<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol. 14 no. 1 3 pp.3064-3072, 1995 <br />  <br />  <br /> The calnexin homologue cnx1+ in <br /> Schizosaccharomyces pombe, essential gene <br /> which complemented soluble ER <br /> domain <br /> Francesco Parlati1l3, Daniel Dignard3,                              GlcNAc2 Mang GlcI intermediate (Ou et al., 1993; <br /> John J.M.Bergeron2                                              Hammond et al., 1994; Le et al., 1994). Recently, it <br /> David Y.Thomas&quot;&apos;2&apos;3&apos;4                                               demonstrated GlcNAc2 Mang GlcI <br />                                                                     oligosaccharide interacts directly calnexin (Ware <br /> &apos;Department Biology 2Department Anatomy Cell          et al., 1995). In addition, tunicamycin (Ou et al., 1993) <br /> Biology, McGill University, Montreal, Quebec 3Genetics Group,   glucosidases I II inhibitors, deoxynojirimycin <br /> Biotechnology Research Institute, National Research Council      castanospermine, (Hammond et al., 1994) inhibit <br /> Canada, 6100 Royalmount Avenue, Montreal, Quebec H4P 2R2, <br /> Canada                                                              association incompletely folded glycoproteins with <br /> 4Corresponding author <br />                                                                     calnexin, supporting observation GlcNAc2 Mang <br />                                                                     GlcI intermediate important calnexin recognition. <br /> Secretory proteins folded action                    Calnexin associated proteins being <br /> number molecular chaperones soon enter                folded action ER molecular chaperones and <br /> the endoplasmic reticulum (ER). In mammalian cells,                 chaperonins. The time association secretory protein <br /> the ER membrane protein calnexin shown                  calnexin reflects time required folding (Ou <br /> be molecular chaperone involved folding                 et al., 1993). Secretory proteins assembled <br /> secretory proteins assembly cell surface              correctly complexes fold correctly <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> receptor complexes. We used PCR strategy <br />                                                                     mutations incorporation amino acid analogues, are <br /> to identify Schizosaccharomyces pombe calnexin <br />                                                                     retained calnexin ER (reviewed Bergeron <br />                                                                     et al., 1994). Thus calnexin function a <br /> homologue, cnxl +. The cnxl + encoded protein, Cnxl,                constituent ER quality control apparatus. <br /> was shown calcium binding type I integral                      In common molecular chaperones, calnexin <br /> membrane glycoprotein. At 5&apos; end, cnxl+ gene                genes identified wide variety eukaryotes <br /> has consensus heat shock transcriptional control ele-               including mammals, nematodes plants (Wada et al., <br /> ments inducible heat shock                    1991; Sulston et al., 1992; Huang et al., 1993). They share <br /> calcium ionophore A23187. Unlike sequence-related               sequence motifs ER lumenal protein calreticulin, <br /> Saccharomyces cerevisiae CNEI gene, S.pombe                     similarly number eukaryotes. <br /> cnxl + gene essential cell viability. The full-             Together appear form gene family possible <br /> <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> Cnxl protein able complement cnxl +               similarity function (Smith Koch, 1989; Wada et al., <br /> gene disruption full-length mammalian calnexin              1991). How proteins perform function a <br /> could not. The ER lumenal domain Cnxl,                     topic considerable interest. <br /> was secreted cells, capable complementing                  We recently cloned CNEJ gene Saccharo- <br /> the cnxl::ura4 lethal <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span>. The equivalent                     myces cerevisiae shares sequence similarity with <br /> region mammalian calnexin shown                      mammalian calnexin calreticulin. The CNEJ gene <br /> possess molecular chaperone activity. It possible                codes integral membrane ER glycoprotein, Cnelp. <br /> that <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> caused absence               Unlike mammalian calnexin, Cnelp does bind calcium <br /> this chaperone activity S.pombe cnxl + gene                  in vitro assay, does cytosolically <br /> disruption.                                                         directed domain carboxy terminus (Parlati et al., <br /> Key words: calnexin/ER/essential gene/molecular chaperone/           1995). We demonstrated CNEJ does effect <br /> Schizosaccharomyces pombe                                           retention mutant proteins ER (Parlati <br />                                                                     et al., 1995) essential viability of <br />                                                                     S.cerevisiae. Thus, properties correspond <br /> Introduction                                                        expected bona fide S.cerevisiae calnexin <br /> In eukaryotes, endoplasmic reticulum (ER) site           homologue. However, Schizosaccharomyces pombe <br /> of folding secretory proteins assembly            cnxl + gene characterized properties <br /> multimeric cell surface receptors. These processes              common mammalian calnexin likely <br /> mediated molecular chaperones (reviewed Bergeron              S.pombe calnexin homologue. <br /> et al., 1994). Some molecular chaperones have <br /> been identified appear present wide variety          Results <br /> of eukaryotic species. One molecular chaperone, the <br /> ER membrane protein calnexin, interacts secretory              Cloning S.pombe member calnexini <br /> glycoproteins soon enter ER (Ou et al.,              calreticulin family, cnx1+ <br /> 1993). Newly synthesized glycoproteins specifically                 We previously used sequence motifs shared <br /> associate calnexin monomeric, incom-            mammalian calnexin ER lumenal protein <br /> pletely folded oligosaccharide modification            calreticulin identify calnexin related gene S.cere- <br />  <br />   3 <br /> 3064 4 Oxford University Press <br />                                                                     Schizosaccharomyces pombe calnexin homologue cnx1+ <br />  <br /> visiae, CNEJ (Parlati et al., 1995). We used similar         heat shock (Figure 3A B). If heat shock treatment <br /> strategy cloning S.pombe member calnexin/     carried extended time, level transcrip- <br /> calreticulin family. Degenerate oligonucleotides coding        tion diminished basal level (Figure 3A B). <br /> for regions amino acid similarity conserved     Other agents known cause stress also <br /> mammalian calnexin calreticulin, YKGK,FWKP             tested effect cnxl&#194;&#177; transcription. Treatment <br /> the repeat motif KPEDWDE (Figure IA) used              glycosylation inhibitor tunicamycin (Figure 3C <br /> prime PCR using S.pombe genomic DNA template            D) 2-deoxyglucose (data shown) no <br /> (Figure iB). <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> organization motifs          effect, treatment calcium ionophore A23187 <br /> calnexin calreticulin, amplified fragment 350 bp     gave marked increase cnxl + transcription (Figure 3C <br /> was expected (Figure 1A) double-stranded (ds) DNA          D). <br /> products approximately size cloned the <br /> plasmid pTZ19R. The DNA sequence               The cnx1+ gene product, Cnxl, calcium- <br /> clones derived amino acid sequence revealed            binding protein <br /> that PCR product shared high amino acid sequence          Calnexin mammalian cells originally described as <br /> similarity mammalian calnexin calreticulin.      major calcium-binding proteins the <br /> Using fragment probe, entire gene cloned     ER membrane (Wada et al., 1991). All calnexins and <br /> and sequenced (Figure IC, Materials methods).          calreticulins tested far bind calcium, for <br /> The cnxl+ genomic sequence identified consensus            S.cerevisiae Cnelp (Parlati et al., 1995). In order to <br /> heat shock elements 5&apos; end (Figure 1C). The coding      determine Cnxl calcium-binding protein, used <br /> sequence does contain introns predicts type I        GST fusion Cnxl expressed Escherichia coli. This <br /> integral membrane protein 560 amino acids        fusion comprised amino acids 23-492 Cnxl, is, it <br /> similar overall arrangement mammalian calnexin. There       excludes predicted signal sequence C-terminal <br /> is predicted N-terminal cleavable signal sequence,        transmembrane cytoplasmic domains Cnxl. Tested <br /> N-glycosylation motif residue 418, membrane-          calcium overlay assay, GST::Cnxl fusion protein <br /> spanning domain proximal cytoplasmic domain (Argos        bound calcium (Figure 4, lane 2) expected, GST <br /> et al., 1982; Kyte Doolittle, 1982; von Heijne, 1986)      alone, phosphorylase B GST::Cnelp (S.cerevisiae) <br /> (Figure 2). S.pombe cnxl + encodes repeats related        (Figure 4, lanes 1, 6 7 respectively) did bind <br /> to motif KPEDWDE, calnexin              calcium assay. The positive controls, calmodu- <br /> suggested high-affinity, low-capacity calcium binding    lin (Figure 4, lane 4), parvalbumin (Figure 4, lane 5) and <br /> sites (Tjoelker et al., 1994). These amino acid repeats        bands corresponding mammalian calnexin pgp <br /> are present Arabidopsis thaliana, mammalian         35 (Figure 4, lane 3) obtained Triton X-1 14-extracted <br /> calnexins mammalian calreticulins. Overall,             canine pancreatic stripped rough microsomes, bound <br /> S.pombe cnxl+ 38% identical A.thaliana calnexin          calcium. Thus, like mammalian calnexin calreticulin, <br /> and 34% identical canine calnexin 25% identical      S.pombe Cnxl calcium-binding protein. <br /> mouse calreticulin, 22% identical S.cerevisiae <br /> CNEJ. Thus, S.pombe cnxl + gene likely       Cnxl integral membrane glycoprotein <br /> a calnexin homologue higher amino acid            Polyclonal antibodies raised E.coli-expressed <br /> identity calnexin calreticulin. Additionally,     GST::Cnxl fusion protein. These antibodies recognize a <br /> encodes C-terminal domain predicted transmem-        protein S.pombe 91 kDa associated a <br /> brane domain, present calnexin genes. We       membrane fraction. To determine Cnxl integral <br /> have looked calnexin calreticulin homologues     membrane protein, S.pombe membrane preparations were <br /> in S.pombe. Hybridization low stringency cnxl +        treated regimen sodium carbonate pH 11.5 or <br /> probe entire genome, presented form         0.5 M NaCl 2.5 M urea did release from <br /> bacteriophage P1 clones, did identify related    membranes; 0.1% SDS 1% Triton X-100 <br /> sequences. We determined cnxl + gene maps        led solubilization 91 kDa protein (Figure <br /> region chromosome I, probes 57bJ2 20h4          SA). Thus, properties S.pombe Cnx 1 correspond to <br /> and location P1 phage clone 3GJOp (data shown)      expected integral membrane protein. Membrane <br /> (Hoheisel et al., 1993). We tentatively conclude cnxl +   association expected calnexin homologue but <br /> is S.pombe calnexin gene.                             calreticulin homologue soluble ER <br />                                                                lumenal protein. <br /> The cnxl+ transcript inducible heat shock                   The apparent molecular mass 91 kDa S.pombe <br /> and calcium ionophore                                        Cnx1 higher predicted Cnxl sequence. <br /> In eukaryotes, consensus sequence heat shock       To confirm predicted site N-glycosylation at <br /> element characterized repeats       residue 418 used (see Figure 2B), S.pombe membrane <br /> sequence nGAAn alternating orientation (Pelham           preparations digested endoglycosidase H <br /> and Bienz, 1982; Amin et al., 1988). Inspection         (endo-H). Immunoblotting antibodies Cnx 1 protein <br /> upstream sequences cnxl+ identified heat shock          revealed increase mobility SDS-PAGE -88 kDa <br /> elements (see Figure IC), correct position    (Figure 5B). This mobility change corresponds that <br /> to act elements controlling transcription cnxl +.        expected single potential site glycosylation (-3 kDa <br /> To test this, exponentially growing S.pombe cells         site) modified addition core sugar <br /> subjected transient heat shock 39&#194;&#176;C RNA        residues (Herscovics Orlean, 1993). From primary <br /> transcripts analysed Northern blots. The cnxl +        sequence, predicted molecular mass non- <br /> transcript induced 1.6-fold transient <br />                                     - <br />                                                                glycosylated protein 63 kDa (Figure 1 C). This <br />  <br />                                                                                                                      3065 <br />   F.Parlati et aL <br />  <br />  <br />  <br />  <br />                              A <br />                              CALNEXIN                             ...   SREIEDPEDQKPEDWDERPKIPDPDAVKPDDWNEDAPAKIPDEEATKPDGWLDDEPEYVPDPDAEKP <br />                              CALRETICUSLIN                        FLPPKKIKDPDAAKPEDWDERAKIDDPTDSKPEDWDK ............EHIPDPDAKKP <br />  <br />  <br />  <br />  <br />                              CALNEXIN <br />                              CALRETICULIN                         EDWDEEMDGEWEPP...........VIQNPEYKGEWKPRQIDNPDYKGTWIHPEIDNPE.. <br />  <br />  <br />                              B                                               PCR S                                                       PCR A <br />  <br />                                                               K         F ED         N D     E                                   V   K G             N       K       F <br />  <br />  <br />                                                          5- AARCCNGARGAYTGGGAYGA 3-                                     3, ATRTTYCCNYTYACCTTYGG 5. <br />  <br />  <br />                                   C <br />  <br />                                                                                              GtTTYGATACCTGACAGTTCGTTATGCGTCAGACTAAAGCATATTCTAATGAT <br />  <br />                                   TGAATAAATCATTgCC                                   CAGTTTACATAATTTACTTTTACCTGTGA                                                                361 <br />  <br />  <br />  <br />  <br />                                   TACTCTrrNIrIACTTAATGGGGnGAACAATTAAAGCCAAATACTTAGTATATTAAAATGCAAACGTTT-ACTGTAAACATGTTACTGC                                                      -271 <br />  <br />  <br />  <br />  <br />                                   AACC7TCATTATAGCAAGACA&apos;rTCGCTCCGTAATTCACAAAGTAAGAAGTAAAATAAATATAAAGATCTAATGCAAATACAACTT1ACCAG                                                    181 <br />  <br />  <br />  <br />  <br />                                   ATGTTTAACATACAG                                ATCATG&apos;=GTCGACTTCTArC=GTAGTGT                                                                    91 <br />  <br />  <br />  <br />  <br />                                   ATCCATCTCATCGATCAAAGCCGTAGTACTACCAAAACACACGCAAAACCGCGCACCGTAGTTATCAACTACACCACCCAACACGTGGAA                                                      1 <br />  <br />  <br />  <br />  <br />                                   ATGAAGTACGGAAAGGTATC&#194;&#182;ITrCTAGCGCTTTSTGTGCCTC=ATATGTTCGGGGATCAC=&apos;GCTGATCCTGAATCCGAACAAGAACCT                                                     90 <br />                                    M     K Y     G   K    V S       F    L A     L   L   C s     L    Y    VSR     GES       L AD        P   E S         E Q             E   P   30 <br />  <br />  <br />                                   TTGGTTTTTAACCCAACTGAAGTGAAAGCGCCTCTTGTTGAGCAGTTTrCAAGGTGCATGGTCTGAACG&#194;&#182;0-GGAT-rCCTTCACATGCTAAA <br />                                    L V      F    N   P T      E     V    K   A   P   L V      E Q     F    Q   G A      W    S E     R   W I     P S             H       A   K   60 <br />  <br />  <br />                                   CGATITTGTAAATGGTATTGAAGAAATGAGCTATGTAGGTGAATGGACTGT7rGAGGAAAGTTCTGGTCCCGGTGCCCTT-AAGGGTGAAGCC                                                  270 <br />                                    S F      V    N   GlIREE              M S     Y VG        S       NTV       E   E S       S   G   P   G AL            K G             E   A   90 <br />  <br />  <br />                                   GGTCTCGTTATGAAAGACGAAGCCGCTCATCATGCTATTTCTCTATGAArGATG.AACCCATAAATGAACCAGAAAAGGA&#194;&#182;TTTGGTAG&#194;&#182;Irr                                                  360 <br />                                    G    L V      M   K    D   E    A     AH H MASIS              Y E       P D     E    P I NE           F E     K       D LV                V   120 <br />  <br />  <br />                                   CAATATGAGGTTAACCCCGAAGAGGGTCTGAACTrGTGGTGGTGCATATITTAAGCTCTrAGCAGAACCCACTCATGGTGAAATGTCAAAC                                                    450 <br />                                    Q Y E VHF ESE G L N C G G AY L K L L A E P T H G E MS N                                                                                       150 <br />  <br />  <br />                                   TCCATCGATTACAGAATCATGTTGGACCCGACAAATGTGGTGTT~AATGACCGTGTrrCA7TTTATCTTCAAGCATAAGAATCCCCTrACG                                                    54 0 <br />                                    ES ID Y R IM F G P D K C G V N D R V N F I F KMH K N PL T                                                                                     180 <br />  <br />  <br />                                   GGTGAGTATTCCGAGAAGCATCTCGACTCTAGACCCGCCAGCCTTTGAAACCTGGTATTACAAACCTTTACACTTTrAATTGTGAAGCCT                                                     630 <br />                                    IG    Y        K     L  D  S     P  A  S  L  L  K  P  G     T  N  L     T  L  I  V  K  P                                                      210 <br />  <br />  <br />  <br />                                                                                                                                                                                  720 <br />                                    D Q      T    F   E VES               N   G D     V   V    E Q     G    S   L   P    Y    DP      F       F   V       L       F       F   V   240 <br />  <br />  <br />                                   GAAATACGATtCCGAAGATATCAAGCCTGCTGATTGGGTTrGACGAACCTGAAATTCCTGACCCCAATGCGGTTAAGCCTGATGATTGG                                                      810 <br />                                    ESI      Y    D       F EDSI          K   P AD        W V     DSEP          ESI      P D      F   NA      V   K       P D             D   W   270 <br />  <br />  <br />                                   GATGAAGACGCTCCCCGTATGATTCCTGATCCTGACGCAGTGAAACCTGAGGATTGG7TAGAGGATGAACCTCTTTACATTCCTGATCCT                                                     900 <br />                                    DRE      D    A   PENSM               PD      P D     AV      K        F ED     W    L ED         E   P LY            SFP D               F   300 <br />  <br />  <br />                                   GAAGCTCAGAAGCCTGAGGATTGGGATGATGAAGAAGATGGTGATTGGATTCCAAGTGAAATCATCAATCCTAAATGCATTGAAGGTGCT                                                     990 <br />                                    S    A   Q K          FED       WND       D E ED          G D          N I PS        ES       SIN     P   K   C           S EG A              330 <br />  <br />  <br />                                   GGCTGTGGTGAATGGAAGCCACCGATGATTCGTAACCCTAACTATCGTGGCCCTTGGTCTCCTCCTATGATCCCTAATCCTGAACTCATT-                                                    1080 <br />                                    G    C   G E      N    K   P PM           SERN        P   N   YER       G   P       NEFFP         MSI     P   N       PEPF                I   360 <br />  <br />  <br />                                                                                                                                                                                  1170 <br />                                    G    E       NY   P    0RK          SFP   N   P   D   Y       D    D    D       P    SN       PFG         LY          G       V       G       3 90 <br />  <br />  <br />                                   G~AACTTTGGACTATGCAGCCCAACATCCGCTTTrAGCAATATCTATGTCGGCCATTCAATTGAGGACGCTGAACGTCTTGGAA.ATGAACT                                                   1260 <br />                                    S    L   N    T   N    Q   P    N     I   S   F   S   N   I   Y    V    G   N   S    S    N   D   A   E   E   L       G       N       N   T   420 <br />  <br />                                   TTCCTACCTAAATTGAAAGCTGAAAGAGAATTACSCCTAAGCAAGAGTCCATGGAAAAGCAATCAATGCATGTITGACGAAGAGTCTAAT                                                     1355 <br />                                    F    L   P    K   L    K   A    S     R   S   L   L   S   K   Q    N    S   M   E    K    Q   S   N   N   V   D       N       S       S   N   450 <br />                                   CAAATTCTTGAGAAGT&apos;=rCTTGACGTATATGATATTVATTAAAGCTAAACTTCCTCCAAATGTTGCCGAAAAAGTTGACTACTATGTTGAA                                                   1440 <br />                                    QIS      L    S   K    F   L    D     V   Y   D   S   I   K   A    K    L   P   F    N    V   A   E   K   V   D       Y       Y       V   E   485 <br />                                   ACACTGGCCTAATGATCATTGCTCTGCCTACGTTATTATGTC                                                                                                     1530 <br />                                    T    I   S    S   T    P   E    I     0   5   A   I   V   A   V    L    G   S   L    T    A   V   S   L   T   C       Y       F       Y   F   515 <br />                                   TTGTCTTCCGCTTTTTCGACAGAGCGGAGGACAAAG7AGAGGCGGC                                                                                                 1620 <br />                                    F    A   S    S   S    P   A    S     L   S   T   G   T   T   S   A     S   K   E    Q I      E   K   P   K   Q       S       T       N   T   540 <br />                                   GAGAAGATAGACGTTTC7rACGCTCCCGAAACTGAATCACCAACTGCGAAGAATGAAGACTAATGAAACTGCTAAAATGTCAAA.AGm                                                   =   1720 <br />                                    S KI D            V    EVYA           F E TES             P TA          K   NE D                                                              560 <br />  <br />  <br />                                        TCTTA=AAGATCGATAGAG7TCAATAACT=CCTTTCCTAAATAATAATCATATAATACCACTATACGCAT1810 <br />                                     A7&apos;&apos;rTGTACAAAGTTGATTGTGGAC7TTGACTGAAGCTCAAAGTAGCGACTCTGTTAAGC=A                         1889 <br />  <br /> Fig. 1. DNA sequence cnxl + gene deduced amino acid sequence. (A) The central domains dog calnexin mouse calreticulin aligned <br /> between amino acids 254-389 185-281 respectively (Wada et al., 199 1; Smith Koch, 1989). Amino acid sequences used design forward and <br /> reverse oligonucleotide primers indicated bold type. (B) Amino acid sequences corresponding nucleotide sequences (PCR S, sense; PCR A, <br /> antisense) used primers PCR amplification yeast (Y = C T; R = A G; N = A, C, G, T). (C) Spombe cnxJ + gene codes 560 amino acid <br /> protein. The predicted protein contains signal sequence amino terminus (italics), single N-glycosylation site N4 18 (CHO), transmembrane <br /> domain (bold type) 48 amino acid predicted cytosolically oriented sequence. Putative heat shock elements indicated (underlined). Nucleotides <br /> matching heat shock element (HSE) consensus sequences marked asterisks. Sequences encoding partial HSE elements denoted by <br /> asterisks, underlined. A putative TATA box shaded. The sequence available GenBank #M98799. <br />  <br /> 3066 <br />                                                                                            :chizosaccnaromyces pombe cainexin ihomologue cnxl+ <br />              A                                                                              106   - <br />  <br />                  3-                                                                          80-                    .. <br />  <br />  <br />  <br />  <br />                  0-                                           A   .   #                    49.5 <br />                  . <br />                       f   Tr         7 r <br />                                      IN                        1VVi&apos; <br />                                                                                                                          _A. <br />                  .3- <br />                                              II <br />                                100     200          300       400         500              27.5 - <br />                                         Residue Number <br />                                                                                            18.5 <br />  <br />              B                                                                                            1     2        3     4     5     6     7 <br />                                                 DS6 <br />                                                                                       Fig. 4. Identification Spombe Cnxl protein calcium binding <br />                                                                                       protein. Lane 1, GST (20 ,ug protein); lane 2, GST::Cnxl <br />                                                 L                                     (S.pombe) fusion (20 jg protein); lane 3, Triton X-1 14 extracted <br />                                                                                       canine pancreas ER (100 ig protein) showing polypeptides of <br />                                                                                       90 kDa (calnexin) 35 kDa (pgp 35) known bind Ca2+ (Wada <br />                                                                                       et al. 1991); lane 4, calmodulin (2 jig protein); lane 5, parvalbumin <br />                                      &gt;- N 411                              ER Lumen   (10 jg protein); lane 6, phosphorylase B (10 ig protein); lane 7, <br />                                                                                       GST::Cnelp (S.cerevisiae) fusion protein (20 jg protein) were <br />                                                                                       electrophoresed 10% SDS-PAGE, transferred nitrocellulose <br />                                                                                       membrane 45Ca binding perforned described previously (Wada <br />                                                                                       et al, 1991). Molecular mass markers indicated left. <br />  <br />                                                                                       anomalous migration SDS-PAGE observed for <br />                                                 D 23 <br />                                                                                       non-glycosylated canine calnexin predicted <br /> Fig. 2. Hydrophobicity plot topology Cnxl. (A) Hydrophobicity                  molecular mass 67 kDa mobility SDS-PAGE <br /> plot. (B) Predicted topology Cnxl showing single predicted                     corresponding 90 kDa (Wada et al., 1991). As with <br /> N-linked glycosylation site, single transmembrane domain proximal                 mammalian calnexin, low pl Cnxl (calculated as <br /> to carboxyl terminus cytoplasmic tail. The predicted signal <br /> sequence cleavage residue aspartate 23 (D 23).                                  pI 4.13) likely responsible discrepancy. <br />                                                                          The cnx1+ gene essential <br />      A                                                 C                Gene disruption performed order determine if <br />                    Heat Shock                   CThemical Stress        cnxl+ essential gene S.pombe. Approximately <br />             200. <br />                                                                         80% coding sequence replaced S.pombe <br />           -6150-                          -a                            ura4+ gene. The sequences flanking cnxl +, is, 500 bp <br />           0                                0 <br />                                                                         5&apos; ORF 1.0 kb 3&apos; retained <br />           L)100-                                          u <br />                                                                         order promote good frequency homologous <br />              50-11                                                      recombination (Figure 6A). A diploid S.pombe strain (see <br />                                                                         Materials methods) transformed linear <br />               0-                                                        construct uracil prototrophs selected. The DNA <br />                    0     15      30 <br />                       Time (min.)                  .R,^9of transformants analysed PCR Southern <br />                                                  &gt; &apos;&gt; ss                blots confirm recombination occurred <br />                                                                         cnxr + locus (see below). Diploid transformants <br />                                                                         heterozygous cnxl + gene disruption grew normally, <br />       B                                   D                             sporulated 17 tetrads dissected. Each tetrad <br />                                                                         gave segregation viable spores grew <br />       28 S <br />                                                                       s form visible colonies apparently non-viable <br />                                                                         spores (Figure 6C). All spores grew uracil <br />       18 -        -    UP _                       -~             - 18 s auxotrophs. Thus cnxr + gene S.pombe essential <br />                                                                         viability. <br />                                                                            Southern blot analysis DNA parental <br />                                                                         diploid strain, ura4+ heterozygous diploid ura- <br />                     1    2     3                    1 2 3               spores probed following probes: (i) entire <br />                                                                         ura4+ gene; (ii) probe corresponding nucleotides -490 <br /> Fig. 3. Induction cnxl + heat shock calIcium ionophore        -1, 5&apos; cnxl + ORF; (iii) NsiIlEcoRV <br /> A23187. Northern blot analysis total S.pombe RNA using probe <br /> from complete ORF cnxl + gene Spon                                                      cnxc <br />                                                                         fragment + (see Figure 6A). These probes used <br /> grown 23&#194;&#176;C shifted 39&apos;C 0 min. (A B, lane 1);         distinguish wild-type recombinant <br /> 15 min (A B, lane 2); 30 min (A B, lane 3) grown      allele. DNA parental diploid strain (Figure 6B, <br /> 30&#194;&#176;C (C D, lane 1) treated tunicamy,cin (C D, lane 2)   lanes 1, 5, 9 13), ura4+ diploid transformant (Figure <br /> or calcium ionophore, A23187 (C D, lan e 3). The results     6B, lanes 2, 6, 10 14), ura- spores (Figure 6B, <br /> three independent experiments evaluated densuitonetry            lanes 3, 4, 7, 8, 11, 12, 15 16) digested with <br /> illustrated panels A C (&#194;&#177;SD). Equal amoi <br /> applied lane gels stained eethidium bromide    EcoRI (Figure 6B, lanes 1-4 9-12) EcoRV (Figure <br /> order verify quantities 28S 18S RNIA.                     6B, lanes 5-8 13-16). Hybridization probe <br />  <br />                                                                                                                                                      3067 <br />  F.Parlati et aL <br />  <br />   A                                                                        A <br />  <br />  <br />                        -           P- <br />                        S       s           S   -&apos;s <br />                                                 P     P     s P <br />  <br />  <br />            106 - <br />  <br />             80             - <br />  <br />  <br />  <br />  <br />            49.5 -- <br />                                                                                   B <br />  <br />        B <br />                                                                                        .. <br />                                                                                                        A <br />                                                                                                            &amp;F <br />  <br />                      ENDO-H        -       +         M.W.                                         -k <br />                                                                                                                           4.: <br />  <br />  <br />                                                 - 106 kDa <br />                                                 - 80 kDa <br />  <br />  <br />                                                 - 49.5 kDa <br />  <br />  <br />                                                                              ,I *? x               mm <br />                                        1   2 <br /> Fig. 5. Cnx integral membrane glycoprotein. (A) Membranes              1.   .9 kD_) <br />                                                                                         _,O <br />                                                                                          _ _    _6a_a <br /> were  prepared extracted 0.1% SDS, 1% Triton X-100, 0.1 M <br /> sodium carbonate, pH 11.5, 0.5 M NaCl (high salt), 2.5 M urea or <br /> Tris-buffered saline, pH 7.5 (mock), followed centrifugation <br /> (30 min X 100 000 g) pellet (P) supernatant (S) fractions <br /> that analysed immunoblotting anti-Cnxl antisera. <br /> Molecular mass markers indicated left. (B) A total <br /> particulate fraction homogenized spheroplasts digested with <br /> endo-H. Analysis immunoblotting revealed change mobility for <br /> Cnxl 91 kDa 88 kDa. After endo-H treatment, minor band <br /> of 74 kDa found, likely originating light 78 kDa <br /> band lane 1. The significance band unknown, we <br /> speculate represent minor Cnxl degradation product. <br /> Molecular mass markers indicated right. <br />  <br /> of entire ura4+ gene resulted 4.7 kb band when <br /> restricted EcoRI (lane 2) bands 2.3 kb                                                          ..   .N   4;   5   6   7 <br />  <br />  <br />  <br /> and 6.0 kb restricted EcoRV (lane 6) the <br /> heterozygous diploid. These bands present the <br /> parental ura- spores. Hybridization probe                   Fig. 6. Gene disruption cnxl+. (A) Schematic representation of <br />                                                                         plasmid pFPP3 containing entire cnxl+ gene pFPP4 <br /> derived PCR fragment 5&apos; cnxl + gene                       containing cnxl::ura4. The indicated restriction sites used to <br /> (nucleotides -490 -1) revealed 2.9 kb band                 evaluate Southern blot analysis presence absence cnxl+ in <br /> wild-type allele 4.7 kb band present                    parental heterozygous diploids, ura- spores. <br /> heterozygous diploid (lane 10). Hybridization                  (B) Analysis cnxl+ disruption Southern blot. DNA the <br />                                                                         parental diploid strain (lanes 1, 5, 9 13), heterozygous diploid <br /> probe encompassing NsiV/EcoRV fragment cnxl +                    strain (lanes 2, 6, 10 14) ura- spores (lanes 3, 4, 7, 8, 11, 12, <br /> resulted 1.7 kb band corresponding wild-type                15 16) restricted EcoRI (lanes 1-4 9-12) or <br /> allele 2.3 kb band present heterozygous               EcoRV (lanes 5-8 13-16) probed with: (i) entire ura4+ <br /> diploid (lane 14) corresponding cnxlJ disrupted allele.              gene (lanes 1-8); (ii) PCR fragment nucleotides -490 -1, 5&apos; <br /> Thus confirmed cnxl + gene disrupted                cnxl+ ORF (lanes 9-12); (iii) NsilIEcoRV fragment cnxl+ <br />                                                                         (lanes 13-16). The ura- spores tetrads, i.e. lanes 3, 7, <br /> by homologous recombination yield cnxl+/cnxl::                     11 15 spore 1, lanes 4, 8, 12 16 spore 2. <br /> ura4 heterozygous diploid. The spores grew                (C) Results tetrad analysis showing viable non- <br /> ura4- contained wild-type cnxl + allele. We                     viable    spores   tetrad. <br />  <br /> 3068 <br />    W <br />       F.X <br />        A <br />  <br />  <br />  <br />  <br />         1-7F <br />       r.X -A <br />             x <br />  <br />  <br />  <br />  <br />              /fF7 <br />                  &quot;ilziziiE <br />  <br />  <br />  <br />  <br />   -%c&apos; x ?:--27172A7/?. <br />  <br />   ;, FL         &apos;cccc2-.YN .xK_ <br />                                                                            L .c                 Corn o. <br />                                                                                                                Schizosaccharomvces nombe calnexin ihomologue cnxl+ <br />  <br />                                                                                                            disruption haploid cells. The full-length Cnx 1 protein <br />                                                                                                            detectable intracellularly membrane fractions <br />                                                                                                            (Figure 7B, lanes 4, 11 12). Plasmid pCNX 524 <br />                                                                                                            expressed Cnxl lacking cytosolic tail produced a <br />                                                                                                            protein lower molecular weight, detect- <br />                                                                                                            able intracellularly membrane fraction (Figure 7B, <br />                                                                                                            lanes 1, 2 3). This truncated protein able to <br />                                                                                                            complement cnxl::ura4 gene disruption haploids. <br />                                                                                                            Plasmid pCNX 484 pCNX 474 expressed Cnxl <br />                                                                                                            protein lacking C-terminal cytosolic domain the <br />                                                                                                            putative transmembrane domain. Remarkably both <br />                                                                                                            complemented lethal phenotype gene disruption. <br />                                                                                                            For constructs, Cnxl protein detectable <br />       B                                                                                                    intracellularly (Figure 7B, lanes 5-10) truncated <br />                                                                                                            Cnxl proteins detected medium (not <br />                                                                                                            shown), retained ER are <br />                     CNX 524          560     CNX 474            ,NX 484        CNX 560         FL 90       secreted (Figure 7A). We attempted complement <br />                                                            T-                  i~1-                        gene disruption strain full-length canine <br />       106-                                                                                                 calnexin expressed plasmid. The mammalian <br />        80 - <br />                                                                                                            protein (FL 90) detected diploid cells heterozygous <br />                                                                                                            cnxl disruption (Figure 7B, lanes 13 14). <br />                                                                                                            However, obtain transformants where <br />   49.5-                                                                                                    mammalian calnexin complemented lethal pheno- <br />                                                                                                            type cnxl::ura4 gene disruption haploid cells. <br />                    1         2   3    .4     -         7             9    10      11   12     13 14 <br />                                                                                                            Discussion <br /> Fig. 7. Complementation cnxl::ura4 disruption haploid cells. <br /> (A) Constructs used complement cnxl::ura4 disruption                                            It established calnexin acts molecular <br /> depicted lumenal (dashed), transmembrane (filled) cytosolic                                       chaperone recognizes secretory glycoproteins and <br /> tails (open) rectangles. Their ability complement (Comp)                                            unassembled multimeric cell surface receptors (see <br /> cnxl::ura4 disruption intracellular (I) extracellular                                 Bergeron et al., 1994). In mammalian cells, function <br /> (E) location (Loc) noted. pCNX560 encodes entire Cnxl <br /> protein; pCNX 524, pCNX 484 pCNX 474 encodes Cnxl                                                      molecular chaperone largely elucidated <br /> truncated amino acid 524, 484 474 respectively. pFL90 encodes                                       vivo studies showing transient association of <br /> the full-length canine calnexin gene. (B) Immunoblot detection                                          incompletely folded secretory proteins calnexin. The <br /> Cnxl canine calnexin S.pombe cells. Protein (10 tg)                                          molecular mechanism recognition proteins by <br /> membrane fraction different cnxl::ura4 haploids <br /> complemented pCNX 560 (lanes 4, 11 12), pCNX 524                                           calnexin remains uncertain. It suggested that <br /> (lanes 1, 2 3), pCNX 484 (lanes 8, 9 10) pCNX 474                                              multiple mechanisms including N-linked glycosylation <br /> (lanes 5, 6 7) electrophoresed 8% SDS-PAGE,                                                 and/or direct binding peptide motifs incompletely <br /> transferred nitrocellulose blotted anti-Cnxl antisera.                                         folded proteins involved (Bergeron et al., 1994; Ware <br /> Canine calnexin detected membrane fraction diploid cells                                       et al., 1995). It certain incompletely folded proteins <br /> heterozygous cnxl deletion transformed pFL90 (lanes 13 <br /> and 14). Membrane proteins (10 ,ug) treated                                                  recognized. For example, transferrin expressed in <br /> (lanes 1-12), immunoblotted anti-canine calnexin antisera.                                        human HepG2 cells released calnexin its <br /> Molecular mass markers (kDa) indicated left.                                                    disulphide bonds form (Ou et al., 1993). Also, is <br />                                                                                                            known calnexin recognize free oligosaccharide <br /> conclude                          cnxl <br />                                                   + <br />                                                       gene            essential        viability.   GlcNAc2 Man9 Glc1 (Ware et al., 1995) affinity <br /> Upon microscopic examination apparently non-                                                        oligosaccharide polypeptide probably <br /> viable spores observed did germinate                                                      higher (Arunachalam Cresswell, 1995). Less known <br /> divided produce microcolonies 20-50 cells. There-                                                    features calnexin necessary its <br /> fore, cnxl + essential spore germination                                                function. Its apparent ubiquity mammalian cells makes <br /> results suggest cell divisions, Cnx                                                     experimental alteration function difficult, the <br /> becomes diluted cells eventually stop dividing.                                                        characterization yeast calnexin homologue be <br />                                                                                                            advantage. We previously cloned CNEJ from <br /> Complementation                                cnxl::ura4 gene                                      S.cerevisiae similar strategy described here. <br /> disruption haploid cells                                                                                We shown CNEJ sequence similarities with <br /> Since cnxl::ura4 gene disruption lethal haploids,                                                calnexin/calreticulin family shares the <br /> we determine region molecule                                                        properties calnexins does bind calcium have <br /> essential growth. The cnxl::ura4 gene disruption strain                                                C-terminal cytoplasmic domain (Parlati et al., 1995). <br /> was transformed series deletion plasmids lacking                                                 The S.pombe cnxl + gene described sequence <br /> fragments cnxl + C-terminus. Their ability com-                                                  similarity properties common calreticulin/ <br /> plement lethal cnxl::ura4 gene disruption,                                                  calnexin, characteristics argue that <br /> the location Cnx protein determined (Figure                                                     S.pombe cnxl + codes homologue calnexin whereas <br /> 7A). Plasmid pCNX 560, expressing full-length Cnx 1,                                                       identity S.cerevisiae CNEJ certain. <br /> complemented lethal phenotype cnxl::ura4 gene                                                      Over complete sequence, Cnxl 38% identical to <br />  <br />                                                                                                                                                                   3069 <br />  F.Parlati et aL <br />  <br />                    A <br />                    S.poznbe Cnxl               249   KPADWVD         266   KPDDWDE         285   KPEDWLE         304   KPEDWDD <br />                    A. thaliana calnexin        230   KPEDWDE         247   KPEDWDE         266   KPEGWLD         285   KPEDWDD <br />                    Canine calnexin             284   KPEDWDE         301   KPDDWNE         320   KPDGWLD         339   KPEDWDE <br />                    S.cerevisiae Cnelp          255   KPHDWDD         272   KLSDRDE         291   EPPEWNS         310   KPSWWKE <br />                    Mouse calreticulin          215   KPEDWDE         232   KPEDWDK                               249   KPEDWDE <br />  <br />  <br />  <br />  <br />                    B <br />                    S.pombe Cnxl                132 CGGAYLKLL               156 IMFGPDKCG              324 PKCIEGAGCG <br />                    A. thaliana calnexin        108 CGGAYLKYL               136 IMFGPDKCG              305 PKCEAAPGCG <br />                    Canine calnexin             161 CGGAYVKLL               188 IMFGPDKCG              358 PKCESAPGCG <br />                    S. cerevisiae Cnelp         124 CGGAFIKLM               154 LVFGPDYCA              330 PLCTAERGCG <br />                    Mouse calreticulin          105 CGGGYVKLF               130 IMFGPDICG <br />  <br /> Fig. 8. Comparison predicted high affinity calcium binding motifs (A) cysteine motifs (B) S.pombe Cnxl, A.thaliana calnexin, canine <br /> calnexin, S.cerevisiae Cnelp mouse calreticulin proteins. Amino acid sequences aligned using GeneWorks program. The numbers <br /> denote place original sequence residue motif. <br />  <br /> canine calnexin 25% identical mouse calreticulin,                     suggested N-linked glycosylation specificity of <br /> compared overall identity Cnelp 24%                       calnexin (Ou et al., 1993), is, inhibition the <br /> canine calnexin 21% mouse calreticulin. As                      oligosaccharide glycosylation precursor sensed by <br /> all mammalian plant calnexins, S.pombe Cnxl                          calnexin. In addition, result suggest Cnxl <br /> has repeats related sequence KPEDWDE,                        protein role Ca2+ regulation sequestration. A <br /> S.cerevisiae Cnelp single related motif KPHDWDD                        heat shock consensus sequence identified the <br /> with conserved variations motif                    S.cerevisiae CNEJ gene, unable find <br /> equivalent positions (Figure 8A). This observation                        experimental conditions altered transcription <br /> noteworthy repeat motif suggested                     (Parlati et al., 1995). <br /> previous studies represent high-affinity calcium                         The essential nature S.pombe cnxl+ gene in <br /> binding domain calnexin calreticulin                           contrast non-essential nature S.cerevisiae <br /> (Michalak et al., 1992; Tjoelker et al., 1994). As shown                     CNEJ gene (Parlati et al., 1995). In S.cerevisiae cnel <br /> in Figure 4, Cnelp does bind calcium, Cnxl                       deleted strain small effect observed the <br /> does. Mammalian calnexins S.pombe Cnxl possess                           function temperature-sensitive mutant a- <br /> four cysteine residues remarkably conserved                         pheromone receptor (ste2-3) non-permissive <br /> amongst calnexins including Cnelp,                      temperature. Furthermore, increase secretion of <br /> also conserved calreticulin (Figure 8B).                                  heterologously expressed mammalian ctl-antitrypsin was <br />    The diagnostic difference calnexins cal-                      observed (Parlati et al., 1995). To explain small <br /> reticulins type I ER membrane                         effects speculated systems for <br /> proteins, calreticulins ER lumenal proteins                   protein folding ER S.cerevisiae. Despite search <br /> carboxy terminal KDEL retrieval signals (Pelham, 1990;                       low stringency hybridization S.cerevisiae <br /> Michalak et al., 1992). For S.cerevisiae shown                       S.pombe genomes, able demon- <br /> that Cnelp integral membrane glycoprotein                       strate presence related calnexins calreticulins <br /> localized ER membrane (Parlati et al., 1995). We                      (Parlati et al., 1995). <br /> have shown S.pombe Cnxl integral mem-                           Complementation lethal cnxl::ura4 disruption <br /> brane glycoprotein. Although demon-                          strain shown C-terminal molecule <br /> strated ER localization, tight banding protein                essential viability. Indeed deletion the <br /> on SDS-PAGE consistent ER, Golgi                             transmembrane predicted cytosolic domain <br /> glycosylation. Hence expect Cnxl localized                       led soluble truncated form Cnxl protein that <br /> the ER. There obvious ER retention motif present                       secreted complemented. We able, <br /> in Cnxl cytosolic tail ER retention/retrieval                    however, detect truncated Cnxl protein intracellularly. <br /> motif mammalian calnexin (an extreme carboxy-                            Our previous studies pointed lumenal domain <br /> terminal RKPRRE motif), present Cnx 1                              calnexin important function molecular <br /> (Rajogopalan et al., 1994).                                                  chaperone mammalian cells (Ou et al., 1993), we <br />    S.pombe cnxl+ inducible heat shock                           speculate function carried ER <br /> treatment calcium ionophore A23187. This                     lumenal soluble Cnx protein ER. A <br /> feature described mammalian calnexin                        similar result observed S.cerevisiae for <br /> (Bergeron et al., 1994) known                  ER lumenal molecular chaperone BiP. When the <br /> ER chaperones BiP, GRP 94 PDI                            carboxy-terminal tetrapeptide ER retrieval motif, HDEL, <br /> mammalian cells yeasts (Normington et al., 1989;                         deleted essential gene BiP, protein is <br /> Rose et al., 1989; Mori et al., 1992; Pidoux Armstrong,                  secreted. However, small BiP remains <br /> 1992). The difference mRNA abundance cnxl+                          intracellular cells viable (Hardwick et al., <br /> response A23187 tunicamycin consistent                     1990). By analogy, propose sufficient levels of <br />  <br /> 3070 <br />                                                                                   Schizosaccharomyces pombe calnexin homologue cnx1+ <br />  <br /> truncated soluble Cnx 1 remain ER order                         Membrane extraction endo-H digestion <br /> perform essential function. We unable prove                       S.pombe membranes (ML fraction) prepared treated essentially <br />                                                                              described (Parlati et al., 1995). Membranes mixed 1 vol <br /> definitively lethal phenotype cnxl::ura4                     I M NaCl, 0.2 M sodium carbonate, pH 11.5, 5 M urea, 2% <br /> disruption lack molecular chaperone                         Triton X-l00 0.2% SDS subsequently analysed described <br /> function. However, studies lumenal domain                        (Feldheim et al., 1992). The Cnxl antiserum used 1:4000 <br /> the mammalian calnexin shown interact                       dilution. Endo-H digestions performed incubating 20 jg ML <br />                                                                              fraction proteins 100 mM sodium acetate, pH 4.9, 150 mM NaCl, <br /> with secretory glycoproteins (W.-J.Ou et al., unpublished                    10 mM DTT, 1% Triton X-100, 0.1% SDS + inhibitors (1 mM PMSF, <br /> results). Thus, likely domain molecular                  1 mg/ml pepstatin, 1 mg/ml leupeptin 1 mg/ml aprotinin) and <br /> chaperone properties functions.                   incubating 2 ,ug endo-H 16 h 37&#194;&#176;C. <br />    The lack complementation cnxl::ura4 dis- <br /> ruptant mammalian (canine) calnexin surprising. We                     45Ca overlay <br />                                                                              The Spombe GST::Cnx 1 fusion protein (residues 23-492) expressed <br /> have shown protein S.pombe                        Ecoli purified (Smith Johnson, 1988). Samples electro- <br /> membrane-localized. The ER lumenal region cnxl+                           phoresed SDS-PAGE prepared 45Ca overlay exactly as <br /> (amino acids 1-474) complement cnxl:: ura4                      previously described. Control experiments carried canine <br /> disruption 50% sequence similarity equivalent                   pancreatic ER membranes obtained extraction Triton X- 1 14 (Wada <br /> region mammalian calnexin. Our observation                       et al., 1991), Cnelp previously described (Parlati et al., 1995). <br /> lumenal domain Cnx 1 complement disrup-                     Gene disruption <br /> tion secreted, argues mammalian                    A 4.3 kb PstI DNA fragment containing complete cnxl+ gene was <br /> calnexin ER retained, present                      cloned pTZ19R creating plasmid pFPP3. A PCR fragment containing <br /> perform ER function. Although expect Cnx 1                       nucleotides -490 -1 (5&apos; coding sequence) amplified, cut <br /> and mammalian calnexin recognize secretory                      SpeI cloned SpeI/SmaI site pBluescript KS+ creating <br />                                                                              plasmid pFPP3. 1. The NsilVEcoRV fragment (3&apos; coding sequence) <br /> glycoprotein substrates, sequence differences                        pFPP3 cut, purified inserted EcoRV site of <br /> these proteins value order map regions                      pFPP3. 1 creating plasmid pFPP3.2. A 1.8 kb HindIII fragment containing <br /> essential molecular chaperone function.                                  ura4+ gene (Grimm et al., 1988) blunt-ended inserted into <br />                                                                              similarly treated EcoRI site pFPP3.2 creating plasmid pFPP4, <br />                                                                              essentially disrupting cnxl+ gene. Plasmid pFPP4 digested with <br /> Materials methods                                                        XbaI, HindlIl (which cleave insert) XmnI <br />                                                                              (which cuts vector) -5 jg linear fragment containing <br /> Strains media                                                            disrupted gene isolated purified GeneClean method. <br /> S.pombe strains Q358 (h- leul-32 ura4-D18 ade6-M210) Q359 (h+            This fragment used disrupt diploid strain h- leul-32 ura4- <br /> leul-32 ura4-DJ8 ade6-M216) used. The strains grown             D18 ade6-M2JO/h+ leul-32 ura4-D18 ade6-M216 uracil prototrophs <br /> 30&#194;&#176;C YPD medium EMM media supplemented <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                   selected. Colonies grouped pools 10 chromosomal <br /> requirements previously described (Moreno et al., 1991). Ecoli strain     DNA prepared (Moreno et al., 1991). PCR used screen for <br /> MC1061 (Maniatis et al., 1982) used.                                     homologous recombinants using sense oligonucleotide coding for <br />                                                                              nucleotides -510 -495 (5&apos; cnxJl ORF) antisense 20 bp <br /> Cloning S.pombe cnx1+                                                     oligonucleotide ura4+ gene. Among 20 pools screened, <br />                                                                              15 positive expected 550 bp PCR product. Four pools were <br /> To amplify calnexin/calreticulin gene equivalent Spombe,        chosen PCR reaction repeated constituent colony. <br /> degenerate oligonucleotides, PCR primer S (5&apos; AARCCNGARGAYT-                 We colonies positive chromosomal DNA was <br /> GGGAYGA 3&apos;) PCR primer A (3&apos; ATRTTYCCNYTYACCTTYGG                        prepared each. Southem blots performed digesting these <br /> 5&apos;) used amplify genomic DNA. Amplification reactions           DNAs EcoRI EcoRV, followed electrophoresis 1% <br /> performed previously described (Parlati et al., 1995). The products       agarose gels transfer nylon membranes. The 32P-labelled probes <br /> of PCR reaction electrophoresed 2% agarose gel             used were: (i) entire ura4+ gene hybridized genomic EcoRI and <br /> stained ethidium bromide. A band -350 bp purified            EcoRV digests; (ii) PCR fragment -490 -1 hybridized a <br /> cloned SmaI site pTZ-19R (Pharmacia). The fragment           genomic EcoRI digest; (iii) Nsil/EcoRV fragment cnxl + hybridized <br /> sequenced encode peptide high amino acid             genomic EcoRV digest. <br /> sequence similarity canine calnexin. The PCR fragment 32P- <br /> radiolabelled using QuickPrime method (Pharmacia) sub- <br /> sequently used probe clone full-length calnexin Spombe        Complementation cnxl gene disruption <br /> genomic library plasmid pWH5 (Wright et al., 1986). This gene         Sequences encoding full-length cnxl +, cnxl + terminated amino <br /> sequenced standard procedures sequence cnxl +              acids 524, 484 474 amplified using PC <br /> </body></html>