<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>512                                                                              Biophysical Journal   Volume 105    July 2013     512&#226;&#8364;&#8220;522 <br />  <br />  <br /> Quantitative Assessment Thermodynamic Constraints Solution <br /> Space Genome-Scale Metabolic Models <br />  <br /> Joshua J. Hamilton, Vivek Dwivedi, Jennifer L. Reed* <br /> Department Chemical Biological Engineering, University Wisconsin-Madison, Madison, Wisconsin <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> Constraint-based methods provide powerful computational techniques allow understanding prediction of <br /> cellular behavior. These methods rely physiochemical constraints eliminate infeasible behaviors space of <br /> available behaviors. One constraint thermodynamic feasibility, requirement intracellular flux distributions <br /> obey laws thermodynamics. The past decade seen constraint-based methods interpret constraint <br /> in different ways, including limited small networks, rely predefined reaction directions, and/or neglect the <br /> relationship reaction free energies metabolite concentrations. In work, utilize approach, <br /> thermodynamics-based metabolic flux analysis (TMFA), make genome-scale, quantitative predictions metabolite con- <br /> centrations reaction free energies absence prior knowledge reaction directions, accounting uncertainties <br /> in thermodynamic estimates. We applied TMFA genome-scale network reconstruction Escherichia coli examined the <br /> effect thermodynamic constraints flux space. We assessed predictive performance TMFA gene <br /> essentiality quantitative metabolomics data, aerobic anaerobic, optimal suboptimal growth condi- <br /> tions. Based results, propose TMFA useful tool validating phenotypes generating hypotheses, and <br /> that additional types data constraints improve predictions metabolite concentrations. <br />  <br />  <br /> INTRODUCTION <br /> Genome-scale network reconstructions provide concise                    (7), loops (8). Because constraints nonlinear, <br /> mathematical representations organism&#226;&#8364;&#8482;s biochemical               EBA restricted small models, scalable algo- <br /> capabilities, serve platform constraint-based              rithm recently proposed (9). The scientists <br /> techniques used understanding predicting            developed method compute eliminate closed <br /> cellular behavior (1,2). The predictive accuracy                     cycles flux-balance models (10,11) based solely on <br /> constraint-based methods depends degree                 stoichiometry, technique remains computa- <br /> constraints eliminate physiochemically biologically                 tionally demanding genome-scale networks. Recently, <br /> infeasible behaviors.                                                   scalable, mixed-integer approach eliminating closed <br />    Flux-balance analysis (FBA) (3) commonly employed                 cycles developed (8). <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> predict state network identifying steady-               Due limited availability free energy data for <br /> state flux distribution maximizing cellular growth,               reactions metabolites (12,13), above <br /> also satisfying mass-balance enzyme capacity con-                   approaches directly account relationships <br /> straints. Reaction directionality typically assigned based           Dr G, metabolite concentrations, free energies <br /> on enzyme assays biological considerations (e.g.,                 formation (Df G). Fortunately, group contribution <br /> free ATP synthesis), consideration given thermo-             methods (GCMs) (14&#226;&#8364;&#8220;16) developed estimate <br /> dynamic feasibility. The second law thermodynamics                   free energies metabolites reactions data are <br /> states negative Gibbs energy reaction (Dr G)               unavailable. A recent model Escherichia coli used one <br /> needed drive forward reaction flux, v, leading              GCM assign reaction directionalities based on <br /> thermodynamic feasibility constraint v $ Dr G &gt; 0                   thermodynamic feasibility (16,17). In approaches, <br /> nonzero v.                                                              experimentally measured thermodynamic data been <br />    The attempt enforce thermodynamic feasibility            combined heuristics and/or group contribution data to <br /> FBA energy balance analysis (EBA) (4,5), incor-               define feasible reaction directions E. coli (18,19). How- <br /> porated nonlinear constraints chemical potentials               ever, approaches (17&#226;&#8364;&#8220;19) neglect thermodynamic <br /> FBA eliminate closed cycles. Closed cycles sets               interactions reactions network arise <br /> reactions overall thermodynamic driving force             shared metabolites. As result, directionality <br /> is zero, net flux occur (e.g., A/              reaction assigned independently reaction <br /> B / C / A). These closed cycles variously                     directions network. For example, reactions <br /> referred Type III pathways (6), internal flux cycles              feasible forward reverse directions, <br />                                                                         shared metabolite, pair reactions must <br />                                                                         proceed direction. These approaches fail to <br /> Submitted October 5, 2012, accepted publication June 5, 2013. <br />                                                                         capture type thermodynamic coupling between <br /> *Correspondence: reed@engr.wisc.edu <br />                                                                         reactions. <br /> Editor: Daniel Beard. <br />   2013 Biophysical Society <br /> 0006-3495/13/07/0512/11 $2.00                                                                     http://dx.doi.org/10.1016/j.bpj.2013.06.011 <br />  Quantitative Assessment TMFA                                                                                                                                  513 <br />  <br />    GCMs used approaches capture                                                 vmin %vj %vmax cj;Jirrev :                                (4) <br /> thermodynamic interactions including metabolite con- <br /> centrations variables. EBA extended predict                      Because network steady state, net production metabolites is <br /> intracellular metabolite concentrations small network                      zero. Each reaction constrained flux appropriate <br /> (20), mixed-integer approaches                           range given enzyme capacities, reactions assumed be <br />                                                                                 irreversible (j &#203;&#8250; Jirrev). The limits reactions set vmax &#194;&#188; <br /> developed, thermodynamic constraints                               1000 mmol/gDW/h vmin &#194;&#188;  1000 mmol/gDW/h, measured <br /> imposed predefined reaction directions. NET                           fluxes (e.g., carbon uptake rates). <br /> analysis (21) integrates quantitative metabolomics data                            Previous implementations TMFA (7,23) imposed thermody- <br /> with thermodynamic constraints predict feasible free                         namic constraints FBA, allowing constraints to <br /> energy ranges reactions network. Another                         restrict reaction directions; however, reaction deemed irreversible <br />                                                                                 FBA reversible indicated thermodynamic <br /> method, thermodynamics-based metabolic flux analysis                            constraints. In contrast, implementation makes assumptions about <br /> (TMFA) (7), extends FBA thermodynamic constraints,                         reaction (ir)reversibility, allowing thermodynamic constraints decide <br /> enabling quantitative prediction feasible ranges                      directions reactions. We replaced Eqs. 3 4 with <br /> metabolite concentrations reaction free energies, <br />                                                                                                           vmin %vj %vmax cj&#203;&#8250;J                                    (5) <br /> without relying metabolomic data. However, of <br /> these methods have, date, relied prior knowledge of <br />                                                                                 rely thermodynamic constraints determine reaction <br /> the reversibility directionality reactions (7,21&#226;&#8364;&#8220;23),                     directions. <br /> thereby restricting predictive capabilities. <br />    In work, examine extent thermody- <br /> namics-based flux-balance methods make genome-                              Calculating free energies reaction <br /> scale, quantitative predictions, absence outside                      Enforcing thermodynamic constraints requires knowledge standard <br />                                                                                                                                 0 <br /> information flux directions, considering presence                   transformed Gibbs energy reaction (Dr G 0) reactions the <br /> and absence uncertainty thermodynamic estimates. To                       model. Due paucity experimental data, group contribution methods <br />                                                                                 (14,15) used provide estimates uncertainties standard trans- <br /> this end, applied TMFA iJR904 model E. coli                                                                  0 <br />                                                                                 formed Gibbs energy formation (Df G 0) metabolites reaction <br /> (24). This model used group contribution esti-                            0 <br />                                                                                         0 <br />                                                                                 (Dr G ) reactions. <br />                                                                                                                  0 <br /> mates available higher fraction metabolites                       GCMs assume Df G 0 metabolite linear combination <br /> in iJR904 model newer models (16). We assessed                      formation energies constituent molecular substructures <br /> the predictive performance TMFA number                          (or groups), k, <br /> large-scale datasets (22,25&#226;&#8364;&#8220;27), encompassing metabolite                                                        0 <br />                                                                                                                         X                      0 <br />                                                                                                        Df G 0 &#194;&#188;                ni;k Dgr Gk0 ;                  (6) <br /> concentrations gene deletion phenotypes,                                                                  k <br /> aerobic anaerobic, optimal suboptimal growth <br />                                                                                               0                                                                   0 <br /> conditions. Through analysis, highlight impor-                      Dgr Gk 0 estimated contribution group k overall Df G 0, <br /> tance quantitative concentration measurements                            ni,k number groups k molecular structure compound i. <br />                                                                                 We used software implementation GCM Jankowski et al. (16), <br /> thermodynamic coupling achieving physiologically real-                       Henry et al. (28), Finley et al. (29) obtain estimates uncertainties <br /> istic predictions growth rates flux distributions. We                             0           0 <br />                                                                                 Df G 0 Dr G 0 metabolites reactions iJR904 network <br /> were able generate hypotheses regarding metabolite                      (see Supporting Methods Table S1 Table S2 Supporting <br /> concentrations thermodynamic bottlenecks,                            Material). <br />                                                                                                                                    0 <br />  <br /> discuss additional types data constraints                          The GCM method returns estimates Df G 0 predominant ionic <br />                                                                                 species biochemical standard state: pH 7, zero ionic strength, temper- <br /> improve predictions metabolite concentrations.                               ature 298 K. Simulations performed conditions 298 K zero <br />                                                                                 ionic strength, pH 7 (7.4) intra- (extra)cellular metabolites, using <br />                                                                                 ionic species represented iJR904 model stoichiometric matrix. <br /> MATERIALS AND METHODS                                                           These differences pH major ionic species required adjusting the <br />                                                                                                          0 <br />                                                                                 GCM estimates Df G 0 (see Supporting Material). <br />                                                                                                                       0                       0 <br /> Overview relationship previous                                              Using new estimates Df G 0, calculated Dr G 0from stoichi- <br /> implementations                                                                 ometry reaction, <br />                                                                                                                     0 <br />                                                                                                                         X                  0 <br /> Given stoichiometric matrix S set reactions J, flux-balance analysis                           Dr G j 0 &#194;&#188;               Si;j Df Gi0 ;                   (7) <br /> (FBA) seeks steady-state flux distribution (v) maximizing flux                                             i <br /> the biomass reaction (vBM), satisfying mass-balance enzyme <br /> capacity constraints individual reactions, j:                               Si, j stoichiometric coefficient metabolite reaction j. We <br />                                                                                                                                                         0 <br />                                                                                 calculated transformed Gibbs energy reaction (Dr G ) a <br />                                 max vBM ;                                (1)    function metabolite concentration, xi, <br />                                                                                                   0         0 <br />                                                                                                                          X                             0 <br />                                                                                            Dr Gj &#194;&#188; Dr Gj0 &#195;&#190; RT                     Si;j ln&#195;&#176;xi &#195;&#382; &#195;&#190; Dt Gj0 ;       (8) <br />                              s:t: S$v &#194;&#188; 0;                               (2)                                                 i <br />                                                                                                                                                              0 <br />                                                                                 R gas constant, T temperature (298 K), Dt G 0 reflects <br />                         0%vj %vmax cj&#203;&#8250;Jirrev ;                           (3)                              0 <br />                                                                                 contribution Dr G transport metabolites the <br />  <br />                                                                                                                         Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  514                                                                                                                                                    Hamilton et al. <br />  <br />                                                                               0 <br /> membrane. Following established derivation (7), calculated Dt G 0                                             vj R0 cj&#203;&#8250;Jfor ;                              (15) <br /> a function electrochemical potential (DJ) pH gradient (DpH) <br /> across membrane, <br />                        0 <br />                                                                                                                      dj &#194;&#188; 0 cj&#203;&#8250;Jrev ;                               (16) <br />                  Dt Gj0 &#194;&#188; cj FDJ   2:3 hj RTDpH;                                  (9) <br />                                                                                                                       vj %0 cj&#203;&#8250;Jrev ;                               (17) <br /> where F Faraday&#226;&#8364;&#8482;s constant, cj net charge transported outside <br /> to inside cell, hj number protons transported the <br /> membrane (see Table S3 details). Our constraints intra- extracel-                                          vj &#194;&#188; 0 cj&#203;&#8250;Jbl :                               (18) <br /> lular proton concentrations resulted values  130 mV DJ, 0.4 <br /> for DpH (7,30). <br />                                                                                         For reactions, flux maximization using RTMFA computationally <br />    We performed simulations uncertainties esti- <br />                  0                                                      0               intensive, FVA simulations performed time limit 5 min. <br /> mates Dgr G 0 group. In TMFA, fixed values Dgr G 0                 As result, possibility true flux ranges larger reported <br />                                    0 <br />                                      0 <br /> their estimated values (Dgr Gest ) given group contribution <br />                                                                                         (see Results). <br /> method, <br />                                       0           0 <br />                                Dgr Gk0 &#194;&#188; Dgr Gk;est <br />                                                0 <br />                                                     :                         (10) <br />                                                                                         Differences <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>phenotype</span>: CONGA <br />                                                                   0 <br /> Whereas relaxed TMFA (RTMFA), allowed Dgr G 0 group                    We used algorithm previously developed, CONGA (33), identify <br /> vary 95% confidence interval, determined standard error            single, double, triple gene deletions predicted lethal FBA but <br /> (SE) reported GCM software (i.e., SEDgr G0 0 ):                                  TMFA (or RTMFA). CONGA employs bilevel optimization problem <br />                                                           k;est <br />                                                                                         identify gene deletion strategies maximizing difference in <br />       00                                  0           0 <br /> Dgr Gk;est  2SEDgr G0 0 %Dgr Gk0 %Dgr Gk;est <br />                                         0 <br />                                              &#195;&#190;2SEDgr G0 0 : (11)                        maximum biomass flux different models, FBA TMFA <br />                            k;est                                      k;est             (or RTMFA). Because maximum possible difference biomass flux <br />                                                                                         occurs model zero biomass flux, CONGA identifies gene <br /> See Supporting Methods Supporting Material additional                    deletions lethal model. Additional details be <br /> details.                                                                                Supporting Methods Supporting Material the <br />                                                                                         original work (33). <br />  <br /> Enforcing thermodynamic consistency <br />                                                                                         Synthetic lethals phenotype correction <br /> Thermodynamic consistency requires reaction fluxes con- <br />                                      0 <br /> sistent predicted values Dr G (i.e., v $ Dr G &lt; 0). We employed                 Once identified gene deletions lethal FBA model, we <br /> a mixed-integer approach enforce requirement, binary                 employed synthetic lethals (SL) Finder (34) identify reaction(s) <br /> variable d indicates reaction operating forward (d &#194;&#188; 1)                  responsible difference predicted phenotype. SL Finder employs <br /> or reverse (d &#194;&#188; 0) direction. We added following equations                  bilevel optimization problem identify reaction deletion strategies that <br /> our model:                                                                              minimize maximum biomass flux network. Because the <br />                                                                                         lowest maximum biomass zero, SL Finder finds lethal reaction <br />                            &#195;&#176;1   d&#195;&#382;vmin %v%dvmax ;                             (12)      deletions. Additional details Supporting Methods in <br />                                                                                         Supporting Material original work (34). <br />                                               0                                            For gene deletions predicted lethal FBA, TMFA pre- <br />                    Md &#195;&#190; &#206;&#181;%Dr G %M&#195;&#176;1   d&#195;&#382;   &#206;&#181;:                                 (13)      diction disagreed experimental observation. This suggested the <br />                                                                                     0 <br />                                                                                         SL reaction did operate vivo direction predicted FBA. In <br /> Here Eq. 13 big-M constraint (31) M upper limit Dr G             instances, developed constraint metabolite concentrations <br /> (300 kcal/mol) &#206;&#181; small nonzero number (10 6). Equations 1, 2, 5&#226;&#8364;&#8220;9,             prevented SL reaction operating TMFA-predicted <br /> and 12&#226;&#8364;&#8220;13 used TMFA equations 1, 2, 5&#226;&#8364;&#8220;8, 10&#226;&#8364;&#8220;13                     direction, correcting phenotype. We refer constraints <br /> used RTMFA.                                                                          phenotype-correction constraints. Additional details in <br />                                                                                         Supporting Methods Supporting Material. <br />  <br /> Flux thermodynamic variability analysis <br />                                                                                         Simulation conditions <br /> We performed flux thermodynamic variability analysis a <br /> variety conditions identify thermodynamically feasible flux                   All simulations performed using CPLEX 12 (IBM, Armonk, NY) <br /> metabolite concentration ranges. In flux variability analysis (FVA) (32),               accessed General Algebraic Modeling System, Ver. 23.3.3 (GAMS; <br /> the flux v reaction minimized maximized subject                  GAMS Development Corporation, Washington, DC). Simulations per- <br /> the constraints TMFA. In thermodynamic variability analysis (TVA)                    formed Red Hat Enterprise Linux server 2.66 GHz Intel Xeon pro- <br />                                                           0 <br /> (7), concentration metabolite (or Dr G reaction)                 cessors 8 GB RAM. The TMFA formulation solves seconds, <br /> minimized maximized subject constraints TMFA. Additional                  RTMFA formulation takes ~1 h prove global optimality. <br /> details Supporting Methods Supporting <br /> Material. <br />    Using FVA results, defined sets reactions operate            Sources experimental data experimental <br /> in forward (Jfor) reverse (Jrev) directions, blocked (Jbl) entirely.      methods <br /> By imposing following constraints model, able reduce <br /> simulation times order magnitude:                                         Details experimental datasets methods used validate our <br />                                                                                         TMFA predictions given Supporting Methods Supporting <br />                                    dj &#194;&#188; 1 cj&#203;&#8250;Jfor ;                           (14)      Material. <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                          515 <br />  <br /> RESULTS                                                          A                                   asp      2H+ <br />  <br /> Optimization aerobic growth glucose using <br /> TMFA                                                                                    ABC-type              proton <br />                                                                                        transporter          symporter <br />                                                                                                                             Membrane <br /> We used thermodynamics-based metabolic flux anal- <br />                                                                                                                                Cytosol <br /> ysis (TMFA) determine maximum growth rate                               H+ <br /> E. coli glucose aerobic conditions, using iJR904                                          asp       2H+ <br />                                                                                     ADP           ATP <br /> genome-scale metabolic model (24). Building previous <br /> work (7), constrained concentration intracellular                              [x]min    [x]max         [x]meas <br />                                                                  B              RTFMA    (mM)      (mM)           (mM) <br /> metabolites range 0.001&#226;&#8364;&#8220;20 mM. We constrained <br />                                                                                   ADP     0.89       20       0.437 0.704 <br /> the concentrations extracellular nutrients of <br />                                                                                    ATP    0.01     0.22        8.13 11.4 <br /> 0.4% glucose MOPS medium (35). The concentration of <br /> intracellular H&#195;&#190; fixed pH 7, extracellular     FIGURE 1 Examples thermodynamically feasible physiologically <br /> H&#195;&#190; pH 7.4. The concentration extracellular gases    implausible behavior. (A) RTMFA predicts ATP synthesized by <br /> (O2 CO2) based experimental measurement (36),       cycling small molecules membrane. One cycle involves <br />                                                                asp. (B) Metabolite concentration ranges predicted RTMFA which <br /> with requirement intracellular concentration     cycle shown panel A thermodynamically feasible (columns 1 <br /> less extracellular concentration. Under con-    2), experimentally measured metabolite concentrations (column <br /> ditions, TMFA predicted maximum growth rate zero,         3). (Highlighted type) Constraining concentration model renders <br /> suggesting growth possible thermo-     cycle thermodynamically infeasible. <br /> dynamic infeasibility essential reactions. We <br /> expanded concentration range seven metabolites to <br /> enable thermodynamic feasibility essential reac-      ATP (Fig. 1 B). By additionally constraining concentra- <br /> tions (see Supporting Results Supporting Material).     tion ATP experimentally measured range (22), in <br /> All simulations performed glucose limiting    ATP present higher concentration ADP, <br />                                                                force RTMFA model use ATP synthase to <br /> substrate, maximum uptake rate 10 mmol/gDW/h. <br />                                                                synthesize ATP. When constraint ATP concentration <br /> The set metabolite concentration exchange flux <br />                                                                added model, predicted growth rate exceeds <br /> constraints Table S6. <br />                                                                FBA 3%. This constraint ATP concentration <br />    We performed simulations neglecting uncertainty <br />                                                                used subsequent RTMFA simulations discussed <br /> in standard Gibbs energy formation group <br />         0                                                      work. <br /> (Dgr G 0) (Eq. 10). The maximum growth rate predicted <br /> by TMFA exceeded FBA 2%, despite the <br /> increased network flexibility possible lack     Flux variability analysis: thermodynamically <br /> predefined reaction directions TMFA.                        feasible reaction directions <br />                                               0 <br />    We introduced uncertainty Dgr G 0 (Eq. 11) and <br />                                                                We used flux variability analysis (FVA) determine <br /> observed maximum growth rate exceeded         thermodynamically feasible directions reactions in <br /> FBA ~12%. This elevated growth rate highlights          network glucose aerobic conditions (Fig. 2 A, <br /> additional network flexibility possible uncer-     Table S7 Table S8). We classified reactions as <br /> tainty free energy estimates arising GCM. In       fully bidirectional, constrained ways: <br /> particular, growth rate difference mecha-       blocked entirely; capable operating forward direc- <br /> nisms relaxed TMFA model (RTMFA) enable            tion only; capable operating reverse direction <br /> ATP synthesized lower energetic cost occurs      only. We performed FVA fully reversible version <br /> physiologically (and reflected FBA). For example,        iJR904 model thermodynamic constraints <br /> RTMFA predict ATP synthesis ATP synthase,           (FBAr), reactions (except biomass) were <br /> using energy released transport protons       allowed reversible. This allowed identify direc- <br /> across plasma membrane. Instead, RTMFA model           tionally constrained reactions based stoichiometry and <br /> identifies numerous cycles synthesize ATP using energy    external environment. <br /> released transport fewer numbers protons,       We observed FBAr, majority reactions are <br /> shuttling metabolites forth plasma mem-    bidirectional (60%), remainder constrained <br /> brane (Fig. 1 A). One metabolite aspartate (asp),      way. On hand, FBA model large <br /> which enables ATP synthesized using energy released      set irreversible reactions, causing number bidirec- <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> transport protons plasma       tional reactions decrease significantly, mere 5% the <br /> membrane. However, shuttling relies aspartate      network. The majority reactions FBA model <br /> ABC-type transporter pumping aspartate, re-          operate forward direction only, all. When <br /> quires ADP present concentration higher        neglect priori reaction directionality assignments and <br />  <br />                                                                                                      Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  516                                                                                                                                                                                         Hamilton et al. <br />  <br />  <br /> A                                                        reactions         thermodynamic   uncertainties <br />                                                                                                                   TMFA                         When reaction directionalities FVA analysis of <br />                                                                                                         &#194;&#180;0      directions <br />                                                           reversible             constraints     &#226;&#710;&#8224;fGest <br />                                                     FBA                  FBAr              TMFA           RTMFA           TMFA-LP <br />                                                                                                                                             TMFA model used constrain fluxes in <br />                                                0 <br />                                                                                                                                             FBAr (TMFA-LP), predicted growth rate 150% of <br />                                          100 <br />                                                                                                                                             predicted FBA (Fig. 2 C). This increase growth <br />                                                                                                                                             rate indicates ATP- energy-generating cycles were <br />   Number Reactions <br />  <br />  <br />  <br />  <br />                                          200                                                                                                present network present thermody- <br />                                          300 <br />                                                                                                                                             namic constraints imposed directly. One cycle <br />                                                                                                                                             involves shuttling sodium ions forth across <br />                                          400                                                                                                membrane, resulting proton gradient used synthe- <br />                                          500 <br />                                                                Blocked                                                                      size ATP (see Fig. S1 Supporting Material). This cycle <br />                                                                Forward <br />                                                                Reverse                                                                      infeasible TMFA, despite reaction operating in <br />                                                                Bidirectional <br />                                          600 <br />                                                                 LP    MILP <br />                                                                                                                                             thermodynamically feasible direction. This emphasizes <br />                                                                                                                                             need actively impose thermodynamic constraints <br /> B                                                                                                                                           account thermodynamic interactions reac- <br />                                                                                                                                             tions (7,8,21), opposed methods impose thermody- <br /> Cumulative Distribution Function (CDF) <br />  <br />  <br />  <br />  <br />                                          0.8                                                                                                namically feasible reaction directions accounting <br />                                                                                                                                             thermodynamic coupling reactions (17&#226;&#8364;&#8220;19). <br />                                          0.6                         C               Growth Rate (Fraction FBA Growth Rate) <br />                                                                                                                                                Finally, used FVA results formulation to <br />                                                                                  0   0.2    0.4   0.6     0.8    1       1.2   1.4    1.6 <br />                                                                           FBA <br />                                                                                                                                             cumulative distribution function (CDF) flux <br />                                          0.4                             TMFA                                                               span, difference maximum minimum <br />                                                                      RTMFA                                           *                      flux given reaction (Fig. 2 B Table S9). <br />                                          0.2        FBA <br />                                                     TMFA <br />                                                             TMFA-LP <br />                                                                                                           * [ATP] constraint <br />                                                                                                                                             The CDF FBA sharpest, &gt;90% reac- <br />                                                     RTMFA <br />                                                     TMFA-LP <br />                                                                                                                                             tions having span &lt;300 mmol/gDW/h. The CDF for <br />                                           0                                                                                                 TMFA similar FBA, despite increase in <br />                                                0   200      400       600         800   1000 1200 1400            1600       1800    2000 <br />                                                                                Flux Span (mmol / gDW / hr)                                  network flexibility. The CDF shallow for <br />                                                                                                                                             RTMFA (more reactions larger spans), reaction&#226;&#8364;&#8482;s <br /> FIGURE 2 Comparison thermodynamic formulations glucose <br /> aerobic conditions. (A) Reactions classified blocked, forward, <br />                                                                                                                                             span increase directionally unconstrained. <br /> reverse, bidirectional based thermodynamic feasibility. Differ-                                                                 The CDF TMFA-LP shallow: jumps the <br /> ences formulation given chart. (B) Cumulative                                                                    CDF value flux spans 1000 2000 mmol/gDW/h <br /> distribution function (CDF) flux span formulation. (C) Plot                                                                point existence thermodynamically infea- <br /> of growth rate formulation, normalized FBA growth rate.                                                                     sible closed cycles (e.g., A/ B / C / A) which <br /> (Formulations) FBAr: FBA reaction directions fully reversible. <br /> TMFA: FBAr thermodynamic constraints, global metabolite bounds, <br />                                                                                                                                             participating reactions operate maximum flux. These <br /> and media constraints. RTMFA: cTMFA uncertainties Df G 0est <br />                                                                         0 <br />                                                                                                                                             results highlight important role thermodynamic interac- <br /> and constraint ATP concentration. TMFA-LP: LP FBAr reaction                                                                       tions play shaping feasible flux space. We observed <br /> directions consistent TMF.                                                                                                             sets bidirectional constrained reactions vary <br />                                                                                                                                             slightly media conditions, overall CDFs <br />                                                                                                                                             remain qualitatively similar (data shown). <br /> instead allow reaction directions determined solely by <br /> thermodynamic constraints (TMFA), fraction direc- <br />                                                                                                                                             Gene deletion studies: comparison FBA <br /> tionally constrained reactions 85%, compared to <br />                                                                                                                                             TMFA <br /> 95% FBA, 40% FBAr. This reveals thermody- <br /> namic constraints play major role eliminating                                                                                  Under glucose aerobic conditions, CONGA identified 22 <br /> the network flexibility resulting eliminating predefined                                                                               single gene deletions TMFA FBA made <br /> reaction directions (FBAr). We that, relative FBA                                                                               different growth phenotype predictions (Table 1). In 19 <br /> (with predefined reaction directions), TMFA enabled previ-                                                                                  cases, TMFA predicted nonlethal phenotype FBA pre- <br /> ously forward- reverse-only reactions bidirec-                                                                                 dicted lethal one, remaining cases <br /> tional, previously blocked reactions feasible                                                                                 TMFA predicted <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> FBA predicted a <br /> in reverse direction.                                                                                                                   nonlethal one. Relative experimental data (25), corre- <br />    Just moving FBA TMFA led decrease                                                                                     sponded better prediction TMFA seven cases, and <br /> the number constrained reactions, does moving                                                                                     worse prediction remaining 15 cases. Using RTMFA <br /> from TMFA RTMFA. The number constrained reac-                                                                                         instead TMFA introduces worse predictions <br /> tions decreases 85% TMFA 77% RTMFA.                                                                                           better prediction (Table 1). <br /> The number bidirectional reactions increases 50%,                                                                                        For TMFA predict growth FBA predicts no <br /> as previously forward-only reactions feasible                                                                                   growth, TMFA enable reaction proceed direc- <br /> in reverse direction.                                                                                                                   tion allowed FBA. We used variant SL Finder <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                                      517 <br />  <br /> TABLE 1 Single-gene deletions FBA and/or (R)TMFA predict different growth phenotypes glucose aerobic <br /> conditions <br />                       Gene locus     Gene       Phenotype        FBA          TMFA        TMFA (corrected)        RTMFA        RTMFA (corrected) <br /> Better TMFA          b0907           serC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b0928           aspC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b2913           serA         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b3359           argD         Nonlethal      Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3429           glgA         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b3430           glgC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b4388           serB         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br /> Worse           b0474            adk          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b0720           gltA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b1207           dnaR          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b2615           nadK          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b2818           argA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3607           cysE          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3608           gpsA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal          Nonlethal <br />                         b3729           glmS          Lethal        Lethal       Nonlethal           Lethal           Nonlethal          Nonlethal <br />                         b3957           argE          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3958           argC          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3959           argB          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b4226            ppa          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3919           tpiA         Nonlethal     Nonlethal      Lethal             Lethal            Lethal             Lethal <br /> Worse TMFA           b1849           purT         Nonlethal     Nonlethal      Lethal             Lethal           Nonlethal          Nonlethal <br />                         b2500           purN         Nonlethal     Nonlethal      Lethal             Lethal           Nonlethal          Nonlethal <br /> Better RTMFA         b0888           trxB          Lethal        Lethal        Lethal             Lethal           Nonlethal          Nonlethal <br /> Worse RTMFA          b1136            icd          Lethal        Lethal        Lethal             Lethal           Nonlethal           Lethal <br />                         b2780           pyrG          Lethal        Lethal        Lethal             Lethal           Nonlethal           Lethal <br /> Columns labeled &#226;&#8364;&#732;&#226;&#8364;&#732;corrected&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicate RTMFA predictions additional phenotype-correction constraints (described Table S10 Supporting <br /> Material) included. <br />  <br />  <br />  <br /> (34) identify reactions <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>knockout</span> mutant. In                  prediction. This suggests ornithine transaminase might <br /> cases TMFA makes better prediction, hypothe-                        rescue DargD. Indeed, argD::kan DastC double <br /> sized SL reaction active mutant pheno-                   mutant proved viable (see Fig. S4), suggesting other <br /> type. This hypothesis tested knocking SL                     reaction isozyme rescues DargD mutant. <br /> reaction single mutant: resulting mutant                      In cases TMFA worse prediction FBA, <br /> be incapable growth. Conversely, TMFA makes                          developed phenotype-correction constraint pre- <br /> worse prediction, SL reaction operate pre-                  vented SL reaction operating rescuing direc- <br /> dicted direction vivo.                                                      tion (see Table S10). In instances, SL Finder predicted <br />    Of seven deletions TMFA better                         multiple reactions acted rescue pheno- <br /> prediction FBA, selected experimental vali-                    type, reaction rescued multiple phenotypes. <br /> dation: DaspC DargD. Both mutants exhibit robust                           For example, SL Finder predicted reactions ornithine <br /> growth, removing SL reactions requires                          transaminase n-acetylornithine deacetylase acted <br /> a single gene deletion. For DaspC mutant, SL Finder                        rescue phenotypes DargA, DargB, and <br /> identified aspartase (encoded aspA) SL reaction.                     DargC. All told, 12 genes TMFA an <br /> After aspA::kan aspC double mutant proved viable                            incorrect nonlethal prediction associated 10 SL <br /> (see Fig. S2), identified studies reporting tyrB                     reactions. We able identify metabolite concentration <br /> an isozyme aspC (37,38). A tyrB::kan DaspC double                          constraints 10 reactions (see Table S10), <br /> mutant proved nonviable (see Fig. S3), confirming                         which, implemented TMFA, resulted correct <br /> tyrB aspA rescues DaspC mutant. We                        predictions 12 genes (Table 1). For RTMFA, there <br /> correct phenotype adding tyrB isozyme                         19 SL reactions associated 14 genes which <br /> aspC model, imposing constraint pre-                      RTMFA incorrect nonlethal predictions. We were <br /> vents aspartase operating reverse direction. SL                    able identify concentration constraints 17 these <br /> Finder predicted DargD rescued ornithine                      reactions (see Table S10), which, implemented in <br /> transaminase, reaction argD astC                           RTMFA, resulted correct predictions 12 14 <br /> reported activity (39,40). However, allowing orni-                     genes (Table 1). With exception DargD noted above, <br /> thine transaminase reversible results TMFA making                     correct TMFA RTMFA predictions unaffected by <br /> four worse predictions correcting single DargD                       additional phenotype-correction constraints. <br />  <br />                                                                                                                  Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  518                                                                                                                                                         Hamilton et al. <br />  <br />  <br />        A                &#206;&#8221;b0720, citrate synthase (2.3.3.1)                                                    Taken single-gene deletion data, this <br />                                                .6                                                             suggests additional concentration measurements are <br />  <br />  <br />  <br />  <br />                                        4.1.3 <br />                                                 ac                                                            needed accurately define metabolic flux space <br />                                                oaa <br />                                                      X <br />                                                      2.3 <br />                                                         .3.1     cit <br />  <br />                                                                accoa                   icit <br />                                                                                                               predict growth phenotypes. <br />                                                                                                                  We identified gene deletions TMFA <br />                                                                                                               falsely predicted lethal phenotype (Db1949, Db2500, and <br />                                                                                                               Db3919) FBA predicted nonlethal (Table 1). In <br />                                       mal                  TCA Cycle                          akg             cases, hypothesized SL reaction active in <br />                                                                                                               FBA thermodynamically infeasible TMFA. Indeed, <br />                                                                                                               cases, uncertainty free energies <br />                         biomass                fum                                succ-CoA                    included, RTMFA correct prediction (the <br />                        precursors                                                              biomass        exception Db3919). RTMFA picked an <br />                                                                 succ                          precursors      additional gene deletion (Db0888) TMFA and <br />        B                    Constraining citrate lyase (4.1.3.6)                                              FBA incorrectly predicted lethal phenotype (Table 1). <br />                                                                                                               Thus, cases thermodynamic <br />                                                                                                               assignment reaction directions led inaccurate <br />                        &#226;&#710;&#8217;4 <br />                        &#226;&#710;&#8217;5 <br />                                                                                                               growth predictions, cases thermody- <br />                        &#226;&#710;&#8217;6                                                                                     namic constraints needed explain observed growth <br />         ln[cit] (mM) <br />  <br />  <br />  <br />  <br />                        &#226;&#710;&#8217;7                                                                                     phenotypes. <br />                        &#226;&#710;&#8217;8 <br />                        &#226;&#710;&#8217;9 <br />                                                                                                                  We evaluated TMFA aerobic growth glycerol <br />                   &#226;&#710;&#8217;10                                                                                         anaerobic growth glucose (see Table S5). We found <br />                   &#226;&#710;&#8217;11                                                                                         enable aerobic growth glycerol M9 medium TMFA <br />                     &#226;&#710;&#8217;4 <br />                             &#226;&#710;&#8217;5 <br />                                  &#226;&#710;&#8217;6                                                                 &#226;&#710;&#8217;5 <br />                                                                                                          &#226;&#710;&#8217;4   required concentration constraints glucose <br />                                       &#226;&#710;&#8217;7                                                      &#226;&#710;&#8217;6 <br />              ln[oaa] (mM)                  &#226;&#710;&#8217;8 <br />                                                 &#226;&#710;&#8217;9                                &#226;&#710;&#8217;8 <br />                                                                                          &#226;&#710;&#8217;7                   case; however, anaerobic growth glucose, TMFA <br />                         TMFA                         &#226;&#710;&#8217;10                     &#226;&#710;&#8217;9          ln[ac] (mM) <br />                                                                        &#226;&#710;&#8217;10 <br />                         RTMFA <br />                                                         &#226;&#710;&#8217;11     &#226;&#710;&#8217;11                                           required expanded concentration ranges slightly <br />                                                                                                               different set metabolites. We performed CONGA <br /> FIGURE 3 Example reduced concentration spaces imposed pheno-                                            single-gene deletions conditions. Under glyc- <br /> type-correction constraints. (A) TMFA incorrectly predicts deletion <br />                                                                                                               erol aerobic conditions, 20 gene deletions for <br /> of gltA (citrate synthase, 2.3.3.1) nonlethal. When gltA deleted <br /> (large X), TMFA predicts citrate lyase (4.1.3.6) synthesizes citrate                                     FBA TMFA different predictions, with <br /> (cit) operating reverse. (B) To correct phenotype, citrate lyase                                    better 14 worse predictions (see Table S12). Under <br /> must constrained operate forward direction. This requires                                   glucose anaerobic conditions, 26 gene deletions <br /> the ac, oaa, cit concentrations lie beneath shaded surface.                                        FBA TMFA different predictions. We <br />                                                                                                               performed growth phenotype screens 20 dele- <br />    An example phenotype-correction constraint illus-                                                    tion strains available Keio collection (see Fig. S5). <br /> trated Fig. 3. TMFA predicts deletion citrate                                                  Assuming deletion strains unavailable in <br /> (cit) synthase (gltA, EC 2.3.3.1) nonlethal, citrate                                                  Keio collection involve essential genes (25), these <br /> lyase (EC 4.1.3.6) rescuing growth. In case citrate                                                   screens reveal TMFA makes better prediction than <br /> synthesized oxaloacetate (oaa) acetate (ac),                                                         FBA cases worse prediction 18 cases <br /> instead oaa acetyl-CoA (accoa) (Fig. 3 A). We                                                    (see Table S13). <br /> the phenotype-correction constraint for <br />        DgltA ln&#195;&#176;ac&#195;&#382; &#195;&#190; ln&#195;&#176;oaa&#195;&#382;   ln&#195;&#176;cit&#195;&#382;&lt;   2:76                                                         Thermodynamic variability analysis: ranges of <br />                                                                                                               metabolite concentration <br /> for TMFA,                                                                                                 Thermodynamic variability analysis (TVA) used to <br />                                                                                                               study ranges metabolite concentrations allow <br />                              ln&#195;&#176;ac&#195;&#382; &#195;&#190; ln&#195;&#176;oaa&#195;&#382;   ln&#195;&#176;cit&#195;&#382;&lt;   7:77                                               maximal growth glucose minimal media absence <br />                                                                                                               uncertainty standard Gibbs energy formation of <br />                                                                                                                                   0 <br /> for RTMFA. In Fig. 3 B, constraints indicated                                                    group (Dgr G 0). Using TMFA, identified total <br /> the shaded planes, concentrations half-space                                                      124 (out 618) intracellular metabolites feasible <br /> below plane satisfying constraint.                                                                    concentration range default global bounds <br />    CONGA identified total 20 double- triple-                                                    (0.001 20 mM, Table 2, Table S14 Fig. S5), <br /> deletions TMFA predicted nonlethal phenotype                                                      indicating thermodynamic constraints impose <br /> and FBA lethal (see Table S11). We able                                                    restrictions metabolite concentrations. <br /> experimental phenotypes 14 multi-gene dele-                                                         The study Bennett et al. (22) examined exponential <br /> tions, TMFA worse prediction cases.                                                         growth E. coli, reported measurements 107 <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                                                                                519 <br />  <br /> TABLE 2 Comparison model-predicted experimentally                                                                      tion range restricted thermodynamic constraints <br /> measured metabolite concentration ranges glucose aerobic                                                                  (Fig. 4 B), 57 did (i.e., ranges as <br /> conditions maximal growth <br />                                                                                                                               global bounds). These results suggest TMFA be <br />                                                                    Predicted TMFA                                          suitable framework incorporating mea- <br />                                                              Unconstrained Constrained                                  sured concentration data constraint-based models, <br />                                                              thermodynamics thermodynamics Total                              tool predict experimental concentration <br /> Experimentally  Overlap                                            57                              16            73           measurements. <br /> Measured       No overlap                                          11                              23            34              A second study, Ishii et al. (27), examined E. coli <br />                 No data                                            426                             85            511          grown continuous culture 0.2 h 1, reported mea- <br /> Total                                                              494                             124           618 <br />                                                                                                                               surements 88 metabolites iJR904 model. We per- <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Constrained&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates concentration ranges tighter global                                                formed TVA growth rate examine effect of <br /> bounds (0.001&#226;&#8364;&#8220;20 mM).                                                                                                         suboptimal growth predicted metabolite concentrations. <br />                                                                                                                               In general, fewer metabolites predicted to <br /> metabolites iJR904 model. Of these, predicted con-                                                                     constrained ranges suboptimal growth (97 <br /> centration ranges overlapped measured values                                                                       compared 124, Table S15 Fig. S7), likely <br /> in 73 instances, failed overlap 34 (Table 2,                                                                        increase network flexibility associated with <br /> Fig. 4, Fig. S6). Of 34 metabolites (Fig. 4                                                                     suboptimal growth. We compared model predic- <br /> A), 12 measurements did overlap predicted                                                                      tions measurements taken dataset Ishii et al. <br /> values measurements fell outside global                                                                       (27) predictions agree experi- <br /> bounds defined model. In addition, 11                                                                     mental measurements (74 88 measurements, Table <br /> 12 metabolites, concentration range predicted                                                                          S16, Fig. S8, Fig. S9, Fig. S10). Of 68 metabo- <br /> TMFA spanned range allowed global bounds,                                                                     lites measured studies, 42 predictions agreed with <br /> indicating changing global bounds likely                                                                       experimental measurements studies. <br /> resolve conflicts. However, doing unlikely to <br /> result tighter ranges predicted metabolite concentra- <br />                                                                                                                               Examination thermodynamic bottlenecks <br /> tions. For remaining 22 (out 34) conflicting metabo- <br /> lites measurements did fall global                                                                           Previous studies utilized thermodynamic constraints to <br /> bounds, thermodynamic consistency (i.e., predicted concen-                                                                    identify candidate reactions regulation (those trans- <br />                                                                                                                                                                           0 <br /> trations consistent experimental measurement)                                                                      formed Gibbs energy reaction (Dr G ) far zero) <br /> be achieved 22 metabolites allowing uncer-                                                                     (21,23), identify thermodynamic bottlenecks in <br /> tainty using RTMFA. However, predicted concentration                                                                      cellular metabolism (7,23). Thermodynamic bottlenecks <br /> range metabolites RTMFA spanned                                                                       defined reactions render metabolic path- <br /> global range, indicating RTMFA unable predict                                                                      ways infeasible given known concen- <br /> these metabolite concentrations great precision. Of                                                                      trations (41,42). The term later used (7,23) to <br />                                                                                                                                                                   0 <br /> the 73 instances overlap measured TMFA                                                                         reactions Dr G close equilibrium. <br /> predicted values, 16 metabolites feasible concentra-                                                                    Such reactions feasible narrow concentration <br />  <br />  <br /> A                adn                                                        B              nadp <br />                 dgsn <br />                  cytd <br />                   gsn                                                                      fum <br />                  ade <br />               dad-2                                                                        trp-L <br />              adpglc <br />                                                                                                                                                    FIGURE 4 Comparison model-predicted and <br />                                                                                            nadh <br />                  anth <br />               ppcoa <br />                                                                                                                                                    experimentally observed metabolite concentrations <br />                 inost <br />                   aps <br />                                                                                            prpp                                                    maximal growth. The dataset Bennett et al. <br />                  glcn                                                                      pep                                                     (22) contains metabolite concentrations 107 <br />              dhor-S <br />                 histd                                                                     arg-L                                                    metabolites iJR904. (A) Plot 34 metabo- <br />                   coa <br />                                                                                                                                                    lites experimental theoretical con- <br /> Metabolite <br />  <br />  <br />  <br />  <br />                                                                              Metabolite <br />  <br />  <br />  <br />  <br />                    fad <br />                                                                                            r5p <br />               nadph <br />              succoa                                                                                                                                centration measurements overlap. (B) Plot <br />                                                                                           ru5p-D <br />                   imp <br />                  cmp <br />                                                                                                                                                    16 metabolites experimental and <br />                  amp                                                                      xu5p-D <br />                   akg <br />                                                                                                                                                    theoretical concentration measurements overlap, <br />                                                                                           mal-L <br />                 succ <br />               cbasp <br />                                                                                                                                                    metabolite concentration ranges <br />                pro-L                                                                       ctp                                                     constrained thermodynamics. (A B) <br />                  adp <br />               accoa                                                                        nad                                                     Circles (diamonds) Mean predicted (measured) <br />                  actp <br />                     cit                                                                    6pgc                                                    metabolite concentration . (Horizontal bars) Full <br />                asp-L <br />                  g6p                                                                       atp <br />                                                                                                                                                    concentration range. Metabolite abbreviations can <br />                  g1p         Global Default Bounds <br />                    f6p       Measured Concentration                                                                                                Supporting Material. <br />                                                                                            fdp <br />                 glu-L        Predicted Concentration <br />                     0.0001     0.001       0.01        0.1     1     10                       0.0001     0.001   0.01   0.1      1     10    100 <br />                                            Concentration (mM)                                                    Concentration (mM) <br />  <br />  <br />  <br />                                                                                                                                                              Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  520                                                                                                                                       Hamilton et al. <br />  <br />                                                                                                                                                  0 <br /> range. Thus, term &#226;&#8364;&#732;&#226;&#8364;&#732;thermodynamic bottleneck&#226;&#8364;&#8482;&#226;&#8364;&#8482; refers                              prediction. Reactions GLUPRT (Dr G &lt; 0) <br /> a bottleneck space potential metabolite concentra-                              remain feasible concentration ranges spanning several <br /> tions. Thermodynamic models provide quantitative                                      orders magnitude. Finally, PRPP synthetase, primary <br /> values feasible concentration range metabolites                                regulatory control point purine biosynthesis (30), also <br /> associated bottleneck. TVA used study                                   free energy range far zero. <br />                                                0 <br /> ranges reaction Gibbs energies (Dr G ) allow <br /> maximal growth glucose minimal media absence <br />                          0                                                                DISCUSSION <br /> of uncertainty Dgr G 0. Using TVA, identified total <br /> of 168 reactions operate single direction                                  TMFA modifies flux balance analysis thermodynamic <br /> and free energy range includes equilibrium (see                                     constraints, allowing expanded predictive capability of <br /> Table S17).                                                                               constraint-based methods. TMFA ensures reactions <br />    One reaction operating close equilibrium                               operate thermodynamically feasible directions, elimi- <br /> phosphoribosylaminoimidazole synthase (PRAIS), inter-                                  nates thermodynamically infeasible closed cycles, and <br /> mediate step purine biosynthesis. We used TVA iden-                                 accounts thermodynamic coupling reactions <br />                            0 <br /> tify range Dr G metabolite concentrations                                      network. In work, demonstrate thermody- <br /> consistent cellular growth reaction metab-                              namic constraints provide guide predicting reac- <br /> olite pathway (Fig. 5). TVA shows driving                                 tion directions absence prior knowledge. We also <br /> force PRAIS large concentration gradient                                 systematically evaluated impacts thermody- <br /> 2-formamido-n(1)-(5-phospho-D-ribosyl)acetamidine                                         namic constraints metabolic flux distributions and <br /> (fpram) 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-                                   cellular growth rates, highlight importance of <br /> carboxamide (air), PRAIS feasible narrow                                explicitly accounting thermodynamic coupling between <br /> range fpram air concentrations. Conversely, reactions                              reactions. We used TMFA make qualitative and <br />                                        0 <br /> with large positive negative Dr G relatively                               quantitative predictions, validated predic- <br /> insensitive metabolite concentration, TMFA confirms                                tions variety genome-scale datasets. We show <br />                                                                                           predictions generate hypotheses regarding <br />                                                                                           reaction directions thermodynamic bottlenecks. We <br />                                                     &#206;&#8221;rG&#226;&#8364;&#178; (kcal / mol) <br />                                            -30    -20        -10        0            10   instances predictions metabolite <br />                       atp <br />                           r5p                                                             concentrations lack precision and/or accuracy, which <br />                             PRPPS <br />                     amp <br />                           prpp <br />                                                                                           case additional types data constraints incorpo- <br />                     gln-L <br />                     glu-L <br />                             GLUPRT                                                        rated TMFA. Identifying additional data most <br />                 atp + gly <br />                           pram <br />                             PRAGS <br />                                                                                           useful important question addressed in <br />                      adp <br />                           gar <br />                                                                                           years ahead. <br />                    10fthf <br />                       thf <br />                             GARFT                                                            In work, TMFA able achieve <br />               atp + gln-L <br />                           fgam <br />                             PRFGS <br />                                                                                           physiologically realistic predictions growth rates and <br />               adp + glu-L <br />                           fpram <br />                                                                                           flux distributions absence uncertainty esti- <br />                       atp                                                                                                          0 <br />                      adp <br />                             PRAIS <br />                                                                                           mated contributions groups (Dgr G 0k,est) formation <br />                                                                                                            0 <br />               hco3 + atp air <br />                                                                                                              0 <br />                             AIRC2 <br />                                                                                           energies (Df G ), concentration measurements for <br />                      adp <br />                           5caiz                                                           ATP required presence uncertainty. While it <br />                             AIRC3* <br />                                                                                           encouraging TMFA reproduce wild-type growth <br />               asp-L + atp 5aizc <br />                             PRASCS <br />                                                                                           rates minimum experimental data, that <br />                     adp <br />                           25aics                                                          additional concentration measurements necessary <br />                                 ADSL2r <br />                        fum                                                                refine growth rate predictions conditions (e.g., <br />                               aicar <br />                      10fthf <br />                                 AICART                                                    knockout mutants). We observed slightly <br />                         thf <br />                               fprica                                                      different global concentration bounds necessary to <br />                                 IMPC* <br />                                                                                           support growth aerobic versus anaerobic conditions, sug- <br />                               imp <br />        Reaction         0 <br />                    v&#194;&#183;&#206;&#8221;rG&#226;&#8364;&#178; &gt; 0 <br />                   Feasible &#206;&#8221;rG&#226;&#8364;&#178; Range 0.01       0.1             1              10        gesting concentration predictions media-dependent. <br />                                                    Concentration (mM) <br />                   Feasible Concentration Range <br />                                                                                              TMFA used generate hypotheses regarding <br /> FIGURE 5 Model-predicted concentrations Dr G ranges for <br />                                                                             0             reaction directions thermodynamic bottlenecks. For <br /> metabolites reactions purine biosynthesis pathway. (Circles                    example, cofactor pairs ATP/ADP appear together <br />                                                       0 <br /> and diamonds) Mean predicted concentrations (Dr G ). (Horizontal bars) <br />                          0 <br />                                                                                           numerous reactions, resulting constraints con- <br /> Full concentration (Dr G ) range. Underlined reaction abbreviations signify               centration ratio metabolites (7). However, we <br />           0          0 <br /> that Dr G Dr G 0 opposite signs. Starred reaction abbreviations <br />                                                                                           constraining ATP concentration was <br /> indicate reaction direction iJR904 model opposite that <br />                               0 <br /> shown figure, Dr G values sign reversed agree                necessary achieve physiologically realistic behavior in <br />                                                                                                                                 0 <br /> with direction shown. Metabolite reaction abbreviations                    presence uncertainty Dgr G 0k,est estimates. Thus, <br /> found Supporting Material                                                          constraints metabolite ratios serve drive <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                              521 <br />  <br /> reactions physiological directions. Physiological re-      improve solver performance different formu- <br /> action directions assigned based vitro charac-      lations (50). We observed priori thermodynamic <br /> terization enzymes conditions vary                constraints reaction directions (Eqs. 14&#226;&#8364;&#8220;18) reduced <br /> significantly vivo. Thus, envision           solution times order magnitude. Alternatively, <br /> TMFA complementing approaches (17&#226;&#8364;&#8220;19,43)                 enforcing strict thermodynamic requirements, <br /> are used calculate thermodynamically feasible reaction           use thermodynamic metabolomic data a <br /> directions new genome-scale models. TMFA prom-             guide, seek solution maximizes consistency <br /> ises useful tool metabolic engineering applica-         available data (e.g., allowing thermodynamic <br /> tions, identifying thermodynamic bottlenecks                  concentration constraints violated, employing <br /> engineered pathways (44) pinpointing reactions          penalty objective). One study suggests metabo- <br /> whose reversible operation enables new routes chemical          lite concentrations remain stable response perturba- <br /> synthesis (45).                                                     tions, implying single set metabolomics data (27) <br />    We observed TMFA model limited           used model variety conditions. <br /> ability predict metabolite concentrations. This be <br /> because majority reactions iJR904 network <br /> are thermodynamically favorable, relatively insen-         SUPPORTING MATERIAL <br /> sitive metabolite concentrations. Obtaining tighter              Methods, results, figures, references (51&#226;&#8364;&#8220;54) available http:// <br /> bounds predicted metabolite concentrations require           www.biophysj.org/biophysj/supplemental/S0006-3495(13)00685-1. <br /> the use penalty function (46), thermodynamic objec- <br />                                                                     The authors acknowledge Mink Arunrattanamook John Friel for <br /> tive (20), use kinetic constraints. Recent studies        experimental contributions. The group contribution method software used <br /> have identified correlations metabolite concentra-          free energy calculations kindly provided Vassily Hatzimanikatis. <br /> tions physiochemical properties (47,48), including          This work funded National Science Foundation a <br /> KM metabolic enzymes (22). Alternatively, incorporation          CAREER grant J.L.R. (NSF No.1053712) Graduate Research <br /> of known metabolite concentrations enable TMFA               Fellowship J.J.H. (grant No. DGE-0718123). V.D. supported by <br /> predict concentrations metabolites data un-        Khorana Program University Wisconsin-Madison. <br /> available. In light results, suggest that, without <br /> additional constraints, TMFA better suited validating <br /> phenotypes generating hypotheses quantitative          REFERENCES <br /> prediction metabolite concentrations.                             1. Oberhardt, M. A., B. &#195;&#732;. Palsson, J. A. Papin. 2009. Applications of <br />    Furthermore, results suggest additional types            genome-scale metabolic reconstructions. Mol. Syst. Biol. 5:320. <br /> data constraints needed improve TMFA&#226;&#8364;&#8482;s pre-           2. Lewis, N. E., H. Nagarajan, B. &#195;&#732;. Palsson. 2012. Constraining the <br /> dictions growth phenotypes metabolite concentra-                 metabolic genotype-phenotype relationship using phylogeny of <br />                                      0                                  silico methods. Nat. Rev. Microbiol. 10:291&#226;&#8364;&#8220;305. <br /> tions uncertainties Df G 0est. A recently published <br />                                              0                       3. Orth, J. D., I. Thiele, B. &#195;&#732;. Palsson. 2010. What flux balance <br /> GCM provides tighter estimates Df G 0est (49),              analysis? Nat. Biotechnol. 28:245&#226;&#8364;&#8220;248. <br /> other approaches successfully combined group contri-            4. Beard, D. A., S. D. Liang, H. Qian. 2002. Energy balance for <br />                                                                0 <br /> bution estimates experimentally measured Df G 0                    analysis complex metabolic networks. Biophys. J. 83:79&#226;&#8364;&#8220;86. <br /> values equilibrium constants (18,21). We observed            5. Qian, H., D. A. Beard, S. D. Liang. 2003. Stoichiometric network <br /> that inclusion additional constraints cofactor con-           theory nonequilibrium biochemical systems. Eur. J. Biochem. <br />                                                                         270:415&#226;&#8364;&#8220;421. <br /> centrations formation energies (ATP, NAD, NADP, etc.) <br /> further constrains flux space (data shown). Thus,            6. Schilling, C. H., D. Letscher, B. &#195;&#732;. Palsson. 2000. Theory the <br />                                         0                               systemic <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> metabolic pathways use interpreting <br /> experimental measurements Df G 0 cofactors                metabolic function pathway-oriented perspective. J. Theor. Biol. <br /> a promising way improve accuracy thermodynamic                203:229&#226;&#8364;&#8220;248. <br /> models. Finally, note approach underestimates            7. Henry, C. S., L. J. Broadbelt, V. Hatzimanikatis. 2007. Thermody- <br />                         0 <br /> uncertainty Dr G 0 neglecting error associated                namics-based metabolic flux analysis. Biophys. J. 92:1792&#226;&#8364;&#8220;1805. <br /> with structural groups unchanged reaction, approach          8. Schellenberger, J., N. E. Lewis, B. &#195;&#732;. Palsson. 2011. Elimination of <br />                                                 0         0             thermodynamically infeasible loops steady-state metabolic models. <br /> that valid contributions Dgr G 0 Df G 0 are,       Biophys. J. 100:544&#226;&#8364;&#8220;553. <br /> in fact, linearly additive. Additional types data likely     9. Heuett, W. J., H. Qian. 2006. Combining flux energy balance <br /> be necessary considering error associated                 analysis model large-scale biochemical networks. J. Bioinform. <br /> these unchanged groups.                                                 Comput. Biol. 4:1227&#226;&#8364;&#8220;1243. <br />                                                              0 <br />    Associated need reduce uncertainty Df G 0est        10. Beard, D. A., E. Babson, ., H. Qian. 2004. Thermodynamic con- <br />                                                                         straints biochemical networks. J. Theor. Biol. 228:327&#226;&#8364;&#8220;333. <br /> is need improve model run-time performance, large <br /> mixed-integer programs TMFA quite                    11. Yang, F., H. Qian, D. A. Beard. 2005. Ab initio prediction of <br />                                                                         thermodynamically feasible reaction directions biochemical <br /> cumbersome. A recent Master&#226;&#8364;&#8482;s thesis examines number                  network stoichiometry. Metab. Eng. 7:251&#226;&#8364;&#8220;259. <br /> of thermodynamic approaches (EBA, TMFA, etc.)                12. Alberty, R. A. 2003. Thermodynamics Biochemical Reactions. John <br /> theoretical practical perspective, provides insights            Wiley &amp; Sons, Hoboken, NJ. <br />  <br />                                                                                                         Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  522                                                                                                                                           Hamilton et al. <br />  <br /> 13. Goldberg, R. N., Y. B. Tewari, T. N. Bhat. 2004. Thermodynamics           34. Suthers, P. F., A. R. Zomorrodi, C. D. Maranas. 2009. Genome- <br />     enzyme-catalyzed reactions&#226;&#8364;&#8221;a database quantitative biochem-                scale gene/reaction essentiality synthetic lethality analysis. Mol. <br />     istry. Bioinformatics. 20:2874&#226;&#8364;&#8220;2877.                                              Syst. Biol. 5:301. <br /> 14. Mavrovouniotis, M. L. 1990. Group contributions estimating stan-          35. Neidhardt, F. C., P. L. Bloch, D. F. Smith. 1974. Culture medium <br />     dard Gibbs energies formation biochemical compounds                      enterobacteria. J. Bacteriol. 119:736&#226;&#8364;&#8220;747. <br />     aqueous solution. Biotechnol. Bioeng. 36:1070&#226;&#8364;&#8220;1082.                           36. Yang, C., Q. Hua, ., K. Shimizu. 2003. Analysis Escherichia coli <br /> 15. Mavrovouniotis, M. L. 1991. Estimation standard Gibbs energy                   anaplerotic metabolism regulation mechanisms meta- <br />     changes biotransformations. J. Biol. Chem. 266:14440&#226;&#8364;&#8220;14445.                    bolic responses altered dilution rates phosphoenolpyruvate <br />                                                                                       carboxykinase knockout. Biotechnol. Bioeng. 84:129&#226;&#8364;&#8220;144. <br /> 16. Jankowski, M. D., C. S. Henry, ., V. Hatzimanikatis. 2008. Group <br />     contribution method thermodynamic analysis complex metabolic           37. Gelfand, D. H., R. A. Steinberg. 1977. Escherichia coli mutants <br />     networks. Biophys. J. 95:1487&#226;&#8364;&#8220;1499.                                               deficient aspartate aromatic amino acid aminotransferases. <br />                                                                                       J. Bacteriol. 130:429&#226;&#8364;&#8220;440. <br /> 17. Feist, A. M., C. S. Henry, ., B. &#195;&#732;. Palsson. 2007. A genome-scale <br />     metabolic reconstruction Escherichia coli K-12 MG1655                38. Berg, C. M., M. D. Wang, ., L. Liu. 1988. Acquisition new meta- <br />     accounts 1260 ORFs thermodynamic information. Mol. Syst.                  bolic capabilities: multicopy suppression cloned transaminase genes <br />     Biol. 3:121.                                                                      Escherichia coli K-12. Gene. 65:195&#226;&#8364;&#8220;202. <br /> 18. Fleming, R. M. T., I. Thiele, H. P. Nasheuer. 2009. Quantitative          39. Billheimer, J. T., H. N. Carnevale, ., E. E. Jones. 1976. Ornithine <br />     assignment reaction directionality constraint-based models                  d-transaminase activity Escherichia coli: identity acetylorni- <br />     metabolism: application Escherichia coli. Biophys. Chem. 145:               thine d-transaminase. J. Bacteriol. 127:1315&#226;&#8364;&#8220;1323. <br />     47&#226;&#8364;&#8220;56.         <br /> </body></html>