<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Transcriptomic Analysis Rice White Tip Nematode, <br /> Aphelenchoides besseyi (Nematoda: Aphelenchoididae) <br /> Feng Wang., Danlei Li*, Zhiying Wang, Airong Dong, Lihong Liu., Buyong Wang., Qiaoli Chen, <br /> Xiaohan Liu <br /> College Forestry, Northeast Forestry University, Harbin, Heilongjiang, China <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: The rice white tip nematode Aphelenchoides besseyi, devastating nematode genome been <br />      sequenced, distributed widely rice-growing regions world. The aims present study <br />      define transcriptome A. besseyi identify parasite-related, mortality-related host resistance- <br />      overcoming genes nematode. <br />  <br />      Methodology Principal Findings: Using Solexa/Illumina sequencing, profiled transcriptome mixed-stage <br />      populations A. besseyi. A total 51,270 transcripts gaps produced based high-quality clean reads. Of all <br />      A. besseyi transcripts, 9,132 KEGG Orthology assignments annotated. Carbohydrate-active enzymes glycoside <br />      hydrolases (GHs), glycosyltransferases (GTs), carbohydrate esterases (CEs) carbohydrate-binding modules (CBMs) were <br />      identified. The presence A. besseyi GH45 cellulase gene verified situ hybridization. Given 13 unique A. <br />      besseyi potential effector genes identified 41 candidate effector homologs, studies homologs are <br />      merited. Finally, comparative analyses conducted A. besseyi contigs Caenorhabditis elegans genes look <br />      orthologs RNAi phenotypes, neuropeptides peptidases. <br />  <br />      Conclusions Significance: The present results provide comprehensive insight genetic makeup A. besseyi. <br />      Many species&#226;&#8364;&#8482; genes parasite related, nematode mortality-related necessary overcome host resistance. The <br />      generated transcriptome dataset A. besseyi reported lays foundation studies molecular <br />      mechanisms related parasitism facilitates development new control strategies species. <br />  <br />   Citation: Wang F, Li D, Wang Z, Dong A, Liu L, et al. (2014) Transcriptomic Analysis Rice White Tip Nematode, Aphelenchoides besseyi (Nematoda: <br />   Aphelenchoididae). PLoS ONE 9(3): e91591. doi:10.1371/journal.pone.0091591 <br />   Editor: Gustavo Bonaventure, BASF Plant Sciences &#226;&#8364;&#8220; Cropdesign, Belgium <br />   Received October 21, 2013; Accepted February 13, 2014; Published March 17, 2014 <br />   Copyright: &#195;&#376; 2014 Wang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work financed Research Fund Doctoral Program Higher Education China (project 20100062120002), Heilongjiang <br />   Postdoctoral Fund (China; project LBH-Z11248) China State Scholarship Fund (2009660509). The funders role study design, data collection and <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: danleili@nefu.edu.cn <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          including important crop species, strawberry <br />                                                                                       (Fragaria grandiflora), sweet corn (Zea mays), sweet potato (Dioscorea <br />    Parasitic nematodes prey plants cause devastating losses                   esculenta), bird&#226;&#8364;&#8482;s nest fern (Asplenium nidus) vegetables and <br /> in agricultural production contribute significant reduction                  grasses [9&#226;&#8364;&#8220;11]. <br /> in yield. It estimated plant parasitic nematodes cause                     Based phylogenetic tree [12], A. besseyi belongs clade <br /> annual losses crop production valued US$125 billion [1]. The                    10B, major plant-parasitic nematodes belong <br /> rice white tip nematode Aphelenchoides besseyi Christie, 1942,                  clade 12. Only high-quality draft genome sequence <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> is widely distributed rice growing regions                  Bursaphelenchus xylophilus available investigation the <br /> of world, considered major contributor seed-                    biological basis clade 10 nematodes [13]. The investigation B. <br /> borne pathogens rice [2&#226;&#8364;&#8220;4]. A. besseyi global burden, causing                 xylophilus genes linked key biological processes parasitism <br /> losses 50% upland rice Brazil [5] 10&#226;&#8364;&#8220;50%                        provide guidance studies A. besseyi other <br /> flooded rice China [6].                                                            clade 10 nematodes. Although model free-living nematodes <br />    As seed-borne nematode, A. besseyi survive stored grain                   Caenorhabditis spp. (clade 9A), human parasite Brugia malayi <br /> in anhydrobiosis years [7]. This species facultative                 (clade 8B) plant-parasitic nematodes (clade 12) have <br /> migratory ecto- endoparasite leaf young tissue rice,                 subjected molecular analysis [14], little known <br /> causing whitening leaf, later                         genomic background A. besseyi. However, recently, a <br /> necrotic, crinkling distortion flag leaf enclosing                   fatty acid retinoid-binding protein gene (Ab-far-1) cloned <br /> the panicle, dies disintegrates [4,8]. Moreover,                   A. besseyi based EST sequencing [15]. In addition, an <br /> A. besseyi polyphagous damages wide range host plants, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                              March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                         White Tip Nematode Transcriptome <br />  <br />  <br /> exciting small-scale transcriptome project A. besseyi recently            assembly critically assessed LC Sciences (Houston, <br /> published [16,17].                                                                TX, USA) subsequent analysis. <br />    Next-generation sequencing technology, Solexa/                        The expression level transcript measured the <br /> Illumina, 454 (Roche) ABI-SOLiD platforms, dramati-                       number clean reads mapped sequence. The mapped clean <br /> cally improved efficiency gene discovery [18]. Using next-                 read number normalized RPKM (reads kilobase of <br /> generation sequencing technology, nematode transcrip-                     exon model million mapped reads) Bowtie 0.12.8 (single- <br /> tomes sequenced, including Trichuris suis, Necator                      end mapping) adjusted normalized factor [28]. <br /> americanus, B. malayi, Trichostrongylus colubriformis, Pratylenchus coffeae          <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> determine unigenes total available our <br /> and P. thornei [14,19&#226;&#8364;&#8220;24]. Whereas genomic analyses B. xylophilus              transcriptome recently published A. besseyi transcrip- <br /> reveal genetic potential clade 10 nematodes, transcriptomic                tome [16], compared unigenes 5,804 MIRA <br /> studies nematodes enable regulatory networks                          assembly sequences downloaded nematode.net [29] using <br /> controlling expression parasitism-related genes space                   BLASTN E-value cutoff 1e-6. <br /> and different stages deciphered. <br />    In present study, Solexa/Illumina sequencing technology                    Annotation KEGG Pathway Determination <br /> was used explore functionally annotate transcriptome of <br />                                                                                       Unigene sequence directions determined searches <br /> a mixed-stage population A. besseyi. The results provide insights <br />                                                                                   NR (non-redundant protein sequences NCBI), <br /> into genetic background A. besseyi. The transcriptomic data <br />                                                                                   Swiss-Prot, TrEMBL, Cdd, Pfam KOG databases (E- <br /> presented serve valuable resource help deepen <br />                                                                                   value,1e-5). <br /> our understanding parasitism nematode. Under- <br />                                                                                       KEGG mapping used determine metabolic pathways <br /> standing basic biology plant-parasitic nematodes, especially <br />                                                                                   metabolism, genetic information processing, environmental <br /> the parasitic mechanisms molecular level, essential the <br /> development effective control strategies.                                      information processing, cellular processes, organismal systems and <br />                                                                                   etc. [30]. Sequences submitted KEGG Automatic <br />                                                                                   Annotation Server (KAAS) [31] enrich pathway annotation. <br /> Materials Methods <br /> Nematode Material                                                                 Annotation Carbohydrate-Active Enzymes <br />    The nematode A. besseyi (NCBI BioSample accession No. -                           The CAZymes Analysis Toolkit [32] used detect A. besseyi <br /> SAMN02420038), derived single female isolated                      carbohydrate-active enzymes (CAZymes) according previous <br /> infected rice grain eastern China, maintained serial                    report [13]. Expansin-like proteins, pectinases arabinanases <br /> subculture Botrytis cinerea mat grown sterilized barley [25]              detected BLAST search using core modules known <br /> and kept dark 25uC. Nematodes freshly extracted                    protein sequences queries. BLASTP used predict protein <br /> sterilized barley used RNA extraction. Eggs, juvenile                    functions searching known protein modules and <br /> stages (J2&#226;&#8364;&#8220;J4) adults mixed ratio 1:2:3.                catalytic sites downloaded NCBI&#226;&#8364;&#8482;s Conserved Domain <br />                                                                                   Database service. Multiple sequence alignments performed <br /> RNA Extraction, cDNA Library Construction Illumina                            using Clustal W, phylogenetic trees constructed based on <br /> Sequencing                                                                        sequences using MEGA 5.05 program. <br />    Total RNA extracted mixed-stage nematodes using <br /> TRIzol regent (Invitrogen, Carlsbad, CA, USA) [26]                       Effector Candidate Identification <br /> cleaned using RNeasy Minikit column (Qiagen, Valencia, CA,                        Known putative effectors selected annotated secreted <br /> USA) according manufacturers&#226;&#8364;&#8482; instructions. The extracted                  protein sequences different plant-parasitic nematodes <br /> RNA assessed quality quantified using BioPhotom-                    (Aphelenchus avenae, B. mucronatus, B. xylophilus, Ditylenchus africanus, <br /> eter D30 (Eppendorf, Hamburg, Germany).                                           Globodera mexicana, G. pallida, G. rostochiensis, G. tabacum, Heterodera <br />    Two micrograms total RNA used extract polyadenyl-                    glycines, H. schachtii, M. arenaria, M. chitwoodi, M. hapla, M. incognita, <br /> ated RNA construct strand-specific RNAseq libraries accord-                   M. javanica, P. coffeae, P. penetrans, Rotylenchulus reniformis, Radopholus <br /> ing Illumina protocol (directional mRNASeq sample                          similis Xiphinema index) [33] downloaded GenBank. <br /> preparation #15018460 Rev. A, October 2010; Illumina,                        These sequences searched A. besseyi contigs with <br /> San Diego, CA, USA) (detailed Text S1 File S1). The library                 TBLASTN identify potential homologs [34]. An E-value cutoff <br /> quality validated Bioanalyzer 2100 (Agilent, CA, USA).                 1e-5 used identify significant matches. SignalP 4.1 [35] <br /> Each sample sequenced 80 cycles lane                        TMHMM 2.0 [36] used identify secreted proteins as <br /> Illumina GAIIX platform (Illumina, San Diego, CA, USA)                     previously described [13]. <br /> 26100 bp module LC Sciences (Houston, TX, USA). The <br /> dataset generated deep-sequencing platforms deposited                   RNAi <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>Phenotype</span> Matches C. elegans Genes <br /> the NIH short-read archive (accession No. SRR1040470).                               Seventy-five C. elegans vitro <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi)-related <br />                                                                                   genes identified retrieved WormBase (version <br /> Data Filtering De Novo Assembly                                               WS236). The A. besseyi contigs compared C. elegans genes <br />   Low-quality reads ambiguous sequences &#226;&#8364;&#732;&#226;&#8364;&#732;N&#226;&#8364;&#8482;&#226;&#8364;&#8482;                           identify orthologs RNAi phenotypes [37]. <br /> removed according sliding window method using threshold <br /> of 35 bp. To examine coverage sequences, 500,000 <br />                                                                                   Neuropeptides Peptidase Detection <br /> randomly selected reads compared NT (non- <br />                                                                                      One thirteen C. elegans neuropeptide genes were <br /> redundant nucleotide sequences NCBI) database using <br />                                                                                   identified retrieved WormBase (version WS236). The A. <br /> BLAST, typical cutoff expectation value (E-value) less <br />                                                                                   besseyi contigs compared C. elegans genes identify <br /> than 1e-10. The trinity (trinityrnaseq_r2013-02-25) [27] and <br />                                                                                   orthologs high identity neuropeptides BLAST. <br /> paired-end methods used generate non-redundant <br /> unigenes novo assembly clean reads. The quality <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                    2                              March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                White Tip Nematode Transcriptome <br />  <br />  <br />  <br />  <br /> Figure 1. The common NR hierarchies identified A. besseyi sequences. <br /> doi:10.1371/journal.pone.0091591.g001 <br />  <br />    The MEROPS database 9.8 used identify putative A.                S2), 77.8% (27,380 35,180) unigenes our <br /> besseyi peptidases [38]. The candidates derived examined          transcriptome unique. Only 2.2% (125 5,804) the <br /> for similarity (E-value,1e-10) MEROPS proteins.                         MIRA assembly-yielded sequences dataset <br />                                                                            (detailed Text S2 File S1). Because E-value cutoff 1e-6 <br /> In Situ Hybridization GH45 Cellulases                                   used, 7,800 (22.2% total 35,180) unigenes our <br />    In situ hybridization performed A. besseyi essentially        transcriptome matched 5,679 (97.8% total 5,804) MIRA <br /> described De Boer et al. [39]. A digoxigenin-labeled A. besseyi         assembly sequences. Therefore, 33,184 unigenes now <br /> GH45 cellulase DNA probe synthesized using cDNA                    available datasets combined. <br /> sequence following general protocol Roche DIG DNA                  The distribution transcript values indicated transcripts <br /> Labeling      Kit     (Roche,    Mannheim,     Germany,         cat.       expressed, 95.69% reads mapped (Table S1 <br /> no. 11175033910). The specificity probe verified             File S1). Of 51,270 transcripts, RPKM values the <br /> Southern blotting. The hybridization signals detected              majority transcripts varied 1 12 (Table S2 and <br /> incubation DIG High Prime DNA Labeling                        Figure S3 File S1). The distribution RPKM values indicated <br /> Detection Starter Kit I (Roche, Mannheim, Germany, cat.                    genes expressed low levels. <br /> no. 11745832910) photographed Bx51 microscope <br /> (Olympus, Tokyo, Japan).                                                   Annotation <br />                                                                               Unigenes determined performing BLASTX searches <br /> Results                                                                    NR protein database. A total 17,338 orthologs were <br />                                                                            identified. Of 35,180 unigenes, 88.03% annotated the <br /> Illumina Sequencing De Novo Assembly                                   common organisms (Figure 1). The majority the <br />     A total 46,826,350 raw reads obtained Solexa                annotated A. besseyi (clade 10B) sequences corresponded known <br /> (Illumina GAIIX) RNA paired-end sequencing, 36,905,372                 nucleotide sequences nematodes belonging clades 2A <br /> high-quality clean reads obtained strict filtering            (Trichinella spiralis), 8B (Ascaris suum, Loa loa B. malayi), 9A <br /> process. The average <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> clean reads 89.89 bp. The           (Caenorhabditis spp.) 10B (B. xylophilus). Other protein databases, <br /> ratio clean reads 78.81%. No RNA contamination B.              Swiss-Prot, TrEMBL, Cdd, Pfam KOG, used to <br /> cinerea organisms based 500,000 randomly             annotate unigenes (detailed Table S3 File S1). A search <br /> selected reads NT database. Using clean reads,                Gene Ontology (GO) identifiers unique sequences was <br /> Trinity produced 51,270 transcripts length .200 bp. The               performed, 14,064 GO terms retrieved. The most <br /> average length N50 transcripts 1,241 bp              abundant GO terms dataset shown Figure 2. <br /> 1,857 bp, respectively. A total 35,180 unigenes average <br /> length 1,050 bp N50 1,626 bp generated                 Pathway Determination <br /> further clustering assembly. Of unigenes, 21,882                    KAAS used annotate EST sequences animal <br /> unigenes (62.2%) .500 bp 13,259 unigenes (37.7%)                  nematodes (B. malayi A. suum), free-living nematode (C. <br /> were .1,000 bp (Figure S1 File S1). The frequency distribution          elegans), A. besseyi plant-parasitic nematode (B. xylophilus <br /> of GC content shown Figure S2 File S1.                        M. incognita) transcripts. Of A. besseyi transcripts, 9,132 <br />     Recently, A. besseyi transcriptome including 5,804             KEGG Orthology (KO) assignments annotated. Two Venn <br /> MIRA assembly sequences published [16]. Compared                  diagrams drawn intersections nematode <br /> this transcriptome (detailed supplementary dataset File          KO assignments (Figure 3). The diagram C. elegans, A. besseyi and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             3                            March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                               White Tip Nematode Transcriptome <br />  <br />  <br />  <br />  <br /> Figure 2. The abundant Gene Ontology terms present dataset cellular component, molecular function and <br /> biological process categories. <br /> doi:10.1371/journal.pone.0091591.g002 <br />  <br /> the plant parasitic nematodes indicated 8,705 A. besseyi KO             To identify pathways unigenes participated, we <br /> assignment hits nematodes, 427              mapped A. besseyi unigenes canonical reference pathways <br /> KO assignments unique (Figure 3A). Another diagram C.              KEGG database. A total 9,133 unigenes annotated <br /> elegans, A. besseyi animal-parasitic nematode A. suum         based BLASTX search KEGG database; 753 enzymes <br /> illustrates 429 unique A. besseyi KO assignments (Figure 3B).              306 KEGG pathways predicted (Table 1, Tables S4&#226;&#8364;&#8220;S8 <br />                                                                            File S1). Of KEGG pathways, 121 pathways were <br />  <br />  <br />  <br />  <br /> Figure 3. The intersections lists nematode KEGG Orthology (KO) assignments. A: Assignment C. elegans, A. besseyi plant- <br /> parasitic nematodes. B: Assignment C. elegans, A. besseyi animal-parasitic nematode A. suum. Ab: A. besseyi, Bx: B. xylophilus, Mi: M. <br /> incognita, Ce: C. elegans, Bm: B. malayi, As: A. suum. <br /> doi:10.1371/journal.pone.0091591.g003 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             4                            March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                       White Tip Nematode Transcriptome <br />  <br />  <br /> consistent corresponding pathways C. elegans. In                    Table 2. The common 5 A. besseyi pathways consistent <br /> addition common 5 pathways (Table 2),                         C. elegans. <br /> important pathways, including ubiquitin-mediated proteolysis (72 <br /> members), glutathione metabolism (39 members), glycolysis/ <br /> gluconeogenesis (38 members), glycerophospholipid metabolism                    Pathway Pathway type                                         Unigene num <br /> (38 members), drug metabolism (35 members) metabolism of <br />                                                                                 cel01100 Metabolic pathways                                  602 <br /> xenobiotics cytochrome (32 members), found. <br />                                                                                 cel03010 Ribosome                                            119 <br />  <br /> CAZymes Cell Wall Degradation                                               cel04141 Protein processing endoplasmic reticulum         118 <br />    Some CAZymes known secreted sedentary                   cel00190 Oxidative phosphorylation                           106 <br /> as migratory plant parasitic nematodes predicted               cel03013 RNA transport                                       99 <br /> be involved nematode migration plant feeding from <br /> host cells [40]. According Carbohydrate-Active enZYmes                   doi:10.1371/journal.pone.0091591.t002 <br /> Database, CAZymes grouped following classes: <br /> glycoside hydrolases (GHs), glycosyltransferases (GTs), polysac- <br /> charide lyases (PLs), carbohydrate esterases (CEs) carbohy-                 Table 3. Carbohydrate-active enzymes identified A. besseyi. <br /> drate-binding modules (CBMs) [24]. A summary CAZymes <br /> detected A. besseyi provided Table 3. In all, 524 contigs were <br /> assigned following classes CAZymes: 136 contigs                  CAZy enzyme classes                     CAZy family    A. besseyi contigs <br /> with GH domains 10 families GHs (Table S9 File S1), 328 <br />                                                                                 Glycoside hydrolases                    10             136 <br /> contigs 28 families GTs (Table S10 File S1), 42 contigs 2 <br /> families CEs (Table S11 File S1) 18 contigs 3 families             Glycosyl transferases                   28             328 <br /> of CBMs (Table S12 File S1). Apparently, PLs present              Carbohydrate esterases                  2              42 <br /> any contigs transcriptome.                                       Polysaccharide lyases                   0              0 <br />    With regard plant cell wall degradation-related CAZymes,               Carbohydrate-binding modules            3              18 <br /> first noted 4 contigs identified homologs GH45 <br />                                                                                 Total                                   43             524 <br /> cellulases previously identified B. xylophilus A. besseyi <br /> transcriptome [41]. Multiple sequence alignments A. besseyi                  doi:10.1371/journal.pone.0091591.t003 <br /> GH45 cellulase B. xylophilus homologs of <br /> Ascomycota Basidiomycota fungi revealed motifs                    Globodera spp. migratory parasitic species P. thornei and <br /> were identical proteins (Figure 4A). The GH45                    P. coffeae) [14,24]. However, GH5 cellulases apparently <br /> cellulase maximum likelihood tree (Figure 4B) showed major                absent A. besseyi transcriptome. Furthermore, GH28 <br /> clusters. Because nematode GH45 cellulases                   polygalacturonase, GH30 xylanase, GH43 arabinase, GH53 <br /> found, B. xylophilus B. mucronatus plant                 arabinogalactan galactosidase, PL3 pectate lyase expansin- <br /> parasitic nematodes used phylogenetic tree study addition            like protein absent. <br /> to A. besseyi.                                                                    In addition plant cell wall degradation-related CAZymes, we <br />    In situ hybridization used determine tissue specificity          noted A. besseyi possessed GH16 GH20 CAZymes, <br /> of A. besseyi GH45 cellulase gene transcription. The anti-sense                involved fungal cell wall degradation (Table S9 in <br /> probe stained esophageal gland cell area (Figure 5). The gland             File S1). GH16, degrades beta-1,3-glucan, core compo- <br /> region strongly stained probes A. besseyi GH45              nent fungal cell wall, present A. besseyi <br /> cellulase gene, sense probes gave signal (Figure 5E).           transcriptome remarkable abundance (a total 90 contigs). <br /> The A. besseyi GH45 cellulase gene expressed lower level              In contrast, GH20, involved chitin degradation, was <br /> in males (Figure 5A B) females (Figure 5C D).                  detected low abundance (2 contigs). <br />    Another plant cell wall degradation-related CAZyme family <br /> secreted plant parasitic nematodes GH5 cellulase, has <br />                                                                                Candidate Effector Identification <br /> been studied plant-parasitic nematodes order <br />                                                                                   Numerous effectors various functions plant cell are <br /> Tylenchida (clade 12, including major plant-sedentary <br />                                                                                secreted plant nematodes [33]. Unlike sedentary or <br /> endoparasitic nematodes Meloidogyne spp., Heterodera spp. <br />                                                                                migratory endoparasites, A. besseyi migratory ecto- and <br />                                                                                endoparasite nematode does rely biotrophy. Candidate <br />  Table 1. The classification A. besseyi pathways.                           A. besseyi parasitism genes effector homologs identified to <br />                                                                                establish relationships parasite strategy and <br />                                                                                parasitism genes. All known annotated putatively secreted <br />                                               Pathways     Members             proteins plant-parasitic nematodes compared the <br />  Metabolism                                   121          2077                A. besseyi contigs identify possible homologs. <br />                                                                                   In addition CAZymes described above, 41 candidate <br />  Genetic Information Processing               22           868 <br />                                                                                effector homologs identified (Table S13 File S1). Thirteen <br />  Environmental Information Processing         25           478 <br />                                                                                unique A. besseyi potential effector genes identified (Table 4) <br />  Cellular Processes                           17           583                 41 candidate effector homologs based defining <br />  Organismal Systems                           60           957                 characteristic effector genes encode proteins a <br />  Human Diseases                               61           1396                putative signal peptide lack transmembrane helix and <br />  Total                                        306          4489                open reading frame thirty amino acids after <br />                                                                                predicted signal peptide cleavage site [42&#226;&#8364;&#8220;44]. In contrast <br />  doi:10.1371/journal.pone.0091591.t001                                         sedentary endoparasites, plant-hormone mimics or <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 5                              March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                 White Tip Nematode Transcriptome <br />  <br />  <br />  <br />  <br /> Figure 4. Alignment polygenetic tree A. besseyi GH45 cellulase amino acid sequence homologs identified from <br /> the NCBI database. A: The alignment shows comparison A. besseyi GH45 cellulase homologs. Active site residues marked <br /> with red asterisks. The GH45 cellulase consensus sequences boxed red line highlighted magnifying lens. Asterisks, double dots <br /> and single dots denote fully conserved, strongly conserved weakly conserved amino acid residues, respectively. B: The maximum likelihood tree <br /> shows relationship A. besseyi GH45 cellulase homologs. One thousand bootstrap replicates performed, node labels <br /> represent percentage bootstrap support. <br /> doi:10.1371/journal.pone.0091591.g004 <br />  <br /> nodulation-related effector homologs A. besseyi           besseyi candidate effectors present migratory <br /> transcriptome.                                                              plant nematodes, C. elegans animal-parasitic nematodes. <br />    Because A. besseyi does establish long-term feeding                Cathepsin-like proteinases migratory <br /> relationship living cells hosts, necessary         nematode, R. similis [45]. Chitinase, egg-hatching-related <br /> to confirm 13 potential effectors functional classification.         protein, Caenorhabditis spp. L. loa. Glutathione <br /> Similar B. xylophilus [13], majority (7 13) A.         peroxidase Caenorhabditis spp., A. suum Haemonchus <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              6                             March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                    White Tip Nematode Transcriptome <br />  <br />  <br />                                                                              genes previously assumed involved process of <br />                                                                              syncytia giant cell formation, findings suggest the <br />                                                                              roles genes different A. besseyi. <br />  <br />                                                                              The Presence Genes Involved RNAi Pathway <br />                                                                                 RNAi effective method gene function identification and <br />                                                                              facilitate identification potential targets developing <br />                                                                              potent antinematodal drugs A. besseyi. RNAi been <br />                                                                              successfully applied sedentary migratory plant- <br />                                                                              parasitic nematodes, including plant-parasitic <br />                                                                              nematodes [46&#226;&#8364;&#8220;50]. Recently, RNAi successfully applied A. <br />                                                                              besseyi [15]. <br />                                                                                 A comparative analysis conducted A. besseyi contigs <br />                                                                              C. elegans genes look RNAi phenotype orthologs. The <br />                                                                              2,899 A. besseyi contigs homologous C. elegans matched 57 RNAi <br />                                                                              pathway effectors. The RNAi-related proteins separated into <br />                                                                              following core functional groups: small RNA biosynthesis, <br />                                                                              dsRNA uptake spreading, argonautes (AGOs) RNA- <br />                                                                              induced silencing complex (RISC), RNAi inhibitors nuclear <br />                                                                              effectors (Table 5 Tables S14&#226;&#8364;&#8220;S18 File S1). Apparently, A. <br />                                                                              besseyi encodes predicted RNAi pathway effectors B. <br />                                                                              xylophilus [13] M. incognita [36]. <br />  <br />                                                                              Neuropeptide Detection <br />                                                                                 Neuropeptides play important roles modulation of <br />                                                                              synaptic activity development nervous and <br />                                                                              function primary neurotransmitters [24,51]. Accord- <br />                                                                              ing WormBook, 113 C. elegans neuropeptide genes, which <br /> Figure 5. Localization A. besseyi GH45 cellulase mRNA               encode 250 distinct neuropeptides, identified [52]. <br /> situ hybridization using digoxigenin-labeled antisense sense                 With sequencing A. besseyi transcriptome, 49 <br /> probes. A B: Antisense probe male nematode. C D:                neuropeptide genes identified (Table 6). Of these, 8 genes <br /> Antisense probe female nematode. E: Control sense probe a <br /> female nematode. The scale bars represent 20 mm. G: esophageal <br />                                                                              encode insulin-like peptides (Table S19 File S1), 17 genes <br /> glands; S: stylet; M: metacarpus.                                            encode FMRFamide-related peptides (Table S20 File S1) and <br /> doi:10.1371/journal.pone.0091591.g005                                        24 genes encode non-insulin non-FLP peptides (Table S21 in <br />                                                                              File S1). <br /> contortus. FAR, A. besseyi EST project, <br /> was Caenorhabditis spp. H. contortus. PDI          Peptidase Detection <br /> Caenorhabditis spp., B. malayi L. loa. The 6 effector                The MEROPS database classifies peptidases seven catalytic <br /> homologs unknown functions. Given identified                types: aspartic, glutamic, cysteine, isoleucine, metallo, serine and <br />                                                                              threonine peptidases. In analysis, 73 peptidase genes (Table 7 <br />  <br />  Table 4. Potential effectors identified A. besseyi. <br />  <br />  <br />  Effector                                   Function                                                 Contigs hit    Bit score    Best E-value <br />  <br />  4D01                                       unknown                                                  21                  135          2e-32 <br />  5G05                                       unknown                                                  6                   175          5e-44 <br />  6F06                                       unknown                                                  56                  66           5e-11 <br />  Acid phosphatase                           digestion                                                30                  324          1e-88 <br />  Calreticulin                               calcium signaling                                        24                  617          e-177 <br />  Cathepsin                                  protein degradation                                      2                   449          e -126 <br />  Chitinase                                  egg hatching                                             7                   125          9e-29 <br />  C-type lectin                              unknown                                                  3                   48           4e-05 <br />  FAR                                        binding host fatty acids; reducing defense response   12                  177          4e-45 <br />  Glutathione peroxidase                     detoxification ROS                                    10                  360          e-100 <br />  PDI                                        protein folding                                          4                   706          0 <br />  Putative esophageal gland cell secretory   unknown                                                  10                  135          2e-32 <br />  protein 3 <br />  SXP-RAL2                                   unknown                                                  2                   105          1e-23 <br />  <br />  doi:10.1371/journal.pone.0091591.t004 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               7                                March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                         White Tip Nematode Transcriptome <br />  <br />  <br />  <br />  Table 5. Presence genes involved RNAi pathway. <br />  <br />  <br />                                          Number RNAi effector proteins                                                      A. besseyi contigs* <br />                                          A. besseyi      C. elegans         B. xylophilus              M. incognita <br />  <br />                                          clade 10B       clade 9A           clade 10B                  clade 12B <br />  <br />  Small RNA biosynthesis                  7               9                  8                          7                       112 <br />  dsRNA uptake spreading              6               11                 6                          4                       84 <br />  AGOs RISC                           27              30                 12                         9                       2212 <br />  RNAi inhibitors                         5               10                 2                          2                       73 <br />  Nuclear effectors                       12              15                 9                          6                       418 <br />  Total                                   57              75                 37                         28                      2899 <br />  <br />  *(bitscore.40 E-value,e-4). <br />  doi:10.1371/journal.pone.0091591.t005 <br />  <br />  <br /> and Table S22 File S1) identified A. besseyi. Proteins <br /> belonging C01A (41 contigs), I02 (89 contigs), M12A (43 <br /> contigs) S09X (110 contigs) families particularly                      Table 7. Summary peptidases A. besseyi. <br /> expanded A. besseyi. Peptidase families involved extracellular <br /> digestion lysosomal activities particularly expanded. <br />                                                                                 Catalytic types            Family number              Best E value <br />  <br /> Discussion                                                                      Aspartic                   2                          1.50E-106 <br />                                                                                 Cysteine                   19                         1.60E-108 <br />    This study provides detailed description the <br />                                                                                 Isoleucine                 6                          1.10E-55 <br /> transcriptome A. besseyi, rice-parasitic nematode a <br /> sequenced genome. We detected annotated repertoire                   Metallo                    26                         1.50E-148 <br /> candidate CAZymes A. besseyi transcriptome. Our dataset                  Serine                     16                         9.60E-100 <br /> extends knowledge arsenal enzymes A. besseyi.                  Threonine                  3                          5.50E-202 <br /> Cellulases classified 14 glycoside hydrolase families&#226;&#8364;&#8221;                 Unassigned                 1                          2.10E-46 <br /> GH5, 6, 7, 8, 9, 10, 12, 26, 44, 45, 48, 51, 61 74 [53] &#226;&#8364;&#8221;most               Total                      73 <br /> of typically produced plant pathogens plant <br /> parasites [54]. GH45 cellulases present Bursaphelenchus                  doi:10.1371/journal.pone.0091591.t007 <br /> species A. besseyi [13,16]. The discovery genes led to <br /> the hypothesis acquired fungi horizontal            fungal cell walls, detected. This enzyme identified in <br /> gene transfer (HGT) [40].                                                   various fungivorous plant-parasitic nematodes, in <br />    Although GH5 cellulase A. fragariae [55],              Bursaphelenchus species [56] recently A. avenae [57]. <br /> transcripts similar GH5 cellulases present A. besseyi        Because facultative fungal feeder, makes sense A. <br /> transcriptome. To confirm result, TBLASTN search A.               besseyi harbors considerable GH16. This result supports the <br /> fragariae GH5 cellulase protein sequences (NCBI accession                   possibility fungi-consuming nematodes acquired endo- <br /> Nos. AFD33557, AFD33558 AFI63769) A. besseyi                1,3-b-glucanase genes bacteria obtain fungal feeding <br /> contigs performed, hits E-value               ability subsequently acquired cellulase genes fungi, <br /> cutoff 1e-4. It worth noting small-scale transcriptome         permitted parasitize plants [56,58]. <br /> project A. besseyi published [16]. GH45 cellulase             Of particular detection candidate effector <br /> found transcriptome. However, transcripts similar             homologs cytokinins, ubiquitin extension protein, <br /> GH5 cellulases identified small-scale transcriptome.            calreticulin, 10A06 NULG1a [59] (Table S13 File S1), <br /> These results consistent study.                                genes thought involved process of <br />    In work, remarkable abundance GH16 (endo-1, 3-beta-             syncytia giant cell formation [47,60&#226;&#8364;&#8220;63]. Because A. <br /> glucanase), thought important degradation         besseyi does generate specific feeding cells, roles these <br />                                                                             genes indicate different function. In addition, effector <br />  Table 6. Neuropeptide genes A. besseyi.                                 identification small-scale transcriptome project A. besseyi <br />                                                                             indicated (12 13) candidate sequences <br />                                                                             matches Caenorhabditis elegans nematodes [16]. <br />  Neuropeptide                    Genes Contigs hit E-value level       These results suggested identified A. besseyi <br />                                                                             homologs really play roles effectors root knot <br />  Insulin-like peptides           8       25             0.003-8e-015 <br />                                                                             cyst nematodes parasitism open question. Obviously, <br />  FMRFamide-related peptides 17           54             5e-005-6e-028       studies homologs merited. <br />  Non-insulin, non-FLP            24      288            3e-005-2e-032          The present results provide insights genetic background <br />  peptides                                                                   A. besseyi. These insights useful future investigations of <br />  Total                           49      367                                nematode parasitism development techniques nematode <br />                                                                             control plant protection. <br />  doi:10.1371/journal.pone.0091591.t006 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              8                                      March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                                          White Tip Nematode Transcriptome <br />  <br />  <br />  <br /> Supporting Information                                                                      nents. Table S17, RNAi inhibitors. Table S18, Nuclear effectors. <br />                                                                                             Table S19, Neuropeptide genes encoding insulin-like peptides. <br /> File S1 Supporting Figures, Tables, Text. Figure S1,                                    Table S20, Neuropeptide genes encoding FMRFamide-related <br /> Length distribution All-Unigene. Figure S2, GC content                                   peptides. Table S21, Neuropeptide genes encoding non-insulin, <br /> frequencies distribution. Figure S3, RPKM distribution                                   non-FLP peptides. Table S22, Peptidases identified A. besseyi. <br /> transcript. Table S1, Expressed transcripts. Table S2, RPKM                              Text S1, Detailed methods RNAseq libraries construction. Text <br /> the A. besseyi transcripts. Table S3, The annotation A. besseyi                          S2, MIRA assembly yielded sequences our <br /> unigenes. Table S4, Metabolism related pathways. Table S5,                                  dataset. <br /> Genetic information processing related pathways. Table S6,                                  (DOC) <br /> Environmental information processing related pathways. Table <br /> S7, Cellular processes related pathways. Table S8, Organismal                               Files S2 Supplementary Dataset. The comparison of <br /> systems related pathways. Table S9, Glycoside hydrolases                                    unigenes A. besseyi transcriptome. <br /> identified A. besseyi. Table S10, Glycosyl transferases identified                       (XLSX) <br /> in A. besseyi. Table S11, Carbohydrate esterases identified A. <br /> besseyi. Table S12, Carbohydrate binding modules identified A.                           Author Contributions <br /> besseyi. Table S13, Candidate effectors potential parasitism                             Conceived designed experiments: FW DL ZW. Performed the <br /> genes identified A. besseyi. Table S14, Small RNA biosynthesis                           experiments: DL FW AD. Analyzed data: FW DL. Contributed <br /> proteins. Table S15, dsRNA uptake spreading effectors. Table                            reagents/materials/analysis tools: FW DL LL BW QC XL. Wrote the <br /> S16, Argonautes RNA-induced silencing complex compo-                                    paper: FW DL QC. <br />  <br /> References <br />  1. Chitwood DJ (2003) Research plant-parasitic nematode biology conducted                  Trichostrongylus colubriformis, using next-generation sequencing approach. Infect <br />     United States Department Agriculture-Agricultural Research Service. Pest               Genet Evol 10: 1199&#226;&#8364;&#8220;1207. <br />     Manage Sci 59: 748&#226;&#8364;&#8220;753.                                                                 22.   Cantacessi C, Mitreva M, Jex AR, Young ND, Campbell BE, et al. (2010) <br />  2. Fortuner R, Williams KJO (1975) Review literature Aphelenchoides besseyi                Massively parallel sequencing analysis Necator americanus transcrip- <br />     Christie, 1942, nematode causing &#226;&#8364;&#732;&#226;&#8364;&#732;White tip&#226;&#8364;&#8482;&#226;&#8364;&#8482; disease rice. Helmintho-                tome. PLoS Negl Trop Dis 4: e684. <br />     logical abstracts Series B, Plant Nematology 44: 1&#226;&#8364;&#8220;40.                                  23.   Choi YJ, Ghedin E, Berriman M, McQuillan J, Holroyd N, et al. (2011) A deep <br />  3. Rajan AL, Mathur VK, Lal A (1989) Improved culturing technique                            sequencing approach comparatively analyze transcriptome lifecycle <br />     Aphelenchoides besseyi inducing anhydrobiosis. Indian J Nematology 19: 10&#226;&#8364;&#8220;13.              stages filarial worm, Brugia malayi. PLoS Negl Trop Dis 5: e1409. <br />  4. Duncan LW, Moens M (2006) Migratory endoparasitic nematodes. In: Perry                  24.   Nicol P, Gill R, Fosu-Nyarko J, Jones MGK (2012) novo analysis and <br />     RN, Moens M, editors. Plant Nematology: CABI publishing. pp. 123&#226;&#8364;&#8220;152.                         functional classification transcriptome root lesion nematode, <br />  5. da Silva GS (1992) White tip national rice production. Informe                            Pratylenchus thornei, 454 GS FLX sequencing. International Journal for <br />     Agropecuario Belo Horizonte 16: 57&#226;&#8364;&#8220;59.                                                        Parasitology 42: 225&#226;&#8364;&#8220;237. <br />  6. Lin MS, Ding XF, Wang ZM, Hou FM, Lin N (2004) Description                           25.   Yoshida K, Hasegawa K, Mochiji N, Miwa J (2009) Early embryogenesis and <br />     Aphelenchoides besseyi abnormal rice &#226;&#8364;&#732;small grains erect panicles&#226;&#8364;&#8482;              anterior-posterior axis formation white-tip nematode Aphelenchoides besseyi <br />     symptom China. Rice Science 12: 289&#226;&#8364;&#8220;294.                                                   (Nematoda: Aphelenchoididae). Journal Nematology 41: 17&#226;&#8364;&#8220;22. <br />  7. Tiwari SP, Khare MN (2003) White tip caused Aphelenchoides besseyi,               26.   Wang F, Wang ZhY, Li DL, Chen QL (2012) Identification characteriza- <br />     important seed borne disease rice. In: Trivedi PC, editors. Advances                    tion Bursaphelenchus xylophilus (Aphelenchida: Aphelenchoididae) thermo- <br />     Nematology. Scientific Publishers. pp. 103&#226;&#8364;&#8220;114.                                               tolerance-related gene: Bx-HSP90. Int J Mol Sci 13: 8819&#226;&#8364;&#8220;8833. <br />  8. Togashi K, Hoshino S (2001) Distribution <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> mortality white tip         27.   Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, et al. (2011) Full- <br />     nematode, Aphelenchoides besseyi (Nematoda: Aphelenchoididae), rice seeds.              length Transcriptome assembly RNA-Seq data reference <br />     Nematology 3:17&#226;&#8364;&#8220;24.                                                                           genome. Nat Biotechnol 29: 644&#226;&#8364;&#8220;652. <br />  9. Wang KM, Tsay TT, Lin YY (1993) The occurrence Aphelenchoides besseyi             28.   Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B (2008) Mapping and <br />     strawberry ecology Taiwan. Plant Prot Bull 35: 14&#226;&#8364;&#8220;29.                              quantifying mammalian transcriptomes RNA-Seq. Nat Methods 5: 621&#226;&#8364;&#8220;628. <br />                                                                                             29.   Martin J, Abubucker S, Heizer E, Taylor CM, Mitreva M (2012) Nematode.net <br /> 10. Yu PC, Tsay TT (2004) Occurrence foliar nematode disease fern in <br />                                                                                                   update 2011: addition data sets tools featuring next-generation <br />     Taiwan. Plant Pathol Bull 13: 35&#226;&#8364;&#8220;44. <br />                                                                                                   sequencing data. Nucleic Acids Res 40: D720&#226;&#8364;&#8220;D728. <br /> 11. Tsay TT, Cheng YH, Chen HY, Lin YY, Wu WS (1995) Occurrence and <br />                                                                                             30.   Kanehisa M, Goto S (2000) KEGG: Kyoto Encyclopedia Genes and <br />     control nematode diseases bulbous flowers. Plant Pathol Bull 4: 180&#226;&#8364;&#8220;192. <br />                                                                                                   Genomes. Nucleic Acids Res 28: 27&#226;&#8364;&#8220;30. <br /> 12. Van Megen H, Van den Elsen S, Holterman M, Karssen G, Mooyman P, et al. <br />                                                                                             31.   Moriya Y, Itoh M, Okuda S, Yoshizawa AC, Kaehisa M (2007) KAAS: an <br />     (2009) A phylogenetic tree nematodes based 1200 full-length small <br />                                                                                                   automatic genome annotation pathway reconstruction server. Nucleic Acids <br />     subunit ribosomal DNA sequences. Nematology 11: 927&#226;&#8364;&#8220;950. <br />                                                                                                   Res 35 (suppl 2):W182&#226;&#8364;&#8220;W185. <br /> 13. Kikuchi T, Cotton JA, Dalzell JJ, Hasegawa K, Kanzaki N, et al. (2011)                  32.   Park BH, Karpinets TV, Syed MH, Leuze MR, Uberbacher EC (2010) <br />     Genomic insights origin parasitism emerging plant pathogen                 CAZymes Analysis Toolkit (CAT): Web-service searching analyzing <br />     (Bursaphelenchus xylophilus). PLoS Pathog 7: e1002219.                                        carbohydrate-active enzymes newly sequenced organism using CAZy <br /> 14. Haegeman A, Joseph S, Gheysen G (2011) Analysis transcriptome                   database. Glycobiology 20: 1574&#226;&#8364;&#8220;1584. <br />     root lesion nematode (Pratylenchus coffeae) generated 454 sequencing                 33.   Haegeman A, Mantelin S, Jones JT, Gheysen G (2012) Functional roles of <br />     technology. Molecular &amp; Biochemical Parasitology 178: 7&#226;&#8364;&#8220;14.                                   effectors plant-parasitic nematodes. Gene 492: 19&#226;&#8364;&#8220;31. <br /> 15. Cheng X, Xiang Y, Xie H, Xu CL, Xie TF, et al. (2013) Molecular                         34.   Haegeman A, Bauters L, Kyndt T, Rahman MM, Gheysen G (2013) <br />     Characterization Functions Fatty Acid Retinoid Binding Protein                     Identification candidate effector genes transcriptome rice root <br />     Gene (Ab-far-1) Aphelenchoides besseyi. PLoS ONE 8: e66011.                                knot nematode Meloidogyne graminicola. Molecular Plant Pathology 14: 379&#226;&#8364;&#8220;390. <br /> 16. Kikuchi T, Cock PJA, Helder J, Jones JT (2013) Characterisation                  35.   Petersen TN, Brunak S, Von Heijne G, Nielsen H (2011) SignalP 4.0: <br />     transcriptome Aphelenchoides besseyi identification GHF 45 cellulase.             discriminating signal peptides transmembrane regions. Nature Methods 8: <br />     Nematology 16: 99&#226;&#8364;&#8220;107.                                                                        785&#226;&#8364;&#8220;786. <br /> 17. Jones JT, Haegeman A, Danchin EGJ, Gaur HS, Helder J, et al. (2013) Top 10              36.   Krogh A, Larsson B, Von Heijne G, Sonnhammer ELL (2001) Predicting <br />     plant-parasitic nematodes molecular plant pathology. Molecular Plant                       transmembrane protein topology hidden markov model: application to <br />     Pathology 14: 946&#226;&#8364;&#8220;961.                                                                        complete genomes. J Mol Biol 305: 567&#226;&#8364;&#8220;580. <br /> 18. Schuster SC (2008) Next-generation sequencing transforms today&#226;&#8364;&#8482;s biology. Nat           37.   Dalzell JJ, McVeigh P, Warnock ND, Mitreva M, Bird DM, et al. (2011) RNAi <br />     Methods 5: 16&#226;&#8364;&#8220;18.                                                                             Effector Diversity Nematodes. PLoS Negl Trop Dis 5: e1176. <br /> 19. Cantacessi C, Gasser RB, Strube C, Schnieder T, Jex AR, et al. (2011) Deep              38.   Rawlings ND, Morton FR (2008) The MEROPS batch BLAST: tool detect <br />     insights Dictyocaulus viviparus transcriptomes provides unique prospects             peptidases non-peptidase homologues genome. Biochimie 90: <br />     new drug targets disease intervention. Biotechnol Adv 29: 261&#226;&#8364;&#8220;271.                        243&#226;&#8364;&#8220;259. <br /> 20. Cantacessi C, Jex AR, Hall RS, Young ND, Campbell BE, et al. (2010) A                   39.   De Boer JM, Yan Y, Smant G, Davis EL, Baum TJ (1998) In situ hybridization <br />     practical, bioinformatic workflow large data sets generated                messenger RNA Heterodera glycines. J Nematol 30: 309&#226;&#8364;&#8220;312. <br />     generation sequencing. Nucleic Acids Res 38: e171.                                      40.   Bakhetia M, Urwin PE, Atkinson HJ (2007) QPCR analysis RNAi define <br /> 21. Cantacessi C, Mitreva M, Campbell BE, Hall RS, Young ND, et al. (2010) First                  pharyngeal gland cell-expressed genes (Heterodera glycines) required initial <br />     transcriptomic analysis economically important parasitic nematode,                     interactions host. Mol Plant-Microbe Interact 20: 306&#226;&#8364;&#8220;312. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                              9                                    March 2014 | Volume 9 | Issue 3 | e91591 <br />                                                                                                                                            White Tip Nematode Transcriptome <br />  <br />  <br /> 41. Kikuchi T, Jones JT, Aikawa T, Kosaka H, Ogura N (2004) A family glycosyl                 54. Lynd LR, Weimer PJ, Van Zyl WH, Pretorius IS (2002) Microbial cellulose <br />     hydrolase family 45 cellulases pine wood nematode Bursaphelenchus                       utilization: fundamentals biotechnology. Microbiol Mol Biol Rev 66: 506&#226;&#8364;&#8220; <br />     xylophilus. FEBS Letters 572: 201&#226;&#8364;&#8220;205.                                                           577. <br /> 42. Davis EL, Hussey RS, Mitchum MG, Baum TJ (2008) Parasitism proteins                       55. Fu Zh, Agudelo P, Wells CE (2012) Differential expression b-1,4- <br />     nematode-plant interactions. Current Opinion Plant Biology 11: 360&#226;&#8364;&#8220;366.                       Endoglucanase induced diet change foliar nematode Aphelenchoides <br /> 43. Elling AA, Mitreva M, Gai XW, Martin J, Recknor J, et al. (2009) Sequence                        fragariae. Phytopathology 102:804&#226;&#8364;&#8220;811. <br />     mining transcript profiling explore cyst nematode parasitism. BMC                     56. Kikuchi T, Shibuya H, Jones JT (2005) Molecular biochemical <br />     Genomics 10:58.                                                                                  characterization endo-beta-1,3-glucanase pinewood nematode <br /> 44. Roze E, Hanse B, Mitreva M, Vanholme B, Bakker J, et al. (2008) Mining                       Bursaphelenchus xylophilus acquired horizontal gene transfer bacteria. <br />     secretome root-knot nematode Meloidogyne chitwoodi candidate                          Biochem J 389:117&#226;&#8364;&#8220;125. <br />     parasitism genes. Molecular Plant 9: 1&#226;&#8364;&#8220;10.                                                   57. Karim N, Jones JT, Okada H, Kikuchi T (2009) Analysis expressed sequence <br /> 45. Jacob J, Mitreva M, Vanholme B, Gheysen G (2008) Exploring                                   tags identification genes encoding cell-wall-degrading enzymes the <br />     transcriptome burrowing nematode Radopholus simi <br /> </body></html>