<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Antiviral RNA Silencing Initiated Absence RDE-4, Double- <br /> Stranded RNA Binding Protein, Caenorhabditis elegans <br /> Xunyang Guo, Rui Zhang, Jeffrey Wang, Rui Lu <br /> Department Biological Sciences, Louisiana State University, Baton Rouge, Louisiana, USA <br />  <br /> Small interfering RNAs (siRNAs) processed double-stranded RNA (dsRNA) virus origins mediate potent antiviral de- <br /> fense process referred <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) RNA silencing diverse organisms. In simple inverte- <br /> brate Caenorhabditis elegans, RNAi process initiated single Dicer, partners dsRNA binding protein <br /> RDE-4 process dsRNA viral siRNAs (viRNAs). Notably, C. elegans RNA-directed viral immunity (RDVI) re- <br /> quires number worm-specific genes antiviral potential. One gene rsd-2 (RNAi spreading defective 2), which <br /> was implicated RDVI previous studies. In current study, established antiviral role showing rsd-2 <br /> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>null mutants</span> permitted higher levels viral RNA accumulation, enhanced viral susceptibility reversed ecto- <br /> pic expression RSD-2. We examined relationship rsd-2 known components RNAi pathways es- <br /> tablished rsd-2 functions novel pathway independent rde-4 likely requires RNA-dependent RNA poly- <br /> merase RRF-1, suggesting critical role RSD-2 secondary viRNA biogenesis, likely coordinated action RRF-1. <br /> Together, results suggest RDVI single-Dicer organism C. elegans depends collective actions RDE- <br /> 4-dependent RDE-4-independent mechanisms produce RNAi-inducing viRNAs. Our study reveals, time, a <br /> novel siRNA-producing mechanism C. elegans bypasses need dsRNA-binding protein. <br />  <br />  <br />  <br /> I n fungi, plants, insects, nematodes, double-stranded RNAs <br />   (dsRNAs), formed complementary viral transcripts or <br /> through intramolecular <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span> pairing, trigger potent antiviral de- <br />                                                                                       (3, 5, 19&#226;&#8364;&#8220;21). Interestingly, RDE-4, key factor RDVI, also <br />                                                                                       conserved fruit fly, genome known encode two <br />                                                                                       Dicer proteins dedicated function biogenesis siRNA <br /> fense process referred RNA interference                         miRNA, respectively (7, 20&#226;&#8364;&#8220;23). So far, closely <br /> (RNAi) (1). Increasing evidence suggested RNA-directed                      related AGO proteins, RDE-1 (RNAi defective 1) C04F12.1, <br /> viral immunity (RDVI) initiated processing viral                       play important role RDVI C. elegans <br /> dsRNA small interfering RNAs (siRNAs) RNase III-like                       (3&#226;&#8364;&#8220;5, 24). RDE-1 slicer activity required the <br /> enzyme called Dicer (2). Subsequently, virus-derived                            maturation RISC cleavage target transcript <br /> siRNAs incorporated RNA-induced silencing complexes                          (25). Currently, C04F12.1 contributes RDVI remains <br /> (RISCs), formed Argonaute protein cofactors, serve                      largely unknown. Efficient RDVI C. elegans requires an <br /> sequence guides target viral transcript selection destruc-                    RdRP, termed RRF-1, produces 22-nucleotide (nt) single- <br /> tion (1). In diverse organisms, dsRNA binding proteins essen-                     stranded secondary siRNAs Dicer-independent manner (4, 5, <br /> tial components RDVI contribute biogenesis                          26, 27) (28). The fact RDE-1 required production of <br /> function virus-derived siRNAs (viRNAs) (3&#226;&#8364;&#8220;8). In plants,                           secondary siRNAs suggests RRF-1 functions downstream of <br /> RDVI potentiated secondary viRNAs processed                        RDE-1 genetic pathway (28, 29). <br /> dsRNAs converted cleaved viral transcripts RNA-depen-                             In addition Dicer, AGO, RdRP proteins, worm <br /> dent RNA polymerases (RdRPs) (9&#226;&#8364;&#8220;12). Plant RDVI involves                         RDVI requires unique components, DRH-1 <br /> an intercellular signal believed restrict systemic spread-                 (Dicer-related RNA helicase 1) RSD-2 (RNAi spreading de- <br /> ing invading virus priming antiviral status prior                     fective 2) (5). DRH-1 encodes putative DEAD box RNA helicase <br /> virus arrival (13, 14). Recently, siRNAs serve                   shares significant sequence homology RIG-I (retinoic <br /> physical carrier mobile silencing signal plants (15, 16). In               acid-inducible gene 1), cytosolic virus sensor initiates inter- <br /> the fruit fly RDVI spreads systemically, mecha-                     feron-mediated viral immunity virus detection mammals <br /> nism involved different (17).                                                      (30). Intriguingly, essential RDVI, DRH-1 appears <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />     Since RNAi triggered artificial dsRNA mechanistically reca-                    dispensable RNAi triggered artificial dsRNAs, suggesting <br /> pitulates RDVI, previous studies artificial dsRNA-triggered                        dedicated role DRH-1 antiviral defense (5). Previously, we <br /> RNAi C. elegans significantly improved understanding                      reported viRNAs detected drh-1 mutants undetect- <br /> of RDVI nematode worms. Current models envision                           able drh-1;rde-4 double mutants (5), suggesting DRH-1, <br /> worm RDVI initiated processing viral dsRNA into <br /> primary viRNAs worm Dicer, called DCR-1 (4, 5, 18). <br /> DCR-1 processes precursor microRNAs (miRNAs) ma-                               Received 15 May 2013 Accepted 18 July 2013 <br /> ture miRNAs, class endogenous small RNAs important                             Published ahead print 24 July 2013 <br /> roles development (19). Whereas efficient processing viral                         Address correspondence Rui Lu, ruilu@lsu.edu. <br /> dsRNA primary viRNA requires dsRNA binding protein,                               Copyright &#194;&#169; 2013, American Society Microbiology. All Rights Reserved. <br /> termed RDE-4 (RNAi defective 4), processing precursor                             doi:10.1128/JVI.01305-13 <br /> miRNAs mature miRNAs appears RDE-4 independent <br />  <br />  <br /> October 2013 Volume 87 Number 19                                           Journal Virology   p. 10721&#226;&#8364;&#8220;10729                                         jvi.asm.org   10721 <br />  Guo et al. <br />  <br />  <br />  <br /> together RDE-1 RRF-1, functions genetic                   licon. The sur-5 promoter, known active worm cells <br /> pathway RDE-4.                                                              (41), used drive expression RSD-2. A construct directs <br />     RSD-2, SID-1 (systemic RNA interference defi-                mCherry expression pharynx tissue used produce visual <br /> cient 1), originally identified key factors             mark Psur5::RSD-2 transgene. The RSD-2 function rescue assay <br /> enable systemic spreading RNAi (31&#226;&#8364;&#8220;33). A recent study sug-                 began microinjection Psur5::RSD-2 construct gonad of <br /> gested RSD-2 contributes accumulation secondary                 rsd-2;FR1gfp worms mCherry reporter construct. Be- <br /> siRNAs exogenous endogenous RNAi pathways (34).                    cause nature worm transformation gonad injection, <br /> Since rsd-2 mutants displayed defects transposon silencing un-              transgenic extrachromosomal arrays randomly passed on <br /> der unfavorable conditions, believed RSD-2 plays im-             generations. As result, progenies free extra- <br />                                                                                chromosomal arrays serve internal negative control. Upon heat <br /> portant role maintaining chromosome integrity (35). SID-1 <br />                                                                                induction, initiate replication FR1gfp replicon, trans- <br /> contains multiple putative transmembrane domains been <br />                                                                                genic worms marked red fluorescence head or <br /> shown required dsRNA intake (33, 36, 37). Interestingly,             decreased green fluorescence Psur5::RSD-2 transgene able re- <br /> a recent study suggested classes silencing                store function RSD-2 rsd-2 mutants. <br /> dsRNAs C. elegans tissues act generate                 RNA preparation Northern analysis. Total RNA prepared 48 <br /> the systemic silencing signal (38). Currently, SID-1 con-              h heat induction using TRI reagent following manufacturer&#226;&#8364;&#8482;s <br /> tributes systemic RNAi transporting classes                 instructions (Sigma-Aldrich, Inc.). Small RNA species total RNA <br /> silencing RNAs remains largely unknown.                                        samples enriched using mirVana kit (Ambion). For the <br />     Previously, high concentrations dsRNAs trig-              detection high-molecular-weight viral RNAs, 4 &#226;?&#174;g total RNA each <br /> ger efficient RNAi absence RDE-4 (39). Currently,                sample fractionated 1.2% agarose gel. For detection small <br /> biological significance observation remains largely un-                RNA, including viRNAs miRNAs, 10 20 &#226;?&#174;g small RNA per <br /> known. By analyzing viral replication viRNA accumulation                sample resolved 15% acrylamide denaturing gel. After electropho- <br /> worm single double mutants defective rsd-2,                resis, RNA samples transferred Hybond N&#226;&#171;&#185; membrane <br /> that RSD-2, important component worm RDVI, func-                      (GE Healthcare Inc.) UV cross-linked using 1.8 &#226;&#171;&#187; 105 &#226;?&#174;J/cm2 the <br /> tions downstream primary viRNA biogenesis. Most impor-                      output power (SpectroLinker). <br /> tantly, increase viral replication, primary viRNAs accu-                The blots detection high-molecular-weight viral RNAs were <br /> mulated higher level rsd-2;rde-4 double mutants                hybridized alkaline phosphatase-labeled cDNA probes prepared us- <br /> rsd-2 single mutants, suggesting rde-4-independent mecha-                   ing AlkPhos direct labeling module (GE Healthcare). The probes for <br /> nism biogenesis primary viRNAs C. elegans. We fur-               FR1gfp transcript detection prepared using cDNA fragments ampli- <br />                                                                                fied green fluorescent protein (GFP) coding sequence within <br /> ther showed RRF-1, RDE-1, contributed to <br />                                                                                FR1gfp. The probes used detect Orsay virus RNA1 prepared using <br /> the rde-4-indepedent RDVI. Notably, playing essen- <br />                                                                                cDNA fragments amplified, RT-PCR, 3= end Orsay <br /> tial role facilitating spreading RNAi triggered artificial <br />                                                                                virus RNA1 genome. The hybridization performed 65&#194;&#176;C overnight. <br /> dsRNAs, SID-1 appears dispensable RDVI targeting natural                  After washing 65&#194;&#176;C times, labeled cDNA probes detected <br /> viral pathogen C. elegans.                                                  using CDP-Star detection reagent following manufacturer&#226;&#8364;&#8482;s in- <br />                                                                                structions (GE Healthcare). <br /> MATERIALS AND METHODS                                                              The detection siRNAs miRNAs adopted protocol described <br /> Worm genetics. The Bristol isolate C. elegans, N2, used          previously (40). For detection FR1gfp-derived siRNAs, DNA oligo- <br /> reference strain study. All strains used study de-   nucleotides covering entire subgenomic RNA sequence labeled <br /> rived N2 include drh-1 (tm1329), drh-2 (ok951), rde-1 (ne300),        using digoxigenin (DIG) oligonucleotide tailing kit (Roche Applied <br /> rde-4 (ne337), rsd-2 (tm1429 pk3307), sid-1 (qt2), rrf-1 (pk1417)      Science). DNA oligonucleotides sequence ATTGCCGTACTGA <br /> mutants. The genotypes drh-1, drh-2, rsd-2, rrf-1 mutants          ACGATCTCA labeled used detection miR-58. Four <br /> confirmed using PCR. The primers used confirm tm1329 drh-2              DNA oligonucleotides sizes 19, 21, 23, 25 nt, respectively, were <br /> alleles described previously (5). The primer pairs used confirm    detected using DIG-labeled DNA oligonucleotides and, with <br /> alleles tm1429intf (CTACTAAACCGGTTACGTGT) tm1429intb                   miR-58, served size references. <br /> (CCACAGGGATTTTGTAGGGA) tm1429 1417EcoRV (AGGAG                             Infectious filtrate preparation Orsay virus inoculation. Orsay <br /> AGCATAGAAGGATATCA) 1417PfoI (GTCACGGGGAGCTATTG                             virus maintained using JU1580 isolate C. elegans room tem- <br /> TGAA) pk1417. The genotypes rde-1, sid-1, rde-4 mutants        perature following protocol described previously (6). To prepare <br /> confirmed using skn-1 feeding RNAi combined genomic DNA se-               Orsay virus inoculum, JU1580 worms infected Orsay virus were <br /> quencing. NGM plates seeded Escherichia coli strain OP50 used        washed off, using M9 buffer, slightly starved 10-cm agar plates. After <br /> to maintain worm strains. Single double mutants containing var- <br />                                                                                spinning low speed, virus-containing supernatant filtered <br /> ious transgenes created standard genetic crosses. <br />                                                                                0.22-&#226;?&#174;m filter <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (Millipore), filtrate used to <br />     Plasmid constructs transgenic worms. The construction of <br />                                                                                resuspend pelleted OP50 E. coli NGM plate seeding. <br /> FR1gfp replicon described previously (5). Psur5::RSD-2 devel- <br /> oped inserting RSD-2 coding sequence, amplified reverse                 RNAi experiments. Feeding RNAi targeting endogenous gene <br /> transcription-PCR (RT-PCR), LR50 vector described previously          skin-1 transgene gfp performed using bacterial feeding pro- <br /> (40). The RSD-2 coding sequence resulting construct con-            tocol described previously (42). NGM agar plates containing 5 mM iso- <br /> firmed DNA sequencing. The plasmid construct used drive gfp              propyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG) 50 mg/ml carbenicillin <br /> dsRNA expression E. coli described previously (40). The chromo-         used prepare feeding RNAi food. All feeding RNAi experi- <br /> somal integrant Psur5::RSD-2 generated using protocol de-            ments carried room temperature. <br /> scribed previously (40).                                                           Imaging microscopy. The green red fluorescence images were <br />     RSD-2 function rescue assay. The RSD-2 function rescue assay uti-          recorded using exposure set images. A Nikon digital <br /> lized ectopic expression RSD-2 coding sequence rsd-2 null mutants        camera p7000 mounted Nikon SMZ1500 microscope used to <br /> (tm1429) carrying nuclear transgene corresponding FR1gfp rep-         record images. <br />  <br />  <br /> 10722    jvi.asm.org                                                                                                                      Journal Virology <br />                                                                                                               RDE-4-Independent Antiviral Silencing <br />  <br />  <br />  <br /> RESULTS                                                                 23 nt. FR1gfp-derived siRNAs detected rsd-2 mutants <br /> rsd-2 essential biogenesis primary viRNAs. rsd-2      <span id='am-241' about='xsp:pattern' typeof='owl:Thing'>pattern</span> similar rde-1 mutants visibly lower <br /> of C. elegans implicated RDVI RNAi screen       level. Previously, shown primary siRNAs pro- <br /> that selected genes downregulation dsRNA feeding           duced worm Dicer predominantly 23 nt size accu- <br /> led loss RDVI (5). To confirm loss RDVI worms      mulate high levels rde-1 mutants (18, 29, 43, 44). This result <br /> fed rsd-2-targeting dsRNA caused partial loss        suggested rsd-2 essential biogenesis primary <br /> of rsd-2 function, examined replication FR1gfp replicon       viRNAs. <br /> in mutant worms containing tm1429 pk3307 allele               rsd-2 contributes rde-4-independent antiviral silencing. <br /> rsd-2. The tm1429 allele contains 251-nt deletion, spanning      We previously reported FR1gfp-derived siRNAs detected in <br /> nt 1376 1626 rsd-2 cDNA, predicted produce            drh-1 mutants undetectable drh-1;rde-4 double mu- <br /> truncated RSD-2 contains 458 amino acid (aa)        tants, suggesting drh-1 rde-4 function linear genetic <br /> residues 1,266-aa wild-type RSD-2. Conversely, pk3307        pathway (5). To rsd-2 functions same <br /> allele contains premature stop codon RSD-2 coding se-          genetic pathway, compared accumulation primary <br /> quence caused truncation 542 aa RSD-2.          viRNAs double mutants corresponding drh-1;rde-4, rsd-2; <br /> Thus, rsd-2 alleles considered null alleles. The FR1gfp        rde-4, rsd-2;drh-1. As shown Fig. 2A, detect- <br /> replicon developed flock house virus (FHV) RNA1             able drh-1;rde-4 double mutants, FR1gfp-derived primary <br /> replacing coding sequence B2, FHV-encoded RNAi sup-          siRNAs detected rsd-2;drh-1 rsd-2;rde-4 double <br /> pressor, green fluorescence protein (GFP). As result,    mutants (compare lane 3 lanes 1 4). Most importantly, an <br /> the replication FR1gfp subdued RDVI wild-type N2            enhanced replication Orsay virus FR1gfp ob- <br />                                                                         served rsd-2;rde-4 rsd-2;drh-1 double mutants compared to <br /> worms rescued RDVI-defective worms, rde-1 <br />                                                                         rde-4 drh-1 single mutants, respectively (Fig. 2B C, left). <br /> mutants. As shown Fig. 1A, similar rde-1 mutants, both <br />                                                                         These observations suggested rde-4-independent <br /> rsd-2 mutants supported elevated accumulation FHV RNAs <br />                                                                         mechanism directs antiviral silencing rsd-2-dependent <br /> (Fig. 1A, compare lanes 3 4 lanes 1 2), confirming a <br />                                                                         manner C. elegans. RDE-1 specifically recruits primary siRNAs, <br /> critical role wild-type RSD-2 FHV-targeting RDVI. En- <br />                                                                         secondary siRNAs, target RNA selection be- <br /> hanced accumulation observed rsd-2 mutants for <br />                                                                         lieved function downstream RDE-4 (29). Thus, expected <br /> Orsay virus, natural viral pathogen C. elegans closely <br />                                                                         enhanced viral replication rsd-2;rde-1 double mutants <br /> related FHV (Fig. 1B, compare lanes 2 3 lane 1) (6). These <br />                                                                         compared rde-1 single mutants rsd-2 contributes to <br /> results strongly suggested RSD-2 plays important <br />                                                                         RDVI rde-4-independent manner. As shown Fig. 2B and <br /> role RDVI. <br />                                                                         C, right, enhanced viral replication observed, a <br />     To reconfirm loss RDVI rsd-2 mutants          lesser extent, rsd-2;rde-1 double mutants compared rde-1 <br /> resulted rsd-2 null alleles closely linked   single mutants. As expected, FR1gfp-derived siRNAs accumulated <br /> genetic alleles, checked ectopic expression wild-type          rsd-2;rde-1 double mutants (Fig. 2A, lane 2). Taken together, <br /> RSD-2 restores RDVI rsd-2 mutants (tm1429) containing            results strongly suggest rde-4-independent <br /> FR1gfp replicon. The ectopic expression wild-type RSD-2 cod-         rsd-2-dependent pathway contributing RDVI C. elegans. <br /> ing sequence achieved gonad microinjection plas-             rsd-2 rrf-1 function RDVI pathways. rrf-1 <br /> mid construct containing RSD-2 coding sequence driven            encodes RdRP initiates novo synthesis secondary <br /> sur-5 promoter (Fig. 1C) (41). A plasmid construct directs         siRNAs using targeted transcript template RNAi (26, <br /> mCherry expression pharynx tissue coinjected produce        27, 45, 46). To rsd-2 functions genetic <br /> visual reporter transgene. As shown Fig. 1C, FR1gfp          pathway rrf-1, known function downstream rde-4 <br /> replication, manifested green fluorescence, suppressed        rde-1 (28, 29), checked viral replication rsd-2;rrf-1 dou- <br /> rsd-2 mutants containing Psur-5::RSD-2 transgene (compare           ble mutants compared single mutants. We expected see <br /> the worms showed red fluorescence heads worms            enhanced viral replication double mutants rsd-2 rrf-1 <br /> that did not). Consistent observation, FR1gfp transcripts     function separate genetic pathways. However, shown Fig. <br /> were detected reduced level worms containing Psur-5::       3A, enhancement replication FR1gfp Orsay virus <br /> RSD-2 transgene (Fig. 1D, compare lane 2 lane 3). These results      observed double mutants compared single <br /> confirmed RSD-2 plays important role worm             mutants (compare lane 4 lanes 2 3 left panel lane <br /> RDVI.                                                                   5 lanes 3 4 right panel). Consistently, FR1gfp-derived <br />     rsd-2 known contribute accumulation secondary,      23-nt primary siRNAs accumulated comparable levels the <br /> but primary, endogenous siRNAs (34). To            rsd-2;rrf-1 double mutants single mutants (Fig. 3B, <br /> RSD-2 contributes RDVI similar mechanism,               compare lanes 1 2 lane 7). This result, the <br /> checked accumulation viRNAs rsd-2 mutants (tm1429)            results shown Fig. 2, suggest rrf-1 rsd-2 function <br /> using Northern blotting. Both rde-1 rde-4 play essential roles      independent genetic pathways mediate rde-4-depen- <br /> in worm RDVI. However, virus-derived siRNAs accumulate               dent rde-4-independent antiviral silencing, respectively. <br /> readily detectable levels rde-1 rde-4 mutants (3, 5).            rrf-1 contributes rde-4-independent antiviral silencing. <br /> Thus, control, accumulation viRNAs rde-1 rde-4       To confirm rrf-1 contributes rde-4-independent <br /> mutants detected test. As shown Fig. 1E, al-        RDVI, compared replication FR1gfp Orsay <br /> though detectable wild-type N2 worms rde-4 mutants,           virus rrf-1;rde-4 rrf-1;drh-1 double mutants the <br /> FR1gfp-derived viRNAs detected high levels rde-1 mu-         single mutants corresponding rrf-1, drh-1, rde-4. We ex- <br /> tants, major band detected position corresponding      pected enhanced viral replication double mutants if <br />  <br />  <br /> October 2013 Volume 87 Number 19                                                                                               jvi.asm.org 10723 <br />  Guo et al. <br />  <br />  <br />  <br />  <br /> FIG 1 rsd-2 essential biogenesis primary viRNA. (A) Northern blot showing accumulation FR1gfp genomic subgenomic transcripts in <br /> different genetic backgrounds. Total RNA extracted heat-induced worms 48 h heat induction. The viral transcripts detected using probes prepared <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> gfp cDNA. RNA1, genomic RNA FR1gfp; RNA3, subgenomic RNA FR1gfp; rRNA, methylene blue-stained rRNA serving equal loading <br /> control. (B) Accumulation Orsay virus RNA1 (ovRNA1) wild-type N2 worms rsd-2 mutants carrying tm1429 NL3307 allele indicated. Total RNA was <br /> prepared 72 h viral inoculation. Orsay virus RNA1 detected using cDNA probes prepared using cDNA fragment amplified 3= end Orsay virus <br /> RNA1. (C) The upper panel shows structure Psur-5::rsd-2. Psur-5, promoter endogenous gene sur-5; RSD-2, coding sequence wild-type rsd-2; <br /> UTR, 3=-end untranslated region unc-54. The lower panel shows visualization green fluorescence rsd-2 mutants (tm1429 allele) carrying FR1gfp replicon <br /> transgene extrachromosmal array corresponding Psur-5::RSD-2 48 h heat induction. Shown merged images recorded, using exposure, <br /> under red fluorescence green fluorescence. Worms showing red color transgenic Psur-5::RSD-2. (D) Accumulation FR1gfp transcripts wild-type N2 <br /> worms, drh-1 mutants, rsd-2 mutants (tm1429) extrachromosomal assay derived Psur-5::RSD-2 construct. RSD-2wt denotes the <br /> Psur-5::RSD-2 transgenic array. (E) Accumulation FR1gfp-derived siRNAs wild-type N2 worms RDVI-defective mutants indicated. Lane M, DNA <br /> oligonucleotides sizes 19, 21, 23, 25 nt detected used, miR-58, size references. <br />  <br />  <br />  <br /> 10724    jvi.asm.org                                                                                                                               Journal Virology <br />                                                                                                                                RDE-4-Independent Antiviral Silencing <br />  <br />  <br />  <br />  <br /> FIG 2 rsd-2 contributes rde-4-independent antiviral silencing. (A) Accu- <br /> mulation FR1gfp-derived siRNAs double mutants defective RDVI.            FIG 3 rsd-2 rrf-1 function antiviral silencing pathways. (A) <br /> Experimental details described legend Fig. 1E.    Accumulation FR1gfp Orsay virus transcripts various genetic back- <br /> (B) Accumulation Orsay virus RNA1 (ovRNA1) wild-type N2 worms           grounds indicated. See legend Fig. 1A experimental details. (B) <br /> RDVI-defective mutants carrying single double mutations indicated. See      The accumulation FR1gfp-derived siRNAs single double mutants con- <br /> the legend Fig. 1B experimental details. (C) Accumulation FR1gfp        taining rsd-2 and/or rrf-1 null alleles. See legend Fig. 1E experimental <br /> transcripts various genetic backgrounds indicated. See legend Fig.   details. <br /> 1A experimental details. <br />  <br />  <br />                                                                                   FR1gfp replicated higher levels double mutants the <br /> rrf-1 contributes rde-4-independent RDVI. As shown                   single mutants. Most importantly, FR1gfp-derived primary <br /> Fig. 4A B, FR1gfp Orsay virus replicated higher                   siRNAs, detectable drh-1;rde-4 double mu- <br /> levels double mutants single mutants. Most                     tants, readily detected rde-1;rde-4 double mutants (Fig. <br /> importantly, detected FR1gfp-derived primary siRNAs                     5B, compare lane 3 lane 2). These results suggested that <br /> rrf-1;rde-4 rrf-1;drh-1 double mutants, contrast               rde-1 plays role rde-4-independent RDVI. <br /> seen drh-1;rde-4 double mutants (Fig. 4C, compare lanes 2                      sid-1 required RDVI targeting Orsay virus. In C. <br /> 4 lane 1). As shown Fig. 4A B, increase FR1gfp                    elegans, RNAi triggered artificial long dsRNA spreads systemi- <br /> replication observed rrf-1;rde-1 double mutants com-                   cally causes sequence-specific silencing distant tissues that <br /> pared single mutants, increase                    exposed dsRNA trigger (32). This observation raised <br /> clear Orsay virus. As expected, FR1gfp-derived primary                        question whether, like plants (13, 14), worm RDVI <br /> siRNAs accumulated rde-1;rrf-1 double mutants (Fig. 4C,                   involves intercellular signal prevents viruses spread- <br /> lane 3). These observations confirmed rrf-1 plays role                  ing systemically. Previously, shown RDVI triggered <br /> rde-4-independnet RDVI.                                                           FHV replicon compromised sid-1 mutants (40) <br />     rde-1 plays role rde-4-independent RDVI. Previously, en-                 defective RNAi spreading (33, 36, 47). However, potent <br /> hanced viral replication observed drh-1;rde-1,                  RDVI triggered individual cell con- <br /> drh-1;rde-4, double mutants compared single mutants (5),                   tains replicon transgene, systemic silencing signal be <br /> suggesting role rde-1 rde-4-independent RDVI. To                    needed silence viral replicon known move <br /> out rde-1 contributes rde-4-independent                         systemically. To worm RDVI spreads <br /> RDVI, compared viral replication primary viRNA                        systemically restrict virus spread natural conditions, we <br /> accumulation rde-1;rde-4 double mutants single mu-                     compared Orsay virus infection wild-type N2 worms and <br /> tants. We expected enhanced viral replication accumu-                  worms defective sid-1. As shown Fig. 6, increase Orsay <br /> lation primary viRNAs rde-1 plays role rde-4-                   virus replication detected sid-1 mutants compared to <br /> independent RDVI. As shown Fig. 5A, Orsay virus                       wild-type N2 worms (compare lane 4 lane 2). Based on <br />  <br />  <br /> October 2013 Volume 87 Number 19                                                                                                                 jvi.asm.org 10725 <br />  Guo et al. <br />  <br />  <br />  <br />  <br />                                                                                 FIG 5 rde-1 contributes rde-4-independent antiviral silencing. (A) Viral <br />                                                                                 replication enhanced rde-1;rde-4 double mutants compared the <br />                                                                                 single mutants. Shown accumulation Orsay virus (left panel) and <br />                                                                                 FR1gfp transcripts single double mutants containing rde-1 and/or rde-4 <br /> FIG 4 rrf-1 contributes rde-4-independent antiviral silencing. (A) Accu-     null alleles. (B) Accumulation FR1gfp-derived primary siRNAs single or <br /> mulation Orsay virus RNA1 (ovRNA1) wild-type N2 worms RDVI-           double mutants containing rde-1 and/or rde-4 null alleles. <br /> defective mutants carrying single double mutations indicated. (B) Accu- <br /> mulation FR1gfp transcripts wild-type N2 worms RDVI-defective <br /> mutants carrying single double mutations indicated. (C) Accumulation      ing viral dsRNA siRNAs Dicers (8, 48). In <br /> of FR1gfp-derived siRNAs double mutants, indicated, defective RDVI.    Drosophila, RDE-4 homologue R2D2 contributes func- <br />                                                                                 tion viRNAs produced drosophila Dicers (7, <br />                                                                                 22, 49). Therefore, demonstrating rde-4-indpendent mech- <br /> this observation, concluded sid-1-mediated systemic                     anism primary viRNA biogenesis, study, time, <br /> RNAi required restricting Orsay virus infection C.                suggested RDVI single-Dicer invertebrates initiated <br /> elegans.                                                                        absence dsRNA binding protein. <br />                                                                                     Previously, shown dsRNA high concentra- <br /> DISCUSSION                                                                      tion triggers RNAi C. elegans absence RDE-4 (39). This <br /> rsd-2 identified gene enables RNAi spreading                observation suggests viruses, powerful replicators rap- <br /> from soma germ line (31). A role rsd-2 RDVI                   idly produce large amounts dsRNA replication intermediates, <br /> revealed genetic screening utilized RNAi-mediated gene                trigger RDVI absence RDE-4. Here, an <br /> knockdown phenocopy genetic mutants (5). Recently, genetic                 increase viral replication level, primary viRNAs detected at <br /> study suggested rsd-2 helps maintain normal chromo-                high levels rde-4 mutants containing rsd-2 rrf-1 <br /> somal function, transposon control, unfavorable                   <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>null allele</span>, suggesting rde-4-independent mechanism the <br /> conditions (35). Currently, rsd-2 contributes biolog- <br /> ical processes remains poorly understood. rsd-2 unique nem- <br /> atode worms, C. elegans, known encode the <br /> single Dicer required biogenesis siRNAs and <br /> miRNAs. Thus, functional mechanistic study rsd-2 is <br /> expected shed light mechanism rsd-2 <br /> helps maintain chromosomal integrity response environ- <br /> mental stresses reveal uniqueness RDVI single- <br /> Dicer invertebrates. Here, rsd-2 functions two <br /> parallel genetic pathways mediate antiviral silencing rde-4-            FIG 6 sid-1 required RDVI C. elegans. Shown accumulation <br /> dependent rde-4-independent manners. In Arabidopsis,                        Orsay virus RNA1 wild-type N2 worms sid-1 mutants. See legend <br /> dsRNA binding protein regulates RDVI facilitating process-               Fig. 1B experimental details. <br />  <br />  <br />  <br /> 10726    jvi.asm.org                                                                                                                        Journal Virology <br />                                                                                                                     RDE-4-Independent Antiviral Silencing <br />  <br />  <br />  <br /> generation primary viRNAs (Fig. 2 4). Apparently an <br /> increase viral replication viRNA accumulation be <br /> attributed residual rde-4 function rde-4 allele used this <br /> study, FR1gfp-derived siRNAs detected rde-4 <br /> single drh-1;rde-4 double mutants conditions <br /> (5) (Fig. 1E, 2A, 4C, 5B). Thus, demonstrating exis- <br /> tence rde-4-independent pathway antiviral silencing, we <br /> explained dsRNAs high levels triggers RNAi absence <br /> of RDE-4 C. elegans (39). Notably, readily detectable <br /> in worms, rde-1, rrf-1, rsd-2 mutants, defective <br /> in secondary viRNA biogenesis, virus-derived primary siRNAs <br /> were hardly detectable wild-type N2 worms worms defective <br /> in rde-4 (Fig. 1E, 2A, 3C, 4C). This simply reflect fact <br /> that antiviral silencing rde-4-dependent rde-4-inde- <br /> pendent mechanisms amplified secondary viRNAs and, <br /> as result, viral dsRNA processed primary <br /> viRNAs Dicer presence secondary viRNAs. <br />     Notably, observations suggested rde-1 plays role <br /> in rde-4-independent RDVI similar rsd-2 (Fig. 5). Inter- <br /> estingly, increase FHV replication level observed rde- <br /> 1;rsd-2 double mutants compared rde-1 single mutants. rde-1 <br /> probably plays major role rde-4-dependent RDVI minor <br /> role rde-4-independent RDVI, result, antiviral silencing <br /> in rde-1 mutants compromised presence an <br /> rsd-2 null allele. Nevertheless, analyzing accumulation        FIG 7 Working model rde-4-dependent rde-4-independent antiviral <br /> viral transcripts primary viRNAs various worm single          silencing C. elegans. WAGO, worm-specific AGO (29). Here, propose <br /> double mutants defective RDVI, study demonstrated,            processing viral dsRNAs primary viRNAs occurs rde-4- <br /> the time, addition conventional rde-4-depen-        dependent rde-4-independent manner. drh-1 contributes rde-4- <br />                                                                          dependent, rde-4-independent, antiviral silencing. We believe that <br /> dent pathway, rde-4-independent pathway antiviral        rsd-2 rrf-1 contribute rde-4-dependent rde-4-independent <br /> silencing C. elegans (Fig. 7). Previously, showed     antiviral silencing, rde-1 plays major function rde-4-dependent <br /> RDE-1-related worm AGO, termed C04F12.1, plays role                 antiviral silencing minor function rde-4-independent antiviral silenc- <br /> RDVI. It interesting test C04F12.1 plays           ing. We believe major function rrf-1 direct the <br />                                                                          synthesis secondary viRNAs, rsd-2 contributes function primary <br /> role rde-4-independent RDVI. C. elegans known encode          viRNAs, initiating synthesis secondary viRNAs. As proposed <br /> large number AGO proteins. Thus, testing rde-4 depen-             previously, WAGOs direct cleavage cognate viral transcripts <br /> dence C04F12.1 allow address question                using secondary viRNAs sequence guide (29). <br /> whether different worm AGO proteins specifically function in <br /> mechanistically distinct RDVI pathways C. elegans. <br />     rsd-2 known required accumulation second-        yield insight mechanistic basis RSD-2 in <br /> ary, primary, endogenous siRNAs (34). Currently, re-          RDVI. <br /> mains unclear rsd-2 contributes RDVI sim-              Like plants, RNAi C. elegans involves intercellular <br /> ilar mechanism. Here, FR1gfp-deirved primary                signal responsible systemic spread RNAi (32, 50, 51). <br /> siRNAs accumulated rsd-2 pattern similar rrf-1      A recent study suggested long dsRNAs pro- <br /> or rde-1 single mutants, known accumulate primary           cessed intermediates physical carrier intercellular <br /> viRNAs (Fig. 1E, 2A, 3B). This finding suggests that, similar     signal (38). Consistent finding, systemic spreading the <br /> that endogenous RNAi, rsd-2 enhances cell-autonomous anti-            intercellular signal requires transmembrane protein SID-1 <br /> viral silencing contributing accumulation secondary,        transports dsRNAs size preference facili- <br /> but primary, viRNAs. This finding explained rsd-2                tate intercellular transportation silencing signal (33, 36, <br /> mutants resistant low doses sensitive high doses        47). Previously, shown SID-1 dispensable for <br /> dsRNAs (34). Presumably, trigger dsRNAs present          RDVI triggered artificial viruses, vesicular stomatitis <br /> high concentrations, sufficient primary siRNAs produced          virus FHV (4, 40). Currently, SID-1 contributes to <br /> to trigger efficient RNAi absence secondary siRNAs.            RDVI natural conditions remains largely unknown. To ad- <br /> However, trigger dsRNAs introduced low levels,           dress question, compared Orsay virus infection between <br /> efficient RNAi require dose-sensitive RNAi factors,      wild-type N2 worms sid-1 mutants Orsay virus <br /> as RSD-2, RDE-10, RDE-11, amplify RNAi promot-              infection enhanced sid-1 mutants (Fig. 6). <br /> ing production/accumulation secondary siRNAs (34). No-            Worm RDVI probably involves intercellular silencing signal <br /> tably, replication FHV Orsay virus           transported SID-1-independent mechanism. <br /> enhanced rsd-2 mutants compared rrf-1 mutants,             Consistent hypothesis, SID-1 shown dispens- <br /> suggesting rsd-2 contributes function primary        able export mobile silencing signal C. elegans (52). <br /> viRNAs. RSD-2 contains 3 tandem domains unknown function              Alternatively, SID-1, key component systemic RNAi, may <br /> at N terminus. Functional characterization domains          active nonintestinal tissues, result role systemic <br />  <br />  <br /> October 2013 Volume 87 Number 19                                                                                                     jvi.asm.org 10727 <br />  Guo et al. <br />  <br />  <br />  <br /> RDVI identified using Orsay virus, mainly targets                     polymerases Dicer-like proteins antiviral defense small inter- <br /> the intestine cells replication (6). Moreover, probably                    fering RNA biogenesis turnip mosaic virus infection. Plant Cell <br />                                                                                       22:481&#226;&#8364;&#8220; 496. <br /> the lack RDVI suppression activity, Orsay virus known               13.   Baulcombe D. 2002. Viral suppression systemic silencing. Trends Mi- <br /> be highly sensitive RDVI (28). Thus, possible rep-                  crobiol. 10:306 &#226;&#8364;&#8220;308. <br /> lication Orsay virus sid-1 mutants suppressed intracel-             14.   Guo HS, Ding SW. 2002. A viral protein inhibits long range signaling <br /> lular antiviral silencing, making intercellular antiviral silenc-                 activity gene silencing signal. EMBO J. 21:398 &#226;&#8364;&#8220; 407. <br /> ing redundant keeping particular virus control.                   15.   Molnar A, Melnyk CW, Bassett A, Hardcastle TJ, Dunn R, Baulcombe <br />                                                                                       DC. 2010. Small silencing RNAs plants mobile direct epigenetic <br />     Unlike RDE-4, plays important role biogenesis                     modification recipient cells. Science 328:872&#226;&#8364;&#8220; 875. <br /> of endogenous siRNAs, DRH-1 appears dedicated factor                 16.   Dunoyer P, Schott G, Himber C, Meyer D, Takeda A, Carrington JC, <br /> worm RDVI (5, 20). DRH-2 shares significant sequence homology                         Voinnet O. 2010. Small RNA duplexes function mobile silencing sig- <br /> with DRH-1 appears negatively regulate worm RDVI (5).                          nals plant cells. Science 328:912&#226;&#8364;&#8220;916. <br />                                                                                 17.   Saleh M-C, Tassetto M, van Rij RP, Goic B, Gausson V, Berry B, <br /> These observations raised interesting question of <br />                                                                                       Jacquier C, Antoniewski C, Andino R. 2009. Antiviral immunity in <br /> whether DRH-2 specifically targets regulates function                      Drosophila requires systemic RNA interference spread. Nature 458:346 &#226;&#8364;&#8220; <br /> DRH-1, negatively regulating rde-4-dependent RDVI.                        350. <br /> Apparently, addressing question able             18.   Ketting RF, Fischer SE, Bernstein E, Sijen T, Hannon GJ, Plasterk RH. <br /> whether worms evolved mechanism allow                          2001. Dicer functions RNA interference synthesis small RNA <br />                                                                                       involved developmental timing C. elegans. Genes Dev. 15:2654 &#226;&#8364;&#8220; <br /> for specific regulation RDVI affecting normal cel-                     2659. <br /> lular function unfavorable environments.                                  19.   Grishok A, Pasquinelli AE, Conte D, Li N, Parrish S, Ha I, Baillie DL, <br />                                                                                       Fire A, Ruvkun G, Mello CC. 2001. Genes mechanisms related to <br /> ACKNOWLEDGMENTS                                                                       RNA interference regulate expression small temporal RNAs that <br />                                                                                       control C. elegans developmental timing. Cell 106:23&#226;&#8364;&#8220;34. <br /> We thank Caenorhabditis Genetics Center (CGC)               20.   Lee RC, Hammell CM, Ambros V. 2006. Interacting endogenous and <br /> worm strains used study. This work possible               exogenous RNAi pathways Caenorhabditis elegans. RNA 12:589 &#226;&#8364;&#8220;597. <br /> without JU1580 worm strain Orsay virus M.-A. F&#195;&#169;lix,            21.   Tabara H, Yigit E, Siomi H, Mello CC. 2002. The dsRNA binding protein <br /> E. A. Miska, D. Wang.                                                             RDE-4 interacts RDE-1, DCR-1, DExH-box helicase direct <br />     This work supported Board Regents, Louisiana                        RNAi C. elegans. Cell 109:861&#226;&#8364;&#8220; 871. <br /> [LEQSF(2011-14)-RD-A-09 LEQSF-EPS(2012)-PFUND-277],                     22.   Zambon RA, Vikram NV, Louisa PW. 2006. RNAi antiviral im- <br /> the NIH (1R03AI092159-01A1).                                                          mune response dsRNA virus Drosophila melanogaster. Cell. <br />                                                                                       Microbiol. 8:880 &#226;&#8364;&#8220; 889. <br />                                                                                 23.   Ghildiyal M, Zamore PD. 2009. Small silencing RNAs: expanding <br /> REFERENCES                                                                            universe. Nat. Rev. Genet. 10:94 &#226;&#8364;&#8220;108. <br />  1. Ding SW, Voinnet O. 2007. Antiviral immunity directed small RNAs.        24.   Lu R, Maduro M, Li F, Li HW, Broitman-Maduro G, Li WX, Ding SW. <br />     Cell 130:413&#226;&#8364;&#8220; 426.                                                                2005. Animal virus replication RNAi-mediated antiviral silencing in <br />  2. Roghiyh Aliyari S-WD. 2009. RNA-based viral immunity initiated             Caenorhabditis elegans. Nature 436:1040 &#226;&#8364;&#8220;1043. <br />     Dicer family host immune receptors. Immunol. Rev. 227:176 &#226;&#8364;&#8220;188.          25.   Steiner FA, Okihara KL, Hoogstrate SW, Sijen T, Ketting RF. 2009. <br />  3. Wilkins C, Dishongh R, Moore SC, Whitt MA, Chow M, Machaca K.                     RDE-1 slicer activity required passenger-strand cleavage during <br />     2005. RNA interference antiviral defence mechanism Caenorhab-            RNAi Caenorhabditis elegans. Nat. Struct. Mol. Biol. 16:207&#226;&#8364;&#8220;211. <br />     ditis elegans. Nature 436:1044 &#226;&#8364;&#8220;1047.                                       26.   Pak J, Fire A. 2007. Distinct populations primary secondary effec- <br />  4. Schott DH, Cureton DK, Whelan SP, Hunter CP. 2005. An antiviral role              tors RNAi C. elegans. Science 315:241&#226;&#8364;&#8220;244. <br />     RNA interference machinery Caenorhabditis elegans. Proc. Natl.   27.   Sijen T, Steiner FA, Thijssen KL, Plasterk RH. 2007. Secondary siRNAs <br />     Acad. Sci. U. S. A. 102:18420 &#226;&#8364;&#8220;18424.                                             result unprimed RNA synthesis form distinct class. Science <br />  5. Lu R, Yigit E, Li WX, Ding SW. 2009. An RIG-I-like RNA helicase                   315:244 &#226;&#8364;&#8220;247. <br />     mediates antiviral RNAi downstream viral siRNA biogenesis Caeno-      28.   Guo X, Lu R. 2013. Characterization virus-encoded RNAi suppressors <br />     rhabditis elegans. PLoS Pathog. 5:e1000286. doi:10.1371/journal.ppat              Caenorhabditis elegans. J. Virol. 87:5414 &#226;&#8364;&#8220;5423. <br />     .1000286.                                                                   29.   Yigit E, Batista PJ, Bei Y, Pang KM, Chen CC, Tolia NH, Joshua-Tor L, <br />  6. F&#195;&#169;lix M-A, Ashe A, Piffaretti J, Wu G, Nuez I, B&#195;&#169;licard T, Jiang Y, Zhao          Mitani S, Simard MJ, Mello CC. 2006. Analysis C. elegans Argo- <br />     G, Franz CJ, Goldstein LD, Sanroman M, Miska EA, Wang D. 2011.                    naute family reveals distinct Argonautes act sequentially during <br />     Natural experimental infection Caenorhabditis nematodes                 RNAi. Cell 127:747&#226;&#8364;&#8220;757. <br />     novel viruses related nodaviruses. PLoS Biol. 9:e1000586. doi:10.1371    30.   Seth RB, Sun L, Chen ZJ. 2006. Antiviral innate immunity pathways. Cell <br />     /journal.pbio.1000586.                                                            Res. 16:141&#226;&#8364;&#8220;147. <br />  7. Wang XH, Aliyari R, Li WX, Li HW, Kim K, Carthew R, Atkinson P,             31.   Tijsterman M, May RC, Simmer F, Okihara KL, Plasterk RH. 2004. <br />     Ding SW. 2006. RNA interference directs innate immunity viruses           Genes required systemic RNA interference Caenorhabditis elegans. <br />     adult Drosophila. Science 312:452&#226;&#8364;&#8220; 454.                                        Curr. Biol. 14:111&#226;&#8364;&#8220;116. <br />  8. Jakubiec A, Yang SW, Chua N-H. 2012. Arabidopsis DRB4 protein            32.   Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC. 1998. <br />     antiviral defense Turnip yellow mosaic virus infection. Plant J.          Potent specific genetic interference double-stranded RNA Cae- <br />     69:14 &#226;&#8364;&#8220;25.                                                                        norhabditis elegans. Nature 391:806 &#226;&#8364;&#8220; 811. <br />  9. Qu F, Ye X, Hou G, Sato S, Clemente TE, Morris TJ. 2005. RDR6         33.   Winston WM, Molodowitch C, Hunter CP. 2002. Systemic RNAi C. <br />     broad-spectrum temperature-dependent antiviral defense role                elegans requires putative transmembrane protein SID-1. Science 295: <br />     Nicotiana benthamiana. J. Virol. 79:15209 &#226;&#8364;&#8220;15217.                                 2456 &#226;&#8364;&#8220;2459. <br /> 10. Dalmay T, Hamilton A, Rudd S, Angell S, Baulcombe DC. 2000. An              34.   Zhang C, Montgomery TA, Fischer SEJ, Garcia SMDA, Riedel CG, <br />     RNA-dependent RNA polymerase gene Arabidopsis required                  Fahlgren N, Sullivan CM, Carrington JC, Ruvkun G. 2012. The Caeno- <br />     posttranscriptional gene silencing mediated transgene           rhabditis elegans RDE-10/RDE-11 complex regulates RNAi promoting <br />     virus. Cell 101:543&#226;&#8364;&#8220;553.                                                          secondary siRNA amplification. Curr. Biol. 22:881&#226;&#8364;&#8220; 890. <br /> 11. Wang X-B, Wu Q, Ito T, Cillo F, Li W-X, Chen X, Yu J-L, Ding S-W.           35.   Han W, Sundaram P, Kenjale H, Grantham J, Timmons L. 2008. The <br />     2010. RNAi-mediated viral immunity requires amplification virus-               Caenorhabditis elegans rsd-2 rsd-6 genes required chromo- <br />     derived siRNAs Arabidopsis thaliana. Proc. Natl. Acad. Sci. U. S. A.           functions exposure unfavorable environments. Genetics <br />     107:484 &#226;&#8364;&#8220; 489.                                                                    178:1875&#226;&#8364;&#8220;1893. <br /> 12. Garcia-Ruiz H, Takeda A, Chapman EJ, Sullivan CM, Fahlgren N,               36.   Feinberg EH, Hunter CP. 2003. Transport dsRNA cells the <br />     Brempelis KJ, Carrington JC. 2010. Arabidopsis RNA-dependent RNA                  transmembrane protein SID-1. Science 301:1545&#226;&#8364;&#8220;1547. <br />  <br />  <br />  <br /> 10728    jvi.asm.org                                                                                                                          Journal Virology <br />                                                                                                                            RDE-4-Independent Antiviral Silencing <br />  <br />  <br />  <br /> 37. Shih JD, Hunter CP. 2011. SID-1 dsRNA-selective dsRNA-gated            45. Sijen T, Fleenor J, Simmer F, Thijssen KL, Parrish S, Timmons L, <br />     channel. RNA 17:1057&#226;&#8364;&#8220;1065.                                                      Plasterk RH, Fire A. 2001. On role RNA amplification dsRNA- <br /> 38. Jose AM, Garcia GA, Hunter CP. 2011. Two classes silencing RNAs              triggered gene silencing. Cell 107:465&#226;&#8364;&#8220; 476. <br />     Caenorhabditis elegans tissues. Nat. Struct. Mol. Biol. 18:    46. Aoki K, Moriguchi H, Yoshioka T, Okawa K, Tabara H. 2007. In vitro <br />     1184 &#226;&#8364;&#8220;1188.                                                                     analyses production activity secondary small interfering <br /> 39. Habig JW, Aruscavage PJ, Bass BL. 2008. In C. elegans, high levels           RNAs C. elegans. EMBO J. 26:5007&#226;&#8364;&#8220;5019. <br />     dsRNA allow RNAi absence RDE-4. PLoS One 3:e4052. doi:10          47. Shih JD, Fitzgerald MC, Sutherlin M, Hunter CP. 2009. The SID-1 <br />     .1371/journal.pone.0004052.                                                     double-stranded RNA transporter selective dsRNA length. RNA <br /> 40. Guo X, Li W-X, Lu R. 2012. Silencing host genes directed virus-           15:384 &#226;&#8364;&#8220;390. <br />     deri <br /> </body></html>