<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>         Basic Research Paper <br />  <br /> Autophagy 7:4, 386-400; April 2011; &#194;&#169; 2011 Landes Bioscience <br />  <br />  <br />  <br />                  The Atg6/Vps30/Beclin 1 ortholog BEC-1 <br />                  mediates endocytic retrograde transport <br />                   addition autophagy C. elegans <br />        Alexander Ruck,1,2 John Attonito,1 Kelly T. Garces,1 Lizbeth N&#195;&#186;nez,1 Nicholas J. Palmisano,1,2 Zahava Rubel,1 Zhiyong Bai,3 <br />                          Ken C.Q. Nguyen,4 Lei Sun,4,5 Barth D. Grant,3 David H. Hall4 Alicia Mel&#195;&#169;ndez1,2,* <br />  1 <br />   Department Biology; Queens College; Flushing, NY USA; 2The Graduate Center; The City University New York; New York, NY USA; 3 Department Molecular Biology <br /> and Biochemistry; Rutgers University; Piscataway, NJ USA; 4Center C. elegans Anatomy; Albert Einstein College Medicine; Bronx, NY USA; 5Center Biological Imaging; <br />                                                     Institute Biophysics; Chinese Academy Sciences; Beijing, China <br />  <br />  <br />                                                 Key words: C. elegans, autophagy, endocytosis, lysosomes <br />  <br />  <br />             Autophagy endocytosis dynamic tightly regulated processes contribute fundamental aspects <br />             biology including <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>survival</span>, longevity development. However, molecular links autophagy and <br />             endocytosis understood. Here, report BEC-1, C. elegans ortholog Atg6/Vps30/Beclin 1, key <br />             regulator autophagic machinery, contributes endosome function. In particular identified defect in <br />             retrograde transport endosomes Golgi bec-1 mutants. MIG-14/Wntless normally recycled endosomes <br />             Golgi action retromer complex associated factor RME-8. Lack retromer RME-8 <br />             activity results aberrant transport MIG-14/Wntless lysosome degraded. Similarly, found <br />             lack bec-1 results mislocalization degradation MIG-14::GFP, reduced levels RME-8 endosomal <br />             membranes, accumulation morphologically abnormal endosomes. A similar <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> observed in <br />             animals treated dsRNA vps-34. We identified requirement BEC-1 clearance apoptotic <br />             corpses hermaphrodite gonad, suggesting role BEC-1 phagosome maturation, process appears <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />             depend retrograde transport. In addition, autophagy genes required cell corpse clearance, we <br />             RNAi atg-18 unc-51 results lack cell corpse clearance. <br />  <br />  <br />  <br />  <br />                                 Introduction                                            Additional regulators autophagy identified in <br />                                                                                         screens mutants cytoplasm-to-vacuole (Cvt) pathway.7,8 <br /> Macroautophagy (hereafter autophagy) cellular bulk degrada-                        The Cvt pathway delivers vesicle-bound proteins vacuole <br /> tion process conserved eukaryotes.1,2 In autophagy, portion                    cytoplasm. However, unlike autophagy degra- <br /> of cytoplasm sequestered double-membrane organelle                          dative mainly nonselective, Cvt pathway biosynthetic <br /> called autophagosome, fuses lysosome                         selects specific cargo cytoplasm delivered to <br /> degrade materials inside. Endocytosis vesicle-mediated                       vacuole.9 <br /> process internalize plasma membrane, and/or macromolecules                              Yeast Atg6/Vps30 orthologs multicellular organ- <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> outside environment. The endosomal sorts                            isms play pivotal role autophagy. The mammalian ortholog <br /> internalized molecules proper intracellular destination.                       Atg6/Vps30 Beclin 1. In addition regulating autophagy, <br /> In yeast, vacuole functionally equivalent lysosome                        Beclin 1 shown tumor suppressor protein and <br /> in higher eukaryotes. In yeast Saccharomyces cerevisiae, muta-                      antiapoptotic protein Bcl-2 negatively regulates autophagy <br /> tions vacuolar protein sorting (VPS) genes result secretion                       binding Beclin 1.10-14 C. elegans affords another <br /> of proteins normally localized vacuole. Characterization                      genetic studying role Beclin 1. The C. elegans <br /> the VPS genes provided insight mechanism pro-                           Beclin 1 ortholog bec-1 mediates autophagy essential <br /> tein sorting vesicle-mediated intracellular transport.                              morphogenesis dauer larva life-span exten- <br />    Genetic studies yeast S. cerevisiae identified                      sion mediated insulin signaling15 dietary restriction.16-18 <br /> genes function autophagy (ATG).3,4 Many ATG                              We studied role C. elegans BEC-1 autophagy and <br /> genes encode proteins required early formation                         endocytosis. <br /> of autophagosome, localized perivacu-                                In yeast, distinct roles Atg6/Vps30 protein autophagy <br /> olar structure called preautophagosomal structure (PAS).5,6                         vacuolar protein sorting reflect related functionally <br />  <br />  *Correspondence to: Alicia Mel&#195;&#169;ndez; Email: Alicia.Melendez@qc.cuny.edu <br />  Submitted: 06/29/10; Revised: 12/05/10; Accepted: 12/06/10 <br />  DOI: 10.4161/auto.7.4.14391 <br />  <br />  <br /> 386	                                                                          Autophagy	                                                                   Volume 7 Issue 4 <br />        Basic Research Paper                                                                                        Basic Research Paper <br />  <br />  <br />  <br /> different phosphatidylinositol-3-kinase (PI3K) complexes:        bec-1 mutant animals display defects recycling MIG- <br /> type I complex functions autophagy type II complex        14/Wntless cargo protein endosome trans-Golgi <br /> functions vacuolar protein sorting, respectively.19 The differ-   network. Recycling MIG-14/Wntless cargo mediated <br /> ence complexes complex I contains Atg14,         retromer complex, conserved cytoplasmic coat recycling <br /> whereas complex II contains Vps38, instead Atg14. Vps38        complex mediates endosome-to-Golgi retrieval vacu- <br /> present endosome vacuolar membranes, required       ole/lysosome hydrolase receptors yeast mammals.36,37 We <br /> for targeting Atg6/Vps30 Vps34 endosome.20         showed RME-8, subunit retromer complex <br /> In contrast, Atg14 localizes PAS vacuolar membrane,       BEC-1 colocalize endosomes adjacent to <br /> and generates autophagosome recruiting Vps34 Atg6/        lysosomes wild-type animals, RME-8 redistributed <br /> Vps30 PAS.20                                                  cytoplasm bec-1 mutants, indicating role bec-1 in <br />     Vps34, corresponds mammalian class III PI3-         retrograde transport. Finally, report lack cell corpse clear- <br /> kinase, important various membrane trafficking pathways       ance observed electron microscopy increased number <br /> such Golgi lysosome pathway, internal vesicle forma-       CED-1::GFP positive cells bec-1 mutant gonads, similar to <br /> tion late endosomes autophagy.21-26 Although      phenotype animals lacking retromer subunit rme-8.38 <br /> well established mammalian Beclin 1/Vps34 complex <br /> is required autophagy, involvement membrane                                      Results <br /> trafficking pathways controversial higher eukary- <br /> otes. Expression C. elegans BEC-1 human Beclin 1            bec-1 maternal zygotic activity. The C. elegans Gene <br /> atg6-depleted yeast restores autophagy vacuolar pro-         <span id='am-394' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-395' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>Knockout</span> Consortium generated deletion mutations bec- <br /> tein sorting function yeast.27 Although Beclin 1 hVps34       1, ok691 ok700. The ok691 allele deletes of <br /> were shown co-localize trans-Golgi network (TGN)        seven exons, including ATG start codon (Fig. 1) a <br /> HeLa cells, role Beclin 1 vesicular trafficking mam-    predicted molecular <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>null allele</span>.31 The ok700 allele deletes amino <br /> malian cells limited. <span id='am-628' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-629' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-630' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-631' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-632' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-633' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-634' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-635' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-636' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-637' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-638' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-639' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-640' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-641' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-642' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-643' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-644' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-645' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-646' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-647' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-648' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-649' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-650' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-651' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-652' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-653' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-654' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-655' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-656' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-657' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-658' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-659' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-660' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-661' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-662' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-663' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-664' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-665' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-666' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-667' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-668' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-669' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-670' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-671' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-672' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-673' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-674' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-675' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-676' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-677' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-678' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-679' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-680' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-681' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-682' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-683' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-684' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-685' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-686' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-687' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-688' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-689' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-690' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-691' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-692' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-693' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-694' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-695' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-696' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-697' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-698' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-699' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-700' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-701' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-702' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-703' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-704' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-705' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) medi-           acids 168 314 &#226;&#8364;&#339;evolutionarily conserved domain&#226;&#8364;? (ECD) <br /> ated suppression beclin 1 impairs autophagy does          BEC-1 causes frameshift removes remain- <br /> affect fluid-phase endocytosis, endocytic sorting epider-     der protein (Materials Methods); allele appears to <br /> mal growth factor receptor (EGFR) cathepsin D transport           strong hypomorph. <br /> from TGN lysosomes.28 In addition, overexpression            Both ok691 ok700 mutations reported display <br /> mutant version human Beclin 1 bind human           highly penetrant embryonic <span id='am-550' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-551' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-553' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-554' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-555' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-556' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-557' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-558' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-559' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-560' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-561' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-562' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-563' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-564' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-565' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-566' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-567' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-568' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-569' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-570' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-571' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-572' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-573' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-574' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-575' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-576' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-577' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-578' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-579' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-580' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-581' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-582' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-583' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-584' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-585' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-586' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-587' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-588' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-589' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-590' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-591' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-592' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-593' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-594' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-595' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-596' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-597' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-598' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-599' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-600' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-601' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-602' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-603' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-604' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-605' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-606' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-607' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-608' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-609' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-610' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-611' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-612' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-613' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-614' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-615' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-616' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-617' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-618' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-619' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-620' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-621' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-622' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-623' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-624' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-625' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-626' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-627' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> segregat- <br /> Vps34 does inhibit ability process cathepsin D,       ing balanced heterozygote.31 In contrast, using bal- <br /> significantly reduced autophagy activity tumor suppres-          ancer marked unc-5(e53) jcIs[ajm-1:gfp], enables us <br /> sor functions MCF7 cells.29 In contrast, expression plant      unambiguously identify bec-1 mutant animals both <br /> Atg6 yeast atg6 mutants restores autophagy vacu-         embryonic adult stages (see Materials Methods), we <br /> olar protein sorting function.30 Furthermore, C. elegans BEC-1       ok691 ok700 homozygous individuals derived <br /> has shown bind VPS-34 affect PI3P localization        bec-1(mutant)/unc-5(e53) jcIs1 heterozygous parent can <br /> and general uptake endocytic marker.31 Biological func-        reach adulthood (Table 1). However, adults sterile and <br /> tions atg6/beclin 1 higher eukaryotes suggest role outside   die day adulthood. These results confirm that <br /> of autophagy. Arabidopsis atg6 mutants exhibit defects auto-      bec-1 activity required viability (Table 1). Further analysis <br /> phagy pollen germination, observed        suggests bec-1 mutants segregating heterozygous moth- <br /> in autophagy mutants, suggesting Atg6 confers        ers reach adulthood maternal rescue. <br /> autophagy-independent function.30 In mice, Beclin 1 knockout             Both bec-1(ok691) bec-1(ok700) homozygous mutants <br /> animals die approximately embryonic day 7.5,13 Atg5       segregate bec-1(mutant)/unc-5(e53) jcIs1 hetero- <br /> and Atg7 knockout mice survive birth,32,33 Beclin     zygous parent appear superficially wild type early larval <br /> 1 complex function mice. Recently, pos-         stages increasingly uncoordinated later larval <br /> sible orthologs Atg14 Vps38 identified mam-      stages display low penetrance molting defect. In addition, <br /> mals,34,35 shown form distinct PI3-kinase complexes      mutants accumulate vacuoles visible the <br /> different intracellular localization, suggesting Vps34-Beclin   L3 stage [Table 2: 70% ok691 (n = 37), 30% ok700 (n <br /> 1 complexes different functions metazoans similar        = 40) contain vacuole, Fig. 1B, middle panel]. These <br /> to yeast cells.                                              vacuoles present intestine, hypodermis, pharynx and <br />     Although C. elegans bec-1 previously implicated      coelomocytes. Both number size vacuoles increases <br /> endocytic trafficking,31 nature contribution      age larvae (Table 2). By L4 stage, 80% ok691 ani- <br /> been described. Here, showed bec-1 mutant animals            mals (n = 39), 52% ok700 (n = 40) contain one <br /> display defects recycling cargo molecules endo-       vacuole, approximately 42% ok691 mutants (n = 39), and <br /> some late-Golgi. We observed maternally rescued          5% ok700 mutants (n = 40) contain 10 vacuoles <br /> bec-1 mutants accumulate large vacuoles mixed endosomal           (Table 2). Lastly, ok691 ok700 mutant animals display <br /> and lysosomal identity. Furthermore, showed bec-1 func-      growth retardation reach adult stage 1.5 days later <br /> tions endosome Golgi retrograde transport. In particular,      wild-type animals. <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                   387 <br />   Figure 1. bec-1 gene, protein, mutations phenotypes. (A) At top, genomic structure bec-1 gene encompassing 7 exons, encod- <br />  ing 375 amino acid protein, shown below. In middle, deletions bec-1 provided C. elegans <span id='am-472' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-473' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-475' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-476' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-484' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-485' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-486' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-488' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-489' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-491' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-499' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-513' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-514' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-515' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-518' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-523' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-528' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-536' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-537' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-545' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>Gene Knockout</span> Consortium. ok691 <br />  deletes bec-1 open reading frame including start ATG exons 1&#226;&#8364;&#8220;6, molecular null. ok700 deletes exons 4&#226;&#8364;&#8220;6 bec-1 and <br />  renders mRNA frame exon 7. A cDNA sequenced RNA extracted ok700 lysate, shows bec-1 <br />  mRNA transcribed ok700 mutants lacks evolutionarily conserved domain BEC-1 required binding VPS-34. (B) Both deletions display an <br />  adult lethal phenotype striking accumulation vacuoles (marked black arrows (B), middle, right panel). ok691 mutants (B, middle panel) <br />  segregating heterozygous parent (B, panel). bec-1 homozygous mutant, ok691 ok700, appear superficially wild type early <br />  larval stages increasingly uncoordinated later larval stages display molting problems (white arrow (B), middle, left panel). <br />  Rescue bec-1 phenotypes including uncoordination, accumulation vacuoles lethality observed transgene containing the <br />  bec-1 wild-type genomic sequences ubiquitously expressed SUR-5::GFP marker (B, bottom, left panel) BEC-1::RFP construct (B, bottom, right <br />  panel). (C) Animals carry rescuing bec-1 genomic extrachromosomal array (Ex[bec-1(+), SUR-5::GFP]) (left panel) rescued the <br />  bec-1 phenotype. Animals lost array early development completely devoid GFP marker expression (C, right panel) <br />  lack maternal zygotic bec-1 expression. A hatched line delineates periphery embryo (C, right panel). These animals die <br />  embryogenesis increase number visible apoptotic cells (marked black arrow heads, (C) right panel). <br />  <br />  <br />    To confirm observed phenotypes muta-                3). Only 15.1% embryos carrying Ex[BEC-1::RFP, <br /> tion bec-1, injected genomic sequence containing wild-               rol-6(d)] transgene 38.6% embryos carrying Ex <br /> type sequences bec-1 ok691 mutant animals (Ex [bec-1(+),             [bec-1(+), SUR-5::GFP] transgene hatch. Since multicopy trans- <br /> SUR-5::GFP]) obtained rescue developmental                   genic arrays efficiently expressed hermaphrodite <br /> defects vacuolar phenotypes (Table 2 Fig. 1B, bot-                   germline,40,41 observation suggested bec-1 maternally <br /> tom panel). Both ok691 ok700 bec-1 deletion alleles              contributed. <br /> likewise rescued functional BEC-1::RFP fusion construct                     To directly test possibility, analyzed phenotype <br /> (kindly provided B. Bamber39) (Table 3). Interestingly, sig-            bec-1 mutant embryos originated fertile homozy- <br /> nificant number transgenic embryos failed hatch (Table                 gous ok691 mutant mothers carrying rescuing transgene, Ex <br />  <br />  <br /> 388	                                                                 Autophagy	                                                        Volume 7 Issue 4 <br />   Table 1. Lack zygotic bec-1 results adult lethality                      Table 2. Lack maternal zygotic bec-1 leads embryonic lethality <br />                           Eggs      hatched          Reach adulthood                                 Aberrant vacuole phenotype <br />                                                  20.3% unc-5 homozygous                     % Animals contain vacuole <br />  bec-1(ok691)/unc-5; <br />                            325       97.8%         53.1% heterozygous                                          L3                          L4 <br />         jcIs1 <br />                                                  26.6% bec-1 homozygous <br />                                                                                       ok691               70% (n = 37)              80% (n = 39) n.s. <br />                                                  24.6% unc-5 homozygous <br />  bec-1(ok700)/unc-5;                                                                  ok700               30% (n = 40)              52% (n = 40) n.s. <br />                            270        94%           50% heterozygous <br />         jcIs1 <br />                                                  25.4% bec-1 homozygous                   % Animals contain 10 vacuoles <br />          N2                195        97%                    100%                                              L3                          L4 <br />                                                                                       ok691                13% (n = 37)              42% (n = 39)* <br />  Table 3. bec-1 mutants accumulate aberrant vacuoles                                  ok700                 0 (n = 40)              5% (n = 40) n.s. <br />                            +GFP          -GFP                       n = eggs <br />    ok691; Ex array                                     Adults <br />                           Hatched       Hatched                     counted <br />  Ex[BEC-1::RFP, pRF4]       15.1%            0         100%         n = 651    human Hrs FYVE domain, 2XFYVE::GFP. As shown previ- <br />                                                                                ously,31 lack bec-1 activity disrupts localization PI3P. <br />   Ex[bec-1(+), pTG96]       38.6%            0          92%         n = 185 <br />                                                                                In bec-1(ok691) mutants observed diffuse localization <br />                                                                                2XFYVE::GFP reporter (Fig. 2A&#226;&#8364;&#8482;) compared the <br /> [bec-1(+), SUR-5::GFP]. The lack transgene embryos                   marker control animals (Fig. 2A). The membrane the <br /> can visualized lack GFP (Fig. 1C right bot-                   enlarged vacuoles mutant intestines labeled the <br /> tom panel). These embryos likely lack maternal zygotic                PI3P marker. Localization PI(4,5)P2 determined a <br /> expression bec-1. We maternal zygotic loss                   reporter expresses PH domain phospholipase C delta. <br /> of bec-1 results complete early embryonic lethality. As               PI(4,5)P2 worm intestine normally enriched apically <br /> embryos fail hatch, inferred rescued                  pronounced localization basolateral membranes <br /> maternally. These embryos display classic morphogenesis defects                (Fig. 2B). The PI(3,4,5)P3 reporter consisted PH-domain <br /> with unattached cells floating outside embryo (Fig. 1C                 Akt fused GFP, localized apically basolaterally <br /> right panel) visible apoptotic cell corpses.31                         intestine (Fig. 2C). In ok691 mutants, expression the <br />    bec-1 <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> disrupts localization PI3P. We ana-                   PI(4,5)P2 PI(3,4,5)P3 reporters largely unaffected when <br /> lyzed phenotype bec-1(ok691), unambiguous molecular                  compared wild-type animals (Fig. 2B&#226;&#8364;&#8482; C&#226;&#8364;&#8482;). <br /> null allele detail. To understand relationship bec-1                 bec-1 mutants defects endocytosis. The endocytic <br /> vps-34, test dysregulation phos-                nature vacuoles defects PI3P localization observed <br /> phoinositides bec-1 mutants, monitored expression                    bec-1 mutant animals prompted directly test whether <br /> and localization PI3P-, PI(4,5)P2- PI(3,4,5)P3-binding                  bec-1 required endocytic trafficking C. elegans. We con- <br /> reporters42 wild-type bec-1(ok691) mutant animals. vps-                 centrated analysis coelomocytes, large scaven- <br /> 34/let-512 encodes class III phosphoinositide 3-kinase,                  ger cells, situated pseudocoelomic space continuously <br /> protein regulates multiple steps endocytosis.43 Since                  endocytose fluid pseudocoelom.44 To define pre- <br /> BEC-1 associates VPS-34,31 expected localiza-                 cisely defect bec-1 endocytosis, performed fluid uptake <br /> tion PI3P affected bec-1(ok691) null mutant.                assays injection Texas Red-coupled BSA (TR-BSA) the <br /> PI3P detected transgene encoding tandem repeat                    body cavity wild-type bec-1 mutant individuals. We used <br />  <br />  <br />  Figure 2 (See page). bec-1 mutants display defects endocytosis. The detection phosphoinositides PI3P, PIP2 PIP3 intestines <br />  wild-type (A&#226;&#8364;&#8220;C), bec-1(ok691) mutants (A&#226;&#8364;&#8482;&#226;&#8364;&#8220;C&#226;&#8364;&#8482;), animals shown. Scale bars indicate 10 &#206;&#188;m. In wild-type animals (A), PI3P enriched apical <br />  membrane weak basolateral labeling. In bec-1(ok691) (A&#226;&#8364;&#8482;) mutants low diffused expression PI3P observed. PIP2 expression is <br />  affected bec-1(ok691) mutants (B&#226;&#8364;&#8482;) compared wild-type animals (B). The PIP3 reporter labels apical membrane strong basolateral <br />  expression wild-type animals (C). The localization PIP3 bec-1(ok691) animals (C&#226;&#8364;&#8482;) similar wild-type animals (C). Micrographs of <br />  LMP-1::GFP positive lysosomes coelomocytes 24 h injection Texas Red BSA (a fluid phase marker endocytosis) body cavity <br />  (pseudocoelom). A hatched line demarcates periphery coeolomocytes. The red dye accumulates lysosomes (LMP-1 positive) localized in <br />  coelomocytes wild-type animals (D), fails accumulate lysosomes (LMP-1 positive) bec-1(ok691) mutants (D&#226;&#8364;&#8482;). Micrographs of <br />  LMP-1::GFP positive lysosomes coelomocytes 1 h soaking Texas Red BSA (a fluid phase marker endocytosis). The red dye accumulates <br />  lysosomes (green) localized coelomocytes wild-type animals (E), fails accumulate lysosomes (green) bec-1 mutants <br />  (E&#226;&#8364;&#8482;). Electron microscopy N2 wild-type (F G) bec-1(ok691) mutant young adults (H I). (F) Low power electron micrograph wild- <br />  type adult intestine cross-section, fixed high pressure freezing. Most basal membrane shows signs endocytosis, is <br />  membrane infolding (black arrow F) small vesicles, particularly near lateral membrane border intestinal cells on <br />  extreme left edge tissue. Higher power view adult wild-type intestine shows examples membrane infolding (black arrows in <br />  G) representing local zone endocytosis active. It notable infolded membranes large, vesicles <br />  nearby cytoplasm. In contrast, chains vesicles (white arrows I) extensive infolded membranes seen basal pole bec-1 <br />  mutant (H I). Thus endocytotic events cleared relatively quickly wild-type intestine. In bec-1 mutant intestines, defective enlarged <br />  endosomes (marked white arrow head H) seen correspond abnormal vacuoles different sizes contain membranes or <br />  condensed material. (I) close region marked (H) shows enlarged vacuoles high number early endosomes appear to <br />  fusing cases (black arrowhead I). This phenotype seen intestines wild-type animals (F G). <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                                     389 <br />   Figure 2. For figure legend, page 389. <br />  <br />  <br /> LMP-1::GFP visualize lysosomes late endosomes. LMP-1        lysosomes.46,49 After injection wild-type animals, fluid- <br /> is similar vertebrate lysosome-associated membrane pro-     phase marker present early endocytic compartments. After <br /> tein LAMP/CD68.45 LMP-1::GFP widely used lyso-         15 min, begins accumulate endosomes (LMP-1::GFP <br /> somal marker.46,47 Injection TR-BSA pseudocoelom       negative), 1 h accumulates lysosomes (LMP-1::GFP <br /> (body cavity) monitored uptake, mediated   positive Fig. 2D).46,50,51 In bec-1 mutants, endocytosis of <br /> by endocytosis basolateral cell membrane bordering      fluid phase marker abnormal (Fig. 2D&#226;&#8364;&#8482;). <br /> pseudocoelom.48 In wild-type animals, fluorescent marker       Texas Red BSA does localize LMP-1 positive endosomes <br /> enters coelomocytes fluid phase mediated endocytosis    24 hours. After 1 h soaking animals fluid <br /> is transported different endocytic compartments     markers (or allowing feed it), Texas Red BSA <br />  <br />  <br /> 390	                                                       Autophagy	                                                   Volume 7 Issue 4 <br />  Figure 3. bec-1 mutants display defect retrograde transport. Confocal images wild-type background shown GFP::RAB-5 (A), GFP::RAB-7 <br /> (B), LMP-1::GFP (C), GFP::hTfR (D), MIG-14::GFP (E), GFP::RME-8 (F), GFP::SNX-1 (G), GFP::VPS-35 (H). Confocal images bec-1(ok691) mutants shown <br /> for GFP::RAB-5 (A&#226;&#8364;&#8482;), GFP::RAB-7 (B&#226;&#8364;&#8482;), LMP-1::GFP (C&#226;&#8364;&#8482;), GFP::hTfR (D&#226;&#8364;&#8482;), MIG-14::GFP (E&#226;&#8364;&#8482;), RME-8::GFP (F&#226;&#8364;&#8482;), GFP::SNX-1 (G&#226;&#8364;&#8482;), GFP::VPS-35 (H&#226;&#8364;&#8482;). (B) GFP::RAB-7 <br /> positive puncta (marked yellow arrow) labels maturing endosomes late endosomes (marked yellow arrowhead). bec-1(ok691) mutants ac- <br /> cumulate maturing endosomes marked GFP::RAB-7, display decrease late endosomes marked GFP::RAB-7 (compare B B&#226;&#8364;&#8482;; See <br /> Fig. 2I). White arrowheads indicate enlarged intestinal endosomes (abnormal vacuoles) labeled GFP::RAB-5 GFP::RAB-7. In bec-1(ok691) animals, <br /> LMP-1::GFP accumulates membrane enlarged endosomes (C&#226;&#8364;&#8482;; marked white arrowheads). In wild-type animals (D), basolateral <br /> plasma membrane basolateral endocytic compartments labeled hTfR::GFP (human transferrin receptor). The accumulation hTfR::GFP is <br /> not affected bec-1(ok691) mutants (D&#226;&#8364;&#8482;), recycling endosome cargo hTfR::GFP does accumulate enlarged endosomes (abnormal <br /> vacuoles) bec-1(ok691) mutants. Confocal images worm intestine expressing GFP-tagged endocytic transmembrane cargo marker MIG- <br /> 14::GFP wild-type (E), bec-1(ok691) (E&#226;&#8364;&#8482;) animals. The retromer-dependent cargo protein Wntless MIG-14::GFP normally localizes basolaterally <br /> in intestine colocalizes early endosomal marker RAB-5.55 In bec-1 mutants, MIG-14::GFP transmembrane cargo protein accumulates <br /> in enlarged endosomes (abnormal vacuoles; E&#226;&#8364;&#8482;). We observed decrease number MIG-14::GFP positive puncta (I). The retromer subunit <br /> RME-8::GFP (F&#226;&#8364;&#8482;), GFP::SNX-1 (G&#226;&#8364;&#8482;) GFP::VPS-35 (H&#226;&#8364;&#8482;), accumulate enlarged intestinal endosomes (abnormal vacuoles). A decrease the <br /> number GFP::RME-8 positive puncta observed bec-1(ok691) mutants (F&#226;&#8364;&#8482;), intensity markedly decreased. For images, scale bars repre- <br /> sent 10 &#206;&#188;m. Quantification endosome number visualized positive labeling endocytic markers shown (I). In quantification of <br /> endosomal compartments, saw significant difference RAB-5 early endosomes, observed increase RAB-7 maturing endosomes, and <br /> a dramatic decrease RAB-7-positive late endosomes. In addition, observed decrease MIG-14 RME-8 labeled compartments, a <br /> slight increase SNX-1 VPS-35 labeled compartments. Error bars represent standard deviation mean (n = 30 each, 10 animals <br /> of genotype sampled different regions intestine). In (J K), analyzed lysosomal degradation GFP::MIG-14 <br /> occurs bec-1 mutant animals. RNAi mediated depletion lysosome biogenesis protein CUP-5/mucolipin1 increased number intensity <br /> of MIG-14::GFP positive puncta (J Fig. S5). After cup-5 RNAi, saw 4X increase number MIG-14 positive puncta bec-1 mutants, when <br /> compared control RNAi, indicating lysosomal processing MIG-14 cargo bec-1 mutants (J, left panel). Similarly, observed a <br /> significant increase intensity MIG-14::GFP bec-1 mutants cup-5 RNAi (J, right panel). Asterisks indicate significant difference the <br /> one-tailed Student&#226;&#8364;&#8482;s t-test (*p &lt; 0.05, **p &lt; 0.005, ***p &lt; 0.0005, n.s.: significant). <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                                      391 <br />  marker clearly endocytosed wild-type coelomocyte lyso-         functional traffic normally mammalian cells C. ele- <br /> somes (LMP-positive Fig. 2D), bec-1(ok691)         gans.53 We abnormal vacuoles bec-1 mutants did <br /> mutant coelomocytes (Fig. 2D&#226;&#8364;&#8482;). These results indicate role      accumulate hTfR::GFP (Fig. 3D D&#226;&#8364;&#8482;), steady state <br /> BEC-1 early step endocytosis.                                 localization unaffected mutation bec-1, indicating <br />     The vacuoles bec-1(ok691) mutants endocytic ori-         hTfR transit recycling endosome unaffected. <br /> gin. To characterize abnormal vacuoles bec-1 mutant ani-        From experiment described using RAB-10 and <br /> mals origin, used combination fluorescent        RAB-11 markers, concluded bec-1 required the <br /> electron microscopy. In electron micrographs, vacuoles         function recycling endosomes. <br /> clearly surrounded membrane and, empty, filled             Endosome TGN retrograde transport defective bec-1 <br /> with membrane whorls electron dense deposits, features typi-        mutants. In yeast, identification yeast VPS34 gene <br /> cal late endosomes lysosomes (Fig. 2H I). We           encoding PI3-kinase screen vacuolar sorting mutants <br /> often observed aggregation endocytic structures (Fig. 2I, black     demonstrated direct role PI3P vesicular trafficking.52 <br /> arrow head).                                                           Specifically, PI3P shown required efficient sort- <br />     To better define origin vacuoles, crossed ok691      ing proteins late-Golgi vacuole.58 Given the <br /> mutants transgenic animals expressing markers vari-         disruption early endosomal trafficking saw bec-1 <br /> ety specific endocytic compartments. Wild-type intestines           mutants, asked loss bec-1 affect <br /> contain large number small GFP::RAB-5 positive punctate           endosome Golgi retrograde trafficking. <br /> structures (Fig. 3A) near basolateral apical plasma mem-           In C. elegans, SNX-1 ortholog mammalian <br /> brane corresponding early endosomes.52 Similarly number           Sorting Nexin 1 Sorting Nexin 2 proteins function in <br /> of GFP::RAB-7 positive punctate structures, presumably corre-          endosome-to-Golgi retrograde transport Vps-26, Vps- <br /> sponding maturing endosomes, observed near plasma           29 Vps-35, retromer complex.36,37 RME-8, a <br /> membrane53 wild-type intestines (Fig. 3B). A number larger       large protein DNA-J domain, binds SNX-1.55 C. elegans <br /> ring-like GFP::RAB-7 positive structures localize deeper        GFP::SNX-1 RME-8::GFP fusion proteins colocalize with <br /> cytoplasm presumed mature late endosomes.47,53 We        early endosomal markers living animals.55 In C. elegans, like all <br /> found ok691 mutants, GFP::RAB-5 early endosomal            invertebrates, Golgi network appears dispersed ministacks <br /> marker labels membrane abnormal vacuoles (Fig.              cell large juxtanuclear stack.59 <br /> 3A&#226;&#8364;&#8482;). The GFP::RAB-5 specifically localized selected       To date, retromer dependent cargo protein known C. <br /> regions abnormal vacuoles. However, number          elegans MIG-14/Wntless.60,61 In absence retromer func- <br /> intensity GFP::RAB-5 punctate dots did change ok691          tion, MIG-14/Wntless depleted Golgi missorted <br /> mutants compared wild-type (Fig. 3I). In contrast,         late endosome lysosome.55,60-64 Endosome Golgi <br /> late endosomal markers, GFP::RAB-7 LMP-1::GFP, <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span>               transport sorting cargo protein MIG-14/Wntless depend <br /> the membrane enlarged abnormal vacuoles ok691,           RME-8 SNX-1.55 Thus, reasoned BEC-1 func- <br /> only rarely membrane small abnormal vacuoles (Fig. 3B&#226;&#8364;&#8482;).        tions retrograde transport, bec-1 mutants abnormally <br /> These results suggest vacuoles aberrant endosomes         transport MIG-14/Wntless cargo similar pheno- <br /> and distinguished classes, representing          type snx-1 rme-8 mutants. <br /> different stages endosome maturation process. Unlike                When compared wild-type control animals, bec-1(ok691) <br /> RAB-5-positive puncta, small RAB-7-positive puncta over-           mutants displayed approximately four-fold reduction the <br /> accumulated bec-1 mutant intestines, (Fig. 3B B&#226;&#8364;&#8482;            number MIG-14::GFP-positive puncta. A similar reduction <br /> I). The accumulation small RAB-7-labeled structures             observed animals treated RNAi bec-1 and <br /> appeared expense larger RAB-7-labeled rings        vps-34 (Fig. 3S). These results consistent idea that <br /> that decreased number (Fig. 3I). Markers endosome         BEC-1 functions VPS-34 retrograde transport (Fig. 3E <br /> types apical recycling endosomes (GFP::RAB-11)             E&#226;&#8364;&#8482;; Fig. 3S). However, observed increase total <br /> basolateral recycling endosomes (GFP::RAB-10) appeared nor-            intensity MIG-14::GFP signal contrast has <br /> mal (Fig. S1 data shown). Moreover, abnormal vacu-         previously reported MIG-14::GFP rme-8 snx-1 <br /> oles bec-1 mutant animals faintly labeled   mutants.55,60,61 In bec-1 vps-34 mutants, MIG-14::GFP is <br /> labeled GFP::RAB-10 (Fig. S1). Similarly, noted         ring-like intralumenal structures (Fig. 3S). The <br /> expression ALX-1::GFP bec-1 mutants significantly        reasons difference unclear, possibility that <br /> different wild-type animals (Fig. S1). ALX-1 Alix/    MIG-14::GFP localizes membrane abnormal and <br /> Bro1p ortholog C. elegans, protein RME-1       enlarged vacuoles accumulate bec-1 mutants (Fig. 3E <br /> regulates recycling endosomes plasma membrane.54           E&#226;&#8364;&#8482;; Fig. 3S), similar localization GFP::RAB-7, <br /> Taken results suggest requirement BEC-1      LMP-1::GFP. Another possibility discrepancy <br /> specific stage endosome maturation.                                 bec-1 rme-8 phenotypes hypomorphic <br />     As control, assayed distribution human       nature rme-8 mutation used. These results suggest that <br /> transferrin receptor (hTfR::GFP), transmembrane cargo          defect retrograde transport MIG-14::GFP <br /> recycled recycling endosome Golgi.55-57        endosome Golgi bec-1 mutants in <br /> An equivalent version GFP-fusion protein shown          bec-1 mutants, substantial MIG-14 degraded by <br />  <br />  <br /> 392	                                                           Autophagy	                                                   Volume 7 Issue 4 <br />  the lysosome, similarly described animals           intensity clathrin-positive endosomes RNAi treatment <br /> with defective retromer subunit activity.                               vps-34 (visualized functional clathrin reporter, <br />     To determine decrease MIG-14::GFP bec-1 mutants        GFP::CHC-1;59 Fig. 6S), obvious changes knock- <br /> is result missorting lysosomal-mediated degradation,         bec-1 activity. However, consistent decrease in <br /> we monitored MIG-14::GFP levels bec-1 mutants RNAi             number intensity RME-8::GFP positive punta observed in <br /> depletion cup-5/mucolipin1, transmembrane protein required         bec-1 mutants, observed significant increase GFP::SNX-1 <br /> for normal lysosome biogenesis normal levels hydrolytic          GFP::VPS-35 positive puncta bec-1 mutants com- <br /> activity.46,55,65 Depletion CUP-5 RNAi increased num-         pared wild-type animals (Fig. 3G&#226;&#8364;&#8482; H&#226;&#8364;&#8482;; quantitation Fig. <br /> ber MIG-14::GFP positive puncta nearly normal levels           3I). Moreover, size GFP::SNX-1 positive puncta was <br /> the bec-1 mutant background (quantitation shown Fig. 3J; Fig.        significantly increased bec-1 mutant animals (Fig. 7S). This <br /> 4S). In addition, significant increase pixel intensity      increase size distribution SNX-1 positive endosomes <br /> MIG-14::GFP observed bec-1 mutants depleted            bec-1 mutants similar observed rme-8 mutants.55 <br /> for CUP-5 cup-5 RNAi (Fig. 3J; Fig. 4S). These results con-             Colocalization BEC-1 RME-8. The retromer com- <br /> firm lysosomal degradation MIG-14::GFP occurs            plex, including RME-8 SNX-1, primarily early <br /> in bec-1 mutants. A similar result previously reported ret-     endosomes, RME-8/SNX-1 positive endosomes <br /> romer mutants, rme-8. In rme-8 mutants, CUP-5 deple-            closely juxtaposed Golgi apparatus.37,55 <br /> tion RNAi restores levels MIG-14::GFP pixel intensity            We C. elegans BEC-1::RFP colocalizes with <br /> to nearly normal levels. Thus, loss BEC-1 activity results      subset RME-8::GFP labeled endosomes intestine, <br /> similar phenotype retromer mutants, degrada-           consistent direct role BEC-1 retrograde transport <br /> tion MIG-14 occurs missorting lysosomal         (Fig. 4C&#226;&#8364;&#8482;). Interestingly, note BEC-1::RFP RME- <br /> pathway (Fig. 3J).                                                      8::GFP double-positive endosomes clearly adjacent auto- <br />     The increased degradation MIG-14::GFP suggested             fluorescent lipofuscin-positive lysosome-like organelles (in blue, <br /> multivesicular body (MVB)-mediated transport membrane                Fig. 4C&#226;&#8364;&#8482; C&#226;&#8364;?). <br /> proteins lysosome does require BEC-1. To examine           bec-1 role germ cell corpse clearance. Loss bec-1 <br /> point further, assayed degradation CAV-1::GFP early        leads increase number visible apoptotic cell <br /> embryos.54,55,66 CAV-1 known transmembrane cargo protein           corpses.31 This phenotype result increased apoptosis <br /> that degraded cell embryo, right fertilization,     clearance defects execution cell death pathway <br /> after metaphase anaphase transition.66-68 Degradation         both. To distinguish possibilities, analyzed <br /> CAV-1::GFP requires endocytosis endosomal sort-                 germ cell corpses surroundings using transmission elec- <br /> ing complex required transport (ESCRT) machinery. CAV-              tron microscopy. Wild-type sheath cells efficient clear- <br /> 1::GFP degradation unaffected animals treated bec-1         ing apoptotic germline cells corpses virtually seen <br /> RNAi (Fig. 2S), animals grown second gen-        wild-type gonad electron microscopy, have <br /> eration RNAi plates, suggesting BEC-1, just like        observed endocytic mutants.70 In contrast, bec- <br /> RME-8 SNX-1,55 required ESCRT-mediated degra-            1(ok691) mutants, examples dying germ <br /> dation integral membrane proteins. Similarly, did       cells engulfed sheath cell digested (Fig. 5A). <br /> defect CAV-1::GFP degradation vps-34 RNAi (Fig. 2S).           We analyzed serial sections span entire diameter each <br />     Localization RME-8, SNX-1 VPS-35 bec-1 mutant             cell corpse determine totally inside sheath cell. <br /> animals. Since observed retrograde trafficking defect bec-1     In bec-1(ok691) mutants, germ cell corpses engulfed by <br /> mutants, considered C. elegans BEC-1 act similarly          gonadal sheath cells. These results suggest dying germ <br /> to Atg6/Vps34 yeast retrograde transport. In yeast, muta-        cell able trigger sheath cell encircle it, that <br /> tions ATG6/VPS30 lead selective sorting maturation          bec-1 activity required promote delivery compo- <br /> phenotype soluble vacuolar protease CPY. Redistribution          nents phagosome completion cell degradation. <br /> of Vps5p Vps17p, subunits retromer complex,              To corroborate findings followed individual <br /> was observed strains deficient Atg6/Vps30.69 To deter-           apoptotic nuclei animals reduced bec-1 function in <br /> mine localization retromer complex proteins altered,       wild-type animals determined time required clear- <br /> we examined localization RME-8::GFP (Fig. 3F F&#226;&#8364;&#8482;),            ance. We apoptotic nuclei wild-type animals are <br /> GFP::SNX-1 (Fig. 3G G&#226;&#8364;&#8482;) GFP::VPS-35 (Fig. 3H                cleared 50 min, 30 min, nuclei <br /> H&#226;&#8364;&#8482;). In bec-1 mutants, intensity number           persist animals reduced bec-1 function 1 h (Fig. <br /> of RME-8::GFP-positive puncta decreased (Fig. 3 compare F            5B). These observations indicate apparent increase in <br /> to F&#226;&#8364;&#8482;; Fig. 5S). Although RNAi bec-1 did            apoptotic nuclei reflects defect cell corpse clearance rather <br /> dramatic effect RME-8::GFP bec-1 mutant, knock-            increase number cells undergoing apoptosis. <br /> down vps-34 RNAi affected number RME-                    To investigate inactivation autophagy genes <br /> 8::GFP positive puncta significantly (Fig. 5S).                         effect cell corpse clearance, counted apoptotic nuclei <br />     RME-8 SNX-1 shown physically associate             gonads animals treated dsRNA <br /> in vivo association functionally relevant regula-   various autophagy genes. The phagocytic receptor cell <br /> tion endosomal clathrin.55 We observed increase          death abnormal CED-1 engulfing cell-specific marker that <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                       393 <br />   Figure 4. BEC-1 colocalizes RME-8. Representative images intestinally expressed RFP-tagged BEC-1 (red, A, A&#226;&#8364;&#8482;) GFP-tagged RME-8 (green, B, <br />  B&#226;&#8364;&#8482;) wild-type intact living animals. Autofluorescent lysosome-like organelles shown blue (+ DAPI filter, D). White arrowheads indicate colocal- <br />  ization BEC-1::RFP RME-8::GFP signals (C), observed yellow fluorescence. The blue autofluorescence shown BEC-1::RFP (A&#226;&#8364;&#8482;) <br />  GFP::RME-8 (B&#226;&#8364;&#8482;) merger (C&#226;&#8364;&#8482;). The colocalized BEC-1::RFP GFP::RME-8 signals adjacent autofluorescent <br />  lysosomes visualized DAPI filter (C&#226;&#8364;&#8482;). A close-up (C&#226;&#8364;&#8482;) shown (C&#226;&#8364;?). Arrowheads yellow BEC-1::RFP associ- <br />  ated RME-8::GFP. Magnification 630x (A&#226;&#8364;&#8220;C&#226;&#8364;&#8482;). <br />  <br />  <br />  <br /> recognizes cell corpses clusters growing phagocyte.71               GFP::LGG-1 localization daf-2 insulin-like receptor mutants <br /> Using CED-1::GFP, accumulation germ cell                        bec-1 RNAi observed form large aggregates, rather <br /> apoptotic nuclei gonads vps-34 RNAi animals (Fig. 5C),               small punctate structures represent &#226;&#8364;&#339;normal&#226;&#8364;? auto- <br /> consistent previous reports.71-73 In addition,              phagosomes daf-2 mutants wild-type bec-1 activity. This <br /> RNAi mediated knockdown unc-51/ATG1, atg-18 result                   lack clearance GFP::LGG-1 protein <br /> an increase CED-1::GFP positive nuclei (Fig. 5D). Although                  defect autophagolysosomal pathway15,16 awaits <br /> we did observe increase CED-1::GFP positive nuclei ani-               clarification. In addition confirming role bec-1 in <br /> mals treated RNAi atg-7, analysis variance               endocytic trafficking,31 used combination endocytic mark- <br /> (ANOVA) multiple comparisons, result                  ers BEC-1 involved various steps endocytosis. <br /> be significant.                                                                In particular, identified role BEC-1 retrograde <br />                                                                                trafficking resembles defects associated loss retromer <br />                              Discussion                                        activity, including defective recycling MIG-14, known cargo <br />                                                                                endosome Golgi retromer transport high number <br /> C. elegans bec-1 shown role organismal size,             CED-1::GFP positive cell corpses bec-1 mutant gonads, as <br /> dauer morphogenesis, resistance pathogens longevity              observed gonads animals lacking retromer subunits.38 <br /> daf-2 (the insulin-like receptor) mutants, longevity                bec-1 maternal effect lethal phenotype. Animals that <br /> of dietary restricted animals.15 daf-2 constitutive mutants             completely lack maternal zygotic BEC-1 activity die during <br /> restrictive temperature shown increase auto-                embryogenesis. These embryos display high incidence apop- <br /> phagy, marked GFP::LGG-1 autophagosome marker.27                     totic nuclei.31 Animals segregate heterozygous parent, <br />  <br />  <br /> 394	                                                                   Autophagy	                                                          Volume 7 Issue 4 <br />   Figure 5. bec-1 mutants display lack cell corpse clearance. (A) Micrograph transmission electron microscopy (TEM) poor degradation of <br />  apoptotic germ cell corpses observed bec-1 mutants. This TEM image taken cross-section midbody. A rounded dying germ <br />  cell shown dark cytoplasm large round vacuoles near nucleus, contains clumped chromatin. The dying cell com- <br />  pletely wrapped somatic sheath cell (tinted purple) early phase apoptosis. The sheath cell wraps normal germ-line (bottom <br />  cells panel), separating mesodermal tissue intestine (top cell panel). (B) Histogram indicating distribution the <br />  duration germ cell corpses completely degraded. The y-axis indicates percentage germ cell corpses lasted period <br />  time indicated x-axis clearance. n number cell corpses analyzed. (C) Representative images CED-1::GFP positive labeled cell <br />  corpses wild-type animals treatment control RNAi RNAi bec-1 vps-34. (D) Quantification germ cell corpses wild-type <br />  animals treatment control RNAi, RNAi unc-51/Atg1, bec-1, vps-34, atg-7 atg-18. Data derived observing adults, 36 hours <br />  post larval L4 stage. Data compared unpaired t tests; 30 animals analyzed experiment. Asterisks indicate significant differ- <br />  ence result analysis variance (ANOVA). This analysis indicated significant differences treatments control, <br />  atg-7 RNAi (p &lt; 0.001). Threshold significance (alpha) t-tests p &lt; 0.01 using &#226;&#8364;&#339;Bonferroni correction&#226;&#8364;? multiple corrections. <br />  <br />  <br /> and maternally derived wild-type BEC-1 function, live                      We analyzed number morphology major <br /> to early adulthood. These homozygous bec-1 mutant animals                       endocytic compartments bec-1 mutant intestinal cells in <br /> display striking accumulation vacuoles different tissues                vivo using set GFP-tagged markers label class of <br /> including intestine hypodermis. They uncoordi-                 compartment. Our goal determine block <br /> nated movement.                                                        particular transport step caused lack bec-1 activ- <br />     Studies mice shown lack beclin 1 activity                ity measuring specific changes endosome morphology, and <br /> also lethal. beclin 1 mutants die day 7.5 development,12,13               intensity number GFP markers. We expect an <br /> whereas atg5 atg7 mutant mice die birth period              endocytic compartment normally receives cargo BEC-1 <br /> starvation animals begin breastfeed.33,74 Therefore,              dependent fashion smaller size lacking com- <br /> it possible difference phenotype beclin              pletely intestinal cells lacking BEC-1 activity. We also <br /> 1 autophagy genes beclin 1 having additional                expect accumulation enlargement BEC-1 dependent <br /> defects endocytosis mammals.                                              endosomes BEC-1 donor compartments bec-1 mutant <br />     bec-1 mutants display endocytic defects. The lack accu-                  intestines. Trafficking cell membrane lysosome <br /> mulation endocytic tracers coelomocytes indicates                requires activity Rab5 Rab7 GTPases.75 Rab GTPases <br /> internalization step endocytosis defective bec-1 mutants.              regulate intracellular trafficking controlling transport <br /> This consistent previous reports pronounced defect                 vesicles membrane compartments endocytic <br /> in bec-1 mutants assaying uptake vitellogenin                   transport. Rabs alternate &#226;&#8364;&#339;active&#226;&#8364;? guanosine tri- <br /> by oocytes,31 process requires endocytosis. Thus, bec-1               phosphate (GTP)-bound state &#226;&#8364;&#339;inactive&#226;&#8364;? guanosine <br /> required early step endocytic trafficking.                             diphosphate (GDP)-bound state. Active Rab5 localizes early <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                                       395 <br />  endosomes Rab7 localizes late endosomes. Rab5          animals knockdown RNAi mediated vps-34. <br /> Rab7 regulate trafficking cargo plasma membrane        Conversely, number GFP::SNX-1 GFP::VPS-35 posi- <br /> the lysosomes.75                                                      tive puncta increases bec-1 mutants compared wild- <br />     In bec-1 homozygous mutants, accumulation             type animals. For SNX-1, size puncta larger bec-1 <br /> of RAB-7 positive puncta structures decrease RAB-7           mutants. These findings similar rme-8 <br /> ring-like structures. As RAB-7 positive puncta presumed        mutants,55 bec-1 vps-34 mutants accumulate aber- <br /> be maturing endosomes, ring-like structures late        rant MIG-14::GFP positive vacuoles. We conclude BEC-1 <br /> endosomes, results suggest defect prior maturation   vps-34 function retrograde transport, possibly con- <br /> the late endosome. In addition, membranes        cert RME-8. Since known RME-8 recruited <br /> the abnormal vacuoles seen bec-1 mutant animals positive      endosome, hypothesize BEC-1 product <br /> for RAB-5, RAB-7 LMP-1, consistent accumulation              VPS-34, PI3P, facilitate endosomal recruitment of <br /> of incompletely matured endosomes. In yeast mammalian             RME-8. Further studies required better understand <br /> cells, Ypt7p Rab7 mediate homotypic fusion vacu-           relationship BEC-1, VPS-34 RME-8, the <br /> oles late endosomes/lysosomes.76,77 Thus, possible     role BEC-1 VPS-34 early endosome maturation and <br /> the accumulation RAB-7 LMP-1 positive large vacuoles           transport Golgi. <br /> is result enhancement homotypic fusion consistent            BEC-1 required cell corpse clearance. We that <br /> with increase RAB-7 activity. Distorted endosomal com-          bec-1 loss function mutant animals display lack cell corpse <br /> partments observed electron microscopy. These data       clearance hermaphrodite gonad increase <br /> demonstrate wild-type BEC-1 activity required     incidence apoptosis.31 Instead, a <br /> endolysosomal pathway. The fact lack PI3P local-     significant lack germ cell corpse degradation bec-1 mutant <br /> ization, product VPS-34, bec-1 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> indi-       animals compared wild-type. In addition, using trans- <br /> cates bec-1 phenotype mislocalization      mission electron microscopy major effect on <br /> of PI3P endosomes.                                                 engulfment, implying loss bec-1 activity does appear <br />     In study enlarged vacuoles bec-1         affect signal dying cell. We rule low- <br /> null mutants result defect maturing endocytic com-          penetrance effect, vivo results contrast previ- <br /> partments. This phenotype previously documented          ously reported requirement autophagy genes beclin 1 <br /> C. elegans intestines defects recycling endosomes      atg5 energy source facilitate signaling dying <br /> as rme-1 rab-10 mutants, recycling blocked        cells phagocytic cell embryoid bodies derived ES <br /> pharmacological agents certain mammalian cell types.52,78          cells.80 <br /> Similar vacuoles enlargement endocytic compartments            We animals deficient vps-34, is <br /> also observed phosphatidylinositol phosphate kinase 3         required RAB-5 recruitment nascent phagosome,71 <br /> mutants, ppk-3, C. elegans ortholog yeast PIKfyve/         increase number germ cell nuclei. The degra- <br /> Fab1p.49 In animals, enlargement RAB-7-positive,         dation cell corpses shown require sequential <br /> LMP-1 positive endocytic compartments occurs.49 Thus,           enrichment early endosomes, late endosomes lysosomes <br /> are multiple steps endocytosis disruption result     nascent phagosome (reviewed ref. 81). RAB-7 and <br /> in enlarged endocytic compartments. However recycling             PI3P proposed function downstream effectors <br /> marker RAB-10::GFP did localize bec-1 mutant vacu-         CED-1 pathway mediate phagolysosome formation.82 <br /> oles, number RAB-10::GFP puncta affected           Since showed RAB-7::GFP PI3P localization <br /> in bec-1 mutants, suggesting lack bec-1 does affect   normal bec-1 mutants, cell clearance defect bec-1 <br /> the recycling endosomes.                                           mutants mislocalization PI3P the <br />     BEC-1 functions retrograde transport. Yeast Atg6/          phagosome. Interestingly, observed knockdown other <br /> Vps30 implicated early endosome Golgi retro-           autophagy genes RNAi results similar accumulation of <br /> grade transport.69,79 We C. elegans bec-1 function      CED-1-positive engulfed germ cells hermaphrodite gonad. <br /> also required retromer function. We BEC-1           This result suggests autophagosomes required the <br /> VPS-34 required rescue MIG-14 degradation       process phagosome maturation. Since defect cell clear- <br /> endocytosis. Our results aberrant sorting MIG-14      ance reported retromer subunit mutants, rme-8 <br /> bec-1 vps-34 mutants similar, dramatic      snx-1,38 point know lack of <br /> that rme-8 snx-1 mutants.55 It known MIG-        cell clearance defect bec-1 mutants reflects requirement for <br /> 14 utilize alternative route retrograte transport    autophagy bec-1-mediated retrograde transport function <br /> in C. elegans mammals.55 LMP-1 MIG-14, differ-          both. This point addressed future experiments. <br /> ent transmembrane cargo proteins, labeled abnormal vacu-              Comparisons mammals yeast. In yeast, differ- <br /> oles accumulate bec-1 mutant intestinal cells,        ent Atg6/Vps30 complexes function autophagy <br /> hTfR::GFP did not, indicating LMP-1 MIG-14 require           vacuolar protein sorting.19,79 The complexes differed only <br /> BEC-1 transport, hTfR does not.                             proteins: Atg14 Vps38. The complex consisting of <br />     bec-1 mutants marked decrease number RME-            Atg14, Atg6/Vps30, Vps15 Vps34 shown have <br /> 8::GFP puncta expression. A similar phenotype observed            autophagy function, complex consisting Vps38, Atg6/ <br />  <br />  <br /> 396	                                                          Autophagy	                                                  Volume 7 Issue 4 <br />  Vps30, Vps15 Vps34 shown specific vacuolar          [Pvha-6 ::RME-8::GFP, unc-119(+)], p[snx-1::GFP::SNX-1], izEx <br /> protein sorting, specifically endosome Golgi retrograde       [Pvha-6 ::MIG-14::GFP, unc-119(+)],55 pwIs61 [GFP::cav-1, unc- <br /> trafficking.69 Thus, subunits Atg14 Vps38      119(-)] 66 bcIs39[P(lim-7)ced-1::GFP, lin-15(+)].71 <br /> are shared complexes. The counterparts                Molecular analyses. To characterize nature bec-1 <br /> these complexes organisms,        transcript present bec-1(ok700) mutant animals, total RNA <br /> orthologs Atg14 Vps38 higher eukaryotes         isolated subjected RT-PCR using oligo dT primers. <br /> seemingly lacking, leading assumption molecu-       Determination DNA sequence RT-PCR product <br /> lar complexes existed yeast. Recently, using highly sensi-    revealed short transcript containing exons 1&#226;&#8364;&#8220;4 resulting a <br /> tive methods purification, mammalian proteins identified      frameshift removes remainder open reading frame <br /> that interact Beclin 1, mammalian ortholog Atg6/          exon 4. <br /> Vps30. In fact, distinct Beclin 1 complexes           Transgenes. For rescue experiments bec-1 mutants, 14.0 <br /> described orthologs yeast Atg14 Vps38            Kb PCR fragment amplified using wild-type genomic DNA <br /> found.34,83 The human ortholog yeast Atg14 Atg14L,      template. This fragment contains entire bec-1(+) gene plus <br /> human ortholog Vps38 proposed Uvrag,          10 Kb flanking 5&apos; DNA 970 bp downstream 3&apos; DNA. <br /> Uvrag primarily localizes late endosomes, shows weak           This PCR fragment directly injected bec-1(ok691)/nT1 <br /> homology yeast Vps38.35 A complex consisting Beclin 1,        mutant animals 40 &#206;&#188;g/ml ubiquitously <br /> hVps34, hVps15 Atg14L, functions early autophagosome           expressed cotransformation marker pTG96 (SUR-5::GFP) at <br /> formation. Another complex consisting Uvrag, Beclin 1, hVps34      100 &#206;&#188;g/ml. The rescuing BEC-1::RFP, pAR 39.1,39 injected <br /> and hVps15 appears act endosome Golgi retrograde traf-       20 &#206;&#188;g/ml cotransformation marker pRF4 [rol- <br /> ficking fusion lysosomes autophagosomes.     6(d)] 100 &#206;&#188;g/ml, Rol lines established. <br /> This consistent findings. A complex consists         Endocytosis assays. To investigate endocytosis baso- <br /> Rubicon, Uvrag, Beclin 1, hVps34 hVps15, Rubicon            lateral membrane intestine, young adult hermaphrodites <br /> negatively regulates autophagosome maturation process,         injected pseudocoelomic space 0.1 mg/ml <br /> well endocytic trafficking.34,83 As Rubicon protein localizes      Texas Red-conjugated BSA (TR-BSA Sigma), previ- <br /> to endosomes lysosomes, directly involved        ously described reference 50. Briefly, TR-BSA dissolved in <br /> regulation membrane fusion processes endosomes/lysosome         egg salts (118 mM NaCl, 48 mM KCl, 2 mM MgCl2, 2 mM <br /> and autophagosomes. Uvrag capacity bind Class       CaCl2, 10 mM HEPES, pH 7.4). We injected TR-BSA the <br /> C VPS complex, involved fusion process auto-       pseudocoelomic space pharyngeal region adult worms <br /> phagosomes endocytosis.84,85 In tissue culture experiments,    immobilized dried agarose pad immersed oil. <br /> the knockdown phenotypes Atg14L Rubicon different,         Injected animals rehydrated M9 buffer transferred <br /> indicating Beclin 1 multiple roles autophagy      seeded NGM plates. At different time points, animals were <br /> the formation different complexes. Orthologs Uvrag          mounted agarose pads 1&#226;&#8364;&#8220;2 &#206;&#188;l drop M9 containing <br /> Rubicon C. elegans sequence homology,      25 mM sodium azide anesthetize view the <br /> although ortholog Uvrag does exist Drosophila.2 Our          confocal microscope. Animals soaked solutions <br /> results support idea BEC-1 complexes exist   investigate endocytosis apical membrane. <br /> C. elegans function autophagy retrograde transport.          RNA interference. dsRNA-mediated gene interference exper- <br /> Significant future work required better understand     iments performed feeding bacteria expressing dsRNA <br /> mechanisms BEC-1 activity functions development,          larval L4 stage individuals, scoring progeny, unless <br /> longevity resistance pathogens, require BEC-1     described otherwise. L4 larvae placed plates containing <br /> autophagy function endocytic function both.                 NGM agar 5 mM IPTG HT115 (DE3) bacteria car- <br />                                                                       rying double stranded RNA expression constructs allowed <br />                    Materials Methods                              lay eggs 24 h 20&#194;&#176;C, case GFP::CAV-1 <br />                                                                       expressing animals, incubated 25&#194;&#176;C. P0s were <br /> C. elegans strains. Standard procedures used culture          transferred day 3 days. F1 progeny raised the <br /> C. elegans worms (Brenner, 1974). All strains grown           plates containing dsRNA bacteria scored 1-day-old adults. <br /> 20&#194;&#176;C, unless stated. The wild-type C. elegans strain        To determine lack GFP::CAV-1 degradation, the <br /> N2 following mutant alleles used: LGIV, bec-             second generation RNAi animals assayed. In all <br /> 1(ok691) bec-1(ok700) provided Knockout consor-            experiments bec-1 vps-34 RNAi, accumulation of <br /> tium, unc-5(e53) jcIs1 (Is[jam-1::GFP]) (gift T. Schedl,         vacuoles monitored obvious phenotype deter- <br /> University St. Louis). The transgenes used: pwIs50[lmp-            RNAi successful. RNAi clones obtained <br /> 1::gfp unc-119(+)],46 pwIs72 [Pvha-6 ::gfp::rab-5, unc-119(+)],       Ahringer Vidal genomic RNAi libraries86 (a gift <br /> pwIs87[Pvha-6 :: gfp::rme-1, unc-119(+)],47,53 pwIs170 [Pvha-         Dr. Malene Hansen). <br /> 6 <br />   ::gfp::rab-7, unc-119(+)]53 [Pvha-6 ::gfp::rab-10, unc-119(+)],         Epifluorescence microscopic analyses. We used Leica <br /> pwIs69 [Pvha-6 :: gfp::rab-11, unc-119(+)],53 pwIs90[Pvha-            TCS-SP5 laser-scanning confocal microscope analyze sub- <br /> 6 <br />   ::hTfR::gfp], izEx1[lgg-1promoter ::gfp::LGG-1],27 izEx5[pAy39.1,   cellular localization fluorescent markers endocytic lyso- <br /> bec-1promoter ::BEC-1::  RFP],39         izEx6[pBEC-1,     pTG96],    somal compartments endocytosis markers mutant <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                  397 <br />  and wild-type animals. Images confocal collected                                    analyzed treatment control. Analysis variance <br /> by PMT (photomultiplier tube) detector, converted Tiff for-                                (ANOVA) indicated significant differences between <br /> mat cropped using Adobe Photoshop CS3. Quantifications                                      treatments t-tests significant differences between <br /> were performed counting punctate-positive structures                                      control treatment treatments atg-7 RNAi <br /> the wild-type mutant intestines using Image J Software                                      (p &lt; 0.001). Threshold significance (alpha) t-tests was <br /> (National Institutes Health). To quantify intensity images,                               p &lt; 0.01 using &#226;&#8364;&#339;Bonferroni correction&#226;&#8364;? multiple corrections. <br /> Metamorph software ver 6.3r2 (Universal Imaging) used.                                         Apoptotic cell clearance assay. Germ cell corpses the <br />     Live worms mounted 2% agarose pads 10 mM                                       adult hermaphrodite gonads scored Nomarski <br /> sodium azide.87 For GFP::RAB-5, RME-8::GFP puncta                                           DIC microscope highly refractile appearance.89 A Zeiss <br /> quantification, structures smaller 0.3 microns                                   ApoTome microscope equipped Time Lapse software was <br /> counted wild-type mutant animals. For MIG-                                         employed capture apoptotic nuclei. In addition, serial Z-section <br /> 14::GFP-positive puncta quantification, structures smaller                                 images regions hermaphrodite gonad recorded at <br /> than 0.2 microns counted wild-type mutant                                     0.5 &#206;&#188;m intervals 5 min. Movement cells focus were <br /> animals. For quantification GFP::SNX-1 GFP::RAB-7,                                       closely monitored ensure region gonad being <br /> GFP-positive puncta counted manually. Images taken                                     recorded did change. Animals closely monitored for <br /> the DAPI filter used identify broad-spectrum intestinal                                 viability recording. Adult hermaphrodites 36&#226;&#8364;&#8220;48 h <br /> autofluorescence caused lipofuscin-positive lysosome-like                                    post-larval L4 stage animals scored. <br /> organelles.47,88 <br />     Electron microscopy. Animals prepared electron                                                                     Acknowledgements <br /> microscopic analysis described previously reference 15                                We thank Drs. Johnny Fares, Hannes B&#195;&#188;low, Malene Hansen, <br /> 27. Briefly, L4 animals packed metal planchette                                   Tim Schedl, C. elegans Knockout Consortium and <br /> using excess bacteria avoid space surrounding                                CGC, strains reagents. We grateful Dr. Areti Tsiola <br /> the animals. The animals frozen using Bal-Tec HPM                                   Dr. Nathalia Holtzman assistance imaging and <br /> 010 high pressure freeze apparatus freeze substituted 1%                                 Image J program, Core Facility Imaging, Cellular <br /> osmium tetroxide acetone beginning -90&#194;&#176;C. The samples                                     &amp; Molecular Biology Queens College. We thankful to <br /> were infiltrated epoxy resin heat cured. These plas-                              Drs. Cathy Savage-Dunn, Hannes B&#195;&#188;low Iva Greenwald for <br /> tic blocks sectioned, post-stained uranyl acetate                                critical reading manuscript helpful discussions. <br /> and lead citrate, examined electron microscopy.                                     This work supported grants NIH/NIA <br />     Quantitation cell corpses. The increase number                                 (AG024882-04S1) Dr. Monica Driscoll (Rutgers University) <br /> cell corpses measured using CED-1::GFP marker. Adult                                    A.M., NSF, Research Initiation Grant 0818802 <br /> hermaphrodites 36 h post-larval L4 stage animals scored.                                A.M. A.M. Ellison Medical Foundation New Scholar <br /> Quantification germ cell corpses wild-type animals                                  Aging (AG-NS-0521-08). D.H.H. supported NIH RR <br /> treatment control RNAi, unc-51/ATG1, bec-1, vps-34                                     12596. B.D.G. supported NIH Grant GM067237. <br /> atg-18 RNAi animals. Data derived experiments <br /> observing adults, 36 h post-larval L4 stage. Data compared                                                                           Note <br /> by analysis variance (ANOVA) two-sample t-tests                                     Supplemental materials at: <br /> between control strain treatments; 30 animals                                 www.landesbioscience.com/journals/autophagy/article/14391 <br />                                                                 7.	  Harding TM, Hefner-Gravink A, Thumm M, Klionsky            13.	 Yue Z, Jin S, Yang C, Levine AJ, Heintz N. beclin 1, an <br />                        References <br />                                                                      DJ. Genetic phenotypic overlap auto-                autophagy gene essential early embryonic develop- <br /> 1.	 Mizushima N, Levine B, Cuervo AM, Klionsky DJ.                   phagy cytoplasm vacuole protein targeting            ment, haploinsufficient tumor suppressor. Proc <br />     Autophagy fights disease cellular self-digestion.        pathway. J Biol Chem 1996; 271:17621-4.                         Natl Acad Sci USA 2003; 100:15077-82. <br />     Nature 2008; 451:1069-75.                                   8.	 Harding TM, Morano KA, Scott SV, Klionsky DJ.               14.	 Pattingre S, Tassa A, Qu X, Garuti R, Liang XH, <br /> 2.	 Melendez A, Neufeld TP. The cell biology autophagy            Isolation characterization yeast mutants          Mizushima N, et al. Bcl-2 antiapoptotic proteins <br />     metazoans: developing story. Development 2008;              cytoplasm vacuole protein targeting pathway. J Cell          inhibit Beclin 1-dependent autophagy. Cell 2005; <br />     135:2347-60.                                                     Biol 1995; 131:591-602.                                         122:927-39. <br /> 3.	 Suzuki K, Kubota Y, Sekito T, Ohsumi Y. Hierarchy        9.	 Teter SA, Klionsky DJ. Transport proteins         15.	 Melendez A, Hall DH, Hansen M. Monitoring role <br />     Atg proteins pre-autophagosomal structure organiza-           yeast vacuole: autophagy, cytoplasm-to-vacuole target-          autophagy C. elegans aging. Methods Enzymol <br />     tion. Genes Cells 2007; 12:209-18.                               ing role vacuole degradation. Semin Cell          2008; 451:493-520. <br /> 4.	 Klionsky DJ, Cregg JM, Dunn WA Jr, Emr SD, Sakai                 Dev Biol 2000; 11:173-9.                                   16.	 Hansen M, Chandra A, Mitic LL, Onken B, Driscoll <br />     Y, Sandoval IV, et al. A unified nomenclature yeast     10.	 Liang XH, Jackson S, Seaman M, Brown K, Kempkes                 M, Kenyon C. A Role Autophagy Extension <br />     autophagy-related genes. Dev Cell 2003; 5:539-45.                B, Hibshoosh H, Levine B. Induction autophagy                Lifespan Dietary Restriction C. elegans. PLoS <br /> 5.	 Suzuki K, Kirisako T, Kamada Y, Mizushima N,                     inhibition tumorigenesis beclin 1. Nature             genetics 2008; 4:24. <br />     Noda T, Ohsumi Y. The pre-autophagosomal structure               1999; 402:672-6.                                           17.	 Jia K, Levine B. Autophagy required dietary <br />     organized concerted functions APG genes            11.	 Liang XH, Kleeman LK, Jiang HH, Gordon G,                       restriction-mediated life span extension C. elegans. <br />     essential autophagosome formation. EMBO J 2001;              Goldman JE, Berry G, et al. Protection fatal            Autophagy 2007; 3:597-9. <br />     20:5971-81.                                                      Sindbis virus encephalitis Beclin 1, novel Bcl-2-     18.	 Mel&#195;&#169;ndez A, Levine B. Autophagy C. elegans <br /> 6.	 Kim J, Huang WP, Klionsky DJ. Membrane recruit-                  interacting protein. J Virol 1998; 72:8586-96.                  (August 24, 2009), WormBook, Ed. The C. elegans <br />     ment Aut7p autophagy cytoplasm             12.	 Qu X, Yu J, Bhagat G, Furuya N, Hibshoosh H, Troxel             Research Community, WormBook, doi/10.1895/ <br />     vacuole targeting pathways requires Aut1p, Aut2p             A, et al. Promotion tumorigenesis heterozygous            wormbook.1.147.1, http://www.wormbook.org. <br />     autophagy conjugation complex. J Cell Biol 2001;             disruption beclin 1 autophagy gene. J Clin Invest   19.	 Kametaka S, Okano T, Ohsumi M, Ohsumi Y. Apg14p <br />     152:51-64.                                                       2003; 112:1809-20.                                              Apg6/Vps30p form protein complex essential for <br />                                                                                                                                      autophagy yeast, Saccharomyces cerevisiae. J Biol <br />                                                                                                                                      Chem 1998; 273:22284-91. <br />  <br />  <br />  <br /> 398	                                                                                  Autophagy	                                                                        Volume 7 Issue 4 <br />  20.	 Obara K, Sekito T, Ohsumi Y. Assortment phos-           39.	 Rowland AM, Richmond JE, Olsen JG, Hall DH,                58.	 Schu PV, Takegawa K, Fry MJ, Stack JH, Waterfield <br />      phatidylinositol-3-kinase complexes&#226;&#8364;&#8221;Atg14p directs              Bamber BA. Presynaptic terminals independently regu-            MD, Emr SD. Phosphatidylinositol-3-kinase encoded <br />      association complex I pre-autophagosomal              late synaptic clustering autophagy GABAA                 yeast VPS34 gene essential protein sorting. <br />      structure Saccharomyces cerevisiae. Mol Biol Cell            receptors Caenorhabditis elegans. J Neurosci 2006;           Science 1993; 260:88-91. <br />      2006; 17:1527-39.                                               26:1711-20.                                                59.	 Sato K, Ernstrom GG, Watanabe S, Weimer RM, Chen <br /> 21.	 Brown WJ, DeWald DB, Emr SD, Plutner H, Balch              40.	 Mello CC, Kramer JM, Stinchcomb D, Ambros V.                    CH, Sato M, et al. Differential requirements clath- <br />      WE. Role phosphatidylinositol-3-kinase               Efficient gene transfer C. elegans: extrachromosomal         rin receptor-mediated endocytosis maintenance <br />      sorting transport newly synthesized lysosom-             maintenance integration transforming sequenc-            synaptic vesicle pools. Proc Natl Acad Sci USA 2009; <br />      al enzymes mammalian cells. J Cell Biol 1995;                es. The EMBO journal 1991; 10:3959-70.                          106:1139-44. <br />      130:781-96.                                                41.	 Kelly WG, Xu S, Montgomery MK, Fire A. Distinct            60.	 Yang PT, Lorenowicz MJ, Silhankova M, Coudreuse <br /> 22.	 Kihara A, Noda T, Ishihara N, Ohsumi Y. Two distinct            requirements somatic germline expression             DY, Betist MC, Korswagen HC. Wnt signaling requires <br />      Vps34 phosphatidylinositol-3-kinase complexes func-             generally expressed Caernorhabditis elegans gene.             retromer-dependent recycling MIG-14/Wntless in <br />      tion autophagy carboxypeptidase Y sorting             Genetics 1997; 146:227-38.                                      Wnt-producing cells. Dev Cell 2008; 14:140-7. <br />      Saccharomyces cerevisiae. J Cell Biol 2001; 152:519-30.    42.	 Bae YK, Kim E, L&#226;&#8364;&#8482;Hernault SW, Barr MM. The CIL-1           61.	 Pan CL, Baum PD, Gu M, Jorgensen EM, Clark SG, <br /> 23.	 Petiot A, Ogier-Denis E, Blommaart EF, Meijer AJ,               PI 5-phosphatase localizes TRP Polycystins cilia             Garriga G. C. elegans AP-2 retromer control Wnt <br />      Codogno P. Distinct classes phosphatidylinositol-            activates sperm C. elegans. Curr Biol 2009;              signaling regulating mig-14/Wntless. Dev Cell <br />      3&apos;-kinases involved signaling pathways              19:1599-607.                                                    2008; 14:132-9. <br />      control macroautophagy HT-29 cells. J Biol Chem         43.	 Roggo L, Bernard V, Kovacs AL, Rose AM, Savoy F,           62.	 Belenkaya TY, Wu Y, Tang X, Zhou B, Cheng L, <br />      2000; 275:992-8.                                                Zetka M, et al. Membrane transport Caenorhabditis            Sharma YV, et al. The retromer complex influences <br /> 24.	 Futter CE, Collinson LM, Backer JM, Hopkins CR.                 elegans: essential role VPS34 nuclear mem-        Wnt secretion recycling wntless endosomes to <br />      Human VPS34 required internal vesicle forma-             brane. EMBO J 2002; 21:1673-83.                                 trans-Golgi network. Dev Cell 2008; 14:120-31. <br />      tion multivesicular endosomes. J Cell Biol 2001;    44.	 Fares H, Greenwald I. Genetic analysis endocytosis      63.	 Franch-Marro X, Wendler F, Guidato S, Griffith J, <br />      155:1251-64.                                                    Caenorhabditis elegans: coelomocyte uptake defective         Baena-Lopez A, Itasaki N, et al. Wingless secretion <br /> 25.	 Davidson HW. Wortmannin causes mistargeting                  mutants. Genetics 2001; 159:133-45.                             requires endosome-to-Golgi retrieval Wntless/Evi/ <br />      procathepsin D. evidence involvement          45.	 Kostich M, Fire A, Fambrough DM. Identification                 Sprinter retromer complex. Nat Cell Biol 2008; <br />      phosphatidylinositol-3-kinase vesicular transport         molecular-genetic characterization LAMP/               10:170-7. <br />      lysosomes. J Cell Biol 1995; 130:797-805.                       CD68-like protein Caenorhabditis elegans. J Cell      64.	 Port F, Kuster M, Herr P, Furger E, Banziger C, <br /> 26.	 Johnson EE, Overmeyer JH, Gunning WT, Maltese                   Sci 2000; 113:2595-606.                                         Hausmann G, Basler K. Wingless secretion promotes <br />      WA. Gene silencing reveals specific function          46.	 Treusch S, Knuth S, Slaugenhaupt SA, Goldin E, Grant            requires retromer-dependent cycling Wntless. <br />      hVps34 phosphatidylinositol-3-kinase late versus             BD, Fares H. Caenorhabditis elegans functional ortho-           Nat Cell Biol 2008; 10:178-85. <br />      early endosomes. J Cell Sci 2006; 119:1219-32.                  logue human protein h-mucolipin-1 required       65.	 Schaheen L, Dang H, Fares H. Basis lethality in <br /> 27.	 Mel&#195;&#169;ndez A, Talloczy Z, Seaman M, Eskelinen EL,                 lysosome biogenesis. Proc Natl Acad Sci USA 2004;               C. elegans lacking CUP-5, Mucolipidosis Type IV <br />      Hall DH, Levine B. Autophagy genes essential                101:4483-8.                                                     orthologue. Dev Biol 2006; 293:382-91. <br />      dauer development life-span extension C.        47.	 Hermann GJ, Schroeder LK, Hieb CA, Kershner AM,            66.	 Sato K, Sato M, Audhya A, Oegema K, Schweinsberg <br />      elegans. Science 2003; 301:1387-91.                             Rabbitts BM, Fonarev P, et al. Genetic analysis lyso-        P, Grant BD. Dynamic regulation caveolin-1 traf- <br /> 28.	 Zeng X, Overmeyer JH, Maltese WA. Functional                    somal trafficking Caenorhabditis elegans. Mol Biol           ficking germ line embryo Caenorhabditis <br />      specificity mammalian Beclin-Vps34 PI 3-kinase           Cell 2005; 16:3273-88.                                          elegans. Mol Biol Cell 2006; 17:3085-94. <br />      complex macroautophagy versus endocytosis               48.	 Grant B, Zhang Y, Paupard MC, Lin SX, Hall DH,             67.	 Bembenek JN, Richie CT, Squirrell JM, Campbell JM, <br />      lysosomal enzyme trafficking. J Cell Sci 2006;              Hirsh D. Evidence RME-1, conserved C. elegans            Eliceiri KW, Poteryaev D, et al. Cortical granule exocy- <br />      119:259-70.                                                     EH-domain protein, functions endocytic recycling.            tosis C. elegans regulated cell cycle components <br /> 29.	 Furuya N, Yu J, Byfield M, Pattingre S, Levine B.               Nat Cell Biol 2001; 3:573-9.                                    including separase. Development 2007; 134:3837-48. <br />      The Evolutionarily Conserved Domain Beclin 1         49.	 Nicot AS, Fares H, Payrastre B, Chisholm AD,               68.	 Sato M, Grant BD, Harada A, Sato K. Rab11 is <br />      Required Vps34 Binding, Autophagy Tumor                 Labouesse M, Laporte J. The Phosphoinositide                    required synchronous secretion chondroitin pro- <br />      Suppressor Function. Autophagy 2005; 1:46-52.                   Kinase PIKfyve/Fab1p Regulates Terminal Lysosome                teoglycans fertilization Caenorhabditis elegans. J <br /> 30.	 Fujiki Y, Yoshimoto K, Ohsumi Y. An Arabidopsis                 Maturation Caenorhabditis elegans. Mol Biol Cell             Cell Sci 2008; 121:3177-86. <br />      homolog yeast ATG6/VPS30 essential pollen             2006; 17:3062-74.                                          69.	 Burda P, Padilla SM, Sarkar S, Emr SD. Retromer <br />      germination. Plant Physiol 2007; 143:1132-9.               50.	 Zhang Y, Grant B, Hirsh D. RME-8, conserved                   function endosome-to-Golgi retrograde transport is <br /> 31.	 Takacs-Vellai K, Vellai T, Puoti A, Passannante M,              J-domain protein, required endocytosis                regulated yeast Vps34 PtdIns 3-kinase. J Cell Sci <br />      Wicky C, Streit A, et al. Inactivation autophagy         Caenorhabditis elegans. Mol Biol Cell 2001; 12:2011-21.         2002; 115:3889-900. <br />      gene bec-1 triggers apoptotic cell death C. elegans.    51.	 Nilsson L, Conradt B, Ruaud AF, Chen CC, Hatzold           70.	 Gumienny TL, Brugnera E, Tosello-Trampont AC, <br />      Curr Biol 2005; 15:1513-7.                                      J, Bessereau JL, et al. Caenorhabditis elegans num-1            Kinchen JM, Haney LB, Nishiwaki K, et al. CED-12/ <br /> 32.	 Komatsu M, Wang QJ, Holstein GR, Friedrich VL Jr,               negatively regulates endocytic recycling. Genetics 2008;        ELMO, novel member CrkII/Dock180/Rac <br />      Iwata J, Kominami E, et al. Essential role autophagy        179:375-87.                                                     pathway, required phagocytosis cell migra- <br />      protein Atg7 maintenance axonal homeostasis      52.	 Apodaca G, Katz LA, Mostov KE. Receptor-mediated                tion. Cell 2001; 107:27-41. <br />      prevention axonal degeneration. Proc Natl            transcytosis IgA MDCK cells apical recy-      71.	 Zhou Z, Hartwieg E, Horvitz HR. CED-1 trans- <br />      Acad Sci USA 2007; 104:14489-94.                                cling endosomes. J Cell Biol 1994; 125:67-86.                   membrane receptor mediates cell corpse engulf- <br /> 33.	 Kuma A, Hatano M, Matsui M, Yamamoto A, Nakaya             53.	 Chen CC, Schweinsberg PJ, Vashist S, Mareiniss DP,              ment C. elegans. Cell 2001; 104:43-56. <br />      H, Yoshimori T, et al. The role autophagy             Lambie EJ, Grant BD. RAB-10 required endo-          72.	 Zou W, Lu Q, Zhao D, Li W, Mapes J, Xie Y, Wang <br />      early neonatal starvation period. Nature 2004;              cytic recycling Caenorhabditis elegans intestine.        X. Caenorhabditis elegans myotubularin MTM-1 nega- <br />      432:1032-6.                                                     Mol Biol Cell 2006; 17:1286-97.                                 tively regulates engulfment apoptotic cells. PLoS <br /> 34.	 Zhong Y, Wang QJ, Li X, Yan Y, Backer JM, Chait            54.	 Shi A, Pant S, Balklava Z, Chen CC, Figueroa V, Grant           Genet 2009; 5:1000679. <br />      BT, et al. Distinct regulation autophagic activ-             BD. A novel requirement C. elegans Alix/ALX-1          73.	 Kinchen JM, Doukoumetzidis K, Almendinger J, <br />      ity Atg14L Rubicon associated Beclin                RME-1-mediated membrane transport. Curr Biol                 Stergiou L, Tosello-Trampont A, Sifri CD, et al. A <br />      1-phosphatidylinositol-3-kinase complex. NatCell Biol           2007; 17:1913-24.                                               pathway phagosome maturation engulfment <br />      2009; 11:468-76.                                           55.	 Shi A, Sun L, Banerjee R, Tobin M, Zhang Y, Grant               apoptotic cells. Nat Cell Biol 2008; 10:556-66. <br /> 35.	 Itakura E, Mizushima N. Atg14 UVRAG: mutu-                  BD. Regulation endosomal clathrin retromer-         74.	 Komatsu M, Waguri S, Ueno T, Iwata J, Murata S, <br />      ally exclusive subunits mammalian Beclin 1-PI3K              mediated endosome Golgi retrograde transport                 Tanida I, et al. Impairment starvation-induced and <br />      complexes. Autophagy 2009; 5:534-6.                             J-domain protein RME-8. EMBO J 2009;                     constitutive autophagy Atg7-deficient mice. J Cell <br /> 36.	 Collins BM. The structure function retromer          28:3290-302.                                                    Biol 2005; 169:425-34. <br />      protein complex. Traffic (Copenhagen, Denmark)             56.	 Burack MA, Silverman MA, Banker G. The role             75.	 Stenmark H. Rab GTPases coordinators vesicle <br />      2008; 9:1811-22.                                                selective transport neuronal protein sorting. Neuron         traffic. Nat Rev 2009; 10:513-25. <br /> 37.	 Bonifacino JS, Hurley JH. Retromer. Curr Opin Cell              2000; 26:465-72.                                           76.	 Bucci C, Thomsen P, Nicoziani P, McCarthy J, van <br />      Biol 2008; 20:427-36.                                      57.	 Lin SX, Grant B, Hirsh D, Maxfield FR. Rme-1                    Deurs B. Rab7: key lysosome biogenesis. Mol Biol <br /> 38.	 Chen D, Xiao H, Zhang K, Wang B, Gao Z, Jian Y,                 regulates distribution function endocytic        Cell 2000; 11:467-80. <br />      et al. Retromer required apoptotic cell clear-           recycling compartment mammalian cells. Nat Cell         77.	 Papini E, Satin B, Bucci C, Bernard M, Telford <br />      ance phagocytic receptor recycling. Science 2010;            Biol 2001; 3:567-72.                                            JL, Manetti R, et al. The small GTP binding protein <br />      327:1261-4.                                                                                                                     rab7 essential cellular vacuolation induced by <br />                                                                                                                                      Helicobacter pylori cytotoxin. EMBO J 1997; 16:15-24. <br />  <br />  <br />  <br /> www.landesbioscience.com	Autophagy	                                                                                                                                                        399 <br />  78.	 van Weert AW, Geuze HJ, Groothuis B, Stoorvogel W.       82.	 Kinchen JM, Ravichandran KS. Identification        87.	 Shaham S, Ed. WormBook: Methods Cell Biology <br />      Primaquine interferes membrane recycling            evolutionarily conserved genes regulating processing        (January 02, 2006), WormBook, Ed. The C. elegans <br />      endosomes plasma membrane direct             engulfed apoptotic cells. Nature 464:778-82.                   Research Community, WormBook, doi/10.1895/ <br />      interaction endosomes does involve        83.	 Matsunaga K, Saitoh T, Tabata K, Omori H, Satoh                wormbook.1.49.1, http://www.wormbook.org. <br />      neutralisation endosomal pH osmotic swelling           T, Kurotori N, et al. Two Beclin 1-binding proteins,      88.	 Clokey GV, Jacobson LA. The autofluorescent &#226;&#8364;&#339;lipo- <br />      endosomes. Eur J Cell Biol 2000; 79:394-9.                 Atg14L Rubicon, reciprocally regulate autophagy            fuscin granules&#226;&#8364;? intestinal cells Caenorhabditis <br /> 79.	 Seaman MN, Marcusson EG, Cereghino JL, <br /> </body></html>