<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, June 1991, p. 3009-3019                                                                  Vol. 11, No. 6 <br /> 0270-7306/91/063009-11$02.00/0 <br /> Copyright &#194;&#169; 1991, American Society Microbiology <br />  <br />  <br />    SPT5, Essential Gene Important Normal Transcription in <br />  Saccharomyces cerevisiae, Encodes Acidic Nuclear Protein with <br />                     Carboxy-Terminal Repeat <br />                        MICHELE S. SWANSON, ELIZABETH A. MALONE, AND FRED WINSTON* <br />                         Department Genetics, Harvard Medical School, Boston, Massachusetts 02115 <br />                                           Received 28 January 1991/Accepted 13 March 1991 <br />  <br />               Mutations SPTS gene Saccharomyces cerevisiae isolated previously suppressors 8 insertion <br />            mutations HIS4 LYS2. In study shown sptS mutations suppress his4-9128 and <br />            lys2-1288 alleles altering transcription. We cloned SPTS gene increase a <br />            decrease copy number wild-type SPTS gene caused Spt- <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Construction analysis <br />            sptS null <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> demonstrated SPT5 essential growth, suggesting SPT5 required <br />            normal transcription large number genes. The SPT5 DNA sequence determined; predicted a <br />            116-kDa protein extremely acidic amino terminus novel six-amino-acid repeat carboxy <br />            terminus (consensus = S-T/A-W-G-G-A/Q). By indirect immunofluorescence microscopy showed a <br />            bifunctional SPT5-0-galactosidase protein located yeast nucleus. This molecular analysis the <br />            SPT5 gene revealed number interesting similarities previously characterized SPT6 gene S. <br />            cerevisiae. These results suggest SPT5 SPT6 act related fashion influence essential <br />            transcriptional processes S. cerevisiae. <br />  <br />    A large number mutations alter transcription          gene suppress 8 insertion mutations altering transcrip- <br /> yeast Saccharomyces cerevisiae isolated selection               tion. We cloned SPT5 gene used cloned gene for <br /> for suppressors Ty solo 8 insertion mutations HIS4             additional molecular genetic studies SPT5. The nu- <br /> and L YS2 (19, 64, 65). These mutations identified 16              cleotide sequence SPT5 gene predicts 1,063-amino- <br /> unlinked genes, named SPT genes (for suppressor Ty).                 acid protein highly acidic amino terminus 15 <br /> Molecular analysis revealed SPT genes             copies six-amino-acid repeat carboxy terminus. <br /> encode proteins believed play central roles              Analysis SPT5 repeat domain indicated this <br /> gene expression: SPT15 encodes TATA-binding protein                 region required SPT5 function. Like SPT6, SPT5 is <br /> TFIID (16), SPTJJ SPT12 the genes               essential growth, proper SPT5 gene dosage critical for <br /> HTA1 HTBJ, encode histone proteins H2A                    function, SPT5 protein appears located the <br /> and H2B, respectively (12). Genetic analysis remain-             yeast nucleus. This molecular analysis supports idea that <br /> ing spt mutants allowed classification of SPT               SPT5 SPT6 functionally related required for <br /> genes groups, based distinct mutant pheno-                normal transcription S. cerevisiae. <br /> types: group includes SPTJJ SPT12, second <br /> includes SPT15, includes SPT131GALII (19, <br /> 20).                                                                                  MATERIALS AND METHODS <br />    The SPT5 gene, characterized study, SPT6 <br /> gene, analyzed previously (13, 45, 61), belong pheno-               Strains genetic methods. All S. cerevisiae strains used <br /> typic class represented histone genes HTAJISPTJJ                 studies (Table 1) constructed laboratory <br /> and HTBJISPTJ2 (12, 19). Mutations genes               derivatives strain S288C (MATot gal2) for <br /> suppress set Ty 8 insertion mutations            strains YPH149 (24), SF402-4D (Yeast Genetic Stock Cen- <br /> HIS4 L YS2 genes (19, 64) cis-                  ter, Berkeley, Calif.), K396-11A K396-22B (36). <br /> trans-acting mutations severely reduce expression              Parentheses indicate autonomous plasmids, brackets indi- <br /> of SUC2 gene Ty elements (11, 26, 45, 46, 62).                  cate integrated plasmids, double brackets indicate au- <br /> Previous studies demonstrated altered dosage               tonomous linear chromosomal fragments. Standard methods <br /> the SPT6 gene HTAJ-HTBJ locus causes similar                     mating, sporulation, tetrad analysis used (43, <br /> mutant phenotypes (12, 13, 45). Molecular analysis demon-               56). The lys2-1288 allele consists solo 8 element inserted <br /> strated SPT6 encodes essential nuclear protein 170           153 bp downstream LYS2 translation initiation codon <br /> kDa extremely acidic amino terminus (13, 45, 61).               (13, 59). The his4-9128 allele insertion solo 8 <br /> These studies suggested SPT6 plays essential role            element bp -97 relative HIS4 transcription initiation <br /> transcription, affecting chromatin structure.                site (8, 18). Strains MS194 MS195 contain SPT5-lacZ <br />   On basis observation spt5 mutants                 hybrid genes integrated spt5A202::LEU2 allele. These <br /> phenotypically similar spt6 histone mutants,                  strains constructed transforming diploid strain <br /> decided analyze SPT5 gene product. In               BM80 pMS50 DNA (MS194) pBM68 DNA (MS195) <br /> study, demonstrated mutations SPT5                  linearized restriction digestion PstI. <br />                                                                         The site integration determined sporulating trans- <br />                                                                         formants, dissecting tetrads, scoring Ura Leu <br />                                                                         phenotypes spore clones. <br />   * <br />       Corresponding author.                                                Media. Rich medium (YPD), minimal medium (SD), sup- <br />                                                                  3009 <br />  3010       SWANSON ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />                                                                    TABLE 1. Yeast strains <br />  Strain                                                                                            Genotype <br /> FY98 .....................................                 MATa ura3-52 leu2AJ <br /> FY360 .............             ........................   MATa spt5-194 Ieu2AJ <br /> FY382 .............             ........................   MATa sptS-8 ura3-52 leu2AJ <br /> FY120 .............             ........................   MATa his4-9128 Iys2-1288 ura3-52 Ieu2Al <br /> FY300 ............              ........................   MATa sptS-194 ura3-52 his4-9128 Iys2-1288 leu2AJ <br /> FY276 ............              ........................   MATa sptS-8 his4-9128 Iys2-1288 ura3-52 leu2Al <br /> MSll ...........              .........................    MATot sptS-8 lys2-1288 ura3-52 his4-917 Ieu2 <br /> MS63 ...........              .........................    MATa spt5-24 his4-9128 lys2-1288 ura3-52 ade2-1 trp5 can1-100 <br /> FW1237 .....................................               MATa his4-9128 lys2-1288 ura3-52 <br /> MS98 ...........              .........................    MATa his4-9128 lys2-1288 ura3-52 [SPT5-pMS15-URA3] <br /> MS17 ...........              .........................    MATa spt5-24 his4-9128 lys2-1288 ura3-52 ade2-1 canl-100 <br /> FY298 ............              ........................   MATa spt5-194 his4-9128 lys2-1288 ura3-52 Ieu2Al ade8 <br /> BM60 ...........              .........................    MA Ta/MA Ta ura3-521ura3-52 his4-91281his4-9128 Iys2-128811ys2-1288 leu2-3,1121leu2-3,112 <br />                                                             trplAlltrplAl <br /> BM80 .....................................                 MATa/MATTa spt5A202::LEU21SPT5 ura3-521ura3-52 his4-91281his4-9128 lys2-12881lys2- <br />                                                             1288 leu2-3,1121leu2-3,112 trplAJ/trpJAl <br /> MS189 .....................................                MATa ura3-52 his4-9128 lys2-1288 leu2Al (pCGS42) <br /> MS190 ....................................                 MATa ura3-52 his4-9128 lys2-1288 leu2AJ (pMS4) <br /> MSl91 ....................................                 MATa ura3-52 his4-9128 lys2-1288 leu2AJ (pMS24) <br /> YPH149 ....................................                MATa ade his7 Iys2 trpJAl ura3-52 [[CFIURA31RAD2 distal]] [[CFITRPJIRAD2 <br />                                                              proximal]] <br /> K396-11A ....................................              MATa Iys7 spoil ura3 adel his] Ieu2 met3 trp5 <br /> K396-22B ....................................              MATot Iys7 spoll ura3 adel his] Ieu2 met3 trpS <br /> SF402-4D .....................................             MATa sec59 <br /> MS113.MATa                                                        sptS-24 sec59 <br />   M 13.......................................................MA ast24            his4-9128 lys2-I288 trpJAJl <br />                                                                            ec9h4918l218               rpA <br /> MS192 ....................................                 MATa lys7 his4-9128 ura3-52 <br /> MS193 ....................................                 MAToa Iys7 his4-9128 ura3-52 adel <br /> MS194 .............            ........................ MA Ta/MA <span id='am-5' about='protege:TO' typeof='owl:Thing'>TO</span> [spt5A202: :LEU2-SPT5(917)-lacZ-URA3]ISPT5 ura3-521ura3-52 his4-91281 <br />                                                              his4-9128 lys2-12881lys2-1288 Ieu2A1/Ieu2AJ trpJAJItrplAJ <br /> MS195 .....................................                MATa/MATot [spt5A202::LEU2-SPT5(1004)-lacZ-URA3]/SPT5 ura3-521ura3-52 his4-91281 <br />                                                              his4-9128 lys2-12881lys2-1288 Ieu2A1JIeu2Al trplAlltrplAl <br /> BM418 .....................................                MAToa spt5-194 his4-9128 lys2-1288 ura3-52 ade8 <br /> BM437 .....................................                MATot spt5-194 his4-9128 lys2-1288 ade8 [ura3-52-SPT5(1004)-1acZ-URA3] <br /> BM443 ....................................                 MATa his4-9128 Iys2-1288 Ieu2Al [ura3-52-SPT5(1004)-lacZ-URA3] <br /> BM444 ....................................                 MAToa his4-9128 Iys2-1288 Ieu2AI ade8 [ura3-52-SPT5(1004)-lacZ-URA3] <br /> BM448 ....................................                 MATa/MATTa his4-91281his4-9128 lys2-128811ys2-1288 Ieu2Al1Ieu2Al ade8/ADE8 [ura3-52- <br />                                                              SPT5(1004)-lacZ-URA3]1[ura3-52-SPT5(1004)-lacZ-URA3] <br /> BM454 ....................................                 MATa/MATa his4-91281his4-9128 Iys2-12881/ys2-1288 Ieu2AI/Ieu2AJ ade8/ADE8 [ura3-52- <br />                                                             SPT5(1004)-lacZ-URA3]1[ura3-52-SPT5(1004)-lacZ-URA3] <br /> BM455 ....................................                 MATTa/MATTa his4-91281his4-9128 Iys2-128811ys2-1288 Ieu2Al1Ieu2AI ade8/ADE8 [ura3-52- <br />                                                             SPT5(1004)-lacZ-URA3]1[ura3-52-SPT5(1004)-lacZ-URA3] <br /> BM467 .....................................                BM454 x BM455 <br /> BM330 .....................................                MATa/MATTa his4-91281his4-9128 Iys2-128811ys2-1288 ura3-521ura3-52 trplA631trplA63 <br /> BM331 .....................................                MA Ta/MA Ta his4-91281his4-9128 lys2-12881Iys2-1288 ura3-521ura3-52 trpIA631trplA63 <br /> BM339 ....................................                 BM331 x BM330 <br />  <br />  <br /> plemented SD (SD amino acids added                             Seakem agarose (FMC Bioproducts, Rockland, Maine) and <br /> liquid stock solutions), synthetic complete medium lacking                      purified electroelution, using device designed Larry <br /> single amino acid, sporulation medium prepared                        Peck (49). Southern blot hybridization analysis per- <br /> described previously Sherman et al. (56). GNA presporu-                        formed described previously (51). Radiolabeled DNA <br /> lation medium contained 10 g yeast extract, 30 g                            probes prepared nick translation (50) random <br /> <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> broth, 50 g dextrose, 20 g agar liter.                     priming (21, 22), using [a-32P]dATP (Amersham) reagent <br />   Transformations. Yeast cells transformed                            kits Boehringer Mannheim. <br /> lithium acetate method (32). Escherichia coli HB101 (3)                          RNA preparation analysis. Yeast RNA prepared <br /> TB1 (Bethesda Research Laboratories, Gaithersburg, Md.)                           described previously (7) cells grown supple- <br /> were transformed described previously (41).                                    mented SD medium 25&#194;&#176;C density 1 x 107 2 x 107 <br />   Enzymes. Restriction enzymes purchased                                cells ml. RNA separated formaldehyde-agarose <br /> Boehringer Mannheim Biochemicals (Indianapolis, Ind.)                         gels. After denaturation 0.05 M NaOH renaturation in <br /> New England BioLabs (Beverly, Mass.). T4 DNA ligase                           0.1 M Tris (pH 7.5), RNA transferred GeneScreen <br /> purchased New England BioLabs. All enzymes                              (New England Nuclear, Boston, Mass.) capillary blotting <br /> used according instructions supplier.                               1x SSC (0.15 M NaCl, 0.015M sodium citrate). RNA was <br />   DNA preparation analysis. Yeast genomic plasmid                         cross-linked membrane UV irradiation (10). Probes <br /> DNAs prepared described previously (28). E. coli                          hybridized dextran sulfate method described <br /> plasmid DNA prepared method Holmes                              GeneScreen instruction manual. Additional probes <br /> Quigley (29) Birnboim Doly (2). DNA restriction                            hybridized membrane stripped 1 x <br /> fragments separated electrophoresis                               SSC-50% formamide 80&#194;&#176;C 2 h. The RNA in <br />  VOL. 1 l, 1991                                                                         SPT5 GENE OF S. CEREVISIAE           3011 <br />  <br /> each lane normalized hybridization TUB2 DNA             acid, 1.25 mM EDTA). After electrophoresis, gel was <br /> (plasmid pYST138; 60).                                            soaked sequentially 20 min 0.25 M HCl, 0.5 M <br />    Plasmids. Vectors used SPT5 subcloning YCpS0          NaOH-1.0 M NaCl, 1.5 M NaCl-1.0 M Tris (pH 7.4). <br /> (34) pMS37 pRS316 (57) pMS18 pMS23.               The DNA transferred nitrocellulose capillary <br /> Subclones tested SPT5 function transforma-         blotting 4x SSC. Radiolabeled pMS4 DNA hybridized to <br /> tion spt5 mutant strain MS63(pMS18) FY298             chromosomes result composition: chro- <br /> (pMS23, pMS37). For integration cloned yeast DNA,          mosome IV (CEN4), chromosome V (URA3), frag- <br /> we constructed plasmid pMS15 cloning 4.0-kb                ments chromosome VII (pBR322, URA3), chromo- <br /> HindIII-KpnI fragment pMS4 HindIII-KpnI sites         XIII (SPT5). Linkage SPT5 centromere to <br /> of integrating plasmid pRS306 (57). To study effects      markers chromosome XIII determined tetrad <br /> of multiple copies SPT5 gene, plasmid pMS24            analysis. sptS-24 scored ability suppress the <br /> constructed cloning 4.8-kb HindIlI-EagI fragment        His- phenotype conferred 8 <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>insertion mutation</span> his4- <br /> pMS4 HindIII-EagI sites vector pCGS42             9128. The secS9 allele MS113 derived strain <br /> (Collaborative Research), contains 2,um circle.     SF402-4D. <br /> Plasmids used probes follows: HIS4, pFW45,            DNA sequence analysis. Restriction fragments se- <br /> a BglII-SalI restriction fragment internal HIS4 cloned      quenced cloned vector M13mp18 M13mpl9 <br /> pBR322 (66); LYS2, pFW47, BglII-XhoI restriction            (48). The nucleotide sequence determined the <br /> fragment internal LYS2 cloned pBR322 (13),              method Sanger et al. (54), using [a-35S]dATP (Amersham) <br /> pFW112, EcoRI-BglII restriction fragment 5&apos;           Sequenase version 2.0 kit (U.S. Biochemical Corp., <br /> region LYS2 cloned pBR322 (13); TUB2, pYST138,          Cleveland, Ohio). In 12 cases, conve- <br /> a 0.24-kb BglII-KpnI restriction fragment internal TUB2        nient restriction sites, 20-<span id='am-396' about='xsp:base' typeof='owl:Thing'>base</span> oligomers synthesized <br /> cloned pGEM vector (60); SPT5, pMS22,              (Mark Fleming, Biopolymers Laboratory, Department of <br /> 4.0-kb HindIII-KpnI fragment pMS4 cloned pBR322.          Genetics, Harvard Medical School) used primers. <br />    We constructed fusions SPT5 E. coli lacZ gene     The complete DNA sequence determined both <br /> using integrating plasmids containing polylinker followed       strands. <br /> by lacZ gene URA3 gene selectable marker            Indirect immunofluorescence. For indirect immunofluores- <br /> (42). Plasmid pBM68 constructed cloning 4.0-kb         cence experiments, used tetraploid cells are <br /> HindIII-KpnI restriction fragment pMS4 HindlIl        larger haploid cells. A tetraploid strain containing an <br /> and KpnI sites YIp356R; SPT5-lacZ fusion gene              SPT5-lacZ fusion constructed steps. First, we <br /> contained pBM68, designated SPT5(1004)-lacZ, encodes           constructed haploid strains containing SPT5(1004)-lacZ <br /> the 1,004 amino acids SPT5. Plasmid pMS50            fusion transforming strain BM418 uracil prototrophy <br /> constructed cloning 3.7-kb HindIII-PvuII restriction       pBM68 DNA linearized digestion with <br /> fragment pMS4 HindIII-SmaI sites YIp357R;          StuI. Next, stable integrant transformation, <br /> the SPT5-lacZ fusion gene contained pMS50, designated          BM437, crossed strain FY120 yield SPT5 strains <br /> SPT5(917)-lacZ, encodes 917 amino acids SPT5.        carrying SPT5(1004)-lacZ fusion. Two strains <br /> Plasmids pBM68 pMS50 unique StuI                  (BM443 BM444) mated form <br /> restriction site URA3 gene unique PstI site   diploid BM448. We isolated diploids homozygous at <br /> SPTS gene useful directing integration        MAT UV irradiating BM448 (300 ergs/mm2) screening <br /> plasmids URA3 SPT5 genes, respectively,             diploids mate cells. Mating of <br /> transformation experiments.                                       MA TalMA Ta diploid BM454 MA TaIMA Ta diploid <br />    We observed E. coli strains carrying SPT5 DNA          BM455 yielded tetraploid strain BM467, carries <br /> high-copy-number plasmids formed unusually small colo-            copies SPT5(1004)-lacZ fusion. Strain BM339, <br /> nies.                                                             similar BM467 does contain SPT5- <br />    Construction spt5 <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>null allele</span>. To construct sptS     lacZ fusion, constructed method. <br /> null allele, sptSA202::LEU2, replaced 0.8-kb Sall-            Cells prepared immunofluorescence the <br /> SalI restriction fragment pMS23 2.2-kb SalI-XhoI        method Kilmartin Adams (35) modifications <br /> restriction fragment containing LEU2 gene (1) gener-       described previously (61). The SPT5-p-galactosidase fusion <br /> ate plasmid pBM19. Next, 3.1-kb PvuII-PstI restriction        protein identified incubating prepared cells a <br /> fragment pBM19, contains LEU2 gene                 monoclonal rabbit anti-,B-galactosidase antibody (Cappel, <br /> flanked SPT5 sequences, used transform              Malvern, Pa.) diluted 1:10,000 fluorescein-conjugated <br /> diploid BM60 leucine prototrophy. Strain BM80 stable      goat anti-rabbit antibody (Sigma, St. Louis, Mo.) diluted <br /> Leu+ transformant resulting recombination            1:250. DNA stained fluorescent dye 4&apos;,6&apos;- <br /> the fragment genome copy SPT5        diamidino-2-phenylindole dihydrochloride (DAPI). Fluores- <br /> gene replaced null allele (53). The structure   cent staining visualized using Zeiss Photomicro- <br /> SPTS loci diploid BM80 verified Southern            scope III equipped epi-illumination fluorescence a <br /> hybridization analysis (data shown). The phenotype         Zeiss Neofluar 63x lens, numerical aperture 1.25, pho- <br /> the spt5 null allele haploids analyzed following           tographed Kodak TMAX-400 film. The exposure time <br /> sporulation BM80 dissection tetrads.                    micrographs fluorescein fluorescence experi- <br />    Mapping SPT5. The method Carle Olson (6)            mental control strains 30 s. <br /> used prepare chromosome-size DNA strain                     Nucleotide sequence accession number. The GenBank ac- <br /> YPH149. Chromosome VII YPH149 fragmented              cession number SPT5 sequence M62882. <br /> at RAD2 allow yeast chromosomes be <br /> resolved discrete bands alternating field gel electro-                               RESULTS <br /> phoresis (24). DNA separated clamped homogeneous <br /> electric field (CHEF) gel electrophoresis (9) 0.9%          sptS mutations alter transcription. Recessive mutations in <br /> agarose 0.5x TBE buffer (45 mM Tris base, 45 mM boric          SPT5 identified suppressors insertion <br />  3012        SWANSON ET AL.                                                                                                                          MOL. CELL. BIOL. <br />  <br />                  A.                                                                      A. <br />                                K/S4       hls4--912 6                                                          L YKS              /ys2i- 128 <br />                                                                                                          i <br />                                                                                                                  Q-                      - <br />                                                                                                                                                     I: <br />                                                                                                          z,S     A)     t. <br />                                                                                                                              C.   K,fI <br />  <br />                                                                                             L KS2        --O           mm <br />  <br />  <br />  <br />                  HIS4     (     _             7 <br />  <br />  <br />  <br />  <br />                  FUB2                 -m                                                    L YS2                                 * <br />                            1    2     3   4    5   6 <br />        B. <br />                  TATA <br />         UAS                                                                                              1 2           3          4          5 6 <br />  <br />                                                                              B <br />               SPT5 *                                       H js <br />                                                                                UAS    TATA          :i <br />                                                                                                                        :1,                   LY52 <br />                spf5                       _Hiis&apos; <br />                                                                                      spT5                                                                   Lys <br />  <br />   FIG. 1. Effects spt5 mutations transcription his4-9128. <br /> (A) Total yeast RNA subjected Northern analysis. The same <br /> membrane hybridized HIS4 probe pFW45 then <br />                                                                                      spf5 <br /> rehybridized TUB2 probe pYST138. Approximately 2.5 ,ug of <br /> RNA loaded lanes 1 3, approximately 10 ,ug was <br /> loaded lanes 4 6. The strains used (left right) FY98,        FIG. 2. Effects spt5 mutations transcription lys2-1288. <br /> FY360, FY382, FY120, FY300, FY276. (B) The line depicts         (A) Total yeast RNA subjected Northern analysis. The same <br /> the structure his4-9128. The gray box represents HIS4 open       membrane hybridized mixture LYS2 probes <br /> reading frame. The lines represent flanking DNA. The box      pFW47 pFW112 rehybridized TUB2 probe <br /> the solid triangle represents solo 8 element. Labels indicate   pYST138. Since smaller L YS2 RNA abundant, the <br /> the relative positions known TATA boxes (TATA) upstream          pFW47 probe contained approximately 10-fold counts per <br /> activating sequences (UAS). The lower lines depict probable         minute pFW112 probe. Approximately 2.5 ,ug RNA was <br /> origins transcripts SPT5+ sptS strains.                       loaded lanes 1 3, approximately 10 ,ug loaded lanes <br />                                                                         4 6. The strains used given Fig. 1A. (B) <br />                                                                         The line depicts structure lys2-1288. The gray box <br />                                                                         represents L YS2 open reading frame. The lines represent <br /> mutations his4-9128 lys2-1288 (19, 64). To determine                flanking DNA. The box solid triangle represents solo 8 <br /> whether phenotypic changes spt5 mutants result              element. Labels indicate relative positions known <br /> alterations transcription, performed Northern (RNA)               TATA boxes (TATA) upstream activating sequences (UAS). <br /> hybridization analysis. Our results (Fig. 1 2) demon-               The lower lines depict probable origins transcripts SPT5S <br /> strate sptS mutations cause altered transcription          spt5 strains. <br /> his4-9128 lys2-1288. <br />    In SPT+ background, transcription his4-9128 ini- <br /> tiates 8 initiation site results mRNA          transcript approximately 580 bases (Fig. 2A, lane 4), <br /> longer wild-type HIS4 mRNA (27, 58; Fig. 1). The               probably initiation L YS2 initiation site and <br /> 5&apos; portion transcript contains in-frame translation             termination 8 transcription termination site (13; Fig. <br /> initiation termination codons presumably prevent              2B). In spt5 mutant strains, second transcript also <br /> normal translation HIS4 coding region (58). In              produced (Fig. 2A, lanes 5 6). For spt5-194 mutants, this <br /> sptS-194 spt5-8 mutants, shorter transcript comi-            second transcript easily seen; spt5-8 mutants, the <br /> grated wild-type HIS4 mRNA produced                     new transcript low level seen <br /> addition 8-initiated transcript (Fig. 1A, lanes 5 6).        clearly longer exposures. (The strains were <br /> The levels transcripts roughly equal. The               equally Lys&apos;.) This mRNA slightly shorter the <br /> size new transcript indicates spt5 mutations allow          wild-type LYS2 transcript likely result tran- <br /> transcription initiation normal HIS4 initiation site (Fig.       scription initiation near 8 initiation site elongation <br /> 1B). This alteration his4-9128 transcription seen         LYS2 gene (Fig. 2B). Mutations SPT6 and <br /> spt6 htal-htbl (histone) mutants (12, 13).                          HTAI-HTBI cause similar changes lys2-1288 transcription <br />    Mutations SPT5 affected transcription lys2-1288.          (12, 13). <br /> In SPT7 strains, transcription lys2-1288 resulted short           Transcription sequences examined. The <br />  VOL . 1 1, 1991                                                                                                              SPT5 GENE OF S. CEREVISIAE        3013 <br />  <br />                                                                                                                 :4 <br />  <br />                                   :1:: <br />                                                                        &quot;Z&apos;.                           C <br />  <br />  <br />  <br />  <br />                                                                         IQ ~                          ~               ~ <br />  <br />  <br />                                                                                                                              pMS4        + <br />                                                                                                                              pMSi8       - <br />                                                                                                                              pMS23      +/- <br />                                                                                                                              pMS37       - <br />                                                                 LEU2 <br />                                                              \                ,/                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~I <br />                                                                                                                    pBMl9 <br />  <br />  <br />   FIG. 3. Restriction map SPT5 locus. The line represents SPT5 locus restriction sites. The thinner portion line <br /> represents vector sequences. The lines represent DNA fragments subcloned test SPTS function. Plasmid pBM19 was <br /> used construct sptS null mutant. <br />  <br />  <br /> level wild-type HIS4 L YS2 transcripts slightly                 gene (see Materials Methods). Integration allele <br /> reduced spt5 mutations (Fig. 1A, lanes 1 3; Fig. 2A,                     diploid strain BM60 resulted strain heterozygous <br /> lanes 1 3). There reduction level                          null allele. After sporulation tetrad dissection of <br /> full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Ty transcripts spt5 mutants (data shown).                   strain (BM80), viability segregated 2:2 (9 10 tetrads) <br /> Similarily, spt6 htal-htbl (histone) mutations                      1:3 (1 10 tetrads). In cases, viable spores were <br /> affect level full-length Ty transcripts (4, 13). In                     Leu-, indicating carried wild-type SPT5 gene. <br /> contrast, spt mutants different phenotypic class (spt3,                   Therefore, SPT5 gene essential growth. <br /> spt7, spt8, sptl5) abolish greatly reduce level                     Altered SPTS gene dosage causes mutant phenotypes. Al- <br /> full-length Ty transcripts (17, 65, 66).                                       tered dosage SPT6 HTAI-HTBI locus confers <br />    Isolation SPT5 gene. An SPT5 gene clone                          Spt- mutant phenotype (12, 13, 45). To test whether <br /> isolated library yeast DNA (52) screening                     altered SPT5 gene dosage causes similar phenotypes, we <br /> plasmids complemented spt5 mutant phenotype. The                       constructed set isogenic strains re- <br /> recipient strain, MS11 (spt5-8 lys2-1288 ura3-52), Lys&apos;                     duced increased copy number SPTS gene. These <br /> due suppression lys2-1288 spt5-8. Since sptS-8                     strains tested suppression insertion <br /> recessive, transformants MS11 carry wild-type                      mutations his4-9128 lys2-1288. A diploid strain (BM80) <br /> SPT5 gene plasmid Lys-. Two Lys- candi-                         contained copy SPT5 gene (SPT5SI <br /> dates identified screen approximately 15,000                      sptSA202::LEU2) strong Spt- phenotype (His&apos; Lys&apos;) <br /> Ura+ transformants; contained DNA insert,                        (Fig. 4). A haploid SPTS+ strain (MS191) transformed a <br /> and plasmids designated pMS4. This                            multicopy SPT5 plasmid (pMS24) weak Spt- pheno- <br /> plasmid conferred Spt+ phenotype retransformed                         type (His+&apos; Lys+&apos;-) (Fig. 4). Therefore, increase a <br /> into strain MS11 transformed strain MS63,                         decrease SPT5 gene copy number causes Spt- <br /> which contains different spt5 allele.                                        phenotype. <br />    We confirmed pMS4 contained SPT5 gene                              Genetic mapping SPTS. To determine SPT5 was <br /> demonstrating cloned DNA directed integration                    previously identified gene, mapped combination <br /> plasmid SPT5 locus. Plasmid pMS15, integrating                       physical genetic methods. To determine chromo- <br /> plasmid contains restriction fragment pMS4,                    SPT5 gene resides, DNA prepared <br /> linearized unique NruI site cloned DNA                     strain YPH149, chromosomes separated by <br /> used transform FW1237 (SPT5+ ura3-52 iys2-1288)                          CHEF gel electrophoresis (9) described Materials and <br /> uracil prototrophy. A Ura+ transformant (MS98) was <br /> crossed strain MS17 (sptS-24 ura3-52 iys2-1288), and <br /> tetrads dissected. In 17 four-spored three- <br /> spored tetrads, Spt+ Ura+ phenotypes cosegregated <br /> in tetrad, demonstrating pMS15 DNA tightly <br /> linked SPTS locus. <br />    To characterize SPTS gene greater detail,                        C)I IL, PT-5 -,-       ,l.        t -1r                    b) sp15-/-94 <br /> subclones constructed tested SPT5 function                            c)    F T54        2caSPT5                                 d) SJT:5 (CEASt-P T5 ) <br /> (Fig. 3). Although HindIII-KpnI fragment (pMS23) par-                               .../&amp;S  f.                                            f spto5A ,; SPT5 <br /> tially complemented spt5-194 allele, sub-                                                                                    ) <br />  <br />  <br /> clones fully complemented mutant phenotype, suggest- <br /> ing SPT5 gene occupies cloned 3.9-kb                        FIG. 4. Suppression insertion mutation Iys2-1288 al- <br /> DNA fragment.                                                                  tered SPTS gene dosage. Patches strains genotypes <br />    Construction analysis spt5 null mutant. We                        indicated grown YPD plate replica plated the <br /> constructed spt5 null allele marked LEU2 gene,                       plate shown, contains SC-lys medium. All strains grew a <br /> spt5A202::LEU2, deleting SalI-Sall restriction frag-                        synthetic complete plate (not shown). (a) MS189(pCGS42); (b) <br /> ment internal SPT5 (Fig. 3) replacing LEU2                      FY300; (c) MS191(pMS24); (d) MS190(pMS4); (e) BM60; (f) BM80. <br />  3014    SWANSON ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />                                                                                    TABLE 2. Mapping SPTS tetrad analysisa <br />                                             %I <br />                    ZcAN#                                                       SegregatingPD <br />                                                                        Crossb Sb <br />                                                                                markers                     NPD      TT           ~~~~Linkage Gene-CEN <br />                                                                                                                                              (cM) <br /> XJII                                                                              markers                                  ~~~~~Gene-gene <br />   FIG. 5. Genetic map position SPT5. The SPT5-CEN13 dis-               1       spt5, sec59       76        0       4           3 <br /> tance strain dependent (see text).                                            spt5, lys7        41        0      39          24 <br />                                                                                   secS9, lys7       43        0      37          23 <br />                                                                                   spt5, trpl        42       38       0                       &lt;0.6 <br />                                                                                   secS9, trpl       39       36       5                         3 <br /> Methods. Southern hybridization analysis demonstrated                        lys7, trpl        20       20      40                        25 <br /> SPTS located chromosome XIII (63).                                         trpl, ura3        36       32      12                         8 <br />                                                                                   sptS, ura3        35       34      11                         7 <br />    Tetrad analysis showed SPT5 located left               2       sptS, sec59       42        0       9           9 <br /> arm chromosome XIII, tightly centromere linked (Table 2;                       sptS, lys7        19        0      32          31 <br /> Fig. 5). The order SPT5-CENJ3-SEC59 determined                             sec59, lys7       28        0      23          23 <br /> examining tetrads SPT5 CEN13 recom-                              spt5, trpl        23       21       7                         7 <br /> bined; case (seven tetrads), SEC59 CEN13 did                         secS9, trpl       23       26       2                         2 <br /> not recombine. This order consistent                            lys7, trpl        11       15      25                        25 <br /> segregation SPT5 tetrads CENJ3 SEC59                           trpl, ura3        23       20       8                         8 <br /> had recombined. Similarly, order SPT5-SEC59-LYS7                              spt5, ura3        19       20      12                        12 <br /> was established scoring LYS7 segregation tetrads                 PD, parental ditype; NPD, nonparental ditype; TT, tetratype. The sptS-24 <br /> which SPT5 SEC59 recombined. In 13 tetrads,               allele scored ability suppress his4-9128; secS9 scored its <br /> SEC59 LYS7 recombined once, SPT5                    temperature-sensitive lethality 37&#194;&#176;C; trpJlW ura3-52 scored by <br /> LYS7 recombined 12 times. These results, conjunc-                tryptophan uracil auxotrophies, respectively; lys7 scored by <br />                                                                         mating spore clones lys7 strains (K396-11A K396-22B) scoring <br /> tion previous work showed L YS7 maps phys-               diploids complementation lysine auxotrophy. The TRPI allele, <br /> ically right arm chromosome XIII (23), indicated              tightly linked CEN4 (&lt;1 cM; 44), URA3 allele, which <br /> that SPT5 resides left arm chromosome XIII.                   tightly linked CENS (8 cM; 44), used calculate map distance <br /> Additional meiotic linkage analysis demonstrated SPT5              SPTS CEN13. <br />                                                                           b Cross 1 MS113 x MS192; cross 2 MS113 x MS193. <br /> was allele centromere-linked TSMOJJJ locus, <br /> since cross (82 tetrads), SPT5 mapped 4.9 centimor- <br /> gans (cM) TSMOJJJ (63). Therefore, based its <br /> unique map position, SPT5 previously uncharacterized               concentration acidic residues amino terminus: <br /> gene S. cerevisiae.                                                  residues 1 220 net charge -63. One region 81 <br />    The map distance SPT5 CEN13 strain                   amino acids (residues 137 217) 60% glutamic and <br /> dependent; representative crosses shown Table 2.             aspartic acid included stretches 11 20 consecutive <br /> In cross strain MS113 strain MS192,               acidic residues. The amino terminus included four <br /> CENJ3-SPT5 recombinants 80 tetrads (&lt;0.6 cM; Table 2).               consensus sites phosphorylation casein kinase II (37) <br /> In contrast, strain MS113 crossed strain                    positions 122, 188, 241, 377. The carboxy terminus of <br /> MS193, seven CEN13-SPT5 recombinants 51                   predicted SPT5 protein (residues 887 1063) 31% <br /> tetrads (7 cM; Table 2). Analysis SPT5 segregation              glycine contained 15 copies six-amino-acid se- <br /> respect different centromere-linked markers, TRPI (chro-             quence S-T/A-W-G-G-A/Q (Table 3). <br /> mosome IV) URA3 (chromosome V), showed                        A search National Biomedical Research <br /> difference strains MS192 MS193 lies                  Foundation Protein Sequence data base (version 26; Novem- <br /> CEN13-SPT5 interval (Table 2). Similar results ob-                 ber 1990) did reveal proteins significant sequence <br /> tained SPT5 crosses: CEN13-SPT5 map distance               similarity SPT5, exception number of <br /> was calculated &lt;1 cM approximately 5 cM (63).              generally acidic glycine-rich proteins, including nucleolins <br /> The observation genetic map position SPT5               (38). No proteins data base contained the <br /> strain dependent indicates polymorphisms                 six-amino-acid repeat S-T/A-W-G-G-A/Q. <br /> (possibly insertions inversions) CEN13 region                 The carboxy-terminal repeat region critical SPT5 <br /> some strains.                                                           function. Results subcloning experiments suggested <br />    SPT5 encodes acidic protein carboxy-terminal               SPT5 carboxy-terminal repeats required for <br /> repeat. The nucleotide sequence SPT5 gene                    SPT5 function. A deletion removed 6 15 repeats <br /> determined method Sanger et al. (54). The open                (pMS23) impaired complementation spt5-194 (Fig. 3) and <br /> reading frame 3,189 bp (Fig. 6) consistent              eliminated complementation spt5 null mutation. A <br />  size SPT5 mRNA (3.4 kb; 63) results                  deletion removed 15 repeats (pMS37) eliminated <br />  subcloning experiments (Fig. 3). The nucleotide sequence               complementation spt5-194 (Fig. 3) spt5 <br /> predicted 1,063-amino-acid protein 115,649 Da             null alleles. <br />  striking features: acidic amino terminus car-               To examine importance SPT5 peptide <br /> boxy-terminal amino acid repeat. The entire SPT5 protein                repeats, constructed SPT5-lacZ fusions: SPT5(1004)- <br />  predicted isoelectric point 4.95,             lacZ encodes residues 1 1004 SPT5 includes nine <br />  <br />  <br />    FIG. 6. Nucleotide sequence SPT5 gene predicted amino acid sequence gene product. Nucleotides numbered the <br /> left; amino acids numbered right. The termination codon represented asterisk. The runs 11 20 consecutive acidic <br /> residues 15 copies six-amino-acid repeat underlined. The PvuII KpnI sites used construct SPT5-lacZ fusions and <br /> the SalI sites used construct spt5 null allele indicated. <br />  VOL. 11, 1991                                                                                                   SPT5 GENE OF S. CEREVISIAE     3015 <br />  <br />                    1                      CTCTTAAGCCACTTCCCCCAAATCCACCTCTCTGCCACAGMCTAATAACCATCCACCATAMTCATCTATTACTCTTAAATTAT <br />                   86 TCATCACTATCAGTAACTGGCATGTCCATCGTACCAGTT&#194;&#167;  CTCGTAAATCTATTACGTCAGTTCCTTCTATAATGACGTCTTCATTCTCACCAGTTT&#194;&#167;CCAAMTTG <br />                  191TCCGCTCTAGTTGCAAATCCTTGAAGACCTTGGTAGTTCATTAACGGTCATATCATGGTCCAGACTGGATTCCGCATATTTTCCAGAAMCATCTTCAACTT <br />                 296 CGATAGGTACTTTCGGGTTCTCGAATAGCTCTACTCTTTCATAACTAACCTTAGTCGAI GICTCATTGAAAAACTCGCAAGGTAM AGGTGACCTTGCGG <br />                 401 MrrMATGCTGACACCTCCATGCGTAATTGTTATTATATATATATATGATAGTATAM CATACTTGTTCCTTCTAAAGTCACCCCGCCTGATATGCGACATA <br />                 506 GMAAAAATGATAAAGTCAAATTG    GTAGCATT      GACAAGACCATTCATAC AAGCTAGTACAGCAGCAAGAAGAGACGGGCC <br />                                                                                                              TGTTGTGATAATT <br />                 611 ATGAGTGACAACTCCGACACAAACGTGAGCATGCAGGACCATGATCAACAAM| CTGATCCCGTAGTCGTTCCGCAGTCAACTCACACTAAAGATGAAATACT <br />                         M S D N S D T N V S M Q D H D Q Q F A D P V V V P Q S T D T K D E N T                                           35 <br />                  716 AGTGACAAAGAACTGTTGATATGCATGCAACOAGTACAG         AACCATATTCCCCCTrTAGATGGGGMAAGAAAGA <br />                                                         AACGTGCAGAAGTAC <br />                      S D K D T V D S G N V T T T E S T E R A E S T S N I P P L D G E E K E                                              70 <br />                  821 GCAAAATCTGACCCACAGCACCTGAGGATAATGCAGAAACGCCGCCCACACACCACCI TrCATCTAGTAACGOGCCTGCTACAGATGATGCCCAAGCAACTTTG <br />                      A K S E P Q Q P E D N A E T A A T E Q V S S S N G P A T D D A Q A T L                                              105 <br />                  926    MTACOGATTCATCCGAAGCAAATGAAATTGTCAAMGAAGGAGAGCCTCTGATGAAACGAAGAGACCTCGCCGAAGAGACACCAAAACAGTGATGGTGATACC <br />                         N T D S S E A N E I V K K E E G S D E R K R P R E E D T K N S D G D T                                           140 <br />                                                                          ATGACGATAACMCTCCACTAAGCGTCGTCAG <br />                                                             GTAAGOGACWAGCA <br />                 1031 AAAGATGGCGATCAAAGATGAAGACGAGTAACCAGACGAT        C <br />                      K D E C D N K D E D        n F nD D D        D D F D D D D E A P T K R R R Q                                       175 <br />                 1136                                                                                                    ICATTACCCAT <br />                                                                                                              GTGA__[____ACGMGC <br />                         E R N R F L D I E A E V S D                  F nD E      F       n   F FE D S E L V R E G F I T H               210 <br />                 1241    OGTGATGATGAAGATGACGAAOCAAGTOCTCCACOCCCCAAGAAACGAAATTACATAGACAACTOGACCAAGACMGAACAAACTTCAGAAAAGMACGCT <br />                         G D D E D D E A S A P G A R R D D R L H R Q L D Q D L N K T S E E D A                                           245 <br />                 1346    CAAAOGTTACeCAAAGAATTAACCCAGCCTTACCCTAGACCAOCTCCAAOCAATACCGTCCTOCTCTCAAGATOGTTACGTCCCCCACAGAGGI |CTCCTACCA <br />                      Q R L A K E L R E R Y C R S S S K Q Y R A A A Q D G Y V P Q R F L L P                                              280 <br />                 1451 ACTCTTGATACACCTACCAMCGCMTGTOCOCTCCAGACCAGCTAAAGAAAOAACGTTCGATTTAAAAAAAT <br />                                                                                   CC      TCAAM ATAGGCATG <br />                      S V D T A T I V G V R C R P G K E K E L I R K L L K K K F N L D R A M                                              315 <br />                  S56W   CGTMGAAAAAACTGMAAATTTTATCCA llTTTTCCAACGTAACACACCAGTCTATATCGACCCTGACGTTATT GAAM AATTTTTMT <br />                         G K K K L K I L S I F Q R D N Y T C R I Y I E A P K Q S V I E K F C N                                           350 <br />                 1661                                                 CAAGAATTACCTCTATTACTAAAACCAACAMTCACAATCTAGATGTTGCTTTGIGIACCMCMCT <br />                         G V P D I Y I S Q K L L I P V Q E L P L L L K P N K S D D V A L E E G                                           385 <br />                 1766    AOCTACGTTCGTATTMGACACACCATCTATAAGCGTGAMTAGCTATOGTCGACCAAATTAGTGAGAATMAATAAAGTTATCCTGMMTTGTTCCTCGTCTG <br />                         S Y V R I K R G I Y K G D L A M V D Q I S E N N L E V M L K I V P R L                                           420 <br />                 1871    CATTATGGTAAATTCGACGAAATTGATCCAACAACACAOCAACCTAAATCCAGAAGACCAACTTTTGCTCATAGAOCACCACCGCAATTATTTAAMTCCAACAATG <br />                         D Y G K F D E I D P T T Q Q R K S R R P T F A H R A P P Q L F N P T M                                           455 <br />                 1976    CCTCTAACATTACACCAAGCTAACCTGTACAAAACCATCATCOCCACTTTAMTATGAACTGAAGATrATATCGATGGTTATCTGTATAAGTCM|CAGAATT <br />                         A L R L D Q A N L Y K R D D R H F T Y K N E D Y I D G Y L Y K S F R I                                           490 <br />                 2081    CAAAGTGACCAAAATATTCACCCAACTGTGGAOGAATTGCAAGCATTCGGTTCCAACAGACCOCCGGTAGATCTMCATCAGMCACAATCAATCAAG <br />                      Q H V E T K N I Q P T V E E L A R F G S K E G A V D L T S V S Q S I K                                              525 <br />                 2186 AACOCTCAACCTOCAAACCTCACM CCAOCCACGTGATCGTATCGAAGTTCTAAATGGTGAACAACGTCGTTCCAAGGCTATTGTAACAAGAAcCAcGAAAGAT <br />                      K A Q A A K V T F Q P C D R I E V L N C E Q R G S K G I V T R T T K D                                              560 <br />                 2291 ATr CTACTATCAAACTAMTGCTTACAACOCCTCTAGAAMCCTATCTCTACTCTGAGCAM TMTCGAACCTCGTGATCACGTTACTGTcATCAATGGT <br />                         I A T I K L N C F T T P L E F P I S T L R K I F E P G D H V T V I N G                                           595 <br />                 2396 CA CT A GT ACTC M M C CC CT CAT                         CAaTTAACAAATCTG <br />                      E H Q C D A G L V L M V E Q C Q V T F M S T Q T S R E V T I T A N N L                                              630 <br />                 2501 TCCAAATCCATCGACACTACACCTACATCAA CTCATATOCCTCATGCATATCGAATTGGTCTAAAAATGCTTOCTTArATTTcAAcCTCCCCACGAT <br />                      S K S I D T T A T S S E Y A L H D I V E L S A K N V A C I I Q A G H D                                              665 <br />                                                           aiII <br />                 2606    ATMCAACMGTTATTGATGAAACTOGTAAAGTGTCGCAAMTTACGAAGCTCTATMGATAAACMTAAATACTGCTCGTGCACGCGTTIIICTAGTGTCGATCCA <br />                         I F K V I D E T G K V S T I T K C S I L S K I N T A R A R V S S V D A                                           700 <br />                 2711 AATIACATGCTCAAATCGTGTACATGTACAGACATCAC OCGGAGCOTCCTCATAACATACAAACCAAAATTffTTTGMCA <br />                      N G N E I K I G D T I V E K V G S R R E C Q V L Y I Q T Q Q I F V V S                                              735 <br />                 2816 AAAGTTCGTMGAAATCTACTMCTTMGTGTCCCTATACGTGC CGGCATCC <br />                                                                   AACGACM GTTM <br />                                                                         TAT  TMGTACAAAGGATCTMGTMAA <br />                      K K I V E N A G V F V V N P S N V E A V A S K D N M L S N K M D L S K                                              770 <br />                 2921 ATGAATCCAGAAATCAMCTAAATGOGACCTCCATCATCTAAGACATTCCAACAACCCATCCAGTCCAGAGGAGGTCGTGAAGTTOCACTCGCACAACAGTA <br />                      V N P E I I S K M G P P S S K T F Q Q P I Q S R C G R E V A L G K T V                                              805 <br />                 3026 AGATCGTTCTG GCTCCTGATTCCTCGM <br />                                   CGTCTT MT                                   AGACAACACATACAATT <br />                      R I R S A G Y K C Q L G I V K D V N G D K A T V E L H S K N K H I T I                                              840 <br />                 3131 GACAACATAAGTTMCATATACAACCGTGAGGGAGTGMOTATCAGTATGTATOTMTA GAGTGTAGATAAGTCCAcAGMCAATcGGCCCA <br />                      D K H K L T Y Y N R E G G E G I T Y D E L V N R R G R V P Q A R M G P                                              875 <br />                 3236    AGTTACGTCAGTGCCCCMCAAGACATCCCCACTOCGGCTATTGCAGCAGGTCTccGCGCTACCTMCTTGTcCTTAGCGGTGGTATGACACCAGGATGGAGCTCC <br />                         S Y V S A P R N M A T G G I A A C A A A T S S G L S G G M T P G W S S                                            910 <br />                                             P-JII <br />                 3341    TTCGATGGTGGCAAACACCAGCTGTAMATGCGCATGGAGGCTCAGGTOGTCGcGGTGTCTCCTCATGGGGTGGTGCTTCCACTTGGGGTGCCAAGGTAATGGA <br />                         F D G G K T P A V N A H G G S G G G G V S S w C C A S T V C C-                              n       G N G        945 <br />                 3446    GGTGCATCCCCMGGGGCGGTGCTGGCGTCCTCCCCGTCTAGMGGCGGCCAAGccGTACTGGTGCTAMCTAMGGTcGTGGTGCTrCAGCCTGGGCTAACAAA <br />                         G A S A W C C A G G G A S A V C C Q G T G A T S T W G C A S A W C N K                                            980 <br />                                                                                                     42nI <br />                 3551 TCAAGTTGCGCGGCTGCATCCACTTGGGCGTCGGGTGGGATCMTAATGGTGCCATGTCTACTTGGGGTGGTACCGGTGATAGGTCAGCCTACGCGGGGCCTTCC <br />                      S S w C C A S T W A S C G E S N G A M S T W C T G D R S A Y C C -A S                                             1015 <br />                 3656 ACCTGGGAGGAATAACAATAATAAAATACMAAGAATGCGGCAGCTTCTGCATGGGGTAACCAAGACGATGGAATAGGTCTGCTTGCAACAACCAAGAAT <br />                      T W C Q-A N N N K S T R D C G A S A                   C     Q D D G N R S A W N N Q G N <br />                                                                                      L                                                  1050 <br />                 3761 AAGTCAAACTATGGTGGTAACAGTACATGGGGAGGTCATTAATCACCAAAAGAGACMCAATATCACCCAAG CAAG MTCAATAAAAAAACMMTATTACCA <br />                      K S N Y C C N S T W C. C H o                                                                                       1063 <br />                 3866 rCrATAATAATACTTATCMAAmATCrATCrmCrATCAAATArTCMATCTrCATCrTCCArCATC <br />  3016       SWANSON ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />               TABLE 3. SPT5 six-amino-acid repeat&apos;                                                      DISCUSSION <br />        Position                                      Sequence <br />                                                                                    Mutations SPT5 initially isolated genetic selec- <br /> 931 ............                             S S W G G                     A    tions designed identify genes required normal <br /> 937 ............                             S T W G G                     Q    transcription S. cerevisiae (19, 64). Previous genetic <br /> 948 ............                             S A W G G                     A    results suggested SPT5 gene related func- <br /> 958 ............                             S A W G G                     Q    tionally SPT6. These results include observations that <br /> 969 ............                              S T W G G                    A    spt5 spt6 mutations suppress set mutant <br /> 975 ............                              S A W G N                    K    alleles, cause double-mutant lethality haploid cells, and <br /> 981 ............                              S S W G G                    A    fail complement diploid cells (26, 63, 64). The similarity <br /> 987 ............                              S T W A S                    G    spt5 mutant phenotypes spt6 also <br /> 1000 ............                             S T W G G                    T    htal-htbl (histone) mutants (4, 12, 13, 19, 26, 45, 46, 62) led <br /> 1009 ............                             S A Y G G                    A    analyze SPT5 gene product step toward <br /> 1015 ............                             S T W G G                    N    understanding roles genes transcription. <br /> 1032 ............                             S A W G N                    Q       Our current work extends genetic similarities between <br /> 1043 ............                             S A W N N                    Q    SPT5 SPT6 genes revealed interesting <br /> 1052 ............                             S N Y G G                    N    features common SPT5 SPT6 gene products. <br /> 1058 ............                             S T W G G                    H    First, spt5 spt6 mutations cause similar alterations in <br />                                                   T                        A    transcription his4-912b lys2-1288 alleles (Fig. 1 <br />   Consensus .............S A W G G                                         Q    2; 13). Second, sptS spt6 null mutations cause <br />   No. matches ............                 15 12 13 13 11               9    lethality (13, 45). Third, increased decreased gene dosage <br />   The predicted SPT5 amino acid sequence includes 15 copies       six-   SPT5 SPT6 confers similar mutant phenotypes <br /> amino-acid repeat carboxy terminus. The serine repeat   (Fig. 4; 13). Finally, judged immunofluorescence <br /> numbered Fig. 6. Residues boldface type match consensus,     microscopy fusion proteins, SPT5 SPT6 appear <br /> was derived empirically.                                                        nuclear proteins (Fig. 7; 61). <br />                                                                                    The nucleotide sequence SPT5 gene revealed <br />                                                                                 similarities SPT6 gene. Both predicted proteins a <br />                                                                                 high concentration acidic residues amino terminus. <br /> copies repeat; SPT5(917)-1acZ encodes residues 1                      The sequence SPT5 predicted region 81 amino acids <br /> 917 SPT5 contains copies repeat (Fig. 6). To                   60% glutamic acid aspartic acid. Likewise, the <br /> determine SPT5-lacZ fusions com-                        SPT6 nucleotide sequence revealed 50% 70 <br /> plemented <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> conferred spt5 null                        amino acids glutamic acid aspartic acid (61). The <br /> mutation, constructed diploid strains containing                     acidic domains proteins contain number poten- <br /> the SPT5-lacZ fusions integrated sptSA202::LEU2                          tial sites phosphorylation casein kinase 11 (37). <br /> allele (see Materials Methods). Tetrad dissection                           One class proteins extremely acidic domains <br /> strains contained SPT5(1004)-lacZ fusion                        proteins thought interact chromatin (see refer- <br /> (MS195) SPT5(917)-lacZ fusion (MS194) resulted                        ence 15 review). One proteins, nucleolin, has <br /> 4:0 segregation viability. However, spore clones                   acidic amino terminus glycine-rich carboxy <br /> contained SPT5(917)-lacZ fusion gene extremely                         terminus (38), similar SPT5. Several transcription factors, <br /> sick, taking 5 7 days form colonies 23&#194;&#176;C [as opposed                   including yeast proteins GAL4 (39) GCN4 (30, 31), <br /> to 2 days wild type 3 days SPT5(1004)-lacZ                      acidic domains required activation of <br /> strains]. Spore clones carried SPT5-lacZ fusion                     transcription; however, acidic domains generally <br /> could grow 30 37&#194;&#176;C phenotypically Spt-.                         extensive acidic predicted the <br /> Western immunoblot analysis indicated strains carrying                      SPT5 SPT6 proteins. The human nucleolar transcription <br /> the fusions (MS194 MS195) contained approxi- <br />                                                                                 factor hUBF does contain highly acidic carboxy-terminal <br /> mately SPT5-p-galactosidase protein <br />                                                                                 region (33). <br />                                                                                    The carboxy-terminal 132 amino acids predicted <br /> (63). Therefore, repeat domain appears critical                   SPT5 protein include 15 copies six-amino-acid repeat <br /> SPT5 activity.                                                                  S-T/A-W-G-G-A/Q. Deletion SPT5 sequences encod- <br />    SPT5 encodes nuclear protein. To characterize                  ing 15 repeats impaired SPT5 function, while <br /> SPT5 protein, used indirect immunofluorescence,                           deletion 15 repeats virtually eliminated SPT5 function, <br /> described Materials Methods, determine cellu-                     judged complementation sptS mutations. Analysis of <br /> lar location SPT5-p-galactosidase hybrid protein. The                     SPT5-lacZ fusion genes encoded zero nine <br /> results (Fig. 7A C) showed immunofluorescence                       copies six-amino-acid repeat indicated the <br /> coincident DAPI staining nuclear DNA. When                          repeat domain critical SPT5 function. In analysis <br /> treated similarly, isogenic strain fusion                    requirement repeat domain, fusions p-ga- <br /> not stained (Fig. 7D F), demonstrating immuno-                       lactosidase appeared increase slightly activity of <br /> fluorescence represented SPT5-,-galactosidase hybrid                         SPT5, possibly stabilizing SPT5 portion hybrid <br /> protein. Omission anti-,-galactosidase antibody                      protein. However, result does necessarily mean that <br /> the staining procedure eliminated fluorescein fluores-                      function repeats stabilize SPT5 protein. <br /> cence. Previous work demonstrated ,-galactosidase                       Other possible roles SPT5 repeats promote <br /> expressed yeast cells located cytoplasm                     protein-protein interactions regulate SPT5 activity. <br /> and nucleus (25, 61). Therefore, observation                            Although gene sequences large data base <br /> SPT5-p-galactosidase hybrid protein contained partial                       predicted proteins significantly similar repeat se- <br /> SPT5 function localized nucleus strongly suggests                     quences, SPT5 repeat somewhat reminiscent the <br /> that wild-type SPT5 protein located nucleus.                       heptapeptide carboxy-terminal repeat largest subunit <br />  VOL . 1 l, 1991                                                                              SPT5 GENE OF S. CEREVISIAE              3017 <br />  <br />  <br />  <br />  <br />   FIG. 7. Nuclear localization SPT5-,3-galactosidase. Strain BM467, containing SPT5(1004)-p-galactosidase hybrid protein (A C), <br /> and strain BM339, control strain lacking hybrid protein (D F), stained anti-,-galactosidase antibody a <br /> fluorescein-conjugated secondary antibody described Materials Methods. Micrographs phase-contrast (A D), fluorescein <br /> fluorescence (B E), DAPI fluorescence, stains DNA (C F). Bar = 10 ,m. <br />  <br />  <br /> of RNA polymerase II (consensus = P-T-S-P-S-Y-S). Both                   Previous genetic analyses spt mutants revealed num- <br /> the SPT5 RNA polymerase repeats contain residues              ber similarities spt5, spt6, histone (htall <br /> that phosphorylated (S, T), residues known                sptll htbl/spt12) mutant strains. Mutations these <br /> disrupt secondary structure proteins (G, P),                genes suppress spectrum Ty 8 insertion <br /> aromatic residue (W, Y). The RNA polymerase repeat                 mutations cis- trans-acting mutations <br /> present eukaryotic RNA polymerases 26 (yeast) 52             greatly reduce expression SUC2 gene Ty <br /> (mouse) copies (see reference 14 review), phosphor-          elements (4, 12, 13, 19, 26, 45, 46, 62). Furthermore, histone <br /> ylated (5; reviewed reference 55), required vivo         gene pairs, like SPT5 SPT6, exhibit interesting dosage <br /> for yeast RNA polymerase function (47), function           effects: increase decrease gene copy number <br /> not known.                                                        causes Spt- phenotype (12). We that <br />   One model consistent gene dosage results           SPT5 SPT6 encode essential nuclear proteins with <br /> the SPT5 SPT6 proteins function complex              highly acidic amino termini. Given mutant phenotypes <br /> that sensitive stoichiometry components.             physical characteristics, favor model the <br /> However, strains contain SPT5 SPT6                   SPT5 SPT6 proteins act affect chromatin <br /> high-copy-number plasmids Spt- phenotype                 structure, influencing gene expression yeast cells. <br /> (data shown), suggesting              Examination chromatin structure spt mutants <br /> components putative complex. Genetic analysis indi-            begin address model directly. <br /> cates gene product, SPT4 protein, <br /> may required function putative complex.                                ACKNOWLEDGMENTS <br /> Mutations SPT4 cause phenotypes similar sptS <br /> and spt6 mutations, cause double-mutant lethality spt5             We grateful Joel Hirschhorn David Eisenmann for <br /> and spt6 mutations, fail complement spt5 spt6              helpful comments manuscript David Eisenmann for <br /> mutations (63, 64). SPT4, however, distinct             bringing attention similarity SPT5 RNA <br />                                                                       polymerase repeat domains. <br /> respects: gene essential growth, altered dosage          This work supported National Institutes Health grant <br /> does cause mutant phenotype, SPI4 protein            GM32967, National Science Foundation grant DCB8451649, a <br /> not acidic (40).                                                      grant Stroh Brewing Company F.W. M.S.S. E.M. <br />  3018      SWANSON ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br /> were supported NIH training grant Genetics Program          27. Hirschman, J. E., K. J. Durbin, F. Winston. 1988. Genetic <br /> (T32-GM07196) Lucille P. Markey Charitable Trust.                 evidence promoter competition Saccharomyces cerevi- <br />                                                                              siae. Mol. Cell. Biol. 8:4608-4615. <br />                            REFERENCES                                    28. Hoffman, C., F. Winston. 1987. A ten-minute DNA prepa- <br />                                                                              ration yeast efficiently releases autonomous plasmids for <br />   1. Andreadis, A., Y.-P. Hsu, G. B. Kohihaw, P. Schimmel.               transformation Escherichia coli. Gene 57:267-272. <br />      1982. Nucleotide sequence yeast LEU2 shows 5&apos;-noncoding          29. Holmes, D. S., M. Quigley. 1981. A rapid boiling method for <br />      region sequences cognate leucine. Cell 31:319-325.               preparation bacterial plasmids. Anal. Biochem. 114:193- <br />  2. Birnboim, H. C., J. Doly. 1979. A rapid alkaline extraction          197. <br />      procedure screening recombinant plasmid DNA. Nucleic            30. Hope, I. A., S. Mahadevan, K. Struhl. 1988. Structural and <br />      Acids Res. 7:1513-1523.                                                 functional characterization short acidic transcriptional <br />   3. Boyer, H. W., D. Roulland-Dussoix. 1969. A complementa-             activation region yeast GCN4 protein. Nature (London) <br />      tion analysis restriction modification DNA E.          333:635-640. <br />      coli. J. Mol. Biol. 41:458-472.                                     31. Hope, I. A., K. Struhl. 1986. Functional dissection a <br />  4. Brown, S. A., F. Winston. Unpublished data.                          eukaryotic transcriptional activator protein, GCN4 yeast. <br />   5. Cadena, D. L., M. E. Dahmus. 1987. Messenger RNA                    Cell 46:885-894. <br />      synthesis mammalian cells catalyzed phosphorylated     32. Ito, H., Y. Fukuda, K. Marata, A. Kimura. 1983. Transfor- <br />      form RNA polymerase II. J. Biol. Chem. 262:12468-12474.              mation intact yeast cells treated alkali cations. J. <br />  6. Carle, G. F., M. V. Olson. 1985. An electrophoretic                  Bacteriol. 153:163-168. <br />      karyotype yeast. Proc. Natl. Acad. Sci. USA 82:3756-3760.       33. Jantzen, H.-M., A. Admon, S. P. Bell, R. Tjian. 1990. <br />  7. Carlson, M., D. Botstein. 1982. Two differentially regulated         Nucleolar transcription factor hUBF contains DNA binding <br />      mRNAs different 5&apos; ends encode secreted intracellular          motif homology HMG proteins. Nature (London) 344: <br />      forms yeast invertase. Cell 28:145-154.                              830-836. <br />  8. Chaleff, D. T., G. R. Fink. 1980. Genetic events associated      34. Johnston, M., R. W. Davis. 1984. Sequences regulate <br />      insertion mutation yeast. Cell 21:227-237.                   divergent GALI-GALIO promoter Saccharaomyces cere- <br />  9. Chu, G., D. Vollrath, R. W. Davis. 1986. Separation large         visiae. Mol. Cell. Biol. 4:1440-1448. <br />      DNA molecules contour-clamped homogeneous electric               35. Kilmartin, J. V., A. E. M. Adams. 1984. Structural rear- <br />      fields. Science 234:1582-1585.                                          rangements tubulin actin cell cycle yeast <br /> 10. Church, G. M., W. Gilbert. 1984. Genomic sequencing.                 Saccharomyces. J. Cell Biol. 98:922-933. <br />      Proc. Natl. Acad. Sci. USA 81:1991-1995.                            36. Klapholz, S., R. Esposito. 1982. A new mapping method <br /> 11. Clark-Adams, C. D. 1988. Ph.D. thesis. Harvard University,               employing meiotic rec- mutant yeast. Genetics 100:387- <br />      Cambridge, Mass.                                                        412. <br /> 12. Clark-Adams, C. D., D. Norris, M. A. Osley, J. S. Fassler,       37. Kuenzel, E. A., J. A. Mulligan, J. Sommercorn, E. G. Krebs. <br />      F. Winston. 1988. Changes histone gene dosage alter tran-            1987. Substrate specificity determinants casein kinase II as <br />      scription yeast. Genes Dev. 2:150-159.                               deduced studies synthetic peptides. J. Biol. Chem. <br /> 13. Clark-Adams, C. D., F. Winston. 1987. The SPT6 gene               262:9136-9140. <br />      essential growth required 8-mediated transcription   38. Lapeyre, B., H. Bourbon, F. Amalric. 1987. Nucleolin, the <br />      Saccharomyces cerevisiae. Mol. Cell. Biol. 7:679-686.                major nucleolar protein growing eukaryotic cells: unusual <br /> 14. Cornelissen, A. W. C. A., R. Evers, J. Kock. 1988. Structure         protein structure revealed nucleotide sequence. Proc. Natl. <br />      sequence genes encoding subunits eukaryotic RNA               Acad. Sci. USA 84:1472-1476. <br />      polymerases. Oxford Surv. Eukaryotic Genes 5:91-131.                39. Ma, J., M. Ptashne. 1987. Deletion analysis GAL4 defines <br /> 15. Earnshaw, W. C. 1987. Anionic regions nuclear proteins. J.            transcriptional activating segments. Cell 48:847-853. <br />      Cell Biol. 105:1479-1482.                                           40. Malone, E. A., J. S. Fassler, F. Winston. Unpublished data. <br /> 16. Eisenmann, D. M., C. Dollard, F. Winston. 1989. SPT15,       41. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />      gene encoding yeast TATA binding factor TFIID, re-               cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />      quired normal transcription initiation vivo. Cell 58:1183-       Cold Spring Harbor, N.Y. <br />      1191.                                                               42. Meyers, A. M., A. Tzagoloff, D. M. Kinney, C. J. Lusty. <br /> 17. Eisenmann, D. M., F. Winston. Unpublished data.                      1986. Yeast shuttle integrative vectors multiple cloning <br /> 18. Farabaugh, P. J., G. R. Fink. 1980. Insertion                 sites suitable construction lacZ fusions. Gene 45:299-310. <br />      eukaryotic transposable element Tyl creates 5-base pair           43. Mortimer, R. K., D. C. Hawthorne. 1969. Yeast genetics, p. <br />      duplication. Nature (London) 286:352-356.                               385-460. In A. H. Rose J. S. Harrison (ed.), The yeasts, vol. <br /> 19. Fassler, J. S., F. Winston. 1988. Isolation analysis        1. Academic Press, Inc., New York. <br />      novel class suppressor Ty insertion mutations Saccha-      44. Mortimer, R. K., D. Schild. 1980. Genetic map Saccha- <br />      romyces cerevisiae. Genetics 118:203-212.                               romyces cerevisiae. Microbiol. Rev. 44:519-571. <br /> 20. Fassler, J. S., F. Winston. 1989. The Saccharomyces              45. Neigeborn, L., J. L. Celenza, M. Carlson. 1987. SSN20 an <br />      cerevisiae SPT131GALJJ gene positive negative              essential gene mutant alleles suppress defects SUC2 <br />      regulatory roles transcription. Mol. Cell. Biol. 9:5602-5609.        transcription Saccharomyces cerevisiae. Mol. Cell. Biol. <br /> 21. Feinberg, A. P., B. Vogelstein. 1983. A technique                7:672-678. <br />      radiolabeling restriction endonuclease fragments high specific   46. Neigeborn, L., K. Rubin, M. Carlson. 1986. Suppressors of <br />      activity. Anal. Biochem. 132:6-13.                                      snJ2 mutations restore invertase derepression cause tem- <br /> 22. Feinberg, A. P., B. Vogelstein. 1984. A technique                perature-sensitive lethality yeast. Genetics 112:741-753. <br />      radiolabeling restriction endonuclease fragments high specific   47. Nonet, M., D. Sweetser, R. Young. 1987. Functional redun- <br />      activity. Addendum. Anal. Biochem. 137:266-267.                         dancy structural polymorphism largest subunit of <br /> 23. Floy, K., P. Heiter (Johns Hopkins University Medical                RNA polymerase II. Cell 50:909-915. <br />      School). Personal communication.                                    48. Norrander, J., T. Kempe, J. Messing. 1983. Construction of <br /> 24. Gerring, S. L., C. Connelly, P. Heiter. 1990. Positional             improved M13 vectors using oligodeoxynucleotide-directed mu- <br />      mapping genes chromosome blotting chromosome                  tagenesis. Gene 26:101-106. <br />      fragmentation. Methods Enzymol. 194:57-77.                          49. Peck, L. (Harvard University). 1983. Personal communication. <br /> 25. Hall, M. N., L. Hereford, I. Herskowitz. 1984. Targeting      50. Rigby, P. W., M. Diekman, C. Rhodes, P. Berg. 1977. <br />      E. coli P-galactosidase nucleus yeast. Cell 36:1057-          Labeling deoxyribonucleic acid high specific activity vitro <br />      1065.                                                                   nick translation DNA polymerase I. J. Mol. Biol. <br /> 26. Happel, A. M., M. S. Swanson, F. Winston. Genetics,               113:237-251. <br />      press.                                                              51. Roeder, G. S., G. R. Fink. 1980. DNA rearrangements <br />  VOL. 11, 1991                                                                                     SPT5 GENE OF S. CEREVISIAE               3019 <br />  <br />       associated transposable element yeast. Cell 21:239-             charomyces cerevisiae controlled SPT genes. <br />       249.                                                                      Proc. Natl. Acad. Sci. USA 81:2431-2434. <br /> 52.   Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R.        60.   Som, T., K. A. Armstrong, F. C. Volkert, J. R. Broach. <br />       Fink. 1987. A Saccharomyces cerevisiae genomic plasmid bank               1988. Autoregulation 2,u circle gene expression provides a <br />       based centromere-containing shuttle vector. Gene 60:237-             model maintenance stable plasmid copy levels. Cell <br />       243.                                                                      52:27-37. <br /> 53.   Rothstein, R. 1983. One step gene disruption yeast. Methods      61.   Swanson, M. S., M. Carlson, F. Winston. 1990. SPT6, an <br />       Enzymol. 101:202-211.                                                     essential gene affects transcription Saccharomyces cere- <br /> 54.   Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequenc-             visiae, encodes nuclear protein extremely acidic <br />       ing chain-terminating inhibitors. Proc. Natl. Acad. Sci.             amino terminus. Mol. Cell. Biol. 10:4935-4941. <br />       USA 74:5463-5467.                                                   62.   Swanson, M. S., J. N. Hirschhorn, S. A. Brown, F. Winston. <br /> 55.   Sentenac, A. 1985. Eukaryotic RNA polymerases. Crit. Rev.                 Unpublished data. <br />       Biochem. 18:31-90.                                                  63.   Swanson, M. S., F. Winston. Unpublished data. <br /> 56.   Sherman, F., G. R. Fink, C. W. Lawrence. 1978. Methods in <br />       yeast genetics: laboratory manual, revised ed. Cold Spring        64.   Winston, F., D. T. Chaleff, B. Valent, G. R. Fink. 1984. <br />       Harbor Laboratory, Cold Spring Harbor, N.Y.                               Mutations affecting Ty-mediated expression HIS4 gene of <br /> 57.   Sikorski, R. S., P. Heiter. 1989. A shuttle vectors         Saccharomyces cerevisiae. Genetics 107:179-197. <br />       yeast host strains designed efficient manipulation       65.   Winston, F., C. Dollard, E. A. Malone, J. Clare, J. G. Kapakos, <br />       DNA Saccharomyces cerevisiae. Ge <br /> </body></html>