<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BCAS2 essential Drosophila viability functions <br /> in pre-mRNA splicing <br />  <br />  <br /> PO-HAN CHEN,1 CHIA-I LEE,1 YU-TZU WENG,1 WOAN-YUH TARN,2 YEOU-PING TSAO,3 PING-CHANG KUO,1 <br /> PANG-HUNG HSU,4,5 CHU-WEI HUANG,1 CHIUN-SHENG HUANG,6 HSIU-HSIANG LEE,7 JUNE-TAI WU,7,8 <br /> and SHOW-LI CHEN1,9 <br /> 1 <br />   Graduate Institute Microbiology, College Medicine, National Taiwan University, Taipei 100, Taiwan <br /> 2 <br />   Institute Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan <br /> 3 <br />   Department Ophthalmology, Mackay Memorial Hospital, Taipei 104, Taiwan <br /> 4 <br />   Department Life Science, College Life Sciences, National Taiwan Ocean University, Keelung 202, Taiwan <br /> 5 <br />   Institute Bioscience Biotechnology, College Life Sciences, National Taiwan Ocean University, Keelung 202, Taiwan <br /> 6 <br />   Department Surgery, College Medicine, National Taiwan University Hospital, Taipei 100, Taiwan <br /> 7 <br />   Institute Molecular Medicine, College Medicine, National Taiwan University, Taipei 100, Taiwan <br /> 8 <br />   Department Medical Research, National Taiwan University Hospital, Taipei 100, Taiwan <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Here, dBCAS2 (CG4980, human Breast Carcinoma Amplified Sequence 2 ortholog) essential viability of <br /> Drosophila melanogaster. We ubiquitous tissue-specific depletion dBCAS2 leads larval lethality, wing <br /> deformities, impaired splicing, apoptosis. More importantly, overexpression hBCAS2 rescues defects. <br /> Furthermore, C-terminal coiled-coil domain hBCAS2 binds directly CDC5L recruits hPrp19/PLRG1 form a <br /> core complex splicing mammalian cells partially restore wing damage induced knocking dBCAS2 in <br /> flies. In summary, Drosophila human BCAS2 share similar function RNA splicing, affects cell viability. <br /> Keywords: BCAS2; Drosophila; viability; splicing; Prp19 complex <br />  <br />  <br />  <br /> INTRODUCTION                                                                      stable association U5 U6 snRNP spliceo- <br />                                                                                   some, release U4 snRNP (Chan et al. 2003; Chan and <br /> BCAS2 26-kDa nuclear protein contains coiled- <br />                                                                                   Cheng 2005). Disruption yeast BCAS2 ortholog (Cwf7 <br /> coil motifs expressed ubiquitously normal human <br />                                                                                   Snt309) using genetic approaches results accumu- <br /> tissue (Kuo et al. 2009). Previously, BCAS2 <br />                                                                                   lation pre-mRNA (Chen et al. 1998, 1999; Ohi Gould <br /> interacts directly tumor suppressor p53 func- <br />                                                                                   2002; Ohi et al. 2002). In addition, Modifier snc1 4 <br /> tions negative regulator (Kuo et al. 2009). Knocking <br />                                                                                   (MOS4) Arabidopsis thaliana (the hBCAS2 ortholog in <br /> down BCAS2 wild-type p53 cancer cells, MCF7, <br />                                                                                   plants) shown affect splicing <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> the <br /> LNCaP, A549, induces programmed cell death. How- <br />                                                                                   resistance gene, snc1 rps4 plants (Xu et al. 2012). <br /> ever, depletion BCAS2 p53-null cells (H1299) and <br />                                                                                   Human Prp19, CDC5L, PLRG1 known re- <br /> p53 mutant (C33A) cells results growth arrest G2/M, <br />                                                                                   quired splicing (Ajuh et al. 2000, 2001; Grillari et al. <br /> suggesting BCAS2 plays essential role control <br />                                                                                   2005), BCAS2 essential pre-mRNA pro- <br /> of cell growth, addition regulation p53 (Kuo <br />                                                                                   cessing remains unclear. <br /> et al. 2009). <br />                                                                                      In Drosophila melanogaster, CG4980 encodes hBCAS2 <br />    BCAS2 predicted component the <br />                                                                                   ortholog, predicted component the <br /> spliceosome (Neubauer et al. 1998). The subspliceosomal <br />                                                                                   dPrp19 complex (Herold et al. 2009). Therefore, used <br /> complex, containing Prp19 (PSO4), Cef1 (CDC5L), Prp46 <br />                                                                                   fly model study examine role <br /> (PLRG1), Snt309 (BCAS2), plays critical role yeast <br />                                                                                   BCAS2 cell viability, structure&#226;&#8364;&#8220;function <br /> and human cells (Supplemental Fig. S1; Ajuh et al. 2000; <br />                                                                                   relationship pre-mRNA splicing. Our results that <br /> Ohi Gould 2002; Ohi et al. 2002, 2005; Grote et al. <br />                                                                                   dBCAS2 hBCAS2 share similar function pre- <br /> 2010). The yeast Prp19p-associated complex required for <br />                                                                                   mRNA splicing. Depletion dBCAS2 body leads <br />                                                                                   larval lethality, knockdown dBCAS2 using tissue- <br />     9 <br />     Corresponding author                                                          specific promoter results deformity phenotypes notum <br />    E-mail showlic@ntu.edu.tw <br />    Article published online ahead print. Article publication date   wings increased apoptosis wing discs, which <br /> http://www.rnajournal.org/cgi/doi/10.1261/rna.034835.112.                         rescued hBCAS2 restored RNA splicing <br />  <br />  <br /> 208                                     RNA (2013), 19:208&#226;&#8364;&#8220;218. Published Cold Spring Harbor Laboratory Press. Copyright &#194;&#169; 2013 RNA Society. <br />                                                                                        BCAS2 required fly viability splicing <br />  <br />  <br />  <br /> function. Therefore, abnormality caused depletion         TABLE 2. hBCAS2 partially rescues lethality caused Act5C- <br /> dBCAS2 is, part, consequence impaired splicing.    GAL4&#226;&#8364;&#8220;driven dBCAS2dsRNA Drosophila <br />  <br />                                                                                         Act5C-GAL4/T(2;3) <br />                                                                                           SM6-TM6B or <br /> RESULTS <br />                                                                                         UAS-dBCAS2dsRNA,            Act5C-GAL4/ <br />                                                                                         UAS-hBCAS2/T(2;3)         UAS-dBCAS2dsRNA, <br /> BCAS2 essential Drosophila viability                         Progeny                 SM6-TM6B                UAS-hBCAS2 <br />  <br /> A previous report indicated depletion hBCAS2             Predicted ratio             2             :           1 <br /> wild-type p53 cells induces apoptosis, p53 mutant        Results (no.)              620            :          91 <br /> or null cells, results G2 growth arrest (Kuo et al. 2009).    UAS-dBCAS2dsRNA, UAS-hBCAS2/T(2;3)SM6-TM6B crossed <br /> Therefore, addition regulation p53, hypothe-        Act5C-GAL4/T(2;3)SM6-TM6B, 91 Act5C-GAL4/UAS- <br /> sized BCAS2 involved basic mechanisms           dBCAS2dsRNA, UAS-hBCAS2 flies survived showed obvious <br />                                                                     defects (data shown). <br /> that maintain regular cell growth. We used D. melanogaster <br /> as animal model investigate function BCAS2 in <br /> vivo. Drosophila BCAS2 encodes 278 amino acids (&#226;&#710;&#188;31 <br /> kDa), BLAST analysis demonstrated human               (620 / 2) flies express UAS-dBCAS2dsRNA,UAS- <br /> and Drosophila BCAS2 protein orthologs 59% identical           hBCAS2. As shown Table 2, 29.4% (n = <br /> over entire sequence (Supplemental Fig. S2). We           91, 91/310) adult progeny Act5C-GAL4/UAS- <br /> used transgenic RNAi flies examine role dBCAS2 us-       dBCAS2dsRNA,UAS-hBCAS2 able survive ob- <br /> ing UAS-GAL4 system. We used Act5C-GAL4               vious defective <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Expression dBCAS2 and <br /> driver express UAS-dBCAS2dsRNA ubiquitously deter-          hBCAS2 confirmed surviving progeny RT-PCR <br /> mined dBCAS2 essential fly <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>survival</span>. As shown       (Supplemental Fig. S3). These results suggest hBCAS2 <br /> in Table 1, ratio offspring genotype Act5C-     compensate loss dBCAS2 function and <br /> GAL4/T(2;3)SM6-TM6B UAS-dBCAS2dsRNA/T(2;3)SM6-                  partially rescue lethality dBCAS2-depleted flies. <br /> TM6B compared Act5C-GAL4/UAS-dBCAS2dsRNA                     In order visible phenotypes induced knocking <br /> expected 2:1. The majority instar larvae            dBCAS2, chose tissue-specific ms1096-GAL4 <br /> Act5C-GAL4/UAS-dBCAS2dsRNA died wandering               driver expressed pouch region wing discs <br /> stage, &lt;10% expected number survived             (Milan et al. 1998) knock dBCAS2. All prog- <br /> pupae stage P1&#226;&#8364;&#8220;P2. In end, viable Act5C-GAL4/            eny carrying UAS-dBCAS2dsRNA exhibited growth defects at <br /> UAS-dBCAS2dsRNA adult flies observed, indicating              notum wings, irregular scutellum margin <br /> that dBCAS2 essential fly viability complete       (Fig. 1A, panel c) twisted shrunken wings (Fig. 1A, <br /> knockdown dBCAS2 lethal. Because human                   panel d). The apterous-GAL4 (ap-GAL4) driver ex- <br /> Drosophila BCAS2 similar (Supplemental Fig. S2),          pressed dorsal compartment instar larvae <br /> we generated hBCAS2 transgenic line investigate         wing discs (Diaz-Benjumea Cohen 1993) selected <br /> whether hBCAS2 rescue lethality. We used                  confirm UAS-dBCAS2dsRNA&#226;&#8364;&#8220;induced wing deformation. <br /> UAS-dBCAS2dsRNA,UAS-hBCAS2/T(2;3)SM6-TM6B crossed                  All flies carrying UAS-dBCAS2dsRNA (Supplemental Fig. <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> Act5C-GAL4/T(2;3)SM6-TM6B predicted 310                S4B,H,N) showed defects discoloration notum; <br />                                                                    irregular scutellum margin; abnormal number <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> of <br />                                                                    bristles, appeared similar Stubble (no Stubble mu- <br />  TABLE 1. The Act5C-GAL4         driven   UAS-dBCAS2dsRNA       tant background) (Supplemental Fig. S4H); twisted <br />  Drosophila causes lethality                                       shrunken wings (Supplemental Fig. S4N). However, flies <br />                    Act5C-GAL4/T(2;3) <br />                                                                    expressing hBCAS2 dBCAS2dsRNA control of <br />                    SM6-TM6B UAS-             Act5C-GAL4/        ms1096-GAL4 restored phenotypes recovery <br />                    dBCAS2dsRNA/T(2;3)              UAS-            scutellum margin (Fig. 1A, panel g) wing shape, <br />  Progeny               SM6-TM6B                 dBCAS2dsRNA        compared wild-type flies (Fig. 1A, panel a,b) flies <br />  Predicted ratio            2              :         1             expressing hBCAS2 (Fig. 1A, panel e,f). But wing veins <br />  Results (no.)             560             :         0             flies expressing hBCAS2 dBCAS2dsRNA par- <br />                                                                    tially recovered, anterior crossvein (ACV) longi- <br />  The genotypes parents Act5C-GAL4/T(2;3)SM6-TM6B <br />  UAS-dBCAS2dsRNA/T(2;3)SM6-TM6B. The predicted progeny         tudinal veins L3 L4 appearing absent (Fig. 1A, panel <br />  Act5C-GAL4/UAS-dBCAS2dsRNA, Act5C-GAL4/T(2;3)SM6-            h, arrow) posterior crossvein (PCV) L4 and <br />  TM6B, UAS-dBCAS2dsRNA/T(2;3)SM6-TM6B. The ratio off-       L5 looking truncated (Fig. 1A, panel h, arrowhead; Blair <br />  spring Act5C-GAL4/T(2;3)SM6-TM6B UAS- <br />  dBCAS2dsRNA/T(2;3)SM6-TM6B Act5C-GAL4/UAS-dBCAS2dsRNA          2007). But UAS-dBCAS2dsRNA,UAS-hBCAS2 ex- <br />  expected 2:1. The results absence        pressed using ap-GAL4 driver (Supplemental Fig. S4D, <br />  Act5C-GAL4/UAS-dBCAS2dsRNA genotype total number         E), observed partial rescue (Supplemental Fig. S4D, <br />  560 progeny flies. <br />                                                                    E, arrow mark) wing shape. The different rescue effects <br />  <br />  <br />                                                                                                          www.rnajournal.org      209 <br />  Chen et al. <br />  <br />  <br />  <br />                                                                         similar defective phenotypes UAS-dBCAS2dsRNA <br />                                                                         (Supplemental Fig. S4B,H,N) indicating rescue <br />                                                                         effect titration RNAi hBCAS2. <br />                                                                         In addition, efficiency dBCAS2 knockdown in <br />                                                                         UAS-dBCAS2dsRNA (Fig. 1B, left panel, lane 3) nearly <br />                                                                         UAS-dBCAS2dsRNA,UAS-hBCAS2 (Fig. 1B, left <br />                                                                         panel, lane 4), measured quantitative RT-PCR. <br />                                                                         Likewise, hBCAS2 expression UAS-dBCAS2dsRNA and <br />                                                                         UAS-dBCAS2dsRNA,UAS-hBCAS2 analyzed <br />                                                                         RT-PCR (Fig. 1B, right panel, lanes 2,4). The titration ef- <br />                                                                         fects GAL4 ruled using UAS con- <br />                                                                         structs. The dBCAS2 RNAi line #26676 VDRC a <br />                                                                         potential target hit CG18375 (dASPP), is <br />                                                                         highly expressed wing discs, mutants dASPP dem- <br />                                                                         onstrate wing overgrowth phenotype (Langton et al. 2007). <br />                                                                         To rule off-target effect, measured expression <br />                                                                         dASPP encoded transcripts (RA RB) dBCAS2- <br />                                                                         knockdown flies using quantitative RT-PCR. As shown in <br />                                                                         Supplemental Figure S5, significant change <br />                                                                         wild type, suggesting wing deformation <br />                                                                         caused dBCAS2 RNAi line #26676 off-target ef- <br />                                                                         fect expression dASPP. Taken together, find- <br />                                                                         ings indicate human Drosophila BCAS2 share a <br />                                                                         similar function human BCAS2 rescue de- <br />                                                                         fects caused depleting Drosophila BCAS2. <br />  <br />  <br />                                                                         dBCAS2 essential RNA splicing <br />                                                                         Because dBCAS2 predicted member subspli- <br />                                                                         ceosomal complex, determined defective <br />                                                                         growth dBCAS2-knockdown flies consequence im- <br /> FIGURE 1. BCAS2 essential normal wing development. (A)           paired pre-mRNA splicing. To test hypothesis, RNA ex- <br /> hBCAS2 rescues growth defects notum wings induced knocking <br /> down dBCAS2. Parental genotype (<span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> bottom): ms1096-GAL4        tracted escaper Act5C-GAL4/UAS-dBCAS2dsRNA third <br /> crossed (1) w1118, (2) UAS-dBCAS2dsRNA/T(2;3)SM6-           instar larvae analyzed quantitative RT-PCR. We chose <br /> TM6B, (3) UAS-hBCAS2, (4) UAS-dBCAS2dsRNA,UAS-hBCAS2/T               transcripts &#206;&#179;-tubulin hpo, reported (Andersen <br /> (2;3)SM6-TM6B. Scale bar, 0.5 mm. The red arrow indicates anteri- <br /> or crossvein (ACV) longitudinal veins L3 L4, ar- <br />                                                                         Tapon 2008), test dBCAS2-knockdown <br /> rowhead indicates posterior crossvein (PCV) L4 L5.      larvae exhibited abnormal pre-mRNA splicing vivo. We <br /> (B) Expression dBCAS2 hBCAS2 described above. RNA iso-       mRNA &#206;&#179;-tubulin hpo decreased by <br /> lated wing discs instar larvae, mRNA       30% 50%, compared wild type, pre- <br /> dBCAS2 measured quantitative RT-PCR (left panel). (Upper right <br /> panel) The mRNA expression hBCAS2 analyzed RT-PCR. The <br />                                                                         mRNA &#206;&#179;-tubulin hpo accumulated 2.7-fold and <br /> internal control, rp49. (Lower right) Genotypes fly.            fourfold, respectively, dBCAS2-knockdown larvae (Fig. <br />                                                                         2A, black box). This suggests lethality (Table 1) <br />                                                                         wing deformation (Fig. 1A) induced depletion of <br /> between drivers (ms1096-GAL4, ap-GAL4, Act5C-                 dBCAS2 correlate impaired splicing. We inves- <br /> GAL4) accounted strength tissue-             tigated rescue effect hBCAS2 related to <br /> specific promoter wing development.                              repair splicing efficiency. Therefore, Act5C-GAL4/UAS- <br />   Because UAS-dBCAS2dsRNA UAS-hBCAS2 driven                    dBCAS2dsRNA,UAS-hBCAS2 instar larvae subject- <br /> by set GAL4, possible activity             ed RNA splicing analysis. As shown Figure 2A (gray <br /> GAL4 activator diluted rescue effects result-          box), pre-mRNA mRNA &#206;&#179;-tubulin hpo <br /> ed decreased expression small interfering RNAs. To              recovered wild-type levels (white box). This verifies our <br /> exclude possibility, generated Drosophila ap-GAL4,              hypothesis human fly BCAS2 share similar func- <br /> UAS-mCD8-GFP/UAS-dBCAS2dsRNA, contained                       tion RNA processing, essential cell viability. <br /> UAS targeting sites, drive UAS-mCD8-GFP UAS-                     Consistent results decreased &#206;&#179;-tubulin, hpo, stg <br /> dBCAS2dsRNAsimultaneously. The fly ap-GAL4,UAS-mCD8-                    mRNA accumulated &#206;&#179;-tubulin hpo pre-mRNA <br /> GFP/UAS-dBCAS2dsRNA (Supplemental Fig. S4F,L,R) showed                  obtained Drosophila S2 cells knocking down <br />  <br />  <br /> 210      RNA, Vol. 19, No. 2 <br />                                                                                                                                                                             BCAS2 required fly viability splicing <br />  <br />  <br />  <br />      A                                                           6                                                                                         pull MCF-7 nuclear extracts analyzed putative <br />                                                                                    Act5C-GAL4                                   P&lt;0.05                     hBCAS2-interacting proteins mass spectrometry. In addi- <br />                           Relative RNA expression level (fold) <br />  <br />                                                                                    Act5C&gt; dBCAS2dsRNA <br />                                                                  5                                                                                         tion hPrp19/CDC5L complex, numer- <br />                                                                                    Act5C&gt; dBCAS2dsRNA,hBCAS2 <br />                                                                                                       P&lt;0.05                                               ous RNA processing&#226;&#8364;&#8220;related proteins, shown Table 3. <br />                                                                  4 <br />                                                                                                                                                            We performed immunoprecipitation analysis. As shown <br />                                                                  3                                                                                         Supplemental Figure S6, PLRG1 hBCAS2 <br />                                                                                                                                                            antibodies able pull endogenous CDC5L, <br />                                                                  2                                                                                         hPrp19, PLRG1 HEK (human embryonic kidney <br />                                                                              P&lt;0.05         P&lt;0.05                 P&lt;0.05 <br />  <br />  <br />                                                                  1 <br />  <br />  <br />                                                                  0 <br />                                                                                                                                                             TABLE 3. The potential hBCAS2-interacting proteins involved in <br />                                                                                                                                                             RNA splicing <br />  <br />                                                                                                                                                                                   Accession <br />                                                                                                                                                                                      no.          No.      Sequence <br />     B                                                                                                                                                       Protein          (Swissprot)    peptidesa   coverage (%) <br />  <br />                                                                                                                                                             hPrpI 9/CDC5L complex <br />                                                                           RT-PCR <br />  <br />  <br />  <br />  <br />                                                                                    dBCAS2 <br />                                                                                                                                                               CDC5L                Q99459           45          38 <br />                                                                                      rp49                                                                     hPrp19               Q9UMS4            8          14.3 <br />                                                                                                                        P&lt;0.05                                 AD-002               Q9P013            2           4.3 <br />      Relative RNA expression level (fold) <br />  <br />  <br />  <br />  <br />                                                                      4 <br />                                                                                                                                                             SR SR-related proteins <br />                                                                  3.5                                                                         mock <br />                                                                                                                                                               ASF/SF2              Q07955           12          20.6 <br />                                                                                                                                              dBCAS2dsRNA <br />                                                                      3                                                                                        SRp55 (SFRS6)        Q13247            5          10.1 <br />                                                                  2.5                         P&lt;0.05 <br />                                                                                                                                                               P54 (SFRS11)         Q6PJY9            3           1.7 <br />                                                                      2                                                                                        U2AF1                Q01081            5          16.7 <br />                                                                               P&lt;0.05                      P&lt;0.05 <br />                                                                  1.5                                                                     P&lt;0.01 <br />                                                                                                                                                               9G8 (SFRS7)          Q16629           15          26.1 <br />                                                                      1                                                                                        CDC2L5               Q9BVE2            5           8.4 <br />                                                                  0.5                                                                                          SRp20 (SFRS3)        P84103           16          31.1 <br />                                                                      0                                                                                      hnRNPs <br />                                                                          m-r-tubulin pre-r-tubulin       m-hpo        pre-hpo        m-stg                    hnRNP R              043390           25          18.8 <br />                                                                                                                                                               hnRNP U              Q00839           54          38.3 <br /> FIGURE 2. hBCAS2 plays similar role dBCAS2, necessary                                                                                           hnRNP F              P52597            3           8.7 <br /> for pre-mRNA splicing. (A) dBCAS2-knockdown larvae displays im-                                                                                               hnRNP H1             P31943           14          21.4 <br /> paired pre-mRNA splicing, hBCAS2 rescues splicing defects.                                                                                                hnRNP H3             P31942            4           9 <br /> Third instar larvae RNAs extracted Act5C-GAL4/UAS-                                                                                                  hnRNP A3             P51991            6          10.8 <br /> dBCAS2dsRNA (black box), Act5C-GAL4/UAS-dBCAS2dsRNA,UAS- <br />                                                                                                                                                               hnRNP A2/B           P22626            5          12.6 <br /> hBCAS2 (gray box), Act5C-GAL4/+ (white box). &#206;&#179;-tubulin and <br />                                                                                                                                                               hnRNP core A         P09651           19          31.2 <br /> hpo pre-mRNA mRNA analyzed quantitative RT-PCR. <br />                                                                                                                                                             snRNP-associated <br /> (B) Depletion dBCAS2 showed impaired pre-mRNA processing in <br /> S2 cells. (Upper panel) Knocking dBCAS2 dsRNA                                                                                                      U5 snRNP 100         Q9BUQ8            4            2.7 <br /> Drosophila S2 cells. (Lower panel) &#206;&#179;-tubulin, hpo, stg mRNA                                                                                           kD protein <br /> &#206;&#179;-tubulin hpo pre-mRNA analyzed quantitative RT-PCR.                                                                                              U4/U6 snRNP          043395            7            4.6 <br /> Data shown means SD relative wild-type controls                                                                                            90kD protein <br /> three independent experiments. The P-values estimated                                                                                                  (Prp3) <br /> Student&#226;&#8364;&#8482;s t-test statistically significant.                                                                                                              SF3B14               Q9Y3B4           12          44 <br />                                                                                                                                                             RNA-binding protein <br />                                                                                                                                                               RNA binding          P49756           34          21.2 <br /> dBCAS2 using dsRNA (Fig. 2B, lower panel). However, sim-                                                                                                      motif protein 25 <br />                                                                                                                                                               RNA binding          Q8IXT5            7            4.6 <br /> ilar previous reports, stg pre-mRNA level not <br />                                                                                                                                                               motif protein 12B <br /> detectable quantitative RT-PCR; suggested                                                                                                   RNA binding          Q96PK6           47          30.2 <br /> attributable low levels instability pre-mRNA                                                                                                     motif protein 14 <br /> (Andersen Tapon 2008). The decreased expression                                                                                                        RNA binding          Q14498           16          17.7 <br /> dBCAS2 RT-PCR shown upper panel. Our                                                                                                            motif protein 39 <br />                                                                                                                                                               RNA binding          Q9Y5S9            4          14.4 <br /> data confirm dBCAS2 plays role RNA splicing a <br />                                                                                                                                                               motif protein 8A <br /> component subspliceosome (Herold et al. 2009).                                                                                                       DEAD-box RNA helicase <br />                                                                                                                                                               DDX 39               B1Q2N1           20          34.4 <br />                                                                                                                                                               DDX17                Q92841           29          26.9 <br /> C-terminal hBCAS2 associates hPrp19 complex <br /> and necessary RNA splicing                                                                                                                           Mass spectrometry analysis GST-hBCAS2 pull proteins <br />                                                                                                                                                             obtained MCF-7 nuclear extracts shows putative <br /> To determine human BCAS2 bona fide com-                                                                                                        hBCAS2-interacting proteins involved RNA processing. <br />                                                                                                                                                             a <br />                                                                                                                                                               Number peptides identified protein. <br /> ponent hPrp19 complex, used GST-hBCAS2 <br />  <br />  <br />                                                                                                                                                                                                 www.rnajournal.org      211 <br />  Chen et al. <br />  <br />  <br />  <br /> epithelial cells) 293 cells (Supplemental Fig. S6, lanes 3,4),          5). On hand, His-CDC5L interacted directly with <br /> confirming hPrp19, CDC5L, PLRG1, hBCAS2                        full-length hBCAS2 C-domain protein (Fig. 3C, lanes <br /> form stable complex mammalian cells. Because human                 3,5). The vitro binding data summarized right <br /> BCAS2 contains coiled-coil motifs, mediate                panel Figure 3A. We conducted vivo binding ex- <br /> protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220;DNA interactions (Mason                  periments members endogenous hPrp19 complex <br /> Arndt 2004), C-terminal region (Fig. 3A), used                transiently expressing Flag-tagged hBCAS2, dCC1, dCC2 <br /> GST-hBCAS2, GST-&#206;&#8221;CC (deletion coiled-coil do-                    (dCC1 dCC2 plasmid constructs refer deletion <br /> mains), GST-C, His-tagged hPrp19, CDC5L proteins,                   coiled-coil domain 1 2, respectively), &#206;&#8221;CC, C- <br /> generated Escherichia coli, vitro protein&#226;&#8364;&#8220;protein           domain (Fig. 3D) constructs HEK 293 cells. The results <br /> interactions investigate association          showed full-length hBCAS2 C-domain protein <br /> the hPrp19 complex. The results showed His-hPrp19                  associated endogenous CDC5L, hPrp19, PLRG1 <br /> could bind full-length hBCAS2 &#206;&#8221;CC protein (Fig.               (Fig. 3E, lanes 8,12). However, coiled-coil deletion mu- <br /> 3B, lanes 3,4), C-domain protein (Fig. 3B, lane             tants Flag-dCC1, Flag-dCC2, Flag-&#206;&#8221;CC interacted <br />                                                                         hPrp19 CDC5L PLRG1 (lanes 9&#226;&#8364;&#8220;11). <br />                                                                         These vivo results (summarized Fig. 3D, right panel) <br />                                                                         consistent vitro binding assays (Fig. 3A&#226;&#8364;&#8220;C). <br />                                                                         Taken together, hBCAS2 N terminus interacts with <br />                                                                         hPrp19, C terminus (containing coiled-coil <br />                                                                         domain) necessary interaction major <br />                                                                         components hPrp19 complex. <br />                                                                            To decipher structure&#226;&#8364;&#8220;function relationship of <br />                                                                         hBCAS2, pSV40-CAT (In1) reporter (Fig. 4A, upper pan- <br />                                                                         el; Lin et al. 2004) used investigate function in <br />                                                                         constitutive splicing. The results showed overexpressing <br />                                                                         hBCAS2 increased pre-mRNA splicing efficiency the <br />                                                                         reporter approximately 1.45-fold, C terminus en- <br />                                                                         hanced 1.34-fold, &#206;&#8221;CC protein inhibited <br />                                                                         splicing 0.69-fold (Fig. 4A, lower panel). The correspond- <br />                                                                         ing hBCAS2 protein expression shown Figure 4B. In ad- <br />                                                                         dition, used adenovirus E1A reporter (Fig. <br />                                                                         4C, upper panel) assay alternative splicing regulation of <br />                                                                         hBCAS2. Selection 5&#226;&#8364;&#178; splice sites E1A produces var- <br />                                                                         ious amounts 13s, 12s, 9s mRNAs modulated <br />                                                                         variety splicing regulatory factors (Lin et al. 2004). The <br />                                                                         results showed hBCAS2 C terminus increased the <br />                                                                         ratio 9s mRNA, decreased 12s mRNA, the <br />                                                                         E1A reporter (Fig. 4C,D). In general, splicing factor <br />                                                                         activates distal 5&#226;&#8364;&#178; splice site (i.e., 9s site), level of <br />                                                                         9s mRNA increased concomitant reduction of <br />                                                                         12s 13s mRNAs (Lai et al. 2003; Lin et al. 2004). <br />                                                                         However, hBCAS2 appeared effect 13s <br />                                                                         mRNA, reduced 12s promoted 9s expres- <br />                                                                         sion, mechanism underlying clear. <br />                                                                         Southern blotting performed confirm RT- <br />                                                                         PCR products shown Figure 4A C (Supplemental <br /> FIGURE 3. The C-terminal coiled-coil region hBCAS2 associates        Fig. S7A,B). To confirm effect hBCAS2 alternative <br /> with hPrp19 complex. (A) Schematic representation GST-tagged     splicing endogenously, chose alternative transcripts <br /> hBCAS2, &#206;&#8221;CC, C constructs summarized data. (B) Full-length      associated cell cycle control, depletion <br /> and &#206;&#8221;CC hBCAS2 interacts directly hPrp19. (C) Full-length   hBCAS2 causing G2 growth arrest (Kuo et al. 2009). The <br /> C-terminal hBCAS2 interacts directly CDC5L. (D) Schematic rep- <br /> resentation Flag-tagged hBCAS2, dCC1, dCC2, &#206;&#8221;CC, C con-          human Cdc25 phosphatase isoforms Cdc25B Cdc25C <br /> structs. (Right) Summarized data vivo interactions E. (E)    critical transition G2 M phase <br /> Mapping interaction domain hBCAS2 CDC5L, PLRG1,             (Boutros et al. 2007). Because data Figure 2B (lower <br /> hPrp19 vivo. 1/20 input analyzed Western blot shown in <br />                                                                         panel) showed mRNA level stg (<span id='am-2' about='xsd:string' typeof='owl:Thing'>String</span>, <br /> the left panel (lanes 1&#226;&#8364;&#8220;6). Proteins precipitated Flag-tagged <br /> hBCAS2 variants revealed blotting using indicated antibodies    Drosophila Cdc25 ortholog) dramatically decreased in <br /> (right panel, lanes 7&#226;&#8364;&#8220;12).                                              dBCAS2-knockdown S2 cells, determined RT-PCR <br />  <br />  <br /> 212      RNA, Vol. 19, No. 2 <br />                                                                                                 BCAS2 required fly viability splicing <br />  <br />  <br />  <br />                                                                                                      hBCAS2 affected splicing <br />                                                                                                      variants Cdc25C (Pacheco et al. <br />                                                                                                      2006; Albert et al. 2011). Our data re- <br />                                                                                                      vealed knocking hBCAS2 <br />                                                                                                      changed ratio Cdc25C3/4 C1 <br />                                                                                                      variants, compared scramble <br />                                                                                                      control, MCF7 cells (Fig. 4E, left panel, <br />                                                                                                      lane 2; Supplemental Fig. S7C). The right <br />                                                                                                      panel shows hBCAS2 efficiently <br />                                                                                                      depleted transient transfection of <br />                                                                                                      pSuper-shBCAS2-434 (Fig. 4E, right pan- <br />                                                                                                      el, lane 2) mRNA protein <br />                                                                                                      levels. This verifies role hBCAS2 <br />                                                                                                      alternative splicing. In summary, the <br />                                                                                                      hBCAS2 C terminus containing the <br />                                                                                                      coiled-coil motifs critical RNA splic- <br />                                                                                                      ing association hPrp19 <br />                                                                                                      complex. <br />  <br />                                                                                                      The C terminus hBCAS2 partially <br />                                                                                                      restores defective phenotypes <br />                                                                                                      dBCAS2-depleted flies <br />                                                                                                      Because C terminus indispensable <br />                                                                                                      function hBCAS2 RNA splic- <br />                                                                                                      ing (Fig. 5A), asked question <br />                                                                                                      C-fragment hBCAS2 res- <br />                                                                                                      cues defects dBCAS2-depleted flies <br />                                                                                                      way full-length hBCAS2. <br />                                                                                                      We generated hBCAS2-C transgenic <br />                                                                                                      fly showed obvious phenotypic <br />                                                                                                      change compared wild type <br />                                                                                                      (data shown). When UAS-hBCAS2- <br />                                                                                                      C UAS-dBCAS2dsRNA expressed <br />                                                                                                      ms1096-GAL4 driver, wing <br />                                                                                                      deformation partially rescued, in <br />                                                                                                      shrunken wrinkled wings <br />                                                                                                      partially unfolded the <br />                                                                                                      wing veins clearly observed, <br />                                                                                                      compared dBCAS2-knockdown <br />                                                                                                      fly (Fig. 5B, panels b,d); however, the <br /> FIGURE 4. The C terminus indispensable function hBCAS2 splicing. (A) C-ter- <br /> minal full-length hBCAS2 enhances CAT(In1) reporter pre-mRNA splicing. (Upper pan-           notum exhibited similar pheno- <br /> el) Schematic representation CAT(In1) reporter. The splicing efficiency (lower panel)     type (Fig. 5B, cf. panels c). <br /> calculated value mRNA/(mRNA + pre-mRNA), relative splicing efficiency            Expression hBCAS2-C mRNA the <br /> was normalized vector control (set 1). Data shown means SD in-       wing discs ms1096-GAL4&#226;&#8364;&#8220;driven <br /> dependent experiments. (&#226;&#710;&#8212; P &lt; 0.05) (B) Western blot analysis anti-Flag antibody deter- <br /> mine ectopically expressed protein reporter HEK 293 cells. (C) C-terminal            UAS-dBCAS2dsRNA UAS-hBCAS2-C <br /> full-length hBCAS2 modulates 5&#226;&#8364;&#178; splice site selection AdE1A reporter. Schematic represen-     flies confirmed RT-PCR (Fig. <br /> tation AdE1A reporter shown upper panel. (Lower panel) AdE1A reporter           5C, lane 2). This indicates the <br /> cotransfected control vector Flag-hBCAS2 variants HEK 293 cells analyzed     hBCAS2-C fragment partially rescue <br /> by RT-PCR determine 13s, 12s, 9s RNAs. (D) Quantitative results alternative splicing. C <br /> showed represented experiment. The data D collected independent exper-        wing deformation dBCAS2-deplet- <br /> iments shown means SD. P values estimated Student&#226;&#8364;&#8482;s t-tests          ed flies. To confirm partial rescue <br /> statistically significant. (E) Depletion hBCAS2 influences alternative splicing patterns   effect hBCAS2-C fragment cor- <br /> Cdc25C variants. The Cdc25C alternative splicing variants examined RT-PCR in <br />                                                                                                      related splicing function, ex- <br /> BCAS2-depleted MCF7 cells. (Left) Knocking BCAS2 (lane 2) alters ratio of <br /> Cdc25C3/4 Cdc25C1 variant compared scramble control (lane 1). (Right) Protein            amined splicing efficiency in <br /> mRNA expression BCAS2 internal control revealed Western blot RT-PCR.               instar larvae expressing UAS- <br />  <br />  <br />                                                                                                                     www.rnajournal.org       213 <br />  Chen et al. <br />  <br />  <br />  <br />                                                                           ment appears potent full-length protein re- <br />                                                                           storing fly viability. Taken together, rescue effect with <br />                                                                           hBCAS2-C weaker full-length protein, <br />                                                                           hBCAS2-C appears potent full-length protein <br />                                                                           mammalian cell splicing assays. The differences of <br />                                                                           BCAS2 function fly mammalian cells interest- <br />                                                                           ing subject investigation. <br />  <br />  <br />                                                                           dBCAS2 knocked cells higher <br />                                                                           degree apoptosis <br />                                                                           Previous studies shown depletion splicing factors <br />                                                                           dMAFP1 dPrp38 affects Drosophila cell prolifer- <br />                                                                           ation induces apoptosis (Andersen Tapon 2008). We <br />                                                                           depletion BCAS2 resulted splicing de- <br />                                                                           fects wing deformation. Other observation of <br />                                                                           splicing defect, notum, wing phenotypes, exam- <br />                                                                           ined cell viability wing discs instar lar- <br />                                                                           vae knocking dBCAS2. The engrailed-GAL4 was <br />                                                                           used driver located posterior wing discs <br />                                                                           UAS-dBCAS2dsRNA (Fig. 6B), UAS-dBCAS2dsRNA,UAS- <br />                                                                           hBCAS2 (Fig. 6C), UAS-dBCAS2dsRNA,UAS-hBCAS2-C <br />                                                                           (Fig. 6D) expressed. As shown Figure 6, knocking <br />                                                                           dBCAS2 significantly increased number apopto- <br />                                                                           tic cells, revealed staining cleaved caspase 3 (Fig. 6F, red <br />                                                                           spot), compared wild type (Fig. 6E). However, co- <br />                                                                           expressing hBCAS2 (Fig. 6G) hBCAS2-C (Fig. 6H) could <br />                                                                           fully partially reduce cleaved caspase 3&#226;&#8364;&#8220;positive cells in <br />                                                                           dBCAS2-knockdown wing discs. Because human BCAS2 <br />                                                                           reported negative regulator tumor sup- <br />                                                                           pressor p53 cancer cell lines (Kuo et al. 2009), possible <br /> FIGURE 5. C-terminal hBCAS2 partially rescues dBCAS2-knockdown            apoptosis phenomenon dBCAS2-knockdown <br /> induced wing splicing defects. (A) C-terminal hBCAS2 plays core     wing discs results direct effects dBCAS2 the <br /> of hPrp19 complex RNA splicing. (B) Ectopic expression of <br /> hBCAS2-C partially rescues dBCAS2dsRNA-induced wing deformation.          Drosophila p53 (dmp53), splicing defects. To <br /> UAS-hBCAS2-C UAS-dBCAS2dsRNA expressed con-            rule effect dmp53, observed caspase 3 activity <br /> trol ms1096-GAL4. Scale bar, 0.5 mm. (C) Expression hBCAS2-C        knocking dBCAS2 ms1096-GAL4 dmp53- <br /> in ms1096-GAL4/+;UAS-dBCAS2dsRNA/+;UAS-hBCAS2-C/+. RNAs                   null mutant flies (Sogame et al. 2003). As shown in <br /> were extracted wing discs instar larvae subjected to <br /> RT-PCR confirm expression hBCAS2-C (lane 2) compared            Supplemental Figure S8C, cleaved caspase 3 staining in <br /> with wild-type control (lane 1). The internal control, rp49. (D) C-   dBCAS2-depleted flies (Supplemental Fig. S8B) still <br /> terminal hBCAS2 partially rescues impaired splicing dBCAS2-deplet-     higher control (Supplemental Fig. S8A) the <br /> ed instar larvae. Act5C-GAL4/UAS-dBCAS2dsRNA;UAS-hBCAS2-            dmp53-null mutant background. In summary, results <br /> C/+ (gray box) instar larvae RNA extracted analyzed <br /> by quantitative RT-PCR described legend Figure 2.            provide evidence function BCAS2 RNA splicing <br />                                                                           essential Drosophila cell viability. <br />  <br /> dBCAS2dsRNA UAS-hBCAS2-C, driven Act5C-GAL4. <br />                                                                           DISCUSSION <br /> As shown Figure 5D, C-terminal hBCAS2 (gray box) <br /> partially rescued level mRNA pre-mRNA &#206;&#179;-tu- <br />                                                                           BCAS2 essential Drosophila viability <br /> bulin hpo, compared UAS-dBCAS2dsRNA alone. <br />                                                                           RNA splicing <br /> On hand, examined level viability <br /> restored hBCAS2-C fragment Act5C-GAL4/                      In study, showed dBCAS2 essential for <br /> dBCAS2dsRNA. We screened 241 adult flies, contrast                 Drosophila viability shRNA-mediated depletion the <br /> with 29.4% restored survival rate full-length hBCAS2,              body (Table 1). In addition, conditional knocking <br /> the progeny expressing dBCAS2dsRNA hBCAS2-C                       dBCAS2 using ms1096-GAL4, ap-GAL4, en- <br /> not survived (data shown). This indicates C-frag-            grailed-GAL4 drivers results damaged phenotypes and <br />  <br />  <br /> 214       RNA, Vol. 19, No. 2 <br />                                                                                                BCAS2 required fly viability splicing <br />  <br />  <br />  <br />                                                                                                     provide evidence hBCAS2 partici- <br />                                                                                                     pates pre-mRNA splicing (Fig. 4). <br />                                                                                                     Furthermore, conservation of <br />                                                                                                     human fly BCAS2, deplet- <br />                                                                                                     ing Drosophila BCAS2 instar lar- <br />                                                                                                     vae S2 cells caused decreased <br />                                                                                                     processing mRNA accumulation <br />                                                                                                     pre-mRNA (Fig. 2), overexpress- <br />                                                                                                     ing hBCAS2 dBCAS2-depleted flies <br />                                                                                                     restore defects RNA processing <br />                                                                                                     (Fig. 2), phenotypes in <br />                                                                                                     wings lethality (Fig. 1; Table 2). <br />                                                                                                     Previous reports illustrated that <br />                                                                                                     depletion spliceosomal factors, such <br />                                                                                                     ASF dMFAP1, cause G2/M arrest <br />                                                                                                     (Bernstein Coughlin 1998; Li et al. <br />                                                                                                     2005; Andersen Tapon 2008). <br />                                                                                                     Similarly, members hPrp19 <br />                                                                                                     complex, hPrp19 PLRG1, are <br />                                                                                                     required RNA splicing, caused <br />                                                                                                     lethality early stage embryo de- <br />                                                                                                     velopment knocked mice <br />                                                                                                     (Fortschegger et al. 2007; Kleinridders <br />                                                                                                     et al. 2009). These reports support our <br />                                                                                                     data impairment RNA splicing <br />                                                                                                     BCAS2 knocked p53- <br />                                                                                                     null cells causes lethality malforma- <br /> FIGURE 6. dBCAS2 knock cells display higher level apoptosis. Apoptotic cells vi- <br />                                                                                                     tions dBCAS2-depleted flies. <br /> sualized cleaved caspase 3 staining dBCAS2-knockdown wing discs, expression of <br /> hBCAS2 hBCAS2-C reduced cleaved caspase 3&#226;&#8364;&#8220;positive staining. engrailed-GAL4 drives GFP <br /> (green, panels A&#226;&#8364;&#8220;D) locating posterior (to right discs) region wing discs where <br /> UAS-dBCAS2dsRNA (panels B,F,J), UAS-dBCAS2dsRNA,UAS-hBCAS2 (panels C,G,K ), UAS-                Structure-function relationship <br /> dBCAS2dsRNA,UAS-hBCAS2-C (panels D,H,L) expressed compared wild type (panels           BCAS2 <br /> A,E,I ) wandering instar larvae. Anti-cleaved caspase 3 staining (red, panels E&#226;&#8364;&#8220;H) re- <br /> vealed apoptotic cells imaginal wing discs. Merged images (panels I&#226;&#8364;&#8220;L). The genotypes   As reported previously (Grote et al. <br /> flies described Materials Methods.                                                 2010), amino acid sequences of <br />                                                                                                   hBCAS2 counterpart Snt309 <br />                                                                                                   (Saccharomyces cerevisiae-simpler yeast), <br /> increased apoptotic cells wings imaginal discs, respec-               lacks coiled-coil domains, share 24% similarity. <br /> tively (Figs. 1, 6; Supplemental Fig. S4; Table 1). These de-                Although Snt309 component Prp19 complex, <br /> formed phenotypes lethality rescued human                     does interact directly Cef1 (CDC5L ortholog) <br /> BCAS2 (Figs. 1, 6; Supplemental Fig. S4; Table 2), indicating                (Ohi et al. 2005). Therefore, deduce data <br /> that hBCAS2 dBCAS2 share similar functions affect                   C-terminal coiled-coil domains hBCAS2 contrib- <br /> cell viability. Previously, hBCAS2 negative               ute primarily interaction CDC5L. Previous re- <br /> regulator p53 direct interaction. Depletion                    ports shown depletion Snt309 protein by <br /> hBCAS2 increases p53 protein activity induces p53-tar-                   anti-Snt309 antibodies does impair pre-mRNA splicing <br /> geted gene expression, resulting apoptosis. (Kuo et al.                   vitro (Chen et al. 1998, 1999). Unlike Snt309, human <br /> 2009). However, knocking human BCAS2 p53-                            BCAS2 enhances efficiency pre-mRNA splicing <br /> null mutant cells induces G2/M arrest, suggesting                   (Fig. 4). Thus, coiled-coil domains hBCAS2 critical <br /> the function BCAS2 growth control involve                        splicing function direct interaction with <br /> mechanism regulation p53. BCAS2 (also                      CDC5L. <br /> named Spf27) reported component mul-                         Moreover, Cwf7 (BCAS2 counterpart) more <br /> ti-protein spliceosome complex mass spectrometry                      complex yeast, Schizosaccharomyces pombe, contains sequenc- <br /> EST-database searching, role RNA splicing remains                 es similar C-terminal domain human BCAS2. Cwf7 <br /> unclear (Neubauer et al. 1998). Here, confirmed hu-                  interact CDC5L ortholog rescue the <br /> man BCAS2 integral component hPrp19 subspli-                    growth defects snt309-deficient cells (Ohi Gould <br /> ceosomal complex (Supplemental Fig. S6; Table 3),                     2002; Ohi et al. 2002), suggesting BCAS2 ortholog <br />  <br />  <br />                                                                                                                  www.rnajournal.org      215 <br />  Chen et al. <br />  <br />  <br />  <br /> has evolved achieve functional complexity. This possibility        capacity core complex formation responsible regular <br /> is supported high similarity Drosophila, plant         splicing mammalian cells. But Drosophila system, <br /> (Palma et al. 2007), human BCAS2, contain               flies transgenic full-length hBCAS2 (Fig. 1A, panel e) <br /> coiled-coil domains interact directly CDC5L.            hBCAS2-C (data shown) effects terms <br /> In addition, Drosophila Protein Interaction Mapping               changes morphology, endogenous <br /> (DPiM) project identified putative dBCAS2-interact-               splicing complexes reaching saturation. However, rescue <br /> ing proteins, including CG6905 (CDC5L) Tango4                     effects measured crossing transgenic full-length or <br /> (PLRG1), similar GST-hBCAS2 pull pro-               hBCAS2-C flies dBCAS2-depleted flies, restora- <br /> teins (Table 3). Combined data (Figs. 3&#226;&#8364;&#8220;5), re-        tion wing-deformation hBCAS2-C (Fig. 5B, panel d) is <br /> ports verify structural functional conservation               weaker full-length hBCAS2 (Fig. 1A, panel <br /> between dBCAS2 hBCAS2.                                            h). The rescue effect hBCAS2-C potent full- <br />    In study, identified novel function consti-      length hBCAS2, suggesting roles for <br /> tutive splicing factor human BCAS2 5&#226;&#8364;&#178; splice site selec-       BCAS2 wing shape development, RNA splicing. <br /> tion adenovirus E1A reporter (Fig. 4C). We provide         Aside RNA splicing, BCAS2 reported to <br /> preliminary evidence depletion hBCAS2 affects             interact estrogen receptor (ER) transcriptional co- <br /> ratio Cdc25C alternative transcripts (Fig. 4E), suggesting         factor (Qi et al. 2005) p53 negative regulator <br /> that BCAS2 potential functions regulation alter-        (Kuo et al. 2009). ER known growth survivor and <br /> native splicing. Alternative splicing important post-           p53 tumor suppressor, proteins crucial <br /> transcriptional regulation diversifies gene expression           cell viability. The detailed roles BCAS2 regulating the <br /> in higher eukaryotes. Two crucial factors control alternative         genes involved cell growth development are <br /> splicing: sequence position cis-acting elements            investigated. Moreover, N fragment hBCAS2 in- <br /> and local concentration post-translational status             teracts directly hPrp19 dominant- <br /> of trans-acting factors. In general, trans-acting factors        negative role RNA splicing (Fig. 4A,C) mammalian <br /> regulate alternative splicing include SR (Ser-Arg) proteins           cells. It interesting investigate whether <br /> and heterogeneous nuclear ribonucleoproteins (hnRNPs)                 hBCAS2-N fragment binds hPrp19, resulting in <br /> particles (Chen Manley 2009). Recent studies showed               dilution core complex components impeding <br /> that hPrp19 interacts directly Blom7&#206;&#177;, novel hnRNP             RNA splicing fly assay. <br /> K homology domain-containing protein modulate <br /> the alternative 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; splice site choices reporter mini- <br /> genes (Grillari et al. 2009). In addition, CDC5L/PLRG1 inter-         MATERIALS AND METHODS <br /> acts hnRNP M affects selection alternative <br /> splice site (Lleres et al. 2010). It interesting inves- <br />                                                                       Fly strains <br /> tigate BCAS2 regulates alternative splicing af- <br /> fecting interaction POS4-Blom7&#206;&#177; CDC5L/PLRG1-                Drosophila stocks kept crossed 25&#194;&#176;C supplied with <br /> hnRNPM. As BCAS2 harbors coiled-coil domains                 cornmeal medium. UAS-dBCAS2dsRNA purchased Vienna <br /> and does belong families SR proteins                 Drosophila RNAi Center (stock no. 26676) balanced T(2;3) <br /> hnRNPs, exclude possibility              SM6-TM6B, translocated chromosome 2&#226;&#8364;&#8220;3 balancer. UAS- <br />                                                                       hBCAS2 UAS-hBCAS2-C constructed insertion of <br /> activity hBCAS2-C protein alternative splicing and <br />                                                                       hBCAS2 hBCAS2-C (278 bp, amino acids 139&#226;&#8364;&#8220;225) cDNA <br /> in restoration knockdown phenotype involve                 HeLa cells pUAST vectors randomly inserted into <br /> its interaction splicing factors (as provided           second chromosome w1118, respectively (kindly <br /> our GST-hBCAS2 pull list, Table 3), known            provided Taiwan Fly Core). UAS-dBCAS2dsRNA,UAS- <br /> factors Prp19 complex. However, possible           hBCAS2 established recombination UAS- <br /> that BCAS2 affects overall kinetics splicing result    dBCAS2dsRNA UAS-hBCAS2 balanced T(2;3)SM6- <br /> the reduction increase general splicing factor.               TM6B. UAS-dBCAS2dsRNA UAS-hBCAS2-C balanced <br />    A previous report concluded hPrp19 core com-              T(2;3)SM6-TM6B chromosome II III, respectively, to <br /> plex, including hBCAS2 (lack N-terminal amino acids),          generate UAS-dBCAS2dsRNA;UAS-hBCAS2-C/T (2;3) SM6-TM6B; <br /> the N-terminal domain hPrp19, C-terminal domain                therefore, UAS-dBCAS2dsRNA UAS-hBCAS2-C cosegregate dur- <br /> of CDC5L, N-terminal domain PLRG1, forms                 ing crosses. UAS-dBCAS2dsRNA,UAS-hBCAS2, UAS-hBCAS2-C <br />                                                                       expressing flies distinguished selectable marker, <br /> salt stable bounded mixture resists high concentration <br />                                                                       Tubby. The dmp53&#226;&#710;&#8217;ns strain kind gift Dr. John <br /> of protease subtilisin (Grote et al. 2010). Consistently, <br />                                                                       M. Abrams (Sogame et al. 2003). ap-GAL4, ms1096-GAL4, Act5C- <br /> our results demonstrate C terminus hBCAS2 (con-           GAL4, engrailed-GAL4, w1118 used study obtained <br /> taining coiled-coil domains) capacity interact         Bloomington Drosophila stock center maintained the <br /> directly CDC5L, turn binds hPrp19              Fly Core Facility College Medicine, National Taiwan <br /> PLRG1 (Fig. 3) carry RNA splicing activities, compa-           University. Phenotypes flies examined 48 h post eclose ster- <br /> rable length hBCAS2 (Figs. 3, 4). This indicates     eomicroscope (Leica) anesthetized carbon dioxide. <br />  <br />  <br /> 216      RNA, Vol. 19, No. 2 <br />                                                                                             BCAS2 required fly viability splicing <br />  <br />  <br />  <br /> Wings isolated adult flies mounted Hoyer&#226;&#8364;&#8482;s mount-          Immunohistochemistry <br /> ing medium imaging. To validate expression UAS- <br /> hBCAS2 UAS-dBCAS2 driven ms1096-GAL4, total RNAs                   Wing imaginal discs dissected wandering instar lar- <br /> were extracted wing discs instar larvae analyzed        vae of following genotypes: (1) engrailed-GAL4,UAS- <br /> by RT-PCR.                                                                GFP/+, (2) engrailed-GAL4,UAS-GFP/UAS-dBCAS2dsRNA, (3) en- <br />                                                                           grailed-GAL4,UAS-GFP/UAS-dBCAS2dsRNA,UAS-hBCAS2, (4) <br />                                                                           engrailed-GAL4,UAS-GFP/UAS-dBCAS2dsRNA;UAS-hBCAS2-C/+. <br /> Cell culture plasmid constructs                                       The discs fixed 20 min PBS 4% paraformal- <br />                                                                           dehyde blocked PBS 0.3% Triton X 5% BSA <br /> HEK 293 MCF7 (breast cancer cells) grown Dulbecco&#226;&#8364;&#8482;s           (bovine serum albumin) 30 min room temperature incu- <br /> modified Eagle&#226;&#8364;&#8482;s medium RPMI 1640 medium, respectively,               bated anti-cleaved caspase 3 antibody (1:600; Abcam) over- <br /> supplied 10% FBS, L-glutamine, penicillin/streptomy-             night 4&#194;&#176;C. After washing PBS-Triton X times, they <br /> cin 37&#194;&#176;C, 5% CO2. To knock dBCAS2 S2 cells, full-              incubated Cy3-conjugated goat anti-rabbit IgG (1:1000, <br /> length dBCAS2 cDNA Drosophila S2 cells used                  Jackson ImmunoResearch Lab) 1 h room temperature. The <br /> dsRNA synthesis vitro transcription (Promega). dBCAS2               fluorescent image acquired fluorescence microscope <br /> dsRNA incubated S2 cells 5 d RNA analysis ac-         (Zeiss AXIO SCOPE A1). <br /> cording protocol reported previously (Rogers Rogers <br /> 2008). GST-tagged Flag-tagged hBCAS2 variants con- <br /> structed according method described previously (Kuo et al.         Mass spectrometry analysis hBCAS2-associated <br /> 2009). Primers used study listed Supplemental          protein <br /> Information. <br />                                                                           Identification putative hBCAS2-interacting proteins per- <br />                                                                           formed vitro recombinant GST-hBCAS2 pull nucle- <br /> Protein&#226;&#8364;&#8220;protein interaction antibody                                  ar extracts MCF7 breast cancer cells eluted SDS-PAGE <br />                                                                           separation in-gel digestion. The digested peptides an- <br /> 6&#195;&#8212;His-CDC5L, hPrp19, GST fusion protein (hBCAS2 variants)             alyzed high-resolution high-mass accuracy nanoflow LC- <br /> were prepared E. coli BL21 (DE3) vitro interaction. GST-      MS/MS (Liquid Chromatography&#226;&#8364;&#8220;Tandem Mass Spectrometry, <br /> fusion protein conjugated beads washed sonication buffer        Thermo Fisher Scientific). <br /> (50 mM pH 8.0 NaH2PO4, 300 mM NaCl, 20% glycerol, DTT, <br /> PMSF) twice co-IP buffer (20 mM Tris pH 7.5, 150 <br /> mM NaCl, 10% glycerol, 0.1% NP-40, 5 mM MgCl2, 1 mM EDTA, <br /> and 1 mM DTT) incubated eluted His-tagged pro- <br />                                                                           SUPPLEMENTAL MATERIAL <br /> teins. After incubation, samples washed eluted       Supplemental material available article. <br /> Western blot analysis. HEK 293 cells transfected Flag-hBCAS2 <br /> variants calcium phosphate subjected vivo immuno- <br /> precipitation (Kuo et al. 2009): anti-hBCAS2, hPrp19, PLRG1               ACKNOWLEDGMENTS <br /> (Bethyl Laboratories); anti-CDC5L (BD, ABcam); anti-GST, Flag <br /> M2, Actin (Sigma).                                                        This work supported National Science Council (NSC99- <br />                                                                           3112-B-002-037, NSC99-2628-B-002-023-MY3), National Taiwan <br />                                                                           University (98R0066-13; 99R311001, 98F008-201), National <br /> In vivo splicing assays                                                   Taiwan University Hospital (UN101-066). We thank Dr. Shu- <br />                                                                           Chun Teng critical discussion appreciate technical sup- <br /> For detection mRNA pre-mRNA, RNA extracted                port staff Sixth Core Lab DNA Sequencing Facility, <br /> three instar larvae following genotypes: (1)        Department Medical Research, NTUH. We thank Dr. John <br /> Act5C-GAL4/+, (2) Act5C-GAL4/UAS-dBCAS2dsRNA, (3) Act5C-                  M. Abrams kind gifts. We thank Taiwan Fly Core <br /> GAL4/UAS-dBCAS2dsRNA,UAS-hBCAS2, (4) Act5C-GAL4/                      Facility Fly Core Facility NTU microinjection main- <br /> UAS-dBCAS2dsRNA;UAS-hBCAS2-C/+. Two micrograms RNA                     taining fly strains. We thank Dr. Tim J. Harrison editing our <br /> was treated DNase (RQ1, Promega) reverse transcrip-           English. <br /> tion (SuperScript III, Invitrogen) analyzed quantitative <br /> PCR (KAPA SYBR FAST qPCR, Kapa Biosystems) according               Received June 10, 2012; accepted November 14, 2012. <br /> manufacturer&#226;&#8364;&#8482;s instructions. Data collected technical <br /> triplication reaction independent samples. <br /> Primers detection mRNA pre-mRNA &#206;&#179;-tubulin <br />                                                                           REFERENCES <br /> and hpo based previous reports (Andersen Tapon <br /> 2008). Details vivo constitutive alternative splicing assays   Ajuh P, Kuster B, Panov K, Zomerdijk JC, Mann M, Lamond AI. 2000. <br /> were described previous reports (Lin et al. 2004). For alternative        Functional analysis human CDC5L complex identifica- <br /> splicing endogenous Cdc25C, MCF7 cells transfected              tion components mass spectrometry. EMBO J 19: 6569&#226;&#8364;&#8220; <br />                                                                              6581. <br /> either pSuper-scramble pSuper-shBCAS2-434 electroporation           Ajuh P, Sleeman J, Chusainow J, Lamond AI. 2001. A direct interaction <br /> and incubated 48&#226;&#8364;&#8220;72 h Western blot RT-PCR analy-              carboxyl-terminal region CDC5L WD40 <br /> sis. Primers used following previous reports (Pacheco et al.            domain PLRG1 essential pre-mRNA splicing. J Biol Chem <br /> 2006; Albert et al. 2011).                                                   276: 42370&#226;&#8364;&#8220;42381. <br />  <br />  <br />                                                                                                                 www.rnajournal.org        217 <br />  Chen et al. <br />  <br />  <br />  <br /> Albert H, Santos S, Battaglia E, Brito M, Monteiro C, Bagrel D. 2011.       Kuo PC, Tsao YP, Chang HW, Chen PH, Huang CW, Lin ST, Weng YT, <br />     Differential expression CDC25 phosphatases splice variants hu-        Tsai TC, Shieh SY, Chen SL. 2009. Breast cancer amplified sequence <br />     man breast cancer cells. Clin Chem Lab Med 49: 1707&#226;&#8364;&#8220;1714.                   2, novel negative regulator p53 tumor suppressor. Cancer Res <br /> Andersen DS, Tapon N. 2008. Drosophila MFAP1 required                    69: 8877&#226;&#8364;&#8220;8885. <br />     pre-mRNA processing G2/M progression. J Biol Chem 283:              Lai MC, Kuo HW, Chang WC, Tarn WY. 2003. A novel splicing regu- <br />     31256&#226;&#8364;&#8220;31267.                                                                lator shares nuclear import pathway SR proteins. EMBO J <br /> Bernstein HS, Coughlin SR. 1998. A mammalian homolog fission                 22: 1359&#226;&#8364;&#8220;1369. <br />     yeast Cdc5 regulates G2 progression mitotic entry. J Biol           Lan <br /> </body></html>