<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 8 March 2012                                                Nucleic Acids Research, 2012, Vol. 40, No. 12 5679&#226;&#8364;&#8220;5693 <br />                                                                                                                  doi:10.1093/nar/gks223 <br />  <br />  <br /> Air2p critical assembly RNA-binding <br /> of TRAMP complex KOW domain of <br /> Mtr4p crucial exosome activation <br /> Peter Holub, Jana Lalakova, Hana Cerna, Josef Pasulka, Marie Sarazova, <br /> Kristyna Hrazdilova, Maria Sanudo Arce, Fruzsina Hobor, Richard Stefl* and <br /> Stepanka Vanacova* <br /> CEITEC&#226;&#8364;&#8221;Central European Institute Technology, Masaryk University, Kamenice 5, CZ-62500 Brno, <br /> Czech Republic <br />  <br /> Received November 16, 2011; Revised Accepted February 21, 2012 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        regulation gene expression chromatin structure <br />                                                                                 [reviewed (1)]. Most, all, RNAs synthesized <br /> Trf4/5p-Air1/2p-Mtr4p polyadenylation complex                                   precursor molecule needs posttran <br /> (TRAMP) essential component nuclear                                    scriptionally processed and/or modi&#239;&#172;?ed order form <br /> RNA surveillance yeast. It recognizes variety                              mature functional molecules. Several human genetic dis- <br /> of nuclear transcripts produced RNA                                orders relate abnormalities RNA processing and <br /> polymerases, adds short poly(A) tails aberrant                               proper ribonucleoprotein assembly, cancer or <br /> or unstable RNAs activates exosome                                  neurodegenerative diseases (2). In yeast, nuclear RNA <br /> their degradation. Despite advances under-                               maturation stability strict control <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> standing structural features isolated                                RNA surveillance carried nuclear exosome <br /> complex subunits fragments, details                             cofactor TRAMP4/5 polyadenylation <br /> complex assembly, RNA recognition exosome                                   complex (3&#226;&#8364;&#8220;6). <br />                                                                                    The TRAMP recognizes variety nuclear transcripts <br /> activation remain poorly understood. Here we <br />                                                                                 produced RNA polymerases, precursors <br /> provide understanding RNA binding                              small nuclear nucleolar RNAs (snRNAs and <br /> mode complex. We Air2p                                   snoRNAs), ribosomal RNAs (rRNAs), tRNAs, as <br /> RNA-binding subunit TRAMP. We identify                                   telomeric, cryptic unstable transcripts (CUTs) (6&#226;&#8364;&#8220;8). It <br /> zinc knuckles (ZnK) 2, 3 4 RNA-binding                               adds short poly(A) tails aberrant transcripts, creating a <br /> domains, reveal essentiality ZnK4                                favorable substrate exosome (3&#226;&#8364;&#8220;6,9), reviewed (10). <br /> TRAMP4 polyadenylation activity. Furthermore,                                Among complex subunits, Mtr4p provides helicase <br /> identify Air2p key component TRAMP4                                   activity Trf4p Trf5p provide polyA-polymerase <br /> assembly providing bridging Mtr4p                                   (PAP) activity, Air2p Air1p zinc-knuckle (ZnK) <br /> Trf4p. The bound N-terminus                                proteins predicted RNA-binding role within <br /> Air2p, bound ZnK5, linker                           TRAMPs. The degradation presumably involves multiple <br />                                                                                 polyadenylation cycles helicase activity Mtr4p <br /> between ZnK4 5 C-terminus the <br />                                                                                 (5). In instances, RNA degradation does not <br /> protein. Finally, uncover RNA binding                               require polyadenylation activity TRAMP (7,11,12). <br /> of Mtr4p arch, KOW domain, essential                             The mechanistic understanding substrate recognition <br /> component          TRAMP-mediated      exosome                              exosome activation currently limited. <br /> activation.                                                                        The minimal TRAMP PAP consists heterodimer of <br />                                                                                 Trf4 Trf5 proteins Air1p Air2p <br />                                                                                 (4&#226;&#8364;&#8220;6). Trf4p Trf5p catalytic subunits the <br /> INTRODUCTION                                                                    complex, Air1p Air2p predicted <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Eukaryotic cells produce diverse types protein-coding                        provide RNA-binding. Air proteins contain &#239;&#172;?ve <br /> as noncoding RNAs,                              tandemly arranged ZnK motifs CCHC-type <br /> recently shown direct regulatory role                            (CX2CX4HX4C) previously shown to <br />  <br />  <br /> *To correspondence addressed. Tel: +420 549495042; Fax: +420 549492556; Email: vanacova@chemi.muni.cz <br /> Correspondence addressed Richard Ste&#239;&#172;&#8218;. Tel: +420 549492436; Fax: +420 549492556; Email: richard.ste&#239;&#172;&#8218;@ceitec.muni.cz <br />  <br /> &#195;&#376; The Author(s) 2012. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  5680 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br /> bind nucleic acids proteins unrelated viral proteins     MATERIALS AND METHODS <br /> (13). The ZnK regions proteins recognize exposed       Yeast cultures manipulation <br /> guanosines RNA loops single-stranded RNA (13). <br /> The RNA helicase Mtr4p required                  Yeast cultured standard conditions media <br /> polyadenylation activity TRAMP; however, regulates        selective markers corresponding particular strains <br /> the <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> poly(A) tail added Trf4p-Air2p dimer         vectors. Yeast transformations performed by <br /> and modulates polyadenylation rate vitro (14).           lithium acetate method. For growth test analyses, yeast <br />    Recently, works reported structure       cultured overnight minimal SD medium 30  C, <br /> full-length Mtr4p heterodimer Trf4p-Air2p frag-        diluted fresh SD media OD600 0.2 grown at <br /> ments (15&#226;&#8364;&#8220;17). The structures Mtr4p revealed               30  C OD600 1. The cells serially diluted in <br /> &#239;&#172;?ve-domain organization domains constitute        decimal dilutions incubated 17  C, 25  C, 30  C and <br /> the DExH helicase core [RecA-1, RecA-2, winged helix            37  C SD media (or indicated experiment). <br /> (WH) helical bundle] (16,17). Compared known           The yeast strains used work listed in <br /> structures DNA RNA helicases, Mtr4p DExH           Supplementary Table S2. <br /> core N-terminal b-hairpin extension located the <br /> surface globular core. The unique feature           Puri&#239;&#172;?cation protein complexes yeast <br /> however insertion 265 amino acids the <br /> WH helices protruding globular core form            The yeast culture grown 30  C YPD medium to <br /> the arch domain arm-like structure &#239;&#172;?st            OD600  2.5. Cells harvested, frozen liquid <br /> (also termed KOW domain) end. The KOW                nitrogen manually grinded. Lysate melted in <br /> domain shares similarities rRNA-binding proteins           D100 buffer [100 mM KCl, 50 mM Tris, 10% (w/v) <br /> and contributes RNA binding vitro (17). The arch       glycerol, 0.02% (v/v) Nonidet P&#226;&#8364;&#8220;40 (NP&#226;&#8364;&#8220;40), 0.2 mM <br /> dispensable Mtr4 helicase ATPase activities     EDTA, pH 7.9] protease inhibitors, soluble fraction <br /> TRAMP integrity vitro. A strain expressing archless       recovered centrifugation bound to <br /> Mtr4p displays slow growth <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>            IgG-sepharose beads 3 h 4  C. Beads washed <br /> result Rrp6p-dependent incomplete processing            thoroughly D150 (same D100 150 mM KCl) <br /> of 5.8S rRNA (16).                                              D1000 (high-salt wash, D100 1 M KCl) <br />    The study structure minimal Trf4p-Air2p        buffer, proteins eluted TEV protease <br /> heterodimer, covering catalytic central domains      cleavage D150 buffer. Elutions snap frozen and <br /> Trf4p ZnKs Air2p, revealed           kept  80  C. Alternatively, elutions merged and <br />                                                                 bound o/n Ni-NTA agarose beads. The beads were <br /> phylogenetically conserved surface central <br />                                                                 washed D150 buffer 10 mM imidazole and <br /> domain Trf4p contacts linker ZnK <br />                                                                 eluted 250 mM imidazole D50 buffer. Elutions <br /> region Air2p (15). Nevertheless, study left <br />                                                                 snap frozen kept  80  C. <br /> many open questions role the <br />                                                                    All immunoprecipitated samples subsequently <br /> other Air2p ZnK domains N- C- terminal <br />                                                                 subjected western blot analysis prepared the <br /> regions Air2p Trf4p. Finally, RNA-binding <br />                                                                 presence 100 mg ml 1 RNase A (added cell <br /> mode Trf4p-Air2p addressed indirectly. <br />                                                                 lysate). Samples used enzymatic assays <br /> The structural determination entire complex has <br />                                                                 (polyadenylation, exosome degradation) prepared <br /> proven dif&#239;&#172;?cult far instability of <br />                                                                 absence RNase order avoid interference <br /> recombinant subunits. Moreover, remains a <br />                                                                 residual RNase enzymatic analysis. <br /> mystery TRAMP interacts activates <br /> the downstream acting exosome. Thus despite the <br /> advances understanding TRAMP function                 RNA isolation analysis <br /> in vivo individual structures,   For RNA analysis, cells grown 30  C o/n liquid <br /> limited knowledge assembly complex         SD media, diluted OD600 0.25 grown 25  C to <br /> the mechanistic details action.                          OD600 1. RNA isolated hot phenol extraction <br />    Here, focused aspects TRAMP4           stored  80  C. Five micrograms total RNA was <br /> biochemistry. On hand, aimed identify <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>        denatured 25% formamide, separated 8% <br /> of recognition structurally complicated substrates      denaturing (8 M urea) polyacrylamide gel transferred <br /> need polyadenylated TRAMP4 exosome              nylon membrane using semi-dry electro-transfer. RNA <br /> degradation, hypomodi&#239;&#172;?ed tRNAiMet. Second,          cross-linked membrane UV light <br /> to tackle TRAMP4 complex assembly molecular             (120 mJ cm 2) hybridized probe Ultra-Hyb <br /> mechanisms function, performed            buffer (Ambion) according manufacturer&#226;&#8364;&#8482;s instructions. <br /> analyses Trf4p, Air2p Mtr4p domain organization <br /> in vitro vivo. Furthermore, succeeded obtain <br /> the solution structure &#239;&#172;?ve zinc-knuckle region        Construction yeast plasmids <br /> Air2p NMR spectroscopy. Overall, uncovered                Yeast genomic DNA WT full-length TRF4, AIR2 and <br /> regions required cell viability, protein&#226;&#8364;&#8220;protein inter-     MTR4 inserted pET22b (5) used PCR-amplify <br /> actions, RNA binding domains regions responsible            untranslated coding regions TRF4, AIR2 and <br /> for exosome activation.                                         MTR4 genes. <br />                                                                        Nucleic Acids Research, 2012, Vol. 40, No. 12 5681 <br />  <br />  <br /> TRF4 constructs. To obtain plasmids containing                 MTR4 constructs. Full-length MTR4 deletion <br /> truncated forms TRF4 gene, desired region           variants cloned pNOPPATA1L vector. <br /> was ampli&#239;&#172;?ed standard PCR reaction using primers          C-terminal His6 tag added PCR ampli&#239;&#172;?cation <br /> listed Supplementary Table S3 inserted             extended reverse primer (MTR4 His SalI). The <br /> pNOPPATA1L vector restriction sites NdeI               full-length MTR4 ampli&#239;&#172;?ed genomic DNA <br /> SalI. To obtain point mutated forms, standard PCR            (primers MTR4 NdeI MTR4 His SalI); MTR4 <br /> mediated site directed mutagenesis pNOPPATA1L-            archless ampli&#239;&#172;?ed pAV674 vector (16), which <br /> TRF4 (5) template performed. All constructs           kind gift Dr Van Hoof.  KOW MTR4 <br /> contained N-terminal fusion proteinA tag TEV             version prepared ligation PCR products <br /> protease recognition sequence. With exception           pNOPPATA1L vector (using NdeI/SalI restriction <br /> C-terminally truncated forms (that TRF4 C 544,              sites). N-terminal (ampli&#239;&#172;?ed primers MTR4 <br /> C 499, C 463 182&#226;&#8364;&#8220;463), constructs                 KOW N) MTR4 (aminoacids 1&#226;&#8364;&#8220;665) extended at <br /> contained fusion C-terminal hexahistidine anchor. The          30 -end coding sequence Gly-Ser-Gly- <br /> His6 tag used analyses observed          Ser-Ala-Ser C-terminal MTR4 (aminoacids <br /> no sign interference results. To express en-   819&#226;&#8364;&#8220;1073, obtained oligos MTR4 KOW C) was <br /> dogenous levels Trf4p, genomic region spanning      extended 50 -end coding sequence Ala-Ser-Gly- <br /> 500 nt upstream TRF4 initiation codon               Ser-Gly-Ser resulting GSGSASGSGS bridge (con- <br /> ampli&#239;&#172;?ed using primers TRF4 506 50 -UTR XbaI For               nected NheI restriction site Ala-Ser coding <br /> and TRF4 506 50 -UTR ATG SphI XhoI Rev                     sequence). <br /> inserted pRS413 vector XbaI XhoI re- <br /> striction sites. Into construct, TRF4 full-length     Preparation unmodi&#239;&#172;?ed tRNAiMet enzymatic and <br /> gene and/or alleles D425A, DXD and/or N 96                NMR binding experiments <br /> pNOPPATTA1L constructs including fusion tags               The plasmid ptRNAiMet vitro transcription of <br /> were inserted SphI site. For Escherichia coli ex-      yeast tRNAiMet (5) linearized BstNI. Unlabeled <br /> pression, TRF4 coding sequence inserted       hypomodi&#239;&#172;?ed tRNAiMet produced vitro T7 poly- <br /> expression vector pET30a(+) allowing N-terminal        merase run transcription (19) using linearized plasmid <br /> fusion G-protein B1 tag (18) using BamHI XhoI           template. The product puri&#239;&#172;?ed anion exchange <br /> restriction sites.                                             HPLC denaturing conditions desalted a <br />                                                                NAP column (Sephadex G-25). The hypomodi&#239;&#172;?ed <br /> AIR2 constructs. Yeast genomic DNA WT AIR2 gene             tRNAiMet denatured 95  C, pH 6.2 5 min and <br /> inserted pET22b (5) used PCR-amplify un-        snap cooled ice, favor monomeric conformation. <br /> translated coding regions AIR2. To prepare AIR2         The concentration RNA determined by <br /> constructs phenotypic analyses vivo, coding         UV-spectroscopy. <br /> sequence AIR2 surrounded 425-bp upstream of <br /> initiation codon 479-bp downstream stop codon           Fluorescence anisotropy <br /> was PCR ampli&#239;&#172;?ed Saccharomyces cerevisiae                 The equilibrium binding Air2p ZnK1-5 (aminoacids <br /> genomic DNA (BY4741 background) primers AIR2              57&#226;&#8364;&#8220;180) different oligonucleotides analyzed by <br /> 50 -UTR_For1 AIR2 30 -UTR_Rev1. The fragment           &#239;&#172;&#8218;uorescence anisotropy. The 15-mer poly-A RNA and <br /> inserted pRS415 vector PstI NotI restric-     DNA oligonucleotides 50 -labeled N,N,N0 ,N0 - <br /> tion sites resulting plasmid V379. To prepare tagged    tetramethyl-6-carboxyrhodamine (TAMRA) was <br /> versions AIR2 af&#239;&#172;?nity puri&#239;&#172;?cations TRAMP             attached hexyl linker. The measurements con- <br /> complexes, full-length CDS AIR2 subcloned           ducted FluoroMax-4 spectro&#239;&#172;&#8218;uorometer (Horiba <br /> to yeast expression vector [pNOPPATA1L, (5)]             Jobin-Yvon, USA). The instrument equipped a <br /> NdeI SalI restriction enzymes, resulting            thermostatted cell holder Neslab RTE7 water bath <br /> plasmid V53. Single point mutations AIR2               (Thermo Scienti&#239;&#172;?c, USA). The operated using <br /> prepared WT AIR2 pNOPPATA1L                       FluorEssence software (version 2.5.3.0, Horiba <br /> template site-directed mutagenesis using oligo-          Jobin-Yvon, USA). The TAMRA &#239;&#172;&#8218;uorophore was <br /> nucleotides listed Supplementary Table S3. AIR2 trun-       excited 561 nm, emission collected at <br /> cation mutants constructed PCR ampli&#239;&#172;?cation            581 nm. The widths excitation emission <br /> with speci&#239;&#172;?c primers listed Supplementary Table S2.         monochromatic slits 8 nm integration time was <br /> The PCR product inserted pNOPPATA1L                   set 3 s. During measurement 10 nM labeled oligo- <br /> vector using NdeI SalI restriction sites. To generate      nucleotide (volume 1.4 ml) titrated increasing <br /> mutants pRS415 derived constructs, &#239;&#172;?rst introduced       amounts protein 50 mM Tris buffer (pH 7.9), <br /> NdeI restriction site position 3-bp upstream         supplemented 250 mM NaCl, 20 mM b-mercap <br /> AIR2 start codon resulting construct V394 WT            toethanol 50 mM ZnSO4. The titrations carried <br /> (with primers AIR2 NdeI ATG For AIR2 NdeI                  20  C stirred 1.5 ml quartz cuvette. A &#239;&#172;?xed delay <br /> ATG       Rev).     Individual    mutants    prepared       30 s set aliquot addition start of <br /> pNOPPATA1L transferred NdeI                  measurement allow reaction reach equilib- <br /> BamHI sites generate constructs listed               rium. This delay suf&#239;&#172;?cient, change in <br /> Supplementary Table S4.                                        anisotropy observable. Every data point average <br />  5682 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br /> of measurements. The data analyzed              E.     coli    strain  BL21-CodonPlus        (DE3)-RIPL <br /> SigmaPlot 11 software (Systat Software, USA). The ex-         (Stratagene). The expression induced 0.4 mM <br /> perimental isotherms &#239;&#172;?t single-site binding         IPTG 37  C 4 h presence 0.5 mM ZnSO4. <br /> model according Heyduk Lee (20) using nonlinear        Cells lysed lysis buffer (50 mM carbonate <br /> least squares regression. The data normalized        buffer, 400 mM NaCl, 25% sacharose, 2 mM MgCl2 <br /> visualization purposes.                                       10% glycerol, 400 mM ZnSO4, pH 10.5), proteins <br />                                                               puri&#239;&#172;?ed buffer containing 50 mM carbonate <br /> Preparation DNA probes northern blot analysis          buffer, pH 10.5, 400 mM NaCl, 10% glycerol, 400 mM <br /> DNA probes NEL025c snR33 ampliEed           ZnSO4 using af&#239;&#172;?nity Ni-NTA chromatography followed <br /> S. cerevisiae BY4741 genomic DNA primers com-        gel &#239;&#172;?ltration. <br /> binations (NEL025c For Rev, snR33   mature + <br /> extension For Rev, Supplementary Table S5) and <br /> used templates Random Primed DNA Labeling Kit          Expression preparation Air2 ZnK protein for <br /> (Roche) according manufacturer&#226;&#8364;&#8482;s instructions. To          NMR studies <br /> detect U14 snR13 30 -extended snoRNAs                 The coding sequence Air2 protein comprising residues <br /> tRNAiMet, oligonucleotides 50 -phosphate labeled     57&#226;&#8364;&#8220;180 S. cerevisiae expressed the <br /> using T4 polynucleotide kinase (NEB) [g-32P]-ATP.         N-terminal fusion Smt3 protein (21). The expression was <br />                                                               induced 100 mM IPTG E. coli BL21-Codon Plus <br /> Whole-cell protein extract preparation western blot       (DE3)-RIPL (Stratagene) strain overnight 16  C M9 <br /> analysis                                                      minimal medium, supplemented 50 mM ZnSO4. For <br /> The yeast culture grown 30  C selective medium      isotope labeling, medium supplemented with <br />                                                               15 <br /> to OD600  1.0. The cells resuspended D150            NH4Cl [U-13C6] glucose. Cells harvested, resus- <br /> buffer protease inhibitors (0.5 mg ml 1 leupeptin,    pended denaturing buffer (50 mM Tris, 500 mM NaCl, <br /> 0.8 mg ml 1 pepstatin A, 0.6 mM PMSF 0.6 mM            8 M Urea, pH 8.0) lysed sonication, proteins <br /> DTT). The cell suspension vortexed glass beads       puri&#239;&#172;?ed using af&#239;&#172;?nity Ni-NTA chromatography. The <br /> at 4  C 10 min. The beads cell debris        protein immobilized beads extensively washed and <br /> pelleted centrifugation, cell-free extract   subsequently refolded buffer containing 50 mM Tris pH <br /> was mixed 1 SDS loading buffer western blot          7.9, 500 mM NaCl, 20 mM b-mercaptoethanol, 5 mM imid- <br /> analysis. The rest extract used protein        azole, 50 mM ZnSO4, protease inhibitors. Refolded <br /> concentration measurement method Bradford.          protein eluted dialyzed buffer containing <br />                                                               50 mM Tris pH 7.9, 500 mM NaCl, 20 mM b-mercap <br /> Antibodies                                                    toethanol 50 mM ZnSO4. The Smt3 fusion protein <br />                                                               cleaved Ulp1p protease removed Ni-NTA <br /> The generation speci&#239;&#172;?c antibodies Trf4p, Air2p, <br />                                                               column. For NMR measurements protein was <br /> Air1p described (5,7). Mtr4p antibodies were <br />                                                               concentrated using Vivaspin 20 concentrator (Sartorius). <br /> kindly provided Dr Patrick Linder (University of <br /> Geneva). The anti-proteinA antibodies anti-His6 <br /> antibodies Sigma. All antibodies used a <br /> 1:5000 dilution.                                              NMR spectroscopy structure determination <br />                                                               The protein used NMR spectroscopy concentrated <br /> Expression puri&#239;&#172;?cation recombinant proteins             150 mM. Due high salt buffer [50 mM sodium <br /> Full-length Trf4p S. cerevisiae recombinantly        phosphate       buffer,    500 mM      NaCl,      10 mM <br /> expressed pET30a vector N-terminally fused          b-mercaptoethanol pH 8.0 (90% H2O/10% D2O)], 3 mm <br /> G-proteinB1 solubility tag (18) E. coli strain             shaped NMR tubes used. All NMR experiments <br /> BL21-CodonPlus (DE3)-RIPL (Stratagene). The expres-           recorded 20  C Bruker AVANCE 600 MHz <br /> sion induced 0.5 mM IPTG 37  C 2 h       900 MHz spectrometers equipped cryogenic <br /> presence 1 mM MgCl2. Cells lysed Lysis Buffer      triple resonance probe (Bruker BioSpin). The chemical <br /> (50 mM carbonate buffer, 300 mM NaCl, 5% glycerol,            shifts backbone side-chains assigned using <br /> 5 mM imidazole, 0.1% NP-40, 1 mM MgCl2, 2 mM                  standard triple resonance experiments (22,23). All <br /> b-mercaptoethanol protease inhibitors, pH 10.5)        distance restraints derived 3D 15N- and <br />                                                               13 <br /> sonication, protein puri&#239;&#172;?ed 50 mM Tris buffer,        C-edited NOESYs (with mixing time 80 ms) collected <br /> pH 8.0 containing 300 mM NaCl, 5% glycerol, 0.02%             900 MHz spectrometer. The structure calculations were <br /> NP-40, 1 mM MgCl2, 1 mM b-mercaptoethanol using               performed automated NOE assignment module <br /> af&#239;&#172;?nity Ni-NTA chromatography. For NMR studies,               implemented CYANA program (24). Initial struc- <br /> the GB1-Trf4 fusion protein dialyzed buffer        tures re&#239;&#172;?ned CYANA-generated <br /> containing 50 mM Tris, pH 8.0, 300 mM NaCl, 3%                restraints restraints tetrahedral <br /> glycerol 1 mM MgCl2 subsequently concentrated         zinc coordination AMBER 10.0 software (25) as <br /> up 230 mM VIVASPIN column (Sartorius).                  described previously (26,27). From 40 re&#239;&#172;?ned structures, <br />    Wild-type mutant forms AIR2                    20 conformers lowest AMBER energy were <br /> recombinantly expressed pET22b vector             selected form &#239;&#172;?nal ensemble structures. <br />                                                                        Nucleic Acids Research, 2012, Vol. 40, No. 12 5683 <br />  <br />  <br /> Enzymatic assays                                               Furthermore, observed chemical shift perturbation in <br />                                                                (L3) fourth linker (L4) regions Trf4p <br /> Polyadenylation assays carried 15 ml reaction <br />                                                                binding. It concerned mainly residues 117&#226;&#8364;&#8220;120 (L3) and <br /> mixtures containing 5&#226;&#8364;&#8220;50 ng af&#239;&#172;?nity puri&#239;&#172;?ed complex, <br />                                                                residues S138, I139, W140, R141 Y143 (L4), of <br /> 50 fmol 50 -end labeled RNA, 0.5 mM ATP, 5 mM MgCl2, <br />                                                                highly conserved Air homologs from <br /> 25 mM Tris, pH 7.9, 20 mM KCl, 10% glycerol, <br />                                                                distant species (Supplementary Figure S1A). The analysis <br /> 0.1 mg ml 1 BSA, 1 mM DTT, 0.02% Nonidet P-40, 5U <br />                                                                additional mutants regions revealed that <br /> of RNasin Plus (Promega) addition of <br />                                                                double mutants ZnKs ZnK4+5 in <br /> 50 ng af&#239;&#172;?nity puri&#239;&#172;?ed Rrp6-TAP eluate (5). Reaction <br />                                                                fourth linker (W140G+R141E), respectively, exhibited <br /> mixture incubated 30  C, 5 ml aliquotes col- <br />                                                                poor &#239;&#172;?tness comparable WT strain tempera- <br /> lected times indicated. The reaction stopped the <br />                                                                tures tested (Figure 1B). The growth defects observed at <br /> addition 25 mM EDTA proteins removed incu- <br />                                                                24  C correlated de&#239;&#172;?ciencies TRAMP function <br /> bation ProteinaseK (Promega) 10 min 37  C. <br />                                                                vivo northern blot analyses revealed strong accu- <br /> Sample mixed volume formamide loading <br />                                                                mulation NEL025c CUT snR33 pre-snoRNA in <br /> buffer resolved denaturing polyacrylamide gel. <br />                                                                ZnK4+5, L4, C 180 ZnK1-5L mutants cultivated <br />                                                                24  C (Figure 1C). Cold sensitivity certain <br /> Database searches sequence analysis                        extent RNA degradation defects seen 24  C were <br /> BLAST searches conducted using NCBI genome         observed ZnK point mutants <br /> databases (http://www.ncbi.nlm.nih.gov/BLAST/)        indicating ZnKs contribute degree to <br /> nonredundant EST databases Swiss EMBnet node         TRAMP activity vivo (Figure 1B C). <br /> (28). The multiple sequence alignment performed            To address vivo phenotypes re&#239;&#172;&#8218;ected <br /> using ClustalW2 (29) http://www.ebi.ac.uk/Tools/        defects TRAMP assembly catalytic activity, we <br /> services/web_clustalw2 interface. The Boxshade server          isolated analyzed yeast complexes containing indi- <br /> (http://www.ch.embnet.org/software/BOX_form.html)          vidual mutated Air2p versions. In agreement our <br /> used color similarity alignment.                 NMR studies, L4 ZnK5 mutants failed ef&#239;&#172;?ciently <br />                                                                copurify Trf4p Mtr4p, point muta- <br />                                                                tions did affect TRAMP4 assembly (Figure 1D). <br /> RESULTS                                                        Western blot analysis input fractions individual <br /> Identi&#239;&#172;?cation Air2p regions required intermolecular     Air2p mutants revealed comparable levels three <br /> contacts Trf4p Mtr4p                                  TRAMP subunits WT, ZnK L4 mutant strains. <br />                                                                Thus lack Trf4p Mtr4p copuri&#239;&#172;?cation L4 <br /> Trf4/5p, unlike canonical PAPs, possess identi&#239;&#172;?able         ZnK5 caused decreased stability or <br /> RNA-binding domain inactive              expression mutant strains. As absence of <br /> own. We previously demonstrated Air2p acti-          Mtr4p does disrupt interaction Trf4p <br /> vates Trf4p, likely RNA-binding properties         (Figure 1D, HSW) (4,5), conclude fourth <br /> (5). To identify Air2p regions contact RNA,               linker zinc knuckle regions Air2p crucial <br /> interact Trf4p Mtr4p, required yeast       binding Trf4p. The lack Trf4p Mtr4p <br /> viability, prepared Air2p mutants carrying point       copuri&#239;&#172;?cation ZnK5 samples re&#239;&#172;&#8218;ected the <br /> mutations second zinc-coordinating cysteine     lack PAP activity puri&#239;&#172;?ed protein sample <br /> of &#239;&#172;?ve ZnKs (ZnK1 ZnK5) disrupt ZnK fold         (Figure 2A). Interestingly, ZnK4 mutant was <br /> (Figure 1A Supplementary Figure S1A). In order          able maintain TRAMP integrity (Figure 1D), was <br /> tackle importance surrounding N-                strongly defective polyadenylation vitro (Figure 2A). <br /> C-termini, generated deletion mutants lacking        Accordingly, recombinant Air2 proteins mutated ZnK4 <br /> the N- C- terminus (Figure 1A). Mutants             ZnK5 failed activate yeast Trf4p vitro (Figure 2B). <br /> ZnK4, ZnK5, C 180 ZnK1-5L able                    We previously reported puri&#239;&#172;?ed TRAMP4 <br /> rescue growth defect air1Dair2  strain higher        complex activates exosomes vitro (5). In order test <br /> temperature mutants &#239;&#172;?rst ZnKs             mutants affect exosome <br /> N 57 showed reduced &#239;&#172;?tness (Figure 1B). The growth             activation, performed coupled polyadenylation/ <br /> defects observed 30  C caused reduced ex-       exosome assays. In assay, 50 -end labeled <br /> pression stability Air2p variants TRAMP       hypomodi&#239;&#172;?ed tRNAiMet (a well-de&#239;&#172;?ned vivo target of <br /> subunits, western blot analysis protein extracts     pathway) incubated presence WT or <br /> prepared cells grown 30  C revealed comparable         mutant TRAMPs yeast nuclear exosome puri&#239;&#172;?ed via <br /> protein levels mutants tested (Supplementary           Rrp6p subunit (5). Figure 2C shows Rrp6p-TAP <br /> Figure S1B S1C). This pointed importance         puri&#239;&#172;?ed fractions used subsequent degrad- <br /> the zinc knuckles Air2p C-terminus. Our       ations assays work compared exosome obtained <br /> previous yeast two-hybrid results indicated minimal        core subunit Rrp4p. The coupled polyadenylation/ <br /> region Air1 Air2 proteins required Trf4p inter-     degradation assays revealed the <br /> actions positioned zinc knuckles ZnK3 ZnK5       polyadenylation-competent Air2p point mutants further <br /> (5). Accordingly, NMR titration experiments,             affected exosome activation vitro (Figure 2D). <br /> observed fourth &#239;&#172;?fth ZnKs Air2p               The analysis deletion mutants indicated import- <br /> associated recombinant Trf4p (data shown).            ance &#239;&#172;?rst 25 N-terminal aminoacids the <br />  5684 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br />  <br />  <br /> Figure 1. Air2p N- C-termini ZnK4 ZnK5 domains important TRAMP functions integrity. (A) Overview mutant forms <br /> of AIR2 used study: ZnK, zinc knuckle. The aminoacid substitutions individual point mutants designated brackets. (B) Growth test <br /> analysis WT mutant forms Air2p episomally expressed air1Dair2  strain, serially diluted SD-Leu media incubated 3 days at <br /> temperatures indicated. (C) Northern blot analysis total RNA isolated AIR2 WT mutant yeast strains grown 25  C, speci&#239;&#172;?c <br /> probes RNAs indicated. The number lane represents percentage unprocessed snoRNA precursor. (D) Western blot analysis the <br /> composition puri&#239;&#172;?ed TRAMPs containing WT mutant Air2p variants. TRAMP subunits detected antibodies indicated: HSW, <br /> high-salt wash (1M KCl). <br />  <br />  <br /> interaction Mtr4p. We observed N 57                     Air2p (ZnK1-5, residues 57-180) NMR spectroscopy. <br /> (residues 57-344) ZnK1-5L (residues 26-205) proteins                   The Air2p puri&#239;&#172;?ed denaturing conditions and <br /> did co-purify Mtr4p (Figure 1D, lanes N 57                         refolded Ni-NTA column presence N-terminal <br /> ZnK1-5L). Interestingly, N 57 protein contrary                      Smt3 fusion protein. We &#239;&#172;?rst con&#239;&#172;?rmed, fragment <br /> pulled signi&#239;&#172;?cantly higher amounts Trf4p                      suf&#239;&#172;?cient activate PAP activity Trf4p <br /> WT presence 1M KCl (HSW). At present,                       (Figure 2B, &#239;&#172;?rst lane). For NMR experiments, high-salt <br /> we know reason higher yields Trf4p               concentration 500 mM NaCl used prevent the <br /> mutant. These results signi&#239;&#172;?cantly altered                 protein aggregation. Although protein construct <br /> levels Mtr4p Trf4p mutants (Supplementary                  suffered low solubility limit (150 mM), we <br /> Figure S1C). In addition, deletion 139 amino               able achieve 1H, 13C 15N resonance assign- <br /> acids weakened interaction Mtr4p Trf4p                   ments ZnK1-5 determine three-dimensional <br /> (Figure 1D, samples C 180 ZnK1-5L).                                    structure (Figure 3A). The chemical shift deviations of <br />                                                                            Ca Ha assigned backbone resonances of <br /> Air2p genuine RNA-binding protein                                     ZnK1-5 sequence-dependent random coil values <br /> To investigate TRAMP4 mutant Air2p                          <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> ZnK1-5 region. This agrees well <br /> ZnK4 lacks PAP activity, studied central region                  absence secondary structure elements the <br />                                                                                       Nucleic Acids Research, 2012, Vol. 40, No. 12 5685 <br />  <br />  <br />  <br />  <br /> Figure 2. Air2p ZnK4 ZnK5 essential TRAMP polyadenylation activity vitro. (A) In vitro polyadenylation assay af&#239;&#172;?nity puri&#239;&#172;?ed <br /> WT mutant Air2p TRAMPs performed presence 0.5 mM ATP 50 fmol radioactively labeled hypomodi&#239;&#172;?ed tRNAiMet <br /> (migration indicated arrow). Reactions stopped 15, 30 60 min, respectively. (B) Reconstitution Trf4p PAP activity with <br /> recombinant Air2p. In vitro polyadenylation assay performed 60 ng WT mutant recombinant Air2p 20 ng WT Trf4p af&#239;&#172;?nity <br /> puri&#239;&#172;?ed air1Dair2  yeast strain presence [a-32P]-ATP 50 fmol unlabeled hypomodi&#239;&#172;?ed tRNAiMet (migration indicated an <br /> arrow). The presence radioactive signal lane result polyadenylation activity. Reactions Trf4p omitted (np) serves a <br /> negative control bacterial PAP contamination. Trf4p puri&#239;&#172;?ed air1Dair2  strain detectable activity (lane 4, yTrf4p). Trf4p <br /> puri&#239;&#172;?ed WT yeast strain (TRAMP4) used positive control (lane 3, TRAMP4). Reactions stopped 60 min case of <br /> recombinant proteins 10 min case TRAMP4. The purity recombinant Air2 proteins used assay separated 12% polyacryl- <br /> amide gel stained Coomassie blue shown right. (C) Rrp6p-TAP puri&#239;&#172;?ed exosome fraction used degradation assays this <br /> work compared pro&#239;&#172;?le exosome puri&#239;&#172;?ed core subunit Rrp4p. The puri&#239;&#172;?ed complexes resolved 10% SDS-PAGE and <br /> silverstained. The migration position Rrp6p marked arrow. M molecular weight marker. (D) ZnK1, 2 3 dispensable for <br /> exosome activation. In vitro coupled polyadenylation/exosome assays 30 ng TRAMP4 obtained af&#239;&#172;?nity puri&#239;&#172;?cation WT mutant <br /> Air2p. The reactions performed presence 0.5 mM ATP 50 fmol radioactively labeled hypomodi&#239;&#172;?ed tRNAiMet (migration <br /> indicated arrow). Reactions stopped 15, 30 60 min, respectively. RNAs resolved 20% PAGE. <br />  <br />  <br />  <br />  <br /> topology zinc knuckles (13,22). The histidines                    Therefore, resulting NMR structure low <br /> CCHC motif ZnKs, H71, H90, H109, H131                             resolution (Supplementary Table S1, Supplementary <br /> and H172, involved coordination zinc ions                      Figure S2). The structure Air2p region consists of <br /> the N-H correlations residues span                        &#239;&#172;?ve independent zinc knuckles adopt canonical <br /> region 2D 1H-15N HSQC (Figure 3B) char-                      zinc knuckle fold CCHC type. The domains are <br /> acteristic zinc knuckles (22). Furthermore, presence                connected &#239;&#172;&#8218;exible linkers, inter-domain <br /> of folded zinc knuckles con&#239;&#172;?rmed addition                     contacts (Figure 3A). The conserved <br /> EDTA excess zinc ions resulted loss res-                 IWRxY motif linker 4 unstructured free <br /> onance dispersion 2D 1H-15N HSQC (data                           form, forms short a-helix binding to <br /> shown). The NOESY spectra suffered low                               Trf4p (15). <br /> signal-to-noise ratio result low solubility                     To investigate Air2p genuine RNA-binding <br /> ZnK1-5 protein presence high-salt buffer.                        protein, performed quantitative solution-binding <br />  5686 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br />  <br />     A                                                                                                B <br />  <br />  <br />  <br />  <br />     C                                                                              D <br />  <br />  <br />  <br />  <br />     E                                                                              F <br />  <br />  <br />  <br />  <br />                                                                                    G <br />  <br />  <br />  <br />  <br /> Figure 3. Structure RNA-binding ZnK1-5 Air2p. (A) The lowest energy structure Air2p ZnK1-5. The protein shown ribbon <br /> model (in red) zinc-coordinating residues (CCHC) shown yellow. The zinc ions shown blue. (B) Close-up view 1H-15N <br /> TROSY spectrum, showing N-H correlations histidines involved coordination zinc ions. (C) Air2p ZnK1-5 binds A15 RNA. <br /> Binding isotherms equilibrium binding ZnK1-5 Air2p &#239;&#172;&#8218;uorescently labeled A15 RNA, monitored &#239;&#172;&#8218;uorescence anisotropy. (D) Air2p <br /> ZnK1-5 binding dA15 DNA. The ionic strength pH binding buffer measurements. The dissociation constant (Kd) <br /> was calculated best &#239;&#172;?t data using single-site binding isotherm. (E) 1H-15N TROSY spectra Air2p ZnK1-5 (in blue) the <br /> presence equivalent hypomodi&#239;&#172;?ed tRNAiMet (in red; 1H-15N HSQC) 20  C. (F) Summary chemical shift perturbations line <br /> broadening Air2p ZnK1-5 binding hypomodi&#239;&#172;?ed tRNAiMet. Affected residues (qualitatively described perturbation index; yes=1, <br /> no=0) plotted amino-acid residue number. The assignments residues indicated asterisks obtained. <br /> (G) Solvent-accessible surface representation RNA-binding ZnK Air2p (ZnK2, ZnK3 ZnK4) colored electrostatic potential <br /> (blue, positive; red, negative) representative structure. <br />                                                                       Nucleic Acids Research, 2012, Vol. 40, No. 12 5687 <br />  <br />  <br /> assay using &#239;&#172;&#8218;uorescence anisotropy (FA) experiments. We       C 499 suf&#239;&#172;?cient TRAMP4 assembly <br /> found Air2p ZnK1-5 binds unstructured A15 RNA            (Supplementary Figure S3D). In vivo, C 499 displayed <br /> with KD low micromolar range (Figure 3C),      cold sensitivity impaired growth 30  C when <br /> the binding af&#239;&#172;?nity dA15 DNA detected      expressed trf4Dtrf5  deletion background (Figure 4C, <br /> our experimental setup (KD &gt;&gt; 100mM) (Figure 3D).             Supplementary Figure S3A S3B). When expressed in <br /> Next, used NMR spectroscopy reveal ZnK            single trf4  strain, C 499 N 96 able <br /> domains Air2p used bind hypomodi&#239;&#172;?ed              suppress RNA degradation/processing defects of <br /> tRNAiMet, substrate used vitro assays     trf4  vivo (Figure 4D). TRAMPs puri&#239;&#172;?ed via <br /> this work. The chemical shift changes resonance           proteinA-tagged N 96 C 499 showed WT-like <br /> broadening observed 1H-15N HSQC spectrum            PAP activities (Figure 4E) able activate <br /> Air2p ZnK1-5 addition tRNAiMet established        tRNAiMet degradation presence Rrp6p-TAP <br /> that tRNAiMet interacts second, third,   exosomes vitro (Figure 4F). <br /> fourth ZnKs (Figure 3E F). In addition, chemical         In agreement previously published data, cata- <br /> shifts linkers connecting ZnKs perturbed    lytically inactive DXD TRF4 mutant suppressed par- <br /> as well. This suggested highly conserved linkers   tially NEL025C accumulation (6,7). This implies that <br /> that unstructured free form, structured     TRAMP act polyadenylation independent <br /> upon RNA-binding akin linkers viral nucleocap-      manner vivo. In addition DXD, identi&#239;&#172;?ed <br /> sid (NC) proteins (13). Our attempts compare binding       D425A second point mutant impaired PAP <br /> of Air2p hypomodi&#239;&#172;?ed tRNAiMet native tRNAiMet          activity (Figure 4F). Compared DXD, D425A <br /> (the poor substrate TRAMPs) failed,     exhibited weak, background accumu- <br /> we able prepare suf&#239;&#172;?cient quantities native    lation TRAMP targets seen northern blot analysis <br /> tRNAiMet yeast NMR experiments.                      (Figure 4D). Moreover, able support growth of <br />                                                               trf4Dtrf5  strain (Figure 4C). The puri&#239;&#172;?ed TRAMP4 con- <br /> The minimal functional Trf4p                                  taining <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>point mutation</span> D425 failed polyadenylate <br /> The recent crystal structure truncated Trf4p bound    hypomodi&#239;&#172;?ed tRNAiMet vitro standard assay <br /> fragment Air2p encompasses conserved              conditions accordingly activate exosome <br /> catalytic (CAT) central domains (CD) Trf4p             (Figure 4E F), complex integrity was <br /> (residues 161&#226;&#8364;&#8220;481) (15). We observed regions       affected (Supplementary Figure S3D). We found <br /> alone (fragments 182&#226;&#8364;&#8220;463 161&#226;&#8364;&#8220;481) functional       increasing ATP concentration resulted gradual <br /> in vivo able rescue viability      increase PAP activity D425A mutant to <br /> double deletion strain trf4Dtrf5  (Figure 4A C        4 mM concentration tested, did change <br /> Supplementary Figure S3A S3B). To search          activity saturated WT protein <br /> minimal functional Trf4p address role indi-     (Figure 4G). This result indicated <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> vidual Trf4p domains, prepared set deletion       reduced Trf4p af&#239;&#172;?nity ATP. We propose that <br /> point mutant constructs TRF4 (Figure 4B). The              lowered ATP substrate af&#239;&#172;?nity D425A slows down <br /> deletions designed remove poorly conserved,           PAP activity complex, suf&#239;&#172;?cient <br /> potentially unstructured parts protein. In addition    TRAMP4 function vivo. <br /> to catalytically inactive DXD mutant (D236A and <br /> D238A) (5), introduced point mutations: (i)            The Mtr4p-arch domain required TRAMP4 <br /> D425A residing highly conserved aspartate residue     complex-mediated exosome activation vitro <br /> the central domain (30), present canonical <br /> PAPs, (ii) K552A, proposed reside       We observed absence Air proteins the <br /> in catalytic center Trf4p dRP lyase activity (31).    stability Trf4p-Mtr4p heterodimer compromised <br />    Deletions entire N- (N 181) C- (C 463)           low salt conditions (Figure 5A) implying that <br /> termini surrounding CAT CD parts Trf4p did         Air proteins mediate interaction Trf4p and <br /> not support viability double trf4Dtrf5  strain         Mtr4p. This agreement similar recent &#239;&#172;?ndings <br /> (Figure 4C, Supplementary Figure S3A S3B)             Schizosaccharomyces pombe (32), suggesting a <br /> exhibited accumulation well-characterized         conserved mode Mtr4p interaction TRAMP in <br /> TRAMP4 substrates, NEL025c CUT precursors              distant yeast species. Based sequence conservation <br /> SnR13 U14 snoRNAs expressed trf4                  molecular surface scanning, patches were <br /> strain (Figure 4D). Although able detect       proposed possible binders Trf4p-Air2p (17). To <br /> expression fragments western blot                 identify interaction surface Mtr4p rest of <br /> (Supplementary Figure S3C), attempts purify            TRAMP complex, prepared set terminal and <br /> these mutants assess interaction            internal deletion variants N-terminally fused cleav- <br /> TRAMP components activities failed. Thus,           able 2 proteinA tag (Figure 5B). As expected, the <br /> we hypothesize deletion resulted protein          mutants conserved core able support <br /> misfolding instability. The analyses rest   growth absence WT Mtr4p, removal of <br /> deletion mutants indicated minimal functional Trf4p       &#239;&#172;?rst 143 amino acids (N 143) resulted strong <br /> region (spanning residues 97&#226;&#8364;&#8220;499) containing CAT          growth defect (Figure 5C). The archless mutant shows <br /> CD Trf4p surrounded short extensions N 96            reduced      growth       reported   previously   (16). <br />  5688 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br />  <br />  <br /> Figure 4. Mutational analysis Trf4p reveals minimal functional Trf4p indicates importance D425 PAP activity. (A) Fragment of <br /> Trf4p (aminoacids 161&#226;&#8364;&#8220;481) used previous structural studies does support growth trf4Dtrf5  background. Growth test analysis wt and <br /> mutants TRF4 episomally expressed trf4Dtrf5  strain complemented WT TRF4 URA3 plasmid serially diluted spotted on <br /> media 5-FOA. (B) Overview mutant forms TRF4 used study: CAT, catalytical domain; CD, central domain. (C) Minimal viable <br /> allele TRF4 spans amino acids 96&#226;&#8364;&#8220;499. Drop tests performed analogously (A). (D) Northern blot analysis total RNA isolated from <br /> WT mutant TRF4 yeast strains DNA probes speci&#239;&#172;?c indicated RNAs. (E) In vitro polyadenylation assays 20 ng af&#239;&#172;?nity puri&#239;&#172;?ed <br /> WT mutant TRAMP4 performed presence 0.5 mM ATP, 10 ng radioactively labeled hypomodi&#239;&#172;?ed tRNAiMet (migration indicated <br /> by arrow). Reactions stopped 15, 30 60 min, respectively. The &#239;&#172;?rst lane (np) represents control sample protein added. <br /> RNAs separated 20% PAGE. (F) Coupled polyadenylation exosome assay. The 50 -end-labeled unmodi&#239;&#172;?ed tRNAiMet incubated with <br /> 40 ng af&#239;&#172;?nity-puri&#239;&#172;?ed TRAMP4 (ProtA-Trf4p) 40 ng nuclear exosome (Rrp6-TAP) 15, 30 60 min. The &#239;&#172;?rst lane (np) represents <br /> control sample protein added. (G) D425A mutation lowers Trf4p af&#239;&#172;?nity ATP. Polyadenylation assay performed 25 ng af&#239;&#172;?nity <br /> puri&#239;&#172;?ed D425A Trf4p, 50 fmol 50 -end-labeled unmodi&#239;&#172;?ed tRNAiMet presence ATP concentrations indicated. Reactions stopped after <br /> 60 min resolved 10% denaturing gel. <br />  <br />  <br />  <br />  <br /> Interestingly, deletion KOW domain suf&#239;&#172;?-                   dispensable TRAMP formation vivo (Figure 5E). <br /> cient reproduce phenotype (Figure 5D).                            Furthermore, observed complex formation <br />    Using af&#239;&#172;?nity puri&#239;&#172;?cation S. cerevisiae subse-                deletion unusual RecA-1N-terminal <br /> quent western blotting, con&#239;&#172;?rmed previous results                  b-hairpin extension (N 143) structure (17), <br /> from vitro (17) showing arch domain                   conserved outside Ski2 protein family <br />                                                                                    Nucleic Acids Research, 2012, Vol. 40, No. 12 5689 <br />  <br />  <br />  <br />  <br /> Figure 5. The RNA-binding KOW domain Mtr4p required activation exosome TRAMP complex. (A) Air2p required Mtr4p <br /> association Trf4p. Western blot analysis af&#239;&#172;?nity puri&#239;&#172;?ed ProteinA-Trf4p WT air1Dair2  strains. TRAMP subunits detected <br /> with antibodies indicated. HSW, 1 M KCl high-salt wash used puri&#239;&#172;?cation. (B) Overview mutant forms MTR4 used study: <br /> N-term, N-terminal b-hairpin extension, WH, winged-helix domain. (C D) Growth test analysis WT mutants MTR4 episomally <br /> expressed mtr4  strain complemented WT MTR4 URA3 plasmid serially diluted spotted media 5-FOA. (E) Western blot <br /> analysis composition puri&#239;&#172;?ed TRAMPs containing different Mtr4p variants. TRAMP subunits detected antibodies indicated. <br /> (F G) Coupled vitro polyadenylation exosome activation assay 40 ng af&#239;&#172;?nity puri&#239;&#172;?ed Mtr4p variants performed 0.5 mM <br /> ATP 50 fmol radioactively labeled hypomodi&#239;&#172;?ed tRNAiMet (migration position indicated arrow). Reactions stopped 15, 30 and <br /> 60 min, respectively. Samples np stand controls protein included. The entire gel shown Supplementary Figure S4A. <br />  <br />  <br /> (Figure 5E). Deletion of domains form                possibility misfolding cellular mislocalization. <br /> helicase core, led loss co-puri&#239;&#172;?cation TRAMP                Since <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span> area helical core RNA <br /> components (Figure 5E). Nevertheless, rule                 exit tunnel proposed serve exosome interaction <br />  5690 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br /> (16), narrowed interaction site          substrates based low-sequence speci&#239;&#172;?city interactions <br /> Trf4p/Air2p heterodimer the barrel formed      sugar-phosphate backbone. <br /> by RecA-2, winged helix helical bundle domains.              The analysis deletion mutants indicated import- <br />    Since    Mtr4p       dispensable        TRAMP       ance &#239;&#172;?rst 25 N-terminal aminoacids inter- <br /> polyadenylation activity required activation   action Mtr4p, N 57 (residues 57&#226;&#8364;&#8220;344) <br /> of exosome (4,5), tested loss           ZnK1-5L (residues 26&#226;&#8364;&#8220;205) proteins did co-purify <br /> N-terminal extension arch domains effect        Mtr4p (Figure 1D, lanes N 57 ZnK1-5L). The <br /> the ability TRAMP complex activate degrad-          N-terminus poorly conserved Air1/2p <br /> ation coupled vitro polyadenylation exosome         homologs. In fact, metazoan Air1-like protein (also <br /> assay. While N 143 mutant did devi-          designated ZCCHC7) contains long N-terminal exten- <br /> ation WT, observed complete loss            sion (data shown). Given observation the <br /> exosome activation presence archless            human Mtr4p (SKIV2L2) forms NEXT (Nuclear <br /> mutant different structured substrates                 EXosome Targeting) complex independently PAPD5/ <br /> (Figure 5F, Supplementary Figure S4A S4B). To in-         ZCCHC7 (the human homologs Trf4p/Air1/2p) (34), it <br /> vestigate importance KOW domain, RNA           possible interaction noncanonical PAP <br /> binding element arch (17), conducted assay      (Trf4p/Air2p) Mtr4p helicase diverged during <br /> in presence Mtr4p variant lacking domain. As      evolution depending organism complexity. <br /> shown Figure 5G, removal KOW domain                    On hand, Trf4p interacting residues in <br /> recapitulated inability archless mutant         Air2p identi&#239;&#172;?ed NMR titration experiments pos- <br /> ef&#239;&#172;?ciently activate exosome vitro. Since observed       itions 139&#226;&#8364;&#8220;143, W140G R141E reside stretch of <br /> apparently lowered polyadenylation KOWless                 aminoacids forming (P/T)xIWRxYxL motif conserved <br /> TRAMPs, compared exosome activation            Air-like proteins distant species including <br /> the presence lowered amounts WT Mtr4p. Despite          humans (Supplementary Figure S1A). The protein <br /> the resulting reduction observed polyadenylation           pulldown experiments revealed strong defects Trf4p <br /> activity, WT Mtr4p TRAMP activated                  Mtr4p copuri&#239;&#172;?cation mutants conserved <br /> Rrp6p-TAP, unlike KOWless mutant (Supplementary               motif (L4 mutant) mutant dis- <br /> Figure S4C). This slow growth pheno-        rupted &#239;&#172;?fth zinc knuckle fold (ZnK5) (Figure 1D). <br /> types (Figure 5D) suggested arch particularly    Mtr4p required Trf4p&#226;&#8364;&#8220;Air2p interaction. It is <br /> KOW domains crucial exosome activation.               likely dimerization Air2p Trf4p needed for <br />                                                               ef&#239;&#172;?cient assembly Mtr4p, absence one <br />                                                               results loss Mtr4p copuri&#239;&#172;?cation (Figures 1D <br /> DISCUSSION                                                    5A). Therefore, believe linker 4 ZnK5 <br /> We identi&#239;&#172;?ed key features assembly,       Air2p crucial regions responsible interaction <br /> RNA binding exosome activation yeast               Trf4p. The ZnK displays highest <br /> TRAMP RNA surveillance complex. While focused              degree conservation &#239;&#172;?ve Air ZnKs <br /> on analysis TRAMP4 complex,             (Supplementary Figure S1A). Likewise, Trf4p <br /> similar composition TRAMP5 (33) high level          surface interacting ZnK4&#226;&#8364;&#8220;ZnK5 region is <br /> sequence conservation Trf5p Trf4p,            conserved Trf4-like proteins (15). This <br /> between Air1p Air2p (5), data presented          implies phylogenetically conserved mode interaction <br /> likely valid TRAMP5.                                  Air1/2p homologs Trf4p-like noncanonical <br />                                                               PAPs. Although human PAPD5 does apparently <br />                                                               require cofactor PAP activity vitro (35), does <br /> Dual function Air1/2p TRAMP complex <br />                                                               form complex ZCCHC7 putative <br /> Here, demonstrate Air2p bona &#239;&#172;?de                 RNA-binding proteins (34), suggesting Air-like <br /> RNA-binding protein. The Air2p ZnKs 2, 3 4 bind           proteins mainly serve RNA substrate speci&#239;&#172;?city <br /> aberrant tRNA (Figure 3E F); however,            factors. During preparation manuscript, <br /> fourth ZnK fully essential TRAMP PAP activity          Fasken co-workers (36) reported similar results on <br /> in vitro (Figure 2A B). Although study           Air1p regard importance zinc <br /> Hamill et al. (15) proposed involvement RNA            knuckles cell viability IWRxY motif conservation, <br /> binding, analyses revealed speci&#239;&#172;?c role      NEL025C degradation Trf4p interaction (36). Our <br /> for ZnK1. It possible ZnK1 binds RNAs,      analysis goes work distinguishing dif- <br /> used tests. Alternatively, involved   ferent key roles ZnK4 RNA binding Trf4p ac- <br /> interaction TRAMP-associated proteins.             tivation (see below) ZnK5 Trf4p interaction. <br /> Structural sequence alignment analyses indicate      Fasken colleagues Air1p Air2p <br /> the RNA-binding ZnKs Air2p use different binding         mutants ZnK5 unstable 37  C lesser <br /> mode compared NC proteins. They lack hydro-        extend 25  C, likely reason strong <br /> phobic pocket, accommodates guanine base         growth defects high temperatures. Our western blot <br /> NC proteins. Instead, large electropositive         analysis expression levels different Air2p <br /> patches (Figure 3G) created lysines, arginines     variants shows slight decrease ZnK5 Air2p <br /> serines. We speculate residues respon-    mutant 25  C (Supplementary Figure S1B). Moreover, <br /> sible recognition broad spectrum RNA          vitro reconstitution assays (Figure 2B) suggest that <br />                                                                             Nucleic Acids Research, 2012, Vol. 40, No. 12 5691 <br />  <br />  <br /> this mutant unable form functional Trf4p-Air2p PAP            (Figure 4G). This result indicated mutation <br /> even amounts comparable WT. Thus conclude                  reduced Trf4p af&#239;&#172;?nity ATP. <br /> that absence Trf4p ZnK5 immunoprecipitates                By examining available crystal structure (15), we <br /> not caused lowered vivo expression Trf4p,          observed D425 residue situated approximately <br /> to weakened interactions proteins              20 A&#203;&#353; active site. Close inspection mechan- <br /> response disruption &#239;&#172;?fth Air2p zinc                   ism homologous yeast canonical poly(A) polymer- <br /> knuckle. Furthermore, Air2p mediates               ase Pap1p catalysis revealed hydrogen bonding this <br /> interaction Trf4 Mtr4 proteins (Figure 5A).             region required stabilization conformational <br /> Our work extends analysis N- C- termini           movements leading open closed conformation <br /> of Air2p revealing crucial interactions Mtr4p. The             protein (37). When N189A (CAT domain) <br /> Air2p termini enhance Mtr4p binding, main                 mutation introduced Pap1p, equilibrium <br /> surface interaction Trf4p located &#239;&#172;?fth           open closed state shifted, resulting in <br /> ZnK adjacent linker 4 regions (Figure 1D).                  increased KM ATP (37). N189 forms hydrogen bond <br /> Based mapping analyses prediction             main chain carbonyl Y307 (CD) the <br /> the Mtr4p helicase barrel serves inter-           closed conformation (Supplementary Figure S3E and <br /> action exosome (16), propose model                 S3F). Interestingly, Y307 Pap1p resides both, <br /> quarternary interactions TRAMP complex. In               sequentially (30) spatially, identical position D425 <br /> this model, Trf4p/Air2p binds Mtr4p DExH                     Trf4p (Supplementary Figure S3G). Therefore, we <br /> globular core, barrel formed RecA-2,              suggest chain D425 serve a <br /> winged helix helical bundle domains. We suggest                 hydrogen bond acceptor analogous Y307 main chain <br /> that RNA substrates &#239;&#172;?rst screened Air2p              carbonyl. We identi&#239;&#172;?ed hydrogen bond <br /> ZnK2 &#226;&#8364;&#8220; ZnK4 region protrudes Mtr4 surface                 formed D425 published structure [(15) and <br /> and consequently loaded Trf4p active site. We            Supplementary Figure S3G]. Notably, structure was <br /> speculate RNA 30 -oligo(A) terminus leaves Trf4p           solved Trf4p apoenzyme; likely rep- <br /> in direction KOW-RecA-2 interface,             resent open state status. We, therefore, suggest that <br /> the RNA 30 -end expected enter helicase core (17)         conformational change analogous Pap1p occur <br /> (Figure 6).                                                         Trf4p RNA ATP binding, indicating the <br />                                                                     conservation catalytic mechanism related <br /> TRF4 catalytical mechanism                                          proteins. <br /> We increasing ATP concentration resulted a <br /> gradual increase PAP activity D425A mutant            RNA binding KOW domain involved exosome <br /> up 4 mM concentration tested, did             activation <br /> change activity saturated WT protein             The role arch domain exosome function vivo <br />                                                                     proposed previously Jackson et al. (16) who <br />                                                                     observed accumulation 5.8S+30 rRNA stabil- <br />                                                                     ization 50 -ETS, speci&#239;&#172;?c substrates Rrp6p <br />                                                                     activity (38). Here report arch, particularly the <br />                                                                     KOW domain, Mtr4p necessary the <br />                                                                     TRAMP-mediated exosome activation vitro. We show <br />                                                                     TRAMPs lacking arch KOW able add <br />                                                                     poly(A) tails, completely fail activate <br />                                                                     exosomes degrade structured RNAs vitro (Figure 1F <br />                                                                     G; Supplementary Figure S4). Recently Jia et al. (14) <br />                                                                     showed Mtr4p regulates individual steps of <br />                                                                     TRAMP polyadenylation length the <br />                                                                     poly(A) tails added. We exclude possibility <br />                                                                     arch/KOW involved regulation. <br />                                                                     However, assays, N 143  KOW TRAMPs <br />                                                                     showed comparable polyadenylation, N 143 <br />                                                                     fully able initiate exosomal degradation, whereas <br />                                                                      KOW highly defective. Therefore, conclude that <br />                                                                     observed differences polyadenylation seen in <br />                                                                     KOW-less TRAMP explain lack exosome <br />                                                                     activation, propose KOW domain is <br />                                                                     crucial TRAMP-mediated exosome activation. <br />                                                                        As helicase ATPase activities Mtr4p <br /> Figure 6. Model TRAMP assembly RNA binding. The Air2         affected arch removal (16,17), speculate the <br /> protein bridging Mtr4p (interaction N- C-terminus) <br /> and Trf4p (interaction L4, ZnK5 C-terminus). The ZnKs 2, <br />                                                                     RNA-binding properties arch necessary to <br /> 3 4 involved RNA binding. The KOW domain Mtr4p     handle substrate different activities the <br /> required exosome activation.                                    TRAMP-exosome pathway. Both close proximity of <br />  5692 Nucleic Acids Research, 2012, Vol. 40, No. 12 <br />  <br />  <br /> the KOW helicase RNA entry site possible       NOESY spectra obtained BMRZ NMR <br /> &#239;&#172;&#8218;exibility elbow arm (16) suggest arch     facility supported EU-NMR program <br /> transports RNA Trf4p-Air2p, helicase core         (RII3-026145); EC FP-7 (grant 205872 to <br /> and potentially Rrp6p. Alternatively, arch         M.S.A.); Brno City Municipality Scholarship for <br /> required displacement RNA-binding            Talented Ph.D. Students (to P.H. F.H.). Funding <br /> factors shuttling substrate helicase core.     open access charge: Senior International Wellcome <br /> We exclude possibility arch          Trust Fellowship. <br /> serves protein&#226;&#8364;&#8220;protein interactions Rrp6p or <br /> other core exosome components. This scenario               Con&#239;&#172;&#8218;ict statement. None declared. <br /> supported presence similar structure recently <br /> uncovered cytoplasmic exosome cofactor Ski2p <br /> (16,17,39).                                                   REFERENCES <br />    The nuclear exosomes contain con&#239;&#172;?rmed <br /> exoribonucleases&#226;&#8364;&#8221;the Rrp6p Rrp44p. Currently,           1. Brosnan,C.A. Voinnet,O. (2009) The long short of <br />                                                                   noncoding RNAs. Curr. Opin. Cell. Biol., 21, 416&#226;&#8364;&#8220;425. <br /> is fully understood subunits inter-    2. Philips,A.V. Cooper,T.A. (2000) RNA processing human <br /> play divergent exosome substrates vivo. However,         disease. Cell. Mol. Life Sci., 57, 235&#226;&#8364;&#8220;249. <br /> was shown conditions used vitro          3. Kadaba,S., Krueger,A., Trice,T., Krecic,A.M., Hinnebusch,A.G. <br /> assays (5 mM Mg2+) nuclease Rrp44p inactive                Anderson,J. (2004) Nuclear surveillance degradation of <br /> (12). Therefore, propose nuclease activity                hypomodi&#239;&#172;?ed initiator tRNAMet S. cerevisiae. Genes Dev., 18, <br />                                                                   1227&#226;&#8364;&#8220;1240. <br /> observed assays fully attributed Rrp6p.       4. LaCava,J., Houseley,J., Saveanu,C., Petfalski,E., Thompson,E., <br /> Therefore, interesting address RNAs         Jacquier,A. Tollervey,D. (2005) RNA degradation the <br /> bound KOW undergo processing core              exosome promoted nuclear polyadenylation complex. Cell, <br /> exosome speci&#239;&#172;?cally triggered         121, 713&#226;&#8364;&#8220;724. <br />                                                                5. Vanacova,S., Wolf,J., Martin,G., Blank,D., Dettwiler,S., <br /> degradation Rrp6p. The exact mechanistic details               Friedlein,A., Langen,H., Keith,G. Keller,W. (2005) A new <br /> remain obscure subject following               yeast poly(A) polymerase complex involved RNA quality <br /> studies.                                                          control. PLoS Biol., 3, e189. <br />                                                                6. Wyers,F., Rougemaille,M., Badis,G., Rousselle,J.C., Dufour,M.E., <br />                                                                   Boulay,J., Regnault,B., Devaux,F., Namane,A., Seraphin,B. et al. <br /> ACCESSION NUMBERS                                                 (2005) Cryptic pol II transcripts degraded nuclear <br />                                                                   quality control pathway involving new poly(A) polymerase. <br /> The atomic coordinates restraints (<span id='am-241' about='Thesaurus:code' typeof='owl:Thing'>code</span> 2lli)            Cell, 121, 725&#226;&#8364;&#8220;737. <br /> been deposited Protein Data Bank, Research              7. San Paolo,S., Vanacova,S., Schenk,L., Scherrer,T., Blank,D., <br /> Collaboratory Structural Bioinformatics, Rutgers              Keller,W. Gerber,A.P. (2009) Distinct roles non-canonical <br />                                                                   poly(A) polymerases RNA metabolism. PLoS Genet., 5, <br /> University, New Brunswick, NJ (http://www.rcsb.org/).             e1000555. <br />                                                                8. Wlotzka,W., Kudla,G., Granneman,S. Tollervey,D. (2011) <br />                                                                   The nuclear RNA polymerase II surveillance targets <br /> SUPPLEMENTARY DATA                                                polymerase III transcripts. EMBO J., 30, 1790&#226;&#8364;&#8220;1803. <br />                                                                9. Houseley,J., Kotovic,K., El Hage,A. Tollervey,D. (2007) Trf4 <br /> Supplementary Data available NAR Online:                   targets ncRNAs telomeric rDNA spacer regions and <br /> Supplementary Tables 1&#226;&#8364;&#8220;5, Supplementary Figures 1&#226;&#8364;&#8220;4               functions rDNA copy number control. EMBO J., 26, <br /> and Supplementary Reference [40].                                 4996&#226;&#8364;&#8220;5006. <br />                                                               10. Houseley,J. Tollervey,D. (2008) The nuclear RNA <br />                                                                   surveillance machinery: link ncRNAs genome <br />                                                                   structure budding yeast. Biochim. Biophys. Acta, 1779, 239&#226;&#8364;&#8220;246. <br /> ACKNOWLEDGEMENTS                                              11. Thiebaut,M., Kisseleva-Romanova,E., Rougemaille,M., Boulay,J. <br /> We thank Leona Svajdova excellent technical assist-           Libri,D. (2006) Transcription termination nuclear <br /> ance; Walter Keller Georges Martin helpful                degradation cryptic unstable transcripts: role the <br />                                                                   nrd1-nab3 pathway genome surveillance. Mol. Cell., 23, <br /> advice; John LaCava, Jonathan Houseley                    853&#226;&#8364;&#8220;864. <br /> Andrzej Dziembowski critical reading comments         12. Callahan,K.P. Butler,J.S. (2010) TRAMP complex enhances <br /> on manuscript. We thank David Tollervey,                 RNA degradation nuclear exosome component Rrp6. <br /> Ambro Van Hoof Domenico Libri yeast strains               J. Biol. Chem., 285, 3540&#226;&#8364;&#8220;3547. <br />                                                               13. D&#226;&#8364;&#8482;Souza,V. Summers,M.F. (2005) How retroviruses select <br /> and constructs, Patrick Lindner Mtr4p antibodies, <br />                                                                   genomes. Nat. Rev. Microbiol., 3, 643&#226;&#8364;&#8220;655. <br /> Gerhard Wagner G-proteinB1 vector Christopher         14. Jia,H., Wang,X., Liu,F., Guenther,U.P., Srinivasan,S., <br /> Lima Smt3p Ulp1p expression vectors.                      Anderson,J.T. Jankowsky,E. (2011) The RNA helicase Mtr4p <br />                                                                   modulates polyadenylation TRAMP complex. Cell, 145, <br />                                              <br /> </body></html>