<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Exp Eye Res. 2011 October ; 93(4): 460&#226;&#8364;&#8220;474. doi:10.1016/j.exer.2011.06.010. <br />  <br />  <br />  <br />                            Phosphatidylinositol synthase required lens structural <br />                            integrity photoreceptor cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span> zebrafish eye <br />                            Taylor R. Murphy1, Thomas S. Vihtelic1, Kristina E. Ile2, Corey T. Watson3, Gregory B. <br />                            Willer3, Ronald G. Gregg3, Vytas A. Bankaitis2, David R. Hyde1,* <br />                            Taylor R. Murphy: tmurphy4@nd.edu; Thomas S. Vihtelic: Thomas.Vihtelic@mpiresearch.com; Kristina E. Ile: <br />                            kristina.ile@tuebingen.mpg.de; Corey T. Watson: ctwatson@sfu.ca; Gregory B. Willer: gregwiller@hotmail.com; Ronald <br />                            G. Gregg: ron.gregg@louisville.edu; Vytas A. Bankaitis: vytas@med.unc.edu; David R. Hyde: dhyde@nd.edu <br />                            1Department Biological Sciences Center Zebrafish Research, University Notre <br />                            Dame, Notre Dame, IN 46556 <br />                            2Department   Cell Developmental Biology, University North Carolina School Medicine, <br />                            Chapel Hill, NC 27599 <br />                            3Mutant    Zebrafish Mapping Facility, University Louisville, Louisville, KY 40202 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 The zebrafish lens opaque (lop) mutant previously isolated genetic screen shown <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                 lack rod cone photoreceptors exhibit lens opacity, cataract, 7 days post-fertilization <br />                                 (dpf). In manuscript, provide different lines evidence demonstrating lop <br />                                 <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> results defect cdipt (phosphatidylinositol (PI) synthase; CDP- <br />                                 diacylglycerol--inositol 3-phosphatidyltransferase) gene. First, DNA sequence analysis revealed <br />                                 lop mutant contained <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>missense mutation</span> lop open reading frame, yields a <br />                                 nonconservative amino acid substitution (Ser-111-Cys) PI synthase catalytic domain. <br />                                 Second, morpholino-mediated knockdown cdipt-encoded PI synthase protein phenocopied <br />                                 cdiptlop/lop mutant, abnormal lens epithelial secondary fiber cell morphologies and <br />                                 reduced numbers photoreceptors. Third, microinjection vitro transcribed, wild-type cdipt <br />                                 mRNA 1-4 cell stage cdiptlop/lop embryos significantly reduced percentage larvae <br />                                 displaying lens opacity 7 dpf. Fourth, cdipt retroviral-insertion allele, cdipthi559, exhibited <br />                                 similar lens retinal abnormalities failed complement cdiptlop mutant phenotype. <br />                                 To determine initial cellular defects associated cdipt mutant, examined <br />                                 homozygous cdipthi559/hi559 mutants prior gross lens opacification 6 dpf. The cdipthi559/hi559 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 mutants exhibited photoreceptor layer disruption photoreceptor cell death 3 4 dpf, <br />                                 respectively, followed lens dismorphogenesis 5 dpf. RT-PCR revealed cdipt gene is <br />                                 maternally expressed continues transcribed development adulthood, <br />                                 wide variety tissues. Using anti-zebrafish PI synthase polyclonal antiserum, localized <br />                                 protein developing eye, including photoreceptor layer lens cortical <br />                                 secondary fiber cells. As expected, polyclonal antiserum revealed PI synthase protein <br />                                 reduced in cdiptlop/lop cdipthi559/hi559 mutants. Furthermore, used a <br />                                 heterologous yeast phenotypic complementation assay confirm wild-type zebrafish <br />  <br />  <br />  <br />                            &#194;&#169; 2011 Elsevier Ltd. All rights reserved. <br />                            * <br />                              Correspondence sent to: David R. Hyde, Ph.D., Department Biological Sciences, Galvin Life Science Building, University of <br />                            Notre Dame, Notre Dame, Indiana, 46556, dhyde@nd.edu, Phone: 574-631-8054, Fax: 574-631-7413. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Murphy et al.                                                                                                  Page 2 <br />  <br />  <br />                                     cdipt allele encodes functional PI synthase activity. Taken together, cdipt-encoded PI synthase <br />                                     required survival photoreceptor cells lens epithelial secondary cortical fiber cells. <br />                                     These zebrafish cdipt alleles represent excellent vivo genetic tools study role of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     phosphatidylinositol phosphorylated derivatives lens photoreceptor development <br />                                     maintenance. <br />  <br />  <br />                               Keywords <br />                                     Phosphatidylinositol synthase; photoreceptor; lens epithelial cell; lens fiber cell; cell death; <br />                                     cataract <br />  <br />  <br />                               Introuction <br />                                                Zebrafish ideal model study vertebrate lens retina their <br />                                                rapid development highly conserved structural organization mammalian eye <br />                                                (Easter Nicola 1996; Glass Dahm 2004; Dahm et al. 2007; Vihtelic 2008). Between <br />                                                16-20 hpf (hours post fertilization), zebrafish lens begins form thickening of <br />                                                overlying surface ectoderm form solid lens placode (Schmitt Dowling 1994; <br />                                                Dahm et al. 2007; Vihtelic 2008; Greiling Clark 2009). Between 22-24 hpf, placode <br />                                                delaminates yield lens mass (Greiling Clark 2009), analogous lens <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                vesicle. By 30 hpf, cuboidal-shaped lens epithelial cells present anterior surface of <br />                                                developing lens, primary fiber cells surround core lens nucleus, secondary <br />                                                fiber cells elongating moving posteriorly (Greiling Clark 2009), inner <br />                                                layer adjacent optic cup begins generate neural retina (Hu 1999; Easter and <br />                                                Malicki 2002; Dahm et al. 2007; Vihtelic 2008). At 52 hpf, newly differentiating rod and <br />                                                cone photoreceptors begin expressing opsin genes (Nawrocki et al. 1985; Raymond et al. <br />                                                1995; Schmitt Dowling 1996; Schmitt Dowling 1999; Vihtelic et al. 1999; Soules <br />                                                Link 2005; Dahm et al. 2007). By 72 hpf, lens spherical visual is <br />                                                functional based behavioral tests (Soules Link 2005; Easter Nicola 1996; Bilotta <br />                                                2000; Li 2001; Neuhauss 2003; Greiling Clark 2009). <br />  <br />                                                A variety zebrafish mutants display abnormal eye development, model <br />                                                human ocular diseases (Barut Zon 2000; Shin Fishman 2002; Fadool Dowling <br />                                                2008). For example, mutants perturb lens formation include laminin alpha 1 (lama1arl) <br />                                                disrupted lens (dsl) (Vihtelic et al. 2001; Vihtelic Hyde 2002; Semina et al. 2006; <br />                                                Zinkevich et al. 2006). The lama1arl mutant exhibits defects lens epithelial cell <br />                                                population 5 dpf, significantly affecting organization survival retinal <br />                                                neurons (Vihtelic et al. 2001; Semina et al. 2006). Interestingly, mutations LAMB2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                (laminin beta 2) protein, directly interacts LAMA1, cause cataracts, glaucoma, <br />                                                microphthalmia humans (Colognato Yurchenco 2000; Semina et al. 2006; <br />                                                Zinkevich et al. 2006; Bredrup et al. 2008). <br />  <br />                                                In chemical mutagenesis screen identified lama1arl allele other <br />                                                zebrafish mutants abnormal eye morphologies (Vihtelic Hyde 2002), isolated <br />                                                lens opaque (lop) mutant exhibited opaque lens beginning 7 dpf (Vihtelic et al. <br />                                                2005). Immunostaining revealed maturation lop lens secondary fiber cells was <br />                                                disrupted. The lop mutant exhibited increased numbers proliferating cells the <br />                                                lens anterior chamber relative controls, suggested abnormal proliferation <br />                                                distal lens epithelial cell population. Surprisingly, lop central retina lacked <br />                                                rod cone photoreceptors 7 dpf, wild-type zebrafish retina possessed a <br />                                                complement retinal neurons, including rods cones (Johns 1981; Easter Nicola <br />                                                1996; Schmitt Dowling 1996; Schmitt Dowling 1999; Vihtelic et al. 2005). Lens <br />  <br />  <br />                                                     Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                            Page 3 <br />  <br />  <br />                                            transplant experiments 31 hpf demonstrated lens retinal phenotypes were <br />                                            lens-autonomous, suggesting mutant lens disrupted retinal development (Vihtelic et <br />                                            al. 2005). The underlying molecular defects lop mutant leading phenotypes, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            however, unknown. <br />  <br />                                            This study reveals lop <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> missense mutation cdipt (CDP- <br />                                            diacylglycerol:myo-inositol 3-phosphatidyltransferase) gene encoding PI synthase <br />                                            enzyme. Using previously identified retroviral-insertion allele cdipthi559 (Amsterdam et <br />                                            al. 2004), PCR-genotyped cdipthi559/hi559 homozygotes determine earliest <br />                                            ocular phenotype occurred photoreceptor layer 3 dpf, followed lens <br />                                            dismorphogenesis beginning 5 dpf. The cdipt gene, initially expressed <br />                                            maternally, continues expressed variety adult tissues. A polyclonal antisera <br />                                            specific zebrafish PI synthase localized protein developing eye and <br />                                            brain. To knowledge, zebrafish cdipt alleles vertebrate PI synthase <br />                                            mutants identified represent excellent vivo genetic model elucidate the <br />                                            function PI synthase maintenance lens transparency photoreceptor survival. <br />  <br />                               2. Materials Methods <br />                               2.1. Zebrafish maintenance <br />                                            Wild-type, cdiptlop mutant (Vihtelic et al. 2005) cdipthi559 mutant (Amsterdam et al. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2004) zebrafish stocks maintained Center Zebrafish Research the <br />                                            University Notre Dame Freimann Life Science Center using standard husbandry protocols <br />                                            (Westerfield 2000). The fish raised daily light cycle 14 hours light/10 hours <br />                                            dark. Embryos wild-type mass-matings mutant pair-matings reared 28.5&#194;&#176;C <br />                                            staged according hours post fertilization (hpf) days post fertilization (dpf) using <br />                                            morphological criteria (Kimmel et al. 1995). All experimental procedures conducted in <br />                                            agreement Animal Care Use Committee University Notre Dame and <br />                                            principles ARVO statement use animals vision research. <br />  <br />                               2.2. Complementation analysis <br />                                            Adult heterozygous cdipthi559/+ cdiptlop/+ mutants crossed, resulting larvae <br />                                            stereoscopically examined 7 dpf lop mutant phenotype. The trans- <br />                                            heterozygous mutants, age-matched wild-type controls, processed for <br />                                            histological analysis. A minimum fish genotype (n = 8 eyes) examined <br />                                            histology. <br />  <br />                               2.3. Histology <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Wild-type mutant larvae fixed 2% formaldehyde/2.5% glutaraldehyde/0.1 M <br />                                            cacodylate (pH 7.4) overnight 4&#194;&#176;C. Larvae washed 100 mM cacodylate 30 <br />                                            minutes, rinsed water, dehydrated series ethanol washes being <br />                                            infiltrated Polybed 812 (Polysciences; Warrington, PA) previously described <br />                                            (Vihtelic et al. 2001; Shi et al. 2005; Harding et al. 2008). Three micron serial sections were <br />                                            taken using glass knife Sorvall JB4 microtome stained 0.5% methylene blue/ <br />                                            0.5% azure II. A minimum fish genotype (n = 8 eyes) examined by <br />                                            histology. <br />  <br />                               2.4. Positional cloning <br />                                            Bulked segregant analysis mapped lop mutation chromosome 3 simple <br />                                            sequence-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> polymorphic markers Z59420 Z59380 (Knapik et al. 1998; Shimoda et <br />                                            al. 1999; Willer et al. 2005). High resolution linkage mapping using single nucleotide <br />                                            polymorphisms refined genomic region 200 Kb BAC DKEY-183N6. <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                           Page 4 <br />  <br />  <br />                                            Candidate genes critical interval evaluated cDNA sequencing (Willer et <br />                                            al. 2005). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.5. Yeast tetrad analysis <br />                                            A single genomic copy essential yeast phosphatidylinositol synthase (PIS1) gene was <br />                                            replaced KanMx (G418r) cassette diploid strain CTYD1 (Mata/&#206;&#177; ura3-52/ <br />                                            ura3-52 lys2-801/lys2-801 his3&#206;&#8221;-200/his3&#206;&#8221;-200 ade2-101/ADE2 TRP1/trp1&#206;&#8221; SEC14/ <br />                                            sec14-1ts) generate PIS1+/pis1&#206;&#8221; heterozygous diploid strain (Bankaitis et al. 1989). The <br />                                            replacement cassette generated PCR-amplification KanMx primers <br />                                            corresponding flanking regions PIS1 (Forward: 5&#226;&#8364;&#178;- <br />                                            GTAATGTAAATAAGAGGGAAAGTGTGATAGTACAAGAGATCTGTTTAGCTTGCC <br />                                            TCGTC-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            CAATTAAAAGGAGAAAAAAAGTAAGAAACTCATCCTAGTTTTCGACACTGGAT <br />                                            GGCGGC-3&#226;&#8364;&#178;). The gel-purified PCR-product transformed CTYD1 strain using <br />                                            standard lithium acetate methods (Ito et al. 1983; Rothstein 1983; Gietz et al. 1995). After a <br />                                            four-hour recovery non-selective media, yeast transformants plated yeast- <br />                                            peptone-dextrose (YPD) solid media supplemented 200 &#206;&#188;g/ml G418. Large G418r <br />                                            colonies picked genomic DNA prepped confirm desired replacement <br />                                            events PCR amplification forward primer hybridized sequence PIS1 <br />                                            5&#226;&#8364;&#178; UTR, located outside replacement cassette, reverse primer KanMX <br />                                            gene. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            To generate zebrafish cdipt expression plasmids, wild-type zebrafish cdipt cDNA <br />                                            (accession number BC155271) subcloned XhoI SacII sites yeast <br />                                            episomal shuttle vector pDR195 (Rentsch et al. 1995). This plasmid, pDR195 <br />                                            vector control, carry selectable URA3 marker transformed CTYD1 <br />                                            PIS1/pis1&#206;&#8221;&#226;&#710;&#183;KanMx heterozygous strain using lithium acetate method. The desired <br />                                            transformants recovered selection uracil prototrophy. <br />  <br />                                            CTYD1 PIS1/pis1&#206;&#8221;&#226;&#710;&#183;KanMx heterozygous diploids induced sporulate incubation <br />                                            2% potassium acetate supplemented amino acids 25&#194;&#176;C days. Four-spore <br />                                            tetrads visually identified light microscope, individually dissected by <br />                                            micromanipulation, individual spores dispersed ordered arrays solid YPD <br />                                            medium standard methods. The individual meiotic progeny derived four-spore <br />                                            germinated grew colonies 25&#194;&#176;C days. For genotyping, meiotic <br />                                            progeny replica plated minimal media dropout plates lacking uracil, histidine, <br />                                            adenine, tryptophan, YPD plates supplemented G418 not. In <br />                                            addition, genomic DNA extracted individual colonies PCR amplified to <br />                                            confirm PIS1 pis1&#206;&#8221;&#226;&#710;&#183;KanMx genotypes using primer sets: wild-type <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PIS1 allele primers (Forward: 5&#226;&#8364;&#178;-GCCCCTCCTATTGTTTTTTCCG-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            GCCATGGTTCCGTCTAGCGCATCC-3&#226;&#8364;&#178;), PIS1 5&#226;&#8364;&#178; UTR (Forward: 5&#226;&#8364;&#178;- <br />                                            GCCCCTCCTATTGTTTTTTCCG-3&#226;&#8364;&#178;) KanMX cassette (Reverse: 5&#226;&#8364;&#178;- <br />                                            CTCTACAGGGGCGCGGCGTGGGG-3&#226;&#8364;&#178;) primers used diagnose pis1&#206;&#8221;&#226;&#710;&#183;KanMx <br />                                            allele, zebrafish cdipt gene primers (Forward: 5&#226;&#8364;&#178;- <br />                                            CCGAAGAAAATATATTCCTGTTCG-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            GGAACAGAAAAGTGTAGGACGGG-3&#226;&#8364;&#178;). <br />  <br />                               2.6. Morpholino-mediated protein knockdown lop phenocopy <br />                                            Two lissamine-tagged anti-sense morpholino oligomers (Gene Tools; Philomath, OR) were <br />                                            designed target cdipt gene. The translation-blocking morpholino (MO1, 5&#226;&#8364;&#178;- <br />                                            ATGATGCCGAGAGTTTCTTTCTTTG-3&#226;&#8364;&#178;) anneals 5&#226;&#8364;&#178; untranslated region the <br />                                            beginning cdipt open reading frame. The splice-inhibiting morpholino (MO2, 5&#226;&#8364;&#178;- <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 5 <br />  <br />  <br />                                            CGAGCTGCAAAAGACCTTGATTGAG-3&#226;&#8364;&#178;) anneals exon 2-intron 2 junction the <br />                                            cdipt pre-mRNA. A control morpholino (5-mis MO2) used contained 5 mismatched <br />                                            bases relative experimental MO2 morpholino (5&#226;&#8364;&#178;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            CGAcCTcCAAAAcACgTTcATTGAG-3&#226;&#8364;&#178;). Morpholinos resuspended nuclease-free <br />                                            water 0.5 mM working concentrations microinjected 1-4 cell stage embryos as <br />                                            previously described (Thummel et al. 2004; Wei et al. 2004; Shi et al. 2005). Both MO1 and <br />                                            MO2, 5-mis MO2, produced phenotypes similar cdiptlop lens retinal <br />                                            phenotypes. <br />  <br />                                            RT-PCR evaluated efficacy cdipt pre-mRNA splice inhibition MO2 morphants <br />                                            (Draper et al. 2001). A forward primer (5&#226;&#8364;&#178;-CTCTCGGCATCATGACCGAAG-3&#226;&#8364;&#178;) a <br />                                            reverse primer (5&#226;&#8364;&#178;-GTGTAGGACGGGTACAGCAGTGCC-3&#226;&#8364;&#178;) anneals sequences <br />                                            exons 1 3, respectively, used amplify cDNA pools wild-type and <br />                                            morphant embryos 3 dpf. <br />  <br />                               2.7. cdipt mRNA microinjections <br />                                            PCR primers (Forward: 5&#226;&#8364;&#178;-CTCGAGGAAGACAATTCAGGGAGCTCCG-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            AGATCTCAGCTTCATCTCGCTGTCAGGA-3&#226;&#8364;&#178;) designed internal XhoI and <br />                                            BglII restriction sites facilitate subcloning zebrafish cdipt open reading frame the <br />                                            pXT7 vector (courtesy Dr. Len Zon) containing 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; UTR h-globin gene. <br />                                            Recombinant plasmids sequenced verify identities (Sequetech; Mountain View, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            CA) linearized vitro transcribed according manufacturer&apos;s protocol <br />                                            (mMessage mMachine, Ambion; Austin, TX). The vitro synthesized capped mRNA <br />                                            quantified using NanoDrop spectrophotometer (Thermo Scientific; Wilmington, DE). <br />                                            Embryos heterozygous cdiptlop/+ pairmating microinjected 1-4 cell stage <br />                                            50 ng/&#206;&#188;l vitro synthesized cdipt mRNA nuclease-free water previously <br />                                            described (Thummel et al. 2004; Wei et al. 2004; Shi et al. 2005). Embryos were <br />                                            stereoscopically analyzed 7 dpf lop lens phenotype. A minimum 44 fish each <br />                                            type (uninjected, water-injected mRNA-injected) examined. <br />  <br />                               2.8. PCR genotyping <br />                                            Adult cdipthi559/+ heterozygous mutants genotyped PCR amplification. To extract <br />                                            genomic DNA, clipped fins homogenized 0.4 M NaCl/10 mM Tris-HCl (pH 8.0)/2 <br />                                            mM EDTA (pH 8.0) adding 40 &#206;&#188;L 20% SDS 160 &#206;&#188;g Proteinase K (New <br />                                            England Biolabs, Inc.; Ipswich, MA). Samples incubated inversion 65&#194;&#176;C 3 <br />                                            hours microcentrifuged 30 minutes. Supernatant transferred new tube <br />                                            microcentrifuged 5 minutes. DNA precipitated supernatant 100% <br />                                            ethanol -80&#194;&#176;C 1 hour, followed microcentrifugation 20 minutes. The DNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pellet washed 70% ethanol, microcentrifuged 5 minutes, vacuum-dried 15 <br />                                            minutes resuspended nuclease-free water. The resulting genomic DNA samples were <br />                                            PCR amplified using primer pair (Forward primer MSL4, 5&#226;&#8364;&#178;- <br />                                            GCTAGCTTGCCAAACCTACAGGT-3&#226;&#8364;&#178;; Reverse primer 559 TJU, 5&#226;&#8364;&#178;- <br />                                            GTCAAACCAACCGATGTCACAATC-3&#226;&#8364;&#178;) 38 cycles 94&#194;&#176;C 30 seconds, 60&#194;&#176;C for <br />                                            30 seconds, 68&#194;&#176;C 1 minute final termination step 68&#194;&#176;C 10 minutes. The <br />                                            cdipthi559 allele produced 191 <span id='am-474' about='xsp:base' typeof='owl:Thing'>base</span> pair product corresponding small region the <br />                                            retroviral insertion intron cdipt gene. <br />  <br />                                            To distinguish cdipthi559/+ heterozygotes cdipthi559/hi559 homozygotes, individual <br />                                            larval bodies removed posterior head genotyping 3, 4, 5, 6 dpf, while <br />                                            corresponding heads processed histology immunohistochemistry. The <br />                                            MSL4 559 TJU primers amplified 191 bp product cdipthi559 mutant allele. A <br />                                            second forward primer (pp2: 5&#226;&#8364;&#178;-CTGCAGTGCGCGGTGCGAG-3&#226;&#8364;&#178;), combination with <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                            Page 6 <br />  <br />  <br />                                            559 TJU reverse primer, amplified 330 bp product wild-type allele was <br />                                            present cdipthi559/+ heterozygotes cdipthi559/hi559 homozygotes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.9. Immunohistochemistry <br />                                            Larvae fixed 9:1 ethanolic formaldehyde 4&#194;&#176;C overnight (Vihtelic et al. 2001; <br />                                            Vihtelic et al. 2005; Harding et al. 2008). Tissues rehydrated ethanol series <br />                                            washed 5% sucrose/PBS embedding 1.5% agarose/5% PBS and <br />                                            cryoprotecting 30% sucrose/PBS overnight 4&#194;&#176;C. The embedded fish infiltrated <br />                                            tissue freezing medium (TFM, Triangle Biomedical Sciences; Durham, NC) 30% <br />                                            sucrose/PBS (1:1) 4 hours room temperature, followed second infiltration with <br />                                            TFM 30% sucrose/PBS (2:1) room temperature 4 hours. Finally, embedded <br />                                            tissues incubated 100% TFM 4&#194;&#176;C overnight. 12 &#206;&#188;m tissue sections were <br />                                            taken dried SuperFrost Plus slides (Fischer Scientific; Pittsburgh, PA) 42&#194;&#176;C 2 <br />                                            hours storing -80&#194;&#176;C. Upon use, tissue sections rehydrated PBS 20 <br />                                            minutes, blocked PBS/2% normal goat serum/1% DMSO/0.3% Triton X-100 1 hour at <br />                                            room temperature, incubated overnight primary antibody diluted blocking buffer <br />                                            (anti-PI synthase diluted 1:250, anti-&#206;&#177;A crystallin diluted 1:500). The sections washed <br />                                            PBS/0.5% Tween-20 (3&#195;&#8212; 10 minutes), incubated 1 hour goat anti-rabbit IgG <br />                                            AF594-conjugated secondary antibody (2 mg/ml, Molecular Probes; Eugene, OR) diluted <br />                                            1:500 blocking buffer, washed PBS/0.5% Tween-20 (3&#195;&#8212; 10 minutes) before <br />                                            final PBS rinse 5 minutes. Cover slips mounted ProLong Gold (Invitrogen, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Carlsbad, CA) imaged using Leica TCS SP2 laser scanning confocal microscope. A <br />                                            minimum fish age genotype (n = 12 eyes) examined each <br />                                            antibody. <br />  <br />                                            The Apoptosis Detection Kit (R&amp;D Systems, Minneapolis, MN) used assay cell <br />                                            death wild-type cdipthi559/hi559 frozen tissue sections 3, 4 5 dpf previously <br />                                            described (Vihtelic et al. 2006). Larvae fixed overnight ethanolic formaldehyde <br />                                            (9:1) processed cryosectioning described above. After rehydration PBS, <br />                                            Neuropore (R&amp;D Systems) used permeablize tissues TUNEL-labeled nicked <br />                                            nucleic acid detected using streptavidin-fluorescein conjugate. A minimum fish <br />                                            age genotype (n = 12 eyes) examined. <br />  <br />                               2.10. Reverse transcriptase-polymerase chain reaction <br />                                            Reverse transcriptase-polymerase chain reaction (RT-PCR) amplification performed <br />                                            essentially described previously (Shi et al. 2006). RNA isolated 3, 12, 18, 24, <br />                                            36, 48 hpf embryos (RNA-aqueous-4PCR, Ambion; Austin, TX) cDNA was <br />                                            synthesized (First Strand cDNA Synthesis, Invitrogen). RNA isolated cDNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            adult brain, caudal fin, mixture internal organs, ocular lens, anterior eye <br />                                            segments lens, posterior eye segments retina, retina. The <br />                                            cDNA pools amplified primers annealing exon 3&#226;&#8364;&#178; UTR of <br />                                            cdipt gene (Forward: 5&#226;&#8364;&#178;- GGGAGCCACTAGTCACAAGGCC-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            GGACGAAAATAATGCCAAAGCC-3&#226;&#8364;&#178;) 38 (embryonic larval tissues) 35 <br />                                            (adult tissues) cycles 94&#194;&#176;C 30 seconds, 60&#194;&#176;C 30 seconds, 68&#194;&#176;C 1 minute a <br />                                            final termination step 68&#194;&#176;C 10 minutes. &#206;&#178;-actin amplified control (Forward: <br />                                            5&#226;&#8364;&#178;-TCAAACGAACGACCAACC-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;-AGACACCCTGGCTTACAT-3&#226;&#8364;&#178;), and <br />                                            negative control reactions lacked reverse transcriptase cDNA template. The PCR <br />                                            products visualized agarose gel electrophoresis. <br />  <br />                               2.11. Anti-PI synthase polyclonal antiserum generation <br />                                            An anti-PI synthase polyclonal antisera generated C-terminal zebrafish PI <br />                                            synthase peptide (NP_996971; SRNMAAIDVADREKERSKAQ) synthesized in <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 7 <br />  <br />  <br />                                            vitro, conjugated KLH, injected rabbits using standard immunization protocol <br />                                            (Proteintech Group; Chicago, IL). The PI synthase antisera purified affinity <br />                                            chromatography. The genomic region corresponding amino acids the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cdipt-encoded protein PCR amplified primers possessing internal BamHI XhoI <br />                                            restriction sites (Forward: 5&#226;&#8364;&#178;-GGATCCCTCATCACCGCTTCC-3&#226;&#8364;&#178;; Reverse: 5&#226;&#8364;&#178;- <br />                                            CTCGAGCTATGTGCTCATGTC-3&#226;&#8364;&#178;) cloned TOPO vector (Invitrogen). The <br />                                            fragment subcloned pET32a expression vector (Novagen; San Diego, CA) and <br />                                            expression fusion protein induced IPTG BL21 (DE3) cells (Novagen). <br />                                            Gel-purified PI synthase fusion protein covalently coupled column matrix <br />                                            (AminoLink Plus Immobilization, Pierce; Rockford, IL). Crude anti-PI synthase serum was <br />                                            passed previously generated non-specific column matrix according the <br />                                            manufacturer&apos;s protocol. The resulting flow-through fraction affinity purified the <br />                                            PI synthase-specific column. Immunoblots frozen tissue sections wild-type and <br />                                            cdipt morphants demonstrated specific knockdown PI synthase protein and <br />                                            confirmed antiserum specificity reactivity. <br />  <br />                               2.12. Immunoblot analysis <br />                                            Protein isolated 20 embryos experimental group: 6 dpf wildtype, 6 dpf <br />                                            cdipthi559/hi559 mutants, 7 dpf wild-type, 7 dpf cdiptlop/lop mutants, 7 dpf cdiptlop/hi559 <br />                                            trans-heterozygotes. Tissues homogenized 3 &#206;&#188;l extraction buffer/embryo, which <br />                                            consisted PBS/10% Glycerol/1 % Triton X-100/5 mM KPO4/0.05 mM EDTA/1x <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Complete Protease Inhibitor Cocktail Tablet (Roche; Basel, Switzerland). Immunoblot <br />                                            analysis performed previously described (Vihtelic et al. 1999; Shi et al. 2005). <br />                                            Protein aliquots combined sample loading buffer reducing agent, heated to <br />                                            90&#194;&#176;C 5 minutes, electrophoresed 15% SDS-PAGE gel according a <br />                                            standard protocol (Novex Tris-Glycine Gel Systems, Invitrogen). Proteins transferred <br />                                            PVDF H-Bond membrane (Amersham; Piscataway, NJ) blocked PBS/0.1% <br />                                            Tween-20/5% nonfat dry milk hour. The membrane incubated either <br />                                            anti-PI synthase antibody (1:250) anti-actin monoclonal antibody (1:10,000, <br />                                            Calbiochem; San Diego, CA) overnight 4&#194;&#176;C blocking buffer. The membrane was <br />                                            washed PBS/0.1% Tween-20, incubated anti-rabbit anti-mouse HRP- <br />                                            conjugated secondary antibody (1:10,000, Amersham). The ECL-Plus (Amersham) <br />                                            used detect secondary antibody described (Vihtelic et al. 1999). <br />  <br />                               3. Results <br />                               3.1. The zebrafish lop mutation cdipt missense allele <br />                                            The zebrafish lens opaque (lop) mutant previously identified ENU-induced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutagenesis screen eye morphological mutants (Vihtelic Hyde 2002; Vihtelic et al. <br />                                            2005). The homozygous lop mutant exhibits lens opacity, cataract, 7 days post <br />                                            fertilization (dpf) relative wild-type siblings (Fig. 1, panels B A, respectively; <br />                                            Vihtelic et al. 2005). At 7 dpf, coronal sections lop mutant eyes revealed lens <br />                                            secondary fiber cells lacked structural organization neighboring lens cortex and <br />                                            contained intracellular organelles (Fig. 1D, arrow). The lop mutant lens epithelial cells <br />                                            distal surface lacked characteristic monolayer organization cuboidal <br />                                            morphology (Fig. 1D, double arrowheads). While wild-type larvae possessed full <br />                                            complement mature photoreceptor cells, lop mutant retina lacked definitive <br />                                            photoreceptor layer (Fig. 1, panels C D, respectively, single arrowhead). Additionally, <br />                                            lop eye was, general, smaller wild type eye. The lop mutant, however, was <br />                                            phenotypically indistinguishable wild-type siblings 4 dpf, suggesting early eye <br />                                            development occurred normally (Vihtelic et al. 2005). <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 8 <br />  <br />  <br />                                            Recombination mapping placed lop mutation chromosome 3, interval on <br />                                            DKEY-183N6 (Supplemental Fig. 1). This region contains seven transcriptional units, <br />                                            including cdipt gene (CDP-diacylglycerol:myo-inositol 3-phosphatidyltransferase). The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cdipthi559/hi559 mutation, identified retroviral-insertional mutagenesis screen <br />                                            (Amsterdam et al. 2004), possesses retroviral insertion intron cdipt gene <br />                                            (Fig. 1L). While cdipthi559/hi559 mutant histologically similar cdiptlop/lop <br />                                            mutant smaller size like cdiptlop/lop mutant (Fig.1, panels F D, <br />                                            respectively), notable differences. First, cdipthi559/hi559 mutant <br />                                            exhibited lens opacity 5 6 dpf, 7 dpf cdiptlop/lop mutant (data <br />                                            shown). Second, cdipthi559/hi559 mutant exhibited histological defects, a <br />                                            disruption lens organization loss photoreceptor layer 6 dpf instead 7 <br />                                            dpf cdiptlop/lop mutant (Fig. 1, panels E D, respectively). Finally, the <br />                                            cdipthi559/hi559 lens degenerated rapidly cdiptlop/lop mutant (Fig. 1, panels F <br />                                            D, respectively). Thus, cdipthi559/hi559 mutant appeared possess severe <br />                                            mutant phenotypes cdiptlop/lop mutant. <br />  <br />                                            To determine lop cdipt allele, cdiptlop/hi559 trans-heterozygotes generated and <br />                                            possess comparable lens retinal phenotypes cdiptlop/lop or <br />                                            cdipthi559/hi559 mutants (Fig. 1, compare panels G H panels D-F, respectively). The <br />                                            cdiptlop/hi559 trans-heterozygous mutants exhibited lens retinal phenotypes 6 dpf like <br />                                            cdipthi559/hi559 mutant (Fig. 1, panels G E, respectively) possessed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pronounced lens degeneration 7 dpf observed cdipthi559/hi559 mutant (Fig. <br />                                            1, panels H F, respectively). The apparently severe lens phenotypes the <br />                                            cdiptlop/hi559 trans-heterozygous mutant, disruption secondary fiber cells <br />                                            6 dpf large increase cells anterior lens 7 dpf (Fig. 1, panels G H, <br />                                            respectively), phenotypic range observed the <br />                                            cdipthi559/hi559 mutant. Thus, histology cdiptlop/hi559 trans-heterozygous <br />                                            demonstrated lop mutation cdipt allele. <br />  <br />                                            We sequenced cdiptlop allele contained single missense mutation <br />                                            relative wild-type sequence, adenine thymine change position 331 third <br />                                            exon (Fig. 1L). This mutation resulted serine-to-cysteine amino acid substitution the <br />                                            invariant position 111, located catalytic domain 213 amino acid <br />                                            phosphatidylinositol (PI) synthase enzyme (Supplemental Fig. 1C, red arrowhead). In <br />                                            contrast, cdipthi559 retroviral insertion intron disrupt PI synthase <br />                                            amino acid sequence near amino terminus (Fig. 1L Supplemental Fig. 1C, yellow <br />                                            arrowhead). These mutations consistent cdipthi559 allele severe than <br />                                            cdiptlop. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.2. Reduced expression cdipt-encoded protein accounts lens retinal <br />                               mutant phenotypes <br />                                            To phenocopy cdiptlop/lop mutant phenotype, lissamine-tagged antisense morpholino <br />                                            oligonucleotides complementary exon 2:intron 2 splice junction (MO2) cdipt <br />                                            gene microinjected 1-4 cell stage wild-type embryos. At 3 dpf, mRNA was <br />                                            isolated pooled lissamine-positive morphants age-matched wild-type <br />                                            control larvae cDNA generation, independent pools used histology. RT- <br />                                            PCR primers used detect correctly spliced wild-type cDNA splice- <br />                                            inhibited morphant cDNA (Fig. 1K, 330 bp 196 bp, respectively). At 7 dpf, cdipt <br />                                            MO2 morphant possessed small eye, nucleated cells lens nucleus, disorganized <br />                                            secondary lens fiber cells lens epithelial cells, small number immature <br />                                            photoreceptors lacked outer segments central retina (Fig. 1J). None these <br />                                            phenotypes observed wild-type eye 5-base mismatch cdipt MO2 <br />                                            (cdipt 5-mis MO2) morphant eye (Fig. 1, panels C I, respectively). However, MO2 <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 9 <br />  <br />  <br />                                            morphant exhibited smaller eye cdipt mutants, severe retinal <br />                                            phenotype cdipt mutants (centrally-located immature photoreceptors compared to <br />                                            photoreceptors), different lens phenotype cdipt mutants. Therefore, we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            knocked PI synthase using independent translation-blocking morpholino (MO1; <br />                                            Supplemental Fig. 2B), resulted morphant eye phenotypically similar to <br />                                            cdipt MO2 morphant retina (Supplemental Fig. 2D Fig. 1J, respectively). While both <br />                                            morphants exhibited eye phenotypes (small eye, centrally-located immature photoreceptors) <br />                                            observed cdipt mutant eyes (Fig. 1, panels D, F, H), the <br />                                            presence retinal lens phenotypes morphants suggested the <br />                                            cdiptlop mutant phenotypes defect PI synthase. <br />  <br />                                            To support loss PI synthase caused lop mutant ocular phenotypes, we <br />                                            microinjected water 50 ng/&#206;&#188;l vitro transcribed wild-type cdipt+ mRNA into <br />                                            1-4 cell stage embryos resulting heterozygous cdiptlop/+ pairmating. At 7 dpf, both <br />                                            uninjected water-injected embryos yielded expected 25% larvae a <br />                                            heterozygote pairmating recessive mutant lens phenotype (Table 1). In contrast, <br />                                            microinjection 50 ng/&#206;&#188;l cdipt+ mRNA significantly reduced percentage larvae that <br />                                            displayed mutant lens opacity phenotype relative control larvae (Table 1). These <br />                                            data confirmed reduced expression cdipt-encoded wild-type protein results the <br />                                            mutant lens phenotype. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.3. Histological examination cdipthi559/hi559 prior gross lens opacification reveals <br />                               early retinal abnormalities <br />                                            To examine onset cdipt mutant phenotypes, PCR genotyped cdipthi559/hi559 <br />                                            individuals heterozygous cdipthi559/+ pairmatings histologically characterized the <br />                                            resulting ocular phenotypes beginning 3 dpf. Individual larvae bisected, the <br />                                            heads processed histology bodies genomic DNA isolation PCR <br />                                            genotyping. Due retroviral insertion cdipthi559 allele, PCR amplification of <br />                                            genomic DNA unambiguously identified homozygous wild-type, heterozygous carriers and <br />                                            homozygous mutant larvae based presence 330 bp wild-type and/or 191 bp <br />                                            cdipthi559 product (Fig. 2A). <br />  <br />                                            The cdipthi559/+ heterozygous phenotype indistinguishable wild-type <br />                                            phenotype time points analyzed (data shown). At 3 dpf, cdipthi559/hi559 <br />                                            homozygous mutant eye possessed retinal layers similar wild-type eyes (Fig. 2, <br />                                            panels C B, respectively). Unlike wild-type retina, however, rods cones in <br />                                            cdipthi559/hi559 mutant disorganized contained small, darkly stained nuclei (Fig. <br />                                            2, panels E D, respectively), represent pyknotic nuclei. The cdipthi559/hi559 <br />                                            lens, similar wild-type (Fig. 2, panels G F, respectively), contained organelle-free <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            nucleus, possessed differentiating secondary fiber cells, intact monolayer of <br />                                            cuboidal-shaped lens epithelial cells (Fig. 2, panels F G, double arrowheads). This <br />                                            suggests early events required lens development, delamination lens <br />                                            mass, differentiation primary secondary fiber cells, initial proliferation the <br />                                            secondary fiber cells occurred normally. Thus, earliest histological defects the <br />                                            cdipthi559/hi559 eye detected photoreceptor layer 3 dpf. <br />  <br />                                            By 5 dpf, wild-type rods cones differentiated begun extend outer <br />                                            segments (Fig. 3C, single arrowhead), cdipthi559/hi559 mutant lacked definitive <br />                                            photoreceptor layer, rod cone nuclei missing (Fig. 3D, single arrowhead). This <br />                                            consistent absence photoreceptors transmission electron microscopy <br />                                            level 7 dpf cdiptlop mutant retina (Vihtelic et al. 2005). At 5 dpf, cdipthi559/hi559 <br />                                            mutant lens phenotype appeared. Secondary fiber cell differentiation lens epithelial <br />                                            cell organization disrupted relative control eyes (Fig. 3, panels F E, <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                Page 10 <br />  <br />  <br />                                            respectively). However, cdipthi559/hi559 lens nucleus remained intact free of <br />                                            intracellular organelles similar control lenses (Fig. 3, panels F E, respectively). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We previously demonstrated cdiptlop mutant lacked rhodopsin, UV opsin, and <br />                                            red opsin expression 7 dpf based immunolocalization (Vihtelic et al. 2005). In contrast, <br />                                            HuC/D-positive amacrine ganglion cells, PKC-positive bipolar cells, glutamine <br />                                            synthetase-positive M&#195;&#188;ller glia present normal number location cdiptlop <br />                                            mutant retina 7 dpf (Vihtelic et al. 2005). We examined absence photoreceptor <br />                                            cells cdipthi559/hi559 mutant histological sections accurate. By 6 dpf, <br />                                            immunohistochemical staining confirmed rhodopsin-positive rods, blue opsin- <br />                                            expressing long single cones ultraviolet opsin-expressing cones absent the <br />                                            cdipthi559/hi559 retinas (Fig. 4, panels E, F G, respectively), present wild-type <br />                                            retinas (Fig. 4, panels A, B C, respectively). Furthermore, Zpr-1, corresponds to <br />                                            arrestin 3-like protein expressed double cone cell bodies (Ile et al. 2010), was <br />                                            detected wild-type retinas, cdipthi559/hi559 mutant retinas (Fig. 4, panels D and <br />                                            H, respectively). Consistent previous described cdiptlop mutant retina <br />                                            (Vihtelic et al. 2005) retinas possessing dying photoreceptors, cdipthi559/hi559 <br />                                            mutant possesses PCNA-positive cells retinal margin inner outer <br />                                            nuclear layers central retina, retinal margin wild-type eyes (Fig. 4, <br />                                            panels E A, respectively). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We examined additional cells associated cdipthi559/hi559 mutant <br />                                            lens derived lens tissue. We immunostained wild-type and <br />                                            mutant lenses monoclonal antibody Zl-1, expressed wild-type secondary <br />                                            lens fiber cells (Fig. 4B, arrowhead). We Zl-1 antigen present the <br />                                            disorganized cells associated cdipthi559/hi559 mutant lens (Fig.4F arrowhead). This <br />                                            suggests mass cells associated cdipthi559/hi559 mutant lens express <br />                                            secondary lens fiber cell proteins. <br />  <br />                               3.4. Photoreceptor cell death precedes lens cell apoptosis cdipthi559/hi559 mutant <br />                                            To investigate basis photoreceptor cell loss lens cell disruption (Fig. 5), progeny <br />                                            cdipthi559/+ heterozygous pair-mating analyzed cell death. Frozen sections <br />                                            immunolabeled TUNEL &#206;&#177;A-crystallin <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> apoptotic cells newly <br />                                            differentiated secondary lens fiber cells, respectively. At 3 dpf, significant <br />                                            differences wild-type cdipthi559/hi559 mutant retinas lenses (Fig. 5, panels A <br />                                            B, respectively). At 4 dpf, cdipthi559/hi559 larvae displayed increased number of <br />                                            TUNEL-positive cells retinal photoreceptor layer relative control retina <br />                                            (Fig. 5, panels D C, respectively; arrows), significant differences the <br />                                            mutant wild-type lenses. However, large increase number TUNEL-positive <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cells observed mutant retina lens 5 dpf (Fig. 5F, arrows and <br />                                            arrowheads, respectively) relative wild-type (Fig. 5E), suggesting observed <br />                                            photoreceptor cell loss attributed cell death developmental absence. &#206;&#177;A- <br />                                            crystallin labeling revealed relatively wild-type appearing population new secondary <br />                                            fiber cells cdipthi559/hi559 mutant larvae 4 dpf (Fig. 5D), started exhibit <br />                                            perturbations TUNEL-positive cells appeared 5 dpf (Fig. 5F). This suggested the <br />                                            mutant lens initially developed normally, does distinguish secondary fiber cell <br />                                            death disrupts lens cell organization perturbing secondary fiber cells arrangement <br />                                            leads death 4 5 dpf. <br />  <br />                               3.5. Spatial temporal analysis cdipt expression <br />                                            The temporal expression wild-type cdipt gene examined RT-PCR polyA+ <br />                                            mRNA isolated early development. The 877 bp cdipt RT-PCR product amplified <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 11 <br />  <br />  <br />                                            3 hpf suggested cdipt transcript maternally inherited (Fig. 6A), because <br />                                            zygotic transcription does begin timepoint. The cdipt product also <br />                                            detected 48 hpf (Fig. 6A), demonstrating gene continued expressed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            early development. Additionally, RT-PCR product observed wide <br />                                            variety adult tissues, including lens retina (Fig. 6B), suggesting broad <br />                                            temporal spatial expression patterns. We examined mutants morphants <br />                                            exhibited broad phenotypic abnormalities 6 (cdipthi559/hi559) 7 dpf (cdiptlop/lop, <br />                                            MO1, MO2). While mutants morphants exhibited small eyes, cloudy and <br />                                            defective lenses, abnormal jaw structures, general brain structures appeared be <br />                                            relatively normal (Supplemental Figure 3). While broad developmental abnormalities may <br />                                            apparent histological level, cellular subcellular defects exist in <br />                                            mutants morphants. <br />  <br />                                            To examine expression PI synthase protein, generated rabbit anti-PI synthase <br />                                            polyclonal antiserum. To confirm specificity anti-PI synthase polyclonal <br />                                            antiserum, performed immunoblot examine expression deduced 24 kDa <br />                                            PI synthase protein. The anti-PI synthase antiserum recognized 21 kDa protein on <br />                                            immunoblots 6 7 dpf wild-type extracts, detected the <br />                                            cdipthi559/hi559 cdiptlop/lop mutants trans-heterozygote (Fig. 7A). This suggests that <br />                                            mutant PI synthase proteins unstable rapidly degraded. Alternatively, <br />                                            cdipthi559/hi559 mutant mRNA undergo nonsense-mediated decay, also <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            lead significant reduction mutant protein. The broad spatial distribution the <br />                                            cdipt transcript (Fig. 6B) confirmed immunolocalizing protein brain <br />                                            (asterisk), retina (double asterisk), lens (arrowhead) wild-type head 6 7 dpf <br />                                            (Fig. 7, panels B D, respectively). In contrast, cdipthi559/hi559 cdiptlop/lop <br />                                            tissue sections (Fig. 7, panels C E, respectively) possessed reduced PI synthase <br />                                            immunostaining brain retinal tissues (asterisk double asterisk, respectively) <br />                                            relative wild-type tissue sections. The residual immunostaining mutant lens <br />                                            (Fig. 7, panels C E, arrowheads) non-specific fluorescence the <br />                                            damaged tissue based negative controls lacked primary antisera (Fig. 7E&#226;&#8364;&#178;). Taken <br />                                            together, cdipt exhibits broad temporal spatial expression patterns the <br />                                            developing zebrafish embryo adulthood. <br />  <br />                                            We used polyclonal antiserum immunolocalize PI synthase protein the <br />                                            developing wild-type eye. At 24 hpf, PI synthase detected lens placode (Fig. <br />                                            8A). <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> 36 72 hpf, PI synthase detected low levels developing retina <br />                                            adjacent brain (Fig. 8, panels B-E), including differentiating photoreceptor layer <br />                                            (Fig. 8, panels D E, arrow). At time points, PI synthase detected a <br />                                            higher level developing lens nucleus (Fig. 8, panels B C), followed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            preferential expression lens secondary fiber cells lens epithelial cells (Fig. 8, <br />                                            panels D E, double arrowheads). By 7 dpf, expression retina, including the <br />                                            photoreceptors, increases begins approximate level expression lens <br />                                            secondary fiber cells (Fig. 8F). To confirm broad retinal lens immunostaining <br />                                            patterns specific PI synthase antiserum, 7 dpf wild-type eye stained in <br />                                            absence primary anti-PI synthase polyclonal antiserum showed signal the <br />                                            retina weak background fluorescence secondary fiber cells (Fig. 8G, <br />                                            exposure settings panel F). <br />  <br />                                            In adult eye, PI synthase expression present outer nuclear layer rods and <br />                                            adjacent cones inner nuclear layer secondary neurons (Fig. 8, panels H I). In <br />                                            adult lens, PI synthase detected nucleus recent secondary <br />                                            lens fiber cells (Fig. 8J). To confirm lens immunostaining <span id='am-475' about='xsp:pattern' typeof='owl:Thing'>pattern</span> specific the <br />                                            PI synthase antiserum, wild-type lens stained absence primary anti-PI <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                          Page 12 <br />  <br />  <br />                                            synthase polyclonal antiserum showed weak background fluorescence secondary <br />                                            fiber cells nucleus (Fig. 8K, exposure settings panel J). Thus, PI <br />                                            synthase expression begins early development eye persists the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            retina lens adulthood. <br />  <br />                               3.6. The zebrafish cdipt gene encodes active PI synthase <br />                                            To demonstrate zebrafish cdipt gene encodes functional PI synthase, examined <br />                                            zebrafish cdipt expression rescue lethality associated deletion the <br />                                            single essential yeast PI synthase gene (PIS1). This heterologous yeast phenotypic <br />                                            complementation assay involved expressing zebrafish cdipt gene episomal <br />                                            pDR195 plasmid heterozygous PIS1/pis1&#206;&#8221;&#226;&#710;&#183;KanMx diploid yeast monitoring fate <br />                                            pis1&#206;&#8221;&#226;&#710;&#183;KanMx meiotic progeny inherited expression plasmid (phenotypically <br />                                            marked Ura prototrophy &#226;&#8364;&#8220; Ura+). <br />  <br />                                            All four-spore tetrads analyzed PIS1/pis1&#206;&#8221;&#226;&#710;&#183;KanMx heterozygous diploid <br />                                            strain containing pDR195 control vector exhibited segregation pattern 2 viable:2 <br />                                            nonviable meiotic progeny YPD medium (Fig. 9A, row). Additionally, the <br />                                            viable progeny (PIS1+) G418s (Fig. 9A, row) &#226;&#8364;&#8220; reporting viable <br />                                            progeny failed inherit pis1&#206;&#8221;&#226;&#710;&#183;KanMx allele (Fig. 9B). These data confirmed that <br />                                            pis1&#206;&#8221;&#226;&#710;&#183;KanMx haploid lethal allele <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> indifferent inheritance <br />                                            pDR195 control vector. In contrast, four-spore tetrads carrying the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pDR195/cdipt vector exhibited segregation pattern 4 viable: 0 nonviable meiotic <br />                                            progeny YPD medium (Fig. 9A). In cases, 2:2 segregation G418r:G418s and <br />                                            4:0 segregation Ura+:Ura- phenotypes recorded. Indeed, tetrads <br />                                            exhibited proper Mendelian segregation markers crosses, demonstrating <br />                                            fidelity meioses. Notably, remaining tetrads, displayed a <br />                                            segregation 3 viable:1 nonviable spore. In cases, viable <br />                                            segregants G418r G418r segregants Ura+. That is, G418r Ura- <br />                                            segregants recovered. The tetrads yielded 2 viable progeny apiece, all <br />                                            cases viable progeny recovered G418s &#226;&#8364;&#8220; i.e. genotypically PIS1 (data shown). <br />                                            Taken together, segregation patterns demonstrated recovery viable pis1&#206;&#8221;&#226;&#710;&#183;KanMx <br />                                            haploids co-inheritance plasmid-borne zebrafish cdipt (Fig. 9A). <br />  <br />                                            The phenotypic results confirmed PCR-based genotyping progeny derived a <br />                                            representative control tetrad, tetrad exhibited 4:0 segregation of <br />                                            viable:nonviable meiotic progeny. For PIS1/pis1&#206;&#8221;&#226;&#710;&#183;KanMx heterozygous diploid strain <br />                                            carrying pDR195, viable progeny recovered dissected tetrad genotyped as <br />                                            PIS1. As expected, failed amplify zebrafish cdipt sequence (Fig. 9B). In <br />                                            contrast, pDR195/cdipt vector present viable meiotic progeny test <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cross evidenced positive amplification zebrafish sequence the <br />                                            corresponding template samples. Moreover, pis1&#206;&#8221;&#226;&#710;&#183;KanMx allele (but PIS1) was <br />                                            recovered viable G418r segregants, PIS1 amplified DNA <br />                                            extracted G418s haploids (Fig. 9B). These collective data definitively <br />                                            demonstrate zebrafish cdipt+ expression rescued pis1&#206;&#8221;&#226;&#710;&#183;KanMx lethality. We as <br />                                            unambiguous evidence cdipt+ encodes PI synthase activity. <br />  <br />                               4. Discussion <br />                                            In study, extended analysis zebrafish lens opaque mutant (cdiptlop/lop) <br />                                            phenotype, displayed lens opacity photoreceptor apoptosis 7 dpf (Vihtelic et al. <br />                                            2005). We determined lop mutation corresponded cdipt allele based on: i) DNA <br />                                            sequence analysis revealed missense mutation changed conserved serine the <br />                                            PI synthase catalytic domain cysteine, ii) cdipt morphant yielded retinal lens <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 13 <br />  <br />  <br />                                            phenotypes severe lop mutant phenotype, iii) vitro transcribed <br />                                            wild-type cdipt mRNA rescued lop mutant phenotype, iv) lop mutation failed to <br />                                            complement previously identified cdipthi559 allele, contains viral insertion the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            intron cdipt gene (Amsterdam et al. 2004). As expected molecular <br />                                            data, cdiptlop allele likely hypomorphic allele, phenotypes severe <br />                                            (based age phenotypes observed) relative cdipthi559 allele. <br />  <br />                                            Surprisingly, cdipt morphants exhibited retinal lens phenotypes similar, <br />                                            severe, cdipt mutants. The extremely small eye size immature- <br />                                            looking retina morphants relative cdipt mutants attributed general <br />                                            developmental delay observed morpholinos, however, phenotype <br />                                            present cdipt 5-mis MO2 morphant retina. It possible severe <br />                                            phenotype MO1 morphant result knocking translation of <br />                                            maternally-loaded cdipt mRNA, however, does explain severe phenotype the <br />                                            splice-blocking MO2 morphant. Another possibility cdiptlop cdipthi559 are <br />                                            true null alleles morphants significantly knockdown PI synthase expression <br />                                            activity greater extent mutant. Finally, severe effects observed <br />                                            morphants combination effects. However, appearance of <br />                                            lens retinal phenotypes mutants, regardless severity, further <br />                                            support lop mutant phenotypes mutation cdipt gene. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Because eye phenotypes observed alleles, disruption lens <br />                                            development death photoreceptors, focused phenotypic characterization on <br />                                            cdipthi559 mutant unambiguously genotype cdipthi559 allele using <br />                                            PCR amplification cellular phenotypes observed. We demonstrated the <br />                                            cdipthi559/hi559 lens developed properly 3 dpf (Fig. 2G), intact lens nucleus <br />                                            composed organelle-free primary fiber cells, monolayer epithelial cells distal <br />                                            surface, newly differentiated secondary fiber cells forming lens cortex (Easter and <br />                                            Nicola 1996; Soules Link 2005; Dahm et al. 2007). The cdipthi559/hi559 mutant, <br />                                            however, displayed disorganized photoreceptor layer containing pyknotic nuclei at <br />                                            3 dpf (Fig. 2E), time wild-type zebrafish retina possesses cell types and <br />                                            laminar organization mature retina (Fig. 2D; Raymond et al. 1995; Easter and <br />                                            Nicola 1996; Schmitt Dowling 1996; Schmitt Dowling 1999). By 5 dpf, the <br />                                            cdipthi559/hi559 mutant lacked definitive outer nuclear layer (Fig. 3D). This loss of <br />                                            photoreceptor cells correlated increased number TUNEL-positive nuclei outer <br />                                            nuclear layer 5 dpf (Fig. 5). Additionally, increased cell proliferation we <br />                                            observed cdiptlop/lop mutant retina (Vihtelic et al. 2005) likely resulted this <br />                                            photoreceptor cell death, known induce proliferative regeneration response in <br />                                            adult zebrafish retina (Vihtelic Hyde 2000). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In contrast, cdipthi559/hi559 mutant lens phenotype evident 5 dpf (Fig. 3F), <br />                                            observed disorganized lens secondary fiber cells, disrupted monolayer of <br />                                            cuboidal epithelial cell nuclei, nucleated cells accumulating distal lens surface that <br />                                            lacked normal cell adhesion. The cdipt mutant possessed increased number of <br />                                            proliferating cells distal lateral lens, suggesting mutant exhibited <br />                                            misregulation epithelial cell proliferation resulted accumulation nucleated <br />                                            cells distal lens surface. Previously, suggested loss photoreceptors was <br />                                            secondary disruption lens (Vihtelic et al. 2005). However, appearance a <br />                                            retinal phenotype prior observed lens phenotype suggests retinal <br />                                            phenotype result lens defect underlying molecular defects not <br />                                            apparent. <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 14 <br />  <br />  <br />                                            The zebrafish bumper (bum) mutant, previously identified large scale ENU- <br />                                            mutagenesis screen (Heisenberg et al. 1996), exhibits phenotypes similar to <br />                                            cdipt mutant. Histology bum mutant revealed hyperproliferation of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            undifferentiated lens epithelial cells 3 dpf, resulting accumulation disorganized <br />                                            cell mass anterior chamber 4 dpf, decreased 5 dpf (Schonthaler et al. <br />                                            2010). Similarly, large increase number undifferentiated epithelial cells <br />                                            cdipthi559/hi559 mutant early 5 dpf (Figure 3F), decreased. Both the <br />                                            cdipt mutant bum mutant exhibited defects secondary fiber cell differentiation <br />                                            resulting lens cortex degeneration. At 4 dpf, bum secondary fiber cells appeared <br />                                            swollen, vacuoles appearing cortical fibers 5 dpf, similar the <br />                                            secondary fiber cell unpacking cdipt mutant (Figure 1). While cdipt mutant <br />                                            exhibited rod cone photoreceptor cell loss, bum mutant possessed retinal <br />                                            layers cell types adulthood, suggesting bum mutant phenotype truly lens <br />                                            specific. SSLP mapping localized bum mutation critical region chromosome 7 <br />                                            (Schonthaler et al. 2010), clearly ruling cdipt allele, located on <br />                                            chromosome 3. Three genes critical region bum mutation, hspa12b, <br />                                            slc8a4b cct7, sequenced potential candidates, genes contained a <br />                                            mutation open reading frames (Schonthaler et al. 2010). However, authors <br />                                            identified gene encoding protein similar human phospholipase C-&#206;&#178; (PLC&#206;&#178;) within <br />                                            critical region (Schonthaler et al. 2010). In light similarity cdipt lens <br />                                            phenotype role cdipt PI metabolism, PLC&#206;&#178; gene viable candidate for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            bum. <br />  <br />                                            We previously demonstrated transplanting cdiptlop/lop lens wild-type eye 31 <br />                                            hpf induced lens retinal defects, transplanting wild-type lens a <br />                                            cdiptlop/lop eye time yielded normal lens mutant retina (Vihtelic et al. <br />                                            2005). While result demonstrated cdipt-encoded PI synthase required <br />                                            cell autonomously photoreceptors, transplantation experiment suggested <br />                                            lens secrete signal required photoreceptor survival (Vihtelic et <br />                                            al. 2005). Phosphatidylinositol (PI) precursor seven different <br />                                            phosphoinositides function multiple signaling networks affect various cellular <br />                                            processes, including cell differentiation cell cycle regulation (Payrastre et al. 2001; <br />                                            Payrastre 2004). While cell-autonomous requirement PI biosynthesis in <br />                                            photoreceptor <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>cell survival</span> appears obvious, PI-dependent signal generated lens to <br />                                            maintain photoreceptors remains unknown. <br />  <br />                                            By rescuing lethality associated deletion single yeast PI synthase gene <br />                                            (PIS1), confirmed zebrafish cdipt gene encodes PI synthase enzyme, which <br />                                            catalyzes condensation myo-inositol CDP diacylglycerol generate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            phosphatidylinositol (Antonsson 1997; Lykidis et al. 1997; Payrastre et al. 2001; <br />                                            Vanhaesebroeck et al. 2001). In developing chick lens, increased PI synthesis was <br />                                            associated fiber cell formation (Zelenka 1980), stimulating proliferation of <br />                                            cultured chick lens epithelial cells increased rate PI turnover (Zelenka Vu 1984). <br />                                            This suggests PI metabolism regulate cell cycle mitotically active lens <br />                                            epithelial cells. <br />  <br />                                            However, known PI3 kinase mediates neuronal survival mouse retina <br />                                            (Pimentel et al. 2002), photoreceptor cell death rd mouse is, part, <br />                                            dependent inactivating PI3 kinase/Akt pathway (Portera-Cailliau et al. 1994; <br />                                            Jomary et al. 2006). Further analysis PI cycle defects cdipt mutant eyes help <br />                                            elucidate role phosphoinositides play zebrafish photoreceptor development and <br />                                            maintenance. <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                           Page 15 <br />  <br />  <br />                                            It known PI3 kinase plays critical role lens. For example, inhibiting PI3 <br />                                            kinase activity suppressed rabbit lens epithelial cell proliferation stimulated the <br />                                            synthesis early differentiation markers chick lens epithelial cells (Chandrasekher and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Sailaja 2003). Additionally, inhibiting PI3 kinase suppressed lens cell differentiation in <br />                                            primary quail lens cell culture (Weber Menko 2006), activated glycogen synthase <br />                                            kinase-3 (GSK3) drive apoptosis differentiating lens fiber cells (Weber Menko <br />                                            2006). These data implicate importance PI cycle lens cell proliferation, <br />                                            differentiation, cell survival. <br />  <br />                                            Recent studies implicated PI3 kinase pathway cataract formation (Chandrasekher <br />                                            Sailaja 2004; Cho et al. 2007; Yao et al. 2008), significant reduction PI3 <br />                                            kinase activity selenite cataracts rats (Chandrasekher Sailaja 2004). Inhibiting PI3 <br />                                            kinase activity blocked TGF-&#206;&#178;-induced epithelial-to-mesenchymal transitions (EMT) in <br />                                            mouse lens, widely implicated models cataract formation (de <br />                                            Iongh et al. 2005; Shirai et al. 2006), based reduced expression myofibroblast <br />                                            marker &#206;&#177;-smooth muscle actin maintenance cuboidal, myofibroblast-like, <br />                                            lens epithelial cell morphology (Cho et al. 2007). We observed loss cuboidal <br />                                            lens epithelial cell morphology cdipt mutants, indicating EMT event <br />                                            cdipt mutant lenses. <br />  <br />                                            Many human cataract conditions typified abnormal lens cell EMT resulting in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            fibrotic cells contribute scattering light (de Iongh et al. 2005). The fibrotic cell <br />                                            accumulation anterior surface, deregulation epithelial cell proliferation and <br />                                            differentiation, aberrant lens fiber cell differentiation characterize human anterior <br />                                            subcapsular cataract (ASC) condition (Nagamoto et al. 2000; Lovicu et al. 2004a; Lovicu et <br />                                            al. 2004b; Iongh et al. 2005). Similarly, zebrafish cdipt mutants an <br />                                            accumulation nucleated cells distal lens surface, misregulation lens epithelial cell <br />                                            proliferation, defects fiber cell differentiation. Thus, zebrafish cdipt mutants may <br />                                            serve model features ASC. Human oculocerebrorenal syndrome of <br />                                            Lowe (OCRL), caused mutation PI(4, 5)-P2 5-phosphatase, displays <br />                                            congenital cataracts demonstrates precedence altering PI cycle leading a <br />                                            human cataract condition (Attree et al. 1992). Thus, zebrafish cdipt mutants represent an <br />                                            unique opportunity model vivo role PI cycle variety diseases lead <br />                                            cataract formation. <br />  <br />                               Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                            We thank D. Bang staff Freimann Life Science Center Providing zebrafish husbandry care. <br />                                            This work funded National Institutes Health grant R01 EY014455 (T.S.V.) Center Zebrafish <br />                                            Research University Notre Dame. KEI VAB supported National Institutes Health grant <br />                                            R01 GM44530 (V.A.B.). <br />  <br />  <br />                               References <br />                                            Amsterdam A, Nissen RM, Sun Z, Swindell E, Farrington S, Hopkins N. Identification 315 genes <br />                                              essential early zebrafish development. Proc Natl Acad Sci USA. 2004; 101:12792&#226;&#8364;&#8220;12797. <br />                                              [PubMed: 15256591] <br />                                            Antonsson B. Phosphatidylinositol synthase mammalian tissues. Biochim Biophys Acta. 1997; <br />                                              1348:179&#226;&#8364;&#8220;186. [PubMed: 9370331] <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                     Page 16 <br />  <br />  <br />                                            Attree O, Olivos IM, Okabe I, Bailey LC, Nelson DL, Lewis RA, McInnes RR, Nussbaum RL. The <br />                                               Lowe&apos;s oculocerebrorenal syndrome gene encodes protein highly homologous inositol <br />                                               polyphosphate-5-phosphatase. Nature. 1992; 358:239&#226;&#8364;&#8220;242. [PubMed: 1321346] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Bankaitis VA, Malehorn DE, Emr SD, Greene R. The Saccharomyces cerevisiae SEC14 gene encodes <br />                                               cytosolic factor required transport secretory proteins yeast Golgi complex. J <br />                                               Cell Biol. 1989; 108:1271&#226;&#8364;&#8220;1281. [PubMed: 2466847] <br />                                            Barut BA, Zon LI. Realizing potential zebrafish model human disease. Physiol <br />                                               Genomics. 2000; 13:49&#226;&#8364;&#8220;51. [PubMed: 11015581] <br />                                            Bilotta J. Effects abnormal lighting development zebrafish visual behavior. Behav Brain <br />                                               Res. 2000; 116:81&#226;&#8364;&#8220;87. [PubMed: 11090887] <br />                                            Bredrup C, Matejas V, Barrow M, Blahova K, Bockenhauer D, Fowler DJ, Gregson RM, Maruniak- <br />                                               Chudek I, Medeira A, Medonca EL, Kagan M, Koenig J, Krastel H, Kroes HY, Saggar A, Sawyer <br />                                               T, Schittkowski M, Swietlinski J, Thompson D, VanDeVoorde RG, Wittebol-Post D, Woodruff G, <br />                                               Zurowska A, Hennekam RC, Zenker M, Russel-Eggitt I. Ophthalmological aspects Pierson <br />                                               syndrome. Am J Ophthalmol. 2008; 146:602&#226;&#8364;&#8220;611. [PubMed: 18672223] <br />                                            Chandrasekher G, Sailaja D. Differential activation phosphatidylinositol 3-kinase signaling during <br />                                               proliferation differentiation lens epithelial cells. Invest Ophthalmol Vis Sci. 2003; 44:4400&#226;&#8364;&#8220; <br />                                               4411. [PubMed: 14507886] <br />                                            Chandrasekher G, Sailaja D. Alterations lens protein tyrosine phosphorylation and <br />                                               phosphatidylinositol 3-kinase signaling selenite cataract formation. Curr Eye Res. 2004; <br />                                               28:135&#226;&#8364;&#8220;144. [PubMed: 14972719] <br />                                            Cho HJ, Baek KE, Saika S, Moon-Jin J, Jiyun Y. Snail required transforming growth factor-beta- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                induced epithelial-mesenchymal transition activating PI3-kinase/Akt signal pathway. Biochem <br />                                                Biophys Res Commun. 2007; 353:337&#226;&#8364;&#8220;343. [PubMed: 17187756] <br />                                            Colognato H, Yurchenco PD. Form function: laminin family heterotrimers. Dev Dyn. 2000; <br />                                                218:213&#226;&#8364;&#8220;234. [PubMed: 10842354] <br />                                            Dahm R, Schonthaler HB, Soehn AS, van Marle J, Vrensen GF. Development adult morphology <br />                                                eye lens zebrafish. Exp Eye Res. 2007; 85:74&#226;&#8364;&#8220;89. [PubMed: 17467692] <br />                                            Iongh RU, Wederell E, Lovicu FJ, McAvoy JW. Transforming growth factor-beta-induced <br />                                                epithelial-mesenchymal transition lens: A model cataract formation. Cells Tissues <br />                                                Organs. 2005; 179:43&#226;&#8364;&#8220;55. [PubMed: 15942192] <br />                                            Draper BW, Morcos PA, Kimmel CB. Inhibition zebrafish fgf8 pre-mRNA splicing with <br />                                                morpholino oligos: quantifiable method gene knockdown. Genesis. 2001; 30:154&#226;&#8364;&#8220;156. <br />                                                [PubMed: 11477696] <br />                                            Easter SS Jr, Malicki JJ. The zebrafish eye: developmental genetic analysis. Results Probl Cell <br />                                                Differ. 2002; 40:346&#226;&#8364;&#8220;370. [PubMed: 12353485] <br />                                            Easter SS Jr, Nicola GN. The development vision zebrafish (Danio rerio). Dev Biol. 1996; <br />                                                180:646&#226;&#8364;&#8220;663. [PubMed: 8954734] <br />                                            Fadool JM, Dowling JE. Zebrafish: A model study eye genetics. Prog Retin Eye Res. <br />                                                2008; 27:89&#226;&#8364;&#8220;110. [PubMed: 17962065] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Gietz RD, Schiestl RH, Willems AR, Woods RA. Studies transformation intact yeast cells by <br />                                                LiAc/SS-DNA/PEG procedure. Yeast. 1995; 11:355&#226;&#8364;&#8220;360. [PubMed: 7785336] <br />                                            Glass AS, Dahm R. The zebrafish model organism eye development. Ophthalmic Res. 2004; <br />                                                36:4&#226;&#8364;&#8220;24. [PubMed: 15007235] <br />                                            Greiling TMS, Clark JI. Early lens development zebrafish: three-dimensional time-lapse <br />                                                analysis. Dev Dyn. 2009; 238:2254&#226;&#8364;&#8220;2265. [PubMed: 19504455] <br />                                            Harding RL, Howley S, Baker LJ, Murphy TR, Archer WE, Wistow G, Hyde DR, Vihtelic TS. <br />                                                Lengsin expression function zebrafish lens formation. Exp Eye Res. 2008; 86:807&#226;&#8364;&#8220;818. <br />                                                [PubMed: 18406404] <br />                                            Heisenberg CP, Brand M, Jiang YJ, Warga RM, Beuchle D, van Eeden FJ, Furutani-Seikl M, Granato <br />                                                M, Haffter P, Hammerschmidt M. Genes involved forebrain development zebrafish, <br />                                                Danio rerio. Development. 1996; 123:191&#226;&#8364;&#8220;203. [PubMed: 9007240] <br />                                            Hu M, Easter SS Jr. Retinal neurogenesis: formation initial central patch postmitotic cells. <br />                                                Dev Biol. 1999; 207:309&#226;&#8364;&#8220;321. [PubMed: 10068465] <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                  Page 17 <br />  <br />  <br />                                            Ile KE, Kassen S, Cao C, Vihtelic T, Shah SD, Mousley CJ, Alb JG, Huijbregts RPH, Stearns GW, <br />                                                 Brockerhoff SE, Hyde DR, Bankaitis VA. Zebrafish Class 1 Phosphatidylinositol Transfer <br />                                                 Proteins: PITP&#206;&#178; Double Cone Cell Outer Segment Integrity Retina. Traffic. 2010; 11:1151&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 1167. [PubMed: 20545905] <br />                                            Ito H, Fukuda Y, Murata K, Kimura A. Transformation intact yeast cells treated alkali cations. <br />                                                 J Bacteriol. 1983; 153:163&#226;&#8364;&#8220;168. [PubMed: 6336730] <br />                                            Johns PR, Fernald RD. Genesis rods teleost fish retina. Nature. 1981; 293:141&#226;&#8364;&#8220;142. [PubMed: <br />                                                 7266666] <br />                                            Jomary C, Cullen J, Jones SE. Inactivation Akt survival pathway photoreceptor apoptosis <br />                                                 retinal degeneration mouse. Invest Ophthalmol Vis Sci. 2006; 47:1620&#226;&#8364;&#8220;1629. [PubMed: <br />                                                 16565401] <br />                                            Kimmel CB, Ballard WW, Kimmel SR, Ullmann B, Schilling TF. Stages embryonic development <br />                                                 zebrafish. Dev Dyn. 1995; 203:253&#226;&#8364;&#8220;310. [PubMed: 8589427] <br />                                            Knapik E, Goodman A, Ekker M, Chevrette M, Delgado J. A microsatellite genetic linkage map for <br />                                                 zebrafish (danio rerio). Nat Genet. 1998; 18:338&#226;&#8364;&#8220;343. [PubMed: 9537415] <br />                                            Li L. Zebrafish mutants: behavioral genetic studies visual defects. Dev Dyn. 2001; 221:365&#226;&#8364;&#8220; <br />                                                 372. [PubMed: 11500973] <br />                                            Lovicu FJ, Ang S, Chorazyczewska M, McAvoy JW. Deregulation lens epithelial cell proliferation <br />                                                 differentiation development TGFbeta-induced anterior subcapsular cataract. Dev <br />                                                 Neurosci. 2004a; 26:446&#226;&#8364;&#8220;455. [PubMed: 15855773] <br />                                            Lovicu FJ, Steven P, Saika S, McAvoy JW. Aberrant lens fiber differentiation anterior subcapsular <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 cataract formation: process dependent reduced levels Pax6. Invest Ophthalmol Vis Sci. <br />                                                 2004b; 45:1946&#226;&#8364;&#8220;1953. [PubMed: 15161862] <br />                                            Lykidis A, Jackson PD, Rock CO, Jackowski S. The role CDP-diacylglycerol synthetase and <br />                                                 phosphatidylinositol synthase activity levels regulation cellular phosphatidylinositol <br />                                                 content. J Biol Chem. 1997; 272:33402&#226;&#8364;&#8220;33409. [PubMed: 9407135] <br />                                            Nagamoto T, Eguichi G, Beebe DC. Alpha-smooth muscle actin expression cultured lens epithelial <br />                                                 cells. Invest Ophthalmol Vis Sci. 2000; 41:1122&#226;&#8364;&#8220;1129. [PubMed: 10752950] <br />                                            Nawrocki L, BreMiller R, Streisinger G, Kaplan M. Larval adult visual pigments zebrafish, <br />                                                 brachydanio rerio. Vision Res. 1985; 25:1569&#226;&#8364;&#8220;1576. [PubMed: 3832580] <br />                                            Neuhauss S. Behavioral genetic approaches visual development function zebrafish. J <br />                                                 Neurobiol. 2003; 54:148&#226;&#8364;&#8220;160. [PubMed: 12486702] <br />                                            Payrastre B. Phosphoinositides: lipid kinases phosphatases. Methods Mol Biol. 2004; 273:201&#226;&#8364;&#8220; <br />                                                 212. [PubMed: 15308803] <br />                                            Payrastre B, Missy K, Giuriato S, Bodin S, Plantavid M, Gratacap M. Phosphoinositides: key players <br />                                                 cell signaling, time space. Cell Signal. 2001; 13:377&#226;&#8364;&#8220;387. [PubMed: 11384836] <br />                                            Pimentel B, Rodriguez-Borlado L, Hernandez C, Carrera AC. A role phosphoinositide 3-kinase in <br />                                                 control cell division survival retinal development. Dev Biol. 2002; 247:295&#226;&#8364;&#8220;306. <br />                                                 [PubMed: 12086468] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Portera-Cailliau C, Sung CH, Nathans J, Adler R. Apoptotic photoreceptor cell death mouse models <br />                                                 retinitis pigmentosa. Proc Natl Acad Sci USA. 1994; 91:974&#226;&#8364;&#8220;978. [PubMed: 8302876] <br />                                            Raymond PA, Barthel LK, Curran GA. Developmental patterning rod cone photoreceptors in <br />                                                 embryonic zebrafish. J Comp Neurol. 1995; 359:537&#226;&#8364;&#8220;550. [PubMed: 7499546] <br />                                            Rentsch D, Laloi M, Rouhara I, Schmelzer E, Delrot S, Frommer WB. NTR1 encodes high affinity <br />                                                 oligopeptide transporter Arabidopsis. FEBS Lett. 1995; 370:264&#226;&#8364;&#8220;268. [PubMed: 7656990] <br />                                            Rothstein RJ. One-step gene disruption yeast. Methods Enzymol. 1983; 101:202&#226;&#8364;&#8220;211. [PubMed: <br />                                                 6310324] <br />                                            Schmitt EA, Dowling JE. Early eye morphogenesis zebrafish, brachydanio rerio. J Comp <br />                                                 Neurol. 1994; 344:532&#226;&#8364;&#8220;542. [PubMed: 7929890] <br />                                            Schmitt EA, Dowling JE. Comparison topographical patterns ganglion photoreceptor cell <br />                                                 differentiation retina zebrafish, danio rerio. J Comp Neurol. 1996; 371:222&#226;&#8364;&#8220;243. <br />                                                 [PubMed: 8835728] <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                  Page 18 <br />  <br />  <br />                                            Schmitt EA, Dowling JE. Early retinal development zebrafish, danio rerio: light electron <br />                                                microscopic analyses. J Comp Neurol. 1999; 404:515&#226;&#8364;&#8220;536. [PubMed: 9987995] <br />                                            Schonthaler HB, Franz-Odendaal TA, Hodel C, Gehring I, Geisler R, Schwarz H, Neuhauss SC, Dahm <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                R. The zebrafish mutant bumper shows hyperproliferation lens epithelial cells fibre cell <br />                                                degeneration leading functional blindness. Mech Dev. 2010; 127:203&#226;&#8364;&#8220;219. [PubMed: <br />                                                20117205] <br />                                            Semina EV, Bosenko DV, Zinkevich NC, Soules KA, Hyde DR, Vihtelic TS, Willer GB, Gregg RG, <br />                                                Link BA. Mutations laminin alpha 1 result complex, lens-independent ocular phenotypes in <br />                                                zebrafish. Dev Biol. 2006; 299:63&#226;&#8364;&#8220;77. [PubMed: 16973147] <br />                                            Shi X, Bosenko DV, Zinkevich NS, Foley S, Hyde DR, Semina EV, Vihtelic TS. Zebrafish pitx3 is <br />                                                necessary normal lens retinal development. Mech Dev. 2005; 122:513&#226;&#8364;&#8220;527. [PubMed: <br />                                                15804565] <br />                                            Shi X, Luo Y, Howley S, Dzialo A, Foley S, Hyde DR, Vihtelic TS. Zebrafish foxe3: roles lens <br />                                                morphogenesis interaction pitx3. Mech Dev. 2006; 123:761&#226;&#8364;&#8220;782. [PubMed: <br />                                                16963235] <br />                                            Shimoda N, Knapik EW, Zinitti J, Sim C, Yamada E. Zebrafish genetic map 2000 microsatellite <br />                                                markers. Genomics. 1999; 58:219&#226;&#8364;&#8220;232. [PubMed: 10373319] <br />                                            Shin JT, Fishman MC. From zebrafish human: modular medical models. Annu Rev Genomics Hum <br />                                                Genet. 2002; 2:311&#226;&#8364;&#8220;340. [PubMed: 12142362] <br />                                            Shirai K, Saika S, Tanaka T, Okada Y, Flanders KC, Ooshima A, Ohnishi Y. A new model anterior <br />                                                subcapsular cataract: involvement TGFbeta/Smad signaling. Mol Vis. 2006; 14:681&#226;&#8364;&#8220;691. <br />                                                [PubMed: 16807527] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Soules KA, Link BA. Morphogenesis anterior segment zebrafish eye. BMC Dev Biol. <br />                                                2005; 5:12. [PubMed: 15985175] <br />                                            Thummel R, Li L, Tanase C, Sarras MPJ, Godwin AR. Differences expression pattern function <br />                                                zebrafish hoxc13 orthologs: recruitment Hoxc13b early embryonic role. Dev <br />                                                Biol. 2004; 274:318&#226;&#8364;&#8220;333. [PubMed: 15385162] <br />                                            Vanhaesebroeck B, Leevers SJ, Ahmadi K, Timms J, Katso R, Driscoll PC, Woscholski R, Parker PJ, <br />                                                Waterfield MD. Synthesis function 3-phosphorylated inositol lipids. Annu Rev Biochem. <br />                                                2001; 70:535&#226;&#8364;&#8220;602. [PubMed: 11395417] <br />                                            Vihtelic TS, Soverly J, Kassen S, Hyde DR. Retinal regional differences photoreceptor cell death <br />                                                regeneration light-lesioned albino zebrafish. Exp Eye Res. 2006; 82:558&#226;&#8364;&#8220;575. [PubMed: <br />                                                16199033] <br />                                            Vihtelic TS. Teleost lens development degeneration. Int Rev Cell Mol Biol. 2008; 269:341&#226;&#8364;&#8220;373. <br />                                                [PubMed: 18779061] <br />                                            Vihtelic TS, Doro CJ, Hyde DR. Cloning characterization zebrafish photoreceptor opsin <br />                                                cDNAs immunolocalization corresponding proteins. Vis Neurosci. 1999; 16:571&#226;&#8364;&#8220;585. <br />                                                [PubMed: 10349976] <br />                                            Vihtelic TS, Hyde DR. Zebrafish mutagenesis yields eye morphological mutants retinal lens <br />                                                defects. Vision Res. 2002; 42:535&#226;&#8364;&#8220;540. [PubMed: 11853770] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Vihtelic TS, Yamamoto Y, Springer SS, Jeffery WR, Hyde DR. Lens opacity photoreceptor <br />                                                degeneration zebrafish lens opaque mutant. Dev Dyn. 2005; 233:52&#226;&#8364;&#8220;65. [PubMed: <br />                                                15765514] <br />                                            Vihtelic TS, Yamamoto Y, Sweeney MT, Jeffery WR, Hyde DR. Arrested differentiation and <br />                                                epithelial cell degeneration zebrafish lens mutants. Dev Dyn. 2001; 222:625&#226;&#8364;&#8220;636. [PubMed: <br />                                                11748831] <br />                                            Weber GF, Menko AS. Phosphatidylinositol 3-kinase necessary lens fiber cell differentiation and <br />                                                survival. Invest Ophthalmol Vis Sci. 2006; 47:4490&#226;&#8364;&#8220;4499. [PubMed: 17003444] <br />                                            Wei X, Cheng Y, Luo Y, Nelson S, Hyde DR. The zebrafish Pard3 ortholog required separation <br />                                                eye fields retinal lamination. Dev Biol. 2004; 269:286&#226;&#8364;&#8220;301. [PubMed: 15081374] <br />                                            Westerfield, M. The zebrafish book: guide laboratory use zebrafish (danio rerio). fourth. <br />                                                University Oregon Press; Eugene: 2000. <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                  Page 19 <br />  <br />  <br />                                            Willer GB, Lee VM, Gregg RG, Link BA. Analysis zebrafish perplexed mutation reveals tissue- <br />                                                specific roles novo pyrimidine synthesis development. Genetics. 2005; 170:1827&#226;&#8364;&#8220; <br />                                                1837. [PubMed: 15937129] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Yao K, Ye PP, Tan J, Tang XJ, Shen Tu XC. Involvement PI3K/Akt pathway TGF-beta2- <br />                                                induced epithelial mesenchymal transition human lens epithelial cells. Ophthalmic Res. 2008; <br />                                                40:69&#226;&#8364;&#8220;76. [PubMed: 18223299] <br />                                            Zelenka PS. Changes phosphatidylinositol metabolism differentiation lens epithelial cells <br />                                                lens fiber cells embryonic chick. J Biol Chem. 1980; 225:1296&#226;&#8364;&#8220;1300. [PubMed: <br />                                                7354028] <br />                                            Zelenka PS, Vu ND. Correlation phosphatidylinositol degradation cell division in <br />                                                embryonic chicken lens epithelia. Dev Biol. 1984; 105:325&#226;&#8364;&#8220;329. [PubMed: 6479442] <br />                                            Zinkevich NS, Bosenko DV, Link BA, Semina EV. laminin alpha 1 gene essential normal lens <br />                                                development zebrafish. BMC Dev Biol. 2006; 6:13. [PubMed: 16522196] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                         Page 20 <br />  <br />  <br />                                            Highlights <br />                                            The lens opaque mutation missense change PI (phosphatidylinositol) synthase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PI synthase required photoreceptor cell viability <br />                                            PI synthase necessary maintain lens epithelial cells <br />                                            PI synthase required lens cortical secondary fiber differentiation <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                               Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 1. The lop mutation cdipt allele <br />                                            Images wild-type eye transparent lens cdiptlop/lop mutant lens opacity <br />                                            (panels A B, respectively; arrow). At 7 days post-fertilization (dpf), wild-type retina <br />                                            (panel C) laminar layers: photoreceptor layer (PL), inner nuclear layer <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (INL), ganglion cell layer (GCL). The wild-type lens lens nucleus (asterisk), <br />                                            lens cortex containing tightly packed secondary fiber cells (C, arrow), monolayer of <br />                                            lens epithelial cells (LEC, double arrowheads) distal surface. In contrast, cdiptlop/lop <br />                                            (panel D), cdipthi559/hi559 (panels E F), cdiptlop/hi559 (panels G H) mutants all <br />                                            lack photoreceptor layers (single arrowhead) possess degenerated lenses abnormal <br />                                            secondary fiber cell organization (arrow) disorganized cuboidal epithelial cells the <br />                                            distal lens surface (arrowheads). Both cdipthi559/hi559 cdiptlop/hi559 lenses 7 dpf <br />                                            (panels F H, respectively) severely degenerated (smaller lens nucleus) than <br />                                            cdipthi559/hi559 cdiptlop/hi559 6 dpf (panels E G, respectively) cdiptlop/lop <br />                                            7 dpf (panel D). To phenocopy cdipt mutant, lissamine-tagged splice- <br />                                            inhibiting morpholino (MO2) 5-base mismatch control morpholino (5-mis MO2) was <br />                                            injected 1-4 cell stage wild-type embryos. At 7 dpf, 5-base mismatch morphant <br />                                            retina lens appeared normal (panel I), cdipt MO2 morphant lens (panel J) <br />                                            exhibited defects cuboidal-shaped lens epithelial cells secondary fiber cell <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                           Page 22 <br />  <br />  <br />                                            differentiation organization similar cdipt mutants. The MO2 morphant lens <br />                                            nucleus exhibited deficiencies primary fiber cell differentiation, presence <br />                                            organelle-containing primary fiber cells (panel J; asterisk). While morphant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            photoreceptor cell layer completely absent, cdipt mutants, morphant <br />                                            photoreceptors appeared immature disorganized (panel J; single arrowhead). Wild-type <br />                                            morphant larvae genotyped 3 dpf reverse transcriptase PCR. The properly <br />                                            spliced cdipt mRNA yielded 330 bp PCR product (wt) abnormally spliced cdipt <br />                                            morphant mRNA (MO2) produced 196 bp PCR product (panel K). A schematic the <br />                                            cdipt gene (panel L), containing exons (rectangles, open reading frame red and <br />                                            untranslated regions blue) introns (lines), shows relative locations the <br />                                            cdipthi559 retroviral insertion cdiptlop adenine thymine missense mutation. Scale <br />                                            bar (panel C) represents 50 &#206;&#188;m scale panels C-J. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                             Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 2. cdipthi559/hi559 mutants display photoreceptor abnormalities 3 dpf <br />                                            PCR-based genotyping identified heterozygous homozygous cdipthi559 mutants prior to <br />                                            cellular phenotypes (A). Two different pairs oligonucleotide primers (insert wt) <br />                                            amplified region cdipthi559 viral insertion allele (191 bp) wild-type gene <br />                                            (330 bp), respectively. This allowed identification wild-type, heterozygous, or <br />                                            homozygous mutant larvae. B-G: Histological analysis revealed onset the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            homozygous mutant phenotype 3 dpf. The wild-type (panels B, D, F) and <br />                                            heterozygous (data shown) eyes histologically indistinguishable. In contrast, the <br />                                            cdipthi559/hi559 mutant retina possessed photoreceptor layer contained disorganized <br />                                            rods cones pyknotic nuclei (panels C E; single arrowheads). However, the <br />                                            cdipthi559/hi559 lens (panels C G) appeared normal intact lens nucleus (asterisk), <br />                                            differentiated secondary fiber cells (arrow), ordered monolayer lens epithelial cells <br />                                            (double arrowheads). Scale bars (panels B D) represent 50 &#206;&#188;m bar panel D is <br />                                            scale panels E-G. <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                            Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 3. The cdipthi559/hi559 mutant exhibits irregular lens morphology absence of <br />                                            photoreceptors 5 dpf <br />                                            Larvae PCR-genotyped unambiguously identify genotype prior histology. <br />                                            Histological sections larval eyes 5 dpf, prior appearance gross lens opacity at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            6 dpf, analyzed. The wild-type (panels A, C, E) heterozygous (data shown) <br />                                            eyes histologically indistinguishable, normal photoreceptor cell layer <br />                                            (arrowhead) lens. The cdipthi559/hi559 mutant eye (panels B, D, F), smaller <br />                                            wild-type, lacked rods cones photoreceptors (panels B D; single <br />                                            arrowheads), consistent defects observed 3 dpf. The cdipthi559/hi559 <br />                                            mutant lens exhibited defects, including cortical secondary fiber cells failed to <br />                                            properly differentiate contained intracellular organelles (panels B F; arrows) <br />                                            lens epithelial cells lost characteristic monolayer distal surface and <br />                                            cuboidal morphology (panels B F; double arrowheads). However, cdipthi559/hi559 <br />                                            lens nucleus intact histologically similar wild-type (panels F E, respectively; <br />                                            asterisk). Scale bars (panels A C) represent 50 &#206;&#188;m bar panel A same <br />                                            scale panel B, bar panel C scale panels D-F. <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                          Page 25 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 4. The cdipthi559/hi559 mutant lacks rod cone photoreceptors <br />                                            Larvae PCR-genotyped unambiguously identify genotype prior histology. <br />                                            Frozen tissue sections larval eyes 6 dpf analyzed. The wild-type eyes exhibited a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            normal complement rhodopsin-expressing rods (panel A, green; arrow), blue <br />                                            opsinexpressing long single cones (panel B, red; arrow), ultraviolet opsin-expressing short <br />                                            single cones (panel C, green; arrow) Zpr-1-expressing double cone cell bodies (panel D, <br />                                            red; arrow). The cdipthi559/hi559 mutant retinas contained small numbers rod <br />                                            photoreceptors (panel E, green; arrow) ultraviolet opsin-expressing cones (panel G, <br />                                            green; arrow), detectable long single cones (panel F, red; arrow) Zpr-1-expressing <br />                                            double cone cells (panel H, red; arrow). PCNA-positive proliferating cells wild-type <br />                                            retina restricted circumferential marginal zone (panel A, red; arrowheads), while <br />                                            cdipthi559/hi559 retina contained PCNA-positive cells circumferential marginal <br />                                            zone retina (panel E, red; arrowheads double arrowheads, <br />                                            respectively). The wild-type lens possessed normal expression Zl-1 antigen, which <br />                                            labels secondary fiber cells (panel B, green, arrowhead). In cdipthi559/hi559 mutant, the <br />                                            Zl-1 protein localized disrupted lens cells photoreceptor layer <br />                                            (panel F, green; arrowhead arrow, respectively). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                                Page 26 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 5. The cdipthi559/hi559 mutant possesses apoptotic photoreceptor cells prior lens cell <br />                                            death <br />                                            Larvae PCR-genotyped unambiguously identify wild-type (panels A, C, E) and <br />                                            cdipthi559/hi559 mutants (panels B, D, F). Frozen tissue sections larval eyes 3 <br />                                            (panels A B), 4 (panels C D), 5 dpf (panels E F) immunostained for <br />                                            cell death (TUNEL, green) secondary fiber cells (&#206;&#177;A-crystallin, red). The wild-type eyes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            exhibited minimal numbers TUNEL-positive cells normal secondary fiber cells. <br />                                            The cdipthi559/hi559 mutants exhibited significant increase number dying cells in <br />                                            outer retina (arrows), corresponds photoreceptor cells, 4 5 dpf an <br />                                            increased number dying lens cells (arrowheads). The dying lens cells correlated a <br />                                            disruption &#206;&#177;A-crystallin immunolocalization cdipthi559/hi559 mutant retinas 5 <br />                                            dpf. The scale bar (panel A) represents 50 &#206;&#188;m scale panels B-F. <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                            Page 27 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 6. RT-PCR analysis reveals broad temporal spatial expression cdipt gene <br />                                            mRNA isolated embryos various times development different <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            adult tissues (panels A B, respectively). The mRNA used reverse transcriptase <br />                                            PCR (RT-PCR) reaction using oligonucleotide primers amplified 877 bp cdipt <br />                                            product (upper portion panel) 415 bp &#206;&#178;-actin product, served loading <br />                                            control reaction (lower portion panel). The cdipt product observed at <br />                                            times tested, 3 hpf (corresponding maternal expression) 48 hpf. The <br />                                            cdipt product detected adult samples analyzed, including: eye anterior <br />                                            segments lens (ant. seg.), eye posterior segments retina (post. seg.), <br />                                            retina, lens, various internal organs (int. organs), fin, brain. A RT-PCR reaction lacking <br />                                            reverse-transcriptase run negative control (panel A, RT(-) cntrl). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Exp Eye Res. Author manuscript; available PMC 2012 October 1. <br />                             Murphy et al.                                                                                              Page 28 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 7. cdipt mutants display reduced levels Phosphatidylinositol synthase protein <br />                                            Protein extracted wild-type cdipt mutant larvae 6 7 dpf for <br />                                            immunoblot analysis (panel A). An approximately 24 kDa protein representing PI synthase <br />                                            detected 6 7 dpf wild-type extracts, cdipthi559/hi559, cdiptlop/lop, or <br />                                            trans-heterozygote. Actin detected immunoblots loading control each <br />                                            lane. 12 &#206;&#188;m coronal frozen tissue sections wild-type cdipt mutant larvae 6 <br />                                            7 dpf immunolabeled anti-PI synthase polyclonal antisera. PI synthase protein <br />                                            detected wild-type head tissues (panels B D), including brain, retina, and <br />                                            lens 6 7 dpf (asterisk, double asterisk arrowhead, respectively). In contrast, <br />                                            brain retinal tissues (asterisk double asterisk, respectively) the <br />                                            cdipthi559/hi559 cdiptlop/lop (panels C E, respectively) did exhibit PI synthase <br />                                            expression, consistent immunoblot (panel A). Non-specific staining of <br />                                            mutant lenses (panels C E, arrowheads) confirmed similar signal tissue <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            sections lacked anti-PI synthase polyclonal antiserum (panel E&#226;&#8364;&#178;, exposure as <br />                                            panel E). The scale bar <br /> </body></html>