<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Dev Biol. 2011 July 1; 355(1): 77&#226;&#8364;&#8220;88. doi:10.1016/j.ydbio.2011.04.013. <br />  <br />  <br />  <br />                            Essential roles snap-29 C. elegans <br />                            Junsu Kang1,#, Zhiyong Bai2,#, Matthew H. Zegarek2, Barth D. Grant2,*, Junho Lee1,3,* <br />                            1 Research Center Functional Cellulomics, IMBG, School Biological Sciences, Seoul <br />  <br />                            National University, Seoul, 151-747, Korea <br />                            2Department Molecular Biology Biochemistry, Rutgers University, Piscataway, NJ 08854, <br />                            USA <br />                            3WCU Department Biophysics Chemical Biology, Seoul National University, Seoul, <br />                            151-742, Korea <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  SNARE domain proteins key molecules mediating intracellular fusion events. SNAP25 family <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  proteins unique target-SNAREs possessing SNARE domains. Here report genetic, <br />                                  molecular, cell biological characterization C. elegans SNAP-29. We snap-29 is <br />                                  essential gene required life-cycle. Depletion snap-29 RNAi adults <br />                                  results sterility associated endomitotic oocytes pre-meiotic maturation oocytes. <br />                                  Many embryos produced multinucleated, indicating defect embryonic <br />                                  cytokinesis. A profound defect secretion oocytes early embryos animals lacking <br />                                  SNAP-29 appears underlying defect connecting phenotypes. Further analysis <br />                                  revealed defects basolateral apical secretion intestinal epithelial cells animals lacking <br />                                  SNAP-29, indicating broad requirement protein secretory pathway. A SNAP-29- <br />                                  GFP fusion protein enriched recycling endosomes, loss SNAP-29 disrupted <br />                                  recycling endosome morphology. Taken results suggest requirement SNAP-29 <br />                                  fusion post-Golgi vesicles recycling endosome cargo reach cell <br />                                  surface. <br />  <br />  <br />                            Keywords <br />                                  C. elegans; snap-29; SNARE domain; Recycling endosome <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Introduction <br />                                              Soluble n-ethylmaleimide sensitive factor attachment protein receptor (SNARE) domain <br />                                              containing proteins known essential mediators intracellular fusion <br />                                              events(Sudhof Rothman, 2009). SNAREs common heptad-repeat SNARE motif <br />                                              constituting 60&#226;&#8364;&#8220;70 amino acids(Fasshauer et al., 1998). There general classes of <br />                                              SNARE proteins. target-SNAREs (t-SNAREs) reside target membranes receiving cargo <br />                                              input. vesicle-SNAREs (v-SNAREs) located transport vesicles bud donor <br />  <br />  <br />                            &#194;&#169; 2011 Elsevier Inc. All rights reserved. <br />                            * <br />                              <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed: elegans@snu.ac.kr; grant@biology.rutgers.edu. <br />                            #These authors equally contributed work. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Kang et al.                                                                                            Page 2 <br />  <br />  <br />                                          compartment interact t-SNARES mediate fusion. Some best-studied <br />                                          SNARES neuronal synapse mediate fusion neurotransmitter- <br />                                          laden synaptic vesicles synaptic plasma membrane neurotransmission. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          synaptic plasma membrane t-SNARE composed Syntaxin1 (one Snare motif) and <br />                                          SNAP25 (two Snare motifs). The cognate v-Snare synaptic vesicles known as <br />                                          synaptobrevin/VAMP (one Snare motif). According &#226;&#8364;&#732;zippering model&#226;&#8364;&#8482;, SNARE <br />                                          motifs provided t-SNARE assembles fourth SNARE motif provided the <br />                                          v-SNARE, forming trans-SNARE complex, SNAREpin, resulting close contact <br />                                          vesicle target membranes. With additional regulatory proteins, trans-SNARE <br />                                          complexes distort lipid bilayers individual membranes fuse merged <br />                                          membrane (Rizo Rosenmund, 2008; Sudhof Rothman, 2009; Wickner and <br />                                          Schekman, 2008). <br />  <br />                                          SNAP29 SNAP25-family double SNARE-domain t-SNAREs. SNAP29 <br />                                          differs SNAP25-family proteins lacks Cys-residues are <br />                                          palmytoylated SNAP25 SNAP23 (Steegmaier et al., 1998). Mammalian SNAP29 has <br />                                          reported localize intracellular membranes &#226;&#8364;&#8220; endocytic vesicles, Golgi, and <br />                                          lysosomes (Rapaport et al., 2010; Steegmaier et al., 1998). Various studies report that <br />                                          SNAP29 bind syntaxins associated intracellular compartments plasma <br />                                          membrane syntaxins (Hohenstein Roche, 2001; Steegmaier et al., 1998; Wong et al., <br />                                          1999). The proposed functions SNAP29 correlate subcellular localization, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          including recycling integral components focal adhesion complexes migrating <br />                                          fibroblasts, recycling post-fusion SNARE complexes neuronal cells during <br />                                          synaptic transmission (Pan et al., 2005; Rapaport et al., 2010; Su et al., 2001). Furthermore, <br />                                          CEDNIK syndrome, novel neurocutanoeus disease, caused <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> SNAP29 <br />                                          (Sprecher et al., 2005). However, precise intracellular membrane fusion event involving <br />                                          SNAP29 remains somewhat unclear. <br />  <br />                                          The C. elegans genome encodes SNAP25 family proteins &#226;&#8364;&#8220; RIC-4, AEX-4, and <br />                                          K02D10.5 (SNAP-29, GenBank ID: NC003281). ric-4 encodes synaptic SNAP25 <br />                                          ortholog C. elegans (Hwang Lee, 2003). ric-4 mutants discovered a <br />                                          screen animals resistant aldicarb, inhibitor acetylcholinesterase. Mutation in <br />                                          ric-4 results decreased acetylcholine release impaired fusion synaptic vesicles <br />                                          synaptic plasma membrane, conferring aldicarb resistance (Miller et al., 1996; <br />                                          Nonet et al., 1998). This work demonstrated RIC-4 dependent synaptic vesicle fusion is <br />                                          essential executing synaptic transmission C. elegans. AEX-4 involved in <br />                                          intracellular trafficking intestine, modulating trafficking AEX-5, pro-protein <br />                                          convertase, turn regulates defecation motor program. Therefore, mutations in <br />                                          aex-4 result defective defecation(Mahoney et al., 2008). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          In study, report physiological role SNAP29 homolog C. elegans. We <br />                                          SNAP-29 required general secretion, affecting transmembrane and <br />                                          secreted proteins multiple tissues. In intestine SNAP-29-GFP enriched <br />                                          recycling endosomes. Analysis transport defects intestine suggest SNAP-29 <br />                                          functions Golgi recycling endosome transport. We document developmental <br />                                          defects associated secretory block germ cells. Loss SNAP-29 leads to <br />                                          failure ovulation associated failure secrete LIN-3/EGF failure pre-meiotic <br />                                          maturation associated failure secrete VAB-1/Ephrin-R. In early embryo lack of <br />                                          SNAP-29 results failure secrete cortical granules failure cytokinesis indicating <br />                                          critical role membrane transport key developmental events. <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                           Page 3 <br />  <br />  <br />                               Materials methods <br />                               Strains <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The Bristol strain, N2, used WT handled using standard protocols. The <br />                                           following N2-derivative strains used: rrf-1(pk1417), <br />                                           dpy-5(e61)daf-16(m26)unc-75(e950), fog-2(q71), oma-1(zu405te33), oma-2(te51), <br />                                           dpy-17(e164), vab-7(e1562), unc-32(e189) rme-1(b1045), rab-10(dx2), ppw-1(pk2505), <br />                                           ppw-1(pk1425), RW10006(ruIs32[ unc-119(+) + Ppie-1::GFP::H2B]. zuIs178[unc-119(+) + <br />                                           his-72(1 kb):: HIS-72::GFP]), WH371(ojIs50[Ppie-1::GFP::air-2 + unc-119(+)]), <br />                                           DG2160(tnIs13[Ppie-1::vab-1::GFP + unc-119(+)] ltIs44[Ppie-1-mcherry::PH(PLC1delta1) <br />                                           + unc-119(+)]), RT130(pwIs23[vit-2::GFP]), OD179(ltIs79[pAA196; <br />                                           Ppie-1::mcherry::RAB-5 + unc-119(+)] [Ppie-1::RME-2::GFP]), RT408(pwIs116 <br />                                           [rme-2::GFP + unc-119(+)]), RT688(pwIs28[Ppie--cav-1::GFP + unc-119(+)]), <br />                                           WH347(ojIs35[Ppie-1::GFP::rab-11.1 + unc-119(+)]), RT123(pwIs21[Ppie-1::GFP::rab-7 + <br />                                           unc-119(+)]), RT163(pwIs33[Ppie-1::GFP::rab-11, unc-119(+)]), <br />                                           RT1100(Ppie-1::GFPrme-1F + unc-119(+)), RT852(pwIs338 [Ppie-1::GFP::rab-5 + <br />                                           unc-119 (+)]), RT2532(pwIs874[ Ppie-1::lin-3.a::GFP + unc-119(+)]), <br />                                           pwIs72[Pvha-6::GFP::RAB-5], pwIs170[Pvha-6::GFP::RAB-7], <br />                                           pwIs206[Pvha-6::GFP::RAB-10], pwIs69[Pvha-6::GFP::RAB-11], <br />                                           pwIs481[Pvha-6::MANS::GFP], pwIs87[Pvha-6::GFP::RME-1], <br />                                           pwIs112[Pvha-6::hTAC::GFP], pwIs90[Pvha-6::hTfR::GFP], <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pwEx102[Pvha-6::MANS::mCherry], pwIs846[Pvha-6::TagRFP::RAB-5], <br />                                           pwIs852[Pvha-6::TagRFP::RME-1], PwEx135[Pvha-6::TagRFP::RAB-11], <br />                                           pwIs695[Pvha-6::GFP::SNAP-29], pwIs603[Pvha-6::GFP::SNB-1], <br />                                           PwIs686[Pvha-6::GFP::SYN-4], pwIs914[Pvha-6::Tag RFP::RAB-6.2], <br />                                           pwIs915[Pvit-2::VIT2SignalSequence::VENUS::LDLR(TM-INT)], SP-12- <br />                                           GFP[Ppie-1::SP-12::GFP], pwEx131[SNAP-29-GFP]. <br />  <br />                               Molecular Biology Transgenic strains <br />                                           The full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> snap-29 genomic DNA, plus 4 kb promoter sequence, cloned <br />                                           GFP-containing vector. To make SNAP-29-GFP transgenic animal, SNAP-29-GFP <br />                                           plasmid injected N2 100 ng/ml pRF4 selection marker. <br />  <br />                                           For germline expression snap-29 lin-3, full-length genomic DNA snap-29 and <br />                                           full-length cDNA lin-3.a cloned pJL1007, gfp-containing modified L1-L2 <br />                                           Multigateway vector. Next, pCG142, pCM5.47 (gifts Seydoux), cloned snap-29- <br />                                           gfp lin-3.a-gfp vector combined make final constructs bombardment. To <br />                                           construct GFP RFP fusion transgenes expression worm intestine, Gateway <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           destination vectors used. Each vector uses promoter region intestine-specific <br />                                           gene vha-6 cloned C. elegans pPD117.01 vector, followed GFP coding <br />                                           sequences, Gateway cassette (Invitrogen, Carlsbad, CA), let-858 3&#226;&#8364;&#178; UTR sequences, and <br />                                           unc-119 gene C. briggsae. The genomic cDNA sequences C. elegans snap-29, <br />                                           syn-4, snb-1,genes cloned individually entry vector pDONR221. A pDONR221- <br />                                           RAB-6.2(cDNA) construct generously provided Dr. Christopher Rongo. Coding <br />                                           regions transferred vha-6-GFP vha-6-RFP vectors LR reaction <br />                                           (Invitrogen). To construct VENUS-LDLR fusion transgenes expression worm <br />                                           intestine, Gateway destination vector constructed using promoter region signal <br />                                           sequence intestine-specific vit-2 gene cloned C. elegans pPD117.01 vector, <br />                                           followed VENUS coding sequence (generous gift Joshua Kaplan), Gateway <br />                                           cassette (Invitrogen, Carlsbad, CA), let-858 3&#226;&#8364;&#178; UTR sequences, followed unc-119 <br />                                           gene C. briggsae. The transmembrane intracellular domains human LDL <br />                                           receptor cloned entry vector pDONR221, transferred vit-2- <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                            Page 4 <br />  <br />  <br />                                           VENUS vector generate N-terminal fusion. All final constructs bombarded <br />                                           unc-119(ed3) mutant animals establish low copy integrated transgenic lines (Praitis et <br />                                           al., 2001). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Microscopy images <br />                                           Live worms mounted 2% agarose pads 10 mM levamisole described <br />                                           previously (Sato et al., 2005). Multi-wavelength fluorescence images obtained using an <br />                                           Axiovert 200M (Carl Zeiss MicroImaging, Oberkochen, Germany) microscope equipped <br />                                           digital CCD camera (C4742&#226;&#8364;&#8220;12ER, Hamamatsu Photonics, Hamamatsu, Japan), <br />                                           captured using Metamorph software (Universal Imaging, West Chester, PA), then <br />                                           deconvolved using AutoDeblur software (AutoQuant Imaging, Watervliet, NY). Images <br />                                           taken DAPI channel used identify broad-spectrum intestinal autofluorescence <br />                                           caused lipofuscin-positive lysosomes (Clokey Jacobson, 1986). To obtain images of <br />                                           GFP fluorescence interference autofluorescence, used spectral <br />                                           fingerprinting function Zeiss LSM510 Meta confocal microscope focusing on <br />                                           510 nm emission wavelength (Carl Zeiss MicroImaging). Quantification images was <br />                                           performed Metamorph software (Universal Imaging). Most GFP/RFP colocalization <br />                                           analysis performed L3 L4 larvae. <br />  <br />                                           Projection images (Figs. 1E, H), fluorescence images (Figs. 1F, G, I&#226;&#8364;&#8220;K, Fig.2E) <br />                                           acquired acquired using confocal microscope (ZEISS LSM 700, Carl Zeiss, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Inc), 40 X 1.20 NA water lens (Carl Zeiss, Inc), ZEN2009 image browser software <br />                                           (Carl Zeiss, Inc.). Figs, 2B, F, G, Fig3&#226;&#8364;&#8220;4 Figs 6E, F images acquired a <br />                                           microscope (Axioplan 2, Carl Zeiss), 10X 0.30 NA lens, 20X 0.50 NA lens, 40X NA 0.75 <br />                                           lens, 63X 1.25 NA oil lens 100X 1.3 NA oil lens (Carl Zeiss), CCD camera (AxioCam <br />                                           HRc), AxioVision 4 software (Carl Zeiss). 3D merged fluorescence images (Figs. 1E, <br />                                           H) taken using LSM700 (Carl Zeiss, Inc). Fig 1E H taken 30 40 <br />                                           sections z-axis 0.5 mm 0.31 mm intervals. Images merged a <br />                                           single projection image using ZEN2009 program. <br />  <br />                               Feeding RNAi <br />                                           RNAi feeding performed using standard methods. E. coli strains containing each <br />                                           RNAi clone obtained RNAi library (J. Ahringer). For snap-29 clones, we <br />                                           used 4 different clones showed effect. One RNAi library. Other <br />                                           constructs cloned L4440 vector. One full-length cDNA. The are <br />                                           N-terminus C-terminus snap-29 cDNA, cut EcoRI enzyme. To examine RNAi <br />                                           effects snap-29 oocytes, L4 stage animals transferred RNAi-medium. <br />                                           After 1day, RNAi treated animals used. To examine RNAi effects snap-29 the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           intestine, synchronized L3-stage worms used RNAi-feeding assays. Phenotypes <br />                                           scored 48h later young adults. rab-1 RNAi performed using clone the <br />                                           Ahringer library described Sato et al (2006)(Sato et al., 2006). <br />  <br />                               Ovulation rate <br />                                           The ovulation rate calculated previously described modification <br />                                           (McCarter et al., 1999). L4 stage animals transferred RNAi medium. After 1day, <br />                                           young adult animals mounted 5% agar pad M9 observe Normarski <br />                                           microscopy calculated total number embryos oocytes uterus each <br />                                           worm. Next, 4~5 animals transferred new RNAi medium let lay eggs for <br />                                           3~4 hours. After completion time interval (3.5~4hrs), total number embryos or <br />                                           oocytes uterus worm counted again. The ovulation rate gonad arm <br />                                           hour (number embryos oocytes end interval &#226;&#8364;&#8220; number embryos or <br />                                           oocytes start interval)/(2 X population size (4 5) X total hours (3.5 4)). <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                           Page 5 <br />  <br />  <br />                               Results <br />                               K02D10.5 encodes C. elegans SNAP-29 widely expressed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           In C. elegans, genes encode snap-25-family proteins. These include RIC-4, AEX-4, <br />                                           K02D10.5 (Figs. 1A, B). Each predicted proteins SNARE domains, the <br />                                           major defining feature SNAP25-family. Among these, K02D10.5 shows greatest <br />                                           similarity human SNAP29. Thus assigned SNAP-29 K02D10.5. To <br />                                           investigate expression localization SNAP-29 vivo, examined animals <br />                                           expressing SNAP-29-GFP fusion gene driven snap-29 promoter. SNAP-29-GFP <br />                                           expressed widely, including excretory canal, gland cells, hypodermis, vulva, <br />                                           coelomocyte, intestine, gonad sheath cells neurons (Figs. 1C &#226;&#8364;&#8220; I). SNAP-29-GFP <br />                                           appeared enriched intracellular puncta (identified endosomal organelles) well <br />                                           near plasma membrane. In intestine, SNAP-29-GFP appeared highly <br />                                           enriched near apical membrane, visible basolateral structures (Fig. 1G). <br />                                           SNAP-29-GFP expressed germline control pie-1 promoter localized <br />                                           intracellular vesicles, enrichment cortex near plasma membrane oocytes <br />                                           early embryos (Figs. 1J, K). Given well-known function SNAP-25-family <br />                                           proteins membrane fusion, broad expression <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> SNAP-29-GFP <br />                                           transgene, suspected SNAP-29 important animal development and <br />                                           cellular function tissues body. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               snap-29 essential gene <br />                                           To determine physiological importance snap-29 vivo, analyzed <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br />                                           snap-29 mutant animals. The snap-29(tm2060) mutation deletes half 2nd <br />                                           exon 2nd intron. This deletion removes sequences encoding first <br />                                           SNARE motif, depending splicing affect second SNARE motif well. <br />                                           Thus snap-29(tm2060) unlikely produce functional protein. We that <br />                                           homozygous snap-29(tm2060) mutant animals invariably arrested development early <br />                                           larval stages (Table 1). Larval arrest common mutants lack essential gene. <br />                                           Proteins mRNAs provided heterozygous mother embryo often <br />                                           sufficient allow embryonic development. The SNAP-29-GFP transgene highly <br />                                           expressed embryos suggesting snap-29 important embryonic <br />                                           development. Many arrested snap-29(tm2060) mutant larvae appeared trapped in <br />                                           partially shed cuticle, phenotype commonly C. elegans membrane trafficking <br />                                           mutants (Greener et al., 2001; Zhang et al., 2001). <br />  <br />                                           To test functionality SNAP-29::GFP fusion assayed ability rescue <br />                                           snap-29 mutant phenotypes. While 100% snap-29(tm2060) mutant worms showed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           early larval arrest, 48% snap-29(tm2060) transgenic worms carrying SNAP-29::GFP <br />                                           construct driven promoter rescued early larval arrest, suggesting <br />                                           SNAP-29::GFP functional (n =90 48 snap-29(tm2060) and <br />                                           snap-29(tm2060);Ex[SNAP-29-GFP], respectively; Fig S1). However, fusion is <br />                                           probably fully functional rescued worms displayed slow growth late <br />                                           larval arrest phenotype (Fig S1). <br />  <br />                               Loss SNAP-29 leads defects ovulation <br />                                           To better determine requirements SNAP-29 larval adult stages, we <br />                                           analyzed phenotype animals depleted SNAP-29 using timed RNAi-feeding <br />                                           protocol. Many snap-29 RNAi treated larvae showed molting defects, described <br />                                           deletion mutant (Fig S2). In general, larvae arrested stages <br />                                           application snap-29 RNAi, indicating continuous requirement SNAP-29 <br />                                           development. <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                           Page 6 <br />  <br />  <br />                                          To examine requirements snap-29 function adult stage, applied <br />                                          snap-29 RNAi late L4 stage animals examined phenotype young adults 24 <br />                                          hours later. Such snap-29(RNAi) animals produced progeny soon became <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          completely sterile. We examined germline snap-29 RNAi treated adults to <br />                                          determine source sterility. The C. elegans hermaphrodite gonad U-shaped <br />                                          tubular organ. Germ cells distal region syncytial mitotic, entering <br />                                          meiotic prophase away distal tip. Cellularization oogenesis <br />                                          begins near bend region gonad arm. Oocytes arrest diakinesis meiotic <br />                                          prophase I stimulated meiotic maturation signals reach proximal-most <br />                                          region gonad near sperm-laden spermatheca. About 20 min proximal- <br />                                          oocyte, nearest spermatheca, matures ovulates. Ovulation allows entry the <br />                                          spermatheca fertilization occurs. We oocytes snap-29 RNAi treated <br />                                          animals displayed enlarged nuclei increased DNA content determined DAPI <br />                                          staining (Figs. 2A&#226;&#8364;&#8220;2C). This phenotype termed Emo (endomitotic replication DNA <br />                                          oocyte), typically occurs ovulation fails. Extended failure ovulation <br />                                          oocyte maturation eventually leads aberrant exit diakinesis chromosomal <br />                                          endoreduplication cytokinesis (Iwasaki et al., 1996; McCarter et al., 1999). While <br />                                          control oocytes displayed condensed chromosomes diakinesis, snap-29 RNAi treated <br />                                          oocytes displayed Emo polyploidy (n = 73 85, Emo phenotype = 0% 70% for <br />                                          control snap-29(RNAi), respectively) (Figs. 2B, 2C). Furthermore, Emo phenotype <br />                                          produced using independent RNAi clones targeting non-overlapping <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          regions snap-29 mRNA, supporting conclusion phenotype resulted <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                          RNAi-mediated depletion SNAP-29, off-target effects (Emo phenotype = 54% <br />                                          [n=55] 64% [n=75] 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; snap-29 RNAi clones, respectively) (Fig 2C). <br />  <br />                                          To confirm Emo phenotype resulted ovulation defect measured the <br />                                          ovulation rate (McCarter et al., 1999). The ovulation rate 2.33 ovulations/hr control <br />                                          animals (n=30), snap-29 RNAi animals produced 0.54 ovulations/hr (n=30; Fig. <br />                                          2D). In addition, using time-lapse microscopy, observed oocytes in <br />                                          snap-29(RNAi) animals failed enter spermatheca completely. The ovulated <br />                                          oocytes occasionally torn, concomitant abnormal constriction spermathecal <br />                                          valve, leaving oocyte debris gonad (Supplementary Videos S1 S2). <br />  <br />                                          We confirmed over-replication DNA proximal oocytes snap-29(RNAi) <br />                                          animals required meiotic cell-cycle progression (maturation) blocking meiotic prophase I <br />                                          using oma-1;oma-2 double mutants(Inoue et al., 2006; Kim et al., 2010)(Detwiler et al., <br />                                          2001). As expected meiotic prophase I block imposed oma-1; oma-2 mutation <br />                                          epistatic snap-29(RNAi), preventing over-replication DNA proximal <br />                                          oocytes (n=12 22 control snap-29(RNAi), respectively; Figs. 2F, G). Another <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          requirement oocyte maturation, prerequisite ovulation Emo phenotype <br />                                          produced defective ovulation, MSP signal sperm. Thus confirmed that <br />                                          snap-29(RNAi) Emo phenotype requires sperm using fog-2 feminized mutant lacks <br />                                          sperm (Ellis Schedl, 2007; McCarter et al., 1999). We snap-29(RNAi) in <br />                                          fog-2 females did cause Emo phenotype gonad (n=44 66, Emo phenotype <br />                                          = 0% 2% fog-2 females snap-29(RNAi) fog-2 females, respectively; Fig. 2C). In <br />                                          contrast, mated males, snap-29(RNAi);fog-2 females did Emo <br />                                          phenotype gonad (n=37 37, Emo phenotype = 0% 95% control fog-2 and <br />                                          snap-29(RNAi) fog-2 females, respectively; Fig. 2C). <br />  <br />                                          Several different types cells contribute ovulation C. elegans. This includes somatic <br />                                          germline tissues (sheath spermatheca) germ cells (oocytes). To determine <br />                                          contribution SNAP-29 ovulation germ cells, performed <br />                                          snap-29 RNAi rrf-1 ppw-1 mutant backgrounds. In rrf-1 mutants germ cells the <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                              Page 7 <br />  <br />  <br />                                           animal sensitive RNAi, ppw-1 mutants somatic cells remain sensitive to <br />                                           RNAi (Sijen et al., 2001; Tijsterman et al., 2002). While control animals <br />                                           displayed Emo phenotype (n=77), 93% snap-29(RNAi);rrf-1 animals Emo <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (n=82; Fig. 2C). In contrast, snap-29(RNAi); ppw-1 animals did Emo <br />                                           phenotype (n=67, 61, 30, 52 control ppw-1(pk1425), snap-29(RNAi) <br />                                           ppw-1(pk1425), control ppw-1(pk2505) snap-29(RNAi) ppw-1(pk2505), respectively; <br />                                           Fig 2C). The ovulation rate control rrf-1 animals 2.41 ovulation/hr (n=29; Fig. <br />                                           2D), snap-29(RNAi);rrf-1 animals produced 0.26 ovulation/hr (n=28; Fig 2D). <br />                                           Therefore, conclude respect ovulation, SNAP-29 required germ <br />                                           cells. <br />  <br />                                           To investigate genes intracellular transport involved sterile <br />                                           phenotype, analyzed phenotype rab-1 RNAi treated worms. RAB-1 member <br />                                           small GTPase family required ER Golgi transport. 82% rab-1 RNAi treated <br />                                           worms showed Emo phenotype (n = 73; Fig. 2C). This result indicates <br />                                           disruption general secretory pathway germ cells leads defective ovulation. <br />  <br />                               snap-29 required trafficking transmembrane signaling proteins oocyte <br />                                           As mentioned above, sperm secrete major sperm proteins (MSPs) signaling molecule <br />                                           recognized proximal oocytes overlying gonad sheath cells (Miller et al., <br />                                           2001)(Miller et al., 2003). The MSP signal recognized VAB-1 ephrin <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           receptor-like protein, promotes oocyte maturation, prerequisite ovulation (Miller et <br />                                           al., 2003). In addition, oocytes signal sheath spermatheca ovulation via <br />                                           secretion LIN-3 EGF-type ligand. This EGF signaling promotes spermathecal dilation <br />                                           sheath cell contraction, events required successful fertilization (Bui Sternberg, <br />                                           2002; Clandinin et al., 1998; McCarter et al., 1999). <br />  <br />                                           Given known role SNAP-25 family SNARES membrane traffic, reasoned that <br />                                           snap-29(RNAi) Emo phenotype result failure oocytes secrete the <br />                                           LIN-3 EGF-like signal. To examine snap-29 required trafficking LIN-3 in <br />                                           oocyte, compared localization LIN-3-GFP fusion protein wild type and <br />                                           snap-29(RNAi) animals. LIN-3-GFP expressed oocyte germline specific <br />                                           pie-1 promoter localized spermatheca oocyte wild type animals <br />                                           (n=25, 100% normal). In contrast, LIN-3-GFP accumulated perinuclear region the <br />                                           oocytes snap-29(RNAi) animals (n=28, 17% 73% normal abnormal, <br />                                           respectively; Fig. 2E). Thus data suggests snap-29 regulates LIN-3 secretion the <br />                                           oocyte, essential ovulation. <br />  <br />                                           We noted phenotype gonad snap-29(RNAi) animals: presence of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ovoid distal oocytes (Fig. 3A). In C. elegans, proximal oocyte the <br />                                           spermatheca (first, -1 position) matures reenters cell cycle. The main hallmarks of <br />                                           meiotic maturation C. elegans cortical rearrangement changes appearance of <br />                                           oocyte cylindrical ovoid shape, nuclear envelope breakdown (NEBD), and <br />                                           AIR-2-GFP chromosomal localization (Govindan et al., 2009; McCarter et al., 1999; Miller <br />                                           et al., 2001). To determine distal oocytes snap-29(RNAi) animals maturing early, as <br />                                           suggested ovoid shape, assayed NEBD AIR-2 localization using GFP- <br />                                           tagged molecular markers(Bao et al., 2006). In wild-type background, diffuse nuclear <br />                                           histone-GFP signals present distal oocytes, condensed chromosomal <br />                                           histone-GFP signal -1 oocyte (n=22; Fig. 3A). The condensed <br />                                           chromosomal histone-GFP signal indicative NEBD mature oocyte. By contrast, we <br />                                           snap-29(RNAi) animals distal oocytes displayed condensed <br />                                           chromosomal GFP signals similar -1 oocyte control, suggesting that <br />                                           NEBD events occurred prematurely (n=25; Fig. 3A). As previous reported, <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                                Page 8 <br />  <br />  <br />                                           chromosomal localization AIR-2-GFP marker MSP response meiotic <br />                                           maturation (Govindan et al., 2009). The chromatin localization AIR-2-GFP strong at <br />                                           -1 oocyte fainter -2 oocyte control animals (n=68; Fig. 3B Table S1). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           However, chromatin-localized AIR-2-GFP strong -2, -3, -4 oocytes in <br />                                           snap-29 (RNAi) animals (n=57; Fig. 3B Table S1). These data suggest distal <br />                                           oocytes snap-29(RNAi) animals undergo premature pre-meiotic maturation. <br />  <br />                                           The VAB-1 MSP/ephrin receptor negatively modulates pre-meiotic maturation distal <br />                                           oocytes. MSP inhibits VAB-1 signaling, helping trigger meiotic maturation, and <br />                                           endocytic trafficking VAB-1 required signaling (Miller et al., 2001; Miller et al., <br />                                           2003)(Cheng et al., 2008). To determine phenotype result misregulation <br />                                           VAB-1, analyzed VAB-1 localization snap-29(RNAi) animals. While VAB-1-GFP <br />                                           localized plasma membrane intracellular vesicles control oocytes (n=54, <br />                                           normal=96%; Fig. 3C), VAB-1-GFP accumulated intracellularly, near nucleus, in <br />                                           snap-29(RNAi) animals (n=56, abnormal=84%; Fig. 3C). This phenotype always <br />                                           observed distal oocytes displaying ovoid shape snap-29(RNAi) animals. Thus <br />                                           data indicate SNAP-29 required trafficking VAB-1-GFP oocyte, <br />                                           potentially explaining observed ectopic maturation distal oocytes animals depleted <br />                                           SNAP-29. <br />  <br />                               SNAP-29 required secretion cortical granules yolk receptors <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Two additional well-described membrane proteins oocyte early embryo are <br />                                           CAV-1 RME-2. RME-2 yolk receptor responsible uptake massive <br />                                           quantities lipoprotein particles (yolk) secreted intestine adult hermaphrodites <br />                                           (Grant Hirsh, 1999). CAV-1 marks cortical granules, large vesicles formed the <br />                                           oocyte synchronously secreted early embryos shortly fertilization <br />                                           (Bembenek et al., 2007; Sato et al., 2006; Sato et al., 2008). Chondroitin proteoglycans are <br />                                           major constituents cortical granules, secretion chondroitin proteoglycans the <br />                                           early embryo required cytokinesis (Hwang et al., 2003; Mizuguchi et al., 2003; Sato et <br />                                           al., 2008)(Bembenek et al., 2007). Thus disruption cortical granule release leads to <br />                                           multinucleated embryos (Bembenek et al., 2007; Sato et al., 2008) <br />  <br />                                           As previously reported, observed CAV-1-GFP distal germline mainly <br />                                           localized plasma membrane. CAV-1-GFP localizes small vesicles bend <br />                                           region gonad oocyte formation begins. In larger oocytes CAV-1-GFP appeared <br />                                           large ring-like structures cytosol (cortical granules). After fertilization, CAV-1-GFP <br />                                           positive cortical granules fused plasma membrane (Fig. 4A). In contrast, <br />                                           snap-29(RNAi) appeared trap CAV-1-GFP near nucleus oocytes, such <br />                                           oocytes lacked identifiable cortical granules (Fig. 4A). Consistent data, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           snap-29(RNAi) resulted production multinucleated embryos, indicating cytokinesis <br />                                           defect (Fig. 4B). In addition, snap-29(RNAi) results abnormal accumulation GFP- <br />                                           tagged yolk receptors (RME-2-GFP) near nucleus, similar localization CAV-1- <br />                                           GFP VAB-1-GFP snap-29(RNAi) (n=46, near nucleus=92%; Fig. 4C). These <br />                                           data suggest SNAP-29 required secretion many, all, transmembrane cargo <br />                                           proteins oocyte cell. <br />  <br />                               Loss SNAP-29 disrupts secretion intestinal epithelial cells <br />                                           In order determine SNAP-29 required secretory function cell types <br />                                           examined intestine, terminally differentiated epithelial tube prodigious <br />                                           secretory activity. Since cells intestine divide embryogenesis, are <br />                                           polarized cells, offer opportunity test requirements SNAP-29 secretion <br />                                           considering potential involvement cell cycle, testing potential <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                            Page 9 <br />  <br />  <br />                                          specificity basolateral versus apical transport. First examined basolaterally secreted <br />                                          yolk protein YP170 expressed GFP-fusion. Normally YP170-GFP driven own <br />                                          promoter expressed intestinal cells adult hermaphrodite secreted <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          basolaterally body cavity internalized oocytes expressing the <br />                                          yolk receptor (Grant Hirsh, 1999). We snap-29(RNAi) animals accumulated <br />                                          YP170-GFP intestinal cells abnormally high levels, YP170-GFP failed to <br />                                          reach oocytes (Figs. 5A, B). We assayed effect previously described <br />                                          basolateral transmembrane cargo markers, hTfR-GFP (human transferrin receptor) and <br />                                          hTAC-GFP (human IL-2 receptor alpha-chain)(Chen et al., 2006; Shi et al., 2007). In <br />                                          snap-29(RNAi) animals hTAC-GFP hTfR-GFP accumulated intracellularly, <br />                                          apparently failing reach basolateral plasma membrane (Figs. 5E&#226;&#8364;&#8220;I). Similar results <br />                                          obtained translation fusion signal-sequence Venus transmembrane and <br />                                          intracellular domains human LDL receptor, mainly localized <br />                                          basolaterally (Figs. 5C, D). Since aspects secretion appear differ between <br />                                          basolateral apical cargo, tested apically localized transmembrane proteins, <br />                                          GFP-SNB-1 GFP-SYN-4, apically localized SNARE proteins. Depletion snap-29 <br />                                          RNAi led increased diffuse cytoplasmic labeling GFP-SNB-1 GFP-SYN-4 with <br />                                          concomitant reduction apical localization, indicating defect apical secretion (Figs <br />                                          5J, K). Thus data indicate SNAP-29 generally required secretion process <br />                                          including transport apical basolateral cargos. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Loss SNAP-29 affects recycling endosomes <br />                                          To help determine step secretion SNAP-29 functions, assayed for <br />                                          snap-29(RNAi) induced changes morphology variety intracellular organelles <br />                                          using collection previously described GFP-tagged markers expressed intestine <br />                                          (Chen et al., 2006). In particular noted dramatic disruption recycling endosomes <br />                                          labeled GFP-RME-1, GFP-RAB-11, GFP-RAB-10 snap-29(RNAi) animals (Figs. <br />                                          6A&#226;&#8364;&#8220;C). <br />  <br />                                          By contrast, Golgi marker MANS-GFP displayed normal morphology distribution in <br />                                          snap-29(RNAi) animals, marker appeared somewhat dimmed intensity (Fig. <br />                                          6D). We noted differential sensitivity recycling endosome markers SNAP-29 <br />                                          depletion, compared Golgi markers, strains expressing markers both <br />                                          compartments simultaneously. In strains co-expressing GFP-RAB-11 Golgi marker <br />                                          MANS-RFP RFP-RAB-6.2, snap-29(RNAi) lead complete diffusion GFP-RAB-11, <br />                                          RFP-tagged MANS RAB-6.2 puncta preserved (Fig. S2). Further analysis <br />                                          indicated recycling endosome markers GFP-RAB-11 GFP-RME-1 were <br />                                          mislocalized cortex juxtanuclear position snap-29(RNAi) treated oocytes <br />                                          (Figs 6E, 6F). However, ER marker SP-12-GFP altered snap-29(RNAi) treated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          oocytes. SP-12-GFP dispersed reticular network cell <br />                                          regardless snap-29 status (Fig 6G; (Poteryaev et al., 2005). Together recent <br />                                          reports recycling endosome acts intermediate secretory traffic between <br />                                          Golgi plasma membrane, results suggest SNAP-29 mediate traffic <br />                                          Golgi apparatus recycling endosome (Ang et al., 2004; Folsch, 2008). <br />  <br />                               SNAP-29 Localizes Recycling Endosome <br />                                          To help determine hypothesis correct, analyzed subcellular localization of <br />                                          SNAP-29 intestine using GFP-tagged-SNAP-29 set previously described <br />                                          RFP-tagged organellar markers (Chen et al., 2006; Shi et al., 2010). GFP-SNAP-29 co- <br />                                          localized best canonical recycling endosome marker RFP-RAB-11 (Fig. 7A-A&#226;&#8364;?). Some <br />                                          GFP-SNAP-29 labeled structures colocalized RFP-RME-1, basolateral-specific <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                           Page 10 <br />  <br />  <br />                                          recycling endosome marker (Fig. 7B-B&#226;&#8364;?). GFP-SNAP-29 co-localized poorly Golgi <br />                                          marker MANS-mCherry early endosomal marker RFP-RAB-5 (Figs. 7C&#226;&#8364;&#8220;D&#226;&#8364;?). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          We tested localization GFP-SNAP-29 recycling endosomes crossing the <br />                                          marker rme-1(b1045) rab-10(dx2) mutant backgrounds known disrupt <br />                                          recycling endosome Golgi morphology (Chen et al., 2006; Shi et al., 2007). In both <br />                                          mutant backgrounds GFP-SNAP-29 localization severely disrupted (Figs. 8A&#226;&#8364;&#8220;C). These <br />                                          results indicate SNAP-29 enriched recycling endosomes a <br />                                          position directly mediate fusion incoming secretory vesicles derived the <br />                                          Golgi. Consistent idea recycling endosomes intermediates Golgi to <br />                                          plasma membrane transport, Golgi marked MANS-RFP RFP- <br />                                          RAB-6.2 generally flanked GFP-RAB-11 marked recycling endosomes. This is <br />                                          similar pattern GFP-SNAP-29, typically flanked RFP-RAB-6.2 <br />                                          labeled Golgi (Figs. S3 D, E). <br />  <br />                                          Finally, wished establish SNAP-29 acts RME-1 the <br />                                          recycling endosome. In rme-1 mutants hTac-GFP overaccumulates enlarged recycling <br />                                          endosomes (Fig. 8D Chen et al, 2006). As described snap-29(RNAi) results in <br />                                          accumulation hTac-GFP diffusely cytoplasm, probably dispersed small vesicles <br />                                          (Figs. 5H,I). We snap-29(RNAi) produces diffuse accumulation hTac- <br />                                          GFP rme-1 mutant background, indicating SNAP-29 functions hTac- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          GFP trafficking prior recycling endosome (Fig. 8E). <br />  <br />                               Discussion <br />                                          In study, showed SNAP-29 plays pivotal roles C. elegans levels of <br />                                          organismal development intracellular transport. Depletion SNAP-29 leads severe <br />                                          defects oocyte ovulation (Emo phenotype), likely observed failure to <br />                                          secrete LIN-3 EGF. We identified ectopic oocyte maturation animals lacking <br />                                          SNAP-29, likely observed defective secretion VAB-1 MSP receptor. While <br />                                          secretion known important ovulation, endocytic trafficking VAB-1 was <br />                                          known important restricting oocyte maturation, direct demonstration <br />                                          secretion important process (Cheng et al., 2008; Iwasaki et al., 1996). Our <br />                                          work demonstration defects late step secretion produce <br />                                          germline development defects profound produced blocking early <br />                                          secretory step controlled EMO-1/Sec61: co-translational insertion transmembrane and <br />                                          secreted proteins ER membrane (Iwasaki et al., 1996). <br />  <br />                                          These results, indicating importance SNAP-29 germline secretion signaling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          proteins, led test general defects secretory pathway animals lacking <br />                                          SNAP-29. In fact assayed oocyte membrane proteins, including oocyte <br />                                          yolk receptor RME-2 cortical granule component CAV-1, failed reach cell <br />                                          surface SNAP-29 depleted animals. Further analysis showed loss SNAP-29 <br />                                          disrupted basolateral apical secretion intestinal cells indicating SNAP-29 is <br />                                          probably universally required efficient secretion C. elegans. We identified Golgi to <br />                                          recycling endosome transport likely step requiring SNAP-29. GFP-SNAP-29 <br />                                          highly enriched recycling endosomes intestinal cells. Loss of <br />                                          SNAP-29 severely disrupted recycling endosome morphology, localization SNAP-29 <br />                                          required intact recycling endosomes. Since SNAP-29 t-SNARE likely function <br />                                          fusion Golgi derived vesicles recycling endosome membrane. The <br />                                          severity secretory defect indicate requirement SNAP-29 transport <br />                                          steps well. <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                                          Page 11 <br />  <br />  <br />                                          The depletion snap-29 rrf-1 strain, germline-specific RNAi strain, showed the <br />                                          Emo phenotype. No Emo phenotype elicited ppw-1 mutant strains restrict <br />                                          RNAi-sensitivity somatic tissues. While SNAP-29 appears ubiquitously expressed, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          data indicate respect ovulation, germ cells require SNAP-29. <br />  <br />                                          SNAP-29 important t-snare involved intracellular trafficking mammals. <br />                                          Deregulation SNAP-29 associated important diseases, as <br />                                          schizophrenia(Saito et al., 2001), CEDNIK syndrome(Sprecher et al., 2005), cardiac septal <br />                                          defects microcephaly(Ogilvie et al., 2009). However, conflicting data been <br />                                          reported subcellular localization function vertebrate Snap29. One report <br />                                          indicated interaction Snap29 Syntaxin6 specific localization Golgi <br />                                          apparatus (Wong et al., 1999). Our data indicate Ce-SNAP-29 localized Golgi- <br />                                          proximal recycling endosomes Golgi itself. Another report indicated equivalent <br />                                          binding Snap29 plasma membrane intracellular syntaxins (Hohenstein and <br />                                          Roche, 2001)}. Snap29 reported interact Syntaxin-1A function as <br />                                          regulator SNARE complex disassembly (Su et al., 2001), interact Syntaxin-19 and <br />                                          function post-Golgi snare (Gordon et al., 2010), interact Rab3A (Schardt et <br />                                          al., 2009). Mammalian Snap29 reported interact recycling endosome <br />                                          protein EHD1, mammalian RME-1 homolog, regulate endocytic recycling <br />                                          (Rapaport et al., 2010; Rotem-Yehudar et al., 2001), However, results based on <br />                                          cultured cells confirmed intact animal. Our data suggest this <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          interaction physiologically relevant, EHD-homolog RME-1 SNAP-29 <br />                                          affect localization. However rme-1 mutant phenotypes severe <br />                                          snap-29, indicating RME-1 absolutely required SNAP-29 <br />                                          function. <br />  <br />                                          Here intact living animal, SNAP-29 functions recycling endosome <br />                                          regulates secretion. In ways phenotypic results similar recently <br />                                          reported Rapaport et al., results differ key respects (Rapaport et al., <br />                                          2010). While studies point important role SNAP-29 recycling <br />                                          endosome, C. elegans severe defect secretion animals depleted of <br />                                          SNAP-29. Experiments human CEDNIK cells, thought null Snap29, were <br />                                          reported defective recycling secretion. There possible <br />                                          explanations apparent discrepancy. One possibility vertebrate-specific <br />                                          protein, SNAP-47, redundant SNAP-29 mediating Golgi recycling <br />                                          endosome transport vertebrates cells (Holt et al., 2006). Alternatively, degree which <br />                                          recycling endosome acts intermediate Golgi plasma membrane transport may <br />                                          differ cell types organisms. Recent work recycling endosome protein <br />                                          EHBP-1 C. elegans suggests high degree dependence pathway C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          elegans germline (Shi et al., 2010). Further work required determine what <br />                                          extent pathway required various vertebrate cell-types. <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                          We thank H. Kim (SNU, Korea) advice initiation study. We thank CGC NBP japan <br />                                          strains, J. Ahringer RNAi plasmids. JL supported Research Center Functional <br />                                          Cellulomics Seoul National University, Korea grant National R&amp;D Program Cancer Control, <br />                                          Ministry Health, Welfare Family Affairs, Korea. BG supported NIH grant GM067237. <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                                  Page 12 <br />  <br />  <br />                               References <br />                                          Ang AL, Taguchi T, Francis S, Folsch H, Murrells LJ, Pypaert M, Warren G, Mellman I. Recycling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             endosomes serve intermediates transport Golgi plasma membrane of <br />                                             MDCK cells. J Cell Biol. 2004; 167:531&#226;&#8364;&#8220;543. [PubMed: 15534004] <br />                                          Bao Z, Murray JI, Boyle T, Ooi SL, Sandel MJ, Waterston RH. Automated cell lineage tracing in <br />                                             Caenorhabditis elegans. Proc Natl Acad Sci U S A. 2006; 103:2707&#226;&#8364;&#8220;2712. [PubMed: 16477039] <br />                                          Bembenek JN, Richie CT, Squirrell JM, Campbell JM, Eliceiri KW, Poteryaev D, Spang A, Golden A, <br />                                             White JG. Cortical granule exocytosis C. elegans regulated cell cycle components including <br />                                             separase. Development. 2007; 134:3837&#226;&#8364;&#8220;3848. [PubMed: 17913784] <br />                                          Bui YK, Sternberg PW. Caenorhabditis elegans inositol 5-phosphatase homolog negatively regulates <br />                                             inositol 1,4,5-triphosphate signaling ovulation. Mol Biol Cell. 2002; 13:1641&#226;&#8364;&#8220;1651. [PubMed: <br />                                             12006659] <br />                                          Chen CC, Schweinsberg PJ, Vashist S, Mareiniss DP, Lambie EJ, Grant BD. RAB-10 required for <br />                                             endocytic recycling Caenorhabditis elegans intestine. Mol Biol Cell. 2006; 17:1286&#226;&#8364;&#8220;1297. <br />                                             [PubMed: 16394106] <br />                                          Cheng H, Govindan JA, Greenstein D. Regulated trafficking MSP/Eph receptor oocyte <br />                                             meiotic maturation C. elegans. Curr Biol. 2008; 18:705&#226;&#8364;&#8220;714. [PubMed: 18472420] <br />                                          Clandinin TR, DeModena JA, Sternberg PW. Inositol trisphosphate mediates RAS-independent <br />                                             response LET-23 receptor tyrosine kinase activation C. elegans. Cell. 1998; 92:523&#226;&#8364;&#8220;533. <br />                                             [PubMed: 9491893] <br />                                          Clokey GV, Jacobson LA. The autofluorescent &#226;&#8364;&#339;lipofuscin granules&#226;&#8364;? intestinal cells of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Caenorhabditis elegans secondary lysosomes. Mech Ageing Dev. 1986; 35:79&#226;&#8364;&#8220;94. [PubMed: <br />                                             3736133] <br />                                          Detwiler MR, Reuben M, Li X, Rogers E, Lin R. Two zinc finger proteins, OMA-1 OMA-2, are <br />                                             redundantly required oocyte maturation C. elegans. Dev Cell. 2001; 1:187&#226;&#8364;&#8220;199. [PubMed: <br />                                             11702779] <br />                                          Ellis, R.; Schedl, T. WormBook. 2007. Sex determination germ line; p. 1-13. <br />                                          Fasshauer D, Sutton RB, Brunger AT, Jahn R. Conserved structural features synaptic fusion <br />                                               complex: SNARE proteins reclassified Q- R-SNAREs. Proc Natl Acad Sci U S A. 1998; <br />                                               95:15781&#226;&#8364;&#8220;15786. [PubMed: 9861047] <br />                                          Folsch H. Regulation membrane trafficking polarized epithelial cells. Curr Opin Cell Biol. 2008; <br />                                               20:208&#226;&#8364;&#8220;213. [PubMed: 18282697] <br />                                          Gordon DE, Bond LM, Sahlender DA, Peden AA. A targeted siRNA screen identify SNAREs <br />                                               required constitutive secretion mammalian cells. Traffic. 2010; 11:1191&#226;&#8364;&#8220;1204. [PubMed: <br />                                               20545907] <br />                                          Govindan JA, Nadarajan S, Kim S, Starich TA, Greenstein D. Somatic cAMP signaling regulates <br />                                               MSP-dependent oocyte growth meiotic maturation C. elegans. Development. 2009; <br />                                               136:2211&#226;&#8364;&#8220;2221. [PubMed: 19502483] <br />                                          Grant B, Hirsh D. Receptor-mediated endocytosis Caenorhabditis elegans oocyte. Mol Biol Cell. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               1999; 10:4311&#226;&#8364;&#8220;4326. [PubMed: 10588660] <br />                                          Greener T, Grant B, Zhang Y, Wu X, Greene LE, Hirsh D, Eisenberg E. Caenorhabditis elegans <br />                                               auxilin: J-domain protein essential clathrin-mediated endocytosis vivo. Nat Cell Biol. <br />                                               2001; 3:215&#226;&#8364;&#8220;219. [PubMed: 11175756] <br />                                          Hohenstein AC, Roche PA. SNAP-29 promiscuous syntaxin-binding SNARE. Biochem Biophys <br />                                               Res Commun. 2001; 285:167&#226;&#8364;&#8220;171. [PubMed: 11444821] <br />                                          Holt M, Varoqueaux F, Wiederhold K, Takamori S, Urlaub H, Fasshauer D, Jahn R. Identification of <br />                                               SNAP-47, novel Qbc-SNARE ubiquitous expression. J Biol Chem. 2006; 281:17076&#226;&#8364;&#8220; <br />                                               17083. [PubMed: 16621800] <br />                                          Hwang HY, Olson SK, Esko JD, Horvitz HR. Caenorhabditis elegans early embryogenesis vulval <br />                                               morphogenesis require chondroitin biosynthesis. Nature. 2003; 423:439&#226;&#8364;&#8220;443. [PubMed: <br />                                               12761549] <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                                Page 13 <br />  <br />  <br />                                          Hwang SB, Lee J. Neuron cell type-specific SNAP-25 expression driven multiple regulatory <br />                                              elements nematode Caenorhabditis elegans. J Mol Biol. 2003; 333:237&#226;&#8364;&#8220;247. [PubMed: <br />                                              14529613] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Inoue T, Hirata K, Kuwana Y, Fujita M, Miwa J, Roy R, Yamaguchi Y. Cell cycle control daf-21/ <br />                                              Hsp90 meiotic prophase/metaphase boundary oogenesis Caenorhabditis <br />                                              elegans. Dev Growth Differ. 2006; 48:25&#226;&#8364;&#8220;32. [PubMed: 16466390] <br />                                          Iwasaki K, McCarter J, Francis R, Schedl T. emo-1, Caenorhabditis elegans Sec61p gamma <br />                                              homologue, required oocyte development ovulation. J Cell Biol. 1996; 134:699&#226;&#8364;&#8220;714. <br />                                              [PubMed: 8707849] <br />                                          Kim J, Kawasaki I, Shim YH. cdc-25.2, C. elegans ortholog cdc25, required promote oocyte <br />                                              maturation. J Cell Sci. 2010; 123:993&#226;&#8364;&#8220;1000. [PubMed: 20200231] <br />                                          Mahoney TR, Luo S, Round EK, Brauner M, Gottschalk A, Thomas JH, Nonet ML. Intestinal <br />                                              signaling GABAergic neurons regulates rhythmic behavior Caenorhabditis elegans. Proc <br />                                              Natl Acad Sci U S A. 2008; 105:16350&#226;&#8364;&#8220;16355. [PubMed: 18852466] <br />                                          McCarter J, Bartlett B, Dang T, Schedl T. On control oocyte meiotic maturation ovulation <br />                                              Caenorhabditis elegans. Dev Biol. 1999; 205:111&#226;&#8364;&#8220;128. [PubMed: 9882501] <br />                                          Miller KG, Alfonso A, Nguyen M, Crowell JA, Johnson CD, Rand JB. A genetic selection for <br />                                              Caenorhabditis elegans synaptic transmission mutants. Proc Natl Acad Sci U S A. 1996; <br />                                              93:12593&#226;&#8364;&#8220;12598. [PubMed: 8901627] <br />                                          Miller MA, Nguyen VQ, Lee MH, Kosinski M, Schedl T, Caprioli RM, Greenstein D. A sperm <br />                                              cytoskeletal protein signals oocyte meiotic maturation ovulation. Science. 2001; <br />                                              291:2144&#226;&#8364;&#8220;2147. [PubMed: 11251118] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Miller MA, Ruest PJ, Kosinski M, Hanks SK, Greenstein D. An Eph receptor sperm-sensing control <br />                                              mechanism oocyte meiotic maturation Caenorhabditis elegans. Genes Dev. 2003; 17:187&#226;&#8364;&#8220; <br />                                              200. [PubMed: 12533508] <br />                                          Mizuguchi S, Uyama T, Kitagawa H, Nomura KH, Dejima K, Gengyo-Ando K, Mitani S, Sugahara K, <br />                                              Nomura K. Chondroitin proteoglycans involved cell division Caenorhabditis elegans. <br />                                              Nature. 2003; 423:443&#226;&#8364;&#8220;448. [PubMed: 12761550] <br />                                          Nonet ML, Saifee O, Zhao H, Rand JB, Wei L. Synaptic transmission deficits Caenorhabditis <br />                                              elegans synaptobrevin mutants. J Neurosci. 1998; 18:70&#226;&#8364;&#8220;80. [PubMed: 9412487] <br />                                          Ogilvie CM, Ahn JW, Mann K, Roberts RG, Flinter F. A novel deletion proximal 22q associated <br />                                              cardiac septal defects microcephaly: case report. Mol Cytogenet. 2009; 2:9. [PubMed: <br />                                              19239688] <br />                                          Pan PY, Cai Q, Lin L, Lu PH, Duan S, Sheng ZH. SNAP-29-mediated modulation synaptic <br />                                              transmission cultured hippocampal neurons. J Biol Chem. 2005; 280:25769&#226;&#8364;&#8220;25779. [PubMed: <br />                                              15890653] <br />                                          Poteryaev D, Squirrell JM, Campbell JM, White JG, Spang A. Involvement actin cytoskeleton <br />                                              homotypic membrane fusion ER dynamics Caenorhabditis elegans. Mol Biol Cell. 2005; <br />                                              16:2139&#226;&#8364;&#8220;2153. [PubMed: 15716356] <br />                                          Praitis V, Casey E, Collar D, Austin J. Creation low-copy integrated transgenic lines in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Caenorhabditis elegans. Genetics. 2001; 157:1217&#226;&#8364;&#8220;1226. [PubMed: 11238406] <br />                                          Rapaport D, Lugassy Y, Sprecher E, Horowitz M. Loss SNAP29 impairs endocytic recycling and <br />                                              cell motility. PLoS One. 2010; 5:e9759. [PubMed: 20305790] <br />                                          Rizo J, Rosenmund C. Synaptic vesicle fusion. Nat Struct Mol Biol. 2008; 15:665&#226;&#8364;&#8220;674. [PubMed: <br />                                              18618940] <br />                                          Rotem-Yehudar R, Galperin E, Horowitz M. Association insulin-like growth factor 1 receptor with <br />                                              EHD1 SNAP29. J Biol Chem. 2001; 276:33054&#226;&#8364;&#8220;33060. [PubMed: 11423532] <br />                                          Saito T, Guan F, Papolos DF, Rajouria N, Fann CS, Lachman HM. Polymorphism SNAP29 gene <br />                                              promoter region associated schizophrenia. Mol Psychiatry. 2001; 6:193&#226;&#8364;&#8220;201. [PubMed: <br />                                              11317222] <br />                                          Sato K, Sato M, Audhya A, Oegema K, Schweinsberg P, Grant BD. Dynamic regulation caveolin-1 <br />                                              trafficking germ line embryo Caenorhabditis elegans. Mol Biol Cell. 2006; 17:3085&#226;&#8364;&#8220; <br />                                              3094. [PubMed: 16672374] <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                                   Page 14 <br />  <br />  <br />                                          Sato M, Grant BD, Harada A, Sato K. Rab11 required synchronous secretion chondroitin <br />                                              proteoglycans fertilization Caenorhabditis elegans. J Cell Sci. 2008; 121:3177&#226;&#8364;&#8220;3186. <br />                                              [PubMed: 18765566] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Sato M, Sato K, Fonarev P, Huang CJ, Liou W, Grant BD. Caenorhabditis elegans RME-6 novel <br />                                              regulator RAB-5 clathrin-coated pit. Nat Cell Biol. 2005; 7:559&#226;&#8364;&#8220;569. [PubMed: <br />                                              15895077] <br />                                          Schardt A, Brinkmann BG, Mitkovski M, Sereda MW, Werner HB, Nave KA. The SNARE protein <br />                                              SNAP-29 interacts GTPase Rab3A: Implications membrane trafficking myelinating <br />                                              glia. J Neurosci Res. 2009; 87:3465&#226;&#8364;&#8220;3479. [PubMed: 19170188] <br />                                          Shi A, Chen CC, Banerjee R, Glodowski D, Audhya A, Rongo C, Grant BD. EHBP-1 functions with <br />                                              RAB-10 endocytic recycling Caenorhabditis elegans. Mol Biol Cell. 2010; 21:2930&#226;&#8364;&#8220; <br />                                              2943. [PubMed: 20573983] <br />                                          Shi A, Pant S, Balklava Z, Chen CC, Figueroa V, Grant BD. A novel requirement C. elegans Alix/ <br />                                              ALX-1 RME-1-mediated membrane transport. Curr Biol. 2007; 17:1913&#226;&#8364;&#8220;1924. [PubMed: <br />                                              17997305] <br />                                          Sijen T, Fleenor J, Simmer F, Thijssen KL, Parrish S, Timmons L, Plasterk RH, Fire A. On role of <br />                                              RNA amplification dsRNA-triggered gene silencing. Cell. 2001; 107:465&#226;&#8364;&#8220;476. [PubMed: <br />                                              11719187] <br />                                          Sprecher E, Ishida-Yamamoto A, Mizrahi-Koren M, Rapaport D, Goldsher D, Indelman M, Topaz O, <br />                                              Chefetz I, Keren H, O&#226;&#8364;&#8482;Brien TJ, Bercovich D, Shalev S, Geiger D, Bergman R, Horowitz M, <br />                                              Mandel H. A mutation SNAP29, coding SNARE protein involved intracellular <br />                                              trafficking, causes novel neurocutaneous syndrome characterized cerebral dysgenesis, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              neuropathy, ichthyosis, palmoplantar keratoderma. Am J Hum Genet. 2005; 77:242&#226;&#8364;&#8220;251. <br />                                              [PubMed: 15968592] <br />                                          Steegmaier M, Yang B, Yoo JS, Huang B, Shen M, Yu S, Luo Y, Scheller RH. Three novel proteins of <br />                                              syntaxin/SNAP-25 family. J Biol Chem. 1998; 273:34171&#226;&#8364;&#8220;34179. [PubMed: 9852078] <br />                                          Su Q, Mochida S, Tian JH, Mehta R, Sheng ZH. SNAP-29: general SNARE protein inhibits <br />                                              SNARE disassembly implicated synaptic transmission. Proc Natl Acad Sci U S A. 2001; <br />                                              98:14038&#226;&#8364;&#8220;14043. [PubMed: 11707603] <br />                                          Sudhof TC, Rothman JE. Membrane fusion: grappling SNARE SM proteins. Science. 2009; <br />                                              323:474&#226;&#8364;&#8220;477. [PubMed: 19164740] <br />                                          Tijsterman M, Okihara KL, Thijssen K, Plasterk RH. PPW-1, PAZ/PIWI protein required for <br />                                              efficient germline RNAi, defective natural isolate C. elegans. Curr Biol. 2002; 12:1535&#226;&#8364;&#8220; <br />                                              1540. [PubMed: 12225671] <br />                                          Wickner W, Schekman R. Membrane fusion. Nat Struct Mol Biol. 2008; 15:658&#226;&#8364;&#8220;664. [PubMed: <br />                                              18618939] <br />                                          Wong SH, Xu Y, Zhang T, Griffiths G, Lowe SL, Subramaniam VN, Seow KT, Hong W. GS32, a <br />                                              novel Golgi SNARE 32 kDa, interacts preferentially syntaxin 6. Mol Biol Cell. 1999; <br />                                              10:119&#226;&#8364;&#8220;134. [PubMed: 9880331] <br />                                          Zhang Y, Grant B, Hirsh D. RME-8, conserved J-domain protein, required endocytosis in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Caenorhabditis elegans. Mol Biol Cell. 2001; 12:2011&#226;&#8364;&#8220;2021. [PubMed: 11451999] <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                      Page 15 <br />  <br />  <br />                                          Highlights <br />                                              &#226;&#8364;&#162;    SNAP-29 constantly required life cycle C. elegans. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              &#226;&#8364;&#162;    snap-29 involved intracellular trafficking proteins. <br />                                              &#226;&#8364;&#162;    SNAP-29 proteins localized recycling endosomes. <br />                                              &#226;&#8364;&#162;    SNAP-29 functions prior recycling endosome. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. SNAP-29 broadly expressed C. elegans <br />                                          (A) Gene structure snap-29. Numbers indicate percentage amino acid identity with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          respect SNAP-29. (B) A phylogenetic tree various snap25 family suggesting that <br />                                          K02D10.5 putative homolog snap-29. (C-K) SNAP-29-GFP expressed various <br />                                          epithelial tissues &#226;&#8364;&#8220; embryo (C), vulva (D), gonad sheath cells (E), hypodermis (F), intestine <br />                                          (G), neurons (H), excretory canal(H), coelomocyte (I). (J, K) SNAP-29-GFP <br />                                          expression pattern oocyte. SNAP-29-GFP expressed promoter (C- <br />                                          I) pie-1 germline promoter (J, K). (C) (H) 3D merged images. Arrows and <br />                                          arrowheads indicate endocytic vesicles plasma membrane, respectively. The scale bars <br />                                          10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                        Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. snap-29 (RNAi) required ovulation <br />                                          (A) Normarski images control (left) snap-29 snap-29 (RNAi) (right). White arrows <br />                                          enlarged nuclei. The arrowhead indicates multinucleated oocyte. sp spermatheca. (B) <br />                                          DAPI staining control snap-29 (RNAi) animals. Arrows Emo oocytes. (C) <br />                                          Quantification Emo phenotype using DAPI staining. (D) Quantification ovulation <br />                                          rates. (E) LIN-3-GFP expression oocytes control snap-29 (RNAi) animals. (F, <br />                                          G) DAPI staining control (F) snap-29(RNAi) oma-1; oma-2 mutant animals <br />                                          (G). The scale bars 10um. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                      Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. snap-29(RNAi) animals showed pre-meiotic maturation phenotype <br />                                          (A) The expression GFP-H2B HIS-72-GFP control snap-29 (RNAi) animals. <br />                                          Oocytes proximal oocyte (-1 position) showed diffused nuclear GFP in <br />                                          control animals. Most proximal oocyte condensed chromosomal GFP signal. Distal <br />                                          oocytes snap-29(RNAi) animals showed condensed GFP signal, diffused <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          signal. Only proximal oocyte showed ovoid shape control, distal <br />                                          oocytes showed ovoid shape snap-29(RNAi). (B) Images AIR-2-GFP. <br />                                          Numbers indicate position oocyte spermatheca. &#226;&#8364;&#732;Minus&#226;&#8364;&#8482; means the <br />                                          gonad. -1 proximal oocyte. (C) The expression pattern VAB-1-GFP control <br />                                          snap-29(RNAi) animals. VAB-1-GFP accumulated nuclear envelope the <br />                                          oocyte snap-29 (RNAi). The scale bar 10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                 Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                       Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. SNAP-29 controls localization CAV-1 RME-2 <br />                                          (A) CAV-1-GFP expression pattern control snap-29 (RNAi). CAV-1-GFP <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          accumulated nuclear envelope oocyte snap-29 (RNAi). Numbers indicate <br />                                          position oocytes spermatheca. &#226;&#8364;&#732;Plus&#226;&#8364;&#8482; &#226;&#8364;&#732;Minus&#226;&#8364;&#8482; mean uterus <br />                                          gonad, respectively. Sp indicates spermatheca. (B) The expression GFP-H2B and <br />                                          HIS-72-GFP control snap-29 RNAi treated embryos. Arrow indicates multinucleated <br />                                          cells. (C) RME-2-GFP expression pattern control snap-29 (RNAi). RME-2-GFP <br />                                          localized plasma membrane oocyte. After fertilization, RME-2-GFP endocytosed <br />                                          showed punctuated signals embryo (arrows). RME-2-GFP accumulated the <br />                                          nuclear envelope oocyte snap-29 (RNAi). After fertilization, RME-2-GFP exists the <br />                                          embryo snap-29 (RNAi). The scale bars 10 &#206;&#188;m. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                          Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. Loss SNAP-29 disrupts trafficking proteins intestine <br />                                          (A&#226;&#8364;&#8220;B) Yolk protein YP170-GFP (VIT-2-GFP) accumulates abnormally high levels in <br />                                          secretory cells (intestine - arrows) fails reach target (oocytes &#226;&#8364;&#8220; arrowheads) in <br />                                          snap-29 RNAi animals. (C-D&apos;) Signal sequence modified Venus fused transmembrane <br />                                          intracellular domains human LDL receptor (VENUS-LDLR) trapped the <br />                                          intestine cells snap-29 RNAi animals. VENUS-LDLR expressed using intestinal <br />                                          specific promoter. Arrows C-C&apos; shows Venus protein secreted body cavity, which <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          observed snap-29(RNAi) D-D&apos;. Arrowheads indicate basolateral plasma <br />                                          membrane associated VENUS-LDLR (C-C&apos;), observed snap-29(RNAi) (D- <br />                                          D&apos;). (E&#226;&#8364;&#8220;F&apos;) GFP fused intracellular transmembrane domain human <br />                                          transferrin receptor (hTfR-GFP) accumulates intracellularly fails reach basolateral <br />                                          plasma membrane snap-29 RNAi animals. Loss basal-lateral plasma membrane <br />                                          localization hTfR observed F-F&apos; compared E-E&apos;, arrows indicate the <br />                                          basolateral plasma membrane localization hTfR. Arrowheads indicate intracellular <br />                                          accumulation. (G&#226;&#8364;&#8220;H&apos;) hTAC-GFP (human IL2-receptor alpha chain fused GFP) <br />                                          accumulates abnormally high levels intestinal cells (F-F&apos; arrows) fails reach the <br />                                          basolateral plasma membrane (E-E&apos; arrowheads)in snap-29 RNAi animals. (I-I&apos;) Intestinal <br />                                          vacuoles observed snap-29 RNAi (indicated arrows). hTAC-GFP labeling shown. (J&#226;&#8364;&#8220; <br />                                          K) Secretion apical membrane proteins impaired snap-29 RNAi animals. (J-J&#226;&#8364;&#8482;) <br />                                          Apical membrane localization GFP-SNB-1 defective snap-29 RNAi animals. SNB-1 <br />                                          transmembrane v-snare localized apical membrane, showed arrows. (K-K&#226;&#8364;&#8482;) Apical <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                   Page 21 <br />  <br />  <br />                                          membrane localization GFP-SYN-4 defective snap-29 RNAi animals. SNB-1 a <br />                                          transmembrane t-snare localized apical membrane, showed arrows. The scale bar <br />                                          represents 10&#206;&#188;m. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                    Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 6. Endosomes Golgi disrupted snap-29 RNAi animals <br />                                          (A) GFP-RME-1 diffusive snap-29 RNAi animals. Arrowheads indicated intracellular <br />                                          punctae. (B) GFP-RAB-11 diffusive snap-29 RNAi animals. Arrowheads indicated <br />                                          intracellular punctae. (C) GFP-RAB-10 diffusive snap-29 RNAi animals. Arrowheads <br />                                          indicated intracellular punctae. (D) MANS-GFP relatively normal snap-29 RNAi <br />                                          animals. Arrowheads indicated intracellular punctas. (E&#226;&#8364;&#8220;F) GFP-RME-1 (E) GFP- <br />                                          RAB-11(F) mis-localized cortex juxtanuclear position snap-29 RNAi <br />                                          animals. (G) SP-12-GFP showed similar localization control snap-29(RNAi). The <br />                                          scale bars represent 10&#206;&#188;m. <br />  <br />  <br />  <br />                                                 Dev Biol. Author manuscript; available PMC 2012 July 1. <br />                             Kang et al.                                                                                        Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 7. SNAP-29 colocalizes recycling endosome markers intestine <br />                                          Representative images deconvolved 3D image stacks shown. All images were <br />                                          acquired intact living animals expressing GFP TagRFP mCherry-tagged proteins <br />                                          specifically intestinal epithelial cells. (A-A&quot;) GFP-SNAP-29 colocalizes TagRFP- <br />                                          RAB-11. Arrowheads indicate endosomes labeled GFP-SNAP-29 TagRFP- <br />                                          RAB-11. (B-B&quot;) GFP-SNAP-29 colocalizes TagRFP-RME-1. Arrowheads indicate <br />                                          endosomes labeled GFP-SNAP-29 TagRFP- RME-1. (C-C&quot;) GFP-SNAP-29 <br />                                          does colocalize mCherry-MANS. Arrowheads indicate endosomes labeled both <br />                                          GFP-SNAP-29 mCherry-MANS. (D-D&quot;) GFP-SNAP-29 does colocalize with <br />                                          TagRFP-RAB-5. Arrowheads indicate endosomes labeled GFP-SNAP-29 and <br />                                          TagRFP-RAB-5. Enlarged images (2.5X) boxed regions shown insets. In each <br />                                          image, autofluorescent lysosome-like organelles seen channels the <br />                                          strongest signal blue, GFP appears green channel TagRFP o <br /> </body></html>