<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Cumate-Inducible Gene Expression System Sphingomonads and <br /> Other Alphaproteobacteria <br /> Andreas Kaczmarczyk, Julia A. Vorholt, Anne Francez-Charlot <br /> Institute Microbiology, ETH Zurich, Zurich, Switzerland <br />  <br /> Tunable promoters represent pivotal genetic tool wide range applications. Here present sphin- <br /> gomonads, phylogenetically diverse group bacteria gained potential bioremediation and <br /> their use industry dedicated inducible gene expression described far. A strong, constitu- <br /> tive synthetic promoter identified genetic screen subsequently combined repressor oper- <br /> ator sites Pseudomonas putida F1 cym/cmt system. The resulting promoter, termed PQ5, responds rapidly inducer <br /> cumate shows maximal induction ratio 2 3 orders magnitude different sphingomonads tested. Moreover, it <br /> was functional Alphaproteobacteria, model organisms Caulobacter crescentus, Paracoccus denitrificans, <br /> and Methylobacterium extorquens. In noninduced state, expression PQ5 low allow gene depletion analysis, <br /> as demonstrated essential gene phyP Sphingomonas sp. strain Fr1. A set PQ5-based plasmids constructed <br /> allowing fusions affinity tags fluorescent proteins. <br />  <br />  <br />  <br /> R    egulated gene expression systems powerful tool study <br />      physiology, allowing, example, dosage-effect studies, con- <br /> ditional expression toxic alleles, depletion analysis essen- <br />                                                                                ducible rapidly responding conditional gene expression sys- <br />                                                                                tem shows low basal expression induction (i.e., is <br />                                                                                tight). We demonstrate functionality number different <br /> tial genes; accordingly, developed model organ-              sphingomonads distantly related Alphaproteo- <br /> isms (1&#226;&#8364;&#8220;7) missing non-model organisms. Most                  bacteria belonging Sphingomonadales. Due mod- <br /> systems rely transcriptional repressor tightly binds              ular design system, core promoter sequences readily <br /> operator sites promoter region target genes absence           exchanged, opening possibility adapt other <br /> of inducer, preventing transcription; inducer               organisms well. <br /> is present, allosterically binds inactivates transcrip- <br /> tional regulator, leading derepression promoters. Despite                MATERIALS AND METHODS <br /> this simple concept, identification inducible promoters, control            Strains, growth conditions, plasmid delivery. Escherichia coli <br /> elements, inducing conditions particular organism              TOP10 (Invitrogen), &#226;&#8364;&#339;ccdB <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span>&#226;&#8364;? (Invitrogen), &#226;&#8364;&#339;dam&#226;&#171;&#186; dcm&#226;&#171;&#186;&#226;&#8364;? <br /> always trivial task. On hand, difficult             (NEB; catalog no. C2925I) used cloning. Sphingomonas Fr1 wild- <br /> exploit particular use organisms                type (JVZ857) &#226;&#338;&#172;phyR &#226;&#338;&#172;phyP::loxP mutant (JVZ1357) strains were <br />                                                                                described previously (18). Other Sphingomonas, Sphingobium, No- <br /> original host need dedicated transporters the <br />                                                                                vosphingobium strains used work obtained German <br /> inducer different promoter specificity RNA polymerase              Collection Microorganisms Cell Cultures (DSMZ). Methylobacte- <br /> holoenzyme requirement coactivator promoter               rium extorquens AM1 rif (19), Caulobacter crescentus CB15 (20), Agro- <br /> activity (8). Some obstacles circumvented engi-             bacterium tumefaciens C58 (21) laboratory stocks. Paracoccus deni- <br /> neering artificial inducible promoters placing constitutive               trificans JVZ2585 spontaneous rifampin-resistant derivative P. <br /> minimal promoter known active particular organism                   denitrificans DSM 413 (DSMZ) obtained selection LB-Lennox con- <br /> with operator sequences repressor heterologous sys-               taining 50 &#226;?&#174;g rifampin/ml; carries A1610T <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> &#226;&#8364;&#339;cluster I&#226;&#8364;? of <br /> tem (2, 9, 10). We sphingomonads,              rpoB, resulting amino acid change Q537L (equivalent Q513L E. <br /> a phylogenetically diverse group environmentally abundant                   coli), likely accounts rifampin resistance (22). E. coli strains were <br />                                                                                routinely grown LB-Lennox 37&#194;&#176;C. Other strains grown 28&#194;&#176;C <br /> bacteria comprising genera Sphingomonas, Sphingobium, No- <br />                                                                                solidified medium (1.5% agar) 20-ml cultures 100-ml baffled <br /> vosphingobium, Sphingopyxis (11). Members group                flasks (with shaking 160 rpm) LB-Lennox (Sphingomonas Fr1, P. <br /> well known unusual ability degrade wide range                denitrificans, A. tumefaciens), PYE (0.2% peptone, 0.1% yeast extract, <br /> compounds, including pesticides, herbicides, xenobiotics,                  0.02% MgSO4&#194;&#183;7H2O, 0.01% CaCl2&#194;&#183;H2O; C. crescentus), <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> broth <br /> many aromatics. Due property, prospective               (Fluka Analytical, catalog no. 17181) (all sphingomonads), min- <br /> candidates bioremediation; addition, strains are <br /> used industrial applications, described as <br /> potential candidates biocontrol (12&#226;&#8364;&#8220;15). Despite               Received 11 July 2013 Accepted 25 August 2013 <br /> in different aspects physiology, dedicated induc-               Published ahead print 30 August 2013 <br /> ible gene expression sphingomonads existed far.                Address correspondence Anne Francez-Charlot, fanne@micro.biol.ethz.ch. <br /> Here, combining operator sequences (CuO) cum-                     Supplemental material article http://dx.doi.org/10.1128 <br /> ate-responsive CymR repressor Pseudomonas putida F1                     /AEM.02296-13. <br /> cym/cmt (16, 17) engineered minimal housekeep-                    Copyright &#194;&#169; 2013, American Society Microbiology. All Rights Reserved. <br /> ing promoter, Psyn2, derived Sphingomonas sp. strain Fr1                    doi:10.1128/AEM.02296-13 <br /> (henceforth Sphingomonas Fr1), developed highly in- <br />  <br />  <br /> November 2013 Volume 79 Number 21                            Applied Environmental Microbiology    p. 6795&#226;&#8364;&#8220; 6802                            aem.asm.org         6795 <br />  Kaczmarczyk et al. <br />  <br />  <br />  <br /> imal medium (23) methanol (M. extorquens). When appropriate,                 Plasmid construction. Standard molecular biology protocols were <br /> concentrations (in &#226;?&#174;g/ml) antibiotics E. coli, sphingomonads,      followed (29). PCR Phusion DNA polymerase (Thermo <br /> M. extorquens used follows: tetracycline, 10; kanamycin, 50; car-     Scientific) GC buffer containing 10% dimethyl sulfoxide (DMSO), and <br /> benicillin, 50; streptomycin, 100; chloramphenicol, 34. For strains,    oligonucleotides purchased Microsynth (Balgach, Switzer- <br /> tetracycline maintain plasmids added media following     land). Restriction enzymes Thermo Scientific, T4 DNA <br /> concentrations (in &#226;?&#174;g/ml): 2 (C. crescentus); 3 (A. tumefaciens); 1 (P.   ligase New England BioLabs. For oligonucleotide annealing, the <br /> denitrificans).                                                               complementary oligonucleotides (10 &#226;?&#174;M each) mixed 100 &#226;?&#174;l <br />     Plasmids delivered sphingomonads conjugal transfer        oligonucleotide annealing buffer (8 mM Tris-HCl [pH 7.5], 40 mM <br /> electroporation described previously (24). For A. tumefaciens C.       NaCl) microcentrifuge tube; tube placed 1 liter boiling <br /> crescentus, electroporation performed GenePulser Xcell        water, allowed cool room temperature several <br /> (Bio-Rad) using standard preset E. coli program; M. extorquens,       hours. Relevant plasmids listed Table 1 oligonucleotides in <br /> electroporation performed described (24). For P. denitrifi-     Table S1 supplemental material. <br /> cans, plasmids delivered conjugal transfer E. coli S17-1&#226;?&#173;pir         PQ5-based plasmids. pQF assembled multiple steps, mostly <br /> (25) mating minimal medium carbon source previ-          synthetic DNA (Invitrogen MWG Operon Eurofins) short <br /> ously described (24). Mating mixtures plated LB-Lennox con-           double-stranded DNA (dsDNA) (generated oligonucleotide anneal- <br /> taining tetracycline (10 &#226;?&#174;g/ml) rifampin (50 &#226;?&#174;g/ml) plasmid se-       ing), details outlined supplemental material. All PQ5- <br /> lection E. coli counterselection, respectively.                           based plasmids derivatives pQF, pQF-lacZ, which <br />     Measurement promoter activity. Cumate (4-isopropylbenzoic              pQF itself derived (see supplemental material). The multiple- <br /> acid) purchased Sigma-Aldrich (catalog no. 269402-5G)            cloning site (MCS) pQF flanked sequences encoding triple-FLAG <br /> dissolved 100% ethanol result 100 mM stock solution. Dilutions     (3&#226;&#171;&#187;FLAG) tags. Derivatives pQF obtained exchanging the <br /> were prepared ethanol 1,000-fold stock. For &#226;&#8364;&#339;no cumate&#226;&#8364;?          N-terminal 3&#226;&#171;&#187;FLAG tag affinity tags fluorescent proteins <br /> controls, equal volume ethanol added cultures.                   maintaining reading frame original MCS C-ter- <br />     For measuring steady-state activity PQ5 Sphingomonas Fr1,       minal 3&#226;&#171;&#187;FLAG constructs. The N-terminal tags different <br /> precultures wild-type strain harboring pQF-lacZ grown         plasmids follows: pQF, 3&#226;&#171;&#187;FLAG tag; pQH, hemagglutinin (HA) <br /> 8 10 h optical density 600 nm (OD600) 1.0, diluted        tag; pQM, myc tag; pQG, enhanced green fluorescent protein (EGFP); <br /> fresh medium OD600 0.0005 0.001 different cumate             pQY, &#226;&#8364;&#339;super&#226;&#8364;? yellow fluorescent protein 2 (SYFP2); pQR, mCherry; pQC, <br /> concentrations (see Results Discussion), grown 20 24 h.        enhanced cyan fluorescent protein (ECFP). Their construction detailed <br />                                                                               following explanation. pQH generated replacing Hin- <br /> For measuring induction kinetics Sphingomonas Fr1, cultures were <br />                                                                               dIII-VspI fragment pQF fragment generated annealing oligo- <br /> grown exponential phase (OD600 0.5 1.5) cumate was <br />                                                                               nucleotides oJVZ1088 oJVZ1089. Similarly, pQM generated by <br /> added reach final concentration 50 &#226;?&#174;M. For depletion analysis, <br />                                                                               replacing HindIII-VspI fragment pQF fragment generated by <br /> cultures grown LB containing 50 &#226;?&#174;M cumate exponen- <br />                                                                               annealing oligonucleotides oJVZ1090 oJVZ1091. pQF derivatives for <br /> tial phase (OD600 1 2), washed 5 times 20 ml LB, and <br />                                                                               C-terminal fusions fluorescence proteins constructed replacing <br /> resuspended 20 ml LB (OD600 0.025) (50 &#226;?&#174;M) without <br />                                                                               HindIII-VspI fragment pQF PCR fragments encoding fluores- <br /> cumate. Cultures grown 20 24 h measuring nepRp- <br />                                                                               cence proteins amplified oligonucleotides oJVZ1107 oJVZ1108. <br /> lacZ&#226;&#171;&#185; activity. <br />                                                                               PCR templates genes encoding fluorescent proteins follows: <br />     Measurements steady-state activity PQ5 sphingomonads <br />                                                                               pBBRBB-egfp (30) (Addgene plasmid 32549) EGFP; pLM-sYFP2 (18) <br /> were performed described Sphingomonas Fr1 using 25 &#226;?&#174;M cumate. <br />                                                                               SYFP2; pMP4516 (31) ECFP; pHC119 (a gift David H. H. <br /> For A. tumefaciens P. denitrificans, cultures inoculated single    Chou) mCherry. Destination vectors based PQ5 compatible with <br /> colony grown overnight (16 20 h) (25 &#226;?&#174;M) cum-         Gateway cloning (Invitrogen) obtained PCR amplification the <br /> ate. For C. crescentus, cultures exponential phase (OD600 0.6    Gateway cassette (including att sites, ccdB, cat) pDEST-565 (Ad- <br /> 0.8) split induced 50 &#226;?&#174;M cumate subjected          dgene plasmid 11520) oligonucleotides oJVZ740 oJVZ739, di- <br /> mock treatment ethanol 4 h measurement PQ5 activity.       gestion XbaI Acc65I, ligation sites pQF and <br /> For M. extorquens, wild-type strain harboring lux reporter plasmid    derivatives. Destination plasmids given parent&#226;&#8364;&#8482;s fol- <br /> (pQ5-lux pQ2148-lux) grown exponential phase (OD600          lowed &#226;&#8364;&#339;D&#226;&#8364;? (e.g., pQHD derived pQH insertion the <br /> 0.3 0.6) PQ5 PQ2148 induced adding 90 &#226;?&#174;l culture       Gateway cassette allows fusion N-terminal HA tag). The pQH <br /> wells containing 10 &#226;?&#174;l 10-fold aqueous stock solution cumate       derivative pQH2 contains pBBR1 origin replication (oriV) was <br /> different concentrations. Measurements started immediately (&#226;&#172;?2           constructed PCR amplification fragment containing rep, mob, and <br /> min induction) followed approximately 3 h 2-min inter-       oriT pBBR1MCS5 (32) using oligonucleotides oJVZ1436 and <br /> vals.                                                                         oJVZ1437, digestion PscI Eco147I, cloning the <br />     &#226;?&#164;-Galactosidase activity determined using chromogenic sub-        sites pQH. <br /> strate 2-nitrophenyl-&#226;?&#164;-D-galactopyranoside (ONPG; Fluka) according             lacZ reporter constructs. The lacZ transcriptional reporter plasmid <br /> the method Miller (26). For M. extorquens, luciferase activity mea-    pAK501 constructed steps involved cloning puta- <br /> sured 96-well black/white isoplates (PerkinElmer) Victor3 multi-      tive transcriptional terminator, &#226;&#8364;&#339;low-background&#226;&#8364;? multiple-cloning site <br /> <span id='am-6' about='rdfs:label' typeof='owl:Thing'>label</span> plate reader (PerkinElmer) essentially described previously (27)     (MCS), reporter gene lacZ plasmid pCM62 (33) followed by <br /> except measured values corrected initial OD600.         exchange antibiotic resistance cassette origin replication. <br />     SDS-PAGE Western blots. Aliquots (1 ml) taken           Individual steps described following explanation. First, <br /> same cultures used measure nepRp-lacZ&#226;&#171;&#185; activity, cells pelleted    TERM193, fragment encompassing tandem copies putative bidirec- <br /> centrifugation (6,000 &#226;&#171;&#187; g, 5 min) resuspended OD600 10        tional transcriptional terminator encoded Sphingomonas Fr1 ge- <br /> SDS-PAGE sample buffer (28), 5-&#226;?&#174;l samples resolved SDS-           nome, amplified oligonucleotide pair oJVZ745 oJVZ746 <br /> PAGE gels (12.5%) followed transfer nitrocellulose membranes.           pTOPO-TERM193 (see Fig. S1 supplemental material) and <br /> Western blotting performed primary mouse monoclonal anti-            cloned pCM62 PscI HindIII sites, generating plasmid <br /> FLAG M2 antibodies (Invitrogen) secondary goat anti-mouse horse-          pAK127. Next, lacZ amplified E. coli BTH101 (Euromedex) <br /> radish peroxidase-coupled IgG antibodies (Bio-Rad) ECL Western            using oligonucleotide pair oJVZ1082 oJVZ1083 cloned between <br /> blotting detection reagents (GE Healthcare).                                  EcoRI Acc65I sites pAK127, resulting lacZ reporter plas- <br />  <br />  <br /> 6796   aem.asm.org                                                                                                  Applied Environmental Microbiology <br />                                                                                                                              Cumate-Inducible Gene Expression Sphingomonads <br />  <br />  <br />  <br /> TABLE 1 Plasmids used study <br />                                                                                                                                                                    Source or <br /> Plasmid category                   Description/relevant featuresa                                                                                         reference <br /> Basic cloning plasmids <br />   pAK200                                    pBBR oriV; Kmr                                                                                                         18 <br />   pAK206                                    pBBR oriV; Gmr                                                                                                         18 <br />   pAK200Cm                                  pAK200 derivative; Cmr                                                                                                 This study <br />   pBBR1MCS5                                 pBBR oriV; Gmr                                                                                                         32 <br />   pCM62                                     IncP oriV; ColE1 oriV; RP4 oriT; Tcr                                                                                   33 <br />   pME6010                                   pVS1 oriV; p15A oriV; Tcr                                                                                              34 <br />  <br /> Standard PQ5-based plasmids <br />   pQF                                       pCM62 cymR*, PQ5, MCS N- C-terminal fusions 3&#226;&#171;&#187;FLAG tag; Tcr                                        This study <br />   pQH                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion HA tag; Tcr                                    This study <br />   pQM                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion myc tag; Tcr                                   This study <br />   pQG                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion EGFP; Tcr                                      This study <br />   pQY                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion SYFP2; Tcr                                     This study <br />   pQR                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion mCherry; Tcr                                   This study <br />   pQC                                       pQF derivative C-terminal fusion 3&#226;&#171;&#187;FLAG N-terminal fusion ECFP; Tcr                                      This study <br />   pQH2                                      pQH derivative pBBR oriV; Tcr                                                                                     This study <br />   pQFD                                      pQF derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />   pQHD                                      pQH derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />   pQMD                                      pQM derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />   pQGD                                      pQG derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />   pQYD                                      pQY derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />   pQRD                                      pQR derivative Gateway cassette; contains ccdB; Cmr Tcr                                                           This study <br />  <br /> Reporter plasmids <br />   pTOPO-TERM193                             pCR2.1-TOPO TERM193; Kmr                                                                                          This study <br />   pAK127                                    pCM62 TERM193; Tcr                                                                                                This study <br />   pAK127lacZ                                Promoter-probe plasmid (intermediate construct); pAK127 E. coli lacZ; Tcr                                         This study <br />   pAK127lacZ(MCS)                           Promoter-probe plasmid (intermediate construct); pAK127lacZ new MCS; Tcr                                          This study <br />   pAK200Cm-lacZ                             Promoter-probe plasmid (intermediate construct); pAK200Cm TERM193-MCS-lacZ fragment                            This study <br />                                               pAK127lacZ(MCS); Cmr <br />      pAK501                                 Promoter-probe plasmid; pME6010 TERM193-MCS-lacZ-cat fragment pAK200Cm-lacZ; Cmr                               This study <br />      pAK501-nepRp                           nepR promoter (<span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> &#226;&#171;&#186;42 &#226;&#171;&#186;2) pAK501; Cmr                                                                          This study <br />      pAK128                                 Promoter-probe plasmid; pAK127 SYFP2; Tcr                                                                         This study <br />      pQF-lacZ                               pCM62-based reporter plasmid cymR* PQ5-lacZ&#226;&#171;&#185; fusion; Tcr                                                      This study <br />      pLM05                                  Promoter-probe plasmid carrying luxCDABE Photorhabdus luminescens; Kmr                                            27 <br />      pQ5-lux                                pLM05 cymR* PQ5-luxCDABE fusion; Kmr                                                                          This study <br />      pQ2148                                 pQH derivative Psyn2 replaced META1_2148 promoter M. extorquens AM1; Tcr                       This study <br />      pQ2148-lacZ                            Reporter plasmid; pQ2148 lacZ; Tcr                                                                                This study <br />      pQ2148-lux                             pLM05 cymR* PQ2148-luxCDABE fusion; Kmr                                                                       This study <br />  <br /> Other plasmids <br />   pQH-PhyP                                  pQH derivative encoding PhyP-3&#226;&#171;&#187;FLAG fusion; Tcr                                                                        This study <br />   pAK200-QPhyP                              pAK200 derivative PQ5 driving PhyP-3&#226;&#171;&#187;FLAG expression; Kmr                                                         This study <br />   pCM62-PhyR                                pCM62 harboring phyR control native promoter RBS; Tcr                                             18 <br />   pCM62-PhyR(D194A)                         pCM62 harboring phyR(D194A) control native promoter RBS; Tcr                                      18 <br /> a <br />     Cmr, chloramphenicol resistance; Gmr, gentamicin resistance; Kmr, kanamycin resistance; Tcr, tetracycline resistance. <br />  <br />  <br />  <br /> mid pAK127lacZ. Because pAK127lacZ high level background                             lacZ constructed cloning NcoI-NheI-digested PCR product <br /> lacZ activity despite presence terminators, MCS pAK127lacZ                      containing lacZ reporter cassette (TERM193-MCS-lacZ) from <br /> was replaced MCS generated cloning annealed oligonucle-                         pAK127lacZ(MCS) amplified oligonucleotides oJVZ1105 and <br /> otides (oJVZ1086 oJVZ1087) HindIII Acc65I sites                        oJVZ1106 PscI NheI sites pAK200Cm. Finally, pAK501 <br /> pAK127lacZ, giving pAK127lacZ(MCS). This plasmid showed approx-                            generated ligating Eco147I-BglII-digested fragment amplified by <br /> imately 10-fold-reduced background compared parent. <span id='am-9' about='protege:TO' typeof='owl:Thing'>To</span> con-                            PCR oligonucleotides oJVZ1102 oJVZ1103 containing <br /> struct pAK501, &#226;&#8364;&#339;lacZ reporter cassette&#226;&#8364;? pAK127lacZ(MCS) com-                           TERM193, MCS, lacZ, cat pAK200Cm-lacZ 5.1-kb <br /> prising TERM193-MCS-lacZ combined steps                                 fragment pME6010 (34) released Eco147I-BamHI digestion <br /> different broad-host-range oriV (pSV1) different antibiotic-resis-                      (pME6010 purified E. coli &#226;&#8364;&#339;dam&#226;&#171;&#186; dcm&#226;&#171;&#186;&#226;&#8364;? strain since <br /> tant marker (cat). To end, pAK200Cm constructed PCR                         Eco147I cleavage methylation impaired context pME6010). <br /> amplification cat pBAD33 (1) using oligonucleotide pair                               Plasmid pAK501 carries broad-host-range pSV1 origin replication <br /> oJVZ1103 oJVZ1104 cloning BamHI-BcuI-digested PCR                                 compatible commonly used origins replication, <br /> product BglII BcuI sites pAK200 (18). pAK200Cm-                            IncP pBBR1. <br />  <br />  <br /> November 2013 Volume 79 Number 21                                                                                                                           aem.asm.org 6797 <br />  Kaczmarczyk et al. <br />  <br />  <br />  <br />     Plasmid pAK501-nepRp reporting general stress response (GSR) ac- <br /> tivity generated cloning annealed oligonucleotides oJVZ1291 and <br /> oJVZ1292 harboring mapped Sphingomonas Fr1 nepR promoter (18) <br /> between HindIII BamHI sites pAK501. <br />     syfp2 reporter constructs. The syfp2 reporter plasmid pAK128 used <br /> for screening strong promoters constructed PCR amplifcation <br /> of syfp2 pLM-sYFP2 (18) oligonucleotides oJVZ743 and <br /> oJVZ744 cloning pAK127 (see above) Acc65I EcoRI sites. <br /> Construction rpmBp mutant promoter library described a <br /> separate section (see below). <br />     luxCDABE reporter constructs. To construct pQ5-lux, fragment <br /> containing PQ5 cymR* subcloned pQF-lacZ lucif- <br /> erase reporter plasmid pLM05 (27) using restriction sites NotI and <br /> Acc65I. pQ2148-lux constructed steps. First, Psyn2 of <br /> plasmid pQH replaced fragment containing META1_2148 <br /> promoter generated oligonucleotide annealing oJVZ1434 and <br /> oJVZ1435 using HpaI BsrGI restriction sites, giving plasmid <br /> pQ2148. Then, intermediary step, XbaI-EcoRI fragment of <br /> pAK127lacZ(MCS) containing lacZ inserted SpeI and <br /> EcoRI sites pQ2148, generating pQ2148-lacZ. Finally, fragment con- <br /> taining cymR* PQ2148 subcloned pQ2148-lacZ pLM05 <br /> using NotI Acc65I sites, giving pQ2148-lux. <br />     Other plasmids. pQH-PhyP constructed amplifying phyP with <br />                                                                             FIG 1 (A) Weblogo output motif MEME 11 23 query <br /> primers oJVZ534 oJVZ1093, digestion HindIII Acc65I, and <br />                                                                             sequences. (B) Alignment 11 sequences MEME harboring <br /> ligation pQH digested enzymes; construct, PhyP   putative promoter motif (boxed) additional putative promoters <br /> is fused 3&#226;&#171;&#187;FLAG tag C terminus expression        aligned manually reinspection promoter regions of <br /> PhyP-3&#226;&#171;&#187;FLAG fusion driven PQ5 natural phyP ribosome           remaining housekeeping genes. Note linker <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> been <br /> binding site (RBS). pAK200-QPhyP derived subcloning PscI-          adjusted manually inserting 1-bp gap promoters better align <br /> EcoRI fragment pQH-PhyP pAK200 digested en-           &#226;&#171;&#186;35 &#226;&#171;&#186;10 boxes promoters. Labels refer gene locus <br /> zymes. This plasmid carries pBBR oriV compatible   tags permanent genome draft (http://img.jgi.doe.gov/cgi-bin/w <br />                                                                             /main.cgi). (C) Sequence Psyn2 corresponding positions &#226;&#171;&#186;43 &#226;&#171;&#185;1 in <br /> pAK501 reporter plasmid (pSV1 oriV) pCM62-based plasmids (IncP <br />                                                                             original rpmB (Sphme2DRAFT_0682) promoter. Note 1 bp <br /> oriV). pCM62-PhyR pCM62-PhyR(D194A) described pre-            inserted linker region align sequences shown in <br /> viously (18).                                                               panel B. <br />     In silico analysis putative promoters housekeeping genes. A <br /> permanent draft Sphingomonas Fr1 genome (genome &#226;&#8364;&#339;Sph- <br /> ingomonas melonis FR1&#226;&#8364;?) available Integrated Microbial       RESULTS AND DISCUSSION <br /> Genomes (IMG) (35) DOE Joint Genome Institute website         Screening strong housekeeping promoter. Since high gene <br /> (http://img.jgi.doe.gov/cgi-bin/w/main.cgi). Upstream regions puta-      expression induction inducible gene expression sys- <br /> tive housekeeping genes (see Table S2 supplemental material) were <br />                                                                             tem desired, sought identify strong housekeeping <br /> subjected motif search MEME (36) following parameters: <br />                                                                             promoters Sphingomonas Fr1. This promoter constitute <br /> distribution motif occurrences, 0 1 sequence; number differ- <br /> ent motifs, 15; minimum number sites, 5; maximum number sites, <br />                                                                             minimal (or core) promoter sequence order minimize the <br /> 30; minimum motif width, 20; maximum motif width, 50; searching,            risk additional factors influencing promoter activity. Regions of <br /> &#226;&#8364;&#339;given strand only.&#226;&#8364;? Putative rpmB promoter sequences sphingomon-        99 120 bp upstream 23 putative Sphingomonas Fr1 house- <br /> ads retrieved IMG website aligned using MultAlin (37)     keeping genes (see Table S2 supplemental material) were <br /> using symbol comparison table &#226;&#8364;&#339;DNA&#226;&#8364;? default parameters.                 subjected motif search using MEME suite (36). The con- <br />     Screening identification Psyn2. A 119-bp region upstream     served motif TTGACN-N17-TANNNGC resembling &#226;?&#180;70-de- <br /> of rpmB (Sphme2DRAFT_0682) open reading frame random-              pendent promoter consensus E. coli (TTGACA-N17-TATAAT) <br /> ized nucleotides putative &#226;&#171;&#186;10 &#226;&#171;&#186;35 promoter regions           identified 11 23 putative promoter regions MEME <br /> generated oligonucleotide annealing extension Phusion           (Fig. 1A), 7 additional putative promoters original <br /> DNA polymerase using oligonucleotides PL28pNTGACN_s                     set manually aligned consensus (Fig. 1B). We <br /> PL28pTANNNGC_as. Products purified, digested HindIII          focused following promoters, rpmBp, which <br /> BamHI, cloned sites pAK128 (a syfp2 reporter plasmid),     drives expression gene encoding ribosomal protein L28 <br /> and transformed E. coli TOP10. Plasmids purified pool   (Sphme2DRAFT_0682). <br /> approximately 10,000 colonies generate rpmB promoter library. <br />                                                                                 Alignment putative rpmBp regions different sphin- <br /> This library subsequently transformed Sphingomonas Fr1, and <br />                                                                             gomonads showed presumed promoter motif most <br /> approximately 20,000 colonies screened high fluorescence using <br /> an IVIS Spectrum (Caliper LifeSciences). A total 14 colonies      conserved rpmB upstream region putative <br /> showing high fluorescence picked, rpmB promoter regions        &#226;&#171;&#186;35 box TTGACN consensus sequence &#226;&#171;&#186;10 box TAN <br /> were amplified colony PCR sequenced (Microsynth, Switzerland).       NNGC consensus sequence (see Fig. S3 supplemental ma- <br />     Nucleotide sequence accession numbers. Sequences pAK501          terial), suggesting identified motif true promoter of <br /> pQF deposited GenBank (http://www.ncbi.nlm.nih.gov/Gen         rpmB. However, wild-type rpmB promoter weak <br /> Bank/index.html) accession numbers KF536587 KF536588,             tested transcriptional fusion E. coli lacZ (data not <br /> respectively.                                                               shown). We reasoned changing nonconserved nucleotides in <br />  <br />  <br /> 6798   aem.asm.org                                                                                                Applied Environmental Microbiology <br />                                                                                                              Cumate-Inducible Gene Expression Sphingomonads <br />  <br />  <br />  <br />  <br /> FIG 2 (A) Organization cymR*, PQ5, multiple-cloning site (MCS) plasmid pQF (not scale). Restriction sites flanking important features are <br /> indicated. T193* denotes putative transcriptional terminator derived Sphingomonas Fr1 (see supplemental material). Gray boxes MCS indicate <br /> 3&#226;&#171;&#187;FLAG tags black circles CuO sites. (B) Nucleotide sequence PQ5 promoter region MCS. cymR* MCS start codons bold. Ribosome <br /> binding sites (RBS), operator sequences (CuO), &#226;&#171;&#186;35 &#226;&#171;&#186;10 promoter elements boxed. Core promoter sequences PQ5 (Psyn2) promoter <br /> driving cymR* expression (Pbla-mut1T) indicated arrows. Translation MCS indicated nucleotide sequence. All restriction sites shown are <br /> unique pQF, marked asterisk. The 3&#226;&#171;&#187;FLAG tag (peptide sequence DYKDHDGDYKDHDIDYKDDDDK) shown schematically. <br />  <br />  <br /> the &#226;&#171;&#186;10 &#226;&#171;&#186;35 boxes (see above) generate stronger                       activity Sphingomonas Fr1 wild type grown without <br /> promoter screened library mutant rpmBp promoters                 cumate. Irrespective cumate concentration, significant <br /> which nucleotides randomized increased activity                &#226;?&#164;-galactosidase activity (&#226;&#172;?5 Miller units) observed, indicat- <br /> (see Materials Methods). Among 14 promoters identified                 ing strain endogenous &#226;?&#164;-galactosidase activity <br /> this screen (see Fig. S4 supplemental material), iden-           conditions tested lacZ appropriate re- <br /> tical core promoter motif TTGACG-N17-TAACTGC,                        porter gene. Thus, characterize PQ5-dependent gene expres- <br /> the corresponding 44-bp fragment ranging position &#226;&#171;&#186;43                     sion, transcriptional fusion PQ5 lacZ (plasmid pQF-lacZ) <br /> position &#226;&#171;&#185;1 referred Psyn2 following (Fig. 1C).                   used promoter activity followed measuring the <br />     Construction cumate-inducible promoter. In order                      &#226;?&#164;-galactosidase activity Sphingomonas Fr1/pQF-lacZ cultures <br /> render Psyn2 regulated, combined control elements                  grown different cumate concentrations. As seen the <br /> of Pseudomonas putida F1 cym/cmt system, i.e., transcrip-                 dose-response curve Fig. 3A, PQ5 steady-state activity de- <br /> tional repressor CymR corresponding operator sites                        pendent inducer concentration showed maximal <br /> (CuO) (17). This chosen inexpensive in-                    induction ratio &#226;&#172;&#381;250-fold 25 50 &#226;?&#174;M cumate (see also <br /> ducer cumate membrane permeative, i.e., does require                   Table 2). We noted cumate concentrations &#226;&#172;&#381;10 &#226;?&#174;M, <br /> dedicated transporter, Sphingomonas Fr1 apparently                     growth somewhat impaired. However, likely to <br /> metabolize genome lacks homologs cumate utiliza-                  high expression level lacZ cumate concentra- <br /> tion genes. These features make induction stable min-                 tion itself Sphingomonas Fr1 plasmid pQF-lacZ <br /> imal interference physiology.                                                carrying gene, phyP, PQ5 control (see below) did <br />     Because difficult predict behavior promoter                 growth defect 100 &#226;?&#174;M cumate. In time <br /> when placed control heterologous regulatory                    course experiments, addition 50 &#226;?&#174;M cumate exponen- <br /> elements (38), arrangements Psyn2 CuO operator                    tially growing noninduced culture strain (Sphingomo- <br /> sequences different promoters cymR tested (data                  nas Fr1/pQF-lacZ) led rapid sustained increase &#226;?&#164;-ga- <br /> shown). In final configuration shown Fig. 2, cumate-                   lactosidase activity apparent 10 min after <br /> inducible promoter, referred PQ5, consists Psyn2 flanked              induction peaked 3 h (Fig. 3B). The differences levels <br /> two CuO operator sequences; expression high-GC-content                       absolute lacZ expression experiments results <br /> codon-optimized gene CymR repressor (cymR*) driven                     shown Fig. 3A B likely different exper- <br /> divergently modified bla promoter (Pbla-mut1T)                  imental setups (see Materials Methods). In summary, these <br /> &#226;&#171;&#186;35 box changed TTGACA &#226;&#171;&#186;10 box TAC                         results demonstrate PQ5 provides rapidly responding and <br /> AAT. This <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> assembled multiple clon-                  tunable gene expression Sphingomonas Fr1. <br /> ing site flanked sequences encoding triple FLAG (3&#226;&#171;&#187;FLAG)                           PQ5 allows gene depletion analysis. One informa- <br /> tags plasmid pCM62 (33), generating plasmid pQF. Details                     tive applications conditional gene expression systems the <br /> on construction pQF given Fig. S2 Materials                     study essential genes depletion analysis. To test this <br /> Methods supplemental material.                                             possible PQ5, chose essential gene phyP Sphin- <br />     Characterization PQ5. To test E. coli lacZ                gomonas Fr1 (18). phyP encodes putative phosphatase the <br /> used reporter gene measuring cumate-dependent gene                       response regulator PhyR master regulator general <br /> expression, determined endogenous &#226;?&#164;-galactosidase                    stress response (GSR) Alphaproteobacteria (39). phyP be <br />  <br />  <br /> November 2013 Volume 79 Number 21                                                                                                             aem.asm.org 6799 <br />  Kaczmarczyk et al. <br />  <br />  <br />  <br />                                                                                    TABLE 2 PQ5 activity different Alphaproteobacteria <br />                                                                                                                   &#226;?&#164;-Galactosidase activity (Miller <br />                                                                                                                   units)a,c <br />                                                                                    Strain                         Without cumate          With cumate           Ratiob,c <br />                                                                                    Sphingomonas sp.               87 &#226;&#171;&#190; 6.9                24,000 &#226;&#171;&#190; 800          270 <br />                                                                                      strain Fr1 (JVZ857) <br />                                                                                    Sphingomonas aerolata          61 &#226;&#171;&#190; 7.0                16,000 &#226;&#171;&#190; 1,300        260 <br />                                                                                      DSM 14751 <br />                                                                                    Sphingomonas melonis           75 &#226;&#171;&#190; 3.6                6,800 &#226;&#171;&#190; 1,400         90 <br />                                                                                      DSM 14444 <br />                                                                                    Sphingobium indicum            1.2 &#226;&#171;&#190; 2.1               1,700 &#226;&#171;&#190; 36            1,400 <br />                                                                                      DSM 16412 <br />                                                                                    Novosphingobium rosa           28 &#226;&#171;&#190; 24                 13,000 &#226;&#171;&#190; 3,000        480 <br />                                                                                      DSM 7285 <br />                                                                                    Caulobacter crescentus         10 &#226;&#171;&#190; 3.1                11,000 &#226;&#171;&#190; 1,200        1,000 <br />                                                                                      CB15 <br />                                                                                    Paracoccus denitrificans       1.8 &#226;&#171;&#190; 0.6               8,700 &#226;&#171;&#190; 1,300         4,700 <br />                                                                                      JVZ2585 <br />                                                                                    a <br />                                                                                      Data represent means &#226;&#171;&#190; standard deviations results independent <br />                                                                                    biological replicates. <br />                                                                                    b <br />                                                                                      Data represent ratios unrounded mean values (induced results/uninduced results). <br />                                                                                    c <br />                                                                                      Values given significant digits. <br />  <br />  <br />  <br />                                                                                    reached stationary phase (OD600 &#226;&#172;&#381;10). Western blots con- <br />                                                                                    firmed PhyP-3&#226;&#171;&#187;FLAG levels dramatically decreased in <br /> FIG 3 (A) Dose-response curve PQ5 assayed &#226;?&#164;-galactosidase (&#226;?&#164;-gal.)           depleted strain compared nondepleted strain (Fig. 4A). <br /> activity PQ5-lacZ&#226;&#171;&#185; transcriptional fusion Sphingomonas Fr1 wild      Decreased levels PhyP-3&#226;&#171;&#187;FLAG correlated increased <br /> type (strain JVZ857/pQF-lacZ) grown different cumate concentrations           nepRp-lacZ&#226;&#171;&#185; activity measured &#226;?&#164;-galactosidase assays <br /> (no cumate 2-fold increments 0.78 50 &#226;?&#174;M cumate). The inset a <br /> closeup dose-response curve illustrating responsiveness low <br />                                                                                    (Fig. 4B), demonstrating that, indeed, loss PhyP leads GSR <br /> cumate concentrations. (B) Induction kinetics PQ5 assayed adding 50      overactivation. As expected, control strain expressing nonphos- <br /> &#226;?&#174;M cumate noninduced culture strain JVZ857/pQF-lacZ mid-            phorylatable PhyRD194A [plasmid pCM62-PhyR(D194A)] instead <br /> exponential phase following &#226;?&#164;-galactosidase activity PQ5-lacZ&#226;&#171;&#185;         wild-type PhyR displayed reduced nepRp-lacZ&#226;&#171;&#185; activity and <br /> time. The inset represents set independent experiments focusing        showed dependence cumate respect growth (data <br /> first 25 min inducer addition. Data represent means &#226;&#171;&#190; standard <br /> deviations results independent biological replicate experiments,   shown) GSR activation (Fig. 4). Similar results ob- <br /> each performed technical replicates.                                      tained isogenic strains lacking pAK501-nepRp (data <br />  <br />  <br /> deleted unless phyR deleted replaced phyRD194A <br /> allele encoding nonphosphorylatable version PhyR, suggest- <br /> ing phosphorylation PhyR cause lethality. Accord- <br /> ingly, proposed phyP essential dele- <br /> tion lead overactivation GSR (18). <br />     To test hypothesis, used strain deleted chromo- <br /> somal phyP phyR (strain JVZ1357) (18) complemented in <br /> trans functional PhyP-3&#226;&#171;&#187;FLAG fusion expressed the <br /> control PQ5 (plasmid pAK200-QPhyP) phyR the <br /> control promoter (plasmid pCM62-PhyR). As read- <br /> out activation GSR, strain harbored plasmid <br /> carrying nepRp-lacZ&#226;&#171;&#185; transcriptional fusion (plasmid                           FIG 4 (A) Western blot PhyP-3&#226;&#171;&#187;FLAG-depletion strain harboring the <br /> pAK501-nepRp) lacZ expression driven previ-                     phyR wild-type (phyR&#226;&#171;&#185;) D194A mutant (phyRD194A) allele grown (&#226;&#171;&#185;) <br /> ously mapped, PhyR-dependent nepR promoter (18). Growth                         (&#226;&#171;&#186;) cumate (see main text details). PhyP-3&#226;&#171;&#187;FLAG was <br /> this strain LB agar liquid LB required addition                    detected using primary mouse monoclonal anti-FLAG M2 antibodies sec- <br /> cumate (data shown), suggesting absence cum-                    ondary goat anti-mouse horseradish peroxidase-coupled IgG antibodies and <br />                                                                                    ECL Western blotting detection reagent. Membranes exposed X-ray <br /> ate, PhyP-3&#226;&#171;&#187;FLAG depleted level required via-                    films short exposure 20 s (SE) long exposure 15 min (LE) <br /> bility. When grown liquid culture containing cumate, removal                    visualize residual PhyP-3&#226;&#171;&#187;FLAG depleted strains. Numbers the <br /> of inducer extensive washing dilution fresh me-                    left indicate molecular mass markers (in kDa). (B) nepRp-lacZ&#226;&#171;&#185; &#226;?&#164;-galactosi- <br /> dium lacking cumate (OD600 0.025) resulted growth arrest                  dase activity assayed strains represented panel A grown with <br />                                                                                    cumate (white bars) cumate (black bars). Three independent <br /> the exponential phase (OD600 ca. 2) 6 7 generations;                biological replicate experiments performed, data displayed as <br /> contrast, culture diluted fresh medium                      means &#226;&#171;&#190; standard deviations. Results panel A representative the <br /> containing cumate, growth normal continued                        results replicate experiments. <br />  <br />  <br />  <br /> 6800   aem.asm.org                                                                                                            Applied Environmental Microbiology <br />                                                                                                          Cumate-Inducible Gene Expression Sphingomonads <br />  <br />  <br />  <br />  <br /> FIG 5 (A) Alignment core promoter regions PQ5 PQ2148. The sequence BsrGI HpaI difference reporter plasmids <br /> pQ5-lux pQ2148-lux. (B C) Time courses cumate-dependent induction PQ5-luxCDABE PQ2148-luxCDABE reporter constructs. Cumate <br /> concentrations used indicated right. Luciferase activity given arbitrary units (RLU [relative luminescence units]). Data single <br /> experiment representative independent biological replicates. <br />  <br />  <br />  <br /> not shown), indicating observed phenotypes                PQ5, PQ2148, Psyn2 core promoter replaced <br /> artifact lacZ expression. Altogether, results verify             minimal promoter M. extorquens AM1 (Fig. 5A) natu- <br /> phyR epistatic phyP (18) support idea phyP               rally driving expression rpsL (META1_2148). This promoter <br /> essential prevents detrimental overactivation                chosen shows clear single transcriptional start <br /> general stress response prevention constitutive PhyR                site active methanol- succinate-grown bac- <br /> phosphorylation. More generally, demonstrate utility               teria according transcriptome sequencing (RNA-Seq) data (A. <br /> PQ5 studying essential gene function.                                      Francez-Charlot J. A. Vorholt, unpublished data). Similar to <br />     Use PQ5 Alphaproteobacteria. To broaden po-                PQ5, PQ2148 rapidly highly inducible showed approx- <br /> tential use system, tested number             imately 2-fold-higher activity PQ5 highest cumate con- <br /> sphingomonads using plasmid pQF-lacZ. As seen Table 1,                 centration tested (Fig. 5C). <br /> all cases high induction ratios approximately 100- 1,000-fold                Altogether, results indicate PQ5 used a <br /> between noninduced cumate-induced cultures obtained                   versatile inducible expression Sphingomonadales and <br /> as measured &#226;?&#164;-galactosidase assays. In cases in-              members orders Alphaproteobacteria. In <br /> duction ratio higher Sphingomonas Fr1,                   addition, results PQ2148 suggest Psyn2 easily ex- <br /> stemmed tighter repression higher pro-                 changed species-specific promoters achieve, exam- <br /> moter activity. Indeed, Sphingomonas Fr1 displayed highest                 ple, tighter repression higher expression levels cases where <br /> absolute &#226;?&#164;-galactosidase activity strains tested. In general,        Psyn2 does provide desired characteristics. <br /> speculate strain-specific differences induction ratios                 A comprehensive set cumate-inducible plasmids. In addi- <br /> absolute &#226;?&#164;-galactosidase activity attributed different               tion pQF, number PQ5-based plasmids con- <br /> strengths PQ5 and/or promoter driving expression CymR,               structed (see Table 1) carry, addition carboxy-termi- <br /> Pbla-mut1T, exclude possibility              nal 3&#226;&#171;&#187;FLAG tag, different amino-terminal epitope tags (HA, myc, <br /> are factors involved.                                                    fluorescent proteins) commonly used immunodetection or <br />     We tested PQ5 alphaproteobacterial species rep-              protein localization studies. In constructs, original reading <br /> resenting orders Caulobacterales (C. crescentus), Rhodobacte-              frame displayed Fig. 2B maintained. Besides conventional <br /> rales (P. denitrificans), Rhizobiales (M. extorquens A. tu-            cloning vectors, plasmids derivatives compatible the <br /> mefaciens). For species M. extorquens, PQ5 activity             Gateway cloning (Invitrogen) constructed. <br /> monitored using plasmid pQF-lacZ; M. extorquens, fragment                Last, constructed PQ5-based plasmid carrying broad- <br /> containing PQ5 cymR* subcloned previously de-                     host-range pBBR origins replication transfer be <br /> scribed bacterial luciferase (luxCDABE) transcriptional reporter               used bacteria pCM62-based plasmids suit- <br /> plasmid (27). Whereas PQ5 essentially inactive A. tumefa-               able, Bradyrhizobium japonicum (H.-M. Fischer, personal <br /> ciens (&#226;&#172;?2 Miller units cumate; data shown),                communication). Plasmids available Addgene <br /> the promoter inducible species tested induc-             plasmid repository (www.addgene.org). <br /> tion ratios approximately 1,000- 5,000-fold (Table 2 Fig.                Conclusions. We developed tight highly inducible gene <br /> 5B). As discussed sphingomonads, high induction                expression sphingomonads, first <br /> ratios tighter repression higher activity         described, demonstrated successful applica- <br /> PQ5. Similar results obtained Sphingomonas Fr1, M.                  tion depletion analysis. This adds valuable the <br /> extorquens response inducer addition rapid (apparent                genetic toolbox sphingomonads, anticipate will <br /> after approximately 10 min) expression tunable                  broad use basic applied research. In addition, <br /> wide range inducer concentrations. We tested derivative              demonstrated work number Alp- <br />  <br />  <br /> November 2013 Volume 79 Number 21                                                                                                        aem.asm.org 6801 <br />  Kaczmarczyk et al. <br />  <br />  <br />  <br /> haproteobacteria used complement                     19. Nunn DN, Lidstrom ME. 1986. Isolation complementation analysis <br /> existing inducible gene expression systems (3, 6, 40&#226;&#8364;&#8220;42).                              10 methanol oxidation mutant classes identification metha- <br />                                                                                        nol dehydrogenase structural gene Methylobacterium sp. strain AM1. J. <br /> ACKNOWLEDGMENTS                                                                        Bacteriol. 166:581&#226;&#8364;&#8220;590. <br />                                                                                    20. Poindexter JS. 1964. Biological properties classification Cau- <br /> We thank James Tauber performing initial experiments PhyP de-                   lobacter group. Bacteriol. Rev. 28:231&#226;&#8364;&#8220;295. <br /> pletion. We grateful David H. H. Chou, Denis Faure, Dieter Haas,            21. Smith EF, Townsend CO. 1907. A plant-tumor bacterial origin. Sci- <br /> Dominic Esposito, Hans-Martin Fischer, Tobias J. Erb, Urs Jenal                ence 25:671&#226;&#8364;&#8220; 673. <br /> providing strains and/or plasmids.                                                 22. Jin DJ, Gross CA. 1988. Mapping sequencing mutations Escherichia <br />     This work supported Swiss National Science Foundation                   coli rpoB gene lead rifampicin resistance. J. Mol. Biol. 202:45&#226;&#8364;&#8220;58. <br /> (SNF) research grant 31003A-135623.                                        23. Peyraud R, Kiefer P, Christen P, Massou S, Portais JC, Vorholt JA. <br />                                                                                        2009. Demonstration ethylmalonyl-CoA pathway using 13C <br /> REFERENCES                                                                             metabolomics. Proc. Natl. Acad. Sci. U. S. A. 106:4846 &#226;&#8364;&#8220; 4851. <br />  1. Guzman LM, Belin D, Carson MJ, Beckwith J. 1995. Tight regulation,             24. Kaczmarczyk A, Vorholt JA, Francez-Charlot A. 2012. Markerless gene <br />     modulation, high-level expression vectors containing arabi-             deletion sphingomonads. Appl. Environ. Microbiol. 78:3774 &#226;&#8364;&#8220; <br />     nose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220; 4130.                                  3777. <br />  2. Iniesta AA, Garcia-Heras F, Abellon-Ruiz J, Gallego-Garcia A, Elias-           25. Simon R, Priefer U, P&#195;&#188;hler A. 1983. A broad-host range mobilization <br />     Arnanz M. 2012. Two systems conditional gene expression Myxo-               vivo genetic engineering: transposon mutagenesis gram- <br />     coccus xanthus inducible isopropyl-beta-D-thiogalactopyranoside              negative bacteria. Nat. Biotechnol. 1:784 &#226;&#8364;&#8220;791. <br />     vanillate. J. Bacteriol. 194:5875&#226;&#8364;&#8220;5885.                                        26. Miller JH. 1972. Experiments molecular genetics. Cold Spring Harbor <br />  3. Meisenzahl AC, Shapiro L, Jenal U. 1997. Isolation characterization            Laboratory Press, Cold Spring Harbor, NY. <br />     xylose-dependent promoter Caulobacter crescentus. J. Bacteriol.      27. Metzger LC, Francez-Charlot A, Vorholt JA. 2013. Single-domain re- <br />     179:592&#226;&#8364;&#8220; 600.                                                                      sponse regulator involved general stress response Methylobacte- <br />  4. Quisel JD, Burkholder WF, Grossman AD. 2001. In vivo effects                    rium extorquens. Microbiology 159:1067&#226;&#8364;&#8220;1076. <br />     sporulation kinases mutant Spo0A proteins Bacillus subtilis. J. Bac-     28. Laemmli UK. 1970. Cleavage structural proteins assembly of <br />     teriol. 183:6573&#226;&#8364;&#8220; 6578.                                                            head bacteriophage T4. Nature 227:680 &#226;&#8364;&#8220; 685. <br />  5. Stephens C, Christen B, Watanabe K, Fuchs T, Jenal U. 2007. Regula-            29. Sambrook J, Russel D. 2001. Molecular cloning: laboratory manual, 3rd <br />     tion D-xylose metabolism Caulobacter crescentus LacI-type re-           ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. <br />     pressor. J. Bacteriol. 189:8828 &#226;&#8364;&#8220; 8834.                                        30. Vick JE, Johnson ET, Choudhary S, Bloch SE, Lopez-Gallego F, Sriv- <br />  6. Thanbichler M, Iniesta AA, Shapiro L. 2007. A comprehensive set                 astava P, Tikh IB, Wawrzyn GT, Schmidt-Dannert C. 2011. Optimized <br />     plasmids vanillate- xylose-inducible gene expression Caulobac-          compatible set BioBrick vectors metabolic pathway engineering. <br />     ter crescentus. Nucleic Acids Res. 35:e137. doi:10.1093/nar/gkm818.                Appl. Microbiol. Biotechnol. 92:1275&#226;&#8364;&#8220;1286. <br />  7. Yansura DG, Henner DJ. 1984. Use Escherichia coli lac repressor         31. Bloemberg GV, Wijfjes AH, Lamers GE, Stuurman N, Lugtenberg BJ. <br />     operator control gene expression Bacillus subtilis. Proc. Natl.          2000. Simultaneous imaging Pseudomonas fluorescens WCS365 popu- <br />     Acad. Sci. U. S. A. 81:439 &#226;&#8364;&#8220; 443.                                                  lations expressing different autofluorescent proteins rhizo- <br />  8. Browning DF, Busby SJ. 2004. The regulation bacterial transcription             sphere: new perspectives studying microbial communities. Mol. Plant <br />     initiation. Nat. Rev. Microbiol. 2:57&#226;&#8364;&#8220; 65.                                         Microbe Interact. 13:1170 &#226;&#8364;&#8220;1176. <br />  9. Choi YJ, Morel L, Bourque D, Mullick A, Massie B, Miguez CB. 2006.             32. Kovach ME, Elzer PH, Hill DS, Robertson GT, Farris MA, Roop RM, II, <br />     Bestowing inducibility cloned methanol dehydrogenase promoter               Peterson KM. 1995. Four new derivatives broad-host-range cloning vector <br />     (PmxaF) Methylobacterium extorquens applying regulatory elements           pBBR1MCS, carrying different antibiotic-resistance cassettes. Gene 166:175&#226;&#8364;&#8220;176. <br />     Pseudomonas putida F1. Appl. Environ. Microbiol. 72:7723&#226;&#8364;&#8220;7729.              33. Marx CJ, Lidstrom ME. 2001. Development improved versatile broad- <br /> 10. Choi YJ, Morel L, Le Francois T, Bourque D, Bourget L, Groleau D,                  host-range vectors use methylotrophs Gram-negative <br />     Massie B, Miguez CB. 2010. Novel, versatile, tightly regulated ex-             bacteria. Microbiology 147:2065&#226;&#8364;&#8220;2075. <br />     pression Escherichia coli strains. Appl. Environ. Microbiol. 76:    34. Heeb S, Itoh Y, Nishijyo T, Schnider U, Keel C, Wade J, Walsh U, <br />     5058 &#226;&#8364;&#8220;5066.                                                                        O&#226;&#8364;&#8482;Gara F, Haas D. 2000. Small, stable shuttle vectors based min- <br /> 11. Takeuchi M, Hamana K, Hiraishi A. 2001. Proposal genus Sphin-               imal pVS1 replicon use gram-negative, plant-associated bacteria. <br />     gomonas sensu stricto new genera, Sphingobium, Novosphingo-              Mol. Plant Microbe Interact. 13:232&#226;&#8364;&#8220;237. <br />     bium Sphingopyxis, basis phylogenetic chemotaxonomic         35. Markowitz VM, Chen IMA, Palaniappan K, Chu K, Szeto E, Grechkin <br />     analyses. Int. J. Syst. Evol. Microbiol. 51:1405&#226;&#8364;&#8220;1417.                             Y, Ratner A, Jacob B, Huang JH, Williams P, Huntemann M, Anderson <br /> 12. Balkwill DL, Fredrickson JK, Romine MF. 2006. Sphingomonas                     I, Mavromatis K, Ivanova NN, Kyrpides NC. 2012. IMG: integrated <br />     related genera, p 605&#226;&#8364;&#8220; 629. In Dworkin M, Falkow S, Rosenberg E, Schle-            microbial genomes database comparative analysis system. Nucleic <br />     ifer K-H, Stackebrandt E, (ed), The prokaryotes: handbook biol-           Acids Res. 40:D115&#226;&#8364;&#8220;D122. <br />     ogy bacteria, vol 7, proteobacteria, delta epsilon subclasses. Deeply   36. Bailey TL, Elkan C. 1994. Fitting mixture model expectation maxi- <br />     rooting bacteria. Springer-SBM, New York, NY.                                      mization discover motifs biopolymers, p 28 &#226;&#8364;&#8220;36. In Proceedings of <br /> 13. Fialho AM, Moreira LM, Granja AT, Popescu AO, Hoffmann K, Sa-                      Second International Conference Intelligent Systems Molecular <br />     Correia I. 2008. Occurrence, production, applications gellan: cur-          Biology. AAAI Press, Menlo Park, CA. <br />     rent state perspectives. Appl. Microbiol. Biotechnol. 79:889 &#226;&#8364;&#8220;900.         37. Corpet F. 1988. Multiple sequence alignment hierarchical clustering. <br /> 14. Innerebner G, Knief C, Vorholt JA. 2011. Protection Arabidopsis thaliana        Nucleic Acids Res. 16:10881&#226;&#8364;&#8220;10890. <br />     leaf-pathogenic Pseudomonas syringae Sphingomonas strains      38 <br /> </body></html>