<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol.19 No.7 pp.1691&#226;&#8364;&#8220;1702, 2000 <br />  <br />  <br />  <br /> A novel SMC protein complex in <br /> Schizosaccharomyces pombe contains Rad18 <br /> DNA repair protein <br />  <br />  <br /> Maria I.Fousteri Alan R.Lehmann1                             arresting cell cycle following damaging treatments <br />                                                                  (for review, Caspari Carr, 1999). <br /> MRC Cell <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>Mutation</span> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, University Sussex, Falmer, Brighton      In previous work used rad18-X mutant show <br /> BN1 9RR, UK <br />                                                                  Rad18 protein involved repair ionizing <br /> 1Corresponding   author                                          radiation damage (Lehmann et al., 1995). Double-strand <br /> e-mail: a.r.lehmann@sussex.ac.uk <br />                                                                  breaks principal lesions responsible killing <br />                                                                  cells ionizing radiation. Both Verkade et al. (1999) and <br /> In Schizosaccharomyces pombe, rad18 essential              (unpublished observations) shown rad18-X <br /> gene involved repair DNA damage produced               cells deficient repair radiation-induced double- <br /> by ionizing radiation tolerance UV-induced             strand breaks. <br /> DNA damage. The Rad18 protein member                        Epistasis analysis response UV-irradiation suggests <br /> the SMC (structural maintenance chromosomes)                  Rad18 involved NER, does play <br /> superfamily, that, like SMC                role second excision repair pathway UV <br /> proteins condensin cohesin, Rad18 compon- <br />                                                                  damage. The step pathway mediated UV <br /> ent high-molecular-weight complex. This complex <br />                                                                  damage-endonuclease (UVDE) (Yonemasu et al., 1997), <br /> contains proteins, largest which <br />                                                                  nicks UV-irradiated DNA 5&#226;&#172;&#732; UV <br /> is Spr18, novel SMC family member closely related <br />                                                                  photoproducts (Avery et al., 1999; Yoon et al., 1999). In <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> Rad18, likely heterodimeric partner. <br />                                                                  uvde background, rad18-X mutation sensitized <br /> SMC proteins ATP-binding domains the <br />                                                                  cells UV-irradiation (Yonemasu et al., 1997), and <br /> N- C-termini, extended coiled-coil domains <br />                                                                  epistasis analysis suggested a <br /> separated hinge middle. We the <br />                                                                  role Rad18 DNA damage tolerance pathway <br /> N-terminal ATP-binding domain Rad18 essential <br />                                                                  (Murray et al., 1997). Furthermore, deletion rad18 <br /> for functions, overexpression N-terminal <br />                                                                  gene demonstrated essential cell prolifera- <br /> mutant dominant-negative effect. We identi- <br />                                                                  tion, data led propose tentatively this <br /> fied important mutation (S1045A) near the <br /> C-terminus Rad18 separates repair                essential role involvement DNA replication <br /> essential roles. Potential models role            (Lehmann et al., 1995). Figure 1 summarizes pathways <br /> Rad18&#226;&#8364;&#8220;Spr18 complex DNA repair dis-                   Rad18 involved, based genetic analysis. <br /> cussed.                                                             Sequence analysis Rad18 (Lehmann et al., 1995) <br /> Keywords: ATPase/coiled coils/DNA repair/fission yeast/          showed 131 kDa Rad18 protein member <br /> SMC proteins                                                     SMC (structural maintenance chromosomes) <br />                                                                  superfamily. SMC proteins globular N- and <br />                                                                  C-terminal domains, involved ATP Mg2&#226;&#171;&#185; <br />                                                                  binding, extended &#206;&#177;-helical coiled-coil domains <br />                                                                  involved protein&#226;&#8364;&#8220;protein interactions, separated a <br /> Introduction                                                     hinge region (for reviews, Hirano, 1998, 1999; <br /> Many genes involved response DNA damage           Jessberger et al., 1998). Recent work using Saccharomyces <br /> been identified fission yeast Schizosaccharomyces         cerevisiae, S.pombe Xenopus laevis identified two <br /> pombe (see Lehmann, 1996). Epistasis analysis enabled        protein complexes containing heterodimeric pairs SMC <br /> these genes assigned different DNA repair               proteins. Smc2 Smc4 (and orthologues) form <br /> pathways. Following treatment ionizing radiation,           core condensin, five-subunit complex involved in <br /> repair DNA double-strand breaks requires genes known          chromosome condensation (Hirano et al., 1997; Sutani <br /> to involved homologous recombination, suggesting           et al., 1999). Smc1 Smc3 (and orthologues) are <br /> that mechanism likely major means           core five-subunit cohesin complex, is <br /> repairing DNA damage fission yeast. Photoprod-           required sister chromatid cohesion (Michaelis et al., <br /> ucts induced DNA UV-irradiation repaired            1997; Losada et al., 1998). There evidence <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> both classical nucleotide excision repair (NER)           bovine cells suggest SMC1 SMC3 proteins <br /> system, conserved eukaryotes,           form complex DNA ligase III DNA <br /> second excision repair process, specific             polymerase &#206;&#181;, DNA strand exchange activity <br /> S.pombe organisms (Yonemasu et al.,              (Jessberger et al., 1996). <br /> 1997; Yasui McCready, 1998). In addition               We paper purification high- <br /> pathways repair DNA damage directly, cells able         molecular-weight complex DNA-dependent ATPase <br /> to tolerate UV damage DNA replication (Murray             activity, contains Rad18 other <br /> et al., 1997) utilize cell cycle checkpoint mechan-       polypeptides. The largest novel SMC protein <br /> isms ameliorate deleterious effects DNA damage         (Spr18), likely heterodimeric partner <br />  <br /> &#194;&#169; European Molecular Biology Organization                                                                              1691 <br />  M.I.Fousteri A.R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 1. Involvement Rad18 different cellular responses DNA <br /> damage. <br />  <br />  <br /> of Rad18. We site-directed mutagenesis the <br /> N-terminal ATP-binding domain Rad18 required for <br /> all functions, C-terminal mutation can <br /> separate repair essential functions. <br />  <br /> Results <br /> Rad18 high-molecular-weight complex <br /> To determine Rad18, like SMC proteins, is <br /> a member protein complex, fractionated extracts <br /> of S.pombe gel filtration columns. For experiments <br /> we constructed cell strain myc-Rad18, rad18 <br /> gene myc-tagged genome 5&#226;&#172;&#732; end, <br /> remaining control promoter. When <br /> extracts myc-Rad18 cells passed Superdex <br /> 200 column, Rad18 protein high- <br /> molecular-weight peak (Mr &#226;&#172;&#381;700 kDa). To obtain more <br /> accurate estimate size complex, peak               Fig. 2. Rad18 high-molecular-weight complex. Extracts of <br /> fractions column pooled analysed                 myc-Rad18 (A) myc-Rad18-X cells (C) precipitated with <br /> a Superose 6 column (Figure 2A). The results                ammonium sulfate, fractionated Superdex 200 high- <br />                                                                       molecular-weight peak analysed gel filtration Superose 6 <br /> Rad18 complex apparent Mr ~1.6 MDa.             column. Fractions subjected immunoblotting anti-myc <br /> This complex stable presence 0.7 M                antibody. The high-molecular-weight band gel non-specific <br /> NaCl (not shown). Using polyclonal antibody raised                  cross-reacting material. (B) Wild-type strain 501 subjected the <br /> against Rad18 peptide, showed gel                  treatment above, analysed anti-Rad18 antibody. <br /> filtration profile identical Rad18 <br /> protein N-terminally myc-tagged (Figure 2A B). <br /> No change gel filtration profile detected             ive washing, bound proteins eluted peptide <br /> extracts &#206;&#179;-irradiated cells used (data shown).            corresponding c-myc epitope, analysed both <br />     As discussed below, mutation rad18-X located            immunoblotting colloidal Coomassie staining. <br /> in hinge region, immediately adjacent second               Immunoblotting showed bound eluted proteins <br /> coiled-coil region (see Figure 5A). This raised possibil-         contained myc-tagged Rad18 (Figure 3B, lane 2). When <br /> ity mutation disrupt coiled-coil structure         extracts untagged control cells (strain 501) were <br /> of protein interfere complex formation. The              subjected identical purification procedures, corres- <br /> Rad18 profiles gel filtration columns using normal                 ponding protein bound beads (Figure 3B, <br /> and rad18-X extracts were, however, indistinguishable                 lane 4). Coomassie staining revealed that, apart from <br /> (compare Figure 2C A), salt concentrations                Rad18, specific bands eluate from <br /> up 0.7 M NaCl. The rad18-X mutation does not,                      tagged cells (Figure 3C, lane 3) found <br /> therefore, interfere complex formation.                          similar experiments using untagged control cells <br />                                                                       (Figure 3C, lanes 1 2). One specific bands, <br /> Purification Rad18                                                 p124, molecular weight close Rad18 and <br /> We developed purification procedure Rad18-             similar intensity. <br /> containing complex, using extracts myc-Rad18 cells <br /> (Figure 3A). After ammonium sulfate precipitation,                ATPase activity <br /> extracts subjected gel filtration Superdex 200.            Mutagenesis studies described implicate <br /> The high-molecular-weight fractions containing Rad18                  ATP-binding activity Rad18 functions, sug- <br /> were pooled fractionated heparin&#226;&#8364;&#8220;Sepharose                   gesting protein ATPase activity. We <br /> column. The 0.6 M KCl fractions, contained                  measured ATPase activity Rad18 <br /> Rad18 protein, incubated protein G beads                    complex. We assayed peptide eluate immuno- <br /> covalently cross-linked anti-myc antibody. After extens-           affinity beads ATPase activities presence and <br />  <br /> 1692 <br />                                                                                                Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 3. The Rad18 complex. (A) Scheme purification Rad18 complex myc-Rad18 cell extracts (WCE, cell extracts; FT, flow- <br /> through). (B) Samples proteins loaded (Load) eluted (B&#226;&#8364;&#8482;nd) anti-myc immuno-affinity beads immunoblotted anti- <br /> myc anti-Spr18 antibodies. (C) Proteins bound beads eluted myc peptide analysed SDS&#226;&#8364;&#8220;PAGE. The gel fixed and <br /> stained colloidal Coomassie Blue. Lanes 1 2, extracts strain 501; lane 3, myc-Rad18. (D) ATPase activity Rad18 complex. <br /> Experiments carried times, typical results shown. <br />  <br />  <br /> absence DNA. Eluate myc-Rad18 cell extracts                    candidate open reading frame (ORF) cosmid <br /> had significant ATPase activity, stimulated                   C14C4 chromosome 1 (DDBJ/EMBL/GenBank <br /> single-stranded, greater extent double-stranded                accession No. Z98596) orthologue the <br /> DNA (Figure 3D). In contrast, eluate untagged                         S.cerevisiae database (ORF SCYOL034W; DDBJ/EMBL/ <br /> control strain 501 containing equal non-                  GenBank accession No. Z74776). The predicted 123 kDa <br /> specific 73 kDa protein (see Figure 3C) minimal                        protein related SMC family, most <br /> activity, unchanged addition DNA.                      closely related Rad18 (see Figure 4A). We designated <br /> The complex associated specific DNA-                     Spr18 (SMC partner rad18). Alignments Rad18 <br /> stimulated ATPase activity.                                                Spr18 S.cerevisiae orthologue shown in <br />                                                                            Figure 4B. There 28% identity S.pombe <br /> Spr18, heterodimeric partner Rad18                                    S.cerevisiae orthologues, 20% identity between <br /> In SMC family, Smc2 Smc4 form heterodimeric                      Rad18 Spr18. Like rad18, spr18 small intron <br /> component condensin complex (Hirano et al., 1997),                  (49 bases) close 5&#226;&#172;&#732; end gene (between nt 187 <br /> which involved chromosome condensation. Smc1                         188). <br /> and Smc3 similarly cohesin, involved sister                    In order determine Spr18 interacts with <br /> chromatid cohesion (Michaelis et al., 1997). We                  Rad18 vivo, carried immunoprecipitations using <br /> anticipated Rad18 heterodimeric partner,                 myc-Rad18 strain (Figure 4C). Extracts immuno- <br /> and considered possibility p124 polypeptide                precipitated antibody directed peptide from <br /> in complex novel SMC protein partner                     Spr18 (Figure 4C, lanes 2, 4, 6 8) or, control, with <br /> Rad18. We searched S.pombe sequence database                       anti-myc antibody (lanes 1, 3, 5 7). The immuno- <br />  <br />                                                                                                                                          1693 <br />  M.I.Fousteri A.R.Lehmann <br />  <br />  <br />  <br />  <br /> 1694 <br />                                                                                                 Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> precipitates electrophoresed SDS&#226;&#8364;&#8220;polyacryl-                      Viability Ura4&#226;&#171;&#185; <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>phenotype</span> segregated 2:0 <br /> amide gel immunoblotted anti-myc antibody.                     manner, confirming spr18, like rad18 SMC <br /> Figure 4C shows myc-tagged Rad18 co-immunopre-                         genes, essential cell proliferation. In order to <br /> cipitated Spr18 experiments presence                   determine terminal phenotype spr18 deletion <br /> of salt concentrations 0.75 M (lanes 1&#226;&#8364;&#8220;6). Similar                    strain, spores germinated liquid medium with <br /> results obtained blots probed                       selection Ura4&#226;&#171;&#185;, cell number morphology <br /> anti-Spr18 (data shown). The results affected                  determined. Germinating spores underwent or <br /> by treatment cell extracts DNase I prior                     divisions ceased growth variety abnormal <br /> immunoprecipitation, demonstrating interaction                     morphologies. Several examples shown Figure 4D. <br /> was mediated binding DNA. When immuno-                        Many cells elongated, cases excessively so <br /> precipitation experiments carried control                     (Figure 4D, examples 1 5), septa <br /> untagged strain 501, bands detected                          (examples 3 5), nuclear material barely <br /> immunoblot (Figure 4C, lanes 7 8). These results                        detectable, present single nucleus <br /> demonstrate Rad18 Spr18 interact,                        (example 4) spread amorphously the <br /> consistent idea proteins form                       cell (example 1). A proportion cells (10&#226;&#8364;&#8220;15%) showed <br /> heterodimer S.pombe cells,                          cut phenotype (Figure 4D, example 2). The distribution <br /> formally eliminated possibility interaction                    different phenotypes shown Figure 4E. The <br /> might mediated protein. In results                      similarity terminal phenotype spr18 deletion <br /> presented elsewhere, shown similar interaction                    mutant, rad18 deletion (Lehmann et al., 1995) wild- <br /> between human orthologues proteins                             type cells overexpressing dominant-negative mutants of <br /> (E.M.Taylor, J.Moghraby A.R.Lehmann, unpublished                        rad18 (see below), provides evidence Rad18 <br /> observations).                                                              Spr18 perform similar functions cell. <br />  <br /> Spr18 p124                                                           Mutagenesis rad18 defines important structural <br /> The peptide elution fractions immuno-affinity                      domains <br /> column obtained using myc-Rad18 501                            The characteristic SMC structure Rad18, globular <br /> extracts (Figure 3) analysed immunoblotting                    head tail regions coiled-coil domains separated <br /> the anti-Spr antibody. Spr18, like Rad18, detected                   hinge, depicted schematically Figure 5A. To <br /> the bound eluted proteins myc-Rad18 extract                    gain insight roles different domains, we <br /> (Figure 3B, lane 6), untagged control                        used site-directed mutagenesis introduce series <br /> extracts (lane 8). The band detected using anti-Spr18                   mutations rad18 cDNA. The cDNA cloned <br /> antibody corresponded p124 protein                      control attenuated thiamine-repressible <br /> the Coomassie-stained gel. These results confirm p124                  nmt1 promoter vectors pRep41MH N (Leu&#226;&#171;&#185;) or <br /> is SMC protein Spr18 consistent                        pRep42MH N (Ura&#226;&#171;&#185;) (Maundrell, 1993; Craven et al., <br /> suggestion Rad18&#226;&#8364;&#8220;Spr18 forms heterodimeric core                      1998), Rad18 protein N-terminally myc- <br /> of large complex similar SMC complexes,                      tagged. The effects mutations examined in <br /> cohesin condensin. To investigate Rad18&#226;&#8364;&#8220;                    different ways, follows: (i) ability mutant <br /> Spr18 complex contains known DNA damage                               plasmids rescue lethal effect rad18 deletion <br /> response genes, carried immunoblotting                        examined transformation rad18::ura4&#226;&#171;&#185;/ <br /> proteins eluted anti-myc affinity column using                   rad18&#226;&#171;&#185; diploid strain; (ii) mutated plasmids were <br /> series antibodies. We unable detect presence                 overexpressed wild-type cells look dominant- <br /> of DNA damage response proteins Rad9, Rad26,                            negative effects mutant gene cell growth and <br /> Cds1, Rqh1 Rad21 Rad18&#226;&#8364;&#8220;Spr18 complex (data                        viability, sensitivity UV- &#206;&#179;-irradiation; <br /> not shown).                                                                 (iii) mutated genes introduced rad18-X <br />                                                                             cells impact sensitivity mutant to <br /> Spr18 essential gene                                                  ionizing UV-irradiation examined following their <br /> If Spr18 partner protein Rad18, anticipated                    transient overexpression. <br /> that, like Rad18, essential gene. We deleted                    The mutations effects summarized in <br /> the entire ORF spr18 gene genome                       Figure 5 Table I. Several mutant constructs <br /> diploid strain replacing copy gene                   dominant-negative effect cell viability <br /> ura4&#226;&#171;&#185; marker. Following sporulation plating                      (Figure 5A). The proportion population lost <br /> absence uracil, haploid colonies obtained,                       viability depended mutation, phenotype of <br /> suggesting spr18 gene essential cell                       dying cells quite similar cases, resembled <br /> proliferation. In order confirm this, tetrads dis-                  terminal phenotype deletion mutants rad18 <br /> sected spores plated yeast extract plates.                  (Lehmann et al., 1995) spr18 (see above). For simplicity <br />  <br />  <br /> Fig. 4. Spr18, SMC partner protein Rad18. (A) Phylogenetic tree SMC family members S.cerevisiae S.pombe. (B) Sequence <br /> alignments Rad18 Spr18 S.cerevisiae orthologue (designated scspr). Identical amino acids highlighted black, conserved <br /> residues grey. (C) Extracts myc-Rad18 cells immunoprecipitated anti-myc anti-Spr18 antibody presence the <br /> salt concentrations indicated. The immunoprecipitates immunoblotted probed anti-myc antibody. The right hand lanes (7 8) show <br /> results untagged strain 501. (D) Terminal phenotypes spr18 deletion mutant. Diploid spr18&#226;&#171;&#185;/spr18::ura4&#226;&#171;&#185;cells sporulated and <br /> grown absence uracil 36 h. Cells fixed stained DAPI calcofluor. (E) Frequency different phenotypes 24 h <br /> growth. <br />  <br />  <br />                                                                                                                                           1695 <br />  M.I.Fousteri A.R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 5. Effects site-directed mutations Rad18. (A) Schematic Rad18 protein positions mutations, ability rescue lethal <br /> phenotype rad18 deletion mutant effect overexpression viability wild-type strain (&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;, cells dead; <br /> &#226;&#171;&#185;&#226;&#171;&#185;, cells dead retain viability; &#226;&#171;&#185;, 30&#226;&#8364;&#8220;40% cells retain viability). (B C) Survivals &#206;&#179;- UV-irradiation wild-type (B) or <br /> rad18-X cells (C) overexpressing different mutant plasmids. Experiments carried times, typical results shown (Mrep42, empty <br /> pREP42 MH vector). <br />  <br />  <br />  <br /> we refer Rad18 phenotype,                         small, septated prior mitosis, resulting <br /> there intended implication phenotype                        long cells nucleus only <br /> unique rad18 mutants. This phenotype consisted                           nucleus multiple septa. Examples shown <br /> cells variety abnormal morphologies                         Figure 6A (compare overexpressed wild-type <br /> nuclear segregation clearly aberrant. Many                         rad18 Figure 6B). <br />  <br /> 1696 <br />                                                                                                    Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 6. Dominant-negative effects mutations rad18 gene. (A B) Effects overexpressing (A) rad18K129Q, rad18L843P or <br /> rad18R706A (B) wild-type rad18 cell growth morphology. (C) Wild-type cells overexpressing wild-type rad18, rad18K129E <br /> mutant plasmid, both, incubated absence thiamine cell numbers counted 24 h time period. (D) Western blots cells <br /> overexpressing myc-tagged rad18K129E (lane 1) myc-tagged rad1 (lane 2), rad2 (lane 3), rad3 (lane 4), pcn1 (lane 5), rad17 (lane 7) <br /> and HA-tagged cdc6 (lane 6). <br />  <br />  <br />  <br /> Table I. Summary effects overexpressing mutant constructs <br />  <br />                       K129E          K129Q          R706C         R706A          L843P          L843F             S1045A   S1045T       D1072A <br />  <br /> WT viabilitya         0              35             &#226;&#171;&#185;             42             20             &#226;&#171;&#185;                 &#226;&#171;&#185;        &#226;&#171;&#185;            &#226;&#171;&#185; <br /> WT UVb                dead           S              N             S              dead           N                 S        S            N <br /> WT &#206;&#179; b                dead           S              N             S              dead           N                 S        N            N <br /> Rad18-X UVc           dead           SS             SS            SS             SS             R                 SS       SS           SS <br /> Rad18-X &#206;&#179; c           dead           SSS            SS            SSS            SS             R                 SSS      SSS          SS <br /> Rad18&#226;&#710;&#8224;d                                       yes           partial                    yes               yes                no <br /> aNumbers  denote percentage viable cells. &#226;&#171;&#185;, &#226;&#172;&#381;70% viability. <br /> bN, normal; S, sensitive normal. <br /> cR,restores wild-type resistance; SS, sensitivity comparable rad18-X; SSS, sensitivity greater rad18-X. <br /> dYes, does; no, does rescue lethality deletion strain. <br />  <br />  <br />  <br />  <br />    The N-terminal domain Rad18 contains highly                         liquid culture, cell growth prevented (Figure 6C) and <br /> conserved Walker Type A box ATPases.                            cells died Rad18 phenotype. <br /> We mutated conserved Lys129 residue GXGKS                             We generated milder mutation Lys129 to <br /> motif glutamic acid (rad18K129E). When rad18K129E                          glutamine (K129Q). When plasmids containing con- <br /> was overexpressed wild-type cells, dramatic                       struct overexpressed wild-type cells, resulting <br /> dominant-negative effect viability, completely abol-                       colonies small, liquid culture, significant <br /> ishing colony-forming ability cells. A similar                     proportion cells (65%) showed Rad18 phenotype <br /> result recently obtained different mutation                    reduced viability (Figure 6A). The viable cells <br /> in ATP-binding site Verkade et al. (1999). When                        sensitized UV- ionizing radiation <br /> the rad18K129E overexpressed cells growing                          (Figure 5B). The mutant plasmid failed rescue the <br />  <br />                                                                                                                                              1697 <br />  M.I.Fousteri A.R.Lehmann <br />  <br />  <br /> viability deletion mutant slightly increased     mutations rad18. Mutations ATP-binding site in <br /> sensitivity rad18-X cells &#206;&#179;-irradiation (Figure 5A    N-terminus disruption coiled-coil domain <br /> and C).                                                         dramatic effects, suggesting ATP- <br />    The C-terminal globular region contains motifs LSGG          binding site coiled-coil structures essential all <br /> and DE, conserved SMC proteins, probably      functions. Effects mutations conserved C-terminal <br /> representing Walker Type B motif involved        domain severe. The S1045A mutation the <br /> ATP&#226;&#8364;&#8220;Mg2&#226;&#171;&#185; binding. We mutated Ser1045 LSGG              conserved LSGG motif enabled separate repair from <br /> sequence alanine threonine (rad18S1045A         essential functions protein, affected <br /> rad18S1045T), Asp1072 alanine (rad18D1072A).             former. <br /> Overexpression mutated genes significant <br /> effect viability wild-type cells. However,        Overexpression damage response genes <br /> rad18S1045A mutant plasmid sensitized cells         rescues dominant-negative phenotype <br /> &#206;&#179;- UV-irradiation damage, rad18S1045T               We looked genetic interactions rad18 <br /> had effect rad18D1072A minimal                     genes examining ability suppress the <br /> effect (Figure 5B). The S1045A S1045T plasmids              dominant-negative effects K129E mutation the <br /> slightly increased &#206;&#179;-irradiation sensitivity         ATP-binding site. If normal mutant cDNAs were <br /> rad18-X mutant (Figure 5C). Importantly, however,           co-overexpressed separate plasmids same <br /> rad18S1045A mutant plasmid, others,          nmt1 promoter (in pRep41 42), normal plasmid <br /> was able rescue lethality rad18 deletion          able rescue growth-inhibitory effects the <br /> strain (Figure 5A). This significant result shows          mutant plasmid (Figure 6C). We used co-expression <br /> the S1045A mutation able separate repair         investigate genes (rad1, <br /> the essential phenotype cells. It possible   rad2, rad3, rad17, pcn1 cdc6) rescue the <br /> rad18 mutant plasmid rescued rad18 deletion         adverse effects mutant plasmid. We that <br /> the mutant gene overexpressed. However, results          cell cycle checkpoint genes rad1 rad3, <br /> to presented elsewhere, eliminated         rad2 gene encoding structure-specific nuclease the <br /> this possibility. We integrated S1045A mutation        cdc6 gene encoding DNA polymerase &#206;&#180; overexpressed <br /> into genomic rad18 gene, expressed        rad18K129E, growth inhibition was <br /> normal levels. This strain viable sensitive           overcome, pcn1 encoding PCNA check- <br /> radiation.                                                      point gene rad17 effect. In order ensure that <br />    Coiled-coil motifs amphipathic &#206;&#177;-helices character-      merely result second plasmid <br /> ized heptad repeats hydrophobic amino acids          reducing expression mutant gene, used <br /> the fourth residues heptad (Lupas et al.,     Western blotting determine levels expression <br /> 1991). We mutated Leu843, middle long heptad        14 h absence thiamine. Since genes <br /> repeat, proline (L843P), expected      myc-tagged (in cases), able to <br /> to destroy coiled-coil structure, phenylalanine       detect Rad18 co-expressing proteins using <br /> (L843F), hydrophobic like leucine,           antibody directed myc epitope tag. The <br /> completely different structure. The mutation behaved     expression mutant rad18K129E affected <br /> like wild type respects (Figure 5B): restored         co-expression second plasmid (Figure 6D). <br /> viability rad18 deletion strain rescued          Our data provide evidence genetic inter- <br /> sensitivity DNA damage rad18-X cells                  action rad18 rad1, rad2, rad3 cdc6. <br /> (Figure 5C). In contrast, L843P dramatic effect.          These interactions studied in future work. <br /> Only wild-type cells overexpressing plasmid <br /> were able form colonies, small,          Discussion <br /> in liquid culture, cells (80%) Rad18 <br /> phenotype. Viability deletion strain restored    Rad18&#226;&#8364;&#8220;Spr18 SMC protein complexes <br /> with mutant plasmid.                                       Although understanding DNA repair pro- <br />    The mutation rad18-X, sensitive UV-       cesses increased dramatically years, <br /> &#206;&#179;-irradiation elevated temperatures, results         functions DNA repair proteins poorly <br /> change R706C. We considered possibility           understood. As shown scheme Figure 1, Rad18 <br /> change exert effects mutant protein       plays important role repair processes, its <br /> contains cysteine residue, form aberrant          exact function unknown. The SMC protein family <br /> disulfide bridges. We mutated Arg706 alanine       members essential chromosome condensation <br /> (R706A) compared effects R706C.           (SMC2 4) cohesion (SMC1 3) (see Hirano, <br /> Overexpression R706C effect wild-           1998; Jessberger et al., 1998), role these <br /> type rad18-X cells restored viability         proteins play molecular level poorly understood. <br /> deletion strain. The effects R706A were, however,       We shown paper Rad18 resembles other <br /> severe, phenotype quite similar            SMC proteins forms heterodimeric complex <br /> rad18K129Q (see Figure 5B C Table I). These             closely related partner protein, Spr18, Rad18&#226;&#8364;&#8220; <br /> findings ruled possibility phenotype        Spr18 higher molecular weight complex. This <br /> rad18-X resulted aberrant disulfide bridges,           complex molecular weight &#226;&#172;&#381;1 MDa, stable in <br /> rather implicated R706 important residue              high salt, contains proteins has <br /> Rad18 function.                                                 DNA-stimulated ATPase activity. Similarly, condensins <br />    In summary, gradation severity         Xenopus S.pombe contain multiple subunits <br />  <br /> 1698 <br />                                                                                     Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> including, Xenopus, XCAP-C XCAP-E (Hirano <br /> et al., 1997) S.pombe, Cut3 Cut14 (Tanaka <br /> et al., 1999), orthologues SMC2 SMC4. <br /> Likewise cohesins, include SMC1 SMC3 <br /> (Losada et al., 1998), dosage compensation <br /> complex Caenorhabditis elegans, contains the <br /> SMC protein DPY27 (Chuang et al., 1996), contain <br /> multiple subunits. A complex designated RC-1, was <br /> able carry strand exchange reactions, composed of <br /> bovine SMC1 SMC3 DNA polymerase &#206;&#181; <br /> and DNA ligase III (Jessberger et al., 1996). Both Rad18 <br /> and Spr18 orthologues S.cerevisiae and <br /> C.elegans databases isolated human ortholog- <br /> ues (E.M.Taylor, J.Moghraby A.R.Lehmann,             Fig. 7. Models involvement Rad18&#226;&#8364;&#8220;Spr18 complex DNA <br /> unpublished results). In view similarity structural    repair. The diagram shows roles SMC proteins chromosome <br />                                                                  condensation cohesion proposed Hirano (1999), two <br /> composition Rad18&#226;&#8364;&#8220;Spr18 complex, con-                possible roles Rad18&#226;&#8364;&#8220;Spr18 repair double-strand breaks. <br /> densin cohesin, consider Rad18&#226;&#8364;&#8220;Spr18            Black lines represent double-stranded DNA, grey symbols <br /> third conserved SMC complex eukaryotes.                       represent SMC proteins globular N- C-termini hinge <br />                                                                  regions, extended coiled coils between. <br /> Structure&#226;&#8364;&#8220;function relationships Rad18 <br /> Structural studies bacterial SMC-like proteins           1990; Gregory et al., 1991). This mutation provides an <br /> been carried recently Melby coworkers (Melby          opportunity investigation repair functions <br /> et al., 1998). MukB SMC-like protein              Rad18 homologues. <br /> Escherichia coli antiparallel homodimeric <br /> structure, globular domains attached rigid           Function Rad18&#226;&#8364;&#8220;Spr18 complex <br /> coiled-coil domains hinge. The coiled-coil regions        The SMC protein complexes function chromosome- <br /> form basis dimerization. The hinge appeared          associated processes, condensation, cohesion and <br /> quite flexible, permitting scissoring movement. It          dosage compensation. It thought putative <br /> probable Rad18 complex, Rad18 Spr18              scissoring action, possible flexibil- <br /> form antiparallel heterodimers.                                  ity SMC dimer (Melby et al., 1998), enables the <br />    Site-directed mutagenesis studies (Figures 5 6)      complexes DNA molecules relative other <br /> shown globular N-terminal domain coiled-        cellular structures. The condensin complex <br /> coil regions required repair essential          Xenopus able supercoil DNA presence <br /> functions Rad18. This consistent structure        topoisomerase I (Kimura Hirano, 1997) to <br /> suggested above. If assume Rad18 forms similar         effect topological reconfigurations DNA molecules <br /> structure heterodimer Spr18, provide        (Kimura et al., 1999). The S.cerevisiae SMC1p and <br /> a ready explanation dominant-negative effects         SMC2p proteins able bind DNA, DNA- <br /> overexpressed mutant Rad18 (Figure 6 Verkade et al.,         binding activity resides C-terminus molecule <br /> 1999). In majority mutants coiled-       (Akhmedov et al., 1998). We Rad18 binds <br /> coil domains remain intact, overexpressed mutant             tightly DNA-cellulose columns (our unpublished data), <br /> Rad18 protein dimerize sequester            suggesting Rad18 likely DNA-binding <br /> cellular Spr18, preventing dimerizing           protein, consistent DNA-stimu- <br /> endogenous protein. The L843P mutant probably causes             lated ATPase activity associated with <br /> local disruption &#206;&#177;-helical structure          Rad18&#226;&#8364;&#8220;Spr18 complex. We previously suggested that <br /> C-terminal coiled coil, N-terminal coiled-          Rad18 involved recombinational repair processes <br /> coil domain likely unaffected capable      (Lehmann et al., 1995). Our current work with <br /> interacting Spr18.                                          published data functions SMC proteins lead us <br />    Interestingly, mutations conserved C-terminal          speculate function Rad18&#226;&#8364;&#8220;Spr18 complex <br /> globular domain relatively mild effects. Particularly        bring and/or hold broken DNA molecules <br /> interesting S1045A mutation (Figure 5),             vicinity double-strand damage allow <br /> destroyed repair function protein, whilst retaining   repair recombination place (see Figure 7). Such <br /> its essential function. This quite surprising view        damage double-strand break produced <br /> of fact motif LSGG, encompassing Ser1045,           directly ionizing radiation, UV photoproduct <br /> is conserved SMC proteins (with exception          opposite gap stalled replication fork generated during <br /> Spr18, contains sequence QSGG). It             replication UV damage. We envisage Rad18&#226;&#8364;&#8220; <br /> found large family ATP transporter proteins (Ames        Spr18 complex holds ends break together <br /> et al., 1992), proposed LSGG       alternatively holds broken chromosome register <br /> motif forms important linker region different         sister homologue allow Rhp51 RecA <br /> domains. In protein family serine residue      homologue initiate strand exchange. In context the <br /> essential transporter function S.cerevisiae       Rad18&#226;&#8364;&#8220;Spr18 complex presumably plays complementary <br /> Ste6p protein involved transport a-factor phero-       role cohesins, known hold sister <br /> mone (Browne et al., 1996), mutated CFTR        chromatids together, required normal <br /> protein cystic fibrosis patients (Kerem et al.,       response radiation, mutants S.pombe rad21, a <br />  <br />                                                                                                                             1699 <br />  M.I.Fousteri A.R.Lehmann <br />  <br />  <br /> non-SMC component cohesin, sensitive radiation.                      Immunoprecipitation <br />                                                                                For experiments constructed S.pombe strain myc-tagged in <br /> Recent data suggest cohesin complex present                        genome (myc-Rad18). A fragment DNA containing 950 bp of <br /> at specific sites chromosome arms                        rad18 upstream sequence coding sequence NcoI site <br /> coating lengths chromosomes (Blat                               nt 1760 cloned integrating vector pSta18 (Carr et al., <br /> Kleckner, 1999; Tanaka et al., 1999). In context                        1989), contains sup3.5 tRNASerUGA insert (Hottinger, 1982). <br /> recombinational repair double-strand DNA damage,                         The His6-myc2 coding sequence inserted NdeI fragment from <br />                                                                                pGEMMH (Craven et al., 1998) NdeI site engineered start <br /> may cohesins necessary maintain overall                     rad18 ORF. This plasmid transformed S.pombe strain <br /> cohesion sister chromatids, Rad18&#226;&#8364;&#8220;                          501 (leu1.32 ura4.D18 ade6.704) selection applied growth <br /> Spr18 complex specific role sites damage.                     absence adenine. (The ade6.704 mutation 501 strain is <br /> The suggestion role Rad18 recombination                         suppressed sup3.5.) Ade&#226;&#171;&#185; colonies grown analysed the <br /> supported 4-fold reduction intrachromosomal                          integration tagged sequence Southern blotting immuno- <br />                                                                                blotting. A similar procedure followed S.pombe rad18-X <br /> recombination recently reported Rad18 mutant                           myc-tagged strain (myc-Rad18-X). <br /> Arabidopsis thaliana (Mengiste et al., 1999). A                              For immunoprecipitations, crude extract containing 2 mg total <br /> complex role Rad18 response DNA damage                           protein incubated overnight anti-Spr18 antibody or <br /> been suggested work Verkade et al. (1999),                       9E10 anti-myc antibody ab binding buffer [40 mM HEPES pH 7.8, <br />                                                                                2 mM EDTA, 2 mM EGTA, 60 mM &#206;&#178;-glycerophosphate, 1 mM NaF, <br /> provided evidence suggest circum-                           0.5 mM phenylmethylsulfonyl fluoride (PMSF), 0.2 mM dithiothreitol <br /> stances Rad18 required maintain checkpoint                            (DTT), 0.1 mM orthovanadate, 10% glycerol protease inhibitor <br /> following DNA damage, establishment                            cocktail, described below] presence different concentrations <br /> checkpoint clearly normal. Our finding genetic                        NaCl. The immune complexes captured protein G&#226;&#8364;&#8220;agarose <br />                                                                                beads, washed times, boiled gel loading buffer elec- <br /> interaction checkpoint genes                           trophoresed 10% SDS&#226;&#8364;&#8220;PAGE gels. They blotted PVDF <br /> rad18 consistent observations. We obtained                       membrane filters probed anti-myc antibody. <br /> no evidence, however, physical association the <br /> Rad18&#226;&#8364;&#8220;Spr18 complex checkpoint proteins                             Purification Rad18 complex <br /> tested. The relationship Rad18 checkpoint                            For purification, 30 l exponentially growing myc-Rad18 cells were <br />                                                                                harvested, washed cell pellet resuspended equal volume <br /> machinery likely, therefore, complex indirect.                    lysis buffer A (40 mM HEPES pH 7.8, 0.4 M KCl, 80 mM <br />    Both Rad18 Spr18 essential cell proliferation.                  &#206;&#178;-glycerophosphate, 12 mM NaF, 5 mM EGTA, 5 mM EDTA, 1 mM <br /> Our results Figure 4D Figure 6                         DTT, 1 mM PMSF, 0.1 mM orthovanadate, 10% glycerol, protease <br /> absence functional Spr18, dominant-negative                         inhibitor cocktail consisting 5 &#194;&#181;g/ml trypsin inhibitor, <br /> mutant rad18 constructs overexpressed, cells appear                        pepstatin, leupeptin, aprotinin, 10 &#194;&#181;g/ml bestatin E-64, and <br />                                                                                50 &#194;&#181;g/ml chymotrypsin inhibitor). The cells snap-frozen liquid <br /> to unable replicate DNA,                     nitrogen lysed using blender, following addition copious <br /> continue grow, lay septa develop grossly                           quantities dry ice. The lysate clarified centrifugation and <br /> abnormal morphologies. This indicative role                       subjected ammonium sulfate precipitation. The 15&#226;&#8364;&#8220;45% cut containing <br /> in DNA replication mitotic control. Occasional                          Rad18 protein dissolved lysis buffer dialysed overnight <br />                                                                                buffer. The extract containing ~300 mg protein was <br /> double-strand breaks occur DNA replication                          applied multiple injections Superdex 200, 10/30 FPLC column <br /> these thought repaired recombination. Cells                       (Pharmacia Biotech). Fractions 0.25 ml collected analysed <br /> deficient recombination-repair genes rhp51 rhp54                     Western blotting using anti-myc monoclonal antibody. The high- <br /> grow poorly (Muris et al., 1993, 1996), presumably                             molecular-weight fractions containing Rad18 protein pooled <br /> because difficulty dealing damage arising                    fractionated heparin&#226;&#8364;&#8220;Sepharose Cl-6B (Pharmacia Biotech) <br />                                                                                equilibrated buffer B/0.1 M KCl (25 mM HEPES pH 7.8, 2 mM <br /> during replication. Double mutants rhp51 rhp54                      EGTA, 3 mM MgCl2, 0.1 mM DTT, 0.5 mM PMSF, 12% glycerol). <br /> the DNA damage checkpoint genes inviable (Muris                            After extensive washing, proteins eluted buffer B/0.2, 0.4, <br /> et al., 1996), showing checkpoint-mediated mitotic                        0.6 1 M KCl. The 0.6 M KCl eluate contained Rad18 <br /> delay necessary cell deal kind                      protein incubated overnight 4&#194;&#176;C protein G&#226;&#8364;&#8220;agarose beads <br />                                                                                (Pharmacia Biotech) crosslinked anti-myc antibody buffer B/ <br /> replicative stress. We propose inviability rad18                   0.3 M KCl, containing 0.4 mM EDTA protease inhibitor cocktail. <br /> and spr18 mutants arises involvement linking                     The beads washed buffer, times 10 vol. <br /> replication, repair mitotic control. Further work                     buffer B/0.5 M KCl, 0.1% NP-40, twice buffer B/0.1 M KCl, <br /> be necessary unravel origin complex                            0.01% NP-40. Bound proteins eluted twice 30&#194;&#176;C 1 h each <br /> phenotypes.                                                                    peptide corresponding myc epitope (AEEQKLISEEDL) at <br />                                                                                3 mg/ml buffer B/0.1 M KCl, 0.2 mg/ml insulin 1 &#194;&#181;g/ml aprotinin <br />                                                                                E64. The eluted fractions dialysed twice buffer B containing <br />                                                                                0.1 M KCl 15% glycerol eliminate peptide. The proteins were <br /> Materials methods                                                          separated 10% SDS&#226;&#8364;&#8220;PAGE stained colloidal Coomassie <br />                                                                                Brilliant Blue (Brilliant Blue G, colloidal concentrate; Sigma). Alternat- <br /> Antibodies                                                                     ively analysed immunoblotting antibodies specific for <br /> The 9E10 anti-myc monoclonal antibody used immunoblotting              different DNA damage response proteins. <br /> obtained Santa Cruz. For immuno-affinity chromatography                   In experiments estimate size complex, pooled Superdex <br /> used supernatants CRL-1729 hybridoma cells obtained              200 fractions contained high-molecular-weight peak, derived as <br /> American Type Culture Collection, cultured RPMI 10%                above, subjected gel filtration Superose 6 FPLC <br /> fetal calf serum. The supernatant bound protein G&#226;&#8364;&#8220;agarose,              column (Pharmacia Biotech). In case 0.5 ml fractions collected <br /> extensively washed covalently crosslinked using dimethylpimel-             analysed Western blotting. The column calibrated with <br /> imidate (Harlow Lane, 1988). Polyclonal antibodies Rad18                HMW Gel Filtration Calibration Markers (Pharmacia Biotech). <br /> and Spr18 protein raised sheep peptides <br /> VYSLAKKEEEYKLLWEQSRE RKREREILQNKNQGQSTLNS-                                 ATPase assay <br /> LKDR, respectively. These antibodies identified corresponding              For assaying ATPase activity, elution affinity column the <br /> in vitro translated proteins. They detected proteins appropriate   myc peptide carried 30&#194;&#176;C 20 min only, eluate <br /> size cell extracts, detected corresponding pre-       dialysed buffer B containing 0.05 M KCl. Protein eluates <br /> immune sera. Antibodies damage response proteins            incubated 2 h 30&#194;&#176;C presence 0.1 &#194;&#181;Ci [&#206;&#179;-32P]ATP <br /> generously supplied Drs T.Caspari, J.Murray A.M.Carr.                   (7000 Ci/mmol; ICN) 10 &#194;&#181;l reaction volume containing 25 mM <br />  <br />  <br /> 1700 <br />                                                                                                   Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> Tris&#226;&#8364;&#8220;HCl pH 7.5, 4 mM MgCl2, 20 &#194;&#181;M ATP, 1 mM DTT                         Carr,A., MacNeil,S.A., Hayles,J. Nurse,P. (1989) Molecular cloning <br /> 50 &#194;&#181;g/ml BSA. The reaction stopped adding 5 &#194;&#181;l 0.5 M                 sequence analysis mutant alleles fission yeast cdc2 <br /> EDTA pH 8.0. One microlitre reaction mixture spotted          protein kinase gene: implications cdc2&#226;&#171;&#185; protein structure and <br /> on PEI-cellulose layer chromatography (TLC) plate (Sigma)               function. Mol. Gen. Genet., 218, 41&#226;&#8364;&#8220;49. <br /> and developed 0.75 M KH2PO4. Radiolabelled ATP inorganic              Caspari,T. Carr,A.M. (1999) DNA structure checkpoint pathways in <br /> phosphate quantitated Storm 840 PhosphorImager (Molecular            Schizosaccharomyces pombe. Biochimie, 81, 173&#226;&#8364;&#8220;181. <br /> Dynamics). DNA-dependent ATPase activity assayed presence         Chuang,P., Lieb,J.D. Meyer,B.J. (1996) Sex-specific assembly a <br /> or absence double- single-stranded DNA final concentration          dosage compensation complex nematode X chromosome. <br /> of 15 ng/&#194;&#181;l.                                                                   Science, 274, 1736&#226;&#8364;&#8220;1739. <br />                                                                              Craven,R.A., Griffiths,D.J.F., Sheldrick,K.S., Randall,R.E., Hagan,I.M. <br /> Site-directed mutagenesis                                                      Carr,A.M. (1998) Vectors expression tagged proteins <br /> A rad18 cDNA clone constructed PCR subcloning                  Schizosaccharomyces pombe. Gene, 221, 59&#226;&#8364;&#8220;68. <br /> total S.pombe cDNA. The internal NdeI site destroyed site-directed <br />                                                                              Gregory,R.J., Rich,D.P., Cheng,S.H., Souza,D.W., Paul,S., Manavalan,P., <br /> mutagenesis, removing PCR errors mutagenesis, the <br />                                                                                Anderson,M.P., Welsh,M.J. Smith,A.E. (1991) Maturation and <br /> cDNA cloned NdeI&#226;&#8364;&#8220;SalI fragment pRep41 MH, and <br />                                                                                function cystic fibrosis transmembrane conductance regulator <br /> pRep42 MH epitope-tagging expression vectors (Craven et al., 1998). <br /> Mutations cDNA introduced specific sites using              variants bearing mutations putative nucleotide-binding domains 1 <br /> procedure Kunkel et al. (1987), mutant clones              2. Mol. Cell. Biol., 11, 3886&#226;&#8364;&#8220;3893. <br /> sequenced completely ensure desired mutations      Harlow,E. Lane,D. (1988) Antibodies. Cold Spring Harbor Laboratory <br /> introduced. We regularly subcloning rad18 constructs         Press, Cold Spring Harbor, NY. <br /> unsuccessful ligated DNA transformed variety          Hirano,T. (1998) SMC protein complexes higher-order chromosome <br /> commonly used E.coli strains. We transformed ligation            dynamics. Curr. Opin. Cell Biol., 10, 317&#226;&#8364;&#8220;322. <br /> mixes directly S.pombe successfully obtained desired            Hirano,T. (1999) SMC-mediated chromosome mechanics: conserved <br /> products.                                                                      scheme bacteria vertebrates? Genes Dev., 13, 11&#226;&#8364;&#8220;19. <br />                                                                              Hirano,T., Kobayashi,R. Hirano,M. (1997) Condensins, chromosome <br /> Cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>survival</span> complementation essential                             condensation protein complexes containing XCAP-C, XCAP-E a <br /> phenotype                                                                      Xenopus homolog Drosophila barren protein. Cell, 89, 511&#226;&#8364;&#8220;521. <br /> Mutant constructs introduced wild-type rad18-X cells     Hottinger,H., Pearson,D., Yamao,F., Gamulin,V., Cooley,L., Cooper,T. <br /> in presence thiamine maintain nmt1 promoter plasmid       Soll,D. (1982) Nonsense suppression Schizosaccharomyces <br /> in repressed state. For survival experiments, cells grown      pombe: S. pombe Sup3-e tRNASerUGA gene active S. ce <br /> </body></html>