<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                              Discovery of Antagonist Peptides against Bacterial Helicase- <br />                                               Primase Interaction in B. stearothermophilus by Reverse Yeast <br />                                               Three-Hybrid <br />   <br />                                               Summary <br />                                                    Developing small-molecule antagonists against protein-protein interactions will provide powerful <br />                                                    tools for mechanistic/functional studies and the discovery of new antibacterials. We have <br />                                                    developed a reverse yeast three-hybrid approach that allows high-throughput screening for <br />                                                    antagonist peptides against essential protein-protein interactions. We have applied our <br />                                                    methodology <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the essential bacterial helicase-primase interaction in Bacillus stearothermophilus <br />                                                    and isolated a unique antagonist peptide. This peptide binds to the primase, thus excluding the <br />                                                    helicase and inhibiting an essential interaction in bacterial DNA replication. We provide proof of <br />                                                    principle that our reverse yeast three-hybrid method is a powerful &#226;&#8364;&#339;one-step&#226;&#8364;&#65533; screen tool for direct <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                    high-throughput antagonist peptide selection against any protein-protein interaction detectable by <br />                                                    traditional yeast two-hybrid systems. Such peptides will provide useful &#226;&#8364;&#339;leads&#226;&#8364;&#65533; for the <br />                                                    development of new antibacterials. <br />  <br />  <br />                                               Introduction <br />                                                                 Many functions in biology involve large macromolecular assemblies. The ability of <br />                                                                 individual protein components to interact with each other in a subtle and highly specific <br />                                                                 manner often defines mechanistic and regulatory aspects of the functional requirements of <br />                                                                 such assemblies. Characterization of the molecular details that underpin specific protein- <br />                                                                 protein interactions is a considerable challenge. Traditional structural biology, X-ray <br />                                                                 crystallography, nuclear magnetic resonance (NMR) spectroscopy, and microscopy, <br />                                                                 provides powerful tools for studying such interactions, but technological and experimental <br />                                                                 limits have made it difficult to apply such approaches in a generic, high-throughput manner. <br />                                                                 Instead, biochemical techniques, such as phage-display, yeast two-hybrid (Y2H), and mass <br />                                                                 spectrometry-based technologies, have been developed as more generic methods [1-4]. Such <br />                                                                 methods have been used extensively to identify protein-protein interaction interfaces, <br />                                                                 interacting domains, and small ligands that may mediate such interactions. <br />  <br />  <br />  <br />                                                           The discovery of agonist and/or antagonist peptides that regulate protein-protein interactions <br />                                                           is of great importance, not only in understanding structural, regulatory, and mechanistic <br />                                                           aspects of macromolecular assemblies, but also in designing new drugs employing <br />                                                           combinatorial and bioinformatics approaches [5, 6]. The use of peptide antagonists to <br />                                                           disrupt protein-protein interactions has been demonstrated in a variety of systems. Examples <br />                                                           include peptides that block IL-1a binding to type I IL-1 receptor [7], insulin-like growth <br />                                                           factor 1 binding to its regulatory binding protein, IGFBP-1 [8], the angiogenesis factor <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           VEFGF binding to its cell surface receptor, KDR [9, 10], IgG Fc binding to streptococcal <br />                                                           protein A [11], and HIV-1 gp120 binding to CD4 [12]. By comparison, the use of such <br />                                                           peptides against DNA replication is less well documented. <br />  <br />                                                           To overcome mechanistic and topological problems associated with DNA replication, <br />                                                           evolution has created large, complex multiprotein assemblies, like the primosome and the <br />                                                           replisome. The functions of these assemblies are dependent on delicate collaborations of <br />                                                           their component proteins [13-15]. These collaborations are based upon allosteric protein- <br />                                                           protein and protein-DNA interactions that ensure the effective progression of the entire <br />                                                           assembly through various functional stages in order to complete the effective and faithful <br />                                                           replication of the genetic information. Interference with any of these essential interactions <br />                                                           will inevitably result in stalling of DNA replication and, ultimately, cell death. Small <br />                                                           molecules and/or peptides may act as antagonists to block such interactions, and may <br />                                                           represent a new avenue to explore and develop new antibacterial drugs. Indeed, three very <br />                                                           recent examples have demonstrated that a peptide can bind to geminin and suppress DNA <br />                                                           replication in HCT116 colon cancer cells by disrupting the geminin-Cdt1 interaction [16], <br />                                                           whereas a small molecule can mimic the growth-inhibitory effect of phage polypeptides by <br />                                                           disrupting the interaction within a prototypic pair, ORF104 and DnaI (the putative helicase <br />                                                           loader) in Staphylococcus aureus [17] and also other small molecules can inhibit the <br />                                                           interaction of the herpes simplex virus DNA polymerase and its processivity subunit UL42 <br />                                                           thus inhibiting viral replication [18]. <br />  <br />                                                           In this article, we report the discovery, using a reverse yeast three-hybrid (Y3H) approach, <br />                                                           of an antagonist peptide against the essential interaction between the main replicative <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           hexameric helicase, DnaB, and the primase, DnaG, in the Gram-positive Bacillus <br />                                                           stearothermophilus. This peptide competes with DnaB for binding to DnaG, thus disrupting <br />                                                           an essential bacterial protein-protein interaction during DNA replication and making it a <br />                                                           powerful &#226;&#8364;&#339;lead compound&#226;&#8364;&#65533; for the development of synthetic peptidomimetics as novel <br />                                                           broad-spectrum antibiotics against bacterial DNA replication. This antagonist peptide will <br />                                                           also be useful in studying the molecular details of the helicase-primase interaction. <br />  <br />                                                           Furthermore, although the potential of Y3H in the analysis of small molecule-protein <br />                                                           interactions, as well as its applications in drug discovery, has been emphasized before [19], <br />                                                           our data presented here provide proof of principle that the use of the Y3H technology in a <br />                                                           reverse manner to screen random peptide libraries is a valuable advance over existing <br />                                                           methods. Apart <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> being a direct screen for isolating antagonist peptides against any <br />                                                           protein-protein interaction, it also offers a unique flexibility, allowing the targeting of either <br />                                                           partner of a binary protein complex. It can also be used to investigate, very rapidly, the <br />                                                           effects of mutations on the activity of the antagonist peptide, and can be easily adapted as a <br />                                                           high-throughput screen in a manner similar to other Y2H high-throughput screens to amplify <br />                                                           its potential as an industrial method for direct antagonist peptide screening. It offers new <br />                                                           opportunities in the field of peptide-based drug discovery against any protein-protein <br />                                                           interaction that is detectable by the traditional Y2H system. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 3 <br />  <br />  <br />                                            Results and Discussion <br />                                            Antagonist Peptides against the Helicase-Primase Interaction <br />                                                           The helicase (DnaB)-primase (DnaG) interaction in B. stearothermophilus is stable and the <br />                                                           complex can be reconstituted and detected from the purified proteins [20]. A C-terminal <br />                                                           domain of DnaG, known as P16, can substitute for the full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> DnaG in the complex and <br />                                                           interacts with a linker region that connects the N- and C-terminal domains of DnaB to <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           mediate this interaction both structurally and functionally [21]. However, the precise <br />                                                           structural details of this interaction still remain unresolved. The stoichiometry of the <br />                                                           complex is an unresolved issue, with a DnaB6-DnaG3 model being the more likely [20], but <br />                                                           with DnaB6-DnaG2 and DnaB6-DnaG1 complexes also detectable [21]. <br />  <br />                                                           The interactions of both DnaG and P16 with DnaB can also be detected by traditional Y2H. <br />                                                           We devised a novel use of the Y3H technology in a reverse manner to screen a commercial <br />                                                           MATCHMAKER random peptide library from Clontech in order to discover antagonist <br />                                                           peptides against this essential bacterial protein-protein interaction. Such peptides will <br />                                                           provide us with potential leads for the development of drugs that will target bacterial DNA <br />                                                           replication. <br />  <br />                                                           Using the pBridgeBP16 plasmid that carries both GAL4BD-DnaB and GAL4AD-P16 <br />                                                           fusions, we proceeded to test whether we could detect the DnaB-P16 interaction. GAL4BD- <br />                                                           DnaB is under the control of the constitutive PADH promoter, whereas GAL4AD-P16 was <br />                                                           placed under the control of the PMET25 inducible promoter (Figures 1A&#226;&#8364;&#8220;1D). In the absence <br />                                                           of methionine, the PMET25 promoter is switched on and GAL4AD-P16 is expressed. We <br />                                                           found that the DnaB-P16 interaction was detectable in trp&#226;&#710;&#8217;/met&#226;&#710;&#8217; media using the pBridge- <br />                                                           BP16 plasmid (data not shown), indicating that we could now proceed to use the <br />                                                           pBridgeBP16 plasmid in a reverse Y3H experiment to screen for antagonist peptides using a <br />                                                           random peptide library, as shown in Figure 1E. <br />  <br />                                                           We screened approximately 6000 clones from a commercial MATCHMAKER random <br />                                                           peptide library from Clontech, cloned into the pGADGH plasmid (Figure 1D). Selection for <br />                                                           antagonist peptides was carried out in trp&#226;&#710;&#8217;/met&#226;&#710;&#8217;/leu&#226;&#710;&#8217; media. Potential clones were isolated <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           for antagonist functions from the primary Y3H screen and verified for binding to the <br />                                                           primase DnaG (or its P16 helicase-interacting domain) by a secondary Y2H screen (data not <br />                                                           shown). One clone was isolated from these screens and sequenced. It was found to <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> for <br />                                                           a unique 16-mer peptide that was named lpg1. The sequence of lpg1 did not resemble any <br />                                                           amino acid sequence motifs (or other amino acid sequence parts) of the DnaB sequence and <br />                                                           is presented in Figure 1F. We proceeded to test its antagonist function, not only against the <br />                                                           DnaB-P16 interaction, but also against the full-length DnaB-DnaG interaction applying <br />                                                           again our reverse Y3H approach, and confirmed that, indeed, it inhibited this interaction <br />                                                           (Figures 2A&#226;&#8364;&#8220;2C). Our data show that lpg1 clearly inhibits the DnaB-DnaG and DnaB-P16 <br />                                                           interactions in our reverse Y3H system. <br />  <br />                                                           The configuration of our reverse Y3H system ensured that the GAL4AD-peptide would <br />                                                           interact exclusively with the GAL4AD-P16 for antagonist activity to be apparent in our <br />                                                           screens (Figure 1E). An interaction between GAL4AD-peptide and GAL4BD-DnaB will <br />                                                           simply result in &#206;&#178;-galactosidase expression. Our system also offers the flexibility for <br />                                                           selecting antagonist peptides that will interact exclusively with the other partner in the <br />                                                           complex. For example, a GAL4BD-peptide fusion would result in negative signal only if it <br />                                                           interacts with GAL4BD-DnaB, whereas, in this case, an interaction with GAL4AD-P16 will <br />                                                           result in positive signal. Therefore, the antagonist activity of the isolated lpg1 peptide must <br />                                                           be due to specific interactions with P16 (or DnaG) that exclude DnaB from the complex. To <br />                                                           verify this we confirmed the lpg1-DnaG interaction with the conventional Y2H system, <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                            Page 4 <br />  <br />  <br />                                                           using the pASG and pGADGH-lpg1 plasmids in tpr&#226;&#710;&#8217;/leu&#226;&#710;&#8217; media. As expected, lpg1 was <br />                                                           found to interact with DnaG (Figures 2D and 2E). In contrast, lpg1 did not interact with <br />                                                           DnaB when tested in a similar manner using the pASB instead of the pASG plasmid (data <br />                                                           not shown). <br />  <br />                                            Antagonist Activity In Vitro <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Confirmation of the antagonist activity of lpg1 against the DnaB-DnaG complex formation <br />                                                           was verified by examining its effect on the DnaG-mediated stimulation of the ATPase <br />                                                           activity of DnaB in vitro. Purified B. stearothermophilus DnaB and DnaG proteins form a <br />                                                           stable complex that can be isolated by gel filtration [20, 21]. The ATPase activity of DnaB is <br />                                                           stimulated significantly when in complex with DnaG [20, 21]. However, the DnaB-DnaG <br />                                                           system is particularly complex, because (1) the stoichiometry of the complex is not absolute <br />                                                           and (2) the ATPase kinetic behavior of DnaB is still unresolved. Although the majority of <br />                                                           the complexes exist in a DnaB6-DnaG3 stoichiometry, a significant percentage of complexes <br />                                                           with DnaB6-DnaG2 and DnaB6-DnaG1 stoichiometry also exist [20, 21]. The complexity of <br />                                                           this system is amplified further by the ATPase kinetic behavior of DnaB. It does not obey <br />                                                           first order Michaelis Menten kinetics [20, 21] and the relative kinetic behavior of the <br />                                                           heterogeneous complexes is far from certain. <br />  <br />                                                           Despite the complexity of this system, we argued that if lpg1 reversibly competes with <br />                                                           DnaB for binding to DnaG, the stimulatory effect on the DnaB ATPase activity by DnaG <br />                                                           should be inhibited in the presence of lpg1, because lpg1 will bind to DnaG and prevent the <br />                                                           DnaB-DnaG complex formation. Obviously, we cannot be certain whether all three DnaG <br />                                                           molecules need to be disengaged from the complex or if only one or two DnaG molecules <br />                                                           need to be disengaged before inhibition of the stimulatory effect can be observed. Indeed <br />                                                           DnaB6-DnaG2 and DnaB6-DnaG1 complexes may still exhibit stimulated ATPase activity. <br />                                                           Despite these complications, we would anticipate that removing all three DnaG molecules <br />                                                           from the complex will lead to inhibition of the DnaG-mediated stimulatory effect on DnaB <br />                                                           ATPase activity. <br />  <br />                                                           We tested this hypothesis employing two different ATPase assays: a continuous NADH- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           linked assay that indirectly measures the rate of ATP hydrolysis by monitoring the oxidation <br />                                                           of NADH to NAD+ and an end-point TLC-based assay that directly measures the conversion <br />                                                           of ATP to ADP and inorganic phosphate (Pi). Both assays indicated that lpg1 does, indeed, <br />                                                           inhibit the DnaG-mediated stimulation of the DnaB ATPase activity. In fact, in the TLC- <br />                                                           based assay, lpg1 exhibited good inhibitory activity from 100- to 3.12-fold molar excess <br />                                                           over DnaG and appeared to lose its inhibitory effect if diluted further at 1.5-fold molar <br />                                                           excess or at half the concentration of DnaG (Figure 3A). We should emphasize that the <br />                                                           TLC-based assay provides a qualitative rather than an absolute quantitative assay. In fact in <br />                                                           this assay the competitive inhibitory effect of lpg1 is likely to be magnified. With this assay <br />                                                           we are measuring the percentage of ATP that has been hydrolysed in 10 min. As ATP is <br />                                                           hydrolysed its concentration is reduced with time thus affecting the complex kinetic <br />                                                           behavior of DnaB. Moreover, since we are not measuring a continuous ATPase rate the lpg1 <br />                                                           dose-response in the former experiment will appear better than the robust end-point assay. <br />                                                           Lpg1 actually reduced the stimulatory effect of DnaG on the DnaB ATPase activity by <br />                                                           30%&#226;&#8364;&#8220;50% as determined by the TLC-based end-point ATPase assay. Interestingly, a <br />                                                           truncated version of lpg1 missing two amino acid residues ([des-Ser1-Trp2] lpg1) from its N <br />                                                           terminus exhibited significantly reduced antagonist activity in the same TLC-based assay <br />                                                           compared to the full length lpg1 (Figure 3B). It inhibited marginally the stimulatory effect of <br />                                                           DnaG by ~10% (compare Figures 3A and 3B). <br />  <br />                                                           The antagonist effect of lpg1 was also evident in the continuous NADH-linked assay that <br />                                                           measures the steady-state rate of ATP hydrolysis. This assay measures real-time continuous <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 5 <br />  <br />  <br />                                                           rate because the ATP is continuously regenerated. The ATPase profile of DnaB with this <br />                                                           assay in the presence and absence of DnaG is complicated, with an initial phase that <br />                                                           resembles first order kinetics from 0 to 2.0 mM ATP, followed by an increase between 2.0 <br />                                                           and 3.5 mM, and a sharp inhibition at higher concentrations of ATP [20, 21, 24]. This <br />                                                           complex kinetic behavior is still unresolved, but has been attributed to negative <br />                                                           cooperativity of the monomers around the hexameric ring [20]. However, this suggestion is <br />                                                           still purely speculative and the absolute kinetic mechanism behind this complex behavior is <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           still a mystery. Initially, we examined the effect of lpg1 on the ATPase activity at a fixed 2.0 <br />                                                           mM ATP and found that the ATPase rate of the DnaB-DnaG mixture was reduced from <br />                                                           approximately 15 &#194;&#183; s&#226;&#710;&#8217;1 to less than 4 &#194;&#183; s&#226;&#710;&#8217;1, comparable to the rate for DnaB alone, <br />                                                           approximately 2 &#194;&#183; s&#226;&#710;&#8217;1 (Figure 3C). <br />  <br />                                                           In order to obtain more reliable quantitative information for the &#226;&#8364;&#339;dose response&#226;&#8364;&#65533; of the lpg1 <br />                                                           peptide, we investigated the ATPase profile of the DnaB and DnaB-DnaG mixtures in the <br />                                                           presence and absence of different concentrations of lpg1 at varying concentrations of ATP <br />                                                           up to 2 mM, corresponding to the initial phase of the complicated ATPase profile of DnaB <br />                                                           (Figure 4). The presence of DnaG in the reaction mixture in the absence of lpg1 results in a <br />                                                           marked stimulation of the DnaB ATPase activity (Figure 4A). Significantly, inclusion of a <br />                                                           large excess of the truncated ([des-Ser1-Trp2] lpg1) or mutant ([Ala2]lpg1) peptides resulted <br />                                                           only in marginal differences in the ATPase profile without an obvious suppression of the <br />                                                           DnaG-mediated stimulatory effect (Figure 4B). In contrast, the inclusion of lpg1 in the <br />                                                           reaction mixture resulted in a marked suppression of the DnaG-mediated stimulatory effect, <br />                                                           suggesting that it inhibits the DnaB-DnaG complex formation (Figure 4C). <br />  <br />                                                           Because of the complexity of the actual stoichiometry of the DnaB-DnaG mixtures and the <br />                                                           presence of heterogeneous complexes with either three, two, or one DnaG molecule per <br />                                                           DnaB hexamer in all our ATPase assays, we used an excess of DnaG (55.2 nM) over the <br />                                                           DnaB hexamer (9.2 nM), but still cannot ensure absolutely that all the complexes will be of <br />                                                           the DnaB6-DnaG3 stoichiometry. Also, any excess of free DnaG is likely to sequester lpg1 <br />                                                           from the solution. The complexity of this system makes it impossible to engage in the <br />                                                           simple protein-ligand type of analysis for the determination of a meaningful IC50 dose <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           response for lpg1. <br />  <br />                                                           Despite these problems, we carried out a semiquantitative analysis of the lpg1-mediated <br />                                                           inhibition, and established that, under our experimental conditions, there was no effect <br />                                                           observed at 50 molar excess of lpg1 (2.76 &#206;&#188;M) over the DnaG (55.2 nM) in the reaction <br />                                                           mixture, whereas the inhibitory effect gradually increased as we raised the concentration of <br />                                                           lpg1 to 5.52, 11, and 27.6 &#206;&#188;M, corresponding to 100, 200, and 500 molar excess of lpg1 <br />                                                           over the DnaG in the reaction mixture, respectively (Figure 4C). <br />  <br />                                                           We also verified direct binding of lpg1 to DnaG by isothermal titration calorimetry (ITC), <br />                                                           but the interaction was very strong and an accurate KB value could not be calculated (in the <br />                                                           Supplemental Data available with this article online). With tight binding (KB &gt; 109 M&#226;&#710;&#8217;1), <br />                                                           ITC titrations lose their curvature and KB cannot be measured accurately [25]. <br />  <br />                                            The N Terminus of lpg1 Is Critical for Antagonist Activity <br />                                                           In our search for a minimal sequence within lpg1 that might be critical for its antagonist <br />                                                           activity, we discovered that a truncated lpg1 missing the first two N-terminal serine and <br />                                                           tryptophan residues has lost most of its antagonist activity. In fact, it suppresses the DnaG- <br />                                                           stimulatory effect on the DnaB ATPase activity only marginally in both TLC-based (Figure <br />                                                           3B) and NADH-linked ATPase assays (Figure 4B). Interestingly, when we substituted the <br />                                                           tryptophan residue close to the N terminus for an alanine we found that the mutant <br />                                                           ([Ala2]lpg1) has also lost most of its antagonist activity, with only very weak activity <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                           Page 6 <br />  <br />  <br />                                                           exhibited by this mutant lpg1 in the NADH-linked assay (Figure 4B). In fact, (Ala2)lpg1 <br />                                                           was marginally worse than the truncated (des-Ser1-Trp2) lpg1 (compare the two curves in <br />                                                           Figure 4B). The fact that both (Ala2)lpg1 and (des-Ser1-Trp2)lpg1 peptides exhibit loss of <br />                                                           antagonist activity indicates that the antagonist activity of lpg1 is specific and that the N- <br />                                                           terminal segment of lpg1 is critical. A subsequent search through the protein database (PDB) <br />                                                           revealed that lpg1 has sequence homology to helices found in the mammalian mevalonate <br />                                                           kinase (pdb: 1KVK) and in a viral polymerase (pdb: 1MUK) (Figures 5A and 5B) [26, 27]. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Interestingly, in both cases the homology was confined to the N-terminal half of the lpg1 <br />                                                           peptide (Figure 5C), indicating that this region of the lpg1 peptide has the propensity to <br />                                                           adopt a helical secondary structure. The homologous helices in these two proteins do not <br />                                                           seem to have any functional significance and only in the case of the mammalian mevalonate <br />                                                           kinase is the homologous helix &#206;&#177;8 close to a classic four helix bundle, composed of helices <br />                                                           &#206;&#177;9 and &#206;&#177;10, which constitutes the dimerization interface of this protein [26]. Detailed <br />                                                           structural analysis will be required to establish whether lpg1 does indeed adopt a stable <br />                                                           secondary structure and whether the integrity of this structure is essential for antagonist <br />                                                           activity. <br />  <br />                                            Lpg1 Inhibits the Stimulatory Effect of DnaG on the DnaB Helicase Activity <br />                                                           Having established that lpg1 inhibits the stimulatory effect of DnaG on the ATPase activity <br />                                                           of DnaB, we then examined the effect on the helicase activity of DnaB. Using a well- <br />                                                           established in vitro helicase assay, we showed that lpg1 does indeed inhibit the stimulatory <br />                                                           effect of DnaG on the helicase activity of DnaB (Figure 6). DnaB has a relatively low <br />                                                           helicase activity level that is stimulated significantly by DnaG (Figure 6A). In the presence <br />                                                           of an excess of lpg1, the stimulatory effect of DnaG is inhibited significantly, whereas no <br />                                                           significant effect on the activity of DnaB alone is observed (Figure 6B). These data are <br />                                                           consistent with our ATPase data and, together, provide strong evidence that lpg1 does <br />                                                           indeed act as an antagonist inhibitor of the DnaG-meidated stimulation of the DnaB activity. <br />  <br />                                            Lpg1 Disrupts the DnaB-DnaG Complex Formation In Vitro <br />                                                           The B. stearothermophilus DnaB-DnaG complex, in contrast to that of Escherichia coli, is <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           stable enough to be detected by simply mixing purified DnaB and DnaG proteins together <br />                                                           and then resolving the mixture through a gel filtration column (Figure 7A) [20, 21]. Three <br />                                                           DnaG molecules are believed to interact with a single DnaB hexamer, although minor <br />                                                           species of DnaB6-DnaG2 and DnaB6-DnaG1 complexes may also exist [20, 21]. We <br />                                                           investigated the direct effect of lpg1 on the complex by gel filtration. The complex elutes <br />                                                           from the column much earlier (10.21 ml elution volume) than DnaG (13.8 ml elution <br />                                                           volume). The resolving power of the Superdex S-200 column is insufficient to resolve <br />                                                           clearly the complex (MW, 504 kDa) from the DnaB hexamer (303 kDa). We discovered that <br />                                                           adding an excess of lpg1 (175 &#206;&#188;M) in the DnaB (0.5 &#206;&#188;M hexamer)-DnaG (0.67 &#206;&#188;M <br />                                                           monomer) mixture resulted in the release of a substantial amount of DnaG from the <br />                                                           complex, confirming that lpg1 competes with DnaB for binding to DnaG. The antagonist <br />                                                           activity of lpg1 is apparent in Figure 7B, with an increase of DnaG eluted free from the <br />                                                           complex and also a comparative decrease in the amount of DnaG remaining in the complex <br />                                                           (compare relative amounts of DnaB and DnaG in the complex in Figure 7A, lanes 1&#226;&#8364;&#8220;3 and <br />                                                           Figure 7B, lanes 1&#226;&#8364;&#8220;4). Lpg1 has to compete with the DnaB helicase and displace it from the <br />                                                           preformed DnaB-DnaG complex. Although the affinity of lpg1 for DnaG is in the <br />                                                           nanomolar region (Supplemental Data), apparently DnaB also binds to DnaG with very high <br />                                                           affinity, and not all of the DnaB could be displaced from DnaG by lpg1. Also, as the <br />                                                           proteins in the mixture are loaded onto the top of the column, they begin to separate almost <br />                                                           immediately. Because of its small size, excess lpg1 starts separating from the rest of the <br />                                                           proteins almost immediately. The remaining DnaB-DnaG complex moves down the column <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                            Page 7 <br />  <br />  <br />                                                           quicker, and some of the DnaB will remain bound to DnaG. Thus, only partial, but <br />                                                           nevertheless significant, displacement of DnaB from the complex by lpg1 is observed. <br />  <br />                                                           Our Y3H, Y2H, and in vitro biochemical data for the lpg1 peptide indicate that it <br />                                                           competitively binds to DnaG and disengages it from the DnaB-DnaG complex. Because of <br />                                                           the complexity of the DnaB-DnaG interaction and the unresolved ATPase kinetics of DnaB, <br />                                                           it was not possible to obtain reliable quantitative information for the dose response of lpg1 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           in this system in a manner analogous to a traditional simple peptide-ligand interaction. <br />                                                           However, our combined data provide proof of principle that the novel use of reverse Y3H <br />                                                           technology, in screening random peptide libraries for antagonist peptides against protein- <br />                                                           protein interactions detectable by traditional Y2H, is a viable method. <br />  <br />                                            Significance <br />                                                           Overall, our data provide proof of principle and show that our reverse Y3H approach can <br />                                                           directly select antagonist peptides against a protein-protein interaction that is detectable by <br />                                                           the traditional Y2H system. We targeted the bacterial helicase-primase interaction and <br />                                                           isolated an antagonist peptide that binds to the primase and disrupts the helicase-primase <br />                                                           complex. Despite the complexity of the DnaB-DnaG system, we verified its antagonist <br />                                                           activity using a range of in vitro experiments. We have also shown that both a truncated and <br />                                                           a point mutant version of this peptide were inactive. Our approach can be adapted to a rapid, <br />                                                           high-throughput direct screening assay for the isolation of antagonist peptides against either <br />                                                           partner of a binary protein complex, in a similar manner to a traditional Y2H high- <br />                                                           throughput screen [28, 29], and can be applied to any protein-protein interaction system that <br />                                                           can be detected by the traditional Y2H method. It is a significant advance over existing <br />                                                           methods because: (1) it selects antagonist peptides that bind directly at the protein-protein <br />                                                           interaction interface (direct antagonists) or peptides that act allosterically from a distant <br />                                                           binding site (allosteric antagonists); (2) it allows the specificity of targeting exclusively <br />                                                           either partner of a binary protein complex; (3) it can be adapted as a high-throughput <br />                                                           method; and (4) once the primary antagonist peptide has been isolated, it can also be used to <br />                                                           rapidly investigate a large number of mutant antagonist peptides for improved activity. It <br />                                                           will provide a powerful tool for the development of peptide-based drugs that target essential <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           protein-protein interactions in a variety of systems. Isolated antagonist peptides can be used <br />                                                           as &#226;&#8364;&#339;chemical seeds&#226;&#8364;&#65533; for further development of chemical compounds that mimic their <br />                                                           antagonist action and also to probe directly the structural/functional molecular aspects of <br />                                                           protein-protein interactions. <br />  <br />                                            Experimental Procedures <br />                                            Plasmid Constructions for Y2H and Y3H <br />                                                           Our Y2H and 32H methodologies were based upon the MATCH-MAKER Two-Hybrid <br />                                                           system 2 from Clontech. The dnaB, dnaG genes, as well as the p16 C-terminal fragment of <br />                                                           dnaG coding for a 16 kDa polypeptide (P16) that interacts with DnaB, were all subcloned <br />                                                           from their pET expression vectors [20] into the pACT2 and pAS2-1 vectors, as shown in <br />                                                           Figures 1A and 1B. <br />  <br />                                                           The dnaB gene was cloned into the NcoI-PstI sites of pAS2-1 to give pASB (Figure 1B) that <br />                                                           codes for the GAL4 DNA binding domain (BD)-DnaB fusion protein, under the control of <br />                                                           the PADH1 constitutive promoter whereas the dnaG and p16 genes were subcloned into the <br />                                                           NcoI-XhoI sites of pACT2 to produce the pACTG and pACTP16 plasmids, respectively <br />                                                           (Figure 1A). Both pACTG and pACTP16 <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> for the GAL4 activation domain (AD)-DnaG <br />                                                           and AD-P16 fusion proteins, respectively, both under the control of the PADH1 promoter. <br />                                                           The dnaG gene was also subcloned as an NcoI-XhoI fragment from the pET21d plasmid <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 8 <br />  <br />  <br />                                                           [20] into the NcoI-SalI sites of pAS2-1 (XhoI and SalI sites are compatible), to produce the <br />                                                           pASG plasmid coding for the GAL4BD-DnaG fusion protein under the control of the PADH1 <br />                                                           promoter (Figure 1B). <br />  <br />                                                           The dnaB gene was subcloned as NdeI(blunt)-PstI fragment from pASB into the <br />                                                           EcoRI(blunt)-PstI sites of pBridge to produce the pBridgeB plasmid, coding for a GALBD- <br />                                                           DnaB fusion protein and placing it under the control of the constitutive PADH1 promoter <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           (Figure 1C). The p16 gene was subcloned as HindIII(blunt)-HindIII(blunt) fragment into the <br />                                                           NotI(blunt) site of pBridgeB placing the GAL4AD-P16 fusion protein under the control of <br />                                                           the inducible PMET25 promoter, to give the pBridgeBP16 plasmid (Figure 1C). The correct <br />                                                           orientation was verified by appropriate restriction endonuclease mapping. The dnaG gene <br />                                                           was subcloned as a BglII-BglII fragment from pACTG into the same sites of pBridgeBP16, <br />                                                           thus replacing the p16 gene to form the pBridgeBG plasmid, placing the GAL4AD-DnaG <br />                                                           fusion protein under the control of the PMET25 promoter. Correct orientation was verified by <br />                                                           appropriate restriction endonuclease mapping. <br />  <br />                                                           The random peptide library was purchased from Clontech and was already cloned into the <br />                                                           BamHI-EcoRI sites of pGADGH (Figure 1D), under the control of the constitutive PADH1 <br />                                                           promoter, with stop codons in all three frames immediately after the EcoRI site. The library <br />                                                           contains 106 independent random peptide clones coding for random peptides of 16 amino <br />                                                           acids maximum length fused to GAL4-AD. <br />  <br />                                            Growth Media for Yeast <br />                                                           The AH109 Saccharomyces cerevisiae yeast strain was used in this study. The components <br />                                                           of the yeast complete medium (YPD) and the different synthetic dropout (SD) media, as <br />                                                           well as the various supplements, were purchased from Becton Dickinson and Co. or Sigma <br />                                                           and prepared according to the manufacturer&#226;&#8364;&#8482;s instructions. Nutritionally selective media <br />                                                           were prepared by omitting leucine, tryptophan, and methionine (or combinations of these) <br />                                                           from the growth media. E. coli XL1-Blue bacteria were cultured in Luria-Bertani (LB) <br />                                                           media in the presence of tetracycline. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Yeast Transformations <br />                                                           Yeast cells were made either electrocompetent or chemically-competent for transformations <br />                                                           with DNA. For electrocompetent yeast preparations, cells were cultured first on a YPD plate <br />                                                           at 30&#194;&#176;C for 3 days. Single colonies were used to inoculate liquid YPD cultures that were <br />                                                           incubated at 30&#194;&#176;C overnight with vigorous shaking. Yeast cells were collected by <br />                                                           centrifugation and washed repeatedly in filtered-sterilized, ice-clod 1 M sorbitol. Finally, <br />                                                           yeast cells were resuspended in 200 &#206;&#188;l of 1 M sorbitol and used for transformation by <br />                                                           electroporation with a MicroPulser (Bio-Rad), according to the manufacturer&#226;&#8364;&#8482;s instructions. <br />  <br />                                                           For chemically-competent yeast, single colonies were isolated, as described above, and used <br />                                                           to inoculate liquid YPD cultures that were incubated at 30&#194;&#176;C overnight with vigorous <br />                                                           shaking. The OD600 of the cultures were measured and a volume containing approximately <br />                                                           2.5 &#195;&#8212; 108 cells was collected and brought up to 50 ml with prewarmed YPD and incubated <br />                                                           for a further 4 hr at 30&#194;&#176;C under shaking. Yeast cells were then collected by centrifugation, <br />                                                           washed three times in double distilled (dd) H2O, resuspended in 1 ml 100 mM LiOAc, and <br />                                                           incubated for 10 min at 30&#194;&#176;C. Aliquots of 100 &#206;&#188;l were used, and from each aliquot, cells <br />                                                           were collected by centrifugation. The supernatant was removed and 240 &#206;&#188;l 50 w/v PEG, 36 <br />                                                           &#206;&#188;l of 1 M LiOAc, 50 &#206;&#188;l of salmon sperm DNA, 2 &#206;&#188;l of plamid DNA, and 32 &#206;&#188;l dd H2O <br />                                                           were added, in that order. The cells were vortexed vigorously and incubated for 30 min at <br />                                                           30&#194;&#176;C. They were then heat shocked at 42&#194;&#176;C for 30 min before being collected by <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 9 <br />  <br />  <br />                                                           centrifugation. The supernatant was removed and the cells were resuspended in 1 ml dd H2O <br />                                                           and then loaded onto plates with appropriate SD media. <br />  <br />                                            Y2H and Y3H Screening <br />                                                           Y2H and Y3H screening were carried out by either the agarose overlay [22] or filter-lift <br />                                                           methods according to the yeast protocols handbook from Clontech. The pBridgeBP16 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           plasmid was cotransformed with a random peptide library cloned into pGADGH plasmid. <br />                                                           Following transformation, double selective plates were incubated at 30&#194;&#176;C for 36 hr. A <br />                                                           typical transformation yielded approximately 2000 colonies. An agarose overlay assay was <br />                                                           performed, as described previously [22]. This allowed the rapid identification of potential <br />                                                           positive white colonies in which the protein-protein interaction appeared to be blocked, thus <br />                                                           preventing &#206;&#178;-galactosidase activity. Such colonies typically would fail to develop blue <br />                                                           coloration overnight. These colonies were then patched out on to double selective plates, <br />                                                           cultured for 36 hr at 30&#194;&#176;C, and selected again for absent &#206;&#178;-galactosidase activity. Colonies <br />                                                           that exhibited no &#206;&#178;-galactosidase activity were then cultured and plasmid DNA coding for <br />                                                           the interfering peptide was prepared. As a further control, a yeast strain containing the <br />                                                           pASG plasmid was transformed with the plasmid pGADGH coding for the potentially <br />                                                           interfering peptide in order to confirm the interaction of the peptide with its target protein, <br />                                                           DnaG. <br />  <br />                                            Peptide Synthesis <br />                                                           Peptide synthesis was carried out using the standard fluoren-9-ylmethoxycarbonyl (Fmoc) <br />                                                           solid-phase approach [23] on a 4-[2&#226;&#8364;&#178;, 4&#226;&#8364;&#178;- <br />                                                           dimethoxyphenyl(amino)methyl]phenoxyacetamido methyl Novagel (Rink Amide Novagel, <br />                                                           100&#226;&#8364;&#8220;200 mesh, 1% crosslinking, 0.61 mmol &#194;&#183; g&#226;&#710;&#8217;1) resin. The peptide assembly was achieved <br />                                                           in a stepwise manner, in which amino acid residues were incorporated by activation using <br />                                                           N-[(dimethylamino)-1H-1,2,3-triazolo[4,5-b]pyridine-1 ylmethylene]-N-methylmethan- <br />                                                           aminium hexafluorophosphate N-oxide:7-aza-1-hydroxybenzotriazole:N,N- <br />                                                           diisopropylethylamine (HATU:HOAt:DIEA, 1.0:0.5:2.0) in N,N-dimethylformamide <br />                                                           (DMF). Exceptionally, the Cys residue was installed by activation using N,N- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           diisopropylcarbodiimide:HOAt (1:1) in DMF. A 4-fold molar excess of the activated Fmoc- <br />                                                           amino acids was used throughout the synthesis and double-couplings (6 hr followed by 18 hr <br />                                                           reaction times) were typically required. A solution of 20% (v/v) piperidine in DMF was <br />                                                           applied for sequential Fmoc-deprotection. <br />  <br />                                                           Following solid-phase peptide assembly, the resin-supported product was collected and <br />                                                           dried. This material was then treated with the mixture trifluoroacetic acid (TFA): <br />                                                           H2O:triisopropylsilane: ethyl methyl sulfide (90:5:3:2 v/v, 10 ml) for 4 hr at ambient <br />                                                           temperature. The reaction mixture was filtered and the filtrate was evaporated to dryness in <br />                                                           vacuo to yield, following trituration with diethyl ether, the peptide amide as beige solid, <br />                                                           which was dissolved in water (5 ml) and lyophilized. The desired peptide amide was then <br />                                                           purified by reversed-phase HPLC on a Kromasil C8 column (150 &#195;&#8212; 10 mm) at 4.5 ml &#194;&#183; <br />                                                           min&#226;&#710;&#8217;1. The eluents were: A, 0.06% TFA in water, and B, 0.06% TFA in 90% aqueous <br />                                                           acetonitrile. The elution profile was 20%&#226;&#8364;&#8220;40% B over a 12 min period, in which the <br />                                                           required peptide amide eluted at 9.7 min. <br />  <br />                                                           Electrospray-high resolution mass spectrometry (ES-HRMS) analysis (Micromass LCT) of <br />                                                           the purified peptide showed the presence of the expected molecular ion (calculated for C78 <br />                                                           H122N23O19S [MH+], 1718.0103; found, 1718.7845). Lpg1 was dissolved as a 10 mg/ml <br />                                                           stock solution in neat DMSO and then brought down to 1 mg/ml in 10% v/v DMSO by the <br />                                                           dropwise addition of dH2O. Aliquots were frozen in liquid N2, stored at &#226;&#710;&#8217;80&#194;&#176;C, and used as <br />                                                           required. <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 10 <br />  <br />  <br />                                                           Both the truncated peptide, (des-Ser1,Trp2)lpg1, and the mutant peptide, (Ala2)lpg1, were <br />                                                           also synthesized and purified using the approach outlined above. When analyzed by ES- <br />                                                           HRMS, the purified 14-mer peptide showed the presence of the required molecular ion <br />                                                           (calculated MH+, 1443.7158; found, 1443.9214 for [des-Ser1-Trp2]lpg1; and calculated <br />                                                           MH+, 1602.8784; found 1602.8606 for [Ala2]lpg1). Both peptides were readily soluble as 1 <br />                                                           mg/ml stock solutions in dH2O. Aliquots were frozen in liquid N2, stored at &#226;&#710;&#8217;80&#194;&#176;C, and <br />                                                           used as required. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Protein Purifications <br />                                                           DnaB and DnaG proteins were purified and quantitated as described previously [21]. <br />  <br />                                            ATPase Assays and Analytical Gel Filtration <br />                                                           The DNA-independent ATPase activity of DnaB was assayed using either TLC-based or <br />                                                           NADH-linked assays. <br />  <br />                                                           The end-point TLC-based ATP hydrolysis assay was carried out in 20 mM Tris (pH 7.5), 50 <br />                                                           mM NaCl, 10 mM MgCl2, 2.5 mM EDTA, 5 mM DTT, 2 mM ATP, 5 pM of 32P-labeled <br />                                                           ATP, 0.6 &#206;&#188;M DnaB (referring to hexamers), and in the presence or absence of 1.6 &#206;&#188;M <br />                                                           DnaG, at 37&#194;&#176;C for 10 min. Reactions were also carried out in the presence or absence of <br />                                                           lpg1 at various concentrations as indicated in the figure legends. All reactions were <br />                                                           terminated by the addition of 50 mM EDTA and run through TLC plates in 1 M formamide <br />                                                           and 0.5 M LiCl for 1 hr. TLC plates were visualized and analyzed using a phosphorimager. <br />                                                           All reactions were carried out in triplicate with average values plotted. <br />  <br />                                                           The continuous NADH-linked ATPase assay was carried out as described previously for the <br />                                                           DnaB-DnaG complex [21] in the presence or absence of varying concentrations of lpg1 over <br />                                                           DnaG. All reactions were carried out in triplicate and the average values were plotted. All <br />                                                           ATPase rates were defined as molecules of ATP hydrolyzed per second per hexamer of <br />                                                           DnaB. We should emphasize here that it is unlikely that all monomers in the hexameric <br />                                                           complex will be hydrolyzing ATP at the same rate. Also, because the stock lpg1 peptide was <br />                                                           dissolved in 10% v/v DMSO, we investigated the effect of increasing concentrations of <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           DMSO on the ATPase activity in both assays and found that, up to 50% v/v, there was no <br />                                                           detectable effect on the ATPase activity (data not shown). DnaG does not exhibit an ATPase <br />                                                           activity ([20] and Figure 3B) and synthesizes primers only in the presence of an appropriate <br />                                                           ssDNA substrate (for a review, see article by Frick and Richardson [30]). <br />  <br />                                                           Analytical gel filtration experiments were carried out using a Superdex S-200 gel filtration <br />                                                           column (Amersham Pharmacia Biotech) as described previously [20, 21], using 0.5 &#206;&#188;M <br />                                                           DnaB hexamer, 0.67 &#206;&#188;M DnaG, and 175 &#206;&#188;M lpg1. In brief, DnaB and DnaG were mixed <br />                                                           and left for 10 min on ice to enable complex formation, at which point lpg1 was added, and <br />                                                           the mixture of proteins was left for a further 5 min before being loaded onto the sizing <br />                                                           column. <br />  <br />                                            ITC <br />                                                           ITC experiments were carried out with a VP-ITC Microcalorimeter and data were analyzed <br />                                                           with the associated Origin software. Typically, the appropriate cell was filled with 1.4 ml <br />                                                           degassed solution of DnaG (16.4 &#206;&#188;M) in 50 mM Tris (pH 7.4), 100 mM NaCl, 10% v/v <br />                                                           DMSO. Samples (5 &#206;&#188;l) from a 300 &#206;&#188;l degassed solution of lpg1 (0.45 mM in 10% v/v <br />                                                           DMSO) peptide in the same buffer were injected into the DnaG solution at 240 s intervals <br />                                                           and data were collected after each injection. A comparative reference control experiment <br />                                                           with 10% v/v DMSO was also carried out, and the reference data were subtracted from the <br />                                                           experimental data. <br />  <br />  <br />  <br />                                            Helicase Assays <br />                                                           Helicase assays were carried out using a well-established in vitro assay system [20, 21, 24, <br />                                                           31]. A radiolabeled oligonucleotide (5&#226;&#8364;&#178;-GTT <br />                                                           ATTGCATGAAAGCCCGGCTGACTCTAGAGGATCCCCGGGTACGTTATTGCATGA <br />                                                           AAGCCCGGCTG-3&#226;&#8364;&#178;) was annealed onto ssM13mp18 DNA to produce a 3&#226;&#8364;&#178;-5&#226;&#8364;&#178;-tailed DNA <br />                                                           substrate. One molecule of DNA substrate is defined as one molecule of ssM13mp18 DNA <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           with one molecule of oligonucleotide annealed to it. DnaB (25 nM) was incubated in the <br />                                                           helicase reaction mixture (20 mM Tris [pH 7.5], 20 mM MgCl2, 10% v/v glycerol, 5 mM <br />                                                           DTT, and 0.5 nM DNA substrate) at 37&#194;&#176;C for 5 min in the presence or absence of 150 nM <br />                                                           DnaG and/or 12.5 &#206;&#188;M lpg1, as appropriate. Reactions were initiated with the addition of 3 <br />                                                           mM ATP and terminated at 5 min intervals by the addition of helicase stop-buffer (0.4% w/v <br />                                                           SDS, 40 mM EDTA, 8% v/v glycerol, 0.1% w/v bromophenol blue). Displaced <br />                                                           oligonucleotide was separated from annealed oligonucleotide through a 10% nondenaturing <br />                                                           polyacrylamide gel at constant voltage (100 V). Gels were dried and quantitative analysis <br />                                                           was performed using a phosporimager and associated software (Molecular Imager FX, Bio- <br />                                                           Rad). Helicase activity was defined as a percentage of radiolabeled oligonucleotide <br />                                                           displaced from M13mp18 DNA. <br />  <br />  <br />  <br />  <br />                                                           Figure 1. <br />                                                           Construction of Plasmids for Y2H and Y3H Experiments <br />                                                           (A) Construction of pACT-based plasmids. The insertion sites of dnaG and p16 genes are <br />                                                           indicated. <br />                                                           (B) Construction of pAS-based plasmids. The insertion sites for the dnaG and dnaG genes <br />                                                           are indicated. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           (C) Construction of pBridge-based plasmids. The insertion sites for the dnaB and p16 genes <br />                                                           are indicated. <br />                                                           (D) The pGADGH plasmid carrying the random peptide library at the BamHI-EcoRI, as <br />                                                           indicated. In all cases the direction of the inserted genes is indicated by arrows. <br />                                                           (E) Schematic representation of the Y2H system with both proteins (P1, P2) on the same <br />                                                           pBridge plasmid (left panel). The plasmid is selected in trp&#226;&#710;&#8217; media. P1 is expressed <br />                                                           constitutively, whereas P2 is expressed only in met&#226;&#710;&#8217; media. A schematic representation of <br />                                                           the reverse Y3H system is presented in the right panel. The pGADGH plasmid (carrying the <br />                                                           peptides, P3) is selected in leu&#226;&#710;&#8217; media and P3 is expressed constitutively. A P2-P3 <br />                                                           interaction will give a negative signal, whereas a P1-P3 interaction will give a positive <br />                                                           signal. Therefore, if P3 interacts with P2 and disrupts the P1-P2 interaction, it can be <br />                                                           detected as a negative signal in the reverse Y3H system. <br />                                                           (F) The sequence of the lpg1 peptide. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 15 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 2. <br />                                                           Lpg1 Antagonist Activity in Yeast Hybrid Systems <br />                                                           Verification of the antagonist activity of lpg1 against the DnaB-DnaG (A) and DnaB-P16 <br />                                                           (B) interactions in the Y3H system. In both panels, positive control experiments for the <br />                                                           DnaB-DnaG and DnaB-P16 interactions are indicated by the blue coloration, as indicated. <br />                                                           Additional controls for the traditional positive interaction between the SV40 large T antigen <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           and p53, and the negative control between SV40 large T antigen and DnaB, are shown in <br />                                                           (C). Verification of the lpg1-DnaG interaction by the filter-lift method in a traditional Y2H <br />                                                           system is shown in (D) (yeast patches) and (E) (filter lifts). Lpg1 exhibits clear DnaG <br />                                                           binding activity. The positive control indicates the DnaB-DnaG interaction and the negative <br />                                                           control indicates no interaction between DnaB and SV40 large T antigen. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                         Page 16 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 3. <br />                                                           Thin-Layer Chromatography-Based ATPase Assays <br />                                                           (A) A TLC-based end-point ATPase assay showing the inhibitory effect of lpg1 at different <br />                                                           concentrations: 100-, 50-, 25-, 12.5-, 6.12-, 3.12-, 1.5-, and 0.5-fold molar excess of lpg1 <br />                                                           over DnaG as indicated in bars 3&#226;&#8364;&#8220;10, respectively. Bars 1 and 2 indicate the ATPase <br />                                                           activities of the DnaB-DnaG mixture and the DnaB protein alone, respectively. Triplicate <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           experiments were carried out, but only one representative TLC plate is shown. The standard <br />                                                           deviation is indicated by error bars. <br />                                                           (B) The results from a similar assay, but using 100-fold molar excess of the N-terminally <br />                                                           truncated version (des-Ser1Trp2)lpg1 of lpg1 missing the first two N-terminal residues (bar <br />                                                           3). Bar 1 and 2 indicate the ATPase activities of the DnaB-DnaG mixture and the DnaB <br />                                                           protein alone, respectively. Bars 4 and 5 indicate controls of DnaG and ATP with no protein, <br />                                                           respectively. Experiments were carried out in triplicate and error bars indicate the standard <br />                                                           deviation. <br />                                                           (C) A synopsis of the steady-state ATPase data collected using the continuous NADH- <br />                                                           linked ATPase assay. Bar 1 indicates the ATPase rate of the DnaB (9.2 nM hexamer)-DnaG <br />                                                           (55.2 nM monomer) mixture, bar 2 the rate of the DnaB (9.2 nM hexamer) alone, and bar 3 <br />                                                           the rate of the DnaB (9.2 nM hexamer)-DnaG (55.2 nM monomer) in the presence of 5.52 <br />                                                           &#206;&#188;M of lpg1. All reactions were carried out at 2.0 mM (ATP). <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                        Page 17 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 4. <br />                                                           NADH-Linked ATPase Assays <br />                                                           (A) A comparison of the ATPase profile of the DnaB hexamer (9.2 nM) in the presence <br />                                                           (closed triangles) and absence (open squares) of DnaG (55.2 nM) from 0 to 2.0 mM (ATP). <br />                                                           (B) The ATPase profile of the DnaB hexamer (9.2 nM) in the presence of DnaG (55.2 nM) <br />                                                           and 27.6 &#206;&#188;M of either the truncated lpg1 (open squares) or the mutant lpg1 (closed <br />                                                           triangles). <br />                                                           (C) The ATPase profile of the DnaB hexamer (9.2 nM) in the presence of DnaG (55.2 nM) <br />                                                           and varying concentrations of lpg1, 2.76 &#206;&#188;M (closed circles), 5.52 &#206;&#188;M (open triangles), 11 <br />                                                           &#206;&#188;M (closed squares), and 27.6 &#206;&#188;M (open squares) corresponding to 50, 100, 200, and 500 <br />                                                           molar excess of lpg1 over DnaG, respectively. All the graphs have the same scale axes for <br />                                                           direct comparison, and error bars indicate the standard deviation from duplicate <br />                                                           experiments. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                         Page 18 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 5. <br />                                                           Sequence Similarities from the Structural Database <br />                                                           The N-terminal half of the lpg1 peptide is homologous to helices found in the mammalian <br />                                                           mevalonate kinase (A) and in the reovirus RNA polymerase lambda3 (B). In both cases the <br />                                                           relevant helices are indicated in black. The actual sequences and their homology to the lpg1 <br />                                                           sequence are shown in (C). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 19 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 6. <br />                                                           Effects on the Helicase Activity <br />                                                           DnaG stimulates the helicase activity of DnaB (A), whereas lpg1 inhibits the stimulatory <br />                                                           effect of DnaG (B). Lanes 1&#226;&#8364;&#8220;6 represent helicase reaction time points 5, 10, 15, 20, 25, and <br />                                                           30 min, whereas lanes a and b represent annealed and boiled controls, respectively. The <br />                                                           actual gels labeled appropriately are shown on the left and the quantitative data are shown as <br />                                                           graphs on the right. In (A), data represent helicase reactions of DnaB in the presence <br />                                                           (circles) and absence (squares) of DnaG. In (B), data represent helicase reactions of DnaB + <br />                                                           lpg1 in the presence (circles) and absence (squares) of DnaG. Annealed and displaced <br />                                                           oligonucleotide is indicated with labeled arrows for clarity. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                       Page 20 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 7. <br />                                                           Lpg1 Competes with the Helicase for Binding to the Primase <br />                                                           Analytical gel filtration experiments of the DnaB (0.5 &#206;&#188;M hexamers)-DnaG (0.67 &#206;&#188;M <br />                                                           monomers) mixtures in the absence (A) and presence (B) of lpg1 (175 &#206;&#188;M). Samples from <br />                                                           the peak fractions were analyzed by SDS-PAGE. The numbers of the lanes in the <br />                                                           polyacrylamide gels correspond to the numbers of the peak fractions, as indicated. Lane M <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           shows molecular weight markers, as indicated. lpg1 causes the dissociation of DnaG (67 <br />                                                           kDa) from DnaB (50.5 kDa) in a competitive manner. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available in PMC 2011 February 08. <br />   <br /> </body></html>