<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  Proc. Nadl. Acad. Sci. USA <br />   Vol. 82, pp. 3785-3789, June 1985 <br />   Genetics <br />  <br />  <br />   RAS2 Saccharomyces cerevisiae required gluconeogenic <br />   growth proper response <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation <br />        (yeast/oncogenes/sporulaion/storage carbohydrates) <br />   KELLY TATCHELL, Lucy C. ROBINSON, AND MICHAEL BREITENBACH* <br />   Department Biology, G5, University Pennsylvania, Philadelphia, PA 19104 <br />   Communicated Edward M. Scolnick, February 15, 1985 <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>         Saccharomyces cerevisiae contains genes                       substitute normal RAS genes yeast a <br />  remarkable homology members ras oncogene                             modified RAS] gene transform NIH-3T3 cells (11). <br />  family. These genes, RAS] RAS2, constitute                                  Given functional biochemical similarities between <br />  essential gene family spores disruptions ofboth genes                    mammalian yeast RAS proteins, likely knowl- <br />  fail grow. We report strains containing RAS2                           edge RAS function yeast direct implica- <br />  disruptions distinct phenotypes. First, fail                     tions forRAS function mammals. We report yeast <br />  grow efficiently nonfermentable carbon sources. Second,                          strains lacking RAS2 function defect gluconeogenic <br />  hyperaccumulate storage carbohydrates glycogen                         growth, accumulate excessive levels storage carbohy- <br />  trehalose. Third, diploid cells homozygous RAS2                             drates, sporulate prematurely. We identified muta- <br />  disruptions sporulate rich media. Extragenic suppressors                         tions additional genes suppress the <br />  isolated suppress gluconeogenic defect.                          gluconeogenic defect. Two mutations allow strains <br />  These suppressors fall complementation                          RAS grow. <br />  groups, mutations bypass normal require- <br />  ment RAS cell viability, allowing cells containing                                MATERIALS AND METHODS <br />  RAS gene grow. The <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span> RAS2 mutant and <br />  extragenic suppressors unplicateRAS function                         Yeast Strains. TX2-530.1.1C (a leu2 ura3 trpl lysi lys2 his3 <br />  normal response <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation.                                             ras2-530) (4) backcrossed MCY638 (a his4-539 lys2- <br />                                                                                      801 ura3-52). MCY638 MCY317 (a ade2-101 his4-539), <br />  Saccharomyces cerevisiae contains genes pro-                            derived strain S288C, kindly provided Marian <br />  tein products share homology mammalian ras                                 Carlson. Meiotic segregants EG81, second backcross <br />                                                                                      MCY638, used quantitative measurements and <br />  oncogene products. YP2, located chromosome VI (1),                               revertant analysis. Although backcrosses performed <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br />  shares 40% homology NH2-terminal 160 amino                             linkage specific phenotypes ras2, mende- <br />  acids mammalian ras, RASI RAS2 (2),                                lian segregation patterns gluconeogenic defect and <br />  located chromosome XV (3, 4) XIV (3), respectively                           glycogen hyperaccumulation clearly observed origi- <br />  &gt;60% homologous mammalian ras                                  nal transformed diploid TX2-530. Specific strains used in- <br />  region (5, 6). RAS] RAS2 constitute essential gene                           clude EG81-40A (a ura3 his4 lys2), EG81-40B (a ras2-530 <br />  family. Cells containing disruptions RAS gene                       leu2 ura3 his4 lys2 lysi), EG81-40C (a ras2-530 leu2 ura3 his4 <br />  viable, haploid spores containing disruptions                       lys2 his3 lysl?), EG81-40D (a ura3 his4 lys2 his3 lysl?), <br /> RAS] RAS2 fail grow (3, 4). YP2 fails complement                           EG81-22B (a ras2-530 his4-539 lys2-801 ura3), 112-699 [a <br />  double disruption encodes separate                              leu2 ura3 can1-100 ade2-1 his3 ras2-699 (His+)]. <br /> function.                                                                              The gluconeogenic revertants described de- <br />    RAS] RAS2 <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins 309 322 amino                             rived EG81-22B. Revertants crossed EG72-16C <br /> acids, respectively, immunoprecipitated                             (a ras2-530 leu2 trpl lys2) subsequent crosses between <br /> antibodies directed mammalian p21 (7, 8). The larger                        strains containing sra mutations meiotic <br /> size proteins relative 21-kDa mammalian p21                         progeny crosses. EG73-14D (a rasl-545 leu2 ura3 <br /> is result 7-amino acid extension NH2-terminal                       lysi his3) EG87-2B (a rasl-545 ura3 his4 lys2) used <br /> end yeast proteins 120- 130-amino acid                             test sra mutations bypass RAS. XC0262485 [a <br /> insertion COOH-terminal portion proteins. The                         rasl485 (Trp&apos;) stell&apos;3 leu2 his4-580 his3 trpl] been <br /> COOH-terminal insertions homologous                           described (7). <br />                                                                                        Media Genetic Techniques. Unless stated, <br /> and location analogous position                        yeast cells grown 1% yeast extract 2% peptone <br /> mammalian proteins greatest divergence (amino acids                            (YEP medium) containing 2% designated carbon <br /> 165-185).                                                                           source; glucose (YEPD medium), ethanol (YEPE <br />    Like mammalian counterparts, yeast RAS pro-                            medium), glycerol (YEPG medium), acetate (YEPA me- <br /> teins bind GTP (7, 8) and, case RAS1,                       dium). Sporulation medium described (13). <br /> an intrinsic GTPase activity similar p21 (9).                      Genetic techniques described Mortimer and <br /> This activity decreased absent RAS1 proteins contain-                      Hawthorne (14). Cell number determinations doubling <br /> ing missense mutations amino acid positions analogous                         time calculations cells grown liquid culture made <br /> transforming mutations p21 (9). Functional similarities                     using Klett colorimeter. <br /> exist p21 yeast RAS. The normal mammalian                                  Glycogen Trehalose Determinations. Qualitative glyco- <br /> Harvey ras gene (10) viral Harvey ras (11)                               gen determinations routinely inverting plate <br /> The publication costs article defrayed page charge          *Permanent address: Institut fur Allgemeine Biochemie und Ludwig <br /> payment. This article marked &quot;advertisement&quot;                Boltzmann Forschungsstelle fuir Biochemie, Universitdt Wien, <br /> in accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                     Wahringerstrasse 38, A-1090, Vienna, Austria. <br />                                                                              3785 <br />  3786     Genetics: Tatchell et al.                                                   Proc. Natl. Acad. Sci. USA 82 (1985) <br />  <br /> of cells iodine vapors. Cells accumulate glycogen            defect, glycogen phenotype segregated LEU2 <br /> stain brown, don&apos;t remain yellow. Quan-             meiosis 52 tetrads remained associated with <br /> titative trehalose glycogen determinations               ras2-530 backcrosses wild-type strain. <br /> according Lillie Pringle (15).                                  Strain 112-699 (ras2-699) stained darker its <br />    DNA Isolation Southern Analysis. Genomic yeast DNA              congeneic RAS2 parent. Confirmation iodine stain <br /> was isolated according Winston et al. (16). Gel electro-            reflected levels ofglycogen accomplished measur- <br /> phoresis, DNA blotting, hybridization analysis de-             ing glycogen levels (15). EG81-40B -40C, which <br /> scribed (4).                                                 contained ras2-530, significantly higher levels glyco- <br />                                                                        gen wild-type spore clones 40A 40D (Table 1). <br />                           RESULTS                                      Quantitative measurements storage carbohydrate, <br />                                                                        trehalose, showed trend. <br />    RAS2 Is Required Efficient Gluconeogenic Growth.                   We observed diploids homozygous ras2 sporulated <br /> RAS] RAS2 disrupted diploid strains                rich medium. Most strains Saccharomyces cerevisiae <br /> inserting genes selectable auxotrophic markers, URA3               undergo meiotic cell cycle (referred sporula- <br /> and LEU2, respectively, coding sequences               tion) medium deficient reduced nitrogen containing <br /> cloned genes replacing normal genes                   nonfermentable carbon source acetate. A series of <br /> disrupted copies (3, 4). No phenotypic change observed             strains, homozygous heterozygous wild-type <br /> in haploid meiotic products contained disruptions              mutant RAS2 gene, tested sporulation different <br /> either RAS] RAS2 alone, observed               media. Only ras2 homozygous diploids (EG81-40B/EG81- <br /> yeast cells containing disruptions RAS2 (allele ras2-530)           40C EG81-44A/EG81-44B) sporulated YEPD or <br /> have impaired ability grow nonfermentable carbon              YEPE medium (5-25%). Heterozygous homozygous <br /> sources ethanol, glycerol, acetate, pyruvate.              wild-type strains (nine closely related strains) showed no <br /> Five independent ras2-530 mutants showed growth                   sporulation media 96 hr. Because strains <br /> defect. The defect segregated 2:2 meiosis LEU2            isogeneic, data sufficient prove <br /> (ras2-530) 52 four-spored asci tested remained              genetic linkage derepressed sporulation pheno- <br /> associated ras2 backcrosses wild-type          type ras2. However, Toda et al. (17) recently <br /> strain (MCY638). Strain 112-699 contains RAS2 gene                   observed similar phenotype independently construct- <br /> disrupted HIS3 gene defect            ed ras2 mutations. <br /> ras2-530 strains (11). The congeneic RAS2 parent                   Increased Gene Dosage RASI Can Suppress Growth <br /> defect. No phenotypic changes associated            Defect ras2-530. Strain XC0262-485 contains multiple <br /> disruptions RAS], yeast ras homolog.                      copies RAS] gene integrated RAS] locus (allele <br />    Growth ethanol glycerol completely elimi-             RAS1485) increased levels RAS] mRNA and <br /> nated strains contain ras2-530. Although failed        protein (7). Strains containing ras2-530 RAS1485 <br /> grow nonfermentable carbon sources 37&#194;&#176;C, significant             growth defect ethanol (YEPE medium). Tetrad <br /> growth observed lower temperatures. Doubling times              analysis strains heterozygous ras2-530 and <br /> for representative RAS2 ras2-530 strains presented          RAS1485 revealed spore clones containing ras2-530 <br /> Table 1. Sister spores tetrad EG81-40, contained            RAS1485 (Leu+,Trp+) grew YEPE medium (total of <br /> ras2-530 (EG81-40B -40C), longer doubling times                 35), spore clones containing ras2-530 RAS] <br /> than wild-type spores (EG81-40A -40D) YEPE                   (total 27) failed grow. The glycogen phenotype of <br /> medium (ethanol) 21&#194;&#176;C 30&#194;&#176;C failed grow 37&#194;&#176;C            ras2-530 suppressed RASS1485 allele. All <br /> (doubling time &gt;24 hr). No difference growth rate            Leu+, Trp+ spore clones stained extent with <br /> the RAS2 ras2-530 strains observed YEPD                      iodine wild-type (Leu-) clones (data shown). <br /> medium (glucose) 21&#194;&#176;C 30&#194;&#176;C, difference was <br /> observed 37&#194;&#176;C (Table 1). Transfer EG81-40B                      The Gluconeogenic Defect Reverts High Frequency. <br /> glucose ethanol medium resulted population largely           Numerous growing clones arose strains containing <br /> unbudded cells, indicated cell cycle arrest G1.                ras2-530 cultured YEPE YEPG medium. In order <br /> EG81-40B failed grow glycerol, acetate, pyru-             characterize events responsible reversions, <br /> vate did grow fermentable sugars sucrose                 individual colonies strain EG81-22B (a ras2-530) were <br /> raffinose. The growth defect recessive diploids               grown stationary phase YEPD medium plated on <br /> heterozygous ras2-530 grew ethanol glycerol.             YEPG medium. After 7 days 37&#194;&#176;C, growing colonies were <br />    ras2-530 Causes Derepression Sporulation Carbo-               observed frequency colony 104-105 cells <br /> hydrate Accumulation. Yeast strains containing ras2-530                 plated. When revertants crossed EG73-14D (a <br /> consistently stained darker iodine wild-type                  ras2-530), resulting diploids utilize ethanol <br />  strains, suggested ras2-530 responsible         glycerol, not. Thus, mutations <br /> increased accumulation glycogen. Like gluconeogenic              responsible reversions dominant or <br /> Table 1. Doubling times carbohydrate levels ras2-530 strains <br />                                                   Doubling time, min <br />                                   210C                   300C                  370C <br />                Relevant     YEPD       YEPG        YEPD       YEPE      YEPD        YEPE         Glycogen,               Trehalose, <br />   Strain      genotype medium medium medium medium medium medium pg/mg dry weight Ag/mg dry weight <br /> EG81-40A        RAS2          240        408         90        198        117         480              9                       1 <br /> EG81-40B        ras2          215       1269         90        600        235      &gt;2400*             93                     87 <br /> EG81-40C        ras2          240       1231         90        855        213      &gt;2400             106                     99 <br /> EG81-40D        RAS2          240        426         96        198        153         450             18                      2 <br />   Glycogen trehalose measurements early stationary-phase cultures. <br /> *After initial period growth, ras2 strains stopped growing 370C. The growth <span id='am-165' about='xsp:pattern' typeof='owl:Thing'>pattern</span> glycerol-, acetate-, or <br />  pyruvate-containing medium. <br />              Genetics: Tatchell et al.                                                        Proc. Natl. Acad. Sci. USA 82 (1985)           3787 <br />            Table 2. Genetic analysis sra mutants <br />                                                                            Ethanol utilizing/ethanol nonutilizingt <br />                                                       Relevant                     Four-spored asci                             % ethanol <br />             Cross         Parental strains           genotype*         4:0       3:1     2:2        1:3     0:4     Total*       positive <br />             RX14        R12b/EG72-16C              sral-1/+              0         0      11         1       0       33/42          44 <br />            RX1          Rla/EG72-16C               sral-2/+              0        0        4         0       0       14/18          44 <br />            RX12         R2b/EG72-16C              sra2/+                 0        0        4         1       0       17/18          49 <br />            RX8          R6e/EG72-16C              sra3/+                 0        0        4         0       1      12/19           39 <br />            RX26        RX14-7B/RX14-4D            sral-1/sral-1                                Sporulation negative <br />            RX15        RX1-1B/RXI-6C              sral-2/sral-2         16        0        0         0       0      77/0           100 <br />            RX16        RX14-7B/RX1-lB             sral-J/sral-2          2        0        0         0       0      38/0           100 <br />            RX19        RX12-1B/RX12-3D            sra2/sra2              5        0        0         0       0      29/0           100 <br />            RX20        RX8-9B/RX8-9C              sra3/sra3             10        0        0        0        0      69/0           100 <br />            RX45        RX31-12B/RX12-4A           sral-I/sra2            0        3        2        0        0      38/23           62 <br />            RX18        RX1-2D/RX12-1B             sral-2/sra2            2        7        2        0       0       54/18           75 <br />            RX58        RX14-1C/RX8-9C             sral-1/sra3            1        3        1        0       0       44/16           73 <br />            RX21        RX1-lB/RX8-9B              sral-2/sra3            2      14         2        0       0       58/22           72 <br />            RX43        RX12-4A/RX8-7B             sra2/sra3              1        0        1        0       0       23/10           70 <br />            *A11 strains contain ras2-530 <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span>. <br />            tGrowth ethanol scored 48 hr 30&#194;&#176;C YEPE medium. <br />            tA high percentage spores failed germinate (&gt;40%) crosses (RX16 RX43, example) therefore <br />             data tetrad analysis significant. For reason, data germinating spores included. <br />  <br />   recessive. In preliminary characterization chose                  The pleiotropic phenotypes observed strains con- <br />   look recessive revertants.                                   tain sral independent ras2-530. We <br />      The mutations responsible reversion                     obtained sral-l strains wild type RAS2 and <br />   gluconeogenic deficiency segregated mendelian genes,                observed phenotypes original revertant <br />   shown 2:2 segregation ethanol growth tetrads                (Rl2b)-namely, accumulation brown pigment, lack of <br />   strains heterozygous suppressors (Table 2). We               glycogen, loss viability nitrogen-deficient media. <br />   placed mutations responsible suppression                 The inability R12b grow ethanol 37&#194;&#176;C also <br />   (designated sra &quot;suppressor ras&quot;) complemen-               shared sral-J strains wild type RAS2. The failure of <br />   tation groups crossing different suppressors observing             R12b grow ethanol 37&#194;&#176;C failure sral-J <br />   ability meiotic progeny utilize ethanol glycerol.            suppress gluconeogenic defect ras2-530 was <br />   The summary crosses presented Table 2. The                 phenotype sral-l itself. <br />   suppressors revertants R12b Rla same <br />   complementation group, suppressors                                                     .0           -0 <br />   revertants, R2b R6e, fell different                                                      L-    CNJ        N     CD <br />  complementation groups. Crosses suppressors                                                        EE    cr    cr     cr <br />  R2b, R6e, R12b Rla gave rise -25% <br />  YEPE- progeny (original SRA ras2 phenotype). These data                                 YEPD 300E <br />  presented Table 2 crosses RX45, RX18, RX58, <br />  RX21, RX43. We designated suppressors R12b,                              YEPD/ 12 stain <br />  Rla, R2b, R6e sral-J, sral-2, sra2, sra3, respec- <br />  tively. <br />     The revertants phenotypes addition                               YEPE 30&#194;&#176;0 <br />  reversion gluconeogenic defect. Some are <br />  documented Fig. 1, parent congeneic <br />  revertants spotted YEPD medium replica-                              YEPE 370 <br />  plated different media. As determined iodine stain- <br />  ing, R12b Rla accumulated glycogen parent. <br />  R6e, however, accumulated high higher levels                            YEPG 300 <br /> parent. Although revertants grew better parent on <br /> YEPE YEPG medium 30&#194;&#176;C, R12b failed grow                                     YEPG 370 <br /> YEPE medium 37&#194;&#176;C. This growth defect observed <br /> on YEPG (Fig. 1) YEPA medium (data shown). <br /> Another phenotype specific R12b Rla decreased                             -NH;/YEPD <br /> viability grown medium deficient (NH4)2SO4. This <br /> was observed strains replica-plated                       FIG. 1. Pleiotropic phenotyes sra mutations. Strain EG81- <br /> (NH4)2SO4-deficient medium, incubated 48 hr                  22B (parent) revertants ras2 gluconeogenic defect <br /> plated rich medium (YEPD medium). As seen                    spotted YEPD plate, allowed grow 24 hr 30C, <br /> the panel Fig. 1, R12b Rla, contain             replica-plated media listed left, photographed <br /> alleles sral, showed decreased plating efficiency com-               48 hr. For second row, glycogen content estimated by <br /> pared strains. R12b meiotic progeny              staining cells iodine vapor. In row, colonies were <br /> containing sral-1 accumulated brown pigment incu-                  incubated minimal medium lacking (NH4)2SO4 48 hr, replica- <br /> bated extended periods time. Spectroscopic examina-                plated YEPD medium, photographed 24 hr 30&#194;&#176;C. <br />                                                                           Most cells R12b Rla nonviable incubation on <br /> tion strain (data shown) revealed absorption                (NH4)2SO4-deficient media, seen decreased plating <br /> band 580-590 nm, indicative porphyrin compound (18).              efficiency. <br />  3788     Genetics: Tatchell et al.                                                Proc. Natl. Acad. Sci. USA 82 (1985) <br />  <br />   We observed strains homozygous sral-J failed          shown recently bypass requirement RAS <br /> sporulate. However, sral-2, appeared allelic         (17). We know bcyl allelic sral. gici, mutation <br /> sral-J shared phenotypes, sporulated           originally characterized glycogen deficiency (12), like <br /> when homozygous heteroallelic sral-1. This             sral, recessive sporulation defect accumulates <br /> was consistent generally weaker phenotype               brown porphyrin pigment (26) does appear be <br /> sral-2. Strains containing sral-2 accumulated glyco-           allelic sral (unpublished data). <br /> gen, did turn brown storage sral-1,                 Yeast cells normally grow unrestrained unless deprived of <br /> showed slightly increased viability sral-J NH4)2SO4-        essential nutrients. Even situations cell cycle <br /> deficient media (Fig. 1). We observed sporulation        inhibited, response haploid cells mating <br /> defect suppressors.                                    hormones, cell growth continues (27). One first <br />   Two Extragenic Revertants Bypass Requirement ras.         responses deprivation nutrients accumulation of <br /> Haploid spores containing disruptions rasi ras2         storage carbohydrates, glycogen trehalose. In the <br /> either fail germinate arrest early cell cycle prior    case glucose limitation, accumulation begins before <br /> to bud emergence (3, 4). Since sra mutations suppressed         consumption of glucose growth <br /> the gluconeogenic defect ras2, asked            medium, cells growing logarithmically (15). <br /> bypass requirement RAS altogether. This         When carbon source finally depleted cells arrest in <br /> crossing strains containing ras2-530 (marked LEU2)           G1 cell cycle, giving rise population largely <br /> sra strains containing rasl-545 disruption (marked         unbudded cells. Deprivation reduced nitrogen, phosphate, <br /> by URA3). Haploid meiotic progeny containing ras2          sulfate elicits similar response (15). In diploid cells, <br /> rasi (Leu&apos;, Ura&apos;) viable spore              induction meiotic cell cycle, sporulation, also <br /> clones sra mutation eliminates bypasses require-      governed <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> levels. With strains, nitrogen and <br /> ment RAS genes. This case          glucose eliminated sporulation occur. <br /> both sral-l sra3. rasi, ras2 double mutants (Leu+, Ura+)        Optimal sporulation occurs acetate buffer. Some strains <br /> were frequently sral-l sra3                induced sporulate limiting phosphate sulfate <br /> present, indicates mutations eliminated        instead nitrogen (28). <br /> requirement normally essential RAS genes. All Ura+,            The increased glycogen levels derepressed sporulation <br /> Leu+ spore clones sral-l crosses phenotypes            ras2-530 strains taken evidence RAS <br /> indicative sral-l (glycogen deficiency, pigment accumula-        function nutritional assessment or, specifically, <br /> tion, decreased viability (NH4)2SO4-deficient media).        lack RAS2 results premature starvation <br /> They cold-sensitive phenotype, growing              response. If decreased RAS expression leads premature <br /> above 30&#194;&#176;C growing poorly 25&#194;&#176;C. All putative        starvation response, overexpression decrease elimi- <br /> rasi ras2 sra3 strains grew poorly temperatures. Ura+,       nate response. Strains carrying RAS2 missense muta- <br /> Leu+ spores observed sral-2 sra2            tion, resulting substitution valine-19 glycine-19, <br /> was present.                                                        dominant phenotype consistent inability to <br />    We ruled duplication RASI explanation        respond starvation conditions; strains (RAS2 vall9) fail <br /> for apparent bypass phenotype sral-J sra3             sporulate (3), fail accumulate glycogen trehalose, <br /> Southern hybridization analysis. Genomic DNA isolated           decreased viability (17). We confirmed these <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> sral sra3 strains, digested proper restric-       observations strain containing RAS2alal6vall9 (unpub- <br /> tion endonucleases distinguish wild-type mutant         lished data). If mutations result decrease the <br /> RAS alleles, electrophoresed agarose, blotted          intrinsic GTPase activity RAS2 protein affect- <br /> nitrocellulose, hybridized nick-translated RAS]         ing GTP binding, RAS1 gene product (9), the <br /> RAS2 DNA (4). In case did observe additional bands            net result analogous overexpression RAS. The <br /> of ras hybridization indicative additional RAS gene. The       similarities sral-1 RAS2Vall9 phenotypes, <br /> rasi, ras2 sra mutants contained restriction pattern       observation sral-1 bypasses the <br /> of disruption alleles (data shown).                         requirement RAS, suggest sral gene products <br />                                                                      acting downstream RAS pathway. <br />                         DISCUSSION                                     While phenotypes described implicate RAS2 with <br />                                                                     function response <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> levels, mechanism of <br /> A number previously characterized mutations yeast              signaling clear. However, cAMP metabolism appears <br /> have phenotypes similar ones discussed here. The spd         involved. Strains mutations CYR], are <br /> mutations described Dawes Calvert (19, 20) sporulate          deficient adenylate cyclase activity, hypersporulate, <br /> efficiently rich medium fail grow effectively           bcyl mutants asporogenous (22). There also <br /> most nonfermentable carbon sources. Unlike ras2, grow           evidence enzymes responsible synthesis and <br /> well fail sporulate medium containing ethanol       degradation glycogen trehalose regulated by <br /> sole carbon source. Two temperature-sensitive cell division          phosphorylation (25, 29-33). cAMP-dependent protein <br /> cycle mutants, cdc25 cdc35, derepressed             kinase directly implicated stimulation trehalase <br /> sporulation (21) cyrl (22), maps close           phosphorylase (25, 33). Taken together, results <br /> cdc35 results loss adenylate cyclase activity (23).    suggest testable hypothesis RAS functions of <br />    spd mutations, like ras2, revert glycerol utilization     mechanism regulating protein kinases. <br /> high frequency, extragenic suppressors. The sup-              We understand strains lacking RAS2 have <br /> pressors share similarities sral-l (24). They        defect gluconeogenic growth. However, defect has <br /> have recessive asporogenous phenotype, decreased vi-               allowed isolate suppressors bypass normal <br /> ability, accumulate brown pigment incubated           requirement RAS. Judging similarities phe- <br /> extended periods. Unlike sral, however, spd suppressors          notypes sral-l dominant RAS2 mutation, is <br /> are dominant suppression respiratory defect. Two         likely sral-l function downstream same <br /> additional mutations similar phenotypes sral-1 . bcyl,      pathway. The recent observations mammalian (10) <br /> a mutation leading loss cAMP dependence               viral Harvey ras (11) substitute normal <br /> normal cAMP-dependent protein kinase (23), shares                 RAS genes yeast modified RAS] gene can <br /> number phenotypes sral-J. bcyl recessive               transform NIH-3T3 cells (11) support hypothesis that <br /> sporulation defect (22), fails accumulate trehalose (25),     RAS equivalent functions organisms. If the <br />             Genetics: Tatchefl et aL                                                         P1oc. NatL. Acad. Sci. USA 82 (1985)        3789 <br />  <br />  case, sra bypass mutants described                           eds. Rose, A. H. &amp; Harrison, J. S. (Academic, New York), <br />  functional equivalents mammals. Molecular cloning                         Vol, 1, pp. 385-460. <br />  suppressors allow isolation oftheir mammalian              15.    Lillie, S. H. &amp; Pringle, J. R. (1980) J. Bacteriol. 143, <br />  counterparts lead understanding RAS                           1384-1394. <br />  function.                                                                16.    Winston, F., Chumley, F. &amp; Fink, G. R. (1983) Methods <br />                                                                                  Enzymol. 101, 211-228. <br />                                                                           17.    Toda, T., Uno, I., Ishikawa, T., Powers, S., Kataoka, T., <br />     We thank Deborah DeFeo-Jones, John Cannon, Richard                       Broek, D., Cameron, S., Broach, J., Matsumoto, K. &amp; Wigler, <br />  Schultz helpful discussions. We thank Ian Dawes bringing           M. (1985) Cell 40, 27-36. <br />  attention similarities betweefl phenotypes         18.    Pretlo*, T. P. &amp; Sherman, F. (1967) Biochim. Biophys. Acta <br />  mutations RAS SPD. This work supported grants                148, 629-644. <br />  General Medical Council (GM29473) National Cancer                19.    Dawes, I. W. (1975) Nature (London) 255, 707-708. <br />  Institute (CA37702) National Institutes Health. L.C.R.     20.    Dawes, I. W. &amp; Calvert, G. R. (1984) J. Gen. Microbiol. 130, <br />  supported National Institutes Health predoctoral training grant           605-613. <br />  M.B. supported Max Kade Fellowship.                 21. Shilo, V., Shimchen, G. &amp; Shilo, B., (1978) Exp. Cell Res. 112, <br />                                                                                  141-248. <br />    1. Gallwitz, D., Donath, C. &amp; Sander, C. (1983) Nature (London)        22. Matsumoto, K., Uno, I. &amp; Ishikawa, T. (1983) Cell 32, <br />        306, 704-707.                                                           417-423. <br />    2. DeFeo-Jones, D., Scolnick, E. M., Koller, R. &amp; Dahr, R.             23. Matsumoto, K., Uno, I., Oshima, Y. &amp; Ishikawa, T. (1982) <br />        (1983) Nature (London) 306, 707-709.                                    Proc. Natd. Acad. Sci. USA 79, 2355-2359. <br />    3. Kataoka, T., Powers, S., McGill, C., Fasano, O., Strathern,         24. Calvert, G. R. &amp; Dawes, I. W. (1984) J. Gen. Microbiol. 130, <br />        J., Broach, J. &amp; Wigler, M. (1984) Cell 37, 437-445.                    615-624. <br />   4. Tatchell, K., Chaleff, D. T., DeFeo-Jones, D. &amp; Scolnick,            25. Uno, I., Matsumoto, K., Adachi, K. &amp; Ishikawa, T. (1983) J. <br />        E. M. (1984) Nature (London) 309, 523-527.                              Biol. Chem. 258, 10867-10872. <br />   5. Dahr, R., Nieto, A., Koller, R., DeFeo-Jones, D. &amp; Scolnick,         26. Padrao, G. R. B., Malamud, D. R., Panek, A. D. &amp; Mattoon, <br />        E. M. (1984) Nucleic Acids Res. 12, 3611-3618.                          J. R. (1982) Mol. Gen. Genet. 185, 255-261. <br />   6. Powers, S., Kataoka, T., Fasano, O., Goldfarb, M., Broach, J,        27. Thorner, 3. (1981) The Molecular Biology Yeast <br />        &amp; Wigler, M. (1984) Cell 36, 607-612.                                   Saccharomyces: Life Cycle Inheritance, eds. Strathern, <br />   7. Temeles, G. L., DeFeo-Jones, D., Tatchell, K., Ellinger,                 J. N., Jones, E. W. &amp; Broach, J. R. (Cold Spring Harbor <br />        M. S. &amp; Scolnick, E. M. (1984) Mol. Cell. Biol. 4, 2298-2305.          Laboratory, Cold Spring Harbor, NY), pp. 143-180. <br />   8. Tamanoi, F., Walsh, M., Kataoka, T. &amp; Wigler, M. (1984)             28. Freese, E. B., Chu, M. I. &amp; Freese, E. (1982) J. Bacteriol. <br />       Proc. Natd. Acad. Sci. USA X1, 6924-6928.                               149, 840-851. <br />   9. Temeles, G. L., GibbS, J. B., D&apos;Alonzo, J. S., Sigal, I. S. &amp;       29. Fosset, M., Muir, L. W., Nielsen, L. D. &amp; Fischer, E. H. <br />       Scolnick, E. M. (1985) Nature (London) 313, 700-703.                    (1971) Biochemistry 10, 4105-4113. <br /> 10. Kataoka, T., Powers, S., Cameron, S., Fasano, O., Goldfarb,          30. Huang, K. P. &amp; Cabib, E. (1974) J. Biol. Chem. 249, <br />       M., Broach, J. &amp; Wigler, M. (1985) Cell 140, 19-26. <br /> 11. DeFeo-Jones, D., Tatchell, K., Robinson, L. C., Sigal, I.,                3851-3857. <br />       Vass, W., Lowy, D. R. &amp; Scolnick, E. M. (1985) Science 228,        31. van Solingen, P. &amp; van der Plaat, J. 13. (1975) Biochem. <br />       179-184.                                                                Biophys. Res. Commun. 62, 553-560. <br /> 12. Rothman-Denes, L. B. &amp; Cabib, E. (1970) Proc. Ndtl. Acad.            32. Ortiz, C. H., Maia, J. C. C., Tenan, M. M., Braz-Padrao, <br />       Sci. USA 66, 967-974.           <br /> </body></html>