<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>STEM CELLS AND DEVELOPMENT                                                    ORIGINAL RESEARCH REPORTS <br /> Volume 21, Number 8, 2012 <br />   Mary Ann Liebert, Inc. <br /> DOI: 10.1089/scd.2011.0285 <br />  <br />  <br />  <br />  <br />                     Characterization Danio rerio Nanog <br />                 Functional Comparison Xenopus Vents <br />  <br />            Maximilian Schuff,1,* Doreen Siegel,1,* Melanie Philipp,2 Karin Bundschu,2 Nicole Heymann,1 <br />                                       Cornelia Donow,1 Walter Kno&#194;&#168;chel1 <br />  <br />  <br />  <br />  <br /> Nanog homeodomain transcription factor associated acquisition pluripotency. Genome analyses <br /> of lower higher vertebrates revealed existence Nanog restricted gnathostomata absent <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> agnatha invertebrates. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> elucidate function Nanog nonmammalia, identified Danio rerio <br /> ortholog Nanog characterized role gain loss function experiments. We Nanog be <br /> crucial <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>survival</span> early zebrafish embryos, depletion Nanog led gastrulation defects with <br /> subsequent lethality. Mouse Nanog overexpression rescue defects. Vice versa, zebrafish Nanog was <br /> found promote proliferation inhibit differentiation mouse embryonic stem cells absence of <br /> leukemia inhibitory factor. These findings indicate functional conservation Nanog teleost fishes to <br /> mammals. However, Nanog lost genome anurans Xenopus laevis Xenopus tropicalis. Phy- <br /> logenetic analysis revealed deletion probably occurred common anuran ancestor chromo- <br /> somal translocations. The closest homologs Nanog Xenopus Vent proteins. We, therefore, <br /> investigated Xvent genes substitute Nanog function Xenopus. Although some <br /> similarities phenotypes overexpression regulation marker genes, Xvent1/2 and <br /> Nanog substitute other. Depletion Nanog zebrafish rescued ectopic expression of <br /> Xvent, Xvent depletion Xenopus overcome ectopic expression zebrafish Nanog. <br />  <br />  <br />  <br />  <br /> Introduction                                                          mammals nonmammals [6,8]. The pluripotency- <br />                                                                       associated factor, Dppa5, completely absent lower ver- <br />  <br /> T   undifferentiated cell state vertebrate embryo- <br />     genesis maintenance pluripotency embry- <br /> onic stem (ES) cells characterized expression of <br />                                                                       tebrates [9]. Dppa2 Dppa4 exist lower vertebrates, such <br />                                                                       Anolis (reptile) Xenopus (amphibian), form one <br />                                                                       homolog [10]. For handful factors, asso- <br /> factors, Oct4, Nanog, Dax1, Tert, Dppa1-5. Ex-             ciated maintenance stem cell character, is, <br /> pression Oct4 appears prerequisite mainte-         Zfp206, Rex1, Dax1, Klf4, Dppa1, Dppa3, Nanog, <br /> nance undifferentiated state self-renewal               data obtained lower species insufficient. Since <br /> mammalian ES cells [1&#226;&#8364;&#8220;3], despite increasing knowl-            stemness-like pluripotent state described <br /> edge target gene regulations molecular interactions         nonmammalian organisms [11,12], clear how <br /> in different species, current data allow       status maintained species. Do factors exist <br /> general view signaling maintaining pluripotency.                lower vertebrates, embedded same <br /> Comparing genes involved pluripotency evo-             signaling cascades higher vertebrates? <br /> lutionary point view, situation                 In current study, focus homeobox tran- <br /> complicated. In placental mammals (eutheria), Oct4           scription factor Nanog, master regulator plur- <br /> encoding gene (known Pou5f1) exists, monotremes            ipotency [13&#226;&#8364;&#8220;15]. Nanog identified screening mouse <br /> (protheria) marsupials (metatheria) additionally posses         cDNA libraries ES cell specific transcripts [16,17] by <br /> Pou2 gene. In amphibian Xenopus, Pou2 gene           expression cloning ES cell cDNA library factor <br /> been triplicated (Oct60, Oct25, Oct91) [4,5], Pou5f1      maintaining cytokine-independent self-renewal murine ES <br /> gene lacking [6]. In vertebrates, chicken,      cells [even absence leukemia inhibitory factor (LIF)] <br /> zebrafish, axolotl, Pou2 Pou5f1 gene exists [7].   [13]. Although Nanog unable re-induce stem cell char- <br /> In addition, Oct4 regulation cascades modified            acter Oct4 [18], Nanog re-expressed fully <br />  <br />  <br />   Institutes 1Biochemistry 2Biochemistry Molecular Biology, University Ulm, Ulm, Germany. <br />   *These authors contributed equally work. <br />  <br />                                                                  1225 <br />  1226                                                                                                            SCHUFF ET AL. <br />  <br /> reprogrammed induced pluripotent stem cells promotes           ctrlMO (5&#194;&#162;-ATGCAAACAATACATACACCTCGTG-3&#194;&#162;) was <br /> transfer pluripotency cell fusions [19,20]. This indi-    used. In vitro test morpholino specificity NanogMO <br /> cates Nanog central player core transcriptional   previously described [10]. <br /> regulatory circuitry pluripotency [14,21,22] gate- <br /> way pluripotent ground state [15]. Nanog - / - mice         Cloning genes vitro transcription <br /> embryos lethal, ES cells form extraembryonic endo- <br /> derm expense epiblast. Since Nanog expressed       A Nanog clone D. rerio obtained imaGenes <br /> in Oct4 - / - mouse embryos, suggested 2         (accession no. BC162318). The open reading frame cloned <br /> transcription factors act parallel pathways [13]. Although      pCS2 expression vector. For rescue experiments, silent <br /> Oct4 substitute Nanog self-renewal murine ES          mutations introduced MO-binding site D.r. <br /> cells absence LIF [23], 2 factors share high     Nanog (mut-Nanog: 5&#194;&#162;-cgGaattcATGTCaCGgCAcGAgGG-3&#194;&#162;) <br /> overlap target genes [14,21]. In addition, mutual protein     polymerase chain reaction (PCR) using mutated primers. <br /> binding Oct4 Nanog found, recruiting               The translation start underlined; silent mutations shown <br /> HDAC1/2 NuRD, Sin3A, Pml, Mta1/2 form re-               lowercase. For GST fusion proteins, Nanog open reading <br /> pression complexes called NODE (Nanog Oct4 associ-             frames (murine D. rerio) cloned pGEX-4T-1 <br /> ated deacetylase) [24].                                            vector. Nanog mutants (HD + : 164&#226;&#8364;&#8220;179 aa; DN: 194&#226;&#8364;&#8220;384 aa; <br />    A vertebrate homolog Nanog nonmammalian or-               DC: 1&#226;&#8364;&#8220;261 aa) Nanog DHD, lacking aa 195&#226;&#8364;&#8220;261, were <br /> ganisms reported chicken [25,26]. It pos-        cloned pCS2 vector. For RNA injection, plasmids were <br /> tulated Nanog amniote-specific gene absent       transcribed SP6. Digoxigenin labeled antisense RNA for <br /> from anamniotes invertebrates [25]. However, Nanog             situ hybridization obtained T7 polymerase. <br /> homologs recently reported teleost fish <br /> (medaka) [27,28] urodele amphibian axolotl [29].       Protein expression GST-pulldown assay <br /> It suggested mechanisms governing plur-                  GST-fusion proteins expressed E. coli BL21 (DE3) <br /> ipotency conserved urodele amphibians             Plus (Stratagene) purified batch Glutathione- <br /> mammals [29]. However, does consider situation        Sepharose  4B (Amersham Biosciences) native con- <br /> in fish, Nanog present, anuran amphibian        ditions according manufacturer&#226;&#8364;&#8482;s protocol. Purified <br /> Xenopus tropicalis, Nanog homolog missing [30].         proteins dialyzed overnight 4 C (50 mM Tris-HCl, <br /> Therefore, interesting know parts          pH 8.0, 1 mM EDTA, 1 mM DTT, 20% glycerol). 35S- <br /> Nanog function conserved different species       methionine labeled proteins translated TNT T7/ <br /> and genes account Nanog function                  SP6 Coupled Reticulocyte Lysate System (Promega). Pull- <br /> Xenopus.                                                           analyses previously described [34]. <br />    Here, characterize Nanog gene Danio rerio and <br /> demonstrate gain loss function (LOF) experiments <br /> that Nanog function essential zebrafish embryogenesis.      Electrophoretic mobility shift assay <br /> We Nanog knockdown zebrafish rescued              Electrophoretic mobility shift assays (EMSAs) per- <br /> by murine Nanog, closest sequence homologs          formed double-stranded oligonucleotides murine <br /> Nanog proteins Xenopus, homeobox transcription              Tcf3 promoter (5&#194;&#162;-AAACCTGTTAATGGGAGCGCAT-3&#194;&#162; <br /> factors Xvent1 Xvent2 [31,32], fail substitute          [35]). Oligonucleotides 5&#194;&#162;-end-labeled g-ATP using <br /> Nanog function D. rerio. Additionally, demonstrate      T4 polynucleotide kinase subsequently annealed. Gel- <br /> zebrafish Nanog contrast Xvent2 promotes proliferation       purified probes incubated Xvent2 Nanog pro- <br /> and inhibits differentiation mouse ES cells ab-     teins (murine Danio) 25 min ice 30 mL binding <br /> sence LIF. These findings indicate Nanog function-      buffer [36] containing unspecific competitor poly(dIC). <br /> ally conserved bony fishes (teleosts) mammals.             For specific competition, unlabeled probes used. Sam- <br />                                                                    ples separated 8% nondenaturing polyacrylamide <br /> Materials Methods                                              gels analyzed autoradiography (BAS 1500&#226;&#8364;&#8220;Fuji). <br /> Database analyses <br />                                                                    ES cell culture <br />    Sequence analyses, EST, genomic database search and <br /> sequence alignments using bioinformatic                  In general, mouse ES cells (E14.1; 129/Ola-derived) were <br /> programs: www.ncbi.nlm.nih.gov/blast; www.ensembl.org/             cultured (DMEM + GlutaMax, 10% FCS, 1.5 g/L <br /> index.html; www.ebi.ac.uk/clustalw/; http://genome.wustl           Na2CO3, 10 mg/L LIF (SIGMA), 0.1% penicillin/strep- <br /> .edu/tools/blast.                                                  tomycin) irradiated neomycin resistant mouse embryonic <br />                                                                    feeder cells (MEFs). M.m. Nanog, D.r. Nanog, Xvent2 <br /> Antisense morpholino oligonucleotides                              sequences integrated eukaryotic expression <br />                                                                    vector pFIEN, ES cells electroporated (Biorad Gene <br />    D.r. NanogMO (5&#194;&#162;-ATCTTCCAGTCCGCCATTTCGCCGT-                     Pulser, 240V) recombinant constructs pFIEN vector <br /> 3&#194;&#162;) (GeneTools) directed translation start site     control (40 mg). To obtain stably transfected clones, ES <br /> (reverse complementary underlined). Xvent1 Xvent2           cells selected G418 (250 mg/mL; Gibco) 8 days <br /> morpholino oligonucleotides (MOs) previously reported         sorted GFP expression (BD, FACS Aria). <br /> [33]. For control injections Xenopus, control MO (ctrlMO)        For differentiation assays, ES cells trypsinized, sep- <br /> derived human b-globin gene (5&#194;&#162;-CCTCTTACCT                arated MEFs 2 &#194;&#183; 103 cells/40 mL differentiation <br /> CAGTTACAATTTATA-3&#194;&#162;) used. For controls zebrafish,           media [IMDM (Gibco), 1.5 g/L Na2CO3, 10% FCS, 0.1% <br />  ZEBRAFISH NANOG AND XENOPUS VENT GENES                                                                                       1227 <br />  <br /> penicillin/streptomycin, 20 mL/L monothioglycerol, 1.1 &#194;&#183;          (Branchiostoma), tunicates [urochordata (ie, Ciona)]. In <br /> nonessential amino acids] incubated hanging drops         addition, Nanog-like genes exist non-vertebrates, for <br /> to form embryoid bodies. For controls, 10 mg/L LIF            example, insects (D. melanogaster) nematodes (Cae- <br /> added differentiation media inhibit differentiation.    norhabditis elegans). Thus, Nanog appeared chordates with <br /> At day 2 incubation, embryoid bodies transferred       evolution jaw (gnathostomata). <br /> gelatinized 24-well plates differentiation media. Dif-          Interestingly, Nanog lacking genome X. tropi- <br /> ferentiation embryoid bodies monitored day 3         calis [30]. Since failed detect corresponding ESTs <br /> morphology day 4 AP staining (Alkaline Phos-            Nanog Xenopus laevis, suppose Nanog absent <br /> phatase Substrate Kit III; Vector). Images taken      entire genus Xenopus com- <br /> Keyence microscope.                                               plete pipidian subfamily Xenopodinae. To understand how <br />                                                                   Nanog deleted Xenopus genome, performed <br /> BrdU cell tracking assays                                     extensive synteny analysis Nanog loci (Fig. 1B). The <br />                                                                   human Nanog located chromosome 12:7. The genomic <br />    Stably transfected ES cells dissociated trypsin      surroundings Nanog identical human to <br /> and re-seeded density 5 &#194;&#183; 105 cells. Cells incu-     chicken. Nevertheless, pro- eutheria some <br /> bated 10 mM BrdU solution (APC BrdU Flow Kit; BD               chromosomal inversions chicken, absence dis- <br /> Biosciences) phosphate-buffered saline (PBS), harvested     tinct genes, GDF3 Dppa3. Even genome of <br /> different time points, re-trypsinized, fixed Cytofix/      reptile Anolis carolinensis, Nanog located with <br /> Cytoperm (BD Sciences) buffer. Washing, re-fixation,          partially existing synteny scaffold (sc) 86. Upstream the <br /> BrdU staining performed according manufac-            Anolis Nanog gene, orthologous neighboring genes, <br /> turer&#226;&#8364;&#8482;s manual. For counterstaining, cells incubated      AM2 APOBEC1. Downstream Nanog 7 <br /> 7-amino-actinomycin (BD Biosciences) analyzed              genes, human orthologs human <br /> FACSCalibur.                                                      chromosome 19. The ortholog genes X. tropicalis lo- <br />    For cell tracking, amounts cells incu-        cated sc_1120. Another block genes neighboring <br /> bated 45 min 15 mM Cell Tracker Red CMPTX (Pro-            Nanog A. carolinensis X. tropicalis sc_735 <br /> mega). Dye working solution replaced, cells          sc_1151, Nanog gene these <br /> washed PBS. Cells trypsinized. Washing, incuba-         scaffolds. In teleost fishes, synteny completely lost. We <br /> tion, fixation performed according manu-          checked surroundings Nanog gene D. rerio and <br /> facturer&#226;&#8364;&#8482;s protocol (Promega). Cells analyzed             identical fugu, tetraodon, medaka <br /> FACSCalibur.                                                      Nanog genes, conservation Nanog locus <br />                                                                   comparison non-fishes. The Nanog gene D. rerio is <br /> Real-time reverse transcriptase-PCR                               located chromosome 24:11. By tracking opposite way, <br />    Total RNA extraction, purification, first-strand cDNA          checked genomic surroundings Nanog in <br /> synthesis, real-time reverse transcriptase (RT)-PCR      teleost fishes (medaka zebrafish) analyzed lo- <br /> done previously described [10]. Data presented          cation orthologous genes vertebrates (Fig. 1C). <br /> relative units. Primers RT-PCR listed Supplemen-       The genes surrounding fish Nanog genes in <br /> tary Table S1A, B; Supplementary Data available online        higher vertebrates orthologous different chromosomes <br /> at www.liebertonline.com/scd.                                     indicating various chromosomal translocations re- <br />                                                                   arrangements Nanog locus. Such chromosomal <br />                                                                   translocations X. tropicalis, there <br /> Embryonic staging situ hybridization                       evidence Nanog gene coupled these <br />    Embryonic stages determined Xenopus [35]          neighboring genes species. <br /> for zebrafish [37]. Whole mount situ hybridizations were <br /> performed according standard procedures [38,39].               Divergence Nanog proteins different species <br />                                                                   relationship Xvents <br /> Results                                                              Murine human Nanog described 3-domain <br /> Nanog present gnathostomata                          proteins containing homeodomain (HD), serine enriched <br /> for Xenopus                                                       N-terminal, C-terminal transactivator domain con- <br />                                                                   taining tryptophan repeats [40,41]. The tryptophan <br />    To identify Nanog orthologs lower vertebrates,           repeats shown crucial dimerization, prolifer- <br /> performed BLAST analyses genomic, EST, protein           ation, maintenance Nanog-dependent cell self- <br /> databases different species. We detected Nanog ortho-        renewal, binding Nac1 [42&#226;&#8364;&#8220;44]. However, when <br /> logs pro- metatheria (platypus opos-          comparing mammalian Nanog proteins me- <br /> sum), bony fishes (Cyprinidae: D. rerio;      daka chicken, highly divergent. Therefore, we <br /> Gasterosteidae: stickleback; Gadidae: Atlantic cod; Salmonidae:   asked common characteristics Nanog proteins and <br /> salmon; tetradontidae: fugu tetraodon) (Fig. 1A). The EST     aligned orthologs different species (Supplementary <br /> CO049347 little skate (Leucoraja erinace) gives rise    Fig. S1). In addition high rates identity inside the <br /> the assumption Nanog exists cartilaginous         HD, conserved aspartate/glutamate stretches and <br /> fishes. However, unable detect Nanog ge-        serine/threonine residues C- N-terminus. <br /> nome lamprey (Petromyzon marinus), representative        The C-terminally located tryptophan repeats mainly <br /> of jaw-less vertebrates (agnatha), acranial vertebrates        eutherians, 2 conserved residues found <br />  FIG. 1. Nanog exists gnathostomata Xenopodinae. (A) Phylogenetic tree verification Nanog. Species red <br /> letters: Nanog present. Species black letters: Nanog missing. (B) Synteny analysis human Nanog neighboring <br /> genes lower vertebrates created Ensembl database (Anolis carolinensis, version 1.0; Xenopus tropicalis, version 4.1). (C) <br /> Synteny analysis fish Nanog neighboring genes higher vertebrates. (D) Alignment homeodomain (HD) of <br /> Nanog zebrafish, medaka, human, HD Xvent2.1, Nkx2.5, HoxB6 X. tropicalis. Amino acids (PAM <br /> homology) shown red (3 6). Percentages homologies zebrafish Nanog given brackets. D.r., Danio rerio; O.l., <br /> Oryzias latipes; H.s., Homo sapiens; X.t., Xenopus tropicalis. <br />  <br />                                                               1228 <br />  ZEBRAFISH NANOG AND XENOPUS VENT GENES                                                                                           1229 <br />  <br /> in pro- metatheria (pos. 198 203 mouse               hybridization (Fig. 3A; Supplementary Table S2A, S2B). We <br /> protein).                                                           tested Sox17 (endodermal marker), sonic hedgehog (Shh), a <br />    Inspection Nanog HD reveals striking aspect be-         marker neural ectoderm, target Ptc-1. We in <br /> ing characteristic BarH subfamily HD proteins [45],      cases disturbed expression patterns loss the <br /> that specific amino acid exchange aliphatic            boundaries expression domains. Southpaw (Spaw), a <br /> nonpolar residue, valine isoleucine, polar        nodal-related gene, normally expressed left <br /> residue threonine (pos. 241 zebrafish protein) (Fig. 1D).    somite stages [46], bilaterally expressed D.r. <br /> This exchange detected Xenopus HD              Nanog RNA injected embryos. Marker gene analysis real- <br /> proteins, Vent proteins Hox11,           time RT-PCR (Fig. 3B, details Supplementary Table <br /> Nkx distal-less (Dll) proteins, share substantial     S3A) revealed Shh repressed embryos injected <br /> identities HD Nanog. The highest rates identity      higher amounts Nanog RNA. Pou2 upregulated, <br /> for Nanog proteins exist Xenopus Vent proteins.                low dose Nanog RNA. This result consistent <br /> Xvent2.1 X. tropicalis (we used gene1         situation mammals, Nanog stimulates <br /> Xvent2, Xvent1/2/3 genes du-           Oct4 expression vice versa [14,44]. We observed no <br /> plicated X. tropicalis) Nanog protein, belongs   change ventral markers (BMP4, Vent, Vox, Ved) low <br /> the BarH class [45] closely related Nanog (Fig. 1D;       amounts Nanog RNA, upregulation Ved (ventrally <br /> Supplementary Fig. S1). However, crucial attribute         expressed dharma/bozozok antagonist) BMP4 higher <br /> Nanog HD highly conserved aliphatic, nonpolar residue          amounts Nanog RNA. Interestingly, goosecoid (Gsc), a <br /> in helix a2 (M pos. 225 zebrafish Nanog), con-          marker dorsal mesoderm, upregulated. This finding <br /> tributes DNA binding minor groove contacts [36].         indicates Nanog signaling does shift cells <br /> All HD proteins, including Vent proteins,          ventral fate, acts complex <br /> basic amino acid (mostly lysine arginine) position.      manner. Moreover, observed strong decrease the <br /> Further, 3 invariant positions Nanog HD             neuroectodermal marker key regulator midbrain- <br /> (zebrafish Q218, Y237, K238) exchanged             hindbrain boundary, Pax2a [47]. Surprisingly, chemokine <br /> Xvent2.1. Further, triplet amino acids (trf),         receptor Cxcr4b, known guidance molecule primor- <br /> invariant Nanog proteins, partially conserved          dial germ cells zebrafish medaka [48,49], re- <br /> Xvent2.1 (S pos. 71 murine Nanog) (Supplementary Fig.         pressed. This unexpected, Cxcr4b shown a <br /> S1), BarH proteins related Dll           direct target Nanog medaka downregulated <br /> proteins (data shown). We S/T resi-          Nanog depletion [50]. Gata4 Gata6, were <br /> dues N-terminus Xvent2.1 described Nanog.          shown regulators endodermal cell fate Xenopus <br /> However, 2 invariant amino acids flanking polar resi-         [51], significantly changed, re- <br /> due adjacent HD domain Nanog C-terminus,           pressed Nanog human ES cells [52,53]. <br /> a basic tryptophan residue (b_r_W), detected             LOF studies performed using morpholino (MO) <br /> in Xvent2.1.                                                        based knockdown strategy. Specificity NanogMO was <br />                                                                     demonstrated vitro coupled transcription/translation <br /> Expression <span id='am-159' about='xsp:pattern' typeof='owl:Thing'>pattern</span> functional characterization                  assays D.r. Nanog RNA mutated Nanog (mut-Nanog) <br /> of D.r. Nanog                                                       RNA, containing 11 silent mutations putative MO <br />                                                                     binding site, presence NanogMO unrelated <br />    To investigate role Nanog lower vertebrates,        ctrlMO (Supplementary Fig. S2). Translation D.r. Nanog <br /> analyzed function zebrafish. The coding sequence        RNA blocked NanogMO ctrlMO. In addition, <br /> of D.r. Nanog obtained image clone BC162318.           NanogMO suppress translation mut-Nanog RNA. <br /> The corresponding protein contains 384 amino acids. We              NanogMO ctrlMO injected zebrafish embryos <br /> determine temporal expression pattern Nanog real-         cell stage. Although NanogMO injection does result <br /> time RT-PCR. Nanog RNA detected un-               disturbed embryogenesis gastrulation (data not <br /> fertilized eggs (Fig. 2A). The transcript strongly     shown), epiboly progression strongly im- <br /> increased beginning gastrulation, peaks germ ring      paired 99% injected embryos (Fig. 3C, Supplementary <br /> stage, decreasing. When segmentation starts          Table S2C). Although blastoderm continues thin <br /> 9 h postfertilization, Nanog transcripts greatly dimin-         covered yolk cells 90% epiboly unin- <br /> ished. From somite stage onward, Nanog detected           jected ctrlMO injected embryos, epiblast Na- <br /> anymore. We examined spatial expression pattern            nogMO injected embryos strongly thickened covered <br /> by mount situ hybridization (Fig. 2B). Nanog           approximately half yolk. Later on, NanogMO in- <br /> already detected 4-, 16-, 128-cell stages. High levels    jected embryos die, control siblings develop normally. <br /> of Nanog observed gastrulation, end        To ensure phenotypes yielded NanogMO injections <br /> gastrulation (75% epiboly), transcripts strongly        specific Nanog depletion toxic effect the <br /> diminished. From segmentation onward (90% epiboly), Na-             NanogMO, performed rescue experiments. In contrast to <br /> nog detected anymore.                                     NanogMO injection alone, NanogMO mut- <br />    Next, performed gain function studies Nanog             Nanog RNA co-injected embryos overcome critical stages and <br /> overexpression zebrafish embryos. Injection D.r. Nanog        develop (Supplementary Table S2C; Fig. 6A). <br /> RNA did prevent gastrulation, embryos did                  We analyzed Nanog depleted embryos situ hy- <br /> display obvious abnormalities gastrulation (data         bridization RT-PCR expression ventral marker <br /> shown). To obtain precise information genetic               genes, BMP4, Ved, Vent (Fig. 3C, D; Supplemen- <br /> dysregulation, marker genes analyzed situ        tary Tables S2D S3B). Although BMP4 expression, <br />  1230                                                                                                           SCHUFF ET AL. <br />  <br />  <br />  <br />  <br /> FIG. 2. Nanog expression during <br /> zebrafish embryogenesis. (A) Temporal <br /> expression D.r. Nanog analyzed real- <br /> time RT-PCR. Histone H4 used as <br /> internal control, expression given <br /> in relative units. (B) Expression analysis <br /> of D.r. Nanog mount situ <br /> hybridization. Four-cell 75% epiboly <br /> are viewed top, insert shows a <br /> lateral view shield (red arrowhead). <br /> About 90% epiboly long pec stage are <br /> shown laterally. RT-PCR, reverse tran- <br /> scriptase&#226;&#8364;&#8220;polymerase chain reaction. <br />  <br />  <br />  <br />  <br /> responsible specification ventro-posterior cell fate   asked overexpression zebrafish Nanog Xvent <br /> in late phase BMP signaling [54], unaf-          Xenopus results similar <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span>. Bi- <br /> fected, maternally expressed Ved [55] strongly re-         lateral injections D.r. Nanog Xvent2 RNA 2-cell stage <br /> pressed. In addition, Vent Vox, transcribed      delay blastopore formation (Fig. 4A, Supplementary Table <br /> midblastula transition (MBT), decreased Nanog de-          S2E). Dorsal injections D.r. Nanog Xvent2 RNA 4-cell <br /> pleted embryos. Gata4 Gata6 expression de-            stage embryos lead lacking head structures (Fig. 4A). We <br /> creased (Fig. 3D). We observe Nanog depleted           performed real-time RT-PCR selected marker genes <br /> embryos repression Gsc Pou2, confirming re-          different developmental stages injection mouse <br /> sults Nanog overexpression activate Gsc Pou2         D.r. Nanog Xvent1/2 RNA (Fig. 4B, Supple- <br /> expression (Fig. 3B, D). This contrast medaka,     mentary Table S3C). We focused markers undif- <br /> Nanog depletion influence Oct4 expression level         ferentiated state (Oct60, Tert, Klf4), germ layer formation <br /> [27]. In addition, Pax2a Cxcr4b, repressed         (such Xbra, Gsc, Siamois), cell cycle (Wee1b), me- <br /> by overexpression, strongly upregulated Nanog de-          tabolism (PepCK, Sod2), including genes microarray <br /> pleted embryos (Fig. 3B, D).                                      analyses reported changed Nanog ES cells <br />    Since Vent genes display close sequence homology      (marked asterisks) [56]. Importantly, mouse D.r. <br /> zebrafish Nanog similar temporal expression profile         Nanog RNA injections mainly lead identical regulations. <br /> with transcriptional activation MBT [31,32],            Although genes similarly affected Xvent RNA <br />  ZEBRAFISH NANOG AND XENOPUS VENT GENES                                                                                       1231 <br />  <br />                                                                   injections, is, Sod2, Wee1b, XSox3, Oct60, an <br />                                                                   inverse regulation Gsc Sox2 (Fig. 4B). <br />  <br />                                                                   Murine D.r. Nanog similar protein <br />                                                                   interactions substitute vivo <br />                                                                      Mammalian Nanog acts dimer maintenance of <br />                                                                   self-renewal absence LIF, dimerization Na- <br />                                                                   nog shown dependent C-terminally located <br />                                                                   tryptophan repeats [42,43]. Axolotl Nanog reported to <br />                                                                   unable dimerization, lacks tryptophan <br />                                                                   repeats [29]. Since repeats obviously specific for <br />                                                                   mammals (Supplementary Fig. S1), absent from <br />                                                                   zebrafish. However, GST-pulldown analyses, found <br />                                                                   D.r. Nanog self-associate (Fig. 5A). Size exclusion <br />                                                                   chromatography radiolabeled D.r. Nanog Sephadex G- <br />                                                                   100 revealed 70% labeled protein eluted dimers <br />                                                                   30% monomers (data shown). We prepared de- <br />                                                                   letion constructs Nanog delineate putative dimer- <br />                                                                   ization domain N-terminal region well <br />                                                                   HD required dimerization process (Fig. 5A). <br />                                                                   Noteworthy, contrast mammalian axolotl Nanog, <br />                                                                   N-terminal region zebrafish Nanog displays en- <br />                                                                   richment tryptophan residues (Supplementary Fig. S1). <br />                                                                   We investigated D.r. Nanog physically <br />                                                                   interacts Pou2 homologous Xenopus protein Oct25. <br />                                                                   Mammalian axolotl Nanog bind Oct4 [22,24,29]. GST- <br />                                                                   pulldown analyses Oct25 murine zebrafish Na- <br />                                                                   nog demonstrated HD transcription factors interact <br />                                                                   Oct25 (Fig. 5B). <br />                                                                      Next, tested EMSA 22mer murine Tcf3 pro- <br />                                                                   moter, shown sufficient murine Nanog <br />                                                                   binding [36], binding D.r. Nanog Xvent2. The re- <br />                                                                   ported consensus binding sequences murine Nanog <br />                                                                   (TAATGG, [36]) Xvent2 (CTAATT, [57,58]) cer- <br />                                                                   tain overlap. However, murine as <br />                                                                   zebrafish Nanog bind 22mer, Xvent2 does <br />                                                                   interact binding site (Fig. 5C). This difference in <br />                                                                   DNA binding Xvent Nanog the <br />                                                                   mentioned amino acid exchange a2 helix of <br />                                                                   HD (basic residue/aliphatic, nonpolar residue). <br />                                                                      We asked D.r. Nanog, mouse Nanog, and <br /> FIG. 3. Gain function (GOF) loss function (LOF)         Xvent substitute LOF zebrafish Xe- <br /> analyses D.r. Nanog. (A) Whole mount situ hybridiza-        nopus embryos, respectively. First, tried overcome Na- <br /> tion wild-type, control (unrelated nucleotide sequence),       nogMO dependent gastrulation defects subsequent death <br /> and Nanog RNA (400 pg) injected embryos hybridized           co-injection mutated D.r. Nanog, mouse Nanog, or <br /> Sox17 (80% epiboly), Shh (90% epiboly), Ptc-1 (90% epiboly), <br />                                                                   Xvent2 RNA (Fig. 6A). Although blastoderm Nanog <br /> or Spaw (12 somite stage). (B) Real-time RT-PCR D.r. Na- <br /> nog RNA injected embryos 6 somite stage indicated          depleted embryos remains does cover yolk, <br /> marker genes relative wild-type embryos. (C) Uninjected        co-injection mouse Nanog RNA successfully substituted for <br /> control, ctrlMO, NanogMO (1.5 ng) injected embryos         zebrafish Nanog, gastrulation defects reverted, <br /> 90% epiboly (lateral view) situ hybridization (animal      embryos survived (Fig. 6A; Supplementary Table <br /> view, ventral left orientated) BMP4, Ved,    S2C). However, co-injection Xvent2 RNA different <br /> Vent (D) Real-time RT-PCR NanogMO injected embryos          amounts does inhibit attenuate NanogMO <br /> relation ctrlMO injected embryos indicated marker          based gastrulation defects. Vice versa, depleted Xvent1 <br /> genes 80% epiboly. Histone H4 used internal control.     Xvent2 Xenopus embryos MO injections tried to <br /> Green: upregulation ( &#226;&#8364;&#161; 1.5), red: decrease ( &#194;&#163; 0.75), gray:   rescue wild-type phenotype co-injection Danio Nanog <br /> change transcript level (0.75 &lt; &#194;&#183; &lt; 1.5). MO, morpholino <br />                                                                   RNA (Fig. 6B). Co-injections Xvent1MO/Xvent2MO result <br /> oligonucleotide; ctrl MO, control MO. <br />                                                                   hyper-dorsalized embryos strongly reduced body axis <br />                                                                   [33]. Co-injections murine (data shown) zebrafish <br />                                                                   Nanog RNA various concentrations did restore dorso- <br />                                                                   ventral body axis attenuate Xvent1/2 depletion phe- <br />                                                                   notype. Therefore, conclude Nanog Xvent cannot <br />  FIG. 4. Comparison Xvent D.r. Nanog overexpression Xenopus laevis. (A) Overexpression Nanog Xvent2 X. <br /> laevis embryos results similar phenotypes. Embryos shown stage 11 (vegetal view), stage 18 (dorsal view), at <br /> stage 29 (lateral view). RNAs bilaterally injected indicated amounts 2-cell stage embryos 2 dorsal cells <br /> at 4-cell stage, respectively. (B) Schematic presentation real-time RT-PCR data derived overexpression Nanog and <br /> Xvent1/2 X. laevis stages 10, 11, 12. M.m. Nanog (400 pg), D.r. Nanog (400 pg), Xvent1 (350 pg), Xvent2 RNA (350 pg) <br /> were bilaterally injected 2-cell stage embryos. Histone H4 used internal control. Green: upregulation ( &#226;&#8364;&#161; 1.5), red: <br /> decrease ( &#194;&#163; 0.75), gray: change transcript level (0.75 &lt; &#194;&#183; &lt; 1.5). Marker genes shown regulated Nanog <br /> in ES cells [56] indicated asterisks. ES, embryonic stem. <br />  <br />                                                              1232 <br />  ZEBRAFISH NANOG AND XENOPUS VENT GENES                                                                                         1233 <br />  <br />  <br />  <br />  <br />                                                                                                     FIG. 5. Protein DNA <br />                                                                                                     binding properties Nanog. <br />                                                                                                     (A) Scheme Nanog deletion <br />                                                                                                     mutants used pulldown <br />                                                                                                     analysis       GST-Nanog. <br />                                                                                                     White box: HD; DHD: dele- <br />                                                                                                     tion HD; HD + : HD of <br />                                                                                                     Nanog surrounding 5 <br />                                                                                                     amino acids. Note di- <br />                                                                                                     merization D.r. Nanog <br />                                                                                                     does require C-termi- <br />                                                                                                     nus. (B) Oct25 interacts with <br />                                                                                                     mouse D.r. <br />                                                                                                     Nanog. 35S methionine-la- <br />                                                                                                     beled mouse zebrafish <br />                                                                                                     Nanog proteins incu- <br />                                                                                                     bated GST-fusion <br />                                                                                                     protein Oct25. (C) Electro- <br />                                                                                                     phoretic mobility shift assay <br />                                                                                                     22mer murine Tcf-3 <br />                                                                                                     promoter. D.r. murine <br />                                                                                                     Nanog Xvent2 inter- <br />                                                                                                     act Nanog binding <br />                                                                                                     site. <br />  <br />  <br />  <br />  <br /> substitute gastrulation processes zebrafish       culturing (Fig. 7A, B). In contrast, M.m. Nanog as <br /> and dorso-ventral patterning X. laevis.                      D.r. Nanog overexpression able inhibit differentia- <br />                                                                    tion, visible round shape blue AP staining of <br /> D.r. Nanog Xvent promotes proliferation                    cultured embryoid bodies. This sub- <br /> and LIF-independent self-renewal mouse                          stantiated expression analysis selected marker <br /> ES cells                                                           genes absence LIF. Although pluripotency genes <br />                                                                    Pou5f1 Sox2 highly expressed D.r. Nanog M.m. <br />    To test M.m. Nanog, D.r. Nanog, Xvent2          Nanog transfected cells, differentiation markers T (bra- <br /> the effects regarding cell differentiation cell prolif-   chyury) Sox17 exhibited strong increase Xvent2 and <br /> eration ES cells, generated stably transfected mouse ES      pFIEN vector transfected cells (Supplementary Fig. S4). <br /> cell lines constructs (Supplementary Fig. S3). The ef-       Previous studies implicate important role Nanog cell <br /> fects cell differentiation monitored embryoid body      proliferation mammalian [42] the <br /> assays, morphology loss pluripotency          axolotl medaka protein [27,29]. To test D.r. Nanog <br /> were analyzed. With LIF, inhibiting differentiation     Xvent2 enhance cell proliferation, performed BrdU <br /> ES cells, embryoid bodies tested cell lines kept round,   cell tracking assays transfected ES cell lines. We <br /> undifferentiated shape, mainly consisted pluripotent        observed higher level BrdU incorporation mouse <br /> cells. Without LIF, control cell lines (E14 pFIEN vector       Nanog D.r. Nanog transfected cells contrast Xvent2 or <br /> control) Xvent2 overexpressing cells started differentia-      vector transfected ES cells (Fig. 7C). To verify results, we <br /> tion lost pluripotent state, indicated      performed cell tracking assay. ES cells incubated <br /> the outgrowing expanding morphology de-                Red CMPTX cell tracker. A high proliferation rate re- <br /> creased blue AP staining embryoid bodies 4 days       sults higher basic value fluorescence intensity coupled <br />  1234                                                                                                             SCHUFF ET AL. <br />  <br />  <br />  <br />  <br /> FIG. 6. Mouse Nanog rescues zebrafish <br /> embryos Nanog knockdown. (A) Co- <br /> injection 400 pg mouse Nanog RNA and <br /> 1.5 ng NanogMO zebrafish embryos <br /> prevents knockdown dependent gas- <br /> trulation defect subsequent death, <br /> whereas co-injection 400 pg Xvent2 <br /> RNA so. Upper: uninjected <br /> control (90% epiboly), NanogMO injected <br /> and knockdown embryos co-injected with <br /> mut-Nanog RNA. Lower: knockdown em- <br /> bryos co-injected Xvent2 RNA co- <br /> injected mouse Nanog RNA 90% <br /> epiboly somitogenesis. (B) Co-in- <br /> jection D.r. Nanog RNA rescue <br /> the Xvent1/2MOs (45 ng) caused pheno- <br /> type X. laevis embryos. Left: uninjected <br /> X. laevis embryos, stage 29, middle: <br /> Xvent1MO Xvent2MO injected em- <br /> bryos. Right: Xvent1MO Xvent2MO <br /> injected embryos co-injected 100 pg <br /> D.r. Nanog RNA. <br />  <br />  <br />  <br />  <br /> with enhanced decrease fluorescence intensity,       chromosomal translocations Xenopodinae. <br /> cell tracker removed. We observed fluorescence         During evolution, Nanog passed 2 chromosomal re- <br /> intensities 45 min incubation higher decreased       arrangements, Osteichthyes Tetrapodae, sec- <br /> much faster D.r. mouse Nanog transfected ES cells       ond amphibians, Nanog depleted from <br /> in pFIEN Xvent2 transfected cells (Fig. 7D). Accordingly,    anurans or, specifically, Xenopus genome. <br /> ratio mean fluorescence intensity 0 h/1 day and, especially <br /> 0 h/2 day removal cell tracker, higher mouse    Characterization Nanog D. rerio <br /> and D.r. Nanog control Xvent2 transfected cells <br /> (Fig. 7D). Both experimental approaches, BrdU assay cell           The Nanog gene zebrafish maternally expressed and <br /> tracking, suggest murine D.r. Nanog enhance         strongly upregulated blastula stage development. <br /> proliferation.                                                      Transcripts detected shield, abundant <br />                                                                     gastrula stage, rapidly degraded 75% <br /> Discussion                                                          epiboly. The murine Nanog expression detected during <br /> The Nanog gene present amphibia                               embryogenesis morula stage onward, later re- <br /> but lacking anuran Xenopus                                 stricted epiblast, excluded primitive endo- <br />                                                                     derm [16,17]. Mouse Nanog detectable primitive <br />   EST data genomic linkage analyses suggest model             streak, downregulated cells ingressing the <br /> according Nanog arose gnathostomata evolved.          streak form mesoderm definitive endoderm [60]. It is <br /> In amphibians, Nanog identified newt             enriched pluripotent cell lines, ES cells, embryonic <br /> Notophthalmus viridescens [59] axolotl Ambystoma         germ cells, embryonic carcinoma cells, absent from <br /> mexicanum [29], Nanog reported absent            somatic tissues [13,17]. Overexpression zebrafish Nanog <br /> the X. tropicalis genome [30]. Since unable         revealed upregulation pluripotency factor Pou2 and <br /> identify ESTs Nanog X. laevis, assume Nanog       ventral (BMP4, Ved) dorsal-mesodermal (Gsc) mark- <br /> missing genus Xenopus. The genomic linkage           ers, Cxcr4b gene downregulated. Vice versa, <br /> analysis indicates Nanog gene deleted             knockdown Nanog led increase Cxcr4b Pax2a, <br />  FIG. 7. Mouse zebrafish Nanog promote self-renewal cell proliferation. (A) Morphology embryoid bodies 3 days <br /> with ( + ) ( - ) LIF (leukemia inhibitory factor). (B) Alkaline phosphatase (AP) staining embryoid bodies and <br /> without LIF 4 days culturing. (C) Nanog promotes cell proliferation. Percentages BrdU positive ES cells detected with <br /> FACSCalibur correlation incubation time hours. (D) Histograms fluorescence intensity transfected ES cells treated <br /> with Cell Tracker Red CMPTX, correlation percentages cell numbers analyzed FACSCalibur, diagram presenting <br /> the ratio mean fluorescence intensity (MFI) 0 h 1 2 days removal cell tracker. <br />  <br />                                                                1235 <br />  1236                                                                                                             SCHUFF ET AL. <br />  <br /> whereas Pou2, Gsc, Vent, Vox, Ved, Gata4, Gata6           derstanding role Nanog embryogenesis it <br /> downregulated. Nanog overexpression does affect            dispensable Xenopus. <br /> phenotype gastrulation, situ hybridizations <br /> revealed delocalized expression Sox17, Shh, Ptc-1, and <br /> Spaw. Similar previously reported medaka [27],         Acknowledgments <br /> Nanog knockdown zebrafish prevents epibolic move-                  The authors thank Alpaslan Tasdogan help the <br /> ments gastrulation lethal.                           proliferation assays, Christian Klein (Max Delbrueck Center <br />                                                                    Molecular Medicine) providing zebrafish embryos for <br /> The proliferative effect Nanog                                  Nanog situ hybridization, Janine Ziermann (In- <br /> is evolutionarily conserved                                        stitute Biology, Leiden University) providing EST se- <br />                                                                    quences Triturus carnifex helpful discussion. This <br />    D.r. Nanog forms homodimers maintains self-renewal          work supported grants W.K. (SFB 497/A1), M.P. <br /> of mouse ES cells absence LIF. This finding correlates   (Marie Curie IRG No. 268333), K.B. (ESF/M.v.W. <br /> to effects murine Nanog, axolotl Nanog          Scholarship). <br /> shown maintain self-renewal, tryptophan residues in <br /> the C-terminus introduced [29]. Moreover, found <br /> that murine Nanog able rescue depletion zebrafish         Author Disclosure Statement <br /> Nanog gastrulation. These findings indicate both <br />                                                                      There conflicts relating work for <br /> D.r. Nanog mouse Nanog behave functional homologs. <br />                                                                    authors. <br />    An important aspect Nanog function proliferative <br /> effect described mammalian, medaka, <br />                                                                    References <br /> and axolotl Nanog [27,29,42]. Here, demonstrate BrdU <br /> incorporation cell tracking D.r. Nanog transfected         1. Chambers I, J Silva, D Colby, J Nichols, B Nijmeijer, M Ro- <br /> mouse ES cells significant increase proliferation.           bertson, J Vrana, K Jones, L Grotewold A Smith. (2007). <br /> We, therefore, postulate regulation proliferation       Nanog safeguards pluripotency mediates germline de- <br /> an important common role Nanog proteins              velopment. Nature 450:1230&#226;&#8364;&#8220;1234. <br /> been evolutionarily conserved.                                      2. Masui S, Y Nakatake, Y Toyooka, D Shimosato, R Yagi, K Ta- <br />                                                                        kahashi, H Okochi, A Okuda, R Matoba, et al. (2007). Plur- <br />                                                                        ipotency governed Sox2 regulation Oct3/4 expression <br /> Is Nanog frogs replaced Xvents?                                  mouse embryonic stem cells. Nat Cell Biol 9:625&#226;&#8364;&#8220;635. <br />                                                                     3. Pan GJ JA Thomson. (2007). Nanog transcriptional <br />    How organism, Xenopus, survive without <br />                                                                        networks embryonic stem cell pluripotency. Cell Res <br /> an important gene vitality? Either basic develop-             17:42&#226;&#8364;&#8220;49. <br /> mental processes anurans/Xenopodinae evolution-             4. Cao Y, D Siegel W Kno&#194;&#168;chel. (2006). Xenopus POU fac- <br /> arily changed, Nanog gene dispensable,             tors subclass V inhibit activin/nodal signaling during <br /> or function(s) Nanog gene replaced           gastrulation. Mech Dev 123:614&#226;&#8364;&#8220;625. <br /> functionally related genes. Therefore, asked whe-           5. Morrison GM JM Brickman. (2006). Conserved roles for <br /> ther Xenopus Vent genes compensated                 Oct4 homologues maintaining multipotency early <br /> missing Nanog. Sequence homologies, especially regarding               vertebrate development. Development 133:2011&#226;&#8364;&#8220;2022. <br /> the BarH specific threonine residue conserved serine        6. Niwa H, Y Sekita, E Tsend-Ayush F Gru&#194;&#168;tzner. (2008). <br /> residue outside HD,             Platypus Pou5f1 reveals steps evolution of <br /> BarH proteins, suggest Vent Nanog arose                trophectoderm differentiation pluripotency mam- <br /> common ancestor. In addition, temporal expression                  mals. Evol Dev 10:671&#226;&#8364;&#8220;682. <br /> profiles zebrafish Nanog Xenopus Vents sim-         7. Frankenberg S, A Pask MB Renfree. (2010). The evolu- <br /> ilar. Overexpression zebrafish Nanog Xenopus Vents           tion class V POU domain transcription factors verte- <br /> frogs revealed similar phenotypes, developmen-                brates characterisation marsupial. Dev Biol <br /> tally important genes regulated manner.               337:162&#226;&#8364;&#8220;170. <br /> Moreover, zebrafish Nanog Xvent2 dimerize            8. Onichtchouk D, F Geier, B Polok, DM Messerschmidt, R <br /> and bind Oct25. However, important                  Mo&#194;&#168;ssner, B Wendik, S Song, V Taylor, J Timmer W <br /> genes, dorsalizing genes Gsc Chordin,            Driever. (2010). Zebrafish Pou5f1-dependent transcriptional <br />                                                                        networks temporal control early development. Mol <br /> are regulated Nanog Xvent opposite way. <br />                                                                        Syst Biol 6:354. <br /> Moreover, Nanog stimulates cell proliferation, whereas <br />                                                                     9. Pierre A, M Gautier, I Callebaut, M Bontoux, E Jeanpierre, P <br /> Xvent stimulating effect. Finally, that <br />                                                                        Pontarotti P Monget. (2007). Atypical structure and <br /> neither Nanog LOF zebrafish overcome Xvent                phylogenomic evolution new eutherian oocyte- and <br /> overexpression LOF Xvents frogs overcome              embryo-expressed KHDC1/DPPA5/ECAT1/OOEP gene <br /> by zebrafish Nanog overexpression. In contrast, murine                 family. Genomics 90:583&#226;&#8364;&#8220;594. <br /> Nanog able rescue depletion Nanog zeb-       10. Siegel D, M Schuff, F Oswald, Y Cao W Kno&#194;&#168;chel. (2009). <br /> rafish embryos gastrulation. These experiments                  Functional dissection XDppa2/4 structural domains in <br /> clearly demonstrate Nanog function Xenopus              Xenopus development. Mech Dev 126:974&#226;&#8364;&#8220;989. <br /> simply substituted Xvent genes. Either       11. Petitte JN, G Liu Z Yang. (2004). Avian pluripotent stem <br /> genes involved Nanog function se dispensable              cells. Mech Dev 121:1159&#226;&#8364;&#8220;1168. <br /> Xenopus. The identification Nanog additional am-           12. Fan L, J Crodian, X Liu, A Alestro&#194;&#168;m, P Alestro&#194;&#168;m P <br /> phibians, particular anurans, help un-            Collodi. (2004). Zebrafish embryo cells remain pluripotent <br />  ZEBRAFISH NANOG AND XENOPUS VENT GENES                                                                                             1237 <br />  <br />       germ-line competent multiple passages culture.            stem cells demonstrates ground state pluripotency is <br />       Zebrafish 14:43&#226;&#8364;&#8220;51.                                                  conserved urodele amphibians mammals. Devel- <br /> 13.   Chambers I, D Colby, M Robertson, J Nichols, S Lee,                  opment 137:2973&#226;&#8364;&#8220;2980. <br />       S Tweedie A Smith. (2003). Functional expression           30.   Hellsten U, RM Harland, MJ Gilchrist, D Hendrix, J Jurka, <br />       cloning Nanog, pluripotency sustaining factor em-            V Kapitonov, I Ovcharenko, NH Putnam, S Shu, et al. (2010). <br />       bryonic stem cells. Cell 113:643&#226;&#8364;&#8220;655.                                The genome Western clawed frog Xenopus tropicalis. <br /> 14.   Loh YH, Q Wu, JL Chew, VB Vega, W Zhang, X Chen,                     Science 328:633&#226;&#8364;&#8220;636. <br />       G Bourque, J George, B Leong, et al. (2006). The Oct4      31.   Gawantka V, H Delius, K Hirschfeld, C Blumenstock and <br />       Nanog transcription network regulates pluripotency                C Niehrs. (1995). Antagonizing Spemann organizer: role <br />       mouse embryonic stem cells. Nat Genet 38:431&#226;&#8364;&#8220;440.                    homeobox gene Xvent-1. EMBO J 14:6268&#226;&#8364;&#8220;6279. <br /> 15.   Silva J, J Nichols, T Theunissen, G Guo, AL van Oosten,        32.   Onichtchouk D, V Gawantka, R Dosch, H Delius, <br />       O Barrandon, J Wray, S Yamanaka, I Chambers A                    K Hirschfeld, C Blumenstock C Niehrs. (1996). The <br />       Smith. (2009). Nanog gateway pluripotent               Xvent-2 homeobox gene BMP-4 signalling <br />       ground state. Cell 138:722&#226;&#8364;&#8220;737.                                      pathway controlling dorsoventral patterning Xenopus <br /> 16.   Wang SH, MS Tsai, MF Chiang H Li. (2003). A novel                mesoderm. Development 122:3045&#226;&#8364;&#8220;3053. <br />       NK-type homeobox gene, ENK (early embryo specific NK),         33.   Sander V, B Reversade EM De Robertis. (2007). The <br />       preferentially expressed embryonic stem cells. Gene Expr          opposing homeobox genes Goosecoid Vent1/2 self- <br />       Patterns 3:99&#226;&#8364;&#8220;103.                                                   regulate Xenopus patterning. EMBO J 26:2955&#226;&#8364;&#8220;2965. <br /> 17.   Mitsui K, Y Tokuzawa, H Itoh, K Segawa, M Murakami,            34.   Cao Y, D Siegel, F Oswald W Kno&#194;&#168;chel. (2008). Oct25 <br />       K Takahashi, M Maruyama, M Maeda S Yamanaka.                     represses transcription nodal/activin target genes in- <br />       (2003). The homeoprotein Nanog required mainte-               teraction signal transducers Xenopus gastrula- <br />       nance pluripotency mouse epiblast ES cells. Cell           tion. J Biol Chem 283:34168&#226;&#8364;&#8220;34177. <br />       113:631&#226;&#8364;&#8220;642.                                                   35.   Nieuwkoop PD J Faber. (1967). Normal Table Xenopus <br /> 18.   Yates A I Chambers. (2005). The homeodomain protein              laevis (Daudin), 2nd edn. North Holland, Amsterdam, Else- <br />       Nanog pluripotency mouse embryonic stem cells.                vier. <br />       Biochem Soc Trans 33:1518&#226;&#8364;&#8220;1521.                                36.   Jauch R, CK Ng, KS Saikatendu, RC Stevens PR Ko- <br /> 19.   Okita K, T Ichisaka S Yamanaka. (2007). Generation            latkar. (2008). Crystal structure DNA binding the <br />       germline-competent induced pluripotent stem cells. Nature            homeodomain stem cell transcription factor Nanog. <br />       448:313&#226;&#8364;&#8220;317.                                                         J Mol Biol 376:758&#226;&#8364;&#8220;770. <br /> 20.   Silva J, I Chambers, S Pollard A Smith. (2006). Nanog      37.   Kimmel CB, WW Ballard, SR Kimmel, B Ullmann TF <br />       promotes transfer pluripotency cell fusion. Nature          Schilling. (1995). Stages embryonic development the <br />       441:997&#226;&#8364;&#8220;1001.                                                        zebrafish. Dev Dyn 203:253&#226;&#8364;&#8220;310. <br /> 21.   Boyer LA, TI Lee, MF Cole, SE Johnstone, SS Levine, JP         38.   Harland RM. (1991). In situ hybridization: improved <br />       Zucker, MG Guenther, RM Kumar, HL Murray, et al. (2005).             whole-mount method Xenopus embryos. Methods Cell <br />       Core transcriptional regulatory circuitry human embry-            Biol 36:685&#226;&#8364;&#8220;695. <br />       onic stem cells. Cell 122:947&#226;&#8364;&#8220;956.                             39.   Concordet JP, KE Lewis, JW Moore, LV Goodrich, <br /> 22.   Wang J, S Rao, J Chu, X Shen, DN Levasseur, TW Theu-                 RL Johnson, MP Scott PW Ingham. (1996). Spatial reg- <br />       nissen SH Orkin. (2006). A protein interaction network           ulation zebrafish patched homologue reflects roles of <br />       pluripotency embryonic stem cells. Nature 444:364&#226;&#8364;&#8220;            sonic hedgehog protein kinase A neural tube and <br />       368.                                                                 somite patterning. Development 122:2835&#226;&#8364;&#8220;2846. <br /> 23.   Chambers I A Smith. (2004). Self-renewal teratocar-     40.   Pan GJ DQ Pei. (2003). Identification distinct <br />       cinoma embryonic stem cells. Oncogene 23:7150&#226;&#8364;&#8220;7160.              transactivation domains pluripotency sustaining fac- <br /> 24.   Liang J, M Wan, Y Zhang, P Gu, H Xin, SY Jung, J Qin,                tor nanog. Cell Res 13:499&#226;&#8364;&#8220;502. <br />       J Wong, AJ Cooney, D Liu Z Songyang. (2008). Nanog         41.   Oh JH, HJ Do, HM Yang, SY Moon, KY Cha, HM Chung and <br />       Oct4 associate unique transcriptional repression            JH Kim. (2005). Identification putative transactivation <br />       complexes embryonic stem cells. Nat Cell Biol 10:731&#226;&#8364;&#8220;739.         domain human Nanog. Exp Mol Med 37:250&#226;&#8364;&#8220;254. <br /> 25.   Can&#203;&#339;o&#194;&#180;n S, C Herranz M Manzanares. (2006). Germ cell       42.   Ma T, Z Wang, Y Guo D Pei. (2009). The C-terminal <br />       restricted expression chick Nanog. Dev Dyn 235:2889&#226;&#8364;&#8220;              pentapeptide Nanog tryptophan repeat domain interacts <br />       2894.                                                                Nac1 regulates stem cell proliferation not <br /> 26.   Lavial F, H Acloque, F Bertocchini, DJ Macleod, S Boast,             pluripotency. J Biol Chem 284:16071&#226;&#8364;&#8220;16081. <br />       E Bachelard, G Montillet, S Thenot, HM Sang, et al. (2007).    43.   Mullin NP, A Yates, AJ Rowe, B Nijmeijer, D Colby, PN <br />       The Oct4 homologue PouV Nanog regulate plur-                     Barlow, MD Walkinshaw I Chambers. (2008). The <br />       ipotency chicken embryonic stem cells. Development                pluripotency rheostat Nanog functions dimer. Biochem J <br />       134:3549&#226;&#8364;&#8220;3563.        <br /> </body></html>