<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                 GBE <br />  <br /> All Nothing: Protein Complexes Flip Essentiality <br /> between Distantly Related Eukaryotes <br /> Colm J. Ryan1,*, Nevan J. Krogan2,3,4, Pa&#194;&#180;draig Cunningham1, Gerard Cagney5 <br /> 1 <br />  School Computer Science Informatics, University College Dublin, Ireland <br /> 2 <br />  Department Cellular Molecular Pharmacology, University California, San Francisco <br /> 3 <br />  California Institute Quantitative Biosciences, San Francisco, California, USA <br /> 4 <br />  J. David Gladstone Institutes, San Francisco, California, USA <br /> 5 <br />  School Biomolecular &amp; Biomedical Science, University College Dublin, Ireland <br /> *Corresponding author: E-mail: colm.ryan@ucd.ie. <br /> Accepted: May 3, 2013 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> In budding yeast Saccharomyces cerevisiae, subunits given protein complex essential mostly <br /> nonessential, suggesting essentiality property molecular machines individual components. There excep- <br /> tions rule, however, is, nonessential genes largely essential complexes essential genes largely nonessential <br /> complexes. Here, provide explanations exceptions, showing redundancy complexes, revealed genetic <br /> interactions, explain cases, &#226;&#8364;&#339;moonlighting,&#226;&#8364;? revealed membership multiple complexes, can <br /> explain latter. Surprisingly, redundancy complexes usually explained gene duplication, suggest- <br /> ing alternate buffering mechanisms. In distantly related Schizosaccharomyces pombe, observe phenomenon of <br /> modular essentiality, suggesting general feature eukaryotes. Furthermore, complexes flip essentiality <br /> in cohesive fashion species, is, tend change essential nonessential, vice <br /> versa, mixed patterns. We flips essentiality explained differing lifestyles yeasts. <br /> Collectively, results support previously proposed model proteins essential involvement essential <br /> functional modules specific topological features degree centrality. <br /> Key words: essentiality, modularity, yeast, redundancy, genetic interactions, protein complexes. <br />  <br />  <br /> Introduction                                                                                   Two global experimental studies sought identify <br /> Much work cells carried molecular machines                                 comprehensive maps protein complexes organism <br /> known protein complexes (Alberts 1998; Hartwell et al.                                      (Gavin et al. 2006; Krogan et al. 2006), others <br /> 1999). These machines composed multiple proteins                                        curated literature identify hundreds com- <br /> that connected single physical <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>.                                         plexes support small-scale experiments (Pu et al. <br /> In ways, canonical example functional                                       2009). The availability genome wide set gene de- <br /> modules biomolecular systems (Pereira-Leal et al. 2006):                                    letion mutants S. cerevisiae (Giaever et al. 2002) also <br /> their protein components densely connected protein                                  organism extremely useful understanding how <br /> interaction network, suggesting modularity network                                      genotypes map phenotypes. Perhaps, easiest pheno- <br /> theory sense (Newman 2006); relatively independent,                                   type measure yeast cell growth viability, and <br /> as reconstituted vitro absence                                  large-scale studies gene deletion collection <br /> surrounding network (Pereira-Leal et al. 2006); work                                  focused it. Giaever et al. (2002) noted ap- <br /> together carry common function.                                                       proximately 81% genes individually deleted in <br />    The budding yeast Saccharomyces cerevisiae proved                                       rich media causing cell death&#226;&#8364;&#8221;that is, were <br /> <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> extraordinary resource understanding                                          nonessential. The remaining approximately 19% es- <br /> organization behavior molecular machines.                                         sential cell <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span>. <br />  <br />  <br /> &#195;&#376; The Author(s) 2013. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), permits <br /> non-commercial re-use, distribution, reproduction medium, provided original work properly cited. For commercial re-use, contact journals.permissions@oup.com <br />  <br />  <br />  <br />  <br /> Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013                                                                             1049 <br />  Ryan et al.                                                                                                                                              GBE <br />  <br />     A disproportionate number essential genes                     property molecular machines individual compo- <br /> noted belong protein complexes (Hart et al. 2007;                         nents (fig. 1A) (Dezso et al. 2003; Hart et al. 2007; Semple et al. <br /> Michaut et al. 2011). Indeed, relationship                        2008; Wang et al. 2009). Wang et al. (2009) connected the <br /> gene&#226;&#8364;&#8482;s essentiality connectivity encoded protein                 observations noting larger protein complexes are <br /> has subject analysis, number models                   likely essential, explaining essential genes are <br /> have proposed explain link two.                         likely high cocomplex interaction degree. <br /> Jeong et al. (2001) noted strong correlation essen-                       More generally, observation entire complexes <br /> tiality protein interaction degree (the &#226;&#8364;&#339;centrality-lethality&#226;&#8364;?              appear essential, henceforth referred modular essential- <br /> rule). Subsequent analyses shown cor-                   ity, indicates phenotypes associated whole <br /> relation weak interaction networks map binary                    complexes restricted individual genes. Related <br /> transient interactions (e.g., yeast two-hybrid) (Yu et al. 2008;                work yeast shown members protein <br /> Zotenko et al. 2008), robust networks map stable                 complex respond coherent fashion drugs <br /> cocomplex interactions. However, result linking                    (Hillenmeyer et al. 2010), analyses humans <br /> essentiality interaction degree replicated                       suggested genes protein products belong <br /> number times, interpretation link sub-                  complex likely result same <br /> ject debate. It initially suggested proteins                   disease <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fraser Plotkin 2007; Lage et al. <br /> with high degree essential role main-                 2007; Oti Brunner 2007). <br /> taining overall connectivity network (Jeong et al.                       In work, seek address outstanding questions <br /> 2001), later certain interactions essential                    related modular essentiality: <br /> high-degree proteins simply likely disrupt essen- <br /> tial interaction (He Zhang 2006). Both interpretations                 First&#226;&#8364;&#8221;if modularity organizing principle explains essen- <br /> convincingly rejected Zotenko et al. (2008) instead                         tiality protein complexes, exceptions arise? that <br />                                                                                    is, nonessential subunits largely essential <br /> proposed certain proteins essential of <br />                                                                                    complexes essential subunits largely <br /> their involvement &#226;&#8364;&#339;Essential Complex Biological <br />                                                                                    nonessential complexes? We redundancy within <br /> Modules&#226;&#8364;?&#226;&#8364;&#8221;large groups essential proteins, den-                        complexes, revealed genetic interactions, explain <br /> sely connected protein interaction network                      cases, &#226;&#8364;&#339;moonlighting,&#226;&#8364;? as <br /> involved common biological function. Similarly,                        revealed membership multiple complexes, explain <br /> have shown essential genes concentrated specific                     latter. <br /> complexes protein complexes essen-                   Second&#226;&#8364;&#8221;is phenomenon modular essentiality con- <br /> tial nonessential, suggesting essentiality                     served species? There evidence number of <br />  <br />  <br />     A <br />                                                                                                C                           2.0 <br />                                                                                                                                                                  * <br />                                                                                                Log2(Observed / Expected) <br />  <br />  <br />  <br />  <br />                                                                                                                            1.5 <br />                                                                                                                            1.0    * <br />                                                                                                                            0.5 <br />                                                                                                                            0.0 <br />  <br />                                           Observed                                                                         -0.5 <br />     B <br />                                                                                                                            -1.0 <br />                                                                                                                            -1.5                   * <br />                                                                                                                                          * <br />                                                                                                                            -2.0 <br />                                                                                                                                   0-20 <br />  <br />  <br />                                                                                                                                          20-40 <br />  <br />  <br />                                                                                                                                                  40-60 <br />  <br />  <br />                                                                                                                                                          60-80 <br />  <br />  <br />                                                                                                                                                                  80-100 <br />  <br />  <br />  <br />  <br />                                                                                                                                   % Essential Genes Complex <br />                                           Expected <br />  <br />       FIG. 1.&#226;&#8364;&#8221;Comparing observed distribution essential genes Saccharomyces cerevisiae expected random. (A) Distribution essential <br /> genes complexes S. cerevisiae. For clarity, nonoverlapping complexes subunits shown. Essential genes shown red, <br /> nonessential blue. (B) An example randomly distributed essential genes set complexes shown (A). (C) Comparing observed <br /> distribution essential genes complexes seen 1,000 randomizations (see Materials Methods). The log 2 ratio observed expected <br /> is shown here; positive values indicate observed frequency random, negative values indicate frequency random. Stars indicate values <br /> that extreme seen randomizations. <br />  <br />  <br />  <br /> 1050 Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013 <br />  Protein Complexes Flip Essentiality Eukaryotes                                                              GBE <br />  <br />   model organisms members complex             &#226;&#8364;&#339;Synthetic Lethality&#226;&#8364;? included list genes <br />   likely share phenotype perturbed           identified nonessential data. <br />   (Lage et al. 2007; Lee et al. 2008); however, lack a <br />   comprehensive set gene deletions organism        Protein Domain Data <br />   S. cerevisiae far prevented global analysis. <br />   Using comprehensive essentiality data distantly         Protein domain annotations obtained precal- <br />   related fission yeast Schizosaccharomyces pombe (Kim             culated InterProScan results (domain.tab) hosted the <br />   et al. 2010), strong evidence modular essential-      Saccharomyces Genome Database (Cherry et al. 2012). <br />   ity, suggesting general feature eukary-      Annotations matching &#226;&#8364;&#339;HMMPfam&#226;&#8364;? extracted, resulting <br />   otes. Furthermore, complexes flip essentiality      31,924 protein pairs sharing Pfam domain. <br />   species cohesive fashion, is, they <br />   change essential nonessential vice      Drug Sensitivity Data <br />   versa flips associated larger scale <br />   biological differences yeasts.                   Drug sensitivity data obtained heterozygous <br />                                                                    fitness screen Hillenmeyer et al. (2008). Only drug&#226;&#8364;&#8220;gene <br />                                                                    interactions P &lt; 0.01 considered. <br /> Materials Methods <br />                                                                    Gene Ontology Data <br /> Essentiality Data <br />                                                                    Gene Ontology annotations obtained the <br /> Saccharomyces cerevisiae data obtained the <br />                                                                    Saccharomyces Genome Database (Cherry et al. 2012), and <br /> OGEE database (Chen et al. 2012), filtered include <br />                                                                    unique annotations &#226;&#8364;&#339;Biological Process&#226;&#8364;? ontol- <br /> the results Giaever et al. (2002). Schizosaccharomyces <br />                                                                    ogy counted. <br /> pombe essentiality data, orthology mapping between <br /> Sch. pombe S. cerevisiae, obtained Kim et al. <br />                                                                    Protein Complex Definitions <br /> (2010). Mouse essentiality data obtained the <br /> OGEE database (Chen et al. 2012).                                  Saccharomyces cerevisiae complex definitions obtained <br />                                                                    manually curated list 408 complexes supported by <br />                                                                    small-scale experiments literature (Pu et al. 2009). <br /> Gene Duplications <br />                                                                    This set covers 56% essential genes budding yeast <br /> A list duplicate gene pairs resulting whole-genome         (588/1,088). <br /> duplication obtained Byrne Wolfe (2005). This             Mouse complex definitions obtained the <br /> list extended include list small-scale duplications    CORUM database (Ruepp et al. 2010). <br /> identified using method Rost (1999). An initial list            Schizosaccharomyces pombe complexes identified <br /> of duplicates identified performing all      Gene Ontology annotations maintained at <br /> Basic Local Alignment Search Tool comparison (E &#194;&#188; 10). These       PomBase (Wood et al. 2012). Descendants term &#226;&#8364;&#339;mac- <br /> were subsequently filtered, <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> aligned    romolecular complex&#226;&#8364;? included term con- <br /> region (L)  80% length longer protein,       tained word &#226;&#8364;&#339;complex&#226;&#8364;? &#226;&#8364;&#339;subunit.&#226;&#8364;? This list was <br /> the identity aligned region greater equal    manually expanded include number complexes that <br /> a minimum identity threshold. For pairs aligned region     followed different naming conventions&#226;&#8364;&#8221;for example, <br /> greater 150 a.a., threshold 30%,            &#226;&#8364;&#339;signalosome&#226;&#8364;? &#226;&#8364;&#339;U1 SnRNP.&#226;&#8364;? Only associations annotated <br /> it equal n + 480 L 0.32(1 + exp( L/1,000)), n &#194;&#188; 6,    &#226;&#8364;&#339;IDA&#226;&#8364;? &#226;&#8364;&#339;IPI&#226;&#8364;? included&#226;&#8364;&#8221;indicating com- <br /> as Gu et al. (2002).                                           plex <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> supported direct experimental evidence in <br />                                                                    Sch. pombe just inferred sequence similarity. Finally, <br /> Genetic Interactions                                               list pruned remove complexes identical subunit <br /> The quantitative genetic interactions used analysis        composition different names, example, &#226;&#8364;&#339;RNA-directed <br /> were obtained recently collated interactome contain-        RNA polymerase complex&#226;&#8364;? &#226;&#8364;&#339;nuclear RNA-directed RNA <br /> ing majority quantitatively measured S. cerevisiae inter-   polymerase complex.&#226;&#8364;? The list 104 complexes available <br /> actions (Ryan et al. 2012). All interactions score     supplementary table S1, Supplementary Material online. <br /> than  2.3 considered strong negative genetic <br /> interaction. To better assess relationship homol-      Assessing Modular Distribution of <br /> ogy within-complex redundancy, complete list            Essential/Flipped Genes <br /> of genetic interactions (including low-throughput       Using methodology similar Hart et al. (2007), assessed <br /> and nonquantitative screens) obtained BioGRID         extent essentiality concentrated specific <br /> (Stark et al. 2011) version 3.1.8.8. Interactions annotated        complexes. The percentage essential genes complex <br /> &#226;&#8364;&#339;Negative Genetic,&#226;&#8364;? &#226;&#8364;&#339;Synthetic Growth Defect,&#226;&#8364;?                  calculated, sorted equal-sized bins, and <br />  <br />  <br /> Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013                             1051 <br />  Ryan et al.                                                                                                                                    GBE <br />  <br /> compared expected background generated ran-                                             If redundancy explains presence nonessential genes <br /> domly assigning essential genes set complexes.                                  essential protein complexes, essential complexes <br /> The methodology used assess distribution                                    enriched negative genetic interactions. Using a <br /> genes &#226;&#8364;&#339;flipped&#226;&#8364;? essentiality.                                                             set quantitatively measured genetic interactions, found <br />                                                                                                case 42% (22/53) measured <br /> Results                                                                                        interactions essential complexes ( 60% subunits essen- <br />                                                                                                tial) negative compared 5% (84/1,819) those <br /> We confirmed observation modular essentiality in <br />                                                                                                nonessential complexes (fig. 2B). Note would <br /> S. cerevisiae using date manually curated set of <br />                                                                                                expect 100% measured interactions between <br /> &#226;&#8364;&#339;gold-standard&quot; protein complexes (Pu et al. 2009) the <br />                                                                                                nonessential genes essential protein complexes neg- <br /> method Hart et al. (fig. 1, Materials Methods). <br />                                                                                                ative, gene need redundant other <br /> We found, previous studies, complexes mostly <br />                                                                                                complex subunit. Indeed, approximately 65% genes <br /> essential nonessential, or <br />                                                                                                covered data set feature negative genetic interaction <br /> nothing phenomenon. Indeed, approximately 26% <br /> (105/401) complexes consist solely essential solely                                   member complex. <br /> nonessential subunits. Despite success concept                                       One plausible explanation redundancy gene duplica- <br /> modular essentiality explaining behavior protein com-                                tion (Guan et al. 2007; Ihmels et al. 2007; Musso et al. 2007; <br /> plex subunits, extensive presence exceptions trou-                                   Dean et al. 2008; Szklarczyk et al. 2008; Li et al. 2010; <br /> bling. If protein complexes truly behave single <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>,                                VanderSluis et al. 2010). To assess extent gene <br /> their activity dependant coordinated function                                    duplication explain within-complex redundancy, <br /> subunits, subunits behave differently                                      assembled list paralogs small-scale and <br /> to others?                                                                                     whole-genome duplications (see Materials Methods). <br />                                                                                                Surprisingly, 22 negative genetic <br /> Redundancy Explains Many Nonessential Members                                           interactions essential protein complexes ex- <br /> of Essential Complexes                                                                         plained gene duplication. Indeed, examination previ- <br /> One possible explanation presence nonessential                                      ously reported negative genetic interactions (Stark et al. 2011) <br /> genes largely essential complexes redundancy&#226;&#8364;&#8221;the ability                                 (including low-throughput nonquantitative <br /> of gene functionally compensate loss                                 screens) suggests redundancy cocomplexed <br /> (fig. 2A). Such relationships ubiquitous biological                                     pairs rarely explained homology. Of 315 neg- <br /> systems (Nowak et al. 1997) identified high                                      ative genetic interactions reported occur members <br /> throughput yeast comparing growth mutants                                         protein complex, 67 explained by <br /> with gene deletions growth mutants                                          gene duplication. Of 67 interactions, 50 occur between <br /> each gene deleted individually (Baryshnikova et al. 2010;                                      members large small cytosolic ribosomal complexes, <br /> Collins et al. 2010). Negative genetic interactions,                                      suggesting that, notable exception ribosome, <br /> known synthetic sick synthetic lethal, identified                                    homology redundant cocomplexed pairs the <br /> when double mutant grows significantly worse                                          exception rule. <br /> expected indicate degree functional redundancy                                            Expanding analysis include proteins share a <br /> between deleted genes.                                                                 Pfam (Punta et al. 2012) domain (see Materials Methods), <br />  <br />                                          Complex <br />                             A                                        B <br />                                                                      interactions betweeen <br />  <br />  <br />  <br />  <br />                                                                                              50% <br />                                                                      nonessential subunits <br />  <br />  <br />  <br />  <br />                                                                                                         22/53 <br />                                                                         % Synthetic sick <br />  <br />  <br />  <br />  <br />                                                                                              40% <br />                                                                                              30% <br />                                                                                              20% <br />                                                                                              10%                         84/1819 <br />  <br />                                                                                              0% <br />                                                                                                    Essential complex   Nonessential <br />                                                                                                                          complex <br />  <br />       FIG. 2.&#226;&#8364;&#8221;Redundancy explains nonessential genes largely essential complexes. (A) Cartoon example redundancy protein complex. Red <br /> circles represent essential genes, blue circles represent nonessential genes, red edge indicates redundant relationship. (B) Interactions the <br /> nonessential members essential complexes likely synthetic sick interactions nonessential members nonessential complexes <br /> (P &#194;&#188; 1.9   10 15, sided Fisher&#226;&#8364;&#8482;s exact test). Complexes  60% essential genes considered essential complexes &lt;40% essential genes <br /> are considered nonessential. Counts bars indicate number synthetic sick interactions total number measured interactions. <br />  <br />  <br />  <br /> 1052 Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013 <br />  Protein Complexes Flip Essentiality Eukaryotes                                                                                                                               GBE <br />  <br /> rather strict paralogs, 89 315                                                 additional roles outside complex&#226;&#8364;&#8221;that is, they <br /> negative genetic interactions occurring members                                                   &#226;&#8364;&#339;moonlighting&#226;&#8364;? (Jeffery 2003). If hypothesis correct, <br /> the complex share protein domain. Again, majority                                                 reasoned protein products moonlighting <br /> (51) domain-sharing pairs ribosomal proteins,                                                   genes likely members multiple protein <br /> suggesting shared domains explain apparent                                                   complexes (fig. 3A). We tested hypothesis comparing <br /> redundancy complexes.                                                                                 essential genes nonessential complexes (&lt;40% subu- <br />                                                                                                              nits essential) nonessential genes complexes <br /> Moonlighting Proteins Explain Many Essential                                                          discovered essential genes significantly <br /> Genes Nonessential Complexes                                                                              likely members multiple complexes (fig. 3B). <br /> A possible explanation existence essential genes                                               These moonlighting proteins essential two <br /> otherwise nonessential complexes essential genes                                               reasons&#226;&#8364;&#8221;their participation essential complex in <br />  <br />  <br />  <br />    A                               Complex 1           B                                                                                                    E   Succinate dehydrogenase <br />                                                                                  50%          23/49 <br />                                                        % In Multiple Complexes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               SDH4 <br />  <br />  <br />                                                                                  40% <br />                                                                                                                                                                     SDH2               SDH1 <br />  <br />  <br />                                                                                  30% <br />                                                                                                                                                                               SDH3 <br />  <br />  <br />  <br />                                                                                  20%                                                      85/564                      TIM22          TIM12 <br />  <br />  <br />  <br />                                                                                  10% <br />                                                                                                                                                                      TIM54           TIM18 <br />  <br />  <br />                                    Complex 2                                     0% <br />                                                                                                                                                                      Tim22 Complex <br />                                                                                        Essential gene non- Nonessential gene non- <br />                                                                                          essential complex       essential complex <br />  <br />  <br />    C                                                                                                          D <br />                                   50%                                                                                                     50% <br />    % In Additional Nonessential <br />  <br />  <br />  <br />  <br />                                                                                                               % In Additional Essential <br />  <br />  <br />  <br />  <br />                                   40%                                                                                                     40% <br />                                                                                                                      Complex <br />  <br />  <br />  <br />  <br />                                                                                                                                                        14/49 <br />              Complex <br />  <br />  <br />  <br />  <br />                                   30%                                                                                                     30% <br />  <br />  <br />                                   20%          9/49                                                                                       20% <br />                                                                                        80/564 <br />                                   10%                                                                                                     10% <br />  <br />                                                                                                                                                                                  5/564 <br />                                   0%                                                                                                      0% <br />                                         Essential gene non- Nonessential gene                                                             Essential gene non- Nonessential gene in <br />                                           essential complex nonessential complex                                                                  essential complex nonessential complex <br />  <br />      FIG. 3.&#226;&#8364;&#8221;Moonlighting explains essential proteins nonessential complexes. (A) Cartoon example moonlighting protein present Complex 1 <br /> and Complex 2. Colors figure 2. (B) Essential genes nonessential complexes likely belong multiple complexes nonessential genes in <br /> nonessential complexes (P &#194;&#188; 6.3   10 7 two-sided Fisher&#226;&#8364;&#8482;s exact test). Complexes &lt;40% essential genes considered nonessential, cytosolic <br /> ribosomal subunits genes excluded analysis, behavior better explained gene duplication. Labels bars indicate actual gene <br /> counts. (C) Essential genes nonessential complexes likely belong additional nonessential complex nonessential genes in <br /> nonessential complexes (P &#194;&#188; 0.40 two-sided Fisher&#226;&#8364;&#8482;s exact test). Details (B). (D) Essential genes nonessential complexes likely belong <br /> to essential complex nonessential genes nonessential complexes (P &#194;&#188; 5.67   10 13 two-sided Fisher&#226;&#8364;&#8482;s exact test). Details (B). (E) SDH3, the <br /> only essential member succinate dehydrogenase complex, recently identified having additional role essential Tim22 <br /> complex. <br />  <br />  <br />  <br /> Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013                                                                                              1053 <br />  Ryan et al.                                                                                                            GBE <br />  <br /> addition role nonessential complex par-    S. pombe separated S. cerevisiae approximately 400 <br /> ticipation distinct nonessential complexes. To distin-       Myr evolution (Sipiczki 2000) eukary- <br /> guish cases, stratified              ote comprehensive essentiality data available <br /> groups asked essential genes nonessential            (Kim et al. 2010). These species differ greatly their <br /> complexes likely member additional           basic biology, Sch. pombe appearing similar to <br /> nonessential complex nonessential genes            metazoan cells respects&#226;&#8364;&#8221;including aspects of <br /> complexes. We slight statistically significant      mRNA splicing (due extensive presence introns in <br /> difference groups (fig. 3C, P &#194;&#188; 0.40, Fisher&#226;&#8364;&#8482;s      Sch. pombe), gene expression controlled RNA <br /> exact test), suggesting simply membership         interference machinery (absent S. cerevisiae), chromosome <br /> multiple complexes renders protein essential. When           structure epigenetic mechanisms, cell-cycle regula- <br /> restricting analysis membership additional essen-      tion G2/M transition control (Wood 2006; Rhind et al. <br /> tial complex, significant difference     2011). Furthermore, divergence, S. cerevisiae has <br /> two groups&#226;&#8364;&#8221;with essential genes approximately 30              undergone whole-genome duplication (Wolfe Shields <br /> times likely belong additional essential complex      1997). Consequently, features conserved species are <br /> than nonessential genes (fig. 3D, P &#194;&#188; 5.7   10 13).                 likely broadly conserved eukaryotes. <br />     This phenomenon widespread explained             We assembled list 104 complexes experimental <br /> small number essential moonlighting proteins present          support Sch. pombe (see Materials Methods) including <br /> many nonessential complexes: membership multiple com-            complexes absent S. cerevisiae. By performing <br /> plexes accounts approximately 50% essential genes        analysis described figure 1, Sch. <br /> in nonessential complexes (excluding cytosolic ribosome).       pombe complexes disproportionately essential or <br /> Nevertheless, reasonable ask accounts rest.   nonessential (fig. 4A), confirming modular essen- <br /> It possible genes actually members       tiality conserved species. Interestingly, the <br /> complexes, associations experimentally        complexes absent S. cerevisiae completely <br /> determined documented databases. An example        nonessential. <br /> SDH3, set complex definitions anno-             It possible modular essentiality feature conserved <br /> tated member succinate dehydrogenase complex            single-celled fungi absent metazoans in- <br /> (Oyedotun Lemire 2004). SDH3 essential              creased complexity requirements multiple cell types. <br /> member complex, suggesting additional        However, analysis essentiality incomplete set of <br /> role outside complex, indeed, recently          <span id='am-399' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-400' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-476' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>knock-out</span> mice suggests distribution essential <br /> identified member essential TIM22 mitochondrial         genes complexes follows similar, albeit weaker, <br /> inner membrane protein insertion complex (Gebert et al.             <span id='am-556' about='xsp:pattern' typeof='owl:Thing'>pattern</span> species (supplementary fig. S1, Supplementary <br /> 2011) (fig. 3E).                                                    Material online). As coverage <span id='am-477' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-478' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-479' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-480' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-481' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-486' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-493' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-509' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-518' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-519' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-526' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-534' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-535' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-542' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-551' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-552' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-553' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-554' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> collections in <br />     If undocumented membership complex             metazoans increases, able assess conserva- <br /> common explanation essential genes annotated belong-         tion trend better. <br /> ing single nonessential complex, would <br /> expect genes display phenotypes per- <br /> turbed nonessential genes complex           Protein Complexes &#226;&#8364;&#339;Flip&quot; Essentiality the <br /> their additional functions. We tested using data        Two Species <br /> large-scale haploinsufficiency screen (Hillenmeyer et al. 2008),    Eighty-three percent one-to-one orthologs between <br /> finding display drug sensitivities nones-       Sch. pombe S. cerevisiae conserved essentiality, <br /> sential genes nonessential complexes (median 33 drug          is, nonessential species essential in <br /> interactions vs. 27). Furthermore, essential genes     species. The remaining 17% genes nonessential in <br /> in nonessential complexes greater number biological       species essential other. Interestingly, Kim et al. <br /> process annotations Gene Ontology (median 9 vs. 6)        (2010) genes change essentiality enriched <br /> (Ashburner et al. 2000). Both observations support         certain functional categories. This enrichment be <br /> the hypothesis essential genes additional func-     statistically significant 50% subunits large <br /> tions outside nonessential complex.                             complex changed essentiality, behavior be <br />                                                                     consistent idea modular essentiality. Instead, if <br />                                                                     essentiality truly modular behavior, that <br /> Schizosaccharomyces pombe Complexes Display                         complexes &#226;&#8364;&#339;flip&quot; essentiality single <span id='am-557' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. By analyzing <br /> Modular Essentiality                                                distribution flipped genes complexes com- <br /> Having confirmed phenomenon modular                          paring expected random, com- <br /> essentiality S. cerevisiae, sought test       plexes behave modular fashion, or <br /> this observation conserved species. The fission yeast     subunits changing essentiality (fig. 4B). <br />  <br />  <br /> 1054 Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013 <br />  Protein Complexes Flip Essentiality Eukaryotes                                                                                                                  GBE <br />  <br />     A                           2.0                                                          B                           2.0 <br />                                                                                                                                                                             * <br />     Log2(Observed / Expected) <br />  <br />  <br />  <br />  <br />                                                                                              Log2(Observed / Expected) <br />                                 1.5                                                                                      1.5 <br />                                          *                                     * <br />                                 1.0                                                                                      1.0 <br />  <br />                                 0.5                                                                                      0.5       * <br />                                 0.0                                                                                      0.0 <br />  <br />                                 -0.5                                                                                     -0.5 <br />  <br />                                 -1.0                                                                                     -1.0                         * <br />                                                               *                                                                              * <br />                                 -1.5                                                                                     -1.5 <br />                                          0-20 <br />  <br />  <br />  <br />                                                     20-40 <br />  <br />  <br />  <br />                                                               40-60 <br />  <br />  <br />  <br />                                                                       60-80 <br />  <br />  <br />  <br />                                                                                80-100 <br />  <br />  <br />  <br />  <br />                                                                                                                                    0-20 <br />  <br />  <br />  <br />                                                                                                                                             20-40 <br />  <br />  <br />  <br />                                                                                                                                                       40-60 <br />  <br />  <br />  <br />                                                                                                                                                                60-80 <br />  <br />  <br />  <br />                                                                                                                                                                             80-100 <br />                                                 % Essential Genes Complex                                                     % Genes Flipped Essentiality Complex <br />  <br />  <br />     C <br />                                         MPRL6 YML6                                      MPRL6 YML6                                                   MPRL6 YML6 <br />  <br />                                                         MRPL16                                                            MRPL16                                       MRPL16 <br />  <br />                                                             MRPL23                                                              MRPL23                                     MRPL23 <br />  <br />  <br />                                                             MRPL10                                                              MRPL10                                    MRPL10 <br />  <br />  <br />  <br />  <br />                                        S. cerevisiae                                    S. pombe                                                      C. albicans <br />       FIG. 4.&#226;&#8364;&#8221;Schizosaccharomyces pombe complexes exhibit modular essentiality, complexes flip essentiality yeast species. <br /> (A) Experimentally characterized S. pombe complexes exhibit modular essentiality. The observed distribution essential genes compared that <br /> expected random, figure 1C (See Materials Methods). (B) Protein complexes flip essentiality S. pombe S. cerevisiae. The observed <br /> distribution &#226;&#8364;&#339;flipped&#226;&#8364;? genes compared expected random, figure 1C. Only genes present single copy species used for <br /> this analysis. The complex definitions used S. cerevisiae, greater coverage S. pombe. (C) The mitochondrial <br /> ribosomal large subunit. This complex essential S. pombe Candida albicans nonessential S. cerevisiae. White circles correspond genes whose <br /> essentiality status experimentally determined. Genes orthologs unknown essentiality status C. albicans unlabeled shown <br /> as smaller circles. <br />  <br />  <br />  <br />  <br />    We 14 complexes, 80%                                              (table 1). For example, significant difference was <br /> subunits flip essentiality coherent fashion                                               seen mitochondrial translation machinery. <br /> two species. A complete list complexes given                                                Saccharomyces cerevisiae cells survive mtDNA <br /> table 1. These complexes range size large mito-                                                &#226;&#8364;&#339;petite&quot; mutants, contrast Sch. pombe (&#226;&#8364;&#339;petite neg- <br /> chondrial ribosomal subunit, features 36 genes flip,                                            ative&quot;) (Jiang et al. 2011). If difference lifestyle causes <br /> to smaller complexes SBF transcription complex,                                                mitochondrial translation machinery flip essentiality be- <br /> whose subunits flip. Furthermore, note                                                    tween species, expect machinery be <br /> complexes (64%) change essentiality                                                      essential petite negative species. Candida albicans is <br /> between species&#226;&#8364;&#8221;for example, DNA-directed RNA poly-                                                    species&#226;&#8364;&#8221;Phylogenetically closer S. cerevisiae <br /> merase III complex, 17 subunits essential                                                Sch. pombe, displays petite negative phenotype. <br /> species, Elongator complex, subunits                                                 Comprehensive essentiality data available for <br /> nonessential species.                                                                              C. albicans; however, number pilot studies (Becker et al. <br />    Complexes function processes organelles                                                 2010; Noble et al. 2010) established status few <br /> have well-documented differences yeasts                                                    genes S. cerevisiae orthologs, allowing to <br /> account half complexes flip essentiality                                                   test hypothesis. Consistent idea changes <br />  <br />  <br /> Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013                                                                                     1055 <br />  Ryan et al.                                                                                                                 GBE <br />  <br /> Table 1 <br /> Protein Complexes That Flip Essentiality Schizosaccharomyces pombe Saccharomyces cerevisiae <br /> Complex Name                              Fraction     No. Essential     No. Essential    Complex        Associated Lifestyle Difference <br />                                           Flipped      S. pombe        S. cerevisiae        Size <br /> Mitochondrial ribosomal small subunit     0.83         23                3                   24           Mitochondrial translation <br /> Mitochondrial ribosomal large subunit     0.85         36                2                   40           Mitochondrial translation <br /> Clathrin                                  1.00          2                0                    2           Golgi associated <br /> GARP complex                              1.00          4                0                    4           Golgi associated <br /> H+-transporting ATPase, Golgi             1.00         11                0                   11           Golgi associated <br /> Mdm12p/Mmm1p/Mdm10p complex               1.00          3                0                    3           Mitochondrial translation (absence <br />                                                                                                             results loss mtDNA) <br /> DASH complex                              1.00          0                9                    9           Spindle/Kinetochore <br /> Gcd10p/Gcd14p complex                     1.00          0                2                    2 <br /> Mtr2p/Mex67p complex                      1.00          0                2                    2 <br /> TORC 2 complex                            1.00          0                3                    3 <br /> Uba2p/Aos1p complex                       1.00          0                2                    2 <br /> Anthranilate synthase complex             1.00          2                0                    2 <br /> SBF complex                               1.00          2                0                    2 <br /> Ula1p/Uba3p complex                       1.00          2                0                    2 <br />  <br />  <br />  <br />  <br /> in lifestyle associated flips essentiality,      Understanding Exceptions Greater Detail <br /> all members large mitochondrial ribosomal subunit              We essential genes nonessential com- <br /> in C. albicans experimentally tested essen-            plexes tend second complex. This sharing of <br /> tial (fig. 4C).                                                           subunits come reasons&#226;&#8364;&#8221;either pro- <br />                                                                           tein question unique structural feature (e.g., a <br /> Discussion                                                                catalytic activity structural recognition domain) is <br />                                                                           required complexes (&#226;&#8364;&#339;moonlighting&quot;) could <br /> In work, shown modular essentiality an <br />                                                                           distinct structural features required complex. <br /> organizing principle conserved distantly re- <br />                                                                           Distinguishing cases require integra- <br /> lated species, suggesting general feature of <br />                                                                           tion structural models (Kim et al. 2006) genetic <br /> eukaryotic systems. Furthermore, provided explana- <br />                                                                           perturbation individual interacting domains (Dreze et al. <br /> tions exceptions principle. Finally, shown <br />                                                                           2009; Ear Michnick 2009; Zhong et al. 2009). <br /> that protein complexes flip essentiality coherent <span id='am-558' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> be- <br />                                                                              Furthermore, nonessential genes other- <br /> tween species. Overall, results reinforce concept <br />                                                                           wise essential complexes frequently exhibit negative genetic <br /> that protein complexes fundamental functional unit of <br />                                                                           interactions member complex. <br /> cell biology phenotypes associated whole <br />                                                                           Previous work complexes enriched neg- <br /> protein complexes individual genes. <br />                                                                           ative genetic interactions tend contain essential <br />                                                                           gene (Bandyopadhyay et al. 2008; Wilmes et al. 2008; <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> Essentiality Other Phenotypes                                     Baryshnikova et al. 2010); however, analyses included <br /> Modular essentiality consistent tendency                   genetic interactions hypomorphic alleles essential <br /> members protein complexes display coherent &#226;&#8364;&#339;mono-                   genes, making interpretation results difficult. <br /> chromatic&quot; genetic interactions (Segre` et al. 2005;                      Here, asking inverse question (&#226;&#8364;&#339;do nonessential <br /> Bandyopadhyay et al. 2008; Michaut et al. 2011), display               genes essential complexes display negative genetic interac- <br /> sensitivity drugs (Fraser Plotkin 2007;                   tions?&#226;&#8364;?) analyzing genetic interactions between <br /> Hillenmeyer et al. 2010; Kapitzky et al. 2010),                 nonessential genes, support idea these <br /> associated disease (Fraser Plotkin 2007;                nonessential subunits redundant. <br /> Lage et al. 2007; Oti Brunner 2007). Our explanations                    We observed approximately 5% genetic <br /> for exceptions modular essentiality applied                  interactions nonessential protein complexes negative. <br /> these contexts well&#226;&#8364;&#8221;providing explanations                     However, complexes essential presence <br /> perturbation member complex cause dif-                  drug stress, negative genetic interactions <br /> ferent disease exhibit different drug sensitivity             observed specific conditions. Indeed, has <br /> members complex                                               noted certain genetic interactions (termed <br />  <br />  <br /> 1056 Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013 <br />  Protein Complexes Flip Essentiality Eukaryotes                                                                        GBE <br />  <br /> conditional genetic interactions) observed specific     Ryan et al. 2012), drug&#226;&#8364;&#8220;gene interactions (Kapitzky et al. <br /> conditions (St Onge et al. 2007; Bandyopadhyay et al. 2010;         2010), transcription factor binding (Tan et al. 2007). In <br /> Gue&#194;&#180;nole&#194;&#180; et al. 2013). Consequently, note likely   work, protein complexes &#226;&#8364;&#339;flip&quot; essentiality <br /> a lower bound estimate frequency redun-            coherent unit species. As gene deletion collec- <br /> dancy nonessential complexes.                                tions available additional species, especially those <br />                                                                     closely related Sch. pombe S. cerevisiae, we <br /> Homology Redundant Proteins Same                     able compare relative contributions lifestyle <br /> Complex Exception Rather Rule                       changes factors. It possible define a <br /> Several groups analyzed retention duplicate genes       core set complexes essential eukaryotes under <br /> in protein complexes roles duplicates play            conditions, example, involved core processes <br /> (Musso et al. 2007; Pereira-Leal et al. 2007; Szklarczyk et al.     transcription, set &#226;&#8364;&#339;conditionally essential&quot; <br /> 2008). Others analyzed extent duplicate           complexes associated specific environments <br /> gene pairs display functional redundancy (Guan et al. 2007;         morphology. <br /> Ihmels et al. 2007; Dean et al. 2008; Li et al. 2010; van <br /> Wageningen et al. 2010; VanderSluis et al. 2010) noted          Supplementary Material <br /> that duplicates coding proteins complex         Supplementary table S1 figure S1 available at <br /> frequently redundant (Li et al. 2010). Here, du-       Genome Biology Evolution online (http://www.gbe. <br /> plicate genes responsible small fraction      oxfordjournals.org/). <br /> redundancy complexes. This suggests although <br /> duplicate genes complex frequently <br />                                                                     Acknowledgments <br /> redundant, majority redundancy complexes <br /> cannot explained duplication. The mechanism                The authors thank A. Roguev, D. Greene, D.C. Shields, and <br /> which nonhomologous genes perform redundant roles             P. Beltrao helpful discussion. This work supported by <br /> is unclear; however, recently shown re-          Irish Research Council Science Engineering <br /> dundant relationships complexes highly conserved         Technology C.J.R., NIH (GM084448, GM084279, <br /> across species (Ryan et al. 2012) suggesting merit        GM081879, GM098101) N.J.K., Science <br /> further investigation.                                              Foundation Ireland (08/SRC/I1407) P.C. G.C. <br />  <br /> Function, Not Topology, Primary Determinant               Literature Cited <br /> Essentiality                                                        Alberts B. 1998. The cell collection protein machines: preparing the <br /> There strong evidence protein&#226;&#8364;&#8220;protein interactions, es-         generation molecular biologists. Cell 92:291&#226;&#8364;&#8220;294. <br />                                                                     Ashburner M, et al. 2000. Gene ontology: tool unification of <br /> pecially members complex, are <br />                                                                         biology. The Gene Ontology Consortium. Nat Genet. 25:25&#226;&#8364;&#8220;29. <br /> highly conserved species (Shevchenko et al. 2008; van        Bandyopadhyay S, et al. 2010. Rewiring genetic networks response to <br /> Dam Snel 2008; Leducq et al. 2012). That protein&#226;&#8364;&#8482;s                DNA damage. Science 330:1385&#226;&#8364;&#8220;1389. <br /> interaction partners remain relatively constant       Bandyopadhyay S, Kelley R, Krogan NJ, Ideker T. 2008. Functional maps of <br /> requirement protein survival change suggests            protein complexes quantitative genetic interaction data. PLoS <br />                                                                         Comput Biol. 4:e1000065. <br /> that topological features primary determining <br />                                                                     Baryshnikova A, et al. 2010. Quantitative analysis fitness genetic <br /> factor essentiality. Indeed, observation            interactions yeast genome scale. Nat Methods. 7:1017&#226;&#8364;&#8220;1024. <br /> changes essentiality species fall specific func-    Becker JM, et al. 2010. Pathway analysis Candida albicans survival and <br /> tional categories (mitochondrial translation, kinetochore,              virulence determinants murine infection model. Proc Natl Acad Sci <br /> Golgi associated) lends strong support proposal          U S A. 107:22044&#226;&#8364;&#8220;22049. <br />                                                                     Beltrao P, et al. 2009. Evolution phosphoregulation: comparison of <br /> is involvement specific biological processes renders a <br />                                                                         phosphorylation patterns yeast species. PLoS Biol. 7:e1000134. <br /> protein essential, network connectivity (Zotenko        Byrne KP, Wolfe KH. 2005. The Yeast Gene Order Browser: combining <br /> et al. 2008).                                                           curated homology syntenic context reveals gene fate polyploid <br />                                                                         species. Genome Res. 15:1456&#226;&#8364;&#8220;1461. <br /> Using Protein Complexes Understand Differences                   Chen W-HH, Minguez P, Lercher MJ, Bork P. 2012. OGEE: online gene <br />                                                                         essentiality database. Nucleic Acids Res. 40:D901&#226;&#8364;&#8220;D906. <br /> between Species <br />                                                                     Cherry JM, et al. 2012. Saccharomyces Genome Database: genomics <br /> We interested essentiality protein com-              resource budding yeast. Nucleic Acids Res. 40:D700&#226;&#8364;&#8220;D705. <br /> plexes differs species. Previous work highlighted       Collins SR, Roguev A, Krogan NJ. 2010. Quantitative genetic interaction <br />                                                                         mapping using E-MAP approach. Methods Enzymol. 470: <br /> the utility protein complexes abstraction under- <br />                                                                         205&#226;&#8364;&#8220;231. <br /> stand evolution cell-cycle regulation (Jensen et al.         Dean EJ, Davis JC, Davis RW, Petrov DA. 2008. Pervasive persistent <br /> 2006), phosphorylation (Beltrao et al. 2009), genetic interac-          redundancy duplicated genes yeast. PLoS Genet. 4: <br /> tions (Roguev et al. 2008; Frost et al. 2012; Koch et al. 2012;         e1000113. <br />  <br />  <br />  <br /> Genome Biol. Evol. 5(6):1049&#226;&#8364;&#8220;1059. doi:10.1093/gbe/evt074 Advance Access publication May 8, 2013                                        1057 <br />  Ryan et al.                                                                                                                               GBE <br />  <br /> Dezso Z, Oltvai ZN, Baraba&#194;&#180;si A-L. 2003. Bioinformatics analysis experi-     Lage K, et al. 2007. A human phenome-interactome network pro- <br />      mentally determined protein complexes yeast Saccharomyces                 tein complexes implicated genetic disorders. Nat Biotechnol. 25: <br />      cerevisiae. Genome Res. 13:2450&#226;&#8364;&#8220;2454.                                            309&#226;&#8364;&#8220;316. <br /> Dreze M, et al. 2009. &#226;&#8364;&#339;Edgetic&#226;&#8364;? perturbation C. elegans BCL2               Leducq J-B, et al. 2012. Evidence robustness protein complexes <br />      ortholog. Nat Methods. 6:843&#226;&#8364;&#8220;849.                                                inter-species hybridization. PLoS Genet. 8:e1003161. <br /> Ear PH, Michnick SW. 2009. A general life-death selection strategy          Lee I, et al. 2008. A single gene network accurately predicts phenotypic <br />      dissecting protein functions. Nat Methods. 6:813&#226;&#8364;&#8220;816.                            effects gene perturbation Caenorhabditis elegans. Nat Genet. 40: <br /> Fraser HB, Plotkin JB. 2007. Using protein complexes predict phenotypic            181&#226;&#8364;&#8220;188. <br />      effects <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>gene mutation</span>. Genome Biol. 8:R252.                             Li J, Yuan Z, Zhang Z. 2010. The cellular robustness genetic redundancy <br /> Frost A, et al. 2012. Functional repurposing revealed comparing                    budding yeast. PLoS Genet. 6:e1001187. <br />      S. pombe S. cerevisiae genetic interactions. Cell 149:1339&#226;&#8364;&#8220;1352.       Michaut M, et al. 2011. Protein complexes central yeast genetic <br /> Gavin A-C, et al. 2006. Proteome survey reveals modularity yeast cell          landscape. PLoS Comput Biol. 7:e1001092. <br />      machinery. Nature 440:631&#226;&#8364;&#8220;636.                                             Musso G, Zhang Z, Emili A. 2007. Retention protein complex member- <br /> Gebert N, et al. 2011. Dual function Sdh3 respiratory chain                 ship ancient duplicated gene products budding yeast. Trends <br />      TIM22 protein translocase mitochondrial inner membrane.               Genet. 23:266&#226;&#8364;&#8220;269. <br />      Mol Cell. 44:811&#226;&#8364;&#8220;818.                                                      Newman M. 2006. Modularity community structure networks. Proc <br /> Giaever G, et al. 2002. Functional profiling Saccharomyces cerevisiae          Natl Acad Sci U S A. 103:8577&#226;&#8364;&#8220;8582. <br />      genome. Nature 418:387&#226;&#8364;&#8220;391.                                                Noble SM, French S, Kohn LA, Chen V, Johnson AD. 2010. Systematic <br /> Gu Z, Cavalcanti A, Chen F-C, Bouman P, Li W-H. 2002. Extent gene                  screens Candida albicans homozygous deletion library decouple <br />      duplication genomes Drosophila, nematode, yeast. Mol               morphogenetic switching pathogenicity. Nat Genet. 42:590&#226;&#8364;&#8220;598. <br />      Biol Evol. 19:256&#226;&#8364;&#8220;262.                                                     Nowak MA, Boerlijst MC, Cooke J, Smith JM. 1997. Evolution genetic <br /> Guan Y, Dunham MJ, Troyanskaya OG. 2007. Functional Analysis                       redundancy. 388:167&#226;&#8364;&#8220;171. <br />      Gene Duplications Saccharomyces cerevisiae. Genetics 175:               Oti M, Brunner HG. 2007. The modular nature genetic diseases. Clin <br />      933&#226;&#8364;&#8220;943.                                                                         Genet. 71:1&#226;&#8364;&#8220;11. <br /> Gue&#194;&#180;nole&#194;&#180; A, et al. 2013. Dissection DNA damage responses using              Oyedotun KS, Lemire BD. 2004. The quaternary structure the <br />      multiconditional genetic interaction maps. Mol Cell. 49:346&#226;&#8364;&#8220;358.                 Saccharomyces cerevisiae succinate dehydrogenase. Homology <br /> Hart TT, Lee I, Marcotte ER. 2007. A high-accuracy consensus map yeast             modeling, cofactor docking, molecular dynamics simulation <br />      protein complexes reveals modular nature gene essentiality. BMC               studies. J Biol Chem. 279:9424&#226;&#8364;&#8220;9431. <br />      Bioinformatics 8:236.                                                      Pereira-Leal JB, Levy E, Kamp C, Teichmann S. 2007. Evolution protein <br /> Hartwell LH, Hopfield JJ, Leibler S, Murray AW. 1999. From molecular               complexes duplication homomeric interactions. Genome Biol. 8: <br />      modular cell biology. Nature 402:C47&#226;&#8364;&#8220;C52.                                        R51. <br /> He X, Zhang J. 2006. Why hubs tend essential protein                Pereira-Leal JB, Levy ED, Teichmann SA. 2006. The origins evolution of <br />      networks? PLoS Genet. 2:e88.                                                     functional modules: lessons protein complexes. Philos Trans R <br /> Hillenmeyer ME, et al. 2008. The chemical genomic portrait yeast:                  Soc Lond B Biol Sci. 361:507&#226;&#8364;&#8220;517. <br />      uncovering phenotype genes. Science 320:362&#226;&#8364;&#8220;365.                 Pu S, Wong J, Turner B, Cho E, Wodak SJ. 2009. Up-to-date catalogues of <br /> Hillenmeyer ME, et al. 2010. Systematic analysis genome-wide fitness               yeast protein complexes. Nucleic Acids Res. 37:825&#226;&#8364;&#8220;831. <br />      data yeast reveals novel gene function drug action. Genome          Punta M, et al. 2012. The Pfam protein families database. Nucleic Acids <br />      Biol. 11:R30.                                                                    Res. 40:D290&#226;&#8364;&#8220;D301. <br /> Ihmels J, Collins S, Schuldiner M, Krogan N, Weissman J. 2007. Backup           Rhind N, et al. 2011. Comparative functional genomics fission <br />      redundancy: genetic interactions reveal cost duplicate            yeasts. Science 332:930&#226;&#8364;&#8220;936. <br />      gene loss. Mol Syst Biol. 3:86.                                            Roguev A, et al. 2008. Conservation rewiring functional mod- <br /> Jeffery CJ. 2003. Moonlighting proteins: old proteins learning new tricks.            ules revealed epistasis map fission yeast. Science 322: <br />      Trends Genet. 19:415&#226;&#8364;&#8220;417.                                                        405&#226;&#8364;&#8220;410. <br /> Jensen LJJ, Jensen TSS, De Lichtenberg U, Brunak S, Bork P. 2006.               Rost B. 1999. Twilight zone protein sequence alignments. Protein Eng. <br />      Co-evolution transcriptional post-translational cell-cycle                12:85&#226;&#8364;&#8220;94. <br />      regulation. Nature 443:594&#226;&#8364;&#8220;597.                                            Ruepp A, et al. 2010. CORUM: comprehensive resource mammalian <br /> Jeong H, Mason SP, Baraba&#194;&#180;si AL, Oltvai ZN. 2001. Lethality centrality         protein complexes&#226;&#8364;&#8221;2009. Nucleic Acids Res. 38:D497&#226;&#8364;&#8220;D501. <br />      protein networks. Nature 411:41&#226;&#8364;&#8220;42.                                        Ryan CJ, et al. 2012. Hierarchical modularity evolution genetic <br /> Jiang H, et al. 2011. Identification characterization mitochon-            interactomes species. Mol Cell. 46:691&#226;&#8364;&#8220;704. <br />      drial RNA polymerase transcription factor fission yeast         Segre` D, DeLuna A, Church GM, Kishony R. 2005. Modular epistasis in <br />      Schizosaccharomyces pombe. Nucleic Acids Res. 39:5119&#226;&#8364;&#8220;5130.                      yeast metabolism. Nat Genet. 37:77&#226;&#8364;&#8220;83. <br /> Kapitzky L, et al. 2010. Cross-species chemogenomic profiling                   Semple JI, Vavouri T, Lehner B. 2008. A simple principle concerning the <br />      reveals evolutionarily conserved drug mode action. Mol Syst                   robustness protein complex activity changes gene expression. <br />      Biol. 6:451.                                                                     BMC Syst Biol. 2:1. <br /> Kim D-U, et al. 2010. Analysis genome-wide set gene deletions           Shevchenko A, et al. 2008. Chromatin central: comparative <br />      fission yeast Schizosaccharomyces pombe. Nat Biotechnol. 28:              proteome accurate mapping yeast proteomic environment. <br />      617&#226;&#8364;&#8220;623.                                                                         Genome Biol. 9:R167. <br /> Kim PM, Lu LJ, Xia Y, Gerstein MB. 2006. Relating three-dimensional             Sipiczki M. 2000. Where does fission yeast sit tree life? Genome <br />       <br /> </body></html>