<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Methods. 2013 July ; 10(7): . doi:10.1038/nmeth.2475. <br />  <br />  <br />  <br />                            The C. elegans Lifespan Machine <br />                            Nicholas Stroustrup1, Bryne E. Ulmschneider, Zachary M. Nash, Isaac F. L&#195;&#179;pez Moyado, <br />                            Javier Apfeld1,*, Walter Fontana1,* <br />                            Department Systems Biology, Harvard Medical School, 200 Longwood Avenue, Boston MA <br />                            02115, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  The measurement lifespan pervades aging research. Because lifespan results complex <br />                                  interactions genetic, environmental stochastic factors, varies widely among <br />                                  isogenic individuals. The action molecular mechanisms lifespan visible only <br />                                  statistical effects populations. <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>Survival</span> assays C. elegans provided critical <br />                                  insights evolutionarily conserved determinants aging. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> enable rapid acquisition of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  survival curves arbitrary statistical resolution, developed scalable imaging analysis <br />                                  platform observe nematodes multiple weeks square meters agar surface 8 &#206;&#188;m <br />                                  resolution. The method generates permanent visual record individual deaths which <br />                                  survival curves constructed validated, producing data consistent manual method <br />                                  mutants standard stressful environments. Our approach allows rapid, <br />                                  detailed reverse-genetic chemical screens effects survival enables quantitative <br />                                  investigations statistical structure aging. <br />  <br />  <br />                            Introduction <br />                                              Aging organisms exhibit functional declines levels biological organization. <br />                                              These declines ultimately conspire kill organism determine lifespan. Because <br />                                              lifespans vary remarkably1 isogenic populations, useful information aging <br />                                              mechanisms obtained identifying statistical signature response of <br />                                              lifespan distributions genetic, chemical physical interventions <br />  <br />                                              C. elegans individuals live self-fertile adults weeks, producing large numbers of <br />                                              isogenic offspring. Early investigations revealed single point mutations insulin/IGF-1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              pathway components capable doubling average lifespan2-5. These findings subsequently <br />                                              generalized organisms, including fruit flies mice5, establishing C. elegans a <br />  <br />  <br />  <br />                            1 <br />                             These authors corresponding authors: stroustr@fas.harvard.edu &#226;&#8364;&#8221; javier_apfeld@hms.harvard.edu &#226;&#8364;&#8221; walter@hms.harvard.edu. <br />                            *These authors contributed equally considered co-last authors. <br />                            Author Contributions: N.S. designed implemented hardware software. N.S. B.U. constructed calibrated equipment. <br />                            N.S. J.A. conceived designed experiments. N.S., B.U., J.A., Z.N. I.F.L.M. performed experiments. N.S. designed analytic <br />                            tools. N.S., J.A. W.F. provided guidance, analyzed data, interpreted results, wrote manuscript. J.A W.F. co-last <br />                            authors. <br />                            Figure 5 Hazard animals thermal stress. (a) Hazard rates (dots) estimated Fig. 2d populations 1,873 <br />                            age-1(hx546), 2,106 daf-16(mu86) 1,726 wild type animals. (b) An acquired thermotolerance experiment shown hazard <br />                            perspective. Roughly 4,000 C. elegans grown UV-killed bacteria 20 &#194;&#176;C. On day adulthood, half animals <br />                            transferred 33 &#194;&#176;C hour allowed recover hours 20 &#194;&#176;C. Treated animals transferred a <br />                            second time 33 &#194;&#176;C survival determined LM. The hazard rate curve 2,022 pre-treated animals compared to <br />                            2,459 untreated animals. (c) The panel shows LM-acquired survival curves obtained populations 1,306 C. elegans (N2 <br />                            Bristol isolate), 255 C. elegans (Hawaiian isolate), 285 C. briggsae, 224 C. species 11, 295 C. brenneri nematodes raised live <br />                            OP50 20 &#194;&#176;C transferred 34.5 &#194;&#176;C second day adulthood. (d) Hazard rate plots Caenorhabditis species panel <br />                            (c). <br />                             Stroustrup et al.                                                                                              Page 2 <br />  <br />  <br />                                                metazoan model studying genetics aging. The acquisition survival curves C. <br />                                                elegans essential aging research. In routine practice, survival curves <br />                                                animals cultured solid agar Petri dish, fed bacterial lawn E. coli, are <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                acquired manually daily observation low-power dissecting microscope6, 7. <br />                                                Death recognized failure individual react motion prodding a <br />                                                wire, requiring labor-intensive, repetitive, subjective observation. This protocol provides <br />                                                strong incentives population sizes small, observations infrequent, number of <br />                                                replicates limited, curtailing reproducibility scope data collected. <br />  <br />                                                We present scalable method producing standardized survival curves arbitrary <br />                                                statistical resolution, allowing &#226;&#8364;&#339;hands-free&#226;&#8364;?, lifelong, accurate observation arbitrarily <br />                                                large nematode populations. Our method combines groups flatbed scanners function as <br />                                                single spatially-extended microscope. Alone groups, scanners produce time- <br />                                                lapse videos highly controlled environmental conditions. We couple imaging <br />                                                approach image-analysis pipeline automatically identifies death times. By <br />                                                creating auditable trail images, allows rapid validation refinement of <br />                                                processed data, supports statistically rigorous analysis aging stress- <br />                                                resistance. <br />  <br />                                                In cases, notably worms flies, focused efforts acquiring <br />                                                high-resolution survival data8-15. Our method formalizes, standardizes, routinizes such <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                observations nematodes, enabling accumulation self-consistent body of <br />                                                quantitative demographic data tightly linking aging, genotype, environment. No such <br />                                                comprehensive body data exists metazoan date, humans included. We refer to <br />                                                integrated combination scanner hardware image-processing software the <br />                                                &#226;&#8364;&#339;Lifespan Machine&#226;&#8364;? (LM). <br />  <br />                                Results <br />                                A distributed microscope large worm populations <br />                                                Lifespan data acquired scales, ranging aging studies spanning multiple <br />                                                months stress-resistance assays lasting day16, involving targeted <br />                                                characterizations strains12 screening projects thousands gene knock- <br />                                                downs2, 3, 5. Throughput important screening projects, targeted studies benefit <br />                                                large populations provide statistical power. To suit purposes, platform <br />                                                combines flatbed scanners scalable fashion monitor arbitrarily large populations over <br />                                                wide-range timescales. <br />  <br />                                                Our approach replaces human observation automated acquires and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                processes time-lapse images identify timing death defined persistent <br />                                                cessation spontaneous movement. Our current reference implementation can <br />                                                simultaneously monitor 30,000 animals 800 plates, imaging 4.5 m2 agar surface at <br />                                                resolution 8 &#206;&#188;m (Supplementary Notes 1&#226;&#8364;&#8221;4). This capacity used run many <br />                                                independent, single-scanner experiments parallel larger investigations involving ten <br />                                                scanners more. <br />  <br />                                                Young worms quickly relative observation frequency attainable scanners, <br />                                                precluding tracking. However, animals slow age, motion be <br />                                                quantified longitudinally, especially limited changes posture fixed <br />                                                location. <br />  <br />                                                Each scanner monitors 16 Petri dishes (plates) sealed face glass sheet rubber <br />                                                mat, plate containing population 35 animals (Online Methods). A <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                            Page 3 <br />  <br />  <br />                                                fluorescent lamp inside scanner lid sends light plates, agar, bacteria, worms, <br />                                                captured sensor chip moving underneath glass surface synchrony the <br />                                                light source (Fig. 1a, Supplementary Notes 1 3). A scanner acts 16 camera- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                equipped dissecting microscopes, costing significantly form factor suited <br />                                                dense shelving inside temperature-controlled incubators. <br />  <br />                                                Traditional protocols monitoring large populations nematodes long periods of <br />                                                time plates incubator6, 7, 11, 12, exposing spatially and <br />                                                temporally heterogeneous conditions. Our apparatus maintains plates fixed position <br />                                                controlled environment duration experiment. This important for <br />                                                accuracy, given exquisite temperature sensitivity C. elegans lifespan17 <br />                                                (Supplementary Note 1). We curbed temperature excursions systems fans, one <br />                                                cooling scanner, circulating air evenly enclosing incubator <br />                                                (Supplementary Note 3). Temperature fluctuations typical scanner surface thus <br />                                                reduced degrees 0.5 &#194;&#176;C, limiting differences between <br />                                                scanners 1 &#194;&#176;C (Supplementary Note 1). <br />  <br />                                                To improve image quality repositioned single fixed lens scanner&apos;s optical path, <br />                                                shifting focal plane millimeters scanner glass, matching agar surface <br />                                                inside plates. To ensure agar surface consistent experiments, developed <br />                                                robust method controlled drying agar plates (Online Methods). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Automated identification death times <br />                                                Following optimized scanning schedule (Online Methods), LM captures 24 images of <br />                                                plate day. The image-processing pipeline identifies worm objects (Supplementary <br />                                                Note 2) position frame (Fig. 1b,c). Individual animals, stationary <br />                                                (Fig. 1d), continue change posture time, moving head tail (Fig. 1e, <br />                                                Supplementary Videos 1-5 Note 5), reminiscent &#226;&#8364;&#339;class C&#226;&#8364;? animals described in <br />                                                studies age-dependent locomotory decline18. Our software determines final cessation <br />                                                spontaneous postural movement retrospective image analysis (Online Methods, <br />                                                Supplementary Note 5 Video 6). Posture analysis stationary animals crucial to <br />                                                avoid underestimation lifespan. We observed wild type animals spending average 1.2 <br />                                                days life state (Fig. 1e), certain mutants, example age-1(hx546), <br />                                                persist twice long (Supplementary Note 6), consistent previous <br />                                                reports19. <br />  <br />                                                We animals exhibit stereotyped morphological change near time <br />                                                final posture change (Supplementary Note 7). Individuals shrink 10% <br />                                                expand shortly thereafter. We observed change in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                posture expansion. The correlation cessation postural motion and <br />                                                morphological change suggests physiological transition, corresponding the <br />                                                animal&apos;s death, corroborating death criterion. <br />  <br />                                                A crucial component platform software package allows rapid validation of <br />                                                death times visual inspection (Supplementary Note 4). This quality control step <br />                                                (Fig. 1f, Supplementary Video 6, Online Methods) allows user determine plates <br />                                                compromised (e.g. desiccation fungal contamination) need exclusion. In <br />                                                particular, permits identification aggregates containing multiple worms (Supplementary <br />                                                Note 8) censoring incorrectly classified objects (e.g. features agar lawn) <br />                                                non-aging related deaths (e.g. body rupture), enabling visual validation of <br />                                                machine operation. <br />  <br />  <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                           Page 4 <br />  <br />  <br />                                The LM produces accurate precise survival curves <br />                                                To evaluate LM produces results consistent human observer, we <br />                                                placed roughly 8,000 age-synchronous wild type animals single liquid suspension, and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                distributed aliquots approximately 35 individuals 170 agar plates seeded E. <br />                                                coli OP50 (Online Methods). 160 plates distributed scanners <br />                                                situated incubator 25 &#194;&#176;C. The remaining 10 plates placed separate <br />                                                incubator monitored daily hand using dissecting microscope. <br />  <br />                                                Because drew animals pool, significant difference between <br />                                                survival curves individual plates attributed measurement error or <br />                                                environmental variation, differences plate temperature. We compared death times <br />                                                grouped plate single scanner acquired manual procedure (Fig. 2a). <br />                                                In methods, plate means varied essentially range, 10.7 and <br />                                                13.1 days. No significant effect observed plates collected manual <br />                                                method (Log-rank test homogeneity P =0.059). A small, significant difference was <br />                                                observed plates collected automated method (Log-rank P =0.002), reflecting <br />                                                inaccuracies estimation death times slightly increased environmental <br />                                                variability plates. In independent replicates, survival curves determined by <br />                                                LM differed mean lifespan 2.5 hours 0.2% (Log-rank P =0.01, Fig. 2b) 11.7 <br />                                                hours 3.2% (Log-rank P &lt;0.001) cohorts assayed using conventional <br />                                                method. These small differences explained practical limitations ability to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                measure maintain temperature constant methodologies. <br />  <br />                                                Where experiments require comparison populations larger 300 <br />                                                individuals, animals distributed multiple scanners. With statistical power <br />                                                (Supplementary Note 9) afforded scanner, slight environmental differences <br />                                                scanners translate statistically significant differences survival curves <br />                                                (Fig. 2c top). Indeed, scanner surface temperature correlated mean lifespan (R2 = <br />                                                0.75; P=0.023; Supplementary Note 1). <br />  <br />                                                Because devices produced small effect lifespan, applied accelerated failure time <br />                                                regression model scanner identity categorical covariate generate &#226;&#8364;&#339;device- <br />                                                corrected&#226;&#8364;? lifespans (Fig. 2c, Supplementary Note 10). This procedure useful assessing <br />                                                data pooling justified standardizing data desired baseline curve, <br />                                                example, aligning large experiments single scanner. <br />  <br />                                                The LM provides data quantity quality appropriate estimating time-dependent <br />                                                hazard (mortality) rates, Supplementary Note 11. We observed rapid increase and <br />                                                subsequent deceleration mortality time, noted previously10-12,14. For 50% <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                deaths, mortality appears better fit power time t (Weibull hazard, &#206;&#177;/&#206;&#178; (t/&#206;&#178;)(&#206;&#177;&#226;&#710;&#8217;1)) <br />                                                exponential time (Gompertz hazard, et/b), Fig. 2d Supplementary Note 11, <br />                                                values Gompertz parameter smaller 1, Gompertz Weibull <br />                                                distributions behave similarly lowest quantiles survival, making <br />                                                disambiguation difficult. An analysis hazard data, accounting deceleration <br />                                                phase, require frailty models incorporate phenotypic diversity potentially present <br />                                                isogenic populations20, 21. However, observed hazard deceleration the <br />                                                result environmental heterogeneity apparatus, persists device-correction <br />                                                (data shown). <br />  <br />                                                The close agreement automated manual survival curves surprising, as <br />                                                scanners expose animals conditions differ manual method, as <br />                                                oscillating lights22 temperature oscillations (Supplementary Note 1). The difference in <br />                                                lifespan animals exposed exposed scanner environment, scored <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                           Page 5 <br />  <br />  <br />                                                hand, small explained limitations ability maintain same <br />                                                temperature environments (Supplementary Note 12). Importantly, LM the <br />                                                manual procedure adopt different operational definitions death&#226;&#8364;&#8221;cessation spontaneous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                vs. stimulated movement, respectively. We sought validation process than <br />                                                comparison manual method developing software assist visual inspection <br />                                                image record generated LM experiment (Supplementary Note 4 and <br />                                                Video 6). Two survival curves constructed 294 user-annotated death times based on <br />                                                images scanners statistically indistinguishable curves produced by <br />                                                automated annotation (Log-rank P &gt; 0.5), Fig. 2e-f. <br />  <br />                                The LM reproduces known lifespans genetic perturbations <br />                                                To test LM used characterize effects mutations RNAi on <br />                                                lifespan, divided wild type animals daf-16(mu86) mutants3 plates seeded <br />                                                E. coli HT115(DE3) strain containing RNAi construct targeting insulin/ <br />                                                IGF receptor daf-23, 22 vector. As expected, daf-16 mutants live shorter than <br />                                                wild type animals23, knockdown daf-2 RNAi extended lifespan wild type <br />                                                animals daf-16 mutants (Fig. 3a)24. We significant difference between <br />                                                manual automated survival curves conditions (Log-rank P &gt; 0.3) those <br />                                                wild type animals feeding control bacteria (Log-rank P &lt; 0.001), appeared to <br />                                                live 21 hours longer manual assay. A subsequent replicate showed significant <br />                                                difference (data shown). These experiments performed 25 &#194;&#176;C; observed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                corresponding effects 20 &#194;&#176;C (Supplementary Note 13). <br />  <br />                                                Consistent previous reports2, 23, 25,5,26, mutations phosphoinositide 3-kinase <br />                                                age-1, syntaxin homolog unc-64, Notch receptor glp-1 result lifespan <br />                                                extension, mutations daf-16, heat shock factor hsf-1, CD91 homolog <br />                                                ced-1 shorten lifespan. Our automated technique correctly identifies known genetic <br />                                                determinants C. elegans lifespan (Fig. 3b&#226;&#8364;&#8221;h). Although population sizes (Supplementary <br />                                                Table 1) small compare parametric models, used data estimate27 the <br />                                                hazard rates mutant (Fig. 3d,f,h, Online Methods). <br />  <br />                                                Visual image inspection (Fig. 3i,j) validates LM-acquired survival curves mutants. <br />                                                We note machine performed movement-defective mutants unc-64(e246), <br />                                                unc-4(e120), unc-50(e306), unc-119(ed3) (Supplementary Note 13), the <br />                                                image analysis parameters optimized wild type movement. The analysis of <br />                                                glp-1(e2141) performed equally wild type, despite mutant animals being <br />                                                thinner absence germline. <br />  <br />                                The LM enables high-resolution stress resistance assays <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Survival assays widely used evaluate ability individuals withstand exogenous <br />                                                stresses. Exposure high temperature (35 &#194;&#176;C) toxins dramatically shortens lifespan <br />                                                C. elegans. Mean survival ranges hours days16, 28, making data <br />                                                collection high frequency challenging manual approach. We evaluated the <br />                                                performance LM stress-resistance scenarios: exposure high temperature16 <br />                                                oxidant tert-butyl hydroperoxide (t-BuOOH)28. The LM required modifications <br />                                                adjustments subset parameters used image analysis software to <br />                                                quantify worm movement identify worm death times (Online Methods). <br />  <br />                                                Age-synchronous wild type, age-1(hx546), daf-16(mu86) mutants grown under <br />                                                standard conditions 25 &#194;&#176;C shifted second day adulthood four <br />                                                conditions: agar plates 25 &#194;&#176;C 35 &#194;&#176;C agar plates 3 mM 6 mM t-BuOOH at <br />                                                25 &#194;&#176;C. While wild type animals died days control plates 25 &#194;&#176;C, <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                            Page 6 <br />  <br />  <br />                                                lifespan shortened dramatically 35 &#194;&#176;C. In agreement previous reports16, <br />                                                age-1(hx546) mutants lived longer wild type daf-16(mu86) mutants showed small <br />                                                statistically significant reduction survival (Fig. 4a,b). The results obtained visual <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                automated image analyses highly correlated (Fig. 4c,d). We observed similar, <br />                                                concentration-dependent effect t-BuOOH lifespan, consistent previous studies28 <br />                                                (Fig. 4e,f). The LM suited assaying effects chemicals survival <br />                                                C. elegans. <br />  <br />                                The hazard rate animals stress <br />                                                We hazard rate function wild type, age-1(hx546) daf-16(mu86) 35 &#194;&#176;C to <br />                                                consist accelerating decelerating phase (Fig. 5a), 25 &#194;&#176;C, suggesting the <br />                                                demographic characteristics aging preserved intense stress conditions. <br />                                                These features did depend presence live bacteria, observed in <br />                                                populations raised assayed UV-killed bacteria (Fig. 5b). To knowledge, is <br />                                                quantification hazard rate function metazoans extreme conditions. <br />  <br />                                                The effect thermal stress appears consist primarily rescaling time, barely <br />                                                impacting coefficient variation (CV) lifespan distribution (Supplementary Table <br />                                                1). Up median lifespan barring frailty effects, 1 hour animals 25 &#194;&#176;C corresponds <br />                                                1.8 minutes animals 35 &#194;&#176;C experience moment 33-fold <br />                                                higher risk death. Likewise, effect age-1(hx546) daf-16(mu86) mutations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                relative 35 &#194;&#176;C consists primarily change time scale (Fig. 5a) a <br />                                                marginal effect CV. This marked departure behavior 25 &#194;&#176;C (Fig. 3d) <br />                                                DAF-16 increases AGE-1 decreases lifespan variation disproportionately relative <br />                                                mean. <br />  <br />                                                Previous studies C. elegans shown brief heat pulse induces heterogeneous <br />                                                stress response, magnitude predicts survival subsequent shift high <br />                                                temperature16, 29. We evaluated effect pretreatment thermotolerance a <br />                                                substantial stretching time scale (Fig. 5b). <br />  <br />                                                Despite separation 100 million years more30, Caenorhabditis species elegans, <br />                                                briggsae, brenneri, species 11 sufficiently similar appearance LM to <br />                                                function correctly additional calibration 35 &#194;&#176;C. We confirmed previous reports31 <br />                                                C. briggsae, C. brenneri thermotolerant C. elegans, to <br />                                                case C. species 11. The thermotolerance profiles species correlate <br />                                                temperatures natural habitats30, tropical species more <br />                                                thermotolerant temperate ones. As observed C. elegans mutants, genetic <br />                                                background Caenorhabditis species, too, causes predominantly rescaling time, while <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                having smaller effect coefficient variation lifespan distribution <br />                                                (Supplementary Table 1). <br />  <br />                                Discussion <br />                                                By using standard nematode culture conditions, method meaningfully extends, within <br />                                                limits (Supplementary Note 16), existing experimental literature checked <br />                                                it. The modular design LM gives researchers flexibility expand an <br />                                                installation simply adding scanners (Supplementary Notes 14 15). A single scanner is <br />                                                sufficient compare lifespan mutant population wild type control, <br />                                                detecting 10% differences survival 99% confidence 99% power. Facilities ten <br />                                                scanners provide sufficient throughput systematically characterize effects of <br />                                                genes environmental conditions survival. At larger scale, scope be <br />                                                paired depth evaluating samples large resolve quantitatively <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                                                Page 7 <br />  <br />  <br />                                                compare hazard rate features, linking interventions molecular level their <br />                                                statistical footprint organismic level. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Supplementary Material <br />                                                Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                                We like thank Joy Alcedo (Wayne State University) providing hsf-1 glp-1 mutant strains, <br />                                                Xavier Mani&#195;&#168;re (Universit&#195;&#169; Paris Descartes) providing NEC937 strain, Becky Ward Debora Marks for <br />                                                critical reading manuscript Catalina Romero, Debora Marks members Fontana lab helpful <br />                                                discussions encouragement project. We like thank Tom Kolokotrones, Eric Smith, and <br />                                                Lee Jen Wei discussions statistical advice, Mason Miranda, departmental IT specialist, patiently <br />                                                meeting needs data storage. This work funded National Institute Health grants R03 <br />                                                AG032481, R03 AG032481-S1, R01 AG034994. <br />  <br />  <br />                                References <br />                                                1. Kirkwood TB, et al. What accounts wide variation life span genetically identical <br />                                                   organisms reared constant environment? Mech Ageing Dev. 2005; 126:439&#226;&#8364;&#8220;443. [PubMed: <br />                                                   15664632] <br />                                                2. Friedman DB, Johnson TE. A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> age-1 gene Caenorhabditis elegans lengthens life <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   reduces hermaphrodite fertility. Genetics. 1988; 118:75&#226;&#8364;&#8220;86. [PubMed: 8608934] <br />                                                3. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R. A C. elegans mutant lives twice long <br />                                                   wild type. Nature. 1993; 366:461&#226;&#8364;&#8220;464. [PubMed: 8247153] <br />                                                4. Kimura KD, Tissenbaum HA, Liu Y, Ruvkun G. daf-2, insulin receptor-like gene regulates <br />                                                   longevity diapause Caenorhabditis elegans. Science. 1997; 277:942&#226;&#8364;&#8220;946. [PubMed: <br />                                                   9252323] <br />                                                5. Kenyon CJ. The genetics ageing. Nature. 2010; 464:504&#226;&#8364;&#8220;512. [PubMed: 20336132] <br />                                                6. Sutphin GL, Kaeberlein M. Measuring Caenorhabditis elegans life span solid media. J Vis Exp. <br />                                                   2009 <br />                                                7. Wilkinson DS, Taylor RC, Dillin A. Analysis aging Caenorhabditis elegans. Methods Cell <br />                                                   Biol. 2012; 107:353&#226;&#8364;&#8220;381. [PubMed: 22226530] <br />                                                8. Curtsinger JW, Fukui HH, Townsend DR, Vaupel JW. Demography genotypes: failure the <br />                                                   limited life-span paradigm Drosophila melanogaster. Science. 1992; 258:461&#226;&#8364;&#8220;463. [PubMed: <br />                                                   1411541] <br />                                                9. Carey JR, Liedo P, Vaupel JW. Mortality dynamics density Mediterranean fruit fly. Exp <br />                                                   Gerontol. 1995; 30:605&#226;&#8364;&#8220;629. [PubMed: 8867529] <br />                                                10. Vaupel JW, et al. Biodemographic trajectories longevity. Science. 1998; 280:855&#226;&#8364;&#8220;860. <br />                                                     [PubMed: 9599158] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                11. Vanfleteren JR, De Vreese A, Braeckman BP. Two-parameter logistic Weibull equations <br />                                                     provide better fits survival data isogenic populations Caenorhabditis elegans axenic <br />                                                     culture does Gompertz model. J Gerontol A Biol Sci Med Sci. 1998; 53:B393&#226;&#8364;&#8220;403. <br />                                                     discussion B404-398. [PubMed: 9823735] <br />                                                12. Johnson TE, Wu D, Tedesco P, Dames S, Vaupel JW. Age-specific demographic profiles of <br />                                                     longevity mutants Caenorhabditis elegans segmental effects. J Gerontol A Biol Sci Med <br />                                                     Sci. 2001; 56:B331&#226;&#8364;&#8220;339. [PubMed: 11487591] <br />                                                13. Mair W, Goymer P, Pletcher SD, Partridge L. Demography dietary restriction death in <br />                                                     Drosophila. Science. 2003; 301:1731&#226;&#8364;&#8220;1733. [PubMed: 14500985] <br />                                                14. Baeriswyl S, et al. Modulation aging profiles isogenic populations Caenorhabditis elegans <br />                                                     bacteria causing different extrinsic mortality rates. Biogerontology. 2009; 11:53&#226;&#8364;&#8220;65. [PubMed: <br />                                                     19444640] <br />                                                15. Wu D, Rea SL, Cypser JR, Johnson TE. Mortality shifts Caenorhabditis elegans: remembrance <br />                                                     conditions past. Aging Cell. 2009; 8:666&#226;&#8364;&#8220;675. [PubMed: 19747231] <br />  <br />  <br />                                                     Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                                      Page 8 <br />  <br />  <br />                                                16. Lithgow GJ, White TM, Melov S, Johnson TE. Thermotolerance extended life-span conferred <br />                                                    single-gene mutations induced thermal stress. Proc Natl Acad Sci U S A. 1995; <br />                                                    92:7540&#226;&#8364;&#8220;7544. [PubMed: 7638227] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                17. Klass MR. Aging nematode Caenorhabditis elegans: major biological environmental <br />                                                    factors influencing life span. Mech Ageing Dev. 1977; 6:413&#226;&#8364;&#8220;429. [PubMed: 926867] <br />                                                18. Herndon LA, et al. Stochastic genetic factors influence tissue-specific decline ageing C. <br />                                                    elegans. Nature. 2002; 419:808&#226;&#8364;&#8220;814. [PubMed: 12397350] <br />                                                19. Huang C, Xiong C, Kornfeld K. Measurements age-related changes physiological processes <br />                                                    predict lifespan Caenorhabditis elegans. Proc Natl Acad Sci U S A. 2004; 101:8084&#226;&#8364;&#8220;8089. <br />                                                    [PubMed: 15141086] <br />                                                20. Vaupel JW, Manton KG, Stallard E. The impact heterogeneity individual frailty the <br />                                                    dynamics mortality. Demography. 1979; 16:439&#226;&#8364;&#8220;454. [PubMed: 510638] <br />                                                21. Weitz JS, Fraser HB. Explaining mortality rate plateaus. Proc Natl Acad Sci U S A. 2001; <br />                                                    98:15383&#226;&#8364;&#8220;15386. [PubMed: 11752476] <br />                                                22. Mathew MD, Mathew ND, Ebert PR. WormScan: technique high-throughput phenotypic <br />                                                    analysis Caenorhabditis elegans. PLoS One. 2012; 7:e33483. [PubMed: 22457766] <br />                                                23. Larsen PL, Albert PS, Riddle DL. Genes regulate development longevity in <br />                                                    Caenorhabditis elegans. Genetics. 1995; 139:1567&#226;&#8364;&#8220;1583. [PubMed: 7789761] <br />                                                24. Dillin A, Crawford DK, Kenyon C. Timing requirements insulin/IGF-1 signaling C. elegans. <br />                                                    Science. 2002; 298:830&#226;&#8364;&#8220;834. [PubMed: 12399591] <br />                                                25. Ailion M, Inoue T, Weaver CI, Holdcraft RW, Thomas JH. Neurosecretory control aging in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Caenorhabditis elegans. Proc Natl Acad Sci U S A. 1999; 96:7394&#226;&#8364;&#8220;7397. [PubMed: 10377425] <br />                                                26. Haskins KA, Russell JF, Gaddis N, Dressman HK, Aballay A. Unfolded protein response genes <br />                                                    regulated CED-1 required Caenorhabditis elegans innate immunity. Dev Cell. 2008; <br />                                                    15:87&#226;&#8364;&#8220;97. [PubMed: 18606143] <br />                                                27. Muller HG, Wang JL. Hazard rate estimation random censoring varying kernels and <br />                                                    bandwidths. Biometrics. 1994; 50:61&#226;&#8364;&#8220;76. [PubMed: 8086616] <br />                                                28. Tullet JM, et al. Direct inhibition longevity-promoting factor SKN-1 insulin-like <br />                                                    signaling C. elegans. Cell. 2008; 132:1025&#226;&#8364;&#8220;1038. [PubMed: 18358814] <br />                                                29. Rea SL, Wu D, Cypser JR, Vaupel JW, Johnson TE. A stress-sensitive reporter predicts longevity <br />                                                    isogenic populations Caenorhabditis elegans. Nat Genet. 2005; 37:894&#226;&#8364;&#8220;898. [PubMed: <br />                                                    16041374] <br />                                                30. Kiontke KC, et al. A phylogeny molecular barcodes Caenorhabditis, numerous new <br />                                                    species rotting fruits. BMC Evol Biol. 2011; 11:339. [PubMed: 22103856] <br />                                                31. Amrit FR, Boehnisch CM, May RC. Phenotypic covariance longevity, immunity stress <br />                                                    resistance caenorhabditis nematodes. PLoS One. 2010; 5:e9978. [PubMed: 20369008] <br />                                                32. Stiernagle T. Maintenance C. elegans. WormBook. 2006:1&#226;&#8364;&#8220;11. [PubMed: 18050451] <br />                                                33. Dong JX, Krzyzak A, Suen CY. Fast SVM training algorithm decomposition large <br />                                                    data sets. IEEE Trans <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>Pattern</span> Anal Mach Intell. 2005; 27:603&#226;&#8364;&#8220;618. [PubMed: 15794164] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                34. Sall, J. SAS Institute. JMP start statistics: guide statistics data analysis using JMP. 5th. <br />                                                    SAS Institute; Cary, NC: 2012. <br />                                                35. R Development Core Team. R: A language environment statistical computing. R <br />                                                    Foundation Statistical Computing; Vienna, Austria: 2006. <br />                                                36. Buckley J, James I. Linear regression censored data. Biometrika. 1979; 66:429&#226;&#8364;&#8220;436. <br />  <br />  <br />  <br />  <br />                                                     Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                             Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Figure 1. <br />                                                Lifespan Machine workflow. (a) Agar plates age-synchronized worms placed face- <br />                                                surface flatbed scanners (Supplementary Note 1). (b) Each scanner captures <br />                                                time series images of plates. Images processed identify foreground <br />                                                objects (Supplementary Note 2). Scale bar = 250 &#206;&#188;m. (c) Various morphological features are <br />                                                quantified object. The population distributions features demonstrate that <br />                                                permits itself discrimination worms non-worms. Based validated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                images worm non-worm objects, classifiers constructed capable of <br />                                                categorizing objects based 65-dimensional feature vectors. The panel depicts a <br />                                                projection point cloud representing validated set feature vectors, indicating that <br />                                                differentiating worms (black) non-worms (red) possible (Supplementary Note 2). (d) <br />                                                Once worm objects identified, individual image stacks analyzed identify <br />                                                animals stationary (Supplementary Note 5). (e) Stationary animals are <br />                                                analyzed detect posture changes, including head tail movements. Death is <br />                                                identified retrospective analysis final cessation postural change (Supplementary <br />                                                Note 5). (f) The LM produces time-lapse image record individual. These records <br />                                                used visual validation resolution ambiguities employing rapid data- <br />                                                inspection tool, &#226;&#8364;&#339;Worm Browser&#226;&#8364;? (Supplementary Video 6 Note 4). Validated death <br />                                                times combined automatic censoring data (Supplementary Note 8) Kaplan- <br />                                                Meier survival curve (actual data shown) hazard rates using statistics package. <br />  <br />  <br />  <br />                                                    Nat Methods. Author manuscript; available PMC 2014 January 01. <br />                             Stroustrup et al.                                                                                        Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Figure 2. <br />                                                Automated wild type survival data. (a) A survival curve generated 484 death times <br />                                                wild type animals co-located single scanner (top) 513 wild type animals <br />                                                observed using manual method (bottom). (b) The entire population 3,578 wild type <br />                                                animals scanners aggregated single curve, nearly indistinguishable <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                manually-scored curve. (c) Scanner-specific microenvironments, mainly differences in <br />                                                temperature, affect lifespan (top). The effect estimated using categorical Accelerated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Failure Time model, obtain device-corrected lifespans registered grand mean <br />                <br /> </body></html>