<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Knockdown Asparagine Synthetase A Renders <br /> Trypanosoma brucei Auxotrophic Asparagine <br /> Ine&#203;&#8224;s Loureiro1, Joana Faria1, Christine Clayton2, Sandra Macedo Ribeiro3, Nilanjan Roy4, <br /> Nuno Santare&#194;&#180;m1, Joana Tavares1.*, Anabela Cordeiro-da-Silva1,5.* <br /> 1 Parasite Disease Group, Instituto Biologia Molecular e Celular da Universidade Porto, Porto, Portugal, 2 Zentrum fu&#194;&#168;r Molekulare Biologie der Universita&#194;&#168;t <br /> Heidelberg, DKFZ-ZMBH Alliance, Heidelberg, Germany, 3 Protein Crystallography Group, Instituto Biologia Molecular e Celular da Universidade Porto, Porto, <br /> Portugal, 4 Ashok Rita Patel Institute Integrated Study Research Biotechnology Allied Sciences, New Vallabh Vidyanagar, Gujarat, India, 5 Departamento <br /> de Cie&#203;&#8224;ncias Biolo&#194;&#180;gicas, Faculdade Farma&#194;&#180;cia da Universidade Porto, Porto, Portugal <br />  <br />  <br />  <br />        <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />        Asparagine synthetase (AS) catalyzes ATP-dependent conversion aspartate asparagine using ammonia or <br />        glutamine nitrogen source. There distinct types AS, asparagine synthetase A (AS-A), known strictly <br />        ammonia-dependent, asparagine synthetase B (AS-B), use ammonia glutamine. The absence AS- <br />        A humans, presence trypanosomes, suggested AS-A potential drug target deserved further <br />        investigation. We report presence functional AS-A Trypanosoma cruzi (TcAS-A) Trypanosoma brucei (TbAS-A): <br />        purified enzymes convert L-aspartate L-asparagine presence ATP, ammonia Mg2+. TcAS-A TbAS-A <br />        use preferentially ammonia nitrogen donor, surprisingly, use glutamine, characteristic far never <br />        described AS-A. TbAS-A knockdown RNAi didn&#226;&#8364;&#8482;t affect vitro growth bloodstream forms parasite. <br />        However, growth significantly impaired TbAS-A knockdown parasites cultured medium reduced <br />        levels asparagine. As expected, mice infections induced non-induced T. brucei RNAi clones similar those <br />        wild-type parasites. However, induced T. brucei RNAi clones injected mice undergoing asparaginase <br />        treatment, depletes blood asparagine, mice exhibited lower parasitemia prolonged <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>survival</span> comparison <br />        similarly-treated mice infected control parasites. Our results TbAS-A important vivo <br />        conditions asparagine limiting, unlikely suitable drug target. <br />  <br />      Citation: Loureiro I, Faria J, Clayton C, Ribeiro SM, Roy N, et al. (2013) Knockdown Asparagine Synthetase A Renders Trypanosoma brucei Auxotrophic <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      Asparagine. PLoS Negl Trop Dis 7(12): e2578. doi:10.1371/journal.pntd.0002578 <br />      Editor: Alejandro Buschiazzo, Institut Pasteur Montevideo, Uruguay <br />      Received August 1, 2013; Accepted October 25, 2013; Published December 5, 2013 <br />      Copyright: &#195;&#376; 2013 Loureiro et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />      unrestricted use, distribution, reproduction medium, provided original author source credited. <br />      Funding: The authors received specific funding study. <br />      Competing Interests: The authors declared competing interests exist. <br />      * E-mail: jtavares@ibmc.up.pt (JT); cordeiro@ibmc.up.pt (ACdS) <br />      . These authors contributed equally work. <br />  <br />  <br />  <br />  <br /> Introduction                                                                              AS-A AS-B share sequence structural similarities. <br />                                                                                        Their three-dimensional structures provided important informa- <br />    Asparagine naturally occurring non-<span id='am-3' about='Thesaurus:Essential_Amino_Acid' typeof='owl:Thing'>essential amino acid</span>                        tion concerning distinct catalytic mechanisms [2,23&#226;&#8364;&#8220;25]. AS- <br /> found proteins. Due high nitrogen/carbon ratio,                         A exists dimer monomer core b- <br /> asparagine likely linked nitrogen homeostasis                          strands flanked a-helices, resembling catalytic domain of <br /> protein biosynthesis [1]. AS protein involved asparagine                     class II aminoacyl-tRNA synthetases aspartyl-tRNA <br /> biosynthesis. There distinct types AS, AS-A AS-B,                       synthetase [24]. AS-A synthesizes asparagine steps: b- <br /> encoded asnA asnB genes, respectively. AS-A encoding                            carboxylate group aspartate activated ATP form an <br /> genes reported archaea [2,3], prokaryotes [4&#226;&#8364;&#8220;7],                      aminoacyl-AMP, followed amidation nucleophilic attack <br /> in protozoan parasite Leishmania [8]. The AS-B encoding gene                       ammonium ion [2]. The AS-B enzyme forms dimer, <br /> is present prokaryotes [5,9] eukaryotes, including                      monomer contains distinct domains, which <br /> mammalian cells [10,11], yeasts [12], algae [13], higher plants                    contains catalytic site. The N-terminal site catalyzes the <br /> [14]. Both types catalyze ATP-dependent conversion                              conversion glutamine glutamic acid ammonia, while <br /> aspartate asparagine. While AS-B use ammonia                             aspartate reacts ATP C-terminal site, generating the <br /> and glutamine (reaction B) amide nitrogen donors [5,15&#226;&#8364;&#8220;20],                         intermediate b-aspartyl-AMP [26,27]. Similarly gluta- <br /> Escherichia coli (E. coli) AS-A reported dependent strictly                  dependent amidotransferases, ammonia released N- <br /> on ammonia (reaction A) [21,22].                                                       terminal domain enzyme travels intramolecular <br />                                                                                        tunnel connecting active sites, reacts reactive <br /> A)      ATP + L-aspartate + NH4+ = . AMP + diphosphate + L-                            acyladenylate intermediate produce asparagine [28]. <br />         asparagine                                                                        An open reading frame encoding putative AS-A present in <br /> B)      ATP + L-aspartate + L-glutamine [or NH4+] = . AMP +                            genome protozoan parasites, Trypanosoma cruzi (T. cruzi) <br />         diphosphate + L-asparagine + L-glutamate                                       Trypanosoma brucei (T. brucei) [29&#226;&#8364;&#8220;31]. T. cruzi T. brucei are <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                                1                           December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                         Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />   Author Summary                                                         Cloning T. brucei T. cruzi ASA genes <br />                                                                             T. brucei asparagine synthetase A (TbASA) T. cruzi <br />   The amino acid asparagine important                    asparagine synthetase (TcASA) genes obtained performing <br />   protein biosynthesis, nitrogen homeostasis.               PCR genomic DNA Trypanosoma brucei brucei TREU927 <br />   Asparagine synthetase catalyzes synthesis                  Trypanosoma cruzi CL Brener Non-Esmeraldo-like. Fragments <br />   amino acid. There forms asparagine synthetase,              open reading frames TbASA (Tb927.7.1110; chromosome <br />   A B. The presence type A trypanosomes,               Tb927_07_v4; 28861 289067) TcASA (Tc00.1047053503 <br />   absence humans, makes protein potential drug <br />                                                                          625.10; chromosome TcChr29-P; 687159&#226;&#8364;&#8220;688206) PCR- <br />   target. Trypanosomes responsible parasitic <br />                                                                          amplified using Taq DNA polymerase proofreading activity <br />   diseases rely limited drug therapeutic options for <br />   control. In study present functional characteriza-            (Roche). The sequences primers follows: sense <br />   tion trypanosomes asparagine synthetase A. We                       primer 59 - CTAATTACATATGGGCGACGACGGTTATTC - <br />   Trypanosoma brucei Trypanosoma cruzi                 39 antisense primer 59 - CCCAAGCGAATTCTTACAACA- <br />   type A enzymes able use ammonia                       AATTGTGC - 39, sense primer 59 - CAAT TTGCATATGACA- <br />   glutamine nitrogen donor, conversion                TCGGGAGATCC - 39 antisense primer 59 - CCCAAGCA- <br />   aspartate asparagine. Furthermore,                   AGCTTTCACAGCAAGGG - 39, respectively. PCR conditions <br />   asparagine synthetase A knockdown renders Trypanosoma                  follows: initial denaturation (2 min 94uC), 35 cycles of <br />   brucei auxotrophic asparagine. Overall, study                  denaturation (30 s 94uC), annealing (30 s 45uC) and <br />   demonstrates interfering asparagine metabolism               elongation (2 min 68uC) TbAS-A, annealing (30 s at <br />   represents way control parasite growth infectivity.           50uC) elongation (2 min 68uC) TcAS-A, final <br />                                                                          extension step (10 min 68uC). The PCR products isolated <br />                                                                          1% agarose gel, purified Qiaex II protocol (Qiagen), <br /> transmitted mammalian host invertebrate vector,          cloned pGEM-T Easy vector (Promega) sent to <br /> and responsible Chagas disease African sleeping              Eurofins MWG (Germany) sequencing. <br /> sickness, respectively. Disease control dependent drug <br /> therapy, treatment options limited, high toxicity        Expression purification poly-His-tagged <br /> and recent emergence drug resistance [32&#226;&#8364;&#8220;34]. Vaccines T.         recombinant TbAS-A TcAS-A <br /> brucei infections unlikely developed          The TbASA TcASA genes subcloned pET28a(+) <br /> extensive antigenic variation [35], infections          expression vector (Novagen). The recombinant 6-His-tagged <br /> compromise host humoral immune competence [36].                          proteins expressed E. coli BL21DE3 induction log- <br />    Trypanosome AS-A drug target absence            phase cultures 0.5 mM IPTG (NZYTech) 3 h 37uC <br /> of homologue humans [8]. AS-A important                 (TcAS-A) 18uC, overnight (O/N) (TbAS-A). Bacteria were <br /> microorganisms. For example, asnA essential gene                harvested resuspended buffer A [0.5 M NaCl (Sigma), <br /> Haemophilus influenzae (DEG10050178) [37], strongly up-           20 mM Tris.HCl (Sigma), pH 7.6]. The sample sonicated and <br /> regulated Pasteurella multocida host infection [38],       centrifuged obtain bacterial crude extract. The recombinant <br /> when Klebsiella aerogenes grown amino acid-limited          proteins purified using Ni2+ resin (ProBond) washing and <br /> ammonia rich environment [5]. We undertook bio-                elution increasing levels (25 mM 1 M) imidazole <br /> chemical genetic studies AS-A trypanosomes ascertain        (Sigma). The presence purity recombinant protein in <br /> its biological role evaluate potentiality drug target.        fractions determined SDS-PAGE and <br />                                                                          Coomassie staining. Dialysis performed PBS <br /> Materials Methods                                                    [137 mM NaCl (Sigma), 2.7 mM KCl (Sigma), 10 mM Na2H- <br />                                                                          PO4.2H2O (Riedel-de Hae&#194;&#168;n), 2 mM KH2PO4 (Riedel-de Hae&#194;&#168;n) <br /> Ethics statement                                                         pH 7.4]. <br />   All experiments involving animals carried accor-              To generate rat rabbit polyclonal antibodies TbAS- <br /> dance IBMC.INEB Animal Ethics Committees                A, animal immunized 150 mg recombinant <br /> Portuguese National Authorities Animal Health guidelines,            TbAS-A protein. After 2 weeks, 4 boosts 100 mg of <br /> according statements directive 2010/63/EU           recombinant TbAS-A given weekly. The collected blood <br /> European Parliament Council. IL, JT ACS           samples centrifuged obtain serum. <br /> accreditation animal research given Portuguese <br /> Veterinary Direction (Ministerial Directive 1005/92).                    Protein extracts western blot analysis <br />                                                                             Extracts obtained RIPA buffer [(20 mM Tris-HCl <br /> Parasite culture                                                         (Sigma) (pH 7.5), 150 mM NaCl (Sigma), 1 mM Na2EDTA <br />    Procyclic bloodstream forms Trypanosoma brucei brucei          (Sigma), 1 mM EGTA (Sigma), 1% Nonidet P-40 (Sigma), 1% <br /> Lister 427 used. Procyclic forms grown MEM-Pros             sodium deoxycholate (Sigma), 2.5 mM sodium pyrophosphate <br /> medium supplemented 7.5 mg/ml hemin, 10% fetal calf                 (Sigma), 1 mM b-glycerophosphate (Sigma), 1 mM Na3VO4 <br /> serum (FCS) 100 IU/mL penicillin/streptomycin 27uC,            (Sigma)], freshly-added complete protease inhibitor cocktail <br /> with cell densities 56105 cells/ml 1&#226;&#8364;&#8220;26107 cells/ml.          (Roche Applied Science). The total protein quantified <br /> Bloodstream forms grown complete HMI-9 medium                    using Biorad Commercial Kit (Reagents A, B S) the <br /> (supplemented 10% FCS 100 IU/mL penicillin/                  samples kept 280uC. For analysis parasites <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> streptomycin) [39] vented tissue culture flasks; cultures       mice, trypanosomes purified mouse blood using DE- <br /> were diluted cultures reached cell density 26106/ml          52 (Whatman) column [40]. <br /> and incubated humidified atmosphere 5% CO2, 37uC.                For Western blotting, 2 mg recombinant TbAS-A TcAS- <br /> Bloodstream RNAi cell cultures supplemented 7.5 mg/            A proteins, 10 mg total soluble cell extract, 16107 parasites, <br /> ml hygromycin 0.2 mg/ml phleomycin.                                  resolved SDS/PAGE transferred nitrocellulose <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                  2                       December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 1. Multiple-sequence alignment AS-A protein. Alignment type A asparagine synthetase T. brucei (NCBI-GeneID:3658321/ <br /> Tb927.7.1110), T. cruzi (NCBI-GeneID:3534325/Tc00.1047053503625.10) E. coli (NCBI-GeneID:948258/pdb:12AS). The residues colored <br /> according ALSCRIPT Calcons (Aline version 011208) using predefined colour scheme (red: identical residues; orange blue: scale conservation <br /> of amino acid properties alignment column; white: dissimilar residues). Secondary structure elements EcAS-A crystal structure (grey) of <br /> TcAS-A (purple) TbAS-A (blue) homology models depicted alignment. In protein sequences, asparagine (squares) AMP <br /> binding residues (circles) identified. The dashed box indicates structurally divergent region. <br /> doi:10.1371/journal.pntd.0002578.g001 <br />  <br /> Hy-bond ECL membrane (Amersham Biosciences). The mem-                            Using query based L-cysteine-S-sulfinic acid inhibitor [42], <br /> brane blocked 5% (w/v) non-fat dried skimmed milk                   ZINC database screened using program ROCS <br /> PBS/0.1% Tween-20 (blocking solution), followed incubation                 (version 2.3.1) compounds good shape similarity <br /> with anti-His-tag rabbit antibody (MicroMol-413) (1:5000)             (measured 3D Tanimoto) similar functional group overlap <br /> combination anti-TbAS-A rabbit antibody (1:1000)                query molecule. L-cysteine-S-sulfate (Sigma; PubChem <br /> anti-aldolase rabbit antibody (1:5000) blocking solution 4uC            Substance ID 24892471) used following conditions: <br /> O/N, respectively. Blots washed PBS/0.1% Tween-20                   2.5 mM aspartate, 1.25 mM ATP, 12.5 mM ammonia, and <br /> (3615 min). Horseradish peroxidase-conjugated goat anti-rabbit                8.4 mM Mg2+. The characterization mechanism of <br /> IgG (Amersham) (1:5000 1 h, room temperature) blocking              inhibition consisted determination Km Vmax each <br /> buffer used secondary antibody. The membranes                 substrate, presence inhibitor concentrations (0.025, <br /> developed using SuperSignal WestPico Chemiluminescent Sub-                    0.050, 0.1 0.2 mM). The following substrate concentration <br /> strate (Pierce). ImageJ software (version 1.43u) used                 ranges 1.25&#226;&#8364;&#8220;10 1.25&#226;&#8364;&#8220;50 mM used aspartate and <br /> protein bands semi-quantification.                                            ammonia, respectively, determine Km ATP, range <br />                                                                               0.625 10 mM (TbAS-A) 0.3125 5 mM (TcAS- <br /> Enzyme assays                                                                 A) assayed. Ki determined &#226;&#8364;&#732;&#226;&#8364;&#732;Km app Method&#226;&#8364;&#8482;&#226;&#8364;&#8482;[43]. <br />    AS activity assessed quantification asparagine <br /> formation [41]. The reactions performed total                    AS-A protein alignments TbAS-A/TcAS-A homology <br /> 150 ml enzyme assay mixture 85 mM Tris-HCl (Sigma)                      models <br /> containing aspartate (Sigma), ammonia (Sigma), ATP (Sigma)                   EcAS-A, TbAS-A TcAS-A protein alignments were <br /> 8.4 mM Mg2+ (Sigma). Following incubation set times 37uC,              performed using ClustalW [44]. Aline, Version 011208 [45], was <br /> enzymatic reactions terminated boiling 4 min,                used editing protein sequence alignments preparing Fig. 1. <br /> centrifuged maximum speed 30 s. 100 ml reaction <br />                                                                               TbAS-A TcAS-A homology models obtained with <br /> mixture supernatant added 900 ml ninhydrin 0.05% in <br />                                                                               SWISS-MODEL, using EcAS-A crystal structure (Protein Data <br /> ethanol. The resulting mixtures boiled 100uC 5 min, <br />                                                                               Bank (PDB) accession <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> 12AS [24]) template (percentage <br /> then centrifuged 30 s maintained ice. 300 ml clear <br />                                                                               sequence identity 56% 57%, respectively) [46&#226;&#8364;&#8220;48]. The <br /> supernatant fluids transferred 96-well plates, the <br />                                                                               3D structures rendered PyMOL (The PyMOL Molecular <br /> absorbance 340 nm determined [41]. Based reaction <br />                                                                               Graphics System, Version 1.3, Schro&#194;&#168;dinger, LLC). <br /> linearity studies, 7.5 mg enzyme 15 min incubation at <br /> 37uC selected final conditions. To determine Kms, the <br /> concentrations substrates varied following ranges:             Generation TbAS-A RNAi cell lines <br /> 1.25&#226;&#8364;&#8220;20 mM (aspartate), 0.78&#226;&#8364;&#8220;50 mM (ammonia) 0.62&#226;&#8364;&#8220;                           The &#226;&#8364;&#732;&#226;&#8364;&#732;stuffer strategy&#226;&#8364;&#8482;&#226;&#8364;&#8482; used generate RNAi-mediated AS- <br /> 10 mM (ATP), remaining substrates concentrations                    A depletion. First, TbASA fragment (amplified sense <br /> were excess ([aspartate] .20 mM, [ATP] .10 mM,                         oligo BglII &#226;&#8364;&#8220; SphI linker 59 - GAGAAGATCTGCA- <br /> [ammonia] .50 mM). Km glutamine determined using                    TGCGCGACGACGGTTATTCGTCATAC - 39, anti- <br /> concentration range 1.5625&#226;&#8364;&#8220;25 mM, ATP aspartate                  sense oligo EcoRI &#226;&#8364;&#8220; SalI 59 &#226;&#8364;&#8220; CGGAATTCGTCGA- <br /> were maintained excess. Measurements performed                     CACTCCGTTTTTCGGATTGCGGC &#226;&#8364;&#8220; 39) cloned twice in <br /> triplicate, initial rate analyzed obtain values Vmax        opposite direction sides &#226;&#8364;&#732;stuffer&#226;&#8364;&#8482; fragment the <br /> and Km curve fitting using GraphPad Prism (5.0 version).                   pHD1144 vector (also digested SphI SalI) (Fig. S1A). The <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                       3                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                           Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 2. Homology models AS-A trypanosomes. (A) Ribbon representation EcAS-A colored according sequence similarity <br /> with TbAS-A TcAS-A shown Fig. 1. (B) Superposition EcAS-A structure (grey) (PDB accession <span id='am-165' about='Thesaurus:code' typeof='owl:Thing'>code</span> 12AS), TbAS-A (blue) TcAS-A <br /> (purple) homology models (obtained SWISS-MODEL server, using PDB 12AS template). A small structurally divergent region marked <br /> by dashed rectangle. Ligand color schemes: asparagine shown yellow (oxygen, red; nitrogen blue) AMP shown brown (oxygen, red; <br /> nitrogen blue; phosphorous orange). <br /> doi:10.1371/journal.pntd.0002578.g002 <br />  <br />  <br /> resulting [(target)-stuffer-(reverse-complement target)] construct         4 groups BALB/c mice (6&#226;&#8364;&#8220;8 weeks old, n = 4) (Harlan <br /> obtained HindIII BglII digestion, generates            Laboratories, United Kingdom) infected intraperitoneal <br /> stem-loop RNA, cloned pHD1145 (also digested                 injection 104 T. brucei bloodstream forms. 2 groups injected <br /> HindIII BglII) (Fig. S1B). The final construct linearized          wt strain (with tetracycline) 2 groups <br /> with NotI 10 mg DNA transfected 26107/ml                   injected RNAi cell line (with tetracycline). <br /> bloodstream form cell line carrying pHD1313 plasmid (contains              48 h prior infection 2 RNAi induced groups given <br /> two copies tet repressor phleomycin resistance                doxycycline (treated 1 mg/ml doxycycline hyclate 5% <br /> cassette) electroporation using Amaxa Basic Parasite Nucleo-            sucrose containing water). The 2 non-induced groups given <br /> fector Kit (Lonza). Transcription occurs induction                 standard water. To evaluate virulence RNAi induced <br /> tetracycline (100 ng/ml), producing mRNA homologs                 parasites mice reduced plasmatic levels asparagine, <br /> the target gene. Stable individual clones selected 5 7         animals treated 50 IU E. coli L- asparaginase <br /> days transfection 7.5 mg/ml hygromycin.                      (ProSpec-Tany TechnoGene) 48 h injection every <br />                                                                            48 h. According literature, L-asparagine be <br /> In vitro analysis TbAS-A RNAi                                           detected blood 48 h following intravenous injection E. <br />    To analyse growth, T. brucei RNAi cell line cells expressing        coli L-asparaginase, dose 50 IU/mouse [50]. Mice were <br /> the tet repressor (wt), seeded 26105 cells/ml              monitored day general appearance behaviour. <br /> complete HMI9 medium, presence absence                       Parasitemia monitored daily fifth day post-infection, <br /> tetracycline. Cell growth monitored microscopically               using tail-vein blood, haematocytometer microscope. <br /> haemocytometer (Marienfeld) culture diluted                Animals parasitemia greater 108 parasites/ml were <br /> 26105 cells/ml daily. The protocol repeated complete           euthanized, previous studies established levels <br /> HMI9 medium basal IMDM asparagine, complete                   consistently lethal 24 h. <br /> HMI9 medium basal IMDM asparagine supple- <br /> mented 6.76104 nM asparagine (levels human                Immunofluorescence <br /> plasma [49]), complete HMI9 medium basal IMDM                       T. brucei bloodstream forms log-phase cultures, or <br /> without asparagine supplemented 1.676105 nM aspara-                RNAi, fixed m-Chamber 12 (Ibidi) for <br /> gine (levels normal medium).                                      15 min, room temperature, PBS containing 3% p-formalde- <br />                                                                            hyde, washed twice PBS, permeabilized PBS <br /> In vivo analysis TbAS-A RNAi                                            containing 0.1% Triton X-100. Fixed cells incubated for <br />   Wild-type transgenic bloodstream T. brucei parasites            60 min PBS containing 10% FCS room temperature (RT), in <br /> cultured absence selecting drugs (hygromycin                 humidified atmosphere, washed twice PBS/2% FCS. <br /> phleomycin) 24 h, tetracycline added. After         Cells incubated primary rat rabbit polyclonal <br /> 48 h, parasites inoculated mice. For experiment,              antibody TbAS-A (1:100 1:5000 respectively, both <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                    4                       December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                   Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 3. Analysis purified recombinant TcAS-A/TbAS-A AS-A expression T. brucei life cycle stages. Purified TcAS-A (A) <br /> and TbAS-A (B) recombinant proteins analyzed using 12% SDS-PAGE visualized using Coomassie blue staining. MW, molecular weight <br /> marker. Rabbit anti-histidine monoclonal antibody (1:1000) used immunoblotting assays TcAS-A (A) TbAS-A (B) purified <br /> recombinant proteins. (C) AS-A expression T. brucei life-cycle stages: 10 mg protein BS (bloodstream forms) PC (procyclic forms) <br /> lysates analysed Western blot. Aldolase used loading control. Rabbit anti-TbAS-A anti-aldolase polyclonal antibodies used <br /> for protein detection. The results representative independent experiments. <br /> doi:10.1371/journal.pntd.0002578.g003 <br />  <br /> diluted blocking solution) overnight 4uC, followed                    In case TbAS-A immunolocalization, T. brucei wt bloodstream <br /> washes PBS/2% FCS. Subsequently, cells incubated                      forms cells co-stained using rat anti-TbAS-A antibody (1:100 <br /> with Alexa Fluor 647 conjugated goat anti-rat Alexa Fluor 488                diluted blocking solution), rabbit anti-aldolase antibody (1:5000 <br /> conjugated goat anti-rabbit secondary antibodies (Molecular                     diluted blocking solution), anti-BiP antibody (kindly provided by <br /> probes Life technologies) (1:500 diluted blocking solution)             Dr. Jay Bangs, 1:500 diluted blocking solution), anti-enolase <br /> for 1 h RT humidified atmosphere, washed twice                    antibody (kindly provided Dr. Paul Michels, 1:5000 diluted in <br /> with PBS. Next, slides stained mounted                        blocking solution) anti-GRASP antibody (kindly provided by <br /> Vectashield-DAPI (Vector Laboratories, Inc.). Images                       Dr. Graham Warren, 1:200 diluted blocking solution). Alexa <br /> captured using fluorescence microscope AxioImager Z1                        Fluor 647 conjugated goat anti-rat (1:500) Alexa Fluor 488 <br /> software Axiovision 4.7 (Carl Zeiss, Germany). Pseudo-coloring                  conjugated goat anti-rabbit (1:500) used secondary <br /> of images carried using ImageJ software (version 1.43u).                antibodies. Staining MitoTracker Orange (Invitrogen) <br />                                                                                 followed Alexa Fluor 488 conjugated goat anti-rabbit (1:500), <br />  <br />  <br />  Table 1. TbAS-A TcAS-A kinetic parameters aspartate, ATP ammonia. <br />  <br />  <br />  Species              Substrate                 Km (mM)                   Vmax 61023(mM.s21)        kcat (s21)               kcat/Km (M21.s21) <br />  <br />  T. brucei            Aspartate                 5.3960.31                 6.7260.14                 5.3160.11                9.856102 <br />                       ATP                       1.8060.32                 7.8860.48                 6.2260.38                3.466103 <br />                       Ammonia                   5.5560.41                 7.2160.16                 5.6960.13                1.036103 <br />  T. cruzi             Aspartate                 6.4562.05                 3.0960.40                 2.4160.32                3.746102 <br />                       ATP                       0.7260.01                 4.4260.02                 3.4560.02                4.796103 <br />                       Ammonia                   7.7561.55                 3.1460.21                 2.4560.17                3.166102 <br />  <br />  The values means 6 standard deviation obtained 3 independent experiments. <br />  doi:10.1371/journal.pntd.0002578.t001 <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                         5                       December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                 Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  Table 2. TbAS-A TcAS-A kinetic parameters                               sequence alignment, performed using ClustalW [44], AS-A from <br />  glutamine.                                                                      T. brucei (Tb927.7.1110, NCBI-GeneID:3658321), T. cruzi <br />                                                                                  (Tc00.1047053503625.10, NCBI-GeneID:3534325) E. coli <br />                                                                                  (NCBI-GeneID:948258) shown Figure 1. The amino acid <br />                             Vmax 61023                                           residues known involved active-site formation E. coli <br />  Species     Km(mM)         (mM.s21)   kcat(s21)       kcat/Km (M21.s21)         [24] highly conserved sequences (Fig. 1). <br />                                                                                  Protein alignments demonstrated 58% similarity EcAS-A versus <br />  T. brucei   8.2061.70      7.0660.61    5.5860.48     6.806102 <br />                                                                                  TbAS-A, 60% EcAS-A versus TcAS-A, 63% TbAS-A <br />  T. cruzi    15.3363.66     4.3360.53    3.3860.41     2.206102                  versus TcAS-A. <br />  The values means 6 standard deviation obtained 3 independent          Like EcAS-A, TbAS-A TcAS predicted dimeric, as <br />  experiments.                                                                    seen superimposed homology models EcAS-A crystal <br />  doi:10.1371/journal.pntd.0002578.t002                                           structure [24] (Fig. 2A). The structurally divergent region <br />                                                                                  (area marked dashed rectangle) (Fig. 1, 2B), present both <br /> as secondary antibody. The labelling parasites                         TbAS-A (from residues Q232 S250) TcAS-A (from residues <br /> MitoTracker adding 250 nM cell culture                        D232 S247), absent EcAS-A. This region distant from <br /> medium (without FCS) 30 minutes, prior washing, fixing                    enzyme active site dimer interface functional <br /> and staining using protocol described above. Images                     structural significance unknown. The amino acids <br /> captured using confocal microscope Leica TCS SP5II                       involved asparagine binding strictly conserved, in <br /> LAS 2.6 software (Leica Microsystems, Germany). Again, image                     AMP binding pocket, majority residues are <br /> analysis using ImageJ version 1.43U software.                           conserved, residues (Fig. 1). In EcAS-A, E103 <br />                                                                                  (D106 E106 TbAS-A TcAS-A, respectively) L109 <br /> Digitonin permeabilization                                                       (I112 T112 TbAS-A TcAS-A, respectively) (Fig. 1) are <br />    For sample condition, 1.06107 bloodstream cells                     directly involved polar interactions nucleotide <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span>, <br /> washed cold trypanosome homogenization buffer                          form outer wall binding site [24]. The main <br /> (THB), containing 25 mM Tris (Sigma), 1 mM EDTA (Sigma)                          chain L249 EcAS-A (V271 L268 TbAS-A TcAS- <br /> and 10% sucrose (Sigma), pH = 7.8. Just cell lysis, leupeptin             A, respectively) directly involved hydrogen bonds ribose <br /> (Sigma) (final concentration 2 mg/ml) different digitonin                 AMP, different chains leucine valine <br /> (Calbiochem) quantities (final concentrations 5, 12.5, 25, 50,                affect shape AMP binding site. <br /> 100, 150 200 ug/ml) added 500 ml cold THB, for <br /> cell pellet resuspension. Untreated cells (0 mg/ml digitonin)             Trypanosome AS-A catalyze asparagine synthesis using <br /> those completely permeabilized (total release, result                     ammonia glutamine nitrogen donors <br /> incubation 0.5% Triton X-100) used controls. Each                        TbAS-A TcAS-A coding sequences cloned the <br /> sample incubated 60 min ice, centrifuged                      bacterial expression vector pET28a. Histidine-tagged fusion <br /> 2000 rpm, 4uC, 10 min. Supernatants transferred new                  proteins purified non-denaturing conditions (Fig. 3A, <br /> chilled tubes 500 ml cold THB added pellet                B). As expected, recombinant proteins recognized by <br /> then mixed. All fractions analysed Western blot                  anti-His Tag monoclonal antibody (Fig. 3A, B). Rabbit polyclonal <br /> described above.                                                                 antibodies produced recombinant TbAS-A recognized the <br />                                                                                  protein total extracts different parasite developmental <br /> Cell cycle analysis                                                              stages, bloodstream forms (mammalian host parasite stage) and <br />    T. brucei bloodstream forms analyzed flow cytometry               procyclic forms (insect vector parasite stage) (Fig. 3C). <br /> DNA content following RNAi induction. Cells collected                       The capacities TbAS-A TcAS-A produce asparagine <br /> centrifugation washed twice PBS containing 2% FCS.                      aspartate presence ATP, ammonia Mg2+ were <br /> Each 26106 cells resuspended 1 mL PBS/2% FCS                      determined using specific quantitative colorimetric assay L- <br /> 3 mL cold absolute ethanol added vortexing. Cells                   asparagine [41]. The pH optimum 7.6, detectable <br /> were fixed 1 hour 4uC washed twice PBS. 1 mL               activity 6.0 9.0 (data shown). Mg2+ essential co- <br /> staining solution [3.8 mM sodium citrate dehydrate (Sigma),                      factor TbAS-A TcAS-A (data shown), previously <br /> 50 mg/mL propidium iodide (Sigma), 0.5 mg/mL RNAse A (Sigma)                     described EcAS-A [22]. We included 8.4 mM Mg2+ final <br /> in PBS] added cell pellets vortex. Samples                   reaction mixture. Lower concentrations (2, 4 6 mM) gave <br /> analysed FACS (Becton Dickinson) incubation 4uC                lower activity increased concentrations (up 16 mM) <br /> 30 min. Data analyzed FlowJo software (Ashland, OR).                      resulted substantial activity improvement (data shown). <br />                                                                                  TbAS-A TcAS-A showed similar Kms aspartate and <br /> Statistical analysis                                                             ammonia (p.0.01), TcAS-A showed higher Km ATP than <br />    One-way ANOVA two-tailed Student&#226;&#8364;&#8482;s test used                     TbAS-A (p = 0.0042) (Table 1). ATP substrate required for <br /> statistical analysis. Statistical analysis performed using                   generation b-aspartyl adenylate intermediate, which <br /> GraphPad Prism Software (version 5.0), p values#0.01                    reacts ammonia, releasing asparagine. In absence, the <br /> considered statistically significant. Asterisks indicate statis-           reaction did occur (data shown). To surprise, both <br /> tically significant differences (*** p#0.001, ** p#0.01).                        TbAS-A TcAS-A use glutamine nitrogen donor <br />                                                                                  (Table 2). TbAS-A showed higher Km nitrogen donor than <br /> Results                                                                          TcAS-A, statistically significant (p.0.01). Both <br />                                                                                  enzymes present higher Km values ammonia for <br /> Conservation AS-A trypanosomes                                             glutamine, differences statistically significant <br />    One open reading frame code putative AS-A present               (p.0.01) (Table 1 2). TbAS-A higher Vmax TcAS- <br /> in genomes T. cruzi CL Brener Non-Esmeraldo-like T.                   A, ammonia (p,0.0001) glutamine (p = 0.0043) <br /> brucei TREU927 (http://tritrypdb.org) [29&#226;&#8364;&#8220;31]. A protein multiple                dependent-activities (Table 1 2). TbAS-A similar catalytic <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                          6                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                    Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org   7   December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                 Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br /> Figure 4. T. brucei T. cruzi AS-A vitro inhibition. (A) Inhibition (%) T. brucei T.cruzi AS-A activity L-cysteine-S-sulfate (L-CS). (B) <br /> Plot Kmapp/Vmax versus L-CS concentration established; Ki corresponds symmetric value X-axis intersection. (C-H) Plots showing the <br /> effect different L-CS concentrations inverse initial velocity versus inverse concentrations ATP, aspartate ammonia <br /> for TbAS-A (C, E G, respectively) TcAS-A (D, F H, respectively) enzymes. Error bars indicate standard deviation means two <br /> replicates data shown representative independent experiments. <br /> doi:10.1371/journal.pntd.0002578.g004 <br />  <br /> rates glutamine ammonia-dependent activities                      aspartate ammonia observed (p.0.01) (Fig. 4E, F, G, H), <br /> (p.0.01), TcAS-A presented slightly higher,                  suggesting inhibition exclusively ATP binding <br /> statistically significant, rate glutamine-dependent activity               interference (p#0.01) . Ki values 137 128.9 mM were <br /> (p.0.01) (Table 1 2). The high conservation active                  determined TbAS-A TcAS-A, respectively (Fig. 4B). <br /> sites small amino acid differences identified the <br /> homology models allow accurate structural interpreta-                AS-A localizes cytosol T. brucei bloodstream <br /> tion small differences observed. Indeed,               forms <br /> been slight differences proportion protein              The subcellular localization TbAS-A determined by <br /> correctly folded.                                                              immunofluorescence digitonin fractionation bloodstream <br />    L-cysteine-S-sulfate, considered putative AS-A inhibitor             forms. As expected, induction RNAi resulted decrease in <br /> a virtual screening, inhibited enzymes, IC50s 126             fluorescence intensity (Fig. S2A, B, C). TbAS-A the <br /> 100 mM TbAS-A TcAS-A, respectively (Fig. 4A). For                 cytosol, revealed colocalization cytosolic enzyme <br /> enzymes, kinetic characteristics suggested competition                enolase [51] (Fig. 5A) colocalization aldolase, BiP, <br /> ATP binding (Fig. 4C, D). No changes Kms Vmaxs                  GRASP mitotracker (Fig. S3), markers glycosomes [52], <br />  <br />  <br />  <br />  <br /> Figure 5. AS-A subcellular localization T. brucei bloodstream forms. (A) Immunofluorescence analysis confocal microscopy TbAS-A <br /> (red) enolase (green) bloodstream forms. DAPI locate nuclear kinetoplast mitochondrial DNA (blue). Bar, 5 mm. Images maximal Z- <br /> projections 50 contiguous stacks separated 0.1 mm. (B) For digitonin permeabilization, selected supernatant (S) pellet (P) fractions obtained <br /> at different digitonin concentrations subjected Western-blot analysis probed rabbit antisera raised TbAS-A, enolase <br /> (cytoplasmic marker) aldolase (glycosome marker). Data shown representative independent experiments. Untreated cells (0 mg/ml of <br /> digitonin) completely permeabilized Triton X-100 0.5% used controls. <br /> doi:10.1371/journal.pntd.0002578.g005 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        8                         December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 6. In vitro effect RNAi-mediated AS-A down-regulation T. brucei bloodstream forms. Growth curve wt versus a <br /> representative AS RNAi cell line performed unmodified medium [complete HMI-9 (HMI-9c) basal IMDM] (A), modified medium [HMI-9c <br /> with basal medium asparagine - HMI-9c (basal IMDM w/o Asn)] (C), modified medium supplemented 6.76104 nM asparagine [HMI-9c <br /> (basal IMDM w/o Asn + 6.76104 nM Asn)] (E), modified medium supplemented 1.676105 nM asparagine [HMI-9c (basal IMDM w/o Asn + <br /> 1.676105 nM Asn)] (G). Circles squares represent wild-type growth absence presence tetracycline, respectively, triangles and <br /> down triangles represent clones growth absence presence tetracycline, respectively. Cumulative cell numbers plotted product <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                       9                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br /> of cell number total dilution. Error bars indicate standard deviation. The effect RNAi AS-A protein levels analyzed Western blots <br /> with extracts noninduced tet(2), RNAi-induced tet(+) cells, isolated unmodified medium (B), modified medium (complete HMI-9 medium <br /> with basal IMDM asparagine) (D), modified medium supplemented 6.76104 nM asparagine (F), modified medium supplemented with <br /> 1.676105 nM asparagine (H). Cells collected day 1, 2, 6 10 RNAi induction. Results representative independent <br /> experiments. <br /> doi:10.1371/journal.pntd.0002578.g006 <br />  <br />  <br />  <br /> endoplasmic reticulum [53], Golgi mitochondria compart-                   HMI-9 medium asparagine fetal calf serum, <br /> ments [54], respectively. Controls performed rat rabbit               growth impaired, increase proportion cells in <br /> pre-immune sera secondary antibody alone, showed                       G0/G1 (Fig. 6C 7B, C). Presumably asparagine the <br /> detectable signal (data shown). Digitonin fractionation              serum allowed slower growth. Levels asparagine usually <br /> resulted similar profiles AS-A enolase (cytosolic                  human serum (6.76104 nM) [49], somewhat <br /> marker) similarity aldolase (glycosomes marker)                     lower normal medium (1.676105 nM; IMDM - Iscove&#226;&#8364;&#8482;s <br /> (Fig. 5B).                                                                    modified Dulbecco&#226;&#8364;&#8482;s basal medium Gibco Invitrogen), were <br />                                                                               sufficient overcome defect (Fig. 6E, G). In complete HMI-9 <br /> AS-A knockdown makes T. brucei bloodstream forms                              medium, asparagine fetal calf serum, the <br /> dependent extracellular asparagine                                         growth defects induced RNAi clones abrogated day 5 <br />    To study biological role AS-A T. brucei bloodstream              post-induction (Fig. 6C, 7D), percentage cells GO/ <br /> forms, cells stably transfected <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>                 G1 S phases cell cycle return ones non- <br /> plasmid construct. RNAi asparagine synthetase A                   induced cells (Fig. 7A), suggesting appearance RNAi <br /> induced normal medium (complete HMI-9) adding                           revertants, visible Western blot (Fig. 6D). Similar <br /> tetracycline. No difference observed cell proliferation                reversion evade lethal RNAi trypanosomes seen <br /> between induced non-induced cells (Fig. 6A), AS-A                times [55]. In presence asparagine, low AS-A <br /> protein reduced &lt;13% normal level 48 hours               levels maintained (Fig. 6B, F H). <br /> (Fig. 6B). When, however, AS-A-depleted cells grown in <br />  <br />  <br />  <br />  <br /> Figure 7. In vitro effect RNAi-mediated AS downregulation T. brucei cell cycle. DNA content &lt;26106 wt RNAi cell line parasites <br /> grown completed HMI-9 medium basal IMDM asparagine, collected tetracycline induction, day 0 (A), day 1 (B), 3 <br /> (C), 5 (D) post-induction, cell cycle analysis. Samples analysed flow cytometry percentage cells phase cell <br /> cycle determined using FlowJo software. Each bar represents average replicates. Error bars indicate standard deviation these <br /> measurements. The statistics calculated one-way ANOVA (*** p#0.001 ** p#0.01). <br /> doi:10.1371/journal.pntd.0002578.g007 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                      10                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                  Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 8. In vivo effect RNAi TbAS-A. (A) Groups 4 mice infected intraperitoneally, 16104 control wt parental cell line <br /> (open squares open diamonds) representative AS RNAi clone (open triangles open circles). The mice untreated (open <br /> squares open triangles) treated 1 mg/ml doxycycline (Dox) (open diamonds open circles) water supply. Parasitemia was <br /> quantified times indicated. Error bars indicate standard deviation means mice wt cell lines, mice case AS <br /> RNAi cell lines. The detection limit assay 56104 trypanosomes ml blood. Mice euthanized parasitemia reached 16108 <br /> cells/ml. (B) Kaplan&#226;&#8364;&#8220;Meier survival analysis mice infected wt parental cell line AS RNAi cell line absence presence doxycycline. <br /> Data representative independent experiments. Western blot analysis AS-A levels trypanosomes injected mice, 48 h in <br /> vitro tetracycline induction (C), trypanosomes isolated mice, euthanized (D). <br /> doi:10.1371/journal.pntd.0002578.g008 <br />  <br />  <br />    Normal T. brucei parasites showed statistically significant           appeared course infection asparaginase-treated, <br /> slower growth conditions asparagine limitation (p#0.01;                untreated, mice (Fig. 9F). Parasites extracts wt <br /> data shown) (compare Fig. 6C, A, E, G). It                infected mice used controls (Fig. 9C). <br /> possible parasite AS, requires <br /> external asparagine optimal vitro growth.                                Discussion <br />  <br /> AS-A dispensable T. brucei infectivity mice                              In study demonstrated trypanosomes AS-A use both <br />    To test AS-A important parasite infection                ammonia glutamine nitrogen donors ATP dependent <br /> disease model, groups mice (n = 4) inoculated              conversion aspartate asparagine. Such hybrid activity was <br /> parental cell line, groups RNAi cells. Two mice              previously demonstrated type B enzymes, prefer <br /> groups fed water containing doxycycline induce                     glutamine ammonia [15&#226;&#8364;&#8220;19]. The small differences Km of <br /> down-regulation TbAS-A, remaining mice kept                   TbAS-A TcAS-A ammonia glutamine (1.5 2 fold, <br /> as non-induced controls. Within days inoculation, mice               respectively) lower difference AS-B <br /> from different groups developed high levels parasitemia                  enzymes, exception human enzyme, has <br /> (Fig. 8A), euthanized seven days               similar affinities [16&#226;&#8364;&#8220;20]. Purified E. coli AS-A used only <br /> post-infection (Fig. 8B). The results confirm asparagine            ammonia nitrogen source, results Klebsiella aerogenes <br /> mouse blood sufficient compensate &lt;87% downreg-                   suggested AS-A preferentially uses ammonia substrate <br /> ulation AS-A (Fig. 8C, D).                                                   [5,21,22]. The conclusions AS-A enzymes Gram- <br />    To assess contribution blood L-asparagine vivo, mice               negative organisms relied biochemical genetic <br /> were treated L-asparaginase [50]. L-asparaginase treatment                 analysis, given technical limitations time, fact <br /> did affect growth normal parasites mice (Fig. 9A)                 background enzyme activity seen absence both <br /> consequently did extend animal survival (Fig. 9B). However L-               ammonia glutamine, re-examination bacteria would <br /> asparaginase treatment mice infected TbAS-A RNAi-                       worthwhile. Moreover overall Km values trypanosomes <br /> induced parasites caused decrease parasitemia (Fig. 9D),               AS-A aspartate, 6 20 fold higher ones found <br /> thus leading increase mice survival (Fig. 9E). Even so,            literature prokaryotic asparagine synthetase type A <br /> infection resulted death. As happened vitro, RNAi revertants              [5,21,22]. Trypanosoma AS-A structures solved <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        11                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                Trypanosoma Asparagine Synthetase A <br />  <br />  <br />  <br />  <br /> Figure 9. In vivo RNAi TbAS-A mice undergoing E. coli L-asparaginase treatment. Time course parasitemia groups <br /> of 4 mice infected intraperitoneally, 16104 RNAi induced [using 1 mg/ml doxycycline, Dox (+)] wt parental cell line (A) representative <br /> AS RNAi cell line (D) untreated (open squares) treated daily 2 days (open diamonds) 50 IU E. coli L-asparaginase/mouse. <br /> Parasitemia quantified peripheral tail blood times indicated. Error bars indicate standard deviation, data representative two <br /> independent experiments using different AS RNAi clones. The detection limit assay 56104 trypanosomes/ml blood. Mice culled <br /> when parasitemia reached 16108 cells/ml. Kaplan&#226;&#8364;&#8220;Meier survival plot mice infected RNAi induced wt cell line (B) AS RNAi cell line (E) either <br /> untreated (open squares) treated (open diamonds) E. coli L-asparaginase. (F) Western blot analysis AS-A levels trypanosomes isolated <br /> from mice, euthanized. (C) Wt cell extracts, collected mice culled, use negative control. <br /> doi:10.1371/journal.pntd.0002578.g009 <br />  <br />  <br /> and protein homology models completely enlightening,               extracellular medium biosynthesis AS-A. <br /> nevertheless speculate differences result              Bloodstream form proliferation, vitro vivo, only <br /> the fact parasite enzymes expressed purified                  significantly affected asparagine sources compro- <br /> recombinant proteins bacteria purified directly                mised. Also consistent idea, published RNAi <br /> trypanosomes extracts. As consequence, differences protein                screen, slight (possibly insignificant) growth disadvantage <br /> post-transcriptional processing and/or changes protein confor-              seen bloodstream forms depleted AS-A [56]. In same <br /> mation excluded.                                                     way, vitro results corroborated previous studies, as <br />    Our results suggest bloodstream-form parasites rely             mammalian cells low expression AS similarly <br /> major sources asparagine ensure normal proliferation: uptake             susceptible asparagine depletion [57&#226;&#8364;&#8220;59], asparaginase <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                       12                        December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                                     Trypanosoma Asparagine Synthetase A <br />  <br />  <br /> isolated E. coli Erwinia carotovora act potent anti-                              inducible EP1 promoter hygromycin resistance cassette, <br /> leukemic agents [60].                                                                         insertion ribosomal spacer). <br />    In trypanosomes genome second open reading                                  (TIF) <br /> frame (Tb927.3.4060) coding putative AS domain. However                                 Figure S2 Validation antibodies TbAS-A. <br /> this apparently classical AS, despite presence good                         Immunofluorescence analysis T. brucei wt representative <br /> Pfam AS domain (PF0073) C-terminus. A BLASTp search                                    AS RNAi clone grown presence absence tetracycline. <br /> using T. brucei sequence revealed variety proteins                                RNAi induced uninduced cells grown 48 h, then <br /> unknown function aligned AS domain,                              fixed probed rat polyclonal anti-TbAS-A (A) rabbit <br /> also N-terminal region, contains N-terminal                                      polyclonal anti-TbAS-A (B) antibody co-stained DAPI. <br /> aminohydrolase domains. Best matches originate extremely                                 Bars, 5 mm. Quantification TbAS-A fluorescence levels in <br /> diverse eukaryotes including plant, alga, member                                    induced cells (AS RNAi tet(+), n = 30) uninduced cells (AS <br /> the fungi amoeba. BLASTp Saccharomyces                                     RNAi tet(2), n = 30), using rat rabbit polyclonal anti- <br /> cerevisiae predicted proteome yielded YML096W, reci-                                  TbAS-A antibodies (C). Data representative independent <br /> procal BLASTp trypanosome genome gave                                           experiments using different clones. ImageJ software (version <br /> Tb927.3.4060 best match. The function YML096W                                    1.43u) used fluorescence quantification. p value was <br /> known, trypanosome RNAi screen growth defect                                  calculated Student&#226;&#8364;&#8482;s t test (*** p#0.001 ** p#0.01). <br /> seen Tb927.3.4060 [56].                                                                   (TIF) <br />    The capacity trypanosomes grow using asparagine from <br /> the extracellular environment, lack growth defect                             Figure S3 TbAS-A cellular localization T. brucei <br /> the levels AS-A reduced, combination                                  bloodstream forms. Immunofluorescence analysis confocal <br /> therapy using TbAS-A inhibitor extracellular                                    microscopy TbAS-A (red) bloodstream forms. Aldolase <br /> asparagine depletor (e.g. L-asparaginase) asparagine                                    (glycosome marker), GRASP (golgi marker), BiP (endoplasmic <br /> transport blocker inhibit parasite growth. This                                  reticulum marker) MitoTracker (labels mitochondria) in <br /> appropriate African sleeping sickness treatment. A combina-                               green. DAPI locate nuclear kinetoplast mitochondrial DNA <br /> tion absolutely required simultaneous activities                                  (blue). Bars, 5 mm. Images maximal Z-projections 50 <br /> different drugs wide open resistance development,                                 contiguous stacks separated 0.1 mm. <br /> and drug combination including intravenously-introduced                                    (TIF) <br /> enzyme likely expensive logistically <br /> inappropriate treatment African trypanosomiasis. Moreover,                             Acknowledgments <br /> L-asparaginase treatment cancer results adverse                                 We like thank Dr. Jay Bangs University Wisconsin- <br /> events [61&#226;&#8364;&#8220;63]. We conclude AS-A good                                 Madison Medical School, USA, providing BiP antibody, Dr. Paul <br /> candidate sleeping sickness drug target. Its role Trypanosoma                         Michels Universite&#194;&#180; Catholique Louvain, Belgium, sending us <br /> cruzi, however, remains established.                                                    enolase antibody Dr. Graham Warren Yale University School of <br />                                                                                               Medicine, USA, GRASP antibody. We thank Claudia Helbig <br />                                                                                               assistance Heidelberg. <br /> Supporting Information <br /> Figure S1 RNAi vectors used generate RNAi-mediat-                                          Author Contributions <br /> ed TbAS-A downregulation. (A) pHD1144 vector stem-                                        Conceived designed experiments: IL JF JT ACdS. Performed the <br /> loop cloning (pSP72 vector stuffer fragment); (B) pHD1145                              experiments: IL JF NS JT. Analyzed data: IL JF SMR CC JT ACdS. <br /> inducible polymerase I vector insertion ready-made stem-                               Contributed reagents/materials/analysis tools: CC SMR NR. Wrote the <br /> loops (pHD677 vector T7 promoter                                        paper: IL JF CC SMR JT ACdS. <br />  <br /> References <br />  1. Richards NG, Schuster SM (1998) Mechanistic issues asparagine synthetase               10. Andrulis IL, Chen J, Ray PN (1987) Isolation human cDNAs asparagine <br />     catalysis. Adv Enzymol Relat Areas Mol Biol 72: 145&#226;&#8364;&#8220;198.                                      synthetase expression Jensen rat sarcoma cells. Mol Cell Biol 7: 2435&#226;&#8364;&#8220; <br />  2. Blaise M, Frechin M, Olieric V, Charron C, Sauter C, et al. (2011) Crystal                    2443. <br />     structure archaeal asparagine synthetase: interrelation aspartyl-tRNA         11. Andrulis IL, Shotwell M, Evans-Blackler S, Zalkin H, Siminovitch L, et al. <br />     asparaginyl-tRNA synthetases. J Mol Biol 412: 437&#226;&#8364;&#8220;452.                                    (1989) Fine structure analysis Chinese hamster AS gene encoding <br />  3. Roy H, Becker HD, Reinbolt J, Kern D (2003) When contemporary aminoacyl-                      asparagine synthetase. Gene 80: 75&#226;&#8364;&#8220;85. <br />     tRNA synthetases invent cognate amino acid metabolism. Proc Natl Acad               12. Ramos F, Wiame JM (1980) Two asparagine synthetases Saccharomyces <br />     Sci U S A 100: 9837&#226;&#8364;&#8220;9842.                                                                     cerevisiae. Eur J Biochem 108: 373&#226;&#8364;&#8220;377. <br />  4. Nakamura M, Yamada M, Hirota Y, Sugimoto K, Oka A, et al. (1981)                          13. Merchant SS, Prochnik SE, Vallon O, Harris EH, Karpowicz SJ, et al. (2007) <br />     Nucleotide sequence asnA gene coding asparagine synthetase E. coli              The Chlamydomonas genome reveals evolution key animal plant <br />     K-12. Nucleic Acids Res 9: 4669&#226;&#8364;&#8220;4676.                                                         functions. Science 318: 245&#226;&#8364;&#8220;250. <br />  5. Reitzer LJ, Magasanik B (1982) Asparagine synthetases Klebsiella aerogenes:            14. Coruzzi GM (2003) Primary N-assimilation Amino Acids Arabidopsis. <br />     properties regulation synthesis. J Bacteriol 151: 1299&#226;&#8364;&#8220;1313.                           Arabidopsis Book 2: e0010. <br />  6. Blattner FR, Plunkett G, 3rd, Bloch CA, Perna NT, Burland V, et al. (1997)                15. Boehlein SK, Richards NG, Schuster SM (1994) Glutamine-dependent nitrogen <br />     The complete genome sequence Escherichia coli K-12. Science 277: 1453&#226;&#8364;&#8220;                     transfer Escherichia coli asparagine synthetase B. Searching catalytic <br />     1462.                                                                                         triad. J Biol Chem 269: 7450&#226;&#8364;&#8220;7457. <br />  7. Sugiyama A, Kato H, Nishioka T, Oda J (1992) Overexpression purification              16. Duff SM, Qi Q, Reich T, Wu X, Brown T, et al. (2011) A kinetic comparison of <br />     asparagine synthetase Escherichia coli. Biosci Biotechnol Biochem 56:                 asparagine synthetase isozymes higher plants. Plant Physiol Biochem 49: <br />                                                                                                   251&#226;&#8364;&#8220;256. <br />     376&#226;&#8364;&#8220;379. <br />                                                                                               17. Ramos F, Wiame JM (1979) Synthesis activation asparagine asparagine <br />  8. Gowri VS, Ghosh I, Sharma A, Madhubala R (2012) Unusual domain <br />                                                                                                   auxotrophs Saccharomyces cerevisiae. Eur J Biochem 94: 409&#226;&#8364;&#8220;417. <br />     architecture aminoacyl tRNA synthetases paralogs Leishmania <br />                                                                                               18. Horowitz B, Meister A (1972) Glutamine-dependent asparagine synthetase from <br />     major. BMC Genomics 13: 621. <br />                                                                                                   leukemia cells. Chloride dependence, mechanism action, inhibition. J Biol <br />  9. Humbert R, Simoni RD (1980) Genetic biomedical studies demonstrating a <br />                                                                                                   Chem 247: 6708&#226;&#8364;&#8220;6719. <br />     second gene coding asparagine synthetase Escherichia coli. J Bacteriol 142: <br />                                                                                               19. Patterson MK Jr OG (1968) Asparagine biosynthesis Novikoff Hepatoma <br />     212&#226;&#8364;&#8220;220. <br />                                                                                                   isolation, purification, property, mechanism studies enzyme system. <br />                                                                                                   J Biol Chem 243: 376&#226;&#8364;&#8220;380. <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                                      13                             December 2013 | Volume 7 | Issue 12 | e2578 <br />                                                                                                                                    Trypanosoma Asparagine Synthetase A <br />  <br />  <br /> 20. Ciustea M, Gutierrez JA, Abbatiello SE, Eyler JR, Richards NG (2005)                    41. Sheng S, Kraft JJ, Schuster SM (1993) A specific quantitative colorimetric assay <br />     Efficient expression, purification, characterization C-terminally tagged,            L-asparagine. Anal Biochem 211: 242&#226;&#8364;&#8220;249. <br />     recombinant human asparagine synthetase. Arch Biochem Biophys 440:                      42. Fresquet V, Thoden JB, Holden HM, Raushel FM (2004) Kinetic mechanism of <br />     18&#226;&#8364;&#8220;27.                                                                                      asparagine synthetase Vibrio cholerae. Bioorg Chem 32: 63&#226;&#8364;&#8220;75. <br /> 21. Cedar H, Schwartz JH (1969) The asparagine synthetase Escherichia coli. II.          43. Kakkar T, Boxenbaum H, Mayersohn M (1999) Estimation Ki a <br />     Studies mechanism. J Biol Chem 244: 4122&#226;&#8364;&#8220;4127.                                           competitive enzyme-inhibition model: comparisons methods of <br /> 22. Cedar H, Schwartz JH (1969) The asparagine synthetase Escherhic coli. I.                 data analysis. Drug Metab Dispos 27: 756&#226;&#8364;&#8220;762. <br />     Biosynthetic role enzyme, purification, characterization              44. Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, et al. (2007) <br />     reaction products. J Biol Chem 244: 4112&#226;&#8364;&#8220;4121.                                              Clustal W Clustal X version 2.0. Bioinformatics 23: 2947&#226;&#8364;&#8220;2948. <br /> 23. Larsen TM, Boehlein SK, Schuster SM, Richards NG, Thoden JB, et al. (1999)              45. Bond CS, Schuttelkopf AW (2009) ALINE: WYSIWYG protein-sequence <br />     Three-dimensional structure Escherichia coli asparagine synthetase B: short            alignment editor publication-quality alignments. Acta Crystallogr D Biol <br />     journey substrate product. Biochemistry 38: 16146&#226;&#8364;&#8220;16157.                            Crystallogr 65: 510&#226;&#8364;&#8220;512. <br /> 24. Nakatsu T, Kato H, Oda J (1998) Crystal structure asparagine synthetase              46. Arnold K, Bordoli L, Kopp J, Schwede T (2006) The SWISS-MODEL <br />     reveals close evolutionary relationship class II aminoacyl-tRNA synthetase.            workspace: web-based environment protein structure homology modelling. <br />     Nat Struct Biol 5: 15&#226;&#8364;&#8220;19.                                                                   Bioinformatics 22: 195&#226;&#8364;&#8220;201. <br /> 25. Nakatsu T, Kato H, Oda J (1996) Crystallization preliminary crystallo-              47. Kiefer F, Arnold K, Kunzli M, Bordoli L, Schwede T (2009) The SWISS- <br />     graphic study asparagine synthetase Escherichia coli. Acta                          MODEL Repository associated resources. Nucleic Acids Res 37: D387&#226;&#8364;&#8220;392. <br />     Crystallogr D Biol Crystallogr 52: 604&#226;&#8364;&#8220;606.                                             48. Peitsch MC, Wells TN, Stampf DR, Sussman JL (1995) The Swiss-3DImage <br /> 26. Boehlein SK, Stewart JD, Walworth ES, Thirumoorthy R, Richards NG, et al.                   collection PDB-Browser World-Wide Web. Trends Biochem Sci 20: <br />     (1998) Kinetic mechanism Escherichia coli asparagine synthetase B.                       82&#226;&#8364;&#8220;84. <br />     Biochemistry 37: 13230&#226;&#8364;&#8220;13238.                                                           49. Cooney DA, Capizzi RL, Handschumacher RE (1970) Evaluation L- <br /> 27. Luehr CA, Schuster SM (1985) Purification characterization beef                      asparagine metabolism animals man. Cancer Res 30: 929&#226;&#8364;&#8220;935. <br />     pancreatic asparagine synthetase. Arch Biochem Biophys 237: 335&#226;&#8364;&#8220;346.                    50. Goldberg AI, Cooney DA, Glynn JP, Homan ER, Gaston MR, et al. (1973) The <br /> 28. Huang XH, HM; Raushel, F (2001) Channeling substrates                            effects immunization L-asparaginase antitumor enzymatic activity. <br />     intermediates enzyme-catalyzed reactions. Annual Review Biochemistry                  Cancer Res 33: 256&#226;&#8364;&#8220;261. <br />     70: 149&#226;&#8364;&#8220;180.                                                                            51. Hannaert V, Albert MA, Rigden DJ, da Silva Giotto MT, Thiemann O, et al. <br /> 29. Berriman M, Ghedin E, Hertz-Fowler C, Blandin G, Renauld H, et al. (2005)                   (2003) Kinetic characterization, structure modelling studies crystallization of <br />     The genome African trypanosome Trypanosoma brucei. Science 309:                      Trypanosoma brucei enolase. Eur J Biochem 270: 3205&#226;&#8364;&#8220;3213. <br />     416&#226;&#8364;&#8220;422.                                                                                52. Clayton CE (1987) Import fructose bisphosphate aldolase glycosomes <br /> 30. El-Sayed NM, Myler PJ, Bartholomeu DC, Nilsson D, Aggarwal G, et al. (2005)                 Trypanosoma brucei. J Cell Biol 105: 2649&#226;&#8364;&#8220;2654. <br />     The genome sequence Trypanosoma cruzi, etiologic agent Chagas disease.            53. Bangs JD, Uyetake L, Brickman MJ, Balber AE, Boothroyd JC (1993) Molecular <br />     Science 309: 409&#226;&#8364;&#8220;415.                                                                       cloning cellular localization BiP homologue Trypanosoma brucei. <br /> 31. El-Sayed NM, Myler PJ, Blandin G, Berriman M, Crabtree J, et al. (2005)                     Divergent ER retention signals lower eukaryote. J Cell Sci 105 (Pt 4): 1101&#226;&#8364;&#8220; <br />     Comparative genomics trypanosomatid parasitic protozoa. Science 309: 404&#226;&#8364;&#8220;                1113. <br />     409.                                                                                    54. He CY, Ho HH, Malsam J, Chalouni C, West CM, et al <br /> </body></html>