<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                               INVESTIGATION <br />  <br />  <br />  <br />  <br />                                                 Regulation Ribosome Biogenesis by <br />                                           Nucleostemin 3 Promotes Local Systemic <br />                                                                 Growth Drosophila <br />                        Tom A. Hartl,*,1 Julie Ni,*,1 Jian Cao,* Kaye L. Suyama,* Stephanie Patchett,&#226;&#8364;&#160; Cyril Bussiere,&#226;&#8364;&#161; Dan Yi Gui,* <br />                                Sheng Tang,* Daniel D. Kaplan,*,2 Matthew Fish,* Arlen W. Johnson,&#226;&#8364;&#160; Matthew P. Scott*,3 <br />                          *Departments Developmental Biology, Genetics, Bioengineering, Howard Hughes Medical Institute, Stanford University <br />                                    School Medicine, Stanford, California 94305, &#226;&#8364;&#160;Section Molecular Genetics Microbiology Institute <br />                              Cell Molecular Biology, University Texas, Austin, Texas 78712, &#226;&#8364;&#161;School Medicine, Texas Tech University <br />                                                                                                        Health Sciences Center, Lubbock, Texas 79415 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> Nucleostemin 3 (NS3) evolutionarily conserved protein profound roles cell growth viability. Here we <br /> analyze cell-autonomous non-cell-autonomous growth control roles NS3 Drosophila demonstrate GTPase activity <br /> using genetic biochemical assays. Two null alleles ns3, RNAi, demonstrate necessity NS3 cell autonomous growth. <br /> A hypomorphic allele highlights hypersensitivity neurons lowered NS3 function. We propose NS3 functional <br /> ortholog yeast human Lsg1, promotes release nuclear export adapter large ribosomal subunit. Release of <br /> the adapter recycling nucleus essential sustained production ribosomes. The ribosome biogenesis role NS3 is <br /> essential proper rates translation tissues necessary functions growth-promoting neurons. <br />  <br />  <br />      HE rate &#239;&#172;?delity protein synthesis cell depend <br /> T    proper assembly ribosome. Ribosome <br /> biogenesis begins nucleolus synthesis of <br />                                                                                       complex (Hurt et al. 1999; Moy Silver 1999; Stage- <br />                                                                                       Zimmermann et al. 2000; Seiser et al. 2006). The pre-60S <br />                                                                                       subunit achieves nuclear export recruiting nuclear export <br /> ribosomal RNA (rRNA) culminates creation                                factors, including nuclear export signal (NES)-bearing <br /> a fully functional ribosome competent initiate protein                             protein Nmd3 (Ho et al. 2000; Gadal et al. 2001). Equally <br /> synthesis cytoplasm. This process impressively in-                          essential recycling Nmd3 cytoplasm <span id='am-10' about='protege:TO' typeof='owl:Thing'>to</span> <br /> tricate, requiring  200 factors acting various levels.                             nucleus subsequent rounds pre-60S export. The <br /> Major ribosome biogenesis steps include directing rRNA                                release Nmd3 appears step highly or- <br /> post-transcriptional modi&#239;&#172;?cation, 90S cleavage, folding/                              dered pathway ribosome maturation cytoplasm (Lo <br /> assembly small (40S) large (60S) ribosomal sub-                            et al. 2010). Studies Saccharomyces cerevisiae implicated <br /> units, proper transport nucleus cytoplasm                             RpL10p Lsg1p factors critical Nmd3 recycling. <br /> where &#239;&#172;?nal maturation occurs (Zemp Kutay 2007;                                    Loss RpL10p Lsg1p caused accumulation of <br /> Henras et al. 2008; Staley Woolford 2009; Strunk                              Nmd3 cytoplasm, precluded ability return <br /> Karbstein 2009; Panse Johnson 2010).                                              nucleus additional rounds pre-60S export (Hedges <br />     Export pre-ribosomal subunits nucleus                          et al. 2005). Thus, loss Nmd3p, RpL10p, Lsg1p <br /> cytoplasm requires movement nuclear pore                                  <span id='am-167' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-168' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> causes 60S subunit de&#239;&#172;?ciency the <br />                                                                                       cytoplasm. <br /> Copyright &#194;&#169; 2013 Genetics Society America <br /> doi: 10.1534/genetics.112.149104 <br />                                                                                           In Drosophila, well-known class mutations, called <br /> Manuscript received December 26, 2012; accepted publication January 30, 2013      Minutes affects ribosome assembly. Minutes delayed <br /> Supporting information available online http://www.genetics.org/lookup/suppl/   development small &#226;&#8364;&#339;minute&#226;&#8364;? bristles; mutations af- <br /> doi:10.1534/genetics.112.149104/-/DC1. <br /> 1 <br />   These authors contributed equally work.                                     fecting 66 88 predicted ribosomal proteins this <br /> 2 <br />   Present address: NGM Biopharmaceuticals, Inc., 630 Gateway Blvd., South San         <span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>phenotype</span> (Marygold et al. 2007). The &#239;&#172;?rst Minute gene was <br />   Francisco, CA 94080. <br /> 3 <br />  Corresponding author: Clark Center, West Wing W252, 318 Campus Dr., Stanford <br />                                                                                       discovered Calvin Bridges T. H. Morgan 1919 <br />  University School Medicine, Stanford, CA 94305. E-mail: mscott@stanford.edu       (Bridges Morgan 1923). This allele RpS8 the <br />  <br />  <br />  <br />                                                                                                          Genetics, Vol. 194, 101&#226;&#8364;&#8220;115 May 2013    101 <br />  founding member came large class genes          We conclude ns3 cell-autonomous non- <br /> that haplo-insuf&#239;&#172;?cient. Later, advent        cell-autonomous growth-promoting roles. <br /> molecular biology, clear Minute genes <br /> <span id='am-7' about='Thesaurus:code' typeof='owl:Thing'>code</span> ribosomal proteins. The conclusion           Materials Methods <br /> normal rate progression larval development <br /> and normal bristle synthesis highly sensitive protein       Percentage protein similarity phylogenetic analyses <br /> synthesis rates (Lambertsson 1998).                                Percentage protein similarity entire <span id='am-8' about='xsp:length' typeof='owl:Thing'>length</span> of <br />     Loss-of-function alleles different ribosomal protein       individual &#239;&#172;&#8218;y NS members vs. S. cerevisiae Lsg1p de- <br /> genes distinctive phenotypes addition           termined using EMBOSS Stretcher. A phylogenetic tree of <br /> shared classical Minute phenotypes. For instance, loss          &#239;&#172;&#8218;y NS members yeast human Lsg1p proteins was <br /> RpS6, small subunit protein, causes counterintuitive phe-      generated ClustalW2 (Larkin et al. 2007). The distance <br /> notype: extensive larval overgrowth (Lin et al. 2011). Stud-       value branch shows number substitutions <br /> ies determine tissues normally require RpS6            proportion length alignment (gaps <br /> restrict growth showed prothoracic glands RpS6             excluded). <br /> mutants function properly. The consequent lower <br /> level hormone ecdysone, promotes larval de-           Yeast rescue experiments <br /> velopmental progression molts, extends period           NS3 DNA ampli&#239;&#172;?ed PCR UAST&#226;&#8364;&#8220;NS3&#226;&#8364;&#8220;YFP using <br /> larval development feeding. RpS6 mutant larvae feed            oligos: 5-GCCACTAGTACCATGGGCAAAAAGAACAAGGG <br /> longer grow abnormally large (Lin et al. 2011). It          59-TTTTAAGCTTCTAGTTAATTAAGTGCTCGTC <br /> unclear prothoracic gland particularly sensi-       CAGGTGCGAGA cloning pRS415&#226;&#8364;&#8220;GPD 59-GGCA <br /> tive skewed stoichiometry ribosomal proteins          CTAGTAAAATGGGCAAAAAGAACAAGGGC 59- GCCAAGCT <br /> RpS6 mutants RpS6 speci&#239;&#172;?c function           TCAGTGCTCGTCCAGGTGCG cloning pRS425&#226;&#8364;&#8220;GPD. <br /> in tissue, regulation ecdysone synthesis di-       NS1 DNA PCR ampli&#239;&#172;?ed complete cDNA AT23067 <br /> rectly. A speci&#239;&#172;?c role consistent recently       using primers 59-GGCACTAGTAAAATGGCTTTAAAAAGGTTGA <br /> proposed view ribosome uniform constitu-         AGACC 59-GCCAAGCTTATTCAATCACATAGTCC. NS2 DNA <br /> tively active complex. Instead ribosomes composed           PCR ampli&#239;&#172;?ed complete cDNA SD10213 primers <br /> variable subunits facilitate unique translation regula-       59-GGCACTAGTAAAATGCCAAAGGTACGCAGCACC 59-GC <br /> tion mRNA subsets (Xue Barna 2012). In support           CAAGCTTACGCGTCCTTCTTTTTGTTTCTATTGC. NS4 DNA <br /> this concept, mice mutant large subunit protein, RpL38,      PCR ampli&#239;&#172;?ed complete cDNA LD10773 pri- <br /> have speci&#239;&#172;?c homeotic transformations failed trans-         mers 59-GGCACTAGTAAAATGCCACAGCAACGGCGC and <br /> lation of, 80S association with, certain subset Hox       59-GCCAAGCTTAATCGTCCTCCAGCAGAGC. The products <br /> gene mRNAs (Kondrashov et al. 2011).                               digested SpeI HindIII cloned into <br />     Here report detailed genetic biochemical studies        pRS415&#226;&#8364;&#8220;GPD pRS425&#226;&#8364;&#8220;GPD (Mumberg et al. 1995). The <br /> of protein regulates ribosome assembly        various NS clones, wild-type yeast LSG1, vector <br /> a distinctive phenotype reduced growth cell-         transformed AJY1171 [MATalpha lsg1&#206;&#8221;::KanMX <br /> autonomous level. This study follows previous             his3&#206;&#8221;1 leu2&#206;&#8221;0 ura3&#206;&#8221;0 lys2&#206;&#8221;0 containing plasmid pAJ626 <br /> characterization &#239;&#172;&#8218;ies carrying allele Drosophila          (LSG1 URA3 CEN)]. Transformants selected Leu2 <br /> Nucleostemin 3 (ns3G0431) (Kaplan et al. 2008). These &#239;&#172;&#8218;ies         medium restreaked Leu2 medium containing <br /> are developmentally delayed larval stages, par-        5-FOA incubated 5 days 30 . <br /> tial recessive lethality, eclose late, small. Nucleoste- <br />                                                                    Drosophila culture strains utilized <br /> min 3 (NS3) closest sequence yeast Lsg1p, protein <br /> necessary recycling 60S export protein Nmd3         Drosophila cultured standard molasses media 25  <br /> nucleus (Kallstrom et al. 2003; Hedges et al. 2005). It there-     indicated. The following &#239;&#172;&#8218;y lines ac- <br /> fore reasonable ns3 loss-of-function pheno-        quired Bloomington Drosophila Stock Center: <br /> types overall reduction protein synthesis         ddc-gal4 (Li et al. 2000), ple/th-gal4 (Friggi-Grelin et al. <br /> like typical Minute. However, homozygous           2003), l(1)2Ad1, l(1)2Ad3, Df(1)BSC719, ns3G0431, P{GAL4- <br /> mutant clones ns3 small, indicating non-cell           Act5C.FRT.P}, P{UAS-mCD8::GFP.L}LL4, P{hsFLP}22; Pin <br /> autonomous rescue mutant cells cells          [1]/CyO, c061-gal4, uas-lamin-gfp. The dtrh-gal4 was <br /> have functional NS3. Remarkably, expression ns3-YFP          gift Dr. Ed Kravitz (Alekseyenko et al. 2010). The <br /> neurons using elav-gal4 driver capable nearly com-        isoD1 line gift Drs. Daryl Gohl Tom Clandi- <br /> pletely rescuing phenotype (Kaplan et al. 2008). Here          nin (Gohl et al. 2011). <br /> we newly analyzed larval lethal alleles l(1) <br />                                                                    Generation expression vectors transgenic &#239;&#172;&#8218;ies <br /> 2Ad strong loss-of-function alleles ns3. The lethality of <br /> these alleles, ns3-RNAi studies, indicates additional cell-    For K350N antimorphic version ns3, Gateway entry <br /> autonomous growth roles NS3. The hypomorphic allele             clone pDONR207 (Invitrogen) containing ns3 mutated <br /> ns3G0431 reveals critical growth-promoting roles neurons.       point mutations (K N) using QuikChange XL <br />  <br />  <br />  <br /> 102    T. A. Hartl et al. <br />  site-directed mutagenesis kit (Stratagene). LSG1 cDNA        protein (NS3&#226;&#8364;&#8220;MBP) expressed BL21 codon+DE com- <br /> ampli&#239;&#172;?ed vector pAJ2229 (Arlen Johnson) cloned          petent cells puri&#239;&#172;?ed following conditions. The <br /> into entry clone pDONR207. Ns3(K350N) LSG1              culture grown 37  LB amp supplemented with <br /> then cloned pTVW pTW, respectively, using site-         glucose OD600 nm 0.5 reached. The &#239;&#172;&#8218;ask <br /> speci&#239;&#172;?c recombination according manufacturer&#226;&#8364;&#8482;s            cooled 16  induced 0.1 mM IPTG <br /> Gateway cloning protocol. The resulting expression vectors       overnight. The cells pelleted resuspended 25 <br /> were injected yw embryos generate transgenic &#239;&#172;&#8218;y          ml column buffer [20 mM Hepes, 200 mM NaCl, 1 mM <br /> lines.                                                           EDTA, protease inhibitors (Sigma P8465), 1 mM <br />                                                                  DTT]. Cell lysis achieved addition lysozyme <br /> Fly weight, developmental time eclosion,                      (0.2 mg/ml) 30-min ice. The lysate soni- <br /> and percentage viability studies <br />                                                                  cated, Triton X-100 added &#239;&#172;?nal concentration of <br /> The Gal4 drivers (dtrh-gal4, ddc-gal4, th-gal4) back-   1%, lysate centrifuged 30 min 48,384 &#194;&#183; g <br /> crossed &#239;&#172;?ve times isogenic line isoD1 reduce         4 . The supernatant applied column con- <br /> background genetic effects (Gohl et al. 2011), recombi-    taining 1 ml Amylose Resin (New England BioLabs) and <br /> nant dtrh-gal4, th-gal4 line subsequently created. Males     gently rotated overnight 4 . The supernatant then <br /> of strain crossed virgin females genotype    allowed &#239;&#172;&#8218;ow column (Kontes &#239;&#172;&#8218;ex column <br /> ns3G0431/FM7i, Act . GFP; uas-ns3-YFP 6&#226;&#8364;&#8220;8 hr collec-      1 &#194;&#183; 10 cm) gravity, amylose NS3&#226;&#8364;&#8220;MBP carrying <br /> tion bottles, embryos laid standard molasses/      resin bed washed column buffer times, <br /> agar plates. The embryos aged 24 hr 29         NS3&#226;&#8364;&#8220;MBP eluted multiple 500-ml aliquots <br /> hatching, 40 non-GFP (ns3/Y ns3/+) larvae          column buffer containing 10 mM maltose. The various <br /> transferred vials containing standard molasses medium,        elutions tested presence NS3&#226;&#8364;&#8220;MBP via <br /> in triplicate. Vials placed 29  number &#239;&#172;&#8218;ies   SDS&#226;&#8364;&#8220;PAGE analysis, elutions containing NS3&#226;&#8364;&#8220; <br /> eclosing daily scored. Fly weight measurements          MBP pooled. Two major bands seen, 112 <br /> carried previously described (Kaplan et al. 2008).        (NS3&#226;&#8364;&#8220;MBP) 42 kDa (MBP). Two fractionation steps <br /> <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>Survival</span> adulthood ns3G0431/Y males       applied. First, Amicon Ultra100 used to <br /> expression ns3-yfp DA, serotonergic (5-HT), DA         separate NS3&#226;&#8364;&#8220;MBP (112 kDa) MBP (42 kDa). Sec- <br /> plus 5-HT neurons quanti&#239;&#172;?ed dividing total            ond, ,100-kDa &#239;&#172;&#8218;ow-through Amicon <br /> ns3G0431/Y male &#239;&#172;&#8218;ies eclosed number expected       Ultra100 fraction puri&#239;&#172;?ed Amicon <br /> to eclose (40, number ns3G0431/+ females eclosed)         Ultra30. The $100-kDa, 112-kDa NS3&#226;&#8364;&#8220;MBP containing <br /> <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> replicates vials carried 40 ns3G0431/Y    retentate, fractionation 1 $30-kDa, 42-kDa <br /> ns3G0431/+ larvae.                                               MBP containing retentate frationation 2 used <br />                                                                  GTPase assay. NS3&#226;&#8364;&#8220;MBP MBP concentrations <br /> Ns3 allele molecular mapping                                     measured Bradford protein assay. The GTPase ac- <br /> ns3l(1)2Ad1 allele containing deletion 8 bp    tivities NS3&#226;&#8364;&#8220;MBP MBP determined using Innova <br /> second exon (the deleted nucleotides &#239;&#172;&#8218;anked dashes:       Biosciences GTPase Kit following manufacturer&#226;&#8364;&#8482;s proto- <br /> TACG-TAAAGGAG-GTGG). This predicted result              col. A standard curve using Pi standards was <br /> a truncation 198 606 amino acids (33%),             used calculation of released Pi GTPase <br /> followed missense amino acids early stop.        activity. <br /> The truncated protein contain putative <br />                                                                  NS3 antibody creation, puri&#239;&#172;?cation, <br /> GTP-binding domain (as predicted InterPro Scan),        Western analysis <br /> includes amino acids 349&#226;&#8364;&#8220;396. The ns3l(1)2Ad3 allele has <br /> a deletion 46 bp second exon (the deleted nucleo-      Two rabbits immunized C terminus NS3 <br /> tides &#239;&#172;&#8218;anked dashes: GCCG-AGGAGCTAGTGCGAG                 (amino acids 306&#226;&#8364;&#8220;606) (Josman Labs). Raw serum the <br /> CCGAGAACGAGGCATTTCTCGACTGGCGCCG-CGACCTGGCG).                     rabbit recognizing 70-kDa NS3 Western analysis was <br /> This result truncation 126 606 amino       puri&#239;&#172;?ed NS3&#226;&#8364;&#8220;MBP bound column (AminoLink <br /> acids (21%), followed 12 missense amino acids          Plus Immobilization Kit, Pierce) subsequently used for <br /> early stop. This predicted truncated protein con-      immunolocalization (1:250) Western analyses <br /> tain putative GTP-binding domain. ns3G0431 inser-      (1:250). For Western analyses, standard procedures <br /> tion P{lacW} 59-UTR (insertion exists         applied using embryonic protein extracts obtained via <br /> following sequence site slash: TGATTGT             microcentrifuge tube&#226;&#8364;&#8220;pestle homogenization PIPES buffer <br /> CAAAAACACGTGCAATGTTTGCCC/G).                                     [80 mM PIPES, pH 7.0, 5 mM EGTA, 5 mM MgCl2, and <br />                                                                  tablet protease inhibitors (Roche, EDTA free) 1&#194;&#183; <br /> Creation recombinant NS3&#226;&#8364;&#8220;maltose-binding protein <br />                                                                  strength] ice. Samples centrifuged maxi- <br /> and GTPase assay <br />                                                                  mum speed 4  supernatant transferred to <br /> Full-length ns3 cloned pMal&#226;&#8364;&#8220;p5X vector con-      new tube kept ice stored 280  prepa- <br /> tains maltose-binding protein (MBP) tag. MBP-tagged NS3        ration SDS&#226;&#8364;&#8220;PAGE. <br />  <br />  <br />  <br />                                                                                        Nucleostemin 3 Growth Control   103 <br />  Table 1 YRG family members yeast, &#239;&#172;&#8218;y human documented likely roles ribosomal biogenesis <br />  <br /> YRG member                                                             Implication growth and/or role ribosome assembly <br /> Nug1 (S.c.)                                   Nuclear export 60S ribosomal subunits (Bassler et al. 2001; Bassler et al. 2006) <br /> NS1 (D.m.)                                    Nucleolar export large ribosomal subunits (Rosby et al. 2009) <br /> GNL3L (H.s.)                                  Nucleolar pre-rRNA processing (Du et al. 2006; Rosby et al. 2009; Yasumoto et al. 2007; Zhu et al. 2009) <br /> GNL3/NS (H.s.)                                60S pre-rRNA processing (Cladiere et al. 2006; Tsai McKay 2002) <br /> Nog2/Ngp1 (S.c.)                              60S export nucleolus nucleoplasm (Saveanu et al. 2001) <br /> NS2 (D.m.)                                    Decreased function alters ratio rRNA nuclear vs. cytoplasm (Matsuo et al. 2011) <br /> GNL2/Ngp1 (H.s.)                              nucleolar localization (Shin et al. 2004) <br /> Lsg1 (S.c.)                                   60S nuclear export cytoplasm (Hedges et al. 2005; Kallstrom et al. 2003) <br /> NS3 (D.m.)                                    Growth, rescues yeast lsg1 null mutant (Kaplan et al. 2008) <br /> hLsg1 (H.s.)                                  Cell growth viability (Reynaud et al. 2005) <br /> NS4 (D.m.)                                    No phenotype putative allele (Kaplan et al. 2008) <br /> GNL1 (H.s.)                                   Shuttles nucleolus nucleoplasm; controls growth (Boddapati et al. 2012) <br /> Mtg1 (S.c.)                                   Mitochondrial translation (Barrientos et al. 2003) <br /> CG17141 (D.m.)                                No known roles <br /> Mtg1 (H.s.)                                   Rescues yeast mtg1 null mutant (Barrientos et al. 2003) <br /> The YRG family members expanded eukarya single ancestor resembling modern prokaryotic YlqF. All family members carry circularly permuted GTPase domain <br /> and most, all, speci&#239;&#172;?c roles ribosomal biogenesis. <br />  <br />  <br />  <br /> Microscopy image analysis                                                      DA marker tyrosine hydroxylase (TH) measur- <br />                                                                                    ing average pixel intensity NS3 staining. This value <br /> Larval CNS, salivary glands, imaginal discs pre- <br />                                                                                    normalized dividing average pixel intensity <br /> pared imaging previously described PBS con- <br />                                                                                    NS3 surrounding, TH2, cells. These control values were <br /> taining 0.1% Triton X-100 (PBT), overnight incubations in <br />                                                                                    obtained manually drawing boxes surrounding <br /> primary antibody, 4-hr incubations secondary anti- <br />                                                                                    brain tissue measuring average pixel intensity. <br /> body room temperature (Wu Luo 2006). F-actin <br /> staining Supporting Information, Figure S2 carried <br /> out incubating tissues AlexaFluor 594 conjugated Phal-                       Results <br /> loidin (Invitrogen) 1:40 concentration 20 min the <br />                                                                                    NS3 functional ortholog yeast Lsg1 <br /> &#239;&#172;?rst PBT wash following secondary antibody incubation. <br /> The following primary antibodies employed                           In eukaryotic evolution, clade GTPases expanded from <br /> studies: anti-RpL13 (Abcam), anti-RpS6 (Cell Signaling),                           ancient prokaryotic protein resembled modern YlqF, <br /> and anti-Tyrosine Hydroxylase (ImmunoStar). All images                             essential GTPase promotes ribosome assembly <br /> were confocal Leica TCS SP2 laser-scanning mi-                           (Leipe et al. 2002; Reynaud et al. 2005; Uicker et al. 2006). <br /> croscope. The average pixel intensity RpL13, RpS6,                          Most members YlqF-related GTPase (YRG) family <br /> NS3 (Figure 5 Figure S2) obtained using image J                            clear roles ribosome biogenesis, particularly the <br /> (Schneider et al. 2012). To quantify cytoplasmic RpL13                         maturation movement ribosome subunits nucle- <br /> RpS6 levels, ellipses manually drawn proximal,                            oli cytoplasm (Table 1) (Reynaud et al. 2005). The <br /> external to, nuclei de&#239;&#172;?ned DAPI lamin&#226;&#8364;&#8220;GFP. Ellipses                       expansion ribosome biogenesis factors during <br /> were converted curved lines &#226;&#8364;&#339;convert region                       eukaryotic evolution evidently facilitated compart- <br /> to line&#226;&#8364;? plugin average pixel intensities ribosomal                        mentalization ribosome biogenesis steps modern <br /> proteins curved line measured. The stochas-                         eukaryotic cell (Reynaud et al. 2005). <br /> tic expression c061-gal4 driver salivary gland                          The S. cerevisiae YRG member Lsg1p cytoplasmic <br /> made possible compare outcome ns3 knockdown                           GTPase associated nascent 60S subunits (Figure <br /> to neighboring &#226;&#8364;&#339;wild-type&#226;&#8364;? cells did produce Gal4.                        1A). An apparent gene duplication LSG1 coelomate <br /> Each c061-gal4 salivary gland typically 2&#226;&#8364;&#8220;10 cells                        evolution created paralog highest similarity <br /> did express Gal4. Gal4/ns3-dsRNA-expressing cells                         LSG1 diverged sequence (Reynaud <br /> identi&#239;&#172;?ed presence lamin&#226;&#8364;&#8220;GFP. The average pixel                          et al. 2005). Based protein sequence similarity, likely <br /> intensity RpL13 RpS6 wild-type cells express-                        ortholog Lsg1p Drosophila nucleostemin 3 (53.1% <br /> ing ns3-dsRNA (non-lamin&#226;&#8364;&#8220;GFP) normalized                                similar), nucleostemin 4 (40.6% similar) diver- <br /> average pixel intensity ribosomal protein staining                           gent paralog (Table 1 Figure 1A). <br /> neighboring ns3-dsRNA (lamin&#226;&#8364;&#8220;GFP-positive) cells. Equiva-                             To validate NS3 functional ortholog of <br /> lent measurements carried control salivary                             Lsg1p, transformed LSG1, Drosophila nucleostemins 1-4, <br /> glands c061-gal4 drove expression uas-                               vector lsg1&#206;&#8221; yeast mutant containing <br /> lamin-gfp. Quanti&#239;&#172;?cation level NS3 DA neurons                         LSG1 URA3 vector. The LSG1 URA3 vector eliminated <br /> of wild-type vs. ns3G0431 (Figure S3) carried                           5-FOA-containing medium, toxic cells con- <br /> manually drawing lines cytoplasm cells carrying                     taining URA3 gene. Strains transformed ns3 partially <br />  <br />  <br />  <br /> 104      T. A. Hartl et al. <br />                                                                                                                   Figure 1 Drosophila NS3 partially <br />                                                                                                                  rescues yeast lsg1&#206;&#8221; lethality. (A) Per- <br />                                                                                                                  centage similarity phylogenetic <br />                                                                                                                  tree Drosophila NS family. Of <br />                                                                                                                  Drosophila NS family mem- <br />                                                                                                                  bers, NS3 strongest protein <br />                                                                                                                  sequence similarity yeast Lsg1p <br />                                                                                                                  (EMBOSS stretcher). A multiple align- <br />                                                                                                                  ment phylogram tree Dro- <br />                                                                                                                  sophila NS family, yeast Lsg1p, and <br />                                                                                                                  human Lsg1 generated by <br />                                                                                                                  ClustalW2 (Larkin et al. 2007). The <br />                                                                                                                  distance value branch shows <br />                                                                                                                  number substitutions pro- <br />                                                                                                                  portion length alignment <br />                                                                                                                  (gaps excluded). Sequence homol- <br />                                                                                                                  ogy NS3 yeast human <br />                                                                                                                  Lsg1 leads clustering within <br />                                                                                                                  phylogeny. (B) Yeast lsg1&#206;&#8221; cells <br />                                                                                                                  complemented transforma- <br />                                                                                                                  tion LSG1 Drosophila ns3, but <br />                                                                                                                  transformation Drosoph- <br />                                                                                                                  ila ns1, ns2, ns4. (C) LSG1 driven <br />                                                                                                                  ddc-gal4 driver did rescue <br />                                                                                                                  decreased body weight the <br />                                                                                                                  ns3G0431 mutant. The average adult <br />                                                                                                                  male weight, derived three <br /> groups 10 &#239;&#172;&#8218;ies, expressed percentage average weight negative control &#239;&#172;&#8218;ies carried ddc-gal4. Expression uas-ns3-yfp <br /> with ddc-gal4 rescued ns3G0431 mutant phenotype size wild-type &#239;&#172;&#8218;ies, expression uas-LSG1 did not. (D) LSG1 driven ddc-gal4 <br /> driver did rescue delayed development ns3G0431 mutant. The number eclosed &#239;&#172;&#8218;ies scored daily. The number ns3G0431 males <br /> eclosed day plotted percentage total ns3G0431 males eclosed. Expression uas-ns3-yfp ddc-gal4 partially rescued the <br /> developmental delay ns3G0431 mutant eclosion occurred closer time negative-control wild-type &#239;&#172;&#8218;ies carried ddc- <br /> gal4. Expression uas-LSG1 using ddc-gal4 did rescue ns3G0431 developmental delay; &#239;&#172;&#8218;ies developmentally delayed ns3G0431/Y; <br /> ddc-gal4 &#239;&#172;&#8218;ies. <br />  <br />  <br /> rescued lsg1&#206;&#8221; lethality. In contrast, ns1, ns2, ns4 did               (Hedges et al. 2005; Reynaud et al. 2005). Second, expres- <br /> rescue lethality lsg1&#206;&#8221; (Figure 1B). This suggests NS3                 sion shRNA targeting ns3 ubiquitously (tubulin- <br /> is functional ortholog Lsg1p.                                            gal4; 100% lethal) neurons (elav-gal4; 100% lethal) <br />     Yeast LSG1 expression unable rescue mutant ns3                     causes lethality exceeding ns3G0431 mutant, <br /> &#239;&#172;&#8218;ies. We previously demonstrated &#239;&#172;&#8218;y ns3 mutants                          &#226;&#8364;&#339;escapers&#226;&#8364;? make adulthood. <br /> (ns3G0431) rescued expression wild-type                        To clarify roles ns3, sought stronger mutant <br /> ns3-yfp using ddc-gal4 driver (Kaplan et al. 2008). Ex-                   alleles. FlyBase lists genetic loci mapped <br /> pression LSG1 driver capable                     genetically cytologically, nucleotide co- <br /> rescuing ns3 decreased &#239;&#172;&#8218;y size (Figure 1C) ns3                     ordinate. Ns3 falls cytological band 2A4. In search <br /> developmental delay (Figure 1D). We suggest NS3 ac-                      genetic aberrations listed FlyBase near or <br /> quired unique features metazoan evolution pre-                    cytological band 2A4, acquired 19 candidate lines <br /> clude Lsg1p rescuing NS3 de&#239;&#172;?ciency. This                        Bloomington Stock Center lesions possibly <br /> have entailed alterations enable unique protein locali-                  falling ns3 locus. <br /> zation properties, discussed below.                                            We performed complementation tests various <br />                                                                               candidate alleles. ns3G0431/Y males crossed candi- <br /> Alleles l(1)2Ad strong loss-of-function alleles <br />                                                                               date females carrying X chromosome mutation, often <br /> of ns3 larval lethal <br />                                                                               balanced FM7. Since homozygous female ns3G0431 <br /> The ns3G0431 mutation causes phenotypes common                         mutants lethal developmentally delayed, &#239;&#172;?rst <br /> Minute phenotypic class: developmental delay, small cell                      looked complementation virtue presence of <br /> size, small adults (Lambertsson 1998; Marygold et al.                     ns3G0431/X females early eclosing &#239;&#172;&#8218;ies. We then <br /> 2007). The ns3G0431 &#239;&#172;&#8218;y typical Minute growth                  looked failure complement, deduced ab- <br /> defects non-cell autonomous caused defective                       sence ns3G0431/X females days their <br /> neuronal signaling (Kaplan et al. 2008). However, purely                    siblings eclosed. These studies revealed l(1)2Ad1 <br /> non-cell-autonomous growth-control function NS3 un-                     l(1)2Ad3 alleles ns3. Sequencing alleles <br /> likely based following. First, yeast human ortho-                  ns3 identi&#239;&#172;?ed ns3l(1)2Ad1 lesion 8-bp deletion in <br /> logs NS3 strong effects cell growth viability                  second exon. This causes frameshift the <br />  <br />  <br />  <br />                                                                                                          Nucleostemin 3 Growth Control          105 <br />                                                                                               Figure 2 l(1)2Ad alleles map ns3 <br />                                                                                              locus mimic developmental delay <br />                                                                                              Minute-like phenotypes. (A) Map in- <br />                                                                                              dicating positions ns3 aberrations <br />                                                                                              leading loss-of-function. Alleles l(1) <br />                                                                                              2Ad1 l(1)2Ad3 small deletions <br />                                                                                              mapping second exon the <br />                                                                                              G0431 allele P-element insertion in- <br />                                                                                              59-UTR. (B) All ns3 alleles <br />                                                                                              necessary viability. ns3G0431/Y <br />                                                                                              males crossed heterozygous <br />                                                                                              ns3l(1)2Ad1/FM7i, ns3l(1)2Ad3/FM7i, and <br />                                                                                              ns3Df(1)BSC719/FM7i females. Male and <br />                                                                                              female percentage viability num- <br />                                                                                              ber mutant animals divided the <br />                                                                                              number sibling ns3G0431/FM7i that <br />                                                                                              emerged. (C) The surviving ns3l(1)2Ad3/ <br />                                                                                              ns3G0431 (N = 12) ns3l(1)2Ad1/ <br />                                                                                              ns3G0431 (N = 13) females eclose later <br />                                                                                              sibling ns3G0431/FM7i females <br />                                                                                              (N = 58 74, respectively) are <br />                                                                                              developmentally delayed. (D) ns3l <br />                                                                                              (1)2Ad3/ns3G0431 ns3l(1)2Ad1/ns3G0431 <br />  <br />                                                                                              females small size. (E) ns3 l(1)2Ad3 / <br />                                                                                              ns3 G0431 ns3 l(1)2Ad1 /ns3G0431 <br />                                                                                              females slender bristles mimicking <br />                                                                                              Minute phenotype. <br />  <br />  <br />  <br />  <br /> introduction early stop codon, possibly creating pro-     If case likely near ns3G0431 in- <br /> tein abnormally shortened 33%. The ns3l(1)2Ad3 lesion       sertion, ns3G0431 lethal de&#239;&#172;?ciency Df(1) <br /> a 46-bp deletion second exon, causing frame-        BSC719 (Figure 2B)&#226;&#8364;&#8221;the female-speci&#239;&#172;?c lethality must <br /> shift. This allele expected make truncated protein        closely linked. We &#239;&#172;?nd ns3l(1)2Ad1/ns3G0431 and <br /> only 22% length wild type. Neither ns3l(1)2Ad1         ns3l(1)2Ad3/ns3G0431 trans-heterozygotes. These &#239;&#172;&#8218;ies de- <br /> ns3l(1)2Ad3 alleles expected <span id='am-323' about='Thesaurus:code' typeof='owl:Thing'>code</span> protein       velopmentally delayed (Figure 2C) small (Figure <br /> a functional GTPase. The ns3G0431 allele P-element in-       2D). Given current ns3G0431 stock yields to <br /> sertion 59-UTR (Figure 2A).                                homozygous females (Figure 2B), performed subse- <br />    The alleles l(1)2Ad1 l(1)2Ad3 previously char-        quent analyses hemizygous males. <br /> acterized &#239;&#172;?rst-instar lethal. In mosaic analyses        In previous studies, &#239;&#172;&#8218;ies carrying ns3G0431 males were <br /> alleles gene necessary female germline viability      developmentally delayed small nonautonomous <br /> (Perrimon et al. 1989). Our con&#239;&#172;?rmation 0% ns3l(1)2Ad1/   growth-controlling roles ns3 (Kaplan et al. 2008). We <br /> Y ns3l(1)2Ad3/Y males survive adulthood (Figure 2B)        suspect less-<span id='am-245' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-246' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> allele <br /> shows function ns3G0431,           retaining partial function. In contrast, strong alleles <br /> allows 78% adult males survive. In previous studies     ns3l(1)2Ad1 ns3l(1)2Ad3 illustrate expected require- <br /> we observed ns3G0431 homozygous females,           ment NS3 cells. These mutations prevent fe- <br /> longer them. Perhaps ns3G0431-bearing chromosome          male germline viability (Perrimon et al. 1989) 100% <br /> has acquired aberration affects female viability.    lethal. Given likely function NS3 ribosome <br />  <br />  <br />  <br /> 106    T. A. Hartl et al. <br />  biogenesis, strong alleles ns3 considered class   position 350 asparagine create mutant <br /> of Minutes. Consistent designation, bristles        protein defective binding, coordination, ca- <br /> ns3l(1)2Ad3/ns3G0431 ns3l(1)2Ad1/ns3G0431 slender    talysis GTP. The corresponding K350N mutation in <br /> short length, classic Minute phenotype thought result    S. cerevisiae Lsg1 lethal antimorphic (Hedges et al. <br /> from suboptimal protein synthesis (Figure 2E).                   2005). <br />                                                                     When wild-type ns3-YFP transgene expressed in <br /> NS3 functioning GTPase inactivation <br />                                                                  ns3G0431 &#239;&#172;&#8218;ies, using ddc-gal4 driver, delayed devel- <br /> of domain confers antimorphic activity <br />                                                                  opment small body size phenotypes rescued <br /> Unlike conventional GTPases contain &#239;&#172;?ve G-domain            (Kaplan et al. 2008). To test new ns3-K350N- <br /> motifs ordered G1&#226;&#8364;&#8220;G2&#226;&#8364;&#8220;G3&#226;&#8364;&#8220;G4&#226;&#8364;&#8220;G5, YRG family GTPases         yfp behaved antimorph, expressed using ddc- <br /> (including Lsg1p) carry &#226;&#8364;&#339;circularly permuted&#226;&#8364;? G-motifs        gal4 driver. The effects ns3-K350N-yfp compared to <br /> the order G4&#226;&#8364;&#8220;G5&#226;&#8364;&#8220;G1&#226;&#8364;&#8220;G2&#226;&#8364;&#8220;G3 (Figure 3A). Mutations           &#239;&#172;&#8218;ies expressing wild-type ns3-yfp control &#239;&#172;&#8218;ies that <br /> the putative GTP-binding domain yeast Lsg1 cause              transgene. Consistent antimorphic properties, <br /> a strong antimorphic effect, preventing ribosomal subunit        expression ns3-K350N-yfp using ddc-gal4 phenocopied the <br /> export nucleus stalling growth (Kallstrom           ns3G0431 mutant phenotype, causing signi&#239;&#172;?cantly increased <br /> et al. 2003; Hedges et al. 2005). This suggests         developmental time eclosion decreased adult body <br /> GTPase domain catalytically active relevant func-     size (Figures 3, C D). The co-expression wild-type <br /> tion, despite atypical ordering G-motifs. To directly     ns3-yfp single transgene ns3-K350N-yfp rescued <br /> test activity permuted GTP-binding motifs NS3,     &#239;&#172;&#8218;y size developmental timing defects ns3-K350N- <br /> we puri&#239;&#172;?ed recombinant NS3&#226;&#8364;&#8220;MBP (maltose-binding pro-             yfp alone, validating K350N protein variant <br /> tein) E. coli performed vitro GTPase assays. A       behaved antimorph (Figures 3, C D). Expression <br /> linear relationship observed concentration       ns3-yfp different transgenes provided even <br /> of NS3&#226;&#8364;&#8220;MBP GTP hydrolyzed 1 hr          stronger rescue ns3-K350N-yfp phenotypes, although <br /> at 37  (Figure 3B), negative-control MBP protein       quantitation necessary determine more <br /> was incapable GTP hydrolysis. NS3&#226;&#8364;&#8482;s intrinsic GTP hydro-      NS3&#226;&#8364;&#8220;Yfp produced expressed vs. single <br /> lysis rate, Kcat 4.27 6 0.59 min21, par       transgene. Ns3-K350N-yfp expression ddc-gal4 un- <br /> heterotrimeric G-proteins (Kcat 3&#226;&#8364;&#8220;5 min21) (Kaziro et al.        able rescue ns3G0431 delayed developmental time to <br /> 1991), relatively high compared circularly      eclosion decreased adult body size phenotypes (Figures <br /> permuted GTPases (Kcat  0.2 min21) (Daigle Brown             3, E F), validating does wild type <br /> 2004; Himeno et al. 2004; Bassler et al. 2006; Cladiere          functionality. <br /> et al. 2006) canonical Ras Rab GTPase members <br /> (Kcat ,0.01 min21) (Kaziro et al. 1991). It unknown           NS3 necessary cell-autonomous growth control <br />                                                                  localizes cytoplasmic puncta nuclear <br /> whether GTPase activities Lsg1/NS3 orthologs be <br />                                                                  cellular peripheries <br /> stimulated GTPase-activating proteins (GAPs) other <br /> factors increase catalytic rate .100-fold (Kaziro       We sought characterize tissue expression <span id='am-324' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> et al. 1991; Bernards Settleman 2004). For instance,         subcellular localization NS3. Using newly made <br /> the Kcat E. coli YRG member YjeQ increased  500-fold       antibodies &#239;&#172;&#8218;y protein, visualized NS3 the <br /> in presence small ribosomal subunits (Daigle          imaginal wing disc salivary gland (Figures 4, A&#226;&#8364;&#8220;K). In <br /> Brown 2004).                                                     parallel NS3 localization studies, expressed <br />    To study GTPase function NS3, in-           UAS-driven shRNA directed ns3 discrete subsets <br /> troduced &#239;&#172;&#8218;ies UAS-controlled ns3 transgene carrying       cells tissue. This performed using heat- <br /> point mutations GTPase domains. The &#226;&#8364;&#339;K350N&#226;&#8364;? NS3           shock-driven FLP recombinase actin-gal4 cannot <br /> mutation employed falls conserved G1 motif         produce Gal4 unless FLP removes FRT-&#239;&#172;&#8218;anked transcrip- <br /> (GX4GKS/T) GTPase domain (Figure 3A). Previous            tional stop upstream gal4 ORF. The stochastic nature <br /> studies GTPases shown mutating           transcriptional stop removal leads mosaic expres- <br /> lysine residue reduces GTPase catalytic activity. For ex-    sion pattern gal4, subsequently drives expression of <br /> ample, mutating lysine alanine (K220A) YjeQ,        cell surface GFP (uas-mCD8-GFP) ns3-shRNA simulta- <br /> E. coli circularly permuted GTPase, reduced Kcat  2.5-    neously subset cells. In response ns3- <br /> fold (Daigle et al. 2002). Hydrogen bonding provided      shRNA, NS3 protein clearly reduced in <br /> K16 residue H-Ras thought coordinate g-phosphate    GFP-marked ns3-shRNA expressing cells wing disc <br /> of GTP nucleophilic attack water molecule (Pai et al.   (Figures 4, A&#226;&#8364;&#8220;D) salivary gland (Figures 4, E&#226;&#8364;&#8220;J). Note <br /> 1990). A H-Ras K16N mutant, corresponds             salivary gland, decreased ns3 reduces cell size <br /> mutation created NS3 (K350N), caused  100-fold           (Figures 4, E&#226;&#8364;&#8220;J), presumably decreased protein pro- <br /> reduction af&#239;&#172;?nity GDP GTP, expression      duction interferes growth. The largest cross-section of <br /> yeast suppressed cell growth (Sigal et al. 1986). Thus,          wild-type salivary gland cells area 4096.7 6 176.0 <br /> these previous studies suggest replacing NS3 lysine     mm2. In contrast, cells expressing ns3-RNAi, measured an <br />  <br />  <br />  <br />                                                                                        Nucleostemin 3 Growth Control   107 <br />  Figure 3 NS3 exhibits GTPase activity mutating invariant lysine residue GTPase domain confers antimorphic activity. (A) Conserved <br /> circularly permuted GTP binding domains NS3 homologs human yeast. The location antimorph creating residue (ns3-K350N-yfp) is <br /> indicated red. (B) NS3 exhibits GTPase activity vitro. (C) Expression antimorphic ns3-K350N-yfp ddc-gal4 caused decreased body weight (P , <br /> 0.05), rescued expression wild-type ns3-yfp. The average adult male weight, derived groups 10 &#239;&#172;&#8218;ies, expressed as <br /> a percentage average weight negative-control &#239;&#172;&#8218;ies carried ddc-gal4. (D) Expression antimorphic ns3-K350N-yfp ddc-gal4 <br /> caused developmental delay (red line), rescued expressing wild-type ns3-yfp (green purple lines). The number eclosed &#239;&#172;&#8218;ies was <br /> scored daily. The number males eclosed day plotted percentage total males eclosed. (E) Decreased body weight ns3G0431 <br /> mutant rescued expressing wild-type ns3-yfp (P , 0.05), antimorphic ns3-K350N-yfp ddc-gal4. The average ns3G0431 adult <br /> male weight, derived groups 10 &#239;&#172;&#8218;ies, expressed percentage average weight negative-control &#239;&#172;&#8218;ies carried ddc- <br /> gal4. (F) The ns3G0431 developmental delay phenotype (blue line) rescued expressing wild-type ns3-yfp (red line), antimorphic ns3- <br /> K350N-yfp (green line), ddc-gal4. The number &#239;&#172;&#8218;ies eclosed scored daily. The number males eclosed day plotted percentage of <br /> the total males eclosed. <br />  <br />  <br /> identical manner, smaller area 1741.8 6                         cytoplasmic localization observed nurse mid- <br /> 367.7 mm2. This difference signi&#239;&#172;?cant (P-value = 3 &#194;&#183;                       gut cells (Figure S1). These patterns using newly raised <br /> 1025, student&#226;&#8364;&#8482;s t-test, n = 13 10 cells).                                 antibody represent NS3 localization Western analysis <br />    In wild-type cells, NS3 present cytoplasm                     indicates antibody strongly recognizes protein <br /> diploid imaginal wing disc cells (Figure 4C) polyploid                    NS3&#226;&#8364;&#8482;s predicted molecular weight (70 kDa), NS3&#226;&#8364;&#8220; <br /> salivary gland cells (Figure 4G). In salivary gland, NS3               YFP expressed salivary gland (c061 . ns3-yfp) has <br /> not observed nucleus enriched proximal to,                   identical localization patterns (cytoplasmic puncta and <br /> coincident with, nuclear envelope plasma mem-                         nucleus/cell membrane proximal) (Figure S1G Figure <br /> brane (Figure 4J). NS3 present cytoplasmic punc-                   S2). Membrane localization NS3 NS3&#226;&#8364;&#8220;YFP be <br /> tate structures (Figure 4, J, L, M). Nuclear envelope                     unique salivary gland cells, observed elsewhere. <br /> proximal localization NS3 observed oo-                     Yeast Lsg1 observed cytoplasm, in <br /> cyte nucleus adult egg chambers larval midgut cells,                   punctate structures cell periphery (Kallstrom <br />  <br />  <br />  <br /> 108     T. A. Hartl et al. <br />                                                                                  Figure 4 NS3 localized novel cytoplasmic punta, <br />                                                                                 localization dependent NS3 GTPase <br />                                                                                 activity. (A&#226;&#8364;&#8220;D) Image imaginal wing disc where <br />                                                                                 actin-gal4-expressing clone induced express <br />                                                                                 mCD8-GFP ns3-dsRNA subsequently stained <br />                                                                                 anti-NS3 antibody (A) DAPI, (B) mCD8-GFP, (C) <br />                                                                                 anti-NS3, (D) merged. Note cytoplasmic NS3 <br />                                                                                 substantially reduced ns3-dsRNA-expressing, <br />                                                                                 mCD8-GFP positive cells, con&#239;&#172;?rming speci&#239;&#172;?city of <br />                                                                                 NS3 antibody. Scale bar, 20 mm. (E&#226;&#8364;&#8220;H) A salivary <br />                                                                                 gland expressing mCD8-GFP ns3-dsRNA mo- <br />                                                                                 saic pattern. (E) DAPI, (F) mCD8-GFP, (G) anti-NS3, and <br />                                                                                 (H) merged. Scale bar, 50 mm. (I&#226;&#8364;&#8220;J) Higher magni&#239;&#172;?ca- <br />                                                                                 tion image salivary gland (see box H) show <br />                                                                                 NS3 localization proximal coincident nu- <br />                                                                                 clear envelope, cellular membrane, cyto- <br />                                                                                 plasmic puncta (I), DAPI (J), anti-NS3. Scale bar, 5 mm. <br />                                                                                 (K) Larval CNS stained anti-NS3 showing ubiq- <br />                                                                                 uitous distribution protein. Scale bar, 50 mm. (L) <br />                                                                                 NS3 antibody staining ddc-gal4; uas-TdTomato-labeled <br />                                                                                 ddc neurons. NS3, green; ddc neurons, red; DAPI, <br />                                                                                 blue. Scale bar, 5 mm. (M&#226;&#8364;&#8220;M99) Wild-type ns3-yfp <br />                                                                                 driven ddc-gal4 demonstrating punctate locali- <br />                                                                                 zation NS3 cytoplasm. (N&#226;&#8364;&#8220;N99) Ns3-K350N-yfp <br />                                                                                 expressed using ddc-gal4 driver shows diffuse pat- <br />                                                                                 tern extends neurites. Scale bars, 50 mm. <br />  <br />  <br />  <br />  <br /> et al. 2003; Hedges et al. 2005). Human hLsg1, like NS3,         stained larval CNS anti-NS3 antibody. NS3 <br /> appears exist cytoplasmic foci nuclear pe-      appears ubiquitous larval CNS (Figure 4K), with <br /> riphery. However, unlike NS3 localization, hLsg1 ob-         sign unique NS3 patterns ddc-expressing cells (Figure <br /> served nucleus colocalized Cajal bodies      4L). Thus, special requirement NS3 ddc cells not <br /> as coincident ER (Reynaud et al. 2005).                     re&#239;&#172;&#8218;ected visibly distinct protein localization pattern. <br />    Our rescue ns3G0431 mutant ns3-yfp expressed         The GTPase domain required NS3 function (Figure 3). <br /> in neurons (elav . ns3-yfp) suggests hypersensitivity neu-    We observe intriguing difference NS3&#226;&#8364;&#8220;YFP and <br /> rons low NS3 dose (Kaplan et al. 2008). Ns3G0431     NS3&#226;&#8364;&#8220;K350N&#226;&#8364;&#8220;YFP protein localization neurons. NS3&#226;&#8364;&#8220;YFP is <br /> rescued expressing ns3-yfp using ddc-gal4, driver   strongly enriched punctate structures, NS3&#226;&#8364;&#8220; <br /> active dopaminergic (DA) 5-HT neurons (Li            K350N&#226;&#8364;&#8220;YFP diffuse spreads from <br /> et al. 2000). Given majority cells express      cell body axons dendrites (Figure 4, M and <br /> both elav ddc-gal4 drivers DA 5-HT neurons,          N). GTP coordination hydrolysis necessary for <br /> NS3 unique role cells. To investigate        proper NS3 localization. <br /> more closely NS3 special localization pattern <br />                                                                  Lower levels ribosomal proteins RpL13 RpS6 <br /> in 5-HT and/or DA neurons, compared staining levels of <br />                                                                  cells reduced ns3 <br /> NS3 ddc-gal4-expressing cells neighboring cells do <br /> not express ddc-gal4 (Figure 4, K L). To visualize ddc-      S. cerevisiae lsg1 loss-of-function mutants cause retention of <br /> gal4-expressing neurons, drove mtd-Tomato ddc-gal4       60S subunit protein RpL25 nucleoli, indicating that <br />  <br />  <br />  <br />                                                                                          Nucleostemin 3 Growth Control          109 <br />  Lsg1 required 60S export. Lsg1 works facilitating <br /> recycling export adapter (Kallstrom et al. 2003). We <br /> examined NS3 functions 60S subunit export <br /> by observing localization 60S subunit protein <br /> RpL13 salivary gland cells ns3 knocked <br /> down RNAi. We expressed ns3-dsRNA c061-gal4 <br /> (c061 . ns3-dsRNA), driver active variable levels <br /> in cells salivary glands. This convenient tool enabled <br /> the assessment ns3-RNAi knockdown consequence rela- <br /> tive nearby wild-type cells ns3-dsRNA was <br /> expressed. A transgene, uas-lamin-gfp, included <br /> to mark cells expressing ns3-dsRNA. <br />    In wild-type cells lacking lamin&#226;&#8364;&#8220;GFP lack- <br /> ing ns3-dsRNA, RpL13 localized nucleolus, espe- <br /> cially outer layer, nucleoplasm, the <br /> cytoplasm. In cells ns3 knocked down, identi&#239;&#172;?ed by <br /> their lamin&#226;&#8364;&#8220;GFP consequently known expressing <br /> ns3-dsRNA, cytoplasmic RpL13 levels lower (Figure 5, <br /> A&#226;&#8364;&#8220;D). To quantify results, average pixel intensities of <br /> cytoplasmic RpL13 measured normalized <br /> by dividing RpL13 levels wild-type cells ns3- <br /> dsRNA-expressing cells. Cytoplasmic RpL13 levels wild- <br /> type cells 1.4 6 0.13 times greater ns3-dsRNA <br /> expressing cells. This contrasts unchanged levels of <br /> RpL13 control salivary gland tissues analyzed parallel <br /> that did carry ns3-dsRNA produced lamin&#226;&#8364;&#8220;GFP <br /> (1.1 6 0.08, P = 0.04, t-test) (Figure 5I). <br />    To determine effect ns3 knockdown was <br /> a general in&#239;&#172;&#8218;uence ribosomes speci&#239;&#172;?c effect <br /> on RpL13, examined localization levels RpS6,       Figure 5 NS3 RNAi reduces cytoplasmic levels RpL13 RpS6 in <br /> a component 40S subunit. RpS6 observed         salivary glands. (A&#226;&#8364;&#8220;D) Anti-RpL13 staining wild-type ns3- <br /> cytoplasm wild-type ns3-dsRNA-expressing cells       dsRNA-expressing salivary gland cells. The variably active salivary gland <br />                                                                  driver c061-gal4 used co-express lamin-gfp ns3-dsRNA. We <br /> although, like RpL13, levels signi&#239;&#172;?cantly reduced    observed consequences RpL13 localization relative nearby non- <br /> cells ns3 knockdown (Figures 5, E&#226;&#8364;&#8220;H). Wild-type cells      lamin-gfp/non-ns3-dsRNA expressing cells. (A) Lamin&#226;&#8364;&#8220;GFP marks nu- <br /> had 6.0 6 1.8 times RpS6 neighboring cells             cleus cell expressing ns3-dsRNA dotted circle highlights <br /> expressing ns3-dsRNA. This contrasts control salivary       nucleus non-ns3-dsRNA-expressing cell. (B) DAPI staining to <br /> glands did express ns3-dsRNA (1.7 6 0.2, P =            visualize DNA. (C) Anti-RpL13 staining indicates RpL13 levels are <br />                                                                  decreased cytoplasm cells expressing ns3-dsRNA. (D) Merged <br /> 0.005, t-test) (Figure 5I). Reduced cytoplasmic RpL13        images A&#226;&#8364;&#8220;C. (E&#226;&#8364;&#8220;H) Anti-RpS6 staining c061 . lamin-gfp, ns3-dsRNA <br /> RpS6 ns3-dsRNA consistent role NS3        salivary glands. (E) Lamin&#226;&#8364;&#8220;GFP staining indicates ns3-dsRNA expressing <br /> 60S subunit export.                                              cells, dotted circle highlights non-ns3-dsRNA expressing cell. <br />                                                                  (F) DAPI visualize DNA. (G) Anti-RpS6 staining indicates RpS6 levels <br /> ns3G0431 developmental defects rescued                  reduced cytoplasm ns3-dsRNA-expressing cells. (H) Merged <br /> ns3-yfp expression driven dopaminergic                    images E&#226;&#8364;&#8220;G. (I) Levels cytoplasmic RpL13 RpS6 non-lamin- <br /> th-gal4 driver                                                   gfp/non-ns3-dsRNA quanti&#239;&#172;?ed imageJ normalized to <br />                                                                  neighboring lamin-gfp/ns3-dsRNA-expressing cells focal <br /> Previously, panel 15 gal4 drivers expression cell   plane. Levels RpL13 RpS6 reduced cytoplasm after <br /> subpopulations used express ns3-yfp ns3G0431      ns3 RNAi. Scale bars, 5 mm. <br /> mutant background test rescue developmental <br /> delay small-body-size phenotypes (Kaplan et al. 2008).          To de&#239;&#172;?ne highly affected defective <br /> From panel, elav-gal4 (elav . ns3-yfp) ddc-    neurons ns3G0431 mutant, tested rescue ns3- <br /> gal4-driven ns3-yfp (ddc . ns3-yfp) rescued mutant.          yfp expression driven newly available dtrh-gal4 <br /> Given elav-gal4 neuron speci&#239;&#172;?c ddc-gal4      (Alekseyenko et al. 2010), active 5-HT neurons. <br /> expressed larval 5-HT- DA-positive cells, ns3        We retested th-gal4 DA neurons (Friggi-Grelin et al. <br /> mutation likely affects cells intersection     2003). As expected, ddc . ns3-yfp provided strong rescue: <br /> drivers&#226;&#8364;&#8482; expression patterns: 5-HT and/or DA neurons. Lack       76.7 6 1.4% ns3G0431/Y; ddc . ns3-yfp &#239;&#172;&#8218;ies eclosed 10 <br /> of rescue th-gal4 driver led earlier conclusion    days embryos deposited (AEL) (Figure 6A). This <br /> that 5-HT neurons important.                       contrasts negative control ns3G0431/Y; ns3-yfp/+ <br />  <br />  <br />  <br /> 110    T. A. Hartl et al. <br />                                                                                   &#239;&#172;&#8218;ies, gal4 driver present &#239;&#172;&#8218;ies (52.2 6 <br />                                                                                  4.1%) eclosed 2 days later day 12 AEL. Driving NS3&#226;&#8364;&#8220;YFP <br />                                                                                  production dtrh . ns3-yfp did rescue, ns3G0431/ <br />                                                                                  Y &#239;&#172;&#8218;ies th . ns3-yfp substantially rescued: the <br />                                                                                  largest percentage &#239;&#172;&#8218;ies eclosed day 10 (41.8 6 <br />                                                                                  1.2%). Adding 5-HT driver, dtrh-gal4 th-gal4 did not <br />                                                                                  improve rescue: 49.5 6 1.8% ns3G0431/Y; dtrh, <br />                                                                                  th . ns3-yfp &#239;&#172;&#8218;ies eclosed day 10. This trend matched <br />                                                                                  looking viability levels. Only 49.3 6 3.5% of <br />                                                                                  ns3G0431/Y; ns3-yfp/+ 49.3 6 5.1% ns3G0431/Y; <br />                                                                                  dtrh . ns3-yfp males survived adulthood, 81.9 6 <br />                                                                                  2.4%, 72.1 6 5.6%, 70.1 6 4.2% expected ns3G0431/ <br />                                                                                  Y males carrying ddc . ns3-yfp, th . ns3-yfp, dtrh, th . <br />                                                                                  ns3-yfp ns3G0431/Y survived (Figure 6B). <br />                                                                                     Given cells activate elav-gal4 th- <br />                                                                                  gal4 DA neurons, suspect DA neurons are <br />                                                                                  especially prone defects NS3 dose reduced the <br />                                                                                  hypomorphic ns3G0431 background. The level NS3 in <br />                                                                                  ns3G0431 DA neurons relative surrounding non-DA neu- <br />                                                                                  rons unchanged relative wild type, ruling an <br />                                                                                  allele-speci&#239;&#172;?c effect NS3 dosage DA neurons (Figure <br />                                                                                  S3). The role NS3 DA neurons support <br />                                                                                  proper translational regulation transcripts unique and <br />                                                                                  necessary DA function, DA neurons espe- <br />                                                                                  cially high demand ef&#239;&#172;?cient protein synthesis. Although <br />                                                                                  role DA wild-type growth directly ob- <br />                                                                                  served, rescue growth defects ns3G0431 elav-, <br />                                                                                  ddc-, th-gal4-driven uas-ns3-yfp implies DA neurons <br /> Figure 6 The delayed development viability ns3G0431 mutants           provide critical functions proper larval developmental <br /> be rescued restoration ns3 function th-gal4-positive cells. (A)         progression. <br /> Developmental time eclosion ns3G0431/Y males without <br /> expression ns3-yfp DA, 5-HT, DA + 5-HT neurons. Five large <br /> crosses set males (1) Gal4 driver, (2) DA neuron-             Discussion <br /> enriched th-gal4, (3) 5-HT neuron enriched dtrh-gal4, (4) DA 5-HT- <br /> enriched ddc-gal4, (5) DA 5-HT enriched th-gal4, dtrh-gal4            Evolutionary conservation possible divergence <br /> ns3G0431/FM7i, act . gfp; uas-ns3-yfp virgin females. Embryos col-          Lsg1/NS3 functionality <br /> lected yeast/molasses agar plates overnight aged 24 hr. Forty <br />                                                                                  In yeast, Lsg1p cytoplasmic GTPase associated the <br /> act . gfp negative, ns3G0431/Y male, ns3G0431/+ female larvae were <br /> transferred standard molasses food vials triplicate. Eclosion &#239;&#172;&#8218;ies   60S ribosomal subunit. It required releasing the <br /> after embryos laid (AEL) scored daily, plot A indicates      nuclear export adapter Nmd3p 60S ribosomal <br /> the percentage &#239;&#172;?nal total number eclosed ns3G0431/Y &#239;&#172;&#8218;ies          subunit indirectly promotes 60S nuclear <br /> eclosed indicated day AEL. Most ns3G0431/Y &#239;&#172;&#8218;ies gal4              export (Kallstrom et al. 2003; Hedges et al. 2005). In this <br /> driver (dark blue) eclosed 12 days AEL (N = 15, 7, 13 ns3G0431/Y &#239;&#172;&#8218;ies <br />                                                                                  study, expression &#239;&#172;&#8218;y NS3 yeast cells was <br /> for 3 replicates). Expression ns3-yfp ns3G0431/Y &#239;&#172;&#8218;ies the <br /> ddc-gal4 (DA + 5-HT, red, N = 25, 20, 16 &#239;&#172;&#8218;ies) th-gal4 (DA,              suf&#239;&#172;?cient rescue Lsg1p de&#239;&#172;?ciency, con&#239;&#172;?rming orthol- <br /> purple, N = 14, 18, 16 &#239;&#172;&#8218;ies) provided substantial rescue,          ogy. Thus NS3 appears ensure proper export 60S <br /> &#239;&#172;&#8218;ies eclosing 10 days AEL. Expression ns3-yfp dtrh-gal4 (5-HT,           subunit nucleus, like Lsg1p. Consistent this <br /> green, N = 10, 7, 15 &#239;&#172;&#8218;ies) did rescue ns3G0431/Y developmental           view, levels RpL13, large ribosomal protein, lower <br /> delay, dtrh-gal4 combination th-gal4 (DA + 5-HT, light blue, <br />                                                                                  cytoplasm salivary gland cells ns3 was <br /> N = 17, 11, 20 &#239;&#172;&#8218;ies) did signi&#239;&#172;?cantly enhance rescue provided by <br /> th-gal4 (purple). (B) Survival adulthood ns3G0431/Y males       knocked RNAi (Figures 5, A&#226;&#8364;&#8220;D I). Levels of <br /> and expression ns3-yfp DA, 5-HT, DA + 5-HT neurons.            small ribosomal subunit protein, RpS6, reduced <br /> The total number ns3G0431/Y male &#239;&#172;&#8218;ies eclosed predicted         ns3 knockdown (Figure 5, E&#226;&#8364;&#8220;H, 5I). Therefore, NS3 <br /> number (40 2 number ns3G0431/+ females) replicates              likely contributes steps culminate in <br /> vials carried 40 ns3G0431/Y ns3G0431/+ larvae plotted. Only <br />                                                                                  proper protein synthesis. <br /> 49.3 6 3.5% ns3G0431/Y; ns3-yfp/+ 49.3 6 5.1% ns3G0431/ <br /> Y; dtrh . ns3-yfp males survived adulthood. In contrast, 81.9 6 2.4% <br /> ns3G0431/Y males carrying ddc . ns3-yfp 72.1 6 5.6% carrying th . <br /> ns3-yfp eclosed. This indicated expression ns3-yfp ns3G0431/Y <br /> ddc/th-gal4+ cells, covering DA neurons, restored ns3G0431/Y viabil-        rescue provided th-gal4 alone, 70.1 6 4.2% ns3G0431/Y; dtrh, th . <br /> ity. Inclusion dtrh-gal4 driver th-gal4 did enhance          ns3-yfp males survived. <br />  <br />  <br />  <br />                                                                                                            Nucleostemin 3 Growth Control        111 <br />     We previously showed expression uas-ns3-yfp            G3&#226;&#8364;&#8220;G4&#226;&#8364;&#8220;G5. Lsg1 NS3 contain circularly permuted G- <br /> transgene control ddc-gal4 strongly res-         domain motifs instead ordered G4&#226;&#8364;&#8220;G5&#226;&#8364;&#8220;G1&#226;&#8364;&#8220;G2&#226;&#8364;&#8220;G3 <br /> cue ns3G0431 mutant. Expression uas-lsg1        (Figure 3A). This structure common GTPases involved <br /> fashion suf&#239;&#172;?cient provide rescue ns3G0431         ribosome biogenesis. Several ribosome biogenesis pro- <br /> mutant (Figures 1, C D). This apparent contradiction            teins belong family circularly permuted GTPases that <br /> may explained features &#239;&#172;&#8218;y NS3 suggest func-           expanded eukarya single circularly permuted <br /> tional divergence yeast Lsg1p. NS3 localizes         GTPase, YlqF (Reynaud et al. 2005) (Table 1). YlqF itself <br /> proximal nuclear envelope plasma membrane                promotes ribosome biogenesis necessary 70S as- <br /> and cytoplasmic puncta. Lsg1p de-           sembly Bacillus subtilis (Uicker et al. 2006). <br /> scribed subcellular regions exists through-          In spite atypical structures, circularly <br /> out cytoplasm (Kallstrom et al. 2003; Hedges et al.             permuted GTPases YRG family been <br /> 2005). The distinctive localization features NS3 re-         demonstrated genuine GTPases using crystal struc- <br /> &#239;&#172;&#8218;ect special features acquired metazoan            tures vitro assays (Daigle Brown 2004; Levdikov <br /> lineage, precluding yeast Lsg1p rescuing &#239;&#172;&#8218;y ns3            et al. 2004; Shin et al. 2004; Reynaud et al. 2005; Bassler <br /> mutant.                                                             et al. 2006; Cladiere et al. 2006; Matsuo et al. 2007; Moreau <br />    The human protein orthologous Lsg1p/NS3            et al. 2008; Sudhamsu et al. 2008). Altered G-domain order- <br /> hLsg1 (Reynaud et al. 2005). Like yeast &#239;&#172;&#8218;y Lsg1p/NS3,           ing does change motif folds tertiary <br /> hLsg1 necessary cell viability. It unclear        structure. Here demonstrate NS3 pro&#239;&#172;?cient in <br /> hLsg1 speci&#239;&#172;?c roles ribosome biogenesis,            GTP hydrolysis proper GTPase activity confer <br /> this likely given protein sequence similarity yeast   special protein localization properties neurons, perhaps <br /> ortholog. NS3 hLsg1 share localization properties. Both         affecting association proteins. <br /> proteins coincident nuclear envelope lo-               Based previous studies GTPases, replacement of <br /> cated unidenti&#239;&#172;?ed cytoplasmic puncta (Figure 4J Fig-         lysine position 350 NS3 asparagine (K350N) is <br /> ure S1) (Reynaud et al. 2005). We did observe NS3            predicted disrupt coordination catalysis GTP <br /> the nucleus, hLsg1 observed nuclear Cajal bodies         (Saraste et al. 1990). We production protein <br /> that contain factors involved pre-ribosomal RNA             mutation conferred antimorphic activities &#239;&#172;&#8218;ies <br /> and mRNA processing (Reynaud et al. 2005). Just NS3              (Figure 3). This mimics occurs S. cerevisiae an <br /> and Lsg1p functional differences, hLsg1 appears             analogous mutation (Hedges et al. 2005). We suggest that <br /> have features functions different NS3.                     antimorphic NS3(K350N) blocks important functions, or <br />                                                                     sequesters essential cofactors away endogenous wild- <br /> Cell-autonomous role NS3 revealed new alleles <br />                                                                     type NS3. GTP hydrolysis necessary remove Nmd3 <br /> In yeast human cells, Lsg1 essential cell viability      60S subunit enable return nucleus for <br /> and growth (Kallstrom et al. 2003; Hedges et al. 2005;              round 60S export. In scenario, NS3(K350N) <br /> Reynaud et al. 2005), Drosophila ns3G0431 allele            prevent GTPase-competent wild-type NS3 re- <br /> was partially lethal. Many ns3G0431 &#239;&#172;&#8218;ies survive            moving Nmd3. On hand, NS3(K350N) altered <br /> adulthood, albeit developmental delay decreased            localization neurons appeared spread cell <br /> overall size (Kaplan et al. 2008). Here identi&#239;&#172;?ed            bodies neurites. The altered localization sequester <br /> alleles previously uncharacterized locus l(1)2Ad          important cofactors away endogenous NS3 thus <br /> new alleles ns3. Both ns3l(1)2Ad1 ns3l(1)2Ad3,         block ribosome biogenesis activities. <br /> are predicted severe truncation protein, cause <br />                                                                     Identifying neuronal subsets involved growth control <br /> 100% lethality likely strong loss-of-function <br /> alleles. In contrast, ns3G0431 lesion P-element inser-     Previously, concluded ns3 functioned 5- <br /> tion 59-UTR, cause subtle loss-of-          HT neurons promote growth (Kaplan et al. 2008). The <br /> function. The apparent inconsistency cell lethality         primary data leading conclusion expression <br /> in lsg1 mutant yeast mammalian cells semiviability          uas-ns3-yfp 5-HT DA neurons (ddc . ns3-yfp) <br /> in &#239;&#172;&#8218;y cells resolved lethality stronger loss-     rescued ns3G0431 growth defects, expression in <br /> of-function ns3 alleles. We suspect Drosoph-       DA neurons (th . ns3-yfp) did provide rescue. <br /> ila cells depend NS3 ribosome biogenesis               We longer believe second, negative, result. Since <br /> therefore growth viability. Flies ns3G0431 allele      study, apparent penetrance of <br /> likely survive adulthood subtle drop NS3        ns3G0431 phenotypes highly dependent genetic back- <br /> levels.                                                             ground. For instance, original background 5-HT <br />                                                                     driver line (dtrh-gal4) nearly 100% lethal placed <br /> A family noncanonical GTPases uni&#239;&#172;?ed roles <br />                                                                     trans ns3G0431 stock. We backcrossed ddc- <br /> in ribosome biogenesis <br />                                                                     gal4, dtrh-gal4, th-gal4 &#239;&#172;?ve generations iso- <br /> The canonical GTPase structure, based Ras, carries G-            genized genetic background (Gohl et al. 2011) we <br /> domain motifs ordered N C terminus G1&#226;&#8364;&#8220;G2&#226;&#8364;&#8220;            semiviable trans ns3G0431. A second <br />  <br />  <br />  <br /> 112    T. A. Hartl et al. <br />  demonstration importance levels rescuing pro-         A higher demand NS3 neurons <br /> tein comes manipulating temperature experi- <br />                                                                    Strong alleles ns3 reveal cell-autonomous growth roles <br /> ments GAL4 activity level. Our previous <br />                                                                    (Figures 1&#226;&#8364;&#8220;5), weak ns3G0431 allele reveals that <br /> studies performed 25 . The present studies were <br />                                                                    ddc/th-gal4-expressing neurons (possibly DA neurons) are <br /> conducted 29 , signi&#239;&#172;?cantly boosts expression of <br />                                                                    sensitive reduced levels NS3. Based high sequence <br /> Gal4-driven transgenes. Under conditions boosted <br />                                                                    homology, likely NS3 promotes ribosome biogenesis. <br /> gal4 expression pure genetic backgrounds, expression of <br />                                                                    Thus, hypomorphic ns3G0431 allele indicate special <br /> uas-ns3-yfp DA neurons (th . ns3-YFP) provided substan- <br />                                                                    ribosome biogenesis requirements ddc/th-gal4-expressing <br /> tial rescue. <br />                                                                    neurons and/or unique dependency speci&#239;&#172;?c types of <br />     Expression ns3 different Gal4 drivers (elav- <br />                                                                    translational regulation. <br /> gal4, ddc-gal4, th-gal4) suf&#239;&#172;?cient provide <br />                                                                        We propose following alternative working models to <br /> signi&#239;&#172;?cant rescue ns3G0431 mutant. The intersection <br />                                                                    explain ddc/th-gal4 hypersensitivity NS3 dosage. As the <br /> between drivers&#226;&#8364;&#8482; expression patterns consists DA <br />                                                                    NS3 yeast ortholog (Lsg1) promotes export 60S subunits <br /> neurons, suggesting ns3 important role these <br />                                                                    nuclei, NS3 metazoans necessary <br /> cells promote growth. Perhaps factors involved DA <br />                                                                    positioning ribosomes specialized structures <br /> signaling-speci&#239;&#172;?c properties especially high require- <br />                                                                    neurons. Most ribosomal subunits appear neuronal <br /> ment protein-translation ef&#239;&#172;?ciency. Such growth-promoting <br />                                                                    cell body, instances ribosomes localized <br /> properties DA neurons include broad-reaching me- <br />                                                                    axons dendrites (Craig Banker 1994; Brittis <br /> tabolism regulation like insulin-like peptides (ILPs), <br />                                                                    et al. 2002; Kiebler Bassell 2006; Rolls et al. 2007). This <br /> as previously suggested (Kaplan et al. 2008). ILPs pro- <br />                                                                    subcellular targeting ribosomes, causing move <br /> duced small cluster neurosecretory cells known as <br />                                                                    long distances, require specialization ribo- <br /> ILP-producing cells (IPCs) (Rulifson et al. 2002). ILPs accu- <br />                                                                    biogenesis proteins like NS3. <br /> mulate IPCs ns3G0431 mutants (Kaplan et al. 2008). <br />                                                                        NS3 instead translationally regulate speci&#239;&#172;?c mRNA <br /> Perhaps speci&#239;&#172;?c signaling defects DA neurons the <br />                                                                    subsets. Given DA neurons appear especially <br /> ns3G0431 mutant lead failed ILP modi&#239;&#172;?cation and/or se- <br />                                                                    sensitive NS3 dose, mRNAs that <br /> cretion IPCs. DA neurons regulate <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br />                                                                    code DA synthesis, degradation, transport, synaptic <br /> acquisition activities levels appetite <br />                                                                    dynamics translationally regulated NS3. A mRNA- <br /> control, <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> absorption, feeding activities. <br />                                                                    speci&#239;&#172;?c translational regulation function make NS3 <br />     5-HT signaling altered ns3G0431 back- <br />                                                                    analogous mouse RpL38. This large subunit ribosomal <br /> ground. In keeping possibility, 5-HT levels were <br />                                                                    protein binds speci&#239;&#172;?cally subset Hox gene mRNAs <br /> elevated mutant adult heads. Feeding 5-HT precursor, <br />                                                                    controls association 80S ribosome. Mice <br /> 5-HTP, enhanced ns3G0431 phenotypes (Kaplan et al. <br />                                                                    carrying mutant copy RpL38 speci&#239;&#172;?c homeotic <br /> 2008). Those results consequence primary <br />                                                                    transformations failed translation Hox <br /> defect ns3 loss. Recent studies implicated 5-HT in <br />                                                                    mRNAs (Kondrashov et al. 2011). <br /> promoting activities IPCs, turn neces- <br />                                                                        Another unique translational regulation role NS3 <br /> sary growth. Developmental growth roles 5-HT were <br />                                                                    play proper localization translational <br /> not demonstrated, expression 5-HT1A recep- <br />                                                                    regulation certain mRNAs synapse. Localized trans- <br /> tor gal4 line IPCs (Luo et al. 2012) suggests 5-HT <br />                                                                    lation unique neurons, RNA-binding <br /> may regulate IPC-driven growth certain circumstances. <br />                                                                    proteins transport, tether, activate, and/or repress <br />     The ns3G0431 mutant appears sensitized genetic <br />                                                                    translation identi&#239;&#172;?ed screens molecules <br /> condition, important growth control ddc-gal4 and <br />                                                                    involved neuronal activities (Costa-Mattioli et al. 2009). <br /> th-gal4 (ddc/th-gal4)-expressing cells revealed. Alterna- <br />                                                                    Perhaps NS3 necessary localized translational regula- <br /> tively, growth control ddc/th-gal4 cells happen <br />                                                                    tion neurons binding certain mRNAs transporting <br /> in wild-type animals, ns3G0431 mutants abnormal <br />                                                                    ribosomal subunits relevant site. Further explorations <br /> circuit sort occurs. If aspect neuronal signal- <br />                                                                    ribosome biogenesis, diversity ribosomal struc- <br /> ing subfunctional ns3G0431 mutant, restoration <br />                                                                    tures activities, likely reveal speci&#239;&#172;?c developmental <br /> of ddc/th-gal4 cellular health expression uas-ns3-yfp <br />                                                                    physiological functions carried presumed &#226;&#8364;&#339;house- <br /> could compensate neuronal defects elsewhere. Although <br />                                                                    keeping&#226;&#8364;? proteins. <br /> ddc/th-gal4-expressing cells convey growth-promoting <br /> activities wild-type &#239;&#172;&#8218;ies, DA independent. <br /> Both drivers active small number non- <br />                                                                    Acknowledgments <br /> dopaminergic neurons various tissues may <br /> be true source nonautonomous growth control. In             We thank Michael Dellinger critiquing manuscript. <br /> addition, ddc/th-gal4 growth control neuronal,          We grateful Kevin Cook Bloomington Dro- <br /> mediated signaling molecule DA             sophila Stock Center help identifying stocks carrying pu- <br /> a neuropeptide, hormone, different neurotransmitter.            tative ns3 alleles. We thank Kathleen Beckingham, Sangbin <br />  <br />  <br />  <br />                                                                                           Nucleostemin 3 Growth Control   113 <br />  Park, Seung Kim, Wendy Neckameyer, Jay Hirsh, Eric Rulifson,            Gadal, O., D. Strauss, J. Kessl, B. Trumpower, D. Tollervey et al., <br /> Ron Alfa, members Tom Clandinin&#226;&#8364;&#8482;s laboratory                     2001 Nuclear export 60s ribosomal subunits depends on <br />                                                                             Xpo1p requires nuclear export sequence-containing factor, <br /> helpful discussions. The research supported National <br />                                                                             Nmd3p, associates large subunit protein Rpl10p. <br /> Institutes Health grant GM53655 A.J.                       Mol. Cell. Biol. 21: 3405&#226;&#8364;&#8220;3415. <br /> Howard Hughes Medical Institute. T.H. Robert Black                 Gohl, D. M., M. A. Silies, X. J. Gao, S. Bhalerao, F. J. Luongo et al., <br /> fellow Damon Runyon Cancer Research Foundation                       2011 A versatile vivo directed dissection gene <br /> (DRG-2045-10). J.N. postdoctoral fellow, M.P.S.                 expression patterns. Nat. Methods 8: 231&#226;&#8364;&#8220;237. <br />                                                                         Hedges, J., M. West, A. W. Johnson, 2005 Release the <br /> Investigator, Howard Hughes Medical Institute. <br />                                                                             export adapter, Nmd3p, 60S ribosomal subunit re- <br />                                                                             quires Rpl10p cytoplasmic GTPase Lsg1p. EMBO J. <br />                                                                             24: 567&#226;&#8364;&#8220;579. <br /> Literature Cited                                                        Henras, A. K., J. Soudet, M. Gerus, S. Lebaron, M. Caizergues-Ferrer <br />                                                                             et al., 2008 The post-transcriptional steps eukaryotic ribo- <br /> Alekseyenko, O. V., C. Lee, E. A. Kravitz, 2010 Targeted ma-            biogenesis. Cellular molecular life sciences. Cell. <br />     nipulation serotonergic neurotransmission affects escala-        Mol. Life Sci. 65: 2334&#226;&#8364;&#8220;2359. <br />     tion aggression adult male Drosophila melanogaster. PLoS      Himeno, H., K. Hanawa-Suetsugu, T. Kimura, K. Takagi, W. <br />     ONE 5: e10806.                                                          Sugiyama et al., 2004 A novel GTPase activated small <br /> Barrientos, A., D. Korr, K. J. Barwell, C. Sjulsen, C. D. Gajewski          subunit ribosome. Nucleic Acids Res. 32: 5303&#226;&#8364;&#8220;5309. <br />     et al., 2003 MTG1 codes conserved protein required        Ho, J. H., G. Kallstrom, A. W. Johnson, 2000 Nmd3p is <br />     mitochondrial translation. Mol. Biol. Cell 14: 2292&#226;&#8364;&#8220;2302.               Crm1p-dependent adapter protein nuclear export the <br /> Bassler, J., P. Grandi, O. Gadal, T. Lessmann, E. Petfalski et al.,         large ribosomal subunit. J. Cell Biol. 151: 1057&#226;&#8364;&#8220;1066. <br />     2001 Identi&#239;&#172;?cation 60S preribosomal particle           Hurt, E., S. Hannus, B. Schmelzl, D. Lau, D. Tollervey et al., <br />     closely linked nuclear export. Mol. Cell 8: 517&#226;&#8364;&#8220;529.                 1999 A novel vivo assay reveals inhibition ribosomal <br /> Bassler, J., M. Kallas, E. Hurt, 2006 The NUG1 GTPase reveals           nuclear export ran-cycle nucleoporin mutants. J. Cell <br />     N-terminal RNA-binding domain essential asso-           Biol. 144: 389&#226;&#8364;&#8220;401. <br />     ciation 60 S pre-ribosomal particles. J. Biol. Chem. 281:      Kallstrom, G., J. Hedges, A. Johnson, 2003 The putative <br />     24737&#226;&#8364;&#8220;24744.                                                            GTPases Nog1p Lsg1p required 60S ribosomal sub- <br /> Bernards, A., J. Settleman, 2004 GAP control: regulating            <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> biogenesis localized nucleus cytoplasm, <br />     regulators small GTPases. Trends Cell Biol. 14: 377&#226;&#8364;&#8220;385.             respectively. Mol. Cell. Biol. 23: 4344&#226;&#8364;&#8220;4355. <br /> Boddapati, N., K. Anbarasu, R. Suryaraja, A. V. Tendulkar, S.       Kaplan, D. D., G. Zimmermann, K. Suyama, T. Meyer, M. P. <br />     Mahalingam, 2012 Subcellular distribution human pu-              Scott, 2008 A nucleostemin family GTPase, NS3, acts sero- <br />     tative nucleolar GTPase GNL1 regulated novel arginine/          tonergic neurons regulate insulin signaling control body <br />     lysine-rich domain GTP binding domain cell cycle-            size. Genes Dev. 22: 1877&#226;&#8364;&#8220;1893. <br />     dependent manner. J. Mol. Biol. 416: 346&#226;&#8364;&#8220;366.                       Kaziro, Y., H. Itoh, T. Kozasa, M. Nakafuku, T. Satoh, <br /> Bridges, C. B., T. H. Morgan, 1923 The Third Chromosome                 1991 Structure function signal-transducing GTP-bind- <br />     Group Mutant Characters Drosophila melanogaster. Carnegie         ing proteins. Annu. Rev. Biochem. 60: 349&#226;&#8364;&#8220;400. <br />     Institution, Washington, DC.                                        Kiebler, M. A., G. J. Bassell, 2006 Neuronal RNA granules: <br /> Brittis, P. A., Q. Lu, J. G. Flanagan, 2002 Axonal protein              movers makers. Neuron 51: 685&#226;&#8364;&#8220;690. <br />     synthesis provides mechanism localized regulation       Kondrashov, N., A. Pusic, C. R. Stumpf, K. Shimizu, A. C. Hsieh <br />     intermediate target. Cell 110: 223&#226;&#8364;&#8220;235.                                 et al., 2011 Ribosome-mediated speci&#239;&#172;?city Hox mRNA <br /> Cladiere, L., K. Hamze, E. Madec, V. M. Levdikov, A. J. Wilkinson           translation vertebrate tissue patterning. Cell 145: 383&#226;&#8364;&#8220;397. <br />     et al., 2006 The GTPase, CpgA(YloQ), putative translation         Lambertsson, A., 1998 The minute genes Drosophila their <br />     factor, implicated morphogenesis Bacillus subtilis. Mol.       molecular functions. Adv. Genet. 38: 69&#226;&#8364;&#8220;134. <br />     Genet. Genomics. 275: 409&#226;&#8364;&#8220;420.                                      Larkin, M. A., G. Blackshields, N. P. Brown, R. Chenna, P. A. McGettigan <br /> Costa-Mattioli, M., W. S. Sossin, E. Klann, N. Sonenberg,               et al., 2007 Clustal W Clustal X version 2.0. Bioinformatics 23: <br />     2009 Translational control long-lasting synaptic plasticity          2947&#226;&#8364;&#8220;2948. <br />     memory. Neuron 61: 10&#226;&#8364;&#8220;26.                                       Leipe, D. D., Y. I. Wolf, E. V. Koonin, L. Aravind, <br /> Craig, A. M., G. Banker, 1994 Neuronal polarity. Annu. Rev.             2002 Classi&#239;&#172;?cation evolution P-loop GTPases re- <br />     Neurosci. 17: 267&#226;&#8364;&#8220;310.                                                  lated ATPases. J. Mol. Biol. 317: 41&#226;&#8364;&#8220;72. <br /> Daigle, D. M., E. D. Brown, 2004 Studies interaction      Levdikov, V. M., E. V. Blagova, J. A. Brannigan, L. Cladiere, A. A. <br />     Escherichia coli YjeQ ribosome vitro. J. Bacteriol.         Antson et al., 2004 The crystal structure YloQ, circularly <br />     186: 1381&#226;&#8364;&#8220;1387.                                                         permuted GTPase essential Bacillus subtilis viability. J. Mol. <br /> Daigle, D. M., L. Rossi, A. M. Berghuis, L. Aravind, E. V. Koonin           Biol. 340: 767&#226;&#8364;&#8220;782. <br />     et al., 2002 YjeQ, essential, conserved, uncharacterized         Li, H., S. Chaney, I. J. Roberts, M. Forte, J. Hirsh, 2000 Ectopic <br />     protein Escherichia coli, unusual GTPase circu-         G-protein expression dopamine serotonin neurons blocks <br />     larly permuted G-motifs marked burst kinetics. Biochemis-           cocaine sensitization Drosophila melanogaster. Curr. Biol. 10: <br />     try 41: 11109&#226;&#8364;&#8220;11117.                                                    211&#226;&#8364;&#8220;214. <br /> Du, X., M. R. Rao, X. Q. Chen, W. Wu, S. Mahalingam et al.,             Lin, J. I., N. C. Mitchell, M. Kalcina, E. Tchoubrieva, M. J. Stewart <br />     2006 The homologous putative GTPases Grn1p &#239;&#172;?ssion                  et al., 2011 Drosophila ribosomal protein mutants control tis- <br />     yeast human GNL3L required growth play              sue growth non-autonomously effects prothoracic <br />     role processing nucleolar pre-rRNA. Mol. Biol. Cell 17:         gland ecdysone. PLoS Genet. 7: e1002408. <br />     460&#226;&#8364;&#8220;474.                                                            Lo, K. Y., Z. Li, C. Bussiere, S. Bresson, E. M. Marcotte et al., <br /> Friggi-Grelin, F., H. Coulom, M. Meller, D. Gomez, J. Hirsh et al.,         2010 De&#239;&#172;?ning pathway cytoplasmic maturation the <br />     2003 Targeted gene expression Drosophila dopaminergic                60S ribosomal subunit. Mol. Cell 39: 196&#226;&#8364;&#8220;208. <br />     cells using regulatory sequences tyrosine hydroxylase. J.      Luo, J., J. Becnel, C. D. Nichols, D. R. Nassel, 2012 Insulin- <br />     Neurobiol. 54: 618&#226;&#8364;&#8220;627.                                                 producing cells brain adult Drosophila regulated by <br />  <br />  <br />  <br /> 114     T. A. Hartl et al. <br />     serotonin 5&#226;&#8364;&#8220;HT1A receptor: cellular molecular life sci-          subunits required late 60S maturation steps. EMBO J. <br />    ences. Cell. Mol. Life Sci. 69: 471&#226;&#8364;&#8220;484.                                 20: 6475&#226;&#8364;&#8220;6484. <br /> Marygold, S. J., J. Roote, G. Reuter, A. Lambertsson, M. Ashburner       Schneider, C. A., W. S. Rasband, K. W. Eliceiri, 2012 NIH <br />    et al., 2007 The ribosomal protein genes Minute loci              Image ImageJ: 25 years image analysis. Nat. Methods 9: <br />    Drosophila melanogaster. Genome Biol. 8: R216.                           671&#226;&#8364;&#8220;675. <br /> Matsuo, Y., T. Oshima, P. C. Loh, T. Morimoto, N. Ogasawara,         Seiser, R. M., A. E. Sundberg, B. J. Wollam, P. Zobel-Thropp, K. <br />    2007 Isolation characterization dominant negative               Baldwin et al., 2006 Ltv1 required ef&#239;&#172;?cient nuclear ex- <br />    mutant Bacillus subtilis GTP-binding protein, YlqF, essential         port ribosomal small subunit Saccharomyces cerevisiae. <br />    biogenesis maintenance 50 S ribosomal subunit. J.         Genetics 174: 679&#226;&#8364;&#8220;691. <br />    Biol. Chem. 282: 25270&#226;&#8364;&#8220;25277.                                         Shin, D. H., Y. Lou, J. Jancarik, H. Yokota, R. Kim et al., <br /> Matsuo, E., T. Nagamine, S. Matsumoto, K. Tsuneizumi,                   2004 Crystal structure YjeQ Thermotoga maritima con- <br />    2011 Drosophila GTPase nucleostemin 2 changes cellular dis-              tains circularly permuted GTPase domain. Proc. Natl. Acad. <br />    tribution larval development GTP-binding motif            Sci. USA 101: 13198&#226;&#8364;&#8220;13203. <br />    essential nucleoplasmic localization. Biosci. Biotechnol. Bi-   Sigal, I. S., J. B. Gibbs, J. S. D&#226;&#8364;&#8482;Alonzo, G. L. Temeles, B. S. Wolanski <br />    ochem. 75: 1511&#226;&#8364;&#8220;1515.                                                    et al., 1986 Mutant ras-encoded proteins altered nucleo- <br /> Moreau, M., G. I. Lee, Y. Wang, B. R. Crane, D. F. Klessig,             tide binding exert dominant biological effects. Proc. Natl. Acad. <br />    2008 AtNOS/AtNOA1 functional Arabidopsis thaliana                   Sci. USA 83: 952&#226;&#8364;&#8220;956. <br />    cGTPase nitric-oxide synthase. J. Biol. Chem. 283:          Stage-Zimmermann, T., U. Schmidt, P. A. Silver, 2000 Factors <br />    32957&#226;&#8364;&#8220;32967. <br />                                                                             affecting nuclear export 60S ribosomal subunit vivo. <br /> Moy, T. I., P. A. Silver, 1999 Nuclear export small <br />                                                                             Mol. Biol. Cell 11: 3777&#226;&#8364;&#8220;3789. <br />    ribosomal subunit requires ran-GTPase cycle certain <br />                                                                          Staley, J. P., J. L. Woolford Jr., 2009 Assembly ribosomes <br />    nucleoporins. Genes Dev. 13: 2118&#226;&#8364;&#8220;2133. <br />                                                                             spliceosomes: complex ribonucleoprotein machines. Curr. <br /> Pai, E. F., U. Krengel, G. A. Petsko, R. S. Goody, W. Kabsch et al., <br />                                                                             Opin. Cell Biol. 21: 109&#226;&#8364;&#8220;118. <br />    1990 Re&#239;&#172;?ned crystal structure triphosphate conforma- <br />                                                                          Strunk, B. S., K. Karbstein, 2009 Powering ribosome <br />    tion H-ras p21 1.35 A resolution: implications the <br />                                                                             assembly. RNA 15: 2083&#226;&#8364;&#8220;2104. <br />    mechanism GTP hydrolysis. EMBO J. 9: 2351&#226;&#8364;&#8220;2359. <br />                                                                          Sudhamsu, J., G. I. Lee, D. F. Klessig, B. R. Crane, 2008 The <br /> Panse, V. G., A. W. Johnson, 2010 Maturation eukaryotic <br />    ribosomes: acquisition functionality. Trends Biochem. Sci. 35:        structure YqeH: AtNOS1/AtNOA1 ortholog couples <br />    260&#226;&#8364;&#8220;266.                                                                 GTP hydrolysis molecular recognition. J. Biol. Chem. 283: <br /> Perrimon, N., L. Engstrom, A. P. Mahowald, 1989 Zygotic                 32968&#226;&#8364;&#8220;32976. <br />    lethals speci&#239;&#172;?c maternal effect phenotypes Drosophila         Tsai, R. Y., R. D. McKay, 2002 A nucleolar mechanism con- <br />    melanogaster. I. Loci X chromosome. Genetics 121:                 trolling cell proliferation stem cells cancer cells. Genes <br />    333&#226;&#8364;&#8220;352.        <br /> </body></html>