<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                      MBoC&#226;&#8364;&#8218; |&#226;&#8364;&#8218; ARTICLE <br />  <br />  <br />  <br />  <br /> The E3 ubiquitin ligase mind bomb 1 <br /> ubiquitinates promotes degradation <br /> of <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> motor neuron protein <br /> Deborah Y. Kwona,b, Maria Dimitriadib, Barbara Terzica, Casey Cablea, Anne C. Hartb, Ajay Chitnisc, <br /> Kenneth H. Fischbecka, Barrington G. Burnetta <br /> a <br />  Neurogenetics Branch, National Institute Neurological Disorders Stroke, National Institutes Health, <br /> Bethesda, MD 20892; bDepartment Neuroscience, Brown University, Providence, RI 02912; cProgram Genomics <br /> of Development, Eunice Kennedy Shriver National Institute Child Health Human Development, National <br /> Institutes Health, Bethesda, MD 20892 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>&#226;&#8364;&#8218; Spinal muscular atrophy inherited motor neuron disease results                                  Monitoring Editor <br /> deficiency survival motor neuron (SMN) protein. SMN ubiquitinated degraded                                 William P. Tansey <br />                                                                                                                             Vanderbilt University <br /> through ubiquitin proteasome (UPS). We previously shown proteasome <br /> inhibition increases SMN protein levels, improves motor function, reduces spinal cord,                                  Received: Jan 18, 2013 <br /> muscle, neuromuscular junction pathology spinal muscular atrophy (SMA) mice. Spe-                                    Revised: Apr 8, 2013 <br /> cific targets UPS efficacious toxic. In study,                            Accepted: Apr 16, 2013 <br /> E3 ubiquitin ligase, mind bomb 1 (Mib1), interacts ubiquitinates SMN facilitates <br /> its degradation. Knocking Mib1 levels increases SMN protein levels cultured cells. <br /> Also, knocking Mib1 orthologue improves neuromuscular function Caenorhabditis <br /> elegans deficient SMN. These findings demonstrate Mib1 ubiquitinates catalyzes <br /> the degradation SMN, represents novel therapeutic target SMA. <br />  <br />  <br />  <br /> INTRODUCTION <br /> Spinal muscular atrophy (SMA) autosomal recessive neurologi-                     rapidly degraded (Coovert et&#194;&#160; al., 1997; Lorson et&#194;&#160; al., 1999; <br /> cal disorder characterized loss lower motor neurons, leading                     Lorson Androphy, 2000; Monani et&#194;&#160; al., 1999; Burnett et&#194;&#160; al., <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> weakness skeletal muscle atrophy, leading                  2009). Human SMN2 transgenic mice mitigates severity the <br /> genetic causes infant death. More 90% SMA results                      SMA disease <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> mouse smn-null background, a <br /> deletion survival motor neuron (SMN1) gene, patients                   gene dosage effect (Hsieh-Li et&#194;&#160;al., 2000). Patients mild SMA <br /> retaining copy homologue SMN2 (Lefebvre et&#194;&#160;al.,                    phenotypes SMN2 copies, indi- <br /> 1995). While SMN1 produces predominantly full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> SMN pro-                          viduals SMN2 genes pheno- <br /> tein, SMN2 contains translationally silent C-to-T transition                  typically normal (Lefebvre et&#194;&#160;al., 1995; McAndrew et&#194;&#160;al., 1997; Prior <br /> exon 7, causing exon skipped mRNA                             et&#194;&#160;al., 2004). <br /> splicing producing truncated protein (SMN&#206;&#8221;7) unstable                        SMN degraded ubiquitin proteasome system <br />                                                                                        (UPS; Chang et&#194;&#160;al., 2004; Burnett et&#194;&#160;al., 2009; Kwon et&#194;&#160;al., 2011). <br />                                                                                        In system, proteins destined degradation tagged link- <br />                                                                                        age ubiquitin action enzymes: E1 (ubiquitin- <br /> This article published online ahead print MBoC Press (http://www <br /> .molbiolcell.org/cgi/doi/10.1091/mbc.E13-01-0042) April 24, 2013.                   activating), E2 (ubiquitin-conjugating), E3 (ubiquitin ligase) <br /> The authors declare conflict interest.                            (Petroski, 2008). E3 ligases confer substrate specificity this <br /> Address correspondence to: Barrington G. Burnett (burnettb@ninds.nih.gov).             process considered candidates targeted <br /> Abbreviations used: Mib1, mind bomb 1; NMJ, neuromuscular junction; RING,              inhibition protein degradation. <br /> really interesting new gene; SMA, spinal muscular atrophy; SMN, survival motor <br /> neuron; UPS, ubiquitin proteasome system.                                                  We shown SMN protein levels increased cell <br /> &#194;&#169; 2013 Kwon et&#194;&#160;al. This article distributed The American Society Cell Bi-    culture animal models inhibition proteasome (Burnett <br /> ology license author(s). Two months publication available   et&#194;&#160;al., 2009; Kwon et&#194;&#160;al., 2011). Because protein degradation the <br /> to public Attribution&#226;&#8364;&#8220;Noncommercial&#226;&#8364;&#8220;Share Alike 3.0 Unported <br /> Creative Commons License (http://creativecommons.org/licenses/by-nc-sa/3.0). <br />                                                                                        UPS essential process cellular homeostasis general inhi- <br /> &#226;&#8364;&#339;ASCB&#194;&#174;,&#226;&#8364;? &#226;&#8364;&#339;The American Society Cell Biology&#194;&#174;,&#226;&#8364;? &#226;&#8364;&#339;Molecular Biology           bition undesired effects, sought identify char- <br /> the Cell&#194;&#174;&#226;&#8364;? registered trademarks The American Society Cell Biology.          acterize E3 ligase(s) responsible specifically targeting SMN <br />  <br />  <br /> Volume 24&#226;&#8364;&#402; June 15, 2013	                                                                                                                             1863&#226;&#8364;&#402; <br />  for degradation. We chose evaluate mind bomb 1 (Mib1)              uitinated proteins, consistent results cultured cells (Figure <br /> candidate, based role inhibiting outgrowth neurites       1C). Given Mib1 ubiquitinates SMN, sought quantify <br /> in cultured neurons (Choe et&#194;&#160;al., 2007). Loss Drosophila melano-        effect Mib1 SMN protein turnover. We performed pulse&#226;&#8364;&#8220; <br /> gaster Mib1 increases number synaptic boutons neuromus-          chase analysis using HEK-293T cells transfected wild-type <br /> cular junctions (NMJs), producing synaptic overgrowth, reduc-        Mib1-myc active-site mutant, Mib1-C1009S-myc, deter- <br /> tion SMN reduces number NMJ boutons D. melanogaster           E3 ligase activity Mib1 alters SMN protein half- <br /> and results aberrantly truncated motor neurons Danio rerio           life. We overexpressing wild-type Mib1 decreased the <br /> (McWhorter et&#194;&#160; al., 2003; Choe et&#194;&#160; al., 2007; Chang et&#194;&#160; al., 2008).        half-life newly synthesized radiolabeled SMN half, &#226;&#710;&#188;4 to <br /> Mib1 multidomain enzyme carboxyl-terminal really           2 h, compared active-site Mib1 mutant (Figure 1D). In addi- <br /> interesting new gene (RING) finger domains (Jin et&#194;&#160;al., 2002; Itoh         tion, overexpressing Mib1 NSC34 cells reduced steady-state <br /> et&#194;&#160;al., 2003; Yoo et&#194;&#160;al., 2006; Zhang et&#194;&#160;al., 2007). Its best-character-   levels SMN protein, effect blocked protea- <br /> ized role activating Delta-Notch signaling pathway,        inhibitor bortezomib (Figure 2A), indicating Mib1 targets <br /> controls differentiation regulating cell&#226;&#8364;&#8220;cell communication (Itoh       SMN proteasomal degradation. <br /> et&#194;&#160;al., 2003; Chen Casey Corliss, 2004; Barsi et&#194;&#160;al., 2005; Koo            The SMN complex assembled cytoplasm is <br /> et&#194;&#160; al., 2005a; Lai et&#194;&#160; al., 2005). Other functions Mib1      imported nucleus, localizes Cajal bodies or <br /> emerged. Mib1 identified postsynaptic densities,            gems. The number SMN-positive nuclear bodies been <br /> which shown function binding partner            used measure evaluating changes functional SMN protein <br /> CDK5 complex regulating morphogenesis postmitotic neurons            levels (Coovert et&#194;&#160;al., 1997; Jarecki et&#194;&#160;al., 2005; Wolstencroft et&#194;&#160;al., <br /> (Choe et&#194;&#160; al., 2007) regulating apoptosis promoting          2005). We counted number SMN-positive nuclear puncta per <br /> proteasomal degradation anti-apoptotic factor death-associ-         nucleus assessed fluorescence intensity HeLa cells over- <br /> ated protein kinase (DAPK; Jin et&#194;&#160;al., 2002). Recently overexpres-         expressing wild-type Mib1 mutant Mib1-C1009S. We that <br /> sion Mib1 shown promote ubiquitination receptor-          number intensity SMN-positive bodies significantly <br /> like tyrosine kinase (RYK), reducing steady-state levels        reduced cells overexpressing wild-type Mib1 compared un- <br /> plasma membrane (Berndt et&#194;&#160;al., 2011).                                     transfected cells (Alexa Fluor 488&#226;&#8364;&#8220;negative) express- <br />     In study, SMN interacts Mib1           ing Mib1-C1009S (p &lt; 0.0001, Figure 2B). Together, data <br /> overexpression Mib1 cultured cells increases SMN ubiquitina-         demonstrate Mib1 determines SMN protein levels promot- <br /> tion degradation. Importantly, Mib1 knockdown results in-           ing ubiquitination subsequent degradation. <br /> creased SMN levels cultured cells ameliorates neuromus- <br /> cular pharyngeal pumping defect Caenorhabditis elegans                  Reducing Mib1 expression increases SMN protein levels <br /> deficient SMN, indicating physiological role Mib1 modu-        Because Mib1 overexpression decreased SMN levels inves- <br /> lating SMN.                                                                tigated decreasing Mib1 increases SMN levels. We trans- <br />                                                                            fected HEK-293T cells Mib1 small interfering RNA (siRNA) a <br /> RESULTS                                                                    scrambled control, confirmed RT-PCR Mib1 levels <br /> Mib1 increases SMN ubiquitination protein turnover                     reduced half 72 h transfection Mib1 <br /> E3 ligases promote protein degradation catalyzing transfer       siRNA (Figure 3A). Importantly, SMN protein levels significantly <br /> ubiquitin molecules E2 enzyme substrate proteins. To         increased Mib1 knocked (p &lt; 0.001, Figure 3A). It <br /> determine Mib1 ubiquitinates SMN, cotransfected             previously reported microRNA miR-137 targets <br /> motor neuron&#226;&#8364;&#8220;derived hybrid cell line, NSC34, hemagglutinin           Mib1 mRNA regulates protein expression (Smrt et&#194;&#160;al., 2010). <br /> (HA)-tagged ubiquitin full-length selected domains myc-          We sought determine overexpressing miR-137 af- <br /> tagged Mib1. The cells lysed buffer containing ubiq-          fects SMN protein levels. We confirmed transfection cells <br /> uitin aldehyde inhibit deubiquitination immunoprecipitated          miR-137 decreased Mib1 mRNA protein levels (Figure 3B), <br /> with antibody SMN. To ensure ubiquitin-positive             SMN levels increased cells transfected <br /> bands Western blot ubiquitinated SMN ubiq-             miR-137 compared scrambled microRNA control (Figure <br /> uitinated proteins associated SMN, disassociated SMN          3B). Importantly, confirmed short hairpin RNA (shRNA) <br /> its binding partners immunoprecipitation denaturing              knockdown Mib1 increases SMN protein levels SMA patient&#226;&#8364;&#8220; <br /> them 1% SDS, followed renaturation 4.5% Triton X-100.           derived fibroblasts (Figure 3C). SMN transcript levels un- <br /> These conditions sufficient dissociate SMN known              changed siRNA Mib1 miR-137 (Figure 3D). Thus, using <br /> binding partners (Supplemental Figure S1). Immunoblots immu-            independent techniques cell lines shown <br /> noprecipitated SMN probed anti-HA antibody detect             reducing Mib1 expression results increased SMN protein <br /> ubiquitinated SMN. The ubiquitination endogenous SMN,                levels. <br /> indicated high-molecular-weight, ubiquitin-positive smear, is <br /> increased cells expressing full-length, truncated active-    Interaction Mib1 SMN mediated the <br /> site mutant forms Mib1 (Figure 1, A B). In contrast, overex-        N-terminal domain Mib1 segment SMN <br /> pressing E3 ligase parkin did increase SMN ubiquitination,         encoded exon 6 <br /> ruling possibility overexpressing E3 ligase         We investigated Mib1 interacts SMN. Using a <br /> indiscriminately increase SMN ubiquitination (Figure S2). We          monoclonal antibody SMN, endogenous Mib1 <br /> performed vitro ubiquitination assay determine            coimmunoprecipitated endogenous SMN (Figure 4A), indicat- <br /> Mib1 directly ubiquitinates SMN. Purified recombinant Mib1             ing proteins associate. No interaction detectable be- <br /> SMN proteins incubated reaction buffer containing ubiq-            tween SMN Mib2, homologue Mib1 (Koo et&#194;&#160; al., 2005b) <br /> uitin, ubiquitin-activating enzyme (E1), ubiquitin-conjugating     similar structural domain organization (Figure 4B), demon- <br /> enzyme (E2) UBCH5b. Mib1 ubiquitinates SMN cell-free sys-          strating specificity interaction Mib1 SMN. To <br /> tem, seen Western blots probed antibody polyubiq-         characterize interaction Mib1 SMN, we <br />  <br />  <br /> 1864&#226;&#8364;&#402; |&#226;&#8364;&#402; D. Y. Kwon et al.	                                                                                             Molecular Biology Cell <br />  FIGURE 1:&#226;&#8364;&#8218; (A) NSC-34 cells transfected 2 &#206;&#188;g Mib1 1 &#206;&#188;g HA-Ub cDNAs. The cells harvested 48 h <br /> later, endogenous SMN immunoprecipitated. Immunoprecipitated proteins resolved SDS&#226;&#8364;&#8220;PAGE, and <br /> the proteins analyzed Western blotting. The blots probed HA antibody detect ubiquitinated <br /> SMN. (B) Schematic representation Mib1 protein domains. (C) Cell-free SMN ubiquitination assay. Recombinant SMN <br /> was incubated E1 E2 (UBCH5B) enzymes Mib1 ubiquitin 1 h 37&#194;&#176;C. Western blots <br /> were probed anti-polyubiquitin antibody (FK1). (D) Pulse&#226;&#8364;&#8220;chase analysis endogenous SMN presence of <br /> 2 &#206;&#188;g Mib1-myc Mib1-C1009S-myc. The data represent mean &#194;&#177; SEM independent experiments. <br />  <br />  <br /> used purified recombinant Mib1 pull-down assays purified        tion mutant, exception SMN&#206;&#8221;6, coimmunoprecipitated <br /> glutathione S-transferase (GST)-SMN GST alone. We         Mib1, indicating SMN exon 6 required interac- <br /> GST-SMN GST pulled Mib1 (Figure 4C), con-            tion Mib1 (Figure 4E). Moreover, deletion of <br /> firming direct interaction SMN Mib1 showing           SMN exon 6 disrupts binding Mib1 markedly re- <br /> that factors, including members SMN complex,       duced ubiquitination Mib1 (Figure 4F). Because domain <br /> required interaction.                                          encoded exon 6 mediates SMN oligomerization, examined <br />     We asked domains Mib1 mediate interaction         SMN oligomerization required Mib1 interact with <br /> with SMN. Using constructs containing various regions Mib1       SMN. We Mib1 coimmunoprecipitated wild-type <br /> gene, performed coimmunoprecipitation assays map do-          SMN patient-derived SMN mutants SMN-G279V <br /> mains Mib1 required SMN binding. While N-terminal re-        SMN-Y272C, efficiently self-associate oli- <br /> gion Mib1 (m132-myc) coimmunoprecipitated SMN, con-             gomerize (Figure 4G). Thus SMN oligomerization required for <br /> structs containing middle ankyrin repeat C-terminal RING        Mib1 association. <br /> domain showed weak interaction (Figure 4D). Additionally,             We sought determine Mib1 contributes in- <br /> the C-terminal active-site <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> C1009S effect SMN         stability SMN&#206;&#8221;7. We overexpressing Mib1 resulted <br /> binding (Figure 4D). To map regions SMN mediate bind-       greater ubiquitination SMN&#206;&#8221;7 full-length SMN, and <br /> ing Mib1, cotransfected FLAG-tagged Mib1 construct            knocking Mib1 increased SMN&#206;&#8221;7 stability (Figure 5, A and <br /> with constructs expressing full-length SMN SMN lacking        B). These data suggest instability SMN&#206;&#8221;7 in <br /> each seven translated exons. Full-length SMN dele-      ubiquitination Mib1. <br />  <br />  <br /> Volume 24&#226;&#8364;&#402; June 15, 2013	                                                                          Mib1 targets SMN degradation&#226;&#8364;&#402; |&#226;&#8364;&#402; 1865&#226;&#8364;&#402; <br />                                                                                                     interference (RNAi) feeding <br />                                                                                                    (Timmons Fire, 1998; Timmons et&#194;&#160; al., <br />                                                                                                    2001). Knocking Mib-1 C. elegans <br />                                                                                                    orthologue increased pharyngeal <br />                                                                                                    pumping rates smn-1&#226;&#8364;&#8220;deficient animals, <br />                                                                                                    did reach statistical significance <br />                                                                                                    (p = 0.06, Figure S3). RNAi feeding is <br />                                                                                                    generally effective neurons in <br />                                                                                                    tissues (Kamath et&#194;&#160;al., 2001; Timmons <br />                                                                                                    et&#194;&#160; al., 2001; Rual et&#194;&#160; al., 2004). To circum- <br />                                                                                                    vent difficulty, increased neuronal <br />                                                                                                    sensitivity RNAi feeding ectopic neu- <br />                                                                                                    ronal expression SID-1 double- <br />                                                                                                    stranded RNA channel smn-1 animals <br />                                                                                                    (Calixto et&#194;&#160;al., 2010). In sensitized back- <br />                                                                                                    ground, mib-1 RNAi significantly amelio- <br />                                                                                                    rated smn-1 pharyngeal pumping de- <br />                                                                                                    fects (Figure 6A; p = 0.04), suggesting that <br />                                                                                                    MIB-1 conserved cross-species modifier <br />                                                                                                    SMN loss-of-function defects. Because <br />                                                                                                    Mib1 plays important role Notch func- <br />                                                                                                    tion (Itoh et&#194;&#160; al., 2003; Lai et&#194;&#160; al., 2005; Le <br />                                                                                                    Borgne et&#194;&#160;al., 2005), examined whether <br />                                                                                                    MIB-1 impact perturbations <br />                                                                                                    Notch signaling. Loss Notch coli- <br />                                                                                                    gand function (Komatsu et&#194;&#160;al., 2008; Singh <br />                                                                                                    et&#194;&#160; al., 2011) smn-1 mutants did not <br />                                                                                                    ameliorate pharyngeal pumping defect <br />                                                                                                    smn-1(ok355);osm-11(rt142) animals <br />                                                                                                    (Figure 6B). While rule SMN- <br />                                                                                                    independent effects, data suggest <br />                                                                                                    MIB-1 directly impacts SMN-1 function <br />                                                                                                    independent Notch signaling. <br />  <br />                                                                                                    DISCUSSION <br />                                                                                                     Through study, provide evidence <br />                                                                                                     E3 ligase Mib1 ubiquitinates and <br /> FIGURE 2:&#226;&#8364;&#8218; Effects Mib1 overexpression SMN protein levels gem number.                     catalyzes SMN protein degradation. These <br /> (A) HEK-293T cells transfected 1 2 &#206;&#188;g Mib1-myc cDNA. At 24 h              effects associated changes in <br /> transfection, cells treated vehicle (water) 1 &#206;&#188;M bortezomib harvested 6 h SMN mRNA expression ob- <br /> later. Cell lysates resolved SDS&#226;&#8364;&#8220;PAGE, densitometry resulting bands          served ligase-defective point mutant <br /> performed using ImageJ software (National Institutes Health). SMN protein levels were <br />                                                                                                     Mib1 expressed. Knocking Mib1 <br /> corrected based actin values. The data represent mean &#194;&#177; SEM independent <br />                                                                                                     increases SMN protein levels cultured <br /> experiments. *, p &lt; 0.05. (B) Overexpression Mib1 decreases gem number intensity in <br /> cultured cells. HeLa cells transfected Mib1-myc Mib-C1009S-myc 24 h. The cells cells partially rescues neuromuscular <br /> were fixed stained antibodies SMN myc appropriate Alexa Fluor               defect SMN deficiency C. elegans. <br /> secondary antibodies visualization microscopy. The number intensity nuclear gems These data indicate Mib1 reduces SMN <br /> were quantitated using Nikon NIS Elements software. At 300 cells counted        protein stability targeting degrada- <br /> transfection condition. That data represent mean &#194;&#177; SEM independent experiments.            tion proteasome represents a <br /> ***, p &lt; 0.001.                                                                                     new modifier SMA phenotype. <br />                                                                                                         The inverse correlation SMN2 <br /> Loss C. elegans orthologue mib-1 ameliorates                     copy number SMA severity indicates increasing SMN pro- <br /> neuromuscular defects caused smn-1 loss function                      tein levels ameliorate disease phenotype. Mechanisms to <br /> Potential cross-species conservation SMN-MIB1 interaction            increase SMN protein levels actively pursued thera- <br /> was examined C. elegans. The C. elegans genome single              peutic options SMA. Inhibiting degradation SMN one <br /> orthologue SMN, smn-1. Homozygous smn-1 loss function                 way increase steady-state level. Inhibiting proteasome in- <br /> causes developmental delay, impaired locomotion, reduced pha-               creases SMN levels cultured cells SMA model mice (Kwon <br /> ryngeal pumping, larval lethality (Briese et&#194;&#160;al., 2009; Dimitriadi      et&#194;&#160;al., 2011), ameliorates disease phenotype. Identifying <br /> et&#194;&#160;al., 2010). Although complete loss SMN function causes le-            enzymes specifically target SMN degradation thus <br /> thality, smn-1 animals survive days partial          provide new therapeutic targets. We overexpression <br /> maternal rescue. The C. elegans orthologue human MIB1             Mib1 increases ubiquitination SMN, indicated increased <br /> gene mib-1 (Berndt et&#194;&#160; al., 2011). To assess genetic interactions,          ubiquitin&#226;&#8364;&#8220;SMN conjugates. We overexpressed Mib1 ac- <br /> we decreased Mib-1 levels smn-1&#226;&#8364;&#8220;deficient animals RNA                 celerates rate SMN degradation, knockdown Mib1 <br />  <br />  <br /> 1866&#226;&#8364;&#402; |&#226;&#8364;&#402; D. Y. Kwon et al.	                                                                                           Molecular Biology Cell <br />                                                                                                       ubiquitinates SMN&#206;&#8221;7 greater extent <br />                                                                                                      full-length SMN. We propose the <br />                                                                                                      increased Mib1-mediated ubiquitination of <br />                                                                                                      SMN&#206;&#8221;7 account the <br />                                                                                                      relative instability SMN&#206;&#8221;7 protein. <br />                                                                                                      Consistent this, reducing Mib1 levels <br />                                                                                                      results increased stability SMN&#206;&#8221;7 <br />                                                                                                      protein. <br />                                                                                                          We interaction SMN <br />                                                                                                      Mib1 likely mediated domain <br />                                                                                                      encoded exon 6. Missense mutations in <br />                                                                                                      exon 6 deletion exon 7 disrupt SMN&#226;&#8364;&#8482;s <br />                                                                                                      ability oligomerize (Lorson et&#194;&#160; al., 1998; <br />                                                                                                      Pellizzoni et&#194;&#160;al., 1999; Zhang et&#194;&#160;al., 2003; Le <br />                                                                                                      et&#194;&#160; al., 2005), resulting severe SMA; this <br />                                                                                                      SMN fails oligomer- <br />                                                                                                      ize rapidly degraded (Burnett et&#194;&#160;al., 2009). <br />                                                                                                      A mutation SMN exon 6 disrupts oli- <br />                                                                                                      gomerization (Y272C) destabilizes the <br />                                                                                                      SMN protein, cells patient with <br />                                                                                                      mutation similar SMN levels to <br />                                                                                                      homozygous deletion SMN1 <br />                                                                                                      (Lefebvre et&#194;&#160;al., 1997). One possibility that <br />                                                                                                      Mib1 interacts SMN is <br />                                                                                                      oligomerized, exon 6 mediates SMN <br />                                                                                                      oligomerization. However, likely <br />                                                                                                      Mib1 interacts SMN has <br />                                                                                                      patient-derived mutations G279V and <br />                                                                                                      Y272C, efficiently oligomerize <br />                                                                                                      (Figure 4G). Thus likely the <br />                                                                                                      inability SMN oligomerize increases <br />                                                                                                      likelihood binding Mib1. Because <br />                                                                                                      exon 6 domain mediates SMN self-asso- <br />                                                                                                      ciation (Lorson et&#194;&#160;al., 1998), domain may <br />                                                                                                      available interaction Mib1 <br />                                                                                                      SMN oligomerizes complex. <br />                                                                                                      This explain increased stability of <br />                                                                                                      complexed SMN, inaccessible to <br />                                                                                                      Mib1 unlikely ubiquitinated <br />                                                                                                      degraded. <br /> FIGURE 3:&#226;&#8364;&#8218; (A) HEK-293T cells transfected 100 pmol scrambled siRNA                While article preparation, <br /> Mib1 siRNA 72 h. Mib1 SMN protein levels determined 72 h Western <br />                                                                                                      Usp9x (also known fat facets, FAM), a <br /> blotting. (B) HEK-293T cells transfected 100 &#206;&#188;M negative control miRNA or <br />                                                                                                      deubiquitinating enzyme (DUB), found <br /> miR-137 72 h. SMN protein levels determined Western blotting. The data represent <br /> the mean &#194;&#177; SEM independent experiments. (C) SMA patient&#226;&#8364;&#8220;derived fibroblasts (3813)          stabilize SMN reducing ubiquitina- <br /> were transfected Mib1 siRNA scrambled control. Cells harvested 72 h post-               tion (Han et&#194;&#160;al., 2012). Interestingly, Usp9x/ <br /> transfection, Mib-1 mRNA SMN protein levels assessed RT-PCR Western              FAM interacts Mib1 (Choe et&#194;&#160; al., <br /> blot analysis, respectively. (D) HEK-293T cells transfected Mib1 siRNA, miR-137,        2007), points possibility that <br /> scrambled control. Cells harvested 72 h posttransfection, SMN mRNA level                Usp9x Mib1 act concert regulate <br /> analyzed RT-PCR. *, p &lt; 0.05; **, p &lt; 0.01; ***, p &lt; 0.001.                                       SMN protein levels. <br />                                                                                                          Two genetic modifiers SMA been <br /> by siRNAs microRNA mir-137 stabilizes SMN. Our data           reported humans: SMN2 plastin3. A genome-wide RNAi <br /> demonstrate Mib1-mediated ubiquitination SMN enhances               screen C. elegans Drosophila deficient SMN identified <br /> its degradation decreases steady-state SMN protein levels.              genes modify disease phenotype model sys- <br />     While SMA patients homozygous deletion                 tems (Dimitriadi et&#194;&#160;al., 2010). SMN deficiency C. elegans char- <br /> SMN1 gene, retain copy SMN2. The major             acterized marked reduction rate pharyngeal pumping. <br /> SMN2 gene product truncated SMN&#206;&#8221;7 protein that, based             We Mib1 knockdown RNAi partially rescues the <br /> its ability extend survival SMA mice, likely retains         SMN-deficient phenotype C. elegans, indicating Mib1 may <br /> function unstable rapidly degraded. The instability           modifier SMA. Our biochemical data suggest Mib1 pro- <br /> SMN&#206;&#8221;7 thought to number factors, including            motes SMN degradation proteasome. However, cannot <br /> inability self-associate (Lorson et&#194;&#160;al., 1998; Burnett et&#194;&#160;al., 2009)     rule possibility SMN-independent pathways involving <br /> and presence degradation signal promotes rapid            functions Mib1 responsible partial rescue the <br /> turnover (Cho Dreyfuss, 2010). In study, Mib1         neuromuscular phenotype. Nonetheless, identification new <br />  <br />  <br /> Volume 24&#226;&#8364;&#402; June 15, 2013	                                                                                Mib1 targets SMN degradation&#226;&#8364;&#402; |&#226;&#8364;&#402; 1867&#226;&#8364;&#402; <br />                                                                                                modifier SMA phenotype partic- <br />                                                                                               ular importance, given currently limited <br />                                                                                               number targets favorable thera- <br />                                                                                               peutic index. <br />                                                                                                   The steady-state level protein is <br />                                                                                               governed balance protein <br />                                                                                               synthesis degradation. Strategies in- <br />                                                                                               crease SMN protein synthesis up-regulat- <br />                                                                                               ing gene, stabilizing mRNA, al- <br />                                                                                               tering splicing promote inclusion exon <br />                                                                                               7 pursued SMA therapeutic <br />                                                                                               development. That SMN2 gene pro- <br />                                                                                               duces full-length SMN protein sug- <br />                                                                                               gests strategies slow degradation <br />                                                                                               therapeutic options. Because <br />                                                                                               SMN ubiquitinated degraded the <br />                                                                                               proteasome, retarding turnover either <br />                                                                                               blocking proteasome inhibit- <br />                                                                                               ing SMN ubiquitination useful in <br />                                                                                               treating SMA. We shown SMA <br />                                                                                               mice treated proteasome inhibitor <br />                                                                                               bortezomib reduced pathology and <br />                                                                                               improved motor function (Kwon et&#194;&#160;al., 2011). <br />                                                                                               However, toxicity available protea- <br />                                                                                               inhibitors preclude chronic use in <br />                                                                                               SMA patients. While small molecule inhibi- <br />                                                                                               tors proteasome devel- <br />                                                                                               oped research clinical use, E3 ubiq- <br />                                                                                               uitin ligases emerging targets for <br />                                                                                               disease treatment, confer the <br />                                                                                               substrate specificity UPS. Drugs that <br />                                                                                               inhibit E3 ligases involved cell cycle regu- <br />                                                                                               lation cell proliferation currently un- <br />                                                                                               der study treat various types cancer. <br />                                                                                               One E3 ligase, murine double minute 2 <br />                                                                                               (MDM2), regulates tumor suppressor <br />                                                                                               p53 represses role cell cycle arrest <br />                                                                                               apoptosis (Fakharzadeh et&#194;&#160; al., 1993; <br />                                                                                               Haupt et&#194;&#160;al., 1997). Inhibitors human <br />                                                                                               homologue MDM2 shown to <br /> FIGURE 4:&#226;&#8364;&#8218; (A) Endogenous SMN immunoprecipitated HEK-293T cell lysates       reduce tumor growth induce tumor cell <br /> antibody SMN, immunoprecipitates resolved SDS&#226;&#8364;&#8220;PAGE. Association           apoptosis vitro, providing incentive for <br /> Mib1 determined Western blot analysis using antibody Mib1. (B) Protein lysates   targeting MDM2 function treat cancer in <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> HEK-293T cells immunoprecipitated SMN antibody. While Mib1                  vivo (Issaeva et&#194;&#160; al., 2004; Koblish et&#194;&#160; al., <br /> coimmunoprecipitated SMN, Mib2 detected Western blot analysis.                2006). High-throughput small molecule <br /> Membranes probed anti-SMN, anti-Mib1, anti-Mib2 antibodies. (C) GST or <br />                                                                                               compound screens used iden- <br /> GST-SMN used pull purified recombinant Mib1 protein. (D) Myc-tagged Mib1 <br />                                                                                               tify inhibitors E3 ligases (Davydov et&#194;&#160; al., <br /> constructs containing full-length truncated transcripts Mib1 gene were <br /> transiently transfected HEK-293T cells. Immunoprecipitation using anti-myc antibody   2004; Huang et&#194;&#160; al., 2005). These screens <br /> was performed, interaction endogenous SMN detected Western blot analysis      adapted identify molecules that <br /> using anti-SMN antibody. (E) HEK-293T cells transfected 0.5 &#206;&#188;g full-length SMN   disrupt interaction ligase and <br /> cDNA various SMN constructs single-exon deletion, 2 &#206;&#188;g Mib1-FLAG. Mib1-         protein substrate, confer <br /> FLAG immunoprecipitated resolved SDS&#226;&#8364;&#8220;PAGE gel. Western blots probed         greater selectivity reduce off-target ef- <br /> with antibodies FLAG Myc determine binding Mib1 SMN. (F) HEK-293T cells       fects. Nutlin, inhibits interaction <br /> were transfected 2 &#206;&#188;g Mib1-FLAG, 1 &#206;&#188;g HA-Ub, 1 &#206;&#188;g full-length SMN-myc      MDM2 p53, identified a <br /> SMN&#206;&#8221;6-myc. The cells harvested 48 h later, myc-tagged SMN immunoprecipitated         small molecule screen (Vassilev et&#194;&#160;al., 2004) <br /> using anti-myc antibody, proteins separated SDS&#226;&#8364;&#8220;PAGE detected by <br />                                                                                               currently evaluated early- <br /> Western blotting. The blots probed HA antibody detect ubiquitinated SMN. <br />                                                                                               phase clinical trials. While new, toxic, <br /> (G) HEK-293T cells transfected 2 &#206;&#188;g Mib1-myc 0.5 &#206;&#188;g GFP-tagged <br /> wild-type SMN, SMN G279V, SMN Y272C. The cells harvested 48 h later, SMN          general proteasome inhibitors de- <br /> was immunoprecipitated anti-myc antibody resolved SDS&#226;&#8364;&#8220;PAGE gel. Western      veloped, targeting E3 ligase selec- <br /> blots probed antibodies myc GFP determine interaction Mib1         tively tags SMN degradation be <br /> wild-type SMN disease-associated mutants.                                              safer effective. Small molecule <br />  <br />  <br />  <br /> 1868&#226;&#8364;&#402; |&#226;&#8364;&#402; D. Y. Kwon et al.	                                                                                      Molecular Biology Cell <br />                                                                                                   (Sigma-Aldrich, St. Louis, MO) used at <br />                                                                                                  1:1000 dilution; anti-FLAG M2 (Sigma- <br />                                                                                                  Aldrich), anti-Myc 9E10 (Santa Cruz Biotech- <br />                                                                                                  nology, Santa Cruz, CA), anti-HA (Sigma- <br />                                                                                                  Aldrich), anti-actin (Sigma-Aldrich) were <br />                                                                                                  used 1:5000 dilution. Anti-Mib1 serum <br />                                                                                                  gift Junmin Peng (St. Jude <br />                                                                                                  Children&#226;&#8364;&#8482;s Research Hospital). The signal <br />                                                                                                  visualized secondary antibody <br />                                                                                                  (anti-mouse anti-rabbit immunoglobulin- <br />                                                                                                  horseradish peroxidase, 1:3000&#226;&#8364;&#8220; <br />                                                                                                  10,000 dilution) using chemiluminescence <br />                                                                                                  detection (Perkin Elmer-Cetus, <br /> FIGURE 5:&#226;&#8364;&#8218; (A) HEK-293T cells transfected 2 &#206;&#188;g full-length SMN SMN&#206;&#8221;7            Waltham, MA). <br /> 1 &#206;&#188;g Mib1-FLAG HA-Ub cDNAs. The cells harvested 48 h later, SMN                         For immunoprecipitation experiments, <br /> immunoprecipitated resolved SDS&#226;&#8364;&#8220;PAGE, proteins analyzed Western           cell lysates clarified centrifugation <br /> blotting. The blots probed HA antibody detect ubiquitinated SMN.                 precleared pan-mouse immuno- <br /> (B) Pulse&#226;&#8364;&#8220;chase analysis full-length SMN SMN&#206;&#8221;7, absence presence 2 &#206;&#188;g        globulin G (IgG) Dynabeads (Invitrogen) for <br /> Mib1-myc. The data represent mean &#194;&#177; SEM experiments.                                1 h 4&#194;&#176;C. The lysates incubated <br />                                                                                                  1 &#206;&#188;g antibody 2 h overnight <br /> screens identify inhibitors Mib1 lead novel thera-     incubated pan-mouse IgG Dynabeads 1 h a <br /> peutics SMA.                                                         rotator 4&#194;&#176;C. The beads washed NP-40 buffer to <br />                                                                          times boiled SDS gel-loading buffer. Eluted proteins <br /> MATERIALS AND METHODS                                                    run SDS&#226;&#8364;&#8220;PAGE Western blotting. <br /> Plasmids miRNA                                                           For pull-down assay, 5 &#206;&#188;g purified recombinant Mib1 protein <br /> Plasmids encoding human full-length truncated Mib1              purchased Origene (Rockville, MD; TP321377) incubated <br /> constructed previously described (Itoh et&#194;&#160;al., 2003). Mib1 shRNA      GST SMN-GST beads 4&#194;&#176;C rotator overnight. Samples <br /> and nonsilencing control purchased SABiosciences               spun down, beads washed times NP-40 <br /> (KM26177G) pre-miR miRNA precursors (control miR-137)            buffer. Following wash, beads boiled SDS gel- <br /> were purchased Ambion (PM10513). Myc-Parkin obtained            loading buffer 5 min, eluted proteins processed for <br /> from Chitnis lab.                                                    Western blotting. <br />  <br /> Cell culture, transfection, Western blotting                          RNA extraction quantification <br /> Human embryonic kidney HEK-293T NSC34 cells main-                Cells homogenized 500 &#206;&#188;l TRIzol reagent (Invitrogen) and <br /> tained 37&#194;&#176;C 5% CO2 DMEM (Invitrogen, Carlsbad, CA)             spun 10 min 4&#194;&#176;C. One microliters chloroform was <br /> supplemented 10% FBS penicillin/streptomycin/glutamine           added supernatant, mixture vortexed spun <br /> (Invitrogen). Cells plated six-well culture plate tran-   again. After aqueous upper phase transferred an- <br /> siently transfected 0.5&#226;&#8364;&#8220;4 &#206;&#188;g plasmid DNA using           tube, 250 &#206;&#188;l isopropanol 50 &#206;&#188;l 3M sodium acetate were <br /> FuGENE HD transfection reagent (Roche Applied Science, India-             added. The RNA pelleted, washed 70% ethanol, <br /> napolis, IN) according manufacturer&#226;&#8364;&#8482;s protocol. The total          resuspended 100 &#206;&#188;l DNase- RNase-free water. Total <br /> amount DNA used transfection kept constant             RNA quantified absorption 260 nm, 450 ng each <br /> experiment adding appropriate vector                RNA sample converted cDNA using High Capacity <br /> plasmid. Cells harvested 24&#226;&#8364;&#8220;48 h posttransfection.                   cDNA Reverse Transcription kit (Applied Biosystems, Bedford, MA). <br />     For siRNA miRNA transfections, HEK-293T cells                Aliquots 20 &#206;&#188;l quantitative PCRs run triplicate using the <br /> plated subconfluently six-well culture dish transfected          ABI Prism 7900 sequence detection (Applied Biosystems). <br /> with 100 pmol siRNA/miRNA 10 &#206;&#188;l Lipofectamine 2000              The levels hypoxanthine phosphoribosyltransferase (HPRT; <br /> (Invitrogen). After 48 h transfection, cells transfected    control), SMN, Mib1 transcripts quantified thresh- <br /> with siRNA/miRNA 24 h harvested total 72 h        old cycle primers purchased Applied Biosystems. <br /> transfection. <br />                                                                           Immunohistochemical analysis gem numbers <br /> Western blotting, immunoprecipitation,                                    The gem number assayed 24 h transfection myc-Mib1 <br /> and pull-down assays                                                      myc-Mib1-C1009S. The cells fixed, permeabilized, im- <br /> Transfected cells lysed NP-40 lysis buffer (1% NP-40,      munostained anti-SMN anti-myc antibodies. Blinded counts <br /> 50 mM Tris-HCl, 150 mM NaCl, pH 8.0) RIPA buffer (1% Triton            gems performed 300 nuclei condition. <br /> X-100, 1% sodium deoxycholate, 150 mM NaCl, 0.1% SDS, pH 8.0) <br /> and quantified using BCA Protein Assay Kit (Pierce, Rockford, IL)     In vitro ubiquitination <br /> for Western blotting. Equal amounts protein lysate (15 &#206;&#188;g)        Recombinant SMN protein (Enzo Life Sciences, Farmingdale, NY) <br /> run separated 10% SDS&#226;&#8364;&#8220;PAGE (Invitrogen) transferred          incubated ubiquitin, E1 ubiquitin-activating enzyme <br /> to polyvinylidene fluoride membrane (Invitrogen). They             (Boston Biochem, Cambridge, MA), E2 ubiquitin-conjugating <br /> then probed appropriate antibodies 5% milk washed         enzyme, UBCH5B, recombinant Mib1 (OriGene Technologies, <br /> with Tris-buffered saline Tween 20. Antibodies: anti-SMN (BD          Rockville, MD) reaction buffer (50 mM Tris-HCl, pH 7.5, 2 mM ATP, <br /> Transduction), anti-Mib1 (Epitomics, Burlingame, CA), anti-Mib2       4 mM MgCl2, 2 mM dithiothreitol [DTT]) 1 h 37&#194;&#176;C. Reactions <br />  <br />  <br /> Volume 24&#226;&#8364;&#402; June 15, 2013	                                                                            Mib1 targets SMN degradation&#226;&#8364;&#402; |&#226;&#8364;&#402; 1869&#226;&#8364;&#402; <br />                                                                                                          Construction RNAi feeding clone for <br />                                                                                                         Y47D3A.22/mib-1 <br />                                                                                                         A 796&#226;&#8364;&#8220;<span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> pair PCR product correspond- <br />                                                                                                         ing C. elegans mib-1 cDNA (Berndt <br />                                                                                                         et&#194;&#160;al., 2011) cloned L4440 RNAi <br />                                                                                                         feeding vector; followed by <br />                                                                                                         transformation HT115(DE3) bacte- <br />                                                                                                         rial strain (Timmons Fire, 1998; <br />                                                                                                         Timmons et&#194;&#160;al., 2001). Primers amplifica- <br />                                                                                                         tion ACH72 5&#226;&#8364;&#8482;-tgatgctagcgcagcttctc- <br />                                                                                                         ctgttgcgtg-3&#226;&#8364;&#8482; ACH73 5&#226;&#8364;&#8482;-tgatctcga&#194;&#173;ggcg&#194;&#173; <br />                                                                                                         ggagccga&#194;&#173;tttgaactg-3&#226;&#8364;&#8482;. <br />  <br />                                                                                                         C. elegans pharyngeal pumping assay <br />                                                                                                       The pharyngeal pumping assay per- <br />                                                                                                       formed previously described (Dimitriadi <br />                                                                                                       et&#194;&#160;al., 2010). Briefly, eggs hatched L4440 <br />                                                                                                       control       vector     (L4440     [RNAi])      or <br />                                                                                                       (Y47D3A.22/mib-1[RNAi]) bacterial feeding <br />                                                                                                       strains (Kamath Ahringer, 2003) were <br /> FIGURE 6:&#226;&#8364;&#8218; Knockdown C. elegans orthologue Mib1 ameliorates pharyngeal <br /> pumping neuromuscular defects smn-1 loss-of-function animals. (A) mib-1 knockdown                  reared 2 d 25&#194;&#176;C 1 d 20&#194;&#176;C. <br /> significantly increases pharyngeal pumping smn-1 deficient animals SID-1 sensitization,       Pumping rates determined using an <br /> which increases RNAi neuronal sensitivity. (B) Loss Notch coligand function smn-1           AxioCam ICc1 camera Zeiss Stemi SV11 <br /> mutants did ameliorate pharyngeal pumping defect smn-1(ok355);osm-11(rt142)                microscope (Carl Zeiss, Jena, Germany). A <br /> animals. Pharyngeal pumping events measured control mib-1 RNAi-fed                   pumping event scored pharyngeal <br /> animals. All error bars SEM; *, p &lt; 0.05.                                                         grinder movement axis, average <br />                                                                                                       pumping rates (&#194;&#177; SEM) derived a <br /> were quenched 30 &#206;&#188;l 5&#195;&#8212; Laemmli sample buffer (200 mM Tris,          blinded manner independent trials (n &#226;&#8240;&#165; 30 ani- <br /> pH 6.8, 32% [vol/vol] glycerol, 6.4% [wt/vol] SDS, 0.32% [wt/vol] bro-      mals total). Heterozygous homozygous smn-1 loss-of-function <br /> mophenol blue, 200 mM DTT), boiled 5 min, separated             animals distinguished green fluorescent protein (GFP) fluo- <br /> via SDS&#226;&#8364;&#8220;PAGE. SMN ubiquitination analyzed Western blot-              rescence. Both wild-type (N2) smn-1 loss-of-function heterozy- <br /> ting, using antibody polyubiquitin (FK1; Enzo Life Sciences).       gous animals showed pumping rates significantly dif- <br />                                                                             ferent. Hence, heterozygous smn-1 loss-of-function animals were <br /> Pulse&#226;&#8364;&#8220;chase protein labeling                                                used controls study. <br /> Pulse&#226;&#8364;&#8220;chase analysis performed previously described (Burnett <br /> et&#194;&#160; al., 2009). In brief, HEK-293T cells incubated cysteine-        Statistics <br /> methionine&#226;&#8364;&#8220;free medium (Sigma-Aldrich) 2 h 12 h;            The biochemical data analyzed GraphPad Prism <br /> followed incubation cysteine-methionine&#226;&#8364;&#8220;free medium contain-          software package (GraphPad Software, San Diego, CA) com- <br /> ing 100 &#206;&#188;Ci [35S]-labeled cysteine-methionine (GE Healthcare,            pared statistically t test one-way analysis variance, <br /> Piscataway, NJ) 1 h. After labeling, cells washed         Newman-Keuls multiple-comparison post hoc correction. <br /> with culture medium containing 10-fold excess unlabeled methi-         To compare differences groups, performed a <br /> onine cysteine (5 mM each) incubated            nonparametric equality medians test, data were <br /> same medium. The cells collected indicated time points          normally distributed. If statistically significant, a <br /> and processed immunoprecipitation anti-SMN antibody (BD            pairwise comparison treatment groups done <br /> Transduction Laboratories). Immunoprecipitations carried           using Mann-Whitney test. <br /> for 6 h 4&#194;&#176;C antibodies bound protein G-Sepharose (Sigma- <br /> Aldrich). After washes lysis buffer (100 mM NaCl, 10 mM          ACKNOWLEDGMENTS <br /> Tris-HCl, pH 7.4, 2.5 mM MgCl2, 0.1% NP-40, protease [Roche])           We thank Liu Yang (University Arkansas) myc-tagged <br /> and phosphatase inhibitor cocktails (Sigma-Aldrich), bound proteins         FL-SMN deletion mutant cDNAs. This work supported by <br /> were eluted boiling SDS&#226;&#8364;&#8220;PAGE sample buffer. Immunocom-                intramural research funds National Institute Neurological <br /> plexes separated 10% SDS&#226;&#8364;&#8220;PAGE. The gels dried              Disorders Stroke National Institutes Health&#226;&#8364;&#8220;National <br /> exposed phosphorimager screen, signal visualized           Institute Neurological Disorders Stroke (NS066888) A.C.H. <br /> with Storm phosphorimager (Molecular Dynamics, Piscat- <br /> away, NJ). Densitometric analysis protein bands carried      REFERENCES <br />                                                                             Barsi JC, Rajendra R, Wu JI, Artzt K (2005). Mind bomb1 ubiquitin ligase <br /> using ImageQuant PhosphorImager software (Molecular Dynamics). <br />                                                                                   essential mouse embryonic development Notch signaling. Mech <br />                                                                                   Dev 122, 1106&#226;&#8364;&#8220;1117. <br /> C. elegans strains                                                            Berndt JD, Aoyagi A, Yang P, Anastas JN, Tang L, Moon RT (2011). Mind- <br /> TU3311 uIs60 [unc-119p::yfp, unc-119p::sid-1] (Calixto et&#194;&#160;al., 2010),             bomb 1, E3 ubiquitin ligase, forms complex RYK activate <br /> HA2258 smn-1(ok355)/hT2[bli-4(e937) let-?(q782) qIs48] (I;III); uIs60             Wnt/&#206;&#178;-catenin signaling. J Cell Biol 194, 737&#226;&#8364;&#8220;750. <br />                                                                               Brenner S (1974). The genetics Caenorhabditis elegans. Genetics 77, <br /> (Briese et&#194;&#160;al., 2009), HA2201 smn-1(ok355)/hT2[bli-4(e937) let-               71&#226;&#8364;&#8220;94. <br /> ?(q782) qIs48] (I;III); osm-11(rt142)X maintained 20&#194;&#176;C ac-            Briese M, Esmaeili B, Fraboulet S, Burt EC, Christodoulou S, Towers PR, <br /> cording standard protocols (Brenner, 1974).                                    Davies KE, Sattelle DB (2009). Deletion smn-1, Caenorhabditis <br />  <br />  <br /> 1870&#226;&#8364;&#402; |&#226;&#8364;&#402; D. Y. Kwon et al.	                                                                                                 Molecular Biology Cell <br />      elegans ortholog spinal muscular atrophy gene, results locomo-      Kwon DY, Motley WW, Fischbeck KH, Burnett BG (2011). Increasing expres- <br />     tor dysfunction reduced lifespan. Hum Mol Genet 18, 97&#226;&#8364;&#8220;104.                   sion decreasing degradation SMN ameliorate spinal muscular <br /> Burnett BG, Munoz E, Tandon A, Kwon DY, Sumner CJ, Fischbeck KH                       atrophy phenotype mice. Hum Mol Genet 20, 3667&#226;&#8364;&#8220;3677. <br />     (2009). Regulation SMN protein stability. Mol Cell Biol 29, 1107&#226;&#8364;&#8220;1115.     Lai EC, Roegiers F, Qin X, Jan YN, Rubin GM (2005). The ubiquitin ligase <br /> Calixto A, Chelur D, Topalidou I, Chen X, Chalfie M (2010). Enhanced neu-             Drosophila Mind bomb promotes Notch signaling regulating the <br />     ronal RNAi C. elegans using SID-1. Nat Methods 7, 554&#226;&#8364;&#8220;559.                     localization activity Serrate Delta. Development 132, <br /> Chang HC et&#194;&#160;al. (2008). Modeling spinal muscular atrophy Drosophila.               2319&#226;&#8364;&#8220;2332. <br />     PLoS One 3, e3209.                                                            Le TT, Pham LT, Butchbach ME, Zhang HL, Monani UR, Coovert DD, <br /> Chang HC, Hung WC, Chuang YJ, Jong YJ (2004). Degradation survival                 Gavrilina TO, Xing L, Bassell GJ, Burghes AH (2005). SMNDelta7, the <br />     motor neuron (SMN) protein mediated ubiquitin/proteasome               major product centromeric survival motor neuron (SMN2) gene, <br />     pathway. Neurochem Int 45, 1107&#226;&#8364;&#8220;1112.                                             extends survival mice spinal muscular atrophy associates <br /> Chen W, Casey Corliss D (2004). Three modules zebrafish Mind bomb                  full-length SMN. Hum Mol Genet 14, 845&#226;&#8364;&#8220;857. <br />     work cooperatively promote Delta ubiquitination endocytosis.           Le Borgne R, Remaud S, Hamel S, Schweisguth F (2005). Two distinct E3 <br />     Dev Biol 267, 361&#226;&#8364;&#8220;373.                                                            ubiquitin ligases complementary functions regulation of <br /> Cho S, Dreyfuss G (2010). A degron created SMN2 exon 7 skipping               delta serrate signaling Drosophila. PLoS Biol 3, e96. <br />     principal contributor spinal muscular atrophy severity. Genes Dev 24,      Lefebvre S et&#194;&#160;al. (1995). Identification characterization spinal <br />     438&#226;&#8364;&#8220;442.                                                                          muscular atrophy-determining gene. Cell 80, 155&#226;&#8364;&#8220;165. <br /> Choe EA, Liao L, Zhou JY, Cheng D, Duong DM, Jin P, Tsai LH, Peng J               Lefebvre S, Burlet P, Liu Q, Bertrandy S, Clermont O, Munnich A, Dreyfuss <br />     (2007). Neuronal morphogenesis regulated interplay              G, Melki J (1997). Correlation severity SMN protein level in <br />     cyclin-dependent kinase 5 ubiquitin ligase mind bomb 1.                   spinal muscular atrophy. Nat Genet 16, 265&#226;&#8364;&#8220;269. <br />     J Neurosci 27, 9503&#226;&#8364;&#8220;9512.                                                     Lorson CL, Androphy EJ (2000). An exonic enhancer required inclusion <br /> Coovert DD, Le TT, McAndrew PE, Strasswimmer J, Crawford TO, Mendell                  essential exon SMA-determining gene SMN. Hum Mol <br />     JR, Coulson SE, Androphy EJ, Prior TW, Burghes AH (1997). The survival            Genet 9, 259&#226;&#8364;&#8220;265. <br />     motor neuron protein spinal muscular atrophy. Hum Mol Genet 6,             Lorson CL, Hahnen E, Androphy EJ, Wirth B (1999). A single nucleotide in <br />     1205&#226;&#8364;&#8220;1214.                                                                        SMN gene regulates splicing responsible spinal muscular <br /> Davydov IV, Woods D, Safiran YJ, Oberoi P, Fearnhead HO, Fang S, Jensen               atrophy. Proc Natl Acad Sci USA 96, 6307&#226;&#8364;&#8220;6311. <br />     JP, Weissman AM, Kenten JH, Vousden KH (2004). Assay ubiquitin            Lorson CL, Strasswimmer J, Yao JM, Baleja JD, Hahnen E, Wirth B, Le T, <br />     ligase activity: high-throughput screen inhibitors HDM2. J Biomol          Burghes AH, Androphy EJ (1998). SMN oligomerization defect corre- <br />     Screen 9, 695&#226;&#8364;&#8220;703.                                                                lates spinal muscular atrophy severity. Nat Genet 19, 63&#226;&#8364;&#8220;66. <br /> Dimitriadi M et&#194;&#160;al. (2010). Conserved genes act modifiers invertebrate      McAndrew PE, Parsons DW, Simard LR, Rochette C, Ray PN, Mendell JR, <br />     SMN loss function defects. PLoS Genet 6, e1001172.                             Prior TW, Burghes AH (1997). Identification proximal spinal muscular <br /> Fakharzadeh SS, Rosenblum-Vos L, Murphy M, Hoffman EK, George DL                      atrophy carriers patients analysis SMNT SMNC gene copy <br />     (1993). Structure organization amplified DNA double minutes             number. Am J Hum Genet 60, 1411&#226;&#8364;&#8220;1422. <br />     containing mdm2 oncogene. Genomics 15, 283&#226;&#8364;&#8220;290.                           McWhorter ML, Monani UR, Burghes AH, Beattie CE (2003). Knockdown of <br /> Han KJ, Foster DG, Zhang NY, Kanisha K, Dzieciatkowska M, Sclafani RA,                survival motor neuron (Smn) protein zebrafish causes defects in <br />     Hansen KC, Peng J, Liu CW (2012). Ubiquitin-specific protease 9x deu-             motor axon outgrowth pathfinding. J Cell Biol 162, 919&#226;&#8364;&#8220;931. <br />     biquitinates stabilizes spinal muscular atrophy protein&#226;&#8364;&#8221;survival      Monani UR, Lorson CL, Parsons DW, Prior TW, Androphy EJ, Burghes AH, <br />     motor neuron. J Biol Chem 287, 43741&#226;&#8364;&#8220;43752.                                       McPherson JD (1999). A single nucleotide difference alters splicing <br /> Haupt Y, Maya R, Kazaz A, Oren M (1997). Mdm2 promotes rapid deg-                 patterns distinguishes SMA gene SMN1 copy gene SMN2. <br />     radation p53. Nature 387, 296&#226;&#8364;&#8220;299.                                             Hum Mol Genet 8, 1177&#226;&#8364;&#8220;1183. <br /> Hsieh-Li HM, Chang JG, Jong YJ, Wu MH, Wang NM, Tsai CH, Li H (2000).             Pellizzoni L, Charroux B, Dreyfuss G (1999). SMN mutants spinal muscular <br />     A mouse model spinal muscular atrophy. Nat Genet 24, 66&#226;&#8364;&#8220;70.                   atrophy patients defective binding snRNP proteins. Proc Natl <br /> Huang J, Sheung J, Dong G, Coquilla C, Daniel-Issakani S, Payan DG                    Acad Sci USA 96, 11167&#226;&#8364;&#8220;11172. <br />     (2005). High-throughput screening inhibitors e3 ubiquitin          Petroski MD (2008). The ubiquitin system, disease, drug discovery. <br />     ligase APC. Methods Enzymol 399, 740&#226;&#8364;&#8220;754.                                         BMC Biochem 9, S7. <br /> Issaeva N, Bozko P, Enge M, Protopopova M, Verhoef LG, Masucci M,                 Prior TW, Swoboda KJ, Scott HD, Hejmanowski AQ (2004). Homozygous <br />     Pramanik A, Selivanova G (2004). Small molecule RITA binds p53,                SMN1 deletions unaffected family members modification the <br />     blocks p53-HDM-2 interaction activates p53 function tumors.                phenotype SMN2. Am J Med Genet A 130, 307&#226;&#8364;&#8220;310. <br />     Nat Med 10, 1321&#226;&#8364;&#8220;1328.                                                        Rual JF et&#194;&#160;al. (2004). Toward improving Caenorhabditis elegans phenome <br /> Itoh M et&#194;&#160;al. (2003). Mind bomb ubiquitin ligase essential           mapping ORFeome-based RNAi library. Genome Res 14, <br />     efficient activation Notch signaling Delta. Dev Cell 4, 67&#226;&#8364;&#8220;82.              2162&#226;&#8364;&#8220;2168. <br /> Jarecki J, Chen X, Bernardino A, Coovert DD, Whitney M, Burghes A,                Singh K et&#194;&#160;al. (2011). C. elegans Notch signaling regulates adult chemosen- <br />     Stack J, Pollok BA (2005). Diverse small-molecule modulators SMN               sory response larval molting quiescence. Curr Biol 21, 825&#226;&#8364;&#8220;834. <br />     expression high-throughput compound screening: early leads           Smrt RD et&#194;&#160;al. (2010). MicroRNA miR-137 regulates neuronal maturation by <br />     therapeutic spinal muscular atrophy. Hum Mol Genet 14,              targeting ubiquitin ligase mind bomb-1. Stem Cells 28, 1060&#226;&#8364;&#8220;1070. <br />     2003&#226;&#8364;&#8220;2018.                                                                    Timmons L, Court DL, Fire A (2001). Ingestion bacterially expressed <br /> Jin Y, Blue EK, Dixon S, Shao Z, Gallagher PJ (2002). A death-associated              dsRNAs produce specific potent genetic interference in <br />     protein kinase (DAPK)-interacting protein, DIP-1, E3 ubiquitin ligase       Caenorhabditis elegans. Gene 263, 103&#226;&#8364;&#8220;112. <br />     promotes tumor necrosis factor-induced apoptosis regulates           Timmons L, Fire A (1998). Specific interference ingested dsRNA. Nature <br />     cellular levels DAPK. J Biol Chem 277, 46980&#226;&#8364;&#8220;46986.                        395, 854. <br /> Kamath RS, Ahringer J (2003). Genome-wide RNAi screening Caenorhab-            Vassilev LT et&#194;&#160;al. (2004). In vivo activation p53 pathway small- <br />     ditis elegans. Methods 30, 313&#226;&#8364;&#8220;321.                                               molecule antagonists MDM2. Science 303, 844&#226;&#8364;&#8220;848. <br /> Kamath RS, Martinez-Campos M, Zipperlen P, Fraser AG, Ahringer J (2001).          Wolstencroft EC, Mattis V, Bajer AA, Young PJ, Lorson CL (2005). A non- <br />     Effectiveness specific RNA-mediated interference ingested              sequence-specific requirement SMN protein activity: role of <br />     double-stranded RNA Caenorhabditis elegans. Genome Biol 2,                     aminoglycosides inducing elevated SMN protein levels. Hum Mol <br />     RESEARCH0002.                                                                     Genet 14, 1199&#226;&#8364;&#8220;1210. <br /> Koblish HK et&#194;&#160;al. (2006). Benzodiazepinedione inhibitors Hdm2:p53          Yoo KW, Kim EH, Jung SH, Rhee M, Koo BK, Yoon KJ, Kong YY, Kim CH <br />     complex suppress human tumor cell proliferation vitro sensitize            (2006). Snx5, Mind bomb-binding protein, expressed he- <br />     tumors doxorubicin vivo. Mol Cancer Ther 5, 160&#226;&#8364;&#8220;169.                        matopoietic endothelial precursor cells zebrafish. FEBS Lett 580, <br /> Komatsu H et&#194;&#160;al. (2008). OSM-11 facilitates LIN-12 Notch signaling             4409&#226;&#8364;&#8220;4416. <br />     Caenorhabditis elegans vulval development. PLoS Biol 6, e196.                 Zhang C, Li Q, Jiang YJ (2007). Zebrafish Mib Mib2 mutual E3 <br /> Koo BK et&#194;&#160;al. (2005a). Mind bomb 1 essential generating functional             ubiquitin ligases common specific delta substrates. J Mol Biol <br />      <br /> </body></html>