<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Bicoid Stability Factor Controls Polyadenylation and <br /> Expression Specific Mitochondrial mRNAs in <br /> Drosophila melanogaster <br /> Ana Bratic1,2., Anna Wredenberg2., Sebastian Gro&#194;&#168;nke2, James B. Stewart2, Arnaud Mourier2, Benedetta <br /> Ruzzenente2, Christian Kukat2, Rolf Wibom1, Bianca Habermann2, Linda Partridge2, Nils-Go&#194;&#168;ran <br /> Larsson1,2* <br /> 1 Department Laboratory Medicine, Karolinska Institutet, Solna, Sweden, 2 Max Planck Institute Biology Ageing, Cologne, Germany <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The bicoid stability factor (BSF) Drosophila melanogaster reported present cytoplasm, it <br />      stabilizes maternally contributed bicoid mRNA binds mRNAs expressed early zygotic genes. BSF have <br />      roles, ubiquitously expressed essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span> adult flies. We performed immunofluorescence <br />      cell fractionation analyses BSF mainly mitochondrial protein. We studied independent RNAi <br />      knockdown fly lines report reduced BSF protein levels lead severe respiratory deficiency delayed <br />      development late larvae stage. Ubiquitous knockdown BSF results severe reduction polyadenylation tail <br />      lengths specific mitochondrial mRNAs, accompanied enrichment unprocessed polycistronic RNA intermediates. <br />      Furthermore, observed significant reduction mRNA steady state levels, despite increased novo transcription. <br />      Surprisingly, mitochondrial novo translation increased abnormal mitochondrial translation products present in <br />      knockdown flies, suggesting BSF role coordinating mitochondrial translation addition role in <br />      mRNA maturation stability. We report novel function BSF flies demonstrate important <br />      intra-mitochondrial role, essential maintaining mtDNA gene expression oxidative phosphorylation. <br />  <br />   Citation: Bratic A, Wredenberg A, Gro&#194;&#168;nke S, Stewart JB, Mourier A, et al. (2011) The Bicoid Stability Factor Controls Polyadenylation Expression Specific <br />   Mitochondrial mRNAs Drosophila melanogaster. PLoS Genet 7(10): e1002324. doi:10.1371/journal.pgen.1002324 <br />   Editor: Gregory S. Barsh, Stanford University School Medicine, United States America <br />   Received April 14, 2011; Accepted August 4, 2011; Published October 13, 2011 <br />   Copyright: &#195;&#376; 2011 Bratic et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The study supported grants Swedish Research Council (2010-2766) (http://www.vr.se) ERC Advanced Investigator grant (http:// <br />   erc.europa.eu/) N-GL. AW funded FEBS Long-Term Fellowship (http://www.febs.org). The funders role study design, data collection and <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: Larsson@age.mpg.de <br />   . These authors contributed equally work. <br />  <br />  <br />  <br />  <br /> Introduction                                                                           bicoid expression [2]. Characterization global gene expression <br />                                                                                        patterns flies shown BSF ubiquitously expressed in <br />    The maternal zygotic transition, control                         adults, indicating role BSF limited <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> development shifts maternally contributed mRNAs genes                          embryogenesis [3]. Studies subcellular localization BSF <br /> expressed zygote, considerable interest. The maternally                   shown present cytoplasmic particles oocytes <br /> contributed bicoid mRNA, encoding protein important                              surrounding nurse cells [1] cytoplasm nucleus <br /> formation anterior body patterning, dependent regulatory                      early embryos [2]. Bioinformatics analyses suggest BSF has <br /> mechanisms controlling cytoplasmic stability localization                      homology mammalian LRPPRC protein [1,4], which <br /> of mRNA zygote. The bicoid stability factor (BSF)                        belongs pentatricopeptide repeat class proteins [5]. The <br /> thought involved process binds 39UTR                    LRPPRC protein reported roles cytoplasmic <br /> bicoid mRNA [1]. <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>Mutation</span> BSF binding site leads                             RNA transport [6&#226;&#8364;&#8220;8] nuclear transcription [9,10], its <br /> reduced abundance bicoid mRNA, P element                                  main localization mitochondrial matrix [4,10,11], it <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>insertion mutation</span> leads drastic reduction BSF protein                    suggested stabilize mitochondrial transcripts [12]. <br /> levels does affect abundance distribution endogenous                        The fly mtDNA small compact, circular molecule, encoding <br /> bicoid mRNA [1]. In study, BSF reported                          13 essential polypeptides, components mitochon- <br /> role regulation early zygotic genes binding short                           drial oxidative phosphorylation system, RNA components of <br /> consensus sequence 59UTR genes expressed early                        mitochondrial translational system, include 22 transfer <br /> zygote [2]. It reported BSF essential, P element                 RNAs 2 ribosomal RNAs [13]. Mitochondrial transcription <br /> insertion bsf open reading frame lethal homozygous                    generates large polycistronic transcripts, processed by <br /> form [2]. However, homozygous bicoid mutant flies viable                       endonucleolytic cleavage generate individual mRNAs [14,15] <br /> BSF likely additional role regulating                  These mRNAs subsequently polyadenylated [16], process <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                        1                          October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                  BSF Drosophila melanogaster <br />  <br />  <br />  <br />   Author Summary                                                         oocytes cytoplasmic nuclear localization early <br />                                                                          embryos [1,2]. We addressed subcellular localization <br />   The majority cellular energy currency ATP formed             BSF transfecting Schneider (S2R+) HeLa cells a <br />   tubular network, termed mitochondria, present              BSF-FLAG-GFP fusion construct. There perfect overlay <br />   virtually eukaryotic cells. The mitochondria unique            Mitotracker Deep Red BSF-FLAG-GFP fluores- <br />   cellular organelles contain               cence co-localization rate 9161% Schneider cells <br />   genome, encodes critical proteins necessary                  (N = 5) 9463% HeLa cells (N = 7) (Figure 1B), thus <br />   cellular energy production. However, vast majority              indicating BSF localized mitochondria. We also <br />   mitochondrial proteins encoded nucleus                  performed subcellular fractionation experiments tissues <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />   imported mitochondria. Gene expression needs                 adult flies BSF present mitochondrial <br />   coordinated genomes ensure <br />                                                                          fraction (Figure 1C). Our results BSF mainly <br />   efficient mitochondrial function sufficient adaptation <br />                                                                          localized mitochondria, good agreement the <br />   different physiological demands. The regulation the <br />   mitochondrial genome poorly understood,                known main localization mammalian homolog LRPPRC <br />   basic regulators characterized. We used              [4]. <br />   RNAi fruit fly study vivo function the <br />   bicoid stability factor (BSF), previously thought              Efficient RNAi induced knockdown BSF vivo <br />   cytoplasmic nuclear protein important fly devel-                  In order analyze vivo function BSF, induced gene <br />   opment. We BSF mainly localized                   silencing using UAS-GAL4 independent bsf <br />   mitochondria, essential mtDNA gene                     RNAi-knock fly lines, target different, non-overlapping <br />   expression, regulating polyadenylation maturation              regions bsf transcript. In case, performed fly crosses <br />   specific mRNAs. Furthermore, BSF coordinates                    generated bsf knockdown lines wDahT;+;UAS-bsf-RNAi#1/ <br />   translation assembly mitochondrial peptides                  daGAL4 wDahT;UAS-bsf-RNAi#2/+;daGAL4/+ (w;;UAS- <br />   inner mitochondrial membrane.                                      bsfRNAi#1/daGAL4 w;UAS-bsfRNAi#2/+;daGAL4/+) and <br />                                                                          control lines wDahT;+;daGAL4/+ (w;;daGAL4/+), and <br />                                                                          wDahT;+;UAS-bsfRNAi#1/+;+ wDahT;UAS-bsfRNAi#2/ <br /> believed place steps. First, oligoadenylated              +;+ (w;;UAS-bsfRNAi#1/+; w;UAS-bsfRNAi#2/+), which <br /> transcripts generated addition short adenine tail         analyzed parallel experiments. Ubiquitous <br /> the 3&#226;&#8364;&#732;ends mitochondrial mRNAs. The enzyme necessary             knockdown (KD), using daughterless-GAL4 (daGAL4) driver <br /> for oligoadenylation identified. Second,       line, resulted 80% down-regulation bsf transcript <br /> poly A polymerase enzyme add ,50 adenines              levels third-instar larvae adult flies (Figure 2A). <br /> oligoadenylated mRNAs create long polyadenylated tail             Western blot analyses revealed BSF protein was <br /> [17].                                                                    undetectable third-instar KD larvae (Figure 2B) KD flies <br />     The vast majority ,103 mitochondrial proteins             (Figure S1A), demonstrating highly efficient KD BSF protein <br /> encoded nuclear genes, including majority                  expression bsf-RNAi lines. <br /> respiratory chain subunits, proteins involved replication <br /> and transcription mtDNA proteins mitochondrial <br />                                                                          BSF deficiency affects climbing ability, fecundity, life <br /> ribosome. The regulation oxidative phosphorylation capacity is <br /> thus dependent crosstalk genomes nuclear            span flies <br /> genes play key role process regulate mtDNA                 Ubiquitous BSF KD caused delay pupal development, with <br /> expression different levels. The mitochondrial genomes        majority pupae line 1 hatching, 15% from <br /> flies mammals gene content          line 2 managing complete eclosure (Figure 2C). KD flies <br /> substantial differences gene order. The high level conserva-       significantly reduced climbing ability, suggesting muscle <br /> tion mtDNA important nuclear-encoded regulators                weakness (Figure 2D). Female flies KD lines weighed <br /> mtDNA expression suggests key regulatory processes              (Figure 2E), consistent weight change in <br /> similar insects mammals. We hypothesized           males (Figure 2F). Continuous bsf KD male/female crossings <br /> BSF role fly mitochondria, addition            resulted offspring, KD flies laid significantly fewer eggs <br /> suggested regulatory roles developmental processes. We report         comparison controls (Figure 2G). Dissecting ovaries from <br /> here BSF bona fide mitochondrial protein involved           adult bsf KD flies revealed significant reduction size in <br /> regulating mtDNA gene expression. Contrary previous reports,          comparison controls (45621% v. 100610%)(Figure 2H and <br /> our data demonstrate BSF involved maturation          Figure S1B), possibly contributing lower body weight the <br /> polyadenylation mitochondrial mRNAs coordinates                   female KD flies. Flies bsf KD lines drastically <br /> mitochondrial translation. We identified essential          reduced life span compared controls (Figure 2I), most <br /> role BSF maintaining mtDNA gene expression oxidative          flies surviving 30 days. Together, results identify <br /> phosphorylation flies.                                                BSF essential protein fly survival. <br />  <br /> Results                                                                  BSF deficiency leads respiratory chain dysfunction <br />                                                                          increased lactate levels <br /> BSF mitochondrial protein fly homologue                     To investigate biochemical consequence reduced BSF <br /> LRPPRC                                                                   expression levels measured respiratory chain enzyme <br />   We previously identified possible fly homologues           activities isolated mitochondria larval tissue. We found <br /> mammalian LRPPRC protein [4]. Here present additional                 respiratory chain complexes containing mitochondrial- <br /> phylogenetic analyses showing BSF closely related       ly-encoded subunits reduced enzyme activities. Complex I <br /> homologue LRPPRC flies (Figure 1A). Previous experiments           affected, complex I+III, complex II+III and <br /> have demonstrated punctuated cytoplasmic localization BSF           complex IV activities profoundly reduced bsf KD <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                           2                       October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                  BSF Drosophila melanogaster <br />  <br />  <br />  <br />  <br /> Figure 1. Phylogenetic analysis subcellular localization BSF. (A) Phylogenetic tree LRPPRC family proteins. Phylogenetic <br /> analyses arthropod families related LRPPRC, i.e. Bicoid stability factor (BSF) family CG14786 family. The BSF family is <br /> orthologous deuterostomian LRPPRC proteins. Dots indicate stable branches non-parametric bootstrap analysis (found 450 of <br /> 500 replicates). Orthologous sequences nematodes divergent, leading long branches. Abbreviations accession numbers as <br /> follows: H. sapiens, Homo sapiens, NP_573566; M. musculus, Mus musculus, NP_082509; M. domesticus, Monodelphis domesticus, XP_001382190, <br /> G. gallus, Gallus gallus, XP_001234903; X. tropicalis, Xenopus tropicalis, NP_001039203; D. rerio, Danio rerio, NP_001136064; S. kowalevskii, <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                            3                      October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                            BSF Drosophila melanogaster <br />  <br />  <br />  <br /> Saccoglossus kowaleskii, XP_002734047; S. purpuratus, Strongylocentrotus purpuratus, XP_001200846; B. floridae, Branchiostoma floridae, <br /> XP_002613352; A. pisum, Acyrthosiphon pisum, BSF: XP_001944507; D. melanogaster, Drosophila melanogaster, BSF: NP_523596, CG14786: <br /> NP_569913; A. gambiae, Anopheles gambiae, BSF: XP_557938, CG14786: XP_321013; C. quinquefasciatus, Culex quinquefasciatus, BSF: <br /> XP_001846253; A. aegypti, Aedes aegypti, BSF: XP_001658446, CG14786: XP_001657384; T. castaneum, Tribolium castaneum, BSF: XP_975329. C. <br /> elegans, Caenorhabditis elegans, C04E6.11: NP_504540, C14C10.4: NP_506151; B. malayi, Brugia malayi, 52970: XP_001902062, 51715: <br /> XP_001901811. (B) S2R+ cells expressing GFP-tagged BSF-FLAG fusion protein (BSF-FLAG-GFP), counterstained Mitotracker Deep Red <br /> (upper panels), scale bar size 10 mm HeLa cells expressing GFP-tagged BSF-FLAG fusion protein (BSF-FLAG-GFP), counterstained with <br /> Mitotracker Deep Red (lower panels), scale bar size 10 mm. (C) Western blot analyses nuclear, mitochondrial cytoplasmic fractions to <br /> determine subcellular localization BSF. Antibodies subunits NDUFS3 mitochondrial complex I, tubulin histone H3 were <br /> used assess purity fractions. <br /> doi:10.1371/journal.pgen.1002324.g001 <br />  <br /> flies (Figure 3A). Complex II, containing exclusively nuclear-                   levels significantly reduced mitochondria third- <br /> encoded subunits, exhibited normal enzyme activity (Figure 3A),                  instar bsf KD larvae flies presence substrates <br /> strongly supporting notion loss BSF specific                     entering respiratory chain level complex I (CPI), <br /> effects mtDNA gene expression. We analyzed                               substrates (SUCC-G3P) deliver electrons to <br /> mitochondrial respiratory capacity monitoring oxygen                          complex II glycerol-3-phosphate dehydrogenase, both <br /> consumption permeabilized tissues extracted third-                       upstream complex III (Figure 3B). There significant <br /> instar larvae thoraces flies. Maximal oxygen consumption                   reduction oxygen consumption combined substrates <br />  <br />  <br />  <br />  <br /> Figure 2. Ubiquitous bsf KD negatively affects hatching rate, climbing ability, body weight, life span Drosophila. (A) QRT-PCR <br /> of BSF transcript levels third-instar control (white grey bars) third-instar bsf KD (black purple bars) larvae six-day old flies (*p,0.05; <br /> ** <br />    p,0.01; ***p,0.001, n = 4-5). (B) Western blot analyses performed mitochondrial protein preparations third-instar larvae. Protein <br /> extracts, 10&#226;&#8364;&#8220;20 mg, separated standard SDS-PAGE followed Western blot analysis antibodies BSF. Coomassie staining the <br /> membrane used assess loading. (C) The relative hatching rate control lines (white grey bars) bsf KD lines (black bars). Hatching rates <br /> are shown relative controls. ***p,0.001, Student-t-test. (D) bsf KD impairs climbing abilities four-day old flies. ***p,0.001, Mann-Whitney U test. <br /> (E, F) Body weight three-day-old female male flies. ***p,0.001, Student-t&#226;&#8364;&#8220;test. (G) Reduced fecundity ubiquitous bsf KD. **p,0.01, <br /> Student-t-test. (H) Ovaries six-day old bsf KD females significantly smaller (small arrow) control ovaries (large arrow), scale bar size <br /> 1 mm. (I) bsf KD flies short lived. Survival curves bsf KD flies control flies standard food. <br /> doi:10.1371/journal.pgen.1002324.g002 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  4                        October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                    BSF Drosophila melanogaster <br />  <br />  <br />  <br />  <br /> Figure 3. Biochemical measurements respiratory chain function. (A) Relative enzyme activities respiratory chain enzyme complex I <br /> (NADH coenzyme Q reductase, NQR), complex I+III (NADH cytochrome c reductase, NCR), complex II (succinate dehydrogenase, SDH), complex II+III <br /> (succinate cytochrome c reductase, SCR) complex IV (cytochrome c oxidase, COX) third-instar control (white grey bars) bsf KD (black <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             5                      October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                       BSF Drosophila melanogaster <br />  <br />  <br />  <br /> and purple bars) larvae shown. Bars indicate mean 6 SD (*p,0.05; **p,0.01; ***p,0.001; n = 6-7). (B) Oxygen consumption normalized protein <br /> content third-instar control (white grey bars) bsf KD (black bars) larvae six-day old flies. Oxygen consumption assessed in <br /> permeabilized larvae thoraces six-day old flies using substrates entering level complex I (CPI), complex II (SUCC), glycerol-3- <br /> phosphate dehydrogenase, (G3P) complex I, II glycerol-3-phosphate dehydrogenase (CPI-SUCC-G3P), (C) Relative ATP/ADP levels six-day <br /> old bsf KD flies. (D) Lactate/pyruvate ratios six-day old bsf KD flies. <br /> doi:10.1371/journal.pgen.1002324.g003 <br />  <br /> (CPI-SUCC-G3P) adult fly mitochondria (Figure 3B). We                     BSF affects steady-state levels mitochondrial <br /> also observed progression decrease uncoupled                 transcripts <br /> oxygen consumption complex I substrates, compar-                       We performed detailed study steady-state levels of <br /> ing third-instar larvae flies (Figure 3B). In summary,               mitochondrial transcripts bsf KD lines using QRT- <br /> results measurements enzyme activities (Figure 3A)               PCR northern blot analyses. The levels analyzed <br /> oxygen consumption (Figure 3B) complex I                    mitochondrial mRNAs reduced third-instar bsf KD larvae <br /> most affected oxidative phosphorylation complexes,                    (Figure 4A, 4C, 4D Figure S1C) flies (Figure 4B), the <br /> perhaps high number mtDNA-encoded subunits.                    exception ND6, significantly reduced adult <br /> As result respiratory chain deficiency [ATP]/                   flies. QRT-PCR demonstrated slightly increased mtDNA levels in <br /> [ADP] ratio adult bsf KD flies reduced 44% ratio             third-instar bsf KD larvae (Figure 4E), showing reduced <br /> controls (Figure 3C). Furthermore, adult bsf KD flies                 mitochondrial mRNA levels explained mtDNA <br /> [lactate]/[pyruvate] ratio considerably increased                        depletion. In contrast, mitochondrially encoded small <br /> (Figure 3D), likely consequence compromised                 ribosomal subunit rRNA (12S rRNA) significantly increased, <br /> respiratory function, leads increased lactic acid                   large ribosomal subunit rRNA (16S rRNA) slightly <br /> fermentation order maintain reduction-oxidation                    decreased (Figure 4C 4D). Additionally, a <br /> homeostasis.                                                                 substantial increase levels analyzed mitochondrial tRNAs <br />  <br />  <br />  <br />  <br /> Figure 4. Steady-state levels mtDNA mitochondrial transcripts. (A-B) QRT-PCR analysis relative levels mitochondrial mRNAs in <br /> comparison nuclear ribosomal protein 49 transcript (A) third-instar larvae (B) six-day old flies. Northern Blot analyses (C) and <br /> quantification (D) mitochondrial transcripts normalized nuclear ribosomal protein 49 transcript third-instar larvae. (E) QRT-PCR analysis of <br /> mtDNA levels third-instar larvae. Northern blot analyses (F) quantification (G) mitochondrial tRNA levels third-instar larvae. (*p,0.05; <br /> ** <br />    p,0.01; ***p,0.001. n = 5). <br /> doi:10.1371/journal.pgen.1002324.g004 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              6                       October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                  BSF Drosophila melanogaster <br />  <br />  <br /> in third-instar bsf KD larvae (Figure 4F 4G). There             mRNA steady-state levels resulted decreased mitochondrial <br /> clear correlation level particular tRNA           translation assessing novo translation isolated mitochon- <br /> location gene fly mtDNA.                                         dria. Surprisingly, third-instar bsf KD larvae demonstrated a <br />                                                                            selective increase synthesis subunits complex I (ND1-6 <br /> Reduced BSF level leads increased novo                               ND4L) complex IV (COXI-III), synthesis of <br /> mitochondrial transcription aberrant translation                       subunits complex III (Cyt b) complex V (ATP6) remained <br />    The presence increased steady-state levels tRNAs makes         unchanged (Figure 5B). We observed exceptionally large <br /> unlikely reduction levels mRNAs explained            increase levels COXII subunit complex IV (Figure 5B, <br /> reduced transcription types mature transcripts              asterisk). We unidentified translation product <br /> produced processing polycistronic precursor transcripts. We          migrating ND1 subunit complex I (Figure 5B, <br /> nevertheless assessed mitochondrial novo transcription               arrow). Interestingly, aberrant translation products <br /> performing organello labeling experiments. Both third-instar bsf        invisible 3-hour chase cold methionine <br /> KD larvae (Figure 5A Figure S2B) bsf KD flies (Figure              (Figure 5B, left panel), suggesting subjected an <br /> S2A) dramatic increase novo transcription, showing             increased degradation shortly synthesis. <br /> that reduced mRNA levels explained increased <br /> degradation. The increased mitochondrial novo transcription             BSF required polyadenylation mitochondrial <br /> and increased mtDNA copy number likely parts                      mRNAs <br /> compensatory mitochondrial biogenesis response induced                 Decreased mitochondrial mRNA steady-state levels, combi- <br /> respiratory chain deficiency, which, turn, caused defective       nation increased novo transcription translation of <br /> post-transcriptional regulation mitochondrial mRNA stability         specific mitochondrial polypeptides (ND1-6 COX1-3) led us <br /> the absence BSF. We investigated decrease           investigate nature mature mitochondrial mRNAs. We <br />  <br />  <br />  <br />  <br /> Figure 5. De novo mitochondrial transcription, novo mitochondrial translation bsf KD larvae, polyadenylation profiles. <br /> (A) De novo transcription mtDNA determined a32P-UTP incorporation. Isolated mitochondria incubated a32P-UTP labeled <br /> transcripts separated MOPS/formaldehyde agarose gels normalized total mitochondrial protein content ml sample (lower <br /> panel). (B) Analysis novo mitochondrial translation determined S35-methionine incorporation, normalized total mitochondrial <br /> protein content ml sample (Figure 5A, lower panel) The asterisk indicates COXII subunit. The arrow indicates unidentified protein <br /> migration just ND1 subunit. (C) Examples electropherograms showing polyadenylation status COXII mRNA mitochondria of <br /> control (w;;daGAL4/+) bsf KD (w;;UAS-bsfRNAi/daGAL4) larvae. (D) A summary polyadenylation status mRNAs mitochondria third- <br /> instar controls (w;;daGAL4/+) bsf KD (w;;UAS-bsfRNAi/daGAL4) larvae. AAAA = 40-60 adenines, AA = ,20 adenines. Polycistronic transcript of <br /> COXIII-ATP6/8 ({), antisense tRNA-Threonine ({). <br /> doi:10.1371/journal.pgen.1002324.g005 <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             7                      October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                    BSF Drosophila melanogaster <br />  <br />  <br /> analyzed 59-and 39-ends mitochondrial transcripts, using            state levels assembled complex I third-instar bsf KD larvae <br /> RNA circularization, followed reverse transcription direct          (Figure 6C, Figure S2E) flies (Figure 6D). There major <br /> sequencing cloning subsequent sequencing (see Materials             reduction steady-state levels nuclear-encoded complex V <br /> and Methods). The 59 39 ends 12S 16S rRNA              subunits (a-subunit ATP synthase) third-instar bsf KD larvae <br /> identical bsf KD control larvae. However, observed               (Figure 6C, Figure S1B) flies (Figure 6D), suggesting normal <br /> severely reduced poly A tail lengths mitochondrial mRNAs            assembly. Together, results third-instar bsf KD <br /> except Cytb ATP6/8 (Figure 5C 5D). Interestingly,            larvae flies fail assemble sufficient levels complex I, III <br /> mRNAs reduced poly A tail <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> encoded polypeptides             IV supercomplexes consisting complexes, <br /> that showed increased levels novo synthesis. We              explains observed profound reduction oxidative <br /> analyzed 59 39ends polycistronic transcripts           phosphorylation capacity. <br /> containing mRNA retained tRNA 39 end (ND3 <br /> plus tRNA-Ala COXII plus tRNA-Lys). In control samples                 Discussion <br /> these polycistronic transcripts polyadenylated 39- <br /> ends, showing polyadenylation process specific             BSF previously suggested stabilize cytoplasmic <br /> fully processed mRNAs, occur free 39end. In          mRNAs oocytes early zygotic cells hours <br /> the bsf KD larvae, polycistronic RNAs severely reduced           fly embryogenesis [1,2]. However, ubiquitous expression of <br /> lengths polyA tail, suggesting lack              BSF RNA [3] punctuate cytoplasmic localization in <br /> polyadenylation direct consequence loss BSF         flies prompted re-investigate function BSF the <br /> a secondary effect impaired translation. In bsf KD larvae,       fly. Surprisingly, able demonstrate BSF protein <br /> consistently failed recover mature COX III mRNA            mainly localized mitochondria, controls polyadenyl- <br /> observed COXIII large polycistronic RNA. The 59part           ation specific mitochondrial mRNAs. Further, BSF plays key <br /> of RNA consisted ATP6/8 3&#226;&#8364;&#732;part COXIII. This             role regulation mtDNA gene expression, coordinating <br /> RNA undergone correct processing 39 end COXIII             mitochondrial translation flies. <br /> but lacked polyadenylation. The lack mature polyadenylated                  GFP-tagged BSF localized mitochondria transfected tissue <br /> COXIII transcripts bsf KD larvae likely explained        culture cells cell fractionation experiments showed BSF is <br /> a combination defective RNA processing defective                    mainly present mitochondria adult flies. Interestingly, BSF <br /> polyadenylation. Cloning subsequent sequencing                  detectable cytoplasmic nuclear fractions of <br /> ND4/ND4L PCR product revealed different RNA species                adult flies. It important note mammalian homolog of <br /> in bsf KD samples, indicating polyadenylation                 BSF, LRPPRC, reported roles regulation <br /> processing differences comparison control samples                  cytoplasmic mRNA transport [6-8] regulation nuclear <br /> (Figure 5D). In summary, sequencing RNA 59 39-ends               transcription [9,10]. However, convincing evidence proposes that <br /> third-instar bsf KD larvae revealed severe mRNA maturation               main proportion LRPPRC localized mitochondria <br /> defect reduced poly A tail lengths enrichment               [4,10,11]. In study tissue culture cells, endogenously <br /> unprocessed polycistronic RNA intermediates containing COXIII              expressed LRPPRC detectable highly purified nuclei <br /> and ND4/ND4L sequences.                                                    lacking mitochondrial contamination [4]. These findings not <br />                                                                            exclude small fraction BSF LRPPRC <br /> The biochemical defect bsf KD larvae caused                       localized subcellular compartments mitochondria. <br /> reduced levels assembled complex I, III, IV                         However, proposed extramitochondrial functions BSF and <br />    Despite observed compensatory increase novo                   LRPPRC revisited new set experiments focusing <br /> transcription translation, third-instar bsf KD larvae           avoiding mitochondrial contamination isolated nuclei and <br /> severe respiratory chain dysfunction presumably causing delayed            cytoplasmic extracts. <br /> hatching reduced lifespan. Sequencing mitochondrial                 Ubiquitous RNAi-induced reduction endogenous bsf levels in <br /> mRNAs showed reduced length poly A tails processing                 flies causes mitochondrial dysfunction decreased respiratory <br /> defects, affect translational initiation lead        chain enzyme activities reduced levels assembled complex I, <br /> production abnormal polypeptides. Subunits complex I             III IV. This mitochondrial dysfunction leads severe <br /> IV translated increased rates, subset newly           phenotypes, including delayed incomplete eclosure, reduced <br /> synthesized subunits preferentially degraded             fecundity, sterility shortened life span. <br /> (Figure 5B), suggesting fail assemble mature                 Mammalian cell lines reduced LRPPRC levels have <br /> complexes. We assessed levels assembled                   reduced mitochondrial mRNA steady-state levels reduced <br /> respiratory chain enzyme complexes using Blue-Native                    mitochondrial translation [12,18]. Interestingly, downregulation of <br /> polyacrylamide gel electrophoresis (BN-PAGE). We observed                  BSF protein levels flies leads decreased steady-state levels <br /> reduced levels assembled complex I, complex III, complex IV             mitochondrial mRNAs but, contrast results in <br /> and supercomplexes third-instar bsf KD larvae (Figure 6A            mammals, translation increased aberrant translation <br /> 6B) flies (Figure S2C S2D). The levels assembled                products generated. Some mitochondrial translation <br /> complex V (ATP synthase) unaffected. This reduction                products synthesized increased rates flies subject <br /> steady-state levels assembled complexes accompanied              increased degradation, suggesting BSF act a <br /> reduced in-gel activity complex I (Figure 6A, Figure S2C)           translational coordinator. Regulation mitochondrial gene <br /> complex IV (Figure 6B, Figure S2D). The BN-PAGE complex                expression response different metabolic demands animal <br /> I in-gel activity analyses showed reduced steady-state levels           cells largely unknown, likely regulation occurs at <br /> supercomplexes presence smaller, partially assembled,         different levels. The basic mtDNA transcription machinery <br /> form complex I third-instar bsf KD larvae (Figure 6A,                defined [19], unclear regulated. <br /> asterisk). Western blot analyses showed reduced levels nuclear        Processing mRNAs required correct translation [20], but <br /> encoded subunit complex I (NDUFS3 subunit), indicating              coupling transcript processing translation is <br /> supporting notion severe reduction steady-          poorly understood. Furthermore, mitochondria essence a <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             8                       October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                          BSF Drosophila melanogaster <br />  <br />  <br />  <br />  <br /> Figure 6. Analyses assembled respiratory chain enzyme complexes bsf KD larvae. (A-B) BN-PAGE analysis mitochondrial protein <br /> extracts third-instar bsf KD control larvae. (A) In-gel activity complex I, (B) In-gel activity complex IV. (C-D) Western blot analyses of <br /> levels nuclear encoded subunit NDUFS3 (complex I) a-subunit ATP synthase (complex V) (C) third-instar bsf KD larvae (D) one- <br /> day-old bsf KD flies. <br /> doi:10.1371/journal.pgen.1002324.g006 <br />  <br />  <br /> prokaryotic compartmentalization                        leads destabilization mitochondrial mRNAs reduced <br /> transcription translation, makes likely              steady-state levels, despite simultaneously increased novo <br /> these processes directly communicate.                                          transcription. In animals mitochondrial transcripts are <br />    Here demonstrate loss BSF results incorrect               polyadenylated mature mRNA ready <br /> processing polycistronic precursor transcripts failure            translation, ND6 vertebrates, humans <br /> to polyadenylate subset processed polycistronic                  fish long 39UTR poly A tail [16,21]. In the <br /> transcripts. This failure mature mitochondrial mRNAs, turn,              mammalian believed nearly mitochondrial <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                9                        October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                       BSF Drosophila melanogaster <br />  <br />  <br /> RNAs oligoadenylated transcription unknown                  mitochondrial translation inhibitor thiamphenicol increased <br /> enzyme, followed polyadenylation approximately 50                     steady-state levels mitochondrial mRNAs [31]. However, <br /> adenines mitochondrial poly A polymerase (mtPAP) [22]. A                 decreased translation does increase mRNA steady- <br /> number mitochondrial transcripts require polyadenylation                  state levels exemplified conditional mouse <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> for <br /> generate functional stop codons, exact function               TFB1M, abolishes mitochondrial translation, does not <br /> of mRNA polyadenylation mitochondria unknown. While                    affect levels mitochondrial mRNAs despite activation <br /> bacteria chloroplasts poly A tail promote                      novo transcription [32] . <br /> transcript degradation, eukaryotes polyadenylate nearly                    In conclusion, present unexpected result the <br /> every fully processed cytosolic mRNA 39 end, resulting                BSF protein mainly localized mitochondria, controls <br /> increased mRNA stability, increased translational efficiency,               polyadenylation specific mitochondrial mRNAs. In <br /> promotion transport processed mRNA nucleus                   addition, results suggests BSF novel role in <br /> to cytoplasm [23]. A primary role BSF maturation               coordinating mitochondrial translation loss BSF leads to <br /> mitochondrial mRNAs supported recent observation                 increased uncoordinated translation increase synthesis of <br /> PPR proteins trypanosomes affect polyadenylation mito-                    unstable translation products. BSF essential role in <br /> chondrial mRNAs. In unicellular protozoon, PPR proteins                regulating mitochondrial function fly. <br /> KPAF1 KPAF2 associate mtPAP, stimulating mRNA <br /> polyadenylation coordinate stability translation             Materials Methods <br /> mRNA [24]. Our results suggest BSF involved actual <br /> mRNA maturation process controlling polyadenylation                       Drosophila stocks maintenance <br /> specific transcripts.                                                              For vivo KD studies independent non-overlapping UAS- <br />    The mammalian BSF homolog LRPPRC shown                        bsf RNAi lines used. w;;UAS-bsfRNAi#1 (#10302R-I) was <br /> stabilized second RNA-binding protein called SLIRP,                   obtained National Institute Genetics (Japan) and <br /> direct interaction [12]. A SLIRP homolog suggested                w;UAS-bsfRNAi#2 (#22839) obtained Vienna <br /> to exist flies, raising possibility interaction        Drosophila RNAi Center (VDRC). Ubiquitous bsf knock down <br /> a second RNA binding protein required fly mitochondria.              achieved crossing UAS-bsf RNAi lines daughterless- <br /> Interestingly, bioinformatics analyses identified additional                 GAL4 (da-GAL4) driver line. UAS-RNAi lines daGAL4 <br /> LRPPRC homolog, BSF, Drosphila melanogaster [4],                     driver lines backcrossed 6 generations the <br /> suggesting homologs work control                      white Dahomey background (wDahT). All fly stocks free <br /> polyadenylation translation different sets mitochondrial              endosymbiontic bacterium Wolbachia. Flies were <br /> mRNAs. This supported observation                        propagated experiments conducted 25uC a <br /> mitochondrial transcripts, cytb ATP6/8,                   12 h:12 h light:dark cycle constant humidity standard <br /> polyadenylated bsf KD lines.                                             sugar-agar-yeast medium. <br />    Loss BSF results increased aberrant translation, <br /> and reduced levels assembled RC complex I, III IV,                       Bioinformatics <br /> suggesting BSF coordinates mitochondrial translation RC <br />                                                                                    Homologs LRPPRC collected using PSI-BLAST [33] <br /> complex assembly. The coupling transcription translation is <br />                                                                                 RefSeq protein database NCBI aligned using <br /> supported studies yeast homologue BSF and <br />                                                                                 ClustalX [34] BLOSUM matrix. Multiple sequence <br /> LRPPRC, PET309 [25]. This factor implicated activation <br />                                                                                 alignments trimmed using GBLOCKS server [35] with <br /> of translation binding 59UTR yeast mitochondrial <br />                                                                                 relaxed settings. The trimmed alignment submitted PhyML <br /> mRNAs, tethering mitochondrial polysomes inner <br />                                                                                 [36] using standard parameters non-parametric bootstrap <br /> mitochondrial membrane, which, turn, thought ensure co- <br />                                                                                 analysis 500 replicates. The resulting tree displayed using <br /> translational insertion newly synthesized polypeptides the <br />                                                                                 Dendroscope [37] prepared publication Illustrator. <br /> assembling respiratory chain complexes [26,27]. Mitochondrial <br /> mRNAs 59-UTRs translated normal levels <br /> followed rapid degradation, suggesting targeting                 Colocalization studies <br /> inner mitochondrial membrane necessary stability                     Full-length bsf cDNA obtained Drosophila <br /> newly translated peptides [28]. It important point               Genomics Resource Center (SD10676, AY058795). Two amino <br /> most mRNAs encoded metazoan mtDNA lack 59UTRs                            acid changing substitutions identified bsf cDNA in <br /> targeting translation inner membrane animal                        comparison reference sequence (FBtr0081087). The <br /> mitochondria, occurs, involve, partly,          corresponding mutations nucleotide positions 415 710 were <br /> different regulatory mechanisms.                                                changed site-directed mutagenesis cDNA, using the <br />    Cytoplasmic mRNAs different types destabiliz-                QuickChange II XL Site-Directed Mutagenesis Kit (Agilent <br /> ing elements translation coupled                        Technologies). A cDNA encoding FLAG-tagged BSF cloned <br /> degradation classes mRNAs [29]. If similar                         plasmid pAcGFP1-N2 (Clontech, Mountain View, USA) <br /> mechanism exists mitochondria, suggest                     generate vector pbsfFLAG-AcGFP1, encodes fusion <br /> increased novo translation BSF KD larvae results                       protein consisting BSF-FLAG in-frame addition of <br /> increased degradation mitochondrial mRNAs. Some                          green fluorescent protein (GFP) carboxy-terminus (BSF- <br /> support proposed mechanism comes characterization                 FLAG-GFP). <br /> of patient microdeletion genes ATP6/8                     Schneider 2R+ HeLa cells transfected microscopy <br /> and COXIII, reported cause incorrect                          dishes (m-Dish, ibidi, Martinsried, Germany) pbsfFLAG- <br /> processing corresponding mitochondrial mRNAs                         AcGFP1 using FuGENE HD Transfection Reagent (Roche <br /> severely reduced steady-state levels ATP6/8 transcript               Diagnostics, Mannheim, Germany). The mitochondrial counter- <br /> to translationally induced deadenylation [30]. Additional support               staining achieved 100 nm MitoTracker Deep Red FM <br /> comes observation human cells treated                    (Invitrogen, Darmstadt, Germany). Live cell image acquisition was <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                10                      October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                    BSF Drosophila melanogaster <br />  <br />  <br /> performed Leica TCS SP5-X confocal microscope (Leica                  Biosciences). Membranes hybridized 32P-labeled probes <br /> Microsystems, Wetzlar, Germany).                                             exposed PhosphoImager Screens and/or X-ray <br />   The colocalization rate determined software LAS AF              films. Labeling mitochondrial double-stranded DNA probes and <br /> (Leica Microsystems, Wetzlar, Germany) following                   oligonucleotides performed described [40]. Primers and <br /> conditions calculations: threshold 30%, background 20%,                  oligonucleotides used Northern blot listed Table S1. <br /> colocalization rate [%] = colocalization area/area foreground, <br /> and area foreground = area image 2 area background.                          Biochemical evaluation respiratory chain function <br />                                                                                 Isolation mitochondria third-instar larvae per- <br /> Subcellular fractionation Western blot analysis                          formed described [41] modifications buffer composition. <br />    All fractions isolated adult wDahT flies. Mitochon-             Briefly, third-instar larvae washed gently homogenized <br /> drial, cytoplasmic nuclear fractions isolated                    ice-cold MSB buffer (210 mM mannitol, 70 mM sucrose, <br /> differential centrifugation previously described [38]. The used           10 mM EDTA, 50 mM Tris, pH 7.5) using 15 ml Dounce <br /> primary antibodies were: HISTONE H3 (Santa Cruz Biotechnol-                  homogenizers. Protein concentration determined using <br /> ogy, dilution 1:200), Complex I-subunit NDUFS3 (Mitoscience                  Bradford assay aliquots corresponding 10 mg mitochondrial <br /> MS112, dilution 1:1000), tubulin (Sigma, dilution 1:1000)                proteins pelleted resuspended resuspension buffer <br /> polyclonal rat antisera raised BSF (kindly provided               (250 mM sucrose, 15 mM K2HPO4, 2 mM MgAc2, 0.5 mM <br /> Professor MacDonald PM, Stanford University, dilution 1:1000).               EDTA 0.5 g/L HSA, pH 7.2). Biochemical activities of <br /> Protein bands visualized ECL western blotting reagents             respiratory chain complexes determined described [42]. <br /> (Bio-Rad). <br />    Western blot analyses performed using fly                   Respiratory rates <br /> mitochondrial protein extracts according Cell Signaling                  Third-instar larvae (n = 10) thoraces adult flies (n = 5) <br /> Technology protocol (CellSignaling). Additional primary antibod-             dissected PBS resuspended 2 ml respiratory <br /> ies used were: complex V (Mitoscience MS504, dilution 1:5000),               buffer (120 mM sucrose, 50 mM KCl, 20 mM Tris-HCl, 4 mM <br /> VDAC (Mitoscience MSAO3, dilution 1:2000), actin (Sigma,                     KH2PO4, 2 mM MgCl2, 1 mM EGTA, 0.01% digitonin, <br /> dilution 1:1000).                                                            pH 7.2). Oxygen consumption measured 25uC using an <br />                                                                              oxygraph chamber (OROBOROS). Complex I-dependent respi- <br /> Hatching rate, lifespan, fecundity, climbing assays                   ration assessed adding substrates proline (10 mM), <br /> flies                                                                        pyruvate (10 mM), malate (5 mM) glutamate (5 mM). <br />    For adult hatching rate measurements eggs collected                  Succinate glycerol-3-phosphate dehydrogenase activities were <br /> during 3 hours time window transferred vials (80 eggs/              measured using 20 mM succinate (SUCC) 15 mM glycerol-3- <br /> vial) ensure standard larval density. Hatching adult flies         phosphate (G3P), respectively. Mitochondrial quality each <br /> monitored regular intervals. After hatching, virgin females           sample assessed measuring respiratory control rate <br /> males collected mated 2 days.                                   (RCR), using 1 mM ADP (state 3) 1 mM ADP 2.5 mg/ml <br />    For lifespan analyses, 50&#226;&#8364;&#8220;100 females genotype used           oligomycin (pseudo state 4). Permeabilized control mitochondria <br /> a density 10 flies vial. Flies transferred new vials          consistently RCR values 4 7 complex I <br /> every days dead flies counted. For fecundity           substrates. <br /> assays, 100 females equally distributed vials,                      The respiration uncoupled addition 400 mM <br /> transferred new vials day number eggs                CCCP rotenone-sensitive flux measured the <br /> counted.                                                                     presence 200 mM rotenone. Finally, protein content was <br />    Climbing assays conducted performance index                  determined Bradford method (BioRad) order to <br /> calculated described [39]. 100 three-day-old males genotype           normalize oxygen consumption flux mitochondrial protein <br /> were tested climbing ability.                                      content. <br />  <br /> DNA isolation quantitative RT&#226;&#8364;&#8220;PCR                                        ATP/ADP lactate/pyruvate ratios <br />    DNA third-instar larvae extracted using DNAeasy                   Flies (n = 10) snap frozen directly liquid nitrogen and <br /> Kit (Qiagen). Mitochondrial DNA levels determined                    kept -80uC. Acidic extraction (PCA 7%) performed, <br /> quantitative real-time PCR (QRT-PCR) 7900HT Real Time                   samples centrifuged (16000 g, 10 min), supernatant was <br /> PCR (Applied Biosystems), using SYBR green master mix                 neutralized 2N KOH, 10 mM MOPS metabolites were <br /> (Invitrogen). Reactions carried triplicates sample        quantified. ADP ATP levels assessed previously <br /> a final volume 20 ml 5 ng DNA 10 pmol specific             described [43,44]. Briefly, ATP quantified using the <br /> primers (primers listed Table S1).                                    ATPlite step kit (PerkinElmer). For ADP levels, samples were <br />                                                                              incubated 10 min 37uC 75 mM KCl, 8 mM MgSO4, <br /> RNA isolation, QRT-PCR, Northern blot analysis                           10 mg/ml pyruvate kinase 2 mM phosphoenolpyruvate. <br />    Total RNA third-instar larvae adult flies extracted           Lactate pyruvate concentrations determined 1 h <br /> using Totally RNA KIT (Ambion). Reverse transcription                incubation horse radish peroxydase 5 U/ml, Amplex red <br /> QRT-PCR performed using High capacity RNA-to-                        20 mM, 0.1 M phosphate, pH 7.2, supplemented lactate <br /> cDNA kit (Applied Biosystems) Taqman 2x Universal                    oxidase pyruvate oxidase, followed fluorimetric analysis <br /> PCR mastermix, No Amperase UNG (Applied Biosystems),                         (ex:560 nm, em:590 nm), using Infinite 200 Pro fluorimeter <br /> respectively. Custom-made TaqMan probes Drosophila                   (Tecan). <br /> mitochondrial transcripts obtained Applied Biosystems <br /> and listed Table S1.                                                  In organello transcription translation assays <br />    For Northern blot analyses, RNA fractionated 1.2%                    For preparation mitochondria, third-instar larvae flies <br /> agarose gels blotted Hybond-N+ membranes (Amersham                    homogenized ice-cold isolation buffer STE+BSA (250 mM <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              11                      October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                      BSF Drosophila melanogaster <br />  <br />  <br /> sucrose, 5 mM Tris, 2 mM EGTA, 1% (w/v) BSA, pH 7.4) using                   Statistical analysis <br /> a 15 ml Dounce homogenizer. Cellular debris pelleted                    Data presented mean 6 SD. The Mann-Whitney test <br /> 1000 g 5 min supernatants transferred new tubes.             used analyze climbing index log-rank test used <br /> Mitochondria washed times final mitochondrial                   analyze lifespan. Unpaired t-test used analyze other <br /> pellets resuspend 1 ml STE buffer presence                 data statistically. <br /> 200 mg/ml emetine (Sigma) 100 mg/ml cycloheximide <br /> (Sigma) inhibit cytoplasmic translation. Protein concentrations           Supporting Information <br /> were determined using Bradford assay. <br />    In organello transcription assays performed described [45]        Figure S1 Phenotypic analysis steady state level of <br /> using 200 mg mitochondria/sample modified transcription                mitochondrial transcripts bsf KD flies. (A) Western blot analyses <br /> buffer (25 mM sucrose, 75 mM sorbitol, 100 mM KCl, 10 mM                     antibodies BSF performed mitochondrial <br /> K2HPO4, 50 mM EDTA, 5 mM MgCl2, 1 mM ADP, 10 mM                              protein preparations six-day old bsf KD control flies. <br /> glutamate, 2.5 mM malate, 10 mM Tris-HCl (pH 7.4) 1%                     Antibodies actin used assess loading. (B) <br /> (w/v) BSA). In short, labeling, mitochondrial RNA                  Quantification ovary sizes six-day old bsf KD control <br /> isolated using Totally RNA kit (Ambion). Mitochondrial RNA               flies. (C) QRT-PCR analysis relative levels mitochondrial <br /> fractionated 1.2% agarose gels blotted Hybond-N+                   mRNAs comparison nuclear ribosomal protein 49 <br /> membranes (Amersham Biosciences).                                            transcript third-instar bsfRNAi#2 KD control flies. <br />    In vitro assays study mitochondrial novo translation           (TIF) <br /> [35S]-methionine performed described [46]. Equal amounts             Figure S2 Level novo mitochondrial transcription and <br /> of total mitochondrial protein loaded 15% SDS-PAGE                   analyses assembled respiratory chain complexes bsf KD <br /> gels. Gels fixed isopropanol-acetic solution, stained           larvae flies. (A B) De novo transcription mtDNA <br /> Coomassie, destained ethanol-acetic acid solution treated             determined a32P&#226;&#8364;&#8220;UTP incorporation. Isolated mitochondria <br /> with Amplify Solution (GE Healthcare). Afterwards gels dried            incubated a32P&#226;&#8364;&#8220;UTP labeled transcripts were <br /> and [35S]&#226;&#8364;&#8220;methionine-labelled proteins visualized autora-            separated MOPS/formaldehyde agarose gels. (A) six-day old <br /> diography. The mitochondrial translation profile compared             bsf KD control flies. (B) third-instar bsf KD control <br /> previously published profiles Schneider cell lines [47], addition-        larvae. De novo transcription (left panel), probing same <br /> ally ND2 ATP6 identified endopeptidase fingerprinting            membrane detect 16S rRNA size comparison (right panel). <br /> in second dimension (data shown) [48].                               (C) BN-PAGE analysis mitochondrial protein extracts six- <br />                                                                              day old bsf KD control flies. The assembled respiratory chain <br /> RNA circularization RT&#226;&#8364;&#8220;PCR                                               complexes supercomplexes shown left panel. The <br />   An RNA circularization protocol modified [49,50].                 right panel shows in-gel activity complex I. (D) BN-PAGE <br /> Approximately 6 ng total mitochondrial RNA circularized                  analysis mitochondrial protein extracts six-day old bsf KD <br /> with 5 U T4 RNA ligase 200 ml 16uC 16 h                flies. The assembled respiratory chain complexes super- <br /> manufacturer-supplied buffer (NEB). The circularized RNA                 complexes shown left panel. The right panel shows in- <br /> precipitated equal volume isopropanol, incubated -             gel activity complex IV. (E) Western blot analyses levels of <br /> 20uC 4 h, centrifuged 20 min speed          nuclear encoded subunit NDUFS3 (complex I) a-subunit <br /> bench centrifuge. The entire precipitate used                    ATP synthase (complex V) third-instar bsf KD larvae. <br /> complementary DNA synthesis gene specific primers using                 Antibodies VDAC used assess loading. <br /> GeneAmp RNA PCR kit (Applied Biosystems). PCR products                       (TIF) <br /> were purified using ExoSAP-IT (Affymetrix) sequenced. <br />                                                                              Table S1 List oligonucleotide sequences Taqman probes <br /> Selected PCR products cloned pCR-II (Invitrogen) <br />                                                                              used cloning BSF-FLAG-GFP construct quantifica- <br /> and sequenced order confirm results direct <br />                                                                              tion steady-state levels mtDNA, mt-tRNAs mt-mRNAs. <br /> sequencing. Primer sequences RT-PCR subsequent PCR <br />                                                                              (TIF) <br /> are contained Table S2. <br />                                                                              Table S2 List oligonucleotide sequences used RT PCR and <br /> Blue native-polyacrylamide gel electrophoresis                               subsequent PCR determine polyadenylation profile mt <br /> (BN-PAGE) in-gel histochemistry                                          transcripts. RT-PCR using forward primer (F) and <br />    For BN-PAGE, 75 mg mitochondria pelleted lyzed                subsequent PCR sequencing using forward primer in <br /> in 50 ml ice-cold digitonin buffer (1% digitonin, 20 mM Tris                 combination reverse (R) primer. Forward primer COXI <br /> pH 7.4, 0.1 mM EDTA, 50 mM NaCl, 10% glycerol, 1 mM                          additional primer sequences used polyadenylation sequencing <br /> PMSF). After 15 min incubation ice, unsolubilized material             designed according Stewart Beckenbach, 2009 [50]. <br /> was removed centrifugation 4uC. The supernatant                    (TIF) <br /> mixed 5 ml 10 x loading dye (5% (w/v) Coomassie Brilliant <br /> Blue G-250, 100 mM Tris pH 7, 500 mM 6-aminocaproic acid)                    Acknowledgments <br /> and loaded 4&#226;&#8364;&#8220;10% gradient BN-PAGE gels [51,52]. In gel                    We thank Ewa Gustafsson technical assistance assays the <br /> complex I activity determined incubating BN-PAGE                  respiratory chain enzyme activities. <br /> gels 2 mM Tris-HCl pH 7.4, 0.1 mg/ml NADH (Roche) and <br /> 2.5 mg/ml iodonitrozolium (Sigma). In gel complex IV activity                Author Contributions <br /> was determined incubating BN-PAGE gels 50 ml of <br />                                                                              Conceived designed experiments: AB AW SG JBS BR LP N-GL. <br /> 0.05 mM phosphate buffer pH 7.4, 25 mg 3.39-diamidobenzidine                 Performed experiments: AB AW AM BR CK RW BH. Analyzed the <br /> tetrahydrochloride (DAB), 50 mg cytochrome c, 3.75 g sucrose                 data: AB AW JBS AM BR RW BH LP N-GL. Contributed reagents/ <br /> and 1 mg catalase. All stainings carried room                    materials/analysis tools: BH SG CK AM. Wrote paper: AB AW LP N- <br /> temperature.                                                                 GL. <br />  <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              12                       October 2011 | Volume 7 | Issue 10 | e1002324 <br />                                                                                                                                              BSF Drosophila melanogaster <br />  <br />  <br />  <br /> References <br />  1. Mancebo R, Zhou X, Shillinglaw W, Henzel W, Macdonald PM (2001) BSF                      25. Mootha VK, Lepage P, Miller K, Bunkenborg J, Reich M, et al. (2003) <br />     binds specifically bicoid mRNA 39 untranslated region contributes              Identification gene causing human cytochrome c oxidase deficiency by <br />     stabilization bicoid mRNA. Mol Cell Biol 21: 3462&#226;&#8364;&#8220;3471. doi:10.1128/                      integrative genomics. Proc Natl Acad Sci USA 100: 605&#226;&#8364;&#8220;610. doi:10.1073/ <br />     MCB.21.10.3462-3471.2001.                                                                    pnas.242716699. <br />  2. De Renzis S, Elemento O, Tavazoie S, Wieschaus EF (2007) Unmasking                       26. Manthey GM, McEwen JE (1995) The product nuclear gene PET309 is <br />     activation zygotic genome using chromosomal deletions Drosophila               required translation mature mRNA stability production intron- <br />     embryo. PLoS Biol 5: e117. doi:10.1371/journal.pbio.0050117.                                 containing RNAs derived mitochondrial COX1 locus Saccharomy- <br />  3. Chintapalli VR, Wang J, Dow JAT (2007) Using FlyAtlas identify better                     ces cerevisiae. EMBO J 14: 4031&#226;&#8364;&#8220;4043. <br />     Drosophila melanogaster models human disease. Nat Genet 39: 715&#226;&#8364;&#8220;720.                  27. Manthey GM, Przybyla-Zawislak BD, McEwen JE (1998) The Saccharomyces <br />     doi:10.1038/ng2049.                                                                          cerevisiae Pet309 protein embedded mitochondrial inner membrane. <br />  4. Sterky FH, Ruzzenente B, Gustafsson CM, Samuelsson T, Larsson N-G (2010)                     Eur J Biochem 255: 156&#226;&#8364;&#8220;161. <br />     LRPPRC mitochondrial matrix protein conserved metazoans.                 28. Sanchirico ME, Fox TD, Mason TL (1998) Accumulation mitochondrially <br />     Biochemical Biophysical Research Communications 398: 759&#226;&#8364;&#8220;764.                            synthesized Saccharomyces cerevisiae Cox2p Cox3p depends targeting <br />     doi:10.1016/j.bbrc.2010.07.019.                                                              information untranslated portions mRNAs. EMBO J 17: 5796&#226;&#8364;&#8220;5804. <br />  5. Lurin C, Andre&#194;&#180;s C, Aubourg S, Bellaoui M, Bitton F, et al. (2004) Genome-wide               doi:10.1093/emboj/17.19.5796. <br />     analysis Arabidopsis pentatricopeptide repeat proteins reveals essential        29. Sachs AB (1993) Messenger RNA degradation eukaryotes. Cell 74: 413&#226;&#8364;&#8220;421. <br />     role organelle biogenesis. Plant Cell 16: 2089&#226;&#8364;&#8220;2103. doi:10.1105/                     30. Temperley RJ, Seneca SH, Tonska K, Bartnik E, Bindoff LA, et al. (2003) <br />     tpc.104.022236.                                                                              Investigation pathogenic mtDNA microdeletion reveals translation- <br />  6. Mili S, Shu HJ, Zhao Y, Pin&#203;&#339;ol-Roma S (2001) Distinct RNP complexes                       dependent deadenylation decay pathway human mitochondria. Hum Mol <br />     shuttling hnRNP proteins pre-mRNA mRNA: candidate intermediates                     Genet 12: 2341&#226;&#8364;&#8220;2348. doi:10.1093/hmg/ddg238. <br />     formation export mRNA. Mol Cell Biol 21: 7307&#226;&#8364;&#8220;7319. doi:10.1128/               31. Chrzanowska-Lightowlers ZM, Preiss T, Lightowlers RN (1994) Inhibition of <br />     MCB.21.21.7307-7319.2001.                                                                    mitochondrial protein synthesis promotes increased stability nuclear-encoded <br />  7. Tsuchiya N, Fukuda H, Nakashima K, Nagao M, Sugimura T, et al. (2004)                        respiratory gene transcripts. J Biol Chem 269: 27322&#226;&#8364;&#8220;27328. <br />     LRP130, single-stranded DNA/RNA-binding protein, localizes outer                32. Metodiev MD, Lesko N, Park CB, Camara Y, Shi Y, et al. (2009) Methylation of <br />     nuclear endoplasmic reticulum membrane, interacts mRNA                       12S rRNA necessary vivo stability small subunit the <br />     vivo. Biochemical Biophysical Research Communications 317: 736&#226;&#8364;&#8220;743.                      mammalian mitochondrial ribosome. Cell Metab 9: 386&#226;&#8364;&#8220;397. doi:10.1016/ <br />     doi:10.1016/j.bbrc.2004.03.103.                                                              j.cmet.2009.03.001. <br />  8. Topisirovic I, Siddiqui N, Lapointe VL, Trost M, Thibault P, et al. (2009)               33. Altschul SF, Madden TL, Scha&#194;&#168;ffer AA, Zhang J, Zhang Z, et al. (1997) Gapped <br />     Molecular dissection eukaryotic initiation factor 4E (eIF4E) export-                  BLAST PSI-BLAST: new generation protein database search <br />     competent RNP. EMBO J 28: 1087&#226;&#8364;&#8220;1098. doi:10.1038/emboj.2009.53.                              programs. Nucleic Acids Res 25: 3389&#226;&#8364;&#8220;3402. <br />  9. Labialle S, Dayan G, Gayet L, Rigal D, Gambrelle J, et al. (2004) New                    34. Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al. (2003) Multiple <br />     invMED1 element cis-activates human multidrug-related MDR1 MVP                           sequence alignment Clustal series programs. Nucleic Acids Res 31: <br />     genes, involving LRP130 protein. Nucleic Acids Res 32: 3864&#226;&#8364;&#8220;3876.                        3497&#226;&#8364;&#8220;3500. <br />     doi:10.1093/nar/gkh722.                                                                  35. Talavera G, Castresana J (2007) Improvement phylogenies removing <br /> 10. Cooper MP, Qu L, Rohas LM, Lin J, Yang W, et al. (2006) Defects energy                    divergent ambiguously aligned blocks protein sequence alignments. <br />     homeostasis Leigh syndrome French Canadian variant PGC-1alpha/                    Syst Biol 56: 564&#226;&#8364;&#8220;577. doi:10.1080/10635150701472164. <br />     LRP130 complex. Genes Dev 20: 2996&#226;&#8364;&#8220;3009. doi:10.1101/gad.1483906.                        36. Guindon S, Delsuc F, Dufayard J-F, Gascuel O (2009) Estimating maximum <br /> 11. Xu F, Morin C, Mitchell G, Ackerley C, Robinson BH (2004) The role                    likelihood phylogenies PhyML. Methods Mol Biol 537: 113&#226;&#8364;&#8220;137. <br />     LRPPRC (leucine-rich pentatricopeptide repeat cassette) gene cytochrome                   doi:10.1007/978-1-59745-251-9_6. <br />     oxidase assembly: mutation causes lowered levels COX (cytochrome c                    37. Huson DH, Richter DC, Rausch C, Dezulian T, Franz M, et al. (2007) <br />     oxidase) I COX III mRNA. Biochem J 382: 331&#226;&#8364;&#8220;336. doi:10.1042/                            Dendroscope: An interactive viewer large phylogenetic trees. BMC <br />     BJ20040469.                                                                                  Bioinformatics 8: 460. doi:10.1186/1471-2105-8-460. <br /> 12. Sasarman F, Brunel-Guitton C, Antonicka H, Wai T, Shoubridge EA, et al.                  38. Cox B, Emili A (2006) Tissue subcellular fractionation protein extraction for <br />     (2010) LRPPRC SLIRP interact ribonucleoprotein complex                         use mass-spectrometry-based proteomics. Nat Protoc 1: 1872&#226;&#8364;&#8220;1878. <br />     regulates posttranscriptional gene expression mitochondria. Mol Biol Cell 21:             doi:10.1038/nprot.2006.273. <br />     1315&#226;&#8364;&#8220;1323. doi:10.1091/mbc.E10-01-0047.                                                  39. Greene JC, Whitworth AJ, Kuo I, Andrews LA, Feany MB, et al. (2003) <br /> 13. Boore JL (1999) Animal mitochondrial genomes. Nucleic Acids Res 27:                          Mitochondrial pathology apoptotic muscle degeneration Drosophila <br />     1767&#226;&#8364;&#8220;1780.                                                                                   parkin mutants. Proc Natl Acad Sci USA 100: 4078&#226;&#8364;&#8220;4083. doi:10.1073/ <br /> 14. Montoya J, Ojala D, Attardi G (1981) Distinctive features 59-terminal                 pnas.0737556100. <br />     sequences human mitochondrial mRNAs. Nature 290: 465&#226;&#8364;&#8220;470.                         40. Freyer C, Park CB, Ekstrand MI, Shi Y, Khvorostova J, et al. (2010) <br /> 15. Ojala D, Montoya J, Attardi G (1981) tRNA punctuation model RNA                           Maintenance respiratory chain function mouse hearts severely <br />     processing human mitochondria. Nature 290: 470&#226;&#8364;&#8220;474.                                       impaired mtDNA transcription. Nucleic Acids Res 38: 6577&#226;&#8364;&#8220;6588. doi:10.1093/ <br /> 16. Temperley RJ, Wydro M, Lightowlers RN, Chrzanowska-Lightowlers ZM                        <br /> </body></html>