<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Eukaryotic resistance &#239;&#172;&#8218;uoride toxicity mediated by <br /> a widespread family &#239;&#172;&#8218;uoride export proteins <br /> Sanshu Lia,b,1, Kathryn D. Smithc,d,1, Jared H. Davisd, Patricia B. Gordond, Ronald R. Breakera,b,d,2, Scott A. Strobelc,d,2 <br /> Departments aMolecular, Cellular Developmental Biology, cChemistry, dMolecular Biophysics Biochemistry bHoward Hughes Medical <br /> Institute, Yale University, New Haven, CT 06520 <br />  <br /> Edited Jay C. Dunlap, Geisel School Medicine Dartmouth, Hanover, NH, approved October 7, 2013 (received review June 3, 2013) <br />  <br /> Fluorine abundant element toxic organisms              partial cause &#239;&#172;&#8218;uoride toxicity enzyme in- <br /> bacteria humans, mechanisms eukaryotes                  hibition interactions important cations cell, such <br /> resist &#239;&#172;&#8218;uoride toxicity unknown. The Escherichia coli gene crcB         Mg2+ Ca2+ (12, 19, 20). Crystal structures enzyme <br /> was recently shown regulated &#239;&#172;&#8218;uoride-responsive ri-              enolase complex &#239;&#172;&#8218;uoride inhibition due <br /> boswitch, implicating &#239;&#172;&#8218;uoride response. There &gt;8,000 crcB         formation magnesium-&#239;&#172;&#8218;uoride-phosphate complex <br /> homologs domains life, indicating im-          (19). Another well-known example interaction &#239;&#172;&#8218;uoride <br /> portant role biology. Here demonstrate eukaryotic                enzymes ability &#239;&#172;&#8218;uoride complexes aluminum <br /> homologs [renamed FEX (&#239;&#172;&#8218;uoride exporter)] function &#239;&#172;&#8218;uoride               beryllium act phosphate mimics. These complexes af- <br /> export. FEX KOs eukaryotic model organisms, Neurospora             fect activity phosphoryl transfer enzymes, large and <br /> crassa, Saccharomyces cerevisiae, Candida albicans, highly          important group macromolecules (12, 21). <br /> sensitized &#239;&#172;&#8218;uoride (&gt;200-fold) halides. Some            Because ubiquitous presence environment its <br /> these KO strains unable grow &#239;&#172;&#8218;uoride concentrations               toxic effects, organisms evolved mechanisms re- <br /> found tap water. Using radioactive isotope &#239;&#172;&#8218;uoride, <br />                                                                             sistance &#239;&#172;&#8218;uoride. However, little known strat- <br /> 18 <br />   F, developed assay measure intracellular &#239;&#172;&#8218;uoride <br />                                                                             egies pathways. Some new clues emerged discovery <br />                                                                             class regulatory RNAs, riboswitches, bind <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> concentration FEX deletion strains accumulate <br />                                                                             &#239;&#172;&#8218;uoride regulate expression genes response this <br /> &#239;&#172;&#8218;uoride excess external concentration, providing direct <br />                                                                             anion eubacteria archaea (22, 23). Riboswitches me- <br /> evidence FEX function &#239;&#172;&#8218;uoride ef&#239;&#172;&#8218;ux. In addition, are <br />                                                                             tabolite ion-sensing structured RNA motifs typically located <br /> more sensitive lower pH presence &#239;&#172;&#8218;uoride. These <br />                                                                             noncoding regions certain mRNAs. They control the <br /> results demonstrate eukaryotic FEX genes encode previously           expression adjoining protein-coding regions several <br /> unrecognized class &#239;&#172;&#8218;uoride exporter necessary <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span>            different mechanisms including transcription termination, trans- <br /> standard environmental conditions.                                          lation blocking, alternative splicing (24&#226;&#8364;&#8220;27). One the <br />                                                                             genes commonly associated &#239;&#172;&#8218;uoride-sensing ribos- <br />                  | <br /> toxicity resistance environmental toxin   | ion transport |                 witches crcB gene. This gene originally identi&#239;&#172;?ed in <br /> dual membrane topology                                                      Escherichia coli implicated previously chromo- <br />                                                                             condensation camphor resistance (28, 29). It encodes <br />  <br /> H     alides long known important biology. <br />       Chloride iodide studied effects <br /> on organisms (1&#226;&#8364;&#8220;5), understood biological <br />                                                                             small protein (127 amino acids) predicted <br />                                                                             transmembrane domains. There evidence CrcB functions <br />                                                                             dimer dual membrane topology, similar small <br /> importance &#239;&#172;&#8218;uoride, smallest electronegative                multidrug transporters (30). <br /> anion series. The size reactivity &#239;&#172;&#8218;uoride               Due association &#239;&#172;&#8218;uoride-binding riboswitch, it <br /> unique chemical biochemical properties, mecha-                  hypothesized crcB involved &#239;&#172;&#8218;uoride re- <br /> nisms cells respond halide incompletely                sistance, speci&#239;&#172;?cally &#239;&#172;&#8218;uoride ef&#239;&#172;&#8218;ux. An E. coli strain lacking crcB <br />                                                                             200-fold sensitive &#239;&#172;&#8218;uoride WT strain (22). <br /> characterized. <br />                                                                             Additionally, strain appears higher intracellular <br />    Fluoride ubiquitous environment, in <br /> soil, water, air (6, 7), highly abundant element in <br /> the earth&#226;&#8364;&#8482;s crust (0.32 g/kg) (8&#226;&#8364;&#8220;10). The distribution ha-              Signi&#239;&#172;?cance <br /> lide soil water variable depending location. Fluoride <br /> concentrations soil range thousands parts                 Although &#239;&#172;&#8218;uoride plentiful environment com- <br /> per million (ppm) (10). In natural water sources, concen-                   monly used high concentrations oral hygiene products, <br /> trations range &lt;25 &#206;&#188;M &gt;100 mM (&lt;0.5 &gt;2,000 ppm;                      little known biological systems overcome <br /> 1 ppm &#226;&#710;&#188; 55 &#206;&#188;M) depending water contact high                   toxic effects anion. We demonstrate protein <br /> levels &#239;&#172;&#8218;uoride-containing minerals (6, 7, 11). In groundwater                called FEX fungi essential <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>cell survival</span> the <br /> speci&#239;&#172;?cally, &#239;&#172;&#8218;uoride concentrations highest                presence high &#239;&#172;&#8218;uoride concentrations. The protein re- <br /> anion (7, 12).                                                                  quired rapid expulsion cytoplasmic &#239;&#172;&#8218;uoride, in- <br />    At high concentration, &#239;&#172;&#8218;uoride toxic effects bacteria,                dicating eukaryotic species carry FEX genes <br /> fungi, plants, animals. More century ago, &#239;&#172;&#8218;uoride                likely avoid &#239;&#172;&#8218;uoride toxicity purging cellular &#239;&#172;&#8218;uoride. <br /> used antimicrobial agent (6). The antimicrobial effects of <br /> this ion continued elucidated decades,             Author contributions: S.L., K.D.S., J.H.D., R.R.B., S.A.S. designed research; S.L., K.D.S., <br /> with emphasis bacteria cause dental caries (13&#226;&#8364;&#8220;              J.H.D., P.B.G. performed research; S.L., K.D.S., J.H.D. analyzed data; S.L., K.D.S., <br /> 15). Fluoride toxic eukaryotic organisms. For example,           R.R.B., S.A.S. wrote paper. <br />  <br /> at beginning 20th century, noted baker&#226;&#8364;&#8482;s             The authors declare con&#239;&#172;&#8218;ict interest. <br /> yeast, Saccharomyces cerevisiae, killed 1% sodium &#239;&#172;&#8218;uoride            This article PNAS Direct Submission. <br /> (about 250 mM) (16). Inhibitory growth effects               1 <br />                                                                              S.L. K.D.S. contributed equally work. <br /> observed species fungi, including pathogens            2 <br />                                                                              To correspondence addressed. E-mail: ronald.breaker@yale.edu scott. <br /> (17, 18).                                                                    strobel@yale.edu. <br />    The mechanisms &#239;&#172;&#8218;uoride toxic different              This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> species complex incompletely understood. It thought              1073/pnas.1310439110/-/DCSupplemental. <br />  <br />  <br />  <br /> 19018&#226;&#8364;&#8220;19023 | PNAS | November 19, 2013 | vol. 110 | no. 47                                                         www.pnas.org/cgi/doi/10.1073/pnas.1310439110 <br />                                                                                   distinct &#239;&#172;?lamentous fungi. <br />                                                                                  Examples present agriculturally important plants such <br />                                                                                  rice, corn, grapes, model plant species <br />                                                                                  Arabidopsis thaliana. Additionally, FEX genes present in <br />                                                                                  animal species, Oikopleura dioica (a tunicate) and <br />                                                                                  Ciona intestinalis (a sea squirt). The widespread distribution of <br />                                                                                  protein suggests important biological function FEX. <br />  <br />                                                                                  Deletion FEX Causes Fluoride Sensitivity Eukaryotes. To test the <br />                                                                                  role eukaryotic FEX proteins &#239;&#172;&#8218;uoride resistance, se- <br />                                                                                  lected important model organisms sample <br />                                                                                  phylogenetic diversity easily genetically manipulated. <br />                                                                                  We deleted FEX gene(s) organisms, N. crassa, <br />                                                                                  S. cerevisiae, C. albicans, tested <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>knockout</span> (KO) <br />                                                                                  strain growth &#239;&#172;&#8218;uoride containing media. <br />                                                                                     N. crassa haploid organism contains FEX gene <br />                                                                                  (NCU06262, chromosome III; fex-1) encoding 526 amino acid <br />                                                                                  (aa) protein conserved FEX domains. This gene was <br />                                                                                  deleted, growth assessed liquid solid media <br />                                                                                  increasing amounts &#239;&#172;&#8218;uoride. When strains were <br /> Fig. 1. FEX proteins architecture distribution. (A) Eubacterial, archaeal, <br />                                                                                  cultured solid agar media &#239;&#172;&#8218;uoride, KO strain had <br /> and eukaryotic FEX domain arrangments. TM represents transmembrane <br /> domain. (B) Phylogenetic tree eukaryotic FEX proteins. <br />                                                                                  similar growth rate compared WT (Table S1). However, <br />                                                                                  solid liquid media containing &#239;&#172;&#8218;uoride, growth of <br />                                                                                  KO organism inhibited &#226;&#710;&#188;200-fold lower concen- <br /> concentration &#239;&#172;&#8218;uoride WT, growth strain                  tration WT (Fig. 2 Fig. S1). WT N. crassa experienced <br /> can rescued high &#239;&#172;&#8218;uoride media addition                      reduction number spores, myce- <br /> eriCF gene (22), variant eriC encodes F&#226;&#710;&#8217;/H+ antiporter               lium &#239;&#172;&#8218;uoride concentration increased (Fig. S1).When <br /> (31). Because eriCF substitute crcB, likely             copy fex-1 gene reintroduced genome the <br /> proteins level functional equivalence,                   his-3 locus, resulting rescue strain fully recovered ability <br /> transportation &#239;&#172;&#8218;uoride ions.                                                  tolerate high &#239;&#172;&#8218;uoride concentrations (Fig. 2, Figs. S1 S2, <br />    Although &#239;&#172;&#8218;uoride riboswitches identi&#239;&#172;?ed                       Table S1). <br /> eukaryotes, organisms crcB homologs. There                       S. cerevisiae contains ORFs, YOR390W (FEX1) and <br /> are 8,000 genes UniProt database annotated                   YPL279C (FEX2), encodes protein 375 aa. Both of <br />  <br />  <br />  <br />  <br />                                                                                                                                                                    GENETICS <br /> containing crcB domain (32). Although majority                  proteins composed FEX domains connected by <br /> of identi&#239;&#172;?ed sequences prokaryotic organisms,                   linker. These proteins, sequences upstream and <br /> a signi&#239;&#172;?cant number eukaryotic (&gt;200; Fig. 1) archaeal                    downstream, nearly identical, suggesting gene duplication <br /> (&gt;200) species well. Many homologs contain                    event. Both genes previously uncharacterized not <br /> four transmembrane domains classi&#239;&#172;?ed transporter                    included systematic yeast deletion library were <br /> superfamily Pfam (30, 33).                                                    implicated yeast &#239;&#172;&#8218;uoride biology, despite <br />    To determine eukaryotic homologs crcB                      large-scale screen mutants growth defects mil- <br /> involved &#239;&#172;&#8218;uoride homeostasis, used genetic approaches                   limolar sodium &#239;&#172;&#8218;uoride (34). <br /> three organisms: &#239;&#172;?lamentous fungus Neurospora crassa,                       To test crcB homologs involved &#239;&#172;&#8218;uoride <br /> budding yeast S. cerevisiae, pathogen Candida albicans.                  resistance yeast, single deletion strains as <br /> Here demonstrate CrcB proteins play role &#239;&#172;&#8218;uoride                    strain ORFs deleted. The single deletion of <br /> resistance eukaryotes provide direct evidence                FEX1 FEX2 does signi&#239;&#172;?cantly increase &#239;&#172;&#8218;uoride <br /> are involved removing &#239;&#172;&#8218;uoride cell. This                 sensitivity, suggesting redundancy function two <br /> known example gene identi&#239;&#172;?ed function &#239;&#172;&#8218;uoride ef&#239;&#172;&#8218;ux                   proteins. Single KOs grow slightly worse WT in <br /> in eukaryotes. Based results, propose renaming                       media containing millimolar quantities &#239;&#172;&#8218;uoride (Fig. S3). For <br /> these genes FEX, &#239;&#172;&#8218;uoride exporters, refer                   WT strain, IC50 &#239;&#172;&#8218;uoride 70 mM minimum <br /> using nomenclature. <br /> Results <br /> Conserved Domains Phylogenetic Tree Eukaryotic FEX <br /> Proteins. Examination FEX homologs eubacteria, archaea, <br /> and eukaryotes reveals clear similarities differences do- <br /> main organization. Most archaeal bacterial genes <br /> encode proteins containing single conserved FEX domain, <br /> whereas eukaryotic homologs encode conserved FEX <br /> domains single polypeptide (Fig. 1A). There evidence that <br /> the CrcB protein E. coli forms dimer, suggesting two <br /> CrcB domains needed biological function (30). If this <br /> is case, prokaryotic FEX proteins predicted to <br /> form dimers, eukaryotic homologs pre- <br /> dicted function monomers, necessary FEX <br /> domains present polypeptide. <br />    Eukaryotic FEX proteins segregate distinct groups <br /> based phylogeny wide variety organisms,                 Fig. 2. Sensitivity N. crassa fex-1 KO strain NaF race tube assay. <br /> including fungi, algae, molds, protozoans, plants, animals                   All strains cultured solid agar media different concentrations of <br /> (Fig. 1B). Fungal FEX genes comprise largest groups                      NaF. (Inset) Curve KO strain. The results independent <br /> with homologs yeast species closely related                     repeats. Gray bars represent SDs. <br />  <br />  <br /> Li et al.                                                                                          PNAS | November 19, 2013 | vol. 110 | no. 47 | 19019 <br />  inhibitory concentration (MIC) &#226;&#710;&#188;120 mM. In case the <br /> single deletion mutants, IC50 &#239;&#172;&#8218;uoride &#226;&#710;&#188;55&#226;&#8364;&#8220;60 mM and <br /> the MIC 110 mM (Table S1). <br />    In contrast single KOs, deletion FEX genes <br /> results greater 1,000-fold increase &#239;&#172;&#8218;uoride sensitivity <br /> (Fig. 3 Table S1). The growth double KO signi&#239;&#172;?- <br /> cantly affected micromolar concentrations &#239;&#172;&#8218;uoride both <br /> solid liquid culture. For double deletion strain, IC50 <br /> for &#239;&#172;&#8218;uoride &#226;&#710;&#188;60 &#206;&#188;M MIC 120 &#206;&#188;M. To determine if <br /> this <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> unique &#239;&#172;&#8218;uoride, tested WT and <br /> double KO strains increasing concentrations other <br />                                                                                  Fig. 4. Sensitivity C. albicans FEX1 KO strain NaF. Growth curves of <br /> sodium halide salts NaCl, NaBr, NaI. We observed no <br />                                                                                  (A) WT (B) homozygous KO. <br /> change sensitivity halides concentrations that <br /> completely inhibited growth (Fig. S4). <br />    To determine large increase &#239;&#172;&#8218;uoride sensitivity <br /> observed double KO solely deletion                    Given deletion FEX organisms produced <br /> FEX proteins, sought restore &#239;&#172;&#8218;uoride resistance sup-                    similar &#239;&#172;&#8218;uoride-sensitive phenotype, performed rest of <br /> plying FEX1 extrachromosomally. A centromeric plasmid (main-                     analyses yeasts. These unicellular organisms <br /> tained copy cell) containing FEX1 transformed                amenable analysis &#239;&#172;?lamentous species. <br /> the double KO strain assayed growth the <br />                                                                                  Intracellular Fluoride Concentration Increases Organisms Without <br /> presence increasing amounts &#239;&#172;&#8218;uoride. The transformed <br /> strain behaves similarly single deletion mutant IC50              FEX. If FEX genes encode proteins remove &#239;&#172;&#8218;uoride <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> &gt;900-fold greater double KO (Table S1). This obser-                     cells, FEX KO cells higher levels &#239;&#172;&#8218;uoride the <br /> vation establishes change &#239;&#172;&#8218;uoride tolerance observed                 WT strain. To quantitatively determine &#239;&#172;&#8218;uoride in <br /> in double KO entirely deletion FEX genes.               WT KO organisms, measured intracellular &#239;&#172;&#8218;uoride <br />    The pathogenic yeast C. albicans diploid organism               concentration using radioactive 18F (35, 36). <br /> one FEX gene (CaO19.7095, chromosome 7; FEX1) encoding                              We &#239;&#172;?rst applied method measure concentration of <br /> a 390 aa protein conserved FEX domains. We gener-                       &#239;&#172;&#8218;uoride WT S. cerevisiae. Yeast cells incubated with <br /> ated strain allele deleted (referred                 mixture nonradioactive &#239;&#172;&#8218;uoride trace concentration of <br /> heterozygous KO) alleles deleted <br />                                                                                  18 <br />                                                                                     F. Any &#239;&#172;&#8218;uoride taken cells washed away, and <br /> (referred homozygous KO). The &#239;&#172;&#8218;uoride sensitivity                   counts minute (cpm) measured cell sus- <br /> all strains tested liquid media growth assays.                            pension. We additionally determined volume WT and <br />    Similarly S. cerevisiae, heterozygous KO does                 double KO cells, allowing calculate concentration of <br /> a signi&#239;&#172;?cant effect &#239;&#172;&#8218;uoride sensitivity, growth                &#239;&#172;&#8218;uoride inside yeast cells. The measured intracellular volume <br /> the homozygous KO compromised concentrations &#239;&#172;&#8218;uo-                       38 &#194;&#177; 5 fL (mean &#194;&#177; SD) matches cell volumes reported pre- <br /> ride orders magnitude lower WT (Fig. 4                   viously (37). <br /> Fig. S5). The WT strain IC50 &#239;&#172;&#8218;uoride 140 mM                       WT S. cerevisiae able maintain low intracellular &#239;&#172;&#8218;uoride <br /> an MIC 175 mM (Table S1). The heterozygous KO in-                         concentration range external &#239;&#172;&#8218;uoride concentrations. <br /> hibition values similar WT, homozygous KO                    When incubated 50 &#206;&#188;M external &#239;&#172;&#8218;uoride, intracellular <br /> has IC50 100 &#206;&#188;M MIC 0.5 mM, &gt;300-fold                       concentration remains 10 &#206;&#188;M (Fig. 5 A B). There is <br /> lower. Neither C. albicans KO strain shows increased sensi-                   essentially change intracellular &#239;&#172;&#8218;uoride concentration for <br /> tivity Cl&#226;&#710;&#8217; compared WT (Fig. S6). A rescue strain                    4 h, upper time limit assay the <br /> constructed integrating WT copy FEX1 ribo-                      short half-life 18F isotope (109 min). When external <br /> somal protein S10 (RPS10) locus. The reintroduction FEX1                   &#239;&#172;&#8218;uoride concentration increased 1 mM, internal <br /> this alternate locus restores ability resist high concentra-              &#239;&#172;&#8218;uoride concentration remains &#226;&#710;&#188;10-fold external con- <br /> tion NaF similarly WT, strain grew                      centration (Fig. 5C). This result implies millimolar <br /> slowly WT unknown reasons (Fig. S5).                                    amounts &#239;&#172;&#8218;uoride, WT S. cerevisiae able maintain <br />                                                                                  low intracellular &#239;&#172;&#8218;uoride concentration. Consistent the <br />                                                                                  results growth assays, single KO cells levels &#239;&#172;&#8218;uoride <br />                                                                                  approximately equal WT (Fig. 5A). <br />                                                                                     In contrast, intracellular &#239;&#172;&#8218;uoride concentration double <br />                                                                                  KO S. cerevisiae cells rapidly equilibrates exceeds the <br />                                                                                  external &#239;&#172;&#8218;uoride concentration. After 30 min incubation, <br />                                                                                  double KO strain contains &gt;10-fold &#239;&#172;&#8218;uoride the <br />                                                                                  WT strain conditions (Fig. 5A). After 2 h, the <br />                                                                                  internal concentration rises 5-fold higher external <br />                                                                                  concentration &#226;&#710;&#188;40-fold higher WT strain (Fig. 5A). <br />                                                                                  When cells incubated 50 &#206;&#188;M NaF, observe nearly <br />                                                                                  linear increase &#239;&#172;&#8218;uoride inside double KO <br />                                                                                  cells time (Fig. 5B). The intracellular concentration also <br />                                                                                  increases external concentration &#239;&#172;&#8218;uoride increased. <br />                                                                                  After 2 h presence varying concentrations &#239;&#172;&#8218;uoride, <br />                                                                                  &#239;&#172;&#8218;uoride inside double KO cells two- <br />                                                                                  &#239;&#172;?vefold higher external concentration (Fig. 5C). <br /> Fig. 3. Fluoride sensitivity S. cerevisiae double KO strain. (A) Serial      When FEX1 supplied double KO cells plasmid, low <br /> dilution assay strains grown YPD + NaF concentrations stated.       intracellular levels &#239;&#172;&#8218;uoride restored (Fig. 5A). After 30 <br /> All plates grown 30 &#194;&#176;C imaged 3 d growth. (B) Liquid        min incubation 50 &#206;&#188;M &#239;&#172;&#8218;uoride, rescue strain an <br /> growth assay double KO strain grown YPD + NaF concen-           intracellular &#239;&#172;&#8218;uoride concentration three- fourfold of <br /> trations stated. Cultures grown 30 &#194;&#176;C. (C) Representative IC50 curves    WT cells. Even 2 h incubation, concentration of <br /> for wild-type S. cerevisiae double KO strain. AUC area      &#239;&#172;&#8218;uoride inside cells signi&#239;&#172;?cantly lower double <br /> growth curve.                                                                    KO, despite small increase compared WT. This slight <br />  <br /> 19020 | www.pnas.org/cgi/doi/10.1073/pnas.1310439110                                                                                                   Li et al. <br />                                                                                   incubation required cell death concentration (Fig. <br />                                                                                  6A). Although 100 mM slightly MIC WT <br />                                                                                  strain, growth observed assay liquid <br />                                                                                  culture growth assays described above, likely dif- <br />                                                                                  ferences conditions experiments (amount <br />                                                                                  oxygen, shaking speeds, etc.). In contrast WT strain, <br />                                                                                  double KO cells incubated 100 mM &#239;&#172;&#8218;uoride, the <br />                                                                                  number colonies decreases sharply 4&#226;&#8364;&#8220;8 h, cells are <br />                                                                                  dead 24 h exposure (Fig. 6B). When double KO cells are <br />                                                                                  incubated 0.5 mM &#239;&#172;&#8218;uoride, observe decrease the <br />                                                                                  number colonies 2.5 h. However, 72 h, few <br />                                                                                  colonies able grow. Curiously, 10 mM &#239;&#172;&#8218;uoride, the <br />                                                                                  number colonies does substantially decrease &#226;&#710;&#188;12&#226;&#8364;&#8220;24 h <br />                                                                                  exposure, cells alive 72 h. <br />                                                                                     For WT C. albicans, 300 mM &#239;&#172;&#8218;uoride required kill the <br />                                                                                  yeast 24 h (Fig. 6C); 200 mM &#239;&#172;&#8218;uoride (close MIC) <br />                                                                                  inhibits growth &#239;&#172;?rst 12 h cells continue to <br />                                                                                  multiply that. The result similar observed for <br />                                                                                  S. cerevisiae, expect differences how <br />                                                                                  cells cultured solid agar plate liquid me- <br />                                                                                  dium assays. In case homozygous KO, 100 mM &#239;&#172;&#8218;uo- <br />                                                                                  ride required kill cells 24 h exposure (Fig. 6D). <br />                                                                                  Similarly S. cerevisiae lower &#239;&#172;&#8218;uoride concentrations, as <br />                                                                                  5 mM here, C. albicans homozygous KO inhibited in <br />                                                                                  growth survive 120 h. <br />                                                                                     These data suggest inhibitory concentrations <br /> Fig. 5. Intracellular &#239;&#172;&#8218;uoride concentrations S. cerevisiae C. albicans. <br />                                                                                  &#239;&#172;&#8218;uoride dramatically different WT cells and <br /> Values graphed average measurements &#194;&#177; SD. (A) <br /> Intracellular concentration &#239;&#172;&#8218;uoride S. cerevisiae strains in- <br />                                                                                  FEX deleted, concentrations that <br /> cubation 50 &#206;&#188;M NaF. (B) Intracellular &#239;&#172;&#8218;uoride concentration func- <br />                                                                                  cause cell death 24 h similar. Lower concentrations of <br /> tion time WT double KO S. cerevisiae. Cells incubated       &#239;&#172;&#8218;uoride kill cells lacking functional FEX, longer <br /> 50 &#206;&#188;M NaF. (C) Intracellular &#239;&#172;&#8218;uoride concentration function external     exposure times needed. <br /> &#239;&#172;&#8218;uoride concentration WT double KO S. cerevisiae. All samples were <br /> incubated 2 h scintillation counting. (D) Intracellular &#239;&#172;&#8218;uoride       Effect pH Growth C. albicans Fluoride. The cellular <br /> concentration C. albicans strains. Cells incubated 500 &#206;&#188;M           permeability HF &gt;1,000,000-fold greater F&#226;&#710;&#8217; (39) due <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  GENETICS <br /> &#239;&#172;&#8218;uoride 2 h.                                                                 fact HF neutral diffuse through <br />                                                                                  lipid bilayer. In contrast, F&#226;&#710;&#8217; repelled large Born <br />                                                                                  charging energy results low dielectric environment <br /> increase likely fact cells grown             membrane small radius F&#226;&#710;&#8217; anion (40). The <br /> in nonselective media assay, plasmid known                pKa HF 3.2 (41), lower extracellular pH results in <br /> to lost low percentage generation grown                 higher ratio HF F&#226;&#710;&#8217;, greater of <br /> selection (38). It possible differences expression            &#239;&#172;&#8218;uoride able enter cells (6, 42). Under low pH conditions, <br /> FEX1 plasmid chromosomal locus exist                         cells lacking FEX predicted accumulate more <br /> and contribute discrepancies observed. <br />    The trends observed C. albicans, ho- <br /> mozygous KO accumulates higher &#239;&#172;&#8218;uoride cell than <br /> the WT heterozygous KO strains (Fig. 5D). Accurate con- <br /> centrations determined dif&#239;&#172;?culty measuring <br /> the internal volume C. albicans. However, observed that <br /> the homozygous KO cells accumulate &#226;&#710;&#188;10-fold &#239;&#172;&#8218;uoride <br /> than WT cells incubated 0.5 mM &#239;&#172;&#8218;uoride. <br /> When FEX1 reintroduced different chromosomal lo- <br /> cus, intracellular concentration &#239;&#172;&#8218;uoride equivalent to <br /> WT (Fig. 5D). These results suggest FEX proteins play an <br /> important role removing &#239;&#172;&#8218;uoride cell establish <br /> that FEX necessary suf&#239;&#172;?cient lower internal &#239;&#172;&#8218;uoride <br /> concentration (Fig. 5). <br />  <br /> Cytotoxic Effects Fluoride FEX KO Organisms. Deleting FEX in <br /> eukaryotic organisms causes increase sensitivity to <br /> &#239;&#172;&#8218;uoride intracellular &#239;&#172;&#8218;uoride concentration. We de- <br /> termined increase intracellular &#239;&#172;&#8218;uoride FEX <br /> KO yeast results cell death lower external &#239;&#172;&#8218;uoride con- <br /> centration shorter exposure time. To assess time- and <br /> concentration-dependent killing cells &#239;&#172;&#8218;uoride, overnight <br /> growths WT double KO cells diluted media <br /> containing &#239;&#172;&#8218;uoride. Periodically, small volume culture <br /> was removed, cells washed plated. The number <br /> of colonies counted days growth.                            Fig. 6. Effects NaF growth survival S. cerevisiae C. <br />    For WT S. cerevisiae, 200 mM &#239;&#172;&#8218;uoride required kill                  albicans. Data measured triplicate data set shown as <br /> total cell population 24 h exposure. Some growth                    example. Colonies formed (A) WT S. cerevisiae, (B) S. cerevisiae double <br /> observed 100 mM &#239;&#172;&#8218;uoride, 48&#226;&#8364;&#8220;72 h                         KO, (C) WT C. albicans, (D) C. albicans homozygous KO cultures. <br />  <br />  <br /> Li et al.                                                                                          PNAS | November 19, 2013 | vol. 110 | no. 47 | 19021 <br />  &#239;&#172;&#8218;uoride experience greater toxic effects KO            orders magnitude, meaning experienced <br /> cells higher pH. To test hypothesis, measured             growth inhibition low mid-micromolar &#239;&#172;&#8218;uoride. These data <br /> growth different C. albicans strains function pH,        indicate organisms able grow the <br /> in absence presence NaF. As expected, growth           absence FEX environmental conditions. In fact, the <br /> the WT, heterozygous KO, rescue strains C. albicans            S. cerevisiae FEX double KO shows growth inhibition level <br /> unaffected pH (Fig. S7). However, absence              &#239;&#172;&#8218;uoride supplementation municipal water <br /> added NaF, growth homozygous KO cells inhibited be-            supplies (&#226;&#710;&#188;50 &#206;&#188;M) (7). Even low micromolar amounts, <br /> low pH 5. We believe effect presence          data using 18F reporter intracellular &#239;&#172;&#8218;uoride con- <br /> residual F&#226;&#710;&#8217; media [according measurement, YPD              centration indicate time internal &#239;&#172;&#8218;uoride concen- <br /> (yeast extract, peptone, dextrose) contains 7.3 &#194;&#177; 0.5 &#206;&#188;M &#239;&#172;&#8218;uoride].       tration reach higher levels &#239;&#172;&#8218;uoride accumulates. <br /> When known &#239;&#172;&#8218;uoride (100 &#206;&#188;M) added                     Lower pH aggravates &#239;&#172;&#8218;uoride stress, presumably facilitating <br /> media, growth homozygous KO inhibited pH 6.5            in&#239;&#172;&#8218;ux &#239;&#172;&#8218;uoride cells conjugate acid HF. This fact <br /> or lower, indicating cells sensitive pH        explain bacteria fungi contain FEX genes. <br /> the presence F&#226;&#710;&#8217;.                                                      In situations, external pH lower intracellular <br />                                                                          pH, promote accumulation &#239;&#172;&#8218;uoride the <br /> Discussion                                                               cell. For instance, tap, distilled, deionized water pH <br /> FEX genes widespread domains life,               &#226;&#710;&#188;5. For environmental water sources pH also <br /> suggesting proteins important function             near 5, dif&#239;&#172;?cult cells survive genes <br /> diverse organisms. Deleting crcB E. coli makes cells         encoding &#239;&#172;&#8218;uoride export proteins. <br /> sensitive &#239;&#172;&#8218;uoride, likely accumulation &#239;&#172;&#8218;uoride          To date, homologs FEX identi&#239;&#172;?ed many <br /> the cell indicated reporter assay (22). Here showed           higher eukaryotes, including humans mammals. <br /> that deleting FEX genes model eukaryotic organisms              However, toxic effects &#239;&#172;&#8218;uoride species been <br /> produces &#239;&#172;&#8218;uoride sensitive phenotype, KO cells              documented. These reports data indicate that <br /> 200&#226;&#8364;&#8220;1,000 times sensitive &#239;&#172;&#8218;uoride corre-              organisms likely evolved mechanisms combat <br /> sponding WT strain. These data demonstrate importance             &#239;&#172;&#8218;uoride toxicity. Although proteins involved &#239;&#172;&#8218;uoride re- <br /> FEX resistance &#239;&#172;&#8218;uoride toxicity. We provided direct           sponse mammals largely unknown, fact FEX does <br /> evidence cells lacking FEX higher intracellular &#239;&#172;&#8218;uoride        appear involved exploited therapeutic <br /> concentrations relative WT counterparts, directly im-           purposes. Because humans animals the <br /> plicating FEX &#239;&#172;&#8218;uoride ef&#239;&#172;&#8218;ux.                                          FEX protein, pathogens parasites do, drug <br />    The FEX protein product predicted multipass                targeting FEX used combination &#239;&#172;&#8218;uoride be <br /> transmembrane protein homology variety membrane             promising novel therapeutic (49, 50). For example, an <br /> proteins responsible transporting different substrates. Com-         antifungal reagent (Amphotericin) combined small <br /> bined data, suggests FEX gene encodes             NaF, ef&#239;&#172;?cacy inhibition greatly im- <br /> a &#239;&#172;&#8218;uoride channel transporter, rule            proved (49). If inhibitors targeting FEX developed, such <br /> the possibility FEX functions required accessory ele-          combination predicted broad activity against <br /> ment protein itself exports &#239;&#172;&#8218;uoride. However,            diverse pathogens parasites relatively nontoxic <br /> a recent report shows conclusively bacterial FEX homologs           human animal cells contain FEX protein. <br /> are highly selective &#239;&#172;&#8218;uoride channels discriminate against <br /> Cl&#226;&#710;&#8217; factor &gt;10,000-fold (43). These channels recon-              Materials Methods <br /> stituted lipid bilayers exist primarily open state          Strains Growth Assays. Strains used study listed Table S2, <br /> export F&#226;&#710;&#8217; passive electrodiffusion using negative             plasmids used listed Table S3, primers used listed Table S4. <br /> membrane potential. Because large degree homology              FEX deletions N. crassa, S. cerevisiae, C. albicans using <br /> that exists bacterial fungal FEX genes,                  previously published methods. For detailed description, SI Materials <br /> eukaryotic FEX proteins likely function selective         Methods. N. crassa growth assays performed liquid and <br /> F&#226;&#710;&#8217; channels. Furthermore, resting membrane voltage                solid (race tubes) minimal media NaF. S. cerevisiae growth <br /> fungi normally highly negative (about &#226;&#710;&#8217;200 mV) (44, 45),              assays performed solid agar plates liquid culture using <br /> which provide suf&#239;&#172;?cient thermodynamic gradient                standard procedures. C. albicans growth assays liquid culture <br /> expel F&#226;&#710;&#8217; mechanism. The results eubac-             using standard procedures. For detailed description, SI Materials <br /> teria eukaryotes indicate FEX proteins conserved         Methods. <br /> function maintaining low levels intracellular &#239;&#172;&#8218;uoride across <br /> multiple domains life proteins promote             18 <br />                                                                            F Assays. Measurements internal &#239;&#172;&#8218;uoride concentrations S. cerevisiae <br /> survival cells &#239;&#172;&#8218;uoride stress.                                  using yeast grown &#226;&#710;&#188;1 OD600; 675 &#206;&#188;L cell suspension was <br />    The conservation FEX proteins species implies          mixed 75 &#206;&#188;L 10&#195;&#8212; NaF solution containing mixture 18F non- <br /> that large number organisms encounter &#239;&#172;&#8218;uoride stress            radioactive NaF. A variable 18F added 10&#195;&#8212; solutions to <br /> require ability remove anion. Environmental &#239;&#172;&#8218;uoride          reduce range speci&#239;&#172;?c activities samples: 0.013 &#206;&#188;Ci/&#206;&#188;L the <br /> concentrations vary low micromolar mid millimolar,               10 50 &#206;&#188;M samples, 0.026 &#206;&#188;Ci/&#206;&#188;L 250 &#206;&#188;M samples, 0.04 &#206;&#188;Ci/&#206;&#188;L for <br /> suggesting survive, organisms tolerant          500 &#206;&#188;M samples, 0.053 &#206;&#188;Ci/&#206;&#188;L 750 &#206;&#188;M samples, 0.067 &#206;&#188;Ci/&#206;&#188;L for <br />                                                                          1,000 &#206;&#188;M samples. After addition NaF, cells incubated at <br /> levels (6, 46). It possible different species resistant <br />                                                                          room temperature variable time. The media removed, cells <br /> to different concentrations ion. Indeed, observe dif- <br />                                                                          washed times vacuum &#239;&#172;?ltration. Cells resuspended 200 &#206;&#188;L <br /> ferences MIC different eukaryotic species         YPD, radiation intensity measured scintillation counting using <br /> have studied here, C. albicans tolerant           18F measurement protocol. Each sample counted 1 min, the <br /> &#239;&#172;&#8218;uoride N. crassa least. A previous study reports          decay 18F counting accounted using half-life 109 <br /> the MIC C. albicans higher, suggesting dif-          min. One microliters 1:1,000 dilution 10&#195;&#8212; stock was <br /> ferent strains species vary level resistance      counted alongside samples determine speci&#239;&#172;?c activity 10&#195;&#8212; <br /> (18). Wide variation observed &#239;&#172;&#8218;uoride tolerance             mixture moles count. This value used determine moles of <br /> higher eukaryotes. For instance, model organism Caeno-               NaF cells multiplying counts cell sample moles per <br /> rhabditis elegans experiences toxic effects low submillimolar      count value obtained. Samples 18F prepared identical <br /> concentrations (47), plants (such cotton             fashion used determine OD600 cells incubation and <br /> plant) tolerate 4,000 ppm (about 220 mM) NaF (48).             washing. The OD600 samples used determine volume the <br /> Relative WT strains, FEX deletion mutants         cells sample described below. The concentration NaF cells <br /> we study exhibited increased sensitivity &#239;&#172;&#8218;uoride         determined dividing moles NaF cell volume. <br />  <br />  <br /> 19022 | www.pnas.org/cgi/doi/10.1073/pnas.1310439110                                                                                            Li et al. <br />     The cell volumes determined using cell displacement method                     fungus, volume cells speci&#239;&#172;?cally determined. Because of <br /> inulin previously described (35). Thirty mL &#226;&#710;&#188;20 OD600 cells pelleted                this, &#239;&#172;&#8218;uoride C. albicans samples reported (pico- <br /> and resuspended 200 &#206;&#188;L water 10,000 counts/&#206;&#188;L 14C puri&#239;&#172;?ed inulin. The                  moles F&#226;&#710;&#8217;)/[OD600(sample) &#195;&#8212; mL (sample)]. <br /> volume resuspension determined, OD600 cells was <br /> measured. The increase volume 200 &#206;&#188;L cellular                      Fluoride Toxicity Assays. A low cell density (about 200 cells) strain was <br /> volume extra cellular volume. To determine extracellular volume, <br />                                                                                                incubated different concentrations &#239;&#172;&#8218;uoride test tubes total <br /> the cells &#239;&#172;?ltered 0.45-&#206;&#188;m &#239;&#172;?lter centrifugation. The radiation <br />                                                                                                volume 2 mL YPD shaking 30 &#194;&#176;C. At different time points, 100 &#206;&#188;L this <br /> intensity &#239;&#172;?ltrate cells measured scintillation counting. <br />                                                                                                cell culture removed spread YPD agar plates. For experiments <br /> The radiation intensity &#239;&#172;?ltrate speci&#239;&#172;?c activity extracel- <br /> lular liquid used determine extracellular volume                       double KO strains, 100 &#206;&#188;L culture centrifuged and <br /> radiation intensity cells. The cell volume determined sub-                       washed twice 1 mL YPD spread agar plates. <br /> tracting extracellular volume total volume increase                         Colonies counted 24&#226;&#8364;&#8220;48 h incubation 30 &#194;&#176;C. <br /> 200 &#206;&#188;L. The OD600 volume cells used determine cell <br /> volume OD600 &#195;&#8212; mL. This analysis times S. cerevisiae                     ACKNOWLEDGMENTS. We thank M. Hochstrasser (Yale University) the <br /> to determine value 0.00038 &#206;&#188;L (cell volume)/&#206;&#188;L (sample volume) &#195;&#8212;                          generous gift BY4741 BY4742 strains resistance cassette <br /> OD600 (sample). This value OD600 volume NaF-incubated                       plasmids; S. M. Noble (University California, San Francisco) C. albicans <br /> cells used determine internal volume cells.                                 SN152 strain plasmids pSN40, pSN52, pSN69; Fungal Genetics Stock <br />                                                                                                Center (Kansas City, MO) Neurospora strains FGSC#4200 FGSC#9717 and <br />    The internal NaF concentrations C. albicans determined by <br />                                                                                                plasmid pCSN44; K. Ruff cloning pRS416-FEX1; D. Spakowicz help with <br /> growing cells density 2 OD600; 10&#195;&#8212; NaF solutions added                     phylogenetic tree; R. Tomko Jr., J. W. Nelson, N. Sudarsan, D. Hiller <br /> cells similar speci&#239;&#172;?c activities S. cerevisiae samples. The picomoles             helpful advice. This work supported National Institutes Health <br /> NaF obtained using method S. cerevisiae                      Grants DE022340 (to R.B.) GM022778 (to S.A.S.). Research R.B. labo- <br /> samples. Because challenge using 14C inulin pathogenic                        ratory supported Howard Hughes Medical Institute. <br />  <br />  <br />  1. Jentsch TJ, Stein V, Weinreich F, Zdebik AA (2002) Molecular structure physio-         26. Breaker RR (2011) Prospects riboswitch discovery analysis. Mol Cell 43(6): <br />     logical function chloride channels. Physiol Rev 82(2):503&#226;&#8364;&#8220;568.                              867&#226;&#8364;&#8220;879. <br />  2. Duran C, Thompson CH, Xiao Q, Hartzell HC (2010) Chloride channels: Often enig-            27. Breaker RR (2012) Riboswitches RNA world. Cold Spring Harb Perspect Biol <br />     matic, rarely predictable. Annu Rev Physiol 72:95&#226;&#8364;&#8220;121.                                         4(2):a003566. <br />  3. Edwards JC, Kahl CR (2010) Chloride channels intracellular membranes. FEBS Lett         28. Hu KH, et al. (1996) Overproduction genes leads camphor resistance and <br />     584(10):2102&#226;&#8364;&#8220;2111.                                                                             chromosome condensation Escherichia coli. Genetics 143(4):1521&#226;&#8364;&#8220;1532. <br />  4. Zimmermann MB, Jooste PL, Pandav CS (2008) Iodine-de&#239;&#172;?ciency disorders. Lancet              29. Sand O, Gingras M, Beck N, Hall C, Trun N (2003) Phenotypic characterization of <br />     372(9645):1251&#226;&#8364;&#8220;1262.                                                                           overexpression deletion Escherichia coli crcA, cspE crcB genes. Micro- <br />  5. Zimmermann MB (2011) The role iodine human growth development. Semin                 biology 149(8):2107&#226;&#8364;&#8220;2117. <br />     Cell Dev Biol 22(6):645&#226;&#8364;&#8220;652.                                                               30. Rapp M, Granseth E, Sepp&#195;&#164;l&#195;&#164; S, von Heijne G (2006) Identi&#239;&#172;?cation evolution of <br />  6. Weinstein LH, Davison AW (2004) Fluorides Environment: Effects Plants            dual-topology membrane proteins. Nat Struct Mol Biol 13(2):112&#226;&#8364;&#8220;116. <br />     Animals (CABI, Cambridge, MA).                                                             31. Stockbridge RB, et al. (2012) Fluoride resistance transport riboswitch-con- <br />  7. Jagtap S, Yenkie MK, Labhsetwar N, Rayalu S (2012) Fluoride drinking water              trolled CLC antiporters. Proc Natl Acad Sci USA 109(38):15289&#226;&#8364;&#8220;15294. <br />                                                                                                32. UniProt Consortium (2012) Reorganizing protein space Universal Protein <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                               GENETICS <br />     de&#239;&#172;&#8218;uoridation water. Chem Rev 112(4):2454&#226;&#8364;&#8220;2466. <br />  8. WHO (1970) Fluorides human health. Switzerland Monograph Series No. 59                     Resource (UniProt). Nucleic Acids Res 40(Database issue):D71&#226;&#8364;&#8220;D75. <br />     (World Health Organization, Geneva).                                                       33. Punta M, et al. (2012) The Pfam protein families database. Nucleic Acids Res 40(Da- <br />  9. WHO (1984) Fluorine Fluorides, Environmental Health Criteria 36, IPCS In-                  tabase issue):D290&#226;&#8364;&#8220;D301. <br />     ternational Programme Chemical Safety (World Health Organization, Geneva).              34. Hillenmeyer ME, et al. (2008) The chemical genomic portrait yeast: Uncovering <br /> 10. Smith FA, Hodge HC, Dinman BD (1977) Airborne &#239;&#172;&#8218;uorides man: Part I. CRC Crit               phenotype genes. Science 320(5874):362&#226;&#8364;&#8220;365. <br />                                                                                                35. Drescher M, Suttie JW (1972) Intracellular &#239;&#172;&#8218;uoride cultured mammalian cells. Proc <br />     Rev Environ Control 8:293&#226;&#8364;&#8220;371. <br />                                                                                                    Soc Exp Biol Med 139(1):228&#226;&#8364;&#8220;230. <br /> 11. WHO (2006) Fluoride Drinking Water (World Health Organization, Geneva). <br />                                                                                                36. Quissell DO, Suttie JW (1972) Development &#239;&#172;&#8218;uoride-resistant strain L cells: <br /> 12. Barbier O, Arreola-Mendoza L, Del Razo LM (2010) Molecular mechanisms &#239;&#172;&#8218;uoride <br />                                                                                                    Membrane metabolic characteristics. Am J Physiol 223(3):596&#226;&#8364;&#8220;603. <br />     toxicity. Chem Biol Interact 188(2):319&#226;&#8364;&#8220;333. <br />                                                                                                37. Tyson CB, Lord PG, Wheals AE (1979) Dependency size Saccharomyces cerevisiae <br /> 13. Lesher RJ, Bender GR, Marquis RE (1977) Bacteriolytic action &#239;&#172;&#8218;uoride ions. Anti- <br />                                                                                                    cells growth rate. J Bacteriol 138(1):92&#226;&#8364;&#8220;98. <br />     microb Agents Chemother 12(3):339&#226;&#8364;&#8220;345. <br />                                                                                                38. Sikorski RS, Hieter P (1989) A shuttle vectors yeast host strains designed <br /> 14. Maltz M, Emilson CG (1982) Susceptibility oral bacteria various &#239;&#172;&#8218;uoride salts. <br />                                                                                                    ef&#239;&#172;?cient manipulation DNA Saccharomyces cerevisiae. Genetics 122(1):19&#226;&#8364;&#8220;27. <br />     J Dent Res 61(6):786&#226;&#8364;&#8220;790. <br />                                                                                                39. Gutknecht J, Walter A (1981) Hydro&#239;&#172;&#8218;uoric nitric acid transport lipid <br /> 15. Ochoa-Herrera V, et al. (2009) Toxicity &#239;&#172;&#8218;uoride microorganisms biological <br />                                                                                                    bilayer membranes. Biochim Biophys Acta 644(1):153&#226;&#8364;&#8220;156. <br />     wastewater treatment systems. Water Res 43(13):3177&#226;&#8364;&#8220;3186. <br />                                                                                                40. Hall JE (1981) Membrane Transport, eds Bonting SL, Pont JJHHM (Elsevier, New <br /> 16. Arthus M, Gavelle J (1903) Action sodium &#239;&#172;&#8218;uoride yeast. Compt Rend Soc Biol <br />                                                                                                    York), pp 107&#226;&#8364;&#8220;121. <br />     55:1481&#226;&#8364;&#8220;1483. <br />                                                                                                41. Ayotte P, H&#195;&#169;bert M, Marchand P (2005) Why hydro&#239;&#172;&#8218;uoric acid weak acid? J Chem <br /> 17. Treshow M (1965) Response pathogenic fungi sodium &#239;&#172;&#8218;uoride. Mycologia <br />                                                                                                    Phys 123(18):184501. <br />     57:216&#226;&#8364;&#8220;221.                                            <br /> </body></html>