<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                      NIH Public Access <br />                       Author Manuscript <br />                       Brain Res. Author manuscript; available PMC 2013 December 13. <br />                   Published final edited form as: <br />                    Brain Res. 2012 December 13; 1489: 66&#226;&#8364;&#8220;80. doi:10.1016/j.brainres.2012.10.035. <br />  <br />  <br />  <br />                   Behavioral electrophysiological outcomes tissue-specific <br />                   Smn knockdown Drosophila melanogaster <br />                   Christina Timmermana Subhabrata Sanyala,b,* <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                   aDepartment Cell Biology Emory University School Medicine, Atlanta, GA 30322, USA <br />  <br />                   bDepartment       Neurology Emory University School Medicine, Atlanta, GA 30322, USA <br />  <br />                   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                         Severe reduction <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>Survival</span> Motor Neuron 1 (SMN1) protein humans causes Spinal Muscular <br />                         Atrophy (SMA), debilitating childhood disease leads progressive impairment the <br />                         neuro-muscular system. Although previous studies attempted identify tissue(s) in <br />                         SMN1 loss critically leads disease, tissue-specific functions widely <br />                         expressed protein remain unclear. Here, leveraged <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>RNA interference</span> methods <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                         manipulate SMN function selectively Drosophila neurons muscles followed behavioral <br />                         electrophysiological analysis. High resolution measurement motor performance shows <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                         profound alterations locomotor patterns following pan-neuronal knockdown SMN. Further, <br />                         locomotor phenotypes elicited SMN knockdown motor neurons, supporting previous <br />                         demonstrations motor neuron-specific SMN function mice. <br />                         Electrophysiologically, SMN modulation muscles reveals largely normal synaptic transmission, <br />                         quantal release trans-synaptic homeostatic compensation larval neuro-muscular <br />                         junction. Neuronal SMN knockdown does alter baseline synaptic transmission, dynamics <br />                         synaptic depletion acute homeostatic compensation. However, chronic glutamate receptor- <br />                         dependent developmental homeostasis neuro-muscular junction strongly attenuated <br />                         following reduction SMN neurons. Together, results support distributed model of <br />                         SMN function distinct neuron-specific roles likely compromised following <br />                         global loss SMN patients. While complementary to, broad agreement with, recent <br />                         mouse studies suggest strong necessity SMN neurons, results uncover hitherto <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                         under-appreciated role SMN homeostatic regulatory mechanisms motor synapses. <br />  <br />  <br />                   Keywords <br />                         Drosophila; motor behavior; synaptic transmission; homeostasis; Spinal Muscular Atrophy; SMN <br />  <br />  <br />                   1. Introduction <br />                                     Spinal Muscular Atrophy (SMA) severely debilitating childhood disorder leads to <br />                                     progressive muscle weakness resulting loss degeneration motor neuron <br />  <br />  <br />  <br />                   &#194;&#169; 2012 Elsevier B.V. All rights reserved <br />                   * <br />                    Corresponding author Room No. 444, 615 Michael St., NE, Cell Biology Dept., Whitehead Building, Emory University, Atlanta, GA <br />                   30322 ssanya2@emory.edu Phone: 404-727-1250 Fax: 404-727-6256. <br />                   Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                   customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                   resulting proof published final citable form. Please note production process errors be <br />                   discovered affect content, legal disclaimers apply journal pertain. <br />                    Timmerman Sanyal                                                                                          Page 2 <br />  <br />  <br />                                     innervation. It relatively common autosomal recessive disorder affect 1 in <br />                                     6000 babies (Lefebvre et al., 1995; Pearn, 1978; Sleigh et al., 2011). SMA results <br />                                     predominantly mutations Survival Motor Neuron 1 gene (SMN1). Severity is <br />                                     related copy number activity near identical homolog humans, SMN2 <br />                                     (Coovert et al., 1997; Lefebvre et al., 1997). The SMN2 gene carries C-to-T transition <br />                                     leading aberrant splicing 80&#226;&#8364;&#8220;90% SMN2 transcript loss exon 7 (SMN&#206;&#8221;7) <br />                                     resulting protein reduced stability (Lorson et al., 1999; Lorson Androphy, <br />                                     2000; Monani et al., 1999). The overall reduction SMN protein leads severe <br />                                     neuromuscular degeneration including loss motor neuron cell bodies time <br />                                     (Balabanian et al., 2007; Jablonka et al., 2000; Monani et al., 2000). SMN highly <br />                                     conserved ubiquitously expressed protein required critically normal mRNA <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     splicing affects biogenesis U snRNP particles species (small nuclear <br />                                     ribonucleoprotein) (Fischer et al., 1997; Kroiss et al., 2008; Lee et al., 2009; Liu et al., 1997; <br />                                     Meister et al., 2001; Pellizzoni et al., 2002). Complete loss SMN cell lethal, therefore, <br />                                     SMA essentially disorder results reduced SMN availability (Feldkotter et al., <br />                                     2002; Mailman et al., 2002; McAndrew et al., 1997). Several mouse models SMA also <br />                                     suggest full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> SMN required cell viability, disease severity closely <br />                                     linked dosage SMN gene product. Thus, total loss Smn leads early lethality <br />                                     (Frugier et al., 2000), introduction copies human SMN2 gene results <br />                                     normal birth followed progressive decline neuromuscular motor function and <br />                                     subsequent death post-natal day 7 (PND 7). Introduction copies SMN2, <br />                                     however, leads complete rescue (Hsieh-Li et al., 2000; Monani et al., 2000). Interestingly, <br />                                     introduction SMN&#206;&#8221;7 &#226;&#8364;&#8220; mutant form human SMN missing exon 7 - Smn&#226;&#710;&#8217;/&#226;&#710;&#8217;; <br />                                     SMN2+/+ background provides additional rescue, animals die PND 6 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     13 (Le et al., 2005). Given SMN required cells, reduction leads to <br />                                     specific neuromuscular symptoms SMA symptoms arise SMN reduction in <br />                                     motor neurons muscle (or tissues), remain outstanding questions, though <br />                                     recent experiments indicate prominent neuronal functions (Gogliotti et al., 2012; Martinez et <br />                                     al., 2012; Park et al., 2010; Imlach et al., 2012; Lotti et al., 2012). <br />  <br />                                     Since SMN highly conserved protein, invertebrate models C. elegans and <br />                                     Drosophila used understand molecular functions SMN more <br />                                     rapidly identify networks genes SMN functions (Briese et al., 2009; Chan et <br />                                     al., 2003; Chang et al., 2008; Dimitriadi et al., 2010; Rajendra et al., 2007; Sen et al., 2011; <br />                                     Sleigh et al., 2011). These studies revealed remarkable similarities SMN function in <br />                                     regulation U snRNP biogenesis (Cauchi, 2010; Cauchi et al., 2008; Kroiss et al., <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     2008; Lee et al., 2009; Shpargel et al., 2009) NMJ development physiology (Cauchi <br />                                     et al., 2008; Chan et al., 2003; Chang et al., 2008; Praveen et al., 2012; Rajendra et al., 2007; <br />                                     Shpargel et al., 2009). While number studies used classical loss-of-function <br />                                     alleles Drosophila Smn, recent work use gene-targeted RNAi <br />                                     methodologies, used knock down, deplete SMN function selected <br />                                     tissues conjunction versatile GAL4-UAS (Brand Perrimon, 1993; <br />                                     Chang et al., 2008). Thus, RNAi based knockdown closer approximation situation <br />                                     SMA patients, permits experiments tissue-specific reduction SMN <br />                                     achieved vivo normal animal. If true different tissues have <br />                                     different thresholds susceptibility reduction SMN function, selective, and <br />                                     relatively mild, knockdown SMN uncover fundamentally conserved tissue-specific <br />                                     functions SMN (Sleigh et al., 2011). Here, use strategy test neuronal versus <br />                                     muscle-specific functions SMN Drosophila regulation motor behavior and <br />                                     synaptic physiology (Figure 1). Our results highlight neuronal muscle functions for <br />                                     Drosophila Smn, similar mouse studies, stronger neuronal outcomes. But more <br />                                     significantly, point precise role SMN regulation homeostatic <br />                                     mechanisms maintain parity NMJ. These findings especially relevant the <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                        Page 3 <br />  <br />  <br />                                     context putative developmental roles SMN mammals suggest unique function <br />                                     SMN homeostatically driven development mature motor neurons (Foust et al., <br />                                     2010; Gogliotti et al., 2012; Hammond et al., 2010; Martinez et al., 2012; Sleigh et al., <br />                                     2011). Behavioral synaptic phenotypes described report provide <br />                                     useful platforms functional testing Smn-interacting genes been <br />                                     discovered genetic biochemical screening approaches (Dimitriadi et al., 2010; <br />                                     Sen et al., 2011). <br />  <br />                      2. Results <br />                      2.1 Smn perturbation neurons muscle causes behavioral deficits <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Gross locomotor defects Drosophila Smn mutants mutants fly homolog of <br />                                     Gemin3 &#226;&#8364;&#8220; DEAD-box RNA helicase complexed SMN, reported previously <br />                                     (Cauchi et al., 2008; Chan et al., 2003; Praveen et al., 2012; Shpargel et al., 2009; Imlach et <br />                                     al., 2012). Since Smn mutants pupal adult lethal, studies revealed <br />                                     abnormal locomotion predominantly larval stages showed systemic cholinergic <br />                                     neuron-specific add-back SMN significantly rescued motor phenotypes. <br />                                     Consequently, relationship normal SMN function motor physiology been <br />                                     established Drosophila, similar human patients mammalian model systems <br />                                     (Cauchi et al., 2008; Chan et al., 2003; Shpargel et al., 2009; Imlach et al., 2012). However, <br />                                     currently unclear locomotor phenotypes Smn mutants arise loss of <br />                                     SMN neurons muscle. In study, attempted answer question using the <br />                                     GAL4-UAS (Brand Perrimon, 1993) method tissue-targeted transgene expression and <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     recently available reagents over-express length wild type SMN gene <br />                                     product knockdown Smn function using RNA interference (Chang et al., 2008; Sen et al., <br />                                     2011). We chose use RNAi reagents efficacy knocking down <br />                                     endogenous levels SMN neurons muscles, potency generating <br />                                     neuro-muscular defects characterized (Chang et al., 2008), reduction in <br />                                     SMN, completely removal, closely approximates situation SMA <br />                                     (western blots Figure 2A C). Since RNAi mediated knockdown Smn does lead <br />                                     lethality cases, able measure adult locomotor behavior a <br />                                     resolution higher possible larvae. Such measurements allow thorough <br />                                     analysis locomotor parameters useful screening tools future genetic <br />                                     interaction experiments. A complementary strategy rescue strong loss-of-function <br />                                     mutations SMN using tissue directed rescue recently reported (Imlach et al., <br />                                     2012). <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Pan-neuronal reduction SMN using elavC155-GAL4 driver results significant <br />                                     changes normal locomotor patterns flies Buridan&apos;s assay (Gotz, 1980). In this <br />                                     assay, 5&#226;&#8364;&#8220;6 day old flies allowed walk diametrically placed landmarks on <br />                                     circular arena movement recorded real time followed off-line analysis <br />                                     (Freeman et al., 2012; Neuser et al., 2008). In general, results suggest neuronal loss <br />                                     SMN leads slower walking speed, overall activity larger number pauses <br />                                     bouts walking compared genetic controls containing GAL4 line (Figure <br />                                     2A). These phenotypes seen Smn knocked using RNAi <br />                                     lines directed C- N-terminal portion Drosophila Smn mRNA (Chang et al., <br />                                     2008) confirming specificity phenotypes. Neuronal over-expression SMN, <br />                                     however, does lead discernible defects locomotion. Single fly tracks and <br />                                     cumulative occupancy plots (heat map coded regions highest occupancy during <br />                                     performance behavior red) demonstrate extent disruption of <br />                                     locomotor activity neuronal Smn knockdown animals (Figure 2B). <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                      Page 4 <br />  <br />  <br />                                     Changes SMN levels muscle produce strong locomotor phenotypes adults <br />                                     (Figure 2C D). Although muscle-specific roles SMN recognized both <br />                                     mammalian Drosophila models (Chan et al., 2003; Chang et al., 2008; Dachs et al., <br />                                     2011; Lee et al., 2012; Rajendra et al., 2007; Sen et al., 2011), RNAi mediated knockdown <br />                                     over-expression wild type SMN does result appreciable locomotor defects. Mild <br />                                     changes seen number pauses SMN control <br />                                     levels, suggesting likely result non-specific effects. Single <br />                                     representative fly tracks occupancy plots confirm changes SMN in <br />                                     muscle tissue affect locomotor performance assay. Overall, results presented in <br />                                     Figure 2 presence profound locomotor deficits adult animals following <br />                                     selective knockdown Smn neurons. These findings suggest Drosophila motor <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     phenotypes previously observed Smn mutants likely to loss SMN in <br />                                     neurons clarify SMN required specific neuronal subsets normal <br />                                     locomotor behavior. <br />  <br />                      2.2. Knockdown Smn glutamatergic neurons, cholinergic neurons, leads to <br />                      behavioral deficits <br />                                     Since pan-neuronal reduction SMN resulted locomotor deficits, asked whether <br />                                     SMN preferentially required subset neurons normal motor behavior. Given that <br />                                     motor neurons strongly implicated SMA, tested Smn knockdown adult <br />                                     motor neurons produces behavioral phenotypes. Drosophila motor neurons are <br />                                     glutamatergic. Therefore, used VGlut-GAL4 line (OK371) over-express or <br />                                     knockdown Smn glutamatergic neurons (note non-motor <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     glutamatergic neurons Drosophila nervous targeted OK371- <br />                                     GAL4) (Mahr Aberle, 2006). OK371-GAL4 driven Smn RNAi resulted phenotypes <br />                                     reminiscent pan-neuronal Smn knockdown (Figure 3A B). While average <br />                                     speed walking significantly altered, total distance moved overall activity <br />                                     significantly reduced, number pauses elevated, following Smn knockdown in <br />                                     glutamatergic nervous system. These phenotypes observed following over- <br />                                     expression Smn glutamatergic neurons. Single fly tracks occupancy plots also <br />                                     confirmed aberrant locomotor behavior knockdown animals, as <br />                                     strong following pan-neuronal knockdown Smn. By contrast, Smn perturbation <br />                                     cholinergic neurons did effect locomotion (Figure 3C D). Thus, <br />                                     Smn over-expressed knocked Cha-GAL4 driver (Salvaterra and <br />                                     Kitamoto, 2001), parameters altered significantly single fly tracks and <br />                                     occupancy plots reiterated normal locomotor behavior genotypes tested. Since all <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     accounts, Cha-GAL4 strong driver elicited phenotypes previous studies <br />                                     including used RNAi mediated knockdown (for examples Ghosh and <br />                                     Feany, 2004; Kitamoto, 2001; Lima Miesenbock, 2005; Sakai et al., 2009), not <br />                                     favor idea absence phenotypes consequence weak expression the <br />                                     SMN-RNAi construct. Instead, taken results described previous section, <br />                                     experiments suggest glutamatergic motor neurons likely a <br />                                     prominent locus Smn-dependent locomotor phenotypes Drosophila. We tested <br />                                     altered synaptic transmission motor synapses underlie behavioral deficits <br />                                     following tissue-specific knockdown SMN. <br />  <br />                      2.3. Neuronal perturbation Smn does alter baseline high-frequency synaptic <br />                      transmission <br />                                     Analysis adult behavior shows reduced SMN motor neurons results behavioral <br />                                     deficits visuo-motor assay. We wondered detect possible basis for <br />                                     motor phenotypes rooted synaptic physiology. Since larval neuro-muscular <br />                                     simpler experimentally accessible adult, used model <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                      Page 5 <br />  <br />  <br />                                     synapse test idea. Baseline synaptic transmission measured muscle 6/7 <br />                                     motor synapse abdominal segment A2 1 mM external calcium reported previously <br />                                     (Freeman et al., 2011; Sen et al., 2011). elavC155-GAL4 used drive expression full- <br />                                     length Smn RNAi constructs described previous sections. Interestingly, did not <br />                                     observe meaningful change synaptic transmission conditions (Figure 4A <br />                                     B). Mean excitatory junction potential (EJP) amplitude, mini evoked junction potential <br />                                     (mEJP) amplitude quantal content release remained unaltered genotypes <br />                                     tested, barring modest increase mEJP amplitude following over-expression Smn. <br />                                     These data indicate normal parameters synaptic transmission affected Smn <br />                                     perturbation neurons, 50% reduction growth synapse the <br />                                     number synaptic boutons conditions Smn knockdown reported <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     previously (Chang et al., 2008). <br />  <br />                                     We tested Smn knockdown impairs synaptic transmission is <br />                                     stressed high-frequency stimulation conditions maximize quantal content <br />                                     release. We drove N4 RNAi line elavC155-GAL4 result the <br />                                     strongest pre-synaptic knockdown Smn. Motor neurons innervating muscle 6/7 were <br />                                     stimulated 10 Hz duration 10 minutes external calcium concentration 10 <br />                                     mM. Under conditions synaptic vesicles released recovered, <br />                                     leading depletion readily releasable pool synaptic vesicles rapid decline <br />                                     size EJP reflects fewer vesicles released action potential. <br />                                     Following rapid decline, synapse recovers partially reaches stable transmission <br />                                     rate vesicle recruitment reserve pool vesicles. After transition 10 <br />                                     Hz stimulation 0.1 Hz stimulation, synapse recovers immediately pre- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     stimulus levels balance vesicle fusion vesicle retrieval restored. <br />                                     This assay, therefore, examines mechanisms underlying rapid recruitment distinct vesicle <br />                                     pools synaptic vesicle fusion recycling (Dickman et al., 2005; Seabrooke and <br />                                     Stewart, 2011). When challenged high-frequency stimulation Smn knockdown <br />                                     synapses performed control ones (Figure 4C). The rate extent decline, <br />                                     reserve pool recruitment recovery post-stimulation significantly different <br />                                     knockdown control animals suggesting normal vesicle dynamics these <br />                                     synapses. Taken together, results section imply behavioral deficits following <br />                                     neuronal knockdown result obvious changes baseline synaptic <br />                                     transmission neuro-muscular junction. This apparent conflict earlier reports <br />                                     Drosophila (Chan et al., 2003) studies rodents (Kong et al., 2009; Martinez <br />                                     et al., 2012; Ruiz et al., 2010). However, studies did selectively remove Smn in <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     neurons, synaptic transmission phenotypes situations arisen <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                     developmental changes triggered reduced SMN neurons muscles indeed <br />                                     tissues. In situation Smn selectively reduced rodent motor neurons, <br />                                     synaptic behavioral defects mild improved dramatically postnatally (Park <br />                                     et al., 2010). <br />  <br />                      2.4. Smn perturbation muscle affects quantal size <br />                                     Increasingly, role normal SMN function muscle appreciated (Dachs et al., <br />                                     2011; Lee et al.; Martinez et al., 2012). These studies suggest SMN muscle might <br />                                     contribute regulation normal muscle development synaptic physiology. In <br />                                     Drosophila, previous experiments clearly documented post-synaptic enrichment of <br />                                     SMN protein, synaptic growth phenotypes larval neuro-muscular junction have <br />                                     resulted Smn knockdown (Chang et al., 2008), strong muscle atrophy Smn mutants <br />                                     (Rajendra et al., 2007) genetic interactions Smn FGF signaling pathways in <br />                                     muscle (Sen et al., 2011). Based information, asked Smn perturbations in <br />                                     muscle result synaptic phenotypes larval NMJ. <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                        Page 6 <br />  <br />  <br />                                     how24B-GAL4 driven expression full-length Smn Smn-RNAi resulted reduced mean <br />                                     EJP mEJP amplitude (Figure 5A B) resulting indistinguishably similar quantal <br />                                     content release genotypes. Although surprising over-expression <br />                                     knockdown Smn lead equivalent synaptic phenotypes, line the <br />                                     observation over-expression knockdown Smn muscle leads pupal/adult <br />                                     lethality (Chang et al., 2008) abnormally long pupal cases (our unpublished <br />                                     observations). While reasons similarity phenotypes unclear, it <br />                                     likely perturbations muscle SMN function. Quantal <br />                                     size phenotypes prompted examine staining intensity distribution post- <br />                                     synaptic glutamate receptors. A severe disorganization post-synaptic Acetylcholine <br />                                     receptors NMJ widely observed rodent models SMA (Dachs et al., <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     2011; Kariya et al., 2008; Kong et al., 2009; Lee et al., 2008; Ling et al.; Murray et al., <br />                                     2008; Ruiz et al., 2010) loss glutamate receptor staining reported in <br />                                     Drosophila Smn mutants (Chan et al., 2003). However, did observe changes in <br />                                     overall glutamate receptor staining intensity distribution Smn knocked in <br />                                     muscles using stronger N4 RNAi transgene (Figure 5C). Similarly, change in <br />                                     glutamate receptor distribution observed following neuronal knockdown Smn either <br />                                     (data shown). These results suggest reduction SMN exclusively muscle <br />                                     (or neurons) sufficient cause strong synaptic transmission defects receptor <br />                                     misaggregation, sufficient cause adult lethality (Chang et al., 2008). <br />                                     Changes quantal size simply small alterations muscle size (Imlach <br />                                     et al., 2012) able detect readily, therefore, input resistance. It is <br />                                     possible neuro-muscular symptoms SMA arise progressive distortion NMJ <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     development maintenance precipitated chronic loss SMN compartments <br />                                     (Martinez et al., 2012). If true, compartment-specific knockdown SMN might <br />                                     reveal initial events trigger collapse. <br />  <br />                      2.5. Pre-synaptic knockdown Smn abolishes long-term homeostatic compensation at <br />                      NMJ <br />                                     Rodent models SMA early synaptic transmission deficits NMJ typically <br />                                     worsen age (Dachs et al., 2011; Kariya et al., 2008; Kong et al., 2009; Lee et al., 2012; <br />                                     Ling et al., 2011; Murray et al., 2008; Ruiz et al., 2010). Additionally, recent studies suggest <br />                                     homeostatic feedback mechanisms maintain parity synaptic transmission might <br />                                     compromised SMN levels reduced motor neurons. In study, selective <br />                                     reduction SMN motor neurons resulted reduced quantal content transmission at <br />                                     post-natal day 8 (PND8) mice (Park et al., 2010). Strikingly, defects sharply <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     reduced PND10&#226;&#8364;&#8220;12, 3 months, NMJs showed overcompensation they <br />                                     displayed higher mini-end plate potentials (mEPPs), EPPs quantal content control <br />                                     animals (Park et al., 2010). These data reveal aberrations homeostatic mechanisms the <br />                                     synapse following neuronal knockdown Smn. Conversely, studies have <br />                                     addressed tissue-specificity SMN rescuing SMN expression neurons or <br />                                     muscle Smn mutant background (Gogliotti et al., 2012; Martinez et al., 2012). While <br />                                     neuronal, muscle, add-back SMN able rescue synaptic transmission defects <br />                                     NMJ Smn mutants, surprisingly, neuronal supplementation SMN prevented <br />                                     loss central motor neuron synaptic input. This observation suggests normal <br />                                     conditions neuronal SMN regulate retrograde signaling mechanisms maintain <br />                                     homeostatic drive synapses. <br />  <br />                                     To directly evaluate SMN plays role homeostatic signaling mechanisms in <br />                                     neurons, used assays test rapid, gene expression-independent long- <br />                                     term, gene expression requiring form homeostasis larval NMJ. An activity- <br />                                     dependent block post-synaptic glutamate receptors larval NMJ using wasp <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                        Page 7 <br />  <br />  <br />                                     venom Philanthotoxin (PhTox) triggers fast homeostatic response pre-synapse <br />                                     results significant increase quantal content, quantal size remains <br />                                     predictably reduced (Frank et al., 2006; Frank et al., 2009). We tested Smn <br />                                     knockdown neurons muscles affected homeostatic response. Figure 6A shows that <br />                                     PhTox-dependent homeostatic responses remained robust case. Thus, reduction of <br />                                     SMN using N4 RNAi transgene did alter synapse&apos;s ability engage fast <br />                                     homeostatic increase quantal content following glutamate receptor attenuation with <br />                                     PhTox (reflected reduced mEJP amplitudes Figure 6A). These data suggest that <br />                                     local signaling mechanisms involved form homeostasis require <br />                                     SMN activity (Figure 6D). <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Since aberrations homeostasis rodent models SMA occur developmental time <br />                                     period, tested form homeostatic compensation larval NMJ longer <br />                                     term likely requires changes gene expression. A strong genetic reduction one <br />                                     glutamate receptor subunit, GluRIIA, leads severely attenuated post-synaptic currents and <br />                                     smaller mEJP amplitudes (Davis et al., 1998; Petersen et al., 1997). However, over <br />                                     developmental time, pre-synapse compensates increase quantal content <br />                                     (Davis et al., 1998 1999; Haghighi et al., 2003; Penney et al., 2012; Petersen et al.,1997). <br />                                     When reduced SMN neurons GluRIIA mutant background, saw virtual <br />                                     absence homeostatic compensation (Figure 6B C Supplementary Table 1). By <br />                                     contrast, SMN reduction muscles did affect form homeostasis. These <br />                                     observations point specific neuronal functions SMN control long-term <br />                                     homeostatic responses. Such function consistent general role for <br />                                     SMN regulation RNA metabolism gene expression (Figure 6D). <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                      3. Discussion <br />                                     Although obvious differences Drosophila mammalian neuro-muscular <br />                                     organization, modeling SMA flies productive approach given high degree of <br />                                     conservation SMN function ease rapidity genetic analysis flies <br />                                     (Dimitriadi et al., 2010). Indeed, previous work Drosophila suggested Smn <br />                                     regulates U snRNP biogenesis (Cauchi, 2010; Kroiss et al., 2008; Lee et al., 2009; Shpargel <br />                                     et al., 2009) loss SMN results synaptic motor defects comparable <br />                                     seen mouse models SMA (Chan et al., 2003; Chang et al., 2008; Rajendra et <br />                                     al., 2007; Shpargel et al., 2009). Recent work use RNA interference to <br />                                     knockdown Smn target tissues (Chang et al., 2008; Sen et al., 2011) better mimic the <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     situation SMA patients. Experiments validated RNAi reagents uncovered <br />                                     morphological phenotypes NMJ SMN knocked neurons or <br />                                     muscle (Chang et al., 2008; Sen et al., 2011). In current study used same <br />                                     RNAi reagents conjunction GAL4-UAS reduce SMN neurons or <br />                                     muscle tissue followed assessment perturbation motor performance and <br />                                     synaptic transmission NMJ (Figure 1). Muscle knockdown SMN did result in <br />                                     strong behavioral synaptic transmission defects, similar manipulations been <br />                                     shown affect synaptic morphology larvae lead adult lethality (Chang et al., 2008; <br />                                     Sen et al., 2011). This suggests situation Drosophila similar mice, <br />                                     SMN plays role muscles ultimately impact lifespan, critically <br />                                     required normal synaptic function (Gogliotti et al., 2012; Martinez et al., 2012). On the <br />                                     hand, distinct deficits visuo-motor performance SMN reduced <br />                                     neuronally, baseline synaptic transmission conditions largely <br />                                     unaffected. This contrasts earlier findings Drosophila (Chan et al., 2003) is <br />                                     likely compartment-specific knockdown SMN experiments. <br />                                     Interestingly, recent report suggests larval locomotor phenotypes strong <br />                                     hypomorphic mutations SMN arise SMN deficiency cholinergic neurons (Imlach <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                        Page 8 <br />  <br />  <br />                                     et al., 2012). While complementary manipulations reveal effect of <br />                                     cholinergic knock SMN locomotor activity adult flies (perhaps to <br />                                     insufficient knock RNAi reagent available us), able identify a <br />                                     specific loss homeostatic compensation pre- post-synapse when <br />                                     SMN reduced neuronally (Figure 6D). We propose reflect early and <br />                                     subtle event SMA etiology predates dramatic SMA sequelae comprising loss of <br />                                     normal NMJs neuro-muscular degeneration. <br />  <br />                                     Severe mouse models SMA (e.g. Smn&#226;&#710;&#8217;/&#226;&#710;&#8217;; SMN2+/+) clearly discernible loss of <br />                                     neuro-muscular morphology, strong synaptic transmission defects, loss pre-synaptic <br />                                     inputs spinal motor neurons, muscle degeneration eventual death motor neurons <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     (Balabanian et al., 2007; Jablonka et al., 2000; Monani et al., 2000). Several studies have <br />                                     used Cre-loxP derived conditional <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> models SMA mice address the <br />                                     question tissue-specificity SMN vis-&#195;&#160;-vis incidence SMA-like symptoms. <br />                                     Complete removal exon 7 specific tissues led cell lethality, situation, though <br />                                     consistent vital role SMN cells, significantly different in <br />                                     SMA patients (Cifuentes-Diaz et al., 2001; Cifuentes-Diaz et al., 2002; Frugier et al., 2000; <br />                                     Nicole et al., 2003; Vitte et al., 2004). Recent studies successfully approximated <br />                                     patient condition, trying distinguish tissue-specific functions SMN vivo <br />                                     (Gogliotti et al., 2012; Martinez et al., 2012; Park et al., 2010). In study, Smn exon 7 <br />                                     removed neurons background SMN2 introduced (Park et al., 2010). <br />                                     Surprisingly, mice showed mild SMA symptoms improved age. However, <br />                                     manipulation did uncover subtle homeostatic regulatory phenotypes neuro- <br />                                     muscular junction (NMJ) noticed presence typically <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     severe SMA phenotypes (Kariya et al., 2008; Michaud et al., 2010; Murray et al., 2010; <br />                                     Murray et al., 2008; Ruiz et al., 2010). In study, full-length SMN protein added <br />                                     neurons muscle mutant SMN background results in <br />                                     viable &#226;&#8364;&#339;SMA&#226;&#8364;? mice (Lutz et al., 2011; Martinez et al., 2012). This study revealed while <br />                                     neuronal muscle add-back SMN significantly rescue survival motor <br />                                     behavior, neuronal expression rescued synaptic function NMJ motor <br />                                     neuron somal synapses, putative homeostatic mechanisms (Martinez et al., 2012). A <br />                                     study showed, adding SMN selectively motor neurons using Hb9-Cre, <br />                                     SMN supplementation motor neurons rescue vast majority motor defects <br />                                     restored normal sensory-motor synapses (Gogliotti et al., 2012). Together, these <br />                                     observations hint neuron-specific roles SMN homeostatic regulation normal <br />                                     synaptic connectivity transmission mammals. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Phenotypic end-points severe SMA mouse models closely mirror SMA pathology in <br />                                     patients, particularly Type1 SMA. However, progressive nature phenotypes, <br />                                     increased fetal expression SMN general observation SMA phenotypes be <br />                                     effectively rescued adding length SMN perinatally, suggest defining <br />                                     events SMA pathology occur early stages development (Baumer et al., <br />                                     2009; Burlet et al., 1998; Butchbach et al., 2010; Foust et al., 2010; Gabanella et al., 2005; <br />                                     Hammond et al., 2010; Murray et al., 2010; Narver et al., 2008). Thus, obvious target tissues <br />                                     SMA motor neurons muscle exquisitely susceptible reduction in <br />                                     SMN function subtle defects neuro-muscular development physiology the <br />                                     appear loci &#226;&#8364;&#8220; consistent threshold hypothesis SMA (Sleigh et al., <br />                                     2011). Milder models SMA mice mentioned previous section, <br />                                     support idea ways reveal aberrations homeostatic <br />                                     regulation synaptic connectivity discernible presence more <br />                                     severe phenotypes (Gogliotti et al., 2012; Martinez et al., 2012; Park et al., 2010). These <br />                                     observations point fundamental impairment communication motor neurons <br />                                     pre-synaptic input following loss SMN. Over time, lead an <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                        Page 9 <br />  <br />  <br />                                     asynchrony tightly coordinated program synaptic development leading to <br />                                     eventual synaptic degeneration. <br />  <br />                                     Our experiments Drosophila probably represent relatively mild manipulations Smn. <br />                                     Consistent idea, animals largely viable dramatic loss of <br />                                     locomotion larval stages seen Smn loss-of-function mutants (Chan et al., 2003; <br />                                     Shpargel et al., 2009). However, neuronal knockdown Smn does result adult locomotor <br />                                     dysfunction discerned sophisticated visuo-motor assays. Further <br />                                     electrophysiological analysis larval NMJ provides possible underlying mechanism, <br />                                     obvious caveat recordings larval stages, behavioral <br />                                     analysis adults. Although parameters synaptic transmission near normal, we <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     detect specific phenotypes Smn perturbed neurons muscles. Muscle <br />                                     knockdown Smn results altered quantal size. Although did detect changes <br />                                     glutamate receptor staining intensity distribution, increased mEJP size result <br />                                     altered post-synaptic receptor sensitivity electrophysiological measures typically <br />                                     sensitive immuno-histochemistry. Alternatively, altered quantal size result <br />                                     changes synaptic vesicle size neurotransmitter packaging, small changes <br />                                     muscle size input impedance. Neuronal knockdown Smn leads defect in <br />                                     homeostatic compensation. A severe impairment synaptic transmission described <br />                                     previously (Chan et al., 2003), allowed detection <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span>. In <br />                                     general, phenotype finds resonance defects homeostatic regulatory mechanisms <br />                                     highlighted previously mild SMA models mice (Gogliotti et al., 2012; <br />                                     Martinez et al., 2012; Park et al., 2010) represent &#226;&#8364;&#339;weak link&#226;&#8364;? most <br />                                     susceptible loss SMN neurons. It conceivable SMN-dependent loss <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     calibration homeostatic drive earliest events disrupts developmental <br />                                     coupling pre- post-synapse critical period synaptogenesis. It is <br />                                     interesting note phenotype uncovered Glutamate receptor <br />                                     expression reduced, suggesting low neuronal SMN limits capacity a <br />                                     homeostatic response demand response high. Further experiments are <br />                                     required test prediction holds mouse models SMA. <br />  <br />                                     Our results help understanding molecular function SMN neurons. <br />                                     While SMN clearly involved U snRNP biogenesis, clear function <br />                                     SMN compromised neurons (Buhler et al., 1999; Gabanella et al., <br />                                     2007; Pellizzoni et al., 1999; Shpargel Matera, 2005; Wan et al., 2005). For example, in <br />                                     Drosophila, Smn <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>null mutants</span> normal mRNA splicing (Praveen et al., 2012; Rajendra <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     et al., 2007). In addition, low transgene-mediated expression wild type SMN rescues <br />                                     SMA-like phenotypes flies improvement snRNA levels (Praveen et al., <br />                                     2012). By contrast, recent report implicates aberrant U12 splicing events collection of <br />                                     U12 intron containing genes following SMN perturbations (Lotti et al., 2012). While snRNA <br />                                     levels tested manipulations, phenotypic analysis shows clearly a <br />                                     gene expression-requiring homeostatic compensation completely abolished a <br />                                     neuron-specific reduction SMN (red arrows Figure 6D). By contrast, rapid gene <br />                                     expression-independent form homeostasis NMJ normal (grey arrows Figure <br />                                     6D), suggesting role SMN regulation nuclear gene expression. <br />                                     Whether occurs control mRNA metabolism, remains investigated. <br />                                     However, interesting note form long-term homeostasis requires BMP/ <br />                                     TGF-&#206;&#178; signaling pre-synapse, signaling pathway recently shown interact with <br />                                     SMN flies (Chang et al., 2008; Haghighi et al., 2003). In model, envisage <br />                                     SMN playing role relay homeostatic &#226;&#8364;&#339;signal&#226;&#8364;? pre-synaptic <br />                                     compartment, execution suitably graded response (Figure 6D). Although our <br />                                     studies previously shown role SMN regulation FGF signaling the <br />                                     muscle, function does necessary homeostatic regulation synaptic <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                    Page 10 <br />  <br />  <br />                                     transmission (Sen et al., 2011). Finally, note motor neurons Drosophila are <br />                                     glutamatergic. Whether GluR-dependent homeostatic signaling flies bearing on <br />                                     cholinergic neuro-muscular synapses glutamatergic sensory-motor synapses mammals <br />                                     remains seen. <br />  <br />                      4. Experimental Procedure <br />                      4.1. Drosophila strains, genetics husbandry <br />                                     Drosophila strains reared standard corn meal&#226;&#8364;&#8220;dextrose&#226;&#8364;&#8220;yeast containing food 25 <br />                                     &#194;&#176;C controlled humidity incubators constant 12 h light:dark cycle. elavC155-GAL4 <br />                                     how24B-GAL4 described previously (Brand Perrimon, 1993; Luo et al., <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     1994). UAS-SMN, UAS-SMN-RNAi[C24] UAS-SMN-RNAi[N4] Spyros <br />                                     Artavanis-Tsakonas validated described earlier (Chang et al., 2008). <br />                                     GluRIIA mutants (GluRIIASp16) deficiency (Df(2l)cl-h4) Pejmun Haghighi <br />                                     (Penney et al., 2012). <br />  <br />                      4.2. Antibodies, immuno-histochemistry western blotting <br />                                     Larval dissection, staining confocal microscopy performed according standard <br />                                     protocols (Franciscovich et al., 2008). Briefly, larvae dissected Ca2+ free HL3 <br />                                     ringer&apos;s solution (in mM): NaCl 70, KCl 5, MgCl2 20, NaHCO3 10, Sucrose 115, Trehalose <br />                                     5, BES 5 (pH 7.2). Dissections fixed 4% paraformaldehyde, stained primary <br />                                     antibody overnight, followed incubation Alexa Fluor conjugated secondary <br />                                     antibody. Rabbit anti-dGluRIII (Marrus et al., 2004) used 1:1000 Alexa-568 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     conjugated anti-HRP (Molecular Probes) used 1:500. Anti-rabbit Alexa 488 <br />                                     conjugated secondary antibody (Molecular Probes) used 1:400. An inverted 510 Zeiss <br />                                     LSM microscope used imaging. For quantitative fluorescence care taken to <br />                                     prepare samples identically. Samples imaged analyzed double blind, all <br />                                     imaging interleaved control experimental samples imaged alternately <br />                                     day. Confocal settings including black level (offset/contrast), gain, pixel dwell <br />                                     time number iterative samplings noise reduction kept constant. Western <br />                                     blotting carried according standard procedures (Freeman et al., 2012). Rabbit <br />                                     polyclonal anti-SMN antibodies used 1:1000 (Chang et al., 2008). <br />  <br />                      4.3. Electrophysiology <br />                                     Baseline evoked spontaneous junction potentials measured described previously <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     (Sen et al., 2011). Briefly, wandering instar larvae dissected normal HL3 <br />                                     ringer&apos;s solution (Stewart et al., 1994) (in mM): NaCl 70, KCl 5, MgCl2 20, CaCl2 1, <br />                                     NaHCO3 10, Sucrose 115, Trehalose 5, BES 5 (pH 7.2). Intracellular recording electrodes <br />                                     tip resistances 25&#226;&#8364;&#8220;50 M&#206;&#169; filled 3 M KCl. Only recordings <br />                                     used resting membrane potential polarized &#226;&#710;&#8217;60 mV the <br />                                     muscle input resistance greater 10 M&#206;&#169;. Muscle 6 (VL3) abdominal segment A2 <br />                                     used recordings. A train 25 supra-threshold stimuli 0.5 Hz delivered in <br />                                     experiment record compound EJPs Is Ib terminals, mean <br />                                     peak EJP values obtained averaging 20 traces. EJP amplitudes were <br />                                     corrected non-linear summation using Martin&apos;s correction factor, assuming reversal <br />                                     potential 0 mV membrane capacitance factor 0.55 (Kim et al., 2009; McLachlan <br />                                     Martin, 1981). 8&#226;&#8364;&#8220;10 separate animals used genotype. A 2 minute <br />                                     continuous recording used measure mEJP amplitude. Traces low-pass filtered at <br />                                     1 kHz analyzed using Clampfit Mini-Analysis programs (Synaptosoft). 8&#226;&#8364;&#8220;10 separate <br />                                     animals analyzed genotype. Quantal content determined dividing the <br />                                     mean corrected EJP amplitude given synapse mean mEJP amplitude. Statistical <br />                                     significance determined using one-way ANOVA. Representative traces plotted <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                                   Page 11 <br />  <br />  <br />                                     using MS-Excel episodic recordings exported Clampfit axon text files. High- <br />                                     frequency stimulation carried modified HL3 Ringer&apos;s solution containing 10 <br />                                     mM Ca2+ described previously (Dickman et al., 2005) (in mM): NaCl 70, KCl 5, MgCl2 <br />                                     10, CaCl2 10, NaHCO3 10, Sucrose 115, Trehalose 5, BES 5 (pH 7.2). Rapid PhTox (Sigma <br />                                     Aldrich Inc.) mediated homeostasis measured described earlier 0.3 mM Ca2+ <br />                                     containing HL3.1 Ringer&apos;s solution (Feng et al., 2004; Frank et al., 2006) (in mM): NaCl 70, <br />                                     KCl 5, MgCl2 10, CaCl2 0.3, NaHCO3 10, Sucrose 115, Trehalose 5, BES 5 (pH 7.2). <br />                                     Recordings loss GluRIIA background carried HL3.1 Ringer&apos;s <br />                                     0.3 mM Ca2+. <br />  <br />                      4.4. Behavioral analysis <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     2&#226;&#8364;&#8220;3 day old flies collected wings cut close thorax. These flies were <br />                                     allowed recover additional 3 days. Flies individually placed circular <br />                                     platform 10 cm diameter surrounded moat water 2 cm width. Flies <br />                                     normally walked forth diametrically opposite vertical black bars <br />                                     brightly illuminated cylinder. A 5 mega-pixel webcam mounted centrally the <br />                                     platform recorded movement period 5 minutes Buridan tracking <br />                                     program Bjorn Brembs. These recordings analyzed using custom designed <br />                                     Buridan analysis software (from Bjorn Brembs) written statistical package R. Results <br />                                     analysis plotted MS-Excel. Statistical significance determined using <br />                                     one-way ANOVA. Single representative fly tracks occupancy plots generated <br />                                     using Buridan analysis software. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                      Supplementary Material <br />                                     Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                      Acknowledgments <br />                                     The authors thank Spyros Artavanis-Tsakonas UAS-SMN UAS-SMN-RNAi reagents, Pejmun Haghighi <br />                                     GluRIIA stocks, Graeme Davis help PhTox experiments Aaron DiAntonio anti-GluRIII <br />                                     antibodies. S.S. thanks Julien Colomb Bjorn Brembs help Buridan&apos;s assay Anindya Sen <br />                                     useful discussions. <br />  <br />  <br />                      Abbreviations <br />                                                 Neuro-muscular junction <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  NMJ <br />                                  SMA            Spinal Muscular Atrophy <br />                                  SMN            Survival Motor Neuron <br />                                  UAS            Upstream Activating Sequence <br />                                  RNAi           RNA interference <br />                                  FGF            Fibroblast Growth Factor <br />                                  BMP            Bone Morphogenetic Protein <br />                                  GluR           Glutamate Receptor <br />  <br />                      6. References <br />                                     1. Balabanian S, Gendron NH, MacKenzie AE. Histologic transcriptional assessment mild <br />                                        SMA model. Neurol Res. 2007; 29:413&#226;&#8364;&#8220;24. [PubMed: 17535551] <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                             Page 12 <br />  <br />  <br />                                     2. Baumer D, Lee S, Nicholson G, Davies JL, Parkinson NJ, Murray LM, Gillingwater TH, Ansorge <br />                                        O, Davies KE, Talbot K. Alternative splicing events late feature pathology mouse <br />                                        model spinal muscular atrophy. PLoS Genet. 2009; 5:e1000773. [PubMed: 20019802] <br />                                     3. Brand AH, Perrimon N. Targeted gene expression means altering cell fates generating <br />                                        dominant phenotypes. Development. 1993; 118:401&#226;&#8364;&#8220;15. [PubMed: 8223268] <br />                                     4. Briese M, Esmaeili B, Fraboulet S, Burt EC, Christodoulou S, Towers PR, Davies KE, Sattelle DB. <br />                                        Deletion smn-1, Caenorhabditis elegans ortholog spinal muscular atrophy gene, results <br />                                        locomotor dysfunction reduced lifespan. Hum Mol Genet. 2009; 18:97&#226;&#8364;&#8220;104. [PubMed: <br />                                        18829666] <br />                                     5. Buhler D, Raker V, Luhrmann R, Fischer U. Essential role tudor domain SMN in <br />                                        spliceosomal U snRNP assembly: implications spinal muscular atrophy. Hum Mol Genet. 1999; <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                        8:2351&#226;&#8364;&#8220;7. [PubMed: 10556282] <br />                                     6. Burlet P, Huber C, Bertrandy S, Ludosky MA, Zwaenepoel I, Clermont O, Roume J, Delezoide AL, <br />                                        Cartaud J, Munnich A, Lefebvre S. The distribution SMN protein complex human fetal tissues <br />                                        alteration spinal muscular atrophy. Hum Mol Genet. 1998; 7:1927&#226;&#8364;&#8220;33. [PubMed: <br />                                        9811937] <br />                                     7. Butchbach ME, Rose FF Jr. Rhoades S, Marston J, McCrone JT, Sinnott R, Lorson CL. Effect of <br />                                        diet survival phenotype mouse model spinal muscular atrophy. Biochem Biophys <br />                                        Res Commun. 2010; 391:835&#226;&#8364;&#8220;40. [PubMed: 19945425] <br />                                     8. Cauchi RJ. SMN Gemins: `we family&apos; &#226;&#8364;&#166; we?: insights partnership between <br />                                        Gemins spinal muscular atrophy disease protein SMN. Bioessays. 2010; 32:1077&#226;&#8364;&#8220;89. <br />                                        [PubMed: 20954180] <br />                                     9. Cauchi RJ, Davies KE, Liu JL. A motor function DEAD-box RNA helicase, Gemin3, in <br />                                        Drosophila. PLoS Genet. 2008; 4:e1000265. [PubMed: 19023405] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     10. Chan YB, Miguel-Aliaga I, Franks C, Thomas N, Trulzsch B, Sattelle DB, Davies KE, van den <br />                                          Heuvel M. Neuromuscular defects Drosophila survival motor neuron gene mutant. Hum Mol <br />                                          Genet. 2003; 12:1367&#226;&#8364;&#8220;76. [PubMed: 12783845] <br />                                     11. Chang HC, Dimlich DN, Yokokura T, Mukherjee A, Kankel MW, Sen A, Sridhar V, Fulga TA, <br />                                          Hart AC, Van Vactor D, Artavanis-Tsakonas S. Modeling spinal muscular atrophy Drosophila. <br />                                          PLoS One. 2008; 3:e3209. [PubMed: 18791638] <br />                                     12. Cifuentes-Diaz C, Frugier T, Tiziano FD, Lacene E, Roblot N, Joshi V, Moreau MH, Melki J. <br />                                          Deletion murine SMN exon 7 directed skeletal muscle leads severe muscular dystrophy. J <br />                                          Cell Biol. 2001; 152:1107&#226;&#8364;&#8220;14. [PubMed: 11238465] <br />                                     13. Cifuentes-Diaz C, Nicole S, Velasco ME, Borra-Cebrian C, Panozzo C, Frugier T, Millet G, <br />                                          Roblot N, Joshi V, Melki J. Neurofilament accumulation motor endplate lack axonal <br />                                          sprouting spinal muscular atrophy mouse model. Hum Mol Genet. 2002; 11:1439&#226;&#8364;&#8220;47. <br />                                          [PubMed: 12023986] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     14. Coovert DD, Le TT, McAndrew PE, Strasswimmer J, Crawford TO, Mendell JR, Coulson SE, <br />                                          Androphy EJ, Prior TW, Burghes AH. The survival motor neuron protein spinal muscular <br />                                          atrophy. Hum Mol Genet. 1997; 6:1205&#226;&#8364;&#8220;14. [PubMed: 9259265] <br />                                     15. Dachs E, Hereu M, Piedrafita L, Casanovas A, Caldero J, Esquerda JE. Defective neuromuscular <br />                                          junction organization postnatal myogenesis mice severe spinal muscular atrophy. J <br />                                          Neuropathol Exp Neurol. 2011; 70:444&#226;&#8364;&#8220;61. [PubMed: 21572339] <br />                                     16. Davis GW, DiAntonio A, Petersen SA, Goodman CS. Postsynaptic PKA controls quantal size and <br />                                          reveals retrograde signal regulates presynaptic transmitter release Drosophila. Neuron. <br />                                          1998; 20:305&#226;&#8364;&#8220;15. [PubMed: 9491991] <br />                                     17. Dickman DK, Horne JA, Meinertzhagen IA, Schwarz TL. A slowed classical pathway than <br />                                          kiss-and-run mediates endocytosis synapses lacking synaptojanin endophilin. Cell. 2005; <br />                                          123:521&#226;&#8364;&#8220;33. [PubMed: 16269341] <br />                                     18. Dimitriadi M, Sleigh JN, Walker A, Chang HC, Sen A, Kalloo G, Harris J, Barsby T, Walsh MB, <br />                                          Satterlee JS, Li C, Van Vactor D, Artavanis-Tsakonas S, Hart AC. Conserved genes act as <br />                                          modifiers invertebrate SMN loss function defects. PLoS Genet. 2010; 6:e1001172. [PubMed: <br />                                          21124729] <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                               Page 13 <br />  <br />  <br />                                     19. Feldkotter M, Schwarzer V, Wirth R, Wienker TF, Wirth B. Quantitative analyses SMN1 and <br />                                          SMN2 based real-time lightCycler PCR: fast highly reliable carrier testing prediction <br />                                          severity spinal muscular atrophy. Am J Hum Genet. 2002; 70:358&#226;&#8364;&#8220;68. [PubMed: 11791208] <br />                                     20. Feng Y, Ueda A, Wu CF. A modified minimal hemolymph-like solution, HL3.1, physiological <br />                                          recordings neuromuscular junctions normal mutant Drosophila larvae. J Neurogenet. <br />                                          2004; 18:377&#226;&#8364;&#8220;402. [PubMed: 15763995] <br />                                     21. Fischer U, Liu Q, Dreyfuss G. The SMN-SIP1 complex essential role spliceosomal <br />                                          snRNP biogenesis. Cell. 1997; 90:1023&#226;&#8364;&#8220;9. [PubMed: 9323130] <br />                                     22. Foust KD, Wang X, McGovern VL, Braun L, Bevan AK, Haidet AM, Le TT, Morales PR, Rich <br />                                          MM, Burghes AH, Kaspar BK. Rescue spinal muscular atrophy phenotype mouse <br />                                          model early postnatal delivery SMN. Nat Biotechnol. 2010; 28:271&#226;&#8364;&#8220;4. [PubMed: 20190738] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     23. Franciscovich AL, Mortimer AD, Freeman AA, Gu J, Sanyal S. Overexpression screen in <br />                                          Drosophila identifies neuronal roles GSK-3 beta/shaggy regulator AP-1-dependent <br />                                          developmental plasticity. Genetics. 2008; 180:2057&#226;&#8364;&#8220;71. [PubMed: 18832361] <br />                                     24. Frank CA, Kennedy MJ, Goold CP, Marek KW, Davis GW. Mechanisms underlying rapid <br />                                          induction sustained expression synaptic homeostasis. Neuron. 2006; 52:663&#226;&#8364;&#8220;77. [PubMed: <br />                                          17114050] <br />                                     25. Frank CA, Pielage J, Davis GW. A presynaptic homeostatic signaling composed Eph <br />                                          receptor, ephexin, Cdc42, CaV2.1 calcium channels. Neuron. 2009; 61:556&#226;&#8364;&#8220;69. [PubMed: <br />                                          19249276] <br />                                     26. Freeman A, Franciscovich A, Bowers M, Sandstrom DJ, Sanyal S. NFAT regulates pre-synaptic <br />                                          development activity-dependent plasticity Drosophila. Mol Cell Neurosci. 2011; 46:535&#226;&#8364;&#8220; <br />                                          47. [PubMed: 21185939] <br />                                     27. Freeman A, Pranski E, Miller RD, Radmard S, Bernhard D, Jinnah HA, Betarbet R, Rye DB, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                          Sanyal S. Sleep fragmentation motor restlessness Drosophila model restless legs <br />                                          syndrome. Curr Biol. 2012; 22:1142&#226;&#8364;&#8220;8. [PubMed: 22658601] <br />                                     28. Frugier T, Tiziano FD, Cifuentes-Diaz C, Miniou P, Roblot N, Dierich A, Le Meur M, Melki J. <br />                                          Nuclear targeting defect SMN lacking C-terminus mouse model spinal muscular <br />                                          atrophy. Hum Mol Genet. 2000; 9:849&#226;&#8364;&#8220;58. [PubMed: 10749994] <br />                                     29. Gabanella F, Carissimi C, Usiello A, Pellizzoni L. The activity spinal muscular atrophy <br />                                          protein regulated development cellular differentiation. Hum Mol Genet. 2005; <br />                                          14:3629&#226;&#8364;&#8220;42. [PubMed: 16236758] <br />                                     30. Gabanella F, Butchbach ME, Saieva L, Carissimi C, Burghes AH, Pellizzoni L. Ribonucleoprotein <br />                                          assembly defects correlate spinal muscular atrophy severity preferentially affect subset <br />                                          spliceosomal snRNPs. PLoS One. 2007; 2:e921. [PubMed: 17895963] <br />                                     31. Ghosh S, Feany MB. Comparison pathways controlling toxicity eye brain in <br />                                          Drosophila models human neurodegenerative diseases. Hum Mol Genet. 2004; 13:2011&#226;&#8364;&#8220;8. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                          [PubMed: 15254017] <br />                                     32. Gogliotti RG, Quinlan KA, Barlow CB, Heier CR, Heckman CJ, Didonato CJ. Motor neuron <br />                                          rescue spinal muscular atrophy mice demonstrates sensory-motor defects a <br />                                          consequence, cause, motor neuron dysfunction. J Neurosci. 2012; 32:3818&#226;&#8364;&#8220;29. [PubMed: <br />                                          22423102] <br />                                     33. Gotz KG. Visual guidance Drosophila. Basic Life Sci. 1980; 16:391&#226;&#8364;&#8220;407. [PubMed: 6779803] <br />                                     34. Haghighi AP, McCabe BD, Fetter RD, Palmer JE, Hom S, Goodman CS. Retrograde control of <br />                                          synaptic transmission postsynaptic CaMKII Drosophila neuromuscular junction. Neuron. <br />                                          2003; 39:255&#226;&#8364;&#8220;67. [PubMed: 12873383] <br />                                     35. Hammond SM, Gogliotti RG, Rao V, Beauvais A, Kothary R, DiDonato CJ. Mouse survival motor <br />                                          neuron alleles mimic SMN2 splicing inducible rescue embryonic lethality early in <br />                                          development late. PLoS One. 2010; 5:e15887. [PubMed: 21249120] <br />                                     36. Hsieh-Li HM, Chang JG, Jong YJ, Wu MH, Wang NM, Tsai CH, Li H. A mouse model spinal <br />                                          muscular atrophy. Nat Genet. 2000; 24:66&#226;&#8364;&#8220;70. [PubMed: 10615130] <br />                                     37. Imlach WL, Beck ES, Choi BJ, Lotti F, Pellizzoni L, McCabe BD. SMN required sensory- <br />                                          motor circuit function Drosophila. Cell. 2012; 151:427&#226;&#8364;&#8220;39. [PubMed: 23063130] <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                              Page 14 <br />  <br />  <br />                                     38. Jablonka S, Schrank B, Kralewski M, Rossoll W, Sendtner M. Reduced survival motor neuron <br />                                         (Smn) gene dose mice leads motor neuron degeneration: animal model spinal muscular <br />                                         atrophy type III. Hum Mol Genet. 2000; 9:341&#226;&#8364;&#8220;6. [PubMed: 10655542] <br />                                     39. Kariya S, Park GH, Maeno-Hikichi Y, Leykekhman O, Lutz C, Arkovitz MS, Landmesser LT, <br />                                         Monani UR. Reduced SMN protein impairs maturation neuromuscular junctions mouse <br />                                         models spinal muscular atrophy. Hum Mol Genet. 2008; 17:2552&#226;&#8364;&#8220;69. [PubMed: 18492800] <br />                                     40. Kim SM, Kumar V, Lin YQ, Karunanithi S, Ramaswami M. Fos Jun potentiate individual <br />                                         release sites mobilize reserve synaptic vesicle pool Drosophila larval motor synapse. <br />                                         Proc Natl Acad Sci U S A. 2009; 106:4000&#226;&#8364;&#8220;5. [PubMed: 19228945] <br />                                     41. Kitamoto T. Conditional modification behavior Drosophila targeted expression a <br />                                         temperature-sensitive shibire allele defined neurons. J Neurobiol. 2001; 47:81&#226;&#8364;&#8220;92. [PubMed: <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         11291099] <br />                                     42. Kong L, Wang X, Choe DW, Polley M, Burnett BG, Bosch-Marce M, Griffin JW, Rich MM, <br />                                         Sumner CJ. Impaired synaptic vesicle release immaturity neuromuscular junctions spinal <br />                                         muscular atrophy mice. J Neurosci. 2009; 29:842&#226;&#8364;&#8220;51. [PubMed: 19158308] <br />                                     43. Kroiss M, Schultz J, Wiesner J, Chari A, Sickmann A, Fischer U. Evolution RNP assembly <br />                                         system: minimal SMN complex facilitates formation UsnRNPs Drosophila melanogaster. <br />                                         Proc Natl Acad Sci U S A. 2008; 105:10045&#226;&#8364;&#8220;50. [PubMed: 18621711] <br />                                     44. Le TT, Pham LT, Butchbach ME, Zhang HL, Monani UR, Coovert DD, Gavrilina TO, Xing L, <br />                                         Bassell GJ, Burghes AH. SMNDelta7, major product centromeric survival motor neuron <br />                                         (SMN2) gene, extends survival mice spinal muscular atrophy associates full- <br />                                         length SMN. Hum Mol Genet. 2005; 14:845&#226;&#8364;&#8220;57. [PubMed: 15703193] <br />                                     45. Lee JB, Lee KA, Hong JM, Suh GI, Choi YC. Homozygous SMN2 deletion major risk factor <br />                                         twenty-five Korean sporadic amyotrophic lateral sclerosis patients. Yonsei Med J. 2012; <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         53:53&#226;&#8364;&#8220;7. [PubMed: 22187232] <br />                                     46. Lee L, Davies SE, Liu JL. The spinal muscular atrophy protein SMN affects Drosophila germline <br />                                         nuclear organization U body-P body pathway. Dev Biol. 2009; 332:142&#226;&#8364;&#8220;55. [PubMed: <br />                                         19464282] <br />                                     47. Lefebvre S, Burglen L, Reboullet S, Clermont O, Burlet P, Viollet L, Benichou B, Cruaud C, <br />                                         Millasseau P, Zeviani M, et al. Identification characterization spinal muscular atrophy- <br />                                         determining gene. Cell. 1995; 80:155&#226;&#8364;&#8220;65. [PubMed: 7813012] <br />                                     48. Lefebvre S, Burlet P, Liu Q, Bertrandy S, Clermont O, Munnich A, Dreyfuss G, Melki J. <br />                                         Correlation severity SMN protein level spinal muscular atrophy. Nat Genet. 1997; <br />                                         16:265&#226;&#8364;&#8220;9. [PubMed: 9207792] <br />                                     49. Lima SQ, Miesenbock G. Remote control behavior genetically targeted <br />                                         photostimulation neurons. Cell. 2005; 121:141&#226;&#8364;&#8220;52. [PubMed: 15820685] <br />                                     50. Ling KK, Lin MY, Zingg B, Feng Z, Ko CP. Synaptic defects spinal neuromuscular <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         circuitry mouse model spinal muscular atrophy. PLoS One. 2011; 5:e15457. [PubMed: <br />                                         21085654] <br />                                     51. Liu Q, Fischer U, Wang F, Dreyfuss G. The spinal muscular atrophy disease gene product, SMN, <br />                                         associated protein SIP1 complex spliceosomal snRNP proteins. Cell. 1997; <br />                                         90:1013&#226;&#8364;&#8220;21. [PubMed: 9323129] <br />                                     52. Lorson CL, Hahnen E, Androphy EJ, Wirth B. A single nucleotide SMN gene regulates <br />                                         splicing responsible spinal muscular atrophy. Proc Natl Acad Sci U S A. 1999; 96:6307&#226;&#8364;&#8220; <br />                                         11. [PubMed: 10339583] <br />                                     53. Lorson CL, Androphy EJ. An exonic enhancer required inclusion essential exon the <br />                                         SMA-determining gene SMN. Hum Mol Genet. 2000; 9:259&#226;&#8364;&#8220;65. [PubMed: 10607836] <br />                                     54. Lotti F, Imlach WL, Saieva L, Beck ES, Hao Le T, Li DK, Jiao W, Mentis GZ, Beattie CE, <br />                                         McCabe BD, Pellizzoni L. An SMN-dependent U12 splicing event essential motor circuit <br />                                         function. Cell. 2012; 151:440&#226;&#8364;&#8220;54. [PubMed: 23063131] <br />                                     55. Luo L, Liao YJ, Jan LY, Jan YN. Distinct morphogenetic functions similar small GTPases: <br />                                         Drosophila Drac1 involved axonal outgrowth myoblast fusion. Genes Dev. 1994; <br />                                         8:1787&#226;&#8364;&#8220;802. [PubMed: 7958857] <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                            Page 15 <br />  <br />  <br />                                     56. Lutz CM, Kariya S, Patruni S, Osborne MA, Liu D, Henderson CE, Li DK, Pellizzoni L, Rojas J, <br />                                         Valenzuela DM, Murphy AJ, Winberg ML, Monani UR. Postsymptomatic restoration SMN <br />                                         rescues disease phenotype mouse model severe spinal muscular atrophy. J Clin Invest. <br />                                         2011; 121:3029&#226;&#8364;&#8220;41. [PubMed: 21785219] <br />                                     57. Mahr A, Aberle H. The expression <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Drosophila vesicular glutamate transporter: a <br />                                         marker protein motoneurons glutamatergic centers brain. Gene Expr Patterns. 2006; <br />                                         6:299&#226;&#8364;&#8220;309. [PubMed: 16378756] <br />                                     58. Mailman MD, Heinz JW, Papp AC, Snyder PJ, Sedra MS, Wirth B, Burghes AH, Prior TW. <br />                                         Molecular analysis spinal muscular atrophy modification phenotype SMN2. Genet <br />                                         Med. 2002; 4:20&#226;&#8364;&#8220;6. [PubMed: 11839954] <br />                                     59. Marrus SB, Portman SL, Allen MJ, Moffat KG, DiAntonio A. Differential localization of <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         glutamate receptor subunits Drosophila neuromuscular junction. J Neurosci. 2004; 24:1406&#226;&#8364;&#8220; <br />                                         15. [PubMed: 14960613] <br />                                     60. Martinez TL, Kong L, Wang X, Osborne MA, Crowder ME, Van Meerbeke JP, Xu X, Davis C, <br />                                         Wooley J, Goldhamer DJ, Lutz CM, Rich MM, Sumner CJ. Survival Motor Neuron Protein in <br />                                         Motor Neurons Determines Synaptic Integrity Spinal Muscular Atrophy. J Neurosci. 2012; <br />                                         32:8703&#226;&#8364;&#8220;8715. [PubMed: 22723710] <br />                                     61. McAndrew PE, Parsons DW, Simard LR, Rochette C, Ray PN, Mendell JR, Prior TW, Burghes <br />                                         AH. Identification proximal spinal muscular atrophy carriers patients analysis SMNT <br />                                         SMNC gene copy number. Am J Hum Genet. 1997; 60:1411&#226;&#8364;&#8220;22. [PubMed: 9199562] <br />                                     62. McLachlan EM, Martin AR. Non-linear summation end-plate potentials frog mouse. J <br />                                         Physiol. 1981; 311:307&#226;&#8364;&#8220;24. [PubMed: 6267255] <br />                                     63. Meister G, Hannus S, Plottner O, Baars T, Hartmann E, Fakan S, Laggerbauer B, Fischer U. <br />                                         SMNrp essential pre-mRNA splicing factor required formation mature <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         spliceosome. EMBO J. 2001; 20:2304&#226;&#8364;&#8220;14. [PubMed: 11331595] <br />                                     64. Michaud M, Arnoux T, Bielli S, Durand E, Rotrou Y, Jablonka S, Robert F, Giraudon-Paoli M, <br />                                         Riessland M, Mattei MG, Andriambeloson E, Wirth B, Sendtner M, Gallego J, Pruss RM, Bordet <br />                                         T. Neuromuscular defects breathing disorders new mouse model spinal muscular <br />                                         atrophy. Neurobiol Dis. 2010; 38:125&#226;&#8364;&#8220;35. [PubMed: 20085811] <br />                                     65. Monani UR, Lorson CL, Parsons DW, Prior TW, Androphy EJ, Burghes AH, McPherson JD. A <br />                                         single nucleotide difference alters splicing patterns distinguishes SMA gene SMN1 from <br />                                         copy gene SMN2. Hum Mol Genet. 1999; 8:1177&#226;&#8364;&#8220;83. [PubMed: 10369862] <br />                                     66. Monani UR, Sendtner M, Coovert DD, Parsons DW, Andreassi C, Le TT, Jablonka S, Schrank B, <br />                                         Rossoll W, Prior TW, Morris GE, Burghes AH. The human centromeric survival motor neuron <br />                                         gene (SMN2) rescues embryonic lethality Smn(&#226;&#710;&#8217;/&#226;&#710;&#8217;) mice results mouse spinal <br />                                         muscular atrophy. Hum Mol Genet. 2000; 9:333&#226;&#8364;&#8220;9. [PubMed: 10655541] <br />                                     67. Murray LM, Lee S, Baumer D, Parson SH, Talbot K, Gillingwater TH. Pre-symptomatic <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         development lower motor neuron connectivity mouse model severe spinal muscular <br />                                         atrophy. Hum Mol Genet. 2010; 19:420&#226;&#8364;&#8220;33. [PubMed: 19884170] <br />                                     68. Murray LM, Comley LH, Thomson D, Parkinson N, Talbot K, Gillingwater TH. Selective <br />                                         vulnerability motor neurons dissociation pre- post-synaptic pathology the <br />                                         neuromuscular junction mouse models spinal muscular atrophy. Hum Mol Genet. 2008; <br />                                         17:949&#226;&#8364;&#8220;62. [PubMed: 18065780] <br />                                     69. Narver HL, Kong L, Burnett BG, Choe DW, Bosch-Marce M, Taye AA, Eckhaus MA, Sumner CJ. <br />                                         Sustained improvement spinal muscular atrophy mice treated trichostatin A plus nutrition. <br />                                         Ann Neurol. 2008; 64:465&#226;&#8364;&#8220;70. [PubMed: 18661558] <br />                                     70. Neuser K, Triphan T, Mronz M, Poeck B, Strauss R. Analysis spatial orientation memory in <br />                                         Drosophila. Nature. 2008; 453:1244&#226;&#8364;&#8220;7. [PubMed: 18509336] <br />                                     71. Nicole S, Desforges B, Millet G, Lesbordes J, Cifuentes-Diaz C, Vertes D, Cao ML, De Backer F, <br />                                         Languille L, Roblot N, Joshi V, Gillis JM, Melki J. Intact satellite cells lead remarkable <br />                                         protection Smn gene defect differentiated skeletal muscle. J Cell Biol. 2003; 161:571&#226;&#8364;&#8220; <br />                                         82. [PubMed: 12743106] <br />                                     72. Park GH, Maeno-Hikichi Y, Awano T, Landmesser LT, Monani UR. Reduced survival motor <br />                                         neuron (SMN) protein motor neuronal progenitors functions cell autonomously cause spinal <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                            Page 16 <br />  <br />  <br />                                         muscular atrophy model mice expressing human centromeric (SMN2) gene. J Neurosci. <br />                                         2010; 30:12005&#226;&#8364;&#8220;19. [PubMed: 20826664] <br />                                     73. Pearn J. Incidence, prevalence, gene frequency studies chronic childhood spinal muscular <br />                                         atrophy. J Med Genet. 1978; 15:409&#226;&#8364;&#8220;13. [PubMed: 745211] <br />                                     74. Pellizzoni L, Charroux B, Dreyfuss G. SMN mutants spinal muscular atrophy patients are <br />                                         defective binding snRNP proteins. Proc Natl Acad Sci U S A. 1999; 96:11167&#226;&#8364;&#8220;72. [PubMed: <br />                                         10500148] <br />                                     75. Pellizzoni L, Yong J, Dreyfuss G. Essential role SMN complex specificity snRNP <br />                                         assembly. Science. 2002; 298:1775&#226;&#8364;&#8220;9. [PubMed: 12459587] <br />                                     76. Penney J, Tsurudome K, Liao EH, Elazzouzi F, Livingstone M, Gonzalez M, Sonenberg N, <br />                                         Haghighi AP. TOR required retrograde regulation synaptic homeostasis the <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         Drosophila neuromuscular junction. Neuron. 2012; 74:166&#226;&#8364;&#8220;78. [PubMed: 22500638] <br />                                     77. Petersen SA, Fetter RD, Noordermeer JN, Goodman CS, DiAntonio A. Genetic analysis of <br />                                         glutamate receptors Drosophila reveals retrograde signal regulating presynaptic transmitter <br />                                         release. Neuron. 1997; 19:1237&#226;&#8364;&#8220;48. [PubMed: 9427247] <br />                                     78. Praveen K, Wen Y, Matera AG. A Drosophila Model Spinal Muscular Atrophy Uncouples <br />                                         snRNP Biogenesis Functions Survival Motor Neuron Locomotion Viability Defects. <br />                                         Cell Rep. 2012; 1:624&#226;&#8364;&#8220;31. [PubMed: 22813737] <br />                                     79. Rajendra TK, Gonsalvez GB, Walker MP, Shpargel KB, Salz HK, Matera AG. A Drosophila <br />                                         melanogaster model spinal muscular atrophy reveals function SMN striated muscle. J <br />                                         Cell Biol. 2007; 176:831&#226;&#8364;&#8220;41. [PubMed: 17353360] <br />                                     80. Ruiz R, Casanas JJ, Torres-Benito L, Cano R, Tabares L. Altered intracellular Ca2+ homeostasis in <br />                                         nerve terminals severe spinal muscular atrophy mice. J Neurosci. 2010; 30:849&#226;&#8364;&#8220;57. [PubMed: <br />                                         20089893] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     81. Sakai T, Kasuya J, Kitamoto T, Aigaki T. The Drosophila TRPA channel, Painless, regulates <br />                                         sexual receptivity virgin females. Genes Brain Behav. 2009; 8:546&#226;&#8364;&#8220;57. [PubMed: 19531155] <br />                                     82. Salvaterra PM, Kitamoto T. Drosophila cholinergic neurons processes visualized Gal4/ <br />                                         UAS-GFP. Brain Res Gene Expr Patterns. 2001; 1:73&#226;&#8364;&#8220;82. [PubMed: 15018821] <br />                                     83. Seabrooke S, Stewart BA. Synaptic transmission plasticity modulated nonmuscle <br />                                         myosin II neuromuscular junction Drosophila. J Neurophysiol. 2011; 105:1966&#226;&#8364;&#8220;76. <br />                                         [PubMed: 21325687] <br />                                     84. Sen A, Yokokura T, Kankel MW, Dimlich DN, Manent J, Sanyal S, Artavanis-Tsakonas S. <br />                                         Modeling spinal muscular atrophy Drosophila links Smn FGF signaling. J Cell Biol. 2011; <br />                                         192:481&#226;&#8364;&#8220;95. [PubMed: 21300852] <br />                                     85. Shpargel KB, Matera AG. Gemin proteins required efficient assembly Sm-class <br />                                         ribonucleoproteins. Proc Natl Acad Sci U S A. 2005; 102:17372&#226;&#8364;&#8220;7. [PubMed: 16301532] <br />                                     86. Shpargel KB, Praveen K, Rajendra TK, Matera AG. Gemin3 essential gene required larval <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                         motor function pupation Drosophila. Mol Biol Cell. 2009; 20:90&#226;&#8364;&#8220;101. [PubMed: 18923150] <br />                                     87. Sleigh JN, Buckingham SD, Esmaeili B, Viswanathan M, Cuppen E, Westlund BM, Sattelle DB. A <br />                                         novel Caenorhabditis elegans allele, smn-1(cb131), mimicking mild form spinal muscular <br />                                         atrophy, provides convenient drug screening platform highlighting new pre-approved <br />                                         compounds. Hum Mol Genet. 2011; 20:245&#226;&#8364;&#8220;60. [PubMed: 20962036] <br />                                     88. Sleigh JN, Gillingwater TH, Talbot K. The contribution mouse models understanding the <br />                                         pathogenesis spinal muscular atrophy. Dis Model Mech. 2011; 4:457&#226;&#8364;&#8220;67. [PubMed: 21708901] <br />                                     89. Stewart BA, Atwood HL, Renger JJ, Wang J, Wu CF. Improved stability Drosophila larval <br />                                         neuromuscular preparations haemolymph-like physiological solutions. J Comp Physiol A. 1994; <br />                                         175:179&#226;&#8364;&#8220;91. [PubMed: 8071894] <br />                                     90. Vitte JM, Davoult B, Roblot N, Mayer M, Joshi V, Courageot S, Tronche F, Vadrot J, Moreau <br />                                         MH, Kemeny F, Melki J. Deletion murine Smn exon 7 directed liver leads severe defect of <br />                                         liver development associated iron overload. Am J Pathol. 2004; 165:1731&#226;&#8364;&#8220;41. [PubMed: <br />                                         15509541] <br />                                     91. Wan L, Battle DJ, Yong J, Gubitz AK, Kolb SJ, Wang J, Dreyfuss G. The survival motor <br />                                         neurons protein determines capacity snRNP assembly: biochemical deficiency spinal <br />                                         muscular atrophy. Mol Cell Biol. 2005; 25:5543&#226;&#8364;&#8220;51. [PubMed: 15964810] <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                  Page 17 <br />  <br />  <br />  <br />                                          Highlights <br />  <br />                                          &#226;&#8364;&#162;    Knock dSmn nervous impairs locomotion <br />                                          &#226;&#8364;&#162;    Knock dSmn muscle does impair locomotion <br />                                          &#226;&#8364;&#162;    Synaptic transmission normal following neuronal muscle knock of <br />                                               dSmn <br />                                          &#226;&#8364;&#162;    Homeostasis synaptic transmission impaired neuronal dSmn knock <br />                                               down <br /> $watermark-text <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                     Page 18 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Figure 1. Experimental paradigm test tissue-specific requirement SMN maintenance <br />                                     normal motor physiology <br />                                     The experimental strategy test compartment-specific functions SMN outlined <br />                                     schematically. SMN expressed widely tissues (grey outer box). In study, <br />                                     SMN over-expressed (black) neurons muscles, reduced <br />                                     RNAi mediated knockdown (lighter shades grey reflected moderate strong <br />                                     knockdown). SMN levels tissues presumed intact comparable wild <br />                                     type control animals. Experimental animals subjected behavioral assays to <br />                                     measure motor function adults. Electrophysiological measurements used evaluate <br />                                     various aspects synaptic function instar larval NMJ. In figures, circles <br />                                     represent neuronal soma, triangles represent pre-synaptic compartment small <br />                                     rectangles represent post-synaptic muscle. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                    Page 19 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Figure 2. Perturbation SMN nervous disrupts normal motor behavior <br />                                     (A) Various parameters locomotion measured Buridan&apos;s assay shown for <br />                                     neuronal perturbations SMN using elavC155-GAL4 driver line. SMN knockdown <br />                                     using RNAi lines results decreased speed, distance covered overall activity, while <br />                                     number pauses increased significantly affecting mean pause length. <br />                                     Western blot adult heads shows strong over-expression (SMN-OE) significant but <br />                                     complete knock (SMN-RNAi) SMN protein levels adult heads (a <br />                                     predominantly neuronal population). (B) Single representative fly traces (left circles) and <br />                                     cumulative occupancy plots (right circles heat maps) disrupted locomotor <br />                                     behavior following pan-neuronal knockdown SMN. (C) Muscle perturbation SMN <br />                                     using how24B-GAL4 driver line. The number pauses significantly increased <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     following over-expression knockdown SMN mesoderm. Western blot of <br />                                     protein extract adult thorax shows strong over-expression (SMN-OE) significant <br />                                     complete knock SMN muscle tissue (SMN-RNAi). (D) Single fly tracks <br />                                     cumulative occupancy plots mild impairment locomotion. Note muscle <br />                                     expression SMN-RNAi-N4 pupal lethal reported previously. 10 individual animals <br />                                     5&#226;&#8364;&#8220;6 days old tested genotype. Error bars SEM, double asterisks <br />                                     represent p&lt;0.01 tested ANOVA. <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                   Page 20 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Figure 3. SMN reduction motor neurons sufficient impair motor function <br />                                     (A, B) Parameters locomotion, single fly tracks occupancy plots animals with <br />                                     glutamatergic neuron specific perturbations SMN using VGlut-GAL4 driver (OK371). <br />                                     RNAi mediated knockdown SMN glutamatergic neurons recapitulates large the <br />                                     effect pan-neuronal knockdown. Distance activity reduced, total number <br />                                     pauses increased. SMN over-expression does discernible influence on <br />                                     locomotion. (C, D) Results Buridan&apos;s assay cholinergic neuron specific <br />                                     perturbations SMN. None parameters locomotion significantly affected by <br />                                     altering SMN function cholinergic neurons. 10 animals tested genotype, error <br />                                     bars SEM double asterisks represent p&lt;0.01 determined ANOVA. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                      Page 21 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Figure 4. Baseline synaptic transmission largely normal conditions neuronal SMN <br />                                     modulation <br />                                     (A, B) Representative traces (A) quantification baseline synaptic transmission the <br />                                     larval NMJ (muscle 6/7 synapse abdominal segment A2) following pan-neuronal SMN <br />                                     perturbation (B). mEJP amplitude (quantal size) significantly different SMN over- <br />                                     expressing synapses. However, EJP amplitudes quantal content remain unchanged. 9&#226;&#8364;&#8220;10 <br />                                     animals tested genotype. Vertical scale bar = 10 mV EJP 5 mV for <br />                                     mEJP; Horizontal scale bar = 100 ms EJP 200 ms mEJP. Mean resting membrane <br />                                     potentials are: Control = 73mV, UAS-SMN = 74mV, SMN-RNAi-C24 = 69mV, SMN- <br />                                     RNAi-N4 = 68.5mV. Mean input resistance values are: Control = 12.5M&#206;&#169;, UAS-SMN = <br />                                     12M&#206;&#169;, SMN-RNAi-C24 = 13.3M&#206;&#169;, SMN-RNAi-N4 = 12.4M&#206;&#169;. (C) High-frequency (10 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Hz) stimulation high-calcium (10 mM) containing Ringer&apos;s solution leads gradual <br />                                     synaptic depression approximately 50% initial EJP amplitude, followed almost <br />                                     instantaneous recovery low-frequency (0.1 Hz) stimulation. This profile unchanged <br />                                     SMN knocked pan-neuronally using N4 RNAi transgene (red spheres) as <br />                                     compared control animals (black spheres). Errors bars SEM. 5 animals tested for <br />                                     genotype. Horizontal scale bar = 1min, vertical scale bar = 50mV. <br />  <br />  <br />  <br />  <br />                                          Brain Res. Author manuscript; available PMC 2013 December 13. <br />                    Timmerman Sanyal                                                                                      Page 22 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                     Figure 5. SMN knockdown muscle affects quantal size pre-synaptic transmitter release <br />                                     (A, B) Representative traces quantification baseline synaptic transmission the <br />                                     instar larval NMJ following muscle-specific perturbation SMN. Both over- <br />                    <br /> </body></html>