<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, July 2011, p. 985&#226;&#8364;&#8220;997                                                                                           Vol. 10, No. 7 <br /> 1535-9778/11/$12.00 doi:10.1128/EC.05025-11 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />     Characterization, Localization, Essentiality, High-Resolution <br />      Crystal Structure Glucosamine 6-Phosphate N-Acetyltransferase <br />                         Trypanosoma brucei&#228;&#338;&#164;&#226;&#8364;&#161;&#194;&#167; <br />                             &#203;&#339;o,1&#226;&#8364;&#160; M. Lucia Sampaio Gu <br />                  Karina Marin                       &#194;&#168;ther,1 Amy K. Wernimont,2 Wei Qiu,2 <br />                                 Raymond Hui, Michael A. J. Ferguson1* <br />                                              2 <br />  <br />       Division Biological Chemistry Drug Discovery, College Life Sciences, University Dundee, Dundee DD1 5EH, <br />                 United Kingdom,1 Structural Genomics Consortium, University Toronto, MaRS South Tower, <br />                                  7th Floor, 101 College St., Toronto, Ontario, Canada M5G 1L72 <br />                                                Received 7 March 2011/Accepted 25 April 2011 <br />  <br />              A gene predicted encode Trypanosoma brucei glucosamine 6-phosphate N-acetyltransferase (TbGNA1; EC <br />           2.3.1.4) cloned expressed Escherichia coli. The recombinant protein enzymatically active, and <br />           high-resolution crystal structure obtained 1.86 &#195;&#8230;. Endogenous TbGNA1 protein localized the <br />           peroxisome-like microbody, glycosome. A bloodstream-form T. brucei GNA1 conditional null mutant was <br />           constructed shown unable sustain growth vitro nonpermissive conditions, demonstrating <br />           metabolic nutritional routes UDP-GlcNAc GlcNAc-6-phosphate. Analysis <br />           protein glycosylation <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> TbGNA1 mutant nonpermissive conditions revealed that <br />           poly-N-acetyllactosamine structures greatly reduced parasite glycosylation profile of <br />           principal parasite surface coat component, variant surface glycoprotein (VSG), modified. The <br />           significance results potential TbGNA1 novel drug target African sleeping sickness are <br />           discussed. <br />  <br />  <br />    The African trypanosomes protozoan parasites                      include ESAG6/ESAG7 heterodimeric transferrin recep- <br /> cause human sleeping sickness nagana cattle,                   tor (45), major lysosomal glycoprotein p67 (1, 38), the <br /> which fatal diseases untreated. The current                   Trypanosoma brucei membrane-bound histidine acid phospha- <br /> drugs highly toxic; example, melarsoprol, front-line              tase MBAP1 (TbMBAP1) (13). All molecules contain <br /> drug sleeping sickness central nervous involve-               mannose, galactose, N-acetylglucosamine (GlcNAc) resi- <br /> ment, kills 5% patients difficult administer. With               dues. The GlcNAc sugar present N-acetylchitobiosyl <br /> the exception eflornithine, unsuccessful anticancer drug                cores glycoprotein N-linked glycans; glucosamine <br /> that registered treatment human African trypano-                   (GlcN), derived GlcNAc de-N-acetylation (11), is <br /> somiasis 1990, new drug registered clinical                present protein-linked free GPI structures. N-Acetyl- <br /> use 50 years.                                                         glucosamine N-acetyllactosamine (LacNAc) <br />    In mammalian bloodstream form (BSF) parasite,                   units Gal&#226;?&#164;(1-4)GlcNAc decorate conventional complex <br /> a dense coat 5 &#226;&#171;&#187; 106 glycosylphosphatidylinositol (GPI)-                   N-linked glycans (28) appear unusual giant poly- <br /> anchored variant surface glycoprotein (VSG) homodimers                        LacNAc-containing N-linked glycans flagellar <br /> decorates plasma membrane. The parasite survives                      pocket endosomal/lysosomal bloodstream <br /> immune attack vertebrate host antigenic varia-                  form parasite (3). LacNAc repeats lacto-N-biose <br /> tion, process involves replacement VSG coat                  repeats Gal&#226;?&#164;(1-3)GlcNAc appear chains the <br /> another composed antigenically different VSG molecules                     procyclin GPI anchor free GPIs procyclic form of <br /> (7). In addition VSG, parasite expresses number                 organism (35, 42, 48). <br /> of abundant glycoproteins, specific                    Sugar nucleotides used glycosyltransferases donor <br /> and essential infectious bloodstream form. Examples                   substrates glycosylation reactions (15). They synthesized <br />                                                                               cytoplasm used and/or transported through <br />                                                                               specific transporters lumen Golgi apparatus <br />    * Corresponding author. Mailing address: Division Biological            (and/or endoplasmic reticulum [ER]). Sugar nucleotides may <br /> Chemistry Drug Discovery, College Life Sciences, University         formed novo pathway salvage pathways, the <br /> Dundee, Dundee DD1 5EH, United Kingdom. Phone: 44 (0)1382384219.              involving activation sugar using kinase a <br /> Fax: 44 (0)1382348896. E-mail: m.a.j.ferguson@dundee.ac.uk. <br />    &#226;&#8364;&#160; Present address: Dublin-Oxford Glycobiology Group, National In-          pyrophosphorylase. The source GlcNAc eukaryotic gly- <br /> stitute Bioprocessing Training (NIBRT), UCD Conway Insti-              coconjugates sugar nucleotide UDP-GlcNAc. The se- <br /> tute, University College Dublin, Dublin, Ireland.                             quence reactions eukaryotic novo pathway is <br />    &#226;&#8364;&#161; Supplemental material article http://ec         represented Fig. 1, starting fructose-6-phosphate (F- <br /> .asm.org/. <br />    &#228;&#338;&#164; <br />      Published ahead print 29 April 2011. <br />                                                                               6P). Two alternative salvage routes exist. In case, <br />    &#194;&#167; The authors paid fee allow immediate free access            GlcN GlcNAc taken cell medium (or <br /> this article.                                                                 degradation oligosaccharide moieties) phosphory- <br />  <br />                                                                         985 <br />  986           &#203;&#339; O ET AL. <br />           MARIN                                                                                                                                   EUKARYOT. CELL <br />  <br />                                                                                     GCCTCGAGCTACAAATCCAGCCTCATTTGTCT-3&#226;&#172;&#732;. The products three <br />                                                                                     separate PCRs cloned BamHI/XhoI site pGEX 6P-1 (GE <br />                                                                                     Healthcare), representative clone PCR sequenced both <br />                                                                                     directions. The TbGNA1 ORF sequence deposited EMBL <br />                                                                                     nucleotide sequence database (accession number FR715998). Oligonucleotides <br />                                                                                     obtained University Dundee oligonucleotide facility. All plas- <br />                                                                                     mids verified sequencing (DNA Sequencing Service, College Life <br />                                                                                     Sciences, University Dundee). <br />                                                                                        Reverse transcription-PCR (RT-PCR). RNA extracted using RNeasy ex- <br />                                                                                     traction kits on-column DNase digestion (RNase-free DNase; Qiagen). <br />                                                                                     RNA samples, 50 ng, treated Omniscript reverse transcriptase (Qia- <br />                                                                                     gen) generate cDNA. The cDNAs amplified PCR using Taq <br />                                                                                     polymerase TbGNA1 ORF primers (forward, 5&#226;&#172;&#732;-ATGACCGATATAGTG <br />                                                                                     GACCTGGA-3&#226;&#172;&#732;; reverse, 5&#226;&#172;&#732;-CTACAAATCCAGCCTCATTTGTCT-3&#226;&#172;&#732;) and <br />    FIG. 1. UDP-GlcNAc biosynthetic pathway eukaryotes. Abbre-                    Dol-P-Man synthetase (DPMS) primers (forward, 5&#226;&#172;&#732;-AATGGATGCGGACCT <br /> viations: GFA1, glucosamine 6-phosphate synthetase (EC 2.6.1.16);                   TCAGCACCCAC-3&#226;&#172;&#732;; reverse, 5&#226;&#172;&#732;-TAGAACCGTGAGCGCGGTGCCATAC- <br /> PAGM, phosphoacetylglucosamine mutase (EC 5.4.2.3).                                 3&#226;&#172;&#732;) constitutively expressed positive control. <br />                                                                                        Southern blotting. Genomic DNA prepared 1 &#226;&#171;&#187; 108 2 &#226;&#171;&#187; 108 cells <br />                                                                                     using DNAzol (Helena Biosciences). The hygromycin phosphotransferase resis- <br />                                                                                     tance gene (HYG), puromycin acetyltransferase resistance gene (PAC), <br /> lated (19) generate GlcN-6-phosphate (GlcN-6P)                               TbGNA1, TbGNA1 5&#226;&#172;&#732; untranslated region (UTR) probes amplified, <br /> GlcNAc-6-phosphate (GlcNAc-6P), respectively,                         purified, 32P labeled Amersham Rediprime II random prime labeling <br />                                                                                     (GE Healthcare), used probe diverse digests 3 &#226;?&#174;g T. <br /> feeds biosynthetic pathway. This salvage pathway been <br />                                                                                     brucei genomic DNA. <br /> confirmed biosynthetic radiolabeling T. brucei glycopro-                         Construction TbGNA1 conditional null mutant (TbGNA1cKO). The gene <br /> teins [3H]GlcN (14), presumably action hexoki-                      replacement cassettes generated PCR amplification 500 bp UTR <br /> nase (GlcN 3 GlcN-6P). As T. brucei unable interconvert                       immediately flanking 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends TbGNA1 ORF Platinum Taq <br /> UDP-GalNAc UDP-GlcNAc (42), second salvage route,                            High Fidelity DNA polymerase (Invitrogen) using forward primers 5&#226;&#172;&#732;-CGTTG <br />                                                                                     TCGACCATTTGGTCAAGGTGCTGGT-3&#226;&#172;&#732; (SalI site, boldface) 5&#226;&#172;&#732;-GTTT <br /> starting galactosamine (GalN), discounted (Fig.                         AAACTTACGGACCGTCAAGCTTTGTATAGCTTCACAACTTTCG-3&#226;&#172;&#732; and <br /> 1). The novo pathway glucose probably                           reverse primers 5&#226;&#172;&#732;-gacggtccgtaagtttaaacGGATCCttttttttgtctgcattttcgat-3&#226;&#172;&#732; and <br /> important vivo, T. brucei does GlcNAc (4)                      5&#226;&#172;&#732;-ATAAGTAAGCGGCCGCTAGCATACAGCACCTGTTAC-3&#226;&#172;&#732; (NotI site, <br /> and free GlcN abundant sugar mammals                         boldface). The PCR products used separate PCR produce <br />                                                                                     construct containing 5&#226;&#172;&#732; UTR linked 3&#226;&#172;&#732; UTR short HindIII, PmeI <br /> insects. <br />                                                                                     (italics), BamHI (capital letters) cloning site. The resulting DNA fragment <br />    The biosynthesis sugar nucleotides                           ligated pGEM-5Zf(&#226;&#171;&#185;) vector (Promega) using SalI/NotI sites. <br /> shown essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span> infectivity                       Antibiotic resistance markers cloned HindIII/BamHI restriction <br /> trypanosomatid parasites. Thus, UDP-galactose essential                       sites UTRs produce constructs, containing PAC and <br /> both bloodstream procyclic forms T. brucei ap-                         containing HYG. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> generate tetracycline-inducible ectopic copy of <br />                                                                                     TbGNA1 ORF, primers 5&#226;&#172;&#732;-GTTTAAACTTACGGACCGTCAAGCTTATG <br /> pears essential T. cruzi (25, 42, 43), GDP-mannose                      ACCGATATAGTGGACCTGGA-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CGCGGATCCCTACAAATCCA <br /> essential infectivity Leishmania major survival T.                    GCCTCATTTGTCT-3&#226;&#172;&#732; used PCR amplify ORF, cloned <br /> brucei (9, 10), UDP-GlcNAc essential L. major                            vector pLew100 using HindIII BamHI sites, respectively (italics). <br /> pathogenesis mammalian host (34) growth                          These constructs purified using Qiagen Maxiprep kit, double digested <br />                                                                                     SalI/NotI linearize DNA, precipitated, washed twice 70% eth- <br /> survival bloodstream-form T. brucei (46). <br />                                                                                     anol, redissolved sterile water. The linearized DNA electroporated <br />    The glucosamine 6-phosphate N-acetyltransferase (GNA1)                           T. brucei bloodstream cells (strain 427, variant 221) stably trans- <br /> enzymes (EC 2.3.1.4; Fig. 1) catalyze acetylation                            formed express T7 RNA polymerase tetracycline repressor protein <br /> GlcN-6P using cofactor acetyl coenzyme A (acetyl-CoA)                           G418 selection. This cell line referred &#226;&#8364;&#339;wild type&#226;&#8364;? here <br /> and characterized eukaryotes, including                        on. Cell culture, transformation, selection carried previously <br />                                                                                     described (55). <br /> Mus musculus Saccharomyces cerevisiae (5, 32). The crystal <br />                                                                                        Protein expression purification. Expression performed using Esche- <br /> structures S. cerevisiae, Aspergillus fumigatus, human                   richia coli ArcticExpress RP(DE3) cells (Stratagene) transformed the <br /> enzymes solved (20, 39, 53). GNA1                           pGEX 6P-1 vector containing TbGNA1 ORF. The cells grown 30&#194;&#176;C <br /> considered potential antifungal drug target Candida                         optical density reached 1.2, isopropyl &#226;?&#164;-D-1-thiogalactopy- <br /> albicans GNA1 mutants reduced virulence mice (31).                          ranoside (IPTG) added final concentration 0.025 mM. The cell <br />                                                                                     culture grown 48 h 12&#194;&#176;C. Cells harvested; washed 50 mM <br /> Here, present biochemical structural properties, sub-                        Tris-HCl, pH 7.5&#226;&#8364;&#8220;0.25 M NaCl (buffer A), 1 mg/ml lysozyme, complete <br /> cellular localization, drug target genetic validation T.                     protease inhibitor cocktail tablets (Roche); lysed French press. The <br /> brucei GNA1 (TbGNA1).                                                               lysate cleared centrifugation (20,000 &#226;&#171;&#187; g, 30 min, 4&#194;&#176;C), superna- <br />                                                                                     tant containing N-terminally glutathione S-transferase (GST)-tagged TbGNA1 <br />                                                                                     protein incubated 4&#194;&#176;C 2 h glutathione-Sepharose 4B beads (GE <br />                         MATERIALS AND METHODS <br />                                                                                     Healthcare) previously equilibrated buffer A. After washing buffer A, <br />    Parasite culture. Strain 427, variant MITat1.2 (also known variant 221), T.   GST tag cleaved situ using PreScission protease (a kind gift Daan <br /> brucei bloodstream-form parasites express T7 polymerase tetracycline       Van Aalten, University Dundee) buffer A 1 mM dithiothreitol (DTT) <br /> repressor protein G418 selection cultured HMI-9 medium (55)        16 h 4&#194;&#176;C. The sample purified gel filtration using Superdex <br /> to density &#226;&#172;&#402;2 &#226;&#171;&#187; 106 cells/ml 37&#194;&#176;C 5% CO2.                              200 HR30 high-performance liquid chromatography (HPLC) column (Amer- <br />    Cloning sequencing TbGNA1. The putative TbGNA1 open reading               sham Biosciences). Proteins eluted 0.5 ml/min buffer A monitored <br /> frame (ORF) identified T. brucei genome database (GeneDB accession           measurement absorbance 280 nm. Fractions (2 ml) collected, <br /> number Tb11.01.2886) amplified PCR genomic DNA Platinum            presence TbGNA1 peak fractions verified SDS-PAGE. The <br /> Taq High Fidelity DNA polymerase (Invitrogen) using forward primer con-           protein concentrated using Vivaspin concentrator (Viva- <br /> taining BamHI site (underlined), 5&#226;&#172;&#732;-CGCGGATCCATGACCGATATAGTG                      science) 3,500-kDa-molecular-mass cutoff 4&#194;&#176;C. TbGNA1 stored at <br /> GACCTGGA-3&#226;&#172;&#732;, reverse primer containing XhoI site (boldface), 5&#226;&#172;&#732;-C          &#226;&#171;&#186;80% buffer A&#226;&#8364;&#8220;10% glycerol. For raising antibodies (see below), GST- <br />  VOL. 10, 2011                                                      T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                                                     987 <br />  <br /> TbGNA1 eluted glutathione-Sepharose 4B beads buffer A                         TABLE 1. Crystallographic parameters TbGNA1a <br /> containing 10 mM glutathione. Recombinant protein identity performed by <br />                                                                                                      Parameter                                            Value(s) TbGNA1b <br /> the Proteomics Mass Spectrometry Facility, College Life Sciences, Uni- <br /> versity Dundee.                                                                    Data collection statistics <br />    Recombinant TbGNA1 (1 mg/ml buffer A) analyzed sedimentation              Space group ...................................................P212121 <br /> velocity centrifugation using Beckman Optima XL-1 analytical ultracentrifuge         <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> cell dimensions <br /> with AN50-Ti rotor 32,000 rpm 20&#194;&#176;C. Absorbance data (72 scans 280             (&#195;&#8230;) ...........................................................50.86 <br /> nm) collected analyzed using SEDFIT program (44). TbGNA1                 b (&#195;&#8230;) ...........................................................64.44 <br /> assumed globular, density predicted amino acid                 c (&#195;&#8230;) ...........................................................82.77 <br /> composition.                                                                              Alpha (&#194;&#176;) ....................................................90.00 <br />    For crystallization purposes, TbGNA1 expressed resistant strain E. coli         Beta (&#194;&#176;).......................................................90.00 <br /> BL21(DE3)-V2R-pRARE2 using Lex bioreactor (Harbinger Biotechnology                      Gamma (&#194;&#176;) .................................................90.00 <br /> and Engineering, Markham, Ontario, Canada) purified according meth-             Wavelength ....................................................1.5418 <br /> ods described previously (51). The protein sample identity purity             Resolution range (&#195;&#8230;) ...................................40&#226;&#8364;&#8220;1.86 <br /> evaluated mass spectrometry SDS-PAGE.                                           No. unique reflections .............................23,634 <br />    Crystallographic analysis, data processing, refinement. TbGNA1         No. reflections observed..........................23,625 <br /> His6 tag intact crystallized mixing 1.5 &#226;?&#174;l protein (at concentration    Rsym .................................................................0.049 (0.237) <br /> 10.9 mg/ml buffer 10 mM HEPES, pH 7.5, 500 mM NaCl) 1.5 &#226;?&#174;l             I/&#226;?&#180;&#229;&#8224;&#172;I&#229;&#8224;&#173; ................................................................28.94 (7.5) <br /> reservoir solution containing 19% polyethylene glycol 3350 0.2 M ammo-             Completeness (%) ........................................99.9 (99.9) <br /> nium tartrate hanging-drop vapor diffusion setup 18&#194;&#176;C VDX plates            Redundancy ...................................................7.0 (6.9) <br /> (Hampton Research, Aliso Viejo, CA). Crystals appeared 2 3 days were <br /> grown 80 150 &#226;?&#174;m 2 weeks. Crystals briefly soaked reservoir       Refinement statistics <br /> solution supplemented 15% ethylene glycol flash frozen        Resolution range...........................................35.96&#226;&#8364;&#8220;1.86 (1.91&#226;&#8364;&#8220;1.86) <br /> liquid nitrogen data collection. Single-wavelength data collected     No. reflections ..........................................22,408 (1,628) <br /> FR-E Superbright rotating anode X-ray source Raxis IV&#226;&#171;&#185;&#226;&#171;&#185; image plate               No. reflections test set ........................1,217 (95) <br /> (Rigaku, The Woodlands, TX).                                                           Rwork/Rfree.......................................................0.243/0.282 (0.309/0.331) <br />    Diffraction data reduced scaled program XDS (22). Mo-             No. atoms <br /> lecular replacement solution obtained program balbes (24) using S.           Protein........................................................2,265 <br /> cerevisiae GNA1 searching model (Protein Data Bank [PDB] accession                   Water..........................................................238 <br /> number 1I1D). Iterative cycles model building refinement carried           Mean B factor (&#195;&#8230;2) ......................................20.24 <br /> out using Coot refmac programs (12, 22, 33) The refinement statistics          Ramachandran (%) <br /> data TbGNA1 (PDB accession number 3I3G) listed Table 1. All                    Favored ......................................................98.15 <br /> structural figures produced using PyMOL program (www.pymol.org).                 Allowed ......................................................1.85 <br />    Glucosamine 6-phosphate N-acetyltransferase activity assay. Acetyl coenzyme            Disallowed .................................................0 <br /> A, GlcN-6P, coenzyme A supplied Sigma. All measurements were <br /> performed room temperature triplicate 96-well-plate format (Nunc).         RMSD <br /> For activity measurement, 500 &#226;?&#174;M GlcN-6P 500 &#226;?&#174;M acetyl-CoA                    Bond <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> (&#195;&#8230;)............................................0.002 <br /> dissolved 25 mM Tris-HCl&#226;&#8364;&#8220;150 mM NaCl, pH 7.2, total volume 50 &#226;?&#174;l.           Bond angle (&#194;&#176;) ...............................................0.531 <br /> The reactions initiated adding 50 ng TbGNA1 protein stopped with <br />                                                                                         a <br /> 50 &#226;?&#174;l solution containing 25 mM bis-Tris-propane, 250 mM NaCl, 2 mM                    PDB accession number 3I3G. <br />                                                                                         b <br />                                                                                             Values parentheses outer shell. <br /> EDTA, 6.4 M guanidine chloride, pH 7.5. Fifty microliters dithio-bis(2- <br /> nitrobenzoic acid) (DTNB) solution (1 mM DTNB 0.1% dimethyl sulfoxide <br /> containing 25 mM Tris-HCl, 250 mM NaCl, 2 mM EDTA, pH 7.5) was <br /> added. Reaction DTNB coenzyme A, by-product acetylation           mounted glass slides, sealed Hydromount containing 2.5% 1,4- <br /> GlcN, generates 2-nitro-5-thiobenzoate anion, absorbance measured         diazabicyclo[2.2.2]octane, left dry dark 30 min. Microscopy was <br /> at 412 nm. Absorbance quantified using Spectra Max 340 PC spectrometer          performed Zeiss LSM 510 META confocal microscope. <br /> (Molecular Devices), data analyzed Grafit (version 5) pro-             Sugar nucleotide analysis. Sugar nucleotide analysis performed de- <br /> gram. For determination kinetic constants, different concentrations      scribed previously (49). Briefly, cells pelleted centrifugation, washed in <br /> 1.5 mM GlcN-6P acetyl-CoA used.                                              ice-cold PBS, lysed 70% ethanol presence 20 pmol GDP- <br />    TbGNA1 localization. Two BALB/c adult mice used raise polyclonal           glucose internal standard. Sugar nucleotides extracted using EnviCarb <br /> antibodies GST-tagged TbGNA1. About 0.1 mg mouse used             columns (Supelco) analyzed using multiple-reaction-monitoring (MRM) <br /> immunization Freund&#226;&#8364;&#8482;s complete adjuvant. Each mouse received fur-            liquid chromatography-tandem mass spectroscopy (LC-MS/MS). <br /> ther immunizations Freund&#226;&#8364;&#8482;s incomplete adjuvant 2 months. Antibod-             Lectin blotting. T. brucei cells washed trypanosome dilution buffer and <br /> ies affinity purified CNBr-Sepharose-immobilized TbGNA1             hypotonically lysed 300 &#226;?&#174;l water containing 0.1 mM 1-chloro-3-tosylamido- <br /> had GST tag removed PreScission protease. A similar protocol         7-amino-2-heptone (TLCK) 1 &#226;?&#174;g/ml leupeptin. Cell ghosts harvested <br /> used immunize New Zealand rabbits 0.5 mg purified TbGNA1               centrifugation (13,000 &#226;&#171;&#187; g 10 min), pellet resuspended in <br /> protein animal. Wild-type bloodstream-form T. brucei cells grown          SDS-sample buffer containing 8 M urea. The lysed extracts subjected to <br /> HMI-9 medium density 1 &#226;&#171;&#187; 106 cells/ml, harvested centrifugation,       electrophoresis reducing conditions 1.5 &#226;&#171;&#187; 107 cell equivalents/lane on <br /> resuspended trypanosome dilution buffer (0.1 M Na2HPO4, 0.01 M NaH2PO4,            NuPAGE 4 12% bis-Tris gradient (Invitrogen) using morpholinepropane- <br /> 0.025 M KCl, 0.4 M NaCl, 5 mM MgSO4, 0.1 M glucose adjusted pH 7.45           sulfonic acid (MOPS)-SDS running buffer. Proteins transferred a <br /> HCl) density 4 &#226;&#171;&#187; 107 cells/ml. Aliquots (15 &#226;?&#174;l) added 13-mm           nitrocellulose membrane normal Western blotting conditions. <br /> coverslips, left 4&#194;&#176;C 15 min, fixed 1 ml 4% paraformaldehyde             Membranes stained Ponceau S solution demonstrate equal load- <br /> phosphate-buffered saline (PBS) 30 min, followed 5-min washes         ing; blocked 5% bovine serum albumin, 0.05% Igepal detergent (Sigma), <br /> 2 ml PBS. Cells permeabilized 0.1% Triton X-100 PBS 10 min           0.15 M NaCl 50 mM Tris-HCl, pH 7.4; incubated 0.33 &#226;?&#174;g/ml <br /> at room temperature. Samples blocked 5% fish skin gelatin (FSG)          biotinylated tomato lectin (Vector Labs), 3 mg/ml chitin <br /> in PBS containing 10% normal goat serum. The coverslips incubated           hydrolysate (Vector Labs), 1:10,000-diluted ExtraAvidin-horse- <br /> immunopurified mouse anti-TbGNA1 (0.5 &#226;?&#174;g/ml) rabbit anti-glyceralde-              radish peroxidase conjugate (Sigma). All membranes developed by <br /> hyde-3-phosphate dehydrogenase (anti-GAPDH) antiserum 1:2,000 dilution           chemiluminescent detection (ECL-plus; GE Healthcare). <br /> or rabbit antienolase antiserum 1:4,000 dilution 1% FSG PBS               Purification sVSG. The VSG coat trypanosomes conveniently <br /> 0.05% Triton X-100. Both anti-GAPDH antienolase kind gifts Paul           released soluble form (sVSG) osmotic cell lysis 37&#194;&#176;C (14). We <br /> Michels (Catholic University Louvain). Samples washed de-             used modified version procedure Cross cell lysis and <br /> scribed PBS incubated 50 &#226;?&#174;l Alexa 594-conjugated anti-           membrane washing step 0&#194;&#176;C (26). Briefly, T. brucei cultures (100 ml) were <br /> mouse IgG Alexa 488-conjugated anti-rabbit IgG 1 h. Coverslips           washed trypanosome dilution buffer, resuspended 300 &#226;?&#174;l lysis buffer (10 <br />  988           &#203;&#339; O ET AL. <br />           MARIN                                                                                                                        EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 2. Alignment GNA1 predicted amino acid sequences. The sequences Homo sapiens (Hsa), Mus musculus (Mmu), Aspergillus <br /> fumigatus (Afm), Saccharomyces cerevisiae (Sce), T. cruzi (Tcr), T. brucei, L. major (Lma) aligned using Clustal W (http://www.ebi <br /> .ac.uk/) Jalview (http://www.jalview.org/) programs. Residues required activity (circles) involved charge distribution (triangles) <br /> are highlighted. The classical motifs GNA1s (motifs A D) boxed, consensus motif [(R/Q)-X-X-Q-X-G] GNC5-related <br /> N-acetyltransferase family underlined motif A. <br />  <br />  <br />  <br /> mM NaH2PO4-Na2HPO4, pH 8.0, 0.1 mM TLCK, 1 &#226;?&#174;g/ml leupeptin, 1                   analysis suggests TbGNA1 shares structural similarities with <br /> &#226;?&#174;g/ml aprotinin), incubated 37&#194;&#176;C 10 min. The lysate cooled      members GNAT superfamily and, particular, with <br /> on ice 2 min centrifuged 16,000 &#226;&#171;&#187; g 5 min. The supernatant was <br /> applied 200 &#226;?&#174;l DE52 (Whatman) preequilibrated 10 mM NaH2PO4- <br />                                                                                     GNA1 orthologues (Fig. 2). The closest relatives TbGNA1 are <br /> Na2HPO4, pH 8.0, buffer eluted 4 times 200 &#226;?&#174;l fresh lysis buffer. The   putative GNA1s predicted kinetoplastid genome se- <br /> eluates pooled concentrated 100 &#226;?&#174;l using YM-10 spin concentrator      quences. T. cruzi contains putative genes TcGNA1 63 <br /> (Microcon), yielding 50 &#226;?&#174;g sVSG. The majority buffer salts     62% identities T. brucei gene sequence, L. <br /> removed diafiltration additions 0.5 ml water. This procedure <br />                                                                                     major gene similar 44% identity. In maximum <br /> yields sVSG preparations sufficiently pure (&#226;&#172;&#381;90% SDS-PAGE and <br /> Coomassie blue staining) analysis mass spectrometry (26, 46).                likelihood tree obtained using PhyML program (2), kineto- <br />    Electrospray mass spectrometry sVSG. Intact sVSG diluted 0.05          plastid GNA1 sequences separate clade (Fig. 3). Com- <br /> &#226;?&#174;g/&#226;?&#174;l 50% methanol&#226;&#8364;&#8220;1% formic acid loaded Micromass type F               pared sequence human orthologue (Fig. 2), T. <br /> nanotips. The sVSG analyzed positive-ion electrospray tandem mass            brucei GNA1 shares 26% identity, the <br /> spectrometry using Applied Biosystems Q-StarXL instrument (26, 46). The <br /> data processed using Bayesian protein reconstruction algorithm (ABI        residues involved activity conserved. Thus, con- <br /> Analyst Software).                                                                  servative changes observed 9 amino acids described to <br />    Accession numbers. DNA sequence T. brucei GNA1 deposited         essential activity (23, 30). The charge distribution the <br /> the EMBL nucleotide sequence database, accession number FR715998. Coor-             proposed GlcN-6P binding pocket important role only <br /> dinates TbGNA1 determined X-ray crystallography deposited <br /> in Protein Data Bank, accession number FR715998 1.86-&#195;&#8230; resolu- <br />                                                                                     substrate affinity catalytic activity. The consensus <br /> tion.                                                                               motif GCN5-related N-acetyltransferases (R/Q)-XX-G-X-G <br />                                                                                     (52), X amino acid, conserved T. brucei <br />                                                                                     GNA1, corresponding R-GA-G-L-G (amino acids 94 99). <br />                                  RESULTS <br />                                                                                     Nevertheless, low sequence similarity TbGNA1 and <br />    Identification, cloning, expression TbGNA1. GNA1                       Homo sapiens GNA1 (HsaGNA1) prompted investigation of <br /> an amino sugar N-acetyltransferase member GCN5-                        differences extended crystal structure. <br /> related N-acetyltransferase (GNAT) superfamily (30, 32). A                             The TbGNA1 ORF cloned pGEX 6P-1 expres- <br /> BLASTp search T. brucei predicted protein database                      sion vector GST tag N terminus PreScission <br /> the Saccharomyces cerevisiae GNA1 amino acid sequence (NCBI                         protease site protein tag. The plasmid was <br /> accession number NP_116637.1) human GNA1 (NCBI ac-                              transformed E. coli DH5&#226;?&#163; cells purified described <br /> cession number NP_932332.1) revealed putative TbGNA1 gene                         Materials Methods. Analysis untagged recombi- <br /> (GeneDB accession number Tb11.01.2886). The 444-bp TbGNA1                           nant TbGNA1 protein gel filtration analytical ultracen- <br /> open reading frame (EMBL accession number FR715998)                             trifugation 0.5 mg/ml 0.75 mg/ml indicated the <br /> amplified PCR genomic DNA prepared T. brucei                           majority TbGNA1 dimeric concentrations (see <br /> strain 427. There single-nucleotide difference, corre-                  Fig. S1 supplemental material). <br /> sponding codon change, 427 sequence reported                          Crystal structure TbGNA1. Several TbGNA1 constructs <br /> and 927 genome strain (G184 replaced C184). Sequence                      prepared order achieve crystallization; con- <br />  VOL. 10, 2011                                            T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                                 989 <br />  <br />  <br />  <br />  <br />   FIG. 3. Phylogenetic analyses glucosamine 6-phosphate N-acetyltransferases. The amino acid sequences GNA1s mouse (NCBI <br /> accession number 54342), rat (NCBI accession number 498486), bovine (NCBI accession number 512299), Homo sapiens (NCBI accession number <br /> 64841), Saccharomyces cerevisiae (NCBI accession number 850529), Aspergillus flavus (NCBI accession number 7920275), Aspergillus niger (NCBI <br /> accession number 4986135), Aspergillus fumigatus (NCBI accession number 3505194), Danio rerio (NCBI accession number 554143), Xenopus laevis <br /> (NCBI accession number 379524), Trypanosoma brucei (NCBI accession number 3665679), T. cruzi (NCBI accession numbers 3551278/3544900), L. <br /> major (NCBI accession number 5653516) aligned, phylogenetic relationships analyzed using Jalview (version 2.6) (54) PhyML <br /> (2) programs. The tree rooted midpoint calculation, numbers colon correspond bootstrap support branch. The <br /> branch lengths given colon calculated WAG model. Asterisks indicate putative GNA1s identified sequence homology. <br />  <br />  <br />  <br /> struct 4 N-terminal residues successful.              respect resembles S. cerevisiae GNA1 <br /> The refined structure contains homodimer, mono-                 structure (PDB accession number 1I1D). The HsaGNA1 struc- <br /> mer (named chains A B crystal structure) composed                ture 35 extra residues N terminus found <br /> of 146 residues amino-terminal valine miss-               T. brucei gene sequence. Overall, root mean square <br /> ing chain A electron density map. Chain B includes               difference (RMSD) TbGNA1 main chains S. <br /> all residues construct, including 6 residues              cerevisiae GNA1 1.41 &#195;&#8230;, compared HsaGNA1 <br /> amino-terminal His tag. Its structure resembles classic                   sequence, RMSD 1.55 &#195;&#8230;, suggesting major confor- <br /> GNAT fold (39, 52), containing motifs (A D; Fig. 2)             mational differences exact secondary structure. How- <br /> conserved GNATs. Unfortunately, structure                      ever, RMSD narrowed just C terminus <br /> with natural ligands (GlcN-6P, acetyl-CoA, GlcNAc-6P,                   (residues 63 134), results S. cerevisiae GNA1 (Sce- <br /> CoA) glucose-6-phosphate (21) obtained. Similar                 GNA1) extremely close, 0.804 &#195;&#8230; compared 1.411 &#195;&#8230; <br /> to GNATs, including Saccharomyces cerevisiae (PDB                 humans. An Aspergillus fumigatus GNA1 (AfmGNA1) <br /> accession number 1I1D) human (PDB accession num-                    structure (PDB accession number 2VEZ) recently re- <br /> ber 2HUZ) homologues, TbGNA1 (PDB accession number                          ported (20) adopts C-terminal fold similar to <br /> 3I3G) features dimeric structure created intertwining              TbGNA1. As common feature, TbGNA1, AfmGNA1, <br /> of C-terminal beta strands (Fig. 4B). The TbGNA1 dimer                  SceGNA1 adopt long beta strand lines bot- <br /> is present crystal lattice solution,         tom sugar binding pocket, human structure <br /> demonstrated gel filtration analytical centrifugation              shows interruption, placing residues outside the <br /> results (see Fig. S1 supplemental material). The GNAT                pocket lining, pointing away active-site entrance (Fig. <br /> superfamily signature clearly apparent: N-terminal strand             4B, arrow). <br /> (&#226;?&#164;1), followed &#226;?&#163; helices (&#226;?&#163;3-&#226;?&#163;4-&#226;?&#163;5), antiparallel               As noted AfmGNA1 structure, acetyl- <br /> &#226;?&#164; strands, signature central helix (&#226;?&#163;8), fifth &#226;?&#164; strand, &#226;?&#163;          CoA pocket relatively unchanged HsaGNA1 and <br /> helices (&#226;?&#163;9 &#226;?&#163;10), sixth final &#226;?&#164; strand (Fig. 4A).             TbGNA1, sugar binding pocket contains signifi- <br /> TbGNA1 adopts overall fold human orthologue                 differences. Several residues essential interactions <br /> and shows typical intertwining C-terminal beta strands,             substrates, K118 Y117 (involved posi- <br /> it maintains longer hydrogen-bonding network area                 tioning phosphate group) D87 (which positions the <br />  990         &#203;&#339; O ET AL. <br />         MARIN                                                                                                                   EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 4. Crystal structure TbGNA1. (A) Representation domains present TbGNA1 monomer N-terminal motif C <br /> (red), motif D (blue), motif A (violet), motif B (yellow). Light blue, second chain forms dimer. (B) Overall structure comparison <br /> of TbGNA1 (cyan blue, domain dimeric structure) HsaGNA1 (gray). Arrows point C-terminal discrepancies between <br /> H. sapiens T. brucei entrance sugar binding site. (C) Close-up HsaGNA1 active site, representing surface charge and <br /> shape pocket. The ribbon cartoon shows specific interactions substrates. (D) TbGNA1 active site, representing surface <br /> charge shape pocket. (E) Ribbon cartoon sticks showing specific interactions substrates H. sapiens GNA1 structure <br /> (green blue, corresponding different domains). (F) Ribbon cartoon sticks showing specific interactions substrates T. <br /> brucei GNA1 structure. As TbGNA1 crystallized presence substrates, figure created modeling N-acetyl-D- <br /> glucosamine-6-phosphate human structure. <br />  <br />  <br />  <br /> carbohydrate residue the pocket), highly                deeper pocket human protein (Fig. 4D). The <br /> conserved structures. In spite high               narrowness TbGNA1 pocket possibly attrib- <br /> conservation observed residues involved activity,             uted fact structure bound substrate. <br /> significantly different environment pocket lining              However, SceGNA1 AfmGNA1 pockets are <br /> also parasite enzyme compared                       affected substrate binding, making unlikely. <br /> HsGNA1. In parasite enzyme, V82 replaces R116 I110                 Lastly, replacement H111 (Fig. 4E; conserved S. cerevi- <br /> replaces T76, changing shape charge pocket. In              siae, H. sapiens, A. fumigatus involved phosphate <br /> addition, TbGNA1 H139 creates smaller entrance                    positioning) R77 interesting, residue is <br /> pocket (compared E174 HsaGNA1 A151 Sce-                        positioned pocket. The longer residue possibly <br /> GNA1), overall pocket shape narrower                    link phosphate linking water outside posi- <br />  VOL. 10, 2011                                          T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                              991 <br />  <br />  <br />  <br />  <br />   FIG. 5. Subcellular localization TbGNA1. Wild-type bloodstream-form T. brucei cells stained affinity-purified mouse anti- <br /> TbGNA1 Alexa 488-conjugated antimouse antibody (green channel) (A E) rabbit anti-GAPDH Alexa 594-conjugated <br /> antirabbit antibody (red channel) (B) mark glycosomes rabbit antienolase Alexa 594-conjugated anti-rabbit antibody (red channel) <br /> (F) mark cytosol. Merged images shown (C G), corresponding phase-contrast images shown (D H). <br />  <br />  <br />  <br /> tion, potentially crowd correct phosphate          combination following electroporation parasites the <br /> positioning moved similar position histidines        presence linear DNA containing HYG flanked 500 <br /> in structures (Fig. 4F).                                       bp TbGNA1 5&#226;&#172;&#732; 3&#226;&#172;&#732; UTRs. Following selection with <br />    Enzyme kinetics. To address question TbGNA1               hygromycin, &#226;&#338;&#172;TbGNA1::HYG clone selected trans- <br /> encodes active acetyltransferase, enzymatic assays             formed ectopic, tetracycline-inducible copy of <br /> recombinant protein performed described Materials             TbGNA1 introduced ribosomal DNA (rDNA) locus <br /> and Methods. Analysis kinetic data demonstrated           phleomycin selection (55). After tetracycline induction, <br /> obeyed simple Michaelis-Menten kinetics substrates              second endogenous allele replaced PAC gene to <br /> with linear Lineweaver-Burke plots (see Fig. S2 supple-           yield desired &#226;&#338;&#172;TbGNA1::HYG/TbGNA1Ti/&#226;&#338;&#172;TbGNA1::PAC <br /> mental material). The apparent Km values GlcN-6P (144 &#226;&#171;&#190;              clone. After round transformation, selected clones <br /> 11 &#226;?&#174;M) acetyl-CoA (234 &#226;&#171;&#190; 44 &#226;?&#174;M) similar                 analyzed Southern blotting using TbGNA1 ORF probe. <br /> reported GNA1s (23, 32). The activities re-             The blot obtained transfection (Fig. 6B) shows <br /> combinant proteins affected presence His6             successful introduction ectopic copy replace- <br /> or GST tags (data shown). Recombinant TbGNA1                  ment endogenous alleles &#226;&#338;&#172;TbGNA1::HYG/ <br /> stored &#226;&#171;&#186;80&#194;&#176;C months 10% glycerol               TbGNA1Ti/&#226;&#338;&#172;TbGNA1::PAC clone used studies. This <br /> significant loss activity.                                            TbGNA1 conditional null mutant cell line referred as <br />    Subcellular localization TbGNA1. Mouse anti-TbGNA1                 TbGNA1cKO. <br /> affinity-purified antibodies used rabbit anti-           The TbGNA1 gene essential bloodstream-form T. brucei <br /> GAPDH antibody glycosomal marker. The secondary an-                 vitro. Triplicate cultures wild-type TbGNA1 condi- <br /> tibodies antimouse Alexa 488 (red) antirabbit Alexa             tional null mutant cells inoculated permissive and <br /> 594 (green). The fluorescence micrographs wild-type cells             nonpermissive conditions (i.e., tetracycline, <br /> show anti-GAPDH colocalized anti-                      respectively) 2.5 &#226;&#171;&#187; 105 cells/ml subcultured day. <br /> TbGNA1, indicating TbGNA1 located predominantly               The TbGNA1 conditional null mutant cultures permis- <br /> glycosome microbodies bloodstream-form T. brucei (Fig. 5A             sive conditions growth rates similar rate wild <br /> to C). A similar experiment performed rabbit anti-              type (Fig. 7A). Under nonpermissive conditions, cells grew <br /> enolase antibodies cytosolic marker (Fig. 5E G),           2 days subculturing failed grow 5 days (Fig. <br /> no colocalization found. Additional images showing              7B). Nevertheless, cultures resumed growth day <br /> cells shown Fig. S3 supplemental material.                 8. Analysis reverse transcription-PCR confirmed that <br />    Construction TbGNA1 conditional null mutant. RT-                 TbGNA1 mRNA undetectable 6 h tetracycline re- <br /> PCR primers described Materials Methods              moval cells resumed growth 10 days after <br /> TbGNA1 ORF expression showed gene transcribed               tetracycline removal reexpressing TbGNA1 mRNA (Fig. <br /> in procyclic bloodstream forms parasite              7C). This phenomenon, conditional <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> be- <br /> (data shown). The conditional null mutant construction               come tetracycline independent, known trypanosoma- <br /> strategy summarized Fig. 6A possible                 tids evidence gene essentiality (43). <br /> genome assembly clearly indicates TbGNA1 present                 In order check addition GlcN sufficient the <br /> a single copy haploid genome. A Southern blot analysis               rescue GNA1 <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>knockout</span>, medium added GlcN (1 <br /> using TbGNA1 ORF probe consistent                   mM) used parallel cultures. However, phenotype <br /> conclusion (Fig. 6B).                                                    changed parasites died 2 days (data not <br />    The TbGNA1 allele replaced homologous re-                shown), showing novo pathway UDP-GlcNAc <br />  992        &#203;&#339; O ET AL. <br />        MARIN                                                                                                     EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 6. Creation bloodstream-form T. brucei TbGNA1 conditional null mutant. (A) Schematic representation genetic <br /> transformations. The endogenous allele TbGNA1 replaced HYG, creating &#226;&#338;&#172;TbGNA1::HYG. The ectopic copy (pLew100- <br /> TbGNA1) inserted rDNA region, creating &#226;&#338;&#172;TbGNA1::HYG/TbGNA1Ti. The second allele TbGNA1 replaced <br /> with PAC, creating &#226;&#338;&#172;TbGNA1::PAC/TbGNA1Ti/&#226;&#338;&#172;TbGNA1::HYG. (B) Southern blots. Genomic DNA wild-type (WT) T. brucei 427 <br /> was digested BssSI, PstI, HindIII, SalI probed TbGNA1 ORF probe, showing TbGNA1 single-copy gene. <br /> Genomic DNA TbGNA1 conditional null mutant (dKO) digested PstI BssSI probed TbGNA1 open reading <br /> frame, confirming copy present mutants ectopic gene. <br />  <br />  <br /> biosynthesis main source sugar nucleotide T.    TbGNA1 gene expression parasite UDP-GlcNAc levels, <br /> brucei.                                                           sugar nucleotides extracted TbGNA1 condi- <br />   Phenotypic characterization TbGNA1 conditional null          tional null mutant permissive nonpermissive condi- <br /> mutant. To analyze effect selective removal         tions quantified ion-pair HPLC-electrospray ioniza- <br />  VOL. 10, 2011                                           T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                                           993 <br />  <br />  <br />  <br />  <br />    FIG. 7. T. brucei TbGNA1 conditional null mutant: confirmation gene essentiality culture. (A) Growth curve TbGNA1 conditional null <br /> mutant cells, subcultured day, grown permissive (plus tetracycline) conditions. (B) Growth curve TbGNA1 conditional null mutant <br /> cells, subcultured day, grown nonpermissive (minus tetracycline) conditions. (C) Ethidium bromide-stained agarose gel reverse <br /> transcription-PCR products RNA extracted TbGNA1 conditional null mutant cells 0, 6, 24, 48 h 10 days without <br /> tetracycline, indicated. The middle panel shows reverse transcription-PCR products using TbGNA1 primers, upper panel control without <br /> reverse transcriptase, lower panel control using DPMS primers equal RNA input. <br />  <br />                                                                                  TABLE 2. Quantification sugar nucleotides liquid <br /> tion-MS/MS method described previously (49). Briefly, sugar                          chromatography-tandem mass spectrometrya <br /> nucleotides extracted T. brucei, separated HPLC,                                          Sugar nucleotide concn (pmol/107 cells) <br />                                                                                Sugar <br /> and quantitated MRM tandem mass spectrometry using                     nucleotide        Wild type         0h             24 h             48 h <br /> internal standard, GDP-glucose, sugar nucleotide not <br /> found trypanosomes. The levels UDP-GlcNAc, UDP-Glc,                 UDP-glucose         182 &#226;&#171;&#190; 15       210 &#226;&#171;&#190; 5        281 &#226;&#171;&#190; 50       267 &#226;&#171;&#190; 40 <br /> UDP-Gal, GDP-Man TbGNA1 conditional null cell                  UDP-galactose        44 &#226;&#171;&#190; 9         68 &#226;&#171;&#190; 15         78 &#226;&#171;&#190; 8        94 &#226;&#171;&#190; 6 <br />                                                                           UDP-GlcNAc          209 &#226;&#171;&#190; 12       270 &#226;&#171;&#190; 14       224 &#226;&#171;&#190; 40        17 &#226;&#171;&#190; 2 <br /> line permissive conditions agree reasonably           GDP-mannose          12 &#226;&#171;&#190; 4         19 &#226;&#171;&#190; 3          26 &#226;&#171;&#190; 5        39 &#226;&#171;&#190; 12 <br /> wild-type levels (46, 49). However, dramatic reduction               GDP-fucose         0.18 &#226;&#171;&#190; 0.03     0.2 &#226;&#171;&#190; 0.04     0.15 &#226;&#171;&#190; 0.07     0.1 &#226;&#171;&#190; 0.03 <br /> UDP-GlcNAc levels, 6%, occurs 48 h tetracy-                   a <br />                                                                                Sugar nucleotides extracted TbGNA1 conditional null mutant 0, <br /> cline removal (Table 2). The cell cultures stopped divid-            24, 48 h withdrawal tetracycline. The experiments performed <br /> ing 48 72 h died, cell lysis, 72 h.        triplicate. <br />  994         &#203;&#339; O ET AL. <br />         MARIN                                                                                                                 EUKARYOT. CELL <br />  <br />  <br />    Effects UDP-GlcNAc starvation protein glycosylation.         HsaGNA1, AfmGNA1, SceGNA1, TbGNA1 replaces <br /> The poly-N-acetyllactosamine-specific lectin Lycopersi-          R181 D146, complete change charge function. A <br /> con esculentum (tomato lectin) used detection         neighboring arginine (R144; Fig. 4F) able the <br /> glycoproteins flagellar pocket endosomal/lysosomal         phosphate positioning duties poking pocket the <br /> system T. brucei (36). The tomato lectin binding oligosac-         correct position forming hydrogen bonds anom- <br /> charide structures shown include family             alous aspartate. As seen Fig. 4D, TbGNA1 there <br /> unusually large N-linked poly-N-acetyllactosamine-containing          large positive patch accommodate the <br /> glycans average 54 N-acetyllactosamine repeats         phosphate moiety negative patch back, similar to <br /> glycan (3). The effect UDP-GlcNAc starvation              HsaGNA1. However, size pocket created the <br /> structures assessed using Western blots whole-cell ly-         dimer actually longer narrower human or- <br /> sates wild-type TbGNA1 conditional null mutant cell          thologue, providing potentially exploitable differences the <br /> lines probed tomato lectin. A large smear detected        design discovery specific inhibitors. <br /> the wild-type cell lysate, decreased progressively 0          GNA1 enzymes, like components hexosamine <br /> to 48 h absence tetracycline TbGNA1 condi-           biosynthesis, typically cytosolic eukaryotes, mu- <br /> tional null mutant (Fig. 8, lanes 1 4). The decreased intensity    rine GNA1 association cytoplasmic <br /> of tomato lectin binding high-molecular-mass glycopro-         face Golgi apparatus intracellular vesicle mem- <br /> teins indicated significant reduction total poly-N-acetyllac-    branes (5). This association based hydrophobic annexin <br /> tosamine synthesis cellular levels UDP-GlcNAc fall.         V-like domain present mammalian GNA1s, is <br /> The specificity tomato lectin blot carbohydrate        absent TbGNA1. In contrast, TbGNA1 localizes gly- <br /> confirmed inhibition chitin hydrolysate (Fig. 8, lanes 5      cosomes. Glycosomes trypanosomatids belong micro- <br /> to 8).                                                                body family organelles, containing glyoxysomes plants <br />    The effects UDP-GlcNAc starvation VSG glycosyla-             peroxisomes eukaryotes. These organelles resem- <br /> tion studied. sVSG purified wild-type          ble respect morphology contain both <br /> TbGNA1 conditional null mutant cells grown presence            common species-specific metabolic functions (29, 37). A <br /> and absence tetracycline 48 h. Analysis SDS-PAGE            unique feature trypanosomatid glycosomes essential <br /> and Coomassie blue staining revealed sVSGs wild-            role carbohydrate metabolism, i.e., glycolysis, gluconeogen- <br /> type TbGNA1 conditional null cells grown permis-            esis, glycerol metabolism, pentose-phosphate pathway <br /> sive conditions appear single bands, sVSG         (18, 29). Glycosomal proteins typically targeted these <br /> TbGNA1 conditional null mutant grown 48 h absence          organelles peroxisomal targeting sequence (PTS), which <br /> of tetracycline appeared doublet (Fig. 9A B). The            located C terminus (PTS-1) N terminus <br /> VSG doublet present mutant represents VSG molecules            (PTS-2) protein. Internal targeting sequences also <br /> that similar wild type (i.e., upper     proposed targeting mechanism, T. <br /> band) VSG molecules selectively lost N-           cruzi phosphoglucomutase T. brucei triosephosphate <br /> linked glycan C-terminal Asn428 site (i.e., lower        isomerase examples (16, 40). However, proteins also <br /> band), previously described T. brucei UDP-       traffic glycosome piggybacking glycosome- <br /> GlcNAc pyrophosphorylase (TbUAP; EC 2.7.7.23) conditional             targeted proteins (47). Since conventional trypanosome PTSs <br /> null mutant (46). <br />  <br />                          DISCUSSION <br />    In paper, identified Tb11.01.2886 gene <br /> encoding GNA1 activity T. brucei. This organism appears <br /> to typical eukaryotic UDP-GlcNAc biosynthetic path- <br /> way (46), indeed, dimeric nature Km TbGNA1 <br /> for GlcN-6P similar human homologue. <br /> However, parasite enzyme approximately 10-fold <br /> higher Km acetyl-CoA human homologue. <br /> TbGNA1 HsaGNA1 26% amino acid sequence <br /> identity share GNAT motifs, like S. <br /> cerevisiae Aspergillus, crystal structures quite sim- <br /> ilar (20, 21, 39, 53). Although apo structure <br /> for TbGNA1 (despite efforts obtain complexes acetyl- <br /> CoA, CoA, GlcN-6P, GlcNAc-6P, Glc-6P), make <br />                                                                          FIG. 8. Phenotypic characterization TbGNA1 conditional null <br /> useful comparisons parasite human active-site         mutant. Tomato lectin blotting wild-type TbGNA1 conditional <br /> pockets: human GNA1 active site, GlcN6P interacts              null mutant extracts. Extracts wild type (lane 1) TbGNA1 <br /> closely R116 R181 (20); residues di-          conditional null mutant cells 0, 24, 48 h tetracycline removal <br /> rectly contact reaction product. Similar Aspergillus       (lanes 2, 3, 4, respectively) subjected SDS-PAGE, trans- <br />                                                                       ferred nitrocellulose, probed biotinylated tomato lectin, <br /> fumigatus enzyme, TbGNA1, R116 replaced V82,               followed horseradish peroxidase-streptavidin. Lanes 5 8, in <br /> smaller, neutral residue, presumably reducing ability in-      lanes 1 4, respectively, tomato lectin inhibitor (chitin <br /> teract phosphate group. Noticeably, contrast       hydrolysate) included lectin. <br />  VOL. 10, 2011                                            T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                                995 <br />  <br />  <br />  <br />  <br />   FIG. 9. Electrospray mass spectrometry sVSG TbGNA1 conditional null mutant. Cells grown presence tetracycline (A) or <br /> absence tetracycline (B) 48 h analyzed positive-ion electrospray mass spectrometry, data processed Bayesian protein <br /> reconstruction produce mass graphs isobaric glycoforms. The SDS-PAGE bands sVSG detected Coomassie blue staining. Models <br /> of principal glycoforms similar observed TbUAP mutant indicated. amu, atomic mass units. <br />  996         &#203;&#339; O ET AL. <br />         MARIN                                                                                                                   EUKARYOT. CELL <br />  <br />  <br /> are apparent TbGNA1 or, indeed, T. cruzi L.      (Asn428) N-glycosylation site. This similar situ- <br /> major orthologues (37), likely TbGNA1 uses            ation described TbUAP TbALG3 mutants (26, 46), <br /> either piggybacking mechanism unidentified inter-       reemphasizes relative importance N-glycosylation <br /> nal targeting sequence. In case, TbGNA1 joins list         Asn263 site Asn428 site. A similar <br /> enzymes novo sugar nucleotide biosynthesis          conclusion recently drawn conditional null mu- <br /> been experimentally identified glycosomal components.         tant T. brucei GDP-Man pyrophosphorylase (10). <br /> These include UDP-GlcNAc pyrophosphorylase (46),                   In summary, paper extended knowl- <br /> GDP-mannose 4,6-deydratase GDP-fucose biosynthesis               edge enzymology location hexosamine biosyn- <br /> (50), UDP-glucose 4&#226;&#172;&#732;-epimerase UDP-galactose biosynthe-          thesis T. brucei determined TbGNA1 es- <br /> sis (42), UDP-glucose pyrophosphorylase UDP-glucose          sential gene bloodstream-form T. brucei and, therefore, <br /> biosynthesis (27). In addition, hexokinase, necessary   potential target drugs activity the <br /> above pathways, phosphoglucose isomerase, necessary         parasite. However, therapeutic potential obviously <br /> GDP-mannose, GDP-fucose, UDP-GlcNAc biosynthesis,               depend ability drug-like parasite-specific <br /> are known glycosomal components. Finally, phosphoman-          TbGNA1 inhibitors, enzyme generally essential for <br /> nose isomerase, contains PTS1 signal necessary       eukaryotes (5, 31, 32). Nevertheless, inhibitors other <br /> for GDP-mannose GDP-fucose biosynthesis,               GNAT family members developed (17, 41, 56), <br /> found glycosome proteome (6). It possible       structure TbGNA1 shows similar <br /> that sugar nucleotide biosynthesis machinery     dimerization mode human protein have <br /> is localized glycosome T. brucei rate     similar activities specificities, active sites dissim- <br /> synthesis sugar nucleotides controlled, degree,    ilar, key residues replaced. These differ- <br /> compartmentalization. Such mechanism             ences exploited drug discovery process. <br /> proposed alternative allosteric regulation gly- <br /> colysis T. brucei (18).                                                                   ACKNOWLEDGMENTS <br />    The construction conditional null mutant               This work supported Wellcome Trust Programme Grant <br /> TbGNA1 gene enabled test essentiality          (085622) M.A.J.F. We grateful University Dundee <br /> confirm alternative routes GlcNAc-6P.          Proteomics Facility Mark Agacan, supported Wellcome <br /> Essentiality expected result, TbUAP known       Strategic Award 083481, performing proteomics ultracen- <br /> be essential (46). However, half genes     trifugation experiments. The Structural Genomics Consortium a <br />                                                                     registered charity (number 1097737) receives funds Ca- <br /> T. brucei encode hypothetical proteins unknown function,         nadian Institutes Health Research, Canadian Foundation for <br /> that fewer experimentally annotated,         Innovation Genome Canada, GlaxoSmithKline, Karolinska Insti- <br /> GNAT enzyme superfamily largest known,            tutet, Knut Alice Wallenberg Foundation, Ontario Inno- <br /> possibility redundancy certainly excluded prior     vation Trust, Ontario Ministry Research Innovation, <br />                                                                     Merck, Novartis Research Foundation, Swedish Agency for <br /> to experimental verification. A recent relevant case         Innovation Systems, Swedish Foundation Strategic Research, <br /> point unexpected nonessentiality UDP-Glc pyrophos-        Wellcome Trust. <br /> phorylase L. major, transpires function       We grateful Alan R. Prescott (Division Cell Signaling and <br /> be compensated relatively nonspecific UDP-sugar            Immunology, University Dundee) valuable help micros- <br /> pyrophosphorylase (8) present T. cruzi (57).        copy. We thank Paul Michels providing anti-GAPDH and <br />                                                                     antienolase antibodies, Daan van Aalten providing vectors, Jim <br />    The phenotype TbGNA1 conditional null mutant un-          Procter Geoff Barton helpful advice sequence comparisons. <br /> der nonpermissive conditions similar TbUAP          We conflicts declare. <br /> conditional null mutant, i.e., rapid cessation growth and <br /> death cell lysis 48 72 h tetracycline removal (Fig.                                     REFERENCES <br /> 7B). Also, like TbUAP mutant, took 48 h             1. Alexander, D. L., K. J. Schwartz, A. E. Balber, J. D. Bangs. 2002. <br />                                                                         Developmentally regulated trafficking lysosomal membrane protein <br /> UDP-GlcNAc fall 5% wild-type levels. However,            p67 Trypanosoma brucei. J. Cell Sci. 115:3253&#226;&#8364;&#8220;3263. <br /> whereas UDP-glucose UDP-galactose levels dropped            2. Anisimova, M., O. Gascuel. 2006. Approximate likelihood ratio test for <br /> to 60% wild-type levels 48 h TbUAP mutant,           branchs: fast, accurate powerful alternative. Syst. Biol. 55:539&#226;&#8364;&#8220;552. <br />                                                                      3. Atrih, A., J. M. Richardson, A. R. Prescott, M. A. Ferguson. 2005. <br /> the levels sugar nucleotides slightly increased         Trypanosoma brucei glycoproteins contain novel giant poly-N-acetyllac- <br /> TbGNA1 mutant (Table 2). Intuitively, expect              tosamine carbohydrate chains. J. Biol. Chem. 280:865&#226;&#8364;&#8220;871. <br />                                                                      4. Azema, L., et al. 2004. Interaction substituted hexose analogues the <br /> latter, lack UDP-GlcNAc reduce rate               Trypanosoma brucei hexose transporter. Biochem. Pharmacol. 67:459&#226;&#8364;&#8220;467. <br /> UDP-Gal (and UDP-Glc precursor) consumption             5. Boehmelt, G., et al. 2000. Cloning characterization murine glu- <br /> the GPI anchor chain N-glycan processing. The rea-             cosamine-6-phosphate acetyltransferase EMeg32. Differential expression <br />                                                                         intracellular membrane association. J. Biol. Chem. 275:12821&#226;&#8364;&#8220;12832. <br /> sons anomalous behavior TbUAP mutant              6. Colasante, C., M. Ellis, T. Ruppert, F. Voncken. 2006. Comparative <br /> currently investigation. The consequences UDP-                 proteomics glycosomes bloodstream form procyclic culture form <br /> GlcNAc starvation, death 72 h, studied            Trypanosoma brucei brucei. Proteomics 6:3275&#226;&#8364;&#8220;3293. <br />                                                                      7. Cross, G. A. 1996. Antigenic variation trypanosomes: secrets surface <br /> isolating parasites 48 h removal tetracycline. In          slowly. Bioessays 18:283&#226;&#8364;&#8220;291. <br /> these cells, discrete sets VSG glycoforms corresponding       8. Damerow, S., et al. 2010. Leishmania UDP-sugar pyrophosphorylase. J. Biol. <br />                                                                         Chem. 285:878&#226;&#8364;&#8220;887. <br /> to bands seen SDS-PAGE observed elec-             9. Davis, A. J., et al. 2004. Properties GDP-mannose pyrophosphorylase, a <br /> trospray MS (Fig. 9): set similar wild-type           critical enzyme drug target Leishmania mexicana. J. Biol. Chem. <br /> VSG, lacking higher-molecular-mass             279:12462&#226;&#8364;&#8220;12468. <br />                                                                     10. Denton, H., S. Fyffe, T. K. Smith. 2010. GDP-mannose pyrophosphory- <br /> glycoforms, set masses consistent              lase essential bloodstream form Trypanosoma brucei. Biochem. J. <br /> the absence oligomannose structures C-terminal                425:603&#226;&#8364;&#8220;614. <br />  VOL. 10, 2011                                                       T. BRUCEI GLUCOSAMINE 6-PHOSPHATE N-ACETYLTRANSFERASE                                               997 <br />  <br /> 11. Doering, T. L., W. J. Masterson, P. T. Englund, G. W. Hart. 1989.                     stages Leishmania major utilize amino sugars major carbon source. <br />     Biosynthesis glycosyl phosphatidylinositol membrane anchor                  PLoS Pathog. 6:e1001245. <br />     trypanosome variant surface glycoprotein: origin non-acetylated glu-         35.   Nagamune, K., et al. 2004. Surface sialic acids taken host allow <br />     cosamine. J. Biol. Chem. 264:11168&#226;&#8364;&#8220;11173.                                                 trypanosome survival tsetse fly vectors. J. Exp. Med. 199:1445&#226;&#8364;&#8220;1450. <br /> 12. Emsley, P., B. Lohkamp, W. G. Scott, K. Cowtan. 2010. Features              36.   Nolan, D. P., M. Geuskens, E. Pays. 1999. N-Linked glycans containing <br />     development Coot. Acta Crystallogr. D Biol. Crystallogr. 66:486&#226;&#8364;&#8220;501.                   linear poly-N-acetyllactosamine sorting signals endocytosis Trypano- <br /> 13. Engstler, M., et al. 2005. The membrane-bound histidine acid phosphatase                  soma brucei. Curr. Biol. 9:1169&#226;&#8364;&#8220;1172. <br />     TbMBAP1 essential endocytosis membrane recycling Trypano-             37.   Opperdoes, F. R., J. P. Szikora. 2006. In silico prediction glyco- <br />     soma brucei. J. Cell Sci. 118:2105&#226;&#8364;&#8220;2118.                                                  somal enzymes Leishmania major trypanosomes. Mol. Biochem. Para- <br /> 14. Ferguson, M. A., M. Duszenko, G. S. Lamont, P. Overath, G. A. Cross.                  sitol. 147:193&#226;&#8364;&#8220;206. <br />     1986. Biosynthesis Trypanosoma brucei variant surface glycoproteins. N-          38.   Peck, R. F., et al. 2008. The LAMP-like protein p67 plays essential role in <br />     Glycosylation addition phosphatidylinositol membrane anchor.                     lysosome African trypanosomes. Mol. Microbiol. 68:933&#226;&#8364;&#8220;946. <br />     J. Biol. Chem. 261:356&#226;&#8364;&#8220;362.                                                         39.   Peneff, C., D. Mengin-Lecreulx, Y. Bourne. 2001. The crystal structures <br /> 15. Freeze, H. H., A. B. Elbein. 2008. Glycosylation precursors, p. 47&#226;&#8364;&#8220;61. In             apo complexed Saccharomyces cerevisiae GNA1 shed light the <br />     A. Varki, R. D. Cummings, J. D. Esko, H. H. Freeze, P. Stanley, C. R.                     catalytic mechanism amino-sugar N-acetyltransferase. J. Biol. Chem. <br />     Bertozzi, G. W. Hart, M. E. Etzler (ed.). Essentials glycobiology, 2nd             276:16328&#226;&#8364;&#8220;16334. <br />     ed. Cold Spring Harbor Laboratory Press, New York, NY.                              40.   Penha, L. L., C. B. Sant&#226;&#8364;&#8482;Anna, L. Mendonca-Previato, N. L. Cunha-e-Silva, <br /> 16. Galland, N., S. Walque, F. G. Voncken, C. L. Verlinde, P. A. Michels.              J. O. Previato, A. P. Lima. 2009. Sorting phosphoglucomutase to <br />     2010. An internal sequence targets Trypanosoma brucei triosephosphate                     glycosomes Trypanosoma cruzi mediated internal domain. Glyco- <br />     isomerase glycosomes. Mol. Biochem. Parasitol. 171:45&#226;&#8364;&#8220;49.                              biology 19:1462&#226;&#8364;&#8220;1472. <br /> 17. Gao, F., et al. 2006. Synthesis structure-activity relationships trun-       41.   Poux, A. N., M. Cebrat, C. M. Kim, P. A. Cole, R. Marmorstein. 2002. <br />     cated bisubstrate inhibitors aminoglycoside 6-N-acetyltransferases.                    Structure GCN5 histone acetyltransferase bound bisubstrate <br />     J. Med. Chem. 49:5273&#226;&#8364;&#8220;5281.                                                               inhibitor. Proc. Natl. Acad. Sci. U. S. A. 99:14065&#226;&#8364;&#8220;14070. <br /> 18. Haanstra, J. R., et al. 2008. Compartmentation prevents lethal turbo-             42.   Roper, J. R., et al. 2005. The suppression galactose metabolism procylic <br />     explosion glycolysis trypanosomes. Proc. Natl. Acad. Sci. U. S. A.                  form Trypanosoma brucei causes cessation cell growth alters procyclin <br />     105:17718&#226;&#8364;&#8220;17723.                                                                          glycoprotein structure copy number. J. Biol. Chem. 280:19728&#226;&#8364;&#8220;19736. <br /> 19. Hinderlich, S., et al. 1998. Purification characterization N-acetylglu-      43.   Roper, J. R., M. L. Guther, K. Milne, M. A. Ferguson. 2002. Galactose <br />   <br /> </body></html>