<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Role SKN-1/Nrf Pathogen Resistance and <br /> Immunosenescence Caenorhabditis elegans <br /> Dia&#194;&#180;na Papp&#194;&#164;, Pe&#194;&#180;ter Csermely, Csaba So&#203;?ti* <br /> Department Medical Chemistry, Semmelweis University, Budapest, Hungary <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     A proper immune response ensures <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>survival</span> hostile environment promotes longevity. Recent evidence indicates <br />     innate immunity, antimicrobial effectors, relies host-defensive mechanisms. The Caenorhabditis elegans <br />     transcription factor SKN-1 regulates xenobiotic oxidative stress responses contributes longevity, however, role <br />     immune defense unknown. Here SKN-1 required C. elegans pathogen resistance Gram- <br />     negative Pseudomonas aeruginosa Gram-positive Enterococcus faecalis bacteria. Exposure P. aeruginosa leads SKN- <br />     1 accumulation intestinal nuclei transcriptional activation SKN-1 target genes, gcs-1 gst-4. Both Toll/IL- <br />     1 Receptor domain protein TIR-1 p38 MAPK PMK-1 required SKN-1 activation PA14 exposure. We <br />     demonstrate early onset immunosenescence concomitant age-dependent decline SKN-1-dependent target <br />     gene activation, requirement SKN-1 enhance pathogen resistance response longevity-promoting <br />     interventions, reduced insulin/IGF-like signaling preconditioning H2O2 treatment. Finally, wdr- <br />     23(RNAi)-mediated constitutive SKN-1 activation results excessive transcription target genes, confers oxidative stress <br />     tolerance, impairs pathogen resistance. Our findings identify SKN-1 novel regulator innate immunity, suggests its <br />     involvement immunosenescence provide important crosstalk pathogenic stress signaling the <br />     xenobiotic/oxidative stress response. <br />  <br />   Citation: Papp D, Csermely P, So&#203;?ti C (2012) A Role SKN-1/Nrf Pathogen Resistance Immunosenescence Caenorhabditis elegans. PLoS Pathog 8(4): <br />   e1002673. doi:10.1371/journal.ppat.1002673 <br />   Editor: Frederick M. Ausubel, Massachusetts General Hospital, Harvard Medical School, United States America <br />   Received November 6, 2011; Accepted March 13, 2012; Published April 26, 2012 <br />   Copyright: &#195;&#376; 2012 Papp et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grants EU (FP6-518230, FP7-200970, TA&#194;&#180;MOP-4.2.2/B-10/1-2010-0013), joint grant Hungarian Science <br />   Foundation Norway Grants (NNF-78794), Hungarian Science Foundation (OTKA K69105 OTKA-K83314). C.S. Bolyai Research Scholar the <br />   Hungarian Academy Sciences. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: soti.csaba@med.semmelweis-univ.hu <br />   &#194;&#164; Current address: Department Genetics, Eo&#194;&#168;tvo&#194;&#168;s Lora&#194;&#180;nd University, Budapest, Hungary <br />  <br />  <br />  <br /> Introduction                                                                         regulation pathogen-specific immune response C. elegans <br />                                                                                      [13,14]. Both p38 MAPK IIS pathways regulate Nrf1/2/3 <br />    A proper immune response ensures survival hostile                            ortholog SKN-1, transcription factor orchestrates both <br /> environment contributes longevity. The nematode                               oxidative xenobiotic stress responses C. elegans [15,16]. <br /> Caenorhabditis elegans provides valuable genetic tool studying                 However, involvement SKN-1 regulation pathogen <br /> innate immunity various aspects host-pathogen interac-                        stress response unknown. <br /> tions. During infection, bacterial virulence factors host                      In nematodes, SKN-1 isoforms exist. While function of <br /> antimicrobial defense mechanisms present oxidative proteo-                       SKN-1A elucidated yet, SKN-1B C provide <br /> toxic noxae inducing tissue-damage, especially intestine [1&#226;&#8364;&#8220;                  distinct biological functions. SKN-1B expressed ASI <br /> 5]. Accordingly, self-protective stress-response regulators                  neurons, mediates lifespan extension response dietary <br /> including forkhead transcription factor DAF-16/FOXO [6],                         restriction [17]. In contrast, intestinal SKN-1C required for <br /> the heat shock transcription factor HSF-1 [7] X-box                          oxidative stress resistance contributes longevity reduced <br /> binding protein 1 (XBP-1) [8] required robust immunity.                      IIS [15]. SKN-1 activity regulated phosphorylation and <br /> Moreover, DAF-16-regulated antioxidant enzymes SOD-3                             degradation. Under normal conditions, inhibitory phosphoryla- <br /> and CTL-2 contribute immunity protecting intestinal cells                      tions GSK-3 IIS kinases, AKT-1/2 SGK-1, retain <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> reactive oxygen species exposure Enterococcus faecalis                SKN-1 cytosol [15,18], rapidly targeted <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> [9]. Strikingly, hyper-activation DAF-16 enhances susceptibility                  proteasomal degradation WD40 repeat protein WDR-23 <br /> to bacterial infection [10].                                                         [19,20]. In response oxidative stress, p38 MAPK ortholog <br />    These data illustrate critical role stress response innate                PMK-1 phosphorylates SKN-1, translocates the <br /> immunity, raise questions co-ordination                             nuclei intestinal cells induces transcription phase 2 <br /> antimicrobial host-defense mechanisms. Antimicrobial re-                         detoxification genes [16]. <br /> sponses mediated canonical p38 mitogen-activated                               Here report SKN-1 required pathogen resistance <br /> protein kinase (MAPK) pathway, conserved                               Gram-negative P. aeruginosa Gram-positive E. <br /> nematodes humans [11,12]. Besides, insulin/IGF-like                           faecalis bacteria, consistently independent study [21] <br /> signaling (IIS) TGF-b pathways involved                          published submission paper. We demonstrate a <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                                    1                             April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                    SKN-1 C. elegans Immunity <br />  <br />  <br />  <br />   Author Summary                                                           Results <br />   Innate immunity promotes survival combating patho-                    SKN-1 required bacterial pathogen resistance C. <br />   genic threat. During infection, tissue damage induced                 elegans <br />   invading pathogens immune effectors                     To study role SKN-1 C. elegans immunity, examined <br />   toxins free radicals. Therefore, important                  pathogen resistance animals absence SKN-1. skn- <br />   elucidate self-protective mechanisms host                    1(zu135) allele considered genetic null <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> it <br />   defends itself pathogenic stress. The conserved                  creates premature stop codon affects SKN-1 isoforms <br />   SKN-1 protein roundworm Caenorhabditis elegans                    [15]. skn-1(zu135) mutant worms exposed Gram- <br />   directs detoxification response neutralizing harmful                   negative Pseudomonas aeruginosa (PA14) strain. As skn-1(zu135) <br />   compounds confers tolerance oxidative <br />                                                                            mutants sterile, eliminated difference them <br />   stress. Here identify SKN-1 novel regulator C. <br />   elegans innate immunity. We SKN-1 contributes                  wild-type N2 strain arising &#226;&#8364;&#732;bag worms&#226;&#8364;&#8482; <br />   resistance infection caused bacterial                  <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, major contributor killing. To end, germline <br />   pathogens. Components pathogen-responsive signal-                   development inhibited silencing cdc-25.1, required for <br />   ing pathway required activate SKN-1 intestinal                 embryonic mitosis meiosis. cdc-25.1(RNAi) animals exhibit <br />   cells site infection. Moreover, SKN-1-                     extended survival pathogenic bacteria, reported previously <br />   dependent response pathogen exposure declines                         [21,22]. In conditions, observed increased suscepti- <br />   aging, mild metabolic oxidative                       bility skn-1(zu135) mutants PA14 (Figures 1A S1A, <br />   stresses, known extend lifespan, evoke SKN-1-                       Tables S1A S1E). This result confirmed using skn- <br />   dependent boosting immunity. Finally,                    1(RNAi) (Figures 1B S1B, Table S1A S1E). Furthermore, <br />   elimination inhibitory protein leads excessive               animals exposed Gram-positive Enterococcus <br />   activation SKN-1 impairs pathogen resistance. Thus,               faecalis SdB262 strain, skn-1(zu135) skn-1(RNAi) exhibited <br />   SKN-1 integrates various chemical, metabolic micro-                  significantly decreased survival, case absence of <br />   bial signals elicit self-protective detoxification                  SKN-1 exerted modest effect (Figures S1C S1D, Table <br />   response, promotes innate immunity                      S1E). These results suggest requirement SKN-1 the <br />   relevant diseases aging human immune                       efficient immune response distinct bacterial pathogens. <br />   system.                                                                  In subsequent experiments, focused defining role <br />                                                                            SKN-1 antibacterial response P. aeruginosa. <br /> Toll Interleukin-1 Receptor domain protein (TIR-1)/PMK-1- <br /> dependent SKN-1 activation P. aeruginosa infection. More-             P. aeruginosa infection triggers SKN-1 activation <br /> over, gradual decrease pathogen resistance            To investigate SKN-1 nuclear translocation occurs upon <br /> activation SKN-1-dependent targets aging,                  PA14 exposure, incubated skn-1::gfp L3 larvae P. aeruginosa <br /> requirement SKN-1 boost immunity response                      lawn 5 hours. We massive accumulation SKN- <br /> longevity-promoting manipulations, reduced IIS                 1::GFP intestinal nuclei infected larvae, compared control <br /> preconditioning H2O2 treatment. Finally, hyper-               animals fed non-pathogenic OP50 Escherichia coli strain <br /> activation SKN-1 impairs pathogen resistance. Our results               (Figures 2A 2B). The specificity response was <br /> indicate intricate regulation innate immunity SKN-1           demonstrated complete inhibition using skn-1-specific <br /> links pathogenic stress signaling xenobiotic stress response.       double-stranded RNA. To reveal SKN-1-dependent transcrip- <br />  <br />  <br />  <br />  <br /> Figure 1. SKN-1 required bacterial pathogen resistance. (A, B) Increased susceptibility Pseudomonas aeruginosa PA14 occurs both <br /> skn-1(zu135) mutant (p,0.0001) skn-1(RNAi) nematodes (p,0.0001). Killing assays performed 90 young adult animals each <br /> condition. EV: vector RNAi. <br /> doi:10.1371/journal.ppat.1002673.g001 <br />  <br />  <br />       PLoS Pathogens | www.plospathogens.org                           2                          April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                           SKN-1 C. elegans Immunity <br />  <br />  <br />  <br />  <br /> Figure 2. P. aeruginosa infection activates SKN-1. (A) Representative epifluorescence image demonstrating translocation SKN-1::GFP in <br /> the Is007[SKN-1::GFP] strain intestinal nuclei L3 larvae, fed vector skn-1 dsRNA, 5-hour exposure P. aeruginosa PA14. <br /> Note intestinal tissue displays autofluorescence, ASI neurons SKN-1::GFP silenced skn-1 RNAi treatment. (B) Quantification <br /> of SKN-1 nuclear translocation data shown panel (A). SKN-1::GFP-positive nuclei counted intestine 78 animals. &#226;&#8364;&#732;&#226;&#8364;&#732;Low&#226;&#8364;&#8482;&#226;&#8364;&#8482; refers to <br /> animals SKN-1::GFP detected 5 intestinal nuclei, &#226;&#8364;&#732;&#226;&#8364;&#732;high&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates SKN-1::GFP signal present 15 <br /> intestinal nuclei. (C) Representative epifluorescence microscopic image showing intestinal expression Pgcs-1::GFP GST-4::GFP L3 larvae upon <br /> a 24-hour PA14 exposure. Images control animals incubated OP50 bacteria shown Figure S2. (D) Quantification reporter expression <br /> demonstrating SKN-1-dependence response. Data obtained panel (C) completed data skn-1(RNAi) animals. <br /> Microscopic images representatives 3 independent experiments. EV: vector RNAi. <br /> doi:10.1371/journal.ppat.1002673.g002 <br />  <br /> tional activation PA14 infection, examined Pgcs-1::gfp             The TIR-1/PMK-1 pathway controls SKN-1 activation upon <br /> and gst-4::gfp reporter strains. While gcs-1 regulated exclusively          P. aeruginosa infection <br /> by SKN-1, gst-4 mutual control DAF-16                    The p38 MAPK ortholog PMK-1 fundamental role C. <br /> SKN-1 [15]. We observed effective intestinal induction                   elegans innate immunity [11]. To investigate PMK-1 <br /> fluorescence comparable extent strains response             regulates SKN-1 response bacterial exposure, monitored <br /> 24 h-exposure PA14 (Figures 2C 2D). Both gcs-1                      activity Pgcs-1::GFP reporter wild-type pmk- <br /> promoter activation GST-4 expression significantly                1(km25) mutant genetic background (Figures 3A 3B). We <br /> suppressed feeding worms skn-1(RNAi), indicating                   silencing pmk-1 entirely prevented SKN-1- <br /> specific requirement SKN-1 elicit responses. Thus,                 dependent activation gcs-1 response PA14 infection. In <br /> PA14 infection induces nuclear translocation SKN-1                      physiological settings, PMK-1 inactivated dual specificity <br /> transcriptional activation targets.                                     MAPK phosphatase VHP-1 [23]. Suppression VHP-1 resulted <br />  <br />  <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                              3                            April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                         SKN-1 C. elegans Immunity <br />  <br />  <br />  <br />  <br /> Figure 3. The pathogen response-specific TIR-1 p38 MAPK PMK-1 required SKN-1 activation P. aeruginosa <br /> infection. (A) Representative epifluorescence microscopic images showing expression Pgcs-1::GFP pmk-1(km25) mutants the <br /> p38 MAPK phosphatase vhp-1(RNAi), Toll/IL-1 resistance (TIR) domain protein tir-1(RNAi) animals response P. aeruginosa infection. L3 <br /> larvae exposed PA14 24 hours. Microscopic images representatives 3 independent experiments. (B) Quantification reporter <br /> expression data shown panel (A) completed data control animals fed OP50 24 h. (C) Quantification SKN-1 nuclear <br /> translocation tir-1(RNAi) L3 larvae 5 h PA14 exposure. Representative epifluorescence images tir(RNAi) L3 larvae shown Figure S3. <br /> Please note data Figure 2B 3C derived set experiments. (D) Suggested model SKN-1 activation P. <br /> aeruginosa infection. Upon exposure PA14, TIR-1/PMK-1 pathway indispensable insufficient elicit SKN-1 transactivation. We propose a <br /> second, unknown factor/pathway required activate SKN-1. Whether pathways act parallel consecutively unclear. Solid arrows <br /> indicate direct, dashed arrows indicate indirect/unknown connection. EV: vector RNAi. <br /> doi:10.1371/journal.ppat.1002673.g003 <br />  <br />  <br /> in increased PMK-1 phosphorylation resistance PA14 [23].                 TIR-1 conserved Toll/IL-1 resistance (TIR) domain protein <br /> However, vhp-1(RNAi) significantly increased Pgcs-1::GFP activa-             known activate p38 MAPK signaling independently Toll- <br /> tion PA14, OP50 exposure, suggesting                  like receptor ortholog tol-1 PA14 infection [24,25]. <br /> PMK-1 indispensable permissive factor SKN-1 activation             Depletion TIR-1 RNAi prevented Pgcs-1::GFP fluorescence <br /> by infection.                                                                PA14 infection (Figures 3A 3B). A similar inhibition in <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                            4                            April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                    SKN-1 C. elegans Immunity <br />  <br />  <br /> Pgcs-1::GFP expression observed tir-1(qd4) mutant             pathogen resistance, nematodes pretreated various <br /> animals (data shown). Moreover, silencing tir-1 prevented         concentrations H2O2, exposed PA14 infection. <br /> nuclear translocation SKN-1 induced PA14 infection,             H2O2 preconditioning induced resistance PA14 a <br /> did affect baseline expression levels (Figures 3C S3.).       concentration-dependent manner, reaching 2-fold increase in <br /> Altogether, results suggest TIR-1/PMK-1 pathway            survival 2 mM H2O2, compared untreated controls <br /> is necessary attain activation SKN-1 PA14 exposure               (Figure 5B Table S1D). Intriguingly, treatment on <br /> (Figure 3D).                                                              skn-1(zu135) mutant nematodes exhibited decreased <br />                                                                           pathogen resistance, strongly suppressed reaction to <br /> Involvement SKN-1 immunosenescence                                  H2O2 (Figure 5C Table S1D). We the <br />    Immune function declines age, leading compromised              mutation major oxidative stress response regulator, <br /> immune responses infections elderly. Accordingly, aged          DAF-16 (daf-16(mu86)), shows shorter basal survival, <br /> nematodes exhibit increased susceptibility infection various        compared skn-1(zu135), poorly responded H2O2 <br /> pathogens, including P. aeruginosa [26&#226;&#8364;&#8220;28]. As SKN-1 required          (Figure 5C Table S1D). Thus, oxidative preconditioning <br /> for longevity pathogen resistance, asked               requires SKN-1 DAF-16 enhanced pathogen <br /> chronological aging affected SKN-1-dependent target gene                  resistance P. aeruginosa. <br /> expression nematodes exposed pathogenic stress. To this <br /> end examined promoter induction gcs-1 PA14 L3             Excessive activation SKN-1 wdr-23(RNAi) impairs <br /> stage larvae, 4-day 9-day old adult worms, respectively            pathogen resistance <br /> (Figures 4A 4B). We observed massive age-dependent                     Finally, investigated increased activation SKN-1 <br /> decrease expression Pgcs-1::GFP reporter 24 h          able promote pathogen resistance. Stabilization SKN-1 <br /> PA14 infection. To investigate, SKN-1 activity involved         RNAi wdr-23 shown induce constitutive <br /> immunosenescence, examined survival 1, 4 9 day-             SKN-1 activation, resistance oxidative stress longevity [19]. <br /> old adult N2 skn-1(zu135) mutant nematodes exposed                 Feeding worms wdr-23(RNAi) resulted an <br /> PA14. We observed pathogen resistance wild-type animals           unexpectedly robust increase expression Pgcs-1::GFP <br /> already declined day 4 previously described Laws et al.          GST-4::GFP (Figure 6A) compared PA14-induced <br /> [26]. Consistent premature decline self defense          expression (Figure 2C). To surprise, wdr-23(RNAi), compared <br /> absence SKN-1 activity, 4 d adult N2 worms showed similar              vector feeding greatly reduced pathogen resistance to <br /> survival PA14 1 d adult skn-1(zu135) animals (p = 0.1429).          PA14 (Figure 6B Table S1A). wdr-23(RNAi) did impair <br /> Furthermore, skn-1(zu135) mutant animals                    survival skn-1(zu135) mutant background excluding SKN-1- <br /> exhibited increased susceptibility PA14, compared N2             independent impact WDR-23 pathogen resistance. Inter- <br /> all ages (p.0.0001) (Figure 4C Table S1B), indicating            estingly, compromised reactivity worms wdr-23(RNAi) was <br /> SKN-1 function required survive infection day 9.        confined pathogenic stress. Determination oxidative <br />    To address potential involvement SKN-1-dependent gene           tolerance revealed wdr-23(RNAi) animals exhibited increased <br /> expression immunosenescence, performed bioinformatics             survival, skn-1(RNAi) nematodes displayed decreased <br /> analysis using microarray data Youngman et al. [27]. From          survival, compared control worms, exposed 3 mM or <br /> the 379 genes exhibiting significant down-regulation             5 mM H2O2, respectively (Figure 6C). Our results suggest an <br /> during aging (.10 fold down-regulation d15 vs. d6)                  excessive post-translational stabilization SKN-1 induces oxida- <br /> identified 46 SKN-1-regulated genes (based skn-1(RNAi) screens         tive stress resistance impairs resistance bacterial infection. <br /> [29,30]) (Figure 4D, Table S2). Next, examined regulation <br /> of genes respect oxidative stress, PA14 PMK-1 <br /> dependent regulation using Wormbase expression data [31]. <br />                                                                           Discussion <br /> Strikingly, SKN-1-regulated genes subject PA14-dependent                  Our present study identified SKN-1 novel regulator of <br /> regulation over-represented compared regulated           pathogen resistance Gram-negative P. aeruginosa and <br /> either oxidative stress PMK-1, respectively. These results             Gram-positive E. faecalis bacteria (Figures 1 S1). We <br /> confirm progressive age-dependent compromise pathogen                demonstrated TIR-1/PMK-1-dependent SKN-1 activation <br /> resistance imply decline SKN-1 function contributes         P. aeruginosa infection (Figures 2 3). Moreover, we <br /> to immunosenescence.                                                      showed early onset immunosenescence parallel <br />                                                                           decline SKN-1-dependent transcriptional activation (Figure 4) <br /> Reduced IIS oxidative preconditioning require SKN-1                   requirement SKN-1 efficient immunity response to <br /> for enhanced pathogen resistance                                          reduced IIS preconditioning H2O2 treatment (Figure 5). <br />    Loss-of-function mutations insulin/IGF-1 receptor gene,         Finally, excessive activation SKN-1 blocking <br /> daf-2 enhance stress resistance extend lifespan,             turnover impaired pathogen resistance (Figure 6). <br /> processes require DAF-16 SKN-1 activity [15]. As reduced                 The xenobiotic stress response provides conserved defense <br /> IIS increases pathogen resistance [6], investigated                mechanism oxidative electrophilic stress the <br /> contribution SKN-1 pathogen resistance daf-2(e1370)              induction phase 2 detoxification enzymes [33]. Nrf2 its <br /> mutant animals. In accordance previously published data [6],         nematode ortholog, SKN-1, transcription factors important in <br /> daf-2(e1370) mutants exhibited robustly increased pathogen                oxidative xenobiotic stress response [34,35]. Previously, <br /> resistance PA14 (Figure 5A Table S1C). However                studies demonstrated importance Nrf2 innate <br /> silencing skn-1 RNAi largely increased susceptibility         immunity mammals [36&#226;&#8364;&#8220;38]. For example, Nrf22/2 mice <br /> PA14. These data suggest SKN-1 required reduced IIS           exhibit increased susceptibility bacterial infection bacterial <br /> to bring enhanced pathogen resistance P. aeruginosa.        lipopolysaccharide (LPS)-induced inflammation [39]. Similarly, <br />    Exposure mild oxidative stress induces tolerance lethal        present study demonstrates SKN-1 deficiency C. elegans <br /> challenge, cross-tolerance stresses extends lifespan         impairs resistance infection (Figures 1 S1). During the <br /> [32]. To address impact oxidative preconditioning               revision manuscript independent paper Garsin <br />  <br />  <br />       PLoS Pathogens | www.plospathogens.org                          5                           April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                         SKN-1 C. elegans Immunity <br />  <br />  <br />  <br />  <br /> Figure 4. Involvement SKN-1 immunosenescence. (A) Representative epifluorescence images showing decreased induction the <br /> gcs-1 promoter. L3 larvae, 4 d/9 d adult Pgcs-1::gfp worms exposed PA14 24 h. (B) Quantification epifluorescence images panel <br /> (A). Epifluorescence images representatives independent experiments. EV: vector RNAi. (C) Pathogen resistance young adult (1 <br /> day-old), 4 day-old 9 day-old adult N2 skn-1(zu135) mutant animals. skn-1(zu135) mutant worms exhibited significantly increased <br /> susceptibility PA14 compared N2 wild-type animals ages (p,0.0001). 1 day-old adult skn-1(zu135) worms similar pathogen resistance <br /> to 4 day-old N2 worms (p = 0.1429) (middle graph). Killing assays performed 3 parallel plates condition 2 independent trials. (D) <br /> Venn diagram showing distribution age-regulated SKN-1 target genes. Data analyzed finding overlaps micro-array <br /> databases containing genes down-regulated 10-fold 15 d adult compared 6 d adult wild-type animals [27] SKN-1 dependent <br /> genes non-stress [29] oxidative stress conditions [30] using expression data Wormbase [31]. Please note majority genes <br /> belong regulated PA14 infection. 10 46 genes assigned groups. For detailed gene list refer Table <br /> S2. <br /> doi:10.1371/journal.ppat.1002673.g004 <br />  <br /> lab appeared, obtained similar results [21]. A previous study           confirmed decrease pathogen resistance. A similar finding <br /> found significant impairment pathogen resistance skn-                reported earlier unpublished result Evans et al. <br /> 1(zu135) skn-1(zu67) mutations wildtype background                 [41]. <br /> [40]. A possible reason discrepancy use cdc-             We observed nuclear translocation transcriptional <br /> 25.1(RNAi) Garsin lab study, suggesting                   activation SKN-1 intestine, consistent primary <br /> selective bagging wild-type vs. sterile skn-1 mutants           site infection (Figures 2 3). Furthermore, not <br /> masked pathogen resistance decrease induced loss skn-1              detect apparent change SKN-1 intensity nuclear <br /> in previous investigation. Consistently note, use        localization ASI neurons PA14 exposure (Figure 2). This <br /> skn-1(RNAi) L1 stage, did induce sterility [21],           finding agreement previous reports constitutive SKN- <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                             6                            April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                         SKN-1 C. elegans Immunity <br />  <br />  <br />                                                                                regulation SKN-1B response infection excluded. <br />                                                                                Hence, tissue-specific analysis SKN-1 function clue <br />                                                                                SKN-1 isoforms co-operate immunity. <br />                                                                                   Our findings confirm van der Hoeven et al. [21] the <br />                                                                                critical role p38 MAPK pathway SKN-1 activation <br />                                                                                (Figure 3). However, inability vhp-1(RNAi) activate SKN-1 <br />                                                                                OP50 suggests additional, unidentified <br />                                                                                signals govern SKN-1 activation response PA14 <br />                                                                                infection, certainly prompt additional studies. We <br />                                                                                showed absolute requirement TIR-1 SKN-1 nuclear <br />                                                                                translocation gcs-1 promoter induction PA14 <br />                                                                                exposure, Garsin lab reported minimal <br />                                                                                involvement TIR-1 gst-4 gcs-1 induction response to <br />                                                                                E. faecalis [21]. Whether difference observations <br />                                                                                differences assays dosage/treatment tir-1(RNAi) <br />                                                                                differential pathogen sensing P. aeruginosa and <br />                                                                                E. faecalis exciting possibility explore. TIR-1 PMK-1 <br />                                                                                related mammalian SARM p38 MAPK proteins, <br />                                                                                respectively [42,43]. Although existence orthologous <br />                                                                                pathway mammals remains elusive, findings indicate that <br />                                                                                SKN-1-mediated response ancient component innate <br />                                                                                immunity. <br />                                                                                   Immunosenescence, age-dependent decline immune <br />                                                                                response, critical problem impeding healthy ageing [44]. C. <br />                                                                                elegans provides useful tool investigate elements innate <br />                                                                                immunity contributing immunosenescence [45]. A recent <br />                                                                                systematic study reported age-dependent progressive increase <br />                                                                                susceptibility PA14, detectable day 6 adulthood [27]. <br />                                                                                Our data similar age-related decline survival, 45% <br />                                                                                decrease pathogen resistance day 4 (Figure 4C) establishes an <br />                                                                                earlier, dramatic onset immunosenescence. Moreover, loss <br />                                                                                SKN-1 function phenocopies decreased resistance d4 <br />                                                                                worms day 1, continues negatively affect survival <br />                                                                                day 9 (Figure 4C). A parallel strong decline gcs-1 <br />                                                                                transactivation day 4 widespread down-regulation of <br />                                                                                SKN-1 targets, including PA14-regulated genes, day 6 <br />                                                                                15 adulthood (Figure 4D, Table S2) consistent this <br />                                                                                observation indicate SKN-1 key player immunosenes- <br />                                                                                cence. <br /> Figure 5. Reduced IIS oxidative preconditioning require                       Youngman colleagues involvement PMK-1 a <br /> SKN-1 enhanced pathogen resistance. (A) daf-2(e1370) mutant                decline innate immune response, hypothesized <br /> nematodes exhibited increased resistance P. aeruginosa, compared            intestinal deterioration primary event immunosenescence <br /> to wild-type N2 worms (p,0.0001). skn-1(RNAi) treatment daf-        [27]. Of note, dependence SKN-1 activation PMK-1 <br /> 2(e1370) animals increased susceptibility P. aeruginosa infection       ([21] study), high number age-dependent SKN-1 <br /> (p,0.0001). Killing assays performed 90 young 1-day <br /> old adult animals condition. (B) H2O2 pretreatment increased           targets PMK-1 targets (13 26; Figure 4D, Table S2 and <br /> survival PA14 concentration-dependent manner. Survival curves          [27]) impact SKN-1 intestinal homeostasis [30] <br /> of N2 wild-type worms treated various concentrations H2O2           suggest dynamic, probably mutual interaction SKN-1 <br /> liquid NGM: 1 mM (p = 0.425), 1.5 mM (p,0.0001) 2 mM                       PMK-1 immunosenescence. We propose SKN-1- <br /> (p,0.0001) 12 h prior killing assay shown. Killing assay        dependent stress responses collapse early adulthood, which <br /> performed 90 3-day old adult animals condition. (C)               manifest vicious circle decreasing intestinal homeostasis, <br /> Oxidative preconditioning-induced pathogen resistance impaired <br /> in absence SKN-1 DAF-16. Increase survival               progressive immunosenescence increasing pathogenic load in <br /> pronounced skn-1(zu135) (p = 0.0156) daf-16(mu86) mutant          C. elegans. This hypothesis consistent short lifespan <br /> (p = 0.0304), wild-type animals (p,0.0001). Survival curves         worms natural conditions allocation resources <br /> the genetic background compared absence                   maintain soma production fit progeny (the <br /> presence H2O2. Data combined experiments             &#226;&#8364;&#732;&#226;&#8364;&#732;disposable soma theory&#226;&#8364;&#8482;&#226;&#8364;&#8482; [46]). <br /> with 89 animals average group. EV: vector RNAi. <br /> doi:10.1371/journal.ppat.1002673.g005 <br />                                                                                   Genetic environmental interventions operate stress- <br />                                                                                responsive mechanisms extend lifespan pathogen <br />                                                                                resistance [6,15,32,47,48]. Our results H2O2-induced pathogen <br /> 1B activity lack interaction SKN-1B WDR-                  resistance (Figure 5B), previous analogous heat- <br /> 23 ASI neurons, respectively [19,34]. Together, data imply              shock experiments [7] suggest mild stresses acting early in <br /> an active role intestinal SKN-1C isoform does allow             adulthood confer resistance pathogenic stress. Further- <br /> a conclusion regarding involvement ASI neuronal SKN-                more, demonstration requirement SKN-1 DAF- <br /> 1B inducible antibacterial response. However, continuous              16 enhanced pathogen resistance H2O2-precondi- <br /> transcriptional output SKN-1B and/or different mode                    tioned daf-2(e1370) mutant nematodes suggests dynamic <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                              7                           April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                      SKN-1 C. elegans Immunity <br />  <br />  <br />  <br />  <br /> Figure 6. Excessive activation SKN-1 wdr-23(RNAi) impairs pathogen resistance. (A) Robust up-regulation Pgcs-1::GFP GST- <br /> 4::GFP 1 d adult wdr-23(RNAi) worms. (B) Pathogen resistance wdr-23(RNAi)-fed N2 skn-1(zu135) mutant worms. N2;wdr-23(RNAi) exhibited <br /> increased susceptibility P. aeruginosa infection (p,0.0001). skn-1(zu135) mutant nematodes fed wdr-23(RNAi) showed significant difference <br /> in survival PA14 (p = 0.1992). Killing assay performed 90 1-day old adult animals condition. Please note data in <br /> Figures 1A 6B derived set experiments. (C) wdr-23(RNAi) treatment increased (p,0.0001, 3 mM 5 mM H2O2, <br /> respectively), skn-1 RNAi treatment decreased oxidative tolerance H2O2 (p,0.0001 3 mM H2O, p,0.05 5 mM H2O2). Worms were <br /> treated 3 mM 5 mM H2O2 1 hour, 24 h challenge survival scored. Data combined experiments 120 <br /> animals average group. EV: vector RNAi. <br /> doi:10.1371/journal.ppat.1002673.g006 <br />  <br />  <br /> cross-talk stress-responsive transcription networks tipping        skn-1(zu135) mutants PA14 non-pathogenic OP50 <br /> the balance responses <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability, oxidative           (51% vs. 19% compared N2, Figures 1 S4, Tables S1 and <br /> and pathogen stress. Though data allow clear                   S3) suggests stronger impact SKN-1 pathogen resistance <br /> conclusion, recent prediction DAF-16-dependent                     longevity. The pathogen-induced activation SKN-1 and <br /> regulation SKN-1 [49] line proposed functional            large number PA14-regulated SKN-1-targets including <br /> interaction SKN-1 DAF-16 subject future             immune-related CUB-like domain proteins (Figure 4D, Table S2) <br /> interesting studies.                                                        [29] support SKN-1&#226;&#8364;&#8482;s active involvement pathogen <br />    Evidence mammals indicates defensive role Nrf2           response. Finally, previously shown knock-down <br /> inflammation-induced tissue damage [36&#226;&#8364;&#8220;39]. An analogous                    skn-1 daf-2(e1370) mutant selectively suppresses stress <br /> nematode model raises question, SKN-1 affects                   resistance lifespan [15]. Thus, findings demonstrating <br /> immunity independently impact aging. Indeed, longevity,           SKN-1-dependent increase pathogen resistance allele <br /> stress resistance pathogen resistance intimately linked          (Figure 5A), suggest immune-specific effect SKN-1. Studies <br /> short-lived C. elegans. However, greater reduction survival         investigating SKN-1-dependent responses OP50 vs. pathogens <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                           8                           April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                       SKN-1 C. elegans Immunity <br />  <br />  <br /> would help reveal downstream mediators SKN-1                   PCR using primers pmk-1-IF (59-TCCTATAAGTTGCCAT- <br /> determine immune-specific branches SKN-1                    GACCTCAG-39) pmk-1-IR (59-CCCGAGCGAGTACATT- <br /> action.                                                                      CAGC-39) inside deletion region. Wild-type animals <br />    SKN-1 activation wdr-23(RNAi) impairs pathogen resistance,             generated 469-bp fragment, homozygous pmk-1(km25) <br /> a result contrast Garsin lab [21]. The reason           allele did produce fragment. The primers tir-1-OF (59- <br /> may lie use cdc-25.1(RNAi) us, different              TGGGTAAATGAGGAAGAGAGAGAG-39) tir-1-OR (59- <br /> dosage/duration RNAi treatment experimental                    TCGGTTGACGAGTCGAATTTGG-39) used detect the <br /> protocols. Nevertheless, findings adverse effects              tir-1(qd4) deletion allele. The wild-type tir-1 allele amplified a <br /> excessive SKN-1 activity consistent reporting            1368-bp fragment compared 228-bp fragment tir-1(qd4) <br /> loss WDR-23 activity slows growth SKN-1 [19], expression              allele. Homozygous tir-1(qd4) mutants identified PCR <br /> of SKN-1 high-copy arrays toxic [15], SKN-1                 using primers tir-1-OF tir-1-IR (59-CACAAGAACGTGCAA- <br /> mediates increased susceptibility PA14 absence BLI-3            CATCG-39) inside deletion region. Wild-type animals <br /> [21]. Combining wdr-23(RNAi) results ([21] study)            generated 327-bp fragment, homozygous tir-1(qd4) <br /> clearly shows type activation dissociate immunity           allele did produce fragment. <br /> from oxidative stress resistance. As potential mechanism, <br /> excessive SKN-1 activation remodel transcriptional                   <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) <br /> response favor anti-oxidative defense and/or repress                  The HT115(DE3) E. coli bacteria producing dsRNA cdc- <br /> pathogen-specific defenses. Indeed, differential SKN-1 transcrip-            25.1 (Andy Golden NIDDK/NIH, Bethesda MD, USA), skn-1 (T. <br /> tional outputs demonstrated [29]. It logical assume          Keith Blackwell, Harvard Medical School, Boston MA, USA), <br /> negative positive inputs regulating SKN-1 allow fine-tuning           wdr-23 (Keith P. Choe, University Florida, Gainesville FL, <br /> stress resistance, growth immunity. An analogous deteriora-              USA), vhp-1 tir-1 (Source BioScience Geneservice, Cambridge, <br /> tion pathogen resistance excessive activation DAF-16            United Kingdom) used study. RNAi feeding E. coli <br /> [10] underscores necessity tight control stress responses       clones grown overnight LB medium containing 100 mg/ml <br /> avoid deleterious consequences infection.                             ampicillin. RNAi treatment performed described by <br />    Taken together, propose optimal enhancement                 Shapira et al. [22]. Worms grown RNAi bacteria from <br /> SKN-1 activity proper time-frame enhance immune                       hatching till young adult stage. If RNAi constructs were <br /> responses delay immunosenescence compromising                    used condition, ON cultures feeding bacteria strains <br /> longevity. In recent years, C. elegans versatile model          mixed equally. Empty vector containing HT115(DE3) <br /> not studying innate immunity, host-pathogen interactions, <br />                                                                              bacteria (EV) used control cases. <br /> but testing pharmacological interventions drug discovery <br /> [50]. The results presented prompt studies drugs <br /> targeting SKN-1/Nrf2 modulate innate immune response.                 Preparation pathogenic bacteria <br /> In conclusion, findings indicate intricate regulation                 Different human opportunistic bacteria, Pseudomonas <br /> innate immunity SKN-1, link pathogenic stress signaling            aeruginosa Enterococcus faecalis [28,52] ubiquitously used as <br /> xenobiotic oxidative stress responses.                                   pathogen models. Gram-negative Pseudomonas aeruginosa PA14 <br />                                                                              (David W. Wareham, Queen Mary University London, <br />                                                                              London, UK) Gram-positive Enterococcus faecalis SdB262 <br /> Materials Methods <br />                                                                              (Jonathan J. Ewbank, Centre d&#226;&#8364;&#8482;Immunologie Marseille- <br /> C. elegans strains maintenance                                           Luminy, Marseille, France) bacteria maintained and <br />    Nematodes maintained propagated E. coli OP50               prepared experiments described Powell Ausubel <br /> described Brenner [51] 20uC. The following C. elegans strains          [53]. BHI agar supplemented 100 mg/ml rifampicin for <br /> were obtained Caenorhabditis Genetics Center               E. faecalis killing assay. <br /> used study: N2, EU31 skn-1(zu67)IV/nT1[unc-?(n754) let- <br /> ?](IV;V), KU25 pmk-1(km25)IV., ZD101 tir-1(qd4)III. Further                  Killing assay <br /> strains used: LD001 Is007 [skn-1::gfp], CF1038 daf-                        Killing assays performed young adult animals at <br /> 16(mu86)I., CB1370 daf-2(e1370)III. (Tibor Vellai, Eo&#194;&#168;tvo&#194;&#168;s Lora&#194;&#180;nd          25uC slow killing plates (P. aeruginosa) rifampicin BHI plates <br /> University, Budapest, Hungary), LD1171 Is003 [Pgcs-1::gfp] (T.               (E. faecalis), noted figure legend. <br /> Keith Blackwell, Harvard Medical School, Boston MA, USA)                 Dead worms scored 12 hours till complete extinction <br /> MJCU017 kIs17[gst-4::gfp, pDP#MM016B]X. (Johji Miwa, Chubu                   population. Viability determined assaying for <br /> University, Kasugai, Japan). Nematodes treated cdc-                movement response gentle prodding. Worms died the <br /> 25.1(RNAi) avoid bacterial infection induced &#226;&#8364;&#732;bag worms&#226;&#8364;&#8482;           wall Petri dish crawled gel censored. 30 <br /> phenotype experiment.                                                 animals condition tested 3 parallel plates least <br />                                                                              independent trials, H2O2-concentration <br /> Crossing genotyping PCR                                               dependence preconditioned pathogen resistance estab- <br />    Pgcs-1::gfp;pmk-1(km25) Pgcs-1::gfp;tir-1(qd4) strains           lished trial. For studying pathogen resistance daf-2 <br /> created mating male pmk-1(km25) tir-1(qd4), respectively,              mutants, nematodes grown 15uC. For oxidative <br /> with LD1171 Is003 [Pgcs-1::gfp] hermaphrodites. Transgenic rol               preconditioning 2-day old adult animals treated 0 <br /> progeny isolated correct genotype scored PCR.             (control), 1 mM, 1.5 mM 2 mM H2O2 (Sigma) liquid <br /> PCR primers obtained Sigma. The primers pmk-1-OF                   NGM 2 hours 20uC. Before killing assay worms were <br /> (59-GGATACGGAAGAAGAGCCAATG-39) pmk-1-OR (59-                             transferred OP50 seeded NGM plates 12-hour recovery <br /> CAACAGTCTGCGTGTAATGC-39) used detect                             period. To test effect aging pathogen resistance, worms <br /> pmk-1(km25) deletion allele. The wild-type pmk-1 allele amplified          maintained OP50-seeded NGM plates the <br /> 1195-bp fragment compared 882-bp fragment pmk-                     challenge. To avoid &#226;&#8364;&#732;bag worms&#226;&#8364;&#8482; phenotype, animals were <br /> 1(km25) allele. Homozygous pmk-1(km25) mutants identified               fed cdc-25.1(RNAi). <br />  <br />  <br />        PLoS Pathogens | www.plospathogens.org                            9                           April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                                                      SKN-1 C. elegans Immunity <br />  <br />  <br />  <br /> Fluorescence microscopy                                                                           O62146, G5EGH6, Q8MNR8, Q19774, O02357, Q09321, <br />    Nematodes treated indicated figure legends                                      Q9UAQ9, P91316, Q20770, Q20840, G5EC22, P90893, <br /> (Figures 2, 3, 4 6). After treatments 40 worms                                   Q20968, Q21009, Q20117, Q9U2Q9, Q21355, Q9XW45, <br /> condition placed 2% agarose pad, immobilized                                     Q21381, Q09991, Q94269, Q94271, P34528, Q17446, Q2PJ68, <br /> adding 40 mM levamisole M9 buffer. Images taken                                      P34707, P41977, O02364, Q9XUC0, Q86DA5, Q10038, <br /> Leica DMI6000B epifluorescence microscope DFC480                                           P90794, Q8WRF1, Q9U309, G5EFR9, Q9GR66, G5ECJ8, <br /> camera. Epifluorescent microscopic images representatives                                  Q86S61, O76725, Q9N4X8, Q9NAB1, Q23498, Q23564 <br /> at 3 experiments. To analyze Pgcs-1::GFP GST-4::GFP                                       Human proteins/genes: Q12778, Q16656, Q16236, Q9Y4A8 <br /> expression PA14 infection, groups animals were <br /> determined depending detected GFP level intestine:                                  Supporting Information <br /> GFP positive GFP negative animals. To study nuclear <br /> localization SKN-1 response PA14 infection minimum 15                                    Figure S1 SKN-1 required pathogen resistance <br /> skn-1::gfp worms condition analyzed 3                                        P. aeruginosa E. faecalis. <br /> independent trials. Images captured Zeiss LSM510                                        (DOC) <br /> confocal laser scanning microscope equipped 406/1.3 oil                                    Figure S2 PA14-induced activation Pgcs-1::GFP and <br /> immersion objective (Plan-Neofluar, Zeiss).                                                       GST-4::GFP expression. <br />                                                                                                   (DOC) <br /> Analysis SKN-1 dependent targets genes <br />                                                                                                   Figure S3 Suppression PA14-induced SKN-1 nuclear <br /> down-regulated aging                                                                           localization tir-1(RNAi). <br />    A list 379 genes exhibiting significant age-dependent <br />                                                                                                   (DOC) <br /> decline expression (.10-fold d6 vs. d15) acquired <br /> from [27]. Data analyzed finding overlaps                                     Figure S4 Lifespan N2 skn-1(zu135) mutant <br /> genes subject SKN-1 dependent genes non-stress [29]                                   worms. <br /> oxidative stress conditions [30]. Then expression                                      (DOC) <br /> identified genes analyzed based Wormbase data, focusing <br />                                                                                                   Table S1 Statistical analysis killing assays. <br /> on PA14-, oxidative stress- PMK-1-dependent regulation [31]. <br />                                                                                                   (DOC) <br /> Oxidative stress tolerance                                                                        Table S2 List SKN-1-dependent genes down-regulat- <br />    Young, 1-day old adult worms incubated liquid NGM                                      ed aging. <br /> for 1 hour 20uC 3 mM 5 mM H2O2 (Sigma). After                                         (DOC) <br /> oxidative challenge animals transferred OP50 seeded <br />                                                                                                   Table S3 Statistical analysis lifespan assays. <br /> NGM plates viability tested 24 hours later. 35 animals per <br />                                                                                                   (DOC) <br /> plate examined condition 3 parallel plates 3 <br /> independent trials. <br />                                                                                                   Acknowledgments <br /> Statistical analysis                                                                              We thank T. Keith Blackwell, Johji Miwa, Tibor Vellai the <br />    Data analyzed using SPSS software 15.0 (SPSS Inc.,                                 Caenorhabditis Genetics Center C. elegans strains, T. Keith Blackwell, <br /> Chicago, IL, USA). Survival curves compared Kaplan-                                       Andy Golden, Keith P. Choe RNAi strains Jonathan J. Ewbank <br /> Meyer log-rank test. To compare means survival (oxidative                                  David W. Wareham Pseudomonas aeruginosa PA14 Enterococcus <br />                                                                                                   faecalis SdB262 strains, respectively, Wormbase collecting and <br /> tolerance assay) GFP expression Pgcs-1::gfp, gst-4::gfp,                            providing data C. elegans. We grateful Attila Mo&#194;&#180;csai Gergo&#203;? <br /> skn-1::gfp strains variables analyzed one-way ANOVA test.                                 Szanda (Semmelweis University) help fluorescence/confocal <br /> Results expressed mean 6 standard deviation (SD).                                          microscopy, Beatrix Gila&#194;&#180;nyi technical help, Jennifer Tullet and <br /> Statistical significance indicated follows: * p,0.05,                                      members So&#203;?ti Group discussions, Tibor Vellai Tama&#194;&#180;s <br /> ** p,0.001, *** p,0.0001.                                                                         Korcsma&#194;&#180;ros critical reading manuscript Editor the <br />                                                                                                   anonymous Reviewers helpful comments. <br /> Accession numbers <br />  C. elegans proteins/genes: Q17941, Q9XTG7, Q2MGF0,                                               Author Contributions <br /> Q17450, O61213, P54145, Q9U3Q6, O02215, G5EC10,                                                   Conceived designed experiments: DP CS. Performed the <br /> Q18198, Q9XUH3, Q9XUF9, O44552, Q27487, Q18938,                                                   experiments: DP. Analyzed data: DP CS. Contributed reagents/ <br /> O17725, O16849, Q968Y9, Q9XVB4, Q9XVA9, Q19223,                                                   materials/analysis tools: PC. Wrote paper: DP CS. <br />  <br /> References <br /> 1.   Bolm M, Jansen WT, Schnabel R, Chhatwal GS (2004) Hydrogen peroxide-                         5.   Mohri-Shiomi A, Garsin DA (2008) Insulin signaling heat shock <br />      mediated killing Caenorhabditis elegans: common feature different                         response modulate protein homeostasis Caenorhabditis elegans intestine <br />      streptococcal species. Infect Immun 72: 1192&#226;&#8364;&#8220;1194.                                                infection. J Biol Chem 283: 194&#226;&#8364;&#8220;201. <br /> 2.   Jansen WT, Bolm M, Balling R, Chhatwal GS, Schnabel R (2002) Hydrogen                        6.   Garsin DA, Villanueva JM, Begun J, Kim DH, Sifri CD, et al. (2003) Long- <br />      peroxide-mediated killing Caenorhabditis elegans Streptococcus pyogenes. Infect             lived C. elegans daf-2 mutants resistant bacterial pathogens. Science 300: <br />      Immun 70: 5202&#226;&#8364;&#8220;5207.                                                                              1921. <br /> 3.   Mahajan-Miklos S, Tan MW, Rahme LG, Ausubel FM (1999) Molecular                              7.   Singh V, Aballay A (2006) Heat-shock transcription factor (HSF)-1 pathway <br />      mechanisms bacterial virulence elucidated using Pseudomonas aeruginosa-                      required Caenorhabditis elegans immunity. Proc Natl Acad Sci U S A 103: <br />      Caenorhabditis elegans pathogenesis model. Cell 96: 47&#226;&#8364;&#8220;56.                                        13092&#226;&#8364;&#8220;13097. <br /> 4.   Chavez V, Mohri-Shiomi A, Garsin DA (2009) Ce-Duox1/BLI-3 generates                          8.   Richardson CE, Kooistra T, Kim DH (2010) An essential role XBP-1 in <br />      reactive oxygen species protective innate immune mechanism                                host protection immune activation C. elegans. Nature 463: 1092&#226;&#8364;&#8220; <br />      Caenorhabditis elegans. Infect Immun 77: 4983&#226;&#8364;&#8220;4989.                                               1095. <br />  <br />  <br />  <br />  <br />         PLoS Pathogens | www.plospathogens.org                                               10                                 April 2012 | Volume 8 | Issue 4 | e1002673 <br />                                                                                                                                                    SKN-1 C. elegans Immunity <br />  <br />  <br /> 9.    Chavez V, Mohri-Shiomi A, Maadani A, Vega LA, Garsin DA (2007) Oxidative                  29. Oliveira RP, Porter Abate J, Dilks K, Landis J, Ashraf J, et al. (2009) Condition- <br />       stress enzymes required DAF-16-mediated immunity generation                    adapted stress longevity gene regulation Caenorhabditis elegans SKN-1/Nrf. <br />       reactive oxygen species Caenorhabditis elegans. Genetics 176: 1567&#226;&#8364;&#8220;1577.                Aging Cell 8: 524&#226;&#8364;&#8220;541. <br /> 10.   Singh V, Aballay A (2009) Regulation DAF-16-mediated Innate Immunity                30. Park SK, Tedesco PM, Johnson TE (2009) Oxidative stress longevity in <br />       Caenorhabditis elegans. J Biol Chem 284: 35580&#226;&#8364;&#8220;35587.                                         Caenorhabditis elegans mediated SKN-1. Aging Cell 8: 258&#226;&#8364;&#8220;269. <br /> 11.   Kim DH, Feinbaum R, Alloing G, Emerson FE, Garsin DA, et al. (2002) A                     31. Yook K, Harris TW, Bieri T, Cabunoc A, Chan J, et al. (2012) WormBase 2012: <br />       conserved p38 MAP kinase pathway Caenorhabditis elegans innate immunity.                   genomes, data, new website. Nucleic Acids Res 40: D735&#226;&#8364;&#8220;741. <br />       Science 297: 623&#226;&#8364;&#8220;626.                                                                     32. Cypser JR, Johnson TE (2002) Multiple stressors Caenorhabditis elegans induce <br /> 12.   Dong C, Davis RJ, Flavell RA (2002) MAP kinases immune response.                       stress hormesis extended longevity. J Gerontol A Biol Sci Med Sci 57: <br />       Annu Rev Immunol 20: 55&#226;&#8364;&#8220;72.                                                                   B109&#226;&#8364;&#8220;114. <br /> 13.   Schulenburg H, Hoeppner MP, Weiner J, 3rd, Bornberg-Bauer E (2008)                        33. Talalay P, Dinkova-Kostova AT, Holtzclaw WD (2003) Importance phase 2 <br />       Specificity innate immune diversity C-type lectin domain                 gene regulation protection electrophile reactive oxygen toxicity <br />       (CTLD) proteins nematode Caenorhabditis elegans. Immunobiology 213:                    carcinogenesis. Adv Enzyme Regul 43: 121&#226;&#8364;&#8220;134. <br />       237&#226;&#8364;&#8220;250.                                                                                  34. An JH, Blackwell TK (2003) SKN-1 links C. elegans mesendodermal specification <br /> 14.   Alper S, McBride SJ, Lackford B, Freedman JH, Schwartz DA (2007) Specificity                  conserved oxidative stress response. Genes Dev 17: 1882&#226;&#8364;&#8220;1893. <br />       complexity Caenorhabditis elegans innate immune response. Mol Cell             35. Motohashi H, Yamamoto M (2004) Nrf2-Keap1 defines physiologically <br />       Biol 27: 5544&#226;&#8364;&#8220;5553.                                                                           important stress response mechanism. Trends Mol Med 10: 549&#226;&#8364;&#8220;557. <br /> 15.   Tullet JM, Hertweck M, An JH, Baker J, Hwang JY, et al. (2008) Direct                     36. Kim J, Cha YN, Surh YJ (2010) A protective role nuclear factor-erythroid 2- <br />       inhibition longevity-promoting factor SKN-1 insulin-like signaling C.            related factor-2 (Nrf2) inflammatory disorders. Mutat Res 690: 12&#226;&#8364;&#8220;23. <br />       elegans. Cell 132: 1025&#226;&#8364;&#8220;1038.                                                             37. Nagai N, Thimmulappa RK, Cano M, Fujihara M, Izumi-Nagai K, et al. (2009) <br /> 16.   Inoue H, Hisamoto N, An JH, Oliveira RP, Nishida E, et al. (2005) The C.                      Nrf2 critical modulator innate immune response model uveitis. <br />       elegans p38 MAPK pathway regulates nuclear localization transcription                  Free Radic Biol Med 47: 300&#226;&#8364;&#8220;306. <br />       factor SKN-1 oxidative stress response. Genes Dev 19: 2278&#226;&#8364;&#8220;2283.                       38. Harvey CJ, Thimmulappa RK, Sethi S, Kong X, Yarmus L, et al. (2011) <br /> 17.   Bishop NA, Guarente L (2007) Two neurons mediate diet-restriction-induced         <br /> </body></html>