<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular cloning, structure and expression of the yeast <br /> proliferating cell nuclear antigen gene <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                              MATERIALS AND METHODS <br /> The budding yeast Saccharomyces cerevisiae is                         Strains and Media. The E. coli strain was DH1. Yeast strains <br /> proving <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> be an useful and accurate model for                        were prototrophic diploids NCYC239 and SKI (12), as well as <br /> eukaryotic DNA replication. It contains both DNA                      PY2/YM599 (ala ura3-52/ura3-52 trplA/trplA leu2-3,112/+ <br /> polymerase (I) and 6 (111). Recently, proliferating cell <br />                  a                                                    +/ade2-101 +/lys2-801 can1l+). All rich and minimal media <br /> nuclear antigen (PCNA), which in mammalian cells is                   were as described (13). <br /> an auxiliary subunit of DNA polymerase 6 and is <br /> essential for in vitro leading strand SV40 DNA                        Isolation of the gene for yPCNA. Fifty Ag of yPCNA, Fraction <br /> replication, was purified <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> yeast. We have now                     VmI (14) was purified by electrophoresis on a preparative 12% <br /> cloned the gene for yeast PCNA (POL30). The gene                      SDS-polyacrylamide gel. The protein was transferred <br /> codes for an essential protein of 29 kDa, which shows                 electrophoretically to a polyvinylidene difluoride membrane (15). <br /> 35% homology with human PCNA. Cell cycle                              Half of the appropriate membrane was directly subjected to <br /> expression studies, using synchronized cells, show                    sequencing on an automated Applied Biosystems sequenator. The <br /> that expression of both the PCNA (POL30) and the DNA                  other half was treated with trypsin, peptides released from the <br /> polymerase 6 (POL3, or CDC2) genes of yeast are                       membrane were separated by reverse phase HPLC and some of <br /> regulated in an identical fashion to that of the DNA                  these sequenced as well. A 41 aminoterminal sequence was <br /> polymerase (POLl) gene. Thus, steady state mRNA <br />                  a                                                    obtained from the intact protein. And an octapeptide sequence, <br /> levels increase 10 1 00-fold in late Gl phase, peak in <br />                          -                                            which was later shown to map near the carboxy terminus, was <br /> early S-phase, and decrease to low levels in late S-                  obtained from a tryptic fragment (Figure 2). Using yeast codon <br /> phase. In addition, in meiosis mRNA levels increase                   usage data, a sense strand 64-fold degenerate 44-mer <br /> prior to initiation of premeiotic DNA synthesis.                      oligonucleotide was made to amino acid residues 1-14 2/3 and <br />                                                                       an antisense strand 32-fold degenerate 42-mer oligonucleotide <br />                                                                       to residues 40-27 (16). These oligonucleotides, at 1 ,uM <br /> INTRODUCTION                                                          nucleotide each, were used as primers in a 25 cycle polymerase <br />                                                                       chain reaction (PCR) with total yeast DNA at 20 itg/ml as <br /> Evidence for the model that, in mammalian cells, DNA                  template (17). A 120 bp fragment was detected and purified by <br /> polymerase 6 is responsible for leading strand DNA replication        10% PAGE and used as a probe for the isolation of the full <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br /> (1) derives from studies of the proliferating cell nuclear antigen    gene from a library of 10-20 kb genomic DNA HindI <br /> (PCNA, or cyclin), which is a known cofactor of DNA                   fragments ligated into the centromere vector yCp5O. Subcloning <br /> polymerase 6 (2-4) in in vitro replication of SV40 DNA (5,6).         using the 120 bp PCR fragment as a probe located the gene to <br /> The high degree of structural and functional conservation of          a 1.1 kb MluI-XbaI fragment, which was sequenced using <br /> components of the DNA replication apparatus between                   sequenase (United States Biochemicals) and the dideoxy <br /> mammalian cells and yeast (7) gives relevance to genetic              methodology. Both strands were completely sequenced from <br /> information obtained in yeast for understanding eukaryotic DNA        deletions generated with restriction enzyme cutting or by the use <br /> replication in general. Recently, we (8), and others (9), have        of appropriate synthetic oligonucleotide primers. <br /> shown that the polymerase subunit of DNA polymerase HI (10), <br /> the yeast analog of mammalian Pol6, is encoded by the CDC2            Plasmids. The 2.1 kb MluI fragment containing the yPCNA gene <br /> gene (11), indicating an essential function for this enzyme during    was ligated into the Sall site of pUC19 after both the MluI and <br /> the S phase of the yeast cell cycle. Here, we describe the cloning,   SaiI sites were filled in with DNA polymerase I, Klenow <br /> primary structure and cell cycle regulated expression of the yeast    fragment, and dNTP&apos;s (pBL203). The orientation of the insert <br /> PCNA (yPCNA) gene, and show that it is an essential gene as           in pBL203 was such that a 1.0 kb XbaI fragment could be deleted <br /> well.                                                                 by cutting and religating using the XbaI site in the polylinker <br />  <br />  <br /> *   To whom correspondence should be addressed <br />  262 Nucleic Acids Research <br /> of pUC1&apos;9 and a single XbaI site in the insert, resulting in plasmid                            fragment from plasmid yEp24 which contained the URA3 gene <br /> pBL205 (containing the full length yPCNA gene on the remaining                                  (18) was made blunt-end with DNA polymerase I, Klenow <br /> 1.1 kb IIlluI-XbaI fragment (Figure 1A). A 1.2 kb HindHI                                        fragment, and dNTP&apos;s and ligated into the EcoRV sites of plasmid <br />                                                                                                 pBL203. This replaced the 160 bp EcoRV fragment internal to <br /> A.                                                                                              the yPCNA gene by the URA3 gene (pBL206, Figure 1B). <br />          M                   S             RM RI RV                                     x       Plasmid pBL205 was cut with EcoRV and StyI and religated after <br />                                       -ORF -.                                                   filling in the StyI site, thus deleting a 360 bp fragment internal <br />                                        0.5                                 1.0 kb               to the yPCNA gene (pBL207). The URA3 gene was inserted in <br />                                                                                                 this plasmid into the HindlIl site present in the polylinker region <br />                                                                                                 (pBL208, Figure lC). Plasmid pBL203 was cut with BamHl and <br /> B. <br />      M        S      (RZ)                                               (R3Z)               x   HindII, which cut at sites in the pUC19 polylinker region such <br />               q     IUURA-3                                                             H       that the fragment contained the 2.1 kb MluI insert. The resulting <br />      0          0.5       1.0 1.5                                  2.0 kb                       fragment was ligated into the BamHI-HindIIl site of centromere <br />                                                                                                 vector yCp5O (pBL2 11). <br /> C.                                               HM        (S/R3)                   X <br />                            UR                                                                   Expression studies of the POL30 (yPCNA) gene. Strain NCYC239 <br />           0               0.5           1.0                  1.5 <br />                                                                    kb                           was synchronized by the feed-starve method (12,19). <br />                                                                                                 Synchronous entry into meiosis for strain SK 1 was affected <br /> Figure 1. Restriction maps of the yeast PCNA gene region and deletion derivatives.              through a shift to potassium acetate containing media (12,20). <br /> All fragments are inserted into pUC19 in a counter clockwise direction as described             At appropriate times cells were collected by centriguation and <br /> in Materials and Methods. A, The MluI-XbaI wild-type fragment in pBL205; <br /> B, The disruption construction in pBL206 used for insertional mutagenesis; C,                   frozen in dry ice. Total RNA was isolated by the glass- <br /> The deletion construction in pBL208 used for making a chromosomal deletion.                     bead method (21), separated by agarose gel electrophoresis <br /> Abbreviations are: M, MluI; S, StyI; RV, EcoRV; RI, EcoM; X, XbaI; H, Hindul.                   and transferred to Gene-screen (DuPont) according to <br />  <br />                                 1                                                       ACGCGTAACTTTTTTTTTTTGGATTTCAACTGATAGTTTTCGTAC <br />                             46      TTTGCTTCCTCTGGTACATAAAATTATATATAAGAAACACTTTTGCTTTAGCCTTCCTTTCTTTCCACTTGCACC <br />                            121 TTTCACTTTCGCCGTCCTTTTTCACTCACAGCAACAAGCAGCAAGCACTAAGTACGCAGTCAAAAGAGAGAAAAA <br />                            196 ATGTTAGAAGCAAAATTTGAAGAAGCATCCCTTTTCAAGAGAATAATTGATGGTTTCAAAGATTGTGTCCAGTTG <br />                              1 MetLeuGluAlaLysPheGluGluAlaSerLeuPheLysArgIleIleAspGlyPheLysAspCysvalGlnLeu <br />                            271 GTCAATTTCCAATGTAAAGAAGATGGTATCATTGCACAAGCTGTCGATGACTCAAGAGTTCTATTGGTCTCCTTG <br />                             26 ValAsnPheGlnCysLysGluAspGlyIleIleAlaGlnAlaValAspAspSerArgvalLeuLeuValSerLeu <br />                            346 GAAATAGGTGTCGAAGCCTTCCAAGAATATAGATGTGACCATCCTGTTACGTTAGGTATGGATCTAACCTCACTA <br />                             51 GluIleGlyValGluAlaPheGlnGluTyrArgCysAspHisProValThrLeuGlyMetAspLeuThrSerLeu <br />                            421      AGTAAAATCCTACGTTGTGGTAACAACACCGATACATTAACACTAATTGCTGACAACACACCGGATTCCATCATC <br />                             76      SerLysIleLeuArgCysGlyAsnAsnThrAspThrLeuThrLeuIleAlaAspAsnThrProAspSerIleIle <br />                            496      TTATTATTTGAGGATACCAAGAAAGACCGTATAGCCGAATACTCTCTGAAATTGATGGATATCGATGCTGATTTC <br />                            101      LeuLeuPheGluAspThrLysLysAspArgIleAlaGluTyrSerLeuLysLeuMetAspIleAspAlaAspPhe <br />                            571      TTAAAGATTGAAGAATTACAGTACGACTCCACCCTGTCATTGCCATCTTCCGAATTCTCTAAAATTGTTCGTGAC <br />                            126      LeuLysIleGluGluLeuGlnTyrAspSerThrLeuSerLeuProSerSerGluPheSerLysIleValArgAsp <br />                            646      TTGTCCCAATTGAGTGATTCTATTAATATCATGATCACCAAAGAAACAATAAAGTTTGTAGCTGACGGTGATATC <br />                            151      LeuSerGlnLeuSerAspSerIleAsnIleMetIleThrLysGluThrIleLysPheValAlaAspGlyAspIle <br />                            721      GGATCAGGTTCAGTCATAATAAAACCATTCGTGGATATGGAACATCCTGAAACAAGCATCAAACTTGAAATGGAT <br />                            176      GlySerGlySerValIleIleLysProPheValAspMetGluHisProGluThrSerIleLysLeuGluMetAsp <br />                            796      CAACCTGTCGACTTGACGTTCGGAGCTAAATATTTATTGGACATCATTAAGGGCTCCTCCTTTCTGATAGAGTT <br />                            201      GlnProValAspLeuThrPheGlyAlaLysTyrLeuLeuAspIleIleLysGlySerSerLeuSerAspArgval <br />                           871 GGTATCAGGCTCTCCAGCGAAGCTCCTGCTTTATTCCAAMTTTGATTAGAGTGGGTTCCTACAGTTTTTCTTG <br />                           226 GlyIleArgLeuSerSerGluAlaProAlaLeuPheGlnPhe  LeuLysSerGlyPheLeuGlnPhePheLeu <br />                           946 GCTCCTAAATTTAATGACGAAGAATAAATGTAAATTATCTATATAGTTGTATACTAAAAATAATAAACAAAAAM <br />                           251 AlaProLysPheAsnAspGluGlu <br />                          1021       AAACAGTAAAGTTTGTTTTAAATGAAAATAAATMCAAAGAAAATAAAGACTAAGTAGTCAGTTAATATCAGCAT <br />                          1096 TTTTGTGTGACTTATACAGTATTTATGACATATCTTACATTAATCTAGA <br /> Figure 2. DNA sequence of the Mlul-XbaI fragment and deduced protein sequence of yPCNA. The POL20 sequence data will appear in the EMBL/GenBank/DDBJ <br /> Nucleotide Sequence Databases under the accession number X 16676. The underlined amino acid residues correspond to those obtained from protein sequencing <br /> of the intact protein (residues 1 -41), or a tryptic fragment thereof (residues 231 -240). Arrows indicate RNA start sites as mapped by primer extension analysis <br /> (34). Both start sites are used with approximately equal efficiency. The underlined DNA sequence is identical with a sequence upstream of the POL3 gene. <br />                                                                                                                           Nucleic Acids Research 263 <br />                                                                                       or a smaller derivative (pBL205) (see Materials and Methods). <br />                                   A&amp; AL AkAIL ift A&amp;   46LALAU, <br /> ACT 1 <br />                             ift <br />                                                                                       Further mapping and sequencing defined the gene to a MluI-XbaI <br />                                                                                       fragment (Figure IA). The DNA sequence of the 1.1 kb MluI- <br />                                                                                       XbaI fragment and the deduced amino acid sequence of yPCNA <br />  POL        1                               ii.r                                      are given in Fig. 2. An open reading frame for a 28,916 dalton <br />                                                                                       protein was found. This is larger than the M, of purified <br />  POL3                                                                                 yPCNA as observed by SDS-PAGE (26-27 kDa), suggesting <br />                                                                         -   l.  -:. <br />                                                                    ~:-- AI_PF.esr <br />                                                                                       that the purified yPCNA may be a proteolytic fragment. <br /> POL 30                                                            I-W       M A.- <br />                                                                                       However, amino acid sequence data from a tryptic fragment <br />                                                                                       obtained from pure yPCNA yielded sequence data near the <br />       100   - <br />                                                                                       carboxy terminus (residues 231-240), indicating that our purified <br />        90 <br />              - <br />  <br />                                                        POL30                          yPCNA could at most be missing the 18 carboxy terminal <br /> z 80-                                                                                 residues. In addition, we have recently expressed yPCNA in E. <br /> cc                                                                                    coli using an appropriate expression system, and the resulting <br /> E 70- <br />                                                                                       purified protein has exactly the same SDS-PAGE size and <br />        60                                                                             enzymatic properties as the protein purified from yeast (Burgers <br /> 0 <br />        50-.                                                                           et   al., unpublished results). <br /> E <br />                                                                                         A null mutant of yPCNA was created in vitro by deleting the <br /> &lt;     40-- <br />                                                                                       internal EcoRV fragment (160 bp) and inserting a 1.1 kb Hindu <br /> (&apos;3 <br />                                                                                       fragment from YEp24 containing the URA3 gene (Fig. iB). The <br /> (&apos;     20-                                                                            resulting modified POl30 gene was removed from the plasmid <br />        10. <br />                                                                                       by digestion with MluI and transformed into diploid strain <br />                                                                                       PY2/YM599 (see Materials and Methods) to disrupt one of the <br />          0.-     --. <br />                         v     e                                    I. <br />  <br />                                                                                       two copies of the gene by a double crossover event (26). Tetrad <br />                  0     30         60    90 120 150 180 210 240 <br />                                           Time (hours) <br />                                                                                       analysis of the sporulated diploid showed that the POl30 gene <br />                                                                                       is essential for spore viability. Of 18 tetrads analyzed, 10 <br />                                                                                       produced two viable spores and 8 produced one viable spore. <br /> Figure 3. Cell cycle regulated expression in mitotic cells. NCYC239 cells,            All viable spores were Ura-. Sporulation of the control diploid <br /> synchronized by the feed-starve method were shifted to rich media. Samples were       strain gave (of 10 tetrads dissected) 6 tetrads with 4 viable spores, <br /> removed at intervals and total RNA extracted for Northern blot analysis. A, The <br /> blot was probed with radioactively labelled fragments of the ACT], POL1, POL3         3 tetrads with 3 viable spores, and 1 tetrad with 2 viable spores, <br /> and POL30 genes. B, Densitometric scanning of the autoradiogram obtained from         indicating that the low spore viability of the parental strain was <br /> probing of the blot with the POL30 gene. The bar in the graph identifies the          the cause of the relatively high number of tetrads with 1 viable <br /> period of DNA synthesis (from ref. 12,20).                                            spore in the disruption strain. These results show that the POL30 <br />                                                                                       gene is essential for growth. A large chromosomal deletion of <br /> the manufacturer&apos;s directions. The blots were probed with                             the yPCNA gene was generated by transforming the PY2/YM599 <br /> 32P-labelled probes to the coding sequences of the ACT],                              diploid to Ura+ with the integrating plasmid pBL208 (Figure <br /> POLI, POl3 and POl30 genes (8,22,23). Probes were prepared                            IC). Cells which had then excised the plasmid by intramolecular <br /> using the random primer method (24).                                                  recombination leaving the deleted copy of the gene in the <br />                                                                                       chromosome were selected by plating on 5-fluoroorotic acid <br /> RESULTS                                                                               containing plates and subsequently screened by Southern blot <br />                                                                                       analysis (27,28). Sporulation and tetrad analysis showed that this <br /> The sequence of the amino-terminal 41 residues of yPCNA was                           deletion was also lethal. The deletion mutant, however, was <br /> determined on an automated sequenator from the protein bound                          complemented by a plasmid containing the POL30 gene on a 2.1 <br /> to a polyvinylidene difluoride membrane (15). Using this                              kb MluI fragment inserted into centromere vector yCp5O <br /> information, we made two oppositely-oriented degenerate                               (pBL21 1, see Materials and Methods) when the heterozygous <br /> oligonucleotides (see Materials and Methods) and subjected                            diploid was transformed with this plasmid prior to sporulation. <br /> them to the polymerase chain reaction (PCR) with genomic yeast                           190 bp upstream from the initiation codon is a MluI (ACGCGT) <br /> DNA (17). The resulting amplified 120 bp DNA fragment was                             site which is also present in one or more copies upstream from <br /> used as a probe to HindlHI-digested yeast genomic DNA in a                            a large number of genes involved in DNA metabolism that are <br /> Southern blot analysis. Hybridization was to a single 17 kb band.                     periodically expressed during the cell cycle (23). This sequence <br /> Accordingly, a yeast genomic library of 10-20 kb HindIll                              is also present upstream from the start site of the POL3 (CDC2) <br /> fragments was made in the centromere vector YCp5O. Three                              gene (8). Even more strikingly, a 14 bp stretch at the MluI site <br /> identical clones were obtained by screening this library with the                     (ACGCGTAACTTTTT) is identical for the POL3 (CDC2) and <br /> 120 bp fragment.                                                                      POL30 (yPCNA) genes (Ref. 8, Fig. 2), suggesting that both <br />    The yPCNA gene is designated POL30 because PCNA is a                               genes may be similarly regulated. Cell cycle dependent expression <br /> subunit of DNA polymerase 6 (RI) and the gene designation for                         of the POL3 and POl30 genes was measured after <br /> the catalytic polypeptide (formerly designated CDC2) should be                        synchronization of NCYC239 cells by the feed-starve method <br /> considered as POL3 (8,9). The POL30 gene was mapped to the                            (12,19). This type of synchronization gave results identical to <br /> left arm of chromosome II (25) (results not shown). After                             elutriation or ca-Factor synchronization when cell-cycle dependent <br /> restriction mapping, a 2.1 kb MluI fragment was subcloned into                        expression of the POLI and CDC9 genes was studied (12,20). <br /> pUC 19 in such a way that the MluI sites were regenerated and                         The timing of P013 and P0130 expression was indistinguishable <br /> all further manipulations were done with this plasmid (pBL203)                        from each other and from that of the POLl gene, with steady- <br />  264 Nucleic Acids Research <br />                                                                                      state mRNA levels of all three genes rising sharply in late GI, <br />                                                                                      continuing through S, and decreasing again in the late S phase <br />                                                                                      (Figure 3). As a control, expression of the actin gene (ACT]) <br />                                                                                      was cell-cycle independent. The poor signal obtained with the <br />                                                                                      POL3 probe was due to the low levels of POL3 mRNA (Figure <br />                                                                                      3). Quantitative dot blot analysis of steady state mRNA levels <br />                                                                                      in asynchronously growing cells showed that the abundance of <br />                                                                                      POL30 mRNA is 2-3 fold higher than POLI, and 10-fold higher <br />                                                                                      than POL3 mRNA (data not shown). <br />                                                                                         The POL30 and POLl mRNA levels also rose coordinately <br />                                                                                      when the diploid strain SK-1 was induced to sporulate in a <br />                                                                                      synchronous fashion (Figure 4). In these blots, however, the <br />                                                                                      POL3 (CDC2) mRNA could not be detected above background <br />                                                                                      noise. Again, increase in the mRNA levels precedes meiotic DNA <br />                                                                                      replication. <br />  <br />                                                                                      DISCUSSION <br />                                                                                      The use of amino acid sequence derived oppositely oriented <br />                                                                                      degenerate oligonucleotides as primers for a polymerase chain <br />                                                                                      reaction with total nuclear DNA proved to be a facile way to <br />                                                                                      generate a probe for isolation of the full-length gene (29). Only <br />                                                                                      one fragment of the expected size (120 bp) was produced. The <br />                                                                                      coding region of the POL30 gene is preceded by a 190 bp <br />                                                                                      regulatory region which contains a consensus TATA box, the <br />                                                                                      RNA start site(s) and a ACGCGT sequence (a Mlul site) <br />                                                                                      presumed to be required for cell cycle regulated gene expression <br />                                                                                      (23). This 190 bp upstream region was sufficient to allow <br />                                                                                      complementation of a POL30 deletion mutant by the wild-type <br />                                                                                      gene on a centromere plasmid. Whether the entire region is <br />                                                                                      required for proper expression has not yet been determined. <br />                                                                                        The putative amino acid sequence of yPCNA shows ca. 35 % <br /> Figure 4. Regulation of mRNA levels during meiosis. Late-log phase SK- 1 cells       homology to that of the human PCNA (30) (Fig. 2). While <br /> were transferred to sporulation medium and samples taken and processed as            yPCNA can fully stimulate calf thymus DNA polymerase 6 by <br /> described in Materials and Methods and in the Legend to Figure 3. The same           increasing its processivity of action, it cannot substitute for human <br /> RNA samples were used for different blots in A and B. A. The blot was probed <br /> with the POL30 gene and the autoradiogram scanned with a densitometer. The           PCNA in the in vitro SV40 DNA replication system (14). This <br /> bar indicates the region of premeiotic DNA synthesis (from ref. 12.20). B. The       low homology at the amino acid level can also explain why human <br /> blot was probed with the ACT] and POLI genes.                                        polyclonal antisera from lupus patients, which react with <br />  <br />  <br /> Figure 5. Protein comparison of human, yeast. and AcMNPV PCNA sequences. Protein sequence homology was determined using the Wilbur/Lipman PRTALN <br /> alignment program (35). Boxed residues indicate identity betwcen all three PCNA sequences. <br />  <br /> mammalian PCNA, do not react with yPCNA (14). Recently it      <br /> has been reported that the baculovirus Autographa californica                 <br /> nuclear polyhedrosis virus (AcMNPV) has a gene that encodes            <br /> a protein, ETL, that has significant homology to rat PCNA (31),                 <br /> which in turn is virtually identical to the human PCNA. The                    <br /> possibility that ETL functions similarly to yeast and calf thymus             <br /> PCNA is further supported by the finding that the AcMNPV also         <br /> encodes a DNA polymerase with 6-like properties, i.e. aphidicolin              <br /> sensitivity, and a 3&apos;-5&apos; exonuclease activity (32). This AcMNPV                   <br /> ETL protein is 34% homologous to yeast PCNA (Fig. 5). When                     <br /> all three PCNA sequences are analyzed together (human, yeast,            <br /> and AcMNPV), there are definite domains of homology which      <br /> might be important for protein-protein interaction with the 6                   <br /> polymerase (33).                                                                 <br />    These results indicate that yPCNA plays a necessary role in           <br /> yeast, presumably in its interaction with DNA polymerase 6          <br /> during DNA replication. This interaction appears to be conserved                 <br /> through evolution, since yeast PCNA can sustitute for calf thymus                   <br /> PCNA in vitro. Also, domains of homology between human,                          <br /> yeast and AcMNPV PCNA suggest a functional conservation                             <br /> between these three very diverse organisms.                                     <br />                                                                                       <br /> </body></html>