<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32 <br />  <br />                                                                                                                            Cell &amp; Bioscience <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                  Open Access <br />  <br /> Transient knockdown overexpression reveal a <br /> developmental role zebrafish enosf1b gene <br /> Steve Finckbeiner1,2, Pin-Joe Ko1, Blake Carrington3, Raman Sood1,3, Kenneth Gross4, Bruce Dolnick5, Janice Sufrin5 <br /> and Paul Liu1* <br />  <br />  <br />  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  Background: Despite detailed vivo knowledge glycolytic enolases bacterial non-enolase members <br />  superfamily, little known vivo function vertebrate non-enolase enolase superfamily <br />  members (ENOSF1s). Results previous studies suggest involvement b splice form ENOSF1 breast and <br />  colon cancers. This study used zebrafish (Danio rerio) vertebrate model ENOSF1b function. <br />  Results: Whole mount situ hybridization (WISH) showed zebrafish ENOSF1b (enosf1b) zygotic and <br />  expressed ubiquitously 24 hours post fertilization (hpf). After 24 hpf, enosf1b expression is <br />  restricted notochord. Embryos injected enosf1b-EGFP mRNA grew slower EGFP mRNA-injected <br />  embryos caught EGFP-injected embryos 48 hpf. Embryos injected ATG exon 10 enosf1b <br />  mRNA-targeting morpholinos kinked notochords, shortened anterior-posterior axes, circulatory edema. <br />  WISH ntl pax2a expression showed embryos injected morpholino deformed notochord <br />  pronephros. TUNEL staining revealed increased apoptosis peri-notochord region. <br />  Conclusions: This study report ENOSF1 function vertebrate shows ENOSF1 required <br />  embryonic development. Increased apoptosis following enosf1b knockdown suggests potential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> <br />  advantage increased ENOSF1b expression human cancers. <br />  <br />  <br /> Background                                                                        wide array substrates. Evolutionarily conserved acidic <br /> Sequence information computational techniques                             residues, located loops end b <br /> be used group proteins evolutionarily meaningful                          sheets lining C-terminal barrel ENOSF fold, <br /> families larger superfamilies. Dayhoff defined pro-                           coordinate essential magnesium shared all <br /> tein families groups proteins high (&gt;50%)                              ENOSFs. Within superfamily, different ENOSF <br /> sequence identity [1]. Members protein superfamilies                           families distinguishable identity third <br /> have lower sequence identities, statistically signifi-                        magnesium ligand different combinations of <br /> cant pairwise alignment scores. Both protein families                             general acid/<span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> catalytic residues ends the <br /> and protein superfamilies thought monophy-                              remaining b sheets C-terminal barrel [4,7]. <br /> letic [1-4]. Since Dayhoff&#226;&#8364;&#8482;s original work, newer semi-                             Two ENOSF families representatives in <br /> automated classification schemes revealed thousands                            eukaryotes. The ubiquitous glycolytic enolases reversibly <br /> protein superfamilies [5,6]. The enolase superfamily                              dehydrate 2-phosphogylcerate phosphoenolpyruvate <br /> (ENOSF), named enolase glycolysis, used                           [7,8]. Despite named mandelate racemase <br /> as model protein superfamily evolution [4,7]. Mem-                           (MR), members MR family measured <br /> bers enolase superfamily share common ENOSF                              enzymatic activities x-ray structures acid sugar <br /> fold catalyze common half reaction:                                dehydratases. While bacterial MR family members <br /> abstract protons adjacent carboxyl groups                               known activities x-ray structures with <br />                                                                                   bound ligands inhibitors [9-20], little known <br /> * Correspondence: pliu@mail.nih.gov                                               roles MR family members perform eukar- <br /> 1 <br />  Oncogenesis Development Section, National Human Genome Research              yotes. Some fungi catalytically active MR family <br /> Institute, 49 Convent Drive, Bethesda MD, 20892, USA <br /> Full list author information available end article <br />                                                                                   members participate sugar metabolizing <br />  <br />                                     &#194;&#169; 2011 Finckbeiner et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                     Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                     reproduction medium, provided original work properly cited. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                               Page 2 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> pathway allows fungi use sugar rham-        Ensembl databases followed MUSCLE alignment <br /> nose sole carbon source [21,22]. The best stu-      phylogenetic tree construction reveal the <br /> died group eukaryotic MR family members splice         ENOSF1b gene likely present common <br /> isoforms human ENOSF1 protein. Human cancer            ancestor cephalochordates (Ciona sp, Figure 1) and <br /> cell lines known ENOSF1 splice forms (a, b,        vertebrates lost different vertebrate classes. <br /> and g). Immunoblotting polyclonal monoclonal         The anole (Anolis) ENOSF1b chicken <br /> antibodies shows b isoform (hsENOSF1b)        (Gallus) zebra finch (Taeniopygia) ENOSF1b homo- <br /> major protein product human ENOSF1 locus               logues found. With exception guinea pig <br /> [23,24]. In cell culture, hsENOSF1b expression appears        (Cavia), rodents order appear lost are <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> regulated environmental factors cell         losing ENOSF1b homologues. The kangaroo rat <br /> density, time culture, exposure chemotherapy        (Dipodomys) 13-lined ground squirrel (Spermophi- <br /> drugs affect thymidylate synthase folate pathway      lus) predicted ENOSF1b genes missing <br /> enzymes [23-27]. Using patient samples Taiwa-          exons animals. Other sequences with <br /> nese population, Kuo et al [28] showed hsENOSF1b         missing exons (Additional file 3) excluded <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> is expressed breast tumor tissue surrounding       MUSCLE alignment phylogenetic tree construction. <br /> tissue. The g splice forms         While rabbit (Oryctolagus), guinea pig (Cavia), rat <br /> patient samples [28]. The group reported          (Rattus), rhesus monkey (Macaca) genomes pre- <br /> statistically significant decrease year survival   dicted contain <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> ENOSF1b homologue, <br /> colon cancer patients tumor hsENOSF1b expres-            predicted sequences divergent they <br /> sion compared colon cancer patients tumor          branch away known evolutionary relatives <br /> hsENOSF1b expression [29]. Taken together, vitro           (Figure 1). The mouse, pig, sheep iden- <br /> cell culture data [23-27] human patient data [28,29]      tifiable ENOSF1b homologue sequences their <br /> suggest connection hsENOSF1b expression             genomes. <br /> and cancer. <br />   In study, used zebrafish (Danio rerio)    The zebrafish genome contains length <br /> in vivo model ENOSF1b expression. Small adult size,        homologue hsENOSF1b <br /> ease spawning, rapidly developing glass-clear          Using hsENOSF1b ([GenBank:NP_059982.2]) the <br /> embryos make zebrafish attractive vertebrate model       query sequence, BLASTP searches latest version <br /> for study genes unknown function. Zebrafish       zebrafish genome revealed single length <br /> also sequenced genome, allowing easy gene          homologue human hsENOSF1b, [GenBank: <br /> finding straightforward comparison zebrafish           NP_001070210.2], named enosf1b. <br /> genes human genes [30-33]. Building previously        Further bioinformatic searching zebrafish genome <br /> published bioinformatics study [23], homology searches        Ensembl genome viewer revealed enosf1b is <br /> of zebrafish genome zebrafish           located linkage group 7, base pairs 59,391,419 <br /> hsENOSF1b homologue (enosf1b). We enosf1b           59,891,418 (Figure 2A). The predicted zebrafish <br /> is expressed ubiquitously early development         ENOSF1b (drENOSF1b) protein 71% identical hsE- <br /> then restricted primarily notochord 24     NOSF1b protein level. BLASTP searching the <br /> hours development. Phenotypes resulting               PDB [34] showed closest homologue hsE- <br /> enosf1b knockdown microinjection enosf1b-tar-           NOSF1b drENOSF1b solved structure and <br /> geted morpholino oligonucleotides provide evidence        known biochemical activity fuconate dehydratase <br /> enosf1b&#226;&#8364;&#8482;s involvement normal development: enosf1b-         (FucD, [PDB:1YEY]) plant pathogenic bacteria <br /> knockdown embryos deformed notochords. Further           Xanthomonas campestris [12]. Alignments protein <br /> characterization apoptosis cell cycle markers        sequence bl2seq algorithm hsE- <br /> show enosf1b-knockdown embryos increased            NOSF1b drENOSF1b 51% 54% identical to <br /> cell death tissues surrounding notochord,          FucD, respectively. MUSCLE alignment hsENOSF1b, <br /> while mitosis appears unaffected.                       drENOSF1b, FucD protein sequences (Figure 2B) <br />                                                               vertebrate ENOSF1b proteins share resi- <br /> Results                                                       dues required coordinating magnesium proton <br /> Phylogenetic analysis reveals complicated evolutionary      abstraction FucD [4,7,12]. <br /> history vertebrate ENOSF1b genes <br /> Protein sequences used phylogenetic analysis       The zebrafish homologue hsENOSF1 expressed <br /> be Additional file 1. The complete MUSCLE            development <br /> alignment length ENOSF1b homologues            Primers located ATG exon 1 3&#226;&#8364;&#8482; <br /> found Additional File 2. BLASTP searches NCBI           end exon 15 (primers &#226;&#8364;&#339;enosf1b length forward &#226;&#8364;&#339; <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                   Page 3 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  Figure 1 Phylogenetic tree length cephalochordate vertebrate hsENOSF1b homologues. Protein sequences Additional file 1 <br />  aligned MUSCLE guide tree constructed automatically. C. intestinalis C. savignyi tunicates genus Ciona. P. <br />  abelli P. pygmaeus orang-utans genus Pongo. <br />  <br />  <br /> and &#226;&#8364;&#339;enosf1b length reverse&#226;&#8364;?, Table 1) used                 Overexpression enosf1b-EGFP impairs embryonic <br /> amplify enosf1b cDNA, pooled RNA multiple                      development <br /> developmental stages. The result shows enosf1b                   Following microinjection EGFP mRNA, fluorescence <br /> expressed early embryonic development                         visible embryos epiboly and <br /> (Figure 3A). Sequencing RT-PCR products cloned                    48 hpf (Figure 4A-C). Embryos microin- <br /> into pEGFP-N1 (p-enosf1b-N1-EGFP) confirmed                          jected enosf1b-EGFP mRNA demonstrated fluorescence <br /> identity enosf1b (data shown).                                    earlier stages 48 hpf (Figure 4C). <br />   WISH performed enosf1b antisense probe                         When embryos staged 24 hpf, approximately <br /> zygote early cleavage stage embryos showed                       90% uninjected EGFP-injected embryos the <br /> that enosf1b zygotic expressed                    correct stage. Only 20-40% enosf1b-EGFP-injected <br /> early embryo (Figure 3B). Embryos undergoing early                       embryos reached 24 hpf stage (Figure 4D, top <br /> somitogenesis (10 somites) ubiquitously expressed                        graphs); suggesting developmentally <br /> enosf1b, embryos mid-somitogenesis (16                          delayed. This delay severe higher dose <br /> somites) stronger expression tissues                  enosf1b-EGFP injected (40% 60 pg/embryo <br /> eye, somite borders, notochord (Figure 3B). Ubiqui-                  20% 180 pg/embryo). When embryos all <br /> tous enosf1b expression decreased notochord                        groups staged 48 hpf, however, delay <br /> enosf1b expression 48 72 hpf embryos remained                      disappeared. Interestingly, proportionate recov- <br /> (Figure 3C). At 96 hpf, faint notochord                        ery correct stage enosf1b-EGFP-injected embryos <br /> enosf1b (see higher magnification panel Figure 3C)                    48 hpf accompanied loss enosf1b-EGFP <br /> expression expression pancreas (Figure 3C).                   fluorescence (Figure 4D, graphs), may <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                  Page 4 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  Figure 2 The zebrafish homologue hsENOSF1b. A: Genomic context enosf1b. Blue bars exons, grey dashed lines introns. B: <br />  Alignment enosf1b, hsENOSF1b, FucD. Residues marked black asterisk involved FucD proton abstraction. Conserved residues <br />  marked red asterisk stabilize magnesium ion required FucD&#226;&#8364;&#8482;s catalytic mechanism. <br />  <br />  <br /> suggest overexpressing enosf1b compatible                    The e10i10 morpholino causes mis-splicing enosf1b <br /> with normal embryonic growth.                                            RT-PCR used assay presence absence of <br />                                                                          altered enosf1b transcripts following injection the <br /> Two separate enosf1b-targeting morpholinos generate                      e10i10 MO. Using primers flank exon 10 (primer <br /> similar dose-dependent phenotypes                                        sequences Table 1), PCR cDNA individual <br /> Embryos injected enosf1b ATG MO 3.5                    standard control MO e10i10 MO-injected embryos <br /> 7 ng/embryo developed severe developmental defects                       (7ng/embryo, 48 hpf) revealed presence an <br /> when compared uninjected standard control-                         additional PCR product ~100 bp longer 804 bp <br /> injected embryos. ATG MO-injected embryos                          expected product (Figure 5C). Sequencing longer <br /> shortened anterior-posterior axis, pericardial edema, and <br /> kinked notochords (Figure 5A). Embryos injected with <br /> enosf1b e10i10 MO concentrations developed                   Table 1 Oligonucleotide primers used study. <br /> a similar <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, lesser severity (Figure 5A).                        Primer                      Primer sequence <br /> When compared clutches embryos injected 3.5                   Enosf1b length forward        ATGCTGGCGATCAAAATCATA <br /> ng MO, clutches embryos injected 7 ng                   Enosf1b length reverse        CTGCTGTTTCTCAATGGCTCT <br /> of MO higher percentage embryos with <br />                                                                          Enosf1b XhoI Kozak forward     ATACTCGAGGACACCATGCTGGCGATCAAA <br /> the respective MO phenotypes: percentage of <br />                                                                             Enosf1b SacII reverse       CTCGAGCCGCGGATTTTTCTGCTGTTTCTC <br /> embryos phenotype respective MO is <br />                                                                           Exon 10 flanking forward           GCCCTTGTGGAAGCTACTTG <br /> dose dependent (Figure 5B, panel). Embryos the <br /> phenotype MO died 48 hpf.                                    Exon 10 flanking reverse           GTGGGGCTTTTGAAGTGTTC <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                       Page 5 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />                                                                        mRNA ATG MO decreased percentage <br />                                                                        embryos ATG phenotype (Figure 5D, <br />                                                                        right panel, p = 0.0227). <br />  <br />                                                                        Notochord pronephros deformed enosf1b <br />                                                                        morphants <br />                                                                        The notochord phenotype confirmed WISH for <br />                                                                        notochord-specific transcription factor tail (ntl) <br />                                                                        (Figure 6A, left). While approximately dia- <br />                                                                        meter overall length, notochord e10i10 and <br />                                                                        ATG MO-injected embryos kinked multiple times. <br />                                                                        The expression gene wider distribution pat- <br />                                                                        tern, pax2a, unaffected brain thymus pri- <br />                                                                        mordium, slightly reduced prospective cranial <br />                                                                        nerves eye, expanded pronephros the <br />                                                                        ATG morpholino-injected embryos (Figure 6A, right). <br />  Figure 3 Expression enosf1b development. A: enosf1b <br />  expression measured RT-PCR. Primers length enosf1b        Knockdown enosf1b increases apoptosis does not <br />  (Table 1) used test presence absence enosf1b <br />                                                                        change mitosis zebrafish development <br />  staged embryo lysate. Bp = base pairs. B: Whole mount situ <br />  hybridization (WISH) sense antisense DIG-labelled            Results TUNEL staining area immedi- <br />  riboprobe early embryonic development. Negative control        ately surrounding notochord apoptotic <br />  WISH, DIG-labelled sense riboprobe, free WISH colored       cells ATG e10i10 MO-injected embryos when <br />  precipitate. C: WISH enosf1b later embryonic               compared standard control injected uninjected <br />  development. Except panel C, original magnification all <br />                                                                        embryos (Figure 6B-C). On hand, antibody <br />  photomicrographs lower right hand corner picture. <br />                                                                        staining specific marker mitotic cells, phos- <br />                                                                        phorylated histone H3, did change differ- <br /> PCR product revealed insertion intron 10 (Additional                ent experimental groups (Figure 6B-C). <br /> file 4). This confirmed presence mis-splicing <br /> induced e10i10 morpholino. As intron 10 con-                    Discussion <br /> tains number base pairs (76 bp), insertion              A previously published phylogenetics study the <br /> alters reading frame 3&#226;&#8364;&#8482; end enosf1b mRNA.                ENOSF1 gene [23] ENOSF1 homologues all <br />                                                                        major phyla plants. Expanding these <br /> Injecting morpholinos p53 mutant embryos does                 results, used bioinformatics animal <br /> alter ATG e10i10 morpholino-induced phenotypes                  homologues ENOSF1 splice form associated with <br /> The ATG, e10i10, standard control MOs (all 7 ng/                   increased cancer risk human patients: hsENOSF1b <br /> embryo) injected p53-/- mutant embryos [35]                  [28,29]. The overall topology ENOSF1b tree (Fig- <br /> and control wild-type embryos. As shown Figure 5B                   ure 1) follows generally accepted model evolu- <br /> (bottom panel), statistically significant differ-          tion chordates [36-40]. Birds apparently lost their <br /> ence percentage embryos ATG                    ENOSF1bs common ancestor of <br /> or e10i10 phenotype injected p53 mutant embryos                     reptiles birds (Figure 1). Within mammalian <br /> when compared injected wild type embryos.                           clade, branching <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> dominated miss- <br />                                                                        ing species rodents branch orders. <br /> The morpholino phenotypes specific knock                   The rat guinea pig ENOSF1bs cluster <br /> of enosf1b                                                             rabbit ENOSF1b outgroup. All <br /> Embryos injected enosf1b-EGFP mRNA                     sequences sister group primates <br /> (60 pg/embryo), ATG e10i10 MO (7 ng/                   ([36,41], see: [42,43]). The missing mouse ENOSF1b, <br /> embryo), combination mRNA                       diverged rat ENOSF1b, &#226;&#8364;&#339;normal&#226;&#8364;? ENOSF1b in <br /> MOs (final concentrations 60 pg/embryo enosf1b-                     guinea pig support hypothesis rodents are <br /> EGFP mRNA 7 ng/embryo MO). Figure 5D (left                         losing ENOSF1b genes. This unusual it <br /> panel) shows coinjecting enosf1b-EGFP mRNA                   seem. A recently published study details the <br /> the e10i10 MO decreased percentage embryos                      loss motilin gene receptor rodent <br /> with e10i10 phenotype. Results unpaired Student&#226;&#8364;&#8482;s               evolution [44]. On broader scale, work Hahn <br /> t test decrease statistically significant (p         colleagues measures kind loss (and gain) <br /> value = 0.0380). In addition, coinjecting enosf1b-EGFP                 mammalian lineages entire gene families [45]. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                   Page 6 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  Figure 4 In vivo expression enosf1b-EGFP compared EGFP. A-C: Lateral views representative embryos injected vitro <br />  transcribed mRNA encoding enosf1b-EGFP EGFP. Original magnification photomicrographs lower right hand corner each <br />  picture. D: Comparison effect injecting equal doses enosf1b-EGFP EGFP mRNA time embryos reach 24 hpf 48 hpf <br />  developmental stage. Data average independent experiments; &#226;&#8364;&#339;n&#226;&#8364;? values total experiments. Error bars standard <br />  deviation. Original magnification photomicrographs lower right hand corner picture. <br />  <br />  <br />   Of vertebrates typically used model systems              share conserved residues needed enzyme&#226;&#8364;&#8482;s cat- <br /> clearly identifiable genomic ENOSF1b, diploid                   alytic activity (Figure 2B) [4,7,12]. <br /> species suitable small scale laboratory work                    Results RT-PCR WISH experiments con- <br /> zebrafish, Danio rerio, frog, Xenopus tropicalis. X.             firm enosf1b expressed development (Fig- <br /> tropicalis developed molecular biology                   ure 3A-C). The pattern enosf1b loss during <br /> resources make zebrafish widely used model                   development suggests involvement drENOSF1 in <br /> vertebrate [46]. Homology searches zebrafish gen-                 notochord function [47-49]. It noted that <br /> ome zebrafish hsENOSF1b homologue                       faint notochord expression 4 dpf (see <br /> (enosf1b), protein product (drENOSF1b)                   higher magnification inset 3C) does photo- <br /> predicted 71% identical human hsENOSF1b. The                    graph well. While appears notochord signal is <br /> predicted drENOSF1b key features                      disappearing, possible decreased enosf1b <br /> human cancer-associated gene bacterial gene                 staining lack probe penetration. Embryos at <br /> with known function: drENOSF1b hsENOSF1b                       4 dpf beginning grow skin known for <br /> both close homologues (slightly &gt; 50% sequence identity)                 having weaker staining medial tissues and <br /> to bacterial protein fuconate dehydratase (FucD)                 higher relative viscera staining [50]. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                       Page 7 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  Figure 5 Morpholino (MO) antisense oligonucleotide knockdown enosf1b expression. A: MO phenotypes. Photomicrograph of <br />  uninjected, standard control (Std Ctrl), e10i10, ATG2 MO-injected embryos 48 hpf. Original magnification lower right hand corner. <br />  B (top panel): Effect injecting morpholinos different doses compared uninjected embryos. B (bottom panel): Morpholino <br />  phenotypes dependent p53 status. Data B average independent experiments; &#226;&#8364;&#339;n&#226;&#8364;? values reported total three <br />  experiments. Embryos scored MO phenotypes 48 hpf. C: RT-PCR uninjected, standard control, e10i10 injected single embryos. <br />  Morpholinos injected final concentration 7 ng/embryo. Single embryos collected 48 hpf processed RT- <br />  PCR described Methods. Bp = base pairs. D (left panel): Rescue e10i10 phenotype coinjection e10i10 morpholino enosf1b-EGFP <br />  mRNA. D (right panel): Titration ATG2 phenotype coinjecting ATG2 morpholino enosf1b-EGFP mRNA. See Methods MO RNA <br />  doses. Data average independent experiments; &#226;&#8364;&#339;n&#226;&#8364;? values total experiments. Error bars standard deviation. <br />  Embryos scored 48 hpf. Unpaired Student&#226;&#8364;&#8482;s t test used evaluate statistical significance observed differences. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                         Page 8 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br />  Figure 6 Characterizing enosf1b knockdown phenotype. A: WISH tail pax2a 36 hpf morpholino-injected embryos. Red, <br />  grey, blue, green arrows indicate pax2a staining pronephros, midbrain-hindbrain boundary, prospective cranial nerves, thyroid <br />  primordium respectively. B (left panel): TUNEL staining uninjected, std ctrl, e10i10, ATG2 injected 48 hpf embryos. B (right panel): Mitotic <br />  index measured uninjected, std ctrl, e10i10, ATG2 injected embryos antibody staining phospho-histone H3. C: Morpholino-injected <br />  embryos increased TUNEL staining unchanged pH3 staining. Data average 3 tail counts condition. Error bars standard <br />  deviation. YSE = yolk sac extension. Original magnification photomicrographs lower right hand corner picture. <br />  <br />  <br />    Expression enosf1b manipulated zebra-                      phenotype: developmental delay 24 hpf (Figure 4A-C). <br /> fish development injecting enosf1b-EGFP mRNA                          While EGFP-injected embryos essentially identical <br /> antisense morpholino oligonucleotides 1-2 cell stage                   uninjected embryos, significantly fewer enosf1b-EGFP- <br /> embryos. Injection enosf1b-EGFP mRNA 1-2 cell                       injected embryos progressed stage of <br /> embryos led relatively weak gain-of-function                           development control embryos. Interestingly, the <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                              Page 9 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> enosf1b-EGFP-injected embryos caught EGFP-          primordium, pronephros. The WISH data pax2a <br /> injected uninjected embryos 48 hpf, exactly       suggests enosf1b involved normal develop- <br /> the enosf1b-EGFP signal disappeared (Figure 4D). The          ment tissues. <br /> relatively mild gain-of-function phenotype               Other assays performed anti-enosf1b morpholino- <br /> explained instability enosf1b-EGFP mRNA and/        injected embryos reveal changes peri-noto- <br /> or enosf1b-EGFP protein [51,52].                          chord tissues. When compared standard control <br />    Morpholino antisense oligonucleotides used         injected uninjected embryos, anti-enosf1b morpho- <br /> generate enosf1b loss-of-function phenotypes. Multiple        lino-injected embryos increased TUNEL staining <br /> lines evidence knockdown phenotypes          [56] peri-notochord tissues. The increased apop- <br /> observed specific enosf1b knockdown. First,        tosis apparently dependent stalled cell cycle. <br /> morpholinos target different evolutionarily con-         Whole mount antibody staining accepted prolif- <br /> served regions enosf1b transcript led similar       eration marker, phospho-histone H3 (pH3) shows that <br /> phenotypes (Additional file 5 Figure 5A). The sec-        visible change cell cycle enosf1b- <br /> ond line evidence supporting specificity        knockdown embryos (Figure 6B). Experimental embryos <br /> observed knockdown phenotypes provided RT-              equivalent pH3 staining, indicating change in <br /> PCR enosf1b following injection e10i10 mor-        achieving M-phase. The TUNEL phospho-histone <br /> pholino. Single embryo RT-PCR shows embryos              H3 (pH3) results uninjected embryos shown Fig- <br /> with e10i10 phenotype express longer transcript         ure 6B similar TUNEL results reported elsewhere <br /> (Figure 5C). Sequencing longer product revealed           [56,57]. Additional support cell cycle-independent <br /> that intron 10 inserted mis-spliced e10i10        increase apoptosis provided previously <br /> transcript (see Additional file 4). Insertion intron 10    described p53-independence morpholino-induced <br /> should cause frame shift remaining coding              phenotypes (Figure 5B, panel). Increased apop- <br /> sequence. This effectively removes 4 8 evolutiona-     tosis following enosf1b knockdown especially interest- <br /> rily conserved catalytic residues (see Figure 2). The   ing given recent clinical studies implicating human <br /> line evidence supporting enosf1b-specificity    ENOSF1b risk factor separate cancers <br /> morpholino phenotypes provided results           [28,29]. Perhaps increased ENOSF1b confers survival <br /> injecting morpholinos p53 mutant embryos.            advantage tumour cells interfering apoptosis <br /> Some morpholino phenotypes, particularly brain apopto-        [58-61]. <br /> sis, shown caused p53-mediated anti-            The increased apoptosis limited area around <br /> sense response [53]. The p53 mutant line expresses        notochord (see especially e10i10 panel Figure 6B). <br /> full length p53 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>mutation</span> transactivation     There possible explanations change. <br /> domain initiate p53-dependent transcription        First, enosf1b involved production pro- <br /> [35]. Enosf1b-targeting morpholino-injected p53 mutant        survival signal consumption pro-death signal by <br /> embryos indistinguishable morpholino-injected        notochord. When enosf1b-targeting morpholinos are <br /> wild type embryos (Figure 5B, panel). The fourth       injected, enosf1b decreases pro-survival signal <br /> line evidence supporting enosf1b-specificity    disappears pro-death signal increases, leading to <br /> morpholino-induced phenotypes results           increased apoptosis. This idea fits previous discov- <br /> mRNA-morpholino coinjection, rescue, experiments.          ery soluble signal produced hsENOSF1b-overex- <br /> Coinjecting enosf1b morpholinos         pressing drug resistant cancer cells [62] known <br /> &#226;&#8364;&#339;rescues&#226;&#8364;? decreases morpholino-induced pheno-          signalling role notochord. In addition providing <br /> type (Figures 5D).                                            structural support developing embryo, noto- <br />    The striking visibly observable characteristic     chord secretes soluble signalling proteins like hedgehog <br /> both morpholino-induced phenotypes kinked              known involved inducing formation of <br /> notochord. WISH shows enosf1b strongly                ventral neural tube [49,63]. Notochord-produced <br /> expressed notochord late somitogenesis         hedgehog probably plays role somite differen- <br /> 48 hpf. WISH notochord-specific gene, tail           tiation inhibition cardiac mesoderm fate para- <br /> [54], clearly highlights severely deformed notochord      xial mesoderm. Removing notochord, interfering <br /> in enosf1b morpholino knockdown embryos (Figure 6A).          hedgehog signalling, interferes pancreas <br /> The salient feature knockdown phenotypes pro-          development [48,49]. In addition hedgehog, noto- <br /> duced different anti-enosf1b, kinked notochord        chords different species known secrete many <br /> 48 hpf, precisely enosf1b highly         soluble proteins growth factors like BMPs, TGFbs, <br /> expressed. WISH Pax2a [55] performed.            FGFs [47,49]. A simpler interesting pos- <br /> Pax2a transcription factor involved devel-    sibility explain increased peri-notochord <br /> opment eye, brain, cranial neurons, thyroid            increase apoptosis: enosf1b expression is <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                               Page 10 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> low affected morpholino knockdown.            100% v/v methylene blue, 8 L water, [75]) until <br /> Cells decreased enosf1b survive die.          reached desired stage. <br />  <br /> Conclusions                                                   Whole mount In Situ Hybridization (WISH) <br /> Given recent publications identifying hsENOSF1b           A ZGC cDNA clone corresponding truncated zeb- <br /> expression risk factor human cancers     rafish homologue human ENOSF1b (enosf1b) the <br /> [28,64], developing vivo animal model                vector pME18S-FL (p-enosf1b) purchased a <br /> ENOSF1b takes added importance. The results             commercial supplier (OPEN Biosystems, Hunstville, <br /> the WISH study presented represent time        AL). Following confirmatory sequencing NHGRI <br /> a vertebrate ENOSF1b visualized intact         zebrafish core, MUSCLE alignment sequence to <br /> model organism. The results mRNA morpho-           identified length zebrafish homologue hsE- <br /> lino experiments overexpression knock-      NOSF1b showed 3&#226;&#8364;&#8482; end clone contained <br /> down phenotypes reported vertebrate ENOSF1b.            multiple gaps non-existent homology polyA <br /> One major findings, increased apoptosis following      tail. As multiple regions gaps polyA tails are <br /> in vivo ENOSF1 knockdown, implications cancer         conducive successful WISH [50], fragment of <br /> therapy. Given results clinical studies         clone better homology 5&#226;&#8364;&#8482; end enosf1b <br /> [28,64] vitro cell culture data [23-27,62,65],   created. The XhoI-EcoRV fragment (532 bp) p- <br /> not unreasonable hypothesize targeting ENOSF1         enosf1b subcloned pBluescript SK(-) producing <br /> catalytic activity possible interactions pro-   plasmid used make enosf1b riboprobe: pBS- <br /> teins lead increased apoptosis tumours.             enosf1b-5&#226;&#8364;&#8482; (see Additional File 6 MUSCLE align- <br />                                                               ments illustrating process). Plasmids containing <br /> Methods                                                       sequences tail [54] Pax2a [55] kind <br /> Phylogenetics bioinformatics                              gifts Dr. Milton English (NIH, National Eye Institute). <br /> BLASTP [66-68], used NCBI map viewer [68]              Linearized plasmids used generate antisense or <br /> and Ensembl genome browser [69], used         sense digoxigenin (DIG)-labelled riboprobes using T3 or <br /> protein sequences vertebrate homologues hsE-            T7 polymerase RNA labelling kits (Roche, Indianapolis, <br /> NOSF1b latest genome version available. Ensembl        IN). Riboprobes purified free salts enzymes <br /> genome browser used locate chromosomal             using Megaclear kit (Ambion, Austin TX). Embryos <br /> location enosf1b, identify exon/intron bound-       staged according Kimmel et al [76] fixed in <br /> aries [69]. Top scoring BLAST hits used search        4% paraformaldehyde PBS overnight (4% PFA/PBS) at <br /> the Zebrafish Gene Collection (ZGC, [70]) length         4&#194;&#176;C. Embryos younger 24 hours post fertilization <br /> clone library. The hsENOSF1b sequence, [Genbank:              (hpf) manually dechorionated watchmaker <br /> NP_059982.2], used search Protein Data             forceps fixation. All reagents WISH proce- <br /> Bank [34] homologues known function. The             dure purchased Sigma. WISH performed <br /> program bl2seq used calculate percent identity         Thisse Thisse [50]. <br /> [67,71]. MUSCLE (2 iterations, CLUSTALW output) <br /> was used align nucleotide protein sequences        Morpholino (MO) antisense oligonucleotide design <br /> to generate phylogenetic trees [72]. JalView used      MOs purchased lyophilized powders Gen- <br /> visualize alignments [73]. NEBCutter used plan         eTools (Philomath, Oregon). In addition FITC- <br /> subcloning experiments [74].                                  tagged standard control MO (abbreviated &#226;&#8364;&#339;std <br />                                                               ctrl,&#226;&#8364;?, [77]), MOs target different evolutio- <br /> Zebrafish husbandry                                           narily conserved regions enosf1b transcript or <br /> Zebrafish maintained used following approved         pre-mRNA obtained (Table 2). The FITC- <br /> NHGRI ACUC protocols. Adult wild type zebrafish (EK           tagged MO designated &#226;&#8364;&#339;ATG&#226;&#8364;? targets conserved <br /> strain) p53-/- mutant zebrafish [35] main-           ATG exon 2 processed enosf1b transcript. The <br /> tained recirculating aquaculture 28.5&#194;&#176;C        FITC-tagged MO designated &#226;&#8364;&#339;e10i10&#226;&#8364;? targets the <br /> and 14:10 light:dark cycle Westerfield [75]. Single <br /> adults separated plastic dividers individual     Table 2 Morpholino antisense oligonucleotide (MO) <br /> pair breeding boxes (Dura-Cross, LPS Inc. Rochester           sequences used study. <br /> NY) night spawning. At light, divider               Name                        MO sequence <br /> was pulled adults allowed spawn natu-          Standard Control (std ctrl)    CCTCTTACCTCAGTTACAATTTATA <br /> rally. Eggs manually separated adults, <br />                                                                          e10i10              GTTTACCTTAGAGATGGAAGCATGA <br /> rinsed free faeces debris, raised 28.5&#194;&#176;C in <br />                                                                          ATG2                CAGAATAATCTGGATCTGTGTGTGCAT <br /> embryo media (480 mg Instant Ocean salts, 16 mL <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                            Page 11 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> exon10-intron10 splice junction (e10i10) enosf1b pre-      UV light. Mis-spliced exon 10 PCR products were <br /> mRNA. Exon 10 chosen targeting splice-             sequenced NHGRI zebrafish core. <br /> blocking MO contains 3 conserved residues <br /> necessary catalysis members MR            In vitro transcription capped EGFP-polyA enosf1b- <br /> family. Alignments protein products con-        EGFP-polyA mRNA <br /> served regions targeted morpholinos         Following digestion XhoI SacII, enosf1b was <br /> found Additional file 5. Morpholino sequences (Table       ligated pEGFP-N1 previously digested XhoI <br /> 2) silico tested specificity using         SacII, generating enosf1b fused frame N- <br /> BLASTN program [67,71] search alternative tar-         terminus EGFP (p-enosf1b-N1-EGFP). P-enosf1b-N1- <br /> gets zebrafish genome. Morpholinos resus-         EGFP sequenced NHGRI zebrafish core to <br /> pended RNase-free water quantified using             verify enosf1b amplified absence <br /> NanoDrop ND-1000 (Thermo Scientific) following                PCR-induced mutations (sequencing data available <br /> quantification instructions. Immediately use,          request). The NheI/NotI fragment p-enosf1b- <br /> diluted MOs heated 65&#194;&#176;C 10 minutes.               N1-EGFP subcloned downstream T7 promo- <br /> Diluted MOs stored room temperature              ter pcDNA3.1(-) (pcDNA3.1-enosf1b-EGFP-polyA). <br /> until injected.                                               The NheI/NotI fragment pEGFP-N1 sub- <br />                                                               cloned downstream T7 promoter pcDNA3.1(-) <br /> RT-PCR                                                        (pcDNA3.1-EGFP-polyA). <br /> Twenty zebrafish embryos following stages:          Both pcDNA3.1-EGFP-polyA pcDNA3.1-enosf1b- <br /> 1-8 cell, 25-50% epiboly, 5-20 somites, 24-48 hpf,        EGFP-polyA linearized 3&#226;&#8364;&#8482; polyadenylation <br /> 72-96 hpf dechorionated pronase needed           signal sequence phenol/chloroform extracted to <br /> and anesthetized Tricaine Westerfield [75].            remove restriction enzyme. The linearized plasmids <br /> Embryos rinsed 3 times sterile phosphate buf-         ethanol precipitated (post-digest cleanup per <br /> fered saline (PBS), drained residual PBS, snap fro-    [51]), resuspended nuclease-free H 2 O, used as <br /> zen liquid nitrogen, stored -80&#194;&#176;C. To assay         templates vitro transcription (mMessage mMa- <br /> developmental expression enosf1b, staged embryos           chine T7 Ultra vitro transcription kit (Ambion, Aus- <br /> were thawed ice, homogenized Trizol (Invitrogen,        tin TX)). Transcribed mRNA phenol/chloroform <br /> Carlsbad CA), total RNA purified Trizol pack-         extracted [78] quantified ND-1000 Nanodrop <br /> age instructions. Total RNA (1 &#206;&#188;g) used prepare        spectrophotometer (Thermo Scientific). The mRNA was <br /> cDNA using SuperScript III kit (also Invitrogen). Full      ethanol precipitated [78], resuspended nuclease-free <br /> length protein coding sequence zebrafish homolo-       H2O, stored aliquots -80&#194;&#176;C use. <br /> gue hsENOSF1 PCR-amplified (&quot;Full length&#226;&#8364;? pri- <br /> mers Table 1) using AmpliTaq DNA polymerase                Embryo microinjection <br /> (Applied Biosystems, Foster City CA) step          Glass capillaries (World Precision Instruments, 1 mm <br /> PCR protocol: 94&#194;&#176;C, 10 minutes; 40 cycles 94&#194;&#176;C/30 s,       inside diameter filament) pulled Kopf <br /> 62&#194;&#176;C/30 s, 72&#194;&#176;C/60 s (annealing temperature               730 vertical capillary puller. Using sequencing gel <br /> decreased 0.5&#194;&#176;C/cycle 7 cycles final anneal-   loading pipette tip, capillaries back-filled with <br /> ing temperature 58&#194;&#176;C); 72&#194;&#176;C 60 s. To clone        enosf1b-GFP GFP mRNA. Pulled capillaries bro- <br /> length enosf1b construction vitro transcrip-     ken narrow tip razor blade stereomi- <br /> tion vector, length protein coding sequence           croscope, calibrated mineral oil [51], used to <br /> enosf1b PCR amplified using proof-reading poly-         inject 0.4 nL pulses vitro transcribed mRNA into <br /> merase (Platinum Pfx, Invitrogen) primers designed        blastomeres 1-2 cell embryos. The proce- <br /> to add 5&#226;&#8364;&#8482; XhoI site, Kozak sequence, 3&#226;&#8364;&#8482; SacII       dure used prepare MO injection capillaries. For <br /> site PCR product (primer sequences Table 1,         MO injection, injection volume adjusted 1.4 <br /> PCR conditions above). To quantify mis-spliced        nL injection target cytoplasmic strea- <br /> enosf1b following e10i10 MO injection, single standard        mers immediately ventral blastomeres 1-2 cell <br /> control injected e10i10 MO-injected embryos           embryos. Coinjections rescue experiments used the <br /> processed total cDNA using scaled version        injection volume mRNA injection. Concentrations of <br /> the Trizol protocol outlined above. Primers flanking          stock mRNA stock MOs adjusted achieve <br /> enosf1b exon 10 (Table 1) used PCRs            final concentrations listed Results above. Follow- <br /> same samples [78]. Twenty percent PCR reactions            ing microinjection, embryos raised 28.5&#194;&#176;C and <br /> were run precast 1% Tris-Borate-EDTA agarose gels          periodically photographed using Leica MZ16F stereo- <br /> containing ethidium bromide. Gels photographed           microscope GFP epifluorescence optics. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                         Page 12 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> Scoring microinjection phenotypes statistics           RT. Equilibration buffer removed replaced with <br /> Fluorescence EGFP enosf1b-EGFP mRNA-                working enzyme solution (50-100 &#206;&#188;L liquid 20 <br /> injected embryos scored tailbud stage night     embryos). Embryos working enzyme solution were <br /> of day injection. GFP-negative embryos           incubated heat block 37&#194;&#176;C 60 minutes with <br /> injected clutches discarded. GFP assessed     300 RPM agitation. While incubating enzyme, Apop- <br /> stage scored surviving embryos 24 48 hpf.   tag Stop/Wash concentrate thawed diluted to <br />   FITC-negative embryos MO-injected clutches       working Stop/Wash solution (1 mL concentrate + 34 <br /> discarded night injection. FITC-positive embryos    mL ddH2O). Enzyme removed replaced 1 <br /> were scored MO-induced phenotypes daily          mL diluted Stop/Wash solution. Embryos incu- <br /> there surviving embryos (usually 48 hpf). &#226;&#8364;&#339;ATG     bated Stop/Wash solution RT 60 minutes with <br /> phenotype&#226;&#8364;? defined decreased anterior-posterior     agitation. Stop/Wash solution washed 3 <br /> axis, severe notochord kinks pericardial edema.        10 minute PBT washes agitation. The wash <br /> &#226;&#8364;&#339;E10i10 phenotype&#226;&#8364;? defined mild severe noto-     aspirated replaced anti-DIG horseradish <br /> chord kinks pericardial edema. Because FITC-       peroxidase conjugated monoclonal antibody (provided <br /> tagged MO masked GFP signal, coinjection           Apoptag kit) diluted 1:9,000 blocking buffer <br /> MO-injected embryos treated MO-injected       (2% Roche blocking reagent, 10% lamb serum, 1% <br /> the day: fluorescence-negative embryos dis-     DMSO ddH2O). Embryos incubated anti-DIG <br /> carded night injection. Control mRNA-           antibody overnight 4&#194;&#176;C. Antibody removed and <br /> injected embryos scored described above. Per-      embryos washed 4 times PBT 10 minutes at <br /> centage embryos clutch scored MO-         RT. Washed embryos stained Tris-buffered <br /> induced phenotype using criteria described above.      DAB, (3,3&#226;&#8364;&#8482;-Diaminobenzidine: SigmaFast, Sigma-Aldrich) <br /> All data values shown figures aver-   5 minutes RT dark. Immediately stain- <br /> age experiments standard deviation bars.     ing, embryos fixed 4% PFA/PBS. Tails of <br /> Total &#226;&#8364;&#339;n&#226;&#8364;? values totals independent   TUNEL-stained embryos photographed as <br /> experiments. When necessary, two-tailed Student&#226;&#8364;&#8482;s t test   described WISH above. TUNEL-positive spots above <br /> was used calculate p values. All statistics tests   yolk sac extension (YSE) counted 3 <br /> were calculated Microsoft Excel.                        embryos condition. Statistics performed <br />                                                            above. <br /> Whole mount TUNEL assay <br /> The protocol used modified package instruc-    Whole mount phosphohistone-H3 (pH3) antibody staining <br /> tions Apoptag TUNEL kit (Chemicon, also:        PH3 staining protocol developed protocol <br /> [56,57]). Fixed 36 hpf embryos, microcentrifuge         [57]. Embryos (36 hpf) fixed 4% PFA/PBS as <br /> tubes, gradually dehydrated minute room       above. Following fixation, PFA/PBS aspirated and <br /> temperature (RT) washes 25%, 50%, 75%, ethanol/         embryos incubated ice cold acetone seven <br /> PBS. After 100% ethanol washes, embryos       minutes, followed brief rinse ddH2O. Embryos <br /> stored overnight -20&#194;&#176;C. Embryos rehy-         washed PBT (1X PBS, 0.1% Tween 20) two <br /> drated PBT (1X PBS, 0.1% Tween 20) reversing         times minutes RT. The PBT wash was <br /> the order washes above. Embryos washed      aspirated replaced blocking buffer (2% Roche <br /> additional 2 times 100% PBT bleached        blocking reagent, 10% lamb serum, 1% DMSO in <br /> 3% peroxide PBT 10 minutes RT. After bleach-     ddH2O). Embryos blocked 30 minutes RT. <br /> ing, embryos washed PBT 3 times 10 min-        While blocking, rabbit anti-pH3 polyclonal antibody <br /> utes RT. Embryos permeabilized        (Santa Cruz Biotechnology) diluted 1:200 block- <br /> adding 5 uL 2 mg/mL proteinase K final PBT       ing buffer. After blocking, embryos incubated in <br /> wash (final volume 1 mL) incubating 10 minutes     diluted anti-pH3 antibody overnight 4&#194;&#176;C. Antibody in <br /> at RT. Proteinase K aspirated permeabilized        blocking buffer aspirated embryos washed <br /> embryos washed twice PBT (7 minutes RT)         4 15 minute PBT washes RT. While washing, goat <br /> and post-fixed 4% PFA/PBS 15 minutes        anti-rabbit HRP conjugated secondary antibody was <br /> RT. Trace amounts PFA removed exhaustive        diluted 1:1000 blocking buffer. The wash was <br /> PBT washes (five - minute washes RT). While        aspirated embryos incubated diluted anti- <br /> washing PBT, Apoptag equilibration reaction buf-    rabbit secondary antibody hours RT gen- <br /> fers thawed. Working enzyme solution         tle agitation. Secondary antibody aspirated and <br /> (154 &#206;&#188;L reaction buffer + 66 &#206;&#188;L enzyme tube            embryos washed 4 15 minute PBT washes at <br /> assayed). The 5 PBT washes aspirated       RT. Embryos stained DAB, fixed PFA, <br /> and replaced equilibration buffer 20 minutes   photographed described WISH. PH3- <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                                   Page 13 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> positive spots YSE counted 3                                 2.    Dayhoff MO: Origin Evolution Protein Superfamilies. Federation <br />                                                                                       Proceedings 1976, 35:2132-2138. <br /> embryos condition. Statistics performed                          3.    Dayhoff MO, McLaughlin PJ, Barker WC, Hunt LT: Evolution Sequences <br /> as above.                                                                             Protein Superfamilies. Naturwissenschaften 1975, 62:154-161. <br />                                                                                 4.    Gerlt JA, Babbitt PC, Rayment I: Divergent evolution enolase <br />                                                                                       superfamily: interplay mechanism specificity. Archives of <br /> Additional material                                                                   Biochemistry Biophysics 2005, 433:59-70. <br />                                                                                 5.    Andreeva A, Howorth D, Chandonia JM, Brenner SE, Hubbard TJP, <br />     Additional file 1: Protein sequences used phylogenetic analysis.               Chothia C, Murzin AG: Data growth impact SCOP <br />     NCBI Ensembl accession numbers follow genus numbers. FucD                   database: new developments. Nucleic Acids Research 2008, 36:D419-D425. <br />     fuconate dehydratase Xanthomonas.                                        6.    Greene LH, Lewis TE, Addou S, Cuff A, Dallman T, Dibley M, Redfern O, <br />                                                                                       Pearl F, Nambudiry R, Reid A, et al: The CATH domain structure database: <br />     Additional file 2: Alignment protein sequences used in <br />                                                                                       new protocols classification levels comprehensive <br />     phylogenetic analysis. Alignment MUSCLE. <br />                                                                                       resource exploring evolution. Nucleic Acids Research 2007, 35: <br />     Additional file 3: Protein sequences predicted ENOSF1bs                   D291-D297. <br />     missing exons used phylogenetic analysis. Ensembl                7.    Babbitt PC, Hasson MS, Wedekind JE, Palmer DRJ, Barrett WC, Reed GH, <br />     accession numbers follow genus name.                                          Rayment I, Ringe D, Kenyon GL, Gerlt JA: The enolase superfamily: A <br />     Additional file 4: E10i10-injected embryos express enosf1b                        general strategy enzyme-catalyzed abstraction alpha-protons <br />     transcript inserted intron 10. Sequence mis-spliced product               carboxylic acids. Biochemistry 1996, 35:16489-16501. <br />     generated e10i10-injected embryo cDNA PCR exon 10             8.    Pancholi V: Multifunctional alpha-enolase: role diseases. Cellular and <br />     flanking primers. Exons introns color-coded.                              Molecular Life Sciences 2001, 58:902-920. <br />                                                                                 9.    Rakus JF, Fedorov AA, Fedorov EV, Glasner ME, Hubbard BK, Delli JD, <br />     Additional file 5: Alignments conserved regions targeted by <br />                                                                                       Babbitt PC, Almo SC, Gerlt JA: Evolution enzymatic activities the <br />     morpholino antisense oligonucleotides used study. Alignment <br />                                                                                       enolase superfamily: L-rhamnonate dehydratase. Biochemistry 2008, <br />     MUSCLE. <br />                                                                                       47:9944-9954. <br />     Additional file 6: Creating enosf1b WISH plasmid. Sequences          10.   Rakus JF, Fedorov AA, Fedorov EV, Glasner ME, Vick JE, Babbitt PC, Almo SC, <br />     enosf1b plasmids p-enosf1b (Additional figure 1) pBS-enosf1b-5&#226;&#8364;&#8482;            Gerlt JA: Evolution enzymatic activities enolase superfamily: D- <br />     (Additional figure 2). See Methods rationale.                                 Mannonate dehydratase Novosphingobium aromaticivorans. <br />                                                                                       Biochemistry 2007, 46:12896-12908. <br />                                                                                 11.   Yew WS, Fedorov AA, Fedorov EV, Almo SC, Gerlt JA: Evolution of <br />                                                                                       enzymatic activities enolase superfamily: L-talarate/galactarate <br /> Acknowledgements                                                                      dehydratase Salmonella typhimurium LT2. Biochemistry 2007, <br /> Barbara Foster, Mike Moser, Jason Kirk, Avinash Srivatsan provided                46:9564-9577. <br /> critical evaluation multiple points conception                12.   Yew WS, Fedorov AA, Fedorov EV, Rakus JF, Pierce RW, Almo SC, Gerlt JA: <br /> development stages study. We thank Colleen Kane, Kevin Bishop,           Evolution enzymatic activities enolase superfamily: L-fuconate <br /> and members Dolnick, Sufrin, Liu Labs helpful                    dehydratase Xanthomonas campestris. Biochemistry 2006, <br /> discussions. This work supported The Intramural Program             45:14582-14597. <br /> the National Human Genome Research Institute, NIH.                              13.   Yew WS, Fedorov AA, Fedorov EV, Wood BM, Almo SC, Gerlt JA: Evolution <br />                                                                                       enzymatic activities enolase superfamily: D-tartrate <br /> Author details                                                                        dehydratase Bradyrhizobium japonicum. Biochemistry 2006, <br /> 1 <br />  Oncogenesis Development Section, National Human Genome Research                  45:14598-14608. <br /> Institute, 49 Convent Drive, Bethesda MD, 20892, USA. 2Program               14.   Gulick AM, Hubbard BK, Gerlt JA, Rayment I: Evolution enzymatic <br /> Molecular Pharmacology Cancer Therapeutics, Roswell Park Graduate                 activities enolase superfamily: Identification general acid <br /> Division, State University New York Buffalo, Elm Carlton Streets,           catalyst active site D-glucarate dehydratase Escherichia <br /> Buffalo NY, 14263, USA. 3Zebrafish Core, National Human Genome Research               coli. Biochemistry 2001, 40:10054-10062. <br /> Institute, 49 Convent Drive, Bethesda MD, 20892, USA. 4Department            15.   Gulick AM, Hubbard BK, Gerlt JA, Rayment I: Evolution enzymatic <br /> Molecular Cellular Biology, Roswell Park Cancer Institute, Elm                activities enolase superfamily: Crystallographic mutagenesis <br /> Carlton Streets, Buffalo NY, 14263, USA. 5Department Pharmacology              studies reaction catalyzed D-glucarate dehydratase from <br /> Therapeutics, Roswell Park Cancer Institute, Elm Carlton Streets, Buffalo         Escherichia coli. Biochemistry 2000, 39:4590-4602. <br /> NY, 14263, USA.                                                                 16.   Hubbard BK, Gulick AM, Babbitt PC, Rayment I, Gerlt JA: Evolution of <br />                                                                                       enzymatic activities enolase superfamily: Mechanism, structure, <br /> Authors&#226;&#8364;&#8482; contributions                                                                metabolic context glucarate dehydratase Escherichia coli. <br /> SF performed bioinformatics phylogenetics studies. SF PJK carried             Faseb Journal 1999, 13:A1446-A1446. <br /> out WISH experiments. SF did embryo microinjection experiments, TUNEL           17.   Wieczorek SJ, Kalivoda KA, Clifton JG, Ringe D, Petsko GA, Gerlt JA: <br /> and pH3 staining, performed statistics. RS SF designed morpholino             Evolution enzymatic activities enolase superfamily: <br /> sequences oligonucleotide sequences used primers. RS planned               Identification &#226;&#8364;&#339;new&#226;&#8364;? general acid catalyst active site D- <br /> guided sequencing experiments. BC performed sequencing experiments. SF,               galactonate dehydratase Escherichia coli. Journal American <br /> KG, BD, RS, JS, PL conceived study participated design          Chemical Society 1999, 121:4540-4541. <br /> of study helped draft manuscript. All authors approved       18.   Palmer DRJ, Hubbard BK, Gerlt JA: Evolution enzymatic activities the <br /> final manuscript.                                                                     enolase superfamily: Partitioning reactive intermediates (D)- <br />                                                                                       glucarate dehydratase Pseudomonas putida. Biochemistry 1998, <br /> Competing interests                                                                   37:14350-14357. <br /> The authors declare competing interests.                      19.   Gulick AM, Palmer DRJ, Babbitt PC, Gerlt JA, Rayment I: Evolution of <br />                                                                                       enzymatic activities enolase superfamily: Crystal structure (D)- <br /> Received: 4 July 2011 Accepted: 26 September 2011                                     glucarate dehydratase Pseudomonas putida. Biochemistry 1998, <br /> Published: 26 September 2011                                                          37:14358-14368. <br />                                                                                 20.   Hubbard BK, Koch M, Palmer DRJ, Babbitt PC, Gerlt JA: Evolution of <br />                                                                                       enzymatic activities enolase superfamily: Characterization the <br /> References                                                                            (D)-glucarate/galactarate catabolic pathway Escherichia coli. <br /> 1. Dayhoff MO: Computer-Analysis Protein Sequences. Federation                     Biochemistry 1998, 37:14369-14375. <br />     Proceedings 1974, 33:2314-2316. <br />  Finckbeiner et al. Cell &amp; Bioscience 2011, 1:32                                                                                                    Page 14 15 <br /> http://www.cellandbioscience.com/content/1/1/32 <br />  <br />  <br />  <br />  <br /> 21. Watanabe S, Saimura M, Makino K: Eukaryotic bacterial gene clusters         44. He J, Irwin DM, Chen R, Zhang YP: Stepwise loss motilin its <br />     related alternative pathway nonphosphorylated L-rhamnose                   specific receptor genes rodents. Journal Molecular Endocrinology <br />     metabolism. Journal Biological Chemistry 2008, 283:20372-20382.                  2010, 44:37-44. <br /> 22. Crystal structure rhamnonate dehydratase Gibberella zeae                45. Demuth JP, De Bie T, Stajich JE, Cristianini N, Hahn MW: The Evolution of <br />     complexed Mg. [http://www.pdb.org/pdb/explore/explore.do?                      Mammalian Gene Families. Plos One 2006, 1. <br />     structureId=3FXG].                                                              46. Flintoft L, Skipper M: Model organisms - Xenopus tropicalis goes genetic. <br /> 23. Liang P, Nair JR, Song L, McGuire JJ, Dolnick BJ: Comparative genomic               Nature Reviews Genetics 2006, 7:588-589. <br />     analysis reveals novel mitochondrial isoform human rTS protein             47. Cleaver O, Krieg PA: Notochord patterning endoderm. <br />     unusual phylogenetic distribution rTS gene. Bmc Genomics                 Developmental Biology 2001, 234:1-12. <br />     2005, 6.                                                                        48. Kim SK, Hebrok M, Melton DA: Notochord endoderm signaling is <br /> 24. Dolnick BJ, Lu K, Yin MB, Rustum YM: Recent advances study rTS            required pancreas development. Development 1997, 124:4243-4252. <br />     proteins. rTS expression growth response thymidylate           49. Stemple DL: Structure function notochord: essential organ <br />     synthase inhibitors human tumor cells. In Advances Enzyme                     chordate development. Development 2005, 132:2503-2512. <br />     Regulation Edited by: Weber G 1997, 37:95-109, Advances Enzyme               50. Thisse C, Thisse B: High-resolution situ hybridization whole-mount <br />     Regulation.                                                                         zebrafish embryos. Nature Protocols 2008, 3:59-69. <br /> 25. Black AR, Dolnick BJ: Expression rTS correlates altered growth          51. Gilmour DT, Jessen JR, Lin S: Manipulating Gene Expression Zebrafish. <br />     regulation thymidylate synthase. Cancer Research 1996, 56:700-705.               In Zebrafish: A Practical Approach. Edited by: Nusslein-Volhard C, Dahm R. <br /> 26. Dolnick BJ, Black AR, Winkler PM, Schindler K, Hsueh CT: rTS gene                   New York: Oxford University Press; 2002:121-144. <br />     expression associated altered cell sensitivity thymidylate           52. Hammerschmidt M, Blader P, Strahle U: Strategies Perturb Zebrafish <br />     synthase inhibitors. In Advances Enzyme Regulation Edited by: Weber G            Development. In The Zebrafish: Biology. Edited by: Detrich HW, Westerfield <br />     1996, 36:165-180, Advances Enzyme Regulation.                                    M, Zon LI. New York: Academic Press; 1999:. <br /> 27. Dolnick BJ, Black AR: Alternate splicing rTS gene product        53. Robu ME, Larson JD, Nasevicius A, Beiraghi S, Brenner C, Farber SA, <br />     overexpression 5-fluorouracil-resistant cell line. Cancer Research 1996,       Ekker SC: p53 activation knockdown technologies. PLoS Genet 2007, <br />     56:3207-3210.                                                                       3:787-801. <br /> 28. Kuo SJ, Wang HC, Chow KC, Chiou SH, Chiang SF, Lin TY, Chiang IP,               54. Schultemerker S, Vaneeden FJM, Halpern ME, Kimmel CB, Nussleinvolhard C: <br />     Chen DR: Expression rTS&#206;&#178; beta 5-fluorouracil resistance marker           No Tail (Ntl) Is Zebrafish Homolog Mouse-T (Brachyury) Gene. <br />     patients primary breast cancer. Oncology Reports 2008, 19:881-888.             Development 1994, 120:1009-1015. <br /> 29. Lin YL, Chow KC: rTS beta Novel 5-fluorouracil Resistance Marker        55. Krauss S, Johansen T, Korzh V, Moens U, Ericson JU, Fjose A: Zebrafish Pax <br />     Colorectal Cancer: A Preliminary Study. Annals Academy Medicine                  [Zf-a] - Paired Box-Containing Gene Expressed Neural-Tube. <br />     Singapore 2010, 39:107-111.                                                         Embo J 1991, 10:3609-3619. <br /> 30. Burgess S: New techniques using zebrafish model                    56. Webb AE, Kimelman D: Analysis Early Epidermal Development in <br />     development. Methods 2006, 39:181-182.                                              Zebrafish. 2004, 289:137-145. <br /> 31. Traver D, Herbomel P, Patton EE, Murphey RD, Yoder JA, Litman GW,               57. Shepard JL, Amatruda JF, Stern HM, Subramanian A, Finkelstein D, Ziai J, <br />     Catic A, Amemiya CT, Zon LI, Trede NS: The zebrafish model                     Finley KR, Pfaff KL, Hersey C, Zhou Y, et al: Zebrafish bmyb mutation <br />     organism study development immune system. Advances                     causes genome instability increased cancer susceptibility. <br />     Immunology, Vol 81 2003, 81:253-+.                                                  Proceedings National Academy Sciences United States of <br /> 32. Lohr JL, Yost J: Vertebrate model systems study early heart               America 2005, 102:13194-13199. <br />     development: Xenopus zebrafish. American Journal Medical                 58. Hector S, Prehn JHM: Apoptosis signaling proteins prognostic <br />     Genetics 2000, 97:248-257.                                                          biomarkers colorectal cancer: A review. Biochimica Et Biophysica Acta- <br /> 33. Kelly GM: Zebrafish: model elucidating mechanisms               Reviews Cancer 2009, 1795:117-129. <br />     development. Biochemistry Cell Biology-Biochimie Et Biologie Cellulaire     59. Singhal S, Vachani A, Antin-Ozerkis D, Kaiser LR, Albelda SM: Prognostic <br />     1997, 75:U3-U3.                                                                     implications cell cycle, apoptosis, angiogenesis biomarkers in <br /> 34. Bernstein FC, Koetzle TF, Williams GJB, Meyer EF, Brice MD, Rodgers JR,             non-small cell lung cancer: A review. Clinical Cancer Research 2005, <br />     Kennard O, Shimanouchi T, Tasumi M: Protein data bank - computer-                   11:3974-3986. <br />     based archival file macromolecular structures. Archives Biochemistry     60. Wang GD, Reed E, Li QDQ: Apoptosis prostate cancer: Progressive and <br />     Biophysics 1978, 185:584-591.                                                   therapeutic implications (Review). International Journal Molecular <br /> 35. Berghmans S, Murphey RD, Wienholds E, Neuberg D, Kutok JL,                          Medicine 2004, 14:23-34. <br />     Fletcher CDM, Morris JP, Liu TX, Schulte-Merker S, Kanki JP, et al: tp53        61. Shivapurkar N, Reddy J, Chaudhary PM, Gazdar AF: Apoptosis lung <br />     mutant zebrafish develop malignant peripheral nerve sheath tumors.                  cancer: A review. Journal Cellular Biochemistry 2003, 88:885-898. <br />     Proceedings National Academy Sciences United States         62. Dolnick BJ, Angelino NJ, Dolnick R, Sufrin JR: A novel function rTS <br />     America 2005, 102:407-412.                                                          gene. Cancer Biology &amp; Therapy 2003, 2:364-369. <br /> 36. Carroll RH: Vertebrate Paleontology Evolution New York: W.H. Freeman        63. Dodd J, Jessell TM, Placzek M: NEUROSCIENCE:The When Where of <br />     Co; 1988.                                                                       Floor Plate Induction. Book NEUROSCIENCE:The When Where Floor <br /> 37. Gnathostomata Jawed Vertebrates. [http://tolweb.org/Gnathostomata/                  Plate Induction 282:1654-1657, (Editor ed.^eds.) City; 1998: 1654-1657. <br />     14843].                                                                         64. Lin YL, Chow KC: rTS beta Novel 5-fluorouracil Resistance Marker of <br /> 38. Janvier P: The daw <br /> </body></html>