<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                            WRN helicase promotes repair of DNA double-strand breaks <br />                            caused by aberrant mismatch repair of chromium-DNA adducts <br />   <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Recent studies in yeast have found that processing of DNA-double strand breaks (DSB) for <br />                                  recombination repair involves Sgs1 helicase. Human cells have five Sgs1 homologues with <br />                                  unknown selectivity and significance for repair of different DSB types. Here we examined the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  importance of WRN helicase in repair of G2-specific DSB caused by abnormal mismatch repair <br />                                  (MMR) of ternary Cr-DNA adducts. We found that Cr(VI) induced a rapid dispersal of WRN <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                  the nucleolus resulting in its prolonged retention in the nucleoplasm. The loss of MSH2 or MLH1 <br />                                  MMR proteins abolished the long-term but not the initial WRN relocalization. WRN-deficient <br />                                  fibroblasts were hypersensitive <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> Cr(VI)-induced clonogenic death and contained high levels of <br />                                  persistent DSB detected by &#206;&#179;-H2AX/53BP1 foci and pulsed-field gel electrophoresis. WRN was <br />                                  involved in recombination repair of Cr-induced DNA damage, as evidenced by WRN-RAD51 <br />                                  colocalization and defective formation of RAD51 foci in the absence of WRN. The accumulation <br />                                  of unrepaired DSB in WRN-depleted cells was rescued by the inactivation of MMR, indicating <br />                                  that MMR-generated DSB were a key substrate for WRN action in Cr(VI)-treated cells. <br />                                  Competition for the limited amounts of WRN in primary cells between G2 processes of telomere <br />                                  rebuilding and recombinational repair is expected to increase persistence of Cr-induced DSB and <br />                                  may cause telomeric abnormalities in tissues of chronically chromate-exposed workers. Our work <br />                                  provides the first demonstration of the major importance of WRN in repair of the specific class of <br />                                  DSB in human cells. <br />  <br />  <br />                            Introduction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Werner syndrome (WS) is a rare genetic disorder of premature ageing that is manifested by <br />                                             the early appearance of gray hair, cataracts, scleroderma, diabetes and increased incidence of <br />                                             cancer.1&#226;&#8364;&#8220;3 WRN, the protein mutated in Werner syndrome, is unique among the five- <br />                                             member human RecQ helicase family as it contains both helicase and exonuclease <br />                                             activities.1,3 Normal localization of WRN is nucleolar4&#226;&#8364;&#8220;6 but in afflicted individuals, this <br />                                             protein is often missing the nuclear localization signal resulting in its exclusion from the <br />                                             nucleus.1,2 WS cells display a prolonged S-phase,7 low proliferative capacity3 and increased <br />                                             sensitivity to drugs with S-phase dependent toxicity.8&#226;&#8364;&#8220;10 The preferred substrates for WRN <br />                                             action include various types of 3- and 4-way DNA junctions, bubbles, extrahelical loops and <br />                                             DNA overhangs.3,11,12 These abnormal DNA structures can arise spontaneously in the areas <br />                                             of DNA repeats and they can be induced by incomplete repair or replication. WRN is <br />                                             capable of resolving a broad range of alternative DNA structures due to its ability to <br />                                             recognize these structures and process them via unwinding and cleavage reactions.1&#226;&#8364;&#8220;3 <br />  <br />  <br />                                             WRN interacts with several proteins involved in repair of DNA double-strand breaks <br />                                             (DSB)13&#226;&#8364;&#8220;18 and localizes to the sites of laser-induced DSB in live cells.19 However, WS <br />                                             cells exhibited only a mild sensitivity toward ionizing radiation in clonogenic <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             experiments,2,16 which was indicative of a minor if any role of WRN in the overall DSB <br />                                             repair. Recent studies have found that the yeast RecQ helicase Sgs1 stimulated <br />                                             recombination repair of DSB through its essential function in one of the two redundant DNA <br />                                             ends resection pathways.20,21 The RecQ helicase BLM appears to play a role analogous to <br />                                             that of Sgs1 in campthothecin-treated human cells where it acts in parallel with EXO1 to <br />                                             promote recombination-associated foci and <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>cell survival</span>.22 The presence of five RecQ <br />                                             members in human cells potentially indicates that individual helicases may participate in <br />                                             repair of different classes of DSB. It is currently unknown whether repair of specific DSB in <br />                                             human cells is particularly dependent on WRN or even involves this helicase. <br />  <br />                                             In this work, we investigated the importance of WRN in cellular survival and repair of DSB <br />                                             induced by the potent human carcinogen chromium(VI). The main source of DSB in Cr(VI)- <br />                                             treated human cells is misprocessing of Cr-DNA damage by mismatch repair (MMR).23&#226;&#8364;&#8220;25 <br />                                             The majority of Cr-induced DSB are formed in G2 phase following replication of Cr- <br />                                             adducted DNA.23,24 Ternary Cr-DNA adducts, such as cysteine-Cr-DNA or ascorbate-Cr- <br />                                             DNA crosslinks, were particularly good substrates for recognition by MMR proteins.24,25 <br />                                             These adducts were also a primary cause of the mutagenic responses generated in shuttle- <br />                                             vector plasmids damaged during Cr(VI) reduction in vitro with ascorbate26,27 or cysteine.28 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Chronic formation of toxic DSB by MMR activity during repetitive exposures to Cr(VI) has <br />                                             been proposed to promote the selection of transformation-prone, Cr-resistant cells lacking <br />                                             MMR, providing an explanation for the unusually high frequency of microsatellite <br />                                             instability (marker of inactive MMR) in lung cancers among chromate workers.29 Unlike <br />                                             other agents with MMR-dependent mechanisms of cytotoxicity,30&#226;&#8364;&#8220;32 the formation of DSB <br />                                             in Cr-exposed cells is unusually rapid which results from the unprecedented activation of <br />                                             both the MSH6 and MSH3 branches of MMR.25 MSH3 specifically recognizes and repairs <br />                                             mismatches containing insertion/deletion loops,33,34 pointing to a potential formation of <br />                                             these structures in Cr-damaged DNA. Extrahelical loops are also high affinity substrates for <br />                                             WRN helicase1&#226;&#8364;&#8220;3 but they can interfere with the activity of other DNA helicases. Therefore, <br />                                             it is possible that repair of MMR-generated DSB in Cr-damaged DNA may engage WRN. <br />                                             Cr-induced DSB are also well suited for testing the involvement of WRN in homologous <br />                                             recombination because it is a principal DSB repair process in S/G2 cells and the main target <br />                                             for the activity of RecQ helicases in lower organisms.20,21 <br />  <br />                               Materials and Methods <br />                               Cell cultures and drug treatments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Human SV40 transformed fibroblast lines GM00637H (NF cells) and GM00847 (LNS cells) <br />                                             were purchased from Coriell Cell Repository. SV40 immortalized Werner syndrome <br />                                             fibroblasts AG11395 (WS cells) were obtained from NIA Aging Cell Culture Repository. <br />                                             Primary human lung IMR90 fibroblasts and colon HCT116 (MLH1&#226;&#710;&#8217;/&#226;&#710;&#8217;) cells were purchased <br />                                             from the American Type Culture Collection. HCT116+ch3 (MLH1+) cell line was a gift <br />                                             from T. Kunkel. NF, WS and IMR90 fibroblasts were cultured in DMEM supplemented <br />                                             with penicillin-streptomycin and serum (15% for NF and WS, 10% for IMR90). LNS cells <br />                                             were grown in MEM with 10% serum. HCT116 cells were grown in DMEM-F12 medium <br />                                             containing 10% serum and penicillin-streptomycin. Media for HCT116+ch3 cells <br />                                             additionally included 600 &#206;&#188;g/ml geneticin. All cell lines were grown at 37&#194;&#176;C in a humidified <br />                                             atmosphere containing 95% air-5% CO2. Cells were exposed to K2CrO4 [Cr(VI)] for 3 hr in <br />                                             serum-free medium. To increase cellular ascorbate concentrations, cells were incubated for <br />                                             90 min with 1 mM dehydroascorbic acid in Krebs-HEPES buffer supplemented with 0.5 <br />                                             mM glucose.35 Cellular ascorbate was measured by a modified HPLC procedure based on <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                          Page 3 <br />  <br />  <br />                                             the fluorescent detection of a specific conjugate with 1,2-diamino-4,5-dimethoxybenzene <br />                                             dihydrochloride.36 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Clonogenic survival <br />                                             A total of 2000 cells were seeded onto 100-mm dishes one day prior to treatments with <br />                                             Cr(VI). Three dishes were used for each dose. Ten to 14 days after exposure, colonies were <br />                                             fixed with 100% methanol and stained with Giemsa solution and counted. Clonogenic <br />                                             experiments were repeated at least twice. <br />  <br />                               Western blotting <br />                                             Cells were collected by scraping, washed twice with cold phosphate-buffered saline (PBS) <br />                                             and resuspended in a buffer containing 50 mM Tris pH 8.0, 250 mM NaCl, 1% NP40, 0.1% <br />                                             SDS, 5 mM EDTA, 2 mM Na3VO4, 10 mM Na2P2O7, 10 mM NaF, 1 mM PMSF and <br />                                             protease inhibitor cocktail. Lysates were incubated on ice for 10 min and then cell debris <br />                                             was spun down at a speed of 10,000 &#195;&#8212; g for 5 min at 4&#194;&#176;C. Samples were denatured by 2X <br />                                             loading buffer with 10% 2-mercaptoethanol and boiled for 10 min. Proteins were separated <br />                                             by SDS-polyacrylamide gel electrophoresis and electrotransferred to the ImmunoBlot PVDF <br />                                             membrane (Bio-Rad). Primary antibodies were rabbit polyclonal anti-WRN (Santa Cruz <br />                                             Biotechnology), mouse anti-MLH1, anti-MSH6 and anti-MSH2 (BD Biosciences). Protein <br />                                             bands were visualized using horseradish peroxidase-conjugated secondary antibodies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             (Upstate) and an enhanced chemiluminescence kit (Amersham). <br />  <br />                               Stable shRNA knockdowns <br />                                             Stable depletion of protein levels was achieved by expression of short hairpin RNA <br />                                             (shRNA) from pSUPER-RETRO retroviral vector (Oligoengine). The sequence for WRN- <br />                                             targeting shRNA was 5&#226;&#8364;&#178;-GTGTATAGTTACGATGCTA-3&#226;&#8364;&#178; and for MSH6 was 5&#226;&#8364;&#178;- <br />                                             GGTGATCCCTCTGAGAACT-3&#226;&#8364;&#178;. The control GFP-targeting sequence was 5&#226;&#8364;&#178;- <br />                                             GCAAGCTGACCCTGAAGTT-3&#226;&#8364;&#178;. The luciferase-targeting sequence was 5&#226;&#8364;&#178;- <br />                                             GCGACCAACGCCTTGATTG-3&#226;&#8364;&#178;. Double-stranded oligonucleotides were ligated into the <br />                                             pSUPER-RETRO vector that was linearized with BamHI and HindIII. The pSUPER- <br />                                             RETRO vectors for depletion of MLH1 and MSH224 and details of retroviral infections37 <br />                                             have been described previously. For double knockdown experiments, NF cells were first <br />                                             infected with pSUPER-puro vectors, selected with puromycin for one week and then re- <br />                                             infected with pSUPER-neo vectors and selected with 400 &#206;&#188;g/ml geneticin for two weeks. To <br />                                             avoid loss of shRNA efficacy in long-term cultures, new stocks of infected cells were used <br />                                             every 4 weeks. <br />  <br />                               Immunofluorescence <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cells were grown on Superfrost Plus slides and exposed to Cr(VI) for 3 hr, then returned to <br />                                             complete medium. At the selected times, cells were washed twice with PBS and fixed with <br />                                             2% paraformaldehyde in PBS for 20 min at room temperature. Cells were then <br />                                             permeabilized for 20 min with 1% Triton X-100, followed by incubation with 3% fetal <br />                                             bovine serum in PBS for 1 hr. Primary antibodies against WRN (rabbit polyclonal <br />                                             antibodies, Santa Cruz) and B23 (mouse monoclonal antibody from Chemicon) were used at <br />                                             1:200 dilution and incubated overnight at 4&#194;&#176;C. For primary cells and Rad51 staining (anti- <br />                                             Rad51 mouse monoclonal antibody from Abcam, 1:400 dilution), cells were incubated at <br />                                             37&#194;&#176;C for 3 hr before and after overnight incubation at 4&#194;&#176;C. DSB repair sites were labeled by <br />                                             simultaneous incubation of primary antibodies for &#206;&#179;-H2AX at 1:1000 dilution (Upstate) and <br />                                             anti-53BP1 at 1:500 dilution (Santa Cruz) for 2 hr at 37&#194;&#176;C in a humidified chamber. The <br />                                             secondary antibodies, Alexa Fluor 488-conjugated anti-mouse immunoglobulin G, Alexa <br />                                             Fluor 594-conjugated anti-mouse immunoglobulin G, Alexa Fluor 488-conjugated anti- <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                           Page 4 <br />  <br />  <br />                                             human immunoglobulin G and Alexa Fluor 594-conjugated anti-rabbit immunoglobulin G <br />                                             (Molecular Probes) were incubated for 1 hr at room temperature. Antibodies were diluted in <br />                                             PBS containing 1% bovine serum albumin and 0.5% Tween 20, except for detection of &#206;&#179;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             H2AX where the antibodies were diluted in a 2% albumin-PBS solution. Slides were <br />                                             mounted with Vectashield hard set mounting medium with DAPI (Vector Laboratories). <br />                                             Fluorescence images were recorded with a Zeiss Axiovert 100 confocal microscope and <br />                                             analyzed by Phoenix software. Experiments were repeated with 3&#226;&#8364;&#8220;6 slides and at least 100 <br />                                             cells were analyzed on each slide. <br />  <br />                               Pulsed-field gel electrophoresis (PFGE) <br />                                             Cells were exposed to Cr(VI) for 3 hr and returned to complete medium for 6 hr or 24 hr. <br />                                             PFGE plugs were prepared using BioRad Mammalian CHEF Genomic Plug Kit at 106 cells/ <br />                                             plug and incubated in a proteinase K buffer [100 mM EDTA, 1% N-laurylsarcosyl, 10 mM <br />                                             Tris, pH 8.0, and proteinase K 1 mg/ml] at 20&#194;&#176;C for 24 hr. The plugs were washed 5 times in <br />                                             a wash buffer [20 mM Tris, pH 8.0 and 50 mM EDTA] with agitation prior to loading. <br />                                             PFGE was performed using a 1% gel (Pulsefield Certified Megabase Agarose, BioRad) in <br />                                             0.5X TBE buffer on a CHEF Mapper XA Pulsed Field Electrophoresis System (BioRad) for <br />                                             18 hr at 14&#194;&#176;C under the following current conditions: 120&#194;&#176; field angle, 240 s switch time, 4 <br />                                             V/cm. DNA was visualized by staining with ethidium bromide. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results <br />                               Increased sensitivity of WRN-deficient cells to Cr(VI) toxicity <br />                                             Expression of telomerase is known to compensate for the absence of WRN in human WS <br />                                             cells.38,39 The presence of active telomerase is also responsible for suppression of premature <br />                                             aging <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span> in Wrn-<span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>knockout</span> mice.40,41 Introduction of active hTERT into BJ human <br />                                             fibroblasts increased resistance and strongly inhibited the induction of chromosomal <br />                                             instability by Cr(VI),42 indicating that telomerase-immortalized cells have altered <br />                                             processing of Cr-induced damage. Therefore, we chose to perform our investigation in <br />                                             telomerase-negative human cells. A potential role of WRN helicase in cellular responses to <br />                                             Cr-DNA damage was first examined by clonogenic survival of Cr-treated WRN-expressing <br />                                             NF and WRN-null WS fibroblasts (Fig. 1A). WS cells displayed high sensitivity to <br />                                             clonogenic lethality by Cr(VI) with only a few colonies surviving at 10 &#206;&#188;M Cr while <br />                                             approximately 50% NF cells were clonogenically viable at this dose. Since NF and WS cells <br />                                             are not isogenic, we created a stable knockdown of WRN protein in NF cells by expressing a <br />                                             targeting shRNA from a chromosomally integrated pSUPER-RETRO vector (Fig. 1B, top <br />                                             panel). The selected shRNA decreased WRN levels about 10-fold as compared to NF cells <br />                                             expressing nonspecific shRNA (GFP). Using this isogenic model of WRN deficiency, we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             found again that the absence of WRN caused hypersensitivity to Cr(VI) lethality (Fig. 1C). <br />                                             To further confirm our findings, we constructed a stable WRN knockdown in telomerase- <br />                                             negative LNS fibroblasts (Fig. 1B, bottom panel). Although LNS cells were more resistant <br />                                             to Cr(VI) relative to NF cells, WRN knockdown in these cells resulted in essentially the <br />                                             same degree of sensitization to Cr(VI) toxicity (Fig. 1D). <br />  <br />                               Altered intranuclear localization of WRN after Cr(VI) damage <br />                                             WRN is typically found in the nucleolus of primary and transformed human cells.5 <br />                                             Costaining for WRN and the nucleolar marker B23 showed that the majority of NF cells also <br />                                             exhibited nucleolar localization of WRN (Fig. 2A). In response to Cr(VI) exposure, WRN <br />                                             relocalized from the nucleolus into the nucleoplasm. The nucleoplasmic localization was <br />                                             observed immediately after 3 hr-long exposures to 10&#226;&#8364;&#8220;20 &#206;&#188;M Cr(VI) and it was retained in <br />                                             treated cells up to 24 hr post-treatment. The majority of relocalized WRN produced a largely <br />                                             uniform diffuse staining <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. Costaining of Cr(VI)-treated cells with antibodies for WRN <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                          Page 5 <br />  <br />  <br />                                             and a marker of DSB, &#206;&#179;&#226;&#8364;&#8220;H2AX,43 showed that most DSB-positive cells displayed the <br />                                             diffuse WRN nuclear distribution and many &#206;&#179;&#226;&#8364;&#8220;H2AX foci colocalized with WRN (Fig. <br />                                             2B,C). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Persistence of DSB in the absence of WRN <br />                                             The observed colocalization of WRN with &#206;&#179;&#226;&#8364;&#8220;H2AX foci suggested that WRN could be <br />                                             involved in repair of Cr(VI)-induced DSB. To test this possibility, we examined the effects <br />                                             of WRN deficiency on the persistence of DSB after Cr(VI) removal. The presence of DSB <br />                                             was detected by the appearance of 53BP1 and &#206;&#179;&#226;&#8364;&#8220;H2AX foci and only cells exhibiting focal <br />                                             staining for both proteins were scored as DSB-positive (Fig. 3A). 53BP1 is recruited <br />                                             following remodeling of chromatin regions flanking the break44 and serves as a second <br />                                             marker of DSB. Levels of DSB induced by moderate doses of Cr(VI) typically approached <br />                                             their peak at about 6 hr post-exposure.23,24 Treatment of NF and LNS cells with moderately <br />                                             toxic Cr(VI) doses (~60% clonogenic survival) also resulted in a strong production of DSB <br />                                             foci at 6 hr post-Cr and WRN depletion had no significant effect on this initial induction of <br />                                             DSB (Fig. 3B,C). However, at 24 hr post-exposure, there were approximately twice as many <br />                                             DSB foci-positive cells with WRN knockdown as compared to their shGFP controls. Cr(VI) <br />                                             requires reductive activation inside cells in order to cause DNA damage.45 Human cells in <br />                                             culture contain low levels of ascorbate,25,35 which is the most rapid cellular reducer of <br />                                             Cr(VI).46,47 Restoration of physiological levels of ascorbate in primary human cells strongly <br />                                             potentiated the formation of DSB by Cr(VI).24 Our HPLC analyses detected approximately <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             25 &#206;&#188;M ascorbate in NF cells, which is much lower than physiological levels of this vitamin <br />                                             in tissues.47 Therefore, we preloaded NF cells with ascorbate to 0.25 mM and then <br />                                             examined DSB accumulation at 24 hr post-Cr (Fig. 3D,E). As in primary cells,24 increasing <br />                                             cellular ascorbate levels enhanced DSB-producing activity of Cr(VI), and WRN depletion <br />                                             resulted in the presence of persistent DSB even after exposure to as low as 2 &#206;&#188;M Cr(VI), <br />                                             which is the current EPA standard for chromium in drinking water.47 Analysis of <br />                                             chromosomal DNA by PFGE further confirmed the importance of WRN in repair of Cr- <br />                                             induced DSB as evidenced by 2.7-times (p&lt;0.01) higher levels of remaining DNA breaks in <br />                                             WRN-depleted LNS cells at 24 hr after Cr(VI) exposure (Fig. 3F). Control LNS cells with <br />                                             normal WRN levels were also capable of repairing the majority of DSB generated even by <br />                                             50 &#206;&#188;M Cr(VI) while WRN-depleted cells were unable to show significant repair of much <br />                                             fewer DSB induced by 30 &#206;&#188;M Cr(VI) (Fig. 3G). <br />  <br />                               Attenuated formation of RAD51 foci in WRN-deficient cells <br />                                             The majority of Cr-induced DSB are present in G2 cells.23,24 Since DSB repair in G2 cells <br />                                             occurs primarily via homologous recombination,48,49 we examined the effect of WRN on <br />                                             this repair process. The activity of recombination repair was assessed by scoring foci of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             RAD51, which is one of the essential recombination repair proteinsand a specific marker of <br />                                             recombination repair sites.50 Treatment of NF cells with either moderate 10 &#206;&#188;M or highly <br />                                             toxic 20 &#206;&#188;M doses of Cr(VI) did not increase the production of RAD51 foci above <br />                                             background levels at 6 hr post-exposure (Fig. 4A,B). At 24 hr post-Cr, NF cells with <br />                                             nonspecific GFP-shRNA displayed a significant induction of RAD51 foci at both Cr(VI) <br />                                             doses while WRN-depleted cells showed no change relative to untreated controls. Thus, <br />                                             activation of recombination repair in NF cells significantly lagged the induction of DSB in <br />                                             the presence of WRN (Fig. 3B,C) or was absent in cells with WRN knockdown. To further <br />                                             confirm these findings, we investigated the formation of RAD51 foci in LNS cells (Fig. 4C). <br />                                             Control LNS cells showed a strong induction of RAD51 foci already at 6 hr post-Cr but <br />                                             WRN depletion completely abrogated recombination foci at this time. At 24 hr post-Cr, <br />                                             WRN-depleted LNS cells displayed a significant increase in the number of RAD51 foci, <br />                                             which was still modestly lower than in control cells. Thus, both cell lines showed a strong <br />                                             positive effect of WRN on the formation of RAD51 foci. A more rapid activation of <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                           Page 6 <br />  <br />  <br />                                             recombination repair in LNS cells may at least be partially responsible for their higher <br />                                             resistance to Cr(VI) lethality (Fig. 1D). We also analyzed RAD51 foci in WRN-null WS <br />                                             cells treated with 10 and 20 &#206;&#188;M Cr(VI), doses that were clearly genotoxic to these cells (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             1A). Neither Cr(VI) dose caused even a marginal increase in the induction of RAD51 foci at <br />                                             24 hr post-exposure (Fig. 4D), further confirming the importance of WRN protein for the <br />                                             initiation of recombination repair of Cr-DNA damage. A large fraction of Cr-induced <br />                                             RAD51 focus-positive NF and LNS cells also showed a partial colocalization (ranging from <br />                                             20 to 80%) of RAD51 foci with WRN (Fig. 4E,F). <br />  <br />                               Mismatch repair (MMR)-mediated DNA lesions and WRN <br />                                             The main cause of DSB in human cells treated with low to moderately toxic doses of Cr(VI) <br />                                             is abnormal processing of Cr-DNA crosslinks by MMR.23&#226;&#8364;&#8220;25 We have also observed that <br />                                             ascorbate, which enhances DSB production by aberrant MMR activity,24 increased the <br />                                             levels of DNA breaks in WRN-depleted cells at Cr(VI) concentrations that were otherwise <br />                                             nongenotoxic (Fig. 3D,E). Therefore, we tested whether WRN was responding to secondary <br />                                             DNA lesions generated by MMR. We first examined the dynamics of WRN localization in <br />                                             Cr(VI)-treated human HCT116 cells lacking or expressing MLH1, which is one of the <br />                                             essential MMR proteins.33,34 The majority of control cells in both lines showed costaining <br />                                             of WRN with the nucleolar marker B23, demonstrating that MLH1 has no effect on the <br />                                             retention of WRN in nucleoli of unstressed cells (Fig. 5A). Immediately after Cr(VI) <br />                                             exposure, WRN dispersed from the nucleolus into the nucleoplasm in the majority of cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             irrespective of their MLH1 status. Similarly to NF cells (Fig. 2), MLH1+ HCT116 cells also <br />                                             retained nucleoplasmic WRN at 24 hr post-Cr. In contrast, MLH1&#226;&#710;&#8217;/&#226;&#710;&#8217; HCT116 cells restored <br />                                             the normal nucleolar localization of WRN at 3&#226;&#8364;&#8220;6 hr post-exposure. To further investigate the <br />                                             MMR-dependence of WRN dynamics, we created stable knockdowns of MLH1 or MSH2 <br />                                             MMR proteins in primary human IMR90 fibroblasts (Fig. 5B). We found that WRN <br />                                             relocalized in all IMR90 constructs immediately after Cr(VI) exposure but while control <br />                                             Luc-shRNA cells retained WRN in the nucleoplasm, cells with MLH1 or MSH2 <br />                                             knockdowns restored nucleolar WRN staining already at 3 hr post-Cr (Fig. 5C). Thus, the <br />                                             initial relocalization of WRN occurred directly in response to Cr(VI)-induced stress but the <br />                                             long-term retention in the nucleoplasm required the presence of MMR proteins. The <br />                                             nucleoplasmic retention of WRN was not due to binding to MMR proteins, as co- <br />                                             immunoprecipitation experiments found no detectable association of WRN with MSH6 or <br />                                             MSH2 in control or Cr(VI)-treated NF cells (Fig. 5D). MSH6 immunoprecipitates contained <br />                                             large amounts of its binding partner MSH2,33,34 validating the effectiveness of the <br />                                             employed procedure. Sensitization of WRN-depleted cells to Cr(VI) toxicity did not appear <br />                                             to result from the alterations in the MMR system as expression of three major MMR <br />                                             proteins MSH6, MSH2 and MLH1 did not vary between GFP-shRNA and WRN-shRNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             cells (Fig. 5E). Cr(VI) exposure also had no significant effect on the protein levels of WRN <br />                                             itself (Fig. 5E). <br />  <br />                                             The observed MMR-dependent dynamics of WRN in the absence of direct interactions with <br />                                             MMR proteins suggested that WRN could be responding to MMR-induced DSB. To test this <br />                                             possibility, we constructed NF cells with double knockdowns of MSH6 and WRN (Fig. 6A). <br />                                             We found that depending on Cr(VI) dose, knockdown of MSH6 in WRN-depleted cells <br />                                             either completely abrogated or significantly suppressed the levels of persistent DSB foci <br />                                             (Fig. 6B). The presence of ascorbate strongly potentiated the formation of DSB by Cr(VI) <br />                                             (Fig. 3E) and at low doses essentially all ascorbate-promoted DSB are MMR-generated.24 <br />                                             We found that depletion of MSH6 completely eliminated the presence of persistent DSB in <br />                                             Cr-treated/ascorbate-preloaded cells with WRN knockdown (Fig. 6C). This effect was quite <br />                                             striking, as even cells with normal WRN levels (GFP/GFP controls) were unable to fully <br />                                             repair DSB. As with DSB induction, the formation of RAD51 foci by Cr(VI) was enhanced <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                          Page 7 <br />  <br />  <br />                                             by preloading of control cells with ascorbate and MSH6 depletion abrogated the appearance <br />                                             of the recombination-active cells (Fig. 6D,E). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Discussion <br />                                             In this work, we found that WRN was actively engaged in cellular responses to Cr(VI). <br />                                             Immediately after short treatments with this carcinogen, WRN relocalized from the <br />                                             nucleolus into the nucleoplasm where it remained even at 24 hr post-exposure. The loss of <br />                                             WRN resulted in increased clonogenic toxicity of Cr(VI), demonstrating the importance of <br />                                             this RecQ-type helicase in cell survival. Cellular reduction of Cr(VI) generates a transient <br />                                             oxidative stress due to the formation of redox-active CrV/IV intermediates51,52 and yields <br />                                             stable Cr-DNA adducts.45 The prolonged nucleoplasmic retention of WRN in Cr-treated <br />                                             cells, but not its initial relocalization, was dependent on the presence of intact MMR. Thus, <br />                                             the early WRN movement was likely caused by the cellular stress during Cr(VI) metabolism <br />                                             but its subsequent nucleoplasmic presence reflected other events involving MMR. <br />                                             Processing of Cr-DNA adducts by human MMR generates DSB as secondary, more toxic <br />                                             DNA lesions.23&#226;&#8364;&#8220;25 We found that WRN played a major role in repair of MMR-induced DSB <br />                                             in Cr(VI)-treated human fibroblasts without significant effects on the initial production of <br />                                             DNA breaks. The presence of unrepaired DSB was likely responsible for the decreased <br />                                             survival of WRN-depleted cells in the clonogenic assay. Poor repair of DSB in WRN- <br />                                             lacking cells was associated with their deficiency in activation of early steps in homologous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             recombination, as revealed by the absence or severely delayed formation of RAD51 foci <br />                                             after Cr-DNA damage. Functional recombination repair is known to play an important role <br />                                             in cellular resistance to Cr(VI) toxicity.53,54 <br />  <br />                                             Induction of DSB by MMR typically begins within 1&#226;&#8364;&#8220;3 hr post-Cr exposure and occurs in <br />                                             G2 cells following their replication of Cr-adducted DNA.23,24 The late steps in MMR- <br />                                             dependent processing of Cr-DNA damage prior to DNA breakage are dependent on the <br />                                             activity of MSH3-MSH2 heterodimer (MutS&#206;&#178; complex).25 MutS&#206;&#178; dimer exhibits high <br />                                             specificity for DNA loops of various sizes,33,34 suggesting that MMR-induced DSB could <br />                                             be associated with the presence of loops or hairpins. While the presence of loops or hairpins <br />                                             can interfere with the activity of many DNA-processing enzymes, DNA ends with adjacent <br />                                             loops are high affinity substrates for WRN.2 Additionally, a strong requirement of WRN for <br />                                             the initiation of RAD51-dependent recombination in Cr-treated cells could have also <br />                                             reflected a limited availability of EXO1 that operates in a parallel pathway leading to the <br />                                             activation of recombination repair of DSB.20&#226;&#8364;&#8220;22 The role of the RecQ helicases in DSB <br />                                             processing was most evident when cells were made deficient in EXO1/Exo1. Because EXO1 <br />                                             is also an essential component of human MMR,33,34 a massive formation of ternary Cr-DNA <br />                                             adducts recruiting MMR complexes and stimulating their activity is expected to sequester a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             major fraction of this low abundance exonuclease.22 <br />  <br />                                             Both MSH2-MSH3 and MSH2-MSH6 mismatch-sensing complexes were capable of <br />                                             stimulating DNA duplex unwinding by WRN in vitro.55 The biological relevance of these <br />                                             findings was further supported by physical interactions between WRN and MSH2 in yeast <br />                                             two-hybrid studies. High duplex unwinding activity should promote DSB repair, as helicase <br />                                             activity is essential for 5&#226;&#8364;&#178;&#226;&#8224;&#8217;3&#226;&#8364;&#178; exonuclease degradation of DSB ends in the RecQ/Sgs1- <br />                                             dependent recombination process.20&#226;&#8364;&#8220;22 Since MMR proteins are present at the sites of Cr- <br />                                             induced DSB,25 they could recruit WRN and/or stimulate its strand unwinding activity and <br />                                             thereby contribute to its prominent role in DSB ends processing that is necessary for RAR51 <br />                                             loading. However, we were so far unable to detect interactions of WRN with MSH2 or <br />                                             MSH6 by co-IP in either control or Cr(VI)-treated cells. It is possible that association of <br />                                             WRN with MMR proteins predominantly occurs in chromatin-bound DSB repair complexes <br />                                             that remained insoluble during the protein extraction procedures or our IP procedure <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                                           Page 8 <br />  <br />  <br />                                              employed too stringent lysis/washing conditions. Alternatively, WRN can be recruited to the <br />                                              sites of Cr-induced DSB by other components of MMR, such as the upstream-acting MLH1- <br />                                              PMS2 heterodimer.55 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Implications for Cr(VI) genotoxicity <br />                                              Long DSB persistence and defective formation of RAD51 recombination foci in the absence <br />                                              of WRN indicate that repair of MMR-induced DSB in Cr(VI)-treated fibroblasts is <br />                                              dependent on this helicase. Thus, unlike other types of DSB that are processed for <br />                                              recombination repair by both RecQ-dependent and independent (EXO1) pathways,20&#226;&#8364;&#8220;22 a <br />                                              reliance on a WRN-dependent process for repair of Cr/MMR-induced breaks can result in <br />                                              more persistent DSB. Cr(VI)-treated cells did display a relatively slow DSB repair and this <br />                                              phenomenon was particularly noticeable in primary human cells which showed a steady <br />                                              accumulation of DSB over many hours even after exposure to subtoxic environmental doses <br />                                              of Cr(VI).24,25 High promoter-repressive activity of Rb complexes in primary cells results in <br />                                              low levels of WRN expression,56 which could limit repair rates for Cr-induced DSB in <br />                                              tissues. Persistent DSB can cause a variety of gross genetic rearrangements, including <br />                                              chromosomal deletions and translocations,57,58 that are likely contributing to the high <br />                                              carcinogenicity of even moderate chromate exposures found in the modern workplace <br />                                              environment.47,59 Consequently, individuals with low WRN levels could be at a greater risk <br />                                              for Cr(VI)-associated genotoxicity and carcinogenesis. Since Cr(VI)-induced DSB are <br />                                              predominantly found in G2 cells and their repair requires WRN, this may restrict WRN <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              availability for its other biological functions such as rebuilding of telomere structures which <br />                                              also occurs in G2 phase.60 Thus, one of the potential consequences of competition for WRN <br />                                              between telomeric functions and DSB repair could be slower rates for both processes and, <br />                                              possibly, accelerated attrition of telomeres in cells of individuals chronically exposed to <br />                                              Cr(VI). <br />  <br />  <br />                               Abbreviations <br />                                             DSB             double-strand break <br />                                             MMR             mismatch repair <br />                                             PBS             phosphate-buffered saline <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             shRNA           short hairpin RNA <br />                                             PFGE            pulsed-field gel electrophoresis <br />  <br />   <br />                                             Figure 1. Loss of WRN decreases survival of cells treated with Cr(VI) <br />                                             Cell survival was determined by clonogenic assay. Results are means&#194;&#177;SD of 2&#226;&#8364;&#8220;3 <br />                                             independent experiments, each with triplicate dishes per dose. Where not seen, error bars <br />                                             were smaller than the symbol. Statistics (Student&#226;&#8364;&#8482;s t-test): *- p&lt;0.05, **- p&lt;0.01 relative to <br />                                             WRN-deficient cells. (A) Clonogenic viability of WS and NF fibroblasts treated with <br />                                             Cr(VI). (B) WRN levels in NF (top panel) and LNS (lower panel) cells expressing targeting <br />                                             and nonspecific (GFP) shRNA. (C) Cr(VI) toxicity in NF cells expressing WRN-targeting or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             nonspecific (GFP) shRNA. (D) Survival of LNS cells expressing WRN or GFP-targeting <br />                                             shRNA. <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                       Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. Cr(VI) causes dispersion of WRN from nucleolus <br />                                             (A) Confocal images of control and 20 &#206;&#188;M Cr(VI)-treated NF cells immunostained for <br />                                             WRN and the nucleolar marker B23. (B) A representative confocal image of Cr-treated cells <br />                                             displaying colocalization WRN with &#206;&#179;-H2AX foci. (C) Presence of &#206;&#179;-H2AX foci in NF cells <br />                                             with different patterns of WRN staining. Cells were treated with 20&#206;&#188;M Cr(VI) and fixed for <br />                                             immunostaining at indicated times. Data are means&#194;&#177;SD from three slides with &gt;100 cells <br />                                             scored per slide. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                        Page 14 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 3. Persistence of Cr-induced DSB in WRN-depleted cells <br />                                             (A) Confocal images of GFP and WRN-shRNA expressing NF cells stained with DAPI and <br />                                             antibodies for 53BP1 and &#206;&#179;-H2AX. Shown are representative images of untreated and 10 <br />                                             &#206;&#188;M Cr-treated cells at 6 hr post-exposure. (B) Frequency of DSB-positive NF cells at 6 and <br />                                             24 hr after exposure to 10 &#206;&#188;M Cr(VI). Cells with &#226;&#8240;&#165;5 foci of both 53BP1 and &#206;&#179;-H2AX were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             counted as DSB-positive. WRN -cells expressing WRN-targeting shRNA, GFP - cells <br />                                             expressing nonspecific shRNA. Data are means &#194;&#177;SD for 5 slides with &gt;100 cells analyzed <br />                                             per slide. Statistics: *- p&lt;0.05, **- p&lt;0.01 relative to shGFP cells. (C) Frequency of DSB- <br />                                             positive LNS cells following exposure to 30 &#206;&#188;M Cr(VI). Definitions of cells are as in panel <br />                                             B. Results are means&#194;&#177;SD for 3 slides with &gt;100 cells counted per slide. (D) DSB levels in <br />                                             normally cultured NF cells at 24 hr post-exposure to low Cr(VI) doses. Data are means &#194;&#177;SD <br />                                             for 4 slides with &gt;100 cells analyzed per slide. (E) Elevated DSB levels in NF cells <br />                                             preloaded with 0.25 mM ascorbic acid prior to Cr(VI) exposure. DSB foci were counted at <br />                                             24 hr post-Cr. Data are means&#194;&#177;SD for 4 slides with &gt;100 cells counted per slide. (F) <br />                                             Determination of DSB repair by PFGE in LNS cells treated with 30 &#206;&#188;M Cr(VI) (means <br />                                             &#194;&#177;SD). (G) PFGE of chromosomal DNA from shGFP- and shWRN-LNS cells treated with <br />                                             50 and 30 &#206;&#188;M Cr(VI), respectively. <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                            Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 4. WRN depletion inhibits the formation of RAD51 foci in Cr-treated cells <br />                                             (A) Representative confocal images of NF and LNS cells stained with DAPI and anti- <br />                                             RAD51 antibody. Cells were treated with 10 &#206;&#188;M (NF cells) or 30 &#206;&#188;M Cr(VI) (LNS cells) for <br />                                             3 hr and fixed 6 and 24 hr later. (B) Induction of RAD51 foci Cr(VI) in NF cells. Cells with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             3 or more foci were scored as focus-positive. Data are means&#194;&#177;SD for 4 slides with &gt;100 <br />                                             cells scored per slide. (C) Formation of RAD51 foci in LNS cells treated with 30 &#206;&#188;M <br />                                             Cr(VI). Data are means&#194;&#177;SD for 4 slides with &gt;100 cells counted per slide. Statistics for <br />                                             panels B and C: *-p&lt;0.05, **- p&lt;0.01 relative to shGFP cells. (D) Number of RAD51 foci- <br />                                             positive WS cells at 24 hr post-Cr. Data are means&#194;&#177;SD for 3 slides with at least 100 cells <br />                                             scored per slide. (E) Colocalization of WRN and RAD51 foci in LNS cells at 6 and 24 hr <br />                                             after treatment with 30 &#206;&#188;M Cr(VI). (F) WRN and RAD51 colocalization in NF cells at 24 hr <br />                                             following exposure to 10 &#206;&#188;M Cr(VI). For panels E and F, nuclei containing at least one clear <br />                                             RAD51 focus colocalizing with WRN were scored as positive. Typical colocalization for <br />                                             positive cells was 20&#226;&#8364;&#8220;80% of all RAD51 foci. &#226;&#8364;&#339;&#226;&#710;&#8217;&#226;&#8364;&#65533; Frequency of RAD51-positive cells <br />                                             without WRN colocalization, &#226;&#8364;&#339;+&#226;&#8364;&#65533; &#226;&#8364;&#8220;frequency of cells with RAD51 and WRN colocalization. <br />                                             Results are means&#194;&#177;SD for 3 slides with &gt;30 RAD51 foci-containing cells scored for <br />                                             colocalization. Statistics for panels E and F: *-p&lt;0.05, **- p&lt;0.01 relative to untreated cells. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 5. Nucleoplasmic retention of WRN in Cr(VI)-treated cells requires intact MMR <br />                                             (A) Frequency of MLH1&#226;&#710;&#8217;/&#226;&#710;&#8217; and MLH1+ HCT116 cells with nucleoplasmic WRN following <br />                                             treatment with 20 &#206;&#188;M Cr(VI) (means&#194;&#177;SD for 3 slides with &gt;100 cells analyzed/slide). <br />                                             Statistics: *- p&lt;0.05, **- p&lt;0.01 relative to MLH1&#226;&#710;&#8217;/&#226;&#710;&#8217; cells. (B) Western blot of protein <br />                                             extracts from IMR90 cells expressing nonspecific (Luc), MSH2 or MLH1-targeting shRNA. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             (C) Quantitation of nuclear WRN dispersion in IMR90 cells treated with 10 &#206;&#188;M Cr(VI) <br />                                             (means&#194;&#177;SD for 3 slides with &gt;100 cells analyzed/slide). Statistics: *- p&lt;0.05, **- p&lt;0.01 <br />                                             relative to MLH1- or MSH2-shRNA. (D) Coimmunoprecipitation of WRN with MSH6 or <br />                                             MSH2 in control or 20 &#206;&#188;M Cr-treated NF cells. (E) Expression of MMR proteins in GFP- <br />                                             shRNA and WRN-shRNA NF cells after exposure to 20&#206;&#188;M Cr(VI). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />                             Zecevic et al.                                                                                        Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 6. Rescue of WRN hypersensitivity by MSH6 depletion <br />                                             (A) Western blot for MSH6 and WRN in NF cells after double shRNA infections. WRN- <br />                                             shRNA was expressed from pSUPER-puro vector, MSH6 &#226;&#8364;&#8220; from pSUPER-neo and <br />                                             nonspecific GFP-shRNA was expressed from either pSUPER-puro or pSUPER-neo <br />                                             depending on the resistance marker in the targeting vector. (B) DSB foci (53BP1 + &#206;&#179;-H2AX) <br />                                             in WRN-depleted NF cells with and without MSH6 knockdown (&#226;&#8212;&#65533; - WRN-GFP shRNAs, &#226;&#8211;&#161; <br />                                             - GFP-GFP shRNAs, &#226;&#8212;&#8225;- GFP-MSH6 shRNAs, &#226;&#8211;&#178;-WRN-MSH6 shRNAs). Cells were <br />                                             treated with Cr(VI) for 3 hr and fixed for immunofluorescence 24 hr later. Data are means <br />                                             &#194;&#177;SD for 3 slides with &gt;100 cells counted per slide (*- p&lt;0.05 for both GFP/GFP and WRN/ <br />                                             MSH6 relative to WRN/GFP cells). (C) Frequency of DSB-positive double knockdown NF <br />                                             cells preloaded with 0.25 mM ascorbate prior to 5 &#206;&#188;M Cr(VI) exposure. Cells were fixed for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DSB foci scoring (53BP1 + &#206;&#179;-H2AX foci) at 24 hr post-Cr. Shown are means&#194;&#177;SD for 3 <br />                                             slides with &gt;100 cells counted per slide (*- p&lt;0.05, **-p&lt;0.01 relative to untreated cells). <br />                                             (D) Frequency of RAD51 foci-positive NF cells expressing nonspecific (GFP) and MSH6- <br />                                             targeting shRNA following exposure to Cr(VI) under normal culture conditions. RAD51 <br />                                             foci were scored at 24 hr post-treatment. Data are means&#194;&#177;SD for 3 slides with &gt;100 cells <br />                                             counted per slide. (E) As panel D except that cells were preloaded with 0.25 mM ascorbate <br />                                             prior to treatments with Cr(VI). Data are means&#194;&#177;SD for 3 slides. Statistics for panels D and <br />                                             E: *- p&lt;0.05, **- p&lt;0.01 relative to shGFP cells. <br />  <br />  <br />  <br />  <br />                                                 Cell Cycle. Author manuscript; available in PMC 2011 November 30. <br />   <br /> </body></html>