<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1989, p. 1659-1666                                                                      Vol. 9, No. 4 <br /> 0270-7306/89/041659-08$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  <br />           Molecular Analysis GPHI, Gene Encoding Glycogen <br />                  Phosphorylase Saccharomyces cerevisiae <br />                   PETER K. HWANG,* STUART TUGENDREICH, AND ROBERT J. FLETTERICK <br />          Department Biochemistry Biophysics, University California, San Francisco, California 94143-0448 <br />                                         Received 19 November 1988/Accepted 13 January 1989 <br />  <br />               In yeast cells, activity glycogen phosphorylase regulated cyclic AMP-mediated phosphorylation <br />            enzyme. We previously cloned gene glycogen phosphorylase (GPH1) Saccharomyces <br />            cerevisiae. <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> assess role glycogen phosphorylase-catalyzed glycogenolysis yeast life cycle, <br />            yeast strains lacking functional GPHI gene containing multiple copies gene constructed. GPHI <br />            essential gene yeast cells. Haploid cells disrupted GPHI lacked phosphorylase <br />            activity attained higher levels intracellular glycogen similar wild-type cells. <br />            Diploid cells homozygous disruption able sporulate rise viable ascospores. Absence <br />            functional GPHI did impair cells synthesizing storing trehalose. Increases phosphorylase <br />            activity 10- 40-fold detected cells carrying multiple copies GPHI-containing 2,um plasmid. <br />            Northern (RNA) analysis indicated GPHI transcription induced late exponential growth phase, <br />            simultaneous onset intracellular glycogen accumulation. Thus, low level glycogen in <br />            exponential cells primarily maintained regulating phosphorylation state constitutive <br />            phosphorylase. GPHI did appear formal glucose repression, transcriptional <br />            induction occurred advance glucose depletion medium. <br />  <br />    Energy required cell activity growth maintained              evolutionary relationships different enzymes (23, <br /> in regulated synthesis breakdown storage                    33a). We begun study biochemical and <br /> carbohydrates. Glycogen phosphorylase, catalyzes                     physiological role glycogen phosphorylase S. cerevi- <br /> the release glucose 1-phosphate glycogen, plays                  siae, using powerful tools genetic manipulations that <br /> vital role process cells organisms. The                available organism (45). <br /> regulation phosphorylase best understood vertebrate                  Yeast cells contain principal storage carbohydrates, <br /> tissues, enzyme activity modulated                 glycogen trehalose (15). Both carbohydrates appear to <br /> changing levels effector ligands hormonal signals (20).             provide energy cells periods nonproliferation. <br /> Vertebrate phosphorylases convert inactive                     A central question study yeast storage metabolism <br /> active forms reversible covalent modification                    organism utilizes different reserve carbohy- <br /> well-characterized regulatory kinase cascade (28). Enzyme                  drates. Only case cells undergoing sporulation or <br /> activity regulated tissue-specific expression              respiratory adaptation does evidence exist distinct roles <br /> of different isozymic forms. At different isozyme              (27, 31). A related issue glycogen trehalose <br /> forms identified mammals (muscle, liver, brain,               pathways directly coupled (17, 34, 38). Previous studies <br /> or fetal forms, named tissues pre-                 shown levels glycogen trehalose yeast <br /> dominate) (19, 43, 54). These isozymes display different                   cells change dramatically response variety different <br /> allosteric properties, reflecting evolution functions          physiological situations (29, 31, 48). The enzymes involved <br /> suit specific environments physiological roles. In                 synthesizing breaking carbohydrates are <br /> case muscle phosphorylase rabbit skeletal tissue, <br /> regulatory mechanisms extensively character-                     regulated cyclic AMP (cAMP)-dependent kinase <br /> ized, biochemically crystallographicaily (12, 13, 25).                 yeasts (22, 36, 49, 50, 52, 53). Glycogen and <br />    Unlike bacterial plant phosphorylases,                    trehalose levels, fact, useful phenotypic indi- <br /> unregulated enzymes (8, 33, 35), yeast phosphorylase               cators studying various mutants RAS, adenylate <br /> regulated allosteric mechanisms. The enzyme                     cyclase, cAMP kinase genes (16, 46, 47). However, <br /> subject secondary-site allosteric control (glucose 6-phos-              complexity regulatory path- <br /> phate inhibition) converted active inac-                way controlling carbohydrates, difficult to <br /> tive forms reversible phosphorylation (14). An important                use abnormal carbohydrate levels basis isolating <br /> distinction yeast phosphorylase phosphorylation             mutants specific defects glycogen trehalose me- <br /> site lacks sequence similarity mammalian enzyme                 tabolism. <br /> counterpart (30). We previously cloned phospho-                      The cloning yeast GPHJ permits address the <br /> rylase gene yeast Saccharomyces cerevisiae (we refer                issues using combined molecular genetic <br /> to gene yeast GPHI) derived primary struc-                  approach. We present preliminary account the <br /> ture enzyme (23). These data, recently                  phenotypic consequences certain genetic manipulations of <br /> determined primary structures bacterial, plant,                    GPHI. We disrupted GPHJ gene yeast cells, <br /> isozymic variants mammalian phosphorylases, pro-                   generating haploid cells disrupted GPHJ diploid <br /> vided foundation comparing structure-function                    cells homozygous disruption, introduced <br />                                                                            cells increased gene dosage GPHI 2,um <br />                                                                            plasmids. We examined expression level of <br />   * <br />       Corresponding author.                                                GPHJ transcripts growth rich medium. <br />                                                                     1659 <br />  1660     HWANG ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />                            Plasmids <br />                                                B            R       X       S             RS        X          R       B <br />                           YCp5O:GPH1                 %11                                  11        I          I <br />  <br />                                                                                     GPH <br />  <br />                                                             R    X          S              S <br />                                                    5&apos; Probe I <br />                                                         Probe --        I             -         Coding   Region <br />                                                                                                    ~~~~~~Probe <br />                            YCpX:GPH1                                x                     RS           x <br />                                                                                            iI <br />                            YEp24:GPH1 <br />  <br />                                                                     X       S        R         (S,X)       x <br />  <br />                          YCpX:(gph 1A::LEU2) <br />  <br />  <br />                                                             R       x           s     R        (SX) X  R <br />                                                     .....   I       I                               I  I1 <br />                                                                                                     ..1W <br />  <br />  <br />                                                                                                                    1   kb <br />  <br />   FIG. 1. Structure wild-type GPHI, disrupted GPH1, plasmid constructions. YCp5O:GPHl contains 15-kb genomic fragment <br /> including entire GPHI gene. The coding region GPHI previously identified (23). Fragments designated 5&apos; probe coding region <br /> probe subcloned M13 vectors used hybridization probes. Plasmids YCpX:GPHI, YEp24:GPHl, YCpX:(gphlA::LEU2) <br /> were constructed described Materials Methods. Arrow indicates coding region GPHI; open bar indicates yeast LEU2 gene. <br /> Restriction enzyme sites: BamHI (B); EcoRI (R); Sall (S); Xhol (X). <br />  <br />  <br />                MATERIALS AND METHODS                                    inserting XhoI GPHJ fragment Sall site of <br />                                                                         YEp24. <br />    Strains media. S. cerevisiae HR125-5d (MA Ta/MATct,                 Preparation yeast crude extract. Yeast cells grown in <br /> homozygous his3 his4 leu2-3 leu2-112 ura3-52 trpl-289),             liquid cultures harvested centrifugation 3,000 x g <br /> provided R. Jensen, used RNA DNA blotting                5 min 5&#194;&#176;C. The cell pellets suspended cold <br /> and gene disruption experiments. Escherichia coli MH1 <br />                                                                         water, repelleted centrifugation, frozen at <br /> (araD139 lacX74 galU galK hsr rpsL) MH6 (lacX74 <br /> galU galK hsr rpsL leuB600 pyrF: :Tn5), M. Hall (18),              -70QC storage. Thawed pellets suspended equal <br /> were used grow various plasmid constructions. <br />                                                                         volumes homogenization buffer (0.1 M Tris [pH 7.9], 10 <br />    YM-P medium (31) contained 6.3 g yeast nitrogen <span id='am-86' about='xsp:base' typeof='owl:Thing'>base</span>              mM magnesium chloride, 2 mM calcium chloride, 1 mM <br /> without amino acids (Difco Laboratories, Detroit, Mich.)            EDTA, 10% glycerol, 1 mM dithiothreitol). Homogenization <br /> liter, 4.5 g yeast extract (Difco) liter, 9 g peptone         carried vortexing samples equal volume <br /> (Difco) liter, 9 g succinic acid liter, 5.4 g NaOH        glass beads (0.5-mm diameter; Biospec Products, Bartles- <br /> per liter, 2% glucose.                                              ville, Okla.) 30-s pulses. The homogenized mixture <br />    Genetic methods. Standard methods yeast genetics             centrifuged 8,000 x g 10 min. A portion of <br /> used tetrad analysis scoring various markers (44).           supernatant extract diluted 1:200 measurement of <br /> Yeast cells transformed method Ito et al. (24).          A280. Typical values optical density range 0.4 <br /> Gene disruption carried homologous recombina-                0.8. <br /> tion cloned DNA fragment genomic DNA,                        Phosphorylase assay. The procedure used assay phos- <br /> using one-step gene replacement technique (42).                     phorylase activity yeast crude extract based the <br />    Plasmids. GPHI originally isolated 15-kilobase              method Carney et al. (7) determines of <br /> (kb) genomic DNA insert BamHI site YCp5O.                 phosphate released formation glycogen glu- <br /> This plasmid, designated YCp5O:GPH1, used re-                   cose 1-phosphate. Enzyme reactions carried 100 <br /> striction mapping genomic fragment (Fig. 1). The                 mM sodium succinate (pH 5.8)-1% glycogen (prepared <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> 3.7-kb XhoI restriction fragment GPHJ subcloned               oyster; Sigma Chemical Co., St. Louis, Mo.)-150 mM glu- <br /> digesting YCp5O:GPHJ XhoI religating                  cose 1-phosphate-1 mM dithiothreitol. Typically, yeast ex- <br /> digested mixture form plasmid YCpX:GPHI, liga-                 tracts diluted 1/100 reaction mixtures. To <br /> tion GPHJ fragment XhoI fragment consisting             check Pi generated assay other <br /> a portion YCp50 (ampr ori) joined short region               sources yeast extract, additional assays were <br /> yeast genomic DNA. For gene disruption, 1.9-kb Sall                 carried absence added glycogen glucose <br /> restriction fragment GPHJ coding region                  1-phosphate. Portions reaction mixture mixed with <br /> removed YCpX:GPHJ replaced 2.1-kb Sall-                   phosphate-reacting colorimetric reagent (7) extracted <br /> XhoI restriction fragment yeast LEU2 gene                equal volume butanol, measured for <br /> YCpX:(gphlA::LEU2). YEp24:GPHI constructed                       A310- Phosphate concentration determined linear <br />  VOL. 9, 1989                                                           GLYCOGEN PHOSPHORYLASE IN S. CEREVISIAE                         1661 <br />  <br />                             A                        B                                       C <br />                              1     2     3     4       1     2     3     4                       .1    2       3 <br />  <br />                                                                                                                         XIII, XVI <br />  <br />  <br />                                                                               -4.6 <br />                                                                               -3.9 <br />                                                                                                                         III <br />  <br />  <br />  <br />          DNA probes:          GPH1 5&apos; region GPH1 coding region                             GPH1           LEU2 <br />    FIG. 2. Disruption chromosomal localization GPH1 gene. (A B) Total cellular DNA prepared haploid yeast strains <br /> segregated tetrad derived transformation HR125-5d gphlA::LEU2 DNA fragment. Then 5 Fg DNA sample <br /> was digested EcoRI, electrophoresed 1.0% agarose gel, transferred nitrocellulose filter. Lanes DNA Leu- (1 and <br /> 3) Leu+ (2 4) segregants. (A) Filter hybridized (high stringency) 32P-labeled 1.7-kb EcoRI-XhoI DNA fragment 5&apos; flanking to <br /> GPHI; (B) hybridized (low stringency) 1.9-kb Sail fragment GPHI coding region. In panel A, shift predominant signal <br /> from 4.6-kb band (lanes 1 3) 3.9-kb band (lanes 2 4) predicted gphlA::LEU2 disruptants. Weaker hybridization bands <br /> indicate homologies yeast genome 5&apos;-flanking region GPHI. In panel B, lack low-stringency hybridization Leu+ <br /> segregants (lanes 2 4) confirms deletion GPHI coding region absence multiple phosphorylase genes. (C) <br /> Chromosomes Leu- (wild-type GPHI) segregant (lanes 1 2) Leu+ segregant (lane&apos;3) separated OFAGE blotted <br /> to nylon membrane filter. Lanes: 1, hybridized radiolabeled 3.7-kb XhoI fragment GPHI; 2 3, hybridized 2.1-kb <br /> SaII-XhoI fragment yeast LEU2 gene. <br />  <br /> interpolation spectrophotometrically measured phosphate              Pa.). Medium glucose levels determined using a <br /> standards. One <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> phosphorylase activity represents            glucose oxidase kit Sigma. <br /> release 1 pLmol phosphate glucose 1-phosphate              Determination glycogen trehalose content. Qualita- <br /> min 30&#194;&#176;C. For determination phosphorylase specific                tive glycogen determinations adding 5 10 ml <br /> activity, total protein concentration estimated              0.4% KI-0.2% iodine solution plates. Cells that <br /> method Bradford (4), using bovine serum albumin                    accumulate glycogen stain brown, not <br /> standard curve.                                                         remain yellow. Quantitative determinations glycogen and <br />    DNA RNA hybridization analysis. Total yeast DNA                  trehalose carried methods described Lillie <br /> prepared saturated cultures (11) digested                 Pringle (31). <br /> EcoRI. The DNA electrophoresed 1.0% agarose gel <br /> and blotted nitrocellulose membrane filter. DNA                                            RESULTS <br /> probes used hybridization gel purified 32p <br /> labeled (109 cpm/nug DNA) random primer method                   Construction null GPHI allele. To determine whether <br /> (21). Hybridization conditions 650C, 6x SSPE (20x                  GPHI essential haploid cell growth, gphlA::LEU2 <br /> SSPE 3.6 M NaCl, 200 mM NaH2PO4, 20 mM EDTA                      allele constructed method Rothstein (42). On a <br /> [pH 7.4]), 5 mM TES [N-tris(hydroxymethyl)methyl-2-ami-                 cloned DNA fragment containing GPHI gene, 1.9-kb <br /> noethanesulfonic acid; Sigma], Sx Denhardt solution, 10 ,ug             Sail restriction fragment coding region de- <br /> of salmon sperm DNA ml, 0.1% sodium dodecyl                     leted replaced insertion LEU2 gene fragment <br /> sulfate. Washes carried nonstringently 2x SSPE              (Fig. 1). The resulting plasmid, YCpX:(gphlA::LEU2), was <br /> or stringently 0.1 x SSPE presence 0.1% sodium             cut XhoI, gel-purified 3.8-kb gphlA::LEU2 <br /> dodecyl sulfate 65&#194;&#176;C.                                                DNA fragment used transform diploid HR125-5d <br />    For chromosomal analysis GPHI, yeast chromosomes                  leucine prototrophy. Three Leu+ diploids sporu- <br /> were separated orthogonal-field alternating-gel electro-             lated, tetrads analyzed. In 13 16 tetrads (5 5, <br /> phoresis (OFAGE) 1.5% agarose gel (6). The gel                 4 5, 4 6), spores viable, 2:2 <br /> blotted nylon membrane filter UV irradiated (9).             segregation Leu+:Leu-, grew similar rates. <br /> The filter hybridized 32P-labeled DNA probes                     DNA prepared spore colonies derived from <br /> described above.                                                        single tetrad, Southern hybridization analysis used <br />    Total cellular RNA isolated described Jensen et            confirm GPHJ replaced disrupted gene in <br /> al. (26). RNA electrophoresed 1.2% agarose gel                 Leu+haploids (see Fig. 2A B). In addition, <br /> containing formaldehyde, transferred nylon membrane                chromosomes Leu- Leu+ haploids separated <br /> filter UV fixation (9), hybridized stringent             OFAGE (6), blotted filters, hybridized GPHI <br /> conditions (420C, 50% formamide, Sx SSPE, 5x Denhardt                   LEU2 probes (Fig. 2C). A preliminary assignment of <br /> solution, 0.1% sodium dodecyl sulfate) 32P-labeled               GPHI chromosome XIII XVI detection of <br /> 1.9-kb Sall fragment GPHJ.                                           GPHI hybridization band 10 OFAGE separation <br />    Cellular medium parameters. For growth curve mon-                (6). The LEU2 probe hybridized band in <br /> itoring, liquid growth carried 500-ml baffled                Leu+ cells, providing additional confirmation correct gene <br /> Erlenmeyer flasks (type 2543; Bellco Glass, Inc., Vineland,             disruption. The absence low-stringency hybridization in <br /> N.J.), containing 100 ml medium, incubated             Leu+ disruptants 1.9-kb DNA encoding catalytic <br /> rotary shaker 200 rpm 30&#194;&#176;C. Cell density moni-               portions yeast phosphorylase (Fig. 2) indicated hap- <br /> tored primarily A6. calibration optical density           loid cells contained copy phosphorylase gene. <br /> to cell number determined. Initial determinations cell           Therefore, GPHI gphlA::LEU2 segregants were <br /> number percentage budded cells using                equally viable, conclude GPHI essential to <br /> a cell-counting chamber (Hausser Scientific, Philadelphia,              vegetative growth. <br />  1662    HWANG ET AL.                                                                                              MOL. CELL. B3IOL. <br />  <br />    To test GPHJ required growth sporula- <br /> tion diploid cells, formed homozygous gphlA::LEU2 <br /> diploid mating gph1A&amp;:LEU2 haploid strains opposite <br /> mating types. Diploid colonies identified basis of <br /> inability mate MATa MATa tester strains. <br /> Homozygous gphlA::LEU2 diploids viable exhib- <br /> ited apparently normal morphology. When replica plated onto <br /> acetate medium, gphlA::LEU2 diploid cells able to <br /> sporulate produce viable gphlA::LEU2 haploid spores. <br />    <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>Phenotype</span> yeast cells containing gphlA::LEU2. The <br /> glycogen content spore clones single tetrads               f: <br />                                                                             }: <br /> examined qualitatively iodine staining. After growth on <br /> YEPD plates (44) 5 7 days, resulted prolonged <br /> incubation cells stationary phase, spore colonies began <br /> to differences staining intensity glycogen (Fig. 3) <br /> that depended GPHI gene disrupted. <br /> Darker lighter iodine staining tended develop an <br /> expected 2:2 ratio tetrad spores. Higher intracellular <br /> glycogen content, indicative reduced glycogen <br /> breakdown, segregated disruption GPHI leucine <br /> prototrophy. <br />    A direct comparison GPHI function gphlA:: <br /> LEU2 GPHI cells obtained assays phos- <br /> phorylase activity homogenized crude extracts yeast <br /> cultures grown stationary phase (Table 1). The assay <br /> procedure monitors release Pi phosphorylase cata-                              B <br /> lyzes incorporation glucose 1-phosphate glyco- <br /> gen. The Leu+ segregants released significantly phos- <br /> phate did Leu- cells, indicated absence                        2                  c      z        _ <br /> phosphorylase activity. The phosphate released gphiA                            3 <br />                                                                                              = <br />                                                                                                            _ <br />  ::LEU2 cells (PH3-6b PH3-6d) likely resulted from <br /> other phosphate-generating activity cell extract. Some <br />  measure contribution obtained assay reac- <br /> tions omitting phosphorylase substrates glucose 1-phosphate <br /> and glycogen (Table 1, columns). In results of <br />  assays glycogen, noted glycogen <br /> from diluted cell extract promote significant                                    * Leu <br />  degree phosphorylase-catalyzed phosphate release.                                            E Leu- <br />     Experiments carried determine pheno- <br />  typic differences Gph+ (GPHI) Gph- (gphiA::             FIG. 3. Glycogen staining haploid progeny derived from <br />  LEU2) cells. Wild-type mutant strains grown        GPHI-disrupted diploid cells. (A) Staining transformed HR125-5d <br />  identical conditions monitored various medium         cells. Cells sporulated, tetrads dissected micro- <br />  cellular parameters. YM-P (2% glucose) used grow-         manipulation. Segregants tetrads germinated on <br />  ing cultures, yeast growth carbohydrate accumu-         YEPD medium (44) 30&#194;&#176;C 2 days streaked the <br />  lation previously characterized medium (31).     YEPD plate shown, grown 5 days, stained solution of <br />  Overall, Gph+ Gph- strains exhibited similar growth       0.4% KI-0.2% 12. (B) Diagrammatic representation plate showing <br />  profiles (see Fig. 4A). Doubling times exponential         grouping spores derived individual.asci segre- <br />  growth identical. The consumption medium glucose          gation <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span> leucine prototrophy. In tetrad 8, four <br />  followed similar time courses. Adaptation fermentative       spores (a GPHI segregant) viable. <br />  respirative growth entry stationary phase also <br />  appeared similar Gph+ Gph- strains. Cell              exhaustion medium. Although basal level of <br />  morphologies, examined microscopically, similar           glycogen exponential Gph- cells appeared some- <br />  growth curve; percentage budded cells       higher Gph+ cells, general similarity of <br />  dropped manner strains reached station-       glycogen profiles presence absence phosphory- <br />  ary phase.                                                        lase indicated phosphorylase principal <br />     Intracellular glycogen trehalose contents        regulator glycogen content proliferative growth. <br />  determined growth Gph+ Gph- cells (Fig.         During stationary phase, Gph- cells contained significantly <br />  4). In wild-type yeast cells, levels carbohydrates   glycogen did Gph+ cells, understandably because <br />  generally low increase significantly cells approach        absence phosphorylase-catalyzed degradation in <br />  growth-arresting conditions (31). (Measurements glyco-         Gph- cells. It suggested glycogen be <br />  gen trehalose vary widely depending medium         used respirative adaptation diauxic growth (31). <br />  strain used, method normalization,         However, indication studies such <br />  factors imprecision intrinsic assay procedures.)        adaptation consequent entry stationary phase were <br />  Both Gph- Gph+ cells exhibited wild-type pattern       affected loss phosphorylase function. <br />  carbohydrate accumulation. In particular, glycogen accumu-           Comparison trehalose levels Gph+ Gph- <br />  lation strains began similar moments, near end     cells enabled determine absence phospho- <br />  exponential growth, advance glucose             rylase-catalyzed glycogenolysis affects pathway of <br />  VOL. 9, 1989                                                                    GLYCOGEN PHOSPHORYLASE IN S. CEREVISIAE                                  1663 <br />  <br />                        TABLE 1. Phosphorylase activity spore clones derived GPHI-disrupted transformanta <br />                                                                                               Phosphate release (nmollmin mg total protein)&apos; <br />            Strain              MAT            ~~~Leucine <br />                                             prototrophy <br />                                                                   Glycogenb <br />                                                                  accumulation         Complete reaction            Without added             Without added <br />                                                                                         Complterection          glucose 1-phosphate             glycogen <br /> PH3-6a                                          -                   -                      28.0                        6.7                      12.2 <br /> PH3-6b                                          +                   +                       6.4                        4.1                       9.3 <br /> PH3-6c                                          -                   -                      19.0                        3.0                      11.7 <br /> PH3-6d                                          +                   +                       9.3                        2.8                       9.1 <br /> PH3-6b/GPHI-2 .md                               +                   -                     573.0 <br /> PH3-6d/GPHI-2 .md                               +                   -                     373.0 <br />   Strains PH3-6a, PH3-6b, PH3-6c, PH3-6d represent single tetrad derived sporulation GPHI-disrupted HR125-5d cells. Cell cultures grown <br /> to early stationary phase liquid medium preparation crude cell extract enzyme assay. <br />   b Detected <br />               qualitatively iodine staining. <br />     Equivalent milliunits phosphorylase activity milligram protein. <br />   d GPHI present multicopy plasmid YEp24. <br />  <br />  <br /> trehalose synthesis. Gph+ Gph- cells contained similar                       differences phosphorylase activity glycogen con- <br /> amounts trehalose exponential growth                            tent, transformants appeared physiologically morpho- <br /> stationary phase. During mid- late log growth, strains                   logically similar untransformed strains. In particular, <br /> exhibited slightly decreasing levels trehalose. At end                    differences growth profile accumulation <br /> of exponential growth, Gph+ Gph- cells showed                              pattern glycogen detected. Furthermore, was <br /> similar pattern trehalose accumulation. As                           difference transformed wild-type strains in <br /> observed wild-type yeast strains (31), trehalose                        cell viability 2-week incubation stationary phase. <br /> synthesis Gph+ Gph- strains lagged                                Growth cycle dependence GPHI mRNA expression. Total <br /> initiation glycogen accumulation continued cells                    RNA prepared wild-type cells harvested several <br /> had reached stationary phase. The absence functional                          time points growth YM-P medium. Various medium <br /> GPHJ did impair capacity cells synthesize                      cellular parameters monitored harvesting <br /> store trehalose, did alter temporal relationship                   points (Fig. 5B). A band RNA corresponding 3.3-kb <br /> trehalose glycogen accumulation.                                             RNA species detected radiolabeled GPHI probe in <br />    Since glycogen yeast cells normally degraded                     samples late exponential stationary cells. GPHJ <br /> stationary phase, expected GPHI gphlA::LEU2                          transcripts observed initial growing cells, an <br /> cells exhibit differences cell viability incuba-                  indication GPHI gene repressed exponential <br /> tion stationary phase. However, observed differ-                        growth. Interestingly, highest level GPHI transcript <br /> ence GPHI gphlA::LEU2 cultures incubated                           detected cells proliferating exponentially <br /> in exhausted YM-P medium 4 weeks. Although                           medium glucose slightly declining. This time point <br /> yeast strain showed detectable loss viability, possible                 corresponded onset glycogen accumulation (Fig. <br /> that longer incubation required difference                    4). The GPHI gene did appear strict glucose <br /> detected. In wild-type yeast strains, net degradation                         repression, peak levels transcripts occurred in <br /> glycogen trehalose undetectable 2 weeks;                       advance glucose exhaustion medium. Glycogen <br /> furthermore, cell viability fails decline 3 months                  accumulation previously shown occur the <br /> incubation room temperature (31).                                             depletion glucose medium (31). The coincidence of <br />    An attempt discover condition GPHI                           initial appearance GPHI transcripts glycogen <br /> confers advantage yeast cells challenging                      accumulation suggests coordinated regulation glycogen <br /> gphlA::LEU2 GPHI cells grow 36&#194;&#176;C. Gph+                             phosphorylase glycogen synthase genes. <br /> Gph- cells plated YEPD medium grown                                It noted detected GPHI transcripts were <br /> 36&#194;&#176;C. Again, however, difference cell viability                         normalized mRNA constitutively expressed <br /> growth rate GPHI gphlA::LEU2                               gene, commonly Northern (RNA) analysis <br /> yeast strains.                                                                   profiles. This study yeast cells <br />    Increased gene dosage GPHI integration                        generate quite different mRNA profiles logarithmic <br /> plasmid YEp24. The phenotypic consequence in-                              stationary phases (3). By loading close equal <br /> creased dosage GPHI determined transfecting                            amounts total RNA possible gel lanes, effect <br /> yeast cells multicopy plasmid containing GPHI                         achieved normalization rRNA species predomi- <br /> gene. YEp24:GPHI, constructed described Materials                          nate total RNA preparations. <br /> and Methods, consisted 3.7-kb GPHI fragment inserted                           As extension experiment described above, GPHI <br /> into 2,um plasmid YEp24 (URA3). YEp24:GPHI                               transcripts monitored yeast cells stationary- <br /> used transform PH3-6b PH3-6d strains. Ura+ trans-                         to-log-phase adaptation. Cells initially grown satu- <br /> formants strain contain levels                      ration YM-P incubated 6 days exhausted <br /> phosphorylase activity 10- 40-fold higher wild-type                      medium, pelleted suspended in <br /> levels (Table 1). Using strains disabled genomic copy                     fresh medium. GPHI mRNA gradually diminished unde- <br /> of GPHJ assured phosphorylase overproduction                             tectable levels, transcripts present cells <br /> involved transcription plasmid-derived GPHI gene                        doublings preexponential phase (data <br /> and gene fragment cloned plasmid did                       shown). <br /> lack cis-acting element required GPHJ <br /> induction. Transformed cells contain                                                   DISCUSSION <br /> glycogen wild-type cells, determined iodine                              Yeast phosphorylase activated allosterically through <br /> staining glycogen assay (data shown). Apart                         reversible phosphorylation specific threonine residue <br />  1664                 HWANG ET AL.                                                                                                                                     MOL. CELL. BIOL. <br />  <br />     A     2- <br />                                                                                                                A                                                                Hrs. of <br />                                                                                                                          10   12 14.5 16     18.5   21       23   35       45   growth <br />                                                                                                                                                                                 GPH mRNA <br />                                                                                                    -1o8        3.2 kb-              w   t <br />                                                                                                                                         .w&apos; <br />                                                                                                                                               _          w        w        _ <br />  <br />  <br />                                                                                                           El <br />     C.)                     I <br />                                 I <br />                                                                                                                B <br />                                 I <br />                                     11 <br />                                                                                                           b <br />                                                                                                           U) <br />                                      I                                                                    Cu <br />                                          r)L <br />                                          Ili   .................    -s-9-98|    -o, <br />                                                                        .......... <br />  <br />  <br />                                                                                           , <br />                                                                                               L1   007             1 <br />                 10     15           20           25      30    35       40          45   50 <br />                                                        Time (hrs)                                                                                                                          .z <br />                                                                                                                                                                                           Z. <br />                                                                                                                    0 <br />                                                                                                                                                                                           -7c) <br />                                                                                     A)                                                                                                     z <br />                                                                                       -1&quot;                                                                                                  rA&gt; <br />                                                                                                                                                                                           -o <br />                                                                                                                    .%)                                                                    Z:L-- <br />                                                                                                                                                                                           C) <br />                                                                                                                                                                                           Q <br />           42- <br />  <br />  <br />                                                                                                                                             25     30        35       40 <br />                                                                                                                                             Time (hrs) <br />           0-                                                                                                      FIG. 5. Time course GPHI mRNA accumulation yeast <br />                                                                                                                cells approach stationary phase rich medium. (A) <br />                                                                                                                Time course Northern blotting analysis GPHI mRNA levels. <br />                                                                                                                Total cellular RNA isolated yeast cells growing YM-P <br />                                                                                                                medium containing 2% glucose. Conditions the <br />     CO          10     15           20            25     30    35       40          45   50 <br />                                                                                                                experiment shown Fig. 4. A 20-S&#194;&#177;g RNA prepa- <br />                                                                                                                ration electrophoresed 1.2% agarose gel containing form- <br />                                                        Time (hrs)                                              aldehyde hybridized 32P-labeled GPHI Sall fragment. <br /> I                                                                                                              (B) Accumulation GPHI mRNA yeast cells relation to <br />                                                                                                                glucose concentration rate cell proliferation the <br />                                                                                                                approach stationary phase. The dashed line indicates relative <br />                                                                                                                cellular levels GPHI transcript determined densitometer <br />                                                                                                                scanning Northern blot. <br />  <br />           0 <br />     -52 -                                                                                                      logical evidence indicated phosphorylase production <br />                                                                                                                Saccharomyces carlsbergensis occurs specifically at <br />                                                                                                                late exponential growth phase (1). We observed that <br />                                                                                                                intracellular levels phosphorylase highest yeast cells <br />      I1-                                                                                                       grown stationary phase (unpublished findings). <br />                                                                                                                   One objectives understand physiological <br />                10     15        20               25     30     35      40           45   50 <br />                                                                                                                role allosteriQ regulatory mechanisms yeast phos- <br />                                    Time (hrs)                                                                  phorylase. Our findings refute model glycogen metabo- <br />       FIG. 4. Growth profiles (A), glycogen accumulation (B), and <br />                                                                                                                lism constitutive level phosphorylase partici- <br />     trehalose accumulation (C) Gph+ Gph- yeast cells approach-                                          pates regulating glycogen content exponential <br />     ing stationary phase rich medium. YM-P medium containing 2%                                           growth phosphorylation-dephosphorylation inter- <br />     glucose inoculated (at t = 0 h) exponentially growing cells                                       conversion mediated cAMP regulatory cascade. Al- <br />     YM-P precultures initial cell densities 104/ml. The                                        study needed define role allosteric <br />     cultures incubated 30&#194;&#176;C rotary shaker 200 rpm.                                        regulation yeast phosphorylase, current information about <br />     Sampling begun cell density reached approximately                                             carbohydrate metabolism cAMP regulation yeasts <br />     107/ml. Symbols: 0, O, Gph- cells (PH3-6b, MATa gphlA::LEU2                                              suggests reasonable model. As phosphorylase glyco- <br />     strain derived HR125-5d); *, *, Gph+ cells (PH3-6a; MATa                                            gen accumulate end exponential growth, en- <br />     GPHI derivative HR125-5d).                                                                              zyme likely exists unphosphorylated b form. A <br />                                                                                                                predominance phosphorylase b phosphorylase is <br />     inhibited allosterically glucose 6-phosphate                                                   consistent observed decreasing concentrations of <br />     binding. Phosphorylation catalyzed specific phospho-                                               cAMP decreasing activity cAMP-dependent kinase as <br />     rylase kinase additionally cAMP-depen-                                                    haploid yeast cells approach stationary conditions (17, 51). <br />     dent protein kinase (39, 53). The unphosphorylated,                                                   Although active enzyme, phosphorylase b has <br />     active enzyme form, phosphorylase b, potently inhibited                                                 enzymatic activity (14) sufficient meet a <br />     glucose 6-phosphate, phosphorylated en-                                                     minimal energy demand stationary cells account for <br />     zymej phosphorylase a, sensitive effector. In                                              gradual diminution intracellular glycogen content (31). <br />     addition allosteric controls, yeast phosphorylase subject                                            Degradation glycogen stationary incubation would <br />     transcriptional regulation. This study indicates GPHJ                                              effectively modulated intracellular levels glucose <br />     transcript levels low exponential growth in-                                                6-phosphate, which, noted, effective <br />     crease entry stationary phase. Previous immuno-                                                inhibitor phosphorylase b. Interestingly, level of <br />  VOL. 9, 1989                                                       GLYCOGEN PHOSPHORYLASE IN S. CEREVISIAE                             1665 <br />  <br /> glucose 6-phosphate proliferating yeast cells rises          functions absence GPHI, glycogen degradation does <br /> medium glucose exhausted drops lower         appear play important role supplying glucose for <br /> level cells stationary phase (41).                     trehalose synthesis. Since observed trehalose <br />    When stationary yeast cells switched rich growth         accumulation continues exhaustion medium <br /> conditions, glycogen levels rapidly reduced (2), sugges-       glucose, gluconeogenesis probable primary source of <br /> tive profound activation phosphorylase activity,      glucose trehalose synthesis. Supporting view the <br /> as afforded phosphorylation. Hence, covalent           observation nonproliferating yeast cells incu- <br /> modification phosphorylase involving cAMP-mediated          bated presence alanine, nonfermentable carbon <br /> activation accessory kinase likely occurs        source, trehalose concentration increases amino <br /> adaptation rich medium. Such conditions shown         acid rapidly depleted (36), indication gluconeogen- <br /> to promote increases cAMP levels cAMP-dependent             esis provides precursors trehalose synthesis. <br /> kinase activity yeasts (17, 32, 40, 51). After glycogen            The likely explanation apparent dispensabil- <br /> depletion, subsequent exponential growth obviates met-         ity phosphorylase growth conditions used <br /> abolic requirement maintain phosphorylase cell.          study, utilization trehalose energy source <br />    The adaptive value maintaining different reserve         sufficiently compensated Gph- defect. To further <br /> carbohydrates yeast cells remains unclear us. Glycogen       examine matter, like determine stationary <br /> and trehalose display nonidentical patterns accumulation        conditions promote metabolism other <br /> and utilization varying nutritional limitations (31),        carbohydrate both. As alternate approach to <br /> which indicates reserves play distinct roles     dissecting metabolic roles glycogen trehalose, it <br /> the cell life cycle. To assess role glycogen            possible isolate cells survive without <br /> phosphorylase-catalyzed glycogenolysis yeast life           functional copy GPHI gene. Since glycogen and <br /> cycle, constructed strains lacking functional            trehalose appear serve overlapping purposes, mutant <br /> phosphorylase gene containing multiple copies            search approach identify defective genes specific <br /> gene. GPHI does appear essential gene. Haploid        trehalose pathway. Another hope identify the <br /> cells disrupted GPHI contained higher levels intracellu-     gene genes yeast glycogen synthase, enzyme that <br /> lar glycogen appeared similar haploid Gph+        catalyzes glycogen formation, possible homology to <br /> cells. Increasing gene dosage GPHI presenting      recently determined DNA protein sequences human <br /> a YEp24-based plasmid resulted expected increase          glycogen synthase (5). Genetic control synthetic <br /> phosphorylase activity unaccompanied ad-            degradative pathways glycogen metabolism pro- <br /> verse phenotype. The high-copy construction enables         vide ultimate tools defining role glycogen the <br /> us purify phosphorylase saturated yeast cultures       yeast organism. <br /> further biochemical structural study (V. L. Rath, P. K. <br /> Hwang, R. J. Fletterick, manuscript preparation).                                  ACKNOWLEDGMENTS <br />    Since phosphorylase principal enzyme glycogenoly-         We indebted Michael Hall Susan Lillie technical <br /> sis, apparent dispensability GPHJ suggests gly-        advice discussions Ira Herskowitz use of <br /> cogen degradation yeasts essential func-          laboratory facilities. <br /> tion. Phosphorolysis, however, mode               This work supported National Science Foundation grant <br /> glycogen breakdown yeasts. Cells undergoing sporulation         DMB84-20489. <br /> exhibit period glycogen degradation final <br /> maturation ascospores (27) result activity                          LITERATURE CITED <br /> sporulation-specific glucoamylase encoded SGA gene            1. Becker, J. U. 1982. Mechanisms regulation glycogen <br /> (10, 55); glycogen phosphorylase, fact, undetectable             phosphorylase activity Saccharomyces carlsbergensis. J. <br /> yeast cells sporulation (10). Interestingly, recent            Gen. Microbiol. 128:447-454. <br /> studies SGA gene cloned disrupted               2.   Becker, J. U., H. J. Vohmann, C. Eilers-Konig. 1979. <br /> indicate gene required successful sporu-             Glycogen metabolism resting growing cells Saccharo- <br />                                                                           myces carlsbergensis. Arch. Microbiol. 123:143-149. <br /> lation (55). This result provides supporting indication       3.   Boucherie, H. 1985. Protein synthesis transition and <br /> glycogen degradation essential.                                stationary phases glucose limitation Saccharomyces <br />    Given elaborate allosteric properties yeast phospho-            cerevisiae. J. Bacteriol. 161:385-392. <br /> rylase growth-dependent regulation GPHI tran-             4.   Bradford, M. M. 1976. A rapid sensitive method the <br /> scription, difficult accept dispensability GPHI               quantitation microgram quantities protein utilizing the <br /> function. The coincidence GPHI induction glycogen                  principle protein-dye binding. Anal. Biochem. 72:248-254. <br /> accumulation end logarithmic growth supports             5.   Browner, M. F., K. Nakano, A. G. Bang, R. J. Fletterick. <br /> primary physiological role phosphorylase utilizing                 1989. Human muscle glycogen synthase cDNA sequence: a <br /> glycogen reserve energy source periods                     negatively charged protein asymmetric charge distribu- <br />                                                                           tion. Proc. Natl. Acad. Sci. USA 86:1443-1447. <br /> <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> starvation. The high degree structural func-         6.   Carle, G. F., M. V. Olson. 1985. An electrophoretic <br /> tional conservation yeast mammalian phospho-                  karyotype yeast. Proc. Natl. Acad. Sci. USA 82:3756-3760. <br /> rylases argues yeast enzyme, like mam-          7.   Carney, I. T., R. J. Beynon, J. Kay, N. Birket. 1978. A <br /> malian counterparts, serves essential function                 semicontinuous assay glycogen phosphorylase. Anal. Bio- <br /> particular distress. It possible enzyme confers            chem. 85:321-324. <br /> yeast cells adaptive growth advantages               8.   Chen, G. S., I. H. Segal. 1968. Purification properties of <br /> apparent homogeneous cultures growth                      glycogen phosphorylase Escherichia coli. Arch. Biochem. <br /> conditions used laboratory.                                        Biophys. 127:175-186. <br />    Several reports suggested yeasts, trehalose          9.   Church, G. M., W. Gilbert. 1984. Genomic sequencing. <br />                                                                           Proc. Natl. Acad. Sci. USA 81:1991-1995. <br /> forms expense glycogen (17, 34). It evident       10.   Colonna, W. J., P. T. Magee. 1978. Glycogenolytic enzymes <br /> our study disruption GPHJ does impair                     sporulating yeast. J. Bacteriol. 95:1750-1757. <br /> capacity yeast cells accumulate high level trehalose.    11.   Davis, R. N., M. Thomas, J. Cameron, T. P. St. John, S. Scherer, <br /> Therefore, unless alternative pathway glycogenolysis                R. A. Padgett. 1980. Rapid DNA isolation enzymatic and <br />  1666       HWANG ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />      hybridization analysis. Methods Enzymol. 65:404 411.               34. Oura, S., E. Oura, H. Suomalainen. 1975. On formation <br /> 12. Fletterick, R. J., N. B. Madsen. 1980. The structures            glycogen trehalose baker&apos;s yeast. Eur. J. Appl. <br />      related functions phosphorylase a. Annu. Rev. Biochem.               Microbiol. 2:29-37. <br />      40:31-61.                                                          35. Palm, D., R. Goerl, K. J. Burger. 1985. Evolution of <br /> 13. Fletterick, R. J., S. R. Sprang. 1982. Glycogen phosphory-           catalytic regulatory sites phosphorylases. Nature <br />      lase structure function. Acc. Chem. Res. 15:361-369.                (London) 313:500-502. <br /> 14. Fosset, M., L. W. Muir, L. D. Nielsen, E. H. Fischer. 1971.     36. Panek, A. C., P. S. Araujo, V. Moura Neto, A. D. Panek. <br />      Purification properties yeast glycogen phosphorylase           1987. Regulation trehalose-P-synthase complex Sac- <br />      b. Biochemistry 10:4105-4113.                                       charomyces. Curr. Genet. 11:459466. <br /> 15. Fraenkel, D. 1982. Carbohydrate metabolism, p. 1-37. In J. N.       37. Panek, A. D. 1975. Trehalose synthesis starvation of <br />      Strathern, E. W. Jones, J. R. Broach (ed.), The molecular           baker&apos;s yeast. Eur. J. Appl. Microbiol. 2:3946. <br />      biology yeast Saccharomyces: metabolism gene            38. Panek, A. D., A. L. Sampaiao, G. C. Braz, S. J. Baker, J. R. <br />      expression. Cold Spring Harbor Laboratory, Cold Spring Har-             Mattoon. 1980. Genetic metabolic control trehalose and <br />      bor, N.Y.                                                               glycogen synthesis. New relationships carbohydrate <br /> 16. Fraenkel, D. G. 1985. On RAS gene function yeast. Proc.               reserves, catabolite repression maltose utilization. Cell. <br />      Natl. Acad. Sci. USA 82:47404744.                                       Mol. Biol. 25:334-354. <br /> 17. Francois, J., P. Eraso, C. Gancedo. 1987. Changes        39. Pohlig, G., R. Wingender-Drissen, J. U. Becker. 1983. <br />      concentration cAMP, fructose 2,6-bisphosphate related            Characterization phosphorylase kinase activities yeast. <br />      metabolites enzymes Saccharomyces cerevisiae              Biochem. Biophys. Res. Commun. 114:331-338. <br />      growth glucose. Eur. J. Biochem. 164:369-373.                   40. Purwin, C., F. Leidig, H. Holzer. 1982. Cyclic AMP- <br /> 18. Hall, M., L. Hereford, I. Herskowitz. 1984. Targeting E.          dependent phosphorylation fructose-1,6-bisphosphatase in <br />      coli P-galactosidase nucleus yeast. Cell 46:1057-1065.        yeast. Biochem. Biophys. Res. Commun. 107:1482-1489. <br /> 19. Henion, W. F., E. W. Sutherland. 1957. Immunological            41. Quain, D. E., J. M. Haslam. 1979. Changes glucose <br />      differences phosphorylases. J. Biol. Chem. 224:477-488.              6-phosphate storage carbohydrates catabolite dere- <br /> 20. Hers, H. G. 1976. The control glycogen metabolism              pression Saccharomyces cerevisiae. J. Gen. Microbiol. 113: <br />      liver. Annu. Rev. Biochem. 45:167-189.                                  195-198. <br /> 21. Hodgson, C. P., R. Z. Fisk. 1987. Hybridization probe size      42. Rothstein, R. J. 1983. One-step gene disruption yeast. Meth- <br />      control: optimized oligolabelling. Nucleic Acids Res. 15:6295.          ods Enzymol. 101:202-211. <br /> 22. Huang, K. P., E. Cabib. 1974. Yeast glycogen synthetase      43. Schane, H. P. 1965. Molecular weight estimation rat uterine <br />      glucose 6-phosphate-dependent form. I. Purification             phosphorylase. Anal. Biochem. 11:371-394. <br />      properties. J. Biol. Chem. 249:3851-3857.                          44. Sherman, F., G. R. Fink, J. B. Hicks. 1983. Methods in <br /> 23. Hwang, P. K., R. J. Fletterick. 1986. Convergent                 yeast genetics. Cold Spring Harbor Laboratory, Cold Spring <br />      divergent evolution regulatory sites eukaryotic phospho-           Harbor, N.Y. <br />      rylases. Nature (London) 324:80-84.                                45. Struhl, K. 1983. The new yeast genetics. Nature (London) 305: <br />                                                                               391-397. <br /> 24. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor-       46. Tatchell, K., L. C. Robinson, M. Breitenbach. 1985. RAS2 of <br />      mation intact yeast cells treated alkali cations. J.             Saccharomyces cerevisiae required gluconeogenic growth <br />       Bacteriol. 153:163-168.                                                 proper response <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation. Proc. Natl. Acad. <br /> 25. Jenkins, J. A., L. N. Johnson, D. I. Stuart, E. A. Stura, K. S.           Sci. USA 82:3785-3789. <br />      Wilson, G. Zanotti. 1981. Phosphorylase: control activ-     47. Toda, T., I. Uno, T. Ishikawa, S. Powers, T. Kataoka, D. Broek, <br />       ity. Phil. Trans. R. Soc. London Ser. B 293:2341.                       S. Cameron, J. Broach, K. Matsumoto, M. Wigler. 1985. In <br /> 26. Jensen, R., G. F. Sprague, I. Herskowitz. 1983. Regulation            yeast, RAS proteins controlling elements adenylate <br />       yeast mating-type interconversion: feedback control HO            cyclase. Cell 40:27-36. <br />       gene expression yeast mating type locus. Proc. Natl.        48. Trevelyan, W. E., J. S. Harrison. 1956. Studies yeast <br />       Acad. Sci. USA 80:3035-3039.                                            metabolism. 7. Yeast carbohydrate fractions. Separation from <br />  27. Kane, S. M., R. Roth. 1974. Carbohydrate metabolism                  nucleic acid, analysis, behavior anaerobic fermenta- <br />       ascospore development yeast. J. Bacteriol. 118:8-14.          tion. Biochem. J. 63:23-33. <br />  28. Krebs, E. G., J. A. Beavo. 1979. Phosphorylation-dephos-        49. Uno, I., K. Matsumoto, K. Adachi, T. Ishikawa. 1983. <br />       phorylation enzymes. Annu. Rev. Biochem. 48:923-959.                 Genetic biochemical evidence trehalase substrate <br />  29. Kuenzi, M. T., A. Fiechter. 1972. Regulation carbohy-             cAMP-dependent protein kinase yeast. J. Biol. Chem. <br />       drate composition Saccharomyces cerevisiae growth              258:10867-10872. <br />       limitation. Arch. Mikrobiol. 84:254-265.                           50. Van der Plaat, J. B., P. Van Solingen. 1974. Cyclic 3&apos;,5&apos;- <br />  30. Lerch, K., E. H. Fischer. 1975. Amino acid sequence           adenosine monophosphate stimulates trehalose degradation in <br />       functional sites yeast glyc <br /> </body></html>