<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RESEARCH ARTICLE <br />  <br />  <br />  <br />  <br /> In Vivo-Validated Essential Genes Identified Acinetobacter <br /> baumannii Using Human Ascites Overlap Poorly Essential <br /> Genes Detected Laboratory Media <br /> Timothy C. Umland,a,b L. Wayne Schultz,a,b Ulrike MacDonald,c,d,e Janet M. Beanan,c,d,e Ruth Olson,c,d,e Thomas A. Russoc,d,e,f <br /> Hauptman-Woodward Medical Research Institute, Buffalo, New York, USAa; Departments Structural Biology,b Medicine,c Microbiology Immunologyd the <br /> Witebsky Center Microbial Pathogenesis,e University Buffalo, State University New York, Buffalo, New York, USA; Veterans Administration Western New York <br /> Healthcare System, Buffalo, New York, USAf <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> A critical feature potential antimicrobial target characteristic essential growth <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>survival</span> dur- <br /> ing host infection. For bacteria, genome-wide essentiality screens usually performed rich laboratory media. This study <br /> addressed genes detected manner optimal identification antimicrobial targets vivo <br /> milieu fundamentally different. Mutant derivatives clinical isolate Acinetobacter baumannii screened growth <br /> on human ascites, ex vivo medium reflects infection environment. A subset 34 mutants unique gene disrup- <br /> tions demonstrated little growth ascites underwent evaluation rat subcutaneous abscess model, establishing 18 <br /> (53%) genes vivo essential. The putative gene products annotated biological functions, represented unrec- <br /> ognized underexploited antimicrobial targets, grouped functional categories: metabolic, two- <br /> component signaling systems, DNA/RNA synthesis regulation, protein transport, structural. These A. baumannii in <br /> vivo essential genes overlapped poorly sets essential genes Gram-negative bacteria catalogued Data- <br /> <span id='am-400' about='xsp:base' typeof='owl:Thing'>base</span> Essential Genes (DEG), including Acinetobacter baylyi, closely related species. However, finding not <br /> due absence orthologs. None 18 vivo essential genes identified study, putative gene products, <br /> were targets FDA-approved drugs drugs developmental pipeline, indicating significant portion available <br /> target space pathogenic Gram-negative bacteria currently neglected. <br /> IMPORTANCE The human pathogen Acinetobacter baumannii increasing clinical importance, growing proportion of <br /> isolates multiantimicrobial-resistant, pan-antimicrobial-resistant, extremely resistant strains. This scenario reflective of <br /> the general problem critical lack antimicrobials effective antimicrobial-resistant Gram-negative bacteria, as <br /> Pseudomonas aeruginosa, Klebsiella pneumoniae, Enterobacter sp., Escherichia coli. This study identified set A. bau- <br /> mannii genes essential growth survival infection demonstrated importance using clinically <br /> relevant media vivo validation screening essential genes purpose developing new antimicrobials. Fur- <br /> thermore, established gene absent Database Essential Genes, excluded potential an- <br /> timicrobial target. Lastly, new set high-value potential antimicrobial targets pathogenic Gram-negative bacteria been <br /> identified. <br />  <br />   Received 11 April 2012 Accepted 23 July 2012 Published 21 August 2012 <br />    Citation Umland TC, et al. 2012. In vivo-validated essential genes identified Acinetobacter baumannii using human ascites overlap poorly essential genes detected <br />    laboratory media. mBio 3(4):e00113-12. doi:10.1128/mBio.00113-12. <br />    Editor George Drusano, University Florida&#226;&#8364;&#8221;Albany <br />    Copyright &#194;&#169; 2012 Umland et al. This open-access article distributed terms Creative Commons Attribution-Noncommercial-Share Alike 3.0 Unported <br />    License, permits unrestricted noncommercial use, distribution, reproduction medium, provided original author source credited. <br />    Address correspondence Timothy C. Umland, umland@hwi.buffalo.edu, Thomas A. Russo, trusso@buffalo.edu. <br />  <br />  <br />  <br />  <br /> T    identification bacterial essential genes (i.e., genes re- <br />      quired growth and/or survival) important tool <br /> for dissecting biological pathways functions, identifying evo- <br />                                                                                             genetic variation strains, random versus systematic mu- <br />                                                                                             tagenesis protocols, growth condition differences, clonal versus <br />                                                                                             mixed populations, working <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> used designate a <br /> lutionary relationships, promoting synthetic biology, predict-                          gene essential) (1&#226;&#8364;&#8220;4). These complexities exemplified by <br /> ing antimicrobial targets. An essential gene considered                          comparison genome-wide essentiality screens conducted upon <br /> member minimal gene set required growth specific                            species using different methodologies. In case of <br /> organism optimal conditions (i.e., <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-rich media                            Escherichia coli, Keio collection identified 303 essential genes <br /> absence environmental stress) (1). This <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> integrates                            (5), earlier screen identified 620 essential genes (6), Eco- <br /> with practical consideration using standard laboratory                            gene reannotated Keio collection representing 289 es- <br /> vitro growth conditions. Despite seemingly simple <span id='am-401' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>                           sential genes using stringent criteria (7). Expanding the <br /> of &#226;&#8364;&#339;essential gene,&#226;&#8364;? practice number experimental subtle-                           comparison species include additional Gram-negative <br /> ties exist influence identification genes (e.g.,                      bacteria (GNB) distantly related Gram-positive bacteria <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                                                     &#194;&#174; <br />  <br />                                                                                                                                                                     mbio.asm.org 1 <br />  Umland et al. <br />  <br />  <br />  <br /> (GPB) emphasizes disparity experimentally           peutics effective A. baumannii MDR, XDR, PDR <br /> determined bacterial essential gene sets. For example, compari-       GNB general, employed previously utilized experimental <br /> son Helicobacter pylori essential genes E. coli, Haemophilus      approach (32) identification unrecognized underex- <br /> influenzae, Mycobacterium tuberculosis indicated 55%       ploited antimicrobial targets. This approach potential <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> <br /> essential species, 11% essen-    effectively promote development new small-molecule anti- <br /> tial considered species (8).                                     microbial therapeutics novel targets circum- <br />     Gene essentiality increasingly viewed contextual,       vent current resistance mechanisms (33). This strategy de- <br /> with decreased <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> levels, changes carbon sources, en-       signed efficiently identify A. baumannii genes essential for <br /> vironmental stress (e.g., change temperature) altering set    growth survival infection rat subcutaneous abscess <br /> genes required growth (1, 9&#226;&#8364;&#8220;11). A recent study testing yeast       model (i.e., vivo essential genes) essential vitro rich <br /> deletion mutants multitude small molecules en-         laboratory media (34, 35). Here, report 18 verified vivo es- <br /> vironmental stresses concluded 97% genes con-         sential genes form set putative antimicrobial targets, rep- <br /> tribute wild-type growth presence chem-        resenting largely new set antimicrobial targets based upon <br /> icals environmental conditions (12), compared ~20%         evaluation drug target databases. These A. baumannii <br /> genes annotated essential optimal laboratory growth            genes compared bacterial genes annotated essential in <br /> conditions. A similar chemical genetics approach E. coli identi-     DEG, revealing 89% readily predicted vivo <br /> fied 116 genes, unique present Keio collection,       essential. Data presented support concept the <br /> that essential rich laboratory media stressed         genes identified catalogued essential DEG need be <br /> 324-chemical screen (13). These studies underscore fact        interpreted context method identification. <br /> microorganisms infrequently encounter ideal growth conditions           Our review literature revealed screens con- <br /> except lab, evolved grow survive      ducted using rich laboratory media, performed <br /> multiple changing environments. For example, pathogenic bac-          using clinically relevant medium, demonstrating limitation of <br /> terium encounters different environment infection         data currently DEG. This highlights cataloguing of <br /> of host (<span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> poor, host defenses) growth lab      essential genes incomplete, labor cost <br /> media (<span id='am-8' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> rich).                                                  intensive, experimental establishment vivo essential genes <br />     Whole-genome essentiality screening requires considerable re-       appropriate environmental context needed. An un- <br /> sources, bacterial species selected screens largely    derstanding limitation DEG critical is <br /> important model systems (e.g., E. coli) human pathogens. The         used decision process assessing validity of <br /> Database Essential Genes (DEG; version 6.8) contains genome-         potential drug targets. Consideration potential drug targets <br /> scale essentiality data 17 unique bacterial species obtained        clinically relevant context enabled antimicrobial tar- <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> 20 published screens (14). For majority bacterial spe-      space expanded, enabled refinement of <br /> cies, experimental data establishing gene essentiality sparse,      bioinformatic methods used prioritize antimicrobial targets. <br /> and annotation &#226;&#8364;&#339;essential&#226;&#8364;? bacterial genes a <br /> prediction based homology experimentally established es- <br /> sential genes. Given modest overlap experimentally deter-        RESULTS <br /> mined whole-genome essentiality screens (pairwise bacterial spe-        Identification A. baumannii genes essential growth hu- <br /> cies comparisons typically exhibit 50 70% overlap,             man ascites ex vivo. A. baumannii strain 307-0294 (AB307-0294) <br /> rapidly decreases species compared [8, 15&#226;&#8364;&#8220;17]), con-        randomly mutagenized using transposon EZ-Tn5&#226;&#172;?kan- <br /> fidence levels silico essentiality predictions vary widely   2&#226;&#172;&#381;. Mutants (2,934 total) isolated Mueller-Hinton (MH) <br /> and cases quantified. Bioinformatic selec-        kanamycin plates subsequently gridded ascites plates <br /> tion strategies target-based antimicrobial discovery rely heav-     containing kanamycin (40 &#226;?&#174;g/ml). Mutant derivatives (224 total; <br /> ily essentiality annotations (18&#226;&#8364;&#8220;20), evaluation      7.6%) AB307-0294 identified displayed significantly <br /> the accuracy essential nonessential gene predictions       decreased absent growth ascites plates. Chromosomal se- <br /> practical importance.                                                   quencing using primers ends EZ-Tn5&#226;&#172;?kan-2&#226;&#172;&#381; was <br />     Essential genes hold promise potential new drug        performed 224 mutants, enabling location trans- <br /> targets. We risk entering postantibiotic era,   poson insertion sites identified. Analysis data re- <br /> evolution multidrug, extreme drug, pan-drug (MDR, XDR,           vealed set 90 unique genes loss function to <br /> PDR) resistance GNB. Identification new drug targets         transposon-mediated gene disruption resulted <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> lead development new antimicrobials, urgently       near wild-type growth rich laboratory (MH) medium plates <br /> needed. Acinetobacter baumannii poster child            decreased absent growth ascites plates. This phenotype is <br /> emerging threat (21), incidence infections         likely predictive AB307-0294 genes dispensable vitro <br /> those XDR PDR strains increasing (22&#226;&#8364;&#8220;29). Treatment          growth optimal conditions (rich laboratory medium, <br /> of infections Acinetobacter challenging,      does reflect vivo environment) essential for <br /> need identify new antimicrobial targets pressing        growth survival systemic infection human host <br /> ever. Unfortunately, 2009 virtually new anti-       (i.e., essential vivo), human ascites similar exuda- <br /> microbial agents active GNB pharmaceutical pipe-         tive inflammatory fluid environment extracellular bacterial <br /> line (21). A 2011 update antimicrobials develop-       pathogens encounter human infection. This simple con- <br /> ment activity GNB, reached phase         cept strength approach, generally been <br /> 3 trials (30, 31).                                                      used screens essential genes. Based bioinformatic <br />     Given urgent need new classes antimicrobial thera-       filter, 34 90 A. baumannii genes selected, primarily <br />  <br />  <br /> 2       &#194;&#174; <br />  <br />             mbio.asm.org                                                                            July/August 2012 Volume 3 Issue 4 e00113-12 <br />                                                                                                                    A. baumannii In Vivo Essential Gene Screening <br />  <br />  <br />  <br />  <br /> FIG 1 Growth/clearance AB307-0294 wild type (wt) mutant derivatives rat subcutaneous abscess infection model. (A) Comparison wild type <br /> (AB307-0294) mutants disruptions metabolic pathway two-component signaling genes. AB307.12 (carA&#226;&#171;&#186;), AB307.68 (aroA&#226;&#171;&#186;), AB307.70 <br /> (rstA&#226;&#171;&#186;), AB307.73 (aceE&#226;&#171;&#186;), AB307.153 (envZ&#226;&#171;&#186;), AB307.170 (aroC&#226;&#171;&#186;). (B) Comparison wild type mutants disruptions DNA/RNA synthesis or <br /> regulation protein transport genes. AB307.09 (pyrC&#226;&#171;&#186;), AB307.155 (rpmA&#226;&#171;&#186;), AB307.162 (trmD&#226;&#171;&#186;), AB307.211 (secE&#226;&#171;&#186;), AB307.216 (relA&#226;&#171;&#186;). (C) Comparison <br /> of wild type mutants disruptions structural genes. AB307.17 (ompF&#226;&#171;&#186;), AB307.27 (pbpG&#226;&#171;&#186;), AB307.30 (ptk&#226;&#171;&#186;), AB307.45 (epsA&#226;&#171;&#186;), AB307.80 (uppP&#226;&#171;&#186;), <br /> AB307.92 (spsC&#226;&#171;&#186;), AB307.177 (ostA&#226;&#171;&#186;). Data means &#226;&#171;&#190; SEM 3 4 experiments time point. <br />  <br />  <br />  <br /> based lack homology human genes, verification                     reduction original bacterial titer 48 h postin- <br /> in vivo essentiality.                                                               fection (hpi). Impressively, 10 18 mutant strains exhib- <br />     Fifty-three percent AB307-0294 mutant derivatives de-                   iting significant vivo growth defect &#226;&#177;&#8211;5 log10 reduc- <br /> creased growth human ascites essential rat subcutane-                  tion and/or complete kill 48 hpi (Fig. 1). Furthermore, <br /> ous abscess model. Thirty-four unique AB307-0294 mutant strains                     48 hpi, mutant strains exhibited &#226;&#177;&#8211;5 log10 reduc- <br /> were tested well-established rat subcutaneous abscess model               tion compared corresponding titer wild-type <br /> has validated use A. baumannii (36). Importantly, mul-                AB307-0294 control. Mutations resulted bacteriostatic <br /> tiple host defense factors, including complement, professional                      mildly decreased growth test period clas- <br /> phagocytes, antimicrobial peptides, present model.                  sified vivo essential genes. All 18 vivo essential <br /> This critical step verifies vivo essentiality monitoring              genes annotated biological functions grouped <br /> quantitative growth survival specific mutant strains time.              functional categories (metabolic, two-component sig- <br /> The use vivo infection model (i.e., infection host)               naling [TCS] systems, DNA/RNA synthesis regulation, <br /> subjects A. baumannii mutants range conditions, including               protein transport, structural). These data demonstrate that <br /> restricted nutrients immune responses, difficult impossible               genes disrupted mutants decreased growth in <br /> completely replicate outside host.                                                human ascites likely essential vivo. <br />     This vivo assessment revealed 18 34 (53%)                            Genes identified essential growth and/or survival <br /> mutated genes essential vivo (Fig. 1; Table 1). The cri-                    clinically relevant conditions essential rich lab- <br /> terion used classify gene vivo essential &#226;&#177;&#8211;2 log10                  oratory medium. Importantly, 18 mutants grew rich <br />  <br />  <br /> TABLE 1 AB307-0294 vivo essential genes gene products <br />                                                                                                                                        Polar effect <br /> Ab307-0294 gene locus         Gene     <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>Mutation</span> no.a      Annotation                                     UniProt no.   OMA no.         exclusion <br /> ABBFA_000154                  aceE     73                 Pyruvate dehydrogenase E1 component            B7GV82        ACIB300151      RT-PCR amplicon <br /> ABBFA_000285                  envZ     153                Osmolarity sensor protein EnvZ                 B7GVK9        ACIB300278      RT-PCR amplicon <br /> ABBFA_000350                  trmD     162                tRNA (guanine-1)-methyltransferase             B7GVS0        ACIB300339      RT-PCR amplicon <br /> ABBFA_000621                  ompF     17                 Outer membrane porin F precursor               B7GWT4        ACIB300599      Genome organization <br /> ABBFA_000700                  uppP     80                 Undecaprenyl-diphosphatase UppP                B7GXE2        ACIB300672      RT-PCR amplicon <br /> ABBFA_000738                  rpmA     155                Ribosomal protein L27                          B7GXH9        ACIB300709      RT-PCR amplicon <br /> ABBFA_000787                  carA     12                 Carbamoyl-phosphate synthase small chain       B7GY14        ACIB300758      RT-PCR amplicon <br /> ABBFA_001168                  aroA     68                 3-Phosphoshikimate 1-carboxyvinyltransferase   B7GZU8        ACIB301129      RT-PCR amplicon <br /> ABBFA_001801                  aroC     170                Chorismate synthase                            B7H3A6        ACIB301758      Genome organization <br /> ABBFA_001929                  ostA     177                Organic solvent tolerance protein OstA         B7H3N4        ACIB301886      RT-PCR amplicon <br /> ABBFA_002478                  pyrC     9                  Dihydroorotase, type II                        B7GX48        ACIB302431      Complementation <br /> ABBFA_002866                  rstA     70                 Transcriptional regulatory protein RstA        B7GZC5        ACIB302811      Downstream gene&#226;&#8364;&#8221; <br />                                                                                                                                          identical signal path <br /> ABBFA_002981                  relA     216                GTP pyrophosphokinase                          B7H017        ACIB302919      Genome organization <br /> ABBFA_003254                  secE     211                Preprotein translocase, SecE subunit           B7H1K4        ACIB303187      RT-PCR amplicon <br /> ABBFA_003295                  pbpG     27                 Penicillin-binding protein 7/8 (PBP7/8)        B7H1P0        ACIB303223      Complementation <br /> ABBFA_003451                  spsC     92                 Perosamine synthase                            B7H2H8        ACIB303374      RT-PCR amplicon <br /> ABBFA_003459                  epsA     45                 EPS I polysaccharide export outer              B7H2I6        ACIB303382      Complementation <br />                                                             membrane protein EpsA precursor <br /> ABBFA_003461                  ptk      30                 Tyrosine-protein kinase Ptk                    B7H2I8        ACIB303384      Complementation <br /> a   Mutation number assigned specific AB307-0294 transposon mutant. <br />  <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                                 &#194;&#174; <br />  <br />                                                                                                                                                 mbio.asm.org 3 <br />  Umland et al. <br />  <br />  <br />  <br /> TABLE 2 Growth characteristics A. baumannii vivo essential gene mutants <br />                                                                                                                      Growtha <br /> AB307-0294 gene                         Function                                                                     LB                      Ascites                      In vivo <br /> aceE                                    Carbohydrate amino acid metabolism                                       &#226;&#171;&#185;2                          &#226;&#171;&#185;1                       &#226;&#171;&#186;2 <br /> envZ                                    Two-component signaling                                               &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> trmD                                    tRNA modification                                                            &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> ompF                                    Outer membrane porin                                                         &#226;&#171;&#185;2                          &#226;&#171;&#186;2                       &#226;&#171;&#186;2 <br /> uppP                                    Peptidoglycan synthesis                                                      &#226;&#171;&#185;2                          &#226;&#171;&#186;2                       &#226;&#171;&#186;2 <br /> rpmA                                    Ribosome assembly translation                                            &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> carA                                    Pyrimidine metabolism                                                        &#226;&#171;&#185;1                          &#226;&#171;&#186;2                       &#226;&#171;&#186;2 <br /> aroA                                    Chorismate aromatic amino acid metabolism                                &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> aroC                                    Chorismate metabolism                                                        &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> ostA                                    Outer membrane LPS assembly                                                  &#226;&#171;&#185;2                          &#226;&#171;&#186;2                       &#226;&#171;&#186;2 <br /> pyrC                                    Pyrimidine metabolism                                                        &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> rstA                                    Two-component signaling                                               &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> relA                                    Nucleotide metabolism; stringent response                                    &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> secE                                    Protein transport                                                            &#226;&#171;&#185;2                           0                       &#226;&#171;&#186;2 <br /> pbpG                                    Peptidoglycan synthesis/modification                                         &#226;&#171;&#185;2                          &#226;&#171;&#186;2                       &#226;&#171;&#186;2 <br /> spsC                                    LPS synthesis/modification                                                   &#226;&#171;&#185;2                          &#226;&#171;&#186;1                       &#226;&#171;&#186;2 <br /> epsA                                    Capsule transport, outer membrane                                            &#226;&#171;&#185;2                          &#226;&#171;&#186;1                       &#226;&#171;&#186;2 <br /> ptk                                     Capsule synthesis                                                            &#226;&#171;&#185;2                          &#226;&#171;&#186;1                       &#226;&#171;&#186;2 <br /> a Growth mutant strain compared wild type LB-rich laboratory medium, human ascites, vivo rat subcutaneous abscess infection model. Growth scale: <br />  <br /> &#226;&#171;&#185;2, wild type; &#226;&#171;&#185;1, wild type; 0, growth kill; &#226;&#171;&#186;1, modest kill; &#226;&#171;&#186;2, significant kill. <br />  <br />  <br />  <br /> Luria-Bertani (LB) laboratory medium, carA&#226;&#171;&#186;                                  The correlation experimentally identified AB307-0294 in <br /> mutant displaying growth near wild-type levels (Table 2).                           vivo essential genes compared present DEG poor. <br /> The optimal density 600 nm (OD600) carA&#226;&#171;&#186; mutant cul-                            The results AB307-0294 vivo essentiality screen were <br /> ture ~66% corresponding wild-type culture 24 h.                          compared bacterial genes annotated essential DEG <br /> Although mutant growth results LB medium ex-                                (version 6.8) (14). DEG contains results 20 genome-wide <br /> pected, based design essentiality screen, empha-                       essentiality screens 17 unique bacterial species (see Table S1 <br /> size fundamental difference screens define                         Fig. S1 supplemental material). <br /> essentiality lack growth rich laboratory medium. In con-                           DEG searched orthologs 18 identified A. bau- <br /> trast, screened set mutants survived LB                         mannii vivo essential genes. The ortholog match criterion was <br /> medium human ascites. This point critical genes,                        selected maximize likelihood A. baumannii DEG <br /> which shown essential growth and/or sur-                          matches possessed identical highly similar biological functions <br /> vival A. baumannii ex vivo human ascites vivo rat                       (37&#226;&#8364;&#8220;39), practice followed (40, 41). This especially <br /> infection model, identified rich labora-                         important large superfamilies, subfamilies have <br /> tory medium screen. Since human host modified minimal                            distinct functional differences. DEG ortholog matches then <br /> medium (e.g., iron depleted) contains host defense factors,                          separated GNB only; GNB only, excluding A. baylyi; non- <br /> the clinical relevance essential genes identified rich labora-                      GNB (i.e., GPB, mycobacteria, mycoplasma); bacteria <br /> tory medium uncertain misleading.                                               species (Fig. 2). The majority A. baumannii vivo essential <br />  <br />  <br />  <br />  <br /> FIG 2 Histograms occurrence orthologs 18 AB307-0294 vivo essential genes identified study bacterial genome-wide <br /> essentiality screen data compiled DEG. Columns left histogram indicate AB307-0294 vivo essential genes having orthologs <br /> annotated essential DEG. (A) Comparison 12 GNB data sets DEG (red columns) set excluding A. baylyi (blue columns); <br /> (B) comparison 8 non-GNB data sets (includes GPB, mycobacterium, mycoplasma species) DEG; (C) comparison 20 bacterial data sets <br /> present DEG. <br />  <br />  <br />  <br /> 4       &#194;&#174; <br />  <br />             mbio.asm.org                                                                                                     July/August 2012 Volume 3 Issue 4 e00113-12 <br />                                                                                                                   A. baumannii In Vivo Essential Gene Screening <br />  <br />  <br />  <br />  <br /> FIG 3 The occurrence orthologs 18 AB307-0294 vivo essential genes identified study bacterial genome-wide essentiality <br /> screen data compiled DEG (red columns) genomes bacterial species (blue columns). (A) Comparison 12 GNB data sets within <br /> DEG; (B) comparison 8 non-GNB bacterial data sets (includes GPB, mycobacterium, mycoplasma species) DEG; (C) comparison 20 <br /> bacterial data sets present DEG. <br />  <br />  <br /> genes/gene products bacterial orthologs annotated                 Over 70% (13/18) A. baumannii genes orthologs in <br /> within DEG essential. The distribution DEG ortholog                       &#226;&#172;&#381;50% bacterial species annotated DEG. Furthermore, <br /> matches skewed slightly greater frequency                 A. baumannii genes, rstA, ortholog &#226;&#172;?25% <br /> the DEG GNB data considered, 13/18 (72%) genes                    DEG bacterial species classified unique to <br /> &#226;&#177;&#8226;1 ortholog 6/18 (33%) matches (Fig. 2A). This                      Acinetobacter. Orthologs frequently present DEG <br /> observation expected, generally DEG GNB species                   GNB species (Fig. 3A) versus DEG non-GNB species (Fig. 3B), <br /> are closely related A. baumannii DEG&#226;&#8364;&#8482;s non-GNB                     unexpected. While 18 A. baumannii vivo essen- <br /> species, 15/18 (83%) genes &#226;&#177;&#8226;1 ortholog 13/18                  tial genes universally conserved bacteria, are <br /> (72%) genes matches (Fig. 2B). Likewise, excluding con-                  widely distributed species, especially GNB. <br /> sideration closely related species, A. baylyi, shifts           Thus, general absence annotated essential in <br /> distribution lower occurrence orthologs A. bau-                  DEG genes unique Acinetobacter. <br /> mannii vivo essential genes DEG&#226;&#8364;&#8482;s GNB set (14/18 [77%]                    The growth environment used essentiality screen ap- <br /> with &#226;&#177;&#8226;1 ortholog 10/18 [56%] matches) (Fig. 2A).                    pears critical. The fact AB307-0294 mutant deriv- <br /> Therefore, fact ortholog listed essential DEG          atives disruptions genes essential growth and/or sur- <br /> or identified essential species         vival vivo identified using screen human ascites <br /> means excludes essential vivo. This obvious im-              likely explains poor correlation genes annotated essential <br /> plications given data DEG typically heavily                  DEG. Nearly screens used identify genes DEG <br /> weighted decisions validity gene products                performed rich laboratory medium. A single genome- <br /> potential antimicrobial targets given genera species.                  wide essentiality screen DEG conducted de- <br />     It noteworthy A. baumannii vivo essential               fined minimal medium, better representation the <br /> genes ortholog hits DEG possess impor-                 nutrient levels available human growth environment. The <br /> tant phenotypical difference matches. All bacterial es-              organism screened belonged genus (Acinetobac- <br /> sentiality screens DEG, exception A. baylyi              ter baylyi), seven 18 identified essential genes were <br /> screen, conducted rich laboratory media. Hence, genes                   common (42) (Table 3). The Keio collection E. coli genome- <br /> annotated essential screens indicate re-               wide-directed single gene mutants, excluding mutants nonviable <br /> spective gene disruption mutants nonviable rich media.                  rich media, evaluated growth characteristics in <br /> However, 18 A. baumannii vivo essential genes iden-               defined minimal medium (10). This study detected E. coli <br /> tified study grew near wild-type levels rich media             genes essential minimal media addition E. coli <br /> (Table 2). Thus, 18 A. baumannii genes form set unique               genes essential rich media orthologous A. baumannii <br /> the essential genes DEG.                                                     vivo essential genes. The results studies com- <br />     The poor correlation experimentally identified AB307-                    pared 18 A. baumannii vivo essential genes determine if <br /> 0294 vivo essential genes compared annotated                     E. coli A. baylyi ortholog mutants&#226;&#8364;&#8482; growth phenotypes in <br /> DEG absence orthologs. A potential reason                 minimal media correlated vivo growth survival pheno- <br /> why A. baumannii vivo essential gene orthologs pres-                types A. baumannii. Only 3 18 A. baumannii vivo <br /> ent DEG 17 bacterial species annotated DEG                essential genes unambiguously correlated minimal medium <br /> simply lack orthologous genes (i.e., gene annotated              growth phenotype mutations orthologous genes both <br /> essential gene present genome wild-type            E. coli A. baylyi. While definitive conclusions be <br /> organism). The OMA ortholog database searched confirm                    drawn, results suggest minimal medium im- <br /> that 18 A. baumannii genes, orthologous genes                   proved screening environment compared rich laboratory me- <br /> were prevalent bacterial species contained DEG (Fig. 3).              dium closely represents vivo environment. <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                                &#194;&#174; <br />  <br />                                                                                                                                                mbio.asm.org 5 <br />  Umland et al. <br />  <br />  <br />  <br /> TABLE 3 A. baumannii vivo essential genes versus essentiality                          TABLE 4 A. baumannii vivo essential genes antimicrobial targets <br /> minimal medium <br />                                                                                              Gene                                   UR/UEa                                     Rankb <br />                         Minimal medium study resulta                                         aceE                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         H <br /> AB307-0294 gene         E. coli                A. baylyi            Prediction vivob      envZ                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         M <br />                                                                                              trmD                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         M <br /> aceE                    N                      N                    NE <br />                                                                                              ompF                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        L <br /> envZ                    N                      Y                    ? <br />                                                                                              uppP                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        H <br /> trmD                    NT                     Y                    EE <br />                                                                                              rpmA                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         M <br /> ompF                    N                      N                    NE <br />                                                                                              carA                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         H <br /> uppP                    N                      N                    NE <br />                                                                                              aroA                                   &#226;&#171;&#185;                                          H <br /> rpmA                    NT                     N                    ? <br />                                                                                              aroC                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         H <br /> carA                    Y                      Y                    E <br />                                                                                              ostA                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        L <br /> aroA                    Y                      N                    ? <br />                                                                                              pyrC                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         H <br /> aroC                    Y                      Y                    E <br />                                                                                              rstA                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        L <br /> ostA                    NT                     N                    ? <br />                                                                                              relA                                   &#226;&#171;&#185;&#226;&#171;&#185;                                         M <br /> pyrC                    Y                      Y                    E <br />                                                                                              secE                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        L <br /> rstA                    No ortho               N                    NE <br />                                                                                              pbpG                                   &#226;&#171;&#185;                                          H <br /> relA                    N                      Y                    ? <br />                                                                                              spsC                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        H <br /> secE                    No ortho               Y                    ? <br />                                                                                              epsA                                   &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        M <br /> pbpG                    N                      N                    NE <br />                                                                                              ptk                                    &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;                                        M <br /> spsC                    No ortho               N                    NE <br />                                                                                              UR/UE, unrecognized/underexploited drug target evaluated comparison to <br /> epsA                    N                      N                    NE <br /> ptk                     N                      N                    NE                       entries DrugBank (56) Therapeutic Target Database (57). &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;, no <br />                                                                                              ortholog present; &#226;&#171;&#185;&#226;&#171;&#185;, ortholog entry present, listed inhibitor(s) classified as <br /> a Keio Collection growth study E. coli mutants defined minimal MOPS                    experimental (i.e., prior preclinical studies) nondrug like; &#226;&#171;&#185;, ortholog entry <br /> (morpholinepropanesulfonic acid) medium (10) A. baylyi essentiality study                present related protein class targeted approved drug chemical <br /> performed defined minimal MA medium (42). N, gene essential minimal                inhibitor commercial use. <br /> medium; Y, gene essential minimal medium; No ortho, absence orthologous gene;          b Evaluation potential antimicrobial drug target: H, high; M, medium; L, low. <br /> NT (E. coli only), tested minimal medium, gene essential rich medium. <br /> b Prediction AB307-0294 gene vivo essential based minimal <br />  <br /> medium studies E. coli A. baylyi. E, predicted vivo essential, growth in <br /> minimal medium E. coli A. baylyi ortholog mutants; EE, predicted                developmental pipeline. Almost half lacked entry drug <br /> essential condition (i.e., rich minimal medium); NE, predicted       target databases. <br /> vivo essential, growth occurs minimal medium E. coli A. baylyi                Two genes (aroA pbpG) lacked entry ortholog, <br /> ortholog mutants; ?, prediction vivo essentiality ambiguous different results <br /> in E. coli A. baylyi minimal medium screens. <br />                                                                                              entries did exist related protein families targets of <br />                                                                                              drug drug-like small molecules. In cases, attention <br />                                                                                              related proteins received antimicrobial targets was <br />                                                                                              largely spillover extensive efforts nonbacte- <br /> However, does serve optimal surrogate vivo                            rial targets. In plants, gene product aroA, <br /> growth environment GNB. A partial explanation                               3-phosphoshikimate 1-carboxyvinyltransferase (PSCVT), the <br /> be vivo conditions possess host defense factors                            target broad-spectrum herbicide glyphosate (Roundup). <br /> bactericidal activity present laboratory me-                             However, plant PSCVTs glyphosate sensitive (class I), <br /> dium. In contrast, use human ascites likely enhanced                              bacterial PSCVT enzymes belong class I class II <br /> identification genes essential growth and/or survival vivo                         (glyphosate resistant). Additionally, previous bacterial <br /> because contains active complement.                                                       genome-wide essentiality screens, aroA ortholog detected <br />    Further, recent genome-wide chemical genetics E. coli study                             essential Caulobacter crescentus (Table 5). Supporting <br /> was designed identify genes essential specific chemical                          data exist indicating vivo importance aroA patho- <br /> environmental stresses (13). Few conditionally essential                              genic GNB. Salmonella enterica aroA mutants yield highly attenu- <br /> genes identified overlapped A. baumannii vivo essential                          ated strains inability produce aromatic metab- <br /> genes identified study, suggesting extensive chemical                            olites cell wall outer membrane defects reduce <br /> genetics screens replacement vivo studies.                                  resistance innate immune responses (43). The gene pbpG en- <br />    The majority identified AB307-0294 vivo essential                               codes penicillin-binding protein 7/8 (PBP-7/8), low-molecular- <br /> genes unrecognized underexploited antimicrobial tar-                                  weight (LMW) PBP. The high-molecular-weight (HMW) PBPs <br /> gets. Bioinformatic selection putative antimicrobial drug tar-                            heavily exploited antibacterial targets, previous the <br /> gets typically requires gene ortholog anno-                            demonstration A. baumannii PBP-7/8 vivo essential <br /> tated essential (18, 40). The lack significant numbers                              (35), LMW PBPs recognized contributing bacterial <br /> orthologs annotated essential DEG identified                           survival largely neglected targets. E. coli LMW <br /> A. baumannii vivo essential genes suggested                             PBPs recently reported provide intrinsic resistance against <br /> genes selected possible drug targets                            &#226;?&#164;-lactams, underscoring previously unappreciated <br /> bioinformatic screen. In order determine orthologs                           roles proteins infection antimicrobial treat- <br /> these genes received attention antimicrobial                              ment (44). A gene, pryC, encoding dihydroorotase <br /> target, DrugBank Therapeutic Target Database                                    (DHOase), noteworthy human Plasmodium <br /> (TTD) searched (Table 4). This analysis revealed                           genes encoding function received attention potential <br /> the 18 A. baumannii genes identified study, orthologs                          anticancer antimalarial targets, respectively. However, the <br /> these genes, targets FDA-approved drugs drugs                              Plasmodium human enzymes belong different DHOase <br />  <br />  <br /> 6        &#194;&#174; <br />  <br />              mbio.asm.org                                                                                                       July/August 2012 Volume 3 Issue 4 e00113-12 <br />                                                                                                                                      A. baumannii In Vivo Essential Gene Screening <br />  <br />  <br />  <br /> TABLE 5 Ortholog distribution AB307-0294 vivo essential genes bacterial species annotated DEGa <br />  <br />  <br />  <br />  <br /> a Green indicates presence ortholog AB307-0294 vivo essential gene annotated essential bacterial species present DEG version 6.8. Yellow indicates <br /> the presence ortholog type (essential nonessential) AB307-0294 vivo essential gene. <br />  <br />  <br />  <br /> class majority bacteria, including exhib-                          1-carboxyvinyltransferase chorismate synthase, respectively, <br /> iting different active-site architectures.                                                   catalyze final steps chorismate biosynthesis. This <br />     Given identification 18 A. baumannii genes                            example screen identifying functionally related genes <br /> vivo essential, evaluated likelihood serving potential                        participate pathway contained within <br /> anti-GNB targets criteria commonly used bioinformatic                              operon. The A. baumannii aroA gene additionally en- <br /> selection targets. This analysis revealed iden-                          codes prephenate dehydrogenase (PD) activity, yielding un- <br /> tified A. baumannii genes possess significant potential thera-                            usual fusion protein. Typically, activities present as <br /> peutic targets GNB despite previously overlooked                                separate proteins bacteria (45). PD member aromatic <br /> (Table 4). Therefore, context identifying essential                            amino acid biosynthesis pathway, uses derivative cho- <br /> genes purpose developing new classes antimicrobials                            rismate initial substrate. The metabolic gene iden- <br /> for human infections, growth medium/environment used                                 tified aceE, putatively encoding pyruvate dehydrogenase E1 <br /> the essentiality screen critical.                                                         component. This protein participates conversion pyru- <br />                                                                                              vate acetyl coenzyme A (acetyl-CoA), feeds the <br /> DISCUSSION                                                                                   citric acid cycle metabolic pathways. Two genes partic- <br /> This study identified unique set A. baumannii genes repre-                              ipating separate TCS systems, envZ rstA, identified as <br /> senting high-value, unrecognized underexploited, potential                                vivo essential (Fig. 1A). The gene envZ putatively encodes the <br /> antimicrobial targets pathogenic GNB. Further, report                               osmolarity sensor protein, trans-inner membrane sensor pro- <br /> established gene DEG, excluded                          tein propagates signal cytoplasmic histidine ki- <br /> from consideration potential drug target. Lastly, data                              nase domain. This protein participates osmoregulation, re- <br /> demonstrated importance using clinically relevant media                               sponding environmental stimuli. The rstA gene putatively <br /> and vivo validation screening essential genes                            encodes response regulator protein, RstA, propagates the <br /> purpose developing new classes antimicrobials human                                signal corresponding sensor protein RstB acting a <br /> infections.                                                                                  transcriptional regulator. The environmental stimulus the <br />    The strategy used identifying vivo essential genes                          RstA/RstB TCS responds unknown. DNA/RNA syn- <br /> A. baumannii efficient, capable identifying                        thesis regulation forms group containing vivo <br /> genes involved multiple pathways functions enhances                               essential genes (Fig. 1B). This group subdivided tRNA <br /> the understanding pathogen biology. The putative products                              modification [trmD, putatively encodes tRNA (guanine- <br /> three vivo essential genes participate metabolic pathways                              1)-methyltransferase], ribosome assembly translation (rmpA, <br /> (Fig. 1A). The genes aroA aroC encode 3-phosphoshikimate                                 putatively encodes ribosomal protein L27), pyrimidine me- <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                                                      &#194;&#174; <br />  <br />                                                                                                                                                                      mbio.asm.org 7 <br />  Umland et al. <br />  <br />  <br />  <br /> tabolism (carA, putatively encodes carbamoyl-phosphate                uted differences growth environment used the <br /> synthase small chain; pyrC, encodes type II dihydro-                process essential gene identification. While conceptually this <br /> orotase), pppGpp synthesis signaling stringent re-              difference easy understand, fully appreciated <br /> sponse modulates transcription profiles decreases tRNA             interpreting results essential gene screens extrapolating <br /> and rRNA synthesis stress conditions (relA, puta-               results computationally select antimicrobial drug <br /> tively encodes GTP pyrophosphokinase). Both carbamoyl phos-                 targets predict essential genes species. This concept is <br /> phate synthase small subunit DHOase key enzymes              applicable identification virulence factors may <br /> pyrimidine biosynthesis pathway. Identification genes partici-           serve antimicrobial targets vaccine candidates. DEG a <br /> pating metabolic pathway distantly located           repository publically released genome-wide essentiality screens <br /> A. baumannii&#226;&#8364;&#8482;s genome increased confidence screening re-             serves common source bioinformatic methods requir- <br /> sults. One identified genes, secE, predicted protein           ing essentiality data. Overall, data contained DEG of <br /> transport function (Fig. 1B). It putatively encodes component          high quality considered context were <br /> the SecYEG translocon. This integral membrane protein                 generated. For bacterial species DEG, genome-wide screen- <br /> complex spanning inner membrane actively translocates              ing conducted vitro using laboratory media, one <br /> proteins periplasm. Seven identified vivo essential      used rich media. Genes annotated nonessential screens <br /> genes possess annotated functions predicted affect              excluded priori having essential role vivo. <br /> structural integrity: outer membrane (ompF, putatively            These screens designed replicate physiological condi- <br /> encodes outer membrane porin F; ostA, putatively encodes              tions encountered pathogen infection. All <br /> organic solvent tolerance protein), peptidoglycan synthesis             18 A. baumannii vivo essential genes identified study <br /> modification (uppP, putatively encodes undecaprenyl-                  carry annotation nonessential vitro conditions <br /> diphosphatase; pbpG, putatively encodes PBP-7/8), capsule             used nearly genome-wide screens published date. It is <br /> synthesis transport (ptk, putatively encodes tyrosine-            important growth conditions used identify essential <br /> protein kinase; epsA, putatively encodes EPS I polysaccha-            genes concordant project&#226;&#8364;&#8482;s goal. The human <br /> ride export outer membrane protein), lipopolysaccharide                 host environment deficient selected requisite nutrients and <br /> synthesis modification (spsC, putatively encodes pero-            possesses host defense factors. For example, certain purines, py- <br /> samine synthase) (Fig. 1C). Their structural role reinforced          rimidines, amino acids variably present various human <br /> the results quantitative growth studies mutants hu-          body fluids, including plasma (48, 49). Based solely concentra- <br /> man serum, demonstrating extreme sensitivity bactericidal            tion measurements, unclear bioavailability is <br /> activity complement (34, 35) (data shown). It notewor-           sufficient support growth corresponding auxotrophic <br /> thy pbpG identified screen (36). PBP-7/8 belongs         bacterial strain. However, numerous biological studies dem- <br /> the LMW class PBPs. LMW PBPs largely neglected                 onstrated pathogen&#226;&#8364;&#8482;s ability synthesize uptake certain <br /> because recognized essential using vitro assays,        noniron nutritional factors, purines, pyrimidines, and <br /> unlike HMW PBPs well-known targets &#226;?&#164;-lactam                 amino acids, critical virulence host (50). This <br /> antibiotics. These results strongly suggest mechanism           study supports concept human body fluids a <br /> action &#226;?&#164;-lactam antibiotics partially inactivation         modified minimal medium iron limitation. A number of <br /> of LMW PBPs.                                                                AB307-0294 mutants identified phenotype being <br />     The strategy identify vivo essential genes study          unable grow human ascites possessed gene disruptions in <br /> should applicable bacterial species cultured        pyrimidine amino acid metabolic pathways. Therefore, the <br /> and subjected gene disruption random mutagenesis. Critical            objective identify potential antimicrobial targets, impor- <br /> to strategy approach screening clinically relevant       tant essential genes identified clinically relevant me- <br /> medium, case human ascites, ex vivo medium              dium environment. Likewise, use genome-wide chemical <br /> reflective inflammatory extracellular fluid. This simple innova-         genetic screening rich laboratory media fail detect a <br /> tion critical efficiency approach. By selecting       significant percentage vivo essential genes. Findings this <br /> transposon mutants laboratory medium screening                  report demonstrate pool potential antimicrobial targets <br /> for essentiality ascites plates, able identify, step,   A. baumannii expanded refined based use <br /> genes (targets) likely essential expressed vivo.       clinically relevant <span id='am-402' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> gene essentiality. This <br /> The strength approach unbiased highly             strategy applicable human microbial pathogens. <br /> efficient. We select targets allow genetic screen,           The number bacterial species experimentally deter- <br /> designed identify phenotype vivo essentially, dictate       mined genome-wide essentiality small (i.e., 20 total screens 17 <br /> the choices. This approach ~50% success rate selecting            unique species DEG version 6.8), conducting screens <br /> genes later verified essential rat abscess infection model.         resource intensive. This set circumstances naturally led to <br /> Target validation appropriate animal model(s) strong             extrapolation data DEG predict essential genes <br /> predictor efficacy humans (46) critical step,          organisms. Cross-species prediction gene essentiality <br /> target essentiality vitro ex vivo translate     recognized difficult, ortholog conservation <br /> in vivo environment (47).                                                   multiple species necessarily marker essentiality <br />     The results study demonstrated set A. bau-       (11). Analysis A. baumannii results bacterial <br /> mannii genes initially identified ascites plates           entries DEG agreement difficulty inferring <br /> subsequently established vivo essential largely distinct        essentiality cross-species. The majority A. baumannii genes ver- <br /> from ortholog sets essential growth rich laboratory         ified vivo essential readily predicted high <br /> media. Broadly, difference essential gene sets attrib-       confidence level based comparison DEG (Fig. 2; Table 5). <br />  <br />  <br /> 8       &#194;&#174; <br />  <br />             mbio.asm.org                                                                               July/August 2012 Volume 3 Issue 4 e00113-12 <br />                                                                                                           A. baumannii In Vivo Essential Gene Screening <br />  <br />  <br />  <br /> Although differences screening methods likely account             studies (i.e., animal infection models) early process eval- <br /> significant variation A. baumannii vivo           uating potential antimicrobial targets. Neglecting may <br /> essential genes identified study essential genes cat-      significantly skewed past selection targets. Antimicrobial <br /> alogued DEG, shown genes identified             drug development general, especially GNB, chal- <br /> one species defined environment necessarily essential         lenging variety reasons. A successful antimicrobial drug <br /> in species genera conditions (8).                characteristics simple inhibition target&#226;&#8364;&#8482;s <br />     Conversely, experimentally established re-        function. Features include permeability properties enable tar- <br /> port, presence gene ortholog gene DEG         site access, resistance degradation efflux, favorable <br /> not guarantee gene essential vivo. Many          pharmacokinetic pharmacodynamic properties, accept- <br /> elements type II fatty acid biosynthesis (FASII) pathway            able toxicity. These necessary desired features are <br /> (i.e., fab gene cluster) number bacteria annotated       odds chemically, requiring multiple <br /> as essential, forming basis development antimicrobi-         rounds optimization (56). Expansion target space allows for <br /> als targeting pathway (51, 52). However, recently         exploration different regions chemical space, increasing <br /> demonstrated pathway essential GPB dur-            probability appropriate lead molecules identified <br /> ing infection. Specifically, inhibited genetically disrupted        developed new classes antimicrobials. <br /> GPB FASII pathway compensated fatty acids present                 The identification vivo essential genes increases un- <br /> serum (47, 53). Thus, vitro essentiality data obtained using rich       derstanding vivo physiology A. baumannii growth, <br /> laboratory medium failed accurately reflect genes required,         important initial step target-based drug develop- <br /> and required, promotion infection clinically rel-        ment strategies facilitate understanding mech- <br /> evant environments.                                                        anism action. A number high-throughput screening (HTS) <br />     Several limitations noted regarding strategy         campaigns, designed identify drug leads targeting essential <br /> identifying essential genes. First, does identify genes essen-      genes target-based approach, not, yet, come <br /> tial rich laboratory media. However, likely      fruition (46). However, important points this <br /> genes identified essential rich laboratory me-         study previous screens been <br /> dia encode proteins targeted present          optimally designed respect target selection. Retrospective <br /> antimicrobial armamentarium, involved cell              analyses failed target-based HTS campaigns identified <br /> wall protein synthesis. Second, genes identified vivo essen-      problems HTS design, especially use chemical librar- <br /> tial bacterial species (e.g., A. baumannii) certain      ies limited chemical diversity primarily designed eukaryotic <br /> be vivo essential genera species,        targets (33, 57). Multiple approaches used identi- <br /> problem present essentiality screening methods. Redundant           fication lead compounds effective specific targets, which <br /> or alternative metabolic biosynthetic pathways exist;               subsequently validated using combination appropriate <br /> therefore, conclusive proof essentiality requires experimental          bacterial constructs clinically relevant vitro vivo mod- <br /> confirmation appropriate conditions. Additional research             els. Structurally enabled fragment-based lead discovery (FBLD) <br /> is required determine level conservation essen-        methods differ significantly methodology philosophy from <br /> tiality phenotype species varies significantly genes        HTS. Importantly, FBLD allows efficient larger <br /> identified vitro using rich laboratory media identified       sampling chemical space simultaneously allowing greater <br /> in vivo. Lastly, A. baumannii genetic screen designed           freedom medicinal chemistry engineer desired properties <br /> examine essentiality extracellular inflammatory fluid. Bacterial        optimization initial drug lead compounds towards <br /> infections occur physiological environments ascites          therapeutics suitable clinical testing (58, 59). Although the <br /> and rat abscess model mimic poorly. An interesting future          concept antimicrobial active multiple indepen- <br /> research direction test results differ-      dent targets appealing, particularly durability, a <br /> ent infection models. It likely variation    necessary requirement effective antimicrobial. Several strat- <br /> observed vivo essentiality based site infection           egies compatible leveraging results drug de- <br /> and/or animal species used. That said, previously              velopment: single pharmacophore/multitarget compounds, hy- <br /> tested A. baumannii mutants rat pneumonia              brids pharmacophores, combinations single-target <br /> model, cases significant decreases bacterial titers     inhibitors avoid resistance development, structure-based <br /> observed (35). It possible bacterial genes     drug design create multiple intramolecular drug&#226;&#8364;&#8220;target inter- <br /> vivo essential rat vivo essential humans, vice   actions (60). Lastly, results allow (re)examination of <br /> versa, specific pathogen-host interactions. For example,            genes. For example, possible screening hits against <br /> Neisseria species obtain iron host transferrin lactoferrin        genes/gene products identified the <br /> proteins. Neisseria&#226;&#8364;&#8482;s receptors proteins evolved      past pursued targets previously con- <br /> specific proteins human closely related        sidered nonessential. <br /> primates (54, 55). Pathogen-host specificity recognized issue             In summary, study demonstrates underappreciated <br /> for animal model solely problem              concept decisions potential validity antimicrobial <br /> strategy.                                                                  targets investigators pharmaceutical companies need be <br />     The results presented expand antimicrobial target space           environmental context (e.g., rich laboratory me- <br /> and increase awareness environment infection               dium versus infection model) gene identified as <br /> may differ significantly laboratory vitro growth condi-            essential. The absence database catalogues clinically rel- <br /> tions. Furthermore, results emphasize need incorpo-           evant essential genes impeded fuller definition anti- <br /> rate appropriate vitro screens clinically relevant vivo          microbial target space. <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                        &#194;&#174; <br />  <br />                                                                                                                                        mbio.asm.org 9 <br />  Umland et al. <br />  <br />  <br />  <br /> MATERIALS AND METHODS                                                              successfully generated (data shown). For RT-PCR studies, RNA <br /> AB307-0294 clinical blood isolate representative A. baumannii.             isolated mutant strain using RNeasy Protect minikit <br /> A. baumannii strain 307-0294 (AB307-0294) (blood isolate Buffalo,             (Qiagen Inc., Valencia, CA). Samples treated DNase I twice to <br /> NY, sequence type [ST] 15, clonal group 1 based Ecker et al. [61])          completely digest contaminating chromosomal DNA. Absence con- <br /> selected model strain studies. We previously used       taminating DNA confirmed testing RNA samples prior RT- <br /> strain related studies (34&#226;&#8364;&#8220;36). The genome AB307-0294               PCR analysis using appropriate primer pair GoTaq Green master <br /> fully sequenced, containing 3.76 Mbp 3,531 predicted open reading              mix (Promega, Madison, WI). The Qiagen OneStep RT-PCR kit used <br /> frames (ORFs) (62). Compared genomes public do-               subsequent RT-PCR analyses samples. Primers designed to <br /> main (ATCC 17978, 0057, AYE, ACICU, SDF) additional iso-                 amplify 100 500 bp RNA transcript assessed. Primers are <br /> lates (900 [perineal isolate], 853 [blood isolate Iraq], 979 [envi-       listed Table S2 supplemental material. A positive control that <br /> ronmental isolate Iraq]) sequenced T. A. Russo, A. Campagnari,             amplified RNA transcript single-stranded binding protein <br /> and S. Gill (unpublished data), available time initiation         (SSB) run concurrently samples tested. <br /> these studies, AB307-0294 shared ~3 Mbp. AB307-0294 relatively drug                  Four AB307-0294 mutants complemented trans using <br /> susceptible, enabling performance genetic manipulations. It              A. baumannii cloning vector pNLAC1 restoration pheno- <br /> closely related A. baumannii strains 0057 AYE, approxi-           type able grow survive human ascites (34, 35) (data not <br /> mately 3% AB307-0294&#226;&#8364;&#8482;s genome differing strains, virtually           shown pyrC). For mutant derivative, polar effect excluded <br /> all prophage clusters. Taken together, data support         downstream gene multisubunit two- <br /> AB307-0294 clinically appropriate, representative con-            component regulatory system. In mutants, downstream <br /> temporary A. baumannii strains, ideal background identify new            gene reversed transcriptional orientation, excluding polar effects. <br /> or unrecognized antimicrobial targets interest.                                 Results summarized Table 1. <br />      Identification AB307-0294 genes essential growth human                   Assessment AB307-0294 mutant derivatives decreased <br /> ascites ex vivo. The experimental approach modification               growth human ascites rat subcutaneous abscess model. The Uni- <br /> method previously used identify virulence factors extraintestinal            versity Buffalo Veterans Administration Institutional Animal Care <br /> pathogenic E. coli (32) subsequently applied A. baumannii (34,          Committees approved rat subcutaneous abscess model. This study <br /> 35). Briefly, EZ-Tn5&#226;&#172;?kan-2&#226;&#172;&#381; transposome (EPICENTRE Biotech-                    carried strict accordance recommendations the <br /> nologies) electroporated AB307-0294 create random mutants.             Guide Care Use Laboratory Animals National Insti- <br /> Mutagenized bacteria selected MH plates containing kanamycin               tutes Health, efforts minimize suffering. Mutants <br /> (40 &#226;?&#174;g/ml), nutrient-rich laboratory medium. Mutants grid-             exhibiting significantly decreased absent growth human ascites were <br /> ded plates comprised solely human exudative ascites, kanamy-             evaluated established rat soft-tissue infection model vivo <br /> cin (40 &#226;?&#174;g/ml), agar (ascites plates). Human ascites collected             growth/survival (36). Briefly, subcutaneous space created the <br /> from deidentified patients undergoing therapeutic paracentesis            injection 30 ml air anesthetized Long-Evans rats (200 <br /> for symptoms abdominal distension. These individuals               225 g), followed injection 1 ml 1% croton oil filter- <br /> being treated antimicrobials infected human im-             sterilized vegetable oil vehicle. The space allowed mature an <br /> munodeficiency, hepatitis B, hepatitis C virus. The ascites cultured        encapsulated, fluid-filled (8- 12-ml) pouch 6 8 days. Neutro- <br /> to confirm sterility, divided aliquots, stored &#226;&#171;&#186;80&#194;&#176;C. The pro-         phils migrated abscess response appropriate <br /> cedures obtaining human ascites reviewed approved the <br />                                                                                    chemotactic signals. The abscess&#226;&#8364;&#8482; subcutaneous location enabled multiple <br /> Western New York Veterans Administration Institutional Review Board. <br />                                                                                    injections samplings performed time. AB307-0294 mu- <br /> Informed consent used obtain human blood preparation of <br />                                                                                    tant derivatives injected abscess anesthetized <br /> serum (approval ID 00063). The Western New York Veterans Adminis- <br />                                                                                    animal, resulting estimated starting abscess concentration 1 &#226;&#171;&#187; <br /> tration Institutional Review Board process obtaining ascites <br />                                                                                    105 CFU/ml. Within 1 min bacteria injected abscess, <br /> waived informed consent (approval ID 00098). An expedited review was <br />                                                                                    0.5 ml abscess fluid removed measure actual starting bacterial <br /> performed, ascites collected deidentified patients <br />                                                                                    titer. Fluid aliquots (0.5 ml) subsequently obtained anesthe- <br /> who undergoing therapeutic paracentesis symptoms ab- <br />                                                                                    tized animals 3, 6, 24, 48 h initial bacterial challenge, and <br /> dominal distension. Mutants exhibiting minimal growth ascites <br /> plates sites transposon insertion identified chromosomal           bacterial titers enumerated. In vivo data presented means &#226;&#171;&#190; <br /> sequencing using previously described methods (35), inter-                standard errors means (SEM). To normalize vivo data, log10 <br /> rupted ORF identified comparison annotated AB307-0294                transformed values utilized, area curve calcu- <br /> genomic sequence (GenBank no. CP001172). Note saturation mu-                  lated, areas compared using two-tailed unpaired t tests <br /> tagenesis achieved. Depending method estimation,             (Prism 4 Macintosh, GraphPad Software Inc.). <br /> predicted number mutants required reach 95% saturation                     Prioritization genes identified vitro ascites screen vivo <br /> AB307-0294 genome 12,000 42,000 (63). Further,                         evaluation. Genes analyzed conservation additional <br /> transposon-mediated mutagenesis likely interrupt longer                 A. baumannii strains possessing sequenced genomes (AB0057, ACICU, <br /> genes, data set biased short genes. However,             ATCC 17978, AYE, SDF) initiation study lack of <br /> given objective evaluating growth phenotype ascites plates        close human homologues using BLAST (64). The identified genes were <br /> predictive vivo essentiality identifying vivo essential   conserved A. baumannii strains exhibited wide range sim- <br /> genes, lack saturation mutagenesis acceptable.                         ilarity human genes. This filtering process performed reduce the <br />      Quantitative vitro growth analyses. Mutant derivatives AB307-           number mutant strains animals used, current study focus- <br /> 0294 demonstrated decreased absent growth ascites plates un-            ing potential targets little homology human genes. <br /> derwent quantitative growth assessment rich laboratory medium (LB)                   Comparison A. baumannii vivo essential genes DEG. Puta- <br /> and human ascites described (35, 36).                                           tive protein sequences generated DNA sequences 18 <br />      Exclusion polar effects mutant derivatives AB307-0294. Po-           AB307-0294 genes experimentally established essential vivo. These <br /> tential transposon-mediated polar effects downstream genes                 proteins compared bacterial genome-wide essentiality screen data <br /> evaluated. For 10 mutants, reverse transcription-PCR (RT-PCR)               available Database Essential Genes, version 6.8 (14), BLASTp <br /> was performed gene immediately downstream              (64). Significant DEG BLASTp hits (E values &#226;&#172;?1e&#226;&#171;&#186;10 coverage of <br /> transcriptional orientation, amplicon expected size              &#226;&#172;&#381;75% AB307-0294 protein) evaluated using OMA <br />  <br />  <br /> 10       &#194;&#174; <br />  <br />              mbio.asm.org                                                                                        July/August 2012 Volume 3 Issue 4 e00113-12 <br />                                                                                                                          A. baumannii In Vivo Essential Gene Screening <br />  <br />  <br />  <br /> Browser determine orthologs (i.e., OMA ortholog close ortholog                    mutant library use identifying motility genes Vibrio cholerae. <br /> groups) A. baumannii targets (65).                                                    Proc. Natl. Acad. Sci. U. S. A. 105:8736 &#226;&#8364;&#8220; 8741. <br />     Comparison known drug targets. Similarly, drug target data-             16.   Gallagher LA, et al. 2007. A comprehensive transposon mutant library of <br /> bases (Therapeutic Targets Database [TTD] DrugBank) (66, 67)                    Francisella novicida, bioweapon surrogate. Proc. Natl. Acad. Sci. U. S. A. <br /> searched using BLASTp proteins similar A. baumannii targets,                  104:1009 &#226;&#8364;&#8220;1014. <br />                                                                                    17.   Langridge GC, et al. 2009. Simultaneous assay Salmonella typhi <br /> significant matches confirmed orthologs OMA browser. <br />                                                                                          gene using million transposon mutants. Genome Res. 19:2308 &#226;&#8364;&#8220;2316. <br /> The putative protein products derived A. baumannii genes ex-              18.   Crowther GJ, et al. 2010. Identification attractive drug targets in <br /> perimentally established vivo essential evaluated             neglected-disease pathogens using silico approach. PLoS Negl. Trop. <br /> potential antimicrobial targets. Criteria similar concept            Dis. 4:e804. <br /> published methods (18, 40) modified GNB. In addition absolute           19.   Payne DJ, Gwynn MN, Holmes DJ, Rosenberg M. 2004. Genomic <br /> criterion genes verified vivo essential A. baumannii,             approaches antibacterial discovery. Methods Mol. Biol. 266:231&#226;&#8364;&#8220;259. <br /> other factors considered included conservation A. baumannii                 20.   Shen Y, et al. 2010. Blueprint antimicrobial hit discovery targeting <br /> strains multiple clinically important GNB, knowledge biolog-                metabolic networks. Proc. Natl. Acad. Sci. U. S. A. 107:1082&#226;&#8364;&#8220;1087. <br /> ical role aid understanding inhibitor/drug lead mechanisms action,         21.   Boucher HW, et al. 2009. Bad bugs, drugs: ESKAPE! An update <br /> defined druggable site protein, availability activity assay,            infectious Diseases Society America. Clin. Infect. Dis. 48:1&#226;&#8364;&#8220;12. <br /> the feasibility biochemical biophysical experimental study,         22.   Garnacho-Montero J, Amaya-Villar R. 2010. Multiresistant Acinetobac- <br />                                                                                          ter baumannii infections: epidemiology management. Curr. Opin. <br /> lack closely related proteins humans. <br />                                                                                          Infect. Dis. 23:332&#226;&#8364;&#8220;339. <br />                                                                                    23.   Manchanda V, Sanchaita S, Singh N. 2010. Multidrug resistant Acineto- <br /> SUPPLEMENTAL MATERIAL                                                                    bacter. J. Global Infect. Dis. 2:291&#226;&#8364;&#8220;304. <br /> Supplemental material article http://mbio.asm.org         24.   Kim BN, et al. 2009. Management meningitis antibiotic- <br /> /lookup/suppl/doi:10.1128/mBio.00113-12/-/DCSupplemental.                                resistant Acinetobacter species. Lancet Infect. Dis. 9:245&#226;&#8364;&#8220;255. <br />     Figure S1, TIF file, 2.5 MB.                                                   25.   Snitkin ES, et al. 2011. Genome-wide recombination drives diversifica- <br />     Table S1, PDF file, 0.1 MB.                                                          tion epidemic strains Acinetobacter baumannii. Proc. Natl. Acad. Sci. <br />                                                                                          U. S. A. 108:13758 &#226;&#8364;&#8220;13763. <br />     Table S2, PDF file, 0.1 MB. <br />                                                                                    26.   Towner KJ. 2009. Acinetobacter: old friend, new enemy. J. Hosp. <br />                                                                                          Infect. 73:355&#226;&#8364;&#8220;363. <br /> ACKNOWLEDGMENTS                                                                    27.   Sengstock DM, et al. 2010. Multidrug-resistant Acinetobacter baumannii: <br /> This work support Telemedicine Advance Technical                    emerging pathogen older adults community hospitals and <br />                                                                                          nursing homes. Clin. Infect. Dis. 50:1611&#226;&#8364;&#8220;1616. <br /> Research Center (TATRC) cooperative agreement W23RYX1055N607, <br />                                                                                    28.   Perez F, et al. 2007. Global challenge multidrug-resistant Acinetobacter <br /> an interdisciplinary grant University Buffalo (L.W.S., T.A.R.,               baumannii. Antimicrob. Agents Chemother. 51:3471&#226;&#8364;&#8220;3484. <br /> and T.C.U.), VA Merit Review grant Department Vet-               29.   Fishbain J, Peleg AY. 2010. Treatment Acinetobacter infections. Clin. <br /> erans Affairs (T.A.R.).                                                                  Infect. Dis. 51:79 &#226;&#8364;&#8220; 84. <br />                                                                                    30.   Boucher H, et al. 2011. 10 &#226;&#171;&#187; &#226;&#8364;&#8482;20 progress: development new drugs <br /> REFERENCES                                                                               active resistant 687 Gram-negative bacilli, abstr LB-27. Abstr. 49th <br />                                                                                          Ann. Meet. Infect. Disease Soc. Am. IDSA, Boston, MA. <br />  1. D&#226;&#8364;&#8482;Elia MA, Pereira MP, Brown ED. 2009. Are essential genes really <br />                                                                                    31.   Butler MS, Cooper MA. 2011. Antibiotics clinical pipeline 2011. <br />     essential? Trends Microbiol. 17:433&#226;&#8364;&#8220; 438. <br />                                                                                          J. Antibiot. 64:413&#226;&#8364;&#8220; 425. <br />  2. Dowell RD, et al. 2010. Genotype phenotype: complex problem. <br />                                                                                    32.   Russo TA, Carlino UB, Mong A, Jodush ST. 1999. Identification genes <br />     Science 328:469. <br />                                                                                          extraintestinal isolate Escherichia coli increased expression <br />  3. Grenov AI, Gerdes SY. 2008. Modeling competitive outgrowth mutant <br />                                                                                          exposure human urine. Infect. Immun. 67:5306 &#226;&#8364;&#8220;5314. <br />     populations: essentiality screens yield divergent results? Methods <br />                                                                                    33.   Gwynn MN, Portnoy A, Rittenhouse SF, Payne DJ. 2010. Challenges of <br />     Mol. Biol. 416:361&#226;&#8364;&#8220;367. <br />  4. Osterman AL, Gerdes SY. 2008. Comparative approach analysis                    antibacterial discovery revisited. Ann. N. Y. Acad. Sci. 1213:5&#226;&#8364;&#8220;19. <br />     gene essentiality. Methods Mol. Biol. 416:459 &#226;&#8364;&#8220; 466.                           34.   Russo TA, et al. 2010. The K1 capsular polysaccharide Acinetobacter <br />  5. Baba T, et al. 2006. Construction Escherichia coli K-12 in-frame, single-         baumannii strain 307-0294 major virulence factor. Infect. Immun. <br />     gene <span id='am-244' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-245' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio collection. Mol. Syst. Biol. 2:2006.0008.            78:3993&#226;&#8364;&#8220; 4000. <br />  6. Gerdes SY, et al. 2003. Experimental determination level            35.   Russo TA, et al. 2009. Penicillin-binding protein 7/8 contributes the <br />     analysis essential genes Escherichia coli MG1655. J. Bacteriol. 185:           survival Acinetobacter baumannii vitro vivo. J. Infect. Dis. <br />     5673&#226;&#8364;&#8220;5684.                                                                           199:513&#226;&#8364;&#8220;521. <br />  7. Rudd KE. 2000. EcoGene: genome sequence database Escherichia coli        36.   Russo TA, et al. 2008. Rat pneumonia soft-tissue infection models for <br />     K-12. Nucleic Acids Res. 28:60 &#226;&#8364;&#8220; 64.                                                 study Acinetobacter baumannii biology. Infect. Immun. 76: <br />  8. Salama NR, Shepherd B, Falkow S. 2004. Global transposon mutagenesis                 3577&#226;&#8364;&#8220;3586. <br />     essential gene analysis Helicobacter pylori. J. Bacteriol. 186:         37.   Altenhoff AM, Dessimoz C. 2009. Phylogenetic functional assess- <br />     7926 &#226;&#8364;&#8220;7935.                                                                          ment orthologs inference projects methods. PLoS Comput. Biol. <br />  9. Dammel CS, Noller HF. 1995. Suppression cold-sensitive mutation                 5:e1000262. <br />     16S rRNA overexpression novel ribosome-binding factor, RbfA.        38.   Fitch WM. 1970. Distinguishing homologous analogous proteins. <br />     Genes Dev. 9:626 &#226;&#8364;&#8220; 637.                                                              Syst. Zool. 19:99 &#226;&#8364;&#8220;113. <br /> 10. Joyce AR, et al. 2006. Experimental computational assessment            39.   Koonin EV. 2005. Orthologs, paralogs, evolutionary genomics. <br />     conditionally essential genes Escherichia coli. J. Bacteriol. 188:                Annu. Rev. Genet. 39:309 &#226;&#8364;&#8220;338. <br />     8259 &#226;&#8364;&#8220; 8271.                                                                   40.   Ag&#195;&#188;ero F, et al. 2008. Genomic-scale prioritization drug targets: the <br /> 11. Zalacain M, et al. 2003. A global approach identify novel broad-                  TDR targets database. Nat. Rev. Drug Discov. 7:900 &#226;&#8364;&#8220;907. <br />     spectrum antibacterial targets proteins unknown function. J.          41.   Doyle MA, Gasser RB, Woodcroft BJ, Hall RS, Ralph SA. 2010. Drug <br />     Mol. Microbiol. Biotechnol. 6:109 &#226;&#8364;&#8220;126.                                              target prediction prioritization: using orthology predict essentiality <br /> 12. Hillenmeyer ME, et al. 2008. The chemical genomic portrait yeast:                 parasite genomes. BMC Genomics 11:222. <br />     uncovering phenotype genes. Science 320:362&#226;&#8364;&#8220;365.                     42.   Berardinis V, et al. 2008. A complete collection single-gene deletion <br /> 13. Nichols RJ, et al. 2011. Phenotypic landscape bacterial cell. Cell              mutants Acinetobacter baylyi ADP1. Mol. Syst. Biol. 4:174. <br />     144:143&#226;&#8364;&#8220;156.                                                                   43.   Sebkova A, Karasova D, Crhanova M, Budinska E, Rychlik I. 2008. aro <br /> 14. Zhang R, Lin Y. 2009. DEG 5.0, database essential genes                 mutations Salmonella enterica cause defects cell wall outer mem- <br />     prokaryotes eukaryotes. Nucleic Acids Res. 37:D455&#226;&#8364;&#8220;D458.                         brane integrity. J. Bacteriol. 190:3155&#226;&#8364;&#8220;3160. <br /> 15. Cameron DE, Urbach JM, Mekalanos JJ. 2008. A defined transposon                44.   Sarkar SK, Dutta M, Chowdhury C, Kumar A, Ghosh AS. 2011. PBP5, <br />  <br />  <br /> July/August 2012 Volume 3 Issue 4 e00113-12                                                                                                      &#194;&#174; <br />  <br />                                                                                                                                                      mbio.asm.org 11 <br />  Umland et al. <br />  <br />  <br />  <br />       PBP6 DacD play different roles intrinsic beta-lactam resistance       56. Hughes D. 20 <br /> </body></html>