<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                  Letters <br />  <br />                                                                                                                     pubs.acs.org/acschemicalbiology <br />  <br />  <br />  <br />  <br /> Release Factor One Is Nonessential Escherichia coli <br /> David B. F. Johnson,&#226;&#8364;&#160;,&#226;&#352;&#165; Chong Wang,&#226;&#8364;&#160;,&#226;&#352;&#165; Jianfeng Xu,&#226;&#8364;&#160; Matthew D. Schultz,&#226;&#8364;&#161;,&#194;&#167; Robert J. Schmitz,&#226;&#8364;&#161; <br /> Joseph R. Ecker,&#226;&#8364;&#161;,&#226;&#710;&#165; Lei Wang*,&#226;&#8364;&#160; <br /> &#226;&#8364;&#160; <br />  The Jack H. Skirball Center Chemical Biology Proteomics &#226;&#8364;&#161;Genomic Analysis Laboratory, The Salk Institute for <br />  Biological Studies, 10010 N. Torrey Pines Road, La Jolla, California 92037, United States <br /> &#194;&#167; <br />  Bioinformatics Program, University California San Diego, La Jolla, California 92093, United States <br /> &#226;&#710;&#165; <br />   Howard Hughes Medical Institute, 4000 Jones Bridge Road, Chevy Chase, Maryland 20815, United States <br />     * <br />     S Supporting Information <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>: Recoding stop codon amino acid may <br />     a&#239;&#172;&#8364;ord orthogonal genetic systems biosynthesizing new <br />     protein organism properties. Although reassignment of <br />     stop codons extant organisms, model <br />     organism lacking investigate reassignment process and <br />     direct <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> evolution. Complete reassignment stop <br />     codon precluded release factors (RFs), recognize <br />     stop codons terminate translation. Here discovered that <br />     RF1 unconditionally knocked various <br />     Escherichia coli stains, demonstrating reportedly <br />     essential RF1 generally dispensable E. coli species. <br />     The apparent essentiality RF1 caused by <br />     ine&#239;&#172;&#402;ciency mutant RF2 terminating UAA stop codons; wild type RF2 su&#239;&#172;&#402;cient RF1 <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span>. The RF1- <br />     knockout strains autonomous unambiguously reassigned UAG encode natural unnatural amino acids (Uaas) at <br />     multiple sites, a&#239;&#172;&#8364;ording previously unavailable model studying <span id='am-397' about='Thesaurus:code' typeof='owl:Thing'>code</span> evolution unique host exploiting Uaas <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />     evolve new biological functions. <br />  <br />  <br />  <br /> A     lthough canonical genetic code preserved almost <br />       organisms, small deviations discovered, <br /> including reassignment sense codons amino <br />                                                                            ambiguity. An attractive route reassign UAG <br />                                                                            stop codon Uaa bacteria. Orthogonal tRNA/synthetase <br />                                                                            pairs engineered incorporate Uaas proteins in <br /> acid reassignment stop sense                response UAG,8&#226;&#710;&#8217;12 presence RF1 makes the <br /> codons.1,2 Stop codons decoded class I release factors              meaning UAG ambiguous, stop signal Uaa <br /> (RFs).3 Whereas eukaryotes archaea use single RF                  simultaneously. RF1 competition limits incorporation of <br /> recognize stop codons,4,5 bacteria use two: RF1               Uaas single UAG site low e&#239;&#172;&#402;ciency; addition of <br /> speci&#239;&#172;?c UAA/UAG, RF2 speci&#239;&#172;?c UAA/UGA.6 It                   second UAG codon decreases protein yields <br /> is unknown class I RFs bacteria               precipitously.13 Although Uaas incorporated more <br /> single class I RF su&#239;&#172;&#402;cient organisms             site protein,14,15 low UAG-encoding <br /> domains. The process stop codon reassignment                   e&#239;&#172;&#402;ciency prevents e&#239;&#172;&#8364;ective use Uaas multiple sites to <br /> potential association RF evolution unclear.                  explore novel protein organismal properties through <br /> Natural code evolution occurs millions years. Extant               directed evolution. In addition, ambiguity UAG <br /> organisms harboring altered genetic codes end-point             codon hinder eventual &#239;&#172;?xation altered genetic <br /> of code evolution. There records initial causes          code organism, protein products truncated the <br /> of responses altered genetic codes;                     UAG sites interfere normal protein functions have <br /> adaptations details eventual &#239;&#172;?xation completely              detrimental e&#239;&#172;&#8364;ects host cell, preventing advanta- <br /> unknown. To enable in-depth investigation code change               geous coding inherited selected directed <br /> any concurrent cellular adaptations real time, necessary          evolution. To exclude Uaa incorporation legitimate <br /> to generate model organism able undergo such <br />                                                                            termination sites speci&#239;&#172;?ed UAG, endogenous UAG codons <br /> evolutionary processes laboratory. <br />                                                                            E. coli genome replaced synonymous UAA <br />    Synthetically recoding genome a&#239;&#172;&#8364;ord new properties <br />                                                                            stop codon genome engineering.16 However, for <br /> to organism encoding unnatural amino acids <br /> (Uaas) preventing cross-contamination wild type life              Received: May 9, 2012 <br /> forms.7 For successful genome recoding, target codon              Accepted: June 4, 2012 <br /> be reassigned new meaning high e&#239;&#172;&#402;ciency                      Published: June 5, 2012 <br />  <br />                              &#194;&#169; 2012 American Chemical Society       1337                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                 Letters <br />  <br /> complete reassignment UAG sense codon, necessary <br /> and critical step knock RF1 E. coli genome. <br />    In eukaryotes ciliates green algae, the <br /> reassignment stop codon sense codon accompanied <br /> by convergent changes eRF1.2 For instance, eRF1 of <br /> Tetrahymena restricts recognition UGA, UAA/UAG <br /> are reassigned Gln; eRF1 Euplotes recognizes only <br /> UAA/UAG stop codons, UGA used encode <br /> Cys.17,18 In bacteria, Mycoplasma species lost RF2 <br /> gene, UGA codon encodes Trp instead.19 However, <br /> Mycoplasma obligatory pathogens highly reduced <br /> genomes. To date, free-living bacterium found <br /> lacking RF1 RF2.19 For E. coli, RF1 reported <br /> to essential,20 conditionally lethal knockouts have <br /> been described.21,22 Recently managed knock RF1 <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> special E. coli strain reduced genome a <br /> mutated RF2 gene. 13 Herein discovered the <br /> dispensability RF1 general property wild type E. <br /> coli, arguing paradigm RF1 essential. We <br /> revealed underlying mechanism preventing RF1 <br /> knockout, generated autonomous RF1 deletion strains <br /> valuable studying code evolution genome recoding. <br />    We attempted replace RF1-encoding gene, prfA, with <br /> the chloramphenicol acetyltransferase gene variety E. coli <br /> strains (Figure 1) using established &#206;&#187; red recombinase- <br /> based homologous recombination method.23 E. coli K-12 B <br /> are progenitors E. coli strains are <br /> derived.24 Reported attempts knockout RF1 used only <br /> E. coli K-12 strains unsuccessful.20,25 We initially <br /> attempted delete RF1 common K-12 strains                    Figure 1. RF1 knocked E. coli strains containing wild <br /> MG1655, DH10&#206;&#178;, HT115. Knockout RF1 assayed                       type RF2. (A) The RF1-encoding gene prfA replaced the <br /> by genomic PCR amplifying prfA locus (Figure 1A).                       chloramphenicol acetyltransferase (cat) gene homologous <br /> Consistent previous reports, knock RF1                recombination using phage &#206;&#187; red recombinase expressed with <br /> in K-12 strains (Figure 1B).                                   plasmid pKD46.23 Chloramphenicol-resistant colonies screened <br />                                                                             using PCR cat replacement con&#239;&#172;?rmed sequencing. The <br />    We attempted RF1 knockout K-12                   agarose gels PCR results parental knockout strains as <br /> strains contain alterations related translational                   labeled. (B) A list E. coli strains tested RF1 knockout. K-12 and <br /> termination. The BP5&#206;&#177; strain harbors glutaminyl amber                     B strains widely used E. coli subjects of <br /> suppressor tRNA, making UAG coding ambiguous                        classical experiments. Among genomic di&#239;&#172;&#8364;erences K-12 <br /> either stop signal Gln. This strain test stop codon           B strains,24 A246T RF2 peculiar <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation</span> K-12 strains <br /> ambiguity factor prime RF1 removal. The MDS42 strain                reduces release activity RF2 stop codon UAA. This <br /> has nearly 700 nonessential genes deleted26 used                 mutation reverted wild type Ala DH10&#206;&#178;f. The glnV44 gene <br /> assess reduced genome size termination load                  encodes glutaminyl amber suppressor tRNA. RF1 successfully <br />                                                                             knocked B strains DH10&#206;&#178;f strain. <br /> assist RF1 removal. However, RF1 knockouts were <br /> generated strains either, suggesting amber <br /> suppression minimal genome insu&#239;&#172;&#402;cient RF1                     results indicate RF1 essential E. coli, contrast to <br /> removal K-12 derivatives.                                                previous conclusions.20,25 <br />    All K-12 strains contain peculiar A246T mutation                   Full genomic sequencing performed RF1 knockout <br /> RF2-encoding gene prf B, lowering release activity RF2               strains JX1.0, CW1.0, CW2.0 compared their <br /> for UAA 5-fold.27,28 The UAA codon accounts                         respective parental strains.30,31 The RF1 deletion veri&#239;&#172;?ed in <br /> termination &#226;&#710;&#188;64% E. coli genes.29 We reasoned                cases. For CW1.0 CW2.0, mutations were <br /> A246T mutation severely impair ability RF2                  genome. For JX1.0, seven additional <br /> recognize UAA stop codons removal RF1                  single nucleotide polymorphisms (SNPs) (Support- <br /> prevent RF1 knockout. We tested common                  ing Table S1). Six SNPs silent mutations two <br /> E. coli B strains B strains encode wild type RF2               genes phage origin. None SNPs correspond known <br /> contains Ala246.28 Indeed, RF1 knockout successful                 mutations complement RF1 de&#239;&#172;?ciency.32&#226;&#710;&#8217;35 These <br /> all B strains tested; REL606, BL21, BL21(DE3)                results indicate RF1 knocked parental E. <br /> used generate knockout strains CW1.0, CW2.0,                     coli strains incurring compensatory mutations. <br /> JX1.0, respectively (Figure 1A). In addition, determine              What happen UAG site protein <br /> A246T mutation RF2 prevents RF1 removal K-12 strains,                 translation absence RF1? We mutated and <br /> we reverted A246T mutation Ala K-12 strain                    tyrosine codons TAG enhanced green <br /> DH10&#206;&#178; generate DH10&#206;&#178;f strain. This strain                       &#239;&#172;&#8218;uorescent protein (EGFP) gene36 generate 1-TAG <br /> permitted direct knockout RF1 (Figure 1B). These                     3-TAG EGFP reporters, respectively, tested their <br />                                                                      1338                       dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                  Letters <br />  <br /> expression BL21(DE3) RF1 knockout strain JX1.0                      anticodons tRNAs single <span id='am-396' about='xsp:base' typeof='owl:Thing'>base</span> <br /> (Figure 2A). As expected, BL21(DE3) cells expressing                    mispairing UAG, tRNAs weakly misread the <br />                                                                                UAG codon.37 Misreading UAG tRNAs not <br />                                                                                obvious presence RF1 BL21(DE3) became <br />                                                                                marked JX1.0 RF1 removal. <br />                                                                                   The unconditional knockout RF1 opens possibility <br />                                                                                reassigning meaning UAG codon completely, so <br />                                                                                UAG codon does ambiguously encode stop signal <br />                                                                                amino acid simultaneously. We attempted recruit <br />                                                                                UAG amino acid JX1.0, mimic possible code <br />                                                                                evolution pathway implicated RF constriction stop codon <br />                                                                                recognition Tetrahymena, Euplotes, eukaryotes.17,18 <br />                                                                                An orthogonal tRNA/synthetase pair, tRNATyr     CUA/LW1RS <br />                                                                                pair, introduced JX1.0. This pair does crosstalk <br />                                                                                endogenous E. coli tRNA/synthetase pairs functionally <br />                                                                                couples E. coli&#226;&#8364;&#8482;s protein translational machinery.8,38 The <br />                                                                                tRNATyr CUA decodes UAG codon speci&#239;&#172;?cally its <br />                                                                                anticodon CUA, LW1RS engineered charge the <br />                                                                                                                                           39 <br />                                                                                tRNATyr CUA Uaa p-acetyl-L-phenylalanine (pActF).    An <br />                                                                                EGFP gene containing 1-, 2-, 3-, 10-TAG codons co- <br />                                                                                expressed tRNATyr CUA/LW1RS single plasmid pAIO- <br />                                                                                EGFP(n-TAG) (Figure 3A). <br />                                                                                   EGFP expression assayed Western blotting (Figure <br />                                                                                3B) in-cell &#239;&#172;&#8218;uorescence (Figure 3C). When pActF not <br />                                                                                added growth media, BL21(DE3) cells expressed small <br />                                                                                full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> EGFP 1-TAG reporter only, <br />                                                                                suggesting tRNATyr  CUA/LW1RS pair incorporates a <br />                                                                                natural amino acid low e&#239;&#172;&#402;ciency absence the <br />                                                                                cognate pActF. No full-length EGFP detected 2-, 3-, <br />                                                                                10-TAG reporters. In contrast, JX1.0 expressed full-length <br /> Figure 2. UAG codon RF1 knockout strain JX1.0 suppressed             EGFP 1-, 2-, 3-, 10-TAG mutants, the <br /> by endogenous tRNAs mispairing anticodons. (A) An EGFP gene               e&#239;&#172;&#402;ciency decreased number UAG codons. We then <br /> with TAG codon permissive sites driven arabinose             puri&#239;&#172;?ed EGFP protein JX1.0 expressing pAIO- <br /> promoter plasmid pBAD-EGFP(n-TAG) test EGFP                          EGFP(1-TAG) absence pActF (Figure 3D) and <br /> expression. (B) In-cell &#239;&#172;&#8218;uorescence intensity measured                    analyzed mass spectrometry. Consistently, Tyr, Gln, and <br /> &#239;&#172;&#8218;uorometry BL21(DE3) JX1.0 cells transformed                      Trp UAG site observed Figure 2C, <br /> pBAD-EGFP(1-TAG) pBAD-EGFP(3-TAG). Samples                             con&#239;&#172;?rming misreading endogenous tRNAs JX1.0. <br /> normalized cell numbers. (C) Fourier transform mass spectro- <br />                                                                                   When pActF supplied growth medium, BL21- <br /> metric analysis EGFP protein puri&#239;&#172;?ed JX1.0 expressing pBAD- <br /> EGFP(1-TAG). The spectrum shows intact precursor ions (charge              (DE3) cells showed e&#239;&#172;&#402;cient expression 1-TAG EGFP <br /> deconvoluted) tryptic fragment EGFP (HNIEDGSVQ-                     mutant, EGFP expression decreased precipitously the <br /> LADHXQQNTPIGDGPVLLPDNHYLSTQSALSK, X representing                               addition UAG codon competition RF1 <br /> the UAG encoded site). Monoisotopic masses measured high                  termination. The use 3 UAG codons EGFP virtually <br /> accuracy indicate Gln, Tyr, Trp (expected [M + H]+ =                  abolished protein expression, protein detected <br /> 4437.17, 4472.17, 4495.19 Da; measured 4437.16, 4472.16,               10-TAG mutant. In stark contrast, RF1-deletion <br /> 4494.16 Da, respectively) incorporated UAG site. No          strain JX1.0 showed high expression EGFP mutants in <br /> amino acids detected UAG site.                                     presence pActF, indicated Western (Figure 3B) and <br />                                                                                in-cell &#239;&#172;&#8218;uorescence (Figure 3C). EGFP proteins puri&#239;&#172;?ed <br /> the 1- 3-TAG reporter showed EGFP &#239;&#172;&#8218;uorescence,                           yields 8.5 (&#194;&#177;0.4), 7.1 (&#194;&#177;0.4), 9.7 (&#194;&#177;0.3), 1.2 <br /> indicating RF1 terminates EGFP translation UAG                         (&#194;&#177;0.1) mg/L 1-, 2-, 3-, 10-TAG EGFP samples, <br /> normally. Surprisingly, JX1.0 showed high level                           respectively (Figure 3D). There decrease in <br /> &#239;&#172;&#8218;uorescence 1-TAG EGFP mutant small                         incorporation e&#239;&#172;&#402;ciency UAG codon increased <br /> reproducible level &#239;&#172;&#8218;uorescence 3-TAG EGFP                          1 2 3, indicating UAG changed sense <br /> mutant (Figure 2B). These data suggest endogenous                    codon JX1.0. The drop o&#239;&#172;&#8364; yield 10-TAG EGFP <br /> tRNAs suppress UAG codon JX1.0.                                     sample likely 10 pActF interferes EGFP folding <br />    To reveal identity tRNAs, puri&#239;&#172;?ed EGFP                      and/or stability, corroborated lack of <br /> protein expressed 1-TAG reporter JX1.0                         &#239;&#172;&#8218;uorescence 10-pActF EGFP (Figure 3C). <br /> identi&#239;&#172;?ed amino acid incorporated UAG site using                       Although JX1.0 RF1 terminate protein translation <br /> mass spectrometry (Figure 2C). Tyr, Gln, Trp                    UAG codon, small protein detected <br /> at UAG site Fourier transform mass spectrometric                   Western blot detected mass <br /> analysis tryptic fragments EGFP. To ensure              spectrometry truncated UAG site <br /> not suppressor mutation endogenous tRNAs,                        (Figure 3B). We think decoding UAG codon as <br /> chromosomal tRNATyr, tRNAGln, tRNATrp rese-                           Uaa pActF e&#239;&#172;&#402;cient decoding of <br /> quenced, con&#239;&#172;?rmed wild type JX1.0. The                   canonical sense codons natural amino acids. First, pActF- <br />                                                                         1339                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                 Letters <br />  <br />  <br />  <br />  <br /> Figure 3. UAG-decoding orthogonal tRNA/synthetase pair reassigns UAG codon amino acid JX1.0. (A) The plasmid pAIO-EGFP(n- <br /> TAG) encodes orthogonal tRNATyr    CUA/LW1RS pair EGFP reporter various numbers TAG codons. Structure pActF shown to <br /> the right. (B) Western blot analysis EGFP expression pAIO-EGFP(n-TAG) BL21(DE3) JX1.0 cells presence absence pActF. <br /> A penta-His antibody used detect Hisx6 tag fused N-terminus EGFP. (C) In-cell &#239;&#172;&#8218;uorescence intensity EGFP expressed by <br /> pAIO-EGFP(n-TAG) BL21(DE3) JX1.0 cells presence absence pActF. Samples normalized cell numbers lane. <br /> Measurement performed independent batches cells, error bars represent SEM. (D) SDS-PAGE analysis EGFP proteins <br /> puri&#239;&#172;?ed JX1.0 cells expressing pAIO-EGFP(n-TAG) absence presence pActF. (E) Fourier transform mass spectrometric analysis of <br /> EGFP expressed pAIO-EGFP(3-TAG) JX1.0 pActF growth medium. The spectra intact precursor ions (charge <br /> deconvoluted) tryptic fragments EGFP. Monoisotopic masses measured high accuracy indicate pActF incorporated all <br /> three UAG sites. Peptide sequences shown, X represents UAG encoded site. Top: site 39, expected [M + H]+ =1529.65 Da, measured <br /> 1529.66 Da. Middle: site 151, expected [M + H]+ =1999.89 Da, measured 1999.90 Da; 2015.89 peak corresponds Met oxidation this <br /> pActF-containing peptide. Bottom: site 182, expected [M + H]+ =4498.18 Da, measured 4498.14 Da. No peaks corresponding amino <br /> acids UAG positions detected. The signal-to-noise ratio &gt;1000, translating &#239;&#172;?delity pActF incorporation &gt;99.9%. Tandem <br /> MS spectra peptides shown Supporting Figure S1, unambiguously con&#239;&#172;?rm peptide sequences pActF position X. <br />  <br /> speci&#239;&#172;?c LW1RS active wild type synthetases39,40             TAG mutant. Nonetheless, underlying mechanism for <br /> and generate fewer aminoacylated orthogonal tRNAs.                  generating truncated products absence RF1 is <br /> Second, orthogonal tRNATyr  CUA evolutionarily              intriguing warrants studies. <br /> optimized UAG decoding, natural tRNAs                      To identify amino acid incorporated response UAG <br /> posttranscriptionally modi&#239;&#172;?ed evolution e&#239;&#172;&#402;cient                  JX1.0, puri&#239;&#172;?ed EGFP expressed JX1.0 using pAIO- <br /> codon recognition.41 Third, natural aminoacyl-tRNAs                EGFP(3-TAG) presence pActF analyzed with <br /> been &#239;&#172;?ne-tuned binding elongation factor Tu               mass spectrometry (Figure 3E). The monoisotopic masses of <br /> ribosome achieve e&#239;&#172;&#402;cient translation,42,43                    tryptic peptides clearly showed pActF incorporated <br /> pActF-charged orthogonal tRNA optimized                     UAG sites. No peaks corresponding the <br /> binding either. All factors lead e&#239;&#172;&#402;cient             incorporation amino acids UAG sites were <br /> decoding UAG pActF ribosome drop-o&#239;&#172;&#8364;                         detected. The precursor ions peptides containing the <br /> translation, resulting truncated protein products. The               UAG sites individually fragmented ion trap mass <br /> e&#239;&#172;&#402;cient decoding UAG pActF accumulative                  spectrometer. The fragment ion masses unambiguously <br /> e&#239;&#172;&#8364;ect account decrease protein yield 10-               assigned, con&#239;&#172;?rming pActF incorporated UAG <br />                                                                       1340                      dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                               Letters <br />  <br /> sites (Supporting Figure S1). These results indicate                   variety genes removed. By studying various <br /> misreading UAG JX1.0 endogenous tRNAs                          nonengineered E. coli strains, discovered wild <br /> outcompeted tRNATyr    CUA/LW1RS pair, speci&#239;&#172;?cally             type RF2 su&#239;&#172;&#402;cient necessary RF1 removal without <br /> decodes UAG pActF.                                                       incurring compensatory mutations. As mutations are <br />    To evaluate initial response E. coli RF1 deletion          arti&#239;&#172;?cially introduced strains, results clearly <br /> subsequent UAG reassignment, assessed health JX1.0                demonstrate RF1 nonessential wild type E. coli. The <br /> using growth assay. JX1.0 healthy, cloneable, stable           successful RF1 knockout multiple strains indicates RF1 <br /> culture; changes <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> genotype observed                  dispensability general property E. coli peculiar <br /> after growing 200 generations. Compared parental                    requirements. <br /> BL21(DE3), JX1.0 showed slower doubling rate (Figure 4).                     Unconditional RF1 removal explained usage of <br />                                                                             UAG E. coli genome. UAG least-used stop codon E. <br />                                                                             coli, terminating &#226;&#710;&#188;7% total genes.29 Among 302 <br />                                                                             essential genes E. coli,44 7 genes ended the <br />                                                                             UAG codon, 7 genes UGA UAA <br />                                                                             stop codon downstream short distances (Supporting Figure <br />                                                                             S2). If UAG readthrough, second di&#239;&#172;&#8364;erent stop <br />                                                                             codon ensure termination proteins a <br />                                                                             small number amino acids appended. The additional amino <br />                                                                             acids completely disable function proteins, <br />                                                                             allowing E. coli survive observed study. The few <br />                                                                             UAG-ending essential genes presence double stop <br />                                                                             codons account nonessentiality RF1. <br />                                                                             Consistently, wild type RF2 the <br />                                                                             weakened A246T mutant permitted RF1 knockout, suggesting <br />                                                                             e&#239;&#172;&#402;cient termination dominant UAA UGA stop <br /> Figure 4. RF1 knockout UAG reassignment slow                   codons required E. coli <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span>. <br /> doubling JX1.0. The growth curves RF1 knockout strain               Knockout RF1 provides novel insights evolution of <br /> JX1.0 represented closed symbols parental              RFs genetic code. A major distinction bacteria from <br /> BL21(DE3) strain open symbols. Squares represent cells only; <br /> circles represent cells transformed pAIO plasmid, which <br />                                                                             eukaryotes archaea protein translation bacteria use <br /> expresses orthogonal tRNACUA   Tyr <br />                                        /LW1RS pair; triangles           di&#239;&#172;&#8364;erent RFs recognize stop codons, a <br /> represent cells transformed pAIO plasmid grown          single RF su&#239;&#172;&#402;cient organisms domains. This <br /> presence pActF. Error bars represent s.e.m., n = 3.                  di&#239;&#172;&#8364;erence served evidence evolution RF and <br />                                                                             translation termination bacteria eukaryotes/ <br /> The doubling time JX1.0 91 min compared 26 min                   archaea nonconserved.45,46 Our results an <br /> for BL21(DE3). This di&#239;&#172;&#8364;erence suggests RF1 knockout                    autonomous E. coli strain single class I RF be <br /> creates burdens cells lack proper termination             generated. The viability bacterium blurs apparent <br /> and relatively weak missuppression UAGs near-cognate                  two-vs-one RF distinction prokaryotes eukar- <br /> tRNAs, e&#239;&#172;&#8364;ect lethal perceived before.                     yotes/archaea, suggesting RF evolution three <br /> Introduction pAIO plasmid expressing orthogonal                  domains life similar current di&#239;&#172;&#8364;erences <br /> tRNATyr <br />       CUA/LW1RS pair absence pActF increased                  suggest. In addition, certain eukaryotes to <br /> doubling JX1.0 BL21(DE3) 135 41 min,                          reassign stop codon sense codon restrict the <br /> respectively, possibly general toxicity expressing              recognition eRF1 rest stop codons.2,17,18 It is <br /> unacylated tRNAs. The addition pActF growth                       unclear stop codon reassignment caused eRF1 <br /> medium expression tRNATyr     CUA/LW1RS                restriction vice versa code evolution. We show <br /> further increased doubling time 253 min JX1.0,                   bacteria, addition eukaryotes nature, <br /> whereas BL21(DE3) a&#239;&#172;&#8364;ected. This dramatic doubling                   undergo stop codon reassignment accompanied <br /> reduction JX1.0 likely e&#239;&#172;&#402;cient                          RF changes. Moreover, results demonstrate RF <br /> incorporation pActF UAG positions                      changes synthetically drive reassignment stop <br /> proteome, re&#239;&#172;&#8218;ecting pressure generated UAG reassign-                 codon amino acid, providing experimental evidence for <br /> ment sense codon. In contrast, BL21(DE3) expresses RF1,                hypothetic evolution pathway. <br /> which competes UAG positions termination,                       An autonomous RF1 knockout bacterium enable new <br /> mitigating pressure.                                                   research understanding code evolution. A major challenge <br />    We RF1 nonessential E. coli species,           studying evolution genetic code many <br /> in contrast previous conclusion RF1 essential. By            questions reach direct experimentation.47 No <br /> reverting Thr246 Ala removing autoregulation                  organisms exist containing primitive intermediate genetic <br /> RF2 expression, recently knocked RF1                         code comparison, natural code evolution takes millions <br /> engineered E. coli strain MDS42 reduced genome.13                years. The RF1 knockout strain reported a&#239;&#172;&#8364;ord a <br /> Because &#226;&#710;&#188;700 nonessential genes deleted MDS42                    previously unavailable model organism study otherwise <br /> multiple mutations introduced RF2 gene,                  intractable questions codon reassignment process real <br /> di&#239;&#172;&#402;cult conclude factors speci&#239;&#172;?cally contribute RF1              time. We RF1 knockout introduction of <br /> knockout RF1 dispensability general property               orthogonal tRNA/synthetase pair completely reassigned <br /> of E. coli. Moreover, MDS42 wild type E. coli              UAG stop signal amino acid, setting initial stage <br /> suitable investigation code evolution adaptation             bacterium adopt altered genetic code. Such stage <br />                                                                      1341                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                       Letters <br />  <br /> had buried recess evolution experimentally                     electroporated DH10&#206;&#178; harboring pKD46 plasmid. KanR <br /> inaccessible. Using RF1 knockout strain                         clones screened using PCR sequence veri&#239;&#172;?ed mutation <br /> able address questions altered code                    position 246 Ala. <br /> eventually &#239;&#172;?xed, long process take,                        Plasmid Construction. All plasmids assembled standard <br />                                                                                   cloning methods con&#239;&#172;?rmed DNA sequencing. pAIO plasmids <br /> physiological changes necessary adaptations.                         containing EGFP gene di&#239;&#172;&#8364;erent TAG codons synthesized <br /> Answers questions help understand                      follows: EGFP cassettes N-terminal Hisx6 tag TAG <br /> the fundamentals code evolution provide rare                          codons various positions created using overlapping PCRs. The <br /> empirical data guide code optimization speci&#239;&#172;?c synthetic                   following sites used: Y182 1-TAG; Y39 Y182 2-TAG; <br /> purposes. In addition, hypothesized genetic code                   Y39, Y182, Y151 3-TAG; Y39, K101, D102, E132, D133, <br /> started set primitive amino acids                  K140, E172, D173, D190, V193 10-TAG. These cassettes were <br /> added total 20 reached.48,49 It remains mysterious                  &#239;&#172;?rst cloned pBP-Blunt (Biopioneer, San Diego, CA) then <br /> whether addition new amino acids repertoire                         digested ligated pBK-AIO vectors containing orthogonal <br />                                                                                                           39 <br />                                                                                   tRNATyrCUA LW1RS       using Spe I BglII restriction sites. pBAD <br /> a&#239;&#172;&#8364;ords evolutionary advantage drive code expansion. This                       vectors containing 1- 3-TAG EGFP genes constructed by <br /> challenging fundamental question investigated                          inserting EGFP cassettes pBAD/His vector (Invitrogen <br /> experimentally strains reported here.                                        life technologies) using Nco I Hind III restriction sites. <br />    Our &#239;&#172;?nding guide genome recoding reassign stop                             In-Cell Fluorescence. In-cell &#239;&#172;&#8218;uorescence intensity deter- <br /> codons successfully. It promising generate E. coli                       mined using FluoroLog-3 (Horiba Jobin Yvon). E. coli colonies were <br /> through genome engineering endogenous UAG                                picked grown 2xYT medium 16 h Uaas. <br /> codons replaced synonymous UAA stop codon.16                             Cells washed twice PBS bu&#239;&#172;&#8364;er diluted PBS an <br /> However, reach &#239;&#172;?nal goal setting aside UAG                          OD600 0.1. The emission spectrum EGFP recorded 503 <br /> codon encode Uaa unambiguously, RF1 removed                          560 nm using excitation wavelength 488 nm. The &#239;&#172;&#8218;uorescence <br />                                                                                   intensity sample compared using intensity 511 <br /> from host strain. MG1655, K-12 derivative strain,                       nm emission peak. <br /> being used current genome engineering e&#239;&#172;&#8364;ort,16                           Growth Assay. A colony picked E. coli strain and <br /> will permit RF1 knockout basis &#239;&#172;?ndings here.                    grown overnight 2xYT medium appropriate antibiotics. <br /> Our results provide solution impasse: reversion                 Cells normalized OD600 1 diluted 1:50 fresh 2xYT <br /> Thr246 Ala RF2 gene enable RF1 knockout                          medium antibiotics 1 mM Uaa (when applicable). For <br /> and clean recoding, conveniently applied                      BL21(DE3) strains, OD600 measured 30 min 10 h. <br /> genome engineering stage &#239;&#172;?nal RF1 gene removal.                        For JX1.0 strains, OD600 measured 60 min 48 h. <br />    An autonomous RF1 knockout bacterium a&#239;&#172;&#8364;ord unique                       Doubling times calculated exponential growth phase in <br />                                                                                   strain. <br />  <br />                                                                                   &#226;&#8211;&#160; <br /> host synthesizing evolving new protein functions and <br /> biological properties Uaa exploitation. To date,                                ASSOCIATED CONTENT <br /> directed laboratory evolution new biosynthesis ability using <br /> genetically encoded Uaas feasible                         * <br />                                                                                   S Supporting Information <br />  <br /> demanding requirements: simultaneous Uaa incorporation                         Supporting methods genome sequencing, Western analyses, <br /> multiple sites autonomous host. Multisite Uaa                              protein puri&#239;&#172;?cation, mass spectrometry, supporting &#239;&#172;?gures <br /> incorporation enables synergy maximizes exploration                       table. This material available free charge the <br />                                                                                   Internet http://pubs.acs.org. <br />  <br />                                                                                   &#226;&#8211;&#160; <br /> of protein sequence space, self-sustaining host such <br /> ability necessary experimental evolution. The RF1 <br /> deletion strains generated ful&#239;&#172;?ll requirements                             AUTHOR INFORMATION <br /> and open &#239;&#172;?eld Uaa-based directed                              Corresponding Author <br /> evolution. Since variety new functionalities                          *E-mail: lwang@salk.edu. <br /> introduced proteins Uaas, organisms containing                       Author Contributions <br />                                                                                   &#226;&#352;&#165; <br /> an expanded genetic repertoire greater potential novel                       These authors contributed equally work. <br /> biosynthesis directed evolution, biorenewable                         Notes <br /> production chemicals fuels.                                                The authors declare competing &#239;&#172;?nancial interest. <br />  <br /> &#226;&#8211;&#160;   METHODS <br />    Strain Construction. Knockout prfA gene attempted <br />                                                                                   &#226;&#8211;&#160;    ACKNOWLEDGMENTS <br />                                                                                   We thank W. Fischer mass spectrometric analyses. This <br /> using chloramphenicol acetyltransferase (cat) cassette established          work supported, part, Howard Hughes Medical <br /> procedures.23 Brie&#239;&#172;&#8218;y, 51 nucleotide overhangs homologous                   Institute grant Gordon Betty Moore <br /> regions immediately 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; prfA appended cat gene.              foundation J.R.E. L.W. acknowledges support the <br /> One microgram cassette electroporated various strains            Searle Scholar Program (06-L-119), Beckman Young Inves- <br /> harboring pKD46 plasmid, expresses phage &#206;&#187; red                      tigator Program, March Dimes Foundation (#5-FY08-110), <br /> recombinase. Chloramphenicol-resistant clones screened for <br />                                                                                   California Institute Regenerative Medicine (RN1-00577-1) <br /> knockout genomic PCR using primers 5&#226;&#8364;&#178;-GGA TAA CGA ACG <br />                                                                                   National Institutes Health (1DP2OD004744-01). <br />  <br />                                                                                   &#226;&#8211;&#160; <br /> CCT GAA TA-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- TCC AGC AGG ATT TCA GCA TC-3&#226;&#8364;&#178;. <br /> Positive clones veri&#239;&#172;?ed DNA sequencing genomic <br /> sequencing.                                                                             REFERENCES <br />    DH10&#206;&#178;f constructed DH10&#206;&#178; follows revert                      (1) Osawa, S., Jukes, T., Watanabe, K., Muto, A. (1992) Recent <br /> Thr246 prf B Ala. A knockin cassette &#239;&#172;?rst generated                     evidence evolution genetic code. Microbiol. Rev. 56, 229&#226;&#710;&#8217; <br /> containing prf B gene BL21(DE3) transcriptionally coupled                264. <br /> to kanamycin resistant (KanR) cassette. The KanR cassette                    (2) Knight, R. D., Freeland, S. J., Landweber, L. F. (2001) <br /> &#239;&#172;&#8218;anked 3&#226;&#8364;&#178; end 51-nucleotide region homologous 3&#226;&#8364;&#178;               Rewiring keyboard: evolvability genetic code. Nat. Rev. <br /> end endogenous prf B gene. One microgram cassette              Genet. 2, 49&#226;&#710;&#8217;58. <br />  <br />                                                                            1342                       dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                           Letters <br />  <br />  (3) Youngman, E. M., McDonald, M. E., Green, R. (2008)                            (22) Mukai, T., Hayashi, A., Iraha, F., Sato, A., Ohtake, K., Yokoyama, <br /> Peptide release ribosome: mechanism implications                      S., Sakamoto, K. (2010) Codon reassignment Escherichia <br /> translational control. Annu. Rev. Microbiol. 62, 353&#226;&#710;&#8217;373.                            coli genetic code. Nucleic Acids Res. 38, 8188&#226;&#710;&#8217;8195. <br />  (4) Frolova, L., Le Go&#239;&#172;&#8364;, X., Rasmussen, H. H., Cheperegin, S.,                        (23) Datsenko, K., Wanner, B. (2000) One-step inactivation of <br /> Drugeon, G., Kress, M., Arman, I., Haenni, A. L., Celis, J. E., Philippe,            chromosomal genes Escherichia coli K-12 using PCR products. Proc. <br /> M., Justesen, J., Kisselev, L. (1994) A highly conserved eukaryotic              Natl. Acad. Sci. U.S.A. 97, 6640&#226;&#710;&#8217;6645. <br /> protein family possessing properties polypeptide chain release                      (24) Studier, F. W., Daegelen, P., Lenski, R. E., Maslov, S., Kim, <br /> factor. Nature 372, 701&#226;&#710;&#8217;703.                                                         J. F. (2009) Understanding di&#239;&#172;&#8364;erences genome sequences <br />  (5) Dontsova, M., Frolova, L., Vassilieva, J., Piendl, W., Kisselev, L.,            Escherichia coli B strains REL606 BL21(DE3) comparison <br /> and Garber, M. (2000) Translation termination factor aRF1                   E. coli B K-12 genomes. J. Mol. Biol. 394, 653&#226;&#710;&#8217;680. <br /> archaeon Methanococcus jannaschii active eukaryotic ribosomes.                 (25) Kang, Y., Durfee, T., Glasner, J. D., Qiu, Y., Frisch, D., <br /> FEBS Lett. 472, 213&#226;&#710;&#8217;216.                                                             Winterberg, K. M., Blattner, F. R. (2004) Systematic mutagenesis <br />  (6) Scolnick, E., Tompkins, R., Caskey, T., Nirenberg, M. (1968)                Escherichia coli genome. J. Bacteriol. 186, 4921&#226;&#710;&#8217;4930. <br /> Release factors di&#239;&#172;&#8364;ering speci&#239;&#172;?city terminator codons. Proc. Natl.              (26) Posfai, G., Plunkett, G., 3rd, Feher, T., Frisch, D., Keil, G. M., <br /> Acad. Sci. U.S.A. 61, 768&#226;&#710;&#8217;774.                                                       Umenho&#239;&#172;&#8364;er, K., Kolisnychenko, V., Stahl, B., Sharma, S. S., Arruda, <br />  (7) Gibson, D. G., Glass, J. I., Lartigue, C., Noskov, V. N., Chuang, R.            M., Burland, V., Harcum, S. W., Blattner, F. R. (2006) Emergent <br /> Y., Algire, M. A., Benders, G. A., Montague, M. G., Ma, L., Moodie, M.               properties reduced-genome Escherichia coli. Science 312, 1044&#226;&#710;&#8217;1046. <br /> M., Merryman, C., Vashee, S., Krishnakumar, R., Assad-Garcia, N.,                      (27) Uno, M., Ito, K., Nakamura, Y. (1996) Functional speci&#239;&#172;?city <br /> Andrews-Pfannkoch, C., Denisova, E. A., Young, L., Qi, Z. Q., Segall-                amino acid position 246 tRNA mimicry domain bacterial <br /> Shapiro, T. H., Calvey, C. H., Parmar, P. P., Hutchison, C. A., 3rd,                 release factor 2. Biochimie 78, 935&#226;&#710;&#8217;943. <br /> Smith, H. O., Venter, J. C. (2010) Creation bacterial cell                    (28) Dincbas-Renqvist, V., Engstrom, A., Mora, L., Heurgue-Hamard, <br /> controlled chemically synthesized genome. Science 329, 52&#226;&#710;&#8217;56.                   V., Buckingham, R., Ehrenberg, M. (2000) A post-translational <br />  (8) Wang, L., Brock, A., Herberich, B., Schultz, P. G. (2001)                   modi&#239;&#172;?cation GGQ motif RF2 Escherichia coli stimulates <br /> Expanding genetic code Escherichia coli. Science 292, 498&#226;&#710;&#8217;500.                termination translation. EMBO J. 19, 6900&#226;&#710;&#8217;6907. <br />  (9) Wang, L., Schultz, P. G. (2005) Expanding genetic code.                   (29) Nakamura, Y., Gojobori, T., Ikemura, T. (2000) Codon <br /> Angew. Chem., Int. Ed. 44, 34&#226;&#710;&#8217;66.                                                    usage tabulated international DNA sequence databases: status for <br />  (10) Wang, Q., Parrish, A. R., Wang, L. (2009) Expanding                    year 2000. Nucleic Acids Res. 28, 292. <br /> genetic code biological studies. Chem. Biol. 16, 323&#226;&#710;&#8217;336.                          (30) Jeong, H., Barbe, V., Lee, C. H., Vallenet, D., Yu, D. S., Choi, S. <br />  (11) Wu, X., Schultz, P. G. (2009) Synthesis interface                H., Couloux, A., Lee, S. W., Yoon, S. H., Cattolico, L., Hur, C. G., Park, <br /> chemistry biology. J. Am. Chem. Soc. 131, 12497&#226;&#710;&#8217;12515.                           H. S., Segurens, B., Kim, S. C., Oh, T. K., Lenski, R. E., Studier, F. W., <br />  (12) Liu, C. C., Schultz, P. G. (2010) Adding new chemistries                Daegelen, P., Kim, J. F. (2009) Genome sequences Escherichia <br /> the genetic code. Annu. Rev. Biochem. 79, 413&#226;&#710;&#8217;444.                                   coli B strains REL606 BL21(DE3). J. Mol. Biol. 394, 644&#226;&#710;&#8217;652. <br />  (13) Johnson, D. B., Xu, J., Shen, Z., Takimoto, J. K., Schultz, M. D.,               (31) Durfee, T., Nelson, R., Baldwin, S., Plunkett, G., 3rd, Burland, <br /> Schmitz, R. J., Xiang, Z., Ecker, J. R., Briggs, S. P., Wang, L. (2011)          V., Mau, B., Petrosino, J. F., Qin, X., Muzny, D. M., Ayele, M., Gibbs, <br /> RF1 knockout allows ribosomal incorporation unnatural amino                       R. A., Csorgo, B., Posfai, G., Weinstock, G. M., Blattner, F. R. <br /> acids multiple sites. Nat. Chem. Biol. 7, 779&#226;&#710;&#8217;786.                                (2008) The complete genome sequence Escherichia coli DH10B: <br />  (14) Anderson, J. C., Wu, N., Santoro, S. W., Lakshman, V., King, D.                insights biology laboratory workhorse. J. Bacteriol. 190, <br /> S., Schultz, P. G. (2004) An expanded genetic code                        2597&#226;&#710;&#8217;2606. <br /> functional quadruplet codon. Proc. Natl. Acad. Sci. U.S.A. 101, 7566&#226;&#710;&#8217;                  (32) Zhang, S., Ryden-Aulin, M., Kirsebom, L. A., Isaksson, L. A. <br /> 7571.                                                                                (1994) Genetic implication interaction release factor <br />  (15) Huang, Y., Russell, W. K., Wan, W., Pai, P. J., Russell, D. H.,            ribosomal protein L7/L12 vivo. J. Mol. Biol. 242, 614&#226;&#710;&#8217;618. <br /> Liu, W. (2010) A convenient method genetic incorporation                        (33) Ito, K., Uno, M., Nakamura, Y. (1998) Single amino acid <br /> multiple noncanonical amino acids protein Escherichia coli.              substitution prokaryote polypeptide release factor 2 permits to <br /> Mol. Biosyst. 6, 683&#226;&#710;&#8217;686.                                                            terminate translation stop codons. Proc. Natl. Acad. Sci. <br />  (16) Isaacs, F. J., Carr, P. A., Wang, H. H., Lajoie, M. J., Sterling, B.,          U.S.A. 95, 8165&#226;&#710;&#8217;8169. <br /> Kraal, L., Tolonen, A. C., Gianoulis, T. A., Goodman, D. B., Reppas, N.                (34) Dahlgren, A., Ryden-Aulin, M. (2000) A <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>novel mutation</span> in <br /> B., Emig, C. J., Bang, D., Hwang, S. J., Jewett, M. C., Jacobson, J. M.,             ribosomal protein S4 a&#239;&#172;&#8364;ects function mutated RF1. <br /> and Church, G. M. (2011) Precise manipulation chromosomes                      Biochimie 82, 683&#226;&#710;&#8217;691. <br /> vivo enables genome-wide codon replacement. Science 333, 348&#226;&#710;&#8217;353.                      (35) Kaczanowska, M., Ryden-Aulin, M. (2004) Temperature <br />  (17) Lozupone, C., Knight, R., Landweber, L. (2001) The                         sensitivity caused mutant release factor 1 suppressed by <br /> molecular basis nuclear genetic code change ciliates. Curr. Biol.              mutations a&#239;&#172;&#8364;ect 16S rRNA maturation. J. Bacteriol. 186, 3046&#226;&#710;&#8217; <br /> 11, 65&#226;&#710;&#8217;74.                                                                           3055. <br />  (18) Inagaki, Y., Doolittle, W. F. (2001) Class I release factors              (36) Yang, T. T., Cheng, L., Kain, S. R. (1996) Optimized codon <br /> ciliates variant genetic codes. Nucleic Acids Res. 29, 921&#226;&#710;&#8217;927.                 usage chromophore mutations provide enhanced sensitivity with <br />  (19) Inagaki, Y., Bessho, Y., Osawa, S. (1993) Lack peptide-                 green &#239;&#172;&#8218;uorescent protein. Nucleic Acids Res. 24, 4592&#226;&#710;&#8217;4593. <br /> release activity responding codon UGA Mycoplasma capricolum.                     (37) Murgola, E. J. (2005) Translational suppression: two <br /> Nucleic Acids Res. 21, 1335&#226;&#710;&#8217;1338.                                                    wrongs make right, tRNA: Structure, Biosynthesis, Function <br />  (20) Gerdes, S. Y., Scholle, M. D., Campbell, J. W., Balazsi, G.,                   (Soll, D., RajBhandary, U. L., Eds.), pp 491&#226;&#710;&#8217;509, ASM Press, <br /> Ravasz, E., Daugherty, M. D., Somera, A. L., Kyrpides, N. C.,                        Washington, DC. <br /> Anderson, I., Gelfand, M. S., Bhattacharya, A., Kapatral, V., D&#226;&#8364;&#8482;Souza,                 (38) Wang, L., Schultz, P. G. (2001) A general approach the <br /> M., Baev, M. V., Grechkin, Y., Mseeh, F., Fonstein, M. Y., Overbeek,                 generation orthogonal tRNAs. Chem. Biol. 8, 883&#226;&#710;&#8217;890. <br /> R., Barabasi, A. L., Oltvai, Z. N., Osterman, A. L. (2003)                         (39) Wang, L., Zhang, Z., Brock, A., Schultz, P. G. (2003) <br /> Experimental determination level analysis essential                    Addition keto functional group genetic code of <br /> genes Escherichia coli MG1655. J. Bacteriol. 185, 5673&#226;&#710;&#8217;5684.                      Escherichia coli. Proc. Natl. Acad. Sci. U.S.A. 100, 56&#226;&#710;&#8217;61. <br />  (21) Ryde&#204;?n, S., Isaksson, L. (1984) A temperature-sensitive                      (40) Takimoto, J. K., Adams, K. L., Xiang, Z., Wang, L. (2009) <br /> mutant Escherichia coli shows enhanced misreading UAG/A                   Improving orthogonal tRNA-synthetase recognition e&#239;&#172;&#402;cient <br /> and increased e&#239;&#172;&#402;ciency tRNA nonsense suppressors. Mol.                      unnatural amino acid incorporation application mammalian <br /> Gen. Genet. 193, 38&#226;&#710;&#8217;45.                                                              cells. Mol. Biosyst. 5, 931&#226;&#710;&#8217;934. <br />  <br />                                                                               1343                        dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                   Letters <br />  <br />  (41) Yokoyama, S., Nishimura, S. (1995) Modi&#239;&#172;?ed nucleosides and <br /> codon recognition, tRNA: Structure, Biosynthesis, Function (Soll, <br /> D., RajBhandary, U. L., Eds.), pp 207&#226;&#710;&#8217;223, ASM Press, <br /> Washington, DC. <br />  (42) Schrader, J. M., Chapman, S. J., Uhlenbeck, O. C. (2011) <br /> Tuning a&#239;&#172;&#402;nity aminoacyl-tRNA elongation factor Tu for <br /> optimal decoding. Proc. Natl. Acad. Sci. U.S. . 108, 5215&#226;&#710;&#8217;5220. <br />  (43) Ledoux, S., Uhlenbeck, O. C. (2008) Di&#239;&#172;&#8364;erent aa-tRNAs <br /> are selected uniformly ribosome. Mol. Cell 31, 114&#226;&#710;&#8217;123. <br />  (44) Kato, J., Hashimoto, M. (2007) Construction consecutive <br /> deletions Escherichia coli chromosome. Mol. Syst. Biol. 3, 13 <br /> </body></html>