<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                        Europe PMC Funders Group <br />                                         Author Manuscript <br />                                         Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                               Published final edited form as: <br />                                                Mol Microbiol. 2006 July ; 61(1): 89&#226;&#8364;&#8220;105. doi:10.1111/j.1365-2958.2006.05216.x. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                               The glycosylphosphatidylinositol (GPI) biosynthetic pathway of <br />                                               bloodstream-form Trypanosoma brucei dependent de <br />                                               novo synthesis inositol <br />                                               Kirstee L. Martin Terry K. Smith* <br />                                               Division Biological Chemistry Molecular Microbiology, The School Life Sciences, <br />                                               University Dundee, Dundee DD1 5EH, Scotland, UK <br />  <br />                                               Summary <br />                                                    In bloodstream-form Trypanosoma brucei (the causative agent African sleeping sickness) the <br />                                                    glycosylphosphatidylinositol (GPI) anchor biosynthetic pathway validated genetically <br />                                                    chemically drug target. The conundrum GPI anchors vivo labelled <br />                                                    [3H]-inositol led hypothesize novo synthesis responsible supplying myo- <br />                                                    inositol phosphatidylinositol (PI) destined GPI synthesis. The rate-limiting step de <br />                                                    novo synthesis isomerization glucose 6-phosphate 1-D-myo-inositol-3-phosphate, <br />                                                    catalysed 1-D-myo-inositol-3-phosphate synthase (INO1). When grown non-permissive <br />                                                    conditions, conditional double <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> demonstrated INO1 essential gene in <br />                                                    bloodstream-form T. brucei. It showed novo synthesized myo-inositol utilized <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                    form PI, preferentially used GPI biosynthesis. We time that <br />                                                    extracellular myo-inositol fact used GPI formation limited extent. <br />                                                    Despite this, extracellular inositol compensate deletion INO1. Supporting these <br />                                                    results, change PI levels conditional double knockout cells grown under <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                    non-permissive conditions, showing perturbation growth specific lack de <br />                                                    novo synthesized myo-inositol general inositol-less death. These results suggest that <br />                                                    distinction novo synthesized myo-inositol extracellular <br />                                                    environment. <br />  <br />  <br />                                               Introduction <br />                                                                 Trypanosoma brucei, causative agent African sleeping sickness, expresses dense <br />                                                                 cell-surface coat consisting approximately 5 &#195;&#8212; 106 dimers variant surface glycoprotein <br />                                                                 attached plasma membrane covalent linkage a <br />                                                                 glycosylphosphatidylinositol (GPI)1 membrane anchor. This coat protects parasite <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                 alternative complement pathway host antigenic variation, from <br />  <br />  <br />                                               &#194;&#169; 2006 The Authors Journal compilation &#194;&#169; 2006 Blackwell Publishing Ltd <br />                                               * <br />                                                For correspondence. T.K.Smith@dundee.ac.uk; Tel. (+44) 1382 388688; Fax (+44) 1382 345764. . <br />                                               Supplementary material <br />                                               The following supplementary material available article online: <br />                                               Fig. S1. Southern blot analysis T. brucei genomic DNA, using TbINO1 ORF probe. <br />                                               Fig. S2. Deamination lipid samples vivo labelling. <br />                                               Fig. S3. sVSG deglycosylation analysis samples vivo labelling. <br />                                               Fig. S4. Analysis exchange reaction GPI-VSG substrate. <br />                                               Table S1. Deamination lipid samples vivo labelling. Cells labelled [3H]-glucose [3H]-mannose, lipids <br />                                               extracted, deaminated subjected butanol water partitioning. [3H]-Levels butanol water phases determined by <br />                                               scintillation spectrometry expressed percentage total cpm. <br />                                               This material available online article http://www.blackwell-synergy.com <br />                                          Martin Smith                                                                                            Page 2 <br />  <br />  <br />                                                            specific immune responses (Cross, 1996). The biosynthesis conserved GPI anchor <br />                                                            previously genetically (Nagamune et al., 2000; Chang et al., 2002) chemically <br />                                                            validated (Smith et al., 2004) drug target bloodstream-form T. brucei. <br />  <br />                                                            The structure biosynthesis GPI membrane anchors related molecules been <br />                                                            recently reviewed (Ferguson et al., 1999; Kinoshita Inoue, 2000; McConville and <br />                                                            Menon, 2000; Morita et al., 2000a; Macedo et al., 2003). The basic GPI core structure <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            attached protein comprises NH2CH2CH2PO4H-6Man&#206;&#177;1-2Man&#206;&#177;1-6Man&#206;&#177;1-4GlcN&#206;&#177;1-6D- <br />                                                            myo-inositol-1-HPO4-lipid (EtN-P-Man3GlcN-PI), lipid diacylglycerol, <br />                                                            alkylacylglycerol ceramide (Ferguson et al., 1999). <br />  <br />                                                            The sequence events underlying GPI biosynthesis studied T. brucei <br />                                                            (Masterson et al., 1989; Masterson et al., 1990; Menon et al., 1990a,b; G&#195;&#188;ther Ferguson, <br />                                                            1995; Morita et al., 2000b), Trypanosoma cruzi (Heise et al., 1996), Toxoplasma gondii <br />                                                            (Striepen et al., 1999), Plasmodium falciparum (Gerold et al., 1999), Leishmania (Smith et <br />                                                            al., 1997; Ralton McConville, 1998; Ralton et al., 2002), Saccharomyces cerevisiae <br />                                                            (Sutterlin et al., 1998; Flury et al., 2000) mammalian cells (Hirose et al., 1992; Puoti <br />                                                            Conzelmann, 1993; Chen et al., 1998). In cases, GPI biosynthesis initiated the <br />                                                            addition GlcNAc UDP-GlcNAc phosphatidylinositol (PI) GlcNAc-PI, <br />                                                            de-N-acetylated form GlcN-PI processed species-specific <br />                                                            manner. To date eukaryotes synthesize PI exchange CMP moiety CDP- <br />                                                            diacylglycerol inositol. <br />  <br />                                                            myo-Inositol six-carbon cyclitol, essential metabolite eukaryotes. The <br />                                                            metabolism myo-inositol plays vital role growth regulation, signal transduction, <br />                                                            membrane biogenesis, osmotolerance essential biochemical processes, as <br />                                                            formation GPI anchors. In prokaryotes inositol sparingly, example an <br />                                                            exception mycobacteria, essential biogenesis mycothiol, PI, <br />                                                            phosphatidylinositol mannosides (PIM) GPIs (Besra Chatterjee, 1994; Haites et al., <br />                                                            2005, references containing therein). <br />  <br />                                                            Cellular myo-inositol arise sources: first, myo-inositol taken from <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            extracellular environment; second, myo-inositol released dephosphorylation of <br />                                                            inositol phosphates produced turnover inositol phospholipids. Finally, myo- <br />                                                            inositol novo synthesized concerted action enzymes: 1-D-myo- <br />                                                            inositol-3-phosphate synthase (INO1) (E.C.5.5.1.4) myo-inositol monophosphatase <br />                                                            (IMPase) (E.C.3.1.3.25). The rate-limiting step novo synthesis myo-inositol is <br />                                                            NAD+-dependent conversion D-glucose 6-phosphate 1-D-myo-inositol-3-phosphate <br />                                                            INO1 stage reaction involving: (i) oxidation D-glucose 6-phosphate 5- <br />                                                            keto-D-glucose 6-phosphate, (ii) cyclization myo-2-inosose1-phosphate (iii) reduction <br />                                                            1-D-myo-inositol-3-phosphate (Majumder et al., 1997). <br />  <br />                                                            INO1s characterized prokaryotic various eukaryotic organisms such <br />                                                            Porteresia coarctata (Majee et al., 2004), Arabidopsis thaliana (Johnson Sussex, <br />                                                            1995), Mycobacterium tuberculosis (Norman et al., 2002; Movahedzadeh et al., 2004), <br />                                                            Mycobacterium smegmatis (Haites et al., 2005), Drosophila melanogaster (Park Kim, <br />                                                            2004), S. cerevisiae (Donahue Henry, 1981), Entamoeba histolytica (Lohia et al., 1999), <br />                                                            Leishmania mexicana (Ilg, 2002) Homo sapiens (Guan et al., 2003; Ju et al., 2004). <br />                                                            Genetic studies pathogenic organisms, L. mexicana (Ilg, 2002) M. <br />                                                            tuberculosis (Movahedzadeh et al., 2004), revealed INO1 important in <br />                                                            regulating intracellular myo-inositol levels, INO1 potential drug target in <br />                                                            organisms. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 3 <br />  <br />  <br />                                                            In study, begin investigate novo synthesis metabolism myo-inositol <br />                                                            T. brucei. Here molecular cloning expression Escherichia coli the <br />                                                            T. brucei myo-inositol-3-phosphate synthase characterization recombinant <br />                                                            enzyme. We report creation T. brucei conditional null mutant which <br />                                                            demonstrates novo synthesis myo-inositol essential bloodstream form <br />                                                            parasite, presence excess myo-inositol. Our results suggest de <br />                                                            novo synthesized myo-inositol preferentially used GPI anchor biosynthesis that <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            differentiation novo synthesized myo-inositol myo-inositol taken up <br />                                                            extracellular environment. <br />  <br />                                            Results <br />                                            Cloning sequencing T. brucei INO1 <br />                                                            An INO1 identified T. brucei genome database (Sanger Centre) using TBLASTN; this <br />                                                            putative open reading frame (ORF) PCR-amplified, cloned sequenced. The <br />                                                            sequence submitted GenBank, Accession No. AJ86670. An alignment the <br />                                                            predicted translated sequence INO1s organisms shown Fig. 1A an <br />                                                            unrooted phylogenetic tree Fig. 1B. The predicted TbINO1 perfect copy motif <br />                                                            GWGGNNG (underlined Fig. 1A), involved NAD+ binding, Rossmann <br />                                                            fold typical oxidoreductase (Majumder et al., 2003). It contains motifs <br />                                                            previously identified eukaryotic INO1s, LWTANTERY, <br />                                                            NGSPQNTFVPGL SYNHLGNNDG (Majumder et al., 2003). Although TbINO1 shows <br />                                                            considerable similarity INO1s variety organisms, closely related to <br />                                                            kinetoplastids, T. cruzi L. mexicana (Fig. 1B). The size the <br />                                                            predicted TbINO1 protein 58 kDa, good agreement INO1s which <br />                                                            generally fall range 58&#226;&#8364;&#8220;67 kDa (Majumder et al., 1997; 2003). <br />  <br />                                            Overexpression INO1 E. coli <br />                                                            To enable biochemical characterization T. brucei INO1 overexpressed E. coli <br />                                                            using pBAD TA vector, encodes C-terminal hexa-His tag. This His-tagged <br />                                                            recombinant protein purified using Ni2+ eluted using increasing concentration of <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            imidazole. This purification protocol resulted pure protein preparation, shown Fig. <br />                                                            2A. The apparent molecular weight recombinant protein approximately 75 kDa <br />                                                            SDS-PAGE analysis, higher predicted size 64 kDa, includes short <br />                                                            leader sequence hexa-His tag. However, molecular weight purified <br />                                                            recombinant protein shown approximately 64 kDa MALDI analysis (data not <br />                                                            shown). <br />  <br />                                                            This recombinant protein shown catalytically active inositol-3-phosphate <br />                                                            synthase use coupled assay, product INO1 activity, <br />                                                            inositol-3-phosphate, used substrate IMPase, phosphate released by <br />                                                            IMPase measured colorimetrically. To study kinetic parameters glucose 6- <br />                                                            phosphate, components standard assay held constant glucose 6- <br />                                                            phosphate concentration varied 0 5 mM. Saturation kinetics shown the <br />                                                            enzyme, apparent Km 0.58 mM Vmax 0.25 &#206;&#188;mole h&#226;&#710;&#8217;1 (Fig. 2B) the <br />                                                            specific activity recombinant TbINO1 756 U mg&#226;&#710;&#8217;1. Like other <br />                                                            studied INO1 proteins, recombinant TbINO1 activity dependent NAD+ activity <br />                                                            observed absence NAD+, NAD+ replaced NADH or <br />                                                            NADPH. Some activity (5% normal) observed NAD+ replaced with <br />                                                            NADP+, suggesting NADP+ act hydrogen acceptor/donor the <br />                                                            reaction limited degree. Interestingly, NAD+ concentrations low <br />                                                            0.1 &#206;&#188;mole, 100% activity observed compared 1 mM, suggesting extremely <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 4 <br />  <br />  <br />                                                            efficient binding NAD+. The enzyme activity T. brucei INO1 stimulated the <br />                                                            presence NH+4 reaction mixture; little activity observed the <br />                                                            absence NH4+ (10 U mg&#226;&#710;&#8217;1), compared activity presence 2 mM <br />                                                            NH4Ac (756 U mg&#226;&#710;&#8217;1), enhancement activity observed the <br />                                                            NH4Ac concentration increased 10 mM. <br />  <br />                                            INO1 essential gene T. brucei bloodstream-form cells <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Southern blot analysis T. brucei genomic DNA showed INO1 present single- <br />                                                            copy gene haploid genome (Fig. S1 Supplementary material). A schematic the <br />                                                            construction conditional double knockout shown Fig. 3A. One allele INO1 <br />                                                            replaced puromycin drug resistance gene homologous recombination and <br />                                                            selection puromycin, creating &#206;&#8221;INO1::PAC cell line (Fig. 3B, lane 2). Attempts to <br />                                                            create null mutant homologous replacement second allele hygromycin <br />                                                            resistance gene unsuccessful, inositol level media increased from <br />                                                            40 &#206;&#188;M (normal HMI-9 media) 100 mM. Therefore, &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; cell line used here <br />                                                            constitutively expresses T7 RNA polymerase tetracycline repressor protein, it <br />                                                            decided introduce tetracycline-inducible (Ti) myc-tagged ectopic copy INO1 <br />                                                            &#206;&#8221;INO1::PAC cell line prior deletion second allele, allowing creation of <br />                                                            conditional double knockout control tetracycline. The ectopic copy <br />                                                            integrated pLew 100 vector, INO1 inserted downstream <br />                                                            trypanosome procyclin promoter tetracycline operators. Integration the <br />                                                            pLew vector known occur rDNA locus, encodes phleomycin <br />                                                            resistance protein (Wirtz et al., 1999). After drug selection puromycin phleomycin, <br />                                                            INO1-mycTi &#206;&#8221;INO1::PAC clones obtained, integration pLew <br />                                                            100 ectopic copy confirmed PCR using primers specific pLew 100 vector <br />                                                            (data shown) later Southern analysis (Fig. 3B, lane 3). The second allele was <br />                                                            replaced hygromycin resistance gene presence tetracycline. One INO1- <br />                                                            mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG clone obtained genotype confirmed by <br />                                                            Southern blotting (Fig. 3B, lane 4). <br />  <br />                                                            To establish INO1 essential gene, growth INO1-mycTi <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG cell line monitored tetracycline containing and <br />                                                            tetracycline-free HMI-9 contained 40 &#206;&#188;M myo-inositol. In presence of <br />                                                            tetracycline INO1-mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG cell line showed normal growth <br />                                                            rates (Fig. 4B) compared wild-type cells (Fig. 4A). However, absence of <br />                                                            tetracycline cells grew normally 2 days, cell numbers declined the <br />                                                            limits detection light microscopy increasing cellular debris observed <br />                                                            suggesting cell death (Fig. 4C). Even medium supplemented an <br />                                                            additional 100 mM myo-inositol, INO1 conditional knockouts unable survive in <br />                                                            tetracycline-free media (Fig. 4D). However, tetracycline-free HMI-9 (with 40 <br />                                                            &#206;&#188;M 100 mM myo-inositol) c. day 8 live cells visible light <br />                                                            microscopy, day 12 cells resumed normal growth kinetics, shown Fig. 4C. <br />                                                            Northern blot analysis (Fig. 4E) showed transcript level ectopic INO1-mycTi <br />                                                            slightly lower endogenous copy seen bloodstream-form wild- <br />                                                            type cells insect-form procyclic cells. In absence tetracycline 48 h the <br />                                                            transcript INO1-mycTi undetectable (Fig. 4E, lane 4), cells had <br />                                                            spontaneously resumed normal growth absence tetracycline 12 days, the <br />                                                            transcript INO1-mycTi clearly visible (Fig. 4E, lane 5). This suggests these <br />                                                            revertant cells able overcome tetracycline control, phenomena been <br />                                                            described essential genes T. brucei (Krieger et al., 2000; Milne et al., 2001; <br />                                                            Chang et al., 2002; Roper et al., 2002; 2005; Martin Smith, 2006). <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 5 <br />  <br />  <br />                                            In vivo labelling wild-type cells <br />                                                            Wild-type bloodstream-form T. brucei labelled [3H]-mannose, [3H]-glucose or <br />                                                            [3H]-inositol pulse pulse-chase experiments. After labelling 1 h (pulse <br />                                                            experiments) 2 h (pulse-chase experiments) samples taken protein lipid <br />                                                            analysis. <br />  <br />                                                            When cells pulse labelled [3H]-mannose mature GPI glycolipids A C <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            observed (Fig. 5A, lane 3), low levels galactosylated versions of <br />                                                            glycolipid A, described previously (Menon et al., 1988). As expected <br />                                                            pulse chase labelling showed decreased amounts [3H]-labelled glycolipids A C <br />                                                            (Fig. 5, lane 4). This decrease addition equal volume HMI-9 media to <br />                                                            labelling start chase longer labelling time pulse-chase <br />                                                            experiment, allowing labelled glycolipids A C used GPI-VSG attachment. <br />                                                            Protein analysis confirmed [3H]-labelled glycolipds A C incorporated into <br />                                                            mature GPI-anchored VSG equivalent signal observed pulse the <br />                                                            pulse-chase experiments (Fig. 5C, lanes 5 6). <br />  <br />                                                            When labelled [3H]-glucose presence absence myo-inositol, <br />                                                            wild-type cells produced main lipid species (1, 2, 3) (Fig. 5A, lane 5). Lipid species 1 <br />                                                            identical Rf PI; digests showed sensitive PI-PLC PLD, indicative of <br />                                                            PI containing phospholipid. Further results <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> treatment, hydrogen fluoride (HF) <br />                                                            treatment deamination consistent [3H]-<span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> head group and <br />                                                            diacylglycerol portion. To confirm identity head group aqueous phase <br />                                                            HF treatment desalted, concentrated analysed high performance thin-layer <br />                                                            chromatography (HPTLC); Rf [3H]-labelled species identical inositol <br />                                                            standard, clearly showing head group inositol (data shown). These results <br />                                                            clearly existence functional INO1 bloodstream-form T. brucei, [3H]- <br />                                                            glucose converted [3H]-inositol. This [3H]-inositol subsequently used the <br />                                                            cell form PI (lipid species 1). <br />  <br />                                                            Lipid species 2 [3H]-glucose labelling identical Rf glycolipid C standard <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glycolipid C obtained [3H]-mannose labellings (lanes 3 4). Lipid species 2&#226;&#8364;&#8482;s <br />                                                            insensitivity PI-PLC sensitivity PLD consistent glycolipid C. <br />                                                            Lipid species 3 identical Rf glycolipid A standard glycolipid A the <br />                                                            [3H]-mannose labelling (lanes 3 4). These results identification lipid species 2 as <br />                                                            glycolipid C suggest lipid species 3 glycolipid A. Furthermore, lipids [3H]- <br />                                                            glucose labelling subjected deamination butanol-water partioning. HPTLC <br />                                                            autoradiography butanol phase revealed PI detected, glycolipids C <br />                                                            A deaminated, losing glycan head group, confirming identities <br />                                                            GPIs (Fig. S2 Supplementary material). Moreover, confirm [3H]-label was <br />                                                            myo-inositol head group (like PI) glycan head group [3H]-GlcN or <br />                                                            [3H]-Man, relative amounts [3H]-label butanol aqueous phases after <br />                                                            deamination measured. As shown Table S1 Supplementary material, deamination <br />                                                            [3H]-glucose labelling lipids resulted [3H]-label remaining the <br />                                                            butanol phase, showing [3H]-label remained bound lipid moiety. If significant <br />                                                            [3H]-label detected aqueous phase indication [3H]- <br />                                                            label released glycan head group deamination. As control, <br />                                                            [3H]-mannose-labelled glycolipids A C deaminated expected of <br />                                                            [3H]-label partitioned aqueous phase [3H]-mannose glycan head <br />                                                            group (Table S1 Supplementary material), indicating deamination process was <br />                                                            working efficiently. These results confirm majority [3H]-glucose is <br />                                                            incorporated GPI precursors [3H]-myo-inositol [3H]-GlcN [3H]- <br />                                                            mannose. Together results [3H]-glucose converted [3H]-inositol, which <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 6 <br />  <br />  <br />                                                            used form [3H]-PI, subsequently utilized GPI biosynthetic pathway to <br />                                                            form [3H]-glycolipids A C. Similar results observed pulse-chase [3H]- <br />                                                            mannose labelling, [3H]-glucose-labelled glycolipids A C decreased <br />                                                            pulse chase, indicative [3H]-glycolipid species utilized GPI-VSG <br />                                                            attachment (Fig. 5A, lane 6). <br />  <br />                                                            Analysis protein samples [3H]-glucose labelling showed [3H]-labelled <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glycolipids A C incorporated mature GPI-anchored VSG (Fig. 5C). A similar <br />                                                            intensity signal observed pulse pulse-chase experiments (Fig. 5C, lanes <br />                                                            3 4). Although analysis lipid species 1 2 clearly shown [3H]-label <br />                                                            inositol head group, wanted confirm [3H]-label observed the <br />                                                            GPI-anchored VSG incorporation N-linked oligosaccharides present <br />                                                            VSG. This confirmed methods. First, labelling [3H]-glucose, <br />                                                            sVSG (VSG soluble portion GPI anchor attached) purified aliquot <br />                                                            deglycosylated PNGaseF. A clear difference molecular weight sVSG <br />                                                            observed SDS-PAGE analysis, suggesting N-linked oligosaccharides had <br />                                                            successfully removed (Fig. S3A Supplementary material, lanes 1 2). <br />                                                            Autoradiography revealed significant decrease intensity [3H] <br />                                                            signal deglycosylation (Fig. S3A Supplementary material, lanes 3 4). Second, <br />                                                            conducted [3H]-glucose [3H]-mannose labellings presence of <br />                                                            tunicamycin (an inhibitor N-glycosylation) [3H]-label remained GPI-anchored <br />                                                            VSG (Fig. S3B C Supplementary material). Together results confirm the <br />                                                            signal observed sVSG labelling [3H]-glucose N- <br />                                                            glycosylation. We checked incorporation [3H]-label mature GPI- <br />                                                            anchored VSG exchange mechanism observed previously the <br />                                                            incorporation myristate (Buxbaum et al., 1996). Labelling performed presence <br />                                                            absence cycloheximide incorporation [3H]-label GPI-anchored <br />                                                            VSG determined. We detect incorporation [3H]-glucose GPI- <br />                                                            VSG presence cycloheximide showing incorporation exchange <br />                                                            reaction uses GPI-VSG substrate (Fig. S4 Supplementary material). The <br />                                                            expected incorporation [3H]-myristate observed presence cycloheximide <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            described previously (Buxbaum et al., 1996). <br />  <br />                                                            Not surprisingly cells pulse labelled [3H]-inositol, PI lyso-PI were <br />                                                            observed (Fig. 5A, lane 1). However, cells pulse chase labelled [3H]- <br />                                                            inositol additional lipid species observed identical Rf values glycolipids A <br />                                                            C respectively (Fig. 5A, lane 2). The identity new lipid species as <br />                                                            glycolipids A C confirmed digests incorporation [3H]-inositol <br />                                                            GPI-anchored VSG (Fig. 5C D), inositol head <br />                                                            group. The appearance increase glycolipids A C pulse-chase experiment <br />                                                            [3H]-inositol opposite observed labelling [3H]-glucose or <br />                                                            [3H]-mannose, levels glycolipids A C decreased. In parallel an <br />                                                            increase incorporation [3H]-labelled glycolipids A C GPI-anchored <br />                                                            VSG pulse-chase experiments compared pulse (compare Fig. 5D <br />                                                            lanes 1 2). In fact pulse labelling incorporation [3H]-inositol mature <br />                                                            GPI-anchored VSG 5% incorporation [3H]-glucose same <br />                                                            conditions (Fig. 5C D). <br />  <br />                                            Biochemical <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> INO1 conditional knockout <br />                                                            The biochemical phenotype INO1 conditional knockout cells investigated in <br />                                                            vivo pulse-labelling experiments [3H]-mannose [3H]-glucose pulse-chase <br />                                                            experiments [3H]-inositol cells grown presence or <br />                                                            absence tetracycline 2 days compared undertaken wild-type cells. <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 7 <br />  <br />  <br />                                                            After labelling, lipids extracted, desalted, separated HPTLC detected by <br />                                                            fluorography. As expected wild-type INO1 conditional knockout cells grown the <br />                                                            presence tetracycline showed identical incorporation [3H]-lipids (see earlier for <br />                                                            identification [3H]-lipids). <br />  <br />                                                            Labelling INO1 conditional knockout cells grown absence tetracycline with <br />                                                            [3H]-glucose showed decrease [3H]-PI formed (Fig. 6, lane 3), <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            confirming deleted gene functional INO1. There concomitant decrease <br />                                                            glycolipid C glycolipid A, suggesting direct knock-on effect <br />                                                            GPI biosynthetic pathway lack PI [3H]-inositol derived <br />                                                            [3H]-glucose. This knock-on effect GPI biosynthetic pathway confirmed by <br />                                                            [3H]-mannose labelling (Fig. 6). There significant reproducible decrease the <br />                                                            amounts glycolipids A C INO1 conditional knockout cells grown in <br />                                                            absence tetracycline (Fig. 6, lane 7). Again suggests GPI biosynthetic <br />                                                            pathway slowed significantly affect dynamic pools glycolipids A C. <br />  <br />                                                            When conditional double knockout cells grown absence tetracycline 2 <br />                                                            days pulse chase labelled [3H]-inositol, significant change the <br />                                                            levels PI, lyso-PI, glycolipids A C observed compared wild-type cells <br />                                                            conditional double knockout cells grown presence tetracycline (Fig. 6). From <br />                                                            [3H]-inositol labellings inferred cells viable point of <br />                                                            labelling able synthesis lipids. To confirm investigated <br />                                                            cells&#226;&#8364;&#8482; ability synthesize new proteins. In parallel [3H]-glucose, [3H]-mannose <br />                                                            [3H]-inositol labellings described, cells labelled [35S]-methionine. After <br />                                                            labelling quenched, proteins separated SDS-PAGE labelled proteins <br />                                                            detected autoradiography. All cells lines (wild type, conditional double knockout <br />                                                            tetracycline conditional double knockout tetracycline) able to <br />                                                            incorporate significant identical amounts [35S]-methionine protein (Fig. <br />                                                            6C, lanes 1, 2 3). These results clearly illustrate point labelling INO1 <br />                                                            conditional knockout cells grown non-permissive conditions viable their <br />                                                            ability synthesize protein level comparable wild-type cells conditional <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            double knockout cells grown presence tetracycline. Therefore, time of <br />                                                            labelling cells viable biochemical phenotype observed direct result of <br />                                                            deletion INO1. <br />  <br />                                                            The myo-inositol containing lipids cell lines (wild-type cells, <br />                                                            conditional double knockout cells grown presence absence tetracycline) was <br />                                                            quantified GC-MS. There significant difference lipid-bound <br />                                                            myo-inositol cell lines (Fig. 7A), suggesting deletion INO1 not <br />                                                            perturbed overall cellular PI levels. We qualitatively analysed inositol containing <br />                                                            phospholipids ES-MS-MS using parent-ion scanning m/z 241 negative ion mode <br />                                                            (indicative collision induced fragment PI species) did detect significant <br />                                                            difference cell lines (Fig. 7B). <br />  <br />                                            INO1 cytosolic protein bloodstream-form T. brucei <br />                                                            The cellular location INO1 T. brucei bloodstream-form cells investigated using a <br />                                                            tetracycline-inducible overexpression vector, pLew82 containing TbINO1 encoding <br />                                                            C-terminal haemagglutinin (HA) tag. This construct introduced bloodstream-form <br />                                                            T. brucei cells, generating cell line INO1-HATi integration confirmed PCR using <br />                                                            primers specific vector (data shown). Transcription ectopic INO1-HATi <br />                                                            induced tetracycline confirmed Northern blotting (data shown). <br />                                                            Tetracycline-induced INO1-HATi cells fixed, allowed adhere poly lysine slides <br />                                                            permeablized. The INO1-HATi protein detected immunofluorescence using a <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                              Page 8 <br />  <br />  <br />                                                            primary antibody HA tag, secondary (which FITC conjugated) <br />                                                            primary antibody. The FITC signal observed cell body, and <br />                                                            seen flagella, nucleus kinetoplast (Fig. 8A&#226;&#8364;&#8220;D), suggesting TbINO1 is <br />                                                            cytoplasmic. This result confirmed subcellular fractionation studies (Fig. 8I), where <br />                                                            INO1-HA protein detected soluble cytosolic fraction differential <br />                                                            centrifugation. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Discussion <br />                                                            Prior results presented study, accepted field the <br />                                                            GPI anchors contain inositol, GPI intermediates mature GPIs vivo labelled <br />                                                            [3H]-inositol bloodstream-form T. brucei, despite able efficiently label PI. <br />                                                            This paradox explained myo-inositol used GPIs novo synthesized <br />                                                            glucose. The results presented support hypothesis, demonstrating the <br />                                                            existence functional myo-inositol novo synthesis pathway bloodstream-form T. <br />                                                            brucei, showing novo synthesis essential <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> parasite and <br />                                                            disruption pathway detrimental effects GPI anchor biosynthesis, not <br />                                                            overcome use extracellular myo-inositol. These results raise interesting <br />                                                            questions distinction novo synthesized myo-inositol obtained <br />                                                            extracellular sources. <br />  <br />                                                            Most eukaryotic cells able novo synthesize myo-inositol glucose INO1 <br />                                                            IMPase. Recombinant expression putative TbINO1 E. coli demonstrated <br />                                                            catalytically active INO1 ability convert glucose 6-phosphate to <br />                                                            inositol-3-phosphate, dependence NAD+ stimulation NH4+ consistent <br />                                                            INO1s organisms (e.g. Donahue Henry, 1981; Johnson Sussex, <br />                                                            1995; Ju et al., 2004). Saturation kinetics shown enzyme, apparent Km <br />                                                            0.58 mM; similar human INO1 Km 0.5 mM (Ju et al., <br />                                                            2004). Interestingly, specific activity recombinant TbINO1 (756 U mg&#226;&#710;&#8217;1) is <br />                                                            approximately 10-fold higher human recombinant INO1 (70 U mg&#226;&#710;&#8217;1) (Ju et <br />                                                            al., 2004), indication importance TbINO1 T. brucei. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            It previously shown GPI anchor biosynthesis essential survival of <br />                                                            bloodstream-form T. brucei (Nagamune et al., 2000; Chang et al., 2002; Smith et al., 2004); <br />                                                            surmized light this, INO1 novo synthesis myo-inositol <br />                                                            essential responsible providing myo-inositol synthesis of <br />                                                            PI GPI anchors. We unable replace allelic copies INO1 create null <br />                                                            mutant, suggesting TbINO1 essential gene surprisingly, addition of <br />                                                            extra myo-inositol (from 40 &#206;&#188;M 100 mM) media did enable creation a <br />                                                            null mutant. Through creation conditional double knockout cessation of <br />                                                            growth non-permissive conditions, clearly demonstrated INO1 an <br />                                                            essential gene bloodstream-form T. brucei. Interestingly, substantial increase the <br />                                                            myo-inositol concentration growth media (40 &#206;&#188;M 100 mM) did enable the <br />                                                            conditional knockout cells overcome deletion INO1 grown absence of <br />                                                            tetracycline (Fig. 4D). This inability overcome deletion INO1 surprising and <br />                                                            unexpected contrast &#206;&#8221;INO1 mutants date, e.g. S. cerevisiae (Henry <br />                                                            et al., 1977), L. mexicana (Ilg, 2002), M. tuberculosis (Movahedzadeh et al., 2004) <br />                                                            require 27 &#206;&#188;M, 200 &#206;&#188;M 77 mM extracellular myo-inositol respectively. This clear <br />                                                            difference T. brucei INO1 conditional mutant null INO1 mutants from <br />                                                            organisms intriguing probed biochemical phenotyping INO1 <br />                                                            conditional double knockout cells grown non-permissive conditions. <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                               Page 9 <br />  <br />  <br />                                                            To surprise vivo labelling wild-type cells time possible <br />                                                            label glycolipids A C [3H]-inositol. Moreover, able that <br />                                                            labelled myo-inositol incorporated GPI-anchored VSG, a <br />                                                            significantly lower level [3H]-mannose [3H]-glucose. However, only <br />                                                            able observe labelling cells pulse chase labelled. This increase the <br />                                                            labelling GPIs intermediates chase pulse chase labelling quite <br />                                                            surprising opposite observed labelling mannose or <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glucose, labelling glycolipids A C actually decrease chase as <br />                                                            used GPI-VSG attachment. These results suggest extracellular myo-inositol <br />                                                            entering GPI biosynthetic pathway slower rate novo <br />                                                            synthesized myo-inositol mannose. This latency explained recycling, where <br />                                                            extracellular inositol PI synthesized extracellular inositol relocated cell <br />                                                            results mixing types myo-inositol/PI. These results suggest that <br />                                                            novo synthesized myo-inositol primary source GPI synthesis extracellular <br />                                                            inositol secondary source. This observation supported twofold: first, vivo <br />                                                            labelling showed GPI biosynthesis slowed conditional double <br />                                                            knockout cells grown non-permissive conditions lack novo synthesized <br />                                                            inositol/PI, concomitant increase levels extacellular inositol feeding <br />                                                            GPI pathway. Second, significant change total cellular levels of <br />                                                            lipid-based myo-inositol wild-type cells conditional double knockout cells <br />                                                            grown non-permissive conditions. Together, results presented clearly demonstrate <br />                                                            deletion INO1 causing &#226;&#8364;&#732;inositol-less death&#226;&#8364;&#8482;, intriguingly cells <br />                                                            dying decrease novo synthesized myo-inositol, particular <br />                                                            function appears supplying GPI pathway myo-inositol/PI. Moreover, these <br />                                                            results explain additional myo-inositol media did allow creation a <br />                                                            null mutant did enable conditional double knockout cells overcome the <br />                                                            deletion INO1 grown absence tetracycline. <br />  <br />                                                            The clear preference use novo synthesized GPI anchors biosynthesis suggests a <br />                                                            distinction compartmentalization types myo-inositol (extracellular de <br />                                                            novo synthesized). To probe idea started investigating location/s of <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            enzymes novo synthesis pathway bloodstream-form T. brucei. Previously we <br />                                                            shown sole PI synthase (PIS) responsible synthesis bulk cellular PI as <br />                                                            PI destined GPI anchor synthesis located Golgi and <br />                                                            endoplasmic reticulum (ER) (Martin Smith, 2006) shown that <br />                                                            TbINO1 localized cytosol, consistent INO1s other <br />                                                            organisims. Based observations propose following: first, extracellular myo- <br />                                                            inositol utilized Golgi PIS make bulk cellular PI. Second, TbINO1 catalyses the <br />                                                            conversion glucose 6-phosphate inositol-3-phosphate cytoplasm, presence of <br />                                                            phosphate group distinguishes novo synthesized myo-inositol acquired <br />                                                            extracellular sources. This inositol-3-phosphate transported ER, <br />                                                            dephosphorylated IMPase used ER PIS form PI, subsequently <br />                                                            used GPI biosynthetic machinery located ER (see Ferguson, 1999, and <br />                                                            references contained therein). Further evidence supporting hypothesis the <br />                                                            identification putative IMPase (GeneDB annotation) appears located the <br />                                                            ER (K.L. Martin T.K. Smith, unpublished). To knowledge, report <br />                                                            organism differentiation novo synthesized extracellular myo- <br />                                                            inositol current efforts focused understanding complex mechanisms of <br />                                                            phenomena further. The physiological reason preference novo <br />                                                            synthesized myo-inositol remains unclear; evolutionary adaptation guarantee <br />                                                            parasites sustain high level GPI anchor production, ensuring integrity <br />                                                            cell surface protective coat VSG, enabling evasion hosts&#226;&#8364;&#8482; immune system. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                         Page 10 <br />  <br />  <br />                                                            In summary, results presented clearly demonstrate bloodstream-form T. brucei <br />                                                            dependent novo synthesis myo-inositol. This dependence confirmed <br />                                                            genetically perturbation growth INO1 conditional double mutants grown <br />                                                            tetracycline-free media. Furthermore, results suggest compartmentalization PI <br />                                                            synthesis, reported organism date. Surprisingly, there <br />                                                            appears compensatory mechanisms extracellular myo-inositol <br />                                                            replace novo synthesized myo-inositol novo synthesis compromised, <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            strongly validating INO1 novo synthesis myo-inositol bloodstream- <br />                                                            form T. brucei potential therapeutic drug targets. Currently, investigating <br />                                                            compounds interfere novo synthesis inositol drug leads African <br />                                                            sleeping sickness related neglected diseases. <br />  <br />                                            Experimental procedures <br />                                            Cloning T. brucei INO1 gene ligation pLew vectors <br />                                                            A putative INO1 gene identified T. brucei genome database (Sanger Centre) <br />                                                            using TBLASTN search S. cerevisiae INO1 genomic sequence query. The ORF <br />                                                            amplified T. brucei genomic DNA using forward reverse primers 5&#226;&#8364;&#178;- <br />                                                            GAGGAGAAGCTTATGCCAGCCGTCCGTACG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            TCGTTAATTAAACTTCCCACGCCGCGAAG-3&#226;&#8364;&#178; containing HindIII PacI restriction <br />                                                            sites respectively (underlined). A band expected size ~1.5 kb amplified using <br />                                                            Pfu polymerase, purified using QIAquick PCR purification kit (Qiagen) cloned into <br />                                                            pCR-Blunt II TOPO (Invitrogen). The TbINO1 ORF excised using HindIII PacI <br />                                                            subsequently ligated tetracycline-inducible expression vectors pLew82 and <br />                                                            pLew100 (Wirtz et al., 1999) HindIII PacI restriction sites. <br />  <br />                                            Construction T. brucei gene replacement cassettes <br />                                                            The 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; untranslated regions (UTRs) immediately adjacent INO1 ORF were <br />                                                            amplified T. brucei genomic DNA using Pfu polymerase. The primers 5&#226;&#8364;&#178;- <br />                                                            ATAAGAATGCGGCCGCAATGGCTGTTGTGGCACAAGG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            GGATCCGTTTAAACTTACGGACCGTCAAGCTTGTACCACAACACGAATCTTCAG <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            G-3&#226;&#8364;&#178; used 5&#226;&#8364;&#178; UTR, amplifying expected 398 bp product primers 5&#226;&#8364;&#178;- <br />                                                            AAGCTTTAAACGGATCCGTAAGTTTAAACGGATCCTGAATGTAACCGTCTACTT <br />                                                            CGTTCCGC-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            ATAAGAATGCGGCCGCATGAAAAGAAAACATCGGGGAAT-3&#226;&#8364;&#178; resulted the <br />                                                            correct 356 bp product 3&#226;&#8364;&#178; UTR. These amplified products used knitting <br />                                                            PCR, resulting 744 bp product 5&#226;&#8364;&#178; UTR joined 3&#226;&#8364;&#178; UTR short <br />                                                            BamHI&#226;&#8364;&#8220;HindIII linker region contained described primers (italics) NotI site <br />                                                            (underlined) end. This PCR product ligated pGEM-5Zf(+) (Promega) via <br />                                                            NotI sites hygromycin phosphotransferase (HYG) puromycin <br />                                                            acetyltransferase (PAC) resistance genes ligated BamHI HindIII <br />                                                            restriction sites. Plasmid DNA prepared using QIAprep Miniprep Plasmid Kit <br />                                                            (Qiagen); digestion NotI, precipitated sodium acetate/ethanol and <br />                                                            redissolved sterile water final concentration 1 &#206;&#188;g &#206;&#188;l&#226;&#710;&#8217;1. <br />  <br />                                            Expression T. brucei INO1 E. coli <br />                                                            The INO1 ORF amplified TbINO1-TOPO plasmid Taq polymerase using <br />                                                            primers 5&#226;&#8364;&#178;-CGG GATCCCGGGATGCCAGCCGTCCGTACGAAAAGT-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            CCTTTCCTTCGCGGCGTGGGAAGTAGGCCGCTCGA GCGGC-3&#226;&#8364;&#178; resulting single <br />                                                            band ~1.5 kb. The amplified product purified using QIAquick PCR purification kit <br />                                                            (Qiagen) ligated pBAD TOPO TA (Invitrogen) vector sequence confirmed. <br />                                                            This construct transformed TOP10 competent cells (Invitrogen) expression. A <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                          Page 11 <br />  <br />  <br />                                                            single colony used inoculate 200 ml Luria&#226;&#8364;&#8220;Bertani media containing 100 &#206;&#188;g ml&#226;&#710;&#8217;1 <br />                                                            ampicillin grown 37&#194;&#176;C OD (600 nm) 0.5. The cells induced 0.2% <br />                                                            arabinose grown 37&#194;&#176;C 3 h, cells harvested by <br />                                                            centrifugation stored &#226;&#710;&#8217;20&#194;&#176;C. <br />  <br />                                            Purification recombinant INO1 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Frozen pellets resuspended lysis buffer (20 mM Tris pH 8.0, 300 mM NaCl) and <br />                                                            incubated ice 0.2 mg ml&#226;&#710;&#8217;1 lysozyme (Sigma) 30 min. Cells disrupted by <br />                                                            sonication (3 &#195;&#8212; 30 s) ice supernatant cleared centrifugation (50 000 g, 30 <br />                                                            min, 4&#194;&#176;C). This supernatant filtered (0.45 &#206;&#188;m) prior loading Hi-Trap Chelating <br />                                                            Sepharose HP column (Amersham) charged Ni2+ equilibrated with <br />                                                            lysis buffer. The column connected AKTA FPLC (Amersham) bound protein <br />                                                            eluted buffer containing 0.5 M imidazole, using linear gradient 0&#226;&#8364;&#8220;0.25 M, a <br />                                                            flow rate 1 ml min&#226;&#710;&#8217;1 collecting 1 ml fractions. Fractions containing TbINO1 <br />                                                            protein pooled dialysed 20 mM Tris pH 8.0, 0.05 mM DTT 24 h with <br />                                                            regular buffer changes. The purity recombinant INO1 checked SDS-PAGE <br />                                                            (10% gel). The dialysed sample adjusted 15% glycerol stored &#226;&#710;&#8217;80&#194;&#176;C. <br />  <br />                                            Enzyme assays <br />                                                            The assay used INO1 activity based described Eisenberg and <br />                                                            Parthasarathy, (1987). Briefly, standard assay reaction consisted 50 mM Tris-Ac pH <br />                                                            8.0, 5 mM glucose 6-phosphate, 1 mM DTT, 1 mM NAD+, 2 mM NH4Ac 5 &#206;&#188;g of <br />                                                            purified protein total volume 150 &#206;&#188;l. The assay incubated 37&#194;&#176;C 1 h, the <br />                                                            assay stopped heating 100&#194;&#176;C 5 min. To determine glucose 6- <br />                                                            phosphate converted inositol-1-phosphate, 1.25 mU IMPase from <br />                                                            bovine brain (Sigma) 4 mM MgCl2 added total 50 &#206;&#188;l incubated 37&#194;&#176;C <br />                                                            1 h; assay stopped heating 100&#194;&#176;C 5 min. As IMPase negligible <br />                                                            activity glucose 6-phosphate, resulted quantitative release phosphate <br />                                                            inositol-1-phosphate formed solely action TbINO1. For <br />                                                            assay substrate blank enzyme blank incubated IMPase account <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            free phosphates present sample non-specific activity of <br />                                                            IMPase. The free phosphate determined malachite green method Itaya Ui <br />                                                            (1966) compared standard curve KH2PO4. Specific activity defined units <br />                                                            (U) milligram protein, 1 U enzyme catalysing formation of <br />                                                            1 nmole inositol-1-phosphate min 37&#194;&#176;C. <br />  <br />                                            Southern Northern blots <br />                                                            The INO1 ORF PCR-amplified using primers described previously ligation <br />                                                            pLew vectors gel-purified QIAquick gel extraction kit (Qiagen). This <br />                                                            fragment labelled fluorescein (Gene ImagesRandom prime module, <br />                                                            Amersham) Southern blotting [&#206;&#177;-32P]-dCTP (RediprimeII random prime labelling <br />                                                            system, Amersham) Northern blotting. <br />  <br />                                                            For Southern blots genomic T. brucei DNA (2 &#206;&#188;g) digested various restriction <br />                                                            enzymes, digestion products separated 0.8% agarose gel transferred to <br />                                                            Hybon-N membrane (Amersham). The membrane hybridized overnight ULTRA- <br />                                                            HYB (Ambion) 42&#194;&#176;C fluorescein labelled INO1 ORF probe. Stringency washes <br />                                                            carried 42&#194;&#176;C, consisted washes low stringency 15 min (2&#195;&#8212; <br />                                                            SSC, 0.1% SDS) washes high stringency 15 min (0.2&#195;&#8212; SSC, 0.1% <br />                                                            SDS). Bound probe detected using CDP-STAR detection module (Amersham) and <br />                                                            autoradiography. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 12 <br />  <br />  <br />                                                            For Northern blots total RNA purified using RNeasy Mini Kit (Qiagen) separated <br />                                                            formaldehyde gel transferred Hybon N+ (Amersham). Hybridization conditions <br />                                                            stringency washes described Southern blotting, using 32P-labelled INO1 <br />                                                            ORF probe. Bound probe detected autoradiography. <br />  <br />                                            Cultivation genetic modification T. brucei <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Bloodstream-form T. brucei strain 427, previously modified express both <br />                                                            T7 polymerase tetracycline repressor protein (Wirtz et al., 1999), referred here <br />                                                            wild-type cells convenience. Cells grown HMI-9 media supplemented with <br />                                                            G418 (2.5 &#206;&#188;g ml&#226;&#710;&#8217;1), 37&#194;&#176;C 5% CO2 described (Wirtz et al., 1999; Milne <br />                                                            et al., 2001; Chang et al., 2002; Roper et al., 2002). Transformation conditions and <br />                                                            subsequent drug selection described (Wirtz et al., 1999; Milne et al., <br />                                                            2001; Chang et al., 2002; Roper et al., 2002). For experiments requiring tetracycline-free <br />                                                            conditions, Tet approved fetal calf serum (Clonetech) used. When tetracycline <br />                                                            added media, final concentration 1 &#206;&#188;g ml&#226;&#710;&#8217;1 used. For tetracycline-free <br />                                                            experiments, cells washed times tetracycline-free HMI-9 resuspended in <br />                                                            tetracycline-free media 5 &#195;&#8212; 104 cells ml&#226;&#710;&#8217;1. Cells counted day and <br />                                                            passaged density 2 &#195;&#8212; 106 3 &#195;&#8212; 106 (normally every <br />                                                            second day). <br />  <br />                                            Subcellular localization studies <br />                                                            Mid-log T. brucei INO1-HATi bloodstream-form cells grown the <br />                                                            presence tetracycline 2 days harvested centrifugation (800 g, 10 min) and <br />                                                            used immunofluorescence differential centrifugation. For immunofluorescence, <br />                                                            cells fixed 4% paraformaldehyde (4&#194;&#176;C, overnight). An aliquot fixed <br />                                                            cells placed poly lysine slide allowed air dry. Cells rehydrated with <br />                                                            PBS washed PBS-glycine (0.1 M) permeablization Triton X-100 <br />                                                            (0.1%). The cells blocked 1% BSA PBS incubation primary <br />                                                            monoclonal antibody; rat anti-HA (Roche) secondary antibody; FITC-conjugated <br />                                                            rabbit anti-rat immunoglobulins (DakoCytomation). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            For differential centrifugation, cells (1 &#195;&#8212; 108) lysed ice-cold water (100 &#206;&#188;l) <br />                                                            containing leupeptin (2 &#206;&#188;g ml&#226;&#710;&#8217;1) TLCK (0.1 &#206;&#188;M) 15 min ice, mechanical <br />                                                            disruption cells addition glass beads (425&#226;&#8364;&#8220;600 &#206;&#188;m, Sigma), vortexing <br />                                                            vigorously 3 &#195;&#8212; 30 s, 2 min intervals ice. Glass beads removed by <br />                                                            centrifugation (500 g, 5 min). Differential centrifugation described previously (Sarkar <br />                                                            et al., 2003). Briefly, unbroken cells nuclei removed centrifugation 1000 g <br />                                                            10 min. The supernatant removed centrifuged 14 500 g 10 min remove <br />                                                            large granular fraction. The resulting supernatant removed centrifuged 140 <br />                                                            000 g 1 h, supernatant contained cytosolic fraction pellet contained small <br />                                                            granular fraction. All pellets resuspended trypanosome dilution buffer (TDB) (25 <br />                                                            mM KCl, 400 mM NaCl, 5 mM MgSO4, 100 mM Na2HPO4, NaH2PO4, 100 mM glucose) <br />                                                            equal volume cytosolic fraction. Proteins separated SDS-PAGE <br />                                                            transferred ECL-Nylon membrane Western blotting. After blocking overnight in <br />                                                            PBS-5% skim milk powder, protein detected using primary monoclonal antibody; <br />                                                            rat anti-HA (Roche) secondary antibody; horseradish peroxidase-conjugated rabbit anti- <br />                                                            rat immunoglobulins (Jackson) ECL western detection reagents. <br />  <br />                                            In vivo T. brucei metabolic labelling <br />                                                            For metabolic labelling, 2 &#195;&#8212; 107 mid-log cells centrifuged (800 g, 10 min), <br />                                                            washed glucose-free minimal essential media, resuspended same <br />                                                            media final concentration 1 &#195;&#8212; 107 cells ml&#226;&#710;&#8217;1. Cells labelled 1 h (pulse) 2 h <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                          Page 13 <br />  <br />  <br />                                                            (pulse chase) 37&#194;&#176;C 50 &#206;&#188;Ci ml&#226;&#710;&#8217;1 D-[2-3H]-glucose (30 Ci mmol&#226;&#710;&#8217;1, ARC), D- <br />                                                            [2-3H]-mannose (14 Ci mmol&#226;&#710;&#8217;1, Amersham) D-[2-3H]-inositol (30 Ci mmol&#226;&#710;&#8217;1, <br />                                                            Amersham) shaking water bath. For pulse-chase experiments, 1 h labelling an <br />                                                            equal volume normal media (HMI-9) added labelling allowed continue for <br />                                                            hour. The cells collected centrifugation (800 g, 10 min) samples taken <br />                                                            lipid protein analysis. Lipids extracted using chloroform:methanol:water <br />                                                            (10:10:3 v/v) 1 h, supernatant removed pellet re-extracted with <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            chloroform:methanol (2:1 v/v) 1 h. The supernantants pooled dried before <br />                                                            samples desalted using butanol/water partioning. Lipids separated HPTLC <br />                                                            using silica 60 HPTLC plates chloroform:methanol:water (10:10:3 v/v) solvent. <br />                                                            Radiolabelled lipids detected fluorography &#226;&#710;&#8217;70&#194;&#176;C, spraying with <br />                                                            En3hance&#226;&#8222;&#162; using Kodak XAR-5 film intensifying screen, when <br />                                                            quantification required HPTLC plates imaged using Fuji FLA-2000 <br />                                                            phosphorimager. To analyse proteins SDS-PAGE sample buffer added cell pellet <br />                                                            heated 95&#194;&#176;C 5 min. Proteins separated 10% SDS-PAGE gel and <br />                                                            visualized Coomassie blue staining. To detect [3H]-labelled proteins destained gel <br />                                                            soaked En3hance&#226;&#8222;&#162; (NEN) 30 min, washed water twice, soaked 10% <br />                                                            glycerol dried exposed XAR-5 film &#226;&#710;&#8217;70&#194;&#176;C. Small-scale sVSG purification and <br />                                                            PNGase F (Calbiochem) performed described previously (Jones et al., 2005). <br />  <br />                                                            When labelling [35S]-methionine, 1 &#195;&#8212; 107 mid-log cells collected by <br />                                                            centrifugation, washed methionine-free minimal essential media resuspended the <br />                                                            media final concentration 1 &#195;&#8212; 107 cells ml&#226;&#710;&#8217;1. The cells labelled 30 min <br />                                                            20 &#206;&#188;Ci [35S]-methionine (MP Biomedicals, 1175 Ci mmol&#226;&#710;&#8217;1) 37&#194;&#176;C. To quench the <br />                                                            labelling, cells diluted 20 ml cold TDB containing 1 mM L-methionine and <br />                                                            centrifuged (800 g, 10 min, 4&#194;&#176;C). After supernatant removed, cells were <br />                                                            resuspended TDB equal volume 2&#195;&#8212; SDS-PAGE sample buffer added prior <br />                                                            heating 100&#194;&#176;C 5 min. Proteins separated 10% SDS-PAGE gel and <br />                                                            visualized Coomassie blue staining. To detect [35S]-labelled proteins destained gel <br />                                                            soaked En3hance&#226;&#8222;&#162; (NEN) 30 min, washed water twice, soaked 10% <br />                                                            glycerol dried. The dried gel exposed XAR-5 film overnight &#226;&#710;&#8217;70&#194;&#176;C. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Enzymatic digests chemical characterization radiolabelled lipid/glycolipid species <br />                                                            Digestion GPI-specific phospholipase D (GPI-PLD, Glyco) PI-specific <br />                                                            phospholipase C (PI-PLC, Glyco), deamination, base hydrolysis HF treatment as <br />                                                            described previously (G&#195;&#188;ther et al., 1994; G&#195;&#188;ther Ferguson, 1995; Smith et al., 1996). <br />                                                            After deamination, reactions subjected butanol/water partitioning total [3H] cpm <br />                                                            fractions determined scintillation spectrometry using Beckman LS6000E <br />                                                            formula 989 scintillation fluid (Packard Biosciences). HPTLC detection the <br />                                                            described vivo T. brucei metabolic labelling. <br />  <br />                                            Inositol analysis <br />                                                            Mid-log cells collected centrifugation (800 g, 10 min) washed TDB stored <br />                                                            &#226;&#710;&#8217;20&#194;&#176;C. Lipids extracted samples addition 500 &#206;&#188;l chloroform <br />                                                            methanol mixture (2:1 v/v) incubated room temperature 1 h. The supernatant was <br />                                                            removed pellet re-extracted chloroform:methanol:water mixture (1:2:0.8 v/v). <br />                                                            Pooled supernatants dried nitrogen prior desalting biphasic partitioning <br />                                                            using 2:1 butanol:water (v/v). An internal standard D6-myo-inositol added samples <br />                                                            prior hydrolysis strong acid (6 M HCl, 110&#194;&#176;C), derivitization TMS analysis <br />                                                            GC-MS, according method Ferguson (1993). myo-Inositol quantified and <br />                                                            means separate analyses determined. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                    Page 14 <br />  <br />  <br />                                            Supplementary Material <br />                                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                            Acknowledgments <br />                                                            This work supported Wellcome Trust Senior Research Fellowship 067441. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            References <br />                                                            Besra, GS.; Chatterjee, D. Lipids carbohydrates Mycobacterium tubercolosis. In: Broom, BR., <br />                                                               editor. Tuberculosis: Pathogenesis, Protection Control. American Society Microbiology; <br />                                                               Washington, DC: 1994. p. 285-306. <br />                                                            Buxbaum LU, Milne KG, Werbovetz KA, Englund PT. Myristate exchange Trypanosoma <br />                                                               brucei variant surface glycoprotein. Proc Natl Acad Sci USA. 1996; 93:1178&#226;&#8364;&#8220;1183. [PubMed: <br />                                                               8577736] <br />                                                            Chang T, Milne KG, G&#195;&#188;ther MLS, Smith TK, Ferguson MAJ. Cloning Trypansoma brucei and <br />                                                               Leishmania major genes encoding GlcNAc-phoshatidylinositol de-N-acetylase of <br />                                                               glycosylphosphatidylinositol biosynthesis essential African sleeping sickness parasite. <br />                                                               J Biol Chem. 2002; 277:50176&#226;&#8364;&#8220;50182. [PubMed: 12364327] <br />                                                            Chen R, Walter EI, Parker G, Lapurga JP, Millan JL, Ikehara Y, et al. Mammalian <br />                                                               glycophosphatidylinositol anchor transfer proteins posttransfer deacylation. Proc Natl Acad <br />                                                               Sci USA. 1998; 95:9512&#226;&#8364;&#8220;9517. [PubMed: 9689111] <br />                                                            Cross GAM. Antigenic variation trypanosomes: secrets surface slowly. Bioessays. 1996; 18:283&#226;&#8364;&#8220; <br />                                                               287. [PubMed: 8967896] <br />                                                            Donahue TF, Henry SA. Myo-inositol-1-phosphate synthase. J Biol Chem. 1981; 256:7077&#226;&#8364;&#8220;7085. <br />                                                               [PubMed: 7016881] <br />                                                            Eisenberg FJ, Parthasarathy R. Measurement biosynthesis myo-inositol glucose-6- <br />                                                               phosphate. Methods Enzymol. 1987; 141:127&#226;&#8364;&#8220;143. [PubMed: 3600356] <br />                                                            Ferguson, MAJ. GPI membrane anchors: isolation analysis. In: Fukuda, M.; Kobata, A., editors. <br />                                                               Glycobiology: A Practical Approach. IRL Press; Oxford: 1993. p. 349-383. <br />                                                            Ferguson MAJ. The structure, biosynthesis functions glycosylphosphatidylinositol anchors, and <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               contributions trypanosome research. J Cell Sci. 1999; 112:2799&#226;&#8364;&#8220;2809. [PubMed: 10444375] <br />                                                            Ferguson MAJ, Brimacombe JS, Brown JR, Crossman A, Dix A, Field RA, et al. The GPI biosynthetic <br />                                                                 pathway therapeutic target African sleeping sickness. Biochim Biophys Acta. 1999; <br />                                                                 1455:327&#226;&#8364;&#8220;340. [PubMed: 10571022] <br />                                                            Flury I, Benachour A, Conzelmann A. L031c belongs novel family membrane proteins involved <br />                                                                 transfer ethanolaminephosphate core structure glycosylphosphatidylinositol <br />                                                                 anchors yeast. J Biol Chem. 2000; 275:24458&#226;&#8364;&#8220;24465. [PubMed: 10823837] <br />                                                            Gerold P, Jung N, Azzouz N, Freiberg N, Kobe S, Schwarz RT. Biosynthesis of <br />                                                                 glycosylphosphatidylinositols Plasmodium falciparum cell-free incubation system: inositol <br />                                                                 acylation needed mannosylation glycosylphosphatidylinositols. Biochem J. 1999; <br />                                                                 344:731&#226;&#8364;&#8220;738. [PubMed: 10585859] <br />                                                            Guan G, Dai P, Shechter I. cDNA cloning gene expression analysis human myo-inositol 1- <br />                                                                 phosphate synthase. Arch Biochem Biophys. 2003; 417:251&#226;&#8364;&#8220;259. [PubMed: 12941308] <br />                                                            G&#195;&#188;ther MLS, Ferguson MAJ. The role inositol acylation inositol deacylation GPI <br />                                                                 biosynthesis Trypanosoma brucei. EMBO J. 1995; 14:3080&#226;&#8364;&#8220;3093. [PubMed: 7621823] <br />                                                            G&#195;&#188;ther MLS, Masterson WJ, Ferguson MAJ. The effects phenylmethylsulfonyl fluoride inositol- <br />                                                                 acylation fatty acid remodelling African trypanosomes. J Biol Chem. 1994; 269:18694&#226;&#8364;&#8220; <br />                                                                 18701. [PubMed: 7518442] <br />                                                            Haites RE, Morita YS, McConville MJ, Billman-Jacobe H. Function phosphatidylinositol in <br />                                                                 mycobacteria. J Biol Chem. 2005; 280:10981&#226;&#8364;&#8220;10987. [PubMed: 15634688] <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                  Page 15 <br />  <br />  <br />                                                            Heise N, Raper J, Buxbaum LU, Peranovich TM, Almeida ML. Identification complete <br />                                                                 precursors glycosylphosphatidylinositol protein anchors Trypanosoma cruzi. J Biol <br />                                                                 Chem. 1996; 271:16877&#226;&#8364;&#8220;16887. [PubMed: 8663209] <br />                                                            Henry SA, Atkinson KD, Kolat AL, Culbertson MR. Growth metabolism inositol-starved <br />                                                                 Saccharomyces cerevisiae. J Bacteriol. 1977; 68:2888&#226;&#8364;&#8220;2898. <br />                                                            Hirose S, Prince GM, Sevlever D, Ravi L, Rosenberry TL, Ueda E, Medof ME. Characterization of <br />                                                                 putative glycoinositol phospholipid anchor precursors mammalian cells. Localization of <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                 phosphoethanolamine. J Biol Chem. 1992; 267:16968&#226;&#8364;&#8220;16974. [PubMed: 1380957] <br />                                                            Ilg T. Generation myo-inositol-auxotrophic Leishmania mexicana mutants targeted replacement <br />                                                                 myo-inositol-1-phosphate synthase gene. Mol Biochem Parasitol. 2002; 120:151&#226;&#8364;&#8220;156. <br />                                                                 [PubMed: 11849714] <br />                                                            Itaya K, Ui M. A new micromethod colorimetric determination inorganic phosphate. Clinica <br />                                                                 Chimica Acta. 1966; 14:361&#226;&#8364;&#8220;366. <br />                                                            Johnson MD, Sussex IM. 1l-myo-inositol 1-phosphate synthase Arabidopsis thaliana. Plant <br />                                                                 Physiol. 1995; 107:613&#226;&#8364;&#8220;619. [PubMed: 12228386] <br />                                                            Jones DC, Mehlert AM, G&#195;&#188;ther ML, Ferguson MAJ. Deletion glucosidase II gene in <br />                                                                 Trypanosoma brucei reveals novel N-glycosylation mechanisms biosynthesis variant <br />                                                                 surface glycoprotein. J Biol Chem. 2005; 280:35929&#226;&#8364;&#8220;35942. [PubMed: 16120601] <br />                                                            Ju S, Shaltiel G, Shamir A, Agam G, Greenberg ML. Human 1d-myo-inositol 3-phosphate synthase is <br />                                                                 functional yeast. J Biol Chem. 2004; 279:21759&#226;&#8364;&#8220;21765. [PubMed: 15024000] <br />                                                            Kinoshita T, Inoue N. Dissecting manipulating pathway glycosylphosphatidylinositol- <br />                                                                 anchor biosynthesis. Curr Opin Chem Biol. 2000; 4:632&#226;&#8364;&#8220;638. [PubMed: 11102867] <br />                                                            Krieger S, Schwarz W, Ariyanayagam MR, Fairlamb AH, Krauth-Siegel RL, Clayton C. <br />                                                                 Trypanosomes lacking trypanothione reductase avirulent increased sensitivity to <br />                                                                 oxidative stress. Mol Microbiol. 2000; 35:542&#226;&#8364;&#8220;552. [PubMed: 10672177] <br />                                                            Lohia A, Hait NC, Majumder AL. 1-myo-Inositol 1-phosphate synthase Entamoeba histolytica. <br />                                                                 Mol Biochem Parasitol. 1999; 98:67&#226;&#8364;&#8220;79. [PubMed: 10029310] <br />                                                            Macedo CS, Shams-Eldin H, Smith TK, Schwarz RT, Azzouz N. Inhibitors glycosyl- <br />                                                                 phosphatidylinositol anchor biosynthesis. Biochimie. 2003; 85:465&#226;&#8364;&#8220;472. [PubMed: 12770785] <br />                                                            McConville MJ, Menon AK. Recent developments cell biology biochemistry of <br />                                                                 glycosylphosphatidylinositol anchors. Mol Membr Biol. 2000; 17:1&#226;&#8364;&#8220;16. [PubMed: 10824734] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Majee M, Maitra S, Dastidar KD, Pattnaik S, Chatterjee A, Hait NC, et al. A novel salt-tolerant 1-myo- <br />                                                                 inositol-1-phosphate synthase form Porteresia coarctata (Roxb.) Tateoka, halophytic wild rice. J <br />                                                                 Biol Chem. 2004; 279:28539&#226;&#8364;&#8220;28552. [PubMed: 15016817] <br />                                                            Majumder AL, Johnson MD, Henry SA. Chapter XIX 1l-myo-Inositol-1-phosphate synthase. Biochim <br />                                                                 Biophys Acta. 1997; 1348:245&#226;&#8364;&#8220;256. [PubMed: 9370339] <br />                                                            Majumder AL, Chatterjee A, Dastidar KD, Majee M. Diversification evolution 1-myo-inositol <br />                                                                 1-phosphate synthase. FEBS Lett. 2003; 553:3&#226;&#8364;&#8220;10. [PubMed: 14550537] <br />                                                            Martin KL, Smith TK. Phosphatidylinositol synthesis essential bloodstream form T. brucei. <br />                                                                 Biochem J. 2006; 396:287&#226;&#8364;&#8220;295. [PubMed: 16475982] <br />                                                            Masterson WJ, Doering TL, Hart GW, Englund PT. A novel pathway glycan assembly: <br />                                                                 biosynthesis glyosyl-phosphatidylinositol anchor trypanosome variant surface <br />                                                                 glycoprotein. Cell. 1989; 56:793&#226;&#8364;&#8220;800. [PubMed: 2924349] <br />                                                            Masterson WJ, Raper J, Doering TL, Hart GW, Englund PT. Fatty acid remodeling: novel reaction <br />                                                                 sequence biosynthesis trypanosome glycosyl-phosphatidylinositol membrane anchors. <br />                                                                 Cell. 1990; 62:73&#226;&#8364;&#8220;80. [PubMed: 1694728] <br />                                                            Menon AK, Major S, Ferguson MAJ, Duszenko M, Cross GAM. Candidate glycophospholipid <br />                                                                 precursor glycosylphosphatidylinositol membrane anchor Trypanosoma brucei variant <br />                                                                 surface glycoproteins. J Biol Chem. 1988; 263:1970&#226;&#8364;&#8220;1977. [PubMed: 3339000] <br />                                                            Menon AK, Mayor S, Schwarz RT. Biosynthesis glycosyl-phosphatidylinositol lipids in <br />                                                                 Trypanosoma brucei: involvement mannosyl-phosphoryldolichol mannose donor. EMBO <br />                                                                 J. 1990a; 9:4249&#226;&#8364;&#8220;4258. [PubMed: 2148289] <br />                                                            Menon AK, Schwarz RT, Mayor S, Cross GAM. Cell-free synthesis glycosyl-phosphatidylinositol <br />                                                                 precursors glycolipid membrane anchor variant surface glycoproteins. Structural <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                 Page 16 <br />  <br />  <br />                                                                characterization putative biosynthetic intermediates. J Biol Chem. 1990b; 265:9033&#226;&#8364;&#8220;9042. <br />                                                                [PubMed: 1693147] <br />                                                            Milne KG, G&#195;&#188;ther MLS, Ferguson MAJ. Acyl-CoA binding protein essential bloodstream form T. <br />                                                                brucei. Mol Biochem Parasitol. 2001; 112:301&#226;&#8364;&#8220;304. [PubMed: 11223138] <br />                                                            Morita YS, Acosta-Serrano A, Buxbaum LU, Englund PT. Glycosyl phosphatidylinositol <br />                                                                myristoylation African trypanosomes. New intermediates pathway fatty acid <br />                                                                remodeling. J Biol Chem. 2000a; 275:14147&#226;&#8364;&#8220;14154. [PubMed: 10799491] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Morita, YS.; Acosta-Serrano, A.; Englund, PT. The biosynthesis GPI enchors. In: Ernst, P.; Sinay, <br />                                                                P.; Hart, G., editors. Oligosaccharides Chemistry Biology &#226;&#8364;&#8220; A Comprehensive Handbook. <br />                                                                Wiley-VCH; Weinheim, Germany: 2000b. p. 417-433. <br />                                                            Movahedzadeh F, Smith DA, Norman RA, Dinadayala P, Murray-Rust J, Russell DG, et al. The <br />                                                                Mycobacterium tuberculosis ino1 gene essential growth virulence. Mol Microbiol. 2004; <br />                                                                51:1003&#226;&#8364;&#8220;1014. [PubMed: 14763976] <br />                                                            Nagamune K, Nozaki T, Maeda Y, Ohishi K, Fukuma T, Hara T, et al. Critical roles of <br />                                                                glycosylphosphatidylinositol Trypanosoma brucei. Proc Natl Acad Sci USA. 2000; 97:10336&#226;&#8364;&#8220; <br />                                                                10341. [PubMed: 10954751] <br />                                                            Norman RA, McAlister MSB, Murray-Rust J, Movahedzadeh F, Stoker NG, McDonald NQ. Crystal <br />                                                                structure inositol 1-phosphate synthase Mycobacterium tuberculosis, key enzyme in <br />                                                                phosphatidylinositol synthesis. Structure. 2002; 10:393&#226;&#8364;&#8220;402. [PubMed: 12005437] <br />                                                            Park S, Kim J. Characterization recombinant Drosophila melanogaster myo-inositol-1-phosphate <br />                                                                synthase expressed Escherichia coli. J Microbiol. 2004; 42:20&#226;&#8364;&#8220;24. [PubMed: 15357287] <br />                                                            Puoti A, Conzelmann A. Characterization abnormal free glycophosphatidylinositols accumulating in <br />                                                                mutant lymphoma cells classes B, E, F, H. J Biol Chem. 1993; 268:7215&#226;&#8364;&#8220;7224. [PubMed: <br />                                                                8463257] <br />                                                            Ralton JE, McConville MJ. Delineation pathways glycosylphosphatidylinositol biosynthesis <br />                                                                Leishmania mexicana. Precursors different pathways assembled distinct pools of <br />                                                                phosphatidylinositol undergo fatty acid remodeling. J Biol Chem. 1998; 273:4245&#226;&#8364;&#8220;4257. <br />                                                                [PubMed: 9461623] <br />                                                            Ralton JE, Mullin KA, McConville MJ. Intracellular trafficking glycosylphosphatidylinositol (GPI)- <br />                                                                anchored proteins free GPIs Leishmania mexicana. Biochem J. 2002; 363:365&#226;&#8364;&#8220;375. <br />                                                                [PubMed: 11931667] <br />                                                            Roper JR, G&#195;&#188;ther MLS, Milne KG, Ferguson MAJ. Galatose metabolism essential African <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                sleeping sickness parasite Trypanosoma brucei. Proc Natl Acad Sci USA. 2002; 99:5884&#226;&#8364;&#8220;5889. <br />                                                                [PubMed: 11983889] <br />                                                            Roper JR, G&#195;&#188;ther MLS, MacRae JI, Prescott AR, Hallyburton I, Acosta-Serrano A, Ferguson MAJ. <br />                                                                The suppression galactose metabolism procyclic form Trypanosoma brucei causes cessation <br />                                                                cell growth alters procyclin glycoprotein structure copy number. J Biol Chem. 2005; <br />                                                                280:19728&#226;&#8364;&#8220;19736. [PubMed: 15767252] <br />                                                            Sarkar M, Hamilton CJ, Fairlamb AH. Properties phosphoenolpyruvate mutase, enzyme in <br />                                                                aminoethylphophonate biosynthetic pathway Trypanosoma cruzi. J Biol Chem. 2003; <br />                                                                278:22703&#226;&#8364;&#8220;22708. [PubMed: 12672809] <br />                                                            Smith TK, Cottaz S, Brimacombe JS, Ferguson MAJ. Substrate specifities dolicol phosphate <br />                                                                mannose: glucosaminyl phospatidylinositol a1-4-mannosyltransferase the <br />                                                                glycosylphosphatidylinositol biosynthetic pathway African trypanosomes. J Biol Chem. 1996; <br />                                                                271:6476&#226;&#8364;&#8220;6482. [PubMed: 8626449] <br />                                                            Smith TK, Milne FC, Sharma DK, Crossman A, Brimacombe JS, Ferguson MAJ. Early steps in <br />                                                                glycosylphosphatidylinositol biosynthesis Leishmania major. Biochem J. 1997; 326:393&#226;&#8364;&#8220;400. <br />                                                                [PubMed: 9291110] <br />                                                            Smith TK, Crossman A, Brimacombe JS, Ferguson MAJ. Chemical validation GPI biosynthesis a <br />                                                                drug target African sleeping sickness. EMBO J. 2004; 23:4701&#226;&#8364;&#8220;4708. [PubMed: <br />                                                                15526036] <br />                                                            Striepen B, Dubremetz J-F, Schwarz RT. Glucosylation glycosylphosphatidylinositol membrane <br />                                                                anchors: identification uridine diphosphate-glucose direct donor chain <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 17 <br />  <br />  <br />                                                                modification Toxoplasma gondii using carbohydrate analogues. Biochemistry. 1999; 38:1478&#226;&#8364;&#8220; <br />                                                                1487. [PubMed: 9931013] <br />                                                            Sutterlin C, Escribano MV, Gerold P, Maeda Y, Mazon MJ, Kinoshita T, et al. Saccharomyces <br />                                                                cerevisiae GPI10, functional homologue human PIG-B, required for <br />                                                                glycosylphosphatidylinositol-anchor synthesis. Biochem J. 1998; 332:153&#226;&#8364;&#8220;159. [PubMed: <br />                                                                9576863] <br />                                                            Wirtz E, Leal S, Ochatt C, Cross GAM. A tightly regulated inducible expression for <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                conditional gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol <br />                                                                Biochem Parasitol. 1999; 99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                     Page 18 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 1. <br />                                                            A. CLUSTALW alignment prediced amino acid sequence Trypanosoma brucei (Tb, <br />                                                            AJ866770) INO1 Leishmania mexicana (Lm, AJ344543), Saccharomyces <br />                                                            cerevisiae (Sc, A30902), Homo sapiens (Hs, AAF26444) Mycobacterium tuberculosis <br />                                                            (Mt, P71703). Identical residues shown reverse type black background, and <br />                                                            conserved residues shown reverse type grey background. The NAD+-binding <br />                                                            motif GXGGXXG (residues 95&#226;&#8364;&#8220;101) identified previously (Majumder et al., 2003) shown <br />                                                            solid line. <br />                                                            B. An unrooted phylogenetic tree deduced amino acid sequences INO1 proteins, <br />                                                            showing relationship protozoan parasites (T. brucei, T. cruzi, L. <br />                                                            mexicana, P. falciparum) yeast, plant, bacterial human origins. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 19 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 2. <br />                                                            Expression purification recombinant TbINO1 E. coli. <br />                                                            A. TbINO1 cloned expression vector pBAD TA (C-terminal hexa-His tag) and <br />                                                            transformed TOP10 E. coli competent cells, production recombinant TbINO1 <br />                                                            induced 0.2% arabinose. After cell disruption, soluble TbINO1 purified by <br />                                                            affinity chromatography using chelating column charged Ni2+, proteins were <br />                                                            separated SDS-PAGE gel stained Coomassie brilliant blue. Lane 1, total E. <br />                                                            coli cellular protein induction; lane 2, soluble E. coli protein induction, was <br />                                                            loaded affinity chromatography column; lane 3, purified recombinant TbINO1 after <br />                                                            affinity chromatography; lane 4, purified recombinant TbINO1 dialysis. <br />                                                            B. Kinetics recombinant TbINO1 glucose 6-phosphate. Enzyme activity was <br />                                                            measured described Experimental procedures, NAD+ concentration held constant <br />                                                            (1 mM) glucose 6-phosphate concentration varied. Insert shows Lineweaver-Burk plot <br />                                                            data. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                     Page 20 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 3. <br />                                                            Construction INO1 conditional double knockout cell line. <br />                                                            A. Schematic representation construction strategy used. (i) One allele INO1 was <br />                                                            replaced puromycin resistance gene (PAC) homologous recombination generating <br />                                                            &#206;&#8221;INO1::PAC cell line. (ii) A tetracycline-inducible ectopic copy INO1 introduced <br />                                                            rDNA, generating INO1-mycTi &#206;&#8221;INO1::PAC cell line. (iii) While tetracycline <br />                                                            induced expression ectopic copy, remaining allele replaced by <br />                                                            hygromycin resistance gene homologous recombination, resulting conditional double <br />                                                            knockout cell line INO1-mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG. <br />                                                            B. Confirmation genotype T. brucei INO1 conditional double knockout cell line. <br />                                                            Southern blot analysis NcoI-digested genomic DNA (~2 &#206;&#188;g) wild-type T. brucei <br />                                                            cells (lane 1), &#206;&#8221;INO1::PAC (lane 2), INO1-mycTi &#206;&#8221;INO1::PAC (lane 3) INO1- <br />                                                            mycTiINO1 &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG (lane 4); INO1 ORF probe shows allelic <br />                                                            TbINO1 1.9 kb ectopic copy TbINO1-mycTi ~7 kb. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                      Page 21 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 4. <br />                                                            Growth curves Northern blot TbINO1 conditional double knockout cell line. <br />                                                            A&#226;&#8364;&#8220;D. Cells washed tetracycline-free HMI-9, transferred tetracycline-free <br />                                                            media counted daily. Growth curves shown wild-type cells medium (A), <br />                                                            TbINO1 conditional double knockout added tetracycline (B), TbINO1 conditional <br />                                                            double knockout tetracycline (C) TbINO1 conditional double knockout without <br />                                                            tetracycline, added myo-inositol (100 mM) (D). <br />                                                            E. A Northern blot total RNA procyclic cells (lane 1), wild-type cells (lane 2), <br />                                                            TbINO1 conditional double knockout tetracycline day 0 (lane 3), TbINO1 <br />                                                            conditional double knockout tetracycline day 2 (lane 4) TbINO1 conditional <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                        Page 22 <br />  <br />  <br />                                                            double knockout tetracycline day 12 (lane 5). The blot probed with <br />                                                            TbINO1 ORF &#206;&#178;-tubulin, detailed Experimental procedures. <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                        Page 23 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 5. <br />                                                            In vivo labelling wild-type cells. <br />                                                            A. Wild-type cells pulse pulse chase labelled lipids extracted, <br />                                                            desalted, separated HPTLC detected fluorography. Lane 1, [3H]-inositol pulse; <br />                                                            lane 2, [3H]-inositol pulse chase; lane 3, [3H]-mannose pulse; lane 4, [3H]-mannose pulse <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            chase; lane 5, [3H]-glucose pulse; lane 6, [3H]-glucos <br /> </body></html>