<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>SCFMet30-mediated control of the transcriptional <br /> activator Met4 is required for the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />                                pounds (reviewed in Thomas and Surdin-Kerjan, 1997). <br />                                In addition <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> the biosynthesis of the two sulfur amino <br />                                   acids methionine and cysteine, reduced sulfur atoms are <br />                                                                          needed to synthesize S-adenosylmethionine (AdoMet), a <br />            molecule central to general metabolism. AdoMet is indeed <br />       next to ATP for the number of reactions in which a <br />  <br />            biological compound is used: AdoMet is the main donor <br />    of methyl groups for methylation of nucleic acids, proteins <br />                             and lipids, serves as a precursor for the biosynthesis of <br />                                                   the polyamines and is also the substrate for numerous <br />                                          reactions in vitamin biosynthesis and nucleotide modi- <br />               fication. <br />                                                                             Sulfur flux through the sulfate assimilation pathway <br /> Progression through the cell cycle requires the coord-                   and the metabolic arms specific for the methionine and <br /> ination of basal metabolism with the cell cycle and                      cysteine syntheses is controlled by the intracellular content <br /> growth machinery. Repression of the sulfur gene net-                     of AdoMet. A key component of this regulation is the MET <br /> work is mediated by the ubiquitin ligase SCFMet30,                       gene transcriptional activator Met4p and its associated <br /> which targets the transcription factor Met4p for                         cofactors Cbf1p, Met28p, Met31p and Met32p (reviewed <br /> degradation. Met30p is an essential protein in yeast.                    in Thomas and Surdin-Kerjan, 1997). Repression of <br /> We have found that a met4&#226;&#710;&#8224;met30&#226;&#710;&#8224; double mutant is                        Met4p-dependent transcription in response to increased <br /> viable, suggesting that the essential function of Met30p                 intracellular AdoMet is mediated by the SCFMet30 complex, <br /> is to control Met4p. In support of this hypothesis, a                    an E3 ubiquitin ligase (Patton et al., 1998; Rouillon et al., <br /> Met4p mutant unable to activate transcription does                       2000). Met4p degradation and transcriptional activity are <br /> not cause inviability in a met30&#226;&#710;&#8224; strain. Also, over-                    governed by an autoregulatory loop. When intracellular <br /> expression of an unregulated Met4p mutant is lethal                      AdoMet is low, Met4p-containing transcription activation <br /> in wild-type cells. Under non-permissive conditions,                     complexes drive expression of the MET gene network, <br /> conditional met30&#226;&#710;&#8224; strains arrest as large, unbudded                     including the MET30 gene. Once expressed, Met30p <br /> cells with 1N DNA content, at or shortly after the                       targets Met4p for degradation via SCFMet30 and thereby <br /> pheromone arrest point. met30&#226;&#710;&#8224; conditional mutants                       limits expression of the MET genes in response to high <br /> fail to accumulate CLN1 and CLN2, but not CLN3                           intracellular AdoMet (Rouillon et al., 2000). <br /> mRNAs, even when CLN1 and CLN2 are expressed                                SCF ubiquitin ligase complexes target specific substrates <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> strong heterologous promoters. One or more                          for degradation, as determined by their F-box component <br /> genes under the regulation of Met4p may delay the                        (reviewed in Craig and Tyers, 1999). In addition to Met4p, <br /> progression from G1 into S phase through specific                        SCFMet30 appears to target Swe1p, a Cdc28p inhibitory <br /> regulation of critical G1 phase mRNAs.                                   kinase necessary for the morphological checkpoint (Kaiser <br /> Keywords: G1 cyclin/G1&#226;&#8364;&#8220;S transition/methionine/SCF                       et al., 1998). SCFMet30 is one of three SCF complexes <br /> ubiquitin ligase                                                         identified in yeast. SCFCdc4 targets the cell cycle and <br />                                                                          transcriptional regulators Sic1p, Far1p, Gcn4p, Ctf13p and <br />                                                                          Cdc6p for degradation, while SCFGrr1 is responsible for <br />                                                                          the degradation of the G1 cyclins, Cln1p and Cln2p, of <br /> Introduction                                                             the Cdc42p effector Gic1p, and regulation of the glucose <br /> Availability of nutrients in the environment is essential                repression pathway. Proteins similar to Met30p exist in <br /> for cell growth and cell cycle progression. In the budding               other fungi, Drosophila and mammals, and are the SconB, <br /> yeast Saccharomyces cerevisiae, <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation can                 Slimb and &#206;&#178;-TrCP proteins, respectively (Kaiser et al., <br /> elicit different responses such as cell cycle arrest in G1               1998; Margottin et al., 1998). Recently, the &#206;&#178;-TrCP/Slimb <br /> phase, sporulation or pseudohyphal growth. In particular,                proteins have been implicated in the ubiquitin-mediated <br /> sufficient sources of carbon, phosphate, nitrogen and sulfur             degradation of protein targets in NF-&#206;&#186;B, Wnt/Wingless <br /> are required for yeast cells to pass through the cell cycle              and Hedgehog pathways (reviewed in Maniatis, 1999). <br /> commitment point called Start in late G1 phase (reviewed                 Also, &#206;&#178;-TrCP was shown to trigger the degradation of <br /> in Pardee, 1989; Hartwell, 1994; Sherr, 1994; Polymenis                  the lymphocyte receptor CD4 via the HIV protein Vpu <br /> and Schmidt, 1999).                                                      (Margottin et al., 1998). <br />    Budding yeast can satisfy their elemental sulfur require-                Regulation of the cell division cycle requires the activa- <br /> ment through the uptake and subsequent metabolism of a                   tion of the essential cyclin-dependent kinase (CDK) <br /> large number of either inorganic or organic sulfur com-                  Cdc28p by the G1 cyclins Cln1p, 2p and 3p (reviewed in <br />  <br /> &#194;&#169; European Molecular Biology Organization                                                                                       1613 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 1. (A) Simplified view of sulfur amino acid biosynthesis in yeast. (B) The met4&#226;&#710;&#8224; disruption <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span> specifically suppresses the lethality <br /> induced by the loss-of-function mutation met30&#226;&#710;&#8224;. See Materials and methods for details of the crosses. <br />  <br />  <br /> Nasmyth, 1996). Cyclin-CDK activity is rate-limiting for                       1A; see Materials and methods). We also included a cross <br /> passage through Start, and control of cyclin-CDK activity                      with a met4&#226;&#710;&#8224; mutant strain in our analysis as Met30p was <br /> has been reported at multiple levels including inhibition by                   shown to trigger Met4p degradation in response to an <br /> cyclin inhibitors, transcription, translation and degradation                  increase in intracellular AdoMet (Rouillon et al., 2000). <br /> (Hartwell, 1994; Sherr, 1994; Nasmyth, 1996; Polymenis                         Single deletions of MET3, MET6 or STR4 genes as well <br /> and Schmidt, 1999). For example, non-degradable forms                          as the double deletion of the SAM1 and SAM2 genes did <br /> of the G1 cyclins, such as Cln3-1, cause hyperactivation                       not alter the met30&#226;&#710;&#8224; lethality <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, indicating that <br /> of Start and hence a decreased critical cell size (Nash                        the met30&#226;&#710;&#8224; lethality was not due to the accumulation of <br /> et al., 1988). Repression of CLN3 mRNA translation is                          a sulfur-containing metabolite. In contrast, deletion of <br /> required for cell cycle arrest in starvation and high                          MET4 bypassed the lethality of the met30&#226;&#710;&#8224; strain (Figure <br /> iron conditions (Polymenis and Schmidt, 1997; Philpott                         1B), suggesting that the essential function of Met30p <br /> et al., 1998).                                                                 might be the regulation of Met4p. <br />    While the MET network is only essential in the absence                         We made a conditionally viable met30&#226;&#710;&#8224; mutant by <br /> of sulfur amino acids, MET30 is an essential gene under                        constructing strains that could express MET4 under the <br /> all <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> conditions (Thomas et al., 1995). Here, we                        GAL1 promoter in the absence of MET30. As shown in <br /> show that the essential function of Met30p is the regulation                   Figure 2A, expression of GAL1&#226;&#8364;&#8220;MET4 in met30&#226;&#710;&#8224; cells <br /> of Met4p transcriptional activity. Surprisingly, Met4p                         induces lethality in minimal or complete galactose-con- <br /> regulation is critical for progression through Start. Our                      taining medium. Also, repressive amounts of extracellular <br /> results argue that the misregulation of genes under the                        methionine or AdoMet, which have been shown to trigger <br /> control of Met4p inhibits passage through Start by pre-                        the degradation of Met4p, did not prevent the detrimental <br /> venting the accumulation of important G1 phase transcripts,                    effect of MET4 expression in met30&#226;&#710;&#8224; mutants. Similar <br /> including those of the genes encoding the G1 cyclins,                          results were obtained with cells grown in sulfur-less B <br /> Cln1p and Cln2p.                                                               medium in the presence of homocysteine or methionine <br />                                                                                used as the sulfur source (data not shown). Altering the <br /> Results                                                                        levels of GAL1&#226;&#8364;&#8220;MET4 expression by spotting the mutants <br />                                                                                on galactose concentrations ranging from 0.1 to 2.5% was <br /> Inactivation of MET4 suppresses met30&#226;&#710;&#8224; lethality                               equally effective in inducing lethality (data not shown). <br /> MET30 is an essential gene and encodes a negative                                 The first functional analyses of Met30p were carried <br /> regulator of the sulfur amino acid biosynthesis pathway                        out in MET30-1 and MET30-2 strains, point mutants of <br /> (Thomas et al., 1995). The unregulated synthesis of a                          MET30 that are unable to fully repress MET gene expres- <br /> particular sulfur-containing metabolite might therefore be                     sion in response to high levels of extracellular methionine <br /> responsible for the lethality of a met30&#226;&#710;&#8224; strain. To test this                 (Thomas et al., 1995). The viable conditional met30&#226;&#710;&#8224; <br /> hypothesis, we crossed a met30::LEU2 strain harboring a                        mutant provided us with the opportunity to verify that <br /> LexA-MET30, HIS3 plasmid with a battery of mutants that                        Met30p function is absolutely required for Met4p regula- <br /> each specifically block one metabolic arm of the sulfur                        tion. We analyzed the AdoMet-mediated repression of two <br /> network (met3&#226;&#710;&#8224;, met6&#226;&#710;&#8224;, str4&#226;&#710;&#8224; and sam1&#226;&#710;&#8224;sam2&#226;&#710;&#8224;; Figure                            structural genes from the sulfate assimilation pathway <br />  <br /> 1614 <br />                                                                                                    Role of Met4 at the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />                                                                         MET genes as a part of multiprotein complexes. Depending <br />                                                                         on the gene, different combinations of the Cbf1p, Met28p, <br />                                                                         Met31p and Met32p factors are assembled and tether <br />                                                                         Met4p to DNA (Kuras et al., 1997; Blaiseau and Thomas, <br />                                                                         1998). To determine whether, in addition to Met4p, com- <br />                                                                         ponents of these complexes were required for the essential <br />                                                                         function of Met30p, we crossed a met30::LEU2 strain <br />                                                                         harboring a LexA-MET30, HIS3 plasmid with the single <br />                                                                         cbf1&#226;&#710;&#8224; or met28&#226;&#710;&#8224; mutants as well as with a met31&#226;&#710;&#8224;met32&#226;&#710;&#8224; <br />                                                                         double mutant. The results of these genetic tests showed <br />                                                                         that met32&#226;&#710;&#8224; met30&#226;&#710;&#8224; spores could be recovered while <br />                                                                         deletion of CBF1, MET28 or MET31 had no effect on the <br />                                                                         lethality of the met30&#226;&#710;&#8224; mutation (Table I). The essential <br />                                                                         function of Met30p might therefore be required to antagon- <br />                                                                         ize a specific Met4p&#226;&#8364;&#8220;Met32p transcription complex. <br />  <br />                                                                         Met4p regulation is essential for cell viability <br />                                                                         We have recently reported the SCFMet30-dependent <br />                                                                         degradation of Met4p in response to high extracellular <br />                                                                         methionine (Rouillon et al., 2000). Our genetic data, <br />                                                                         demonstrating that the essential function of Met30p could <br />                                                                         be bypassed by the deletion of MET4 or MET32, suggested <br />                                                                         that the lethality of a met30&#226;&#710;&#8224; strain might be due to <br /> Fig. 2. Expression of Met4p from the GAL1 promoter is lethal in         unregulated activity of Met4p, as Met32p lacks transcrip- <br /> met30&#226;&#710;&#8224; cells. (A) Serial dilutions of the CC932-6D (met4::GAL1&#226;&#8364;&#8220; <br /> MET4 MET30) and CC932-8B (met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2)                  tional activation function (Blaiseau et al., 1997). Detailed <br /> strains were plated on media containing 2% glucose (Glu), raffinose     analysis of the functional organization of Met4p has <br /> (Raf) or galactose (Gal) as carbon source in the absence or presence    identified one activation domain (AD), two regulatory <br /> of 1 mM L-methionine (Met) or 0.2 mM AdoMet. (B) CC932-6D               regions required for the AdoMet response, called the <br /> (met4::GAL1&#226;&#8364;&#8220;MET4 MET30) and CC932-8B (met4::GAL1&#226;&#8364;&#8220; <br /> MET4 met30::LEU2) cells were grown in raffinose medium and <br />                                                                         inhibitory region (IR) and the auxiliary domain (AUX), <br /> expression was induced by transferring the cells to a fresh galactose   and two domains required for tethering Met4p to DNA <br /> (2%) medium for 90 min. A repressing amount of L-Met (1 mM) was         via protein&#226;&#8364;&#8220;protein interactions with Met31p/Met32p and <br /> then added and total RNA was extracted at the times indicated.          Met28p/Cbf1p, called the INT domain and the bZIP <br /> Expression of MET16, MET25 and GAL1&#226;&#8364;&#8220;MET4 were determined                domain, respectively (Kuras and Thomas, 1995; Blaiseau <br /> by Northern analysis. The actin probe was used as a control of the <br /> amount of RNA loaded.                                                   and Thomas, 1998). To determine which particular aspect <br />                                                                         of Met4p function is required to induce lethality in the <br />                                                                         absence of Met30p, different mutant derivatives of Met4p <br /> (MET16 and MET25) in cells expressing MET4 from the                     were expressed from the GAL1 promoter in met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; <br /> GAL1 promoter in the presence or absence of MET30.                      cells and tested for viability. As shown in Figure 3A, <br /> The two corresponding strains (met4::GAL1&#226;&#8364;&#8220;MET4 and                      deletion of the AD (pGalMet4&#226;&#710;&#8224;12) or INT (pGalMet4&#226;&#710;&#8224;38) <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;) were first grown in the                         domains of Met4p rescued the GAL1&#226;&#8364;&#8220;MET4-induced leth- <br /> presence of raffinose for eight generations, transferred to             ality in met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells. In contrast, deletion of the <br /> a medium containing galactose to induce GAL1&#226;&#8364;&#8220;MET4                       two AdoMet regulatory regions, the IR (pGalMet4&#226;&#710;&#8224;30) <br /> gene expression for 90 min before a repressive concentra-               and AUX (pGalMet4&#226;&#710;&#8224;37) domains, did not prevent lethal- <br /> tion of methionine was added, and samples extracted at                  ity. This is consistent with our findings that met4&#226;&#710;&#8224; and <br /> regular time intervals after methionine addition. As shown              met32&#226;&#710;&#8224;, but not cbf1&#226;&#710;&#8224; or met28&#226;&#710;&#8224;, can bypass the met30&#226;&#710;&#8224; <br /> in Figure 2B, Northern blot analysis demonstrated that                  arrest, and reinforces the idea that an unregulated transcrip- <br /> repression of MET16 and MET25 transcription in response                 tional activity of a DNA-bound complex containing Met4p <br /> to high methionine was dependent on MET30. MET16                        and Met32p is lethal. This notion might be too simplified, <br /> and MET25 genes were repressed with wild-type kinetics                  however, because removal of the Met4p bZIP domain <br /> in met4::GAL1&#226;&#8364;&#8220;MET4 strains, but were insensitive to high                thought to bind Met28p and Cbf1p results in a partial <br /> extracellular methionine in met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                      rescue of met30&#226;&#710;&#8224; (pGalMet4&#226;&#710;&#8224;146; Figure 3A), while the <br /> cells. Insensitivity of the MET16 and MET25 genes to                    simple deletion of MET28 or CBF1 is not sufficient to <br /> high extracellular methionine in met4::GAL1&#226;&#8364;&#8220;MET4                        bypass the met30&#226;&#710;&#8224; lethality. <br /> met30&#226;&#710;&#8224; cells is consistent with the above results showing                  Given the expectation that Met30p-mediated regulation <br /> that repressive amounts of extracellular methionine or                  of Met4p requires an interaction between the two factors, <br /> AdoMet did not suppress the lethality induced by MET4                   it could be predicted that mutations within Met4p should <br /> expression in met30&#226;&#710;&#8224; cells. As a control, the transcription             exist that by impairing the Met30p&#226;&#8364;&#8220;Met4p interaction <br /> of the GAL1&#226;&#8364;&#8220;MET4 fusion gene was not affected by the                    would cause lethality even in cells expressing MET30. <br /> presence of methionine. These results confirmed our                     We had previously mapped the IR domain as a region of <br /> previous findings that Met30p is required for repression                Met4&#226;&#8364;&#8220;Met30 protein interaction by two-hybrid analysis <br /> of the MET gene network.                                                (Thomas et al., 1995); however, our IR deletion mutant <br />    To function as a transcriptional activator, Met4p is                 was unable to induce lethality in the presence of Met30p. <br /> recruited to the sequence-specific upstream regions of                  This suggested that the Met4 IR domain may include <br />  <br />                                                                                                                                 1615 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 3. (A) Expression of Met4p derivatives in met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells. Schematic representations of the modified Met4p derivatives expressed from the <br /> GAL1 promoter region are shown. Plasmids encoding the fusion genes were introduced into CC807-1C (met4::TRP1 met30::LEU2) cells. Serial <br /> dilutions of the resulting transformants were plated on media containing 2% raffinose or galactose as carbon source. (B) Expression of Met4p <br /> derivatives in met4&#226;&#710;&#8224; MET30 cells. Details of the deletions are scaled up. Plasmids encoding the fusion genes were introduced into CD106 <br /> (met4::TRP1) cells. Serial dilutions of the resulting transformants were plated on media containing 2% raffinose or galactose as carbon source. <br />  <br />  <br />  <br /> Table I. Suppression of met30&#226;&#710;&#8224;-induced lethality by inactivation of the MET32 gene <br />  <br /> Genotype             Phenotype         Genotype                 Phenotype        Crosses <br />  <br /> met30&#226;&#710;&#8224;               lethal <br /> met4&#226;&#710;&#8224;                Met&#226;&#8364;&#8220;              met4&#226;&#710;&#8224; met30&#226;&#710;&#8224;             Met&#226;&#8364;&#8220;             see Figure 1 <br /> met28&#226;&#710;&#8224;               Met&#226;&#8364;&#8220;              met28&#226;&#710;&#8224; met30&#226;&#710;&#8224;            lethal           CC958 &#226;&#171;&#189; CD130-1A &#226;&#171;&#187; CC850-2B <br /> cbf1&#226;&#710;&#8224;                Met&#226;&#8364;&#8220;              cbf1&#226;&#710;&#8224; met30&#226;&#710;&#8224;             lethal           CC806 &#226;&#171;&#189; CD127-1AT &#226;&#171;&#187; R31-5C <br /> met31&#226;&#710;&#8224;               Met&#226;&#171;&#185;              met31&#226;&#710;&#8224; met30&#226;&#710;&#8224;            lethal           CC867 &#226;&#171;&#189; C847-1D &#226;&#171;&#187; CD179 <br /> met32&#226;&#710;&#8224;               Met&#226;&#171;&#185;              met32&#226;&#710;&#8224; met30&#226;&#710;&#8224;            Met&#226;&#171;&#185;             CC867 &#226;&#171;&#189; C847-1D &#226;&#171;&#187; CD179 <br />  <br />  <br />  <br /> amino acid residues outside of 189&#226;&#8364;&#8220;235, consistent with                     The essential function of Met30p is required for <br /> data reported by Omura and colleagues, who isolated a                       passage through G1 phase <br /> Met4p point mutant (F156S) that was unresponsive to                         Conditional expression of the MET4 gene from a GAL1 <br /> high extracellular methionine (Omura et al., 1996). We                      promoter in met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; double mutants allowed us <br /> thus constructed a new set of deletions around the IR                       to characterize the phenotype of met30&#226;&#710;&#8224; mutants. We <br /> region of GAL1&#226;&#8364;&#8220;MET4 and expressed them in met4&#226;&#710;&#8224;                             performed growth curve analysis to determine the number <br /> MET30 cells. Two derivatives, Met4&#226;&#710;&#8224;47 (amino acids                          of generations before arrest, fluorescence-activated cell <br /> 160&#226;&#8364;&#8220;221) and Met4&#226;&#710;&#8224;26 (amino acids 160&#226;&#8364;&#8220;186), induced                         sorter (FACS) analysis to measure DNA content, and <br /> lethality in met4&#226;&#710;&#8224; MET30 cells (Figures 3B and 4E),                         determined the budding index. Early log phase cultures <br /> indicating that the Met30p-mediated regulation of Met4p                     of met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; were grown in raffinose, <br /> was dependent on a larger IR domain than previously                         split into two, galactose added to one culture, and cell <br /> reported, and supporting the notion that unregulated Met4p                  number measured over time. Within one doubling the <br /> activity might be responsible for the lethality of met30&#226;&#710;&#8224;                   GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells had arrested, while the raffinose <br /> mutants.                                                                    culture continued to increase in cell number (Figure 4A). <br />  <br /> 1616 <br />                                                                                                          Role of Met4 at the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 4. Conditional met30&#226;&#710;&#8224; mutant cells arrest as large unbudded cells with 1N DNA. (A) CC932-6D (met4::GAL1&#226;&#8364;&#8220;MET4 MET30) (j) and CC932- <br /> 8B (met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2) (d) cells were grown in the presence of 2% raffinose as carbon source. At the time indicated by the arrow, <br /> 2% galactose was added to each culture and growth was followed by measuring the OD650 for the times indicated. (B) Wild type, met4::GAL1&#226;&#8364;&#8220; <br /> MET4 MET30 and met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2 were grown in the presence of raffinose, cultures were divided into two, filtered and transferred <br /> to raffinose or galactose medium. FACS analyses were then performed at the times indicated. (C) CC807-1C (met4::TRP1 met30::LEU2) cells were <br /> transformed by the pGal316, pGalMet4-1 or pGalMet4&#226;&#710;&#8224;12 plasmids. Resulting transformants were grown in the presence of 2% raffinose as carbon <br /> source. Cultures were then filtered and transferred to a galactose (2%) medium. Photographs of the cells were then taken at the times indicated. <br /> (D) Budding index of the cells grown in (A). (E) CC807-1C (met4::TRP1 met30::LEU2) or CD106 (met4::TRP1) were transformed by the plasmids <br /> indicated. Resulting transformants were grown in raffinose medium to early log phase and arrested by &#206;&#177; factor for 3 h in 2% galactose. The cells <br /> were released from &#206;&#177; factor by washing with fresh galactose medium and analyzed for their DNA content by flow cytometry at the times indicated. <br />  <br />  <br /> To examine whether the cells had any particular cell                          There are multiple biological pathways that can cause <br /> cycle arrest phenotype, we grew parallel cultures of                       an unbudded G1 phenotype when disrupted (Nasmyth, <br /> met4::GAL1&#226;&#8364;&#8220;MET4 and met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                                 1996). To gain insight into the mechanism responsible for <br /> strains in raffinose and galactose, and took samples at 0,                 the G1 arrest observed in the GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells, <br /> 3 and 6 h. As shown in Figure 4B, expression of MET4                       we undertook a detailed study of the characteristics of the <br /> in the absence of MET30 causes the cells to accumulate                     GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; arrest. To map the arrest position <br /> in G1 with unreplicated DNA. Microscopic observation                       of the met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutant, we arrested an <br /> showed that &#226;&#172;&#381;90% of the GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells                             asynchronous culture growing in raffinose with mating <br /> remain unbudded 6 h after the shift to the galactose                       pheromone for 2 h, split the culture into two, added <br /> medium (Figure 4C and D).                                                  raffinose to one half and galactose to the other half for <br />    The G1 arrest phenotype of met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                       20 min, released the culture into pheromone-free medium, <br /> cells correlated with the requirement for the Met4p activa-                and took samples at 10 min intervals to measure DNA <br /> tion domain to induce lethality in these strains. Expression               content and budding morphology. Cells in raffinose pro- <br /> of GAL1&#226;&#8364;&#8220;MET4&#226;&#710;&#8224;AD produced little alteration of the FACS                     ceeded through the cell cycle with a similar FACS profile <br /> profile, while expression of GAL1&#226;&#8364;&#8220;MET4 in met4&#226;&#710;&#8224; met30&#226;&#710;&#8224;                     to wild type (Figure 5A), demonstrating that in the absence <br /> cells caused a G1 arrest (Figure 4E). Also, expression of                  of Met4p, the deletion of MET30 has only a marginal <br /> GAL1&#226;&#8364;&#8220;MET4&#226;&#710;&#8224;IR(160&#226;&#8364;&#8220;221), which had been shown to                             effect on cell cycle progression. In contrast, the met4:: <br /> induce lethality in the presence of MET30, also arrested                   GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants remained in G1 phase <br /> the cells in G1 phase. Expression of other GAL1&#226;&#8364;&#8220;MET4                       without undergoing DNA replication (Figure 5A) or initiat- <br /> derivatives, described above, showed FACS profiles sim-                    ing budding (Figure 5B). When GAL1&#226;&#8364;&#8220;MET4 was induced <br /> ilar to wild type. Thus, unregulated activity of Met4p,                    in the met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; mutants after the release from <br /> either by lack of the IR(160&#226;&#8364;&#8220;221) domain or by deletion                    pheromone, the cells progressed through the cell cycle <br /> of MET30, is sufficient to cause a G1 arrest.                              before accumulating in G1 phase (data not shown), <br />  <br />                                                                                                                                          1617 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 5. The essential function of MET30 is required at or after the pheromone arrest position, but prior to the initiation of budding or DNA <br /> replication. (A) Wild-type, met4::GAL1&#226;&#8364;&#8220;MET4 and met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells were grown in rich raffinose medium to early log phase and <br /> arrested with &#206;&#177; factor for 2 h. The cells were then split into two cultures and raffinose added to one, while galactose was added to another for <br /> 20 min. The cells were released from &#206;&#177; factor by washing with fresh medium and time points taken at the designated intervals. (B) Photographs of <br /> the cells in (A) demonstrate that the met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; arrest as large unbudded cells. The tear shape of the cells is due to effects of the <br /> pheromone. (C) MET30 is required before the cdc7-1 arrest point. Wild-type, cdc7-1 and cdc7-1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cultures were incubated <br /> at 37&#194;&#176;C for 2 h, had galactose added for 30 min, and were then shifted to 25&#194;&#176;C and samples taken at the time indicated. <br />  <br />  <br />  <br /> suggesting that the effects of GAL1&#226;&#8364;&#8220;MET4 are specific to                    replication, but not for budding (Yoon et al., 1993). <br /> the G1 phase of the cell cycle. Unregulated Met4p might,                    cdc7-1 mutants arrested at 37&#194;&#176;C with small buds and 1N <br /> therefore, function at or shortly after the pheromone arrest                DNA content (Figure 5C), and progressed through S and <br /> point to inhibit passage through Start. Consistent with this                G2 phase upon shift to 25&#194;&#176;C. Similarly, met4::GAL1&#226;&#8364;&#8220; <br /> hypothesis, overexpression of MET4, still under the control                 MET4 met30&#226;&#710;&#8224; cdc7-1 mutants arrested in G1 phase at <br /> of endogenous levels of Met30p, induces a 20 min delay                      37&#194;&#176;C, and progressed through S and G2 phase when shifted <br /> through Start (Figure 5A). Met30p was originally identified                 to 25&#194;&#176;C in galactose, placing the met30&#226;&#710;&#8224; arrest position <br /> as a mutant that was defective for repression of MET                        after the pheromone block but before the Cdc7p require- <br /> gene transcription (Thomas et al., 1995). Given that                        ment point in the cell cycle. <br /> overexpression of MET4 could delay passage through G1 <br /> of the cell cycle by 20 min, we examined the progression                    G1-phase-arrested met30&#226;&#710;&#8224; mutants are <br /> of MET30-1 mutant cells through the cell cycle by FACS                      compromised for efficient growth and translation <br /> analysis. MET30-1 cells were delayed for passage into S                     Start mutants can be categorized into those affecting <br /> phase by 10 min (data not shown), indicating that non-                      growth such as mutations in the cAMP pathway or the <br /> lethal mutations in MET30 that are partially defective for                  translational machinery, and those affecting division such <br /> Met4p regulation are also capable of producing a G1&#226;&#8364;&#8220;S                       as mutations in CDC28 or the G1 cyclins (reviewed in <br /> phase delay.                                                                Hartwell, 1994). To determine which component of Start <br />    The shift from a pheromone-induced arrest to a met30&#226;&#710;&#8224;                    was altered in the conditional met30&#226;&#710;&#8224; mutants, we mon- <br /> arrest demonstrated that the met30&#226;&#710;&#8224; arrest phenotype was                    itored the rate of growth in small, unbudded met4::GAL1&#226;&#8364;&#8220; <br /> at or after the pheromone arrest point but prior to the                     MET4 met30&#226;&#710;&#8224; G1 cells collected by centrifugal elutriation <br /> DNA replication and budding initiation point. In the                        and then grown in raffinose or galactose (Figure 6A). As <br /> converse experiment, i.e. arresting the met4::GAL1&#226;&#8364;&#8220;MET4                     expected, in raffinose medium cells grew steadily over <br /> met30&#226;&#710;&#8224; cells in G1 phase in galactose and releasing them                    time and passed through Start (25% budded) at a cell <br /> into glucose and pheromone medium, the cells were unable                    volume of 50 fl. These cells progressed through S phase <br /> to recover from the prolonged GAL1&#226;&#8364;&#8220;MET4 arrest. As an                       and into G2&#226;&#8364;&#8220;M phase, as indicated by the increase in bud <br /> alternative, we constructed a met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                        index and DNA content (Figure 6B and C). In contrast, <br /> cdc7-1 mutant, arrested the cells at 37&#194;&#176;C and released                      cells grown in galactose medium did not initiate budding <br /> them into a galactose-containing medium at 25&#194;&#176;C. The                        (Figure 6B) and maintained a 1N DNA content (Figure <br /> Cdc7p kinase is required for the initiation of DNA                          6C), but did, however, accumulate mass over time (Figure <br />  <br /> 1618 <br />                                                                                                               Role of Met4 at the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 6. Conditional G1-arrested met30&#226;&#710;&#8224; mutants are slowed for cell growth and translation. Cultures of small G1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells <br /> grown in raffinose were collected by centrifugal elutriation, divided into two, and raffinose added to one culture and galactose to the other. Samples <br /> were taken at the times indicated to measure (A) cell volume, (B) budding index and (C) DNA content by flow cytometry. (D) Polysome profiles <br /> of wild-type or met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells through the Start position. G1 cells were collected by centrifugal elutriation in a rich, raffinose <br /> medium, galactose was added at time 0, and samples were taken at the times indicated for polysome profiles, flow cytometry and budding analysis. <br />  <br />  <br />  <br /> 6A). Initially, the conditional met30&#226;&#710;&#8224; mutants grown in                        galactose. Similarly, the number of discrete polysome <br /> galactose medium accumulated mass at a rate equal to                           peaks and UV absorbance of the polysome peaks were <br /> those in raffinose medium, but after 100 min the rate of                       slightly reduced in the met30&#226;&#710;&#8224; mutants after 2.5 h (data <br /> growth was slowed somewhat.                                                    not shown). To perform a more detailed analysis, we used <br />    To determine whether the decreased rate of mass accu-                       synchronized G1 cells isolated by centrifugal elutriation <br /> mulation in the met30&#226;&#710;&#8224; mutants might be due to defects                         and examined polysome profiles of wild-type and met30&#226;&#710;&#8224; <br /> in translation, we measured the incorporation of [35S]me-                      cells as a function of time. In comparison with wild-type <br /> thionine/cysteine into total protein, and examined the                         cells, the extent of polyribosome formation was reduced <br /> polysome profiles of the conditional met30&#226;&#710;&#8224; mutants.                           and, correspondingly, the 80S ribosome peak was increased <br /> [35S]methionine/cysteine incorporation was decreased by                        in the met30&#226;&#710;&#8224; mutants at 2.5 h, by which time wild-type <br /> 20% in the conditional met30&#226;&#710;&#8224; mutants after 2.5 h in                           cells had passed through Start (Figure 6D). It therefore <br />  <br />                                                                                                                                                 1619 <br />  E.E.Patton et al. <br />  <br />  <br /> appears that met30&#226;&#710;&#8224; cells are able to reach the critical      budding yeast. Our genetic analysis shows that the essential <br /> cell size required for passage through Start but gradually    function of MET30 is bypassed by deletion of MET4, the <br /> become impaired in translation.                               gene that encodes the main transcriptional activator of <br />                                                               the sulfur amino acid biosynthesis pathway, and that <br /> Various G1 cyclin RNA transcripts are reduced in              unregulated Met4p activity is lethal. The creation of a <br /> met30&#226;&#710;&#8224; mutants                                                met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; conditional mutant has <br /> As the conditional met30&#226;&#710;&#8224; strain was only slightly com-       allowed us to confirm the function of Met30p in regulating <br /> promised for translation, we reasoned that other compon-      Met4p function (Figure 2B) and to characterize the arrest <br /> ents of the cell division machinery might be altered in       phenotype of met30&#226;&#710;&#8224; cells (Figures 3 and 4). Surprisingly, <br /> these cells. We therefore tested whether overexpression       met30&#226;&#710;&#8224; conditional mutants arrest as large, unbudded cells <br /> of the G1 cyclins could rescue the met30&#226;&#710;&#8224; phenotype.          with unreplicated DNA. Genetic analysis has allowed us <br /> Expression of the genes encoding the G1 cyclins, CLN1,        to determine that the G1 arrest phenotype of the met30&#226;&#710;&#8224; <br /> CLN2, CLN3, CLN3-1, PCL1 or PCL2, and the S and G2            conditional mutant is most likely not a result of an <br /> cyclins, CLB2 or CLB5, from the GAL1 promoter in              unbalanced synthesis of a sulfur metabolite because <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants was unable to bypass           mutations in structural genes of the MET pathway do not <br /> the <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> (data not shown). Intriguingly, even     alter the lethal phenotype of met30&#226;&#710;&#8224; strains (Figure 1). <br /> if CLN1 and CLN2 genes were expressed from the GAL1           The G1 arrest phenotype is dependent on the transcriptional <br /> promoter for 3 h, Cln1 or Cln2-Cdc28 kinase activity          activity of Met4p, because Met4p mutants without a <br /> could not be detected in the met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;           transcriptional activation domain (Met4&#226;&#710;&#8224;AD), or that are <br /> mutant strain. Similarly, Western analysis revealed that      incapable of binding the cofactors Met31p and Met32p <br /> Cln1p and Cln2p were barely detectable (data not shown).      (Met4&#226;&#710;&#8224;INT), are not lethal in a met30&#226;&#710;&#8224; background <br /> To ensure that the decrease in protein level was not due      (Figure 3A). Consistent with these results, the met32&#226;&#710;&#8224; <br /> to repression of the GAL1 promoter, we also examined          mutation is also capable of bypassing the met30&#226;&#710;&#8224; lethal <br /> the levels of Cln2p expressed from the endogenous pro-        phenotype (Figure 1B). The fact that the met32&#226;&#710;&#8224; mutation, <br /> moter or from the constitutive ADH1 promoter. In each         but not the met31&#226;&#710;&#8224; mutation, was specifically capable of <br /> case, Cln2p was undetectable in the met4::GAL1&#226;&#8364;&#8220;MET4           bypassing the met30&#226;&#710;&#8224; mutation was somewhat unexpected <br /> met30&#226;&#710;&#8224; cells (data not shown). We next performed North-       since Met31p and Met32p, which are two highly homolo- <br /> ern analysis with total RNA extracted from met4::GAL1&#226;&#8364;&#8220;        gous zinc finger factors, are thought to have redundant <br /> MET4 met30&#226;&#710;&#8224; cells grown in the presence of galactose          functions (Blaiseau et al., 1997). For instance, only the <br /> and found that CLN2 transcripts were undetectable, while      double met31&#226;&#710;&#8224; met32&#226;&#710;&#8224; mutant is a methionine auxotroph, <br /> in contrast, ACT1 mRNA levels were clearly detectable,        while single met31&#226;&#710;&#8224; and met32&#226;&#710;&#8224; mutants are not (Blaiseau <br /> although slightly reduced (Figure 7A). As the protein level   et al., 1997). Likewise, mobility shift assays revealed no <br /> had suggested, CLN1 transcripts were also undetectable, as    difference between the Met4&#226;&#8364;&#8220;Met28&#226;&#8364;&#8220;Met31 and Met4&#226;&#8364;&#8220; <br /> were PCL2 transcripts, while CLN3, WHI3 and WHI4              Met28&#226;&#8364;&#8220;Met32 complexes assembled on the MET3 and <br /> transcripts appeared normal (data not shown).                 MET28 promoter regions (Blaiseau and Thomas, 1998). <br />    It was important to determine whether CLN1 and CLN2        Our results suggest, however, that either the precise <br /> mRNAs failed to accumulate during G1 phase in the             function of Met31p and Met32p, or at least the amount <br /> met30&#226;&#710;&#8224; mutant cells, or whether the lack of detectable        of the two proteins required for the assembly of the <br /> transcripts was a secondary consequence of the prolonged      Met4p-containing complexes, might differ. <br /> G1 arrest. We therefore collected small, unbudded, G1            Regulation of Met4p by Met30p is dependent on a <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells by centrifugal elutri-           Met30p binding region called the IR (Thomas et al., <br /> ation, added raffinose to one half of the culture and         1995). Accordingly, a Met4p derivative lacking the IR <br /> galactose to the other, and collected samples for Northern,   region (amino acids 189&#226;&#8364;&#8220;235) was recently shown to be <br /> FACS and budding analysis. The cells grown in raffinose       less sensitive to SCFMet30-triggered degradation in response <br /> medium expressed CLN1, CLN2 and CLN3 transcripts              to high extracellular methionine (Rouillon et al., 2000). <br /> appropriately, grew in cell size, initiated budding and       However, during our analysis of the regions of Met4p <br /> replicated their DNA (Figure 7B and C). In contrast, the      required for the G1 arrest in met30&#226;&#710;&#8224; mutants, we found <br /> cells grown in galactose medium failed to accumulate          that the Met30p&#226;&#8364;&#8220;Met4p interaction region previously <br /> CLN1 and CLN2 mRNAs but did have wild-type levels             defined by two-hybrid analysis did not entirely correlate <br /> of CLN3 mRNAs (Figure 7B). As before, the cells did           with the Met4p region required for inhibition by Met30p <br /> not initiate budding or DNA replication (Figure 7C). The      because deletion of the former was not sufficient to induce <br /> inability of met30&#226;&#710;&#8224; cells to accumulate CLN1/2 and PCL1/      a G1 arrest in cells carrying MET30. Deletion of a slightly <br /> 2 mRNAs is sufficient to account for the arrest at Start      greater domain (amino acids 160&#226;&#8364;&#8220;220, Met4&#226;&#710;&#8224;47) was, in <br /> (Espinoza et al., 1994; Measday et al., 1994). However,       contrast, capable of inducing a G1 arrest in a MET30 <br /> we cannot rule out the possibility that other G1 phase        background. We suggest that our previously defined IR <br /> transcripts are also affected.                                domain is required for the Met30p&#226;&#8364;&#8220;Met4p two-hybrid <br />                                                               interaction, yet full Met30p repression of Met4p requires <br /> Discussion                                                    amino acids 160&#226;&#8364;&#8220;220 of Met4p. <br />                                                                  Conditional met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants arrest <br /> Met4p and passage through the G1 phase of the                 in G1 phase within the first cell cycle after a shift to a <br /> cell cycle                                                    galactose medium (Figures 4A and 5). The essential <br /> In this study, we demonstrate that components of the MET      function of Met30p is required at or after the pheromone <br /> gene network are critical for passage through Start in        arrest point, but before budding or DNA replication has <br />  <br /> 1620 <br />                                                                                                           Role of Met4 at the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 7. Conditional met30&#226;&#710;&#8224; mutants lack CLN1 and CLN2 mRNA transcripts. (A) Northern analysis of CLN mRNA transcripts demonstrates a lack <br /> of CLN2 mRNA expressed from the endogenous or the ADH1 promoter in galactose-grown met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells. (B) CLN1 and CLN2, <br /> but not CLN3 mRNAs, fail to accumulate in elutriated G1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells grown in galactose, but accumulate normally in <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells grown in raffinose. Cells were grown in a rich, raffinose medium, the culture split into two, and raffinose added to <br /> one culture and galactose to the other. Samples were taken for (B) Northern analysis and (C) flow cytometry. <br />  <br /> been initiated. Overexpression of MET4 alone does not                       series of genes that inhibits passage though G1 phase. <br /> cause a G1 arrest, but it is sufficient to cause a slight                   Since the G1 arrest is dependent upon the Met4p activation <br /> delay from G1 into S phase, suggesting that overexpression                  domain, transcriptional activity in this model would arise <br /> of Met4p can transiently overcome a limiting amount of                      specifically from Met4p while Met32p would act by <br /> repression by SCFMet30. Indeed, the MET30-1 mutation,                       recruiting Met4p to DNA, as Met32p does not appear <br /> which is partially defective in Met4p repression in response                to have transcriptional activity (Blaiseau et al., 1997). <br /> to high extracellular methionine (Thomas et al., 1995),                     Transcription of this G1 phase &#226;&#8364;&#732;inhibitor&#226;&#8364;&#8482; gene might not <br /> also exhibits a delay in passage through G1 phase. That                     depend on the known Met4p transcription complexes that <br /> the MET30-1 mutation is not lethal may indicate that                        are assembled upstream of the classical MET genes because <br /> while some excess Met4p activity is admissible in the                       deletion of CBF1 and MET28 does not suppress the <br /> cell, a complete lack of regulation, such as in met30&#226;&#710;&#8224;, is                  lethality of the met30&#226;&#710;&#8224; mutation (Figure 1B). Likewise, <br /> intolerable.                                                                the &#226;&#8364;&#732;inhibitor&#226;&#8364;&#8482; gene(s) might not be required for methionine <br />                                                                             biosynthesis because the met32&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells are methi- <br /> Models for Met4p function in the G1 phase                                   onine prototrophs. Thus, Met4p might be responsible for <br /> We are currently exploring two models for the manifesta-                    the regulation of multiple gene networks and assemble <br /> tion of the met30&#226;&#710;&#8224; G1 arrest phenotype. In the first one,                   into numerous transcriptional activation complexes. An <br /> a specific transcription activation complex comprising                      alternative model for the met30&#226;&#710;&#8224; G1 arrest might be that, <br /> Met4p and Met32p regulates the expression of a gene or                      due to a lack of Met4p degradation, increased protein <br />  <br />                                                                                                                                            1621 <br />  E.E.Patton et al. <br />  <br />  <br /> Table II. Yeast strains <br />  <br /> Strain                         Genotype                                                                  Reference <br />  <br /> C180                           MATa,   ade2, his3, leu2, ura3, str4::URA3                                Che&#194;&#180; rest and Surdin-Kerjan (1992) <br /> CD127-1AT                      MATa,   ade2, his3, leu2, ura3, met30::LEU2, pLexMet30-4                  Thomas et al. (1995) <br /> CD127-1CT                      MATa,   ade2, his3, leu2, ura3, met30::URA3, pLexMet30-4                  Thomas et al. (1995) <br /> CD106                          MATa,   ade2, his3, leu2, ura3, met4::TRP1                                Thomas et al. (1992) <br /> CD130-1A                       MATa,   ade2, his3, leu2, ura3, met28::LEU2                               Kuras et al. (1996) <br /> CD155                          MATa,   ade2, his3, leu2, ura3, met3::URA3                                this study <br /> CD171                          MATa,   ade2, his3, leu2, ura3, met4::URA3, met30::LEU2                   this study <br /> CD179                          MATa,   ade2, his3, leu2, ura3, met31::LEU2, met32::TRP1                  Blaiseau et al. (1997) <br /> CC435-6C                       MATa,   ade2, his3, leu2, ura3, sam1::URA3, sam2::URA3                    Thomas et al. (1988) <br /> CC807-1C                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::LEU2                   this study <br /> CC807-7D                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::LEU2                   this study <br /> CC847-1C                       MATa,   ade2, his3, leu2, ura3, met30::URA3, pLexMet30-4                  this study <br /> CC849-8A                       MATa,   ade2, his3, leu2, ura3, met4::TRP1                                this study <br /> CC850-2B                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::URA3                   this study <br /> CC864-1B                       MATa,   ade2, his3, leu2, ura3, met6                                      this study <br /> CC867-1B                       MATa,   ade2, his3, leu2, ura3, met30::URA3, met31::LEU2, met32::TRP1     this study <br /> CC867-1D                       MATa,   ade2, his3, leu2, ura3, met30::URA3, met32::TRP1                  this study <br /> CC874-18D                      MATa,   ade2, his3, leu2, ura3, str4::URA3                                this study <br /> CC906-13C                      MATa,   ade2, his3, leu2, ura3, met3::URA3                                this study <br /> CC932-6D                       MATa,   ade2, his3, leu2, ura3, met4::GAL1&#226;&#8364;&#8220;MET4                           this study <br /> CC932-8B                       MATa,   ade2, his3, leu2, ura3, met4::GAL1&#226;&#8364;&#8220;MET4, met30::LEU2              this study <br /> CC954-10A                      MATa,   leu2, ura3, cdc7-1                                                this study <br /> CC954-10C                      MATa,   his3, leu2, ura3, cdc7-1, met4::GAL1&#226;&#8364;&#8220;MET4, met30::LEU2            this study <br /> CC970-1A                       MATa,   ade2, his3, leu2, ura3, cdc7-1, met4::GAL1&#226;&#8364;&#8220;MET4                   this study <br /> R31-5C                         MATa,   ade2, his3, leu2, lys2, ura3, cbf1::TRP1                          O&#226;&#8364;&#8482;Connell and Baker (1992) <br /> W303-1A                        MATa,   ade2, his3, leu2, trp1, ura3,                                     R.Rothstein <br /> W744-1A                        MATa,   ade2, his3, leu2, ura3, sam1::LEU2, sam2::HIS3                    Bailis and Rothstein (1990) <br />  <br />  <br />  <br />                                                                             1996; Polymenis and Schmidt, 1999). Because our GAL1&#226;&#8364;&#8220; <br /> Table III. Plasmid list <br />                                                                             MET4 met30&#226;&#710;&#8224; mutants were unable to pass through Start, <br /> Plasmid          Relevant characteristics             Source                we examined both possibilities as the potential route to <br />                                                                             the G1 arrest phenotype. We used synchronized cells to <br /> pGalMet4-1       pGal1-MET415&#226;&#8364;&#8220;666, URA3, CEN          this study            determine whether the met30&#226;&#710;&#8224; mutants were capable of <br /> pGalMet4&#226;&#710;&#8224;12      pGal1-MET4&#226;&#710;&#8224;79&#226;&#8364;&#8220;180, URA3, CEN         this study <br /> pGalMet4&#226;&#710;&#8224;13      pGal1-MET4&#226;&#710;&#8224;189&#226;&#8364;&#8220;221, URA3, CEN        this study            growth in G1 phase, and to determine whether translation <br /> pGalMet4&#226;&#710;&#8224;21      pGal1-MET4&#226;&#710;&#8224;115&#226;&#8364;&#8220;184, URA3, CEN        this study            was compromised (Figure 6). While the met30&#226;&#710;&#8224; mutant <br /> pGalMet4&#226;&#710;&#8224;26      pGal1-MET4&#226;&#710;&#8224;160&#226;&#8364;&#8220;186, URA3, CEN        this study            did continue to accumulate mass in G1 phase after a <br /> pGalMet4&#226;&#710;&#8224;30      pGal1-MET4&#226;&#710;&#8224;211&#226;&#8364;&#8220;232, URA3, CEN        this study            prolonged growth arrest, translation was slightly slowed <br /> pGalMet4&#226;&#710;&#8224;37      pGal1-MET4&#226;&#710;&#8224;352&#226;&#8364;&#8220;366, URA3, CEN        this study            in comparison with MET30 cells, and polysome profiles <br /> pGalMet4&#226;&#710;&#8224;38      pGal1-MET4&#226;&#710;&#8224;375&#226;&#8364;&#8220;430, URA3, CEN        this study <br /> pGalMet4&#226;&#710;&#8224;47      pGal1-MET4&#226;&#710;&#8224;160&#226;&#8364;&#8220;221, URA3, CEN        this study            showed a corresponding defect in translational apparatus. <br /> pGalMet4&#226;&#710;&#8224;146     pGal1-MET4&#226;&#710;&#8224;616&#226;&#8364;&#8220;666, URA3, CEN        this study            The delayed kinetics of the protein synthesis response <br /> pMT291           CLN2HA, LEU2, CEN                    Tyers laboratory      compared with the first cycle G1 arrest suggest that the <br /> pMT1241          CLN1HA, LEU2, CEN                    Tyers laboratory      effect on translation is a secondary consequence of the <br /> pMT485           pGAL1-CLN1HA, LEU2, CEN              G.Tokiwa <br /> pMT634           pGAL1-CLN2HA, LEU2, URA3,            Tyers laboratory      arrest. In contrast, we unexpectedly found that CLN1, <br />                  CEN                                                        CLN2 and PCL2 transcripts were entirely absent from the <br /> pCB1317          Sc-pADH1-CLN2HA, LEU2, CEN           K.Arndt               met30&#226;&#710;&#8224; cells, while CLN3, WHI3 and WHI4 transcripts <br />                                                                             were unaffected. Since the loss of G1 cyclin transcripts <br />                                                                             occurs well before any effects on protein synthesis, we <br /> levels of Met4p inhibit passage through G1 phase by                         suggest that the effect on G1 transcripts might directly <br /> associating specifically or non-specifically with other tran-               result from unregulated Met4p activity. Because cln1 cln2 <br /> scriptional complexes. However, the specificity of the                      pcl1 pcl2 mutants arrest at Start, the absence of these <br /> met32&#226;&#710;&#8224; mutation to bypass the G1 arrest, as well as the                     transcripts may account for the phenotype of met30&#226;&#710;&#8224; cells. <br /> altered CLN1 and CLN2 but not CLN3 transcript levels,                       As CLN transcripts are very unstable even in wild-type <br /> argue for more than Met4p non-specifically sequestering                     cells, unregulated Met4p activity in the met30&#226;&#710;&#8224; mutant <br /> the general transcription machinery away from promoters                     may hyperactivate a G1 cyclin RNA degradation mechan- <br /> of G1-specific genes.                                                       ism. Our results further indicate that any such pathway <br />                                                                             would presumably not affect CLN3 mRNA. In possible <br /> Potential mechanisms for the conditional met30&#226;&#710;&#8224;                             comparison, an SCF-like complex called VCB&#226;&#8364;&#8220;CUL2 <br /> G1 arrest                                                                   (comprised of VHL&#226;&#8364;&#8220;ElonginB/C&#226;&#8364;&#8220;CUL2) is thought to <br /> A variety of G1 mutants that arrest as unbudded cells with                  regulate, either directly or indirectly, accumulation of <br /> unreplicated DNA have been identified in S.cerevisiae,                      hypoxia-inducible mRNAs by targeting a protein for <br /> including cell division mutants (such as cdc28 and cln                      degradation in hypoxic conditions (reviewed in Kaelin <br /> mutants) and cell growth and translation mutants (such as                   and Maher, 1998; Tyers and Rottapel, 1999). <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span>- <br /> the Ras-cAMP and tor&#226;&#8364;&#8220; mutants; Hartwell, 1994; Nasmyth,                     dependent modifications of RNA stability have also been <br />  <br /> 1622 <br />                                                                                                              Role of Met4 at the G1&#226;&#8364;&#8220;S transition <br />  <br />  <br /> identified as a mechanism of glucose repression (Lombardo                    loss. Diploids were sporulated, and the progeny of each cross were <br />                                                                              analyzed (20 tetrads each, Figure 1B). For the crosses involving the <br /> et al., 1992; Cereghino and Scheffler, 1996). While control                  met3&#226;&#710;&#8224;, met6&#226;&#710;&#8224;, str4&#226;&#710;&#8224; or the double sam1&#226;&#710;&#8224;sam2&#226;&#710;&#8224; mutants with the <br /> over transcription, translation and protein stability has                    met30&#226;&#710;&#8224; LexA-MET30, HIS3 strain, the progeny tetrads contained two or <br /> been explored as a means to control G1 cyclin levels, to                     three viable spores, and those spores that were Leu&#226;&#171;&#185; (i.e. met30&#226;&#710;&#8224;) were <br /> our knowledge, our results constitute the first suggestion                   also His&#226;&#171;&#185; (i.e. LexA-Met30). The progeny of the cross involving the <br /> of a control at the level of G1 cyclin mRNA stability.                       met4&#226;&#710;&#8224; mutation showed a high number of tetrads giving rise to four <br />                                                                              viable spores (13 for a total of 20 tested tetrads) and, of 67 viable <br />                                                                              spores, 17 were found to be leucine prototrophs (i.e. met30&#226;&#710;&#8224;) and <br /> Methionine and the cell cycle                                                histidine auxotrophs (i.e. they did not harbor the pLexM30-4 plasmid). <br /> Connections between the cell cycle and methionine were                       In addition, all these 17 spores were found to be tryptophan prototrophs <br />                                                                              and methionine auxotrophs, proving that they were all met4&#226;&#710;&#8224;. Conversely, <br /> first identified by Unger and Hartwell (1976) with the                       no leucine prototroph spores that were both histidine and tryptophan <br /> study of met mutants that arrested in G1 phase in the                        auxotrophs were recovered. These results therefore suggested that met30&#226;&#710;&#8224; <br /> absence of methionine, and of methionine tRNA synthase                       spores could be recovered if they contain the met4&#226;&#710;&#8224; disruption mutation. <br /> mutants that arrested in G1 phase even in the presence of                    To confirm this result, a met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; spore was backcrossed to a wild- <br /> methionine. They predicted that a &#226;&#8364;&#732;signal&#226;&#8364;&#8482; allowed the                       type strain or to a met4&#226;&#710;&#8224; MET30 strain. In the former cross (CC808), <br />                                                                              only 25% of the recovered spores were leucine prototroph spores and <br /> cell to arrest specifically in G1 phase and acted at the                     all these spores were tryptophan prototrophs, while in the latter cross <br /> level of protein biosynthesis. Whether the met30&#226;&#710;&#8224; G1                         (CC809) the leucine character segregated perfectly 2&#226;&#171;&#185;/2&#226;&#8364;&#8220;. <br /> arrest is similar to the arrest described by Unger and <br />                                                                              Plasmids <br /> Hartwell (1976) has yet to be determined. DNA microarray                     To express the different Met4p derivatives from the GAL1 promoter <br /> analysis has also revealed that the expression of most of                    region from an autonomous replicating plasmid, we used the pGal316 <br /> the MET genes is specifically activated during S phase                       and pGal313 plasmids (Rouillon et al., 2000). They were cleaved by <br /> (Spellman et al., 1998), suggesting that cell cycle and                      EcoRI and BamHI and ligated with different EcoRI&#226;&#8364;&#8220;BamHI fragments <br />                                                                              encoding deleted derivatives of Met4p from the set of pLexMet4 plasmids <br /> sulfur amino acid metabolism are closely linked. Our                         described in Kuras and Thomas (1995). New Met4p derivatives were <br /> study reinforces this connection by linking the regulation                   constructed as described in Kuras and Thomas (1995) except that the <br /> of the main regulator of the MET network, Met4p, with                        ligations were made directly in the pGal313 or pGal316 plasmids cleaved <br /> the cell cycle. Important questions that arise from this                     by EcoRI and BamHI (Table III). <br /> work are whether and how regulation of Met4p might                              To integrate the GAL1&#226;&#8364;&#8220;MET4 fusion gene into the genome, we first <br />                                                                              cloned the StuI&#226;&#8364;&#8220;EcoRI fragment, blunt ended with Klenow fragment <br /> couple methionine with passage through G1 phase. Cell                        from plasmid pM4-1 (Thomas et al., 1992) into pGalM4-1, cleaved by <br /> size experiments and the rate of passage through Start in                    Asp718, blunt ended with Klenow fragment and dephosphorylated. Since <br /> synchronized, small G1 cells suggest that in the presence                    correct ligation events restored one Asp718 site, the resulting plasmid <br /> of high extracellular methionine, cells reset their critical                 contained one Asp718&#226;&#8364;&#8220;BamHI fragment corresponding to the GAL1&#226;&#8364;&#8220; <br />                                                                              MET4 gene fusion flanked by 5&#226;&#172;&#732; and 3&#226;&#172;&#732; regions of the MET4 gene. <br /> cell size to a larger volume and delayed passage through                     Owing to the one-step gene disruption method (Rothstein, 1983), this <br /> Start (E.E.Patton and M.Tyers, unpublished data). This                       fragment was used to replace the met4::TRP1 locus of strain CC849- <br /> initial finding is superficially at odds with the clear                      8A by the GAL1&#226;&#8364;&#8220;MET4 fusion gene by selecting for Met&#226;&#171;&#185; transformants <br /> inhibition of Start by excessive Met4p activity. However,                    in galactose medium. Correct integration events were verified by Southern <br /> as the met30&#226;&#710;&#8224; met32&#226;&#710;&#8224; mutant is a methionine prototroph,                      blot analyses. <br /> it is possible that a specific Met4&#226;&#8364;&#8220;Met32 transcription                      Northern blot analyses <br /> complex responds to a different signal than an increase                      Northern blotting was performed as described by Thomas (1980), with <br /> of intracellular AdoMet.                                                     total cellular RNA extracted from yeast as described by Schmitt et al. <br />                                                                              (1990) and oligolabeled probes (Hodgson and Fisk, 1987). <br />    Finally, the sulfur amino acid biosynthesis pathway is <br /> highly conserved in eukaryotes (Griffith, 1987). Connec-                     Flow cytometry <br /> tions between methionine metabolism and cell cycle                           For flow cytometry analyses, cells were harvested, fixed in 70% ethanol <br /> control have already been noted in mammals. Both in vitro                    and stained with propidium iodide as described (Mann et al., 1992). <br /> and in vivo studies have demonstrated the methionine                         Polysome profiles <br /> dependence of numerous transformed cells and tumors                          Polysomes were isolated as described (Zhong and Arndt, 1993). Briefly, <br /> (Hoffman, 1997). More recently, it was shown that athymic                    cells were grown to log phase, and 60 &#194;&#181;g/ml cycloheximide was added <br />                                                                              for 5 min before collecting the cells. Cells were lysed in a Tris-based <br /> mice that are grafted with human cancers and fed a                           buffer containing heparin, and extracts were loaded onto sucrose gradients <br /> methionine-free diet are greatly reduced in their tumor                      ranging from 7 to 47% sucrose, spun at 35 000 r.p.m. in an SV41 rotor <br /> burden (Kokkinakis et al., 1997). The results reported                       for 2 h, and read at A254 via continuous drip. <br /> here emphasize the importance of future studies on the <br /> coordination of nutrients, such as methionine, with the                    <br /> cell cycle.                                                                  <br />  <br /> Yeast strains and media                                                       <br /> Saccharomyces cerevisiae strains used in this work are listed in Table II.    <br /> They are all isogenic to the W303 background. Standard yeast media            <br /> were prepared as described by Che&#194;&#180; rest and Surdin-Kerjan (1992).            <br /> Saccharomyces cerevisiae was transformed after acetate chloride treat-       <br /> ment as described by Gietz et al. (1992). Genetic crosses, sporulation, <br /> dissection and scoring of nutritional markers were as described by            <br /> Sherman et al. (1979). <br />    For crosses as shown in Figure 1, diploids were selected and grown        <br /> for several generations in the presence of histidine to induce plasmid          <br />  <br /> </body></html>