<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Complexity Mycoplasma fermentans M64 Genome <br /> and Metabolic Essentiality Diversity among <br /> Mycoplasmas <br /> Hung-Wei Shu1., Tze-Tze Liu2., Huang-I Chan3, Yen-Ming Liu4, Keh-Ming Wu2, Hung-Yu Shu2&#194;&#164;, Shih- <br /> Feng Tsai2,4,5&quot;, Kwang-Jen Hsiao6,7&quot;, Wensi S. Hu1*&quot;, Wailap Victor Ng1,3*&quot; <br /> 1 Laboratory Science Medicine, Department Biotechnology, Institute Biotechnology Medicine, National Yang Ming University, Taipei, Taiwan, Republic China, <br /> 2 Genome Research Center, National Yang Ming University, Taipei, Taiwan, Republic China, 3 Institute Biomedical Informatics, National Yang Ming University, Taipei, <br /> Taiwan, Republic China, 4 Institute Genome Sciences, Department Life Sciences, National Yang Ming University, Taipei, Taiwan, Republic China, 5 Division of <br /> Molecular Genome Medicine, National Health Research Institute, Zhunan Town, Miaoli County, Taiwan, Republic China, 6 Department Medical Research and <br /> Education, Taipei Veterans General Hospital, Taipei, Taiwan, Republic China, 7 Department Education Research, Taipei City Hospital, Taipei, Taiwan, Republic of <br /> China <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Recently, genomes Mycoplasma fermentans strains, M64 JER, completely sequenced. <br />      Gross comparison indicated genome M64 significantly bigger strain difference mainly <br />      contributed repetitive sequences including seven families simple complex transposable elements ranging <br />      973 23,778 bps. Analysis repeats resulted identification new distinct family Integrative <br />      Conjugal Elements M. fermentans, designated ICEF-III. Using concept &#226;&#8364;&#732;&#226;&#8364;&#732;reaction connectivity&#226;&#8364;&#8482;&#226;&#8364;&#8482;, metabolic <br />      capabilities M. fermentans manifested complete partial connected biomodules revealed. A comparison of <br />      reported M. pulmonis, M. arthritidis, M. genitalium, B. subtilis, E. coli essential genes genes predicted the <br />      M64 genome indicated 73% Mycoplasmas essential genes preserved M. fermentans. Further <br />      examination highly partly connected reactions novel combinatorial phylogenetic tree, metabolic network, <br />      essential gene analysis indicated pathways (e.g. purine pyrimidine metabolisms) partial <br />      connected reactions important conversions intermediate metabolites. Taken together, light systems <br />      network analyses, diversity Mycoplasma species manifested variations limited <br />      metabolic abilities evolution. <br />  <br />   Citation: Shu H-W, Liu T-T, Chan H-I, Liu Y-M, Wu K-M, et al. (2012) Complexity Mycoplasma fermentans M64 Genome Metabolic Essentiality and <br />   Diversity Mycoplasmas. PLoS ONE 7(4): e32940. doi:10.1371/journal.pone.0032940 <br />   Editor: J. Ross Fitzgerald, University Edinburgh, United Kingdom <br />   Received September 27, 2011; Accepted February 2, 2012; Published April 3, 2012 <br />   Copyright: &#195;&#376; 2012 Shu et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Grant 96-2628-B-010-003-MY3 National Science Council intramural funding Aim Top <br />   University Grant awarded National Yang Ming University Ministry Education Taiwan, Republic China. The funders role study design, <br />   data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: huws@ym.edu.tw (WSH); wvng@ym.edu.tw (WVN) <br />   . These authors contributed equally work. <br />   &quot; These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                           crucial information better understanding interesting <br />                                                                                        group microorganisms. <br />    Mycoplasma, member class Mollicutes, genus lacking                       In spite intensive basic clinical research M. fermentans <br /> a rigid bacterial cell wall. A number species genus                     recent years, role involved molecular mechanism HIV <br /> medically agriculturally important. Extensive genetic                          pathogenesis, sexually-transmitted genital tract infection, systemic <br /> genomic investigations carried shed light                     infection, rheumatic disorders, chronic fatigue syndrome and <br /> biology pathogenicity [1,2,3,4,5,6,7,8,9,10]. In addition,                         diseases remained elusive [25,26,27,28,29,30,31,32, <br /> Mycoplasmas genetically simple bacteria drastically                           33,34,35]. M. fermentans fastidious microorganism isolated or <br /> reduced genomes interesting study                           detected commonly human genitourinary respiratory <br /> the presumably limited crucial metabolic capabilities                          tracts [36,37] correlated diseases healthy <br /> biological activities. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> date genomes species,                 individuals AIDS patients. It described several <br /> with sizes ranging 0.58 (M. genitalium) 1.36 Mbp (M.                           decades detection patients AIDS late 1980s <br /> penetrans), completely determined [4,5,6,7,10,11,                            considered opportunistic pathogen sexually <br /> 12,13,14,15,16,17,18,19,20,21,22,23,24] (http://www.ncbi.nlm.                          transmissible cofactor contributing pathology patho- <br /> nih.gov/genomes/lproks.cgi). These sequences contained basic                       genesis HIV-associated diseases [38]. The detection this <br />                                                                                        species peripheral blood lymphocytes urine AIDS <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                 1                                   April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                   Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> patients previous study suggested ability       ICEF (22.3 23.8 kb) included copies the <br /> to act polyclonal activators T B lymphocytes              previously sequenced ICEF-1A (23.8 kb) partly charac- <br /> stimulate replication HIV. It thought behave              terized ICEF-IIA, B, C (22.3 kb), complete (20.2 and <br /> cofactor immunomodulator HIV-related diseases [39]. M.                20.8 kb) partial (20.7 kb; 52 bp truncation 39-end) <br /> fermentans systemic pathogen, causing fatal disease              copies ICEF-III (ICEF-IIIA B) belonging new family <br /> owing infection bone marrow non-HIV-infected               discovered recent study [52]. Two copies (20.2 20.8 kb) <br /> patients [40]. It actively invade cultured cells HeLa           ICEF-III complete, truncated by <br /> cells survive intracellular pathogen [41]. They          IS1550A. In addition, copies WMFV1 prophage <br /> been present intracellularly adhered cell surface         DNAs (15.8 15.6 kb) inserted chromosome. A total of <br /> [42]. On hand, M. fermentans play critical role          15 complete 4 incomplete copies families IS <br /> genital tract infection rheumatic disorders rheumatoid          elements including IS1550 (973 1,416 bp; 6 complete <br /> arthritis [43,44]. In addition, studies conducted         copies, copy interrupted ICEF-IIA, copy <br /> to elucidate relationship Mycoplasma chronic fatigue             truncated transposition ICEF-IA), seven <br /> syndrome (CFS) [45]. These studies showed Mycoplasma species                IS1630 (1,178 1,384 bp; interrupted ICEF-IIIA), and <br /> including M. fermentans, M. hominis, M. penetrans              ISMf1 (1,395 1,570 bp; complete incomplete <br /> detected patients CFS, solid evidence           elements) identified. <br /> these organisms act cause CFS reported. In <br /> conclusion, M. fermentans clinically interesting Mycoplasma            M. fermentans strains M64 JER Significant <br /> species recent studies brought attention            Difference Large Transposable Elements <br /> possible involvement critical human diseases.                       Sequence comparison indicated genome M. fermentans <br />    Three types transposable elements including insertion                 M64 (1,118,751 bp) [52] 141 115 kb larger the <br /> sequences (IS), ICEFs (integrative conjugal element M.                   JER strain (977,524 bp) [21] nearly complete PG18 strain <br /> fermentans), prophage M. fermentans              (1,004,014 bp; Accession no. AP009608) [53]. The differences are <br /> strains [46,47,48,49]. These elements wide range size              mainly contributed differences copy numbers <br /> distribution ranging 973 23,778 bp present            ICEF WMFV1 prophage DNAs genomes (Figure 2). <br /> as multiple copies. For instance, copies IS1630,              Notably, gaps unknown size potential <br /> encoded IS30 family like transposase diverse target site         assembly problems PG18 genome, actual difference <br /> specificity, detected M. fermentans [47]. IS1550,         M64 genome remains accurately demon- <br /> known ISMi1, demonstrated possibly active               strated. The M. fermentans JER genome essentially lacked the <br /> element E. coli. In addition, IS1550                previously identified ICEF elements WMFV1 prophage DNA <br /> integrated IS1630 inserted large transposable            contained partial copies newly identified ICEF-III <br /> element ICEF [46,47,48,49]. ICEF similar conjugative self-            elements [48,49]. Regardless differences, organization <br /> transmissible integrating elements (constins),               M64 JER genome sequence largely similar. On <br /> absence homologs integrases, transposases recombi-             hand, apparently complete ICEFs one <br /> nases, considered distinct typical constins         WMFV1 prophage DNA complete <br /> mechanism integration-excision. Moreover, approximately              PG18 genome. Nonetheless, expectably, organization the <br /> 16-kb M. fermentans prophage WMFV1 integrated                   PG18 genome considerably different M64 <br /> single multiple copies genome [49]. Taken together,           possibly gaps sequence arrangement problems an <br /> presence elements important plasticity         incomplete genome. <br /> the genome evolution M. fermentans [49,50,51].                          Analyses integrity flanking sequences IS <br />    To increase understandings biology M. fermentans,          elements ICEFs suggested current M. fermentans M64 <br /> which helpful revealing pathogenic roles            genome architecture result the <br /> suspected diseases, comprehensive comparative genomic analysis            transposition and/or recombination events elements in <br /> was performed M. fermentans strains M64 JER            different periods time. Almost elements, for <br /> Mycoplasmas. We examined DNA sequences contrib-                   members IS1550A-1, -2 -3 subfamilies, polymorphic in <br /> uted dramatic difference size M64 JER             DNA sequences. As shown Figure 1, ICEFs (ICEF-I, - <br /> genomes. The metabolic abilities M. fermentans M64                  IIA, -IIB, -IIC) terminated 24-bp imperfect inverted <br /> analyzed systems analysis method based                repeats flanked distinct 8-bp target site duplications. The <br /> evaluation reaction connectivity. Finally, integrated analysis        new ICEF-IIIs, incomplete ICEF-IIIB is <br /> a phylogenetic tree, metabolic network, essential genes             disrupted IS1550A, terminated 23-bp imperfect <br /> carried uncover essentiality diversity metabolic          inverted repeats flanked 8-bp target site duplications. <br /> reactions M. fermentans M64 evolution.                            Sequence comparison ICEF-IA, IIA, IIIA phylogenetic <br />                                                                             analysis 11 intact ICEs (Integrative Conjugal Elements) six <br /> Results Discussion                                                      Mycoplasma species indicated ICEF-III distinct the <br />                                                                             ICEF families M. fermentans unique reported <br /> M. fermentans M64 Harbors Large Number                                 ICEs (Figure S3). Both copies WMFV1 prophage genomes are <br /> Transposable Elements                                                       terminated 4-bp inverted repeats flanked identical 6- <br />   Similar Mycoplasma mycoides subsp. mycoides [16], sequence             bp (TTTTTA) target site duplications, suggesting could <br /> analyses indicated M. fermentans M64 genome                   preferred transposition target sequence, reported <br /> possessed high density transposable elements (Figures S1               previously [49]. The seven complete IS1550 terminated with <br /> and S2) [52]. Nine copies types large repetitive sequences        29-bp imperfect inverted repeats flanked by <br /> and copies relative small insertion sequence (IS) elements          3-bp direct repeats. The seven complete IS1630 terminated <br /> accounted 21.6% genome (Table 1 Figure 1)                    27-bp heterogeneous inverted repeats these <br /> [46,47,48,49]. Among large repeats families               elements flanked 16-, 23-, 25-, 26-bp duplicated <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                            April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                          Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  Table 1. General features M. fermentans M64 genome. <br />  <br />  <br />                                                                  Repetitive sequences included*             Repetitive sequences excluded* <br />  <br />  Genome size (bp)                                                1,118,751                                  887,179 <br />  GC content (%)                                                  26.9                                       27 <br />  Gene density (rRNA and, tRNA genes included) (%)                89.7                                       90.6 <br />  Total number CDSs                                            1050                                       820 <br />    Number CDSs annotated function                        576 (54.9%)                                507 (61.8%) <br />    Number hypothetical proteins                               332 (31.6%)                                210 (25.6%) <br />    Number conserved hypothetical proteins                     142 (13.5%)                                103 (12.6%) <br />  Average protein <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> (a.a.)                                   314                                        321 <br />  Start codon usage <br />    AUG                                                           963 (91.7%)                                752 (91.7%) <br />    UUG                                                           55 (5.2%)                                  43 (5.2%) <br />    GUG                                                           32 (3.1%)                                  25 (3.1%) <br />  Total number rRNA genes                                      5                                          3* <br />  Total number tRNA genes                                      35                                         35 <br />  Repetitive transposable elements <br />    IS elements                                                   19 (4 incomplete)                          - <br />    ICEFs                                                         7                                          - <br />    Prophage, WMFV1                                               2                                          - <br />    Total length repetitive sequences (bp)                     241,287                                    - <br />    Repetitive sequences (%)                                      21.6                                       - <br />  <br />  *Repetitive sequences included transposable elements duplicated 16 23S rRNA. <br />  doi:10.1371/journal.pone.0032940.t001 <br />  <br />  <br /> junction sequences. The complete copies ISMf1 contain                     PG18 (872) M64 (849) JER (774), corroborating the <br /> 47-bp imperfect terminal inverted repeat                     closer relationship M64 PG18 obtained the <br /> noticeable target site duplication. The absence target site                     analysis concatenated sequence dnaE, pyk, rpoA <br /> duplication elements suggested post-                         (Figure 3B, Table S1). Expectably, M64 largest <br /> transposition DNA rearrangements additional transposition                       chromosome rich mobile elements possesses strain <br /> events occurred prior strain evolution                   specific genes (83 genes) JER (17) PG18 (11). Further <br /> families transposable elements, WMFV1 prophage,                      sequence analysis indicated strain specific genes encoded <br /> are present M. fermentans strains M64, JER, PG18. The high                  proteins annotated function. Based annotations in <br /> frequency DNA rearrangements M. fermentans shown                    KEGG Orthology database, gene pool core genome <br /> by Hu et al. previous study [51].                                             (762,837 genes) contained 378 well-annotated orthologous <br />                                                                                    groups genes approximately half belonging genetic <br /> Pan-genome Analysis Across M64, JER, PG18                                  information processing biomodules (Figure 3C). <br /> Strains <br />    To unveil variations gene content M. fermentans                    M. fermentans M64 Metabolic Capabilities Revealed by <br /> M64, JER, PG18, sequence similarity searches                        Reaction Connectivity-Based Analyses <br /> predicted proteins carried using BLASTP dissect                    Global analyses M. fermentans M64 metabolism indicated only <br /> gene repertoire strains. The homologs recognized                     131 1,050 predicted proteins mapped metabolic <br /> under criterion E-value ,1025. Due differential activities               pathways KEGG database (KEGG Mapper; http://www. <br /> of events gene duplication and/or horizontal gene transfer,                 genome.jp/kegg/tool/map_pathway1.html). Apparently, a <br /> the number genes shared present                      relatively large number proteins carbohydrate (57 proteins) <br /> strains different. For instance, gene MfeM64YM_0098                   nucleotide (31 proteins) metabolic networks, scattered <br /> of M64 total 8 homologs (MfeM64YM_0098, Mfe-                              distribution proteins amino acid (25 proteins), cofactors and <br /> M64YM_0108, MfeM64YM_0229, MfeM64YM_0238, Mfe-                                     vitamin (18 proteins), lipid (8 proteins) metabolic networks <br /> M64YM_0385, MfeM64YM_0474, MfeM64YM_0483, Mfe-                                 (Figure S4). A gross examination reactions predicted <br /> M64YM_0992), PG18 JER, respectively, 4                        proteins global metabolic network immediately perceived <br /> (MBIO_0117, MBIO_0267, MBIO_0276, MBIO_0750) 5                             M. fermentans synthesize degrade limited <br /> (MFE_08000, MFE_02760, MFE_02860, MFE_04903,                                   number metabolites established Mycoplasma <br /> MFE_04960) gene homologs. Thus, pan-genome analysis exhib-                         species. <br /> ited 762,837 genes constituting core genome M. fermentans                      Evaluation functionality 58 pathways M64 <br /> strains M64 (837), JER (762), PG18 (809) (Figure 3B). In                       proteins examination connectivity reactions [54] <br /> agreement phylogenetic relationship                     pathways contained connected <br /> strains (Figure 3A), gene homologs shared M64 (955)                    reactions (Table 2). Several carbohydrate metabolic pathways <br />  <br />  <br />        PLoS ONE | www.plosone.org                                              3                            April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                     Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 1. Terminal junction sequences IS elements, ICEFs, WMFV1 prophage DNAs M. fermentans M64 <br /> genome. The terminal inverted repeats flanking sequences 15 complete 4 partial copies families IS elements including the <br /> IS1550, IS1630, ISMf1, 7 copies families ICEFs, 2 prophage genomes shown. Bold uppercase letters arrows indicate <br /> the terminal inverted repeats. The target site duplications shown underlined bold lowercase. The elements interrupted transposable <br /> elements indicated. <br /> doi:10.1371/journal.pone.0032940.g001 <br />  <br /> seemed sufficient connected reactions drive synthesis         reactions driven predicted proteins. Due lack of <br /> or degradation pertinent metabolites. The presence               citrate cycle enzymes, use D-glucose fermentative <br /> complete glycolysis/gluconeogenesis pathway (18 connected                    degradation. On hand, pentose phosphate pathway, <br /> reactions) suggested M. fermentans M64 capable                    contained large cluster 13 small cluster two <br /> catabolizing D-glucose pyruvate using Embden-Meyerh-                  connected reactions, utilized synthesize phosphoribosyl <br /> off-Parnas (EMP) pathway. The pyruvate converted                 diphosphate (PRPP) enter purine (11 connected <br /> to acetyl-CoA, acetate, D-lactate pyruvate metabolism            reactions) pyrimidine (10 reactions) metabolism pathways for <br /> pathway contained single cluster connected                   nucleotide synthesis. Although 6 genes mapped the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        4                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                     Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 2. Dotplot genomes M. fermentans strains M64 versus JER (A) PG18 (B). The graphs generated Gepard <br /> 1.21 word length 20 window size 0 [78]. The long repetitive sequences indicated horizontal arrows: ICEF-I -II (unfilled <br /> arrows), ICEF-III (black arrows), WMFV1 (gray arrows). The locations IS elements ISMf1, IS1550 IS1630 families indicated by <br /> small black gray (elements interrupted ICEFs) triangles. The types non-IS small duplications indicated small filled (16&#226;&#8364;&#8220;23S rRNA <br /> operon) open triangles (uncharacterized repetitive sequence containing hypothetical protein-coding gene). <br /> doi:10.1371/journal.pone.0032940.g002 <br />  <br /> fructose mannose metabolism pathway, encoded                       microorganism. For instance, proline iminopeptidase (Pip) may <br /> proteins form cluster connected reactions drive              release proline peptide aspartate-ammonia ligase <br /> conversion D-fructose glyceraldehyde-3-phosphate,                (AsnA) catalyzes interconversion aspartate asparagine. <br /> may enter glycolysis pathway degradation.               In addition, S-adenosylmethionine synthetase (MetK) an <br />     Neither purine pyrimidine pathways novo               indispensable enzyme conversion methionine S- <br /> nucleotides synthesis complete M. fermentans           adenosyl-L-methionine, used substrate of <br /> M64 rely external supply nucleotides nucleotide                cytosine-specific methyltransferase (Dcm) DNA, rRNA <br /> synthesis intermediates produce building blocks DNA            methylation. Therefore, overall network connectivity <br /> RNA (Table 2 Figure S5). Apparently, M. fermentans M64                   analysis suggested M. fermentans M64 rely external <br /> possesses enzymes form subnetworks connected reactions             supply amino acids protein synthesis. <br /> between RNA synthesis metabolism (i) guanosine, <br /> guanine, GMP, GDP, GTP, (ii) adenosine, adenine, AMP,                    Conservation Genes Biomodules the <br /> ADP, ATP, (iii) UTP CTP; DNA                         Mycoplasmas <br /> synthesis metabolisms (iv) guanosine, guanine,                       Clustering analysis proteins 27 sequenced <br /> deoxyguanosine, dGMP, dGDP, dGTP, (v) dAMP,                          Mycoplasma Phytoplasma species indicated genes <br /> dADP, dATP.                                                              responsible DNA replication, nucleotide excision, homologous <br />     M. fermentans M64, similar Mycoplasma              recombination, transcription, translation significant <br /> species, amino acid metabolism proteins predicted               fraction membrane transporters including ABC <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> genome. Only amino acid metabolic pathways               transporters members protein transport bacterial <br /> had connected reactions &#226;&#8364;&#8220; &#226;&#8364;&#732;&#226;&#8364;&#732;arginine praline&#226;&#8364;&#8482;&#226;&#8364;&#8482;                   secretion highly conserved species <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;cysteine methionine&#226;&#8364;&#8482;&#226;&#8364;&#8482; pathways clusters 3 2,                (Figure 4). The genes related carbohydrate metabolism are <br /> respectively, reactions. The presence arginine deiminase (ArcA),          conserved species M. haemofelis, M. suis, M. <br /> ornithine carbamoyltransferase (ArcB), carbamate kinase                  arthritidis, M. hominis. Several genes accounting pyruvate <br /> (ArcC) formed cluster reactions catalyzing degradation              metabolism carbohydrate metabolisms lost in <br /> of arginine ammonia &#226;&#8364;&#732;&#226;&#8364;&#732;arginine proline metabolism&#226;&#8364;&#8482;&#226;&#8364;&#8482;            them, pdhA, pdhB, pdhC, pdhD. Moreover, distinct <br /> is agreement previous study M. fermentans           patterns metabolisms observed M. <br /> shown capable utilizing arginine (62). Notwithstanding              haemofelis M. suis. Intriguingly, M. haemofelis M. suis are <br /> some proteins did form highly connected reaction                  hemotropic Mycoplasmas. They belong special group of <br /> clusters, important <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span>              Mycoplasmas, known hemoplasma, tropism red <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                     Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 3. Overview pan-genome analysis M. fermentans strains M64, JER, PG18. A. The phylogenetic tree the <br /> M. fermentans strains M64, JER, PG18. The concatenated sequences genes dnaE, pyk, rpoA shared strains M. fermentans <br /> were used phylogenetic analysis. The maximum-likelihood method performed reconstruct tree using PHYML 3.0. The scale bar stands <br /> for estimated number nucleotide substitutions site. B. The pan-genome M. fermentans strains M64, JER, PG18. Venn diagram <br /> exhibits number (including duplications) genes unique strain shared strains (BLASTP E-value ,1025). Superscripts represent <br /> the initials strains indicated numbers genes homologies strains. The numbers genes nearly <br /> complete PG18 genome slightly underestimated presence gaps potential unresolved duplications genome assembly. C. <br /> Functional distribution 378 orthologous groups genes core genome. Some proteins KEGG pathway/biomodule <br /> annotation. <br /> doi:10.1371/journal.pone.0032940.g003 <br />  <br /> blood cells [55,56]. This distinctive propensity associated           carry glucose fermentation capable arginine <br /> with evolution pool metabolism- cellular process-          hydrolysis [57,58]. All Mycoplasma species exhibited <br /> related genes. M. arthritidis M. hominis, hosts rodent         conserved energy metabolism-related genes including genes <br /> and human respectively, form unique phylogenetic clade,               encoding NADH dehydrogenase F-type ATPase aside from <br /> referred M. hominis cluster, Mycoplasmas. They             cytochrome-containing complex, indicating truncated electron <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        6                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                               Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  Table 2. Summary number M. fermentans M64 predicted genes, reactions, connected reactions KEGG metabolic <br />  pathways. <br />  <br />  <br />                                                                                                                                        Connectivity in <br />  Metabolism                              KEGG Pathway                                              # Genes             # Reactions     Reactions <br />  <br />  Amino Acid                              Arginine proline metabolism                           5                   5               3,1,1 <br />                                          Cysteine methionine metabolism                        2                   2               2 <br />  Carbohydrate                            Glycolysis/Gluconeogenesis                                15                  18              17,1 <br />                                          Pentose phosphate pathway                                 12                  15              13,2 <br />                                          Pyruvate metabolism                                       9                   8               8 <br />                                          Amino sugar nucleotide sugar metabolism               10                  11              7,2,2 <br />                                          Fructose mannose metabolism                           5                   6               6 <br />                                          Galactose metabolism                                      9                   7               2,1,1,1,1,1 <br />                                          Pentose glucuronate interconversions                  6                   6               2,1,1,1,1 <br />                                          Starch sucrose metabolism                             4                   5               2,1,1,1 <br />                                          Propanoate metabolism                                     2                   2               2 <br />  Cofactors Vitamins                  Nicotinate nicotinamide metabolism                    5                   6               5,1 <br />                                          Pantothenate CoA biosynthesis                         3                   3               3 <br />                                          One carbon pool folate                                 3                   4               4 <br />                                          Thiamine metabolism                                       2                   2               2 <br />                                          Riboflavin metabolism                                     1                   2               2 <br />                                          Lipoic acid metabolism                                    1                   2               2 <br />  Energy                                  Oxidative phosphorylation                                 9                   2               2 <br />  Lipid                                   Glycerophospholipid metabolism                            6                   5               5 <br />                                          Glycerolipid metabolism                                   3                   2               2 <br />  Nucleotide                              Pyrimidine metabolism                                     22                  22              8,3,2,2,2,1,1(64) <br />                                          Purine metabolism                                         20                  27              6,6,3,3,3,2,2,1,1 <br />  Other Amino Acids                       Selenoamino acid metabolism                               3                   5               2,1,1,1 <br />  <br />  *The number reactions &#226;&#8364;&#732;&#226;&#8364;&#732;group linked reactions&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;orphan reaction&#226;&#8364;&#8482;&#226;&#8364;&#8482; separated commas. <br />  doi:10.1371/journal.pone.0032940.t002 <br />  <br />  <br /> transport chain. Thus, ATP generation proceeds inefficient                   structure Mycoplasmas subunit protein resembles the <br /> substrate-level phosphorylation flavin-terminated respirato-                  PolC-type (Gram (+) type) subunit, contains the <br /> ry pathway oxidative phosphorylation [59,60,61]. In                      DnaE (a subunit) DnaQ (e subunit) domains DNA <br /> addition, pentose phosphate pathway, alternative                           polymerase 39&#226;&#8364;&#8220;59 exonuclease, respectively, activities suggest- <br /> glycolysis, generate NADPH reductive biosynthesis                            ing B. subtilis-like mechanism DNA replication. <br /> reactions Mycoplasma species. The presence limited                         The DNA-directed RNA polymerase a, b, b9 subunits are <br /> number recognizable genes encoding proteins                            sufficient form minimal core (bb9a2) RNA polymerase for <br /> biomodules metabolisms carbohydrates, amino acids,                       catalyzing polymerization nucleoside triphosphates into <br /> and cofactors vitamins conserved, suggesting most,                   RNA [64]. The RNA polymerase primary s factor, which <br /> not all, metabolic pathways defective unless                  promotes attachment RNA polymerase specific site in <br /> cryptic genes carry missed function [62]. This                     promoter transcription initiation, highly conserved <br /> observation agreement requirements wide                          sequenced Mycoplasma species. In addition these <br /> spectrum substrates factors growth Mycoplasma.                     subunits, d subunit ubiquitous Gram (+) <br />     The analysis proteins involved genetic information                     bacteria conserved Mycoplasma <br /> processing unambiguously indicated widely distrib-                     species including M. haemofelis, M. suis, M. leachii, M. capricolum, M. <br /> uted members sequenced Mycoplasmataceae family                      mycoides, M. penetrans, M. pneumoniae, M. genitalium, M. <br /> (Figure 4). Interestingly, nucleotide metabolism genetic                     gallisepticum M. fermentans. Although gene <br /> information processing deservedly exhibited highly conserved                         (MfeM64YM_0186) encoding protein containing domain <br /> profiles involved genes, M. haemofelis M. suis                 RNA polymerase d subunit (pfam05066) present the <br /> to deficient genes. The DNA polymerase III                                chromosome M. fermentans M64, E-value 6.361025 <br /> holoenzyme M. fermentans Mycoplasma species                         protein size (89 a.a.) markedly smaller those <br /> resembles Gram (+) type simpler Gram                     (134,170 a.a.) sequenced Mycoplasmas. Importantly, the <br /> (-) model organism [63]. The Mycoplasma species polymerase III                       sequence similarity analysis protein MfeM64YM_0186 <br /> holoenzymes subunits                          demonstrated significant hits (E-value,1025) were <br /> Gram (+) Bacillus subtilis comprises a, b, c, t, d, d9                 annotated putative uncharacterized protein. Therefore, Mfe- <br /> subunits. Although gene encoding e subunit 39&#226;&#8364;&#8220;59                        M64YM_0186 considered hypothetical protein might <br /> exonuclease activity genomes,                        degenerated typical RNA polymerase d subunit. The d <br />  <br />  <br />          PLoS ONE | www.plosone.org                                              7                                    April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                         Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 4. Clustering analysis metabolic cellular process proteins 27 Mycoplasma species/strains Phytoplasma OY. <br /> The order Mycoplasma species according phylogenetic relationship (23S rRNA tree). Each column represents proteins one <br /> single species/strain row (horizontal line) represents copy number (grey color scale top) gene. The sizes genomes in <br /> Mbp indicated 3-letter abbreviated names organisms. Functional clustering comparison proteins species were <br /> based KO (KEGG Orthology) assignments. The conservation gene Mycoplasma species (i.e. number Mycoplasma species <br /> possessing gene) shown right clustering result. Abbreviations &#226;&#8364;&#8220; mha: M. haemofelis; mss: M. suis Illinois; msk: M. suis KI3806; mpe: <br /> M. penetrans; mga: M. gallisepticum; mge: M. genitalium; mpn: M. pneumoniae; mmy: M. mycoides; mcp: M. capricolum; mlc: M. leachii; mat: M. <br /> arthritidis; mho: M. hominis; mmo: M. mobile; mhr: M. hyorhinis; mhy: M. hyopneumoniae 232; mhj: M. hyopneumoniae J; mhp: M. hyopneumoniae 7448; <br /> mco: M. conjunctivae; mpu: M. pulmonis; mfm: M. fermentans M64; mfr: M. fermentans JER; maa: M. agalactiae PG2; mal: M. agalactiae 5632; mbv: M. <br /> bovis PG45; msy: M. synoviae; mcd: M. crocodyli; poy: Phytoplasma OY. <br /> doi:10.1371/journal.pone.0032940.g004 <br />  <br /> subunit involved initiation recycling phases              recycling. Previous studies reported <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> d subunit <br /> transcription, serve virulence factor [65]. The               gene result extended lag phase growth [66] defect <br /> specificity transcription efficiency RNA synthesis            starvation-induced stationary-phase survival recovery [67]. <br /> increased presence d subunit enhanced                  This subunit does exist Mycoplasma species likely <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           8                               April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                       Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> responsible complex regulation flexibility gene                 conserved analyzed species. On hand, the <br /> expression Mycoplasma species bearing d subunit genes.            proteins essential all <br />    The aminoacy-tRNA synthetases ribosomal genes                 species. It conceivable genome reduction was <br /> a high degree conservation sequenced Mycoplasmas.                 accompanied shift essential gene sets the <br /> Similar Gram (+) B. subtilis, Mycoplasmas possess                evolution shown carbohydrate metabolism membrane <br /> glutaminyl-tRNA synthetase. They similar strategy                transport (Figure 5). <br /> B. subtilis organisms lack enzyme charging <br /> glutamate tRNAGlu tRNAGln nondiscriminat-                   Conservation Residual Metabolism Essential <br /> ing glutamyl-tRNA synthetase converting glutamyl-                     Genes Mycoplasmas <br /> tRNAGln glutaminyl-tRNAGln glutamyl-tRNAGln amido-                        Analysis conservation genes the <br /> transferase (GatABC complex). This partly supported                      Mycoplasmas indicated residual partial <br /> observation M. fermentans 21 Mycoplasma species             metabolic capabilities M. fermentans highly conserved among <br /> and strains protein homolog (MfeM64YM_0687 M.                        Mycoplasma species. A cross examination essential genes <br /> fermentans) similar nondiscriminating glutamyl-tRNA                M. pulmonis [69], M. arthritidis [12], M. genitalium [70] <br /> synthetase B. subtilis (61.7% similarity) discriminating           suggested conserved genes residual <br /> E. coli enzyme (53% similarity). Mycoplasma species slightly          metabolic functions likely essential Mycoplasmas. The <br /> ribosomal subunit proteins E. coli (total 55).                   incomplete nucleotide carbohydrate metabolism chosen <br /> Furthermore, genes encoding GatABC complex                    examples illustrate property. Notwithstanding <br /> present M. fermentans M64 sequenced Mycoplasmas.                  Mycoplasmas lack capability novo nucleotide synthesis, <br /> Upon manually curated annotations KEGG database                       total 21 22 genes purine pyrimidine metabolic <br /> (http://www.genome.jp/kegg/pathway/ko/ko03010.html)                        pathways, respectively, M. fermentans genome. <br /> validations smaller proteins (,100 a.a.) performing                     Phylogenetic comparison nucleotide metabolic networks <br /> TBLASTN, total 47 proteins ribosome complex                    indicated genes present M. fermentans more <br /> highly conserved sequenced Mycoplasma species                    conserved purine metabolism pyrimidine <br /> strains. The S1, S6, S21, L9, L32 subunits absent              pathway 21 sequenced Mycoplasma species (Figure 6A). <br /> of species S22, L25, L30 subunits               The enzymes catalyzed clusters connected reactions in <br /> in species. In line observations, Kawauchi et al.          purine metabolic pathway, Dgk DeoB present <br /> [68] employed two-dimensional polyacrylamide gel electrophore-                 species, respectively, highly conserved <br /> sis analyze M. capricolum ribosomes, identified 30                  Mycoplasmas. Most species enzymes the <br /> proteins large ribosomal subunit 21 proteins small                   synthesis ATP adenine adenosine, GTP dGTP <br /> ribosomal subunit. The M. capricolum ribosome lacks S1, S22, L25,              guanine guanosine. However, phylogenetically <br /> and L30 proteins. S1 protein present seven species belonging             related species - M. fermentans, M. agalactiae, M. bovis M. <br /> to close phylogenetic clusters including M. lipophilum, M.           lipophilum cluster related species - M. arthritidis and <br /> synoviae, M. pulmonis clusters [58] species.                 M. hominis M. hominis cluster [58] lack ribonucleotide <br /> Furthermore, S21 protein restrictively exists species           diphosphate reductase complex conversion ADP to <br /> hemoplasma, pneumoniae, spiroplasma groups [57]. Togeth-                   dADP dATP synthesized dADP dAMP <br /> er, observations suggested proteins               uptake extracellular source. On hand, the <br /> show specificity certain phylogenetic clades                    enzymes pyrimidine metabolic pathway relatively less <br /> dispensable Mycoplasmas reported previously [12,69].                     conserved Mycoplasmas. Due lack one <br />                                                                                key enzyme, sequenced species rely external supply of <br /> Potential Essential Genes M. Fermentans                                     UTP, dCTP, dTTP hosts culture media. For <br />    The essential genes closely related M. pulmonis, M.                instance, M. fermentans short genes encoding dCTP deaminase <br /> arthritidis, M. genitalium distantly related B. subtilis       involved conversion CTP UTP nucleoside- <br /> E. coli identified previous studies compared                  diphosphate kinase responsible conversion UDP UTP, <br /> proteins predicted M. fermentans M64 genome shed                   dCDP dCTP, dTDP dTTP. Moreover, ribonucleoside- <br /> light M. fermentans likely essential genes necessary minimal          triphosphate reductase catalyzing conversion CTP dCTP <br /> self-replicating cell (Figure 5) [12,69,70,71,72]. It notable          absent it. Some species M. fermentans carry <br /> more essential genes M. pulmonis, M. arthritidis, M. genitalium         pyrG gene able convert UTP CTP. <br /> than species homologs M. fermentans.                        Of carbohydrate metabolism, M. fermentans M64 the <br /> Approximately 303 (97.7%) 310 M. pulmonis, 315 (75.7%)               Mycoplasma species possess highly conserved <br /> the 416 M. genitalium, 282 (73.8%) 382 M. genitalium, 150               glycolysis pathway incomplete partially <br /> (55.3%) 271 B. subtilis, 168 (55.4%) 303 E. coli             conserved pentose phosphate pyruvate metabolic networks <br /> essential gene homologs M. fermentans M64                    (Figure 6B). Upstream glycolysis pathway, a-D-glucose b- <br /> genome. Although matched essential genes M. fermentans                  D-glucose enter pathway enzyme GlcK <br /> have validated, subset essential genes Mycoplasma            M. fermentans M64 seven other <br /> species distinct distantly related                species. The joint presence AmyC MalL species <br /> bacteria identified. The higher similarity M. fermentans          including M. fermentans, M. pulmonis, M. hyorhinis, M. crocodyli <br /> M. pulmonis M. arthritidis M. genitalium                   suggested starch converted dextran a- <br /> accordance phylogenetic distance (Figure S6). A                     D-glucose subsequently enter glycolysis pathway these <br /> relatively higher degree conservation observed                   species. In addition, presence FruA, FruK, Fba, <br /> essential genes participating translation transcription. In             different degree conservation, suggested that <br /> addition, significant fraction essential genes involved            fructose utilized M. fermentans M64 some <br /> replication repair, nucleotide metabolism, amino acid                      Mycoplasma species. In contrast highly conserved Fba <br /> metabolism, membrane transport appeared highly                  interconverts D-fructose 1,6-bisphosphate glycerone <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          9                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                      Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 5. Comparison predicted genes M. fermentans M64 reported essential genes bacteria species. BLASTP <br /> was implemented compare proteome M. fermentans M64 (mfm) proteomes M. pulmonis (mpu), M. arthritidis (mat), M. <br /> genitalium (mge), B. subtilis (bsu), E. coli (eco). The gene names (or KEGG Orthology numbers) KEGG functional categories indicated. Black <br /> rectangles indicate essential genes mpu, mat, mge, bsu, eco. Light grey rectangles indicate genes shown be <br /> essential. Dark gray rectangles indicate potential essential genes M. fermentans. White rectangles indicate missing genes. The genes <br /> involved functional category indicated asterisk. <br /> doi:10.1371/journal.pone.0032940.g005 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        10                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                     Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br />  <br />  <br /> Figure 6. Phylogenetic conservation analysis nucleotide (A) carbohydrate (B) metabolic networks M. fermentans M64 and <br /> 20 Mycoplasma species. M. fermentans M64 metabolic networks served references comparisons. The small open squares <br /> represent compounds colored circles represent proteins (enzymes) participating metabolisms. Circles light blue <br /> circumference denote potential essential genes inferred comparison M. pulmonis, M. arthritidis, M. genitalium (Figure 5). The <br /> arrows lines linking enzymes substrates indicate direction reactions. Enzymes involved metabolism are <br /> annotated single letter <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> metabolisms (A: Amino acids; C: Carbohydrate; N: Nucleotide; V: cofactors Vitamins) the <br /> circles. The color scale (bottom) indicates phylogenetic conservation proteins analyzed Mycoplasma species. <br /> doi:10.1371/journal.pone.0032940.g006 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       11                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                     Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> phosphate plus D-glyceraldehyde 3-phosphate, FruA does                   different organisms BLASTP results, further, M. <br /> exist hemoplasma species species M.             fermentans M64 organisms analyzed the <br /> hominis group including M. lipophilum, M. hominis, M. synoviae           23S rRNA phylogeny elucidate evolutionary relationship. <br /> clusters. FruK present species belonging           Regarding ORF MfeM64YM_1027, M. fermentans M64 <br /> spiroplasma group, species pneumoniae group,              occupied distinct, individual branch strong branch support <br /> species hominis group including M. hyorhinis M.                   distant Clostridium perfringens 23S rRNA <br /> fermentans. Within pentose phosphate pathway, enzymes                phylogenetic tree (Figure S8A). However, M. fermentans M64 <br /> Tkt, Rpe, RpiB conversion D-glyceraldehyde 3-                 MfeM64YM_1027 closest homolog C. perfringens, <br /> phosphate D-ribulose 5-phosphate conserved species             raising possibility gene remnant after <br /> except hemoplasma. The enzyme PrsA, converts D-                genome minimization (Figure S8B). Similar result also <br /> ribulose 5-phosphate 5-Phospho-alpha-D-ribose 1-diphosphate               observed gene (MfeM64YM_0060) product <br /> (PRPP) Mycoplasma species. Many Mycoplasma             closest homolog Gram-negative bacterium (Figure <br /> species able catabolize pyruvate acetyl-CoA             S8C D). Taken together, preserved genes onset of <br /> and acetate conserved pathway,                   genome reduction horizontally transferred gene candidates <br /> have complete set enzymes metabolize pyruvate             appeared play critical roles versatility small <br /> lactate ethanol. M. fermentans, M. agalactiae, M. bovis, M.           genome shaping limited metabolism. <br /> crocodyli hominis group, M. mycoides M. leachii the <br /> spiroplasma group, M. penetrans lactate dehydrogenase               Conclusion <br /> catalyzing interconversion lactate pyruvate. Of                 The genome M. fermentans M64, despite interference a <br /> these species, M. fermentans, M. crocodyli, M. penetrans        large proportion repetitive sequences assembling the <br /> capable converting acetate ethanol exerting aldehyde                genome shotgun sequences, successfully deter- <br /> dehydrogenase alcohol dehydrogenase exist                mined previous study [52]. Rare conservation of <br /> couple species. This implies M. fermentans M64           transposable elements Mycoplasmas suggests most <br /> other Mycoplasma species able undergo lactic alcoholic          elements integrated genomes speciation. <br /> fermentation. Similarly, enzymes incomplete                  Clustering analysis Mycoplasma genes indicated conservations <br /> amino acid vitamin cofactor metabolic networks                  degenerated metabolisms cellular processes, suggesting that <br /> examined, enzymes conserved,                    remaining metabolic functions indispensable. <br /> suggesting residual metabolic activities           On account lack complete amino acid, nucleotide, and <br /> may related survival (figure S7).                                vitamin cofactor synthesis pathways, compounds their <br />                                                                              intermediate metabolites obtained external sources <br /> Genes Outside Mycoplasma Clade                                        entered cells forms peptides, amino acids, <br />    Sequence analysis identified 15 genes M. fermentans M64           nucleotides poorly understood transport systems. The <br /> no homolog significant BLASTP hits coming                  completion M64 JER strains M. fermentans the <br /> Mycoplasma species, suggesting conserved            Mycoplasma genomes facilitate revelation these <br /> Mycoplasma clade (See <span id='am-3' about='rdfs:comment' typeof='owl:Thing'>comment</span> Table S2). These genes                  systems biology interesting group microorgan- <br /> might vestiges genome reduction                  isms reduced genomes. <br /> common Gram-positive ancestor, horizontally trans- <br /> ferred genes. The genes dispersedly distributed                  Materials Methods <br /> chromosome. Among these, 7 genes closest Gram- <br /> positive bacteria, 8 genes closest Gram-negative                 Comparative Metabolomics <br /> bacteria. All 7 Gram-positive bacteria belonged Firmicutes,               The metabolism M. fermentans M64 Mycoplasma <br /> of 6 belonged Clostridia. Those 7 genes highest                species compared clustering analysis homologous <br /> homologies Firmicutes regarded possible preserved             genes. The clustering analysis comparison presence and <br /> genes onset evolution genome minimization. In            absence genes different functional groups based the <br /> addition, 8 genes closest Gram-negative bacteria, including           KO (KEGG Orthology) assignments homologous proteins <br /> 4 species belonging Bacteroidetes/Chlorobi group, 3                    KEGG database. M. fermentans M64 protein sequences were <br /> Cyanobacteria, 1 Spirochaetes, trans-                 submitted KEGG Automatic Annotation Server (KAAS) <br /> ferred laterally M. fermentans M64. No UGA codon detected             [74] obtain KO number pathway assignments. While <br /> in 8 putative horizontally transferred gene candidates,                similar information Mycoplasma species Phytoplasma <br /> suggesting genes transferred            OY extracted organism-specific files KEGG <br /> long period insufficient induce codon bias. The          database. <br /> proteins encoded 15 genes consisted 5 conserved <br /> hypothetical proteins, 3 AAA+superfamily ATPases, 7                      Conservation Metabolisms <br /> different proteins. During evolution M. fermentans M64,                  A new method evaluating phylogenetic conservation of <br /> AAA+superfamily ATPase play important role                   metabolism genus/group organisms developed to <br /> compensation genome minimization protein                 predict conserved metabolism Mycoplasma <br /> accountable large variety diverse functions molecular           species. This analysis based hypothesis the <br /> remodeling dependent energy ATP hydrolysis,              probability preservation enzymatic reaction two <br /> protein degradation, DNA recombination, replication, repair,             closely related species different distantly <br /> so [73]. Moreover, hsdS 15 genes encodes                  related species. The M. fermentans M64 metabolic networks <br /> subunit type I restriction enzyme protect hosts           constructed KO number KEGG pathway informa- <br /> against foreign DNA bacteriophages. For             tion served reference networks analysis. The <br /> 15 gene candidates, protein phylogeny reconstructed              conservation enzymes, reactions, metabolic pathway <br /> M. fermentans M64 homolog similar homologs               M. fermentans M64 19 Mycoplasma species and <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        12                              April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                       Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> Phytoplasma OY calculated following equations:                    Supporting Information <br />                                                                                 Figure S1 The circular representation M. <br />    Enzyme conservation score (ECS)~E|D,                                     fermentans M64 genome. The ticks outermost <br />                                       X      X                                  concentric circle indicate relative genomic positions <span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span> <br />    Reaction conservation score (RCS)~   ECS~   E|D <br />                                                                                 pairs, position base upstream <br /> in which, ECS calculates conservation enzyme                  intergenic region dnaA gene. Second concentric <br /> compared species; E presence (1) absence (0)                   circles: predicted protein coding genes plus minus <br />                                                                                 strands, respectively. The various functional groups the <br /> enzyme; D &#226;&#8364;&#732;&#226;&#8364;&#732;Phylogenetic Distance&#226;&#8364;&#8482;&#226;&#8364;&#8482; M. fermentans <br />                                                                                 predicted genes (color coded) categorized according the <br /> and compared species expressed substitution <br />                                                                                 COG classification. Fourth concentric circle: locations rRNA <br /> rate (D) indicated 23S rRNA phylogenetic tree 21 <br />                                                                                 (gray) tRNAs (green) genes. Fifth concentric circle: ICEF-I <br /> Mycoplasma species outgroup species. The phylogenetic <br />                                                                                 IIs (red arrows), ICEF-III (green arrows), WMFV1 prophage <br /> tree constructed PHYML 3.0 [75] evaluated the <br />                                                                                 (blue arrows), IS elements [IS1550 (dark brown), IS1630 (brown), <br /> aLRT method [76]. The phylogenetic distance considered a <br />                                                                                 ISMf1 (orange)], 16S-23S rRNA operons (blue), non-IS <br /> weight enzyme conservation analysis. If enzyme <br />                                                                                 repeats (yellow). Sixth seventh concentric circles: GC content <br /> exists distantly related species, enzyme likely higher <br />                                                                                 GC skew calculated 5,000 bp sliding window with <br /> conservation genus/group organisms, vice versa. Each <br />                                                                                 100 bp step window, respectively. <br /> RCS represents conservation enzyme orthologs of <br />                                                                                 (TIF) <br /> compared species catalyze reaction. <br />                                                                                 Figure S2 Validation large repeats genome <br /> Essential Genes                                                                 assembly M. fermentans M64. A. C. EtBr-stained gel <br />    The essential genes M. pulmonis, M. arthritidis, M. genitalium, B.        images restriction enzymes (BglI, MluI, KpnI, AdhI)-digested <br /> subtilis, E. coli identified previous studies [12,69,70,71,72]           M64 genomic DNA. These fragments resolved CHEF gel <br /> and proteins predicted genomes submitted                electrophoresis 1% agarose 0.5X TBE 6.0 v/cm for <br /> to KAAS obtain KO numbers sequence homology                        approximately 21.5 hr pulsing times 0.92 19.89 s and <br /> searches. The M. fermentans M64 essential genes predicted               0.92 37.34 s, respectively. B. D. Southern blot analyses of <br /> clustering analysis essential putative non-essential               DNA fragments A C, respectively. The BglI MluI, <br /> proteins according KO number BLASTP [77]                     KpnI, AhdI-digested fragments hybridized the <br /> analysis protein consolidated database                 ICEF-I ICEF-II (926 bp), ICEF-III (848 bp), prophage <br /> proteomes analyzed species.                                          WMFV1 (920 bp) specific probes, respectively. The Southern <br />                                                                                 blotting conducted DIG High Prime DNA Labeling and <br />                                                                                 Detection Starter Kit (Roche, Basel, Switzerland). Lane M: <br /> Genes Outside Mycoplasma Clade                                           MidRange I PFG Marker (New England Biolabs, Ipswich, <br />   The M. fermentans M64 genes outside Mycoplasma clade                   Massachusetts); Lane U: Uncut genomic DNA; The size the <br /> were identified examining significant hits BLASTP              marker bands indicated right. <br /> [77] search results proteomes sequenced prokaryotic              (TIF) <br /> organisms retrieved UniProt database (http://www. <br /> uniprot.org/downloads). Genes considered outside                  Figure S3      Comparison ICEF-IA, ICEF-IIA, and <br /> the Mycoplasma clade hit significant matches                    ICEF-IIIA M. fermentans M64 indicated ICEF-III a <br /> belonged Mycoplasma species described previously [15]. Then               new family ICE. A., B., C. Results sequence <br /> the closest organisms genes outside Mycoplasma clade                 comparison ICEF-IIIA ICEF-IA, ICEF-IA ICEF-IIA, <br /> were assigned non-Mycoplasma hit                 ICEF-IIA ICEF-IIIA, respectively. The sequence <br /> 70% sequence coverage alignment M. fermentans                       similarities ICEF-IIIA ICEF-IA, ICEF-IA and <br /> sequence.                                                                       ICEF-IIA, ICEF-IIA ICEF-IIIA are, 45.6%, 91.9%, <br />                                                                                 45.4%, respectively. The GC content (sliding window: <br />                                                                                 120 bp) element plotted each <br /> Genome Sequence Accession Numbers <br />                                                                                 panel. The arrows indicate ORFs four-digit numbers <br />    The accession numbers M. fermentans M64 genome <br />                                                                                 represent locus (i.e., number follow &#226;&#8364;&#732;&#226;&#8364;&#732;MfeM64YM_&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> sequence NC_014921. The species comparative analyses <br />                                                                                 locus tag). The strips dark gray stand forward and <br /> included M. fermentans strain JER (NC_014552), M. haemofelis <br />                                                                                 reverse strands DNA. The direct complementary matched <br /> (NC_014970), M. suis Illinois (NC_015155), M. suis KI3806 <br />                                                                                 regions 2 elements linked blue red lines, <br /> (NC_015153), M. penetrans (NC_004432), M. gallisepticum <br />                                                                                 respectively. D. Phylogenetic tree 11 intact Integrative <br /> (NC_004829), M. genitalium (NC_000908), M. pneumoniae <br />                                                                                 Conjugal Element (ICE) Mycoplasmas. ICEB, ICEA, ICEF, <br /> (NC_000912), M. mycoides (NC_005364), M. capricolum                             ICEH, ICEM, ICEC represent elements M. bovis, M. <br /> (NC_007633),        M.   leachii  (NC_014751),         M.    arthritidis        agalactiae, M. fermentans, M. hyopneumoniae, M. mycoides, M. <br /> (NC_011025), M. hominis (NC_013511), M. mobile (NC_006908),                     capricolum genomes, respectively. The scale bar stands the <br /> M. hyorhinis (NC_014448), M. hyopneumoniae 232 (NC_006360), M.                  estimated number nucleotide substitutions site. <br /> hyopneumoniae J (NC_007295), M. hyopneumoniae 7448 (NC_007332),                 (TIF) <br /> M. conjunctivae (NC_012806), M. pulmonis (NC_002771), M. <br /> agalactiae PG2 (NC_009497), M. agalactiae 5632 (NC_013948), M.                  Figure S4 A global view M. fermentans metabolic <br /> bovis PG45 (NC_014760), M. synoviae (NC_007294), M. crocodyli                   networks showing connected reactions carbohy- <br /> (NC_014014), Phytoplasma OY (NC_005303), B. subtilis subsp.                     drate (green), nucleotide (blue), amino acids (red), <br /> subtilis str. 168 (NC_000964), E. coli str. K-12 substr. MG1655             cofactors vitamins (orange), lipid (pink) <br /> (NC_000913).                                                                    metabolisms. The map produced matching M64 <br />                                                                                 predicted proteins metabolic pathways &#226;&#8364;&#732;&#226;&#8364;&#732;color <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          13                             April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                                         Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> pathway&#226;&#8364;&#8482;&#226;&#8364;&#8482; tool KEGG (http://www.genome.jp/kegg/tool/                                      cofactors Vitamins) circles. The color scale <br /> color_pathway.html).                                                                         (bottom) indicates phylogenetic conservation proteins in <br /> (TIF)                                                                                        21 Mycoplasma species. <br /> Figure S5    M. fermentans M64 purine (A) pyrimi-                                        (TIF) <br /> dine (B) metabolic pathways contain small                                               Figure      S8 Phylogenetic         relationships           Mfe- <br /> clusters connected reactions. Small circles represent                                 M64YM_1027 MfeM64YM_0060 outside the <br /> metabolic intermediates green rectangles enzyme catalog                             Mycoplasma clade. A. C. The 23S rRNA trees the <br /> numbers denote proteins M. fermentans.                                                species shown B D, respectively. B. D. The <br /> (TIF)                                                                                        phylogenetic trees MfeM64YM_1027 MfeM64YM_0060, <br /> Figure S6 Phylogenetic tree 21 Mycoplasma                                             respectively, closest homologs present 12 species. <br /> species analyzed Figure 4. Maximum-likelihood method                                      Maximum-likelihood method used reconstruct trees. <br /> was performed 23S rRNA reconstruct tree using                                 The branch reliability evaluated implementing aLRT <br /> PHYML 3.0. The branch reliability evaluated implement-                                method. The scale bar stands estimated number of <br /> ing aLRT method. Phytoplasma OY set outgroup.                                 nucleotide (A C) amino acid (B D) substitutions per <br /> The scale bar stands estimated number nucleotide                                  site. <br /> substitutions site.                                                                      (TIF) <br /> (TIF)                                                                                        Table S1 Unique common genes M. fermen- <br /> Figure S7 Phylogenetic comparative analysis cofac-                                        tans strains M64, JER, PG18. <br /> tors vitamins (A) amino acid (B) metabolic                                           (XLSX) <br /> networks M. fermentans M64 20 Myco- <br />                                                                                              Table S2 Genes M. fermentans M64 outside the <br /> plasma species. The small open squares represent the <br />                                                                                              Mycoplasma clade. <br /> compounds colored circles represent proteins <br />                                                                                              (XLSX) <br /> (enzymes) participated metabolism. Circles and <br /> light blue circumference represent predicted essential genes. <br /> The arrows lines linking enzymes substrates indicate                             Author Contributions <br /> the direction reactions. Small rectangle B represents                            Conceived designed experiments: HWS WSH SFT WVN. <br /> putative horizontally transferred gene candidate. Enzymes in-                                Performed experiments: HWS TTL YML HYS KJH SFT. Analyzed <br /> volved metabolism indicated                                  data: HWS HIC KMW WVN. Contributed reagents/materials/ <br /> letter <span id='am-320' about='Thesaurus:code' typeof='owl:Thing'>code</span> (A: Amino acids metabolism; C: Carbohydrate                                      analysis tools: WSH SFT WVN. Wrote paper: HWS WVN. Initiated <br /> metabolism; N: Nucleotide metabolism; V: metabolism                                      study: KJH WSH. <br />  <br />  <br /> References <br /> 1.    Yus E, Maier T, Michalodimitrakis K, van Noort V, Yamada T, et al. (2009)              15. Jaffe JD, Stange-Thomann N, Smith C, DeCaprio D, Fisher S, et al. (2004) The <br />       Impact genome reduction bacterial metabolism regulation. Science             complete genome proteome Mycoplasma mobile. Genome Res 14: <br />       326: 1263&#226;&#8364;&#8220;1268.                                                                            1447&#226;&#8364;&#8220;1461. <br /> 2.    Guell M, van Noort V, Yus E, Chen WH, Leigh-Bell J, et al. (2009)                      16. Westberg J, Persson A, Holmberg A, Goesmann A, Lundeberg J, et al. (2004) <br />       Transcriptome complexity genome-reduced bacterium. Science 326:                       The genome sequence Mycoplasma mycoides subsp. mycoides SC type strain <br />       1268&#226;&#8364;&#8220;1271.                                                                                 PG1T, causative agent contagious bovine pleuropneumonia (CBPP). <br /> 3.    Fraser CM, Gocayne JD, White O, Adams MD, Clayton RA, et al. (1995) The                    Genome Res 14: 221&#226;&#8364;&#8220;227. <br />       minimal gene complement Mycoplasma genitalium. Science 270: 397&#226;&#8364;&#8220;403.                17. Vasconcelos AT, Ferreira HB, Bizarro CV, Bonatto SL, Carvalho MO, et al. <br /> 4.    Peterson SN, Hu PC, Bott KF, Hutchison CA 3rd (1993) A survey                       (2005) Swine poultry pathogens: complete genome sequences two <br />       Mycoplasma genitalium genome using random sequencing. J Bacteriol 175:                  strains Mycoplasma hyopneumoniae strain Mycoplasma synoviae. <br />       7918&#226;&#8364;&#8220;7930.                                                                                 J Bacteriol 187: 5568&#226;&#8364;&#8220;5577. <br /> 5.    Papazisi L, Gorton TS, Kutish G, Markham PF, Browning GF, et al. (2003) The            18. Chambaud I, Heilig R, Ferris S, Barbe V, Samson D, et al. (2001) The complete <br />       complete genome sequence avian pathogen Mycoplasma gallisepticum                    genome sequence murine respiratory pathogen Mycoplasma pulmonis. <br />       strain R(low). Microbiology 149: 2307&#226;&#8364;&#8220;2316.                                                Nucleic Acids Res 29: 2145&#226;&#8364;&#8220;2153. <br /> 6.    Sasaki Y, Ishikawa J, Yamashita A, Oshima K, Kenri T, et al. (2002) The                19. Brown DR, Farmerie WG, May M, Benders GA, Durkin AS, et al. (2011) <br />       complete genomic sequence Mycoplasma penetrans, intracellular bacterial              Genome sequences Mycoplasma alligatoris A21JP2T Mycoplasma <br />       pathogen humans. Nucleic Acids Res 30: 5293&#226;&#8364;&#8220;5300.                                       crocodyli MP145T. J Bacteriol 193: 2892&#226;&#8364;&#8220;2893. <br /> 7.    Pereyre S, Sirand-Pugnet P, Beven L, Charron A, Renaudin H, et al. (2009) Life         20. Liu W, Fang L, Li S, Li Q, Zhou Z, et al. (2010) Complete genome sequence of <br />       arginine Mycoplasma hominis: clues minimal genome                      Mycoplasma hyorhinis strain HUB-1. J Bacteriol 192: 5844&#226;&#8364;&#8220;5845. <br />       comparison human urogenital mycoplasmas. PLoS Genet 5:                      21. Rechnitzer H, Brzuszkiewicz E, Strittmatter A, Liesegang H, Lysnyansky I, et al. <br />       e1000677.                                                                                  (2011) Genomic features insights biology Mycoplasma <br /> 8.    Kitzerow A, Hadding U, Henrich B (1999) Cyto-adherence studies                      fermentans. Microbiology 157: 760&#226;&#8364;&#8220;773. <br />       adhesin P50 Mycoplasma hominis. J Med Microbiol 48: 485&#226;&#8364;&#8220;493.                        22. Wise KS, Calcutt MJ, Foecking MF, Roske K, Madupu R, et al. (2011) <br /> 9.    Baseman JB, Cole RM, Krause DC, Leith DK (1982) Molecular basis                        Complete genome sequence Mycoplasma bovis type strain PG45 (ATCC <br />       cytadsorption Mycoplasma pneumoniae. J Bacteriol 151: 1514&#226;&#8364;&#8220;1522.                        25523). Infect Immun 79: 982&#226;&#8364;&#8220;983. <br /> 10.   Himmelreich R, Hilbert H, Plagens H, Pirkl E, Li BC, et al. (1996) Complete            23. Barker EN, Helps CR, Peters IR, Darby AC, Radford AD, et al. (2011) <br />       sequence analysis genome bacterium Mycoplasma pneumoniae. Nucleic            Complete genome sequence Mycoplasma haemofelis, hemotropic <br />       Acids Res 24: 4420&#226;&#8364;&#8220;4449.                                                                   mycoplasma. J Bacteriol 193: 2060&#226;&#8364;&#8220;2061. <br /> 11.   Calderon-Copete SP, Wigger G, Wunderlin C, Schmidheini T, Frey J, et al.               24. Messick JB, Santos AP, Guimaraes AM (2011) Complete genome sequences of <br />       (2009) The Mycoplasma conjunctivae genome sequencing, annotation                       hemotropic mycoplasmas, Mycoplasma haemofelis strain Ohio2 and <br />       analysis. BMC Bioinformatics 10 Suppl 6: S7.                                               Mycoplasma suis Strain Illinois. J Bacteriol 193: 2068&#226;&#8364;&#8220;2069. <br /> 12.   Dybvig K, Zuhua C, Lao P, Jordan DS, French CT, et al. (2008) Genome                25. Watanabe T, Shibata K, Yoshikawa T, Dong L, Hasebe A, et al. (1998) <br />       Mycoplasma arthritidis. Infect Immun 76: 4000&#226;&#8364;&#8220;4008.                                        Detection Mycoplasma salivarium Mycoplasma fermentans synovial <br /> 13.   Sirand-Pugnet P, Lartigue C, Marenda M, Jacob D, Barre A, et al. (2007) Being              fluids temporomandibular joints patients disorders joints. FEMS <br />       pathogenic, plastic, sexual living nearly minimal bacterial               Immunol Med Microbiol 22: 241&#226;&#8364;&#8220;246. <br />       genome. PLoS Genet 3: e75.                                                             26. Haier J, Nasralla M, Franco AR, Nicolson GL (1999) Detection mycoplasmal <br /> 14.   Minion FC, Lefkowitz EJ, Madsen ML, Cleary BJ, Swartzell SM, et al. (2004)                 infections blood patients rheumatoid arthritis. Rheumatology (Oxford) <br />       The genome sequence Mycoplasma hyopneumoniae strain 232, agent swine             38: 504&#226;&#8364;&#8220;509. <br />       mycoplasmosis. J Bacteriol 186: 7123&#226;&#8364;&#8220;7133. <br />  <br />  <br />  <br />          PLoS ONE | www.plosone.org                                                     14                                   April 2012 | Volume 7 | Issue 4 | e32940 <br />                                                                                                                            Comparative Genomic Analyses Mycoplasmas <br />  <br />  <br /> 27. Gilroy CB, Keat A, Taylor-Robinson D (2001) The prevalence Mycoplasma                  52. Shu HW, Liu TT, Chan HI, Liu YM, Wu KM, et al. (2011) Genome sequence <br />     fermentans patients inflammatory arthritides. Rheumatology (Oxford)                   repetitive-sequence-rich Mycoplasma fermentans strain M64. J Bacteriol <br />     40: 1355&#226;&#8364;&#8220;1358.                                                                                193: 4302&#226;&#8364;&#8220;4303. <br /> 28. Kraft M, Cassell GH, Henson JE, Watson H, Williamson J, et al. (1998)                     53. Ishida N, Irikura D, Matsuda K, Sato S, Sone T, et al. (2009) Molecular cloning <br />     Detection Mycoplasma pneumoniae airways adults chronic asthma.              expression novel cholinephosphotransferase involved glycoglycer- <br />     Am J Respir Crit Care Med 158: 998&#226;&#8364;&#8220;1001.                                                      ophospholipid biosynthesis Mycoplasma fermentans. Curr Microbiol 58: <br /> 29. Vojdani A, Choppa PC, Tagle C, Andrin R, Samimi B, et al. (1998) Detection                 535&#226;&#8364;&#8220;540. <br />     Mycoplasma genus Mycoplasma fermentans PCR patients Chronic                54. Chu LJ, Yang H, Shih P, Kao Y, Tsai YS, et al. (2011) Metabolic capabilities <br />     Fatigue Syndrome. FEMS Immunol Med Microbiol 22: 355&#226;&#8364;&#8220;365.                                     systems fluctuations Haloarcula marismortui revealed integrative <br /> 30. Cultrera R, Roulland-Dussoix D, Romani R, Contini C (1998) Use PCR                      genomics proteomics analyses. J Proteome Res. <br />     detect mycoplasma DNA respiratory tract specimens adult HIV-positive              55. Neimark H, Johansson KE, Rikihisa Y, Tully JG (2001) Proposal transfer <br />     patients. J Med Microbiol 47: 983&#226;&#8364;&#8220;986.                                                        members genera Haemobartonella Eperythrozoon genus <br /> 31. Sasaki Y, Honda M, Naitou M, Sasaki T (1994) Detection Mycoplasma                          Mycoplasma descriptions &#226;&#8364;&#732;Candidatus Mycoplasma haemofelis&#226;&#8364;&#8482;, <br />     fermentans DNA lymph nodes acquired immunodeficiency syndrome                         &#226;&#8364;&#732;Candidatus Mycoplasma haemomuris&#226;&#8364;&#8482;, &#226;&#8364;&#732;Candidatus Mycoplasma haemosuis&#226;&#8364;&#8482; <br />     patients. Microb Pathog 17: 131&#226;&#8364;&#8220;135. <br />                                                                                                   &#226;&#8364;&#732;Candidatus Mycoplasma wenyonii&#226;&#8364;&#8482;. Int J Syst Evol Microbiol 51: 891&#226;&#8364;&#8220;899. <br /> 32. Dawson MS, Hayes MM, Wang RY, Armstrong D, Kundsin RB, et al. (1993) <br />                                                                                               56. Rikihisa Y, Kawahara M, Wen B, Kociba G, Fuerst P, et al. (1997) Western <br />     Detection isolation Mycoplasma fermentans urine human <br />                                                                                                   immunoblot analysis Haemobartonella muris comparison 16S rRNA <br />     immunodeficiency virus type 1-infected patients. Arch Pathol Lab Med 117: <br />                                                                                                   gene sequences H. muris, H. felis, Eperythrozoon suis. J Clin Microbiol <br />     511&#226;&#8364;&#8220;514. <br /> 33. Hawkins RE, Rickman LS, Vermund SH, Carl M (1992) Association                              35: 823&#226;&#8364;&#8220;829. <br />     mycoplasma human immunodeficiency virus infection: detection                       57. Weisburg WG, Tully JG, Rose DL, Petzel JP, Oyaizu H, et al. (1989) A <br />     amplified Mycoplasma fermentans DNA blood. J Infect Dis 165: 581&#226;&#8364;&#8220;585.                      phylogenetic analysis mycoplasmas: basis classification. J Bacteriol <br /> 34. Ainsworth JG, Clarke J, Lipman M, Mitchell D, Taylor-Robinson D (2000)                        171: 6455&#226;&#8364;&#8220;6467. <br />     Detection Mycoplasma fermentans broncho-alveolar lavage fluid specimens             58. Pettersson B, Tully JG, Bolske G, Johansson KE (2000) Updated phylogenetic <br />     AIDS patients lower respiratory tract infection. HIV Med 1: 219&#226;&#8364;&#8220;223.                description Mycoplasma hominis cluster (Weisburg, et al. 1989) based on <br /> 35. Ainsworth JG, Hourshid S, Webster AD, Gilroy CB, Taylor-Robinson D (2000)                     16S rDNA sequences. Int J Syst Evol Microbiol 50 Pt 1: 291&#226;&#8364;&#8220;301. <br />     Detection Mycoplasma fermentans healthy students patients                  59. Van Demark PJ (1967) Respiratory pathways mycoplasma. Ann N Y Acad <br />     congenital immunodeficiency. J Infect 40: 138&#226;&#8364;&#8220;140.                                            Sci 143: 77&#226;&#8364;&#8220;84. <br /> 36. Chingbingyong MI, Hughes CV (1996) Detection Mycoplasma fermentans                  60. Smith SL, Vandemark PJ, Fabricant J (1963) Respiratory Pathways the <br />     human saliva polymerase chain reaction-based assay. Arch Oral Biol 41:                 Mycoplasma. I. Lactate Oxidation Mycoplasma Gallisepticum. J Bacteriol <br />     311&#226;&#8364;&#8220;314.                                                                                      86: 893&#226;&#8364;&#8220;897. <br /> 37. Shibata K, Kaga M, Kudo M, Dong L, Hasebe A, et al. (1999) Detection                   61. Razin S, Yogev D, Naot Y (1998) Molecular biology pathogenicity of <br />     Mycoplasma fermentans saliva sampled infants, preschool school                    mycoplasmas. Microbiol Mol Biol Rev 62: 1094&#226;&#8364;&#8220;1156. <br />     children, adolescents adults polymerase chain reaction-based assay.              62. Pollack JD (2002) The necessity combining genomic enzymatic data to <br />     Microbiol Immunol 43: 521&#226;&#8364;&#8220;525.                                                                infer metabolic function pathways smallest bacteria: amino acid, <br /> 38. Montagnier L, Blanchard A (1993) Mycoplasmas cofactors infection                 purine pyrimidine metabolism Mollicutes. Front Biosci 7: d1762&#226;&#8364;&#8220;1781. <br />     human immunodeficiency virus. Clin Infect Dis 17 Suppl 1: S309&#226;&#8364;&#8220;315.                   63. Baker TA, Wickner SH (1992) Genetics enzymology DNA replication in <br /> 39. Iyama K, Ono S, Kuwano K, Ohishi M, Shigematsu H, et al. (1996) Induction                     Escherichia coli. Annu Rev Genet 26: 447&#226;&#8364;&#8220;477. <br />     tumor necrosis factor alpha (TNF alpha) enhancement HIV-1                       64. Lampe M, Binnie C, Schmidt R, Losick R (1988) Cloned gene encoding the <br />     replication J22HL60 cell line Mycoplasma penetrans. Microbiol                       delta subunit Bacillus subtilis RNA polymerase. Gene 67: 13&#226;&#8364;&#8220;19. <br />     Immunol 40: 907&#226;&#8364;&#8220;914.                                                                      65. Jones AL, Needham RH, Rubens CE (2003) The Delta subunit RNA <br /> 40. Beecham HJ 3rd, Lo SC, Lewis DE, Comer SW, Riley KJ, et al. (1991)                            polymerase required virulence Streptococcus agalactiae. Infect Immun <br />     Recovery fulminant infection Mycoplasma fermentans (incognitus                      71: 4011&#226;&#8364;&#8220;4017. <br />     strain) non-immunocompromised host. Lancet 338: 1014&#226;&#8364;&#8220;1015.                             66. Lopez Saro FJ, Yoshikawa N, Helmann JD (1999) Expression, abundance, <br /> 41. Dallo SF, Baseman JB (2000) Intracellular DNA replication long-term <br />                                                                                                   RNA polymerase binding properties delta factor Bacillus subtilis. <br />     survival pathogenic mycoplasmas. Microb Pathog 29: 301&#226;&#8364;&#8220;309. <br />                                                                                                   J Biol Chem 274: 15953&#226;&#8364;&#8220;15958. <br /> 42. Taylor-Robinson D, Sarathchandra P, Furr PM (1993) Mycoplasma fermen- <br />                                                                                               67. Watson SP, Antonio M, Foster SJ (1998) Isolation characterization of <br />     tans--HeLa cell interactions. Clin Infect Dis 17 Suppl 1: S302&#226;&#8364;&#8220;304. <br />                                                                                                   Staphylococcus aureus starvation-induced, stationary-phase mutants defective in <br /> 43. Blanchard A, Hamrick W, Duffy L, Baldus K, Cassell GH (1993) Use the <br />     polymerase chain reaction detection Mycoplasma fermentans Mycoplasma               survival recovery. Microbiology 144 (Pt 11): 3159&#226;&#8364;&#8220;3169. <br />     genitalium urogenital tract amniotic fluid. Clin Infect Dis 17 Suppl 1:        68. Kawauchi Y, Muto A, Osawa S (1982) The protein composition Mycoplasma <br />     S272&#226;&#8364;&#8220;279.                                                                                     capricolum. Mol Gen Genet 188: 7&#226;&#8364;&#8220;11. <br /> 44. Johnson S, Sidebottom D, Bruckner F, Collins D (2000) Identification                   69. French CT, Lao P, Loraine AE, Matthews BT, Yu H, et al. (2008) Large-scale <br />     Mycoplasma fermentans synovial fluid samples arthritis patients                  transposon mutagenesis Mycoplasma pulmonis. Mol Microbiol 69: 67&#226;&#8364;&#8220;76. <br />     inflammatory disease. J Clin Microbiol 38: 90&#226;&#8364;&#8220;93.                                         70. Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, et al. (2006) <br /> 45. Choppa PC, Vojdani A, Tagle C, Andrin R, Magtoto L (1998) Multiplex PCR                       Essential genes minimal bacterium. Proc Natl Acad Sci U S A 103: <br />     detection Mycoplasma fermentans, M. hominis M. penetrans                    425&#226;&#8364;&#8220;430. <br />     cell cultures blood samples patients chronic fatigue syndrome. Mol            71. Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, et al. (2003) <br />     Cell Probes 12: 301&#226;&#8364;&#8220;308.                                                                      Essential Bacillus subtilis genes. Proc Natl Acad Sci U S A 100: 4678&#226;&#8364;&#8220;4683. <br /> 46. Hu WS, Yang CC (2001) IS1550 Mycoplasma fermentans transposable                   72. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction <br />     Escherichia coli. FEMS Microbiol Lett 198: 159&#226;&#8364;&#8220;164.                                        Escherichia coli K-12 in-frame, single-gene <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio <br /> 47. Calcutt MJ, Lavrrar JL, Wise KS (1999) IS1630 Mycoplasma fermentans, novel               collection. Mol Syst Biol 2: 2006 0008. <br />     IS30-type insertion element targets duplicates inverted repeats               73. Snider J, Thibault G, Houry WA (2008) The AAA+superfamily functionally <br />     variable length sequence insertion. J Bacteriol 1 <br /> </body></html>