<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Previous Exposure RNA Virus Does Not Protect <br /> against Subsequent Infection Drosophila <br /> melanogaster <br /> Ben Longdon*, Chuan Cao, Julien Martinez, Francis M. Jiggins <br /> Department Genetics, University Cambridge, Cambridge, United Kingdom <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Background: Immune priming shown occur wide array invertebrate taxa, individuals exposed a <br />     pathogen showing increased protection subsequent exposure. However, mechanisms underlying immune <br />     priming poorly understood. The antiviral RNAi response Drosophila melanogaster ideal candidate providing a <br />     specific acquired response subsequent infection. We exposed D. melanogaster challenges virus known <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />     produce antiviral RNAi response, examine protective effects prior exposure <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> observed. <br />  <br />     Results: In experiment evidence prior exposure Drosophila C Virus (DCV) protects flies a <br />     subsequent lethal challenge, identical levels mortality flies previously exposed DCV control. <br />  <br />     Conclusions: Our results confirm finding &#226;&#8364;&#732;acquired&#226;&#8364;&#8482; immune responses ubiquitous invertebrate- <br />     pathogen interactions. We discuss observed effect study, focus mechanistic basis of <br />     RNAi pathway. <br />  <br />   Citation: Longdon B, Cao C, Martinez J, Jiggins FM (2013) Previous Exposure RNA Virus Does Not Protect Subsequent Infection Drosophila <br />   melanogaster. PLoS ONE 8(9): e73833. doi:10.1371/journal.pone.0073833 <br />   Editor: Nicole M. Gerardo, Emory University, United States America <br />   Received May 24, 2013; Accepted July 31, 2013; Published September 11, 2013 <br />   Copyright: &#195;&#376; 2013 Longdon et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported European Research Council grant (281668, DrosophilaInfection: FMJ BL), Junior Research Fellowship Christ&#226;&#8364;&#8482;s <br />   College, Cambridge (BL), Royal Society University Research Fellowship (FMJ), Wellcome Trust Grant (WT094664MA: JM) Cambridge Overseas Trust/ <br />   Chinese Scholarship Council studentship (CC). The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: b.longdon@gen.cam.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                         Similarly, Drosophila primed Streptococcus pneumonia bacteria <br />                                                                                      protected secondary lethal challenge homologous <br />    Over past decade, number studies demonstrated                       taxonomically diverse range bacteria [12]. In <br /> that invertebrates previously encountered pathogen/                      case Toll pathway phagocytes <br /> parasite appear protected secondary exposure,                       underlie increased survival greater bacterial clearance <br /> has termed &#226;&#8364;&#732;&#226;&#8364;&#732;immune priming&#226;&#8364;&#8482;&#226;&#8364;&#8482; [1&#226;&#8364;&#8220;5]. Experimental studies                       observed. Specific immune priming reported against <br /> of immune priming demonstrated following exposure                       different strains bacterial pathogens beetle Tribolium <br /> either dead/non-infectious pathogen, sub-lethal dose                  castaneum [13] crustacean Daphnia magna [7], the <br /> subsequently cleared, host protected later lethal                     copepod Macrocyclops albidus infected tapeworms [2]. Immune <br /> challenge [2,3,6]. The effect immune priming                        priming shown occur double-stranded <br /> even cross generations, offspring infected parents                 DNA (dsDNA) viruses. Penaeus shrimp previous exposure to <br /> protected [7&#226;&#8364;&#8220;10]. While vertebrates produce acquired immune                          white spot syndrome virus increased survival subsequent <br /> responses parasites antibody-mediated immunity,                           exposure virus, protection persisting months <br /> occurrence immune priming invertebrates suggests &#226;&#8364;&#732;acquired&#226;&#8364;&#8482;                    [14,15]. Similarly, individuals moth Plodia interpunctella <br /> responses achievable alternative mechanisms.                             exposed low dose natural DNA virus increased <br />    Despite insect innate immune recognising killing                   survival subsequent challenge lethal dose, their <br /> invaders using receptors effectors target molecules                         offspring [10]. Whilst little known mechanisms of <br /> conserved broad taxonomic range pathogens, immune                        specific immune priming invertebrates, possibility the <br /> priming highly specific [11]. For example,                          hypervariable immunoglobulin domain-encoding gene Dscam. By <br /> shortly exposure bacterial pathogens, bumblebees                      alternatively spliced, Dscam able produce sufficient <br /> initially general priming response little specificity [5].               receptor diversity distinguish different pathogen strains <br /> However, weeks initial infection, insects                 species [16,17]. <br /> show increased protection secondary exposure bacteria                         A promising candidate acquired specific immunity <br /> they previously exposed to, priming response                      viruses RNAi [18,19]. The RNAi pathway processes <br /> able distinguish bacteria genus [5].                        viral double-stranded RNA (dsRNA) short interfering RNAs <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                           September 2013 | Volume 8 | Issue 9 | e73833 <br />                                                                                                       Effect Previous Viral Exposure Drosophila <br />  <br />  <br /> (siRNAs), called viRNAs viral derived [20]. These              infectivity virus, serial dilutions virus 1021 10212 <br /> used guide cleavage viral RNA complementary                   carried Schneider&#226;&#8364;&#8482;s medium, dilution was <br /> sequence viRNA, resulting degradation viral RNA            added 8 wells plate DL2 cells. After 7 days wells were <br /> sequence-specific manner [19]. This pathway shown                examined classed &#226;&#8364;&#732;&#226;&#8364;&#732;infected&#226;&#8364;&#8482;&#226;&#8364;&#8482; cell death and <br /> protect Drosophila number positive negative sense              cytopathic effects clearly visible. The Tissue Culture Infective <br /> RNA viruses, DNA virus [21]. Therefore, RNAi              Dose 50 (TCID50) calculated Reed-Muench end-point <br /> potentially mechanism highly specific immune priming;            method [40]. <br /> individuals exposed sub-lethal dose virus triggers             To infect flies DCV, 0.0175 mm diameter anodized steel <br /> the RNAi pathway, viRNAs persist, confer                      needle (26002&#226;&#8364;&#8220;15, Fine Science Tools, CA, USA) bent <br /> protection subsequent exposure lethal dose               ,0.25 mm end (,half dorsal width thorax), <br /> virus.                                                                       dipped DCV control solution (uninfected cell culture <br />    Rather host clearing sub-lethal dose virus,        medium), bent needle pricked pleural <br /> acute viral infections develop persistent infections,         suture thorax flies. A pilot study used calculate a <br /> the virus present low level cause little damage [22].        suitable sub-lethal dose DCV (Figure S1). The primary <br /> RNAi important controlling persistent infections low           challenge dose (TCID50 2.326106) used maximum non- <br /> levels [23]. In Drosophila cells, persistent infections RNA viruses       lethal dose, try ensure replication virus. The second <br /> result viral RNA reverse transcribed chimeric DNA              challenge (TCID50 4.646107) dose killed ,85% flies <br /> molecules containing retro-transposon viral sequences [23].              pilot study (Figure S1). The secondary challenge dose was <br /> These DNA forms turn produce transcripts appear                chosen ensure flies simply overwhelmed a <br /> processed siRNA pathway, produce viRNAs                      large viral dose. At doses greater resistance <br /> inhibit viral replication. Furthermore, inhibiting reverse transcrip-        mechanisms &#226;&#8364;&#8220; e.g. Wolbachia &#226;&#8364;&#8220; able confer protection <br /> tion adult flies results greater mortality viral infection.         (Julien Martinez, unpublished data). Virus extract uninfected <br /> Along classical antiviral RNAi, provide                    cell culture medium diluted desired concentration using <br /> mechanism low level persistent infections act              Drosophila Ringer&#226;&#8364;&#8482;s solution [39]. <br /> protect flies subsequent acute viral infection manner akin <br /> to immune priming (we define immune priming                    Experimental Design <br /> protection following clearance initial infection).                       For main experiment, 3&#226;&#8364;&#8220;4 day old female flies first <br />    To examine previous exposure insect RNA virus               challenged low virus dose (TCID50 2.326106) control <br /> can protect subsequent infection pathogen,            medium challenged high virus dose <br /> carried experiment using Drosophila melanogaster              (TCID50 4.646107) control medium, 3 days first <br /> naturally occurring pathogen, Drosophila C Virus (DCV). DCV               challenge. Flies fall treatments (first-second <br /> a positive sense RNA virus family Dicistroviridae [24]           challenge): control-control, control-virus, virus-virus, virus-control. <br /> naturally infects D. melanogaster Drosophila species        The 3 day gap second challenge was <br /> wild [25&#226;&#8364;&#8220;27]. DCV reported predominantly                      designed allow viral challenge replicate produce <br /> horizontally transmitted individuals laboratory,              immune response. Our aim produce immune <br /> although vertical transmission occur [25,28]. The RNAi              response loitered conferred protection subsequent <br /> pathway shown important antiviral defence                  exposure. DCV viral load shown increase day <br /> against DCV, flies lacking major RNAi components                 post-infection, confident virus be <br /> showing increased susceptibility [29&#226;&#8364;&#8220;31], viRNAs                     replicating producing dsRNA [30]. Previous studies have <br /> produced DCV [32]. In addition RNAi, JAK-                     RNAi response appears active controlling viral <br /> STAT pathway gene called Pastrel implicated               load, significant differences viral load wild-type <br /> protecting flies DCV [21,33,34].                                     RNAi deficient flies, day post-infection <br />    Our approach expose flies DCV challenges.                [29,30]. <br /> The challenge low dose replicate &#226;&#8364;&#8220;                The experiment carried days the <br /> produce immune response &#226;&#8364;&#8220; cause minimal mortality. The                treatments performed day treatment order <br /> second lethal dose DCV challenge           randomised day. In total challenged measured <br /> would protect flies. This different approach past           survival 2310 flies. Challenged flies kept vials of <br /> priming literature, primary infection dead rapidly           cornmeal medium 256C 70% relative humidity. After the <br /> cleared [1], designed resemble persistent DCV                  second challenge flies tipped fresh cornmeal medium <br /> infections occur wild [35,36].                               days. On average 19 flies vial, with <br />                                                                              approximately twice number vials treatment the <br /> Methods                                                                      second challenge virus (number flies treatment: control- <br />                                                                              control = 377, control-virus = 784, virus-virus = 758, virus-con- <br /> Virus Production Infections                                              trol = 391). Flies pricked opposite thorax <br />   DCV produced Schneider Drosophila line 2 (DL2) cells                second challenge. As systemic RNAi <br /> [37] described [38]. Cells cultured 26.5uC                  response Drosophila, inoculating flies different location not <br /> Schneider&#226;&#8364;&#8482;s Drosophila Medium 10% Fetal Bovine Serum,                   expected alter RNAi-mediated protection [19,41]. Mortality <br /> 100 U/ml penicillin 100 mg/ml streptomycin (all Invitrogen,              recorded daily 14 days second challenge. <br /> UK). Cells freeze-thawed twice lyse cells                      The flies used experiment the <br /> centrifuged 4000 g 10 minutes 46C remove cellular           DrosDel w1118 isogenic line kindly provided Luis Teixeira <br /> components bacteria. Virus aliquoted frozen               [37,42], confirmed free DCV Wolbachia <br /> 2806C. Uninfected cell culture control solution produced             experiment RT-PCR [38]. Flies reared in <br /> by growing DL2 cells virus production DCV                     bottles fixed densities, flies collected bottle <br /> replaced Drosophila Ringer&#226;&#8364;&#8482;s solution [39]. To calculate                randomly assigned treatment groups. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                        September 2013 | Volume 8 | Issue 9 | e73833 <br />                                                                                                          Effect Previous Viral Exposure Drosophila <br />  <br />  <br />  <br /> Measuring Viral RNA Load <br />    To confirm flies infected DCV <br /> through laboratory contamination experiment, to <br /> see challenge cleared persisted, viral <br /> titre measured quantitative reverse transcription (qRT-) <br /> PCR flies control-control virus-control treatment <br /> respectively day 14 second challenge. Additionally, to <br /> confirm challenge dose replicated, 140 flies were <br /> stabbed challenge dose (TCID50 2.326106), were <br /> then snap frozen immediately days later (7 <br /> replicates 10 flies time point). <br />    Flies snap frozen liquid nitrogen immediately <br /> homogenised Trizol reagent (Invitrogen, UK). RNA then <br /> extracted using chloroform-isopropanol extraction, described <br /> in [43]. The copy-number viral RNA measured relative to <br /> an endogenous control reference gene (Elongation factor 1a 100E <br /> (EF1a100E)) using QuantiTect Virus qRT-PCR kit (Qiagen, <br /> UK) Taqman hydrolysis probes (Sigma-Aldrich, UK) a <br /> BioRad iQ5 thermocycler. Reactions carried duplex <br /> with probes using Cy5 (DCV) FAM (Ef1a100E) reporter dyes, <br /> following manufactures instructions. Relative viral RNA copy <br /> number calculated relative endogenous control                 Figure 1. Survival following challenge. Survival flies inoculated <br /> using DCt (critical threshold) method, relative viral           low initial dose DCV control solution (uninfected cell culture <br /> RNA load given 22(Virus Ct&#226;&#8364;&#8220;Control Ct) assuming 100%                    medium) followed secondary infection high dose DCV or <br />                                                                               control solution. <br /> amplification efficiency. A dilution series used calculate <br />                                                                               doi:10.1371/journal.pone.0073833.g001 <br /> primer efficiencies, 104% 102% Ef1a100E and <br /> DCV respectively. The qRT-PCR cycle was: 506C 20 min, <br />                                                                               dose virus itself caused mortality, 16% virus- <br /> 956C 5 min 40 cycles of: 956C 15 sec, 606C for <br />                                                                               control flies dead day 14 compared 4% control-control <br /> 45 sec. Two technical replicates carried the <br />                                                                               flies (Figure 1; z = 3.69 P,0.001). <br /> day 14 control-control virus-control samples, single <br />                                                                                   We examined hazard ratios [45] standard errors. <br /> technical replicate carried samples looking the <br />                                                                               The hazard ratios control-virus virus-virus treatments <br /> replication challenge dose. For primer probe <br />                                                                               identical small standard errors (hazard ratio of <br /> sequences table S1. <br />                                                                               virus-virus flies relative control-virus flies = 20.04, SE = 0.11), <br />                                                                               indicating mortality estimates good precision. <br /> Analysis <br />    The effect treatment survival rates analysed using a <br />                                                                               Viral RNA Load <br /> Cox&#226;&#8364;&#8482;s proportional hazards mixed effect model, accounted <br />                                                                                   Our qRT-PCR data confirmed 20 samples day 14 <br /> for between-vial variation survival rates. The hazard ith <br />                                                                               flies (363 flies total) control-control treatment contained <br /> individual vial j time t modelled as: <br />                                                                               detectable viral RNA. This suggests flies accidentally <br />                                                                               contaminated DCV experiment. <br />                         Hij (t)~H0 (t)eXi bzbj                                    The challenge (low dose) virus replicated all <br />                                                                               samples. Only seven samples (each sample consisted 10 <br />                                                                               flies) contained detectable amounts viral RNA immediately after <br />    Where H0(t) baseline hazard time t, Xi vector        infection, compared seven samples days post- <br /> fixed effects, b corresponding vector coefficients, bj       infection. There significantly greater quantities viral RNA <br /> a random effect vial j nested day experiment. The            day samples (relative viral load qRT-PCR: day <br /> fixed effects comprised treatment day infection. Flies alive           0 = 0.45 (SE = 0.22), day3 = 2509.16 (SE = 1529.73), Wil- <br /> at end experiment censored. The model fitted              coxon rank sum test: W = 44, P = 0.01). By day 14 main <br /> by maximum likelihood using coxme package [44] R (R                    experiment, 50% virus-control treated vials contained detect- <br /> Foundation Statistical Computing, Vienna, Austria). Data              able amounts virus (10/20, total 327 flies; mean viral load for <br /> separated control virus secondary treatments              infected samples = 110.2, SE = 103.1), suggesting infection <br /> assumption proportional hazards met. The inclusion vial             persistent/maintained low levels flies. In <br /> as random effect led improvement AIC score. The                vials DCV RNA detectable day 14, was <br /> DCt values qRT-PCR data viral load normally               significantly greater mortality compared vials DCV <br /> distributed analysed using Wilcoxon Rank Sum test.                  RNA detected (24% vs 9% mortality day 14, Survival <br />                                                                               analysis: z = 2.64, p = 0.0084, hazard ratio flies vials with <br /> Results                                                                       virus relative virus day 14 = 1.17, SE = 0.45). <br />  <br /> Survival                                                                      Discussion <br />    Exposing flies low dose virus effect rate at <br /> which die subsequently infected larger lethal                  We instance flies previously exposed DCV <br /> dose (Figure 1A; z = 20.34, P = 0.73). By day 14, control-               protected mortality subsequent exposure. <br /> virus virus-virus treatments 88% mortality. The initial low           Given DCV targeted RNAi pathway [29&#226;&#8364;&#8220;31], had <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                3                         September 2013 | Volume 8 | Issue 9 | e73833 <br />                                                                                                          Effect Previous Viral Exposure Drosophila <br />  <br />  <br /> hypothesised sub-lethal challenge lead                      experiment. Exposing flies virus orally &#226;&#8364;&#8220; natural <br /> production viRNAs protect flies subsequent                route infection &#226;&#8364;&#8220; alter result, exposing <br /> lethal infection. It previously shown injection                gut, known preferential site DCV <br /> dsRNA corresponding DCV restrict replication                    replication [36,51]. Similarly, genetic variation ability of <br /> subsequent DCV infection antiviral RNAi [46]. Given                  flies respond secondary infection [34], genotype by <br /> primary challenge used appears replicated,                   environment interactions [52&#226;&#8364;&#8220;54] coinfection symbionts <br /> should lead immune response production dsRNA,                  [37,55,56] pathogens [57] affect outcome such <br /> why protection DCV experiment?                    interactions. <br />     There possible explanations primary                       Immune priming universal phenomenon inverte- <br /> exposure did protect mortality secondary                     brates, previous studies finding variation degree of <br /> challenge experiment. Firstly, DCV known encode                 protection, pathogens eliciting response [12,13]. <br /> viral suppressor RNAi (VSR) binds dsRNA inhibits                Other studies evidence priming [58&#226;&#8364;&#8220;60]. <br /> production siRNAs [30]. Therefore, challenge              Priming variable host-parasite system, <br /> replicated, VSR prevented prophylactic effect.                 suggesting context dependent [10,61]. <br /> Giving flies primary challenge using DCV strain                      Priming costly [62,63], prevent the <br /> mutated VSR abolished activity allow hypothesis                 evolution priming responses costs outweigh benefits. <br /> to tested. A recent study positive sense genomic              In effects evolved to <br /> strand DCV primary target antiviral RNAi, possibly              ecology interaction D. melanogaster DCV the <br /> a result VSR preventing targeting dsRNA replication                      wild. DCV prevalence natural populations estimated <br /> intermediates [47]. It possible approach taken             low (less 1%) [26]. Therefore, speculate that <br /> in previous studies giving large dose dead pathogen,               encounter rates low &#226;&#8364;&#732;acquired&#226;&#8364;&#8482; response have <br /> dose rapidly cleared, resulted protection                evolved [64]. <br /> subsequent exposure. <br />     It case immune response                     Conclusions <br /> primary challenge does persist long provide                      Immune priming demonstrated number of <br /> prophylaxis. Previous studies systems shown priming               invertebrates. We hypothesised RNAi pathway could <br /> can long-lived [5,13,15]. Injection dsRNA complementary                provide acquired sequence-specific antiviral immune <br /> DCV Sindbis virus shown protect flies                        response. Here, examined exposure insect <br /> subsequent infection to days,                  RNA virus results protection subsequent exposure. We <br /> shown dose dependant Sindbis virus, lower doses               evidence previous exposure DCV protected flies <br /> dsRNA producing shorter period protection [46]. If                     subsequent lethal challenge experiment. <br /> primary challenge dose did produce suitably large quantities of <br /> dsRNA, response short lived. The                   Supporting Information <br /> possibility primary dose did produce sufficient quantities <br /> of dsRNA result prophylactic effect secondary                    Figure S1 Dilution pilot survival data. Flies stabbed <br /> exposure days later.                                                      various doses DCV (TCID50 4.646109 4.646104). <br />     Furthermore, flies experiment                       Control flies stabbed uninfected cell culture medium. <br /> produced systemic antiviral response primary challenge,                Each treatment consists 3 vials 20 flies exception of <br /> as D. melanogaster lacks mechanisms organisms                109 treatment 2 vials. Following this <br /> that important generating systemic RNAi response                   study, pilot study carried using TCID50 of <br /> viral infection (reviewed in: [19,41]). Firstly, D. melanogaster lacks       2.326106 9.286105 (i.e. 1:2 1:5 dilution previous <br /> RNA-dependent RNA polymerase amplify dsRNA [48],                       non-lethal TCID50 4.646105) determine maximum sub- <br /> can enhance systemic RNAi response taxa [19,41]. D.                lethal dose. With doses observable differences <br /> melanogaster lacks transmembrane transporter channel                  control vs virus treatment day 10 post-infection. <br /> mediate cell-cell spread RNAi response, SID-1               (TIFF) <br /> is Caenorhabditis elegans [49]. Instead, systemic responses         Table S1 Primer Probe sequences used qRT-PCR. <br /> viral infection Drosophila appear rely active cellular uptake          (DOCX) <br /> of dsRNA greater 31 bp long receptor mediated <br />                                                                                 Dataset S1 Experimental mortality data, columns are: day of <br /> endocytosis [50], flies deficient uptake dsRNA <br /> showing increased susceptibility viral infection [46]. However,              death, day experiment, primary secondary <br /> the proposed mechanism thought rely infected              challenge control (C) virus (V), fly censored <br /> cells releasing viral dsRNAs (through lysis shedding)               (0 = yes, 1 = no), vial identity. <br /> taken uninfected cells (with spread dsRNAs                 (CSV) <br /> haemolymph active cell-cell transport) [46]. There- <br /> fore, low dose DCV result sufficient cell lysis              Acknowledgments <br /> shedding release suitable quantities viral dsRNA trigger             Thanks Matt Tinsley, Darren Obbard, Pedro Vale Mike Boots for <br /> systemic response. As DCV known display tissue                       useful discussions, Rodrigo Cogni Jon Day help lab. <br /> tropism [51], primary exposure experiment                 Thanks anonymous reviewers useful comments. <br /> had stimulate immune response tissues successful <br /> protection subsequent infection.                                             Author Contributions <br />     Whilst did detect effect prior exposure               Conceived designed experiments: BL FMJ. Performed the <br /> experiment, case different conditions                experiments: BL CC JM. Analyzed data: BL. Contributed reagents/ <br /> protective effect secondary infection does occur; example                materials/analysis tools: BL CC JM FMJ. Wrote paper: BL CC JM <br /> altering dose timing infection alter outcome              FMJ. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  4                       September 2013 | Volume 8 | Issue 9 | e73833 <br />                                                                                                                            Effect Previous Viral Exposure Drosophila <br />  <br />  <br />  <br />  <br /> References <br />  1. Little TJ, Kraaijeveld AR (2004) Ecological evolutionary implications            31. Wang XH, Aliyari R, Li WX, Li HW, Kim K, et al. (2006) <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br />     immunological priming invertebrates. Trends Ecol Evol 19: 58&#226;&#8364;&#8220;60.                         directs innate immunity viruses adult Drosophila. Science 312: 452&#226;&#8364;&#8220; <br />  2. Kurtz J, Franz K (2003) Innate defence: evidence memory invertebrate                 454. <br />     immunity. Nature 425: 37&#226;&#8364;&#8220;38.                                                            32. Wu Q, Luo Y, Lu R, Lau N, Lai EC, et al. (2010) Virus discovery deep <br />  3. Moret Y, Siva-Jothy MT (2003) Adaptive innate immunity? Responsive-mode                     sequencing assembly virus-derived small silencing RNAs. Proc Natl Acad <br />     prophylaxis mealworm beetle, Tenebrio molitor. Proc Biol Sci 270: 2475&#226;&#8364;&#8220;              Sci U S A 107: 1606&#226;&#8364;&#8220;1611. <br />     2480.                                                                                   33. Dostert C, Jouanguy E, Irving P, Troxler L, Galiana-Arnoux D, et al. (2005) <br />  4. Roth O, Sadd BM, Schmid-Hempel P, Kurtz J (2009) Strain-specific priming                 The Jak-STAT signaling pathway required sufficient antiviral <br />     resistance red flour beetle, Tribolium castaneum. Proceedings Royal           response drosophila. Nat Immunol 6: 946&#226;&#8364;&#8220;953. <br />     Society B-Biological Sciences 276: 145&#226;&#8364;&#8220;151.                                             34. Magwire MM, Fabian DK, Schweyen H, Cao C, Longdon B, et al. (2012) <br />  5. Sadd BM, Schmid-Hempel P (2006) Insect immunity shows specificity                        Genome-wide association studies reveal simple genetic basis resistance to <br />     protection secondary pathogen exposure. Current Biology 16: 1206&#226;&#8364;&#8220;1210.                 naturally coevolving viruses Drosophila melanogaster. Plos Genetics 8: <br />  6. McTaggart SJ, Wilson PJ, Little TJ (2012) Daphnia magna shows reduced                       e1003057. <br />     infection secondary exposure pathogen. Biol Lett 8: 972&#226;&#8364;&#8220;975.                  35. Gomariz-Zilber E, Poras M, Thomas-Orillard M (1995) Drosophila C virus: <br />  7. Little TJ, O&#226;&#8364;&#8482;Connor B, Colegrave N, Watt K, Read AF (2003) Maternal transfer                experimental study infectious yields underlying pathology Drosophila <br />     strain-specific immunity invertebrate. Curr Biol 13: 489&#226;&#8364;&#8220;492.                      melanogaster laboratory populations. J Invertebr Pathol 65: 243&#226;&#8364;&#8220;247. <br />                                                                                             36. Filipe D, Thomas-Orillard M (1998) Experimental study Drosophila <br />  8. Sadd BM, Kleinlogel Y, Schmid-Hempel R, Schmid-Hempel P (2005) Trans- <br />                                                                                                 melanogaster laboratory population infected food contamination. Endocy- <br />     generational immune priming social insect. Biol Lett 1: 386&#226;&#8364;&#8220;388. <br />                                                                                                 tobiosis Cell Research 12: 163&#226;&#8364;&#8220;176. <br />  9. Roth O, Joop G, Eggert H, Hilbert J, Daniel J, et al. (2010) Paternally derived <br />                                                                                             37. Teixeira L, Ferreira A, Ashburner M (2008) The Bacterial Symbiont Wolbachia <br />     immune priming offspring red flour beetle, Tribolium castaneum. <br />                                                                                                 Induces Resistance RNA Viral Infections Drosophila melanogaster. Plos <br />     Journal Animal Ecology 79: 403&#226;&#8364;&#8220;413. <br />                                                                                                 Biology 6: 2753&#226;&#8364;&#8220;2763. <br /> 10. Tidbury HJ, Pedersen AB, Boots M (2011) Within transgenerational                    38. Longdon B, Fabian DK, Hurst GDD, Jiggins FM (2012) Male-killing Wolbachia <br />     immune priming insect DNA virus. Proceedings Royal Society                protect Drosophila bifasciata viral infection. BMC Microbiology <br />     B-Biological Sciences 278: 871&#226;&#8364;&#8220;876.                                                         12. <br /> 11. Schmid-Hempel P (2005) Evolutionary ecology insect immune defenses. Annu             39. Sullivan W, Ashburner M, Hawley S (2000) Drosophila Protocols. New York: <br />     Rev Entomol 50: 529&#226;&#8364;&#8220;551.                                                                    Cold Spring Harbor Laboratory Press. <br /> 12. Pham LN, Dionne MS, Shirasu-Hiza M, Schneider DS (2007) A specific primed               40. Reed LJ, Muench H (1938) A simple method estimating fifty cent <br />     immune response Drosophila dependent phagocytes. PLoS Pathog 3:                    endpoints. The American Journal Hygiene 27: 493&#226;&#8364;&#8220;497. <br />     e26.                                                                                    41. Karlikow M, Goic B, Saleh MC (2013) Rnai Antiviral Defense in <br /> 13. Roth O, Sadd BM, Schmid-Hempel P, Kurtz J (2009) Strain-specific priming                 Drosophila: Setting systemic immune response. Dev Comp Immunol. <br />     resistance red flour beetle, Tribolium castaneum. Proc Biol Sci 276: 145&#226;&#8364;&#8220;        42. Ryder E, Blows F, Ashburner M, Bautista-Llacer R, Coulson D, et al. (2004) <br />     151.                                                                                        The DrosDel collection: set P-element insertions generating custom <br /> 14. Witteveldt J, Cifuentes CC, Vlak JM, van Hulten MC (2004) Protection                     chromosomal aberrations Drosophila melanogaster. Genetics 167: 797&#226;&#8364;&#8220;813. <br />     Penaeus monodon white spot syndrome virus oral vaccination. J Virol          43. Longdon B, Obbard DJ, Jiggins FM (2010) Sigma viruses species of <br />     78: 2057&#226;&#8364;&#8220;2061.                                                                              Drosophila form major new clade rhabdovirus phylogeny. Proceedings <br /> 15. Wu JL, Nishioka T, Mori K, Nishizawa T, Muroga K (2002) A time-course                       Royal Society B 277: 35&#226;&#8364;&#8220;44. <br />     study resistance Penaeus japonicus induced artificial infection       44. Therneau T (2012) coxme: Mixed Effects Cox Models. 2.2&#226;&#8364;&#8220;3 ed. <br />     white spot syndrome virus. Fish Shellfish Immunol 13: 391&#226;&#8364;&#8220;403.                          45. Andersen PK, Gill RD (1982) Cox Regressio <br /> </body></html>