<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Mol Biol. 2011 November 4; 413(4): 751&#226;&#8364;&#8220;761. doi:10.1016/j.jmb.2011.09.004. <br />  <br />  <br />  <br />                            Systematic chromosomal deletion bacterial ribosomal protein <br />                            genes <br />                            Shinichiro Shoji&#226;&#8364;&#160;,&#226;&#8364;&#161;, Corey M. Dambacher&#226;&#8364;&#160;,&#226;&#8364;&#161;, Zahra Shajani&#226;&#8364;&#160;,&#194;&#167;, James R. Williamson&#226;&#8364;&#160;,&#194;&#167;, and <br />                            Peter G. Schultz&#226;&#8364;&#160;,* <br />                            &#226;&#8364;&#160;Departments Chemistry, The Scripps Research Institute, 10550 North Torrey Pines Road, La <br />  <br />                            Jolla, CA 92037, USA <br />                            &#194;&#167;Molecular   Biology Skaggs Institute Chemical Biology, The Scripps Research <br />                            Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Detailed studies ribosomal proteins, essential components protein biosynthetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 machinery, hampered lack readily accessible chromosomal deletions the <br />                                 corresponding genes. Here, report systematic genomic deletion 41 individual ribosomal <br />                                 protein genes Escherichia coli, included Keio collection. Chromosomal <br />                                 copies genes replaced antibiotic resistance gene presence an <br />                                 inducible, easy-<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-exchange plasmid-born allele. Using <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> collection, 9 ribosomal <br />                                 proteins (L15, L21, L24, L27, L29, L30, L34, S9 S17) nonessential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> <br />                                 induction conditions various temperatures. Taken previous results, this <br />                                 analysis revealed 22 54 E. coli ribosomal protein genes individually deleted <br />                                 genome. These strains allow expression truncated protein variants probe the <br />                                 importance RNA-protein interactions functional sites ribosome. This set strains <br />                                 enhance vivo studies ribosome assembly/function, ultimately allow <br />                                 systematic substitution ribosomal proteins RNA. <br />  <br />  <br />                            Keywords <br />                                 ribosome; translation; recombineering; ribonucleoprotein complex; protein engineering <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2011 Elsevier Ltd. All rights reserved. <br />                            * <br />                              Corresponding author: Tel: 858-784-9273. Fax: 858-784-9440. schultz@scripps.edu Address: 10550 N. Torrey Pines Road, La <br />                            Jolla, CA 92037, USA . <br />                            &#226;&#8364;&#161;These authors contributed equally <br />                            Author email addresses: <br />                            Shinichiro Shoji: ichishoji@gmail.com <br />                            Corey M. Dambacher: cdambach@scripps.edu <br />                            Zahra Shajani: zsyi@scripps.edu <br />                            James R. Williamson: jrwill@scripps.edu <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                            Supporting Information Available. <br />                            Propensity complementary vector removal/exchange, genomic deletion strategies, verification deletion strains PCR can <br />                            supplementary material online. <br />                             Shoji et al.                                                                                                  Page 2 <br />  <br />  <br />                                Introduction <br />                                            Ribosomes large ribonucleoprotein particles iteratively translate triplet codons of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mRNA corresponding amino acids growing polypeptide chain. Upon each <br />                                            round translational elongation, mRNA codon decoding center A <br />                                            (Aminoacyl) site decoded cognate aminoacyl-tRNA. A peptidyl moiety attached to <br />                                            tRNA P (Peptidyl) site transferred &#206;&#177;-amino group A-site aminoacyl- <br />                                            tRNA nucleophilic attack ester bond linkage peptidyl-tRNA, tRNAs in <br />                                            A P sites translocate adjacent P E (Exit) sites, respectively. The <br />                                            functional ribosome (70S prokaryotes 80S eukaryotes) contains 2 subunits: one <br />                                            large subunit small subunit. In E. coli, large (50S) subunit consists 2 <br />                                            ribosomal RNAs (rRNA; 23S 5S) 33 different ribosomal proteins (L1-L7, L9-L36), <br />                                            small (30S) subunit consists 1 rRNA 21 ribosomal proteins (S1-S21). <br />  <br />                                            Ribosomal proteins highly specialized, RNA-interacting proteins&#226;&#8364;&#8221;approximately half of <br />                                            conserved domains life, domain-specific. <br />                                            Moreover, composition ribosomal proteins highly conserved domain 1. <br />                                            Crystal structures revealed intricate protein-RNA interactions peripheral <br />                                            internal regions ribosome 2; 3. In addition, ribosomal proteins are <br />                                            suggested involved essential ribosomal functions 4; 5; 6; 7; 8; 9; 10; 11; 12; 13; 14. Over <br />                                            decades, significant efforts elucidate roles of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ribosomal proteins translation, numerous mutations confer diverse phenotypes <br />                                            antibiotic resistance E. coli isolated 15; 16; 17; 18. For instance, restrictive <br />                                            mutations S12, originally identified streptomycin-resistant/dependent strains, confer a <br />                                            hyperaccurate <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> (where frequency miscoding stop-codon readthrough are <br />                                            decreased) ribosome 19; 20, ram mutations S4 S5, identified revertants <br />                                            streptomycin-dependence, rise error-prone phenotype (where frequency of <br />                                            miscoding stop-codon readthrough increased) 21; 22. Additionally, restrictive <br />                                            mutations S12 (R86S P91Q), known slow rate ribosome <br />                                            translation, rise increased folding expression aggregation-prone <br />                                            eukaryotic proteins E. coli 23. However, functions ribosomal proteins not <br />                                            fully deciphered, primarily lack readily accessible mutant lines containing <br />                                            genetic deletions individual essential ribosomal proteins. <br />  <br />                                            &#206;&#187;-Red-mediated recombination powerful tool precisely alter bacterial genomes, <br />                                            providing simple method study protein function, genetic networks, genome <br />                                            evolution living cells 24. Previous efforts generate complete set gene deletion <br />                                            strains (of non-essential genes) E. coli resulted collection 3985 individual <br />                                            mutants covering ~90% known E. coli genes (Keio collection 25). However, 13 of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            54 ribosomal protein genes (encoding L1, L9, L11, L25, L31, L32, L33, L35, L36, S6, <br />                                            S15, S20, S21) deleted way. Here, employed one-step E. coli <br />                                            genome recombineering technique 26, create series genetic deletion mutants covering <br />                                            41 ribosomal protein genes (excluding covered the <br />                                            publically-available Keio collection). Deleterious effects deletions minimized by <br />                                            presence complementing plasmid carrying gene interest, precise <br />                                            genomic replacement using antibiotic marker. We begun use deletion <br />                                            strains probe functions individual ribosomal proteins translation. <br />  <br />                                Results <br />                                Deletion ribosomal protein genes E. coli chromosome <br />                                            Because ribosomal proteins thought essential, deleted chromosomal <br />                                            gene presence plasmid-born, inducible copy gene (see Supplemental <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                             Page 3 <br />  <br />  <br />                                            Experimental Procedures). We individually cloned 41 targeted ribosomal <br />                                            protein (RP) genes (amplified E. coli DH10B genomic DNA) target plasmid <br />                                            (pCDSSara) arabinose-inducible araBAD promoter (Fig. 1). E. coli strain DH10B <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            used parental strain ability stably maintain complementing <br />                                            plasmids. A selective toxic marker sacB, encoding levansucrase Bacillus subtilis, was <br />                                            included target plasmid facilitate subsequent exchange plasmid by <br />                                            vector interest. The toxic effects conferred sacB successful exchange of <br />                                            pCDSSara-RP RP expression plasmid confirmed presence 5% <br />                                            sucrose (Fig. S1). pCDSSara-RP plasmids transformed E. coli DH10B, and <br />                                            corresponding genes chromosome replaced chloramphenicol <br />                                            acetyltransferase (CAT) gene &#206;&#187; Red-mediated recombination presence of <br />                                            arabinose 26. The CAT gene amplified polymerase chain reaction it <br />                                            included 40-100 bp extensions homologous genomic regions flanking target <br />                                            gene. These PCR fragments purified transformed cells expressing &#206;&#187;-Red <br />                                            recombination presence complementing plasmid. <br />  <br />                                            Roughly 70% targeted genes (29 genes) precisely replaced CAT gene; <br />                                            remainder required partial replacement initial attempts failed (Tables 1; Fig. <br />                                            S2). For rplD (encoding L4), rplO (L15), rpmD (L30), rpsE (S5) contain an <br />                                            internal Shin-Dalgarno (SD) sequence 3&#226;&#8364;&#178;-downstream gene, stop codon the <br />                                            CAT gene introduced immediately 5&#226;&#8364;&#178; SD sequence, preserving translation the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            downstream gene. For rplE (encoding L5), rplM (L13), rplW (L23), rpmB (L28), rpsB (S2), <br />                                            rpsC (S3), 5&#226;&#8364;&#178; region gene left intact, 3&#226;&#8364;&#178; portion replaced by <br />                                            in-frame stop codon followed SD sequence CAT gene. Likewise, rpsD <br />                                            (encoding S4) rpsJ (S10), 3&#226;&#8364;&#178; region gene left intact, 5&#226;&#8364;&#178; portion <br />                                            replaced CAT gene, followed SD sequence start codon minimize <br />                                            possible polar effects downstream genes operon. Probable reasons for <br />                                            difficulties associated complete replacement RP genes the <br />                                            presence important regulatory element polycistronic mRNAs and/or stable <br />                                            secondary structures recombination sites. Unexpectedly, replacement rplL and <br />                                            rpsR genes, encoding L7/L12 S18, respectively, occurred plasmid copy <br />                                            gene expressed control IPTG (isopropyl &#206;&#178;-D-1- <br />                                            thiogalactopyranoside)-inducible trc promoter, requirement higher <br />                                            expression levels (Fig. 1). All recombination events verified genomic PCR with <br />                                            primer pairs targeting CAT gene flanking regions genome (Fig. S3). <br />  <br />                                Essentiality ribosomal protein genes <br />                                            In deletion mutants, full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>, wild-type protein conditionally expressed an <br />                                            inducible copy targeted gene encoded complementing plasmid. If gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            essential survival cell given condition, absence arabinose IPTG <br />                                            induce gene expression result cell death. If gene nonessential, cells would <br />                                            survive absence inducer. It 5 mutants (with chromosomal <br />                                            deletion gene encoding L15, L21, L29, L30, S9) showed obvious growth the <br />                                            absence inducer (Fig. 2a). To test essentiality genes thoroughly, all <br />                                            mutants grown liquid media absence inducer 30 &#194;&#176;C 37 &#194;&#176;C, <br />                                            plated media containing 5% sucrose select cells lost the <br />                                            complementing plasmid. Surviving colonies analyzed absence target gene <br />                                            presence desired chromosomal recombination event (Fig. S4). <br />  <br />                                            Mutants lost complementing plasmid included extensive set non- <br />                                            essential genes, encoding L15, L21, L24, L27, L29, L30, L34, S9, S17 <br />                                            (Fig. 2b). Growth mutant lacking L21 inhibited lower temperature (30 &#194;&#176;C), <br />                                            growth mutants lacking L24 L27 inhibited higher temperature (42 &#194;&#176;C) <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                               Page 4 <br />  <br />  <br />                                            (Fig. 2b). To knowledge, mutants lacking L21 L34 reported. The <br />                                            absence L21 confirmed mass spectroscopy analysis 14N-labeled <br />                                            ribosomes isolated L21 deletion strain, spiked 15N-labeled ribosomes isolated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            parent DH10B strain 27. The results displayed complete absence protein L21 <br />                                            intact 50S 70S subunits isolated L21 deletion strain, confirmed the <br />                                            stoichiometric presence essential protein components ribosomes (Fig. 3). <br />                                            Here, &#226;&#8364;&#339;protein level&#226;&#8364;? refers concentration (or abundance) individual ribosomal <br />                                            proteins present 14N labeled ribosomes isolated &#206;&#8221;L21 line, normalized that <br />                                            15N labeled ribosomes isolated parent DH10B strain. The L35 signal also <br />                                            missing data, L35 detected wild-type sample either. In general, <br />                                            peptides shorter ribosomal proteins L31-L36 difficult resolve this <br />                                            technique 27. A similar analysis performed L34 deletion strain, same <br />                                            reason, L34 peptides identified 14N-ribosomes deletion strain <br />                                            15N-labeled wild-type ribosomes (data shown). Mutants lacking L15, L24, L27, <br />                                            L29, L30, S9, S17 isolated previously various groups diverse <br />                                            contexts 28; 29; 30; 31; 32. Mutants lacking L28 S13 30; 31; 33 insertion mutant <br />                                            S8 gene 34 isolated previously, unable remove the <br />                                            complementing plasmid corresponding strains created, possibly to <br />                                            differences nature <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> strain background. <br />  <br />                                Probing importance protein-RNA contacts ribosomal 30S A 50S P sites <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Crystal structures ribosomal complexes revealed intricate interactions between <br />                                            ribosomal proteins RNAs, including rRNA, tRNA, mRNA, the <br />                                            ribosome 4; 35; 36. The importance interactions poorly understood, to <br />                                            lack facile experimental create analyze mutations ribosomal <br />                                            proteins. By using mutant strains obtained study, able create series <br />                                            truncations ribosomal proteins L5, L27, S19, positioned proximity to <br />                                            functional ribosomal sites, evaluate significance interactions formed these <br />                                            proteins (Fig. 4a). Mutant versions genes placed control trc promoter <br />                                            pTrc99A-derived plasmid compatible pCDSSara carrying wild-type copy, and <br />                                            introduced corresponding deletion mutants. Because absence genomic <br />                                            allele, homogeneous population mutant wild-type protein expressed within <br />                                            cell presence IPTG arabinose, respectively; presence sucrose, <br />                                            surviving cells lost sacB-containing vector, resulting production only <br />                                            mutant protein cell. <br />  <br />                                            With system, C-terminal tail S19, protrudes the <br />                                            decoding center 30S A site, truncated 7 amino acids (S19C&#206;&#8221;7), cell <br />                                            growth substantially perturbed (Fig. 4b). Furthermore, removal 3 consecutive Lys <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            residues C terminus (S19C&#206;&#8221;3), unresolved crystal structures 4, <br />                                            did significantly affect growth. In 50S P site, removal L5 loop interacting with <br />                                            tRNA (L5&#206;&#8221;70-83) caused severe growth defect, consistent recent study suggesting <br />                                            yeast counterpart loop (&#226;&#8364;&#339;P-site loop&#226;&#8364;?) important cell viability tRNA <br />                                            binding 37. Finally, truncation 9 amino acids N-terminal tail L27 (L27N&#206;&#8221;9), <br />                                            remove interactions A-site P-site tRNAs, minimally affected cell <br />                                            growth over-expressed system. This surprising fact the <br />                                            presence L27 non-essential cell viability. <br />  <br />                                            Mutated truncated ribosomal protein variants altered affinities ribosome. <br />                                            Indeed, high level truncated L27 (L27&#206;&#8221;N16) expression required restore growth <br />                                            L27 deletion (Fig. 4b), imply reduced interaction ribosome. To <br />                                            assess ribosomal incorporation truncated protein variants used study <br />                                            (L5&#206;&#8221;70-83, L27N&#206;&#8221;16, L27N&#206;&#8221;20, S19C&#206;&#8221;16; able grow &#206;&#8221;S19 cells <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                Page 5 <br />  <br />  <br />                                           expressing S19C&#206;&#8221;23 truncation, suggesting truncation lethal) used <br />                                           quantitative mass spectrometry. 14N-labeled proteins expressed deletion lines <br />                                           harboring various truncations assessed comparison 15N-labeled wild-type <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ribosomes. We truncated variants analyzed retained ribosome. In <br />                                           &#206;&#8221;L27 mutants expressing L27N&#206;&#8221;16 L27N&#206;&#8221;20 truncations, observed elevated <br />                                           levels truncated L27 protein 14N-labeled samples compared wild-type 15N- <br />                                           labeled samples. We observed retention S19C&#206;&#8221;16 protein &#206;&#8221;S19 cells, we <br />                                           able detect changes relative abundance truncated S19, other <br />                                           ribosomal proteins &#206;&#8221;S19 cells expressing S19C&#206;&#8221;16. Ribosomes &#206;&#8221;L5 cells <br />                                           expressing L5 truncation mutant (L5&#206;&#8221;70-83) contain slightly elevated <br />                                           levels L5 protein compared wild-type ribosomes, proteins detected in <br />                                           ribosomes wild-type abundance. Additionally, exception ribosomes <br />                                           isolated &#206;&#8221;S19 cells expressing S19C&#206;&#8221;16, ribosomes containing truncated <br />                                           variants (L5&#206;&#8221;70-83, L27N&#206;&#8221;16, L27N&#206;&#8221;20) slightly elevated levels of <br />                                           ribosomal protein L31 (Fig. S5). <br />  <br />                                Discussion <br />                                           The deletion lines generated accelerate studies various activities of <br />                                           ribosome individual ribosomal proteins, shed light mechanism of <br />                                           translation assembly ribonucleoprotein complexes 38; 39; 40. For example, studies of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ribosome assembly pathways limited small number available deletion <br />                                           mutants difficulties associated vitro reconstitution ribosome assembly 41. <br />                                           Analysis vivo assembly intermediates depletion various ribosomal proteins <br />                                           provide insights regarding involvement proteins assembly of <br />                                           prokaryotic ribosomes. Moreover, deletion collection generated allows for <br />                                           extensive analysis point deletion mutations specific ribosomal protein genes. Our <br />                                           data suggest E. coli DH10B survive deletion 9 ribosomal protein <br />                                           genes targeted, including 2 genes (encoding L21 L34) considered <br />                                           essential. L21 encoded rplU gene, co-transcribed its <br />                                           downstream gene, rpmA, encoding L27. Previous attempts generate viable L21 deletion <br />                                           lines failed unintentional alteration L27 expression. L21 assembled <br />                                           ribosome near 5&#226;&#8364;&#178; end 23S rRNA forms extensive interactions L20. <br />                                           A previous vitro study showed omitting L21 affects binding L30 assembly <br />                                           intermediates 42. However, unable detect substantial alteration relative <br />                                           abundance ribosomal proteins large subunit mass spectroscopy L21 <br />                                           absent vivo. These results suggest vivo ribosome assembly pathways may <br />                                           differ aspects vitro 43. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           To illustrate utility strain collection, carried preliminary study by <br />                                           generating truncation mutants S19, L27, L5. Each nucleotide triplet mRNA is <br />                                           decoded specific aminoacyl-tRNA molecule, contingent specific <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> <br />                                           pairing decoding center ribosomal 30S A site. The incoming aminoacyl-tRNA <br />                                           forms interactions anticodon stem loop proteins, S12 S19. S12 been <br />                                           extensively investigated mutations gene affect translational fidelity and <br />                                           antibiotic resistance (e.g., &#226;&#8364;&#339;restrictive&#226;&#8364;? mutations causing hyperaccuracy) 14. However, S19 <br />                                           poorly studied recent crystal structures revealed C-terminal interactions <br />                                           tRNA possible involvement decoding 4. The terminal 3 amino acids (4 amino <br />                                           acids Thermus thermophilus) unresolved crystal structures, the <br />                                           nature interactions tRNA remains unclear. We observed nearly wild-type <br />                                           growth C-terminal tail S19 truncated 3 residues, suggesting <br />                                           minimal participation residues protein translation. However, 4 residues <br />                                           preceding play important role S19 function, truncation analysis <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                            Page 6 <br />  <br />  <br />                                           revealed poor growth C-terminal 7-13 residues protein deleted. <br />                                           Although data indicate importance S19 30S A site, studies are <br />                                           necessary elucidate precise role S19 translation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The peptidyltransferase center catalytic center peptide bond formation large <br />                                           subunit mainly comprised rRNA, supports hypothesis peptide bond <br />                                           formation catalyzed solely RNA prebiotic environment 44; 45. However, <br />                                           crystal structures bacterial ribosomes, N-terminal tail L27 protrudes into <br />                                           peptidyltransferase center contacts backbone P-site A-site tRNAs 4; 36. <br />                                           Although L27 non-essential, complete depletion protein strongly perturbs growth as <br />                                           ribosomal functions (Fig. 2) 10; 28. It previously shown truncation of <br />                                           3 amino acids N-terminus L27 decreases peptidyltransferase <br />                                           activity 10, catalysis peptide bond formation directly involve L27 46. We <br />                                           increased expression levels truncated L27 (as observed expressing <br />                                           N-terminal deletions 3-16 residues, observed expressing N-terminal <br />                                           deletion 20 residues) alleviate deleterious effects growth, suggesting L27 <br />                                           N-terminal tail play role assembly 50S subunit. <br />  <br />                                           L5 lies interface 30S 50S subunits, generating inter-subunit bridge B1b, and <br />                                           makes extensive contacts 5S rRNA. L5 forms interactions tRNA the <br />                                           50S P site, specifically elbow region tRNA. We postulated removal the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tRNA-interacting loop L5 perturb precise positioning P-site tRNA at <br />                                           peptidyltransferase center affect catalytic efficiency peptide bond <br />                                           formation. Indeed, cells expressing L5 mutant loop (L5&#206;&#8221;70-83) showed <br />                                           severely impaired growth. Although plausible removal loop caused <br />                                           unfolding substantial decrease affinity L5 ribosome, yeast L11 (eukaryotic <br />                                           homolog L5) similar truncation retained ribosome affected tRNA <br />                                           binding 37. Further investigation precise role L5-tRNA interaction P site <br />                                           remains performed future. Mutated truncated ribosomal protein variants may <br />                                           altered affinities ribosome. The methods described allow <br />                                           investigators determine ribosomal retention relative abundance proteins in <br />                                           sucrose gradient-isolated ribosomes. <br />  <br />                                           Engineering cellular translational machinery allowed biosynthesis proteins <br />                                           containing non-native amino acids diverse chemical properties 47. These amino acids <br />                                           genetically incorporated cellular proteins orthogonal aminoacyl-tRNA <br />                                           synthetase (AARS)/tRNA pairs response nonsense codons frame-shift (4- <br />                                           base) codons 48; 49. Notably, recently shown 16S rRNA engineered to <br />                                           enhance decoding frame-shift codons non-natural aminoacyl-tRNA E. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           coli 50; 51. Just rRNA, ribosomal proteins involved conformational changes of <br />                                           ribosome interactions translation components. It likely engineering <br />                                           functionally critical ribosomal proteins, directed evolution, improve the <br />                                           translational efficiency non-native amino acids incorporated response <br />                                           nonsense frame-shift condons. Fine-tuning ribosomal protein function also <br />                                           provide differential modulation ribosomal activities, including decoding, peptidyltransfer, <br />                                           intermolecular interactions. Since ribosomal proteins considered later <br />                                           additions primitive all-RNA ribosome core 52, examine whether <br />                                           ribosomal protein components functionally replaced entirety by <br />                                           RNA, cis-acting insertion libraries placed near site protein deletion, by <br />                                           trans-acting vivo RNA aptamers. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                         Page 7 <br />  <br />  <br />                                Materials Methods <br />                                Plasmid design <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pCDSSara pCDSStrc constructed combining PCR-amplified fragments <br />                                           containing pCloDF13 origin spectinomycin/streptomycin-resistance marker <br />                                           derived pCDF-1b (Novagen), sacB gene B. subtilis, araBAD trc <br />                                           promoter pBAD (Invitrogen) pTrc99A (GenBank: A13038.1), respectively. The <br />                                           ribosomal protein genes PCR amplified genomic DNA E. coli DH10B (F&#226;&#710;&#8217; <br />                                           mcrA &#206;&#8221;(mrr-hsdRMS-mcrBC) &#206;&#166;80dlacZ&#206;&#8221;M15 &#206;&#8221;lacX74 endA1 recA1 deoR <br />                                           &#206;&#8221;(ara,leu)7697 araD139 galU galK nupG rpsL(StrR) &#206;&#187;&#226;&#710;&#8217;) alteration native stop <br />                                           codon TAA (if already) cloned XhoI SpeI sites pCDSSara or <br />                                           pCDSStrc (only rplL rpsR). <br />  <br />                                Knockout experiments <br />                                           These plasmids (pCDSSara pCDSStrc rplL rpsR) transformed DH10B <br />                                           carrying pKD46, expression plasmid &#206;&#187;-Red recombinase 26. These cells grown <br />                                           30 &#194;&#176;C 2YT media containing 50 &#206;&#188;g/ml spectinomycin 100 &#206;&#188;g/ml ampicillin. 0.2% <br />                                           arabinose (pCDSSara) 0.2% arabinose 1 mM IPTG (pCDSStrc) added to <br />                                           media optical density 600 nm (OD600) reached 0.15-0.2; cells were <br />                                           harvested OD600 approximately 0.5; electrocompetent cells prepared by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           washing 10% glycerol 3 times 4 &#194;&#176;C. Freshly-prepared electrocompetent cells were <br />                                           transformed 300-1000 ng DNA containing CAT gene sequence PCR-amplified <br />                                           pEVOL 53. Primers containing 40-70 nt 3&#226;&#8364;&#178; overhangs homologous target <br />                                           genomic regions used amplify CAT gene. A spacer sequence containing Shine- <br />                                           Dalgarno sequence included template necessary (Fig. S2). For rplL, rpsB, <br />                                           rpsR deletions, DNA fragments containing CAT gene ~100-bp extension with <br />                                           homology flanking genomic regions constructed overlap extension PCR to <br />                                           increase propensity recombination. Transformants recovered 2YT containing <br />                                           0.2% arabinose (pCDSSara) 1 mM IPTG (pCDSStrc), plated LB agar containing 50 <br />                                           &#206;&#188;g/ml spectinomycin, 7-15 &#206;&#188;g/ml chloramphenicol, arabinose IPTG, incubated <br />                                           37 &#194;&#176;C 2-4 days. Surviving colonies individually analyzed genomic PCR using <br />                                           taq polymerase (Invitrogen) primers annealing CAT gene (5&#226;&#8364;&#178;-CAT Fwd, 5&#226;&#8364;&#178;- <br />                                           AATTAACTGCAGTTACTGCCACTCATCGCAGTACTGTTG-3&#226;&#8364;&#178;; 3&#226;&#8364;&#178;-CAT Rev, 5&#226;&#8364;&#178;- <br />                                           AATTAACATATGGAGAAAAAAATCACTGGATATACC-3&#226;&#8364;&#178;) and/or flanking <br />                                           genomic regions ~100-bp upstream downstream target gene (Fig. S3). Loss of <br />                                           pKD46 strains confirmed loss ampicillin resistance (data shown). <br />  <br />                                Vector removal/exchange <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           To remove pCDSSara pCDSStrc plasmids deletion mutants, cells grown <br />                                           overnight LB media containing 7 &#206;&#188;g/ml chloramphenicol. 100 &#206;&#188;l culture was <br />                                           spread LB agar containing 7 &#206;&#188;g/ml chloramphenicol 5% sucrose incubated 30 <br />                                           &#194;&#176;C 37 &#194;&#176;C 2-4 days. Loss plasmid confirmed LB agar containing 50 &#206;&#188;g/ <br />                                           ml spectinomycin appropriate inducer. To test facility vector exchange, the <br />                                           IPTG-inducible pTrc-S16 plasmid used electro-transformation S16 deletion <br />                                           strain. These cells recovered overnight (at 8 h) 2YT + IPTG 37 &#194;&#176;C, then <br />                                           plated LB agar plates containing 100ug/ml ampicillin 1mM IPTG 5% sucrose. <br />                                           Single colonies surviving sucrose counter-selection picked streaked LB <br />                                           agar plates containing 100ug/ml ampicillin + 1mM IPTG, 100ug/ml ampicillin, 50ug/ml <br />                                           spectinomycin + 0.2% arabinose (Fig. S1). <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                              Page 8 <br />  <br />  <br />                                Truncation analysis <br />                                            For expression truncated ribosomal proteins, pTrc99A vector PCR amplified <br />                                            using primers SpeI-pTrc-Fwd 5&#226;&#8364;&#178;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            AATTAAACTAGTGGCGGATGAGAGAAGATTTTCAGCC-3&#226;&#8364;&#178; XhoI-pTrc-Rev 5&#226;&#8364;&#178;- <br />                                            AATTAACTCGAGTTCCTCCTGTGTGAAATTGTTATCCGCTCACAATTCC-3&#226;&#8364;&#178; and <br />                                            digested XhoI SpeI. The truncated L5 &#206;&#8221;70-83 gene amplified overlap <br />                                            extension PCR, truncated L27 S19 genes amplified PCR. These <br />                                            fragments digested cloned XhoI SpeI sites pTrc99A vector <br />                                            created above. The resulting plasmids contain genes control trc promoter. The <br />                                            &#206;&#8221;rplE pL5 &#206;&#8221;rpmA pL27, &#206;&#8221;rpsS pS19 strains transformed plasmids, <br />                                            transformants recovered 2YT containing 0.2% arabinose 2 h selected on <br />                                            LB agar supplemented 50 &#206;&#188;g/ml spectinomycin, 100 &#206;&#188;g/ml ampicillin, 0.2% <br />                                            arabinose overnight 37 &#194;&#176;C. <br />  <br />                                Sample preparation mass spectrometry <br />                                            Cells grown 37 &#194;&#176;C minimal media (per liter: 12.08 g Na2HPO4, 3 g KH2PO4, 0.5 <br />                                            g NaCl, 2 g glucose, 20.3 mg MgCl2, 24.6 mg MgSO4, 30 mg Na2EDTA, 7.5 mg CaCl2, <br />                                            25.05 mg FeCl3, 0.27 mg ZnSO4, 0.27 mg CuSO4, 0.27 mg CoCl2, 0.18 mg MnSO4, 20 mg <br />                                            biotin, 20 mg folic acid, 50 mg thiamine, 50 mg niacinamide, 50 mg riboflavin, 50 mg Ca- <br />                                            pantothenate, 0.001 mg vitamin B12, 1 g (14NH4)2SO4 1 g (15NH4)2SO4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            14N- 15N-labeling mutant DH10B ribosomal proteins, respectively), harvested <br />                                            OD600 approximately 0.5 centrifugation 7,000 &#195;&#8212; g 10 min, stored at <br />                                            &#226;&#710;&#8217;80 &#194;&#176;C subsequent analysis. Frozen cell pellets L21 L34 mutants the <br />                                            parent DH10B strain thawed re-suspended high magnesium lysis buffer (20 mM <br />                                            Tris HCl pH 7.5, 100 mM NH4Cl, 10mM MgCl2, 0.5 mM EDTA, 6 mM &#206;&#178;- <br />                                            mercaptoethanol) lysed bead-beater (BioSpec Products, Inc, Bartesville, OK) <br />                                            using 0.1 mm zirconia/silica beads. Insoluble debris removed centrifugation <br />                                            steps: initial spin 13000 &#195;&#8212; g 10 min followed second spin 13000 &#195;&#8212; g 90 <br />                                            min. A 1mL aliquot supernatant layered 35 mL 10-40% sucrose gradient (50 <br />                                            mM Tris HCl pH 7.8, 10 mM MgCl2, 100 mM NH4Cl) SW 32 rotor (Beckman Coulter, <br />                                            Fullerton, CA) centrifuged 26,000 rpm 4 &#194;&#176;C 16 h order separate out <br />                                            subunits. The gradients fractionated Brandel fractionator. Subunit concentration <br />                                            fractions estimated measuring optical density 260 nm. The quantitative <br />                                            mass spectrometry experiments performed fractions sucrose gradient <br />                                            peak corresponding 50S subunit fractions 70S subunit the <br />                                            &#206;&#8221;L21 &#206;&#8221;L34 mutants. Eights fractions run total. For fractions, 50 pmol <br />                                            14N sample used 50 pmol 15N 70S added standard. Samples were <br />                                            prepared electrospray ionization time-of flight (ESI-TOF) analysis described <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            previously 43. <br />  <br />                                ESI-TOF mass spectrometry <br />                                            The peptides analyzed Agilent 1100 Series high performance liquid <br />                                            chromatography (HPLC) instrument coupled Agilent ESI-TOF instrument with <br />                                            capillary flow electrospray. Peptides separated ACN gradient 0.1% formic <br />                                            acid flow rate 7 &#206;&#188;L/min. The steps gradient 5-15% ACN 10 min, <br />                                            15-50% ACN 70 min 50-95% ACN 4 min. Data collected m/z <br />                                            range 100-1300. <br />  <br />                                Quantitative analysis protein levels using mass spectrometry <br />                                            Briefly, feature list generated using software Agilent (Mass Hunter Mass <br />                                            Profiler), feature entire isotopic envelope single ion contains the <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                             Page 9 <br />  <br />  <br />                                            monoisotopic peak isotopomers. Features defined m/z value the <br />                                            monoisotopic peak, charge ion retention time column. Feature lists <br />                                            compared theoretical digest ribosomal proteins E. coli 70S <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ribosome. Experimental features assigned possible identities theoretical ion&#226;&#8364;&#8482;s <br />                                            charge identical mass 50 ppm experimental feature&#226;&#8364;&#8482;s <br />                                            mass. 14N/15N pairs extracted possible identities features which <br />                                            matched retention time 0.1 min other. For cases multiple <br />                                            identities feature peptide matched multiple <br />                                            proteins, features discarded. For remaining feature pairs, segments the <br />                                            complete mass spectrum extracted theoretical isotope distributions fit to <br />                                            extracted spectra using program Isodist 54. Two distributions fit, unlabeled <br />                                            (natural abundance deletion strains) fully labeled (15N fixed 99.3%, <br />                                            corresponding external standard, case wild-type ribosomes). The fits all <br />                                            extracted spectra evaluated visually, spectra poor fitting distributions were <br />                                            rejected. The amplitudes given Isodist yield relative amounts unlabeled 15N <br />                                            labeled peptide sample used calculate abundance protein 44. <br />                                            In addition samples run searching extracting 15N standard <br />                                            peaks. By searching presence 15N 14N verify absent L21 <br />                                            protein L21 deletion strain truly absent deletion strain standard <br />                                            (Fig. 3b 3c). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                            We thank Drs. Huiwang Ai, Tsotne Javahishvili, Charles E. Melancon helpful comments discussions. <br />                                            We grateful Lyn&#226;&#8364;&#8482;Al L. Nosaka Han Xiao technical assistance. We thank Dr. Eli Chapman for <br />                                            providing pTrc99A plasmid (supplied pTrc99A-GroEL). This work supported Uehara <br />                                            Memorial Foundation (fellowship S.S.), National Institutes Health (NIH) grants R37-GM-053757 (to <br />                                            J.R.W) grant 5R01 GM062159 (to P.G.S.). <br />  <br />  <br />                                Abbreviations used <br />  <br />                                           CAT          chloramphenicol acetyltransferase <br />                                           IPTG         isopropyl &#206;&#178;-D-1-thiogalactopyranoside <br />                                           RP           ribosomal protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SD           Shine-Dalgarno <br />  <br />                                References <br />                                            1. Roberts E, Sethi A, Montoya J, Woese CR, Luthey-Schulten Z. Molecular signatures ribosomal <br />                                               evolution. Proc. Natl. Acad. Sci. USA. 2008; 105:13953&#226;&#8364;&#8220;13958. [PubMed: 18768810] <br />                                            2. Ban N, Nissen P, Hansen J, Moore PB, Steitz TA. The complete atomic structure large <br />                                               ribosomal subunit 2.4 &#195;&#8230; resolution. Science. 2000; 289:905&#226;&#8364;&#8220;920. [PubMed: 10937989] <br />                                            3. Wimberly BT, Brodersen DE, Clemons WM Jr. Morgan-Warren RJ, Carter AP, Vonrhein C, <br />                                               Hartsch T, Ramakrishnan V. Structure 30S ribosomal subunit. Nature. 2000; 407:327&#226;&#8364;&#8220;339. <br />                                               [PubMed: 11014182] <br />                                            4. Jenner L, Demeshkina N, Yusupova G, Yusupov M. Structural rearrangements ribosome at <br />                                               tRNA proofreading step. Nat. Struct. Mol. Biol. 2010; 17:1072&#226;&#8364;&#8220;1078. [PubMed: 20694005] <br />                                            5. Hoang L, Fredrick K, Noller HF. Creating ribosomes all-RNA 30S subunit. Proc. Natl. <br />                                               Acad. Sci. USA. 2004; 101:12439&#226;&#8364;&#8220;12443. [PubMed: 15308780] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                      Page 10 <br />  <br />  <br />                                           6. Takyar S, Hickerson RP, Noller HF. mRNA helicase activity ribosome. Cell. 2005; 120:49&#226;&#8364;&#8220; <br />                                              58. [PubMed: 15652481] <br />                                           7. Schmeing TM, Voorhees RM, Kelley AC, Gao YG, Murphy F. V. t. Weir JR, Ramakrishnan V. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              crystal structure ribosome bound EF-Tu aminoacyl-tRNA. Science. 2009; 326:688&#226;&#8364;&#8220; <br />                                              694. [PubMed: 19833920] <br />                                           8. Gao YG, Selmer M, Dunham CM, Weixlbaumer A, Kelley AC, Ramakrishnan V. The structure of <br />                                              ribosome elongation factor G trapped posttranslocational state. Science. 2009; <br />                                              326:694&#226;&#8364;&#8220;699. [PubMed: 19833919] <br />                                           9. Ogle JM, Ramakrishnan V. Structural insights translational fidelity. Annu. Rev. Biochem. <br />                                              2005; 74:129&#226;&#8364;&#8220;177. [PubMed: 15952884] <br />                                           10. Maguire BA, Beniaminov AD, Ramu H, Mankin AS, Zimmermann RA. A protein component at <br />                                                heart RNA machine: importance protein L27 function bacterial <br />                                                ribosome. Mol. Cell. 2005; 20:427&#226;&#8364;&#8220;435. [PubMed: 16285924] <br />                                           11. Cukras AR, Southworth DR, Brunelle JL, Culver GM, Green R. Ribosomal proteins S12 S13 <br />                                                function control elements translocation mRNA:tRNA complex. Mol. Cell. 2003; <br />                                                12:321&#226;&#8364;&#8220;328. [PubMed: 14536072] <br />                                           12. Diaconu M, Kothe U, Schlunzen F, Fischer N, Harms JM, Tonevitsky AG, Stark H, Rodnina MV, <br />                                                Wahl MC. Structural basis function ribosomal L7/12 stalk factor binding and <br />                                                GTPase activation. Cell. 2005; 121:991&#226;&#8364;&#8220;1004. [PubMed: 15989950] <br />                                           13. Fei J, Kosuri P, MacDougall DD, Gonzalez RL. Coupling ribosomal L1 stalk tRNA <br />                                                dynamics translation elongation. Mol. Cell. 2008; 30:348&#226;&#8364;&#8220;359. [PubMed: 18471980] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           14. Zaher HS, Green R. Hyperaccurate error-prone ribosomes exploit distinct mechanisms during <br />                                                tRNA selection. Mol. Cell. 2010; 39:110&#226;&#8364;&#8220;120. [PubMed: 20603079] <br />                                           15. Dabbs ER. Mutational alterations 50 proteins Escherichia coli ribosome. Mol. Gen. Genet. <br />                                                1978; 165:73&#226;&#8364;&#8220;78. [PubMed: 362167] <br />                                           16. Isono S, Isono K. Mutations affecting structural genes genes coding modifying <br />                                                enzymes ribosomal proteins Escherichia coli. Mol. Gen. Genet. 1978; 165:15&#226;&#8364;&#8220;20. [PubMed: <br />                                                362162] <br />                                           17. Zengel JM, Young R, Dennis PP, Nomura M. Role ribosomal protein S12 peptide chain <br />                                                elongation: analysis pleiotropic, streptomycin-resistant mutants Escherichia coli. J. Bacteriol. <br />                                                1977; 129:1320&#226;&#8364;&#8220;1329. [PubMed: 321423] <br />                                           18. Zaman S, Fitzpatrick M, Lindahl L, Zengel J. Novel mutations ribosomal proteins L4 L22 <br />                                                confer erythromycin resistance Escherichia coli. Mol. Microbiol. 2007; 66:1039&#226;&#8364;&#8220;1050. <br />                                                [PubMed: 17956547] <br />                                           19. Gorini L, Kataja E. Phenotypic Repair Streptomycin Defective Genotypes E. Coli. Proc. <br />                                                Natl. Acad. Sci. USA. 1964; 51:487&#226;&#8364;&#8220;493. [PubMed: 14171463] <br />                                           20. Ozaki M, Mizushima S, Nomura M. Identification functional characterization protein <br />                                                controlled streptomycin-resistant locus E. coli. Nature. 1969; 222:333&#226;&#8364;&#8220;339. [PubMed: <br />                                                4181187] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           21. Birge EA, Kurland CG. Reversion streptomycin-dependent strain Escherichia coli. Mol. <br />                                                Gen. Genet. 1970; 109:356&#226;&#8364;&#8220;369. [PubMed: 4925047] <br />                                           22. Brownstein BL, Lewandowski LJ. A mutation suppressing streptomycin dependence. I. An effect <br />                                                ribosome function. J. Mol. Biol. 1967; 25:99&#226;&#8364;&#8220;109. [PubMed: 5340532] <br />                                           23. Siller E, DeZwaan DC, Anderson JF, Freeman BC, Barral JM. Slowing bacterial translation speed <br />                                                enhances eukaryotic protein folding efficiency. J Mol. Biol. 2010; 396s:1310&#226;&#8364;&#8220;1318. [PubMed: <br />                                                20043920] <br />                                           24. Sharan SK, Thomason LC, Kuznetsov SG, Court DL. Recombineering: homologous <br />                                                recombination-based method genetic engineering. Nat. Protoc. 2009; 4:206&#226;&#8364;&#8220;223. [PubMed: <br />                                                19180090] <br />                                           25. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner <br />                                                BL, Mori H. Construction Escherichia coli K-12 in-frame, single-<span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: the <br />                                                Keio collection. Mol. Syst. Biol. 2006; 2 2006.0008. <br />                                           26. Datsenko KA, Wanner BL. One-step inactivation chromosomal genes Escherichia coli K-12 <br />                                                using PCR products. Proc. Natl. Acad. Sci. USA. 2000; 97:6640&#226;&#8364;&#8220;6645. [PubMed: 10829079] <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                 Page 11 <br />  <br />  <br />                                           27. Sharpe Elles LM, Sykes MT, Williamson JR, Uhlenbeck OC. A dominant negative mutant the <br />                                               E. coli RNA helicase DbpA blocks assembly 50S ribosomal subunit. Nucleic Acids Res. <br />                                               2009; 37:6503&#226;&#8364;&#8220;6514. [PubMed: 19734347] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           28. Wower IK, Wower J, Zimmermann RA. Ribosomal protein L27 participates 50 S subunit <br />                                               assembly peptidyl transferase reaction. J. Biol. Chem. 1998; 273:19847&#226;&#8364;&#8220;19852. [PubMed: <br />                                               9677420] <br />                                           29. Stoffler-Meilicke M, Dabbs ER, Ehrlich R, Stoffler G. A mutant Escherichia coli lacks <br />                                               ribosomal proteins S17 L29 used localize proteins ribosomal surface. Eur. <br />                                               J. Biochem. 1985; 150:485&#226;&#8364;&#8220;490. [PubMed: 3926498] <br />                                           30. Dabbs ER. Selection Escherichia coli mutants proteins missing ribosome. J. <br />                                               Bacteriol. 1979; 140:734&#226;&#8364;&#8220;737. [PubMed: 387748] <br />                                           31. Bubunenko M, Baker T, Court DL. Essentiality ribosomal transcription antitermination <br />                                               proteins analyzed systematic gene replacement Escherichia coli. J. Bacteriol. 2007; <br />                                               189:2844&#226;&#8364;&#8220;2853. [PubMed: 17277072] <br />                                           32. Franceschi FJ, Nierhaus KH. Ribosomal protein L20 replace assembly-initiator protein L24 <br />                                               low temperatures. Biochemistry. 1988; 27:7056&#226;&#8364;&#8220;7059. [PubMed: 2461735] <br />                                           33. Cukras AR, Green R. Multiple effects S13 modulating strength intersubunit <br />                                               interactions ribosome translation. J. Mol. Biol. 2005; 349:47&#226;&#8364;&#8220;59. [PubMed: <br />                                               15876367] <br />                                           34. Kang Y, Durfee T, Glasner JD, Qiu Y, Frisch D, Winterberg KM, Blattner FR. Systematic <br />                                               mutagenesis Escherichia coli genome. J. Bacteriol. 2004; 186:4921&#226;&#8364;&#8220;4930. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               15262929] <br />                                           35. Korostelev A, Trakhanov S, Laurberg M, Noller HF. Crystal structure 70S ribosome-tRNA <br />                                               complex reveals functional interactions rearrangements. Cell. 2006; 126:1065&#226;&#8364;&#8220;1077. <br />                                               [PubMed: 16962654] <br />                                           36. Selmer M, Dunham CM, Murphy FV, Weixlbaumer A, Petry S, Kelley AC, Weir JR, <br />                                               Ramakrishnan V. Structure 70S ribosome complexed mRNA tRNA. Science. <br />                                               2006; 313:1935&#226;&#8364;&#8220;1942. [PubMed: 16959973] <br />                                           37. Rhodin MH, Dinman JD. A flexible loop yeast ribosomal protein L11 coordinates P-site tRNA <br />                                               binding. Nucleic Acids Res. 2010; 38:8377&#226;&#8364;&#8220;8389. [PubMed: 20705654] <br />                                           38. Schmeing TM, Ramakrishnan V. What recent ribosome structures revealed the <br />                                               mechanism translation. Nature. 2009; 461:1234&#226;&#8364;&#8220;1242. [PubMed: 19838167] <br />                                           39. Kaczanowska M, Ryden-Aulin M. Ribosome biogenesis translation process Escherichia <br />                                               coli. Microbiol. Mol. Biol. Rev. 2007; 71:477&#226;&#8364;&#8220;494. [PubMed: 17804668] <br />                                           40. Cech TR. Crawling RNA world. Cell. 2009; 136:599&#226;&#8364;&#8220;602. [PubMed: 19239881] <br />                                           41. Sykes MT, Williamson JR. A complex assembly landscape 30S ribosomal subunit. Annu <br />                                               Rev Biophys. 2009; 38:197&#226;&#8364;&#8220;215. [PubMed: 19416066] <br />                                           42. Herold M, Nierhaus KH. Incorporation additional proteins complete assembly map of <br />                                               50 S subunit Escherichia coli ribosomes. J. Biol. Chem. 1987; 262:8826&#226;&#8364;&#8220;8833. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               3298242] <br />                                           43. Sykes MT, Shajani Z, Sperling E, Beck AH, Williamson JR. Quantitative proteomic analysis of <br />                                               ribosome assembly turnover vivo. J. Mol. Biol. 2010; 403:331&#226;&#8364;&#8220;345. [PubMed: 20709079] <br />                                           44. Nissen P, Hansen J, Ban N, Moore PB, Steitz TA. The structural basis ribosome activity in <br />                                               peptide bond synthesis. Science. 2000; 289:920&#226;&#8364;&#8220;930. [PubMed: 10937990] <br />                                           45. Noller HF. Evolution Protein Synthesis RNA World. Cold Spring Harb. Perspect Biol. <br />                                               2010:a003681. <br />                                           46. Rodnina MV, Beringer M, Wintermeyer W. How ribosomes make peptide bonds. Trends Biochem. <br />                                               Sci. 2007; 32:20&#226;&#8364;&#8220;26. [PubMed: 17157507] <br />                                           47. Liu CC, Schultz PG. Adding new chemistries genetic <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span>. Annu. Rev. Biochem. 2010; <br />                                               79:413&#226;&#8364;&#8220;444. [PubMed: 20307192] <br />                                           48. Wang L, Brock A, Herberich B, Schultz PG. Expanding genetic <span id='am-396' about='Thesaurus:code' typeof='owl:Thing'>code</span> Escherichia coli. <br />                                               Science. 2001; 292:498&#226;&#8364;&#8220;500. [PubMed: 11313494] <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                 Page 12 <br />  <br />  <br />                                           49. Anderson JC, Wu N, Santoro SW, Lakshman V, King DS, Schultz PG. An expanded genetic code <br />                                               functional quadruplet codon. Proc. Natl. Acad. Sci. USA. 2004; 101:7566&#226;&#8364;&#8220;7571. [PubMed: <br />                                               15138302] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           50. Wang K, Neumann H, Peak-Chew SY, Chin JW. Evolved orthogonal ribosomes enhance the <br />                                               efficiency synthetic genetic code expansion. Nat. Biotechnol. 2007; 25:770&#226;&#8364;&#8220;777. [PubMed: <br />                                               17592474] <br />                                           51. Neumann H, Wang K, Davis L, Garcia-Alai M, Chin JW. Encoding multiple unnatural amino acids <br />                                               evolution quadruplet-decoding ribosome. Nature. 2010; 464:441&#226;&#8364;&#8220;444. [PubMed: <br />                                               20154731] <br />                                           52. Bokov K, Steinberg SV. A hierarchical model evolution 23S ribosomal RNA. Nature. 2009; <br />                                               457:977&#226;&#8364;&#8220;980. [PubMed: 19225518] <br />                                           53. Young TS, Ahmad I, Yin JA, Schultz PG. An enhanced unnatural amino acid <br />                                               mutagenesis E. coli. J. Mol. Biol. 2010; 395:361&#226;&#8364;&#8220;374. [PubMed: 19852970] <br />                                           54. Sperling E, Bunner AE, Sykes MT, Williamson JR. Quantitative analysis isotope distributions in <br />                                               proteomic mass spectrometry using least-squares Fourier transform convolution. Anal. Chem. <br />                                               2008; 80:4906&#226;&#8364;&#8220;4917. [PubMed: 18522437] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                          Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 1. Complementing plasmids rescue genomic deletions ribosomal protein genes <br />                                           All genes encoding 41 targeted ribosomal proteins E. coli genome were <br />                                           disrupted presence complementing plasmid. Of these, 39 genes disrupted in <br />                                           presence pCDSSara-RP (RP corresponds protein encoded gene) <br />                                           expresses ribosomal protein presence arabinose induction. rplL rpsR <br />                                           (encoding L12 S18, respectively) disrupted presence pCDSStrc-L12 or <br />                                           S18, respectively, expresses protein presence IPTG induction. Spc, <br />                                           specitinomycin; Sm, streptomycin. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                           Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 2. Growth analysis deletion strains <br />                                           (a) Growth dependence deletion mutants complementing genes. All strains <br />                                           contained genomic deletion (&#206;&#8221;) plasmid (p) copy ribosomal protein gene <br />                                           indicated. Overnight cultures mutants grown presence 0.2% arabinose (for all <br />                                           &#206;&#8221;rplL pL12 &#206;&#8221;rpsR pS18) 1 mM IPTG (for &#206;&#8221;rplL pL12 &#206;&#8221;rpsR pS18) <br />                                           diluted 100-fold, spotted LB agar supplemented antibiotics the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           inducer (+) 0.2% glucose (&#226;&#710;&#8217;), incubated 15-36 h 37 &#194;&#176;C. (b) Temperature <br />                                           dependence knockout strains lacking complementing plasmid. pCDSSara plasmids <br />                                           containing nonessential ribosomal protein genes removed sacB-based <br />                                           counterselection, overnight cultures resulting cells diluted 100-fold, spotted, <br />                                           grown LB agar 48 h various temperatures. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                          Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 3. Protein Level 70S subunit &#206;&#8221;L21 mutant ribosomes <br />                                           (a) Protein level single fraction 70S subunit peak. Error bars represent the <br />                                           standard deviation multiple measurements different peptides/ions same <br />                                           protein. In case single measurement obtained, open circle used. (b) A <br />                                           peptide L5 (MWEFFER) identified present L21 mutant strain <br />                                           (blue) 15N ribosomal standard (red). (c) A peptide L21 (MYAVFQSGGK) is <br />                                           identified present 15N ribosomal standard (red) absent L21 mutant <br />                                           strain. The position 14N peptide appear shown blue arrow. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                         Page 16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 4. Mutational analysis L27, L5, S19 function <br />                                           (a) A schematic view protein-tRNA interactions catalytic centers ribosome. <br />                                           The N-terminal 16 residues L27, residues 70-83 L5, C-terminal 16 residues S19 <br />                                           shown red. tRNA nucleotides comprising sites potential interaction the <br />                                           proteins shown stick representation (Protein Data Bank entries 318H 318I). (b) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Growth analysis mutants containing homogeneous populations various truncated <br />                                           ribosomal proteins. Overnight cultures mutants carrying pCDSSara pTrc <br />                                           plasmids containing full-length truncated version protein, respectively, were <br />                                           diluted 100-fold spotted LB agar supplemented antibiotics inducer <br />                                           indicated. pTrc plasmids containing wild-type gene green fluorescent protein (GFP) <br />                                           gene used controls. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                                                                  NIH-PA Author Manuscript                                     NIH-PA Author Manuscript        NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                    Table 1 <br />                                                                                                                               a <br />                                                                 Summary deletion strains ribosomal protein genes . <br />  <br />                                                                                            Replaced   Remaining <br />                                                                                                                                                                                         Shoji et al. <br />  <br />  <br />  <br />  <br />                                                                                                 b          c             d                     e <br />                                                                  Protein   Gene   Operon   (bp)       (bp)        Plasmid         Essentiality <br />                                                                  L2        rplB   S10      All        None        pCDSSara-L2     Essential <br />                                                                  L3        rplC   S10      All        None        pCDSSara-L3     Essential <br />                                                                  L4        rplD   S10      1-585      [586-606]   pCDSSara-L4     Essential <br />                                                                  L5        rplE   spc      130-540    1-129       pCDSSara-L5     Essential <br />                                                                  L6        rplF   spc      All        None        pCDSSara-L6     Essential <br />                                                                  L10       rplJ   rif      All        None        pCDSSara-L10    Essential <br />                                                                  L7/L12    rplL   rif      All        None        pCDSStrc-L12    Essential <br />                                                                  L13       rplM   L13      31-429     1-30        pCDSSara-L13    Essential <br />                                                                  L14       rplN   spc      All        None        pCDSSara-L14    Essential <br />                                                                  L15       rplO   spc      1-424      [425-435]   pCDSSara-L15    Nonessential <br />                                                                  L16       rplP   S10      All        None        pCDSSara-L16    Essential <br />                                                                  L17       rplQ   alpha    All        None        pCDSSara-L17    Essential <br />                                                                  L18       rplR   spc      All        None        pCDSSara-L18    Essential <br />                                                                  L19       rplS   S16      All        None        pCDSSara-L19    Essential <br />                                                                  L20       rplT   L20      All        None        pCDSSara-L20    Essential <br />                                                                  L21       rplU   L21      All        None        pCDSSara-L21    Nonessential <br />                                                                  L22       rplV   S10      All        None        pCDSSara-L22    Essential <br />                                                                  L23       rplW   S10      31-303     1-30        pCDSSara-L23    Essential <br />  <br />  <br />  <br />  <br /> J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                                                                  L24       rplX   spc      All        None        pCDSSara-L24    Nonessential <br />                                                                  L27       rpmA   L21      All        None        pCDSSara-L27    Nonessential <br />                                                                                                                                            f <br />                                                                  L28       rpmB   L28      31-237     1-30        pCDSSara-L28    Essential <br />                                                                  L29       rpmC   S10      All        None        pCDSSara-L29    Nonessential <br />                                                                  L30       rpmD   spc      1-171      [172-180]   pCDSSara-L30    Nonessential <br />                                                                  L34       rpmH   L34      All        None        pCDSSara-L34    Nonessential <br />                                                                  S1        rpsA   S1       All        None        pCDSSara-S1     Ess <br /> </body></html>