<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Patterns Evolutionary Conservation Essential Genes <br /> Correlate Their Compensability <br /> Tobias Bergmiller1,2&#194;&#164;*, Martin Ackermann1,2, Olin K. Silander1,2,3* <br /> 1 Department Environmental Systems Science, ETH Zurich, Zurich, Switzerland, 2 Department Environmental Microbiology, Eawag, Dubendorf, Switzerland, <br /> 3 Computational Systems Biology, Biozentrum, University Basel, Basel, Switzerland <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Essential genes <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> fundamental cellular functions required viability organism. For reason, essential <br />     genes highly conserved organisms. However, case: orthologues genes are <br />     essential organism essential organisms absent genomes. This suggests <br />     that, course evolution, essential genes rendered nonessential. How gene non-essential? Here <br />     used genetic manipulation deplete products 26 different essential genes Escherichia coli. This depletion <br />     results lethal <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, rescued overexpression non-homologous, non-essential <br />     gene, likely replacement essential function. We that, smaller number cases, the <br />     essential genes fully deleted genome, suggesting complete functional replacement possible. Finally, <br />     essential genes function replaced laboratory likely non-essential not <br />     present taxa. These results consistent notion patterns evolutionary conservation essential <br />     genes influenced compensability&#226;&#8364;&#8221;that is, easily functionally replaced, example through <br />     increased expression genes. <br />  <br />   Citation: Bergmiller T, Ackermann M, Silander OK (2012) Patterns Evolutionary Conservation Essential Genes Correlate Their Compensability. PLoS <br />   Genet 8(6): e1002803. doi:10.1371/journal.pgen.1002803 <br />   Editor: Ivan Matic, Universite&#194;&#180; Paris Descartes, INSERM U1001, France <br />   Received January 27, 2012; Accepted May 12, 2012; Published June 28, 2012 <br />   Copyright: &#195;&#376; 2012 Bergmiller et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: TB, MA, OKS supported grant Swiss National Science Foundation MA. In addition, OKS supported Ambizione <br />   fellowship Swiss National Foundation. The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: tobias.bergmiller@ist.ac.at (TB); olinsilander@gmail.com (OKS) <br />   &#194;&#164; Current address: IST Austria, Klosterneuburg, Austria <br />  <br />  <br />  <br /> Introduction                                                                             The possibility connection compensability and <br />                                                                                       gene conservation raised occasion <br />    Essential genes <span id='am-475' about='Thesaurus:code' typeof='owl:Thing'>code</span> central cellular processes required                   previously. Geissler et al. [17] observed Escherichia coli cell <br /> the viability organism. Many recent studies used <br />                                                                                       division protein ZipA poorly conserved taxa. Under the <br /> experimental data determine gene essentiality large number <br />                                                                                       assumption proteins fulfill role other <br /> of bacteria [1&#226;&#8364;&#8220;11]. The central role essential genes play <br /> suggests highly conserved evolution,                        taxa, looked suppressor mutations obviate the <br /> several comparative genomic analyses confirmed                              requirement ZipA, single <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> FtsA <br /> hypothesis [12&#226;&#8364;&#8220;15]. A second implication <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                           suppressed lethal zipA phenotype. <br /> conservation essential genes tend remain essential                     Here used systematic approach investigate how <br /> evolution: gene essential organism, orthologues                   frequently functions essential genes Escherichia coli can <br /> of gene usually essential organisms (Figure 1).                     replaced laboratory conditions, the <br />    However, genes follow                             frequency process correlates patterns evolutionary <br /> patterns: genes essential organism non-                      conservation. <br /> essential organisms; cases, genes essential                   <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> gain insight question, used following <br /> in organism absent lost genomes                      methodology. First, compromised function essential <br /> other organisms [13,15,16]. Instances essential genes                        gene Escherichia coli decreasing expression tightly <br /> have non-essential, lost completely                          regulated promoter. We increased expression level a <br /> genomes, suggest changes physiological environ-                     large number genes identify genes capable of <br /> mental conditions altered essentiality gene,                    compensating function impaired essential gene. <br /> the genetic context changed way allow loss                           Repeating process large number essential genes, we <br /> previously essential function. In case, second gene (either                    isolated set genes functionally replace essential <br /> paralogous unrelated original essential gene)                       genes overexpressed. We majority of <br /> perform essential function. This raises question                   compensating genes homologous impaired <br /> there connection compensability conservation                         essential genes, exhibit similar functions. In cases, the <br /> level essential genes.                                                             compensating genes capable fully replacing functions of <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                       1                               June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                      Gene Compensability Influences Conservation <br />  <br />  <br />  <br />   Author Summary                                                            transformation maintenance separate plasmids that <br />                                                                             complement function gene unlikely, plasmids <br />   In given organism, fraction genes                     share replication origins resistance markers. <br />   genome required viability; experimen- <br />   tally deleted, organism dies. Interestingly, set               Screening Overexpression Library All E. coli Open <br />   essential genes usually identical closely                 Reading Frames High Copy Suppressors <br />   related organisms. Genes essential one <br />                                                                                Next, assembled library overexpression plasmids using <br />   organism nonessential sister taxa or <br />   missing genomes. This suggests that,                   ASKA(-) strain collection [23]. The ASKA(-) library consists of <br />   course evolution, genes rendered non-                  4123 clones contain plasmid E. coli open <br />   essential consequently lost. How genes                     reading frame cloned IPTG-inducible Plac promoter <br />   non-essential? It possible changes                   N-terminal (His)6-tag. Addition IPTG induces strong <br />   organism&#226;&#8364;&#8482;s living conditions render previously essential                  expression downstream open reading frame. We removed <br />   functions unessential. Alternatively, possible that,                26 clones containing plasmids essential genes <br />   evolution, function essential gene                mentioned ASKA library. We pooled the <br />   taken gene, essential gene                    remaining 4097 clones extracted plasmid DNA this <br />   dispensable. Here, tested second                           mixed pool. <br />   hypothesis experimentally laboratory. We tried                     Each conditional lethal mutant transformed aliquot <br />   replace functions essential genes bacterium                 purified plasmid pool plated restrictive agar plates <br />   Escherichia coli. We genes easily               (where essential genes expressed) IPTG to <br />   replaced laboratory likely lost             induce expression plasmid-encoded genes. We induced expres- <br />   course evolution. This suggests differences                sion plasmid encoded genes 50 mM IPTG, because <br />   evolutionary fate essential genes                      higher induction levels known deleterious growth: <br />   partially explained easily functions                  51% E. coli proteins expressed control Plac 1 mM <br />   taken genes.                                                IPTG cause lethality [23]. We measured transformation <br />                                                                             efficiency: transformed plated permissive <br />                                                                             conditions selection plasmid-encoded chloramphen- <br /> the essential gene, allowing deletion essential gene        icol resistance, strains murA fldA gave rise least <br /> the genome. Finally, essential genes               1.26104 colonies, minimal number clones required <br /> function compensated laboratory likely           coverage 95% variants transformed [24] (see Table <br /> to non-essential present bacterial genomes,              S1). <br /> raising possibility similar compensatory mechanisms               We recovered 10 transformants plate, and <br /> allow essential gene loss occur natural populations.                  restreaked restrictive agar plates IPTG confirm <br />    Many previous studies shown gene essentiality             growth. Upon successful growth clones, plasmids were <br /> mutable characteristic dependent genetic                 extracted. To discriminate possible chromosomal sup- <br /> background organism environmental conditions [18&#226;&#8364;&#8220;             pressor mutations high copy suppressors (HCS) encoded on <br /> 21] (such genes termed conditionally essential genes). The              plasmids, plasmid retransformed ancestral <br /> results present imply cases possible         conditional lethal mutant permissive conditions, and <br /> to predict essential genes likely conditionally        colonies tested growth restrictive conditions. In <br /> essential.                                                                  case successful growth, plasmid sequenced. To control <br />                                                                             suppression multiple plasmids, purified plasmids <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Results/Discussion                                                          ASKA library repeated retransformation test, yielding <br />                                                                             results before. For cases high copy <br /> Construction Strains Conditional Expression                      suppression purified plasmid confirmed, a <br /> Essential Genes                                                             second round transformation selection described above <br />    We constructed collection Escherichia coli strains         carried out. <br /> essential genes placed control conditionally               For strains (Para-aspS, plsC, plsB, ffh, glmUS, gltX), <br /> expressed promoter. The essential genes selected                colonies recovered, HCS plasmids isolated <br /> variety functional classes [22], exhibit wide range             rounds transformation selection (Figure 1). In <br /> conservation levels [15]. In addition, genes             cases possible chromosomal mutations were <br /> been consistently essential bacteria       responsible rescuing conditional lethal phenotypes, possibly <br /> been examined empirically, essential         mutation Para promoter mitigate repression. In three <br /> (Figure 1). A total 26 genes chosen (approximately 10%           cases, colonies recovered (Para-gyrA, metK, murA). For <br /> the essential gene content E. coli); genes           murA, poor transformation efficiency. <br /> essential tandem operons (nrdAB, dnaTC glmUS). To control               Finally, conditional lethal mutants (Para-adk, dnaTC, proS, yeaZ) <br /> the expression essential genes, replaced native             recovered repeatedly plasmid coding gene ybiV. <br /> promoters arabinose-inducible araBAD promoter (Para;           We presumed ybiV interfered function the <br /> Methods). By shifting mutant strains medium L-              arabinose promoter: essential genes functionally <br /> arabinose medium L-arabinose supplemented               different, expression ybiV promoted growth these <br /> D-glucose, expression essential gene repressed,           mutants restrictive conditions. Furthermore, ybiV has <br /> all cases resulted growth inhibition severe growth defects       previously screens using Para construct [25]. We <br /> (Figure 2). We tested plasmid encoding corresponding          excluded HCS ybiV subsequent analyses. <br /> essential gene rescued lethal growth phenotype,               This left essential genes identified <br /> the case, operons dnaTC nrdAB (Table S1). This           HCS (Figure 1). For genes, we <br /> is likely fact tandem operons,           subjectively chose HCS analysis &#226;&#8364;&#8220; the <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              2                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                           Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  <br /> Figure 1. Few high copy suppressors conserved consistently essential genes. We gathered information on <br /> orthologous genes additional taxa essentiality experimentally investigated. All genes listed essential E. coli; each <br /> gene, orthologue taxa indicated essential (E; white background), non-essential (N; light grey), associated large fitness reduction <br /> (N*), unknown (U). Cases orthologue indicated A (absent; dark grey background). For example, spoT, is <br /> essential E. coli, essential taxa absent one. The second column indicates high <br /> copy suppressors isolated (genes suppressors isolated highlighted light grey). In parentheses high copy <br /> suppressors recovered screen analyzed further. A cladogram showing evolutionary relationships these <br /> taxa shown table. Abbreviations: S.ty, Salmonella typhi Ty2; A.ba, Acinetobacter baylyi; C.cr, Caulobacter crescentus NA100; F.tu, <br /> Francisella tularensis U112; S.pn, Streptococcus pneumonia TIGR4; B.su, Bacillus subtilis 168; S.au, Staphylococcus aureus 8325; M.ge, Mycoplasma <br /> genitalium G37; M.pu, Mycoplasma pulmonis UAB CTIP. a) No reciprocal best hit orthologue ftsK exists B. subtilis apparent duplication <br /> (genes BSU16800 BSU29805); gene essential. b) No reciprocal best hit orthologue plsC exists F. tularensis apparent <br /> duplication (FTN1749 FTN1750); FTN1749 essential. <br /> doi:10.1371/journal.pgen.1002803.g001 <br />  <br /> exception essential gene degS, included                essential genes strains harboring HCS plasmid, with <br /> HCS.                                                                            expression suppressor induced using 50 mM IPTG. We were <br />                                                                                 successful cases (these essential gene - HCS pairs were: <br /> Complementation Essential Genes High Copy                                 dapA/nanA, spoT/mutT, pyrH/cmk fldA/fldB; Figure 3). We were <br /> Suppressors                                                                     able delete essential genes strain carrying <br />    Next, tested recovered HCS plasmids                     control plasmid. <br /> replace functions respective essential genes,              The essential genes deleted strains <br /> viability rely low-level transcription repressed              containing HCS plasmids. This suggested presence <br /> Para promoter. We attempted knock corresponding                      HCS, low-level expression essential genes was <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 3                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                           Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  <br /> Figure 2. Differential colony formation conditional lethal mutant strains arabinose glucose-containing LB agar plates. The <br /> native promoters 23 genes operon pairs replaced arabinose-inducible araBAD promoter (see Methods). By shifting mutant <br /> strains medium 0.1% L-arabinose medium L-arabinose supplemented 0.4% D-glucose, expression essential gene <br /> was repressed. In cases resulted growth inhibition severe growth defects. The gene native promoter replaced indicated on <br /> the right hand row; ancestral strain TB741 shown top. In case degS, substantial growth occurs repression, <br /> suggesting growth inhibited gene products sufficiently depleted; observed previously [19]. This also <br /> true, lesser extent, spoT, ftsK, ygjD. To grow spot plates, cultures grown overnight 0.1% arabinose, diluted LB <br /> medium, 5 ml indicated dilution spotted plates incubated 37u 10 hours. Those strains indicated cross were <br /> incubated 24 hours. <br /> doi:10.1371/journal.pgen.1002803.g002 <br />  <br /> sufficient allow growth. Without HCS, residual low-level               The single exception pattern HCS ftnA (coding for <br /> expression did allow growth (Figure 4). Alternatively, possible       ferritin), rescued ftsK- nrdAB depletion. ftnA nrdB exhibit <br /> that high copy suppressor increased expression Para,               structural homology (Table 2), suggesting specific <br /> thus restored normal levels essential proteins. Therefore,            functional replacement. In case ftsK, mechanism of <br /> tested seven HCS (two HCS included                suppression clear. One possibility FtnA protein <br /> degS) increased expression Para promoter. We used                    alleviates oxidative stress [26] results loss FtsK, a <br /> chromosomally encoded Para-phoA fusion monitor expression               consequence double strand breaks chromosomal DNA [27]. <br /> Para conditions expression HCS induced. HCS               However, cases, mechanism does appear be <br /> clones overexpressing yciR, yhbJ, ftnA exhibited slightly lower             moderated FtnA restoring expression Para (Figure S1). <br /> levels PhoA-activity compared controls, overexpression              These data 23 essential genes operons <br /> degP, rho dpiA resulted slightly elevated levels PhoA-activity        assessed, functions completely replaced <br /> (approximately 1.5-fold increase control). However,               non-orthologous genes. In additional cases, functions of <br /> activity 50-fold activity Para                  essential genes completely replaced: over- <br /> induced 0.1% L-arabinose (Figure S1). This suggested             expression second gene enabled cellular viability the <br /> of HCS rescued conditional <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>                   expression essential gene largely abolished. In contrast, <br /> increasing expression essential gene.                                    overexpression second gene, growth occurred. <br />    To test HCS genes caused general non-specific rescue, <br /> we purified HCS plasmids ASKA collection                       Dosage Dependence <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>Knockout</span> Mutant Strains <br /> transformed plasmids conditional lethal mutant.                    We quantified high copy suppressors restored <br /> We tested regrowth way retransformation               growth strains knock the <br /> test described above. With exception (see below),               essential gene, measuring growth yield depended on <br /> HCS plasmids restored growth conditional lethal                    dosage HCS expression. The complemented knockout <br /> strain strain recovered from. Therefore                    mutants showed qualitatively different responses increasing <br /> assumed observed high copy suppression                     expression HCS, measured the inducer <br /> specific mechanistic link depletion phenotype                   IPTG added growth medium (Figure 5). The dapA knockout <br /> the high copy suppressor (Table 1), consequence                   exhibits low levels growth levels inducer, suggesting <br /> of high copy suppressor-mediated induction expression                      NanA low level activity DapA <br /> from Para.                                                                      substrate, high levels activity required sustain <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 4                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                        Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  <br /> Figure 3. Differential colony formation essential <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutant strains. We attempted knock essential genes in <br /> strains isolated HCS. We successful cases shown here. For essential gene, row illustrates growth <br /> that observed native promoter replaced Para, permissive (0.1% arabinose) restrictive (0.4% glucose) growth conditions. The <br /> bottom row indicates growth observed essential gene knocked out, conditions suppressor induced <br /> (0 mM IPTG) induced (1 mM IPTG). In absence HCS, growth observed restrictive conditions. However, HCS <br /> is induced, robust growth observed, cases essential gene knocked out. Note DspoT strain is <br /> complemented mutT plasmid induction mutT expression. To grow spot plates, cultures grown overnight 0.1% arabinose <br /> with 15 mg/ml chloramphenicol 1 mM IPTG, diluted LB medium, 5 ml indicated dilution spotted plates containing 0.1% <br /> arabinose 15 mg/ml chloramphenicol, 0.4% glucose 1 mM IPTG. Arabinose plates incubated 14 hours 37u, glucose <br /> plates incubated 40 hours. <br /> doi:10.1371/journal.pgen.1002803.g003 <br />  <br /> growth. On hand, spoT pyrH knockouts                  sequence protein structure suppressed essential gene: <br /> exhibit growth suppressor uninduced, suggesting             dapA nanA share amino acid homology (Table 2), nanA <br /> that proteins promiscuous,           closest homologue dapA E. coli. However, <br /> low levels activity required growth (Figure 5).                   phylogenetic analysis shows distantly related, <br />                                                                              likely diverged recent common ancestor <br />                                                                              bacteria (Figure S2B). The second HCS &#226;&#8364;&#8220; essential gene pair <br /> Amino Acid Structural Similarities High                          showing amino acid homology fldA fldB (Table 2), fldB <br /> Copy Suppressors Essential Genes                                         closest homologue fldA E. coli genome. <br />    To gain insight mechanisms suppression,                    Phylogenetic analysis suggests proteins diverged <br /> compared amino acid sequences (using Smith-Waterman                      origin gamma-proteobacteria (Figure S2A). <br /> alignments [28]) protein structures (using pairwise structural              In set HCS unable knock the <br /> alignments [29]; Table S2) complementing HCS &#226;&#8364;&#8220; essential              corresponding essential gene, pair exhibits sequence homol- <br /> gene pairs. Of gene pairs knock              ogy: proteins DegS DegP [30] (Table 2). Again, degP (with <br /> the essential gene, HCS share homology amino acid                     degQ) closest homologue degS E. coli genome. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              5                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                        Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  <br /> Figure 4. Differential colony formation conditional lethal mutant strains suppressive plasmids. In cases isolated HCS, but <br /> were unable knockout corresponding essential gene presence HCS. For essential gene, row illustrates the <br /> growth observed native promoter replaced Para, permissive (0.1% arabinose) restrictive (0.4% glucose) conditions. The <br /> bottom row indicates growth observed strain containing plasmid corresponding HCS, permissive (0.1% <br /> arabinose) restrictive (0.4% glucose) conditions. Expression HCS induced 50 mM IPTG. When expression essential gene is <br /> repressed, robust growth observed presence high copy suppressor. To grow spot plates, cultures grown overnight in <br /> 0.1% arabinose 15 mg/ml chloramphenicol, diluted LB medium, 5 ml indicated dilution spotted plates containing either <br /> 0.1% arabinose, 0.4% glucose 50 mM IPTG. The arabinose plates incubated 24 hours, glucose IPTG plates incubated <br /> for 48 hours 37u, Para-degS, incubated 14 hours. <br /> doi:10.1371/journal.pgen.1002803.g004 <br />  <br />  <br /> Functional Similarities HCS Essential Genes                      (dapA-nanA, fldA-fldB, degS-degP) known functional <br />   In cases, functions essential gene              similarities. <br /> complementing HCS appear related. Besides having amino                    DapA NanA belong N-acetylneuraminate lyase <br /> acid structural similarities, pairs mentioned            subfamily catalyze similar biochemical reactions [31], <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              6                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                                        Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  Table 1. Possible functional similarities non-complementing high copy suppressors essential genes. <br />  <br />  <br />  Essential gene                            High copy suppressor                            Depletion phenotype                          Possible functional similarity <br />  <br />  degS &#226;&#8364;&#8220; serine endoprotease                degP &#226;&#8364;&#8220; serine protease                          Sensitivity outer membrane stress,        RseA cleavage sigma E <br />                                                                                            temperature sensitivity                      activation [30] <br />  degS &#226;&#8364;&#8220; serine endoprotease                yciR &#226;&#8364;&#8220; cyclic di-GMP phosphodiesterase/         Sensitivity outer membrane stress,        csgD repression silencing of <br />                                            modulator RNase II                           temperature sensitivity                      curli expression [57]; reduction of <br />                                                                                                                                         outer membrane protein folding <br />                                                                                                                                         stress analogous silencing of <br />                                                                                                                                         ompA opmC expression [58]; <br />                                                                                                                                         modulation RNase II activity <br />                                                                                                                                         analogous HicA [59] <br />  ftsK &#226;&#8364;&#8220; cell division protein              yhbJ &#226;&#8364;&#8220; regulator glmS stability              Inhibition cell division, DNA damage      Modulation peptidoglycan <br />                                                                                            (filamentation) [60]                         structure [61] similar loss of <br />                                                                                                                                         function dacA [62] <br />  lolA &#226;&#8364;&#8220; periplasmic chaperone              dpiA &#226;&#8364;&#8220; response regulator                       Membrane defects [63]                        Signaling outer membrane <br />                                                                                                                                         stress [64] <br />  nrdAB &#226;&#8364;&#8220; ribonucleoside phosphate          ftnA - ferritin                                 Inhibition DNA replication cell       Reduction oxidative damage <br />  reductase                                                                                 division (filamentation) [65]                ROS-induced cell death iron <br />                                                                                                                                         sequestration [26] <br />  pssA &#226;&#8364;&#8220; phosphatidyl                       ipsU &#226;&#8364;&#8220; undecaprenyl diphosphate synthase        Filamentation [66]                           Structural lipid analogue <br />  serine synthase                                                                                                                        similar cardiolipin [67] <br />  ygjD &#226;&#8364;&#8220; t6A transfer-RNA                   rho &#226;&#8364;&#8220; transcriptional termination               Inhibition translation [68]               Adjustment transcript levels <br />  modification enzyme                                                                                                                    binding free RNA [69]; <br />                                                                                                                                         Restoration transcriptional <br />                                                                                                                                         attenuation [70] <br />  <br />  We derived depletion phenotypes conditional lethal mutants original function high copy suppressor genes literature. Proposed modes action of <br />  high copy suppressor genes based evidence literature. <br />  doi:10.1371/journal.pgen.1002803.t001 <br />  <br />  <br />  <br />  <br /> although dapA essential nanA non-essential. It previously                 DapA NanA example increased <br /> been shown single amino acid exchange turn NanA,                          concentration enzyme promiscuous activity promote <br /> N-acetylneuraminate lyase, efficient dihydrodipicolinate                        essential biochemical reaction restore viability absence <br /> synthase, dedicated function DapA. This amino acid                      gene originally encoding essential function. <br /> exchange hypothesized optimize turnover rate                             FldA FldB flavodoxins. FldB non-essential, contrast <br /> than specificity reaction [32]. Therefore, replacement                    FldA [1], replace function FldA expressed <br />  <br />  Table 2. Sequence structural homology essential genes high copy suppressors. <br />  <br />  <br />  Essential gene                           HCS                 Amino acid alignment e-value                           Structural homology p-value (2ln) <br />  <br />  dapA*                                    nanA                4.7e222                                                31.26 <br />  fldA*                                    fldB                1.7e233                                                22.49 <br />  pyrH*                                    cmk                 0.26                                                   1.23 <br />  spoT*                                    mutT                0.14                                                   2.28 <br />  degS                                     degP                2.7e247                                                20.99 <br />  degS                                     yciR                0.11                                                   2.59 <br />  ftsK                                     yhbJ                0.58                                                   1.59 <br />  lolA                                     dpiA                0.85                                                   0.91 <br />  nrdA                                     ftnA                0.071                                                  0.85 <br />  nrdB                                     ftnA                0.065                                                  7.64 <br />  pssA                                     ispU                0.95                                                   2.48 <br />  ygjD                                     rho                 0.01                                                   2.30 <br />  <br />  The Protein Information Resource (http://pir.georgetown.edu/pirwww/search/pairwise.shtml) used generate Smith-Waterman amino acid alignments the <br />  corresponding e-values; lower e-values imply higher homology sequences [28]. MAMMOTH (Matching Molecular Models Theory) [29] used to <br />  generate pairwise structural alignments; value 4.5 (p = 0.01) indicates significant structural homology [29]; structure available, structure model from <br />  http://modbase.compbio.ucsf.edu/modbase-cgi/index.cgi used. Essential genes knocked presence HCS indicated an <br />  asterisk. <br />  doi:10.1371/journal.pgen.1002803.t002 <br />  <br />  <br />  <br />          PLoS Genetics | www.plosgenetics.org                                       7                                 June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                         Gene Compensability Influences Conservation <br />  <br />  <br />  <br />  <br /> Figure 5. Gene dosage dependence essential gene knockouts complemented high copy suppressors. The strains we <br /> were able knockout essential genes grown presence respective HCS growth yields 7.5 hours growth were <br /> measured using optical density (OD) 600 nm. The dapA knockout complemented nanA exhibited nearly undetectable growth levels, <br /> suggesting complementation weak. All mutants exhibited significantly increased yields 50 mM IPTG, near maximal <br /> growth yields 500 mm IPTG. For pyrH spoT, growth 0 mM IPTG presumably results leaky expression Plac, carryover gene <br /> products small amounts IPTG overnight growth cultures (see Figure 3 Methods). Error bars indicate 95% confidence intervals. <br /> doi:10.1371/journal.pgen.1002803.g005 <br />  <br />  <br /> from native promoter [33]. However, results that FldB       guanosine longer triggers stringent response, thus <br /> can replace function FldA expressed high level.              allowing cell growth. <br />    DegS activates sigma E stress response proteolytic                     Other possible functional similarities pairs high copy <br /> degradation anti-sigma factor RseA [34] triggered                   suppressors essential genes listed Table 1. <br /> misfolded outer membrane porins [35], DegP                      We objectively evaluated hypothesis specific <br /> can fulfill function [30]. Although implies DegP           functional relationships essential genes HCS by <br /> overexpression fully compensate DegS function,              testing functional annotations essential genes their <br /> unable delete degS overexpressing degP.                               HCS similar expected chance. We collected the <br />    Two essential gene &#226;&#8364;&#8220; HCS pairs exhibit apparent             GO molecular function annotations [39] essential gene and <br /> amino acid structural similarity exhibit functional similarity:         corresponding HCS, calculated functional distance between <br /> pyrH/cmk spoT/mutT. cmk pyrH code nucleotide               pair (see Methods). Using functional distance measure, we <br /> kinase: PyrH converts uracil monophosphate uracil diphosphate,             essential genes complementary HCS genes are <br /> while Cmk converts cytosine monophosphate cytosine diphos-                 similar function expected chance <br /> phate. It known Cmk use cytidine uridine            (p = 0.0024, one-tailed Kolmogorov-Smirnov test). When exclude <br /> (the primary substrate PyrH) substrates. Additionally,           homologous pairs genes (degS-degP, dapA-nanA, fldA-fldB, a <br /> been shown previously cmk act high-copy suppressor           weak signal functional similarity remains (p = 0.05, one-tailed <br /> a temperature-sensitive pyrH allele [36]. Here shown             Kolmogorov-Smirnov test). Thus, appears HCS genes <br /> cmk fully suppressive deleting entire pyrH locus.                   compensate deleted essential genes specific comple- <br />    Both mutT spoT recognize phosphorylated guanosines                 mentation missing function. On hand, ybiV, is <br /> as substrates, cleave phosphoryl groups. SpoT key enzyme             likely non-specific HCS, shows pattern having greater <br /> of stringent response hydrolyzes penta/tetra guanosine                functional similarity paired essential genes be <br /> phosphate ((p)ppGpp) [37,38]. Deletion spoT leads                   expected chance. <br /> accumulation (p)ppGpp, turn activates stringent <br /> response leads cessation cell growth. One possible                  Correlation Evolutionary Patterns <br /> mechanism suppression mutT cleave phosphoryl                      Comparative genomic analyses shown genes are <br /> groups (p)ppGpp, converting phosphorylated                     essential E. coli tend conserved bacterial taxa [12&#226;&#8364;&#8220; <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               8                            June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                       Gene Compensability Influences Conservation <br />  <br />  <br /> 15]. In addition, recent empirical results shown                  Finally, shown function essential <br /> essential functions genes tend conserved: genes            gene replaced laboratory, orthologues gene <br /> essential taxon orthologues essential          likely non-essential absent genomes of <br /> taxa (Figure 1). There are, however, exceptions: essential              bacterial taxa. This observation suggests compensability <br /> genes conserved, non-essential               influence patterns evolutionary conservation: functions <br /> some taxa.                                                                   essential genes easier replace others, the <br />    Here, shown laboratory conditions,                 genes perform functions lost over <br /> functions essential genes completely partially             evolutionary time. <br /> replaced homologous unrelated non-essential genes. A                      Previous studies looked high copy suppressors lethal <br /> simple explanation connect observations:            phenotypes [25,40,41]. The majority studies been <br /> difficult replace function essential gene, gene       performed smaller scale screening suppressors of <br /> should highly conserved consistently essential            mutations cause non-lethal phenotypes, aim to <br /> bacterial taxa. We asked difficult           investigate gene function. Our study comparative and <br /> genes providing compensatory functions genes               systematic attempt quantify frequency suppressors of <br /> conserved consistently essential.                                        essential genes, test statistical association <br />    We used data conservation (see Methods) empirical                  gene conservation compensability. <br /> assessments essentiality [1,3&#226;&#8364;&#8220;11] test hypothesis. Of            The potential finding redundant, non-orthologous genes <br /> 23 essential genes operon pairs investigated,           functionally replace essential genes function of <br /> both conserved essential bacteria essentiality          genome size. Previous work shown bacterial species with <br /> has empirically assessed (Figure 1). Within set,          large genomes fewer essential genes species small <br /> an HCS single gene, ygjD. 15 essential genes            genomes [4,5]. One explanation observation large <br /> operons considered fully conserved             genomes, greater chance second gene encodes a <br /> bacteria nonessential taxa, both. Of these,             similar function. Thus, chance replace essential gene <br /> found HCS 9 15; exclude operon pairs,             functions functions greater species with <br /> may generally difficult compensatory functions           larger genome sizes generalist lifestyle. It be <br /> both genes, fraction increases 8 12. The probability       interesting test results study compare with <br /> finding HCS conserved consistently essential genes            additional studies bacteria having larger (or smaller) <br /> by chance 0.037 0.025, respectively (one-tailed Fisher&#226;&#8364;&#8482;s <br />                                                                              genomes. Indeed, bacteria small genomes, all <br /> exact test). The estimated odds ratios (with 95% upper limits in <br />                                                                              essential genes highly conserved; finding conditionally <br /> parentheses): 0.11 (0.89) 0.083 (0.75), respectively. These data <br />                                                                              essential genes prove far difficult. <br /> suggest suppressors conserved consistently essential genes <br />                                                                                 Overall, work provides novel explanation different <br /> are approximately fifth likely suppressors for <br />                                                                              patterns conservation observed essential genes, and <br /> genes conserved non-essential taxa. <br />                                                                              emphasizes gene essentiality fluid characteristic, over <br />    Thus, genes ancient, strongly conserved, consis- <br />                                                                              short periods evolutionary time. <br /> tently essential taxa appear persistently essential under <br /> laboratory conditions. <br />                                                                              Methods <br /> Conclusions                                                                  Strains Media <br />    The data support hypothesis cases, simply              All strains grown LB media (Sigma) LB agar plates <br /> increasing expression level specific non-essential genes          (1.5% agar, Sigma), L-arabinose D-glucose (both Sigma) <br /> render essential genes non-essential (i.e. high copy suppression).           supplemented indicated. E. coli strains MG1655 and <br /> Sequence structural comparisons showed                  DY330 described previously [42,43] grown 37uC and <br /> HCS genes homologous essential genes function              32uC, respectively, vigorous shaking. AB330 Lac+ <br /> they replaced. However, homologues tended distantly              derivative DY330, received Alex Boehm, <br /> related, divergence times ranging root           University Wurzburg, Germany. P1 transduction TSS <br /> bacteria, soon origin gamma-proteobacteria. In               transformation described [44,45]. Strains <br /> addition, majority cases, homology required            harboring pKD4 derived kanamycin resistance cassette were <br /> HCS occur, highlighting possibility is <br />                                                                              grown 50 mg/ml kanamycin sulfate (Sigma), strains with <br /> no detectable homology, elevated expression act a <br />                                                                              ASKA(-) plasmids 15 mg/ml chloramphenicol (Calbiochem). <br /> mechanism allowing functional replacement. For example, <br />                                                                              Ampicillin (Fluka) 25 mg/ml used select Para-phoA <br /> although essential gene &#226;&#8364;&#8220; HCS pairs spoT-mutT pyrH-cmk <br />                                                                              insertion attB. Strains transformed pCP20 [46] grown <br /> do share detectable similarity protein structure sequence <br />                                                                              32u presence 15 mg/ml chloramphenicol (Calbiochem). <br /> level, biochemical activity apparently similar to <br />                                                                              IPTG (isopropyl thiogalactopyranoside) Sigma. <br /> allow complete functional complementation. <br />    In instances, HCS recovered did allow <br /> the deletion corresponding essential gene. We hypothesize             Assembling Collection Conditional Lethal Mutant <br /> that combination low expression levels essential            Strains <br /> gene expression relevant HCS allowed suppression                  No comprehensive collection conditional lethal mutants of <br /> the lethal phenotype. However, cases, suppression          essential genes available. To construct collection, selected <br /> of conditional lethal phenotype based                  23 essential genes operon pairs E. coli exhibited <br /> specific mechanistic link, hypothesis supported           varying levels conservation bacterial taxa [15]. <br /> by finding molecular functions essential genes           Genes balanced functional categories, otherwise <br /> their dedicated high-copy suppressors far similar              random. This group essential genes covers nearly 10% the <br /> would expected chance.                                                 essential gene content E. coli MG1655. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              9                           June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                             Gene Compensability Influences Conservation <br />  <br />  <br />    Before selected essential genes experiment,                       Construction Plasmid Library <br /> discarded genes located operons coding essential                         We used ASKA(-) strain collection [50] construct a <br /> genes, insertion Para construct inside                  plasmid pool contained Escherichia coli open reading frames. <br /> operons strong polar effects. Three exceptions                     The ASKA(-) library consists 4123 clones, carrying a <br /> made: nrdAB, dnaTC glmUS essential tandem operons                          plasmid open reading frame. We pin-replicated clones <br /> whose gene products interact physically involved                     96-well plates containing LB medium (Sigma) 15 mg/ml <br /> same cellular processes. We assumed construct use                   chloramphenicol. Plates incubated 48 hours 37uC. <br /> repress transcription abolishes expression genes.                          Then, 20 ml pooled, clones containing <br />                                                                                    plasmids coded essential genes our <br /> Template Strain Construction                                                       experiment excluded. Plasmids extracted using a <br />    We used previously described strain TB55 [47] PCR                        plasmid preparation kit (Promega), following recommenda- <br /> template construction arabinose-inducible conditional lethal                tions manufacturer. <br /> mutants (analogous Roux et al. [48]), aim tightly linking <br /> a kanamycin marker arabinose-inducible promoter                      Transformation Conditional Lethal Mutants <br /> araBAD operon. This strain allows generation PCR product                     Each conditional lethal mutant grown LB medium with <br /> that contains outward facing kanamycin resistance marker                 0.1% arabinose (Sigma) OD600 nm 0.4 0.8. During the <br /> end, end outward facing arabinose-inducible                    preparation electrocompetent cells, density cultures <br /> promoter. Insertion construct essential genes              adjusted OD600 nm 1 guarantee equal number of <br /> fusion Para promoter transcriptional translational start sites        cells transformation event. Each clone electroporated with <br /> allows control expression selected essential genes [48]. We used             1 ml plasmid pool (DNA concentration approximately 330 ng/ <br /> TB55 generate PCR products flanked 40 42 <span id='am-474' about='xsp:base' typeof='owl:Thing'>base</span> pairs                       ml). Cells rescued 1 ml LB medium 15 mg/ml <br /> homology upstream region essential genes interest. The                chloramphenicol, 100 ml immediately removed and <br /> PCR product spanned kanamycin resistance gene, araC                    transferred 900 ml LB medium 0.1% arabinose and <br /> full intergenic region araC araB.                                      15 mg/ml chloramphenicol estimate transformation efficiency. <br />    Next, constructed TB741, strain allowed monitor                 To select high copy suppressors, cells spread LB agar <br /> expression Para second, independent arabinose-inducible <br />                                                                                    plates containing 0.4% glucose (to enhance repression Para), <br /> araBAD promoter. To end, combined phoA knockout <br />                                                                                    50 mM IPTG 15 mg/ml chloramphenicol, incubated at <br /> acquired KEIO clone JW0374 [1], and, removal the <br />                                                                                    37uC colonies appeared, maximally 3 days minimize the <br /> kanamycin resistance marker pCP20 [46], Para-phoA <br />                                                                                    formation colonies arise chromosomal <br /> construct inserted attB (derived E. coli strain SA22 <br />                                                                                    suppressor mutations. <br /> (a gift Prof. Winfried Boos, University Konstanz, <br /> Germany) P1 phage transduction. All strains used this <br /> study Table S3.                                                    Selection Verification High Copy Suppressor <br />                                                                                    Plasmids <br /> Primer Design Conditional Lethal Mutants                                          We recovered 10 colonies transformation event and <br />    All conditional lethal mutant strains constructed initially                restreaked plates glucose, IPTG, chloram- <br /> with primer design, included following:                         phenicol verify growth. After successful regrowth, clones were <br /> deletion 40 100 base pairs upstream region gene                grown liquid cultured overnight 0.1% arabinose and <br /> of insertion PCR product generated TB55,                   plasmid extracted (Promega). The purified plasmid was <br /> and fusion start codon gene start               retransformed [44] fresh ancestral conditional lethal mutant <br /> codon araB.                                                                     strains permissive conditions, 4 independent colonies of <br />    We able recover clones conditional lethal                    transformation event tested permissive and <br /> character yeaZ murA following methodology. Therefore                  restrictive plates growth. This procedure directly tested for <br /> we fused transcription initiation site araB predicted                suppression mediated plasmid: upon <br /> transcriptional start sites murA yeaZ (from www.regulondb.                  successful regrowth 4 clones, plasmids sequenced <br /> ccg.unam.mx), yielding conditional lethal clones. All oligonucle-                  using primer 59-GCGGATAACAATTTCACACAGA-39. <br /> otide sequences Tables S4 S5.                                  Cases clones did grow discarded from <br />                                                                                    analysis. The outcome retransformation test was <br /> Promoter Swap                                                                      based transformation method exhibits comparably <br />    As mentioned above, strain TB55 used generate PCR                        low efficiency [44], decreasing probability transforming two <br /> products contained kanamycin cassette adjacent araC,                 different plasmids cell, making unlikely more <br /> full Para-region 42 45 base pairs 59 39 -prime ends              plasmid responsible high copy suppression. <br /> that homologous upstream N-terminal region                     After verification procedure, went original <br /> essential gene interest. DY330 cells grown LB medium                    ASKA(-) library, purified plasmids recovered the <br /> supplemented 0.2% arabinose electro-                             screen (except yciR; gene contained clone the <br /> recombination competent described previously [49]. After                        indicated position collection), repeated procedure. <br /> electroporation, cells rescued LB medium containing                        This led exclusion high copy suppressors aspS, and <br /> 0.2% arabinose incubated 32u 1.5 hours prior plating                 verified suppressive plasmids screen. <br /> on arabinose- kanamycin - containing LB plates. Clones were <br /> checked LB plates supplemented 0.4% glucose confirm                     PhoA Assay <br /> their conditional lethal character. The constructs moved                    To determine expression high copy suppressor lead to <br /> by P1-transduction TB741, conditional lethality                       strongly increased expression Para promoter, assayed the <br /> assessed LB plates 0.4% glucose. All promoter fusions                activity Para-phoA fusion inserted lambda attachment <br /> as adjacent araC gene verified sequencing.                     site, using previously described procedure [51]. Briefly, we <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                   10                           June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                     Gene Compensability Influences Conservation <br />  <br />  <br /> transformed plasmids coding high copy suppressors (and            Assignment Orthology <br /> control plasmid pCA24N) TB741, grew clones                 We used assignments based previously published data [15]. <br /> overnight 50 mM IPTG, 0.4% glucose 15 mg/ml                       Briefly, used reciprocal shortest distance [52] potential <br /> chloramphenicol 96 plates, replicating clone                  orthologues relevant E. coli genes respective genomes. <br /> independently 16 times. To estimate maximum expression                 Two genes reciprocally closely related were <br /> level Para-phoA, induced 16 replicates TB741 harboring            denoted orthologues aligned 60% the <br /> pCA24N 0.1% arabinose. After overnight growth, cultures               longer gene. In cases orthologues found, used <br /> were spun down, resuspended phoA buffer (150 mM TrisHCl                 MicrobesOnline database search genes named as <br /> adjusted pH 9), diluted 1:2 fresh phoA buffer volume          putative orthologues. In way, additional putative <br /> of 180 ml, OD600 nm measured. One drop                         orthologues, ftsK S. pneumonia, second plsC S. <br /> (approximately 10 ml) 1% SDS solution (Sigma) 25 ml               aureus. <br /> of 10 mg/ml PNPP (4-nitrophenylphosphate, Sigma) solution                   We used data empirical studies essentiality [1,3&#226;&#8364;&#8220;11] <br /> was added. After incubation room temperature 24 hours           determine genes orthologous E. coli <br /> OD550 nm OD420 nm measured, PhoA activity                     essential bacterial taxa. <br /> determined using formula described [51]. <br />                                                                            Phylogenetic Analyses <br /> Knockout Essential Genes                                                   Orthologous homologous genes range bacterial <br />    To delete spoT, pyrH, fldA dapA, plasmids encoding mutT,            taxa selected aligned using Muscle v3.8.31 [53] with <br /> cmk, fldB nanA transformed AB330 expression              default parameters. The alignments cleaned using GBlocks <br /> was induced 50 mM IPTG (or 1 mM nanA). Knockouts               0.91b [54] <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> non-conserved positions set 32, the <br /> were achieved following previously described methods [43,49]               number flank conserved positions set minimum values, <br /> using pKD4-derived kanamycin cassette flanked homologous              minimum block length 2, allowed gaps set all. This <br /> ends. Successful deletions moved MG1655 (harboring               alignment used input MrBayes 3.1.2 [55] a <br /> ASKA(-) plasmids coding HCS) P1 transduction [45]            mixed amino acid model invariant plus gamma distributed <br /> addition IPTG verified PCR using primers upstream                rate variation sites. The chains run 200,000 <br /> and downstream insertion.                                           (dapA/nanA), 1,000,000 (fldA/fldB) generations, the <br />                                                                            20% run used construction majority <br /> Cross-Transformation Tests                                                 rule tree. <br />    To test specific interactions depletion of <br /> essential genes expression plasmid-based non-complement-            Assessment Functional Similarity <br /> ing high copy suppressors, HCS plasmids purified                 We obtained molecular function annotations GO <br /> ASKA(-) library transformed permissive conditions               database       (www.geneontology.org/GO.downloads.annotations. <br /> into conditional lethal mutant, regrowth checked           shtml; 5/20/2011) annotated E. coli genes. We also <br /> described initial screening procedure.                             obtained relationships GO categories (www. <br />                                                                            geneontology.org/GO.downloads.ontology.shtml; OBO v1.2). <br /> Dosage Response Measurements                                               GO annotations related tree-like manner, beginning with <br />    Gene deletion mutants grown overnight 37uC 96-well           broad, non-specific parent categories (e.g. &#226;&#8364;&#732;&#226;&#8364;&#732;binding&#226;&#8364;&#8482;&#226;&#8364;&#8482;), of <br /> plates shaking 400 rpm 8-fold replication, LB                specific child categories (e.g. &#226;&#8364;&#732;&#226;&#8364;&#732;acyl binding&#226;&#8364;&#8482;&#226;&#8364;&#8482;). <br /> medium supplemented 1 mM IPTG 15 mg/ml chlor-                     Thus, quantified functional distance number parent <br /> amphenicol. Cultures spun down, washed LB                     categories separate genes, normalized total <br /> medium, diluted 1:1024 fresh medium IPTG                     number parent categories gene. We calculated this <br /> concentrations indicated. Optical density 600 nm                 distance essential gene genes the <br /> measured 30 minutes, 7.5 hours total.                         genome, compared distance essential <br />                                                                            genes complimentary HCS. This yields number <br />                                                                            0 1, specifying fraction genes genome <br /> Differential Growth Conditional Lethal Mutant Strains                   functionally similar essential gene its <br /> and Plasmid-Complemented Mutants                                           HCS. If functional similarity does play role essential <br />    To analyze differential growth conditional lethal mutant             gene - HCS pairs, expect number 0.5, on <br /> strains (with suppressive plasmids, control plasmids,                average, distributed uniformly 0 1. Instead, we <br /> deletion essential genes ancestral conditional lethal                12 13 essential gene HCS pairs, distance <br /> mutants), grew corresponding clones 96-well plates               0.5 (i.e. similar average <br /> overnight. As control, conditional lethal mutant            pair genes); 8 13 pairs, distance than <br /> ancestral TB741 strain transformed plasmid             0.25. We used Kolmogorov-Smirnov test compare this <br /> pCA24N. Conditional lethal mutants grown 0.1% L-                 distribution distribution expected functional <br /> arabinose and, required, presence 15 mg/ml chloram-               relation essential gene HCS (the uniform <br /> phenicol select ASKA(-) plasmids. Essential gene deletion           distribution). <br /> mutants cultured 1 mM IPTG induce expression of <br /> high copy suppressors decrease likelihood genetic            Statistical Methods <br /> suppressor mutations. After overnight growth, cultures                  All statistical tests R v2.13.1 [56]. <br /> serially diluted repeatedly transferring 20 ml culture into <br /> 180 ml LB medium. Of dilution series, 5 ml indicated <br />                                                                            Supporting Information <br /> dilutions spotted plates supplemented arabinose, <br /> glucose, chloramphenicol IPTG indicated incubated               Figure S1 Expression Para measured phoA activity. Strain <br /> as indicated.                                                              TB741, phoA reporter fused Para promoter, was <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                            11                           June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                                                 Gene Compensability Influences Conservation <br />  <br />  <br /> transformed indicated plasmids grown overnight                                  dnaTC, presumably single gene capable of <br /> 50 mM IPTG 0.4% glucose 0.1% arabinose. None                                        replacing function genes. <br /> expressed HCS led phoA expression comparable induction                                      (DOC) <br /> with 0.1% arabinose, differences HCS                                    Table S2 PDB input chains pairwise structural alignments. <br /> apparent. Each strain replicated 16-fold; Error bars indicate                                 The PDB names essential genes complementing <br /> one standard error.                                                                               high copy suppressors indicated. <br /> (PDF)                                                                                             (DOC) <br /> Figure S2     Phylogenetic relationships homologous essential                                  Table S3 Strains used study. <br /> gene &#226;&#8364;&#8220; HCS pairs. A. fldA fldB duplicated origin                                 (DOC) <br /> gamma-proteobacteria; outside gamma-proteobacteria, the <br /> genes present single copy presumably                                        Table S4 Oligonucleotides used construct strains with <br /> orthologous fldA. B. dapA nanA anciently duplicated                                    conditional expression essential genes. <br /> genes, genes present bacterial clades                                     (DOC) <br /> diverged soon root bacteria (e.g. Bacilli                                    Table S5 Oligonucleotides used construct deletion strains. <br /> proteobacteria). Circles nodes trees indicate                                   (DOC) <br /> posterior probability support node; black: greater than <br /> 0.99; white: 0.5 0.9.                                                                 Acknowledgments <br /> (PDF) <br />                                                                                                   We thank Alex Boehm discussions comments. <br /> Table S1 Transformation efficiencies complementing <br /> plasmids. All genes fldA murA (bolded)                                            Author Contributions <br /> transformed greater 1.26104 high copy suppressor <br />                                                                                                   Conceived designed experiments: TB MA OKS. Performed the <br /> plasmids, number required cover 95% genes                                  experiments: TB. Analyzed data: TB MA OKS. Contributed reagents/ <br /> in E. coli. No complementing plasmids nrdAB                                    materials/analysis tools: TB OKS. Wrote paper: TB MA OKS. <br />  <br /> References <br />  1. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                     18. Cardinale CJ, Washburn RS, Tadigotla VR, Brown LM, Gottesman ME, et al. <br />     Escherichia coli K-12 in-frame, single-gene knockout mutants: Keio                         (2008) Termination factor Rho cofactors NusA NusG silence foreign <br />     collection. Molecular Systems Biology 2: 2006.0008.                                               DNA E-coli. Science 320: 935&#226;&#8364;&#8220;938. <br />  2. Kato JI, Hashimoto M (2007) Construction consecutive deletions                      19. Alba BM, Zhong HJ, Pelayo JC, Gross CA (2001) degS (hhoB) essential <br />     Escherichia coli chromosome. Molecular Systems Biology 3: 132.                                    Escherichia coli gene indispensable function provide sigma(E) <br />  3. Christen B, Abeliuk E, Collier JM, Kalogeraki VS, Passarelli B, et al. (2011) The                 activity. Molecular Microbiology 40: 1323&#226;&#8364;&#8220;1333. <br />     essential genome bacterium. Molecular Systems Biology 7.                                 20. Costa CS, Pettinari MJ, Mendez BS, Anton DN (2003) Null mutations the <br />  4. Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, et al. (2006)                        essential gene yrfF (mucM) lethal rcsB, yojN rcsC strains of <br />     Essential genes minimal bacterium. Proceedings National Academy                    Salmonella enterica serovar Typhimurium. Fems Microbiology Letters 222: 25&#226;&#8364;&#8220; <br />     Sciences United States America 103: 425&#226;&#8364;&#8220;430.                                            32. <br />  5. Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, et al. (2003)                     21. Peist R, Koch A, Bolek P, Sewitz S, Kolbus T, et al. (1997) Characterization of <br />     Essential Bacillus subtilis genes. Proceedings National Academy                         aes gene Escherichia coli encoding enzyme esterase activity. <br />     Sciences United States America 100: 4678&#226;&#8364;&#8220;4683.                                          Journal Bacteriology 179: 7679&#226;&#8364;&#8220;7686. <br />  6. Berardinis V, Vallenet D, Castelli V, Besnard M, Pinet A, et al. (2008) A                  22. Serres MH, Riley M (2000) MultiFun, multifunctional classification scheme for <br />     complete collection single-gene deletion mutants Acinetobacter baylyi                       Escherichia coli K-12 gene products. Microb Comp Genomics 5: 205&#226;&#8364;&#8220;222. <br />     ADP1. Molecular Systems Biology 4: 174.                                                       23. Saka K, Tadenuma M, Nakade S, Tanaka N, Sugawara H, et al. (2005) A <br />  7. Langridge GC, Phan MD, Turner DJ, Perkins TT, Parts L, et al. (2009)                              complete set Escherichia coli open reading frames mobile plasmids <br />     Simultaneous assay Salmonella Typhi gene using million                               facilitating genetic studies. DNA Research 12: 63&#226;&#8364;&#8220;68. <br />     transposon mutants. Genome Research 19: 2308&#226;&#8364;&#8220;2316.                                            24. Firth AE, Patrick WM (2005) Statistics protein library construction. <br />  8. van Opijnen T, Bodi KL, Camilli A (2009) Tn-seq: high-throughput parallel                         Bioinformatics 21: 3314&#226;&#8364;&#8220;3315. <br />     sequencing fitness genetic interaction studies microorganisms. Nature              25. Handford JI, Ize B, Buchanan G, Butland GP, Greenblatt J, et al. (2009) <br />     Methods 6: 767&#226;&#8364;&#8220;U721.                                                                              Conserved Network Proteins Essential Bacterial Viability. Journal of <br />  9. Gallagher LA, Ramage E, Jacobst MA, Kaul R, Brittnacher M, et al. (2007) A                        Bacteriology 191: 4732&#226;&#8364;&#8220;4749. <br />     comprehensive transposon mutant library Francisella novicida, bioweapon                  26. Touati D, Jacques M, Tardat B, Bouchard L, Despied S (1995) Lethal Oxidative <br />     surrogate. Proceedings National Academy Sciences United States                   Damage Mutagenesis Are Generated Iron Delta-Fur Mutants of <br />     America 104: 1009&#226;&#8364;&#8220;1014.                                                                        Escherichia-Coli - Protective Role Superoxide-Dismutase. Journal of <br /> 10. French CT, Lao P, Loraine AE, Matthews BT, Yu HL, et al. (2008) Large-scale                       Bacteriology 177: 2305&#226;&#8364;&#8220;2314. <br />     transposon mutagenesis Mycoplasma pulmonis. Molecular Microbiology 69:                     27. McCool JD, Long E, Petrosino JF, Sandler HA, Rosenberg SM, et al. (2004) <br />     67&#226;&#8364;&#8220;76.                                                                                            Measurement SOS expression individual Escherichia coli K-12 cells using <br /> 11. Chaudhuri RR, Allen AG, Owen PJ, Shalom G, Stone K, et al. (2009)                                 fluorescence microscopy. Molecular Microbiology 53: 1343&#226;&#8364;&#8220;1357. <br />     Comprehensive identification essential Staphylococcus aureus genes using                   28. Smith TF, Waterman MS (1981) Identification Common Molecular <br />     Transposon-Mediated Differential Hybridisation (TMDH). Bmc Genomics 10.                           Subsequences. Journal Molecular Biology 147: 195&#226;&#8364;&#8220;197. <br /> 12. Jordan IK, Rogozin IB, Wolf YI, Koonin EV (2002) Essential genes                     29. Ortiz AR, Strauss CEM, Olmea O (2002) MAMMOTH (Matching molecular <br />     evolutionarily conserved nonessential genes bacteria. Genome                          models obtained theory): An automated method model comparison. <br />     Research 12: 962&#226;&#8364;&#8220;968.                                                                             Protein Science 11: 2606&#226;&#8364;&#8220;2621. <br /> 13. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al. (2003)                        30. Bass S, Gu QM, Christen A (1996) Multicopy suppressors prc mutant <br />     Experimental determination level analysis essential genes                        Escherichia coli include HtrA (DegP) protease homologs (HhoAB), DksA, <br />     Escherichia coli MG1655. Journal Bacteriology 185: 5673&#226;&#8364;&#8220;5684.                                  truncated RlpA. Journal Bacteriology 178: 1154&#226;&#8364;&#8220;1161. <br /> 14. Rocha EPC, Danchin A (2004) An analysis determinants amino acids                        31. Barbosa JARG, Smith BJ, DeGori R, Ooi HC, Marcuccio SM, et al. (2000) <br />     substitution rates bacterial proteins. Molecular Biology Evolution 21: 108&#226;&#8364;&#8220;116.            Active site modulation N-acetylneuraminate lyase sub-family revealed <br /> 15. Silander OK, Ackermann M (2009) The constancy gene conservation                         structure inhibitor-complexed Haemophilus influenzae enzyme. <br />     divergent bacterial orders. BMC Research Notes 2: 2.                                              Journal Molecular Biology 303: 405&#226;&#8364;&#8220;421. <br /> 16. Koonin EV (2003) Comparative genomics, minimal gene-sets                         32. Joerger AC, Mayer S, Fersht AR (2003) Mimicking natural evolution vitro: <br />     universal common ancestor. Nature Reviews Microbiology 1: 127&#226;&#8364;&#8220;136.                                An N-acetylneuraminate lyase mutant increased dihydrodipicolinate <br /> 17. Geissler B, Elraheb D, Margolin W (2003) A gain-of-function mutation ftsA                      synthase activity. Proceedings National Academy Sciences United <br />     bypasses requirement essential cell division gene zipA Escherichia                 States America 100: 5694&#226;&#8364;&#8220;5699. <br />     coli. Proceedings National Academy Sciences United States                 33. Gaudu P, Weiss B (2000) Flavodoxin mutants Escherichia coli K-12. Journal <br />     America 100: 4197&#226;&#8364;&#8220;4202.                                                                           Bacteriology 182: 1788&#226;&#8364;&#8220;1793. <br />  <br />  <br />  <br />          PLoS Genetics | www.plosgenetics.org                                                12                                June 2012 | Volume 8 | Issue 6 | e1002803 <br />                                                                                                                               Gene Compensability Influences Conservation <br />  <br />  <br /> 34. Alba BM, Leeds JA, Onufryk C, Lu CZ, Gross CA (2002) DegS YaeL                          52. Wall DP, Fraser HB, Hirsh AE (2003) Detecting putative orthologs. <br />     participate sequentially cleavage RseA activate sigma(E)-dependent             Bioinformatics 19: 1710&#226;&#8364;&#8220;1711. <br />     extracytoplasmic stress response. Genes &amp; Development 16: 2156&#226;&#8364;&#8220;2168.                        53. Edgar RC (2004) MUSCLE: multiple sequence alignment high accuracy <br /> 35. DeLasPenas A, Connolly L, Gross CA (1997) The sigma(E)-mediated response                        high throughput. Nucleic Acids Research 32: 1792&#226;&#8364;&#8220;1797. <br />     extracytoplasmic stress Escherichia coli transduced RseA RseB,              54. Talavera G, Castresana J (2007) Improvement phylogenies removing <br />     negative regulators sigma(E). Molecular Microbiology 24: 373&#226;&#8364;&#8220;385.                        divergent ambiguously aligned blocks protein sequence alignments. <br /> 36. Yamanaka K, Ogura T, Koonin EV, Niki H, Hiraga S (1994) Multicopy                               Systematic Biology 56: 564&#226;&#8364;&#8220;577. <br />     Suppressors, Mssa Mssb, Smba Mutation Escherichia-Coli.                        55. Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference <br />     Molecular &amp; General Genetics 243: 9&#226;&#8364;&#8220;16.                                                         mixed models. Bioinformatics 19: 1572&#226;&#8364;&#8220;1574. <br /> 37. Xiao H, Kalman M, Ikehara K, Zemel S, Glaser G, et al. (1991) Residual                      56. R Development Core Team (2007) R: A Language Environment for <br />     Guanosine 39,59-Bispyrophosphate Synthetic Activity Rela <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>Null Mutants</span> Can                    Statistical Computing. <br />     Be Eliminated Spot Null Mutations. Journal Biological Chemistry 266:                  57. Weber H, Pesavento C, Possling A, Tischendorf G, Hengge R (2006) Cyclic-di- <br />     5980&#226;&#8364;&#8220;5990.                                                                                      GMP-mediated signalling sigma(S) network Escherichia coli. <br /> 38. Cashel M, Gentry DR, Hernandez VJ, Vinella D (1996) The stringent response.                     Molecular Microbiology 62: 1014&#226;&#8364;&#8220;1034. <br />     In: Neidhardt FC, Ingraham JL, Lin ECC, Low KB, Magasanik B et al., editors.                58. Douchin V, Bohn C, Bouloc P (2006) Down-regulation porins small <br />     Escherichia coli Salmonella: cellular molecular biology. 2 ed.                          RNA bypasses essentiality regulated intramembrane proteolysis <br />     Washington D.C.: American Society Microbiology.                                              protease RseP Escherichia coli. Journal Biological Chemistry 281: 12253&#226;&#8364;&#8220; <br /> 39. Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al. (2000) Gene                        12259. <br />     Ontology: tool unification biology. Nature Genetics 25: 25&#226;&#8364;&#8220;29.                   59. Cairrao F, Chora A, Zilhao R, Carpousis AJ, Arraiano CM (2001) RNase II <br /> 40. Campbell TL, Ederer CS, Allali-Hassani A, Brown ED (2007) Isolation                      levels change according growth conditions: characterization gmr, new <br />     rstA gene multicopy suppressor YjeE, essential ATPase unknown                     Escherichia coli gene involved modulation RNase II. Molecular <br />     function Escherichia coli. Journal Bacteriology 189: 3318&#226;&#8364;&#8220;3321.                           Microbiology 39: 1550&#226;&#8364;&#8220;1561. <br /> 41. Campbell TL, Brown ED (2008) Genetic interaction screens ordered                       60. Begg KJ, Dewar SJ, Donachie WD (1995) A New Escherichia-Coli Cell-Division <br />     overexpression deletion clone sets implicate Escherichia coli GTPase                    Gene, Ftsk. Journal Bacteriology 177: 6211&#226;&#8364;&#8220;6222. <br />     YjeQ late ribosome biogenesis. Journal Bacteriology 190: 2537&#226;&#8364;&#8220;2545.                   61. Kalamorz F, Reichenbach B, Marz W, Rak B, Gorke B (2007) Feedback control <br />                                                                                                     glucosamine-6-phosphate synthase GlmS expression depends small <br /> 42. Blattner FR, Plunkett G, Bloch CA, Perna NT, Burland V, et al. (1997) The <br />                                                                                   <br /> </body></html>