<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>.-. 1993 Oxford University Press                                     Nucleic Acids Research, 1993, Vol. 21, No. 23 5391 -5397 <br />  <br />  <br /> SnR31, snR32, snR33: novel, non-essential <br /> snRNAs Saccharomyces cerevisiae <br /> Andrey G.Balakin, Gregory S.Schneider, Max S.Corbett, Jingwei Ni Maurille J.Fournier* <br /> Department Biochemistry Molecular Biology Program Molecular Cellular Biology, <br /> Lederle Graduate Research Center, University Massachusetts, Amherst, MA 01002, USA <br />  <br /> Received August 9, 1993; Revised Accepted October 6, 1993                    GenBank accession nos L22433-L22435 (incl.) <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Genes novel yeast snRNAs                          viability, loss species alters flux of <br /> identified tested essentiality. Partial sequence              intermediates rRNA processing pathway leads a <br /> information developed RNA extracted                      cold-sensitive <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (12). Effects nearly identical those <br /> isolated nuclei respective gene sequences                     observed loss essential snRNAs been <br /> were discovered screening DNA sequence                           observed depletion nucleolar proteins, NOP1, G- <br /> database. The RNAs contain 222, 188 183                     ARI, NOP3, essential, non-essential <br /> nucleotides designated snR31, snR32                       protein NSR1 (13-17). GAR1, NOPI protein known as <br /> snR33, respectively. Each RNA derived single                SSB1 shown associated snoRNAs <br /> copy gene. The SNR31 gene adjacent gene                   (14,18,19). Taken together, results suggest processing <br /> an unnamed protein associated cap-binding                    rRNA, like mRNA, mediated assembly <br /> protein elF-4E. The SNR32 gene gene                  snRNA particles large &apos;processome&apos; complex (4). <br /> ribosomal protein L41 gene SNR33                       The pool uncharacterized snRNAs expected include <br /> chromosome Ill, open reading frames                  additional snRNAs participate ribosome biogenesis-at <br /> no known function. Genetic disruption analyses                        stage-but functions unrelated manufacture ribosome <br /> showed snRNAs required                  possible. The present report describes progress made <br /> growth. The new RNAs bring number non-                         ongoing effort characterize novel snRNAs yeast <br /> spliceosomal snRNAs characterized far                         Saccharomyces cerevisiae. Our approach clone new snRNA <br /> S.cerevisiae 14, essential.                genes determine genes essential growth. If <br />                                                                       phenoyWpe observed inactive gene, conditional alleles <br /> INTRODUCTION                                                          developed investigate role new RNA. <br />                                                                       The effectiveness strategy established the <br /> The yeast Saccharomyces cerevisiae contains upwards 30 small       spliceosomal snRNAs extended vital non- <br /> nuclear RNAs (1,2). Sixteen RNAs          spliceosomal snRNAs. The properties novel snRNAs <br /> partially characterized, including snRNAs involved        described communication. <br /> mRNA splicing 11 RNAs appear associated with <br /> the nucleolus (reviewed ref.3,4). The nucleolar small RNAs         MATERIALS AND METHODS <br /> in yeast organisms implicated ribosome <br /> biogenesis. Several species shown            Strains media <br /> important processing ribosomal RNA precursors, based           Yeast nuclear RNA prepared strain BJ2407 (a/ca, <br /> on: i) disruption rRNA processing cells extracts depleted    leu2/leu2, trpl/trpl, ura3-52/ura3-52, prbl-1122/prbl-1122, <br /> of specific snRNAs; ii) co-purification precursor rRNAs,         prc-407/prc-407, pep4-3/pep4-73, gal2/gal2; 20,21) genomic <br /> and; iii) cross-linking rRNA (reviewed refs 3-5). Only        DNA strain GRF167 (a, his3A, ura3-167; G.Fink, <br /> three 11 non-spliceosomal snRNAs characterized far        unpublished). Disrupted yeast SNR genes characterized in <br /> <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> S. cerevisiae essential growth, U3, U14 snR30        His-, Ura- strain MH2 (a/ca, ade2-101/ade2-101, <br /> (6-8). The genes dispensable, situation begs         trpl-A901/trpl-A901, ura3-52/ura3-52, leu2-3,112/leu2-3,112, <br /> explanation.                                                          his3/his3; M. Fitzgerald-Hayes, unpublished). Yeast strains were <br />    Each essential nucleolar snRNAs required       grown YPD liquid medium/plates (1% yeast extract, 2% <br /> normal processing rRNA. Loss impairs processing         peptone 2% glucose) selective plates SD medium <br /> of primary transcript leads shortage mature 18S       (2% glucose 0.67% yeast nitrogen <span id='am-166' about='xsp:base' typeof='owl:Thing'>base</span>) containing 20tg/ml <br /> rRNA (9-11). A fourth snRNA, snR1O, essential              histidine uracil 32&#194;&#176;C unless stated otherwise. Alternate <br />  <br /> *   <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed <br />  5392 Nucleic Acids Research, 1993, Vol. 21, No. 23 <br /> carbon sources included 2% galactose, 2% glycerol 2%              Northern hybridization characterization terminal <br /> potassium acetate.                                                   nucleotides <br />   Yeast transformed lithium acetate procedure (22).        Total yeast RNA isolated sodium dodecyl sulfate/hot <br /> Tetrad dissections carried essentially described      phenol method (32). Glass beads used break cells. <br /> Sherman Hicks (23). Diploid cells sporulated 1%          RNA separated 8% polyacrylamide gel, blotted onto <br /> potassium acetate, asci treated zymolyase (ICN             Gene Screen membranes UV cross-linked recommended <br /> Biochemicals, Inc.) spores separated YPD plates.          manufacturer. The membranes probed with <br />   Cloning manipulations carried E. coli DH5aF&apos;         radiolabeled DNA fragments solution containing 5 x SSPE <br /> (F&apos;/endAl, hsdR17(r-m+), supE44, thi-J, gyrA(Nalr), recAl,           buffer, 50% formamide, 0.2% SDS 50,ug/ml heparin (33). <br /> relAl, A(argF-lacZYA)U169 (480dlacZAM15) strain JM56              20xSSPE contained 3M NaCl, 0.2M Na2HPO4 20mM <br /> (F&apos;lacIQ-proAB, dam3,rpsL; M.J.Fournier, unpublished)                Na2.EDTA. <br /> essentially described Sambrook et al (24).                        A Northern blot anti-TMG immunoprecipitated RNAs from <br /> RNA isolation, 3&apos; end labeling enzymatic sequencing              S. cerevisiae kindly provided R.A.Lempicki. <br />                                                                         Primer extension RNA dideoxy sequencing carried <br /> Our nuclear RNA isolation protocol combination          essentially described (34). Three oligonucleotides were <br /> protocols. Spheroplasts prepared lyticase (Sigma; 25),     used primers: (5&apos;-TAGGAACTCATGGTGTAATT-3&apos;), <br /> lysed nuclei purified centrifugation       complementary snR31 position 8 position 27; <br /> 0.5M sucrose cushion (26). RNA extracted washed             (5&apos;-CTGATTCCCAGTGTCTCGTTTC-3&apos;), complementary to <br /> nuclei phenol 3&apos; end labeled variation method   snR32 positions 153-174; and, (5&apos;-GATTGTCCACACAC- <br /> developed Bruce Uhlenbeck (27). RNA incubated             TTC-3&apos;), complementary snR33 positions 167-183. <br /> with 50 pmols 5&apos;[32P]pCp 36 units RNA ligase 50 I1            RNA 3&apos; end sequences determined variation a <br /> of solution containing: 50 mM potassium HEPES, pH 7.5, 20            T4 RNA ligase/PCR-based approach described Kiss and <br /> mM MgCl2, 1 mM ATP, 10mM DTT, 10% glycerol, 10%                      Filipovicz (35). Instead oligoribonucleotide, an <br /> DMSO, 0.1 mg/ml BSA 40 units RNasin. Individual RNAs             oligodeoxiribonucleotide sequence (35) used for <br /> were purified rounds electrophoresis: 1 mm      ligation total yeast RNA. PCR products obtained with <br /> thick gel sequencing gel. Electrophoresis      oligonucleotides specific snR31, snR32 snR33 gel- <br /> the gel continued RNAs differing             purified sequenced directly snRNA-specific <br /> nucleotide resolved. 3&apos; end sequences pCp labeled RNAs       oligonucleotides using dsDNA Cycle Sequencing System <br /> were determined RNA sequencing kit (Pharmacia).               kit BRL Life Technolgies, Inc. Oligonucleotides were: <br /> Gene disruption                                                      (5 &apos;-GGAAGCCTTTCTCTTTCACC-3&apos;), corresponding to <br />                                                                      positions 161-170 snR31; (5&apos;-GATAGATTGAACGTTGC- <br /> DNA segments containing e new snRNA genes amplified             TGG-3&apos;), corresponding positions 111-130 snR32; and, <br /> by PCR using following primers:                                  (5 &apos;-GGTTTGAGTCGGTTCCTTCG-3&apos;), corresponding to <br /> SNR31 gene: 5&apos; side, GTGCGGAATTCGATATCGGGTGC-                        positions 124- 143 snR33. <br /> AAT 3&apos; side, GGTGCGATATCGAATTCAGAAAAGTG, <br /> both EcoRI sites                                                Computer sequence analysis <br /> SNR32 gene: 5&apos; side, GTGCGGAATTCCTCTAGAGAGACGT-                      Analyses RNA DNA sequences performed the <br /> CGTC 3&apos; side, CGTGGGAATTCCCGTGAACG-                              programs designed University Wisconsin Genetics <br /> AGCCGTACA, EcoRI sites                                     Computer Group (36). <br /> SNR33 gene: 5&apos; side, GCTTCCTCGAGCACACACTCTCTC- <br /> TTATT 3&apos; side, CAAGGGAATTCCTGATCTCTGGCG- <br /> AACGT, XoI EcoRI sites, respectively.                       RESULTS <br />   The PCR products cloned shuttle vector pRS314 <br /> (SNR32, SNR33; 28) E.coli vector pACYC 184 (SNR3J;            Isolation 3&apos; sequencing new snRNAs <br /> 29) using indicated restriction enzymes. Unique restriction sites    The snRNA genes described identified partial <br /> in SNR32 SNR33 coding regions used marker           sequencing purified RNA searching GenBank DNA <br /> gene insertions. An AseI site witiin pACYC184 sequence           sequence database related sequences. The RNA sequences <br /> eliminated AseI site middle SNR31      developed following manner. Nuclei purified <br /> used insert marker gene. BamHI fragments derived           centrifugation 0.5M sucrose cushion extracted <br /> plasmids YDp-U YDp-H (30) containing URA3 HIS3               spheroplast lysis buffer. The snRNAs resistant this <br /> genes used gene disruption analyses.                    treatment isolated phenol extraction. The resulting <br />                                                                      RNAs labeled 3&apos; ends [32P]-pCp and <br /> DNA isolation analysis                                           fractionated gel electrophoresis (Fig. 1). Several RNAs which <br /> The procedure used prepare yeast genomic DNA described         appeared amenable isolation purified rounds <br /> in Rose et al (31). Southern analyses carried according     electrophoresis. Nine novel snRNAs number species <br /> to Sambrook et al (SNR32 SNR33; 24) conditions          characterized identified enzymatic <br /> similar Northern hybridization protocol (SNR31).              sequencing. <br /> Radioactive DNA probes prepared nick translation <br /> (24) using New England Biolab Random Priming               Discovery gene sequences DNA database <br /> System I kit. A Southern blot EcoRI-digested genomic DNA          Homologous DNA sequences GenBank <br /> from different organisms (&apos;Zoo-Blot&apos;) obtained         database novel RNA sequences (Figs. 1 and <br /> Clontech Labs.,Inc.                                                  2). After work completed learned RNA <br />                                                                                        Nucleic Acids Research, 1993, Vol. 21, No. 23 5393 <br />                                                                                       SMR31 <br />                                                                                                10           20          30             40                   50 <br />  <br />                                                                                       GAATTCAGAA AAAATTACACC ATCAG                          &apos;ATTAACTCA 50 <br />                                                                                       GCCrCI TACTCATAAT G                      TlTTTG                            100 <br />  <br />                                                                                       CACTGCGTC           TOOT AQATTAATTA TTA              C    TATCTCCAG 150 <br />  <br />                                                                                       CQTAT TAGGGG           AGC7IT=          TCA                              200 <br />  <br />                          snR3                                                         CACCTGATAC AT          T GA            ACArI-iTAAT TTCTTTATGT              250 <br />                          snR5 ._ <br />                        snRI89,                                                        AACCACGTTG TATATTTTCA ATCTGGAAAG TAACCTAAAA GAGTAAAAAC 300 <br />                        snR190 <br />                        (l90nt)                snR32                                   TGTTTATTAA AAAAAATGTT ATTCAAAATT TATACATAGT ACACGTACAG 350 <br />                          snR9    - <br />  <br />                                                                                       CTGCCTGAAG AAAGCATGAG CGAATGACTT ATGCTTCGTC GTCTTCGTCT 400 <br />                                               snR33 <br />  <br />  <br />                                                                                       SNR32 <br />                                      .I&quot;&apos;                                                      10           20          30            40               50 <br />  <br />                                                                                       TTCGTGCTAG CCCTACAACG GAACCCAAAA AATCAATCAC CAGGGTAAGC                      50 <br />  <br />                                                                                       GGGAAAAAAA AAAAGTGTAG GAACCGCACT ATAAAAAGTC GAAGGTTTGT                     100 <br />  <br />                                                                                       ATTAATTTTC CACACCTCTT TAGTTTTGTT ATAATCAAAT TCCTTTTGGT                     150 <br />  <br />                                                                                       TTTTGTTTTA AAAAGGCCAA CATCATAATT ATATCGCGAA TGTACTACTA                     200 <br />  <br />                                                                                       GTATATGCAG    TTTTC&apos;1AAA ATTGCACA ATGACTAAAA GCCCACGTTT                    250 <br />  <br />                                                                                       TTCCCACTTT TAGTAGTACT GAAGCGTAGA       TAGATT AAC GTTGCTGGGC               300 <br />  <br /> Figure 1. Fractionation nuclear small RNAs. Small RNA    prepared         GCCCTGGTGTT   GATCATTTCT GAAATaAGAT ATTGGGAATC        AGCTGTATCT           350 <br /> isolated nuclei pre-extracted 25mM K HEPES pH7.6, 5mM magnesium                  A2CaTrATT     TATACATATT TATTTTTACT GGTTCTTAGT AACGTACTTT                  400 <br /> acetate, 0.25 mM sucrose 0.1 % NP-40. SnRNAs resistant treatment <br /> were labeled 3&apos; end [32p] pCp fractionated electrophoresis         AGTAGCAATG CTGTGTTTTA GTCGAACGCC TACGTGTCTT GTTAACTTAT                     450 <br /> in 8% polyacrylamide gel. The <span id='am-167' about='xsp:pattern' typeof='owl:Thing'>pattern</span> shown snRNAs size range         CATGGTTGCC AACGCGAACG TTTACAACAG GTTCAAAGGC AGGATGAAAA                     500 <br /> of 158-222 nucleotides. The bands corresponding snRNA species snR31, snR32 <br /> and snR33 purified second cycle electrophoresis subjected to <br /> direct sequence analysis. The identity known RNAs marked left <br /> was   established direct sequencing.                                              SMR33 <br />                                                                                               10            20          30             40               50 <br />  <br />                                                                                      GCTTGCGCAT CTTTTTTTAC TTTATATACT AGTTTTTTTT TTTTTTTTCT                            50 <br /> estimated contain approximately 220 nucleotides corresponds                       TTTTCCCAAA TTTTTTCATG AAAAATTTGG CGGAACGGTA CATAAGAATA                       100 <br /> to species named snR31 (3 D.Tollervey, pers. <br />                                                                                      GAAGAGATTC ATTATGAAAA TTTTCTGCTC TCTTCCACAT TTTTTTTTCA <br /> commun.). To avoid confusion elected use                                                                                                      150 <br />  <br /> identifying RNA, adopt consecutively                            TAAGAATTAA AAAAAATAAC CTTAATCCCC TCTTTCTACC ATGCTCTCAC                       200 <br /> numbered designators RNAs described here,                                                                                                       250 <br /> i.e., snR32 snR33. This nomenclature also <br />                                                                                      GGTTCCTTCG ATCAGTGATT TGGAAAG aT CAAACT                   GGGACTC            300 <br /> be used new snRNAs study. <br />    Each RNA coding sequences database                              ,GTCTGTTTTCT&apos;G            TGGAATTGAT ATAGAAuS           GVGGj2Ac&quot;A         350 <br /> occurs genomic segments sequenced                              GATTTGCAAA ACTTTTTATT TTATGTATTA CTACTGAGCT TGTTTATCTG                       400 <br /> unrelated reasons. DNA corresponding snR31 segment                        TATACTTTTT TCTTATTATT ATGTAGTTAG AGGTTAGTTC TTATCGTTTT                       450 <br /> (<span id='am-4' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> X15731) specifies unnamed <br /> 20 kDa protein believed associated translation factor                 CTTTAAAAGC GACCTTTCTT CGCACAAAGC AATGTTGACA ATCGAGCCAA                       500 <br />  <br /> eIF-4E (37; Figs.2A 3). A DNA segment (GenBank <br /> accession number M62392) corresponding snR32 adjacent <br /> to sequenced genes ribosomal protein L41 (38;                    Figure 2. Sequences gene regions encoding snR31, snR32 snR33. <br />                                                                                 The snRNA genes identified searching GenBank DNA sequence <br /> Figs.2B 4). The snR31 snR32 coding sequence                         database related sequences. The snRNA sequences highlighted italics <br /> located 99 bp 268 bp respectively downstream protein                 underlines. The nearest candidate TATA elements marked overhead <br /> genes, tail-to-tail orientation.                                           asterisks (*). The 5&apos; 3&apos; ends coding sequences determined by <br />    The segment encoding snR33 completely                       primer extension T4 RNA ligase/PCR-based sequencing strategy (see <br />                                                                                 Materials Methods). <br /> sequenced chromosome mII (<span id='am-5' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> X59720), <br /> between nucleotide positions 141,410 142,838 (39; Figs.2C <br /> and 5). The snR33 sequence occurs uncharacterized <br /> open reading frames located 1200 200 bp up- and <br />                                                                                 Sequence termini new snRNAs <br />  downstream, respectively; ORF elements snR33                       The RNA sequences initially deduced from: 3&apos; end <br />  coding sequence oriented direction. A gene                 partial sequences, sizes RNAs (estimated <br />  glycine tRNA (<span id='am-8' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> K02654) solo                    electrophoretically), sequences DNA database. <br />  delta element occur upstream, approximately 160 310                   The actual boundaries coding sequences (Fig.2) later <br />  nucleotides start snR33 coding sequence.                       determined precise mapping 5&apos; 3&apos; ends RNAs. <br />  5394 Nucleic Acids Research, 1993, Vol. 21, No. 23 <br />  <br />         A-\ <br />  <br />  <br />  <br />  <br />        K.B <br />                   L. .i   U J..:.- ......- <br />  <br />  <br />  <br />  <br /> Flgure 3. Coning disruption SNR3J   gene region. A: A 1.3 kb fragment <br /> of genomic DNA PCR amplified, cloned disrupted URA3 gene. <br /> The snR31 coding sequence located 99 bp downstream gene 20 kDa <br /> protein believed associated m7G cap binding protein eIF-4E. The <br /> UR43 gene fragment inserted AseI site midst SNR31 gene. <br /> B: Southern analysis DNA URA+ transformants. DNA digested with <br /> EcoRI Clal probed 847 bp EcoRl-ClaI fragment radiolabeled <br /> by nick-translation. Lane 1, DNA homozygous wild type cells; lanes 2-7, <br /> DNA heterozygotes contaning wild type null alleles SANR3. The <br /> wild type DNA fragment 860 bp disrupted gene fragment 1.9 kb. <br />                                                                                                          .~         *:$j~~~~~&quot;ii i. <br />  <br />  <br />  <br />  <br />                                                                                    derived wild type allele. <br />                                                                                                                           1i: <br />                                                                                                                          :;     s:: <br />                                                                                                                                       ..AMM <br />                                                                                                                                               .         iE-- <br />  <br />  <br />                                                                                                                                                   !J w. <br />                                                                                                                                                          _ S//= <br />  <br />  <br />                                                                                                                                                             ~~~~f*&apos; *I F:3 s <br />  <br />  <br />  <br />  <br />                                                                                    Fgure 4. Cloning disruption analysis SNR32 gene locus. A: The snR32 <br />                                                                                    coding segment located 268 bp downstream gene ribosomal protein <br />                                                                                    L4lb (SaC4Jb), opposite orientation. A 1.8 kb DNA fragmnent containing <br />                                                                                    genes cloned PCR amplification SNR32 gene disrupted <br />                                                                                    HIS3 gene fragment inserted BclJ site. The cloned region extends <br />                                                                                    nucleotide -378 upstream snRNA segment position 1253 bp <br />                                                                                    snR32 coding sequence; site 211 bp upstream SCL41b. <br />                                                                                    The structures wild type snrl85::HIS3 alleles shown. B: Southern <br />                                                                                    blot analysis DNA HIS+ transformants wild type cells. Genomic <br />                                                                                    DNA digested Hinda probed 525 bp EcoRI-BcI fragment <br />                                                                                    radiolabeled random priming. Lanes 1,3,5 6, DNAs heterozygous <br />                                                                                    isolates; lanes 2,4 7, DNA cells homozygous SNR32 (wild type). <br />                                                                                    The allele disrupted HIS3 yields fragment 900bp smaller fragment <br />  <br /> The snR32 snR33 sequences verified reverse <br /> transcriptase dideoxy sequencing 5&apos; ends established <br /> at time. The 5&apos; end snR31 mapped primer                         AUl snRNAs encoded single copy, non-essential <br /> extension analysis only.                                                           genes <br />    The 3&apos; terminal nucleotides identified clever                         The availability genomic sequence information it <br /> approach described Kiss Filipovicz (35), applied                       possible clone gene PCR amplification strategy <br /> some modifications. In original procedure RNA                               (Figs.3 -5). Cloned DNAs used prepare hybridization <br /> oligonucleotide ligated 3&apos; end RNA cDNA                 probes construct null alleles test essentiality the <br /> synthesis primed DNA oligonucleotide complementary                       genes. The gene copy number determined Southern <br /> to segment. The cDNA amplified PCR using                         analysis genomic DNA digested variety restriction <br /> second DNA oligonucleotide corresponding internal RNA                        enZymes. The data showed snRNA encoded single <br /> sequence. The PCR product subcloned plasmid                     copy gene. <br /> sequenced. We determined readily available DNA                              The essentiality snRNA genes tested replacing <br /> oligonucleotides used initial ligation RNA                  wild type SNR genes alleles disrupted marker genes. <br /> that sequence joint RNA-DNA molecule                             URA3 HIS3 simply inserted midst SNR31 and <br /> successfully amplified PCR. We omitted subcloning                       SNR32 genes respectively (Figs.3A, 4A). A <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>null allele</span> SNR33 <br /> sequenced PCR products directly gel-purification,                     prepared replacing 3&apos; half snRNA coding <br /> the 3&apos; ends RNAs determined                            sequence UR43 (Fig.5A). The disrupted alleles were <br /> homogeneous enzymatic sequencing.                                             transformed Ura- His- test strains transformants <br />    SnR31, snR32 snR33 determined 5&apos;                             heterozygous SNR loci identified growth <br /> trimethylguanosine cap (TMG), based hybridization probing                       phenotype Southern blot analysis (Figs.3B, 4B, SB). <br /> of electrophoretically fractionated snRNA prepared                                Tetrad analysis showed genes snR31, snR32 snR33 <br /> precipitation TMG antibodies (data shown).                                dispensable. Northern assays carried cells from <br />                                                                                                                          Nucleic Acids Research, 1993, Vol. 21, No. 23 5395 <br />                                                                                                                 A.                               B.                   C. <br />       A.                    ORI <br />                                                        1.2 ki, <br />                                                                          SNR33                     ORF <br />                                                                                                                 + + - - LRA                      - + - + HIS1          -        -   + URA 3 <br />                             87l1 t,                                              2(X)p        K,       ip <br />  <br />  <br />                                                                      SNR33       ~        ~            N <br />                      -I.l                      liha          E-co RV Cla N RsiI)KF <br />                                                                                                                       * ..   .....&quot;, <br />                                                                                                                                        &quot;                        -nR32__                4&lt;- snR33 <br />  <br />               Xhl)                                 kuRI <br />                                                      Rl <br />                                                                                                                               :. _         R3S                                         -   5.8s <br />  <br />                                                                  I           \ <br />                                                                                                                                                                                          -I <br />                      Sal              Illh                 |N            \             si                                                                                            -&gt;71 <br />  <br />                                                                      I <br />                         I <br />                                           1 2 kb                           9011hbp <br />                                                                                                                   1    2 3 4                      1   2   3 4           1   2   3 4 <br />  <br />  <br />        B.                                                                                                       Figure 6. Northern analysis cells containing disrupted SAR genes.Haploid cells <br />                                                                                                                 containing single disrupted alleles snR31, snR32 snR33 analyzed <br />                                                                                                                 Northern blotting (panels A-C). Signals corresponding test RNAs were <br />                                                                                                                 absent case. Each series includes RNA isolated spores of <br />                                                                                                                 single tetrad (lanes 1-4). (-) (+) corresponds spores containing wild <br />                                                                                                                 type alleles alleles disrupted marker genes (URA3 HIS3). A 844 bp <br />                                                                                                                 Hindlll-EcoRl fragment (see Fig.4) labeled random priming used as <br />                              -W.r                                                                               probe snR32. Probes snR31 snR33 identified Figs.3 5. <br />                                                                                                                 The positions 5S 5.8S RNAs marked. <br />  <br />  <br />                                                                                                                    No impressive relationship sequenced snRNAs was <br />                                                                                                                 found, The search snRNA-conserved elements also <br />                                                                                                                 negative, recognizable element species. <br />                                                                                                                 Conservation snR32 snR33 sequences also <br />                  1                    2            3             4               5                 6        7   examined experimentally Southern analysis. No signals were <br />                                                                                                                 detected genomic DNAs human, monkey, rat, mouse, <br />                                                                                                                 dog, cow, rabbit chicken (data shown). <br /> Figure 5. Cloning disruption analysis SNR33 gene region. A: A 1.4                                       As expected, segments complementarity exist between <br /> kb DNA fragment containing snR33 gene cloned PCR amplification                                       snRNAs pre-rRNA, large size of <br /> and 115 bp EcoRV-ClaI gene replaced URA3 marker <br /> gene. The structures wild type disrupted alleles (snr33::URA3)                                   rDNA coding <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. While relevant the <br /> shown. The cloned region extends nucleotides 141,410 142,838                                         largest regions described highest <br /> chromosome Em. B: DNA URA+ transformants screened Southern                                          potential direct binding; course, important interactions <br /> blot analysis, probing NsIl/SalI digests 500 bp NslI-HhaI fragment                                  occur pairing short regions. The longest <br /> radiolabeled random priming. Lanes 1 3, DNA homozygous wild <br /> type strain plasmid containing intact SNR33 (1kb fragment); lane 2, plasmid                                 complementary regions rRNA were: 1) 12 base segment <br /> with disrupted allele (0.9 kb 1.2 kb fragments); lanes 4,6, 7, genomic                                5&apos; external transcribed sequence, nucleotides <br /> DNA transformants heterozygous wild type disrupted alleles;                                    567-578 5&apos; end pre-rRNA (snR32); 2) 10 base <br /> lane 5, DNA heterozygote containing disrupted allele URA3                                   segment internal transcribed sequence 1, nucleotides <br /> gene inserted opposite orientation.                                                                      2447-2456 (snR33); and, 3) 19 base segment region that <br />                                                                                                                 includes enhancer element termination signal T2, at <br /> individual spores confirmed absence test RNAs                                                     nucleotides 6843-6861 (snR31; 40). Although these <br /> haploid cells containing disrupted allele (Fig.6). No                                                       snRNA-related segments imagined role rRNA <br /> phenotypic difference evident haploid cells 16&#194;&#176;C,                                                synthesis, clear complementarities are <br /> 25&#194;&#176;C, 32&#194;&#176;C 38&#194;&#176;C. Cells lacking snR33 exhibited similar                                                       statistically significant. A similar search using spliceosomal <br /> growth rates media containing 2% glucose, 2% galactose,                                                      snRNA sequences controls yielded complementarities upwards <br /> 2% glycerol 2% potassium acetate; effect carbon source                                                10 contiguous bases. Many test control <br /> was tested cells lacking snRNAs. The fact                                                 complementarities involve stretches adenines uridines and <br /> that RNAs dispensable precludes possibility                                                  high AT content yeast genome argues these <br /> of using conditional mutants study function                                                 relationships important statistically. <br /> for essential snRNAs. <br /> Relatedness snR31, snR32 snR33 RNAs                                                             DISCUSSION <br /> Computer searches carried RNA sequence                                                       The genes identified bring number S. cerevisiae <br /> to determine: i) snRNAs evolutionarily conserved; ii)                                                snRNA genes analyzed date 19, corresponding 5 <br /> if sequence motifs conserved snRNAs present,                                                  spliceosomal 14 non-spliceosomal RNAs (reviewed 3,4). <br /> as box A-D elements nucleolar snRNAs; and,                                                       The novel snRNAs described extraction properties like <br /> iii) new RNAs contain segments complementarity                                                   known nucleolar snRNAs (7,12, data shown), <br /> rRNA suggest roles rRNA synthesis.                                                                however, premature conclude RNAs actually <br />  5396 Nucleic Acids Research, 1993, Vol. 21, No. 23 <br /> nucleolar. Resistance extraction modest salt concentration        nucleolar well, number dispensable small RNAs <br /> is believed property nucleolar snRNAs (12),        associated complex increases 11, corresponding to <br /> behavior apply RNAs associated stable          80% known nucleolar small RNAs. <br /> complexes. Consistent observed precursor           As number non-essential species expands seems <br /> tRNAs resistant extraction low salt (A.Balakin         increasingly unlikely RNAs simply redundant, with <br /> and M.Fournier, prep.). The snRNAs identified          distinguishing function. More plausible possibility that <br /> be nucleolar, direct evidence needed        RNAs provide overlapping unique advantages that <br /> be considered established. Other criteria include:                essential. Dispensable snRNAs could, example, <br /> localization nucleolus hybridization probes, co-            influence kinetics accuracy rRNA processing rate <br /> purification cross-linking precursor rRNA,                  assembly ribosomal RNP intermediates little no <br /> association known nucleolar proteins.                              effect growth rate. Unless ribosome content limiting <br />    The new RNAs lack box C box D elements characteristic         growth phenotypes observed loss non- <br /> of nucleolar RNAs yeast sources (41 -44),        essential snRNAs. The contributions RNAs only <br /> but does argue nucleolar location. A number          apparent quantitative biochemical assays discovery <br /> of snRNAs exhibit nucleolar localization properties,          genetic conditions deleterious effects snRNA loss <br /> lack box C D. These include yeast snRNA species snR3,           amplified. <br /> -4, -5, -8, -9 -10 human HeLa snRNAs U17 (El), E2, <br /> and E3 (12,34,45). The yeast snR31, snR32 snR33 species             ACKNOWLEDGEMENTS <br /> could belong group snRNAs. <br />     Several snRNAs participating processing mRNA rRNA          We thank Richard Lempicki providing blot snRNAs <br /> are believed function complementary interactions        immuno-precipitated trimethylguanosine antibodies, Molly <br />  substrate molecules (4,46). The yeast nucleolar         Fitzgerald-Hayes yeast strain MH2 David Tollervey for <br />  snRNAs U3 U14, example, contain rRNA-related                   sharing snR31 gene sequence data prior publication. This work <br />  sequences implicated rRNA processing                supported NIH grant GM-19351. <br /> experimentally. Yeast U3 cross-linked rRNA at <br /> complementary sites (47) mutational analysis shown         REFERENCES <br />  U14 activity depends segments complementary               1. Riedel, N., Wise, J. A., Swerdlow, H., Mak, A. Guthrie, C. (1986) <br />  18S rRNA (48). At point link snRNA            Proc. Natl. Acad. Sci. USA 83, 8097-8101. <br />  species described rRNA synthesis tenuous                2. Thompson, J.R., Zagorski, J., Woolford, J.L. Fournier, M.J. (1988) <br />  nucleolar localization data complementarity               Nucleic Acids Res. 16, 5587-5601. <br />  regions ribosomal DNA. It remains seen           3. Mattaj, I.W., Tollervey, D. Seraphin, B. (1993) FASEB J. 7, 47-53. <br />  new snRNAs role ribosome biogenesis.                         4. Fournier, M.J. Maxwell, E.S. (1993) Trends Biochem. Sci. 18, 131-135. <br />                                                                          5. Sollher-Webb, B., Tyc, K. Steitz, J. (1993) Ribosomal RNA: Stnuture, <br />     The finding the novel snRNAs derived           Ewvuoion, Gene Etpression Function Protein Synthesis (Zimmermann, <br />  single gene surprising. All S. cerevisiae         R.A. Dahlberg, A.E., eds), CRC Press (in press). <br />  snRNAs characterized far encoded single-copy DNA            6. Hughes, J.M.X., Konings, D.A.M. Cesareni, G. (1987) EMBO J. 6, <br />  (49). The exception U3 specified genes,            2145-2155. <br />  containing introns (6,50). No snRNA genes                    7. Zagorski, J., Tollervey, D. Foumier, M.J. (1988) Mol. Cell. Biol, 8, <br />                                                                             3282-3290. <br />  S.cerevisiae known contain introns, including          8. Bally, M., Hughes, J. Cesareni, G. (1988) Nucleic Acids Res. 16, <br />  described present study.                                            5291 -5303. <br />     The proximity new snRNA genes protein genes              9. Hughes, J.M.X. Ares, Jr., M. (1991) EMBO J. 10, 4231-4239. <br />  ORF elements locus raises possibility          10. Li, H.V., Zagorski, J. Fournier, M. (1990) Mol. Cell. Biol. 10, <br />                                                                             1145-1152. <br />  functional relationship RNA protein genes. The         11. Morrissey, J.P. Tollervey, D. (1993) Mol. Cell. Biol. 13, 2469-2477. <br />  involvement snRNAs ribosome biogenesis makes linkage         12. Tollervey, D. (1987) EMBO J. 6, 4169-4175. <br />  snRNA genes genes putative mRNA cap              13. Tollervey, D., Lehtonen, H., Carmo-Fonseca, M. Hurt, E.C. (1991) <br />  binding protein (snR3 1; 37) ribosomal protein (snR32; 38)             EMBO J. 10, 573-583. <br />  especially intriguing. A similar situation exists yeast   14. Girard, J.-P., Lehtonen, H., Caizergues-Ferrer, M., Amalric, F., Tollervey, <br />                                                                             D. Lapeyre, B. (1992) EMBO J. 11, 673-682. <br />  snRNA species snR189 located 300 bp gene           15. Russell, I.D. Tollervey, D. (1992) J. Cell Biol. 119, 737-747. <br />  ribosomal protein rp59 (CRYJ; 2). The SNR31 SNR32 genes            16. Lee, W-C., Zabetakis, D. Melese, T. (1992) Mol. Cell. Biol. 12, <br />  tail-to-tail arrangement protein genes,          3865-3871. <br />  SNR189 CRYI tail-to-head arrangement. The                 17. Kondo, K., Kowalski, L.R. Inouye, M. (1992) J. Biol. Chem. 267, <br />  closeness genes suggests possible regulatory link                16259-16265. <br />                                                                         18. Schimnang, T., Tollervey, D., Kern, H., Frank, R., Hurt, E.C. (1989) <br />  ribosome biogenesis translation. This notion                EMBO J. 8, 4015-4024. <br />  strengthened recent discoveries showing small          19. Clark, M.W., Yip, M.L., Campbell, J. Abelson, J. (1990) J. Cell Biol. <br />  nucleolar RNAs higher eukaryotes encoded introns             111, 1745-1751. <br />  ribosomal protein genes genes heat shock protein          20. Jones, E.W. (1977) Genetics 85, 23-33. <br />                                                                          <br /> </body></html>