<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Cross-Species Array Comparative Genomic Hybridization <br /> Identifies Novel Oncogenic Events Zebrafish and <br /> Human Embryonal Rhabdomyosarcoma <br /> Eleanor Y. Chen1,2,3,4., Kimberly P. Dobrinski4,5,6., Kim H. Brown4,5,7., Ryan Clagg1,2,3, Elena Edelman3, <br /> Myron S. Ignatius1,2,3,4, Jin Yun Helen Chen4,5, Jillian Brockmann1, G. Petur Nielsen8, <br /> Sridhar Ramaswamy3,4, Charles Keller9, Charles Lee4,5, David M. Langenau1,2,3,4* <br /> 1 Division Molecular Pathology, Massachusetts General Hospital, Charlestown, Massachusetts, United States America, 2 Harvard Stem Cell Institute, Boston, <br /> Massachusetts, United States America, 3 Cancer Center, Department Medicine, Massachusetts General Hospital, Boston, Massachusetts, United States America, <br /> 4 Harvard Medical School, Boston, Massachusetts, United States America, 5 Department Pathology, Brigham Women&#226;&#8364;&#8482;s Hospital, Boston, Massachusetts, United <br /> States America, 6 Department Pathology Cell Biology, College Medicine, University Southern Florida, Tampa, Florida, United States America, <br /> 7 Department Biology, Portland State University, Portland, Oregon, United States America, 8 Department Pathology, Massachusetts General Hospital, Boston, <br /> Massachusetts, United States America, 9 Pediatric Cancer Biology Program, Department Pediatrics, Oregon Health Science University, Portland, Oregon, United <br /> States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Human cancer genomes highly complex, making challenging identify specific drivers cancer growth, <br />      progression, tumor maintenance. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> bypass obstacle, applied array comparative genomic hybridization <br />      (array CGH) zebrafish embryonal rhabdomyosaroma (ERMS) utilized cross-species comparison rapidly identify <br />      genomic copy number aberrations novel candidate oncogenes human disease. Zebrafish ERMS contain small, focal <br />      regions low-copy amplification. These regions commonly amplified human disease. For example, 16 19 <br />      chromosomal gains identified zebrafish ERMS exhibited focal, low-copy gains human disease. Genes in <br />      amplified genomic regions assessed functional roles promoting continued tumor growth human and <br />      zebrafish ERMS &#226;&#8364;&#8220; identifying critical genes associated tumor maintenance. Knockdown studies identified important <br />      roles Cyclin D2 (CCND2), Homeobox Protein C6 (HOXC6) PlexinA1 (PLXNA1) human ERMS cell proliferation. <br />      PLXNA1 knockdown enhanced differentiation, reduced migration, altered anchorage-independent growth. By <br />      contrast, chemical inhibition vascular endothelial growth factor (VEGF) signaling reduced angiogenesis tumor size <br />      ERMS-bearing zebrafish. Importantly, VEGFA expression correlated poor clinical outcome patients ERMS, <br />      implicating inhibitors VEGF pathway promising therapy improving patient <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span>. Our results demonstrate <br />      utility array CGH cross-species comparisons identify candidate oncogenes essential pathogenesis of <br />      human cancer. <br />  <br />   Citation: Chen EY, Dobrinski KP, Brown KH, Clagg R, Edelman E, et al. (2013) Cross-Species Array Comparative Genomic Hybridization Identifies Novel Oncogenic <br />   Events Zebrafish Human Embryonal Rhabdomyosarcoma. PLoS Genet 9(8): e1003727. doi:10.1371/journal.pgen.1003727 <br />   Editor: Gregory S. Barsh, Stanford University School Medicine, United States America <br />   Received April 5, 2013; Accepted July 1, 2013; Published August 29, 2013 <br />   Copyright: &#195;&#376; 2013 Chen et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: DML supported NIH grants K01 AR055619, RO1 CA154923, R21 CA156056, Alex&#226;&#8364;&#8482;s Lemonade Stand Foundation (ALSF), American <br />   Cancer Society, Harvard Stem Cell Institute. EYC supported NIH K08 AR063165-01 St. Baldrick&#226;&#8364;&#8482;s Foundation Scholar Award. KHB supported <br />   Grant K99/R00 ES018892 (NIH&#226;&#8364;&#8220;National Institute Environmental Health Sciences). MSI supported &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482; ALSF ward. CL supported Grants <br />   5R01CA111560 (NIH&#226;&#8364;&#8220;National Cancer Institute) 5P41HG004421 (NIH&#226;&#8364;&#8220;National Human Genome Research Institute). The funders role study design, <br />   data collection analysis, decision publish, preparation manuscript <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: dlangenau@partners.org <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                         growth tumor maintenance. Cytogenetic studies, including <br />                                                                                      array Comparative Genomic Hybridiation (array CGH), identify <br />    Rhabdomyosaroma (RMS) common soft tissue                              frequent inconsistent gains losses partial <br /> sarcoma childhood [1] falls major histopathologic                    chromosome arms rare focal high-level amplifications both <br /> subtypes children - embryonal alveolar. Embryonal                             human ERMS ARMS [5&#226;&#8364;&#8220;9], largely precluding identi- <br /> rhabdomyosaroma (ERMS) accounts approximately 60%                             fication specific drivers cancer disease. Moreover, <br /> childhood cases frequently associated RAS pathway                        array CGH cross-species comparisons mouse and <br /> activation [2&#226;&#8364;&#8220;5]. Treatment RMS subtype requires                          human RMS largely failed identify functionally important <br /> surgical resection, chemotherapy, radiation overall poor                    genes contained common copy number alterations (CNAs). <br /> prognosis patients high-risk features, metastasis, relapse               In report, RMS arose Ptch1+/2 Blmtm3Brd/tm3Brd (a <br /> disease. Thus, great elucidating key molecular                  hypomorphic Blm allele) mice exhibited gain chromosome 10 <br /> pathways genetic factors involved continued RMS                      80% cases [10], oncogenes associated this <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                             1                            August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                  Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />   Author Summary                                                           co-amplified high-copy gains human MPNST [19]. High- <br />                                                                            resolution array CGH applied zebrafish T-ALL <br />   Cancer complex genetic disease                        identified subset genes contained CNAs were <br />   associated regional gains losses genomic                     amplified deleted human disease [20]. These two <br />   DNA segments. These changes result aberrant gene                      studies demonstrated utility array CGH technology in <br />   expression drive continued tumor growth. Because                     detecting copy number aberrations candidate driver genes in <br />   amplified deleted DNA segments tend span large                    zebrafish tumor models, functional relevance identified <br />   regions chromosomes, challenging                       genes human disease reported these <br />   identify genes required continued tumor                 genes assessed roles regulating tumor maintenance &#226;&#8364;&#8220; <br />   growth progression. Array comparative genomic                        providing novel targets therapy established tumors. <br />   hybridization (array CGH) effective technology in <br />                                                                               Capitalizing zebrafish model kRASG12D-induced ERMS <br />   identifying abnormal copy number variations cancer <br />                                                                            shares common histopathological, genetic, molecular <br />   genomes. In study, array CGH used zebrafish <br />   model embryonal rhabdomyosarcoma - pediatric                        characteristics human ERMS [4,21,22], utilized high- <br />   muscle tumor. Our work shows zebrafish cancer                   resolution array CGH identify novel CNAs ERMS. <br />   genome contains small number recurrent DNA copy                     Remarkably, array CGH analysis revealed focal CNAs that <br />   number changes, commonly amplified                     span short genomic regions contain 1&#226;&#8364;&#8220;3 genes. To <br />   human disease. Moreover, chromosomal regions                   validate functional significance amplified genes human <br />   small, facilitating rapid identification candidate                ERMS, prioritized genes initial characterization in <br />   oncogenes. A subset genes identified zebrafish array               human ERMS cell lines. Of genes, gene knockdown of <br />   CGH prioritized functional characterization                   Cyclin D2 (CCND2), Homeobox C6 (HOXC6), PlexinA1 (PLXNA1) <br />   human ERMS, identifying evolutionarily conserved path-                   inhibited proliferation human ERMS. PLXNA1 exhibited <br />   ways regulate proliferation, migration, differentiation,            important roles blocking ERMS cells early stages muscle <br />   neovascularization. Our results demonstrate broad                differentiation, enhancing migration, altering anchorage- <br />   utility cross-species array CGH comparisons human                  independent growth. CCND2, HOXC6, PLXNA1, Vascular <br />   zebrafish cancer provide needed                           Endothelial Growth Factor A (VEGFA) highly expressed <br />   discovery platform identifying critical cancer-causing               large fraction human primary RMS, supporting prominent <br />   genes wide range malignancies.                                   roles genes rhabdomyosarcoma. Chemical inhibition <br />                                                                            VEGF signaling reduced tumor growth vivo associated <br /> chromosomal gain remain undefined large number               decrease angiogenesis, implicating VEGF inhibitors prom- <br /> candidate genes region. Moreover, extension           ising therapeutic agents ERMS. Taking advantage tractable <br /> these findings human RMS reported. Rubin et al.            features zebrafish cancer genomes smaller CNA <br /> recently showed greater 30% ERMS arising mice              intervals regions conserved homology human disease, <br /> that harbor p53 homozygous deletion and/or Ptch1 heterozygous              study demonstrates effective use array CGH identify <br /> deletion lack defined molecular signature genetic lesion,             oncogenes required continued tumor growth human <br /> suggesting undiscovered pathways likely contribute ERMS                 rhabdomyosaroma, providing novel therapeutic targets the <br /> transformation, growth, tumor maintenance [11]. To date,               treatment ERMS. <br /> there remains need novel gene discovery methods identify <br /> genes pathways essential tumor growth, progression,            Results <br /> maintenance human cancer &#226;&#8364;&#8220; including ERMS. <br />    Zebrafish cancer shares molecular pathological similarities         Zebrafish array CGH identifies novel conserved CNAs <br /> to human disease [4,12&#226;&#8364;&#8220;16]. For example, Lam et al. (2006)             human ERMS <br /> the use comparative analysis microarray data                 Array CGH performed genomic DNA isolated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> zebrafish human liver tumors demonstrate conserved                kRASG12D-induced zebrafish ERMS compared to <br /> molecular profile tumor progression [13]. Building          adjacent normal tissue. Array CGH revealed complex CNA <br /> work, microarray gene expression studies zebrafish ERMS             <span id='am-317' about='xsp:pattern' typeof='owl:Thing'>pattern</span> relative gains observed frequently than <br /> cross-species comparison human disease identified RAS                   losses. For example, identified 190 regions amplification and <br /> pathway activation common initiating event zebrafish               35 deletions recurrent $3 zebrafish tumors analyzed (Table S1). <br /> and human ERMS. Activating RAS mutations                    Remarkably, 2 20 zebrafish samples exhibit evidence of <br /> identified numerous studies human ERMS [2&#226;&#8364;&#8220;5,17]. Most                aneuploidy, contrasting starkly human ERMS nearly <br /> recently, Paulson et al reported 11 26 (42%) human ERMS            human RMS harbored regions extensive aneuploidy [5]. <br /> samples harbored activating RAS mutations acquisition           While 10 zebrafish ERMS showed evidence CNAs coding <br /> of additional CNAs detected array CGH [5], suggesting              regions genome, 3 exhibited high frequency of <br /> additional genetic lesions likely required drive oncogenic          multiple gains (Table 1 S1). In total, identified 19 gains <br /> transformation ERMS. Not surprisingly, zebrafish cancers           2 losses gene-containing amplicons recurrent at <br /> exhibit recurrent chromosomal gains losses similar            zebrafish ERMS samples. Candidate genes these <br /> found human cancer. For example, transgenic models                   regions predominantly amplified low-level gains, which <br /> zebrafish melanoma, T-cell acute lymphoblastic leukemia (T-                averaged 1&#226;&#8364;&#8220;3 genes spanned 48+/227 kb (+/2 SD, <br /> ALL), ERMS contain genomic imbalances including high-                  Table 1; Fig. 1A). Copy number alterations validated by <br /> level gains losses [18]. However, specific driver events         qPCR genomic DNA (Fig. S1). <br /> not identified studies low resolution          To assess CNAs identified zebrafish ERMS were <br /> platform. Using high-resolution array CGH, Zhang et al (2010)              conserved human disease, zebrafish array CGH data was <br /> demonstrated aneuploid nature zebrafish malignant nerve             compared high-resolution array CGH data 26 human <br /> sheath tumors (MPNST), feature characterizes               ERMS samples [5]. Of 26 samples, 11 carried activating RAS <br /> human disease, identified subset genes                   mutations assessed Sanger sequencing analysis [5]. The two <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   2                        August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                Table 1. Comparison array CGH analyses zebrafish human ERMS. <br />  <br />  <br />                                                                                                                                                                                                                                             with <br />                                                                                                  Samples                                                                                                                                    Amplified   Genes in <br />                                               Zebrafish Chr Start         End          Size (bp) CNA      Genes Interval                   Human Gene Homologues            Chr   Start             End              Size (bp)      CNA         interval <br />  <br />                                               2              3.2E+07      3.2E+07      31975      4 20      patched 2 (ptch2)                   PTCH2                            1     45022354          45089190         66836          1 26     9 <br />                                               2              4.6E+07      4.6E+07      21677      4 20      mbnl1                               MBNL1                            3     153448149         153676279        228130         2 26     2 <br />                                               3              2.1E+07      2.1E+07      72013      3 20      hoxb3a, hoxb4a, hoxb5a,             HOXB3, HOXB4, HOXB5              17    43902466          44075242         172776         5 26     7 <br />                                                                                                                hoxb6a, hoxb7a, hoxb8a,             HOXB6, HOXB7, HOXB8 <br />                                                                                                                hoxb9a, mir10c                      HOXB9 <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org <br />                                               4              5776500      5833215      56715      3 20      rsph9, mrps18a, vegfa               VEGFA                            6     43803475          43880447         76972          6 26     2 <br />                                               4              1.2E+07      1.2E+07      85059      3 20      braf, cry1a, tnnt1                  BRAF                             7     139802093         140482012        679919         5 26     7 <br />                                                                                                                                                    CRY1A                            12    105791516         106162074        370558         9 26     4 <br />                                                                                                                                                    TNNT1                            19    60246821          60592535         345714         8 26     25 <br />                                               4              1.8E+07      1.8E+07      41361      3 20      apaf1                               APAF1                            12    96951092          97889931         938839         10 26    10 <br />                                               6              1563960      1584371      20411      4 20      plxna1                              PLXNA1                           3     128138430         128299614        161184         4 26     3 <br />                                               9              2.4E+07      2.4E+07      24615      6 20      stk17b                              STK17B                           2     196716864         196782315        65451          10 26    2 <br />                                               9              2.4E+07      2.4E+07      32894      6 20      tmeff2a                             TMEFF2                           2     (192522992)       (192767889)      244897         11 26    Aneuploid <br />                                               10             1.8E+07      1.9E+07      70481      3 20      pcdh1g11, pcdh1gb9,                 PCDHGC5                          5     140848162         140948899        100737         5 26     7 <br />                                                                                                                pcdh1g9, pcdh1gb2, <br />  <br />  <br />  <br />  <br /> 3 <br />                                                                                                                pcdh1g3, pcdh1g18 <br />                                               13             3.4E+07      3.4E+07      16915      6 20      smap1                               SMAP1                            6     71434200          71628437         194237         0 26     Not Amplified <br />                                               15             2.1E+07      2.1E+07      50974      5 20      aldh3a2                             ALDH3A                           17    19492656          19521500         28844          0 26     Not Amplified <br />                                               15             186438       238755       52317      6 20      limk1                               LIMK1                            17    72888258          73599600         711342         7 26     9 <br />                                               17             2.1E+07      2.1E+07      82203      3 20      ppp1r3ca                            PPP1R3C                          10    92581714          93961386         1E+06          1 26     17 <br />                                               19             1.4E+07      1.4E+07      29638      4 20      hoxa5a, hoxa4a,                     HOXA1, HOXA2, HOXA3              7     27095146          27150002         54856          7 26     5 <br />                                                                                                                hoxa3a, hoxa1a                      HOXA4, HOXA5 <br />                                               20             4.6E+07      4.6E+07      69342      3 20      tram2                               TRAM2                            6     52500302          52578833         78531          0 26     3 <br />                                               22             3E+07        3E+07        87467      3 20      acbd4, hexim1                       ACBD4, HEXIM1                    17    40559338          40681751         122413         2 26     3 <br />                                               23             3.5E+07      3.5E+07      94371      3 20      hoxc3a, hoxc4a, mir10b-2,           HOXC13, HOXC12, HOTAIR,          12    52461084          52737632         276548         12 26    10 <br />                                                                                                                hoxc5a, hoxc6a, hoxc8a,             HOXC11, HOXC10, HOXC9, <br />                                                                                                                hoxc9a, mir196a-1, hoxc10a,         HOXC8, HOXC6, HOXC5 <br />                                                                                                                hoxc11a, hoxc12a, hoxc13a           HOXC4 <br />                                               25             1.4E+07      1.4E+07      20581      5 20      ccnd2a, tigara                      CCND2                            12    3866904           4377212          510308         8 26     3 <br />                                                                                                                                                                                                             Average:         328,193+/2278,351          7.3 <br />                                               ERMS Deleted <br />                                               19             3289509      3305879      16370      3 20      illr3                               No homologue Human <br />                                               7              1.7E+07      1.7E+07      42336      3 20      nitr1h, nitr1j                      No homologues Human <br />  <br />                                               Zebrafish gene-containing, CNAs compared identified 26 primary human ERMS samples Paulson et al. (2011). Genes selected characterization study bold. <br />                                                                                                                                                                                                                                                                         Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                               doi:10.1371/journal.pgen.1003727.t001 <br />                                                                                                  Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 1. Array CGH reveals cancer-specific chromosomal abnormalities zebrafish ERMS. (A) Summary common gene-containing <br /> CNA gains (green) losses (red) 20 animals examined. Only recurrent CNAs $3 samples shown. The height bar correlates <br /> with frequency aberration. Detailed view regional gains vegfa chromosome 4 (B), ccnd2a chromosome 25 (C), hoxc6a on <br /> chromosome 23 (D), plxna1 chromosome 6 (E). Y-axis denotes log2 ratio probes X-axis denotes genomic coordinates. <br /> doi:10.1371/journal.pgen.1003727.g001 <br />  <br />  <br /> regions chromosomal loss zebrafish ERMS contained                   CNAs focal, low-copy gains spanning 328 kB+/2278 kB <br /> zebrafish-specific genes failed human homologues             contained 7.3 genes average (Table 1 S1). <br /> (Table 1). By contrast, genes contained 19 CNA gains              To demonstrate efficacy array CGH approach to <br /> mapped 21 distinct homologous genomic regions human.                identify evolutionarily conserved oncogenes essential driving <br /> Utilizing statistical algorithms threshold settings       tumor progression maintenance, prioritized CNAs that <br /> outlined Paulson et al [5], discovered 18 21                contained genes amplified zebrafish human <br /> homologous regions amplified low-copy gains               ERMS. In total, candidate genes prioritized further <br /> human ERMS samples (Table 1). Like zebrafish ERMS,                  study human ERMS based following criteria. 1) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                  4                         August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                     Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br /> Candidate oncogenes differentially expressed human                   differentiation arrest [24,25]. As result, ERMS composed of <br /> ERMS compared ARMS and/or normal muscle assessed                    heterogeneous subpopulations proliferating tumor cells that <br /> microarray gene expression studies. 2) Genes known                 vary differentiation status. Therefore, oncogenes are <br /> oncogenic activity cancer types, ascribed                   essential regulating proliferation ERMS cells likely play <br /> functional roles ERMS, prioritized additional study              role modulating differentiation status. Thus, we <br /> to serve &#226;&#8364;&#732;&#226;&#8364;&#732;proof concept&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes cross-species                  determined CCND2, HOXC6, PLXNA1 also <br /> comparative study. 3) A subset genes selected              played role blocking differentiation ERMS. Knockdown <br /> unknown function ERMS represent potential novel                       PLXNA1 resulted increased formation multinucleated <br /> oncogenes. 4) Amplified CNA regions harbored single                   myocytes induction myosin heavy chain expression RD <br /> human homologue prioritized. Based criteria,              cells (Fig. 3 B, p = 0.03). By contrast, siRNA knockdown CCND2 <br /> CCND2, HOXC6, PLXNA1, VEGF, BRAF LIMK1 selected                     HOXC6 did alter differentiation status human RD <br /> for study (Table 1). CCND2, PLXNA1, VEGFA, LIMK1                 cell (Fig. 3 E). To validate <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> PLXNA1 knockdown, <br /> were single genes contained amplified CNA                     independent PLXNA1 shRNA knockdown stable lines were <br /> intervals human disease. BRAF gene                    generated cultured differentiation condition. Both <br /> interval overexpressed human ERMS, CRY1                  PLXNA1 shRNAs induced robust gene knockdown compared to <br /> and TNNT1 identified amplified interval             control scrambled shRNA (Fig. 3 F), resulting increased <br /> differentially expressed comparing human ERMS normal                 numbers multinucleated-myocytes myosin heavy chain <br /> muscle (Fig. S2). HOXC6 reported highly expressed             expression (Fig. 3 D&#226;&#8364;&#8220;E, p = 0.01). <br /> in human ERMS compared ARMS [23], suggesting                            PLXNA1 played critical role regulating anchorage- <br /> possible role modulating tumor growth. While CCND2, cell                independent growth colony formation assays. Stable knockdown <br /> cycle regulator, VEGFA, essential regulator angiogenesis          PLXNA1 resulted impaired anchorage-independent growth <br /> variety cancer types BRAF, oncogene variety                decreased colony formation two-fold 15 days when <br /> cancers, likely serve &#226;&#8364;&#732;&#226;&#8364;&#732;proof concept&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes             compared RD cells transduced control shRNA (Fig. 3 G&#226;&#8364;&#8220;I, <br /> demonstrating functional significance human ERMS. LIMK1,                  p = 0.0003). Moreover, colonies smaller size, likely <br /> HOXC6 PLXNA1 represent potential novel candidate genes               reflecting prominent role PLXNA1 regulating cell <br /> driving tumor growth ERMS.                                                growth. Together, findings indicate PLXNA1 plays an <br />                                                                              essential important role regulating proliferation differenti- <br /> A subset amplified genes play essential roles                          ation transformed ERMS. <br /> regulating ERMS proliferation <br />    The candidate genes assessed anti-proliferative        Knockdown PLXNA1 results impaired migration of <br /> effects human RD SMS-CTR ERMS cell lines siRNA                     human ERMS cell lines <br /> knockdown, establishing role genes continued tumor               Migratory behavior tumor cells vitro useful <br /> growth maintenance. Importantly, human                     predictive index cell invasion metastasis vivo. Genes and <br /> ERMS cell lines contains mutationally-activated RAS alleles,                 pathways essential regulating migratory behavior <br /> mimicking zebrafish model. Effective gene knockdown                  tumors cells likely serve important functions mediating <br /> validated quantitative RT-PCR and/or Western analysis (Fig. 2             metastasis potential targets novel therapy. <br /> A Fig. S3 B S4 A). A quantitative VEGFA ELISA                        Wound healing transwell migration assays used assess <br /> confirmed lower levels secreted VEGFA growth medium                role CCND2, HOXC6, PLXNA1 VEGFA migration of <br /> from cells transfected VEGFA siRNA (Fig. S3 A, p,0.05).                 human RD SMS-CTR ERMS cell lines. Knockdown of <br /> Gene-specific siRNA knockdown CCND2, HOXC6 PLXNA1                     PLXNA1 siRNAs (smart-pool individual siRNAs) and <br /> resulted reduced cell viability assessed luminescent cell         shRNAs resulted impaired migration RD SMS- <br /> viability assay cell lines (Fig. 2 B S4 B&#226;&#8364;&#8220;H). By contrast,       CTR cells 22 hours (p,0.02 RD p#0.04 SMS-CTR, <br /> knockdown BRAF, LIMK1, VEGFA failed alter viability                 Fig. 4 G&#226;&#8364;&#8220;I, Fig. S6). By contrast, knockdown CCND2, HOXC6 <br /> and/or growth cell lines (Fig. 2 B 2 C data              VEGFA did affect migration RD SMS-CTR <br /> shown). Following validation growth effects using additional          cells (p.0.25, Fig. 4 A&#226;&#8364;&#8220;F, I). As independent assessment of <br /> siRNAs CCND2, HOXC6 PLXNA1 RD SMS-CTR                         ERMS cell migration, PLXNA1 stable shRNA knockdown cells <br /> cell lines (Fig. 2 C S4 B&#226;&#8364;&#8220;H), genes prioritized           assessed migration transwell assay. Knockdown of <br /> additional functional studies. For example, knockdown CCND2,              PLXNA1 RD cells independent gene-specific shRNAs <br /> HOXC6 PLXNA1 resulted reduced EDU incorporation                       resulted .50% reduction transwell migration (p = 0.03 for <br /> when compared cells transfected control siRNA                shRNA-A p = 0.0038 shRNA-B, Fig. 4 J). Together, these <br /> RD SMS-CTR cell lines, suggesting inhibition cell                results support additional role PLXNA1 regulating <br /> growth resulted block proliferation (Fig. 2 D). Apoptosis          migratory behavior human ERMS cells. <br /> was altered gene knock assessed Annexin V <br /> staining (Fig. S5). In total, data uncovered important roles         Inhibition VEGFA results reduced angiogenesis and <br /> CCND2, HOXC6 PLXNA1 regulating ERMS prolifera- <br />                                                                              tumor growth <br /> tion, validating role novel genes regulating <br />                                                                                 The VEGFA pathway exerts powerful roles regulating <br /> continued tumor cell proliferation human ERMS cells. <br />                                                                              cancer-induced angiogenesis, missed in <br />                                                                              human cell culture assays. To assess role VEGFA in <br /> Knockdown PLXNA1 results increased terminal                            modulating tumor growth vivo, ERMS-bearing zebrafish were <br /> differentiation impaired anchorage-independent                           treated VEGF receptor tyrosine kinase inhibitor, <br /> growth human ERMS                                                         cediranib, DMSO vehicle 7 days assessed effects <br />   ERMS expresses myogenic factors MYOD MYF5                      tumor growth. Relative tumor growth determined the <br /> yet fails complete normal myogenesis secondary                      ratio tumor volume change pre- post-treatment <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     5                        August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                       Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 2. CCND2, HOXC6 PLXNA1 exert important roles human ERMS cell proliferation. (A) Western analysis following siRNA <br /> knockdown RD cell line. The percentage knockdown (% KD) indicated lane (representative example shown three <br /> independent replicates). (B) Viability following siRNA gene knockdown RD cells assessed cell-titer glo assay. (C) Graph summarizing the <br /> results cell-titer glo assay. OD-fold change 3 days RD cell line knockdown smart-pool individual gene-specific siRNAs is <br /> indicated. (D) Summary EDU proliferation analysis RD SMS-CTR ERMS cell lines,. (Asterisks denote significant differences siRNA <br /> knockdown compared control siRNA (p,0.05). Each error bar B, C, D denotes standard deviation 3 independent experiments. <br /> doi:10.1371/journal.pgen.1003727.g002 <br />  <br />  <br /> was reduced three-fold cediranib-treated fish                      knockdown human ERMS cell lines. Together, data <br /> compared treated vehicle (Fig. 5A&#226;&#8364;&#8220;M, p = 0.0017,                suggest activation VEGF pathway promotes ERMS <br /> Student&#226;&#8364;&#8482;s T-test). As VEGFA known promote angiogenesis                  tumor progression enhanced angiogenesis. <br /> during tumor progression variety cancers, assessed <br /> whether inhibition VEGFA blocked angiogenesis ERMS                 CCND2, HOXC6, PLXNA1 VEGFA commonly <br /> in vivo. In order visualize angiogenesis established tumors,            expressed human rhabdomyosaroma <br /> ERMS co-expressing rag2-KRASG12D rag2-dsRED                             Having established roles CCND2, HOXC6, PLXNA1 and <br /> transplanted irradiated fli1-GFP fish exhibit vessel-               VEGFA ERMS growth, wanted assess extent to <br /> specific GFP expression [26]. Fish engrafted ERMS                   proteins expressed human primary RMS. <br /> treated cediranib DMSO vehicle 7 days. Animals             Immunohistochemistry performed using antibodies to <br /> were assessed differences overall tumor growth                CCND2, HOXC6, PLXNA1 VEGFA primary human <br /> microvessel density determined cryosections tumors.                  tumors fetal muscle (Supplemental Table S2). In all, 8 <br /> ERMS-affected animals treated cediranib showed signifi-                pediatric 11 adult ERMS 3 pediatric 4 adult alveolar <br /> cant reduction tumor growth accompanied two-fold                   RMS (ARMS) analyzed. Remarkably, CCND2, HOXC6, <br /> reduction tumor microvessel density compared                 PLXNA1 VEGFA protein expression detected a <br /> treated vehicle control (N = 5 group, p = 0.006, Fig. 5         majority RMS samples antibody staining was <br /> N&#226;&#8364;&#8220;P). Cediranib-treated ERMS did exhibit difference                  largely negative fetal muscle (Fig. 6). Specifically, HOXC6 <br /> proliferation compared vehicle control-treated tumors                 protein expression detected 14 19 ERMS strong, <br /> (Fig. 5 Q&#226;&#8364;&#8220;S), consistent results VEGFA gene                      diffuse staining 6 14 cases (1 adult 5 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      6                          August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                       Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 3. Knockdown PLXNA1 induced differentiation impaired anchorage-independent growth human ERMS cells. RD <br /> cells stained myosin heavy chain (MF20) DAPI following culture differentiation conditions 72 hrs. (A) Control siRNA. (B) PLXNA1 <br /> smart-pool siRNA. (C) Control scrambled shRNA. (D) PLXNA1 shRNA-1. DAPI, blue; MF20-positive cell, green. (E) Quantification MF-20 <br /> immunofluorescence siRNA shRNA-knockdown RD cells. Asterisk indicates significant differences gene knock- control cells <br /> (p,0.05). Error bars denote standard deviation. (F) Western analysis PLXNA1 shRNA stable knockdown; sc, scrambled control shRNA; 1, PLXNA1 <br /> shRNA-1; 2, PLXNA1 shRNA-2. A soft agar colony formation assay assess PLXNA1 knockdown effects anchorage-independent growth (G&#226;&#8364;&#8220;I). (G) <br /> Control scrambled shRNA. (H) PLXNA1 shRNA. (I) Quantification colony formation assay results. Error bar indicates standard deviation triplicate <br /> experiments. <br /> doi:10.1371/journal.pgen.1003727.g003 <br />  <br />  <br />  <br /> pediatric). In contrast, 2 7 ARMS showed weak, positive               High VEGFA expression correlates clinical outcome <br /> staining HOXC6, consistent lower-level gene transcript                  To assess dysregulated expression CCND2, HOXC6, <br /> levels detected pediatric ARMS compared ERMS                      PLXNA1 VEGFA correlates clinical outcome, Kaplan <br /> (Supplemental Fig. S7). CCND2, PLXNA VEGFA                           Meier analyses completed using microarray gene expression <br /> expressed comparable frequency subtypes RMS.                    data primary ERMS ARMS [23]. Samples were <br /> For example, CCND2 detected 15 19 ERMS 5                     stratified based high low median expression gene <br /> 7 ARMS, PLXNA1 expression 17 19                         assessed independent predictor survival. Based <br /> ERMS 6 7 ARMS. VEGFA antibody staining                             analysis, differential expression CCND2 PLXNA1 did <br /> detected tumor cells vasculature 10 19                   correlate overall survival outcome ERMS or <br /> ERMS (strong staining 1 adult 1 pediatric case), 6               ARMS (Fig. S8). HOXC6 differentially upregulated ERMS <br /> of 7 ARMS exhibited weak staining cases analyzed.                      compared ARMS (Fig. S7); thus, high expression HOXC6 <br /> Additional immunohistochemical analysis tissue microarray                correlated better overall survival (Fig. 7 A), finding <br /> from Children&#226;&#8364;&#8482;s Oncology Group revealed positive VEGF                         consistent previous studies demonstrating better clinical <br /> expression 31 38 ERMS 3 6 ARMS (Table S3).                       outcome ERMS patients compared ARMS [27]. <br /> Of 38 cases ERMS, 29 cases showed strong diffuse                   Finally, samples high mRNA expression VEGFA correlated <br /> staining. Our analysis suggests despite oncogenes                  low overall clinical survival ERMS cohort did not <br /> being infrequently amplified human disease, protein                  predict survival outcome ARMS (Fig. 7 B). In addition, VEGFA <br /> expression levels elevated majority human ERMS.                   expression did correlate clinical stage, indicating is <br /> These data imply important roles genes regulating                likely independent prognostic indicator (Fig. S9). These data <br /> tumor growth large fraction human ERMS                            implicate important roles VEGFA promoting ERMS tumor <br /> suggesting additional, undiscovered mechanisms                 progression identify VEGFA biomarker likely use in <br /> regulate expression genes.                                           stratifying ERMS patients high low-risk groups. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      7                          August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                      Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />                                                                                Discussion <br />                                                                                   Prior cytogenetic array CGH studies human ERMS <br />                                                                                demonstrate inconsistent non-specific partial whole <br />                                                                                chromosomal aneuploidy different primary tumors <br />                                                                                making difficult identify critical genes essential driving <br />                                                                                tumor growth. Utilizing zebrafish model RAS-induced <br />                                                                                ERMS mimics human disease [4,21] subsequent <br />                                                                                array CGH analyses genomic DNA tumor vs. matched <br />                                                                                normal, able rapidly identify candidate gene- <br />                                                                                containing regions likely contribute ERMS pathogenesis. <br />                                                                                The 19 CNA gains recurrently amplified zebrafish <br />                                                                                ERMS mapped 21 homologous regions human <br />                                                                                genome. Remarkably, 18 regions demonstrated <br />                                                                                low-level genomic amplification human ERMS. To validate <br />                                                                                candidate genes contained intervals exert <br />                                                                                important roles continued tumor growth maintenance, <br />                                                                                characterized function amplified genes human <br />                                                                                ERMS cell lines conclusively demonstrated functional <br />                                                                                significance CCND2, HOXC6, PLXNA1 proliferation of <br />                                                                                human ERMS. PLXNA1 important roles regulation <br />                                                                                differentiation migration ERMS cells. As vitro <br />                                                                                analyses performed study able assess <br />                                                                                aspects tumorigenesis neovascularization and <br />                                                                                tumor initiation, utilized zebrafish vivo model to <br />                                                                                demonstrate important role VEGF-A pathway in <br />                                                                                mediating angiogenesis tumor growth. In total, our <br />                                                                                work identified roles 4 6 candidate genes identified our <br />                                                                                cross-species array CGH studies eliciting important roles in <br />                                                                                human ERMS. Importantly, strategy limited to <br />                                                                                zebrafish ERMS, likely provide powerful new methods <br />                                                                                identify novel tumor-suppressor oncogenes wide <br />                                                                                range zebrafish human tumors. <br />                                                                                   Data array CGH study previous studies of <br />                                                                                zebrafish cancer revealed low-level CNA gains frequent <br />                                                                                DNA alteration cancer, class <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> not <br />                                                                                commonly studied difficulty identifying <br />                                                                                relevant meaningful genes regions. Importantly, <br />                                                                                zebrafish allows easy identification low-level gene <br />                                                                                amplifications. In total, data consistent model <br />                                                                                zebrafish tumor cells undergo acquisition low- <br />                                                                                amplitude gains, likely represented single copy gains within <br />                                                                                CNA regions. For example, observed clonal- <br />                                                                                populations purified T-ALL cells (90% enriched blasts) <br />                                                                                contain low-amplitude gains [Blackburn et al., unpub- <br />                                                                                lished]. Moreover, Rudner et al. (2011) recently showed a <br />                                                                                majority amplified, gene-containing CNAs zebrafish <br />                                                                                T-ALL amplified human disease [20]. Upon re- <br />                                                                                analysis data, 72% reported amplified <br />                                                                                regions detected low-level gains zebrafish T-ALL, <br />                                                                                reported such. Zhang et al. identified large <br />                                                                                regions aneuploidy high-level CNA gains zebrafish <br />                                                                                malignant peripheral nerve sheath tumors assessed by <br /> Figure 4. Knockdown PLXNA1 impairs migration human                       array CGH, identified numerous regions low-level <br /> ERMS cells. Representative images ERMS cells transfected               CNA gains, dismissed potential causative lesions <br /> gene-specific siRNAs 0 hr (A, control siRNA; C, CCND2 siRNA; E, HOXC6 <br /> siRNA; G, PLXNA1 siRNA) 22 hrs (B, control siRNA; D, CCND2 siRNA;          cancer. Thus, despite previous reports observing <br /> F, HOXC6 siRNA; H, PLXNA1 siRNA) following gap creation. Scale bar             low-level CNA gains zebrafish malignancy, reported <br /> indicates 100 mm. (I) Quantification data wound healing assay.         functional importance class genes promote <br /> Each error bar indicates standard deviation 5&#226;&#8364;&#8220;6 independent             tumor progression maintenance zebrafish human <br /> replicates. (J) A Transwell migration assay performed RD cells          disease. Although formally possible low level gains <br /> that stably express control shRNA independent PLXNA1 <br />                                                                                detected zebrafish ERMS represent high-copy gains masked <br /> shRNAs. Migration assessed 24 hours. Each error bar indicates <br /> standard deviation fields 2006 magnification. Asterisks          high degree tumor cell heterogeneity and/or <br /> denote p,0. 05.                                                                contamination normal DNA non-transformed blood, <br /> doi:10.1371/journal.pgen.1003727.g004                                          fibroblasts stroma, data strongly argue low-copy <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       8                        August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                       Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 5. Chemical inhibition VEGF signaling cediranib reduces ERMS growth vivo. Syngeneic CG1 fish transplanted with <br /> ERMS cells co-expressed rag2-KRASG12D rag2-dsRED. Fish engrafted tumors treated DMSO vehicle (A&#226;&#8364;&#8220;F) 100 nM of <br /> cediranib 7 days (G&#226;&#8364;&#8220;L). Pre-treatment (A&#226;&#8364;&#8220;C G&#226;&#8364;&#8220;I) post-treatment images (D&#226;&#8364;&#8220;F J&#226;&#8364;&#8220;L) representative fish. Bright field (A,D,G,J), dsRED <br /> fluorescence (B,E,H,K) merged image planes (C,F,I,L). Scale bar 3 mm. (M) Quantification relative volume change individual animals. (N&#226;&#8364;&#8220;O) <br /> fli1-GFP transgenic zebrafish transplanted dsRED-labeled ERMS treated DMSO (N) cediranib (O). Scale bar equals 50 mm. (P) <br /> Microvessel density quantification. Asterisk indicates statistically significant difference DMSO cediranib-treated groups based on <br /> student t-test. Each error bar indicates standard deviation 3 fields microvessels animal. EDU incorporation analysis DMSO (Q) or <br /> cediranib (R) treated fish. Scale bar 50 mm. (S) Quantification EDU analysis cohort animals. Each error bar indicates standard <br /> deviation percent EDU+ cells 3 fields animal. <br /> doi:10.1371/journal.pgen.1003727.g005 <br />  <br />  <br />  <br /> amplification common attribute zebrafish                    important functional roles human cancer. Finally, also <br /> human cancer.                                                                 genes CNA highly expressed a <br />    Interestingly, functional relevance low-                majority human RMS despite infrequently amplified as <br /> level gains genomic duplication events                      low-copy CNAs, suggesting importance gene <br /> infrequently reported human cancer, type DNA                       pathways regulating large fraction human ERMS and <br /> alteration predicts important clinical parameters               additional mechanisms underlying dysregulation this <br /> disease susceptibility, therapy resistance adverse prognosis.             class genes cancer likely. <br /> For example, duplication region chromosome 6q27                       Our work identified essential roles genes in <br /> detected individuals affected familial chordoma, rare               modulating ERMS growth, maintenance, migration, neovas- <br /> bone cancer, unaffected individuals                  cularization. Of genes, CCND2, HOXC6 PLXNA1 <br /> same family [28]. MYB tandem duplication occurs pediatric                  exhibited important roles regulating proliferation human <br /> T-ALL results homologous recombination ALU                        ERMS cell lines. PLXNA1 additional roles arresting <br /> repetitive sequences flanking MYB locus. Elevated MYB                     ERMS cells early stages muscle differentiation, enhancing <br /> expression associated poor outcome T-ALL [29].                     tumor cell migration, altering anchorage-independent <br /> Similarly, focal tandem duplication contributes chemo-                growth. Despite fact genes and/or related family <br /> therapy resistance patients high-grade ovarian cancer                 members ascribed functions cancer types, their <br /> [30]. These findings indicate low-level CNA gains                        contributions pathogenesis ERMS been <br /> have important clinical prognostic relevance likely play                  previously characterized. For example, HOXC6, homeobox <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      9                          August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                     Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 6. Genes contained low copy CNAs expressed primary human rhabdomyosaroma normal fetal muscle. <br /> Immunohistochemistry human primary RMS fetal muscle tissue samples. Hematoxylin Eosin stained sections (A&#226;&#8364;&#8220;C). Expression HOXC6, <br /> CCND2, PLXNA1 VEGFA embryonal rhabdomyosaroma (D, G, J, M), alveolar rhabdomyosaroma (E, H, K, N) fetal muscle (F, I, L, O). Magnified <br /> views staining shown insets. In fetal muscle, PLXNA1 VEGFA expressed vasculature (examples indicated arrowheads in <br /> L O corresponding insets). The cumulative frequency positive staining tumor subtype shown right corner panel <br /> (pediatric adult samples combined). Scale bar (panel A) = 50 mm. <br /> doi:10.1371/journal.pgen.1003727.g006 <br />  <br />  <br />  <br /> transcription factor, regulates expression genes including            B-Cell Lymphoblastic Lymphoma cells invasion ovarian <br /> BMP7, FGFR2, IGFP3 PDGFRA influence oncogenic                         breast tumor cells [36&#226;&#8364;&#8220;38]. Plexin-A1, gene identified in <br /> activities prostate cancer [31]. HOXC6 highly expressed             study, shown activate VEGF receptor NF- <br /> ERMS ARMS [23], suggesting specific independent                kB promote survival malignant mesothelioma cells [39], <br /> role regulating growth human ERMS subtype. A role              suggesting complex interplay PLXNA1 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> and <br /> HOXC6 regulating continued RMS growth                        neovascularization. Taken together, study demonstrated <br /> reported study. CCND2 belongs D-type G1 cyclins            prominent novel roles CCDN2, HOXC6, PLXNA1 <br /> (D1, D2 D3). While cyclin D1 frequently dysregulated               modulating ERMS proliferation PLXNA1 exerts impor- <br /> cancer marker disease progression [32],                     tant additional roles regulating differentiation migration. <br /> involvement cyclin D2 cancer characterized.             None genes previously implicated important <br /> CCND2 amplified 2% gliomas zebrafish human               modulators ERMS growth maintenance, suggesting that <br /> MPNSTs [19,33]. Finally, PLXNA1 belongs highly conserved                cross species array CGH studies valuable for <br /> family transmembrane receptors bind semaphorins                  uncovering genetic lesions wide range zebrafish and <br /> have shown mediate neuronal cell migration, guidance,                human cancers. <br /> and patterning [34,35]. In humans, plexins group                 VEGF pathway activation promotes tumor angiogenesis and <br /> subfamilies implicated having roles              progression variety human cancers, elevated VEGF <br /> cancer progression growth. In particular, plexin-B1                  expression correlates poor prognosis certain tumor types <br /> function oncogene promoting proliferation survival           [40&#226;&#8364;&#8220;42]. However, report, prognostic impact of <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    10                         August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                  Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br />  <br />  <br /> Figure 7. Elevated VEGFA expression correlates poor clinical outcome rhabdomyosarcoma. Kaplan-Meier analysis completed <br /> using microarray gene expression data patient outcome available. Comparison RMS patients, ERMS patients only, and <br /> translocation-positive ARMS patients only. (A) HOXC6. (B) VEGFA. <br /> doi:10.1371/journal.pgen.1003727.g007 <br />  <br />  <br />  <br /> VEGFA expression human ERMS described.                      small-size CNAs containing genes region of <br /> Here, VEGFA amplified low-copy gain               chromosomal aberration, positioning zebrafish an <br /> small cohort zebrafish human RMS highly                      effective model discovering novel genes required for <br /> expressed majority human patient samples. High VEGFA                continued tumor growth maintenance wide <br /> mRNA expression correlated poor clinical outcome human              diversity cancer types. <br /> ERMS, underscoring importance pathway driving <br /> continued tumor growth. As VEGFA expression level linked             Methods <br /> to clinical stage, represents important independent prognostic <br /> indicator potential biomarker therapy stratification.             Animal human protocol approval <br /> Chemical inhibition VEGF signaling pre-clinical vivo             Studies approved Massachusetts General Hospital <br /> model effectively suppressed tumor growth reducing angiogen-             Subcommittee Research Animal Care protocol <br /> esis, consistent findings pre-clinical testing           #2011N000127 (zebrafish) Partners Human Research <br /> VEGFR inhibitors small number human RMS xenografts                  Committee IRB protocol #2009-P-002756 (human). <br /> into mice [43]. Although clinical trials VEGF inhibitors other <br /> types cancers exhibited mixed results [44&#226;&#8364;&#8220;46], data             Array comparative hybridization <br /> suggest targeting VEGF pathway promising                     TuAB-strain zebrafish co-injected one-cell stage <br /> therapeutic option curb tumor growth subset high-risk            linearized rag2-KRASG12D rag2-dsRED DNA con- <br /> ERMS patients.                                                              structs previously described [4,47]. dsRED-labeled ERMS <br />    In summary, array CGH studies zebrafish cancer               adjacent non-neoplastic tissues dissected tumor- <br /> identified conserved CNA gains functional significance              bearing animals 30&#226;&#8364;&#8220;40 days life. RNA DNA were <br /> human ERMS. As proof principle, demonstrat-                 extracted Trizol (Sigma). Tumor DNA labeled Cy5 <br /> ed utility zebrafish array CGH studies identify                   (Bioprime system, Invitrogen, Carlsbad, CA) hybridized <br /> oncogenes essential continued tumor growth                  matched normal samples labeled Cy3 the <br /> both zebrafish human ERMS. Our work provides 13                    custom SurePrint G3 400k CGH microarray (Agilent Technol- <br /> additional CNA gains conserved zebrafish                    ogies, Santa Clara, CA). Array image scans extracted using <br /> human ERMS essential genetic lesion              Agilent Feature Extraction software (Agilent Technologies, Inc, <br /> identified &#226;&#8364;&#8220; providing potential genes interrogate                Santa Clara, CA), normalized signal intensity, imported <br /> future. Our studies suggest amplified CNAs                   Nexus Copy Number software program version 5.1 <br /> contain genes regulate important processes cancer                   (Biodiscovery, Inc., El Segundo, CA). CNA calls generated <br /> maintenance growth. Moreover, study reveals                       based log2 ratio output files using rank segmentation <br /> number tractable features zebrafish cancer genomes               algorithm. Settings optimized using self-self hybridizations <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   11                       August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                   Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br /> to reduce false positive calls. The parameters include signifi-             secondary antibody (1:2000) performed 5% milk/TBST <br /> cance threshold 1.0 E-5, maximum continuous probe spacing                2 hours. <br /> 200 kb, minimum number probes sequence 3, log2 <br /> ratios 1.0, 0.25, 20.25 21.0 high-level amplifications,          Cell proliferation apoptosis assays <br /> gains, losses deletions, respectively. CNAs               siRNA transfected cells assessed Cell Titer Glo assay as <br /> determined using aggregate function Nexus. Aggregates                manufacturer&#226;&#8364;&#8482;s instructions (Promega). Cells also <br /> are represented segmented regions gain loss shared            pulsed EDU 2 hours, harvested 48 hours post- <br /> set samples number samples sharing event                 transfection processed using EDU ClickIt Flow Cytometry <br /> referred aberration frequency. The minimum                        Assay kit (Alexa Fluor 647 dye, Invitrogen). Unstained cells were <br /> aberration frequency required analysis study set             used negative sample facilitate gating flow cytometry. <br /> at $15% (n$3 20 zebrafish ERMS contained common                        To assess apoptosis, cells harvested 48 hours post- <br /> region gene amplification).                                              transfection labeled PE Annexin V 7-AAD using <br />    For human ERMS sample analysis, normalized log2                      PE Annexin V Apoptosis Detection Kit (BD Pharmagin). <br /> intensity files (series number GSE27392) downloaded               Unstained cells, cells treated PE Annexin V 7-AAD <br /> Gene Expression Omnibus (GEO) National Center                    used set gates flow cytometry. Each analysis <br /> Biotechnology Information (NCBI) imported                      performed triplicate. A student&#226;&#8364;&#8482;s T-test performed to <br /> analyzed using Nexus Copy Number software (version 5.1,                     assess difference percentage Annexin V- <br /> BioDiscovery). This program analyzes log2 ratio output files using          positive cells test samples control siRNA-transfected <br /> a rank segmentation algorithm similar circular binary segmen-            cells significant. <br /> tation. Samples segmented following removal the <br /> greatest 3% outliers minimum five-probe requirement at <br /> a significance threshold 1E-08. Gains losses defined         Cell migration assays <br /> regions exhibiting log2 values 0.2 20.18, respectively,            A wound-healing assay performed cells transiently <br /> high-level amplifications deletions defined log2 values              transfected siRNA and/or cells stably express gene- <br /> greater 0.5 20.5. Following identification        specific shRNA. Cell seeded 6-well plates grown to <br /> human ERMS aberrations, homologous human regions                         nearly confluent density. A scrape using a <br /> zebrafish ERMS CNAs analyzed determine                      pipette tip, cell migration gap assessed after <br /> common low-level amplifications present zebrafish              22 hours. Images taken 0 22 hrs calculate the <br /> and human ERMS samples.                                                     percentage gap closure. ERMS cells analyzed for <br />                                                                             altered migration transwell assay. Specifically, 26104 cells <br />                                                                             seeded 6.5 mm-membrane inserts (Corning) DMEM <br /> Immunohistochemistry <br />                                                                             allowed migrate permeable membranes <br />    Paraffinized human primary rhabdomyosaroma (5 ERMS <br /> and 3 ARMS), US Biomax tissue microarray (14 ERMS 4                     (8.0 mM pore size) chamber containing <br /> ARMS, Rockville, MD), Children&#226;&#8364;&#8482;s Oncology Group                       medium 10% FBS. Cells fixed 4% <br /> tissue microarray analyzed immunohistochemical                      paraformaldehyde 24 hours stained hematoxylin <br /> staining previously described [48]. HOXC6 (Sigma, 1:200),                30 minutes. Unmigrated cells inserts removed. <br /> CCND2 (Ab-Cam, 1:100), PLXNA1 (Ab-Cam; 1:200)                           Six random fields migrated cells membranes were <br /> VEGFA (Ab-Cam; 1:125). BGAR- biotinylated goat anti-rabbit                  imaged using Olympus light microscope (Model MVX10, <br /> (Vector #BA-1000) used secondary antibody.                       4006 magnification) manually counted. A Student&#226;&#8364;&#8482;s T-test <br /> Pathology review completed independently E.C.                    performed assess differences control and <br /> G.P.N.                                                                      experimental groups. <br />  <br /> Cell lines, siRNA transfection, stable shRNA knockdown                      Soft-agar colony formation assay <br /> and Western analysis                                                           A <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> layer 1% agar 10% FBS/DMEM prepared <br />    The human RD cell line obtained ATCC cell                       6-well plates. Cells resuspended 0.5% low-melting <br /> biology collection (Manassas, Virginia) SMS-CTR cell                point agarose/10%FBS/DMEM overlaid base layer <br /> line provided Dr. Corrine Linardic (Duke University, North               2.56103 cells subsequently kept the <br /> Carolina). Cells seeded density 56102 cells 6-well          humidified incubator 5% CO2 media change 3 <br /> plates 2 ml antibiotic-free 10% FBS/DMEM. 50 pg                    days 15 days. Cells fixed 4% paraformaldehyde <br /> gene-specific smart-pool control siRNA transfected             stained 0.05% crystal violet. Colony formation assay <br /> cells using RNAiMax lipofectamine transfection reagent (In-                 shRNA stable knockdown line performed in <br /> vitrogen). For stable knockdowns, scrambled gene-specific               triplicate. Image containing soft agar colonies was <br /> shRNAs pLKO.1-based lentiviral vectors packaged                  taken low magnification light microscopy. Colony count <br /> 293T cells. shRNAs obtained molecular profiling                   performed using ImageJ software differences <br /> laboratory Cancer Center Massachusetts General                    assessed Student T-test. <br /> Hospital (Table S4). RD cells infected viral particles <br /> for 24 hours 37 degrees polybrene (Millipore) 4 mg/              Chemical treatment zebrafish ERMS <br /> mL selected puromycin (In Vivo Gene) 10 mg/                   Six-week old CG1 syngeneic fish transplanted 36104 <br /> mL 10%FBS/DMEM 15 days obtain stable lines. Total                 unsorted tumor cells arising dsRED-positive ERMS from <br /> cell lysates knockdown experiments immunoblotted                  CG1 strain fish (Mizgireuv Revskoy 2006; Smith et al., 2010). <br /> using primary antibodies HOXC6 (1:500), CCND2                       Engrafted animals treated 6-days post-transplantation with <br /> (1:1000), PLXNA1 (1:1000) VEGFA (1:1000). All Western                   100 nM cediranib (Selleck) vehicle control (DMSO) 7 <br /> analysis completed times experiment average               days (including 2 24-hr drug holidays). Tumor volume was <br /> percent knockdown noted. Incubation HRP-conjugated                  assessed imaging animals pre-treatment post-treatment. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   12                        August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                    Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br /> Tumor volume calculated multiplying tumor area                    control siRNA treatment Student&#226;&#8364;&#8482;s t-test (p,0.05). Cell-titer <br /> fluorescent intensity using image J. A Student&#226;&#8364;&#8482;s T-test                 glo assay showing OD change 3 days RD (C&#226;&#8364;&#8220;E) SMS- <br /> performed assess differences tumor size control           CTR (F&#226;&#8364;&#8220;H) cells transfected individual gene-specific <br /> and experimental groups.                                                    siRNAs. Each error bar indicates standard deviation triplicate <br />                                                                             experiments. <br /> Estimating microvessel density                                              (PDF) <br />    Six-week old fli1-GFP fish irradiated 25 Gy                  Figure S5 Knockdown CCND2, HOXC6, PLXNA1 and <br /> transplanted 36104 unsorted ERMS cells fish                  VEGFA did affect apoptosis. Annexin V analysis was <br /> dsRED-positive ERMS. Fish engrafted tumors treated                performed RD SMS-CTR cells transfected siRNA <br /> with cediranib described above. At end treatment period,          targeted CCND2, HOXC6, PLXNA1 VEGFA. Each <br /> tumor tissues isolated, fixed 4% paraformaldehyde               error bar indicates standard deviation triplicate experiments. <br /> 30 minutes snap frozen. 5 mM Frozen sections mounted <br />                                                                             (PDF) <br /> in DAPI-containing Vectashield (Invitrogen). GFP dsRED <br /> images obtained 2006 magnification using Nikon                    Figure S6     Knockdown PLXNA1 results impaired <br /> confocal microscope. Microvessel density quantified using               migration wound-healing assay. Cells transfected siRNA <br /> Weidner et al. criteria [49] differences assessed Student T-         shRNA allowed migrate scratch wound over <br /> test.                                                                       22 hours. Representative images cells control siRNA, two <br />                                                                             gene-specific PLXNA1 siRNAs, control shRNA gene- <br /> Kaplan-Meier analysis                                                       specific shRNAs 0 hr (A&#226;&#8364;&#8220;F) 22 hrs (G&#226;&#8364;&#8220;L) shown. Scale <br />    Kaplan-Meier analysis completed using R survival            bar indicates 50 mm. (M) Summary assessing PLXNA1 <br /> package. Median expression level gene used group            knockdown wound healing assays using gene-specific <br /> samples high low expression. Chi-squared tests used           siRNAs RD SMS-CTR cell lines. Eight random <br /> to assess overall survival differences groups. Statistical          measurements time point siRNA. <br /> significance defined p-value 0.05.                       The ratio wound closure determined difference in <br />                                                                             distance migrated total distance. Each error bar denotes <br /> Supporting Information                                                      standard deviation. (N) Summary wound healing assays using <br />                                                                             gene-specific PLXNA1 shRNAs. <br /> Figure S1 Validation copy number changes CNAs                     (PDF) <br /> identified array CGH zebrafish ERMS. A. Quantitative <br /> PCR performed genomic DNA extracted represen-                   Figure S7 HoxC6 differentially expressed ERMS when <br /> tative tumor/matched normal tissues. Each tumor sample                  compared ARMS normal muscle. Wisker plot showing <br /> normalized matched normal tissue. Each error bar indicates               relative mRNA expression levels HOXC6 human RMS in <br /> standard deviation triplicate experiments. B. Corresponding            comparison juvenile adult muscle assessed by <br /> array CGH analysis showing CNA calls based log2 scale.                 microarray gene expression. Asterisk denotes statistical significance <br /> (PDF)                                                                       based Student&#226;&#8364;&#8482;s t-test (p,0.001). PAX3-FKHR+ ARMS (P3+ <br />                                                                             ARMS), PAX7-FKHR+ ARMS (P7+ ARMS). <br /> Figure S2 BRAF gene CNA interval            (PDF) <br /> upregulated human ERMS compared normal muscle. Wisker <br /> plots showing relative mRNA expression levels BRAF (A), CRY1             Figure S8 Kaplan-Meier analysis correlating expression levels of <br /> (B) TTNT1 (C) human ERMS ARMS PAX3-                         CCND2 PLXNA1 clinical survival. Kaplan-Meier analysis <br /> FKHR fusion PAX7-FKHR fusion comparison juvenile                completed using microarray data Davicioni et al (2010) <br /> muscle assessed microarray gene expression.                           correlate expression levels CCND2 PLXNA1 clinical <br /> (PDF)                                                                       survival. Comparison RMS patients, ERMS <br />                                                                             patients only, ARMS patients non-RMS patients. (A) <br /> Figure S3 Knockdown candidate genes inhibition                    CCND2. (B) PLXNA1. <br /> VEGFA activity ERMS cell lines. (A) ELISA assess VEGFA                (PDF) <br /> protein level growth media transfected cells. (B) qRT-PCR to <br /> assess knockdown mRNA levels candidate genes. (Error bars             Figure S9     VEGFA transcript expression does correlate with <br /> indicate standard deviation triplicate                  clinical stage ERMS. Kaplan-Meier analysis completed <br /> experiment). Asterisks denote significant differences                  using microarray data Davicioni et al (2010). (A) Stage 1. (B) <br /> p,0.003 p,0.008, respectively comparison.                   Stage 2. (C) Stage 3. (D) Stage 4. Normalized mRNA expression <br /> (PDF)                                                                       levels VEGFA RMS (E), ERMS (F) ARMS (G) <br />                                                                             samples assessed box plots. <br /> Figure S4 Knockdown CCND2, HOXC6 PLXNA1 <br />                                                                             (PDF) <br /> results reduced growth human SMS-CTR RD cell lines. <br /> (A) Western analysis CCND2, HOXC6 PLXNA1                             Table S1 Summary zebrafish samples recurrent regional <br /> knockdowns SMS-CTR cell line using smart-pool (sp)                   gains deletions array CGH analysis. <br /> two individual gene-specific siRNAs. Bottom row: GAPDH.                     (PDF) <br /> Percentage knockdown siRNA follows: CCND2                    Table S2 Summary clinical information immunohisto- <br /> siRNA-sp, 98%; CCND2 siRNA#1, 57%; CCND2 siRNA#2,                           chemical staining results human RMS samples. <br /> 91%; HOXC6 siRNA-sp, 82%; HOXC6 siRNA#1, 88%; HOXC6 <br />                                                                             (PDF) <br /> siRNA#2, 75%; PLXNA1 siRNA-sp, 77%; PLXNA1 siRNA#1, <br /> 94%; PLXNA1 siRNA#2, 54%; VEGFA siRNA, 97%. (B)                             Table S3 Summary clinical information immunohisto- <br /> Summary cell-titer glo analysis SMS-CTR cell line. OD                 chemical staining results RMS samples Children&#226;&#8364;&#8482;s <br /> fold changes 3 days smart pool (sp) individual siRNAs          Oncology Group tissue microarray. <br /> are shown. Asterisk indicates statistical significance comparison        (PDF) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   13                        August 2013 | Volume 9 | Issue 8 | e1003727 <br />                                                                                                                         Novel Oncogenic Events Rhabdomyosarcoma <br />  <br />  <br /> Table S4 Summary shRNA sequences quantitative                                         Author Contributions <br /> primers used array CGH validation studies. <br />                                                                                              Conceived designed experiments: EYC KPD KHB CL DML SR. <br /> (PDF) <br />                                                                                              Performed experiments: EYC KPD KHB RC EE JYHC MSI JB. <br />                                                                                              Analyzed data: EYC KPD KHB SR DML CL CK MSI GPN. <br /> Acknowledgments                                                                              Contributed reagents/materials/analysis tools: EYC KPD KHB EE SR. <br />                                                                                              Wrote paper: EYC KPD KHB DML. <br /> Jason C. Dobrinski created chromosome design Fig. 1 Amy <br /> Chen provided helpful editorial editing. <br />  <br /> References <br />  1. Linabery AM, Ross JA (2008) Trends childhood cancer incidence U.S.             24. Merlino G, Helman LJ (1999) Rhabdomyosarcoma&#226;&#8364;&#8220;working pathways. <br />     (1992&#226;&#8364;&#8220;2004). Cancer 112: 416&#226;&#8364;&#8220;432.                                                            Oncogene 18: 5340&#226;&#8364;&#8220;5348. <br />  2. Chen Y, Takita J, Hiwatari M, Igarashi T, Hanada R, et al. (2006) Mutations           25. Tapscott SJ, Thayer MJ, Weintraub H (1993) Deficiency rhabdomyosarco- <br />     PTPN11 RAS genes rhabdomyosarcoma pediatric hematological                     mas factor required MyoD activity myogenesis. Science 259: 1450&#226;&#8364;&#8220; <br />     malignancies. Genes Chromosomes Cancer 45: 583&#226;&#8364;&#8220;591.                                          1453. <br />  3. Hettmer S, Liu J, Miller CM, Lindsay MC, Sparks CA, et al. (2011) Sarcomas               26. Lawson ND, Weinstein BM (2002) In vivo imaging embryonic vascular <br />     induced discrete subsets prospectively isolated skeletal muscle cells. Proc            development using transgenic zebrafish. Dev Biol 248: 307&#226;&#8364;&#8220;318. <br />     Natl Acad Sci U S A 108: 20002&#226;&#8364;&#8220;20007.                                                    27. Raney RB, Anderson JR, Barr FG, Donaldson SS, Pappo AS, et al. (2001) <br />  4. Langenau DM, Keefe MD, Storer NY, Guyon JR, Kutok JL, et al. (2007) Effects                  Rhabdomyosarcoma undifferentiated sarcoma decades of <br />     RAS genesis embryonal rhabdomyosarcoma. Genes Dev 21: 1382&#226;&#8364;&#8220;                     life: selective review intergroup rhabdomyosarcoma study group experience <br />     1395.                                                                                        rationale Intergroup Rhabdomyosarcoma Study V. J Pediatr Hematol <br />  5. Paulson V, Chandler G, Rakheja D, Galindo RL, Wilson K, et al. (2011) High-                  Oncol 23: 215&#226;&#8364;&#8220;220. <br />     resolution array CGH identifies common mechanisms drive embryonal                   28. Yang XR, Ng D, Alcorta DA, Liebsch NJ, Sheridan E, et al. (2009) T <br />     rhabdomyosarcoma pathogenesis. Genes Chromosomes Cancer 50: 397&#226;&#8364;&#8220;408.                         (brachyury) gene duplication confers major susceptibility familial chordoma. <br />  6. Bridge JA, Liu J, Qualman SJ, Suijkerbuijk R, Wenger G, et al. (2002) Genomic                Nat Genet 41: 1176&#226;&#8364;&#8220;1178. <br />     gains losses similar genetic histologic subsets rhabdomyosar-          29. Lahortiga I, De Keersmaecker K, Van Vlierberghe P, Graux C, Cauwelier B, et <br />     coma, amplification predominates embryonal anaplasia                     al. (2007) Duplication MYB oncogene T cell acute lymphoblastic <br />     alveolar subtypes. Genes Chromosomes Cancer 33: 310&#226;&#8364;&#8220;321.                                     leukemia. Nat Genet 39: 593&#226;&#8364;&#8220;595. <br />  7. Goldstein M, Meller I, Issakov J, Orr-Urtreger A (2006) Novel genes implicated           30. Ng CK, Cooke SL, Howe K, Newman S, Xian J, et al. (2011) The role of <br />     embryonal, alveolar, pleomorphic rhabdomyosarcoma: cytogenetic                  tandem duplicator phenotype tumour evolution high-grade serous ovarian <br />     molecular analysis primary tumors. Neoplasia 8: 332&#226;&#8364;&#8220;343.                                  cancer. J Pathol 226: 703&#226;&#8364;&#8220;12. <br />  8. Missiaglia E, Selfe J, Hamdi M, Williamson D, Schaaf G, et al. (2009) Genomic            31. McCabe CD, Spyropoulos DD, Martin D, Moreno CS (2008) Genome-wide <br />     imbalances rhabdomyosarcoma cell lines affect expression genes frequently              analysis homeobox C6 transcriptional network prostate cancer. Cancer <br />     altered primary tumors: approach identify candidate genes involved               Res 68: 1988&#226;&#8364;&#8220;1996. <br />     tumor development. Genes Chromosomes Cancer 48: 455&#226;&#8364;&#8220;467.                                 32. Musgrove EA, Caldon CE, Barraclough J, Stone A, Sutherland RL (2011) <br />  9. Pandita A, Zielenska M, Thorner P, Bayani J, Godbout R, et al. (1999)                        Cyclin D therapeutic target cancer. Nat Rev Cancer 11: 558&#226;&#8364;&#8220;572. <br />     Application comparative genomic hybridization, spectral karyotyping,              33. Buschges R, Weber RG, Actor B, Lichter P, Collins VP, et al. (1999) <br />     microarray analysis identification subtype-specific patterns genomic            Amplification expression cyclin D genes (CCND1, CCND2 CCND3) <br />     changes rhabdomyosarcoma. Neoplasia 1: 262&#226;&#8364;&#8220;275.                                           human malignant gliomas. Brain Pathol 9: 435&#226;&#8364;&#8220;442; discussion 432&#226;&#8364;&#8220;433. <br /> 10. Davari P, Hebert JL, Albertson DG, Huey B, Roy R, et al. (2010) Loss Blm <br />                                                                                              34. Kruger RP, Aurandt J, Guan KL (2005) Semaphorins command cells move. <br />     enhances basal cell carcinoma rhabdomyosarcoma tumorigenesis in <br />                                                                                                  Nat Rev Mol Cell Biol 6: 789&#226;&#8364;&#8220;800. <br />     Ptch1+/2 mice. Carcinogenesis 31: 968&#226;&#8364;&#8220;973. <br />                                                                                              35. Negishi M, Oinuma I, Katoh H (2005) Plexins: axon guidance signal <br /> 11. Rubin BP, Nishijo K, Chen HI, Yi X, Schuetze DP, et al. (2011) Evidence an <br />                                                                                                  transduction. Cell Mol Life Sci 62: 1363&#226;&#8364;&#8220;1371. <br />     unanticipated relationship undifferentiated pleomorphic sarcoma and <br />     embryonal rhabdomyosarcoma. Cancer Cell 19: 177&#226;&#8364;&#8220;191.                                     36. Granziero L, Circosta P, Scielzo C, Frisaldi E, Stella S, et al. (2003) CD100/ <br />                                                                                                  Plexin-B1 interactions sustain proliferation survival normal leukemic <br /> 12. Goessling W, North TE, Loewer S, Lord AM, Lee S, et al. (2009) Genetic <br />                                                                                                  CD5+ B lymphocytes. Blood 101: 1962&#226;&#8364;&#8220;1969. <br />     interaction PGE2 Wnt signaling regulates developmental specification of <br />     stem cells regeneration. Cell 136: 1136&#226;&#8364;&#8220;1147.                                        37. Valente G, Nicotra G, Arrondini M, Castino R, Capparuccia L, et al. (2009) Co- <br /> 13. Lam SH, Wu YL, Vega VB, Miller LD, Spitsbergen J, et al. (2006) Conservation                 expression plexin-B1 Met human breast ovary tumours enhances <br />     gene expression signatures zebrafish human liver tumors                   risk progression. Cell Oncol 31: 423&#226;&#8364;&#8220;436. <br />     tumor progression. Nat Biotechnol 24: 73&#226;&#8364;&#8220;75.                                             38. Ye S, Hao X, Zhou T, Wu M, Wei J, et al. (2010) Plexin-B1 silencing inhibits <br /> 14. Liu S, Leach SD (2011) Zebrafish models cancer. Annu Rev Pathol 6: 71&#226;&#8364;&#8220;93.                ovarian cancer cell migration invasion. BMC Cancer 10: 611. <br /> 15. Patton EE, Widlund HR, Kutok JL, Kopani KR, Amatruda JF, et al. (2005)                   39. Catalano A, Lazzarini R, Di Nuzzo S, Orciari S, Procopio A (2009) The plexin- <br />     BRAF mutations sufficient promote nevi formation cooperate                   A1 receptor activates vascular endothelial growth factor-receptor 2 nuclear <br />     p53 genesis melanoma. Curr Biol 15: 249&#226;&#8364;&#8220;254.                                       factor-kappaB mediate survival anchorage-independent growth of <br /> 16. White RM, Cech J, Ratanasirintrawoot S, Lin CY, Rahl PB, et al. (2011)                       malignant mesothelioma cells. Cancer Res 69: 1485&#226;&#8364;&#8220;1493. <br />     DHODH modulates transcriptional elongation neural crest                       40. Kaseb AO, Morris JS, Hassan MM, Siddiqui AM, Lin E, et al. (2011) Clinical <br />     melanoma. Nature 471: 518&#226;&#8364;&#8220;522.                                                               prognostic implications plasma insulin-like growth factor-1 vascular <br /> 17. Stratton MR, Fisher C, Gusterson BA, Cooper CS (1989) Detection point                     endothelial growth factor patients hepatocellular carcinoma. J Clin <br />     mutations N-ras K-ras genes human embryonal rhabdomyosarcomas                      Oncol 29: 3892&#226;&#8364;&#8220;3899. <br />     using oligonucleotide probes polymerase chain reaction. Cancer Res 49:           41. Maae E, Olsen DA, Steffensen KD, Jakobsen EH, Brandslund I, et al. (2012) <br />     6324&#226;&#8364;&#8220;6327.                                                                                   Prognostic impact placenta growth factor vascular endothelial growth <br /> 18. Freeman JL, Ceol C, Feng H, Langenau DM, Belair C, et al. (2009)                             factor A patients breast cancer. Breast Cancer Res Treat 133: 257&#226;&#8364;&#8220; <br />     Construction application zebrafish <br /> </body></html>