<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Activation Essential Calcium Signaling Pathway in <br /> Saccharomyces cerevisiae Kch1 Kch2, Putative Low-Affinity <br /> Potassium Transporters <br /> Christopher P. Stefan,a Nannan Zhang,a Takaaki Sokabe,b Alberto Rivetta,c Clifford L. Slayman,c Craig Montell,b Kyle W. Cunninghama <br /> Department Biology, Johns Hopkins University, Baltimore, Maryland, USAa; Departments Biological Chemistry Neuroscience, Center Sensory Biology, The <br /> Johns Hopkins University School Medicine, Baltimore, Maryland, USAb; Department Cellular Molecular Physiology, Yale School Medicine, New Haven, <br /> Connecticut, USAc <br />  <br /> In budding yeast Saccharomyces cerevisiae, mating pheromones activate high-affinity Ca2&#226;&#180;&#8482; influx (HACS) acti- <br /> vates calcineurin essential cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>survival</span>. Here identify extracellular K&#226;&#180;&#8482; homologous pair transmembrane <br /> proteins, Kch1 Kch2 (Prm6), necessary components HACS activation mechanism. Expression Kch1 espe- <br /> cially Kch2 strongly induced response mating pheromones. When forcibly overexpressed, Kch1 Kch2 lo- <br /> calized plasma membrane activated HACS fashion depended extracellular K&#226;&#180;&#8482; pheromones. They <br /> also promoted growth trk1 trk2 mutant cells low K&#226;&#180;&#8482; environments, suggesting promote K&#226;&#180;&#8482; uptake. Voltage-clamp <br /> recordings protoplasts revealed diminished inward K&#226;&#180;&#8482; currents kch1 kch2 double-mutant cells relative wild type. <br /> Conversely, heterologous expression Kch1 HEK293T cells caused appearance inwardly rectifying K&#226;&#180;&#8482; currents. Col- <br /> lectively, findings suggest Kch1 Kch2 directly promote K&#226;&#180;&#8482; influx HACS electrochemically respond <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> K&#226;&#180;&#8482; influx way homologous voltage-gated Ca2&#226;&#180;&#8482; channels animal cell types. <br />  <br />  <br />  <br />      a2&#226;&#171;&#185; key messenger helps translate extracellular stimuli                analogous subunits mammalian L-type voltage-gated Ca2&#226;&#171;&#185; <br /> C    intracellular responses. Controlled increases cytosolic <br /> free Ca2&#226;&#171;&#185; concentrations ([Ca2&#226;&#171;&#185;]cyt) regulate numerous pro- <br />                                                                                       channels (10, 11, 13&#226;&#8364;&#8220;15). Cch1, homolog catalytic &#226;?&#163;1- <br />                                                                                       subunit voltage-gated Ca2&#226;&#171;&#185; channels (VGCCs), contains posi- <br /> cesses, including membrane fusion, cytoskeleton organization,                         tively charged residues putative voltage-sensing S4 seg- <br /> gene transcription, cell polarization, cell death. Saccharomyces                  ments (10, 11), suggesting HACS, too, regulated by <br /> cerevisiae cells elevate [Ca2&#226;&#171;&#185;]cyt using Ca2&#226;&#171;&#185;-selective channels                      membrane depolarization. Indeed, sudden rise extracellular <br /> that activate evolutionarily conserved signaling pathway                      pH, thought depolarize S. cerevisiae cell membranes <br /> that involves calmodulin, calmodulin-dependent protein kinases,                       (15, 16), leads rapid activation HACS elevation of <br /> and calmodulin-dependent protein phosphatases (calcineurin                            [Ca2&#226;&#171;&#185;]cyt. Hyphal cells yeast Candida albicans utilize <br /> [Cn]), highly expressed mammalian cells                          Cch1 responses electrical stimuli (17). However, coexpres- <br /> (reviewed reference 1).                                                            sion Cch1 Mid1 fungus Cryptococcus neoformans <br />    Increases [Ca2&#226;&#171;&#185;]cyt resulting activation Cn                      HEK293 cells produced currents largely insensitive to <br /> essential survival S. cerevisiae cells responding mating                    voltage narrow range tested (18). Therefore, yet <br /> pheromones (2). Haploid cells propagate &#226;?&#163;                        clear physiological HACS activation fungi involves <br /> mating types, secrete peptide mating pheromones, a-fac-                      membrane depolarization. <br /> tor &#226;?&#163;-factor, coordinate mating process (3). Enhanced                           S. cerevisiae cells grow best acidic media, employ an <br /> Ca2&#226;&#171;&#185; influx low-affinity Ca2&#226;&#171;&#185; influx (LACS)                        electrogenic H&#226;&#171;&#185;-ATPase pH homeostasis maintenance of <br /> and high-affinity Ca2&#226;&#171;&#185; influx (HACS) begins approx-                        large resting voltage, estimated close &#226;&#171;&#186;200 mV (19&#226;&#8364;&#8220;21). <br /> imately 40 min pheromone stimulation (4, 5). LACS active                     K&#226;&#171;&#185; contributes resting membrane voltage (22, 23), and <br /> in rich media depends transmembrane protein                           K&#226;&#171;&#185; uptake proliferating S. cerevisiae cells largely depends the <br /> resembles regulatory TARP-subunits ionotropic glutamate                        related K&#226;&#171;&#185; transporters Trk1 Trk2 (24). In addition, whole- <br /> receptors mammalian cells (6). HACS active rich                     cell patch recordings protoplasts trk1 trk2 mutant cells <br /> synthetic media usually required survival yeast cells                   demonstrated ensemble additional K&#226;&#171;&#185;-permeable <br /> during prolonged exposures mating pheromones (5). Similar                       &#226;&#8364;&#339;channels&#226;&#8364;? blocked divalent cations (25, 26). Al- <br /> HACS-deficient mutants, calmodulin- Cn-deficient cells                            formally proved function ion channels not <br /> slowly die exposed mating pheromones absence                        identified molecular level, low-affinity transporters <br /> mating partners (2, 5, 7&#226;&#8364;&#8220;11). This calcium signaling network <br /> therefore regulates cell death response stresses induced by <br /> mating pheromones. The homologous calcium network patho-                               Received 23 October 2012 Accepted 26 November 2012 <br /> genic yeasts molds regulates cell death response                           Published ahead print 30 November 2012 <br /> endoplasmic reticulum (ER) stresses azole-class antifungals                           Address correspondence Kyle W. Cunningham, kwc@jhu.edu. <br /> (12).                                                                                     Copyright &#194;&#169; 2013, American Society Microbiology. All Rights Reserved. <br />    In S. cerevisiae, HACS defined interacting                           doi:10.1128/EC.00299-12 <br /> proteins&#226;&#8364;&#8221;Cch1, Mid1, Ecm7&#226;&#8364;&#8221;which homologous or <br />  <br />  <br /> 204   ec.asm.org                                          Eukaryotic Cell   p. 204 &#226;&#8364;&#8220;214                                             February 2013 Volume 12 Number 2 <br />                                                                                                                                         Novel K Transporters Yeast <br />  <br />  <br />  <br /> TABLE 1 Yeast strains used studya                                              TABLE 2 Plasmids used study <br />                                                                           Source                                                                     Source or <br /> Strain                 Description genotype                            reference    Plasmid                  Description                             reference <br /> K432                   bar1::hisG                                         50           pJB207                   2&#226;?&#174; URA3 FUS1::lacZ                      29 <br /> NZY164                 bar1::hisG kch2::G418                              This study   pEVP11/AEQ89             2&#226;?&#174; LEU2 PADH-aequorin                   30 <br /> NZY138                 bar1::hisG kch1::TRP1                              This study   pSM10                    CEN LEU2 PMET25-HA3                     31 <br /> NZY165                 bar1::hisG kch1::TRP1 kch2::G418                   This study   pCS01                    CEN LEU2 PMET25-KCH2-HA3                This study <br /> EMY113                 bar1::hisG cch1::TRP1                              This study   pCS02                    CEN LEU2 PMET25-KCH1-HA3                This study <br /> K601                   Wild type                                          33           pCS42                    CEN URA3 PMET25-HA3                     This study <br /> CS12                   Wild type                                          This study   pCS43                    CEN URA3 PMET25-KCH2-HA3                This study <br /> CS01                   kch2::G418                                         This study   pCS44                    CEN URA3 PMET25-KCH1-HA3                This study <br /> CS02                   kch1::TRP1                                         This study   pEG311                   URA3 PTDH3-GFP                          51 <br /> CS03                   kch1::TRP1 kch2::G418                              This study   pCS47                    URA3 PTDH3-GFP-KCH2-HA3                 This study <br /> CS04                   cch1::HIS3                                         This study   pCS48                    URA3 PTDH3-GFP-KCH1-HA3                 This study <br /> CS05                   cch1::HIS3 kch2::G418                              This study   pEGFP-C3                 PCMB-EGFP                               Clontech <br /> CS06                   cch1::HIS3 kch1::TRP1                              This study   pCS49                    PCMB-EGFP-KCH2                          This study <br /> CS07                   cch1::HIS3 kch1::TRP1 kch2::G418                   This study   pCS50                    PCMB-EGFP-KCH1                          This study <br /> CS30                   trk1::LEU2 trk2::HIS3                              This study <br /> CS163                  trk1::LEU2 trk2::HIS3 mid1::G418                   This study <br /> CS34                   KCH2-MYC13::HIS3                                   This study <br /> CS83                   KCH1-MYC13::HIS3                                   This study   pEG311 yield plasmids pCS47 pCS48. Plasmids pEG311, <br /> a <br />  All strains isogenic derivatives strain W303-1A (MATa ade2-1 can1-100   pCS47, pCS48 linearized BglII transformation. <br /> his3-11,14 leu2-3,112 trp1-1 ura3-1) W&#226;&#338;&#172;3 (trk1::LEU2 trk2::HIS3 derivative      Genomic KCH2 KCH1 sequences PCR amplified linkers <br /> W303-1A) (22, 23).                                                                     using forward primers KCH2-PstI-F KCH1-PstI-F reverse <br />                                                                                        primers KCH2-XmaI-R KCH1-XmaI-R cloned into <br />                                                                                        pEGFP-C3 digestion PstI XmaI yield plasmids pCS49 <br /> are thought supply K&#226;&#171;&#185; necessary proliferation trk1                       pCS50. <br /> trk2 double-<span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>knockout</span> mutants grown limiting K&#226;&#171;&#185; (25, 26).                                 45 <br />                                                                                                Ca2&#226;&#180;&#8482; uptake assays. Total cellular accumulation Ca2&#226;&#171;&#185; mea- <br />     In present study, previously uncharacter-                      sured described previously (32). Briefly, yeast cultures grown to <br /> ized proteins, Kch1 Kch2, promote low-affinity K&#226;&#171;&#185;                            log phase SC YPD medium overnight, harvested, resuspended in <br /> uptake essential K&#226;&#171;&#185;-dependent activation HACS                        fresh SC-100 pH 4 medium spiked tracer 45CaCl2 (Perkin-Elmer) <br /> S. cerevisiae cells responding mating pheromones. Both pro-                         &#226;?&#163;-factor FK506, indicated figures. SC-100 pH 4 <br />                                                                                        medium similar SC medium, YNB component was <br /> teins localize distinct zones yeast plasma membrane and <br />                                                                                        replaced yeast nitrogen <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span> calcium magnesium (Sun- <br /> are induced response mating pheromones. They                        rise Scientific Products), pH lowered pH 4.0 using HCl, the <br /> promote <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>cell survival</span> presence mating pheromones. Kch1                       MgSO4 CaCl2 restored 0.5 g/liter 100 &#226;?&#174;M, respectively. <br /> and Kch2 define novel family fungus-specific pro-                       To maintain selection plasmids, uracil leucine omitted as <br /> teins promote K&#226;&#171;&#185; influx utilization.                                          necessary. The effects extracellular potassium sodium rubid- <br />                                                                                        ium Ca2&#226;&#171;&#185; accumulation tested resuspending log-phase <br /> MATERIALS AND METHODS                                                                  cells SC-100 pH 4 medium lacking sodium chloride potassium <br /> Yeast strains, plasmids, culture media, reagents. The S. cerevisiae                phosphate adding solutions KCl, NaCl, RbCl concentra- <br /> strains used present study (Table 1) obtained original                tions indicated figures. Cultures incubated 30&#194;&#176;C 96-well <br /> sources derived parental strain W303-1A means stan-                 filtration plates (Millipore MultiScreen HTS plates) glass tubes for <br /> dard genetic crosses PCR-based methods introducing knockout                     specified times, cells harvested filtration, washed four <br /> mutations epitope-tags (27). Yeast strains cultured rich yeast             times ice-cold buffer A (10 mM CaCl2, 5 mM Na-HEPES [pH 6.5]) <br /> extract-peptone-dextrose (YPD) medium synthetic SC medium (28)                      vacuum filtration <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (Millipore), dried room temperature. <br /> and shifted alternative media described below. Purified synthetic                The filters covered Microscint20 scintillation cocktail (Perkin- <br /> &#226;?&#163;-factor mating pheromone obtained Johns Hopkins Uni-                     Elmer), counted using TopCount NXT (Packard) Tri-Carb <br /> versity Synthesis Sequencing Facility dissolved dimethyl                2200 (Packard) liquid scintillation counter. The data fit sig- <br /> sulfoxide (DMSO). FK506 obtained Astellas Pharma dis-                     moidal (variable slope) equation using nonlinear regression (Prism). <br /> solved DMSO. Aqueous 45CaCl2 purchased MP Biosciences.                          Aequorin luminescence measurements. Yeast strains trans- <br />     The plasmids oligonucleotides used present study listed             formed aequorin expression plasmid pEVP11/AEQ89 (30) and <br /> in Tables 2 3. Plasmids pJB207 (FUS1-lacZ) pEVP11/AEQ89                        grown log phase SC minus leucine. Cells (at optical density 600 <br /> (ADH-aequorin) described previously (29, 30). Genomic KCH2                    nm [OD600] 1.0) harvested, washed, resuspended SC lacking <br /> KCH1 sequences PCR amplified linkers, using forward                      leucine supplemented coelenterazine (Molecular Probes), incu- <br /> primers KCH2-PstI-F KCH1-PstI-F reverse primers KCH2-                      bated room temperature 30 min. The loaded cells collected, <br /> SalI-R KCH1-SalI-R. They digested PstI SalI                 resuspended fresh YPD medium, recovered 90 min described <br /> cloned pSM10 (31) yield plasmids pCS01 pCS02. The KCH2-                    previously (5). The cells transferred fresh tubes, incubated at <br /> HA3 KCH1-HA3 genes subcloned p416MET25 diges-                      room temperature, read tube luminometer (LB9507; EG&amp;G Wallac), <br /> tion SmaI XhoI yield plasmids pCS43 pCS44. HA3                     injected equal volume YPD medium adjusted pH <br /> subcloned pSM10 p416MET25 using SalI XhoI enzymes,                    10.0 NaOH recording luminescence 0.2-s intervals. <br /> yield plasmid pCS42. Green fluorescent protein (GFP)-KCH2-HA3                           Western blotting. Cultures grown log phase 30&#194;&#176;C YPD <br /> GFP-KCH1-HA3 PCR amplified pCS35 pCS36 using                             medium, adjusted OD600 &#226;&#172;&#402;0.1, split aliquots. Sam- <br /> the forward primer MET25(GFP)-BglII-F reverse primer                           ples treated 1 &#226;?&#174;g FK506/ml, 25 &#226;?&#174;M &#226;?&#163;-factor, DMSO <br /> MET25(HA3)-XhoI-R, digested BamHI SalI, cloned                       control (Sigma-Aldrich) 3 h 30&#194;&#176;C processing. Processing <br />  <br />  <br /> February 2013 Volume 12 Number 2                                                                                                                   ec.asm.org 205 <br />  Stefan et al. <br />  <br />  <br />  <br /> TABLE 3 Oligonucleotides used study <br /> Name                                           Sequence (5=&#226;&#8364;&#8220;3=) <br /> KCH2-PstI-F                                    GACGACCTGCAGATGGAATCAAGTTTACAAAAATTG <br /> KCH2-SalI-R                                    GACGACGTCGACTATGTATTTATGTTTGTATTTCTG <br /> KCH1-PstI-F                                    GACGACCTGCAGATGTTTAACCATGATTGGAAGTAC <br /> KCH1-SalI-R                                    GACGACGTCGACAGTATAATTATATGTACGATCTTC <br /> KCH2-HindIII-R                                 GACGACAAGCTTTTATATGTATTTATGTTTGTATTTCTG <br /> KCH2-BamHI-F                                   GACGACGGATCCATGGAATCAAGTTTACAAAAATTG <br /> KCH1-HindIII-R                                 GACGACAAGCTTTTAAGTATAATTATATGTACGATCTTC <br /> KCH1-BamHI-F                                   GACGACGGATCCATGTTTAACCATGATTGGAAGTAC <br /> PMET25(GFP)-BglII-F                            GACGACAGATCTATGTCTAAAGGTGAAGAATTATTCAC <br /> PMET25(HA3)-XhoI-R                             GACGACCTCGAGCTAGCTAGTTCTAGAAGC <br /> KCH2-XmaI-R                                    GACGACCCCGGGTATGTATTTATGTTTGTATTTCTG <br /> KCH1-XmaI-R                                    GACGACCCCGGGAGTATAATTATATGTACGATCTTC <br /> KCH2-F1                                        ATTTTTTACCACATTGTCTAAGAGGGAAAGGAAAATGACAACAAAGCGCTAAATAcggatccccgggttaattaa <br /> KCH2-R1                                        GAGAATCGATTTACTTTTAATGCATTTTTGCAGTCAAAAAAAGCTTCGTTTACAcatcgatgaattcgagctcg <br /> KCH1-F1                                        GCAAACACTTTGGTAATTTCATGAATATTTGACTAGCAGGAACAGAAAATCAAAAcggatccccgggttaattaa <br /> KCH1-R1                                        TTTCTAATCACGTCTATCTAATTCATGAAATATGCAAAGGGCTTTTGCCAGCCTAgaattcgagctcgtttaaac <br />  <br />  <br />  <br />  <br /> involved centrifuging 1 OD600 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cells 4&#194;&#176;C, lysis cells tri-   0.2% glucose, plus 5 mM MES [morpholineethanesulfonic acid] titrated <br /> chloroacetic acid, extraction proteins sodium dodecyl sulfate          pH 7.2 Tris base). With occasional gentle mixing, protoplasts were <br /> (SDS) sample buffer, SDS-PAGE Western blotting de-             stable suspension hours. For use, 2 &#226;?&#174;l suspension was <br /> scribed previously (31). Blots probed anti-MYC monoclonal            pipetted the recording chamber allowed settle <br /> 9E10 antibodies (Covance).                                                     &#226;&#172;&#402;10 min, chamber filled carefully flushed with <br />     Growth measurements. Yeast strains transformed plasmids          stream standard recording buffer (150 mM KCl, 10 mM CaCl2, 5 <br /> pCS42, pCS43, pCS44 grown overnight saturation SC me-            mM MgCl2, plus 1 mM MES titrated pH 7.5 Tris base) remove <br /> dium lacking uracil methionine supplemented 100 mM KCl.           debris nonadherent cells. Patch pipettes pulled &#226;&#172;&#402;1.5 &#226;?&#174;m tip <br /> The cells harvested, washed twice distilled water, re-      diameter 1.2 mm borosilicate glass, heat-polished shape a <br /> suspended OD600 1. The cells serially diluted 10-fold         pointed dome, filled standard intracellular buffer (175 mM <br /> water, 7 &#226;?&#174;l dilution spotted pH 4.5 SDAP medium           KCl, 1 mM EGTA, 0.15 mM CaCl2 yield 100 nM free Ca2&#226;&#171;&#185;, 4 mM <br /> (lacking uracil methionine) 2% noble agar. The pH 4.5 SDAP              MgCl2, 4 mM K2ATP titrated pH 7.0 KOH). Visibly clean <br /> medium contained YNB trace elements vitamins, 2% glucose, 10 mM            protoplasts &#226;&#172;&#402;6 &#226;?&#174;m diameter selected bright-field illumi- <br /> arginine-HCl buffered pH 6.5 phosphoric acid, 200 &#226;?&#174;M CaCl2,            nation (&#226;&#171;&#187;400) picked chamber patch pipette <br /> and 200 &#226;?&#174;M MgSO4, described previously (25), additional          suction &#226;&#172;&#402;8 cm HO. Gigaseals formed usable cells &#226;&#172;&#402;5 <br /> HCl adjustment pH 4.5 50 100 mM KCl. The plates           min allowed stabilize 5 10 min. Whole-cell <br /> were photographed 3 days incubation 30&#194;&#176;C.                          recording obtained high-voltage pulse (750 mV, 100 &#226;?&#174;s) coupled <br />     &#226;?&#164;-Galactosidase assays. To measure activation mating            brief doubling suction pressure. Control voltage-clamp <br /> pheromone pathway, yeast strains transformed plasmid pJB207          protocol, collection current data, preliminary analysis carried <br /> bearing FUS1-lacZ reporter gene grown overnight log phase        EPC9 amplifier Pulse software (HEKA Elektronik, Lam- <br /> SC medium lacking leucine. Cells harvested split pH 4       brecht, Germany), interfaced PowerMac G4 microcomputer. The <br /> SC-100 medium containing 0 10 mM KCl presence absence          data collected 2 kHz filtered 250 Hz. <br /> 20 &#226;?&#174;M &#226;?&#163;-factor. The cells incubated 30&#194;&#176;C 1.5 h assayed         The control membrane voltage held &#226;&#171;&#186;40 mV, scans <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> &#226;?&#164;-galactosidase activity described previously (33).                         level generated using 2.5-s pulses 20 mV decrements (range, <br />     Microscopy. Wild-type yeast strains transformed GFP-             &#226;&#171;&#185;20 &#226;&#171;&#186;180 mV). Control scans run standard recording <br /> KCH1 GFP-KCH2 genes constitutive promoter grown                buffer, test scans run similar solution lacking CaCl2 <br /> overnight log phase SC medium. The cells treated 10 &#226;?&#174;M         MgCl2 supplemented 1 mM EGTA. Averaged currents all <br /> &#226;?&#163;-factor 1 h imaging. HEK293T cells cultured Dulbecco       scans voltage shown (see Fig. 6A, left panel) stacked traces, <br /> modified Eagle medium (Gibco) containing 10% fetal bovine serum,               negative (downward) currents indicating cation flow bath to <br /> transfected plasmids pCS49 pCS50 using Lipofectamine 2000             cell interior. Averaged currents kch1 kch2 mutant cells (see Fig. 6A, <br /> (Invitrogen), fixed 4% paraformaldehyde, stained DAPI              right panel) subtracted averaged currents wild-type <br /> (4=,6=-diamidino-2-phenylindole). Both cell types imaged 510         cells, generating noisy traces displayed Fig. 6C. These traces <br /> Meta Confocal LSM (Zeiss) using &#226;&#171;&#187;100 &#226;&#171;&#187;40 objective lenses, respec-         fitted using IGOR (WaveMetrics, Inc.) equation current as <br /> tively, processed using ImageJ.                                            follows: A0 &#226;&#171;&#185; A1&#194;&#183;[1 &#226;&#171;&#186; exp(&#226;&#171;&#186;&#226;?&#163;1&#194;&#183;time)] &#226;&#171;&#185;A2&#194;&#183;[1 &#226;&#171;&#186; exp(&#226;&#171;&#186;&#226;?&#163;2&#194;&#183;time)], keep- <br />     Electrophysiology yeast cells. Whole-cell patch-clamp measure-          ing reaction constants, &#226;?&#163;1 (separately) &#226;?&#163;2 six <br /> ments protoplasts kch1 kch2 double-deletion strain         traces. The summary current-voltage plots Fig. 6B obtained from <br /> NZY165 parent strain K432, previously described (34). Briefly,      grand averages 0.5 s averaged trace. <br /> the strains grown liquid YPD medium 25&#194;&#176;C orbital                  Electrophysiology using HEK293T cells. HEK239T cells trans- <br /> shaker, washed twice 50 mM KH2PO4 (pH 7.2) containing 0.2%                  fected pEGFP-C3 pCS50 cultured 24 36 h. Cells that <br /> &#226;?&#164;-mercaptoethanol, resuspended buffer plus 1.2 M sorbitol          expressed GFP subjected whole-cell patch clamp recordings. The <br /> and 0.6 U zymolyase 20T (catalog no. 320921; IMP Biomedicals, Inc.,         patch pipettes contained solution 140 mM KCl, 5 mM EGTA, 10 <br /> Irvine CA), incubated slow rocking 45 min 30&#194;&#176;C. The            mM K-HEPES (pH 7.4). The bath contained high divalent K&#226;&#171;&#185; <br /> resulting protoplasts spun low speed resuspended         buffer (130 mM KCl, 5 mM MgCl2, 10 mM CaCl2, 10 mM glucose, 10 mM <br /> stabilizing salt solution (220 mM KCl, 10 mM CaCl2, 5 mM MgCl2,            K-HEPES [pH 7.4]) low divalent K&#226;&#171;&#185; buffer (130 mM KCl, 1 mM <br />  <br />  <br /> 206   ec.asm.org                                                                                                                              Eukaryotic Cell <br />                                                                                                                                  Novel K Transporters Yeast <br />  <br />  <br />  <br /> EGTA, 10 mM glucose, 43 mM mannitol, 10 mM K-HEPES [pH 7.4]). <br /> After achieving whole-cell patch, membrane potential initially <br /> clamped &#226;&#171;&#186;40 mV, series voltage steps &#226;&#171;&#185;100 &#226;&#171;&#186;120 mV <br /> was applied 20 mV decrements 1 s. The data sampled 10 kHz <br /> and filtered 5 kHz analysis (Axopatch 200B amplifier pClamp <br /> 10 software; Molecular Devices). Current values nearly end each <br /> step pulse used plot I-V relationships. All values nor- <br /> malized cell capacitance (pF) represented averages &#226;&#171;&#190; <br /> the standard errors mean (SEM). Cells expressing GFP-Kch1 (n &#226;&#171;&#189; <br /> 19) analyzed 3 different days, control samples (n &#226;&#171;&#189; 9) were <br /> analyzed different days. For cation permeability experiments, bath <br /> contained high-divalent Na&#226;&#171;&#185; buffer (130 mM NaCl, 5 mM MgCl2, <br /> 10 mM CaCl2, 10 mM glucose, 10 mM K-HEPES [pH 7.4]) low- <br /> divalent Na&#226;&#171;&#185; buffer (140 mM NaCl, 1 mM EGTA, 10 mM glucose, 10 mM <br /> Na-HEPES [pH 7.4]). Voltage ramps (&#226;&#171;&#185;100 &#226;&#171;&#186;120 mV 3 s) were <br /> applied whole-cell patch. Reversal potentials obtained the <br /> I-V plots crossing point high- low-divalent buffers <br /> both GFP-Kch1-expressing cells control cells. The change rever- <br /> sal potential (&#226;&#338;&#172;Vrev) GFP-Kch1 cells control cells used to <br /> calculate ion selectivity follows: PNa/PK &#226;&#171;&#189; exp(&#226;&#338;&#172;VrevF/RT), F is <br /> the Faraday constant, R universal gas constant, T absolute <br /> temperature. <br />     Cell death measurements. Cultures grown overnight log <br /> phase SC medium 30&#194;&#176;C. At OD600 0.1, cells harvested, <br /> resuspended SC-100 medium containing 25 &#226;?&#174;M &#226;?&#163;-factor pres- <br /> ence absence 100 mM CaCl2, incubated 30&#194;&#176;C 7 h a <br /> flat-bottom 96-well dish (Becton Dickinson). For plasmid expression, <br /> strains transformed plasmids pSM10, pCS01, pCS02 and <br /> grown overnight saturation SC medium lacking leucine methi- <br /> onine. The cells diluted fresh medium allowed to <br /> recover. After recovery, 50 &#226;?&#174;M &#226;?&#163;-factor added, cells were <br /> incubated 30&#194;&#176;C 6 h flat-bottom 96-well dish (Becton Dickin- <br /> son). In experiments, preincubated cells (20 &#226;?&#174;l) mixed with <br /> 180 &#226;?&#174;l phosphate-buffered saline containing 1 &#226;?&#174;g propidium iodide/ <br /> ml, live dead cells counted automatically using 96-well <br /> flow cytometer FACSArray (Becton Dickinson). At 5,000 cells in <br /> each sample counted. <br />  <br /> RESULTS <br /> KCH1 KCH2 genes promote HACS activation physiolog- <br /> ical stimuli. HACS activity yeast measured uptake of <br /> 45 <br />    Ca2&#226;&#171;&#185; culture medium, particularly strong <br /> when Cn-dependent feedback inhibition eliminated by <br /> the addition FK506 Cn inhibitors (5, 35). Recently, a <br /> collection <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants screened HACS de- <br /> ficiencies responses different stimuli: ER <br /> stressor tunicamycin mating pheromone &#226;?&#163;-factor (15). <br /> Strong HACS deficiencies observed cch1, mid1, ecm7                 FIG 1 Kch1 Kch2 regulate HACS activation response &#226;?&#163;-factor mat- <br />                                                                                ing pheromone. 45Ca2&#226;&#171;&#185; uptake log-phase cultures S. cerevisiae strains <br /> mutants conditions reexamination data re-                 include combinations Kch1, Kch2, Cch1, secreted protease <br /> vealed mutants weaker HACS deficiencies. For                Bar1 (strains K432, NZY164, NZY138, NZY165, EMY113) measured after <br /> example, prm6 mutant exhibited partial HACS deficiency                4 h incubation SC-100 medium presence absence 10 &#226;?&#174;M <br /> response &#226;?&#163;-factor yjr054w mutant exhibited partial                &#226;?&#163;-factor (A) plus 0.2 &#226;?&#174;g FK506/ml (B), indicated. Averages three <br />                                                                                biological replicates (&#226;&#171;&#190; standard deviations [SD]) shown. (C) The <br /> HACS deficiency response tunicamycin. The homologous                     wild-type strain kch1, kch2, kch1 kch2 double mutants without <br /> YJR054w PRM6 genes, designated KCH1 KCH2,                    functional CCH1 gene (strains CS12, CS01, CS02, CS03, CS04, CS05, CS06, <br /> respectively, arose whole-genome duplication event                CS07) expressing cytoplasmic aequorin plasmid (pEVP11/AEQ89) <br /> ancestral species yeast (36) predicted encode closely            monitored luminescence YPD medium sudden <br /> related transmembrane proteins. To test genes                    jump pH 9. <br /> function redundantly HACS activation re- <br /> duced 45Ca2&#226;&#171;&#185; uptake consequence diminished sensitivity <br /> to mating pheromones, single- double-knockout mutants                  bar1 mutant behaved similarly parent strain, the <br /> were reconstructed pheromone-hypersensitive W303-1A                     isogenic kch2 bar1 double mutant kch1 kch2 bar1 triple <br /> bar1 mutant background assayed 45Ca2&#226;&#171;&#185; uptake                    mutant exhibited mild strong HACS deficiencies, respectively <br /> exposure &#226;?&#163;-factor FK506. Under conditions, kch1               (Fig. 1A B). 45Ca2&#226;&#171;&#185; uptake cch1 bar1 double mutant <br />  <br />  <br /> February 2013 Volume 12 Number 2                                                                                                            ec.asm.org 207 <br />  Stefan et al. <br />  <br />  <br />  <br /> was slightly kch1 kch2 bar1 triple mutant, <br /> indicating nearly complete loss Cn-sensitive HACS activ- <br /> ity double mutant. These findings suggest Kch1 and <br /> Kch2 proteins redundantly participate HACS activation during <br /> the response mating pheromones consequence of <br /> desensitization Bar1/SstI protease degrades &#226;?&#163;-factor. <br />     Sudden elevation environmental pH causes rapid acti- <br /> vation HACS elevation [Ca2&#226;&#171;&#185;]cyt levels independent of <br /> Cn, detected measurements aequorin luminescence (16). <br /> Surprisingly, kch1 kch2 double mutants single <br /> mutants exhibited wild-type levels aequorin luminescence after <br /> high-pH shock (Fig. 1C). The loss CCH1 strain <br /> backgrounds completely eliminated elevation of <br /> [Ca2&#226;&#171;&#185;]cyt (Fig. 1C). Thus, KCH1 KCH2 encode essen- <br /> tial subunits HACS instead encode regulators spe- <br /> cifically required HACS activation response stimulation <br /> with mating pheromones. <br />     Roles Kch1 Kch2 yeast cell death. HACS Cn are <br /> not required mating instead required survival of <br /> yeast cells responding mating pheromones unable <br /> to mates (2, 5, 7&#226;&#8364;&#8220;11). To determine KCH1 KCH2 <br /> are necessary survival prolonged episodes pher- <br /> omone signaling, cultures kch1 kch2 double mutant the <br /> single mutants exposed &#226;?&#163;-factor analyzed dead <br /> cells staining propidium iodide described previously <br /> (2). Both kch1 kch2 showed low levels cell death response <br /> to mating pheromone; however, kch1 kch2 double mutant <br /> exhibited large increase cell death (Fig. 2A). Moreover,      FIG 2 Kch1 Kch2 essential maintaining cell viability. (A) The <br /> pheromone-induced death kch1 kch2 double mutant             strains listed Fig. 1C exposed 25 &#226;?&#174;M &#226;?&#163;-factor SC-100 media with <br /> suppressed elevation extracellular Ca2&#226;&#171;&#185; concentrations      additional 100 mM CaCl2 stained propidium iodide. Live <br /> the addition 100 mM CaCl2 (Fig. 2A). These conditions allow         dead cells counted flow cytometry. (B) Strains kch1 kch2 cch1 <br /> Ca2&#226;&#171;&#185; leak cell pathways bypass neces-    kch1 kch2 (CS03 CS07) transformed control plasmid plas- <br />                                                                        mids overexpress KCH2-HA3 KCH1-HA3 genes (pSM10, pCS01, <br /> sity Cch1 Kch1/2. Overexpression Kch1 Kch2         pCS02), exposed 50 &#226;?&#174;M &#226;?&#163;-factor, analyzed pheromone- <br /> was able rescue viability kch1 kch2 double mutant    induced cell death panel A. The averages biological replicates (&#226;&#171;&#190; <br /> cch1 kch1 kch2 triple mutant (Fig. 2B). Collectively, data       SD) shown. <br /> suggest Kch1 Kch2 function upstream HACS that <br /> cell death double mutant results inability up <br /> Ca2&#226;&#171;&#185; extracellular environment activate calcium       response &#226;?&#163;-factor (37, 38), led original <br /> signaling pathway.                                                     naming KCH2 pheromone-regulated membrane protein 6 <br />     Expression, regulation, localization Kch1 Kch2.         (PRM6). The induced Kch1 Kch2 proteins migrated as <br /> The products KCH1 KCH2 predicted contain             doublets SDS-PAGE, changes gel mobility and <br /> least transmembrane spans (38). PSI-BLAST searches cur-         expression levels reproducibly altered addi- <br /> rent protein databases revealed homologous proteins encoded         tional exposure cells FK506. These results suggest that <br /> the genomes nearly fungi eukaryotic         upregulated expression Kch1 Kch2 mediates activation <br /> prokaryotic species, suggesting fungus-specific gene family (data    HACS response mating pheromones Cn <br /> not shown). Multiple sequence alignments representative             likely inhibits HACS directly (39) inhibiting Kch1 or <br /> members Kch2 family proteins revealed strong amino           Kch2. <br /> acid sequence conservation N-terminal 290 amino acid            To determine spatial distributions Kch1 Kch2 the <br /> residues Kch1 Kch2, included predicted         cell, proteins tagged GFP N termini and <br /> transmembrane segments (data shown). The C-terminal tails          overexpressed constitutive promoter. Both GFP-tagged <br /> were hydrophilic highly variable <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> sequence.           proteins able prevent cell death expressed kch1 <br /> None proteins Kch1 family studied previ-       kch2 mutant cells (data shown), suggesting fusion pro- <br /> ously.                                                                 teins functional. GFP-Kch1 fluorescence localized the <br />     To study expression patterns Kch1 Kch2, MYC13       plasma membrane region log-phase cells, slight enrich- <br /> epitope tag inserted C termini proteins         ment growing bud, compared mother cell body (Fig. <br /> homologous recombination yeast genome. Western blots          3B). GFP-Kch2 fluorescence localized distal tip grow- <br /> of whole-cell lysates showed expression Kch1 Kch2 pro-          ing buds vacuole lumen (Fig. 3C). The simplest inter- <br /> teins enhanced cells exposed &#226;?&#163;-factor (Fig. 3A). In    pretation findings Kch2 initially delivered sites <br /> absence &#226;?&#163;-factor, basal expression Kch2 undetectable.        cell growth subsequently endocytosed trafficked to <br /> Transcription KCH1 KCH2 genes strongly induced          vacuole, likely different rates regions the <br />  <br />  <br /> 208   ec.asm.org                                                                                                                        Eukaryotic Cell <br />                                                                                                                                      Novel K Transporters Yeast <br />  <br />  <br />  <br />  <br />                                                                                   FIG 4 K&#226;&#171;&#185; essential Ca2&#226;&#171;&#185; accumulation response &#226;?&#163;-factor <br />                                                                                   mating pheromone. (A) Kch1/2-dependent 45Ca2&#226;&#171;&#185; uptake determined in <br />                                                                                   SC-100 medium containing indicated concentrations KCl subtract- <br />                                                                                   ing values obtained kch1 kch2 double-mutant cultures (CS03) from <br /> FIG 3 Expression localization Kch1 Kch2 yeast cells responding <br />                                                                                   obtained wild-type cultures (K601) 4 h exposure to <br /> to &#226;?&#163;-factor. (A) Western blots whole-cell lysates wild-type yeast strains <br />                                                                                   25 &#226;?&#174;M &#226;?&#163;-factor plus 0.2 &#226;?&#174;g FK506/ml. (B) &#226;?&#164;-Galactosidase activity wild- <br /> (K601) expressing Kch2-MYC13 Kch1-MYC13 chromosomal loci <br />                                                                                   type cells (K601) transformed FUS1-lacZ reporter gene (pJB207) in <br /> (CS34 CS83) presence absence 1 &#226;?&#174;g FK506/ml 25 &#226;?&#174;M <br />                                                                                   SC-100 medium presence absence 25 &#226;?&#174;M &#226;?&#163;-factor 10 mM KCl. <br /> &#226;?&#163;-factor. Wild-type yeast cells (K601) expressing GFP-KCH1 (B) GFP- <br />                                                                                   Bars indicate averages (arithmetic means) biological replicates (&#226;&#171;&#190; <br /> KCH2 (C) constitutive plasmids (pCS47, pCS48) imaged ex- <br />                                                                                   SD). <br /> posure 10 &#226;?&#174;M &#226;?&#163;-factor solvent. Pseudocolored images panel B depict <br /> the scale pixel intensity. <br />  <br />  <br />                                                                                   lular K&#226;&#171;&#185; (Fig. 4A). However, removal K&#226;&#171;&#185; did block other <br /> plasma membrane. This interpretation supported                       cellular responses &#226;?&#163;-factor, expression phero- <br /> broader distribution GFP-Kch2 fluorescence plasma                       mone-responsive FUS1-lacZ reporter gene (Fig. 4B). Thus, extra- <br /> membranes endocytosis-deficient mutant (end3 [unpub-                        cellular K&#226;&#171;&#185; specifically required HACS activation re- <br /> lished observations]). In shmoo-shaped cells responding &#226;?&#163;-fac-                 sponse mating pheromone. <br /> tor, GFP-Kch1 fluorescence concentrated growing pro-                       Yeast cells normally accumulate K&#226;&#171;&#185; high-affinity trans- <br /> jection, GFP-Kch2 fluorescence localized tip                  porters, termed Trk1 Trk2 (24), required pro- <br /> the projection vacuole lumen.                                   liferation low-K&#226;&#171;&#185; culture media. Knockout mutations de- <br />     Kch1 Kch2 promote K&#226;&#180;&#8482; uptake K&#226;&#180;&#8482;-dependent acti-                        stroy Kch1 Kch2 functions did detectably exacerbate the <br /> vation HACS. Given high sequence similarities                      K&#226;&#171;&#185;-requirements strains lacking Trk1 Trk2 (data not <br /> Cch1 catalytic subunits mammalian voltage-gated Ca2&#226;&#171;&#185;                   shown). However, low-pH media designed block residual K&#226;&#171;&#185; <br /> channels (10, 11), supposed activation HACS                 uptake (25), overexpression Kch1 Kch2 trk1 trk2 <br /> response mating pheromones likely involve electrical de-                 double-mutant cells significantly improved growth relative to <br /> polarization cell membrane. To test HACS activity                  controls (Fig. 5A). Similar growth enhancement observed <br /> requires influx K&#226;&#171;&#185; Na&#226;&#171;&#185;, measured 45Ca2&#226;&#171;&#185; uptake                  trk1 trk2 mid1 triple mutants, implying Kch1 Kch2 can <br /> wild-type kch1 kch2 mutant yeast cells responding &#226;?&#163;-factor                 promote K&#226;&#171;&#185; uptake independently HACS. <br /> and FK506, medium lacking cations. Omission K&#226;&#171;&#185;                        In absence mating pheromones presence of <br /> and Na&#226;&#171;&#185; media resulted impairment HACS                          FK506, overexpressed Kch1 Kch2 proteins stimulated <br />                                                                                   45 <br /> activity complemented addition extracel-                     Ca2&#226;&#171;&#185; uptake wild-type cells cch1 mutant cells (Fig. <br />  <br />  <br /> February 2013 Volume 12 Number 2                                                                                                                 ec.asm.org 209 <br />  Stefan et al. <br />  <br />  <br />  <br />  <br /> FIG 5 Kch1 Kch2 promote K&#226;&#171;&#185; uptake K&#226;&#171;&#185;-dependent activation HACS. (A) A trk1 trk2 double mutant trk1 trk2 mid1 triple mutant (CS30 and <br /> CS163) bearing control plasmid plasmids overexpressing KCH1-HA3 KCH2-HA3 genes (pCS42, pCS43, pCS44) serially diluted spotted <br /> onto SDAP pH 4.5 agar medium lacking methionine containing 100 50 mM KCl indicated. Colonies photographed 3 days incubation 30&#194;&#176;C. <br /> (B) 45Ca2&#226;&#171;&#185; uptake log-phase cultures kch1 kch2 double mutants cch1 kch1 kch2 triple mutants (CS03 CS08) transformed control plasmid or <br /> plasmids overexpressing KCH1-HA3 KCH2-HA3 genes, SC-100 medium supplemented 1 &#226;?&#174;g FK506/ml incubated 1.5 h. (C) 45Ca2&#226;&#171;&#185; uptake <br /> into kch1 kch2 double mutants expressing KCH1-HA3 KCH2-HA3 genes performed SC-100 media supplemented 0.2 &#226;?&#174;g of <br /> FK506/ml plus indicated concentrations KCl, RbCl, NaCl. The data points normalized corresponding plasmid controls represent Kch1- and <br /> Kch2-dependent calcium uptake. Plots depict averages (&#226;&#171;&#190; SEM). <br />  <br />  <br />  <br /> 5B). This Kch1- Kch2-dependent HACS activation                     terials Methods). The rate constants&#226;&#8364;&#8221;7.73/s 0.21/s <br /> observed culture medium depleted K&#226;&#171;&#185; Na&#226;&#171;&#185; (Fig. 5C                 (corresponding half times 0.09 3.27 s)&#226;&#8364;&#8221;satisfied six <br /> D). Titration Na&#226;&#171;&#185;, K&#226;&#171;&#185;, Rb&#226;&#171;&#185; salts medium demon-                 tracings. The kch1 kch2 double-mutant cells appeared lack the <br /> strated K&#226;&#171;&#185; Rb&#226;&#171;&#185; 4- 40-fold effective                 faster evolving current. These findings Kch1 Kch2 <br /> Na&#226;&#171;&#185; restoring Kch2- Kch1-dependent activation HACS.                  necessary components NSC currents S. cerevisiae that <br /> Taken together, data suggest Kch1 Kch2 medi-                additional unidentified K&#226;&#171;&#185;-permeable channels transporters <br /> ate influx K&#226;&#171;&#185; external environment               exist. <br /> K&#226;&#171;&#185; influx essential activation HACS                 To test hypothesis Kch1 Kch2 directly promote K&#226;&#171;&#185; <br /> accumulation Ca2&#226;&#171;&#185;.                                                          flux membrane, expressed GFP-Kch1 and <br />     Kch1 Kch2 define novel family K&#226;&#180;&#8482; transporters. S.                 GFP-Kch2 fusion proteins human HEK293T cells carried <br /> cerevisiae cells display variety inwardly rectified cation cur-           whole-cell voltage-clamp measurements. GFP-Kch1 clearly <br /> rents, dubbed NSC currents (25, 26), identified <br />                                                                                localized cortex HEK293T cells (Fig. 7A), GFP- <br /> at molecular level. To determine Kch1 Kch2 <br />                                                                                Kch2 primarily localized punctae cell bodies was <br /> might contribute currents, kch1 kch2 double-knock- <br />                                                                                studied further. At &#226;&#171;&#186;120 mV, average inward current in <br /> out strain wild-type parent strain grown stan- <br /> dard conditions, protoplasted, analyzed whole-cell patch-               cells expressing GFP-Kch1 (34.9 &#226;&#171;&#190; 7.5 pA/pF; n &#226;&#171;&#189; 19) was <br /> clamping (see Materials Methods). The kch1 kch2 mutant cells               significantly higher control cells (13.5 &#226;&#171;&#190; 2.6 pA/pF; <br /> exhibited inward K&#226;&#171;&#185; currents negative membrane voltages                n &#226;&#171;&#189; 9). Similar total inward K&#226;&#171;&#185; currents wild-type S. <br /> were &#226;&#172;&#402;40% currents wild-type cells. This seen            cerevisiae cells (Fig. 6C), Kch1-dependent currents in <br /> by comparing individual downward traces Fig. 6A               HEK293T cells exhibited inward rectification (Fig. 7D) were <br /> the averaged steady-state values (plotted membrane volt-               blocked divalent cations bath (Fig. 7C). In addition, the <br /> age) Fig. 6B. Figure 6A demonstrates amplitudes            Kch1-dependent currents HEK293T cells equally perme- <br /> the sets currents evolved differently time. The implied              able Na&#226;&#171;&#185; versus K&#226;&#171;&#185; (PNa/PK &#226;&#171;&#189; 1.088), suggesting Kch1 <br /> difference tracings shown expanded ordinate scale,              induces nonselective cation channels assay conditions. <br /> in Fig. 6C, fitted (smooth red curves)             Collectively, findings support model Kch1 directly <br /> sum simple exponential functions plus offset (see Ma-                functions K&#226;&#171;&#185;-permeable transporter channel S. cerevi- <br />  <br />  <br /> 210   ec.asm.org                                                                                                                             Eukaryotic Cell <br />                                                                                                                                 Novel K Transporters Yeast <br />  <br />  <br />  <br />                                                                                     gesting Kch1 Kch2 incapable promoting Li&#226;&#171;&#185; <br />                                                                                     transport. Consistent view, overexpression nor <br />                                                                                     loss Kch1 Kch2 S. cerevisiae altered sensitivity Li&#226;&#171;&#185; in <br />                                                                                     medium (data shown). Overexpressed Kch1 Kch2 also <br />                                                                                     promoted growth trk1 trk2 double mutants limiting concen- <br />                                                                                     trations K&#226;&#171;&#185; acidic medium (Fig. 5A). Importantly, Kch1 and <br />                                                                                     Kch2 sole sources K&#226;&#171;&#185; uptake kch1 kch2 <br />                                                                                     trk1 trk2 quadruple mutants exhibited K&#226;&#171;&#185; requirements <br />                                                                                     trk1 trk2 double mutants (unpublished observations) and <br />                                                                                     kch1 kch2 double mutant retained significant K&#226;&#171;&#185; <br />                                                                                     currents (Fig. 6). Our observation Kch1 generate <br />                                                                                     K&#226;&#171;&#185; currents HEK293T cells selectivity, inhibitor sen- <br />                                                                                     sitivity, rectification properties resembling Kch1/2-de- <br />                                                                                     pendent currents S. cerevisiae argues strongly Kch1 and <br />                                                                                     Kch2 function catalytic subunit ion transporters or <br />                                                                                     channels. However, experiments necessary rule <br />                                                                                     remote possibility Kch1 regulates unidentified K&#226;&#171;&#185; <br />                                                                                     transporter conserved S. cerevisiae human cells. <br />                                                                                         Kch1 Kch2 exhibit strong sequence similarity an- <br />                                                                                     uncharacterized proteins present fungi. <br /> FIG 6 Kch1 Kch2 mediate inward K&#226;&#171;&#185; currents. (A) Averaged patch-                 No homologs family animals, plants, or <br /> clamp traces protoplasts wild-type strain (K432, n &#226;&#171;&#189; 5, left panel)     eukaryotic species. Many K&#226;&#171;&#185; channels bacteria eu- <br /> and kch1 kch2 double mutant (NZY165, n &#226;&#171;&#189; 6, right panel), recorded        karyotes function oligomers subunits six <br /> 10 min incubation Mg2&#226;&#171;&#185;- Ca2&#226;&#171;&#185;-free recording buffer (see Materials        transmembrane segments conserved ion selectivity filter <br /> Methods). (B) Current-voltage plots steady-state currents panel <br /> A, plus controls obtained protoplasts, 10 mM extra- <br />                                                                                     located final transmembrane segments (40). The <br /> cellular Ca2&#226;&#171;&#185;. Steady-state values calculated grand average data,     yeast Trk1 Trk2 K&#226;&#171;&#185; transporters contain internal re- <br /> the interval 2.0 2.5 s, trace. (C) Plotted differences        peats minimal channel domain present homote- <br /> corresponding traces panels panel A. These difference curves      trameric KcsA channels Streptomyces lividans (41). Tok1, a <br /> well fitted sum simple exponentials plus offset (see equation   bona fide, outwardly rectifying K&#226;&#171;&#185; channel yeast, appears to <br /> Materials Methods), pair rate constants plots: <br /> &#226;?&#163;1 &#226;&#171;&#189; 7.725 s, &#226;?&#163;2 &#226;&#171;&#189; 0.212 s, corresponding half times 90 ms 3.27       form dimer subunits contain internal repeats the <br /> s, respectively.                                                                    channel domain (42&#226;&#8364;&#8220;44). Each K&#226;&#171;&#185; channel domains con- <br />                                                                                     tains TVGYG-related sequence helps form ion selectivity <br />                                                                                     filter (41, 45). Kch1 homologs fungi contain <br /> siae physiological roles stimulus-dependent activation                  TVGYG-related sequence motifs conserved mo- <br /> of HACS.                                                                            tifs indicative channel transporter activity. A <br />                                                                                     better understanding transmembrane topology, oligo- <br /> DISCUSSION                                                                          meric states, hypothetical selectivity filter, transport and <br /> The evidence presented suggests Kch1 Kch2 activate                    gating kinetics, regulatory regions Kch1 Kch2 will <br /> HACS downstream components essential calcium sig-                        help resolve means novel family proteins <br /> naling pathway S. cerevisiae cells response mating                 promotes K&#226;&#171;&#185; influx fungal cells. <br /> pheromones ability transport K&#226;&#171;&#185;, likely                        Activation HACS Kch1 Kch2. We identified <br /> the partial depolarization plasma membrane. The mainte-                      kch1 kch2 mutants functional screen genes were <br /> nance Kch1-related proteins Cch1-related proteins                    required activation HACS response mating phero- <br /> other fungal species suggests fungi general utilize                     mones ER stress (15). We mating pheromones <br /> membrane depolarization regulation HACS activity, sim-                    induced expression Kch1 Kch2 overexpressed <br /> ilar mechanism mammalian cells regulate VGCCs.                      Kch1 Kch2 localized plasma membrane activated <br /> These findings implications discussed                   HACS independent pheromone stimulation. Thus, Kch1 and <br /> detail.                                                                             Kch2 play crucial roles HACS activation mating phero- <br />     Kch1 Kch2 define novel family ion transporters                      mones. In addition, extracellular K&#226;&#171;&#185; required HACS acti- <br /> fungi. Most K&#226;&#171;&#185; uptake S. cerevisiae occurs high-affinity                 vation conditions, higher concentrations Rb&#226;&#171;&#185; <br /> and moderate-affinity K&#226;&#171;&#185; transporters Trk1 Trk2 (24). Mu-                       Na&#226;&#171;&#185; partially replace K&#226;&#171;&#185; (Fig. 5). The apparently strong <br /> tant cells lack Trk1 Trk2 fulfill nutritional                   preference K&#226;&#171;&#185; Na&#226;&#171;&#185; observed electrophysiolog- <br /> requirements K&#226;&#171;&#185; using ensemble unidentified low-affin-                    ical recordings unstimulated S. cerevisiae protoplasts in <br /> ity K&#226;&#171;&#185; uptake systems distinguished electrophysiologi-                  HEK293T cells expressing Kch1. This discrepancy attrib- <br /> cally rates evolution voltage changes (e.g., instan-              uted factors. First, HACS activity assays per- <br /> taneous, fast, slow components), permeability Li&#226;&#171;&#185;,                     formed standard culture media simple buffers, and <br /> and sensitivities inhibitors (25, 26). Kch1 Kch2                  represent physiological state additional <br /> specifically required fast-evolving K&#226;&#171;&#185; influx current S.                 factors posttranslational modifications promote ion dis- <br /> cerevisiae cells (Fig. 6). The residual K&#226;&#171;&#185; currents kch1 kch2 cells              crimination. The ability Trk1 discriminate K&#226;&#171;&#185; Na&#226;&#171;&#185; dra- <br /> closely resembled Li&#226;&#171;&#185; currents wild-type cells (25, 26), sug-                matically depends signaling pathways respond environ- <br />  <br />  <br /> February 2013 Volume 12 Number 2                                                                                                           ec.asm.org 211 <br />  Stefan et al. <br />  <br />  <br />  <br />  <br /> FIG 7 Heterologous expression Kch1 HEK293T cells induces inward K&#226;&#171;&#185; currents. (A) HEK293T cells transfected GFP-KCH1 gene (pCS50) were <br /> fixed, stained DAPI, imaged microscopically. (B) Representative traces HEK293T cells transfected plasmid plasmid containing GFP-KCH1 <br /> gene (pEGFP-C3 pCS50) subjected whole-cell voltage-clamp measurements buffers containing high- low-divalent cations. (C D) Current <br /> voltage traces summarizing raw data panel B high (C)- low (D)-divalent cation buffers. The plots averages (&#226;&#171;&#190; SEM) 9 control cells (white <br /> symbols) 19 GFP-Kch1 expressing cells (gray symbols) normalization whole-cell capacitance. <br />  <br />  <br />  <br /> mental conditions (46). Second, K&#226;&#171;&#185; Na&#226;&#171;&#185; differentially                   potentially altering speed sensitivity responses to <br /> couple activation HACS chemical elec-               changes membrane voltage. Cch1-dependent ion currents have <br /> trical intermediates.                                                           directly recorded S. cerevisiae fungi. After <br />     The catalytic subunit HACS, Cch1, homologous                       coexpression Cch1 Mid1 homologs fungus Cryp- <br /> VGCCs mammals responsive changes transmem-                    tococcus neoformans HEK293 cells, Ca2&#226;&#171;&#185;-selective channel was <br /> brane electrical potential effects repeated voltage-            recorded, currents largely insensitive voltages <br /> sensing domains (47). Mammalian VGCCs generally contain 19                      narrow range tested (&#226;&#171;&#186;80 &#226;&#171;&#185;70 mV) (18). It pos- <br /> positioned lysine arginine voltage sensing &#226;&#8364;&#339;S4&#226;&#8364;? do-                  sible voltage sensitivity HACS-like fungal channel <br /> mains, Cch1 homologs fungi typically contain                 observed physiological membrane potentials for <br /> 11 12 lysine arginine residues sites (10, 11, 18),              fungi (approaching &#226;&#171;&#186;200 mV) cells coexpress the <br />  <br />  <br /> 212   ec.asm.org                                                                                                                               Eukaryotic Cell <br />                                                                                                                               Novel K Transporters Yeast <br />  <br />  <br />  <br /> regulatory &#226;?&#165;-subunit, recently identified S. cerevisiae      HACS shown necessary survival S. cerevisiae <br /> (15). Interestingly, reconstituted Cch1-Mid1 channel C.          Candida glabrata cells exposure azole-class com- <br /> neoformans voltage sensitive Ba2&#226;&#171;&#185; substituted             pounds (18, 35, 49). Although known ho- <br /> for Ca2&#226;&#171;&#185; charge carrier (18), potentially indicating          mologs Kch1 Cch1 contribute Cn activation survival <br /> feedback regulation Ca2&#226;&#171;&#185;. Although direct evidence         clinically relevant situations, compounds specifically in- <br /> that fungal HACS channels sense respond membrane de-               hibit Kch1 HACS fungi augment potency azole- <br /> polarization, sudden dissipation transmembrane H&#226;&#171;&#185; gradi-           class compounds antifungal therapies diminishing acti- <br /> ent led rapid activation HACS S. cerevisiae (16) C.          vation Cn downstream processes essential for <br /> albicans (48) fashion bypassed requirement Kch1         fungal cell survival. <br /> and Kch2 (Fig. 1). Although rule possibility that <br /> Kch1 Kch2 activate HACS complex mechanisms,               ACKNOWLEDGMENTS <br /> simplest interpretation available data Kch1 Kch2       This research supported grants GM053082 NS074072 to <br /> activate HACS electrical depolarization yeast plasma           K.W.C. EY10852 GM085335 C.M. National Institutes <br /> membrane associated K&#226;&#171;&#185; influx.                                       Health (NIH). A.R. C.L.S. supported NIH grant GM60696. <br />     K&#226;&#171;&#185; plays definite role regulating resting membrane               We thank Alonso Rodriguez-Navarro trk1 trk2 double-mutant <br /> voltage yeast, reflected relative hyperpolarization       yeast strain. <br /> the plasma membrane trk1 trk2 mutants relative                  A.R. C.L.S. performed analyzed patch-clamp studies yeast <br />                                                                           protoplasts. C.M. T.S. performed analyzed whole-cell record- <br /> depolarization tok1 mutants (22, 23). It consistent <br />                                                                           ing experiments HEK293T cells. C.S., N.Z., K.W.C. planned and <br /> that pheromone-dependent induction Kch1 Kch2 expres-               performed remaining experiments. All authors contributed the <br /> sion promote K&#226;&#171;&#185; influx partial depolarization          writing editing manuscript. <br /> plasma membrane. Until now, direct evidence <br /> of enhanced K&#226;&#171;&#185; uptake membrane depolarization dur-              REFERENCES <br /> ing response mating pheromones. Early studies failed             1. Cunningham KW. 2011. Acidic calcium stores Saccharomyces cerevi- <br /> detect significant effect &#226;?&#163;-factor exposure uptake           siae. Cell Calcium 50:129 &#226;&#8364;&#8220;138. <br /> Rb&#226;&#171;&#185; surrogate K&#226;&#171;&#185; (4), activities Kch1 Kch2         2. Iida H, Yagawa Y, Anraku Y. 1990. Essential role induced Ca2&#226;&#171;&#185; influx <br />                                                                               followed [Ca2&#226;&#171;&#185;]i rise maintaining viability yeast cells late the <br /> in experiment overwhelmed nu- <br />                                                                               mating pheromone response pathway: study [Ca2&#226;&#171;&#185;]i single Saccha- <br /> tritional activities Trk1 Trk2. It interesting revisit         romyces cerevisiae cells imaging fura-2. J. Biol. Chem. 265:13391&#226;&#8364;&#8220; <br /> this question using cells lack Trk1 Trk2 overexpress              13399. <br /> Kch1 Kch2 combined new probes direct measure-                 3. Sprague GF, Jr, Thorner JW. 1992. Pheromone response signal <br /> ment transmembrane potential (22, 23).                                     transduction mating process Saccharomyces cerevisiae, p <br />                                                                               657&#226;&#8364;&#8220;744. In Jones EW, Pringle JR, Broach JR (ed), The molecular and <br />     Kch1 Kch2 function apex death-suppressing                   cellular biology yeast Saccharomyces: gene expression. Cold Spring <br /> regulatory pathway. HACS, calmodulin, Cn neces-                   Harbor Laboratory Press, Cold Spring Harbor, NY. <br /> sary efficient mating S. cerevisiae cells, nec-    4. Ohsumi Y, Anraku Y. 1985. Specific induction Ca2&#226;&#171;&#185; transport activity <br /> essary survival nonmating cells exposed mating phero-           MATa cells Saccharomyces cerevisiae mating pheromone, alpha <br />                                                                               factor. J. Biol. Chem. 260:10482&#226;&#8364;&#8220;10486. <br /> mones. The kch1 kch2 double mutant died dramatically cch1 <br />                                                                            5. Muller EM, Locke EG, Cunningham KW. 2001. Differential regulation <br /> mutants, lethal effects mating pheromones                 Ca2&#226;&#171;&#185; influx systems pheromone signaling Saccharomyces <br /> rescued addition high Ca2&#226;&#171;&#185; culture medium (Fig.              cerevisiae. Genetics 159:1527&#226;&#8364;&#8220;1538. <br /> 2). These experiments position Kch1 Kch2 apex            6. Muller EM, Mackin NA, Erdman SE, Cunningham KW. 2003. Fig1p <br /> calcium signaling pathway used S. cerevisiae cells             facilitates Ca2&#226;&#171;&#185; influx cell fusion mating Saccharomyces <br />                                                                               cerevisiae. J. Biol. Chem. 278:38461&#226;&#8364;&#8220;38469. <br /> survival prolonged exposures mating pheromones.                  7. Cyert MS, Kunisawa R, Kaim D, Thorner J. 1991. Yeast homologs <br />     Ca2&#226;&#171;&#185; influx HACS activation Cn neces-              (CNA1 CNA2 gene products) mammalian calcineurin, calmod- <br /> sary survival S. cerevisiae cells exposed compounds            ulin-regulated phosphoprotein phosphatase. Proc. Natl. Acad. Sci. U. S. A. <br /> stress endoplasmic reticulum (35). These stressors include                88:7376 &#226;&#8364;&#8220;7380. <br />                                                                            8. Cyert MS, Thorner J. 1992. Regulatory subunit (CNB1 gene product) of <br /> tunicamycin, dithiothreitol, Ca2&#226;&#171;&#185; starvation, dis- <br />                                                                               yeast Ca2&#226;&#171;&#185;/calmodulin-dependent phosphoprotein phosphatases re- <br /> rupt biogenesis trafficking secretory proteins conse-              quired adaptation pheromone. Mol. Cell. Biol. 12:3460 &#226;&#8364;&#8220;3469. <br /> quently activate unfolded protein response (UPR) signaling             9. Moser MJ, Geiser JR, Davis TN. 1996. Ca2&#226;&#171;&#185;-calmodulin promotes sur- <br /> pathway. The UPR factors Ire1 Hac1 required                  vival pheromone-induced growth arrest activation calcineurin <br /> death-promoting effects tunicamycin death-suppressing               Ca2&#226;&#171;&#185;-calmodulin-dependent protein kinase. Mol. Cell. Biol. 16: <br />                                                                               4824 &#226;&#8364;&#8220; 4831. <br /> activation HACS. Instead, stress-responsive Slt2/Mpk1 pro-         10. Fischer M, Schnell N, Chattaway J, Davies P, Dixon G, Sanders D. 1997. <br /> tein kinase upstream regulators required             The Saccharomyces cerevisiae CCH1 gene involved calcium influx and <br /> HACS-dependent Ca2&#226;&#171;&#185; uptake response ER stressors (39).                  mating. FEBS Lett. 419:259 &#226;&#8364;&#8220;262. <br /> Preliminary findings suggest Kch1, Kch2, responds          11. Paidhungat M, Garrett S. 1997. A homolog mammalian, voltage-gated <br />                                                                               calcium channels mediates yeast pheromone-stimulated Ca2&#226;&#171;&#185; uptake and <br /> tunicamycin ER stressors activates HACS (unpub- <br />                                                                               exacerbates cdc1(Ts) growth defect. Mol. Cell. Biol. 17:6339 &#226;&#8364;&#8220; 6347. <br /> lished observations). Kch2 appears function specifi-         12. Steinbach WJ, Reedy JL, Cramer RA, JR, Perfect JR, Heitman J. 2007. <br /> cally response mating pheromones induction              Harnessing calcineurin novel anti-infective agent invasive fun- <br /> pheromone-responsive transcription factor (38).                               gal infections. Nat. Rev. Microbiol. 5:418 &#226;&#8364;&#8220; 430. <br />     Diverse pathogenic fungi, ranging yeast Candida al-          13. Iida H, Nakamura H, Ono T, Okumura MS, Anraku Y. 1994. MID1, a <br />                                                                               novel Saccharomyces cerevisiae gene encoding plasma membrane pro- <br /> bicans basidiomycete Cryptococcus neoformans, require              tein, required Ca2&#226;&#171;&#185; influx mating. Mol. Cell. Biol. 14:8259 &#226;&#8364;&#8220; <br /> activation Cn order survive assault common             8271. <br /> class antifungal drugs, inhibitors sterol biosynthesis (12).    14. Locke EG, Bonilla M, Liang L, Takita Y, Cunningham KW. 2000. A <br />  <br />  <br />  <br /> February 2013 Volume 12 Number 2                                                                                                         ec.asm.org 213 <br />  Stefan et al. <br />  <br />  <br />  <br />       homolog voltage-gated Ca2&#226;&#171;&#185; channels stimulated depletion secre-             H&#226;&#171;&#185;/Ca2&#226;&#171;&#185; exchange induces Ca2&#226;&#171;&#185; ATPases Saccharomyces cerevi- <br />       tory Ca2&#226;&#171;&#185; yeast. Mol. Cell. Biol. 20:6686 &#226;&#8364;&#8220; 6694.                                 siae. Mol. Cell. Biol. 16:2226 &#226;&#8364;&#8220;2237. <br /> 15.   Martin DC, Kim H, Mackin NA, Maldonado-Baez L, Evangelista CC,                 34.   Bertl A, Bihler H, Reid JD, Kettner C, Slayman CL. 1998. Physiological <br />       Beaudry VG, Dudgeon DD, Naiman DQ, Erdman SE, Cunningham                             characterization yeast plasma membrane outward rectifying K&#226;&#171;&#185; <br />       KW. 2011. New regulators high-affinity Ca2&#226;&#171;&#185; influx (HACS)                channel, DUK1 (TOK1), situ. J. Membr. Biol. 162:67&#226;&#8364;&#8220; 80. <br />       revealed genome-wide screen yeast. J. Biol. Chem. 286:            35.   Bonilla M, Nastase KK, Cunningham KW. 2002. Essential role cal- <br />       10744 &#226;&#8364;&#8220;10754.                                                                        cineurin response endoplasmic reticulum stress. EMBO J. 21:2343&#226;&#8364;&#8220; <br /> 16.   Viladevall L, Serrano R, Ruiz A, Domenech G, Giraldo J, Barcelo A,                   2353. <br />       Arino J. 2004. Characterization calcium-mediated response al-        36.   Scannell DR, Frank AC, Conant GC, Byrne KP, Woolfit M, Wolfe KH. <br />       kaline stress Saccharomyces cerevisiae. J. Biol. Chem. 279:43614 &#226;&#8364;&#8220; 43624.         2007. Independent sorting-out thousands duplicated gene pairs in <br /> 17.   Brand A, Shanks S, Duncan VMS, Yang M, Mackenzie K, Gow NAR.                         yeast species descended whole-genome duplication. Proc. <br />       2007. Hyphal orientation Candida albicans regulated calcium-              Natl. Acad. Sci. U. S. A. 104:8397&#226;&#8364;&#8220; 8402. <br />       dependent mechanism. Curr. Biol. 17:347&#226;&#8364;&#8220;352.                                   37.   Roberts CJ, Nelson B, Marton MJ, Stoughton R, Meyer MR, Bennett <br /> 18.   Hong MP, Vu K, Bautos J, Gelli A. 2010. Cch1 restores intracellular Ca2&#226;&#171;&#185;             HA, He YD, Dai H, Walker WL, Hughes TR, Tyers M, Boone C, Friend <br />       fungal cells endoplasmic reticulum stress. J. Biol. Chem. 285:             SH. 2000. Signaling circuitry multiple MAPK pathways revealed by <br />       10951&#226;&#8364;&#8220;10958.                                                                         matrix global gene expression profiles. Science 287:873&#226;&#8364;&#8220; 880. <br /> 19.   Slayman CL. 1965. Electrical properties Neurospora crassa: effects       38.   Heiman MG, Walter P. 2000. Prm1p, pheromone-regulated multispan- <br />       external cations intracellular potential. J. Gen. Physiol. 49:69 &#226;&#8364;&#8220;92.         ning membrane protein, facilitates plasma membrane fusion yeast <br /> 20.   Slayman CL, Long WS, Lu CY. 1973. The relationship ATP                   mating. J. Cell Biol. 151:719 &#226;&#8364;&#8220;730. <br />       electrogenic pump plasma membrane Neurospora crassa. J.           39.   Bonilla M, Cunningham KW. 2003. Mitogen-activated protein kinase <br />       Membr. Biol. 14:305&#226;&#8364;&#8220;338.                                                             stimulation Ca2&#226;&#171;&#185; signaling required survival endoplasmic re- <br /> 21.   Pe&#195;&#177;A A, S&#195;&#161;nchez NS, Calahorra M. 2010. Estimation electric                    ticulum stress yeast. Mol. Biol. Cell 14:4296 &#226;&#8364;&#8220; 4305. <br />       plasma membrane potential difference yeast fluorescent dyes:           40.   McCoy JG, Nimigean CM. 2012. Structural correlates selectivity and <br />       comparative study methods. J. Bioenerg. Biomembr. 42:419 &#226;&#8364;&#8220; 432.                   inactivation potassium channels. Biochim. Biophys. Acta 1818:272&#226;&#8364;&#8220; <br /> 22.   Madrid R, G&#195;&#179;mez MJ, Ramos J, Rodr&#195;&#173;guez-Navarro A. 1998. Ectopic                      285. <br />       potassium uptake trk1 trk2 mutants Saccharomyces cerevisiae corre-       41.   Durell SR, Guy HR. 1999. Structural models KtrB, TrkH, and <br />       lates highly hyperpolarized membrane potential. J. Biol. Chem.                Trk1,2 symporters based structure KcsA K&#226;&#171;&#185; channel. Bio- <br />       273:14838 &#226;&#8364;&#8220;14844.                                                                    phys. J. 77:789 &#226;&#8364;&#8220; 807. <br /> 23.   Maresova L, Urbankova E, Gaskova D, Sychrova H. 2006. Measure-                 42.   Ketchum KA, Joiner WJ, Sellers AJ, Kaczmarek LK, Goldstein SA. 1995. <br />       ments plasma membrane potential changes Saccharomyces cerevisiae               A new family outwardly rectifying potassium channel proteins two <br />       cells reveal importance Tok1 channel membrane potential                pore domains tandem. Nature 376:690 &#226;&#8364;&#8220; 695. <br />       maintenance. FEMS Yeast Res. 6:1039 &#226;&#8364;&#8220;1046.                                     43.   Reid JD, Lukas W, Shafaatian R, Bertl A, Scheurmann-Kettner C, Guy <br /> 24.   Ko CH, Gaber RF. 1991. TRK1 TRK2 encode structurally related K&#226;&#171;&#185;                  HR, North R <br /> </body></html>