<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNA surveillance required endoplasmic <br /> reticulum homeostasis <br /> Kenjiro Sakakia,b,1, Sawako Yoshinab,c, Xiaohua Shend, Jaeseok Hana,e, Melinda R. DeSantisa, Mon Xionga, <br /> Shohei Mitanib,c,f, Randal J. Kaufmana,e,1 <br /> a <br />  Department Biological Chemistry, University Michigan Medical Center, Ann Arbor, MI 48109; bDepartment Physiology, Tokyo Women&#226;&#8364;&#8482;s Medical <br /> University School Medicine, Shinjuku-ku, Tokyo 162-8666, Japan; cCore Research Evolutional Science Technology, Japan Science Technology <br /> Agency, Kawaguchi, Saitama 332-0012, Japan; dSchool Medicine, Tsinghua University, Beijing 100084, China; eDegenerative Disease Research Program, <br /> Neuroscience, Aging Stem Cell Research Center, Sanford-Burnham Medical Research Institute, La Jolla, CA 92037; fTokyo Women&#226;&#8364;&#8482;s Medical University <br /> Institute Integrated Medical Science (TIIMS), Shinjuku-ku, Tokyo 162-8666, Japan <br />  <br /> Edited David Ron, University Cambridge, Cambridge, United Kingdom, accepted Editorial Board March 26, 2012 (received review August <br /> 4, 2011) <br />  <br /> The unfolded protein response (UPR) intracellular stress-               cytosolic domain (ATFp50) translocation nucleus <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> signaling pathway counteracts accumulation misfolded              activate transcription (5, 6). sXBP1 ATF6(p50) bind the <br /> proteins endoplasmic reticulum (ER). Because defects ER             UPR element ER stress element (ERSE) up-regulate <br /> protein folding associated pathological states, in-             transcription genes function ER protein folding, <br /> cluding metabolic, neurologic, genetic, in&#239;&#172;&#8218;ammatory diseases,          modi&#239;&#172;?cation, traf&#239;&#172;?cking, degradation (7&#226;&#8364;&#8220;9). The UPR <br /> is important understand UPR maintains ER protein-folding <br />                                                                               transducer, PERK, protein kinase phosphorylates the <br />                                                                               &#206;&#177;-subunit eukaryotic translation initiation factor 2 (eIF2&#206;&#177;) to <br /> homeostasis. All metazoans conserved fundamental UPR <br />                                                                               attenuate mRNA translation initiation, reducing un- <br /> transducers IRE1, ATF6, PERK. In Caenorhabditis elegans, UPR          folded protein load (10&#226;&#8364;&#8220;12). In manner, UPR signaling <br /> is required prevent larval lethality intestinal degeneration.          attempts maintain &#239;&#172;?delity protein folding minimize <br /> Although ire-1-null worms viable, particularly sensitive         accumulation unfolded proteins ensuring balance <br /> to ER stress. To identify genes required development          protein-folding load protein-folding capacity. <br /> ire-1-null worms, performed comprehensive <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>RNA interference</span>                  Recently, new mechanisms described ensure ERQC. <br /> screen &#239;&#172;?nd 10 genes exhibit synthetic growth intestinal           ER stress induces autophagy, intracellular degradation system <br /> defects ire-1(v33) mutant atf-6(tm1153) pek-         bulky aggregated proteins dysfunctional organelles (13&#226;&#8364;&#8220; <br /> 1(ok275) mutants. The expression genes, exos-3            16). Although mechanisms signal ER stress-induced <br /> F48E8.6, induced ER stress, knockdown wild-             autophagy remain unclear, thought autophagy of <br /> type strain caused ER stress. Because genes encode subunits          ERQC eliminate misfolded and/or aggregated proteins the <br /> the exosome complex functions mRNA surveillance, ana-              ER. In addition, preemptive ERQC reported degrade <br /> lyzed gene products required nonsense-mediated mRNA                 misfolded nascent polypeptides step cotranslational <br /> decay (NMD). Our results demonstrate defects smg-1, smg-4, <br />                                                                               translocation ER (17&#226;&#8364;&#8220;19). Finally, proposed that <br />                                                                               regulated IRE1-dependent mRNA decay initiates degrada- <br /> and smg-6 C. elegans SMG6 mammalian cells cause ER stress <br />                                                                               tion mRNAs encoding proteins translocated the <br /> and sensitize lethal effects ER stress. Although ER stress did      ER lumen (20, 21). Therefore, appears multiple mecha- <br /> not activate mRNA surveillance complex assembly, ER stress did in-            nisms evolved prevent accumulation misfolded <br /> duce SMG6 expression, NMD regulators constitutively lo-              proteins ER. <br /> calized ER. Importantly, &#239;&#172;?ndings demonstrate unique             In metazoans, newly synthesized mRNAs subject pio- <br /> fundamental interaction NMD-mediated mRNA quality control               neer round translation necessary ensure presence <br /> is required prevent ER stress.                                             intact ORF provides essential mechanism mRNA <br />                                                                               quality control. Nonsense-mediated mRNA decay (NMD) an <br /> endoplasmic reticulum quality control   | premature termination codons        mRNA surveillance mechanism degrades mRNAs contain- <br />                                                                               ing premature termination codons (PTCs) generate <br />                                                                               polypeptides prone misfolding (22). The mechanism NMD <br /> T   endoplasmic reticulum (ER) site folding and <br />     modi&#239;&#172;?cation newly synthesized proteins destined <br /> for intracellular organelles, plasma membrane, ex- <br />                                                                               extensively characterized. The decay-inducing complex, <br />                                                                               including UPF1(SMG2), UPF2(SMG3), UPF3(SMG4), ribo- <br />                                                                               releasing factor (eRF)2, eRF3, PI3-kinase&#226;&#8364;&#8220;like <br /> tracellular milieu. Protein folding ER facilitated by <br />                                                                               protein SMG1, captures substrate mRNA pioneer <br /> molecular chaperones monitored stringent ER quality <br /> control (ERQC) allows properly folded proteins               round translation. Subsequent release ribosomes and <br /> to traf&#239;&#172;?c Golgi apparatus. In ERQC, misfolded proteins                 eRFs SMG1-mediated UPF1 phosphorylation repress <br /> are retained ER attempt attain appropriate             translation initiation promote decapping, SMG6 SMG5/ <br /> conformation, irreversibly misfolded proteins targeted             SMG7 binds phosphorylated UPF1 promotes protein <br /> ER-associated protein degradation (ERAD) autophagy (1).                 phosphatase 2A-mediated dephosphorylation UPF1 (23&#226;&#8364;&#8220;26). <br />                                                                                                                                                                                  BIOCHEMISTRY <br />  <br />  <br />  <br />  <br />    The accumulation misfolded proteins ER caused                 SMG6 endoribonuclease cleaves PTC-proximal site <br /> alterations ER homeostasis initiates signaling unfolded             mRNA subsequent 3&#226;&#8364;&#178;-5&#226;&#8364;&#178; degradation exo- <br /> protein response (UPR) attempts resolve protein-                  5&#226;&#8364;&#178;-3&#226;&#8364;&#178; decay XRN1 (27&#226;&#8364;&#8220;29). <br /> folding defect (2). In metazoans, UPR signaled through <br /> three ER transmembrane transducers, IRE1, ATF6, PERK, <br /> that sense accumulation misfolded proteins transmit                Author contributions: K.S., X.S., S.M., R.J.K. designed research; K.S., S.Y., J.H., M.R.D., <br /> signals cytosol nucleus. IRE1 protein kinase/             M.X. performed research; K.S. analyzed data; K.S., S.M., R.J.K. wrote <br />                                                                               paper. <br /> endoribonuclease initiates unconventional splicing mRNA <br /> encoding transcription factor X box-binding protein 1 (XBP1),             The authors declare con&#239;&#172;&#8218;ict interest. <br /> to create translational frameshift produces active tran-            This article PNAS Direct Submission. D.R. guest editor invited Editorial Board. <br /> scription factor (sXBP1) (3, 4). ATF6 basic leucine zipper              1 <br />                                                                               To correspondence addressed. sakaki@research.twmu.ac.jp rkaufman@ <br /> domain cytosolic N terminus. Upon accumulation un-                  sbmri.org. <br /> folded proteins ER, ATF6 transits Golgi apparatus,              This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> where processed site 1 site 2 proteases release           1073/pnas.1110589109/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1110589109                                                          PNAS | May 22, 2012 | vol. 109 | no. 21 | 8079&#226;&#8364;&#8220;8084 <br />     In report, demonstrate genes required NMD          embryonic development. The &#239;&#172;?ndings suggest the <br /> also contribute ER homeostasis necessary Cae-           ire-1(v33) deletion mutant adapted constitutive ER stress <br /> norhabditis elegans larval development mammalian cell              survive propagate. <br /> proliferation response ER stress. In particular, SMG6           To identify additional genes required development <br /> two exosomal subunits, DIS3 EXOSC3, induced ER             ire-1(v33) deletion mutant C. elegans, used feeding- <br /> stress. Although UPF1 phosphorylation RNA surveillance             mediated RNAi analysis screen genes exhibit synthetic <br /> complex assembly affected ER stress, NMD regu-             larval lethal/severe growth defect intestinal degeneration the <br /> lators constitutively localized ER. We propose        ire-1(v33) mutant pek-1(ok275) mutant. Screening of <br /> an effective RNA surveillance ER membrane             16,757 genes covering 87% C. elegans genome did not <br /> required maintain ER function prevent generation         identify genes ire-1 xbp-1 exhibit synthetic <br /> misfolded proteins pioneer round translation.            lethality pek-1(ok275) mutant. In contrast, screen <br />                                                                        identi&#239;&#172;?ed 97 genes evolutionally conserved mammals <br /> Results <br />                                                                        required normal growth ire-1(v33) mutant did <br /> C. elegans RNAi Screen Identi&#239;&#172;?es Genes Required Growth ire-1    affect growth control rrf-3(pk1426) mutant, hy- <br /> (v-33). Although basic mechanisms tripartite UPR signaling      persensitive RNAi treatment (31) (Fig. 1A Table S1). Of <br /> are conserved metazoan species, different species cell      97 genes, knockdown 62 genes displayed synthetic growth <br /> types rely different subpathways function <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span>.   defects ire-1(v33) atf-6(tm1153) mutants, all <br /> Unfortunately, presence Ire1 homologs Atf6          UPR mutants [including pek-1(ok275)]. Within 62 <br /> homologs mammals limited analysis functional       genes set nuclear-encoded mitochondrial genes (F59A3.3, <br /> signi&#239;&#172;?cance. Fortunately, C. elegans single gene of <br />                                                                        nuo-2, F54C4.1, Y53G8AL.2, tag-313, phb-2, mrps-5, T22B11.5, <br /> the key UPR regulators [PERK (pek-1), ATF6 (atf-6), IRE1 (ire- <br /> 1), XBP1 (xbp-1)]. However, previous studies revealed         Y43F8C.8, C47E12.2, F53F4.10), knockdown <br /> the ire-1/xbp-1 pathway regulates vast majority UPR tran-       did cause intestinal degeneration UPR mutants. <br /> scriptional control response acute stress (induced UPR)      Knockdown mitochondrial genes deplete ATP or <br /> in larval development (constitutive UPR) (30). Although C. ele-        cause oxidative stress inhibit worm development through <br /> gans deletions single UPR sensor viable, double     mechanism intestinal degeneration. Knockdown 10 <br /> deletion pek-1 atf-6 combination ire-1 produces   62 genes caused intestinal degeneration ire-1 <br /> the typical <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> UPR de&#239;&#172;?ciency C. elegans,         (v33) atf-6(tm1153) mutants (Table S2). This group includes <br /> L2/L4 larval lethality intestinal degeneration. In contrast,   master regulators ER protein biosynthesis [hsp-4 (GRP78) and <br /> double mutant pek-1 atf-6 exhibits normal phenotype (3).      F08D12.1 (SRP72)]. Interestingly, group contained nfyc-1, <br /> These &#239;&#172;?ndings suggest ire-1/xbp-1 pathway         encode subunit nuclear transcription factor Y (NF- <br /> important UPR signaling pathway ER functional homeostasis          Y) binds ATF6 mammalian ERSE (32, 33). It is <br />  <br />  <br /> A      C. elegans RNAi library                             B <br />            (16,757 genes) <br />  <br />                                                                                                          Fig. 1. ire-1 interacts genetically <br />                                                                                                          components exosome. (A) <br /> Synthe c growth defect ire-1(v33),                                                                    A comprehensive RNAi analysis iden- <br />     normal rrf-3(pk1426)                                                                          ti&#239;&#172;?es genes required growth and <br />             (124 genes)                                                                                  intestinal proliferation ire-1(v33) <br />                                                                                                          worms. Knockdown 97 evolutionally <br />                                                                                                          conserved genes exhibited synthetic <br />                             Not conserved mammals                                                     growth defects ire-1(v33) strain. <br />                                    (2 7 genes)             D                                             Of these, 10 caused synthetic <br />        Conserved mammals <br />                                                                                                          severe growth defect intestinal <br />              (97 genes)                                                                                  degeneration ire-1(v33) mutant, <br />                                                                                                          UPR mutants <br />                           Synthe c growth defect                                                         tested. (B) TM induction F48E8.6 <br />                          UPR mutants                                                        does require single UPR path- <br />                                                                                                          way. Worms strain L2 <br />        Synthe c growth defect    (62 genes)                                                              larval stage treated 10 &#206;&#188;g/mL <br />            ire-1(v33)                                                                            TM 3 h analyzed qRT-PCR. <br />                (35 genes)                                  E                                             The expression gene nor- <br />                                                                                                          malized act-2 (the C. elegans homo- <br />                                                                                                          log mammalian ACT&#206;&#178;). The relative <br />                                                                                                          mRNA abundance strain was <br />                                                                                                          normalized expression wild <br />        Intes nal degenera on                                                                             type N2. (C) TG induces expression of <br />              (10 genes)                                                                                  CRT1, DIS3, EXOSC3 murine im- <br />                                                                                                          mortalized hepatocytes. Cells were <br /> C                                                                                                        treated 1 &#206;&#188;M TG 8 h ana- <br />                                                                                                          lyzed qRT-PCR. (D E) Neither <br />                                                                                                          single UPR pathway required TG <br />                                                                                                          induction CALR, TG in- <br />                                                                                                          duction DIS3 EXOSC3 requires <br />                                                                                                          eIF2&#206;&#177; phosphorylation. Cells trea- <br />                                                                                                          ted TG analyzed qRT-PCR <br />                                                                                                          described (C). Average values &#194;&#177; SD <br />                                                                                                          shown (B, n = 3; C&#226;&#8364;&#8220;E, n = 4). P values <br />                                                                                                          calculated using Student&#226;&#8364;&#8482;s <br />                                                                                                          t test. *P &lt; 0.05; **P &lt; 0.01; bars no <br />                                                                                                          asterisks, P &gt; 0.05. <br />  <br />  <br /> 8080 | www.pnas.org/cgi/doi/10.1073/pnas.1110589109                                                                                    Sakaki et al. <br />  Table 1. Genes caused synthetic severe growth defect intestinal degeneration RNAi-mediated knockdown ire-1(v33) <br />                                                                                     GFP level                   Ratio expression level N2 (SD) <br />  <br /> Gene                             Gene annotation                                  zcIs4             ire-1(v33)          atf-6(tm1533)         pek-1(ok275) <br />  <br /> crt-1            Calreticulin                                                       Induced           1.26   (&#194;&#177;0.112)*      0.441   (&#194;&#177;0.212)*    0.993   (&#194;&#177;0.119) <br /> arf-6            ADP ribosylation factor                                            Induced           1.71   (&#194;&#177;0.159)**      1.09   (&#194;&#177;0.186)      1.07   (&#194;&#177;0.301) <br /> 4R79.2           GTP-binding protein SEC4                                           Induced           1.67   (&#194;&#177;0.579)       0.969   (&#194;&#177;0.0384)    0.981   (&#194;&#177;0.355) <br /> F48E8.6          Exosomal 3&#226;&#8364;&#178;-5&#226;&#8364;&#178; exoribonuclease subunit Rrp44/Dis3                  Induced           2.23   (&#194;&#177;0.520)*       1.34   (&#194;&#177;0.388)      1.39   (&#194;&#177;0.169)* <br /> exos-3           Exosomal 3&#226;&#8364;&#178;-5&#226;&#8364;&#178; exoribonuclease subunit Rrp40/EXOSC3                Induced           2.78   (&#194;&#177;0.392)**     0.978   (&#194;&#177;0.0427)     1.08   (&#194;&#177;0.281) <br /> Y39G10AR.8       Translation initiation factor 2, &#206;&#179;-subunit                         Reduced           1.35   (&#194;&#177;0.323)       0.688   (&#194;&#177;0.145)*    0815    (&#194;&#177;0.0612)* <br /> rpl-1            Large ribosomal subunit L 10a protein                              Reduced           1.67   (&#194;&#177;0.459)       0.991   (&#194;&#177;0.102)      1.08   (&#194;&#177;0.158) <br /> ZK1098.4         Translation initiation factor 2B, &#206;&#177;-subunit                      Not changed         1.51   (&#194;&#177;0.599)        1.32   (&#194;&#177;0.282)      1.22   (&#194;&#177;0.021)** <br /> cpl-1            Cysteine proteinase Cathepsin L                                  Not changed       0.1.67   (&#194;&#177;0.545)        1.11   (&#194;&#177;0.299)      1.17   (&#194;&#177;0.136) <br /> K06A5.8          WD-repeat containing protein                                     Not changed         2.43   (&#194;&#177;0.858)*      0.799   (&#194;&#177;0.296)     0.870   (&#194;&#177;0.312) <br />  <br />    The 10 genes exhibit synthetic lethality intestinal defects RNAi knockdown ire-1(v33) worms listed. Gene annotations WormBase. <br /> ER stress induction RNAi knockdown evaluated zcIs4 strain (Fig. S1). The relative expression gene, analyzed qRT-PCR, mutant <br /> strains compared wild type (N2). Average values &#194;&#177; SD shown (n = 3). P values calculated using Student&#226;&#8364;&#8482;s t test. *P &lt; 0.05; **P &lt; 0.01; bars <br /> with asterisks, P &gt; 0.05. <br />  <br />  <br />  <br /> possible NFYC-1 functions XBP1 and/or ATF6 acti-                       ER, induced CALR (crt-1), DIS3 (F48E8.6), EXOSC3 <br /> vate UPR genes C. elegans.                                                       (exos-3) twofold wild-type hepatocytes (Fig. 1C and <br />    The synthetic lethal screen identi&#239;&#172;?ed 35 genes speci&#239;&#172;?cally                  Table S3). Similar C. elegans, basal expression CALR <br /> exhibited growth defects ire-1(v33) mutant                   reduced 22% Atf6&#206;&#177;&#226;&#710;&#8217;/&#226;&#710;&#8217; cells compared wild <br /> the atf-6(tm1153) and/or pek-1(ok275) mutants. RNAi-mediated                        type (Fig. S4), TG induction signi&#239;&#172;?cantly reduced <br /> knockdown 35 genes identi&#239;&#172;?ed 10 genes caused in-                      mutants, including Ire1&#206;&#177;&#226;&#710;&#8217;/&#226;&#710;&#8217; hepatocytes (Fig. <br /> testinal degeneration ire-1(v33) mutant (Fig. 1A Table                   1D). These &#239;&#172;?ndings suggest ATF6 requirement basal <br /> 1). In addition, screen identi&#239;&#172;?ed atf-6 pek-1,                  expression CALR evolutionally conserved. However, no <br /> were used positive controls synthetic lethal screen ire-              single UPR pathway required induction CALR re- <br /> 1(v33). We assessed effect knockdown 10 genes                    sponse acute ER stress mammals, ire-1/xbp-1 <br /> ER homeostasis RNAi treatment zcIs4 transgenic strain                       pathway regulates crt-1 induction C. elegans. In contrast the <br /> harboring hsp-4 promoter-driven GFP transgene exhibits                     C. elegans homologs, basal expression DIS3 EXOSC3 <br /> ER stress-dependent induction GFP &#239;&#172;&#8218;uorescence. RNAi-me-                          increased Ire1&#206;&#177;&#226;&#710;&#8217;/&#226;&#710;&#8217; cells (Fig. S4), TG <br /> diated knockdown &#239;&#172;?ve genes (crt-1, arf-6, 4R79.2, F48E8.6,                   induction reduced eIF2&#206;&#177;(AA) mutant cells (Fig. 1E). <br /> exos-3) elevated GFP expression intestinal cells (Table 1                    Therefore, ER stress induction DIS3 EXOSC3 apparently <br /> Fig. S1). This suggests &#239;&#172;?ve genes required                        requires eIF2&#206;&#177; phosphorylation mammalian cells. Together, <br /> maintain ER protein-folding homeostasis. In addition crt-1                       &#239;&#172;?ndings indicate DIS3 EXOSC3 important for <br /> encoding ER chaperone, genes, 4R79.2 arf-6,                     ER functional homeostasis larval development C. elegans. <br /> encode vesicle traf&#239;&#172;?cking-associated proteins. The reduced ex-                      The exosomal complex mediates mRNA decay NMD sur- <br /> pression genes directly disrupt ER function                       veillance. Consistent notion, knockdown <br /> perturbing ER protein folding and/or traf&#239;&#172;?cking. Interestingly,                     DIS3 EXOSC3 caused accumulation mutant <br /> the genes, F48E8.6 exos-3, encode core subunit                      protein derived PTC-containing mutant GABA <br /> (RRP44/DIS3) regulatory subunit (RRP40/EXOSC3)                             receptor &#206;&#177; 1 (GABRA1), degraded ERAD (37) (Fig. <br /> the 3&#226;&#8364;&#178;-5&#226;&#8364;&#178; exoribonuclease (exosome) complex.                                        S5). These &#239;&#172;?ndings suggest functional interaction exists <br />    Quantitative RT-PCR (qRT-PCR) revealed basal                            mRNA quality control ER homeostasis. <br /> expression 3 10 genes (F48E8.6, exos-3, K06A5.8) <br /> was elevated twofold ire-1(v33) mutant com-                        NMD Genes Are Required ER Homeostasis Survival Response <br /> pared wild type, expression crt-1 reduced                   ER Stress. To assess ER protein-folding homeostasis <br /> to 44% atf-6(tm1153) mutant (Table 1). Because RNAi                          requires NMD-mediated mRNA surveillance, analyzed dele- <br /> knockdown genes caused synthetic lethality ire-1                    tions NMD genes smg-1, smg-4, smg-6 zcIs4 <br /> (v33) mutant, &#239;&#172;?ndings suggest up-regulation F48E8.6, <br /> exos-3, K06A5.8 ATF6-mediated expression crt-1 <br /> may compensate loss ire-1 ensure ER homeostasis <br /> during larval development. Furthermore, expression of <br /> F48E8.6 crt-1 up-regulated twofold tuni- <br /> camycin (TM), inhibits N-linked glycosylation disrupts <br /> ER protein folding (Fig. 1B Fig. S2), other <br />                                                                                                                                                                        BIOCHEMISTRY <br />  <br />  <br />  <br />  <br /> genes induced twofold (Fig. S3). TM <br /> induction F48E8.6 did require ire-1, atf-6, pek-1, sug- <br /> gesting induction did require single UPR pathway <br /> or uses unconventional UPR pathway (Fig. 1B). On the <br /> other hand, induction crt-1 required ire-1 atf-6, <br /> indicating crt-1 requires atf-6 constitutive expression <br /> during larval development requires ire-1/xbp-1 induction <br /> upon acute ER stress, consistent previous &#239;&#172;?ndings (34).                        Fig. 2. The UPR activated smg-1(tm849) smg-6(ok1794) mutant strains. <br />    We analyzed ER stress induction murine homologs                      (A) Expression hsp-4-GFP increased smg-1(tm849) smg-6(ok1794) <br /> of 10 genes immortalized murine hepatocytes deleted                     mutants. Differential interference contrast (DIC) microscopy &#239;&#172;&#8218;uorescence <br /> either Ire1&#206;&#177; Atf6&#206;&#177; harboring serine 51 alanine mu-                     microscopy GFP L4 larvae smg-1(tm849);zcIs4, smg-6(ok1794);zcIs4, <br /> tation PERK-mediated phosphorylation site eIF2&#206;&#177;                           parental zcIs4 worms. (B) hsp-4 mRNA expression increased smg-1 <br /> [eIF2&#206;&#177;(AA)] (10, 35, 36). Thapsigargin (TG) treatment,                        (tm849) smg-6(ok1794) mutants. L2-stage worms analyzed qRT-PCR <br /> depletes ER Ca2+ inhibition Ca2+-dependent ATPase                         hsp-4. Error bars asterisks described Fig. 1 (n = 3). <br />  <br />  <br /> Sakaki et al.                                                                                                 PNAS | May 22, 2012 | vol. 109 | no. 21 | 8081 <br />  transgenic hsp-4 GFP reporter strain. All strains displayed greater           SMG6 siRNA compared nonspeci&#239;&#172;?c siRNA (Fig. 4B). <br /> GFP &#239;&#172;&#8218;uorescence compared parental zcIs4 strain (Fig. 2A               The overexpression dominant-negative SMG6 mutant <br /> Fig. S6), qRT-PCR revealed expression endoge-                 lacking PIN domain, required endoribonuclease <br /> nous hsp-4 mRNA constitutively elevated smg-1(tm849)               activity (28), reduced <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>cell survival</span> compared over- <br /> and smg-6(ok1794) mutant strains compared wild type N2                    expression wild-type SMG6 (Fig. 4C). Finally, overexpression <br /> (Fig. 2B). The smg-1, smg-4, smg-6 mutants exhibited synthetic            wild-type SMG6 increased survival HeLa cells response <br /> growth defects silencing ire-1 (Fig. S7). Although smg-1,             ER stress (Fig. 4D). Importantly, overexpression SMG6 <br /> smg-4, smg-5, smg-7 represented bacterial RNAi                attenuated CHOP induction long-term treatment TM, <br /> library, included 10 genes obtained               induction GRP78 affected (Fig. S11). <br /> screen, possibly knockdown insuf&#239;&#172;?cient. smg-6               Because CHOP encodes proapoptotic transcription factor, this <br /> was included 10 genes smg-6 repre-                 observation suggests proapoptotic response reduced <br /> sented bacterial RNAi library. smg-2 smg-3                overexpression SMG6. These observations support idea <br /> tested RNAi treatment gene disruption.                           NMD critical survival response ER stress. <br />    To analyze NMD requirement ER homeostasis in <br /> mammalian cells, silenced SMG6 HeLa cells. Analysis               SMG6 Expression Is Induced Response ER Stress Is Localized <br /> NMD reporter assay using Renilla luciferase fused minigene               ER. An NMD reporter assay demonstrated PTC- <br /> of &#206;&#178;-globin demonstrated siRNA-mediated knockdown                     containing mRNA accumulated response TM treatment (Fig. <br /> SMG6 caused accumulation PTC-containing mRNA (Fig. S8                    S12). Therefore, analyzed effect ER stress NMD <br /> A B). Immunoblotting qRT-PCR analysis revealed                   activation analysis UPF1 phosphorylation interaction <br /> SMG6 knockdown increased expression CHOP GRP78                         components RNA surveillance complex HeLa <br /> (Fig. 3A Fig. S9). The activity CHOP promoter up-              cells treated TG. TG treatment did alter UPF1 phos- <br /> regulated SMG6 knockdown, indicating transcriptional                  phorylation coimmunoprecipitation UPF3B SMG7 (Fig. <br /> activation CHOP occurs NMD inhibition (Fig. S8C).                     5A). Because secretory proteins frequently possess signal <br /> Analysis using ATF4 reporter 5&#226;&#8364;&#178; UTR ATF4                peptide N terminus targeting ER the <br /> fused GFP (38) demonstrated increased GFP expression                  pioneer round translation, analyzed NMD regu- <br /> SMG6 knockdown (Fig. S8D). In addition, knockdown SMG6                     lators localized ER. Immuno&#239;&#172;&#8218;uorescence microscopy <br /> increased Xbp1 mRNA splicing compared nonspeci&#239;&#172;?c siRNA-                  revealed colocalization endogenous SMG6 UPF3B the <br /> treated cells approximately threefold (Fig. 3 B C). The             ER protein chaperone GRP78/BiP (Fig. 5B). These observations <br /> silencing SMG1 slightly increased CHOP mRNA,                 suggest NMD regulators constitutively reside ER. <br /> it did alter expression GRP78 sensitize cells ER stress,            We asked expression NMD genes is <br /> possibly inef&#239;&#172;?cient knockdown (Fig. S10). These &#239;&#172;?ndings                regulated ER stress. HeLa cells treated ER stress- <br /> support notion NMD required ER homeostasis,                   inducing agents expression SMG1, UPF1, UPF2, <br /> namely absence NMD, cells activate UPR.                    UPF3A, UPF3B, SMG5, SMG6, SMG7 analyzed by <br />    To determine NMD required cell survival                  qRT-PCR. The results demonstrated expression of <br /> response ER stress, analyzed cell survival response               SMG6 up-regulated approximately threefold treatment <br /> TM treatment. Whereas TM slightly reduced survival HeLa                    TM TG (Fig. 5C). The expression SMG6 was <br /> cells treated nonspeci&#239;&#172;?c siRNA, TM reduced survival                   induced two- threefold response TG treatment <br /> &#226;&#710;&#188;50% cells treated SMG6 siRNA (Fig. 4A). TM                      wild type immortalized hepatocytes (Fig. 5D). However, ER <br /> reduced survival immortalized murine hepatocytes treated                   stress-mediated induction SMG6 reduced of <br />                                                                               UPR mutant hepatocyte cell lines (Fig. 5D), indicating that <br />                                                                               single UPR signaling pathway required induction <br />                                                                               SMG6 response ER stress. <br />  <br />                                                                               Discussion <br />                                                                               In study, discovered genes encoding exosome and <br />                                                                               NMD machinery required ER functional homeostasis, <br />                                                                               cell survival ER stress, normal development worms <br />                                                                               lacking ire-1. Our &#239;&#172;?ndings suggest component ER <br />                                                                               quality control relies intact mRNA surveillance to <br />                                                                               minimize ER stress. The following observations support this <br />                                                                               conclusion: (i) The loss reduced expression NMD genes <br />                                                                               (F48E8.6, exos-3, smg-1, smg-4, smg-6 C. elegans; SMG6 in <br />                                                                               mammals) causes ER stress; (ii) F48E8.6, exos-3, smg-1, smg-4, <br />                                                                               smg-6 display synthetic growth defects ire-1(v33) <br />                                                                               deletion mutant; (iii) SMG6 knockdown mammalian cells <br />                                                                               reduces SMG6 overexpression increases cell survival re- <br />                                                                               sponse ER stress; (iv) DIS3 (F48E8.6) induced acute ER <br /> Fig. 3. Knockdown SMG6 causes ER stress HeLa cells. (A) Knockdown <br />                                                                               stress C. elegans mammals, EXOSC3 (exos-3) and <br /> of SMG6 increases GRP78 CHOP protein HeLa cells. HeLa cells were <br />                                                                               SMG6 induced acute ER stress mammals; (v) <br /> transfected SMG6 siRNA #2 (see Table S4) nonspeci&#239;&#172;?c (NS) siRNA,   NMD factors constitutively localized ER. Our &#239;&#172;?ndings <br /> after 72 h treated 1 &#206;&#188;M TG 8 h. The expression GRP78,        lead propose defect NMD decrease the <br /> CHOP, &#206;&#178;-tubulin analyzed immunoblotting. (B) Knockdown          &#239;&#172;?delity mRNA quality control increase synthesis of <br /> SMG6 induces splicing Xbp1 mRNA. HeLa cells transfected          proteins having aberrant carboxyl termini, case of <br /> SMG6 siRNA #2 nonspeci&#239;&#172;?c siRNA. After 72 h, cells treated 1 &#206;&#188;M   secretory proteins misfold translocation ER. <br /> TG, 10 &#206;&#188;g/mL TM, DMSO 4 h. Xbp1 mRNA splicing analyzed          On hand, accumulation misfolded proteins in <br /> described Materials Methods. S, spliced Xbp1 mRNA; U1 U2,          ER activate NMD preemptive quality control <br /> digested unspliced fragments 1 2; UD, undigested unspliced fragments,     mechanism degrade mRNAs cytoplasmic trans- <br /> likely representing heteroduplex DNA. (C) The intensities band   lation attenuate protein misfolding. However, point, we <br /> determined described Materials Methods, ef&#239;&#172;?ciencies Xbp1     rule requirement exosomal components <br /> splicing quanti&#239;&#172;?ed calculating percentage spliced Xbp1 mRNA    survive ER stress re&#239;&#172;&#8218;ects exosome-dependent pro- <br /> relative total Xbp1 mRNA (S, U1, U2 plus UD). Induction Xbp1 splicing   cess, RNA processing RNA degradation. <br /> by knockdown SMG6 comparison nonspeci&#239;&#172;?c siRNA treatment            Remarkably, &#239;&#172;?ndings indicate ER stress induces the <br /> shown. Error bars asterisks described Fig. 1 (n = 10).          accumulation PTC-containing mutant mRNAs expression <br />  <br /> 8082 | www.pnas.org/cgi/doi/10.1073/pnas.1110589109                                                                                    Sakaki et al. <br />                                                                                    Fig. 4. SMG6 required cell survival response ER <br />                                                                                   stress. (A B) SMG6 siRNA reduces survival ER-stressed cells. <br />                                                                                   HeLa cells (A) immortalized murine hepatocytes (B) were <br />                                                                                   transfected SMG6 siRNA. At 24 h transfection, cells <br />                                                                                   treated 250 ng/mL TM 48 h, cell viability was <br />                                                                                   measured MTS assay (Materials Methods). Two different <br />                                                                                   SMG6 siRNAs analyzed HeLa cells, mixture four <br />                                                                                   different siRNAs analyzed hepatocytes. (C) Domi- <br />                                                                                   nant-negative (DN) SMG6 reduces cell survival response ER <br />                                                                                   stress. HeLa cells transfected plasmid DNAs encoding <br />                                                                                   wild-type SMG6 (pEGFP-hSMG6) dominant-negative SMG6 <br />                                                                                   mutant lacking PIN domain (pEGFP-hSMG6&#206;&#8221;PIN). At 24 h <br />                                                                                   transfection, cells treated 300 nM TG 48 h <br />                                                                                   analyzed cell viability MTS assay. (D) Overexpression <br />                                                                                   SMG6 improves survival ER-stressed cells. HeLa cells were <br />                                                                                   transfected pEGFP-hSMG6 (SMG6) vector (EV). At <br />                                                                                   24 h transfection, cells treated 2 &#206;&#188;g/mL TM for <br />                                                                                   48 h analyzed cell viability MTS assay. Error bars and <br />                                                                                   asterisks described Fig. 1 (A, B, D, n = 3; C, n = 4). <br />  <br />  <br />  <br /> of SMG6 exosomal subunits (DIS3 EXOSC3),              It recently suggested IRE1-dependent sur- <br /> the phosphorylation UPF1 assembly RNA sur-           veillance exists degrade mRNAs associated the <br /> veillance complex affected (Figs. 1 5 Fig. S12).   ER membrane limit synthesis proteins require <br /> In addition, NMD regulatory proteins (SMG6 UPF3B)          folding ER lumen (20, 21). The NMD machinery may <br /> localize ER (Fig. 5B). These observations suggest      provide unique role maintain ER homeostasis does not <br /> NMD-mediated mRNA surveillance functions ER mem-            involve NMD, contributing decay mRNAs at <br /> brane, possibly association translocon, ensure      ER membrane initially cleaved IRE1 response <br /> quality control mRNAs encoding secretory proteins. In-         ER stress. To elucidate molecular mechanism which <br /> duction SMG6 exosomal subunits required       ERQC NMD machinery interact, future studies should <br /> increase ef&#239;&#172;?ciency cleavage degradation PTC-         investigate mechanism(s) recruits NMD factors ER <br /> containing mRNAs NMD attenuate generation             membrane function coordinated processes <br /> misfolded proteins response ER stress.                       protein translocation folding, ERAD, UPR signaling <br />    Our &#239;&#172;?ndings induction exosomal subunits     maintain ER protein-folding homeostasis. <br /> regulated eIF2&#206;&#177; phosphorylation. In mammals, eIF2&#206;&#177;              NMD prevents generation misfolded proteins many <br /> phosphorylated eIF2&#206;&#177; kinases, including PERK. Because      diseases caused nonsense gene mutations. Recent studies suggest <br /> TM induction F48E8.6 altered pek-1 deletion (Fig.    inhibition NMD suppression translation termination <br /> 1B), speculate eIF2&#206;&#177; phosphorylation result     therapeutic human diseases. For example, <br /> other eIF2&#206;&#177; kinases, gcn-2. In contrast, &#239;&#172;?ndings in-   group patients Ullrich disease nonsense mutations in <br /> dicate UPR signaling pathways (IRE1&#206;&#177;,          type II collagen IV [COL&#206;&#177;2(IV)] gene, mRNA is <br /> ATF6&#206;&#177;, PERK) required ER stress induction SMG6        subject NMD. Although translation product mutant <br /> in mammalian cells (Fig. 5D). To unravel interplay     mRNA truncated, structurally stable functional cell <br /> ER quality control mRNA surveillance, studies      surface. Inhibition NMD siRNA-mediated knockdown of <br /> required investigate mechanism NMD factors     SMG1 improved cell adhesion cells patients Ullrich <br /> recruited ER membrane, SMG6 induced ER          disease (39). Therefore, NMD inhibition clinical utility in <br /> stress, coordinately function ERQC            management diseases. However, &#239;&#172;?ndings that <br /> maintain ER functional homeostasis. The notion NMD        NMD de&#239;&#172;?ciency causes ER stress reduces cell proliferation in <br /> ERAD cooperate limit production abnormal proteins        response ER stress. Therefore, NMD required cells <br /> was recently suggested analysis nonsense mutations      exposed ER stress. As UPR required differ- <br /> the GABAA receptor GABRA1 (37) (Fig. S5).                          entiation survival cells secrete large amounts protein, <br />  <br />  <br />                                                                                  Fig. 5. ER stress does alter NMD activation, induces <br />                                                                                  SMG6 expression. (A) ER stress does alter UPF1 phos- <br />                                                                                  phorylation assembly NMD complex. HeLa cells <br />                                                                                  transfected pCMV5FLAG-hUPF1 vector. <br />                                                                                  After 36 h, cells treated 1 &#206;&#188;M TG 8 h. UPF1 <br />                                                                                  phosphorylation RNA surveillance complex formation <br />                                                                                                                                                          BIOCHEMISTRY <br />  <br />  <br />  <br />  <br />                                                                                  analyzed immunoprecipitation (IP) anti-FLAG <br />                                                                                  antibody Western blotting antibodies indicated. <br />                                                                                  (B) SMG6 UPF3B colocalize GRP78. HeLa cells were <br />                                                                                  stained anti-UPF3B anti-SMG6 anti-GRP78 anti- <br />                                                                                  bodies observed confocal microscopy. (C) ER stress <br />                                                                                  induces expression SMG6. HeLa cells treated 10 <br />                                                                                  &#206;&#188;g/mL TM 1 &#206;&#188;M TG 8 h, mRNA expression was <br />                                                                                  analyzed qRT-PCR. The expression gene was <br />                                                                                  normalized expression ACT&#206;&#178;, abundance <br />                                                                                  relative mock treatment shown. (D) Induction SMG6 <br />                                                                                  ER stress does require single UPR pathway. Wild <br />                                                                                  type eIF2&#206;&#177;(A/A), Ire1&#206;&#177;&#226;&#710;&#8217;/&#226;&#710;&#8217;, Atf6&#206;&#177;&#226;&#710;&#8217;/&#226;&#710;&#8217; immortalized <br />                                                                                  hepatocytes treated DMSO 1 &#206;&#188;M TG 16 h, <br />                                                                                  SMG6 expression relative ACT&#206;&#178; analyzed qRT- <br />                                                                                  PCR. Error bars asterisks described Fig. 1 (n = 3). <br />  <br />  <br /> Sakaki et al.                                                                              PNAS | May 22, 2012 | vol. 109 | no. 21 | 8083 <br />  as metabolic in&#239;&#172;&#8218;ammatory responses,                                Cell Proliferation Luciferase Reporter Assays. Cell proliferation ana- <br /> be caution treating patients NMD inhibitors. We propose                                 lyzed MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4- <br /> that development NMD inhibitors disrupt ER                                   sulfophenyl)-2H-tetrazolium] assay using CellTiter 96 Aqueous One <br /> homeostasis advantageous treatment individuals                               Solution Assay (Promega). Luciferase reporter assays performed using <br /> with nonsense gene mutations, muscular dystrophy.                                    Dual Luciferase Reporter System (Promega). <br />  <br /> Materials Methods                                                                           Detection UPF1 Phosphorylation RNA Surveillance Complex Assembly. <br /> Detailed procedures available SI Materials Methods.                                  Human UPF1 cDNA ampli&#239;&#172;?ed RT-PCR total mRNA extracted <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                 HeLa cells inserted pCMV5-FLAG express FLAG-hUPF1 HeLa <br /> Nematode Strains. smg-1(tm849), smg-4(tm5409), atf-6(tm1153)                           cells. Detection UPF1 phosphorylation RNA surveillance complex <br /> obtained described (40). All strains obtained                         performed described (44). <br /> Caenorhabditis Genetics Center. Genetic crosses performed de- <br /> scribed (41). The details mutant strains, smg-4(tm5409),                  ACKNOWLEDGMENTS. We thank Ms. Jan Mitchell [University Michigan <br /> shown WormBase (http://www.wormbase.org). smg-4(tm5409) gen-                             (UM)] assistance preparing manuscript; Mr. Ed Stuart, Ms. Xiaoli <br /> erated study lacks 338 bp exon 2.                                                Zhou (UM), Ms. Hitomi Moriya, Ms. Miwa Sato-Nakajima [Tokyo Women&#226;&#8364;&#8482;s <br />                                                                                                 Medical University (TWMU)] supporting C. elegans manipulations; <br />                                                                                                 Drs. Yoshiro Maru Tsuyoshi Tomita (TWMU) luciferase analysis; and <br /> Feeding-Mediated RNAi Treatment Worms. Comprehensive feeding-medi-                           Mr. Yasuhide Shigematsu (TWMU) confocal microscopy analysis. We espe- <br /> ated RNAi analysis performed described (42) using C. elegans RNAi                    cially thank Dr. Elisa Izaurralde (Max Planck Institute) providing hSMG6 <br /> library obtained Source BioScience.                                                        hSMG6(&#206;&#8221;PIN) cDNAs. We thank members R.J.K. S.M. <br />                                                                                                 laboratories fruitful discussions. Portions work supported by <br /> RNA Preparation, qRT-PCR, XBP1 Splicing Analysis. cDNA synthesized                     National Institutes Health Grants DK042394, DK088227, HL052173, and <br />                                                                                                 HL057346 (to R.J.K.); Program Promoting Establishment Strate- <br /> 1 &#206;&#188;g total RNA analyzed qPCR using POWER SYBR Green PCR <br />                                                                                                 gic Research Centers, Special Coordination Funds Promoting Science and <br /> Master Mix 7500 Real-Time PCR System (Applied Biosystems).                              Technology (S.M.); Ministry Education, Culture, Sports, Science and <br /> Threshold cycles (CT) normalized act-2 mRNA worms ACT&#206;&#178;                            Technology (K.S. S.M.); Japan Society Promotion Science <br /> mRNAs mammals. XBP1 splicing analyzed described (43)                              Grant-in-Aid Scienti&#239;&#172;?c Research &#226;&#8364;&#339;Protein Community&#226;&#8364;? (to K.S.) Grant- <br /> quanti&#239;&#172;?ed using Image Gauge version 4.12 (Fuji&#239;&#172;?lm).                                             in-Aid Scienti&#239;&#172;?c Research Innovative Areas &#226;&#8364;&#339;RNA Regulation&#226;&#8364;? (to K.S.). <br />  <br />  <br />  1. M&#195;&#164;&#195;&#164;tt&#195;&#164;nen P, Gehring K, Bergeron JJ, Thomas DY (2010) Protein quality control        25. Isken O, et al. (2008) Upf1 phosphorylation triggers translational repression during <br />     ER: The recognition misfolded proteins. Semin Cell Dev Biol 21:500&#226;&#8364;&#8220;511.                      nonsense-mediated mRNA decay. Cell 133:314&#226;&#8364;&#8220;327. <br />  2. Ron D, Walter P (2007) Signal integration endoplasmic reticulum unfolded             26. Hwang J, Sato H, Tang Y, Matsuda D, Maquat LE (2010) UPF1 association cap- <br />     protein response. Nat Rev Mol Cell Biol 8:519&#226;&#8364;&#8220;529.                                              binding protein, CBP80, promotes nonsense-mediated mRNA decay distinct <br />  3. Shen X, et al. (2001) Complementary signaling pathways regulate unfolded                    steps. Mol Cell 39:396&#226;&#8364;&#8220;409. <br />     protein response required C. elegans development. Cell 107:893&#226;&#8364;&#8220;903.             27. Isken O, Maquat LE (2007) Quality control eukaryotic mRNA: Safeguarding cells <br />  4. Calfon M, et al. (2002) IRE1 couples endoplasmic reticulum load secretory capacity           abnormal mRNA function. Genes Dev 21:1833&#226;&#8364;&#8220;1856. <br />     processing XBP-1 mRNA. Nature 415(6867):92&#226;&#8364;&#8220;96.                                       28. Huntzinger E, Kashima I, Fauser M, Sauli&#195;&#168;re J, Izaurralde E (2008) SMG6 catalytic <br />  5. Haze K, Yoshida H, Yanagi H, Yura T, Mori K (1999) Mammalian transcription factor               endonuclease cleaves mRNAs containing nonsense codons metazoan. RNA 14: <br />     ATF6 synthesized transmembrane protein activated proteolysis re-              2609&#226;&#8364;&#8220;2617. <br />     sponse endoplasmic reticulum stress. Mol Biol Cell 10:3787&#226;&#8364;&#8220;3799.                         29. Eberle AB, Lykke-Andersen S, M&#195;&#188;hlemann O, Jensen TH (2009) SMG6 promotes endo- <br />  6. Ye J, et al. (2000) ER stress induces cleavage membrane-bound ATF6               nucleolytic cleavage nonsense mRNA human cells. Nat Struct Mol Biol 16(1):49&#226;&#8364;&#8220;55. <br />     proteases process SREBPs. Mol Cell 6:1355&#226;&#8364;&#8220;1364.                                        30. Shen X, Ellis RE, Sakaki K, Kaufman RJ (2005) Genetic interactions constitutive <br />  7. Yoshida H, et al. (2003) A time-dependent phase shift mammalian unfolded                 inducible ge <br /> </body></html>