<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Escherichia coli Deletion Mutants Illuminate Trade-Offs <br /> between Growth Rate Flux Foreign <br /> Anabolic Pathway <br /> Kelly C. Falls&#194;&#164;a, Aimee L. Williams&#194;&#164;b, Anton V. Bryksin&#194;&#164;c, Ichiro Matsumura* <br /> Department Biochemistry, Emory University School Medicine, Atlanta, Georgia, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Metabolic engineers strive improve production yields microbial fermentations, mutating the <br />      genomes production strains. Some mutations detrimental health organism, quantitative and <br />      mechanistic understanding trade-offs inform better designs. We employed bacterial luciferase operon <br />      (luxABCDE), uses ubiquitous energetic cofactors (NADPH, ATP, FMNH2, acetyl-CoA) host cell, proxy a <br />      novel anabolic pathway. The strains Escherichia coli Keio collection, contains single deletion non- <br />      essential gene, represent mutational choices engineer make optimize fermentation yields. The Keio strains <br />      parental BW25113 strain transformed luxABCDE expression vector. Each transformant propagated <br />      defined M9 medium 37uC 48 hours; cell density (optical density 600 nanometers, OD600) luminescence <br />      measured 30 minutes. The trade-offs visualized plotting maximum growth rate luminescence/ <br />      OD600 transformant &#226;&#8364;&#732;&#226;&#8364;&#732;production possibility frontier&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Our results loss-of-function mutations <br />      enhance growth vitro light production, improvement trait generally comes expense other. <br />  <br />   Citation: Falls KC, Williams AL, Bryksin AV, Matsumura I (2014) Escherichia coli Deletion Mutants Illuminate Trade-Offs Growth Rate Flux a <br />   Foreign Anabolic Pathway. PLoS ONE 9(2): e88159. doi:10.1371/journal.pone.0088159 <br />   Editor: Patrick C. Cirino, University Houston, United States America <br />   Received August 26, 2013; Accepted January 4, 2014; Published February 4, 2014 <br />   Copyright: &#195;&#376; 2014 Falls et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Science Foundation. KF IM supported grant number EF-1104988 (NSF3); AB, AW IM were <br />   supported MCB-0951076 (NSF2). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: imatsum@emory.edu <br />   &#194;&#164;a Current address: Medical Scientist Training Program, The University Iowa, Carver College Medicine, Iowa City, Iowa, United States America <br />   &#194;&#164;b Current address: Department Cellular Biology, University Georgia, Athens, Georgia, United States America <br />   &#194;&#164;c Current address: Parker H. Petit Institute Bioengineering Bioscience, Georgia Institute Technology, Atlanta, Georgia, United States America <br />  <br />  <br />  <br /> Introduction                                                                           used homologous recombination create Keio collection, <br />                                                                                        consists 3985 isogenic Escherichia coli strains, <br />    Metabolic engineers seek optimize environmentally                            missing single open reading frame [6]. Hara et al. <br /> benign biosynthesis pharmaceuticals, materials, fuels                     subsequently propagated strain rich medium 24 hours, <br /> economically valuable compounds [1]. The economic impact                            permeated cells osmotic shock, reacted released <br /> any project, however, predicated yield. For example,                      ATP firefly luciferase substrate luciferin [7]. They <br /> the pathway produces artemisinin, anti-malarial drug,                       measured optical density (600 nm) culture, light <br /> engineered yeast received well-deserved attention [2]. It                         emission, microplate luminometer, identified numerous <br /> would not, however, cost-effective substantial,                      deletion mutations exhibited increased ATP production (per <br /> but appreciated, strain improvement fermentation process                      cell minute) stationary phase. This observation <br /> development [3]. Most economically valuable pathways                               suggests wild-type E. coli cells produce ATP processes not <br /> anabolic (energy-consuming) non-essential host                              essential growth rich medium (Lysogeny Broth +2.8% <br /> organism vitro. They utilize intermediates produced                              glucose), inactivation processes certain <br /> endogenous pathways, including common energetic cofactors                          deletion mutations increase pool intracellular ATP for <br /> that entered pathways: adenosine                            artificial anabolic processes. This interpretation, the <br /> triphosphate (ATP), nicotinamide adenine dinucleotide (NADPH),                         &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch hypothesis,&#226;&#8364;&#8482;&#226;&#8364;&#8482; sensible, domestication of <br /> flavin mononucleotide (reduced form, FMNH2), acetyl- <br />                                                                                        plants animals generally produces breeds econom- <br /> coenzyme A (acetyl-CoA). The metabolic pathways cell <br />                                                                                        ically valuable fit wild. <br /> are interconnected, remains difficult know priori how <br />                                                                                            We wondered free lunch hypothesis be <br /> perturbation pathway affect flux a <br />                                                                                        generalized energetic cofactors bacterial life <br /> heterologously expressed pathway health cell. <br />                                                                                        cycle. We employed bacterial Lux pathway utilizes <br />    The experimental tools functional genomics make possible <br />                                                                                        common exchangeable cofactors (ATP, NADPH, <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> address question. Transposons inserted nearly <br />                                                                                        FMNH2 fatty acids produced acetyl-CoA) functions <br /> random chromosomal positions, creating populations of <br />                                                                                        living cells. The bacterial lux operon encodes proteins <br /> insertion mutants [4,5]. Alternatively, Mori colleagues <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         1                             February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                             Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 1. Deletion mutations affect growth Escherichia coli, production light bacterial luciferase. E. coli Keio <br /> strains, parental BW25113 strain, transformed bacterial luciferase expression vector (pIMBB-T5-lux). The transformants were <br /> shaken minimal medium 37uC 48 hours; optical density 600 nm (green blue) luminescence (red) recorded 30 <br /> minutes. Data (A) parental lux/BW25113 control, representative Keio transformants, (B) lux/DybhC, (C) lux/Dlon (D) lux/Dfiu, are <br /> presented here. The growth curves Keio strains differ BW25113 control, luminescence curves similar in <br /> shape. <br /> doi:10.1371/journal.pone.0088159.g001 <br />  <br /> that form complexes. The LuxCDE complex uses ATP                      nucleotide triphosphate (dNTP) kits Roche Applied <br /> NADPH catalyze reduction fatty acids fatty                     Science (Indianapolis, IN). <br /> aldehydes. The LuxAB luciferase uses FMNH2 oxygen                         The polyethylene glycol (MW = 8,000) Fluka (now a <br /> catalyze conversion fatty aldehyde fatty acid,         division Sigma-Aldrich, vide infra). DNA purified with <br /> with concomitant release photon [8]. Here            QIAprep Gel Extraction Spin Miniprep kits (Qiagen, <br /> expression lux operon Keio strains,             Valencia, CA). Isopropyl-b-D-thio-galactoside (IPTG) <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> the measurement light emission cell density 48 hours              Gold Biotechnology (St. Louis, MO). All chemicals were <br /> of growth. We wild-type E. coli genome nearly                Sigma-Aldrich (St. Louis, MO). The microtiter plates were <br /> optimal growth rate light production, deletion               Nunc (96-well, clear, flat bottom) Corning (384-well, white <br /> mutations enhance trait usually diminish other.                  white clear bottom). The construction pIMBB-T5- <br />                                                                               lux expression vector previously described [9]. Briefly, series <br /> Materials Methods                                                         lux ORFs (luxA, luxB, luxC, luxD luxE) PCR amplified <br />                                                                               Photorhabdus luminescens (ATCC 29999) chromosome and <br /> Bacterial strains, media plasmids                                         separately cloned pIMBB. The internal Xba I site luxD <br />    The Keio <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>knockout</span> collection [6] parent BW25113 E. <br />                                                                               eliminated site-directed mutagenesis. Each BioBrick was <br /> coli strain obtained Hirotada Mori (Nara Institute of <br />                                                                               combined optimized ribosomal binding site (rbs). The rbs- <br /> Science Technology, Nara, Japan) March 2006, immedi- <br />                                                                               ORF BioBricks combined other, T5-promoter, <br /> ately available; stocks been <br />                                                                               lac operators form pIMBB-T5-lux. <br /> propagated. The Lysogeny Broth (or Luria-Bertani, LB) from <br /> EMD, M9 medium bacto-agar Difco. <br /> Restriction enzymes, T4 DNA ligase, Phusion DNA polymerase,                   High Throughput transformation KEIO collection with <br /> and 1 kb ladder New England Biolabs (Ipswich,                   pIMBB-T5-Lux <br /> MA). DNA oligonucleotides IDT (Coralville, IA),                   The modified polyethylene glycol/dimethyl sulfoxide (PEG/ <br /> custom DNA sequencing services provided Macrogen                      DMSO) protocol, originally developed Chung et al. [10], was <br /> (Rockville, MD). Polymerase chain reaction (PCR)-grade deoxy-                 used transform Keio strains pIMBB-T5-Lux <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                2                           February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                           Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />                                                                               (OD600 = 0.4&#226;&#8364;&#8220;0.7). Growth temperature (17 37uC) did affect <br />                                                                               transformation efficiency. <br />                                                                                  A Thermo Scientific Multidrop 384 coupled Titertek Titan <br />                                                                               plate stacker used add 20 microliters 2X TSS (2X LB, <br />                                                                               50 mM MgCl2, 50 mM MgSO4, 20% PEG 8000, 10% DMSO) <br />                                                                               containing pIMBB-T5-Lux concentration 1 ng/microliter <br />                                                                               micro-culture. Plates shaken briefly 2 minutes at <br />                                                                               600 rpm incubated ice 30&#226;&#8364;&#8220;60 minutes. The Multidrop <br />                                                                               384 dispenser used add 200 microliters LB each <br />                                                                               microculture. The microplates transferred ATR <br />                                                                               Microtitertron, shaken 33uC 1 hr 600 rpm allow <br />                                                                               expression ampicillin (Amp)-resistance gene. The dispenser <br />                                                                               used add 10 microliters ampicillin stock solution <br />                                                                               (3.5 mg/mL) (final concentration 140 micrograms/ <br />                                                                               mL. The microcultures replicated using 96-pin microplate <br />                                                                               replicator new plates; contained 200 microliters of <br />                                                                               fresh LB supplemented Amp (100 micrograms/mL), <br />                                                                               BW25113 strain, Amp kanamycin (Kan, 50 micro- <br />                                                                               grams/mL), Keio mutants. The E. coli cells transformed <br />                                                                               96 microtiter plates, resulting transformants were <br />                                                                               arrayed order configuration original <br />                                                                               (untransformed) Keio collection [6]. The E. coli micro-cultures <br />                                                                               allowed grow saturation overnight 33uC and <br />                                                                               600 rpm. Saturated cultures supplemented glycerol <br />                                                                               (final concentration 10%), shaken 2 minutes 600 rpm, <br />                                                                               frozen stored 280uC. <br />                                                                                  The transformants propagated saturation liquid LB <br />                                                                               supplemented ampicillin (and kanamycin Keio <br />                                                                               strains), reformatted 384-well microtiter plates; lux/ <br />                                                                               BW25113 replicated wells 384-well microtiter plate <br />                                                                               3747 lux/Keio strains distributed 126384- <br />                                                                               plates. The micro-cultures propagated overnight at <br />                                                                               30uC, subsequently frozen -80uC. PCR using primers <br />                                                                               designed recognize kanamycin phosphotransferase gene <br />                                                                               (used knock genes), specific adjacent regions, <br />                                                                               used confirm identities arbitrarily chosen <br />                                                                               transformed Keio strains 12 microtiter plates (data <br />                                                                               shown). <br />  <br />                                                                               Luminescence Growth Assays <br />                                                                                  Frozen, transformed Keio strains stored 384-well plates were <br />                                                                               thawed diluted 50-fold 384-pin replicator <br />                                                                               new plates; contained 50 microliters M9 <br />                                                                               supplemented 1 mM thiamine, 0.4% glucose, 250 micromo- <br />                                                                               lar isopropyl-b-D-thiogalactoside (IPTG), 100 micrograms/ <br />                                                                               mL ampicillin (no kanamycin). The kanamycin resistance marker <br />                                                                               Keio strains does affect cell growth absence of <br /> Figure 2. Light production cell normally distributed         antibiotic, knock-outs single copies multicopy genes result <br /> 384 lux/BW25113 parental control replicates (A), 3747       wild-type-like strains (data 42 Keio strains not <br /> (N = 3) mutant lux/Keio cultures (B). The average maximum luminescence <br />                                                                               shown). Each microtiter plate sampled times different <br /> (Relative Light Units) transformant divided maximum <br /> OD600, resulting values plotted histograms.                   days, recipient plates separately assayed with <br /> doi:10.1371/journal.pone.0088159.g002                                         BioTek Synergy2 microplate reader. OD600 luminescence <br />                                                                               measured 30 minute intervals 48 hours. Plates were <br /> plasmid. This method, unlike others, doesn&#226;&#8364;&#8482;t require prepa-               shaken continuously medium speed, temperature kept <br /> ration competent cells little 5&#226;&#8364;&#8220;             37uC. Absorbance read 600 nm. Luminescence was <br /> 6 hours batch. The Keio strains delivered 96-well                 recorded following settings: 1.0 sec integration time, a <br /> plates. Each seeded 96-pin microplate replicator              4.5 mm read height, 130 gain. <br /> flat 96-well plates (Nunc); contained 20 <br /> microliters fresh LB supplemented 10 mM MgSO4                     Data Analysis <br /> 50 mM 2-(N-morpholino)ethanesulfonic buffer (pH 6.1). The                        Data BioTek Synergy2 microplate reader acquired <br /> microtiter plates agitated 600 rpm ATR Micro-                   analyzed Gen5 software, exported Excel files <br /> titertron shaker cells exponential phase                (raw data available request). The derived values, namely <br />                                                                               maximum growth rate (mOD600/min), maximum optical density, <br />                                                                               maximum luminescence, integrated OD600 integrated lumi- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                3                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                         Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 3. Maximum growth rates 384 lux/BW25113 parental control replicates (A) normally distributed corrected edge effects (B). <br /> The corrected maximum growth rates 3747 (N = 3) mutant lux/Keio cultures (C) distributed widely control population of <br /> the size. <br /> doi:10.1371/journal.pone.0088159.g003 <br />  <br /> nescence, technical replicates lux/Keio              pin replicator microtiter plates containing fresh minimal <br /> plate manually combined Excel document plate.             medium supplemented ampicillin select non- <br /> Average values standard error calculated Microsoft              transformants isopropyl-b-D-thiogalactoside (IPTG) induce <br /> Excel, resulting parameters derived entire Keio            expression luxABCDE operon. Each plate shaken at <br /> collection consolidated single Excel document (Table              37uC 48 hours Synergy2 microplate spectrophotometer/ <br /> S1). Data technical replicates lux/BW25113 plate          luminometer; optical density (600 nm) light emission of <br /> were similarly combined separate document (Table S2).                  micro-culture measured 30 minutes. Three <br /> Kaleidagraph 3.5 (Synergy Software) used create                  technical replicates stock micro-culture, so <br /> figures.                                                                    parental lux/BW25113 strain propagated monitored <br />    Liquids outermost wells 384-well microtiter plates tend        1152 times (36384). <br /> to evaporate quickly situated interior;                 Light production rose cell density increased, suggesting that <br /> bacterial cultures edges increase cell density 20%          observed lag times function cell physiology rather <br /> faster middle. Such edge effects well-                sensitivity spectrophotometer. Luminescence <br /> documented [11,12], commonplace difficult avoid. To                  peaked cells entered stationary phase (Figure 1a), which <br /> demonstrate latter, parental control strain (lux/BW25113)           suggests intracellular concentration one <br /> was propagated 384 microtiter plates lids containing           energetic cofactor decreases cells left log phase, presumably <br /> standard media (50 microliters M9-ampicillin) humidity-                <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> depletion. The mutant cultures exhibit same <br /> controlled ATR Microtitertron (600 rpm 80% humidity, 33uC                patterns light production, growth patterns vary <br /> for 23 hours). The OD600 manually measured                         significantly regard lag time, maximum growth velocity, <br /> SpectraMax M5 plate reader (Molecular Devices) 5, 8                  cell death rate (Figure 1b&#226;&#8364;&#8220;c). Some mutants exhibit phase <br /> 23 hours; edge effects similar recorded growth           growth, light production diminishing phase <br /> the Biotek Synergy2 observed. We calculated              (Figure 1d); hypothesize second phase begins growth <br /> average maximum growth rate (mOD600/min) values cultures                 exported acetate commences [13], leading decreased <br /> in loops wells, outermost (A1-24, P1-         light production. <br /> 24, 1A-P, 24A-P) innermost (H8-17, I8-17) during <br /> continuous growth Synergy2. The values derived the <br />                                                                             Some lux/Keio strains exhibit faster growth, others <br /> outer loops average 1.35, 1.16 1.05-fold greater, <br /> respectively, inner loops. The maximum               produce light <br /> growth rate values cultures (lux/BW25113 lux/Keio)               We sought test free lunch hypothesis, understand <br /> the outer wells corrected multiplying 0.74,           biochemical nature tradeoffs anabolism and <br /> 0.86 0.95 respectively. Some mutants probably respond                   cellular health, focused analysis parameters <br /> differently parental control strain reductions culture       derived data. We identified maximum <br /> volume, reasoned did not.                                  luminescence (Relative Light Units) maximum optical density <br />                                                                             (OD600) culture. Both values recorded every <br />                                                                             experiment cells reached end logarithmic growth, so <br /> Results <br />                                                                             maximum luminescence divided maximum optical density <br /> The productivity bacterial luciferase pathway                     (lum/OD600) reflects rate cells produce light <br /> growth phase dependent                                                      (Figure 2). We derived maximum growth rate <br />    We attempted transform 3985 Keio strains (in 96-well              (mOD600/minute), reflects rate cells produce <br /> microtiter plates) parental BW25113 E. coli strain         cells (Figure 3). The mean standard error values both <br /> plasmid-borne luxABCDE operon. Approximately 6% (238/3985)                  parameters calculated transformed mutant (N = 3). <br /> of Keio strains failed transform repeated attempts        The maximum growth rate values corrected (Materials and <br /> our high throughput protocol (Table S3). A micro-aliquot            Methods) compensate increased evaporation wells <br /> culture, lux/BW25113 lux/Keio, transferred 384-              edges plate. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              4                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                              Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />                                                                                     The corrected maximum growth rate values 384 lux/ <br />                                                                                  BW25113 replicates normally distributed (Shapiro Wilk <br />                                                                                  test statistic = 0.98, P-value = 0.0002); values range 0.55 <br />                                                                                  0.94 mOD600/min, mean 0.68 standard <br />                                                                                  deviation 0.063 (Figure 3b). Again, mutants exhibited <br />                                                                                  greater variation did parental controls (Kolmogorov&#226;&#8364;&#8220; <br />                                                                                  Smirnov maximum D = 0.230, p,0.0000001). The comparable <br />                                                                                  maximum growth rate values lux/Keio strains ranged from <br />                                                                                  0.01 1.75, mean 0.71 standard deviation 0.17 <br />                                                                                  (Figure 3c). A population 3747 lux/BW25113 replicates should <br />                                                                                  exhibit range maximum velocities 0.46 0.90 mOD600/ <br />                                                                                  min (presuming null hypothesis mutants wild- <br />                                                                                  type-like), 148 lux/Keio transformants (3.9%) grew more <br />                                                                                  slowly expected 450 (12%) grew quickly. Previous <br />                                                                                  workers did observe E. coli strains enhanced fitness <br />                                                                                  collection 226 transposon insertion mutants [14], but <br />                                                                                  growth cells hampered constitutive <br />                                                                                  expression vector. <br />                                                                                     The fitness luminescence values associated 3747 lux/ <br />                                                                                  Keio strains plotted dimensional trade-off curve, <br />                                                                                  maximum growth rate (mOD600/min) X-axis and <br />                                                                                  maximum luminescence/maximum optical density Y-axis <br />                                                                                  (Figure 4). Economists use similar plot, &#226;&#8364;&#732;&#226;&#8364;&#732;production <br />                                                                                  possibility frontier&#226;&#8364;&#8482;&#226;&#8364;&#8482; (PPF), depict quantities two <br />                                                                                  commodities manufactured organization (or <br />                                                                                  country) given interval time fixed of <br />                                                                                  factors production. Our PPF plot (Figure 4) shows most <br />                                                                                  lux/Keio strains similar parental control (lux/BW25113) <br />                                                                                  growth rate light production. Many grow more <br />                                                                                  slowly and/or produce light control. A either <br />                                                                                  produce light ancestor strain tend grow <br />                                                                                  slowly, grow quickly produce light. We <br />                                                                                  investigated clones margins frontier see <br />                                                                                  E. coli offers free lunches. <br />  <br />                                                                                  A lux/Keio transformants outshine parental lux/ <br />                                                                                  BW25113 ancestor <br />                                                                                     Metabolic engineers generally concerned fermen- <br />                                                                                  tation yields growth rates. Chromosomal mutations that <br />                                                                                  increase productivity foreign anabolic pathway, represented <br />                                                                                  case luminescence/OD600, economically <br />                                                                                  valuable [4]. Such mutations, however, combined to <br />                                                                                  improve yields unless neutral net effect upon <br />                                                                                  cellular health. Several lux/Keio transformants exhibited statisti- <br /> Figure 4. The maximum growth rate luminescence cell 384 <br /> lux/BW25113 control replicates (A) lux/Keio transformants (B) are <br />                                                                                  cally significant improvements lux/BW25113 control in <br /> shown scatter plots. The distances points (green squares, A)          light production modest effects growth rate: DhyfC, <br /> reflect variations isogenic lux/BW25113 cultures 384              DyccC, DdcuC, DbglG, DyfiR, DyheV, DnusB, DthrL. Colonies <br /> well plate; error bars represent variations repetitions       (biological replicates, N = 6) separately propagated 300 <br /> the experiment (N = 3). The data (green squares, B) plotted             microliter cultures M9-ampicillin, diluted 1:50 same <br /> among lux/Keio transformant (blue squares, B) facilitate         medium, supplemented 250 micromolar IPTG, 96 well <br /> direct comparisons. <br /> doi:10.1371/journal.pone.0088159.g004 <br />                                                                                  microtiter plates. The plates agitated (medium) our <br />                                                                                  spectrophotometer/luminometer (Biotek Synergy 2) 24 hours; <br />    The average lum/OD600 values 384 replicates lux/                    optical density (600 nm) luminescence (Relative Light <br /> BW25113 normally distributed (Shapiro Wilks test statistic                   Units) measured 15 minutes. <br /> .0.99, P-value = 0.03). The values range 52,400                       Only clones re-tested, lux/DthrL, exhibited <br /> 113,000 RLU/OD600, average 79,800 standard                      significant improvement parental lux/BW25113, both <br /> deviation 10,400 (Figure 2a). The comparable lum/OD600 values                 cell growth (Figure 5a) light production (Figure 5b). One <br /> of 3747 lux/Keio transformants distributed broadly                  other, lux/DyveH, exhibited enhanced light production without <br /> asymmetrically (Shapiro Wilk test statistic = 0.86), ranging                significant reduction growth (data shown). It be <br /> 2,820 479,000 RLU/OD600, average 82,900                      possible combine &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch&#226;&#8364;&#8482;&#226;&#8364;&#8482; mutations produce a <br /> standard deviation 39,100 (Figure 2b). A normally distributed                 strain evinced better performance, under <br /> population 3747 lux/BW25113 replicates expected                   laboratory conditions. Three transformants, lux/DhyfC <br /> vary 43,800 116,000, 357 lux/Keio transformants                      (Figure 5c&#226;&#8364;&#8220;d), lux/DyfiR lux/DnusB produced light at <br /> (9.5%) fell range 524 (14%) above.                         expense growth. Three others, lux/DyccC, lux/DdcuC and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   5                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                           Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 5. The lux/DthrL lux/DhyfC strains exhibit improvements parental lux/BW25113. Each strain propagated (N = 6) <br /> for 24 hours M9 medium supplemented ampicillin IPTG Biotek Synergy2 plate reader (37uC, medium shaking); OD600 and <br /> luminescence recorded culture 15 minutes. The lux/DthrL strain (blue squares, A) grows quickly higher cell density <br /> than does lux/BW25113 (green squares), produces light (B). The lux/DhyfC transformant grows slowly parental control (C), but <br /> produces light (D). <br /> doi:10.1371/journal.pone.0088159.g005 <br />  <br /> lux/DblgG, produced light control lux/BW25113                   ATP. Contrary expectation, exhibited relative activity <br /> transformant (data shown), indicative observed                     (maximum luminescence divided maximum OD600) over <br /> imprecision initial screen (Figure 4a).                                double lux/BW25113 control assay (although <br />                                                                               112 strains assay exceeded criterion). Similarly, <br /> Different assays produce different results                                    seven 20 dec mutants exhibited half the <br />    Hara et al. mixed ATP Keio strains stationary phase            relative activity control (319 strains fell this <br /> with firefly luciferase luciferin substrate. They reported        threshold assay); dec mutants actually exhibited <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;cellular ATP synthetic activity,&#226;&#8364;&#8482;&#226;&#8364;&#8482; quantity light relative        higher relative activity control. The disparity not <br /> the wild-type value divided cell density (also relative wild-       surprising. The bacterial luciferase used study utilizes <br /> type value) strain. They identified 21 &#226;&#8364;&#732;&#226;&#8364;&#732;inc&#226;&#8364;&#8482;&#226;&#8364;&#8482; strains           cofactors (ATP, NADPH, FMNH2 fatty acids), and <br /> exhibited .200% relative activity (per cell, compared wild-            active log phase, firefly luciferase employed <br /> type) 20 &#226;&#8364;&#732;&#226;&#8364;&#732;dec&#226;&#8364;&#8482;&#226;&#8364;&#8482; exhibited ,50% relative activity [7]. We            Hara et al. required ATP (as luciferase exogenously <br /> expected priori strains similar phenotypes            added) assayed stationary phase. <br /> in assay, luxABCDE pathway partially dependent <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                6                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                                         Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />    Baptist et al. recently cloned lux operon downstream                        Our study inform efforts improve biosynthetic <br /> acetyl-CoA synthetase (acs) promoter. They transformed Keio                        yields genetically modified microorganisms. Engineers are <br /> strains lux expression vector used colony-based                       taught formulate quantitative models, generally useful <br /> screen identify E. coli genes normally regulate acs. The                       skill lead oversimplify. Some synthetic biologists long <br /> genes identified naturally quite different                          free-living organism minimal genome [22], we <br /> strongest effectors (vide supra). The lux operon constitutively                    showed deletions non-essential genes impart <br /> expressed experiment, regarded negative                     precipitous decreases biosynthetic yield. Photorhabdus luminescens, <br /> control study like it. Alternatively, well-                       source lux genes, apparently optimized evolution <br /> characterized reporter genes, lacZ gfp, sensitive                  produce light, outshines best deletion mutants (data not <br /> to metabolic state cell                         shown). It expresses additional specialized genes, luxG [23], <br /> appropriate genome-wide screens modifiers                               phenocopied loss-of-function mutations. E. coli <br /> particular gene interest.                                                           engineered produce 1,3 propanediol 82% theoretical <br />                                                                                        yield [24], transformable species likely prove be <br /> Discussion                                                                             better starting points production challenging target <br />    We sought test &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch hypothesis,&#226;&#8364;&#8482;&#226;&#8364;&#8482; notion                    compounds. <br /> the inactivation anabolic processes essential vitro <br /> will increase energy available competing processes. Our                         Supporting Information <br /> observations suggest &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch&#226;&#8364;&#8482;&#226;&#8364;&#8482; mutations exist                    Table S1 Lux Keio parameters. The lux/Keio strains were <br /> (Figure 5ab), vast majority mutations enhance                     propagated times 384 microtiter plates, the <br /> light production exhibit average growth rate (Figure                        luminescence optical density 600 nm measured every <br /> 5cd). We conclude light production competes                              30 minutes, maximum growth rate (max V OD), integrated <br /> production new cells. The shape PPF informs economists                       optical density (area growth curve, int OD), integrated <br /> about opportunity costs producing commodity. A                             luminescence (area luminescence curve, int lum), maximum <br /> straight line reflects constant opportunity costs, means that <br />                                                                                        optical density 600 nm (max OD), maximum luminescence (max <br /> resources specialized production output. <br />                                                                                        lum) calculated technical replicate. <br /> An inward curving PPF (Figure 4b) reflects economies <br />                                                                                        (XLSX) <br /> of scale, means opportunity cost manufacturing <br /> one output falls produced. Our plot suggests                    Table S2      Lux BW25113 parameters. The parental lux/ <br /> that mutations largest phenotypic effects tend                      BW25113 control strain used seed wells 384 well <br /> pleiotropic, means disrupt multiple pathways                      microtiter plate. Each micro-cultures propagated three <br /> normally affect growth light production opposite ways.                          times, luminescence optical density 600 nm were <br />    ATP flux does determine growth rate E. coli, ATP                     measured 30 minutes, growth light emission <br /> yield does determine cell density stationary phase [15]. The                    parameters (see Table S1 Legend) calculated. <br /> trade-offs fitness light production                      (XLSX) <br /> indirect nature. Genome-scale metabolic models used <br /> to formulate mechanistic hypotheses. The thrL gene                           Table S3 Unsuccessful transformation. The Keio strains we <br /> negative regulator threonine operon [16], deletion                       unable transform pIMBB-T5-lux expression <br /> likely increases production amino acid minimal                          vector listed. <br /> medium. YccC, Etk, membrane tyrosine autokinase                           (XLSX) <br /> participates form polysaccharide secretion [17] essential <br /> in laboratory. HyfC component hydrogenase 4,                         Acknowledgments <br /> converts formate CO2 H2 [18]. Even                                   We thank Prof. Susan Gilbert teaching IM Production <br /> comprehensive models [19,20] remain incomplete. The primary                            Possibilities Frontier model, Matsumura lab their <br /> functions quarter genes E. coli K-12, including yveH                  helpful suggestions. <br /> and yfiR, remain unknown [21]. Fundamental biochemical <br /> parameters (including concentrations, rate constants, Michaelis                    Author Contributions <br /> constants hundreds catalysts metabolites) regulatory <br /> properties annotated genes                                 Conceived designed experiments: IM. Performed experiments: <br />                                                                                        AW KF. Analyzed data: IM KF AB. Contributed reagents/materials/ <br /> experimentally determined. We hope data set <br />                                                                                        analysis tools: AB. Wrote paper: IM KF. <br /> (Table S2), like it, help refine their <br /> genome-scale models. <br />  <br /> References <br />  1 <br /> </body></html>