<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>OASIS: Online Application <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>Survival</span> Analysis of <br /> Lifespan Assays Performed Aging Research <br /> Jae-Seong Yang1, Hyun-Jun Nam1, Mihwa Seo1, Seong Kyu Han2, Yonghwan Choi1, Hong Gil Nam1,2, <br /> Seung-Jae Lee1,2,3*, Sanguk Kim1,2,3* <br /> 1 School Interdisciplinary Bioscience Bioengineering, Pohang University Science Technology, Pohang, South Korea, 2 Department Molecular Life Science, <br /> Pohang University Science Technology, Pohang, South Korea, 3 World Class University Information Technology Convergence Engineering, Pohang University of <br /> Science Technology, Pohang, South Korea <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: Aging fundamental biological process. Characterization genetic environmental factors that <br />      influence lifespan crucial step understanding mechanisms aging organism level. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> capture the <br />      different effects genetic environmental factors lifespan, appropriate statistical analyses needed. <br />  <br />      Methodology/Principal Findings: We developed online application survival analysis (OASIS) helps conduct <br />      various novel statistical tasks involved analyzing survival data user-friendly manner. OASIS provides standard survival <br />      analysis results including Kaplan-Meier estimates mean/median survival time taking censored survival data. OASIS <br />      provides various statistical tests including comparison mean survival time, overall survival curve, survival rate at <br />      specific time point. To visualize survival data, OASIS generates survival log cumulative hazard plots enable <br />      researchers easily interpret experimental results. Furthermore, provide statistical methods analyze <br />      variances survival datasets. In addition, users analyze proportional effects risk factors survival. <br />  <br />      Conclusions/Significance: OASIS provides platform essential facilitate efficient statistical analyses survival data <br />      field aging research. Web application detailed description algorithms accessible http://sbi. <br />      postech.ac.kr/oasis. <br />  <br />   Citation: Yang J-S, Nam H-J, Seo M, Han SK, Choi Y, et al. (2011) OASIS: Online Application Survival Analysis Lifespan Assays Performed Aging <br />   Research. PLoS ONE 6(8): e23525. doi:10.1371/journal.pone.0023525 <br />   Editor: Enrico Scalas, Universita&#226;&#8364;&#8482; del Piemonte Orientale, Italy <br />   Received April 18, 2011; Accepted July 19, 2011; Published August 15, 2011 <br />   Copyright: &#195;&#376; 2011 Yang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Core Research Center Systems Bio-Dynamics (20090091509); World Class University program (R31-2010- <br />   000-10100-0), POSTECH BRI Grant. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: seungjaelee@postech.ac.kr (S-JL); sukim@postech.ac.kr (SK) <br />  <br />  <br />  <br />  <br /> Introduction                                                                           characteristics aging relatively homogeneous <br />                                                                                        individuals controlled environmental conditions often <br />    During decades, witnessed explosion                     display variations lifespan [10,11]. That is, populations <br /> the field aging research. For identifying mechanisms aging,                      homogeneous genetic background precipitous <br /> many approaches attempted discover genetic                            survival curve specific time point display <br /> environmental factors regulate aging various organisms                         gradual survival curve. One possible explanation this <br /> [1,2]. A key experiment examining effects genetic                           phenomenon stochastic components epigenetic <br /> modulation chemical compounds aging measurement                           switch noisy gene expression, influenced some <br /> of lifespan, requires analysis survival time                      unknown factors, play important role variation in <br /> aging processes. By performing appropriate statistical analyses                     lifespan. In addition, genetic components suggested to <br /> survival data, extract wealth useful information (Table                   contribute variances lifespan [12]. Analyzing contribu- <br /> S1) [3&#226;&#8364;&#8220;6]. For example, log-rank (Mantel-Cox) test                               tion factors require novel statistical test can <br /> introduced determine experimental treatments signif-                        quantify variances lifespan data. <br /> icantly affected lifespan [7,8]. In addition, analysis hazard                    Here report open-access service survival analysis, the <br /> function lifespan data gained popularity                          online application survival analysis (OASIS) provides <br /> shape cumulative hazard plots proposed reflect                      canonical survival analysis methods advanced <br /> the rate aging [9]. Therefore, accurate efficient execution                  statistical tests comparing variances survival datasets. <br /> statistical analyses crucial step better understanding                  OASIS user-friendly online application runs a <br /> of aging molecular level.                                                       browser downloading installation. These features of <br />    Despite development statistical analyses lifespan data,                   OASIS help researchers field aging research <br /> there need developing statistical methods explain                  analyze data depth potentially facilitate the <br /> complex phenomena involved aging. One interesting                            standardization survival analysis. <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                 1                                August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                            Online Application Survival Analysis <br />  <br />  <br />  <br /> Results                                                                     di number deaths group 1, ei (estimated as <br />                                                                             &#195;&#176;di n1i &#195;&#382;=ni ) number expected deaths group 1. n1i the <br /> OASIS web application                                                       size population group 1 risk ith interval, and <br />    To provide standardized platform biologists aging               ni total size population risk ith interval. <br /> research fields perform survival analyses, developed OASIS               1.2 Fisher&#226;&#8364;&#8482;s exact test. Fisher&#226;&#8364;&#8482;s exact test frequently used <br /> server accessible using majority modern web              survival analysis [6,13,18&#226;&#8364;&#8220;24]. To test different survival <br /> browsers. After surveying experiments, recorded survival data           functions specific time points instead overall lifespan, the <br /> imported input OASIS web server simple format                 program calculate probability observed data with <br /> (Figure 1A). The input format consist following items        Fisher&#226;&#8364;&#8482;s exact test different time points using following <br /> given order: experimental identifier observed data. The              formula. <br /> line started &#226;&#8364;&#732;&#226;&#8364;&#732;%&#226;&#8364;&#8482;&#226;&#8364;&#8482; sign indicates experimental identifier. <br /> Observed data columns: observed time, the <br /> number dead subjects, number censored (e.g.                                           !              !,                      ! <br />                                                                                               azb         czd                 azbzczd <br /> missing) subjects observation interval (Figure 1B;                         pt ~                                                     ~ <br /> sample inputs available OASIS webpage). These                                                    c                   azc <br />                                                                                                                                                    ; <br /> columns separated tabs.                                                       (azb)!(czd)!(azc)!(bzd)! <br />    OASIS composed statistical frameworks can <br />                                                                                                 (azbzczd)!a!b!c!d! <br /> facilitate proper analysis survival data (Figure 1C). In basic <br /> survival analysis, OASIS provides various statistical methods          b numbers living subjects group 1 and <br /> as Kaplan-Meier statistics, mean median survival time,                  group 2 respectively c d dead subjects group <br /> survival log cumulative hazard plots depicting                  1 group 2 respectively specific time t. P-value Fisher&#226;&#8364;&#8482;s <br /> characteristics dataset (Figure 2). For statistical compar-         exact test calculated sum probabilities equal <br /> isons different survival datasets, OASIS performs hypothesis             pt combinations. Generally, 90% mortality rate used for <br /> testing log-rank test, Fisher&#226;&#8364;&#8482;s exact test, Kolmogorov-             Fisher&#226;&#8364;&#8482;s exact test. However, cases, comparisons between <br /> Smirnov test, Neyman&#226;&#8364;&#8482;s smooth test (Figure 3). In particular,           datasets 90% mortality statistically significant <br /> survival time F-test, partial slopes rank-sum test, normalized          difference reasons including drastic death an <br /> chow test novel testing methods comparison               old age. This suggests want emphasis <br /> shape survival hazard functions. Furthermore, OASIS                   earlier deaths later ones later deaths result <br /> generates hazards regression evaluate effect        causes unrelated normal aging. <br /> risk factors (Figure 4). By using outputs produced OASIS,             1.3 Weighted log-rank test. As mentioned previous <br /> users interpret survival data comprehensively.                          paragraph, want emphasis earlier deaths <br />                                                                             later ones vice versa. To generalize log-rank test for <br /> Statistical testing differences <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> lifespan               needs, Fleming Harrington developed G(rho, gamma)- <br />    A comprehensive comparison survival datasets               weighted log-rank test [25,26]. The weighted test statistics is <br /> experimental group control group important determine            calculated following equation. <br /> the effects experimental treatments survival. For example, <br /> some drug treatments increase average survival time, <br /> whereas increase average maximum survival                                            2                        32 <br /> times. Therefore, distinguish differences OASIS provides                                            X <br />                                                                                                          4             wi (di {ei )5 <br /> various statistical comparison methods (Figure 3). For example,                                          i:ti &#198;&#8217;t <br />                                                                                                     2 <br /> statistics average survival time obtained using log-                                   x ~X                               ; <br /> rank test, specific time point obtained                                               w2i Var(di {ei ) <br />                                                                                                          i:ti &#198;&#8217;t <br /> using Fisher&#226;&#8364;&#8482;s exact test [13]. OASIS provides log-rank test, <br /> Fisher&#226;&#8364;&#8482;s exact test, tests explained the <br />                                                                             wi represents G(rho, gamma) weight defined S(t)rho(1- <br /> followings. <br />                                                                             S(t))gamma. Generally rho .0 gamma = 0, test sensitive <br />    1.1 Log-rank test. Mantel-Cox test, so-called log-rank test, is <br />                                                                             early differences, rho = 0 gamma .0, test is <br /> a kind nonparametric test frequently used comparing <br />                                                                             sensitive later differences [26]. <br /> two survival functions overall lifespan data [7,8,14&#226;&#8364;&#8220;17]. <br /> The log-rank statistics groups experimental a <br /> control groups calculated follows.                                    Statistical testing differences shape hazard <br />                                                                             function <br />                                                                                2.1 Kolmogorov-Smirnov test. While log-rank test is <br />                            2                   32                           commonly used comparing survival data samples, is <br />                                X <br />                            4           (di {ei )5                           optimized special assumptions underlying distributions <br />                              i:ti &#198;&#8217;t                                        hazard ratio relative risk l2(t) / l1(t) constant in <br />                       x2 ~ X                        ;                       time t. In case, log-rank test generally gives optimal results. <br />                                      Var(di {ei )                           However, need statistical tests depend the <br />                            i:ti &#198;&#8217;t <br />                                                                             distribution survival data. The Kolmogorov-Smirnov test is <br />                                                                             suitable purpose robustly works condition <br />                                                                             hazard functions l1(t) l2(t) cross time <br />                                di (ni {di )n1i (ni {n1i )                   t. The Kolmogorov-Smirnov test based following <br />               Var(di {ei )~                               ; <br />                                       ni 2 (ni {1)                          equation. <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                               August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                               Online Application Survival Analysis <br />  <br />  <br />  <br />  <br /> Figure 1. Statistical components OASIS. (A) Overall flowchart survival analysis (B) Input basic survival analysis statistical testing. <br /> Observed data columns; time observation started, number dead subjects, number censored subjects during <br /> the interval. (C) The web application provides uniform platform comprises analysis parts: basic survival analysis, statistical testing, and <br /> Cox proportional hazards regression. <br /> doi:10.1371/journal.pone.0023525.g001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           3                             August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                         Online Application Survival Analysis <br />  <br />  <br />  <br />  <br /> Figure 2. Results basic survival analysis. (A) The output Kaplan-Meier estimator. (B) Mean/median survival time data. Survival time at <br /> 25%, 50%, 75%, 90%, 100% mortality shown. (C) Survival log cumulative hazard plots. <br /> doi:10.1371/journal.pone.0023525.g002 <br />  <br />                                                                            different survival data comparing null model, S1(t) = S2(t) to <br />                                                                            various alternative models. The alternative models embedded the <br />                   D&#194;&#189;0,t ~ sup jS1 (t){S2 (t)j ;                            null model Legendre polynomials based Neyman&#226;&#8364;&#8482;s <br />                             0&#198;&#8217;t&#198;&#8217;t                                          goodness-of-fit idea following equation. <br />  <br /> where sup represents supremum set gives smallest <br /> real number greater equal number the <br /> set D represents largest absolute vertical deviation. OASIS                             S2 (t)~S1 (t) expfhT y(t)g ; <br /> adopted surv2.ks function implemented R packages [27] to <br /> provide Kolmogorov-Smirnov test. We note Kolmogorov-              h = (h1,&#226;&#8364;&#166;, hd)T parameter set bounded functions y <br /> Smirnov test OASIS applicable survival data              (t) = (y1(t), &#226;&#8364;&#166;, yd(t))T, models possible differences between <br /> contain tied observations [e.g. multiple events (deaths failures)       S1(t) S2(t). Therefore, h = 0, null hypothesis accepted. <br /> during observed time interval]. OASIS provides warning                Since Neyman&#226;&#8364;&#8482;s smooth test selects optimal smooth model in <br /> message tied observation survival data        Legendre polynomials Schwarz&#226;&#8364;&#8482;s selection rule, different <br /> between samples.                                                           Kolmogorov-Smirnov test respect providing idea <br />    2.2 Neyman&#226;&#8364;&#8482;s smooth test. Another statistical comparison                types difference survival data [28]. The <br /> method detecting wide spectrum alternatives Neyman&#226;&#8364;&#8482;s           selected dimension represents type difference S1(t) <br /> smooth test. It developed test homogeneity               S2(t). If selected dimension (d) 1, suggests S1(t) is <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       4                             August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                        Online Application Survival Analysis <br />  <br />  <br />  <br />  <br /> Figure 3. Results statistical testing input format. Output statistical tests differences lifespan data samples. <br /> doi:10.1371/journal.pone.0023525.g003 <br />  <br />  <br />  <br />  <br /> different S2(t) constant hazard ratio. If selected d       used analysis detecting structural differences between <br /> 2, relationship samples likely monotonic.            different log cumulative hazard functions using the <br /> If selected d 3, relationship samples likely        following equation. <br /> to convex concave form. OASIS adopted surv2.neyman <br /> function implemented R packages [28] provide the <br /> Neyman&#226;&#8364;&#8482;s smooth test. Similar Kolmogorov-Smirnov test,                                     (RSSp {(RSS1 zRSS2 ))=k <br /> the Neyman&#226;&#8364;&#8482;s smooth test currently applicable                            F~                              ; <br />                                                                                                 (RSS1 zRSS2 )=(N1 zN2 {2k) <br /> are tied observations survival data. <br />    2.3 Chow test. Chow test, variant F-test, invented                RSSp represents sum squared residuals pooled <br /> by economist Gregory Chow test coefficients                 log cumulative hazard data. RSS1 RSS2 represent sum of <br /> two linear regressions different data sets                 squared residuals different log cumulative hazard data <br /> [29]. This test generally used detecting structural break              respectively. N1 N2 numbers observation each <br /> that unexpected shift time series data. In OASIS,                 data k, 3 case, total number of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                         August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                             Online Application Survival Analysis <br />  <br />  <br />  <br />  <br /> Figure 4. Results Cox proportional hazard regression input format. (A) Input Cox proportional hazard regression. Observed <br /> data columns; observed time, status events (dead censored), values risk factors. (B) Output Cox proportional <br /> hazard regression. <br /> doi:10.1371/journal.pone.0023525.g004 <br />  <br /> parameters linear regression model. The test statistic follows             3.2 Partial slopes rank-sum test. We devised another <br /> F distribution (k, N1 + N2 &#226;&#8364;&#8220; 2k) degrees freedom.                      statistical test method comparing differences slopes of <br />                                                                                log cumulative hazard plots. We calculated partial slopes the <br /> Statistical testing differences variances                        log cumulative hazard plot. With null hypothesis different <br /> survival time                                                                  log cumulative hazard plots slopes, conducted rank- <br />    3.1 Survival time F-test. We provide survival time F-test,                  sum tests set partial slopes following definitions. <br /> which used examine normal populations have <br /> the variance not. Because censored data generally <br /> used survival analysis, estimate number dead <br />                                                                                         ln ({ ln (S1 (ti ))){ ln ({ ln (S1 (ti{1 ))) <br /> animals using survival function S(t) perform F-test               D1 ~f                                             ji~2:::kg; <br /> comparison variances different survival data. The F-test                                        ti {ti{1 <br /> is used condition survival times individuals <br /> follow normal distribution. As normality check method a <br /> given dataset, provide Shapiro-Wilk test OASIS                             ln ({ ln (S2 (tj ))){ ln ({ ln (S2 (tj{1 ))) <br />                                                                                     D2 ~f                                             jj~2:::lg; <br /> website. If P-value generated Shapiro-Wilk test                                              tj {tj{1 <br /> smaller 0.01, chance survival data following the <br /> normal distribution 0.01. In case, provide              D1 D2 sets partial slopes group. These sets <br /> warning message results F-test applicable.              compared rank-sum test. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          6                            August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                          Online Application Survival Analysis <br />  <br />  <br />    The partial slopes rank-sum test based non-parametric <br /> statistics requires sufficient number samples (in case, <br /> partial slopes) reliable analysis. Since partial slope is <br /> defined changes log cumulative hazard divided the <br /> corresponding change survival time neighbouring <br /> time points, number observed time points the <br /> total sample size important non-parametric analysis. To <br /> obtain statistically meaningful results, observed time <br /> points needed. <br />    3.3 Normalized chow test. Chow test used testing <br /> whether coefficients linear regressions different <br /> datasets [29]. However, researchers perform <br /> survival analysis tend interested examining difference <br /> in slope determining difference y-intersect. <br /> For purpose, conducting Chow test, normalized the <br /> log cumulative hazard data mean zero. In case, <br /> the linear regression dataset zero y-intersect. Thus, one <br /> can examine differences slopes datasets pooled <br /> data. We verified difference lifespan variations through <br /> normalized Chow test, statistical test examines the <br /> coefficients linear regressions different normalized data <br /> sets equal. Similar log-rank test, assumption that <br /> survival rate constant time apply normalized Chow <br /> test. <br />  <br /> Application basic survival analysis experiments <br />     As test case, analyzed lifespan dataset roundworm <br /> C. elegans using OASIS. Mutations daf-2, encodes an <br /> insulin/IGF-1 receptor homolog, extend lifespan C. elegans <br /> [30,31]. The FOXO transcription factor DAF-16 required for <br /> daf-2 mutants live long [32-35]. We measured lifespan of <br /> wild-type daf-2(e1370) mutant animals without <br /> knocking daf-16 transcription factors using RNA <br /> interference. OASIS provided result daf-2(e1370) <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> significantly extended lifespan decreased mortality <br /> rate (Figure 5). By using OASIS, generated survival log <br /> cumulative hazard plots illustrated differences (Figure 5). <br /> Among groups experiments, daf-2(e1370) mutants <br /> showed different survival <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> using graphs. <br /> The survival curve daf-2(e1370) mutants shifted right <br /> compared (Figure 5A). In addition, y-intercept          Figure 5. Survival analysis wild-type daf-2 mutant <br />                                                                              C. elegans combination daf-16 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>RNA interference</span>. (A) <br /> log cumulative hazard plot daf-2(e1370) mutants smaller               Survival plots (B) log cumulative hazard plots wild type, daf-2 <br /> than (Figure 5B). To statistically validate            mutant [daf-2(e1370)], daf-16(RNAi), daf-2 mutant treated daf- <br /> results daf-2(e1370) mutants live long, compared mean            16 RNAi [daf-16(RNAi); daf-2(e1370)] animals. <br /> lifespan conducted various statistical tests. The mean lifespan          doi:10.1371/journal.pone.0023525.g005 <br /> of daf-2(e1370) mutants larger 30 days, of <br /> other groups approximately 20 days (Table 1). By using log-              survival variations reach conventional <br /> rank test Fisher&#226;&#8364;&#8482;s exact test, determined statistical             analysis. Three different types hypothetical survival datasets, A, <br /> significance lifespan differences. By using log-rank test,       B C generated developing test (Figure 6). <br /> showed wild type daf-2(e1370) significantly                Datasets A B mean lifespans (20 days) but <br /> different mean lifespans (P,1.0610210), wild type                different lifespan variances (2 days A; 4 days B), whereas <br /> and daf-2(e1370) mutants treated daf-16 RNAi similar               dataset B C lifespan variances (4 days) but <br /> mean lifespans (P = 0.113). In addition, proportions survivors         different mean lifespans (20 days B; 30 days C). The <br /> specific time points compared using Fisher&#226;&#8364;&#8482;s exact tests.          characteristics dataset depicted survival <br /> These statistical analyses acquired using OASIS consistent            (Figure 6A) log cumulative hazard plots (Figure 6B) the <br /> with previous reports [30&#226;&#8364;&#8220;35]. Furthermore, examined                  differences lifespan variations evident. It difficult to <br /> OASIS analysis comparing publicly available datasets             distinguish lifespan variations using conventional methods for <br /> confirmed validity (Table S2).                                           survival analysis. Specifically, log-rank test suitable the <br />                                                                              comparison mean lifespan, Neyman&#226;&#8364;&#8482;s test effective <br /> Advanced statistical analyses comparison                          distinguish structural differences survival curves <br /> variances survival data                                                   (Figure 6C). In contrast, statistical methods we <br />    To examine usefulness advanced statistical methods              implemented OASIS, survival time F-test, partial slopes <br /> OASIS, conducted model-based test different                 rank-sum test, normalized Chow test, able to <br /> statistical methods implemented OASIS work complex                     discriminate variances lifespan data (Figure 6C). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                            August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                                           Online Application Survival Analysis <br />  <br />  <br />  <br />  Table 1. Statistical analysis lifespan data daf-2 mutants combination daf-16 RNAi knock down. <br />  <br />  <br />  Statistics <br />  <br />  Name                     No. subjects           Restricted mean                                        Age days % mortality (days) <br />  <br />                                                     Days          Std. Err.         95% C.I.               25%         50%           75%            90%              100% <br />  <br />  Wild type                52                        19.35         1.07              17.24,21.45            15          18            25             30               32 <br />  daf-2(e1370)             223                       32.87         0.64              31.63,34.12            29          34            39             41               - <br />  daf-16(RNAi)             127                       17.74         0.43              16.91,18.57            15          18            22             25               28 <br />  daf-16(RNAi); daf-       366                       20.84         0.46              19.94,21.73            15          21            24             29               39 <br />  2(e1370) <br />  <br />  Comparison samples <br />  <br />  Type             Name                                  Log-rank test                            Fisher exact test % mortality (P-value) <br />                                                             2                        2 <br />                                                         x                Prob . x               25%               50%                  75%                90% <br />  <br />  Control          Wild type <br />                   daf-2(e1370)                          119.2            0.00E+00*              4.04E-11          1.09E-13             7.53E-07           9.35E-05 <br />                   daf-16(RNAi)                          5.03             2.50E-02               1.00E+00          2.52E-01             6.03E-02           3.69E-03 <br />                   daf-16(RNAi); daf-2(e1370)            2.51             1.13E-01               5.55E-02          4.15E-01             1.83E-01           2.59E-02 <br />  Control          daf-2(e1370) <br />                   Wild type                             119.2            0.00E+00*              4.04E-11          1.09E-13             7.53E-07           9.35E-05 <br />                   daf-16(RNAi)                          206.8            0.00E+00*              1.16E-13          1.10E-12             8.49E-13           1.06E-10 <br />                   daf-16(RNAi); daf-2(e1370)            146.1            0.00E+00*              2.69E-10          9.56E-13             1.60E-12           5.06E-02 <br />  <br />  Users compare restricted mean &#226;&#8364;&#732;&#226;&#8364;&#732;maximum survival time,&#226;&#8364;&#8482;&#226;&#8364;&#8482; generally 90th percentile survival. C.I. indicates confidence interval. In case daf- <br />  2(e1370) age days 100% mortality (days) determined &#226;&#8364;&#732;&#226;&#8364;&#732;-&#226;&#8364;&#8482;&#226;&#8364;&#8482;, individual C. elegans survived censored. * The log-rank test OASIS <br />  provides &#226;&#8364;&#732;0.00E+00&#226;&#8364;&#8482; P,1.0610210. <br />  doi:10.1371/journal.pone.0023525.t001 <br />  <br />  <br />  <br />  <br />     We applied statistical methods survival experi-                            Discussion <br /> ments C. elegans. We analyzed lifespan data daf-2(e1370) <br /> mutant C. elegans treated RNAi targeting daf-16/FOXO                                    Rigorous analysis survival data crucial aging research. <br /> with RNAi targeting mag-1, encodes C. elegans homolog                             Several statistical tools survival analysis available (Table <br /> Drosophila mago nashi known regulate hermaphrodite germ-line                              S3), time, provide OASIS the <br /> sex determination [36]. We daf-16 RNAi mag-1                             comprehensive analysis survival data including generation of <br /> RNAi daf-2(e1370) mutant background significantly shortened                               KAPLAN-MEIER statistics, visualization survival log <br /> lifespan compared control RNAi (P,1.0610210, log-rank test)                               cumulative hazard plots, statistical test hypothesis, hazards <br />                                                                                              regression. OASIS based Django [38], python-based web <br /> (Figure 7A). We daf-16 RNAi- mag-1 RNAi-treated <br />                                                                                              framework, R statistical environment [27] integrate <br /> daf-2(e1370) mutants similar mean lifespans (P = 0.82, log-rank <br />                                                                                              essential advanced statistical features user-friendly <br /> test). However, analyses lifespan variances using OASIS <br />                                                                                              graphical interface survival analysis. <br /> revealed mag-1 RNAi resulted significantly different <br />                                                                                                 Historically, comparison average survival time predom- <br /> variance lifespan compared daf-16 RNAi (Figure 7B, <br />                                                                                              inantly used analysis survival data. However, many <br /> P,1.0610210, survival time F-test; P,0.05, partial slopes rank-sum <br />                                                                                              cases survival data comparisons shown examples <br /> test; P,1.0610210, normalized Chow test). As example,                             (Figure 7), survival curves obviously different shapes may <br /> analyzed lifespan datasets wild-type tax-2(p671); isp-                            similar mean survival times researchers may <br /> 1(qm150) double mutant C. elegans, mutations tax-2 [a                           conclude differences survival datasets not <br /> subunit cyclic-nucleotide gated calcium channel] isp-1 [an                          statistically significant. As described recently [10], a <br /> iron-sulfur protein complex III respiratory chain]. In                         population organisms relatively homogeneous genetic <br /> previous report, showed mean lifespan values                            environmental factors showed gradual lifespan curves instead <br /> two lifespan datasets similar (P = 0.12, log-rank test)                         sharp precipitated line. Stochastic factors including epigenetic <br /> lifespan curves crossed [37]. Using OASIS,                                switch noisy gene expression [10] and/or genetic components <br /> found difference variances lifespan                         [12] underlie phenomenon crucial to <br /> datasets statistically significant (P,1.0610210, survival time F-                         identify characterize factors future. Here, we <br /> test; P,0.05, partial slopes rank-sum test; P,0.05, normalized                               provide advanced statistical methods analyzing differences <br /> Chow test). Together, analyses indicate advanced                              variances survival datasets believe methods <br /> statistical methods useful distinguishing differences                         useful objectively quantify variances based on <br /> lifespan variances. Moreover, results shown using C. elegans                        statistical significance. <br /> mutants RNAi suggest genetic components underlie                               For proper survival analysis, OASIS users consider <br /> in cases lifespan variances.                                                     underlying statistical assumptions. For example, log-rank test is <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                       8                            August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                               Online Application Survival Analysis <br />  <br />  <br />  <br />  <br /> Figure 6. Advanced statistical results analyzing differences survival variations using hypothetical datasets. (A) Survival plots <br /> and (B) log cumulative hazard plots different types hypothetical survival data. (C) Concept figures statistical test illustrated. P-values <br /> of statistical test shown. Significant differences indicated bold font red. <br /> doi:10.1371/journal.pone.0023525.g006 <br />  <br /> suitable baseline survival rate changing time. In           methods multiple statistical comparisons. Together the <br /> addition, advanced statistical tests, survival time F-test,            consideration assumptions, performing lifespan experiments <br /> partial slopes rank-sum test normalized chow test, used               times independently reliable results for <br /> identify differences lifespan variations based specific                distinguishing differences lifespan variations. <br /> assumptions. In statistical testing, OASIS automatically generates <br /> all pair-wise comparison results, increase type I error [39].          Availability Requirements <br /> To adjust multiple testing, OASIS provides corrected P-values                  The software available public use http://sbi.postech.ac. <br /> with Bonferroni method, commonly used correction                 kr/oasis. <br />  <br />  <br />  <br />  <br /> Figure 7. Examples lifespan variations experimental datasets. (A) Survival plots daf-2(e1370) mutants treated control <br /> RNAi, mag-1 RNAi, daf-16 RNAi. (B) Gaussian fitting curves survival time F-test analyzes variances daf-2(e1370) mutants treated <br /> with control RNAi, mag-1 RNAi, daf-16 RNAi. <br /> doi:10.1371/journal.pone.0023525.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            9                             August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                                               Online Application Survival Analysis <br />  <br />  <br />   Because researchers&#226;&#8364;&#8482; unpublished data uploaded                                                  hi (tjXi,1 ,:::,Xi,k ) <br /> website, OASIS encrypts input data does permit access of <br /> other users ensure security privacy.                                                                  ~h0 (t) exp (b1 Xi,1 zb2 Xi,2 zb3 Xi,3 :::zbk Xi,k ) <br />                                                                                                              ~h0 (t) exp (bT Xi ) ; <br /> Methods <br /> Basic survival analysis <br />                                                                                            Xi,1, &#226;&#8364;&#166;, Xi,k represent k risk factors assumed act <br />    To estimate survival time area survival curve, it <br />                                                                                            independently, b1, &#226;&#8364;&#166;, bk regression coefficients, h0(t) is <br /> is necessary characterize survival function a <br />                                                                                            baseline hazard time t, subscript observation. <br /> probability death specific time t. <br />                                                                                            OASIS provides standard Cox proportional-hazards regression <br />                                                                                            robust methods [43] (Text S1). Both methods provide <br />                                                                                            regression coefficient risk factors statistical signif- <br />                                S(t)~P(Twt);                                                icance. <br />                                                                                               To identify risk factors explain hazard function with <br /> where t specific time, T random variable time,                         proportion, input data format different of <br /> and P denotes probability death.                                                    survival analysis. As shown Figure 4A, OASIS takes following <br />    Generally, observe censored data include missing                           format input data Cox proportional hazards regression in <br /> subjects. To consider missing subjects, Kaplan-Meier                             given order; fields observed data. The line <br /> estimator proposed 1958 [40,41] right censored data                             starting &#226;&#8364;&#732;&#226;&#8364;&#732;%&#226;&#8364;&#8482;&#226;&#8364;&#8482; sign indicates fields. Observed data <br /> analysis. The survival function S(t) estimated Kaplan-Meier                      columns; observed time, status events <br /> method following formula.                                                               (dead not), values risk factors. Other columns are <br />                                                                                            considered values risk factors. These columns are <br />                                                                                            separated tabs. <br />                     _                _          dj <br />                     S (ti )~ P p j ~ P (1{ );                                              Supporting Information <br />                             j:tj &#198;&#8217;ti   j:tj &#198;&#8217;ti nj <br />                                                                                            Text S1 Detailed explanations statistical methods for <br />         _ <br /> where p j conditional probability survival jth                        survival analyses <br /> interval, dj number deaths nj size                             (PDF) <br /> population risk jth interval.                                                Table S1        Statistical methods used survival analyses <br />     Using Kaplan-Meier estimator, OASIS gives various outputs                          (PDF) <br /> such variance survival function, mean median lifespan, <br /> lifespan % mortality, survival plots, log-cumulative hazard                         Table S2 Comparisons OASIS results reference <br /> plots (Figure 2). These outputs generally character-                          data <br /> istics survival data. For example, slope log-cumulative                          (PDF) <br /> hazard plots indicates age-dependent increase death rate                            Table S3 Web-based statistical methods survival <br /> and Y-intercept represents hazard rate beginning                         analysis <br /> the observation. More detailed explanations Text S1.                            (PDF) <br />  <br /> Cox proportional hazard regression                                                         Acknowledgments <br />    Whereas hypothesis testing useful comparing survival                           We sincerely thank Dr. Kraus comments Kolmogorov-Smirnov <br /> data groups, Cox proportional-hazards                                    test Neyman&#226;&#8364;&#8482;s smooth test implemented R package, the <br /> regression suitable analyzing proportional effects                           reviewers valuable comments improved manuscript. We are <br /> several risk factors survival [42]. OASIS provides Cox                                  grateful members S. Kim lab S.J. Lee lab helpful <br /> proportional hazards regression, evaluate effect                          suggestions OASIS manuscript. <br /> several risk factors sex, age, weight survival <br /> (Figure 4B). Mortality rate explained proportional                           Author Contributions <br /> sum risk factors. Cox formulated semi-parametric model                             Conceived designed experiments: J-SY H-JN S-JL SK. Performed <br /> the following equation.                                                                    experiments: J-SY MS S-JL. Analyzed data: J-SY H-JN S-JL. <br />                                                                                            Contributed reagents/materials/analysis tools: J-SY MS SKH YC HGN. <br />                                                                                            Wrote paper: J-SY H-JN S-JL SK. <br />  <br /> References <br />  1. Guarente L, Partridge L, Wallace DC (2007) Molecular biology aging;                  6. Harrison DE, Strong R, Sharp ZD, Nelson JF, Astle CM, et al. (2009) <br />     Guarente L, Partridge L, Wallace DC, eds. CSHL Press.                                      Rapamycin fed late life extends lifespan genetically heterogeneous mice. <br />  2. Antebi A (2007) Genetics aging Caenorhabditis elegans. PLoS Genet 3:                 Nature 460: 392&#226;&#8364;&#8220;395. <br />     1565&#226;&#8364;&#8220;1571.                                                                              7. Mantel N (1966) Evaluation survival data new rank order statistics <br />  3. Wu D, Cypser JR, Yashin AI, Johnson TE (2009) Multiple mild heat-shocks                    arising consideration. Cancer Chemother Rep 50: 163&#226;&#8364;&#8220;170. <br />     decrease Gompertz component mortality Caenorhabditis elegans. Exp             8. Peto R, Pike MC (1973) Conservatism approximation sigma (O-E)2-E the <br />     Gerontol.                                                                                  logrank test survival data tumor incidence data. Biometrics 29: 579&#226;&#8364;&#8220;584. <br />  4. Honjoh S, Yamamoto T, Uno M, Nishida E (2009) Signalling RHEB-1                 9. Luder HU (1993) Onset human aging estimated hazard functions <br />     mediates intermittent fasting-induced longevity C. elegans. Nature 457:                 associated various causes death. Mech Ageing Dev 67: 247&#226;&#8364;&#8220;259. <br />     726&#226;&#8364;&#8220;730.                                                                               10. Kenyon CJ The genetics ageing. Nature 464: 504&#226;&#8364;&#8220;512. <br />  5. Valenzano DR, Terzibasi E, Genade T, Cattaneo A, Domenici L, et al. (2006)             11. Herndon LA, Schmeissner PJ, Dudaronek JM, Brown PA, Listner KM, et al. <br />     Resveratrol prolongs lifespan retards onset age-related markers            (2002) Stochastic genetic factors influence tissue-specific decline ageing C. <br />     short-lived vertebrate. Curr Biol 16: 296&#226;&#8364;&#8220;300.                                             elegans. Nature 419: 808&#226;&#8364;&#8220;814. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                    10                                  August 2011 | Volume 6 | Issue 8 | e23525 <br />                                                                                                                                   Online Application Survival Analysis <br />  <br />  <br /> 12. Promislow DE, Tatar M, Khazaeli AA, Curtsinger JW (1996) Age-specific                       26. Fleming TR, Harrington DP (1991) Counting Processes Survival Analysis. New <br />     patterns genetic variance Drosophila melanogaster. I. Mortality. Genetics 143:            York: Wiley. <br />     839&#226;&#8364;&#8220;848.                                                                                    27. R Development Core Team contributors worldwide, (2008). <br /> 13. Fisher RA (1922) On interpretation x2 contingency tables,               28. Kraus D (2009) Adaptive Neyman&#226;&#8364;&#8482;s smooth tests homogeneity samples <br />     calculation P. Journal Royal Statistical Society 85: 87&#226;&#8364;&#8220;94.                           survival data. Journal Statistical Planning Inference 139: 3559&#226;&#8364;&#8220;3569. <br /> 14. Carvalho GB, Kapahi P, Benzer S (2005) Compensatory ingestion dietary                  29. Chow GC (1960) Tests Equality Between Sets Coefficients Two Linear <br />     restriction Drosophila melanogaster. Nat Methods 2: 813&#226;&#8364;&#8220;815.                                 Regressions. Econometrica 28. <br /> 15. Dai C, Whitesell L, Rogers AB, Lindquist S (2007) Heat shock factor 1                  30. Kimura KD, Tissenbaum HA, Liu Y, Ruvkun G (1997) daf-2, insulin <br />     powerful multifaceted modifier carcinogenesis. Cell 130: 1005&#226;&#8364;&#8220;1018.                          receptor-like gene regulates longevity diapause Caenorhabditis elegans. <br /> 16. Kapahi P, Zid BM, Harper T, Koslover D, Sapin V, et al. (2004) Regulation                    Science 277: 942&#226;&#8364;&#8220;946. <br />     lifespan Drosophila modulation genes TOR signaling pathway.                 31. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans <br />     Curr Biol 14: 885&#226;&#8364;&#8220;890.                                                                          mutant lives twice long wild type. Nature 366: 461&#226;&#8364;&#8220;464. <br /> 17. Wilks MF, Fernando R, Ariyananda PL, Eddleston M, Berry DJ, et al. (2008)                   32. Lee RY, Hench J, Ruvkun G (2001) Regulation C. elegans DAF-16 its <br />     Improvement survival paraquat ingestion following introduction                    human ortholog FKHRL1 daf-2 insulin-like signaling pathway. Curr Biol <br />     new formulation Sri Lanka. PLoS Med 5: e49.                                                  11: 1950&#226;&#8364;&#8220;1957. <br /> 18. Ravi D, Wiles AM, Bhavani S, Ruan J, Leder P, et al. (2009) A network                    33. Lin K, Dorman JB, Rodan A, Kenyon C (1997) daf-16: An HNF-3/forkhead <br />                                                                                                     family member function double life-span Caenorhabditis elegans. <br />     conserved damage survival pathways revealed genomic RNAi screen. PLoS <br />                                                                                                     Science 278: 1319&#226;&#8364;&#8220;1322. <br />     Genet 5: e1000527. <br />                                                                                                 34. Ogg S, Paradis S, Gottlieb S, Patterson GI, Lee L, et al. (1997) The Fork head <br /> 19. Matheu A, Maraver A, Klatt P, Flores I, Garcia-Cao I, et al. (2007) Delayed <br />                                                                                                     transcription factor DAF-16 transduces insulin-like metabolic longevity <br />     ageing damage protection Arf/p53 pathway. Nature 448: <br />                                                                                                     signals C. elegans. Nature 389: 994&#226;&#8364;&#8220;999. <br />     375&#226;&#8364;&#8220;379. <br />                                                                                                 35. Henderson ST, Johnson TE (2001) daf-16 integrates developmental and <br /> 20. Managbanag JR, Witten TM, Bonchev D, Fox LA, Tsuchiya M, et al. (2008)                          environmental inputs mediate aging nematode Caenorhabditis elegans. <br />     Shortest-path network analysis useful approach identifying genetic                  Curr Biol 11: 1975&#226;&#8364;&#8220;1980. <br />     determinants longevity. PLoS One 3: e3802.                                               36. Li W, Boswell R, Wood WB (2000) mag-1, homolog Drosophila mago nashi, <br /> 21. Andrade MC, Gu L, Stoltz JA (2005) Novel male trait prolongs survival                        regulates hermaphrodite germ-line sex determination Caenorhabditis elegans. <br />     suicidal mating. Biol Lett 1: 276&#226;&#8364;&#8220;279.                                                          Dev Biol 218: 172&#226;&#8364;&#8220;182. <br /> 22. Gallagher IM, Jenner P, Glover V, Clow A (2000) CuZn-superoxide dismutase                   37. Lee SJ, Kenyon C (2009) Regulation longevity response temperature by <br />     transgenic mice: effect longevity, locomotor activity 3H-mazindol                 thermosensory neurons Caenorhabditis elegans. Curr Biol 19: 715&#226;&#8364;&#8220;722. <br />     3H-spiperone binding 19 months. Neurosci Lett 289: 221&#226;&#8364;&#8220;223.                            38 <br /> </body></html>