<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The essential polysome-associated RNA-binding protein <br /> RBP42 targets mRNAs involved Trypanosoma brucei <br /> energy metabolism <br />  <br />  <br /> ANISH DAS,1 RACHEL MORALES,1 MAHRUKH BANDAY,1 STACEY GARCIA,1 LI HAO,2 GEORGE A.M. CROSS,3 <br /> ANTONIO M. ESTEVEZ,4 VIVIAN BELLOFATTO1,5 <br /> 1 <br />  Department Microbiology Molecular Genetics, UMDNJ-NJ Medical School, Newark, New Jersey 07103, USA <br /> 2 <br />  Center Genome Informatics, UMDNJ-NJ Medical School, Newark, New Jersey 07103, USA <br /> 3 <br />  The Rockefeller University, New York, New York 10065, USA <br /> 4 <br />  Instituto Parasitologia y Biomedicina &#226;&#8364;&#732;&#226;&#8364;&#732;Lopez-Neyra,&#226;&#8364;&#8482;&#226;&#8364;&#8482; CSIC Avda. Del Comocimiento s/n. Armilla, 18100 Granada, Spain <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> RNA-binding proteins target mRNA coding regions emerging regulators post-transcriptional processes in <br /> eukaryotes. Here newly identified RNA-binding protein, RBP42, targets coding region mRNAs the <br /> insect form African trypanosome, Trypanosoma brucei. RBP42 essential protein associates polysome-bound <br /> mRNAs cytoplasm. A global survey RBP42-bound mRNAs performed applying HITS-CLIP technology, which <br /> captures protein&#226;&#8364;&#8220;RNA interactions vivo using UV light. Specific RBP42&#226;&#8364;&#8220;mRNA interactions, mRNA interactions <br /> with known RNA-binding protein, purified using specific antibodies. Target RNA sequences identified and <br /> quantified using high-throughput RNA sequencing. Analysis revealed RBP42 bound mainly coding region of <br /> mRNAs encode proteins involved cellular energy metabolism. Although mechanism RBP42&#226;&#8364;&#8482;s function unclear at <br /> present, speculate RBP42 plays critical role modulating T. brucei energy metabolism. <br /> Keywords: trypanosomes; mRNA translation; NTF-2 containing proteins; HITS-CLIP analysis <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                   enable parasite respond environmental <br />                                                                                changes cell little, any, differential transcription. <br /> African trypanosomes undergo complex life cycle re- <br />                                                                                   RNA-binding proteins interact target mRNAs, <br /> quiring morphological metabolic changes controlled <br />                                                                                cis-acting elements, post-transcriptional <br /> by gene expression programs. In absence of <br />                                                                                reactions lead regulated mRNA stability trans- <br /> regulated transcription, trypanosome gene expression is <br />                                                                                lation. In trypanosomes, cis-acting elements been <br /> regulated mainly post-transcriptional level, where <br />                                                                                identified mainly 39 UTRs stage-specific cell cycle&#226;&#8364;&#8220; <br /> environmental signals likely integrated mRNA <br />                                                                                specific mRNAs (Fernandez-Moya Estevez 2010; Kramer <br /> maturation, localization, translation, decay (Fernandez- <br />                                                                                Carrington 2011). Some extensively studied <br /> Moya Estevez 2010; Kramer Carrington 2011). In <br />                                                                                stage-specific mRNAs trypanosomes procyclin <br /> trypanosomes, post-transcriptional reactions likely ex- <br />                                                                                mRNAs (Acosta-Serrano et al. 2001). GPEET, EP1, EP2 <br /> tensive intricately networked account large <br />                                                                                procyclins major surface glycoproteins encoded by <br /> number transcripts proteins fluctuate in <br />                                                                                procyclic mRNAs; cover Trypanosoma brucei <br /> abundance different stages parasite life-cycle <br />                                                                                surface parasite resides midgut tsetse <br /> (for review, Siegel et al. 2011). It unclear post- <br />                                                                                vector. The 39 UTRs EP1 GPEET contain three <br /> transcriptional reactions drive differential gene expression <br />                                                                                stem&#226;&#8364;&#8220;loop structures encoding cis-acting elements <br />                                                                                regulate mRNA translation stability. Specifically, <br />                                                                                EP1 translation increased RNA-binding protein <br />    5 <br />     Corresponding author                                                       TbZPF3 binds cis-acting elements 39 <br />    E-mail bellofat@umdnj.edu <br />    Article published online ahead print. Article publication date   UTR EP1 (Walrad et al. 2009). Similarly, GFP reporter <br /> at http://www.rnajournal.org/cgi/doi/10.1261/rna.033829.112.                   gene translation increased RNA-binding proteins <br />  <br /> 1968                                 RNA (2012), 18:1968&#226;&#8364;&#8220;1983. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2012 RNA Society. <br />                                                                                  Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br /> ALB3 ALB4 bind cis-acting 39 UTR element           parasites (Das et al. 2006). Protein sequence analysis revealed <br /> chimeric mRNA containing GFP open reading frame              RBP42 harbors regions similar known protein <br /> (ORF) flanked GPEET 39 UTR (Mani et al. 2011).            domains. The N-terminal 110 amino acids display homology <br /> Intriguingly, cases effect RNA-binding        NTF2-like protein domain (PROSITE domain <br /> protein&#226;&#8364;&#8220;39 UTR interaction modulated coding region         PS50177), C-terminal 90 amino acids display ho- <br /> sequences target mRNAs. Additional RNA-               mology RNA recognition motif (RRM) protein <br /> binding proteins probably contribute translational effi-      domain (PROSITE domain PS50102) (Fig. 1A,B; Bandziulis <br /> ciency procyclin. Furthermore, extremely little known      et al. 1989). NTF2-like domains proteins with <br /> about translational regulation mRNAs encode          diverse activities usually involved protein&#226;&#8364;&#8220;protein <br /> nonmembrane proteins (Kramer et al. 2010).                       interactions. RRM domains common RNA- <br />    In addition playing role translational regulation,    binding domains eukaryotic RNA-binding proteins. <br /> RNA-binding proteins contribute mRNA stability                A search proteins similar domain architecture <br /> trypanosomes. The best-characterized RNA-binding pro-            revealed RBP42 closely resembles mammalian <br /> tein involved mRNA stability regulation T. brucei          G3BP proteins fly homolog, Rasputin (Parker <br /> DRBD3, called PTB1, stabilizes cohort            et al. 1996; Irvine et al. 2004). G3BP interacts RAS <br /> mRNAs binding 39 UTRs (Estevez 2008;             GTPase-activating protein p120, protein involved the <br /> Stern et al. 2009). In RNA decay experiments, depletion       mammalian RAS signaling pathway. G3BP essential for <br /> DRBD3/PTB1 resulted reduced half-life subset        mouse development, expression increased in <br /> this RNA cohort. These alterations mRNA stability         human cancer cell types (Barnes et al. 2002; Kim et al. <br /> explain DRBD3/PTB1 essential parasite viability.      2007). Interestingly, G3BP reported multiple <br /> Interestingly, study indicated DRBD3-       activities related cellular RNA metabolism, including <br /> modulated mRNAs encode cellular membrane proteins                RNA helicase activity, phosphorylation-dependent endo- <br /> (Estevez 2008). This finding consistent growing        ribonuclease activity, cytoplasmic stress granule facil- <br /> literature verifying post-transcriptional coregulation re-    itator activity (Atlas et al. 2007; Solomon et al. 2007; Ortega <br /> lated genes yeasts, Drosophila, mammalian cells           et al. 2010). <br /> (Keene 2007).                                                       In spite similar domain architecture between <br />    Here, investigated uncharacterized RNA-binding          RBP42 G3BP, noticeable differences exist. First, only <br /> protein, named RBP42 (Gene identification Tb927.6.4440),         25% primary amino acid sequence conserved <br /> <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> determine role trypanosome RNA metabolism.             T. brucei RBP42 mammalian G3BPs. Second, <br /> (RBP means RNA-binding protein, 42 clarifies            RNA-binding domain, RRM, contains conserved <br /> RBP42 42nd identified RNA-binding protein              helical regions G3BPs conserved helix in <br /> trypanosomatids.) The RBP42 protein sequence suggests            RBP42. Finally, arginine-glycine-rich (RGG) domain, <br /> that RBP42 distant homolog metazoan RAS-             present C terminus G3BPs, absent RBP42. <br /> GTPase activating protein SH3-binding protein, G3BP              The PxxP motifs, modulate protein&#226;&#8364;&#8220;protein <br /> (Parker et al. 1996; Irvine et al. 2004). G3BP broadly      interactions, vary number different mammalian <br /> functioning mRNA-binding protein known integrate              G3BP family members, different try- <br /> extracellular signals intracellular metabolic responses.    panosomatid RBP42 orthologs. <br /> The intriguing possibility RBP42 plays important <br /> role trypanosome mRNA metabolism inspired to <br />                                                                  RBP42 essential normal cell growth <br /> characterize RNA targets. To accomplish this, utilized <br /> a highly powerful technique, named HITS-CLIP,              To determine RBP42 essential parasite <br /> traps protein&#226;&#8364;&#8220;RNA interactions vivo UV crosslinking        viability, used conditional double-stranded RNAi-me- <br /> (Ule et al. 2003; Licatalosi et al. 2008). A global picture   diated knockdown (RNAi) RBP42 procyclic cell line <br /> RBP42&#226;&#8364;&#8220;RNA interactions demonstrates RBP42 binds             RBP42 allele; second allele inser- <br /> within mRNA coding sequences. These interactions occur           tionally inactivated puromycin-resistant cassette (Fig. <br /> within cohort mRNAs function cellular energy        2A,B). (A cell line lacking RBP42 alleles be <br /> metabolism.                                                      established despite trials.) Three clonal cell lines were <br />                                                                  selected, exhibited slow growth <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br />                                                                  altered morphology 2 d RNAi induction. These <br /> RESULTS <br />                                                                  data indicate RBP42 essential protein procyclic <br />                                                                  T. brucei <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>survival</span>. Immunoblot analysis typical clone, <br /> RBP42 similar mammalian RNA-binding <br />                                                                  called AM1, showed expected reductions RBP42 <br /> protein, G3BP <br />                                                                  protein (Fig. 2C). Depletion RBP42 caused AM1 cells to <br /> RBP42 originally identified copurifying protein         stop dividing, reflected growth curve (Fig. 2D). <br /> with RNA polymerase II subunit RPB4 T. brucei procyclic       Microscopic observation RBP42-depleted AM1 cells on <br />  <br />                                                                                                    www.rnajournal.org       1969 <br />  Das et al. <br />  <br />  <br />  <br />                                                                                                day 4 revealed noticeable phenotypic <br />                                                                                                alterations, cells exhibiting ab- <br />                                                                                                normal shape (Ploubidou et al. 1999; <br />                                                                                                Hammarton 2007). Cells exhibited ap- <br />                                                                                                parent cytokinesis defects, inferred <br />                                                                                                large number nuclei in <br />                                                                                                parasites (Fig. 2E). DAPI-stained <br />                                                                                                nuclei kinetoplast photographed <br />                                                                                                counted cell basis; z30% <br />                                                                                                RBP42 depleted cells aber- <br />                                                                                                rantly shaped, containing more <br />                                                                                                nuclei (including 1K2N, 2K3N, 2K2N, <br />                                                                                                0K4N). Approximately 3% the <br />                                                                                                cells zoids, containing nucleus <br />                                                                                                (1K0N). In comparison, noninduced cells, <br />                                                                                                intact RBP42, contained &lt;1% of <br />                                                                                                aberrant cells multiple nuclei and <br />                                                                                                &lt;1% zoids. Interestingly, recent RNAi- <br />                                                                                                based screen designed identify essential <br />                                                                                                T. brucei genes (Alsford et al. 2011) in- <br />                                                                                                dicated RNAi knockdown RPB42 <br />                                                                                                resulted defect cellular fitness of <br />                                                                                                bloodstream procyclic form para- <br />                                                                                                sites, defect life cycle dif- <br />                                                                                                ferentiation. Taken together, data <br />                                                                                                suggest decreased levels RBP42 <br />                                                                                                negatively impact parasite morphology <br />                                                                                                growth multiple stages their <br />                                                                                                life cycle. <br />  <br />                                                                                                RBP42 cytoplasmic protein <br />                                                                                                associates polysomes <br />                                                                                                To assess cellular distribution of <br />                                                                                                RBP42, fractionated cellular extracts <br />                                                                                                differential centrifugation assayed <br />                                                                                                fraction immunoblot analysis <br />                                                                                                using specific antibodies. Known nuclear <br />                                                                                                cytoplasmic proteins fractionated as <br />                                                                                                expected; RPB1, largest subunit of <br />                                                                                                RNA polymerase II, served nuclear <br />                                                                                                marker Golgi Reassembly Stacking <br /> FIGURE 1. Alignment T. brucei RBP42 human G3BP1. (A) Multiple sequence                  Protein (GRASP) ER chaperone <br /> alignments trypanosomatid RBP42s metazoa G3BP1s. Gene annotations           BiP served cytoplasmic markers. Our <br /> T. brucei (Tb927.6.4440), Leishmania major (LmjF.30.3090), Homo sapiens (G3BP1,Q13283,         fractionation analysis demonstrated that <br /> Gene identification 10146), Mus musculus (P97855). The NTF-2 domain overlined by <br /> a gray bar; RNA recognition motif (RRM) domain underlined white bar;       RBP42 predominantly present par- <br /> three PxxP motifs T. brucei overlined black line. The single PxxP motif   asite cytoplasm (Fig. 3A,B). Further sep- <br /> human mouse genes underlined. Numbering ends lines identifies amino acid      aration, high-speed centrifugation, of <br /> location organism. (B) Schematic comparison T. brucei RBP42 human              cytoplasmic extracts soluble frac- <br /> G3BP1. RBP42 79% size G3BP1 (drawing approximately scale). The NTF-2 <br /> domain (light gray), weakly acidic acidic regions (dashed-stripe striped), triple   tion particulate fraction, pre- <br /> single PxxP motifs (PxxP), RRM domain (white), RRG-rich region (black) shown.          dominantly consists ribosomes and <br /> RBP42 G3BP1 similar identical; therefore, trypanosome protein           polysomes, resulted RBP42. Immu- <br /> named accordance published trypanosomatid RNA-binding protein designation <br /> scheme (De Gaudenzi et al. 2005). Within RRM, RNP1 signature sequence motif <br />                                                                                                nofluorescence microscopy fixed par- <br /> (K/R) G (F/Y)(G/A)FVX(F/Y) present T. brucei RBP42 KGYVFFDF (amino acids              asites indicated RBP42 localization <br /> 311&#226;&#8364;&#8220;319).                                                                                      consistent observation <br />  <br /> 1970         RNA, Vol. 18, No. 11 <br />                                                                                                Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br />  <br /> FIGURE 2. Effect RBP42 depletion cell viability. (A) RNAi-dependent ablation RBP42 mRNA T. brucei strain AM1. This strain <br /> contains single intact RBP42 allele, RNAi construct. (B) Analysis genomic DNA assess AM1 genotype. PCR reactions <br /> utilized primers-A, -B, -C, -D located shown panel A. The 0.6-kb DNA lane 4 0.9-kb DNA lane 6 confirm AM1 <br /> contains puromycin cassette place endogenous RBP42 alleles. The 1.3-kb band lane 2 shows remaining RBP42 <br /> allele intact. (Lanes 1,3,5) Controls showing presence intact RBP42 alleles parental clone 3 cell line. (C) RBP42 decreased after <br /> ablation mRNA tetracycline-dependent RNAi induction, demonstrated immunoblot analysis. a-Tubulin levels shown loading <br /> control. (D) T. brucei requires RBP42 cell viability. Tetracycline, activates RNAi-dependent ablation RBP42, added day 0. <br /> Parasites directly observed counted daily using hemocytometer. Three clones studied AM1 chosen representative <br /> example observed phenotype. (E) Cellular morphology cytokinesis altered result RBP42 depletion. Examples the <br /> predominant phenotypes shown fluorescent micrographs containing DAPI-stained nuclei kinetoplasts (both blue) parasites that <br /> represent phenotype observed day 4 RBP42 depletion. Quantification DAPI-stained nuclei cell shown n = 445 cells the <br /> histogram right micrographs. Open bars, normal phenotypic distribution. <br />  <br />  <br />                                                                                                                   www.rnajournal.org          1971 <br />  Das et al. <br />  <br />  <br />  <br />  <br /> FIGURE 3. RBP42 S100-precipitated fraction cellular cytoplasm. (A) Subcellular fractionation scheme shown. (B) <br /> Immunoblotting fractionated lysates identifies RBP42 cytoplasmic associated ribosome-containing subcellular fraction (S100 <br /> pellet). Protein detection performed using antibodies listed left panel. RPB1, largest subunit RNA polymerase II, <br /> is nuclear marker; GRASP BiP cytoplasmic markers fractionate S100 supernatant early detergent step the <br /> fractionation (Ho et al. 2006). (C) Immunofluorescent detection fixed parasites shows cytoplasmic localization RBP42. Four 15 Z-stacks <br /> are shown. Z-stacks collected refine resolution. <br />  <br />  <br />  <br /> above; immunostained RBP42 visible                         80S ribosome monomers enriched fractions 3&#226;&#8364;&#8220;5 <br /> cytoplasm dotted granular appearance (Fig. 3C).                     (Fig. 4A). Immunoblot assays polysomal fractions revealed <br /> RBP42 observed nucleus. Some                        significant portion RBP42 associated with <br /> dotted stains appeared outline nuclear periphery.                      polysomes. Poly(A)-binding protein (PABP), binds <br /> However, RBP42 did colocalize nuclear pore                       mRNA polysomes, serves control presence <br /> complex protein, NUP158.                                                      mRNA polysomal fractions. To test whether <br />    To determine RBP42 associated poly-                        RBP42&#226;&#8364;&#8482;s interaction polysomes mRNA-dependent, <br /> somes, fractionated polysomes monosomes                            purified polysomes treated limiting amounts of <br /> sucrose-density gradient centrifugation cytoplasmic ex-                    RNase prior fractionation. RNase treatment clipped the <br /> tracts tested presence RBP42 polysomal                      mRNAs polysomes released RBP42 and <br /> fractions. Distribution rRNA large ribosomal                       PABP lower-molecular-weight complexes (Fig. 4B). <br /> protein L5 showed polysomes enriched                         When purified polysomes fractionated prior <br /> fractions 7&#226;&#8364;&#8220;10 40S 60S ribosomal subunits,                    RNase treatment, RBP42 PABP remained tightly <br />  <br /> 1972         RNA, Vol. 18, No. 11 <br />                                                                                             Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br />                                                                                                  copurified poly(A)+ RNA, indicat- <br />                                                                                                  ing direct interaction cellular <br />                                                                                                  mRNA vivo (Fig. 4C). PABP, which <br />                                                                                                  directly binds mRNA, served a <br />                                                                                                  control copurified poly(A)+ <br />                                                                                                  fraction (Fig. 4C). In control experi- <br />                                                                                                  ment, similar purification poly(A)+ <br />                                                                                                  mRNA noncrosslinked parasites <br />                                                                                                  did copurify RBP42 PABP. <br />                                                                                                  Taken together, data indicate that <br />                                                                                                  RBP42 directly associates translat- <br />                                                                                                  ing mRNAs parasite. <br />                                                                                                     To assess RBP42 affects mRNA <br />                                                                                                  translation T. brucei, measured <br />                                                                                                  overall translation rate normal or <br />                                                                                                  RBP42-depleted cells. Protein synthesis <br />                                                                                                  rates measured incorporation <br />                                                                                                  35S-methionine/cysteine cells <br />                                                                                                  (Supplemental Fig. 1). RBP42-depleted <br />                                                                                                  cells showed 34% diminished rate of <br />                                                                                                  protein synthesis compared normal <br />                                                                                                  cells. (Protein synthesis rate linear <br />                                                                                                  duration assay 35S- <br />                                                                                                  incorporation similar dif- <br />                                                                                                  ferent cell densities.) Although reduced <br />                                                                                                  protein synthesis reflect slow- <br />                                                                                                  growth rate RBP42-depleted cells, <br />                                                                                                  direct effect RBP42 on <br />                                                                                                  mRNA translation (Barquilla et al. 2008; <br />                                                                                                  Freire et al. 2011). <br />  <br />                                                                                                  Crosslinking immunoprecipitation <br /> FIGURE 4. RBP42 fractionates predominantly polysomes. (A) Polysome fractionation            RBP42 RNA targets <br /> profile shown. Immunoblotting fractionated material accomplished using RBP42, <br /> L5, PABP antibodies listed left panel. L5, component 60S      Having established RBP42 directly <br /> ribosomal subunit, monosome (80S) polysome marker; PABP, cytoplasmic poly(A)-      binds mRNA, sought identify <br /> binding protein, mRNA marker. Locations small subunit (40S), large subunit       cellular substrates using crosslinking <br /> (60S), intact monosomes (80S), polysomes gradients accomplished using       immunoprecipitation (CLIP) method- <br /> ethidium bromide (ETBR)&#226;&#8364;&#8220;stained Agarose gel. T. brucei 28S naturally present large <br /> fragments, migrate 18S rRNA. (B) Purified polysomes obtained fractions       ology. We reasoned identification <br /> 7&#226;&#8364;&#8220;10 A fractionated mock (solid line) limited RNase A treatment      RBP42&#226;&#8364;&#8482;s mRNA substrates help <br /> (dotted line). Immunoblotting shown A. (C) Immunoblotting fractionated      reveal function trypanosome gene <br /> lysates vivo protein&#226;&#8364;&#8220;RNA crosslinking UV light. An equal each <br />                                                                                               expression. To set CLIP, estab- <br /> fraction present lanes relative abundance RBP42 PABP each <br /> fraction. Fractions, defined lane, obtained oligo-d(T) celluloselished optimum conditions specific <br /> chromatography.                                                                               efficient immunoprecipitation of <br />                                                                                               RBP42 protein (Fig. 5). Anti-RBP42 an- <br />                                                                                               tibody specifically efficiently removes <br /> associated polysomes control experiments. The                     RBP42 cellular extracts, indicated immunoblot <br /> release RBP42 polysomes limiting RNase treat-                  analysis (Fig. 5A, cf. lanes 4&#226;&#8364;&#8220;6 7&#226;&#8364;&#8220;9). In addition, we <br /> ment indicated RBP42 association polysomes                   tested control antibody recognizes known RNA- <br /> mediated protein&#226;&#8364;&#8220;mRNA interactions.                                    binding protein, DRBD3 (Estevez 2008). The anti-DRBD3 <br />    To confirm RBP42 interacts mRNA, parasite                    antibody equally specific efficient removing <br /> proteins crosslinked substrate RNAs vivo using                 DRBD3 cellular extract (Fig. 5A, cf. lanes 16&#226;&#8364;&#8220;18 and <br /> UV light. Purified parasite protein crosslinked poly(A)+               22&#226;&#8364;&#8220;24). The specificity anti-RBP42 anti-DRBD3 <br /> RNA isolated using oligo-d(T) cellulose chro-                   antibodies verified immunoblot analyses of <br /> matography. As shown immunoblot analysis, RBP42                        immunocaptured proteins: Anti-RBP42 antibody only <br />  <br />                                                                                                              www.rnajournal.org     1973 <br />  Das et al. <br />  <br />  <br />  <br />                                                                                              immunoprecipitate protein&#226;&#8364;&#8220; <br />                                                                                              RNA complexes after <br />                                                                                              crosslinking (lanes 1,2). Protein-bound <br />                                                                                              RNAs subsequently extracted, re- <br />                                                                                              verse transcribed cDNAs, sub- <br />                                                                                              jected high-throughput DNA sequenc- <br />                                                                                              ing (HITS-CLIP). <br />  <br />                                                                                              RBP42 predominantly binds mRNAs <br />                                                                                              coding regions <br />                                                                                                  HITS-CLIP RBP42 DRBD3 cDNA <br />                                                                                                  libraries performed Illumina <br />                                                                                                  platform. A total 41 M 38 M reads <br />                                                                                                  obtained RBP42 DRBD3, <br />                                                                                                  respectively. Unique reads mapped <br />                                                                                                  T. brucei genome RBP42 and <br />                                                                                                  DRBD3 (1.8 M 1.1 M, respectively). <br />                                                                                                  The distribution mapped reads dif- <br />                                                                                                  ferent chromosomal features revealed <br />                                                                                                  surprisingly large number (&gt;94%) <br />                                                                                                  RBP42 reads mapped within <br />                                                                                                  mRNA coding sequences. In contrast, <br />                                                                                                  z30% DRBD3 reads mapped <br />                                                                                                  mRNA coding regions (Fig. 6A). <br />                                                                                                  Neither RBP42 DRBD3 targeted non- <br />                                                                                                  coding RNAs (including rRNAs, tRNAs, <br />                                                                                                  snRNAs). The distribution RBP42 <br />                                                                                                  reads mRNA sequences indicates <br />                                                                                                  RBP42 predominantly binds within <br />                                                                                                  mRNA coding regions trypanosomes. <br /> FIGURE 5. RBP42 antibody recognizes RBP42 cell extracts. (A) Cell extracts depleted         Visualization RBP42 DRBD3 <br /> using preimmune serum RBP42 DRBD3 antibodies. Immunoblots probed <br /> with indicated antibody (anti-RBP42 used lanes 1&#226;&#8364;&#8220;12, anti-DRBD3 used       reads  IGV browser showed four <br /> lanes 13&#226;&#8364;&#8220;24). None indicates no-antibody-treated samples. (Lanes 1,4,7,10,13,16,19,22) Undiluted types profiles (Fig. 6B). Profile 1, an <br /> samples; (lanes 2,5,8,11,14,17,20,23) 1:5 diluted samples; (lanes 3,6,9,12,15,18,21,24) 1:25 example frequently observed <br /> diluted samples starting immunodepleted extracts. (B) Proteins captured <br />                                                                                                  profile, shows RBP42 binding the <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> crosslinked (+) crosslinked ( ) parasites using antibodies indicated the <br /> lanes. Immunocaptured proteins detected anti-RBP42 (lanes 1&#226;&#8364;&#8220;8) anti-DRBD3 coding region gene DRBD3 <br /> (lanes 9&#226;&#8364;&#8220;16) antibodies. None indicates no-antibody-treated samples. (C) Immunoprecipitated binding. Profile 2, frequently <br /> protein&#226;&#8364;&#8220;RNA complexes 59-end radiolabeled RNA, separated denaturating- profile, shows RBP42 bound <br /> polyacrylamide gel electrophoresis (SDS-PAGE) transferred nitrocellulose membranes. <br /> A PhosphorImager scan protein&#226;&#8364;&#8220; 32P-radiolabeled RNA complexes shown. RBP42 and <br />                                                                                                  coding region DRBD3 <br /> DRBD3 migrate similarly proteins similar molecular mass, RNA bound 39 UTR gene. <br /> fragments associated proteins 30&#226;&#8364;&#8220;50 nt long.                                        Profile 3 shows DRBD3 bound the <br />                                                                                                  39 UTR gene RBP42 binding. <br />                                                                                                  Profile 4 shows example DRBD3 <br /> captured RBP42, DRBD3 detected (Fig. 5B, cf.                      bound coding region gene. <br /> lanes 5,6 13,14). Similarly, anti-DRBD3 antibody                        To verify nonspecific associations did cause the <br /> only captured DRBD3, RBP42 detected (Fig.                         binding RBP42 mRNA coding sequences the <br /> 5B, cf. lanes 15,16 7,8).                                                CLIP procedures, determined number of <br />    RNA targets RBP42 DRBD3 purified                            RBP42 reads correlated mRNA <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> mRNA <br /> immunoprecipitating protein&#226;&#8364;&#8220;RNA complexes using anti-                        cellular abundance. The comparison RBP42 read num- <br /> RBP42 anti-DRBD3 antibodies. RNAs detected                           ber mRNA transcript lengths indicated correlation <br /> after radiolabeling 59 ends (Fig. 5C). RNA ob-                     RBP42 binding mRNA length (R2 = 0.07) <br /> tained protein&#226;&#8364;&#8220;RNA complexes crosslinking (lanes                    (Supplemental Fig. 2). Similarly, comparison RBP42 <br /> 4,6). No RNAs observed absence crosslinking                   read number mRNA levels, obtained tran- <br /> (lanes 3,5). In control experiments, preimmune serum did                     scriptome study procyclic parasites (Siegel et al. 2010), <br />  <br /> 1974         RNA, Vol. 18, No. 11 <br />                                                                                                Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br />  <br /> FIGURE 6. HITS-CLIP data, RNA read assignments, quantification. (A) Illumina CLIP sequencing data RNA tag count data. Pie charts <br /> depict relative abundance RNA targets, derived RNA tag count data. Coding sequences defined annotated <br /> open reading frames mRNAs version 4 T. brucei brucei 927 genome TriTrypDB GeneDB (tritrypdb.org/tritrypdb and <br /> www.genedb.org). Intercoding sequences contain 59 39 UTRs, sequences contain primary transcripts. rRNA/tRNA, anti- <br /> sense, RNAs defined database referenced above. Reads poor quality sequence ambiguity, reads generated <br /> by PCR amplification, eliminated prior mapping. (B) RBP42&#226;&#8364;&#8220;mRNA interactions DRBD3&#226;&#8364;&#8220;mRNA interactions (control protein&#226;&#8364;&#8220;RNA <br /> interactions) specific genes T. brucei genome. (Profile 1) RBP42-bound sequences coding region succinyl-CoA <br /> synthetase (a-subunit shown here; Tb927.3.2230). Two upstream genes (Tb927.3.2250 Tb927.2240) downstream gene (Tb927.2220) <br /> that bound RBP42 reveal binding specificity, does absence DRBD3 binding genes shown. (Profile 2) RBP42-bound <br /> sequences CDS DRBD3-bound sequences 39 UTR Gim5B (Tb09.211.2740). The absence reciprocal binding, as <br /> well absence binding flanking genes, indicates binding specificities. (Profile 3) DRBD3-bound sequences (predicted) <br /> 39 UTR region MFS mRNA (major facilitator superfamily protein; Tb927.8.7740). DRBD3-targeting MFS mRNA 39 UTR initially <br /> discovered Estevez (2008). (Binding specificity revealed low binding DRBD3 RBP42 CDS gene, low <br /> binding RBP42 39 UTR gene. The small signal indicating DRBD3 interactions 39 UTR upstream gene, <br /> Tb927.8.7730, investigated.) (Profile 4) DRBD3-bound sequences coding region hypothetical protein mRNA (Tb927.1.2140). <br /> Binding specificity revealed absence RBP42 binding gene, absence DRBD3 RBP42 binding upstream <br /> flanking genes (Tb927.1.2120 Tb927.1.2130) downstream flanking gene (Tb927.1.2150). Scales, 0&#226;&#8364;&#8220;1500 (Profile 1), 0&#226;&#8364;&#8220;300 (Profile 2), 0&#226;&#8364;&#8220;100 <br /> (Profile 3), 0&#226;&#8364;&#8220;600 (Profile 4). Green denotes RBP42-bound sequences, orange denotes DRBD3-bound mRNA sequences. <br />  <br />  <br /> indicated RBP42 read number independent                        RBP42 binding coding regions verified by <br /> mRNA abundance. Whereas large number reads                        analyzing HITS-CLIP data using 835-gene set which <br /> obtained low abundant mRNAs, reads                        39 UTRs determined high confidence <br /> were detected highly abundant mRNAs (R2 =                       (Siegel et al. 2010, 2011). The total unique reads RBP42 <br /> 0.4) (Supplemental Fig. 2).                                                 253,315; 237,261 (z94%) CDS <br />  <br />                                                                                                                   www.rnajournal.org         1975 <br />  Das et al. <br />  <br />  <br />  <br /> regions 16,054 (z6%) 39 UTRs. Total                     reliable RBP42 targets, compared F-seq <br /> unique reads DRBD3 87,811, 38,603                     analysis RBP42 read profiles IGV browser. <br /> (z43%) CDS regions 49,208 (z57%)                   On basis inspection, considered F-Seq <br /> within 39 UTRs. The number RBP42 DRBD3 reads                     values $25 represent reliable binding, with <br /> within CDS 39 UTR plotted pairwise com-                      easily detectable peak binding regions. These criteria re- <br /> parisons scatterplot matrix (Fig. 7). These analyses               sulted identification 188 mRNAs F-Seq <br /> confirmed RBP42 strongly prefers bind                    enrichment values $25 corresponding z3% <br /> mRNA coding sequences.                                                  RBP42 targets. These 188 genes selected further <br />                                                                         analysis. <br />                                                                            Among 188 gene targets, 137 (73%) genes a <br /> Genes involved cellular energy metabolism <br />                                                                         known function, annotated TriTrypDB; http://tritrypdb. <br /> are robust RBP42 targets <br />                                                                         org/tritrypdb/. The rest unknown trypanosome-specific <br /> To identify mRNA substrates RBP42, analyzed                   genes annotated hypothetical. GO terminology used <br /> our HITS-CLIP data using F-Seq algorithm, uses                categorize 137 genes according biological <br /> kernel density measurements define peak regions                   process, cellular component, molecular function (Fig. 8; <br /> binding (Boyle et al. 2008). F-Seq analysis detected peaks           Supplemental Fig. 3). Significantly enriched GO terms (P &lt; <br /> RBP42 reads 2000 genes peak-enrichment                    0.01) include broad categories, example, carbohy- <br /> values ranging 1.7&#226;&#8364;&#8220;169 background. To identify                drate metabolic process (7 3 10 3), cellular metabolic <br />                                                                                             process (2 3 10 5), primary metabolic <br />                                                                                             process (1 3 10 3), cytoplasmic (3 3 <br />                                                                                             10 8), intracellular membrane&#226;&#8364;&#8220; <br />                                                                                             bounded organelle (4 3 10 7). Interest- <br />                                                                                             ingly, 137-gene set overrepresented <br />                                                                                             genes involved glucose proline <br />                                                                                             metabolism. GO terms describing car- <br />                                                                                             boxylic acid metabolic process (1 3 <br />                                                                                             10 6), cellular amino acid metabolic <br />                                                                                             process (4 3 10 6), cellular respiration <br />                                                                                             (1 3 10 5), glucose metabolic process <br />                                                                                             (3 3 10 3), glycosome (8 3 10 6), <br />                                                                                             mitochondrion (3 3 10 5), catalytic <br />                                                                                             activity (8 3 10 3), oxidoreductase <br />                                                                                             activity (5 3 10 5)&#226;&#8364;&#8221;processes integral <br />                                                                                             glucose proline metabolism&#226;&#8364;&#8221;are <br />                                                                                             significantly overrepresented. This over- <br />                                                                                             representation genes involved cel- <br />                                                                                             lular energy generation pathway indicates <br />                                                                                             RBP42 play key role the <br />                                                                                             regulation mRNAs de- <br />                                                                                             termine cellular metabolic responsiveness. <br />                                                                                                The main energy sources procyclic <br />                                                                                             parasites D-glucose L-proline. <br />                                                                                             These nutrients metabolized a <br />                                                                                             combination glycolytic enzymes the <br />                                                                                             glycosomes, TCA cycle enzymes, the <br />                                                                                             respiratory chain mitochondrion, <br />                                                                                             intermediate steps cyto- <br />                                                                                             plasm (Besteiro et al. 2005). Recent RNAi <br /> FIGURE 7. A comparison RBP42 DRBD3 unique reads CDS 39 UTRs gene <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>knockout</span> studies con- <br /> 835 genes defined 39 UTRs. RBP42 DRBD3 unique reads CDS 39 UTR <br /> of 835 genes normalized gene length read counts obtain RPKM (reads tributed current model procyclic <br /> kilobase million mapped reads) values. Gene length gene determined energy metabolic pathways (Bringaud <br /> adding CDS length 39 UTR length. The highest RPKM value obtained RBP42 CDS et al. 2006; Zikova et al. 2009). Mapping <br /> regions 7.4; RBP42 39 UTR region, 1.4; DRBD3 CDS, 4.6, DRBD3 39 UTR, <br />                                                                                             RBP42 targets procyclic- <br /> 9.9. Pairwise comparisons RPKM values protein CDs 39 UTR regions are <br /> shown scatterplot matrix. The axes RPKM values reads mapped specific metabolic pathway shown in <br /> CDs 39 UTR RBP42 DRBD3 experiments.                                          Figure 9. <br />  <br /> 1976         RNA, Vol. 18, No. 11 <br />                                                                                              Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br />  <br /> FIGURE 8. Gene Ontology (GO) assignments RBP42-bound mRNAs. Highly significant (P &lt; 0.01) associations RBP42 targets GO <br /> terms shown. GO terms derived GO Slim set designed incorporate trypanosome-specific processes cellular compartments <br /> (Alsford et al. 2011). GO terms broad categories indicated light blue bars; GO terms specific categories are <br /> indicated dark blue bars. Statistical analysis GO term associations carried using 137 annotated genes. Since annotated genes lack a <br /> comprehensive set appropriate GO terms, significant association verified. <br />  <br />  <br />    In glycosome, trypanosome-specific                       RBP42 target. In case multi-subunit TCA-cycle <br /> peroxisome-like organelle, 14 enzymes involved                   enzymes, enzyme subunits RBP42 targets <br /> in glucose metabolism (enzymes 1&#226;&#8364;&#8220;9, 13&#226;&#8364;&#8220;18) (Parsons 2004;                  (including enzymes 20, 23, 27). Among enzymes <br /> Besteiro et al. 2005; Coustou et al. 2008). Seven                     metabolize proline 2-ketoglutarate (enzymes 31&#226;&#8364;&#8220;34), <br /> among 3% RBP42 targets (enzymes 3&#226;&#8364;&#8220;6, 13&#226;&#8364;&#8220;15).                    5% RBP42 targets. Finally, <br /> Four additional enzymes 10%                          multiple respiratory chain components top <br /> RBP42 targets (enzymes 8, 9, 16, 17). In addition,                     5% RBP42 targets well. These included the <br /> glycosomal membrane proteins, PEX11 GIM5,                        cytochrome oxidase subunits complex IV, of <br /> maintain glycosomal integrity, RBP42 targets. Among                   F-type ATPase subunits (Zikova et al. 2009), cytochrome <br /> the cytosolic glycolytic enzymes involved pyruvate                 C, 20-kDa subunit complex I, Rieske iron&#226;&#8364;&#8220;sulfur <br /> production (enzymes 9&#226;&#8364;&#8220;12, 19), enzymes                    protein complex III. Taken together, findings show <br /> the 5% RBP42 targets (enzymes 9, 11, 12, 19). In                RBP42 targets large portion cellular mRNAs that <br /> mitochondrion, pyruvate metabolized acetyl CoA,                      encode energy generating enzymes function maintain <br /> which enters main pathways. Energy generated                 cell viability. <br /> acetate production enzymes (enzymes 20&#226;&#8364;&#8220;23, 39, 40) <br /> directly indirectly involved substrate-level ATP pro- <br />                                                                            DISCUSSION <br /> duction. Five 3% RBP42 targets. The <br /> sixth enzyme, enzyme 22, unidentified trypanosomes                   RNA-binding proteins play pivotal role trypanosome <br /> (Riviere et al. 2004). The TCA cycle contains enzymes                gene expression. Although important trypanosome <br /> (enzymes 23&#226;&#8364;&#8220;30), enzymes                  RNA-binding proteins studied date, their <br /> 5% RBP42 targets. Of additional enzymes                     comprehensive set target mRNAs remain unknown. In <br /> within 5%, enzyme (enzyme 26) enriched                     report characterized new RNA-binding protein, <br /> above background, (enzyme 29)                        RBP42, identified cellular targets using global <br />  <br />                                                                                                                www.rnajournal.org         1977 <br />  Das et al. <br />  <br />  <br />  <br /> unbiased approach. We RBP42 binds               How does RBP42 binding target mRNAs contrib- <br /> coding region mRNAs involved cellular energy        ute metabolic adaptation trypanosomes? Our findings <br /> metabolism. RBP42 essential procyclic (Fig. 2)      translational rates decreased cells de- <br /> bloodstream-form viability (Alsford et al. 2011). We pro-      pleted RBP42 suggestive RBP42 functioning a <br /> pose newly characterized protein plays important     translational regulator. It possible RBP42 promotes <br /> roles mRNA metabolism parasite&#226;&#8364;&#8482;s life cycle.     translation target mRNAs helping recruit ini- <br />    The identification RBP42 targets revealed RBP42     tiation elongation factors polysomes facilitate protein <br /> binds large number mRNAs encode enzymes           production. Alternatively, RBP42 stall ribosomes, slow- <br /> involved energy metabolism. Modulation cellular          ing abrogating translation enzymes utilized in <br /> energy metabolism response changing nutritional          energy metabolism cell. Expression amyloid pre- <br /> sources requires adjustments metabolic pathways. In         cursor protein (APP) human neuronal cells relies a <br /> procyclic parasites, glucose amino acids          combination positive negative translational <br /> utilized primary nutritional sources. When glucose   regulation (Lee et al. 2010). The hnRNP C protein binds to <br /> major <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> source, glycolytic enzymes <br /> in glycosome mitochondrion are <br /> involved energy production. When <br /> amino acids main nutritional <br /> sources, additional mitochondrial en- <br /> zymes utilized. As trypanosomes lack <br /> differential transcription, post-transcrip- <br /> tional reactions play defining role <br /> in adjustment metabolic <br /> pathways. We predict RBP42 helps <br /> regulate cellular response nutri- <br /> tional sources binding mRNAs that <br /> encode metabolic enzymes. <br />    It interesting RBP42 appears <br /> to similar mammalian protein <br /> G3BP (Fig. 1). Both RBP42 G3BP <br /> are cytoplasmic, associate polysomes <br /> (Angenstein et al. 2002; Atlas et al. 2007), <br /> and bind mRNAs (Atlas et al. 2004; <br /> Taniuchi et al. 2011). Whereas G3BP&#226;&#8364;&#8220; <br /> mRNA interactions limited to <br /> studies c-Myc, Tau, b-F1-ATPase <br /> mRNAs, identified RBP42&#226;&#8364;&#8220; <br /> mRNAs targets using unbiased, global <br /> approach. Our findings RBP42 binds <br /> F1-ATPase mRNAs more <br /> mRNAs involved energy metabolism <br /> in trypanosomes help facilitate the <br /> discovery additional G3BP targets in <br /> metazoans. Indeed, G3BP altered in <br /> several cancer cells, binding to <br /> F1-ATPase c-Myc mRNAs been <br /> speculated play role meta- <br /> bolic changes occur altered <br /> cell growth (Guitard et al. 2001; Tourriere <br /> et al. 2001; Ortega et al. 2010; Shim <br /> et al. 2010; Cairns et al. 2011). It be <br /> interesting RBP42 also <br /> plays role adjusting bioenergetics <br /> in trypanosomes, possibly response to <br /> the constantly changing nutritional envi- <br /> ronment parasite life cycle.                            FIGURE 9. (Legend page) <br />  <br />  <br /> 1978         RNA, Vol. 18, No. 11 <br />                                                                                                  Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br /> the coding region APP promotes translation,                    partners. We speculate similar scenario occurs in <br /> FMRP, fragile X-mental retardation protein, binds                  trypanosomes. Specifically, trypanosomal RBP4 par- <br /> same region stalls ribosome movement (Brown et al.                        ticipate post-transcriptional gene control, partnering <br /> 2001; Lee et al. 2010; Darnell et al. 2011). In trypanosomes,                 RBP42 stimulate translation, modulate mRNA decay, <br /> RBP42 targeting mRNAs stimulatory                        possibly pause translation certain environmental <br /> or inhibitory effect dictated environment                    conditions. <br /> different transcripts.                                                           In summary, RBP42 new member growing <br />    As RBP42 originally identified interaction                 family RNA-binding proteins target coding <br /> the RPB4 subunit RNA polymerase II, cytoplasmic                        region mRNAs. By taking panoramic approach RNA <br /> localization surprising. Although RBP42 function                      target discovery, begin define networks in <br /> in nucleus cytoplasm, immunofluo-                      RBP42 participates. The studies described herein <br /> rescence microscopy data cell fractionation analysis                      RBP42 integrated vitally important <br /> show bulk RBP42 functions cytoplasm                     energy metabolic network procyclic life cycle stage <br /> association polysomes. Accordingly, speculate                    T. brucei. How RBP42 RNA-binding pro- <br /> the copurification RBP42 RPB4 indicates                          teins coordinate cellular processes organism heavily <br /> trypanosomal RPB4 functions cytoplasm. In yeast,                       reliant post-transcriptional control remains be <br /> Rpb4p functions cytoplasm, participating post-                      discovered. <br /> transcriptional reactions Rbp4/Rbp7 dimer, which <br /> is independent core RNA polymerase II complex                          MATERIALS AND METHODS <br /> (Selitrennik et al. 2006; Harel-Sharvit et al. 2010). The <br /> Rbp4/Rpb7 complex regulates decay mRNAs                          Trypanosome strains transformations <br /> products involved protein synthesis, including ri- <br /> bosomal proteins RPL25 translation factors                      Wild-type T. brucei Lister 427 procyclic (tsetse midgut form) <br /> such TIF1 (Pillai et al. 2003; Lotan et al. 2005). Regulation            parasites grown SDM-79 medium, supplemented with <br />                                                                             10% fetal bovine serum (FBS) 27&#194;&#176;C humidified chamber <br /> of specific group mRNAs requires Rbp4/Rbp7 <br />                                                                             containing 5% CO2. TbRPB4-tagged cells (described Das et al. <br /> to interact directly mRNA decay machinery. In <br />                                                                             2006) contain NEO marker (neomycin resistance gene) place <br /> addition, Rpb4/Rpb7 stimulates translation initiation                    RPB4 allele BLE marker (phleomycin resistance gene) <br /> mRNAs, probably direct interaction                       associated in-frame C-terminal tandem affinity protein <br /> translation initiation factor eIF3 (Harel-Sharvit et al.                    (TAP) tag second RPB4 allele. The transgenes are <br /> 2010). The dual roles RBP4 mRNA decay                             maintained TbRPB4-tagged cells 15 mg/mL G418 and <br /> translation suggest possesses dynamic set protein              2.5 mg/mL phleomycin. (See Table 1 strains.) <br />                                                                                                     T. brucei Lister 427 cell line 29-13 (Wirtz <br />                                                                                                  et al. 1999) constitutively expressing T7 <br />                                                                                                  RNA polymerase tetracycline repressor <br /> FIGURE 9. A combination visual computer-based analysis reveals transfected RBP42-RNAi con- <br /> significant RBP42-bound mRNAs encode proteins function energy production. <br />                                                                                                  struct, pZJM (BLE) derivative <br /> Metabolism-associated genes highest enrichment values (top 10%) RBP-42&#226;&#8364;&#8220;bound <br /> RNA tags depicted dark green ovals; genes above-background values depicted contained region RBP42 ORF. The <br /> light green ovals. Enzymes (1) hexokinase (HK); (2) glucose-6-phosphate isomerase (GPI); clonal cell lines produced named RBP42- <br /> (3) phosphofructokinase (PFK); (4) aldolase; (5) triose-phosphate isomerase (TIM); (6) glycerol- clones 1&#226;&#8364;&#8220;4 grew presence of <br /> 3-phosphate dehydrogenase (G3PDH); (7) glycerol kinase-glycosomal (GKg); (8) glyceralde- 2.5 mg/mL phleomycin. RBP42-clone 3 was <br /> hyde-3-phosphate dehydrogenase (GAPDH); (9) phosphoglycerate kinase (cytosolic, PGKc, used create AM1-5 cell lines which <br /> glycosomal [PGKg]); (10) phosphoglycerate mutase (PGAM); (11) enolase (ENO); (12) pyruvate PUR cassette replaced RBP42 <br /> kinase (PYK); (13) phosphoenolpyruvate carboxykinase (PEPCK); (14) pyruvate phosphate <br /> dikinase (PPDK); (15) malate dehydrogenase-glycosomal (MDHg); (16) fumarase-cytosolic alleles non RNAi-induced conditions. <br /> (FHc); (17) NADH-dependent fumarate reductase (FRDg); (18) adenylate kinase-glycosomal Double-stranded RNAi induced add- <br /> (AK), GIM5-glycosomal membrane protein (GIM5), PEX11-glycosomal membrane protein ing tetracycline (2 mg/mL). <br /> (PEX11); (19) malic enzyme-cytoplasmic (MEc); (20) pyruvate dehydrogenase complex (PDH); <br /> (21) acetate:succinate CoA transferase (ASCT); (22) unknown enzyme; (23) succinyl CoA <br /> synthetase (SCoAS); (24) citrate synthase (CS); (25) aconitase (ACO); (26) isocitrate                 Plasmid constructions <br /> dehydrogenase (IDH); (27) 2-ketoglutarate dehydrogenase (ODGC); (28) succinate dehydroge- <br /> nase (SDH); (29) fumarase-mitochondrial (FHm); (30) malate dehydrogenase-mitochondrial                T. brucei RBP42 (Gene ID Tb927.6.4440) <br /> (MDHm); (31) proline dehydrogenase (PRODH); (32) pyrroline-5-carboxylate reductase (P5C               previously designated A7 Table 2 <br /> reductase); (33) L-alanine aminotransferase (ALAT); (34) glutamate dehydrogenase (GDH); (35)          Das et al. (2006). The RBP42-RNAi con- <br /> glycerol-3-phosphate oxidase (FAD-G3PDH); (36) rotenone-insensitive NADH dehydrogenase                struct completed pZJM inserting <br /> (R&#226;&#8364;&#8220;I); (37) alternative oxidase (AO); (38) F0/F1-ATP synthase (b-, g-subunits subunit 9)      486 bp DNA sequence containing 55- <br /> in complexes I&#226;&#8364;&#8220;IV respiratory chain (in complex I, subunit 20-kDa subunit indicated; in <br /> complex II, subunits indicated; complex III, Rieske iron-sulfur protein indicated; <br />                                                                                                       540-bp region RBP42 ORF. Primers <br /> in complex IV, cytochrome oxidase subunits IV, V, VII&#226;&#8364;&#8220;IX indicated; cytochrome C          AD277 (59-ACAAAGCTTTTGCTACGGCT <br /> [Cyto C]); (39) threonine-3-dehydrogenase (TDH); (40) 2-amino-3-ketobutyrate CoA ligase               CCTGAAAATC-39) AD278 (59-ACACT <br /> (ABK ligase); (41) transaldolase; (42) transketolase.                                             CGAGCAGGTTTCTTATTTGGGGCAC-39) <br />  <br />                                                                                                                   www.rnajournal.org      1979 <br />  Das et al. <br />  <br />  <br />  <br />  TABLE 1. Strains <br />  <br />  Cell line           Background                  Genotype                         Drug selection                    Experimental purpose <br />  <br />  Wild type          Lister 427                                                                                RNA-CLIP, RBP42 localization <br />  RPB4-tagged        Wild type         Drpb4TNEO RPB4TTAP BLE           G418, phleomycin                     Biochemical purification of <br />                                                                                                                 RPB4-associated proteins <br />  29-13              Wild type         TETR HYG T7RNAP NEO b              Hygromycin, G418                     Parent strain conditional gene <br />                                                                                                                 expression experiments <br />  RBP42-clone 3      29-13             TETR HYG T7RNAP NEO                Hygromycin, G418, phleomycin         RBP42 conditional knockdown <br />                                         RIT- RBP42Ti BLE c                                                      studies <br />  AM1-3              RBP42-clone 3     Drbp42TPUR                         Hygromycin, G418, phleomycin,        RBP42 conditional knockdown <br />                                                                           puromycin                             studies <br />  a <br />   Das et al. (2006). <br />  b <br />    Wirtz et al. (1999). <br />  c <br />   RIT adapted Alsford et al. (2011), define term RIT-Seq <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) target sequencing. RIT-RBP42Ti describes the <br />  use tetracycline-regulated expression RNAi ablate RBP42 mRNA cells. <br />  <br />  <br />  <br /> contain HindIII XhoI recognition sequences facilitate               1 mM DTT, complete protease inhibitors) containing 1% NP-40. <br /> insertion pZJM used PCR-amplify insert                Total lysate cleared centrifugation (10,000g, 10 min), and <br /> from T. brucei genomic DNA. The DNA sequences recombi-              post-nuclear supernatants collected. To isolate purified <br /> nant plasmids verified.                                               polysomes, 3 mL cytoplasmic extract layered 7 mL <br />                                                                            30% sucrose cushion centrifuged (33,000 rpm, 2.5 h) a <br /> Antibodies                                                                 SW41 rotor. The polysome pellet resuspended polysome <br />                                                                            buffer stored 80&#194;&#176;C. Approximately 90 OD260 units of <br /> Anti-RBP42 antibodies produced injecting rabbits           polysomes recovered 1010 cells. Polysome profiles <br /> recombinant RBP42 protein Escherichia coli GST-               generated fractionating cytoplasmic extracts 5 mL <br /> fusion protein. GST-tagged RBP42 purified using glutathione-           15%&#226;&#8364;&#8220;50% sucrose gradients (41,000 rpm, 2 h) SW55 rotor. <br /> Sepharose beads, according manufacturer&#226;&#8364;&#8482;s protocol (GE              Ten 0.5 mL gradient fractions collected manually the <br /> Healthcare), soluble RBP42, separated GST domain              tube punching hole, fraction was <br /> by Precision protease, used raise polyclonal antibodies          analyzed RNA proteins. <br /> the Lampire Corporation. Anti-DRBD3 antibodies raised in <br /> rabbits N-terminal 83-amino-acid polypeptide from <br /> DRBD3 (Estevez 2008). Anti-GRASP antibodies gift               In vivo crosslinking immunocapture <br /> Graham Warren; anti-BiP antibodies gift James                  protein&#226;&#8364;&#8220;RNA complexes <br /> Bangs; anti-ribosomal protein L5 anti-PABP1 antibodies <br /> were gifts Noreen Williams. Anti-a tubulin antibody               Wild-type T. brucei procyclic cells harvested mid-log- <br /> from Sigma Corporation.                                                    phase cells (4 3 106 5 3 106 cells/mL), washed cold <br />    Antibody specificities assayed incubating parasite ex-          phosphate buffered saline (PBS) resuspended cold PBS to <br />                                                                            concentration 5 3 108/mL. Cell suspensions (10 mL) were <br /> tracts, RIPA buffer (20 mM Tris-HCl pH 7.5, 150 mM <br /> NaCl, 1 mM EDTA, 1 mM EGTA, 1% NP-40, 1% Na-deoxycholate,                  transferred 100-mm Petri dish, placed ice tray, and <br /> 2.5 mM Na-pyrophosphate, 1 mM B-glycerophosphate, 1 mM                     UV-irradiated twice 400 mJ/cm2 Stratalinker 1800 <br /> Na3VO4, 1 mg/mL leupeptin), anti-RBP42 anti-DRBD3                  (Stratagene) 4&#226;&#8364;&#8220;5 cm distant UV source (each irradia- <br /> antibodies bound Dynabeads protein G (Invitrogen). Immu-                tion step takes 1.5 min). Cells rapidly pelleted, snap frozen <br /> nodepleted samples analyzed immunoblotting using                   liquid N2, stored 80&#194;&#176;C small aliquots. <br /> ECL reagents GE Healthcare.                                              Immunocapture crosslinked protein&#226;&#8364;&#8220;RNA complexes was <br />                                                                            carried according method described Ule et al. <br /> Cellular fractionation polysome purification                           (2003). Cellular extracts concentrating UV- <br />                                                                            irradiated cells centrifugation resuspending in <br /> The cellular fractionation protocol adapted method            buffer A (PBS, 0.1% SDS, 0.5% Na-deoxycholate, 0.5% NP- <br /> described Michaeli et al. (1993). Briefly, 109 procyclic T. brucei      40). Extracts treated RQ1 DNase (15 min, 37&#194;&#176;C) to <br /> cells disrupted 5 mL hypotonic buffer (10 mM Tris-HCl           reduce viscosity incubated titrated amounts of <br /> pH 7.5, 10 mM KCl, 2 mM Mg2Cl, 1 mM DTT, complete protease                 RNase T1 fragment protein-bound RNAs. The protein- <br /> inhibitor; Roche) containing 0.2% NP-40 15&#226;&#8364;&#8220;20 strokes              bound RNA samples clarified centrifugation (25 min, <br /> Dounce homogenizer. Nuclei isolated pelleting            90K) Beckman TLA120.2 rotor, specific protein&#226;&#8364;&#8220;RNA <br /> 3 mL 0.8 M sucrose cushion (8000g, 10 min). The cytoplasmic                complexes captured incubation (2 h, 4&#194;&#176;C) anti- <br /> extracts, collected sucrose cushion,      RBP42 anti-DRBD3 antibodies attached Dynabeads Pro- <br /> separated S100 sup pellet centrifugation (100,000g,            tein G beads. Captured protein&#226;&#8364;&#8220;RNA complexes washed <br /> 1 h) SW55 rotor.                                                     extensively, twice buffer A twice buffer B (buffer A <br />    For polysome analysis, cellular extracts polysome          supplemented 53 PBS), analyzed immunoblot or <br /> buffer (20 mM Tris-HCl pH 7.5, 120 mM KCl, 2 mM MgCl2,                  processed CLIP. <br />  <br /> 1980         RNA, Vol. 18, No. 11 <br />                                                                                         Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br /> Purification poly(A)+ RNA&#226;&#8364;&#8220;protein complexes                        alignment mismatches seed. The reads with <br />                                                                       identical index sequence 59 end collapsed in <br /> Cytoplasmic extracts, collected described above, adjusted     order minimize PCR-based amplification artifacts generate <br /> to 0.5% SDS 1% 2-mercaptoethanol incubated 5 min          unique reads set downstream peak analysis. The reads were <br /> at 90&#194;&#176;C. The heated extracts quickly chilled ice in-       annotated grouped categories based their <br /> cubating 5 min, LiCl added final concen-        mapped locations: coding region, intercoding region, abundant <br /> tration 0.5 M. Subsequently, poly(A)+ RNA&#226;&#8364;&#8220;protein complexes        RNAs (rRNA, tRNA), anti-sense RNAs, (i.e., ncRNA, <br /> were isolated subsequent rounds selection oligo-       snRNA). The read distribution groups compared <br /> d(T) cellulose (type 7, GE Healthcare) collected elution       RBP42-bound DRBD3-bound RNAs. <br /> buffer containing 10 mM Tris-HCl (pH 7.4), 0.05% SDS. The <br /> protein components isolated poly(A)+ RNA&#226;&#8364;&#8220;protein com-          Peak profiling Gene Ontology analysis <br /> plexes analyzed immunoblotting specific antibodies. <br />                                                                       F-Seq (version 1.84), algorithm based kernel density estima- <br /> RBP42 DRBD3 HITS-CLIP                                             tion, run identify candidate regions (peaks) significant <br />                                                                       enrichment RBP42 (or DRBD3)-bound RNA sequence tags <br /> The cDNA libraries protein-bound RNAs              (Boyle et al. 2008). The fragment size, f, set 40. The threshold, <br /> according method described Licatalosi et al. (2008).        s, 6, instead 4 (default). The generated list peaks then <br /> After capturing protein&#226;&#8364;&#8220;RNA complexes anti-RBP42         annotated. To select list significant genes high peaks, we <br /> anti-DRBD3 antibodies, RNA linker (RL3; 59P-GUGUCAGU               plotted distribution F-Seq enrichment score (ES) peaks <br /> CACUUCCAGCGG-39-puromycin) ligated 39 end               performed extensive visual inspection top-ranked peaks using <br /> the protein-bound-RNAs using T4 RNA ligase, 59 ends           IGV browser (Integrative genomic browser; version 1.5 www. <br /> were phosphorylated T4 polynucleotide kinase sus-          broadinstitute.org/igv). An empirical cutoff (ES $ 25) used <br /> pended beads. To purify protein&#226;&#8364;&#8220;RNA complexes, small         generate list significant genes, categorized by <br /> amount 32P-g-ATP used tracer T4 PNK reaction.      Gene Ontology (GO) analysis. GO analysis performed using <br /> Protein&#226;&#8364;&#8220;RNA complexes resolved PAGE-SDS, transferred          GOToolBox (Martin et al. 2004) using nonredundant gene list <br /> to nitrocellulose membrane, excised detection X-ray      GOSlim set Alsford et al. (2011). GO associations with <br /> film. RNAs extracted treating excised nitrocellulose      P-value &lt;0.001 considered overrepresented GO categories. <br /> filters proteinase K purified ethanol precipitation. A <br /> 59 RNA linker (RL5D; 59-AGGGAGGACGAUGCGGNNNNG-39)                     Fluorescence light microscopy <br /> was ligated 59 end purified RNA using T4 RNA ligase. <br /> The random assortment 4 nucleotides (nt) positions     Cells fixed 2% paraformaldehyde 15 min room <br /> at 39 end 59 linker resulted set 256 unique 59     temperature, adhered slides coated poly L-lysine, stained <br /> linkers. During subsequent analysis, linker sequences serve   DAPI, visualized 1003/1.30 oil-immersion <br /> as indices distinguish individual RNAs PCR-generated          objective (combined 103 ocular magnification of <br /> amplifications. First-strand cDNA produced             10003) using UplanF1 objective. An Olympus BX61 used in <br /> linker-ligated RNAs using Superscript (Invitrogen) primer       combination DAPI FITC-sensitive filters. Vectashield <br /> complementary 39 linker sequence (DP3; 59-CCGCTG               (Vector Laboratories) used anti-fade. Intact cells were <br /> GAAGTGACTGACAC-39). Libraries cDNA generated                  detected using bright field optics. Images captured using <br /> by PCR amplification first-strand cDNA using 59 primer         Hamamatsu ORCA-ER camera acquired 0.065-mM 3 <br /> (DP5; 59-AGGGAGGACGATGCGG-39) DP3. As quality-                  0.065-mM pixel resolution. <br /> control measure, fraction PCR-generated cDNAs            RBP42 immunolocalization performed using anti-RBP42 <br /> inserted TOPO plasmids transformed bacteria. The          antibodies. After fixing adhering poly-L-lysine coated <br /> presence inserts confirmed restriction analysis          slides, cells permeabilized using 0.2% NP40 5 min at <br /> plasmid DNAs isolated 20 randomly selected bacterial             room temperature. Z-stacks obtained 0.2-mM spacing <br /> colonies. Capillary sequencing 20 clones confirmed         stack. A total 15 stacks generated field. <br /> heterogeneous pool cDNA inserts desired 30- 60-bp <br />                                                                       Protein alignments <br /> length. To prepare cDNA libraries sequencing an <br /> Illumina sequencer, cDNAs RBP42 DRBD3&#226;&#8364;&#8220;               Multiple alignments RBP42 proteins T. brucei, Trypanosoma <br /> bound RNAs reamplified primers (DSFP5; 59-AAT           cruzi, Leishmania major, Leptomonas seymouri performed <br /> GATACGGCGACCACCGACTATGGATACTTAGTCAGGGAGGA                             using CLUSTALW. Alignments refined manually visual- <br /> CGATGCGG-39 59-CAAGCAGAAGACGGCATACGACCGCT                         ized using GeneDoc (http://www.psc.edu/biomed/genedoc). <br /> GGAAGTGACTGACAC-39) 15 25 cycles, respectively. <br />  <br /> Mapping short reads T. brucei genome                               SUPPLEMENTAL MATERIAL <br /> High-quality reads (after signal filtering Illumina)        Supplemental material available article. <br /> aligned genome T. brucei strain 927 using CASAVA <br /> (version 1.7). The 10 bp read masked during <br />                                                                       ACKNOWLEDGMENTS <br /> the mapping accommodate RNA sequences fragment <br /> length shorter read length. In CASAVA1.7, mapping         We thank James Bangs, Graham Warren, Noreen Williams for <br /> algorithm (ELAND version 2) allows multi-seed gapped              antibodies. We thank Rose Garcia administrative assistance <br />  <br />                                                                                                            www.rnajournal.org          1981 <br />  Das et al. <br />  <br />  <br />  <br /> and Han Wu Edgar Garcia assistance graphics. We             Das A, Li H, Liu T, Bellofatto V. 2006. Biochemical characterization of <br /> thank Michael Mathews, Andrew Parrott, members                    Trypanosoma brucei RNA polymerase II. Mol Biochem Parasitol <br /> R. Darnell laboratory extremely helpful discussions. This                150: 201&#226;&#8364;&#8220;210. <br />                                                                          De Gaudenzi J, Frasch AC, Clayton C. 2005. RNA-binding domain <br /> work supported National Institutes Health <br />                                                                              proteins kinetoplastids: A comparative analysis. Eukaryot Cell 4: <br /> grants RO1 29478 RO1 53835 Foundation UMDNJ                     2106&#226;&#8364;&#8220;2114. <br /> award V.B.                                                            Estevez AM. 2008. The RNA-binding protein TbDRBD3 regulates the <br />                                                                              stability specific subset mRNAs trypanosomes. Nucleic <br /> Received April 12, 2012; accepted August 2, 2012.                            Acids Res 36: 4573&#226;&#8364;&#8220;4586. <br />                                                                          Fernandez-Moya SM, Estevez AM. 2010. Posttranscriptional control <br />                                                                              role RNA-binding proteins gene regulation in <br />                                                                              trypanosomatid protozoan parasites. Wiley Interdiscip Rev RNA <br /> REFERENCES                                                                   1: 34&#226;&#8364;&#8220;46. <br /> Acosta-Serrano A, Vassella E, Liniger M, Kunz Renggli C, Brun R,         Freire ER, Dhalia R, Moura DM, da Costa Lima TD, Lima RP, Reis <br />     Roditi I, Englund PT. 2001. The surface coat procyclic                CR, Hughes K, Figueiredo RC, Standart N, Carrington M, et al. <br />     Trypanosoma brucei: Programmed expression proteolytic                2011. The trypanosomatid eIF4E homologues fall two <br />     cleavage procyclin tsetse fly. Proc Natl Acad Sci 98:          separate groups, distinct features primary sequence and <br />     1513&#226;&#8364;&#8220;1518.                                                               biological properties. Mol Biochem Parasitol 176: 25&#226;&#8364;&#8220;36. <br /> Alsford S, Turner DJ, Obado SO, Sanchez-Flores A, Glover L,              Guitard E, Parker F, Millon R, Abecassis J, Tocque B. 2001. G3BP is <br />     Berriman M, Hertz-Fowler C, Horn D. 2011. High-throughput                overexpressed human tumors promotes S phase entry. <br />     phenotyping using parallel sequencing RNA interference targets        Cancer Lett 162: 213&#226;&#8364;&#8220;221. <br />     African trypanosome. Genome Res 21: 915&#226;&#8364;&#8220;924.                  Hammarton TC. 2007. Cell cycle regulation Trypanosoma brucei. <br /> Angenstein F, Evans AM, Settlage RE, Moran ST, Ling SC, Klintsova            Mol Biochem Parasitol 153: 1&#226;&#8364;&#8220;8. <br />     AY, Shabanowitz J, Hunt DF, Greenough WT. 2002. A receptor       Harel-Sharvit L, Eldad N, Haimovich G, Barkai O, Duek L, Choder M. <br />     activated C kinase messenger ribonucleoprotein com-           2010. RNA polymerase II subunits link transcription mRNA <br />     plexes associated polyA-mRNAs neurons. J Neurosci 22:            decay translation. Cell 143: 552&#226;&#8364;&#8220;563. <br />     8827&#226;&#8364;&#8220;8837.                                                           Ho HH, He CY, Graffenried CL, Murrells LJ, Warren G. 2006. <br /> Atlas R, Behar L, Elliott E, Ginzburg I. 2004. The insulin-like growth       Ordered assembly duplicating Golgi Trypanosoma brucei. <br />     factor mRNA binding-protein IMP-1 Ras-regulatory                 Proc Natl Acad Sci 103: 7676&#226;&#8364;&#8220;7681. <br />     protein G3BP associate tau mRNA HuD protein              Irvine K, Stirling R, Hume D, Kennedy D. 2004. Rasputin, more <br />     differentiated P19 neuronal cells. J Neurochem 89: 613&#226;&#8364;&#8220;626.              promiscuous ever: A review G3BP. Int J Dev Biol 48: 1065&#226;&#8364;&#8220; <br /> Atlas R, Behar L, Sapoznik S, Ginzburg I. 2007. Dynamic association          1077. <br />     polysomes P19 neuronal differentiation un-        Keene JD. 2007. RNA regulons: Coordination post-transcriptional <br />     translated-region-dependent translation regulation tau            events. Nat Rev Genet 8: 533&#226;&#8364;&#8220;543. <br />     mRNA tau mRNA-associated proteins IMP1, HuD,              Kim MM, Wiederschain D, Kennedy D, Hansen E, Yuan ZM. <br />     G3BP1. J Neurosci Res 85: 173&#226;&#8364;&#8220;183.                                       2007. Modulation p53 MDM2 activity novel inter- <br /> Bandziulis RJ, Swanson MS, Dreyfuss G. 1989. RNA-binding proteins            action Ras-GAP binding proteins (G3BP). Oncogene 26: <br />     developmental regulators. Genes Dev 3: 431&#226;&#8364;&#8220;437.                       4209&#226;&#8364;&#8220;4215. <br /> Barnes CJ, Li F, Mandal M, Yang Z, Sahin AA, Kumar R. 2002.              Kramer S, Carrington M. 2011. Trans-acting proteins regulating <br />     Heregulin induces expression, ATPase activity, nuclear local-        mRNA maturation, stability translation trypanosomatids. <br />     ization G3BP, Ras signaling component, human breast              Trends Parasitol 27: 23&#226;&#8364;&#8220;30. <br />     tumors. Cancer Res 62: 1251&#226;&#8364;&#8220;1255.                                    Kramer S, Queiroz R, Ellis L, Hoheisel JD, Clayton C, Carrington M. <br /> Barquilla A, Crespo JL, Navarro M. 2008. Rapamycin inhibits                  2010. The RNA helicase DHH1 central correct expression <br />     trypanosome cell growth preventing TOR complex 2 formation.           developmentally regulated mRNAs trypanosomes. <br />     Proc Natl Acad Sci 105: 14579&#226;&#8364;&#8220;14584.                                     J Cell Sci 123: 699&#226;&#8364;&#8220;711. <br /> Besteiro S, Barrett MP, Riviere L, Bringaud F. 2005. Energy generation   Lee EK, Kim HH, Kuwano Y, Abdelmohsen K, Srikantan S, Subaran <br />     insect stages Trypanosoma brucei: Metabolism flux. Trends       SS, Gleichmann M, Mughal MR, Martindale JL, Yang X, et al. <br />     Parasitol 21: 185&#226;&#8364;&#8220;191.                                                   2010. hnRNP C promotes APP translation competing with <br /> Boyle AP, Guinney J, Crawford GE, Furey TS. 2008. F-Seq: A feature           FMRP APP mRNA recruitment P bodies. Nat Struct Mol <br />     density estimator high-throughput sequence tags. Bioinfor-           Biol 17: 732&#226;&#8364;&#8220;739. <br />     matics 24: 2537&#226;&#8364;&#8220;2538.                                                Licatalosi DD, Mele A, Fak JJ, Ule J, Kayikci M, Chi SW, Clark TA, <br /> Bringaud F, Riviere L, Coustou V. 2006. Energy metabolism                 Schweitzer AC, Blume JE, Wang X, et al. 2008. HITS-CLIP yields <br />     trypanosomatids: Adaptation available carbon sources. Mol             genome-wide insights brain alternative RNA processing. <br />     Biochem Parasitol 149: 1&#226;&#8364;&#8220;9.                                              Nature 456: 464&#226;&#8364;&#8220;469. <br /> Brown V, Jin P, Ceman S, Darnell JC, O&#226;&#8364;&#8482;Donnell WT, Tenenbaum SA,         Lotan R, Bar-On VG, Harel-Sharvit L, Duek L, Melamed D, Choder <br />     Jin X, Feng Y, Wilkinson KD, Keene JD, et al. 2001. Microarray           M. 2005. The RNA polymerase II subunit Rpb4p mediates decay of <br />     identification FMRP-associated brain mRNAs altered                specific class mRNAs. Genes Dev 19: 3004&#226;&#8364;&#8220;3016. <br />     mRNA translational profiles fragile X syndrome. Cell 107:         Mani J, Guttinger A, Schimanski B, Heller M, Acosta-Serrano A, <br />     477&#226;&#8364;&#8220;487.                                                                 Pescher P, Spath G, Roditi I. 2011. Alba-domain proteins of <br /> Cairns RA, Harris IS, Mak TW. 2011. Regulation cancer cell                Trypanosoma brucei cytoplasmic RNA-binding proteins that <br />     metabolism. Nat Rev Cancer 11: 85&#226;&#8364;&#8220;95.                                    interact translation machinery. PLoS ONE 6: e22463. doi: <br /> Coustou V, Biran M, Breton M, Guegan F, Riviere L, Plazolles N,              10.1371/journal.pone.0022463. <br />     Nolan D, Barrett MP, Franconi JM, Bringaud F. 2008. Glucose-         Martin D, Brun C, Remy E, Mouren P, Thieffry D, Jacq B. 2004. <br />     induced remodeling intermediary energy metabolism              GOToolBox: Functional analysis gene datasets based on <br />     procyclic Trypanosoma brucei. J Biol Chem 283: 16342&#226;&#8364;&#8220;16354.              Gene Ontology. Genome Biol 5: R101. doi: 10.1186/gb-2004- <br /> Darnell JC, Van Driesche SJ, Zhang C, Hung KY, Mele A, Fraser CE,            5-12-r101. <br />     Stone EF, Chen C, Fak JJ, Chi SW, et al. 2011. FMRP stalls           Michaeli S, Goldring A, Agabian N. 1993. Identification mRNA- <br />     ribosomal translocation mRNAs linked synaptic function             binding proteins lower trypanosomatid Leptomonas collosoma. <br />     autism. Cell 146: 247&#226;&#8364;&#8220;261.                                           Exp Parasitol 76: 308&#226;&#8364;&#8220;313. <br />  <br />  <br /> 1982         RNA, Vol. 18, No. 11 <br />                                                                                            Coordination mRNA metabolism RBP42 <br />  <br />  <br />  <br /> Ortega AD, Willers IM, Sala S, Cuezva JM. 2010. Human G3BP1              Siegel TN, Gunasekera K, Cross GA, Ochsenreiter T. 2011. Gene <br />     interacts b-F1-ATPase mRNA inhibits translation.            expression Trypanosoma brucei: Lessons high-throughput <br />     J Cell Sci 123: 2685&#226;&#8364;&#8220;2696.                                               RNA sequencing. Trends Parasitol 27: 434&#226;&#8364;&#8220;441. <br /> Parker F, Maurier F, Delumeau I, Duchesne M, Faucher D, Debussche        Solomon S, Xu Y, Wang B, David MD, Schubert P, Kennedy D, <br />     L, Dugue A, Schweighoffer F, Tocque B. 1996. A Ras-GTPase-               Schrader JW. 2007. Distinct structural features caprin-1 mediate <br />     activating protein SH3-domain-binding protein. Mol Cell Biol 16:         interaction G3BP-1 induction phosphorylation <br />     2561&#226;&#8364;&#8220;2569.                                                               eukaryotic translation initiation factor 2a, entry cytoplasmic <br /> Parsons M. 2004. Glycosomes: Parasites divergence      <br /> </body></html>