<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Human eIF4AIII interacts an <br /> eIF4G-like partner, NOM1, revealing <br /> an evolutionarily conserved function <br /> outside exon junction complex <br /> Andrei Alexandrov, David Colognori, Joan A. Steitz1 <br /> Department Molecular Biophysics Biochemistry, Howard Hughes Medical Institute, Yale University School Medicine, <br /> New Haven, Connecticut 06536, USA <br />  <br />  <br />  <br /> Despite lack exon junction complex (EJC), Saccharomyces cerevisiae contains Fal1p, DEAD-box <br /> helicase highly homologous eIF4AIII. We yeast Fal1p functionally orthologous human eIF4AIII, <br /> since expression human eIF4AIII complements lethal <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> 18S rRNA biogenesis defect <br /> of fal1D(null) yeast. We yeast Fal1p interacts genetically eIF4G-like protein, Sgd1p: One <br /> allele sgd1 acts dominant extragenic suppressor <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> predicted RNA-binding residue Fal1p, <br /> whereas synthetically exacerbates growth defect fal1 mutation. Both sgd1 mutations map <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> a single, short, evolutionarily conserved patch matches key eIF4A-interacting residues eIF4G when <br /> superimposed X-ray structure eIF4A/eIF4G complex. We demonstrate direct physical interactions <br /> between yeast Sgd1p Fal1p, human orthologs (NOM1 eIF4AIII) vitro vivo, <br /> identifying human NOM1 missing eIF4G-like interacting partner eIF4AIII. Knockdown eIF4AIII and <br /> NOM1 human cells demonstrates novel conserved eIF4A/eIF4G-like complex acts pre-rRNA <br /> processing, adding established functions eIF4A/eIF4G translation initiation eIF4AIII core <br /> component EJC. <br /> [Keywords: eIF3AIII; NOM1; FAL1; SGD1; exon junction complex; ribosome biogenesis] <br /> Supplemental material available article. <br /> Received July 20, 2010; revised version accepted March 28, 2011. <br />  <br />  <br />  <br />  <br /> RNA helicases ubiquitously present organisms                         functions, substrates, relevant interacting partners of <br /> ranging viruses humans. Helicases participate                       RNA helicases known cases (Bleichert <br /> in numerous aspects RNA metabolism, including                            Baserga 2007). The DEAD-box subfamily largest <br /> transcription, editing, mRNA splicing, 39-end formation,                    group RNA helicases, displaying considerable sequence <br /> export, mRNA translation initiation termination,                        homology, including conservation helicase motifs and <br /> quality control, degradation, ribosome biogenesis                       helicase core (Tanner Linder 2001). Yet, DEAD- <br /> (Rocak Linder 2004). Known molecular functions                          box helicases specific: They able com- <br /> of RNA helicases include unwinding RNA duplexes                          plement deletion DEAD-box helicase even <br /> (Ray et al. 1985), removal mRNP protein components                       overexpressed (Tanner et al. 2003). <br /> (Fairman et al. 2004), serving RNA clamps (Ballut et al.                    The translation initiation factor eIF4A (encoded by <br /> 2005), annealing RNA strands (Yang Jankowsky 2005),                     eIF4AI eIF4AII) best studied DEAD-box <br /> and stabilizing RNA folding intermediates (Solem et al.                     helicases. eIF4A considered &#226;&#8364;&#732;&#226;&#8364;&#732;minimal&#226;&#8364;&#8482;&#226;&#8364;&#8482; DEAD-box <br /> 2006). For functions, energy NTP                       helicase core eIF4A-like domain lacks N-termi- <br /> hydrolysis strictly required. For others, like serving                nal C-terminal extensions (Tanner Linder 2001). <br /> RNA clamps placeholders, ATP hydrolysis appears                    eIF4A functions cap-dependent translation initiation <br /> dispensable (Ballut et al. 2005; Shibuya et al. 2006). Despite              required melt secondary structure the <br /> their variety (Saccharomyces cerevisiae, example,                   59 untranslated region (UTR) allow scanning mRNA <br /> 39 putative helicases) essential roles, exact                 recognition start codon. It hetero- <br />                                                                             trimeric complex eIF4F (eIF4G, eIF4A, eIF4E) (Rozen <br /> 1 <br />                                                                             et al. 1990), eIF4G central scaffolding protein <br />  Corresponding author. <br /> E-MAIL joan.steitz@yale.edu; FAX (203) 624-8213.                            binds m7Gppp cap-binding protein eIF4E and <br /> Article online http://www.genesdev.org/cgi/doi/10.1101/gad.2045411.   recruits 43S preinitiation complex (consisting the <br />  <br />  <br /> 1078       GENES &amp; DEVELOPMENT 25:1078&#226;&#8364;&#8220;1090 &#195;&#8220; 2011 Cold Spring Harbor Laboratory Press ISSN 0890-9369/11; www.genesdev.org <br />                                                                                         eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br />  <br /> 40S ribosomal subunit initiation factors eIF1, eIF1A,      vitro, demonstrated coprecipitation endogenous <br /> eIF2, eIF3, eIF5, Met-tRNAIni) eIF3 subunit        eIF4AIII Flag-tagged NOM1 nuclear extracts of <br /> (Hinnebusch 2006).                                             human cells GST pull-downs heterologously ex- <br />    The DEAD-box helicase eIF4AIII striking homol-          pressed purified proteins. Consistent conservation <br /> ogy eIF4A, functionally distinct:     function complex yeast humans, <br /> It known role translation initiation, localized   knockdown eIF4AIII NOM1 affects pre-rRNA <br /> mostly nucleus, does substitute eIF4A in     processing site 1 human cells, knockdown of <br /> vitro ribosome-binding assays, inhibits translation     eIF4AIII additionally affects pre-rRNA processing site 2. <br /> vitro (Li et al. 1999). Human eIF4AIII reported       Therefore, depending interacting partners, human <br /> to bind eIF4G vitro (Li et al. 1999), functional     eIF4AIII plays dual role central biogenesis <br /> consequences association described.          pathways cells: leading rRNA mRNA. <br /> Instead, eIF4AIII plays central role coupling mRNA <br /> splicing downstream events core component the <br /> exon junction complex (EJC). This multiprotein complex         Results <br /> is deposited mRNA splicing ;25 nucleo- <br />                                                                Human eIF4AIII specifically complements lethal <br /> tides (Le Hir et al. 2000a) upstream exon&#226;&#8364;&#8220;exon <br />                                                                phenotype restores 18S rRNA levels <br /> junction (Le Hir et al. 2000b). The EJC binds mRNA a <br />                                                                fal1D yeast cells <br /> sequence-independent manner high affin- <br /> ity; certain components EJC stably associate       Since S. cerevisiae Fal1p highly homologous meta- <br /> mRNA nuclear export later cyto-         zoan eIF4AIII (Fig. 1A), used plasmid shuffle (Supple- <br /> plasm. The EJC plays key role nonsense-mediated           mental Fig. S1A) ask eIF4AIII humans <br /> mRNA decay (NMD) (Le Hir et al. 2001; Lykke-Andersen           D. melanogaster complement <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> <br /> et al. 2001), mRNA localization (Palacios et al. 2004),        yeast Fal1p deletion (Fig. 1B). The observed complemen- <br /> mRNA export nucleus (Le Hir et al. 2001),         tation specific eIF4AIII, yeast human <br /> the enhancing effect splicing translation (Nott et al.   DEAD-box helicases tested (human eIF4AI, human <br /> 2004). The crystal structure EJC core proteins com-     eIF4AII, yeast eIF4A [Tif1p]) complement (Fig. <br /> plexed RNA revealed eIF4AIII&#226;&#8364;&#8482;s RecA-like              1B). Expression analysis Protein A-tagged proteins <br /> domains enclosing ATP molecule, Magoh/Y14              demonstrated lack complementation not <br /> and MLN51 stabilizing eIF4AIII ATP-bound             lower protein expression (Supplemental Fig. S1B), <br /> RNA-locked state (Andersen et al. 2006; Bono et al. 2006).     levels noncomplementing proteins (human <br />    Surprisingly, eIF4AIII-like proteins comprise separate    eIF4AI eIF4AII yeast Tif1p) eIF4AIII mu- <br /> branch evolutionary tree (Li et al. 1999; Linder        tants (K88N E188Q) higher similar those <br /> 2003), &gt;60% sequence identity entire           human eIF4AIII(WT). <br /> <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> humans yeast (Fig. 1A). At odds           To confirm observed complementation, used <br /> this conservation reports EJC components           promoter shutoff approach (Fig. 1C; Supplemental Fig. <br /> Drosophila melanogaster Caenorhabditis elegans         S1C). fal1D strains expressing human eIF4AIII the <br /> not required NMD (Gatfield et al. 2003; Palacios et al.    GAL1 promoter (fal1D pPGAL1H.eIF4AIII) viable when <br /> 2004; Longman et al. 2007), S. cerevisiae lacks        grown galactose, glucose (vector Fig. 1C). <br /> other known EJC components NMD yeast does           Introduction additional plasmid expressing human <br /> depend splicing (Gonzalez et al. 2001). Instead, S.     eIF4AIII GST fusion PCUP1 promoter (Cu2+- <br /> cerevisiae eIF4AIII-like protein Fal1p essential 18S    responsive; expression increased Cu2+ present) <br /> rRNA biogenesis, specifically 35S pre-rRNA process-        rescued growth fal1D pPGAL1H.eIF4AIII strain on <br /> ing sites A0, A1, A2 (Kressler et al. 1997).            glucose (H.eIF4AIII Fig. 1C). Rescue depended Cu2+ <br />    Here, using genetic screen dominant allele-spe-       concentration medium, double mutations <br /> cific extragenic suppressors fal1 cold-sensitive (CS)   K88A, T89P (Walker A box) D187E, E188L (Walker B <br /> phenotype, identified essential S. cerevisiae com-       box) human eIF4AIII completely abolished comple- <br /> plex, Fal1p/Sgd1p, homologous eIF4A/eIF4G          mentation. Notably, eIF4AIII D. melanogaster also <br /> but acts outside translation initiation. Instead,     complemented fal1D dependent Cu2+, weakly <br /> required 18S rRNA biogenesis. Whereas mutation          human protein. <br /> Fal1p affected growth pre-rRNA processing sites            To test function, asked human eIF4AIII can <br /> A0, A1, A2, suppressing mutation Sgd1p allevi-        rescue 18S rRNA biogenesis defect reported pre- <br /> ated growth pre-rRNA processing defects.          viously fal1 temperature-sensitive (TS) strain (Fig. <br /> We demonstrated human eIF4AIII,             2A; Kressler et al. 1997). Indeed, expression human <br /> highly homologous yeast Fal1p, functional         eIF4AIII GAL1 promoter galactose (but not <br /> ortholog Fal1p, heterologous expression hu-        glucose) restored growth fal1D(null) strain liquid <br /> man eIF4AIII yeast specifically complemented growth         culture (Fig. 2B,C). 18S rRNA levels restored near <br /> and rRNA biogenesis defects FAL1 deletion. Consis-          normal (Fig. 2D, panel), levels 20S rRNA, an <br /> tent evolutionary conservation Fal1p/Sgd1p         18S precursor, partially restored (Fig. 2D, middle <br /> complex, human NOM1, homologous yeast              panel) accumulation 23S rRNA partially <br /> Sgd1p, interacts directly eIF4AIII vivo         relieved (Fig. 2D, panel). Together, results <br />  <br />  <br />                                                                                            GENES &amp; DEVELOPMENT            1079 <br />  Alexandrov et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Human eIF4AIII homologous replace S. cerevisiae Fal1p vivo. (A) Amino acid sequence alignment eIF4AIII <br /> homologous proteins humans, Mus musculus, Xenopus laevis, D. melanogaster, C. elegans, S. cerevisiae. Walker A B <br /> boxes shown gray rectangles. The sequence D. melanogaster VASA bottom. Also indicated amino acids the <br /> Walker boxes human eIF4AIII, mutations (K88N E188Q) reported previously (Shibuya et al. 2006; Zhang and <br />                                                                                                 &#226;&#8364;&#162; <br /> Krainer 2007) abolish ATPase activity vitro, NMD vivo EJC assembly vitro ( ), conserved threonine (T322), <br /> whose mutation valine S. cerevisiae Fal1p resulted CS phenotype (m). (B) Expression human D. melanogaster eIF4AIII <br /> complements lethal phenotype yeast fal1D, observed plasmid shuffle (Supplemental Fig. S1A); Walker A B boxes are <br /> required complementation. (H.eIF4AIII) human eIF4AIII; [H.eIF4AIII(K88N) H.eIF4AIII(E188Q)] Walker A B box mutants of <br /> human eIF4AIII; (D.eIF4AIII) D. melanogaster eIF4AIII; (Fal1) S. cerevisiae Fal1p; (H.eIF4AI H.eIF4AII) human eIF4AI eIF4AII. <br /> (C) Expression human D. melanogaster eIF4AIII complements lethal phenotype yeast fal1D, observed GAL1 <br /> promoter shutoff activation expression copper-inducible promoter (CUP1) increasing concentrations Cu2+ <br /> (Supplemental Fig. S1C). (H.eIF4AIII) human eIF4AIII; (KT/AP) K88T, A89P; (DE/EL) D188E, E189L; (D.eIF4AIII) D. melanogaster <br /> eIF4AIII; (Fal1)S. cerevisiae Fal1p. <br />  <br />  <br />  <br />  <br /> 1080       GENES &amp; DEVELOPMENT <br />                                                                                                    eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br />  <br />  <br /> Figure 2. Expression human eIF4AIII restores 18S rRNA biogenesis S. cerevisiae fal1D strain. (A) Schematic rRNA <br /> processing S. cerevisiae. (B) Depletion Fal1p yeast S. cerevisiae results growth defect liquid culture. (C) Expression of <br /> human eIF4AIII GAL1 promoter rescues growth defect yeast fal1D strain liquid culture. (D, panel) Expression <br /> of human eIF4AIII restores 18S rRNA levels yeast fal1D strain (probes L2 L9). (Middle panel) Expression human eIF4AIII <br /> partially restores 20S rRNA levels yeast fal1D strain (probe L3). (Bottom panel) Expression human eIF4AIII partially relieves <br /> accumulation 23S rRNA yeast fal1D strain (probe L1). Supplemental Figure S2 shows probes. <br />  <br />  <br /> demonstrate human eIF4AIII substitute Fal1p                sgd1(S340Y) dominant extragenic suppressor <br /> and function 18S ribosome biogenesis yeast.                       T322V mutation putative RNA-binding <br />                                                                         domain Fal1p <br /> Mutations conserved Walker A B boxes <br />                                                                         Fal1p considered &#226;&#8364;&#732;&#226;&#8364;&#732;minimal&#226;&#8364;&#8482;&#226;&#8364;&#8482; helicase it <br /> of human eIF4AIII abolish rescue fal1D <br />                                                                         contains RecA domains extensions <br /> yeast phenotype <br />                                                                         (Kressler et al. 1997). This suggests factor(s) <br /> The amino acid sequences Walker A B boxes                 responsible substrate specificity. To iden- <br /> human eIF4AIII completely conserved yeast                   tify factor(s) or, possibly, substrates Fal1p, first <br /> humans (Fig. 1A). Yet, human eIF4AIII bearing K88N                  attempted tandem affinity purification (TAP) approach <br /> E188Q single mutations Walker A B boxes,                     successful, probably transient <br /> respectively, lacking detectable ATPase activity,              nature interactions. We focused genetic <br /> can surprisingly support NMD vivo assemble                    screens, aiming identify spontaneous extragenic domi- <br /> EJC vitro (Shibuya et al. 2006). Moreover, non-                  nant allele-specific suppressors mutations conserved <br /> hydrolyzable ATP analog AMP-PNP substitute                      RNA-interacting amino acids Fal1p (Supplemental Fig. <br /> ATP EJC core assembly vitro (Ballut et al. 2005;              S8A,B). An advantage interacting partners (suppres- <br /> Andersen et al. 2006; Bono et al. 2006), confirming                sors) identified physical interaction required <br /> the ATPase activity eIF4AIII required               cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span>, regardless weak transient the <br /> functions human eIF4AIII bearing K88A muta-                     contact is. Based results genetic screens <br /> tion Walker A box active Y14/Magoh                     (Supplemental Figs. S8&#226;&#8364;&#8220;S10), constructed yeast strains <br /> loading complementation assay despite lack heli-                 carrying individual chromosomal sgd1(S340Y) and <br /> case activity (Zhang Krainer 2007).                                 sgd1(N343Y) mutations fal1(T322V) background <br />    Since conservation Walker A/B boxes                     [abbreviated fal1(T/V)]. Growth curves (Fig. 3A) <br /> Fal1p orthologs nonetheless striking, tested                  serial dilutions (Fig. 3B; Supplemental Fig. S3A) of <br /> K88N E188Q mutations human eIF4AIII com-                     yeast strains demonstrate sgd1(S340Y) acts as <br /> plementation yeast fal1D lethal phenotype. Figure                dominant (Supplemental Fig. S3A, cf. lanes 3,4 5,6) <br /> 1B shows mutations completely abolish rescue,                extragenic suppressor fal1(T/V) CS growth phenotype <br /> highlighting crucial role conserved Walker A B             (Fig. 3B, cf. lanes 2 3 16&#194;&#176;C). <br /> boxes function human eIF4AIII shared                  Three lines evidence demonstrate allele specificity <br /> Fal1p heterologous system.                                      suppression fal1(T/V) CS phenotype by <br />  <br />  <br />                                                                                                       GENES &amp; DEVELOPMENT            1081 <br />  Alexandrov et al. <br />  <br />  <br />  <br />                                                                                   Figure 3. SGD1 interacts genetically with <br />                                                                                   FAL1. (A) Extragenic suppressor sgd1(S340Y) <br />                                                                                   alleviates growth defect CS <br />                                                                                   fal1(T322V) strain liquid culture (YPD) <br />                                                                                   (strains exponentially grown 30&#194;&#176;C <br />                                                                                   shifted 16&#194;&#176;C time 0). (B) Whereas <br />                                                                                   sgd1(S340Y) suppresses growth defect <br />                                                                                   fal1(T322V) strain 16&#194;&#176;C, nearby <br />                                                                                   mutation [sgd1(N343Y)] protein <br />                                                                                   results growth defect synthetic with <br />                                                                                   fal1(T322V) 37&#194;&#176;C (YPD). <br />  <br />  <br />  <br />  <br /> sgd1(S340Y). First, phenotype different fal1(Q144R,    partner Fal1p, 60% identical eIF4A <br /> V167T, A194V, M221T, D223G, C272G, A372T, P390T)                (Fig. 1A; Kressler et al. 1997). <br /> mutant strain TS (Supplemental Fig. S3D,E)             Several lines evidence suggested potential <br /> not suppressed tested temperature sgd1(S340Y),        physical eIF4G/eIF4A-like interaction Sgd1p and <br /> sgd1(N343Y), SGD1(WT) expressed multicopy              Fal1p yeast, NOM1 eIF4AIII (their <br /> single copy (Supplemental Fig. S3B,C). Second,          respective orthologs) humans. First, dominant allele- <br /> SGD1(WT), sgd1(S340Y), sgd1(N343Y) suppressed           specific suppressor mutation, sgd1(S340Y), syn- <br /> lethal fal1D(null) phenotype, demonstrated plasmid        thetic mutation sgd1(N343Y) appeared MIF4G (the <br /> shuffle (Supplemental Fig. S4A,B). Third,           major eIF4A-interacting) domain Sgd1p (Fig. 4A). Second, <br /> chromosomal sgd1(S340Y) mutation suppressed                 MIF4G domain itself, mutations <br /> fal1(T/V) CS phenotype (Fig. 3B, cf. lanes 2 3           remarkably appeared amino acids apart 12- <br /> 16&#194;&#176;C), nearby sgd1(N343Y) mutation little          amino-acid conserved motif (NSSLNKLSDSNI yeast <br /> effect growth fal1(T/V) strain temper-    Sgd1p KSLLNKLTLEMF yeast eIF4G [Tif1p]) (Fig. <br /> ature (Fig. 3B, cf. lanes 2 5 16&#194;&#176;C). Instead,        4A), representing major surface eIF4G contacts <br /> sgd1(N343Y) fal1(T/V) double mutant displayed syn-            eIF4A (Schutz et al. 2008). Together, S612 N615 of <br /> thetic growth defect elevated temperatures (Fig. 3B,         yeast eIF4G (corresponding S340 N343 yeast <br /> cf. lanes 2, 5, 6 37&#194;&#176;C); complementation          Sgd1p) form direct intermolecular hydrogen bonds <br /> synthetic growth defect sgd1(N343Y) fal1(T/V)            eIF4A, based X-ray structure yeast <br /> double mutant plasmid-expressed SGD1 FAL1             eIF4A/eIF4G complex (Fig. 4B; Schutz et al. 2008). Third, <br /> shown Supplemental Figure S4C (cf. lanes 2,5,6 Fig.       human NOM1 shown previously interact with <br /> 3B lanes 2&#226;&#8364;&#8220;4 Supplemental Fig. S4C).                     eIF4A yeast two-hybrid screens pull-down assays <br />                                                                 (Simmons et al. 2005); vitro interaction GST- <br />                                                                 tagged NOM1 eIF4AIII mentioned &#226;&#8364;&#732;&#226;&#8364;&#732;data not <br /> Genetically interacting residues Sgd1p (S340 <br />                                                                 shown&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Simmons et al. 2005). <br /> and N343) identify conserved interface between <br />                                                                    Here, present lines evidence conserved <br /> yeast Sgd1p/Fal1p human NOM1/eIF4AIII <br />                                                                 direct eIF4G/eIF4A-like physical interaction between <br /> The existence dominant allele-specific suppressor          Sgd1p Fal1p yeast, NOM1 eIF4AIII <br /> often reflects functional physical interaction        humans: First, TAP-Fal1p Myc-Sgd1p specifically <br /> two proteins. Sgd1p highly conserved (Fig. 4A)        coimmunoprecipitate extracts formaldehyde <br /> known essential S. cerevisiae (Akhtar et al. 2000),    cross-linked yeast cells (Fig. 5A; Supplemental Fig. S6A). <br /> D. melanogaster (http://www.flybase.org), C. elegans        Second, comparable specific interaction exists between <br /> (Maeda et al. 2001); Sgd1p homologs            human counterparts: Myc-eIF4AIII Flag-NOM1 <br /> majority sequenced eukaryotes (Supplemental Fig. S5),        coimmunoprecipitate nuclear extracts human <br /> including human. Sgd1p eIF4G-like protein (Ponting        HEK293 cells (Fig. 5B), cross-linking required. <br /> 2000; Marintchev Wagner 2005) containing MIF4G              Third, direct interaction human eIF4AIII and <br /> and MA3 domains (Fig. 4A). Both MIF4G MA3                   NOM1 reconstituted using heterologously ex- <br /> domains eIF4G shown previously bind eIF4A            pressed purified proteins (Fig. 5D). Finally, identified <br /> (Morino et al. 2000; Oberer et al. 2005; Schutz et al. 2008).   endogenous human eIF4AIII 24 proteins specifi- <br /> It plausible MIF4G-containing Sgd1p        cally copurifying nuclear extract human HEK293 <br />  <br />  <br /> 1082       GENES &amp; DEVELOPMENT <br />                                                                                                   eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br />  <br />  <br /> Figure 4. The suppressor mutation sgd1(S340Y) synthetic mutation sgd1(N343Y) map known conserved 12-amino-acid motif <br /> that comprises largest interface eIF4G eIF4A. (A) Amino acid sequence alignment S. cerevisiae protein Sgd1p with <br /> putative Sgd1p-like proteins humans, M. musculus, D. melanogaster, C. elegans. Predicted MIF4G MA3 domains (Ponting <br /> 2000) indicated blue purple boxes, respectively. The conserved 12-amino-acid motif (KSLLNKLTLEMF yeast eIF4G and <br /> NSSLNKLSDSNI yeast Sgd1p), representing largest surface eIF4G interacting eIF4A (Schutz et al. 2008), indicated with <br /> a red box. Mutations Sgd1p result genetic interactions fal1(T/V) indicated triangles; residues is <br /> known form intermolecular hydrogen bonds eIF4A, based crystal structure eIF4G/eIF4A complex (Schutz et al. <br /> 2008). (B) Structure eIF4G/eIF4A complex (Schutz et al. 2008) showing locations residues homologous starting <br /> mutation Fal1p (eIF4A) suppressor synthetic mutations Sgd1p (eIF4G). The major eIF4A/eIF4G interaction interface is <br /> marked red oval. <br />  <br />  <br />  <br />  <br />                                                                                                      GENES &amp; DEVELOPMENT            1083 <br />  Alexandrov et al. <br />  <br />  <br />  <br />                                                                                               Figure 5. Direct physical interactions be- <br />                                                                                               tween S. cerevisiae Fal1p Sgd1p and <br />                                                                                               human eIF4AIII NOM1. (A) <br />                                                                                               Yeast Sgd1p Fal1p coimmunoprecipi- <br />                                                                                               tate. Western blot IgG precipitates <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                               extracts formaldehyde cross-linked yeast <br />                                                                                               cells expressing tagged untagged ver- <br />                                                                                               sions yeast Sgd1p Fal1p (Supplemen- <br />                                                                                               tal Fig. S6A), indicated. All proteins were <br />                                                                                               expressed promoters. A TAP- <br />                                                                                               tagged yeast protein (Trm8p) known <br />                                                                                               nuclear localization provided negative con- <br />                                                                                               trol. TAP-tagged proteins precipitated <br />                                                                                               using bead-immobilized rabbit IgG (the ZZ <br />                                                                                               domain Staphylococcus aureus protein A <br />                                                                                               present TAP tag binds IgG) (Rigaut <br />                                                                                               et al. 1999). The noncovalent complex of <br />                                                                                               horseradish peroxidase (HRP) poly- <br />                                                                                               clonal rabbit anti-peroxidase (PAP) used <br />                                                                                               Western blot detection TAP-tagged <br />                                                                                               proteins (Puig et al. 2001) (the ZZ domain <br />                                                                                               protein A TAP tag binds PAP); <br />                                                                                               detection c-myc epitope used HRP- <br />                                                                                               conjugated mouse monoclonal antibody 9E10. <br />                                                                                               (B) Human eIF4AIII NOM1 coimmuno- <br />                                                                                               precipitate. Western blot Flag-purified sam- <br />                                                                                               ples nuclear extract human HEK293 <br />                                                                                               cells expressing transiently transfected tagged <br /> and untagged versions human eIF4AIII NOM1, indicated (no cross-linking required). Flag-tagged human protein METTL1, with <br /> known nuclear localization provided negative control. (C) Endogenous human eIF4AIII identified mass spectrometry 24 <br /> protein bands specifically copurifying nuclear extract human HEK293 cells Flag-tagged NOM1 (silver staining) (Supplemental <br /> Fig. S6B). (D) Direct interaction heterologously expressed purified human GST-eIF4AIII NOM1 demonstrated GST pull- <br /> down (the gel Coomassie-stained). The arrow shows position full-length NOM1 copurified GST-eIF4AIII. GST-tagged purified <br /> yeast protein Trm82p provided negative control. (Lane 8) Additional bands NOM1 preparation represent N terminally truncated <br /> forms NOM1. I1 I2 denote inputs pull-downs shown lanes 1 2, lanes 3 4, respectively. <br />  <br />  <br />  <br /> cells Flag-tagged NOM1 (Fig. 5C; Supplemental Fig.                  levels, partially alleviates increases 35S pri- <br /> S6B) using mass spectrometry.                                            mary transcript (Fig. 6A, middle panels) and <br />                                                                          23S rRNA (Fig. 6A, middle panels). Third, <br /> Sgd1p/Fal1p NOM1/eIF4AIII form functional <br />                                                                          consistent Fal1p Sgd1p acting 18S <br /> complexes acting 18S rRNA biogenesis <br />                                                                          rRNA biogenesis, depletion Sgd1p results pre-rRNA <br /> in S. cerevisiae humans, respectively <br />                                                                          processing defects identical Fal1p: Slow <br /> Sgd1p NOM1 documented nucleolar proteins                      growth (Supplemental Fig. S7B,C) yeast strain (Sup- <br /> both yeast humans (Akhtar et al. 2000; Simmons                       plemental Fig. S7A) expressing Sgd1p GAL1 pro- <br /> et al. 2005); function Sgd1p yeast rRNA biogenesis              moter medium containing glucose accompanied <br /> has reported (Li et al. 2009). We demonstrated                 drastic decreases 18S (Fig. 6B, panel; Supple- <br /> the genetic physical interactions observed be-                    mental Fig. S7D) 20S (Fig. 6B, panel) rRNA <br /> tween Sgd1p Fal1p correlate collaborative                 levels, accumulation 23S rRNA (Fig. 6B, middle <br /> function 18S rRNA biogenesis. First, showed                   panels) 35S rRNA primary transcript <br /> the CS yeast strain fal1(T/V) displays pre-rRNA process-                 (Fig. 6B, middle panels). <br /> ing defects sites A0, A1, A2, resulting decrease                Furthermore, consistent evolutionary conserva- <br /> in 18S [the 18S/25S ratio fal1(T/V) strain 75%              tion function Sgd1p/Fal1p complex from <br /> that wild-type strain] (Fig. 6A, panel) 20S               yeast humans, siRNA-mediated knockdown eIF4AIII <br /> (Fig. 6A, panel) rRNA levels, accumu-                  NOM1 (Supplemental Fig. S11) resulted decreased <br /> lation 23S rRNA (Fig. 6A, middle panels)                   rate 18S rRNA formation HEK293T cells. When in <br /> and 35S rRNA primary transcript (Fig. 6A, middle                     vivo pulse-labeled cells 3H-methyl-methionine fol- <br /> bottom panels). Second, observed suppressor                  lowed chase cold methionine, observed lower <br /> mutation sgd1(S340Y) rescues rRNA biogen-                    amounts 18S rRNA (Fig. 7B,C, cf. lanes 1&#226;&#8364;&#8220;5 7&#226;&#8364;&#8220;11). <br /> esis defects fal1(T/V): It partially restores mature 18S              Also decreased 41S pre-rRNA 15 30 min of <br /> rRNA [the 18S/25S ratio suppressor strain fal1(T/V)               chase (Fig. 7B,C, cf. amounts 41S pre-rRNA lanes <br /> sgd1(S/Y) 85% wild-type strain] (Fig. 6A,              2,3 8,9) 21S pre-rRNA (Fig. 7B,C, small bottom <br /> top panel) 20S (Fig. 6A, panel) pre-rRNA                      panels, cf. amounts 21S pre-rRNA lanes 1&#226;&#8364;&#8220;5 and <br />  <br />  <br /> 1084       GENES &amp; DEVELOPMENT <br />                                                                                                eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br />  <br /> 7&#226;&#8364;&#8220;11); formation 41S 21S pre rRNA (Fig. 7A) <br /> requires cleavage site 1, creates 59 end the <br /> mature 18S rRNA. A decrease ratio newly <br /> synthesized (3H-labeled) mature small-subunit (18S) <br /> rRNA large-subunit (28S) rRNA (as observed <br /> after 3 h chase) seen NOM1 knockdown <br /> cells (Fig. 7C, cf. 18S/28S ratio lanes 6 12). A <br /> processing delay site 2 observed eIF4AIII <br /> knockdown cells, judged increased ratio of <br /> unprocessed 45S pre-rRNA 30S/32S pre-rRNA during <br /> the hour chase (Fig. 7B, cf. ratio lanes 4,5 and <br /> 10,11). To confirm specificity siRNAs used for <br /> knockdowns, cotransfected knockdown cells with <br />  <br />  <br />  <br />  <br />                                                                    Figure 7. siRNA-mediated knockdown eIF4AIII or <br />                                                                    NOM1 results 18S rRNA biogenesis defect human cells. <br />                                                                    (A) Schematic rRNA processing humans. (B) siRNA- <br /> Figure 6. The extragenic suppressor mutation sgd1(S340Y)           mediated knockdown eIF4AIII results decreased rate of <br /> alleviates 18S rRNA biogenesis defects CS fal1(T322V)       18S rRNA formation, determined vivo pulse-labeling of <br /> strain; SGD1 required rRNA processing sites     HEK293T cells [methyl-3H] -methionine followed chase <br /> (A0, A1, A2) FAL1. (A) The 18S rRNA biogenesis defect    cold methionine. (Middle panel) Processing 45S and <br /> the S. cerevisiae CS strain fal1(T322V) suppression     formation 41S, 30S/32S, 21S pre-RNAs delayed, <br /> sgd1(S340Y) visualized Northern blot using DNA oligo-       resulting decrease mature 18S rRNA, suggesting defects <br /> nucleotide probes L2 L9 (top panel), L1 (middle panel),    sites 1 2. (Right panel) Cotransfection plasmid ex- <br /> L3 (bottom panel); probes shown Supplemental Figure S2.     pressing siRNA-resistant version eIF4AIII partially rescues <br /> Total RNA prepared strains grown Figure 3A. (B)     rate formation 18S rRNA precursors, confirming <br /> SGD1 required 18S rRNA biogenesis sites A0, A1,      specificity. (C) siRNA-mediated knockdown NOM1 results in <br /> A2 (same sites Fal1p). Depletion PGAL1-expressed Sgd1p    decreased rate 18S rRNA formation, determined pulse- <br /> glucose-containing media 16&#194;&#176;C results decrease 18S      chase performed B. (Middle panel) Formation 41S 21S <br /> rRNA levels (top panel), increase 23S rRNA levels (middle    pre-RNAs delayed, resulting decrease mature 18S rRNA, <br /> panel), decrease 20S rRNA levels (bottom panel),          suggesting processing defect site 1. Consistently, decreased <br /> accumulation 35S rRNA primary transcript (middle        ratio mature small (18S) large (28S) subunit rRNA ob- <br /> bottom panels) yeast sgd1D strain (Northern blot        served 3 h chase (cf. lanes 6 12). (Right panel) Co- <br /> probed A; schematics strains shown Supple-     transfection plasmid expressing siRNA-resistant variant <br /> mental Fig. S7A, growth curves shown Supplemental       NOM1 partially rescues rate appearance 18S rRNA as <br /> Fig. S7B). Arrows left indicate changes levels    21S 41S precursors. <br /> rRNAs intermediates fal1(T/V) (A) sgd1D (B) strains <br /> compared wild type. <br />  <br />  <br />                                                                                                   GENES &amp; DEVELOPMENT            1085 <br />  Alexandrov et al. <br />  <br />  <br />  <br /> plasmids expressing siRNA-resistant eIF4AIII (peIF4AIIIr)       complementation fal1D phenotype by <br /> (Fig. 7B, right panel) NOM1 (pNOM1r) (Fig. 7C, right        human eIF4AIII requires intact Walker A B boxes, <br /> panel), resulted alleviation of ob-        identifying function eIF4AIII strictly <br /> served processing defects, vector (Fig. 7B,C,       depends ATP hydrolysis (at eIF4AIII is <br /> middle panels).                                                 expressed heterologously S. cerevisiae). The other <br />                                                                 known functions eIF4AIII&#226;&#8364;&#8221;EJC formation, NMD acti- <br />                                                                 vation, binding GPx1 SECIS element&#226;&#8364;&#8221;are not <br /> Discussion                                                      abolished mutations despite loss ATPase <br />                                                                 consequent helicase activity vitro (Shibuya et al. <br /> We identified eIF4G-like human protein NOM1            2006; Zhang Krainer 2007; Budiman et al. 2009). <br /> conserved interacting partner eIF4A-like protein         Finally, showed conservation function the <br /> eIF4AIII. This interaction yeast S. cerevisiae (Fal1p/   Fal1p/Sgd1p complex 18S rRNA biogenesis yeast <br /> Sgd1p) humans (eIF4AIII/NOM1) similar             humans demonstrating pre-rRNA processing de- <br /> established eIF4A/eIF4G interaction eIF4F. In yeast,     fects arising siRNA-mediated knockdown eIF4AIII <br /> each partner proteins essential required   NOM1 HEK293T cells. Interestingly, though <br /> for 18S rRNA biogenesis. In addition physical in-      knockdown eIF4AIII NOM1 similarly resulted <br /> teraction involvement steps 18S rRNA         decreased rates formation mature 18S rRNA <br /> biogenesis, established Fal1p Sgd1p ge-         similar processing defect site 1 (formation 41S <br /> netically functionally tied dominant compensa-        21S), delayed processing 45S pre-rRNA site 2 <br /> tory suppressor relationship: Whereas mutation T322V         (which specific eIF4AIII knockdown) reflect a <br /> an RNA-interacting residue Fal1p results cold sensi-      specific requirement eIF4AIII NOM1 site 2. <br /> tivity defect yeast 18S rRNA biogenesis,           Alternatively, factor substitute NOM1 in <br /> dominant allele-specific extragenic suppressor mutation         processing site 2 (but site 1) following NOM1 <br /> S340Y Sgd1p alleviates defects. The suppressor         down-regulation, or, possibly, degree NOM1 knock- <br /> mutation large protein Sgd1p (899 amino acids)           sufficient observe defect site 2 <br /> spontaneously appeared MIF4G domain (the major           processing. <br /> eIF4A-interacting domain eIF4G) 12-amino-acid          The versatility displayed eIF4AIII interacting <br /> conserved motif eIF4G NOM1 orthologous             different sets partners beginning to <br /> proteins represents &#226;&#8364;&#732;&#226;&#8364;&#732;the largest contiguous interaction   appreciated feature DEAD-box helicases. Mam- <br /> surface&#226;&#8364;&#8482;&#226;&#8364;&#8482; eIF4G eIF4A (Schutz et al. 2008). The     malian eIF4AIII interacts hand Magoh and <br /> two identified SGD1 mutations interact genetically         Y14, core protein deposits EJC newly <br /> with FAL1 (extragenic suppressor mutation S340Y             spliced mRNAs nucleoplasm, hand <br /> synthetic mutation N343Y) replace conserved Sgd1p               NOM1 nucleolus. Whether distinct as- <br /> amino acids previously shown yeast eIF4G          sociations dictated simply dual subcellular <br /> (S612 N615) form direct intermolecular          localization clear. Interestingly, eIF4AIII has <br /> hydrogen bonds eIF4A, based X-ray structure         reported reside nucleoli plants (Pendle et al. <br /> of yeast eIF4A/eIF4G complex (Schutz et al. 2008).          2005), human eIF4AIII mutants surprisingly <br />    Our results illuminate novel function       localize nucleolus nucleoplasm <br /> evolutionary origin eIF4AIII protein. Previously         (Shibuya et al. 2006). Other DEAD-box helicases have <br /> reported functions eIF4AIII EJC, NMD, neuronal        implicated cellular process are <br /> protein expression, selenocysteine regulation (Giorgi       Prp43 splicing rRNA processing (Lebaron et al. <br /> et al. 2007; Isken Maquat 2008; Budiman et al. 2009)        2005), eIF4A translation initiation regulation of <br /> were sufficient explain high degree conser-       SMAD degradation D. melanogaster (Li Li 2006), <br /> vation eIF4AIII family (60% identity             Dbp5 mRNA export translation termi- <br /> human eIF4AIII yeast Fal1p entire protein          nation (Gross et al. 2007). The lack requirement for <br /> length) Walker A B boxes proteins.         ATP hydrolysis release eIF4AIII (the EJC) its <br /> This (1) EJC identified yeast        mRNA substrates ascribed power of <br /> (Tange et al. 2004), (2) function eIF4AIII NMD     ribosome progressing mRNA trans- <br /> not conserved D. melanogaster C. elegans (Palacios       lation; rRNA biogenesis, ATP hydrolysis presumably <br /> et al. 2004; Longman et al. 2007), (3) components     leads release eIF4AIII rRNA substrate, <br /> of EJC conserved metazoans (Tange et al.        allowing progression step processing <br /> 2004).                                                          pathway. It interesting learn RNA <br />    We uncovered conserved ability human D.           helicases differential requirements ATP <br /> melanogaster eIF4AIII function 18S rRNA biogen-           hydrolysis performing multiple cellular roles. <br /> esis heterologous expression proteins          Our results consistent idea eIF4G <br /> S. cerevisiae strain lacking essential yeast DEAD-box       NOM1 evolved common ancestor protein <br /> helicase Fal1p. Restoration strain viability        (Marintchev Wagner 2005). Whereas eIF4G functions <br /> 18S rRNA biogenesis specific,           eIF4AI/II translation initiation eukaryotes, the <br /> closely related human eIF4AI eIF4AII yeast Tif1         human eIF4G-like protein NOM1 evolved cooperate <br /> proteins did complement fal1D. We showed            similar protein, eIF4AIII, completely <br />  <br />  <br /> 1086       GENES &amp; DEVELOPMENT <br />                                                                                          eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br />  <br /> different process: 18S rRNA biogenesis. It possible    DEAD-box helicases eIF4A family (Tanner <br /> additional partners DEAD-box helicases          Linder 2001). In crystal structures EJC <br /> exhibit interactions similar eIF4AIII/NOM1          core (Andersen et al. 2006; Bono et al. 2006) VASA <br /> Fal1p/Sgd1p pairs identified future.            complexed RNA (Sengoku et al. 2006), threo- <br />    In retrospect, initial lack success identifying   forms hydrogen bonds RNA backbone phos- <br /> Sgd1p Fal1p-interacting protein using TAP purifica-       phates. The T546A mutation VASA (corresponding to <br /> tion Fal1p surprising. Instead, highlights    T322 Fal1p) abolishes vitro ATPase helicase <br /> similarity Fal1p/Sgd1p interaction relatively    activities VASA, results 50% decrease cross- <br /> weak, functionally crucial, interaction eIF4A      linking efficiency U10 oligomer (Sengoku et al. <br /> eIF4G eIF4F. Specifically, eIF4A readily         2006). The eIF4A/eIF4G interaction model suggests that <br /> dissociated mammalian eIF4G ion exchange               suppressing (S340) synthetically interacting (N343) <br /> columns (Ray et al. 1985; Etchison Milburn 1987)       residues Sgd1p contact Fal1p opposite the <br /> can exchange free eIF4A (Yoder-Hill et al. 1993).         putative RNA-binding domain (Fig. 4B). One possibility is <br /> Binding eIF4A eIF4G weaker eIF4E         mutations Sgd1p (S340Y N343Y) <br /> to eIF4G (Gingras et al. 1999). Purified eIF4F complexes       acts direct interaction Fal1p Sgd1p/ <br /> from plant (Lax et al. 1985), D. melanogaster (Sierra      Fal1p interface induce slight conformational change in <br /> Zapata 1994), yeast (Goyer et al. 1989) lack eIF4A,    Fal1p(T322V) increases function lower tempera- <br /> presumably eIF4A dissociates isolation.         ture. Mechanistically, suppression conforma- <br /> Thus, switched identifying Sgd1p/Fal1p-            tional change RNA-binding region Fal1p induced <br /> associated components using vivo formaldehyde cross-        Sgd1p binding opposite Fal1p be <br /> linking followed affinity selection using TAP-tagged        consistent model (Oberer et al. 2005) eIF4G <br /> Sgd1p (in yeast) Flag-tagged NOM1 (without cross-          serving soft clamp stabilize relative orientation <br /> linking, human cells). Mass-spectrometric analysis       helicase domains favoring closed <br /> proteins copurifying NOM1-Flag human cell            &#226;&#8364;&#732;&#226;&#8364;&#732;active&#226;&#8364;&#8482;&#226;&#8364;&#8482; conformation eIF4A. Such mechanism would <br /> nuclear extract identified 24 proteins, 18        consistent numerous reports stimula- <br /> known ribosomal proteins (five small 13 large sub-         tion eIF4A helicase activity eIF4G binding <br /> units) (Supplemental Fig. S6B). The presence           (Marintchev Wagner 2004). Alternatively, possible <br /> r proteins NOM1 preparations indicate associ-        genetically interacting residues Sgd1p (S340Y or <br /> ation NOM1 preribosomes stages preceding        N343Y) make direct contact CS residue Fal1p <br /> cleavage 35S pre-rRNA transcript (consistent       (T322), gross structural rearrangement would <br /> the reported nucleolar localization NOM1 [Simmons           required. Detailed biochemical characterization the <br /> et al. 2005] yeast ortholog, Sgd1p [Akhtar et al.      context members Sgd1p/Fal1p complex <br /> 2000]), small large ribosomal subunits    shed light mechanism fal1(T322V) cold <br /> at later nuclear steps rRNA processing. Remarkably,         sensitivity extragenic suppression. <br /> endogenous human eIF4AIII 24 proteins              Human NOM1 originally identified nucleolar <br /> identified mass spectrometry copurifying NOM1-      eIF4A-binding protein encoded chromosome <br /> Flag (Fig. 5C; Supplemental Fig. S6B). We currently        7q36 breakpoint region majority pediatric <br /> investigating exact composition eIF4AIII/NOM1       acute myeloid leukemia (AML) patients carrying the <br /> complex.                                                       t(7;12) translocation (Simmons et al. 2005; von Bergh <br />    The apparent difference interaction strength      et al. 2006), associated particularly poor <br /> human NOM1 eIF4AIII relative yeast Sgd1p          prognosis. The genomic segment (7q35-7q36) been <br /> Fal1 similar eIF4G eIF4A       implicated critical region del(7q) leukemias (Dohner <br /> organisms: Human eIF4G eIF4A interact            et al. 1998). Despite proximity (&lt;1 kb) NOM1 <br /> strongly yeast counterparts (Gingras et al.         gene translocation breakpoint t(7;12)(q36;p13) <br /> 1999). This difference explained lack         (Simmons et al. 2005), NOM1 mRNA levels not <br /> of second C-terminal eIF4A-binding region (the MA3         affected (von Bergh et al. 2006). Whether NOM1&#226;&#8364;&#8482;s <br /> domain) yeast eIF4G, yeast Sgd1p             proximity breakpoint contributes AML will <br /> human NOM1 similar structures; possesses             require study. <br /> one MIF4G MA3 domain (Fig. 4A; Ponting 2000; <br /> Marintchev Wagner 2005). Further biochemical stud-         Materials methods <br /> ies explain detection Sgd1p/Fal1p inter- <br /> action yeast requires formaldehyde cross-linking,           Strains plasmids <br /> whereas robust copurification human eIF4AIII with <br /> NOM1-Flag nuclear extracts human cells occurs          Strains plasmids described Supplemental Material. <br /> without cross-linking. <br />    The CS defect fal1(T322V) strain extra-      Yeast growth <br /> genic suppression sgd1(S340Y), synthetic     Yeast strains shown Figure 2B grown synthetic <br /> growth defect fal1(T322V) sgd1(N343Y) double            medium lacking uracil containing 2% galactose 2% <br /> mutant, require investigation. The mutated thre-       rafinose (SGR-ura) shifted synthetic medium lacking <br /> onine residue T322 Fal1p conserved numerous           uracil containing 2% dextrose 2% rafinose (SDR-ura). <br />  <br />  <br />                                                                                             GENES &amp; DEVELOPMENT            1087 <br />  Alexandrov et al. <br />  <br />  <br /> Strains Figure 2C Supplemental Figure S7, B C,        proteins siRNA depletion affected rRNA biogenesis in <br /> grown SGR-his SGR-ura media, respectively,            HeLa cells; human eIF4AIII listed proteins <br /> shifted SGR-his/SDR-his SGR-ura/SDR-ura media, re-             (Wild et al. 2010). <br /> spectively. <br />                                                                       References <br /> Construction isolation TS CS mutants <br />                                                                       Akhtar N, Pahlman AK, Larsson K, Corbett AH, Adler L. 2000. <br /> Mutations Fal1p specific positions (149, 152, 304, 322)        SGD1 encodes essential nuclear protein Saccharomy- <br /> were generated using PCR QuickChange-based methods                   ces cerevisiae affects expression GPD1 gene for <br /> using DNA oligonucleotides specific degenerate (NNN)             glycerol 3-phosphate dehydrogenase. FEBS Lett 483: 87&#226;&#8364;&#8220;92. <br /> triplets. Mutations Fal1p random positions generated       Andersen CB, Ballut L, Johansen JS, Chamieh H, Nielsen KH, <br /> using error-prone PCR using Taq polymerase elevated Mg2+.            Oliveira CL, Pedersen JS, Seraphin B, Le Hir H, Andersen GR. <br /> Libraries resulting fal1*-HIS5 cassettes (Supplemental Fig.           2006. Structure exon junction core complex with <br /> S9B) linearized transformed yeast, FAL1(WT)-           trapped DEAD-box ATPase bound RNA. Science 313: <br /> containing plasmid counterselected, growth            1968&#226;&#8364;&#8220;1972. <br /> resulting strains assessed different temperatures. The         Ballut L, Marchadier B, Baguet A, Tomasetto C, Seraphin B, Le <br /> phenotypes isolated mutants [fal1(CS), fal1(CS2), fal1(TS),       Hir H. 2005. The exon junction core complex locked onto <br /> fal1(TS2)] confirmed tetrad analysis (Supplemental Figs.         RNA inhibition eIF4AIII ATPase activity. Nat Struct <br /> S3D, S9E) complementation FAL1(WT) (Supplemental                Mol Biol 12: 861&#226;&#8364;&#8220;869. <br /> Figs. S3E, S9C).                                                      Bleichert F, Baserga SJ. 2007. The long unwinding road RNA <br />                                                                          helicases. Mol Cell 27: 339&#226;&#8364;&#8220;352. <br />                                                                       Bono F, Ebert J, Lorentzen E, Conti E. 2006. The crystal <br /> Northern blot <br />                                                                          structure exon junction complex reveals it <br /> RNA separated formaldehyde agarose (1%) transferred               maintains stable grip mRNA. Cell 126: 713&#226;&#8364;&#8220;725. <br /> to Zeta-Probe membrane (Bio-Rad), cross-linked using UV254            Budiman ME, Bubenik JL, Miniard AC, Middleton LM, Gerber <br /> (Stratalinker 2400), visualized synthetic 59-end 32P-           CA, Cash A, Driscoll DM. 2009. Eukaryotic initiation factor <br /> labeled DNA oligonucleotide probes (Supplemental Fig. S2).               4a3 selenium-regulated RNA-binding protein selec- <br />                                                                          tively inhibits selenocysteine incorporation. Mol Cell 35: <br />                                                                          479&#226;&#8364;&#8220;489. <br /> Protein purification <br />                                                                       Corpet F. 1988. Multiple sequence alignment hierarchical <br /> Protein purification described Supplemental Material.          clustering. Nucleic Acids Res 16: 10881&#226;&#8364;&#8220;10890. <br />                                                                       Dohner K, Brown J, Hehmann U, Hetzel C, Stewart J, Lowther <br />                                                                          G, Scholl C, Frohling S, Cuneo A, Tsui LC, et al. 1998. <br /> Western blot <br />                                                                          Molecular cytogenetic characterization critical region in <br /> Western Lightning Plus-ECL Enhanced Chemiluminescence                    bands 7q35-q36 commonly deleted malignant myeloid <br /> Substrate PerkinElmer SuperSignal West Pico/Femto               disorders. Blood 92: 4031&#226;&#8364;&#8220;4035. <br /> Maximum Sensitivity Substrates Pierce used              Etchison D, Milburn S. 1987. Separation protein synthesis <br /> detection.                                                               initiation factor eIF4A p220-associated cap binding <br />                                                                          complex activity. Mol Cell Biochem 76: 15&#226;&#8364;&#8220;25. <br />                                                                       Fairman ME, Maroney PA, Wang W, Bowers HA, Gollnick P, <br /> Mass spectrometry <br />                                                                          Nilsen TW, Jankowsky E. 2004. Protein displacement <br /> Flag-eluted preparations NOM1 separated 4%&#226;&#8364;&#8220;20%                DExH/D &#226;&#8364;&#732;RNA helicases&#226;&#8364;&#8482; duplex unwinding. <br /> SDS PAGE stained SYPRO Ruby, gel slices                Science 304: 730&#226;&#8364;&#8220;734. <br /> analyzed ESI-LC-MS/MS Columbia University Protein               Gatfield D, Unterholzner L, Ciccarelli FD, Bork P, Izaurralde E. <br /> Core Facility.                                                           2003. Nonsense-mediated mRNA decay Drosophila: at <br />                                                                          intersection yeast mammalian pathways. <br />                                                                          EMBO J 22: 3960&#226;&#8364;&#8220;3970. <br /> Software used <br />                                                                       Gingras AC, Raught B, Sonenberg N. 1999. eIF4 initiation <br /> Sequences aligned using Multalin (Corpet 1988),            factors: effectors mRNA recruitment ribosomes and <br /> shaded Boxshade 3.21.                                               regulators translation. Annu Rev Biochem 68: 913&#226;&#8364;&#8220;963. <br />                                                                       Giorgi C, Yeo GW, Stone ME, Katz DB, Burge C, Turrigiano G, <br /> Acknowledgments                                                          Moore MJ. 2007. The EJC factor eIF4AIII modulates synap- <br />                                                                          tic strength neuronal protein expression. Cell 130: 179&#226;&#8364;&#8220; <br /> We thank Shobha Vasudevan, Nikolay Kolev, Mei-Di Shu, Sumit              191. <br /> Borah, Abiodun Kukoyi help, valuable discussions,         Gonzalez CI, Bhattacharya A, Wang W, Peltz SW. 2001. Non- <br /> sharing reagents. We thank Adrian Krainer sharing               sense-mediated mRNA decay Saccharomyces cerevisiae. <br /> antibody human eIF4AIII, Christine Guthrie critically        Gene 274: 15&#226;&#8364;&#8220;25. <br /> reading manuscript, Angie Miccinello editorial help,      Goyer C, Altmann M, Trachsel H, Sonenberg N. 1989. Identi- <br /> the rest Steitz laboratory stimulating discussions. This      fication characterization cap-binding proteins from <br /> work supported grant GM026154 NIH. J.A.S              yeast. J Biol Chem 264: 7603&#226;&#8364;&#8220;7610. <br /> an investigator Howard Hughes Medical Institute.               Gross T, Siepmann A, Sturm D, Windgassen M, Scarcelli JJ, <br />                                                                          Seedorf M, Cole CN, Krebber H. 2007. The DEAD-box RNA <br /> Note added proof                                                      helicase Dbp5 functions translation termination. Science <br />                                                                          315: 646&#226;&#8364;&#8220;649. <br /> While manuscript review, laboroatory Ulrike        Hinnebusch AG. 2006. eIF3: versatile scaffold translation <br /> Kutay reported high-throughput identification 153 human             initiation complexes. Trends Biochem Sci 31: 553&#226;&#8364;&#8220;562. <br />  <br />  <br /> 1088       GENES &amp; DEVELOPMENT <br />                                                                                                  eIF4AIII eIF4G-like partner, NOM1 <br />  <br />  <br /> Isken O, Maquat LE. 2008. The multiple lives NMD factors:          Oberer M, Marintchev A, Wagner G. 2005. Structural basis for <br />    balancing roles gene genome regulation. Nat Rev                enhancement eIF4A helicase activity eIF4G. Genes <br />    Genet 9: 699&#226;&#8364;&#8220;712.                                                     Dev 19: 2212&#226;&#8364;&#8220;2223. <br /> Kressler D, la Cruz J, Rojo M, Linder P. 1997. Fal1p         Palacios IM, Gatfield D, St Johnston D, Izaurralde E. 2004. An <br />    essential DEAD-box protein involved 40S-ribosomal-sub-             eIF4AIII-containing complex required mRNA localiza- <br />    <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> biogenesis Saccharomyces cerevisiae. Mol Cell Biol            tion nonsense-mediated mRNA decay. Nature 427: 753&#226;&#8364;&#8220; <br />    17: 7283&#226;&#8364;&#8220;7294.                                                        757. <br /> Lax S, Fritz W, Browning K, Ravel J. 1985. Isolation              Pendle AF, Clark GP, Boon R, Lewandowska D, Lam YW, <br />    characterization factors wheat germ exhibit              Andersen J, Mann M, Lamond AI, Brown JW, Shaw PJ. <br />    eukaryotic initiation factor 4B activity overcome                 2005. Proteomic analysis Arabidopsis nucleolus sug- <br />    7-methylguanosine 59-triphosphate inhibition polypeptide           gests novel nucleolar functions. Mol Biol Cell 16: 260&#226;&#8364;&#8220;269. <br />    synthesis. Proc Natl Acad Sci 82: 330&#226;&#8364;&#8220;333.                         Ponting CP. 2000. Novel eIF4G domain homologues linking <br /> Lebaron S, Froment C, Fromont-Racine M, Rain JC, Monsarrat               mRNA translation nonsense-mediated mRNA decay. <br />    B, Caizergues-Ferrer M, Henry Y. 2005. The splicing ATPase            Trends Biochem Sci 25: 423&#226;&#8364;&#8220;426. <br />    prp43p component multiple preribosomal particles.          Puig O, Caspary F, Rigaut G, Rutz B, Bouveret E, Bragado- <br />    Mol Cell Biol 25: 9269&#226;&#8364;&#8220;9282.                                          Nilsson E, Wilm M, Seraphin B. 2001. The tandem affinity <br /> Le Hir H, Izaurralde E, Maquat LE, Moore MJ. 2000a. The                  purification (TAP) method: general procedure protein <br />    spliceosome deposits multiple proteins 20&#226;&#8364;&#8220;24 nucleotides              complex purification. Methods 24: 218&#226;&#8364;&#8220;229. <br />    upstream mRNA exon-exon junctions. EMBO J 19: 6860&#226;&#8364;&#8220;             Ray BK, Lawson TG, Kramer JC, Cladaras MH, Grifo JA, <br />    6869.                                                                 Abramson RD, Merrick WC, Thach RE. 1985. ATP-depen- <br /> Le Hir H, Moore MJ, Maquat LE. 2000b. Pre-mRNA splicing                  dent unwinding messenger RNA structure eukaryotic <br />    alters mRNP composition: evidence stable association           initiation factors. J Biol Chem 260: 7651&#226;&#8364;&#8220;7658. <br />    proteins exon&#226;&#8364;&#8220;exon junctions. Genes Dev 14: 1098&#226;&#8364;&#8220;1108.          Rigaut G, Shevchenko A, Rutz B, Wilm M, Mann M, Se&#194;&#180;raphin B. <br /> Le Hir H, Gatfield D, Izaurralde E, Moore MJ. 2001. The exon&#226;&#8364;&#8220;            1999. A generic protein purification method protein <br />    exon junction complex provides binding platform                 complex characterization proteome exploration. Nat <br />    factors involved mRNA export nonsense-mediated                 Biotechnol 17: 1030&#226;&#8364;&#8220;1032. <br />    mRNA decay. EMBO J 20: 4987&#226;&#8364;&#8220;4997.                                  Rocak S, Linder P. 2004. DEAD-box proteins: driving forces <br /> Li J, Li WX. 2006. A novel function Drosophila eIF4A               RNA metabolism. Nat Rev Mol Cell Biol 5: 232&#226;&#8364;&#8220; <br />    negative regulator Dpp/BMP signalling mediates              241. <br />    SMAD degradation. Nat Cell Biol 8: 1407&#226;&#8364;&#8220;1414.                      Rozen F, Edery I, Meerovitch K, Dever TE, Merrick WC, <br /> Li Q, Imataka H, Morino S, Rogers GW Jr, Richter-Cook NJ,                Sonenberg N. 1990. Bidirectional RNA helicase activity of <br />    Merrick WC, Sonenberg N. 1999. Eukaryotic translation                 eucaryotic translation initiation factors 4A 4F. Mol Cell <br />    initiation factor 4AIII (eIF4AIII) functionally distinct      Biol 10: 1134&#226;&#8364;&#8220;1144. <br />    eIF4AI eIF4AII. Mol Cell Biol 19: 7336&#226;&#8364;&#8220;7346.                   Schutz P, Bumann M, Oberholzer AE, Bieniossek C, Trachsel H, <br /> Li Z, Lee I, Moradi E, Hung NJ, Johnson AW, Marcotte EM. 2009.           Altmann M, Baumann U. 2008. Crystal structure the <br />    Rational extension ribosome biogenesis pathway <br /> </body></html>