<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Nonlinear Fitness Consequences Variation Expression Level <br />           Eukaryotic Gene <br />           Joshua S. Rest,*,1 Christopher M. Morales,1 John B. Waldron,1 Dana A. Opulente,1 Julius Fisher,1 <br />           Seungjae Moon,z,1 Kevin Bullaughey,2 Lucas B. Carey,3 Demitri Dedousis1 <br />           1 <br />            Department Ecology Evolution, Stony Brook University <br />           2 <br />            Department Ecology Evolution, University Chicago <br />           3 <br />            Department Computer Science Applied Mathematics, Weizmann Institute Science, Rehovot, Israel <br />           z <br />             Present address: Weill Cornell Graduate School Medical Science, New York, NY 10065 <br />           *Corresponding author: E-mail: joshua.rest@stonybrook.edu. <br />           Associate editor: Matthew Hahn <br />  <br />           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />           Levels gene expression considerable variation eukaryotes, fine-scale maps fitness <br />           consequences variation controlled genetic backgrounds environments. <span id='am-9' about='protege:TO' typeof='owl:Thing'>To</span> address this, assayed fitness many <br />           levels up- down-regulated expression single essential gene, LCB2, involved sphingolipid synthesis budding yeast <br />           Saccharomyces cerevisiae. Reduced LCB2 expression rapidly decreases cellular fitness, increased expression little effect. The <br />           wild-type expression level perched edge nonlinear fitness cliff. LCB2 upregulated cells exposed <br />           osmotic stress; consistent this, entire fitness curve shifted upward higher expression osmotic stress, <br />           illustrating selective force gene regulation. Expression levels LCB2 lower wild yeast strains the <br />           experimental lab strain, suggesting higher levels lab strain idiosyncratic. Reports indicate effect sizes of <br />           alleles contributing variation complex phenotypes differ environments genetic backgrounds; results suggest <br />           differences explained simple shifts position nonlinear fitness curves. <br />           Key words: sphingolipids, gene regulatory evolution, Saccharomyces cerevisiae, fitness function, gene expression. <br />  <br />                                                                                                 The shape expression&#226;&#8364;&#8220;fitness curves lac genes been <br />           Introduction                                                                          studied bacteria, resulting curves nonlinear and <br />           Some mutations large effects fitness organisms,                        change different environments (Dekel Alon 2005; <br />           little effect fitness (Fisher 1928;                      Perfeito et al. 2011). Expression&#226;&#8364;&#8220;fitness curves eukaryotic <br />           Wright 1929; Haldane 1930; Orr 1991; Deutschbauer et al.                              genes similarly yield insight distribution the <br />           2005). One explanation impact muta-                           consequences mutations. However, published studies <br />           tions fitness (healthy growth reproduction)                           characterized fitness function gene expression at <br /> Article <br />  <br />  <br />  <br />  <br />           linearly affected <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>mutation</span> changes                           high resolution eukaryotes (Bayer 2010), would <br />           gene activity expression. Although linear relationship                        permit powerful test hypothesis genetic <br />           assumed, demonstrated allelic differ-                          variation nonlinear effect fitness. <br />           ences nonlinear effect fitness (Wright 1934;                               As step goal, chose quantitatively <br />           Hartl et al. 1985; Dykhuizen et al. 1987; Birchler et al. 2001;                       measure expression&#226;&#8364;&#8220;fitness curve, Saccharomyces cer- <br />           Veitia 2002; Papp et al. 2003). For example, increase                       evisiae, essential conserved metabolic gene, LCB2, <br />           decrease expression gene create stoi-                          encodes key enzyme sphingolipid synthesis <br />           chiometric imbalance proteins complex,                            pathway (fig. 1). This pathway synthesizes manages a <br />           type change decrease fitness (Birchler et al. 2001;                     spectrum functionally diverse products metabolites <br />           Veitia 2002; Papp et al. 2003). Another nonlinear fitness func-                       key structural signaling roles mediate responses <br />           tion describes diminishing returns increases ef-                        changing physiologic environmental cues (Hannun <br />           fective activity metabolic enzymes alleles vary                        Obeid 2008). Sphingolipids essential components <br />           protein sequence expression level enzyme                             plasma membrane, involved processes <br />           (Wright 1934; Hartl et al. 1985; Dykhuizen et al. 1987). Gene                         including signaling, proteolysis, cytoskeletal changes, <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br />           expression <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> shows extensive natural vari-                          uptake, regulation cell growth, stress response (Nagiec <br />           ation, comparative analyses suggest                             et al. 1994; Cowart Obeid 2007). Lcb2 binds stoi- <br />           variation little physiological effect (Khaitovich et al. 2005;                    chiometric complex Lcb1 form serine palmitoyltra- <br />           Yanai Hunter 2009). The shape expression&#226;&#8364;&#8220;fitness                           nsferase (SPT), catalyzes committed step <br />           curve gene tool directly determining                          sphingolipid synthesis (fig. 1) (Gable et al. 2002). <br />           causes consequences variation species.                           Misregulation sphingolipids shown cause <br />  <br />           &#195;&#376; The Author(s) 2012. Published Oxford University Press behalf Society Molecular Biology Evolution. <br />           This Open Access article distributed terms Creative Commons Attribution Non-Commercial License <br />           (http://creativecommons.org/licenses/by-nc/3.0/), permits unrestricted non-commercial use, distribution, reproduction <br />           medium, provided original work properly cited.                                                                      Open Access <br />           448                            Mol. Biol. Evol. 30(2):448&#226;&#8364;&#8220;456 doi:10.1093/molbev/mss248 Advance Access publication October 27, 2012 <br />  Fitness Expression Level Variation . doi:10.1093/molbev/mss248                                                                             MBE <br />                         serine + palmitoyl-CoA <br />                                                                                                    LCB2 Expression Level <br />                          SPT                Orm1/2                                                     &#226;&#710;&#8217;1.8 fold underexpressed <br />                     Lcb1, Lcb2* <br />                                                           C18:1                                        wild-type expression <br />                              Tsc3 <br />                        dihydrosphingosine <br />                                                           C18 <br />                               Sur2 <br />  <br />                        phytosphingosine                   C20 <br />                     phyto-       Lag1 <br />                                  Lac1 <br />                     ceramide     Lip1 <br />                                                           C22:1 <br />                       dihydroceramide <br />  <br />                                            ceramide C20:1 <br />  <br />                                                                   0.0002 0.0005 0.001 0.002         0.005 0.01 0.02        0.05    0.1 <br />                                                                             pmole / nmole cellular phosphate <br />  <br /> FIG. 1. The cellular sphingolipid synthesis pathway corresponding responses changes LCB2 expression. On left, sphingolipid biosynthesis <br /> pathway Saccharomyces cerevisiae shown. Lcb2, product titrated gene LCB2, catalyzes committed step pathway. <br /> Sphingolipid compounds measured response changing levels LCB2, sphingolipids displayed significant response shown to <br /> the right corresponding step pathway. All comparisons shown sphingolipid levels cells wild-type LCB2 expression and <br /> with reduced ( 2.7-fold) expression significant according tailed t test P value 0.05. Bars indicate medians standard errors <br /> of biological replicates. Means, standard errors, P values measured sphingolipid compounds LCB2 levels provided in <br /> supplementary table S1, Supplementary Material online. <br />  <br /> disruptions key structural signaling roles                   phenotypes single gene (Hughes et al. 2000; Peng <br /> have severe consequences cell (Hannun Obeid                          et al. 2003; Mnaimneh et al. 2004). We mimicked graded allelic <br /> 2008). The role LCB2 sphingolipid synthesis makes                    variation gene expression level using strain TetO7&#226;&#8364;&#8220;LCB2, <br /> intriguing gene studying effects variation gene                    native promoter gene LCB2 been <br /> expression cellular fitness. Importantly, sphingolipids                   replaced doxycycline-regulated promoter (Tet-Off <br /> their diverse downstream products measured,                           system) (Mnaimneh et al. 2004). We titratable <br /> thereby additional quantitative phenotype                        promoter excellent tool exploring expres- <br /> gene expression fitness. In addition, LCB2 good choice                  sion&#226;&#8364;&#8220;fitness curves affected changing <br /> for studying expression&#226;&#8364;&#8220;fitness curves, 1) exhibits                   environments genetic backgrounds. <br /> stoichiometric binding, dosage balance affect <br /> fitness (Gable et al. 2002), 2) shows variation expression                 Results <br /> patterns ecologically divergent yeast strains (Rossouw <br /> et al. 2009; Eng et al. 2010), shed light shape                 Fitness Sphingolipid Flux Consequences of <br /> of fitness curve, 3) known affect cell                  Variation LCB2 Expression <br /> growth, resulting severe growth defect repressed                       We determined dose-response curve doxy- <br /> (Mnaimneh et al. 2004). For experimental reasons, LCB2                     cycline concentration LCB2 expression TetO7&#226;&#8364;&#8220;LCB2 <br /> appropriate candidate cell cycle regulated                     (fig. 2A). Using expression&#226;&#8364;&#8220;titration function, mea- <br /> (Spellman et al. 1998), periodically expressed                        sured fitness 29 expression levels head-to-head com- <br /> metabolic bursts (Tu et al. 2005), dynamically expressed                     petition mixed culture fluorescent reference strain <br /> over course fermentative growth (Rossouw et al. 2009),                    (fig. 2C). We defined fitness rate cell growth and <br /> and does cell-to-cell variation expression                           division batch culture relative wild-type (parental <br /> (Newman et al. 2006). Therefore, expect pheno-                       strain R1158). <br /> typic consequences changes expression                           As expression reduced wild-type level in <br /> the level expression, changes temporal dy-                     standard media, fitness rapidly decreased, dropping 19% in <br /> namics expression.                                                            conjunction 2-fold (50%) drop LCB2 expression <br />    One challenges assessing fitness consequences                   (solid red curve fig. 2C). This represents dramatic fitness <br /> of variation level gene expression identifying genetic                cost&#226;&#8364;&#8221;such alleles quickly removed wild <br /> system gene expression experimentally                            populations natural selection. Our high-resolution data <br /> manipulated fitness consequences measured. One                           reveal wild-type expression level located directly <br /> possible approach mutate gene&#226;&#8364;&#8482;s promoter,                          edge fitness cliff, small decrease in <br /> but difficult achieve promoter muta-                     expression large fitness cost. In contrast, overexpression <br /> tions effect lead coarse-grained decrease                 LCB2 level 7.3-fold greater wild-type did not <br /> in expression (Patwardhan et al. 2009). To solve problem,                   markedly change fitness. The fact slope fitness <br /> we used chemically titratable promoter does                        curve steep response decreases LCB2 expression but <br /> not directly affect fitness create range expression                      flat response increases expression indicates that <br />  <br />                                                                                                                                                        449 <br />  Rest et al. . doi:10.1093/molbev/mss248                                                                                                                                                                                                  MBE <br />       A                                                                                                                                            B <br />       Fold Change LCB2 Expression   16                                                                                                                                               16 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        Fold Change LCB2 Expression <br />                                         4                                                                                                                                                4 <br />  <br />  <br />  <br />  <br />                                         1                                                                                                                                                1 <br />  <br />  <br />  <br />  <br />                                        &#226;&#710;&#8217;4                                                                                                                                               &#226;&#710;&#8217;4 <br />  <br />  <br />  <br />  <br />                                                        -6                                                              -4 <br />                                             0   1x10                                                            1x10        0.01   1                                                         0                 0.01                0.1   1 <br />                                                                                                          &#194;&#181;g/mL Doxycycline                                                                                        &#194;&#181;g/mL Doxycycline <br />                                                                                                          1.05 <br />  <br />  <br />  <br />  <br />                                                        C <br />  <br />                                                                                                                                     std.    o.s. <br />                                                                                                          1.00 <br />                                                         Fitness Relative Wild&#226;&#8364;&#8220;type (Media Specific) <br />                                                                                                          0.95 <br />                                                                                                          0.90 <br />                                                                                                          0.85 <br />                                                                                                          0.80 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                      Standard Media (std.) <br />                                                                                                                                                                                                      Osmotic Stress Media (o.s.) <br />  <br />  <br />                                                                                                                  &#226;&#710;&#8217;4           &#226;&#710;&#8217;2       1           2                                             4            8             16 <br />                                                                                                                                    Fold Change LCB2 Expression <br />  <br /> FIG. 2. Titrated levels gene expression reveal fitness cliff gene LCB2 Saccharomyces cerevisiae strain TetO7&#226;&#8364;&#8220;LCB2. We measured the <br /> relationship doxycycline concentration LCB2 expression level (A) standard media (B) osmotic stress media. Expression levels were <br /> determined qPCR, curve fitted five-parameter log-logistic regression. Box whisker plots (gray bars dashed lines) indicate <br /> error estimates qPCR measurements biological replicates, capped solid lines indicate error estimates based regression <br /> model. (C) We assessed cellular fitness relative levels LCB2 expression TetO7&#226;&#8364;&#8220;LCB2 grown environments. The sharp fitness cost for <br /> decreases LCB2 expression shows wild-type level expression edge fitness cliff. The shape fitness function osmotic <br /> stress media (dashed line) similar shape standard media (solid line), shifted higher levels LCB2 expression former. <br /> Gene expression relative level expression parental wild-type (R1158) standard media; arrows indicate wild-type expression levels in <br /> each environment. Fitness normalized wild-type equivalent environment. Semitransparent boxes indicate error boundaries fitness <br /> (standard error three, average five, biological replicates) expression estimates (regression model-based error). <br />  <br />  <br /> changes LCB2 expression highly nonlinear fitness                                                                                           quantitated earliest sphingolipid metabolites <br /> consequences.                                                                                                                                      novo synthesis pathway, including dihydrosphingo- <br />    We expected changes LCB2 gene expression                                                                                          sine 20 species ceramides, intermediates in <br /> affect sphingolipid levels key position                                                                                      formation complex sphingolipids. In cells 7.3-fold <br /> head sphingolipid synthesis pathway (fig. 1). We                                                                                            increase LCB2 expression, did observe changes in <br />  <br /> 450 <br />  Fitness Expression Level Variation . doi:10.1093/molbev/mss248                                                        MBE <br /> any sphingolipids measured (supplementary table S1, Supple-          accordingly predicts expected levels standing allelic <br /> mentary Material online), consistent lack ob-         variation populations (Eanes 2011). Because low-expression <br /> vious fitness consequence (fig. 2C). However, decrease        alleles pronounced negative fitness consequences <br /> fitness observed titrated LCB2 expression 2.7-fold        (fig. 2C), expect rapidly purged <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> (63%) wild type associated changes             populations natural selection. By token, high- <br /> levels ceramides. Specifically, observed signifi-    expression alleles measurable fitness consequence <br /> cant decrease level C18:1 dihydroceramide (two-            expect alleles segregate freely popu- <br /> tailed t test, P = 0.022; fig. 1), ceramide       lations. To test hypothesis, measured expression levels <br /> species increased [two-tailed t test: C18 (P = 0.037), C20           LCB2 ecologically genetically divergent strains <br /> (P = 0.044), C22:1 dihydroceramide (P = 0.011), C20:1 cer-       S. cerevisiae: vineyard strain RM11-1a, pathogenic strain <br /> amide (P = 0.038)]. Individual ceramide species influence key        YJM145-a, oak strain YPS 3332. Contrary the <br /> aspects longevity, including cell growth, regulation, differ-     LCB2 fitness curve led expect, wild strains had <br /> entiation, death (reviewed Hannun Obeid 2011).            significantly lower levels LCB2 expression lab <br /> We saw decreases increases stearoyl (C18) cer-           strain standard media osmotic stress media <br /> amides, 18 carbon fatty acid ceramide        (fig. 3). The level LCB2 expression 1.5-fold (33%) lower <br /> dihydroceramide moiety. Effects stearoyl ceramides cell        pathogenic strain YJM145 1.4-fold (27%) lower in <br /> growth observed elsewhere, example,           vineyard strain RM11-1a lab strain both <br /> involved progression squamous cell carcinomas           grown standard media (two-tailed t test, P = 0.028 and <br /> human head neck tumors (Koybasi et al. 2004).                    P = 0.047, respectively). This among-strain analysis requires <br />     Reduction LCB2 levels wild-type results      reference gene used quantitative polymerase <br /> altered transcription genes. According data col-          chain reaction (qPCR), ALG9, changed, been <br /> lected Mnaimneh et al. (2004), 3.8-fold (74%) reduction         shown previous studies variety growth condi- <br /> in LCB2 expression results severe growth defect        tions genetic backgrounds (Kvitek et al. 2008; Teste et al. <br /> genome-wide transcriptional changes including repression          2009). We checked results using stable qPCR <br /> structural ribosomal genes induction genes associated         reference gene according studies, TFC1, found <br /> with cellular stress response (supplementary table S2, Supple-       YJM145 significantly different (1.5-fold lower) <br /> mentary Material online).                                            expression comparison lab strain (two-tailed t test, <br />                                                                      P = 0.009). <br /> The Fitness Curve Shifts Response Altered                          If fitness curves LCB2 expression in <br /> Environmental Conditions                                             YJM145 RM11-1a lab strain, wild strains <br /> The levels types sphingolipids required cell change      presumably lower fitness. A likely inter- <br /> across different environments, example, response           pretation LCB2 fitness curve itself shifted in <br /> osmotic stress (Patton et al. 1992; Hannun Obeid                 lab strain epistasis genetic background <br /> 2008), predicted effect varying LCB2 levels           and/or selection sphingolipid levels. To extent that <br /> would change environments. To              different strains exhibit different expression&#226;&#8364;&#8220;fitness curves, <br /> determine extent environmental change affects           level permissible expression variation corres- <br /> the LCB2 expression&#226;&#8364;&#8220;fitness curve, measured curve             pondingly affected. We hypothesize se- <br /> when cells subjected osmotic stress. We noted            lection higher levels LCB2 expression lab strain. <br /> that wild-type parental strain (R1158), LCB2 expression       Given sphingolipids involved amino acid sensing <br /> was upregulated 1.6-fold osmotic stress media compared            transport (Dickson 2010), plausible cause this <br /> with standard media (arrows fig. 2C; fig. 3; two-tailed t test,   shift deletion lab strain genes <br /> P = 0.004). We determined dose-response function           involved amino acid synthesis (Brachmann et al. <br /> for TetO7&#226;&#8364;&#8220;LCB2 strain osmotic stress media (fig. 2B). We      1998). <br /> then measured changes LCB2 expression affected fit- <br /> ness osmotic stress media. As expression increased 7.3-fold,      Expression Responses Stoichiometric Interactors <br /> there consistent change fitness (dashed blue line       To perform function, Lcb2 binds stoichiometric <br /> fig. 2C). As expression decreased 2.2-fold, 12.6%        complex Lcb1 form SPT (fig. 1). In turn, SPT regu- <br /> reduction fitness. Interestingly, expression&#226;&#8364;&#8220;fitness          lated stoichiometric interactions repressor <br /> curve osmotic stress similar shape curve          complex (Orm1/2) activator (Tsc3) (Monaghan <br /> in standard media shifted region higher expression.      et al. 2002; Breslow et al. 2010). It important consider <br /> This shift fitness curve indicates extent selection    LCB1 expression studying variation LCB2 expression, <br /> for environment-specific responses gene regulation.               extent subunits coregulated or <br />                                                                      subunit limiting determines level functional SPT <br /> Evidence Distinct Fitness Curves Different                    complex. To investigate this, measured levels LCB1 in <br /> Genetic Backgrounds                                                  lab wild strains standard media osmotic <br /> The shape fitness curve indicates relative fitness        stress media. We LCB2 LCB1 covary each <br /> cost mutant alleles compared wild-type               strains environments, consistent with <br />  <br />                                                                                                                                  451 <br />  Rest et al. . doi:10.1093/molbev/mss248                                                                                               MBE <br />                                                                               complex phenotypes (Manolio et al. 2009) enigma <br />                                                                               instead consistent fundamental theory the <br />                                                                               rugged landscape outcomes results nonlinear <br />                                                                               outcomes loci recombine population (Wright <br />                                                                               1932). For nonlinear curves, predicting effects mutation <br />                                                                               ab initio difficult, magnitude effect de- <br />                                                                               pends specific region fitness curve traversed by <br />                                                                               mutation. In particular, challenging distinguish <br />                                                                               neutral functional variation. However, map- <br />                                                                               ping fitness curves, results variation different envir- <br />                                                                               onments genetic backgrounds predicted. <br />                                                                               Titration expression alleles different environments <br />                                                                               (as demonstrated here) different genetic backgrounds <br />                                                                               holds great utility interpreting variation expression <br />                                                                               cells, tissues, individuals, populations. <br />                                                                                   We observed decreases expression level of <br />                                                                               LCB2 wild-type striking fitness cost (fig. 2C). <br />                                                                               Consistent this, LCB2 low cell-to-cell variation in <br />                                                                               gene expression (Newman et al. 2006), expect <br /> FIG. 3. LCB2 LCB1 gene expression levels covary wild strains        gene perched fitness cliff, variation that <br /> of Saccharomyces cerevisiae growth environments. LCB2             results expression decrease harmful. A large <br /> gene expression upregulated osmotic stress media (blue points)       cost relatively small LCB2 dosage reductions makes sense, <br /> comparison levels expression standard media (orange points).       considering sphingolipids require precise regulation <br /> LCB2 levels significantly lower wild strains YJM145 RM11-1a        (Hannun Obeid 2008; Lebman Spiegel 2008). This <br /> than auxotrophic lab strain R1158, suggesting expres-         precise regulation SPT (Lcb1/Lcb2) activity achieved not <br /> sion&#226;&#8364;&#8220;fitness function evolved lab strain (asterisk; two-tailed t <br />                                                                               transcriptional responses direct interaction a <br /> test, P = 0.028 P = 0.047, respectively). Fold change ascertained <br /> by qPCR relative expression level gene the <br />                                                                               repressor, Orm1/2 (fig. 1) (Breslow et al. 2010). The Orm1/2 <br /> lab strain (R1158) grown standard media. Standard errors           complex phosphorylated response low levels cellular <br /> three biological replicates shown.                                sphingolipids, diminishes ability complex to <br />                                                                               oligomerize inhibit SPT; resulting increase SPT <br /> this stoichiometric expectation (linear regression model,                     activity yields increased sphingolipid flux. Reduced LCB2 ex- <br /> P = 0.005; fig. 3).                                                           pression leads changes flux sphingolipid <br />                                                                               pathway altered levels essential ceramides (fig. 1); <br />                                                                               reduced LCB2 expression phenocopies Orm over- <br /> Discussion <br />                                                                               expression (Breslow et al. 2010). Sensitivity decreased LCB2 <br /> The position wild-type LCB2 gene expression edge                 expression consistent hypothesis small <br /> a fitness cliff, slope rapidly changes, supports                changes regulation sphingolipids humans can <br /> hypothesis effects mutations highly non-                   cause diseases childhood asthma, Crohn&#226;&#8364;&#8482;s disease, <br /> linear. It important know variation simple trait,              type 1 diabetes, biliary cirrhosis. We LCB2 <br /> such gene expression, affect complex outcomes,                    expression edge fitness cliff, consistent with <br /> such disease fitness. If complex outcomes respond lin-                  SPT having high level metabolic control. Sensitivity <br /> early trait variation, contribution variation               variation SPT activity position <br /> many simple traits (e.g., loci) complex out-                     near head sphingolipid synthesis pathway (fig. 1), <br /> comes easily calculated inferred (e.g., genetic                 variation suggested exert strongest <br /> association studies). Early work bacteria suggested                   control (Eanes 1999; Wright Rausher 2010). However, <br /> linearity prevail (Elena Lenski 1997). However,                     note result fitness cliff reduction of <br /> the lack success genome-wide association studies                     LCB2 haploid yeast contrasts lack significant <br /> humans associated missing heritability suggested                 fitness cost hemizygous deletion LCB2 diploids <br /> that picture instead complicated                         (Deutschbauer et al. 2005). Determining discrep- <br /> (Manolio et al. 2009). Other work indicated                      ancy different effects LCB2 reduction among <br /> quantitative contributions loci complex-trait variation                 haploids diploids, differences growth <br /> may change completely different combinations environ-                   conditions fitness assay (batch growth) and <br /> ments genetic backgrounds (Gerke et al. 2010). Indeed,                    hemizygous fitness assay (exponential growth) require <br /> we LCB2 effects genetic variation                  additional experimental work. <br /> are nonlinear differ environments genetic                          Why overexpression LCB2 close neutral re- <br /> backgrounds (Dekel Alon 2005; Perfeito et al. 2011).                      spect fitness (fig. 2C)? Increases LCB2 transcription do <br /> Yet continuous fitness curve (fig. 2C) shows                    necessarily mediate proportional increase Lcb2 pro- <br /> missing information regarding genetic basis variation                  tein levels. In particular, Lcb2 unstable unless associated <br />  <br /> 452 <br />  Fitness Expression Level Variation . doi:10.1093/molbev/mss248                                                             MBE <br /> with Lcb1 (Gable et al. 2000; Yasuda et al. 2003), LCB1          examine effect varying expression activity at <br /> expression does increase LCB2, excess Lcb2          level cascade. In particular, highest two <br /> would degraded. However, LCB1 levels appear                   levels cascade (MAPKKK MAPKK), effect of <br /> upregulated response upregulation LCB2 (fig. 3),              variation large low expression levels saturates at <br /> and unlikely Lcb1 limitation sole          higher expression levels, observe LCB2. The authors <br /> factor underlying absence fitness effect. Even add-        point wild-type model parameters, expres- <br /> itional SPT formed, increased dose affect              sion level, typically unknown. However, suggest that <br /> sphingolipid flux fitness Orm1/2             wild-type phenotypes unlikely occur deep plat- <br /> inhibit SPT activity. Because wild-type levels Lcb1 Lcb2        eaus fitness curve fitness curve steeply <br /> are high sphingolipid flux levels toxic         sloped. Instead, conclude likely that <br /> absence Orm1/2 repression (Breslow et al. 2010), find-          wild-type phenotypes lie near edges fitness <br /> ings suggest Orm1/2 capable buffering cell          curves, observe LCB2. <br /> against SPT levels greater                    Although fitness curve measured lab real, is <br /> wild-type.                                                             unclear reflects aspects fitness important the <br />     It advantageous, therefore, wild-type           wild. Yeast grows exponentially glucose, growth <br /> expression fall region high expression           slows glucose used population saturates, and <br /> offers buffer genetic, environmental, stochastic         serially diluted culture fresh media 24 h so <br /> perturbations lower LCB2 levels. In words,           growth continue. As culture went through <br /> the LCB2 expression level higher, mutations             cycles lag phase, exponential growth, satur- <br /> decrease LCB2 expression result severely com-             ation, fitness measurements include absolute <br /> promised fitness. There possible explanations          growth rate factors recovery exit <br /> the precarious position wild-type expression LCB2         lag phase. However, likely effect LCB2 <br /> edge fitness cliff (fig. 2C). One possibility    expression relative fitness differ growth <br /> are growth environments cost              conditions. For example, components fitness that <br /> overexpression (Hillenmeyer et al. 2008). A second possibility         investigated sporulation efficiency wild <br /> is small deleterious fitness consequence for <br />                                                                        (Gerke et al. 2006), exponential growth chemostats <br /> expressing LCB2 wild-type levels our <br />                                                                        (Dykhuizen Hartl 1983), long-term <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> under <br /> measurement resolution. It estimated other- <br />                                                                        <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limited conditions (Gresham et al. 2008). <br /> wise neutral 2-fold increase gene expression selected <br />                                                                            The laboratory adapted strain S288C, TetO7&#226;&#8364;&#8220; <br /> against solely metabolic costs mRNA pro- <br />                                                                        LCB2 derived, experienced recurrent bottlenecks and <br /> tein production, genes lowest <br />                                                                        shows high evolutionary rates (Gu et al. 2005). This <br /> levels expression (Wagner 2005, 2007). The critical selection <br />                                                                        mosaic strain lacks key gene flocculation (Liu et al. 1996), <br /> coefficient Saccharomyces, selection an <br /> allele predominate drift, calculated          TetO7&#226;&#8364;&#8220;LCB2 lacks genes required synthesis three <br /> very small ( 2.93   10 7) based estimates mutation               amino acids. Consistent history bottlenecks, gen- <br /> rate large historical population size (Wagner 2005).          etic manipulation, elevated rates evolution, S288C and <br /> This resolution fitness measurements,         derivatives outliers S. cerevisiae variety of <br /> so fitness curve figure 2C peak        phenotypes including growth ethanol maltose (War- <br /> wild-type expression level, albeit shallow slope         ringer et al. 2011) sporulation efficiency (Deutschbauer <br /> right-hand side. It possible accessible       Davis 2005). It possible observation LCB2 <br /> mutational path LCB2 promoter achieve higher                wild-type expression edge fitness cliff simi- <br /> expression, mutational target LCB2 regulation          larly outlier. It important future work to <br /> is larger expected, drift predominates              measure LCB2 expression&#226;&#8364;&#8220;fitness curves wild strains <br /> (i.e., critical selection coefficient higher estimated).   determine wild-type expression levels are <br /> The position edge cliff permit sphingo-           similarly positioned. <br /> lipid synthesis rapidly reduced environments                 LCB2 essential gene relatively low expression <br /> where cell longevity important rapid growth               level, factors likely contribute steepness its <br /> (Huang et al. 2012). An additional possibility, propose,         fitness curve expression decreases. Previous work indicates <br /> that lab strain recently experienced strong selection          consequences decreased expression essential <br /> for increased LCB2 expression associated loss              genes distributed similarly nonessential genes (Delneri <br /> amino acid synthesis pathways, given time,              et al. 2008). However, substantial heterogeneity <br /> expression level evolve away precipitous position         genes phenotypic fitness effects of <br /> at edge fitness cliff.                                      overexpression (Sopko et al. 2006) reduced expres- <br />     The cliff-like shape LCB2 expression&#226;&#8364;&#8220;fitness curve          sion (Giaever et al. 2002; Mnaimneh et al. 2004; Deutschbauer <br /> mirrors shape observed varying expression levels           et al. 2005; Delneri et al. 2008). There likely many <br /> model mitogen-activated protein kinase (MAPK) sig-              different shapes expression&#226;&#8364;&#8220;fitness curves, reflecting the <br /> naling cascade developed Nijhout et al. (2003). The pheno-          proteome&#226;&#8364;&#8482;s diversity functions structures com- <br /> typic output level active MAPK,          plex web interactions genes proteins. <br />  <br />                                                                                                                                       453 <br />  Rest et al. . doi:10.1093/molbev/mss248                                                                                      MBE <br />    These caveats highlight need expand work       strain competition culture, measured changes in <br /> panel wild-strains, additional genes envir-   Fcompetition, fluorescence <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> OD competition <br /> onments. We expect multiple classes expres-       culture: <br /> sion&#226;&#8364;&#8220;fitness curves, foresee systematic                                                                                 ! <br /> investigations begin quantitatively stand-                                      FcompetitionC, E   FparentalC, E <br />                                                                                                ln <br /> ing variation results evolutionary integration                                       FreferenceonlyC, E   FcompetitionC, E <br /> environment genetic-background specific fitness curves.        !C, E &#194;&#188; 1   log2 &#195;&#176;d&#195;&#382;   <br />                                                                                                                  t <br /> Systems like Tet-Off promoter, use here, allow genes <br /> to systematically titrated, physiological fitness                                                                       &#195;&#176;2&#195;&#382; <br /> consequences quantitatively examined. Our results        d dilution factor (1:50) <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> time t <br /> such systems hold possibility producing larger scale       (1,440 min), Fparental Freferenceonly fluorescence <br /> catalogs expression&#226;&#8364;&#8220;fitness curves serve        <span id='am-244' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> OD parental cells reference cells only, <br /> tool interpreting natural variation gene expression         respectively. We used F A fell 0.57 and <br /> levels.                                                            0.70 (equivalent 4.2   107 5.2   107 cells/ml), which <br />                                                                    allowed compare data wells days fell in <br /> Materials Methods                                              similar optical range. Each doxycycline concentration and <br /> Titration LCB2 Expression                                       environment combination had, average, biological <br /> We altered levels LCB2 expression S. cerevisiae strain       replicates unique plates days. <br /> TetO7&#226;&#8364;&#8220;LCB2 (pLCB2::kanR-TetO7-TATA URA3::CMV-tTA <br /> MATa his3-1 leu2-0 met15-0), native LCB2 pro- <br /> moter replaced tetracycline/doxycycline re-          Measurement LCB2 LCB1 mRNA Levels <br /> pressible promoter, TetO7 (Mnaimneh et al. 2004). LCB2             TetO7&#226;&#8364;&#8220;LCB2 grown measure levels LCB2 expression <br /> mRNA levels measured 29 different con-          different concentrations doxycycline envir- <br /> centrations doxycycline, depending growth                onment. All growth conditions identical com- <br /> condition.                                                         petition protocol (e.g., microwell plates), that <br />                                                                    TetO7&#226;&#8364;&#8220;LCB2 grown monoculture. Subset samples <br /> Fitness Competitions                                               grown test tubes microwell plates. In <br />                                                                    addition, LCB2 expression environment assayed <br /> Fitness level doxycycline measured via <br /> head-to-head competition TetO7&#226;&#8364;&#8220;LCB2 fluor-           parental strain (R1158) oak (YPS 3332; MATa <br /> escent reference strain, GPM1&#226;&#8364;&#8220;GFP (Huh et al. 2003). Our           ho::Nat) (Murphy et al. 2006), vineyard (RM11-1a; MATa <br /> growth protocol follows: TetO7&#226;&#8364;&#8220;LCB2 reference           leu2D ura3D ho::Kan) (Brem et al. 2002), pathogenic <br /> cells grown separately liquid media 24 h, diluted   (YJM145a; MATa gal2 HO) (McCusker et al. 1994) wild strains. <br /> a density 1.5   107 cells/mL, grown  8 add-        Growth treatment strain combination re- <br /> itional hours. The strains mixed 10:1 (TetO7&#226;&#8364;&#8220;     peated times unique days plates. RNA <br /> LCB2: reference) ratios 8   106 cells inoculated     extracted second day growth cultures <br /> 150 mL media appropriate concentration doxy-          reached optical density 0.6 ( 4.4   107 cells/ml; <br /> cycline hyclate (Calbiochem). Additional wells contained pure      Zymolyase, Spin Columns, Zymo Research; Turbo DNA-free, <br /> cultures reference strain parental strain (R1158;    Ambion). Expression levels quantified using quantitative <br /> URA3::CMV-tTA MATa his3-1 leu2-0 met15-0). Cultures           reverse transcriptase-PCR LCB2 primers 50 -TTGCTGT <br /> grown triplicate black-wall, clear-bottom 96-well plates     TGTTGTTGTTGCTTATCCTGCT-30 50 -CGTCGTAACTGG <br /> (Nunc) covered foil seals (Corning) shaken             ATTTGCCGGAATTTGAT-30 LCB1 primers 50 -GCACACA <br /> 1,300 rpm (DTS4, Elmi) 30  C. The standard growth envir-        TCCCAGAGGTTTT-30 50 -TGGTCCTGTATGGATCGT <br /> onment synthetic defined media tryptophan              CA-30 (Brilliant II SYBR 1-Step, Agilent; Mx3000P, Stratagene). <br /> (SD-Trp, Sunrise Scientific). The osmotic stress environment       ALG9 served reference gene normalization the <br /> contained 0.3 M (instead 0.1 M) NaCl. Competitions         qPCR reactions, primers 50 -CACGGATAGTGGCTTTGG <br /> diluted 1:50 fresh media 24 h 4 days. The optical   TGAACAATTAC-30 50 -TATGATTATCTGGCAGCAGGAA <br /> density A (OD units 600 nm) raw fluorescence B              AGAACTTGGG-30 . We normalized subsample reactions <br /> (485 nm excitation, 530 nm emission) microwell cul-        additional reference gene, TFC1, primers: 50 -GCT <br /> ture recorded hourly (F500, Tecan). We calculated F,          GGCACTCATATCTTATCGTTTCACAATGG-30 50 -GAAC <br /> fluorescence <span id='am-245' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> OD:                                          CTGCTGTCAATACCGCCTGGAG-30 . None primer <br />                                                                    sets (LCB2, ALG9, TFC1) contained mismatches their <br />                                  B                                 cognate sequences wild strains analyzed. Each qPCR <br />                             F&#194;&#188;                              &#195;&#176;1&#195;&#382; <br />                                  A                                 reaction repeated times separate plates. <br /> where unit OD contains 7.39   107 cells/mL. Relative           qPCR data analyzed according 2   Ct method <br /> fitness, !, TetO7&#226;&#8364;&#8220;LCB2 doxycycline concentration,       (Livak Schmittgen 2001). For environment, fitted <br /> C, environment, E, derived rate change         relationship doxycycline concentration and <br /> over 4 days proportion TetO7&#226;&#8364;&#8220;LCB2 reference       fold change five-parameter log-logistic function and <br />  <br /> 454 <br />  Fitness Expression Level Variation . doi:10.1093/molbev/mss248                                                                       MBE <br /> used function predict final values estimate error               Deutschbauer AM, Davis RW. 2005. Quantitative trait loci mapped to <br /> (R v2.14; package drc v2.2-1).                                                 single-nucleotide resolution yeast. Nat Genet. 37:1333&#226;&#8364;&#8220;1340. <br />                                                                            Deutschbauer AM, Jaramillo DF, Proctor M, Kumm J, Hillenmeyer ME, <br />                                                                                Davis RW, Nislow C, Giaever G. 2005. Mechanisms haploinsuffi- <br /> Sphingolipid Analysis                                                          ciency revealed genome-wide profiling yeast. Genetics 169: <br /> Sphingolipid metabolites critical points sphingolipid                1915&#226;&#8364;&#8220;1925. <br /> synthesis pathway extracted monocultures                      Dickson RC. 2010. Roles sphingolipids Saccharomyces cerevisiae. In: <br /> TetO7&#226;&#8364;&#8220;LCB2 grown conditions identical                       Chalfant C, Del Poeta M, editors. Sphingolipids signaling and <br /> qPCR. Nine biological replicate cultures grown,                   regulatory molecules. New York: Springer. p. 217&#226;&#8364;&#8220;231. <br /> three concentrations doxycycline: 0, 0.0293, 2.5 mg/ml,             Dykhuizen DE, Dean AM, Hartl DL. 1987. Metabolic flux fitness. <br /> equivalent 7.3-fold higher, as, 2.7-fold lower                 Genetics 115:25&#226;&#8364;&#8220;31. <br /> than wild type expression, respectively. Sphingolipid levels               Dykhuizen DE, Hartl DL. 1983. Selection chemostats. Microbiol Rev. 47: <br /> were quantitated using LC-MS/MS lipidomics core                         150. <br /> facility Medical University South Carolina, following                Eanes WF. 1999. Analysis selection enzyme polymorphisms. Annu <br /> methods described Bielawski et al. (2006).                         Rev Ecol Syst. 30:301&#226;&#8364;&#8220;326. <br />                                                                            Eanes WF. 2011. Molecular population genetics selection the <br /> Supplementary Material                                                         glycolytic pathway. J Exp Biol. 214:165&#226;&#8364;&#8220;171. <br /> Supplementary tables S1 S2 available Molecular                  Elena SF, Lenski RE. 1997. Test synergistic interactions dele- <br /> Biology Evolution online (http://www.mbe.oxfordjournals.                   terious mutations bacteria. Nature 390:395&#226;&#8364;&#8220;398. <br /> org/).                                                                     Eng KH, Kvitek DJ, Keles&#194;&#184; S, Gasch AP. 2010. Transient genotype- <br />                                                                                by-environment interactions following environmental shock pro- <br />                                                                                vide source expression variation essential genes. Genetics <br /> Acknowledgments                                                                184:587&#226;&#8364;&#8220;593. <br /> The authors thank Walter Eanes, Geoffrey Morris, Aman Gill,                Fisher RA. 1928. The possible modification response wild <br /> Niamh O&#226;&#8364;&#8482;Hara, Bin He, Jeffrey Rest; anonymous reviewers                    type recurrent mutations. Am Nat. 62:115&#226;&#8364;&#8220;126. <br /> for comments manuscript; Jason O&#226;&#8364;&#8482;Rawe, Geoff                    Gable K, Han G, Monaghan E, Bacikova D, Natarajan M, Williams R, <br /> Bolen, Laura Praissman assistance lab. Paul                     Dunn TM. 2002. Mutations yeast LCB1 LCB2 genes, <br /> Sniegowski provided strain YPS 3332. This work sup-                        including corresponding hereditary sensory neuropathy <br /> ported startup funds Stony Brook University J.S.R.                  type I mutations, dominantly inactivate serine palmitoyltransferase. <br />                                                                                J Biol Chem. 277:10194&#226;&#8364;&#8220;10200. <br /> References                                                                 Gable K, Slife H, Bacikova D, Monaghan E, Dunn TM. 2000. Tsc3p an <br /> Bayer TS. 2010. Using synthetic biology understand evolution         80-amino acid protein associated serine palmitoyltransferase <br />     gene expression. Curr Biol. 20:R772&#226;&#8364;&#8220;R779.                                  required optimal enzyme activity. J Biol Chem. 275: <br />                                                                                7597&#226;&#8364;&#8220;7603. <br /> Bielawski J, Szulc ZM, Hannun YA, Bielawska A. 2006. Simultaneous <br />     quantitative analysis bioactive sphingolipids high-performance   Gerke J, Lorenz K, Ramnarine S, Cohen B. 2010. Gene- <br />     liquid chromatography-tandem mass spectrometry. Methods 39:                environment interactions nucleotide resolution. PLoS Genet. 6: <br />     82&#226;&#8364;&#8220;91.                                                                     e1001144. <br /> Birchler JA, Bhadra U, Bhadra MP, Auger DL. 2001. Dosage-dependent         Gerke JP, Chen CTL, Cohen BA. 2006. Natural isolates Saccharomyces <br />     gene regulation multicellular eukaryotes: implications dosage       cerevisiae display complex genetic variation sporulation efficiency. <br />     compensation, aneuploid syndromes, quantitative traits. Dev            Genetics 174:985. <br />     Biol. 234:275&#226;&#8364;&#8220;288.                                                     Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br /> Brachmann CB, Davies A, Cost GJ, Caputo E, Li J, Hieter P, Boeke JD.           Lucau-Danila A, Anderson K, Andre B. 2002. Functional profiling of <br />     1998. Designer deletion strains derived Saccharomyces cerevi-         Saccharomyces cerevisiae genome. Nature 418:387&#226;&#8364;&#8220;391. <br />     siae S288C: useful set strains plasmids PCR-mediated      Gresham D, Desai MM, Tucker CM, Jenq HT, Pai DA, Ward A, DeSevo <br />     gene disruption applications. Yeast 14:115&#226;&#8364;&#8220;132.                  CG, Botstein D, Dunham MJ. 2008. The repertoire dynamics of <br /> Brem RB, Yvert Gl, Clinton R, Kruglyak L. 2002. Genetic dissection          evolutionary adaptations controlled <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-limited environ- <br />     transcriptional regulation budding yeast. Science 296:752&#226;&#8364;&#8220;755.          ments yeast. PLoS Genet. 4:e1000303. <br /> Breslow DK, Collins SR, Bodenmiller B, Aebersold R, Simons K,              Gu Z, David L, Petrov D, Jones T, Davis RW, Steinmetz LM. 2005. <br />     Shevchenko A, Ejsing CS, Weissman JS. 2010. Orm family proteins            Elevated evolutionary rates laboratory strain of <br />     mediate sphingolipid homeostasis. Nature 463:1048&#226;&#8364;&#8220;1053.                    Saccharomyces cerevisiae. Proc Natl Acad Sci U S A. 102:1092&#226;&#8364;&#8220;1097. <br /> Cowart LA, Obeid LM. 2007. Yeast sphingolipids: recent developments        Haldane JBS. 1930. A note Fisher&#226;&#8364;&#8482;s theory origin dominance, <br />     understanding biosynthesis, regulation, function. Biochim           correlation dominance linkage. Am Nat. 64: <br />     Biophys Acta. 1771:421&#226;&#8364;&#8220;431.                                                87&#226;&#8364;&#8220;90. <br /> Dekel E, Alon U. 2005. Optimality evolutionary tuning ex-       Hannun YA, Obeid LM. 2008. Principles bioactive lipid signaling: les- <br />     pression level protein. Nature 436:588&#226;&#8364;&#8220;592.                           sons sphingolipids. Nat Rev Mol Cell Biol. 9:139&#226;&#8364;&#8220;150. <br /> Delneri D, Hoyle DC, Gkargkas K, et al. (12 co-authors). 2008.             Hannun YA, Obeid LM. 2011. Many ceramides. J Biol Chem. 286: <br />     Identification characterization high-flux-control genes          27855&#226;&#8364;&#8220;27862. <br />     yeast competition analyses continuous cultures. Nat         Hartl DL, Dykhuizen DE, Dean AM. 1985. Limits adaptation: the <br />     Genet. 40:113&#226;&#8364;&#8220;117.                                                         evolution selective neutrality. Genetics 111:655&#226;&#8364;&#8220;674. <br />  <br />                                                                                                                                                  455 <br />  Rest et al. . doi:10.1093/molbev/mss248                                                                                                     MBE <br /> Hillenmeyer ME, Fung E, Wildenhain J, et al. (14 co-authors). 2008. The        Papp B, Pal C, Hurst LD. 2003. Dosage sensitivity evolution of <br />      chemical genomic portrait yeast: uncovering phenotype            gene families yeast. Nature 424:194&#226;&#8364;&#8220;197. <br />      genes. Science 320:362&#226;&#8364;&#8220;365.                                               Patton JL, Srinivasan B, Dickson RC, Lester RL. 1992. Phenotypes of <br /> Huang X, Liu J, Dickson RC. 2012. Down-regulating sphingolipid synthe-             sphingolipid-dependent strains Saccharomyces cerevisiae. J <br />      sis increases yeast lifespan. PLoS Genet. 8:e1002493.                         Bacteriol. 174:7180&#226;&#8364;&#8220;7184. <br /> Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, Armour CD,            Patwardhan RP, Lee C, Litvin O, Young DL, Pe&#226;&#8364;&#8482;er D, Shendure J. 2009. <br />      Bennett HA, Coffey E, Dai H, He YD. 2000. Functional discovery          High-resolution analysis DNA regulatory elements synthetic <br />      compendium expression profiles. Cell 102:109&#226;&#8364;&#8220;126.                          saturation mutagenesis. Nat Biotechnol. 27:1173&#226;&#8364;&#8220;1175. <br /> Huh W-K, Falvo JV, Gerke LC, Carroll AS, Howson RW, Weissman JS,               Peng W-T, Robinson MD, Mnaimneh S, Krogan NJ, Cagney G, Morris Q, <br />      O&#226;&#8364;&#8482;Shea EK. 2003. Global analysis protein localization budding           Davierwala AP, Grigull J, Yang X, Zhang W. 2003. A panoramic view <br />      yeast. Nature 425:686&#226;&#8364;&#8220;691.                                                    yeast noncoding RNA processing. Cell 113:919&#226;&#8364;&#8220;933. <br /> Khaitovich P, Pa&#194;&#168;a&#194;&#168;bo S, Weiss G. 2005. Toward neutral evolutionary          Perfeito L, Ghozzi S, Berg J, Schnetz K, La&#194;&#168;ssig M. 2011. Nonlinear fitness <br />      model gene expression. Genetics 170:929&#226;&#8364;&#8220;939.                               landscape molecular pathway. PLoS Genet. 7:e1002160. <br /> Koybasi S, Senkal CE, Sundararaj K, et al. (12 co-authors). 2004. Defects   Rossouw D, Olivares-Hernandes R, Nielsen J, Bauer FF. 2009. <br />      cell growth regulation C18:0-ceramide longevity assurance              Comparative transcriptomic approach investigate differences in <br />      gene 1 human head neck squamous cell carcinomas. J Biol                wine yeast physiology metabolism fermentation. Appl <br />      Chem. 279:44311&#226;&#8364;&#8220;44319.                                                        Environ Microbiol. 75:6600&#226;&#8364;&#8220;6612. <br /> Kvitek DJ, Will JL, Gasch AP. 2008. Variations stress sensitivity       Sopko R, Huang D, Preston N, et al. (12 co-authors). 2006. Mapping <br />      genomic expression diverse S. cerevisiae isolates. PLoS Genet. 4:          pathways phenotypes systematic gene overexpression. Mol <br />      e1000223.                                                                     Cell. 21:319&#226;&#8364;&#8220;330. <br /> Lebman DA, Spiegel S. 2008. Thematic review series: sphingolipids.             Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, Eisen MB, <br />      Cross-talk crossroads sphingosine-1-phosphate, growth               Brown PO, Botstein D, Futcher B. 1998. Comprehensive identifica- <br />      factors, cytokine signaling. J Lipid Res. 49:1388&#226;&#8364;&#8220;1394.                   tion cell cycle-regulated genes yeast Saccharomyces cere- <br /> Liu H, Styles CA, Fink GR. 1996. Saccharomyces cerevisiae S288C              visiae microarray hybridization. Mol Biol Cell. 9:3273. <br />      mutation FLO8, gene required filamentous growth. Genetics        Teste M-A, Duquenne M, Francois J, Parrou J-L. 2009. Validation of <br />      144:967&#226;&#8364;&#8220;978.                                                                  reference genes quantitative expression analysis real-time <br /> Livak KJ, Schmittgen TD. 2001. Analysis relative gene expression data           RT-PCR Saccharomyces cerevisiae. BMC Mol Biol. 10:99. <br />      using real-time quantitative PCR 2(-Delta Delta C(T)) <br />                                                                                Tu BP, Kudlicki A, Rowicka M, McKnight SL. 2005. Logic yeast <br />      method. Methods 25:402&#226;&#8364;&#8220;408. <br />                                                                                    metabolic cycle: temporal compartmentalization cellular pro- <br /> Manolio TA, Collins FS, Cox NJ, et al. (27 co-authors). 2009. Finding          cesses. Science 310:1152&#226;&#8364;&#8220;1158. <br />      missing heritability complex diseases. Nature 461:747&#226;&#8364;&#8220;753. <br />                                                                                Veitia RA. 2002. Exploring etiology haploinsufficiency. BioEssays <br /> McCusker JH, Clemons KV, Stevens DA, Davis RW. 1994. Genetic char-                 24:175&#226;&#8364;&#8220;184. <br />      acterization pathogenic Saccharomyces cerevisiae isolates. <br />                                                                                Wagner A. 2005. Energy constraints evolution gene expression. <br />      Genetics 136:1261&#226;&#8364;&#8220;1269. <br />                                                                                    Mol Biol Evol. 22:1365&#226;&#8364;&#8220;1374. <br /> Mn <br /> </body></html>