<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Depletion C. elegans NAC Engages Unfolded <br /> Protein Response, Resulting Increased Chaperone <br /> Expression Apoptosis <br /> Paul T. Arsenovic, Anthony T. Maldonado, Vaughn D. Colleluori, Tim A. Bloss* <br /> Department Biology, James Madison University, Harrisonburg, Virginia, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The nascent polypeptide-associated complex (NAC) highly conserved heterodimer important metazoan <br />      development, molecular function understood. Recent evidence suggests NAC component of <br />      cytosolic chaperone network interacts ribosomal complexes emerging nascent peptides, that <br />      loss NAC chaperone-depleted cells results increase misfolded protein stress. We tested the <br />      NAC functions similarly Caeonorhabditis (C.) elegans homologous NAC subunits, i.e. ICD-1 -2, have <br />      chaperone-like characteristics. Loss NAC appears induce misfolded protein stress ER triggering unfolded <br />      protein response (UPR). Depletion NAC altered response heat stress, led up-regulation hsp-4, a <br />      homologue human chaperone ER stress sensor GRP78/BiP. Worms lacking ICD-1 UPR transcription <br />      factor XBP-1 generated higher proportion defective embryos, showed increased embryonic apoptosis a <br />      diminished <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>survival</span> rate relative ICD-1-depleted animals intact UPR. Up-regulation hsp-4 NAC-depleted <br />      animals specific certain regions embryo; embryos lacking ICD-1, posterior region embryo showed <br />      strong up-regulation hsp-4, anterior region did not. Furthermore, loss ICD-1 produced prominent lysosomes <br />      gut region adults embryos putatively containing lipofuscins, lipid/protein aggregates associated cellular <br />      aging. These results set evidence consistent role C. elegans NAC protein folding localization <br />      translation. Further, findings confirm C. elegans valuable model studying organismal cell-type <br />      specific responses misfolded protein stress. <br />  <br />   Citation: Arsenovic PT, Maldonado AT, Colleluori VD, Bloss TA (2012) Depletion C. elegans NAC Engages Unfolded Protein Response, Resulting in <br />   Increased Chaperone Expression Apoptosis. PLoS ONE 7(9): e44038. doi:10.1371/journal.pone.0044038 <br />   Editor: Alejandro Aballay, Duke University Medical Center, United States America <br />   Received June 15, 2012; Accepted August 1, 2012; Published September 5, 2012 <br />   Copyright: &#195;&#376; 2012 Arsenovic et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The research presented manuscript funded grant Jeffress Memorial Trust (grant number J-925), start-up funding <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />   James Madison University. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: Tim A. Bloss author patent# 20060199781 &#226;&#8364;&#732;&#226;&#8364;&#732;Assays based BTF3 activity&#226;&#8364;&#8482;&#226;&#8364;&#8482;. <br />   * E-mail: blossta@jmu.edu <br />  <br />  <br />  <br /> Introduction                                                                         chaperones prevent inappropriate associations secondary <br />                                                                                      structures allow formation functional domains the <br />    <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> ensure survival, organisms tightly control                        peptide released cytosol. Translational chaperone <br /> synthesis folding proteins. To end, specialized proteins                 interaction important protein localization, <br /> called chaperones regulate novo protein-folding                  specifically organelles ER mitochondria. The <br /> translation, recuperative protein-folding denaturing                      translational chaperone best understood Saccharomyces <br /> stress. Disruption protein folding underlies pathologies                   cerevisiae, heat shock protein (HSP) 70/40 triad complex <br /> most, all, neurodegenerative disorders, including Alzhei-                     NAC associate growing peptide chains the <br /> mer&#226;&#8364;&#8482;s, Huntington&#226;&#8364;&#8482;s, Parkinson&#226;&#8364;&#8482;s diseases, amyo-                      ribosome [6]. <br /> trophic lateral sclerosis (ALS) [1,2]. During disease development,                      The HSP 70/40 triad complex chaperones, of <br /> the accumulation improperly folded aggregation-prone                           form ribosome-associated complex (RAC) binds <br /> proteins leads formation toxic oligomers trigger                  directly ribosomal complex, third, Ssb, binds both <br /> neuronal cell death [3]. To prevent protein aggregation, cells                       ribosomal complex nascent peptide chain [7]. The <br /> initiate stress responses forestall accumulation misfolded <br />                                                                                      NAC consists subunits, b, bind the <br /> proteins. These responses decrease translation, increase turnover, <br />                                                                                      polypeptide; bNAC binds ribosomal complex [8,9]. <br /> and manage existing proteins aggregate [4]. <br />                                                                                      Previous studies yeast associate NAC promoting <br /> Misfolded protein management requires increased availability of <br />                                                                                      protein localization away ER mitochondria <br /> chaperones, inhibit aggregation direct interaction <br />                                                                                      translation, recent evidence suggests NAC <br /> with targets [5]. <br />                                                                                      functionally connected HSP 70/40 chaperone <br />    Control protein-folding begins ribosomal complex <br />                                                                                      network plays important role proper protein folding <br /> during translation. Highly conserved chaperones, known the <br />                                                                                      [10,11]. Deletion NAC Ssb-deficient yeast results in <br /> translational chaperone system, interact ribosomal <br /> complex nascent polypeptide emerges. By doing so, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                       September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                C. elegans NAC Unfolded Protein Response <br />  <br />  <br /> significant growth defects accumulation misfolded protein            theorized result autophagy associated with <br /> [12].                                                                       postmitotic cellular aging [23]. <br />    Although NAC RAC conserved throughout <br /> eukaryotic evolution, clear chaperone functions        Results <br /> in yeast conserved complex organisms [12&#226;&#8364;&#8220;14]. Our <br /> studies examine NAC functions chaperone C.              Depletion NAC modulates stress response, <br /> elegans, genetically tractable like yeast,                mortality movement presence heat <br /> substantially complex regards stress response. For                To characterize putative chaperone role NAC, we <br /> example, eukaryotes respond accumulation misfolded                examined ability NAC-depleted animals cope with <br /> protein ER unfolded protein response (UPR),               protein-denaturing heat stress. We reasoned NAC a <br /> mechanism engages pathways involved managing mis-                   component translational chaperone system, removal either <br /> folded protein stress. In yeast, UPR mediated single            NAC subunit (ICD-1 -2) exacerbate misfolded protein <br /> transmembrane ER protein, Ire-1. However, C. elegans contains               accumulation heat stress, decrease worm survival. <br /> three transmembrane ER proteins, Ire-1, ATF6, PERK/                     Synchronized worms treated icd-1 icd-2(RNAi) were <br /> PEK-1, independently cooperatively initiate UPR,               exposed chronic heat stress, survival measured the <br /> and conserved mammalian homologues [15]. Activation                 presence pharyngeal pumping. Unexpectedly, animals depleted <br /> these misfolded protein sensors initiate range responses        ICD-1 -2 sensitive heat stress relative to <br /> maintain cell viability functionality acute stress             wild-type controls, measured mortality rates (Figure 1A). <br /> resolved proteins refolded properly. These responses                ICD-2-depleted worms instead resistant death than <br /> include attenuation protein translation, stimulation              ICD-1-depleted wild-type animals. To determine the <br /> protein degradation, increase ER Golgi biogenesis, up-           loss NAC affected vitality worms heat stress, the <br /> regulation chaperone expression [16]. If misfolded protein stress        movement NAC(2) worms measured relative wild-type <br /> overwhelms cell&#226;&#8364;&#8482;s ability manage it, UPR engages                 presence heat. Animals decreased levels ICD-1 or <br /> apoptotic pathways, kill cell, autophagic                 -2 mobile assays, differences robustness; <br /> pathways, lead cell death. The ability             worms lacking ICD-2 showed greatest movement, followed by <br /> UPR kill chronically stressed cells links disease pathology,       ICD-1-deficient wild-type animals (Figure 1B). Movement was <br /> specifically proteopathic neurodegenerative diseases [17]. With             strictly defined escape response touch metal <br />                                                                             pick, cases, icd-1 icd-2(RNAi)-treated worms were <br /> this mind, role NAC stress response particularly <br />                                                                             quite functional moving feeding absence of <br /> interesting C. elegans, model organism well-defined and <br />                                                                             prodding, wild-type counterparts immobile. <br /> precisely lineaged nervous allows study NAC <br />                                                                             Taken together, results indicate depletion NAC <br /> chaperone function neurons [18]. Induction protein folding <br />                                                                             RNAi treatment did increase mortality post- <br /> stress depletion NAC C. elegans recapitulate <br />                                                                             embryonic animals positively affected movement the <br /> aspects neurodegenerative pathologies, establishing partic- <br />                                                                             presence heat stress. One possible explanation increased <br /> ular experimental strategy viable model stress-related <br />                                                                             vitality NAC(2) worms heat stress induction <br /> neurodegeneration [19,20]. <br />                                                                             compensatory stress response. <br />    The bNAC homologue ICD-1 best understood two <br /> NAC subunits C. elegans. Deletion ICD-1 results in <br /> inappropriate cell death embryogenesis, neurons <br />                                                                             ER chaperone hsp-4 up-regulated absence of <br /> more sensitive removal ICD-1 cell types [21]. In           ICD-1 <br /> addition, depletion ICD-1 induces severe morphological                 To determine depletion NAC up-regulates chaperone <br /> defects embryogenesis independent cell death              expression, examined expression chaperones the <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, icd-1 <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> mutations embryonic lethal.               embryos icd-1(RNAi)-treated adult worms. icd-1(RNAi) been <br /> The subunit NAC C. elegans characterized           shown generate strong phenotypes embryogenesis in <br /> outside predicted association ICD-1. We propose            progeny treated adults, ranging increases apoptosis <br /> this subunit ICD-2.                                                         severe morphological defects embryonic lethality [21]. If <br />                                                                             loss ICD-1 up-regulating chaperone expression, might <br />    To determine role NAC plays proteostasis stress <br />                                                                             expect effect embryogenesis. Four separate <br /> response C. elegans, depleted ICD-1 -2 time using <br />                                                                             hsp::GFP reporter strains depleted ICD-1 time, and <br /> <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>RNA interference</span>, measured specific stress response outputs. <br />                                                                             expression patterns determined embryogenesis. <br /> We depletion NAC modulated responses heat <br />                                                                             hsp-16, cytosolic chaperone, hsp-6 -60, mitochon- <br /> stress up-regulated ER chaperone gene hsp-4, a <br />                                                                             drial-specific chaperones, showed detectible increases in <br /> homologue mammalian GRP78/BiP [22]. Worms a <br />                                                                             expression (data shown) [24,25]. hsp-4::GFP, ER-specific <br /> partially defective Ire-1 pathway significantly sensitive         chaperone, showed dramatic up-regulation, detected mid- <br /> to icd-1(RNAi) effects wild-type animals, including increases          embryogenesis, increasing embryonic develop- <br /> in apoptosis mortality. Worms deficient hsf-1, cytoplasmic         ment (Figure 2A&#226;&#8364;&#8220;C). GFP levels consistently higher in <br /> stress responder, sensitive ICD-1-depletion          embryos treated icd-1(RNAi) relative untreated controls <br /> wild-type, able mount stress response heat. Up-           (Figure 2D) unc-22(RNAi) controls (data shown), as <br /> regulation hsp-4 appeared specific particular regions icd-         measured signal generated embryo, the <br /> 1(RNAi) embryos, high levels expression regions                  average level expression different points the <br /> containing gut epithelial cells, low levels expression        embryo (Figure 2E). Within embryos, hsp-4 expression patterns <br /> neuronal regions. We observed increase large lysosomal           consistently localized specific regions embryo. <br /> structures gut region animal possibly containing              Regions embryo contain gut epithelial cells showed <br /> lipofsucin granules. Lipofuscins, known &#226;&#8364;&#732;&#226;&#8364;&#732;age pigment&#226;&#8364;&#8482;&#226;&#8364;&#8482;,        high levels expression relative regions contain primarily <br /> aggregates cross-linked protein lipids lysosomes            neurons; regions showed little increased expression <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              2                       September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                 C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br /> Figure 1. Mortality movement ICD-1 ICD-2-depleted animals heat stress. Developmentally synchronized wild-type (N2) <br /> C. elegans larvae fed icd-1 icd-2(RNAi)-specific bacteria 30 hours exposed continuous heat stress 36uC 850 minutes. A) <br /> Survival animals measured periodic assessment pharyngeal pumping. B) Movement measured periodic assessment escape <br /> reaction physical prodding. N = 50 populations tested experiment. Data presented A) B) represent results of <br /> two independent trials experiment. The experimental populations exposed heat (20uC) measure basal rates of <br /> mortality effects movement, showed inherent lethality immobility (N = 50 population) (data shown). <br /> doi:10.1371/journal.pone.0044038.g001 <br />  <br /> (Figure 3A&#226;&#8364;&#8220;H). Many embryos displayed phenotypes                   Animals lacking XBP-1 increased sensitivity loss <br /> consistent previous icd-1(RNAi) results: severe morphological          ICD-1 <br /> defects increases cell death degeneration (Figure 3A, E)           Up-regulation hsp-4 absence ICD-1 consistent with <br /> [21].                                                                       increase misfolded protein ER initiation the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                        September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                       C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br />                                                                                   Figure 3. Localization hsp-4 expression embryos depleted <br />                                                                                   ICD-1. Animals containing hsp-4::GFP expression vector were <br />                                                                                   treated icd-1(RNAi) 48&#226;&#8364;&#8220;56 hours, embryos were <br />                                                                                   assessed GFP expression. Localization hsp-4::GFP signal was <br />                                                                                   determined independent experiments, totaling 30 GFP-positive <br />                                                                                   embryos assessed. All embryos showed similar localization patterns. In <br />                                                                                   time period RNAi treatment, majority embryos are <br />                                                                                   morphologically defective high levels cell death. A&#226;&#8364;&#8220;C) DIC, GFP <br />                                                                                   merged images, respectively, representative icd-1(RNAi) <br /> Figure 2. Embryonic expression hsp-4 ICD-1-depleted                         embryo ,470 minutes post fertilization. D) Diagram embryo <br /> animals. Animals containing hsp-4::GFP expression vector fed              470 minutes post fertilization. E&#226;&#8364;&#8220;G) DIC, GFP merged images, <br /> (A) icd-1(RNAi)-specific bacteria (B) OP50 (E. coli) bacteria expressing       respectively, representative icd-1(RNAi) embryo ,360 minutes <br /> no double stranded RNA 36 hours progeny embryos                post-fertilization. H) Diagram embryo 360 minutes post fertilization. <br /> randomly assessed expression GFP. C) Two examples embryos               In D) H) green cells represent location neurons, red cells <br /> selected icd-1(RNAi) population. Both embryos viable                represent location developing epithelial layer. Diagrams D) H) <br /> and morphologically wild-type. D) Average GFP signal hsp-4::GFP-               adapted Chin-sang Chisholm, 2000 [45]. <br /> containing embryos treated icd-1(RNAi) compared GFP signal              doi:10.1371/journal.pone.0044038.g003 <br /> generated control hsp-4::GFP-containing embryos. Populations of <br /> embryos chosen random, embryos given field                responses closely associated icd-1(RNAi)-induced phenotypes, <br /> observation assessed averaged regardless GFP status. Up-              up-regulation chaperone expression the <br /> regulation hsp-4::GFP observed independent experi-                initiation apoptosis. If increased expression hsp-4 icd- <br /> ments, up-regulation quantified experiments. <br /> Error bars represent standard deviation mean GFP intensity of <br />                                                                                   1(RNAi) associated Ire-1-dependent stress response, we <br /> the embryos assessed experiments (two-tailed t-test, P-              hypothesized xbp-1(ko) animals sensitive the <br /> value = 0.01). E) Averaged line scans measuring pixel intensity        depletion ICD-1 deficient UPR. <br /> assessed icd-1(RNAi) embryos compared control embryos; vertical                 To determine effects ICD-1-depletion xbp-1(ko) <br /> bars represent standard deviation line pixels.                             animals versus wild-type, characterized basal rate of <br /> doi:10.1371/journal.pone.0044038.g002 <br />                                                                                   defective embryo generation xbp-1(ko) worms. This rate was <br />                                                                                   significant, lower rate defective embryo <br /> UPR. HSP-4 C. elegans homologue GRP78/BiP, ER-                         generation xbp-1(ko);icd-1(RNAi) animals (Figure 4A). When <br /> localized chaperone sensor misfolded protein responsible                   compared wild type;icd-1(RNAi) animals 20uC, xbp-1(ko);icd- <br /> for initiating UPR. To determine putative role UPR                 1(RNAi) higher rate mortality, concomitant <br /> in response loss NAC, performed icd-1(RNAi) assays                increase morphological defects cell death embryos of <br /> xbp-1 knockout (ko) animals. XBP-1 downstream effector                    surviving worms (Figure 4B,C). The rate defective embryo <br /> Ire-1, UPR component regulated GRP78/BiP [26,27]. Ire-1                      generation xbp-1(ko) worms depleted ICD-1 increased <br /> pathways particular initiate stress                  dramatically temperature dropped 18uC <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                    4                        September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                 C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br /> Figure 4. Increased sensitivity xbp-1(ko) animals depletion ICD-1. [hsp-4::GFP] xbp-1(ko) worms treated icd-1(RNAi) and <br /> characterized effects mortality embryonic development. A) Profile embryonic development [hsp-4::GFP] xbp-1(ko) animals untreated <br /> or treated icd-1(RNAi). Embryos scored phenotypically wild-type defective. The stage wild-type embryos determined <br /> primarily extent vermiform structure, e.g. early embryos showed tube-like formation, late embryos contained fully formed <br /> worms. Defective embryos morphologically abnormal high levels cell death. Error bars represent standard deviation mean <br /> frequency embryos developmental stage independent experiments. B) Survival curve [hsp-4::GFP] xbp-1(ko) wild-type animals <br /> during icd-1(RNAi) treatment day period; survival determined presence pharyngeal pumping. Error bars represent the <br /> standard deviation mean independent experiments. C,D) Percentage embryos characterized defective wild-type [hsp- <br /> 4::GFP] xbp-1(ko) animals treated icd-1(RNAi) 48&#226;&#8364;&#8220;56 hours grown 20uC (C) 18uC (D) independent experiments. E&#226;&#8364;&#8220;G) <br /> Representative defective embryos wild-type adults treated icd-1(RNAi) 48 hours. All embryos displayed moderate morphological <br /> defects increased levels cell death, display vermiform structure extent. H&#226;&#8364;&#8220;J) Representative defective embryos [hsp- <br /> 4::GFP] xbp-1(ko) embryos treated icd- 1(RNAi) 48 hours. All embryos showed severe morphological defects significant cell death, <br /> with detectable tube-like formations. <br /> doi:10.1371/journal.pone.0044038.g004 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              5                        September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                 C. elegans NAC Unfolded Protein Response <br />  <br />  <br /> (Figure 4D), indicating elements UPR         ICD-1-depleted embryos contain enlarged lipofuscin <br /> engaged response loss NAC, response            granules <br /> acutely temperature sensitive. The morphology defective                 During later stages icd-1(RNAi) treatment, wild type <br /> embryos presence absence XBP-1 differed                  xbp-1(ko) embryos contained abnormally high levels corpse- <br /> significantly; wild type;icd-1(RNAi) embryos showed morpholog-              like structures, phenotype previously observed ICD-1-depleted <br /> ical defects cell death, expected, higher                  embryos [21]. We observed populations large, bi- <br /> percentage discernible worm-like form (Figure 4E&#226;&#8364;&#8220;G)                  refringent, donut-like structures gut region embryos that <br /> relative xbp-1(ko);icd-1(RNAi) embryos, generally showed           previously described lysosomal organelles [30]. These <br /> no worm-like form, deformed stage                     structures observed untreated embryos, were <br /> developmentally (Figure 4H&#226;&#8364;&#8220;J). The small percentage xbp-                 fewer number smaller size (data shown). We <br /> 1(ko);icd-1(RNAi) embryos staged showed                    confirmed enlarged granules lysosomal nature using <br /> increased sensitivity early embryonic apoptosis relative              glo-1::GFP reporter strain. GLO-1 RAB GTPase expressed <br /> wild-type;icd-1(RNAi) populations; 22.3+/211 cell corpses                   gut known marker lysosome-related gut cell <br /> (n = 10) versus 11.75+/23.9 cell corpses (n = 28) respectively              granules [30]. In embryos, glo-1::GFP expression correlated <br /> (Figure 5A). In previous ICD-1 studies, control RNAi-treated                bi-refringent granules observed DIC channel <br /> worms displayed 6.3+/22.3 cell corpses embryo [21]. Corpse              (Figure 7A&#226;&#8364;&#8220;D). <br /> numbers xbp-1(ko);icd-1(RNAi) embryos comma stage                 Due bi-refringent properties gut localization, we <br /> development ranged 6 43, generally localized            hypothesized enlarged lysosomal structures specif- <br /> regions neuronal cell lineage, consistent apoptotic             ically lipofuscin granules, described Clokely et. al (Figure 8D) <br /> effects seen previous ICD-1 studies (Figure 5B&#226;&#8364;&#8220;K) [21]. We          [31]. Lipofuscins contain mixture aggregated proteins and <br /> observed increase corpse-like structures enlarged gut             lipids formed damaged organelles proteins that <br /> granules late embryogenesis. These cell deaths          improperly degrade lysosome. Lipofuscin known to <br /> in neuronal cell lineage, embryo,               generate unique auto-fluorescent signal likely presence <br /> including hypodermal gut cell regions, indicating             Schiff bases [23]. To confirm presence lipofuscin <br /> cell types xbp-1(ko) animals sensitive ICD-1-                granules, examined auto-fluorescent properties using two- <br /> depletion.                                                                  photon microscopy. Following methods described Yen et. <br />    xbp-1(ko) mutants sensitive loss NAC               al, observed two-photon induced auto-fluorescence that <br /> due, part, inability increase chaperone expression           localized enlarged, GLO-1-specific granules icd-1(RNAi) <br /> in response Ire-1 activation. Previous studies shown                embryos (Figure 8A,C, F,G) [32]. The lipofuscin granules were <br /> that hsp-4 expression dramatically reduced xbp-1(ko)                  identified transmission channel using binary contrast <br /> worms relative wild-type treatment tunicamycin,               enhancement (thresholding) (Figure 8 B,E,G). <br /> an ER-stress inducer [28]. In accordance, than <br /> two-fold increase hsp-4::GFP expression xbp-1(ko); icd-               Discussion <br /> 1(RNAi)-treated worms (Figure S1A,B), compared an <br /> almost five-fold increase wt:icd-1(RNAi) animals,                 Here, demonstrate C. elegans NAC exhibits many <br /> were compared untreated controls (Figure 2D), indicating                 characteristics translational chaperone. Depletion either <br /> that xbp-1(ko) worms up-regulate hsp-4 expression effi-                NAC subunit did diminish worm&#226;&#8364;&#8482;s viability presence <br /> ciently wild-type animals depleted                    heat stress, increased worm&#226;&#8364;&#8482;s movement relative wild- <br /> ICD-1.                                                                      type worms, indicating protective effect face protein- <br />                                                                             denaturing stress. Depletion ICD-1 up-regulated expression <br />                                                                             hsp-4, ER chaperone GRP78/BiP homologue associated <br /> Movement ICD-1-depleted animals partially                             UPR function. Furthermore, effects icd-1(RNAi) were <br /> dependent HSF-1                                                          pronounced xbp-1(ko) animals, consistent loss of <br />    Accumulation misfolded protein ER absence            NAC inducing ER stress initiation UPR wild-type <br /> the NAC consistent NAC&#226;&#8364;&#8482;s proposed role modulating                worms. hsf-1(ko);icd-1(RNAi) animals resistant heat <br /> the interaction signal recognition particle (SRP)               stress wild-type;icd-1(RNAi) animals, considerably more <br /> nascent polypeptides, preventing inappropriate localization              robust hsf-1(ko) worms exposed heat, indicating the <br /> proteins ER [11]. To determine compartments              stress response induced ICD-1-depletion partially dependent <br /> the cell experience misfolded protein stress triggered loss       cytosolic stress sensor HSF-1, stress responders <br /> the NAC, icd-1(RNAi) performed hsf-1 knockout strain              able rescue vitality heat-stressed HSF-1(2) animals. <br /> under heat stress. HSF-1 (heat shock factor 21) cytosolic              Finally, observed differential phenotypic effects ICD-1- <br /> detector misfolded protein stress well-characterized                 depletion corresponding specific anatomical regions cell <br /> transcription factor cytosolic chaperones [29]. If cytosolic            types, indicating different cells tissues unique stress <br /> chaperone expression contributes heat stress resistance             responses loss NAC. <br /> depletion NAC, hsf-1(ko);icd-1(RNAi) animals                The NAC individual subunits assigned many <br /> increased sensitivity heat stress relative wild-type; icd-            roles bNAC characterized transcription factor <br /> 1(RNAi) worms. We true; hsf-1(ko); icd-1(RNAi)             BTF3 1987 [33]. Recently, mounting evidence supports the <br /> worms mobile relative wild-type;icd-1(RNAi) popu-              NAC translational chaperone, role previously hypothesized <br /> lations stressed heat (Figure 6). However,              [1,12]. In eukaryotes, subunits NAC two <br /> hsf-1(ko);icd-1(RNAi) maintain movement strikingly higher           proteins interact nascent polypeptide emerges <br /> rate hsf-1(ko) animals alone. These results indicate              ribosomal complex. In prokaryotes, trigger factor (TF) is <br /> depletion NAC engages stress response absence            protein bind polypeptides, plays vital role de <br /> HSF-1that partially rescue movement worms                  novo protein folding bacteria [1,14]. Despite importance in <br /> heat stress.                                                                proteostasis, TF knockouts growth defects the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              6                        September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                    C. elegans NAC Unfolded Protein Response <br />  <br />  <br />                                                                                 compensatory activity redundant chaperones DNAk DNAj, <br />                                                                                 members HSP-70/40 family. Only TF chaperone <br />                                                                                 DnaK deleted does lethality result. Similar trigger <br />                                                                                 factor, yeast NAC associates ribosomal complex and <br />                                                                                 emerging polypeptide, complexes share sequence <br />                                                                                 homology, readily evident function similarly <br />                                                                                 [14]. Recent discoveries, however, indicate TF NAC <br />                                                                                 functionally related; yeast NAC knockouts growth <br />                                                                                 defects translational chaperone Ssb, HSP-70/40 <br />                                                                                 family member, eliminated, indicating that, loss of <br />                                                                                 TF bacteria, yeast compensate loss NAC with <br />                                                                                 redundant chaperone activity, protecting organism death <br />                                                                                 [12]. <br />                                                                                    If NAC(2) yeast maintain wild-type levels growth by <br />                                                                                 employing redundant chaperones, wondered C. elegans did <br />                                                                                 same. We hypothesized absence redundant <br />                                                                                 chaperone activity, NAC depletion induce toxic levels of <br />                                                                                 misfolded protein, resulting lethality. This hypothesis based, <br />                                                                                 part, previous findings showing gross morphological defects <br />                                                                                 increased cell death progeny icd-1(RNAi) worms, and <br />                                                                                 embryonic lethality icd-1 knockout mice, Drosophila C. elegans <br />                                                                                 [21,34,35]. These phenotypes inconsistent presence of <br />                                                                                 ICD-1-redundant activity; surprising icd-1 and <br />                                                                                 icd-2(RNAi)-treated animals survived wild-type <br />                                                                                 worms heat stress, icd-2(RNAi)-treated worms appeared <br />                                                                                 survive higher rate. More striking differences in <br />                                                                                 movement ICD-1 ICD-2-depleted animals relative wild- <br />                                                                                 type worms heat stress, indicating worms lacking the <br />                                                                                 NAC just surviving thriving conditions that <br />                                                                                 debilitated wild-type counterparts. One explanation this <br />                                                                                 increase vitality up-regulation chaperones NAC(2) <br />                                                                                 animals allow withstand heat stress readily than <br />                                                                                 NAC (+) worms. If so, NAC-depleted animals show <br />                                                                                 chaperone up-regulation. We observed response form of <br />                                                                                 increased hsp-4 expression. <br />                                                                                    The up-regulation hsp-4 expression NAC-depleted animals <br />                                                                                 specifically implicates generation misfolded protein stress in <br />                                                                                 ER, initiation UPR consequence NAC- <br />                                                                                 depletion. HSP-4 homologue GRP78/BiP, chaperone and <br />                                                                                 sensor ER misfolded protein stress. As ER-specific <br />                                                                                 chaperone, GRP78/BiP helps refold misfolded proteins this <br />                                                                                 organelle, engages ER-specific stress sensors carry <br />                                                                                 UPR functions [25,26,36]. An accumulation misfolded <br />                                                                                 proteins ER NAC(2) animals consistent NAC&#226;&#8364;&#8482;s <br />                                                                                 recently proposed role yeast cotranslational chaperone that <br />                                                                                 modulates interaction SRP nascent polypeptides. <br />                                                                                 Alamo et al. shown NAC appears associate a <br />                                                                                 vast majority translating ribosomes polypeptides and <br />                                                                                 association allows NAC regulate polypeptide interac- <br />                                                                                 tions SRP [11]. As such, absence NAC may <br />                                                                                 allow SRP bind localize greater number of <br /> Figure 5. Embryonic apoptosis xbp-1(ko) animals depleted                  polypeptides ER, including normally targeted <br /> ICD-1. [hsp-4::GFP] xbp-1(ko) animals treated icd-1(RNAi),        there, resulting overload ER protein folding systems and <br /> their comma-stage embryos scored apoptotic cell corpses. <br /> Developmental apoptosis occurs normally C. elegans <br />                                                                                 misfolded protein stress. As earlier discussed, NAC-deficient yeast <br /> embryogenesis, comma-stage development generates                   appear maintain viability activities chaperones that <br /> high number apoptotic cell corpses relative stages. A) The          redundant NAC function and/or aid protein <br /> number comma-stage embryos containing specific number cell              folding processes. In C. elegans, increase hsp-4 expression may <br /> corpses wild-type (n = 28) [hsp-4::GFP] xbp-1(ko) (n = 11) embryos       serve purposes, chaperone function and/ <br /> treated icd-1(RNAi). B&#226;&#8364;&#8220;F) Multiple focal planes comma-stage           up-regulation chaperones initiation UPR. <br /> embryo produced [hsp-4::GFP] xbp-1(ko) worm treated icd- <br /> 1(RNAi). Cell corpses distinguishable surrounding cells          The increased sensitivity xbp-1(ko) animals icd-1(RNAi) <br /> raised-up, button-like structures. G&#226;&#8364;&#8220;K) Duplicate images A&#226;&#8364;&#8220;E                 suggests loss NAC engages Ire-1 receptor the <br /> highlighting 43 cell corpses observed embryo.               UPR. Ire-1 induces stress responses separate <br /> doi:10.1371/journal.pone.0044038.g005                                           pathways activate XBP-1 Jun kinase (JNK) [16]. XBP-1 <br />                                                                                 transcription factor up-regulates chaperone expression <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  7                       September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                               C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br /> Figure 6. Effects ICD-1 depletion movement hsf-1(ko) animals exposed heat stress. hsf-1(ko) animals fed icd-1(RNAi)- <br /> expressing bacteria time 0 exposed heat stress 36uC 750 minutes. Movement worms measured 450 minutes heat <br /> stress escape response prodding, compared wild-type;icd-1(RNAi) animals heat stress (n = 400 population). All <br /> experimental populations showed 100% movement beginning time course (data shown). As control, strains worms were <br /> assessed movement 20uC (n = 400) (data shown). <br /> doi:10.1371/journal.pone.0044038.g006 <br />  <br /> and ER-associated protein degradation (ERAD),             indicate accumulation misfolded protein ER <br /> JNK controls induction autophagy apoptosis                   cytosol NAC(2) animals, inducing activities the <br /> response accumulation misfolded protein. If loss          UPR HSF-1. Again, results consistent previous <br /> NAC initiates Ire-1 signaling, data indicates, xbp-1(ko)            findings Alamo et al. suggest NAC controls the <br /> animals lack increased chaperone activity                    localization proteins cell. <br /> accelerated protein turnover provided functional XBP-1,                  In process identifying general cell stress response the <br /> but able initiate apoptosis functional JNK             loss NAC, noticed potential cell-type specific <br /> pathway. In context, xbp-1(ko);icd-1(RNAi) animals              responses NAC depletion. Tissue- cell-specific responses to <br /> accumulate misfolded protein rapidly relative wild-type worms           misfolded protein stress range pathways engaged <br /> and readily initiate apoptosis, consistent observa-          specifically response exogenous stress are <br /> tions xbp-1(ko);icd-1(RNAi) animals showed increased rates         essential normal differentiation [38]. In C. elegans, Kern et al. <br /> cell death overall mortality. The increased sensitivity xbp-        identified stress responses heat differ significantly <br /> 1(ko) worms absence NAC indicates              neurons muscle cells [39]. Specifically, found <br /> other arms C. elegans UPR. i.e. ATF6 Perk/Pek-1,            newly formed neurons heat stress showed delayed up- <br /> provide partial, complete, redundancy XBP-1. xbp-              regulation chaperone expression increased protein aggre- <br /> 1(ko);icd-1(RNAi) animals up-regulated hsp-4 relative untreated         gations relative young muscle cells, exhibited robust <br /> controls, level wild-type;icd-1(RNAi) worms,             stress response. In studies, highly reproducible lack hsp-4 <br /> consistent previous results up-regulation hsp-4       up-regulation neuronal regions icd-1(RNAi) embryos may <br /> C. elegans response depletion chaperones reliant        explain neurons sensitive ICD-1 depletion, as <br /> XBP-1 [22].                                                                measured increases cell death, relative cell types. <br />    The engagement ER-associated UPR NAC(2) animals               This result consistent findings Kern et al. show <br /> doesn&#226;&#8364;&#8482;t preclude contributions stress sensing pathways          suboptimal chaperone expression neurons undergoing mis- <br /> in response NAC-depletion, particularly light                 folded protein stress. On hand, regions embryo <br /> possibility NAC controls localization proteins             containing gut epidermal cells consistently displayed strong <br /> throughout cell. We vast majority hsf-1(ko) animals         hsp-4 up-regulation, gut cells specifically showed unique <br /> immobile presence heat, consistent role up-             stress response loss NAC, generation large <br /> regulating cytosolic chaperones stress, worms             lysosomal structures we&#226;&#8364;&#8482;ve putatively identified lipofuscin <br /> became dramatically mobile depleted ICD-1,                    granules. <br /> revealing robust stress response absence HSF-1 [37].              The generation lipofuscin granules gut cells NAC(2) <br /> hsf-1(ko);icd-1(RNAi) animals were, however, mobile        animals causally link structures UPR-mediated <br /> wild-type;icd-1(RNAi) worms exposed heat, HSF-1                  autophagy, process shuttles misfolded proteins ER <br /> contribute portion stress response triggered absence       lysosomal structures relieve overburdened ERAD [40]. <br /> of NAC. These results suggest animals complete UPR              Lipofuscins chemical by-product incomplete <br /> functional HSF-1 sensitive loss NAC            degradation proteins lipids lysosome, forming a <br /> animals missing stress response element. If so, results       polymeric non-degradable substance exhibits unique auto- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             8                       September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                    C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br /> Figure 7. Generation GLO-1-positive lysosomal structures icd-1(RNAi) embryos. Animals containing glo-1::GFP expression vector <br /> were treated icd-1(RNAi) 48&#226;&#8364;&#8220;56 hours resulting embryos scored presence large lysosomal structures gut cells. <br /> GLO-1 marker C. elegans lysosomes distinguishable bio-fluorescence polarized light. A majority embryos (n = 20) <br /> displayed glo-1::GFP signal coincident lysosomal structures. A&#226;&#8364;&#8220;D) A representative [glo-1::GFP] icd-1(RNAi) embryo visualized A) fluorescent <br /> light; B) fluorescent light DIC; C) fluorescent light polarized DIC D) polarized DIC. White arrows indicate lysosomes visualized DIC with <br /> overlapping glo-1::GFP signal, red ovals outline shell embryo. <br /> doi:10.1371/journal.pone.0044038.g007 <br />  <br /> fluorescent properties [23]. Since cellular materials brought           apoptosis gut cell lipofuscins. Many questions concerning C. <br /> the lysosome autophagic mechanisms, lipofuscin formation                   elegans NAC remain, including scope stress response <br /> necessarily depends type autophagy. UPR-mediated                    initiated depletion NAC, aspects UPR are <br /> autophagy mammalian cells dependent expression                 engaged, stress responses contributing. <br /> GRP78/BiP, previous studies observed abnormal                         Also great activities NAC subunit the <br /> lysosomal formation C. elegans chemically induced ER                  absence other. Both subunits assigned individual <br /> stress. We observed large lipofuscin granules NAC(2) gut cells              functions, likely depletion subunit frees other <br /> presumably undergoing misfolded protein stress, resulting               affect cellular function way [10]. We fully <br /> up-regulation hsp-4 engagement UPR;                       understand outcomes resulting loss subunit <br /> postulate lipofuscin granules direct result UPR-              understanding remaining subunit functions. <br /> mediated autophagy shuttling misfolded proteins,                   Ultimately, understanding NAC function normal <br /> damaged ER fragments, lysosomal structures.                           conditions times stress essential our <br />    We presented evidence C. elegans NAC displays                understanding proteostasis, likely provide valuable <br /> characteristics translational chaperone absence triggers            insights pathology diseases associated aberrant <br /> the UPR, resulting up-regulation chaperone expression,               protein folding aggregation, particularly neurodegeneration. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 9                        September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                 C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  <br />  <br /> Figure 8. Identification lipofuscin granules icd-1(RNAi) embryos. Wild-type embryos treated icd-1(RNAi) scored the <br /> presence lipofuscin granules using two-photon microscopy. A&#226;&#8364;&#8220;C) representative icd-1(RNAi) embryo visualized A) transmission; B) thresholded <br /> transmission and; C) two-photon auto-fluorescence. Transmission microscopy used visualize large lysosomal structures gut cells, <br /> thresholded transmission used visualize saturated pixels correlated lysosomal structures two-photon excitation used visualize <br /> specifically lipofuscin granules using 500 nm bandwidth filter. D&#226;&#8364;&#8220;F) magnified regions A&#226;&#8364;&#8220;C respectively, demarcated red border. G) An <br /> overlay images D&#226;&#8364;&#8220;F. Arrows point auto-flourescent lipofuscin granules. <br /> doi:10.1371/journal.pone.0044038.g008 <br />  <br /> aNAC down-regulated Alzheimer&#226;&#8364;&#8482;s disease, recent                    (pH 6), 1.5 mL 1 M MgSO4, 15 uL 0.1 M IPTG and <br /> research shown exogenous amyloid aggregates expressed               1.5 mL 25 mg/mL carbenicillin added. Seeded plates <br /> in human cells preferentially associate subproteome,                  dried incubator 24 hours 37uC. <br /> including proteins involved proteostasis; NAC in <br /> these aggregates [41,42]. Using genetically tractable           Mobility survival assays heat stress <br /> characterized model C. elegans, valuable insights metazoan              Gravid adult wild type worms synchronized bleaching <br /> NAC function continue gained. These insights                 worms 1:1 NaOH bleach. Worms bleached 10 uL <br /> contribute significantly understanding human NAC               lysis solution perimeter medium sized OP50 plates. <br /> and roles proteostasis healthy stressed cells.                 28&#226;&#8364;&#8220;30 hrs later synchronized F1 worms moved their <br />                                                                              respective experimental plates. Worms moved 3 plate <br /> Materials Methods                                                        types: 1) icd-1(RNAi) 2) icd-2(RNAi) 3) OP50 bacteria. 50 worms <br />                                                                              moved plate type. Plate types duplicate <br /> Nematode strains                                                             order temperature controls maintained 20uC. All <br />    C. elegans strains maintained 20uC standard methods            worm populations grown 20uC 30 hours. After <br /> [43]. N2 (wild-type), hsp-4::GFP (SJ4005), hsp-16::GFP (CL2070), <br />                                                                              30 hours growth, heat shock populations moved 36uC <br /> hsp-6::GFP (SJ4100), hsp-60::GFP (SJ4058), [hsp-4::GFP] xbp-1(ko) <br />                                                                              incubator. Worms heat shocked continuously 6 hours. <br /> (SJ17), hsf-1(ko) (PS3551), glo-1::GFP (VS17) provided <br />                                                                              Sampling began 6 hours heat exposure behavioral <br /> by Caenorhabditis Genetics Center. <br />                                                                              heat shock effects measurable [29]. After 6 hours <br />                                                                              heat shock, worms periodically removed the <br /> RNA interference                                                             incubator prodded platinum pick examine mobility <br />    RNA interference performed feeding C. elegans double               survival. Responses recorded movement/survival or <br /> stranded RNA correlating target genes icd-1 icd-2,           lack movement/survival. Worms sampled hour. <br /> the unc-22 control. Feeder bacteria obtained Addgene,              Movement measured displacement escape reaction to <br /> and grown single colonies liquid LB shaking overnight             physical prodding. Survival measured presence of <br /> at 37uC. During 4 hours shaking IPTG added                pharyngeal pumping stimulus. <br /> the overnight culture final concentration 20 uM. After <br /> overnight shaking, cultured feeder seeded RNAi plates. <br /> RNAi plates combining 4.5 g NaCl, 25.5 g agar,                  Monitoring chaperone reporters icd-1(RNAi) <br /> 3.75 g peptone, 1.5 mL 2 mg/mL uracil, 0.22 g CaCl, 0.75 g                  L4s following strains SJ4005, CL2070, SJ4100 and <br /> of 10 mg/ml cholesterol 1463 mL dH2O. This mixture                 SJ4058 moved icd-1(RNAi)or OP50 control plates. Worms <br /> autoclaved, allowed cool, 37.5 mL phosphate buffer                 grown 20uC 48 hours. Adults transferred fresh <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              10                       September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                              C. elegans NAC Unfolded Protein Response <br />  <br />  <br /> plates 24 hours differentiate newly hatched embryos. After                   feeder plates 48 hours examined. Morphologically <br /> 48 hours, randomly chosen embryos mounted glass slides                        defective embyros mounted glass slides viewed with <br /> and viewed Nikon C1 confocal microscope fluorescence.                      DIC widefield flourescent optics using Nikon C1 Confocal <br />                                                                                       microscope. DIC GFP flourescent channels overlaid <br /> Quantification hsp-4::GFP signal                                                   NIS elements software. Polarized DIC images captured <br />    Fluorescent images embryos expressing hsp-4::GFP                           setting polarizer extinction. <br /> captured Nikon C1 confocal microscope. Images were <br /> captured using brightfield settings analyzed NIS                             Two-Photon imaging analyses <br /> Elements software. Fluorescent intensity measured                           Embryos worms fed icd-1(RNAi) 48 hrs mounted <br /> line-scan measurement function. The line-scan measurement tool                        agar pads M9 solution, sealed nail polish remover and <br /> was centered oriented long-axis embryos. This                        imaged Zeiss 510 Meta System coupled Axioplan <br /> measurement outputs fluorescent intensity (in arbitrary units)                    200 M epifluorescence microscope hours mounting. <br /> each pixel line-scan. Data exported Microsoft Excel                   Embryos viewed 636 Oil objective light <br /> and average intensity calculated OP50 icd-1(RNAi)                     transmission TPEF channels. The TPEF signal captured <br /> treatments. After random selection, 16 wild type 13 xbp-1(ko)                     500/86 nm bandpass filter detected photo- <br /> embryos scanned quantified.                                                  multiplier tube. LSM images imported Image J using <br />                                                                                       LSM Reader LSM Toolbox. Dual channel images were <br /> Lifespan assay                                                                        separated stacks images function pixel window <br />   Pretzel-staged wild type [hsp-4::GFP] xbp-1(ko) embryos                         adjusted 160 remove background noise TPEF <br /> were moved icd-1(RNAi) plates percent survival                          (green) channel. <br /> worms determined daily. Survival measured prodding <br /> worms manually platinum pick viewing pharyngeal <br />                                                                                       Supporting Information <br /> pumping. To determine baseline viability, control xbp-1(ko) and <br /> wild type strains grown OP50. Controls remained viable                        Figure S1 Embryonic expression hsp-4 xbp-1(ko) <br /> throughout measurement period. Error bars represent                           animals depleted ICD-1. [hsp-4::GFP] xbp-1(ko) animals <br /> variation independent experiments n = 30.                                 fed icd-1(RNAi)-expressing bacteria OP50 (E. coli) bacteria <br />                                                                                       expressing double stranded RNA 36 hours their <br /> Embryo imaging                                                                        progeny embryos randomly assessed expression GFP. <br />   Embryo images (Figures 3, 4, 5) imported GIMP                         Two independent experiments displayed hsp-4::GFP up-regulation <br /> image free ware. GFP DIC images overlain embryo                          experimental population, quantified levels of <br /> outlines cropped embryo shell using intelligent                    GFP expression relative control population. A) Average <br /> edge cropping function GIMP. Image panels compiled                         GFP signal [hsp-4::GFP] xbp-1(ko) embryos treated icd- <br /> Inkscape vector image free ware. Apoptotic cells embryo shells                    1(RNAi) compared GFP signal generated control embryos <br /> were identified manually highlighted Inkscape (red black                    (n = 8 population). Populations embryos chosen at <br /> outlines).                                                                            random. Error bars represent standard deviation mean <br />                                                                                       GFP intensity embryos. B) DIC GFP overlapped images of <br /> Profiling [hsp-4::GFP] xbp-1(ko) embryos                                           representative [hsp-4::GFP] xbp-1(ko) control embryos (inset) or <br />    [hsp-4::GFP] xbp-1(ko) L4s transferred OP50 icd-                       treated icd-1(RNAi). <br /> 1(RNAi) plates. Embryos monitored microscope                           (TIF) <br /> after 48 hours feeding. Embryo profiles generated by <br /> classifying embryos morphologically 1006 magnification using                       Acknowledgments <br /> DIC optics early, mid, late defective. The stage wild-type <br /> embryos determined primarily extent vermiform                       We like thank Alex Bannigan James Madison University and <br />                                                                                       Ammasi Periasamy W.M. Keck Center Cellular Imaging the <br /> structure, e.g. early embryos showed tube-like formation, while <br />                                                                                       University Virginia help microscopy; Ann Rougvie and <br /> late embryos contained fully formed worms. Defective embryos                          Aric Daul C. elegans strains provided Caenorhabditis Genetics <br /> were morphologically abnormal high levels cell death.                         Center, funded NIH Office Director; Kim Slekar Caylin <br /> Error bars represent standard deviation independent                        Murray comments manuscript; members Bloss Slekar <br /> experiments.                                                                          labs discussions, Robert Rotzin technical support. <br />  <br /> Assessment embryonic apoptosis                                                     Author Contributions <br />   Cell corpses embryos comma stage embryogenesis <br />                                                                                       Conceived designed experiments: PTA TAB. Performed the <br /> were quantified using DIC optics described [44].                                   experiments: PTA TAB ATM VDC. Analyzed data: PTA TAB ATM <br />                                                                                       VDC. Contributed reagents/materials/analysis tools: PTA TAB ATM <br /> Lysosome imaging                                                                      VDC. Wrote paper: PTA TAB. <br />   To determine identity bi-refringent objects icd- <br /> 1(RNAi) embryos, glo-1::GFP reporter worms grown on <br />  <br /> References <br /> 1. Hartl FU, Bracher A, Hayer-Hartl M (2011) Molecular chaperones protein           4. Hetz C (2012) The unfolded protein response: controlling cell fate decisions <br />    folding proteostasis. Nature 475: 324&#226;&#8364;&#8220;332.                                         ER stress beyond. Nat Rev Mol Cell Biol 13: 89&#226;&#8364;&#8220;102. <br /> 2. Voisine C, Pedersen JS, Morimoto RI (2010) Chaperone networks: tipping          5. Vabulas RM, Raychaudhuri S, Hayer-Hartl M, Hartl FU (2010) Protein folding <br />    balance protein folding diseases. Neurobiology Disease 40: 12&#226;&#8364;&#8220;20.                cytoplasm heat shock response. Cold Spring Harb Perspect Biol 2: <br /> 3. Waldmeier PC (2003) Prospects antiapoptotic drug therapy neurodegen-            1&#226;&#8364;&#8220;18. <br />    erative diseases. Prog Neuropsychopharmacol Biol Psychiatry 27: 303&#226;&#8364;&#8220;321. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       11                           September 2012 | Volume 7 | Issue 9 | e44038 <br />                                                                                                                      C. elegans NAC Unfolded Protein Response <br />  <br />  <br />  6. Wegrzyn RD, Deuerling E (2005) Molecular guardians newborn proteins:                  26. Pincus D, Chevalier MW, Aragon T, van Anken E, Vidal SE, et al. (2010) BiP <br />     ribosome-associated chaperones role protein folding. Cell Mol Life               binding ER-stress sensor Ire1 tunes homeostatic behavior the <br />     Sci 62: 2727&#226;&#8364;&#8220;2738.                                                                            unfolded protein response. PLoS Biology 8: e1000415. <br />  7. Gautschi M, Mun A, Ross S, Rospert S (2002) A functional chaperone triad               27. Bertolotti A, Zhang Y, Hendershot LM, Harding HP, Ron D (2000) Dynamic <br />     yeast ribosome. Proc Natl Acad Sci USA 99: 4209&#226;&#8364;&#8220;4214.                                     interaction BiP ER stress transducers unfolded-protein response. <br />  8. Wiedmann B, Sakai H, Davis TA, Wiedmann M (1994) A protein complex                            Nat Cell Biol 2: 326&#226;&#8364;&#8220;332. <br />     required signal-sequence-specific sorting translocation. Nature 370: 434&#226;&#8364;&#8220;         28. Calfon M, Zeng H, Urano F, Till JH, Hubbard SR, et al. (2002) IRE1 couples <br />     440.                                                                                          endoplasmic reticulum load secretory capacity processing XBP-1 <br />  9. Franke J, Reimann B, Hartmann E, Kohlerl M, Wiedmann B (2001) Evidence                        mRNA. Nature 415: 92&#226;&#8364;&#8220;96. <br />     nuclear passage nascent polypeptide-associated complex subunits               29. Walker GA, Thompson FJ, Brawley A, Scanlon T, Devaney E (2003) Heat shock <br />     yeast. J Cell Sci 114: 2641&#226;&#8364;&#8220;2648.                                                             factor functions convergence stress response developmental <br /> 10. Rospert S, Dubaqui Y, Gautschi M (2002) Nascent-polypeptide-associated                        pathways Caenorhabditis elegans. FASEB J 17: 1960&#226;&#8364;&#8220;1962. <br />     complex. Cell Mol Life Sci 59: 1632&#226;&#8364;&#8220;1639.                                                 30. Hermann GJ, Schroeder LK, Hieb CA, Kershner AM, Rabbitts BM, et al. <br /> 11. del Alamo M, Hogan DJ, Pechmann S, Albanese V, Brown PO, et al. (2011)                        (2005) Genetic analysis lysosomal tracking Caenorhabditis elegans. Mol <br />     Defining specificity cotranslationally acting chaperones systematic                 Biol Cell 16: 3273&#226;&#8364;&#8220;3288. <br />     analysis mRNAs associated ribosome- nascent chain complexes. PLoS                 31. Clokey GV, Jacobson LA (1986) The autouorescent &#226;&#8364;&#732;&#226;&#8364;&#732;lipofuscin granules&#226;&#8364;&#8482;&#226;&#8364;&#8482; the <br />     Biol 9: e1001100.                                                                             intestinal cells Caenorhabditis elegans secondary lysosomes. Mech Ageing <br /> 12. Koplin A, Preissler S, Ilina Y, Koch M, Scior A, et al. (2010) A dual function            Dev 35: 79&#226;&#8364;&#8220;94. <br />     chaperones SSB-RAC NAC nascent polypeptide-associated complex                  32. Yen K, Le TT, Bansal A, Narasimhan SD, Cheng JX, et al. (2010) A <br />     ribosomes. J Cell Biol 189: 57&#226;&#8364;&#8220;68.                                                            comparative study fat storage quantitation nematode Caenorhabditis <br /> 13. Otto H, Conz C, Maier P, Wolfle T, Suzuki CK, et al. (2005) The chaperones                    elegans using <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> label-free methods. PloS ONE 5. e12810 <br />     MPP11 Hsp70L1 form mammalian ribosome-associated complex. Proc                    33. Zheng XM, Moncollin V, Egly JM, Chambon P (1987) A general transcription <br />     Natl Acad Sci USA 102: 10064&#226;&#8364;&#8220;10069.                                                           factor forms stable complex RNA polymerase B (II). Cell 50: 361&#226;&#8364;&#8220;368. <br /> 14. Preissler S, Deuerling E (2012) Ribosome-associated chaperones key players             34. Deng JM, Behringer RR (1995) An insertional <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> BTF3 <br />     proteostasis. Trends Biochem Sci 37: 274&#226;&#8364;&#8220;283. <br />                                                                                                   transcription factor gene leads early postimplantation lethality mice. <br /> 15. Mori K (2009) Signalling pathways unfolded protein response:                           Transgenic Res 4: 264&#226;&#8364;&#8220;269. <br />     development yeast mammals. J Biochem 146: 743&#226;&#8364;&#8220;750. <br />                                                                                               35. Markesich DC, Gajewski KM, Nazimiec ME, Beckingham K (2000) bicaudal <br /> 16. Hetz C, Glimcher LH (2009) Fine-tuning unfolded protein response: <br />                                                                                                   encodes Drosophila beta NAC homolog, component ribosomal <br />     Assembling IRE1alpha interactome. Mol Cell 35: 551&#226;&#8364;&#8220;561. <br />                                                                                                   translational machinery*. Development 127: 559&#226;&#8364;&#8220;572. <br /> 17. Doyle KM, Kennedy D, Gorman AM, Gupta S, Healy SJ, et al. (2011) Unfolded <br />                                                                                               36. Shen X, Ellis RE, Lee K, Liu CY, Yang K, et al. (2001) Complementary <br />     proteins endoplasmic reticulum stress neurodegenerative disorders. J Cell <br />                                                                                                   signaling pathways regulate unfolded protein response required for <br />     Mol Med 15: 2025&#226;&#8364;&#8220;2039. <br />                                                                                                   C. elegans development. Cell 107: 893&#226;&#8364;&#8220;903. <br /> 18. Sulston JE, Schierenberg E, White JG, Thomson JN (1983) The embryonic cell <br />     lineage nematode Caenorhabditis elegans. Dev Biol 100: 64&#226;&#8364;&#8220;119.                     37. Hsu AL, Murphy CT, Kenyon C (2003) Regulation aging age-related <br /> 19. David DC, Ollikainen N, Trinidad JC, Cary MP, Burlingame AL, et al. (2010)                    disease DAF-16 heat-shock factor. Science 300: 1142&#226;&#8364;&#8220;1145. <br />     Widespread protein aggregation inherent aging C. elegans. PLoS           38. Ma Y, Hendershot LM (2004) ER chaperone functions normal stress <br />     Biology 8: e1000450.                                                                          conditions. J Chem Neuroanat 28: 51&#226;&#8364;&#8220;65. <br /> 20. Silva MC, Fox S, Beam M, Thakkar H, Amaral MD, et al. (2011) A genetic                    39. Kern A, Ackermann B, Clement AM, Duerk H, Behl C (2010) HSF1-controlled <br />     screening strategy identifies novel regulators proteostasis network. PLoS              age-associated chaperone capacity neurons muscle cells C. elegans. <br />     Genet 7: e1002438.                                                                            PLoS ONE 5: e8568. <br /> 21. Bloss TA, Witze ES, Rothman JH (2003) Suppression CED-3-indepen <br /> </body></html>