<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Single methyl group determines prion propagation and <br /> protein degradation activities of yeast heat shock <br /> protein (Hsp)-70 chaperones Ssa1p and Ssa2p <br />  <br /> Organisms encode multiple homologous heat shock protein (Hsp)-                  ogous Hsp70 protects animals <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> various stresses (5, 6). <br /> 70s, which are essential protein chaperones that play the major                 Nevertheless, even highly homologous isoforms display clearly <br /> role in cellular protein &#226;&#8364;&#339;quality control.&#226;&#8364;&#65533; Although Hsp70s are func-           distinct functions (4, 7&#226;&#8364;&#8220;11). Whereas the ampli&#239;&#172;&#65533;cation of Hsp70 <br /> tionally redundant and highly homologous, many possess distinct                 genes has long been considered advantageous for allowing reg- <br /> functions. A regulatory motif underlying such distinctions, how-                ulation of spatiotemporal abundance of Hsp70 according <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ever, is unknown. The 98% identical cytoplasmic Hsp70s Ssa1p and                cellular need for its chaperone function, the ability of Hsp70 to <br /> Ssa2p function differently with regard to propagation of yeast                  function distinctly implies that expansion of Hsp70 genes also <br /> [URE3] prions and in the vacuolar-mediated degradation of gluco-                provides an advantage as a source of functional diversity. Subtle <br /> neogenesis enzymes, such as FBPase. Here, we show that the                      differences in intrinsic or cochaperone-regulated activities likely <br /> Hsp70 nucleotide binding domain (NBD) regulates these functional                allow certain Hsp70 isoforms to perform better than others in <br /> speci&#239;&#172;&#65533;cities. We &#239;&#172;&#65533;nd little difference in ATPase, protein refolding,            speci&#239;&#172;&#65533;c tasks. Despite intensive study, however, the presence of a <br /> and amyloid inhibiting activities of puri&#239;&#172;&#65533;ed Ssa1p and Ssa2p, but               regulatory element within Hsp70 that can confer such speciali- <br /> show that interchanging NBD residue alanine 83 (Ssa1p) and gly-                 zation has not been described. <br /> cine 83 (Ssa2p) switched functions of Ssa1p and Ssa2p in [URE3]                    The S. cerevisiae Ssa subfamily of cytosolic Hsp70 is composed <br /> propagation and FBPase degradation. Disrupting the degradation                  of the constitutively expressed Ssa1p and Ssa2p and the stress- <br /> pathway did not affect prion propagation, however, indicating                   inducible Ssa3p and Ssa4p (12). Amino acid identity within this <br /> these are two distinct processes where Ssa1/2p chaperones func-                 family is highest between isoforms expressed under similar con- <br /> tion differently. Our results suggest that the primary evolutionary             ditions. Constitutively expressed Ssa1p and Ssa2p are 98% iden- <br /> pressure for Hsp70 functional distinctions is not to specify inter-             tical, inducible Ssa3p and Ssa4p are 88% identical, and Ssa3/4p <br /> actions of Hsp70 with substrate, but to specify the regulation of               are 80% identical to Ssa1/2p. Ssa family Hsp70 function is es- <br /> this activity. Our data suggest a rationale for maintaining multiple            sential for viability, and any of these four isoforms can support <br /> Hsp70s and suggest that subtle differences among Hsp70s evolved                 cell growth if expressed abundantly enough (13, 14). <br /> to provide functional speci&#239;&#172;&#65533;city without affecting overall enzy-                   Despite their near identity, Ssa1p and Ssa2p have clear dif- <br /> matic activity.                                                                 ferences in physiologic activity. They were &#239;&#172;&#65533;rst shown to differ in <br />                                                                                 the vacuolar import and degradation (Vid) pathway, which <br />                                                                                 speci&#239;&#172;&#65533;cally requires Ssa2p for degradation of gluconeogenic <br /> H    eat shock protein (Hsp)-70 molecular chaperones are <br />      among the most highly conserved proteins. They act through <br /> an ATP-regulated cycle of binding and release of hydrophobic <br />                                                                                 enzymes after addition of glucose to starved cells (15). Ssa Hsp70 <br />                                                                                 isoforms also function distinctly with regard to yeast prions. <br /> surfaces on incompletely folded proteins. This interaction helps                Prions are infectious amyloid-like aggregates of misfolded cel- <br /> other proteins adopt and maintain native conformation and                       lular proteins that propagate by inducing similar misfolding of <br /> allows Hsp70 to play a central role in many essential processes                 the native form of protein. Yeast [URE3] prions, which propa- <br /> including protein traf&#239;&#172;&#65533;cking, signaling, and degradation (1&#226;&#8364;&#8220;3).                 gate as amyloid forms of Ure2p, are unaffected by depletion of <br /> Due to its versatile roles, Hsp70 is a promising therapeutic target             Ssa1p but are lost frequently from cells lacking Ssa2p (16, 17). <br /> in several human diseases such as cancer and amyloid-based                      Also, increasing abundance of Ssa1p in wild-type cells causes <br /> neurodegenerative diseases and diabetes. Not surprisingly, there                them to lose [URE3], whereas increasing Ssa2p does not (18). <br />  <br />  <br />  <br />                                                                                                                                                                            GENETICS <br /> is a broad interest in understanding the molecular basis of                        Here, we used this distinction in prion <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> to identify <br /> Hsp70 action.                                                                   the structural elements of Hsp70 that regulate functional dif- <br />    Hsp70 activity is regulated by communication among its                       ferences between Ssa1p and Ssa2p. After determining that the <br /> amino-terminal ATPase domain (nucleotide binding domain,                        ATPase domain was responsible, we show a single amino acid in <br /> NBD), an adjacent substrate binding domain (SBD), and a less                    this domain is all that is necessary for the functional distinction <br /> conserved carboxyl-terminal region (CTD), which forms a lid                     between Ssa1p and Ssa2p with regard to propagation of [URE3]. <br /> over the binding pocket of the SBD and interacts with a variety                 Remarkably, this same residue speci&#239;&#172;&#65533;ed Ssa2p function in pro- <br /> of cochaperones. Binding of ATP to the NBD opens the lid and                    tein degradation mediated by the Vid pathway. This pathway was <br /> allows rapid reversible binding of substrate. ATP hydrolysis                    not important for [URE3] propagation, however, indicating the <br /> induces conformational changes that close the lid, trapping                     same minimal change in Hsp70 can specify functional differences <br /> substrate within the binding pocket. Release of ADP and res-                    in different processes. <br /> toration of the ATP-bound &#226;&#8364;&#339;open&#226;&#8364;&#65533; state promotes substrate re- <br /> lease. Substrate binds with high af&#239;&#172;&#65533;nity to the ADP bound state, <br /> whereas ATP binding decreases this af&#239;&#172;&#65533;nity by approximately                     Author contributions: D.S. and D.C.M. designed research, performed research, contrib- <br /> two orders of magnitude.                                                        uted new reagents/analytic tools, analyzed data, and wrote the paper. <br />    Most organisms encode multiple Hsp70 isoforms. Humans                        The authors declare no con&#239;&#172;&#8218;ict of interest. <br /> and Saccharomyces cerevisiae each have six canonical cytosolic                  *This Direct Submission article had a prearranged editor. <br /> Hsp70s. Their functions overlap to the extent that human Hsp70                  1 <br />                                                                                 To whom correspondence should be addressed. E-mail: masisond@helix.nih.gov. <br /> (Hsc70) can support viability of yeast lacking its essential Ssa                This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10. <br /> Hsp70 subfamily (4). Similarly, exogenously expressed heterol-                  1073/pnas.1107421108/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                    PNAS | August 16, 2011 | vol. 108 | no. 33 | 13665&#226;&#8364;&#8220;13670 <br />  Results                                                                          denaturing it to less than 1% activity by treatment with guani- <br /> Functional Difference Between Ssa1p and Ssa2p Is Not Apparent in                 dinium chloride. Adding Ydj1p restored only 10% of the original <br /> Their Biochemical Activities. Previously, we and others showed that              luciferase activity (Fig. 1C). Coincubating denatured luciferase <br /> [URE3] propagation is inhibited by increasing Ssa1p or de-                       and Ydj1p with a 10-fold molar excess of either Ssa1p or Ssa2p, <br /> pleting Ssa2p, but not by increasing Ssa2p or depleting Ssa1p                    however, resulted in nearly complete restoration of luciferase <br /> (16&#226;&#8364;&#8220;19). We assessed functions of puri&#239;&#172;&#65533;ed Ssa1p and Ssa2p                        activity. The rates of refolding assisted by Ssa1p and Ssa2p were <br /> in vitro to determine whether these differences might be due to                  nearly identical. Therefore, these Hsp70s did not differ in <br /> differences in their biochemical activities. The kinetics of binding             functional cooperation with Ydj1p to refold denatured protein. <br /> and release of substrates by Hsp70 depend upon its ATP cycle.                       When puri&#239;&#172;&#65533;ed, Ure2p spontaneously assembles into amyloid <br />                                                                                  &#239;&#172;&#65533;laments, and this assembly is inhibited by addition of puri&#239;&#172;&#65533;ed <br /> Intrinsic ATPase activity of various Hsp70s is very low, ranging <br />                                                                                  Ssa1p (23, 24). In our hands assembly of puri&#239;&#172;&#65533;ed Ure2p into <br /> from about 0.01 to 1 ATP hydrolyzed per minute (20). The <br />                                                                                  amyloid showed a sigmoidal curve with a very short lag phase of <br /> ATPase activity of both Ssa1p and Ssa2p were found to be 0.02/ <br />                                                                                  about 5 min (Fig. 1D). Preincubation of Ure2p (18 &#206;&#188;M) with <br /> min, which agrees with previous results (Fig. 1A) (21, 22).                      either Ssa1p or Ssa2p (6 &#206;&#188;M) increased the lag time of Ure2p <br />    Low intrinsic ATPase activity provides a key step for regula-                 amyloid formation and decreased the yield of amyloid formed to <br /> tion and &#239;&#172;&#65533;ne-tuning of Hsp70 function by cochaperones. Ydj1p,                    the same degree. These data suggest that the differences in the <br /> the major yeast cytosolic Hsp40, is an extensively studied                       way Ssa1p and Ssa2p in&#239;&#172;&#8218;uence [URE3] propagation are not due <br /> cochaperone that stimulates Ssa1p ATPase. Ydj1p stimulated                       to differences in the way they in&#239;&#172;&#8218;uence Ure2p amyloid forma- <br /> ATPase activity of Ssa1p and Ssa2p roughly 10-fold and 7-fold,                   tion. Together our data did not uncover any differences in the <br /> respectively, which agrees well with previous data (Fig. 1A) (21,                enzymatic activities of Ssa1p and Ssa2p that were substantial <br /> 22). We tested whether this small difference might be due to                     enough to explain their functional distinction in vivo. <br /> a difference in interaction of Ydj1p with Ssa1p and Ssa2p by <br /> using surface plasmon resonance. Ydj1p consistently interacted                   [URE3] Is Unstable in Cells Expressing only Ssa1p. [URE3] prop- <br /> with Ssa2p slightly better than with Ssa1p (Fig. 1B), suggesting                 agates as insoluble amyloid aggregates of Ure2p, which is a re- <br /> that the lower stimulation of Ssa2p ATPase by Ydj1p was not                      pressor of nitrogen catabolic genes such as the allantoate <br /> due to reduced physical interaction of Ydj1p and Ssa2p.                          transporter Dal5p. When [URE3] is present, depletion of Ure2p <br />    We compared the ability of Ssa1p and Ssa2p to refold mis-                     into prion aggregates relieves this repression. In strains with <br /> folded protein by measuring reactivation of &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase after               ADE2 controlled by the DAL5 promoter, expression of Ade2p <br />                                                                                  can be used to monitor the presence of [URE3] (25, 26). When <br />                                                                                  [URE3] is present in these strains, they express Ade2p so they <br />                                                                                  grow without adenine and are white. When they are [ure-o] (i.e., <br />                                                                                  lack [URE3]) they require adenine to grow and are red on media <br />                                                                                  containing limiting adenine (Materials and Methods). <br />                                                                                     Amino acid identity of the Ssa1p and Ssa2p NBD, SBD, and <br />                                                                                  CTD is &#226;&#710;&#188;99, 99, and 93% (Fig. 2A). Hsp70 amino acids con- <br />                                                                                  served across species that are essential for Hsp70 function or <br />                                                                                  known to interact with cochaperones or substrates are present in <br />                                                                                  both Ssa1p and Ssa2p. For example, they both contain R169, <br />                                                                                  which is crucial for interaction with Hsp40 cochaperones, linker <br />                                                                                  residues 382&#226;&#8364;&#8220;392 between NBD and SBD, which are required for <br />                                                                                  interdomain communication, residues I400, A403, L410, F425, <br />                                                                                  A428, V435, and I437, which form a hydrophobic substrate- <br />                                                                                  interacting core, and the C-terminal GPT(I/V)EEVD motif of <br />                                                                                  eukaryotic Hsp70s, which interacts with tetratricopeptide repeat <br />                                                                                  (TPR) cochaperones. It was therefore dif&#239;&#172;&#65533;cult to predict a priori <br />                                                                                  the residues critical for Hsp70 functional differences. <br />                                                                                     Growth of strain 1161, which lacks all four chromosomal SSA <br />                                                                                  family genes (SSA1&#226;&#8364;&#8220;SSA4), is supported equally well by Ssa1p or <br />                                                                                  Ssa2p expressed from a plasmid (17), indicating that all essential <br />                                                                                  Hsp70 functions of Ssa1p and Ssa2p are completely in- <br />                                                                                  terchangeable. However, whereas [URE3] propagates stably in <br />                                                                                  cells expressing only Ssa2p, it is unstable in cells expressing only <br /> Fig. 1. Ssa1p and Ssa2p have comparable enzymatic activities. (A) ATPase <br />                                                                                  Ssa1p (17). This instability is seen as the appearance of red [ure- <br /> activity of Ssa1p or Ssa2p (0.4 &#206;&#188;M) in the absence or presence of Ydj1p (1.6 <br /> &#206;&#188;M). ATP was 50 &#206;&#188;M. Values are averages &#194;&#177; SD of two to three replicate <br />                                                                                  o] colonies arising among a population of cells spread on a plate <br /> experiments. (B) Representative surface plasmon resonance experiment of          with limiting adenine (Fig. 2B, compare 111 with 222). Pheno- <br /> physical interaction of Ssa1p or Ssa2p (concentration indicated on Right)        typic differences between the cells expressing only Ssa1p or <br /> with immobilized Ydj1p. For each assay, background resonance from control        Ssa2p are not explained by differences in abundance of Ssa <br /> cells lacking Ydj1p was subtracted. Highly reproducible results were             protein or the major Hsp70 cochaperones known to in&#239;&#172;&#8218;uence <br /> obtained in two additional replicate experiments on separate machines us-        [URE3] propagation (Ydj1p, Sse1p, and Sis1p; Fig. S1A). We <br /> ing proteins from two different preparations and &#239;&#172;&#65533;ve concentrations of Ssa       used the readily distinguishable prion phenotypes to determine <br /> protein. (C) Denatured &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase (0.5 &#206;&#188;M) was incubated with Ydj1p        the molecular basis of the functional differences between Ssa1p <br /> (0.2 &#206;&#188;M) in the presence or absence of Ssa1p or Ssa2p (2 &#206;&#188;M) as indicated. All <br />                                                                                  and Ssa2p. <br /> reactions contained 20 &#206;&#188;M ATP. Values are averages &#194;&#177; SD of two replicate <br /> experiments. (D) Puri&#239;&#172;&#65533;ed Ure2p was dialyzed from buffer containing 1 M <br />                                                                                  ATPase Domain Determines Prion Stability. We &#239;&#172;&#65533;rst examined the <br /> GdmCl into buffer without GdmCl and then centrifuged at 12,000 g for 1 h <br /> immediately before adding Ure2p (18 &#206;&#188;M) to cuvettetes with or without            regulatory effect of each Ssa functional region by swapping the <br /> Hsp70 (6 &#206;&#188;M). Reactions were stirred at 25&#194;&#176; and 10-&#206;&#188;L aliquots were removed      major Hsp70 structural domains. We constructed six Ssa1p&#226;&#8364;&#8220; <br /> and added to 490 &#206;&#188;L of 100 &#206;&#188;M Thio&#239;&#172;&#8218;avin-T. Fluorescence was monitored at         Ssa2p hybrid proteins representing all of the possible combina- <br /> 485 nm with excitation at 450 nm.                                                tions of the three domains described above. Names of wild-type <br />  <br /> 13666 | www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                                               Sharma and Masison <br />                                                                                   determine the minimal difference that conferred the functional <br />                                                                                  distinction, we began by switching individual residues of Ssa2p to <br />                                                                                  the amino acid found in Ssa1p. Switching residues 20, 41, or 62 <br />                                                                                  did not in&#239;&#172;&#8218;uence [URE3] stability (Fig. 2C). Remarkably, how- <br />                                                                                  ever, cells expressing Ssa2p with the G83A substitution <br />                                                                                  (Ssa2G83A) displayed an unstable Ssa1p-like [URE3] phenotype <br />                                                                                  (Fig. 2 C and E). Thus, Ssa2G83A behaves like Ssa1p even though <br />                                                                                  it is identical to Ssa2p except in having a methyl group in place of <br />                                                                                  the hydrogen as the side chain of residue 83. <br />                                                                                      We then switched glycine 83 to the bulkier residues leucine or <br />                                                                                  phenylalanine to con&#239;&#172;&#65533;rm the crucial role of glycine at this lo- <br />                                                                                  cation. Both alterations similarly reduced [URE3] stability like <br />                                                                                  G83A (Fig. 2C). To determine whether a glycine at residue 83 was <br />                                                                                  enough to promote stable [URE3] propagation, we made the re- <br />                                                                                  ciprocal A83G substitution in Ssa1p (Ssa1A83G). Although Ssa1A83G <br />                                                                                  is identical to Ssa1p except for residue 83, cells expressing it as the <br />                                                                                  only source of Ssa protein had a stable Ssa2p-like [URE3] pheno- <br />                                                                                  type. These data demonstrate that glycine residue 83 determines an <br />                                                                                  Ssa2p-speci&#239;&#172;&#65533;c function important for [URE3] prion propagation. <br />                                                                                      Structural information for Ssa Hsp70 is not known. Hsp70 <br />                                                                                  chaperones are highly conserved, however, and structural in- <br />                                                                                  formation is available for Hsp70 proteins from many species. <br />                                                                                  Using a Web-based program (http://swissmodel.expasy.org/), we <br />                                                                                  modeled Ssa1p structure. Residue 83 lies in a short &#206;&#177;-helix in the <br />                                                                                  ATPase domain (residues 80&#226;&#8364;&#8220;88), which is conserved among <br />                                                                                  Hsp70s whose structure is known (Fig. S2). None of these <br />                                                                                  Hsp70s has a glycine at this homologous position. Because gly- <br />                                                                                  cine is unfavorable for helix stability, Ssa2p might have an al- <br />                                                                                  ternative structure due to the presence of G83. We tested <br /> Fig. 2. ATPase domain determines speci&#239;&#172;&#65533;city of Hsp70 function. (A) Linear        whether making substitutions Q82G or P, and M85G or P in <br /> diagram of Ssa1/2p with structural domains indicated and locations of dif-       Ssa1p, which we expected would similarly disrupt a putative <br /> ferences in amino acid residues between Ssa1p and Ssa2p highlighted as red       helix, would restore [URE3] stability like A83G. We also tested <br /> bars. (B&#226;&#8364;&#8220;D) Cells taken from white colonies grown nonselectively on 1/2 YPD      a deletion of residue 83 in both Ssa1p and Ssa2p (A83&#206;&#8221; and <br /> were diluted in water, spread onto 1/2 YPD plates and grown for 3 d. (B) <br />                                                                                  G83&#206;&#8221;, respectively) to determine the importance of this residue <br /> Leftmost panels show wild-type strain 1075 [URE3] and [ure-o] colonies (WT+ <br /> and WT&#226;&#8364;&#8220;, respectively), which display characteristic white and red color. <br />                                                                                  directly. None of these changes stabilized [URE3] like A83G, <br /> Degree of mitotic instability of [URE3] in strain 1161 expressing Ssa1p (111),   and most of them reduced [URE3] stability more than G83A <br /> Ssa2p (222), or Ssa1/2p hybrid proteins (as indicated) is proportional to the    (Fig. 2D). Moreover, the prion phenotypes of cells expressing <br /> frequency of pigmented colonies. (C and D) [URE3] phenotype of cells             Ssa1p or Ssa2p lacking residue 83 were indistinguishable, in- <br /> expressing Ssa1p (A1) and Ssa2p (A2) with the indicated single amino acid        dicating the other nonconserved residues do not contribute <br /> changes. (E) Quanti&#239;&#172;&#65533;ed frequencies of loss of [URE3]. Cells taken from white     functional speci&#239;&#172;&#65533;city of Ssa1p and Ssa2p regarding [URE3] <br /> colonies expressing the indicated Hsp70 were grown on 1/2 YPD, diluted in        propagation. Therefore, G83 seems to provide a unique se- <br /> water, and spread onto 1/2 YPD plates. The percentage (&#194;&#177;SD) of red [ure-o]       quence element important for Ssa2p-speci&#239;&#172;&#65533;c Hsp70 activity. <br /> colonies measured among a total of 900&#226;&#8364;&#8220;1,500 colonies is shown. <br />                                                                                      Overexpressing Ssa1p, but not Ssa2p, cures cells of [URE3] <br />                                                                                  (18). We tested whether interchanging residue 83 would also <br />                                                                                  switch this speci&#239;&#172;&#65533;city and found it did not (Fig. S3). This result <br /> and hybrid proteins re&#239;&#172;&#8218;ect their domain composition. For ex- <br />                                                                                  shows that Hsp70 activities important for prion propagation <br /> ample, 111 is Ssa1p and 122 has the NBD of Ssa1p and the SBD <br />                                                                                  differ from those important for prion elimination. <br /> and CTD of Ssa2p. All hybrids (and point mutants, see below) <br />                                                                                      We also tested whether the minor G/A83 difference dis- <br />  <br />  <br />  <br />                                                                                                                                                            GENETICS <br /> used here were expressed at a similar level (Fig. S1 B&#226;&#8364;&#8220;D) and                    tinguishes Ssa1p and Ssa2p in another cellular process known <br /> supported growth of cells like wild-type Ssa1p and Ssa2p (gen-                   to depend strictly on Ssa2p. Starving cells of glucose induces <br /> eration times of 104 &#194;&#177; 5 (SD) minutes in YPAD liquid medium                      expression of gluconeogenic enzymes, such as fructose-1,6- <br /> at 30 &#194;&#176;C), indicating that they function like the wild-type proteins             bisphosphatase (FBPase), and restoring glucose after 3 d causes <br /> with regard to essential Ssa functions.                                          degradation of these enzymes through a vacuolar import and <br />    Half of the hybrid proteins (122, 121, and 112) produced an                   degradation (Vid) pathway (15). FBPase was degraded similarly <br /> Ssa1p-like unstable [URE3] phenotype (Fig. 2B). In contrast,                     in wild-type cells and in cells expressing only Ssa2p, but it was <br /> cells expressing the remaining hybrids (211, 212, and 221)                       stabilized in cells expressing only Ssa1p (Fig. 3A). These results <br /> propagated [URE3] stably, like those expressing Ssa2p. Quan-                     con&#239;&#172;&#65533;rmed the speci&#239;&#172;&#65533;c requirement of Ssa2p for Vid pathway <br /> ti&#239;&#172;&#65533;ed frequencies of [URE3] loss are shown in Fig. 2E. Con-                      function. Amazingly, FBPase was stabilized in cells expressing <br /> spicuously, all hybrids that impaired [URE3] propagation had                     Ssa2G83A and was degraded in cells expressing Ssa1A83G. <br /> the NBD of Ssa1p and all those that supported stable [URE3]                      Therefore, this minimal difference in structure also switched the <br /> propagation had the NBD of Ssa2p. These results show that the                    roles of Ssa1p and Ssa2p in Vid pathway function. <br /> ATPase domains of Ssa1p and Ssa2p were enough to confer the                          Because these &#239;&#172;&#65533;ndings suggested a link between [URE3] <br /> speci&#239;&#172;&#65533;c Hsp70 isoform character regarding prion stability.                       propagation and Vid pathway function, we tested whether de- <br />                                                                                  leting SEC28 reduced [URE3] stability. Sec28p is a coatomer <br /> Glycine Residue 83 Determines Ssa2p-Speci&#239;&#172;&#65533;c Hsp70 Function. The                  complex subunit that regulates Golgi to ER traf&#239;&#172;&#65533;cking and <br /> NBDs of Ssa1p and Ssa2p differ at only four residues: 20 (A/S),                  facilitates import of Vid cargo into Vid vesicles (27). Deleting <br /> 41 (A/G), 62 (S/A), and 83 (A/G) (Ssa1p residue listed &#239;&#172;&#65533;rst), and                SEC28 stabilized FBPase (Fig. 3B), indicating it disrupted the <br /> each is a relatively minor difference in structure (Fig. 2A). To                 Vid pathway, but it did not affect [URE3] stability (Fig. 3C). <br />  <br /> Sharma and Masison                                                                                  PNAS | August 16, 2011 | vol. 108 | no. 33 | 13667 <br />                                                                                    function of the Ssa1p and Ssa2p cytoplasmic Hsp70s in two in- <br />                                                                                   dependent cellular processes. Additionally, deleting this residue <br />                                                                                   from Ssa1p and Ssa2p generated proteins that were functionally <br />                                                                                   equivalent, yet behaved differently than either Ssa1p or Ssa2p. <br />                                                                                   Because Hsp70s with much less homology still overlap func- <br />                                                                                   tionally, it is a bit surprising that such nearly identical proteins <br />                                                                                   act so differently and rather astonishing that such a small change <br />                                                                                   is responsible for the distinction in function. Despite the known <br />                                                                                   regulatory activity of the NBD, this &#239;&#172;&#65533;nding was unexpected be- <br />                                                                                   cause the NBDs of Ssa1p and Ssa2p differ by only a few mini- <br />                                                                                   mally different and spatially separated residues. We anticipated <br />                                                                                   the C-terminal region was more likely to provide the speci&#239;&#172;&#65533;city <br />                                                                                   because it is the most divergent region among Hsp70s, including <br />                                                                                   Ssa1p and Ssa2p, and it possesses elements that interact with <br />                                                                                   a variety of cochaperones that regulate Hsp70 function. Our <br />                                                                                   &#239;&#172;&#65533;nding that residue 83 does not in&#239;&#172;&#8218;uence speci&#239;&#172;&#65533;city of overex- <br /> Fig. 3. Residue 83 speci&#239;&#172;&#65533;es Hsp70 function in the Vid pathway. (A) Cultures       pressed Ssa1p and Ssa2p in curing cells of [URE3] indicates that <br /> of strain 1161 [ure-o] cells were starved of glucose for 3 d and then split in <br />                                                                                   these other amino acid differences do, in fact, contribute addi- <br /> half. Glucose was added to one half where indicated and after 2 h the <br /> abundance of FBPase was monitored by Western analysis. Numbers 1 and 2 <br />                                                                                   tional regulatory speci&#239;&#172;&#65533;city. <br /> designate Ssa1p and Ssa2p, respectively; WT is wild type (strain 1075). Similar      The major human Hsp70s HspA1A and HspA1B differ by only <br /> results were obtained with [URE3] cells. (B) The experiment was repeated          two amino acids, one of which resides in the NBD, and these <br /> using an isogenic sec28&#206;&#8221; strain. (C) [URE3] phenotypes of wild-type and           proteins are believed to be fully interchangeable (10). Our &#239;&#172;&#65533;nd- <br /> sec28&#206;&#8221; cells grown on 1/2 YPD plates for 3 d at 30 &#194;&#176;C.                            ings raise the alternative possibility that their differences de- <br />                                                                                   termine speci&#239;&#172;&#65533;c functions in as yet unidenti&#239;&#172;&#65533;ed tasks. They also <br />                                                                                   raise concerns about commonly performed experiments using <br /> Thus, stable [URE3] propagation did not require a functional                      exogenously expressed Hsp70s. As mentioned already, the 98% <br /> Vid pathway, which was somewhat unexpected because we sup-                        identical Ssa1p and Ssa2p differ in their ability to cure yeast of <br /> posed the minimal alanine/glycine difference might have a unique                  prions when overexpressed, and similar differences in effects us- <br /> and limited effect on Hsp70 activity. Therefore, residue 83                       ing different Hsp70s might be expected in other model systems. <br /> appears to have differentiated to specify a functional distinction                   There is strong allosteric coordination among the structural <br /> that is important for de&#239;&#172;&#65533;ned yet unrelated Hsp70 tasks in the cell.               domains of Hsp70. Although substrate binds at the SBD, the <br />                                                                                   kinetics of binding and release of substrate are regulated pri- <br /> Discussion                                                                        marily through ATP hydrolysis and nucleotide exchange by the <br /> We identify a minimal amino acid change as a basis for the                        NBD. We have not ruled out the possibility that an intrinsic <br /> functional distinction between highly homologous Hsp70 family                     Hsp70 activity is altered, but among a large cohort of Hsp70 <br /> chaperones, a phenomenon that is conserved throughout evo-                        cochaperones many interact at the NBD to regulate these pro- <br /> lution. This adaptability of the ATPase domain to regulate                        cesses. Although residue 83 is not known to interact directly with <br /> speci&#239;&#172;&#65533;city of Hsp70 function in different cellular processes                      cochaperones, the subtle Gly/Ala difference might in&#239;&#172;&#8218;uence the <br /> widens its versatility beyond a mere structural domain that en-                   ef&#239;&#172;&#65533;ciency by which they regulate the ATP reaction cycle. Resi- <br /> zymatically in&#239;&#172;&#8218;uences substrate binding. Our &#239;&#172;&#65533;ndings demon-                       due 83 lies in NBD lobe IB, which is near the nucleotide-binding <br /> strate that exquisitely de&#239;&#172;&#65533;ned variations of Hsp70 structure can                  cleft (Fig. S2). A structural implication of the Gly/Ala difference <br /> strongly in&#239;&#172;&#8218;uence the actions of Hsp70 in speci&#239;&#172;&#65533;c and complex                     is that it could affect the movement of this lobe upon interaction <br /> physiological processes. They also show that we could success-                    with nucleotide exchange factors (NEFs) (32), possibly altering <br /> fully switch functional speci&#239;&#172;&#65533;city between Ssa1p and Ssa2p by                     the way different NEFs are able to induce a structural transition <br /> coupling protein engineering with use of yeast prions to monitor                  that facilitates release of phosphate or ADP. <br /> Hsp70 activity, which shows potential for designing Hsp70 to                         The Gly/Ala difference might also in&#239;&#172;&#8218;uence the ef&#239;&#172;&#65533;ciency of <br /> perform in desired ways.                                                          Hsp70 interaction with major chaperone machinery components <br />    The major Hsp70 activities in the cell depend on the same                      such as Hsp104 and Hsp90 or other nonchaperone factors in- <br /> ATP-regulated cycle of binding and release of exposed hydro-                      volved in speci&#239;&#172;&#65533;c processes that require a particular Hsp70 for <br /> phobic surfaces on partially folded proteins. The necessity of                    the job. Alternatively, a modest change in the kinetics of either <br /> possessing the same basic structural information to carry out this                speci&#239;&#172;&#65533;c steps or the overall rate of the Hsp70 reaction cycle <br /> important reaction is evident in the conservation of both sec-                    might have little affect on general functions of Hsp70 but be of <br /> ondary and tertiary structures of even widely diverged Hsp70s.                    importance for certain tasks. <br /> Gross structural information obtained by recombining domains                         In addition to its de&#239;&#172;&#65533;ned functions in essential housekeeping <br /> of Hsp70 has shown that although the SBD can in&#239;&#172;&#8218;uence in-                         processes, Hsp70 protects a large number of proteins that are <br /> trinsic activity of the adjacent NBD (28), the SBD, with its broad                prone to aggregation under various kinds of stresses. During <br /> substrate speci&#239;&#172;&#65533;city, can be interchanged without destroying                      evolution there would have been a clear advantage to possess <br /> functions associated with divergent Hsp70s (4, 7). The NBD,                       different Hsp70s that allow interaction with many protein se- <br /> however, which regulates substrate binding by the SBD, is much                    quence elements while maintaining speci&#239;&#172;&#65533;city for performing <br /> more important for determining speci&#239;&#172;&#65533;c activities of different                    certain tasks. Our &#239;&#172;&#65533;ndings suggest that the NBD has been evo- <br /> Hsp70s in vivo. For example, a majority of Hsp70 alterations that                 lutionarily optimized for conferring such functional speci&#239;&#172;&#65533;city. It <br /> affect yeast prion propagation are in the NBD, and the capacity                   is likely that the SBD is more optimized for interacting with <br /> of Hsp70 to support yeast viability is governed primarily by the                  many proteins that collectively have high diversity in amino acid <br /> NBD (4, 29&#226;&#8364;&#8220;31). These observations suggest that the regulation                    sequence. The unique combination of functional speci&#239;&#172;&#65533;city by <br /> of Hsp70 activity, and not substrate binding per se, is the critical              the NBD and broader range to bind various substrates at the <br /> parameter for determining speci&#239;&#172;&#65533;city of function.                                 SBD allows Hsp70 to perform a wide variety of cellular tasks not <br />    In line with this conclusion, we now show that a minimal Gly/                  known for any other cellular protein. The genomic ampli&#239;&#172;&#65533;cation <br /> Ala difference in the NBD is enough to interconvert speci&#239;&#172;&#65533;city of                 and differentiation of Hsp70 genes magni&#239;&#172;&#65533;es and diversi&#239;&#172;&#65533;es <br />  <br /> 13668 | www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                                                Sharma and Masison <br />  these bene&#239;&#172;&#65533;ts. The observation that such a minimal tweaking of                                 Plasmids expressing Ssa2p (pPROEX-HTb-SSA2) or Ydj1p (pPROEX-HTb- <br /> structure determines major distinctions of function points to                               YDJ1) were obtained by replacing SSA1 of plasmid pPROEX-HTb-SSA1 using <br /> Hsp70 as a powerfully malleable object of selection.                                        standard molecular biology techniques. The Ure2p expression vector (pKT41- <br />                                                                                             Ure2) was a gift from Dr. Reed Wickner (National Institutes of Health, <br /> Materials and Methods                                                                       Bethesda, MD) (23). The vector encodes Ure2p with an N-terminal hexa-His <br /> Yeast Strains and Plasmids. Strain 1075 is MATa, kar1-1, SUQ5, PDAL5::ADE2,                 tag. All proteins were puri&#239;&#172;&#65533;ed using Talon metal af&#239;&#172;&#65533;nity resin. Except for <br /> his3&#206;&#8221;202, leu2&#206;&#8221;1, trp1&#206;&#8221;63, ura3-52 (17). Strain 1161 has in addition ssa1::                 Ure2p, the polyhistidine tag was removed before assaying proteins for <br /> KanMX, ssa2::HIS3, ssa3::TRP1, ssa4::ura3-2f and carries plasmid pJ401 (see                 their activity. <br /> below). Strain 1366 is 1075 with sec28::KanMX. Strain 628&#226;&#8364;&#8220;4B (18), used for <br /> monitoring curing of [URE3], is MAT&#206;&#177;, kar1-1, SUQ5, ade2-1, his3&#206;&#8221;202, ura2.                 ATPase Assay. ATP hydrolysis was measured essentially as described (22). <br /> Plasmid pJ401 is a URA3-based plasmid with SSA2. Plasmid pC210 is a single-                 Brie&#239;&#172;&#8218;y, ATPase activity of 0.4 &#206;&#188;M Ssa1p or Ssa2p together with 1.6 &#206;&#188;M Ydj1 <br /> copy LEU2-based plasmid containing SSA1 with an NdeI site at the initiator                  was measured in 200 &#206;&#188;L assay buffer (25 mM Tris-Cl pH 7.4, 100 mM KCl, 5 <br /> ATG codon, under control of the SSA2 promoter (18). Plasmid p111 is pC210                   mM MgCl2, 1 mM DTT) at 25 &#194;&#176;C. Reactions were incubated for 10 min and the <br /> with XbaI and XhoI sites created without altering amino acid coding in-                     assay started by addition of [&#206;&#179;-32P] ATP to a &#239;&#172;&#65533;nal concentration of 50 &#206;&#188;M. <br /> formation at SSA1 codons 391 and 540, respectively (4). Plasmid p222 is p111                Aliquots were removed at 10, 20, and 30 min and the amount of ATP hy- <br /> with the coding region of SSA1 replaced by that of SSA2. Plasmids with <br />                                                                                             drolysis determined as described. ATP turnover rates were linear over the <br /> SSA1/SSA2 hybrid genes were constructed by replacing the NdeI&#226;&#8364;&#8220;XbaI (NBD), <br />                                                                                             course of the assay and were highly reproducible over multiple separate <br /> XbaI&#226;&#8364;&#8220;XhoI (SBD), and XhoI&#226;&#8364;&#8220;SphI (CTD) fragments of p111 with PCR-ampli&#239;&#172;&#65533;ed <br />                                                                                             protein preparations. <br /> fragments of SSA2. Substitutions of single amino acids were done using the <br /> Stratagene Quick-Change kit. All HSP70 alleles were veri&#239;&#172;&#65533;ed by DNA se- <br /> quencing. Plasmids pA1-83G and pA2-83A are pC210 encoding SSA1A83G and                      Luciferase Refolding. Promega (catalog no. E1701) recombinant &#239;&#172;&#65533;re&#239;&#172;&#8218;y lucif- <br /> SSA2G83A, respectively. These alleles were cloned into pRS313 (single-copy                  erase (0.5 &#206;&#188;M) was denatured in 50 &#206;&#188;L of denaturation buffer (6 M GdmCl, <br /> HIS3 vector) to generate pRS313A1-83G and pRS313A2-83A for monitoring                       30 mM Hepes, 50 mM KCl, 5 mM MgCl2, 5 mM &#206;&#178;-mecaptoethanol, pH 7.3) for <br /> curing of [URE3].                                                                           1 h at 30 &#194;&#176;C. Luciferase was diluted in two sequential steps. First, 75 &#206;&#188;L of <br />    To assess effects of hybrid Hsp70 proteins, plasmid pJ401 in strain 1161 was             refolding buffer (denaturation buffer without GdmCl and &#206;&#178;-mecaptoethanol) <br /> replaced by the LEU2-based plasmids carrying hybrid alleles. The only Ssa                   was added and the reaction incubated at room temperature for 1 min. Sec- <br /> protein in these cells is the one on the plasmid. Because Ssa protein is es-                ond, 1 &#206;&#188;L of the diluted solution was added to 50 &#206;&#188;L refolding buffer con- <br /> sential, these cells must retain the plasmid to grow, even on rich media.                   taining 0.2 uM Ydj1p with or without 2&#206;&#188;M Ssa1p or Ssa2p and further <br />                                                                                             incubated at room temperature for 5 min. Luciferase refolding was initiated <br /> Media, Growth Conditions, and Monitoring Prions. Media and growth con-                      by adding 5 mM ATP. Aliquots were removed at the indicated intervals and <br /> ditions were as described (29) or as indicated in Results. [URE3] was moni-                 luciferase refolding was recorded using a Zylux FB15 luminometer and lucif- <br /> tored by using the wild-type ADE2 gene regulated by the DAL5 promoter                       erase assay reagent (Promega; catalog no. E1501). Spontaneous refolding of <br /> (PDAL5::ADE2) (25, 26). On standard ammonium-containing media Ure2p                         luciferase without added chaperones was subtracted as background. <br /> represses expression of the DAL5 promoter so Ade2p is not produced and <br /> cells are Ade&#226;&#8364;&#8220; and red. When [URE3] is present the depletion of functional <br />                                                                                             Ure2p Amyloid Formation. Puri&#239;&#172;&#65533;ed Ure2p (120 &#206;&#188;L of 30 &#206;&#188;M stock) freshly di- <br /> Ure2p into insoluble prion aggregates activates the DAL5 promoter and <br />                                                                                             alyzed in PBS was added to 76 &#206;&#188;L of PBS reaction buffer. Final concentrations <br /> ADE2 expression, making cells Ade+ and white. Weakened [URE3] propa- <br /> gation can be seen as intermediate colony color (pink) or increased fre-                    were 1 mM ATP, 20 mM MgCl2, 1 mM KCl, 18 &#206;&#188;M Ure2p and 6 &#206;&#188;M Ssa1p or <br /> quency of appearance of [ure-o] cells in a population. The presence of                      Ssa2p. Samples were incubated with stirring in a 500-&#206;&#188;L cuvettete. At in- <br /> [URE3] was con&#239;&#172;&#65533;rmed by its dominant infectious phenotype and ability to be                  dicated times, 10 &#206;&#188;L samples were removed and added to 490 &#206;&#188;L of 100 &#206;&#188;M <br /> cured when cells are grown in the presence of 3 mM guanidine hydrochlo-                     Thio&#239;&#172;&#8218;avin-T. Fluorescence was recorded at 485 nm with excitation at 450 nm. <br /> ride, which inactivates Hsp104 (33&#226;&#8364;&#8220;37). <br />                                                                                             Western Analysis. Yeast were harvested by centrifugation and suspended in <br /> Protein Puri&#239;&#172;&#65533;cation. Plasmid pPROEX-HTb-SSA1 for expressing His6&#226;&#8364;&#8220;Ssa1 fu-                   lysis buffer (PBS with 0.2% Triton X-100 and protease inhibitor mixture; <br /> sion protein was obtained from Addgene (38). The plasmid encodes from                       Roche). Cells were broken by vortexing with glass beads, and cell lysate was <br /> 5&#226;&#8364;&#178; to 3&#226;&#8364;&#178; direction hexa-His-tag, TEV cleavage site, and Ssa1p. Protein was                  centrifuged at 3,000 rpm for 2 min. Supernatant was boiled in loading dye for <br /> puri&#239;&#172;&#65533;ed from Escherichia coli strain Rosetta 2(DE3) (Invitrogen) grown in LB                15 min at 95 &#194;&#176;C and 5 &#206;&#188;g of protein was separated in 4&#226;&#8364;&#8220;12% gradient SDS <br /> medium with 300 mM NaCl at 30 &#194;&#176;C until OD600 = 0.6&#226;&#8364;&#8220;0.8. The culture was                     gels and then transferred onto PVDF membranes. Hsp70 antibodies were <br /> induced with 0.5 mM IPTG and grown overnight at 18 &#194;&#176;C. Cells were har-                      from Stressgen (catalog no. SPA-822), FBPase antibodies were a gift from <br /> vested and lysed in buffer A (20 mM Hepes, 150 mM NaCl, 20 mM MgCl2,                        C. Randell Brown (Pennsylvania State College of Medicine, Hershey, PA) (27), <br /> mM KCl 20, complete protease inhibitor mixture) using lysozyme. Cell debris                 and Sec28p antibodies were a gift from R. Duden (University of L&#195;&#188;beck, <br /> was cleared by centrifugation (12,000 g) and the supernatant loaded on <br />                                                                                             L&#195;&#188;beck, Germany) (39). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             GENETICS <br /> cobalt-based Talon metal af&#239;&#172;&#65533;nity resin. After washing, protein was eluted <br /> with buffer A containing 250 mM imidazole. Puri&#239;&#172;&#65533;ed protein was incubated <br />                                                                                         <br /> with His-TEV (Invitrogen; catalog no. 12575-023) protease at 30&#194;&#176; for 1 h. The <br />                                                                                              <br /> sample was extensively dialyzed at 4&#194;&#176; and again passed through Talon metal               <br /> af&#239;&#172;&#65533;nity resin to remove the cleaved His tag and His-TEV protease. Protein                   <br /> purity was more than 99% as determined by SDS/PAGE and Coomassie                        <br /> staining. Protein identity was con&#239;&#172;&#65533;rmed using mass spectrometry.                        <br /> </body></html>