<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                        Europe PMC Funders Group <br />                                         Author Manuscript <br />                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                               Published final edited form as: <br />                                                Cell Host Microbe. 2013 December 11; 14(6): . doi:10.1016/j.chom.2013.11.004. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                               Natural Killer Cell-Mediated Host Defense Uropathogenic <br />                                               E. coli Is Counteracted Bacterial HemolysinA-Dependent <br />                                               Killing NK Cells <br />                                               Chamutal Gur1,4,6, Shunit Coppenhagen-Glazer2,6, Shilo Rosenberg3, Rachel Yamin1, <br />                                               Jonatan Enk1, Ariella Glasner1, Yotam Bar-On1, Omer Fleissig1,2, Ronit Naor2, Jawad <br />                                               Abed2, Dror Mevorach4, Zvi Granot5, Gilad Bachrach2,6,*, Ofer Mandelboim1,6,* <br />                                               1The Lautenberg Center General Tumor Immunology, The Hebrew University Hadassah <br />  <br />                                               Medical School, IMRIC, Jerusalem 91120, Israel <br />                                               2The Institute Dental Sciences, Hebrew University-Hadassah School Dental Medicine, <br />                                               Jerusalem 91120, Israel <br />                                               3Urology    Department Hadassah Medical Center, Jerusalem 91120, Israel <br />                                               4Internal   Medicine Department Hadassah Medical Center, Jerusalem 91120, Israel <br />                                               5Department Developmental Biology Cancer Research, The Hebrew University Hadassah <br />                                               Medical School, IMRIC, Jerusalem 91120, Israel <br />  <br />                                               SUMMARY <br />                                                    Uropathogenic Escherichia coli (UPEC) common cause urinary tract infections (UTIs) in <br />                                                    humans. While importance natural killer (NK) cells innate immune protection against <br />                                                    tumors viral infections documented, role defense bacterial infections is <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                    emerging, involvement UPEC-mediated UTI practically unknown. Using a <br />                                                    systematic mutagenesis approach, UPEC adheres NK cells primarily type I <br />                                                    fimbriae employs hemolysinA toxin kill NK cells. In absence hemolysinA, NK <br />                                                    cells directly respond bacteria secrete cytokine TNF-&#206;&#177;, results decreased <br />                                                    bacterial numbers vitro reduction bacterial burden infected bladders. Thus, NK <br />                                                    cells control UPEC TNF-&#206;&#177; production, UPEC counteracts hemolysinA-mediated <br />                                                    killing NK cells, representing previously unrecognized host defense microbial <br />                                                    counterattack mechanism context UTI. <br />  <br />  <br />                                               INTRODUCTION <br />                                                                 E. coli colonize gastrointestinal tract human infants hours birth <br />                                                                 (Kaper et al., 2004). This commensal bacterium rarely causes disease in <br />                                                                 immunocompromised hosts (Kaper et al., 2004). However, highly adapted <br />                                                                 pathogenic E. coli strains cause broad spectrum diseases, including enteric disease, <br />                                                                 sepsis/meningitis, UTI (Emody et al., 2003; Johnson, 1991). UPEC responsible for <br />                                                                 approximately 80%&#226;&#8364;&#8220;90% community-acquired UTI cases (Sivick et al., 2010; Ulett et al., <br />                                                                 2013), 50% women 12% men experience episode <br />  <br />  <br />                                               &#194;&#169;2013 Elsevier Inc. <br />                                               * <br />                                                 Correspondence: giladba@ekmd.huji.ac.il (G.B.), oferm@ekmd.huji.ac.il (O.M.) http://dx.doi.org/10.1016/j.chom.2013.11.004. <br />                                               6The authors authors contributed equally work <br />                                               SUPPLEMENTAL INFORMATION Supplemental Information includes figures table this <br />                                               article http://dx.doi.org/10.1016/j.chom.2013.11.004. <br />                                          Gur et al.                                                                                               Page 2 <br />  <br />  <br />                                                        clinically significant infection lifetime (Sivick et al., 2010). UPEC utilize <br />                                                        virulence factors encoded pathogenicity islands (Ulett et al., 2013) infect an <br />                                                        immunocompetent host colonizing periurethral area subsequently ascend through <br />                                                        urethra bladder (Kucheria et al., 2005). In bladder, uroepithelial cells the <br />                                                        early sensors microbial challenge (Ragnarsd&#195;&#179;ttir et al., 2011). Neutrophils first <br />                                                        most-abundant cell type migrate bladder event UTI, they <br />                                                        constitute crucial limiting factor bacterial growth urinary tract (Haraoka et al., <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                        1999). In addition, immune cells implicated host defense against <br />                                                        UTI (Engel et al., 2006; Jones-Carson et al., 1999). However, practically unknown <br />                                                        NK cells, critical players innate immune response, present in <br />                                                        bladder involved UTIs. <br />  <br />                                                        NK cells lymphocytes make 15% peripheral blood lymphocytes <br />                                                        (Seidel et al., 2012). They best known ability kill virally infected and <br />                                                        transformed cells secreting cytokines, specifically TNF-&#206;&#177; IFN-&#206;&#179; (Jewett et al., <br />                                                        1996). The NK cell activity regulated balance signals derived inhibitory <br />                                                        activating receptors (Koch et al., 2013). Their ligands numerous stress <br />                                                        induced, tumor derived, pathogen derived, self-ligands (Seidel et al., 2012). While <br />                                                        importance NK cells innate immune protection tumors viral infections is <br />                                                        documented (Koch et al., 2013), ability directly recognize bacteria well <br />                                                        defined. In regard, previously shown NK cells able directly <br />                                                        recognize Fusobacterium nucleatum killer receptor NKp46 this <br />                                                        interaction exacerbated periodontal disease (Chaushu et al., 2012). <br />  <br />                                                        In study vitro strains UPEC adhere human murine NK cells <br />                                                        primarily type I fimbria kill NK cells hemolysinA toxin. We <br />                                                        demonstrate vivo NK cells accumulate bladder UTI the <br />                                                        absence hemolysinA interaction NK cells E. coli leads TNF-&#206;&#177; <br />                                                        secretion, attenuates infection. In contrast, pathogenic UPEC strains express <br />                                                        hemolysinA evade NK cell control killing NK cells. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             RESULTS <br />                                             Strains UPEC Kill Human NK Cells <br />                                                        <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> test bacteria directly recognized NK cells, incubated bacterial <br />                                                        strains (Figure S1 available online), including GFP-expressing E. coli strains (Figure 1) with <br />                                                        human NK cells. The incubation performed 4&#194;&#176;C 37&#194;&#176;C. NK cells were <br />                                                        analyzed flow cytometry GFP (indicative bacterial binding) PI (to detect <br />                                                        dead cells). As seen Figure 1A, 4&#194;&#176;C (upper dot plots) observed enteric <br />                                                        (EPEC) urinary pathogenic E. coli strains: UPEC SR71 UPEC CFT073 (cystitis and <br />                                                        pyelonephritis isolates, respectively) adhered 20% NK cells. When the <br />                                                        bacteria incubated human NK cells 37&#194;&#176;C, interacted higher <br />                                                        percentages NK cells (lower dot plots, Figure 1A), surprise substantial <br />                                                        (around 72%, upper quadrants) PI staining (indicative cell death) observed with <br />                                                        UPEC CFT073 (Figure 1A). This <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span> PI staining seen XL1 the <br />                                                        UPEC SR71 strain, observed EPEC bacterial species tested <br />                                                        (Figure S1). <br />  <br />                                                        The PI staining observed UPEC CFT073 bacterium accompanied changes in <br />                                                        morphology NK cells. As seen Figure 1B, 37&#194;&#176;C, presence of <br />                                                        UPEC CFT073 NK cells appeared smaller present gate 2 (the cells in <br />                                                        gate stained positive PI). In contrast, NK cells incubated with <br />                                                        UPEC CFT073 4&#194;&#176;C, normal NK cell morphology evident, cells were <br />  <br />  <br />                                                           Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                Page 3 <br />  <br />  <br />                                                         present gate 1 (the NK cells gate mainly PI negative). Furthermore, 4&#194;&#176;C, the <br />                                                         NK cell population gate 1 equivalent size population NK <br />                                                         cells observed UPEC CFT073 absent (Figure 1B). Figures 1C 1D summarize <br />                                                         quantify observations. As seen, little changes observed when <br />                                                         bacteria UPEC CFT073 used. To corroborate findings, used <br />                                                         confocal microscopy, seen Figure 1E, nearly NK cells stained <br />                                                         positive PI incubated 37&#194;&#176;C 3 hr UPEC CFT073. The bacteria, the <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         hand, remained PI negative incubated NK cells 37&#194;&#176;C (Figure <br />                                                         1E) 4&#194;&#176;C (data shown), suggesting bacteria killed NK cells. <br />                                                         To determine UPEC CFT073 invades NK cells, incubated NK cells 3 <br />                                                         hr 37&#194;&#176;C GFP-labeled UPEC CFT073 bacterium stained human NK cells <br />                                                         anti-CD56 mAbs primarily stain NK cell membrane (Figure 1F, left). As <br />                                                         seen, UPEC CFT073 bacterium does penetrate dead human <br />                                                         NK cells. Scanning electron microscopy (SEM), (Figure 1F, right) transmission electron <br />                                                         microscopy (TEM) (Figure 1F, bottom) experiments demonstrated bacteria <br />                                                         inside NK cells. <br />  <br />                                                         To investigate (1) NK cell killing phenomenon unique UPEC CFT073 <br />                                                         strain (2) NK cell killing relates type pathology caused different isolates <br />                                                         (cystitis versus pyelonephritis), incubated additional strains urinary pathogenic E. coli <br />                                                         (UTI536, ATCC 25922, UTI89) human NK cells. The various E. coli strains were <br />                                                         incubated various time points (Figure 1G) varying ratios bacteria NK cells <br />                                                         (Figure 1H). As seen Figure 1G, following 4 hr incubation SR71 and <br />                                                         XL1 NK cell death detected. At longer time points bacteria grow an <br />                                                         extent overabundance caused nonspecific NK bacteria death. Following 4 hr of <br />                                                         incubation CFT073, UTI536, UTI89, ATCC 25922 NK cells, nearly 100% <br />                                                         NK cell death observed (Figure 1G). The UTI536 bacterium exceptionally efficient <br />                                                         killing human NK cells, 60% NK cells killed following 2 hr <br />                                                         incubation strain, opposed 30% killing observed UPEC <br />                                                         &#226;&#8364;&#339;killer&#226;&#8364;? strains (Figure 1G). When repeating experiments 3 hr various bacteria to <br />                                                         NK cells ratios, UTI536 extremely efficient killing NK cells (Figure 1H). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         To test UPEC-mediated killing specific NK cells, incubated primary T <br />                                                         NK cells GFP-labeled UPEC CFT073 4&#194;&#176;C 37&#194;&#176;C (Figure 2). As be <br />                                                         seen, bacterium binds T NK cells equivalent levels (Figure 2A), T <br />                                                         cells (Figure 2B) completely resistant UPEC CFT073-mediated killing, <br />                                                         NK cells highly sensitive (Figure 2B). <br />  <br />                                             UPEC CFT073 Adheres NK Cells Type I Fimbriae Kills NK Cells HemolysinA <br />                                                         To identify component(s) UPEC adheres kill NK cells, generated <br />                                                         transposon-based insertion-inactivation mutant library GFP-expressing UPEC CFT073 <br />                                                         bacterium. To identify genes involved UPEC CFT073 adhesion, incubated the <br />                                                         mutant bacteria 4&#194;&#176;C, identify killing/lytic components, incubated the <br />                                                         mutants 3 hr NK cells 37&#194;&#176;C. More 1,000 clones screened. Nine UPEC <br />                                                         CFT073 mutants impaired adhesion (Figure 3A), NK cell <br />                                                         killing (Figure 3B), mutants identified, having intact adhesion ability <br />                                                         (Figure 3C) impaired killing activity (Figure 3D). One irrelevant mutant, B5, shown in <br />                                                         Figure 3 paper. In UPEC CFT073 adhesion mutants, the <br />                                                         transposon inserted operon encoding type I fimbriae, type I fimbriae- <br />                                                         associated genes (Figure 3E). The gene disrupted ninth mutant tRNA uridine <br />                                                         (Figure 3E). Interestingly, mutants completely lost adhesion potential, <br />                                                         seen, ~50% reduction adhesion sufficient kill NK cells <br />                                                         3 hr incubation 37&#194;&#176;C (Figures 3A 3B). The disrupted genes three <br />  <br />  <br />                                                            Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                             Page 4 <br />  <br />  <br />                                                         killing mutants identified (Figure 3E) reside hlyA operon <br />                                                         encodes hemolysinA toxin. Two mutants defective in <br />                                                         adhesion NK cells killing NK cells (Figures S2A S2B, respectively). <br />                                                         Sequencing disrupted genes revealed housekeeping genes (Figure S2C), <br />                                                         essential bacterial growth (Figure S2D). The growth biological <br />                                                         properties relevant mutants normal (Figure S2D). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             Increased Adhesion Type I Fimbriae Leads Increased Killing NK Cells <br />                                                         The expression fimbriae regulated invertible element alternates <br />                                                         &#226;&#8364;&#339;on&#226;&#8364;? &#226;&#8364;&#339;off&#226;&#8364;? orientation (Kaper et al., 2004; Ulett et al., 2013; Hannan et <br />                                                         al., 2008). To validate findings, additional GFP-expressing mutants UPEC <br />                                                         CFT073 bacteria tested: fimbriae locked &#226;&#8364;&#339;ON&#226;&#8364;? <br />                                                         orientation (Fim on) locked &#226;&#8364;&#339;OFF&#226;&#8364;? orientation (Fim off). In <br />                                                         addition, tested P-fimbria (Mobley et al., 1993) involved the <br />                                                         residual adhesion NK cells using GFP-expressing P-fimbriae mutant (PAP&#226;&#8364;&#8220;, also <br />                                                         known UPEC76). As seen (Figure 4A), Fim bacterium used, the <br />                                                         adhesion reduced 50%, absence P-fimbriae effect. The <br />                                                         binding Fim mutant stronger bacteria, 85% of <br />                                                         NK cells bound bacterium. <br />  <br />                                                         The increased adhesion observed Fim led increased killing NK cells. <br />                                                         As seen Figure 4B, ratio 30:1, 1 hr following incubation Fim on <br />                                                         bacterium NK cells 30% killing observed, practically no <br />                                                         killing detected bacteria. Following 4 hr incubation, 100% <br />                                                         killing NK cells detected bacteria, Fim bacterium, in <br />                                                         killing NK cells did exceed 70%. Regarding varying bacteria:NK cell <br />                                                         ratios, 7.5:1 ratio killing NK cells Fim bacterium very <br />                                                         efficient compared bacteria tested (Figure 4C). At 30:1 ratio, 80%&#226;&#8364;&#8220;90% <br />                                                         NK cells killed, depending bacteria used; however, (Figure 4B), <br />                                                         killing NK cells Fim bacterium did exceed 70% (Figure 4C). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             The Extent HemolysinA Activity Determines Efficiency NK Cell Killing <br />                                                         HemolysinA cytolytic pore-forming toxin contributes tissue injury and <br />                                                         erythrocyte hemolysis (Cavalieri et al., 1984; Gadeberg et al., 1983). We next <br />                                                         investigated bacteria killed NK cells able cause erythrocyte <br />                                                         hemolysis. As seen Figure 5A, clear zone blood agar plate evident with <br />                                                         &#226;&#8364;&#339;NK killer&#226;&#8364;? bacteria (Figure 1). In contrast, SR71 XL1 did perform hemolysis <br />                                                         (Figure 5A data shown, respectively) lacks hlyA gene (Totsika et al., <br />                                                         2009). <br />  <br />                                                         Type 1 fimbriae operon resides pathogenicity island hemolysinA a <br />                                                         frequent spot insertional mutations (Hannan et al., 2008). To confirm increased <br />                                                         killing seen Fim mutant (Figures 4B 4C) resulted better adhesion <br />                                                         properties mutant test higher killing seen UTI536 (Figures 1G <br />                                                         1H) strain expresses copies hemolysinA operon (Wiles <br />                                                         et al., 2008), quantified hemolysis sheep red blood cells (RBCs) performed. As can <br />                                                         seen Figure 5B, hemolysis levels bacteria tested equivalent, the <br />                                                         hemolysis induced UTI536 significantly higher. <br />  <br />                                                         To directly demonstrate hemolysinA responsible human NK cell killing, we <br />                                                         restored hemolysis <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> hemolysinA mutant bacteria transforming them <br />                                                         plasmid harboring entire UPEC CFT073 hemolysinA operon (the complemented <br />  <br />  <br />                                                            Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                 Page 5 <br />  <br />  <br />                                                         bacteria designated C93C, D57C, E49C). As seen Figure 5C, hemolysis <br />                                                         restored complemented bacteria, regained ability kill the <br />                                                         NK cells, determined flow cytometry (Figures 5D S3) confocal microscopy <br />                                                         (Figure 5E). <br />  <br />                                             NK Cells Accumulate Infected Bladder following UPEC CFT073 Infection <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         To investigate physiological relevance findings, initially isolated murine NK <br />                                                         cells incubated various UPEC strains hemolysinA <br />                                                         mutants. As seen, murine NK cells efficiently killed &#226;&#8364;&#339;killer&#226;&#8364;? bacteria, <br />                                                         killing observed hemolysinA mutants (C93, D57, E49) or <br />                                                         XL1 SR71 bacteria (Figure 6A). Interestingly, mouse T cells, opposed human <br />                                                         T cells (Figure 2), quite efficiently killed &#226;&#8364;&#339;killer&#226;&#8364;? bacteria (data shown). <br />  <br />                                                         We tested murine NK cells present bladder prior following <br />                                                         UPEC CFT073 infection. For this, used Ncr1gfp/+ mice generated, which <br />                                                         NK cells labeled GFP easily detected (Gazit et al., 2006). UPEC <br />                                                         CFT073 hemolysinA mutant C93 bacteria injected catheters inserted <br />                                                         bladders 6-to 8-week-old C57BL/6 Ncr1gfp/+ female mice, NK cells were <br />                                                         counted bladder prior following bacterial inoculation. We used C93 bacterium <br />                                                         following experiments mutant disrupted the <br />                                                         structural hlyA gene itself. The NK cell accumulation bladder monitored 6 <br />                                                         consecutive days post-UTI induction, harvesting bladders counting GFP- <br />                                                         positive NK cells using flow cytometry. In absence bacterial infection, NK cells were <br />                                                         hardly detected bladder; however, following bacterial inoculation, NK cells <br />                                                         accumulated bladder (Figure 6B), significantly NK cells observed in <br />                                                         mice inoculated hemolysinA mutant bacterium C93 compared wild-type <br />                                                         bacterium (Figure 6B). The highest number NK cells bladder observed day <br />                                                         2 following UTI induction (Figure 6B). At later time points, NK cells gradually <br />                                                         disappeared practically undetectable day 5 postinfection (data shown). <br />                                                         Cryosections bladders obtained day 2 post-UTI induction, NK cells in <br />                                                         bladder viewed using confocal microscopy (Figure 6C). In agreement flow <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         cytometry experiments (Figure 6B), NK cells detected PBS- <br />                                                         injected bladders, highest number NK cells observed following inoculation <br />                                                         hemolysinA C93 mutant (Figure 6C, depicting piece bladders&#226;&#8364;&#8482; uroepithelial <br />                                                         cells). Figure S4 illustrates following UPEC infection NK cells mainly in <br />                                                         uroepithelial cells, especially mesenchymal area. <br />  <br />                                             NK Cells Attenuate UTI TNF-&#206;&#177; <br />                                                         NK cells kill bacteria directly, incubation NK cells wild-type UPEC <br />                                                         CFT073 bacterium (Figure 1E) hemolysinA mutants (Figure 5E) did result <br />                                                         PI staining bacteria. Furthermore, decrease bacterial viability detected <br />                                                         following incubation various bacteria NK cells (Figure S5). To investigate <br />                                                         NK cells produce cytokines direct interaction bacteria, we <br />                                                         incubated mouse NK cells wild-type bacteria used work determined the <br />                                                         TNF-&#206;&#177; concentrations supernatants. NK cells incubated bacteria lacking the <br />                                                         hemolysinA toxin XL1 SR71, secreted high levels TNF-&#206;&#177; (Figure 7A), while <br />                                                         TNF-&#206;&#177; secreted NK cells incubated hemolytic UPEC strains <br />                                                         greatly reduced (Figure 7A). This NK cell killing hemolytic <br />                                                         UPEC strains (verified PI staining end ELISA assays, data shown). <br />                                                         Similar results obtained complemented hemolysinA bacteria little <br />                                                         secretion TNF-&#206;&#177; noted (data shown). <br />  <br />  <br />  <br />                                                            Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                           Page 6 <br />  <br />  <br />                                                      To identify NK cell receptor mediating TNF-&#206;&#177; secretion, initially investigated <br />                                                      major murine NK-activating receptors, NKG2D Ncr1, able to <br />                                                      directly recognize bacteria. To end, used Ncr1-Ig NKG2D-Ig fusion <br />                                                      proteins, seen Figure 7B, bacteria tested stained the <br />                                                      fusion proteins. In contrast, murine thymoma PD1.6 cells previously shown express <br />                                                      ligand Ncr1 (Halfteck et al., 2009) recognized Ncr1-Ig NKG2D-Ig, <br />                                                      Ncr1-ligand-negative YAC-1 cells (Gazit et al., 2006) recognized by <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      NKG2D-Ig (Figure 7B). It previously shown UPEC bacteria directly activate <br />                                                      NK cells, leading secretion TNF-&#206;&#177; TLR-4 (Mian et al., 2010). To test whether <br />                                                      TLR-4 involved TNF-&#206;&#177; secretion, incubated irrelevant mutant B5 the <br />                                                      hemolysinA mutant C93 bacteria murine NK cells. The C93 bacterium incubated <br />                                                      NK cells pretreated blocking anti-TLR-4 mAb. As seen, <br />                                                      incubation NK cells C93 led substantial TNF-&#206;&#177; secretion partially <br />                                                      reduced anti-TLR-4 mAb (Figure 7C), suggesting TLR-4 involved TNF- <br />                                                      &#206;&#177; secretion following direct interaction bacteria. <br />  <br />                                                      To investigate importance NK cells TNF-&#206;&#177; vivo, initially performed <br />                                                      calibration experiments bacterial loads evaluated bladders, kidneys, and <br />                                                      blood 6 days following bacterial inoculation, depletion NK cells <br />                                                      using anti-NK1.1 mAb. The efficiency anti-NK1.1 mAb depleting NK cells was <br />                                                      monitored using Ncr1gfp/+ mice generated (Gazit et al., 2006). As be <br />                                                      seen Figure 7D, NK cells present blood kidney, bladder, <br />                                                      prior bacteria inoculation. Upon bacterial injection, NK cell numbers were <br />                                                      reduced blood concomitantly increased kidneys bladders (Figure <br />                                                      7D). Injection anti-NK1.1 mAb led near-complete reduction numbers NK <br />                                                      cells present blood, kidneys, bladders (Figure 7D). Following calibration <br />                                                      experiments, investigated role played NK cells TNF-&#206;&#177; UTI. Three groups <br />                                                      mice catheterized: group 1 pretreated anti NK1.1 antibody, 1 day prior to <br />                                                      inoculation bacteria 1 day following catheter inoculation; group 2 was <br />                                                      pretreated soluble receptor TNF-&#206;&#177; (Etanercept) day bacterial inoculation <br />                                                      following day; group 3 left untreated (Figure 7E). The groups mice <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      divided subgroups: group received PBS, second group <br />                                                      received wild-type UPEC CFT073 bacterium, group inoculated the <br />                                                      irrelevant control mutant B5, fourth group inoculated hemolysinA <br />                                                      mutant C93. The bladders kidneys harvested day 2, according our <br />                                                      calibration experiments day highest number bacteria present the <br />                                                      bladder, highest number NK cells. As seen Figure 7F, groups <br />                                                      mice did undergo prior treatments injected UPEC CFT073 or <br />                                                      irrelevant mutant (B5) significantly bacteria compared C93- <br />                                                      injected group. Importantly, NK cell-depleted groups (the NK cell depletion was <br />                                                      verified blood of NK1.1-treated mice, prior removal bladders) <br />                                                      Etanercept (TNF-&#206;&#177; blocker)-treated groups bacterial load similar to <br />                                                      untreated B5 UPEC CFT073 groups (Figure 7F). This indicates NK cells and <br />                                                      TNF-&#206;&#177; play significant role attenuating/controlling bacterial infection bladder in <br />                                                      vivo. The UPEC CFT073 bacterium mutant C93 bacterium hardly detected <br />                                                      kidneys, irrespective NK cells depleted TNF-&#206;&#177; was <br />                                                      blocked (Figure 7G). To demonstrate directly NK cells responsible TNF-&#206;&#177; <br />                                                      secretion following bacterial inoculation, took bladder extracts used determine the <br />                                                      bacterial loads Figure 7F measured TNF-&#206;&#177; concentrations. The untreated bladders <br />                                                      injected C93 mutant lowest CFU (Figure 7F), accordingly levels of <br />                                                      TNF-&#206;&#177; highest (Figure 7H). In contrast, untreated mice injected B5 <br />                                                      mutant, bacterial loads elevated (Figure 7F), TNF-&#206;&#177; levels reduced by <br />                                                      50% (Figure 7H). When NK cells depleted, TNF-&#206;&#177; levels equivalent those <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                               Page 7 <br />  <br />  <br />                                                      observed following B5 inoculation, indicating response UPEC CFT073 infection, <br />                                                      absence hemolysinA, NK cells secrete TNF-&#206;&#177; vivo. <br />  <br />                                                      During acute cystitis, UPEC invade bladder epithelial cells regulated process, named <br />                                                      &#226;&#8364;&#339;developmental cycle&#226;&#8364;? (Jorgensen Seed, 2012), includes bacterial adhesion, <br />                                                      invasion, formation intracellular bacterial communities (IBCs) release <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                      tissue exfoliation. Because NK cells observed significant numbers the <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      bladders 1 day following infection (Figure 6B), assumed NK cells not <br />                                                      involved initial UPEC adhesion invasion uroepithelial cells. We therefore <br />                                                      investigated, using TEM, IBC observed bladders infected C93 <br />                                                      mutant bacterium (as growth bacterium NK cell sensitive; Figure 7F). The <br />                                                      experiments performed day 2, peak NK cell accumulation. As seen <br />                                                      Figure 7I, IBCs observed infected bladders following C93 inoculation, <br />                                                      demonstrating NK cells probably inhibit IBC formation. <br />  <br />                                             DISCUSSION <br />                                                      We started research noticing UPEC CFT073 adheres NK cells that <br />                                                      binding bacterium led killing NK cells. The killing dependent the <br />                                                      type urinary tract infection, i.e., cystitis pyelonephritis, occurred the <br />                                                      bacteria expressed hemolysinA protein. The importance findings highlighted <br />                                                      fact UPEC strains common etiological agents responsible for <br />                                                      uncomplicated UTI healthy women (Ulett et al., 2013). Mutant screening <br />                                                      performed 4&#194;&#176;C identified mutants defective adhesion NK cells. Interestingly, <br />                                                      adhesion mutants defective type I fimbriae genes in <br />                                                      type I fimbriae-related genes. Type I fimbriae produced 80% UPEC <br />                                                      strains required bacterial attachment mannose residues glycoprotein receptor <br />                                                      uroplakin Ia surface urinary epithelium cells bacterial invasion and <br />                                                      persistence target cells (Bouckaert et al., 2006). <br />  <br />                                                      After screening 1,000 bacterial mutants, did mutants were <br />                                                      defective adhesions used UPEC adhere uroepithelium P <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      (Pap), F1C, S, M, Dr (Stapleton, 2005). Thus, concluded adhesion <br />                                                      molecules probably involved UPEC CFT073 adhesion NK cells. <br />                                                      Interestingly, adhesion mutants presented complete absence attachment. <br />                                                      This explained presence adhesion protein, composed of <br />                                                      redundant subunits, nonreceptor-mediated binding UPEC CFT073 NK <br />                                                      cells. Nevertheless, type I fimbriae play significant role bacterial adhesion NK <br />                                                      cells, mutant fimbriae locked &#226;&#8364;&#339;on&#226;&#8364;? orientation used, <br />                                                      approximately 90% NK cells bound bacteria, led efficient killing <br />                                                      NK cells. <br />  <br />                                                      Three mutants identified completely defective NK killing ability, and <br />                                                      mutants disrupted different genes belonging hemolysin operon (hlyA, <br />                                                      hlyC, hlyD). Importantly, hemolysinA associated human pathogenic strains E. <br />                                                      coli causing clinically severe forms UTI (Gadeberg et al., 1983). Approximately half of <br />                                                      UPEC strains cause upper UTIs (pyelonephritis) cause <br />                                                      lower UTIs (cystitis) express hemolysinA. This contrast 10% fecal <br />                                                      isolates express hemolysinA. The hemolysinA protein unique E. coli toxins <br />                                                      secreted membranes (Cavalieri et al., 1984). The ability the <br />                                                      hemolysin toxin secreted outer-membrane vesicles (Balsalobre et al., 2006) may <br />                                                      explain adhesion mutants able efficiently kill NK cells also <br />                                                      explain dead NK cells seen presence UPEC CFT073, the <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                               Page 8 <br />  <br />  <br />                                                      bacteria directly attached them. Still, mentioned above, efficient killing of <br />                                                      NK cells bacterial adhesion NK cells required. <br />  <br />                                                      NK cells particularly sensitive hemolysin-mediated killing, human T <br />                                                      cells completely resistant. The reason hemolysinA-mediated <br />                                                      hypersensitivity NK cells unknown. Virtually known sensitivity of <br />                                                      lymphocytes hemolysinA-mediated killing. A literary search identified paper, from <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      1983 (Gadeberg et al., 1983), presents data supports findings. The authors <br />                                                      showed hemolysinA toxic monocytes granulocytes, lymphocytes were <br />                                                      relatively resistant, NK cells, discovered decade <br />                                                      beforehand, tested (Gadeberg et al., 1983). Indeed, observed that <br />                                                      neutrophils efficiently killed hemolysinA-expressing bacteria (data shown). <br />  <br />                                                      It previously shown hemolysinA trigger rapid degradation paxillin as <br />                                                      host proteins causes serine proteases caspase activation which <br />                                                      promote apoptosis host cells (Dhakal Mulvey, 2012). This explains why, in <br />                                                      addition lysing erythrocytes killing NK cells (shown here), hemolysinA toxic to <br />                                                      range host cells. Interestingly, shown (Dhakal Mulvey, 2012; K&#195;&#182;nig et <br />                                                      al., 1994) addition tissue injury, hemolysinA indirectly disrupt various aspects <br />                                                      host cell signaling suppress cytokine (such IL-6 TNF-&#206;&#177;) release human <br />                                                      lymphocytes/monocytes basophils. Thus, hemolysinA impairs TNF-&#206;&#177; activity in <br />                                                      numerous ways: killing NK cells directly, shown here, indirectly, impairing <br />                                                      host cell signaling shown previously. <br />  <br />                                                      To evaluate role played NK cells UTI, initially established NK cells are <br />                                                      absent bladder infection present accumulate bladder <br />                                                      UTI induction. Because NK cell numbers decreased blood following the <br />                                                      bacterial inoculation, suggest NK cells recruited bladder kidney, at <br />                                                      partially, blood. <br />  <br />                                                      Mice best model studying UTI, abundantly express TLR-11 their <br />                                                      bladder kidneys, TLR absent humans (Lauw et al., 2005; Zhang et al., <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      2004). This leads TLR-11-dependent bacterial clearance (Zhang et al., 2004). Indeed, we <br />                                                      observed 5&#226;&#8364;&#8220;6 days postinfection mice free bacteria, irrespective whether <br />                                                      NK cells present absent independently type injected bacteria. <br />                                                      Nevertheless, despite limitations vivo model, able convincingly <br />                                                      demonstrate NK cells important UTI attenuation vivo. While direct killing <br />                                                      bacteria observed, demonstrated vitro vivo NK cells secrete <br />                                                      large amounts TNF-&#206;&#177; response nonhemolytic UPEC infection. When blocked <br />                                                      TLR-4, observed partial reduction TNF-&#206;&#177; secretion, suggesting TLR-4 partially <br />                                                      participates production TNF-&#206;&#177; NK cells following direct interaction UPEC <br />                                                      receptor(s) identity currently unknown sense UPEC and <br />                                                      secrete TNF-&#206;&#177;. Supporting observations, shown NK cells secrete TNF-&#206;&#177; <br />                                                      TLR-4 response stimulation FimH LPS UPEC (Mian et al., 2010). <br />                                                      Finally, demonstrate IBCs observed bladders mice infected <br />                                                      NK-sensitive strain C93. It interesting thoroughly study future <br />                                                      NK cells affect UPEC developmental cycle uroepithelial cells. <br />  <br />                                                      In summary, discovered host defense microbial counterattack mechanism which <br />                                                      NK cells control UTI UPEC retaliates killing NK cells using hemolysinA. <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                Page 9 <br />  <br />  <br />                                             EXPERIMENTAL PROCEDURES <br />                                             Mice <br />                                                         All experiments performed using 6- 8-week-old C57BL/6 female mice. The <br />                                                         generation Ncr1gfp/+ mice described previously (Gazit et al., 2006). All <br />                                                         experiments performed accordance guidelines local ethics committee. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             Cells, Plasmids, Bacterial Strains <br />                                                         Primary human mouse NK cells isolated PBLs various healthy donors and <br />                                                         various mice, respectively, using human mouse NK cell isolation kits the <br />                                                         autoMACS instrument according manufacturer instructions (Miltenyi Biotec). Some of <br />                                                         bacterial strains plasmids used study described Figure S1 in <br />                                                         Tables S1A S1B. <br />  <br />                                             Creation UPEC CFT073 Transposon Library, Mutant Screen, Bacterial Adhesion, and <br />                                             Killing Assays <br />                                                         EZ::TnKan transposomes (Bahar et al., 2009) purified pMODKan plasmid <br />                                                         according manufacturer&#226;&#8364;&#8482;s instructions electroporated UPEC CFT073 bacteria <br />                                                         harboring GFP pCM18 plasmid. Primary NK cells (either human mice) washed <br />                                                         twice RPMI 1640 antibiotics incubated various bacteria (mutants <br />                                                         wild-type) various bacterial NK cells ratios indicated. Incubations were <br />                                                         performed, various time points, indicated, usually, 3 hr, 4&#194;&#176;C determine <br />                                                         adhesion 37&#194;&#176;C determine NK cell killing, cells analyzed flow <br />                                                         cytometry. <br />  <br />                                             Determining Transposon Location <br />                                                         Genomic DNA purified selected clones (GenElute, Sigma) restricted with <br />                                                         HpaI, MfeI, NdeI (New England Biolabs) cleave inserted <br />                                                         transposon. The resulting fragments self-ligated (TaKaRa) used template <br />                                                         inverse PCR (Herculase II, Agilent) amplification transposons&#226;&#8364;&#8482; flanking <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         sequences. The generated sequences blasted CFT073 genome determine <br />                                                         location <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>insertion mutation</span>. <br />  <br />                                             Hemolysis Hemolysis Quantification <br />                                                         Bacteria streaked blood agar plates supplemented appropriate antibiotic, <br />                                                         hemolysis determined visually clear zone colony. Colonies were <br />                                                         viewed using Zeiss stemi SVII microscope 30.6 magnification photographed <br />                                                         using Infinity I camera. Hemolysis quantification performed described previously <br />                                                         (Warawa et al., 1999). <br />  <br />                                             In Vivo Assays Determination Bacterial Loads <br />                                                         Female mice 6&#226;&#8364;&#8220;8 weeks age anesthetized Ketamine Xylazine before <br />                                                         catheter insertion infected 108 bacteria 50 &#206;&#188;l sterile PBS. Tissue samples <br />                                                         homogenized using Fastprep (MP Biomedicals, USA), serially diluted, spread on <br />                                                         Maconkey plates (Novamed, Israel). Plates incubated overnight, colonies were <br />                                                         enumerated. <br />  <br />  <br />  <br />  <br />                                                            Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                               Page 10 <br />  <br />  <br />                                             Cytokine Secretion Assays <br />                                                         Various bacteria incubated murine NK cells ratio 10:1 5 hr 37&#194;&#176;C, 5% <br />                                                         CO2. Supernatants collected, TNF-&#206;&#177; levels determined using standard <br />                                                         ELISA. <br />  <br />                                             NK Cell Depletion Etanercept Treatment <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Mice injected i.p 100 &#206;&#188;g anti-NK1.1 Ab (PK136) 200 &#206;&#188;l sterile PBS. The <br />                                                         mAb injected 1 day prior 1 day bacterial inoculation. Sterile PBS was <br />                                                         injected control. Depletion verified treated mice, prior bladder removal. To <br />                                                         block TNF-&#206;&#177;, soluble receptor (Etanercept) injected. Each mouse received 0.4 mg of <br />                                                         Etanercept day, day catheterization, following bacterial inoculation 1 <br />                                                         day later. <br />  <br />                                             Statistical Methods <br />                                                         For statistical significance, Student&#226;&#8364;&#8482;s t test analysis used. A statistical test was <br />                                                         considered significant (*) p &lt; 0.05. ANOVA used identify significant group <br />                                                         differences. <br />  <br />                                             Supplementary Material <br />                                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                             Acknowledgments <br />                                                         This study supported Advanced ERC grant, The Israeli Science Foundation, The Israeli- I-CORE, <br />                                                         GIF Foundation, The ICRF Professorship Grant (all O.M.) Israeli Science Foundation <br />                                                         (to G.B.). The work supported Israeli Science Foundation (to C.G.). O.M. Crown Professor of <br />                                                         Molecular Immunology. C.G. supported Foulkes Foundation. The authors like thank Professors <br />                                                         Hirotada Mori Toru Nakayashiki useful discussion. The authors like thank Professor Saul <br />                                                         Burdman, kindly supplied pMODkan plasmid. The pGNH404 plasmid gift Professor Agneta <br />                                                         Richter-Dahlfors. Bacterial strains CFT073, UPEC76, CFT073ON, CFT073OFF, CFT fim/pap from <br />                                                         Professor Harry Mobley&#226;&#8364;&#8482;s collection. The authors thank Professor Ilan Rosenshine EPEC E2348/69 strain <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Professor Nahum Shpigel UTI89 UTI536 strains. The authors grateful EM <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> Core <br />                                                         Research Facility Faculty Medicine, The Hebrew University Jerusalem, assistance electron <br />                                                         microscopy. <br />  <br />  <br />                                             REFERENCES <br />                                                         Bahar O, Goffer T, Burdman S. Type IV Pili required virulence, twitching motility, biofilm <br />                                                           formation acidovorax avenae subsp. Citrulli. Mol. Plant Microbe Interact. 2009; 22:909&#226;&#8364;&#8220;920. <br />                                                           [PubMed: 19589067] <br />                                                         Balsalobre C, Silv&#195;&#161;n JM, Berglund S, Mizunoe Y, Uhlin BE, Wai SN. Release type I secreted <br />                                                           alpha-haemolysin outer membrane vesicles Escherichia coli. Mol. Microbiol. 2006; 59:99&#226;&#8364;&#8220; <br />                                                           112. [PubMed: 16359321] <br />                                                         Bouckaert J, Mackenzie J, Paz JL, Chipwaza B, Choudhury D, Zavialov A, Mannerstedt K, <br />                                                           Anderson J, Pi&#195;&#169;rard D, Wyns L, et al. The affinity FimH fimbrial adhesin receptor-driven <br />                                                           quasi-independent Escherichia coli pathotypes. Mol. Microbiol. 2006; 61:1556&#226;&#8364;&#8220;1568. <br />                                                           [PubMed: 16930149] <br />                                                         Cavalieri SJ, Bohach GA, Snyder IS. Escherichia coli alpha-hemolysin: characteristics probable <br />                                                           role pathogenicity. Microbiol. Rev. 1984; 48:326&#226;&#8364;&#8220;343. [PubMed: 6394977] <br />                                                         Chaushu S, Wilensky A, Gur C, Shapira L, Elboim M, Halftek G, Polak D, Achdout H, Bachrach G, <br />                                                           Mandelboim O. Direct recognition Fusobacterium nucleatum NK cell natural cytotoxicity <br />                                                           receptor NKp46 aggravates periodontal disease. PLoS Pathog. 2012; 8:e1002601. [PubMed: <br />                                                           22457623] <br />  <br />  <br />  <br />                                                             Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                      Page 11 <br />  <br />  <br />                                                      Dhakal BK, Mulvey MA. The UPEC pore-forming toxin &#206;&#177;-hemolysin triggers proteolysis host <br />                                                         proteins disrupt cell adhesion, inflammatory, <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>survival</span> pathways. Cell Host Microbe. 2012; <br />                                                         11:58&#226;&#8364;&#8220;69. [PubMed: 22264513] <br />                                                      Emody L, Ker&#195;&#169;nyi M, Nagy G. Virulence factors uropathogenic Escherichia coli. Int. J. Antimicrob. <br />                                                         Agents. 2003; 22(Suppl 2):29&#226;&#8364;&#8220;33. [PubMed: 14527768] <br />                                                      Engel D, Dobrindt U, Tittel A, Peters P, Maurer J, G&#195;&#188;tgemann I, Kaissling B, Kuziel W, Jung S, Kurts <br />                                                         C. Tumor necrosis factor alpha- inducible nitric oxide synthase-producing dendritic cells are <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         rapidly recruited bladder urinary tract infection dispensable bacterial clearance. <br />                                                         Infect. Immun. 2006; 74:6100&#226;&#8364;&#8220;6107. [PubMed: 16966414] <br />                                                      Gadeberg OV, Orskov I, Rhodes JM. Cytotoxic effect alpha-hemolytic Escherichia coli strain on <br />                                                         human blood monocytes granulocytes vitro. Infect. Immun. 1983; 41:358&#226;&#8364;&#8220;364. [PubMed: <br />                                                         6345395] <br />                                                      Gazit R, Gruda R, Elboim M, Arnon TI, Katz G, Achdout H, Hanna J, Qimron U, Landau G, <br />                                                           Greenbaum E, et al. Lethal influenza infection absence natural killer cell receptor <br />                                                           gene Ncr1. Nat. Immunol. 2006; 7:517&#226;&#8364;&#8220;523. [PubMed: 16565719] <br />                                                      Halfteck GG, Elboim M, Gur C, Achdout H, Ghadially H, Mandelboim O. Enhanced vivo growth of <br />                                                           lymphoma tumors absence NK-activating receptor NKp46/NCR1. J. Immunol. 2009; <br />                                                           182:2221&#226;&#8364;&#8220;2230. [PubMed: 19201876] <br />                                                      Hannan TJ, Mysorekar IU, Chen SL, Walker JN, Jones JM, Pinkner JS, Hultgren SJ, Seed PC. LeuX <br />                                                           tRNA-dependent -independent mechanisms Escherichia coli pathogenesis acute cystitis. <br />                                                           Mol. Microbiol. 2008; 67:116&#226;&#8364;&#8220;128. [PubMed: 18036139] <br />                                                      Haraoka M, Hang L, Frend&#195;&#169;us B, Godaly G, Burdick M, Strieter R, Svanborg C. Neutrophil <br />                                                           recruitment resistance urinary tract infection. J. Infect. Dis. 1999; 180:1220&#226;&#8364;&#8220;1229. <br />                                                           [PubMed: 10479151] <br />                                                      Jewett A, Gan XH, Lebow LT, Bonavida B. Differential secretion TNF-alpha IFN-gamma by <br />                                                           human peripheral blood-derived NK subsets association functional maturation. J. Clin. <br />                                                           Immunol. 1996; 16:46&#226;&#8364;&#8220;54. [PubMed: 8926285] <br />                                                      Johnson JR. Virulence factors Escherichia coli urinary tract infection. Clin. Microbiol. Rev. 1991; <br />                                                           4:80&#226;&#8364;&#8220;128. [PubMed: 1672263] <br />                                                      Jones-Carson J, Balish E, Uehling DT. Susceptibility immunodeficient gene-<span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>knockout</span> mice to <br />                                                           urinary tract infection. J. Urol. 1999; 161:338&#226;&#8364;&#8220;341. [PubMed: 10037434] <br />                                                      Jorgensen I, Seed PC. How make urinary tract: tutorial Escherichia coli. PLoS Pathog. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           2012; 8:e1002907. [PubMed: 23055921] <br />                                                      Kaper JB, Nataro JP, Mobley HL. Pathogenic Escherichia coli. Nat. Rev. Microbiol. 2004; 2:123&#226;&#8364;&#8220;140. <br />                                                           [PubMed: 15040260] <br />                                                      Koch J, Steinle A, Watzl C, Mandelboim O. Activating natural cytotoxicity receptors natural killer <br />                                                           cells cancer infection. Trends Immunol. 2013; 34:182&#226;&#8364;&#8220;191. [PubMed: 23414611] <br />                                                      K&#195;&#182;nig B, Ludwig A, Goebel W, K&#195;&#182;nig W. Pore formation Escherichia coli alpha-hemolysin: <br />                                                           role mediator release human inflammatory cells. Infect. Immun. 1994; 62:4611&#226;&#8364;&#8220;4617. <br />                                                           [PubMed: 7523294] <br />                                                      Kucheria R, Dasgupta P, Sacks SH, Khan MS, Sheerin NS. Urinary tract infections: new insights into <br />                                                           common problem. Postgrad. Med. J. 2005; 81:83&#226;&#8364;&#8220;86. [PubMed: 15701738] <br />                                                      Lauw FN, Caffrey DR, Golenbock DT. Of mice man: TLR11 (finally) finds profilin. Trends <br />                                                           Immunol. 2005; 26:509&#226;&#8364;&#8220;511. [PubMed: 16111920] <br />                                                      Mian MF, Lauzon NM, Andrews DW, Lichty BD, Ashkar AA. FimH directly activate human and <br />                                                           murine natural killer cells TLR4. Mol. Ther. 2010; 18:1379&#226;&#8364;&#8220;1388. [PubMed: 20442710] <br />                                                      Mobley HL, Jarvis KG, Elwood JP, Whittle DI, Lockatell CV, Russell RG, Johnson DE, Donnenberg <br />                                                           MS, Warren JW. Isogenic P-fimbrial deletion mutants pyelonephritogenic Escherichia coli: the <br />                                                           role alpha Gal(1-4) beta Gal binding virulence wild-type strain. Mol. Microbiol. 1993; <br />                                                           10:143&#226;&#8364;&#8220;155. [PubMed: 7968511] <br />                                                      Ragnarsd&#195;&#179;ttir B, Lutay N, Gr&#195;&#182;nberg-Hernandez J, K&#195;&#182;ves B, Svanborg C. Genetics innate immunity <br />                                                           UTI susceptibility. Nat. Rev. Urol. 2011; 8:449&#226;&#8364;&#8220;468. [PubMed: 21750501] <br />                                                      Seidel E, Glasner A, Mandelboim O. Virus-mediated inhibition natural cytotoxicity receptor <br />                                                           recognition. Cell. Mol. Life Sci. 2012; 69:3911&#226;&#8364;&#8220;3920. [PubMed: 22547090] <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                                 Page 12 <br />  <br />  <br />                                                      Sivick KE, Schaller MA, Smith SN, Mobley HL. The innate immune response uropathogenic <br />                                                          Escherichia coli involves IL-17A murine model urinary tract infection. J. Immunol. 2010; <br />                                                          184:2065&#226;&#8364;&#8220;2075. [PubMed: 20083670] <br />                                                      Stapleton A. Novel mechanism P-fimbriated Escherichia coli virulence pyelonephritis. J. Am. <br />                                                          Soc. Nephrol. 2005; 16:3458&#226;&#8364;&#8220;3460. [PubMed: 16280466] <br />                                                      Totsika M, Heras B, Wurpel DJ, Schembri MA. Characterization homologous disulfide bond <br />                                                          systems involved virulence factor biogenesis uropathogenic Escherichia coli CFT073. J. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                          Bacteriol. 2009; 191:3901&#226;&#8364;&#8220;3908. [PubMed: 19376849] <br />                                                      Ulett GC, Totsika M, Schaale K, Carey AJ, Sweet MJ, Schembri MA. Uropathogenic Escherichia coli <br />                                                          virulence innate immune responses urinary tract infection. Curr. Opin. Microbiol. <br />                                                          2013; 16:100&#226;&#8364;&#8220;107. [PubMed: 23403118] <br />                                                      Warawa J, Finlay BB, Kenny B. Type III secretion-dependent hemolytic activity enteropathogenic <br />                                                          Escherichia coli. Infect. Immun. 1999; 67:5538&#226;&#8364;&#8220;5540. [PubMed: 10496946] <br />                                                      Wiles TJ, Kulesus RR, Mulvey MA. Origins virulence mechanisms uropathogenic Escherichia <br />                                                          coli. Exp. Mol. Pathol. 2008; 85:11&#226;&#8364;&#8220;19. [PubMed: 18482721] <br />                                                      Zhang D, Zhang G, Hayden MS, Greenblatt MB, Bussey C, Flavell RA, Ghosh S. A toll-like receptor <br />                                                          prevents infection uropathogenic bacteria. Science. 2004; 303:1522&#226;&#8364;&#8220;1526. [PubMed: <br />                                                          15001781] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                        Page 13 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 1. UPEC Kill Human NK Cells <br />                                                      (A) Flow cytometry analysis various GFP-expressing strains E. coli (indicated above <br />                                                      quadrants) incubated primary human NK cells stained PI (y axis). The <br />                                                      bacterium NK ratio 30:1, incubations performed 3 hr 4&#194;&#176;C <br />                                                      (upper quadrants) 37&#194;&#176;C (lower quadrants). The percentages NK cells each <br />                                                      quadrant indicated. <br />                                                      (B) Forward scatter analysis NK cells incubated UPEC CFT073, <br />                                                      4&#194;&#176;C 37&#194;&#176;C (indicated inside quadrants). Gate 1 shows live human NK cells, and <br />                                                      gate 2 shows nonviable ones. <br />                                                      (C) Ratios &#195;&#8212;100 viable NK cells present gate 1 following 3 hr incubation the <br />                                                      indicated bacteria 37&#194;&#176;C divided number NK cells incubated same <br />                                                      bacteria 3 hr 4&#194;&#176;C. <br />                                                      (D) Percentages PI-positive NK cells present gate 2 following 3 hr incubation 37&#194;&#176;C <br />                                                      various GFP expression E. coli strains, indicated x axis. <br />                                                      (E) Confocal microscopy PI-stained human NK cells incubated different GFP <br />                                                      expressing E. coli strains 3 hr 37&#194;&#176;C. The different bacteria indicated green <br />                                                      staining, DAPI blue, PI red. Magnification &#195;&#8212;1,800. <br />                                                      (F) Confocal microscopy, SEM, TEM human NK cells incubated UPEC <br />                                                      CFT073 3 hr 37&#194;&#176;C. The bacterium NK cells ratio 30:1. The bacteria seen the <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                          Page 14 <br />  <br />  <br />                                                      confocal microscopy indicated green staining, PI blue, anti-CD56 <br />                                                      staining red. The TEM SEM pictures marked. <br />                                                      (G) Shown percentages PI-positive NK cells following incubation NK cells 37&#194;&#176;C <br />                                                      indicated bacteria various time points (indicated x axis). The bacterium to <br />                                                      NK cells ratio 30:1. <br />                                                      (H) Percentages PI-positive NK cells following 3 hr incubation 37&#194;&#176;C indicated <br />                                                      E. coli strains bacteria NK cells ratios (indicated x axis). Figures show <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      representative experiment independent experiments performed and <br />                                                      average &#194;&#177;SD duplicate (C), (D), (G), (H). For (C) (D), *p &lt; 0.00001 for <br />                                                      UPEC CFT073 versus bacteria. For (G) (H) *p &lt; 0.01 UTI536 versus all <br />                                                      bacteria. 25922 represent ATCC 25922. See Table S1 Figure S1. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                        Page 15 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 2. UPEC CFT073 Preferentially Kills NK Cells <br />                                                      (A) Adhesion UPEC CFT073 human NK T cells determined percentages of <br />                                                      GFP-positive cells following 3 hr incubation 4&#194;&#176;C. <br />                                                      (B) Killing human NK T cells following 3 hr incubation 37&#194;&#176;C UPEC CFT073 <br />                                                      determined percentages cells stained PI. Figures representative of <br />                                                      independent experiments, average &#194;&#177;SD duplicate presented. *p &lt; 0.0005. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                       Page 16 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 3. Transposon-Based Mutagenesis Identifies Type I Fimbriae HemolysinA <br />                                                      (A) More 1,000 GFP-expressing UPEC CFT073 mutants screened adhesion to <br />                                                      primary human NK cells 4&#194;&#176;C (A C, determined percentages GFP-positive NK <br />                                                      cells) killing human NK cells 37&#194;&#176;C (B D determined PI-positive NK <br />                                                      cells). B5, irrelevant mutant UPEC CFT073 used control. Data are <br />                                                      representative independent analyses relevant mutants. Data are <br />                                                      represented mean &#194;&#177; SEM. *p &lt; 0.001. (E) Table summarizing transposon location in <br />                                                      disrupted genes relevant mutants. The adhesion mutants white, hemolysinA <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      mutants dark gray, irrelevant mutant B5 (bottom) light gray. See also <br />                                                      Figure S2. <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                          Page 17 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 4. Increased Type I Fimbriae-Mediated Attachment Leads Increase Killing of <br />                                                      Human NK Cells <br />                                                      (A) Percentages binding human NK cells following 3 hr incubation GFP- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      expressing bacteria indicated x axis. Binding performed 4&#194;&#176;C was <br />                                                      calculated percentages GFP-positive NK cells. *p &lt; 0.0001 Fim versus all <br />                                                      bacteria, **p &lt; 0.01 B5 PAP&#226;&#8364;&#8220; versus A15, D33, Fim off. <br />                                                      (B C) Killing human NK cells 37&#194;&#176;C following incubation GFP-labeled <br />                                                      bacteria, indicated legend, time points (B) various bacteria NK <br />                                                      cells (E:T) ratios (C). The percentages NK cell killing various bacteria were <br />                                                      determined PI staining. Data representative independent experiments and <br />                                                      average &#194;&#177; SD duplicate shown. For (B) (C), *p &lt; 0.001 Fim bacteria <br />                                                      versus bacteria. <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                       Page 18 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 5. Hemolytic Bacteria Kill NK Cells <br />                                                      (A) Hemolysis sheep RBCs mediated CFT073, ATCC 25922, UTI89, UTI536, <br />                                                      SR71 UPEC strains streaked blood agar plates. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      (B) Quantification sheep RBCs hemolysis (represented y axis O.D) induced by <br />                                                      UPEC CFT073, Fim Fim mutants, UPEC UTI536 strains. RBCs <br />                                                      treated 1% triton used positive control, treatment RPMI (phenol red <br />                                                      minus) used negative control. *p &lt; 0.01 UTI536 1% triton versus other <br />                                                      bacteria. <br />                                                      (C) Hemolysis mediated hemolysinA mutants (C93, D57, E49, upper panels) <br />                                                      complemented hemolysinA bacteria (C93C, D57C, E49C, lower panels). <br />                                                      (D E) Killing NK cells wild-type UPEC CFT073 bacterium, hemolysinA <br />                                                      mutants (C93, D57, E49) complemented bacteria (C93C, D57C, E49C) <br />                                                      37&#194;&#176;C, determined flow cytometry (D) using confocal microscopy (E). One <br />                                                      (A, C, E) (B D) independent experiments shown. An average &#194;&#177;SD <br />                                                      duplicate shown (B) (D). In (D), *p &lt; 0.0001 UPEC CFT073 E49C, <br />                                                      D57C, C93C versus E49, D57, C93. See Figure S3. <br />  <br />  <br />  <br />  <br />                                                         Cell Host Microbe. Author manuscript; available PMC 2014 June 11. <br />                                          Gur et al.                                                                                       Page 19 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      Figure 6. NK Cells Accumulate Bladder following UPEC CFT073 Infection <br />                                                      (A) Percentages PI-positive murine NK cells incubated 37&#194;&#176;C 3 hr wild-type <br />                                                      hemolytic (CFT073, ATCC 25922, UTI89, UTI536) nonhemolytic (XL1, SR71) UPEC <br />                                                      bacteria, irrelevant mutant B5 hemolysinA-deficient mutants (C93, D57, <br />                                                      E49). *p &lt; 0.005 CFT073, B5, ATCC 25922, UTI89, UTI536 versus other <br />                                                      bacteria. <br />                                                      (B) Number GFP-positive NK cells bladder day 1 day 2 following <br />                                                      infection UPEC CFT073 hemolysinA-deficient C93 mutant. Injection of <br />                                                      PBS used control (five mice used group). *p &lt; 0.005 NK <br />                                                      cells present bladders injected C93 day 2 versus day 1 versus day 2- <br />                                                      injected UPEC CFT073 bladders. <br />                                                      (C) Representative confocal microscopy images uroepithelial cells obtained murine <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                      bladders injected PBS (top), wild-type UPEC CFT073 (middle), the <br />                                                      hemolysinA-deficient mutant C93 (bottom). NK cells visualized GFP. Left panels <br />                                                      depict merge staining DAPI-stained bladder cells GFP-labeled NK cells; right <br />                                                      panels NK cells only. For (A) (B), data representative three <br />                                                      independent experiments, average &#194;&#177;SD triplicate shown. Slides confocal <br />                                                      microscopy prepared bladders mice group t <br /> </body></html>