<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Ptenb Mediates Gastrulation Cell Movements Cdc42/ <br /> AKT1 Zebrafish <br /> Chen-Min Yeh1,2., Yi-Ching Liu1,2., Ching-Jen Chang1, Shih-Lei Lai1, Chung-Der Hsiao4, Shyh-Jye <br /> Lee1,2,3* <br /> 1 Institute Zoology, National Taiwan University, Taipei, Taiwan, Republic China, 2 Center Biotechnology, National Taiwan University, Taipei, Taiwan, Republic of <br /> China, 3 Department Life Science, National Taiwan University, Taipei, Taiwan, Republic China, 4 Department Bioscience Technology, Chung Yuan Christian <br /> University, Chung-Li, Taiwan, Republic China <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Phosphatidylinositol 3-kinase (PI3 kinase) mediates gastrulation cell migration zebrafish regulation PIP2/PIP3 <br />      balance. Although PI3 kinase counter enzyme PTEN reported essential gastrulation, role in <br />      zebrafish gastrulation controversial lack gastrulation defects pten-<span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> clarify issue, <br />      knocked pten isoform, ptenb using anti-sense morpholino oligos (MOs) zebrafish embryos that <br />      ptenb MOs inhibit convergent extension affecting cell motility protrusion gastrulation. The ptenb MO-induced <br />      convergence defect rescued PI3-kinase inhibitor, LY294002 overexpressing dominant negative Cdc42. <br />      Overexpression human constitutively active akt1 showed similar convergent extension defects zebrafish embryos. We <br />      observed clear enhancement actin polymerization ptenb morphants cofocal microscopy actin <br />      polymerization assay. These results suggest Ptenb antagonizing PI3 kinase downstream Akt1 Cdc42 to <br />      regulate actin polymerization critical proper cell motility migration control gastrulation zebrafish. <br />  <br />   Citation: Yeh C-M, Liu Y-C, Chang C-J, Lai S-L, Hsiao C-D, et al. (2011) Ptenb Mediates Gastrulation Cell Movements Cdc42/AKT1 Zebrafish. PLoS ONE 6(4): <br />   e18702. doi:10.1371/journal.pone.0018702 <br />   Editor: Austin John Cooney, Baylor College Medicine, United States America <br />   Received October 29, 2010; Accepted March 15, 2011; Published April 11, 2011 <br />   Copyright: &#195;&#376; 2011 Yeh et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Science Council, Taiwan, Republic China, grant number 98-2311-B-002-006-MY3 (URL: http://web1.nsc.gov. <br />   tw/). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: jefflee@ntu.edu.tw <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            Phosphorylation PIP2 results phophatidylinositol-3,4,5-triphos- <br />                                                                                         phate (PIP3) signaling involved cell proliferation, migration, <br />    Gastrulation morphogenetic process involving cell migration                     <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>survival</span>, apoptosis Akt/PKB signaling [9]. Blocking PI3 kinase <br /> and rearrangements establish germ layers: ectoderm,                            causes convergence extension defects reduced directional <br /> mesoderm, endoderm [1]. In zebrafish, distinct                                protrusions leading cells mesoderm zebrafish [10]. This implies <br /> morphogenetic cell movements occur gastrulation, includ-                         possible involvement PIP2/PIP3 balance gastrulation cell <br /> ing epiboly, involution, convergence extension [2]. Gastrula-                       movements. A counter enzyme PI3 kinase PTEN (Phosphatase <br /> tion starts blastula stage embryo proper appears                      TENsin homolog deleted chromosome 10, named <br /> a mass cells situated yolk cells. The yolk sphere                  MMAC1 TEP1), famous tumor suppressor gene [11,12]. Its <br /> forming dome cap pushes mass blastomeres                               mutations reported numerous human cancers, like brain <br /> become thinner start spread yolk sphere                            cancer, breast cancer, prostate cancer [13], melanoma [14], some <br /> process called epiboly. After 50% yolk sphere enclosed                     autosomal dominant cancer syndromes, like Cowden&#226;&#8364;&#8482;s disease [15], <br /> the blastoderm, runner cells putative dorsal                      Bannayan-Zonana [16], Lhermitte-Duclos disease [17]. PTEN <br /> begin involute retrogradely future anterior                       phosphatase domain [18], negatively regulates PI3 <br /> form mesoderm endoderm progenitor cells. At                                   kinase/Akt pathway dephosphorylating PIP3 [19,20]. It regulates <br /> midgastrulation, convergence extension movements occur                           cell polarity cell migration [21] antagonizing PI3 kinase for <br /> narrow medio-lateral elongate anterior-posterior body axis,                      balance control PIP2/PIP3 mediate chemotaxis [22,23]. It is <br /> respectively, essential set dorsal-ventral                        intriguing PTEN functions concert PI3 <br /> anterior-posterior axes [3,4,5]. These gastrulation cell movements                      kinase embryogenesis. <br /> are demonstrated mediated cell adhesion                                  In mouse, fruit fly chicken, PTEN known regulate cell <br /> cytoskeleton rearrangement [6,7,8].                                                     migration, cell cycle <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> early <br />    Cell adhesion cytoskeleton rearrangement associated                    embryogenesis [24,25,26]. Pten2/2 <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice die around <br /> the metabolism membrane lipids. One key enzymes                           embryonic day 7.5 [24,27] altering pten expression before <br /> metabolizing membrane lipid phosphoinositide 3-kinases (PI3                          midblatula transition causes gastrulation delay Xenopus embryos <br /> kinase). PI3 kinase phosphorylate D3 position hydroxyl group                    [28]. It appears Pten indispensable gene during <br /> of inositol ring phosphatidylinositol-4,5-diphosphate (PIP2).                    embryogenesis. However, effects Pten dynamic <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                  1                                  April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                          Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br /> gastrulating cell movements examined                 Results <br /> experimental constraints. Zebrafish well-established model to <br /> study dynamic processes gastrulation cell movements [29].              Ptenb expresses maternally exists early <br /> There zebrafish pten isoforms, ptena ptenb. Each isoform          embryogenesis <br /> has alternatively splicing forms. Using morpholino (MO)                  Previous studies reported ptenb ubiquitously <br /> knockdown approach, Croushore et al. [30] showed ptena                expressed zebrafish early embryos selected stages <br /> ptenb MO-injected embryos exhibit distinct morphological defects              [30,32], expression patterns cleavage and <br /> in 24&#226;&#8364;&#8220;48 hours post fertilization (hpf), effects MOs         gastrulation stages lacking. RT-PCR analysis showed that <br /> on gastrulation described. Comparing genomic synteny,                ptenb mRNA expressed early embryonic stage <br /> it reveals zebrafish ptenb human PTEN                   examined, reduced 30% epiboly gradually recovered <br /> conserved locus. It suggests ptenb probably ortholog           afterward (Fig. 1A). Whole-mount situ hybridization (WISH) <br /> human PTEN [31]. Thus, set examine effects                   analysis revealed ubiquitous expression patterns ptenb in <br /> Ptenb gastrulation cell movements knockdown                 embryos 18-somite stage (Fig. 1B). The maternal and <br /> of ptenb MO cell non-autonomously disturbs epiboly                     ubiquitous expression patterns ptenb early embryonic <br /> convergent-extension cell movements gastrulation                    stages suggested play pivotal role early <br /> zebrafish.                                                                    embryogenesis. <br />  <br />  <br />  <br />  <br /> Figure 1. Expression patterns zebrafish ptenb. (A) Expression ptenb designated developmental stage examined RT-PCR analysis <br /> of 1183-bp ptenb fragment, 530-bp b-actin fragment used internal control. hpf: h post fertilization. (B) Representative photographs <br /> of embryos fixed designated stages underwent whole-mount situ hybridization ptenb. Scale bar: 200 mm. <br /> doi:10.1371/journal.pone.0018702.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                              April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                         Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br /> Specificity potency examination                                        ptenb, tested effects second published ptenb MO, <br /> translation-blocking ptenb MO                                                 tMO2 [30] 5-bp mismatched tMO1 (mis-tMO1). At dosage <br />    Two published antisense translational blocking MOs [30]               15 ng, 2.761.1% tMO1, 20.9613.8% tMO2 and <br /> adopted study role Ptenb gasrulation. These              16.6611.7% 5-bp mis-tMO1 morphants reached 95% <br /> MOs bind ptenb mRNA 59 untranslated region (59 UTR) non-                epiboly stage 10 hpf. Eventually (,1&#226;&#8364;&#8220;3 h lag) them <br /> overlapping sites indicated Fig. 2A. To confirm             complete epiboly MO dosages tested. This delay in <br /> specificity translational blocking efficiency ptenb          epiboly-induced ptenb MOs partially rescued co- <br /> MO (tMO1), co-injected plasmids PCS2+ construct                       injecting ptenb mRNAs (Fig. 3B). We examined effect of <br /> containing tMO1 binding site (PCS2+_ptenb 59 UTR) check                ptenb MOs 14 hpf shortening embryonic axis. <br /> the tMO1 translation blocking ability. Embryos (n = 148) injected             The tail rudiment tMO1 morphants extended <br /> with PCS2+_ptenb 59 UTR plasmids expressed green                    (intermediate <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, Fig. 4B) stalled vegetal pole (severe <br /> fluorescent protein (GFP) mosaic <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span> high ratio            phenotype, Fig. 4C). This tail defect tMO1-dose dependent <br /> 87.260.23% (N = 3) 40% epiboly stage (Fig. 2C). By contrast,               (Fig. 4D). mis-tMO1 (15 ng)-injected embryos (4.561.6% severe <br /> co-injection 5 ng tMO1 (n = 160) completely blocked                    92.061.9% intermediate phenotype, n = 201) exhibited a <br /> expression GFP (Fig. 2E). These results indicated ptenb               milder defect compared tMO1 morphants (81.063.6% severe <br /> tMO1 specifically potently blocks zebrafish ptenb translation.            14.265.1 intermediate phenotype, n = 140). ptenb tMO2 <br />                                                                               (15 ng) caused 11.061.6% severe 89.061.6% interme- <br /> Knockdown ptenb impairs gastrulation                                       diate defect (n = 168). Co-injection 15 ng tMO1 50 pg ptenb <br />    To understand role Ptenb early embryogenesis,             mRNAs partially rescued defects 3.660.9% severe <br /> injected embryos tMO1 examined effect embryonic               phenotype 92.961.5% intermediate phenotype (n = 195, <br /> development. A standard control MO (StdMO),                       Fig. 4E). Taken together, ptenb MO-induced defects appeared <br /> minimum effect embryonic development, used MO                     caused specific loss ptenb. With higher potency <br /> control. The MO-injected embryos called morphants                     tMO1 observed used rest experiments unless <br /> hereafter. The epibolic progression ptenb morphants notably            stated. <br /> slower morphants displayed cylinder-like shape. Most                  Since observed shortening embryonic axis typical <br /> of StdMO morphants (98.760.9%) reached 95% epiboly                   convergent extension defect [33], examined effect ptenb <br /> stage 10 hpf (Fig. 3A). However, epiboly progression               MO convergence. Embryos fixed checking early <br /> notably lagged tMO1 morphants dose-dependent manner                   morphological defects subjected WISH myod reveal <br /> (N&quot;4). To test observed defects specific loss           somites. The width widest somite posterior <br />                                                                               somites measured estimate extent convergence (Fig. 5A). <br />                                                                               The average somite width 173.763.8 mm StdMO (15 ng) <br />                                                                               morphants, comparable untreated embryos (data <br />                                                                               shown). Embryos injected tMO1 showed expanded somite <br />                                                                               width dose-dependent manner (Fig. 5B). The somite width was <br />                                                                               189.962.0 mm, 196.361.4 mm 215.966.3 mm embryos <br />                                                                               injected 5 ng, 10 ng 15 ng tMO1, respectively. tMO2 <br />                                                                               (15 ng) morphants exhibited severe convergence defect with <br />                                                                               236.466.0 mm somite width. This convergence defect be <br />                                                                               rescued co-injecting 15 ng tMO1 50 pg ptenb mRNA a <br />                                                                               somite width 170.461.8 mm, similar StdMO- <br />                                                                               injected embryos. <br />                                                                                  To examine effect tMO1 dorsal axis extension, we <br />                                                                               measured angle anterior end prechordal plate <br />                                                                               (as revealed WISH ctsl1b) posterior end myod. <br />                                                                               The tMO1 morphants showed notable increase angle (i.e. the <br />                                                                               inhibition dorsal axis extension) compared StdMO <br />                                                                               morphants (Fig. 6A). The ptenb MO-induced extension defect <br />                                                                               partially rescued ptenb mRNA co-injection as <br />                                                                               shown Fig. 6B. Since ctsl1b probe-labeled prechordal plates <br />                                                                               observed control embryos morphants (Fig. 6A), it <br />                                                                               appeared mesendodermal involution affected by <br />                                                                               loss Ptenb. <br />  <br />                                                                               Ptenb knockdown effects alleviated inhibiting <br /> Figure 2. ptenb MO target sites potency examination. (A) <br />                                                                               PI3 kinase <br /> Partial mRNA map ptenb. The black white boxes represent           To understand ptenb MOs impair gastrulation cell <br /> the coding sequence (CDS) ATG translation initiation site 59         movements reinforcing PI3 kinase-mediated responses, we <br /> untranslated region (UTR), respectively. The ptenb tMO1 tMO2              examined ptenb MO-induced gastrulation defects be <br /> binding sites located 59 UTR shown. The potency tMO1         mitigated inhibiting PI3 kinase. By analyzing ratio somite <br /> to reduce ptenb expression shown co-injecting 150 pg                   width embryo diameter (Fig. 7A&#226;&#8364;&#8220;C), observed the <br /> PCS2+_ptenb 59 UTR plasmid, containing tMO1 binding site, <br /> with 5 ng tMO1. Representative photographs control (B,C) <br />                                                                               presence 5 mM PI3 kinase inhibitor LY294002 (n = 72), the <br /> and morphant embryos (D,E) bright (B,D) dark fields (C,E)       somite width ro embryo diameter ratio significantly affected <br /> shown. These experiments repeated 3 times.                               (0.30860.013 mm v.s. 0.29560.003 mm, n = 85) 10 ng StdMO- <br /> doi:10.1371/journal.pone.0018702.g002                                         injected embryos (N = 3). By contrast, LY294002 fully <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                             April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                     Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> Figure 3. ptenb MOs delays epibolic progression. Knockdown ptenb MOs caused delay epibolic movement dose-dependent <br /> manner. (A) Embryos injected standard MO (StdMO), tMO1, 5-bp mismatched tMO1 (mis-tMO1) tMO2 designated amount, incubated <br /> to 10 hpf percentages embryos epiboly stage shown. (B) Embryos injected 15 ng StdMO tMO1 absence <br /> or presence ptenb mRNAs (50 pg), incubated degrees epiboly progression shown. Each treatment repeated 6 times. <br /> doi:10.1371/journal.pone.0018702.g003 <br />  <br />  <br /> rescued impaired somite-convergent defect tMO1 morphants           Ptenb regulates cell movements modulating cell <br /> (Fig. 7G). However, appeared potent rescue           protrusive activity <br /> extension defect induced tMO1 dosage tested (Fig. 7F,G).           To examine ptenb regulates convergence extension <br /> We attempted use higher dosages 30 mm LY294002,           cellular movements, analyzed cell migration different regions <br /> the decorinated embryos fragile examined             gastrulating zebrafish embryos. As shown movie S1 S2, <br /> those conditions. These results indicated ptenb regulate         leading edge hypoblast cells StdMO morphant formed <br /> zebrafish convergent movement antagonizing PI3 kinase.                 different types protrusion mainly long thick <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      4                             April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                       Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> Figure 4. ptenb MOs specifically impairs gastrulation dose-dependent manner. Knockdown ptenb MOs caused gastrulation <br /> defects graded severity. Representative photographs ptenb morphants different severity shown (A) normal (B) intermediate (C) <br /> severe. (D) Embryos injected StdMO, tMO1, mis-tMO1 tMO2 designated amount, incubated 10 somite stage percentages <br /> of embryos different severity shown. (E) Embryos injected 15 ng StdMO tMO1 absence presence ptenb mRNAs <br /> (50 pg), incubated degrees gastrulation defect shown. Each treatment repeated 4 times. <br /> doi:10.1371/journal.pone.0018702.g004 <br />  <br />  <br /> lamellapodia lateral cells converge actively              Ptenb functions non cell-autonomously gastrulation <br /> toward dorsal (arrow, Fig. 8A). In 4 trials,          cell migration <br /> significant difference number lamellapodia formed                To assess cell-autonomy ptenb function mediating migration <br /> cell StdMO (0.14060.006/min, n = 28) tMO1                       lateral hypoblast cells, performed cell transplantation <br /> (0.16860.011/min, n = 29) morphants. By observing 29 lamella-               experiments using rhodamine dextran cell tracer. <br /> podia 75% epiboly stage 4 StdMO morphants,                   Blastomeres StdMO tMO1 morphants transplanted <br /> these lamellapodia persisted 5.860.6 min (Fig. 8C). More           host embryos treated tMO1, examined <br /> half lamellapodia (60.761.0%, Fig. 8D) pointed              confocal microscopy epifluorescent microscopy. Under <br /> anterodorsal direction. In contrast, cells tMO1                 confocal microscopy, cells StdMO morphant showed <br /> morphants formed lamellapodia (n = 32) (Fig. 8B), average          highly protrusive activities formed protrusions when <br /> persistence time shorter (4.261.0 min, N = 4) compared             transplanted untreated host embryo (&#226;&#8364;&#732;&#226;&#8364;&#732;STD.UT&#226;&#8364;&#8482;&#226;&#8364;&#8482;, arrow, <br /> that StdMO morphants (Fig. 8C) 46.265.0% (N = 4,                Fig. 9A; movie S5) protrusions shorter well <br /> Fig. 8D) lamellapodia extended anterodorsal                   formed cells transplanted tMO1 morphant host <br /> direction.                                                                  (&#226;&#8364;&#732;&#226;&#8364;&#732;STD.MO&#226;&#8364;&#8482;&#226;&#8364;&#8482;, arrows, Fig. 9B; movie S5). Moreover, cells a <br />     The behaviors prechordal plate leading edge cells 90%             tMO1 morphant displayed polygonal shape (&#226;&#8364;&#732;&#226;&#8364;&#732;MO.UT&#226;&#8364;&#8482;&#226;&#8364;&#8482;, <br /> epiboly analyzed extension abnormality. In StdMO              Fig. 9C; movie S5) transplanted untreated host <br /> morphants, prechordal plate cells formed multiple layers                compared cells rounder shape reduced <br /> migrated sheet leading edge cells formed protrusions          number protrusions tMO1 morphant host (&#226;&#8364;&#732;&#226;&#8364;&#732;MO.MO&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> at adhered cells back. The leading edge cells              Fig. 9D; movie S5). To examine migration velocity and <br /> (n = 17) quite active, formed protrusions, including              direction transplanted cells different hosts, took time-lapse <br /> filapodia, lamellapodia, blebbing, migrated fast           movies epifluorescent microscopy 6 embryos each <br /> yolk sphere velocity 2.360.2 mm/min (N = 5, movie S3).              treatment monitored 8&#226;&#8364;&#8220;10 cells embryo. The <br /> The leading-edge cells tMO1 morphants (n = 18)              STD.UT cells traveled faster longer STD.MO <br /> same protrusions StdMO morphants velocity             cells (Fig. 9E). The curvilinear velocity (Vcl, curvilinear distance/ <br /> of leading-edge cell migration significantly slower                     time) STD.UT cells 0.025760.0031 mm/sec (n = 61), but <br /> at1.860.2 mm/min (N = 5, p,0.01, movie S4).                                 0.020260.0024 mm/sec (n = 50), STD.MO cells <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       5                              April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                         Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />                                                                               overall intensity F-actin enhanced disturbing actin <br />                                                                               distribution patterns (Fig. 10B). To confirm increase of <br />                                                                               F-actin tMO1 morphants, measured F-actin contents of <br />                                                                               embryos using actin polymerization assay [36]. In 3 <br />                                                                               experiments, tMO1 morphants exhibited ,1.8 folds fluores- <br />                                                                               cence compared control embryos (Fig. 10C). These <br />                                                                               results demonstrated knockdown ptenb results increase <br />                                                                               actin polymerization. <br />  <br />                                                                               Dominant-negative Cdc42 rescues ptenb MO-induced <br />                                                                               convergence extension defects <br />                                                                                  Small GTPases Rac1 Cdc42 known downstream of <br />                                                                               PTEN in vitro studies [35]. To examine Cdc42 and/or <br />                                                                               Rac1 involved Ptenb-mediated regulation gastrulation <br />                                                                               cell movements, dominant negative, T17NCdc42 T17NRac1 <br />                                                                               used effects ptenb MO-induced convergent <br />                                                                               extension defects. Co-injection T17NCdc42 mRNAs (25 pg) <br />                                                                               StdMO (10 ng) did change somite width, but <br />                                                                               increased extension angle (N = 4, Fig. S1A). In contrast, <br />                                                                               T17NCdc42 mRNAs partially restore somite width and <br />                                                                               extension angle tMO1 morphants (Table 1, N = 4). Similar <br />                                                                               experiments performed using T17NRac1 (Table 1, N = 3). <br />                                                                               Co-injection T17NRac1 mRNAs (50 pg) StdMO (10 ng) <br />                                                                               caused 93.0%60.0% embryos severe epiboly arrest (Fig. <br />                                                                               S1C,D) compared 2.4%60.0% embryos-injected with <br />                                                                               StdMO (Fig. S1B,D, N = 4). T17NRac1 mRNAs (50 pg) <br />                                                                               ptenb tMO1 (10 ng) co-injected embryos slightly improved <br />                                                                               convergent extension defect compared tMO1 <br />                                                                               morphants statistical difference (p.0.05) groups. <br />  <br /> Figure 5. ptenb MOs cause convergence defect gastru- <br /> lation. Embryos injected designated MO, <br />                                                                               Overexpression human constitutively active AKT1 <br /> incubated 9 11 somites subjected WISH myod.              causes convergence extension defects <br /> The somite width treated embryos measured indicated (A).            Knockdown ptenb lead increase downstream Akt/ <br /> The average somite width number embryos used (n)               PKB activity, investigate elevation AKT/PKB <br /> treatment shown (B). These experiments repeated 4        activity result similar ptenb MO-induced phenotypes. We <br /> times. Values groups significant difference (p,0.05) are <br /> denoted different superscript letters.                                     overexpressed human constitutively active akt1 (caakt1) zebrafish <br /> doi:10.1371/journal.pone.0018702.g005                                         embryos observe effect convergent extension. Embryos- <br />                                                                               injected 16 buffer caakt1 mRNAs 50, 100, 200 pg <br />                                                                               collected subjected WISH myod ctsl1b. As <br /> (p,0.05). The MO.UT cells traveled faster speed                  shown Table 2, lower doses (50 100 pg) human caakt1 <br /> longer distance Vcl (0.029060.0043 mm/sec, n = 48) compared                mRNAs caused slight convergent extension defects, profound <br /> to MO.MO cells (0.021260.0025 mm/sec, n = 54), did                  inhibition convergent extension observed embryos- <br /> not reach statistical difference (p.0.05). Furthermore,                   treated 200 pg caakt1 mRNAs. These results implied the <br /> STD.UT MO.UT cells migrated straighter shown                        ptenb MO-induced convergent extension defects to <br /> a higher straight line velocity (Vsl, straight line distance/time,            elevation AKT1 activity. <br /> Fig. 9E), 0.021960.0036 mm/sec (n = 61) and <br /> 0.024060.0053 mm/sec (n = 48), respectively. By contrast,                 Ptenb MO causes convergence extension defects in <br /> STD.MO (n = 50) MO.MO (n = 54) cells migrated                        ptenb2/2 embryos <br /> linearly Vsl 0.016360.0029 mm/sec (n = 50)                         The lack notable early embryonic defect ptenb2/2 mutant <br /> 0.016360.0031 mm/sec, respectively, compared cells                  fish contrast observations described might <br /> UT host. These results suggested zebrafish Ptenb mediates                presence maternal pten transcripts [32]. To <br /> gastrulation cell migration non cell-autonomous manner.                  examine possibility, obtained heterozygous ptenb+/2 <br />                                                                               embryos, raised adults rise F1 offspring. Those F1 fish <br /> Knockdown ptenb enhanced actin polymerization                              screened PCR presence mutated ptenb get <br />    PTEN known control protrusion formation adhesion                 ptenb2/2 homozygous fish according Faucherre et al. [32]. <br /> during cell migration [34] modulating actin dynamics [35]. To             Different dosages ptenb tMO1 injected ptenb2/2 <br /> determine actin polymerization gastrulation                 embryos, fixed subjected WISH myod ctsl1b. <br /> tMO1 morphants affected, used rhodamine phalloidin                     Similar results ptenb MO-induced convergent extension defects <br /> staining visualize filamentous actin (F-actin) gastrulating             obtained wild type embryos (Table 3). Extension <br /> embryos confocal microscopy. The vegetal pole view                      angle somite width ptenb2/2 embryos significantly <br /> showed that, F-actins mainly located cell boundaries                  increased presence ptenb tMO1. These data implied that <br /> corresponding cortical actin structure underneath cell                 presence maternal ptenb transcripts sufficient to <br /> membrane StdMO-treated embryos (Fig. 10A). In contrast,                complete normal gastrulation ptenb2/2 embryos. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                             April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                       Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> Figure 6. ptenb MOs cause extension defect gastrulation. Embryos treated described Fig. 5, subjected WISH against <br /> ctsl1b instead myod. The extension angles treated embryos measured indicated (A). The average extension angle number of <br /> embryos used (n) treatment shown (B). Rescue experiments performed shown (C). These experiments repeated least <br /> 4 times. Values groups significant difference (p,0.05) denoted different superscript letters. <br /> doi:10.1371/journal.pone.0018702.g006 <br />  <br />  <br /> Discussion                                                                   enhanced AKT activity [30] analyze effects gastrulation <br />                                                                              cell movements zebrafish. Our results clearly indicated that <br />    The necessity PTEN embryogenesis demonstrat-               block ptenb activity affects convergent extension movements <br /> ed animal species including mice, chickens fruit flies        gastrulation zebrafish. <br /> [24,25,26]. However, role Pten zebrafish gastulation has <br /> been controversial. In study, demonstrated ptenb            Zebrafish Ptenb regulates gastrulation coordination <br /> regulate zebrafish gastrulation cell movements controlling actin <br />                                                                              PI3 kinase <br /> polymerization directed cell migration antagonizing PI3                 Pten known counter enzyme PI3 kinase dephosphor- <br /> kinase downstream Cdc42 Akt activity.                                ylating PIP3 PIP2 [9,18], acts downstream effectors <br />                                                                              AKT/PKB regulate early developmental processes [22,25,28]. <br /> Ptenb regulates convergence extension                             To examine coordination PI3 kinase Ptenb in <br /> gastrulation                                                                 regulating zebrafish gastrulation, showed LY294002 could <br />    PI3 kinase shown required cell polarization         rescue convergence defect tMO1 morphants. It clearly <br /> gastrulating mesendodermal cells overexpressing dominant-                 demonstrated PI3 kinase PTEN linkage required to <br /> negative form PI3 kinase using PI3 kinase inhibitor                modulate cell movements zebrafish gastrulation. <br /> LY294002 [10]. The authors unequivocally demonstrated the <br /> necessity maintaining PIP2/PIP3 balance mesendodermal cell             Zebrafish Ptenb regulates protrusive activities lateral <br /> polarity directed migration gastrulation zebrafish. It       mesendodermal cells prechordal plate cells <br /> logical hypothesize negative regulator PI3 kinase,            gastrulation <br /> PTEN, necessary regulating gastrulation cell                 In LY294002 rescue experiment, result suggested that <br /> movements zebrafish, gastrulation defects observed             Ptenb coordinates PI3 kinase regulate gastrulation cell <br /> in zebrafish pten mutants [37]. Based pten mutant study, Pten         movements presumably regulation PIP2/PIP3 balance. <br /> appears play role gastrulation zebrafish. However,           Activated PI3 kinase increase PIP3 concentration, which <br /> the presence maternal pten messages early mutant embryos               localize frontal edge protrusions induce the <br /> cannot excluded. Secondly, lack functional Pten             directional migration, Pten acts rear edge cells to <br /> zebrafish pten mutants demonstrated lack           dephophorylate PIP3 PIP2 restrict spatial expression of <br /> antibodies recognize zebrafish Pten. Therefore, took            PIP3 cell [22,23]. Our time-lapse recordings clearly <br /> an alternative approach using pten MOs                    demonstrated active protrusive activity required cell <br /> previously shown specifically lowered pten translation           movements zebrafish gastrulation. The ptenb MO-induced <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                            April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                        Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> Figure 7. Rescue ptenb MO-induced convergence defect inhibiting PI3 kinase. Embryos injected 10 ng StdMO (A,D) or <br /> tMO1 (B,E), incubated (A,B,D,E) 5 mM LY294002 (C,F) 9 11 somites subjected WISH myod ctsl1b. The somite <br /> width (solid line), embryo diameter (dash line) (A&#226;&#8364;&#8220;C) extension angle (E&#226;&#8364;&#8220;F) treated embryos measured indicated. The average somite <br /> width/embryo diameter ratio, extension angle sample number treatment shown (G). These experiments repeated 5 times. <br /> Values groups significant difference (p,0.05) denoted different superscript letters. <br /> doi:10.1371/journal.pone.0018702.g007 <br />  <br />  <br />  <br />  <br /> gastrulation defect change protrusion                  Zebrafish Ptenb mediates gastrulation cellular <br /> numbers. In stead, directionality persistency protrusion          movements controlling actin polymerization and <br /> were relevant. Lamellapodia formed leading                  downstream Akt1 Cdc42 <br /> edge lateral hypoblast cells protrusions mainly               In vitro cell studies shown that, increase PIP3 <br /> pointing anterodorsally direction cell movement.                   concentration cells activates Rac1 Cdc42 lead <br /> However, tMO1 morphants&#226;&#8364;&#8482; lateral hypoblasts exhibited interfer-              actin polymerization lamellapodia filapodia formation <br /> ence persistence directionality protrusions. In lateral        [35,39]. Rac1 reported important Drosophila <br /> hypoblasts tMO1 morphants, increase lamellapodia                   mesoderm migration gastrula [40] zebrafish conver- <br /> turnover evident reduction lamellapodia persistence.           gence controlling lamellapodia formation [41]. In results, <br /> At time, directionality affected decreasing         dominant negative Cdc42 extent dominant <br /> the number protrusions pointed anteriodorsally. The directional           negative Rac1 showed rescue convergent extension defects <br /> defect observed transplantation experiment.                  ptenb morphants. <br /> StdMO cells transplanted untreated embryos revealed better                 AKT/PKB known downstream factor PI3 kinase. <br /> linearity tMO1 morphant cells transplantated tMO1                The binding AKT/PKB PIP3 recruit membrane for <br /> hosts. The abnormality protrusion persistence migration               regulation actin arrangement protrusion formation <br /> directionality observed prechordal plate cells, result       [42]. Knockdown ptenb lead increase PIP3 and <br /> might fact prechordal plate cells migrate              subsequent elevation AKT activity zebrafish embryos. The <br /> multicellular sheets cell-cell highly contact. The rear area         elevated AKT activity disturb actin cytoskeleton, <br /> of leading edge cells close contact cells        protrusion formation cell movements. Over-expression of <br /> might help maintain cell polarity absence Ptenb [38].           caakt1 showed convergence extension defects. These results <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        8                             April 2011 | Volume 6 | Issue 4 | e18702 <br />                                   Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   9    April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                           Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br /> Figure 8. Cell protrusions lateral hypoblast cells affected ptenb MO. Embryos injected 10 ng StdMO (A) ptenb <br /> tMO1 (B), imaged DIC microscopy recorded 75% 85% epiboly stages. Representative snapshots shown arrows are <br /> pointing typical cell protrusions. By examining 1-h recording, average persistent time protrusion (C) percentage of <br /> protrusions pointed anterodorsally (D) calculated shown treatment. * : p,0.05, scale bars: 25 mm. <br /> doi:10.1371/journal.pone.0018702.g008 <br />  <br /> demonstrated Ptenb regulates zebrafish convergence                    (3) The defects ptenb morphants rescued exogenous <br /> extension gastrulation PI3 kinase-Akt pathway               ptenb mRNAs, LY294002 dominant-negative Cdc42. (4) Since <br /> consistent rescue ptenb tMO1 morphants PI3                  generation functional truncated Ptenb protein be <br /> kinase inhibitor.                                                              excluded ptenb mutants [32], ptenb2/2 embryos was <br />    Gastrulation cell movements tightly regulated actin                  treated ptenb tMO1 mutant embryos showed similar <br /> cytoskeletons. Here, showed actin polymerization                   morphological defects wild type embryos. In addition our <br /> enhanced folds ptenb tMO1 morphants                  results, Croushore et al., [30] shown ptenb tMO1 inhibited <br /> control embryos. These results suggest zebrafish Ptenb                    Ptenb protein vitro translation assay elevated phospho-Akt <br /> downregulates small GTPases Rac1 Cdc42,                      (pAkt). Those results supported ptenb-induced gastrulation <br /> rearrange actin polymerization, controls convergence              defects observed specifically loss ptenb. <br /> extension gastrulation.                                                    In summary, provide vivo evidence Ptenb <br />                                                                                coordinates PI3 kinase modulate downstream AKT1 and <br /> ptenb MO induces convergence extension defects                          Cdc42 rearranging actin polymerization protrusion <br /> ptenb2/2 embryos                                                               formation lead proper cell migration regulate <br />    The lack gastrulation defect pten mutants argues role           convergence extension cell movements gastrulation in <br /> of PTEN zebrafish authors questioned                  zebrafish schematically depicted Fig. 11. <br /> effects pten MOs zebrafish early embryos reported by <br /> Croushore et al. [30] specific loss Pten              Materials Methods <br /> [32]. However, demonstrated specificity ptenb MO used <br /> with following evidences: (1) Co-injection ptenb tMO1               Ethics Statement <br /> ptenb 59 UTR-GFP plasmid showed fluorescence signal                            All animal handling procedures approved use of <br /> compared ptenb 59 UTR-GFP injection alone. (2) Ptenb tMO1                   laboratory animal committee National Taiwan University, Taipei, <br /> caused convergence extension dose-dependent manner.                   Taiwan (IACUC Approval ID: 97 Animal Use document No. 55). <br />  <br />  <br />  <br />  <br /> Figure 9. ptenb functions non cell-autonomously regulate cell protrusions cell migration gastrulation. (A&#226;&#8364;&#8220;D) <br /> Rhodamine labeled blastomeres transplanted embryos injected 10 ng StdMO (STD) tMO1 (MO) rhodamine dextran to <br /> untreated hosts (UT) tMO1 morphant hosts (MO). Host embryos imaged (animal pole vegetal pole bottom) under <br /> confocal microscopy representative snapshots shown (A) STD.UT: StdMO treated cells untreated host. (B) STD.MO: StdMO treated <br /> cells tMO1 morphant. (C) MO.UT: tMO1 treated cells untreated host. (D) MO.MO: tMO1 treated cells tMO1 morphant. Arrows indicate <br /> the representative cellular protrusions embryo. (E) Cell transplantations performed described (A&#226;&#8364;&#8220;D), imaged epifluorescent <br /> microscopy time-lapse movies taken treatment. The transplanted cells traced curvillinear velocity (Vcl) strait line <br /> velocity (Vcl) calculated shown. Values groups significant difference (p,0.05) denoted different letters. Scale bar: <br /> 25 mm. <br /> doi:10.1371/journal.pone.0018702.g009 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         10                               April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                            Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  <br /> Figure 10. Knockdown ptenb enhances actin polymerization. The 10 ng StdMO (A) 10 ng ptenb tMO1 (B)-treated embryos fixed <br /> at 70% epiboly, stained rhodamine-phalloidin reveal F-actin, observed confocal microscopy, photographed vegetal view. <br /> The F-actin contents shield stage-embryos (n = 60) measured actin polymerization assay described &#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Relative <br /> fluorescence (Rf) values independent experiments corrected subtracting background Rf. * indicates value significantly <br /> different StdMO treatment (p,0.05). Scale bar: 25 mm. <br /> doi:10.1371/journal.pone.0018702.g010 <br />  <br />  <br /> Zebrafish maintenance embryo collection                                    manufacturer&#226;&#8364;&#8482;s instructions. cDNAs generated total <br />   Wild-type TU/AB zebrafish inbreeded maintained                   RNAs served templates 1183-bp fragment Ptenb <br /> a 14-hr light/10-hr dark cycle 28.5uC incubator. Embryos                   amplification following primers: 59-GGCTGCGATCA- <br /> were collected natural spawning raised 0.36 Danieau&#226;&#8364;&#8482;s                TAAAGGAAT-39 (forward) 59-CTGTTCTTCATCG- <br /> buffer (diluting 16 Danieau&#226;&#8364;&#8482;s buffer: 58 mM NaCl, 0.7 mM                    TACTGTTCA-39 (reverse). A 530-bp b-actin amplified with <br /> KCl, 0.4 mM MgSO4, 0.6 mM Ca(NO3)2, 5.0 mM HEPES                           primers: 59-TTGGTATGGGACAGAAAGACAGCTAC-39 (for- <br /> (pH 7.6), double distilled water) observation fixation.          ward) 59-AAGGGCCACATAGCAGAGCTTC-39 (reverse) <br /> The <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> embryo stage according Kimmel et al.                  used RT-PCR control. <br /> [43], stages indicated hours post-fertilization (hpf). <br />                                                                                Morpholino oligonucleotide microinjections <br /> RNA isolation RT-PCR analysis                                                All antisense morpholino oligonucleotides (MOs) custom <br />   Zebrafish RNAs selected embryonic stages isolated                 Gene Tools, LLC (Philomath, OR). Standard control <br /> TriSolution Reagent Plus (GeneMark, Atlanta, Georgia, USA).                    MO (Std MO, 59-CCTCTTACCTCAGTTACAATTTATA-39) <br /> RT-PCR applied cDNA synthesis according                         homology sequence known zebrafish sequence <br />  <br />  <br />  Table 1. The ptenb MO-induced convergent extension defects rescued dominant-negative Cdc42. <br />  <br />  <br />                                                       Somite width (mm)                               Extension angle (6) <br />                                                       (n)                                             (n) <br />  <br />  StdMO 10 ng                                          204.961.7a                                      71.661.8a <br />                                                       (155)                                           (157) <br />  ptenb tMO1 10 ng                                     228.964.5b                                      113.968.7b <br />                                                       (105)                                           (105) <br />  ptenb tMO1 10 ng+T17NCdc42 25 pg                     212.662.6c                                      98.963.6c <br />                                                       (85)                                            (87) <br />  StdMO 10 ng                                          192.064.6a                                      71.264.0a <br />                                                       (124)                                           (124) <br />  StdMO 10 ng+T17NCdc42 25 pg                          192.664.0a                                      79.364.6b <br />                                                       (92)                                            (92) <br />  StdMO 10 ng                                          172.964.5a                                      75.568.3a <br />                                                       (75)                                            (75) <br />  ptenb tMO1 10 ng                                     195.664.4b                                      115.061.9b <br />                                                       (91)                                            (92) <br />  ptenb tMO1 10 ng+T17NRac1 50 pg                      188.765.0b                                      110.564.5b <br />                                                       (112)                                           (114) <br />  <br />  Embryos injected StdMO ptenb tMO1 presence absence T17NCdc42 T17NRac1 mRNAs, subjected WISH myod ctsl1b and <br />  somite width extension angle embryos measured (N&quot;4). <br />  doi:10.1371/journal.pone.0018702.t001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         11                               April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                                Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />  <br />  Table 2. Constructively-active human AKT1 cause convergent                          Table 3. The ptenb MO induces convergent extension defects <br />  extension defects zebrafish embryos.                                             ptenb homozygous mutant. <br />  <br />  <br />                      Somite width (mm)          Extension angle (6)                                          Somite width (mm)       Extension angle (6) <br />                      (n)                        (n)                                  Treatment               (n)                     (n) <br />  <br />  16 buffer           173.763.3a                 77.765.3a                            StdMO 10 ng             222.466.5a              65.662.5a <br />                      (78)                       (79)                                                         (137)                   (144) <br />  caakt1 50 pg        176.462.0b                 84.064.0b                            ptenb tMO1 5 ng         231.266.3b              92.864.2b <br />                      (96)                       (97)                                                         (n = 126)               (126) <br />  caakt1 100 pg       175.962.7b                 82.065.8b                            ptenb tMO1 10 ng        238.065.9c              95.466.2b <br />                      (85)                       (89)                                                         (138)                   (151) <br />  caakt1 150 pg       202.463.0c                 113.063.7c <br />                      (75)                       (82)                                 Embryos collected homozygous ptenb mutants injected StdMO <br />                                                                                      ptenb tMO1, subjected WISH myod ctsl1b somite width <br />                                                                                      extension angle embryos measured (N = 4). <br />  Embryos injected different constructively- <br />                                                                                      doi:10.1371/journal.pone.0018702.t003 <br />  active human akt1 (caakt1) mRNAs, subjected WISH myod ctsl1b <br />  somite width extension angle embryos measured (N = 4). <br />  doi:10.1371/journal.pone.0018702.t002                                              wild type Cdc42 Rac pGEM-T easy, primers with <br />                                                                                     <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> sites used obtain dominant negative zebrafish <br /> which used. Two published ptenb translational blocking MOs,                     Cdc42 (T17NCdc42) Rac1 (T17NRac1) constructs by <br /> tMO1 tMO2 used following sequences: Ptenb                         performing PCR again. The primers mutation sites of <br /> tMO1 (59-CTTTCGGACGGTCGGTCGTCTTTA-39),                                        Cdc42 are: 59-GATGGTGCAGTGGGTAAAAACTGTCTA-39 <br /> targeting 274 251 59 UTR region; Ptenb tMO2                          (forward); 59-GTATAGGAGATTAATAGACAGTTTTTA-39 <br /> (59- GGCTGTGACAGGAG-TCTTTAGGGTT-39), tar-                                     (reverse). The primers mutation sites Rac are: 59- <br /> geting 225 21 [30]. A 5-<span id='am-474' about='xsp:base' typeof='owl:Thing'>base</span> mismatch tMO1                              GGGCTGTGGGAAAAAACTGCCTTCTGATCAGC-39 (for- <br /> designed (59 -CTTTgGGACcGTCcGTgGTgTTTA-39), mis-                                ward); 59-GCTGATCAGAAGGCAGTTTTTTCCCACAGC- <br /> matched bases indicated subscripts. All MOs                            CC-39 (reverse). The T17NCdc42 T17NRac1 linearized <br /> dissolved sterile mini Q water 1 mM stocks stored                      SalI ApaI transcribed mRNA mMESSAGE <br /> room temperature. MOs diluted proper working                                mMACHINEH Sp6 T7 Kits, respectively. The human <br /> concentration 16 Danieau&#226;&#8364;&#8482;s buffer 0.5% phenol red                       constitutively active human Akt1 (caakt1) constructed by <br /> keep room temperature. Thin-wall 3.5 inch glass capillaries                      Gataway cloning make pCSDEST-myrAkt1 myrAkt <br /> (1.1460.50 mm, O.D.6I.D., World Precision Instrument, Sar-                          delta4&#226;&#8364;&#8220;129 plasmid (Addgene, Cambridge, MA). The caakt1 <br /> asota, FL) pulled using horizontal puller (P-97, Sutter                      plasmids digested SacP mRNAs synthesized <br /> Instrument, Navato, CA). Embryos one-cell stage                             Sp6 Kit. <br /> stationary injection tough 100-mm 1% agar plate. <br /> MOs diluted described desired concentrations                         Whole-mount situ hybridization (WISH) <br /> loaded pulled capillary. The tip loaded capillary                        Embryos fixed desired stages 4% paraformaldehyde <br /> forced chorion yolk cells reach                     phosphate-buffered saline overnight, dechorionated manually <br /> junction yolk cells blastodisc MO                         fine forceps stored 100% methanol 220uC use. <br /> ejected using oil Nanoliter injector (Nanoliter 2000, World                   Antisense digoxigenin (DIG)-labeled RNA riboprobes were <br /> Precision Instruments) 2.3 nl. After injection, embryos                synthesized following manufacture instructions (Roche Applied <br /> out injection troughs cultured 0.36 Danieau&#226;&#8364;&#8482;s                       Science, Penzberg,Germany). The myod construct [45] was <br /> buffer 28.5uC examined.                                              linearized XbaI transcribed T7 RNA polymerase <br />                                                                                     (Roche Applied Science, Penzberg, Germany). The ctsl1b <br /> Cloning preparing mRNA                                                          construct [46] linearized NotI transcribed T7 <br />    After RNA isolation, ptenb cDNA obtained RT-PCR                           RNA polymerase. In situ hybridization detection were <br /> using primers: 59-CCGCTCGAGATGGCTGCGATCATAAA-                                       performed according Thisse et al. [47] phosphatase- <br /> GGAATTTG-39 (forward) 59-GAATTCTCAAACTTTAG-                                     coupled anti-DIG antibody. The processed embryos then <br /> TAATCTGTTC-39 (reverse), cloned pGEM-T easy                               transferred 100% glycerol photographed using Nikon <br /> vector. The clone subcloned expression vector,                  CoolPIX 995 digital camera. <br /> pcGlobin2 [44], NotI digestion. The ptenb construct was <br /> linearized XbaI, capped RNA transcribed                              Measurement counting embryos <br /> according manufacturer&#226;&#8364;&#8482;s instruction using mMESSAGE                            For convergence extension assay, ptenb MOs STD <br /> mMACHINEH T7 Kit (Applied Biosystems, Carlsbad, CA)                                 MO treated embryos incubated 28.5uC 8 11- <br /> according manufacturer&#226;&#8364;&#8482;s instructions. The dominant-negative                     somite stages morphological observation stereoscope. <br /> zebrafish Rho-GTPase, T17NCdc42 T17NRac1, cloned                           After morphology classification, embryos fixed and <br /> separately pGEM-T easy vector (Promega, Madison, WI)                        subjected WISH myod measure width myod <br /> using QuikChange Site-Directed Mutagenesis kit (Agilent                           signal widest somite somites. <br /> Technologies, Carlsbed, CA). The primers wild type zebrafish                    Furthermore, extension defects characterized by <br /> Cdc42 are: 59-ATGCAGACGATCAAGTGCGT-39 (forward), 59                                 measuring angle prechordal plate staining using <br /> ACGCTCATAGCAGCACACAT-39 (reverse) ; wild type                               ctsl1b vegetal end myod signals. All experiments were <br /> Rac1 are: 5-ATGCAGGCCATAAAGTGTGT-39 (forward), 59-                                  repeated times measurements made <br /> CAGAAGGAGACATCTTCTCC-39 (reverse); cloned                                 using tools Adobe Photoshop CS4 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                               12                               April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                      Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br />                                                                             end points movement divided total distance moved) <br />                                                                             analyses. <br />  <br />                                                                             PI3-kinase inhibitor (LY294002) treatment <br />                                                                                Embryos treated Ptenb tMO1 dechorionated 30% <br />                                                                             epiboly stage treated 5 mM LY294002 (Calbiochem, <br />                                                                             Darmstadt, Germany) 30% epiboly 90% epiboly E3 <br />                                                                             media transferred 0.36 Danieau&#226;&#8364;&#8482;s buffer 7 somite <br />                                                                             stage collected situ hybridization examination. <br />  <br />                                                                             Actin polymerization assay <br />                                                                                The filamentous actin (F-actin) embryo determined by <br />                                                                             fluorometric assay based binding affinity of <br />                                                                             rhodamine phalloidin F actin [48,49]. Embryos injected with <br />                                                                             different MOs fixed shield stage 5% formaldehyde in <br />                                                                             Actin Stabilizing Buffer (ASB) [50] overnight. The embryos were <br />                                                                             incubated 150 mM glycine-ASB 3 hours, washed by <br />                                                                             ASB, labeled 165 nM rhodamine phalloidin <br />                                                                             (Molecular Probes, Inc., Eugene, OR). Control embryos were <br />                                                                             incubated 20 mg/mL unlabeled phalloidin 50 minutes <br />                                                                             prior rhodamine phalloidin staining. All samples group of <br />                                                                             60 embryos washed ASB extracted dark with <br />                                                                             methanol homogenized 24G needles, incubated with <br />                                                                             rotating 36 hours 4uC. The fluorescence signals were <br />                                                                             detected extractant measured using fluorospec- <br />                                                                             trophotometer (Hitachi F-2000, Tokyo, Japan) excitation <br />                                                                             wavelength 565 nm emission wavelength 580 nm <br />                                                                             [36]. <br />  <br />                                                                             Cell transplantation <br />                                                                               For transplantation preparation, donor embryos injected <br />                                                                             0.5% rhodamine dextran Std MO 10 ng Ptenb tMO1 <br />                                                                             10 ng respectively. Donor embryos transplanted wild <br />                                                                             type Ptenb morphant host separately sphere stage as <br />                                                                             described previously [51]. Transplanted cells further <br />                                                                             recorded using Leica DM5000 B microscope Leica TCS <br /> Figure 11. Ptenb-mediated signal transduction gastru-                SP5 Confocal Microscope Imaging System. <br /> lation zebrafish. By coordinating PI3 kinase, Ptenb controls <br /> the balance PIP2 PIP3 activate downstream <br /> Cdc42 AKT1 mediating actin dynamics regulate cell                ptenb2/2 zebrafish screening <br /> movements gastrulation zebrafish.                                    Heterozygous ptenb+/2 zebrafish embryos kindly provided <br /> doi:10.1371/journal.pone.0018702.g011                                       Jeroen den Hertog (Hubrecht Institute, Utrecht, The Nether- <br />                                                                             lands). Those embryos raised adults mated produce <br />                                                                             F1 fish. Nested PCR performed select homozygous ptenb 2/2 <br /> DIC time-lapse cell migration recording <br />                                                                             zebrafish according Faucherre et al. [32]. <br />    To monitor migration lateral hypoblast leading edge cells <br /> (marginal deep cells), embryos 75% epiboly stage were <br /> dechorionated 0.01 mg/mL protease (Sigma, St. Louis, MO)               Statistical analysis <br /> and mounted 0.8% low-melting agarose (Amresco, Solon, OH).                  All experimental values presented mean 6 standard error <br />                                                                             analyzed unpaired-sample Student&#226;&#8364;&#8482;s t-test in <br /> The migrating cells recorded using 406 water immersion <br />                                                                             Microsoft Excel. N indicates number experiments repeated; <br /> objective Leica DM5000 B DIC microscope (Leica <br />                                                                             n indicates total sample number experimental <br /> Microsystems, Wetzlar, Germany). One hour long movies were <br />                                                                             condition. Different superscript lettering values stands <br /> recorded 30-sec intervals CoolSNAP fx CCD camera <br />                                                                             significant difference p,0.05. <br /> (Roper Scientific, Tucson, AZ). The movies acquired and <br /> analyzed Simple PCI Imagine System software (Compix, <br /> Sewickley, PA). The protrusions merged cell body            Supporting Information <br /> an angle ,135uC width .2 mm defined                         Figure S1 Embryos injected 10 ng StdMO <br /> lamellapodium extension. To monitor migration prechordal             25 pg T7 NCdc42 mRNAs treated <br /> plate cells, embryos 90% epiboly stage dechorionated            described Table 1. The somite width extension angle <br /> and mounted described, monitored, filmed (5 min                 embryo measured shown (A). Embryos were <br /> movies 5 sec intervals) analyzed using 636 water                   injected 10 ng StdMO (B) 50 pg T7 <br /> immersion described. To track transplanted cells, 1 hour long            NRac1 mRNAs (C), incubated 10-somite stage photo- <br /> time-lapse recordings 10-sec intervals taken                  graphed. The percentages normal abnormal embryos in <br /> analyzed using Simple PCI software migration path,               treatment shown (D). <br /> velocity linearity (the shortest distance start         (PDF) <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       13                            April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                                                 Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br /> Movie S1     Time-lapse imaging cellular migration                                      epiboly stage embryo-injected 10 ng ptenb tMO1. The <br /> protrusions lateral hypoblast cells StdMO-                                             prechordal plate cells migrating anteriorly top. <br /> injected morphant. Shown 1-h 30-s intervals                                       (AVI) <br /> DIC time-lapse image sequence 75% epiboly stage embryo- <br />                                                                                                Movie S5 Time-lapse confocal imaging cells treated <br /> injected 10 ng StdMO. The animal pole the <br />                                                                                                StdMO ptenb tMO1 transplanted an <br /> dorsal left.                                                                    untreated ptenb tMO1-treated embryo. Shown is <br /> (AVI)                                                                                          10-min movie 10-s intervals 4 combined time-lapse <br /> Movie S2     Time-lapse imaging cellular migration                                      confocal image sequences designated. The transplanted cells <br /> protrusions lateral hypoblast cells ptenb tMO1-                                        migrating vegetal pole. (A) STD.UT: StdMO <br /> injected morphant. Shown 1-h 30-s intervals DIC                                   treated cells untreated host. (B) STD.MO: StdMO treated <br /> time-lapse image sequence 75% epiboly stage embryo-injected                               cells tMO1 morphant. (C) MO.UT: tMO1 treated cells an <br /> with 10 ng ptenb tMO1. The animal pole                                   untreated host. (D) MO.MO: tMO1 treated cells tMO1 <br /> dorsal left.                                                                    morphant. <br /> (AVI)                                                                                          (AVI) <br /> Movie S3     Time-lapse imaging cellular migration and <br />                                                                                                Acknowledgments <br /> protrusions involuting prechordal plate cells a <br /> StdMO-injected morphant. Shown 5-min movie                                        We thank Dr. Jeroen den Hertog providing ptenb (+/2) embryos and <br /> 5-s intervals DIC time-lapse image sequence 90% epiboly                                   staffs TC5 Bio-Image Tools, Technology Commons, College Life <br /> stage embryo-injected 10 ng StdMO. The prechordal plate                                   Science, National Taiwan University, particularly Yi-Chun Chuang for <br />                                                                                                technical assistance confocal microscopy. <br /> cells migrating anteriorly top. <br /> (AVI) <br />                                                                                                Author Contributions <br /> Movie S4    Time-lapse imaging cellular migration                                       Conceived designed experiments: C-MY YCL CJC SJL. <br /> protrusions involuting prechordal plate cells                                          Performed experiments: C-MY YCL CJC. Analyzed data: C-MY <br /> ptenb tMO1-injected morphant. Shown 5-min                                            YCL CJC SLL SJL. Contributed reagents/materials/analysis tools: CDH. <br /> movie 5-s intervals DIC time-lapse image sequence 90%                                  Wrote paper: C-MY YCL SJL. <br />  <br /> References <br />  1. Keller R (2002) Shaping vertebrate body plan polarized embryonic cell                     kinase/Akt pathway cause human Lhermitte-Duclos disease adults. Am J Hum <br />     movements. Science 298: 1950&#226;&#8364;&#8220;1954.                                                               Genet 73: 1191&#226;&#8364;&#8220;1198. <br />  2. Solnica-Krezel L, Cooper MS (2002) Cellular genetic mechanisms                      18.   Lee JO, Yang H, Georgescu MM, Di Cristofano A, Maehama T, et al. (1999) <br />     convergence extension. Results Probl Cell Differ 40: 136&#226;&#8364;&#8220;165.                                Crystal structure PTEN tumor suppressor: implications phospho- <br />  3. Schier AF (2001) Axis formation patterning zebrafish. Curr Opin Genet                     inositide phosphatase activity membrane association. Cell 99: 323&#226;&#8364;&#8220;334. <br />     Dev 11: 393&#226;&#8364;&#8220;404.                                                                           19.   Maehama T, Dixon JE (1998) The tumor suppressor, PTEN/MMAC1, <br />  4. Warga RM, Kimmel CB (1990) Cell movements epiboly gastrulation                        dephosphorylates lipid second messenger, phosphatidylinositol 3,4,5-trispho- <br />     zebrafish. Development 108: 569&#226;&#8364;&#8220;580.                                                          sphate. J Biol Chem 273: 13375&#226;&#8364;&#8220;13378. <br />  5. Solnica-Krezel L (2005) Conserved patterns cell movements vertebrate             20.   Myers MP, Pass I, Batty IH, Van der Kaay J, Stolarov JP, et al. (1998) The lipid <br />     gastrulation. Curr Biol 15: R213&#226;&#8364;&#8220;228.                                                            phosphatase activity PTEN critical tumor supressor function. Proc <br />  6. Solnica-Krezel L (2006) Gastrulation zebrafish &#226;&#8364;&#8220; just adhesion? Curr                Natl Acad Sci U S A 95: 13513&#226;&#8364;&#8220;13518. <br />     Opin Genet Dev 16: 433&#226;&#8364;&#8220;441.                                                                21.   Sulis ML, Parsons R (2003) PTEN: pathology biology. Trends Cell Biol <br />  7. Hammerschmidt M, Wedlich D (2008) Regulated adhesion driving force                       13: 478&#226;&#8364;&#8220;483. <br />     gastrulation movements. Development 135: 3625&#226;&#8364;&#8220;3641.                                        22.   Iijima M, Devreotes P (2002) Tumor suppressor PTEN mediates sensing of <br />  8. Zhu S, Liu L, Korzh V, Gong Z, Low BC (2006) RhoA acts downstream                             chemoattractant gradients. Cell 109: 599&#226;&#8364;&#8220;610. <br />     Wnt5 Wnt11 regulate convergence extension movements                          23.   Funamoto S, Meili R, Lee S, Parry L, Firtel RA (2002) Spatial temporal <br />     involving effectors Rho kinase Diaphanous: use zebrafish vivo                    regulation 3-phosphoinositides PI 3-kinase PTEN mediates <br />     model GTPase signaling. Cell Signal 18: 359&#226;&#8364;&#8220;372.                                             chemotaxis. Cell 109: 611&#226;&#8364;&#8220;623. <br />  9. Sun H, Lesche R, Li DM, Liliental J, Zhang H, et al. (1999) PTEN modulates                 24.   Di Cristofano A, Pesce B, Cordon-Cardo C, Pandolfi PP (1998) Pten essential <br />     cell cycle progression cell survival regulating phosphatidylinositol 3,4,5,-              embryonic development tumour suppression. Nat Genet 19: 348&#226;&#8364;&#8220;355. <br />     trisphosphate Akt/protein kinase B signaling pathway. Proc Natl Acad                   25.   Huang H, Potter CJ, Tao W, Li DM, Brogiolo W, et al. (1999) PTEN affects cell <br />     Sci U S A 96: 6199&#226;&#8364;&#8220;6204.                                                                         size, cell proliferation apoptosis Drosophila eye development. <br /> 10. Montero JA, Kilian B, Chan J, Bayliss PE, Heisenberg CP (2003) Phosphoino-                       Development 126: 5365&#226;&#8364;&#8220;5372. <br />     sitide 3-kinase required process outgrowth cell polarization                 26.   Leslie NR, Yang X, Downes CP, Weijer CJ (2007) PtdIns(3,4,5)P(3)-dependent <br />     gastrulating mesendodermal cells. Curr Biol 13: 1279&#226;&#8364;&#8220;1289.                                       -independent roles PTEN control cell migration. Curr Biol 17: <br /> 11. Besson A, Robbins SM, Yong VW (1999) PTEN/MMAC1/TEP1 signal                                   115&#226;&#8364;&#8220;125. <br />     transduction tumorigenesis. Eur J Biochem 263: 605&#226;&#8364;&#8220;611.                                27.   Suzuki A, la Pompa JL, Stambolic V, Elia AJ, Sasaki T, et al. (1998) High <br /> 12. Cantley LC, Neel BG (1999) New insights tumor suppression: PTEN                             cancer susceptibility embryonic lethality associated mutation the <br />     suppresses tumor formation restraining phosphoinositide 3-kinase/AKT                      PTEN tumor suppressor gene mice. Curr Biol 8: 1169&#226;&#8364;&#8220;1178. <br />     pathway. Proc Natl Acad Sci U S A 96: 4240&#226;&#8364;&#8220;4245.                                           28.   Ueno S, Kono R, Iwao Y (2006) PTEN required normal progression of <br /> 13. Li J, Yen C, Liaw D, Podsypanina K, Bose S, et al. (1997) PTEN, putative                       gastrulation repressing cell proliferation MBT Xenopus embryos. Dev <br />     protein tyrosine phosphatase gene mutated human brain, breast, prostate                   Biol 297: 274&#226;&#8364;&#8220;283. <br />     cancer. Science 275: 1943&#226;&#8364;&#8220;1947.                                                            29.   Rohde LA, Heisenberg CP (2007) Zebrafish gastrulation: cell movements, <br /> 14. Zhou XP, Kuismanen S, Nystrom-Lahti M, Peltomaki P, Eng C (2002) Distinct                        signals, mechanisms. Int Rev Cytol 261: 159&#226;&#8364;&#8220;192. <br />     PTEN mutational spectra hereditary non-polyposis colon cancer syndrome-                 30.   Croushore JA, Blasiole B, Riddle RC, Thisse C, Thisse B, et al. (2005) Ptena and <br />     related endometrial carcinomas compared sporadic microsatellite unstable                      ptenb genes play distinct roles zebrafish embryogenesis. Dev Dyn 234: 911&#226;&#8364;&#8220;921. <br />     tumors. Hum Mol Genet 11: 445&#226;&#8364;&#8220;450.                                                         31.   Fu CT, Zhu KY, Mi JQ, Liu YF, Murray ST, et al. (2010) An evolutionarily <br /> 15. Liaw D, Marsh DJ, Li J, Dahia PL, Wang SI, et al. (1997) Germline mutations                   conserved PTEN-C/EBPalpha-CTNNA1 axis controls myeloid development <br />     PTEN gene Cowden disease, inherited breast thyroid cancer                          transformation. Blood 115: 4715&#226;&#8364;&#8220;4724. <br />     syndrome. Nat Genet 16: 64&#226;&#8364;&#8220;67.                                                             32.   Faucherre A, Taylor GS, Overvoorde J, Dixon JE, den Hertog J (2008) <br /> 16. Marsh DJ, Dahia PL, Zheng Z, Liaw D, Parsons R, et al. (1997) Germline                           Zebrafish pten genes overlapping non-redundant functions in <br />     mutations PTEN present Bannayan-Zonana syndrome. Nat Genet 16:                         tumorigenesis embryonic development. Oncogene 27: 1079&#226;&#8364;&#8220;1086. <br />     333&#226;&#8364;&#8220;334.                                                                                   33.   Hammerschmidt M, Pelegri F, Mullins MC, Kane DA, Brand M, et al. (1996) <br /> 17. Zhou XP, Marsh DJ, Morrison CD, Chaudhury AR, Maxwell M, et al. (2003)                           Mutations affecting morphogenesis gastrulation tail formation the <br />     Germline inactivation PTEN dysregulation phosphoinositol-3-                        zebrafish, Danio rerio. Development 123: 143&#226;&#8364;&#8220;151. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                        14                                     April 2011 | Volume 6 | Issue 4 | e18702 <br />                                                                                                                           Ptenb Mediates Gastrulation Cdc42/AKT1 <br />  <br />  <br /> 34. Le Clainche C, Carlier MF (2008) Regulation actin assembly associated          44. Ro H, Soun K, Kim EJ, Rhee M (2004) Novel vector systems optimized for <br />     protrusion adhesion cell migration. Physiol Rev 88: 489&#226;&#8364;&#8220;513.                        injecting vitro-synthesized mRNA zebrafish embryos. Mol Cells 17: <br /> 35. Liliental J, Moon SY, Lesche R, Mamillapalli R, Li D, et al. (2000) Genetic                373&#226;&#8364;&#8220;376. <br />     deletion Pten tumor suppressor gene promotes cell motility activation        45. Weinberg ES, Allende ML, Kelly CS, Abdelhamid A, Murakami T, et al. (1996) <br />     Rac1 Cdc42 GTPases. Curr Biol 10: 401&#226;&#8364;&#8220;404.                                          Developmental regulation zebrafish MyoD wild-type, tail spadetail <br /> 36. Becker KA, Hart NH (1999) Reorganization filamentous actin myosin-II                embryos. Development 122: 271&#226;&#8364;&#8220; <br /> </body></html>