<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Curr Opin Microbiol. 2013 December ; 16(6): 779&#226;&#8364;&#8220;785. doi:10.1016/j.mib.2013.09.007. <br />  <br />  <br />  <br />                            On Essentiality Lipopolysaccharide Gram-Negative <br />                            Bacteria <br />                            Ge Zhanga, Timothy C. Meredithb, Daniel Kahnea,c,* <br />                            aDepartment Chemistry Chemical Biology, Harvard University, 12 Oxford St, Cambridge, <br />  <br />                            MA 02138 USA <br />                            bDepartment  Microbiology Immunobiology, Harvard Medical School, HIM building, 4 <br />                            Blackfan Circle, Boston, MA 02115 USA <br />                            cDepartment Biological Chemistry Molecular Pharmacology, Harvard Medical School, <br />                            Boston, MA 02115 USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Lipopolysaccharide highly acylated saccharolipid located outer leaflet outer <br />                                 membrane Gram-negative bacteria. Lipopolysaccharide critical maintaining barrier <br />                                 function preventing passive diffusion hydrophobic solutes antibiotics detergents <br />                                 cell. Lipopolysaccharide considered essential component outer membrane <br />                                 biogenesis cell viability based pioneering studies model Gram-negative organisms <br />                                 Escherichia coli Salmonella. With isolation lipopolysaccharide-<span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> in <br />                                 Neisseria meningitides, Moraxella catarrhalis, recently Acinetobacter baumannii, it <br />                                 increasingly apparent lipopolysaccharide essential outer membrane <br />                                 building block organisms. We suggest accumulation toxic intermediates, misassembly <br />                                 essential outer membrane porins, outer membrane stress response pathways are <br />                                 activated mislocalized lipopolysaccharide collectively contribute observed strain- <br />                                 dependent essentiality lipopolysaccharide. <br />  <br />  <br />                            Introduction <br />                                              Gram-positive bacteria contain cytoplasmic membrane surrounded layer of <br />                                              peptidoglycan; contrast, Gram-negative bacteria contain cytoplasm surrounded what <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              appears layers: inner membrane, layer peptidoglycan outer <br />                                              membrane [1,2]. The outer membrane (OM) Gram-negative bacteria asymmetric <br />                                              bilayer inner leaflet consisting phospholipids outer leaflet consisting of <br />                                              lipopolysaccharide (LPS). Much know LPS derives early work <br />                                              beginning 1960s Escherichia coli Salmonella typhimurium. Using newly <br />                                              developed analytical technique allow separation inner membrane (IM) the <br />                                              OM [3], Osborn coworkers established LPS fractionates OM [4]. <br />                                              Remarkably, LPS subsequently shown localized exclusively outer leaflet of <br />                                              OM [5&#226;&#8364;&#8220;7]. At time, site (bio)synthesis LPS determined take <br />                                              place inner membrane [8]. Work Osborn, Raetz, established <br />  <br />  <br />                            &#194;&#169; 2013 Elsevier Ltd. All rights reserved. <br />                            * <br />                              Correspondence: Tel: (617) 496-0208, kahne@chemistry.harvard.edu. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Zhang et al.                                                                                           Page 2 <br />  <br />  <br />                                           steps biosynthesis LPS [9&#226;&#8364;&#8220;14] and, recently, details LPS transport <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           site synthesis cell surface begun uncovered [15]. LPS shown <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                           essential LPS-defective mutants hypersusceptible antibiotics [16&#226;&#8364;&#8220;18]. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           picture emerged sum studies individual LPS molecules <br />                                           interact cell surface mediated divalent cations form a <br />                                           permeability barrier, prevents entry small hydrophobic compounds, as <br />                                           antibiotics, bile salts detergents, allows Gram-negative bacteria survive in <br />                                           harsh environments [19&#226;&#8364;&#8220;22]. Because proper assembly LPS cell surface is <br />                                           required create effective permeability barrier, genes involved biosynthesis and <br />                                           assembly (biogenesis) LPS targets design novel classes of <br />                                           antibiotics [23,24]. <br />  <br />                                           Historically, developing inhibitors LPS biosynthesis predominantly based <br />                                           view LPS essential structural component necessary create OM. LPS <br />                                           large detergent-like molecule comprising regions; highly acylated di-GlcNAc <br />                                           backbone (lipid A) connected polysaccharide containing repeating sugars (O-antigen) <br />                                           linked highly conserved oligosaccharide Kdo/heptose core (Figure 1A). The <br />                                           minimal LPS structure supporting functional OM cell viability enteric bacteria was <br />                                           shown Kdo2-lipid A [14,25]. However, understanding importance LPS in <br />                                           physiology Gram-negative bacteria clouded discovery certain <br />                                           genera require LPS assemble OM survive. Remarkably, certain strains of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Neisseria live lpxA gene encoding enzyme LPS biosynthesis is <br />                                           inactivated, depleting organisms LPS [26]. One early hypothesis explain <br />                                           strains Neisseria survive capsular polysaccharide a <br />                                           structural substitute LPS essential LPS-deficient strains [27]. <br />                                           However, ability construct double mutants lacking lpxA capsule expression <br />                                           N. meningitidis disproved theory [28]. Subsequently viable strains Moraxella and <br />                                           Acinetobacter completely lacking LPS isolated characterized [29,30]. Taken <br />                                           together, studies called question generality conclusion drawn the <br />                                           classic experiments E. coli Salmonella. Clearly assumption LPS simply <br />                                           required structural component outer leaflet OM Gram-negative <br />                                           bacteria correct. The essentiality LPS varies considerably, depending only <br />                                           genera Gram-negative bacteria species, cases, on <br />                                           particular strain background. This review consider alternate explanations account <br />                                           strain-dependent LPS essentiality Gram-negative bacteria discuss underlying <br />                                           implications developing antibiotics targeting LPS. <br />  <br />                               LPS synthesis assembly pathways E. coli <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The LPS biosynthesis/transport pathway spans compartments Gram-negative <br />                                           bacteria [14]. In E. coli, Kdo2-lipid A domain synthesized inside cytoplasm [11&#226;&#8364;&#8220; <br />                                           13]. After sequential addition sugars produce lipid A-oligosaccharide core the <br />                                           cytoplasmic membrane, molecule flipped periplasmic face inner <br />                                           membrane ABC transporter (MsbA) O-antigen added [31&#226;&#8364;&#8220;36]. In the <br />                                           canonical (Wzy-dependent) O-antigen pathway E. coli, O-antigen biosynthesis begins in <br />                                           cytoplasm sequential addition monosaccharides the <br />                                           undecaprenyl monophosphate carrier lipid (Und-P) make O-antigen repeat subunit. <br />                                           This oligosaccharide subunit transported periplasmic face IM Wzx and <br />                                           polymerized en bloc Wzy form mature O-antigen polysaccharide chain containing as <br />                                           40 200 repeat units. This polysaccharide transferred lipid A- <br />                                           core acceptor O-antigen ligase WaaL [14] prior transit periplasm the <br />                                           cell surface Lpt pathway [36] (Figure 1B). The Lpt pathway consists seven <br />                                           proteins form trans-envelope structure containing IM complex (LptB/F/G/C) <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                              Page 3 <br />  <br />  <br />                                           required extract LPS IM, bridge (LptA) IM OM permit <br />                                           transit aqueous periplasmic compartment, OM translocon (LptD/E) to <br />                                           allow large detergent-like LPS molecule pass OM final destination <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           cell surface [37&#226;&#8364;&#8220;54] (Figure 1C). <br />  <br />                                           The loss LPS biosynthesis given organism deep-seeded consequences the <br />                                           assembly components cell envelope. Syntheses O-antigen, peptidoglycan, <br />                                           secondary cell wall polymers outer-membrane proteins (OMPs) impacted the <br />                                           absence LPS [14,15]. While LPS itself maybe required viability, extent to <br />                                           essential functions cell envelope compromised loss LPS could <br />                                           ultimately determine LPS essential given strain. <br />  <br />                               Inhibition LPS biosynthesis cause accumulation cell envelope <br />                               components inappropriate compartments <br />                                           Inhibition LPS biosynthesis (e.g. LpxC deletion, committed step LPS <br />                                           biosynthesis) depletes levels oligosaccharide lipid A core IM. The lack of <br />                                           oligosaccharide lipid A core acceptor available O-antigen transfer potentially cause <br />                                           unligatable Und-PP O-antigen precursors accumulate. Accumulation precursors <br />                                           shown toxic Salmonella, leading suggestion undecaprenyl <br />                                           sequestration influences essential Und-P dependent pathways [55]. Both O-antigen and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           peptidoglycan biogenesis utilize membrane-bound carrier addition nucleotide <br />                                           sugars. Because undecaprenyl levels limited bacterial membranes, blocking transfer of <br />                                           Und-PP O-antigen lipid A core leads sequestration Und-P, depleting the <br />                                           Und-P pool available essential pathways peptidoglycan biosynthesis. This <br />                                           toxicity predicted highly dependent strain background. <br />  <br />                                           There different factors contribute bacterial strain is <br />                                           susceptible accumulation Und-PP O-antigen. For instance, O-antigen <br />                                           serotypes utilize second distinct biosynthetic pathway, ABC transporter-dependent <br />                                           pathway [14]. The O-antigen homopolymer assembled single Und-P carrier the <br />                                           cytoplasm prior transport ligation oligosaccharide-lipid A acceptor the <br />                                           periplasmic face IM [14,56]. Those strains utilize ABC pathway would <br />                                           clearly susceptible accumulation O-antigen demand the <br />                                           carrier lipid far less. Secondly, different strains bacteria modify lipid A core with <br />                                           O-antigen varying extents [57]. Some bacteria tend cap larger portion lipid A- <br />                                           oligosaccharides O-antigen, producing smooth LPS, organisms tend to <br />                                           predominantly underivatized lipid A-oligosaccharide core present surface <br />                                           (rough LPS) [14]. The extent lipid A-oligosaccharides end-capped O- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           antigen (smooth rough ratio) reflects flux pathway, turn is <br />                                           related usage Und-P carrier. A final factor contributing sensitivity given <br />                                           strain O-antigen accumulation resulting Und-P sequestration cellular <br />                                           mechanisms exist process stalled intermediates. Ordinarily, O-antigen <br />                                           polysaccharide transferred lipid A-core WaaL, newly released Und-PP is <br />                                           recycled active monophosphoryl form Und-P pyrophosphatases, which <br />                                           liberates carriers enabling round O-antigen peptidoglycan biosynthesis [58]. <br />                                           It possible certain pyrophosphatases cleave O-antigen precursors its <br />                                           biosynthesis ligation stalls allow Und-P lipid carrier recycled. Indeed, in <br />                                           certain E. coli strains group 4 capsules, fraction O-antigen normally released by <br />                                           hydrolysis form extracellular capsule polysaccharide layer [59,60]. This discussion is <br />                                           simply meant illustrate strain-specific mechanisms relieve the <br />                                           buildup O-antigen intermediates result toxicity to <br />                                           sequestration lipid carrier. <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                           Page 4 <br />  <br />  <br />                               Inhibition LPS biosynthesis affect assembly function of <br />                               membrane proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           In addition LPS, outer membrane Gram-negative bacteria contains major <br />                                           classes proteins: lipoproteins integral membrane proteins &#206;&#178;-barrel structure. The <br />                                           exact function membrane &#206;&#178;-barrel proteins known, believed to <br />                                           form pores (porins) membrane provide nonspecific channels OM to <br />                                           allow entry nutrients, generally small hydrophilic [22,61]. It believed <br />                                           LPS facilitates porin assembly function acting molecular chaperone [35]. <br />                                           For example, porins OmpC OmpF depend LPS trimerization [62&#226;&#8364;&#8220;64] for <br />                                           maintaining proper channel gating function [65], protease OmpT requires LPS for <br />                                           proteolytic activity [66]. While complement essential OMPs defined <br />                                           limited number species, outer-membrane &#206;&#178;-barrel proteins known be <br />                                           essential E. coli. One, LptD component heterodimeric OM translocon <br />                                           responsible LPS transport assembly cell surface [38,39,41,49&#226;&#8364;&#8220;51]. In those <br />                                           strains Neisseria LPS essential, LptD non-essential [39]. <br />                                           The BamA essential component five-protein complex responsible for <br />                                           assembling OMPs [67&#226;&#8364;&#8220;70]. In fact, endosymbionts evolved <br />                                           minimal genomes contain genes involved LPS biogenesis pathway (either Lpx <br />                                           Lpt) [71,72]. However BamA generally essential minimal genomes, <br />                                           suggesting &#206;&#178;-barrel proteins present permit passage metabolites across <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           outer membrane. Clearly, different strains bacteria unique <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> requirements <br />                                           depend specific repertoire porins essential <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake. In the <br />                                           case porins depend LPS folding/function, LPS become <br />                                           indispensable. <br />  <br />                                           The loss LPS disrupt structure function inner membrane. It is <br />                                           possible inhibition LPS biosynthesis leads accumulation glycolipid (e.g., Und- <br />                                           PP O-antigen, Und-PP enterobacterial common antigen [73,74], secondary cell <br />                                           envelope polymers [75]) intermediates. One imagine scenarios which <br />                                           accumulated glycolipid intermediates compromise IM functions. The <br />                                           accumulation glycolipid intermediates influence functions IM proteins. For <br />                                           example, various essential IM proteins involved peptidoglycan biosynthesis [76]. <br />                                           Since O-antigen Lipid II contain Und-PP activating group, accumulation of <br />                                           Und-PP O-antigen compete Lipid II inhibiting cell wall synthesis. Additional <br />                                           glycolipids influence proper assembly inner-membrane proteins by <br />                                           affecting lipid environment IM. Simply affecting IM bilayer packing, these <br />                                           accumulated glycolipids create physical defects IM cause problems by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           dissipation membrane potential. Here again, presence strain-specific mechanisms to <br />                                           relieve buildup O-antigen intermediates (or secondary cell envelope polymers) <br />                                           determine given Gram-negative organism susceptible LPS <br />                                           deletion. <br />  <br />                               Inhibition LPS biosynthesis trigger stress response pathways <br />                               causing inhibition growth <br />                                           Inhibition growth result directly loss LPS cellular response <br />                                           loss &#226;&#8364;&#8220; activation alternative genetic program response stress LPS <br />                                           deletion. It shown accumulation mistargeted and/or misfolded outer <br />                                           membrane proteins periplasm detected sensor protein, DegS, initiating a <br />                                           proteolytic cascade results activation &#207;&#402;E -dependent envelope stress response <br />                                           [77,78] (Figure 1D). The &#207;&#402;E -transcription factor up-regulates expression of <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                            Page 5 <br />  <br />  <br />                                           genes involved targeting assembly OMPs, genes encoding <br />                                           proteases order clear misfolded substrates [79]. Literally hundreds genes turned <br />                                           order restore intracellular trafficking OMPs. Because various OMPs require <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           LPS fold, inhibition LPS biosynthesis cause misfolding porins initiating <br />                                           envelope stress response resulting growth stasis intracellular levels LPS <br />                                           restored [80&#226;&#8364;&#8220;82]. A recent report suggested surveillance systems <br />                                           directly detect mistargeted LPS periplasm E. coli, triggers envelope <br />                                           stress response activating &#207;&#402;E (Figure 1D) [83]. The fact bacteria respond LPS and <br />                                           OMP defects loss activating quality control mechanism emphasizes the <br />                                           essentiality LPS maintaining OM integrity. In E. coli Salmonella, particularly <br />                                           stringent OM stress response presumably evolved order allow organisms to <br />                                           colonize gut, high concentration detergent-like molecules (e.g. bile salts) <br />                                           tolerated [84]. Hence, LPS may, part, required prevent growth stasis <br />                                           triggered stress response systems. Although relatively known stress response <br />                                           systems Gram-negative organisms, conceivable OM quality control <br />                                           surveillance systems prominent counterparts E. coli Salmonella <br />                                           [35]. These bacteria continue grow divide OM compromised state, whereas <br />                                           strains cease growing stress response signaling. <br />  <br />                               Conclusion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           For decades LPS thought essential structural component OM of <br />                                           Gram-negative bacteria just amino acids essential structures proteins. <br />                                           However, discovery LPS-deficient organisms clear essentiality <br />                                           LPS Gram-negative bacteria complex. It reasonable LPS was <br />                                           selected Nature combined asymmetric bilayer phospholipids it <br />                                           produces unusual membrane prevents passage toxic hydrophobic molecules <br />                                           cell. However, point, maintaining proper LPS permeability barrier is <br />                                           tangential essentiality. Having heavily integrated cell envelope <br />                                           physiology, removal affect metabolic processes indirect means. <br />                                           Whether possible delete LPS given organism depend cellular <br />                                           context removed. While LPS non-essential some <br />                                           organisms, loss inconsequential. Strains lacking LPS virulent [85] and <br />                                           susceptible antibiotics normally penetrate OM [29,30,86]. It is <br />                                           believed compounds interfere functions LPS synthesis, transport, <br />                                           assembly, potential function antibiotics to <br />                                           potentiate entry existing antibiotics normally excluded OM. In fact, a <br />                                           better understanding importance LPS biogenesis bacterial physiology could <br />                                           provide clues specific vulnerabilities given Gram-negative pathogen to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           inhibition LPS different steps synthesis assembly. <br />  <br />                               Acknowledgments <br />                                           This work supported Blavatnik Biomedical Accelerator Fund National Institutes Health (NIH) <br />                                           grant AI081059 D.K. <br />  <br />  <br />                               References <br />                                           1. Kellenberger E, Ryter A. Cell wall cytoplasmic membrane Escherichia coli. J Biophys <br />                                              Biochem Cytol. 1958; 4:323&#226;&#8364;&#8220;326. [PubMed: 13549505] <br />                                           2. Bladen HA, Mergenhagen SE. Ultrastructure Veillonella Morphological Correlation an <br />                                              Outer Membrane Particles Associated Endotoxic Activity. J Bacteriol. 1964; 88:1482&#226;&#8364;&#8220; <br />                                              1492. [PubMed: 14234809] <br />  <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                   Page 6 <br />  <br />  <br />                                           3. Miura T, Mizushima S. Separation density gradient centrifugation types membranes <br />                                              spheroplast membrane Escherichia coli K12. Biochim Biophys Acta. 1968; 150:159&#226;&#8364;&#8220;161. <br />                                              [PubMed: 4868245] Developed sucrose density/gradient centrifugation technique separate IM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              OM. <br />                                           4. Osborn MJ, Gander JE, Parisi E, Carson J. Mechanism assembly outer membrane of <br />                                              Salmonella typhimurium. Isolation characterization cytoplasmic outer membrane. J Biol <br />                                              Chem. 1972; 247:3962&#226;&#8364;&#8220;3972. [PubMed: 4555955] Analyzed composition OM and <br />                                              established LPS localization membrane. <br />                                           5. Muhlradt PF, Golecki JR. Asymmetrical distribution artifactual reorientation of <br />                                              lipopolysaccharide outer membrane bilayer Salmonella typhimurium. Eur J Biochem. <br />                                              1975; 51:343&#226;&#8364;&#8220;352. [PubMed: 807474] <br />                                           6. Kamio Y, Nikaido H. Outer membrane Salmonella typhimurium: accessibility phospholipid <br />                                              head groups phospholipase c cyanogen bromide activated dextran external medium. <br />                                              Biochemistry. 1976; 15:2561&#226;&#8364;&#8220;2570. [PubMed: 820368] Classic paper, established the <br />                                              outer leaflet OM devoid phospholipids, leading hypothesis OM an <br />                                              asymmetric bilayer. <br />                                           7. Funahara Y, Nikaido H. Asymmetric localization lipopolysaccharides outer membrane of <br />                                              Salmonella typhimurium. J Bacteriol. 1980; 141:1463&#226;&#8364;&#8220;1465. [PubMed: 6988417] <br />                                           8. Osborn MJ, Gander JE, Parisi E. Mechanism assembly outer membrane Salmonella <br />                                              typhimurium. Site synthesis lipopolysaccharide. J Biol Chem. 1972; 247:3973&#226;&#8364;&#8220;3986. <br />                                              [PubMed: 4624447] Established site (bio)synthesis LPS IM. Pointed that <br />                                              observation raises question transport LPS cellular compartments -- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              important unanswered question field. <br />                                           9. Rick PD, Fung LW, Ho C, Osborn MJ. Lipid A mutants Salmonella typhimurium. Purification <br />                                              characterization lipid A precursor produced mutant 3-deoxy-D- <br />                                              mannooctulosonate-8-phosphate synthetase. J Biol Chem. 1977; 252:4904&#226;&#8364;&#8220;4912. [PubMed: 17608] <br />                                           10. Rick PD, Osborn MJ. Lipid A mutants Salmonella typhimurium. Characterization a <br />                                                conditional lethal mutant 3-deoxy-D-mannooctulosonate-8-phosphate synthetase. J Biol Chem. <br />                                                1977; 252:4895&#226;&#8364;&#8220;4903. [PubMed: 326782] <br />                                           11. Crowell DN, Anderson MS, Raetz CR. Molecular cloning genes lipid A disaccharide <br />                                                synthase UDP-N-acetylglucosamine acyltransferase Escherichia coli. J Bacteriol. 1986; <br />                                                168:152&#226;&#8364;&#8220;159. [PubMed: 3531165] <br />                                           12. Kelly TM, Stachula SA, Raetz CR, Anderson MS. The firA gene Escherichia coli encodes <br />                                                UDP-3-O-(R-3-hydroxymyristoyl)-glucosamine N-acyltransferase. The step endotoxin <br />                                                biosynthesis. J Biol Chem. 1993; 268:19866&#226;&#8364;&#8220;19874. [PubMed: 8366125] <br />                                           13. Young K, Silver LL, Bramhill D, Cameron P, Eveland SS, Raetz CR, Hyland SA, Anderson MS. <br />                                                The envA permeability/cell division gene Escherichia coli encodes second enzyme lipid <br />                                                A biosynthesis. UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase. J Biol <br />                                                Chem. 1995; 270:30384&#226;&#8364;&#8220;30391. [PubMed: 8530464] <br />                                           14. Raetz CR, Whitfield C. Lipopolysaccharide endotoxins. Annu Rev Biochem. 2002; 71:635&#226;&#8364;&#8220;700. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                [PubMed: 12045108] <br />                                           15. Ruiz N, Kahne D, Silhavy TJ. Transport lipopolysaccharide cell envelope: long <br />                                                road discovery. Nat Rev Microbiol. 2009; 7:677&#226;&#8364;&#8220;683. [PubMed: 19633680] <br />                                           16. Tamaki S, Sato T, Matsuhashi M. Role lipopolysaccharides antibiotic resistance and <br />                                                bacteriophage adsorption Escherichia coli K-12. J Bacteriol. 1971; 105:968&#226;&#8364;&#8220;975. [PubMed: <br />                                                4926688] <br />                                           17. Sanderson KE, MacAlister T, Costerton JW, Cheng KJ. Permeability lipopolysaccharide- <br />                                                deficient (rough) mutants Salmonella typhimurium antibiotics, lysozyme, agents. <br />                                                Can J Microbiol. 1974; 20:1135&#226;&#8364;&#8220;1145. [PubMed: 4213795] <br />                                           18. Vaara M. Antibiotic-supersusceptible mutants Escherichia coli Salmonella typhimurium. <br />                                                Antimicrob Agents Chemother. 1993; 37:2255&#226;&#8364;&#8220;2260. [PubMed: 8285603] <br />                                           19. Macgregor DR, Elliker PR. A comparison properties strains Pseudomonas <br />                                                aeruginosa sensitive resistant quaternary ammonium compounds. Can J Microbiol. 1958; <br />                                                4:499&#226;&#8364;&#8220;503. [PubMed: 13573235] <br />  <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                    Page 7 <br />  <br />  <br />                                           20. Leive L. A Nonspecific Increase Permeability Escherichia Coli Produced Edta. Proc Natl <br />                                               Acad Sci U S A. 1965; 53:745&#226;&#8364;&#8220;750. [PubMed: 14324530] <br />                                           21. Leive L. The barrier function gram-negative envelope. Ann N Y Acad Sci. 1974; 235:109&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               129. [PubMed: 4212391] <br />                                           22. Nikaido H. Molecular Basis Bacterial Outer Membrane Permeability Revisited. Microbiol Mol <br />                                               Biol Rev. 2003; 67:593&#226;&#8364;&#8220;656. [PubMed: 14665678] <br />                                           23. Onishi HR, Pelak BA, Gerckens LS, Silver LL, Kahan FM, Chen MH, Patchett AA, Galloway SM, <br />                                               Hyland SA, Anderson MS, et al. Antibacterial agents inhibit lipid A biosynthesis. Science. <br />                                               1996; 274:980&#226;&#8364;&#8220;982. [PubMed: 8875939] Most studied antibiotic target LPS biosynthesis. <br />                                           24. Srinivas N, Jetter P, Ueberbacher BJ, Werneburg M, Zerbe K, Steinmann J, Van der Meijden B, <br />                                               Bernardini F, Lederer A, Dias RL, et al. Peptidomimetic antibiotics target outer-membrane <br />                                               biogenesis Pseudomonas aeruginosa. Science. 2010; 327:1010&#226;&#8364;&#8220;1013. [PubMed: 20167788] First <br />                                               report small molecule targets component (LptD) OM biogenesis (LPS assembly) <br />                                               machinery. <br />                                           25. Gronow S, Brade H. Lipopolysaccharide biosynthesis: steps bacteria need survive? J <br />                                               Endotoxin Res. 2001; 7:3&#226;&#8364;&#8220;23. [PubMed: 11521077] <br />                                           26. Steeghs L, den Hartog R, den Boer A, Zomer B, Roholl P, van der Ley P. Meningitis bacterium is <br />                                               viable endotoxin. Nature. 1998; 392:449&#226;&#8364;&#8220;450. [PubMed: 9548250] Defined and <br />                                               characterized strain Gram-negative bacteria completely lacking LPS. This paper called <br />                                               question generality LPS essentiality. <br />                                           27. Steeghs L, Cock H, Evers E, Zomer B, Tommassen J, van der Ley P. Outer membrane <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               composition lipopolysaccharide-deficient Neisseria meningitidis mutant. EMBO J. 2001; <br />                                               20:6937&#226;&#8364;&#8220;6945. [PubMed: 11742971] <br />                                           28. Bos MP, Tommassen J. Viability capsule- lipopolysaccharide-deficient mutant of <br />                                               Neisseria meningitidis. Infect Immun. 2005; 73:6194&#226;&#8364;&#8220;6197. [PubMed: 16113348] <br />                                           29. Peng D, Hong W, Choudhury BP, Carlson RW, Gu XX. Moraxella catarrhalis bacterium without <br />                                               endotoxin, potential vaccine candidate. Infect Immun. 2005; 73:7569&#226;&#8364;&#8220;7577. [PubMed: <br />                                               16239560] <br />                                           30. Moffatt JH, Harper M, Harrison P, Hale JD, Vinogradov E, Seemann T, Henry R, Crane B, St <br />                                               Michael F, Cox AD, et al. Colistin resistance Acinetobacter baumannii mediated complete <br />                                               loss lipopolysaccharide production. Antimicrob Agents Chemother. 2010; 54:4971&#226;&#8364;&#8220;4977. <br />                                               [PubMed: 20855724] This paper determined LPS essential Acinetobacter baumannii, <br />                                               Peng et al.&#226;&#8364;&#8482;s study Moraxella raised question makes LPS <br />                                               essential organisms, others. <br />                                           31. Karow M, Georgopoulos C. The essential Escherichia coli msbA gene, multicopy suppressor of <br />                                               null mutations htrB gene, related universally conserved family ATP-dependent <br />                                               translocators. Mol Microbiol. 1993; 7:69&#226;&#8364;&#8220;79. [PubMed: 8094880] <br />                                           32. Clementz T, Bednarski JJ, Raetz CR. Function htrB high temperature requirement gene of <br />                                               Escherchia coli acylation lipid A: HtrB catalyzed incorporation laurate. J Biol Chem. <br />                                               1996; 271:12095&#226;&#8364;&#8220;12102. [PubMed: 8662613] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           33. Zhou Z, White KA, Polissi A, Georgopoulos C, Raetz CR. Function Escherichia coli MsbA, an <br />                                               Essential ABC Family Transporter, Lipid A Phospholipid Biosynthesis. J Biol Chem. 1998; <br />                                               273:12466&#226;&#8364;&#8220;12475. [PubMed: 9575204] <br />                                           34. Doerrler WT, Gibbons HS, Raetz CR. MsbA-dependent translocation lipids inner <br />                                               membrane Escherichia coli. J Biol Chem. 2004; 279:45102&#226;&#8364;&#8220;45109. [PubMed: 15304478] <br />                                           35. Bos MP, Robert V, Tommassen J. Biogenesis gram-negative bacterial outer membrane. <br />                                               Annu Rev Microbiol. 2007; 61:191&#226;&#8364;&#8220;214. [PubMed: 17506684] <br />                                           36. Liechti G, Goldberg JB. Outer membrane biogenesis Escherichia coli, Neisseria meningitidis, <br />                                               Helicobacter pylori: paradigm deviations H. pylori. Front Cell Infect Microbiol. 2012; 2:29. <br />                                               [PubMed: 22919621] <br />                                           37. Sampson BA, Misra R, Benson SA. Identification characterization new gene of <br />                                               Escherichia coli K-12 involved outer membrane permeability. Genetics. 1989; 122:491&#226;&#8364;&#8220;501. <br />                                               [PubMed: 2547691] Clever selection. Discovery gene (originally known Imp -- <br />                                               Increased membrane permeability) LPS transport/assembly pathway. <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                  Page 8 <br />  <br />  <br />                                           38. Braun M, Silhavy TJ. Imp/OstA required cell envelope biogenesis Escherichia coli. Mol <br />                                               Microbiol. 2002; 45:1289&#226;&#8364;&#8220;1302. [PubMed: 12207697] Established function LptD <br />                                               (previously called Imp) OM biogenesis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           39. Bos MP, Tefsen B, Geurtsen J, Tommassen J. Identification outer membrane protein required <br />                                               transport lipopolysaccharide bacterial cell surface. Proc Natl Acad Sci U S A. <br />                                               2004; 101:9417&#226;&#8364;&#8220;9422. [PubMed: 15192148] Identified functional role Imp (LptD) the <br />                                               outer-membrane transporter LPS cell surface knocking LptD Neisseria. This <br />                                               particular strain LPS essential. <br />                                           40. Serina S, Nozza F, Nicastro G, Faggioni F, Mottl H, Deho G, Polissi A. Scanning Escherichia <br />                                               coli chromosome random transposon mutagenesis multiple phenotypic screening. Res <br />                                               Microbiol. 2004; 155:692&#226;&#8364;&#8220;701. [PubMed: 15380559] <br />                                           41. Wu T, McCandlish AC, Gronenberg LS, Chng SS, Silhavy TJ, Kahne D. Identification protein <br />                                               complex assembles lipopolysaccharide outer membrane Escherichia coli. Proc Natl <br />                                               Acad Sci U S A. 2006; 103:11754&#226;&#8364;&#8220;11759. [PubMed: 16861298] <br />                                           42. Sperandeo P, Cescutti R, Villa R, Di Benedetto C, Candia D, Deho G, Polissi A. Characterization <br />                                               lptA lptB, essential genes implicated lipopolysaccharide transport outer <br />                                               membrane Escherichia coli. J Bacteriol. 2007; 189:244&#226;&#8364;&#8220;253. [PubMed: 17056748] <br />                                           43. Tran AX, Trent MS, Whitfield C. The LptA protein Escherichia coli periplasmic lipid A- <br />                                               binding protein involved lipopolysaccharide export pathway. J Biol Chem. 2008; <br />                                               283:20342&#226;&#8364;&#8220;20349. [PubMed: 18480051] <br />                                           44. Sperandeo P, Lau FK, Carpentieri A, De Castro C, Molinaro A, Deho G, Silhavy TJ, Polissi A. <br />                                               Functional analysis protein machinery required transport lipopolysaccharide the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               outer membrane Escherichia coli. J Bacteriol. 2008; 190:4460&#226;&#8364;&#8220;4469. [PubMed: 18424520] <br />                                           45. Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ. Identification inner-membrane proteins <br />                                               required transport lipopolysaccharide outer membrane Escherichia coli. Proc <br />                                               Natl Acad Sci U S A. 2008; 105:5537&#226;&#8364;&#8220;5542. [PubMed: 18375759] <br />                                           46. Sperandeo P, Deho G, Polissi A. The lipopolysaccharide transport Gram-negative <br />                                               bacteria. Biochim Biophys Acta. 2009; 1791:594&#226;&#8364;&#8220;602. [PubMed: 19416651] <br />                                           47. Narita S, Tokuda H. Biochemical characterization ABC transporter LptBFGC complex <br />                                               required outer membrane sorting lipopolysaccharides. FEBS Lett. 2009; 583:2160&#226;&#8364;&#8220;2164. <br />                                               [PubMed: 19500581] <br />                                           48. Ruiz N, Chng SS, Hiniker A, Kahne D, Silhavy TJ. Nonconsecutive disulfide bond formation an <br />                                               essential integral outer membrane protein. Proc Natl Acad Sci U S A. 2010; 107:12245&#226;&#8364;&#8220;12250. <br />                                               [PubMed: 20566849] <br />                                           49. Chng SS, Ruiz N, Chimalakonda G, Silhavy TJ, Kahne D. Characterization two-protein <br />                                               complex Escherichia coli responsible lipopolysaccharide assembly outer membrane. <br />                                               Proc Natl Acad Sci U S A. 2010; 107:5363&#226;&#8364;&#8220;5368. [PubMed: 20203010] <br />                                           50. Chimalakonda G, Ruiz N, Chng SS, Garner RA, Kahne D, Silhavy TJ. Lipoprotein LptE is <br />                                               required assembly LptD beta-barrel assembly machine outer membrane of <br />                                               Escherichia coli. Proc Natl Acad Sci U S A. 2011; 108:2492&#226;&#8364;&#8220;2497. [PubMed: 21257909] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           51. Freinkman E, Chng SS, Kahne D. The complex inserts lipopolysaccharide bacterial <br />                                               outer membrane forms two-protein plug-and-barrel. Proc Natl Acad Sci U S A. 2011; 108:2486&#226;&#8364;&#8220; <br />                                               2491. [PubMed: 21257904] <br />                                           52. Freinkman E, Okuda S, Ruiz N, Kahne D. Regulated assembly transenvelope protein <br />                                               complex required lipopolysaccharide export. Biochemistry. 2012; 51:4800&#226;&#8364;&#8220;4806. [PubMed: <br />                                               22668317] <br />                                           53. Okuda S, Freinkman E, Kahne D. Cytoplasmic ATP hydrolysis powers transport of <br />                                               lipopolysaccharide periplasm E. coli. Science. 2012; 338:1214&#226;&#8364;&#8220;1217. [PubMed: <br />                                               23138981] This paper established Lpt proteins directly transport LPS. Observed intermediates <br />                                               LPS tansport opening door mechanistic studies. <br />                                           54. Chng SS, Xue M, Garner RA, Kadokura H, Boyd D, Beckwith J, Kahne D. Disulfide <br />                                               rearrangement triggered translocon assembly controls lipopolysaccharide export. Science. <br />                                               2012; 337:1665&#226;&#8364;&#8220;1668. [PubMed: 22936569] <br />  <br />  <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                  Page 9 <br />  <br />  <br />                                           55. Yuasa R, Levinthal M, Nikaido H. Biosynthesis cell wall lipopolysaccharide mutants of <br />                                               Salmonella. V. A mutant Salmonella typhimurium defective synthesis cytidine <br />                                               diphosphoabequose. J Bacteriol. 1969; 100:433&#226;&#8364;&#8220;444. [PubMed: 4899003] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           56. Greenfield LK, Whitfield C. Synthesis lipopolysaccharide O-antigens ABC transporter- <br />                                               dependent pathways. Carbohydr Res. 2012; 356:12&#226;&#8364;&#8220;24. [PubMed: 22475157] <br />                                           57. Burrows LL, Lam JS. Effect wzx (rfbX) mutations A-band B-band lipopolysaccharide <br />                                               biosynthesis Pseudomonas aeruginosa O5. J Bacteriol. 1999; 181:973&#226;&#8364;&#8220;980. [PubMed: 9922263] <br />                                           58. Tatar LD, Marolda CL, Polischuk AN, van Leeuwen D, Valvano MA. An Escherichia coli <br />                                               undecaprenyl-pyrophosphate phosphatase implicated undecaprenyl phosphate recycling. <br />                                               Microbiology. 2007; 153:2518&#226;&#8364;&#8220;2529. [PubMed: 17660416] <br />                                           59. Whitfield C, Roberts IS. Structure, assembly regulation expression capsules in <br />                                               Escherichia coli. Mol Microbiol. 1999; 31:1307&#226;&#8364;&#8220;1319. [PubMed: 10200953] <br />                                           60. Whitfield C. Biosynthesis assembly capsular polysaccharides Escherichia coli. Annu Rev <br />                                               Biochem. 2006; 75:39&#226;&#8364;&#8220;68. [PubMed: 16756484] <br />                                           61. Danese PN, Silhavy TJ. Targeting assembly periplasmic outer-membrane proteins in <br />                                               Escherichia coli. Annu Rev Genet. 1998; 32:59&#226;&#8364;&#8220;94. [PubMed: 9928475] <br />                                           62. Ried G, Hindennach I, Henning U. Role lipopolysaccharide assembly Escherichia coli <br />                                               outer membrane proteins OmpA, OmpC, OmpF. J Bacteriol. 1990; 172:6048&#226;&#8364;&#8220;6053. [PubMed: <br />                                               2170338] <br />                                           63. Sen K, Nikaido H. In vitro trimerization OmpF porin secreted spheroplasts Escherichia <br />                                               coli. Proc Natl Acad Sci U S A. 1990; 87:743&#226;&#8364;&#8220;747. [PubMed: 1689050] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           64. Sen K, Nikaido H. Lipopolysaccharide structure required vitro trimerization Escherichia <br />                                               coli OmpF porin. J Bacteriol. 1991; 173:926&#226;&#8364;&#8220;928. [PubMed: 1702785] <br />                                           65. Schindler H, Rosenbusch JP. Matrix protein planar membranes: clusters channels native <br />                                               environment functional reassembly. Proc Natl Acad Sci U S A. 1981; 78:2302&#226;&#8364;&#8220;2306. <br />                                               [PubMed: 6264473] <br />                                           66. Kramer RA, Brandenburg K, Vandeputte-Rutten L, Werkhoven M, Gros P, Dekker N, Egmond <br />                                               MR. Lipopolysaccharide regions involved activation Escherichia coli outer membrane <br />                                               protease OmpT. Eur J Biochem. 2002; 269:1746&#226;&#8364;&#8220;1752. [PubMed: 11895445] <br />                                           67. Voulhoux R, Bos MP, Geurtsen J, Mols M, Tommassen J. Role highly conserved bacterial <br />                                               protein outer membrane protein assembly. Science. 2003; 299:262&#226;&#8364;&#8220;265. [PubMed: 12522254] <br />                                           68. Wu T, Malinverni J, Ruiz N, Kim S, Silhavy TJ, Kahne D. Identification multicomponent <br />                                               complex required outer membrane biogenesis Escherichia coli. Cell. 2005; 121:235&#226;&#8364;&#8220;245. <br />                                               [PubMed: 15851030] <br />                                           69. Ruiz N, Falcone B, Kahne D, Silhavy TJ. Chemical conditionality: genetic strategy probe <br />                                               organelle assembly. Cell. 2005; 121:307&#226;&#8364;&#8220;317. [PubMed: 15851036] <br />                                           70. Hagan CL, Kim S, Kahne D. Reconstitution outer membrane protein assembly purified <br />                                               components. Science. 2010; 328:890&#226;&#8364;&#8220;892. [PubMed: 20378773] <br />                                           71. Lin M, Rikihisa Y. Ehrlichia chaffeensis Anaplasma phagocytophilum lack genes lipid A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               biosynthesis incorporate cholesterol <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span>. Infect Immun. 2003; 71:5324&#226;&#8364;&#8220;5331. <br />                                               [PubMed: 12933880] <br />                                           72. Foster J, Ganatra M, Kamal I, Ware J, Makarova K, Ivanova N, Bhattacharyya A, Kapatral V, <br />                                               Kumar S, Posfai J, et al. The Wolbachia genome Brugia malayi: endosymbiont evolution within <br />                                               human pathogenic nematode. PLoS Biol. 2005; 3:e121. [PubMed: 15780005] <br />                                           73. Rick PD, Wolski S, Barr K, Ward S, Ramsay-Sharer L. Accumulation lipid-linked <br />                                               intermediate involved enterobacterial common antigen synthesis Salmonella typhimurium <br />                                               mutants lacking dTDP-glucose pyrophosphorylase. J Bacteriol. 1988; 170:4008&#226;&#8364;&#8220;4014. [PubMed: <br />                                               2842298] <br />                                           74. Rick PD, Hubbard GL, Kitaoka M, Nagaki H, Kinoshita T, Dowd S, Simplaceanu V, Ho C. <br />                                               Characterization lipid-carrier involved synthesis enterobacterial common antigen <br />                                               (ECA) identification novel phosphoglyceride mutant Salmonella typhimurium <br />                                               defective ECA synthesis. Glycobiology. 1998; 8:557&#226;&#8364;&#8220;567. [PubMed: 9592122] <br />  <br />  <br />  <br />  <br />                                           Curr Opin Microbiol. Author manuscript; available PMC 2014 December 01. <br />                             Zhang et al.                                                                                                     Page 10 <br />  <br />  <br />                                           75. Meredith TC, Mamat U, Kaczynski Z, Lindner B, Holst O, Woodard RW. Modification of <br />                                               lipopolysaccharide colanic acid (M-antigen) repeats Escherichia coli. J Biol Chem. 2007; <br />                                               282:7790&#226;&#8364;&#8220;7798. [PubMed: 17227761] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           76. Typas A, Banzhaf M, Gross CA, Vollmer W. From regulation peptidoglycan synthesis to <br />                                               bacterial growth morphology. Nat Rev Microbiol. 2012; 10:123&#226;&#8364;&#8220;136. [PubMed: 22203377] <br />                                           77. Walsh NP, Alba BM, Bose B, Gross CA, Sauer RT. OMP peptide signals initiate envelope- <br />                                               stress response activating DegS protease relief inhibition mediated PDZ domain. <br />                                               Cell. 2003; 113:61&#226;&#8364;&#8220;71. [PubMed: 12679035] <br />                                           78. Alba BM, Gross CA. Regulation Escherichia coli sigma-dependent envelope stress response. <br />                                               Mol Microbiol. 2004; 52:613&#226;&#8364;&#8220;619. [PubMed: 15101969] <br />                                           79. Rhodius VA, Suh WC, Nonaka G, West J, Gross CA. Conserved variable functions the <br />                                               sigmaE stress response related genomes. PLoS Biol. 2006; 4:e2. [PubMed: 16336047] <br />                                           80. Tam C, Missiakas D. Changes lipopolysaccharide structure induce sigma(E)-dependent <br />                                               response Escherichia coli. Mol Microbiol. 2005; 55:1403&#226;&#8364;&#8220;1412. [PubMed: 15720549] <br />                                           81. Klein G, Lindner B, Brabetz W, Brade H, Raina S. Escherichia coli K-12 Suppressor-free Mutants <br />                                               Lacking Early Glycosyltransferases Late Acyltransferases: minimal lipopolysaccharide <br />                                               structure induction envelope stress response. J Biol Chem. 2009; 284:15369&#226;&#8364;&#8220;15389. <br />                                               [PubMed: 19346244] <br />                                           82. Chaba R, Alba BM, Guo MS, Sohn J, Ahuja N, Sauer RT, Gross CA. Signal integration DegS <br />                                               RseB governs &#207;&#402;E- mediated envelope stress response Escherichia coli. Proc Natl Acad <br />                                               Sci U S A. 2011; 108:2106&#226;&#8364;&#8220;2111. [PubMed: 21245315] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           83. Lima S, Guo MS, Chaba R, Gross CA, Sauer RT. Dual molecular signals mediate bacterial <br />                                               response outer-membrane stress. Science. 2013; 340:837&#226;&#8364;&#8220;841. [PubMed: 23687042] <br />                                               Established off-pathway intermediates LPS acts separate activating signal initiate the <br />                                               &#207;&#402;E envelope stress response. <br />                                           84. Prouty AM, Brodsky IE, Falkow S, Gunn JS. Bile-salt-mediated induction antimicrobial and <br />                                               bile resistance Salmonella typhimurium. Microbiology. 2004; 150:775&#226;&#8364;&#8220;783. [PubMed: <br />                                               15073288] <br />                                           85. Lin L, Tan B, Pantapalangkoor P, Ho T, Baquir B, Tomaras A, Montgomery JI, Reilly U, Barbacci <br />                                               EG, Hujer K, et al. Inhibition LpxC protects mice resistant Acinetobacter baumannii by <br />                                               modulating inflammation enhancing phagocytosis. MBio. 2012; 3 <br />                                           86. Li J, Nation RL, Owen RJ, Wong S, Spelman D, Franklin <br /> </body></html>