<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Advantages Limitations Different p62-Based <br /> Assays Estimating Autophagic Activity Drosophila <br /> Karolina Pircs, Peter Nagy, Agnes Varga, Zsolt Venkei, Balazs Erdi, Krisztina Hegedus, Gabor Juhasz* <br /> Department Anatomy, Cell Developmental Biology, Eotvos Lorand University, Budapest, Hungary <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Levels selective autophagy substrate p62 established recent years specific readout basal <br />     autophagic activity. Here compared different experimental approaches using assay Drosophila larvae. Similar <br />     commonly used western blots, quantifying p62 dots immunostained fat body cells L3 stage larvae <br />     detected strong accumulation endogenous p62 aggregates <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> Atg genes S6K. Importantly, genes <br />     <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span> silencing results early stage lethality analyzed microscopy using clonal analysis. The <br />     loss numerous general housekeeping genes <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> large-scale screens including autophagy, p62 <br />     assay potentially suitable distinguishing bona fide autophagy regulators silencing DNA polymerase <br />     subunit ribosomal gene likely non-specific effect autophagy. p62 accumulation RNAi silencing of <br />     known autophagy regulators dependent duration knockdown effect, unlike case starvation- <br />     induced autophagy. The endogenous p62 assay sensitive constitutively overexpressed p62-GFP reporter, <br />     showed self-aggregation large-scale accumulation control cells. We recommend western blots for <br />     following conversion overexpressed p62-GFP reporters estimate autophagic activity sample collection <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />     mutant larvae adults possible. In addition, showed overexpressed p62 Atg8 reporters strongly <br />     influence phenotypes other, potentially giving rise false contradicting results. Overexpressed p62 <br />     aggregates incorporated Atg8 reporter molecules lead wrong conclusion strongly enhanced <br />     autophagy, expression Atg8 reporter transgene rescued inhibitory effect dominant-negative Atg4 <br />     mutant basal starvation-induced autophagy. <br />  <br />   Citation: Pircs K, Nagy P, Varga A, Venkei Z, Erdi B, et al. (2012) Advantages Limitations Different p62-Based Assays Estimating Autophagic Activity in <br />   Drosophila. PLoS ONE 7(8): e44214. doi:10.1371/journal.pone.0044214 <br />   Editor: Mel B. Feany, Brigham Women&#226;&#8364;&#8482;s Hospital, Harvard Medical School, United States America <br />   Received May 17, 2012; Accepted July 30, 2012; Published August 31, 2012 <br />   Copyright: &#195;&#376; 2012 Pircs et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Wellcome Trust (087518/Z/08/Z), Hungarian Academy Sciences (BO/00552/11), European Union the <br />   European Social Fund contributed purchase electron microscope (TA&#194;&#180;MOP 4.2.1./B-09/1/KMR-2010&#226;&#8364;&#8220;0003). The funders role study design, <br />   data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: szmrt@elte.hu <br />  <br />  <br />  <br /> Introduction                                                                           thought inversely correlate dysregulation basal <br />                                                                                        autophagy. [5,6]. <br />    Human p62/SQSTM1 (sequestosome-1) multidomain                                     Different p62-based assays implemented recent <br /> scaffold protein involved various signaling pathways regulating                     years various experimental systems, including immunostaining <br /> a number processes including apoptosis, stress responses,                       cultured cells tissue samples, western blots, GFP-tagged <br /> cell growth. [1] Its single Drosophila homolog known                        reporters. [7] Drosophila melanogaster key vivo model <br /> Ref(2)P (refractory sigma P), based implicated roles                      organism, traditionally used tool discovery genes and <br /> sigma rhabdovirus multiplication. [2] For simplicity,                     genetic interactions. Previous reports shown the <br /> refer Drosophila gene p62. Its encoded protein product                       progressive formation p62/ubiquitin aggregates adult brains, <br /> shows similar domain structure human p62, containing                         greatly enhanced rates genetic inhibition autophagy <br /> an N-terminal PB1 domain required self-oligomerization                         fly models neurodegenerative disorders. [3,4,8,9] Here <br /> binding PB1-domain proteins, ZZ-type zinc finger                            set test different p62-based assays fat body or <br /> domain, LIR (LC3-interacting region) required in-                           larvae quantitative manner, comparing results and <br /> teraction Atg8/LC3 family members, C-terminal                               relative efficiency various experimental approaches. Our <br /> ubiquitin-binding UBA (ubiquitin-associated) domain. [3] The                           work important implications model systems well: <br /> Atg8/LC3 interaction enables selective degradation p62                           I. statistical analysis p62-positive aggregates in <br /> autophagy, acting specific adaptor protein                         immunostained cells tissues similarly effective western <br /> ensures targeting ubiquitinated proteins lysosomal                          blots estimation basal autophagy levels, II. immunos- <br /> degradation. Numerous human degenerative disorders ac-                             taining mosaic fat bodies allows testing specific role of <br /> companied formation cytoplasmic aggregates containing                        genes lethal phenotypes basal autophagy, III. endogenous <br /> p62 ubiquitinated proteins. [4] These abnormal aggregates                          p62 provides sensitive measure autophagy levels <br /> also form response impaired autophagy, p62 levels                         constitutively overexpressed GFP-tagged reporter in <br />                                                                                        microscopy, IV. duration efficient RNAi knockdown is <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         1                                August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                           p62 Assays Drosophila <br />  <br />  <br /> very important p62 assay unlike case induced             Figure S2). Silencing Atg2, Atg9 Atg8a promoted <br /> autophagy, V. p62 Atg8 reporters strongly interact            p62 dot formation. Enhancing TOR signaling overexpression <br /> each other, &#226;&#8364;&#8482;&#226;&#8364;&#8482;autophagy phenotypes&#226;&#8364;&#8482;&#226;&#8364;&#8482;.                            activator Rheb knockdown Rheb inhibitor Tsc2 also <br />                                                                            increased p62 dot number, case Rheb the <br /> Results                                                                    phenotypes variable change statistically <br />                                                                            significant (Figure 3b, d; Figure S2). Silencing Atg7 <br />    Given importance p62 assay characterizing                 did lead increased p62 dots, consistent observation <br /> autophagy phenotypes selected mutants RNAi treatments,              line weaker effect starvation-induced autophagy <br /> we generated polyclonal antibodies endogenous                 relative lines used, potentially caused less <br /> Drosophila p62. The antibody recognized single prominent                 efficient knockdown kinetics Atg7 E1-like enzyme <br /> band 100 kDa western blots prepared form L3 stage              reaction, is, lowered levels Atg7 RNAi being <br /> larvae adult heads (Figure 1a). The intensity band          potentially sufficient support basal autophagy (data shown). <br /> reduced 10-fold relative tubulin levels silencing           Numerous genes required fundamental cellular processes <br /> of p62 cases ubiquitously expressed different RNAi              transcription translation. It surprising these <br /> transgenes L3 larvae. Null mutation Atg8a resulted                genes identified putative hits various genome-wide <br /> a substantial increase p62 levels larvae adult heads,         RNAi screens, effect likely indirect cases. <br /> respectively, similar Atg7 mutant heads. Expression p62-GFP          While knockdown of genes reduces cell size an <br /> in larval fat body gave rise slower mobility band        extent evaluation autophagy phenotypes practi- <br /> 130 kDa, corresponding tagged version protein               cally impossible, silencing genes coding RNA <br /> (Figure 1a). Interestingly, presence p62-GFP increased         polymerase II. complex member Atms ribosomal subunit <br /> the levels endogenous p62 3.3-fold organismal level               RpS8 strongly interfered starvation-induced mCherry-Atg8 <br /> (probably higher case fat body, tissue         Lysotracker puncta formation (Figure S3). On hand, <br /> the tagged protein expressed). This finding suggests              knockdowns did enhance p62 aggregation expected <br /> overexpression p62 reporter strongly drives aggregate                 condition indirectly interferes autophagy <br /> formation, endogenous p62 captured                induction perturbing fundamental cellular process <br /> stabilized structures.                                            and/or expression transgenic marker (Figure 3c, d; see <br />    Next analyzed distribution p62 larval fat body cells       Figure S2), suggesting p62 assay potentially suitable <br /> by immunostaining whole-mount tissues. We endog-                distinguishing specific regulators autophagy (&#226;&#8364;&#732;&#226;&#8364;&#732;hits&#226;&#8364;&#8482;&#226;&#8364;&#8482;) from <br /> enous p62 formed numerous small cytoplasmic dots cells,           indirect effects (&#226;&#8364;&#732;&#226;&#8364;&#732;noise&#226;&#8364;&#8482;&#226;&#8364;&#8482;). <br /> and RNAi silencing p62 using different transgenes                A GFP-tagged p62 reporter previously used successfully in <br /> strongly significantly reduced number dots                Drosophila detect basal autophagy defects Atg13 mutant <br /> (Figure 1b&#226;&#8364;&#8220;d, Figure 1f statistics). As expected,                  cells. [12] We decided investigate phenotypes obtained by <br /> expression p62-GFP fat body cell clones strongly promoted            p62-GFP compare based endogenous protein. <br /> aggregate formation, resulting 9-fold increase number          Expression p62-GFP RNAi Atg1, Atg7, <br /> and 2.8-fold increase size p62-positive structures,              Atg8a, Atg9, Atg14, Atg18a, Tsc2, Pten dominant- <br /> respectively (Figure 1e, f).                                               negative Vps34 decrease number GFP- <br />    Since defective basal autophagy known enhance p62                 positive aggregates (Figure 3e&#226;&#8364;&#8220;g; Figure S4). In case of <br /> aggregation, analyzed previously described null mutants             Pten Tsc2 size GFP-positive aggregates increased <br /> selected genes involved autophagy. Both number size          significantly, did change cases. Importantly, <br /> p62-positive dots significantly increased fat bodies               number aggregates decreased size <br /> dissected L3 stage larvae mutant following core               increased case RpS8 Atms. As clear conclusions <br /> autophagy genes: Atg1 Atg13 (encoding members Atg1              drawn based results, looked exposure <br /> kinase complex), Vps34 (encoding phosphatidyl-inositol 3-              times used image acquisition. With approach could <br /> kinase), Atg2 Atg18a (encoding binding partners                 detect significantly increased GFP signal Atg1, Atg8a, Atg9, <br /> transmembrane protein Atg9), Atg7 Atg8a (encoding E1-              Atg14, Atg18a, Pten RNAi Vps34KD expressing cells, not <br /> like enzyme ubiquitin-like proteins                  case Atg7, Tsc2, RpS8 Atms RNAi cells (Figure 3e&#226;&#8364;&#8220; <br /> autophagy-specific protein conjugation systems) (Figure 2a, b,             h; Figure S4). These results agreed data <br /> d, Figure S1 additional images). S6 kinase                 obtained endogenous p62 antibody differing the <br /> a physiologic substrate TOR kinase (Target Of Rapamycin),               case Tsc2, several-fold difference each <br /> a central regulator autophagy cell growth. Activation            positively scoring line favor endogenous p62 assay. <br /> TOR suppresses autophagy results phosphorylation                    While testing large number RNAi lines genome-wide <br /> S6K. Based correlations, S6K long considered            RNAi screen (<span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> described elsewhere), striking observation <br /> inhibitor autophagy. Therefore, finding Drosophila             p62 assay occasionally showed large differences in <br /> S6K actually required starvation-induced autophagy             case different RNAi lines targeting gene. For <br /> surprising. [10,11] In line observations,          example, different transgenic RNAi lines Atg18a had <br /> found loss S6K significantly increased number (but          quantitatively different effects p62 aggregation, potentially <br /> this case size) p62 aggregates larval fat body cells,        reflecting differences knockdown efficiencies. Interestingly, one <br /> consistent suggested positive role autophagy (Figure 2c,       different RNAi lines Atg16 (encoding protein <br /> compare Figure 2a; Figure 2d quantification).              required Atg8a lipidation) failed enhance p62 aggregation, <br />    We evaluated inactivation known regulators               similar case PTEN, inactivation strongly <br /> autophagy somatic clones mosaic animals affected p62 levels.         enhances insulin signaling promote cell growth suppress <br /> RNAi silencing Atg1 Atg14 (the autophagy-specific subunit            autophagy (Figure 3d; Figure S2). As RNAi lines for <br /> of Vps34 complex) enhanced p62 aggregation, similar                 Atg18a, Atg16 PTEN effectively blocked starvation- <br /> overexpression dominant-negative Vps34 (Figure 3a, d;               induced autophagy L3 larvae, decided look potential <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             2                            August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                               p62 Assays Drosophila <br />  <br />  <br />  <br />  <br /> Figure 1. p62 levels Atg mutants, p62 RNAi p62 overexpression cells. A. Western blot analysis shows p62 accumulation Atg8a and <br /> Atg7 mutant heads Atg8a mutant larvae. RNAi knockdowns p62 (i1, i2) greatly decrease endogenous protein levels, overexpression of <br /> p62-GFP increases endogenous p62 levels addition appearance 130 kDa extra band corresponding tagged protein. Numbers <br /> refer p62 protein level relative tubulin loading control sample. B-D. RNAi knockdown p62 fat body cell clones (marked by <br /> expression Lamp-GFP) strongly decreases p62 puncta formation. E. Expression p62-GFP cell clones increases aggregate formation. Arrowhead <br /> in E&#226;&#8364;&#8482; indicates large aggregate p62-GFP expressing cell, arrow marks endogenous p62 dot control cell. F. Statistical evaluation the <br /> number size p62 dots samples panels B&#226;&#8364;&#8220;E. ** indicates significant difference (p,0.01), based two-tailed two-sample unequal <br /> Student&#226;&#8364;&#8482;s t tests. Scalebar panel B equals 30 mm panels B&#226;&#8364;&#8220;E. Genotypes are: (A) lane 1: w [1118], lane 2: Atg7[d77]/Atg7[d14], lane 3: Atg8a[d4], <br /> lane 4: UbiGal4/+; p62[HMS00551]/+, lane 5: UbiGal4/+; p62[HMS00938]/+, lane 6: w [1118], lane 7: Atg8a[d4], lane 8: cgGal4/UAS-p62-GFP; (B) hs-Flp; <br /> UAS-LampGFP/p62[KK108193]; Act.CD2.Gal4, UAS-Dcr2/+; (C) hs-Flp; UAS-LampGFP/+; Act.CD2.Gal4, UAS-Dcr2/p62[HMS00551]; (D) hs-Flp; UAS- <br /> LampGFP/+; Act.CD2.Gal4, UAS-Dcr2/p62[HMS00938]; (E) hsFlp; UAS-p62-GFP/+; Act.CD2.Gal4,UAS-Dcr2/+. <br /> doi:10.1371/journal.pone.0044214.g001 <br />  <br /> reasons discrepancies. Quantification mCherry-Atg8a                case PTEN RNAi lines (Figure 4k&#226;&#8364;&#8220;o), showed strong <br /> dots starved fat bodies revealed Atg18aJF showed better                suppression mCherry-Atg8a dot formation L3 stage, while <br /> suppression Atg18aKK L3 L2 stage animals, likely              PTENKK did significantly inhibit starvation-induced autop- <br /> explaining quantitatively different p62 results (Figure 4f&#226;&#8364;&#8220;j). In          hagy L2 animals, consistent failure enhance p62 dot <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 3                            August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                              p62 Assays Drosophila <br />  <br />  <br />  <br />  <br /> Figure 2. Endogenous p62 accumulates Atg S6K mutants. A. p62 immunostaining detects small dots scattered the <br /> cytoplasm fat body cells wild-type larvae. B. p62 aggregates appear bigger numerous Atg1 null mutants, loss S6K (C) only <br /> increases number, size p62 aggregates. D. Statistical evaluation p62 puncta fat bodies various Atg mutants S6K null <br /> animals. * indicates significant difference (p,0.05), ** indicates significant difference (p,0.01), based two-tailed two-sample unequal <br /> Student&#226;&#8364;&#8482;s t tests. Scalebar panel A equals 30 mm panels A&#226;&#8364;&#8220;C. Genotypes are: (A) w [1118]; (B) Atg1 [25]; (C) S6K[l&#226;&#8364;&#8220;1]. <br /> doi:10.1371/journal.pone.0044214.g002 <br />  <br /> formation. Interestingly, differences seen starved L3          readily detected 4-hour starvation wandering stage <br /> stage animals RNAi lines case Atg16.                    animals undergoing developmental autophagy. Expression of <br /> However, Atg16HMS reduced size dots 8-fold,                      Atg1 RNAi dominant-negative Vps34 strongly decreased <br /> Atg16KK showed two-fold reduction L2 stage                          starvation-induced conversion p62-GFP, Atg1 silencing <br /> (Figure 4a&#226;&#8364;&#8220;e). Larger mCherry-Atg8a dots thought                       showed weaker effect wandering animals (Figure 5d). <br /> represent mature autolysosomes, Atg16HMS stronger                      Atg4 encodes cysteine protease required cleavage of <br /> block formation structures L2 stage larvae.                    C-terminal amino acids Atg8a, necessary its <br /> Lysotracker stainings supported hypothesis,              subsequent activation conjugation membrane lipid <br /> Atg16HMS reduced size Lysotracker puncta statistically                 phosphatidyl-ethanolamine. Atg8a needs delipidated after <br /> significantly L2 stage animals (Figure S5). Altogether,              successful completion autophagosome allow lysosomal <br /> results suggest differences seen starvation-induced autop-            fusion. A mutant form human Atg4B substitution of <br /> hagy L2 stage progressive formation p62                     cysteine 74 alanine shown act dominant-negative <br /> aggregates (analyzed L3 stage) reflect                    fashion, inhibiting proper formation clearence autopha- <br /> knockdown efficiency given RNAi line earlier stages (that           gosomes. [13] We reconstituted mutant Drosophila, <br /> is, L2).                                                                      creating transgenic flies inducible expression Atg4aC98A. <br />    Finally, evaluated impact overexpressed p62                 Expression dominant-negative Atg4a strongly suppressed <br /> and Atg8a reporters (which potentially create gain function              Lysotracker staining fat body cells starved larvae, and <br /> condition) autophagy phenotypes, possibility genetic             caused accumulation p62 (Figure 5e, Figure 3d; see <br /> interactions excluded. In fact, p62-GFP                 Figure S2). As expression mCherry-Atg8a Atg8a null <br /> co-expressed mCherry-Atg8a, large colocalizing GFP-                  mutants rescue p62 accumulation phenotype (Figure <br /> mCherry-positive aggregates formed well-fed                      S6a, b), tested overexpressed mCherry-Atg8a can <br /> animals (Figure 5a). These aggregates likely caused                   rescue effect dominant-negative Atg4. Co-expression <br /> the specific interaction overexpressed proteins                mCherry-Atg8a dominant-negative Atg4 restored both <br /> represented artefact bona fide autophagic structures,              starvation-induced autophagy starved animals normal <br /> as Atg8a strongly binds LC3-interacting region p62. In              turnover p62 (Figure 5f; Figure S6c, d). <br /> line that, did detect obvious increase                      Ultrastructural analysis confirmed expression of <br /> autophagy p62-GFP expressing cells Lysotracker staining                 dominant-negative Atg4 strongly inhibited starvation-induced <br /> and transmission electron microscopy (Figure 5b, c). Similarly,               autophagy: autolysosomes seen cross-sections fat <br /> we saw conversion p62-GFP free GFP western blots                  body cells, small autophagosome-like structures were <br /> of larvae coexpressing mCherry-Atg8a p62-GFP fat                   occasionally observed (Figure 5g). Coexpression Atg8a with <br /> body, suggesting mCherry-Atg8a expression does                       dominant-negative Atg4 restored starvation-induced autophagy, <br /> obviously enhance autophagic degradation p62-GFP                        numerous autolysosomes autophagosomes seen in <br /> well-fed L3 stage control animals. As expected, free GFP                  cells (Figure 5h), similar previous findings mammalian <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                4                            August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                                p62 Assays Drosophila <br />  <br />  <br />  <br />  <br /> Figure 3. Loss known autophagy regulators enhances p62 puncta formation cell-autonomously. Knockdown Atg1 (A) Tsc2 (B) <br /> increases p62 aggregate formation, silencing RpS8 results slight reduction p62 dot number (C). Panel D shows statistical evaluation of <br /> the effect RNAi overexpression lines p62 accumulation. Overexpressed p62-GFP forms multiple aggregates control Atg1 RNAi cells <br /> (panels E F, respectively). Exposure times indicated right corner panels E F. Statistical evaluation p62-GFP aggregate size <br /> and number various RNAi overexpression lines reveals changes difficult interpret (G), p62-GFP levels inferred exposure <br /> times image acquisition qualitatively similar data obtained anti-p62 immunostaining (compare H D). * indicates significant <br /> difference (p,0.05), ** indicates significant difference (p,0.01), based two-tailed two-sample unequal Student&#226;&#8364;&#8482;s t tests panels D, G, H. <br /> Scalebar panel A equals 30 mm panels A&#226;&#8364;&#8220;C E&#226;&#8364;&#8220;F. Genotypes are: (A) hsFlp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/Atg1[JF02273]; (B) hsFlp; <br /> UAS-Dcr2/TSC2[KK103417]; Act.CD2.Gal4, UAS-GFPnls/+; (C) hsFlp; UAS-Dcr2/RpS8[KK106835]; Act.CD2.Gal4, UAS-GFPnls/+; (E) hsFlp; UAS-p62-GFP/ <br /> +; Act.CD2.Gal4, UAS-Dcr2/+ (F) hsFlp; UAS-p62-GFP/+; Act.CD2.Gal4, UAS-Dcr2/Atg1[JF02273]. <br /> doi:10.1371/journal.pone.0044214.g003 <br />  <br /> cells. [13] Similarly, knockdown Atg4a using RNAi line                  hypothesis partial loss Atg4a function rescued <br /> Atg4aKK blocked starvation-induced Lysotracker staining                    overexpression Atg8a. <br /> induced p62 accumulation fat body cells failed inhibit <br /> mCherry-Atg8a dot formation (not shown), supporting <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 5                             August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                     p62 Assays Drosophila <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   6   August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                                   p62 Assays Drosophila <br />  <br />  <br />  <br /> Figure 4. The effect Atg16, Atg18a Pten RNAi lines starvation-induced autophagy L3 L2 larval stages. Both RNAi <br /> lines Atg16 similar block mCherry-Atg8a puncta formation L3 (compare A C), size dots reduced more <br /> efficiently Atg16HMS L2 (compare B D; panel E statistics). Both RNAi lines Atg18a strongly inhibit mCherry-Atg8a dot formation <br /> in L3 (compare F H), Atg18aJF shows complete block L2, reducing size number puncta (compare G I; also <br /> panel J statistics). Both RNAi lines Pten strongly inhibit mCherry-Atg8a dot formation L3 (compare K M), PtenJF shows more <br /> complete block L2, reducing size number puncta (compare L N; panel O statistics). * indicates significant <br /> difference (p,0.05), ** indicates significant difference (p,0.01), based two-tailed two-sample unequal Student&#226;&#8364;&#8482;st tests panels E, J, O. <br /> Scalebar panel A equals 30 mm panels A, C, F, H, K, M, scalebar panel B equals 30 mm panels B, D, G, I, L, N. Genotypes are: (A, B) hsFlp; <br /> UAS-Dcr2/Atg16[KK105993]; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/+; (C, D) hsFlp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/ <br /> Atg16[HMS01347]; (F, G) hsFlp; UAS-Dcr2/Atg18a[KK105366]; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/+; (H, I) hsFlp; UAS-Dcr2/+; Act.CD2.Gal4, <br /> UAS-GFPnls, r4-mCherry-Atg8a/Atg18a[JF02898]; (K, L) hsFlp; UAS-Dcr2/Pten[KK101475]; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/+; (M, N) hsFlp; <br /> UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/Pten[JF01987]. <br /> doi:10.1371/journal.pone.0044214.g004 <br />  <br />  <br /> Discussion                                                                       autophagy onset efficient gene knockdown delayed <br />                                                                                  larval Drosophila development. Based observa- <br />    Determining levels specific autophagy substrate p62             tions, perdurance maternally supplied gene products during <br /> becoming standard assay estimate basal autophagy rates. [7]                 development account differences seen in <br /> Western blots widely used experimental setup                     p62 levels different Atg mutants. <br /> detect endogenous p62 levels possibility distinguish                    Additionally, certain problems arise using over- <br /> soluble aggregated forms using differential detergent extrac-                expressed reporter transgenes autophagy. First, overexpressed <br /> tions, experiments simply feasible studies             Atg8a reporters captured overexpressed p62 <br /> involve genetic mosaic animals. Here presented evidence                  aggregates, giving false impression increased autophagy. <br /> an immunostaining approach similarly effective western blots               Second, overexpression Atg8a reporter genetically <br /> to detect subtle changes p62 aggregation larval fat               rescue partial loss gene seen case Atg4a, again <br /> body, actually provide additional information compared                complicating interpretation potentially contradicting results. <br /> with western blots, number, size intracellular <br />                                                                                  While examples shown rare, seen several <br /> distribution aggregates detected way. We <br />                                                                                  instances inactivation certain genes led high <br /> recommend using p62-specific antibody constitu- <br />                                                                                  accumulation endogenous p62 aggregates sequestered <br /> tively overexpressed GFP-tagged reporter led to <br />                                                                                  Atg8a reporter molecules, overexpressed Atg8a reporters also <br /> extensive self-aggregation increased level the <br />                                                                                  genetically rescue effect number RNAi lines <br /> endogenous protein, size number p62 <br />                                                                                  autophagy. <br /> aggregates longer indicative autophagic activity. Still, <br />                                                                                     Finally, experiments involved fat body cells of <br /> comparing GFP signal intensities yield qualitatively similar <br />                                                                                  Drosophila larvae, convinced results and <br /> information data obtained looking endogenous p62 levels <br />                                                                                  limitations presented applicable cells and <br /> in cases. Endogenous p62 protein levels also <br /> be influenced changes unrelated autophagy,                    organisms used autophagy research. <br /> not problem tagged reporter expressed an <br /> artificial promoter. In addition comparing fluorescence in-                   Methods <br /> tensities p62-GFP expressing cell clones, probably more <br />                                                                                  Fly Husbandry <br /> reliable approach follow generation free GFP produced <br />                                                                                     Drosophila stocks maintained standard cornmeal/ <br /> by autolysosomal degradation western blots larvae expressing <br />                                                                                  sugar/agar medium 25uC 50% humidity 12 h light/ <br /> p62-GFP tissue-specific promoter. Alternatively, p62-GFP <br /> expression induced heat shock mutant animals                  12 h dark cycle. Thestocks used study were: UAS-p62-GFP, <br /> the decay tagged protein followed western blots                     Atg13[D81], [12] p62[KK108193], Atg8a[KK109654], At- <br /> microscopy measure tagged p62 half-life pulse-chase assay.               g18a[KK105366],         Atg14[KK108559],         Atg16[KK105993], <br /> This efficient method determine flux basal                   TSC2[KK103417],          Pten[KK101475],         RpS8[KK106835], <br /> starvation-induced autophagy cultured cells. [7] Regarding                    Atms[GD20876], [15] Atg7[d77]/Atg7[d14], [8] Vps34[Dm22], <br /> endogenous p62 expression, transcriptionally                 UAS-Vps34[KD],      [16]    p62[HMS00551],        p62[HMS00938], <br /> upregulated 4.8-fold larval fat body cells response 4-hour            Atg1[JF02273], Atg18a[JF02898], Atg9[JF02891], Atg7[JF02787], <br /> starvation, [14] similar starvation-induced upregulation                Atg16[HMS01347], Atg2[JF02786], Pten[JF01987], [17] At- <br /> also obvious western blots. Interestingly, starvation mainly                  g2[EP3697]/Df(3L)BSC119], Atg18a[KG03090]/Df(3L)Exel6112], <br /> increases soluble p62 levels decreases the                     S6K[l-1], UAS-Rheb[EP50.084cre(w-)]/TM6B, w [1118], [11] Atg1 <br /> aggregated fraction. [4] This observation suggests                     [25]. [18]. The Atg8a[d4] mutant used study generated <br /> evaluation endogenous p62 aggregates reliable measure                 imprecise excision P element KG07569 harbors <br /> autophagic flux.                                                                 deletion removing 25 codons Atg8a 53 <br />    In case autophagy induced certain stimulus                    codons CG1826, gene unknown function (kindly <br /> starvation, levels proteins required autophagy time             provided Tom Neufeld). The mutants homozygous viable <br /> the stimulus response critical determinants                fertile gross morphological developmental <br /> the magnitude autophagy follows. In contrast, p62 levels                 abnormalities. UAS-Atg4[C98A] transgenics established as <br /> are strongly influenced basal autophagic activity,                    described below. <br /> formation excess p62 aggregates progressive, time- <br /> dependent process. Therefore, duration certain treatment                 Molecular Cloning Embryo Injections <br /> effective gene silencing RNAi major impact type                    The full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> coding sequence Atg4a/CG4428 with <br /> of assay. In line that, shown p62 levels              Cysteine 98 Alanine mutation changing coding triplet <br /> remain normal despite strong block starvation-induced                       TGC GCC chemically synthesized (Genscript), and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   7                             August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                     p62 Assays Drosophila <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   8   August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                               p62 Assays Drosophila <br />  <br />  <br />  <br /> Figure 5. The effect p62 Atg8a reporters autophagy phenotypes. A. Coexpression p62-GFP mCherry-Atg8a results in <br /> formation aggregates containing tagged proteins fed animals, potentially suggesting enhanced autophagy. B. Expression p62-GFP fails <br /> to induce Lysotracker puncta fed animals. C. p62-GFP expression leads appearance protein aggregates ultrastructural images fat <br /> body cells (asterisks), autophagic structures seen cytoplasm. Inset shows enlarged aggregate illustrate inclusion <br /> bodies membrane-bound. D. Western blot analysis using anti-GFP antibodies larval extracts expressing mCherry-Atg8a p62-GFP the <br /> fat body. No generation free GFP autolysosomal degradation seen well-fed control larvae, induced 4-hour starvation the <br /> wandering stage. Silencing Atg1 expression dominant-negative Vps34 inhibits autophagy-mediated conversion p62-GFP free GFP. E, F. <br /> Expression dominant-negative Atg4 blocks starvation-induced Lysotracker puncta formation, effect overexpressed mCherry-Atg8a <br /> dots. G. Ultrastructural analysis starved fat bodies reveal small autophagosomes (marked arrowheads G&#226;&#8364;&#8482; G&#226;&#8364;&#8482;), autolysosomes are <br /> seen. Coexpression GFP-Atg8a dominant-negative Atg4 rescues inhibition: numerous autolysosomes (arrow) autophagosomes <br /> (arrowheads) seen H, H&#226;&#8364;&#8482;. Scalebar panel A equals 30 mm panels A, B, E, F scalebars panels C, G, H equal 2 mm. Genotypes are: (A) <br /> hsFlp; UAS-p62-GFP/+; Act.CD2.Gal4, UAS-Dcr2/r4-mCherry-Atg8a; (B, C) hsFlp; UAS-p62-GFP/+; Act.CD2.Gal4, UAS-Dcr2/+; (D) lanes 1, 2, 3: cgGal4, <br /> UAS-p62-GFP/+; Atg1[JF02273]/+, lanes 4, 5, : cgGal4, UAS-Vps34[KD]/UAS-p62-GFP, lanes 6, 7, 8, : cgGal4, UAS-p62-GFP/+; (E, G) hs-Flp; UAS-Dcr2/+; <br /> Act.CD2.Gal4, UAS-GFPnls/UAS-Atg4[C98A]; (F, H) hs-Flp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/UAS-Atg4[C98A]. <br /> doi:10.1371/journal.pone.0044214.g005 <br />  <br /> cloned pUAST using EcoRI XhoI restriction sites.                      Zeiss Axioimager M2 microscope equipped Apotome2 <br /> Drosophila embryo transformation carried according                  grid confocal <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, Plan-NeoFluar 4060.75 NA objective, <br /> standard methods (Bestgene).                                                   Axiocam Mrm camera, Axiovision software using MinMax <br />                                                                                setting automatically adjusting image levels. Lysotracker <br /> Polyclonal Anti-p62 Anti-GFP Antibodies                                stainings photographed widefield mode, single optical <br /> Western Blots                                                                  sections shown colocalisations mCherry-Atg8a assays. <br />    The polyclonal affinity-purified p62 antibody raised                 For p62 stainings, 3 subsequent optical sections taken 0.55 mm <br /> rabbits using peptide antigene PRTEDPVTTPRSTQ corre-                       intervals projected single plane using Maximum <br /> sponding amino acids 297&#226;&#8364;&#8220;311 (Genscript). Polyclonal anti-                  Intensity Projection. <br /> GFP antibodies raised using standard procedures by <br /> immunizing rats bacterially expressed His-tagged eGFP                     Statistical Analysis <br /> purified Ni affinity columns (Sigma). Samples separated                   For western blots, image scanned film inverted in <br /> by SDS-PAGE 8% acrylamide gel transferred                         Adobe Photoshop, saved loaded ImageJ. The strongest <br /> Immobilon-P PVDF membrane (Millipore). Membranes                          band selected first, individual bands measured <br /> blocked 3% milk/TBS 1 h room temperature washed                  (Analyze, Set measurements, Integrated density, Measure) using <br /> three times 10 min TBST (TBS +0.1% Tween-20).                      selection area width height strongest <br /> Blots incubated primary antibodies: rabbit polyclonal                band. Mean p62 values normalized tubulin mean scores <br /> anti-p62 [1:8,000], mouse monoclonal anti-tubulin AA4.3 (DSHB)                 first, expression levels calculated relative the <br /> [1:200], rat polyclonal anti-GFP [1:10,000] 1.5% milk/TBST                  control sample (wt head, lane 1). For immunostainings, images <br /> for 1 h room temperature, followed 10-min washes                loaded ImageJ, image type changed from <br /> TBST. Blots incubated AP-conjugated goat anti-rat                      RGB Color RGB Stack. In case mutants, 3006300 dpi <br /> (Sigma), anti-rabbit anti-mouse secondary antibody (Millipore)              area chosen randomly red channel, threshold <br /> diluted 1:10,000 1.5% milk/TBST 1 h room tempera-                    adjusted select dots. Each selected area analyzed <br /> ture. Blots washed 3610 min TBST                          (Analyze Particles, Show, Masks) count average size <br /> incubated Immobilon Western Chemiluminescent AP Sub-                      (in pixel2) noted summary masked images. At <br /> strate (Millipore), followed exposure Super RX film (Fuji).              10 images evaluated 4&#226;&#8364;&#8220;6 animals genotype. <br />                                                                                The number/size data summarized Excel, normalized <br /> Histology Imaging                                                          control (average dot number size control animals set <br />    Clonal analysis using spontaneously activated Gal4/UAS                  1). For clonal analysis, image type changed from <br /> system larval fat body carried described                     RGB Color RGB Stack. Then threshold adjusted for <br /> previously. [11,12,16,19] Bisected instar larvae inverted           adequte channel, clone cells neighbouring control <br /> and fixed 3.7% paraformaldehyde PBS overnight 4uC.                  cells evaluated. First, randomly selected GFP-positive or <br /> Next, samples rinsed twice washed 2 hours PBS,                 control cells manually encircled Freehand selection <br /> permeabilized 15 minutes PBTX-DOC (PBS 0.1%                        tool, selected area analyzed channel of <br /> Triton X-100 0.05% sodium deoxycholate) blocked                    (Analyze Particles, Show, Masks), count and <br /> 3 h 3% goat serum PBTX-DOC. Samples                            average size (in pixel2) noted summary masked <br /> incubated overnight 4uC primary antibodies rabbit                      images. At 10 images 4&#226;&#8364;&#8220;6 animals evaluated for <br /> polyclonal anti-p62 [1:2,000] mouse monoclonal anti-GFP                    genotype. These number/size data processed the <br /> [1:1,500] (Invitrogen) 1% goat serum PBTX-DOC. After                     case mutants, GFP-positive cells compared to <br /> 3630 minutes washes PBTX-DOC, samples incubated                        controls cells image, data normalized to <br /> with secondary antibodies goat anti-mouse Alexa 488 goat                   cell size. Two-tailed two-sample unequal Student&#226;&#8364;&#8482;s t test used <br /> anti-rabbit Alexa 568 [1:1,500] (Invitrogen) 1% goat serum               estimate p values cases. <br /> PBTX-DOC 4 hours room temperature. Finally, 3615 <br /> minutes washes PBTX-DOC 1615 minutes PBS, fat                        Transmission Electron Microscopy <br /> bodies dissected mounted 50% glycerol/PBS                       Larvae genotypes hs-Flp; UAS-p62-GFP/+; <br /> 0.2 mM DAPI. For p62 staining mCherry-Atg8a expressing cells                Act.CD2.Gal4, UAS-Dcr2/+ hs-Flp; UAS-Dcr2/+; <br /> Alexa 647-conjugated goat anti-rabbit antibody used avoid               Act.CD2.Gal4, UAS-GFPnls/UAS-Atg4[C98A] hs-Flp; <br /> detection signal mCherry. Lysotracker stainings              UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry- <br /> carried described previously. Images captured                   Atg8a/UAS-Atg4[C98A] heat shocked 1 hour 37uC <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 9                            August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                                                p62 Assays Drosophila <br />  <br />  <br /> to activate Act.CD2.Gal4 expression hs-Flp fat body                           expression dominant-negative Vps34 (F) p62-GFP <br /> cells. Larvae processed electron microscopy 24 hours later                     expressing cell clones. Exposure times indicated top <br /> as described previously. [16] Ultrathin sections prepared                          right corner image. Scalebar panel A equals 30 mm for <br /> from animals genotype images captured using                            images. Genotypes are: (A) hsFlp; UAS-p62-GFP/+; Act.CD2.- <br /> a JEOL JEM-1011 transmission electron microscope                                Gal4, UAS-Dcr2/Atg7[JF02787]; (B) hsFlp; UAS-p62-GFP/At- <br /> Olympus Morada 11 megapixel camera iTEM software                                    g8a[KK109654]; Act.CD2.Gal4, UAS-Dcr2/+; (C) hsFlp; UAS- <br /> (Olympus).                                                                              p62-GFP/+; Act.CD2.Gal4, UAS-Dcr2/Atg9[JF02891]; (D) hsFlp; <br />                                                                                         UAS-p62-GFP/Atg14[KK108559]; Act.CD2.Gal4, UAS-Dcr2/+; <br /> Supporting Information                                                                  (E) hsFlp; UAS-p62-GFP/Atg18a[KK105366]; Act.CD2.Gal4, <br />                                                                                         UAS-Dcr2/+;       (F)      hsFlp;    UAS-p62-GFP/UAS-Vps34[KD]; <br /> Figure S1 Endogenous p62 dots Atg mutants. Loss of <br />                                                                                         Act.CD2.Gal4, UAS-Dcr2/+; (G) hsFlp; UAS-p62-GFP/+; <br /> Atg2 (A), Atg7 (B), Atg8a (C), Atg13 (D), Atg18a (E) Vps34 (F) <br />                                                                                         Act.CD2.Gal4, UAS-Dcr2/Pten[JF01987]; (H) hsFlp; UAS-p62- <br /> all increase p62 aggregation. Scalebar panel A equals 30 mm for <br />                                                                                         GFP/TSC2[KK103417]; Act.CD2.Gal4, UAS-Dcr2/+; (I) hsFlp; <br /> all images. Genotypes are: (A) Atg2[EP3697]/Df(3L)BSC119]; (B) <br /> Atg7[d77]/Atg7[d14]; (C) Atg8a[d4]; (D) Atg13[D81]; (E) At-                             UAS-p62-GFP/RpS8[KK106835]; Act.CD2.Gal4, UAS-Dcr2/+; <br /> g18a[KG03090]/Df(3L)Exel6112]; (F) Vps34[Dm22].                                         (J) hsFlp; UAS-p62-GFP/Atms[GD20876]; Act.CD2.Gal4, UAS- <br /> (TIF)                                                                                   Dcr2/+. <br />                                                                                         (TIF) <br /> Figure S2 The effect additional RNAi over- <br /> expression lines p62 aggregation. The effect over-                                Figure S5 The effect Atg16 RNAi lines starvation- <br /> expressed dominant-negative Atg4 (B), dominant-negative Vps34                           induced punctate Lysotracker staining L3 L2 <br /> (G), wild-type Rheb (H), knockdown Atg2 (A), Atg7 (C),                           larval stages. Both RNAi lines Atg16 similar block of <br /> Atg8a (D), Atg9 (E), Atg14 (F), Atg16 (I, J), Atg18a (K, L), Pten (M,                   Lysotracker puncta formation L3 (panels A C), the <br /> N) Atms (O) p62 aggregation. Scalebar panel A equals                          size dots reduced significantly Atg16HMS L2 <br /> 40 mm images. Genotypes are: (A) hsFlp; UAS-Dcr2/+;                             (compare D B; panel E statistics). * indicates <br /> Act.CD2.Gal4, UAS-GFPnls/Atg2[JF02786]; (B) hsFlp; UAS-                                 significant difference (p,0.05), ** indicates significant <br /> Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/UAS-Atg4[C98A]; (C) hsFlp;                             difference (p,0.01), based two-tailed two-sample unequal <br /> UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/Atg7[JF02787]; (D)                                 Student&#226;&#8364;&#8482;s t tests. Scalebar panel A equals 30 mm panels A, C, <br /> hsFlp; UAS-Dcr2/Atg8a[KK109654]; Act.CD2.Gal4, UAS-GFPnls/                              scalebar panel B equals 30 mm panels B, D. Genotypes are: <br /> +; (E) hsFlp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/                                     (A,B) hs-Flp; UAS-LampGFP/Atg16[KK105993]; Act.CD2.Gal4, <br /> Atg9[JF02891];      (F)     hsFlp;    UAS-Dcr2/Atg14[KK108559];                         UAS-Dcr2; (C,D) hs-Flp; UAS-LampGFP/+; Act.CD2.Gal4, UAS- <br /> Act.CD2.Gal4, UAS-GFPnls/+; (G) hsFlp; UAS-Dcr2/UAS-                                    Dcr3/Atg16[HMS01347]. <br /> Vps34[KD]; Act.CD2.Gal4, UAS-GFPnls/+; (H) hsFlp; UAS-                                  (TIF) <br /> Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/UAS-Rheb[EP50.084cre(w- <br />                                                                                         Figure S6 Overexpression mCherry-Atg8a rescues <br /> )]; (I) hsFlp; UAS-Dcr2/Atg16[KK105993]; Act.CD2.Gal4, UAS- <br />                                                                                         effect Atg8a mutation expression dominant- <br /> GFPnls/+; (J) hsFlp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/ <br />                                                                                         negative Atg4 p62 accumulation. Expression mCherry- <br /> Atg16[HMS01347]; (K) hsFlp; UAS-Dcr2/Atg18a[KK105366]; <br />                                                                                         Atg8a reduces size number p62 aggregates Atg8a null <br /> Act.CD2.Gal4, UAS-GFPnls/+; (L) hsFlp; UAS-Dcr2/+; <br />                                                                                         mutants (A; panel B statistics). No accumulation p62 dots <br /> Act.CD2.Gal4, UAS-GFPnls/Atg18a[JF02898]; (M) hsFlp; UAS- <br />                                                                                         observed fat body cells coexpressing mCherry-Atg8a and <br /> Dcr2/Pten[KK101475]; Act.CD2.Gal4, UAS-GFPnls/+; (N) hsFlp; <br />                                                                                         dominant-negative Atg4 relative control cells (C; panel D for <br /> UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls/Pten[JF01987]; (O) <br /> hsFlp; UAS-Dcr2/Atms[GD20876]; Act.CD2.Gal4, UAS-GFPnls/+.                              statistics &#226;&#8364;&#8220; significant difference seen). Scalebar panel A <br /> (TIF)                                                                                   equals 30 mm. Genotypes are: (A)Atg8a[d4]; +/+; r4-mCherry-Atg8a/+; <br />                                                                                         (B) hs-Flp; UAS-Dcr2/+; Act.CD2.Gal4, UAS-GFPnls, r4-mCherry- <br /> Figure S3 Knockdown RpS8 Atms inhibits starva-                                    Atg8a/UAS-Atg4[C98A]. <br /> tion-induced mCherry-Atg8a Lysotracker puncta                                       (TIF) <br /> formation. Silencing RpS8 Atms strongly interferes with <br /> starvation-induced mCherry-Atg8a dot formation (panels A B,                         Acknowledgments <br /> respectively) blocks punctate Lysotracker staining (panels C <br /> and D, respectively). Scalebar panel A equals 30 mm                          We thank Sarolta Palfia excellent technical assistance, Tom Neufeld <br /> images. Genotypes are: (A) hs-Flp; UAS-Dcr2/RpS8[KK106835];                             stocks, sharing unpublished Atg8a mutant, helpful discussions and <br /> Act.CD2.Gal4, UAS-GFPnls, r4-mCherry-Atg8a/+; (B) hs-Flp;                               critical reading manuscript, stock centers BDSC, VDRC, and <br />                                                                                         TRiP Harvard Medical School providing fly strains used this <br /> UAS-Dcr2/Atms[GD20876]; Act.CD2.Gal4, UAS-GFPnls, r4- <br />                                                                                         study. <br /> mCherry-Atg8a/+; (C) hs-Flp; UAS-LampGFP/RpS8[KK106835]; <br /> Act.CD2.Gal4, UAS-Dcr2; (D) hs-Flp; UAS-LampGFP/ <br /> Atms[GD20876]; Act.CD2.Gal4, UAS-Dcr2.                                                  Author Contributions <br /> (TIF)                                                                                   Conceived designed experiments: KP KH GJ. Performed the <br />                                                                                         experiments: KP PN AV ZV BE KH GJ. Analyzed data: KP GJ. <br /> Figure S4 p62-GFP aggregation additional RNAi                                    Contributed reagents/materials/analysis tools: KP PN KH GJ. Wrote the <br /> overexpression lines. Knockdown Atg7 (A), Atg8a (B), Atg9                            paper: KP PN GJ. <br /> (C), Atg14 (D), Atg18a (E), Pten (G), Tsc2 (H), RpS8(I), Atms (J) <br />  <br /> References <br />  1. Moscat J, Diaz-Meco MT (2009) p62 crossroads autophagy, apoptosis,         2. Dezelee S, Bras F, Contamine D, Lopez-Ferber M, Segretain D, et al. (1989) <br />     cancer. Cell 137: 1001&#226;&#8364;&#8220;1004.                                                        Molecular analysis ref(2)P, Drosophila gene implicated sigma rhabdovirus <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         10                                August 2012 | Volume 7 | Issue 8 | e44214 <br />                                                                                                                                                      p62 Assays Drosophila <br />  <br />  <br />       multiplication necessary male fertility. The EMBO journal 8: 3437&#226;&#8364;&#8220;              11. Scott RC, Schuldiner O, Neufeld TP (2004) Role regulation starvation- <br />       3446.                                                                                       induced autophagy Drosophila fat body. Dev Cell 7: 167&#226;&#8364;&#8220;178. <br />  3.   Nezis IP, Simonsen A, Sagona AP, Finley K, Gaumer S, et al. (2008) Ref(2)P,         12. Chang YY, Neufeld TP (2009) An Atg1/Atg13 complex multiple roles in <br />       Drosophila melanogaster homologue mammalian p62, required                     TOR-mediated autophagy regulation. Molecular Biology Cell 20: 2004&#226;&#8364;&#8220; <br />       formation protein aggregates adult brain. J Cell Biol 180: 1 <br /> </body></html>