<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 19 No. 23 pp. 6440&#194;&#177;6452, 2000 <br />  <br />  <br /> LHS1 SIL1 provide lumenal function is <br /> essential protein translocation the <br /> endoplasmic reticulum <br />  <br />  <br /> John R.Tyson Colin J.Stirling1                                      precursor translocon (ii) promote the <br /> School Biological Sciences, 2.205 Stopford Building, University of <br />                                                                         vectorial transport polypeptide chains the <br /> Manchester, Oxford Road, Manchester M13 9PT, UK                         translocon ER lumen (Sanders et al., 1992; Lyman <br /> 1 <br />                                                                         Schekman, 1995, 1997). This role involves the <br />  Corresponding author <br /> e-mail: colin.stirling@man.ac.uk <br />                                                                         Hsp70 ATP-dependent reaction cycle, drives <br />                                                                         multiple rounds Kar2p binding translocating <br /> Lhs1p Hsp70-related chaperone localized                    polypeptide enters lumen. Such binding the <br /> endoplasmic reticulum (ER) lumen. Dlhs1 mutant cells                    incoming precursor lumenal factor prevents `back- <br /> are viable constitutively induced                       slip&apos; suf&#194;&#174;cient drive import (Matlack et al., <br /> unfolded protein response (UPR). Here, demon-                        1999). <br /> strate severe growth defect Dire1Dlhs1 double                         A second ER-resident Hsp70-related protein, encoded <br /> mutant cells UPR longer                          LHS1 gene, characterized yeast (Baxter <br /> induced. In addition, identi&#194;&#174;ed UPR-                          et al., 1996; Craven et al., 1996; Hamilton Flynn, <br /> regulated gene, SIL1, overexpression suf&#194;&#174;-                     1996). The Lhs1 protein (Lhs1p) represents novel branch <br /> cient suppress Dire1Dlhs1 growth defect. SIL1                    Hsp70 superfamily (Craven et al., 1997) appears <br /> encodes ER-localized protein interacts directly                 ubiquitous eukaryotes, homologues includ- <br /> with ATPase domain Kar2p (BiP), suggesting                       ing mammalian Grp170/Orp150 (Chen et al., 1996; <br /> some role modulating activity vital                      Kuwabara et al., 1996; Craven et al., 1997). Unlike <br /> chaperone. SIL1 non-essential gene                         KAR2, LHS1 gene essential viability, lhs1 <br /> Dlhs1Dsil1 double <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> lethal correlates                     null mutant cells display partial defect post- <br /> with complete block protein translocation                 translational translocation (Baxter et al., 1996; Craven <br /> ER. We conclude IRE1-dependent induction                       et al., 1996; Hamilton Flynn, 1996) also <br /> of SIL1 vital adaptation Dlhs1 cells,                  defective repair misfolded proteins ER <br /> the activities associated Lhs1 Sil1                        (Saris et al., 1997). However, interpretation the <br /> proteins constitute essential function required                  Dlhs1 mutant <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> complicated fact that <br /> protein translocation ER. The Sil1 protein                     cells exhibit constitutive induction unfolded <br /> appears widespread eukaryotes, homo-                       protein response (UPR) (Baxter et al., 1996; Craven et al., <br /> logues Yarrowia lipolytica (Sls1p), Drosophila                   1996). The UPR triggered reduction levels of <br /> mammals.                                                                free Kar2p/BiP, presumably resulting sequestra- <br /> Keywords: endoplasmic reticulum/IRE1/LHS1/                              tion misfolded polypeptides (Kohno et al., 1993; <br /> SIL1,PER100,YOL031c/translocation/UPR                                   Bertolotti et al., 2000). In yeast, leads activation of <br />                                                                         transmembrane kinase/nuclease Ire1p (Cox et al., <br />                                                                         1993; Mori et al., 1993; Shamu Walter, 1996) which <br />                                                                         initiates novel splicing mechanism removes an <br /> Introduction                                                            intron HAC1 mRNA (Cox Walter, 1996; <br /> Protein translocation membrane yeast                  Mori et al., 1996; Sidrauski et al., 1996; Sidrauski and <br /> endoplasmic reticulum (ER) occur distinct                    Walter, 1997). Translation spliced mRNA produces <br /> pathways. The co-translational pathway targets nascent                  functional Hac1p transcription factor then <br /> polypeptides ER membrane signal recognition                  mediates transcriptional induction 381 <br /> particle (SRP), post-translational pathway                  genes yeast genome (Travers et al., 2000). Many of <br /> translocates full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> precursor polypeptides                induced genes known encode ER-resident <br /> targeted ER independently SRP (for review               chaperones, response presumed maximize <br /> Stirling, 1999). Recent evidence suggests speci&#194;&#174;c role                cell&apos;s capacity process misfolded molecules thus <br /> for BiP, ER lumenal Hsp70, gating translocon                  tolerate variety stresses affecting folding pathways <br /> prior initiation co-translational translocation               (Cox et al., 1993; Mori et al., 1993). Interestingly, the <br /> (Hamman et al., 1998). Such activity expected               LHS1 gene itself UPR regulated, suggesting Lhs1p <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> crucial living cells, Kar2p (yeast BiP;             plays role normal cellular response folding <br /> Normington et al., 1989; Rose et al., 1989) recently                stress (Baxter et al., 1996; Craven et al., 1996). However, <br /> been shown essential translocation SRP-                fact UPR induced Dlhs1 cells raises <br /> dependent precursors vivo (B.P.Young, R.A.Craven,                    possibility UPR-regulated chaperones <br /> P.J.Reid C.J.Stirling, submitted). Kar2p                    required compensate loss Lhs1p. <br /> essential post-translational translocation pathway              Here report functional UPR required the <br /> in yeast required distinct stages: (i)           near normal growth rate observed Dlhs1 cells when <br /> initiation translocation prior stable interaction          compared parental strain. Moreover, have <br />  <br /> 6440                                                                                      &#195;&#163; European Molecular Biology Organization <br />                                                                           Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br />  <br /> identi&#194;&#174;ed single UPR-regulated gene, have <br /> named SIL1, essential <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>survival</span> Dlhs1 cells and <br /> whose overexpression suf&#194;&#174;cient suppress pheno- <br /> types associated lhs1 deletion. Furthermore, we <br /> show lethal Dsil1Dlhs1 double mutation results in <br /> a complete defect protein translocation ER. <br />  <br />  <br /> Results <br /> The role UPR cellular adaptation the <br /> Dlhs1 mutation <br /> Yeast cells carrying null mutation lhs1 (Dlhs1) grow <br /> well exhibit constitutive induction UPR <br /> (Craven et al., 1996). As discussed above, IRE1 gene <br />                                                                 Fig. 1. Dlhs1Dire1 double mutant cells exhibit severe growth defect. <br /> encodes key effector yeast UPR essential   JTY21 (IRE1/ire1::KanMX4, LHS1/lhs1::TRP1) diploids carrying <br /> the induction UPR-regulated genes. Cells deleted         pRC43 (2m, URA3, LHS1) sporulated tetrads dissected. Four <br /> IRE1 (Dire1) viable highly sensitive             plasmid-containing haploids single tetrad, genotypes as <br /> tunicamycin, dithiothreitol (DTT) range          indicated, streaked minimal medium containing 5-FOA <br />                                                                 incubated 30&#194;&#176;C select cells lost pRC43 by <br /> treatments perturb protein folding ER (Cox          spontaneous mis-segregation. <br /> et al., 1993; Mori et al., 1993). We reported <br /> previously sporulation heterozygous diploid <br /> strain (Dire1/IRE1, Dlhs1/LHS1) gave rise viable          <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> point onwards, report results relating to <br /> Dire1Dlhs1 haploids (Craven et al., 1996). However,        W303-derived strains. <br /> requirement Ire1p speci&#194;&#174;c germination after <br /> sporulation re&#194;&#175;ected vegetative cells. In <br /> order test latter, &#194;&#174;rst transformed heterozygous    Multicopy KAR2 SCJ1 fail suppress the <br /> diploid (JTY21; Dire1/IRE1, Dlhs1/LHS1) URA3-            Dire1Dlhs1 double mutant phenotype <br /> based plasmid containing functional LHS1 gene               The Dire1Dlhs1 double mutant severe growth <br /> (pRC43). Following sporulation, viable Dire1Dlhs1 hap-          defect compared single mutation alone. <br /> loids recovered which, case,        The simplest interpretation &#194;&#174;ndings propose <br /> be Ura+, indicating presence pRC43 plasmid.          Ire1p-dependent unfolded protein response is <br /> These haploids strains inoculated medium         required induce factors then <br /> containing 5-&#194;&#175;uoro-orotic acid (5-FOA) counter-              compensate loss Lhs1p. Perhaps most <br /> select URA3-based pRC43 described previously             obvious candidate suppressor Lhs1p be <br /> (Wilkinson et al., 1997). Incubation 30&#194;&#176;C gave rise     Kar2p, primarily UPR-regulated member of <br /> low frequency slow growing colonies            Hsp70 superfamily, functional <br /> barely visible Figure 1. This contrasted sharply    overlap Lhs1p (Baxter et al., 1996; Craven et al., <br /> numerous rapidly growing colonies observed pRC43           1996; Hamilton Flynn, 1996). A second obvious <br /> was lost single mutant cells wild-      candidate SCJ1, encodes UPR-regulated DnaJ <br /> type parent (Figure 1). As expected, results con&#194;&#174;rm         homologue yeast ER lumen previously been <br /> that Dlhs1 Dire1 single mutants viable     reported suppress translocation phenotype associ- <br /> grow vigorously. However, &#194;&#174;ndings clearly demon-            ated lhs1 deletion present multicopy <br /> strate severe synthetic effect mutations     plasmid (Hamilton Flynn, 1996). We therefore <br /> combined. From results, conclude Ire1p-       decided test multicopy plasmids containing <br /> dependent induction UPR represents vital               KAR2 (pJT41) SCJ1 (pJT42) able to <br /> adaptation Dlhs1 cells.                                      suppress growth defect Dire1Dlhs1 mutant cells. <br />                                                                 First, transformed pJT41 pJT42 JTY38 <br /> The requirement IRE1 strain dependent                {Dire1Dlhs1 [pRC43 (LHS1, URA3)]} tested the <br /> The phenotypes lhs1 deletion reported       ability strains lose pRC43 5-FOA medium. <br /> be severe W303 strain backgrounds compared         Our results indicated multicopy KAR2 nor <br /> with used (Baxter et al., 1996; Craven et al.,      SCJ1 suppressed growth defect Dire1Dlhs1 <br /> 1996). We constructed Dlhs1 mutation             double mutant (Figure 2A). <br /> W303 (strain JTY33) signi&#194;&#174;cant differences <br /> in extent induction UPR             Isolation multicopy suppressors the <br /> accumulation untranslocated form prepro-a-            Dire1Dlhs1 double mutation <br /> factor (data shown). We tested        We screened multicopy plasmid-based library of <br /> was synthetic interaction Dire1 Dlhs1          yeast genomic DNA order identify genes <br /> W303 similar described TR (above). As before,       capable suppressing Dire1Dlhs1 growth phenotype. <br /> the Dire1Dlhs1 double mutant lethal sporulation        Unfortunately, frequency spontaneous <br /> and vegetative cells (data shown). From            suppressors double mutant arose plasmid-loss <br /> results, conclude IRE1-dependent adaptation      assay (on 5-FOA medium) precluded means with <br /> the loss Lhs1p required strain backgrounds.       screen library. Consequently, necessary <br />  <br />                                                                                                                                 6441 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br />                                                                         ADE3) transformants selected minimal medium <br />                                                                         appropriate supplements. A total 18 000 trans- <br />                                                                         formants examined, 29 authentic <br />                                                                         sectoring clones identi&#194;&#174;ed. Plasmids isolated from <br />                                                                         clones restored sectoring re-trans- <br />                                                                         formed JTY62(pJT40), rescued 5-FOA <br />                                                                         sensitivity strain. Results suppressor <br />                                                                         plasmids, named pIRE1c pSIL1, shown in <br />                                                                         Figure 2B. Sequence data obtained insert&#194;&#177; <br />                                                                         vector boundaries 29 plasmids. Of these, 27 <br />                                                                         contained intact LHS1 gene (with different <br />                                                                         overlapping insert fragments represented), contained <br />                                                                         portion IRE1 gene contained 13.2 kbp <br />                                                                         insert chromosome XV. Clearly, 27 LHS1- <br />                                                                         containing clones complement Dlhs1 mutation, <br />                                                                         resulting cells simple Dire1 phenotype. The <br />                                                                         remaining suppressors, pIRE1c pSIL1, of <br />                                                                         interest. <br />  <br />                                                                         Constitutive induction UPR C-terminal <br />                                                                         fragment Ire1p <br />                                                                         The pIRE1c suppressor plasmid contained 5.54 kbp <br />                                                                         insert corresponding residues 254 693&#194;&#177;260 233 from <br />                                                                         yeast chromosome VIII (Johnston et al., 1994). This insert <br />                                                                         contains C-terminal 661 codons IRE1 open <br />                                                                         reading frame (ORF) fused in-frame AUG codon <br />                                                                         located 62 codons upstream vector sequence. <br />                                                                         Initiation AUG codon rise fusion <br />                                                                         protein lacking ER-luminal dimerization domain of <br />                                                                         Ire1p, contain transmembrane <br />                                                                         domain cytosolic kinase/nuclease domains (see <br /> Fig. 2. Plasmid suppressors growth defect Dire1Dlhs1 mutant   Figure 3A) (Cox et al., 1993; Mori et al., 1993). We <br /> cells. (A) Multicopy KAR2 SCJ1 fail suppress Dire1Dlhs1. JTY38    tested ability pIRE1c induce UPR by <br /> (Dire1Dlhs1 + pRC43) transformed pRS423 (vector control), <br /> pRC42 (HIS3, LHS1), pJT41 (2m, HIS3, KAR2) pJT42 (2m, HIS3,          measuring induction b-galactosidase activity cells <br /> SCJ1) restreaked minimal medium containing 5-FOA           carrying plasmid lacZ gene expressed <br /> incubated 30&#194;&#176;C. (B) Isolation suppressors multicopy        control UPR-regulated promoter element <br /> library. JTY38 containing pRC43 transformed pRS423      (UPRE; Wilkinson et al., 2000). Cells carrying the <br /> (vector control), pRC42 (HIS3, LHS1), pIRE1c (2m, LEU2, IRE1 <br /> C-terminus) pJT43 (2m, HIS3, SIL1). Transformants then <br />                                                                         UPRE&#194;&#177;lacZ construct transformed with <br /> inoculated 5-FOA medium incubated before.                   pIRE1c vector control. As expected, wild-type <br />                                                                         cells containing vector control exhibited low <br />                                                                         level LacZ activity, lower levels detected in <br /> for devise robust suppression screen based               Dire1 cells inactive UPR (Figure 3B). In <br /> plasmid sectoring assay. First, strain constructed             contrast, vector-transformed Dlhs1 cells exhibited a <br /> which Dire1Dlhs1 mutations combined                       constitutive UPR induction reported previously <br /> mutant alleles ade2 ade3 (JTY62)                  (Figure 3B) (Craven et al., 1996). When transformed <br /> viability maintained presence LHS1 plasmid              pIRE1c, strains showed substantial <br /> pJT40 (URA3, ADE3, LHS1). In strain, pJT40 stably               expression UPR-regulated LacZ levels excess of <br /> maintained strong selection functional                 observed Dlhs1 cells (Figure 3B). The ability of <br /> LHS1 gene. Since pJT40 contains ADE3 gene,                Ire1Cp induce UPR con&#194;&#174;rmed its <br /> stability visualized easily virtue uniformly           ability rescue tunicamycin sensitivity Dire1 cells <br /> red colonies characteristic Ade3+, Ade2&#194;&#177; cells              (Figure 3C). These observations demonstrate the <br /> (Koshland et al., 1985). However, strain                  truncated C-terminal fragment Ire1p, termed Ire1Cp, is <br /> transformed second LHS1-containing                       constitutive inducer UPR pathway yeast. The <br /> plasmid, pRC42 (HIS3, LHS1), cells lose                      ability induce UPR levels excess those <br /> pJT40, giving rise colonies containing Ade3&#194;&#177; white                   normally observed Dlhs1 cells appear to <br /> sectors (data shown). Having established reliable                 suf&#194;&#174;cient suppress Dire1Dlhs1 double mutant <br /> sectoring assay, &#194;&#174;rst tested multicopy plasmids                  phenotype. <br /> containing KAR2 (pJT41) SCJ1 (pJT42). As <br /> expected, plasmid gave rise sectoring colonies,              Identi&#194;&#174;cation YOL031c multicopy <br /> con&#194;&#174;rming inability suppress Dire1Dlhs1                    suppressor Dire1Dlhs1 <br /> growth phenotype (data shown).                                      The class suppressor identi&#194;&#174;ed termed SIL1 for <br />    A multicopy LEU2-based library transformed                  (suppressor Dire1 Dlhs1 double mutant number 1). <br /> JTY62 (Dire1Dlhs1) containing pJT40 (LHS1, URA3,                        Unlike pIRE1c, pSIL1 plasmid did restore <br />  <br /> 6442 <br />                                                                                          Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br />  <br />  <br />  <br /> Fig. 3. IRE1c encodes C-terminal portion IRE1 constitutively induces UPR suppresses tunicamycin hypersensitivity Dire1 <br /> mutants. (A) Schematic views Ire1p predicted Ire1Cp domain structures. The black region N-terminal end Ire1Cp indicates <br /> potential vector coding sequence in-frame junction vector genomic insert. (B) Constitutive UPR induction pIRE1c- <br /> containing cells. Strains JTY18 (WT), JTY32 (Dire1) JTY33 (Dlhs1) harbouring UPRE-lacZ reporter plasmid pJT30 transformed with <br /> the plasmids pIRE1c (IRE1 fragment) YEplac181 (vector control) assayed b-galactosidase activity. Fold induction relative activity <br /> measured JTY18 containing YEplac181. (C) pIRE1c rescues tunicamycin sensitivity Dire1 cells. JTY32 (Dire1) transformed either <br /> pIRE1c (IRE1 fragment) YEplac181 (vector control), streaked YPD containing 1 mg/ml tunicamycin incubated 30&#194;&#176;C. <br />  <br />  <br />  <br /> tunicamycin resistance Dire1 cells,                           strain JTY33 transformed pJT43 (SIL1) or <br /> appears unlikely functions inducer UPR                    vector control cell extracts prepared and <br /> (data shown). The insert pSIL1 corresponds                         examined immunoblotting. As expected, Dlhs1 cells <br /> 13.2 kbp portion chromosome XV (Rad et al., 1997).                          containing control vector accumulated cytosolic pre- <br /> Subcloning pSIL1 identi&#194;&#174;ed YOL031c ORF                             cursor forms pre-Kar2, pre-PDI prepro-a-factor <br /> encoding SIL1 activity. The SIL1/YOL031c ORF                               (Figure 4A). The extent precursor accumulation was <br /> comprises 421 codons encode polypeptide                            signi&#194;&#174;cantly reduced cell extracts prepared JTY33 <br /> with predicted mol. wt 48.26 kDa. The primary                             cells carrying pJT43 (Figure 4A). These results indicate <br /> sequence contains potential cleavable N-terminal signal                      pJT43 largely suppresses translocation phenotype <br /> sequence direct protein ER lumen,                        associated Dlhs1 mutation. In contrast, neither <br /> and C-terminal RDELCOOH likely function ER-                         multicopy KAR2 SCJ1 effect levels of <br /> retrieval sequence. We predict product                   precursor accumulated Dlhs1 cells (Figure 4A), result <br /> the SIL1 gene ER-localized protein. The SIL1/                            entirely consistent failure suppress growth <br /> YOL031c protein sequence previously noted                            defect Dlhs1Dire1 double mutant. <br /> homologous Sls1 protein (Sls1p) from <br /> Yarrowia lipolytica, ER-resident protein <br /> known interact Kar2p involved                          Phenotypic analysis Dsil1 null mutant <br /> the biosynthesis secretion alkaline extracellular                    The SIL1 ORF (YOL031c) disrupted the <br /> protease (Boisrame et al., 1996, 1998).                                        Eurofan project shown non-essential for <br />                                                                                viability. We obtained cloned sil1::kanMX4 disruption <br />                                                                                cassette (pYORC_YOL031c) Euroscarf used this <br /> Overexpression Sil1p suppresses                                         recreate Dsil1 mutation transformation a <br /> translocation phenotype associated Dlhs1                                  haploid W303-derived strain, JTY19, geneticin resist- <br /> Previous studies shown Dlhs1 mutant cells                        ance. The site integration transformant, <br /> defective protein translocation ER. In order                    JTY63, con&#194;&#174;rmed PCR strain used for <br /> characterize nature Sil1p suppressor further,                    studies. As expected, results con&#194;&#174;rmed the <br /> next sought determine rescued                         Dsil1 mutant cells viable. The Y.lipolytica SLS1 gene is <br /> translocation phenotype Dlhs1 cells. The Dlhs1 mutant                       non-essential implicated secretory <br />  <br />                                                                                                                                                 6443 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br />  <br />  <br /> Fig. 4. Characterization SIL1 suppressor. (A) Multicopy SIL1 suppresses translocation defect Dlhs1 cells. Wild-type (JTY18), JTY33 <br /> (Dlhs1) transformed vector control (pRS423), pJT43 (SIL1), pJT42 (SCJ1) pJT41 (KAR2) grown mid-log phase 30&#194;&#176;C in <br /> YNB appropriate supplements. Whole-cell extracts prepared separated SDS&#194;&#177;PAGE immunoblotting antibodies speci&#194;&#174;c <br /> to Lhs1p, Kar2p, PDI a-factor serum indicated. Untranslocated precursor forms various proteins indicated (preKar2p, prePDI <br /> and prepro-a-factor). Translocated forms subject ER processing; signal-processed form Kar2p indicated, signal-processed <br /> and core-glycosylated forms PDI. Processed forms a-factor evident gel system. (B) Dsil1 cells obvious defect protein <br /> translocation. Strains JTY33 (Dlhs1) JTY63 (Dsil1) grown mid-log phase YNB harvested whole-cell extracts prepared <br /> and analysed immunoblotting indicated. Control extracts prepared wild-type cells (JTY18) treated presence or <br /> absence tunicamycin (10 mg/ml) 2 h prior harvesting cells. Protein species annotated include unglycosylated, signal- <br /> processed forms Lhs1p (ugLhs1p) PDI (ugPDI). (C) Constitutive levels UPR induction. Wild-type (JTY18), Dsil1 (JTY63) Dlhs1 <br /> (JTY33) cells transformed pJT30 (UPRE&#194;&#177;lacZ) grown mid-log phase assaying b-galactosidase activity. JTY33 (Dlhs1) <br /> containing pJT30 assayed transformed multicopy SIL1 (pJT43). (D) The Dsil1 Dlhs1 synthetically lethal in <br /> combination. JTY64 (SIL1/sil1::KanMX4, LHS1/lhs1::KanMX4) diploids carrying pRC43 (2m, URA3, LHS1) sporulated tetrads dissected. <br /> A representative tetrad (with genotypes indicated) shown streaked 5-FOA-containing medium select pRC43. <br />  <br />  <br />  <br /> protein biogenesis system. We analysed                       appeared identifying SIL1/YOL031c UPR-regulated <br /> the translocation processing various precursor                          gene authors named PER100 (Travers et al., <br /> proteins Dsil1 null mutant cells immunoblotting. No                      2000). <br /> defects observed biogenesis PDI or <br /> Kar2, protein pro&#194;&#174;les Dsil1 cell extracts                        SIL1 essential viability Dlhs1 cells <br /> indistinguishable wild-type extracts                             Thus far, data indicate overexpression Sil1p is <br /> (Figure 4B). We examined UPR status Dsil1 cells                        suf&#194;&#174;cient suppress Dire1Dlhs1 double mutant. The <br /> and modest induction compared wild-type                           SIL1 gene itself UPR inducible (Travers et al., 2000) and <br /> cells, suggesting minor role Sil1p protein folding                    Sil1p play role Ire1-dependent <br /> within lumen (Figure 4C). We tested effect                    adaptation Dlhs1 cells described earlier. If induction of <br /> of multicopy SIL1 level UPR induction Dlhs1                       Sil1p important suppressing Dlhs1 mutation, then <br /> cells reduced 4-fold 3-fold                    expect synthetic effect Dlhs1 and <br /> induction relative wild-type control (Figure 4C).                         Dsil1. To test this, mated JTY33 JTY63 generate <br /> While manuscript preparation, paper                              doubly heterozygous diploid JTY64 (Dlhs1/LHS1 <br />  <br /> 6444 <br />                                                                          Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br />  <br /> Dsil1/SIL1). After sporulation, tetrads dissected      mutant medium lacking methionine (data not <br /> YPD medium incubated 30&#194;&#176;C, viable            shown). The plasmid transformed into <br /> haploid Dsil1Dlhs1 progeny obtained. Genotyping             Dlhs1Dsil1 cells carrying pRC43 (LHS1, URA3), selecting <br /> and PCR analysis viable spores demonstrated               HIS3 marker pJT44. Transformants then <br /> recovery single mutants wild-type spores           inoculated 5-FOA medium, lacking methionine in <br /> expected frequencies. Where double mutant             order select cells cured pRC43. The resultant <br /> inferred genotypes siblings,              strain (JTY66) grew medium lacking methionine <br /> examined microscopically shown germinated            medium supplemented 0.2 mM methionine <br /> but failed progress cell      (Figure 5A). Addition methionine (0.2 mM) liquid <br /> divisions. We transformed JTY64 (Dlhs1/LHS1 Dsil1/          culture led severe growth defect 3 h, suggesting <br /> SIL1) pRC43 (LHS1, URA3) repeated tetrad            Lhs1ptrun activity limiting growth that <br /> analysis. In case, recovered viable haploid              time (Figure 5B). Immunoblotting analysis veri&#194;&#174;ed that <br /> Dsil1Dlhs1 double mutants              JTY66 cells expressed truncated form Lhs1p <br /> contain pRC43. Incubation strains 5-FOA               (Lhs1ptrun), undetectable 3 h addition of <br /> medium failed produce viable colonies, con&#194;&#174;rming          methionine (Figure 5C, lanes 3 4), con&#194;&#174;rming the <br /> that double mutant combination lethal (Figure 4D).       growth defect cells correlates loss the <br />                                                                  Lhs1ptrun antigen. We examined processing of <br /> Overexpression Sil1p does accelerate                      various secretory precursors cells. As before, <br /> ER-associated degradation                                        Dlhs1 mutant cells accumulated precursor forms pre- <br /> It reported deletion SIL1/PER100 leads       Kar2, pre-PDI prepro-a-factor absent in <br /> a minor reduction rate misfolded               wild-type cells (Figure 5C, lanes 1 2). This trans- <br /> proteins exported ER lumen degradation          location defect complemented MET3p-lhs1trun in <br /> in cytosol (Travers et al., 2000). If growth defect   absence methionine, slowly re-appeared upon <br /> Dire1Dlhs1 cells accumulation                 depletion Lhs1ptrun following addition methionine to <br /> misfolded proteins ER lumen, obvious              medium (Figure 5C, compare lane 2 lanes 6&#194;&#177;8). <br /> source suppression induction factor         When Dlhs1Dsil1 double mutant cells carrying pJT44 were <br /> that accelerate ER-associated degradation (ERAD).          grown absence methionine, precursors accumu- <br /> We tested multicopy SIL1 led                lated level intermediate wild-type Dlhs1, <br /> accelerated degradation mutant form carboxy-             suggesting partial complementation MET3p-lhs1trun <br /> peptidase Y (CPY), known CPY*, misfolds          (Figure 5C, lanes 1&#194;&#177;3). However, 3 h repression in <br /> ER lumen shown substrate          presence methionine, cells exhibited an <br /> ERAD (Knop et al., 1996). W303-1C cells (expressing              enormous accumulation precursors tested (Figure 5C, <br /> CPY* prc1-1 allele) transformed        lane 4). These results demonstrate direct correlation <br /> pJT43 (SIL1) vector control, rate                lethal consequences Dlhs1Dsil1 double <br /> degradation CPY* monitored pulse&#194;&#177;chase                 mutation major defect translocation protein <br /> studies. We half-life CPY* fact         precursors ER. <br /> marginally extended cells carrying multicopy SIL1                Previous studies shown precursors be <br /> (pJT43) compared wild-type control (data            translocated ER quite different routes, i.e. <br /> shown). Clearly, overexpression Sil1p does            SRP-dependent SRP-independent pathways (Ng <br /> accelerate CPY* degradation, appears               et al., 1996). In order determine these <br /> unlikely observed suppression Dire1Dlhs1              pathways affected Dlhs1Dsil1 cells, next <br /> pSIL1 accelerated ERAD.                                examined fate SRP-dependent precursor, <br />                                                                  dipeptidylaminopeptidase B (DPAPB; Ng et al., 1996), <br /> A Dlhs1Dsil1 double mutant severely defective              SRP-independent precursor, CPY (Ng et al., <br /> protein translocation ER                                1996), pulse-labelling depletion Lhs1ptrun. <br /> The fact Dlhs1Dsil1 double mutation lethal           Our results substantial defect CPY translocation <br /> renders phenotypic analysis dif&#194;&#174;cult. In order overcome       60 min incubation presence methionine, <br /> this, created conditional expression allele LHS1.        &gt;50% newly synthesized protein accumu- <br /> Initially, placed complete LHS1 ORF             lated precursor form (Figure 6, lanes 1 2). This <br /> control methionine-repressible MET3 promoter              rose 100% precursor accumulation 2 h depletion <br /> (Cherest et al., 1985). However, construct        (Figure 6, lane 3). A defect DPAPB translocation was <br /> to complement Dire1Dlhs1 double mutant           observed, onset defect delayed. <br /> presence methionine (data shown). Immunoblot              Minor accumulation pre-DPAPB evident 1 h, <br /> analysis indicated marginal reduction levels         &gt;50% defect 2 h total defect 3 h <br /> Lhs1p antigen methionine-containing medium               (Figure 6). These &#194;&#174;ndings demonstrate SRP- <br /> over 8 h time course (data shown). We reasoned       dependent -independent pathways blocked in <br /> the level Lhs1p antigen observed cells               Dsil1Dlhs1 cells SRP-independent pathway is <br /> repressing conditions re&#194;&#175;ect long half-life      &#194;&#174;rst affected Lhs1ptrun depleted. <br /> protein ER. We sought destabilize the <br /> protein deleting C-terminal RDELCOOH retrieval            Sil1p conserved mammals Drosophila <br /> motif. A plasmid, pJT44, expressing truncated copy       Sil1p homologous Sls1 protein Y.lipolytica <br /> Lhs1p control MET3 promoter (MET3p-                 identi&#194;&#174;ed homologues human and <br /> lhs1trun) complement Dire1Dlhs1 double          mouse expressed sequence tag (EST) sequence databases, <br />  <br />                                                                                                                        6445 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br /> and Drosophila genome sequence. A multiple         Hsp70 regulators present compartments of <br /> alignment shown Figure 7, illustrating signi&#194;&#174;cant   eukaryotic cell. <br /> similarities &#194;&#174;ve proteins. Previous studies <br /> have shown Y.lipolytica Sls1p interacts Kar2p   Sil1p interacts ATPase domain Kar2p <br /> and, intriguingly, human       In order test yeast Sil1p interacts directly with <br /> mouse Sil1p sequences share signi&#194;&#174;cant simil-   Kar2p, tested co-puri&#194;&#174;cation recombin- <br /> arity cytosolic Hsp70-binding proteins HspBP1      ant forms proteins. Sil1p fused GST <br /> (human: NP_036399; rat: AAF35834) HSBp2               (GST&#194;&#177;Sil1p) puri&#194;&#174;ed Escherichia coli extracts <br /> (human: AAF35833) (Raynes Guerriero, 1998,            binding glutathione&#194;&#177;Sepharose (Figure 8). When <br /> 2000). These represent major family    cell extract containing recombinant His-tagged Kar2p <br />                                                           incubated GST&#194;&#177;Sil1p-bound beads, a <br />                                                           substantial quantity recombinant Kar2p to <br />                                                           bind manner dependent presence of <br />                                                           GST&#194;&#177;Sil1p. As Hsp70, Kar2p N-terminal <br />                                                           ATPase domain C-terminal domain respon- <br />                                                           sible binding unfolded polypeptides (Craven et al., <br />                                                           1997). The binding Kar2p indicate <br />                                                           merely GST&#194;&#177;Sil1p fusion protein partially <br />                                                           misfolded. We excluded possibility demon- <br />                                                           strating recombinant fragment Kar2p, <br />                                                           corresponding ATPase domain, interacts <br />                                                           speci&#194;&#174;cally GST&#194;&#177;Sil1p (Figure 8). These results <br />  <br />  <br />  <br />  <br />                                                           Fig. 6. The translocation defects Dsil1Dlhs1 occur rapidly are <br />                                                           complete. Strains JTY66 (Dsil1Dlhs1) JTY33 (Dlhs1) containing <br />                                                           pJT44 (MET3p-lhs1trun) grown mid-log phase YNB medium <br />                                                           addition 0.2 mM methionine repress MET3 promoter. <br />                                                           Cultures labelled cocktail [14C]amino acids (as <br />                                                           described Materials methods) time points indicated, then <br />                                                           extracts prepared subjected sequential immunoprecipitation <br />                                                           antibodies speci&#194;&#174;c DPAPB followed CPY (see Materials <br />                                                           methods). Immunoprecipitates resolved SDS&#194;&#177;PAGE, <br />                                                           8% 10% gels, respectively, detected &#194;&#175;uorography. The <br />                                                           untranslocated precursor form CPY indicated (prepro-CPY), are <br />                                                           ER-processed (p1 CPY) Golgi-modi&#194;&#174;ed (p2 CPY) forms. The <br />                                                           untranslocated mature forms DPAPB indicated as <br />                                                           preDPAPB DPAPB, respectively. <br />  <br />  <br />  <br />  <br />                                                           Fig. 5. Conditional expression truncated Lhs1p reveals severe <br />                                                           translocation defect Dsil1Dlhs1 double mutant cells. (A) JTY65 <br />                                                           (Dsil1Dlhs1 + pRC43) transformed pJT44 (HIS3, MET3- <br />                                                           LHS1trun) passaged 5-FOA medium remove pRC43 <br />                                                           (URA3, LHS1). The resulting strain, JTY66 (Dsil1Dlhs1 + pJT44), was <br />                                                           viable medium lacking methionine medium <br />                                                           containing 0.2 mM methionine. (B) Effect methionine on <br />                                                           logarithmically growing cultures. JTY66 (Dsil1Dlhs1 + pJT44) cells <br />                                                           grown minimal medium lacking methionine log phase, the <br />                                                           culture split, 0.2 mM methionine added half the <br />                                                           effect growth examined time. Sample points (C) are <br />                                                           indicated arrows time methionine addition which <br />                                                           correspond. (C) Translocation defects Lhs1ptrun depletion. <br />                                                           Aliquots cells (B) harvested whole-cell extracts <br />                                                           prepared analysed imunoblotting Lhs1p, Kar2p, PDI, CPY <br />                                                           a-factor indicated. Protein species annotated but <br />                                                           include untranslocated (preproCPY) mature (mCPY) forms <br />                                                           CPY, C-terminally truncated form Lhs1p (Lhs1ptrun). <br />  <br />  <br /> 6446 <br />                                                                                      Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br />  <br />  <br />  <br /> Fig. 7. Sequence alignment Sil1p-like proteins. The Sil1p sequence S.cerevisiae (Sc) aligned Sls1p Y.lipolytica (Yl) with <br /> sequences Homo sapiens (Hs; DDBJ/EMBL/GenBank accession No. AJ299442), Mus musculus (Mm; accession No. AJ297884) Drosophila <br /> melanogaster (Dm). Alignments created using CLUSTAL_X program, black shading representing 100% conservation amino acid <br /> similarity, dark grey 80% light grey 60%. The boxed region highlighting conserved region motif used identify human and <br /> mouse EST sequences, allowing complete sequence assembled overlapping ESTs. <br />  <br />  <br />  <br /> demonstrate biochemical interaction Sil1p                    Kar2p required compensate absence of <br /> the ATPase domain yeast Kar2p.                                          Lhs1p. However, &#194;&#174;ndings indicate multicopy <br />                                                                            KAR2 effect growth Dire1Dlhs1 <br />                                                                            double mutant. Interestingly, multicopy KAR2 been <br /> Discussion                                                                 shown rescue growth defect associated with <br /> Previous characterization yeast Dlhs1 null mutant                   accumulation misfolded protein Dire1 cells <br /> revealed modest growth phenotype identi&#194;&#174;ed                    (Umebayashi et al., 1999). It follows the <br /> substantial induction UPR-regulated genes. Genetic                      growth defect Dire1Dlhs1 cells unlikely due <br /> studies revealed Dire1Dlhs1 double mutant spores                      solely accumulation misfolded proteins. <br /> were non-viable, leading propose Ire1p-                     Similarly, overexpression Kar2p does rescue the <br /> dependent UPR represents important physiological                        translocation phenotype Dlhs1 mutant, we <br /> adaptation Dlhs1 mutant cells (Craven et al., 1996,                     conclude phenotype indirect consequence <br /> 1997). This hypothesis con&#194;&#174;rmed study                  sequestration Kar2p unfolded polypeptides. <br /> have shown severe synthetic growth defect vegetative                     We attempted isolate multicopy plasmid <br /> cells Dlhs1 Dire1 mutations combined.                 suppressors Dire1Dlhs1 double mutant growth <br /> Given Lhs1p Kar2p related, suggested                  defect. As expected, isolated complementing clones of <br /> previously (Craven et al., 1996) UPR induction                 LHS1 IRE1, struck isolation a <br />  <br />                                                                                                                                          6447 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br />  <br />  <br /> Fig. 8. Biochemical interaction GST&#194;&#177;Sil1p His-Kar2p His-Kar2pATPase. Crude extracts prepared bacterial expression GSTp, <br /> GST&#194;&#177;Sil1p, His-Kar2p His-Kar2pATPase, respectively (lanes 1&#194;&#177;4). Each incubated glutathione&#194;&#177;agarose bound <br /> fraction resolved SDS&#194;&#177;PAGE (lanes 5&#194;&#177;8). Lanes 9 10 contain fraction His-Kar2p His- Kar2pATPase crude extract binds GSTp- <br /> bound glutathione&#194;&#177;agarose. Lanes 11 12 contain fraction His-Kar2p His- Kar2pATPase crude extract binds GST&#194;&#177;Sil1p-bound <br /> glutathione&#194;&#177;agarose. <br />  <br /> clone capable expressing C-terminal portion                  tion Sil1p suf&#194;&#174;cient explain observed <br /> Ire1p (Ire1Cp). This clone led constitutive induction               requirement Ire1p Dlhs1 cells. An absolute require- <br /> the UPR provides novel insights                   ment Sil1p Dlhs1 cells demonstrated the <br /> into function Ire1p. Current evidence suggests                  synthetically lethal interaction observed Dsil1 <br /> the N-terminal `dimerization&apos; domain Ire1p interacts                     Dlhs1 mutations. We conclude IRE1- <br /> with Kar2p/BiP ER lumen and, sequestration                      dependent induction SIL1 critical adaptation in <br /> of Kar2p/BiP, promotes oligomerization Ire1p                         Dlhs1 cells. <br /> subsequent activation phosphorylation (Shamu                            The SIL1/YOL031c gene encodes polypeptide pre- <br /> Walter, 1996; Welihinda Kaufman, 1996; Bertolotti                       viously noted 27.5% identical (52% similar) the <br /> et al., 2000). Given induction occur cells                 Sls1 protein (Sls1p) Y.lipolytica (Boisrame et al., <br /> expressing C-terminal kinase/nuclease domain,                      1996). The SLS1 gene identi&#194;&#174;ed virtue a <br /> then conclude dimerization domain                       mutation lethal combination defect the <br /> Ire1p essential Ire1p activation. We                   Y.lipolytica SRP, suggesting role Sls1p protein <br /> propose lumenal domain functions primarily                    translocation (Boisrame et al., 1996). <br /> Kar2p-dependent inhibitor oligomerization/activation                     However, like Dsil1 mutants Saccharomyces <br /> of Ire1p.                                                                   cerevisiae, Y.lipolytica Dsls1 mutant viable and <br />    In addition complementing clones LHS1 IRE1,                    does accumulate untranslocated precursor forms of <br /> our screen identi&#194;&#174;ed bona &#194;&#174;de suppressor named SIL1.                    secretory proteins. Nonetheless, Dsls1 mutant is <br /> The SIL1 gene corresponds reading frame unknown                     defective rate synthesis secretory <br /> function, YOL031c, identi&#194;&#174;ed yeast genome                            protein translocation defect inferred <br /> sequence. The SIL1/YOL031c gene char-                         (Boisrame et al., 1998). Some role translocation is <br /> acterized recently PER100, 381 UPR-inducible                      supported &#194;&#174;nding Sls1p co- <br /> genes yeast transcriptome (Travers et al., 2000). The                immunoprecipitated Sec61p (Boisrame et al., 1996), <br /> &#194;&#174;nding SIL1 UPR regulated allow IRE1-                     core component translocon complex responsible <br /> dependent induction Sil1p Dlhs1 cells. Such induc-                    protein translocation ER membrane (Stirling <br />  <br /> 6448 <br />                                                                            Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br />  <br /> et al., 1992; Broughton et al., 1997). Moreover, Sls1p       absence Lhs1p, propose Ire1-dependent <br /> been shown interact ATP-binding domain            induction Sil1p required activate/potentiate Kar2p <br /> Kar2p, leading proposal Sls1p function      order enhance contribution import reaction. <br /> a co-chaperone responsible modulating activity        The &#194;&#174;nding Sil1p interacts directly ATPase <br /> Kar2p ATP-dependent reaction cycle (Boisrame          domain Kar2p suggests direct mechanism which <br /> et al., 1998; Kabani et al., 2000).                              Sil1p modulate Kar2p activity. The proposed role of <br />    The translocation defects previously reported Dlhs1        Kar2p translocon gating independent Sil1p, <br /> cells consistent defect exclusive     non-essential, translocation is <br /> the post-translational SRP-independent protein trans-            blocked Dsil1 mutant. Our model makes number <br /> location pathway (Baxter et al., 1996; Craven et al.,            experimentally veri&#194;&#174;able predictions currently are <br /> 1996; Hamilton Flynn, 1996). However, lethal             investigation. Finally, identi&#194;&#174;cation homo- <br /> depletion truncated form Lhs1p Dsil1Dlhs1           logues Sil1p variety eukaryotes suggests this <br /> cells correlates severe translocation phenotype       newly de&#194;&#174;ned role Sil1p fact widespread. <br /> all precursors. This gross perturbation of <br /> the structural integrity ER simply <br /> eliminate ER functions. However, observed                 Materials methods <br /> substantial ER glycosylation newly synthesized <br /> DPAPB cells translocation prepro-CPY              Materials <br /> blocked (Figure 6, lanes 2 3). From this, evident      DNA restriction modi&#194;&#174;cation enzymes purchased Roche <br /> that defect CPY translocation precede            Molecular Biochemicals. [35S]methionine [14C]amino acid <br />                                                                  cocktail NEN Life Science Products. Oligonucleotides were <br /> gross loss ER function. Our data entirely       purchased Perkin-Elmer (Warrington, UK). All reagents were <br /> consistent direct role Lhs1p post-             Roche Molecular Biochemicals, Sigma Melford Labs (Suffolk, <br /> translational translocation reaction. We cannot,         UK) analytical grade. <br /> time, exclude possibility secondary consequence <br />                                                                  Strain growth conditions <br /> of defect lead observed defect           Escherichia coli yeast strains listed Table I. The E.coli <br /> co-translational translocation. Alternatively, delayed       cells grown 37&#194;&#176;C LB (1% tryptone, 0.5% yeast extract, 1% <br /> onset co-translational translocation defect           NaCl). Antibiotics used appropriate follows: ampicillin <br /> re&#194;&#175;ect direct requirement Lhs1p/Sil1p satis&#194;&#174;ed     (100 mg/ml), kanamycin (40 mg/ml) chloramphenicol (34 mg/ml). <br /> by lower levels Lhs1ptrun needed post-        Yeast strains grown routinely 30&#194;&#176;C YP medium (2% peptone, <br />                                                                  1% yeast extract) containing 2% glucose (YPD) minimal medium <br /> translational reaction. Circumstantial evidence support       (0.67% yeast nitrogen <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span>; YNB) 2% glucose plus appropriate <br /> of direct role includes fact Y.lipolytica    supplements selective growth. When screening sectoring colonies <br /> SLS1 gene identi&#194;&#174;ed genetic interactions        using ade2 ade3 mutations, 15 mM adenine added medium; <br /> SRP (Boisrame et al., 1996), mammalian         level adenine suf&#194;&#174;cient allow growth allows red/ <br />                                                                  white colony phenotype develop observed. Solid media were <br /> homologue Lhs1p, GRP170, major component           supplemented 2% Bacto-agar. All media Difco <br /> ATP-binding fraction ER lumenal proteins required         Laboratories. Repression LHS1trun expressed MET3 promoter <br /> co-translational translocation microsomes vitro          achieved addition L-methionine early log-phase cultures a <br /> (Dierks et al., 1996).                                           &#194;&#174;nal concentration 0.2 mM. Yeast transformations 5-FOA <br />    The translocation defect observed Dsil1Dlhs1 double        counter-selection Ura3+ cells carried described previously <br />                                                                  (Wilkinson et al., 2000). Diploids sporulated 1% KOAc, 0.1% <br /> mutant cells clearly demonstrates Kar2p              yeast extract, 0.05% glucose, plus appropriate supplements 24&#194;&#176;C. <br /> suf&#194;&#174;cient drive ER translocation vivo. We        Tetrad dissection described previously (Wilkinson et al., 2000). <br /> conclude Sil1p/Lhs1p provide novel lumenal                Where appropriate, geneticin used &#194;&#174;nal concentration of <br /> function(s) essential translocation process.     200 mg/ml YPD. Cell density liquid culture monitored by <br />                                                                  A600 nm using UVmini1240 spectrophotometer (Shimadzu). <br /> Clearly, function affect translocon gating, <br /> vectorial transport lumen, termination trans-        Yeast strain construction <br /> location recycling components translocon.       JTY33 (Dlhs1) created transformation JTY18 linear <br /> Current evidence indicates Kar2p/BiP suf&#194;&#174;cient        Dlhs1::kanMX4 cassette replacing LHS1 codons 1&#194;&#177;881 kanMX4. <br /> gate translocon initiation translocation           The cassette generated PCR ampli&#194;&#174;cation kanMX4 module <br />                                                                  pFA6-kanMX4 (Wach et al., 1994) using primers 5&#194;&#162;-TGCAGT- <br /> (Hamman et al., 1998). However, Lhs1 protein                 ATTCTGGCATTATTAGTGCAAATAAGTACGCATATTACCCGT- <br /> (Lhs1p) shown bind peptides ATP-            ACGCTGCAGGTCGAC-3&#194;&#162; 5&#194;&#162;-TGATATCGGAAAATAAATCTA- <br /> dependent manner similar demonstrated BiP            GTGCTATATATTATAAAGATTCTTATCGATGAATTCGAGCTCG- <br /> (Hamilton et al., 1999). We propose Lhs1p         3&#194;&#162; 5&#194;&#162; homology LHS1 ORF-&#194;&#175;anking regions 3&#194;&#162; homology <br />                                                                  kanMX4. JTY18 transformed 1 mg resultant PCR <br /> binds translocating polypeptides, contributing        product selection YPD containing 200 mg/ml geneticin. Correct <br /> the driving force post-translational translocation      integration Dlhs1::kanMX4 cassette con&#194;&#174;rmed PCR <br /> manner similar previously suggested Kar2p.           analysis integrated locus, loss Lhs1p con&#194;&#174;rmed by <br /> There evidence Sil1p interacting directly         immunoblotting. <br /> precursors, but, given observed interactions            Insertion Dire1::kanMX4 deletion allele TR (TR3) <br />                                                                  W303 (JTY19) backgrounds produce JTY20 JTY32, <br /> Sil1p Kar2p, propose complex Sil1p&#194;&#177;             respectively, performed method previously described <br /> Kar2p acts concert Lhs1p drive import             (Wilkinson et al., 2000). JTY20 mated RCY104 (Craven <br /> reaction. Lhs1p suf&#194;&#174;cient activity Dsil1   et al., 1996) produce TR-derived heterozygous diploid (IRE/Dire1, <br /> mutant translocation defect. Indeed, fact        LHS1/Dlhs1). Similarly, strains JTY32 (Dire1) JTY33 (Dlhs1) were <br />                                                                  mated produce W303-based heterozygous diploid (IRE/Dire1, LHS1/ <br /> Dlhs1 cells exhibit signi&#194;&#174;cant translocation defect,           Dlhs1). The heterozygous diploids transformed with <br /> despite UPR induction Sil1p, suggests Lhs1p              pRC43, multicopy yeast vector containing URA3 LHS1 genes, to <br /> plays major role protein translocation yeast. In     produce W303- TR-derived heterozygous diploids, JTY21 JTY34, <br />  <br />  <br />                                                                                                                                     6449 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br /> Table I. Strains <br /> Strains                     Genotype <br />  <br /> Saccharomyces cerevisiae <br />   CH1305                 MATa ade2 ade3 lys2, leu2, ura3 <br />   RCY104                 MATa ade2 his3 lys2 trp1 ura3 Dlhs1::TRP1 <br />   TR3                    MATa ade2 his3 lys2 trp1 ura3 <br />   W303                   MATa/MATa ade2-1/ade2-1 CAN1-100/CAN1-100 his3-11,15/his3-11,15 leu2-3,112/leu2-3,112 trp1-1/trp1-1 ura3-1/ura3-1 <br />   W303-1C                MATa ade2 CAN1 his3 leu2 trp1 ura3 prc1-1 <br />   JTY18                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1 (derived W303) <br />   JTY19                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1 (derived W303) <br />   JTY20                  MATa ade2 his3 lys2 trp1 ura3 Dire1::kanMX4 <br />   JTY21                  MATa/MATa ade2/ade2 his3/his3 lys2/lys2 trp1/trp1 ura3/ura3 Dire1::kanMX4/+ Dlhs1::TRP1/+ [pRC43] <br />   JTY25                  MATa ade2 his3 lys2 trp1 ura3 Dire1::kanMX4 Dlhs1::TRP1 [pRC43] <br />   JTY30                  MATa ade2 his3 lys2 trp1 ura3 Dlhs1::TRP1 <br />   JTY32                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1 Dire1::kanMX4 <br />   JTY33                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1 Dlhs1::kanMX4 <br />   JTY34                  MATa/MATa ade2-1/ade2-1 CAN1-100/CAN1-100 his3-11,15/his3-11,15 leu2-3 112/leu2-3 112 trp1-1/trp1-1 ura3-1/ <br />                          ura3-1Dire1::kanMX4/+ Dlhs1::kanMX4/+ [pRC43] <br />   JTY38                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1 Dire1::kanMX4 Dlhs1::kanMX4 <br />   JTY62                  MATa ade2 ade3 his3 leu2 trp1 ura3 Dire1::kanMX4 Dlhs1::kanMX4 [pJT40] <br />   JTY63                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1Dsil1::kanMX4 <br />   JTY64                  MATa/MATa ade2-1/ade2-1 CAN1-100/CAN1-100 his3-11,15/his3-11,15 leu2-3 112/leu2-3 112 trp1-1/trp1-1 ura3-1/ <br />                          ura3-1Dsil1::kanMX4/+ Dlhs1::kanMX4/+ <br />   JTY65                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1Dsil1::kanMX4 Dlhs1::kanMX4 [pRC43] <br />   JTY66                  MATa ade2-1 CAN1-100 his3-11,15 leu2-3 112 trp1-1 ura3-1Dsil1::kanMX4 Dlhs1::kanMX4 [pJT44] <br /> Escherichia coli <br />   DH5a                   supE44 DlacU169 (f80lacZM15) hsdR17 recA1 endA1 gryA96 thi-1 relA1 <br />   BL21(DE3)pLysS         F&#194;&#177;, ompT, hsdSB (rB&#194;&#177;, mB&#194;&#177;) dcm, gal, (DE3) pLysS, Cmr <br />  <br />  <br />  <br />  <br /> respectively. Diploids sporulated dissected produce           4 kbp SacI&#194;&#177;BamHI fragment cloning pRS423 the <br /> plasmid-containing haploid derivatives JTY38 JTY25.                        SacI&#194;&#177;BamHI sites. pJT43 (2m, HIS3, SIL1) constructed ligation of <br />    As &#194;&#174;rst step creating JTY62 {ade2 ade3 Dire1 Dlhs1 [pJT40 (2m),        2.17 kbp XhoI fragment pSIL1 containing SIL1 single <br /> URA3, ADE3, LHS1]}, CH1305 (Dr D.Sweet) mated JTY33,                  XhoI site pRS423. pJT44 (CEN6, ARS4, HIS3, MET3p-LHS1trun) a <br /> and sporulation subsequent tetrad dissection haploid progeny        single copy yeast vector based pRS313 containing MET3 regulatable <br /> with ade2 ade3 Dlhs1::kanMX4 genotype identi&#194;&#174;ed                  allele truncated LHS1 gene. As &#194;&#174;rst stage construction of <br /> geneticin resistance white colony colour YPD medium. This             pJT44, ~700 bp SalI&#194;&#177;EcoRV fragment pHAM8 (Mountain and <br /> strain transformed pJT40 heterozygous diploid              Korch, 1991) containing MET3 promoter ligated pRS315, <br /> created mating haploid strain containing Dire1::kanMX4             cut HindIII, blunted Klenow &#194;&#174;ll-in cut <br /> allele. This diploid strain sporulated and, tetrad dissection,   SalI. This cut BamHI, 3265 bp LHS1-containing <br /> JTY62 isolated.                                                            BamHI fragment ligated in. The LHS1-containing 3265 bp BamHI <br />    The SIL1 gene deleted JTY19 replacement                  fragment obtained &#194;&#174;rst cloning 3261 bp PstI&#194;&#177;SalI LHS1 <br /> Dyol031c::kanMX4 allele obtained Eurofan project carried           fragment pRC43 PstI&#194;&#177;SalI-digested pRS315 digesting <br /> the plasmid pYORC_YOL031c create JTY63. The pYORC_YOL031c                   resulting vector BamHI. After joining MET3 promoter <br /> plasmid digested NotI transformed JTY19 followed          LHS1-containing fragments, excised single 3823 bp <br /> growth YPD medium containing 2 mg/ml geneticin select                SalI&#194;&#177;NotI fragment ligated SalI&#194;&#177;NotI-digested pRS313. This <br /> integration events. Correct replacement con&#194;&#174;rmed using diagnostic          vector digested SmaI remove 12 bp palindromic <br /> PCRs. The heterozygous Dlhs1::kanMX4/+ Dsil1::kanMX4/+ diploid,                sequence MET3 promoter LHS1 gene, step required <br /> JTY64, created mating JTY33 JTY63. JTY65 (Dlhs1, Dsil1,            allow methionine-regulatable expression Lhs1p. To create a <br /> [pRC43]) isolated sporulation tetrad dissection JTY64 <br />                                                                                truncated allele LHS1 (amino acids 1&#194;&#177;835 plus alanine residue), <br /> transformed pRC43. JTY66 (Dlhs1, Dsil1, [pJT44]) created by <br />                                                                                pRC44 (Craven et al., 1996) digested HindIII followed by <br /> transforming pJT44 (HIS3, MET3p-LHS1trun) JTY65 (Dlhs1, Dsil1, <br />                                                                                Klenow &#194;&#174;ll-in re-circularization. A 1239 bp BamHI&#194;&#177;Bsu36I fragment <br /> [pRC43]) counter-selecting pRC43 5-FOA containing minimal <br />                                                                                containing truncated allele excised vector, puri&#194;&#174;ed and <br /> medium lacking methionine. <br />                                                                                used replace 1235 bp BamHI&#194;&#177;Bsu36I fragment the <br /> Plasmids nucleic acid manipulation                                         MET3p&#194;&#177;LHS1 allele-containing vector, generating pJT44. JT45 is <br /> The E.coli&#194;&#177;S.cerevisiae shuttle vectors pRS423 (2m, HIS), pRS313               derivative pGEX-4T (Invitrogen) expressing Sil1p (30&#194;&#177;413) <br /> (CEN6, ARS4, HIS), pRS315 (CEN6, ARS4, LEU2) YEplac181 (2m,                N-terminal GST-tagged protein. The SIL1 fragment generated by <br /> LEU2) described (Gietz Sugino, 1988; Sikorski Hieter,        PCR using primers 5&#194;&#162;-CGGGATCCACAATATTGCATTCATCCA- <br /> 1989; Christianson et al., 1992). The plasmid pJT30 (CEN, URA3)                TAC-3&#194;&#162; 5&#194;&#162;-GCGTCGACGCCAAAGATCAAGTGTCTGC-3&#194;&#162;. This <br /> contains UPRE&#194;&#177;lacZ reporter cassette used measuring UPR activity         digested BamHI SalI cloned corresponding <br /> (Wilkinson et al., 2000); pPS177 (2m, TRP1) encoding wild-type SCJ1            sites pGEX-4T. pJT46 derivative pET30a (Novagen) expressing <br /> (Blumberg Silver, 1991); pRC42 (CEN, HIS3) encoding wild-type              Kar2p (39&#194;&#177;674) His-tagged protein, His-Kar2p. The KAR2 fragment <br /> LHS1; pRC43 (2m, URA3) encoding wild-type LHS1 (Craven                generated PCR using primers 5&#194;&#162;-CGGGATCCGTTA- <br /> et al., 1996) described previously. pJT40 (2m, URA3, ADE3,           GAGGTGCCGATGATG-3&#194;&#162; 5&#194;&#162;-GCGTCGACTTCGTCGTCATAA- <br /> LHS1) constructed ligation 3.7 kbp BamHI fragment             TCAGCG-3&#194;&#162;. This digested BamHI SalI cloned <br /> pBamADE3 (Dr D.Sweet) containing ADE3 gene single                 corresponding sites pET30a. pJT47 derivative pET30a <br /> BamHI site pRC43. pJT41 (2m, HIS3, KAR2) constructed                 (Novagen) expressing Kar2p (39&#194;&#177;425) His-tagged protein, His- <br /> ligation 2.8 kbp XhoI fragment carrying KAR2 pMR109 (Dr              Kar2pATPase. The KAR2 fragment generated PCR using the <br /> J.Leighton) single XhoI site pRS423. pJT42 (2m, HIS3, SCJ1)        primers 5&#194;&#162;-CGGGATCCGTTAGAGGTGCCGATGATG-3&#194;&#162; 5&#194;&#162;-GCG- <br /> was constructed &#194;&#174;rst cloning 4 kbp KpnI fragment containing SCJ1          TCGACTAAGACACCAGCTTGAACG-3&#194;&#162;. This digested with <br /> from pPS177 single KpnI site YEplac181, followed removal        BamHI SalI cloned corresponding sites pET30a. <br />  <br />  <br /> 6450 <br />                                                                                         Roles IRE1 SIL1 adaptation loss Lhs1p <br />  <br /> Immunoblotting                                                                immunoprecipitations, CPY DPAPB antibodies used 1 ml <br /> Whole yeast extracts prepared glass bead lysis SDS sample          OD600 radiolabelled cells. <br /> buffer cultures grown mid-log phase, resolved SDS&#194;&#177;PAGE, <br /> transferred nitrocellulose membrane (HybondC, Amersham)              b-galactosidase assays <br /> probed reagent antisera essentially described previously (Stirling    b-galactosidase assays performed strains transformed pJT30 <br /> et al., 1992).                                                                (CEN, URA3, UPRE-lacZ) previously described (Wilkinson et al., <br />                                                                               2000). <br /> Radiolabelling immunoprecipitation <br /> Strains grown 30&#194;&#176;C YNB medium containing 1 mM <br /> (NH4)2SO4 required supplements. For pulse labelling strains <br /> with [35S]methionine, 5 OD600 equivalents cells pelleted,             Acknowledgements <br /> resuspended 500 ml culture supernatant, 100 mCi                  We thank P.A.Silver (Harvard), J.Leighton (Dubendorf), J.Winther <br /> [35S]methionine added prior 5 min incubation 30&#194;&#176;C. For pulse          (Carlsberg Research Centre) D.Sweet (MRC Cambridge) for <br /> labelling strains methionine gene repression experiments, 5 OD600       providing various plasmids, strains antibodies. This work was <br /> equivalents cells pelleted, resuspended 150 ml 23 YNB, 2%       supported The Wellcome Trust. <br /> glucose 150 ml 0.1 mCi/ml [14C]L-amino acid mix (NEN) added <br /> prior 5 min incubation 30&#194;&#176;C. Labelling reactions stopped by <br /> addition equal volume ice-cold 20 mM NaN3, incubation on <br /> ice 5 min. Cells pelleted resuspended 1 ml sphaeroplast    References <br /> buffer (1.4 M sorbitol, 50 mM Tris&#194;&#177;HCl pH 7.4, 2 mM MgCl2, 10 mM <br />                                                                               Baxter,B.K., James,P., Evans,T. Craig,E.A. (1996) SSI1 encodes a <br /> NaN3), followed addition yeast lytic enzyme (ICN) a <br />                                                                                 novel Hsp70 Saccharomyces cerevisiae endoplasmic reticulum. <br /> concentration 1.5 U/OD600 incubation 30&#194;&#176;C 30 min. <br />                                                                                 Mol. Cell. Biol., 16, 6444&#194;&#177;6456. <br /> Sphaeroplasts pelleted lysed resuspension 200 ml IP <br /> lysis buffer (50 mM Tris&#194;&#177;HCl pH 7.4, 1 mM EDTA, 1% SDS)                   Bertolotti,A., Zhang,Y., Hendershot,L.M., Harding,H.P. Ron,D. <br /> heating 95&#194;&#176;C 5 min. After 2 min cooling step ice, 800 ml IP      (2000) Dynamic interaction BiP ER stress transducers the <br /> buffer (187.7 mM NaCl, 62.5 mM Tris&#194;&#177;HCl pH 8.0, 6.25 mM EDTA,                   unfolded-protein response. Nature Cell Biol., 2, 326&#194;&#177;332. <br /> 1.25% Triton X-100) added sample followed 50 ml             Blumberg,H. Silver,P.A. (1991) A homologue bacterial heat- <br /> insoluble protein A suspension (Sigma). Samples cleared rotation        shock gene DnaJ alters protein sorting yeast. Nature, 349, <br /> at 4&#194;&#176;C 30 min, insoluble protein A pelleted 5 min     627&#194;&#177;630. <br /> microcentrifugation. Supernatants transferred clean tube        Boisrame,A., Beckerich,J.M. Gaillardin,C. (1996) Sls1p, an <br /> 1 ml appropriate antibody added OD600 cells used followed         endoplasmic reticulum component, involved protein <br /> by incubation room temperature rotation 1 h. After this, 50 ml      translocation process yeast Yarrowia lipolytica. J. Biol. <br /> of protein A&#194;&#177;Sepharose beads (20% w/v suspension IP buffer)              Chem., 271, 11668&#194;&#177;11675. <br /> added samples rotated room temperature hour.        Boisrame,A., Kabani,M., Beckerich,J.M., Hartmann,E. Gaillardin,C. <br /> The beads pelleted microfuge washed times         (1998) Interaction Kar2p Sls1p required ef&#194;&#174;cient co- <br /> 1 ml IP buffer followed dissociation antigen addition 50 ml      translational translocation secreted proteins yeast Yarrowia <br /> of 23 SDS&#194;&#177;PAGE sample buffer heating 95&#194;&#176;C 5 min. For                 lipolytica. J. Biol. Chem., 273, 30903&#194;&#177;30908. <br /> sequential immunoprecipitation, bead-free supernatant taken       Broughton,J., Swennen,D., Wilkinson,B.M., Joyet,P., Gaillardin,C. and <br /> cleared insoluble protein A 15 min pelleting, transfer     Stirling,C.J. (1997) Cloning SEC61 homologues from <br /> clean tube addition second antibody.                                     Schizosaccharomyces pombe Yarrowia lipolytica reveals the <br />                                                                                 extent functional conservation core component the <br /> GST pull-down experiments                                                       ER translocation machinery. J. Cell Sci., 110, 2715&#194;&#177;2727. <br /> GST (GSTp GST&#194;&#177;SIL1p) His-tagged (His-Kar2p His-                     Chen,X., Easton,D., Oh,H.J., Lee-Yoon,D.S., Liu,X. Subjeck,J. <br /> Kar2pATPase) proteins expressed BL21(DE3) pLysS (Novagen)               (1996) The 170 kDa glucose regulated stress protein large <br /> by induction 0.4 mM isopropyl-b-D-thiogalactopyranoside (IPTG)             HSP70-, HSP110-like protein endoplasmic reticulum. FEBS <br /> for 4 h. Crude extracts freeze&#194;&#177;thaw cycles GST         Lett., 380, 68&#194;&#177;72. <br /> binding buffer [20 mM HEPES pH 7.4, 100 mM KCl, 5 mM MgCl2,                   Cherest,H., Nguyen,N.T. Surdin-Kerjan,Y. (1985) Transcriptional <br /> 0.1% NP-40, 2% glycerol, 1 mM DTT, 1 mM EDTA, 1 mg/ml                           regulation MET3 gene Saccharomyces cerevisiae. Gene, 34, <br /> aminoethylbenzene sulfonyl &#194;&#175;uoride (AEBSF), 2 mg/ml E64, 1 mg/ml                269&#194;&#177;281. <br /> aprotinin, 1 mg/ml pepstatin A 1 mg/ml chemostatin] cleared        Christianson,T.W., Sikorski,R.S., Dante,M., Shero,J.H. Hieter,P. <br /> centrifugation 17 000 g 30 min. Bindings glutathione&#194;&#177;agarose          (1992) Multifunctional yeast high-copy-number shuttle vectors. Gene, <br /> were performed adding 100 ml crude extract 40 ml                    110, 119&#194;&#177;122. <br /> glutathione&#194;&#177;agarose (50% slurry) (Sigma) followed incubation            Cox,J.S. Walter,P. (1996) A novel mechanism regulating activity <br /> 4&#194;&#176;C 1 h. The agarose beads washed times 100 ml         transcription factor controls unfolded protein response. <br /> of GST binding buffer incubation 100 ml crude extract            Cell, 87, 391&#194;&#177;404. <br /> from strains expressing His-Kar2p His-Kar2pATPase 1 h        Cox,J.S., Shamu,C.E. Walter,P. (1993) Transcriptional induction of <br /> 4&#194;&#176;C. Following washes GST binding buffer, bound          genes encoding endoplasmic reticulum resident proteins requires a <br /> samples eluted SDS sample buffer 95&#194;&#176;C. Samples             transmembrane protein kinase. Cell, 73, 1197&#194;&#177;1206. <br /> analysed SDS&#194;&#177;PAGE using 10% polyacrylamide gels Coomassie              Craven,R.A., Egerton,M. Stirling,C.J. (1996) A novel Hsp70 the <br /> Blue staining.                                                                  yeast ER lumen required ef&#194;&#174;cient translocation number <br />                                                                                 protein precursors. EMBO J., 15, 2640&#194;&#177;2650. <br /> Antibodies                                                                    Craven,R.A., Tyson,J.R. Stirling,C.J. (1997) A novel subfamily of <br /> Lhs1p Kar2p antiserum raised using His6-tagged C-terminal               Hsp70s endoplasmic reticulum. Trends Cell Biol., 7, 277&#194;&#177;282. <br /> portions Lhs1p (residues 504&#194;&#177;834) Kar2p (residues 494&#194;&#177;652). The        Dierks,T. et al. (1996) A microsomal ATP-binding protein involved in <br /> fusion proteins protein fragments expressed using           ef&#194;&#174;cient protein transport mammalian endoplasmic reticulum. <br /> pET16b- (Novagen) derived expression vectors described previously            EMBO J., 15, 6931&#194;&#177;6942. <br /> for His-tagged proteins (Wilkinson et al., 1996). These antigens    Gietz,R.D. Sugino,A. (1988) New yeast&#194;&#177;Escherichia coli shuttle <br /> used inoculate sheep Diagnostics Scotland (Lanark, Scotland).             vectors constructed vitro mutagenized yeast genes lacking six- <br /> Antibodies CPY (residues 155&#194;&#177;525), a-factor (residues 10&#194;&#177;120)            base pair restriction sites. Gene, 74, 527&#194;&#177;534. <br /> DPAPB (residues 200&#194;&#177;642) raised laboratory using        Hamilton,T.G. Flynn,G.C. (1996) Cer1p, novel Hsp70-related <br /> same His-tagging approach. The following antibodies used            protein required posttranslational endoplasmic reticulum <br /> dilutions indicated parentheses immunoblotting: a-factor (sheep,         translocation yeast. J. Biol. Chem., 271, 30610&#194;&#177;30613. <br /> 1:10 000; Stirling Lab), (CPY, 1:10 000; Stirling Lab), Lhs1p (sheep,         Hamilton,T.G., Norris,T.B., Tsuruda,P.R. Flynn,G.C. (1999) Cer1p <br /> 1:30 000; study), Kar2p (sheep, 1:30 000; study), PDI                 functions molecular chaperone endoplasmic reticulum of <br /> (rabbit, 1:30 000; Dr Jakob Winther, Carlsberg Reasearch Centre),               Saccharomyces cerevisiae. Mol. Cell. Biol., 19, 5298&#194;&#177;5307. <br /> peroxidase-conjugated goat anti-rabbit IgG (1:10 000; Sigma)              Hamman,B.D., Hendershot,L.M. Johnson,A.E. (1998) BiP maintains <br /> peroxidase-conjugated rabbit anti-sheep IgG (1:10 000; Daco). For               permeability barrier ER membrane sealing lumenal <br />  <br />  <br />                                                                                                                                                6451 <br />  J.R.Tyson C.J.Stirling <br />  <br />  <br />   end translocon pore early translocation. Cell, 92,     Stirling,C.J. (1999) Protein targeting endoplasmic reticulum in <br />   747&#194;&#177;758.                                                                       yeast. 1997 Fleming Lecture. Microbiology, 145, 991&#194;&#177;998. <br /> Johnston,M. et al. (1994) Complete nucleotide sequence Saccharo-           Stirling,C.J., Rothblatt,J., Hosobuchi,M., Deshaies,R. Schekman,R. <br />   myces cerevisiae chromosome VIII. Science, 265, 2077&#194;&#177;2082.                     (1992) Protein translocation mutants defective insertion of <br /> Kabani,M., Boisrame,A., Beckerich,J.M. Gaillardin,C. (2000) A                integral membrane proteins endoplasmic reticulum. Mol. Biol. <br />   highly representative two-hybrid genomic library yeast                 Cell, 3, 129&#194;&#177;142. <br />   Yarrowia lipolytica. Gene, 241, 309&#194;&#177;315.                                    Travers,K.J., Patil,C.K., Wodicka,L., Lockhart,D.J., Weissman,J.S. and <br /> Knop,M., Finger,A., Braun,T., Hellmuth,K. Wolf,D.H. (1996) Der1,             Walter,P. (2000) Functional genomic analyses reveal essential <br />   novel protein speci&#194;&#174;cally required endoplasmic reticulum                 coordination unfolded protein response ER- <br />   degradation yeast. EMBO J., 15, 753&#194;&#177;763.                                    associated degradation. Cell, 101, 249&#194;&#177;258. <br /> Kohno,K., Normington,K., Sambrook,J., Gething,M.J. Mori,K.                Umebayashi,K., Hirata,A., Horiuchi,H., Ohta,A. Takagi,M. (1999) <br />   (1993) The promoter region yeast KAR2 (BiP) gene contains               Unfolded protein response-induced BiP/Kar2p production protects cell <br />   regulatory domain responds presence unfolded proteins         growth accumulation misfolded protein aggregates the <br />   endoplasmic reticulum. Mol. Cell. Biol., 13, 877&#194;&#177;890.                   yeast endoplasmic reticulum. Eur. J. Cell Biol., 78, 726&#194;&#177;738. <br /> Koshland,D., Kent,J.C. Hartwell,L.H. (1985) Genetic analysis       Wach,A., Brachat,A., Pohlmann,R. Philippsen,P. (1994) New <br />   mitotic transmission minichromosomes. Cell, 40, 393&#194;&#177;403.                    heterologous modules classical PCR-based gene disruptions in <br /> Kuwabara,K. et al. (1996) Puri&#194;&#174;cation characterization novel            Saccharomyces cerevisiae. Yeast, 10, 1793&#194;&#177;1808. <br />   stress protein, 150-kDa oxygen-regulated protein (ORP150),         Welihinda,A.A. Kaufman,R.J. (1996) The unfolded protein response <br />   cultured rat astrocytes expression ischemic mouse brain.            pathway Saccharomyces cerevisiae. Oligomerization trans- <br />   J. Biol. Chem., 271, 5025&#194;&#177;5032.                                                phosphorylation Ire1p (Ern1p) required kinase activation. <br /> Lyman,S.K. Schekman,R. (1995) Interaction BiP                    J. Biol. Chem., 271, 18181&#194;&#177;18187. <br />   Sec63p required completion protein translocation     Wilkinson,B.M., Critchley,A.J. Stirling,C.J. (1996) Determination of <br />   ER Saccharomyces cerevisiae. J. Cell Biol., 131, 1163&#194;&#177;1171.                 transmembrane topology yeast Sec61p, essential component <br /> Lyman,S.K. Schekman,R. (1997) Binding secretory precursor                 endoplasmic reticulum translocation complex. J. Biol. Chem., <br />   polypeptides translocon subcomplex regulated BiP. Cell, 88,         271, 25590&#194;&#177;25597. <br />   85&#194;&#177;96.                                                                      Wilkinson,B.M., Esnault,Y., Craven,R.A., Skiba,F., Fieschi,J., Kepes,F. <br /> Matlack,K.E., Misselwitz,B., Plath,K. Rapoport,T.A. (1999) BiP               Stirling,C.J. (1997) Molecular architecture ER translocase <br />   acts molecular ratchet posttranslational transport prepro-      probed chemical crosslinking Sss1p complementary <br />   factor ER membrane. Cell, 97, 553&#194;&#177;564.                            fragments Sec61p. EMBO J., 16, 4549&#194;&#177;4559. <br /> Mori,K., Ma,W., Gething,M.J. Sambrook,J. (1993) A trans-                  Wilkinson,B.M., Tyson,J.R., Reid,P.J. Stirling,C.J. (2000) Distinct <br />   membrane protein cdc2+/CDC28-related kinase activity                 domains yeast sec61p involved post-translational <br />   required signaling ER nucleus. Cell, 74, 743&#194;&#177;756.          translocation protein dislocation. J. Biol. Chem., 275, 521&#194;&#177;529. <br /> Mori,K., Kawahara,T., Yoshida,H., Yanagi,H. Yura,T. (1996) <br />                                                                                       Received August 15, 2000; revised October 16, 2000; <br />   Signalling endoplasmic reticulum nucleus: transcription <br />                                                                                                   accepted October 18, 2000 <br />   factor basic-leucine zipper motif required unfolded <br />   protein-response pathway. Genes Cells, 1, 803&#194;&#177;817. <br /> Mountain,H.A. Korch,C. (1991) TDH2 linked MET3 on <br />   chromosome X Saccharomyces cerevisiae. Yeast, 7, 873&#194;&#177;880. <br /> Ng,D.T., Brown,J.D. Walter,P. (1996) Signal sequences specify the <br />   targeting route endoplasmic reticulum membrane. J. Cell Biol., <br />   134, 269&#194;&#177;278. <br /> Normington,K., Kohno,K., Kozutsumi,Y., Gething,M.J. Sambrook,J. <br />   (1989) S.cerevisiae encodes essential protein homologous in <br />   sequence function mammalian Bip. Cell, 57, 1223&#194;&#177;1236. <br /> Rad,M.R., Habbig,B., Jansen,G., Hattenhorst,U., Kroll,M. and <br />   Hollenberg,C.P. (1997) Analysis DNA sequence a <br />   34,038 bp region left arm yeast chromosome XV. Yeast, <br />   13, 281&#194;&#177;286. <br /> Raynes,D.A. Guerriero,V.,Jr (1998) Inhibition Hsp70 ATPase <br />   activity protein renaturation novel Hsp70-binding protein. <br />   J. Biol. Chem., 273, 32883&#194;&#177;32888. <br /> Raynes,D.A. Guerriero,V. (2000) Isolation characterization of <br />   isoform <br /> </body></html>