<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Billerbeck Panke Microbial Cell Factories 2012, 11:110 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  TECHNICAL NOTES                                                                                                                               Open Access <br />  <br /> A genetic replacement selection-based <br /> engineering essential proteins <br /> Sonja Billerbeck Sven Panke* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Essential genes represent core biological functions required viability. Molecular <br />   understanding essentiality design synthetic cellular systems includes engineering essential <br />   proteins. An impediment effort lack growth-based selection systems suitable directed evolution <br />   approaches. <br />   Results: We established simple strategy genetic replacement essential gene (library of) variant(s) <br />   transformation. <br />   The validated using different essential genes plasmid combinations reproducibly <br />   shows transformation efficiencies order 107 transformants microgram DNA identifiable <br />   false positives. This allowed reliable recovery functional variants 105-fold excess of <br />   non-functional variants. This outperformed selection conventional bleach-out strains orders of <br />   magnitude, recombination functional non-functional variants interfered reliable recovery <br />   recA negative strains. <br />   Conclusions: We propose selection extremely suitable evaluating large libraries of <br />   engineered essential proteins resulting reliable isolation functional variants clean strain background <br />   readily used vivo applications expression purification use in vitro studies. <br />  <br />  <br /> Background                                                                           proteins complicated knock-outs cause le- <br /> About percent E. coli genes essential                           thality. This means phenotypes engineered proteins <br /> cell [1]. Essential genes particular scientific                      easily evaluated vivo suitable clean strain <br /> as encode proteins required important biological                            backgrounds available. <br /> functions, building minimal core cellular                                In decade approaches investi- <br /> viability tends conserved species.                                gated identify essential genes study func- <br /> Knowledge essential genes protein pro-                               tion vivo conditional elimination protein <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> ducts important drug design [2,3], biotechnological                           cell. This achieved triggering interference of <br /> applications [4], minimal genome approaches [5-8] and,                            synthesis target protein transcrip- <br /> general, crucial understanding engineering                               tional translational level [10-15]. However, these <br /> basic cellular functions required life [9]. While                            &#226;&#8364;&#339;bleach-out&#226;&#8364;? methods rely conditional protein elimin- <br /> construction Keio-collection, collection single                          ation elimination target gene itself and <br /> gene knock-outs E. coli [1], enormously facilitated                           retain wild-type copy essential gene the <br /> systematic investigation physiology E. coli                        cell. This sets limitations utility systems as <br /> as protein strain-engineering approaches,                            ready-<span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>-use selection systems directed evolution <br /> restricted non-essential genes protein pro-                             experiments recombination library members with <br /> ducts. Engineering approaches involving essential genes                              chromosomal wild-type gene mutations sys- <br />                                                                                      tem regulating expression wild-type protein can <br />                                                                                      lead selection false positive variants. This par- <br /> * Correspondence: sven.panke@bsse.ethz.ch <br /> ETH Z&#195;&#188;rich, Department Biosystems Science Engineering (D-BSSE), <br />                                                                                      ticularly true using library small <br /> Mattenstrasse 26, 4058 Basel, Switzerland                                            fraction variants expected functional. In this <br />                                        &#194;&#169; 2012 Billerbeck Panke; licensee BioMed Central Ltd. This Open Access article distributed terms the <br />                                        Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, <br />                                        distribution, reproduction medium, provided original work properly cited. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                      Page 2 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> case recombination events preferentially selected     contains 18 bp I-SceI cleavage site allows <br /> functional library members, results selec-      conditional elimination pKOCOMP the <br /> tion effort turning laborious screen bona fide      presence I-SceI. I-SceI expressed helper <br /> functional library members.                                    plasmid pI-SceI control rhamnose- <br />    Besides evaluation large libraries, desire       inducible RhaSR/PrhaBAD-system [18]. A variant this <br /> during protein engineering essential genes re-        helper plasmid, pParaI-SceI, carries arabinose- <br /> place wild-type gene single engineered heter-      inducible regulatory AraC/ParaBAD [19] instead of <br /> ologous variant vivo functional studies      rhamnose-based system. After testing various set- <br /> construction specialized strains used       ups, promoters chosen exhibited high <br /> purify mutant protein free wild-type protein.           expression level presence rhamnose arabin- <br /> Phage P1-mediated transduction chromosomal                ose efficiently switched presence <br /> <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> strain expressing variant essen-     glucose catabolite repression [19,20]. It should <br /> tial gene plasmid current            noted specific set-up complementation <br /> method choice achieve genetic replacement          vector flexible long carries I-SceI site, <br /> essential target gene variant (e.g [16]). The knock-      easily introduced PCR, compatible <br /> out created complementing                pKD46, required &#206;&#187; red-mediated <br /> chromosomal gene loss plasmid-encoded version          knock-out target gene cured growth <br /> the essential gene.                                            42&#194;&#176;C. The completed subse- <br />    Although P1-transduction widely used, protocol       quent transformation pI-SceI pParaI-SceI. <br /> is time-consuming restricted variants a <br /> time efficiencies successful transductions low,      Validation characterization using the <br /> often requiring empirical testing proper phage         essential gene product adenylate kinase (Adk) <br /> concentration followed re-plating PCR-screening         The target gene chosen validation and <br /> for correct genotypes. Therefore, aim de-        characterization adk, encoding E. coli&#226;&#8364;&#8482;s <br /> velop general genetic set-up turns working        Adk, essential gene product required biosyn- <br /> essential genes engineering gene pro-         thesis purine ribonucleotides regulation of <br /> ducts straight-forward approach facile work-      intracellular nucleotide availability [1,21,22]. For comple- <br /> ing non-essential genes. Here, present simple        mentation, adk control Ptac inserted into <br /> transformation-based system. Establishment sys-         pKOCOMP, giving rise pKOCOMP-adk. Upon induc- <br /> tem begins chromosomal replacement             tion pKOCOMP-adk IPTG, chromosomal <br /> essential gene PCR-derived selection          copy adk replaced PCR-generated kanamycin <br /> marker [17], conjunction complementary               resistance cassette [17]. The genotype resulting <br /> vector-encoded version target. A central element        strain E. coli adk::kan [pKOCOMP-adk] confirmed <br /> of method complementation vector car-          PCR using primers binding chromosomal regions <br /> ries I-SceI nuclease recognition site       up- downstream adk-locus (Additional file 1: <br /> rapidly conditionally eliminated presence        Table S1 Additional file 2 Figure S1). To complete <br /> an I-SceI nuclease-expressing helper plasmid. During           selection system, strain adk::kan [pKOCOMP-adk] <br /> elimination complementation vector, cells           transformed helper plasmid pI-SceI. The result- <br /> made electrocompetent transformed vector-           ing strain adk::kan [pKOCOMP-adk; pI-SceI] grown <br /> encoded library specific variant essential         presence glucose maintenance repress <br /> gene. Thus, actual step gene exchange reduced        I-SceI nuclease production. However, cells were <br /> to single transformation plus preparation suitable      transferred glucose-free LB medium, I-SceI produc- <br /> knock-out strain.                                              tion apparently efficiently induced addition <br />                                                                rhamnose, colony-forming units recov- <br /> Results                                                        ered induction-8period 180 minutes <br /> Overview replacement                             (Figure 2A). Importantly, copy adk <br /> The selection flexibly assembled vari-        gene, cells remained viable generations, <br /> ous user-defined ways, discussed sim-       presumably mRNA protein depleted. <br /> plest version first. It relies vectors: knock-out   This expected linear DNA rapidly degraded by <br /> and complementation vector pKOCOMP helper              cellular exonucleases half-life range of <br /> plasmid pI-SceI (Figure 1A Table 1). Plasmid pKO-          minutes [23] proteins relatively stable <br /> COMP derivative pKD46 [17] encodes             half-lives range hours [24]. <br /> arabinose-inducible &#206;&#187; red recombination (genes            During intermediate period cells elec- <br /> &#206;&#178;, &#206;&#179; exo) essential gene interest. It    trocompetent washing H2O glycerol and <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 3 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 1 Overview transformation-based replacement essential genes. A: Plasmids required establishment selection <br />  B: General procedure: The essential target gene adk replaced antibiotic resistance cassette pKOCOMP-adk complements for <br />  chromosomal loss. Plasmid pKOCOMP conditionally eliminated co-expression I-SceI nuclease helper plasmid pI-SceI. During <br />  elimination, cells electrocompetent transformed variant library investigation evaluate functionality to <br />  select functional library members complement loss pKOCOMP. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                  Page 4 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> Table 1 Bacterial strains plasmids used study <br /> Bacterial strain plasmid               Characteristics                                                             Source <br /> Strains <br /> W3110                                     F- &#206;&#187;- rph-1 INV(rrnD, rrnE)                                                 [33], internal <br />                                                                                                                       strain collection <br /> DH10B                                     F- endA1 recA1 galE15 galK16 nupG rpsL &#206;&#8221;lacX74                              [34], internal <br />                                           &#206;&#166;80lacZ&#206;&#8221;M15 araD139 &#206;&#8221;(ara,leu)7697 mcrA &#206;&#8221;(mrr-hsdRMS-mcrBC) &#206;&#187;-              strain collection <br /> W3110 adk::kan                            Chromosomal adk replaced KmR cassette.                                 This work <br />                                           The strain viable adk deletion complemented. <br /> W3110 groE::kan                           Chromosomal groS groL replaced KmR cassette.                       This work <br />                                           The strain viable groE deletion complemented. <br /> W3110                                     Chromosomal secB gpsA replaced KmR cassette.                       This work <br /> secB-gpsA::kan                            The strain viable gpsA deletion complemented. <br /> SB&#206;&#8221;recA                                   lacIq rrnB3 &#206;&#8221;lacZ4787 hsdR514 &#206;&#8221; (araBAD) 567 &#206;&#8221; (rhaBAD)                     This work <br />                                           568 rph-1 groE::ParaBAD-groE-KmR recA::FRT <br /> Plasmids <br /> pKD46                                     ori pSC101ts, ApR. Encodes &#206;&#187; red recombination genes                        [1] <br />                                           &#206;&#179;, &#206;&#178; exo control arabinose-responsive promoter ParaBAD. <br /> pKOCOMP-adk                               pKOCOMP-derived vector adk control                         This work <br />                                           IPTG-responsive tac promoter <br /> pSEVA432                                  ori pBBR1, SpecR, multiple cloning site                                     provided by <br />                                                                                                                       Victor Lorenzo <br /> pCOMP-adk                                 pSEVA432 encoding adk control IPTG-responsive               This work <br />                                           tac promoter, contains I-SceI cleavage site <br /> pCOMP-ESL                                 pSEVA432 encoding groS groL control                           This work <br />                                           IPTG-responsive tac promoter, contains I-SceI cleavage site <br /> pCOMP-secB-gpsA                           pSEVA432 encoding natural secB-gpsA transcriptional <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>,           This work <br />                                           contains I-SceI cleavage site <br /> pSEVA132                                  ori pBBR1, ApR, multiple cloning site                                       provided by <br />                                                                                                                       Victor Lorenzo <br /> pSEVA132-adk                              pSEVA132 encoding adk control natural promoter             This work <br /> pSEVA132-adkStop                          pSEVA132-adk internal stop codon                                    This work <br /> pSEVA132-adkwatermark                     pSEVA132-adk peptide insertion position P140                  This work <br /> pSEVA132-groE                             pSEVA132 coding groS groL control                    This work <br />                                           natural promoter <br /> pSEVA132-groEStop                         pSEVA132-groE internal stop codon                                   This work <br />                  watermark <br /> pSEVA132-groE                             pSEVA132-groE peptide insertion site I301                     This work <br /> pSEVA132-secBgpsA                         pSEVA132 encoding secB gpsA control                  This work <br />                                           natural promoter <br /> pSEVA132- secBgpsAStop                    pSEVA132- secBgpsA internal stop codon                              This work <br /> pSEVA671                                  ori p15A, GmR, multiple cloning site                                        provided by <br />                                                                                                                       Victor Lorenzo <br /> pI-SceI                                   pSEVA671, I-SceI nuclease control rhamnose                This work <br />                                           inducible promoter PRha response regulators <br />                                           RhaS RhaR, derived rhammnose metabolizing <br />                                           transcriptional <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> E. coli. <br /> pParaI-SceI                               pI-SceI PrhaBAD regulators RhaS RhaR exchanged                 This work <br />                                           arabinose promoter ParaBAD regulator araC <br />  <br />  <br /> transformed various test plasmids (Table 1) order                   transformation efficiencies. Transformation efficiencies <br /> to determine transformation efficiencies fre-                       107 &#206;&#188;g-1 DNA routinely achieved. The system <br /> quency false positive variants. Vector pSEVA132-adk                      intensively characterized regarding frequency of <br /> encoding wild-type adenylate kinase control                       false positive variants arise recom- <br /> of natural promoter used determination                       bination library plasmid residual linearized <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 5 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 2 Characterization pKOCOMP-adk/pI-SceI pCOMP-adk/pParaI-SceI systems. A: Elimination pKOCOMP-adk from <br />  adk::kan rhamnose-induced expression I-SceI nuclease helper plasmid pI-SceI. Red arrow: Time point harvest competent <br />  cell preparation. The inlet gives better resolution elimination dynamics. The chromosomal adk-replacement established using <br />  pKOCOMP-adk. B: Elimination pCOMP-adk adk::kan arabinose-induced expression I-SceI nuclease helper plasmid <br />  pParaI-SceI. The chromosomal adk-replacement established using pKD46 pCOMP-adk. <br />  <br />  <br />  <br />  <br /> pKOCOMP-adk uninduced subpopula-                             transformed cell (Table 2). This demonstrates the <br /> tion cells maintained pKOCOMP (Table 2).                          importance careful management plasmid <br /> Recombination-based false positives tested                        elimination step. <br /> transformation pSEVA132-adkstop, harboring stop <br /> codon adk gene repaired recom-                    Flexibility replacement system <br /> bination allowing growth corresponding carrier                     To verify selection set al- <br /> cell. False positives incomplete induction                    ternative combinations, constructed pCOMP-adk. <br /> examined transformation vector                          This vector based pBBR1 ori expected <br /> pSEVA132. In cases false positive colonies                   copy number 10-20 cell [25], carrying Ptac <br /> detected I-SceI expression induced pKO-                      promoter-controlled adk gene, lac-repressor LacI, <br /> COMP eliminated competent cell prepar-                         I-SceI cleavage site. While previous comple- <br /> ation (Table 2), corresponding frequency                     mentation vector pKOCOMP-adk used temperature- <br /> than 2 x 10-4 false positives transformed cell. How-                  sensitive pSC101 ori 2-3 copies cell when <br /> ever, detectable frequency recombination                   grown 37&#194;&#176;C [26], increased copy number of <br /> events pKOCOMP eliminated                             pCOMP-adk allowed examination plas- <br /> transformation test plasmids tried                    mid elimination sufficiently efficient higher intra- <br /> to eliminate pKOCOMP solely growth non-                         cellular plasmid concentration. We used pCOMP-adk in <br /> permissive temperature. In case, I-SceI nucle-                 combination pKD46 replace adk kanamycin <br /> ase expression induced transformation                   cassette. After curing cells pKD46 growth 43&#194;&#176;C, <br /> plating arabinose-supplemented agar plates,                  resulting strain adk::kan [pCOMP-adk] trans- <br /> both plasmids simultaneously present                      formed pParaI-SceI used gene replacement <br /> cells short period time. Here il-                       described using arabinose induce I-SceI pro- <br /> legitimate events frequency 3 x 10-3                      duction. Despite higher copy number pCOMP, the <br />  <br />  <br /> Table 2 Transformation efficiencies frequency false positive variants Adk-specific selection systems <br />                   Strain                          I-SceI induction                            Number transformants <br />                                                 competent cell preparation                      (frequency CFU transformant) <br />                                                                                    pSEVA132-adk         pSEVA132-adkstop           pSEVA132 <br /> W3110 adk::kan [pKOCOMP-adk, pI-SceI]                        No                     20 200 &#194;&#177; 2900        66 &#194;&#177; 21 (~3x10-3)       28 &#194;&#177; 13 (~1x10-3) <br />                                                              Yes                    21 000 &#194;&#177; 6000           0 (&lt;2x10-4)            0 (&lt;2 x10-4) <br />                                                                                                                           -3 <br /> W3110 adk::kan [pCOMP-adk, pParaI-SceI]                      No                     32 100 &#194;&#177; 3700         61 &#194;&#177; 17 (2x10 )        35 &#194;&#177; 11 (1x10-3) <br />                                                              Yes                    28 000 &#194;&#177; 2200           0 (&lt;2x10-4)             0 (&lt;2x10-4) <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                     Page 6 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> plasmid rapidly eliminated cells       pCOMP-plasmid carrying adk (Figure 3). After that, <br /> the presence arabinose (Figure 2B). Probably       systems routinely showed transformation efficien- <br /> the faster on-set possibly higher expression         cies 106-107 colonies &#206;&#188;g-1 DNA transformed <br /> levels arabinose responsive promoter ParaBAD          positive control vectors pSEVA132-groE or <br /> [19,20], elimination-dynamics pCOMP/pParaI-SceI       pSEVA132-secBgpsA. No false positive variants be <br /> system faster pKOCOMP/pI-SceI         detected transformation test plasmids <br /> system. Although elimination data sys-        pSEVA132-groEstop pSEVA132-secBgpsAstop, con- <br /> tems difficult compare differences          structed analogy pSEVA132-adkstop before. Import- <br /> copy number promoters, results indicate       antly, general feature cells stay <br /> the described approach set multiple ways,       viable - judged doubling time comparison <br /> making easy adapt plasmid strategies spe-       non-induced control culture did produce <br /> cific purposes. Transformation efficiencies           I-SceI - generations, depending the <br /> pCOMP/pParaI-SceI (~107 colonies &#206;&#188;g-1 DNA)            target gene, loss complementing plasmid. <br /> were comparable pKOCOMP/pI-SceI                This important characteristic cells <br /> (Table 1) identify false positive           competent transformation library <br /> transformants.                                               variant essential gene period the <br />                                                              complementing plasmid lost no <br /> Generality                                     longer contribute recombination. <br /> To confirm utility limited <br /> to adk easily extended essential       Performance selection <br /> genes, constructed vivo selection systems    To test regarding performance reliably <br /> essential gene products: chaperonin GroEL        identifying functional variants large library without <br /> co-chaperonin GroES (encoded groE operon con-         contamination false positive variants challenged the <br /> taining genes groL groS), glycerol-3-            GroEL- Adk-specific selection mock li- <br /> phosphate dehydrogenase (GpsA encoded gpsA). For          braries various excesses non-functional variants. As <br /> establishment GroEL-specific intro-         conventional bleach-out available for <br /> duced I-SceI cleavage site vector pSEVA431-      GroEL (the E. coli MGM100 strain [10]), compared <br /> groE PCR giving rise vector pCOMP-groE. Plasmid        performance replacement developed here <br /> pSEVA431-groE encodes groE-operon control          bleach-out system. In strain MGM100, groE pro- <br /> of IPTG-inducible Ptac promoter. It harbors     moter replaced araBAD promoter the <br /> lacI gene, spectinomycin resistance cassette repli-    regulatory gene araC. The strain main- <br /> cates pBBR1 ori. Plasmid pCOMP-groE used          tained presence arabinose GroEL production <br /> in combination pKD46 replace chromosomal         chromosomal locus fully repressed the <br /> groE-operon kanamycin resistance cassette (Add-         presence glucose. This way, functional variants be <br /> itional file 2 Figure S1). The resulting strain groE::kan    selected library presence glucose. However, <br /> [pCOMP-groE] cured pKD46 43&#194;&#176;C           copy wild-type gene present selec- <br /> transformed helper plasmid pParaI-SceI.             tion, recombination lead selection false positive <br />    The GpsA-specific constructed cloning       variants, dominate stringent selection <br /> the natural gpsA transcriptional <span id='am-318' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (consisting genes   conditions. To improve strain regarding expected <br /> secB gpsA control natural promoter)       recombination frequency deleted recA leading the <br /> into vector pSEVA431 using primers encoding I-        strain SB&#206;&#8221;recA (Additional file 2: Figure S1). <br /> SceI restriction site, resulting vector pCOMP-               The created mock libraries contained decreasing ratios <br /> secBgpsA. Chromosomally encoded secB gpsA           functional non-functional gene variants 1:102 <br /> then replaced kanamycin resistance cassette using       1:105 (functional:non-functional) mimicking libraries <br /> pKD46 (Additional file 2 Figure S1). To complete         decreasing number variants expected be <br /> set-up, strain secBgpsA::kan [pCOMP-secBgpsA]            functional large pool non-functional variants, <br /> cured pKD46 transformed helper plasmid           typical e.g. libraries created error-prone <br /> pParaI-SceI. Both systems characterized regarding       PCR different error rates. As non-functional var- <br /> elimination dynamics complementing plasmids           iants used pSEVA132-groEstop pSEVA132-adkstop. <br /> pCOMP-groE pCOMP-secBgpsA I-SceI induc-            As functional variants used pSEVA132-groEwatermark <br /> tion, regarding transformation efficiencies    pSEVA132-adkwatermark. These plasmids encode a <br /> electrocompetent cells prepared pCOMP-                GroEL Adk-variant in-frame insertion a <br /> elimination frequency false positive variants. Both   short peptide previously identified permissive sites. <br /> pCOMP-type plasmids lost comparable rate        Both variants fully functional identified <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                                          Page 7 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />     Figure 3 Characterization pCOMP-groE/pParaI-SceI pCOMP-secBgpsA/pParaI-SceI system. Elimination (A) pCOMP-groE <br />     groE::kan (B) pCOMP-secBgpsA secBgpsA::kan presence arabinose helper plasmid pParaI-SceI. Elimination induced <br />     arabinose-induced expression I-SceI nuclease helper plasmid pParaI-SceI. Red arrow: Time point harvest competent cell <br />     preparation. <br />  <br />  <br /> by PCR using watermark-specific primers. To compare                                functional non-functional variants. <br /> the GroEL-specific selection GroEL                                 After plating incubation overnight LB-agar plates <br /> bleach-out SB&#206;&#8221;recA, electrocompetent W3110                                  supplemented arabinose, corresponding anti- <br /> groE::kan [pCOMP-groE, pParaI-SceI] cells electro-                              biotics, and, case W3110 groE::kan [pCOMP- <br /> competent SB&#206;&#8221;recA cells transformed dif-                             groE, pParaI-SceI], glucose, colonies scored a <br /> ferent mock libraries separate experiments. As                                  subset colonies genotyped watermark- <br /> positive negative controls, cells transformed                             specific primers (Table 3). <br />  <br />  <br /> Table 3 Recovery functional GroEL-variants Adk-variants mock libraries increasing excess of <br /> non-functional variants using established selection comparison conventional bleach-out system <br /> Strain                         Library (functional: non-functional)        Number colonies         Number genotyped           Number detected <br />                                                                               transformationa                    variants                false positive variants <br /> W3110 groE::kan                            Only functional                           ~ 27 000                           20                           0 <br /> [pCOMP-groE, pParaI &#226;&#8364;&#8220;SceI] <br />                                                  1:102                                  231                             20                           0 <br />                                                  1:103                                   33                             20                           0 <br />                                                  1:104                                   4                              4                            0 <br />                                                  1:105                                   1a                             1                            0 <br />                                          Only non-functional                             0                              0                            0 <br /> SB&#206;&#8221;recA                                    Only functional                           ~ 25 000                           20                           0 <br />                                                  1:102                                  253                             20                           0 <br />                                                  1:103                                   25                             20                           1 <br />                                                  1:104                                   7                              7                            5 <br />                                                  1:105                                   6                              6                            5 <br />                                          Only non-functional                             7                              7                            7 <br /> W3110 adk::kan                             Only functional                           ~ 46 000                           20                           0 <br /> [pCOMP-adk, pParaI-SceI] <br />                                                  1:102                                  459                             20                           0 <br />                                                  1:103                                   36                             20                           0 <br />                                                  1:104                                   3                              3                            0 <br />                                                  1:105                                   1                              1                            0 <br />                                          Only non-functional                             0                              0                            0 <br /> a            9 <br />  Typically, 10 cells used transformation, cases allowed isolate positive colony high-stringency case. In these <br /> cases, number transformed cells increased 3*109 cells. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                        Page 8 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />    In GroEL-specific system, numbers colonies         transformation-competent. Efficient plasmid elimination <br /> correlated numbers expected trans-            essential prerequisite high performance of <br /> formation efficiencies decreasing number              evaluation large libraries under <br /> functional variants library. Even          stringent conditions elimination wild-type <br /> stringent library (functional variants frequency         gene cells prior introduction variants pre- <br /> 10-5) functional variants carrying watermark            vents recombination-based background growth. The <br /> were recovered transformation ~ 81,000 cells. No        major shortcoming conventionally used bleach-out <br /> colony detected transformation approxi-             systems recombination silenced <br /> mately number cells non-                   wild-type gene selection. Even recA strain, <br /> functional variants. In contrast using SB&#206;&#8221;recA          RecA-independent recombination homologous <br /> the experiment, false positive variants                regions occurs frequencies 10-3-10-4 [27]. <br /> detected transformation negative control          As soon number expected functional variants <br /> (transformation non-functional variants)          library drops 10-3 selection primarily <br /> a frequency ~ 2 x 10-4. In addition, identified false      recombination events instead functional library <br /> positive variants genotyping subset col-          members. This clearly seen selection ex- <br /> onies selected different mock            periment using GroEL bleach-out strain SB&#206;&#8221;recA <br /> libraries (Table 3). Selection stringency positively corre-      mock libraries decreasing ratio functional <br /> lated false positive rate, requiring laborious     variants. Even strain deficient recA, <br /> orthogonal assays differentiate true false posi-         frequently isolated false positive variants. <br /> tives. In directed evolution experiment re-           Besides recombination-free, advantage of <br /> sult requirement intensive post-screening          presented set-up bleach-out systems that <br /> selected variants true functional library members.           selected variants directly expressed clean strain- <br /> Plasmids false positive variants,          background circumventing laborious post-transformation <br /> had isolated transformation non-            work P1-transductions. This allows subsequent <br /> functional variants, analyzed sequencing.        purification engineered proteins vitro <br /> For stop codon reverted              characterization user-defined vivo applications <br /> the wild-type codon, probably recombination               running risk wild-type contamination. <br /> with chromosomal groL copy. The analyzed                 We elimination com- <br /> variant stop codon groL indicating          plementing vector knock-out strain, cells re- <br /> that ability grow arisen muta-           main viable generations - depending on <br /> tion araBAD promoter preventing repression.          gene - electrocompetent cells prepared after <br /> This phenomenon described strain             elimination yield 107 transformants &#206;&#188;g-1 DNA. <br /> MGM100 [16].                                                       As set-up essential target genes are <br />    The experiment performed Adk-               expressed inducible promoter competent <br /> specific selection using mock libraries decreas-     cell preparation, appropriate bleach-out times be <br /> ing ratios functional Adk-variants non-functional          adjusted individual gene products tuning in- <br /> Adk-variants. Also set-up did identify       ducer concentration. <br /> false positive variant transformation           The validated different essential E. <br /> most stringent library reliably identify functional   coli proteins: Adk, glyceraldehyde-3-phosphate dehydro- <br /> variant containing watermark transformation            genase chaperonin GroEL. Transformation efficien- <br /> of ~ 138,000 cells (Table 3).                                    cies absence detectable recombination events <br />                                                                  proved independent essential target gene. <br /> Discussion <br /> In study present facile efficient set-up a <br /> (within tested boundaries) background-free selection         Conclusions <br /> system functional, engineered, essential proteins. It     In current work present straightforward, <br /> based conditional elimination complemen-             transformation-based enables genetic <br /> tary plasmid-based copy essential gene knock-         replacement wild-type essential gene a <br /> out strain order replace essential gene genes       library variant. It directly results isola- <br /> from library (for example).                                    tion functional variants clean strain background <br />   We I-SceI nuclease-based cleavage proves          considerably reduced effort. <br /> be suitable strategy fast efficient elimin-           Furthermore, substantially facilitates working and <br /> ation complementing plasmid knock-out                engineering essential genes protein products <br /> strain occur strain            making experimentally easy, fast scalable task. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                  Page 9 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />   Finally, possible adapt intro-    operon. groS amplified W3110 PCR with <br /> duced replacement strategy hosts &#226;&#8364;&#8220; like e.g.        primers groS-fw groS-rv groL PCR- <br /> yeast, Bacillus subtilis Clustridium spec. -    amplified primers groL-fw groL-rv se- <br /> homologous recombination-based knock-out strategies          quentially cloned KpnI HindIII sites of <br /> are available [28-30].                                       pACT3. Plasmid pSEVA132-groE constructed by <br />                                                              amplification natural groE operon E. coli <br /> Materials methods                                        genomic DNA using primers groE-forward groE- <br /> Chemical enzymes                                         reverse cloning unique sites XmaI and <br /> Restriction enzymes ligase obtained New        XbaI. Plasmid pCOMP-secBgpsA constructed clon- <br /> England Biolabs (Ipswich, MA, USA) used according        ing natural secB-gpsA transcriptional <span id='am-319' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, amplified <br /> to manufacturers&#226;&#8364;&#8482; instructions. Chemicals purchased     primers secBgpsA-forward gpsA_I-SceI-reverse, <br /> in highest purity available Sigma-Aldrich, Fluka    pSEVA431 restriction sites XmaI XbaI. <br /> (Buchs, Switzerland) Roth (Lauterbourg, France). Tryp-    Plasmids pSEVA132-adkstop, pSEVA132-adkwatermark , <br /> ton yeast extract BD Bioscience (Basel,        pSEVA132-groEstop,       pSEVA132-groEwatermark       and <br />                                                                                  stop <br /> Switzerland). Oligonucleotides Sanger-sequencing         pSEVA132-secBgpsA        constructed amplifica- <br /> service purchased Microsynth (Balgach,             tion re-ligation pSEVA132-adk, pSEVA132-groE <br /> Switzerland).                                                pSEVA132-secBgpsA using primers adk-stop-fw/ <br />                                                              adk-stop-rv, adk-watermark-fw/adk-watermark-rv, groE- <br /> Strains plasmids                                         watermark-fw/groE-watermark-rv, groE-stop-fw/groE- <br /> E. coli DH10B used general cloning procedures.       stop-rv secBgpsA-stop-fw/secBgpsA-stop-rv. <br /> E. coli W3110 used chassis chromo-          Helper plasmids pI-SceI pParaI-SceI derivatives <br /> somal deletions (see Table 1 overview strains      pSEVA671 (ori p15A, GmR). The gene I-SceI nu- <br /> used study). SB&#206;&#8221;recA derivative strain       clease amplified plasmid pSTKST [32] using <br /> MGM100 [10]. It constructed BW25113 recA::          primers I-SceI-fw I-SceI-rv cloned into <br /> FRT [1] P1-phage transduction using lysate         pSEVA671 PacI EcoRI restriction sites. The <br /> MGM100 selecting KmR. The final clone            RhaR-RhaS/PrhaBAD regulatory amplified <br /> confirmed PCR analysis groE recA::FRT          E. coli genomic DNA using primers Rha-forward <br /> locus inability grow glucose          Rha-reverse cloned I-SceI restric- <br /> GroEL-bleach-out. Plasmid pKOCOMP-adk deriva-           tion sites NsiI SpeI. The AraC/ParaBAD regulatory <br /> tive pKD46 [17] constructed cloning      amplified E. coli genomic DNA using <br /> the adk gene multiple cloning site expression    primers ParaBAD-fw ParaBAD-rv used ex- <br /> vector pACT3 [31] restriction sites BamHI Hin-       change RhaR-RhaS/Prha using sites NsiI SpeI. <br /> dIII using primers pKOCOMP-adk-fw pKOCOMP- <br /> adk-rv (see Additional file 1 Table S1 primer            Preparation competent cells transformation <br /> sequences). The resulting vector pACT-adk used        E. coli adk::kan [pKOCOMP-adk, pI-SceI] [pCOMP- <br /> template amplify Ptac promoter-controlled adk         adk, pParaI-SceI] grown overnight LB liquid <br /> gene primers pACT-forward pACT-reverse, en-         broth supplemented 50 &#206;&#188;g mL-1 kanamycin, 10 &#206;&#188;g <br /> coding I-SceI recognition site, cloned        mL-1 gentamicin, 100 &#206;&#188;M IPTG 0.5% (wt/vol) glu- <br /> pKD46 unique NcoI-site. For construction          cose (to efficiently repress I-SceI production) 30&#194;&#176;C or <br /> pCOMP-adk Ptac promoter-controlled adk gene          37&#194;&#176;C. Cells pelleted, washed LB and <br /> amplified primers pACT-SceI-Spe pACT-Pac            diluted 1:100 fresh LB broth, supplemented the <br /> and PCR product cloned unique SpeI          antibiotics glucose. Cells <br /> and PacI sites pSEVA432 (ori pBBR1, SpecR resist-         grown 37&#194;&#176;C. At OD600 0.2, 10 mM rham- <br /> ance). Plasmid pSEVA132-adk derivative               nose 0.2% arabinose (wt/vol) added induce I- <br /> pSEVA132 (ori pBBR1, ApR) encodes adk controlled         SceI nuclease production. When reaching OD600 of <br /> by natural promoter fused C-terminal            0.4-0.5, cells chilled ice 30 min, harvested <br /> 6xHis-tag. It amplified genomic E. coli DNA         washed twice chilled water 10% <br /> using primers adk-forward adk-reverse cloned         glycerol described [26]. For transformation the <br /> into pSEVA132 restriction sites XmaI SacI. Plas-     test plasmids mock libraries, 50 &#206;&#188;L cells (OD600 <br /> mid pCOMP-groE constructed amplification           100) mixed 1.5 ng DNA, exposed to <br /> the Ptac-controlled groE-operon pACT-ESL using          electrical pulse 1.3 kV recovered 1 mL LB <br /> primers pACT-SceI-Spe pACT-Pac cloned           broth supplemented 10 mM rhamnose 0.2% ara- <br /> the unique SpeI PacI sites pSEVA431. pACT-ESL         binose 1 h 37&#194;&#176;C. Selection overnight at <br /> is pACT3 derived encodes Ptac controlled groE        37&#194;&#176;C LB agar plates containing 50 &#206;&#188;g mL-1 <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                            Page 10 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> kanamycin, 10 &#206;&#188;g mL-1 gentamicin, 100 &#206;&#188;g mL-1 ampi-                  Authors&#226;&#8364;&#8482; contributions <br /> cillin 10 mM rhamnose 0.2% arabinose. The                     SB performed experiments, SB SP designed experimental <br />                                                                      approach wrote manuscript, SP supervised research. All authors <br /> GroEL-specific selection based W3110 groE::                read approved final manuscript. <br /> kan [pCOMP-groE, pParaI-SceI] GpsA-specific <br /> selection based W3110 secBgpsA::kan                        Acknowledgment <br /> [pCOMP-secBgpsA, pParaI-SceI] treated                  This work funded EU FP6 projects Eurobiosyn NANOMOT <br /> way. For preparation competent SB&#206;&#8221;recA, cells                ESF project Nanocell. The authors indebted Tania Roberts for <br />                                                                      proofreading manuscript. <br /> grown overnight LB supplemented 50 &#206;&#188;g mL-1 <br /> kanamycin 0.2% arabinose. Cells washed twice                Received: 13 June 2012 Accepted: 8 August 2012 <br /> with water diluted 1:100 fresh LB medium. When                Published: 16 August 2012 <br /> cells reached OD600 0.1 0.5% glucose added to <br /> repress chromosomal GroEL production. Cellular GroEL                 References <br /> was bleached generations cells har-              1. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br />                                                                          Tomita M, Wanner BL, Mori H: Construction Escherichia coli K-12 <br /> vested competent cell preparation (at OD600 0.4).                 in-frame, single-gene <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio collection. Mol Syst Biol <br />                                                                          2006, 2:2006&#226;&#8364;&#8220;0008. <br />                                                                      2. Rosamond J, Allsop A: Harnessing power genome the <br />                                                                          search new antibiotics. Science 2000, 287:1973&#226;&#8364;&#8220;1976. <br /> Determination plasmid loss                                        3. Haselbeck R, Wall D, Jiang B, Ketela T, Zyskind J, Bussey H, Foulkes JG, <br /> Loss pKOCOMP-adk pCOMP-adk, pCOMP-groE                             Roemer T: Comprehensive essential gene identification platform for <br /> and pCOMP-secBgpsA determined follows: The                        novel anti-infective drug discovery. Curr Pharm Des 2002, 8:1155&#226;&#8364;&#8220;1172. <br />                                                                      4. Johnson DBF, Xu JF, Shen ZX, Takimoto JK, Schultz MD, Schmitz RJ, Xiang Z, <br /> corresponding knock-out strains adk::kan, groE::kan                  Ecker JR, Briggs SP, Wang L: RF1 knockout allows ribosomal incorporation <br /> secBgpsA::kan containing helper plasmid pI-SceI                       unnatural amino acids multiple sites. Nat Chem Biol 2011, 7:779&#226;&#8364;&#8220;786. <br /> pParaI-SceI grown LB liquid broth supplemented               5. Gil R, Silva FJ, Pereto J, Moya A: Determination core minimal <br />                                                                          bacterial gene set. Microbiol Mol Biol Rev 2004, 68:518&#226;&#8364;&#8220;537. <br /> with 50 &#206;&#188;g mL-1 kanamycin 10 &#206;&#188;g mL-1 gentamicin                  6. Posfai G, Plunkett G 3rd, Feher T, Frisch D, Keil GM, Umenhoffer K, <br /> until exponential growth reached. Next, I-SceI nu-                   Kolisnychenko V, Stahl B, Sharma SS, Arruda M, et al: Emergent properties <br /> clease production induced 10 mM rhamnose                        reduced-genome Escherichia coli. Science 2006, 312:1044&#226;&#8364;&#8220;1046. <br />                                                                      7. Trinh CT, Unrean P, Srienc F: Minimal Escherichia coli cell most <br /> or 0.2% arabinose. After induction aliquots taken                   efficient production ethanol hexoses pentoses. Appl Environ <br /> after 0, 10, 20, 40, 60, 120 180 min normalized                  Microbiol 2008, 74:3634&#226;&#8364;&#8220;3643. <br /> to OD600. Serial dilutions subsequently plated               8. Lee JH, Sung BH, Kim MS, Blattner FR, Yoon BH, Kim JH, Kim SC: Metabolic <br />                                                                          engineering reduced-genome strain Escherichia coli for <br /> LB agar supplemented 0.5% glucose. The day                     L-threonine production. Microb Cell Fact 2009, 8:2. <br /> the number colony forming units (CFU) counted.                9. Danchin A: Natural selection immortality. Biogerontology 2009, 10:503&#226;&#8364;&#8220;516. <br /> The number colonies resulting aliquots                 10. McLennan N, Masters M: GroE vital cell-wall synthesis. Nature 1998, <br />                                                                          392:139&#226;&#8364;&#8220;139. <br /> had taken control culture grown ab-               11. Herring CD: Introduction conditional lethal amber mutations in <br /> sence rhamnose arabinose set 100% CFU.                     Escherichia coli. Methods Mol Biol 2008, 416:323&#226;&#8364;&#8220;334. <br />                                                                      12. Jin Y, Watt RM, Danchin A, Huang JD: Use riboswitch-controlled <br />                                                                          conditional hypomorphic <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>mutation</span> uncover role essential csrA <br />                                                                          gene bacterial autoaggregation. J Biol Chem 2009, 284:28738&#226;&#8364;&#8220;28745. <br /> Chromosomal knock-out essential genes                             13. Ji Y, Zhang B, Van SF, Horn, Warren P, Woodnutt G, Burnham MK, <br /> Knock-outs &#206;&#187; red-based recombination                        Rosenberg M: Identification critical staphylococcal genes using <br />                                                                          conditional phenotypes generated antisense RNA. Science 2001, <br /> with PCR-encoded kanamycin resistance cassette as <br />                                                                          293:2266&#226;&#8364;&#8220;2269. <br /> described earlier [17]. The kanamycin-cassettes                 14. Tang YC, Chang HC, Chakraborty K, Hartl FU, Hayer-Hartl M: Essential role of <br /> generated pKD13 template primers adk-                      chaperonin folding compartment vivo. EMBO J 2008, 27:1458&#226;&#8364;&#8220;1468. <br />                                                                      15. Herring CD, Blattner FR: Conditional lethal amber mutations essential <br /> H1 adk-H2, groE-H1 groE-H2 secBgpsA-H1 <br />                                                                          Escherichia coli genes. J Bacteriol 2004, 186:2673&#226;&#8364;&#8220;2681. <br /> and secBgpsA-H2. Knock-outs PCR-verified                   16. van der Vies SM, Lund PA: Determination chaperonin activity vivo. <br /> primers P1-P6 indicated Additional file 2 Figure S1                Methods Mol Biol 2000, 140:75&#226;&#8364;&#8220;96. <br />                                                                      17. Datsenko KA, Wanner BL: One-step inactivation chromosomal genes in <br /> and Additional file 1 Table S1. <br />                                                                          Escherichia coli K-12 using PCR products. Proc Natl Acad Sci U S A 2000, <br />                                                                          97:6640&#226;&#8364;&#8220;6645. <br />                                                                      18. Tobin JF, Schleif RF: Transcription rha operon psr promoter. J Mol <br /> Additional files                                                         Biol 1990, 211:1&#226;&#8364;&#8220;4. <br />                                                                      19. Guzman LM, Belin D, Carson MJ, Beckwith J: Tight regulation, modulation, <br />                                                                          high-level expression vectors containing arabinose PBAD <br />  Additional files file 1: Table S1. Primers used study.          promoter. J Bacteriol 1995, 177:4121&#226;&#8364;&#8220;4130. <br />  Additional files file 2: Figure S1. PCR verification knock   20. Egan SM, Schleif RF: A regulatory cascade induction rhaBAD. <br />  strains.                                                                J Mol Biol 1993, 234:87&#226;&#8364;&#8220;98. <br />                                                <br /> </body></html>