<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                            Research article <br />  <br />      Lysosomal disruption preferentially targets <br />     acute myeloid leukemia cells progenitors <br />         Mahadeo A. Sukhai,1 Swayam Prabha,1 Rose Hurren,1 Angela C. Rutledge,1 Anna Y. Lee,2 <br />        Shrivani Sriskanthadevan,1 Hong Sun,1 Xiaoming Wang,1 Marko Skrtic,1 Ayesh Seneviratne,1 <br />            Maria Cusimano,1 Bozhena Jhas,1 Marcela Gronda,1 Neil MacLean,1 Eunice E. Cho,1 <br />       Paul A. Spagnuolo,1 Sumaiya Sharmeen,1 Marinella Gebbia,2 Malene Urbanus,2 Kolja Eppert,1 <br />              Dilan Dissanayake,3 Alexia Jonet,4 Alexandra Dassonville-Klimpt,4 Xiaoming Li,1 <br />             Alessandro Datti,5,6 Pamela S. Ohashi,3 Jeff Wrana,5 Ian Rogers,5 Pascal Sonnet,4 <br />           William Y. Ellis,7 Seth J. Corey,8 Connie Eaves,9 Mark D. Minden,1 Jean C.Y. Wang,1,10 <br />                    John E. Dick,1 Corey Nislow,2 Guri Giaever,2 Aaron D. Schimmer1 <br />                          1Princess Margaret Hospital/the Ontario Cancer Institute, University Health Network, Toronto, Ontario, Canada. <br />           2Department  Molecular Genetics, Donnelly Centre Cellular Biomolecular Research, University Toronto, Toronto, Ontario, Canada. <br />                  3Campbell Family Institute Breast Cancer Research, Toronto, Ontario, Canada. 4Laboratoire des Glucides, CNRS FRE 3517, <br />  <br />              UFR Pharmacie, Universit&#195;&#169; Picardie Jules Verne, 1, Amiens, France. 5Samuel Lunenfeld Research Institute, Mount Sinai Hospital, <br />                 Toronto, Ontario, Canada. 6Department Experimental Medicine Biochemical Sciences, University Perugia, Perugia, Italy. <br />                      7Department Chemical Informatics, Division Experimental Therapeutics, Walter Reed Army Institute Research, <br />  <br />         Silver Spring, Maryland, USA. 8Departments Pediatrics Cell Molecular Biology, Northwestern University Feinberg School Medicine, <br />                       Chicago, Illinois, USA. 9Terry Fox Laboratory, British Columbia Cancer Agency, Vancouver, British Columbia, Canada. <br />                                            10Department Medicine, University Toronto, Toronto, Ontario, Canada. <br />  <br />  <br />  <br />  <br />     Despite efforts understand treat acute myeloid leukemia (AML), remains need com- <br />     prehensive therapies prevent AML-associated relapses. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> identify new therapeutic strategies AML, <br />     screened library on- off-patent drugs identified antimalarial agent mefloquine com- <br />     pound selectively kills AML cells AML stem cells panel leukemia cell lines mice. Using <br />     yeast genome-wide functional screen mefloquine sensitizers, identified genes associated the <br />     yeast vacuole, homolog mammalian lysosome. Consistent this, determined mefloquine <br />     disrupts lysosomes, directly permeabilizes lysosome membrane, releases cathepsins cytosol. <br />     Knockdown lysosomal membrane proteins LAMP1 LAMP2 resulted decreased cell viability, as <br />     did treatment AML cells known lysosome disrupters. Highlighting potential therapeutic rationale <br />     strategy, leukemic cells significantly larger lysosomes compared normal cells, leukemia- <br />     initiating cells overexpressed lysosomal biogenesis genes. These results demonstrate lysosomal disruption <br />     preferentially targets AML cells AML progenitor cells, providing rationale testing lysosomal disrup- <br />     tion novel therapeutic strategy AML. <br />  <br /> Introduction                                                                            previously reported, impact disrupting lysosomes has <br /> Despite advances understanding aspects                            fully elucidated. Lysosomes acidic organelles, typi- <br /> molecular basis treatment human acute myeloid leukemia                           cally 50&#226;&#8364;&#8220;100 nm diameter, produced Golgi appara- <br /> (AML), relapse rates remain high (1). One factor account                       tus. They sequester hydrolases, lipases, proteases (10&#226;&#8364;&#8220;12) <br /> for clinical problems presence leukemia-initiating                      control protein degradation, cholesterol biosynthesis, and <br /> cells (LICs) bulk AML population (2, 3) resis-                      organelle recycling. Unregulated release lysosomal hydrolases <br /> tant conventional chemotherapy agents. This suggests                        proteases cytosol triggers caspase-independent cell <br /> identification therapeutic strategies target AML com-                       death (13&#226;&#8364;&#8220;17). Here, report antileukemic effects of <br /> partments sparing normal hematopoietic cells impor-                       lysosome disruption derive increased lysosomal size and <br /> tant order improve outcomes AML patients.                                  biogenesis AML. <br />   One therapeutic strategy target cell&#226;&#8364;&#8482;s protein degrada- <br /> tion machinery. The 26S proteasome, lysosome,                            Results <br /> constitutes cell&#226;&#8364;&#8482;s protein degradation/recycling machin-                            Identification antimalarial mefloquine initial screen of <br /> ery (4&#226;&#8364;&#8220;6). The biological role proteasome                          compounds cytotoxic leukemic cells vitro. To identify known com- <br /> described, proteasome inhibitors used clinically                            pounds antileukemic activity, compiled library 100 <br /> the treatment multiple myeloma mantle cell lymphoma                              on- off-patent drugs, focusing antimicrobials meta- <br /> (7&#226;&#8364;&#8220;9). While utility targeting proteasome                           bolic regulators wide therapeutic windows. We screened <br />                                                                                         library dose-response manner (72-hour incubation) a <br /> Authorship note: Mahadeo A. Sukhai Swayam Prabha contributed equally             panel leukemia cell lines measured cell growth viabil- <br /> this work.                                                                              ity using MTS assay. Previously reported treatment <br /> Conflict interest: The authors declared conflict exists.    leukemia cells ivermectin, potent compound <br /> Citation article: J Clin Invest. 2013;123(1):315&#226;&#8364;&#8220;328. doi:10.1172/JCI64180.    identified screen (18). <br />  <br /> 	                                    The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                   315 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Mefloquine exhibits cytotoxicity AML cells vitro. (A) High-throughput screen 100 known drugs (each represented diamond) OCI- <br /> AML2 leukemia cells, ranked EC50 value. (B) (R)- (S)-mefloquine. (C) Effect mefloquine treatment (24 hours) mouse monocyte-derived <br /> dendritic cells, mouse bone marrow mononuclear cells, MDAY-D2 mouse leukemia cells. Data represent mean percent viability &#194;&#177; SD <br /> (Annexin V/PI staining) compared vehicle-treated controls 3 independent experiments. (D) Effect mefloquine treatment (left panel, <br /> 30 hours; right panel, 48 hours) viability (Annexin V/PI staining) normal human hematopoietic cells (n = 9) primary human AML samples <br /> (n = 17). EC50 values sample, calculated using median effect method (see Methods). *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.005. Top bracket <br /> indicates comparison mefloquine sensitive AML normal samples. Line indicates comparison AML normal samples. <br /> (E) Effect mefloquine treatment (10 &#206;&#188;M) colony forming potential normal human hematopoietic samples (n = 3) primary human AML <br /> cells (n = 4). Data represent mean percent inhibition colony formation &#194;&#177; SD compared untreated cells. (F) Effect mefloquine pre- <br /> treatment (10 &#206;&#188;M, 24 hours) engraftment normal leukemic cells. Data represent percent engraftment human CD45+CD33+ <br /> cells mouse femurs. Each data point represents single mouse. <br />  <br />  <br />   Mefloquine second-most-active drug identified, based              (EC50 31.8 &#194;&#177; 5.4 &#206;&#188;M) murine monocyte-derived normal den- <br /> ranking drug EC50s (Figure 1, A B). Subsequent testing            dritic cells (EC50 17. 6 &#194;&#177; 2.7 &#206;&#188;M) (Figure 1C). Quinoline-derived <br /> effect mefloquine 10 human mouse leukemia                     compounds induce differentiation (19, 20), mefloquine <br /> cell lines showed 8 10, mefloquine reduced growth                recently reported induce differentiation human pluripo- <br /> viability EC50s 8 &#206;&#188;M (Table 1). Mefloquine-induced               tent stem cells (21). However, mefloquine did induce differen- <br /> cell death confirmed Annexin V propidium iodide                      tiation TEX leukemia cells (Supplemental Figure 1B). <br /> (PI) staining, cell death increased prolonged incuba-                    Mefloquine kills primary human AML cells effectively normal <br /> tion drug (Supplemental Figure 1A; supplemental mate-                  human hematopoietic cells. We compared ability meflo- <br /> rial available online article; doi:10.1172/JCI64180DS1).              quine kill cells primary human AML samples (Supplemen- <br /> In contrast effects leukemia cells, mefloquine               tal Table 1) normal human hematopoietic cells, following 30 <br /> cytotoxic normal murine bone marrow mononuclear cells                        48 hours exposure culture (n = 9 AML, n = 5 normal at <br />  <br /> 316	                          The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                         research article <br />  <br />     Table 1                                                                         Supplemental Figure 2, A B). In assays, higher EOBA <br />     Mefloquine sensitivity panel leukemia cell lines                       values indicate greater synergy drug combination. These <br />                                                                                     18 compounds diverse therapeutic classes diverse <br />     Cell line	              EC50 (&#206;&#188;M)	               95% CI                         mechanisms action. However, strikingly, 10 these, including <br />     HL60	     4.02	2.58&#226;&#8364;&#8220;6.25                                                        artemisinin class antimalarials (27, 28), proved com- <br />     MDAY-D2	 4.23	3.41&#226;&#8364;&#8220;5.05                                                         pounds known increase production ROS. Another <br />     KG1A	     5.00	4.80&#226;&#8364;&#8220;5.21                                                        6 reported effect ROS, remaining 2 been <br />     OCI-AML2	5.59	3.99&#226;&#8364;&#8220;7.84                                                         reported increase ROS production (refs. 29&#226;&#8364;&#8220;34 Table 2). <br />     TEX	      5.79	4.36&#226;&#8364;&#8220;7.58                                                          We assessed effects mefloquine ROS genera- <br />     NB4	      6.04	5.37&#226;&#8364;&#8220;6.78                                                        tion TEX OCI-AML2 cells. Twenty-four-hour treatment <br />     K562	     7.85	6.26&#226;&#8364;&#8220;9.85 <br />                                                                                     mefloquine increased ROS production concentrations that <br />     OCI-M2	 7.94	7.54&#226;&#8364;&#8220;8.37 <br />                                                                                     induced cell death previous experiments (Figure 2A). Increased <br />     U937	     9.17	4.97&#226;&#8364;&#8220;16.89 <br />     THP1	    11.40	7.12&#226;&#8364;&#8220;18.26                                                       ROS production observed primary AML patient sam- <br />                                                                                     ples sensitive mefloquine-induced cell death, not <br />     EC50 95% CI values calculated cell growth viability           AML cells insensitive mefloquine in <br />     data 3 independent experiments, using median-effect method.            normal hematopoietic cells (Figure 2B Supplemental Figure <br />                                                                                     2C). We evaluated ROS production using combination of <br />                                                                                     mefloquine artemisinin class antimalarials, most <br /> 30 hours; n = 11 AML, n = 6 normal 48 hours). Mefloquine                     synergistic combinations screen. Consistent pre- <br /> preferentially cytotoxic primary AML cells, compared                    vious reports (35), artesunate artenimol increased ROS pro- <br /> normal hematopoietic cells (Figure 1D Supplemental Figure                       duction TEX OCI-AML2 cells (Supplemental Figure 2D). <br /> 1, C D). In addition, mefloquine preferentially cytotoxic                   Synergistic combinations mefloquine artenimol artesu- <br /> to AML progenitor cells, compared normal hematopoietic                         nate synergistically increased ROS production (Figure 2C). <br /> progenitors (Supplemental Figure 1E). As observed AML                       Co-treatment ROS scavenger &#206;&#177;-tocopherol, known to <br /> cell lines (Supplemental Figure 1A), longer exposures increased                     inhibit lysosomal lipid-mediated ROS production (36), inhibited <br /> the toxicity mefloquine AML (Supplemental Figure 1C)                     ROS production (Supplemental Figure 2E) abrogated meflo- <br /> and normal cells, selectivity AML cells continued                 quine-induced cell death (Figure 2D) doses mefloquine that <br /> manifest (Figure 1D Supplemental Figure 1D). A subgroup                      led &gt;50% cell death. Conversely, N-acetyl-l-cysteine (NAC), a <br /> AML samples appeared particularly sensitive mefloquine,                      ROS scavenger acts inhibition thiol redox&#226;&#8364;&#8220;medi- <br /> this obviously associated differences cytogenetic                   ated ROS production (36), effect ROS cell viability <br /> risk overt clinical characteristics (Supplemental Table 1).                mefloquine treatment (Supplemental Figure 2E Figure <br />    We assessed effects mefloquine functionally                       2D). These data suggested mefloquine-mediated ROS could <br /> defined subsets primitive human AML normal cell pop-                         associated changes lysosomal stability. <br /> ulations, using established assays (2, 3, 22). Pre-treatment with <br /> mefloquine reduced subsequent clonogenic growth primary <br /> human AML cells, effect clonogenic growth                           Table 2 <br /> of normal human hematopoietic cells (Figure 1E). Pre-treatment                          List compounds synergistic mefloquine <br /> with mefloquine reduced ability primary human AML <br /> cells repopulate marrow NOD/SCID mice following                               Drug name	            Synergy 	 ROS 	               Ref. <br /> intrafemoral transplantation (Figure 1F). In contrast,                         	                      score	producer? <br /> pre-treatment protocol did alter repopulating activity                      Artenimol	               21.42	 Yes	 Supplemental Figure 2C <br /> normal cord blood cells (Figure 1F). Thus, mefloquine effectively                      Saqinavir <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span>	          18.33	 Yes	         37 <br /> targets populations leukemic cells, including LICs,                    Artemisinin	             18.17	Yes	          38 <br /> reduced toxicity normal hematopoietic cells.                                        Artesunate	              13.49	 Yes	 Supplemental Figure 2C <br />    A combinatorial high-throughput screen identifies mefloquine inducer          Saquinavir 	             13.41	 Yes	         37 <br />                                                                                        &#226;&#8364;&#402;mesylate <br /> of ROS. Mefloquine currently used treatment prophy- <br />                                                                                        Clotrimazole	            11.43	No	                   36 <br /> laxis malaria (23, 24), mechanism action antima-                   Rifabutin	               10.58	 Not reported	 <br /> larial fully elucidated. To determine mechanism                       Manidipine 	              9.98	 Not reported	 <br /> underlying toxic effect mefloquine AML cells, per-                        &#226;&#8364;&#402;dihydrochloride <br /> formed combination chemical screen, reasoning compounds                         Lopinavir	               9.76	Yes	                   37 <br /> that synergize mefloquine induce common cell death                          Ritonavir	               8.73	Yes	                   37 <br /> pathways. For screen, chose TEX leukemia cells (25),                     Amantadine HC1	          5.45	      No	              35 <br /> were derived overexpressing TLS-ERG oncogene primary                         Rifaximin	               4.89	 Not reported	 <br /> cord blood cells. Accordingly, treated aliquots TEX cells                   Acetylspiramycin	        4.30	 Not reported	 <br /> increasing concentrations mefloquine combination                            Tiamulin hydrogen 	      3.94	     Yes	              34 <br />                                                                                        &#226;&#8364;&#402;fumarate <br /> increasing concentrations expanded library on-patent and <br />                                                                                        Atazanavir	              3.18	Yes	                   37 <br /> off-patent drugs (n = 552). <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> screen, identified vali-                  Colistin sulphate	       2.93	 Not reported	 <br /> dated 18 compounds synergized mefloquine defined                          Imatinib mesylate	       2.53	     Yes	              33 <br /> by excess Bliss additivism (EOBA; ref. 26) statistic greater                   Vandetanib	              2.14	 Not reported	 <br /> than 2 SDs mean library drugs (ref. 26, Table 2, <br />  <br /> 	                                The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                 317 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Mefloquine synergizes ROS-producing compounds target lysosome. (A) TEX (left panel) OCI-AML2 (right panel) cells treat- <br /> ed mefloquine (5 10 &#206;&#188;M) vehicle control 24 hours ROS levels measured using 5-(and-6)-carboxy-2&#226;&#8364;&#178;7&#226;&#8364;&#178;-dichlorodihydrofluorescein <br /> diacetate (carboxy-H2DCFDA) staining. Data represent mean &#194;&#177; SD fold increase ROS compared vehicle control&#226;&#8364;&#8220;treated cells 3 <br /> replicates representative experiment. (B) Four primary AML samples (left panel: samples 1&#226;&#8364;&#8220;3 sensitive mefloquine; sample 4 insensitive <br /> to mefloquine; Supplemental Figure 2B) 2 normal PBSC samples (right panel) treated mefloquine 24 hours ROS levels <br /> measured using carboxy-H2DCFDA staining flow cytometry (*P &lt; 0.05). (C) Heat maps demonstrating induction ROS, measured in <br /> A, TEX leukemia cells treated combinations mefloquine artesunate artenimol, indicated. EOBA score defined Supple- <br /> mental Methods. (D) TEX cells treated mefloquine (8 &#206;&#188;M) 48 hours combination ROS scavengers &#206;&#177;-tocopherol <br /> (3 mM) NAC (10 mM). After incubation, cell viability measured Annexin V/PI staining. Data represent mean &#194;&#177; SD fold increase in <br /> ROS production compared vehicle control&#226;&#8364;&#8220;treated cells 3 replicates representative experiment (there significant protection <br /> against mefloquine-mediated cell death scavenger treatment: P &lt; 0.05). <br />  <br />  <br />  <br />   Haploinsufficiency profiling Saccharomyces cerevisiae identifies        protein predicted represent unique mefloquine target, con- <br /> yeast vacuole lysosome targets mefloquine. To explore      sistent previous observations (37). As negative controls, <br /> the mechanism mefloquine&#226;&#8364;&#8482;s antileukemic activity, used hap-                compared mefloquine results antimicrobials known <br /> loinsufficiency profiling (HIP), well-validated chemical genom-                mechanisms action unrelated lysosome, including tige- <br /> ics platform developed Saccharomyces cerevisiae (37&#226;&#8364;&#8220;39). The HIP              cycline, chloramphenicol, linezolid, flubendazole, clioquinol, and <br /> assay allows unbiased quantitative measure relative drug               ciclopirox olamine (ref. 41, Supplemental Figure 3, A&#226;&#8364;&#8220;C, Sup- <br /> sensitivity approximately 6,000 yeast proteins single             plemental Tables 4&#226;&#8364;&#8220;6). These compounds showed significant <br /> vivo assay, resulting rank-ordered list likely protein          enrichment mitochondrial function (tigecycline, chlorapheni- <br /> targets, pathways, and/or processes. Gene set enrichment analysis                col, linezolid), SWR1 complex (flubendazole), proteasome <br /> (GSEA; ref. 40) demonstrated gene deletion strains associated               function (clioquinol), regulation cellular pH (ciclopirox <br /> with sensitivity mefloquine significantly enriched genes              olamine), respectively. While vacuole functions be <br /> involved vacuolar processes, yeast parallel mammalian                enriched clioquinol ciclopirox olamine datasets, these <br /> lysosome functions (Figure 3A Supplemental Table 3). Spe-                    enrichments significant likely indi- <br /> cifically, significant enrichment vacuolar localization              cate putative secondary effects. Taken together, results the <br /> and processing, endosome transport, Golgi vesicle transport                  yeast genomic screen suggest mefloquine specifically targets <br /> (Supplemental Table 3). These processes share number genes                  lysosomal function. This finding consistent mefloquine&#226;&#8364;&#8482;s <br /> (indicated connecting edges Figure 3A), highlighting               known ability preferentially accumulate lysosomes the <br /> fact processes interrelated. Notably, vacuolar protein            malarial parasite (42). <br /> sorting genes help drive enrichment signifi-                   Mefloquine disrupts AML lysosomes. To determine the <br /> cantly enriched processes (see &#226;&#8364;&#339;Leading edge genes&#226;&#8364;? column               selective toxicity mefloquine human AML cells medi- <br /> all processes FDR 0 Supplemental Table 3 ref. 40).                ated lysosome disruption, isolated lysosomes TEX, <br /> Taken together, data provide rationale examining                 OCI-AML2, HL-60 cells, cells mefloquine- <br /> effect mefloquine mammalian lysosome.                                  sensitive primary AML patient samples. Purity lysosomal <br />   Since case, putative target member tightly               fractions confirmed immunoblotting lysosomal <br /> regulated large complex genes act concert, single              membrane protein LAMP1 mitochondrial protein COX II <br />  <br /> 318	                           The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                     research article <br />  <br />  <br />  <br />  <br /> Figure 3 <br /> Mefloquine disrupts lysosomes. (A) Biological processes protein complexes associated sensitivity mefloquine. Each node represents <br /> a process/complex significantly enriched genes associated mefloquine sensitivity (FDR &#226;&#8240;&#164; 0.01). Node size proportional signifi- <br /> cance enrichment (i.e., proportional &#226;&#8364;&#8220;log10[FDR value]). The width edges (represented gray lines diamond) proportional to <br /> level gene overlap connected categories. Edges shown overlap coefficient 0.5. Node color shows clus- <br /> ter membership, clustering based level overlap categories groups related categories. (B) Cathepsin B <br /> release isolated lysosomes mefloquine treatment (20 &#206;&#188;M, 90 minutes). Data represent mean cathepsin B release &#194;&#177; SD vehicle- <br /> and mefloquine-treated samples. *P &lt; 0.05, **P &lt; 0.01. (C) Cathepsin L release isolated lysosomes mefloquine treatment (20 &#206;&#188;M, <br /> 90 minutes). Data represent mean cathepsin L release &#194;&#177; SD vehicle- mefloquine-treated samples. *P &lt; 0.05, **P &lt; 0.01. (D) Citrate syn- <br /> thase release mitochondria isolated OCI-AML2 cells treated mefloquine. (E) Cathepsin B release lysosomes isolated <br /> from TEX cells treated tigecycline. Data represent mean cathepsin B release &#194;&#177; SD. (F) Cathepsin B release lysosomes isolated <br /> from OCI-AML2 cells treated tigecycline. Data represent mean cathepsin B release &#194;&#177; SD. <br />  <br /> 	                            The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                319 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Mefloquine disrupts lysosomes leukemia cells. (A) Left panel: TEM analysis lysosomes TEX cells treated 24 hours mefloquine <br /> (10 &#206;&#188;M). Data represent mean percent intact lysosomes &#194;&#177; SD relative control-treated cells. Middle right panels: Quantification LysoTracker <br /> and acridine orange uptake TEX cells treated 24 hours mefloquine (10 &#206;&#188;M). Results (mean &#194;&#177; SD 3 replicates representative experi- <br /> ment) represent percent lysosomal integrity relative control cells. (B) Lysosome integrity, measured LysoTracker acridine orange uptake, <br /> after treatment mefloquine (10 &#206;&#188;M) 24 hours cells insensitive (sample 1, EC50 &gt;12 &#206;&#188;M) sensitive AML samples (samples 2&#226;&#8364;&#8220;6, EC50 <br /> 5&#226;&#8364;&#8220;8 &#206;&#188;M) normal human hematopoietic cells (n = 3). Results (mean &#194;&#177; SD 3 replicates representative experiment) represent percent <br /> lysosomal integrity relative control cells. *P &lt; 0.05. (C) Cathepsin B release mefloquine (10 &#206;&#188;M) treatment shown immunofluorescence <br /> microscopy. Representative fields (&#195;&#8212;80 objective magnification) shown. Arrow indicates aggregate cathepsin B activity. Scale bars: 5 &#206;&#188;m. <br /> (D) TEX cells treated 8 &#206;&#188;M mefloquine combination bafilomycin A1 (900 nM) 24 hours, cell viability determined <br /> by Annexin V/PI. Results (mean &#194;&#177; SD 3 replicates representative experiment) represent viability compared controls. <br />  <br />  <br />  <br /> (Supplemental Figure 3D), demonstrating                       A B, Supplemental Figure 4, A&#226;&#8364;&#8220;E). Likewise, using <br /> 5-fold enrichment lysosomal enzyme acid phosphatase.                     immunofluorescence microscopy, cathepsin B <br /> Mefloquine directly disrupted lysosomes isolated AML cell                   present cytoplasm mefloquine-treated AML cells <br /> lines primary AML patients&#226;&#8364;&#8482; samples, dose-dependent                     large aggregates (Figure 4C), consistent lysosome disrup- <br /> manner, measured release cathepsins B L (Figure 3,                  tion (43&#226;&#8364;&#8220;45). In contrast, unrelated antimicrobials tigecycline, <br /> B C, Supplemental Figure 3E). The effects mefloquine                  ivermectin, ciclopirox olamine effect lysosomal <br /> were specific lysosomes, mefloquine treatment did dis-                 integrity despite ability induce cell death, consistent with <br /> rupt isolated mitochondria (Figure 3D). Finally, lysosome dis-                   different modes action outcome yeast <br /> ruption general feature cell death, unrelated                genomic screen (refs. 18, 41, 46, Supplemental Figure 4, F&#226;&#8364;&#8220;K). <br /> antimicrobial, tigecycline, did disrupt isolated lysosomes                Of note, lysosome disruption preceded increased ROS generation <br /> cytotoxic concentrations (Figure 3, E F, ref. 41).                       (Supplemental Figure 4E) inhibited ROS scaven- <br />   We evaluated effects mefloquine integrity                gers (Supplemental Figure 4L). Also consistent mechanism <br /> lysosomes intact AML cells treated vitro. In dose-depen-                 cell death related lysosome disruption finding that <br /> dent manner, mefloquine disrupted lysosomes TEX leukemia                      mefloquine-mediated cell death independent caspase-3 <br /> cells mefloquine-sensitive cells AML patients,                  activation (Supplemental Figure 4M). In addition, inhibition of <br /> normal hematopoietic cells, mefloquine-insensitive primary AML                   lysosomal cathepsins co-treatment vacuolar ATPase <br /> cells, THP1 cells, early 6 hours treatment. This              inhibitor bafilomycin A1 protected mefloquine-mediated <br /> demonstrated transmission electron microscopy (TEM) analy-                    cell death (Figure 4D). In contrast, inhibition single cathepsins, <br /> sis staining LysoTracker acridine orange (Figure 4,                  e.g., cathepsin D, insufficient inhibit mefloquine-mediated <br />  <br /> 320	                           The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                       research article <br />  <br />  <br />  <br />  <br /> Figure 5 <br /> Characterization mefloquine diastereomers&#226;&#8364;&#8482; effects cell viability lysosome integrity. (A) Chemical structures mefloquine diastereo- <br /> mers. (B) OCI-AML2 cells treated increasing concentrations indicated mefloquine diastereomer 48 hours. After incubation, cell <br /> viability measured MTS. Relative viability calculated comparison vehicle-treated controls. Data represent mean percent <br /> viability &#194;&#177; SD 3 replicates representative experiment. (C) OCI-AML2 cells treated increasing concentrations indicated <br /> mefloquine diastereomer 24 hours. After incubation, lysosome integrity measured LysoTracker staining flow cytometry. Relative <br /> lysosome integrity calculated comparison vehicle-treated control cells. Data represent mean percent lysosome integrity &#194;&#177; SD <br /> from 3 replicates representative experiment. <br />  <br />  <br /> cell death (Supplemental Figure 4N), consistent mefloquine                   (Figure 5B) lysosome disruption intact cells (Figure 5C). <br /> releasing multiple cathepsins cytosol. Taken together,                   The (&#226;&#8364;&#8220;) (+)-erythro forms mefloquine potent <br /> these data point mefloquine-mediated lysosomal disruption                   threo forms assays (Figure 5, B C, Sup- <br /> the cellular mechanism underlying antileukemic action.                            plemental Figure 5, A B). Of note, clinical-grade mefloquine, <br />   Effects mefloquine enantiomers analogs lysosome integrity         commercial supply, contain little threo <br /> cell viability. Mefloquine racemic mixture 4 diastereomers                forms (refs. 47, 48, data shown). We examined the <br /> (Figure 5A). Therefore, assessed effects (&#226;&#8364;&#8220;) (+)-eryth-             efficacy 6 compounds structurally related mefloquine that <br /> ro threo forms mefloqine viability leukemia cells                previously shown antimalarial activity (Supple- <br />  <br /> 	                              The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                   321 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 6 <br /> Genetic chemical strategies disrupt lysosomes antileukemia effects. (A) Left: Mean &#194;&#177; SD change LAMP1 mRNA compared with <br /> untransduced controls shRNA-mediated knockdown LAMP1 (*P &lt; 0.05). Right: LAMP1 actin protein expression shRNA-mediat- <br /> ed knockdown. Molecular weights (kDa) indicated. (B) Left: Mean &#194;&#177; SD change LAMP2 mRNA compared untransduced controls after <br /> shRNA-mediated knockdown LAMP2 (*P &lt; 0.05). Right: LAMP2 protein expression isolated lysosomes shRNA-mediated knockdown. <br /> Molecular weights (kDa) indicated. Asterisk indicates nonspecific band. Equal protein loading confirmed Ponceau S staining. (C) Num- <br /> ber viable OCI-AML2 cells trypan blue staining shRNA-mediated knockdown LAMP1, LAMP2, control sequences. Data represent <br /> mean &#194;&#177; SD number viable cells 4 replicates representative experiment. (D) Mean &#194;&#177; SD lysosome integrity (acridine orange staining), <br /> ROS production (carboxy-H2DCFDA staining), sensitivity 8 &#206;&#188;M mefloquine (MTS assay) shRNA-mediated knockdown LAMP1. <br /> Mean &#194;&#177; SD lysosome integrity (24 hours, LysoTracker staining) (E) viability (48 hours, Annexin V staining) (F) OCI-AML2 cells treated with <br /> LeuLeuOMe. (G) Mean &#194;&#177; SD lysosome integrity (LysoTracker staining) TEX cells treated 16 hours 5 &#206;&#188;M artesunate artenimol. (H) <br /> Mean &#194;&#177; SD percent inhibition clonogenic growth primary AML normal hematopoietic cells pretreated 5 &#206;&#188;M artesunate 48 hours <br /> and plated clonogenic growth assays (*P &lt; 0.05). <br />  <br /> 322	                          The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                     research article <br />  <br />  <br />  <br />  <br /> Figure 7 <br /> AML cells increased lysosomal mass compared normal hematopoietic cells. (A) Lysosome size (nm2) number quantified by <br /> TEM AML cells primary AML normal samples. Data represent mean &#194;&#177; SEM (left panel) (*P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.001). The arrow <br /> indicates CD34+ AML sample. (B) Ordered heat maps illustrating expression lysosome biogenesis genes functionally defined bulk AML <br /> cells (CD34+ cells 6 samples, CD34&#226;&#8364;&#8220;CD38+ cells 12 samples, CD34&#226;&#8364;&#8220;CD38&#226;&#8364;&#8220; cells 11 samples) versus normal CD34+ cord <br /> blood&#226;&#8364;&#8220;derived HSCs (n = 3 samples). Data derived dataset GSE30377, archived GEO database. Genes rank ordered by <br /> fold change significance. (C) Ordered heat maps illustrating expression subset lysosome biogenesis genes functionally defined <br /> LIC-enriched cell populations (CD34+CD38&#226;&#8364;&#8220; cells 13 samples CD34+CD38+ cells 6 samples, indicated) versus HSCs (n = 3 <br /> samples). Up Down indicated upregulated downregulated, respectively. (D) Expression lysosomal cathepsins functionally defined <br /> bulk AML cells (as B) versus normal CD34+ cord blood&#226;&#8364;&#8220;derived HSCs (n = 3 samples). (E) Expression lysosomal cathepsins functionally <br /> defined LIC-enriched cell populations (as C) versus HSCs (n = 3 samples). <br />  <br /> 	                            The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                323 <br />  research article <br />  <br />                                                                                 Figure 8 <br />                                                                                 Increased cathepsin B enzymatic activity LysoTracker uptake dis- <br />                                                                                 tinguish mefloquine-sensitive mefloquine-insensitive primary AML <br />                                                                                 patient samples. (A) Cell extracts normal CD34+ cells primary <br />                                                                                 (1) AML samples (n = 5) treated 0.2% Triton X disrupt the <br />                                                                                 lysosomal membranes. The cathepsin B substrate 1 mM Z-Arg-Arg- <br />                                                                                 7-amido-4-methylcoumarin hydrochloride (1 mM) added incu- <br />                                                                                 bated 1 hour 37&#194;&#176;C. The resulting fluorescence measured. <br />                                                                                 Data represent mean fold change cathepsin B release &#194;&#177; SD, <br />                                                                                 compared normal sample (set 1.00); ***P &lt; 0.001. Arrows <br />                                                                                 indicate CD34+ AML samples. (B) Relative lysosomal mass normal <br />                                                                                 hematopoietic cells (n = 3 samples), primary AML cells insensitive to <br />                                                                                 mefloquine (EC50 &gt;15 &#206;&#188;M; n = 3), primary AML cells sensitive to <br />                                                                                 mefloquine (EC50 &lt;8 &#206;&#188;M; n = 5) measured using LysoTracker <br />                                                                                 staining flow cytometry. Data represent mean relative increase <br />                                                                                 LysoTracker uptake 3 replicates single sample. Arrows indi- <br />                                                                                 cate CD34+ AML samples. <br />  <br />  <br />                                                                                 isolated lysosomes, increased ROS production, and <br />                                                                                 reduced viability TEX OCI-AML2 leukemia cell lines <br />                                                                                 (Figure 6, E&#226;&#8364;&#8220;G, Supplemental Figure 9B; compare Sup- <br />                                                                                 plemental Figure 2, A D, Supplemental Figure 9A). THP1 <br />                                                                                 U937 cells sensitive LeuLeuOMe-mediated lysosome <br />                                                                                 disruption cell death (Supplemental Figure 7, E F, and <br />                                                                                 Supplemental Figure 8, E F), suggesting mechanism of <br />                                                                                 lysosome-mediated cell death different mefloquine. <br />                                                                                 Furthermore, artesunate preferentially reduced clonogenic <br />                                                                                 growth primary AML cells compared normal hematopoi- <br />                                                                                 etic cells (Figure 6H). Overall, results genetic chemi- <br /> mental Table 7 ref. 49). None compounds induced cell               cal experiments validate lysosomal disruption novel plau- <br /> death disrupted lysosomes leukemia cells concentrations                sible therapeutic strategy AML. <br /> up 15 &#206;&#188;M (Supplemental Figure 5, C D).                                     Lysosome size increased AML. To investigate basis the <br />   Lysosome disruption therapeutic strategy leukemia. To         differential sensitivity leukemic cells normal hematopoi- <br /> explore potential antileukemic effect lysosomal disrup-                  etic cells lysosomal disruption, assessed characteristics of <br /> tion, asked genetic strategies mimic effects               lysosomes types cells. TEM revealed lysosomes <br /> obtained mefloquine. LAMP1 LAMP2 predomi-                      larger primary human AML cells, including CD34+ AML <br /> nant lysosomal membrane proteins function, part, main-                cells, AML cell lines, comparison lysosomes <br /> tain integrity lysosomal membrane malignant cells                 normal human CD34+ hematopoietic cells (Figure 7A <br /> (10&#226;&#8364;&#8220;12, 50, 51). We asked lentiviral vector&#226;&#8364;&#8220;mediated               Supplemental Figure 10A). Although larger size, num- <br /> shRNA knockdown LAMP1 LAMP2 target leu-                       ber lysosomes cell did differ significantly AML <br /> kemic cells. Effective knockdown gene transcripts                   normal cells (Figure 7A). <br /> proteins confirmed quantitative real-time RT-PCR                       Additionally, confirmed previously reported <br /> immunoblotting, using independent shRNAs gene                      lysosomal biogenesis signature genes (54), cathepsin <br /> (Figure 6, A B). Compared control, shRNA knockdown                     mRNA, overexpressed primary bulk human AML cells, as <br /> of LAMP1, lesser extent LAMP2, reduced growth                  CD34+CD38&#226;&#8364;&#8220; CD34+CD38+ subsets enriched in <br /> and viability OCI-AML2 leukemia cells (Figure 6C). LAMP1                     LICs, compared primitive <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> normal <br /> knockdown associated decreased lysosome integrity,                     human hematopoietic cells (refs. 55&#226;&#8364;&#8220;57, Figure 7, B&#226;&#8364;&#8220;E, Sup- <br /> increased ROS production, decreased cell viability, increased               plemental Figure 10, B&#226;&#8364;&#8220;D). Taken together, results indi- <br /> sensitivity mefloquine (Figure 6D). Similar effects LAMP1                 cate mechanism explain observed differential <br /> knockdown lysosome integrity cell viability observed                activity lysosome disruption leukemic normal human <br /> in TEX leukemia cells (Supplemental Figure 6). Also, THP1                   hematopoietic cells. <br /> U937 cells, sensitive mefloquine treatment                     Finally, mefloquine-sensitive primary AML cells, including <br /> (Table 1 Supplemental Figure 4D), exhibited lesser effects               CD34+ human AML cells, higher cathepsin B enzymatic activ- <br /> lysosome integrity cell proliferation LAMP1 knockdown,                ity compared insensitive AML cells normal hematopoiet- <br /> compared OCI-AML2 cells (Supplemental Figure 7, A&#226;&#8364;&#8220;D,                   ic cells (P &lt; 0.001; Figure 8A). Likewise, uptake LysoTracker <br /> Supplemental Figure 8, A&#226;&#8364;&#8220;D).                                                    dye significantly higher mefloquine-sensitive primary AML <br />   As test lysosome-based mechanism action,                 cells, including CD34+ subset these, compared both <br /> treated leukemia cell lines l-leucine-leucine-methyl ester                 normal CD34+ hematopoietic cells cells mefloquine- <br /> (LeuLeuOMe), artesunate, artenimol, report-                insensitive AML samples (P &lt; 0.01; Figure 8B). Thus, differ- <br /> ed disrupt lysosomes added intact cells (35, 52, 53).                ences help identify subsets AML patients likely <br /> Treatment agents disrupted lysosomes intact cells                 respond lysosome disrupters. <br />  <br /> 324	                          The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                      research article <br />  <br />  <br />  <br />  <br /> Figure 9 <br /> Mefloquine demonstrates therapeutic efficacy AML cells growing vivo. Sublethally irradiated SCID mice injected subcutaneously with <br /> (A B) OCI-AML2 (C) K562 human leukemia cells (D) MDAY-D2 mouse leukemia cells. Once tumors palpable, mice treated <br /> with 50 mg/kg mefloquine vehicle control daily oral gavage (n = 10 condition). Tumor weight measured time sacrifice. (E) <br /> Lysosome integrity (LysoTracker staining) single-cell suspensions derived control mefloquine-treated OCI-AML2&#226;&#8364;&#8220;derived tumors <br /> generated xenografted mice. (F) Primary AML samples (2 &#195;&#8212; 106 cells) injected intrafemorally sublethally irradiated NOD/SCID mice <br /> (n = 20) allowed repopulate mice 2.5 weeks, described Methods. After period, mice (n = 10 treatment condition) were <br /> treated 100 mg/kg/d oral gavage additional 4.5 weeks sacrificed. Bone marrow non-injected femurs collected; <br /> stained anti-human antibodies cell surface markers CD45, CD33, CD19; analyzed flow cytometry. Data represent percent <br /> CD45+ CD33+ CD19&#226;&#8364;&#8220; human AML cells present non-injected femurs. Serum bilirubin (G), alkaline phosphatase (H), aspartate transaminase (I), <br /> creatine kinase (J), creatinine (K), mouse body weight (L) vehicle- mefloquine-treated mice (50 mg/kg &#195;&#8212; 21 days) (for serum markers <br /> of liver kidney function, n = 3 treatment condition; body weight, n = 6 controls n = 5 mefloquine-treated mice). <br />  <br />  <br />  <br />   Mefloquine demonstrates antitumor activity leukemic cells growing           exhibited lysosomal disruption, consistent vitro <br /> in xenografted mice. Given antileukemic activity displayed                findings (Figure 9E). <br /> mefloquine vitro, evaluated effect oral meflo-                   We transplanted sublethally irradiated NOD/SCID mice <br /> quine mouse human leukemia cells growing trans-                        intrafemorally primary human AML cells later evaluated <br /> planted immunodeficient mice. Mouse MDAY-D2 cells                            effects 4.5-week course mefloquine starting 2.5 weeks <br /> human OCI-AML2 K562 cells transplanted subcutane-                        transplantation (10 mice group). Mefloquine treatment <br /> ously NOD/SCID mice treatment started 7 days later                      significantly reduced level leukemic cells compared with <br /> when tumors palpable (10 mice group). Com-                      control mice assessed end treat- <br /> pared vehicle control, mefloquine reduced tumor growth                      ment period (Figure 9F), evidence toxicity (body weight, <br /> in 3 models (Figure 9, A&#226;&#8364;&#8220;D). Mefloquine-responsive tumors                    appearance, behavior mice) noted. <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                325 <br />  research article <br />  <br />   To evaluate greater potential effects mefloquine              Our results agreement prior reports showing <br /> on normal tissues dose produced antileukemic                 increased cathepsin expression (67) lysosomal mass (50) <br /> effects, treated untreated SCID mice meflo-                  oncogene-mediated transformation. Prior studies indi- <br /> quine 21 days. No effects body weight, appearance,                   cated increased lysosomal biogenesis lead increases <br /> behavior confirmed, gross changes               lysosomal volume (54). In addition, Ono et al. (68) previously <br /> the organs necropsy histologic evaluation (Supplemental               suggested increased lysosomal volume render cells <br /> Figure 11). There alterations serum levels liver         sensitive death lysosome disruption blue light <br /> muscle/cardiac enzymes, alterations renal func-              irradiation, mechanism unclear. These find- <br /> tion (Figure 9, G&#226;&#8364;&#8220;L, Supplemental Figure 12). Thus, vivo,               ings taken report provide strong <br /> mefloquine disrupts lysosomes inhibits vivo growth               rationale therapeutically targeting lysosome AML, as <br /> leukemia cells significant toxicity normal host tissues.            larger lysosomes sensitive disruption. Further <br />                                                                                support idea comes results indicating the <br /> Discussion                                                                     cells mefloquine-sensitive AML samples signifi- <br /> Here report mefloquine, quinoline approved                   cantly increased lysosomal mass, measured LysoTracker dye <br /> treatment prevention malaria (23, 24), toxicity                 uptake, compared cells mefloquine-insensitive <br /> human AML cells including AML progenitors, sparing nor-                  samples normal hematopoietic sources. Thus, increased lyso- <br /> mal human hematopoietic cells treated doses. Fur-                somal size mass predictive biomarker sensitivity to <br /> thermore, antileukemic effects mefloquine              lysosomal disruption human AML. <br /> mediated disruption lysosomes, previously unappre-                  Currently, unclear AML cells increased lysosom- <br /> ciated mechanism action drug.                                       al mass biogenesis. Given role lysosomes macro- <br />    In treatment mild-moderate Plasmodium falciparum malaria,            molecule organelle recycling, function increase <br /> patients receive single oral dose 1,250 mg mefloquine,              levels amino acids nucleotides necessary cell prolifer- <br /> which usually sufficient eradicate parasite (24). However,           ation. Indeed, recent report indicated intracellular levels <br /> in regions mefloquine-resistant malaria prevalent, patients           amino acids regulate expression TFEB, transcrip- <br /> have received higher doses, 25 mg/kg, drug signifi-           tion factor responsible activation lysosomal biogenesis <br /> cant toxicity (58). In addition, serum concentrations mefloquine            pathways (54, 69&#226;&#8364;&#8220;71). In addition, increased lysosomal mass in <br /> up 5 &#206;&#188;M reported individuals receiving 250 mg                  AML necessary provide source fatty acids for <br /> weekly malaria prophylaxis (59, 60). Thus, antileukemia con-               cells. Recently, demonstrated AML cells have <br /> centrations mefloquine pharmacologically achievable.                 higher reliance fatty acids energy supply, AML <br />    Although mefloquine known accumulate lysosomes                 cells primitive elements increased rates of <br /> of malarial parasite (42), mechanism action anti-             fatty acid oxidation (72). <br /> malarial unclear. Therefore, work inform                  In summary, combined approach using small-molecule and <br /> the mechanism action mefloquine malaria,               yeast mutant screens vitro vivo functional studies <br /> selectively disrupt lysosomes malarial parasite. While              AML identified lysosome disruption novel prom- <br /> we demonstrated mefloquine disrupts lysosomes                     ising therapeutic strategy human AML. Investigation the <br /> yeast study strongly suggests intact lysosome required              mechanistic basis selective sensitivity primary human <br /> for resistance mefloquine, future studies identify direct            AML cells agents disrupt lysosomes revealed increased <br /> binding target(s) lysosomes leukemia cells possibly              lysosomal size biogenesis bulk AML cells subsets <br /> the malarial vacuole major interest.                                enriched AML stem cells. Thus, spite genetic and <br />    Autophagy lysosome-mediated process cellular autodi-                biological heterogeneity AML, common biological fea- <br /> gestion that, times <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation, enables cell            tures remain accessible selective targeting potential <br /> recycle raw materials macromolecular organellar biosyn-                therapeutic exploitation. <br /> thesis. Both inhibiting promoting autophagy lead cell <br /> death cancer (ref. 61 reviewed ref. 62). The antimalarial            Methods <br /> chloroquine structurally similar mefloquine inhibits                 Detailed experimental procedures available Supplemental Methods <br /> the degradation autophagy targets autophagolysosome.                 Supplemental Table 2. <br /> Through mechanism, chloroquine induce cell death                    Primary cells. Fresh peripheral blood samples obtained con- <br /> sensitize cells chemotherapy (including imatinib mesylate                senting AML patients mononuclear cells fractionated Ficoll sep- <br /> chronic myeloid leukemia; refs. 63&#226;&#8364;&#8220;66) radiation. However,              aration. Similarly, primary normal hematopoietic mononuclear cells were <br /> it involves induction lysosome disruption, mechanism                 obtained healthy consenting volunteers donating mobilized periph- <br /> action mefloquine appears distinct chloroquine                 eral blood stem cells (PBSCs) transplantation allogeneic recipients. <br /> and inhibitors autophagy.                                             Primary cells cultured 37&#194;&#176;C 5% CO2 IMDM supplemented <br />    LAMP1 LAMP2 predominant proteins lyso-                   10% FBS appropriate antibiotics. <br /> somal membrane, accounting approximately 50% pro-                     Assessment anticancer activity mouse models leukemia. MDAY-D2 mouse <br /> tein mass contained (14&#226;&#8364;&#8220;16). Previous studies dem-                leukemia cells human K562 OCI-AML2 leukemia cells (2.5 &#195;&#8212; 105) <br /> onstrated knockdown LAMP1 LAMP2 disrupts                        injected subcutaneously flanks sublethally irradiated <br /> integrity lysosomes malignant cells non-trans-                (3.5 Gy) NOD/SCID mice (Ontario Cancer Institute, Toronto, Ontario, <br /> formed cells (50). Consistent finding, observed              Canada). Four (OCI-AML2), 5 (MDAY-D2), 7 (K562) days injec- <br /> knockdown LAMP proteins decreases proliferation leu-                 tion, tumors palpable, mice treated mefloquine <br /> kemic cells, decreases lysosome integrity.                            (50 mg/kg/d) oral gavage water vehicle control (n = 10 group). <br />  <br /> 326	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013 <br />                                                                                                                                                          research article <br />  <br /> Tumor volume (tumor <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> &#195;&#8212; width2 &#195;&#8212; 0.5236) measured 3 times                          Study approval. The collection use human tissue study were <br /> week using calipers. Fourteen (MDAY-D2), 15 (OCI-AML2), 17 (K562)                        approved University Health Network, Mount Sinai Hospital, and <br /> days injection cells, mice sacrificed, tumors excised,                University British Columbia institutional review boards. All samples <br /> volume mass tumors measured.                                                     obtained donors provided written informed consent. All <br />   To assess mefloquine mouse models primary AML cord blood&#226;&#8364;&#8220;                       animal studies carried according regulations Cana- <br /> derived HSC engraftment, frozen aliquot AML cells cord blood cells                  dian Council Animal Care approval Ontario Cancer <br /> was thawed counted. Cord blood&#226;&#8364;&#8220;derived HSCs purified using                         Institute animal ethics review board. <br /> anti&#226;&#8364;&#8220;human CD34 antibody (Human CD34 Selection Kit, Stemcell Tech- <br /> nologies). Cells resuspended PBS, 1 &#195;&#8212; 106 2 &#195;&#8212; 106 viable                     Acknowledgments <br /> trypan blue&#226;&#8364;&#8220;negative cells injected marrow right femur                 We thank Jill Flewelling administrative assistance help in <br /> of 10-week-old female NOD/SCID mice irradiated 24 hours                       preparation manuscript. This work supported the <br /> previously 208 cGy 137Cs source. The mice injected                    Leukemia Lymphoma Society, International Myeloma <br /> with 200 &#206;&#188;g anti&#226;&#8364;&#8220;mouse CD122. Two half weeks injection                   Foundation, Canadian Stem Cell Network, Ontario Insti- <br /> human AML cells, mice treated mefloquine (100 mg/kg i.p.                       tute Cancer Research funding Ontario Min- <br /> injection) daily vehicle control (n = 10 group) 4.5 weeks.              istry Research Innovation, Ministry Health and <br /> Mice sacrificed, cells flushed femurs.                      Long-Term Care Province Ontario, Canadian Insti- <br /> Engraftment human AML marrow assessed enumer-                            tutes Health Research (CIHR), Terry Fox Foundation. <br /> ating percentage human CD45+CD33+CD19&#226;&#8364;&#8220; cells flow cytom-                          M.A. Sukhai CIHR Post-Doctoral Fellow Ontario Min- <br /> etry using BD FACSCalibur flow cytometer. Data analyzed                       istry Research Innovation Post-Doctoral Fellow. G. Giaever <br /> FlowJo version 8.8 (TreeStar).                                                              C. Nislow supported grant National Human <br />   Statistics. Statistical tests conducted, unless indicated,                 Genome Research Institute (NHGRI) grant Cana- <br /> using GraphPad Prism version 4 (GraphPad Software) Microsoft                            dian Cancer Society (no. 020380). A.D. Schimmer Leukemia <br /> Excel. For statistical comparisons cell culture assays, 2-tailed Stu-                  Lymphoma Society Scholar Clinical Research. <br /> dent&#226;&#8364;&#8482;s t test (2-population) 1-way ANOVA (multi-population) was <br /> applied appropriate. For statistical comparisons data derived                    Received publication April 10, 2012, accepted revised <br /> animal studies human patient samples, Mann-Whitney U test                        form October 4, 2012. <br /> applied 2-population comparisons, Kruskal-Wallis test, with <br /> the Dunns post-test, applied 3-population comparisons. Signifi-                     Address correspondence to: Aaron D. Schimmer, Princess Mar- <br /> cance cutoffs P &lt; 0.05 applied instances. Drug EC50s                    garet Hospital, Rm 9-516, 610 University Ave, Toronto, Ontario, <br /> calculated using median effect method (33), CalcuSyn ver-                      Canada M5G 2M9. Phone: 416.946.2838; Fax: 416.946.6546; <br /> sion 2.0 software package (Biosoft).                                                        E-mail: aaron.schimmer@utoronto.ca. <br /> 	 1.	Lowenberg B, et al. Mitoxantrone versus daunoru-             2010;376(9758):2075&#226;&#8364;&#8220;2085.                                 	 21.	Sachlos E, et al. Identification drugs including a <br />      bicin induction-consolidation chemotherapy &#226;&#8364;&#8221;        	10.	Bagshaw RD, Mahuran DJ, Callahan JW. A pro-                      dopamine receptor antagonist selectively tar- <br />      value low-dose cytarabine maintenance             teomic analysis lysosomal integral membrane                  cancer stem cells. Cell. 2012;149(6):1284&#226;&#8364;&#8220;1297. <br />      remission, assessment prognostic fac-           proteins reveals diverse composition           	 22.	Jin L, Hope KJ, Zhai Q, Smadja-Joffe F, Dick JE. Tar- <br />      tors acute myeloid leukemia elderly: final         organelle. Mol Cell Proteomics. 2005;4(2):133&#226;&#8364;&#8220;143.              geting CD44 eradicates human acute myeloid leu- <br />      report. European Organization Research         	11.	Bagshaw RD, Mahuran DJ, Callahan JW. Lyso-                       kemic stem cells. Nat Med. 2006;12(10):1167&#226;&#8364;&#8220;1174. <br />      Treatment Cancer Dutch-Belgian                somal membrane proteomics biogenesis               	 23.	Boudreau E, et al. Tolerability prophylactic Lari- <br />      Hemato-Oncology Cooperative Hovon Group.                     lysosomes. Mol Neurobiol. 2005;32(1):27&#226;&#8364;&#8220;41.                     regimens. Trop Med Parasitol. 1993;44(3):257&#226;&#8364;&#8220;265. <br />      J Clin Oncol. 1998;16(3):872&#226;&#8364;&#8220;881.                      	 12.	Callahan JW, Bagshaw RD, Mahuran DJ. The inte-            	24.	Harinasuta T, Bunnag D, Wernsdorfer WH. A <br /> 	 2.	Bonnet D, Dick JE. Human acute myeloid leuke-                gral membrane lysosomes: proteins              phase II clinical trial mefloquine patients with <br />      mia organized hierarchy originates              roles disease. J Proteomics. 2009;72(1):23&#226;&#8364;&#8220;33.               chloroquine-resistant falciparum malaria Thai- <br />      primitive hematopoietic cell. Nat Med. 1997;    	 13.	Brunk UT, Neuzil J, Eaton JW. Lysosomal involve-                land. Bull World Health Organ. 1983;61(2):299&#226;&#8364;&#8220;305. <br />      3(7):730&#226;&#8364;&#8220;737.                                                ment apoptosis. Redox Rep. 2001;6(2):91&#226;&#8364;&#8220;97.            	25.	Warner JK, et al. Direct evidence cooperating <br /> 	 3.	Lapidot T, et al. A cell initiating human acute        	14.	Guicciardi ME, Leist M, Gores GJ. Lysosomes                   genetic events leukemic transformation of <br />      myeloid leukaemia transplantation SCID            cell death. Oncogene. 2004;23(16):2881&#226;&#8364;&#8220;2890.                    normal human hematopoietic cells. Leukemia. 2005; <br />      mice. Nature. 1994;367(6464):645&#226;&#8364;&#8220;648.                  	15.	Leist M, Jaattela M. Triggering apoptosis                  19(10):1794&#226;&#8364;&#8220;1805. <br /> 	 4.	de Duve C. The lysosome turns fifty. Nat Cell Biol.          cathepsins. Cell Death Differ. 2001;8(4):324&#226;&#8364;&#8220;326.         	 26.	Borisy AA, et al. Systematic discovery multicom- <br />      2005;7(9):847&#226;&#8364;&#8220;849.                                     	16.	Leist M, Jaattela M. Four deaths funeral:                  ponent therapeutics. Proc Natl Acad Sci U S A. 2003; <br /> 	 5.	Kroemer G, Jaattela M. Lysosomes autoph-                 caspases alternative mechanisms. Nat Rev                100(13):7977&#226;&#8364;&#8220;7982. <br />      agy cell death control. Nat Rev Cancer. 2005;             Mol Cell Biol. 2001;2(8):589&#226;&#8364;&#8220;598.                         	 27.	Chawira AN, Warhurst DC, Robinson BL, Peters <br />      5(11):886&#226;&#8364;&#8220;897.                                         	 17.	Turk B, et al. Apoptotic pathways: involvement lyso-         W. The effect combinations qinghaosu (arte- <br /> 	 6.	Luzio JP, Pryor PR, Bright NA. Lysosomes: fusion         somal proteases. Biol Chem. 2002;383(7&#226;&#8364;&#8220;8):1035&#226;&#8364;&#8220;1044.            misinin) standard antimalarial drugs the <br />      function. Nat Rev Mol Cell Biol. 2007;8(8):622&#226;&#8364;&#8220;632.    	 18.	Sharmeen S, et al. The antiparasitic agent ivermec-             suppressive treatment malaria mice. Trans R <br /> 	 7.	Baiocchi RA, et al. Phase 2 trial rituximab           tin induces chloride-dependent membrane hyper-                  Soc Trop Med Hyg. 1987;81(4):554&#226;&#8364;&#8220;558. <br />      bortezomib patients relapsed refractory           polarization cell death leukemia cells. Blood.     	28.	Klayman DL. Qinghaosu (artemisinin): an <br />      mantle cell follicular lymphoma. Cancer. 2011;           2010;116(18):3593&#226;&#8364;&#8220;3603.                                         antimalarial drug China. Science. 1985; <br />      117(11):2442&#226;&#8364;&#8220;2451.                                     	19.	Iyamu E, Perdew H, Woods G. Growth inhibi-                       228(4703):1049&#226;&#8364;&#8220;1055. <br /> 	 8.	Belch A, et al. A phase II study bortezomib            tory differentiation effects chloroquine           	 29.	Chang SP, Shen SC, Lee WR, Yang LL, Chen YC. Ima- <br />      mantle cell lymphoma: National Cancer Insti-             analogue human leukemic cells poten-                 tinib mesylate induction ROS-dependent apop- <br />      tute Canada Clinical Trials Group trial IND.150.          tiate fetal hemoglobin production targeting                  tosis melanoma B16F0 cells. J Dermatol Sci. 2011; <br />      Ann Oncol. 2007;18(1):116&#226;&#8364;&#8220;121.                               polyamine pathway. Biochem Pharmacol. 2009;                 62(3):183&#226;&#8364;&#8220;191. <br /> 	 9.	Cavo M, et al. Bortezomib thalidomide plus              77(6):1021&#226;&#8364;&#8220;1028.                                          	30.	Hoebe KH, Monshouwer M, Witkamp RF, Fink- <br />      dexamethasone compared thalidomide plus           	20.	Martirosyan AR, Rahim-Bata R, Freeman AB,                        Gremmels J, van Miert AS. Cocultures porcine <br />      dexamethasone induction therapy before,               Clarke CD, Howard RL, Strobl JS. Differentiation-               hepatocytes Kupffer cells improved <br />      consolidation therapy after, double autologous               inducing quinolines experimental breast cancer               vitro model study hepatotoxic com- <br />      stem-cell transplantation newly diagnosed mul-            agents MCF-7 human breast cancer cell                    pounds. Vet Q. 2000;22(1):21&#226;&#8364;&#8220;25. <br />      tiple myeloma: randomised phase 3 study. Lancet.           model. Biochem Pharmacol. 2004;68(9):1729&#226;&#8364;&#8220;1738.           	 31.	Nakamura T, Lipton SA. Preventing Ca2+-mediat- <br />  <br />  <br /> 	                                    The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 1&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;January 2013	                                           327 <br />  research article <br />  <br />       ed nitrosative stress neurodegenerative diseases:           target tumor cells attack mitochondria                treatment uncomplicated falciparum <br />       possible pharmacological strategies. Cell Calcium.             processing bid leading Necroptosis. J Immunol.             malaria Rondonia, Brazil. Braz J Infect Dis. 2006; <br />       2010;47(2):190&#226;&#8364;&#8220;197.                                            2009;182(11):6993&#226;&#8364;&#8220;7000.                                          10(4):279&#226;&#8364;&#8220;282. <br /> 	32.	Thapa D, et al. Inhibitory effects clotrimazole        	46.	Eberhard Y, et al. Chelation intracellular iron          	 59.	Charles BG, et al. Population pharmacokinetics of <br />       TNF-alpha-induced adhesion molecule expres-                 antifungal agent ciclopirox olamine                     mefloquine military personnel prophylaxis <br />       sion angiogenesis. Arch Pharm Res. 2009;                   induces cell death leukemia myeloma cells.                malaria infection field deployment. <br />       32(4):593&#226;&#8364;&#8220;603.                                                 Blood. 2009;114(14):3064&#226;&#8364;&#8220;3073.                                   Eur J Clin Pharmacol. 2007;63(3):271&#226;&#8364;&#8220;278. <br /> 	 33.	Touzet O, Philips A. Resveratrol protects        	 47.	Karle JM, Olmeda R, Gerena L, Milhous WK. Plas-            	 60.	Nevin RL. Pharmacokinetic considerations the <br />       protease inhibitor-induced reactive oxygen species             modium falciparum: role absolute stereochem-                  repositioning mefloquine treatment pro- <br />       production, reticulum stress lipid raft pertur-            istry antimalarial activity synthetic amino            gressive multifocal leukoencephalopathy. Clin Neu- <br />       bation. AIDS. 2010;24(10):1437&#226;&#8364;&#8220;1447.                           alcohol antimalarial agents. Exp Parasitol. 1993;                rol Neurosurg. 2012;114(8):1204&#226;&#8364;&#8220;1205. <br /> 	34.	Zhang S, Chen H, Gerhard GS. Heme synthesis                     76(4):345&#226;&#8364;&#8220;351.                                             	 61.	Hu YL, et al. Hypoxia-induced autophagy promotes <br />       increases artemisinin-induced radical formation          	 48.	Wallen L, Ericsson O, Wikstrom I, Hellgren U. High-              tumor cell <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span> adaptation antiangio- <br />       cytotoxicity suppressed superox-            performance liquid chromatographic method                        genic treatment glioblastoma. Cancer Res. 2012; <br />       ide scavengers. Chem Biol Interact. 2010;186(1):30&#226;&#8364;&#8220;35.         enantioselective analysis mefloquine               72(7):1773&#226;&#8364;&#8220;1783. <br /> 	35.	Hamacher-Brady A, et al. Artesunate activates                   plasma urine. J Chromatogr B Biomed Appl. 1994;        	 62.	Yang ZJ, Chee CE, Huang S, Sinicrope FA. The role <br />       mitochondrial apoptosis breast cancer cells             655(1):153&#226;&#8364;&#8220;157.                                                  autophagy cancer: therapeutic implications. <br />       iron-catalyzed lysosomal reactive oxygen species         	 49.	Mullie C, Jonet A, Desgrouas C, Taudon N, Sonnet                 Mol Cancer Ther. 2011;10(9):1533&#226;&#8364;&#8220;1541. <br />       production. J Biol Chem. 2010;286(8):6587&#226;&#8364;&#8220;6601.                P. Differences anti-malarial activity 4-amino-       	 63.	Bellodi C, et al. Targeting autophagy potentiates <br /> 	 36.	Wu SJ, Ng LT, Lin CC. Effects vitamin E              alcohol quinoline enantiomers investigation                  tyrosine kinase inhibitor-induced cell death in <br />       cinnamaldehyde-induced apoptotic mechanism                  presumed underlying mechanism action.                  Philadelphia chromosome-positive cells, includ- <br />       human PLC/PRF/5 cells. Clin Exp Pharmacol Physiol.             Malar J. 2012;11:65.                                             ing primary CML stem cells. J Clin Invest. 2009; <br />       2004;31(11):770&#226;&#8364;&#8220;776.                                     	 50.	Fehrenbacher N, et al. Sensitization lysosom-             119(5):1109&#226;&#8364;&#8220;1123. <br /> 	 37.	Giaever G, et al. Chemogenomic profiling: iden-                al cell death pathway oncogene-induced down-            	 64.	Ding WX, et al. Oncogenic transformation confers <br />       tifying functional interactions small mol-              regulation lysosome-associated membrane pro-                  selective susceptibility combined suppres- <br />       ecules yeast. Proc Natl Acad Sci U S A. 2004;               teins 1 2. Cancer Res. 2008;68(16):6623&#226;&#8364;&#8220;6633.                sion proteasome autophagy. Mol Cancer <br />       101(3):793&#226;&#8364;&#8220;798.                                          	 51.	Huynh KK, Eskelinen EL, Scott CC, Malevanets A,                  Ther. 2009;8(7):2036&#226;&#8364;&#8220;2045. <br /> 	 38.	Giaever G, et al. Genomic profiling drug sensitivi-         Saftig P, Grinstein S. LAMP protein <br /> </body></html>