<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Volume 16 Number 19 1988                                               Nucleic Acids Research <br />  <br /> Molecular characterization GCDI, yeast gene required general control amino acid <br /> biosynthesis cell-cycle initiation <br />  <br />  <br /> David E.Hill+ Kevin Struhl* <br />  <br />  <br /> Department Biological Chemistry, Harvard Medical School, Boston, MA 02115, USA <br />  <br /> Received April 22, 1988; Revised Accepted, September 8, 1988                Accession no. X07846 <br />  <br />  <br /> 4BSTRACT <br />        The GCD1 gene product Saccharomyces cerevisiae implicated the <br /> coordination cell cycle general control amimo acid biosynthesis (M. Wolfner et <br /> al., J. Mol. Biol. 96:273-290, 1975). Strains containing gcdl-I allele constitutively express <br /> the amino acid biosynthetic genes induced levels normally conditions of <br /> amino acid starvation. In addition, gcdl-l strains grow high temperatures because <br /> under conditions unable proceed START step cell division <br /> cycle. We cloned sequenced GCDI gene examined various aspects cellular <br /> metabolism order elucidate role(s) regulating gene expression cell cycle. GCDJ <br /> encodes 1.7 kb RNA expression regulated function amino acid starvation. <br /> Overexpression RNA does affect regulation amino acid biosynthetic genes or <br /> cell growth. GCDI essential gene cells containing gcdl-HIS3 disruption are <br /> unable grow. The essential function Qf GCDJ involved protein synthesis a <br /> gcdl-1 strain incorporates low levels -methionine protein cells shifted the <br /> restrictive temperature. GCDI encodes protein 511 amino acids predicted sequence <br /> does exhibit significant homology known proteins appears large a <br /> ribosomal protein. We suggest GCDI encodes component normal protein synthesis <br /> machinery involved translational regulation GCN4, protein coordinately <br /> activates transcription amino acid biosynthetic genes. GCDI sensing <br /> mechanism cells monitor protein synthesis capacity prior initiating new cell <br /> division cycle. <br />  <br /> INTRODUCTION <br />      Saccharomyces cerevisiae coordinately regulates expression different amino <br /> acid biosynthetic genes response starvation various amino acids (reviewed 1,2). This <br /> cross-pathway regulation, termed general control, affects isoleucine tRNA synthetase <br /> gene (ILSI) ATRI gene involved aminotriazole resistance (3). Coordinate regulation <br /> occurs level transcription mediated binding GCN4 protein specific <br /> sequences promoter regions coregulated genes (4,5). Saturation mutagenesis of <br /> the HIS3 regulatory site examination sequences coregulated genes identified the <br /> 9-bp dyad symmetric sequence ATGACTCAT consensus element optimal site for <br /> GCN4 binding (6). <br />      Expression GCN4 gene regulated level translation (7,8). During normal <br /> conditions, translation GCN4 low consequently transcription general control <br />  <br />  <br /> &#194;&#169; I RL Press Limited, Oxford, England.                                                         9253 <br />  Nucleic Acids Research <br />  <br /> genes occurs basal level. Under conditions amino acid limitation, translation of <br /> GCN4 increased 30 fold GCN4 mRNA synthsis increases 2-3 fold. <br />  The increase GCN4 protein directly results coordinate induction amino acid <br />  biosynthetic genes. <br />        Translational regulation GCN4 appears involve small open reading frames in <br />  5&apos; non-coding sequences GCN4 mRNA. Under normal circumstances, translation is <br />  precluded presence AUG codons upstream AUG initiation <br />  codon GCN4 protein. After deletion <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>mutation</span> ORFs, GCN4 translation is <br />  constitutively high non-starvation conditions (9,10). Moreover, segment of <br />  GCN4 mRNA confers translational control fused heterologous mRNA coding sequences <br />  (1 1). It postulated upstream AUG codons present GCN4 mRNA leader <br />  bypassed translational initiation cell starved amino acids. <br /> However, mechanism translational regulation various effector molecules <br />  involved unknown. <br />        In addition GCN4, recessive mutations number genes alter regulation of <br /> the amino acid biosynthetic genes. Mutations GCN genes confer inability <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> induce transcription general control genes amino acid limitation (12-15). Mutations <br /> in GCD genes exhibit constitutive induction general control genes growth <br /> conditions (13,16,17). Interestingly, gcd mutations temperature-sensitive growth <br /> even non-starvation conditions. The GCD genes postulated negative regulators of <br /> GCN4 gcd mutants override upstream AUG codons constitutively overexpress <br /> GCN4 protein independently growth conditions. It constitutive overexpression of <br /> GCN4 results induced transcription general control genes. <br />        The best characterized GCD gene GCDI (originally called TRA3), initially <br /> identified single mutation conferred constitutive induction amino acid biosynthetic <br /> genes temperature sensitive growth (13). The gcdl-l mutation causes constitutive <br /> high level translation GCN4 (7,8), confers cell cycle arrest <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> cells. <br /> When gcdl- 1 cells shifted 230C, permissive growth temperature, 370C, the <br /> non-permissive temperature, uniformly arrest unbudded stage yeast cell cycle. <br /> This point arrest coincident position a-factor arrest haploid strains (13), <br /> which operationally defined start cell cycle. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> dual phenotype cell cycle <br /> arrest regulation general control, suggested GCDI function to <br /> integrate cellular requirements amino acids cell growth decision enter new <br /> cell-division cycle (13). In previous work, excluded trivial possibility gcdl- 1 <br /> artifactually caused amino acid starvation failure tRNA charging (18). In this <br /> communication, cloned sequenced GCDI gene examined certain <br /> aspects cellular metabolism gcdl-1 strains order elucidate function(s) GCDI in <br /> general control cell cycle regulation. <br />  <br />  <br /> 9254 <br />                                                                       Nucleic Acids Research <br />  <br /> MATERALS AND METHODS <br /> Genetic manipuain <br />       The yeast strains used work KY403 (a gcdl-l ura3-52 leu2-3,112 cyh2 <br /> canl -k2), KY 1 19 (a/a ura3-52 ade2- 101 lys2-801 trpl-A1 his3-A200), KY29 (a ura3-52 <br /> trpl-289). The growth, mating, sporulation, tetrad dissection, DNA transformation yeast <br /> strains carried standard procedures (19). <br /> Cloning GCDI <br />       A library yeast DNA segments cloned YEpl3, vector containing LEU2 selectable <br /> marker 2g origin replication, introduced spheroplasts strain KY403. After <br /> the treated spheroplasts plated minimal medium lacking leucine, allowed to <br /> grow 230C 18 hours imposing temperature selection 370C. DNA prepared <br /> from putative yeast transformants introduced E.coli selecting ampicillin <br /> resistance, plasmid DNAs introduced KY403 confirm phenotype. <br /> Disrugtion chromosomal GCDI gene <br />       The 1.7 kb DNA BamEH fragment encoding HIS3 gene, Sc2676, (20), cloned into <br /> the unique Bgll site GCDI coding region Sc4OlO. The resulting 5.5 kb BamHI <br /> fragment, Sc4016, containing GCDI disrupted HIS3 gene, purified used to <br /> transfonn diploid strain KY1 19 His+. Transfornants containing gcdl-HIS3 disruption <br /> on chromosome identified genomic hybridization analysis using GCDJ or <br /> HIS3 sequences probe. The diploid strain sporulated subject tetrad <br /> dissection. In control experiment, Y1p5-Sc4016 integrated KY1 19 loss the <br /> endogenous GCDI allele, resulting strain analyzed tetrad dissection. <br /> Sequencing GCDI <br />       Various DNA fragments produced restriction endonuclease cleavage Sc4O1O, 2.4 <br /> kb genomic fragment fully complements gcdl-l, subcloned M13mpl8 and <br /> M13mpl9 (21). Templates DNA sequencing generated digesting double-stranded <br /> DNA DNase I presence Mn2+ generate double-strand breaks random <br /> positions DNA subsequent circularization T4 DNA ligase (22). In addition, a <br /> Ba3l3 deletion series Sc4010 generated cloned M13mpl8. The <br /> M13-derived clones sequenced dideoxy method (23). Both strands GCD1 were <br /> sequenced entire open reading frame, 5&apos; 3&apos; non-coding ends Sc4010 were <br /> sequenced twice strand generally strands. <br /> RNA nails* <br />       Yeast strains grown 230C medium containing amino acid nucleotide <br /> supplements culture reached density A600 = 1.0. Aminotriazole added a <br /> final concentration 10 mM half culture, cells permitted grow an <br /> additional 4 hrs. For experiments involving quantitation GCDI mRNA, total RNA was <br /> isolated yeast cells, clectrophoretically separated 1.7% agarose gel containing 6% <br />  <br />                                                                                             9255 <br />  Nucleic Acids Research <br />  <br /> formaldehyde transferred nitrocellulose filter (24). The separated RNAs were <br /> hybridized mixture 32P-labeled probes prepared nick-translation. The hybridization <br /> probes Sc4014 DNA (Fig. 1) measuring GCDI expression Sc3l19 measuring <br /> HIS3 DEDI expression internal controls (19,24). The 5&apos; ends GCD1 mRNA were <br /> mapped standard primer extension method. Specifically, synthetic oligonucleotide <br /> corresponding nucleotides +42 +65 32P-labeled 5&apos; end T4 polynucleotide <br /> kinase, hybridized total yeast RNA, extended reverse transcriptase. The products of <br /> this primer extension compared di-deoxy sequencing reactions (23) using the <br /> identical labeled oligonucleotide primer. <br /> Protein synthesis <br />       Overnight cultures various yeast strains inoculated fresh synthetic medium <br /> containing 0.3% casamino acids plus uracil adenine grown 230C. The methionine <br /> concentration determined 500 gM based quantitative amino acid analysis 1% <br /> casamino acids solution using Beckman amino acid analyzer. When cultures reached an <br /> A600 approximately 0.5, aliquots transferred 370C left 230C. For <br /> continuous labelling protein synthesis, 35S-methionine added final specific <br /> activity 10 mCi/mmole minutes cultures shifted 370C. At various times <br /> after addition <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span>, 1 ml aliquots withdrawn mixed 0.25 ml 50% <br /> trichloroacetic acid (TCA) rapidly stop protein synthesis. The TCA-treated samples were <br /> heated 900C 10 min deacylate methionyl-tRNA complexes, subjected 15000 X g <br /> centrifugation 3 min, resuspended 1 ml 5% TCA. A 200 p1 aliquot each <br /> sample, duplicate, passed Whatman GFA glass fiber filter, retained <br /> radiolabelled material quantitated liquid scintillation counting. For pulse labelling, <br /> cultures divided 5 ml aliquots shifted 370C. At various times shift, <br /> 35S-methionine added 10 mCi/ mmole incubation continued 5 min which <br /> point unlabelled methionine added 10 mM incubation continued 10 min. <br /> Aliquots removed processed described above. <br />  <br /> RESULTs <br /> Cloning GCD) <br />      The GCDI gene obtained complementation temperature sensitive phenotype <br /> of strain KY403 (relevant genotype gcdl-I leu2-2,112). A library S. cerevisiae DNA <br /> fragments cloned YEpl3 shuttle vector introduced KY403 spheroplasts, 8 <br /> transformants able grow 370C absence leucine obtained. Restriction <br /> endonuclease cleavage plasmid DNAs obtained KY403 transformants identified <br /> common fragments defmed overlapping non-identical genomic clones (Fig. 1). <br />      Restriction mapping identified BamHI fragments encompassed Sc4001 and <br /> indicated Sc4005 contained BamHI site fortuitously created construction of <br />  <br /> 9256 <br />                                                                        Nucleic Acids Research <br />  <br />                                     GCD1 <br />                            I          11    1~!        1 <br />                            B    S/B HG HBHGB H             H S/B        B   H     S <br />  <br />              Sc4001 + <br />              Sc4005 +                      _. <br />              Sc4010 + <br />              Sc401 1- <br />              Sc4012 - <br />              Sc4013 - <br />              Sc4014 + <br />              Sc4015 - <br />              Sc4016 - <br />  <br />  <br />                                     HIS3 <br />  <br /> Figure 1: Restriction map phenotype GCD1 DNAs. The upper line shows composite <br /> restriction map overlapping genomic DNAs Sc4001 Sc4005 complement <br /> gcdl-I mutants. The letters line refer following restriction endonuclease sites: <br /> (B) BamHI; (G) BglI; (H) HindIl; (R) EcoRI; (S) Sau3A; (S/B) Sau3A sites fortuitously <br /> regenerate BamHI sites Sc4001 Sc4005, arrow line indicates the <br /> size direction GCDI mRNA. The genomic clones, Sc4001 Sc4005, with <br /> various subclones derived aligned beneath restriction map endpoints <br /> corresponding BamHI BglII restriction sites utilized subcloning. The + - <br /> symbols refer ability indicated DNA fragment complement gcdl-1 mutant when <br /> inserted multicopy vector YEpl3 (Sc4001 Sc4005), single copy vector <br /> (Sc4OlO-Sc4O15), integrating vector (Sc4010, Sc401 1, Sc4016). The DNA fragment <br /> Sc4016 constructed inserting 1.7 Kb BamHlI fragment containing HIS3 gene into <br /> the unique BgIH site GCDI plasmid Ylp5-Sc4OlO. <br />  <br />  <br /> the library (Fig. 1). These BamHI DNA fragments subcloned YCp5O, URA3 <br /> centromeric vector, YIp5, URA3 integration vector. Plasmids containing Sc4010 or <br /> Sc4014 complemented gcdl-l mutation, localizing presumptive GCDI gene 2.4 kb <br /> fragment yeast DNA. GCDI appears single copy gene unique 3.8 kb <br /> BamHI DNA fragment seen genomic hybridization experiments (Fig. 2). <br />       To confirm cloned fragments contain GCDI gene, KY403 transformant <br /> containing integrated copy YIp5-Sc4O1O mated appropriate haploid strain, and <br /> the resulting diploid strain analyzed tetrad dissection. If plasmid integrated the <br /> genome homologous recombination GCDI locus, URA3 marker plasmid <br /> should map chromosome XV roughly 30 centimorgans away HIS3, normal position <br /> of GCDJ. This result observed directly proving DNA sequences were <br /> cloned basis complementation gcdl-l fact encode GCDI gene. <br />  <br />  <br />                                                                                               9257 <br />  Nucleic Acids Research <br />  <br />  <br />  <br />  <br />                                          B               p   BG            B          P <br />  <br />                           ^=:; <br />                             ;1&gt;                          ~~~ti I           I <br />                         ;. <br />                          :4 E                                      S~~~~~~~~&apos;YS1&apos;L9 <br />  <br />                                                     PBXPX                      B          p <br />  <br />                                                       KY1I9Sc4 i._ <br />                                              PBG     B             P      B F             B <br />  <br />                                                    KY119:YIp5-Sc4QI6 <br />  <br /> Figure 2: Gene disruption GCDI. Panel A shows structure GCDI locus the <br /> KY1 19 diploid strains transformed indicated DNAs Genomic DNAs cleaved with <br /> BamHI (lanes A-B) PstI (lanes C-E) probed 2-p labeled Sc4010 DNA. Panel B <br /> shows restriction map relevant chromosomes (B, BamHI; G, BglH; P, PstI; X, <br /> BamHI/BgM hybrid site). <br />  <br /> GCD1 essential gene <br />       The fact gcdl-l confers temperature senstive growth suggests GCDI gene <br /> product essential cell viability. To prove GCDJ essential gene, perfonned a <br /> gene disruption experiment using one-step direct gene replacement technique (25). The 5.5 <br /> kb BamHI fragment, Sc4016, containing GCD1 gene disrupted HIS3 gene, used <br /> to transform diploid strain KY1 19 His+. Genomic hybridization (Fig. 2) confirmed that <br /> the His+ transformants KY1 19 expected GCDI/gcdl-HIS3 heterozygotes. <br />       If GCDJ essential gene HIS3 insertion abolishes GCDI function, tetrad <br /> should contain viable His- spores. Dissection 31 tetrads yielded 28 contained <br /> two viable His- spores non-viable spores. Microscopic examination non-viable <br /> spores showed 45 56 spores failed germinate 11 spores produced only <br /> very tiny buds. Two tetrads yielded viable His- spore tetrad gave viable His- <br /> spores; presumably, tetrads gene conversion random spore inviability. In a <br /> control experiment, YIp5-Sc4Ol6 integrated KYl 19 GCD1 locus loss of <br /> the endogenous GCDI allele. For 11 tetrads tested, 4 viable spores obtained, being <br /> His+Ura+ His-Ura-. These results demonstrate meiosis KY1 19 <br /> transfonnants occurs normally affected presence GCDJ disrupted gene. <br /> DNA mRNA codingy seguence GCDl <br />       To characterize GCDJ gene product, determined nucleotide sequence Sc4014, <br /> the 2.4 kb fragment fully complements gcdl-I mutation (Fig. 3). A single open reading <br />  <br /> 9258 <br />                                                                                    Nucleic Acids Research <br />  <br />         -415     GATCCACTCTAGAGGCTCCTCTTCATGATGTTCTTCCGTTTCCTGTTGGTTGTTCTGGTTGTTGTTCGTGTGACTGTTGGGTGTGATT <br />         -327 GTCGCCGGACGTTTCCCCCAATCCTGCTAATAAAGGGTCCTGTTCTTCACTCATTCTTAGTTATTAATATCTTTGCTTCGTCTGTTATCT <br />  <br />         -237 TGGAACCGGTGCGAGGTACTAGATTTAATTTACTTCACTATGGTATTTCAAGCTTCCTTTCCTTCGGTGGCAAATTCCACTATTTGAAAG <br />         -147 GTCACCCGCCTATCGCAAAATTTTTCACTTCTrCATCACTATATTGAAATATAACAATCAAGCACTTTACGATGCAACAGAAAATGCTGAC <br />  <br />          -57 AGCTGTGCTGACCAAAAGTTCCCACCGTTGATAGCTCCCCCCCCTATTGTCGTAGTCATGTCAATTCAGGCTTTTGTCTTTTGCGGTAAA <br />                                                                       MetSerIleGlnAlaPheValPheCysGlyLys <br />  <br />           34 GGTTCCAATTTGGCTCCCTTCACCCAGCCAGATTTTCCATTCCAAACGCAGAACAAAGACAGTACAGCTGCCACGAGCGGTGACAAACTT <br />                GlySerAsnLeuAlaProPheThrGlnProAspPheProPheGlnThrGlnAsnLysAspSerThrAlaAlaThrSerGlyAspLysLeu <br />         124 AATGAGTTGGTCAATAGCGCCCTCGATTCAACTGTCATAAATGAGTTCATGCAACATTCAACGCGCTTGCCCAAGGCTCTTTTGCCCATC <br />                AsnGluLeuValAsnSerAlaLeuAspSerThrValIleAsnGluPheMetGlnHisSerThrArgLeuProLysAlaLeuLeuProIle <br />         214 GGTAATAGACCTATGATTGAATACGTCTTGGATTGGTGTGATCAGGCAGATTTCAAAGAAATCAGTGTGGTCGCACCCGTTGACGAAATC <br />                GlyAsnArgProMetIleGluTyrValLeuAspTrpCysAspGlnAlaAspPheLysGluIleSerValValAlaProValAspGluIle <br />         304 GAATTAATTGAAAGTGGACTGACTTCGTTTTTGTCCCTAAGAAAGCAACAATTTGAACTAATATACAAGGCTTTGTCAAATTCCAACCAC <br />                GluLeuIleGluSerGlyLeuThrSerPheLeuSerLeuArgLysGlnGlnPheGluLeuIleTyrLysAlaLeuSerAsnSerAsnHis <br />         394 AGTCATCACTTGCAAGATCCTAAGAAAATTAATTTCATCCCTTCGAAGGCAAATTCTACAGGTGAGTCCTTGCAAAAAGAGCTTTTGCCT <br />                SerHisHisLeuGlnAspProLysLysIleAsnPheIleProSerLysAlaAsnSerThrGlyGluSerLeuGlnLysGluLeuLeuPro <br />         484 AGAATCAATGGCGATTTTGTAATCTTGCCCTGTGATTTTGTCACAGATATACCTCCACAAGTCTTGGTCGATCAATTTAGGAATAGGGAT <br />                ArgIleAsnGlyAspPheValIleLeuProCysAspPheValThrAspIleProProGlnValLeuValAspGlnPheArgAsnArgAsp <br />         574 GATAATAACCTAGCAATGACTATCTACTATAAGAACTCTTTAGATAGTAGTATCGATAAAAAGCAACAGCAAAAGGCAAAAAACAACAGC <br />                AspAsnAsnLeuAlaMetThrIleTyrTyrLysAsnSerLeuAspSerSerIleAspLysLysGlnGlnGlnLysAlaLysAsnAsnSer <br />         664 AATTTTTCACTGTTTATTCAGAAAACGAAGACTCAGAGAGGCAGCCAATACTTTTGGAACGTTTATTCTCAAAGGGACGTCACAAAGACA <br />                AsnPheSerLeuPheIleGlnLysThrLysThrGlnArgGlySerGlnTyrPheTrpAsnValTyrSerGlnArgAspValThrLysThr <br />         754 AAATATCTACAGATCAGATCTCATTTATTATGGAACTATCCAAATTTAACAGTATCCACTAAGTTACTGAACTCATTCATCTACTTTTGT <br />                LysTyrLeuGlnIleArgSerHisLeuLeuTrpAsnTyrProAsnLeuThrValSerThrLysLeuLeuAsnSerPheIleTyrPheCys <br />         844 TCCTTTGAACTTTGCCAGTTGTTAAAATTAGGACCTCAATCAATGTCAAGACAAGCTTCATTCAAAGATCCATTTACTGGAAACCAACAA <br />                SerPheGluLeuCysGlnLeuLeuLysLeuGlyProGlnSerMetSerArgGlnAlaSerPheLysAspProPheThrGlyAsnGlnGln <br />         934 CAGCAAAACCCTCCTACTACTGATGATGATGAAGATCGCAATCATGATGATGACGATGATTACAAACCTTCGGCTACATCTATCCAGCCT <br />                GlnGlnAsnProProThrThrAspAspAspGluAspArgAsnHisAspAspAspAspAspTyrLysProSerAlaThrSerIleGlnPro <br />        1024 ACCTACTTCAAAAAAAAGAATGATCTCATCTTGGACCCAATAAACTGTAATAAATCATTGAGTAAGGTTTTTAGAGATTTATCTCGTCGT <br />                ThrTyrPheLysLysLysAsnAspLeuIleLeuAspProIleAsnCysAsnLysSerLeuSerLysValPheArgAspLeuSerArgArg <br />        1114 TCGTGGCAACATTCGAAACCGAGGGAACCAATAGGTATTTTTATTTTACCAAACGAAACCTTGTTCATCAGAGCCAATAACTTGAATGCT <br />                SerTrpGlnHisSerLysProArgGluProIleGlyIlePheIleLeuProAsnGluThrLeuPheIleArgAlaAsnAsnLeuAsnAla <br />        1204 TACATGGACGCTAATAGATTTGTACTAAAGATAAAATCACAAACGATGTTCACGAAAAATATACAGATTCAATCTGCCGCTATCGGTGCT <br />                TyrMetAspAlaAsnArgPheValLeuLysIleLysSerGlnThrMetPheThrLysAsnIleGlnIleGlnSerAlaAlaIleGlyAla <br />        1294 GATGCCATAGTAGATCCCAAATGCCAAATCTCTGCTCATAGTAATGTCAAGATGTCTGTTCTCGGTACTCAGGCCAATATTGGTTCCAGA <br />                AspAlaIleValAspProLysCysGlnIleSerAlaHisSerAsnValLysMetSerValLeuGlyThrGlnAlaAsnIleGlySerArg <br />        1384 TGTCGTGTTGCAGGGTCTCTCTTATTTCCTGGAGTCCATCTTGGTGACGAAGTAATCCTGGAAAATTGTATTATTGGACCTATGGCAAAA <br />                CysArgValAlaGlySerLeuLeuPheProGlyValHisLeuGlyAspGluValIleLeuGluAsnCysIleIleGlyProMetAlaLys <br />        1474 TCGGTTTCAAAGTGTAAACTCAGCAATTCTTATATCGTAAGGCCATTATGTTGTGGAGCCTAAATAACTTTAAAGGTGAAACACTGCCAA <br />                SerValSerLysCysLysLeuSerAsnCysTyrIleValArgProLeuCysCysGlyAla   * <br />  <br />        1564 CGTTTATTTGGATGAAGATGAGGAGGACGAGTTAATATATGATGATAGTGTTATTGCTGGAGAAAGTGAAATCGCCGAAGAAACTGACAG <br />        1654 TGATGATAGAAGCGATGAAGATTCTGATGATAGTGAATATACCGACGAGTACGAGTACGAAGATGACGGATTATTTGAGCGTTAATATAA <br />        1744 ACTTATGTATATTCCTTCAGTCTCTTTAATAGAGACCAAGTAAGTAAGTGACGAGAATAGAGGCGATATTATGCATTTATATTTGAAATT <br />        1834 CAACAGAACATAATTATGCTTAAGCAAATTAACATGTTGTATTAATACATAGCGTTTTACACATCTATTTCATATATTTATGTTATGTTA <br />        1924 ATTTTTGATTAATTTTTCTTGTTCTTTCGTTGGATC      1959 <br />  <br /> Figure 3: Nucleotide sequence GCDI. The DNA sequence Sc4014, 2975 bp, shown <br /> with putative TATA element (double line) major mRNA initiation sites detemined SI <br /> mapping (lines attached arrows). The sequence numbered ATG codon <br /> observed single open rading frame Sc4014. <br />  <br />                                                                                                             9259 <br />  Nucleic Acids Research <br />  <br />  <br />  <br />  <br />                                                4-- 4 8 <br />  <br />  <br />  <br />  <br />                                                4-         .2 <br />  <br />  <br />  <br />                                                           4 <br />                                                4--    +   1 <br />  <br />  <br />  <br />  <br /> Figure 4: 5&apos; mapping GCDI RNA. The products primer extension shown the <br /> major bands indicated; bands observed absence yeast RNA reverse <br /> transcriptase. The location 5&apos; ends (defined respect presumptive AUG initiation <br /> codon shown Fig. 3) determined comparison di-deoxy sequencing reactions <br /> using identical primer analyzed gel. <br />  <br /> frame encoding 511 contiguous amino acids found. A search available protein and <br /> DNA data bases did identify sequences similar presumptive GCDI coding <br /> sequence. No TACTAAC elements splicing (26,27) found, suggesting GCDI does <br /> not contain introns. The codon usage putative GCDI protein yielded codon bias index <br /> (28) 0.14, suggesting GCDI highly expressed gene. Primer extension <br /> analysis revealed major RNA species 5&apos; endpoints located positions -48, -12, <br /> -4, +1 respect presumptive AUG initiation codon (Fig. 4). This presumptive <br /> initiation codon 5&apos;-proximal AUG codon largest GCDI RNA species. <br />       The GCDI promoter region contains TATA-like sequence 47 95 nucleotides upsteam <br /> from mRNA initiation sites, location typical yeast genes. However, extended <br /> poly(dA-dT) sequence, acts upstrm element constitutive transcription some <br /> yeast genes (29), observed. Although GCDI involved general control amino <br />  <br /> 9260 <br />                                                                        Nucleic Acids Research <br />  <br />                                   GCD1            YEp:Sc400l <br />                                    -   ++_ <br />  <br />                                           - Ded1`l <br />  <br />  <br />  <br />  <br />                                              H 3 <br />  <br />  <br />  <br />  <br /> Figure 5: Analysis GCDI RNA levels. Total RNA wild-type (GCDI) a <br /> gcdl-l strain transformed multicopy plasmid YEpl3-Sc400l prepAred from <br /> cultures grown presence (+) absence (-) aminotriazole probed J LP-labeled <br /> Sc4014 Sc3119. The positions GCDI, HIS3, DEDI transcripts indicated. In <br /> the strain containing multiple copies GCDJ, slight increase GCDI band intensity <br /> upon aminotriazole addition fact approximately 50% RNA loaded in <br /> that lane. <br />  <br /> acid biosynthesis, sequences resembling GCN4 binding sites (6) observed 360 <br /> <span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span> pairs mRNA start site. <br /> Transcriptional regulation GCDI <br />       To measure GCDI RNA levels, performed standard hybridization analysis total yeast <br /> RNA using levels HIS3 DEDI RNAs internal controls (Fig. 5). RNA was <br /> prepared cells grown non-starvation amino acid limitation conditions to <br /> determine GCDI mRNA itself subject general control. We examined the <br /> effect GCDI copy number ability HIS3 induced amino acid starvation <br /> by assaying strain containing approximately 10 copies GCDI form YEpl3-Sc400l <br /> (as assayed densitometric scans genomic DNA probed Sc4014; data shown). The <br /> amount GCDI RNA appears correlated copy number gene is <br /> independent growth conditions. Aminotriazole addition results induction HIS3 RNA <br /> that independent GCDI copy number. Thus, high gene dosage GCDI does not <br /> substantially alter cell&apos;s ability respond amino acid starvation does relative <br /> expression level GCDI change result amino acid starvation. <br /> Protein synthesis cell viability GCDl gcdl-l strains <br />       Although original characterization gcdl-- mutation suggested minimal defect in <br /> protein synthesis non-pernissive growth temperature 370C (13), reexamined <br /> this issue using different methods. In experiment, protein synthesis measured the <br /> continuous labeling proteins 35S-methionine (Fig. 6A). At 230C, rate protein <br /> synthesis strain KY403 2 4 fold lower corresponding rate wild type strain. <br />  <br />                                                                                              9261 <br />  Nucleic Acids Research <br />  <br />                          A.            10 <br />  <br />  <br />  <br />                                        8 <br />  <br />  <br />  <br />                                        6 <br />  <br />  <br />                                   8 <br />                            6 <br />                            E <br />                                   &lt;    4 <br />  <br />  <br />  <br />                                        2 <br />  <br />  <br />  <br />                                         0 <br />  <br />                          B. <br />                                        24                           n <br />                                                   00 <br />                                        20 <br />                                                                                   0 <br />                            T <br />                               , <br />                                        16   0\ <br />                            x <br />                                12 <br />                                   oD                      O~~~~~~~ <br />                            E &lt;                      0 0~~~ <br />                                         8           O     .-----O ------------------ <br />                                         4 <br />  <br />                                             -15 0 15 30   60       120            240 <br />                                                                Time (Min.) <br />  <br />  <br /> Figure 6: Protein synthesis GCDI gcdl-- strains. Panel A shows total <br /> protein synthesis continuous labeling GCDI (O,) gcdl-l(IQ,) cultures incubated <br /> at 230 C (0,0) 370 C (,O) varying times. The cpm S-methionine labelled <br /> TCA precipitable material divided total cell number determined absorbance <br /> measurements 600 nm wavelength normalize data cell basis. Panel B shows the <br /> amount protein synthesis 15 minute pulse labelling GCDI (0) gcdl-l (0) <br /> cultures incubation 370C varying times. The -15 time point represents of <br /> pulse-labeled mateial cells incubated 230C. <br />  <br /> This reduced rate protein synthesis correlates 2 fold slower growth rate gcdl <br /> strains 230C. After shifting culture 370C, rate proteins synthesis gcdl strain <br /> is 8 10 fold lower wild-type strain. <br />       In set experiments (Fig. 6B), yeast cells pulse-labeled minutes with <br />  <br /> 35S-methionine various times shifting cultures 230C 370C. Within 15 minutes <br /> after shifting culture 370C, GCDI strain impaired ability incorporate <br /> 35S-methionine protein wild type strain unaffected. (While yeast exhibit a <br /> mild heat shock response 370C, cells significantly alter <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> protein <br /> synthesis.) The inability incorporate 35S-methionine protein 370C gcdl strain is <br /> not inability methionine medium mutant wild-type <br />  <br />  <br /> 9262 <br />                                                                        Nucleic Acids Research <br />  <br /> strains contain similar amounts radioactive material harvesting washing cells. <br /> However, directly excluded unlikely possibility gcdl mutation alters the <br /> intracellular pool methionine. These experiments strongly suggest, prove, that <br /> GCDJ performs important function general protein synthesis. <br />       Yeast strains canying gcdl-l allele behave START mutants arrest the <br /> start cell cycle shifting culture 370C (13). By incubating KY403 cultures at <br /> 370C periods 1 72 hours retuming culture 230C, observed that <br /> nearly 100% cells survived 24 hour incubation 50% cells survived 48 <br /> hour incubation. The fact gcdl mutant survive long periods 370C a <br /> significant loss viability suggests gcdl restrictive condition characterized an <br /> orderly shutdown overall cellular metabolism. <br />  <br /> DISCUSSION <br />      GCDJ originally defined single allele prevented growth high temperature <br /> and caused constitutive induction amino acid biosynthetic genes lower temperatures <br /> (13). In paper, obtained overlapping genomic segments complemented the <br /> temperature-sensitive phenotype caused gcdl-1 allele mapped genetically GCDI <br /> locus. A 2.4 kb GCDJ DNA fragment encoding single open reading frame 511 amino acids <br /> restored normal regulation HIS3 presumably amino acid biosynthetic genes. In <br /> addition, fragment fully complemented phenotypes exhibited new gcdl <br /> mutant alleles (17). Thus, likely single protein product carries known GCDI <br /> functions. <br />       The temperature-sensitive phenotype gcdl-l suggested GCDI encoded essential <br /> function. This confirmed showing disruption GCDI locus lethal event. <br /> This <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> probably analogous cell-cycle phenotype observed gcdl-1 cells <br /> that shifted restrictive temperature. Although GCDI clearly involved the <br /> general control amino acid biosynthesis translational control GCN4 activator <br /> protein, GCDI RNA regulated function amino acid starvation GCDI <br /> promoter region does contain sequence recognized GCN4. Overexpression of <br /> GCDI mRNA does affect regulation amino acid biosynthesic genes cell growth. <br />       Pulse labeling cells 35S-methionine suggests protein synthesis gcdl strain <br /> is impaired 15 minutes shift 370C. This suggests essential function of <br /> GCDJ involved protein synthesis GCD1 important component basic <br /> protein synthetic machinery. Although specific function GCDI protein synthesis is <br /> unknown, possibilities excluded. GCDI unlikely amino acyl-tRNA <br /> synthetase level charged amino acyl-tRNA complexes remains normal gcdl <br /> strain 4 hr incubation 370C (18). It unlikely GCD1 encodes ribosomal <br /> protein predicted <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> 511 amino acids considerably larger ribosomal <br />  <br />                                                                                               9263 <br />  Nucleic Acids Research <br />  <br />  protein L3, encoded TCM gene (30). More likely candidates GCD1 <br />  function translational initiation, elongation, termination factors. In regard, however, <br />  predicted GCD1 sequence does exhibit significant homology known proteins <br />  does contain motifs resemble GTP-binding proteins. <br />        It likely proposed involvement GCDJ general protein synthesis is <br />  related role translational regulation GCN4. One possibility GCDl of <br />  machinery determines translation initiated. In view, viable gcdl mutants <br />  relax &quot;first AUG rule&quot; permit initiation (fifth) AUG codon specifies <br />  GCN4. In wild-type cells, GCDI directly indirectly sense amino acid starvation and <br />  possibly undergo conformational change bind effector molecule generate altered <br /> GCDI1 protein relaxed initiation specificity. One problem view mutation of <br />  frst AUG codon abolishes translation GCN4 mRNA gcdl-l background (9). <br />        An alternative view, favor, GCDI involved translational <br /> termination, specifically mechanism prevents reinitiation downstream AUG codons. <br /> In wild-type cells grown nonral conditions, translational termination GCN4 upstream <br /> ORFs occur, ribosomes released GCN4 mRNA GCN4 protein <br /> would synthesized. During amino acid starvation viable gcdl strains, termination <br /> event and/or ribosome release upstream ORFs GCN4 prevented, thereby <br /> allowing translating ribosomes initiate translation correct AUG GCN4. As mentioned <br /> above, GCD1 presumably exist states depending state amino acid <br /> biosynthesis. To explain mutation AUG codon abolishes translation GCN4 <br /> mRNA circumstances, alteration normal termination machinery permits <br /> reinitiation require specific initiation fist AUG codon. This idea reminiscent of <br /> antitermination transcription bacteriophage x N Q proteins specific sequences <br /> in template necessary alter properties transcription machinery (31). <br />       Yeast cells initiate new cell cycles sufficient amino acids complete the <br /> cycle. Auxotrophs particular amino acid arrest start cell cycle being <br /> switched medium lacking amino acid. Interestingly, gcdl mutants viable days after <br /> a shift 370C, situation similar auxotrophic cells starving required amino acid. <br /> Although gcdl strains retain normal levels charged tRNAs restrictive temperature and <br /> hence actually starving amino acids (18), GCDI sensing <br /> mechanism cells assess state amino acid biosynthesis initiating new <br /> cycle. As component protein synthesis, GCDI ideally situated monitor level <br /> of amino acid precursors. During mild starvation conditions, GCDI sense reduced <br /> levels charged tRNAs insufficient allow completion cell cycle. Consequently, <br /> GCDI signal cell arrest growth unless cell able increase levels <br /> of charged tRNAs increasing GCN4 translation inducing genes general control <br /> and increasing amino acid levels. By potentially involved directly translational <br />  <br /> 9264 <br />                                                                      Nucleic Acids Research <br />  <br /> control mechanism synthesis GCN4 activator protein, GCDI important link <br /> between capacity protein synthesis controlling amino acid precursors <br /> and decision initiate new round <br /> </body></html>