<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                               The Bacillus subtilis DnaD protein: a putative link between DNA <br />                                               remodeling and initiation of DNA replication <br />                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                                    The Bacillus subtilis DnaD protein is an essential protein and a component of the oriC and PriA <br />                                                    primosomal cascades, which are responsible for loading the main replicative ring helicase DnaC <br />                                                    onto DNA. We present evidence that DnaD also has a global DNA architectural activity, <br />                                                    assembling into large nucleoprotein complexes on a plasmid and counteracting plasmid <br />                                                    compaction in a manner analogous <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> that recently seen for the histone-like Escherichia coli HU <br />                                                    proteins. This DNA-remodeling role may be an essential function for initiation of DNA replication <br />                                                    in the Gram +ve B. subtilis, thus highlighting DnaD as the link between bacterial nucleoid <br />                                                    reorganization and initiation of DNA replication. <br />  <br />                                                    DNA replication; Atomic force microscopy; DnaD; Nucleoid <br />  <br />  <br />                                               1. Introduction <br />                                                                Bacterial chromosomal DNA is organized into a high order condensed structure called the <br />                                                                nucleoid [1]. Histone-like DNA remodeling proteins form the structural scaffolds of the <br />                                                                nucleoid, which is a dynamic nucleoprotein structure that changes during cell growth [2-4]. <br />                                                                Such architectural transitions are promoted by growth-phase dependent variations in the <br />                                                                histone-like protein (and other DNA-binding proteins) composition of the bacterial <br />                                                                proteome [5,6]. Global nucleoid remodeling is associated with crucial biological functions <br />                                                                such as regulation of gene expression [7], chromosome segregation [8] and initiation of <br />                                                                DNA replication [9,10]. <br />  <br />                                                                The Bacillus subtilis DnaD protein has been implicated in a primosomal DNA initiation <br />                                                                cascade that involves DnaB and DnaI and loads the replicative ring helicase, DnaC, onto the <br />                                                                oriC or non-specific DNA as part of the PriA-mediated restart primosome [11-13]. The <br />                                                                essential role of DnaD has also been confirmed genetically in Staphylococcus aureus [14]. It <br />                                                                interacts with both DnaA [15] and PriA [16] in the early steps of the primosomal cascade to <br />                                                                set the stage for helicase loading. The primosomal cascade is sequential in the DnaA (or <br />  <br />  <br />                                               &#194;&#169; 2004 Published by Elsevier B.V. on behalf of the Federation of European Biochemical Societies. <br />                                               * <br />                                                 Corresponding author. Fax: +44-115-8468002. panos.soultanas@nottingham.ac.uk (P. Soultanas).. <br />                                          Turner et al.                                                                                          Page 2 <br />  <br />  <br />                                                         PriA)&#226;&#8364;&#8220;DnaD&#226;&#8364;&#8220;DnaB order to set up an initial complex that then recruits DnaC [16]. DnaC <br />                                                         exists in a complex with DnaI [17,18], whilst the role of DnaB is still unclear. It may act <br />                                                         together with DnaI forming a pair of helicase loaders [18] or alternatively as a membrane <br />                                                         attachment protein to regulate initiation of DNA replication by regulating the recruitment of <br />                                                         DnaD to the membrane [19,20]. The role of the cell membrane in the initiation of DNA <br />                                                         replication in B. subtilis is well documented [21], but the precise molecular mechanisms that <br />                                                         underpin this function have just started to be unraveled [22]. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         We provide evidence that in addition to its role in the initiation of DNA replication, DnaD <br />                                                         also functions as a global regulator of DNA architecture in a manner similar to that seen for <br />                                                         the HU and H-NS proteins of Escherchia coli. It forms a variety of higher order <br />                                                         nucleoprotein structures that can remodel a supercoiled plasmid into an open circular form. <br />                                                         The DNA binding activity of DnaD is resistant to high NaCl concentration and appears to <br />                                                         induce the assembly of higher order DnaD oligomers. Finally, DnaD disrupts the stable <br />                                                         interaction between the helicase and DnaI in vitro, a property that may be functionally <br />                                                         relevant during helicase loading. The role of DnaD in the initiation of DNA replication and <br />                                                         its ability to alter the global architecture of DNA make it an obvious candidate for a link <br />                                                         between nucleoid reorganization and DNA replication in B. subtilis. A speculative model for <br />                                                         combining this dual role is presented. <br />  <br />                                            2. Materials and methods <br />                                            2.1. Protein purifications <br />                                                         The dnaD gene was cloned by PCR <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> B. subtilis (strain 168 EMG50) genomic DNA into <br />                                                         the NdeI-BamHI sites of pET22b (Novagen). Overexpression of DnaD was carried out in <br />                                                         B834 (DE3) E. coli with 1 mM IPTG for 7&#226;&#8364;&#8220;8 h from exponentially growing cultures (OD600 <br />                                                         = 0.6). DnaD overexpressing E. coli grew noticeably slower than un-induced cells. Cells <br />                                                         were harvested in 20 mM Tris, pH 7.5, 2 mM EDTA, 1 mM DTT, 500 mM NaCl, and 20 w/ <br />                                                         v sucrose, sonicated incubated for 30 min at 37&#194;&#176;C with benzonase (Novagen) and the cell <br />                                                         extract was clarified by centrifugation at 17 500 rpm for 30 min. Total protein in the <br />                                                         clarified supernatant was precipitated with ammonium sulfate (2.9 mg per 10 ml) and <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         centrifuged at 17 500 rpm, 30 min. The protein pellet was re-suspended in GE (20 mM <br />                                                         glycine, pH 9.8, 2 mM EDTA, and 1 mM DTT) buffer until the conductivity reached 40 mS <br />                                                         and loaded onto a sourceQ column, pre-equilibrated in GE, at 10&#226;&#8364;&#8220;12 mS. The column was <br />                                                         washed with GE/100 mM NaCl and DnaD was eluted with a 100&#226;&#8364;&#8220;400 mM NaCl gradient in <br />                                                         GE. Relevant fractions were pooled and DnaD was precipitated with ammonium sulfate, as <br />                                                         described before. The protein pellet was re-suspended in BisTE (Bis-Tris, pH 6.7, 2 mM <br />                                                         EDTA, and 1 mM DTT) buffer maintaining the conductivity at 40 mS and loaded onto a <br />                                                         HiTrap 5 ml Heparin, pre-equilibrated in BisTE, at 15&#226;&#8364;&#8220;20 mS. The column was washed with <br />                                                         Bis-TE/100 mM NaCl and DnaD was eluted with 100&#226;&#8364;&#8220;400 and stepping up to 700 mM <br />                                                         NaCl. The relevant fractions were pooled and loaded onto a superdex S75 gel filtration <br />                                                         column pre-equilibrated in TE/300 (20 mM Tris, pH 7.5, 2 mM EDTA, 1 mM DTT, and 300 <br />                                                         mM NaCl) buffer. The molecular weight of DnaD is 27.5 kDa and its elution profile <br />                                                         indicated an oligomer. Typically, 8&#226;&#8364;&#8220;9 mg of DnaD per liter of bacterial culture was <br />                                                         obtained. Purity was assessed by SDS&#226;&#8364;&#8220;PAGE and concentration was determined <br />                                                         spectrophotometrically. <br />  <br />                                            2.2. Sedimentation velocity ultracentrifugation experiments <br />                                                         Analytical ultracentrifugation sedimentation velocity experiments were carried out in a <br />                                                         Beckman XL-I analytical ultracentrifuge using cells with two channel centerpieces, loaded <br />                                                         into a four-hole AnTi 60 rotor. All experiments were carried out at 40 000 rpm. Cells were <br />  <br />  <br />  <br />                                                             FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                            Page 3 <br />  <br />  <br />                                                          scanned every 5 min at 280 nm and the temperature was set at 20 &#194;&#176;C. DnaD concentrations <br />                                                          were 4.9, 2.4 and 1.2 &#206;&#188;M. <br />  <br />                                                          Sedimentation coefficient distributions were generated using the program SEDFIT [23] <br />                                                          using the c(s) method of analysis. Errors in the distribution were estimated at the 67% and <br />                                                          95% confidence levels using 1000 runs of a Monte Carlo simulation. Frictional coefficients <br />                                                          for theoretical distributions of oligomers were estimated using the program SEDNTERP <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                          [24]. <br />  <br />                                                          Experiments designed to observe the DNA dependent behavior of aggregation were carried <br />                                                          out with a 5&#226;&#8364;&#178; HEX labeled 33mer oligonucleotide (5&#226;&#8364;&#178;-HEX- <br />                                                          TGCATGCCTGCAGGTCGACTCTAGAGGATCCCC-3&#226;&#8364;&#178;) and data were obtained at a <br />                                                          wavelength of 536 nm. <br />  <br />                                            2.3. Gel shift assays <br />                                                          DNA binding reactions were carried out using a radioactively labeled 33mer oligonucleotide <br />                                                          (5&#226;&#8364;&#178;-TGCATGCCTGCAGGTCGACTCTAGAGGATCCCC-3&#226;&#8364;&#178;) at 2.5 nM in 50 mM Tris, <br />                                                          pH 7.4, 100 mM NaCl, 1 mM EDTA, 1 mM DTT and increasing concentrations of DnaD <br />                                                          (123&#226;&#8364;&#8220;738 nM). Mixtures were incubated for 10 min at room temperature and complexes <br />                                                          were resolved by non-denaturing PAGE. Gels were dried and radioactive bands were <br />                                                          visualized using a phosphorimager. <br />  <br />                                            2.4. Analytical gel filtration <br />                                                          Gel filtration experiments were carried out as described before [17] using mixtures of B. <br />                                                          stearothermophilus DnaB (4.52 &#206;&#188;M), DnaD (1.96 M) and DnaI (11.22 &#206;&#188;M), as appropriate. <br />                                                          Proteins were incubated in 50 mM Tris, pH 7.5, 250 mM NaCl, 1 mM EDTA, and 1 mM <br />                                                          DTT for 10 min on ice before fractionated through a superdex S200 10/30 column <br />                                                          (Amersham Pharmacia Biotech). Fractions from the peaks were analyzed by SDS-PAGE. <br />                                                          Mixtures of DnaB/DnaD/DnaI, DnaB/DnaI, DnaB/DnaD, DnaI/DnaD as well as the <br />                                                          individual proteins alone were analyzed in a similar manner. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            2.5. Atomic force microscopy imaging <br />                                                          DnaD samples were diluted to between 0.5 and 0.005 &#206;&#188;g/ml in 20 mM Tris, pH 7.5, 2 mM <br />                                                          EDTA, and 300 mM NaCl. Plasmid pBR322 was purchased from Sigma-Aldrich. Plasmid <br />                                                          samples were made up to a stock solution of 10 &#206;&#188;g/ml in dH2O before being diluted further <br />                                                          to a desired concentration in 10% v/v PBS containing 0.5mM MgCl2. All solutions were <br />                                                          filtered through 0.2 &#206;&#188;M pore size filters (Sartorius) prior to use. Protein DNA complexes <br />                                                          were prepared by mixing protein samples diluted to 0.25&#226;&#8364;&#8220;0.0025 &#206;&#188;g/ml in 20 mM Tris, pH <br />                                                          7.5, 2 mM EDTA, 300 mM NaCl, with between 0.5 and 0.15 &#206;&#188;g/ml of pBR322 diluted in <br />                                                          10% v/v PBS containing 0.5 mM MgCl2 to generate varying molar ratios of protein to DNA. <br />  <br />                                                          For imaging in air, samples of 5&#226;&#8364;&#8220;10 &#206;&#188;l were incubated with freshly cleaved mica (Agar <br />                                                          scientific) for 5&#226;&#8364;&#8220;10 s and rinsed with 5 ml of dH2O and allowed to dry under a gentle flow <br />                                                          of nitrogen gas. For imaging in liquid, samples of 30 &#206;&#188;l were spotted as a single aliquot onto <br />                                                          the underside of a cantilever, placed inside a liquid cell and placed directly onto freshly <br />                                                          cleaved mica. <br />  <br />                                                          All atomic force microscopy (AFM) imaging was carried out with a Digital Instruments <br />                                                          Nanoscope III Mutlimode AFM with a type E scanner (Veeco) in tapping mode. The <br />                                                          cantilevers used were silicon tapping probes with a spring constant of 34.4&#226;&#8364;&#8220;74.2 N/m <br />                                                          (Olympus, OMCL-AC160TS). The tapping set point was adjusted to minimize probe&#226;&#8364;&#8220; <br />                                                          sample interactions. Images were recorded in both topography and phase modes with a pixel <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                             Page 4 <br />  <br />  <br />                                                         size of 512 &#195;&#8212; 512, flattened and analyzed with WSxM software 3.0 Beta 2.3 (Nanotec <br />                                                         Electronic S.L, Spain). <br />  <br />                                            3. Results and discussion <br />                                            3.1. DnaD forms higher order oligomers in a concentration dependent manner <br />                                                         DnaD forms mainly dimers or trimers and also a smaller proportion of higher order <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         oligomers [15,16]. A variety of techniques ranging from yeast two hybrid technology, <br />                                                         glycerol density and sucrose density centrifugation, to gel filtration and glutaraldehyde <br />                                                         fixing were used to establish the DnaD oligomeric state, but in all cases purified DnaD was <br />                                                         tagged with either a His-tag or an intein tag to facilitate purification. We purified untagged <br />                                                         DnaD and subjected it to analytical gel filtration through a calibrated superdex S-75 gel <br />                                                         filtration column. DnaD eluted as an apparent tetramer (data not shown). However, <br />                                                         oligomerization estimations by gel filtration may be inaccurate because they depend on a <br />                                                         combination of shape and size [25]. <br />  <br />                                                         Sedimentation velocity experiments over three concentrations revealed sedimentation <br />                                                         coefficient distributions consistent with three species in solution that interact in a <br />                                                         concentration dependent manner (Fig. 1A). At the lowest concentration (1.2 &#206;&#188;M), frictional <br />                                                         coefficients expected for each oligomeric species (Table 1) reveal that only a monomer <br />                                                         (sedimenting at around 2.0 s) has a physically reasonable value (f/f0 = 1.488). Increasing the <br />                                                         concentration leads to a second peak at around 4.5 s (corresponding to a compact dimer or <br />                                                         an extended trimer; see Table 1), with higher order species in excess of 6 s (pentamer and <br />                                                         hexamer, as judged by reasonable frictional coefficients) also present. Separation of the <br />                                                         species into distinct sedimentation coefficient distributions also reveals a specific type of <br />                                                         behavior. It is known both theoretically and experimentally that it is impossible to separate <br />                                                         out a rapidly reversible monomer/dimer equilibrium using sedimentation analysis; a <br />                                                         weighted average distribution is produced [26]. If the second species is a dimer in the <br />                                                         presented studies, this would mean that the rate of dimerization is slow on the timescale of <br />                                                         sedimentation. A second possibility is that the samples undergo irreversible aggregation, and <br />                                                         hence separate into distinct species as there is no appreciable off rate of aggregation. This, <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         however, is unlikely as dilution of the sample for sedimentation yields only one small <br />                                                         species: irreversible aggregation would yield the same proportion of species, irrespective of <br />                                                         concentration, upon dilution. A third scenario is that the species are in a monomer/n-mer <br />                                                         rapidly reversible equilibrium, where n &gt; 2. Theory and experiment show that separation of <br />                                                         species is physically possible if one accepts that there is a cooperative transition from <br />                                                         monomer to trimer or tetramer. These would be present as highly extended conformations <br />                                                         (as judged by physically reasonable frictional coefficients). AFM images of DnaD at various <br />                                                         concentrations verified the formation of higher extended oligomeric species at high <br />                                                         concentrations, giving weight to this third hypothesis. Elongated strand-like and also circular <br />                                                         and semi-circular structures were visible at high concentrations (Fig. 1B). The smallest <br />                                                         features representing DnaD monomers were approximately 25 nm in their longest dimension <br />                                                         and larger features of 50 nm (dimers), 79 nm (trimers) and also longer oligomers were also <br />                                                         apparent. <br />  <br />                                            3.2. DnaD-mediated nucleoprotein complexes <br />                                                         We proceeded to investigate the binding of DnaD to DNA with a HEX-tagged synthetic <br />                                                         oligonucleotide so as to follow independently the sedimentation of the protein (280 nm) or <br />                                                         the Hex-oligonucleotide (536 nm). We observed that upon addition of the oligonucleotide to <br />                                                         DnaD, an extremely large nucleoprotein complex was formed (1&#226;&#8364;&#8220;2 million Daltons) that was <br />                                                         impossible to investigate with analytical ultracentrifugation. Instead, the DNA binding <br />                                                         activity was investigated by gel shifts using the same untagged but radioactively labeled <br />  <br />  <br />                                                             FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                              Page 5 <br />  <br />  <br />                                                         oligonucleotide (Fig. 2A). DnaD bound to the oligonucleotide producing a clear shifted band <br />                                                         and as the DnaD concentration increased a streak started to appear higher up the gel until the <br />                                                         shifted band ended up in the well at the highest DnaD concentration. The remarkable <br />                                                         observation was that additional DnaD molecules were directed exclusively to the initial <br />                                                         nucleoprotein complex rather than to the free oligonucleotide, although at very high DnaD <br />                                                         concentrations all of the substrate could be shifted into the well (data not shown). The <br />                                                         stoichiometry of DnaD versus oligonucleotide in the initial complex is not certain, but <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         comparisons with the shifted bands from other proteins implied that it may be either two or <br />                                                         three DnaD molecules bound to one oligonucleotide (data not shown). The simplest <br />                                                         interpretation of these data is that once DnaD forms an initial nucleoprotein complex, it acts <br />                                                         as the nucleus to recruit additional DnaD molecules eventually forming a very large <br />                                                         complex. Furthermore, the interaction of DnaD with DNA is salt resistant, with stable <br />                                                         DnaD&#226;&#8364;&#8220;DNA complexes observed at NaCl concentrations as high as 800 mM (data not <br />                                                         shown). This is a feature that has been observed in other nucleoid associated proteins such <br />                                                         as the H-NS protein in E. coli [27]. <br />  <br />                                            3.3. AFM imaging of DnaD-mediated nucleoprotein complexes <br />                                                         We investigated the nature of the nucleoprotein complexes by AFM in liquid and air using <br />                                                         pBR322 plasmid. We analyzed 149 images of DnaD&#226;&#8364;&#8220;pBR322 complexes and observed that <br />                                                         when DnaD binds to pBR322 it opens up the supercoiled plasmid into an open circular <br />                                                         shape (Fig. 2B and C). This remarkable global structural effect on pBR322 resembles a <br />                                                         similar effect observed for the HU binding to supercoiled pUC19 that reflects its ability to <br />                                                         constrain DNA supercoils [28]. HU counteracts the H-NS mediated global DNA compaction <br />                                                         effect in E. coli. A similar structural transition from a supercoiled to a relaxed state has also <br />                                                         been observed in the case of the replication initiator RepE54 protein encoded by the mini-F <br />                                                         plasmid and in this case this relaxation is not the result of DNA strand break or local melting <br />                                                         of the double strand [29]. Whether the DnaD-induced relaxation is the result of strand break <br />                                                         or not is not known and is the subject of current investigations in our laboratory. Two <br />                                                         remarkable features were prominent in the DnaD&#226;&#8364;&#8220;pBR322 structures. In many images, <br />                                                         recorded in both air and liquid, there was a build up of DnaD inside the circular plasmid and <br />                                                         also at foci on the pBR322 plasmid (Fig. 2C). Similar features have been observed for the <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         HU protein [28]. Our data thus suggest that DnaD can impose global DNA architectural <br />                                                         transitions, a property that may also be biologically important during primosomal assembly. <br />                                                         DnaD may be forming large scaffolds for global (or localized in the vicinity of oriC) <br />                                                         remodeling of the bacterial nucleoid rendering it replication proficient, while at the same <br />                                                         time attaching the bacterial chromosome to the membrane via its interaction with the <br />                                                         membrane associated DnaB primosomal protein (see below). In this respect, the DnaD&#226;&#8364;&#8220; <br />                                                         DnaB interaction may be functionally analogous to the RacA&#226;&#8364;&#8220;DivIVA interaction near the <br />                                                         oriC that attaches the chromosome to the membrane during B. subtilis sporulation [30]. <br />  <br />                                            3.4. DnaD disrupts the helicase&#226;&#8364;&#8220;DnaI complex in vitro <br />                                                         The precise role of DnaD in primosomal assembly is still unknown. It appears to act early on <br />                                                         in primosomal assembly [15,16,31], while its interaction with the membrane associated <br />                                                         DnaB protein may provide a mechanism for regulating the initiation of DNA replication in <br />                                                         B. subtilis [20]. We have previously reported a stable interaction in vitro between the <br />                                                         putative B. subtilis helicase loader DnaI and the main replicative ring helicase DnaB from B. <br />                                                         stearothermophilus [17]. In order to avoid confusion with the nomenclature, we should make <br />                                                         it clear that the main replicative helicase in B. subtilis is known as DnaC and is homologous <br />                                                         to B. stearothermophilus DnaB (82% identity, 92% similarity). We investigated the effect of <br />                                                         DnaD on the DnaB&#226;&#8364;&#8220;DnaI complex and found that in vitro DnaD disrupts the complex (Fig. <br />                                                         3A). Incubating all three DnaD, DnaB, and DnaI proteins together disrupted the DnaB&#226;&#8364;&#8220;DnaI <br />                                                         interaction whilst, no complex could be detected between either DnaB&#226;&#8364;&#8220;DnaD (Fig. 3B) or <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                                  Page 6 <br />  <br />  <br />                                                         DnaI&#226;&#8364;&#8220;DnaD (data not shown). The biological significance of this observation may be <br />                                                         relevant to the process of helicase loading. The interaction of the DnaI&#226;&#8364;&#8220;DnaC complex with <br />                                                         DnaD may facilitate DnaC release and loading onto the DNA. <br />  <br />                                            4. Model <br />                                                         We have provided evidence that DnaD forms large nucleoprotein assemblies affecting <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         global DNA architecture and that it disrupts a complex between a ring helicase and a <br />                                                         helicase loader. We propose that DnaD is the link between nucleoid reorganization and <br />                                                         initiation of DNA replication in B. subtilis (Fig. 3C). It has been suggested previously that <br />                                                         initiation of DNA replication is regulated by recruitment of DnaD to the membrane [20]. <br />                                                         Maintaining DnaD in the cytoplasm prevents excessive over-initiation and its regulated <br />                                                         recruitment to the membrane by DnaB (referring to the primosomal protein from B. subtilis <br />                                                         and not the replicative helicase from B. stearothermophilus) activates initiation of DNA <br />                                                         replication [20]. DnaD may bind to chromosomal DNA forming a scaffold to open up the <br />                                                         nucleoid (either locally in the vicinity of oriC or globally) and render it replication <br />                                                         proficient. DnaC is recruited from the DnaI&#226;&#8364;&#8220;DnaC complex in the cytoplasm to the <br />                                                         membrane attachment site by a direct interaction with DnaB [16-18]. DnaD will then disrupt <br />                                                         the DnaI&#226;&#8364;&#8220;DnaC complex thus facilitating its loading onto the oriC. DnaD provides a <br />                                                         possible link between nucleoid reorganization and initiation of DNA replication. Its <br />                                                         interaction with the membrane associated DnaB may be the prime regulatory event that <br />                                                         couples chromosomal remodeling to DNA replication. <br />  <br />   <br />  <br />                                                         Fig. 1. <br />                                                         (A) Analysis of DnaD oligomers by sedimentation velocity ultracentrifugation. Data were <br />                                                         analyzed by the c(s) method of Schuck [23] and errors were plotted at the 95% confidence <br />                                                         limit. At low 1.2 &#206;&#188;M (blue lines), there is a species corresponding to a monomer of DnaD. <br />                                                         At 2.4 &#206;&#188;M (black) and 4.9 &#206;&#188;M (red), species that correspond to dimers or trimers and to <br />                                                         higher order aggregates of at least pentamers and hexamers are apparent. (B) AFM images <br />                                                         in air of DnaD at increasing concentrations. The arrow indicates increasing concentrations <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         (0.49, 4.9 and 49 nM). The formation of `chains&apos; of individual DnaD molecules can be seen <br />                                                         clearly with higher oligomers evident as concentration increases. Scale bars represent 100 <br />                                                         nm. <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                           Page 9 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Fig. 2. <br />                                                         (A) Gel shift analysis of DnaD binding to an oligonucleotide at increasing concentrations. <br />                                                         Binding reactions were carried out at 2.5 nM oligonucleotide substrate and 123, 246, 369, <br />                                                         492, 615 and 738 nM DnaD, lanes 1&#226;&#8364;&#8220;6, respectively. Lane C shows the control radioactively <br />                                                         labeled oligonucleotide. (B) Control AFM images of naked pBR322 plasmid in air. A <br />                                                         typical field view and a few representative individual pBR322 molecules adopting a variety <br />                                                         of compact supercoiled structures are shown. The scale bars in the field view and individual <br />                                                         images represent 250 and 50 nm, respectively. (C) AFM topography images of DnaD&#226;&#8364;&#8220; <br />                                                         pBR322 complexes. A typical field view in air and a few representative individual <br />                                                         complexes in air and liquid are shown. All complexes show the DNA in an open circular <br />                                                         conformation around the periphery of the circular structures with DnaD molecules forming <br />                                                         large scaffolds inside the circles. In several images, DnaD also appears to form large foci on <br />                                                         the pBR322 plasmid around the periphery of the circles. The scale bars in the field views <br />                                                         and individual images represent 1 &#206;&#188;m and 100 nm, respectively. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                         Page 10 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Fig. 3. <br />                                                         (A) Gel filtration analysis of a DnaB/DnaI/DnaD mixture showing that no complexes can be <br />                                                         detected when all three proteins are incubated together. (B) Gel filtration analysis of DnaB/ <br />                                                         DnaI and DnaB/DnaD mixtures showing the formation of a DnaB&#226;&#8364;&#8220;DnaI complex but no <br />                                                         stable complex between DnaB and DnaD. In a similar manner DnaI did not form a stable <br />                                                         complex with DnaD (data not shown). In all cases, samples from the peaks were analyzed by <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         SDS&#226;&#8364;&#8220;PAGE and the numbers on the peaks correspond to the numbers of the lanes. Lanes M, <br />                                                         4, 5, 6 indicate molecular weight markers and controls of DnaB, DnaI and DnaD proteins, <br />                                                         respectively. In both panels A and B, DnaB refers to the replicative helicase from B. <br />                                                         stearothermophilus. (C) Speculative model for the coordination of nucleoid remodeling and <br />                                                         initiation of DNA replication in B. subtilis. Indicated stoichiometries of complexes and the <br />                                                         membrane attachment site are not precise and have not been established. The sizes of <br />                                                         various components depicted are not drawn to size. Although DnaD is depicted to remodel <br />                                                         the whole of the nucleoid, also localized remodeling in the vicinity of the oriC may be the <br />                                                         case. DnaB refers to the primosomal protein from B. subtilis. <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />                                          Turner et al.                                                                                                                           Page 11 <br />  <br />  <br />  <br />  <br />                                                                                                           TABLE 1 <br />                                         Table of possible frictional coefficients versus species type for DnaD <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                          Species        2s       4s       6s        9s <br />  <br />                                          Monomer        1.488 <br />  <br />                                          Dimer          2.363    1.182 <br />  <br />                                          Trimer         3.059    1.549    1.032 <br />  <br />                                          Tetramer       3.750    1.875    1.250 <br />  <br />                                          Pentamer       4.352    2.176    1.451 <br />  <br />                                          Hexamer        4.914    2.457    1.638     1.092 <br />  <br />                                          Heptamer       5.446    2.723    1.815     1.210 <br />  <br />                                          Octamer        5.953    2.977    1.984     1.323 <br />  <br />                                          Nonamer        6.440    3.220    2.147     1.431 <br />  <br />                                          Decamer        6.908    3.454    2.3027    1.535 <br />  <br />                                         Physically unreasonable values are in grey. Frictional coefficients are calculated using the program SEDNTERP [24]. Higher values indicate a <br />                                         more elongated molecule. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                       FEBS Lett. Author manuscript; available in PMC 2011 February 04. <br />   <br /> </body></html>