<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Panjkovich et al. BMC Genomics 2014, 15:36 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br />  METHODOLOGY ARTICLE                                                                                                                            Open Access <br />  <br /> antibacTR: dynamic antibacterial-drug-target <br /> ranking integrating comparative genomics, <br /> structural analysis experimental <br /> annotation <br /> Alejandro Panjkovich1 , Isidre Gibert1,2 Xavier Daura1,3* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Development novel antibacterial drugs urgent healthcare necessity partially <br />   neglected field. The decades seen substantial decrease discovery novel antibiotics, which <br />   combined recent thrive multi-drug-resistant pathogens generated scenario general concern. The <br />   procedures involved discovery development novel antibiotics economically challenging, time <br />   consuming lack warranty success. Furthermore, return-on-investment antibacterial drug is <br />   usually marginal compared therapeutics, explains decrease private investment. <br />   Results: In work present antibacTR, computational pipeline designed aid researchers selection of <br />   potential drug targets, initial steps antibacterial-drug discovery. The approach designed and <br />   implemented publicly funded initiatives aimed discovering novel antibacterial targets, mechanisms <br />   drugs priority list Gram-negative pathogens: Acinetobacter baumannii, Escherichia coli, Helicobacter pylori, <br />   Pseudomonas aeruginosa Stenotrophomonas maltophilia. However, present list extended cover <br />   total 74 fully sequenced Gram-negative pathogens. antibacTR based sequence comparisons queries <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />   multiple databases (e.g. gene essentiality, virulence factors) rank proteins according potential as <br />   antibacterial targets. The dynamic ranking potential drug targets easily executed, customized accessed <br />   user web interface integrates computational analyses performed in-house and <br />   visualizable on-site. These include three-dimensional modeling protein structures prediction active sites <br />   functionally relevant ligand-binding sites. <br />   Conclusions: Given versatility ease-of-use integrating experimental annotation computational <br />   analyses, antibacTR effectively assist microbiologists, medicinal-chemists researchers working the <br />   field antibacterial drug-discovery. The public web-interface antibacTR available &#226;&#8364;&#732;http://bioinf.uab.cat/ <br />   antibactr&#226;&#8364;&#8482;. <br />   Keywords: Antibacterial-drug target, Gram-negative bacteria, Target druggability <br />  <br />  <br />  <br />  <br /> *Correspondence: xavier.daura@uab.cat <br /> 1 Institute Biotechnology Biomedicine (IBB), Universitat Aut&#195;&#178;noma de <br /> Barcelona (UAB), 08193 Bellaterra, Spain <br /> 3 Catalan Institution Research Advanced Studies (ICREA), 08010 <br /> Barcelona, Spain <br /> Full list author information available end article <br />  <br />  <br />                                          &#194;&#169; 2014 Panjkovich et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                          Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                          reproduction medium, provided original work properly cited. <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                       Page 2 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> Background                                                      The approach based protein sequence comparisons, <br /> Since initial discovery application        developed unbiased measure described in <br /> early 20th century, antibiotics playing key role    Methods Section. <br /> in public health worldwide. These &#226;&#8364;&#732;miracle drugs&#226;&#8364;&#8482;             The interface used interrogate database and <br /> contributed significantly increase life expectancy    access results form web-based tool, <br /> since end World War II. Besides curing infec-            developed following suggestions of <br /> tions, prevent amputations blindness          experimentalists involved target-discovery <br /> are involved multiple healthcare procedures          initiatives. It includes access thousands three- <br /> joint-replacement, surgery, new cancer treatments,          dimensional protein-structure models visu- <br /> [1]. However, peaking 1960&#226;&#8364;&#8482;s, dis-         alized downloaded analysis the <br /> covery new antibiotics fallen dramatically. The      web-interface. To exploit structural models, we <br /> present scarcity novel antibiotics major           incorporated predictive approach evaluates puta- <br /> health concern light remarkable ability bacte-     tive ligand-binding pockets terms potential to <br /> ria rapidly evolve resistance mechanisms erode         affect protein function ligand binding [12,13]. Links <br /> the therapeutic effect known antibiotics [2]. Nowa-          DrugBank [14] Virulence Factors DataBase <br /> days multi-drug-resistant bacterial infections increas-     (VFDB) [15], predictions active-site residues <br /> ing developing developed countries           provided well. It noted final <br /> both community nosocomial settings [3]. It         aim tool rank proteins according the <br /> reported number pathogens, including Staphylo-        chosen set criteria (with weights defined user) <br /> coccus aureus, Mycobacterium tuberculosis, Pseudomonas          provide protein ranked list infor- <br /> aeruginosa, Acinetobacter baumannii Enter-             mation relevant antibacterial-drug-target <br /> obacteriaceae developed resistance wide range         selection. Clearly, drug-target property result <br /> of antimicrobial agents alarming rise,          complex combination variable number non- <br /> strains truly pan-resistant [4,5]. However, phar-      universal factors, having opposite sign for <br /> maceutical companies investing             different types targets mechanisms. Thus, is <br /> development new antibacterial drugs correspond-         tool support target discovery efforts, target- <br /> ing efforts, mainly economic criteria favour        prediction tool. In words, spare user <br /> other therapeutic areas better return-on-investment        scanning evaluating large number proteins, <br /> ratios [6,7]. The antibacterial agents       simply provide him/her additional means to <br /> launched decade (e.g. linezolid, dapto-         it. <br /> mycin) good activity Gram-positive bac-             All measurements predictions pre-computed, <br /> teria methicillin-resistant S. aureus (MRSA)        allows application return rankings and <br /> vancomycin-resistant enterococci [8]. However, cases         links relevant information seconds. This <br /> resistance new Gram-positive antibiotics         characteristic distinguishes tool similar ones <br /> been reported recently [9].                             UniDrug-Target (UDT) database, can <br />    The situation worse Gram-negative bacteria,      used perform comparative analyses online with <br /> as P. aeruginosa A. baumannii, common             computation time request [16]. Besides execution <br /> among nosocomial infections [10] new           speed, approach differs UDT related ones <br /> antibiotics reached advanced stages development         Prokaryotic-genome Analysis Tool (PGAT) <br /> [1]. In addition, increase prevalence        [17] focus. While tools succeed providing <br /> extended spectrum &#206;&#178;-lactamase (ESBL)-producing Enter-           comparative-analysis means used a <br /> obacteriaceae, use carbapenems, potential alter-       web-interface, dynamic approach focuses speed, <br /> native treat infections caused microorganisms,      ease use integrative solution allows user <br /> is leading emergence multi-drug-resistant Enter-      quickly scan putative antibacterial targets relevant <br /> obacteriaceae including resistance carbapenems [11].         information three-dimensional structural models <br />    This scenario emphasizes relevance initiatives        predictions, comparative analysis is <br /> focused discovery novel targets antibacte-        just underlying features. <br /> rials combating Gram-negative pathogens. Here,              Other researchers focused extensive, manu- <br /> describe tool (antibacTR: antibacterial Target Rank-          ally curated analyses single organism strain, like <br /> ing) support initial stages selection potential    Shanmugham Pan [18] Mycobacterium abscessus <br /> antibacterial-drug targets, developed context     ATCC 19977. Instead, chosen sacrifice of <br /> two initiatives. antibacTR integrates database      depth analysis cover larger selec- <br /> a pipeline ranks filters proteins according set   tion pathogens. The methodology presented was <br /> of criteria commonly associated antibacterial targets.       originally developed specific projects their <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                          Page 3 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> target bacteria, including big Gram-           [16,21]. In case, experimental analysis <br /> negative pathogens relation systemic infections,           reveals given protein suitable drug target, <br /> i.e. Escherichia coli, Klebsiella pneumoniae, Acinetobac-        work continue protein ranking. <br /> ter baumannii Pseudomonas aeruginosa [4]. However,           Moreover, straightforward incorporate new <br /> we expanded coverage include 74 Gram-            criteria ranking scheme needed. <br /> negative pathogens, make useful            The pipeline proteome was <br /> to researchers field. Through arti-        subjected illustrated Figure 1, summarizes the <br /> cle, approach make web-based             approach. <br /> tool publicly available. <br />                                                                  Sequence-based analysis <br /> Results                                                          Currently, database covers 74 Gram-negative <br /> Computational target-ranking pipeline                            pathogens, including 224 distinct strains. In case of <br /> Typically, steps target-discovery          Acinetobacter baumannii, Escherichia coli, Helicobacter <br /> project readily select, thousands pro-            pylori, Pseudomonas aeruginosa Stenotrophomonas <br /> teins composing pathogens&#226;&#8364;&#8482; proteomes,         maltophilia, pathogens distinguished their <br /> highest chance useful therapeutic targets.           prevalence community and/or nosocomial infections <br /> Following lines defined previous studies [19],         incidence drug-resistant isolates [3,10], we <br /> developed algorithm score rank potential drug          included available fully sequenced strains (82 strains, <br /> targets pathogenic organisms evaluating modular          conform &#226;&#8364;&#732;priority set&#226;&#8364;&#8482;). As rest the <br /> set criteria commonplace antimicrobial-           species, included strains marked as <br /> development efforts [7]: 1) presence protein          &#226;&#8364;&#732;human pathogens&#226;&#8364;&#8482; HAMAP (142 strains) [22]. This <br /> in different pathogens, 2) evolutionary conservation, 3)         query data set compared human proteome <br /> essentiality, 4) presence isoforms paralogs        reference set 770 Gram-negative proteomes (494 <br /> proteome, 5) similarity human proteins. We imple-             distinct species), means BLASTPGP program <br /> mented set weighted scores cover            [23] using default parameters. BLASTPGP searches are <br /> criteria defined scoring function combining them.          fast, resulting E-values depend align- <br />   The concepts incorporated inde-          ment itself parameters size the <br /> pendent scores, measuring conservation protein        database scanned. We needed unbiased similarity scores <br /> among Gram-negative organisms different                proteins matched sequence-based <br /> strains species, respectively. Conservation          search results valid case increasing <br /> among strains basic requirement target considera-       size data sets. To attain objective, fur- <br /> tion. Conservation Gram-negative species highly         ther aligned BLASTPGP matches (E-value &lt;= 0.0001) <br /> desirable enables development broad-spectrum        using Smith-Waterman algorithm calculated <br /> solutions increases economic viability. In addition,         &#226;&#8364;&#732;normalized sequence similarity scores&#226;&#8364;&#8482; (NS). NS values <br /> well conserved targets presumably low toler-           used pre-compute toxicity, presence iso- <br /> ance mutations, decreasing chance resistance        forms paralogs conservation scores for <br /> emerge type mechanism.                                protein query data set, described further <br />   Essential proteins, inhibition compromises bac-          Methods Section. <br /> terial viability, potential antibacterial targets defi- <br /> nition. We implemented binary score marking genes           Queries external databases <br /> known essential previous experimental work            Besides comparative analysis, round database queries <br /> [20].                                                            performed integrate additional information. <br />   The remaining scores given negative weights.           Thus, protein sequences compared list of <br /> If protein consideration isoforms and/or           known drug-targets available DrugBank database <br /> paralogs pathogen readily develop resistance          [14] sequence-based searches performed <br /> functional substitution, effect antibac-          virulence factors available Virulence Factors <br /> terial reduced competitive binding             DataBase (VFDB) [15]. Out total 777,585 proteins <br /> non-essential forms. We considered similarity human           query data set, 375,016 matched known target in <br /> proteins negative well, close human homologs            DrugBank (48%) 193,135 proteins matched known <br /> to target interact drug, giving rise         virulence factor VFDB (25%). This information not <br /> unwanted side-effects.                                           incorporated ranking scores, available through <br />   The scoring ranking scheme, partially following            web-interface described researchers eval- <br /> the work White Kell [19] provides advantage            uate relevance matches each <br /> when compared static selection filtering approaches        particular case. <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                                                  Page 4 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br />  Figure 1 Pipeline flowchart. Each proteome subjected computational pipeline depicted flowchart. Each protein <br />  sequence compared using BLASTPGP against: (1) human proteome look similarities lead toxicity, (2) Gram-negative <br />  bacteria estimate evolutionary conservation, (3) proteome isoforms paralogs reduce effectivity promote <br />  resistance, (4) database essential genes (DEG). The results queries combined scoring function described main text. <br />  To integrate relevant information, protein sequences queried known drug targets (DrugBank) virulence factors (VFDB). In <br />  complementary manner, active sites predicted and, proper structural templates found, protein structure modeled by <br />  homology. If structure modeled successfully, ligand-binding sites predicted analyzed described main text. The pipeline&#226;&#8364;&#8482;s <br />  output ranked list organism&#226;&#8364;&#8482;s proteins based comparative genomics analysis (i.e. essentiality, evolutionary conservation, toxicity <br />  paralogs scores) links external databases (VFDB DrugBank) possible, corresponding structural modeling and <br />  analysis results. <br />  <br />  <br />  <br />  <br /> Three-dimensional homology modeling                                           position active site annotated. After <br /> Researchers evaluating prospective drug targets                           analyzing query set (777,585 proteins), pro- <br /> benefit availability protein structural data.                     cedure predicted location active-site residues for <br /> For organisms priority set, performed                         90,482 proteins (11.6%). <br /> large-scale homology modeling protein sequences                          Proteins predicted active site display link the <br /> for valid structural templates explained                    details prediction web interface described <br /> in Methods Section. In total, generated three-                         below. <br /> dimensional homology models 136,141 proteins (cov- <br /> ering 47% priority set). This number obtained                      Pocket analysis <br /> after discarding models presenting 30% sequence                     For proteins build three-dimensional <br /> identity (target-template) G-factors -1.00 [24].                     homology model, predicted location ligand- <br /> All models generated means MODELLER                            binding sites structure using LIGSITEcs [27]. <br /> program [25] using default parameters.                                        We analyzed ligand-binding sites using two <br />   To save computational power, proteins belonging                          previously developed methodologies estimate the <br /> other strains modeled automatically. However,                     regulatory potential particular ligand-binding pockets. <br /> the user interested obtaining homology                      When possible, structural conservation predicted <br /> models, implemented option web-                             pockets measured considering evolutionary <br /> interface automatic submission selected model-                     record protein family, given conserved pock- <br /> ing task.                                                                     ets relevant biological role [12]. Furthermore, <br />                                                                               using Normal Mode Analysis estimated effect of <br /> Active-site prediction                                                        ligand binding overall protein flexibility, measure <br /> To add relevant information putative targets,                      used combination structural con- <br /> we applied sequence-based approach [26] predict                          servation predict location allosteric sites [13]. <br /> the location active-site residues. The method based                     As described below, user visualize protein <br /> on comparing query sequences homologs                            structure predictions online. <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                         Page 5 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> Interface access results                                  drug discovery criticized limited <br /> Interactive access results available web-      success finding new active compounds [31,32]. Yet, <br /> interface &#226;&#8364;&#732;http://bioinf.uab.cat/antibactr&#226;&#8364;&#8482;.                  comparative genomics proteomics continue shed <br />    This interface allows user select organisms        light workings bacterial drug-resistance and <br /> and strain interest, set custom weights different      virulence [33,34]. <br /> scores proceed calculate corresponding              Far attempting solve problem tar- <br /> ranking. If user wishes ignore specific ranking         identification strike, motivation to <br /> parameter, weight 0 (zero) applied. The       implement straightforward computational approach <br /> has built way normalization scores        prove useful initial filtering rank- <br /> is performed selected set strains          ing step, aiding researchers quest novel drug <br /> parameters. To facilitate analysis results,   targets. <br /> user limit top-ranked entries           At time writing equivalent tool one <br /> are displayed. Once ranking procedure finished (it        presented available, servers provide <br /> takes seconds), ranking printed browser.     slightly related functionalities used com- <br /> An option available downloading ranking        plementary fashion. For example Prokaryotic-genome <br /> local tab-delimited text format, useful          Analysis Tool (PGAT), developed Brittnacher col- <br /> researchers interested processing data. Tar-      laborators general comparative genomics tool focused <br /> gets displayed ranked order individual scores         particularly comparing different strains same <br /> are shown protein normalization prior         species [17]. PGAT allows user carry series <br /> to weighting. A brief description biological function     interesting analyses including information metabolic <br /> is displayed protein but, facilitate immediate       pathways, does provide specific drug-target <br /> access annotation relevant data, link        related information, unlike UniDrug-Target (UDT) <br /> to related Uniprot entry provided [28]. In        database clearly focuses aspect [16]. The <br /> cases target shows sequence similarity           presents candidate targets proteins are <br /> already known drug target virulence factor, cor-          present pathogenic bacteria absent commen- <br /> responding links provided. In addition, specific        sal strains. This reasonable approach our <br /> links details predicted active sites homol-          tool achieved setting negative value the <br /> ogy models given. If homology model supplied,           strain conservation score functions <br /> the user download model coordinates PDB for-              available PGAT. However, given focus pathogen- <br /> mat target-template alignments generated          specific proteins UDT&#226;&#8364;&#8482;s approach tends discard evo- <br /> modeling process, sequence identity, DOPE             lutionary conserved proteins, known <br /> score relevant modeling data [25]. Furthermore,        broad-spectrum targets [35]. <br /> available protein structures visualized using Jmol           The known characterized protein <br /> (http://www.jmol.org) results pocket       drug targets currently order hundreds [35]. <br /> analysis previously described [13].                              To illustrate potential tool presented here, <br />                                                                  provide examples known antibac- <br /> User query sequences                                             terial targets. Certain proteins involved replica- <br /> Besides ranking complete proteomes, researchers           tion DNA targeted fluoroquinolones [35,36]. <br /> may want look ranking selected proteins       For example, Ciprofloxacin targets DNA topoisomerase 4 <br /> of particular interest. To achieve functionality,     subunit B DNA gyrase subunit A. Even resis- <br /> we added possibility include user&#226;&#8364;&#8482;s query         tance fluoroquinolones observed pathogens <br /> sequences optional field. These sequences         mutations proteins, new compounds with <br /> compared using BLASTPGP query data set               antibacterial activity resistant strains being <br /> (224 strains). Scoring ranking proceed normally,          developed studying targets [37]. These pro- <br /> but results displayed significant hits         teins appear 3% proteome ranking <br /> within data set. Details BLASTPGP search         Escherichia coli K12 (positions 68 158, respec- <br /> also available user.                                      tively) build ranking using default param- <br />                                                                  eters. This proteins essential, show <br /> Discussion                                                       high levels evolutionary conservation low similar- <br /> Large-scale comparison organisms genome level          ity human proteins (minimal potential toxicity) and <br /> is technique common fields biology              present isoforms paralogs according pipeline <br /> medicine. In past years complex approaches involv-           parameters. <br /> ing phylogenetic metabolic studies pub-               Beyond filtering ranking targets, researchers can <br /> lished [29,30]. However, comparative genomics initiatives        gain insight potential targets the <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                                                 Page 6 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> structural analysis methods implemented                        Briefly, case illustrates situation poorly <br /> the tool. For example, peptide deformylase [Swiss-                          characterized protein, automatic structural analysis <br /> Prot:Q9I7A8] Pseudomonas aeruginosa essen-                       pinpoint potential protein a <br /> tial protein targeted antibiotic Actinonin [38]. This                drug target, precise location drug-binding <br /> protein ranked position 81 complete                         pocket. <br /> Pseudomonas aeruginosa proteome (top 2%) using                           Another interesting example 3-oxoacyl-[acyl-carrier- <br /> default parameters. Our pipeline automatically builds                       protein] synthase III, involved fatty-acid <br /> three-dimensional homology models possible,                         synthesis. This protein targeted Cerulenin, anti- <br /> then predicts putative ligand-binding sites evaluates                   fungal effects, potential antibacterial target as <br /> their potential regulate protein activity described                   [39-41]. It ranked pipeline position 49 <br /> previously [13]. In case, structural analysis (which               (top 2%) proteome Escherichia coli K12 <br /> is pre-calculated available web-interface)                  essential conserved, showing prin- <br /> predicts putative ligand-binding sites signifi-               ciple toxicity (similarity human proteins). Further- <br /> cantly affect protein flexibility shown Figure 2. When                more, structural analysis reveals single pocket that <br /> we superimpose automatically generated homology                         affect protein&#226;&#8364;&#8482;s function perturbing over- <br /> model known structure protein bound                         flexibility, shown Figure 3. When evaluate the <br /> to antibiotic ([PDB:1LRY] RMSD 0.5) position                        structural prediction performed homology model <br /> of cavity predicted significant matches pre-                      superimposing known structure inhibitor- <br /> cisely location antibiotic molecule. This cavity                 bound protein, observe case match of <br /> is considered relevant evolutionary perspec-                   cavity&#226;&#8364;&#8482;s geometric center precise the <br /> tive, shows 100% structural conservation                previous example peptide deformylase. Nevertheless, <br /> domain family according corresponding automatic                      visual inspection shows large cavity indicated by <br /> analysis [12]. The structural conservation pocket                   pocket structural analysis occupied the <br /> was expected, protein&#226;&#8364;&#8482;s active site. <br />  <br />  <br />  <br />  <br />  Figure 2 Peptide deformylase, automatic structural modeling                 Figure 3 Automatic structural modeling pocket analysis <br />  pocket analysis. The spheres displayed homology model            3-oxoacyl-[acyl-carrier-protein] synthase 3. The spheres <br />  Pseudomonas aeruginosa peptide deformylase [Swiss-Prot:Q9I7A8]           displayed Escherichia coli&#226;&#8364;&#8482;s 3-oxoacyl-[acyl-carrier-protein] <br />  (based template [PDB:1N5N]) represent putative ligand-binding            synthase 3 [Swiss-Prot:P0A6R0] homology model (based template <br />  sites predicted automatic pocket analysis. The orange             [PDB:1UB7]) represent putative ligand-binding sites predicted by <br />  sphere marks cavity predicted significantly affect overall      automatic pocket analysis. The orange sphere marks only <br />  protein flexibility. To illustrate relevance prediction,     cavity predicted significantly affect overall protein flexibility. This <br />  location antibiotic ligand (in &#226;&#8364;&#732;sticks&#226;&#8364;&#8482; representation)     predicted position occupies cavity close (4.1 &#195;&#8230;) <br />  superimposing homology model known structure            known inhibitor molecule (in &#226;&#8364;&#732;sticks&#226;&#8364;&#8482; representation) after <br />  antibiotic-bound protein [PDB:1LRY] (RMSD 0.5 &#195;&#8230;). The position       superimposing homology model known structure the <br />  antibiotic Actinonin matches precisely cavity marked             inhibitor-bound protein [PDB:1MZS] (RMSD 1.1 &#195;&#8230;). This shows how <br />  procedure. The cavity estimated                automatic homology modeling pocket analysis combined to <br />  conserved structural level (100% presence protein family).    correctly identify regulatory cavity. <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                                 Page 7 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> inhibitor, automatic procedure                     Because function unequivocally assign <br /> be pointing researcher right direction,                 target scores proteins highly complex full <br /> if inhibition information available                  unknowns, pipeline presented been <br /> a priori.                                                               developed sole aim assist selection of <br />    Of course, multiple factors reach                   prospective candidates, meant provide final <br /> of mere computational approach participate defin-                  complete list antibacterial targets. Very often, it <br /> ing protein good antibacterial target. Indeed                used ranking tool retrieve target- <br /> final outcome clinical trials hardly predicted                relevant information specific protein evaluate <br /> [31]. However, considered relevant infer                 pros cons respect potential candi- <br /> the pipeline presented work ranks known                 dates. The versatility tool, dynamic features <br /> targets. We gathered known targets &#226;&#8364;&#732;approved&#226;&#8364;&#8482;                    assignation relative weights user&#226;&#8364;&#8482;s cri- <br /> drugs, annotated DrugBank [14], belonged                  terion, availability original information, such <br /> any strains analyzed work. We                    predicted, functionally relevant ligand-binding sites, <br /> total 57 proteins identified Uniprot ID                prove valuable arguments microbiologist <br /> [28]. Of 57, majority (48) belong Escherichia                medicinal chemist researching new antibacterial <br /> coli K12. When proceed rank organism&#226;&#8364;&#8482;s pro-                  targets. <br /> teome, half known drug targets appear top <br /> 10% ranking. The distribution 48 known tar-               Conclusions <br /> gets ranking Escherichia coli K12 displayed                We developed database web-based tool the <br /> in Figure 4. It interesting note half           ranking proteins set user-selected bacterial <br /> top 10% correspond essential proteins. Since essential-              proteomes according series antibacterial-drug- <br /> ity binary score (i.e. genes essential not),             target-like properties. Specifically, tool evaluates (1) <br /> it divides ranking sections seen                presence protein Gram-negative species <br /> the histogram (Figure 4). This illustrates difficulty            (currently 494) assess conservation level and, <br /> a priori ranking antibacterial targets,               extension, potential spectrum drug targetting <br /> essentiality considered desirable property            protein, (2) identification protein essen- <br /> candidate target [32,42,43], represents half             tial bacterial <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>survival</span> growth, (3) presence of <br /> known targets organism. Moreover, assessing gene                homologs human genome assess potential tox- <br /> essentiality trivial task, given vitro results         icity effects drug targetting protein and <br /> do correlate gene essentiality determined               (4) presence paralogs isoforms protein in <br /> in vivo [43].                                                           proteome, facilitate develop- <br />                                                                         ment resistance reduce effectivity. The user may <br />                                                                         choose weight of properties in <br />                                                                         ranking, including negative weights (e.g. homol- <br />                                                                         ogy human presence isoforms paralogs). <br />                                                                         When available, ranked proteins incorporate, addi- <br />                                                                         tional information, matches proteins the <br />                                                                         DrugBank (i.e. known drug targets) Virulence <br />                                                                         Factors Database (VFDB), model pro- <br />                                                                         tein&#226;&#8364;&#8482;s three-dimensional structure analysis active <br />                                                                         regulatory sites assess druggability poten- <br />                                                                         tial target. This currently single resource combining <br />                                                                         sequence-based structural information identi- <br />                                                                         fication potential antibacterial-drug targets multiple <br />                                                                         genomes. <br />                                                                            Currently, database covers 74 Gram-negative <br />                                                                         pathogens, including 224 distinct strains. The num- <br />  Figure 4 Ranks distribution known antibacterial targets. The       ber strains particularly extended Acinetobacter <br />  histogram displays distribution ranks 48 DrugBank known     baumannii, Escherichia coli, Helicobacter pylori, Pseu- <br />  targets Escherichia coli strain K12 complete proteome    domonas aeruginosa Stenotrophomonas maltophilia <br />  ranking using default parameters. Target proteins known          (82 strains total), group <br />  essential according DEG appear higher ranking positions (solid <br />                                                                         Gram-negative pathogens distinguished prevalence <br />  black), non-essential proteins appear the <br />  ranking (hatch <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span>).                                               community nosocomial infections inci- <br />                                                                         dence drug-resistant isolates. Additional strains and <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                        Page 8 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> species added database subsequent             Evolutionary conservation Gram-negative <br /> updates.                                                        organisms <br />                                                                 We defined score estimate evolutionary con- <br /> Methods                                                         servation potential targets Gram-negative (GN) <br /> Normalized sequence-similarity score (NS)                       organisms follows: <br /> We used BLASTPGP program [23] default pa- <br /> rameters scan complete proteomes. Since BLASTPGP                              <br />                                                                               i=n&#226;&#710;&#8217;1 <br />                                                                    GNC p =            max(NSp )                           (1) <br /> E-values vary depending size queried                         i=1 <br /> database, aligned matched pairs calculated their <br /> Smith-Waterman similarity score [44]. We ignored align-         Where GNC p Gram-negative conservation score <br /> ments scores lower 100, previously described       protein p, computed adding highest NS value <br /> [45].                                                           (max(NSp )) obtained different GN <br />    Given Smith-Waterman similarity score            species (i) data set, n total number of <br /> related size alignment, divided score      GN species. <br /> by <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> alignment obtain normalized <br /> sequence-similarity score (NS). The Smith-Waterman              Conservation strains <br /> algorithm computes optimal local alignment, meaning          We estimated evolutionary conservation proteins <br /> that NS measure similarity proteins          different strains species using the <br /> is equivalent similarity closely      following score: <br /> related pair domains regions.                                         j=m&#226;&#710;&#8217;1 <br />                                                                               j=1 max(NSp ) <br />                                                                     SC p =                                         (2) <br /> Essentiality                                                                      m <br /> Experimental information regarding gene essentiality         SC p strain conservation score protein <br /> available organisms database essential      p, computed adding highest NS value (max(NSp )) <br /> genes (DEG) [20]. If particular strain avail-         obtained strain ( j) selected <br /> able DEG, mapped query proteins essential              species data set, m total number distinct <br /> genes using BLASTPGP. For annotated essen-              strains particular species. <br /> tial gene related strain, scanned proteome <br /> of marked best hit essential             Scoring function ranking potential drug targets <br /> gene. Only E-values 1e-10 better considered          Each different scores normalized largest <br /> acceptable task. At time writing,       value obtained selected organisms. Normalized <br /> were able gather large-scale essential gene infor-      values multiplied 100 obtain percentages, <br /> mation for: Acinetobacter baumannii, Escherichia coli,          i.e. final scores range 0 100. <br /> Helicobacter pylori, Pseudomonas aeruginosa Vibrio             Each independent score associated weight, which <br /> cholerae.                                                       negative positive. These weighting values can <br />                                                                 set user. However, default values provided <br /> Toxicity                                                        follows. A priori negative features putative target <br /> An antibacterial drug acting protein targets       (i.e. Toxicity Paralogs) given default weight - <br /> similar human proteins bind causing           1, positive features (e.g. Evolutionary conservation, <br /> adverse effects and/or toxicity. We estimated potential     Essentiality) corresponding default weight 1. <br /> toxicity putative target proportional largest       For protein selected data set, normalized <br /> NS value obtained pairwise alignment          scores multiplied respective weights. The final <br /> whole human proteome.                                           score protein obtained summing all <br />                                                                 weighted scores. Finally, proteins selected data <br /> Isoforms paralogs                                           set ranked according final score terms of <br /> If given drug target presents multiple isoforms par-       drug-target potential. <br /> alogs (&#226;&#8364;&#732;variants&#226;&#8364;&#8482;), pathogen readily develop resis- <br /> tance functional substitution mechanisms. It         Comparative-genomics reference data set <br /> possible drug bind target         Sequence data Gram-negative (GN) organisms was <br /> variants, decreasing antibiotic effect. To assess      gathered total 770 fully sequenced GN pro- <br /> this parameter potential drug target, counted       teomes covering 494 distinct species. GN bacteria species <br /> the variants present proteome. We         identified &#226;&#8364;&#732;http://www.bacterialphylogeny.com/ <br /> considered variants protein similar proteins   bacteria.html&#226;&#8364;&#8482; fully sequenced bacterial proteomes <br /> a NS value equal larger 2.                              &#226;&#8364;&#732;http://www.uniprot.org/taxonomy&#226;&#8364;&#8482; using query <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                                                        Page 9 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span>: &#226;&#8364;&#732;bacteria AND complete:yes&#226;&#8364;&#8482;. A total 770                                     Society America: Bad bugs need drugs: update the <br /> bacterial strains common listings. We                                      development pipeline Antimicrobial Availability Task <br />                                                                                         Force Infectious Diseases Society America. Clin Infect Dis <br /> downloaded sequence data &#226;&#8364;&#732;ftp://ftp.expasy.org/                                    2006, 42(5):657&#226;&#8364;&#8220;668. <br /> databases/complete_proteomes/fasta/bacteria/&#226;&#8364;&#8482;.                                    7.    Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL: Drugs bad bugs: <br />                                                                                         confronting challenges antibacterial discovery. Nat Rev Drug <br />                                                                                         Discov 2007, 6:29&#226;&#8364;&#8220;40. <br /> Known drug targets virulence factors                                          8.    Tally FP, DeBruin MF: Development daptomycin gram-positive <br /> Each proteome compared means                                      infections. J Antimicrob Chemother 2000, 46(4):523&#226;&#8364;&#8220;526. <br /> the BLASTPGP program [23], default parameters,                               9.    van Hal SJ, Paterson DL: New Gram-positive antibiotics: better than <br />                                                                                         vancomycin?. Curr Opin Infect Dis 2011, 24(6):515&#226;&#8364;&#8220;520. <br /> against known drug targets available DrugBank [14] and <br />                                                                                   10.   Bereket W, Hemalatha K, Getenet B, Wondwossen T, Solomon A, <br /> virulence factors available VFDB [15]. Proteins showing                              Zeynudin A, Kannan S: Update bacterial nosocomial infections. Eur <br /> a match BLASTPGP E-value &lt;= 1e-2 display link                                  Rev Med Pharmacol Sci 2012, 16(8):1039&#226;&#8364;&#8220;1044. <br /> to related hits output table.                                          11.   Akova M, Daikos GL, Tzouvelekis L, Carmeli Y: Interventional strategies <br />                                                                                         current clinical experience carbapenemase-producing <br />                                                                                         Gram-negative bacteria. Clin Microbiol Infect 2012, 18(5):439&#226;&#8364;&#8220;448. <br /> Three-dimensional homology models                                                 12.   Panjkovich A, Daura X: Assessing structural conservation of <br /> An automated homology-modeling pipeline imple-                                      protein pockets study functional allosteric sites: implications <br />                                                                                         drug discovery. BMC Struct Biol 2010, 10:9. <br /> mented based program MODELLER v9.5 [25].                                   13.   Panjkovich A, Daura X: Exploiting protein flexibility predict the <br /> Briefly, given protein sequence scans                                  location allosteric sites. BMC Bioinformatics 2012, 13:273. <br /> a database structural templates. It proceeds                           14.   Wishart DS, Knox C, Guo AC, Cheng D, Shrivastava S, Tzur D, Gautam B, <br />                                                                                         Hassanali M: DrugBank: knowledgebase drugs, drug actions <br /> generate homology models using best possible set                                 drug targets. Nucleic Acids Res 2008, 36(Database issue):D901&#226;&#8364;&#8220;D906. <br /> diverse templates display 30% sequence iden-                        15.   Yang J, Chen L, Sun L, Yu J, Jin Q: VFDB 2008 release: enhanced <br /> tity. Finally, best resulting models selected using                           web-based resource comparative pathogenomics. Nucleic Acids <br /> combination DOPE GA341 scores [25].                                              Res 2008, 36(Database issue):D539&#226;&#8364;&#8220;D542. <br />                                                                                   16.   Chanumolu SK, Rout C, Chauhan RS: UniDrug-target: computational <br /> Competing interests                                                                     tool identify unique drug targets pathogenic bacteria. PLoS One <br /> The authors declare competing interests.                              2012, 7(3):e32833. <br />                                                                                   17.   Brittnacher MJ, Fong C, Hayden HS, Jacobs MA, Radey M, Rohmer L: <br /> Authors&#226;&#8364;&#8482; contributions                                                                  PGAT: multistrain analysis resource microbial genomes. <br /> XD, IG AP conceived project. AP performed computational work.               Bioinformatics 2011, 27(17):2429&#226;&#8364;&#8220;2430. <br /> AP XD wrote manuscript. All authors read approved final           18.   Shanmugham B, Pan A: Identification characterization of <br /> manuscript.                                                                             potential therapeutic candidates emerging human pathogen <br />                                                                                         Mycobacterium abscessus: novel hierarchical silico approach. <br /> Acknowledgements                                                                        PLoS One 2013, 8(3):e59126. <br /> This work supported funding Seventh Research                19.   White TA, Kell DB: Comparative genomic assessment novel <br /> Framework Programme European Union (ref. HEALTH-F3-2009-223101,                  broad-spectrum targets antibacterial drugs. Comp Funct Genomics <br /> AntiPathoGN) Spanish Ministry Science Innovation                        2004, 5(4):304&#226;&#8364;&#8220;327. <br /> (BFU2010-17199). AP acknowledges FPU Scholarship received            20.   Zhang CT, Zhang R: Gene essentiality analysis based DEG, a <br /> Spanish Government. The authors acknowledge Manuel Alonso Tarajano                  database essential genes. Methods Mol Biol 2008, 416:391&#226;&#8364;&#8220;400. <br /> the initial compilation Gram-negative reference set.                       21.   Sakharkar KR, Sakharkar MK, Chow VTK: A novel genomics approach for <br />                                                                                         identification drug targets pathogens, special <br /> Author details                                                                          reference Pseudomonas aeruginosa. In Silico Biol 2004, 4(3):355&#226;&#8364;&#8220;360. <br /> 1 Institute Biotechnology Biomedicine (IBB), Universitat Aut&#195;&#178;noma       22.   Pedruzzi I, Rivoire C, Auchincloss AH, Coudert E, Keller G, Castro E, <br /> Barcelona (UAB), 08193 Bellaterra, Spain. 2 Departament Gen&#195;&#168;tica                Baratin D, Cuche BA, Bougueleret L, Poux S, Redaschi N, Xenarios I, <br /> Microbiologia, UAB, 08193 Bellaterra, Spain. 3 Catalan Institution Research         Bridge A: UniProt-Consortium: HAMAP 2013, new developments <br /> and Advanced Studies (ICREA), 08010 Barcelona, Spain.                                   protein family classification annotation system. Nucleic <br />                                                                                         Acids Res 2013, 41(Database issue):D584&#226;&#8364;&#8220;D589. <br /> Received: 30 September 2013 Accepted: 11 January 2014                             23.   Altschul SF, Madden TL, Schaeffer AA, Zhang J, Zhang Z, Miller W, Lipman <br /> Published: 17 January 2014                                                              DJ: Gapped BLAST PSI-BLAST: new generation protein <br />                                                                                         database search programs. Nucleic Acids Res 1997, 25(17):3389&#226;&#8364;&#8220;3402. <br /> References                                                                        24.   Laskowski RA, Rullmannn JA, MacArthur MW, Kaptein R, Thornton JM: <br /> 1. Arias CA, Murray BE: Antibiotic-resistant bugs 21st century &#226;&#8364;&#8220;               AQUA PROCHECK-NMR: programs checking quality of <br />     clinical super-challenge. N Engl J Med 2009, 360(5):439&#226;&#8364;&#8220;443.                        protein structures solved NMR. J Biomol NMR 1996, 8(4):477&#226;&#8364;&#8220;486. <br /> 2. Baquero F, Alvarez-Ortega C, Martinez JL: Ecology evolution             25.   Eswar N, Eramian D, Webb B, Shen MY, Sali A: Protein structure <br />     antibiotic resistance. Environ Microbiol Rep 2009, 1(6):469&#226;&#8364;&#8220;476.                    modeling MODELLER. Methods Mol Biol 2008, 426:145&#226;&#8364;&#8220;159. <br /> 3. Boucher HW, Talbot GH, Bradley JS, Edwards JE, Gilbert D, Rice LB, Scheld      26.   Mistry J, Bateman A, Finn RD: Predicting active site residue <br />     M, Spellberg B, Bartlett J: Bad bugs, drugs: ESKAPE! An update                annotations Pfam database. BMC Bioinformatics 2007, 8:298. <br />     Infectious Diseases Society America. Clin Infect Dis 2009,        27.   Huang B, Schroeder M: LIGSITEcsc: predicting ligand binding sites <br />     48:1&#226;&#8364;&#8220;12.                                                                            using Connolly surface degree conservation. BMC Struct <br /> 4. European Academies Science Advisory Council: Tackling antibacterial                  Biol 2006, 6:19. <br />     resistance Europe. Tech. rep., EASAC 2007.                                 28.   UniProt Consortium: The Universal Protein Resource (UniProt) 2009. <br /> 5. Souli M, Galani I, Giamarellou H: Emergence extensively                           Nucleic Acids Res 2009, 37(Database issue):D169&#226;&#8364;&#8220;D174. <br />     drug-resistant pandrug-resistant Gram-negative bacilli                 29.   Fournier PE, Vallenet D, Barbe V, Audic S, Ogata H, Poirel L, Richet H, <br />     Europe. Euro Surveill 2008, 13(47):19045.                                           Robert C, Mangenot S, Abergel C, Nordmann P, Weissenbach J, Raoult D, <br /> 6. Talbot GH, Bradley J, Edwards JE, Gilbert D, Scheld M, Bartlett JG:                  Claverie JM: Comparative genomics multidrug resistance in <br />     Antimicrobial Availability Task Force Infectious Diseases                    Acinetobacter baumannii. PLoS Genet 2006, 2:e7. <br />  Panjkovich et al. BMC Genomics 2014, 15:36                                                                                                    Page 10 10 <br /> http://www.biomedcentral.com/1471-2164/15/36 <br />  <br />  <br />  <br />  <br /> 30. Lee DS, Burd H, Liu J, Almaas E, Wiest O, Barab&#195;&#161;si AL, Oltvai ZN, Kapatral V: <br />     Comparative genome-scale metabolic reconstruction flux <br />     balance analysis multiple Staphylococcus aureus genomes <br />     identify novel anti-microbial drug targets. J <br /> </body></html>