<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Kinome-Wide RNAi Screen Drosophila Glia Reveals <br /> That RIO Kinases Mediate Cell Proliferation and <br /> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>Survival</span> TORC2-Akt Signaling Glioblastoma <br /> Renee D. Read1&#194;&#164;*, Tim R. Fenton2, German G. Gomez2, Jill Wykosky2, Scott R. Vandenberg3, Ivan Babic2, <br /> Akio Iwanami4, Huijun Yang2, Webster K. Cavenee2,5, Paul S. Mischel2,3, Frank B. Furnari2,3, <br /> John B. Thomas1 <br /> 1 Molecular Neurobiology Laboratory, Salk Institute Biological Studies, La Jolla, California, United States America, 2 Ludwig Institute Cancer Research, University <br /> of California San Diego, La Jolla, California, United States America, 3 Department Pathology, School Medicine, University California San Diego, La Jolla, California, <br /> United States America, 4 Department Orthopaedic Surgery, Keio University School Medicine, Tokyo, Japan, 5 Department Medicine, School Medicine, <br /> University California San Diego, La Jolla, California, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Glioblastoma, common primary malignant brain tumor, incurable current therapies. Genetic molecular <br />      analyses demonstrate glioblastomas frequently display mutations activate receptor tyrosine kinase (RTK) Pi-3 <br />      kinase (PI3K) signaling pathways. In Drosophila melanogaster, activation RTK PI3K pathways glial progenitor cells <br />      creates malignant neoplastic glial tumors display features human glioblastoma. In human Drosophila, <br />      activation RTK PI3K pathways stimulates Akt signaling as-yet-unknown changes drive <br />      oncogenesis. We used Drosophila glioblastoma model perform kinome-wide genetic screen new genes required <br />      RTK- PI3K-dependent neoplastic transformation. Human orthologs novel kinases uncovered screens <br />      functionally assessed mammalian glioblastoma models human tumors. Our results revealed atypical <br />      kinases RIOK1 RIOK2 overexpressed glioblastoma cells Akt-dependent manner. Moreover, that <br />      overexpressed RIOK2 formed complex RIOK1, mTor, mTor-complex-2 components, overexpressed <br />      RIOK2 upregulated Akt signaling promoted tumorigenesis murine astrocytes. Conversely, reduced expression of <br />      RIOK1 RIOK2 disrupted Akt signaling caused cell cycle exit, apoptosis, chemosensitivity glioblastoma cells by <br />      inducing p53 activity RpL11-dependent ribosomal stress checkpoint. These results imply that, glioblastoma <br />      cells, constitutive Akt signaling drives RIO kinase overexpression, creates feedforward loop promotes and <br />      maintains oncogenic Akt activity stimulation mTor signaling. Further study RIO kinases other <br />      kinases identified Drosophila screen reveal new insights defects underlying glioblastoma related cancers <br />      reveal new therapeutic opportunities cancers. <br />  <br />    Citation: Read RD, Fenton TR, Gomez GG, Wykosky J, Vandenberg SR, et al. (2013) A Kinome-Wide RNAi Screen Drosophila Glia Reveals That RIO Kinases <br />    Mediate Cell Proliferation Survival TORC2-Akt Signaling Glioblastoma. PLoS Genet 9(2): e1003253. doi:10.1371/journal.pgen.1003253 <br />    Editor: Anders I. Persson, University California San Francisco, United States America <br />    Received June 6, 2012; Accepted December 4, 2012; Published February 14, 2013 <br />    Copyright: &#195;&#376; 2013 Read et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />    unrestricted use, distribution, reproduction medium, provided original author source credited. <br />    Funding: This work supported American Brain Tumor Association postdoctoral fellowship awarded RDR, K99 Award National Institute of <br />    Neurological Disorders Stroke RDR, Salk-Sanofi Discovery Grant Sanofi-Aventis awarded RDR JBT. WKC fellow National <br />    Foundation Cancer Research. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />    Competing Interests: The authors declared competing interests exist. <br />    * E-mail: renee.read@emory.edu <br />    &#194;&#164; Current address: Department Pharmacology, Emory University School Medicine, Atlanta, Georgia, United States America <br />  <br />  <br />  <br />  <br /> Introduction                                                                               forms EGFR GBMs potently drive tumor <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>, <br />                                                                                            migration, proliferation [2,3]. The frequent <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> in <br />    Glioblastoma (GBM), common primary malignant                                   PI3K pathway GBM loss PTEN lipid phosphatase, <br /> brain tumor, infiltrates brain, grows rapidly, refractory                       results unopposed signaling PI3K robust <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> current therapies. Signature genetic lesions GBM include                             stimulation Akt, especially context EGFR activation <br /> amplification, mutation, and/or overexpression receptor tyro-                           [1]. In mouse models, co-activation pathways glia, glial <br /> sine kinases (RTKs), EGFR PDGFRa,                                   progenitor cells, and/or neuro-glial stem cells induces GBM <br /> activating mutations components PI-3 kinase (PI3K)                               [4,5,6,7]. However, range signaling events acting <br /> pathway (reviewed [1]). More 40% GBMs EGFR                                 downstream combination EGFR PI3K drive <br /> gene amplification, amplification events ac-                           oncogenesis remain determined. While normal <br /> companied mutations EGFR [1]. The prevalent                                     effectors RTK PI3K signaling, Ras, Akt, mTor, <br /> mutant form EGFR DEGFR (EGFRvIII, de2-7EGFR,                                         used EGFR PI3K GBM required for <br /> EGFR*), intragenic truncation mutant displays constitutive                         gliomagenesis [1], constitutive activation RTK PI3K <br /> kinase activity [2]. DEGFR constitutively active mutant                          pathways evoke changes distinct induced by <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                   1                            February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                               RIOKs Glioblastoma <br />  <br />  <br />  <br />   Author Summary                                                             characterized presence RIO kinase domain, kinase <br />                                                                              fold structurally homologous eukaryotic serine-threonine <br />   Glioblastomas, common primary brain tumor,                        protein kinase domains, lacks classic activation and <br />   harbor mutations receptor tyrosine kinases (RTKs),                 substrate binding loops (reviewed [15]). While kinases <br />   EGFR, components Pi-3 kinase (PI3K)                          undergo autophosphorylation phosphorylate nonspecific <br />   signaling pathway. However, genes act down-                       substrates vitro, actual vivo substrates RIO kinases are <br />   stream RTK PI3K signaling drive glioblastoma                     unknown [15]. In yeast human cells, RIOK1 and <br />   remain unclear. To investigate genetic molecular                   RIOK2, functionally redundant, required for <br />   basis disease, created glioblastoma model                  processing 18S rRNA cytoplasmic maturation the <br />   fruit fly Drosophila melanogaster. To identify new                     40S ribosomal subunit, kinase integral <br />   genes involved glioblastoma development, per-                        component ribosome [16,17,18,19]. Recent studies demon- <br />   formed screen genes required tumor cell <br />                                                                              strate that, yeast, Rio2p transiently associates im- <br />   proliferation using Drosophila glioblastoma model and <br />   functionally assessed activity human versions                  mature ribosomes block translation initiation, how <br />   novel genes identified screen. Our results                      RIOK2 regulated context unclear [20]. To date, several <br />   revealed RIO kinases overexpressed                      studies provided suggestive evidence RIO kinases <br />   human glioblastomas normal human glial                       involved RTK PI3K signaling: RIOK2 becomes <br />   neuronal cells. We overexpression RIO                    rapidly phosphorylated response EGFR stimulation; Rio2p <br />   kinases promotes maintains signals drive tumor                    binds Tor2p, ortholog mTor kinase, RIOK1 is <br />   cell proliferation survival RTK- PI3K-dependent                 required proliferation survival Ras-dependent <br />   human glioblastoma, reduction RIO kinase expres-                    cancer cells [21,22,23]. However, date, specific function has <br />   sion decreased proliferation prompted cell death                   ascribed RIOK1 RIOK2 context EGFR or <br />   chemosensitivity glioblastoma cells. Therefore, disrup-                 PI3K signaling. <br />   tion RIO kinases provide new therapeutic                           In manuscript demonstrate RIOK1 RIOK2 <br />   opportunities target glioblastoma RTK-                     overexpressed GBM tumor cells relative normal <br />   PI3K-dependent cancers.                                                    brain cells; RIOK1 RIOK2 overexpression occurs in <br />                                                                              response constitutive Akt signaling; RIOK2 forms a <br /> normal developmental signaling. Notably, treatments phar-               complex RIOK1, mTor, signaling components to <br /> macologic inhibitors EGFR mTor cytostatic best             drive activation Akt signaling tumorigenesis; that, in <br /> subset patients, indicating other, unidentified factors           GBM cells, RIOK1 RIOK2 loss causes reduction Akt <br /> compensatory signals affect survival growth tumor cells           signaling provokes p53-dependent apoptosis, cell cycle exit, <br /> [8].                                                                         chemosensitivity RpL11-dependent ribosomal <br />    To uncover new factors required EGFR- PI3K-                       stress checkpoint. Our data demonstrate RIO kinases play <br /> mediated gliomagenesis, developed GBM model Drosophila               key role Akt-mediated transformation GBM cells. <br /> melanogaster [9]. Drosophila offers advantages modeling <br /> cancers like GBM. Flies orthologs 75% human disease              Results <br /> genes [10], including nearly known gliomagenic genes; <br /> signaling pathways highly conserved; versatile genetic tools             A kinome-wide screen modifiers glial neoplasia <br /> are available cell-type specific gene manipulation [11,12];             To discover new genes involved glial pathogenesis, we <br /> Drosophila neural cell types homologous mammalian               performed genetic screen using Drosophila GBM model. Co- <br /> counterparts [13,14]. While Drosophila model address            overexpression constitutively active forms Drosophila EGFR <br /> aspects human GBM, model recapitulates important                      (dEGFRl) PI3K catalytic subunit p110a (dp110CAAX) <br /> pathologic features. Specifically, constitutive activation EGFR-          stimulates malignant transformation post-embryonic larval glia, <br /> Ras PI3K signaling Drosophila glial progenitor cells gives rise       inducing lethal glial neoplasia (Figure 1A&#226;&#8364;&#8220;1C) (characterized in <br /> to proliferative, invasive neoplastic glia create transplantable        [9]). Using larval Drosophila GBM model, we <br /> malignant tumors [9]. These tumors induced                       performed RNAi-based modifier screen genes suppress <br /> activation synergistic genetic network composed down-                (inhibit) enhance (worsen) neoplastic phenotypes caused by <br /> stream pathways commonly mutated and/or activated human                   constitutive EGFR PI3K signaling. In scheme, an <br /> GBMs, Akt mTor signaling [9]. However, activating                enhancer-suppressor screen, modifier kinases block/inhibit fly <br /> these known downstream pathways combination               glial neoplasia expression reduced referred as <br /> sufficient induce glial neoplasia Drosophila, indicating          &#226;&#8364;&#732;suppressors,&#226;&#8364;&#8482; modifier kinases exacerbate neoplasia when <br /> additional, unidentified, genetic pathways involved            expression reduced referred &#226;&#8364;&#732;enhancers.&#226;&#8364;&#8482; This in <br /> transformation. Thus, undertook genetic screens using                 keeping standard Drosophila nomenclature, genes are <br /> Drosophila GBM model discover new genes underlying EGFR                   classified loss-of-function phenotypes. As side-note, in <br /> and PI3K mediated neoplastic transformation, tested              context, term suppressor does refer mammalian <br /> human orthologs genes identified Drosophila represent new          tumor suppressors. <br /> human genes involved GBM.                                                    To enrich new pathway components, screened nearly of <br />    Our analyses Drosophila human systems uncovered               kinases encoded Drosophila genome (Table S1). Our choice <br /> that RIOK1 RIOK2 kinases drive survival                      screen kinases based considerations: (1) kinases <br /> proliferation GBM cells. RIOK1 RIOK2 members                   regulate broad array biological cellular processes, including <br /> the RIO (right open reading frame) family atypical protein                underlying oncogenesis; (2) kinases highly conserved <br /> kinases, named yeast (S. cerevisiae) Rio1p Rio2p,                    Drosophila humans Drosophila kinase has <br /> respectively [15]. The RIOK1 RIOK2 proteins highly                   clear human ortholog; (3) group, characterized, <br /> conserved, present phylogenetic kingdoms,                facilitating functional analysis; (4) drug discovery efforts are <br /> yeast mammals eukaryotes. Kinases family                focused development specific kinase inhibitors. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     2                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                     RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 1. A kinome-wide screen modifiers EGFR- PI3K-dependent glial neoplasia. (A) Optical projections brain-nerve <br /> cord complexes late 3rd instar larvae approximately 130 hrs old, displayed scale. Dorsal view; anterior up. GFP labels glia (green). <br /> Each brain composed 2 hemispheres nerve cord. Knockdown strong suppressor loci decreased brain size, relative wild-type <br /> controls, CG11859dsRNA;dEGFRl;dp110CAAX CG11660dsRNA;dEGFRl;dp110CAAX animals. Glial-specific overexpression DEGFR drives increased <br /> glial cell numbers, brain enlargement, lethality, knockdown strong suppressor loci grossly decreased brain size relative controls, in <br /> CG11859dsRNA;DEGFR CG11660dsRNA;DEGFR animals. (B&#226;&#8364;&#8220;H) 3 mm optical projections brain hemispheres late 3rd instar larvae approximately <br /> 130 hrs old, displayed scale. Frontal sections, midway brains. Anterior up; midline left. Glial cell nuclei labeled Repo (red); <br /> glial cell bodies labeled GFP (green). Brains counter-stained anti-HRP (blue), reveals neuropil high intensity neuronal cell <br /> bodies low intensity. Dark areas contain unstained neuronal precursor cells. dEGFRl;dp110CAAX (C) DEGFR (F) brains showed dramatic increase <br /> in glial cell number (red nuclei, green) relative wild-type (B). Upon suppression, CG11859dsRNA;dEGFRl;dp110CAAX (D), CG11859dsRNA;DEGFR (G), <br /> and CG11660dsRNA;DEGFR (H), excess glia (red nuclei), remaining glial cells abnormal development (green). Reduction both <br /> glial (green) neuronal cell types (low intensity blue) account reduced brain size CG11660 CG11859 knockdown context of <br /> dEGFRl;dp110CAAX DEGFR, suggests remaining abnormal glia properly support neuronal cell survival. Modifier constructs were <br /> also tested effects wild-type glia, CG11859dsRNA animals (E). (I) Breakdown screen results kinases tested. Unconfirmed modifiers are <br /> defined RNAi construct each. (J) Functional classifications confirmed modifiers. Individual kinases noted Table S6. <br /> doi:10.1371/journal.pgen.1003253.g001 <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        3                         February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                             RIOKs Glioblastoma <br />  <br />  <br />    We tested 553 conditional RNAi constructs targeting 223           [30], indicating modifiers generically <br /> 243 kinases fly genome (Tables S1, S2) [24,25]. RNAi                 required cell survival. The largest group modifiers roles <br /> constructs expressed specifically glial lineage         cell proliferation (Figure 1J, Table S6), these <br /> were tested phenotypic effects proliferation, migration,       yielded reduced cell lineages knockdown neuroblasts <br /> morphology, and/or viability dEGFRl;dp110CAAX neoplastic                 (Table S4) [31], consistent known requirements RTK and <br /> glia. The specificity modifier loci confirmed testing             PI3K signaling neural progenitor cells [9,32]. Several the <br /> multiple RNAi constructs, dominant negative constructs, and/or              modifiers involved cell proliferation, warts (Table S2), <br /> available mutant alleles determine produced analogous            components hippo pathway, pathway a <br /> phenotypes dEGFRl;dp110CAAX model (Tables S2, S3).                   documented role glial cell proliferation [33]. Broad comparisons <br /> Control assays performed effects modifier RNAi              orthologs species yeast (S. cerevisiae), revealed <br /> constructs expressed specifically cell types, including       modifiers kinases implicated protein translation, as <br /> normal glia, neuroblasts (Drosophila neural stem cells), neurons        dRIOK1 dRIOK2 [16,20], cell shape change and <br /> in order distinguish RNAi constructs caused non-              migration. Finally, comparison human kinases shows the <br /> specific toxicity RNAi constructs caused specific changes         majority modifier kinases orthologs previously <br /> in neoplastic glia (Table S4, Text S1). Constructs caused early        implicated GBM (Figure 1J, Figure S2, Table S7), leaving a <br /> organismal lethality cellular contexts tested excluded          set 16 novel modifiers, including dRIOK1 dRIOK2. <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> analysis (Tables S2, S4). <br />    To test modifiers act oncogenic signaling down-               Overexpression RIOK kinases human GBM correlates <br /> stream specific EGFR mutations human GBM, we <br />                                                                             Akt activity <br /> created flies overexpress human DEGFR. Glial-specific <br />                                                                                Novel modifier kinases identified Drosophila screens may <br /> expression DEGFR caused lethal glial neoplasia phenotypes, <br />                                                                             represent human kinases directly involved GBM pathogenesis. <br /> alone combination dp110CAAX, similar to <br />                                                                             Kinases block fly glial neoplasia expression is <br /> dEGFRl, phenotypes required EGFR kinase activity <br />                                                                             reduced human orthologs be <br /> and core PI3K effectors, dAkt (Figure 1A&#226;&#8364;&#8220;1F, Figure S1, <br />                                                                             promising new targets therapeutic inhibition. There 27 <br /> Table S5). We tested modifier RNAi constructs ability to <br />                                                                             human orthologs 16 novel Drosophila modifier kinases <br /> alter glial-specific DEGFR DEGFR;dp110CAAX phenotypes <br /> (Table S5); results mirrored genetic interactions            (Table S8). To determine human kinases are <br /> dEGFRl;dp110CAAX (Table S2, S5), indicating modifiers                  expressed mutated GBM, analyzed tumor genomic <br /> identified screen common neoplastic phenotypes                databases, proteomic atlases, GBM cell lines, provided <br /> conferred Drosophila human EGFR.                                suggestive evidence 12 modifier orthologs subject to <br />    We identified total 45 modifier genes (Figure 1I, Tables S2         genetic alteration and/or elevated gene protein expression in <br /> and S3). Suppressor RNAi constructs targeting 39 genes reduced              GBMs (see Text S1, Tables S9 S10, Figures S3 S4). <br /> neoplasia induced smaller brain size lower glial cell               Among these, RIOK1 RIOK2 showed increased protein <br /> numbers relative dEGFRl;dp110CAAX controls, enhanc-              expression consistent involvement GBM. Given loss of <br /> er RNAi constructs targeting 6 genes worsened tumorigenesis             dRIOKs strongly specifically blocks growth survival of <br /> neoplasia induced increased glial cell numbers, and/or                  EGFR PI3K mutant glia, recent publications suggest <br /> aberrant glial morphologies (Figure 1A&#226;&#8364;&#8220;1D, 1I, Tables S2                RIO kinases contribute EGFR and/or mTor <br /> S3). Modifiers suppressed glial neoplasia include genes                signaling [21,22], functional roles RIOK1 RIOK2 in <br /> identified previous studies, including dAkt [9] (Tables S2, S5).         GBM particular interest. <br /> A small subset suppressor constructs caused strong phenotypes               A range GBM cells cell lines examined determine <br /> in context constitutive EGFR-PI3K signaling. Constructs              RIOK1 RIOK2 expression correlated tumor cell <br /> targeting modifier loci, Raf, Src42A, Taf1, rescued               genotype <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>phenotype</span>. Our analyses showed RIOK1 and <br /> dEGFRl;dp110CAAX animals adult viability, allowing neoplastic            RIOK2 expressed PTEN-null U87MG GBM cells and <br /> glia differentiate function normally despite presence         upregulated U87MG cells engineered express DEGFR <br /> dEGFRl dp110CAAX (Table S2). Constructs targeting                levels detected tumors [34] (Figure 2A). RIOK1 RIOK2 <br /> the strongest modifiers, CG11660 CG11859 (the Drosophila                upregulated GBM tumors EGFR overexpres- <br /> orthologs RIOK1 RIOK2 kinases, respectively), caused             sion/mutation activated Akt (Figure 2B), these <br /> severe reduction brain size glial cell number                   correlations clear specimens. Primary neurosphere <br /> combined dEGFRl;dp110CAAX, DEGFR;dp110CAAX, and/or                     cultures, composed neural stem cell-like human GBM <br /> DEGFR, compared wild-type control animals (Figure 1A&#226;&#8364;&#8220;1H,              cells propagated EGF-supplemented media [35,36,37], can <br /> Figure S1). dRIOK2 knockdown gave stronger effect (Figure 1D,             maintain mutations/gene expression parent tumors <br /> 1G). In contrast, RNAi constructs targeting dRIOK1                      [37]. Neurosphere cultures showed strong RIOK1 RIOK2 <br /> dRIOK2 did produce dramatic growth reduction                     expression (Figure 2C); included neurosphere lines with <br /> targeted normal glia (Figure 1E, Table S4), indicating              DEGFR, neurosphere lines displaying PDGFRa <br /> dRIOK1 dRIOK2 knockdown does simply cause                            overexpression, PTEN loss, and/or mutant forms EGFR <br /> nonspecific cellular toxicity.                                              (Figure 2C). In panel standard GBM cell lines, RIOK1 and <br />    Modifier kinases classified bioinformatic annotations            RIOK2 showed strong expression cell lines known harbor <br /> using Flybase, Gene Ontology, KEGG pathways, <span id='am-2' about='xsd:string' typeof='owl:Thing'>STRING</span>                     PTEN and/or EGFR mutations, RIOK1 RIOK2 <br /> database [26], comparisons Drosophila RNAi                   expression comparatively lower GBM cell line with <br /> screens. These classifications (Figure 1J, Figure S2, Table S6)        intact PTEN (Figure S5) [38,39]. In contrast, RIOK1 RIOK2 <br /> that kinases core functions RTK PI3K pathways               nearly undetectable mixed glial cultures freshly derived <br /> were highly represented, validating model screening                 adult human cortex (Figure 2D). Thus, RIOK1 RIOK2 <br /> methodology [27,28,29]. Notably, modifiers             overexpression appeared correlated RTK mutation/ <br /> emerged Drosophila RNAi screens cell viability (Table S7)          overexpression and/or PTEN loss GBM tumor cells. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    4                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                           RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 2. Expression RIO kinases associated EGFR Akt activity GBM cells. (A) U87MG U87MG-DEGFR cells cultured <br /> with .1% serum 36 hrs enrich DEGFR signaling. DEGFR runs full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> EGFR. (B) Biopsies new (de novo) recurrent GBMs. RIOK1 <br /> (,75 kDa) RIOK2 (,63 kDa) antibodies used serially blot. Serine-473 phosphorylation proxy Akt activation. (C) Primary <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                  5                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                               RIOKs Glioblastoma <br />  <br />  <br />  <br /> neurosphere GBM cultures. GBM39 isolated DEGFR-positive serial xenograft [41]. Others low-passage cultures established fresh <br /> tumors; expression DEGFR, EGFR, PDFGRa derives parent tumors [35]. EGFRhw high molecular weight (.200 kDa) mutant version <br /> detected GBM 281. (D) RIOK expression Akt signaling neurospheres compared fresh culture mixed human glia astrocytes <br /> (established normal adult cortex) grown conditions. (E) GBM301 treated 24 hrs growth factor withdrawl (.1% GFs, .1% <br /> of normal growth factor dosage) 5 mM gefitinib. GBM301 cells DEGFR-positive, EGFR-amplified, PTEN-negative (see C). (F) Extracts from <br /> PTEN2/2; Ink4a/arf2/2 astrocytes transduced vector (left) DEGFR (right), grown .5% serum treated gefitinib 24 hrs <br /> (far right). EGFR inhibition evidenced reduced Tyrosine-1068 phosphorylation. (G) Neurosphere cultures treated 24 hours DMSO or <br /> indicated inhibitors. P110 inhibitors: 50 mM LY294002, 1 mM BEZ-235, 2 mM GDC-0941 [71], 2 mM PI-103 [71]. Akt inhibitors: 1 mM A443654, 10 mM Akt <br /> inhibitor IV [40], 8 mM MK-2206 [72]. Inhibition PI3K-Akt signaling evidenced reduced phosphorylation PRAS40, direct Akt substrate, <br /> and reduced Akt and/or PRAS40 protein levels. With A443654, increased Akt-Ser473 phosphorylation occurs despite Akt inhibition [40]; decreased <br /> Akt-Ser473 phosphorylation occurs Akt inhibitor IV MK-2206 [72]. (H) U87MG-DEGFR cells treated pan-Akt siRNAs compared cells <br /> treated nontargeting control siRNAs, harvested 72 hours post-transfection. <br /> doi:10.1371/journal.pgen.1003253.g002 <br />  <br />    To determine elevated RIOK1 and/or RIOK2 expression                 displayed strong, heterogeneous, cytoplasmic <br /> GBM cells depends EGFR and/or PI3K signaling, neuro-                      RIOK2 immunoreactivity, ranging giant cell small <br /> sphere lines U87MG-DEGFR cells treated relevant                cell populations (Figure 3B&#226;&#8364;&#8220;3E, Figures S10 S11), the <br /> inhibitors (Figure 2E 2G, Figure S6). RIOK1 RIOK2                    strongest expression mitotic cells densely cellular pseudo- <br /> levels decreased growth factor withdrawal gefitinib           pallisades (Figure 3C 3D, Figure S10). Heterogeneity in <br /> treatment primary neurosphere cultures (Figure 2E), indicating            RIOK2 expression possibly reflect heterogeneity RTK <br /> that up-regulation EGFR-dependent. Consistent              expression tumors [42,43], reflect upregulation RIOK2 <br /> this, Pten2/2; Ink4a/arf2/2 mouse astrocytes transformed                  actively cycling cells given increased immunoreactivity <br /> DEGFR showed increased RIOK1 RIOK2 levels,                         observed mitotic cells. Cytoplasmic localization RIOK2 in <br /> were reduced gefitinib treatment (Figure 2F). RIOK1                   GBM consistent observations RIO kinase localization in <br /> RIOK2 protein levels decreased neurosphere cells                 yeast human cells [16,17,18]. In contrast, RIOK2 did show <br /> U87MG-DEGFR cells treated inhibitors p110 PI3K                   appreciable immunoreactivity neural cells matched normal <br /> catalytic subunit, BEZ-235, inhibitors Akt,           control brain (n = 14) (Figure 3F, 3H), did show <br /> A443654 (Figure 2G) [40]. siRNA-mediated Akt knockdown                    immunoreactivity tumor stroma (Figure 3B), demonstrating that <br /> restoration PTEN function U87MG-DEGFR cells                       RIOK2 upregulation tumor-specific. Akt signaling tumors was <br /> reduced RIOK protein levels (Figure 2H, Figure S7). Treatments               assessed staining Akt phosphorylated Serine-473 (Akt- <br /> with p110 Akt inhibitors demonstrated p110 Akt             S473-P, example shown Figure 3I), EGFR status tumors <br /> signaling required RIOK1 RIOK2 expression                      primarily assessed staining EGFR phosphorylated on <br /> PDGFRa-overexpressing neurospheres (Figure 2G). Taken to-                    Tyrosine-1068 (EGFR-Y1068-P, example shown Figure 3J), <br /> gether, data indicate RIOK1 RIOK2 overexpres-                 indicates EGFR activation. Statistical analysis demonstrated <br /> sion GBM cells driven Akt activity downstream RTK                RIOK2 expression significantly correlated EGFR <br /> mutation/overexpression and/or PTEN loss.                                    status tumor specimens (Figure 3L), EGFR- <br />    However, role factors act downstream Akt          negative tumors showed RIOK2 immunoreactivity (Figure 3G, <br /> clear (mTor inhibitors did reduce RIOK levels,                3L), EGFR-negative tumors did (Figure 3K). The <br /> Figure S6), suggesting RIOK1 RIOK2 levels                     correlation RIOK2 expression EGFR activity likely <br /> directly regulated Akt. Given mRNA expression levels              secondary Akt-mediated regulation RIOK2: RIOK2-express- <br /> RIOK1 RIOK2 did significant upregulation                     ing specimens positive EGFR-Y1068-P showed staining for <br /> tumor samples PTEN and/or EGFR alterations (Table S9),                  Akt-S473-P (n = 20). Indeed, specimens showed RIOK2 <br /> and given RIOK1 and/or RIOK2 levels decline short-                immunoreactivity, EGFR-positive EGFR-negative, <br /> term treatments Akt inhibitors (Figure S7), hypothesized             showed staining Akt-S473-P (Figure 3L). <br /> that Akt signaling regulate RIOK2 and/or RIOK1 levels by <br /> modulating protein stability post-translationally. Consistent           RIOK2 overexpression astrocytes induces invasive glial <br /> this, addition proteosome inhibitor, MG132, prevented               tumors TORC2-Akt activation <br /> reduction RIOK1 and/or RIOK2 protein levels observed                    To determine elevated levels RIOK2 drives oncogenic <br /> A443654 treatment PTEN add-back (Figure S7). RIOK2                    changes mammalian cells, tested effects RIOK2 <br /> several mapped serine phosphorylation sites (www.phosphosite.                overexpression astrocytes. Pten2/2; Ink4a/arf2/2 murine astro- <br /> org), including putative Akt target sites (Figure S8). However,              cytes, immortalized tumor suppressor mutations <br /> mutation single site did abolish detection RIOK2           common GBM, express little endogenous RIOK2 not <br /> a phospho-Akt-substrate antibody (Figure S9). Thus, Akt-mediated             gliomagenic intracranial grafts assays [44]. In experiments, <br /> regulation RIO kinase levels does hinge phosphorylation            mice intracranially grafted Pten2/2; Ink4a/arf2/2 astrocytes <br /> at single residue RIOK2, likely involves complex             overexpressing RIOK2 showed symptoms hydrocephaly and <br /> mechanism requires investigation.                                  neurological deficits 3 weeks following implantation, unlike mice <br />    To confirm RIOKs expressed GBM,                        grafted Pten2/2; Ink4a/arf2/2 control astrocytes. Histological <br /> performed immunohistochemistry (IHC) RIOK2 group                 analysis showed control Pten2/2; Ink4a/arf2/2 astrocytes <br /> typed tumor specimens (RIOK1 antibodies unsuitable                  yielded intracranial tumors 12 total animals tested, whereas <br /> IHC). Xenograft specimens GBM39, DEGFR-positive                  RIOK2overexpresion; Pten2/2; Ink4a/arf2/2 astrocytes formed invasive <br /> [41], showed RIOK2 expression tumor cells, cells                     high-grade glial tumors composed invasive spindle-shaped cells in <br /> displaying diffuse cytoplasmic sub-surface RIOK2 localization            7 10 total animals tested (p,.001 chi-squared test) <br /> (Figure 3A). In contrast, murine stromal cells little              (Figure 4A, 4B). <br /> RIOK2 immunoreactivity, antibody detect mouse                  In GBM, DEGFR drives strong Akt activation context of <br /> RIOK2. DEGFR-positive EGFR-overexpressing specimens                      PTEN loss, drive gliomagenic transformation Pten2/2; <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     6                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                              RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 3. RIOK2 overexpression GBM tumors associated Akt signaling. (A&#226;&#8364;&#8220;E) Immunohistochemistry RIOK2 (reddish brown) <br /> showing cytoplasmic submembraneous enrichment RIOK2 tumor cells. Hematoxilin counterstain, (A) GBM39 tissue, subcutanteous <br /> xenograft, showing RIOK2 staining tumor cells (arrows), formed lobules delineated RIOK2-negative host stromal cells. (B) DEGFR-positive <br /> human GBM RIOK2-positive giant cell component (inset shows conspicuous giant cell), RIOK2-negative tumor stroma composed of <br /> abnormal blood vessels (&#226;&#8364;&#732;&#226;&#8364;&#732;BV&#226;&#8364;&#8482;&#226;&#8364;&#8482;). (C) DEGFR-positive human GBM, abnormal mitotic cells high RIOK2 staining denoted asterisks shown in <br /> inset close-up. (D) DEGFR-positive human GBM, lower magnification highlight enriched RIOK2 pseudopallisades (&#226;&#8364;&#732;&#226;&#8364;&#732;PSS&#226;&#8364;&#8482;&#226;&#8364;&#8482;), inset shows enriched <br /> RIOK2 staining present dense cellular regions pseudopallisades. (E) RIOK2 expression EGFR-overexpressing human GBM (F) matched <br /> normal control tissue surgical specimen, arrows denote normal astrocytes (recognized open nuclei). (G) RIOK2 expression in <br /> an EGFR-negative/Akt-S473-P-positive GBM shown alongside (H) example normal control brain tissue. Arrows denote normal neuronal <br /> cells (recognized basophilic cell bodies) low/undetectable RIOK2 expression. (I) (J) examples Akt-S473-P EGFR-Y1068-P <br /> immunoreactivity RIOK2-positive GBM tumor specimens. (K) RIOK2- negative GBM negative abnormal blood vessel (&#226;&#8364;&#732;&#226;&#8364;&#732;BV&#226;&#8364;&#8482;&#226;&#8364;&#8482;). (L) Statistical <br /> analysis RIOK2-positive negative tumor specimens showing significant correlation RIOK2 expression phosphorylation EGFR <br /> at Tyrosine-1068 phosphorylation Akt Serine-473. More stains tumors shown Figures S10 S11. <br /> doi:10.1371/journal.pgen.1003253.g003 <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    7                        February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                   RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 4. Overexpression RIOK2 murine astrocytes promotes tumorigenesis TORC2-Akt signaling. (A) H&amp;E stain showing <br /> high-grade glioma derived RIOK2overexpression; Pten2/2; Ink4a/arf2/2 astrocytes grafted mouse brain. Tumor cells (purple) generate <br /> masses composed spindle-shaped cells infiltrative neoplastic cells invasion parenchyma blood vessels, <br /> animals sacrificed ,19 days following injection. (B) representative needle tract (arrows) mouse brain grafted control Pten2/2; Ink4a/arf2/2 <br /> astrocytes engineered vector; note slight concentration astrocytic cells needle tract tumor mass infiltrates. (C) <br /> Western blots RIOK2overexpression; Pten2/2; Ink4a/arf2/2 astrocytes compared Pten2/2; Ink4a/arf2/2 astrocytes vector, grown vitro. <br /> (D) Immunoreactivity Akt phosphorylated Serine-473 (reddish brown) tumor derived RIOK2overexpression; Pten2/2; Ink4a/arf2/2 <br /> astrocytes, tumor margin shown, surrounding normal cells (purple nuclei, faint staining). Arrow indicates strong staining invasive cells at <br /> tumor margin, asterisk indicates distant individual invasive cells; shown close-up (right). (E) Epitope tagged RIOK2 (RIOK2-flag, runs <br /> slightly larger endogenous untagged RIOK2) overexpressed 293T cells immunoprecipitated associated proteins. Blots <br /> were probed indicated proteins, lysates (pre-IP) (post-IP) included control protein expression. <br /> doi:10.1371/journal.pgen.1003253.g004 <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       8                        February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                              RIOKs Glioblastoma <br />  <br />  <br /> Ink4a/arf2/2 astrocytes [6,45]. Thus, wondered                   GBM6, DEGFR-positive neurosphere line, did undergo <br /> RIOK2 overexpression drives astrocyte transformation                apoptosis RIOK1 RIOK2 knockdown (Figure 5D). <br /> activating Akt. Consistent this, RIOK2overexpression; Pten2/2;         Moreover, LNZ308 cells, PTEN mutant [38], did not <br /> Ink4a/arf2/2 astrocytes displayed increased phosphorylation Akt          cell cycle defects strong reduction RIOK1 expression <br /> at Serine-473 (Figure 4C), tumor tissue RIOK2overexpression;       RIOK2 knockdown (Figure 5B). Both GBM6 LNZ308 <br /> Pten2/2; Ink4a/arf2/2 cells showed specific staining Akt-               mutant null p53, GBM cells cell <br /> Ser473-P (Figure 4D). Phosphorylation Serine-473 required             cycle defects apoptosis RIOK loss, U87MG, are <br /> for Akt activity select substrates FOXO3 [46],            wild-type p53 [38,41]. We observed similar lack apoptosis <br /> direct Akt substrate governs GBM cell tumorigenicity [47].             RIOK1/RIOK2 loss p53 mutant/null GBM cells, <br /> FOXO3 displayed increased phosphorylation RIOK2overexpression;         U373 (data shown). This implies survival and <br /> Pten2/2; Ink4a/arf2/2 astrocytes (Figure 4C). However, did           proliferation defects induced RIOK1 RIOK2 loss rely on <br /> observe increased phosphorylation Akt targets, including             p53. Consistent this, concomitant knockdown p53 with <br /> PRAS40, RIOK2 overexpression, suggesting effect            RIOK1 RIOK2 U87MG cells blocks apoptosis observed <br /> RIOK2 phosphorylation Akt targets selective Serine-            RIOK2 RIOK1 knockdown (Figure 5E). <br /> 473 dependent substrates [46].                                                 In human cells, RIOK1 RIOK2 transiently associate with <br />    Akt phosphorylated Serine-473 mTor-complex-2                    immature cytoplasmic 40S ribosomal subunits promote their <br /> (TORC2), multi-protein complex composed mTor kinase                maturation stimulate rRNA processing, like yeast <br /> and signaling components, including Rictor [46],            counterparts [18,19,22,51]. Defects ribosome biogenesis and <br /> protein elevated glioblastomas drives             rRNA processing activate p53-dependent ribosomal-stress <br /> gliomagenesis overexpressed astrocytic cells [48,49]. In            checkpoint suppress growth induce cell cycle arrest and <br /> yeast proteomic analyses, Rio2p shown bind Rio1p             apoptosis, process relies p53 upregulation tran- <br /> and Tor2, yeast mTor ortholog forms equivalent              scriptional activation mediated release RpL11 ribosomal <br /> TORC2 [22]. From human cells overexpressing RIOK2, RIOK2                    protein (reviewed [52,53]). In U87MG cells GBM cell <br /> co-immunoprecipitated RIOK1, mTor, Rictor, protein               lines, RIOK1 RIOK2 knockdown induced up-regulation of <br /> which definitive TORC2 complex (Figure 4E) [46]. mTor             p53 p21 cdk inhibitor, p53 transcriptional target <br /> also associates complex, mTor-complex-1 (TORC1),               (Figure 5F, Figure S13, S14), coincident knockdown RpL11 <br /> which phosphorylates mTor substrates, EIF-4E,                 RIOK1 RIOK2 blocked induction p53 p21 <br /> and composed signaling components including Raptor                (Figure 5F Figure S14). Therefore, RIOK1 RIOK2 loss <br /> protein [46]. Raptor did co-immunoprecipitate RIOK2-             leads p53 activation, requires p53-RpL11-dependent <br /> RIOK1-mTor-Rictor complex (Figure 4E), suggesting                      ribosomal stress checkpoint. <br /> RIOK2 specifically associates TORC2. Taken together, <br /> these data suggest RIOK2 directly binds TORC2                    Loss RIOK function chemosensitizes GBM cells a <br /> stimulate phosphorylation Akt Serine-473 activation            p53-dependent manner <br /> Akt select substrates, FOXO3,                    p53 downregulated GBM tumors tumor cells <br /> process directly involve RIOK1 recruitment.                             activated Akt, PTEN loss, and/or EGFR mutation/ <br />                                                                             overexpression [54]. Yet, majority tumors PTEN loss <br /> Requirement RIOK kinases proliferation                          EGFR mutation/amplification intact p53 loci (EGFR: 85&#226;&#8364;&#8220; <br /> survival GBM cells                                                       92%, PTEN: 92&#226;&#8364;&#8220;95%) [55,56]. Given RIOK loss upregulates <br />    Given knockdown Drosophila cognates yields growth          p53 levels, tested knockdown RIO kinases could <br /> reduction neoplastic glia, tested RNAi constructs targeting           potentiate response GBM cells treatments DNA- <br /> human orthologs novel suppressor kinases requirement           damaging agents doxorubicin, cooperates p53 <br /> in GBM cell survival proliferation (Text S1, Figure S12).               provoke apoptosis, [57,58] temozolomide, DNA <br /> Among these, RIOK1 RIOK2 knockdown yielded strong                        alkylator used treat GBM. In GBM cells wild-type p53, such <br /> effects, inhibiting U87MG-DEGFR U87MG proliferation                     GBM301 U87MG, knockdown RIOK1 RIOK2 <br /> (Figure 5A, 5B). DEGFR-positive neurosphere cultures,               potentiated apoptotic responses doxorubicin and/or temozolo- <br /> GBM301 GBM39, showed pronounced apoptotic                        mide (Figure 6A&#226;&#8364;&#8220;6B, Figure S15). In contrast, cells mutant for <br /> response RIOK2 RIOK1 knockdown, RIOK2 loss                       p53, LNZ308 cells, did apoptosis RIOK1 <br /> yielding stronger effects (Figure 5C, 5D). In U87MG cells,            RIOK2 knockdown doxorubicin-temozolomide treatments <br /> are dependent Akt signaling growth [50], RIOK1                    (Figure 6A). Therefore, inhibition RIO kinases chemosensi- <br /> RIOK2 RNAi provoked G2 cell cycle arrest reduced                        tizes EGFR- and/or PTEN mutant GBM cells. <br /> proliferation (Figure 5A&#226;&#8364;&#8220;5B). The phenotypes caused RIOK1                   Our results suggest elevated p53 activity potentiate <br /> and RIOK2 knockdown observed GBM cell lines                   elimination EGFR and/or PTEN mutant GBM cells. One way <br /> that PTEN and/or EGFR mutant, A172 (Figure 6,                   increase p53 levels activity nutlin-3, small molecule <br /> Figure S14, data shown). Of note, RIOK2 knockdown                       know cause cell cycle arrest sensitivity DNA- <br /> typically triggered reduction RIOK1 expression, regardless          damaging agents U87MG cells [59]. However, nutlin-3 did not <br /> the RIOK2 RNAi constructs used, suggesting RIOK2                       provoke cell cycle defects observed RIO kinase <br /> regulates RIOK1 protein levels (Figure 5D, Figure S13,             knockdown, despite inducing high levels p53 p21 (Figure <br /> Figure 6).                                                                  S16). Thus, changes induced RIO kinase loss must <br />    Given RIO kinases stimulate                  contribute cell cycle arrest apoptosis. <br /> ribosome maturation, initially suspected functional <br /> reduction RIOK1 RIOK2 cause generic cellular                      Loss RIOK function antagonizes Akt signaling <br /> toxicity. However, testing multiple GBM cell lines showed             We tested signaling alterations occur RIOK1 or <br /> strong RIOK expression, subset GBM cells                 RIOK2 knockdown explain reduced proliferation and <br /> were far affected knockdown RIOK1 RIOK2.                     survival GBM cells. The caspase inhibitor ZVAD used to <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    9                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                       RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org   10   February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                   RIOKs Glioblastoma <br />  <br />  <br />  <br /> Figure 5. RIOKs drive proliferation survival GBM cells p53-dependent manner. (A) WST-1 assays. Following selection shRNA <br /> expression, proliferation measured WST-1 reagent quantified fold increase absorbance day 0 day 3, normalized <br /> to controls treated nontargeting shRNA. 2 shRNAs tested RIO kinase. P values refer one-way ANOVA Dunnett post test. Knockdown <br /> verified Figure S13. (B) FACS analysis cell cycle. Fixed cells stained propidium iodide DNA content 72 hrs siRNA transfection. <br /> Knockdown verified Figure S13. (C, D) RIOK1 RIOK2 knockdown stimulated apoptosis neurosphere cultures, GBM39 GBM301 <br /> (brightfield, D), evidenced apoptotic features (D) PARP cleavage, indicates pro-apoptotic caspase activity (PARPfl indicates full-length <br /> PARP, PARPcl indicates cleaved PARP, long short exposures). RIOK1 RIOK2 knockdown did significantly stimulate apoptosis p53- <br /> mutant cells, GBM6, evidenced lack significant PARP cleavage. DEGFR-positive: GBM39, GBM301, GBM6. PTEN-mutant: GBM301. <br /> PTEN-intact: GBM39 GBM6. Cells harvested 96 hrs post-lentiviral infection. (E) Annexin V staining apoptosis U87MG cells, quantified by <br /> FACS. Controls (left) treated equivalent amounts nontargeting control siRNAs (light blue) control siRNAs mixed 1:1 p53 <br /> siRNAs (dark blue). RIOK1 RIOK2 siRNAs mixed 1:1 control (light blue) p53 siRNAs (dark blue). Data represented fold change <br /> in Annexin V-positive cells RIOK knockdown samples compared corresponding control samples. Knockdown verified Figure S13. (F) RIOK1 <br /> or RIOK2 knockdown upregulates p53 p21 expression, blocked concurrent RpL11 knockdown. For RpL11 experiments, cells were <br /> treated equivalent amounts siRNAs mixed 1:1 control experimental samples. Cells harvested 72 hrs post-transfection. <br /> doi:10.1371/journal.pgen.1003253.g005 <br />  <br /> dampen apoptosis preserve signaling defects. Compared                 intact p53 (Figure 7). Our results broad relevance to <br /> to controls, RIOK1 RIOK2 knockdown led reduced                           cancers RIOK2 strongly expressed range of <br /> phosphorylation Akt Serine-473 reduced phosphoryla-                  common tumor types associated high Akt activity, <br /> tion Akt target proteins FOXO3 (Figure 6C, Figure                   breast prostate cancers (Figure S18). Further study of <br /> S17). This occurred p53 wild-type p53 mutant GBM                   RIO kinases kinases identified Drosophila <br /> cells (Figure 6C, Figure S17). Serine-473 phosphorylated                 screen reveal new insights signaling defects <br /> Tor complex 2, (TORC2) [46], yeast human cancer                     underlying GBM related cancers. <br /> cells, TORC2 phosphorylation Akt stimulated mature                       RIOK1 RIOK2 upregulation associated Akt <br /> ribosomes, bind TORC2 Akt mediate                     activity GBM tumor specimens cultured cells, our <br /> their interaction, TORC2 activity blocked defects                 results Akt signaling regulates RIO kinase protein <br /> ribosome biogenesis [60]. Given RIOK1 RIOK2 loss                      stability, exact mechanism Akt regulates <br /> causes defects 40S ribosome maturation [17,18,19],              RIO kinase levels remains undetermined. RIOK2 several <br /> discovered RIOK1 RIOK2 bind TORC2 compo-                           putative mapped phosphorylation sites, including one <br /> nents, hypothesized RIOK1 RIOK2 knockdown                          putative Akt phosphorylation site (www.phosphosite.org, Figure <br /> interferes TORC2 activity. Consistent this, rea-               S8). Other studies RIOK2 phosphorylation be <br /> douts TORC2 activity, phosphorylation levels                 stimulated EGFR, carried Polo-like kinase 1 <br /> NDRG1 [60], reduced (Figure 6C, Figure S17), demon-                       [21,61], events contribute Akt-mediated <br /> strating TORC2 activity downregulated RIOK1                     regulation RIO kinase levels. Of note, standard GBM <br /> RIOK2 loss. This consistent recent findings demonstrating              cells lacking PTEN showed high levels RIO kinase expression, <br /> a requirement TORC2 signaling Drosophila glial neoplasia                non-transformed astrocytes lacking PTEN did high levels <br /> as human GBM cells [9,48,49]. However, effects                  endogenous RIO kinase protein expression relative astrocytes <br /> RIOK loss Akt signaling limited TORC2-                      intact PTEN. Therefore, factors present GBM cells <br /> dependent substrates. Phosphorylation Akt substrates,                 contribute elevated RIO kinase levels. <br /> such PRAS40 TSC2, reduced RIO kinase                      To date, published studies RIO kinases act as <br /> knockdown (Figure 6C). Thus, RIOK1 RIOK2 necessary                     ribosome assembly factors transiently associate 40S <br /> for Akt signaling GBM cells. Over-all, results strongly                 subunit promote ribosome maturation translation initiation <br /> suggest functional reduction RIOK1 RIOK2 results                [17,18,20]. Given mature ribosomes required TORC2 <br /> loss Akt activity p53 activation drive cell cycle arrest,            activation Akt phosphorylation Serine-473 [60], disruption <br /> chemosensitivity, apoptosis Akt-dependent GBM cells                Akt signaling RIOK knockdown result of <br /> intact p53 (Figure 7).                                                         defective ribosome biogenesis caused RIO kinase loss. How- <br />                                                                                ever, RIO kinases direct role in <br /> Discussion                                                                     promoting maintaining Akt activity given RIOK2 binds <br />                                                                                RIOK1 components TORC2 complex, is <br />    From Drosophila genetic screen, identified genes encoding              consistent recent studies yeast showing Rio2p bind <br /> 16 novel kinases affect EGFR- PI3K- dependent                         Tor2 [22]. Given Rio2p released mature ribosomes <br /> neoplastic glial transformation. We subset human               regulated process [20], possible reason mature <br /> orthologs novel kinases, including RIOK1 RIOK2,                  ribosomes promote TORC2 signaling release free <br /> are subject alterations GBM. RIOK1 RIOK2,                        cytoplasmic RIOK2 stimulates TORC2 assembly or <br /> related highly conserved atypical kinases, upregulated              activity. The specific interplay RIO kinases mTor <br /> in Akt-dependent manner GBM cells. Our results                 signaling, ribosome biogenesis, protein translation, Akt <br /> RIOK2 forms complex RIOK1 TORC2 signaling                           signaling require additional investigation context of <br /> components, drives activation TORC2-dependent Akt signal-                   normal abnormal PI3K RTK signaling, may <br /> ing, stimulates glial tumorigenesis. Furthermore,                 involve undetermined factors. <br /> that, GBM cells, RIOK1 RIOK2 loss causes reduction                     Although RIOK1 RIOK2 loss cause defects in <br /> Akt signaling TORC2-depdendent targets provokes                    ribosome maturation [17,18], GBM cells effects RIO <br /> p53-dependent apoptosis, cell cycle exit, chemosensitivity.                loss generic instead genotype-specific: p53 null <br /> Thus, loss-of-function gain-of-function data imply                mutant GBM cells showed major cell cycle defects apoptosis <br /> RIOK2 creates feedforward loop promotes maintains                   loss kinases. This specificity derived p53 <br /> Akt activity, disruption loop sufficient trigger             upregulation activation induced RpL11 ribosomal <br /> chemosensitivity apoptosis Akt-dependent GBM cells                 protein response RIOK loss. In humans, activation the <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      11                        February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                               RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 6. Loss RIOK1 RIOK2 function chemosensitizes GBM cells reduces TORC2-Akt signaling. (A) Knockdown RIOK1 or <br /> RIOK2 sensitizes GBM cells apoptosis response treatment doxorubicin (doxo) temozolomide (tmz), evidenced blots active <br /> caspase-3 PARP cleavage (A). All samples blotted RIOK1 RIOK2 confirm changes RIOK1 levels RIOK2 knockdown, evident p53- <br /> wild-type GBM cell lines. The RIOKs decline doxorubicin treatment. GBM301 cells treated 24 hrs 1 mg/mL doxorubicin <br /> beginning 96 hrs post infection viral vectors. U87MG, A172, LNZ308 cells treated 24 hrs 1 mg/mL doxorubicin 100 mM <br /> temozolomide beginning 72 hrs post transfection siRNAs. (B) FACS-based quantification chemosensitivity. 96 hours post shRNA infection, <br /> U87MG samples split half treated 12 hours DMSO (light blue) 1 mg/mL doxorubicin 100 mM temozolomide (red). <br /> Live cells collected stained 7AAD Annexin-V. Data represented percentage Annexin V-positive 7AAD-negative cells in <br /> each sample, averaged 2 experiments. P-values refer student&#226;&#8364;&#8482;s two-tailed t-test used compare doxorubicin temozolomide-treated <br /> control RIOK-shRNA cells. Validation knockdown shown. FACS plots raw data shown Figure S15. (C) GBM301 cells treated 25 mM <br /> ZVAD 48 hrs beginning 3 days post-infection viral vectors. Reduced phosphorylation Akt TORC2 target site, Serine-473, visible <br /> relative total Akt protein. Reduced phosphorylation Akt targets, FOXO3 transcription factor, clear phospho-epitope <br /> signal compared total protein controls. PARP cleavage read-out apoptosis; PARP cleavage fragment RIOK2 knockdown cells indicates <br /> residual caspase activity, strong effect RIOK2 loss. p53 upregulation evident GBM301 cells absence residual caspase <br /> activity. <br /> doi:10.1371/journal.pgen.1003253.g006 <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    12                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                             RIOKs Glioblastoma <br />  <br />  <br />  <br />  <br /> Figure 7. RIOK1 RIOK2 required EGFR- PI3K-mediated tumorigenesis. Pathway diagram placing RIOK1 RIOK2 in <br /> relation Akt downstream EGFR PI3K signaling GBM. RIOK2 mediates signaling upstream downstream Akt stimulation of <br /> TORC2 (left). Loss RIOK1 RIOK2 reduces Akt signaling downstream oncogenic EGFR PI3K signaling, induces p53-dependent <br /> ribosomal stress checkpoint RpL11 (right). <br /> doi:10.1371/journal.pgen.1003253.g007 <br />  <br /> RpL11-p53-dependent ribosomal-stress checkpoint associated                 (Figure 7). Further studies address mechanisms which <br /> with diseases caused ribosomal protein haploinsufficiency,                 RIO kinases modulate Akt p53 activity lead to <br /> such Diamond-Blackfan anemia, characterized                   important new insights interactions these <br /> stem progenitor cell failure [52,53]. Similarly, Drosophila,           pathways normal cancer cells. <br /> haploinsufficiency genes encode ribosomal proteins retards <br /> developmental cell proliferation [62]. Given cancer cells share          Materials Methods <br /> many properties stem progenitor cells, induction of <br /> the RpL11-p53 ribosomal stress checkpoint prove useful                    Fly stocks, genetics, culture conditions, <br /> to deplete cancer cells. Indeed, recent experimental evidence                 immunohistochemistry, gefitinib treatment <br /> indicates RpL11-p53-dependent ribosomal stress check-                   Flies cultured 25uC unless noted. Genotypes <br /> point suppresses tumorigenesis mouse cancer models [63].                   established standard genetics. Larval brain phenotypes <br /> Moreover, chemotherapeutic drugs induce ribosomal                 assessed imaged previously described [9]. Stocks were <br /> stress checkpoint [64,65]. However, drugs                  obtained VDRC, NIG, Bloomington stock centers <br /> deleterious effects unrelated ribosomal stress, limiting use.        (Table S1). wor-Gal4 lines C. Doe. To create UAS- <br /> More specific induction ribosomal stress checkpoint,                   DEGFR constructs, full-length human DEGFR cDNA was <br /> perhaps RIO kinase inhibition, prove therapeutically              cloned pUAS-T, fly stocks stable insertions were <br /> useful GBM related cancers.                                    created. <br />    The importance RIO kinases cancer cell survival been <br />                                                                                  The screen based crosses (see Text S1 genetic <br /> validated independent studies. RIOK2 recently identified in <br />                                                                               methodology) generated progeny containing single RNAi <br /> an RNAi-based screen kinases required survival of <br />                                                                               construct exclusively expressed GFP-labeled glia with <br /> glioma stem-like cells, confirms results, the <br />                                                                               dEGFRl dp110CAAX. Transgenes overexpressed using <br /> functionality RIOK2 glioma explored [66]. In <br />                                                                               glial-specific repo-Gal4 transcriptional driver. Screening was <br /> addition, RIOK1 identified cell-based RNAi screen for <br />                                                                               performed using fluorescence microscopy visualize GFP-labeled <br /> genes required Ras-mediated cell survival, the <br />                                                                               glia living larvae, phenotypes confirmed confocal <br /> functionality RIOK1 explored study [23]. Of <br />                                                                               microscopy. Each positive-scoring RNAi construct tested at <br /> note, published cell culture-based RNAi screens in <br />                                                                               twice. Positive scoring RNAi constructs tested in <br /> GBM cell lines did pick RIOK1 RIOK2 these <br />                                                                               wild-type glia, neuroblasts, neurons (Text S1, Table S4) <br /> screens designed distinguish kinases were <br /> required genotypic-specific survival growth GBM cell <br /> lines, instead focused kinases showed common                    Mammalian tissue culture techniques RNAi <br /> requirement glioblastoma cell lines tested, mutant            Established primary neurosphere cultures (gifts H. Korn- <br /> wild-type p53, EGFR, PTEN [67,68,69]. Our results,               blum) maintained previously described DMEM/F12 <br /> are derived independent multidisciplinary assays,          medium supplemented bFGF EGF [35,41]. Neurosphere <br /> to establish functional connections RIO kinases,                  cultures GBM39 GBM6 created serial <br /> oncogenesis, Akt signaling, RpL11-p53-dependent ribo-                 xenografts human GBMs (gifts C.D. James). Cultured <br /> somal stress checkpoint (Figure 7). We envision RIOK loss-of-            normal human glia derived fresh surgical specimen of <br /> function phenotypes GBM cells combined effects              normal human cortex (gift J. Olson) procured protocol <br /> of Akt inhibition p53 induction, stimulate                 approved Emory University institutional review board. <br /> apoptosis cell cycle exit EGFR- PTEN- mutant GBM                   Cultured mouse PTEN2/2; Ink4a/arf2/2 astrocytes (gift R. <br /> cells, share common dependence Akt signaling                       Bachoo) maintained DMEM 10% serum. The <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     13                      February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                          RIOKs Glioblastoma <br />  <br />  <br /> RIOK2 cDNA (Origene) overexpressed PTEN2/2; Ink4a/                 RIOK2 overexpression immunoprecipitation <br /> arf2/2 astrocytes pBabe retroviral vector.                         293T cells transiently transfected Myc-DDK-tagged <br />    The following drugs used: Nutlin-3 (Cayman), MG132,               RIOK2 constructs. Cells lysed 50 mM HEPES pH 7.5, <br /> Akt inhibitor IV (Calbiochem), temozolomide (Tocris), doxorubi-           150 mM NaCl, 5 mM EDTA, 1 mM DTT buffer protease <br /> cin, rapamycin, PP242 (Santa Cruz), PI-103, ZVAD (Enzo),                  phosphatase inhibitors [22]. RIOK2 immunoprecipitated <br /> A443654 (gift Greg Riggins), gefitinib (LC Laboratories), BEZ-         M2-agarose (Sigma) cleared lysates, washed im- <br /> 235 (Biovision), LY294002 (Cell Signaling Technology), MK-                munoprecipitates subjected immunobloting. <br /> 2206, GDC-0941 (Selleck). Doses LY294002, BEZ-235, <br /> PI-103, GDC-0941, MK-2206 used GBM cells                      Biopsy lysates immunohistochemistry tumor <br /> determined using dose response assays concentra- <br />                                                                           samples <br /> tions cells showed substantial reduction (approximately <br />                                                                             Human brain tumor biopsies tissues obtained the <br /> ,20% normal) Akt-mediated phosphorylation PRAS40 (as <br />                                                                           Brain Tumor Translational Resource protocol approved <br /> detected immunoblot). <br />                                                                           University California, Los Angeles institutional review <br />    Lentiviral shRNA pLKO.1 plasmids obtained the <br />                                                                           board. Paraffin embedded human brain tumor specimens and <br /> Broad Institute MIT. RIOK1 shRNAs: TRCN0000196278 <br />                                                                           tumor tissue microarrays matched control tissue were <br /> and TRCN0000196981. RIOK2 shRNAs: TRCN0000197250, <br />                                                                           prepared sectioned using UCLA Pathology Histology <br /> TRCN0000196672, TRCN0000196684. pLKO.1-GFP and <br />                                                                           Tissue Core Facility. Immunohistochemical staining was <br /> a nontargeting shRNA lacZ (in pLKO.1) used as <br />                                                                           performed previously described [70] specified by <br /> controls. Lentivirus produced used standard <br />                                                                           manufacturer guidelines (Sigma). The results scored by <br /> protocols (Sigma). Knockdown evident western blot 96 hrs <br />                                                                           neuropathologists according standard clinical criteria, and <br /> post-infection. For neurosphere cultures, lentivirus prepared <br />                                                                           images RIOK2 immunoreactivity taken Olympus <br /> in DMEM/F12 serum, infections cells <br />                                                                           DP72. <br /> were plated adherently [36]. <br />    For siRNAs, constructs transfected 50&#226;&#8364;&#8220;100 mM with <br /> RNAimax (Invitrogen). Unless noted, siRNA-treated <br />                                                                           Mouse intracranial graft assays <br /> cells harvested 72 hrs post-transfection. 2 sets pooled           For orthotopic implantation mouse astrocytes engineered in <br /> siRNAs tested RIOK1 RIOK2 (Dharmacon),                  vitro, low passage cells (no 8&#226;&#8364;&#8220;10 passages) used in <br /> and different nontargeting siRNAs GFP luciferase           separate experiments. 16105 cells 5 ml PBS injected <br /> were used controls (Dharmacon). For knockdown p53, p53              stereotactically 2 mm lateral midline 1 mm anterior to <br /> siRNAs used compared matched control nontargeting             bregma brains 5&#226;&#8364;&#8220;6 week old athymic nu/nu mice. <br /> siRNAs (Cell Signaling Technologies). Target sequences listed         Mice monitored animals sacrificed upon <br /> in Text S1. For dual-knockdown experiments, U87MG cells              evidence neurological symptoms experimental groups such <br /> preferred because, necessary higher doses siRNAs,             samples time-matched. Brains removed for <br /> U87MG-DEGFR cells showed nonspecific alterations DEGFR                 processing histological analysis. Sections scored inde- <br /> expression affected RIOK levels.                                     pendently neuropathologists presence tumors and <br />                                                                           injection-associated needle scars. Animals injected RIOK2- <br />                                                                           expresing cells developed tumors outside brain (n = 1) <br /> Immunoblot analysis                                                       excluded final tally. All animal experiments were <br />   Cells lysed RIPA buffer cleared lysates were <br />                                                                           approved conducted according animal welfare guidelines of <br /> subjected standard immunobloting. The following primary <br />                                                                           IACUC University California, San Diego. <br /> antibodies used: RIOK1 (Novus), RIOK2 (Sigma), p53 <br /> (Santa Cruz), p21, EGFR (BD), actin (DSHB), RpL11 (Invitro- <br /> gen), NRBP2 (Abcam), STK17a/DRAK1 (Anaspec), PDGFRa, <br />                                                                           Supporting Information <br /> VRK1, CDK9, CDK7, STK17B/DRAK2, TLK1, phospho-                            Figure S1 DEGFR signaling drives glial neoplasia. (A) Optical <br /> Akt(S473), phospho-Akt(T308), phospho-PRAS40(T246), phos-                 projections brain-ventral nerve cord complexes late <br /> pho-FOXO1(T24)/FOXO3(T32), Akt, phospho-NDRG1(T346),                      3rd instar larvae, approximately 130 hrs old, displayed same <br /> NDRG1, PRAS40, phospho-4E-BP1, 4E-BP1, PARP, cleaved                      scale. Genotypes matched displayed close-ups B&#226;&#8364;&#8220;G. <br /> caspase, FOXO3, TSC2, phospho-TSC2, mTor, phospho-mTor                    Dorsal view; anterior up. Glia labeled GFP (green) driven <br /> (T2448) (Cell Signaling Technologies)                                     repo-Gal4. (B&#226;&#8364;&#8220;G) 3 mm optical projections brain hemispheres <br />                                                                           late 3rd instar larvae, displayed scale. Frontal <br /> WST1 FACS assays                                                      sections, midway brains. Anterior up; midline left. Glial <br />    For WST-1 assays, cell lines infected lentiviral             cell nuclei labeled Repo (red); glial cell bodies labeled with <br /> shRNA constructs placed selection 48 hrs.                   GFP (green). Brains counter-stained anti-HRP (blue), which <br /> Following selection, cells plated WST1 assays cell           reveals neuropil high intensity neuronal cell bodies low <br /> proliferation/viability manufacturer&#226;&#8364;&#8482;s instructions (Clon-         intensity. (B, C) glial-specific overexpression DEGFR the <br /> tech). For flow cytometry (FACS) analysis DNA content, cells           larval brain induced excess glial cell numbers, brain enlargement, <br /> were dissociated stained propidium iodide (PI). For              lethality rescued gefitinib (A, F, H). Co- <br /> FACS analysis apoptosis, cells treated indicated            overexpression DEGFR dp110CAAX produced lethal <br /> siRNAs lentiviral vectors stained Annexin V-FITC              neoplasia, similar dEGFRl;dp110CAAX animals, that <br /> and PI 7AAD (Invitrogen, BD Biosciences). Assays                  partially suppressed gefitinib treatment (A, D, G). <br /> performed FACScaliber II flow cytometer data                However, gefitinib treatment did fully suppress growth <br /> collected using FACSdiva software (BD Biosciences). Cell cycle            neoplastic DEGFR; dp110CAAX glia did rescue lethality <br /> profiles generated using ModFit LT (Verity Software House).          caused DEGFR; dp110CAAX overexpression (G, H). <br /> In cases, 5000 cells analyzed sample.               (TIF) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                 14                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                               RIOKs Glioblastoma <br />  <br />  <br /> Figure S2 Classification EGFR; PI3K modifiers. (A) Ven                    Inhibition mTor kinase activity evident reduced Akt <br /> diagram comparing confirmed modifiers screen              phosphorylation Serine-473 and/or reduced 4E-BP1 phosphor- <br /> relevant RNAi-screens Drosophila orthologous human                 ylation. Increased 4E-BP1 phosphorylation induced by <br /> kinases implicated GBM, individual kinases noted Table S6.             rapamycin treatment, likely positive feedback [74]. RIOK2 <br /> (B, C) Network diagrams adapted STRINGS showing                         clearly elevated presence EGFR, decreased <br /> functional connections Drosophila modifier kinases (B),          mTor inhibition. RIOK1 shows reduction PP242 <br /> functional connections modifier kinases according                 treatment, rapamycin treatment. <br /> orthology information using COG (Clusters Orthologous                     (TIF) <br /> Groups) analysis (C) [26]. Confidence views networks are <br />                                                                              Figure S7 Akt signaling regulates RIO kinase protein stability. <br /> presented stronger associations represented by <br /> thicker lines. Orthologs kinases implicated GBM shown              (A) U87MG-DEGFR cells infected retroviruses <br /> in black, novel modifiers shown green (suppressors)            containing PTEN, PTEN-G129R (catalytically inactive), empty <br /> red (enhancers). Many novel modifiers                     vector. Cells serum-starved 48 hours treated for <br /> established functional links RTK PI3K signaling Drosophila          8 hours 10 mM MG132, proteosomal inhibitor. Reduced <br /> (B). When network analysis takes account datasets                  Akt phosphorylation Serine-473 evidence inhibition by <br /> orthologous kinases organisms, yeast (C),        PTEN. (B, C) U87MG-DEGFR (B) cells GBM301 (C) cells were <br /> the novel modifiers connections with                treated DMSO 2 mM A443654 10 mM <br /> other categorized modifiers RTK PI3K signaling,                       MG132 10 hrs. Akt inhibition evidenced reduced <br /> suggesting novel modifiers represent new pathway                  PRAS40 phosphorylation A443664 treated samples. <br /> components.                                                                  (TIF) <br /> (TIF)                                                                        Figure S8 RIOK2 motif scan. The RIOK2 protein sequence <br /> Figure S3 Expression human orthologs modifier kinases               examined Scansite Motif Scanner (http://scansite. <br /> high-grade human gliomas. (A&#226;&#8364;&#8220;F) Representative immunohisto-                  mit.edu/motifscan_seq.phtml) [75]. The kinase domain, is <br /> chemical stains indicated protein performed high-                highly similar RIOK1 (RIO1), indicated blue. <br /> grade malignant glioma tumor tissue,                 Potential phosphorylation sites indicated residue, lower <br /> Human Protein Atlas Project. CDK7 CDK9 nuclear                       scores indicate predicted site falls percentiles <br /> proteins, enriched immunoreactivity tumor cells.                 set high stringency. Serine-483 RIOK2 confirmed <br /> TNK2 RIOK2 known predicted cytoplasmic proteins.                  site phosphorylation unpublished proteomic <br /> Antibodies extensively validated described HPA [73],              analyses available PhosphoSitePlus (http://www.phosphosite. <br /> and data available www.proteinatlas.org. Immunostains             org) [76]. <br /> for protein performed panels 10&#226;&#8364;&#8220;24 tumors,               (TIF) <br /> this data summarized Table S10.                                        Figure S9 Potential phosphorylation RIOK2 Akt. <br /> (TIF)                                                                        Epitope-tagged RIOK2 (RIOK2-flag) immunoprecipitated <br /> Figure S4     Expression modifier orthologs cultured GBM               detected antibodies specific Akt substrates phosphory- <br /> cells expressing DEGFR. U87MG U87MG-DEGFR cells                          lated serine threonine characteristic Akt target sites <br /> were cultured .1% serum 36 hrs isolate DEGFR                     (RXXS/T). RIOK2-KD-S483A-flag epitope tagged mutant <br /> signaling, extracts immunobloted indicated                form RIOK2 contains serine-to-alanine change in <br /> proteins. DEGFR runs full-length EGFR. Proteins                   Serine-483, candidate Akt phosphorylation site (see <br /> show upregulation U87MG-DEGFR cells indicated                    Figure S8) point mutations render RIOK2 kinase <br /> with arrows.                                                                 dead block potential autophosphorylation [18]. Mutation of <br /> (TIF)                                                                        Serine-483 kinase domain did block ability the <br />                                                                              Akt substrate antibody detect RIOK2, indicating other <br /> Figure S5     RIO kinase expression panel GBM cell lines.            phosphorylation sites RIOK2 recognized the <br /> Indicated cell lines cultured .1% serum 36 hrs              antibody. <br /> reduce expression artifacts serum treatment,                  (TIF) <br /> extracts immunobloted indicated proteins. PTEN mutant <br /> status shown; SF767 documented PTEN wild-type,             Figure S10 RIOK2 expression giant cell pseudo- <br /> all documented PTEN protein null mutant.              pallisade fractions GBM tumors. Immunohistochemical staining <br /> (TIF)                                                                        RIOK2 (reddish brown), hematoxilin counterstain. Wider <br />                                                                              views heterogeneous RIOK2 immunoreactivity DEGFR- <br /> Figure S6     p110 Akt inhibition, mTor inhibition,              positive human GBMs giant cell components (A, B), cropped <br /> alters RIOK2 expression. (A) U87MG (parent) compared                      section A shown Figure 4B. Wider view showing RIOK2 <br /> U87MG-DEGFR cells, cultured .1% serum treated                     immunoreactivity pseudopallisades (C), shown cropped in <br /> 48 hrs DMSO, 500 nM BEZ-235, 2 mM PI-103,                         Figure 4D, high magnification (D) enrichment for <br /> treated 24 hrs DMSO 1 mM A443654. PI3K inhibition                RIOK2 present cellular fraction pseudopallisades. <br /> by BEZ-235 PI-103 shown reduced Akt phosphorylation at <br />                                                                              (TIF) <br /> Serine-473; blot Akt-Ser473 phosphorylation been <br /> overexposed highlight degree inhibition PI3K signaling          Figure S11    Akt EGFR GBM tumor tissues positive for <br /> by BEZ-235 compounds differences                RIOK2 expression. Immunohistochemical stains EGFR and <br /> Akt-Ser473 phosphorylation U87MG U87MG-                          Akt sections tumor samples stained for <br /> DEGFR (see Figure 2A). (B) U87MG compared U87MG-                          RIOK2 Figure 4B/Figure S10A (A) Figure 4E (B). Both <br /> DEGFR cells, cultured .1% serum treated 24 hrs               tumors strong expression EGFR phosphorylated Akt <br /> DMSO (both U87MG U87MG-DEGFR), 1 nM rapamycin,                           (Akt-S473-P). <br /> or 2 mM PP242, inhibitor mTor kinase activity.                (TIF) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    15                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                             RIOKs Glioblastoma <br />  <br />  <br /> Figure S12    WST1 proliferation assays reveal requirement             knockdown prominently stimulated apoptosis, yielding many <br /> modifier kinases GBM cells. (A&#226;&#8364;&#8220;C) WST-1 assays performed               pyknotic vacuolated cells. <br /> U87MG cells indicated genes. Following selection shRNA               (TIF) <br /> expression, proliferation measured WST-1 reagent and <br />                                                                              Figure S17 Loss RIOK1 RIOK2 function reduces <br /> quantified fold increase absorbance day 0 and <br />                                                                              TORC2-Akt signaling. (A) GBM6, DEGFR-positive p53 <br /> day 3, normalized controls treated nontargeting shRNA. <br />                                                                              mutant neurosphere line. RIOK1 RIOK2 knockdown caused <br /> 2&#226;&#8364;&#8220;3 shRNAs tested kinase. P values refer one-way ANOVA <br />                                                                              reduced phosphorylation Akt TORC2 target site, Serine- <br /> with Dunnett post test. <br />                                                                              473, clear relative total Akt protein. Cells harvested 5 <br /> (TIF) <br />                                                                              days post-infection viral vectors target RIOK1/2 a <br /> Figure S13      RIOK knockdown verification Figure 5. (A, B)             control vector, treated 25 mM ZVAD 48 hrs prior to <br /> Verification RIOK1 RIOK2 knockdown U87MG (A)                       harvest. (B) RIOK1 knockdown caused reduced phosphorylation <br /> and U87MG-DEGFR (B) cells treated indicated shRNAs                  Akt Serine-473 U87MG-DEGFR cells. Reduced <br /> and subjected WST1 assays shown Figure 5A (left panels).            phosphorylation FOXO3 TORC2-dependent Akt target <br /> (C) Verification RIOK2 knockdown LNZ308 (C) cells treated              site detected RIOK1 knockdown. Cells harvested 5 days <br /> with indicated shRNAs subjected WST1 assays                    post-infection viral vectors target RIOK1/2 control <br /> shown Figure 5A (right panel). (D, E) Verification RIOK1               vector, treated 25 mM ZVAD 48 hrs prior harvest. (C) <br /> and RIOK2 knockdown U87MG (D) LNZ308 (E) cells                        RIOK2 knockdown caused reduced phosphorylation Akt on <br /> treated indicated siRNAs subjected FACS analysis             Serine-473 U87MG cells. RIOK2 knockdown also <br /> for cell cycle progression shown Figure 5B. (F) Upregulation           associated reduced phosphorylation NDRG1, aTORC2 <br /> of p53 RIOK2 knockdown shRNAs. (G) Verification                 read-out, increased p53 protein levels. Cells treated with <br /> p53 knockdown U87MG cells treated siRNAs p53                 nontargeting control siRNAs RIOK2 siRNAs, harvested 96 hrs <br /> and RIOKs subject FACS analysis apoptosis                   post-transfection. <br /> shown Figure 5E.                                                          (TIF) <br /> (TIF) <br />                                                                              Figure S18 Expression RIOK2 tumor types. (A&#226;&#8364;&#8220;F) <br /> Figure S14    Loss RIOK1 RIOK2 induces L11-dependent                  Representative immunohistochemical stains RIOK2 per- <br /> p53 upregulation. RIOK1 RIOK2 knockdown upregulates p53                   formed malignant tumor tissue, Human <br /> and p21 expression A712 (A) U178 (B) GBM cells,              Protein Atlas Project [73]. (A) breast cancer (duct carcinoma), (B) <br /> blocked concurrent RpL11 knockdown. A172 cells PTEN-                  lung cancer (squamous cell carcinoma), (C) colorectal cancer <br /> mutant EGFR-mutant, U178 cells PTEN-mutant,                      (adenocarcinoma), (D) head neck cancer (squamous cell <br /> both wild-type p53 [38,39,50]. For RpL11 co-knockdown                carcinoma), (E) prostate cancer (adenocarcinoma), (F) <br /> experiments, cells treated equivalent amounts siRNAs            malignant melanoma. RIOK2 showed strong expression 92&#226;&#8364;&#8220; <br /> mixed 1:1 control experimental samples. Cells               100% colon head neck tumors examined, 50&#226;&#8364;&#8220;58% <br /> harvested 72 hrs post-transfection siRNAs.                              lung, prostate, melanoma tumors examined, 25% of <br /> (TIF)                                                                        breast tumors examined (complete analysis images available at <br /> Figure S15 RIOK loss chemosensitizes GBM cells. Represen-                    www.proteinatlas.org). <br /> tative scatter plots FACS-based quantification chemosensi-             (TIF) <br /> tivity. U87MG cells treated 1 mg/mL doxorubicin                Table S1 All kinases Drosophila genome. Kinases are <br /> 100 mM temozolomide 12 hrs beginning 96 hrs post shRNA                   confirmed hits dEGFRl;dp110CAAX screen bold, <br /> infection. Treated cells collected stained live 7AAD            kinases tested lack available constructs grey <br /> and Annexin-V. 7AAD identifies dead cells (Q1). Annexin V              italics. <br /> identifies cells early apoptosis (Q4) late stage        (XLS) <br /> apoptosis coincident 7AAD (Q2). Viable cells (Q3) <br /> stained neither. The percentage Annexin-V-positive cells              Table S2 Screen results UAS-dsRNA stocks tested. <br /> present doxorubicin temozolomide treatment signifi-                 VDRC stock ID numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;v,&#226;&#8364;&#8482;&#226;&#8364;&#8482; Blooming stock ID <br /> cantly increased RIOK1-shRNA RIOK2-shRNA treated                      numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;b,&#226;&#8364;&#8482;&#226;&#8364;&#8482; NIG stock ID numbers prefaced an <br /> cells compared cells treated non-targeting control                 &#226;&#8364;&#732;&#226;&#8364;&#732;n,&#226;&#8364;&#8482;&#226;&#8364;&#8482; TRIP stock numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;t.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Bold highlights all <br /> shRNA.                                                                       stocks yielded reproducible genetic interactions clear <br /> (TIF)                                                                        phenotypic alterations. Genes considered confirmed hits <br />                                                                              multiple UAS-dsRNA stocks listed bold highlighted <br /> Figure S16       Nutlin-3 treatment does phenocopy RIOK2                 yellow. Key genetic interactions: N: interaction/no <br /> loss. (A) U87MG cells treated 10 mM nutlin-3 (red) DMSO              difference, WS: weak suppressor, S: moderate suppressor, SS: <br /> (control, black) 48 hrs prior fixation propidium iodide           strong suppressor, WE: weak enhancer, E: moderate enhancer, <br /> staining DNA content cell cycle analysis FACS. (B)                SE: strong enhancer. &#226;&#8364;&#732;&#226;&#8364;&#732;?&#226;&#8364;&#8482;&#226;&#8364;&#8482; denotes interactions milder <br /> Immunnoblot showing nutlin-3 treatment significantly <br />                                                                              &#226;&#8364;&#732;&#226;&#8364;&#732;weak&#226;&#8364;&#8482;&#226;&#8364;&#8482; designation subtly different controls. <br /> increased p53 p21 levels U87MG cells, compared to <br />                                                                              (XLS) <br /> both control cells siRNA-RIOK2 treated cells. (C) GBM301 <br /> cells plated adherently infected GFP control lentivirus           Table S3 Dominant negative constructs, overexpression con- <br /> (left panel) shRNA lentivirus targeting RIOK2 (right panel).           structs, mutant alleles tested. Blooming stock ID numbers <br /> Brightfield images taken 96 hrs post-infection. RIOK2 protein                prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;b.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Bold highlights confirmed stocks yielded <br /> levels drop starting 72 hrs post-infection, 96 hrs        reproducible genetic interactions clear phenotypic alterations. <br /> equivalent 24 hrs knockdown. Adherent GBM301 cells                     Key genetic interactions: N: interaction/no difference, WS: <br /> treated 10 mM nutlin-3 24 hrs (middle panel). Nutlin-3              weak suppressor, S: moderate suppressor, SS: strong suppressor, <br /> largely decreased growth GBM301 cells, RIOK2                      WE: weak enhancer, E: moderate enhancer, SE: strong enhancer. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    16                       February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                                              RIOKs Glioblastoma <br />  <br />  <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;?&#226;&#8364;&#8482;&#226;&#8364;&#8482; denotes interactions milder &#226;&#8364;&#732;&#226;&#8364;&#732;weak&#226;&#8364;&#8482;&#226;&#8364;&#8482;                                     glioblastoma highlighted green (suppressors) red <br /> designation subtly different controls.                                                   (enhancers). *Drosophila genes did emerge annotated <br /> (XLS)                                                                                             RNAi screens, established roles RTK PI3K <br /> Table S4 Testing modifier kinase RNAi constructs wild-type                                     signaling and/or cell proliferation survival vivo. <br /> glia, neuroblasts, neurons, eye epithelia. VDRC stock ID                                      (XLS) <br /> numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;v,&#226;&#8364;&#8482;&#226;&#8364;&#8482; Blooming stock ID numbers prefaced                                     Table S8 Human Orthologs novel modifiers of <br /> by &#226;&#8364;&#732;&#226;&#8364;&#732;b,&#226;&#8364;&#8482;&#226;&#8364;&#8482; NIG stock ID numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;n.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Not stocks                                  dEGFRl;dp110CAAX. Orthologs curated http:// <br /> tested assays. Bold highlights confirmed dEGFRl;                                       kinase.com/ names updated current NCBI <br /> dp110CAAX modifier stocks yielded reproducible genetic                                       annotations. Human kinases selected study human GBM <br /> interactions. Key dEGFRl; dp110CAAX genetic interactions: N:                                   model systems bold, Drosophila counterparts. <br /> no interaction, WS: weak suppressor, S: moderate suppressor, SS:                                  (XLS) <br /> strong suppressor, WE: weak enhancer, E: moderate enhancer, <br /> SE: strong enhancer. &#226;&#8364;&#732;&#226;&#8364;&#732;?&#226;&#8364;&#8482;&#226;&#8364;&#8482; denotes interactions marginal.                               Table S9 TCGA Microarray data human orthologs novel <br /> Testing wild-type neuroblasts neurons (26 wor-Gal4, elav-                                  modifiers. Each value AgilentG4502A_07 log2 tumor/normal <br /> Gal4) UAS-dcr background potentiate                                       ratio taken TCGA profiling, http://tcga-portal.nci. <br /> RNAi. Data RNAi constructs tested neuroblasts insc-                                  nih.gov/tcga-portal/AnomalySearch.jsp. Significant overexpres- <br /> Gal4 derived http://neuroblasts.imba.oeaw.ac.at 1. &#226;&#8364;&#732;&#226;&#8364;&#732;Le-                                     sion 3-fold (.1.5) noted pink. Significantly <br /> thal,&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;semi-lethal,&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;viable&#226;&#8364;&#8482;&#226;&#8364;&#8482; refer whole-animals effects                         decreased expression noted blue. Average expression each <br /> modifier constructs wild-type background. In glial-                               gene samples noted <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span> table. Average <br /> specific assays, animal lethality indicates genes                               expression standard deviation gene EGFR- <br /> question essential aspect normal glial function                                overexpressors noted base table. <br /> or development, necessarily glial proliferation                                    (XLSX) <br /> blocking glial proliferation lethal. For analysis larval glia,                          Table S10 Summary Human Protein Atlas data adapted from <br /> fixed specimens stained glial cell nuclei/numbers                                    www.proteinatlas.org. Genes highlighted italics notable <br /> viewed higher magnification.                                                                   upregulation indicated cells tissues, assessed IHC. <br /> (XLS)                                                                                             (XLSX) <br /> Table S5 DEGFR DEGFR;dp110CAAX modifiers, listed                                           Text S1 Supplemental Results Materials Methods. <br /> stocks tested. VDRC stock ID numbers prefaced &#226;&#8364;&#732;&#226;&#8364;&#732;v.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Bold                                      Additional validation novel kinases mammalian <br /> highlights confirmed stocks yielded reproducible genetic                                 systems, screening methodology, shRNA siRNA sequenc- <br /> interactions clear phenotypic alterations. Genes                                     es. <br /> considered &#226;&#8364;&#732;confirmed hits&#226;&#8364;&#8482; initial screen listed bold. <br />                                                                                                   (DOC) <br /> Key genetic interactions: N: interaction/no difference, WS: <br /> weak suppressor, S: moderate suppressor, SS: strong suppressor, <br /> WE: weak enhancer, E: moderate enhancer, SE: strong enhancer.                                     Acknowledgments <br /> (XLSX)                                                                                            We thank Greg Riggins, Geoff Wahl, Mark Wade, Harley Kornblum, C. <br />                                                                                                   David James, Robert Bachoo, Daniel Brat, Chunhai Hao, Jeffrey Olson, <br /> Table S6 Human orthologs functional classification of <br />                                                                                                   members Thomas, Cavenee Mischel laboratories useful <br /> confirmed dEGFRl;dp110CAAX modifier kinases. Orthologs                                       discussions and/or reagents. <br /> curated http://kinase.com/ names were <br /> updated according current Gene ID numbers NCBI <br /> annotations. GO terms GOEast, http://omicslab.genetics. <br />                                                                                                   Author Contributions <br /> ac.cn/GOEAST/.                                                                                    Conceived designed experiments: RDR TRF JBT WKC FBF. <br /> (XLS)                                                                                             Performed experiments: RDR TRF GGG JW AI HY. Analyzed the <br />                                                                                                   data: RDR TRF JBT FBF WKC GGG JW SRV. Contributed reagents/ <br /> Table S7 Overlap related RNAi screens Drosophila                                      materials/analysis tools: GGG IB PSM. Wrote paper: RDR TRF. <br /> human systems. Novel modifier genes previously implicated                                  Edited manuscript: RDR JBT WKC FBF. <br />  <br /> References <br />  1. Furnari FB, Fenton T, Bachoo RM, Mukasa A, Stommel JM, et al. (2007) Malignant                      cell-cycle arrest pathways induce glioma-like lesions mice. Genes Dev 12: <br />     astrocytic glioma: genetics, biology, paths treatment. Genes Dev 21: 2683&#226;&#8364;&#8220;2710.              3675&#226;&#8364;&#8220;3685. <br />  2. Nishikawa R, Ji XD, Harmon RC, Lazar CS, Gill GN, et al. (1994) A mutant                       8.   Cloughesy TF, Yoshimoto K, Nghiemphu P, Brown K, Dang J, et al. (2008) <br />     epidermal growth factor receptor common human glioma confers enhanced                            Antitumor activity rapamycin Phase I trial patients recurrent <br />     tumorigenicity. Proc Natl Acad Sci U S A 91: 7727&#226;&#8364;&#8220;7731.                                             PTEN-deficient glioblastoma. PLoS Med 5: e8. doi:10.1371/jour- <br />  3. Mukasa A, Wykosky J, Ligon KL, Chin L, Cavenee WK, et al. (2010) Mutant                             nal.pmed.0050008 <br />     EGFR required maintenance glioma growth vivo, ablation leads              9.   Read RD, Cavenee WK, Furnari FB, Thomas JB (2009) A Drosophila model for <br />     escape receptor dependence. Proc Natl Acad Sci U S A 107: 2616&#226;&#8364;&#8220;2621.                        EGFR-Ras PI3K-dependent human glioma. PLoS Genet 5: e1000374. doi: <br />  4. Holland EC, Celestino J, Dai C, Schaefer L, Sawaya RE, et al. (2000) Combined                       10.1371/journal.pgen.1000374 <br />     activation Ras Akt neural progenitors induces glioblastoma formation                10.   Reiter LT, Bier E (2002) Using Drosophila melanogaster uncover human <br />     mice. Nat Genet 25: 55&#226;&#8364;&#8220;57.                                                                       disease gene function potential drug target proteins. Expert Opin Ther <br />  5. Marumoto T, Tashiro A, Friedmann-Morvinski D, Scadeng M, Soda Y, et al.                             Targets 6: 387&#226;&#8364;&#8220;399. <br />     (2009) Development novel mouse glioma model using lentiviral vectors. Nat                11.   St Johnston D (2002) The art design genetic screens: Drosophila <br />     Med 15: 110&#226;&#8364;&#8220;116.                                                                                    melanogaster. Nat Rev Genet 3: 176&#226;&#8364;&#8220;188. <br />  6. Bachoo RM, Maher EA, Ligon KL, Sharpless NE, Chan SS, et al. (2002)                           12.   Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) A genome- <br />     Epidermal growth factor receptor Ink4a/Arf: convergent mechanisms                               wide transgenic RNAi library conditional gene inactivation Drosophila. <br />     governing terminal differentiation transformation neural stem cell                    Nature 448: 151&#226;&#8364;&#8220;156. <br />     astrocyte axis. Cancer Cell 1: 269&#226;&#8364;&#8220;277.                                                    13.   Bellen HJ, Tong C, Tsuda H (2011) 100 years Drosophila research its <br />  7. Holland EC, Hively WP, DePinho RA, Varmus HE (1998) A constitutively                                impact vertebrate neuroscience: history lesson future. Nat Rev <br />     active epidermal growth factor receptor cooperates disruption G1                            Neurosci 11: 514&#226;&#8364;&#8220;522. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                         17                            February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                                         RIOKs Glioblastoma <br />  <br />  <br /> 14. Freeman MR, Doherty J (2006) Glial cell biology Drosophila vertebrates.          43. Szerlip NJ, Pedraza A, Chakravarty D, Azim M, McGuire J, et al. (2012) <br />     Trends Neurosci 29: 82&#226;&#8364;&#8220;90.                                                                  Intratumoral heterogeneity receptor tyrosine kinases EGFR PDGFRA <br /> 15. LaRonde-LeBlanc N, Wlodawer A (2005) A family portrait RIO kinases.                  amplification glioblastoma defines subpopulations distinct growth factor <br />     J Biol Chem 280: 37297&#226;&#8364;&#8220;37300.                                                               response. Proc Natl Acad Sci U S A 109: 3041&#226;&#8364;&#8220;3046. <br /> 16. Vanrobays E, Gelugne JP, Gleizes PE, Caizergues-Ferrer M (2003) Late                    44. Chen AJ, Paik JH, Zhang H, Shukla SA, Mortensen R, et al. (2012) STAR <br />     cytoplasmic maturation small ribosomal subunit requires RIO proteins              RNA-binding protein Quaking suppresses cancer stabilization specific <br />     Saccharomyces cerevisiae. Mol Cell Biol 23: 2083&#226;&#8364;&#8220;2095.                                      miRNA. Genes Dev 26: 1459&#226;&#8364;&#8220;1472. <br /> 17. Widmann B, Wandrey F, Badertscher L, Wyler E, Pfannstiel J, et al. (2011) The           45. Ying H, Zheng H, Scott K, Wiedemeyer R, Yan H, et al. (2010) Mig-6 controls <br />     kinase activity human Rio1 required final steps cytoplasmic                    EGFR trafficking suppresses gliomagenesis. Proc Natl Acad Sci U S A 107: <br />     maturation 40S subunits. Mol Biol Cell.                                                  6912&#226;&#8364;&#8220;6917. <br /> 18. Zemp I, Wild T, O&#226;&#8364;&#8482;Donohue MF, Wandrey F, Widmann B, et al. (2009) Distinct              46. Guertin DA, Stevens DM, Thoreen CC, Burds AA, Kalaany NY, et al. (2006) <br />     cytoplasmic maturation steps 40S ribosomal subunit precursors require                    Ablation mice mTORC components raptor, rictor, mLST8 reveals <br />     hRio2. J Cell Biol 185: 1167&#226;&#8364;&#8220;1180.                                                          mTORC2 required signaling Akt-FOXO PKCalpha, not <br /> 19. Baumas K, Soudet J, Caizergues-Ferrer M, Faubladier M, Henry Y, et al. (2012)               S6K1. Dev Cell 11: 859&#226;&#8364;&#8220;871. <br />     Human RioK3 novel component cytoplasmic pre-40S pre-ribosomal                   47. Sunayama J, Sato A, Matsuda K, Tachibana K, Watanabe E, et al. (2011) <br />     particles. RNA Biol 9: 162&#226;&#8364;&#8220;174.                                                             FoxO3a functions key integrator cellular signals control glioblastoma <br /> 20. Strunk BS, Novak MN, Young CL, Karbstein K (2012) A translation-like cycle               stem-like cell differentiation tumorigenicity. Stem Cells 29: 1327&#226;&#8364;&#8220;1337. <br />     quality control checkpoint maturing 40S ribosome subunits. Cell 150: 111&#226;&#8364;&#8220;         48. Tanaka K, Babic I, Nathanson D, Akhavan D, Guo D, et al. (2011) Oncogenic <br />     121.                                                                                        EGFR signaling activates mTORC2-NF-kappaB pathway promotes <br /> 21. Olsen JV, Blagoev B, Gnad F, Macek B, Kumar C, et al. (2006) Global, vivo,               chemotherapy resistance. Cancer Discov 1: 524&#226;&#8364;&#8220;538. <br />     site-specific phosphorylation dynamics signaling networks. Cell 127: 635&#226;&#8364;&#8220;        49. Bashir T, Cloninger C, Artinian N, Anderson L, Bernath A, et al. (2012) <br />     648.                                                                                        Conditional astroglial rictor overexpression induces malignant glioma mice. <br /> 22. Breitkreutz A, Choi H, Sharom JR, Boucher L, Neduva V, et al. (2010) A global               PLoS ONE 7: e47741. doi:10.1371/journal.pone.0047741 <br />     protein kinase phosphatase interaction network yeast. Science 328: 1043&#226;&#8364;&#8220;         50. Furnari FB, Lin H, Huang HS, Cavenee WK (1997) Growth suppression of <br />     1046.                                                                                       glioma cells PTEN requires functional phosphatase catalytic domain. Proc <br /> 23. Luo J, Emanuele MJ, Li D, Creighton CJ, Schlabach MR, et al. (2009) A                       Natl Acad Sci U S A 94: 12479&#226;&#8364;&#8220;12484. <br />     genome-wide RNAi screen identifies multiple synthetic lethal interactions          51. Vanrobays E, Gleizes PE, Bousquet-Antonelli C, Noaillac-Depeyre J, Caizer- <br />     Ras oncogene. Cell 137: 835&#226;&#8364;&#8220;848.                                                        gues-Ferrer M, et al. (2001) Processing 20S pre-rRNA 18S ribosomal RNA <br /> 24. Manning G, Plowman GD, Hunter T, Sudarsanam S (2002) Evolution                           yeast requires Rrp10p, essential non-ribosomal cytoplasmic protein. <br />     protein kinase signaling yeast man. Trends Biochem Sci 27: 514&#226;&#8364;&#8220;520.                 EMBO J 20: 4204&#226;&#8364;&#8220;4213. <br /> 25. Morrison DK, Murakami MS, Cleghon V (2000) Protein kinases                          52. Fumagalli S, Thomas G (2011) The role p53 ribosomopathies. Semin <br />     phosphatases Drosophila genome. J Cell Biol 150: F57&#226;&#8364;&#8220;62.                             Hematol 48: 97&#226;&#8364;&#8220;105. <br /> 26. Szklarczyk D, Franceschini A, Kuhn M, Simonovic M, Roth A, et al. (2011) The            53. Narla A, Ebert BL (2010) Ribosomopathies: human disorders ribosome <br />     STRING database 2011: functional interaction networks proteins, globally              dysfunction. Blood 115: 3196&#226;&#8364;&#8220;3205. <br />     integrated scored. Nucleic Acids Res 39: D561&#226;&#8364;&#8220;568.                                  54. Simmons ML, Lamborn KR, Takahashi M, Chen P, Israel MA, et al. (2001) <br /> 27. Bjorklund M, Taipale M, Varjosalo M, Saharinen J, Lahdenpera J, et al. (2006)               Analysis complex relationships age, p53, epidermal growth factor <br />     Identification pathways regulating cell size cell-cycle progression               receptor, survival glioblastoma patients. Cancer Res 61: 1122&#226;&#8364;&#8220;1128. <br />     RNAi. Nature 439: 1009&#226;&#8364;&#8220;1013. <br />                                                                                             55. McLendon R, Friedman A, Bigner D, Van Meir EG, Brat DJ, et al. (2008) <br /> 28. Bettencourt-Dias M, Giet R, Sinka R, Mazumdar A, Lock WG, et al. (2004)                     Comprehensive genomic characterization defines human glioblastoma genes <br />     Genome-wide survey protein kinases required cell cycle progression.                  core pathways. Nature 455: 1061&#226;&#8364;&#8220;1068. <br />     Nature 432: 980&#226;&#8364;&#8220;987. <br />                                                                                             56. Ohgaki H, Dessen P, Jourde B, Horstmann S, Nishikawa T, et al. (2004) Genetic <br /> 29. Friedman A, Perrimon N (2006) A functional RNAi screen regulators of <br />                                                                                                 pathways glioblastoma: population-based study. Cancer Res 64: 6892&#226;&#8364;&#8220;6899. <br />     receptor tyrosine kinase ERK signalling. Nature 444: 230&#226;&#8364;&#8220;234. <br />                                                                                             57. Lowe SW, Ruley HE, Jacks T, Housman DE (1993) p53-dependent apoptosis <br /> 30. Boutros M, Kiger AA, Armknecht S, Kerr K, Hild M, et al. (2004) Genome- <br />                                                                                                 modulates cytotoxicity anticancer agents. Cell 74: 957&#226;&#8364;&#8220;967. <br />     wide RNAi analysis growth viability Drosophila cells. Science 303: <br />                                                                                             58. Stupp R, Mason WP, van den Bent MJ, Weller M, Fisher B, et al. (2005) <br />     832&#226;&#8364;&#8220;835. <br />                                                                                                 Radiotherapy plus concomitant adjuvant temozolomide glioblastoma. <br /> 31. Neumuller RA, Richter C, Fischer A, Novatchkova M, Neumuller KG, et al. <br />                                                                                                 N Engl J Med 352: 987&#226;&#8364;&#8220;996. <br />     (2011) Genome-Wide Analysis Self-Renewal Drosophila Neural Stem Cells <br />     Transgenic RNAi. Cell Stem Cell 8: 580&#226;&#8364;&#8220;593.                                          59. Villalonga-Planells R, Coll-Mulet L, Martinez-Soler F, Castano E, Acebes JJ, <br /> 32. Sousa-Nunes R, Yee LL, Gould AP (2011) Fat cells reactivate quiescent                       et al. (2011) Activation p53 nutlin-3a induces apoptosis cellular <br />     neuroblasts TOR glial insulin relays Drosophila. Nature 471: 508&#226;&#8364;&#8220;                senescence human glioblastoma multiforme. PLoS ONE 6: e18588. <br />     512.                                                                                        doi:10.1371/journal.pone.0018588 <br /> 33. Reddy BV, Irvine KD (2011) Regulation Drosophila glial cell proliferation         60. Zinzalla V, Stracka D, Oppliger W, Hall MN (2011) Activation mTORC2 by <br />     Merlin-Hippo signaling. Development 138: 5201&#226;&#8364;&#8220;5212.                                         association ribosome. Cell 144: 757&#226;&#8364;&#8220;768. <br /> 34. Huang HS, Nagane M, Klingbeil CK, Lin H, Nishikawa R, et al. (1997) The                 61. Liu T, Deng M, Li J, Tong X, Wei Q, et al. (2011) Phosphorylation right open <br />     enhanced tumorigenic activity mutant epidermal growth factor receptor                  reading frame 2 (Rio2) protein kinase polo-like kinase 1 regulates mitotic <br />     common human cancers mediated threshold levels constitutive                     progression. J Biol Chem 286: 36352&#226;&#8364;&#8220;36360. <br />     tyrosine phosphorylation unattenuated signaling. J Biol Chem 272: 2927&#226;&#8364;&#8220;             62. Marygold SJ, Roote J, Reuter G, Lambertsson A, Ashburner M, et al. (2007) <br />     2935.                                                                                       The ribosomal protein genes Minute loci Drosophila melanogaster. <br /> 35. Laks DR, Masterman-Smith M, Visnyei K, Angenieux B, Orozco NM, et al.                       Genome Biol 8: R216. <br />     (2009) Neurosphere formation independent predictor clinical outcome         63. Macias E, Jin A, Deisenroth C, Bhat K, Mao H, et al. (2010) An ARF- <br />     malignant glioma. Stem Cells 27: 980&#226;&#8364;&#8220;987.                                                   independent c-MYC-activated tumor suppression pathway mediated by <br /> 36. Pollard SM, Yoshikawa K, Clarke ID, Danovi D, Stricker S, et al. (2009) Glioma              ribosomal protein-Mdm2 Interaction. Cancer Cell 18: 231&#226;&#8364;&#8220;243. <br />     stem cell lines expanded adherent culture tumor-specific phenotypes         64. Burger K, Muhl B, Harasim T, Rohrmoser M, Malamoussi A, et al. (2010) <br />     suitable chemical genetic screens. Cell Stem Cell 4: 568&#226;&#8364;&#8220;580.                   Chemotherapeutic drugs inhibit ribosome biogenesis various levels. J Biol <br /> 37. Lee J, Kotliarova S, Kotliarov Y, Li A, Su Q, et al. (2006) Tumor stem cells                Chem 285: 12416&#226;&#8364;&#8220;12425. <br />     derived glioblastomas cultured bFGF EGF closely mirror             65. Sun XX, Dai MS, Lu H (2007) 5-fluorouracil activation p53 involves an <br />     phenotype genotype primary tumors serum-cultured cell lines.                 MDM2-ribosomal protein interaction. J Biol Chem 282: 8052&#226;&#8364;&#8220;8059. <br />     Cancer Cell 9: 391&#226;&#8364;&#8220;403.                                                                 66. Goidts V, Bageritz J, Puccio L, Nakata S, Zapatka M, et al. (2012) RNAi <br /> 38. Ishii N, Maier D, Merlo A, Tada M, Sawamura Y, et al. (1999) Frequent co-                   screening glioma stem-like cells identifies PFKFB4 key molecule <br />     alterations TP53, p16/CDKN2A, p14ARF, PTEN tumor suppressor genes                     important cancer cell survival. Oncogene 31: 3235&#226;&#8364;&#8220;3243. <br />     human glioma cell lines. Brain Pathol 9: 469&#226;&#8364;&#8220;479.                                       67. Wurdak H, Zhu S, Romero A, Lorger M, Watson J, et al. (2010) An RNAi <br /> 39. Ciesielski MJ, Fenstermaker RA (2000) Oncogenic epidermal growth factor                     screen identifies TRRAP regulator brain tumor-initiating cell <br />     receptor mutants tandem duplication: gene structure effects                     differentiation. Cell Stem Cell 6: 37&#226;&#8364;&#8220;47. <br />     receptor function. Oncogene 19: 810&#226;&#8364;&#8220;820.                                                68. Wiedemeyer WR, Dunn IF, Quayle SN, Zhang J, Chheda MG, et al. (2010) <br /> 40. Gallia GL, Tyler BM, Hann CL, Siu IM, Giranda VL, et al. (2009) Inhibition               <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>Pattern</span> retinoblastoma pathway inactivation dictates response CDK4/6 <br />     Akt inhibits growth glioblastoma glioblastoma stem-like cells. Mol Cancer            inhibition GBM. Proc Natl Acad Sci U S A 107: 11501&#226;&#8364;&#8220;11506. <br />     Ther 8: 386&#226;&#8364;&#8220;393.                                                                        69. Kim YW, Liu TJ, Koul D, Tiao N, Feroze AH, et al. (2011) Identification of <br /> 41. Pandita A, Aldape KD, Zadeh G, Guha A, James CD (2004) Contrasting vivo                  novel synergistic targets rational drug combinations PI3 kinase <br />     vitro fates glioblastoma cell subpopulations amplified EGFR.                 inhibitors using siRNA synthetic lethality screening GBM. Neuro Oncol <br />     Gene Chromosome Canc 39: 29&#226;&#8364;&#8220;36.                                                              13: 367&#226;&#8364;&#8220;375. <br /> 42. Inda MD, Bonavia R, Mukasa A, Narita Y, Sah DW, et al. (2010) Tumor                     70. Mellinghoff IK, Wang MY, Vivanco I, Haas-Kogan DA, Zhu S, et al. (2005) <br />     heterogeneity active process maintained mutant EGFR-induced                      Molecular determinants response glioblastomas EGFR kinase <br />     cytokine circuit glioblastoma. Genes Dev 24: 1731&#226;&#8364;&#8220;1745.                                  inhibitors. N Engl J Med 353: 2012&#226;&#8364;&#8220;2024. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                   18                            February 2013 | Volume 9 | Issue 2 | e1003253 <br />                                                                                                                                         <br /> </body></html>