<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Sroka et al. BMC Bioinformatics 2011, 12:196 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  SOFTWARE                                                                                                                                      Open Access <br />  <br /> Acorn: A grid computing constraint <br /> based modeling visualization genome <br /> scale metabolic reaction networks web <br /> interface <br /> Jacek Sroka1, &#197;?ukasz Bieniasz-Krzywiec1, Szymon Gw&#195;&#179;&#197;&#186;d&#197;&#186;1, Dariusz Leniowski1, Jakub &#197;?&#196;&#8230;cki1, Mateusz Markowski1, <br /> Claudio Avignone-Rossa2, Michael E Bushell2, Johnjoe McFadden2 Andrzej M Kierzek2* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Constraint-based approaches facilitate prediction cellular metabolic capabilities, based, turn <br />   predictions repertoire enzymes encoded genome. Recently, genome annotations been <br />   used reconstruct genome scale metabolic reaction networks numerous species, including Homo sapiens, <br />   allow simulations provide valuable insights topics, including predictions gene essentiality of <br />   pathogens, interpretation genetic polymorphism metabolic disease syndromes suggestions novel <br />   approaches microbial metabolic engineering. These constraint-based simulations integrated the <br />   functional genomics portals, activity requires efficient implementation constraint-based simulations <br />   web-based environment. <br />   Results: Here, present Acorn, open source (GNU GPL) grid computing constraint-based <br />   simulations genome scale metabolic reaction networks interactive web environment. The grid- <br />   based architecture allows efficient execution computationally intensive, iterative protocols Flux <br />   Variability Analysis, readily scaled numbers models (and users) increase. The web <br />   interface uses AJAX, facilitates efficient model browsing search functions, intuitive <br />   implementation appropriate simulation conditions. Research groups install Acorn locally create user <br />   accounts. Users import models familiar SBML format link reaction formulas major <br />   functional genomics portals choice. Selected models simulation results shared different <br />   users publically available. Users construct pathway map layouts import server <br />   using desktop editor integrated system. Pathway maps used visualise numerical results <br />   web environment. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> illustrate features deployed Acorn created web server <br />   allowing constraint based simulations genome scale metabolic reaction networks E. coli, S. cerevisiae <br />   M. tuberculosis. <br />   Conclusions: Acorn free software package, installed research groups create web based <br />   environment simulations genome scale metabolic reaction networks. It facilitates shared access to <br />   models creation publicly available constraint based modelling resources. <br />  <br />  <br />  <br />  <br /> * Correspondence: a.kierzek@surrey.ac.uk <br /> 2 <br />  Faculty Health Medical Sciences, University Surrey, Guildford, GU2 <br /> 7XH, UK <br /> Full list author information available end article <br />  <br />                                         &#194;&#169; 2011 Sroka et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                         Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                         medium, provided original work properly cited. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                Page 2 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> Background                                                   conditions, determined terms set nutrients <br /> Analysis biochemical reaction networks, enable         present medium. <br /> predictions <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> catalogue            The concept gene essentiality provides good illus- <br /> molecular parts encoded genome, fulfils major       tration new capabilities. Currently, functional <br /> objective Systems Biology. Due scarcity         genomics portals gene terms com- <br /> detailed quantitative information individual           parison similar sequences organisms, direct <br /> enzyme rate constants intracellular concentrations,      experimental evidence function particu- <br /> genome scale models have, far, constructed      lar experimental conditions high throughput data- <br /> for special case metabolic reaction networks      sets expression gene been <br /> operating steady state conditions. These models        observed. A constraint-based analysis appropriate <br /> capitalize fact metabolic conversions occur      GSMN, integrated system, allow this <br /> in cell high rates therefore,      content dependent experimental conditions, <br /> given set genes active experimental conditions      specified user. The user specify compo- <br /> of interest, network metabolic reactions       sition growth medium compute the <br /> considered quasi-steady state (where metabo-      function gene expressed set metabolic cap- <br /> lite concentrations change). The       abilities gene essential user- <br /> modelled set linear relationships        defined experimental conditions. <br /> calculated reaction fluxes. Although, linear        So far, following attempts at <br /> corresponding genome scale models underdeter-          creating web-based resources, allowing computation <br /> mined exploited Linear Programming (LP)         GSMN models subsequent linking models <br /> based optimization objective function repre-     results functional genomics resources. The <br /> sents metabolic capability interest. The objective    GSMN-TB [6] reconstruction GSMN <br /> function defined flux specific reac-     Mycobacterium tuberculosis, causative agent of <br /> tion flux specific metabolite. LP opti-     Tuberculosis. In case, web interface allows the <br /> mization results unique value objective         specification growth medium composition, and <br /> function, corresponding flux distribution     using information, provides calculation the <br /> unique. The range individual reaction flux    maximal theoretical flux biomass a <br /> flux distributions consistent maximal      gene essentiality prediction. It implements reac- <br /> objective function value exploited iterative   tion essentiality scan Flux Variability Analysis <br /> LP protocols, described following sections. This      (FVA) [14], iterative protocols performing <br /> approach referred constraint based [1],     respectively linear programming optimiza- <br /> it evaluates metabolic capabilities set con-    tions reaction model. All gene <br /> straints expressed balance equations, known quanti-       names model linked Tuberculist [15], a <br /> tative values certain fluxes (determined vitro      functional genomics portal dedicated Mycobacteria. <br /> experimentation) information (<span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> literature)       The CycSim [16] allows computation the <br /> about maximal capacity reversibility reac-        maximal theoretical flux selected metabolites <br /> tions. The acquisition information initi-     does implement iterative protocols <br /> ared analysis genome sequence interest,     described above. However, implements sophisticated <br /> this approach applied reconstruction      user interface, enabling specification experimental <br /> genome scale metabolic reaction networks (GSMNs)         conditions perturbations. It allows visualisation <br /> major model organisms [2-5], pathogens [6-8]             numerical data pathway maps imported from <br /> microorganisms used industrial bioprocesses [9,10].       KEGG [17] PathwayTools [18] software. The WEbcoli <br /> The reconstruction human metabolism         [19] server allows modification constraint based <br /> available [11] currently used investigate    simulation Escherichia coli iJR904 model. The ser- <br /> mechanisms metabolic diseases [12,13].                    ver displays metabolic pathway maps overlays simu- <br />    The availability genome sequences moti-       lation results diagrams. Only single <br /> vated development functional genomics internet        evaluation objective function provided no <br /> portals, multiplicity sequence-derived, high-     iterative simulation approaches available. We also <br /> throughput traditional experimental data inte-       note web servers described are <br /> grated appropriate comparative analysis proto-       available software downloaded used <br /> cols. The integration constraint-based simulations     create new web servers users institution. <br /> GSMNs portals equip users,     Recently, BioMet Toolbox [20] server as <br /> time, tools enable predictions meta-   set constraint-based modelling tools developed by <br /> bolic phenotype cells defined experimental          Nielsen&#226;&#8364;&#8482;s group available web interface. The <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                  Page 3 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> advantage BioMet Toolbox wide range meth- <br /> ods run. However, comparison the <br /> servers mentioned integrated func- <br /> tional genomics portals visualisation tools. The user <br /> is expected create text file format particular <br /> tool download file repository models and <br /> subsequently upload file form web <br /> interface particular tool. Other servers inte- <br /> grated databases models format models for <br /> presentation web interface. Reactions linked <br /> through gene names functional genomics portals, <br /> thus facilitating interpretation models context <br /> of genome annotation. The results formatted <br />                                                                Figure 1 Components Acorn system. <br /> for web presentation numerical data visualised <br /> on network maps. <br />    In work present Acorn, grid computing sys- <br /> tem, deployed users institution         2. The worker console application performing the <br /> used create web based resources constraint based       computations requested users web interface, <br /> simulations GSMN models. Research groups                 3. A graphical editor desktop application defining <br /> install Acorn, set user accounts import models         parts model visualized how. <br /> in SBML format [21]. Users define silico        The user interface main application web <br /> experiments specifying nutritional conditions run      based, designed ergonomics mind optimised <br /> virtual experiments order calculate maximum         handling large models. As example lat- <br /> theoretical value objective function, identify         ter feature, user quickly filter long lists an <br /> essential reactions, predict gene essentiality perform    AJAX (Asynchronous JavaScript XML) based <br /> Flux Variability Analysis (FVA) [14]. Models experi-      mechanism, i.e., delays be <br /> ments shared users. Our grid-based             incurred transferring data browser and <br /> Acorn architecture facilitates implementation          web server. An example ergonomic approach <br /> computationally intensive, iterative protocols        positioning key control buttons they <br /> FVA interactive web environments            visible place screen using <br /> easily scaled number users models            specially defined CSS (Cascading Style Sheets). This <br /> grow. The web server allows            facilitates working long lists. A useful outcome of <br /> visualization numerical results pathway maps        approach user does scroll to <br /> defined user. Acorn includes integrated        start simulation accepting choice optimi- <br /> graphic editor fast creation metabolic map layouts     sation target reaction list. <br /> and command line tool, used connec-           Since computations Acorn time intensive, <br /> tion interfaces. To demonstrate features     chosen architecture (Figure 2) allows <br /> we used Acorn implement web resource pro-           easy distribution workload many <br /> viding constraint-based simulations GSMNs           machines, enabling rapid easy creation a <br /> Saccharomyces cerevisiae, E. coli M. tuberculosis :       computational cluster. Computations commissioned by <br />    http://sysbio3.fhms.surrey.ac.uk:8080/acorn/homepage.      users automatically divided subtasks, which <br /> jsf.                                                          independently computed workers running on <br />                                                               different machines. The main application <br /> Implementation                                                places subtask specifications queue pulled by <br /> In section implementation details     workers. Each worker cyclically consumes the <br /> Acorn. We explain justify technological      current oldest subtask queue, accesses cor- <br /> architectural choices made, design approaches         responding model data main applications&#226;&#8364;&#8482; data- <br /> applied implementation system.                  <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span>, performs computation stores results back <br />   The Acorn consists components               database. In addition submission date, less <br /> (Figure 1):                                                   time consuming tasks given higher priority. Then, if <br />   1. The main application, includes web based         queue contains subtasks, worker pulls the <br /> user interface defining executing required        or, subtasks currently available, <br /> computational task database storing mod-        waits new task appears. The use queue <br /> els, experiments experimental results,                    based pull model processing jobs pulled by <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                      Page 4 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 2 Interaction worker, main application web client. <br />  <br />  <br /> free workers opposed pushed                  users standalone command line tool, be <br /> main application results optimal workload distri-             used implement interfaces. <br /> bution. It renders administration                  The graphical editor standalone Java desktop <br /> easier, content queue moni-               application built NetBeans Platform, a <br /> tored workers added removed according                 generic framework Swing applications http://plat- <br /> to needs availability free processing power.                 form.netbeans.org/. It runs platforms the <br />   The implemented Java use             Java runtime environment available. It accesses the <br /> Java Enterprise Edition (Java EE) technology stack. The             main application obtain model data store <br /> main application deployed Java EE 5.0 com-              defined visualisations presentation web inter- <br /> patible application server Open Source GlassFish             face. This communication performed Web <br /> 2.1 https://glassfish.dev.java.net/. Acorn profits         Service provided main application, which, the <br /> maturity enterprise features Java EE component               user, added advantage blocked <br /> technologies. For example, operations queue              firewalls. <br /> database subject transaction control, com- <br /> putations consistently completed situations like          Results <br /> concurrent execution workers, failure                 Distribution Acorn implemented down- <br /> active worker restart main application. The               loading Google <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>Code</span> repository http://<span id='am-398' about='Thesaurus:code' typeof='owl:Thing'>code</span>.google. <br /> GlassFish server equipped necessary tools          com/p/a-c-o-r-n/. The main application installed <br /> monitoring tuning. Furthermore, flexibility          platform running Java VM, Glassfish applica- <br /> enterprise programming environment allows deploy-               tion server relational database management sys- <br /> ment Acorn relational database management               tem. The binaries linear programming engine are <br /> system providing JDBC drivers. The current version              available Linux, MacOS &#195;&#8212; Windows operating <br /> been tested Java Derby http://db.apache.org/derby/             systems and, Java VM, necessary <br /> and MySQL http://www.mysql.com databases.                           run worker. The graphical editor desktop applica- <br />   Linear programming calculations performed using               tion used running Java VM. The <br /> the appropriate functions GLPK library http://               current version Acorn provides following features. <br /> www.gnu.org/software/glpk/. The linear programming <br /> engine binary executable file implemented C++.              A web-based user interface GSMN models <br /> The worker starts linear programming engine using               The user logs web interface can <br /> the operating interface Java, sends             browse GSMN models tabular format (Figure 3). The <br /> linear programming problem standard input                rows reaction tables contain reaction name, reaction <br /> retrieves results standard output creating             formula lower upper flux bounds. The cell <br /> any temporary files. The engine available               reaction entry contains gene-reaction association <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                    Page 5 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 3 The web page representingt GSMN. The table containing reaction names interactively filtered keyword &#226;&#8364;&#339;alanine&#226;&#8364;? the <br />  reaction name. Reaction bounds editable. The gene names <span id='am-5' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> gene-reaction association formula linked functional <br />  genomics portal chosen user created model. <br />  <br />  <br /> formula representing genes encoding protein subunits                     results calculation flux range consistent <br /> enzymes catalysing reaction. The genes encoding                        maximal value objective function. <br /> components multisubunit enzymes linked                          The flux ranges unique and, unlike flux values <br /> AND boolean statement paralogous genes                              obtained particular single FBA solution, be <br /> represented OR statements. Gene names linked                         used reason internal flux distribution under <br /> to gene pages selected functional genomics resources                     nutritional conditions [22]. <br /> integrating GSMN model literature annotations <br /> and comparative sequence analysis results. Reaction                       To run simulation Acorn web interface the <br /> bounds editable used set initial conditions              user selects methods specifies <br /> of constraint-based simulations.                                    objective function nutritional conditions. The <br />                                                                         objective function specified metabolite name <br /> A web interface constraint-based analysis                       (maximal flux metabolite) reaction name <br /> protocols                                                               (maximal flux reaction). Nutritional condi- <br /> The following computational methods available:                      tions defined constraining lower upper <br />                                                                         bounds transport reaction fluxes. The Acorn interface <br />     i) Flux Balance Analysis (FBA) - calculation                 allows specification bounds reactions the <br />     maximal theoretical value objective function                 system, allowing complex analysis scenarios <br />     nutritional conditions specified user;                 incorporation internal metabolic flux measure- <br />     iI) Gene essentiality prediction - fluxes           ments simulations multiple gene knockouts. The <br />     reactions require functionality selected             keyword searches used quickly reactions <br />     gene constrained 0 maximal theoretical               metabolites interest. Simulations scheduled <br />     value objective function calculated. If               user wait completion com- <br />     objective function value equals 0, gene declared             putational tasks, log return later <br />     essential particular metabolic function;                  examine results. The simulation scenario, together <br />     iii) Reaction essentiality scan - reaction, turn,           simulation results saved database as <br />     inactivated FBA optimisation run. The                 silico experiment. Users share silico experi- <br />     result lists maximal value objective func-               ments visualise results pathway diagrams. <br />     tion single reaction <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>knock-out</span>; <br />     iv) Flux Variability Analysis (FVA) [14] - flux                 A grid-based environment distributing computational <br />     corresponding objective function constrained               load <br />     maximal value flux reac-                   A single execution FBA computationally effi- <br />     tion, turn, maximised minimised. This                     cient. For example, calculation maximal growth <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                     Page 6 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> rate yeast GSMN model iND750 takes 0.73s 2.13                     A user management, model silico experiment <br /> GHz Intel Core 2 Duo processor CPU calculated                       sharing <br /> with linear programming engine Acorn sys-                     Every new user Acorn register web <br /> tem. However, iterative protocols reaction essentiality               interface. The user registered, software <br /> scans FVA require (respectively) evalua-                  installation step, given administration privileges. The <br /> tions objective function reaction                 models created SBML file upload, initi- <br /> system. Therefore, execution FVA yeast                        ally used user uploaded the <br /> GSMN model containing 1265 reactions takes 15                      SBML file. Likewise, silico experiments parti- <br /> minutes CPU time. This computational time                      cular model initially belong user created the <br /> substantial web-based applications, especially mul-               model. The user chose share models in <br /> tiple users simultaneously using server. The                     silico experiments users system. This <br /> Acorn server, hand, using grid-based                     allows creation online communities where <br /> solutions distribute computational load mul-                  users share models results numerical simulations. <br /> tiple computers. The worker program installed                      The models experiments available <br /> each cluster. The worker connects                     publicly accessible account, enables creation <br /> the Acorn server receives set FBA objective                     web based resources constraint based simulations <br /> function evaluation tasks execute. Results                   GSMN models. <br /> tasks sent server (see Implementation <br /> section details). Therefore, FBA calculations                   Desktop metabolic pathway layout editor <br /> from Acorn server distributed arbi-                   Acorn provides desktop editor metabolic pathway <br /> trary number computers local area network                   diagrams (Figure 5). Diagrams created Petri net <br /> the internet, rendering scalable large num-                [24] notation reactions substances represented <br /> ber users computational tasks.                                    different symbols. The editor connects Acorn <br />                                                                          database, using ID password Acorn <br /> SBML file import                                                         users prompts user select GSMN <br /> The user import GSMN models SBML format                       models available user&#226;&#8364;&#8482;s Acorn account. Subse- <br /> (Figure 4). The reaction capacity bounds gene-reaction               quently, user creates diagram every <br /> association formulas, features specific FBA simulations               substance reaction symbol associated the <br /> and implemented Acorn yet, provided                 substance reaction GSMN. The editor find <br /> annotations following BiGG database conventions [23].                    substances (reactions) associated given reac- <br /> The user upload file using web interface.                    tion (substance) place map. <br /> Optionally, user input HTTP address <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span>,                  Therefore, placing symbol diagram, <br /> which concatenated gene names create links               user does check subsequent metabolite <br /> to gene pages functional genomics portal               names quickly build network map simply <br /> user&#226;&#8364;&#8482;s choice. The SBML file parsed stored                 following connections GSMN. The diagram is <br /> relational database backend area Acorn system.                    sent server saved database. <br />  <br />  <br />  <br />  <br />  Figure 4 The model import interface. The user prompted provide the SBML file BiGG database annotations <br />  representing FBA specific information. Subsequently, user specifies model organism. The user link gene names the <br />  functional genomics portal choice providing HTTP address gene replaced &#226;&#8364;&#339;%s&#226;&#8364;? string. This address template then <br />  used link individual genes model pages corresponding pages functional genomics portal. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                     Page 7 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 5 Screenshot desktop editor session S. cerevisiae iND750 model. The user create network map circles <br />  represent substances reactions represented rectangles. The symbols linked substances reactions GSMN. Therefore, <br />  map does need represent genome scale network, numerical results, mapped diagram obtained <br />  computations entire model. The map layout send Acorn server stored database used generation of <br />  visualisations web interface. Numerical results visualised desktop editor. The figure shows results FBA <br />  optimisation biomass flux. <br />  <br />  <br /> A means visualisation numerical results                         script languages used bioinformatics community <br /> metabolic pathway diagrams                                               annotation pipeline. It <br /> Numerical results FBA FVA calculations                     used create graphics user interfaces other <br /> visualised pathway diagrams (Figure 6). The pathway                   Acorn. Finally, users Acorn web server can <br /> diagrams created user available                    use command line tool run their <br /> web associated simulation                         computations local machines resources of <br /> results. Subsequently, user create image files                   web server limited constrained. <br /> where numerical simulation results, representing the <br /> fluxes flux ranges, printed boxes corre-               The GSMN web resource <br /> sponding reactions. Numerical results                 To demonstrate features Acorn, con- <br /> visualised way stand-alone path-                  structed web resource constraint-based simulation <br /> way editor.                                                              GSMNs research group. The <br />                                                                          resource contains GSMN-TB model M. tuberculo- <br /> A stand command line tool                                          sis [6], reconstructed group reconstruc- <br /> The computational engine Acorn software                        tions published groups: E. coli iAF1260 [2] and <br /> available stand-alone tool. The tool used                 S. cerevisiae iND750 [3]. The iAF1260 iND750 have <br /> evaluate objective function GSMN model                        directly uploaded Acorn SBML files <br /> written simple text file format. The command line                     available BiGG database. This shows the <br /> tool interprets boolean gene-reaction association formu-                 Acorn model upload tool compatible SBML <br /> las used perform evaluate                    specification constraint-based specific annotations <br /> objective function single gene <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>knockout</span> version                 used COBRA toolbox software BiGG <br /> the model. Availability Acorn computational                       database [23]. In case GSMN-TB iND750 <br /> engine form command line tool binary                      models, flat files accompanying publications <br /> useful users intend develop                     converted annotated SBML file sub- <br /> interfaces. The tool easily integrated                 sequently uploaded server. The models have <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                        Page 8 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 6 Visualisation numerical results pathway maps web interface Acorn system. Network map created with <br />  desktop editor stored database used. The reaction fluxes FVA flux ranges printed reaction symbols. The figure shows <br />  image generated web interface showing reaction fluxes the central metabolism map S. cerevisiae iND750 model. The <br />  results obtained FBA optimisation biomass flux. <br />  <br />  <br /> been uploaded tested users sys-                  pathway layouts choice. The fluxes are <br /> tem upload privileges subsequently published,                    obtained calculations involving reactions of <br /> by sharing &#226;&#8364;&#339;guest&#226;&#8364;? user account available                   entire GSMN, reactions to <br /> to public.                                                            users displayed format, most <br />    Each models available web resource                   informative them. <br /> is linked relevant functional genomics portal. The <br /> gene names GSMN-TB model linked                             Conclusions <br /> Tuberculist database, E. coli model linked                      In paper, present Acorn software package, <br /> KEGG database, S. cerevisiae model linked                   deployed user&#226;&#8364;&#8482;s institution used <br /> the SGD database. These implementations demon-                       create web-based resources constraint based <br /> strated model upload mechanism Acorn                      simulations GSMNs. Our software used to <br /> be used link reaction lists results files GSMN                  make collection GSMNs available community <br /> models major functional genomics resources,                       users hold accounts system. Users <br /> facilitating interpretation simulations                upload models run constraint-based simulation <br /> context wealth knowledge comparative                        methods make predictions metabolic cap- <br /> analysis results available knowledge bases.                  abilities organisms investigation. The models <br />    The user use web resource run simula-                   results linked functional genomic <br /> tion protocols implemented Acorn models                         resources gene names. Therefore, the <br /> mentioned above. To demonstrate visualisation                         results constraint based calculations examined <br /> functionality Acorn, used desktop editor                   context genome sequences, annotations and <br /> to create layout glycolytic pathway Yeast                     sequence similarity-based predictions available these <br /> uploaded layout iND750 model. Users                     portals. The grid-based architecture Acorn allows for <br /> now visualise fluxes calculated simulation                    implementation computationally expensive <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                 Page 9 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> simulation protocols requiring execution thou-        code distribution, used create collabora- <br /> sands linear programming evaluations entire       tive model development environments public web <br /> model. Users visualise numerical results simula-      resources laboratories. <br /> tion methods pathway maps loaded system.           To summarise, believe Acorn server will <br /> These maps represent tthe users view,      useful research groups reconstructing and <br /> model specific user concerned,      studying GSMNs. Its application result establish- <br /> numerical data visualised map calculated       ment online communities sharing models results <br /> constraint-based modelling entire network. Path-      silico experiments. This facilitate improvement <br /> way diagrams constructed desktop tool,          metabolic models used identification novel <br /> which supports rapid assignment graphical sym-        drug targets microbial pathogens metabolic engi- <br /> bols substances reactions present model.       neering industrially valuable microbial strains. <br />    One important features Acorn user <br /> management model sharing. Select users upload        Availability Requirements <br /> starting models share others. All        Project name: Acorn <br /> users create variations starting models        Project home page: http://code.google.com/p/a-c-o-r- <br /> modifying parameters set simula-         n/ <br /> tions affecting models simulations run         Surrey GSMN server: http://sysbio3.fhms.surrey.ac. <br /> by users. They choose render           uk:8080/acorn/homepage.jsf <br /> models, simulation scenarios results visible      Operating systems(s): Platform independent <br /> users and, guest user account,       Programming language: Java, C++ <br /> available research community. Therefore, Acorn          Other requirements: Java 6.0 higher, GlassFish Java <br /> can used create research community websites            EE Server v2 higher <br /> dedicated annotation investigation meta-         License: GNU GPL v2 <br /> bolic reaction network models.                                 Any restrictions use non-academics: The use of <br />    In addition software used create    constraint based modelling results obtained using <br /> a constraint-based modelling server, provide       iAF1260 iND750 models Surrey FBA resource is <br /> web resource giving access GSMN models S.          subjected BiGG database license. <br /> cerevisiae, E. coli, M. tuberculosis. Users either <br /> use guest account register gain access these <br />                                                              Acknowledgements <br /> models need installing learning mod-      This work supported BBSRC EraSysbioPlus grant BB/ <br /> elling environments as, example, MATLAB             I00453X/1. <br /> based COBRA toolkits. Users use server to <br />                                                              Author details <br /> upload models, represented          1 <br />                                                               Institute Informatics University Warsaw, Poland. 2Faculty Health and <br /> SBML format, use computational tools implemented         Medical Sciences, University Surrey, Guildford, GU2 7XH, UK. <br /> within study models (and share the <br />                                                              Authors&#226;&#8364;&#8482; contributions <br /> results users).                                   AMK JS designed software functionality. JS designed software <br />    We believe Acorn provides advantages         architecture, supervised software development participated software <br /> and alternative solutions compared existing web ser-      implementation. LBK, SG, DL, JL implemented database worker modules. <br />                                                              MM implemented pathway visualisation. JL implemented command line <br /> vers constrained-based modelling metabolic net-       computational engine. CAR developed case studies visualisation yeast <br /> works. It provides advantages compared           pathways. MB JMF tested software functionality participated in <br /> GSMN-TB server respect visualisation             writing manuscript. AMK JS wrote manuscript. All authors <br />                                                              read approved final manuscript. <br /> implements convenient interface model spe- <br /> cification. Its main advantage CycSim WEbcoli       Received: 18 August 2010 Accepted: 24 May 2011 <br /> is implementation FVA reaction essenti-       Published: 24 May 2011 <br /> ality scans. FVA currently method, which <br />                                                              References <br /> allows exploration internal flux distribution      1. Price ND, Reed JL, Palsson BO: Genome-scale models microbial cells: <br /> GSMN.s It efficiently implemented Acorn,            Evaluating consequences constraints. Nature Reviews Microbiology <br /> its grid based architecture. Acorn provides alter-       2004, 2(11):886-897. <br />                                                              2. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> native mechanism visualising metabolic pathway               Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale metabolic <br /> maps. While CyCsim uses standard KEGG BioCyc                 reconstruction Escherichia coli K-12 MG1655 accounts 1260 <br /> maps, Acorn server allows individual user cre-        ORFs thermodynamic information. Molecular Systems Biology 2007, <br />                                                                  3:18. <br /> ate views, linked model.        3. Duarte NC, Herrgard MJ, Palsson BO: Reconstruction validation of <br /> Another major advantage Acorn environment                 Saccharomyces cerevisiae iND750, fully compartmentalized genome- <br /> over CycSim WEbcoli provides source               scale metabolic model. Genome Research 2004, 14(7):1298-1309. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                                      Page 10 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> 4.    Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome-scale <br />       reconstruction metabolic network Bacillus subtilis based high-        doi:10.1186/1471-2105-12-196 <br />                                                                                      Cite article as: Sroka et al.: Acorn: A grid computing for <br />       throughput phenotyping gene essentiality data. Journal Biological <br />                                                                                      constraint based modeling visualization genome scale <br />       Chemistry 2007, 282(39):28791-28799. <br />                                                                                      metabolic reaction networks web interface. BMC Bioinformatics <br /> 5.    Poolman MG, Miguet L, Sweetlove LJ, Fell DA: A Genome-Scale Metabolic          2011 12:196. <br />       Model Arabidopsis Some Its Properties. Plant Physiology 2009, <br />       151(3):1570-1581. <br /> 6.    Beste DJV, Hooper T, Stewart G, Bonde B, Avignone-Rossa C, Bushell M, <br />       Wheeler P, Klamt S, Kierzek AM, McFadden J: GSMN-TB: web-based <br />       genome scale network model Mycobacterium tuberculosis <br />       metabolism. Genome Biology 2007, 8(5). <br /> 7.    Becker SA, Palsson BO: Genome-scale reconstruction metabolic <br />       network Staphylococcus aureus N315: initial draft two- <br />       dimensional annotation. Bmc Microbiology 2005, 5:12. <br /> 8.    Oberhardt MA, Puchalka J, Fryer KE, dos Santos V, Papin JA: Genome-scale <br />       metabolic network analysis opportunistic pathogen Pseudomonas <br />       aeruginosa PAO1. Journal Bacteriology 2008, 190(8):2790-2803. <br /> 9.    Borodina I, Krabben P, Nielsen J: Genome-scale analysis Streptomyces <br />       coelicolor A3(2) metabolism. Genome Research 2005, 15(6):820-829. <br /> 10.   Chung BKS, Selvarasu S, Andrea C, Ryu J, Lee H, Ahn J, Lee DY: Genome- <br />       scale metabolic reconstruction silico analysis methylotrophic <br />       yeast Pichia pastoris strain improvement. Microbial Cell Factories 2010, <br />       9:15. <br /> 11.   Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, <br />       Palsson BO: Global reconstruction human metabolic network <br />       based genomic bibliomic data. Proceedings National <br />       Academy Sciences United States America 2007, 104(6):1777-1782. <br /> 12.   Jerby L, Shlomi T, Ruppin E: Computational reconstruction tissue- <br />       specific metabolic models: application human liver metabolism. Mol <br />       Syst Biol 2010, 6:401. <br /> 13.   Gille C, B&#195;&#182;lling C, Hoppe A, Bulik S, Hoffmann S, Ganeshan R, K&#195;&#182;nig M, <br />       Rother K, Weidlich M, Behre J, Holzhutter HG: HepatoNet1: a <br />       comprehensive metabolic reconstruction human hepatocyte for <br />       analysis liver physiology. Mol Syst Biol 2010, 6:411. <br /> 14.   Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br />       constraint-based genome-scale metabolic models. Metabolic Engineering <br />       2003, 5(4):264-276. <br /> 15.   Camus JC, Pryor MJ, M&#195;&#169;digue C, Cole ST: Re-annotation genome <br />       sequence Mycobacterium tuberculosis H37Rv. Microbiology 2002, <br />       148:2967-2973. <br /> 16.   Le Fevre F, Smidtas S, Combe C, Durot M, d&#226;&#8364;&#8482;Alche-Buc F, Schachter V: <br />       CycSim-an online tool exploring experimenting genome- <br />       scale metabolic models. Bioinformatics 2009, 25(15):1987-1988. <br /> 17.   Kanehisa M, Goto S, Furumichi M, Tanabe M, Hirakawa M: KEGG for <br />       representation analysis molecular networks involving diseases <br />       drugs. Nucleic Acids Res 2009, 38:D355-D360. <br /> 18.   Karp PD, Paley SM, Krummenacker M, Latendresse M, Dale JM, Lee TJ, <br />       Kaipa P, Gilham F, Spaulding A, Popescu L, Altman T, Paulsen I, Keseler IM, <br />       Caspi R: Pathway Tools version 13.0: integrated software pathway/ <br />       genome informatics systems biology. Brief Bioinform 2010, 11. <br /> 19.   Jung TS, Yeo HC, Reddy SG, Cho WS, Lee DY: WEbcoli: interactive and <br />       asynchronous web application silico design analysis of <br />       genome-scale E.coli model. Bioinformatics 2009, 25(21):2850-2852. <br /> 20.   Cvijovic M, Olivares-Hernandez R, Agren R, Dahr N, Vongsangnak W, <br />       Nookaew I, Patil KR, Nielsen J: Nucleic Acids Research 2010, 38:W145-W149. <br /> 21.   Hucka M, Finney A, Sauro HM, Bolouri H, Doyle JC, Kitano H, Arkin AP, <br />       Bornstein BJ, Bray D, Cornish-Bowden A, et al: The systems biology <br />       markup language (SBML): medium representation exchange of <br />       biochemical network models. Bioinformatics 2003, 19(4):524-531.                   Submit manuscript BioMed Central <br /> 22.   Bushell ME, Sequeira SIP, Khannapho C, Zhao H, Chater KF, Butler MJ,              advantage of: <br />       Kierzek AM, Avignone-Rossa CA: The use genome scale metabolic flux <br />       variability analysis process feed formulation based an <br />                                                                                         &#226;&#8364;&#162; Convenient online submission <br />       investigation effects zwf <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> antibiotic production <br />       Streptomyce <br /> </body></html>