<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An Unusual Fibrillarin Gene and Protein: Structure <br /> and Functional Implications <br />  <br />  <br />         The diploid germinal nucleus of the ciliated protozoan Tetrahymena thermophila is unusual <br />         among eukaryotes in that it encodes a single copy of the gene for rRNA allowing <br />         identification of cis-acting mutations in rDNA affecting rRNA structure, function, and <br />         processing. The generally conserved nucleolar protein fibrillarin has been characterized <br />         <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> a number of systems and is involved in pre-rRNA processing. We have demon- <br />         strated that Tetrahymena has fibrillarin and have analyzed the cDNA and the genomic <br />         DNA encoding this protein. The derived amino acid sequence of the N-terminal region <br />         of Tetrahymena fibrillarin shows little similarity with the generally highly conserved <br />         glycine/arginine-rich N-terminal domain of other eukaryotic fibrillarins. The remainder <br />         of the amino acid sequence of the molecule is more conserved. Polyclonal antibodies <br />         generated against the full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> Tetrahymena fibrillarin expressed in bacteria recognize a <br />         protein of Mr approximately 32,000 in whole-cell or nucleolar preparations. Immunocy- <br />         tochemistry localizes fibrillarin <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> nucleoli in the somatic macronuclei of vegetative cells. <br />         Transformation experiments demonstrate that fibrillarin is an essential protein in Tetra- <br />         hymena. The Tetrahymena fibrillarin is expressed but does not complement a NOP1 null <br />         <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span> when transformed into the yeast Saccharomyces cerevisiae, indicating less func- <br />         tional conservation among fibrillarins than previously suggested. <br />  <br /> INTRODUCTION                                                           region is the site of further assembly and maturation <br /> In eukaryotes, transcription of the genes encoding <br />                                                                        of preribosomal particles (Hozak, 1995). Raska et al. <br />                                                                        (1995) have presented a somewhat different view. <br /> rRNA (rDNA), processing of cytoplasmic rRNA pre-                       They suggest that transcription sites are present also at <br /> cursors, and assembly of ribosomal particles occur in                  the border region between the DFC and FC and pro- <br /> the nucleolus. Subnucleolar components have been                       pose a hypothesis for a dynamic process in metaboli- <br /> identified in some systems by electron microscopy in                   cally active mammalian nucleoli where FC and DFC <br /> combination with immunoelectron microscopy                             form a single functional domain for rDNA transcrip- <br /> (Scheer and Rose, 1984; Spector et al., 1984; Hugle et al.,            tion. <br /> 1985a,b; Ochs et al., 1985a; Schmidt-Zachmann et al.,                    Although it is assumed that the complex structural <br /> 1987) and some of the pathways of rDNA metabolism                      organization of the nucleolus is required for its func- <br /> and ribosome biogenesis have been localized to spe-                    tion, a great deal remains to be determined about the <br /> cific nucleolar regions. For example, it has been sug-                 role of nucleolar proteins in ribosome biogenesis. A <br /> gested that RNA polymerase I-catalyzed transcription                   number of nucleolar proteins have been identified and <br /> of rDNA occurs in the fibrillar centers (FCs) of nucle-                some have been localized to various subnucleolar re- <br /> oli. Surrounding the fibrillar centers is the dense fibril-            gions (Ochs et al., 1985b). Fibrillarin, an abundant <br /> lar component (DFC), the site of preribosome forma-                    nucleolar protein of Mr 34,000-38,000 found in the <br /> tion. The granular compartment (GC) outside the FC                     DFC, has been identified in a number of systems in- <br />                                                                        cluding unicellular eukaryotes and mammals. Identi- <br />   * Present address: Amgen Research   Institute,   Toronto, Ontario,   fication and characterization of fibrillarin in mamma- <br />     Canada.                                                            lian cells has been aided by its precipitation with <br />   tCorresponding author.                                               autoimmune serum from patients with scleroderma <br />  <br /> i 1997 by The American Society for Cell Biology                                                                             1051 <br />  E. David et al. <br />  <br /> (Lischwe et al., 1985; Ochs et al., 1985b; Parker and          tionally silent rRNA gene on micronuclear chromo- <br /> Steitz, 1987). A protein in the budding yeast Saccharo-        some 2r is rearranged to yield multiple extrachromo- <br /> myces cerevisiae immunologically related to vertebrate         somal copies of a palindromic transcriptionally active <br /> fibrillarins is encoded by the NOP1 gene. The yeast            rDNA molecule in the macronucleus (Yao and Gall, <br /> Nopl protein is essential for growth and required for          1977; Engberg, 1985). This allows for the isolation and <br /> pre-rRNA processing (Schimmang et al., 1989).                  characterization of cis-acting mutations in rDNA, a <br />   Ribonucleoproteins (RNPs) containing a number of             situation unique among eukaryotes. In addition, the <br /> small nucleolar RNAs (snoRNAs) are precipitated by             extrachromosomal nature of rDNA in Tetrahymena al- <br /> anti-fibrillarin antibodies (Lischwe et al., 1985; Parker      lows the free nucleoli to be regarded as a single gene <br /> and Steitz, 1987; Tyc and Steitz, 1989, Maxwell and            organelle, quite distinct from the situation in most <br /> Fournier, 1995; Balakin et al., 1996). This association of     other eukaryotes where the content of rDNA may be <br /> fibrillarin and snoRNAs implies a role for fibrillarin in      only a few percent of the total nucleolar DNA. These <br /> processing of primary transcripts of rRNA. Direct ev-          genetic and biochemical characteristics make this a <br /> idence for the involvement of fibrillarin, as well as a        useful system for addressing questions about nucleo- <br /> number of fibrillarin-associated snoRNAs, in pre-              lar structure and function. <br /> rRNA processing has been obtained (Kass et al., 1990;            As part of our ongoing studies on rDNA metabolism <br /> Hughes and Ares, 1991; Tollervey et al., 1991). These          in Tetrahymena, we have initiated experiments aimed <br /> data reinforce the notion that nucleolar function is           at addressing questions about structure/function re- <br /> dependent on nucleolar structure and that fibrillarin          lationships in nucleoli. The data presented support a <br /> may be a key component in this interrelationship.              suggestion that there is less structural conservation <br />   Genes or cDNAs encoding fibrillarin from a number            and functional interchangeability among fibrillarins <br /> of eukaryotes have been cloned and sequenced                   than previously thought and this may have important <br /> (Schimmang et al., 1989; Henriquez et al., 1990; Lap-          implications for rDNA metabolism and for nucleolar <br /> eyre et al., 1990; Aris and Bloebel, 1991; Girard et al.,      structure and function. <br /> 1993; Turley et al., 1993; Cappai et al., 1994). The amino <br /> acid sequences and predicted secondary structures are          MATERIALS AND METHODS <br /> highly conserved among eukaryotic fibrillarins. One <br /> of the most highly conserved regions among eukary-             Strains and Microbiological Methods <br /> otic fibrillarins is the N-terminal region of approxi-         The diploid S. cerevisiae strain Y242 is that designated TF38NULL by <br /> mately 85 amino acids, which is very rich in glycine           Jansen et al. (1991) and was provided by Dr. D. Tollervey (Heidel- <br /> and arginine residues and is referred to as a glycine/         berg, Germany). It is heterozygous for the NOP1 alleles, one chro- <br /> arginine-rich (GAR) domain. Furthermore, human                 mosomal copy of NOP1 being disrupted by the URA3 gene. Trans- <br />                                                                formation of yeast was carried out using the one-step protocol of <br /> and Xenopus fibrillarins can complement NOP1 dis-              Chen et al. (1992). Yeast media and manipulations with yeast were <br /> ruption strains of S. cerevisiae (Jansen et al., 1991). This   as described in Sherman et al. (1986). Tetrads from Y242 transformed <br /> extent of conservation over a great diversity of organ-        with plasmids expressing Tetrahymena fibrillarin were picked and <br /> isms is generally seen in only a few &quot;housekeeping&quot;            grown on YEPD medium (YEPgal when the inducible GAL pro- <br />                                                                moter was used) at 25&#194;&#176;C and then replicated to appropriate selective <br /> proteins and is suggestive of a very conserved func-           medium and grown at the same temperature. The plasmid pVT-U of <br /> tion of fibrillarin. These observations suggest that           Vemet et al. (1987) was modified by replacing the URA3 gene on a <br /> there is structural conservation and functional inter-         BglII fragment with a BglII fragment of LEU2. The Tetrahymena <br /> changeability among fibrillarins, supporting the idea          fibrillarin cDNA was inserted into the polylinker of this plasmid at <br /> that this protein affects both nucleolar structure and         the BamHI site in the proper orientation to be expressed from the <br />                                                                ADH promoter. Plasmid constructs in YEp 51 (Broach et al., 1983) <br /> function.                                                      with fibrillarin expressed from an inducible GAL promoter and a <br />   A number of unique aspects of the biology of the             construct using the PGK promoter were also used. <br /> ciliated protozoan Tetrahymena thermophila offer an op- <br /> portunity to address important biological questions in         Recombinant DNA Work <br /> a system amenable to genetic, molecular, and cell bi-          Enzymes used in recombinant DNA experiments were from New <br /> ological analysis. One such aspect is nuclear dimor-           England Biolabs (Mississauga, Canada) and were used according to <br /> phism, a characteristic of the ciliates. Two functionally      the supplier&apos;s directions. Transformation, growth of plasmid con- <br /> distinct nuclei, a transcriptionally active polyploid so-      taining cells, phage isolation and screening, and DNA isolation <br /> matic (macro) and a transcriptionally silent diploid           were according to standard procedures as described in Sambrook et <br />                                                                al. (1989). The Tetrahymena cDNA library in AgtlO was a gift from Dr. <br /> germinal (micro) nucleus are present in a single cell.         T. Takemasa (University of Tsukuba, Tsukuba, Japan; Takemasa et <br /> Genetic continuity is maintained through the micro-            al., 1990). <br /> nucleus and extensive sequence rearrangements occur <br /> during the sexual cycle (conjugation) and develop-             Transfornation of Tetrahymena <br /> ment of a new macronucleus (reviewed in Prescott,              Tetrahymena were transformed by using a biolistic transformation <br /> 1994; Yao, 1996). With regard to rDNA metabolism,              protocol (Cassidy-Hanley et al., 1997). The biolistic somatic trans- <br /> Tetrahymena offers a system where a single transcrip-          formation (Gorovsky, Bruns, Cassidy-Hanley, and Bowen, personal <br />  <br /> 1052                                                                                                 Molecular Biology of the Cell <br />                                                                                                   An Unusual Fibrillarin Gene and Protein <br />  <br /> communication) used 0.6 mg of a linear fibrillarin gene construct       LEU2 containing pVT-U vector, YEp5l, or the PGK vector for trans- <br /> containing a NEO cassette (Gaertig et al., 1994; Shen et al., 1995))    formation into yeast as described above. <br /> coated onto 1.5 mg of 1.0-,Lm gold beads. Cells were grown to 2 x <br /> 105 cells/ml, starved overnight, and used for transformation. Two <br /> bombardments were used for each transformation and each trans-          Phylogenetic Analysis <br /> formation was done in duplicate. Paromomycin sulfate (Sigma,            Phylogenies were reconstructed by using neighbor-joining distance <br /> St.Louis, MO; 120 jig/ml) in SPP medium with antibiotic/antimy-         matrix methods (Saitou and Nei, 1987). The phylogenetic trees were <br /> cotic mixture (Gorovsky et al., 1975) was added 4 h after the shoot,    constructed with the LASERGENE software package of DNAstar <br /> and transformants were obtained 4-5 d later as paromomycin-             corporation (http://www.dnastar.com). Alignments were done <br /> resistant clones.                                                       with the MEGALIGN program within LASERGENE, making minor <br />                                                                         adjustments by eye to optimize alignments. <br /> Immunoprecipitation and Western Blotting <br /> Immunoprecipitation of U3-containing RNPs with human sclero-            Fusion Proteins and Antibody Production <br /> derma sera was essentially as described in Baserga et al. (1991). <br /> Immunoprecipitated material and nonprecipitated supernatants            E. coli carrying fusion constructs were induced with isopropyl (-D- <br /> from equal amounts of cell lysates were extracted with phenol/          thiogalactoside, and insoluble expressed fibrillarin fusion protein <br /> chloroform, and the recovered RNA was subjected to PAGE and             was isolated and used to produce antibodies in mice as described <br /> electroblotted to charged nylon membranes for Northern blot hy-         previously (Dobson et al., 1994). <br /> bridization. <br />    Extracts of Escherichia coli induced with isopropyl 3-D-thiogalac- <br /> toside and containing fibrillarin fusion proteins in either pGEX2T      Indirect Immunofluorescence <br /> (Pharmacia) or pET29 (Novagen) and total cell or nucleolar extracts     Cells were harvested, fixed, and incubated with primary antibody <br /> (Higashinakagawa et al., 1992) of Tetrahymena were separated by         diluted at least 1:1000 exactly as described by Wenkert and Allis <br /> SDS-PAGE (Laemmli, 1970). Staining of gels, transfer to nitrocellu-     (1984). Additional staining was with propidium iodide rather than <br /> lose membranes, and visualization with antibodies was as described      ethidium bromide. Secondary antibody was fluorescein isothiocya- <br /> previously (Dobson et al., 1994). Secondary antibody was a 1:3000       nate-conjugated goat anti-mouse. Cells were visualized using a <br /> dilution of goat anti-human or goat anti-mouse conjugated to alka-      Bio-Rad laser scanning confocal microscope. <br /> line phosphatase (Bio-Rad, Richmond, CA) and the positive <br /> polypeptides were visualized with the p-nitro blue tetrazolium-5- <br /> bromo-4chloro-3-idolyl phosphate (Promega, Madison, WI) color <br /> reaction (Sambrook et al., 1989).                                       RESULTS <br />                                                                         Tetrahymena Has Fibrillarin <br /> DNA Sequencing and Polymerase Chain Reaction                            To address the question of whether T. thermophila has <br /> (PCR)                                                                   fibrillarin, we used serum from a number of sclero- <br /> Double-strand plasmid DNA was sequenced by the dideoxynucle-            derma patients (Yang et al., 1989), serum from a non- <br /> otide chain-termination protocol of Sanger et al. (1977) using the      scleroderma individual, and antibody against yeast <br /> Sequenase Version 2 kit (United States Biochemical, Cleveland, OH)      fibrillarin (Noplp; gift of Dr. D. Tollervey) to immu- <br /> and [35S]dATP (Dupont New England Nuclear, Mississauga, On- <br /> tario, Canada) following protocols provided by the supplier. Some       noprecipitate Tetrahymena extracts. Serum 538 is the <br /> sequencing was done by using automated cycle sequencing with            only scleroderma patient serum used that recognizes <br /> dye-labeled terminators at the Core Molecular Biology DNA Se-           nucleoli in mammalian cells (Yang et al., 1989), and it <br /> quencing Facility (York University, Toronto, Ontario, Canada). Both     is virtually monospecific for yeast fibrillarin (Hen- <br /> strands of the Tetrahymena fibrillarin cDNA were sequenced by <br /> using universal forward and reverse primers and oligonucleotides        riquez et al., 1990). This serum also recognizes the <br /> designed according to already determined sequence. Oligonucleo-         peripheral nucleoli in Tetrahymena macronuclei in im- <br /> tides were purchased from Dalton Chemical (Toronto, Ontario,            munofluorescence experiments (unpublished results). <br /> Canada). PCRs were carried out in a Coy thermal cycler using a          Gel electrophoresis and Northern blotting of phenol- <br /> hot-start touchdown protocol (Don et al., 1991) as described previ- <br /> ously (Golding et al., 1994; Tourancheau et al., 1995). Degenerate      extracted extracts immunoprecipitated with serum <br /> primers were designed based on highly conserved regions of S.           538 identifies a 265-nucleotide (nt) RNA species the <br /> cerevisiae, human, and Xenopus fibrillarins (see Figure 3) and con-     size of U3 snoRNA (Orum et al., 1993), when hybrid- <br /> tained EcoRI and BamHI restriction sites for cloning. The upstream      ized with cloned DNA encoding Tetrahymena U3 <br /> fibrillarin primer was GGAATTCGTNTAYGCNGTNGARTTY and                    snoRNA (Figure 1A, lane 3). None of the other human <br /> the downstream primer was CGGATCCRTCNGGYTRNGCNA- <br /> CRTC. The amplified fibrillarin fragment was labeled by random          sera nor   anti-Noplp immunoprecipitates Tetrahymena <br /> priming (Sambrook et al., 1989) and used to screen the Agtl0 library    U3 snoRNA (Figure 1A, lanes 1, 2, and 4-6). Further- <br /> as described previously (Vohra et al., 1992). A fragment from the       more, a protein of approximately 32 kDa is identified <br /> full-length cDNA, from the ATG initiation codon to the TGA ter-         in Tetrahymena whole-cell extracts on Western blots <br /> mination codon, was amplified by PCR using the primers CTGG- <br /> GATCCATGGGTAAAGATTTCAAG (FIBSTART) and CTGG-                            stained with the 538 serum (Figure 1B, lanes 1 and 2). <br /> GATCCTCAAGCTCTGTAAGAACC (FIBSTOP). This was cloned                      These experiments demonstrate that U3 snoRNA in <br /> into pEMBL18 at the BamHI site and the four TAA codons were             Tetrahymena is associated with a fibrillarin-like protein <br /> individually changed to CAA by using oligonucleotide-directed           in extrachromosomal macronuclear nucleoli and sug- <br /> site-specific mutagenesis (Sambrook et al., 1989). The full-length      gests that Tetrahymena fibrillarin plays a role in rRNA <br /> mutated fibrillarin construct was introduced into pGEX2T (Pharma- <br /> cia) at the BamHI site to produce an in-frame fusion with glutathi-     processing similar to that demonstrated in other eu- <br /> one S-transferase (GST). This fragment was also introduced into the     karyotes (Kass et al., 1990). <br />  <br /> Vol. 8,   June   1997                                                                                                                   1053 <br />  E. David et al. <br />  <br /> A                                                                          Cloning and Analysis of the cDNA Encoding <br />          SUPT <br />     1   2 3 4     5   6           1 <br />                                         PELLE T <br />                                       2 3 4           6 <br />                                                                            Fibrillarin <br />                                                                  U3 <br />                                                                            Degenerate upstream and downstream primers based <br />                                                                            on highly conserved amino acid sequences of human, <br />                                                                            yeast, and Xenopus fibrillarins (Figure 3) were used in <br />                                                                            a PCR amplification with T. thermophila macronuclear <br />                                                                            DNA. The 165-bp EcoRI-BamHI product was cloned <br />                                                                            into the plasmid vector pEMBL18 (Dente et al., 1983) <br />                                                                            and propagated in E. coli DH5aF&apos;. DNA sequence <br />                                                                            analysis revealed that the fragment encoded a portion <br /> B                                                                          of a fibrillarin protein. This fragment was used to <br />                                       2   3                                screen a AgtlO cDNA library (Takemasa et al., 1990) <br />                           W; .I                                            and a 1.4-kbp cDNA was isolated. This 1.4-kbp EcoRI <br />                                                                            fragment was subcloned into pUC and pEMBL vec- <br />                                                                            tors, a restriction enzyme map was constructed, and <br />                                                                            the fragment was sequenced on both strands in its <br />                                                   _ <br />  <br />  <br />  <br />  <br />                                                                            entirety. Southern blot hybridization analysis demon- <br />                                                                            strated that a single copy gene encoded the protein <br />                                                                            represented by this cDNA. <br />                                                                              The complete 1439-bp cDNA sequence and the 294- <br />                                                                            amino acid sequence derived from the longest open <br />                                                                            reading frame are presented in Figure 2. The 560-bp 3&apos; <br />                                                                            untranslated region is unusually long for a Tetrahy- <br />                                                                            mena gene (Vohra et al., 1992; Stargell and Gorovsky, <br />                                                                            1994; Heinonen and Pearlman, 1994). Four possible <br /> C                                                                          polyadenylation signals (AATAAA) are present, all of <br />                                                                            which are further upstream of the poly(A) tail than <br />                                                                            usually found in Tetrahymena (Vohra et al., 1992; Hei- <br />                                                                            nonen and Pearlman, 1994). Northern blot hybridiza- <br />                                                                            tion analysis identified a single polyadenylated <br />                                                                            mRNA of approximately 1600 nt. This size is consis- <br />                                                                            tent with the isolated cDNA being full length and the <br />                                                                            mRNA having a poly(A) tail of 50-100 nt. Four TAA <br />                                                                            codons, which in Tetrahymena encode the amino acid <br />                                                                            glutamine (Q), are present. An additional five glu- <br />                                                                            tamines are encoded by CAA, one of the universal <br />                                                                            codons. No TAG codons are found and the translation <br />                                                                            termination codon is TGA as in all known Tetrahymena <br />                                                                            genes (Tourancheau et al., 1995). The G+C content of <br />                                                                            the coding region is 43.4% greater than the 39.5% G+C <br />                                                                            content of 35 other Tetrahymena coding regions <br />                                                                            (Stargell and Gorovsky, 1994). This is due in large part <br /> Figure 1. (A) Northern blot hybridization analysis of RNA isolated         to the unusually high content (48.2%) of glycine (G) <br /> by phenol/chloroform extraction from the supernatant and pellets           residues, which are encoded by G+C-rich codons. The <br /> of Tetrahymena cells immunoprecipitated with various antisera. <br /> Lanes 1, 3, 5, and 6, human scleroderma sera, the serum used in lane       longest open reading frame predicts a protein of Mr <br /> 3 (538) being the only one to recognize human fibrillarin (Yang et al.,    31,359 having approximately 60% identity with a <br /> 1989). Lane 2, human serum from a nonscleroderma individual; lane          number of other known fibrillarins (Table 1). <br /> 4, anti-Noplp serum. U3, U3 snoRNA. Only the serum recognizing               PCR amplification of Tetrahymena genomic DNA <br /> human fibrillarin immunoprecipitates Tetrahymena U3 snoRNA and <br /> the supernatant is correspondingly depleted in this RNA. Anti-             with the FIBSTART and FIBSTOP primers yielded an <br /> Noplp serum does not recognize heterologous material as de- <br /> scribed by others (e.g., Jansen et al., 1991). (B) Western blot analysis <br /> of whole-cell extracts and nucleolar protein from T. thermophila. <br /> Lanes 1 (supernatant) and 2 (pellet), sonicated T. thermophila centri-     Figure 1 (cont). markers in kDa (Mr)d (C) Indirect immunofluores- <br /> fuged at 9000 x g for 10 min, blot stained with human scleroderma          cence and propidium iodide staining of vegetatively growing T. <br /> serum 538; lane 3, protein from purified Tetrahymena nucleoli              thermophila. Immunofluorescence with the primary antibody as in B, <br /> stained with mouse polyclonal antibody against bacterially pro-            lane 3, stains the extrachromosomal nucleoli at the periphery of <br /> duced GST-Tetrahymena fibrillarin fusion protein. Thick arrow, a           macronuclei with no staining of micronuclei. Propidium iodide <br /> protein of Mr approximately 32,000; thin arrows, molecular size            stains both macronuclei and micronuclei. <br />  <br />  <br /> 1054                                                                                                           Molecular Biology of the Cell <br />                                                                                                                        An Unusual Fibrillarin Gene and Protein <br />  <br />              CCAAAACCAAAAAAACTAATTAAATAACATAACTAAAGCAAAAATAAA                    -1 <br />  ATGGGTAAAGATTTCAAGTCAGGTGGTGGCAATGCTGGTGGAAAGCCCTTCAATAAAGGT                    60 <br />                                                                                          Table 1. Sizes and overall amino acid identities of some fibrillarin <br /> M    G   K   D   F   K   S   G   G   G   N A     G   G   K   P   F   N   K   G   20      sequences <br />  CCTGGAGGTCCTGGTGGAAGACCCTTCAATAAGGGTCCTGGTGGTCCCGGTGGTCCTGGT                    120 <br />  P   G   G   P   G   G   R   P   F   N   K   G   P   G   G   P   G   G   P   G   40      A. Size <br />  GGCAAATTCGGTGGTGGCAGACCCGGTGGCCCTGGTGGCAAATTTGGTGCCAAGGGTCCT                    180 <br />  G   K   F   G   G   G   R   P   G   G   P   G   G   K   F   G   A   K   G   P   60                Protein                     Residues (no.)           Mr <br />  AGAGGTCCCAAGACTATTATTGTTAAGCATAGATTAGAAGGTGTITTCATCTGCAAAGGT                    240 <br />  R   G   P   K   T   I   I   V   K   H   R   L   E   G   V   F   I   C   K   G   80        Yeast fibrillarin (NOP1)                  327               34,470 <br />  CAATAAGAAGCTTTGGTAACTAAAAATTTCTTCCCTGGTGAATCTGTTTACAATGAAAAA                    300       Xenopus fibrillarin                       323               34,330 <br />  Q   Q   E   A   L   V   T   K   N   F   F   P   G   E   S   V   Y   N   E   K   100       Human fibrillarin                         321               33,823 <br />  CGTATGTCCGTTGAAGAAAACGGTGAAAAGATTGAATATCGTGTCTGGAATCCCTACAGA                    360       Tetrahymena fibrillarin                   292               31,359 <br />  R   M   S   V   E   E   N   G   E   K   I   E   Y   R   V   W   N   P   Y   R   120 <br />  TCTAAAATTGCTGCTGCTGTTGTTrGTGGTATTTCTGATATTCATATTAAACCTGGCTCC                    420     B. Amino acid alignments                                  % Identity <br />  S   K   I   A   A   A   V   V   G   G   I   S   D   I   H   I   K   P   G   S   140 <br />                      V <br />  AAGGTTCTCTATTTGGGTGGTGCCTCTGGTACAACTGTCTCTCACGTTGCTGATATTGTT                    480       Human vs. Xenopus                                          81 <br />  K   V   L   Y   L   G   G   A   S   G   T   T   V   S   H   V   A   D   I   V   160 <br />                                                                                            Human vs. yeast                                            67 <br />  GGACCCACTGGTGTTGTCTATGCTGTCGAATCTCTCACAGAAGTGGTAGAGATTTGGTA                     540       Tetrahymena vs. Xenopus                                    65 <br />  G   P   T   G   V   V   Y   A   V   E   F   S   H   R   S   G   R   D   L   V   180 <br />                                                                                            Tetrahymena vs. human                                      63 <br />  AATATGGCTAAGAAGAGAACCAATGTTGTTCCCATTATCGGTGATGCTAGAAAGCCCCAA                    600       Tetrahymena vs. yeast                                      59 <br />  N  M A K K R T N V V P I I G D A R K P Q                                        200 <br />                                                                                            Tetrahymena vs Xenopus (C-term)a                           71 <br />  GAATACAGATTCTTAGTCGGTATGGTCGATGTCGTCTTCGCTGATGTTGCCCAACCTGAT                    660 <br />  E Y R F L V G N V D V V F A D V A Q P D                                         220       Tetrahymena vs. human (C-term)                             69 <br />                                                                                            Tetrahymena vs. yeast (C-term)                             64 <br />  CA1.GCTAGAATTATGGGTATGAACTGTTAATACTTCTTAAAGAATGGTGGTCACTTCCTT                   720 <br />  Q A R I M G M N C Q Y F L K N G G H F L                                         240 <br />  ATCTCCATTAAGGCTTGCTGTATTGATTCAACCAACGAACCTGCAGTCGTTTTCGCTGCT                    780 <br />                                                                                          aC-term represents approximately 230 C-terminal amino acids of the <br />  I   S   I   K   A   C   C   I   D   S   T   N   E   P   A   V   V   F   A   A   260     Tetrahymena protein and approximately 240 C-terminal amino acids <br />  GAAGTACAAAAATTAAAAGAAGAAGGTCTCAAGCCTGAATAATAACTTACTCTTGAACCT                    840 <br />                                                                                          of the other fibrillarins. Data are from Figure 3. <br />  E V Q K L K E E G L K P E Q Q L T L E P                                         280 <br />  TATGAAAGAGATCACGCTATGGTCCTCGGTTCTTACAGAGCTTGATAGATACTTCTTTAT                    900 <br />  Y   E   R   D   H   A   M   V   L   G   S   Y   R   A   *                       294 <br />                                                                                          served over the C-terminal approximately two-thirds <br />  TAGTTTTCTAGCGTAAAAGTGAAAATCGTCACCACAGCAGATTCCTGATAGTTTIGGATA                    1020    of the protein. For example, the C-terminal 230 amino <br />  ATAAAAATTAACAAACAATAAATTAGCAGAATTAATAAATCGAAGTTAGTAATTACAAGA                    1080    acids of the Tetrahymena protein are 65-70% identical <br />  AAAGTAGAATCAATCACAAGCTAAATATGAAATTTTATCAGTTTTAAAGACAATAAAATC                    1140    to comparable regions of other fibrillarins (Table 1). <br />  AAl TAGTCCTAGTACACAAATTTTAATCAAGTGAAGCAAACAGATAAAATTCAC ACACTA                  1200 <br />                                                                                          The Tetrahymena cDNA we have isolated clearly en- <br />  ACAAAAATTTGATTTTACCAGAAACCATGTGATTAATGCACACAGATATTCCTGTAATTA                    1260 <br />                                                                                          codes a fibrillarin or fibrillarin-like protein. The N- <br />  TAGCAATAAAATTATATATCCTACATACATAAAAAAAATTGCTGAAATAAAATAAATAAT                    1320 <br />                                                                                          terminal region of the Tetrahymena protein differs quite <br />                                                                                          extensively, however, from the comparable region in <br />  TAATTAAITTTTTAGAAAATTATAATATTATATATGATATGATGACTATCGATACATACAT                    1380 <br />                                                                                          other eukaryotic fibrillarins (Figure 3). This region <br />  TTGCTATCTGTTTATTGTAACTTAATT&#194;&#182;ATTATTATCTCTGCTGACTGATGAAGAGGCAGA                    1440   normally consists of an approximately 85-amino acid <br />  TTTTTCCA                                                                         1448   domain rich in glycine (G) and arginine (R) residues <br />                                                                                          with a moderately repetitive structure (GAR domain). <br /> Figure 2. Nucleotide and amino acid sequence of the fibrillarin                          This domain generally contains few proline (P) resi- <br /> cDNA from T. thermophila. TAA triplets encoding glutamine and                            dues but is bounded by conserved N- and C-terminal <br /> four putative polyadenylation signals (AATAAA) are designated. <br /> The TGA stop is designated with *. V, position of the intron in                          proline residues, the C-terminal proline as part of a <br /> genomic DNA. The <span id='am-1' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> for the Tetrahymena                            highly conserved PH dipeptide. The T. thermophila <br /> fibrillarin cDNA sequence is X77962.                                                     N-terminal domain is, however, only 64 amino acids, <br />                                                                                          is not bounded by prolines, but contains many inter- <br />                                                                                          nal proline residues (17% P). The N-terminal region in <br /> approximately 1700-bp fragment. This genomic frag-                                       Tetrahymena also contains only three arginines. How- <br /> ment was cloned into pUC18, and DNA sequence                                             ever, because of the presence of nine lysines in this <br /> analysis showed that it contained the entire fibrillarin                                 region, the total percentage of basic amino acids <br /> coding sequence interrupted by an approximately                                          (R+K) is approximately the same as in N-terminal <br /> 800-bp intron between the first and second nucleotides                                   domains of other fibrillarins where arginine predom- <br /> of codon 146 (G; Figure 2).                                                              inates as the basic amino acid (Table 2). Although the <br />                                                                                          overall hydrophobicity index of the Tetrahymena pro- <br />                                                                                          tein is -29.30, the N-terminal region is very hydro- <br /> The N-Terminal Region of Tetrahymena Fibrillarin                                         philic and shows little sequence repetition as illus- <br /> Is Unusual; It Is Not a GAR Domain                                                       trated in the hydrophobicity plot (Figure 4). The <br /> Comparative analysis of the amino acid sequences of                                      hydrophilic nature of this N-terminal region resem- <br /> 10 fibrillarin or fibrillarin-like proteins is presented in                              bles that for other fibrillarins (Henriquez et al., 1990) <br /> Figure 3. The amino acid sequences are quite con-                                        despite the lack of similarity in amino acid sequence. <br />  <br /> Vol. 8, June 1997                                                                                                                                          1055 <br />  E. David et al. <br />  <br /> H. <br /> M. <br />      sapiens <br />      musculus <br />                    MK-PGFSPRGG--GF--GGRGGFGDRGG-RGG-----RGGFGGGRGRGGGFRGRGRGG-- <br />                    MK-PGFRPRGG--GF--GGRGGFGDRGG-RGGGRGG-RGGFGGGRGGFGG-GGRGRGG-- <br />                                                                                                         Phylogenetic Reconstructions <br /> S.   cerevisiae    M---SFRP-GSRGGSRGGSRGGFGGRGGSRGGARGGSRGGFGGRGGSRGG----ARGGSR <br /> S. <br /> X. <br />      pombe <br />      laevis <br />                    xx-MAYTP-GSRGG-RGGSRGG------------    RGGFNGGRG---GFGG-GRGGAR <br />                  MR-PGFSPRGGRGGF--GDRGGFGGRGG-FGD-----RGGFRG--GSRGGFGGRGRGGDR <br />                                                                                                         The data in Figure 3 and Table 2 illustrate that the <br /> T. <br /> G. <br />      thermophila xMGKDFKSGGGNAGGKP---FNKGPGGPGGRP----FN-----------KGPGGPG <br />      lamblia       M-GTDYRNSGR------GGRDGPGGRGPGN-DRRDSGRS-FGDRRPERPDFK-RGDGGRG                         N-terminal region of Tetrahymena fibrillarin is very <br />                    M------------------RGGFGRGGGGRGGSRGG-RGGF-------------GROGGR <br /> L. <br /> M. <br />      major <br />      vannielii     ME-------------KIKVKEIFSNV .-------------YGVD----------------                        different from the similar region (GAR domain) in <br /> M.   voltae        MA-------------KIKEK--FDNV--------------FELD---------------- <br />                                                                                                         other eukaryotic fibrillarins. To confirm and extend <br /> H. <br /> M. <br />      sapiens <br />      musculus <br />                    -GGGG--GGGGGGGRGGGGFHSGGNRGRG--RGGKRGNQSGKI EVMEPHR-HEGVFICRG <br />                    -GGGGFRGRGGGGVRGGG-FQSGGNRGRGGGRGGKRGNQSGKNVMVEPHR-HEGVFICRG <br />                                                                                                         these observations, we have reconstructed phylog- <br /> S. <br /> S. <br />      cerevisiae <br />      pombe <br />                    GGFGG-RGGSRGGARGGS ------RGGRGGAAGGARG---GAKVVIEPHR-HAGVYIAR- <br />                    GG-------GRGGARGGRGGRGGARGGRGGSSGGRGGAKGGAKVIIEPHR-HAGVFIARG                         enies using fibrillarin amino acid sequences. One data <br /> X. <br /> T. <br />      laevis <br />      thermophila <br />                    GGRGGFRGGFSSPGRGGP--RGGGRGGFGGGRGG---FGAGRKVIVEPHR-HEGIFICRG <br />                    GPGGKFGGG----------- RPGGPGGKFGAKGPR ------GPKTI IVKHRL-EGVFICKG                     set (Figure 5A) uses N-terminal sequence as defined in <br /> G.   lamblia       FGDRRGSGPPGGPDRGDRRGPRDGPGGRGGPGGPGGGFKGGAKTMVKPHP-KDGIFISHG <br /> L. <br /> M. <br />      major <br />      vannielii <br />                    GGG---RGGGRGGGRGG---------GRGGGRGGGRG-GAGAKVIVEPHMLHPGVFISKA <br />                    --------------------------------------------------LGDGL----- <br />                                                                                                         Figure 3, from the first amino acid to the conserved PH <br /> M.   voltae        --------------------------------------------------LGDGI-----                         in the eukaryotic fibrillarins, KH in the Tetrahymena <br /> H.   sapiens       KEDALVTKNLVPGESVYGEKRVSISEGDD.-------KFEYRAWNPFRSKLAAAILGGVDQ <br />                                                                                                         sequence. The archaebacterial fibrillarin-like proteins <br /> M. <br /> S. <br />      musculus <br />      cerevisiae <br />                  KEDALFTKNLVPGESVYGEKRVSISEGDD-------TIEYRAWNPFRSKLAAAILGGVDQ <br />                  KEDLLVTKNMAPGESVYGEKRISVEEPSKEDGVPPTKVEYRVWNPFRSKLAAGIMGGLDE                           lack a similar region and are not included in this <br /> S. <br /> X. <br />      pombe <br />      laevis <br />                  KEDLLVTRNLVPGESVYNEKRISVDSPDG------TKVEYRVWNPFRSKLAAGILGGLDN <br />                  KEDALVTKNLVPGESVYGEKRISVEDGEV-------KTEYRAWNPFRSKIAAAILGGVKQ                           reconstruction. Figure 5B presents a phylogeny using <br /> T.   thermophila QQEALVTKNFFPGESVYNEKRMSVEENGE-------KIEYRVIiNPYRSKIAAAVVGGISD <br /> G. <br /> L. <br />      lamblia <br />      major <br />                  RGDVLVTKSLAPGVAVYGEKRISVEGTES-------KIEYREWNPFRSKLGAAVRLNVLD <br />                  KTDSLCTLNMVPGISVYGEKRIELGATQGGDD----KKEYRLIJNPYRSKLASAIYAGVSS <br />                                                                                                         the C-terminal approximately 230 amino acids of the <br /> M. <br /> M. <br />      vannielii <br />      voltae <br />                  --KRIATKSLAPGKRVYGEKLIYSENK-----------EYRIWNPNKSKLGAAIINGLKK <br />                  --KRIGTKSLVPNKRVYGEKLVNVKNT-----------EYRVWWNPNYSKLGASIINGLKE <br />                                                                                                         eukaryotic proteins starting at the conserved PH (KH) <br />                                   0* <br />                                    00*00                        sacece~*   .   *a*..   . <br />                                                                                                         and the entire archaebacterial sequence. The data pre- <br /> H. <br /> M. <br />      sapiens <br />      musculus <br />                    IHIKPGAKVLYLGAASGTTVSHVSDIVGPDGLVYAVEFSHRSGRDLINLAKKRTNIIPVI <br />                 IHIKPGAKVLYLGAASGTTVSHVSDIVGPDGLVYAVEFSHRSGRDLINLAKKRTNIIPVI                            sented were obtained with a neighbor-joining distance <br /> S. <br /> S. <br />      cerevisiae <br />      pombe <br />                 LFIAPGKKVLYLGAASGTSVSHVSDVVGPEGVVYAVEFSHRPGRELISMAKKRPNIIPII <br />                 IYIKPGARVLYLGAANGTSVSHVADVVGPEGLVYAVEFSHRSGRDLLNMAKKRTNVIPIV                            matrix method (Saitou and Nei, 1987). Maximum par- <br />     laevis      IHIKPGVKVLYLGAASGTTVSHVSDVVGPEGLVYAVEFSHRSG;RDLINVAKKRTNIIPVI <br /> X. <br /> T.  thermophila IHIKPGSKVLYLGGASGTTVSHVADIVGPTGVVYAVEFSHRSGRDLVNMAKKRTNVVPII                            simony analysis produced similar phylogenies to <br /> G.  lamblia     MPIKPGAKVLYLGAASGTTVSHVSDIVGPTGAVYAVEFSQRSGRDLLEVAKARTNVYPII <br /> L.  major       IHMGPGSKVLYLGGATGTTVSHVSDLVGPEGMVYAVEFSHRVGRDLVEMSKRRP-IVPIV <br /> M1. vannielii MPIKKGTKVLYLGASAGTTPSHVSDI-SEDTIVYAVEFAPRIMREFIDSCNERINLIPIL <br />                                                                                                         those in Figure 5 and statistical analyses supported the <br /> M. voltae       MPIKKGSKVLYLGASAGTTPSHVADV-AEDSPVYAVEFAPRIMREFIESCEGRKNLFPIL <br />                         a   .aa     *a   ***a...a* a   0.*0**     .   *.a <br />                                                                                                         reconstructions presented. <br />                                                                                        a .*** ....**...*.* <br />  <br /> H. sapiens         EDARHPHKYRMLIAMVDVIFADVAQPDQTRIVALNAHTFLRNGGHFVISIKANCIDSTAS <br /> M. musculus        EDARHPHKYRMLIAMVDVIFADVAQPDQTRIVALNAHTFLRNGGHFVISIKANCIDSTAS <br /> S. cerevisiae      EDARHPQKYRMLIGMVDCVFADVAQPDQARIIALNSHMFLKDQGGVVISIKANCIDSTVD <br /> S. pombe <br /> X. laevis <br />                    EDARHVQKYRMLVGMVDVVFADVAQPDQARIVALNAAAFLKNEGGVVISVKASCIDSTAD <br />                    EDARHPHKYRILVGMVDVVFADVAQPDNTRIVALNAHNFLKNGGHFVISIKANCIDSTAA                         Fibrillarin Is Essential in Tetrahymena <br /> T. thermophila     GDARKPQEYRFLVGMVDVVFADVAQPDQARIMGMNCQYFLKNGGHFLISIKACCIDSTNE <br /> G. lamblia <br /> L. maj or <br />                    ADARHPYKYRMIVPEVDCIFSDVAQPDQARIVAENARYYLKANGGMLISIKASSVDSTLK <br />                    EDARYPMKYRMLVPMVDCIFMDVAQPDQARILALNAQAFLQNGGHYVISIKANCIDSTLE <br />                                                                                                         An EcoRV-Smal fragment of p4T2-1 (Gaertig et al., <br /> M. vannielii <br /> M1. voltae <br />                    GDANRPFEYSNIVGKVDVIFEDVAQPNQAEILVKNAKWFLNKDGYAMISIKARSIDVTKN <br />                    GDANKPEEYANIVEKVDVIFEDVAQPNQAEILIKNAKWFLKKGGYGMISIKARSVDVTEN                         1994; Shen et al., 1995) that contains a NEO gene ca- <br />      H. a.A <br />         *V <br />         *a 00.0N <br />            . *. E *          .P.K*K* <br />                           ,V**,0** <br />                     ,,P *,R* <br />                                                                                                         pable of conferring paromomycin resistance in Tetra- <br /> H.   sapiuens <br /> 14. musculus <br />                    AEAVFASEVKKE4QQENMKPQEQLTLEPYERDHAVVVGVYRPPPKVKN <br />                    AEAVFASEVKKMIQQENMKPQEQLTLEPYERDMAVVVGVYRPPPKVKM                                     hymena was introduced into the genomic fibrillarin <br /> S.   cerevisiae    AETVFAREVQKLREERIKPLEQLTLEPYERDHCIVVGRYMRSG-LKK <br /> S. <br /> X. <br />      pombe <br />      laevis <br />                    AAVVFAREVKKMQE --------------------------------- <br />                PEAVFAAEVKKMQQENMKPQEQLTLEPYERDHAVVVGIYRPPPKQKI <br />                                                                                                         construct at the blunt-ended BglII restriction site (nt <br /> T. thermophila PAVVFAAEVQKLKEEGLKPEQQLTLEPYERDHAMVLGSYRA ----- <br /> G. lamblia     PEAVFAREIETLREHDFKCKEQLDIGEFHRNHAIVVGRFRVKA---A <br />                                                                                                         358, Figure 2). Digestion with BamHI yielded a frag- <br /> L. major <br /> M. vannielii <br />                APVVIASELNKLKKDKLKPLEQVSLEPFERDHAVVVGVYRPVKKSKQ <br />                PKEIFLEQKKILIEGGFDIVDEINIEPFEKDHIMFVGIWKG-----N <br />                                                                                                         ment with 360 bp of Tetrahymena sequence 5&apos; and <br /> M. voltae      PRVIFEAQKEIMEQNGFKIVDAINIEPFEKDHMLFVGIWNG-----Q                                          approximately 1325 bp 3&apos; of the NEO cassette. After <br />                                                                                                         introduction into Tetrahymena, 12 transformants resis- <br /> Figure 3. Clustal (Higgins and Sharp, 1989) alignment of the                                            tant to 120  jig/ml paromomycin were analyzed. Each <br /> amino acid sequences of 10 fibrillarin proteins. Accession numbers <br /> for the sequences are as follows: Homo sapiens, X56597; Mus muscu-                                      transformant was grown for at least 100 doublings in <br /> lus, Z22593; S. cerevisiae, J05230 and X51676; S. pombe, X69930; Xe-                                    the highest concentration of paromomycin that an in- <br /> nopus laevis, M28874; T. thermophila, X77962; G. lamblia, L28115;                                       dividual transformant could tolerate, which ranged <br /> Leishmania major, L26252; Methanococcus vanielli, X73987; and Meth-                                     from 400 ,tg/ml to &gt;3 mg/ml. After growth in a high <br /> anococcus voltae, X73988. The conserved PH residues (except T. <br /> thermophila, KH) and upstream conserved K defining the C terminus                                       concentration of paromomycin, transformants were <br /> of the glycine-rich domain and the mostly conserved P residue                                           maintained in 250 jig/ml paromomycin. PCR analysis <br /> defining the N terminus of this domain are highlighted as are the                                       of genomic DNA from all of the transformants using <br /> glutamines encoded by TAA in T. thermophila and the highly con-                                         the FIBSTART and FIBSTOP primers showed an ap- <br /> served VYAVEF and FADVAQ sequences to which primers were <br /> designed for the PCR amplification of the T. thermophila sequence.                                      proximately 1700-bp fibrillarin fragment and an ap- <br />                                                                                                         proximately 3100-bp disrupted fibrillarin fragment <br />                                                                                                         present in all transformants. The disrupted fibrillarin <br /> Furthermore, although the N-terminal region of the                                                      was present at the same locus as the undisrupted <br /> Tetrahymena protein does show very limited sequence                                                     fibrillarin confirming homologous recombination oc- <br /> repetition, the sequence of the repeats differs mark-                                                   curred. There is no obvious <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> other than <br /> edly from the consensus repeat sequence of other                                                        paromomycin resistance of the transformants. The in- <br /> fibrillarins (Henriquez et al., 1990; Russell and Toll-                                                 ability to eliminate the undisrupted fibrillarin genes <br /> ervey, 1992). Comparison of the predicted secondary                                                     by phenotypic assortment even after extensive growth <br /> structures of Tetrahymena, S. cerevisiae, and human                                                     in high concentrations of paromomycin demonstrates <br /> fibrillarins show extensive differences not only in the                                                 that fibrillarin is essential in T. thermophila, as it is in S. <br /> N-terminal region but throughout the protein.                                                           cerevisiae (Schimmang et al., 1989). <br />  <br /> 1056                                                                                                                                       Molecular Biology of the Cell <br />                                                                                                                                An Unusual Fibrillarin Gene and Protein <br />  <br />  <br /> Table 2. Glycine, proline, and basic amino acid (arginine and lysine) composition of the N-terminal domain of eight eukaryotic fibrillarins <br />                                                                              GAR domain <br /> Source                  Total aa            No.a/%c G            No./% R                            No. / 7cK                                    No. 1% R+ K                                          No. / c P <br /> H. sapienis                84                 48/57                14/17                                 2/2                                             16/19                                             2/2 <br /> M. musculus                90                 51/57                16/18                                 2/2                                             18/20                                             2/2 <br /> X. laevis                  86                 44/51                17/20                                 1/1                                             18/21                                             4/5 <br /> S. cerevisiae              86                 43/50                17/20                                 1/1                                             18/21                                             2/2 <br /> S. pombe                   73                 39/53                14/19                                 2/3                                             16/22                                             2/3 <br /> L. major                   62                 38/61                13/21                                 1/2                                             14/23                                             1/2 <br /> T. thermophila             64                 27/42                 3/5                                  9/14                                            12/19                                            11/17 <br /> G. lamblia                 93                 35/38                19/20                                 3/3                                             22/24                                            11/12 <br /> See Figure 3 for a <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of the limits of this domain. <br /> aNumber of residues of the designated amino acid in the domain; 7c, percentage of that amino acid in the domain. <br />  <br />  <br />  <br /> Antibodies Generated against a Fibrillarin Fusion                      onto YEPD plates and gave two viable spores. These <br /> Protein Produced in E. coli Recognize Tetrahymena                      were replicated to selective medium and no Ura+ <br /> Fibrillarin                                                            haploids were found, indicating that the plasmid <br /> To confirm that the cDNA we cloned and analyzed                        borne Tetrahymena fibrillarin could not functionally <br /> encodes fibrillarin, we prepared a polyclonal antibody                 complement NOP1. Western blot analysis (Figure 6) <br /> in mouse against a GST fusion protein of the full-                     indicates that Tetrahymena fibrillarin was produced <br /> length cDNA with all four TAA codons changed to                        but was not able to complement the NOP1 disruption <br /> CAA to allow expression in E. coli. A Western blot of                  mutant. The antibody used in these experiments was a <br /> a Tetrahymena nucleolar extract probed with this anti-                 mouse polyclonal antiserum against a C-terminal frag- <br /> body identifies an approximately 32-kDa polypeptide                    ment (from BglII at nt 358 to the termination codon of <br /> (Figure 1B, lane 3), the same size as that identified by               the cDNA, Figure 2) of bacterial-expressed Tetrahy- <br /> scleroderma serum 538 (Figure 1B, lane 1). Indirect                    mena fibrillarin. This antibody does not recognize <br /> immunofluorescence labeling of vegetative cells with                   yeast Noplp (Figure 6, lane 1). A Leu+ Ura- spore <br /> the same antibody identifies the extrachromosomal                      from the above dissection grown under selective con- <br /> nulceoli at the periphery of macronuclei (Nilsson and                  ditions expresses Tetrahymena fibrillarin (Figure 6, lane <br /> Leick, 1970) with no labeling of micronuclei (Figure                   2). Leu+ Ura+ transformants of the Y242 diploid yeast <br /> 1C).                                                                   also express Tetrahymena fibrillarin (Figure 6, lanes 3 <br />                                                                        and 5). <br />                                                                           To test whether the inability to functionally comple- <br /> Expression and Function of Tetrahymena Fibrillarin                     ment is due to the difference in the N-terminal do- <br /> Despite Tetrahymena fibrillarin being quite distinct in                mains between the Tetrahymena and S. cerevisiae fibril- <br /> primary and secondary structure from other known                       larins, we constructed a chimeric gene encoding the <br /> fibrillarins and having an unusual N-terminal domain,                  N-terminal yeast GAR domain and the C-terminal <br /> it appears to be functionally similar to other eukary-                 two-thirds of Tetrahymena fibrillarin. Experiments as <br /> otic fibrillarins. It is associated with U3 snoRNA and is              described above for the full-length Tetrahymnena fibril- <br /> restricted to macronuclei (Figure 1C), and its localiza- <br /> tion during nuclear development correlates with nu- <br /> cleolar structure and active rDNA transcription (Dav-                   lndex <br />                                                                            2. ............................. ........................ ...................... ...........................   ............................. <br /> id and Pearlman, unpublished data). To further <br /> address questions about fibrillarin structure and func- <br /> tion, we asked whether Tetrahymena fibrillarin could <br /> function in S. cerevisiae. Fibrillarin in S. cerevisiae is <br /> encoded by an essential gene called NOP1. A 2- tm-                                                         Ai~~~~~~~~~I          ~ <br /> based plasmid with the Tetrahymena fibrillarin cDNA                                                                            120                         18I                        240                           300 <br /> encoding the full-length protein, the LEU2 gene, and <br /> either the constitutive ADH or PGK promoters or the                    Figure 4. Hydropathy plot for the 294-amino acid fibrillarin pro- <br /> inducible GAL promoter was transformed into the                        tein from T. thermophila obtained with the PROSIS program. The <br />                                                                        hydrophilic N-terminal 64-amino acid domain is clearly visible but <br /> diploid yeast strain Y242. Leu+ Ura+ transformants                     distinct repeating units in this domain are not present as they are for <br /> were selected and sporulated. Tetrads were dissected                   the yeast fibrillarin (Nopl) protein (Henriquez et al., 1990). <br />  <br /> Vol. 8, June 1997                                                                                                                                                                                                1057 <br />  E. David et al. <br />  <br /> A                                                                          H. sapiens <br />                                              u <br />                                              0 <br />                                                           -L~M. <br />                                                             [       - musculus <br />                                                               ~~~~~~X. <br />                                                                    laevis <br />                                                                                                      Figure 5. Phylogenetic reconstructions <br />                                                       I              S. cerevisiae                   using neighbor-joining distance matrix <br />                                                                    S. pombe             G. lamblia   methods for fibrillarin amino acid se- <br />                                 T. thernophila                                                       quence data. The data used in the recon- <br />                                                                                                      structions are those presented in Figure <br />                                                                                                      3. (A) Phylogeny for the N-terminal por- <br />                                                                                                      tion of eukaryotic fibrillarins as defined <br /> B                                                                                                    in Figure 3. (B) Phylogeny reconstruc- <br />                                                                                                      tion using data for the remaining ap- <br />                                                                                                      proximately 230 C-terminal amino acids <br />                                                                                                      of eukaryotic fibrillarins. Included in this <br />                                                                                                      C-terminal analysis are complete se- <br />                                                                                                      quences for two archaebacterial fibrilla- <br />                                                                                                      rin-like proteins that do not have an N- <br />                                                                                                      terminal region similar to that in the <br />                                                                                                      eukaryotic fibrillarins. <br />  <br />  <br /> larin were carried out with similar results. With all                         et al., 1991). Of additional interest is that the Tetrahy- <br /> ADH, PGK, and GAL promoter plasmid constructs,                                mena cDNAs have the same nucleotides as does S. <br /> tetrads obtained on transformation and sporulation of                         cerevisiae at positions that were suggested to be impor- <br /> Y242 gave two viable spores and no Ura+ haploids                              tant for fibrillarin function as determined from the <br /> were obtained. Western blots (Figure 6, lanes 4 and 6)                        analysis of intragenic suppressors of the human and <br /> showed that the diploid Leu+ transformants produced                           Xenopus fibrillarin cDNAs expressed in S. cerevisiae <br /> the chimeric protein that is larger than the full-length                      (Jansen et al., 1991). Furthermore, although specific <br /> Tetrahymena protein. The inability of Tetrahymena                             amino acid residues in yeast Noplp thought to be <br /> fibrillarin to complement a yeast NOP1 disruption                             associated with the synthesis and maturation of ribo- <br /> strain is in contrast to the ability of human and Xeno-                       somes that show strong conservation among the <br /> pus fibrillarins to complement this yeast strain (ansen                       mouse, human, and Xenopus protein sequences (Tur- <br />                                                                               ley et al., 1993) are somewhat less conserved in the <br />                                                                               Tetrahymena protein, the conservation is still quite <br />        a      2      3         4        v         6                           high. <br />  <br />                                                                               DISCUSSION <br />                                                                               rDNA in T. thermophila has a distinctive structure and <br />                                                                4              organization in the genome, very different from that <br />                                                                               found in most eukaryotes. The single-copy rRNA gene <br />                                                                               on chromosome 2r of the germinal micronucleus is <br />                                                                               excised and amplified to approximately 9000 copies of <br />                                                                               an extrachromosomal palindromic rDNA dimer dur- <br />                                                                               ing macronuclear development (reviewed in Engberg, <br />                                                                               1985). It is this molecule that is transcribed at a high <br />                                                                               rate in extrachromosomal macronuclear nucleoli to <br />                                                                               provide the rRNA required for ribosome biogenesis in <br /> Figure 6. Westem blot analysis of protein from untransformed or               vegetative cells. Tetrahymena nucleoli differ from most <br /> transformed S. cerevisiae stained with mouse polyclonal antibody              eukaryotic nucleoli in a number of ways, one being <br /> against a C-terminal fragment of bacterially produced Tetrahymena             that their DNA is homogeneous containing a single <br /> fibrillarin. Lane 1, haploid untransformed S. cerevisiae; lane 2, Leu+        extrachromosomal gene. <br /> Ura- spore from Y242 transformed with a plasmid containing full-                We have shown that T. thermophila nucleoli have a <br /> length Tetrahymena fibrillarin cDNA expressed from the ADH pro- <br /> moter; lane 3, Y242 transformed with the same plasmid as in lane 2;           fibrillarin-like protein that is associated with U3 <br /> lane 4, Y242 transformed with a plasmid containing a chimeric                 snoRNA. To initiate molecular studies on structure/ <br /> fibrillarin cDNA, the N-terminal region from S. cerevisiae (GAR               function relationships in these distinctive nucleoli, we <br /> domain) and the remainder Tetrahymena sequences, expressed from               have characterized the cDNA encoding fibrillarin <br /> an ADH promoter; lane 5, as in lane 3 except that the cDNA was                from T. thermophila and have demonstrated that fibril- <br /> expressed from the PGK promoter; lane 6, as in lane 4 except that <br /> the cDNA was expressed from the PGK promoter. Arrows indicate                 larin in vegetative cells is localized to extrachromo- <br /> molecular size markers in kDa (Mr)                                            somal macronuclear nucleoli. Fibrillarin in Tetrahy- <br />  <br /> 1058                                                                                                            Molecular Biology of the Cell <br />                                                                                    An Unusual Fibrillarin Gene and Protein <br />  <br /> mena is associated with nuclei actively transcribing        (Ghisolfi et al., 1992). The only other eukaryotic fibril- <br /> rDNA, suggesting it has a function similar to that of       larin having an N-terminal domain rich in proline is <br /> fibrillarins in other eukaryotes.                           that from the primitive eukaryote Giardia lamblia, but <br />    Even though Tetrahymena fibrillarin is similar to        the sequence of repeats in this region is characteristic <br /> other eukaryotic fibrillarins in that it is localized to    of that in other eukaryotic GAR domains, unlike the <br /> nucleoli and is associated with snoRNAs, it differs in      sequence of the N-terminal region of Tetrahymena <br /> primary and secondary structure from other fibrilla-        fibrillarin. Arginine residues in GAR domains are <br /> rins. It does not have an N-terminal region rich in         highly modified. The almost complete absence of ar- <br /> glycine and arginine residues bounded by highly con-        ginine residues in the N-terminal domain of Tetrahy- <br /> served proline residues (GAR domain), characteristic        mena fibrillarin represents a significant difference with <br /> of other eukaryotic fibrillarins. GAR domains are           other eukaryotic fibrillarins, although the presence of <br /> found in a number of nucleolar proteins (Ochs et al.,       many basic lysine residues could provide a similar <br /> 1985a; Jong et al., 1987; Lapeyre et al., 1987; Schimmang   overall charge for the region. The absence of methyl- <br /> et al., 1989; Henriquez et al., 1990; Lee et al., 1991;     atable arginine residues may be functionally signifi- <br /> Girard et al., 1992; Russell and Tollervey, 1992) and are   cant (Najbauer et al., 1993), and it is not known <br /> characterized by slightly variable repeating units sim-     whether or not the lysine residues can substitute for <br /> ilar to RGGFRGG or RGGFGGR, where F can be sub-             the arginine residues in the proposed RNA-binding <br /> stituted with S, Y, or A (Russell and Tollervey, 1992).     function. Genes encoding fibrillarin-like proteins have <br /> Most of the arginines in the repeats are dimethylated,      also been reported in two species of archaebacteria. <br /> and dimethylarginine-containing proteins likely inter-      Although fibrillarin-like, the proteins putatively tran- <br /> act with RNA (Najbauer et al., 1993). It appears that       scribed from these genes are unusual in that they do <br /> the secondary or tertiary structure of the GAR domain       not have a GAR domain (Figure 3). Little is known <br /> rather than the primary sequence has been conserved         about structure/function relationships in rDNA tran- <br /> in evolution. Helix-destabilizing proline residues gen-     scription and processing in the domain Archea al- <br /> erally flank GAR domains and structurally separate          though in vitro processing studies in an archaeon <br /> them from the remainder of the protein. GAR domains         (Potter et al., 1995) suggest the presence of a U3 <br /> appear able to function independent of their location       snoRNA homologue. <br /> within proteins and this may be due to the structural          Phylogenetic reconstructions using N-terminal and <br /> isolation described above. The GG dipeptide in GAR          C-terminal amino acid sequence data for eukaryotic <br /> domains is predicted to form (3-turns, yielding a           fibrillarins confirm the unusual nature of the Tetrahy- <br /> highly flexible structure. Recent studies with the iso-     mena N-terminal region. The uncharacteristic position- <br /> lated GAR domain of the vertebrate nucleolar protein        ing of Tetrahymena on the phylogenetic tree con- <br /> nucleolin provide direct evidence for a repeated            structed with N-terminal sequence data reflects a <br /> {3-turn/,8-spiral secondary structure (Ghisolfi et al.,     higher rate of evolution of this region of the molecule <br /> 1992).                                                      compared with the rest of the sequence, which is well <br />    The structurally independent and very basic GAR          conserved compared with other fibrillarins. The mech- <br /> domain is capable of binding RNA and other single-          anism(s) leading to the evolution of this unusual eu- <br /> stranded nucleic acids, and this interaction leads to       karyotic fibrillarin are not presently understood. <br /> unstacking of nucleotide bases and an unfolding of             Tetrahymena fibrillarin cannot substitute for the es- <br /> RNA secondary structure (Ghisolfi et al., 1992). It has     sential yeast Nopl protein even though human and <br /> been postulated that this nonspecific interaction may       Xenopus fibrillarins can, demonstrating functional con- <br /> result in unfolding of regions of rRNA allowing access      sequences associated with the primary and secondary <br /> of a second domain of the GAR protein for specific          structure differences between the Tetrahymena and <br /> RNA/protein interaction. Fibrillarins do contain a          other fibrillarins. Because fibrillarin-like proteins that <br /> conserved putative RNA-binding domain within a se-          appear to be functional in rRNA metabolism and that <br /> quence resembling the RNP consensus sequence (Gi-           lack an N-terminal GAR domain or even an N-termi- <br /> rard et al., 1992), although there is as yet no direct      nal glycine-rich region are present in archaebacteria, a <br /> evidence for sequence-specific RNA binding to this          role for a GAR or glycine-rich domain in nucleolar <br /> region. It has also been suggested that the GAR do-         targeting and/or specific protein/protein interaction <br /> main may be important in protein-protein interactions       may be likely. Replacing the Tetrahymena N-terminal <br /> or for nucleolar targeting (Schimmang et al., 1989;         domain with the S. cerevisiae GAR domain did not <br /> Ghisolfi et al., 1992).                                     produce a construct capable of complementation of <br />    The N-terminal region of Tetrahymena fibrillarin is      NOP1 mutant strains, confirming that there is less <br /> moderately repetitive and rich in glycine residues but      functional conservation among fibrillarins than previ- <br /> it is heavily interspersed with proline residues. This is   ously suggested (Jansen et al., 1991). Perhaps the <br /> likely to result in a structure very different from that    uniqueness among eukaryotes of the single gene con- <br /> demonstrated for the highly conserved GAR domain            taining extrachromosomal nucleoli of Tetrahymena <br />  <br />  <br /> permits the unusual Tetrahymena fibrillarin to be func-                 <br /> tional. Genetic analysis in S. cerevisiae of chimeric                   <br /> yeast/Tetrahymena fibrillarin genes such as isolation of                <br /> intra- and extragenic suppressors as well as screening                <br /> for synthetic lethal mutations should provide impor-                   <br /> tant information about fibrillarin structure and func-                 <br /> tion and about proteins that may interact with fibril-                 <br />                                                                        <br /> larin. These ongoing experiments will complement in                    <br /> vitro rRNA processing experiments, taking advantage                    <br /> of the unusual Tetrahymena fibrillarin and nucleoli in                  <br /> addressing questions about the function of this impor-                <br /> tant protein.                                                          <br />  <br /> </body></html>