<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                               INVESTIGATION <br />  <br />  <br />  <br />  <br />                             Integration New Genes Cellular Networks, <br />                                              Their Structural Maturation <br />                                                                                                                                  Gy&#195;&#182;rgy Abrus&#195;&#161;n1 <br />                        Synthetic Systems Biology <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Institute Biochemistry, Biological Research Centre Hungarian Academy Sciences, <br />                                                                                                                               Szeged H-6701, Hungary <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> It recently discovered new genes originate novo noncoding DNA, biological traits <br /> including expression sequence composition form continuum noncoding sequences conserved genes. In article, using <br /> yeast genes I test integration new genes cellular networks structural maturation shows continuum <br /> by analyzing changes gene age. I 1) The number regulatory, protein&#226;&#8364;&#8220;protein, genetic interactions increases <br /> continuously gene age, different rates. New regulatory interactions emerge rapidly million years, <br /> while number protein&#226;&#8364;&#8220;protein genetic interactions increases slowly, rate 2&#226;&#8364;&#8220;2.25 3 1028/year 4.8 3 1028/year, <br /> respectively. 2) Gene essentiality evolves relatively quickly: youngest essential genes appear proto-genes  14 MY old. 3) In <br /> contrast interactions, secondary structure proteins robustness mutations indicate new genes face bottle- <br /> neck evolution: proto-genes characterized high b-strand content, high aggregation propensity, low robustness <br /> against mutations, conserved genes characterized lower strand content higher stability, likely higher <br /> probability gene loss young genes accumulation neutral mutations. <br />  <br />  <br />  <br /> T   HE established view new genes evolve primarily <br />     duplications recombination, i.e., reorganiza- <br /> tion existing domains genes (Long et al. 2003; <br />                                                                                       close relatives sequenced. However, fact that <br />                                                                                       detected model organisms indi- <br />                                                                                       cates emergence exceptionally rare, and <br /> Kaessmann 2010). Recent studies, however, highlighted                            ongoing process many, most, genomes. <br /> genes emerge noncoding DNA: novo emer-                                  Although novo origination genes questioned <br /> gence genes demonstrated number cases                             anymore, little known frequency sub- <br /> in Drosophila (Levine et al. 2006; Begun et al. 2007; Zhou                            sequent fate genes genome, i.e., they <br /> et al. 2008), humans (Toll-Riera et al. 2009; Knowles                             subject turnover, rapidly new protein&#226;&#8364;&#8220;protein <br /> Mclysaght 2009; Wu et al. 2011; Xie et al. 2012), rodents                             interactions formed (Lynch 2012) integrated into <br /> (Heinen et al. 2009; Murphy Mclysaght 2012), yeast                                regulatory networks (Ding et al. 2010; Chen et al. 2012), <br /> (Cai et al. 2008; Carvunis et al. 2012), viruses (Rancurel                        structural changes affect novel pro- <br /> et al. 2009; Sabath et al. 2012). While orphan genes&#226;&#8364;&#8221;genes                            teins. Real proteins random structures; i.e., they <br /> with homologs&#226;&#8364;&#8221;are common genomes (Tautz                                     characterized distinct biophysical properties like <br /> Domazet-Lo&#203;&#8225;so 2011), proving gene emerged novo                              stability aggregation propensity (DePristo et al. 2005; <br /> is straightforward, presence nonfunctional                            Monsellier Chiti 2007), although, quite surprisingly, <br /> parent sequence demonstrated, presence                           basic features, like presence secondary <br /> can convincingly demonstrated species                           structural elements (a-helices b-strands), already <br />                                                                                       formed random sequences (Schaefer et al. 2010). Ge- <br /> Copyright &#194;&#169; 2013 Genetics Society America <br /> doi: 10.1534/genetics.113.152256                                                      nome-wide studies transcription species indi- <br /> Manuscript received April 15, 2013; accepted publication August 27, 2013;         cate large fraction genome, including the <br /> published Early Online September 20, 2013. <br /> Supporting information available online http://www.genetics.org/lookup/suppl/   noncoding part, transcribed species (Kapranov <br /> doi:10.1534/genetics.113.152256/-/DC1. <br /> 1 <br />                                                                                       et al. 2007; Nagalakshmi et al. 2008; Xu et al. 2009), and <br />  Address correspondence: Synthetic Systems Biology <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Institute of <br />  Biochemistry, Biological Research Centre Hungarian Academy Sciences, <br />                                                                                       noncoding transcripts associated ribo- <br />  Temesv&#195;&#161;ry krt. 62, Szeged, H-6701, Hungary. E-mail: abrusan@brc.hu                   somes occasionally translated (Wilson Masel 2011). <br />  <br />  <br />  <br />                                                                                                 Genetics, Vol. 195, 1407&#226;&#8364;&#8220;1417 December 2013     1407 <br />  A recent large-scale study yeast demonstrated exis- <br /> tence continuous change level expression, <br /> selective constraints, codon adaptation index recently <br /> emerged &#226;&#8364;&#339;proto-genes&#226;&#8364;? highly conserved, ancient genes <br /> (Carvunis et al. 2012), suggested genes placed <br /> in continuum nongenic sequences conserved genes, <br /> and novo emergence genes common emer- <br /> gence classic duplication&#226;&#8364;&#8220;divergence mechanism. This <br /> is supported phylogenetic <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene emer- <br /> gence vertebrates (Neme Tautz 2013), indicating that <br /> young genes typically short <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, number <br /> of exons, domains change gradually time. <br />    This study goals. The &#239;&#172;?rst examine as <br /> broadly possible fate new translatable <br /> open reading frames (ORFs) emergence, i.e., <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> test rapidly integrated regulatory, protein&#226;&#8364;&#8220; <br /> protein, genetic interaction networks, are <br /> stable genome, characteristic <br /> sequence structural changes process func-         Figure 1 A schematic phylogenetic tree fungal species conserva- <br /> tionalization. The second test observed            tion yeast genes (modi&#239;&#172;?ed Carvunis et al. 2012). The main bi- <br /> patterns support theory proposed Carvunis                furcation events Saccharomyces lineage numbered, 0 to <br /> et al. (2012), nongenic sequences well-              10, age estimates obtained using TimeTree (Hedges et al. <br />                                                                      2006), S. castellii, split Saccharomyces lineage <br /> characterized genes form continuum sharp <br />                                                                      100&#226;&#8364;&#8220;150 MYA, whole-genome duplication yeasts (Cliften <br /> boundaries exist them.                                       et al. 2006). Yeast genes classi&#239;&#172;?ed according conservation <br />                                                                      level, corresponds phylogenetic spread orthologs; for <br />                                                                      example, yeast gene conservation level 5 means ortho- <br /> Materials Methods                                                logs S. castellii fungal species split earlier the <br />                                                                      Saccharomyces lineage, conservation level 7 means yeast <br /> Categorization yeast genes age groups                        gene orthologs Debaryomyces hansenii C. albicans. Con- <br />                                                                      servation level 0 marks putative ORFs identi&#239;&#172;?ed Carvunis et al. <br /> Similarly Carvunis et al. (2012), genes Saccharomyces          (2012), excluding sequences shorter 50 aa, conservation level <br /> cerevisiae divided 11 groups according            1 indicates genes annotated Saccharomyces Genome Consortium, <br /> evolutionary conservation fungi. Conservation level           having orthologs species. <br /> 0 corresponds unannotated genes identi&#239;&#172;?ed by <br /> Carvunis et al. (2012) longer 50 amino acids.           org/regev/orthogroups). The orthology data set was <br /> Genes conservation level 1 genes anno-            used determine gene losses S. paradoxus S. mika- <br /> tated genes Saccharomyces Genome Database                  tae (see Results). <br /> (Cherry et al. 2012) orthologs species,            The age estimates main branching events the <br /> and higher conservation levels indicate presence              phylogeny fungi (Figure 1) obtained using TimeTree <br /> orthologs fungi (see Figure 1); i.e., conservation          (Hedges et al. 2006), S. castellii, split <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> level N means orthologs gene present             Saccharomyces lineage 100&#226;&#8364;&#8220;150 MYA, whole <br /> in species branch rooted Nth bifurcation           genome duplication yeasts (Cliften et al. 2006). For nodes <br /> (Figure 1), species split earlier           age estimates estimates unavailable <br /> Saccharomyces lineage. Following Carvunis et al. (2012), I           con&#239;&#172;&#8218;ict phylogeny species. <br /> use term proto-genes genes present in <br />                                                                      Data sources <br /> S. cerevisiae closest relatives (conservation levels 0&#226;&#8364;&#8220;4). <br />     The classi&#239;&#172;?cation genes proto-genes (conservation        The yeast genome protein sequences downloaded <br /> levels 0&#226;&#8364;&#8220;4) taken directly Carvunis et al. (2012);          Saccharomyces Genome Database (Cherry et al. <br /> the sequences proto-genes conservation level 0               2012); recent duplications removed protein <br /> (which absent Saccharomyces Genome Database)              data set clustering 80% amino acid similarity <br /> were obtained using genomic coordinates provided              uclust (Edgar 2010). <br /> the study, using R56 (20070406) release                 The &#239;&#172;?tness effect yeast genes (i.e., effect single- <br /> genome. Only sequences minimum length 50 amino             gene deletion growth) obtained Deutschbauer <br /> acids used analysis. Conserved genes clas-          et al. (2005). Yeast protein&#226;&#8364;&#8220;protein interactions genetic <br /> si&#239;&#172;?ed conservation groups (Figure 1) Carvunis             interactions downloaded BioGRID (v. 3.1.91) <br /> et al. (2012) using fungal gene orthology data set pro-          database (Stark et al. 2011); regulatory interactions were <br /> vided Wapinski et al. (2007; http://www.broadinstitute.           downloaded YEASTRACT database (Abdulrehman <br />  <br />  <br />  <br /> 1408     G. Abrus&#195;&#161;n <br />                                                                               et al. 2011). The strength genetic interactions (genetic <br />                                                                              interaction score, e) obtained Costanzo et al. <br />                                                                              (2010); largest interaction data set &#226;&#8364;&#339;lenient <br />                                                                              cutoff&#226;&#8364;? (P , 0.05) used analysis, it <br />                                                                              includes weak interactions. Each gene charac- <br />                                                                              terized average |e| values signi&#239;&#172;?cant <br />                                                                              genetic interactions, calculated separately for <br />                                                                              proto-genes, conserved genes. The number strength <br />                                                                              protein&#226;&#8364;&#8220;protein, genetic, regulatory interactions for <br />                                                                              gene determined Perl scripts, developed in <br />                                                                              house. (See supporting information, Table S1.) Due their <br />                                                                              low expression low conservation, BioGRID database <br />                                                                              probably signi&#239;&#172;?cantly biased proto-genes I <br />                                                                              included genes analysis interac- <br />                                                                              tions databases, majority genes the <br />                                                                              conservation group reported interactions (this the <br />                                                                              case protein&#226;&#8364;&#8220;protein genetic interactions con- <br />                                                                              servation levels 1&#226;&#8364;&#8220;3). <br />                                                                                  Since Saccharomyces cerevisiae RNAi absent, &#239;&#172;?t- <br />                                                                              ness effect individual genes, genetic interactions, and <br />                                                                              epistasis genes determined deletion of <br />                                                                              genes (see Deutschbauer et al. 2005) opposed RNAi <br />                                                                              knockdown used mutlicellular organisms (Dixon <br />                                                                              et al. 2009). However, means case over- <br />                                                                              lapping genes, deletions target gene, thus <br />                                                                              &#239;&#172;?tness measurements estimates strength of <br />                                                                              genetic interactions accurate. In consequence, anal- <br />                                                                              yses involving &#239;&#172;?tness, genetic interactions, epistasis, <br />                                                                              overlapping genes excluded data set. <br />                                                                              Determination secondary structure mutagenesis <br />                                                                              proteins <br />                                                                              In structural analysis, addition sequences with <br />                                                                              conservation levels 0&#226;&#8364;&#8220;10 I used amino acid sequences that <br />                                                                              generated randomly, length amino <br />                                                                              acid frequencies youngest proto-genes conserva- <br />                                                                              tion levels 0&#226;&#8364;&#8220;1. The secondary structures sequences <br />                                                                              determined PSSpred (http://zhanglab.ccmb.med. <br />                                                                              umich.edu/PSSpred/), aggregation propensity Tango <br />                                                                              (Fernandez-Escamilla et al. 2004) (see Table S1). Since pro- <br />                                                                              tein sequences different conservation levels very <br />                                                                              different numbers protein homologs, secondary struc- <br />                                                                              ture prediction includes homology search, avoid biases, <br />                                                                              PSSpred run use homology search <br />                                                                              step (psi-blast), reduces accuracy secondary <br />                                                                              structure prediction 81 69%. <br />                                                                                 The mutagenesis yeast proteins based on <br />                                                                              Schaefer et al. (2010) performed follows (see <br />                                                                              Figure 6). Each sequence mutated gradually 70 <br />                                                                              steps, step 1% residues changed in <br />                                                                              sequence. Since protein sequences evolve entirely <br /> Figure 2 Integration new genes regulatory networks. Proto-genes      randomly, simulate realistic mutations, new residues were <br /> acquire regulatory interactions rapidly; genes conservation <br /> level 1 co-regulated thousands genes (A), (B) regulated by <br /> several transcription factors (B), rapidly gain regulatory motifs <br /> (feed-forward loops) (C). The difference proto-genes con-        million-year-old genes (P . 0.05 conservation levels 5,6,8,10, <br /> served genes largely disappears conservation level 4, representing  14    ANOVA, Bonferroni post hoc tests). <br />  <br />  <br />  <br />                                                                                                 Dynamics Protogene Functionalization   1409 <br />  introduced context speci&#239;&#172;?c manner, using csbuild <br /> tool CS-BLAST suite (Biegert S&#195;&#182;ding 2009). In <br /> each step, secondary structure mutated sequence <br /> was determined, fraction residues similar <br /> secondary structure original secondary structure (Q3 <br /> value) calculated. For yeast protein sequence, the <br /> mutagenesis repeated &#239;&#172;?ve times independently, the <br /> average &#239;&#172;?ve replicates used analyses. To <br /> save computing time, genes conservation level 10, <br /> only 10% sequences mutated (which cho- <br /> sen randomly). <br />  <br /> Results <br /> Rapid emergence regulatory interactions <br /> The functionalization new gene means that, addition <br /> to evolving translatable open reading frame, needs        Figure 3 The percentage essential genes genes different <br /> become transcriptionally active, i.e., acquire promoter     conservation level; proportions indicated bars. The youn- <br />                                                                   gest essential genes overlap older genes (YEL035C, <br /> must co-regulated genes interacts or <br />                                                                   YPL124W) appear conservation level 4. (Note proto-genes <br /> other genes expressed life stages         overlap conserved genes, essentiality present con- <br /> environmental conditions. Studies novel retrogenes indi-       servation level 1; however, cases &#239;&#172;?tness effect in- <br /> cate regulatory elements new genes acquired        dependent overlapping conserved gene.) <br /> from nearby genes distant locations <br /> (Kaessmann et al. 2009; Kaessmann 2010). Using YEAS-          type interactions components, FFLs <br /> TRACT database (Abdulrehman et al. 2011) I analyzed           speed delay response target genes (Mangan <br /> integration proto-genes yeast regulatory network      Alon 2003). I identi&#239;&#172;?ed FFL-like motifs net- <br /> by quantifying number genes share            work YEASTRACT TFs tested number of <br /> transcription factors (TFs), co-regulated, num-      FFLs regulating gene changes gene age. Due the <br /> ber TFs regulating genes, emergence sim-      high connectivity network overlaps between <br /> ple network motif, feed-forward loop. The results        motifs, large number FFLs identi&#239;&#172;?ed YEASTRACT. <br /> that integration genes regulatory network         Nevertheless, I similar, increasing pattern with <br /> happens gradually, quickly; young proto-           number TFs; i.e., conserved genes regulated by <br /> genes conservation levels 1&#226;&#8364;&#8220;3 co-regulated          proportionally FFLs proto-genes (P , 0.05, <br /> a signi&#239;&#172;?cantly smaller number genes genes            ANOVA, Bonferroni post hoc tests, Figure 2C), indicating <br /> conservation level 5&#226;&#8364;&#8220;10 (P , 0.05 comparisons,            gradual rapid acquisition regulatory motifs that <br /> ANOVA, Bonferroni post hoc tests, Figure 2A), genes        enable complex regulatory behaviors. <br /> the youngest group (conservation level 1) co- <br />                                                                   Essential genes present proto-genes <br /> regulated thousands genes, oldest <br /> proto-genes conservation level 4 ( 14 milion years           Recent &#239;&#172;?ndings Drosophila indicate essential genes <br /> old) signi&#239;&#172;?cantly different older groups        (i.e., genes deletion mutants knockouts not <br /> (conservation levels 5,6, 8, 10, P . 0.05, ANOVA, Bonferroni      viable) evolve rapidly (Chen et al. 2010) though <br /> post hoc tests). A comparable pattern visible number   common ancient genes. Using &#239;&#172;?t- <br /> of transcription factors regulating genes (Figure 2B)     ness data set provided Deutschbauer et al. (2005) I tested <br /> proto-genes conservation level 1&#226;&#8364;&#8220;3 regulated sig-     time necessary emergence essential <br /> ni&#239;&#172;?cantly transcription factors conserved genes,        genes yeast. The results youngest essential <br /> or proto-genes conservation level 4 (P , 0.05,          genes (YEL035C, YPL124W) present proto- <br /> ANOVA, Bonferroni post hoc tests); however,           genes conservation level 4, homologs S. baya- <br /> youngest group (conservation level 1) regulated     nus (Figure 3), split S. cerevisiae lineage  14 <br /> transcription factors.                                            MYA. However, essentiality evolve faster: de- <br />    Regulatory networks characterized signi&#239;&#172;?cantly        letion proto-genes (already 7 genes with <br /> overrepresented patterns TFs, termed network motifs,           conservation level 1) lethal phenotypes, which, how- <br /> which capable performing distinct regulatory func-         ever, likely to overlap conserved genes. <br /> tions (Lee et al. 2002; Odom et al. 2006). One        Determining independent &#239;&#172;?tness effect require <br /> important motifs feed-forward loop (FFL),       &#239;&#172;?tness estimates independent effect the <br /> consists genes: TFs, regulates      genes overlap with, e.g., obtained RNAi introduced <br /> other, regulating target gene. Depending        yeasts (Drinnenberg et al. 2009). <br />  <br />  <br />  <br /> 1410      G. Abrus&#195;&#161;n <br />                                                                             Protein secondary structure aggregation propensity <br />                                                                            The analysis secondary structure motifs genes with <br />                                                                            different conservation level shows surprising pattern. The <br />                                                                            abundance a-helices in&#239;&#172;&#8218;uenced age the <br />                                                                            proteins; helices  40% sequence, regardless <br />                                                                            conservation level, random amino acid sequences <br />                                                                            (Figure 4A, P . 0.05 comparisons proto-genes <br />                                                                            conserved genes, Bonferroni post hoc tests, ANOVA). In <br />                                                                            contrast, predicted b-sheets highest in <br />                                                                            proto-genes random sequences ( 20%) shows de- <br />                                                                            cline  10% increasing level conservation (Figure <br />                                                                            4B, P ,, 0.001 comparisons proto-genes <br />                                                                            conserved genes, Bonferroni post hoc tests, ANOVA). <br />                                                                            This suggests b-strands evolutionarily unsta- <br />                                                                            ble, gradually disappear protein sequence accu- <br />                                                                            mulates neutral mutations, declining abundances <br />                                                                            caused natural selection. One selective force be <br />                                                                            aggregation propensity (Monsellier Chiti 2007), it <br />                                                                            depends largely presence b-strands (Jahn et al. <br />                                                                            2010). Predicted aggregation propensity shows even <br />                                                                            stronger pattern b-strands; average aggregation <br />                                                                            propensity random sequences proto-genes almost <br />                                                                            times higher conserved genes (Figure 4C, <br />                                                                            P ,, 0.001, comparisons proto-genes and <br />                                                                            conserved genes, ANOVA). <br />                                                                            Turnover proto-genes <br />  <br />                                                                            Functional noncoding sequences short lived subject <br />                                                                            rapid turnover mammalian genomes (Ponting et al. <br />                                                                            2011). If natural selection cause b-strand loss, it <br />                                                                            manifest itself gradual loss strands muta- <br />                                                                            tions, loss entire genes high strand content. I <br />                                                                            examined importance mechanism testing <br />                                                                            probability gene loss depends age the <br />                                                                            gene. I used genes conservation levels 4&#226;&#8364;&#8220;10, using <br />                                                                            orthogroup data set provided Wapinski et al. (2007) I <br />                                                                            quanti&#239;&#172;?ed gene losses youngest Saccharomyces spe- <br />                                                                            cies, S. paradoxus S. mikatae (conservation levels 2&#226;&#8364;&#8220;3). <br />                                                                            The results relatively young genes conserva- <br />                                                                            tion levels 4&#226;&#8364;&#8220;5 lost signi&#239;&#172;?cantly higher frequencies in <br />                                                                            species conserved genes (40% lost as <br />                                                                            opposed 7&#226;&#8364;&#8220;10%, P , 0.05 comparisons, x-square <br />                                                                            tests, Figure 5), corresponds pattern of <br />                                                                            b-strand loss indicates turnover proto-genes <br />                                                                            likely contribute observed structural change. <br />                                                                            Robustness secondary structures mutations <br />                                                                            To test effect <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> accumulation second- <br />                                                                            ary structure robustness yeast proteins, I performed <br />                                                                            silico mutagenesis studies. I followed procedure was <br />                                                                            based Schaefer et al. (2010); mutations introduced <br /> Figure 4 Changes secondary structure aggregation propensity <br />                                                                            gradually protein sequences, I examined how <br /> with gene age. While a-helices does depend pro- <br /> tein age (A), b-strands declines signi&#239;&#172;?cantly con- <br /> servation levels 4 6 (B). Aggregation propensity, partly      b-strands, random amino acid sequences proto-genes being <br /> caused presence b-strands, shows stronger trend     prone aggregation conserved genes (C). <br />  <br />  <br />  <br />                                                                                               Dynamics Protogene Functionalization   1411 <br />                                                                              structure (Zhang et al. 2012), demonstrated <br />                                                                             interacting surfaces frequently characterized <br />                                                                             similar arrangement secondary structure elements <br />                                                                             (Tuncbag et al. 2008; Zhang et al. 2010; Gao Skolnick <br />                                                                             2010). The size age protein complexes highly <br />                                                                             variable, currently consensus rate <br />                                                                             evolution (Qian et al. 2011; Lewis et al. 2012). In pre- <br />                                                                             vious study Capra et al. (2010) demonstrated that <br />                                                                             yeast genes younger whole-genome duplication <br />                                                                             integrated protein&#226;&#8364;&#8220;protein interaction networks <br />                                                                             older genes. I tested age (conservation <br />                                                                             level) yeast proteins in&#239;&#172;&#8218;uences number phys- <br />                                                                             ical interactions mirrors changes sec- <br />                                                                             ondary structure proteins. Unlike secondary structure <br />                                                                             elements, protein&#226;&#8364;&#8220;protein interactions increase gradually <br />                                                                             age proteins (Figure 8A) appears to <br /> Figure 5 The probability gene loss S. paradoxus S. mikatae. Only   monotonous process; using conservation levels 6 and <br /> proteins emerged S. cerevisiae&#226;&#8364;&#8220;S. mikatae split were <br />                                                                             10 estimate, median number interactions <br /> examined. Genes conservation levels 4&#226;&#8364;&#8220;5 lost signi&#239;&#172;?cantly <br /> higher frequencies conserved genes (P , 0.05          increases rate 2&#226;&#8364;&#8220;2.25 3 1028/year (7/311 MY and <br /> comparisons conservation levels 4&#226;&#8364;&#8220;5 vs. 6&#226;&#8364;&#8220;10, x-square tests).      15/760 MY). <br />                                                                                Genetic interactions mean synergistic effect of <br />                                                                             genes j &#239;&#172;?tness (fij) different expected by <br /> rapidly original secondary structure degrades                  multiplicative effects: fij = fi fj + e, e 6&#194;&#188; 0, fj <br /> change amino acid composition sequence (see                   denotes effect gene j &#239;&#172;?tness, e magnitude <br /> Figure 6 Materials Methods details). The results                epistasis genes (Dixon et al. 2009; <br /> show proto-genes sensitive mutations                  Costanzo et al. 2010). An extreme example synthetic <br /> are older genes; i.e., secondary structure changes faster             lethal <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, independent deletion two <br /> with sequence change secondary structure con-                   genes cases results viable phenotypes, their <br /> served genes (Figure 7A, P ,, 0.001, Bonferroni post hoc                    double deletion mutant lethal. I tested num- <br /> tests, ANCOVA conservation level categorical predic-                ber genetic interactions depends gene age also <br /> tor sequence divergence continuous predictor). How-                  positive relationship (Figure 8B): <br /> ever, structural composition different           number genetic interactions increases rate of <br /> of conserved genes (Figure 4), by-product                4.8 3 1028/year (15/311 MYA); however, appears to <br /> of compositional differences. The comparison muta-                   level conservation level 6 (311 MYA). Similarly to <br /> tional robustness a-helices b-sheets indicates                  number genetic interactions, strength changes <br /> b-sheets decay considerably faster random mutations                    age genes: magnitude epistasis (|e|) of <br /> than a-helices (Figure 7B, P ,, 0.001, ANCOVA),                     interactions proto-genes signi&#239;&#172;?cantly lower than <br /> robustness proteins mutations negatively correlated               conserved genes (P ,, 0.001, Mann&#226;&#8364;&#8220;Whitney <br /> with b-strand content (Figure 7C, P ,, 0.001, R =                     U-test, Figure 9). <br /> 20.54). This suggests b-strands dif&#239;&#172;?cult to <br /> maintain longer evolutionary periods a-helices,                   Discussion <br /> and reduced conserved genes be <br /> the result accumulation neutral mutations. Repeat-                Overall, integration new genes cellular networks <br /> ing stability analysis nonstrand regions            supports hypothesis Carvunis et al. (2012) con- <br /> proteins indicates proto-genes remain sensitive               tinuum exists conserved genes proto-genes. <br /> mutations (Figure 7D, P ,, 0.001, Bonferroni post hoc tests,                The fastest process integration proto-genes into <br /> ANCOVA), exclusion b-strands                     regulatory networks (Figure 2): youngest <br /> analysis; higher robustness old genes compared                  proto-genes regulated TFs, co-regulated with <br /> to proto-genes merely by-product lower                    large number genes, regulated network <br /> b-strand content.                                                           motifs (FFLs). The oldest proto-genes appeared  14 <br />                                                                             MYA integrated regulatory net- <br /> The number strength genetic protein&#226;&#8364;&#8220;protein <br />                                                                             work ancient fraction yeast pro- <br /> interactions increases age genes <br />                                                                             teome, appeared S. cerevisiae&#226;&#8364;&#8220;S. pombe <br /> Proteins typically perform function association               split. Although analysis focused topological fea- <br /> other proteins form protein complexes. Physical inter-                  tures entire regulatory network (i.e., ignored that <br /> actions proteins largely determined                    different environmental conditions life stages only <br />  <br />  <br />  <br /> 1412     G. Abrus&#195;&#161;n <br />                                                                                 subsets genes expressed), agreement <br />                                                                                observations reporting acquisition regula- <br />                                                                                tory sequences new (retro)genes happen rapidly <br />                                                                                (Kaessmann et al. 2009) horizontally transferred genes <br />                                                                                bacteria acquire regulatory interactions mil- <br />                                                                                lion years (Lercher P&#195;&#161;l 2008) &#239;&#172;?ndings <br />                                                                                proportion proto-genes proximity known TF <br />                                                                                binding sites changes little conservation level (Carvunis <br />                                                                                et al. 2012). The fact youngest essential genes are <br />                                                                                present proto-genes (Figure 3), in <br />                                                                                agreement experimental &#239;&#172;?ndings reporting differences <br />                                                                                sets essential genes yeast strains <br />                                                                                (Dowell et al. 2010), indicates proto-genes gain <br />                                                                                functions rapidly. Unlike regulatory evolution, acquisi- <br />                                                                                tion protein&#226;&#8364;&#8220;protein interactions (PPI) dramatically <br />                                                                                slower process, does appear level pro- <br />                                                                                tein age, range examined conser- <br />                                                                                vation levels (Figure 8). The observed rate PPI gain at <br />                                                                                order magnitude faster loss interac- <br />                                                                                tions, estimated occur rate 1029 (Lewis <br />                                                                                et al. 2012) 1.6&#226;&#8364;&#8220;2.6 3 10210 (Qian et al. 2011). In addi- <br />                                                                                tion, interaction databases incomplete, rate of <br />                                                                                PPI gain likely underestimated. In case genetic <br />                                                                                interactions increase number new interactions <br />                                                                                pronounced conservation level 6 (311 MYA, <br />                                                                                Figure 8), oldest group (conservation level <br />                                                                                10) characterized signi&#239;&#172;?cantly higher number <br />                                                                                interactions genes conservation levels 6&#226;&#8364;&#8220;9. <br />                                                                                Unlike PPIs, genetic interactions allow measurement <br />                                                                                strength interactions (e), indicates stronger <br />                                                                                epistasis conserved genes proto- <br />                                                                                genes; change quantitative qual- <br />                                                                                itative (Figure 9). <br />                                                                                    In contrast, structural maturation new genes shows <br />                                                                                different pattern: secondary structures structural <br />                                                                                stability proteins indicate proto-genes conserved <br />                                                                                genes form relatively distinct groups: proto-genes with <br />                                                                                high b-strand content, high aggregation propensity, low <br />                                                                                robustness mutations conserved genes with <br />                                                                                lower strand content higher stability. Proto-genes with <br />                                                                                conservation levels 0&#226;&#8364;&#8220;3 different random <br />                                                                                sequences (with similar amino acid frequency), the <br />                                                                                transition states falls conservation <br />                                                                                levels 4 ( 14 MYA) 6 ( 311 MYA), requiring <br />                                                                                considerably time regulatory evolution (Figures <br />                                                                                4 7). There mechanisms cause <br />                                                                                shift; selection high b-strand content neutral <br />                                                                                process. Although selection favoring mutations re- <br /> Figure 6 An overview analysis protein structural robustness,      duce aggregation propensity proto-genes seem <br /> the example yeast ORF YDR103W. (A) The tertiary structure the <br /> protein (PDB id: 4F2H). a-helices highlighted blue b-strands <br /> with yellow. (B) The sequence protein gradually mutated 70 <br /> steps; step 1% residues changed, step       original protein, particularly b-strands. (C) For protein the <br /> secondary structure determined. The change location heli-        mutagenesis repeated &#239;&#172;?ve times, Q3 value&#226;&#8364;&#8221;the percentage <br /> ces sheets occurs mutagenesis indicted           residues secondary structure original structure&#226;&#8364;&#8221; <br /> respective colors. As sequence similarity original sequence declines,   calculated step. Every line represents mutagenesis path <br /> fewer fewer residues secondary structure        (replicate); analyses average &#239;&#172;?ve replicates used. <br />  <br />  <br />  <br />                                                                                                      Dynamics Protogene Functionalization        1413 <br />  Figure 7 Structural robustness proteins. (A) The robustness secondary structures mutations depends conservation level. Proto-genes <br /> and ancient genes highly signi&#239;&#172;?cant difference (ANCOVA, P ,, 0.001 comparisons proto-genes conserved genes, Bonferroni <br /> post hoc tests; whiskers represent 95% con&#239;&#172;?dence intervals); secondary structure ancient genes sensitive mutations (i.e., Q3 value is <br /> higher). (B) b-strands decay faster random mutations a-helices (P ,, 0.001, ANCOVA). (C) The b-strands proteins correlates <br /> negatively structural stability protein. Q3 values calculated 50% sequence similarity original sequence. (D) The structural <br /> stability proteins, excluding regions b-strands. The difference proto-genes ancient genes highly signi&#239;&#172;?cant (P ,, 0.001, <br /> ANCOVA), indicating merely by-product compositional differences ancient proto-genes (see Figure 4). <br />  <br />  <br />  <br /> an attractive explanation reduction seen                             est number founder genes typically form peak the <br /> b-strands, &#239;&#172;?tness cost misfolding-induced aggre-                     youngest evolutionary strata (Tautz Domazet-Lo&#203;&#8225;so <br /> gation demonstrated experimentally yeast                            2011), implying proto-genes subject form <br /> (Geiler-Samerotte et al. 2011), tests selection                       turnover, similarly recently shown for <br /> Carvunis et al. (2012) reported weak purifying selection                     functional noncoding sequence mammals (Meader et al. <br /> proto-genes, strong adaptive evolution. Alternatively,                  2010). Gene deletion inactivation studies that <br /> the reduction strand content aggregation propensity                      80&#226;&#8364;&#8220;90% genes eukaryotes prokaryotes lost <br /> may re&#239;&#172;&#8218;ect that, despite partly integrated                   individually signi&#239;&#172;?cant &#239;&#172;?tness effect (Korona <br /> regulatory networks, considerable fraction proto-genes                     2011), laboratory conditions, been <br /> does make higher conservation levels lost                     suggested genes lost easily evolution <br /> relatively quickly appearance. This suppor-                 important, i.e., lower expression levels, fewer <br /> ted phylostratigraphic studies, high-                    protein&#226;&#8364;&#8220;protein interactions (Krylov et al. 2003), higher <br />  <br />  <br />  <br /> 1414      G. Abrus&#195;&#161;n <br />                                                                                Figure 9 Genetic interactions proto-genes weaker epistasis than <br />                                                                               conserved genes (P ,, 0.001, Mann&#226;&#8364;&#8220;Whitney U-test). For proto- <br />                                                                               conserved gene mean absolute genetic interaction scores <br />                                                                               (|e|) calculated; histograms represent positive and <br />                                                                               negative epistatic interactions. <br />  <br />  <br />  <br />  <br />                                                                               consideration higher b-strand content (Figure 7). The <br />                                                                               mutations proteins accumulate without <br />                                                                               change structure key determinants <br />                                                                               capacity produce evolutionary innovations, be- <br />                                                                               cause robust proteins better tolerate innovative <br />                                                                               destabilizing mutations (Bloom et al. 2006b; Ferrada <br />                                                                               Wagner 2008). The pattern observed yeast pro- <br />                                                                               teome line experimental &#239;&#172;?ndings individual <br />                                                                               proteins, suggest robustness mutations <br />                                                                               promotes evolvability (Bloom et al. 2006b) evo- <br />                                                                               lution favors mutational robustness (Bloom et al. 2007; <br />                                                                               Bershtein et al. 2008). Additionally, large-scale analyses <br />                                                                               folding energies (DG) protein structures indicate <br />                                                                               ancient, eukaryote-wide proteins thermodynami- <br /> Figure 8 The dependence protein&#226;&#8364;&#8220;protein genetic interactions <br /> from gene age. Note y-axis logarithmic        cally stable robust mutations younger <br /> genes included interactions, correct research biases.   ones present metazoans vertebrates (Toll-Riera <br /> (A) The number protein&#226;&#8364;&#8220;protein interactions increases continuously         et al. 2012). Moreover, lower tolerance b-strands <br /> with conservation level; new protein&#226;&#8364;&#8220;protein interactions emerge rate    mutations predicts strands, conserved <br /> 2&#226;&#8364;&#8220;2.25 3 1028/year. (B) The number genetic interactions increases at <br />                                                                               proteins, evolve lower rates helices coils, <br /> a rate  4.8 3 1028; however, rate change slows above <br /> conservation level 6.                                                         yeast (Bloom et al. 2006a), al- <br />                                                                               effect weaker effects struc- <br />                                                                               tural traits like solvent accessibility. <br /> evolutionary rate (Zhang He 2005). The analysis gene                      To conclude, results somewhat contradictory <br /> loss study indicates proto-gene turnover exists;                 picture evolution new genes yeast: integra- <br /> young genes lower conservation levels (4&#226;&#8364;&#8220;5) lost                     tion proto-genes cellular networks shows contin- <br /> considerably easily S. paradoxus S. mikatae                       uum, rates regulatory evolution gain <br /> than ancient genes (Figure 5), integrated                      protein genetic interactions different. In con- <br /> into cellular networks. An alternative explanation                    trast, structural point view, new genes seem <br /> decay b-strands neutral process,                 face bottleneck: proto-genes conserved genes form <br /> the a-helices maintained selection, loss                  relatively distinct groups, different b-strand content, <br /> of strands neutral mutations compensated.                       aggregation propensity, robustness mutations. This, <br />    The robustness secondary structure mutations                         &#239;&#172;?nding young genes lost easily <br /> shows similar shift conservation levels 4 6                   conserved ones, indicates that, have <br /> b-strand content aggregation propensity: proto-genes                      functionality, young genes unstable the <br /> are robust conserved genes, takes                  genome. <br />  <br />  <br />  <br />                                                                                                     Dynamics Protogene Functionalization       1415 <br />  Acknowledgments                                                             esis regulating Y-linked male fertility genes. PLoS <br />                                                                             Genet. 6: e1001255. <br /> I thank Bal&#195;&#161;zs Papp referees useful comments                 Dixon, S. J., M. Costanzo, A. Baryshnikova, B. Andrews, C. <br /> and suggestions. This study supported Hungarian                  Boone, 2009 Systematic mapping genetic interaction net- <br /> Scienti&#239;&#172;?c Research Fund (OTKA) grant PD83571 G.A.                        works. Annu. Rev. Genet. 43: 601&#226;&#8364;&#8220;625. <br />                                                                          Dowell, R. D., O. Ryan, A. Jansen, D. Cheung, S. Agarwala et al., <br /> The author declares competing interests. <br />                                                                             2010 Genotype phenotype: complex problem. Science <br />                                                                             328: 469. <br />                                                                          Drinnenberg, I. A., D. E. Weinberg, K. T. Xie, J. P. Mower, K. H. <br /> Literature Cited                                                            Wolfe et al., 2009 RNAi budding yeast. Science 326: 544&#226;&#8364;&#8220; <br />                                                                             550. <br /> Abdulrehman, D., P. T. Monteiro, M. C. Teixeira, N. P. Mira, A. B.       Edgar, R. C., 2010 Search clustering orders magnitude <br />     Louren&#195;&#167;o et al., 2011 YEASTRACT: providing programmatic               faster BLAST. Bioinformatics 26: 2460&#226;&#8364;&#8220;2461. <br />     access curated transcriptional regulatory associations Sac-    Fernandez-Escamilla, A.-M., F. Rousseau, J. Schymkowitz, L. <br />     charomyces cerevisiae web services interface. Nucleic         Serrano, 2004 Prediction sequence-dependent muta- <br />     Acids Res. 39: D136&#226;&#8364;&#8220;D140.                                               tional effects aggregation peptides proteins. Nat. <br /> Begun, D. J., H. A. Lindfors, A. D. Kern, C. D. Jones,                  Biotechnol. 22: 1302&#226;&#8364;&#8220;1306. <br />     2007 Evidence novo evolution testis-expressed genes        Ferrada, E., A. Wagner, 2008 Protein robustness promotes <br />     Drosophila yakuba/Drosophila erecta clade. Genetics 176:         evolutionary innovations large evolutionary time-scales. <br />     1131&#226;&#8364;&#8220;1137.                                                              Proc. Biol. Sci. 275: 1595&#226;&#8364;&#8220;1602. <br /> Bershtein, S., K. Goldin, D. S. Taw&#239;&#172;?k, 2008 Intense neutral          Gao, M., J. Skolnick, 2010 Structural space protein-protein <br />     drifts yield robust evolvable consensus proteins. J. Mol.           interfaces degenerate, close complete, highly con- <br />     Biol. 379: 1029&#226;&#8364;&#8220;1044.                                                   nected. Proc. Natl. Acad. Sci. USA 107: 22517&#226;&#8364;&#8220;22522. <br /> Biegert, A., J. S&#195;&#182;ding, 2009 Sequence context-speci&#239;&#172;?c pro&#239;&#172;?les        Geiler-Samerotte, K. A., M. F. Dion, B. A. Budnik, S. M. Wang, D. L. <br />     homology searching. Proc. Natl. Acad. Sci. USA 106: 3770&#226;&#8364;&#8220; <br />                                                                             Hartl et al., 2011 Misfolded proteins impose dosage-dependent <br />     3775. <br />                                                                             &#239;&#172;?tness cost trigger cytosolic unfolded protein response in <br /> Bloom, J. D., D. A. Drummond, F. H. Arnold, C. O. Wilke, <br />                                                                             yeast. Proc. Natl. Acad. Sci. USA 108: 680&#226;&#8364;&#8220;685. <br />     2006a Structural determinants rate protein evolution <br />                                                                          Hedges, S. B., J. Dudley, S. Kumar, 2006 TimeTree: public <br />     yeast. Mol. Biol. Evol. 23: 1751&#226;&#8364;&#8220;1761. <br />                                                                             knowledge-<span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> divergence times organisms. Bioin- <br /> Bloom, J. D., S. T. Labthavikul, C. R. Otey, F. H. Arnold, <br />                                                                             formatics 22: 2971&#226;&#8364;&#8220;2972. <br />     2006b Protein stability promotes evolvability. Proc. Natl. Acad. <br />                                                                          Heinen, T. J. A. J., F. Staubach, D. H&#195;&#164;ming, D. Tautz, <br />     Sci. USA 103: 5869&#226;&#8364;&#8220;5874. <br />                                                                             2009 Emergence new gene intergenic region. <br /> Bloom, J. D., Z. Lu, D. Chen, A. Raval, O. S. Venturelli et al., <br />                                                                             Curr. Biol. 19: 1527&#226;&#8364;&#8220;1531. <br />     2007 Evolution favors protein mutational robustness suf&#239;&#172;?- <br />                                                                          Jahn, T. R., O. S. Makin, K. L. Morris, K. E. Marshall, P. Tian et al., <br />     ciently large populations. BMC Biol. 5: 29. <br /> Cai, J., R. Zhao, H. Jiang, W. Wang, 2008 De novo origination           2010 The common architecture cross-beta amyloid. J. Mol. <br />     new protein-coding gene Saccharomyces cerevisiae. Ge-           Biol. 395: 717&#226;&#8364;&#8220;727. <br />     netics 179: 487&#226;&#8364;&#8220;496.                                                 Kaessmann, H., 2010 Origins, evolution, phenotypic impact <br /> Capra, J. A., K. S. Pollard, M. Singh, 2010 Novel genes exhibit         new genes. Genome Res. 20: 1313&#226;&#8364;&#8220;1326. <br />     distinct patterns function acquisition network integra-       Kaessmann, H., N. Vinckenbosch, M. Long, 2009 RNA-based <br />     tion. Genome Biol. 11: R127.                                            gene duplication: mechanistic evolutionary insights. Nat. <br /> Carvunis, A.-R., T. Rolland, I. Wapinski, M. A. Calderwood, M. A.           Rev. Genet. 10: 19&#226;&#8364;&#8220;31. <br />     Yildirim et al., 2012 Proto-genes novo gene birth. Na-        Kapranov, P., A. T. Willingham, T. R. Gingeras, <br />     ture 487: 370&#226;&#8364;&#8220;374.                                                      2007 Genome-wide transcription implications ge- <br /> Chen, S., Y. E. Zhang, M. Long, 2010 New genes Drosoph-              nomic organization. Nat. Rev. Genet. 8: 413&#226;&#8364;&#8220;423. <br />     ila quickly essential. Science 330: 1682&#226;&#8364;&#8220;1685.                Knowles, D. G., A. McLysaght, 2009 Recent novo origin of <br /> Chen, S., X. Ni, B. H. Krinsky, Y. E. Zhang, M. D. Vibranovski et al.,      human protein-coding genes. Genome Res. 19: 1752&#226;&#8364;&#8220;1759. <br />     2012 Reshaping global gene expression networks sex-           Korona, R., 2011 Gene dispensability. Curr. Opin. Biotechnol. 22: <br />     biased gene expression integration young gene. EMBO J.          547&#226;&#8364;&#8220;551. <br />     31: 2798&#226;&#8364;&#8220;2809.                                                       Krylov, D. M., Y. I. Wolf, I. B. Rogozin, E. V. Koonin, <br /> Cherry, J. M., E. L. Hong, C. Amundsen, R. Balakrishnan, G. Binkley         2003 Gene loss, protein sequence divergence, gene dispens- <br />     et al., 2012 Saccharomyces Genome Database: genomics                ability, expression level, interactivity correlated eu- <br />     resource budding yeast. Nucleic Acids Res. 40: D700&#226;&#8364;&#8220;D705.            karyotic evolution. Genome Res. 13: 2229&#226;&#8364;&#8220;2235. <br /> Cliften, P. F., R. S. Fulton, R. K. Wilson, M. Johnston,             Lee, T. I., N. J. Rinaldi, F. Robert, D. T. Odom, Z. Bar-Joseph et al., <br />     2006 After duplication: gene loss adaptation Sac-            2002 Transcriptional regulatory networks Saccharomyces <br />     charomyces genomes. Genetics 172: 863&#226;&#8364;&#8220;872.                              cerevisiae. Science 298: 799&#226;&#8364;&#8220;804. <br /> Costanzo, M., A. Baryshnikova, J. Bellay, Y. Kim, E. D. Spear et al.,    Lercher, M. J., C. P&#195;&#161;l, 2008 Integration horizontally trans- <br />     2010 The genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431.             ferred genes regulatory interaction networks takes many <br /> DePristo, M. A., D. M. Weinreich, D. L. Hartl, 2005 Missense            million years. Mol. Biol. Evol. 25: 559&#226;&#8364;&#8220;567. <br />     meanderings sequence space: biophysical view protein         Levine, M. T., C. D. Jones, A. D. Kern, H. A. Lindfors, D. J. <br />     evolution. Nat. Rev. Genet. 6: 678&#226;&#8364;&#8220;687.                                 Begun, 2006  <br /> </body></html>