<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                          Generation of a set of conditional <br />                         analog-sensitive alleles of essential <br />                         protein kinases in the fission yeast <br />                            Schizosaccharomyces pombe <br />  <br />                              Key words: kinase, analog-sensitive, conditional allele, fission yeast, phosphorylation <br />  <br />  <br />           The genome of the fission yeast Schizosaccharomyces pombe encodes for 17 protein kinases that are essential for viability. <br />           Studies of the essential kinases often require the use of mutant strains carrying conditional alleles. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> inactivate these <br />           kinases conditionally, we applied a recently developed chemical genetic strategy. The <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>mutation</span> of a single residue in the <br />           ATP-binding pocket confers sensitivity to small-molecule inhibitors, allowing for specific inactivation of the modified <br />           kinase. Using this approach, we constructed conditional analog-sensitive alleles of 13 essential protein kinases in the <br />           fission yeast S. pombe. <br />  <br />  <br />  <br />                                Introduction                                            viability.7,8 We created heterozygous diploid deletion strains for <br />                                                                                        each of these 18 kinases and analyzed the haploid products after <br /> It is well-established that reversible protein phosphorylation is                      sporulation. We confirmed that 17 kinase genes are essential for <br /> one of the major mechanisms by which cellular processes are                            viability. However, the sporulation of a heterozygous diploid <br /> regulated.1,2 The analysis of protein kinases and phosphatases has                     deletion strain, followed by the dissection of the asci on rich <br /> been instrumental to various fields of cell biology. The fission                       medium, showed that one of the kinases, ppk28, is not essential <br /> yeast Schizosaccharomyces pombe is an excellent model organism                         for viability (data not shown). This observation is consistent with <br /> for studying eukaryotic biology, because it is amenable to genetic,                    previous reports showing that ppk28 is dispensable for normal <br /> biochemical and cytological techniques as well as genome-wide                          cell growth.9 Of the 17 essential kinases, only one analog-sen- <br /> approaches.3-5 The S. pombe genome encodes 107 predicted pro-                          sitive allele (ark1&#194;&#160;as3)10 was available when we started this proj- <br /> tein kinases, and 17 of them are known to be essential for cell                        ect. Therefore, we decided to create conditional analog-sensitive <br /> growth. To date, studies of the essential protein kinases in fission                   alleles for 16 remaining protein kinases essential for viability <br /> yeast have relied mostly on the use of mutant strains carrying                         (Table 1). <br /> conditional temperature-sensitive alleles. However, these alleles                          To create conditional analog-sensitive alleles, we mutated <br /> are often leaky and require analysis under non-physiological                           the gate-keeper residue within the kinase ATP-binding site to a <br /> conditions, and their inactivation requires a long period of time.                     glycine (as1) or an alanine (as2). The mutations were expressed <br /> A recently developed chemical genetic strategy for sensitizing                         in an S. pombe strain lacking the wild-type allele of the corre- <br /> protein kinases to small-molecule inhibitors provides an elegant                       sponding kinase according to the protocol of Gregan et al.11 <br /> tool for constructing conditional analog-sensitive kinase alleles                      Out of the 16 kinases with a mutated gate-keeper residue, 13 <br /> (Fig.&#194;&#160;1).6 In our current study, we used this strategy to create                       remained functional as indicated by growth of haploid cells car- <br /> analog-sensitive alleles of S. pombe protein kinases that are essen-                   rying the corresponding gate-keeper mutation in YES medium <br /> tial for cell growth.                                                                  (Table&#194;&#160;1 and Figs. 3 and S1). Tetrad analysis of heterozygous <br />                                                                                        diploid strains showed that the remaining three mutants (cdc7- <br />                                   Results                                              as, hsk1&#226;&#8364;&#8216;as and sid1-as) were not functional (Table 1). We also <br />                                                                                        observed that two mutants (cdc2-as1 and pat1-as) were sensitive <br /> A recent systematic deletion analysis of genes in the fission                          to elevated temperature (32&#194;&#176;C), but they grew normally at a low <br /> yeast S. pombe revealed that 18 protein kinases are essential for                      temperature (25&#194;&#176;C). The temperature sensitivity of cdc2-as1 has <br />  <br />  *Correspondence to: Juraj Gregan; Email: juraj.gregan@univie.ac.at <br />  Submitted: 07/26/11; Accepted: 08/17/11 <br />  http://dx.doi.org/10.4161/cc.10.20.17791 <br />  <br />  <br /> www.landesbioscience.com	                                                     Cell Cycle	                                                                            3527 <br />   Figure 1. Chemical genetic strategy for sensitizing protein kinases to small-molecule inhibitors. Mutation of the gate-keeper residue in a protein ki- <br />  nase (kinase-as) creates a new pocket where an ATP analog (inhibitor) can bind. Binding of the inhibitor to kinase-as leads to inactivation of the kinase <br />  (A). However, the kinase-as mutation typically does not interfere with the kinase function in the absence of the inhibitor (B). The inhibitor cannot bind <br />  to wild-type kinases, allowing for specific inactivation of the mutant kinase (C).6 <br />  <br />  <br />  <br />                                                                                                   In addition to the mutants created in this <br />                                                                                                   study, we included the previously constructed <br />                                                                                                   ark1-as3 mutant in our analysis.10 Spot test on <br />                                                                                                   YES plates showed that eight mutants were <br />                                                                                                   highly sensitive to one or more of the tested <br />                                                                                                   inhibitors (&#226;&#8240;&#164;10 mM ATP analog), and six <br />                                                                                                   mutants exhibited sensitivity only to higher <br />                                                                                                   concentrations of inhibitors (Table&#194;&#160;2 and <br />                                                                                                   Fig.&#194;&#160;3). To further characterize the mutant <br />                                                                                                   phenotypes, we added an inhibitor to cultures <br />                                                                                                   exponentially growing in liquid YES medium <br />                                                                                                   and analyzed the effect of inhibitors on cell <br />                                                                                                   growth by counting cell number. Four hours <br />                                                                                                   after adding the inhibitor, we harvested cells <br />                                                                                                   and analyzed the morphology of the cells by <br />                                                                                                   microscopy and the cellular DNA content by <br />                                                                                                   flow cytometry (Fig.&#194;&#160;S1). These analyses con- <br />                                                                                                   firmed that all 13 kinase mutants were sen- <br />                                                                                                   sitive to at least one of the tested inhibitors, <br />                                                                                                   and the growth of the mutant cells was either <br />                                                                                                   inhibited (7 mutants) or completely blocked <br />  Figure 2. Chemical structures of the inhibitors used in this study. <br />                                                                                                   (6&#194;&#160;mutants) in the presence of the inhibitor <br />                                                                                                   (Fig. S1). Thus, we conclude that analog- <br />                                                                                                   sensitive alleles were constructed for 13 out of <br /> been described previously in references 12 and 13. Thus, out                   16 of the essential kinases. Therefore, a chemical genetic strat- <br /> of the 16 essential kinases, we constructed 13 functional alleles              egy of conditional inactivation of protein kinases by creating <br /> where the gate-keeper residue was replaced by a glycine or an                  mutants sensitive to ATP analogs is a viable option for studying <br /> alanine.                                                                       S. pombe protein kinases that are essential for cell growth and <br />    To test the sensitivity of the 13 kinase mutants to inhibitors,             should encourage wider use of this strategy in yeast as well as <br /> we synthesized eight different inhibitors (ATP analogs) (Fig.&#194;&#160;2).              other organisms. <br />  <br />  <br />  <br /> 3528	                                                                  Cell Cycle	                                                       Volume 10 Issue 20 <br />                              Discussion                                     Table&#194;&#160;1. Functionality of analog-sensitive kinase mutants as determined <br />                                                                            by growth on YES plates <br />  <br /> In this study, we created conditional analog-sensitive alleles of               Kinase <br />                                                                                              Gate keeper           Functionality of <br /> 13 S. pombe protein kinases. Four of these mutants have been                                   residue                mutants <br /> described by others during the course of this work.12,14-17 Although                                                   as1 (Gly)*         as2 (Ala)* <br /> seven mutants were highly sensitive to one or more of the tested                Cdc2              F84                     yes&#226;&#8364;&#160;                    - <br /> inhibitors, six mutants exhibited sensitivity only to higher con-               Cdc7              L85                     no                  no <br /> centrations of inhibitors. In the future, it will be useful to further <br />                                                                                 Cdk9              T120                    yes&#226;&#8364;&#160;                    - <br /> optimize these mutants to increase their sensitivity to inhibitors. <br />                                                                                 Cka1              F117                    yes&#226;&#8364;&#160;                no <br /> In addition, the three kinases that did not tolerate mutation of <br /> the gate-keeper residue may be potentially salvaged by second-site               Crk1             L87                     yes                 no <br /> suppressor mutations.18 A growing number of studies have shown                  Hsk1              L173                    no                  no <br /> that small-molecule inhibitors selective for target kinases are ben-            Ksg1              L177                    yes   &#226;&#8364;&#160; <br />                                                                                                                                                   - <br /> eficial for conditional inactivation.19,20 Therefore, we believe that           Nak1              M88                     yes                 no <br /> the analog-sensitive alleles constructed in this study will provide a           Orb6             M170                     yes&#226;&#8364;&#160;                    - <br /> valuable tool for studying these protein kinases. ATP analogs are <br />                                                                                  Pat1             L95                     yes                 yes <br /> usually cell permeable, allowing for rapid and reversible inhibi- <br />                                                                                  Plo1             L117                    yes&#226;&#8364;&#160;                    - <br /> tion of the target protein kinase. An important advantage of rapid <br /> inactivation of the kinase is that the cell has little time to com-             Ppk37            M537                     yes                 yes <br /> pensate for the missing kinase activity. In addition, depending on               Prp4             F238                    no                  yes <br /> the amount of inhibitor added, total or partial inhibition of the               Shk1             M460                     no                  yes <br /> target kinase can be achieved. Finally, a small-molecule inhibitor               Sid1             M84                     no                  no <br /> and a genetic mutation can perturb the activity of a kinase in                   Sid2            M285                     yes                 no <br /> different ways and may result in different phenotypes. Therefore, <br />                                                                            *gate-keeper residue mutated to a glycine (Gly) or an alanine (Ala), <br /> analog-sensitive alleles may reveal new biological functions of            respectively; &#226;&#8364;&#160;sensitive to &#226;&#8240;&#164;10 &#206;&#188;M ATP analog. <br /> kinases.11,19,21 <br />     The analog-sensitive alleles described here not only provide <br /> a new tool for studying protein kinases, but they also comple-            Sytox-Green staining (FACS analysis) were followed as previ- <br /> ment currently available conditional alleles. For example, the            ously described in reference 24. <br /> analog-sensitive alleles could be combined with temperature-                 Deletion of genes encoding for essential kinases. To delete <br /> sensitive alleles to create double mutants in which the temper-           genes encoding for the essential kinases, we used the approach <br /> ature-sensitive protein can be inactivated by shifting cells to           based on <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>knockout</span> constructs that contain 150&#226;&#8364;&#8220;800 bp long <br /> non-permissive temperature, and the analog-sensitive kinase can           regions homologous to the target genes cloned into vector- <br /> be independently inactivated by adding inhibitor. Notably, some           carrying dominant drug-resistance markers.23 The oligonucle- <br /> of the tested ATP analogs inhibit only a subset of kinase mutants         otides were designed according to the database of S. pombe <br /> (Table 2), which raises the possibility that some of the analog-          deletion     constructs     (mendel.imp.ac.at/Pombe_deletion). <br /> sensitive alleles could be combined in double-mutant strains and          Sequences flanking the genes encoding the essential kinases <br /> separately inactivated with specific inhibitors.                          were PCR-amplified <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> S. pombe genomic DNA using the <br />     The fission yeast Schizosaccharomyces pombe is an excellent           primers up-out and up-in for upstream regions and down-out <br /> model organism for studying various aspects of biology, and it            and down-in for downstream regions (Table S1). The resulting <br /> is likely that the analog-sensitive alleles described in this work        PCR products were ligated to each other, and the proximal ends <br /> will become a useful tool for future studies. It is notable that          were cloned into a pCloneNat1 vector (EF101285) carrying <br /> only 3&#194;&#160;out of the 16 kinases did not tolerate mutation of the gate-       drug resistance markers for E. coli (Ampicillin) and S. pombe <br /> keeper residue. As the gate-keeper residue can be easily identified       (Nourseothricin), resulting in pCloneNat1-&#206;&#8221;kinase plasmids. <br /> in most protein kinases (reviewed in ref. 11 and sequoia.ucsf.edu/        The plasmids were amplified in E. coli, purified and linearized <br /> ksd), we expect that this strategy will also work for many other          with restriction enzymes (Table&#194;&#160;S2). The linearized plasmids <br /> protein kinases.                                                          were used to delete the respective kinases in a diploid strain <br />                                                                           (JG11315). Successful deletion was confirmed by colony PCR <br />                     Materials and Methods                                 and tetrad analysis. <br />                                                                              Construction of analog sensitive (as) kinase mutants. The <br /> Strains and general methods. The genotypes of the yeast strains           coding sequences of the kinase genes with their promoter and <br /> used in this study are listed in Table 3. Schizosaccharomyces             terminator regions were PCR-amplified from S. pombe genomic <br /> pombe strains were maintained and grown using standard condi-             DNA using the forward and reverse primers, kinase promotor <br /> tions.22 The transformation of S. pombe was performed using the           and kinase terminator, respectively (Table S2). The PCR prod- <br /> lithium acetate method as previously described in reference&#194;&#160;23.           ucts were cloned into a pCloneHyg1 vector (EF101286) carry- <br /> Procedures for Hoechst staining (fluorescence microscopy) and             ing drug resistance markers for E. coli (Ampicillin) and S.&#194;&#160;pombe <br />  <br />  <br /> www.landesbioscience.com	                                         Cell Cycle	                                                                         3529 <br />   Table&#194;&#160;2. Sensitivity of analog-sensitive kinase mutants to various ATP analogs <br />                 Sensitivity to inhibitors [&#206;&#188;M] <br />                           1-NM-PP1                 2-NA-PP1       9-CH-ANPUR        9-TOL-ANPUR         2,3-DMP-PP1       Bn-PP1      3-MB-PP1         3-BrB-PP1 <br />   ark1-as3                    5.0                      5.0             n.s.               40.0               5.0            10.0          10.0            10.0 <br />   cdc2-as1                    0.5                      5.0             5.0                5.0                5.0             5.0          5.0              5.0 <br />   cdk9-as1                   10.0                     20.0             20.0               10.0               n.s.            n.s.         10.0            10.0 <br />   cka1-as1                   n.s.                      n.s.            5.0                30.0               n.s.            n.s.         n.s.            40.0 <br />   crk1-as1                   n.s.                      n.s.            n.s.               n.s.               n.s.            n.s.         n.s.            30.0 <br />   ksg1-as1                    5.0                     20.0             20.0               10.0               n.s.           20.0          5.0              5.0 <br />   nak1-as1                   n.s.                      n.s.            n.s.               n.s.               n.s.            n.s.         n.s.            40.0 <br />   orb6-as1                   10.0                     10.0             20.0               10.0               10.0           10.0          10.0             5.0 <br />   pat1-as2                   10.0                     20.0             n.s.               10.0               30.0           30.0          n.s.            30.0 <br />   plo1-as1                   30.0                      n.s.            n.s.               n.s.               n.s.            n.s.         40.0            10.0 <br />  ppk37-as1                   30.0                      n.s.            n.s.               n.s.               n.s.            n.s.         30.0            30.0 <br />   prp4-as2                   n.s.                      n.s.            n.s.               n.s.               n.s.            n.s.         n.s.            30.0 <br />   shk1-as2                   n.s.                      n.s.            n.s.               30.0               n.s.            n.s.         n.s.            20.0 <br />   sid2-as1                   n.s.                      n.s.            n.s.               n.s.               n.s.            n.s.         n.s.            40.0 <br />  n.s., not sensitive. <br />  <br />  <br />  <br />  <br />  Figure 3. Sensitivity of cells expressing analog-sensitive kinase alleles to various inhibitors. Serial dilutions of wild-type cells or cells expressing indi- <br />  cated analog-sensitive kinase alleles were spotted on YES plates containing or lacking the indicated inhibitors and grown at 32&#194;&#176;C for 3&#226;&#8364;&#8220;4 days, with <br />  the exception of the temperature-sensitive mutants cdc2-as1 and pat1-as2, which were grown at 25&#194;&#176;C for 4 days. Concentrations of inhibitors were as <br />  described in the Table 2. <br />  <br />  <br />  <br /> (Hygromycin&#194;&#160;B), resulting in integrative pCloneHyg1-kinase                        unique restriction enzyme that cuts within the promoter region <br /> plasmids. The plasmids were amplified in E. coli, purified and                    of the kinase gene (Table S2) and transformed into a diploid <br /> sequenced to exclude the clones with PCR-introduced mutations.                    strain heterozygous for the deletion of the respective kinase. The <br /> To create mutant strains carrying conditional analog-sensitive                    selected transformants were sporulated at 25&#194;&#176;C, and haploids car- <br /> alleles, we employed site directed mutagenesis (QuikChangeII,                     rying analog-sensitive kinase alleles were selected based on resis- <br /> Site Directed Mutagenesis Kit) and mutated the predicted gate-                    tance to Nourseothricin and Hygromycin. Correct integration of <br /> keeper residues to small amino acid residues (Table 1, sequoia.                   mutant kinase alelles was confirmed by colony PCR. The pres- <br /> ucsf.edu/ksd).11 The sequences of primers used for mutagen-                       ence of mutant alleles in the selected haploid strains was verified <br /> esis are listed in Table&#194;&#160;S3. The resulting plasmids pCloneHyg1-                   by sequencing of the PCR-amplified kinase alleles from genomic <br /> kinase-as1 and pCloneHyg1-kinase-as2 were linearized with a                       DNA. <br />  <br />  <br /> 3530	                                                                     Cell Cycle	                                                            Volume 10 Issue 20 <br />   Table&#194;&#160;3. Yeast strains used in this study <br />            Strain                                                                                   Genotype <br />           JG15459                                                                                        h- <br />           JG15401                                                                 h+ ark1-as(L166A, S229A, Q28R, Q176R)-HphR <br />           JG11315                                                        h+/h- leu1&#226;&#8364;&#8220;32/leu1&#226;&#8364;&#8220;32 ura4-D18/ura4-D18 ade6-M210/ade6-M216 <br />           JG15442                                                    h leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 cdc2::ClonNatR cdc2-as(F84G)-HphR <br />                                                                          - <br />  <br />  <br />           JG15172                                                   h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 cdk9::ClonNatR cdk9-as(T120G)-HphR <br />           JG15446                                                   h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 cka1::ClonNatR cka1-as(F117G)-HphR <br />           JG15452                                                              h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 crk1-as(L87G)-HphR <br />           JG15433                                                   h leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 ksg1::ClonNatR ksg1-as(L177G)-HphR <br />                                                                          - <br />  <br />  <br />           JG15448                                                             h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 nak1-as(M88G)-HphR <br />           JG15439                                                   h leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 orb6::ClonNatR orb6-as(M170G)-HphR <br />                                                                      - <br />  <br />  <br />           JG15404                                                    h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 pat1::ClonNatR pat1-as(L95G)-HphR <br />           JG15403                                                    h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 pat1::ClonNatR pat1-as(L95A)-HphR <br />           JG15431                                                    h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 plo1::ClonNatR plo1-as(L117G)-HphR <br />           JG16103                                                  h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M216 ppk37::ClonNatR ppk37-as(M537G)-HphR <br />           JG16101                                                  h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 ppk37::ClonNatR ppk37-as(M537A)-HphR <br />           JG15934                                                   h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 prp4::ClonNatR prp4-as(F238A)-HphR <br />           JG15927                                                   h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 shk1::ClonNatR shk1-as(M460A)-HphR <br />           JG15450                                                             h- leu1&#226;&#8364;&#8220;32 ura4-D18 ade6-M210 sid2-as(M285G)-HphR <br />  <br />  <br />    Analysis of sensitivity of kinase mutants to ATP analogs. To                                                               <br /> test if the strains with a mutated gate-keeper residue are sensitive                        <br /> to ATP analogs, the dilution spot test was used. Serial dilutions                  <br /> of cells were spotted on YES plates supplemented with various                                 <br /> ATP analogs or DMSO, and grown at 32&#194;&#176;C for 3&#226;&#8364;&#8220;4 d, with the                                 <br /> exception of the temperature-sensitive mutants, cdc2-as1 and                <br /> pat1-as2, which were grown at 25&#194;&#176;C for 4 d. The determination                              <br /> of growth rates and cellular morphology and FACS analysis were                       <br /> performed with cells growing in liquid YES medium supple-                                     <br /> mented with various ATP analogs.25 <br />                                                                                                 <br />                 Disclosure of Potential Conflicts of Interest                            <br /> No potential conflicts of interest were disclosed.                                           <br /> </body></html>