<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> Dependence RNase E Essentiality Escherichia coli <br /> Masaru Tamura,a Christopher J. Moore,a Stanley N. Cohena,b <br /> Departments Geneticsa Medicine,b School Medicine, Stanford University, Stanford, California, USA <br />  <br /> Escherichia coli cells normally require RNase E activity form colonies (colony-forming ability [CFA]). The CFA-defective phe- <br /> notype cells lacking RNase E partly reversed overexpression related endoribonuclease RNase G <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> of <br /> the gene encoding RNA helicase DeaD. We carbon source utilization rne deaD doubly mutant bacteria dif- <br /> fers rne&#226;&#180;&#8482; cells cells mutated deaD loss rne function bacteria limits <br /> conversion glycolytic pathway product phosphoenolpyruvate tricarboxylic acid (TCA) cycle intermediate oxalo- <br /> acetic acid. We mechanism underlying effect reduced production enzyme phosphoenolpyruvate car- <br /> boxylase (PPC) adventitious overexpression PPC, facilitates phosphoenolpyruvate utilization connects <br /> the glycolytic pathway TCA cycle, restored CFA rne deaD mutant bacteria cultured carbon sources otherwise <br /> were unable sustain growth. We bacteria producing full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> RNase E, allows formation degra- <br /> dosomes, nutritional requirements different cells supplied N-terminal catalytic region RNase <br /> E mitigation RNase E deficiency overexpression related RNase, RNase G, affected carbon source. Our <br /> results reveal previously unsuspected effects RNase E deficiency degradosome formation <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> utilization E. coli <br /> cells. <br />  <br />  <br />  <br />  <br /> R    Nase E initially discovered Escherichia coli endori- <br />      bonuclease processes 9S rRNA (1) later shown <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> have multifunctional role processing various other <br />                                                                                       ditions: colony formation observed cells grown <br />                                                                                       Luria-Bertani (LB) medium grown on <br />                                                                                       M9 medium containing glucose sole carbon source. This <br /> structural RNAs (1, 2, 3, 4), global degradation mRNA (5, 6,                   finding prompted investigate nutritional factors affecting the <br /> 7, 8), control plasmid DNA replication (9), matu-                      growth &#226;&#338;&#172;rne bacteria, ability cells containing muta- <br /> ration/processing variety small catalytic RNAs (9, 10).                       tions deaD gene (26), encodes RNA helicase (28), <br /> Commonly, cleavage targeted RNA RNase E fol-                         propagate enabled absence comple- <br /> lowed digestion RNase E-generated fragments ri-                        menting RNase. The results report combina- <br /> bonucleases tightly coupled events (11). DNA sequence analysis                     tion glycolytic pathway sugar plus tricarboxylic acid (TCA) <br /> has predicted orthologs E. coli RNase E exist dozens                    cycle sugar needed propagation &#226;&#338;&#172;rne deaD doubly mutant <br /> of evolutionarily disparate bacterial species (12, 13). The ribonu-                   bacteria, requirement observed cells mu- <br /> cleolytic activity RNase E resides N-terminal half, N-Rne                   tated deaD alone. They indicate, shown Western <br /> (14), C-terminal half (CTH) includes binding sites                    blotting confirmed genetically, RNase E-deficient cells <br /> several proteins form multicomponent complex termed                        defective production enzyme phosphoenolpyruvate <br /> &#226;&#8364;&#339;degradosome&#226;&#8364;? (15, 16). Although degradosome formation                         carboxylase (PPC), converts phosphoenolpyruvate ox- <br /> essential E. coli growth (14, 17, 18), deletion CTH                    aloacetic acid connects glycolytic pathway TCA <br /> been reduce rate decay RNase E substrates                     cycle, establishing previously unsuspected role PPC RNase <br /> (19, 20).                                                                             E essentiality. <br />     In E. coli, actions RNase E essential colony-form- <br />                                                                                       MATERIALS AND METHODS <br /> ing ability (CFA) (21), transfer liquid cultures rne(Ts) <br /> mutant bacteria nonpermissive temperature prevents normal                        Strains plasmids. The strains used study listed Table 1. <br />                                                                                       MT4000 constructed method described Lee et al. (6). <br /> cell division results formation elongated filamentous <br />                                                                                       MT4001, MT4002, MT4003, MT2105, MT2005 constructed by <br /> structures containing connected cell bodies (22). Multiple studies                    transformation MT4000 pNRNE4, MT4000 pRNG2S, MT4000 <br /> aimed elucidating basis RNase E essentiality                      pRNG2S&#226;&#338;&#172;H, CM2100 pRNG2S&#226;&#338;&#172;H, MT2000 pRNG2S&#226;&#338;&#172;H, <br /> carried (6, 23, 24, 25, 26). While CFA restored rne                     respectively. MT2113 obtained P1 transduction deaD::Tn10 into <br /> deletion mutants overproduction mutation RNase G (6,                         CM2100, colonies selected tetracycline (Tc) plates containing <br /> 24, 25, 27), RNase E paralog (2, 4) structurally resembles <br /> the amino-terminal catalytic region RNase E (14), cells <br /> grow slowly rne&#226;&#171;&#185; bacteria retain filamentation                         Received 29 August 2012 Accepted 17 December 2012 <br /> <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> characteristic RNase E-deficient mutants liquid                          Published ahead print 28 December 2012 <br /> culture (25). The filamentation RNase G-overproducing &#226;&#338;&#172;rne                            Address correspondence Stanley N. Cohen, sncohen@stanford.edu. <br /> mutants reversed overexpression FtsZ; however, FtsZ-                            Supplemental material article http://dx.doi.org/10.1128 <br /> overproducing cells remain defective rate division (25).                     /JB.01558-12. <br />     During recently reported investigations insertion mutations                       Copyright &#194;&#169; 2013, American Society Microbiology. All Rights Reserved. <br /> that restore CFA E. coli cells lacking RNase E (26), observed                      doi:10.1128/JB.01558-12 <br /> that <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>survival</span> &#226;&#338;&#172;rne cells dependent culture con- <br />  <br />  <br /> March 2013 Volume 195 Number 6                                            Journal Bacteriology   p. 1133&#226;&#8364;&#8220;1141                                           jb.asm.org       1133 <br />  Tamura et al. <br />  <br />  <br />  <br /> TABLE 1 Bacterial strains plasmids <br /> Strain plasmid           Description                                                                       Reference(s) source <br /> Strains <br />    MG1655                   ilvG rfb-50 rph-1 fnr-267 eut                                                     E. coli Genetic Stock Center (CGSC 6300) <br />    MG165D                   Same MG1655 carrying deaD::Tn10 (clone 2)                                  This study <br />    NCM3416                  Same MG1655 carrying rph&#226;&#171;&#185; zib-207::Tn10                                40 <br />    MT4000                   Same NCM3416 carrying rne::cat pBAD-RNE                                This study <br />    MT4001                   Same MT4000 carrying pNRNE4                                                This study <br />    MT4002                   Same MT4000 carrying pRNG2S                                                This study <br />    MT4003                   Same MT4000 carrying pRNG2S&#226;&#338;&#172;H                                              This study <br />    MT2000                   lacZ43(Fs) LAM&#226;&#171;&#186; relA1 spoT1 thi-1 rne::cat; carries pBAD-RNE                      25 <br />    MT2005                   Same MT2000 carrying pRNG2S&#226;&#338;&#172;H                                              This study <br />    MT2013                   Same MT2000 carrying deaD::Tn10 (clone 2)                                  26 <br />    CM2100                   Same MG1655 carrying rne::cat pBAD-RNE                                 This study <br />    MT2103                   Same MG1655 carrying rne::cat pLAC-RNE2                                This study <br />    MT2105                   Same CM2100 carrying pRNG2S&#226;&#338;&#172;H                                              This study <br />    MT2113                   Same CM2100 carrying deaD::Tn10 (clone 2)                                  This study <br />    MT2114                   Same MT2113 carrying ASKA-ppc                                              This study <br />    MT2115                   Same MT2113 carrying pACYC177                                              This study <br />    MT2116                   Same MT2113 carrying pMT177Z                                               This study <br />    MT2117                   Same MT2113 carrying pCA24N(Ap)                                            This study <br />  <br /> Plasmids <br />   pBAD-RNE                  pSC101 ori Kmr, rne control PBAD                                         6, 25 <br />   pLAC-RNE2                 pSC101 ori Apr, rne control lacUV5 promoter                              41 <br />   pNRNE4                    P15A ori Apr, His-tagged N-rne control lacUV5 promoter                   25 <br />   pRNG2                     P15A ori Apr, His-tagged rng (also N-terminally extended)                   Gift Kangseok Lee <br />                               control lacUV5 promoter <br />   pRNG2S                    P15A ori Apr, His-tagged rng control lacUV5 promoter                     Gift Kangseok Lee <br />   pRNG2S&#226;&#338;&#172;H                  P15A ori Apr, natural short-form rng control lacUV5 promoter             Gift Kangseok Lee <br />   pCA24N(Ap)                ColE1 ori Apr                                                                     26 <br />   ASKA-ppc                  ColE1 ori Apr, ppc control PT5-lac                                       This study <br />   pACYC177                  P15A ori Kmr Apr                                                                  42 <br />   pMT177Z                   P15A ori Apr, ftsZ control natural promoter                              25 <br />  <br />  <br />  <br />  <br /> 0.1% arabinose (0.1% Ara) induce adventitious production RNase E      CTAGTGAATTCAGTGGTGGTGGTGGTGGTGCATCATTACGACGT <br /> in &#226;&#338;&#172;rne strain (6, 26). P1(deaD::Tn10) prepared MT2013      GCAAACT-3=). The pRNG2S&#226;&#338;&#172;H, pRNG2S, pRNG2 plasmids were <br /> described previously (26). MT2103 constructed replacing pBAD-        kindly provided Kangseok Lee (C. J. Moore, H. Go, K. Lee, S. N. <br /> RNE CM2100 pLAC-RNE2 method described Lee et al.          Cohen, unpublished data). Construction ASKA plasmid Ap <br /> (6). To test effects PPC overexpression, introduced      resistance gene described previously (26). <br /> modified ASKA library plasmid (29) expressing PPC (i.e., ASKA-ppc)           Media culture conditions. LB medium M9 minimal medium <br /> an vector [pCA24N(Ap)] MT2113 strain, generating         (30) containing 0.1 mM CaCl2, 1 mM MgSO4, appropriate carbon <br /> MT2114 MT2117 strains, respectively.                                    sources following concentrations used: D-(&#226;&#171;&#185;)-glucose, 0.2%; <br />     pRNG2S&#226;&#338;&#172;H, expresses natural form RNase G (i.e.,         D-mannitol, 0.2%; D-(&#226;&#171;&#186;)-fructose, 0.2%; glycerol, 0.5%; sodium pyru- <br /> N-terminal C-terminal extension), constructed ligating ClaI-      vate, 0.2%; sodium acetate, 0.2%; succinic acid, 0.5%; L-(&#226;&#171;&#186;)-malic acid, <br /> and EcoRI-digested PCR products encoding C-terminal region           0.5%; fumaric acid, 0.5%; Casamino Acids (CAA) (see BD Bionutri- <br /> RNase G lacking hexahistidine tag pRNG2S using restric-     ents Technical Manual), 0.05%. Where indicated, LB components the <br /> tion enzyme sites. The PCR primers used rng-948F (5=-TGTCGGTC          following concentrations used: yeast extract (Bacto Yeast Extract; <br /> ATCGCAATCT-3=) 3=-rng-no-His (5=-AGACTAGTGAATTCACAT                     BD), 5 g/liter; tryptone (Bacto Tryptone; BD), 10 g/liter; sodium <br /> CATTACGACGTCAAACTG-3=). pRNG2S, expresses C-terminal                  chloride (Invitrogen), 10 g/liter. Where indicated, antibiotics used at <br /> His-tagged RNase G lacking N-terminal extension, constructed        following concentrations: ampicillin, 50 &#226;?&#174;g/ml; chloramphenicol, 10 <br /> by ligating NdeI- NsiI-digested PCR product encoding             &#226;?&#174;g/ml; kanamycin, 20 &#226;?&#174;g/ml; tetracycline, 5 &#226;?&#174;g/ml; streptomycin 50 <br /> N-terminal region RNase G containing natural N-terminal se-          &#226;?&#174;g/ml. For experiments, agar plates freshly prepared day <br /> quence (ATGACGGCTGAA. . .) pRNG2 using restriction            used (never stored 4&#194;&#176;C overnight bench), bacteria were <br /> enzyme sites. The PCR primers used 5=-rng-nat (5=-ATACATATG            streaked plates containing 0.1% L-arabinose glycerol stock <br /> ACGGCTGAATTGTTAGTA-3=) rng-595R (5=-AGACGCGTTTCA                        (containing 40% glycerol) incubated 37&#194;&#176;C overnight, colonies <br /> GATAAG-3=). pRNG2, expresses C-terminal His-tagged RNase G            picked inoculations. <br /> lacking N-terminal extension, constructed ligating NotI-         Plating experiments. For plating experiments, cells freshly <br /> SpeI-digested PCR DNA encoding region rng containing              grown single colony (never grown overnight) mid-log phase <br /> extended N-terminal sequence (ATGAGAAAAGGGATAAAC)                      (optical densities 600 nm [OD600] ranged 0.3 0.6) 37&#194;&#176;C LB <br /> pNRNE4 (25) using restriction enzyme sites. The PCR primers        medium containing indicated supplements antibiotics. Cells were <br /> used 5=-rng-2 (5=-GGATCCGCGGCCGCTTTAAGAAGGAGATAT                       diluted 10&#226;&#171;&#186;4 10&#226;&#171;&#186;5, 100 &#226;?&#174;l diluted culture spread LB or <br /> ACATATGAGAAAAGGGATAAAC-3=) 3=-rng-His (5=-GTCTAGA                       M9 plates containing carbon sources, supplements, antibiotics. The <br />  <br />  <br /> 1134   jb.asm.org                                                                                                                 Journal Bacteriology <br />                                                                                                                                            RNase E Essentiality E. coli <br />  <br />  <br />  <br />  <br /> FIG 1 Effects individual LB components CFA &#226;&#338;&#172;rne deaD::Tn10 strain. (A) A culture MT2113 LB containing 0.1% arabinose, induces <br /> adventitious expression plasmid-borne rne, spread LB containing lacking 0.1% arabinose, LB containing additional components M9-glucose <br /> but lacking inducer, LB containing 0.2% glucose arabinose. (B) A culture MT2113 spread M9-glucose lacking additive or <br /> supplemented yeast extract, tryptic digest casein (labeled tryptone), NaCl, absence arabinose. (C) Identification amino acid <br /> requirements CFA &#226;&#338;&#172;rne deaD::Tn10 strain cultured M9-glucose. A culture MT2113 spread M9-glucose containing variety amino acid <br /> mixtures, absence arabinose. A mixture glycolytic pathway-derived amino acids, i.e., Ser, Cys, Gly, Trp, Tyr, Phr, Ala, Val, Leu, Ile, His, defined <br /> as &#226;&#8364;&#339;glycolytic a.a.,&#226;&#8364;? mixture TCA cycle-derived amino acids, i.e., Lys, Met, Thr, Asn, Asp, Glu, Gln, Pro, Arg, His, defined &#226;&#8364;&#339;TCA a.a.&#226;&#8364;? The <br /> concentration amino acid 160 &#226;?&#174;M. Plates scanned incubation 2 days (A B) 4 days (C) 37&#194;&#176;C. <br />  <br />  <br />  <br /> plates incubated 37&#194;&#176;C 2 7 days depending experiment            VersaDoc model 1000 imaging (Bio-Rad), corrected loading <br /> and scanned, colony number determined. Different <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                    based S1 protein control difference total <br /> combinations resulted different rates colony growth, in-             protein loaded MT2114 strains. Values then <br /> dicated, incubation period extended 2 days enable                normalized relative RNase E PPC band CM2100 cells. <br /> colonies better visualized. Incubation conditions experi- <br /> ment described figure legends. The absence detectable colony           RESULTS AND DISCUSSION <br /> formation 7 days incubation defined CFA-negative                   RNase E required glycolysis E. coli. We recently reported <br /> (CFA&#226;&#171;&#186;) phenotype. Colony-forming efficiency (CFE) determined                  Tn10 <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>insertion mutation</span> deaD gene, encodes <br /> dividing colony number observed indicated growth condi-                RNA helicase, enables colony formation E. coli N3433 cells <br /> tion colony number observed condition                  lacking RNase E function (26). During experiments, ob- <br /> arabinose-inducible RNase E gene expressed (26).                                 served ability affected carbon source included <br />     Western blotting. Cultures CM2100, MT2113, MT2114,                         medium. While N3433 E. coli strain RNase E <br /> MT2117 cells grown overnight 37&#194;&#176;C LB plates containing 0.1% <br />                                                                                      discovered widely employed study <br /> arabinose appropriate antibiotics. Randomly selected colonies <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> plate suspended LB diluted OD600 0.001             RNase E essentiality, strain contains multiple mutations <br /> fresh LB containing appropriate selective antibiotics,           potentially affect nutritional requirements [i.e., lacZ43(Fs), <br /> 0.1% arabinose (as indicated) 100 &#226;?&#174;M isopropyl-&#226;?&#164;-D-thiogalac-               relA1, spoT1, thi-1] consequently affect ability even <br /> topyranoside (IPTG) (for strains MT2114 MT2117). The cultures                    rne&#226;&#171;&#185; cells form colonies efficiently carbon sources (see <br /> were grown approximate OD600 0.8, minimum 6 h,              Fig. S1 supplemental material); chose further <br /> then harvested prepared loading polyacrylamide gel. The               investigate nutritional effects loss RNase E function E. <br /> cells diluted OD600 0.01 &#226;?&#174;l water SDS loading               coli strain MG1655, prototrophic loci, con- <br /> buffer boiled. Lysate CM2100, MT2113, MT2117 cells               firming &#226;&#338;&#172;rne deaD derivative MG1655 (i.e., strain <br /> an OD600 0.24 cultured inducing agents indicated loaded               MT2113) forms colonies LB 100% colony-forming effi- <br /> alongside lysate MT2114 cells OD600 0.1, proteins <br />                                                                                      ciency relative cells producing RNase E (Fig. 1A). <br /> were separated Criterion XL 3 8% Tris-acetate polyacrylamide gel <br /> (Bio-Rad). The samples transferred 0.1-&#226;?&#174;m nitrocellulose mem-                  As seen Fig. 1, absence arabinose induce <br /> brane probed anti-PPC (Agrisera rabbit polyclonal), anti-RNase              plasmid-borne RNase E, &#226;&#338;&#172;rne deaD MT2113 strain formed <br /> E (mouse monoclonal), anti-S1 (rabbit polyclonal) antibodies,            colonies LB medium M9 medium containing glu- <br /> order, intervening membrane stripping. The numbers provided rep-                cose mixture amino acids carbon sources; however, no <br /> resent densitometric measurements band determined                  MT2113 colonies observed yeast extract mixture <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                        jb.asm.org 1135 <br />  Tamura et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Effect carbon source combination CFA &#226;&#338;&#172;rne deaD::Tn10 strain. A culture MT2113 LB spread carbon sources indicated. The <br /> presence absence 0.1% arabinose indicated. Plates scanned incubation 37&#194;&#176;C 4 days. <br />  <br />  <br />  <br />  <br /> of amino acids omitted (Fig. 1B). In contrast, absence             cultured M9-glucose medium. No restoration CFA occurred <br /> any amino acid supplement, strain MG165D, MG1655                     absence deaD mutation (data shown), is <br /> derivative containing intact rne gene mutated Tn10 in-                 consistent previously reported ability mutation of <br /> sertion deaD, formed colonies M9 medium containing                       deaD suppress need RNase E E. coli cells (26). To- <br /> any series tested sugars carbon source (see Fig. S2          gether, findings suggest PPC production neces- <br /> supplemental material), arguing failure MT2113                    sary CFA cells lacking RNase E function, sufficient, <br /> form colonies media lacking supplemental amino acids                  CFA-promoting function mediated mutation <br /> supplied glucose sole carbon source results de-                 deaD distinct PPC-related function. Additionally, <br /> fective RNase E function. Testing additives individual amino               adventitious expression PPC (Fig. 4) did result CFA on <br /> acids (Fig. 1C) indicated amino acids specifically synthesized              M9-malate medium, supporting view upregulation of <br /> through glycolytic pathway did enable CFA MT2113                      PPC does restore normality &#226;&#338;&#172;rne deaD mutant cells. <br /> supplied glucose, addition amino acids generated                 During experiments, noted faint &#226;&#8364;&#339;shad- <br /> through TCA cycle did. Moreover, addition malate, <br /> fumarate, succinate, TCA cycle components and <br /> can generate TCA cycle-dependent amino acids, enabled CFA <br /> on M9-glucose (Fig. 2). Together, results indicate the <br /> pathway TCA cycle-dependent amino acid biosynthesis in- <br /> tact &#226;&#338;&#172;rne deaD::Tn10 mutant strain suggest defect <br /> in enzymatic connection glycolytic pathway and <br /> TCA cycle mutant bacteria prevents products glycolysis <br /> from entering TCA cycle. <br />     Functional interactions RNase E PPC. The gly- <br /> colytic pathway TCA cycle connected enzyme phos- <br /> phoenolpyruvate carboxylase (PPC), converts phospho- <br /> enolpyruvate oxaloacetic acid (31). To test hypothesis that <br /> deficiency PPC expression occurs &#226;&#338;&#172;rne deaD bacteria, we <br /> used Western blotting examine effects RNase E expres-                  FIG 3 Effect RNase E expression PPC expression &#226;&#338;&#172;rne strain, &#226;&#338;&#172;rne <br /> sion expression PPC encoded endogenous                   deaD::Tn10 strain, &#226;&#338;&#172;rne deaD::Tn10 strain complemented ASKA- <br /> gene plasmid-borne PPC gene (Fig. 3). As seen, expression               ppc vector. Cultures CM2100, MT2113, MT2117, MT2114 <br />                                                                                  grown 0.1% arabinose induce deplete full-length <br /> of PPC protein highly dependent expression RNase E.                 RNase E pBAD-RNE plasmid (MT2114 MT2117 addition- <br /> When full-length RNase E depleted undetectable levels                  ally grown 100 &#226;?&#174;M IPTG), separated 3 8% Tris-acetate gel, trans- <br /> growth absence arabinose, PPC protein reduced                  ferred nitrocellulose membrane, probed appropriate anti- <br /> by 79 91%. To determine overexpression PPC res-                    bodies presence RNase E, PPC, S1 proteins. Due high level <br />                                                                                  PPC overexpression ASKA-ppc MT2114 induced 100 &#226;?&#174;M <br /> cues CFA bacteria fail produce RNase E, tested                 IPTG, smaller culture loaded lane other <br /> ability MT2114 strain form colonies M9-glucose. As                  strains (as described Materials Methods). Protein expression deter- <br /> seen Fig. 4, overexpression PPC restored CFA bacteria                   mined described Materials Methods. <br />  <br />  <br />  <br /> 1136   jb.asm.org                                                                                                                        Journal Bacteriology <br />                                                                                                                                        RNase E Essentiality E. coli <br />  <br />  <br />  <br />                                                                                   shown). We observed overproduction PPC protein <br />                                                                                   ASKA-ppc plasmid increased RNase E deficiency, <br />                                                                                   contrast observed cells expressing PPC the <br />                                                                                   endogenous gene. As ASKA-ppc plasmid expresses open <br />                                                                                   reading frame PPC gene lacks 5= 3= untranslated <br />                                                                                   regions, result suggests RNase E production a <br />                                                                                   dual role PPC synthesis affecting mRNA synthesis and <br />                                                                                   translation. <br />                                                                                       Interestingly, introduction FtsZ-producing plasmid <br />                                                                                   (pMT177Z), previously shown reverse filamen- <br />                                                                                   tation RNase E-deficient cells (25) (see Fig. S3 supple- <br />                                                                                   mental material), &#226;&#338;&#172;rne deaD::Tn10 bacteria enabled CFA on <br />                                                                                   M9-malate M9-glucose absence amino acids <br />                                                                                   synthesis dependent TCA cycle (Fig. 5). As the <br />                                                                                   genes expressed ftsQAZ operon known con- <br />                                                                                   tribute glycolysis gluconeogenesis, effect FtsZ the <br />                                                                                   nutritional requirements RNase E-deficient cells suggests that <br />                                                                                   additional step glycolytic gluconeogenetic pathway <br />                                                                                   (32, 33) impaired RNase E-deficient bacteria. <br /> FIG 4 Effect PPC complementation CFA &#226;&#338;&#172;rne deaD::Tn10 strain                Role degradosome carbon source utilization E. <br /> cultured M9-glucose M9-malate. Cultures MT2114 spread on <br /> M9-glucose 0.5% malate, M9-glucose, M9-malate without <br />                                                                                   coli. The C-terminal half RNase E functions binding site for <br /> 100 &#226;?&#174;M IPTG absence arabinose. Plates scanned incuba-        multiple proteins (15); glycolytic pathway <br /> tion 37&#194;&#176;C 7 days.                                                          enzyme enolase, converts 2-phosphoglycerate phos- <br />                                                                                   phoenolpyruvate (34), cellular substrate PPC. <br />                                                                                   Interaction RNase E enolase normally required <br /> ows&#226;&#8364;? colonies appeared following plating bacteria                propagation E. coli, N-terminal catalytic region of <br /> on M9-malate plates (see Fig. 5), presumed minute                      RNase E (N-Rne), lacks enolase binding site (35), is <br /> colonies did increase size incubation                   sufficient colony formation rne mutant bacteria (14). As <br /> upon replating did result colony formation (data                       mannitol glucose upstream glycolytic pathway <br />  <br />  <br />  <br />  <br /> FIG 5 Effect FtsZ complementation CFA &#226;&#338;&#172;rne deaD::Tn10 strain M9-glucose M9-malate. Cultures MT2115 MT2116 spread M9 <br /> plates containing glucose and/or malate, indicated, containing lacking 0.1% arabinose. Plates scanned incubation 37&#194;&#176;C 4 days or, where <br /> indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                    jb.asm.org 1137 <br />  Tamura et al. <br />  <br />  <br />  <br />  <br /> FIG 6 (A B) Growth wild-type E. coli (A) full-length RNase E-complemented &#226;&#338;&#172;rne strain (B) indicated carbon sources. Cultures NCM3416 <br /> or MT2103 spread LB M9 plates indicated (10 &#226;?&#174;M IPTG MT2103). (C) Effect RNase E degradosome formation growth E. coli on <br /> different carbon sources. Cultures MT4001 spread LB M9 plates containing lacking 0.1% arabinose indicated, 100 &#226;?&#174;M IPTG added <br /> as indicated. Plates scanned incubation 37&#194;&#176;C 2 days and, indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> step requires enolase (i.e., conversion mannitol pyru-             (pNRNE4)] did form colonies carbon sources <br /> vate) combinations glucose plus pyruvate mannitol plus              utilized effectively full-length RNase E (i.e., fumarate <br /> pyruvate did restore CFA RNase-E deficient cells (Fig. 2),             succinate), colony growth certain carbon <br /> the sugar-related defect CFA RNase E deficiency nec-             sources occurred slowly observed &#226;&#338;&#172;rne <br /> essarily independent enolase.                                              cells complemented trans plasmid expressing full-length <br />     As PPC production impaired RNase E-deficient bacteria               RNase E (Fig. 6C Table 2). Thus, degradosome forma- <br /> and component RNase E-based degradosome generates                    tion necessary utilization glucose E. coli (Fig. 6C), <br /> the cellular substrate PPC (i.e., phosphoenolpyruvate), hy-            absence scaffold region enzyme affected utiliza- <br /> pothesized presence scaffold region RNase E              tion sugars. It noted overproduction of <br /> affect carbon source utilization E. coli cells. To test           full-length RNase E &#226;&#338;&#172;rne strain (MT2103) did enable <br /> notion, wild-type bacteria deaD&#226;&#171;&#185; rne deletion mutant strain             formation colonies M9 medium containing 0.2% lactose, <br /> complemented N-Rne full-length Rne tested               wild-type NCM3416 bacteria able form colonies <br /> their ability grow M9 plates supplemented specific sug-            (data shown). The basis effect currently known. <br /> ars. As shown Fig. 6, wild-type bacteria (Fig. 6A)                  Complementation &#226;&#338;&#172;rne bacteria different forms of <br /> &#226;&#338;&#172;rne strain carrying plasmid expressing full-length Rne (Fig. 6B)           RNase G results differential sugar utilization. Lee et al. re- <br /> formed normal-size colonies tested carbon sources               ported overproduction His-tagged RNase G restores CFA <br /> acetate 2 days culture M9 plates. However, &#226;&#338;&#172;rne bacte-            &#226;&#338;&#172;rne bacteria LB plates LB liquid medium (6). <br /> ria carrying plasmid expressing N-Rne [i.e., MT4001                         Subsequent investigations shown complementation of <br />  <br />  <br /> TABLE 2 Summary plating experiments <br />                                 Resulta plating on: <br />                                                    M9 with: <br /> Strain                          LB medium          Glucose    Glycerol          Pyruvate       Acetate              Malate      Fumarate         Succinate <br /> NCM3416                         ND                 &#197;&#8217;          &#197;&#8217;                 &#197;&#8217;              o                    &#197;&#8217;           &#197;&#8217;                &#197;&#8217; <br /> MT4001(pNRNE4)                  &#197;&#8217;                  &#197;&#8217;          o                 o              &#226;&#171;&#187;                    o           &#226;&#171;&#187;                &#226;&#171;&#187; <br /> MT4002(pRNG2S)                  &#197;&#8217;                  &#197;&#8217;          46% CFE, slow     o              37% CFE, slow        &#197;&#8217;           &#197;&#8217;                &#197;&#8217; <br /> MT2105(pRNG2S&#226;&#338;&#172;H)                &#197;&#8217;                  &#226;&#171;&#187;          ND                ND             ND                   &#226;&#171;&#187;           ND               ND <br /> MT2113 (deaD::Tn10)             o                  &#226;&#171;&#187;          ND                o              ND                   &#226;&#171;&#187;           ND               ND <br /> a <br />     &#197;&#8217;, 100% CFE; o, 100% CFE slow growth; &#226;&#171;&#187;, growth. <br />  <br />  <br />  <br /> 1138     jb.asm.org                                                                                                                  Journal Bacteriology <br />                                                                                                                                     RNase E Essentiality E. coli <br />  <br />  <br />  <br />  <br /> FIG 7 Growth natural-form RNase G-complemented &#226;&#338;&#172;rne strain LB variety carbon sources. (A B) Cultures MT2105 (A) MT2005 (B) <br /> were spread LB M9 plates containing lacking 0.1% arabinose (as indicated), 1 mM IPTG added indicated. Plates scanned incubation <br /> at 37&#194;&#176;C 2 days (A) 4 days (B). (C) Growth &#226;&#338;&#172;rne strain expressing His-tagged RNase G. Cultures MT4002 spread LB M9 plates containing <br /> or lacking 0.1% arabinose indicated carbon sources, 1 mM IPTG added indicated. Plates scanned incubation 37&#194;&#176;C 2 days and, <br /> where indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> rne-defective mutants RNase G facilitated unnatural ex-                (37% CFE) (Fig. 7C). Complementation natural form of <br /> tensions termini RNase G protein, conse-               RNase G observed, LB, N3433-derived <br /> quently concluded RNase G function constrained                   &#226;&#338;&#172;rne strain (Fig. 7B), noted differs MG1655 <br /> termini enzyme (24). In instances, mutations                derivatives carbon sources needed CFA, suggesting that <br /> catalytic region RNase G reported facilitate com-               strain differences account disparate results reported in <br /> plementation rne mutants (27). Given information                    investigations complementation RNase E deficiency by <br /> the results described above, interested learning             RNase G overexpression (6, 24, 27, 36, 37). <br /> whether &#226;&#338;&#172;rne cells complemented viability overproduction                      Proposed mechanism role glycolytic pathway <br /> of natural versus elongated forms RNase G different                 TCA cycle RNase E essentiality. The results reported here <br /> nutritional requirements.                                                       indicate absence RNase E E. coli viable a <br />     As shown Fig. 7, overproduction natural form                   second-site Tn10 insertion mutation deaD (26) results defec- <br /> RNase G restored CFA introduced MG1655-derived                     tive production PPC, links glycolysis TCA cycle <br /> &#226;&#338;&#172;rne strain plated LB medium failed enable CFA                   (Fig. 8) (31). We conclude deficiency enzyme results <br /> cells plated M9 containing tested sugar              inability cells use glycolytic pathway carbon sources for <br /> combination glucose malate (Fig. 7A). In contrast, His-                  synthesis amino acids generated TCA cycle. Conse- <br /> tagged RNase G enabled CFA 100% efficiency, LB                 quently, rne deaD mutant cells failed grow solely on <br /> but M9 medium containing glucose, pyruvate, malate,                     glucose glycolytic pathway intermediates, formed <br /> fumarate, succinate, indicating nutritional require-                colonies minimal medium containing sugars the <br /> ments complementation resemble bacteria com-                  cellular connection glycolytic pathway TCA cycle <br /> plemented full-length RNase E (Fig. 7C). However, relative                enhanced adventitious overexpression cloned ppc gene <br /> cells complemented plasmid-encoded RNase E, CFA occurred                     TCA cycle amino acids added adventitiously. How- <br /> in &#226;&#338;&#172;rne cells reduced efficiency His-tagged RNase G com-               ever, defective PPC synthesis nutritional conse- <br /> plementation growth glycerol (46% CFE) acetate                     quence RNase E deficiency, cells overexpressing PPC re- <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                 jb.asm.org 1139 <br />  Tamura et al. <br />  <br />  <br />  <br />                                                                                  3. Ray A, Apirion D. 1980. Cloning gene ribonuclease E, RNA <br />                                                                                     processing enzyme. Gene 12:87&#226;&#8364;&#8220;94. <br />                                                                                  4. Wachi M, Umitsuki G, Shimizu M, Takada A, Nagai K. 1999. Esche- <br />                                                                                     richia coli cafA gene encodes novel RNase, designated RNase G, in- <br />                                                                                     volved processing 5= end 16S rRNA. Biochem. Biophys. Res. <br />                                                                                     Commun. 259:483&#226;&#8364;&#8220; 488. <br />                                                                                  5. Bernstein JA, Khodursky AB, Lin PH, Lin-Chao S, Cohen SN. 2002. <br />                                                                                     Global analysis mRNA decay abundance Escherichia coli sin- <br />                                                                                     gle-gene resolution using two-color fluorescent DNA microarrays. Proc. <br />                                                                                     Natl. Acad. Sci. U. S. A. 99:9697&#226;&#8364;&#8220;9702. <br />                                                                                  6. Lee K, Bernstein JA, Cohen SN. 2002. RNase G complementation rne <br />                                                                                     null mutation identifies functional interrelationships RNase E in <br />                                                                                     Escherichia coli. Mol. Microbiol. 43:1445&#226;&#8364;&#8220;1456. <br />                                                                                  7. Ono M, Kuwano M. 1979. A conditional lethal mutation Escherichia <br />                                                                                     coli strain longer chemical lifetime messenger RNA. J. Mol. Biol. <br />                                                                                     129:343&#226;&#8364;&#8220;357. <br />                                                                                  8. Stead MB, Marshburn S, Mohanty BK, Mitra J, Castillo LP, Ray D, van <br />                                                                                     Bakel H, Hughes TR, Kushner SR. 2011. Analysis Escherichia coli <br /> FIG 8 Proposed model RNase E effects glycolytic pathway TCA          RNase E RNase III activity vivo using tiling microarrays. Nucleic <br /> cycle. PEP, phosphoenolpyruvate.                                                    Acids Res. 39:3188 &#226;&#8364;&#8220;3203. <br />                                                                                  9. Lin-Chao S, Cohen SN. 1991. The rate processing degradation of <br />                                                                                     antisense RNAI regulates replication ColE1-type plasmids vivo. <br /> mained unable form colonies supplied certain                           Cell 65:1233&#226;&#8364;&#8220;1242. <br />                                                                                 10. Lundberg U, Altman S. 1995. Processing precursor catalytic <br /> sugars. Moreover, combination glucose malate did                    RNA subunit RNase P Escherichia coli. RNA 1:327&#226;&#8364;&#8220;334. <br /> not support CFA &#226;&#338;&#172;rne cells lack additional mutation               11. Alifano P, Rivellini F, Piscitelli C, Arraiano CM, Bruni CB, Carlomagno <br /> deaD. Thus, RNase E essentiality attributable single gene               MS. 1994. Ribonuclease E provides substrates ribonuclease P-depen- <br /> or pathway.                                                                         dent processing polycistronic mRNA. Genes Dev. 8:3021&#226;&#8364;&#8220;3031. <br />     Whereas model proposed explains bulk                 12. Lee K, Cohen SN. 2003. A Streptomyces coelicolor functional orthologue of <br />                                                                                     Escherichia coli RNase E shows shuffling catalytic PNPase-binding <br /> findings observed, findings remain unexplained. (i)                    domains. Mol. Microbiol. 48:349 &#226;&#8364;&#8220;360. <br /> A Keio collection E. coli K-12 BW25113 strain deleted ppc                    13. Schein A, Sheffy-Levin S, Glaser F, Schuster G. 2008. The RNase E/G- <br /> (JW3928-1) (38), expected proposed model (see                      type endoribonuclease higher plants located chloroplast and <br /> reference 39), did form colonies M9 medium containing                        cleaves RNA similarly E. coli enzyme. RNA 14:1057&#226;&#8364;&#8220;1068. <br /> either glucose pyruvate sole carbon source. However, rne                14. McDowall KJ, Cohen SN. 1996. The N-terminal domain rne gene <br />                                                                                     product RNase E activity non-overlapping arginine- <br /> deaD doubly mutant bacteria form colonies M9-pyruvate,                       rich RNA-binding site. J. Mol. Biol. 255:349 &#226;&#8364;&#8220;355. <br /> as reported previously (26). This finding consistent             15. Miczak A, Kaberdin VR, Wei CL, Lin-Chao S. 1996. Proteins associated <br /> conclusion inactivation PPC does fully explain                    RNase E multicomponent ribonucleolytic complex. Proc. Natl. <br /> the modified metabolism observed E. coli absence                       Acad. Sci. U. S. A. 93:3865&#226;&#8364;&#8220;3869. <br /> RNase E. (ii) M9 minimal medium containing CAA mixture                     16. Py B, Higgins CF, Krisch HM, Carpousis AJ. 1996. A DEAD-box RNA <br />                                                                                     helicase Escherichia coli RNA degradosome. Nature 381:169 &#226;&#8364;&#8220;172. <br /> the sole carbon source did confer CFA rne deaD mutant                    17. Lopez PJ, Marchand I, Joyce SA, Dreyfus M. 1999. The C-terminal half <br /> cells, contrary expected. As combination pyru-                   RNase E, organizes Escherichia coli degradosome, participates <br /> vate (a glycolytic pathway intermediate) malate (a TCA cycle                    mRNA degradation rRNA processing vivo. Mol. Microbiol. <br /> intermediate) supported CFA (as shown Fig. 2), glycolytic                    33:188 &#226;&#8364;&#8220;199. <br /> pathway (and gluconeogenetic pathway) steps upstream                       18. Ow MC, Liu Q, Kushner SR. 2000. Analysis mRNA decay rRNA <br />                                                                                     processing Escherichia coli absence RNase E-based degrado- <br /> pyruvate principle active cultured rne deaD                     assembly.  <br /> </body></html>