<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Cell. 2008 October 24; 32(2): 259&#226;&#8364;&#8220;275. doi:10.1016/j.molcel.2008.10.002. <br />  <br />  <br />  <br />                            Atomic structure KEOPS complex: ancient protein <br />                            kinase-containing molecular machine <br />                            Daniel Y.L. Mao1,*, Dante Neculai1,*, Michael Downey1,2, Stephen Orlicky1, Yosr Z. Haffani1, <br />                            Derek F. Ceccarelli1, Jenny S.L. Ho3, Rachel K. Szilard1, Wei Zhang1,2, Cynthia S. Ho1, Leo <br />                            Wan1, Christophe Fares4, Sigrun Rumpel4, Igor Kurinov5, Cheryl H. Arrowsmith4, Daniel <br />                            Durocher1,2,&#194;&#167;, Frank Sicheri1,2,&#194;&#167; <br />                            1 Samuel Lunenfeld Research Institute, Mount Sinai Hospital, 600 University Avenue, Toronto, <br />  <br />                            ON, Canada M5G 1X5 <br />                            2   Department Molecular Genetics, University Toronto, Toronto, Ontario, Canada <br />                            3Program Developmental Stem Cell Biology, The Hospital Sick Children University <br />                            Toronto, 555 University Avenue, Toronto, ON, Canada, M5G 1X8 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Ontario Cancer Institute, Department Medical Biophysics, University Toronto, 101 College <br />                            Street, Toronto, ON, Canada, M5G 1L7 <br />                            5Cornell University, Department Chemistry Chemical Biology, NE-CAT, Bldg. 436E, <br />                            Advanced Photon Source, 9700 S. Cass Ave., Argonne, IL, 60439 <br />  <br />                            SUMMARY <br />                                  Kae1 universally conserved ATPase essential gene set bacteria. In archaea <br />                                  eukaryotes, Kae1 embedded protein kinase-containing KEOPS complex. <br />                                  <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>Mutation</span> KEOPS subunits yeast leads striking telomere transcription defects the <br />                                  exact biochemical function KEOPS known. As step elucidating function, we <br />                                  solved atomic structure archaea-derived KEOPS complexes involving Kae1, Bud32, Pcc1 <br />                                  Cgi121 subunits. Our studies suggest Kae1 regulated levels primordial <br />                                  protein kinase Bud32, itself regulated Cgi121. Moreover, Pcc1 appears function as <br />                                  dimerization module, suggesting KEOPS processive molecular machine. <br />                                  Lastly, Bud32 lacks conventional substrate-recognition infrastructure eukaryotic protein <br />                                  kinases including activation segment, Bud32 provide glimpse evolutionary history <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  protein kinase family. <br />  <br />  <br />                            INTRODUCTION <br />                                              The telomere nucleoproteic structure forms end linear chromosomes. In <br />                                              eukaryotes, structure accomplishes critical functions. First, telomeres recruit and <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier Inc. All rights reserved. <br />                            &#194;&#167; <br />                              Address correspondence <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>: Frank Sicheri, Ph.D., Centre Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai <br />                            Hospital, Room 1090, 600 University Avenue, Toronto, ON, CANADA M5G 1X5, Tel: 416-586-4800 ext. 8471, Fax: 416-586-8869, <br />                            sicheri@mshri.on.ca. Daniel Durocher, Ph.D., Centre Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai <br />                            Hospital, Room 1073, 600 University Avenue, Toronto, ON, CANADA M5G 1X5, Tel: 416-586-4800 ext. 2544, Fax: 416-586-8869, <br />                            durocher@mshri.on.ca. <br />                            *These authors contributed equally work <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Mao et al.                                                                                           Page 2 <br />  <br />  <br />                                         activate telomerase, reverse transcriptase polymerizes short TG-rich repeats 3&#226;&#8364;&#178; <br />                                         end chromosomes (Gilson Geli, 2007; Smogorzewska Lange, 2004; Verdun <br />                                         Karlseder, 2007). Second, telomeres protect chromosome ends nucleolytic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         degradation, unscheduled DNA repair activation DNA damage checkpoints. This <br />                                         activity, termed telomere capping, essential cell viability genomic integrity <br />                                         (Feldser et al., 2003; Gilson Geli, 2007; Verdun Karlseder, 2007). <br />  <br />                                         To identify genes modulate telomere capping, screened viable gene deletion <br />                                         collection Saccharomyces cerevisiae suppressors temperature sensitivity of <br />                                         cdc13-1 (Downey et al., 2006), allele telomere capping protein Cdc13. This screen <br />                                         identified Cgi121 protein promotes telomere uncapping triggered cdc13-1. <br />                                         Cgi121 member complex yeast termed KEOPS (or EKC) complex <br />                                         (Downey et al., 2006; Kisseleva-Romanova et al., 2006). In addition Cgi121, has <br />                                         recognizable domains motifs, KEOPS complex composed protein kinase, <br />                                         Bud32; putative ATPase, Kae1 (see below); proteins small size no <br />                                         detectable enzymatic function recognizable domains, Pcc1 Gon7. The KEOPS <br />                                         complex conserved humans, exception Gon7 detected. <br />                                         Interestingly, human orthologues Pcc1 characterized cancer antigens, <br />                                         indicating upregulation KEOPS subunits associated cancer. <br />  <br />                                         In addition promoting telomere uncapping, KEOPS complex profoundly affects <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         telomere <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> homeostasis required cell growth budding yeast. Indeed, <br />                                         deletion BUD32 GON7 results dramatic telomere shortening, drastic reduction in <br />                                         ability telomerase add new telomere sequences DNA double-strand breaks and <br />                                         severe growth defects. In fact, component KEOPS promotes telomere uncapping <br />                                         elongation (Downey et al., 2006; Gatbonton et al., 2006; Shachar et al., 2008) indicating <br />                                         proteins function collectively <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. <br />  <br />                                         In parallel, KEOPS/EKC complex identified novel transcription factor <br />                                         (Kisseleva-Romanova et al., 2006). Components KEOPS nuclear proteins that <br />                                         associate chromatin vivo important inducible gene transcription <br />                                         (Kisseleva-Romanova et al., 2006). It clear KEOPS promotes transcription but <br />                                         KEOPS mutant yeast strains defective recruitment coactivators, SAGA <br />                                         Mediator, target genes. Collectively, studies indicate KEOPS promotes the <br />                                         access nucleases telomerase chromosome end access of <br />                                         transcriptional complexes chromatin. A model explains seemingly disparate <br />                                         observations KEOPS regulates accessibility chromatin, modulating its <br />                                         higher order structure. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Remarkably, KEOPS component Kae1 universally conserved domains of <br />                                         life. A Kae1 homolog present smallest bacterial genome, Mycoplasma genitalium, <br />                                         smallest archaeal genome, Nanoarchaeum equitans, essential gene all <br />                                         bacterial species tested (Arigoni et al., 1998). Kae1 ATPase ASKHA (acetate &amp; <br />                                         sugar kinase/Hsp70/actin) superfamily. As proteins superfamily carry diverse <br />                                         functions, clear specific biochemical function Kae1 possesses although <br />                                         Pasteurella haemolytica Kae1 act O-sialoglycoprotein protease (Abdullah et al., <br />                                         1991) Pyrococcus abyssi Kae1 proposed apurinic (AP) endonuclease (Hecker <br />                                         et al., 2007). Since functions dramatically different, conserved function of <br />                                         Kae1 remains open question. No subunits KEOPS identified in <br />                                         bacteria. However, archaea, nearly complete KEOPS complex identified (see <br />                                         below). This observation suggests Kae1-interacting proteins evolved be <br />                                         regulatory infrastructure modulates activity Kae1. Given importance <br />                                         KEOPS telomere biology gene transcription, evidence KEOPS an <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                           Page 3 <br />  <br />  <br />                                           evolutionarily conserved molecular machine, determined atomic structure KEOPS <br />                                           complexes reconstituted archaeal proteins probed functional significance <br />                                           vitro vivo. Our findings validate importance KEOPS architecture <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           discovered provide framework deciphering functional relationships <br />                                           KEOPS subunits. <br />  <br />                               RESULTS <br />                               Architecture KEOPS complex <br />                                           We identified homologs Pcc1 Cgi121 archaea iterative BLAST searches <br />                                           (Figures S3 &amp; S4). We did identify Gon7 homolog archaea human <br />                                           genome (data shown). We determined underlying architecture KEOPS <br />                                           complex using pull experiments recombinant forms archaeal Kae1 (residues <br />                                           1-328 Methanococcus jannaschii MJ1130); Pcc1 (Pyrococcus furiosus PF2011); Cgi121 <br />                                           (M. jannaschii MJ0187); Bud32 (residues 328-535 MJ1130). Each protein was <br />                                           expressed GST fusion Escherichia coli purified near homogeneity. The GST <br />                                           tags used immobilize proteins glutathione-Sepharose resin cleaved TEV <br />                                           protease, followed purification, isolate untagged protein. GST-Pcc1 immobilized to <br />                                           glutathione resin selectively retained Kae1, Bud32 (Figure 1A) Cgi121 (Figure <br />                                           1B). In contrast, GST-Cgi121 retained Bud32 Kae1 (Figure 1C) GST-Kae1 <br />                                           retained Bud32 (Figure 1D). Lastly, Bud32 Cgi121 bound immobilized GST-Pcc1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           presence Kae1 (Figure 1B) Kae1 bound immobilized GST-Cgi121 only <br />                                           presence Bud32 (Figure 1C). These data indicate KEOPS organized a <br />                                           linear fashion Cgi121 binds Bud32, binds Kae1, turn binds to <br />                                           Pcc1. A model depicting architecture KEOPS complex shown Figure 1E. <br />  <br />                               X-ray NMR structure determination <br />                                           The linear topology KEOPS enabled crystallize solve binary ternary <br />                                           subcomplexes X-ray crystallography means develop atomic model whole <br />                                           KEOPS complex. We solved total structures using X-ray NMR methods (see <br />                                           Table 1 &amp; S1 data collection refinement statistics). Structure based sequence <br />                                           alignments Bud32, Kae1, Pcc1, Cgi121 shown Figures S1&#226;&#8364;&#8220;5. A composite <br />                                           model monomer KEOPS complex generated subcomplex structures is <br />                                           shown Figure 1F. <br />  <br />                                           The following sections structure functional characterizations four <br />                                           individual KEOPS subunits followed characterizations binary interacting <br />                                           pairs KEOPS subunits. For clarity, residue numbering corresponds M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           jannaschii proteins (except Pcc1, numbers P. furiosus), and, appropriate, <br />                                           residue numbers corresponding S. cerevisiae ortholog shown parentheses. See <br />                                           Table S3 Figures S1&#226;&#8364;&#8220;5 residue equivalence. <br />  <br />                               Individual subunit analyses <br />                                           i. Bud32 structure function&#226;&#8364;&#8221;The crystal structure Bud32 reveals bilobal <br />                                           architecture, characteristic eukaryotic protein kinases, major atypical features <br />                                           (Figure 2A). The N-lobe canonical protein kinases consists minimally stranded <br />                                           &#206;&#178;-sheet flanking helix &#206;&#177;C. These features conserved primary (Figure S1) and <br />                                           atomic structures Bud32 (Figure 2A) exception strand &#206;&#178;1, present but <br />                                           disordered different crystal complexes analyzed. The C-lobe Bud32 unusually <br />                                           small, consisting &#206;&#178;-strands (&#206;&#178;7 &amp; &#206;&#178;8) &#206;&#177;-helices (&#206;&#177;D, &#206;&#177;E, &#206;&#177;F, &#206;&#177;J &amp; &#206;&#177;R) of <br />                                           (&#206;&#177;D, &#206;&#177;E &amp; &#206;&#177;F) direct equivalents canonical protein kinase <br />                                           structures (Figure 2A). Strikingly, C-lobe Bud32 lacks infrastructure normally <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                             Page 4 <br />  <br />  <br />                                         responsible phospho-acceptor site recognition, including activation segment (A- <br />                                         loop), substituted direct link strand &#206;&#178;8 helix &#206;&#177;F (Figure 2A). <br />                                         Despite atypical structure, Bud32 retains conserved elements required ATP binding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         catalysis (Figure 2A, Table S2) auto-phosphorylate vitro (Figure 2B) <br />                                         (Facchin et al., 2002) manner dependent integrity conserved catalytic <br />                                         elements. <br />  <br />                                         The closest structural homologues Bud32 RIO kinases (Rio1 Rio2) (Figure <br />                                         2C, panel). The Rio kinases exhibit abbreviated C-lobe absence A- <br />                                         loop (for comparison PKA, A-loop, Figure 2C, panel). The <br />                                         question protein kinase recognizes phosphorylates substrate absence <br />                                         A-loop remains open question. <br />  <br />                                         To test biological importance unique features identified archaeal KEOPS structures, <br />                                         mutated residues budding yeast counterpart identified structure based sequence <br />                                         alignments. These mutants introduced S. cerevisiae strains low copy <br />                                         plasmids context chromosomal copy KEOPS component deleted. <br />                                         We assessed biological readouts KEOPS function: cell growth telomere length. <br />                                         Consistent Bud32 competent phospho-transfer vitro, ATP binding and <br />                                         catalytic residues required biological function yeast [Figure 2DE, Table S3 &amp; <br />                                         (Downey et al., 2006)]. Indeed, catalytic cleft mutants K360A (K52A), D467A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (D182A), N456A (N166A), D451A (D161A), D451R (D161R), N456A/D467A (N166A/ <br />                                         D182) result cell growth defects telomere shortening. The severity two <br />                                         phenotypes imparted BUD32 mutations parallel other, parallel loss <br />                                         protein kinase activity vitro caused equivalent mutation archaeal Bud32 <br />                                         (Figure 2B). Interestingly, kinase inactivating mutants displayed phenotypes as <br />                                         strong bud32&#206;&#8221;, indicating phosphotransfer activity Bud32 sole <br />                                         function. Finally, catalytic cleft mutations tested destroy protein structure each <br />                                         mutant binds archaeal Cgi121 vitro (Figure S6A) mutant Bud32 expression levels in <br />                                         yeast similar wild type levels (Figure S6B). <br />  <br />                                         ii. Kae1 structure function&#226;&#8364;&#8221;We solved Kae1 structures corresponding two <br />                                         species (M. jannaschi, T. acidophilum) different protein complexes (Kae1-Bud32, <br />                                         Kae1-Bud32-Cgi121 &amp; Kae1-Pcc1) nucleotide. The structures Kae1 very <br />                                         similar to published structure P. furiosus Kae1 (PDB 2IVO, 2IVP <br />                                         2IVN) (1.20 &#195;&#8230;, 1.17 &#195;&#8230; &amp; 1.20 &#195;&#8230; C&#206;&#177; rmsd respectively). As structure isolated <br />                                         Kae1 subunit described previously (Hecker et al., 2007), features most <br />                                         relevant study. Kae1 belongs actin-like ATPase domain superfamily of <br />                                         proteins, known ASKHA superfamily. The ASKHA fold serves broad <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         functions: 1) conformational hydrolases (e.g. Hsp70, actin) 2) metabolite kinases <br />                                         (e.g. glycerol kinase). The ASKHA fold consists subdomains (denoted I &amp; II) with <br />                                         highly similar RNase H-like folds (Hurley, 1996). The fold subdomain is <br />                                         characterized central strand &#206;&#178;-sheet flanking &#206;&#177;-helices. The active site resides <br />                                         cleft region subdomains, sandwiched &#206;&#178;-sheets. ATP <br />                                         binding dependent extensive protein contacts adenine ring sugar. ATPase <br />                                         catalytic function dependent bound metal ions. Ribbons schematic representations <br />                                         Figures 3AB summarize conserved nucleotide metal ion binding infrastructure of <br />                                         Kae1. <br />  <br />                                         The specific function ASKHA family members attributable presence of <br />                                         unique inserts core fold (Hurley, 1996). Kae1 possess unique inserts (denoted <br />                                         specific inserts 1 &amp; 2) comprise surface exposed &#206;&#177;-helices (&#206;&#177;4, &#206;&#177;5 &amp; &#206;&#177;6) a <br />                                         long intervening linker &#206;&#177;5 &#206;&#177;6 insert, C-terminal tail <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                             Page 5 <br />  <br />  <br />                                         adopting extended conformation interspersed short 310 helices second <br />                                         insert (Figure 3C). The C-terminal tail threads catalytic cleft Kae1. Comparison <br />                                         Kae1 ASKHA family members, kinase hydrolase function, did give <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         insight specific function. <br />  <br />                                         To begin characterization Kae1, purified wild type site-directed mutants M. <br />                                         jannaschii Kae1 assessed ATP-binding, ATPase activity, DNA binding AP <br />                                         endonuclease activity. We wild type Kae1 binds ATP Kd 27 &#206;&#188;M (Table <br />                                         S3), intracellular ATP levels. We detected low reproducible <br />                                         ATPase activity affected KEOPS subunits (data shown) but <br />                                         depended presence metal ion binding residues (Figure S7A). We did detect <br />                                         apurinic (AP) endonuclease DNA binding activity (Figure S7B&#226;&#8364;&#8220;D). Mutation residues <br />                                         coordinate ATP, metal ion, perturbed ATP-binding function Kae1 <br />                                         (Table S3). None Kae1 mutations destroyed protein fold mutant proteins <br />                                         bind Pcc1 vitro (Figure S7E). To assess contribution ATP- metal ion- <br />                                         binding Kae1 biological function, kae1 mutant yeast strains perturbed one <br />                                         function other. kae1 mutants perturbed ATP-binding vitro disrupted the <br />                                         putative metal ion binding site, resulted phenocopy kae1&#206;&#8221; mutation yeast [near <br />                                         absence cell growth (Figure S7F)] despite wild type protein expression levels (Figure <br />                                         S6C). This effect yeast growth precluded analysis telomere length <br />                                         <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span>. However, able culture ATP-binding mutant (E176R) and, as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         predicted, mutant short telomeres (Figure 3D). <br />  <br />                                         iii. Pcc1 structure function&#226;&#8364;&#8221;The Pcc1 structure solved complex <br />                                         Kae1. The Pcc1 structure consists three-stranded anti-parallel &#206;&#178;-sheet packed with <br />                                         &#206;&#177; helices face, solvent exposed (Figure 4A panel). This fold <br />                                         shared 700 functionally diverse proteins (DALI Z score &gt;2.0). A close structural <br />                                         homologue Pcc1 KH domain (Z score=6.7), functions versatile RNA, <br />                                         ssDNA RNA hairpin binder (Figure S8A). The ability KH domain bind nucleic <br />                                         acids dependent signature motif (GXXG loop), located kink helix &#206;&#177;1 <br />                                         (Valverde et al., 2008), notably absent Pcc1 (Figure S8A). <br />  <br />                                         Pcc1 forms tightly interdigitated dimer contact area 555 &#195;&#8230;2 (1110 &#195;&#8230;2 buried <br />                                         surface area) (Figure 4AB). Dimerization involves anti-parallel arrangement two <br />                                         monomers giving rise continuous six-stranded anti-parallel &#206;&#178;-sheet packed &#206;&#177; <br />                                         helices face. Interestingly, mode dimerization widely used KH <br />                                         domains (Figure S8A). Indeed, Pcc1 dimerizes solution, concentrations tested <br />                                         (Figure S8B). Equilibrium centrifugation analysis purified Kae1-Pcc1 complex revealed <br />                                         2:2 binding stochiometry (Figure S8C), suggesting Pcc1 functions dimerization <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         module presence KEOPS subunits. Our attempts disrupt Pcc1 homo- <br />                                         dimerization mutation yielded insoluble protein (data shown), suggesting Pcc1 is <br />                                         obligate dimer, notion supported conserved hydrophobic nature dimer <br />                                         interface (Figure 4B &amp; S9A). <br />  <br />                                         iv. Cgi121 structure&#226;&#8364;&#8221;We solved structures human Cgi121 as <br />                                         archaeal Cgi121 complex Bud32 Kae1 (Figures 4C &amp; 6A). The <br />                                         human archaeal Cgi121 structures similar (2.0&#195;&#8230; C&#206;&#177; rmsd), consisting a <br />                                         central anti-parallel &#206;&#178;-sheet abutted &#206;&#177; helices face. In surveying the <br />                                         PDB, identified close structural ortholog P. furiosus (PDB 1ZD0). Aside <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                         protein, did identify proteins similar fold. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                              Page 6 <br />  <br />  <br />                               The Kae1-Bud32 interaction <br />                                          Binding Bud32 Kae1 involves contact surface 1583 &#195;&#8230;2 (3167 &#195;&#8230;2 buried surface) <br />                                          (Figure 5A) highly conserved Kae1 Bud32 orthologs (Figure S9BC &amp; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          S10A). Bud32 engages Kae1 using forward projecting surfaces N- C- <br />                                          terminal lobes, clamps subdomain II Kae1. Overall, binding mode of <br />                                          Bud32 Kae1 observed study similar Kae1-Bud32 structure recently <br />                                          reported Hecker et al. (2008) (PDB 2VWB, 1.75 &#195;&#8230; C&#206;&#177; rmsd). Binding Bud32 does not <br />                                          greatly alter conformation Kae1 (1.2 &#195;&#8230; C&#206;&#177; rmsd) exception &#206;&#178;3-&#206;&#177;1 <br />                                          linker. The reverse assessed, Bud32 structure absence <br />                                          Kae1. <br />  <br />                                          The N-lobe structural elements Bud32 involved Kae1 binding include C-terminal <br />                                          end helix &#206;&#177;C linker strand &#206;&#178;3 (Figure 5B). C-lobe structural elements involved <br />                                          Kae1 binding include &#206;&#177;E-&#206;&#178;7 linker containing catalytic loop, helix &#206;&#177;F, helix &#206;&#177;R <br />                                          extended C-terminal tail. Kae1 structural elements involved Bud32 binding <br />                                          include &#206;&#178;3&#226;&#8364;&#178; strand, extended linker sequence helices &#206;&#177;5 &#206;&#177;6 (within <br />                                          Kae1-specific insert 1), helix &#206;&#177;1&#226;&#8364;&#178;, helix &#206;&#177;2&#226;&#8364;&#178;. The contacts subdomain I Kae1 <br />                                          mediated C-terminal tail Bud32 interacts C-terminal region of <br />                                          helix &#206;&#177;1 &#206;&#178;4-&#206;&#177;2 linker. <br />  <br />                                          The Bud32-Kae1 interface composed mixture hydrophobic, electrostatic and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          hydrogen bonding interactions. Interesting interaction clusters include salt network <br />                                          Bud32 residues R380 K365 Kae1 residues E236 E233, Bud32 <br />                                          K489 Kae1 E148, catalytic <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span> Bud32, D451, Kae1 R287. <br />                                          Hydrophobic interactions include network involving V486 V482 Bud32 L150 <br />                                          K193 Kae1 (Figure 5B panel). Conspicuously, R534 C-terminal tail of <br />                                          Bud32 forms hydrogen bond N132 Kae1 immediately adjacent metal ion <br />                                          binding site (Figure 5B panel). The R534 interaction correlates large <br />                                          displacement &#206;&#178;3-&#206;&#177;1 linker subdomain I Kae1, giving rise partial opening of <br />                                          ATP binding cleft reduction inter-subdomain contacts (Figure 5B bottom <br />                                          panel). <br />  <br />                                          To assess functional significance Bud32-Kae1 interaction, probed the <br />                                          interaction surfaces site-directed mutagenesis. The K365E, R380E, V486Y, K489E <br />                                          single site mutations Bud32 E233R E236K single site mutations Kae1 <br />                                          markedly reduced Bud32-Kae1 interaction vitro (Figure 5C) perturbing <br />                                          binding Cgi121 Pcc1 (Figures 5D). In contrast, deletion C-terminal tail of <br />                                          Bud32 impact Bud32-Kae1 interaction (Figure S11A). We the <br />                                          equivalent mutations yeast Kae1 Bud32 test functional relevance vivo. As <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          shown Figure S10B S6C, Kae1 E233R (E292R) E236K (E295K) mutations <br />                                          severely impaired growth budding yeast adversely affecting protein expression. <br />                                          Similarly, Bud32 single site mutations K489E (R204E), V486Y (V201Y) double <br />                                          site mutations K365E/R380E (K57E/R72E) impaired Bud32 function vivo (Figure S10C) <br />                                          adversely affecting protein expression (Figure S6B). These results indicate the <br />                                          binding Bud32 Kae1 critical KEOPS function, conclusion reached by <br />                                          Hecker et al. (2008). As catalytic base D451 (D161) Bud32 lies Kae1 <br />                                          contact surface, perturbation binding Kae1 explain potent vivo <br />                                          effect caused D451R (D161R) mutant relative Bud32 active site mutants <br />                                          (Figure 2D). Consistent notion, combination mutants targeting Bud32 <br />                                          catalytic residues contact residues Kae1 pronounced phenotypic effects <br />                                          approaching severity BUD32 deletion (Figure S10E, Table S3) adversely <br />                                          affecting protein expression (Figure S6B). Finally, tested importance Bud32 C- <br />                                          terminal tail biological function. Despite fact deletion tail did affect <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                             Page 7 <br />  <br />  <br />                                          Bud32 binding Kae1 vitro, deletion yeast Bud32 severe growth defect <br />                                          (Figure S6B, Figure S11B). This result suggests possible role Bud32 tail in <br />                                          regulating Kae1 function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Kae1 substrate Bud32 <br />                                          Since Kae1 binds close catalytic cleft Bud32, explored possibility Kae1 <br />                                          substrate Bud32. Indeed, Bud32 phosphorylate Kae1 vitro (Figure S11C). <br />                                          Secondly, phosphorylation Kae1 Bud32 functionally relevant, acceptor sites in <br />                                          Kae1 conserved Kae1 proteins species containing Bud32 ortholog. Of <br />                                          11 conserved S/T residues, T308 T316 M. jannaschii Kae1 localize Kae1- <br />                                          specific insert 2 (Figure S11D). This insert notable meandering path the <br />                                          catalytic cleft Kae1 conservation archaeal eukaryotic Kae1 <br />                                          orthologs bacterial counterparts lack Bud32 orthologue (Figure S5). The <br />                                          extended surface exposed nature Kae1 insert 2 leaves accessible to <br />                                          active site Bud32. In contrast, remaining 9 conserved S/T sites deeply buried <br />                                          fold Kae1 integrated secondary structure elements. <br />  <br />                                          If Bud32 regulates Kae1 function phosphorylation sites Kae1-specific insert <br />                                          2, deleting insert pronounced effect vivo. Supporting idea, <br />                                          partial (1-372) deletion (1-364) C-terminal tail yeast Kae1 caused severe <br />                                          growth defect (Figure S11E). Moreover, T308A (T369A) T316A (T377A) mutant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          showed similar phenotype, adversely affecting protein expression (Figure S6C). <br />                                          This result consistent notion T308 (T369) Kae1 Bud32 phospho- <br />                                          regulatory site. Together, data support possibility Kae1 biologically <br />                                          relevant substrate Bud32 phospho-acceptor site lies Kae1-specific <br />                                          insert 2. However, deletion Kae1-specific insert 2 did abolish Bud32 <br />                                          phosphorylation Kae1 vitro (data shown), implying residues also <br />                                          targeted Bud32 vitro. <br />  <br />                               The Bud32-Cgi121 interaction <br />                                          The binding mode Cgi121 Bud32 revealed Bud32-Cgi121-Kae1 ternary <br />                                          structure (Figures 6A &amp; S12A). Cgi121 binds exclusively N-lobe Bud32, a <br />                                          surface composed helix &#206;&#177;C, adjacent strand &#206;&#178;4 &#206;&#178;2-&#206;&#178;3 linker (Figure 6B). The <br />                                          reciprocal binding surface Cgi121 composed helices &#206;&#177;2 &#206;&#177;8. The total contact <br />                                          surface 700 &#195;&#8230;2 (1400 &#195;&#8230;2) relative inter-subunit contact surfaces of <br />                                          KEOPS complex, conserved (Figure S9CD). <br />  <br />                                          The Cgi121-Bud32 interface composed combination hydrophobic, hydrogen <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          bonding, electrostatic interactions (Figure 6B). Hydrophobic interactions consist two <br />                                          clusters. The formed L353 Y352 tip Bud32 &#206;&#178;2-&#206;&#178;3 loop, L134 <br />                                          Cgi121 helix &#206;&#177;8. The second formed A389 V404 Bud32 A138 L139 <br />                                          Cgi121. Salt interactions link K392 Bud32 D135 Cgi121, D403 Bud32 <br />                                          K46 Cgi121. Lastly, Y352 Bud32 hydrogen bonds H46 Cgi121, K46 <br />                                          Cgi121 hydrogen bonds backbone carbonyl D403 Bud32. <br />  <br />                                          Of inter-subunit contacts characterized, Bud32 displays largest conformational <br />                                          change binding Cgi121 (1.38 &#195;&#8230; C&#206;&#177; rmsd comparison 0.9 &#195;&#8230; C&#206;&#177; rmsd Kae1). <br />                                          The apparent conformational change consists 10 &#195;&#8230; shift &#206;&#178;2-&#206;&#178;3 loop (Figure <br />                                          S12B). This shift appears stabilized involvement Y352 Bud32 both <br />                                          hydrophobic hydrogen bond interactions Cgi121. All N-lobe elements of <br />                                          Bud32 including helix &#206;&#177;C entire C-lobe appear minimally affected Cgi121 <br />                                          binding. <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                              Page 8 <br />  <br />  <br />                               Cgi121 stimulates Bud32 activity <br />                                           We examined impact Cgi121 mutations Cgi121-Bud32 interaction. While <br />                                           L134E, L139R A138R single site mutations effect Bud32 binding vitro <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (Figure 6C) L134E/A138R double mutation perturbed binding severely. We confirmed <br />                                           NMR analysis overall fold Cgi121 affected double mutation <br />                                           (Figure S12C). The dispersed nature HSQC spectra produced double mutant <br />                                           consistent folded state close correspondence HSQC spectra <br />                                           produced wild type mutant Cgi121 proteins consistent absence gross <br />                                           structural changes. <br />  <br />                                           The binding Cgi121 Bud32 N-lobe site centered helix &#206;&#177;C reminiscent of <br />                                           intermolecular mode Cdk (cyclin-dependent protein kinase) regulation cyclins <br />                                           (Jeffrey et al., 1995). In case Cdks, binding cyclins reorients helix &#206;&#177;C a <br />                                           productive position supports catalysis. While position &#206;&#177;C Bud32 not <br />                                           outwardly affected Cgi121, large conformational changes &#206;&#178;2-&#206;&#178;3 loop apparent. <br />                                           This led test Cgi121 influences catalytic activity Bud32. Cgi121 potent <br />                                           activator (&gt;20 fold) Bud32 auto-phosphorylation vitro (Figures 2B 6D). In <br />                                           addition, activator function dependent ability Cgi121 bind Bud32, as <br />                                           Cgi121 L134E/A138R double mutant did stimulate Bud32 kinase activity (Figure <br />                                           6D). To investigate impact loss binding Cgi121 Bud32 vivo, we <br />                                           analogous double mutation (L172E/I176R) yeast Cgi121. As shown Figure <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           6E, Cgi121 (L172E/I176R) strain displayed slow growth phenotype range the <br />                                           cgi121&#206;&#8221; mutation, strongly suggesting binding Bud32 central Cgi121&#226;&#8364;&#8482;s <br />                                           biological function. <br />  <br />                                           A role Cgi121 regulating Bud32 protein kinase activity consistent the <br />                                           observation phenotypes imparted cgi121&#206;&#8221; mutation milder those <br />                                           caused bud32&#206;&#8221; kae1&#206;&#8221; mutations (Downey et al., 2006), closely resemble the <br />                                           phenotype Bud32 catalytic site mutants. Moreover, Cgi121 expressed lower <br />                                           levels members KEOPS (Ghaemmaghami et al., 2003). This observation is <br />                                           reminiscent cyclin-Cdk paradigm activity Cdks regulated by <br />                                           proteosome-mediated destruction cyclins, Cdk levels remain fixed. This similarity <br />                                           led test yeast Cgi121 Bud32 unstable proteins GAL1 promoter shut off <br />                                           experiments. We observed Cgi121 highly unstable half life ~30 min whereas <br />                                           Bud32 remained stable experiment (Figure 6F). Consistent possibility <br />                                           binding Cgi121 Bud32 enhances stability vivo, expression level the <br />                                           Cgi121 (L172E/I176R) mutant unexpectedly low relative wild type protein <br />                                           (Figure S6D). Together, data suggest Cgi121 function positive <br />                                           regulator Bud32 function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               The Kae1-Pcc1 interaction <br />                                           We unable crystallize 2:2 stochiometric complex Pcc1 Kae1 but <br />                                           serendipitously crystallized solved structure 2:1 complex resulting from <br />                                           purification Kae1 presence excess Pcc1 (Figure 7A). Pcc1 Kae1 not <br />                                           undergo significant conformational changes response complex formation (0.73&#195;&#8230; &amp; <br />                                           1.20&#195;&#8230; C&#206;&#177; rmsd, respectively). The Kae1-Pcc1 binding interface extensive (contact <br />                                           area: 922 &#195;&#8230;2; buried area: 1844 &#195;&#8230;2) conserved (Figure S9AB) Kae1-Bud32 <br />                                           binding interface. <br />  <br />                                           Kae1 binds exclusively monomer Pcc1 dimer. Kae1 helices &#206;&#177;1 &#206;&#177;2 pack <br />                                           helices &#206;&#177;1 &#206;&#177;2 Pcc1, giving rise tightly packed four-helix bundle (Figure <br />                                           S13A). Additional contacts Kae1 310-1 helix Pcc1 helix &#206;&#177;2. <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                            Page 9 <br />  <br />  <br />                                           The binding interactions, primarily hydrophobic nature, augmented small number <br />                                           salt hydrogen bonds (Figure S13A). The majority hydrophobic interactions involve <br />                                           small chain residues, including T89, A90, T93 Kae1 A75, V79, A26 Pcc1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           allowing close approach participating &#206;&#177; helices. Non-hydrophobic contacts <br />                                           include salt interaction E30 Pcc1 R92 Kae1. <br />  <br />                                           Consistent Pcc1 functioning dimerization module (Figure S8C), fully <br />                                           accessible surfaces Kae1 binding presented Pcc1 dimer structure. <br />                                           Superposition second Kae1 molecule free binding surface Pcc1 revealed <br />                                           contacts involving subdomains II juxtaposed Kae1 molecules (Figure S13B). We <br />                                           reason contacts accommodated small changes inter-subunit <br />                                           closure angle Kae1 ASKHA fold yield 2:2 stochiometric complex observed <br />                                           solution (Figure S8C). Extension modeling studies include Bud32 Cgi121 in <br />                                           fully dimeric KEOPS complex revealed prohibitory contacts involving Cgi121 and <br />                                           Bud32 subunits (Figure S13C). <br />  <br />                               Functional characterizations Kae1-Pcc1 interaction <br />                                           Introduction Kae1 T88R mutation, combination T92R R91E, <br />                                           perturbed binding Kae1 Pcc1, Bud32, vitro (Figures 7B). Mutation of <br />                                           small hydrophobic contact residues Pcc1 large polar residues (Pcc1 A75Y V79R), <br />                                           impaired binding Kae1, double mutant A75Y/V79R pronounced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           effect (Figure 7C). A charge reversal mutation E30R Pcc1 disrupted binding Kae1. <br />                                           The Pcc1 mutations did destroy protein fold mutant easily purified (Figure <br />                                           7D) eluted behaved dimer purification (data shown). Analogous <br />                                           mutations yeast PCC1 KAE1 counterparts introduced a <br />                                           temperature sensitive pcc1-4 yeast strain (Kisseleva-Romanova et al., 2006). The Pcc1 <br />                                           V79R (A83R) A75Y (T79R) mutations disrupt binding Kae1 vitro did not <br />                                           rescue pcc1-4 temperature sensitivity phenotype wild type Pcc1 did (Figure 7E, <br />                                           Table S3). Similarly, T88R (I123R) R91E (R126A) mutations Kae1 resulted in <br />                                           severe loss function vivo (Figure 7F). Note R91E (R126A) Kae1 mutant was <br />                                           expressed yeast (Figure S6C). These data indicate Pcc1-Kae1 interaction is <br />                                           required biological function. <br />  <br />                                           To test functional requirement Pcc1 dimerization vivo, Pcc1- <br />                                           Pcc1 fusion. The close proximity N- C-termini adjacent protomers Pcc1 <br />                                           homodimer suggested Pcc1-Pcc1 fusion readily form intramolecular dimers. As <br />                                           observed wild type Pcc1, Pcc1-Pcc1 wild type fusion rescued pcc1-4 temperature <br />                                           sensitive growth phenotype (Figure S13D). Consistent need functional copies <br />                                           Kae1 binding surface Pcc1 dimer, disruption binding sites the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Pcc1-Pcc1 fusion introduction A75R (T79R), V79R (A83R) A75R (T79R)/A75R <br />                                           (T79R) mutations abolished ability Pcc1-Pcc1 fusion rescue pcc1-4 growth <br />                                           defect restrictive temperature (Figure S13D). These data consistent Pcc1 <br />                                           functioning dimer suggests KEOPS holocomplex acts dimer in <br />                                           vivo. Indeed, yeast KEOPS subunits Kae1, Gon7, Pcc1, Cgi121 Bud32 co- <br />                                           migrate complex apparent MW ~300,000 Da (Downey et al., 2006); (Downey et <br />                                           al., 2006; Kisseleva-Romanova et al., 2006), larger predicted <br />                                           monomer mass 116,872 Da closely matches predicted dimer mass 233,744 <br />                                           Da. <br />  <br />                               DISCUSSION <br />                                           The KEOPS complex ancient, protein kinase-containing molecular machine critical for <br />                                           telomere function gene transcription budding yeast. This work provides framework <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                            Page 10 <br />  <br />  <br />                                           in-depth functional biochemical characterization providing atomic model of <br />                                           complex. We report catalytic activity KEOPS subunits, Bud32 <br />                                           (protein kinase) Kae1 (ATPase), inter-subunit interaction surfaces, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           important overall function KEOPS complex vivo. A challenge the <br />                                           future identify substrate(s) KEOPS order unravel molecular <br />                                           machine exerts profound impact diverse aspects cell biology. <br />  <br />                               Kae1: universal ASKHA-type ATPase <br />                                           Kae1 minimal gene set organisms. In eukaryotes, nuclear encoded Kae1 <br />                                           paralog (Qri7 S. cerevisiae) localized mitochondria, itself bacterial symbiont <br />                                           (Ghaemmaghami et al., 2003; Reinders et al., 2006), underscoring ancient critical <br />                                           role Kae1 play. ASKHA type ATPases function primarily conformational <br />                                           hydrolases metabolite kinases. Moreover, Kae1 proposed glycoprotease <br />                                           AP endonuclease, strikingly different biochemical functions. Our studies M. <br />                                           jannaschii Kae1 revealed ATPase activity DNA binding AP endonuclease <br />                                           activities. Hence, exact function Kae1 remains mysterious. <br />  <br />                               Bud32 regulator Kae1 <br />                                           It striking archaeal species produce single protein encoding Bud32 Kae1, <br />                                           indicating intimate relationship proteins. Since Kae1 conserved species, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tempting speculate Bud32 (and members KEOPS) evolved to <br />                                           modulate Kae1 function. However, rule Kae1 modulates Bud32 that <br />                                           Kae1 Bud32 collaborate modify substrates. In fact, Kae1 recently proposed <br />                                           inhibitor Bud32 kinase activity (Hecker et al. 2008). Nevertheless, work <br />                                           suggests means Bud32 regulate Kae1, phosphorylating Kae1, and <br />                                           second binding Kae1. Dual regulation Kae1 Bud32 experimentally supported by <br />                                           observation bud32&#206;&#8221; mutation phenotypically severe disruption of <br />                                           Bud32 catalytic activity binding Kae1. In fact, close phenocopy the <br />                                           bud32&#206;&#8221; mutation bud32 mutants occurs catalysis binding Kae1 <br />                                           impaired (Figure S10E, Table S3). <br />  <br />                                           ATPases ASKHA fold superfamily regulated intermolecular binding <br />                                           partners (Hurley, 1996; Mayer Bukau, 2005). For example, low ATPase activity of <br />                                           E. coli chaperone DnaK enhanced &gt;1000 fold binding peptide substrate and <br />                                           co-chaperone regulator DnaJ (Liberek et al., 1991). Additionally, catalytic cycle of <br />                                           DnaK ASKHA family ATPases limited ADP exchange. This necessitates the <br />                                           action exchange factors GrpE, case DnaK, opens nucleotide <br />                                           binding cleft (Harrison et al., 1997). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           How does binding Bud32 influence Kae1 catalytic function? The C-terminal tail of <br />                                           Bud32 binds catalytic cleft Kae1 directly adjacent metal ion coordinating <br />                                           center. This leaves Bud32 tail placed influence ATPase activity binding a <br />                                           biological substrate. Alternatively Bud32 induced movement &#206;&#178;3-&#206;&#177;1 linker Kae1 <br />                                           creates open catalytic cleft potentiate nucleotide exchange. Investigating <br />                                           possibilities awaits better understanding biochemical function Kae1 the <br />                                           key players Kae1&#226;&#8364;&#8482;s catalytic cycle. Indeed, detect Kae1 ATPase activity vitro, <br />                                           low levels suggest key factor missing reconstitution assays. <br />  <br />                               Bud32 Rio kinase: living protein kinase fossils? <br />                                           The protein kinase superfamily diverse group enzymes phospho-transfer activity <br />                                           carried evolutionary conserved bilobal structure. Bud32, Rio <br />                                           paralogs, form minimal set protein kinases conserved archaeal eukaryotic <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                               Page 11 <br />  <br />  <br />                                           species (Kannan et al., 2007). A general feature Rio Bud32 protein kinases a <br />                                           minimal C-lobe absence A-loop. The role C-lobe A-loop substrate <br />                                           recognition raises question Rio Bud32 kinases recognize substrates. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           We suggest higher order binding mode revealed Bud32-Kae1 structure may <br />                                           reflect unconventional kinase substrate targeting mechanism, involving the <br />                                           recognition tertiary fold substrate simple linear peptide motif. This <br />                                           approach substrate recognition employed infrequently canonical eukaryotic protein <br />                                           kinases visualized atomic resolution eIF2&#206;&#177; protein kinase PKR <br />                                           (Dar et al., 2005) splicing related kinase SRPK1 (Ngo et al., 2008). The latter <br />                                           kinase-substrate complex notably similar Bud32-Kae1 complex substrate <br />                                           SRPK1 binds face kinase domain manner bridges two <br />                                           kinase lobes. <br />  <br />                                           Our work identified Cgi121 positive regulator Bud32 kinase activity. This <br />                                           conclusion based stimulatory effect Cgi121 binding Bud32 Bud32 <br />                                           autocatalytic activity. Moreover, yeast Cgi121 substochiometric component KEOPS <br />                                           abundance regulated protein degradation. Genetic analysis CGI121 in <br />                                           budding yeast consistent positive regulator KEOPS. We posit that <br />                                           identification cellular conditions Cgi121 stabilized provide important <br />                                           clue core function KEOPS complex. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Pcc1: dimerization subunit <br />                                           Our study identifies Pcc1 dimeric KH domain-like protein. Highlighting functional <br />                                           importance Pcc1 KEOPS complex, deletion PCC1 yeast results phenotypes <br />                                           severe BUD32 KAE1 deletions (Kisseleva-Romanova et al., 2006). <br />                                           Also, ability Pcc1 dimer bind Kae1 molecules needed for <br />                                           function vivo. These data consistent studies place Pcc1 center <br />                                           dimeric KEOPS holocomplex. Why KEOPS function dimer remains open <br />                                           question. One answer question KEOPS processive molecular <br />                                           machine. Indeed, dimerization multimerization common theme processive <br />                                           enzymes [e.g. helicases, kinesins, ubiquitin ligases polymerases (Lohman et al., 1998; <br />                                           Tang et al., 2007)] perform multiple catalytic cycles single substrate. The <br />                                           identification KEOPS substrate(s) provide invaluable tool decipher inner <br />                                           workings complex, regulation biological function. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               Bud32 vitro kinase assays <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Bud32 purified vitro hexahistidine tag fusion protein (pProEx-HTa) and <br />                                           concentrated buffer containing 10 mM HEPES pH 7.5, 300 mM NaCl 2 mM <br />                                           dithiothreitol (DTT). Kinase assays performed using [&#206;&#179;-32P]-ATP, 30 &#206;&#188;M cold ATP, <br />                                           20 &#206;&#188;M MnCl2 0.2&#226;&#8364;&#8220;1 &#206;&#188;M Bud32 60&#194;&#176;C. Reactions resolved sodium <br />                                           dodecylsulfate polyacrylamide gel electrophoresis (SDS-PAGE) visualized a <br />                                           PhosphorImager (Molecular Dynamics). <br />  <br />                               GST pull assays <br />                                           GST-tagged Cgi121, Kae1, Pcc1 proteins expressed Escherichia coli BL21 (DE3) <br />                                           Codon Plus cells lysed buffer containing 20 mM HEPES pH 7.5, 300 mM NaCl and <br />                                           2 mM DTT. Clarified lysate mixed glutathione-Sepharose resin 20&#194;&#176;C and <br />                                           allowed bind 30 min. 15 &#206;&#188;l aliquots resin bound fusion proteins incubated with <br />  <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                                Page 12 <br />  <br />  <br />                                           molar equivalents purified prey proteins 30 min, washed 6 times using 100X bed <br />                                           volumes buffer resolved SDS-PAGE. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Plasmids <br />                                           A description plasmids used Supplementary Experimental <br />                                           Procedures. <br />  <br />                               Telomere Length Determination <br />                                           Telomere length assays performed previously described (Downey et al., 2006). <br />  <br />                               Yeast Strains <br />                                           See Table S4 list yeast strains. Generation yeast strains study were <br />                                           performed using standard genetic techniques. <br />  <br />                               Protein Expression, Crystallization Data Collection <br />                                           All proteins described study purified using conventional affinity tag purification <br />                                           techniques (either glutathione-Sepharose nickel chelate chromatography). Affinity were <br />                                           tags removed TEV protease cleavage indicated. Seleno-methionine substituted <br />                                           proteins prepared described previously (Tang et al., 2007). Heavy atom <br />                                           derivatizations carried soaking crystals mother liquor solutions containing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           saturated [Ta6Br12]Br2. For x-ray diffraction experiments, crystals flash frozen in <br />                                           mother liquors supplemented 20&#226;&#8364;&#8220;35% (v/v) glycerol. All x-ray data sets processed <br />                                           using software XDS (Kabsch, 1993). <br />  <br />                                           Cgi121 isolated structures&#226;&#8364;&#8221;The length Homo sapiens Methanococcus <br />                                           jannaschii Cgi121 proteins expressed E. coli BL21 (DE3) CodonPlus cells a <br />                                           glutathione S-transferase (GST) fusion protein using pGEX2T plasmid modified for <br />                                           affinity tag cleavage Tobacco Etch Virus (TEV) protease. H. sapiens CGi121 <br />                                           crystals space group P62 (a=b=93.56&#195;&#8230;, c=87.16&#195;&#8230;, &#206;&#177;=&#206;&#178;=90&#194;&#176;, &#206;&#179;=120&#194;&#176;) two <br />                                           molecules asymmetric <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> obtained using hanging drop method mixing <br />                                           equal volumes 15 mg/mL protein solution 20% (w/v) PEG1500, 0.2 M calcium <br />                                           acetate, 0.1 M imidazole pH 8 4&#194;&#176;C. A selenium SAD data set collected NE- <br />                                           CAT beamline 24-ID-C. NMR analyses M. jannaschii Cgi121 performed using a <br />                                           Bruker AVANCE 600 800 spectrometer 298K. NMR samples contained 1.6 <br />                                           mM 15N, 13C-labeled M. jannaschii CGI121 20 mM sodium phosphate pH 6.3 100 <br />                                           mM NaCl 10% D2O. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Kae1-Bud32 complex&#226;&#8364;&#8221;The Methanococcus jannaschii protein MJ1130 (Kae1-Bud32 <br />                                           fusion) internal deletion amino acids (MJ1130 &#206;&#8221;R332-K333) expressed <br />                                           purified N-terminal GST-fusion protein human Cgi121. Crystals the <br />                                           space group P4122 (a=b=148.73&#195;&#8230;, c=136.42&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) molecules the <br />                                           asymmetric <span id='am-163' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> obtained using hanging drop method mixing equal volumes <br />                                           6 mg/ml protein solution 10 mM HEPES pH 7.5, 10% PEG3000 11% spermine <br />                                           tetrahydrochloride 20&#194;&#176;C. A tantalum SAD data set collected NE-CAT beamline <br />                                           24-ID-C. <br />  <br />                                           Kae1-Bud32-Cgi121 complex&#226;&#8364;&#8221;A 1:1 molar complex M. jannaschii Cgi121 and <br />                                           MJ1130 obtained mixing excess molar Cgi121 MJ1130 followed size <br />                                           exclusion chromatography remove excess Cgi121. Crystals space group P212121 <br />                                           (a=76.99&#195;&#8230;, b=106.91&#195;&#8230;, c=209.99&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) 1:1 complexes Cgi121 to <br />                                           MJ1130 asymmetric <span id='am-164' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> obtained using hanging drop method mixing <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                            Page 13 <br />  <br />  <br />                                           equal volumes 8 mg/mL protein complex 0.1 M MES pH 6.5 15% PEG3000. <br />                                           A tantalum SAD data set collected NE-CAT beamline 24-ID-C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Pcc1 structure&#226;&#8364;&#8221;Full length Pyrococcus furiosus Pcc1 protein (pfuPcc1) expressed <br />                                           purified N-terminal GST-fusion protein human Cgi121. Crystals space <br />                                           group P63 (a=b=78.81&#195;&#8230;, c=60.02&#195;&#8230;, &#206;&#177;=&#206;&#178;=90&#194;&#176;, &#206;&#179;=120&#194;&#176;) molecule asymmetric <br />                                           unit obtained using hanging drop method mixing equal volumes 5 mg/mL <br />                                           protein solution 45% PEG300, 0.1 M Na/K phosphate pH 6.1 20&#194;&#176;C. A selenium <br />                                           SAD data set collected CHESS East - F2 beamline processed using software <br />                                           XDS (Kabsch, 1993). <br />  <br />                                           Kae1-Pcc1 complex&#226;&#8364;&#8221;The Thermoplasma acidophilum Kae1 protein (taKae1) was <br />                                           expressed purified N-terminal GST-fusion protein human Cgi121. A 2:1 <br />                                           molar complex taKae1-pfuPcc1 obtained mixing 20 fold molar excess pfuPcc1 <br />                                           taKae1 followed size exclusion chromatography remove excess Pcc1. Crystals of <br />                                           space group P65 (a=b=66.56&#195;&#8230;, c=435.56&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) 2:1 complexes of <br />                                           Pcc1:Kae1, respectively, asymmetric unit obtained using hanging drop <br />                                           method mixing equal volumes 5 mg/mL protein complex 0.2 M NaCl, 40% <br />                                           PEG300, 0.1 M HEPES pH 7.5 20&#194;&#176;C. A native data set collected NE-CAT <br />                                           beamline 24-ID-E. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               X-ray Structure Determinations <br />                                           For experimental phasing, heavy atoms positions located using SHELXD (Schneider <br />                                           Sheldrick, 2002) refined using SHARP/AUTOSHARP (Vonrhein et al., 2007). <br />                                           RESOLVE (Terwilliger, 2004) used density modification model autobuilding. <br />                                           Coot (Emsley Cowtan, 2004) used manual building electron density map <br />                                           inspection. Refinement carried using Refmac (Winn et al., 2003) Phenix (Zwart <br />                                           et al., 2008). The HsCgi121 PfuPcc1 crystal structures solved selenium <br />                                           SAD method. The MJ1130 crystal structure solved tantalum SAD method. The <br />                                           structure MjCgi121/MJ1130 complex solved combination tantalum SAD <br />                                           method molecular replacement using Phaser (Storoni et al., 2004). Although the <br />                                           resolution MjCgi121/MJ1130 complex analysis low (3.6 &#195;&#8230;) comparison the <br />                                           structure analyses study, high quality experimentally phased electron <br />                                           density maps (Figure S6A) conjunction knowledge M. jannaschii Cgi121 <br />                                           NMR structure allowed unambiguously model position Cgi121 respect to <br />                                           Bud32. <br />  <br />                                           The crystal structure TaKae1-PfuPcc1 complex solved molecular replacement <br />                                           using aforementioned Kae1 Pcc1 structures search models Phaser (Storoni et <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           al., 2004). <br />  <br />                               MjCGI121 NMR resonance assignment structure calculation <br />                                           The 3D spectra measured non-uniform sampling (NUS) indirect <br />                                           dimensions (down 30% sampling) reconstructed using multidimensional <br />                                           decomposition (MDD) using MDDNMR (Jaravine Orekhov, 2006). All spectra were <br />                                           processed using NMRPipe/NMRDraw (Delaglio et al., 1995) analyzed using Sparky <br />                                           (Goddard T. D.). Backbone assignments obtained using standard triple-resonance <br />                                           experiments (Bax Grzesiek, 1993). Aliphatic chain assignments obtained from <br />                                           3D HCCH-TOCSY aromatic chain assignments 2D (H&#206;&#178;)C&#206;&#178;(C&#206;&#179;C&#206;&#180;)H&#206;&#180; and <br />                                           (H&#206;&#178;)C&#206;&#178;(C&#206;&#179;C&#206;&#180;C&#206;&#181;)H&#206;&#181; experiments (Yamazaki et al., 1993). Distance restraints, derived from <br />                                           15N- 13C-edited NOESY-HSQC (using cross-relaxation mixing time 100 ms) and <br />                                           dihedral angle restraints, predicted chemical shifts software TALOS <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                                        Page 14 <br />  <br />  <br />                                         (Cornilescu et al., 1999; Jaravine Orekhov, 2006), used input combined <br />                                         automated NOE assignment structure calculation program CYANA (Guntert, <br />                                         2004). Residual dipolar couplings (RDCs) gained difference between <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         1HN-15N splittings 2D IPAP-1H-15N HSQC experiments measured isotropic <br />                                         anisotropic conditions (Ottiger et al., 1998). Anisotropic media prepared by <br />                                         addition filamentous phages concentration 10 mg/mL (Hansen et al., 1998; <br />                                         Zweckstetter Bax, 2001). The final 20 structures lowest target function the <br />                                         final CYANA cycle used refinement presence 122 HN-RDCs in <br />                                         explicit solvent (Linge et al., 2003). Structure statistics 10 lowest energy structures <br />                                         refinement shown Table S1. <br />  <br />                               Supplementary Material <br />                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                         We thank Steve Bell help identifying archaeal KEOPS orthologs Domenico Libri strains/plasmids. <br />                                         Research conducted NE-CAT beamlines (Advanced Photon Source) supported National Center <br />                                         Research Resources (NIH, award RR 15301) U.S. Department Energy (Office Basic Energy <br />                                         Sciences, contract DE-AC02-06CH11357). DD Canada Research Chair (Tier II). This work funded by <br />                                         grants CIHR (MOP 79441) DD Canadian Cancer Society FS (Grant 017220). DYLM DN <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         supported NCIC CIHR post-doctoral fellowships, respectively. SR supported postdoctoral <br />                                         fellowship German Academic Exchange Service (DAAD). <br />  <br />                                         Coordinates deposited PDB (accession codes 3EN9, 3ENH, 3ENP, 3ENC, 3ENO, 2K8Y). <br />  <br />  <br />                               References <br />                                         Abdullah KM, Lo RY, Mellors A. Cloning, nucleotide sequence, expression Pasteurella <br />                                           haemolytica A1 glycoprotease gene. J Bacteriol. 1991; 173:5597&#226;&#8364;&#8220;5603. [PubMed: 1885539] <br />                                         Arigoni F, Talabot F, Peitsch M, Edgerton MD, Meldrum E, Allet E, Fish R, Jamotte T, Curchod ML, <br />                                           Loferer H. A genome-based approach identification essential bacterial genes. Nat <br />                                           Biotechnol. 1998; 16:851&#226;&#8364;&#8220;856. [PubMed: 9743119] <br />                                         Bax A, Grzesiek S. Methodological Advances Protein Nmr. Accounts Chemical Research. 1993; <br />                                           26:131&#226;&#8364;&#8220;138. <br />                                         Cornilescu G, Delaglio F, Bax A. Protein backbone angle restraints searching database for <br />                                           chemical shift sequence homology. J Biomol NMR. 1999; 13:289&#226;&#8364;&#8220;302. [PubMed: 10212987] <br />                                         Dar AC, Dever TE, Sicheri F. Higher-order substrate recognition eIF2alpha RNA-dependent <br />                                           protein kinase PKR. Cell. 2005; 122:887&#226;&#8364;&#8220;900. [PubMed: 16179258] <br />                                         Delaglio F, Grzesiek S, Vuister GW, Zhu G, Pfeifer J, Bax A. NMRPipe: multidimensional spectral <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           processing based UNIX pipes. J Biomol NMR. 1995; 6:277&#226;&#8364;&#8220;293. [PubMed: 8520220] <br />                                         Downey M, Houlsworth R, Maringele L, Rollie A, Brehme M, Galicia S, Guillard S, Partington M, <br />                                           Zubko MK, Krogan NJ, et al. A genome-wide screen identifies evolutionarily conserved <br />                                           KEOPS complex telomere regulator. Cell. 2006; 124:1155&#226;&#8364;&#8220;1168. [PubMed: 16564010] <br />                                         Emsley P, Cowtan K. Coot: model-building tools molecular graphics. Acta crystallographica. 2004; <br />                                           60:2126&#226;&#8364;&#8220;2132. <br />                                         Facchin S, Lopreiato R, Stocchetto S, Arrigoni G, Cesaro L, Marin O, Carignani G, Pinna LA. <br />                                           Structure-function analysis yeast piD261/Bud32, atypical protein kinase essential normal <br />                                           cell life. Biochem J. 2002; 364:457&#226;&#8364;&#8220;463. [PubMed: 12023889] <br />                                         Feldser DM, Hackett JA, Greider CW. Telomere dysfunction initiation genome instability. <br />                                             Nat Rev Cancer. 2003; 3:623&#226;&#8364;&#8220;627. [PubMed: 12894250] <br />                                         Gatbonton T, Imbesi M, Nelson M, Akey JM, Ruderfer DM, Kruglyak L, Simon JA, Bedalov A. <br />                                             Telomere length quantitative trait: genome-wide survey genetic mapping telomere <br />                                             length-control genes yeast. PLoS Genet. 2006; 2:e35. [PubMed: 16552446] <br />  <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                                  Page 15 <br />  <br />  <br />                                         Ghaemmaghami S, Huh WK, Bower K, Howson RW, Belle A, Dephoure N, O&#226;&#8364;&#8482;Shea EK, Weissman <br />                                              JS. Global analysis protein expression yeast. Nature. 2003; 425:737&#226;&#8364;&#8220;741. [PubMed: <br />                                              14562106] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Gilson E, Geli V. How telomeres replicated. Nat Rev Mol Cell Biol. 2007; 8:825&#226;&#8364;&#8220;838. [PubMed: <br />                                              17885666] <br />                                         Goddard, TD.; KDG. SPARKY. Vol. 3. University Of California; San Francisco: <br />                                         Guntert P. Automated NMR structure calculation CYANA. Methods Mol Biol. 2004; 278:353&#226;&#8364;&#8220; <br />                                              378. [PubMed: 15318003] <br />                                         Hansen MR, Mueller L, Pardi A. Tunable alignment macromolecules filamentous phage yields <br />                                              dipolar coupling interactions. Nature Structural Biology. 1998; 5:1065&#226;&#8364;&#8220;1074. <br />                                         Harrison CJ, Hayer-Hartl M, Di Liberto M, Hartl F, Kuriyan J. Crystal structure nucleotide <br />                                              exchange factor GrpE bound ATPase domain molecular chaperone DnaK. Science. <br />                                              1997; 276:431&#226;&#8364;&#8220;435. [PubMed: 9103205] <br />                                         Hecker A, Leulliot N, Gadelle D, Graille M, Justome A, Dorlet P, Brochier C, Quevillon-Cheruel S, <br />                                              Le Cam E, van Tilbeurgh H, et al. An archaeal orthologue universal protein Kae1 iron <br />                                              metalloprotein exhibits atypical DNA-binding properties apurinic-endonuclease activity <br />                                              vitro. Nucleic Acids Res. 2007; 35:6042&#226;&#8364;&#8220;6051. [PubMed: 17766251] <br />                                         Hurley JH. The sugar kinase/heat shock protein 70/actin superfamily: implications conserved <br />                                              structure mechanism. Annu Rev Biophys Biomol Struct. 1996; 25:137&#226;&#8364;&#8220;162. [PubMed: <br />                                              8800467] <br />                                         Jaravine VA, Orekhov VY. Targeted acquisition real-time NMR spectroscopy. Journal the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              American Chemical Society. 2006; 128:13421&#226;&#8364;&#8220;13426. [PubMed: 17031954] <br />                                         Jeffrey PD, Russo AA, Polyak K, Gibbs E, Hurwitz J, Massague J, Pavletich NP. Mechanism CDK <br />                                              activation revealed structure cyclinA-CDK2 complex. Nature. 1995; 376:313&#226;&#8364;&#8220;320. <br />                                              [PubMed: 7630397] <br />                                         Kabsch W. Automatic processing rotation diffraction data crystals initially unknown <br />                                              symmetry cell constants. Journal Applied Crystallography. 1993; 26:795&#226;&#8364;&#8220;800. <br />                                         Kannan N, Taylor SS, Zhai Y, Venter JC, Manning G. Structural functional diversity the <br />                                              microbial kinome. PLoS Biol. 2007; 5:e17. [PubMed: 17355172] <br />                                         Kisseleva-Romanova E, Lopreiato R, Baudin-Baillieu A, Rousselle JC, Ilan L, Hofmann K, Namane <br />                                              A, Mann C, Libri D. Yeast homolog cancer-testis antigen defines new transcription <br />                                              complex. Embo J. 2006; 25:3576&#226;&#8364;&#8220;3585. [PubMed: 16874308] <br />                                         Liberek K, Marszalek J, Ang D, Georgopoulos C, Zylicz M. Escherichia coli DnaJ GrpE heat <br />                                              shock proteins jointly stimulate ATPase activity DnaK. Proc Natl Acad Sci U S A. 1991; <br />                                              88:2874&#226;&#8364;&#8220;2878. [PubMed: 1826368] <br />                                         Linge JP, Williams MA, Spronk CA, Bonvin AM, Nilges M. Refinement protein structures in <br />                                              explicit solvent. Proteins. 2003; 50:496&#226;&#8364;&#8220;506. [PubMed: 12557191] <br />                                         Lohman TM, Thorn K, Vale RD. Staying track: common features DNA helicases and <br />                                              microtubule motors. Cell. 1998; 93:9&#226;&#8364;&#8220;12. [PubMed: 9546385] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Mayer MP, Bukau B. Hsp70 chaperones: cellular functions molecular mechanism. Cell Mol Life <br />                                              Sci. 2005; 62:670&#226;&#8364;&#8220;684. [PubMed: 15770419] <br />                                         Ngo JC, Giang K, Chakrabarti S, Ma CT, Huynh N, Hagopian JC, Dorrestein PC, Fu XD, Adams JA, <br />                                              Ghosh G. A sliding docking interaction essential sequential processive phosphorylation <br />                                              SR protein SRPK1. Mol Cell. 2008; 29:563&#226;&#8364;&#8220;576. [PubMed: 18342604] <br />                                         Ottiger M, Delaglio F, Bax A. Measurement J dipolar couplings simplified two- <br />                                              dimensional NMR spectra. Journal Magnetic Resonance. 1998; 131:373&#226;&#8364;&#8220;378. [PubMed: <br />                                              9571116] <br />                                         Reinders J, Zahedi RP, Pfanner N, Meisinger C, Sickmann A. Toward complete yeast <br />                                              mitochondrial proteome: multidimensional separation techniques mitochondrial proteomics. J <br />                                              Proteome Res. 2006; 5:1543&#226;&#8364;&#8220;1554. [PubMed: 16823961] <br />                                         Schneider TR, Sheldrick GM. Substructure solution SHELXD. Acta crystallographica. 2002; <br />                                              58:1772&#226;&#8364;&#8220;1779. <br />                                         Shachar R, Ungar L, Kupiec M, Ruppin E, Sharan R. A systems-level approach mapping the <br />                                              telomere length maintenance gene circuitry. Mol Syst Biol. 2008; 4:172. [PubMed: 18319724] <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                               Page 16 <br />  <br />  <br />                                         Smogorzewska A, Lange T. Regulation telomerase telomeric proteins. Annu Rev Biochem. <br />                                             2004; 73:177&#226;&#8364;&#8220;208. [PubMed: 15189140] <br />                                         Storoni LC, McCoy AJ, Read RJ. Likelihood-enhanced fast rotation functions. Acta crystallographica. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             2004; 60:432&#226;&#8364;&#8220;438. <br />                                         Tang X, Orlicky S, Lin Z, Willems A, Neculai D, Ceccarelli D, Mercurio F, Shilton BH, Sicheri F, <br />                                             Tyers M. Suprafacial orientation SCFCdc4 dimer accommodates multiple geometries for <br />                                             substrate ubiquitination. Cell. 2007; 129:1165&#226;&#8364;&#8220;1176. [PubMed: 17574027] <br />                                         Terwilliger T. SOLVE RESOLVE: automated structure solution, density modification model <br />                                             building. Journal synchrotron radiation. 2004; 11:49&#226;&#8364;&#8220;52. [PubMed: 14646132] <br />                                         Valverde R, Edwards L, Regan L. Structure function KH domains. Febs J. 2008; 275:2712&#226;&#8364;&#8220; <br />                                             2726. [PubMed: 18422648] <br />                                         Verdun RE, Karlseder J. Replication protection telomeres. Nature. 2007; 447:924&#226;&#8364;&#8220;931. <br />                                             [PubMed: 17581575] <br />                                         Vonrhein C, Blanc E, Roversi P, Bricogne G. Automated structure solution autoSHARP. Methods <br />                                             molecular biology (Clifton, NJ). 2007; 364:215&#226;&#8364;&#8220;230. <br />                                         Winn MD, Murshudov GN, Papiz MZ. Macromolecular TLS refinement REFMAC moderate <br />                                             resolutions. Methods enzymology. 2003; 374:300&#226;&#8364;&#8220;321. [PubMed: 14696379] <br />                                         Yamazaki T, Forman-Kay JD, Kay LE. Two-dimensional NMR experiments correlating 13C-beta <br />                                             1H-delta/epsilon chemical shifts aromatic residues 13C-labeled proteins scalar <br />                                             couplings. J Am Chem Soc. 1993; 115:11054&#226;&#8364;&#8220;11055. <br />                                         Zwart PH, Afonine PV, Grosse-Kunstleve RW, Hung LW, Ioerger TR, McCoy AJ, McKee E, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Moriarty NW, Read RJ, Sacchettini JC, et al. Automated structure solution PHENIX <br />                                             suite. Methods molecular biology (Clifton, NJ). 2008; 426:419&#226;&#8364;&#8220;435. <br />                                         Zweckstetter M, Bax A. Characterization molecular alignment aqueous suspensions Pf1 <br />                                             bacteriophage. Journal Biomolecular Nmr. 2001; 20:365&#226;&#8364;&#8220;377. [PubMed: 11563559] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                     Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. Architecture KEOPS complex <br />                                         (A&#226;&#8364;&#8220;D) GST pull experiments purified KEOPS subunits. Proteins separated <br />                                         SDS-PAGE visualized Coomassie blue staining. <br />                                         (E) Architectural model KEOPS complex derived (A&#226;&#8364;&#8220;D). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (F) Composite model KEOPS complex based X-ray crystal structures single <br />                                         subunits, binary complexes ternary complex summarized Table 1. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                            Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. Structure catalytic function Bud32 <br />                                         (A) Ribbons representation Bud32 showing important catalytic residues. For figures <br />                                         tables, residues archaeal ortholog listed appropriate, the <br />                                         S. cerevisiae ortholog bracketed. Residue labels colored according effect the <br />                                         indicated mutations growth yeast (see color legend). The N-lobe Bud32 colored <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         light green C-lobe colored dark green. A metal ion ATP modeled into <br />                                         catalytic cleft based structure RIO1 (PDB 1ZP9). <br />                                         (B) Autophosphorylation analysis Bud32 wild type mutant proteins presence or <br />                                         absence Cgi121. Assays carried [&#206;&#179;-32P]-ATP 60&#194;&#176;C stopped the <br />                                         indicated time points. Products reactions separated SDS-PAGE 32P <br />                                         incorporation detected autoradiography. <br />                                         (C) Ribbons comparison Bud32 Rio1 (PDB 1ZP9; panel) PKA (PDB 2CPK; <br />                                         panel). Bud32 colored (A) Rio kinase shown red unique N- <br />                                         terminal lobe insertion (labeled &#226;&#8364;&#732;Rio insert&#226;&#8364;&#8482;) orange. PKA colored yellow A- <br />                                         loop shown red. <br />                                         (D) Effect Bud32 catalytic cleft mutations growth yeast. A bud32&#206;&#8221; yeast strain <br />                                         transformed centromeric low copy vector expressing wild type BUD32, the <br />                                         indicated Bud32 mutant vector control. Liquid cultures strain were <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                      Page 19 <br />  <br />  <br />                                         spotted minimal medium series 10 fold dilutions. Growth yeast 30&#194;&#176;C was <br />                                         monitored 6 day period. Representative data shown. A summary doubling times <br />                                         liquid culture selection mutants provided Table S3. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (E) Bud32 catalytic mutants short telomeres. Terminal restriction fragment (TRF) <br />                                         analysis yeast strains described (D). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                           Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. Structure catalytic function Kae1 <br />                                         (A) Ribbons representation Kae1 highlighting catalytic cleft residues. Subdomains I and <br />                                         II colored dark light blue respectively. The metal ion ADP modeled into <br />                                         cleft based structure P. abyssi Kae1 (PDB 2IVP). <br />                                         (B) Schematic catalytic cleft residues Kae1 coordinate ATP metal ion (based <br />                                         PDB 2IVP). <br />                                         (C) Kae1 characteristic inserts (colored pink red) distinguish other <br />                                         ASKHA fold proteins. Secondary structure elements unique inserts labeled. <br />                                         (D) The kae1-E176R (E213R) mutant short telomeres. TRF analysis performed on <br />                                         kae1&#206;&#8221; strain transformed low copy vector encoding wild type Kae1 E176 <br />                                         (E213R) mutant. Telomeres wild type, bud32&#206;&#8221; cgi121&#206;&#8221; strains used as <br />                                         reference. <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                     Page 21 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. Structures Pcc1 Cgi121 <br />                                         (A) Ribbons representation Pcc1 monomer (top) dimer (bottom). Pcc1 protomers <br />                                         1 2 colored light dark orange respectively. <br />                                         (B) Stereo view conserved hydrophobic contacts Pcc1 dimer interface. <br />                                         (C) Ribbons representation human Cgi121 (left) M.jannaschii Cgi121 (right) <br />                                         (rmsd=1.92 &#195;&#8230;). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                           Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 5. Binding mode Bud32 Kae1 <br />                                         (A) Ribbons representation Bud32-Kae1 complex. Kae1 Bud32 colored in <br />                                         Figures 2A 3A. Interface residues mutation gives rise growth phenotypes in <br />                                         yeast shown stick representation. <br />                                         (B) Stereo zoom-in view Kae1-Bud32 binding interface centered (top) Kae1- <br />                                         specific insert 1 (bottom) C-terminal tail region Bud32. Figures colored (A). <br />                                         (C,D) Mutation Kae1-Bud32 binding interface. Binding experiments the <br />                                         indicated GST fusion baits prey proteins performed Figure 1. (C) <br />                                         Examination Bud32-Kae1 interaction. Lane 1 (lower panel) loading control for <br />                                         Bud32. (D) Examination Cgi121-Bud32 interaction using Bud32 mutants (top) and <br />                                         Pcc1-Kae1 interaction using Kae1 mutants (bottom). <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                       Page 23 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 6. Binding mode Cgi121 Bud32 <br />                                         (A) Ribbons representation Cgi121-Bud32-Kae1 complex. Bud32, Kae1 Cgi121 <br />                                         colored Figures 2A, 3A 4B, respectively. Residues mutation gives rise <br />                                         strong phenotypes yeast highlighted. <br />                                         (B) Stereo zoom-in view Bud32-Cgi121 binding interface showing chains that <br />                                         participate interaction. <br />                                         (C) In vitro binding experiments indicated GST fusion baits purified prey <br />                                         proteins. <br />                                         (D) Disruption Cgi121 binding Bud32 impairs ability Cgi121 potentiate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Bud32 autophosphorylation activity vitro. Bud32 kinase reactions performed in <br />                                         Figure 2B. <br />                                         (E) Functional analysis yeast Bud32-Cgi121 binding interface mutations cell <br />                                         growth. cgi121&#206;&#8221; yeast transformed vector, vector encoding wild type <br />                                         Cgi121 L134E/A138R (L172E/I176R) double mutant. All strains plated by <br />                                         serial dilution minimal medium described Figure 2D. <br />                                         (F) Analysis Cgi121 Bud32 stability vivo. Yeast encoding galactose-inducible <br />                                         3xHA-Cgi121 Bud32-FLAG proteins grown 4 hrs media containing 2% <br />                                         glucose 2% galactose. Cells grown galactose washed resuspended 2% <br />                                         glucose-containing media repress GAL1 promoter. Cells harvested times <br />                                         following glucose addition. &#206;&#177;-HA &#206;&#177;-FLAG immunoblots shown. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                             Mao et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 7. Binding mode Pcc1 Kae1 <br />                                         (A) Ribbons representation binding interface Pcc1 Kae1. Pcc1 Kae1 <br />                                         colored Figures 3 4, respectively. Residues mutation gives rise a <br />                                         phenotype yeast highlighted. See Figure S13A zoom-in view Pcc1-Kae1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         binding interface participating chains. <br />                                         (B&#226;&#8364;&#8220;D) Mutational analysis Pcc1-Kae1 binding interface. Binding experiments with <br />                                         indicated GST fusion baits prey proteins performed Figure 1. (B) <br />                                         Examination various Kae1 proteins binding Pcc1 (upper panel) Bud32 (lower <br />                                         panel). (C) Examination various Pcc1 proteins binding Kae1. (D) Input gel shows <br />                                         equivalent quantities soluble wild type mutant Pcc1 proteins used binding <br />                                         reactions (C). <br />                                         (E) Analysis Kae1-Pcc1 interface mutations yeast growth. Left panel: Serial dilutions <br />                                         thermosensitive pcc1-4 strains harboring plasmid encoding wild type Pcc1, pcc1 mutants <br />                                         vector grown 37&#194;&#176;C (non-permissive) 23&#194;&#176;C (permissive) 3&#226;&#8364;&#8220;5 days. <br />                                         Right panel: kae1&#206;&#8221; yeast harboring plasmid encoding wild type Kae1, Kae1 mutants or <br />                                         vector grown 3&#226;&#8364;&#8220;6 days. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 May 20. <br />                                                               NIH-PA Author Manuscript                                                  NIH-PA Author Manuscript                                                NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                               Table 1 <br />                                                              Data collection refinement statistics. <br />  <br />                                                                                            mjBud32/Kae1                   mjBud32/Kae1/mjCgi121                hsCgi121                      pfuPcc1                     taKae1/pfuPcc1 <br />                                                                                                                                                                                                                                                                          Mao et al. <br />  <br />  <br />  <br />  <br />                                                               Wavelength (&#195;&#8230;)               0.97888                        0.97888                              0.9793                        0.979                       0.97935 <br />                                                               Resolution (&#195;&#8230;)               2.67                           3.6                                  2.48                          2.63                        3.02 <br />                                                               Rsym (%)                     5.78 (22.86)                   7.59(18.38)                          5.6(31.17)      <br /> </body></html>