<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cell. Author manuscript; available PMC 2012 August 17. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell. 2012 February 17; 148(4): 639&#226;&#8364;&#8220;650. doi:10.1016/j.cell.2011.12.033. <br />  <br />  <br />  <br />                            TAK1 (MAP3K7) inhibition promotes apoptosis KRAS- <br />                            dependent colon cancers <br />                            Anurag Singh1,3, Michael F. Sweeney1, Min Yu1, Alexa Burger1, Patricia Greninger1, Cyril <br />                            Benes1, Daniel A. Haber1,2,*, Jeff Settleman1,4,* <br />  <br />                            1MassachusettsGeneral Hospital Cancer Center Harvard Medical School, Charlestown, MA <br />                            02129 2Howard Hughes Medical Institute, Chevy Chase, Maryland <br />  <br />                            Summary <br />                                  Colon cancers frequently harbor KRAS mutations, subset KRAS-mutant colon <br />                                  cancer cell lines dependent KRAS signaling <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span>. In screen kinases that <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  promote survival KRAS-dependent colon cancer cells, TAK1 kinase <br />                                  (MAP3K7) required tumor cell viability. The induction apoptosis RNAi-mediated <br />                                  depletion pharmacologic inhibition TAK1 linked suppression hyperactivated Wnt <br />                                  signaling, evident endogenous genetically reconstituted cells. In APC-mutant/KRAS- <br />                                  dependent cells, KRAS stimulates BMP-7 secretion BMP signaling, leading TAK1 <br />                                  activation enhancement Wnt-dependent transcription. An vitro-derived &#226;&#8364;&#339;TAK1- <br />                                  dependency signature&#226;&#8364;? enriched primary human colon cancers mutations APC <br />                                  KRAS, suggesting potential clinical utility stratifying patient populations. Together, these <br />                                  findings identify TAK1 inhibition potential therapeutic strategy treatment-refractory <br />                                  subset colon cancers exhibiting aberrant KRAS Wnt pathway activation. <br />  <br />  <br />                            Introduction <br />                                              Targeted cancer therapies exploited specific mutations drive survival signals in <br />                                              subsets tumors, leading successful genotype-directed clinical applications small <br />                                              molecule inhibitors (reviewed (Haber et al., 2011). However, KRAS activating mutations, <br />                                              common multiple human cancers, remain critical therapeutic challenge. <br />                                              KRAS mutations generally associated treatment-refractory tumors (Downward, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              2003). For instance, KRAS mutant lung cancers generally refractory EGFR-targeted <br />                                              small molecule inhibitors colon cancer, KRAS mutations predict failure response <br />                                              antibodies targeting overexpressed wild-type EGFR (Normanno et al., 2009). Thus, <br />                                              patients KRAS-mutant colon cancers excluded targeted EGFR therapies and <br />                                              faced limited therapeutic options. <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2012 Elsevier Inc. All rights reserved. <br />                            * <br />                             Co-corresponding authors: Daniel Haber (Haber@helix.mgh.harvard.edu) Jeff Settleman (settleman.jeffrey@gene.com). <br />                            3Current addresses: Departmental Pharmacology Experimental Therapeutics, Division Medical Oncology Hematology, <br />                            Cancer Research Center, Boston University School Medicine, Boston MA 02118. <br />                            4Discovery Oncology, Genentech, Inc., South San Francisco, CA 94080 <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Singh et al.                                                                                          Page 2 <br />  <br />  <br />                                           <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> date, pharmacologic targeting activated KRAS successful. Mutationally- <br />                                           activated KRAS proteins susceptible agents target GTPase function drug <br />                                           therapies disrupt KRAS post-translational modifications clinically <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           efficacious (Whyte et al., 1997). More recently, genome-wide shRNA screens <br />                                           revealed synthetic lethal interactions knockdown candidate gene appeared to <br />                                           suppress proliferation KRAS mutant cancer cell lines (Barbie et al., 2009; Luo et al., <br />                                           2009a; Scholl et al., 2009; Singh et al., 2009). To date, findings led the <br />                                           development effective inhibitors treat KRAS-mutant cancers, possibly to <br />                                           contextual complexity KRAS mutations difficulty generalizing synthetic <br />                                           lethal interactions broad range tumor-specific backgrounds. <br />  <br />                                           In analyzing large panels KRAS-mutant tumor-derived cell lines, noted that <br />                                           approximately half cell lines underwent apoptosis following shRNA-mediated <br />                                           knockdown KRAS (so-called KRAS-dependent), half maintained <br />                                           viability (KRAS-independent) (Singh et al., 2009). These differences independent of <br />                                           particular KRAS <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, KRAS-dependent cells generally exhibited higher <br />                                           KRAS protein expression levels. Cellular context appeared play significant role the <br />                                           dependency KRAS mutant cells continued KRAS signaling survival. In both <br />                                           lung pancreatic cancer cells harboring KRAS mutations, presence epithelial <br />                                           markers highly correlated KRAS dependency, epithelial mesenchymal <br />                                           transformation (EMT) associated KRAS independence, despite presence a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           KRAS mutation. Reversion mesenchymal cells epithelial <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> associated <br />                                           restored dependence KRAS, suggesting mesenchymal-associated signals may <br />                                           provide alternative survival pathways KRAS activity disrupted (Singh et al., 2009). <br />                                           Interestingly, association KRAS dependency expression epithelial markers <br />                                           evident KRAS-mutant colon cancers prompting identifying <br />                                           lineage-specific determinants KRAS-dependency colon cancers. <br />  <br />                                           The cellular context KRAS mutations colon cancer complex. APC loss-of-function <br />                                           mutations arise early adenomas lead inappropriate activation canonical Wnt <br />                                           signaling, &#206;&#178;-catenin-mediated transcriptional changes (Behrens et al., 1996). KRAS <br />                                           mutations common progression adenoma carcinoma, occurring <br />                                           approximately 50% cases, subsequently followed frequent deletions of <br />                                           SMAD4, targeting transforming growth factor beta(TGF-&#206;&#178;) signaling pathway. <br />                                           (Vogelstein et al., 1988). The contribution KRAS mutations colon carcinogenesis is <br />                                           uniquely linked altered Wnt TGF-&#206;&#178; signaling. <br />  <br />                                           Here, compared KRAS-dependent -independent colon cancer cells using a <br />                                           combination gene expression shRNA knockdown studies, led the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           identification MAP3K7, encoding TGF-&#206;&#178; activated kinase (TAK1), driver cell <br />                                           survival KRAS-dependent, APC-deficient cells. Together recently reported <br />                                           enhancement Wnt signaling KRAS (Janssen et al., 2006; Phelps et al., 2009), these <br />                                           observations point unappreciated critical signaling node subset colon <br />                                           cancers. We demonstrate that, KRAS-dependent, KRAS-independent cells, <br />                                           KRAS activates Bone Morphogenetic Protein-7 (BMP-7) signaling, leading TAK1 <br />                                           activation, &#206;&#178;-catenin nuclear localization, transcriptional upregulation Wnt target <br />                                           genes. This accompanied KRAS- TAK1-regulated activation NF-&#206;&#186;B <br />                                           pathway. Reconstitution studies confirm activation KRAS-dependent signaling <br />                                           network underlies exquisite sensitivity TAK1 inhibition. Together, observations <br />                                           point potential therapeutic strategy, based targeting vulnerable node an <br />                                           identifiable subset APC/KRAS mutant colon cancers. <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                           Page 3 <br />  <br />  <br />                               Results <br />                               Identification KRAS-dependent colon cancer cell lines <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           We used lentiviral-based shRNA assay quantitate KRAS dependency (Singh et al., <br />                                           2009) 21 KRAS-mutant colon cancer cell lines, measuring cell viability 6 days post- <br />                                           infection (Figure 1A). KRAS-mutant colon cancer cells showed variable KRAS- <br />                                           dependencies (Figures 1A 1B), allowing derive quantitative Ras Dependency <br />                                           Index (RDI) compare multiple cell lines varying viral transduction efficiencies (see <br />                                           Experimental Procedures). An RDI &gt;2.0 represented threshold classify cells KRAS- <br />                                           dependent. Among 21 KRAS-mutant cell lines, 10 classified KRAS-dependent <br />                                           11 KRAS-independent (Figure 1B). KRAS dependency associated with <br />                                           particular KRAS activating mutations (Table S2). Examples KRAS-dependent cell <br />                                           lines (SW620 SK-CO-1) selected comparison KRAS-independent <br />                                           lines (LS-174T SW1463) (Figure 1A). <br />  <br />                                           KRAS depletion KRAS-dependent colon cancer cells triggered apoptosis, measured by <br />                                           caspase-3 polyADP ribose polymerase (PARP) cleavage 6-days following shRNA <br />                                           knockdown (Figure 1C). Cells classified KRAS-independent despite presence of <br />                                           mutant KRAS showed apoptotic response KRAS depletion. Reduced Erk Akt <br />                                           phosphorylation preceded apoptosis KRAS-dependent cells, KRAS-independent <br />                                           cells displayed weak KRAS coupling Erk phosphorylation. Moreover, KRAS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           independent cells, KRAS depletion resulted paradoxically increased Akt phosphorylation, <br />                                           agreement recent reports (Ebi et al., 2011) (Figure 1D). Thus, KRAS-dependent and <br />                                           -independent colon cancer cells demonstrate distinct patterns signaling downstream of <br />                                           mutant KRAS, KRAS-dependent cells showing suppression key survival <br />                                           signals following KRAS knockdown. <br />  <br />                               TAK1 KRAS dependency-associated kinase <br />                                           To identify potentially &#226;&#8364;&#339;druggable&#226;&#8364;? pro-survival effectors KRAS-dependent colon cancer <br />                                           cells, compared gene expression profiles KRAS-dependent KRAS- <br />                                           independent cell lines (Figure 2A; Experimental Procedures). A core &#226;&#8364;&#339;KRAS <br />                                           Dependency Gene Set&#226;&#8364;? identified, comprising 687 genes overexpressed KRAS- <br />                                           independent cells (IND genes) 832 genes overexpressed KRAS-dependent cells (DEP <br />                                           genes). Hierarchical clustering KRAS Dependency Gene Set 40 colon cancer <br />                                           cell lines wild-type mutant KRAS demonstrated 3 clusters: IND, DEP and <br />                                           intermediate (Figure S1A). Gene ontology analysis DEP gene set, using DAVID <br />                                           algorithm (Dennis et al., 2003) identified major functional classes, kinases were <br />                                           abundant (Figure S1B). We selected analysis, given possibility <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           identifying novel tractable therapeutic targets. The 47 DEP protein, lipid nucleotide <br />                                           kinase genes showed significant overexpression KRAS-dependent colon cancer cells, <br />                                           confirmed subset protein level (Figures 2B C). The DEP gene set <br />                                           prominently featured genes relevant mitotic checkpoint control DNA replication/ <br />                                           repair pathways (KEGG pathway database) (Figure S1C). Of note, Wnt signaling <br />                                           components significantly enriched KRAS-dependent cells, compared KRAS- <br />                                           independent cells, despite classes having comparable frequency APC mutations <br />                                           (Figure S1C). <br />  <br />                                           We selected candidate protein kinase-encoding genes list 47, based on <br />                                           ranking DEP scores literature searches genes putative cancer- <br />                                           associated function. To establish functional relevance DEP kinases, we <br />                                           compared consequences knockdown cell lines comparable lentiviral <br />                                           infection profiles (KRAS-independent SW837 cells KRAS-dependent SW620 cells; <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                            Page 4 <br />  <br />  <br />                                           Figures S1D-F). We targeted 17 kinases, using 5 shRNAs 3 different viral MOIs, <br />                                           measuring relative cell densities 6 days post-infection (Figure S1E S1F). Among all <br />                                           kinases tested, TAK1 (MAP3K7) depletion potent selective effect on <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           viability SW620 versus SW837 cells, measured cumulative effect shRNA <br />                                           constructs tested (Figure 2D). Two genes, VRK2 encoding vaccinia related kinase <br />                                           isoform 2 CHUK encoding I-&#206;&#186;B kinase alpha, demonstrated selective albeit less <br />                                           potent effects SW620 cell viability (Figures S1E, S1F 2D). TAK1 depletion in <br />                                           KRAS-dependent SW620 cells remarkable producing strong, viral titer-dependent <br />                                           apoptotic response, assessed PARP cleavage (Figure 2E). Since not <br />                                           functionally validated kinase-encoding genes original list 47, remains <br />                                           formally possible additional untested kinases play stronger pro-survival roles than <br />                                           tested. <br />  <br />                               Validation TAK1 therapeutic target KRAS-dependent colon cancer <br />                                           To validate TAK1 candidate therapeutic target context, used potent <br />                                           selective TAK1 kinase inhibitor, 5Z-7-oxozeaenol (Rawlins et al., 1999). We tested <br />                                           sensitivity 5Z-7-oxozeaenol panel 47 colon cancer cell lines various <br />                                           genotypes (Figures 3A S2A). KRAS BRAF genotypes procured <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           Sanger Institute&apos;s Catalog Somatic Mutations (COSMIC) determined targeted <br />                                           resequencing (Table S2). Among KRAS-mutant cells, classified KRAS-dependent <br />                                           virtue sensitivity KRAS shRNA knockdown highly sensitive TAK1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           inhibition, KRAS-independent cells generally resistant (P&lt;0.0001). Notably, <br />                                           10 BRAF-mutant cell lines tested, 5 sensitive 5Z-7-oxozeaenol (Figure 3A). <br />                                           The majority cells wild-type KRAS BRAF 5Z-7-oxozeaenol-resistant, <br />                                           mutations potentially impact Ras pathway (e.g. NF1 mutation in <br />                                           HT55 cells ALK mutation CoCM-1 cells &#226;&#8364;&#8220; COSMIC, Sanger Institute) were <br />                                           moderately sensitive 5Z-7-oxozeaenol. Consistent colon cancer derivation, <br />                                           cell lines tested harbored APC mutations; note, cell lines wild-type <br />                                           APC harboring downstream CTNNB1 (&#206;&#178;-catenin) activating mutations (either S33Y or <br />                                           S45 missense mutations) resistant TAK1 inhibition (Table S2). <br />  <br />                                           To determine sensitivity TAK1 inhibition specific colon cancer-derived cell lines, <br />                                           assessed sensitivity 5Z-7-oxozeaenol 5 KRAS mutant pancreatic ductal <br />                                           adenocarcinoma (PDAC) 4 non-small cell lung cancer (NSCLC) cells, are <br />                                           APC wild-type (Figure S2B). Whether previously classified KRAS-dependent - <br />                                           independent (Singh et al., 2009), PDAC NSCLC cells largely refractory 5Z-7- <br />                                           oxozeaenol treatment. Finally, non-transformed epithelial cell lines 5Z-7- <br />                                           oxozeaenol-refractory &#226;&#8364;&#8220; MCF10A (IC50 = 5.5&#206;&#188;M) MDCK (IC50 = 22&#206;&#188;M) (Figure <br />                                           S2A). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Pharmacologic TAK1 inhibition triggered apoptosis KRAS-dependent colon cancer cells, <br />                                           measured PARP caspase-3 cleavage (Figure 3B). In cells, 5Z-7-oxozeaenol <br />                                           treatment caused reduced threonine 172 phosphorylation AMP-activated kinase (p- <br />                                           AMPK), established TAK1 regulated kinase (Xie et al., 2006). In contrast, KRAS- <br />                                           independent cells displayed little 5Z-7-oxozeaenol-mediated caspase-3 PARP <br />                                           cleavage, high doses, AMPK phosphorylation unaffected. Thus, low <br />                                           concentrations 5Z-7-oxozeaenol, range 0.625 1.25 &#206;&#188;M, promote apoptosis <br />                                           selectively KRAS-dependent colon cancer cells. <br />  <br />                                           To validate efficacy 5Z-7-oxozeaenol vivo, generated subcutaneous xenografted <br />                                           tumors NOD/SCID mice using representative KRAS mutant cell lines: HCT8 and <br />                                           SW837 (KRAS-independent), SK-CO-1 SW620 (KRAS-dependent) (Figure 3C). <br />                                           Palpable tumors evident weeks post-implantation, time mice treated <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                          Page 5 <br />  <br />  <br />                                           daily intraperitoneal 15mg/kg 5Z-7-oxozeaenol vehicle (Rawlins et <br />                                           al., 1999). Tumor imaging demonstrated remarkable regression KRAS-dependent <br />                                           tumors 6 days treatment. In contrast, tumors derived KRAS- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           independent cell lines showed significant response TAK1 inhibition. No overt toxicity <br />                                           evident 5Z-7-oxozeaenol-treated mice selected dosing regimen. <br />  <br />                               A gene expression signature associated sensitivity TAK1 inhibition <br />                                           To identify molecular mechanisms underlying sensitivity TAK1 inhibition, isolated <br />                                           subsets genes KRAS DEP gene set highly correlated 5Z-7- <br />                                           oxozeaenol sensitivity. We employed K-means clustering (Gasch Eisen, 2002) for <br />                                           unsupervised <span id='am-393' about='xsp:pattern' typeof='owl:Thing'>pattern</span> recognition KRAS dependency gene set test set 21 colon <br />                                           cancer cell lines sensitivity TAK1 inhibition determined (Figure S3A). <br />                                           By setting parameters k=3 clusters, identified 10 nodes (0 9) representing <br />                                           synexpression groups co-regulated genes. We correlated average expression scores <br />                                           genes node IC50 values 5Z-7-oxozeaenol linear regression <br />                                           modeling, computed coefficients determination (r2) p-values node/ <br />                                           IC50 correlation (Figure S3B). This analysis revealed nodes genes (Figure S3B and <br />                                           C) expression strongly correlated sensitivity TAK1 inhibition. We <br />                                           combined genes nodes generate 32 gene &#226;&#8364;&#339;TAK1 dependency signature&#226;&#8364;?. <br />  <br />                                           Clustering 32 genes 21 colon cancer cell lines demonstrated high degree of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           concordance expression TAK1 dependency gene set, sensitivity TAK1 <br />                                           inhibition degree KRAS dependency (Figure 4A). Average expression the <br />                                           TAK1 dependency signature significantly correlated previously derived RDI <br />                                           values KRAS mutant cell lines shown Figure 4A (P&lt;0.0001, Figure S3E). Three <br />                                           general classes cell lines appear analysis: KRAS-dependent, TAK1 inhibitor- <br />                                           sensitive cell lines highest expression TAK1 dependency signature; KRAS- <br />                                           independent, TAK1 inhibitor-refractory cells weak expression signature; a <br />                                           cluster cell lines intermediate levels expression, demonstrating enrichment for <br />                                           BRAF mutations (4 6 cell lines). <br />  <br />                                           Since Wnt pathway enrichment KRAS-dependent cells (Figure S1C), we <br />                                           overlapped TAK1 dependency signature dataset binding sites Wnt- <br />                                           regulated transcription factor TCF4, derived ChIP-on-Chip analyses (Hatzis et al., <br />                                           2008). Of 32 TAK1 dependency genes, 18 contained proximal TCF4 binding sites. A <br />                                           number genes, BAMBI, PROX1 NAV2 (HELAD1) been <br />                                           previously linked colon tumorigenesis context deregulated Wnt signaling (Lin et <br />                                           al., 2008; Petrova et al., 2008; Sekiya et al., 2004). Indeed, KRAS-dependent <br />                                           cell lines higher basal Wnt signaling activity KRAS-independent cell lines as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           measured TOP-FLASH TCF4-responsive luciferase assays (Figure 4B). <br />  <br />                                           Remarkably, applied primary colon cancer dataset (Reid et al., 2009), TAK1 <br />                                           dependency signature distinguished tumors mutations APC KRAS from <br />                                           APC mutations (Figure S3C). In particular, subset TAK1 dependency <br />                                           genes identified Wnt targets expressed higher levels APC/KRAS mutant <br />                                           primary colon cancers compared APC mutant/KRAS wild-type tumors (Figure 4C). <br />                                           While observations imply increased Wnt signaling KRAS-mutant cancers, note <br />                                           established Wnt target genes (e.g. MYC TCF7 (Figure S3D)) not <br />                                           enriched APC/KRAS mutant tumors. Taken altogether, gene expression analyses <br />                                           suggest combination APC KRAS mutations colon cancers associated <br />                                           Wnt pathway hyperactivation correlated susceptibility TAK1 inhibition. <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                           Page 6 <br />  <br />  <br />                               KRAS TAK1 regulate &#206;&#178;-catenin transcriptional activity nuclear localization <br />                                           To explore role KRAS TAK1 modulating Wnt signaling, assessed the <br />                                           effect KRAS depletion &#206;&#178;-catenin/TCF transcription panel KRAS mutant cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           lines using TOP-FLASH reporter (Figures 5A S4A). The KRAS-dependent cells <br />                                           SW1116 SK-CO-1 exhibited decreased TOP-FLASH reporter activity following KRAS <br />                                           depletion, correlated level KRAS knockdown (Figure 5A). In <br />                                           contrast, KRAS depletion effect KRAS-independent line (SW1463) and <br />                                           increased TOP-FLASH activity (LS174T). In KRAS-dependent cells, 24h 5Z-7- <br />                                           oxozeaenol treatment strongly suppressed TOP-FLASH activity dose-dependent manner <br />                                           (IC50 0.8&#206;&#188;M 2.5&#206;&#188;M) (Figure 5B). In contrast, TAK1 inhibition weaker effect <br />                                           TOP-FLASH activity KRAS-independent cells (IC50 &gt; 10&#206;&#188;M). Of note, SW837 cells <br />                                           exhibited biphasic response 5Z-7-oxozeaenol, increased TOP-FLASH activity at <br />                                           low doses reduced activity high dose 5&#206;&#188;M. To define effect TAK1 <br />                                           signaling physiological Wnt target gene, measured protein expression levels the <br />                                           endogenous Axin 2 gene (Lustig et al., 2002), following treatment 5Z-7-oxozeaenol. <br />                                           TAK1 inhibition resulted dose-dependent reduction Axin 2 expression KRAS- <br />                                           dependent cells, KRAS-independent cells (Figure 5C). Thus, KRAS and <br />                                           TAK1 suppression selectively suppress &#206;&#178;-catenin-mediated transcription Wnt target <br />                                           gene expression KRAS-dependent cells. <br />  <br />                                           Since activation Wnt signaling associated nuclear translocation &#206;&#178;-catenin, we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           analyzed subcellular localization following TAK1 suppression immunofluorescence <br />                                           microscopy. Parental vehicle-treated KRAS-dependent SW1116 SK-CO-1 cells <br />                                           nuclear &#206;&#178;-catenin localization, addition co-localization E-cadherin at <br />                                           adherens junctions. TAK1 inhibition cells resulted loss nuclear &#206;&#178;-catenin <br />                                           24h. No effect seen KRAS-independent LS174T SW1463 cells. <br />                                           (Figures 5D S4B). Thus, inhibition TAK1 signaling causes reduced &#206;&#178;-catenin nuclear <br />                                           localization KRAS-dependent KRAS-independent cells. <br />  <br />                               Reconstitution TAK1 dependency KRAS Wnt activation <br />                                           To determine TAK1-independent cells driven TAK1 dependency <br />                                           enhanced KRAS/Wnt signaling, undertook series reconstitution experiments. <br />                                           HT29 C2BBe1 colon cancer cells, mutant APC wild-type KRAS, exhibit very <br />                                           little basal TCF/LEF reporter activity demonstrate low undetectable nuclear &#206;&#178;-catenin <br />                                           signal (Figures 5H S4D). These cell lines insensitive 5Z-7-oxozeaenol. To <br />                                           determine activation KRAS sufficient increase Wnt signaling hence <br />                                           lead sensitivity TAK1 inhibition, ectopically introduced mutant KRAS(G12V) in <br />                                           HT29 cells phosphoglycerate kinase (PGK) promoter-driven expression. Expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           4A 4B splice isoforms mutant KRAS cells resulted 3-fold <br />                                           reduction IC50 5Z-7-oxozeaenol (Figures S4C 5E). In contrast, ectopic <br />                                           expression mutant NRAS equivalent expression levels caused slightly increased <br />                                           resistance 5Z-7-oxozeaenol. <br />  <br />                                           The increased TAK1 dependency resulting ectopic mutant KRAS HT29 cells was <br />                                           correlated 5-fold upregulated &#206;&#178;-catenin transcriptional activity, blocked a <br />                                           dose-dependent manner TAK1 inhibition (Figures 5F). Similar results obtained a <br />                                           second KRAS wild-type, TAK1-independent cell line (C2BBe1) (Figures S4D). Ectopic <br />                                           expression mutant KRAS resulted increased TAK1 autophosphorylation, Erk <br />                                           phosphorylation, elevated Axin 2 levels nuclear &#206;&#178;-catenin localization (Figures 5G and <br />                                           5H). TAK1 inhibition reversed KRAS-induced &#206;&#178;-catenin nuclear localization. Finally, <br />                                           pretreatment cells 5Z-7-oxozeaenol, prior transduction mutant KRAS, <br />                                           abrogated KRAS-mediated increase Wnt signaling (Figure S4E). Taken together, <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                         Page 7 <br />  <br />  <br />                                           ectopic expression mutant KRAS sufficient activate TAK1 APC-deficient cells, <br />                                           leading increased Wnt signaling sensitization TAK1 inhibition. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           To test role Wnt signaling context, investigated KRAS mutant <br />                                           pancreatic cancer (PDAC) cell lines, PANC-1 YAPC, APC wild-type. <br />                                           PANC-1 cells KRAS-independent, YAPC cells KRAS-dependent, a <br />                                           distinction linked increased KRAS signaling YAPC cells (Singh et al., <br />                                           2009). Activation canonical Wnt signaling inhibition GSK-3 using selective <br />                                           inhibitor BIO caused strong, dose-dependent TOP-FLASH reporter induction KRAS- <br />                                           dependent YAPC cells, compared weak induction KRAS-independent PANC-1 <br />                                           cells (Figure S4F). Simultaneous treatment YAPC cells 5Z-7-oxozeaenol abrogates <br />                                           BIO-mediated TOP-FLASH induction. In contrast, PANC-1 cells undergo stronger <br />                                           induction TOP-FLASH activity combined GSK-3 TAK1 inhibition (Figure S4G, <br />                                           lower upper panel). While viability KRAS-dependent YAPC cells greatly <br />                                           suppressed combined GSK-3 TAK1 inhibition, effect seen with <br />                                           PANC-1 cells (Figure S5H). Finally, introduced constitutively-activated mutant &#206;&#178;- <br />                                           catenin (CTNNB1-CA) containing S33Y S45A missense mutations, KRAS- <br />                                           dependent SW620 cells. This mutant caused partial losses KRAS TAK1 <br />                                           dependencies (Figures S6A-C). Taken altogether, reconstitution studies indicate that <br />                                           KRAS Wnt pathway hyperactivation contribute TAK1 dependency. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               KRAS activates TAK1 enhanced BMP signaling <br />                                           TAK1 encodes effector BMP receptor, activated response BMP <br />                                           ligand binding. Our TAK1 dependency signature notably enriched TGF-&#206;&#178;/BMP <br />                                           pathway components, including BMP7, BAMBI INHBB (Figure 4A). To test a <br />                                           direct role BMP pathway KRAS-mediated activation TAK1, measured the <br />                                           expression BMP receptor ligand BMP7 markers BMP activation following <br />                                           KRAS depletion. In KRAS-dependent SW620 SK-CO-1 cells, KRAS depletion caused <br />                                           reduced BMP-7 expression, BMP-mediated phosphorylation effector Smad1, and <br />                                           TAK1 autophosphorylation (Figure 6A). Of note, predominant phospho-TAK1 <br />                                           immunoreactive band context 40kD isoform, isoforms (40kD and <br />                                           75kD) observed depleted TAK1 shRNA (Figure 2E). Finally, Axin 2 levels were <br />                                           suppressed following KRAS depletion, indicating KRAS signaling enhances BMP <br />                                           signaling Wnt activation (Figure 6A). These effects KRAS depletion seen in <br />                                           KRAS-independent LS-174T SW837 cells. <br />  <br />                                           Given observed KRAS-regulated expression BMP7 SW620 cells, tested the <br />                                           functional role ligand using lentiviral shRNA-mediated knockdown (Figure 6B). <br />                                           BMP-7 depletion using panel 5 different shRNAs caused pronounced viral titer- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           dependent apoptosis (Figure 6B). Similarly, knockdown BMPR1A, encoding BMP <br />                                           receptor type 1A (Alk-3) component KRAS dependency gene set, <br />                                           suppressed proliferation viability KRAS-dependent SW620 cells (Figure 2D). In <br />                                           KRAS-independent SW837 cells did observe significant proliferation viability <br />                                           defects following BMP7 depletion (Figure S5A S5B). Thus, autocrine BMP-7 ligand <br />                                           expression receptor activation required maintain viability KRAS-dependent <br />                                           cells. <br />  <br />                                           To determine BMP-7 induction direct consequence KRAS activation, as <br />                                           opposed indirect effect cell transformation, introduced inducible mutant <br />                                           KRAS-estrogen receptor chimera (ER-KRAS(12V)) HT29 cells, normally <br />                                           express wild-type endogenous KRAS. At 24h following KRAS induction using 4- <br />                                           hydroxytamoxifen (4-HT), BMP7 mRNA levels increased, cellular and <br />                                           secreted BMP-7 protein levels (Figure 6C). Endogenous levels Axin 2 also <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                          Page 8 <br />  <br />  <br />                                           increased following KRAS induction, TOP-FLASH reporter activity. The activation <br />                                           downstream markers Wnt signaling inducible KRAS effectively <br />                                           suppressed depletion BMP7, BMPR1A TAK1 (Figure 6D), consistent their <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           function key mediators KRAS-potentiated Wnt pathway activation. <br />  <br />                                           To define role BMP signaling TAK1 dependency, ectopically expressed <br />                                           constitutively-activated (CA) variant (Q233D) BMPR1A (Zou et al., 1997) HT29 <br />                                           cells. Expression BMPR1A-CA conferred increased sensitivity 5Z-7-oxozeaenol with <br />                                           IC50 value 1.1&#206;&#188;M compared 7.7&#206;&#188;M vector control cells (Figure 6E). TAK1 <br />                                           inhibition BMPR1A-CA-expressing cells resulted apoptosis shown caspase-3 and <br />                                           PARP cleavage (Figure 6F). This accompanied dose-dependent decreases Axin 2 <br />                                           phosphorylated Smad2 levels. Finally, BMPR1A-CA-induced nuclear accumulation of <br />                                           &#206;&#178;-catenin, suppressed following TAK1 inhibition (Figure 6G). Thus, APC- <br />                                           mutant cells low baseline &#206;&#178;-catenin transcriptional activity, artificial BMP activation <br />                                           enhances Wnt signaling TAK1 activation, conferring TAK1 dependency. <br />                                           Surprisingly, introduction BMPR1A-CA KRAS-dependent SW620 cells not <br />                                           rescue KRAS depletion-induced cell death (Figure 6D), suggesting BMP receptor <br />                                           activation necessary sufficient promote <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>cell survival</span> KRAS-dependent <br />                                           cancer cells. In summary, mutant KRAS promotes autocrine BMP signaling causing TAK1 <br />                                           activation leading enhanced Wnt signaling APC-deficient colon cancer cells. In <br />                                           KRAS-dependent cells, components necessary anti-apoptotic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           signaling. <br />  <br />                               Discussion <br />                                           By analyzing KRAS-dependent subset KRAS-mutant colon cancer cells, we <br />                                           uncovered pathway KRAS enhances Wnt activity BMP/TAK1 <br />                                           activation. Approximately half colon cancer cell lines KRAS APC <br />                                           mutations appear rely pathway viability, rendering sensitive TAK1 <br />                                           kinase inhibition. As such, TAK1 inhibition provide clinical paradigm context- <br />                                           dependent targeting KRAS-dependent colon cancers. Our data suggest TAK1 <br />                                           functions pro-survival mediator cancer cells displaying hyperactive KRAS-dependent <br />                                           Wnt signaling. This seen basal conditions colon cancers relevant <br />                                           genotypes synthetically achieved activating Wnt signaling GSK3 kinase <br />                                           inhibition KRAS-dependent/APC-wild-type pancreatic cancer cells enforced <br />                                           expression mutant KRAS APC mutant/KRAS wild-type colon cancer cells. The ability <br />                                           reconstitute pathway dependency unusual &#226;&#8364;&#339;oncogene addiction&#226;&#8364;? models, and <br />                                           facilitates molecular dissection critical signaling components drive drug <br />                                           susceptibility. An underlying basis explained emerging concept of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           &#226;&#8364;&#339;non-oncogene addiction&#226;&#8364;?, describing acquired dependence cells non-mutated <br />                                           genes drive malignant progression, function essential for <br />                                           cell tolerate oncogenic stress-induced states (Luo et al., 2009b). While TAK1 <br />                                           dependency restricted colon cancer, elevated Wnt signaling activity in <br />                                           KRAS-dependent colon cancer cells highlights importance cellular context the <br />                                           role lineage-specific pathways informing effective therapeutic strategy. <br />  <br />                                           Through combination knockdown reconstitution experiments, attempted to <br />                                           dissect key signaling components linking mutant KRAS TAK1 Wnt activation <br />                                           (Figures S6 7). A strong relationship Wnt KRAS signaling underscored <br />                                           observation constitutive activation Wnt signaling causes loss KRAS <br />                                           dependency KRAS-dependent colon cancer cells. Our data indicate KRAS regulates <br />                                           TAK1 Wnt signaling APC-deficient cells upregulation BMP-7 levels BMP <br />                                           receptor activation. However, BMP receptor activation necessary KRAS- <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                            Page 9 <br />  <br />  <br />                                           driven survival signaling, sufficient. Thus, network KRAS-regulated signaling <br />                                           components likely contribute tumor cell survival. For instance, KRAS regulates NF- <br />                                           &#206;&#186;B TAK1 activation (Figures S6E 7). The relative contribution NF-&#206;&#186;B <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pathway KRAS-driven survival signaling remains determined, evidence <br />                                           suggests pathway critical KRAS-driven lung tumorigenesis (Meylan et al., <br />                                           2009; Starczynowski et al., 2011). Additional parallel pathways likely components <br />                                           KRAS-TAK1 survival signaling axis, data suggest Wnt pathway <br />                                           activation critical. <br />  <br />                                           Our studies highlight context-specific role KRAS driving Wnt signaling the <br />                                           sensitized background APC deficiency. This consistent recent studies reporting <br />                                           KRAS-mediated enhancement Wnt signaling zebrafish developmental model (Phelps <br />                                           et al., 2009). Indeed, observed APC-deficient colon cancers low &#206;&#178;-catenin <br />                                           activity, introduction mutant KRAS causes sharp increase levels nuclear &#206;&#178;-catenin, <br />                                           accompanied increased TCF/LEF transcriptional activity. This effect partly involves <br />                                           KRAS-mediated up-regulation BMP signaling subsequent TAK1 activation, leading <br />                                           enhanced TCF/LEF activity. Interestingly, C. elegans TAK1 ortholog Mom-4 <br />                                           promotes nuclear retention &#206;&#178;-catenin ortholog Wrm-1 asymmetrically 2-cell <br />                                           stage EMS cell defining polarity axis specification (Nakamura et al., <br />                                           2005; Shin et al., 1999). Such context-specific TAK1/&#206;&#178;-catenin interaction points a <br />                                           remarkable degree evolutionary conservation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           From clinical perspective, role secreted BMP-7 particular since <br />                                           autocrine paracrine activation pathway detectable targetable in <br />                                           tumors. Importantly, expression BMP pathway components help stratify colon <br />                                           cancer patients TAK1 inhibitor response groups. To end, suggest 10 <br />                                           genes vitro derived TAK1 dependency signature (GGH, BMP7, BAMBI, <br />                                           MBOAT2, HSPA12A, FYN, NAV2, RGL1, SYK RUNX1) provide clinically <br />                                           annotated signature TAK1 inhibitors sufficiently developed clinical trials. This <br />                                           eventually applied clinical diagnostic test measure relative mRNA levels <br />                                           corresponding ten-gene TAK1 dependency signature patient tumors. We note that <br />                                           half KRAS mutant colon cancer cell lines KRAS-dependent and <br />                                           sensitive TAK1 inhibition, account quarter colon <br />                                           cancers. As such, guided accurate molecular profiles, TAK1 inhibitors provide <br />                                           significant clinical benefit recalcitrant form colon cancer. Beyond tool <br />                                           compounds 5Z-7-oxozeaenol, synthetic TAK1 inhibitors tested in <br />                                           preclinical models (Melisi et al., 2011). However, given potential toxicity, administration <br />                                           regimens need modeled using highly TAK1-dependent cancers. Finally, study <br />                                           illustrates presence KRAS mutation does identify homogenously drug- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           resistant tumor type, specific histological type. Instead, degrees KRAS <br />                                           dependency different cancers modulated associated signaling pathways the <br />                                           Wnt pathway colon cancers. This adds complexity analysis ultimately <br />                                           expected inform unique therapeutic opportunities. <br />  <br />                               Experimental Procedures <br />                               Derivation Ras Dependency Index (RDI) <br />                                           Weighted averages relative cell densities MOIs 5 1 KRAS A B <br />                                           shRNAs calculated. The inverse averages calculated. This number <br />                                           multiplied transduction efficiency respective cell line (the proportion of <br />                                           cells expressing control shRNA following puromycin selection compared not <br />                                           treated puromycin), yielding RDI value. An RDI 2 calibrated 50% <br />                                           reduction cellular proliferation following KRAS depletion. <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                                         Page 10 <br />  <br />  <br />                               Gene Expression Microarray Analyses <br />                                           Comparative whole-genome expression profiling performed Affymetrix U133A <br />                                           Microarrays. The dataset colon cancer cell lines used publically available <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           BROAD Institute (http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi) under <br />                                           Sanger Cell Line Project. Expression data normalized using GCRMA (Bolstad et al., <br />                                           2003). To derive KRAS dependency gene set, p-values computed comparing <br />                                           average normalized probe intensity probe set cell lines shown Figure <br />                                           2A. Fold differences average probe intensities calculated. The p-value &#226;&#710;&#8217;log <br />                                           transformed fold difference log2 transformed. The product log-transformed <br />                                           values designated &#226;&#8364;&#339;DEP SCORE&#226;&#8364;? (Table S1) genes ranked based on <br />                                           score. Significantly over- under-expressed genes (p&lt;0.05; fold difference&gt;2) were <br />                                           identified method. To generate heatmaps, complete linkage hierarchical clustering by <br />                                           Euclidean city-block distance performed using Cluster (Eisen et al., 1998) Java <br />                                           Treeview (http://sourceforge.net/projects/jtreeview/). Statistical analyses correlations with <br />                                           gene expression data performed Graphpad Prism. Patient data available <br />                                           NCBI GEO database (accession #: GSE16125). <br />  <br />                               In vivo Pharmacology Xenografted Mouse Tumors <br />                                           Human colorectal cancer tumor cells trypsinized resuspended single cell <br />                                           suspensions injected left right flanks NOD/SCID mice. Mice housed in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pathogen-free environment Massachusetts General Hospital handled in <br />                                           strict accordance Good Animal Practice defined Office Laboratory Animal <br />                                           Welfare, animal experiments approval Massachusetts General <br />                                           Hospital Subcommittee Research Animal Care. Tumor size monitored daily and <br />                                           tumor volume reached approximately 200mm3, treatment 5Z-7-oxozeaenol <br />                                           initiated (7 14 days post-implantation). Mice injected daily 15mg/kg 5Z-7- <br />                                           oxozeaenol intraperitoneal administration. <br />  <br />                               Reporter assays <br />                                           For TOP-FLASH NF-&#206;&#186;B reporter assays, cells co-transfected 0.5&#206;&#188;g <br />                                           FOP-FLASH TOP-FLASH plasmids pGL3-&#206;&#186;B-LUC plus 50ng pRL-TK <br />                                           (expressing Renilla luciferase). Normalized luciferase activity obtained using the <br />                                           Dual-Luciferase Reporter Assay System (Promega Inc). Alternatively, stable cell lines <br />                                           expressing TOP-FLASH reporter generated transducing cells 7TFP (Fuerer <br />                                           Nusse, 2010) recombinant lentiviruses followed drug selection. <br />  <br />                               Supplementary Material <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                           This work supported NIH K99 CA149169 A.S. NIH R01 CA109447 Lustgarten <br />                                           Foundation grant J.S NIH R01 CA129933 D.A.H. We grateful Randall Peterson, David Ting and <br />                                           Andre Bernards critical evaluation comments manuscript. We thank Laura Libby Brian <br />                                           Brannigan technical assistance mouse xenograft experiments KRAS/BRAF sequencing analyses, <br />                                           respectively. We thank Vijay Yajnik providing TOP-FLASH reporter plasmids. Finally, thank Kevin <br />                                           Haigis, Moon Yee Hang, Miguel Rivera, Jianmin Zhang, Mingzhu Liu Matt Zubrowski intellectual and <br />                                           technical advice. <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                                 Page 11 <br />  <br />  <br />                               References <br />                                           Barbie DA, Tamayo P, Boehm JS, Kim SY, Moody SE, Dunn IF, Schinzel AC, Sandy P, Meylan E, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Scholl C, et al. Systematic <span id='am-315' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-316' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>RNA interference</span> reveals oncogenic KRAS-driven cancers require <br />                                              TBK1. Nature. 2009; 462:108&#226;&#8364;&#8220;112. [PubMed: 19847166] <br />                                           Behrens J, von Kries JP, Kuhl M, Bruhn L, Wedlich D, Grosschedl R, Birchmeier W. Functional <br />                                              interaction beta-catenin transcription factor LEF-1. Nature. 1996; 382:638&#226;&#8364;&#8220;642. <br />                                              [PubMed: 8757136] <br />                                           Bolstad BM, Irizarry RA, Astrand M, Speed TP. A comparison normalization methods high <br />                                              density oligonucleotide array data based variance bias. Bioinformatics. 2003; 19:185&#226;&#8364;&#8220;193. <br />                                              [PubMed: 12538238] <br />                                           Campeau E, Ruhl VE, Rodier F, Smith CL, Rahmberg BL, Fuss JO, Campisi J, Yaswen P, Cooper PK, <br />                                              Kaufman PD. A versatile viral expression depletion proteins mammalian cells. <br />                                              PLoS One. 2009; 4:e6529. [PubMed: 19657394] <br />                                           Dennis G Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, Lempicki RA. DAVID: Database <br />                                              Annotation, Visualization, Integrated Discovery. Genome Biol. 2003; 4:P3. [PubMed: <br />                                              12734009] <br />                                           Downward J. Targeting RAS signalling pathways cancer therapy. Nat Rev Cancer. 2003; 3:11&#226;&#8364;&#8220;22. <br />                                              [PubMed: 12509763] <br />                                           Ebi H, Corcoran RB, Singh A, Chen Z, Song Y, Lifshits E, Ryan DP, Meyerhardt JA, Benes C, <br />                                              Settleman J, et al. Receptor tyrosine kinases exert dominant control PI3K signaling human <br />                                              KRAS mutant colorectal cancers. J Clin Invest. 2011; 121:4311&#226;&#8364;&#8220;4321. [PubMed: 21985784] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Eisen MB, Spellman PT, Brown PO, Botstein D. Cluster analysis display genome-wide <br />                                              expression patterns. Proc Natl Acad Sci U S A. 1998; 95:14863&#226;&#8364;&#8220;14868. [PubMed: 9843981] <br />                                           Fuerer C, Nusse R. Lentiviral vectors probe manipulate Wnt signaling pathway. PLoS One. <br />                                              2010; 5:e9370. [PubMed: 20186325] <br />                                           Gasch AP, Eisen MB. Exploring conditional coregulation yeast gene expression fuzzy <br />                                                k-means clustering. Genome Biol. 2002; 3 RESEARCH0059. <br />                                           Haber DA, Gray NS, Baselga J. The evolving war cancer. Cell. 2011; 145:19&#226;&#8364;&#8220;24. [PubMed: <br />                                                21458664] <br />                                           Hatzis P, van der Flier LG, van Driel MA, Guryev V, Nielsen F, Denissov S, Nijman IJ, Koster J, <br />                                                Santo EE, Welboren W, et al. Genome-wide pattern TCF7L2/TCF4 chromatin occupancy in <br />                                                colorectal cancer cells. Mol Cell Biol. 2008; 28:2732&#226;&#8364;&#8220;2744. [PubMed: 18268006] <br />                                           Janssen KP, Alberici P, Fsihi H, Gaspar C, Breukel C, Franken P, Rosty C, Abal M, El Marjou F, <br />                                                Smits R, et al. APC oncogenic KRAS synergistic enhancing Wnt signaling intestinal <br />                                                tumor formation progression. Gastroenterology. 2006; 131:1096&#226;&#8364;&#8220;1109. [PubMed: 17030180] <br />                                           Lin Z, Gao C, Ning Y, He X, Wu W, Chen YG. The pseudoreceptor BMP activin membrane- <br />                                                bound inhibitor positively modulates Wnt/beta-catenin signaling. J Biol Chem. 2008; 283:33053&#226;&#8364;&#8220; <br />                                                33058. [PubMed: 18838381] <br />                                           Luo J, Emanuele MJ, Li D, Creighton CJ, Schlabach MR, Westbrook TF, Wong KK, Elledge SJ. A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                genome-wide RNAi screen identifies multiple synthetic lethal interactions Ras oncogene. <br />                                                Cell. 2009a; 137:835&#226;&#8364;&#8220;848. [PubMed: 19490893] <br />                                           Luo J, Solimini NL, Elledge SJ. Principles cancer therapy: oncogene non-oncogene addiction. <br />                                                Cell. 2009b; 136:823&#226;&#8364;&#8220;837. [PubMed: 19269363] <br />                                           Lustig B, Jerchow B, Sachs M, Weiler S, Pietsch T, Karsten U, van Wetering M, Clevers H, Schlag <br />                                                PM, Birchmeier W, et al. Negative feedback loop Wnt signaling upregulation of <br />                                                conductin/axin2 colorectal liver tumors. Mol Cell Biol. 2002; 22:1184&#226;&#8364;&#8220;1193. [PubMed: <br />                                                11809809] <br />                                           Melisi D, Xia Q, Paradiso G, Ling J, Moccia T, Carbone C, Budillon A, Abbruzzese JL, Chiao PJ. <br />                                                Modulation pancreatic cancer chemoresistance inhibition TAK1. J Natl Cancer Inst. 2011; <br />                                                103:1190&#226;&#8364;&#8220;1204. [PubMed: 21743023] <br />                                           Meylan E, Dooley AL, Feldser DM, Shen L, Turk E, Ouyang C, Jacks T. Requirement NF-kappaB <br />                                                signalling mouse model lung adenocarcinoma. Nature. 2009; 462:104&#226;&#8364;&#8220;107. [PubMed: <br />                                                19847165] <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                                   Page 12 <br />  <br />  <br />                                           Moffat J, Grueneberg DA, Yang X, Kim SY, Kloepfer AM, Hinkle G, Piqani B, Eisenhaure TM, Luo <br />                                               B, Grenier JK, et al. A lentiviral RNAi library human mouse genes applied arrayed <br />                                               viral high-content screen. Cell. 2006; 124:1283&#226;&#8364;&#8220;1298. [PubMed: 16564017] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Nakamura K, Kim S, Ishidate T, Bei Y, Pang K, Shirayama M, Trzepacz C, Brownell DR, Mello CC. <br />                                               Wnt signaling drives WRM-1/beta-catenin asymmetries early C. elegans embryos. Genes Dev. <br />                                               2005; 19:1749&#226;&#8364;&#8220;1754. [PubMed: 16077004] <br />                                           Naldini L, Blomer U, Gallay P, Ory D, Mulligan R, Gage FH, Verma IM, Trono D. In vivo gene <br />                                               delivery stable transduction nondividing cells lentiviral vector. Science. 1996; <br />                                               272:263&#226;&#8364;&#8220;267. [PubMed: 8602510] <br />                                           Normanno N, Tejpar S, Morgillo F, De Luca A, Van Cutsem E, Ciardiello F. Implications KRAS <br />                                               status EGFR-targeted therapies metastatic CRC. Nat Rev Clin Oncol. 2009; 6:519&#226;&#8364;&#8220;527. <br />                                               [PubMed: 19636327] <br />                                           Petrova TV, Nykanen A, Norrmen C, Ivanov KI, Andersson LC, Haglund C, Puolakkainen P, Wempe <br />                                               F, von Melchner H, Gradwohl G, et al. Transcription factor PROX1 induces colon cancer <br />                                               progression promoting transition benign highly dysplastic phenotype. Cancer Cell. <br />                                               2008; 13:407&#226;&#8364;&#8220;419. [PubMed: 18455124] <br />                                           Phelps RA, Chidester S, Dehghanizadeh S, Phelps J, Sandoval IT, Rai K, Broadbent T, Sarkar S, Burt <br />                                               RW, Jones DA. A two-step model colon adenoma initiation progression caused APC <br />                                               loss. Cell. 2009; 137:623&#226;&#8364;&#8220;634. [PubMed: 19450512] <br />                                           Rawlins P, Mander T, Sadeghi R, Hill S, Gammon G, Foxwell B, Wrigley S, Moore M. Inhibition of <br />                                               endotoxin-induced TNF-alpha production macrophages 5Z-7-oxo-zeaenol fungal <br />                                               resorcylic acid lactones. Int J Immunopharmacol. 1999; 21:799&#226;&#8364;&#8220;814. [PubMed: 10606001] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Reid JF, Gariboldi M, Sokolova V, Capobianco P, Lampis A, Perrone F, Signoroni S, Costa A, Leo E, <br />                                               Pilotti S, et al. Integrative approach prioritizing cancer genes sporadic colon cancer. Genes <br />                                               Chromosomes Cancer. 2009; 48:953&#226;&#8364;&#8220;962. [PubMed: 19672874] <br />                                           Scholl C, Frohling S, Dunn IF, Schinzel AC, Barbie DA, Kim SY, Silver SJ, Tamayo P, Wadlow RC, <br />                                               Ramaswamy S, et al. Synthetic lethal interaction oncogenic KRAS dependency and <br />                                               STK33 suppression human cancer cells. Cell. 2009; 137:821&#226;&#8364;&#8220;834. [PubMed: 19490892] <br />                                           Sekiya T, Adachi S, Kohu K, Yamada T, Higuchi O, Furukawa Y, Nakamura Y, Nakamura T, Tashiro <br />                                               K, Kuhara S, et al. Identification BMP activin membrane-bound inhibitor (BAMBI), an <br />                                               inhibitor transforming growth factor-beta signaling, target beta-catenin pathway in <br />                                               colorectal tumor cells. J Biol Chem. 2004; 279:6840&#226;&#8364;&#8220;6846. [PubMed: 14660579] <br />                                           Shin TH, Yasuda J, Rocheleau CE, Lin R, Soto M, Bei Y, Davis RJ, Mello CC. MOM-4, MAP <br />                                               kinase kinase kinase-related protein, activates WRM-1/LIT-1 kinase transduce anterior/posterior <br />                                               polarity signals C. elegans. Mol Cell. 1999; 4:275&#226;&#8364;&#8220;280. [PubMed: 10488343] <br />                                           Singh A, Greninger P, Rhodes D, Koopman L, Violette S, Bardeesy N, Settleman J. A gene expression <br />                                               signature associated &#226;&#8364;&#339;K-Ras addiction&#226;&#8364;? reveals regulators EMT tumor cell survival. <br />                                               Cancer Cell. 2009; 15:489&#226;&#8364;&#8220;500. [PubMed: 19477428] <br />                                           Starczynowski DT, Lockwood WW, Delehouzee S, Chari R, Wegrzyn J, Fuller M, Tsao MS, Lam S, <br />                                               Gazdar AF, Lam WL, et al. TRAF6 amplified oncogene bridging RAS NF-kappaB <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               pathways human lung cancer. J Clin Invest. 2011; 121:4095&#226;&#8364;&#8220;4105. [PubMed: 21911935] <br />                                           Vogelstein B, Fearon ER, Hamilton SR, Kern SE, Preisinger AC, Leppert M, Nakamura Y, White R, <br />                                               Smits AM, Bos JL. Genetic alterations colorectal-tumor development. N Engl J Med. 1988; <br />                                               319:525&#226;&#8364;&#8220;532. [PubMed: 2841597] <br />                                           Whyte DB, Kirschmeier P, Hockenberry TN, Nunez-Oliva I, James L, Catino JJ, Bishop WR, Pai JK. <br />                                               K- N-Ras geranylgeranylated cells treated farnesyl protein transferase inhibitors. J <br />                                               Biol Chem. 1997; 272:14459&#226;&#8364;&#8220;14464. [PubMed: 9162087] <br />                                           Xie M, Zhang D, Dyck JR, Li Y, Zhang H, Morishima M, Mann DL, Taffet GE, Baldini A, Khoury <br />                                               DS, et al. A pivotal role endogenous TGF-beta-activated kinase-1 LKB1/AMP-activated <br />                                               protein kinase energy-sensor pathway. Proc Natl Acad Sci U S A. 2006; 103:17378&#226;&#8364;&#8220;17383. <br />                                               [PubMed: 17085580] <br />                                           Zou H, Wieser R, Massague J, Niswander L. Distinct roles type I bone morphogenetic protein <br />                                               receptors formation differentiation cartilage. Genes Dev. 1997; 11:2191&#226;&#8364;&#8220;2203. <br />                                               [PubMed: 9303535] <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                  Page 13 <br />  <br />  <br />                                                                            Research Highlights <br />                                           &#226;&#8364;&#162;    The TAK1 kinase promotes survival KRAS-dependent colon cancer cells. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           &#226;&#8364;&#162;    TAK1 dependency correlates KRAS-mediated activation Wnt signaling. <br />                                           &#226;&#8364;&#162;    KRAS activates Wnt signaling TAK1 APC-deficient cancer cells. <br />                                           &#226;&#8364;&#162;    KRAS upregulates BMP signaling leading increased TAK1 kinase activity. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                          Page 14 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 1. Classification KRAS mutant colon cancer cells KRAS-independent KRAS- <br />                                           dependent groups <br />                                           (A) Representative 6-day 96-well viability assays 4 KRAS mutant colon cancer cell lines <br />                                           transduced control 2 independent KRAS-directed lentiviral shRNAs (A and <br />                                           B), 2 viral MOIs. Cell lines red text KRAS-independent, green text are <br />                                           KRAS-dependent. Quantitation transformation relative cell density values yields the <br />                                           Ras Dependency Index depicted Fig. 1B. <br />                                           (B) Ras Dependency Index plot panel 21 KRAS mutant colon cancer cell lines. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Dashed line represents &#226;&#8364;&#339;Dependency Threshold&#226;&#8364;? 2.0. Data presented mean <br />                                           independent experiments +/- SEM. <br />                                           (C) KRAS protein depletion 4 days post-infection KRAS-directed shRNAs effects <br />                                           apoptosis, assessed caspase-3 PARP cleavage, representative panel of <br />                                           KRAS-dependent versus-KRAS independent cell lines. Lanes 1, 2 3 panel A. <br />                                           Data representative independent experiments. <br />                                           (D) Activating phosphorylations Erk (p-Erk1/2) Akt (p-Akt) kinases, following <br />                                           KRAS depletion SW837 KRAS-independent versus SW620 2 KRAS-dependent cells, 4 <br />                                           days post-infection 3 different viral titres (MOIs 1, 2 4) shKRAS-B. Total <br />                                           protein levels (t-Erk1 and t-Akt) shown gel loading controls. Note: different <br />                                           exposure times used individual panels. Data representative two <br />                                           independent experiments. <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 2. Analysis kinases &#226;&#8364;&#339;KRAS dependency signature&#226;&#8364;? colon cancer cell lines <br />                                           (A) Schematic representation methodology used derive colon cancer KRAS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           dependency gene expression data set. Gene expression microarray data 4 indicated <br />                                           KRAS-independent versus KRAS-dependent cell lines analyzed significantly <br />                                           underexpressed (IND) overexpressed (DEP) genes student T-test analysis (two-tailed, <br />                                           homoscedastic) followed selection probe sets average expression 2-fold <br />                                           higher lower, yielding 687 IND genes 832 DEP genes. <br />                                           (B) Hierarchical clustering gene expression 47 DEP &#226;&#8364;&#339;druggable&#226;&#8364;? protein, lipid or <br />                                           ATP-dependent kinase genes kinase regulatory genes. Heat map shows log2 <br />                                           median-centered intensity values similarly expressed genes clustered using <br />                                           Euclidean distance similarity metric. MAP3K7 (encoding TAK1) highlighted an <br />                                           asterisk. <br />                                           (C) Protein expression levels indicated kinases panel KRAS-independent and <br />                                           KRAS-dependent cell lines. GAPDH serves loading control. <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                            Page 16 <br />  <br />  <br />                                           (D) Depletion DEP kinase genes SW620 versus SW837 cells. Each colored bar <br />                                           represents individual shRNA sequence gene, color-coding Figure <br />                                           S1F. Fold growth inhibition shRNA kinase computed dividing relative <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           cell density SW837 SW620 cells using weighted average account for <br />                                           viral titre. The plot shows cumulative log2 fold growth inhibition shRNA per <br />                                           kinase; i.e., value 1 plot indicates 2-fold greater growth inhibitory effect a <br />                                           given shRNA SW620 compared SW837 cells. The log2 fold growth inhibition each <br />                                           individual shRNA cumulated kinase gene. Data represented the <br />                                           mean value corresponding shRNA independent experiments. <br />                                           (E) Knockdown TAK1 increasing viral titres shTAK1-D encoding lentiviruses <br />                                           (MOI) associated apoptotic effects assessed PARP cleavage. GAPDH serves a <br />                                           loading control. Data representative independent experiments. <br />                                           See Figure S1 Table S1. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 3. Validation MAP3K7/TAK1 pro-survival mediator KRAS-dependent colon <br />                                           cancers <br />                                           (A) IC50 values (&#206;&#188;M) effects cellular proliferation viability TAK1 <br />                                           kinase inhibitor 5Z-7-oxozeaenol panel colon cancer cell lines been <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           genotyped KRAS mutant (KRAS-independent &#226;&#8364;&#8220; red circles KRAS-dependent &#226;&#8364;&#8220; green <br />                                           squares), BRAF mutant (blue triangles) wild-type KRAS BRAF (OTHER &#226;&#8364;&#8220; <br />                                           grey diamonds). Effects growth measured 3 days post-treatment. Data are <br />                                           represented mean 3 independent experiments error bars indicate median &#194;&#177; <br />                                           interquartile range. *denotes p&lt;0.00001; n.s. &#226;&#8364;&#8220; significant. <br />                                           (B) Effects TAK1 inhibition apoptosis signaling representative panel of <br />                                           KRAS-independent KRAS-dependent cell lines, 24h treatment. PARP and <br />                                           caspase-3 cleavage shown indicators apoptosis, AMPK threonine 172 (T172) <br />                                           phosphorylation shown downstream indicator TAK1 signaling activity. GAPDH <br />                                           serves gel loading control. <br />                                           (C) TAK1 inhibition mice xenografted human tumors derived HCT8/ <br />                                           SW837 (KRAS-independent) SK-CO-1/SW620 (KRAS-dependent) cell lines. Cells <br />                                           expressing firefly luciferase injected subcutaneously flanks nude mice. <br />                                           Tumors shown imaged IVIS detection luminescence counts (in photons/sec) <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                     Page 18 <br />  <br />  <br />                                           following 14 days tumor growth followed 6 days treatment 15mg/kg <br />                                           5z-7-oxozeaenol vehicle (5% DMSO arachis oil), IP delivery q.d. Quantitation of <br />                                           tumor volume (mm3) shown. Tumor volume data represented mean 4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tumors 2 mice group +/- SEM. <br />                                           See Figure S2 Table S2. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                       Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 4. Associations KRAS dependency gene set, TAK1 dependence KRAS- <br />                                           driven canonical Wnt signaling colon cancer patients <br />                                           (A) Heat map representation gene expression correlated TAK1 dependence <br />                                           KRAS dependency gene set panel colon cancer cell lines various <br />                                           genotypes. Cell lines ordered IC50 values 5Z-7-oxozeaenol, leftmost the <br />                                           highest rightmost lowest. Clustering genes performed Euclidean <br />                                           distance similarity metric. Values presented log2 median-centered intensities. <br />                                           Genes highlighted orange text putative bona fide TCF4 target genes. <br />                                           (B) Basal normalized TCF4 luciferase reporter activity (TOP-FLASH) nominal units a <br />                                           panel KRAS-independent KRAS-dependent colon cancer cell lines. Data are <br />                                           represented means 3 independent experiments +/- SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (C) Average expression non-TCF4 TCF4 target genes depicted Figure 4A colon <br />                                           cancer patients genotyped APC mutant/KRAS-wild-type (red circles) APC <br />                                           mutant plus KRAS mutant (green squares). P-values represent comparison mean <br />                                           expression scores genes class. <br />                                           See Figure S3. <br />  <br />  <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                          Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 5. KRAS TAK1 regulate canonical Wnt signaling KRAS-dependent cancer cells <br />                                           (A) TOP-FLASH luciferase reporter activity function lentiviral shRNA-mediated <br />                                           KRAS depletion increasing MOIs LS174T/SW1463 (KRAS-independent) versus <br />                                           SW620/SK-CO-1 (KRAS-dependent) cells. Cell lines transduced stably express <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           luciferase control TCF4 response elements. Reporter activity plotted relative <br />                                           shGFP (vector) expressing cells. Data represented mean triplicate <br />                                           experiments +/- SEM. <br />                                           (B) TOP-FLASH activity KRAS-independent KRAS-dependent cell lines following <br />                                           TAK1 inhibition increasing concentrations 5Z-7-oxo (&#206;&#188;M). Data represented as <br />                                           means triplicate experiments &#194;&#177; SEM. <br />                                           (C) Protein expression levels endogenous Wnt target gene Axin 2 following treatment <br />                                           cells indicated concentrations 5Z-7-oxo. GAPDH serves loading control. <br />                                           (D) Laser confocal micrographs SW1116 KRAS-dependent cells treated either <br />                                           DMSO vehicle 5&#206;&#188;M 5Z-7-oxo 24h. E-cadherin localization shown red <br />                                           channel, &#206;&#178;-catenin green DAPI-stained nuclei blue. Scale bar = 20&#206;&#188;M. <br />                                           (E) Forced overexpression epitope-tagged oncogenic G12V mutated RAS protein <br />                                           isoforms HT29 cells sensitivity TAK1 pharmacological inhibition 5Z-7- <br />                                           oxozeaenol. Expression levels exogenous endogenous Ras proteins shown by <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                     Page 21 <br />  <br />  <br />                                           immunoblotting pan-ras monoclonal antibody. NRAS/KRAS4B HA-tagged and <br />                                           KRAS4A V5-tagged. <br />                                           (F) Overexpression mutant KRAS(12V) followed TAK1 inhibition HT29 cells and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           effects TOP-FLASH reporter activity. Data presented means three <br />                                           independent experiments +/- SEM. <br />                                           (G) Overexpression KRAS(12V) HT29 cells effects TAK1 Erk <br />                                           phosphorylation (p-TAK1/p-Erk) Axin 2 levels. Total TAK1 Erk1 serve as <br />                                           loading controls. <br />                                           (H) Confocal micrographs showing E-cadherin KRAS (red) &#206;&#178;-catenin (green) <br />                                           localization vector control oncogenic HA tagged KRAS-4B(12V) expressing HT29 <br />                                           cells. KRAS expression visualized using HA polyclonal antibody. Scale bar = 25&#206;&#188;m. <br />                                           See Figure S4. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                          Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 6. Oncogenic KRAS regulates BMP-7/BMPR1A signaling <br />                                           (A) Depletion KRAS KRAS-independent (LS-174T SW837) KRAS- <br />                                           dependent cell lines (SW620 SK-CO-1) subsequent effects expression BMP-7 <br />                                           downstream effects Smad1/TAK1 phosphorylation (p-Smad1/p-TAK1). The <br />                                           20kD secreted form BMP7 shown. Phospho-TAK1 represents TAK1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           autophosphorylation site measure TAK1 activity. Total Smad1/5/8 total <br />                                           TAK1 (t-Smad1/5/8/t-TAK1) proteins shown gel loading controls. Data are <br />                                           representative independent experiments. <br />                                           (B) Effects BMP7 depletion proliferation viability SW620 KRAS-dependent <br />                                           cells. Plot shows cell density 6 days post-infection shGFP control 5 different <br />                                           BMP7-directed lentiviral shRNAs. Data represented mean independent <br />                                           experiments &#194;&#177; SEM. Western blots right panel BMP-7 levels apoptotic <br />                                           effects measured PARP Caspase3 cleavage following BMP-7 depletion two <br />                                           independent lentiviral shRNAs (D E). <br />                                           (C) Effects BMP7 protein transcript levels following induced activation ER- <br />                                           KRAS(12V) fusion protein various doses 4-HT HT29 cells. Left panel shows <br />                                           levels total secreted BMP-7 following ER-KRAS(12V) induction 4-HT. Levels <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                       Page 23 <br />  <br />  <br />                                           Axin 2 phosphorylated Erk (p-Erk1/2) shown following ER-KRAS(12V). <br />                                           Total Erk (t-Erk1) serves loading control. <br />                                           (D) TOP-FLASH reporter activity following 4-HT induced activation ER-KRAS(12V) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           depletion indicated genes lentiviral shRNA delivery various viral titres. <br />                                           Reporter activity shown relative shGFP control. <br />                                           (E) Introduction V5-tagged constitutively activated (CA) mutant BMP receptor, <br />                                           BMPR1A (Q233D) control vector HT29 cells effects 5Z-7-oxozeaenol <br />                                           sensitivity terms IC50 values. <br />                                           (F) Signaling apoptotic effects TAK1 inhibition using 5Z-7-oxozeaenol the <br />                                           indicated concentrations 24h post-treatment BMPR1A-CA expressing cells. Caspase3 and <br />                                           PARP cleavage indicators apoptotic cell death. Axin 2 levels shown readout <br />                                           Wnt signaling. Phosphorylated smad1/5/8 levels serve readout BMP signaling. <br />                                           GAPDH serves gel loading control. BMPR1A-CA expression visualized using a <br />                                           monoclonal V5 antibody. <br />                                           (G) Effects BMPR1A-CA expression &#206;&#178;-catenin localization (red) HT-29 cells <br />                                           following treatment 5&#206;&#188;M 5Z-7-oxozeaenol vehicle control 24h, assessed by <br />                                           immunofluorescence confocal microscopy. DAPI-stained nuclei shown blue. Scale <br />                                           bar = 10&#206;&#188;M. <br />                                           See Figure S5. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2012 August 17. <br />                             Singh et al.                                                                                         Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 7. A model context specific KRAS dependency colon cancers <br />                                           In KRAS-independent colon cancers, APC loss function results hyperactivation of <br />                                           canonical Wnt signaling stabilization &#206;&#178;-catenin cooperation upstream <br />                                           Wnt activators. TAK1 negative regulator canonical Wnt signaling cells. <br />                                           In KRAS-dependent cells, oncogenic KRAS upregulates BMP-7 expression/secretion, <br />                                           activating BMP receptor <br /> </body></html>