<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                           ORIGINAL RESEARCH ARTICLE <br />                                                                                                                                 published: 25 May 2012 <br />                                                                                                                          doi: 10.3389/fmicb.2012.00183 <br />  <br />  <br />  <br /> Genomic biochemical analysis diaminopimelate <br /> and lysine biosynthesis pathway Verrucomicrobium <br /> spinosum: identi&#239;&#172;?cation partial characterization of <br /> L,L-diaminopimelate aminotransferase and <br /> UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-meso- <br /> diaminopimelate ligase <br /> Victoria R. Nachar 1 , Francisco C. Savka 1 , Sean E. McGroty 1 , Katherine A. Donovan 2 , Rachel A. North 2 , <br /> Renwick C. J. Dobson 2,3 , Larry J. Buckley 1 Andr&#195;&#169; O. Hudson 1 * <br /> 1 <br />   The Thomas H. Gosnell School Life Sciences, Rochester Institute Technology, Rochester, NY, USA <br /> 2 <br />   Biomolecular Interaction Centre, School Biological Sciences, University Canterbury, Christchurch, New Zealand <br /> 3 <br />   Department Biochemistry Molecular Biology, Bio21 Molecular Science Biotechnology Institute, The University Melbourne, Parkville, VIC, Australia <br />  <br />  <br />  <br /> Edited by:                                The Gram-negative bacterium Verrucomicrobium spinosum attracted recent <br /> Naomi L. Ward, University of <br />                                           years following sequencing annotation genome. Comparative genomic <br /> Wyoming, USA <br />                                           analysis V. spinosum using diaminopimelate/lysine metabolic genes Chlamydia tra- <br /> Reviewed by: <br /> Uli Stingl, King Abdullah University   chomatis suggests V. spinosum employs L,L-diaminopimelate aminotransferase <br /> Science Technology, Saudi Arabia      (DapL) pathway diaminopimelate/lysine biosynthesis. The open reading frame corre- <br /> Evelyn Toh, Indiana University            sponding putative dapL ortholog cloned recombinant enzyme was <br /> Bloomington, USA <br />                                           shown possess L,L-diaminopimelate aminotransferase activity vitro. In vivo analysis <br /> Yufeng Wang, University Texas at <br /> San Antonio, USA                          using functional complementation con&#239;&#172;?rmed dapL ortholog able func- <br /> *Correspondence:                          tionally complement E. coli mutant confers auxotrophy diaminopimelate and <br /> Andr&#195;&#169; O. Hudson, The Thomas H.            lysine. In addition role lysine biosynthesis, intermediate diaminopimelate an <br /> Gosnell School Life Sciences,          integral role peptidoglycan biosynthesis. <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> end, UDP-N-acetylmuramoylalanyl- <br /> Rochester Institute Technology, 85 <br />                                           D-glutamyl-2,6-meso-diaminopimelate ligase ortholog identi&#239;&#172;?ed, cloned, was <br /> Lomb Memorial Drive, Rochester NY <br /> 14623-5603, USA.                          shown possess meso-diaminopimelate ligase activity vivo. The L,L-diaminopimelate <br /> e-mail: aohsbi@rit.edu                    aminotransferase pathway experimentally con&#239;&#172;?rmed bacteria, some <br />                                           deemed pathogenic animals. Since animals, particularly humans, lack <br />                                           genetic machinery synthesis diaminopimelate/lysine novo, enzymes <br />                                           involved pathway attractive targets development antibiotics. Whether dapL <br />                                           essential gene bacteria currently known. V. spinosum excellent candi- <br />                                           date investigate essentiality dapL, bacterium employs DapL pathway <br />                                           lysine cell wall biosynthesis, non-pathogenic humans, facile grow, can <br />                                           genetically manipulated. <br />                                           Keywords: lysine, diaminopimelate, L,L-diaminopimelate aminotransferase, peptidoglycan, functional complemen- <br />                                           tation <br />  <br />  <br /> INTRODUCTION                                                                      diaminopimelate/lysine novo bacterium pro- <br /> Verrucomicrobium spinosum rod-shaped heterotrophic Gram-                     totrophic lysine. Lysine synthesized using general <br /> negative bacterium fresh water soil envi-                    pathways evolutionarily divergent respect the <br /> ronments characterized non-motile contains                    intermediates used. One pathway utilizes intermediate &#206;&#177;- <br /> appendages known prosthecae. V. spinosum important                       amino adipic acid, derived metabolism of <br /> the biotechnology medical sectors closely related                    2-ketoglutarate product citric acid cycle. The &#206;&#177;-amino <br /> to Chlamydia (Wagner Horn, 2006). A recent study con-                         adipic acid pathway primarily used fungi narrowly <br /> &#239;&#172;?rmed organism contains genetic components                    distributed species belonging domain archaea <br /> encode Type III Secretion System able kill Drosophila                (Nishida et al., 1999; Velasco et al., 2002). The pathway <br /> melanogaster Caenorhabditis elegans, model invertebrate                   employs intermediate diaminopimelate, derived <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> hosts (Sait et al., 2011).                                                        oxaloacetate. The diaminopimelate pathway bac- <br />    Genomic analysis V. spinousm organ-                           teria photosynthetic organisms including cyanobacteria, algae, <br /> ism contains genes necessary synthesis                         plants. <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                           May 2012 | Volume 3 | Article 183 | 1 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />     Four variants diaminopimelate/lysine pathway                   L,L-diaminopimelate tetrahydrodipicolinate transam- <br /> been discovered far (Figure 1): acyl pathways,                      ination reaction using glutamate amino donor and <br /> which use succinyl acetyl intermediates; meso-                   tetrahydrodipicolinate amino acceptor single reac- <br /> diaminopimelate dehydrogenase pathway; recently dis-                   tion bypassing 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate <br /> covered L,L-diaminopimelate aminotransferasae (DapL) pathway                   -acyl-transferase, acyl-diaminopimelate aminotransferase, acyl- <br /> (Hudson et al., 2006, 2008; McCoy et al., 2006). The biosyn-                   diaminopimelate deacylase steps present succinylated or <br /> thesis lysine aspartate diaminopimelate/lysine                 acetylated diaminopimelate/lysine pathways (Table 1, Figure 1). <br /> pathway divided main events. The &#239;&#172;?rst event                  The ultimate event diaminopimelate variant path- <br /> is synthesis tetrahydrodipicolinate aspartate. This             ways de&#239;&#172;?ned enzyme meso-diaminopimelate decarboxy- <br /> a general feature diaminopimelate pathway variants                 lase catalyzes decarboxylation meso-diaminopimelate <br /> and carried enzymes, aspartate kinase, aspartate                 synthesize lysine. This decarboxylation step shared four <br /> semialdehyde dehydrogenase, dihydrodipicolinate synthase,                  variants (Table 1, Figure 1). <br /> dihydrodipicolinate reductase, respectively (Table 1). The sec-                    Since animals contain machinery synthesize <br /> ond event constitutes conversion tetrahydrodipicolinate                 diaminopimelate lysine, used bacteria pep- <br /> to penultimate intermediate meso-diaminopimelate. The syn-                 tidoglycan biosynthesis, enzymes involved bacterial <br /> thetic steps tetrahydrodipicolinate meso-diaminopimelate               diaminopimelate/lysine biosynthesis pathways to <br /> de&#239;&#172;?ne uniqueness diaminopimelate variant pathways.                  scienti&#239;&#172;?c community. meso-Diaminopimelate serves one <br /> In acyl pathways, enzymes required. These reactions               cross linking amino acids cell wall Gram- <br /> are carried enzymes 2,3,4,5-tetrahydropyridine-2,6-                 negative bacteria, lysine role Gram- <br /> dicarboxylate N -acyl-transferase, acyl-diaminopimelate amino-                 positive bacteria (Hutton et al., 2007). Inhibition enzymes <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  diaminopimelate/lysine biosynthesis pathways have <br /> late epimerase, respectively (Table 1, Figure 1). In dehy-                 detrimental effect bacterial growth perspectives: <br /> drogenase pathway, tetrahydrodipicolinate converted meso-                diaminopimelate necessary cell wall synthesis, cell lysis <br /> diaminopimelate enzyme meso-diaminopimelate dehydro-                    occur result osmotic pressure lack pep- <br /> genase step circumventing 2,3,4,5-tetrahydropyridine-               tidoglycan (Cox, 1996; Baizman et al., 2000); addition, the <br /> 2,6-dicarboxylate-acyl-transferase, acyl-diaminopimelate amino-                inhibition enzymes pathway prevent protein syn- <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  thesis, lysine 20 common proteogenic amino <br /> late epimerase enzymatic reactions (Table 1, Figure 1). The L,L-               acids. meso-Diaminopimelate incorporated cross <br /> diaminopimelate aminotransferase (DapL) pathway synthesizes                    linking amino acids cell wall Gram-negative bacteria <br />  <br />  <br />  <br />  <br />   FIGURE 1 | Diaminopimelate/lysine biosynthesis pathways. The            pathways, (B) meso-diaminopimelate dehydrogenase pathway, (C) the <br />   variants diaminopimelate/lysine biosynthetic pathways starting   L,L-diaminopimelate aminotransferase (DapL) pathway. The abbreviations of <br />   tetrahydrodipicolinate denoted (A) succinylated acetylated     enzymes enzyme reaction de&#239;&#172;?nitions listed Table 1. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                        May 2012 | Volume 3 | Article 183 | 2 <br />  Nachar et al.                                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 1 | List diaminopimelate/lysine biosynthesis genes V. spinosum DSM 4136T . <br />  <br /> Locus tag                   Gene symbol          Gene                         EC #             Reaction <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> <br />  <br /> VspiD_010100025585          lysC                 Aspartate kinase                  2.7.2.4          ATP+aspartate &#226;&#8224;&#8221; ADP+4-phospho-aspartate <br /> VspiD_010100030480          asd                  Aspartate semialdehyde            1.2.1.11         aspartate 4-semialdehyde + orthophosphate + NADP+ &#226;&#8224;&#8221; 4- <br />                                                  dehydrogenase                                      phospho-L-aspartate + NADPH + H+ <br /> VspiD_010100011370          dapA                 Dihydrodipicolinate               4.2.1.52         Aspartate 4-semialdehyde + pyruvate &#226;&#8224;&#8221; L-2,3- <br />                                                  synthase                                           dihydrodipicolinate + 2H2 O <br /> VspiD_010100011365          dapB                 Dihydrodipicolinate               1.3.1.26         2,3,4,5-Tetrahydrodipicolinate + NADP + &#226;&#8224;&#8221; L-2,3- <br />                                                  reductase                                          dihydrodipicolinate + NADPH + H+ <br /> -                           dapD                 2,3,4,5-tetrahydropyridine-       2.3.1.117        Acyl-CoA + 2,3,4,5-tetrahydrodipicolinate + H2 O &#226;&#8224;&#8221; CoA + N- <br />                                                  2,6-dicarboxylate                                  acyl-2-L-amino-6-oxoheptanedioate <br />                                                  Nacyl-transferase <br /> -                           dapC                 Acyl-diaminopimelate              2.6.1.17         N-acyl-L,L-2,6-diaminoheptanedioate + 2-oxoglutarate&#226;&#8224;&#8221;N- <br />                                                  aminotransferase                                   acyl-2-L-amino-6-oxoheptanedioate + L-glutamate <br /> -                           dapE                 Acyl-diaminopimelate              3.5.1.18         N-acyl-L,L-2,6-diaminoheptanedioate + H2 O &#226;&#8224;&#8221; acyl + L,L- <br />                                                  deacylase                                          2,6-diaminoheptanedioate <br /> -                           ddh                  Diaminopimelate                   1.4.1.16         meso-2,6-Diaminoheptanedioate + NADP+ + H2O &#226;&#8224;&#8221; L-2- <br />                                                  dehydrogenase                                      amino-6-oxoheptanedioate + NH3 + NADPH <br /> VspiD_010100012510          dapL                 L,L-diaminopimelate               2.6.1.83         L,L-2,6-diaminopimelate + 2-oxoglutarate &#226;&#8224;&#8221; 2,3,4,5- <br />                                                  aminotransferase                                   tetrahydrodipicolinate + L-glutamate + H2 O <br /> VspiD_010100011375          dapF                 Diaminopimelate                   5.1.1.7          L,L-2,6-diaminopimelate&#226;&#8224;&#8221;meso-2,6-diaminopimelate <br />                                                  epimerase <br /> VspiD_010100003650          lysA                 Diaminopimelate                   4.1.1.20         meso-2,6-Diaminopimelate&#226;&#8224;&#8217;lysine + CO2 <br />                                                  decarboxylase <br />  <br /> The list generated using genomic information deposited IMG database (http://img.jgi.doe.gov/cgi-bin/w/main.cgi). <br />  <br />  <br />  <br /> by enzyme UDP-N -acetylmuramoylalanyl-d-glutamyl-2,6-                          diaminopimelate/lysine biosynthesis. In addition, identify and <br /> meso-diaminopimelate ligase (MurE) (E.C. 6.3.2.15),                       characterize MurE V. spinosum, suggesting bacterium <br /> encoded murE gene.                                                          incorporates meso-diaminopimelate peptidoglycan a <br />     The DapL pathway identi&#239;&#172;?ed numerous bacteria                       cross linking amino acid. <br /> and experimentally con&#239;&#172;?rmed limited number bac- <br /> teria, deemed pathogenic (McCoy et al., 2006;                    RESULTS <br /> Hudson et al., 2008; Liu et al., 2010). The three-dimensional crys-                IDENTIFICATION OF PUTATIVE dapL AND murE GENES IN V. SPINOSUM <br /> tal structure DapL Arabidopsis thaliana solved                    The dapL ortholog V. spinosum identi&#239;&#172;?ed using the <br /> (Watanabe et al., 2007, 2008) Chlamydia trachomatis (Watan-                   DapL protein C. trachomatis Ct 390 (NP_219900) the <br /> abe et al., 2011) algae Chlamydomonas reinhardtii                     query BlastP algorithm Integrated Microbial <br /> (Dobson et al., 2011; Hudson et al., 2011a). Inhibitors Ara-               Genomes (IMG) database1 . This search identi&#239;&#172;?ed putative L,L- <br /> bidopsis DapL ortholog reported using vitro studies                   diaminopimelate aminotransferase annotated locus tag <br /> (Fan et al., 2010). It noted inhibitors                    VspiD_010100012510 (ZP_02927470) 38% amino acid <br /> the Arabidopsis DapL found, known                     identity C. trachomatis DapL. The murE ortholog iden- <br /> compounds inhibit aminotransferases.                                         ti&#239;&#172;?ed using C. trachomatis MurE (NP_219774) query. This <br />     The identi&#239;&#172;?cation dapL murE genes, addition                       search resulted identi&#239;&#172;?cation murE ortholog V. <br /> the easy culturability genetic manipulation V. spinosum,                    spinosum annotated locus tag VspiD_010100019130, which <br /> makes bacterium excellent model organism experi-                        37% identical C. trachomatis MurE. <br /> ments addressing essentiality dapL murE respect <br /> to lysine peptidoglycan biosynthesis (Domman et al., 2011).                    CRUDE SOLUBLE PROTEIN EXTRACT FROM V. SPINOSUM CONTAINS <br /> Most bacterial species contain DapL pathway dif&#239;&#172;?-                     DapL ACTIVITY <br /> cult work with, slow growing, dif&#239;&#172;?cult                 If V. spinosum utilizes DapL pathway diaminopime- <br /> culture, anaerobic, pathogenic humans.                                      late/lysine biosynthesis, suspected genomic analysis, a <br />     Here present &#239;&#172;?rst genomic biochemical analysis                      crude soluble protein extract possess DapL activity. The <br /> of diaminopimelate/lysine biosynthesis pathway V. spin-                   ortho-aminobenzaldehyde assay employed using crude sol- <br /> osum. We identify genes necessary diaminopimelate/lysine                   uble protein extract V. spinosum. Using L,L-diaminopimelate <br /> biosynthesis genome V. spinosum demonstrate that <br /> the bacterium uses recently discovered DapL pathway                        1 http://www.jgi.doe.gov/ <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            May 2012 | Volume 3 | Article 183 | 3 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> as amino donor 2-ketoglutarate amino accep-                     consistent genomic analysis regarding absence a <br /> tor, assay measures production dihydroquinazolium                   diaminopimelate dehydrogenase ortholog genome V. <br /> which product interaction tetrahydrodipicol-             spinosum. <br /> inate ortho-aminobenzaldehyde. DapL activity detected <br /> from crude soluble extract showing rate                    THE GENE ANNOTATED BY THE LOCUS TAG VspiD_010100012510 <br /> reaction proportional V. spinosum pro-                     (VsdapL) ENCODES AN AUTHENTIC L,L-DIAMINOPIMELATE <br /> tein extract added assay (Figure 2). Enzymatic activity                 AMINOTRANSFERASE <br /> was observed amino donor, amino accep-                         The putative VsdapL gene cloned recombinant <br /> tor, omitted assay. In addition, activity                  enzyme puri&#239;&#172;?ed homogeneity using af&#239;&#172;?nity chromatogra- <br /> observed protein extract heated boiling water                  phy (Figure 3). The ortho-aminobenzaldehyde assay used to <br /> for 5 min.                                                                     determine putative VsDapL L,L-diaminopimelate <br />                                                                                aminotransferase activity. The results analysis that <br /> DapL IS THE ONLY ROUTE FOR DIAMINOPIMELATE/LYSINE                              open reading frame annotated locus tag encodes an <br /> BIOSYNTHESIS IN V. SPINOSUM                                                    authentic L,L-diaminopimelate aminotransferase enzyme. With <br /> The genome V. spinosum searched catalog genes                    L,L-diaminopimelate amino donor 2-ketoglutarate as <br /> necessary novo anabolism lysine aspartate.                  amino acceptor, speci&#239;&#172;?c activity VsDapL 4.1 &#194;&#177; 0.25 <br /> Since DapL activity detected soluble extract,                    &#206;&#8221;A 440 min&#226;&#710;&#8217;1 mg&#226;&#710;&#8217;1 . Unlike plant algae enzyme, VsDapL <br /> wanted know DapL pathway sole route                     possesses transamination activity diamine donors <br /> toward lysine biosynthesis. There examples litera-                  structurally similar L,L-diaminopimelate, including the <br /> ture multiple pathways lysine biosynthesis bacteria;                 racemic isomer meso-diaminopimelate (Table 2), albeit rela- <br /> for example, genomes Bacteroides fragilis Clostridium               tively low rates. The enzyme able use oxoacids <br /> thermocellum contain DapL meso-                     amino acceptors addition 2-ketoglutarate (Table 2); again <br /> diaminopimelate dehydrogenase pathways (Hudson et al., 2011b).                 relative rates low. <br /> Also, Corynebacterium glutamicum employs acyl pathway in <br /> conjunction Ddh pathway (Schrumpf et al., 1991). A                    VsdapL IS ABLE TO FUNCTIONALLY COMPLEMENT THE E. COLI <br /> search V. spinosum genome suggests DapL path-                  &#206;&#8221;dapD/dapE (AOH1) MUTANT <br /> way route diaminopimelate/lysine biosynthesis,                 The E. coli strain AOH1 harbors deletion dapD <br /> since orthologs acyl dehydrogenase genes                  <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>gene mutation</span> dapE renders enzyme non- <br /> be identi&#239;&#172;?ed genome (Table 1; Figure 1). Even                    functional (Hudson et al., 2006). As such, mutant unable <br /> the search did identify diaminopimelate dehydrogenase                    synthesize meso-diaminopimelate lysine cell wall <br /> ortholog, tested dehydrogenase activity using                  biosynthesis. As result, cells lyse osmotic pressure <br /> protein extract used detect DapL activity. This test               lack meso-diaminopimelate cross linking <br /> was using 1.0 mg total protein different buffer                 amino acid cell wall deemed auxotrophic for <br /> systems, &#239;&#172;?nding activity. The recombinant diaminopime-                      diaminopimelate lysine. For complementation analysis, E. <br /> late dehydrogenase Clostridium thermocellum used                      coli AOH1 transformed vector (pBAD33), <br /> as positive control (Hudson et al., 2011b). This result                  DapL expression vectors (pBAD33 + dapL) from <br />                                                                                plant Arabidopsis thaliana (AtdapL; Hudson et al., 2006), <br />                                                                                alga C. reinhardtii (CrdapL; Dobson et al., 2011) the <br />                                                                                bacterium V. spinosum (VsdapL). While E. coli AOH1 able <br />                                                                                grow media supplemented L,L-diaminopimelate, <br />                                                                                auxotrophic mutant expressing authentic orthologous <br />  <br />  <br />  <br />  <br />                                                                                 FIGURE 3 | Recombinant expression purification VsDapL using <br />  FIGURE 2 | DapL activity crude protein extract V. spinosum.          affinity chromatography. Lane (1) Protein Marker (kDa), Lane (2) 10 &#206;&#188;g of <br />  The graph shows relationship protein reaction       soluble protein uninduced cells, Lane (3) 10 &#206;&#188;g soluble proteins <br />  rate. The DapL ortho-aminobenzaldehyde assay described Section       induced cells, Lane (4) 1 &#206;&#188;g puri&#239;&#172;?ed VsDapL. The proteins were <br />  &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? The assay triplicates protein    resolved 10% (w/v) acrylamide gel stained Coomassie <br />  concentration.                                                                 blue visualization. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                       May 2012 | Volume 3 | Article 183 | 4 <br />  Nachar et al.                                                                    Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> DapLs able grow L,L-diaminopimelate-free media                  VsmurE IS ABLE TO FUNCTIONALLY COMPLEMENT THE E. COLI murE <br /> (Figure 4). This analysis demonstrates recombinant               MUTANT <br /> enzymes able convert tetrahydrodipicolinate L,L-                meso-Diaminopimelate serves penultimate pre- <br /> diaminopimelate, bypassing acyl enzymatic reactions present         cursor lysine biosynthesis, incorporated as <br /> in E. coli acyl pathway, facilitate synthesis meso-         cross linking amino acids cell wall Gram- <br /> diaminopimelate lysine cell wall biosynthesis (Figures 1          negative bacteria. This facilitated enzyme UDP-N - <br /> and 5).                                                                   acetylmuramoylalanyl-D-glutamyl-2,6-meso-diaminopimelate lig- <br />                                                                           ase (MurE; Figure 5). Since V. spinosum classi&#239;&#172;?ed Gram- <br /> VsDapL BELONGS TO THE DAPL1 FORM OF L,L-DIAMINOPIMELATE                   negative bacterium, possess MurE ortholog (Schlesner <br /> AMINOTRANSFERASES                                                         et al., 2006). The VsmurE ortholog identi&#239;&#172;?ed open <br /> Diaminopimelate aminotransferases approximately 400 amino             reading frame cloned expression vector test via <br /> acids <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> members pyridoxal-5  phosphate             functional complementation gene encodes authen- <br /> dependent family class I/II transaminases. A recent study              tic MurE enzyme. The E. coli mutant TKL-11 harbors muta- <br /> revealed diverged forms DapL enzymes;               tion murE gene results temperature sensitive <br /> DapL1 DapL2 (Hudson et al., 2008). The forms                   growth <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>phenotype</span> mutant able grow per- <br /> DapLs share approximately 30% homology amino acid                  missive temperature 30&#203;&#353;C, 42&#203;&#353;C (Lugtenberg and <br /> level. The DapL ortholog C. trachomatis evolutionar-              van Schijndel-van Dam, 1972). The mutant transformed with <br /> ily related DapL1 enzymes (Hudson et al., 2008). Since             vector (pBAD33) vector expressing puta- <br /> homology V. spinosum DapL 38% C. tra-               tive MurE (pBAD33 + VsmurE). Using replica-plating, results <br /> chomatis ortholog, form V. spinosum DapL               analysis demonstrate permissive temperature <br /> apparent. Using phylogenetic analysis, DapL ortholog             30&#203;&#353;C, mutant harboring vector control vec- <br /> V. spinosum clusters enzymes                 tor expressing recombinant enzyme able grow. However, <br /> experimentally con&#239;&#172;?rmed DapL1 orthologs patho-                exposed non-permissive temperature 42&#203;&#353;C, the <br /> genic organisms (Hudson et al., 2008, 2011b). The analysis shows          mutant expressing MurE ortholog able grow (Figure 7A). <br /> two distinct clades, DapL1, DapL2. The aspartate aminotrans-              This result corroborated assessing bacterial growth a <br /> ferase Yersinia pseudotuberculosis, classi&#239;&#172;?ed          period 10 h. At 30&#203;&#353;C mutant harboring vector and <br /> as class I/II transaminase used outgroup (Figure 6).           murE expression vector grew expected. However, the <br /> Four analyses performed (ML, MP, ME, NJ)                cultures switched non-permissive temperature 42&#203;&#353;C <br /> moderate strong support DapL2 clade (57&#226;&#8364;&#8220;100%)                  5 h 30&#203;&#353;C, mutant harboring murE expres- <br /> for DapL1 clade (56&#226;&#8364;&#8220;99%). The variable maximum                   sion vector continued grow. The optical density vector <br /> likelihood bootstrap support alignment dependent alter-           culture declined attributed rapid lysis the <br /> native alignments techniques (Clustal vs. MUSCLE) exhibiting              cell lack proper peptidoglycan synthesis (Figure 7B). <br /> higher ML bootstrap values DapL1 DapL2 clades                 The assessment crude soluble protein extracts com- <br /> respectively. Muscle-aligned sequences generally exhibit higher           plementation experiment using SDS-PAGE analysis con&#239;&#172;?rmed the <br /> bootstrap support DapL1 DapL2 clades                    expression recombinant MurE (59 kDa) TKL-11 cells <br /> Clustal.                                                                  harboring expression vector (pBAD33 + VsmurE) grown at <br />  <br />  <br />  <br />  <br />   FIGURE 4 | Functional complementation E. coli &#206;&#8221;dapD/dapE (AOH1) mutant. The AOH1 mutant harboring plasmids pBAD33, pBAD33 + AtdapL, <br />   pBAD33 + CrdapL, pBAD33 +VsdapL replica-plated LB medium supplemented 0.2% (w/v) arabinose 50 &#206;&#188;g/mL <br />   L,L-diaminopimelate. For presentation purposes colony vector recombinant plasmids represented diagram. <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                May 2012 | Volume 3 | Article 183 | 5 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 5 | Schematic representation bacterial peptidoglycan           muramoylalanine-D-glutamate ligase), MurE (UDP-N-acetylmuramoyl- <br />   synthesis using m-DAP cross linking amino acids. The           L-alanyl-D-glutamate 2,6-meso-diaminopimelate ligase), MurF <br />   cytosolic, membrane, periplasmic steps shown. The abbreviations   (UDP-N-acetylmuramoyl-tripeptide-D-alanyl-d-alanine ligase), Ddl <br />   enzymes follows: MurA (UDP-N-acetylglucosamine enolpyruvyl        (D-alanine-D-alanine ligase B), MraY (phospho-N-acetylmuramoyl- <br />   transferase), MurB (UDP-N-acetylenolpyruvoylglucosaminereductase), MurC      pentapeptide-transferase), MurG (pyrophosphoryl-undecaprenol- <br />   (UDP-N-acetylmuramate-L-alanine ligase), MurD (UDP-N-acetyl-                 N-acetylglucosamine transferase), PBPs (penicillin binding proteins). <br />  <br />  <br />  <br /> 42&#203;&#353;C present extract TKL-11 harboring              (residues red bold), putative active site residues well <br /> vector control (pBAD33) grown 30&#203;&#353;C (Figure 7C).                        conserved DapL enzymes. In addition, loop <br />                                                                                regions form active site cleft generally con- <br /> THE AMINO ACIDS THAT CONSTITUTE THE ACTIVE SITE ARE                            served. In particular, loops A C conserved, although <br /> CONSERVED IN VsDapL                                                            loop B shows variability. From previous structural studies <br /> To examine closely VsDapL variant amino acid                   (Watanabe et al., 2007, 2011; Dobson et al., 2011), known <br /> level, aligned amino acid sequence DapL                loop B disordered role catalysis clear. <br /> orthologs functionally structurally characterized             However, noting increased substrate promiscuity displayed by <br /> Arabidopsis thaliana (Watanabe et al., 2007), C. reinhardtii (Dob-             VsDapL, possible loop B involved substrate <br /> son et al., 2011), C. trachomatis (Watanabe et al., 2011;                  binding step. <br /> Figure 8). The VsDapL sequence similarity                         A homology model VsDapL generated using the <br /> CrDapL At DapL sequences, like Ct DapL sequence,                   Swiss-Model Protein Modeling Server (Figure 9). The tertiary <br /> has truncated N-terminal region. The longer N-terminal regions               structure DapL monomer annotated hav- <br /> for eukaryotic sequences attributed subcellular localiza-            ing large small domain, includes arm <br /> tion sequences, used A. thaliana C. reinhardtii target               region (Figure 9A). Most aminotransferases approximately <br /> the enzyme chloroplast, lysine biosynthesis known              100 kDa mass, representing homodimeric quaternary struc- <br /> to occur (Mills Wilson, 1978). As seen Figure 8                  ture. Assuming functional <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> VsDapL enzyme is <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                          May 2012 | Volume 3 | Article 183 | 6 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />  FIGURE 6 | Phylogenetic relationship DapL amino acid sequences.           Branch lengths proportional divergence; scale tree <br />  The VsDapL ortholog annotated black circle. Minimum evolution      indicates percent divergence branch lengths. The tree rooted <br />  (ME) topology shown maximum likelihood (ML), maximum                    single outgroup aspartate aminotransferase (AspC) from <br />  parsimony (MP), minimum evolution (ME), neighbor-joining (NJ)            Y. pseudotuberculosis. Figure drawing produced using FigTree (ver. 1.6.1; <br />  bootstrap values DapL1 DapL2 clades (from left right).   Rambaut, 2009). <br />  <br />  <br />  <br /> a dimer, required proper orientation conserved            essentiality dapL mutagenic experiments using transposon <br /> active site residues active site, VsDapL probably              homologous recombination. Moreover, organism is <br /> two active sites (Figure 9B). Closer examination active site           aerobic, relatively easy culture non-pathogenic humans, <br /> (Figure 9C) highlights residues conserved             unlike bacteria contain DapL pathway, Lep- <br /> multiple sequence alignment (Figure 8) key loop regions               tospira interrogans, Bacteriodes fragilis, C. trachomatis (McCoy <br /> that line active site. Importantly, model predicts loop          et al., 2006; Hudson et al., 2008). <br /> B positioned entrance active site cleft, ideally placed          Recently, laboratories demonstrated orthologous <br /> to interact substrates entering active site.                         dapL model plant Arabidopsis thaliana essen- <br />                                                                               tial growth development (Dobson et al., 2011). However, <br /> DISCUSSION                                                                    known true bacteria exclusively <br /> The discovery new biosynthetic route diaminopimelate                  utilize DapL pathway diaminopimelate/lysine synthesis. It <br /> and lysine plants, algae, bacteria provides fresh target         known aminotransferase capable inter- <br /> the discovery novel herbicides, algaecides, antibiotics. The           converting tetrahydrodipicolinate L,L-diaminopimelate exists <br /> amino acids constitute active site DapL conserved             bacteria solely contain DapL pathway. Thus, suit- <br /> in V. spinosum ortholog based protein alignment mod-               able dapL bacterial mutant auxotrophic diaminopime- <br /> eling. Therefore, inhibitors Ct DapL At DapL likely             late/lysine important step forward especially since <br /> inhibit VsDapL. To end, V. spinosum suitable bacterial              examples literature regarding substrate promis- <br /> system development vivo assays discovery             cuity aminotransferases involved amino acid metabolism. <br /> compounds able inhibit DapL facilitate antibiotic              For instance, aspartate aminotransferase, tyrosine amino- <br /> development.                                                                  transferase, branched-chain amino acid aminotransferase <br />     Since V. spinosum genetically altered using exogenous              shown possess overlapping activities E. coli (Gu <br /> DNA, V. spinosum excellent candidate determining                et al., 1998). In addition, recent study demonstrated three <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                       May 2012 | Volume 3 | Article 183 | 7 <br />  Nachar et al.                                                                           Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />                                                                                  supplemented diaminopimelate medium observe <br />                                                                                  growth phenotype. If inhibitory compound speci&#239;&#172;?c for <br />                                                                                  DapL, expect effect the <br />                                                                                  mutant. However, compound effect on <br />                                                                                  wild-type parental strain used create dapL mutant. The <br />                                                                                  inhibition DapL prevent m-DAP production would <br />                                                                                  lead cell death caused lack proper protein cell wall <br />                                                                                  biosynthesis. <br />                                                                                     Here present elucidation biosynthetic pathway <br />                                                                                  diaminopimelate/lysine V. spinosum using genomic, bio- <br />                                                                                  chemical, structural approaches. The enzymes involved the <br />                                                                                  bacterial diaminopimelate/lysine biosynthetic pathways pre- <br />                                                                                  sumed targets antibiotic development. The identi&#239;&#172;?cation and <br />                                                                                  characterization enzymes provides valuable information <br />                                                                                  pertaining experiments aiming elucidate essentiality of <br />                                                                                  genes involved bacterial diaminopimelate/lysine biosyn- <br />                                                                                  thesis pathways. The DapL pathway sole route toward <br />                                                                                  diaminopimelate/lysine V. spinosum. In addition, bacterium <br />                                                                                  genetically modi&#239;&#172;?ed, organism relatively easy cul- <br />                                                                                  ture non-pathogenic animals. These criteria make V. spin- <br />                                                                                  osum excellent bacterial model elucidating essentiality <br />                                                                                  genes involved diaminopimelate/lysine metabolism. <br />  <br />                                                                                  MATERIALS AND METHODS <br />                                                                                  V. SPINOSUM GROWTH CONDITIONS <br />                                                                                  The plasmids strains used study listed Table 3. The <br />                                                                                  V. spinosum DSM 4136T organism cultured R2A medium <br />                                                                                  26&#203;&#353;C. <br />  <br />                                                                                  MULTIPLE-PROTEIN SEQUENCE ALIGNMENT, PHYLOGENETIC TREE <br />                                                                                  CONSTRUCTION, AND HOMOLOGY MODEL OF VsDapL <br />                                                                                  A DapL protein sequence alignment generated using the <br />   FIGURE 7 | Functional complementation E.coli murE mutant.               ClustalW server2 . Phylogenetic trees constructed MUS- <br />   (A) The E. coli mutant TKL-11 harboring pBAD33 pBAD33 +VsmurE. The <br />                                                                                  CLE aligned amino acid sequences using maximum likelihood <br />   bacteria grown OD 0.1 600 nm serially diluted to <br />   10&#226;&#710;&#8217;1 , 10&#226;&#710;&#8217;2 , 10&#226;&#710;&#8217;3 using 0.85% (w/v) saline. The bacteria             (ML), maximum parsimony (MP) minimum evolution (ME), <br />   replica-plated LB medium supplemented 0.2% (w/v) arabinose. The        neighbor-joining (NJ) techniques MEGA 5.05 (Tamura <br />   growth assessed 30 42&#203;&#353;C 24 h. Only colony        et al., 2011). ML searches performed using likelihood- <br />   vector recombinant plasmid used create diagram.            based WAG (Whelan Goldman, 2001) model amino acid <br />   (B) Growth curve E. coli TKL-11 harboring vector <br />                                                                                  character evolution, gamma distributed rates sites, and <br />   pBAD33 (circles), pBAD33 +VsmurE (triangles). The cells grown at <br />   30&#203;&#353;C 5 h shifted 42&#203;&#353;C additional &#239;&#172;?ve hours (shaded area).   proportion invariant sites estimated data set <br />   (C) SDS-PAGE analysis proteins extracted TKL-11 grown LB            (WAG + G + I). ME NJ tree searches performed using <br />   supplemented 0.2% arabinose (w/v). Lane (1) protein ladder (kDa),         parsimony-based JTT model (Jones et al., 1992) gamma <br />   Lane (2) 10 &#206;&#188;g soluble protein extract TKL-11 harboring pBAD33         distributed rates (JTT + G). Optimal models determined <br />   grown 30&#203;&#353;C, Lane (3) 10 &#206;&#188;g soluble protein extract harboring <br />   pBAD33 +VsmurE grown 42&#203;&#353;C. The proteins resolved 10% <br />                                                                                  MEGA 5.05 iteratively tests partition hierar- <br />   (w/v) acrylamide gel stained Coomassie blue                  chically nested models evolution obtain best &#239;&#172;?t. Node <br />   visualization.                                                                 strength assessed using bootstrap technique 100 <br />                                                                                  pseudo-replicate data sets analysis type (ML, MP, ME, and <br />                                                                                  NJ). The accessions numbers following DapL proteins are: <br /> aminotransferases involved alanine biosynthesis E. coli                Arabidopsis thaliana (AEE86265), V. spinosum (ZP_02927470), <br /> (Yoneyama et al., 2011).                                                         C. reinhardtii (XP_001693061), C. trachomatis (NP_219900), <br />    Aminotransferases common genome V. spinosum                     Parachlamydia UWE25 (YP_007684), Synechocystis sp. PCC 6803 <br /> given integral role multiple anabolic catabolic path-               (BAA10583), Thermosynechococcus elongatus BP1 NP_682892), <br /> ways. If compound inhibits bacterial DapL vitro,                Bacteroides fragilis NCTC 9343 (YP_212286), Clostridium <br /> case Arabidopsis enzyme, question com-                thermocellum 27405 (YP_001039489), Methanothermobacter <br /> pound speci&#239;&#172;?c DapL vivo investigated. <br /> One way answer question use vivo studies <br /> by exposing putative inhibitory compound dapL mutant                    2 http://www.ebi.ac.uk/Tools/msa/clustalw2/ <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                              May 2012 | Volume 3 | Article 183 | 8 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 8 | Protein sequence alignment DapL V. spinosum, C.          loops line active site, asterisk refers loop <br />   trachomatis, A. thaliana, C. reinhardtii generated using ClustalW   contributing active site opposing monomer. The ClustalW scores <br />   server (http://www.ebi.ac.uk/Tools/msa/clustalw2/). Putative active-site    relative V. spinosum sequence were: vs. C. reinhardtii = 49 <br />   residues, based structural studies ligand bound At DapL   (length = 443 residues, identity); vs. A. thaliana = 45 (length = 426 residues); <br />   Cr DapL enzymes, shown red. The loop regions correspond key   vs. C. trachomatis = 39 (length = 394 residues). <br />  <br />  <br />  <br /> thermoautotrophicus (NP_275195), Geobacter sulfurreducens                     (NP_927054), Methanosaeta thermophila PT (YP_843230), <br /> (NP_951224), Leptospira interrogans (YP_002757), Nostoc                       Methanospirillum hungatei JF1 (YP_504354), Dehalococcoides <br /> PCC 7120 (NP_488367), Gloeobacter violaceus PCC 7421                          CBDB1 (YP_307791), Methanococcoides burtonii DSM 6242 <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                        May 2012 | Volume 3 | Article 183 | 9 <br />  Nachar et al.                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 9 | VsDapL homology model generated                 N-terminal arm region compared Cr DapL At DapL. (B) Assuming <br />   Swiss-Model Protein Modeling Server (Arnold et al., 2006) using     dimeric structure, required place conserved active site <br />   Cr DapL apo structure template (PDB id: 3QGU Dobson        resides (highlighted red bold Figure 8), active sites the <br />   et al., 2011; Hudson et al., 2011a, BLAST identity = 51% (207/404   dimer shown. (C) A close view active site 1 showing the <br />   residues), QMEAN4 score = 0.673). (A) The domain structure   conserved active site residues loops lining active site (as <br />   monomer. Note that, seen Figure 8, VsDapL truncated     Figure 8). <br />  <br />  <br />  <br /> (YP_565702), Methanosarcina barkeri Fusaro (YP_306095),               appropriate geometry using visualization software COOT <br /> Methanosarcina acetivorans C2A (NP_616639), Syntrophobac-             (Emsley Cowtan, 2004). <br /> ter fumaroxidans MPOB (YP_844192), Planctomyces brasilien- <br /> sis DSM 5305 (YP_004269630), Planctomyces limnophilus DSM             PCR AMPLIFICATION AND CLONING OF THE V. SPINOSUM dapL AND <br /> 3776 (YP_003632005), Verrucomicrobiae bacterium DG1235                murE OPEN READING FRAMES <br /> (ZP_05058547). The accession number Yersinia pseudo-          The length ORFs annotated locus tags VspiD_0101000 <br /> tuberculosis IP 32953 aspartate aminotransferase (AspC)            12510 (dapL, L,L-diaminopimelate aminotransferase) and <br /> (CAH20674).                                                           VspiD_010100019130 (murE, UDP-N -acetylmuramoylalanyl-d- <br />     A homology model VsDapL protein generated using        glutamyl-2,6-meso-diaminopimelate ligase) ampli&#239;&#172;?ed by <br /> the Swiss-Model Protein Modeling Server (Arnold et al., 2006)         PCR. The ORFs ampli&#239;&#172;?ed using: 12 pmol forward and <br /> using CrDapL apo structure template (PDB id: 3QGU;           reverse primers, 1 mM MgSO4 , 0.5 mM of four <br /> Dobson et al., 2011). The model examined hand clashes      deoxynucleotide triphosphates, 0.5 ng genomic DNA 1 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                May 2012 | Volume 3 | Article 183 | 10 <br />  Nachar et al.                                                                               Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 2 | Activity VsDapL various amino donors amino                     Table 3 | Plasmids strains used study. <br /> acceptors. <br />                                                                                      Plasmid/strains                                 Vendor/Reference <br /> Amino donor                  Relative         Amino acceptor        Relative <br />                              activity (%)                           activity (%)     pET30A                                          Novagen, USA <br />                                                                                      pET100D/Topo                                    Invitrogen, USA <br /> L,L-diaminopimelate          100              2-Ketoglutarate       100              pBAD33                                          Guzman et al. (1995) <br /> meso-Diaminopimelate         1.6 &#194;&#177; 0.35       Pyruvate              8.1 &#194;&#177; 0.3        pBAD33 + AtdapL                                 Hudson et al. (2006) <br /> L-Lysine                     3.0 &#194;&#177; 0.2        Oxaloacetate          7.4 &#194;&#177; 0.15       pBAD33 + CrdapL                                 Dobson et al. (2011) <br /> L-Ornithine                  5.1 &#194;&#177; 0.2        Oxovalerate           7.9 &#194;&#177; 0.15       pET30A +VsdapL                                  This study <br />                                                                                      pET100D +VsmurE                                 This study <br /> The assay measures production dihydroquinazolium 440 nm using <br />                                                                                      pBAD33 +VsdapL                                  This study <br /> the ortho-aminobenzaldehyde assay. The assays pertaining amino <br />                                                                                      pBAD33 +VsmurE                                  This study <br /> donor/acceptor combination triplicates, relative activity based <br />                                                                                      Verrucomicrobium spinosum DSM 4136              Schlesner (1987) <br /> on activity enzyme L,L-diaminopimelate 2-ketoglutarate as <br />                                                                                      5-alpha competent cells                         New England Biolabs, USA <br /> substrates. <br />                                                                                      BL21 codon plus RIPL                            Agilent Technologies, USA <br /> of Platinum Pfx DNA polymerase (Invitrogen Corporation, Carls-                       AOH1 (&#206;&#8221;dapD/dapE )                              Hudson et al. (2006) <br /> bad, CA, USA) using following PCR conditions: 1 cycle                         TKL-11 (murE )                                  Lugtenberg van <br /> 94&#203;&#353;C 2 min, followed 30 cycles 94&#203;&#353;C 15 s, 60&#203;&#353;C                                                                     Schijndel-van Dam (1972) <br /> 30 s 72&#203;&#353;C 2 min. The forward reverse primers used to <br /> amplify ORFs VsdapL F VsdapL R dapL ORF                         Table 4 | List Primers used PCR amplification, cloning and <br /> and VsmurE F VsmurE R murE ORF. The nucleotide                           nucleotide sequencing. <br /> sequences primers listed Table 4. The underlined <br /> sequence represents restriction enzyme sites used facilitate                  Primer Sequence (from 5  3  ) <br /> cloning ORF bolded italicized sequences repre- <br />                                                                                      VsdapL F         CCCCGAATTCATGGCCCTCATCAACGAAAACTTCCTCAAG <br /> sent initiation termination codons. For cloning, dapL PCR <br />                                                                                      VsdapL R         CCCCGTCGACCTACTTCAGCGCGGCGATACGGCGGCAGAC <br /> fragment digested EcoRI SalI ligated the <br />                                                                                      VsmurE F         CACCATGACCATTTTGCGCGATCTTATCGAGGGT <br /> plasmid pET30a (EMD Biosciences, Gibbstown, NJ, USA) pro- <br />                                                                                      VsmurE R         GTCGACTCACTGACGGTCATCCCTCCTTTGGCGTGC <br /> duce plasmid pET30a + VsdapL. The murE PCR fragment was <br />                                                                                      T7 promoter      TAATACGACTCACTATAGGG <br /> ligated plasmid pET100D/topo (Invitrogen Corporation, <br />                                                                                      T7 R             TAGTTATTGCTCAGCGGTGG <br /> Carlsbad, CA, USA) produce plasmid pE100D + VsmurE. <br />                                                                                      T7 terminator TATGCTAGTTATTGCTCAG <br /> The recombinant protein derived plasmid carries hexa- <br /> histidine tag derived pET30a pET100D plasmids                        The underlined sequences represent restriction enzyme sites.The bolded ital- <br /> amino terminus. To con&#239;&#172;?rm &#239;&#172;?delity PCR reactions,                      icized sequences denoted start stop codons open reading <br /> dapL ORF sequenced pET30a using T7 promoter                             frames (ORFs). <br /> and T7 terminator primer (Table 4). The murE ORF was <br /> sequenced pET100D using T7 promoter T7 R primer                         V. spinosum (pBAD33 + VsdapL), Arabidopsis thaliana <br /> (Table 4). Both dapL murE ORFs 100% identical                     (pBAD33 + AtdapL), C. reinhardtii (pBAD33 + CrdapL). <br /> sequences deposited Integrated Microbial Genomes (IMG)                        Transformants selected LB agar medium supplemented <br /> public database3 .                                                                   50 &#206;&#188;g mL&#226;&#710;&#8217;1 DAP 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol. Ten <br />                                                                                      individual colonies vector recombinant <br /> FUNCTIONAL COMPLEMENTATION PLASMID CONSTRUCTS                                        plasmid replica-plated LB medium plus 0.2% <br /> The plasmids used functional complementation E.                           (w/v) arabinose 50 &#206;&#188;g mL&#226;&#710;&#8217;1 DAP. The cultures <br /> coli &#206;&#8221;dapD/dapE double mutant E. coli murE mutant                            grown 30&#203;&#353;C 24 h. The E. coli murE mutant (TKL-11) <br /> were produced sub-cloning XbaI SalI fragment                         (thr-1, leuB6 (Am), murE1, fhuA21, codA1, lacY1, tsx-95, glnV44 <br /> pET30a + VsdapL pET100D + VsmurE plasmids                               (AS), &#206;&#187;&#226;&#710;&#8217; , pyrF101, his-108, thyA6, argG66, ilvA634, thi-1, deoC1) <br /> pBAD33 produce pBAD33 + VsdapL pBAD33 + VsmurE                                transformed pBAD33 pBAD33 + VsmurE (Table 3). <br /> (Guzman et al., 1995). The fusion proteins produced                         Transformants selected LB agar medium supplemented <br /> pBAD33 constructs identical proteins produced                        50 &#206;&#188;g mL&#226;&#710;&#8217;1 thymine 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol at <br /> the pET30a pET100D constructs.                                                   30&#203;&#353;C. Ten individual colonies vector recombi- <br />                                                                                      nant plasmid transformation replica-plated LB medium <br /> FUNCTIONAL COMPLEMENTATION OF THE E. COLI dapD/E AND murE                            plus 0.2% (w/v) arabinose 50 &#206;&#188;g mL&#226;&#710;&#8217;1 thymine. The cul- <br /> MUTANTS                                                                              tures grown 30 42&#203;&#353;C 24 h assess functional <br /> The E. coli mutant AOH1 ( dapD::Kan2, dapE6 ) trans-                             complementation. <br /> formed pBAD33 plasmids harboring dapL orthologs <br />                                                                                      PROTEIN EXPRESSION AND PURIFICATION OF VsDapL <br />                                                                                      The E. coli BL21-CodonPlus-RIPL strain transformed with <br /> 3 http://img.jgi.doe.gov/cgi-bin/w/main.cgi                                          plasmid pET30a + VsdapL grown LB broth containing <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            May 2012 | Volume 3 | Article 183 | 11 <br />  Nachar et al.                                                                                 Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> 50 &#206;&#188;g mL&#226;&#710;&#8217;1 kanamycin 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol 37&#203;&#353;C                           MESO-DIAMINOPIMELATE DEHYDROGENASE ACTIVITY <br /> to OD600 0.5. Protein expression induced 1.0 L                        Ddh activity assessed using oxidative deamination. The assay <br /> culture using isopropyl &#206;&#178;-d-1-thiogalactopyranoside &#239;&#172;?nal                         consisted 0.5 mM m-DAP, 0.5 mM NADP+ , 1 mg crude <br /> concentration 0.5 mM 6 h 20&#203;&#353;C. The cell pellet lysed                    soluble V. spinosum extract 10 &#206;&#188;g pure recombinant meso- <br /> by sonication solution 50 mM sodium phosphate (pH 8.0)                        diaminopimelate dehydrogenase C. thermocellum (Hudson <br /> and 300 mM NaCl. The soluble extract incubated 1 mL                          et al., 2011b) 100 mM glycine-KOH (pH 10.5) 100 mM <br /> bed volume Talon Metal Af&#239;&#172;?nity Resin (Clontech, Mountain                           HEPES-KOH (pH 7.6) &#239;&#172;?nal volume 0.5 mL. The produc- <br /> View, CA, USA) 30 min 4&#203;&#353;C. The resin washed &#239;&#172;?ve                            tion NADPH measured continuously 340 nm 30&#203;&#353;C <br /> times 30 mL sonication buffer containing 10 mM imi-                           using Beckman DU640 spectrophotometer <br /> dazole (pH 8.0) 15 min each. The enzyme eluted with <br /> 10 mL sonication buffer containing 250 mM imidazole (pH                            ACKNOWLEDGMENTS <br /> 8.0). The pure protein concentrated Amicon Ultra                            This research supported United States National Science <br /> 10,000 molecular weight cutoff &#239;&#172;?lter <span id='am-243' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> replacing elution                       Foundation (NSF) award AOH (MCB-1120541). Victoria R. <br /> buffer 100 mM HEPES-KOH containing 1 mM DTT, 2 mM                                Nachar supported Undergraduate Summer Research <br /> EDTA (pH 7.6). The recombinant protein stored 50%                              Fellowship College Science RIT. Victoria R. Nachar and <br /> glycerol. Protein concentration measured using Brad-                          Francisco C. Savka enrolled Biology Program SEM <br /> ford assay bovine serum albumin standard (Bradford,                       enrolled Bioinformatics Program RIT. Victoria R. Nachar, <br /> 1976).                                                                                Francisco C. Savka, Sean E. McGroty supported the <br />                                                                                       NSF award (MCB-1120541) Andr&#195;&#169; O. Hudson. Renwick C. J. <br /> L,L-DIAMINOPIMELATE AMINOTRANSFERASE ENZYME                                           Dobson acknowledges CR Roper Bequest Fellowship sup- <br /> ORTHO-AMINOBENZALDEHYDE ASSAY                                                         port, New Zealand Royal Society Marsden Fund (UOC1013) <br /> The ortho-aminobenzaldehyde assay measured formation                           funding support material based work sup- <br /> THDPA L,L-DAP using ortho-aminobenzaldehyde,                               ported U. S. Army Research Laboratory U. S. <br /> forms dihydroquinazolium adduct maximum absorbance                           Army Research Of&#239;&#172;?ce contract/grant number W911NF-11- <br /> at 440 nm. The assay contained 100 mM HEPES-KOH (pH 7.6),                             1-0481. Katherine A. Donovan Rachel A. North acknowledge <br /> 0.5 mM amino donor, 2 mM amino acceptor 1.25 mM ortho-                            University Canterbury Summer Scholarship Scheme. Andr&#195;&#169; <br /> aminobenzaldehyde 10.0 &#206;&#188;g puri&#239;&#172;?ed recombinant VsDapL                           O. Hudson thanks Naomi L. Ward form University Wyoming <br /> enzyme 0.5 mL reaction. The reactions incubated                          graciously providing V. spinosum strain/genomic DNA. The <br /> 30&#203;&#353;C change absorbance measured continuously                           authors thank Michael A. Savka (RIT) helpful discussions and <br /> at 440 nm using Beckman DU640 spectrophotometer.                                    revisions manuscript. <br />  <br /> REFERENCES                                    development. PLoS ONE 6, e20439.          novel diaminopimelate biosynthe-          Jones, D. T., Taylor,W. R., Thornton, <br /> Arnold, K., Bordoli, L., Kopp, J.,        doi:10.1371/journal.pone.0020439          sis pathway prokaryotes iden-             J. M. (1992). The rapidgeneration <br />    Schwede, T. (2006). The SWISS-          Domman, D. B., Steven, B. T.,            ti&#239;&#172;?es diverged form L,L-                mutationdatamatrices pro- <br />    MODEL workspace: web-based               Ward, N. L. (2011). Random trans-         diaminopimelate aminotransferase.            teinsequences. Comput. Appl. Biosci. <br />    environment protein structure          poson mutagenesis Verrucomi-           J. Bacteriol. 190, 3256&#226;&#8364;&#8220;3263.                8, 275&#226;&#8364;&#8220;282. <br />    homology modeling. Bioinformatics          crobium spinosum DSM 4136(T).           Hudson, A. O., Gir&#195;&#179;n, I., Dobson,       Liu, Y., White, R. H., Whitman, <br />    22, 195&#226;&#8364;&#8220;201.                               Arch. Microbiol. 193, 307&#226;&#8364;&#8220;312.            R. C. J. (2011a). Crystallization        W. B. (2010). Methanococci use the <br /> Baizman, E. R., Branstrom, A. A., Lon-     Emsley, P., Cowtan, K. (2004). Coot:     preliminary X-ray diffraction analy-         diaminopimelate aminotransferase <br />    gley, C. B., Allanson, N., So&#239;&#172;?a, M.        model-building tools molecular        sis L,L-diaminopimelate amino-            (DapL) pathway lysine biosyn- <br />    J., Gange, D., Goldman, R.             graphics. Acta Crystallogr. D Biol.       transferase (DapL) Chalmy-              thesis. J. Bacteriol. 192, 3304&#226;&#8364;&#8220;3310. <br />    C. (2000). Antibacterial activity       Crystallogr. 60, 2126&#226;&#8364;&#8220;2132.               domonas reinhardtii. Acta Crystal-        Lugtenberg, E. J. J., van Schijndel- <br />    synthetic analogues based        Fan, C., Clay, M. D., Dehyholos, M.          logr. Sect. F Struct. Biol. Cryst. Com-      van Dam, A. (1972). Temperature- <br />    disaccharide structure moeno-           K., Vederas, J. C. (2010).            mun. 67, 140&#226;&#8364;&#8220;143.                            sensitive mutants Escherichia <br />    mycin, inhibitor bacterial           Exploration inhibitors           Hudson, A. O., Klartag, A., Gilvarg,           coli K-12 low activity of <br />    transglycosylase. Microbiology 146,        diaminopimelate aminotransferase.         C., Dobson, R. C. J., Marques, F.            diaminopimelic acid adding <br />    3129&#226;&#8364;&#8220;3140.                                 Bioorg. Med. Chem. 18, 2141&#226;&#8364;&#8220;2151.         G., Leustek, T. (2011b). Dual            enzyme. J. Bacteriol. 110, 41&#226;&#8364;&#8220;46. <br /> Bradford, M. M. (1976). A rapid        Gu, W., Song, J., Bonner, C. A., Xie,        diaminopimelate biosynthesis path-        McCoy, A. J, Adams, N. E., Hud- <br />    sensitive method quanti&#239;&#172;?ca-        G., Jensen, R. A. (1998). PhhC        ways Bacteroides fragilis             son, A. O., Gilvarg, C., Leustek, T., <br />    tion microgram quantities pro-       essential aminotransferase      Clostridium thermocellum. Bochim.            Maurelli, A. T. (2006). L,L- <br />    tein utilizing principle pro-       aromatic amino acid metabolism         Biophys. Acta 1814, 1162&#226;&#8364;&#8220;1168.               Diaminopimelate aminotransferase, <br />    tein dye binding. Anal. Biochem. 72,       Pseudomonas areuginosa. Micrbiol-       Hudson, A. O., Singh, B. K., Leustek,          trans-kingdom enzyme shared <br />    248&#226;&#8364;&#8220;254.                                   ogy 144, 3127&#226;&#8364;&#8220;3134.                       T., Gilvarg, C. (2006). An L,L-          Chlamydia plants syn- <br /> Cox, R. J. (1996). The DAP pathway         Guzman, L. M., Belin, D., Carson, M. J.,     diaminopimelate aminotransferase             thesis diaminopimelate/lysine. <br />    lysine target antimi-          Beckwith, J. (1995). Tight reg-       de&#239;&#172;?nes novel variant lysine         Proc. Natl. Acad. Sci. U.S.A. 103, <br />    crobial agents. Nat. Prod. Rep. 13,        ulation, modulation, high level       biosynthesis pathway plants. Plant        17909&#226;&#8364;&#8220;17914. <br />    29&#226;&#8364;&#8220;43.                                     expression vectors containing      Physiol. 140, 292&#226;&#8364;&#8220;301.                    Mills, W. R., Wilson, K. G. <br /> Dobson, R. C. J., Gir&#195;&#179;n, I.,              arabinose pBAD promoter. J. Bacte-      Hutton, C. A., Perugini, M. A.,            (1978). Amino acid biosynthe- <br />    Hudson, A. O. (2011). L,L-                 riol. 177, 4121&#226;&#8364;&#8220;4130.                     Gerrard, J. A. (2007). Inhibition            sis isolated pea chloroplast: <br />    diaminopimelate aminotransferase        Hudson, A. O., Gilvarg, C.,              lysine biosynthesis: evolving          metabolism labeled aspar- <br />          Chlamydomonas        rein-      Leustek, T. (2008). Biochemical       antibiotic strategy. Mol. Biosyst. 3,        tate sulfate. FEBS Lett. 92, <br />    hardtii: target algaecide            phylogenetic characterization        458&#226;&#8364;&#8220;465.                                     129&#226;&#8364;&#8220;132. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                                 May 2012 | Volume 3 | Article 183 | 12 <br />  Nachar et al.                                                                                   Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Nishida, H. M., Nishiyama, N.,                  Sahm, H. (1991). A functionally            James, M. N. (2011). The struc-          conducted absence com- <br />     Kobayashi, T., Kosuge, T., Hoshimo,         split pathway lysine synthesis         ture L,L-diaminopimelate amino-       mercial &#239;&#172;?nancial relationships that <br />     T., Yamane, H. (1999). A                Corynebacterium glutamicum. J.          transferase Chlamydia tra-          construed potential con- <br />     prokaryotic gene cluster involved           Bacteriol. 173, 4510&#226;&#8364;&#8220;4516.                 chomatis: implications broad     &#239;&#172;&#8218;ict interest. <br />     synthesis lysine           Tamura, K., Peterson, D., Peterson,           substrate speci&#239;&#172;?city. J. Mol Biol. 19, <br />     amino adipate pathway:                N., Stecher, G., Nei, M.,              649&#226;&#8364;&#8220;660.                                 Received: 14 January 2012; accepted: 02 <br />     key evolution amino               Kumar, S. (2011). MEGA5: mole-          Watanabe, N., Clay, M. D., van Belkum,      May 2012; published online: 25 May <br />     acid biosynthesis. Genome Res. 9,           cular evolutionary genetics analy-         M. J., Marcus, S. L., Flegel, M.         2012. <br />     1175&#226;&#8364;&#8220;1183.                                  sis using maximum likelihood,              D., Clay, M. D., Deyholos, M.            Citation: Nachar VR, Savka FC, <br /> Rambaut, A. (2009). FigTree: Tree Figure        evolutionary distance, maxi-           K., Vederas, J. C., James, M.        McGroty SE, Donovan KA, North RA, <br />     Drawing Tool. Version 1.3.1, Insti-         mumparsimony. Mol. Biol. Evol. 28,         N. (2007). Crystal structure          Dobson RCJ, Buckley LJ Hudson <br />     tute Evolutionary Biology, Uni-          2731&#226;&#8364;&#8220;2739.                                 L,L-diaminopimelate aminotrans-          AO (2012) Genomic biochemi- <br />     versity Edinburgh. Available at:      Velasco, A. M., Leguina, J. I., Laz-      ferase Arabidopsis thaliana:      cal analysis diaminopimelate <br />     http://tree.bio.ed.ac.uk/                   cano, A. (2002). Molecular evolution       recently discovered enzyme        lysine biosynthesis pathway in <br /> Sait, M., Kamneva, O. K., Fay, D. S.,           lysine biosynthetic pathways.       biosynthesis L-Lysine plants       Verrucomicrobium spinosum: identi&#239;&#172;?- <br />     Kirienko, N. V., Polek, J., Shirasu-        J. Mol. Evol. 55, 445&#226;&#8364;&#8220;459.                 Chlamydia. J. Mol. Biol. 371,        cation partial characterization of <br />     Hiza, M. M., Ward, N. L.             Wagner, M., Horn, M. (2006).              685&#226;&#8364;&#8220;702.                                 L,L-diaminopimelate aminotransferase <br />     (2011). Genomic experimental            The Planctomycetes, Verrucomi-          Whelan, S., Goldman, N. (2001).               UDP-N-acetylmuramoylalanyl- <br />     evidence suggests Verrucomi-           crobia, Chlamydiae sister              A general empirical model pro-        D-glutamyl-2,6-meso- <br />     crobium spinosum interacts             phyla comprise superphylum               tein evolution derived multiple     diaminopimelate ligase. Front. Microbio. <br />     eukaryotes. Front. Microbiol. 2:211.        biotechnological medical          protein families using maximum-        3:183. doi: 10.3389/fmicb.2012.00183 <br />     doi:10.33389/fmicb.2011.00211               relevance. Curr. Opin <br /> </body></html>