<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 84, pp. 779-783, February 1987 <br /> Genetics <br />  <br />  <br /> Characterization members rho gene family the <br /> yeast Saccharomyces cerevisiae <br />      (ras/oncogenes/sporulation/cAMP-dependent protein kinase) <br /> PASCAL MADAULE*t, RICHARD AXEL*t, AND ALAN M. MYERS&#194;&#167;&#194;&#182; <br /> *Howard Hughes Medical Institute Institute Cancer Research, College Physicians Surgeons, Columbia University, New York, NY 10032; <br /> and &#194;&#167;Department Biological Sciences, Columbia University, New York, NY 10027 <br /> Contributed Richard Axel, October 6, 1986 <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>           The rho genes comprise evolutionarily                   ciated inactivation RHO] suppressed by <br /> conserved family significant homology ras oncogene                overproduction yeast ras gene product cAMP- <br /> family. Two members rho family isolated                  dependent protein kinase, suggesting ras rho genes <br /> yeast Saccharomyces cerevisiae characterized DNA                       function independent pathways. <br /> sequence analysis. The yeast genes RHO) RHO2 70% <br /> and 57% identical, respectively, rho gene marine                               MATERIALS AND METHODS <br /> snail Aplysia, 53% identical other. <br /> Inactivation genes showed RHOI required                      Strains, Media, General Genetic Methods. The yeast <br /> cell viability, RHO2 essential gene. A mutant                  strains used study listed Table 1. Nonselective <br /> allele RHOI (RHOI-His68) constructed <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span>                    selective media growth yeast described <br /> analogous activates transforming potential              (8). Random spore analysis dissection tetrads were <br /> human HRAS gene. Diploid strains containing RHOI-His68                      performed standard methods (9). Yeast strains were <br /> either low high copy number unable sporulate,                transformed 1-3 gg purified linear DNA covalently <br /> mutant allele dominant wild-type RHO). The require-                    closed circular plasmid DNA method Hinnen et al. <br /> ment RHOI circumvented introduction high                   (10). <br /> copy number plasmids containing gene encoding                      Preparation Analysis Plasmid DNA. Standard tech- <br /> catalytic subunit cAMP-dependent protein kinase                      niques used preparation recombinant plasmids <br /> mutant allele RAS2-ValU9. Despite conservation                     Escherichia coli, restriction enzyme digestions, agarose <br /> the rho ras gene families, finding RHOI functions                 gel electrophoresis, isolation restriction fragments <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> independently adenylate cyclase cAMP-dependent pro-                     agarose gels, ligation specific fragments, transformation of <br /> tein kinase cascade suggests rho ras involved                  E. coli, screening recombinant plasmids, conversion of <br /> distinct biochemical pathways.                                                 recessed ends blunt ends using Klenow fragment (11). E. <br />                                                                                coli strain RRI maintained Luria broth used the <br />  The ras genes comprise large multigene family,                        plasmid host. <br />  shows striking evolutionary conservation. Members                         Hybridization Analyses. The 1.6-kilobase (kb) EcoRI frag- <br />  ras gene family identified transforming genes               ments containing RHO] RHO2 radioactively labeled <br />  sarcoma viruses (Ha-, Ki-, N-ras; review, ref.                 nick-translation hybridized Southern blots yeast <br />  1) subsequently genomes various                      genomic DNA described (12). The conditions used for <br />  eukaryotes (2-6). We identified family genes,                  screening Charon 4A library Aplysia rho <br />  designated rho, share properties members                    cDNA probe, moderate stringency hybridizations of <br />  ras family (7). Like ras genes, rho genes                   probe yeast genomic DNA described Madaule <br />  highly conserved evolution: rho genes                        Axel (7). <br />  yeast, molluscs, humans share &gt;70% amino acid homol-                          DNA Sequence Analysis. The 1.6-kb EcoRI fragment con- <br />  ogy. The proteins encoded ras rho genes share                       taining RHO] sequenced entirety the <br />  following additional properties: mass -21                       chemical cleavage method Maxam Gilbert (13) by <br />  kDa, maintain short blocks highly homologous                           chain-termination technique Sanger et al. (14) using <br />  sequences implicated GTP binding hydrolysis,                        M13 derivative vectors (15). All restriction sites used 5&apos; <br />  reveal COOH-terminal sequences required attach-                       end labeling crossed distal sites, entire <br />  ment inner leaflet cytoplasmic membrane. The                     sequence confirmed complementary strands. <br />  common properties ras rho genes suggest                        The coding region 5&apos; 3&apos; flanking sequences RHO2 <br />  share common functions; however, regions                     obtained using chain-termination method and <br />  complete sequence divergence ras rho, suggest-                     confirmed complementary strand. <br />  ing members gene families utilize common                        Construction RHOI-His68. The 1.6-kb EcoRI fragment <br />  functions achieve different effects.                                        containing RHO) cloned phage vector M13mp9 <br />     In initial experiments determine function rho                 plus strand contained single- <br />  genes possible relationship ras gene function,                 stranded phage DNA. A mutant primer synthesized with <br />  isolated characterized different members                   sequence 5&apos; AATCTTCATGACCAGC 3&apos;, complemen- <br />  rho gene family yeast Saccharomyces cerevisiae. <br />  Genetic experiments demonstrate genes,                       tary RHO] codons 66-71 exception single <br />  RHO], essential gene yeast, second gene, <br />  RHO2, required cell viability. The lethality asso-                  tPresent address: Centre National la Recherche Scientifique, <br />                                                                                  Laboratoire Physiologie Nerveuse, Department 1, BP1, 91190 <br />                                                                                  Gif-sur-Yvette, France. <br />  The publication costs article defrayed page charge     iTo reprint requests addressed. <br />  payment. This article marked &quot;advertisement&quot;          $Present address: Department Biochemistry Biophysics, Iowa <br />  accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                State University, Ames, IA 50011. <br />  780      Genetics: Madaule     et   al.                                             Proc. Natl. Acad. Sci. USA 84 (1987) <br /> Table 1. Genotypes sources S. cerevisiae strains              E       G          C         CE                              x <br />         Strain                       Genotype              Source            _            _           I                          --A   RHO I <br />                                                                           MRP2-4    4-RHOI <br /> W303                   a/a, leu2/leu2, his3/his3,             * <br />                           ura3/ura3, trpl/trpl,                           G <br />                           ade2/ade2, canl/can)                       E ---I-            C                                    E                       x <br /> W303-1A                a, leu2, his3, ura3, trpl,             * <br />                                                                                        _i-1772}-I- <br />                                                                                     mRHOI     - - -       =          - - -   - <br />                                                                           MRP2--&gt;         &lt;- HIS 3                                     RHOI ::HIS3 <br />                           ade2, can) <br /> W303VRHO1              a/a, leu2/leu2, his3/his3,             t <br />                           ura3/ura3, trpl/trpl, ade2/ade2, <br />                           canl/can), +/RHO)::HIS3                    E              AN                E <br /> W303VRHO2              a/a, leu2/Ieu2, his3/his3,             t                  RH02-&gt; <br />                                                                                                                                        RH02* <br />                           ura3/ura3, trpl/trpl, ade2/ade2, <br />                           cani/can), +/RH02::URA3                     E           A                       AN                 E <br /> W303VRHO1 + pWT a, leu2, his3, ura3, trpl, ade2,              t                  _mm~ -{- :- <br />                                                                                            -- 2                                        RH02:: URA3 <br />                           can), RHOI::HIS3,                                   RH02&apos;-*         URA3-&gt;          RHO2   - <br />  <br />  <br />                           plasmid pWT (URA3, RHO])                     FIG. 1. Restriction maps RHO), RHO2, corresponding <br /> KL14-4Bp&#194;&#176;              a, aux                                 f     disrupted alleles RHO): :HIS3 RHO2:: URA3. Solid lines repre- <br /> 5lp0                   a, aux                                 f     sent coding flanking regions ofthe wild-type genes dotted <br /> *R. Rothstein, College Physicians Surgeons, Columbia         lines represent coding flanking regions inserted DNA used <br />  University, New York, NY.                                          create gene disruptions. The positions RHO), RHO2, and <br /> tThis study.                                                        neighboring gene MRP2 coding mitochondrial ribosomal <br /> tP. Slonimski, Centre Genetique Moleculaire du Centre National   protein (33) determined DNA sequence analysis (see Fig. 2). <br />  la Recherche Scientifique, Gif-sur-Yvette, France.              Solid open bars indicate coding regions wild-type and <br />                                                                     inserted genes, respectively, direction transcription <br />                                                                     indicated arrows. Restriction enzyme recognition sites indi- <br /> <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> mismatch codon 68, result                cated EcoRI (E), Bgl II (G), Cla I (C), Xba I (X), Acc I (A), and <br /> substitution histidine glutamine residue. An intact      BstNI (N). <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> construct disrupted allele RHO), 2.8-kb Bgl <br /> RHO) gene carrying single mutation obtained              II/Xba I fragment containing gene cloned the <br /> using gapped duplex approach oligonucleotide-direct-         plasmid vector pUC18. The resultant plasmid digested the <br /> ed mutagenesis described Kramer et al. (16). The              enzymes EcoRI Cla I remove 460-base-pair fragment <br /> complete nucleotide sequence RHO) gene phage          beginning 185 base pairs upstream RHO) initiator codon and <br />                                                                     extending 275 base pairs coding sequence gene. A <br /> demonstrated presence single <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>point mutation</span> con-           1.7-kb EcoRI/Cla I fragment containing yeast HIS3 gene (18) was <br /> verting codon 68 glutamine histidine        isolated plasmid pDIS5 (A.M.M., unpublished data) and <br /> wild-type sequence.                                                 used replace missing segment RHO). A 4.4-kb Pvu II <br />                                                                     fragment containing disrupted RHO) gene isolated the <br />                          RESULTS                                    resultant plasmid used transform yeast strain W303 to <br />                                                                     histidine independence, resulting replacement wild-type <br />    Isolation Nucleotide Sequence Analysis RHOI           copy RHO) disrupted allele RHOJ::HIS3. To disrupt <br /> RHO2. In previous experiments demonstrated genomic            RHO2 1.6-kb EcoRI fragment containing gene first <br />                                                                     cloned plasmid vector pSP65. The recombinant plasmid was <br /> Southern blotting sequences homologous rho gene         cleaved unique Acc I site RHO2 treated with <br /> of Aplysia present yeast (7). To isolate rho            Klenow fragment recessed ends. The 1.3-kb HindIII <br /> homologs yeast, hybridized radioactively labeled            fragment containing yeast URA3 gene (19) treated with <br /> segment Aplysia rho gene plaques genomic           Klenow fragment ligated RHO2 sequence. The 2.9-kb <br /> library yeast DNA constructed A vector Charon 4A          EcoRI fragment containing disrupted RHO2 gene used to <br /> (17). Two different A clones hybridized probe          replace wild-type copy RHO2 mutant allele RHO2:: <br /> conditions moderate stringency. Each clone contained           URA3 genome W303. <br /> distinct 1.6-kb EcoRI fragment, hybridized rho <br /> probe subcloned A recombinants (Fig. 1).              followed highly variable COOH-terminal domain 24 <br /> Genomic blotting experiments confirm presence             amino acids. Comparison known rho gene prod- <br /> least rho gene homologs yeast           ucts reveals strikingly similar <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> conserved and <br /> genome (data shown).                                            divergent regions. The rho genes highly conserved from <br />    The nucleotide sequences deduced amino acid se-              amino terminus position 129 [numbering refers the <br /> quences yeast rho homologs RHO) RHO2                 amino acid sequence RHO1 protein (Fig. 2)], is <br /> shown Fig. 2. The predicted sizes products RHO]       equivalent position 123 Ha-ras. They diverge the <br /> and RHO2 23 21 kDa, respectively. Fig. 3 presents        11 residues reveal region moderate <br /> alignment proteins products Aplysia        homology residues 141-162. A second stretch ex- <br /> rho Ha-ras. RHO) 70% identical Aplysia rho           treme conservation seen residues 163-184 (143-164 in <br /> a single gap, RHO2 57% homologous Aplysia           Ha-ras), followed divergent stretch charged COOH- <br /> gene alignment requires introduction gaps.       terminal amino acids. <br /> The yeast genes 53% homologous               The products RHO) RHO2 share 25-32% <br /> 50% residues conserved RHO), RHO2,                homology ras proteins; however, identities are <br /> Aplysia rho. Thus, RHO2 slightly diverged              clustered discrete highly conserved domains <br /> RHO) Aplysia gene.                                    (Fig. 3). For example, residues 12-23 64-71 RHO) are <br />    Sequence comparisons members ras           75% 88% identical, respectively, corresponding ras <br /> and rho gene families reveal similar interpositions con-         domains. Interestingly, regions include specific <br /> served divergent regions (ref. 5; Fig. 3). For example,     residues cellular ras genes, mutated result <br /> three human ras proteins &gt;90% identical amino          genes transforming potential (20-26). <br /> terminus residue 120. Residue 121 demarcates                Inactivation RHOI RHO2. In initial experiments to <br /> divergent region extends residue 132. A second highly       determine functions RHO) RHO2, chromo- <br /> conserved region residues 133 165,             somal copy gene inactivated method of <br />          Genetics: Madaule et al.                                                                                                   Proc. Natl. Acad. Sci. USA 84 (1987)                         781 <br />                               A Sequence RH01 <br />                               -100 CICMAAAAGGAACAAAATAATMAACMAAGAhTCGCTGTTAGAGGTTTATTGTTGCACTAATAGMAATCATAGAACTTTAAMATTATACTAGMAG <br />                                          Met Ser Gln Gln Val Gly Asn Ser Ile Arg Arg Lys Leu Val Ile Val Gly Asp Gly Ala Cys Gly Lys Thr Cys <br />                                    +1    ATG TCA CAA CAA GTT GGT AAC AGT ATC AGA AGA AAG CTG GTA ATC GTT GGT GAT GGT GCC TGT GGT AAG ACA TGT <br />                                          Leu Leu Ile Val Phe Ser Lys Gly Gln Phe Pro Glu Val Tyr Val Pro Thr Val Phe Glu Asn Tyr Val Ala Asp <br />                                   +76    TTA TTA ATC GTC TTT TCC AAG GGC CAA TTT CCA GAA GTC TAC GTA CCA ACT GTC TTT GAA AAC TAT GTA GCA GAT <br />                                          Val Glu Val Asp Gly Arg Arg Val Glu Leu Ala Leu Trp Asp Thr Ala Gly Gln Glu Asp Tyr Asp Arg Leu Arg <br />                               +151       GTT GAA GTT GAT GGG CGT CGT GTA GAG CTA GCG CTA TGG GAT ACC GCT GOT CGA GAG GAT TAT GAT AGA CTA AGA <br />                                           Pro Leu Ser Tyr Pro Asp Ser Asn Val Val Leu Ile Cys Phe Ser Ile Asp Leu Pro Asp Ser Leu Glu Asn Val <br />                               +226        CCA TTG TCA TAC CCA GAC TCC MT GTC GTA TTA ATT TGT TTC TCT ATC GAT CTT CCA GAT TCT TTA GAG AAT GTA <br />                                           Gln Glu Lys Trp Ile Ala Glu Val Leu His Phe Cys Gln Gly Val Pro Ile Ile Leu Val Gly Cys Lys Val Asp <br />                               +301        CAA GAA MA TGG ATT GCC GAA GTA TTA CAT TTC TGT CM GGT GTG CCA ATT ATT CTT GTT GGT TGT AM GTG GAT <br />                                           Leu Arg Asn Asp Pro Gln Thr Ile Glu Gln Leu Arg Gln Glu Gly Gln Gln Pro Val Thr Ser Gln Glu Gly Gln <br />                               +376        TTG AGA AAC GAC CCA CAA ACC ATT GM CAA TTA AGA CAA GM GGT CAA CAA CCC GTT ACA TCA CAG GAG GGA CM <br />                                           Ser Val Ala Asp Gln Ile Gly Ala Thr Gly Tyr Tyr Glu Cys Ser Ala Lys Thr Gly Tyr Gly Val Arg Glu Val <br />                               +451        TCT GTA GCA GAC CAG ATT GGC GCA ACC GGA TAC TAC GAG TGT TCG GCC AAG ACT GGT TAT GGT GTC AGA GAA GTG <br />                                           Phe Glu Ala Ala Thr Arq Ala Ser Leu Met Sly Lys Ser Lys Thr Asn Gly Lys Ala Lys Lys Asn Thr Thr Glu <br />                                +526       TTT GAG GCC GCC ACT AGA GCT TCA TTG ATG GGT AAA TCT MA ACG MT GGT AAA GCT AAG AAG MC ACT ACT GM <br />                                           Lys Lys Lys Lys Lys Cys Val Leu Leu *** <br />                                +601       AAG AG GAGAAGMAAG TGT GTC TTG TTA TAG GAGTGTCTCGATCTCTCAGTCGTTAACTTAGTGGCAGATACTGCCCTTTTTCAATCCCA <br />                                B Sequence RH02 <br />                                - 100     GTCTTMTTAAAMCGCCCTTGCATACCGCTGTAMCGATTTGTGTGCAGACATCMTTGCTGAAACGTTCTGCTTTGGTTGTGCTTTTGATCCCGTACT <br />                                           Met Ser Glu Lys Ala Val Arg Arg Lys Leu Val Ile Ile Gly Asp Gly Ala Cys Gly Lys Thr Ser Leu Leu Tyr <br />                                    +1     ATG TCT GMA MG GCC GTT AGA AGG AAA CTT GTT ATT ATT GGT GAT GGT GCT TGT GGC AAG ACC TCT TTA CTA TAT <br />                                           Val   Phe   Thr Leu GiY   Lvs   Phe   Pro   Glu    Gln   Tyr   His Pro Thr Val Phe Glu Asn Tvr        Val Thr ASP Cys Arg    Val <br />                                   +76     GTA   TTT   ACA TTA GGA   AAA   TTC   CCT   GAA    CAA   TAT   CAT CCG ACA GTG TTC GAG AAT TAT        GTC ACT GAT TCG AGA    GTT <br />                                           Asp   Gly   Ile Lys Val   Ser   Leu   Thr   Leu    Trp   Asp   Thr Ala Gly Gln lu Gblu Tyr Glu        Arg Leu Arg Pro Phe    Ser <br />                                   +151    GAC   GGA   ATA AAA GTG   TCC   TTA   ACG   CTC    TGG   GAT   ACA GCG GGA CAA GAG GAA TAT GAA         CGT TTA CGT CCA TTC   TCA <br />                                           Tyr Ser Lys Ala Asp Ile Ile Leu Ile Gly Phe Ala Val Asp Asn Phe Glu Ser Leu Ile Asn Ala Arg Thr Lys <br />                                +226       TAT TCA AAA GCA GAT ATA ATA TTA ATT G0G TTT GCT GTA GAC AAT TTT GAA TA CT.U AOTT AAC GCA AGG ACG AAA <br />                                           Trp Ala Asp Glu Ala Leu Arg Tyr Cys Pro Asp Ala Pro Ile Val Leu Val Gly Leu Lys Lys Asp Leu Arg Gln <br />                                   +301    TGG GCG GAT GAG GCA TTA CGA TAT TOT CCT GAC GCA CCA ATC GTT CTT (TA GGC TTr AAA AAA GAT TTG AGO CAA <br />                                           Glu Ala His Phe Lys Glu Asn Ala Thr Asp Glu Met Val Pro Ile Glu Asp Ala Lys Cin Val Ala Aro Ala Ile <br />                                   +376    GAA GCC CAT TTT AAA GAG AAT GCT ACC GAT GAA ATG GTT CCC ATT GAA GAT GCA AAA CAA OTT GCA AGO GCC ATT <br />                                           Gly Ala Lys Lys Tyr Met Glu Cys Ser Ala Leu Thr Gly Glu Gly Val Asp                      ASp       Val Phe Plu Val Ala Thr Ara <br />                                   +451    GGG GCC AAG AAA TAC ATG GAA TGT AGT GCA CTG AGT PGT GAG GOT GTG GAT                      PAT       GTC TTT GAP GTA GCT ACA AG4 <br />                                           Thr Ser Leu Leu Met Lys Lys Glu Pro Gly Ala Asn Cys Cys Ile Ile leu *** <br />                                   +526    ACC AGT TTG CTT ATG AAG AAG GAA CCA GG0 GCT AGC TOT TGC ATA ATT TTA TAG                            AAAAAACTTACACTGAAATGATACATT <br />  <br />                                    FIG. 2. Nucleotide sequence RHO], RHO2, respective flanking regions. <br />  <br /> one-step gene replacement (ref. 27; Fig. 1). Southern hybrid-                                                  effort obtain haploid strains containing null <br /> ization analysis chromosomal DNA diploid                                                           allele. None 200 haploid meiotic products W303VRHO1 <br /> strains W303VRHO1 W303VRHO2 containing dis-                                                            able grow absence histidine, indicating that <br /> rupted alleles RHOJ: :HIS3 RH02:: URA3, respectively,                                                      RHOI::HIS3 recessive lethal allele required <br /> confirmed diploid contained wild-type allele                                                     spore germination. Attempts inactivate RHO] by <br /> and inactivated allele (data shown).                                                                   transformation haploid strain W303-1A histidine <br />   W303VRHOl W303VRHO2 induced sporulate                                                            independence unsuccessful, indicating RHOJ:: <br />                                                          10                             20                           30                         40                         50 <br />                 Aplysia rho                                                         KIDQ F P E V Y V P T V F E N Y V A D I[E V <br />                                                 K L V I V G D GA C G K T C L L I V F S <br />                                              M AA <br />                               M S Q Q V G NS I R R K L V I V G D G A C G K T C L L I V F SK G <br />                                                                                                                                                                                       D G K Q <br />                 RHO1                                                                             YVPTVFENYVA V              V                                                          G <br />                 RHO2                MS EK _      R            KLVIIGDGACGKTRLLTVFTLGKFPEQYHPTVF                YVT TLDC                                                               IJCK <br />                 H-ras                                 MT E Y|K L <br />                                                        1 <br />                                                                       VFV~~~1G]AGG <br />                                                                              0 <br />                                                                                    VG KS A ]T2Q L I Q N HF V D E[ PDT1 I E D SO R K QWfV I0 <br />                                                                                            20               30                lo50 <br />                                                                                                                                             ET <br />  <br />  <br />                                 60                 70                   80            90                   100                  110 <br />                 Aplysia rho V E L A L W D T A G QE D Y D R L R P L S Y P D |T S|ISP D S L E N I P|E K W P[            T       IiPN <br />                 RHO1        V E LA L  W D T A Q  E D  Y D R L R P L S Y  P D S LI C F S  I D L P D S L E N  V Q E K W I A E V L  H F C G <br />                 RHO2                                                         A               N             AR K A D ETL1YC D <br />                 H-ras         C    L[OD    I LOD T A G Q E E Y S A <br />                                                         60 <br />                                                                             MEWD 70Q Y M R          T G E G       F[OC8o V    NA <br />                                                                                                                                I JT KWFS OWDIH Q Y R <br />                                                                                                                                                 90 <br />                                                                                                                                                                      E Q I <br />                                                                                                                                                                        100 <br />                                                                                                                                                                                 KEV   K DS <br />  <br />  <br />                                                 120                         130                             140                        150                     160                        170 <br />                 Aplysia rho        V P I I L V G N K K D LRN D|ES EE M K M K| E P V R PF GR A              K      EY <br />                 RHO1               V P I I L V G C KEID L NJP Q    QE JR Q E G0[ P VIT 9Q ,JQ S V A D Q I GAIT                                                         Y E C     S.AKTG1 <br />                 RHO2               A             L KKDLRQEA HF [N A T           DIE J J P I JE D A K QV AR A I G A K                                            K      M          A <br />  <br />                 H-ras         DEIMIV         L V G N     KGC LA                                                A R    TFE]E    R QAQD          LA-0S YrI PWIFT                    AKTR Q <br />                                    110                        120                                                                130                          140                     150 <br />  <br />                                                         180                        190         200 <br />                 Aplysia rho G V R D V F EIT R AALMGI <br />                               IGVRFJEJVFEIAIAT <br />                                                     QV K <br />                                                 S LIM G <br />                                                                                         _G G[51V         L <br />                 RHO1Bb                                                            KTNGKAKK NNTKKKGYLL <br />                                                                                              T T E KP <br />                                                                                                    K K K K VLL <br />  <br />                 H-ras         0V EMDA[YWL V[E I <br />                                                        160 <br />                                                                     RBQH[WL       R K L N P P D E S G P GC M 0 C K G V L S <br />                                                                                       170                           180 <br />  <br />   FIG. 3. Comparison amino acid sequences encoded RHO], RHO2, Aplysia rho, Ha-ras. The alignments generated by <br /> computer described (8). Numbering rho amino acid sequences refers RHO1 protein. Identical amino acids boxed. The glutamine <br /> residue position 68 RH01 (arrow) changed histidine residue mutation RHOJ-His68. Residues Ha-ras homologous to <br /> residues rho gene boxed Ha-ras sequence. Amino acids designated single-letter <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>. <br />  782      Genetics: Madaule   et   al.                                             Proc. Natl. Acad. Sci. USA 84 (1987) <br />  <br /> HIS3 recessive lethal allele. In contrast, tetrads      Table 2. Segregation high copy number plasmids carrying <br /> derived W303VRHO2 showed 2:2 segregation           RHO), TPKI, RAS2-Va1)9 absence a <br /> uracil auxotrophy conferred RHO2:: URA3. Therefore,              chromosomal RHO] gene <br /> RHO2 required cell viability.                                                                       Suppression by <br />    RHO) Function Is Not Mediated cAMP-Dependent Pro-                            Suppression TPKI          RAS2-Va1J9 <br /> tein Kinase. The simultaneous inactivation yeast RAS]          <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>Phenotype</span>     Plasmid Number Plasmid           Number of <br /> and RAS2 genes lethal (28, 29). This phenotype be <br /> suppressed increased expression adenylate          URA3 LEU2 genotype segregants genotype segregants <br /> cyclase (30) catalytic subunit cAMP-dependent                +     + pWT;pKIN             55    pWT;pVall9         12 <br /> protein kinase (M. Wigler, personal communication). To                 +     - pWT                 145    pWT               188 <br /> determine rho ras proteins elements               -     +         pKIN          0         pVall9          0 <br /> common pathway involving kinase cascade, asked                 -       -         -             0           -             0 <br /> whether lethality associated disruption            Strain aW303VRHO1 + pWT transformed leucine inde- <br /> RHO) suppressed overproduction catalytic           pendence pKIN (carrying TPKI) pVal19 (carrying <br /> subunit cAMP-dependent protein kinase. To end            RAS2-Val9). Individual transformants grown 30-40 gen- <br /> constructed (see Table 3) haploid strain designated               erations nonselective medium single colony segregants were <br /> aW303VRHO1 + pWT, contained disrupted chro-               tested presence URA3 (RHO)) LEU2 (TPKI or <br /> mosomal allele RHO)::HIS3 plasmid pWT.               RAS2-Val9). Neither pKIN pVall9 segregated independently of <br /> This plasmid consists wild-type 1.6-kb EcoRI fragment        pWT, indicating TPKI RAS2-Va1l9 unable to <br /> containing RHO) cloned URA3-bearing episomal               suppress lethality induced disrupted allele RHOI::HIS3. <br /> vector YEp352 (31). Strain aW303VRHO1 + pWT then <br /> transformed leucine independence introducing              compensate loss RHO) function. We conclude ras <br /> plasmid pKIN, contains TPKI gene encoding             rho genes function independent pathways. <br /> catalytic subunit cAMP-dependent protein kinase cloned             Phenotype RHOl-His68 Allele. Since disruption of <br /> into LEU2-bearing episomal vector YEp13 (ref. 32; M.            RHO) results lethality, disruption RHO2 causes <br /> Wigler, personal communication). The resultant transform-           detectable phenotype, used site-specific muta- <br /> ants requiring leucine uracil growth presum-        genesis create allele RHO) confers phenotype <br /> ably harbor independent plasmids, bearing TPKI          easily accessible genetic biochemical analysis. A <br /> a second bearing RHO).                                              point mutation introduced RHOJ changes the <br />    The plasmids allowed segregate growth         glutamine residue position 68 histidine residue. The <br /> for 30-40 generations nonselective medium, single            analogous mutation ras genes position 61 been <br /> colony segregants assayed auxotropic             shown result protein transforming potential in <br /> markers involved experiment. All surviving           mammalian cells greatly reduced ability hydro- <br /> segregants uracil independent retention          lyze bound GTP (22). The 1.6-kb EcoRI fragment containing <br /> plasmid carrying RHO), 27.5% leucine indepen-            mutant allele RHOl-His68 cloned YEp352 to <br /> dent retained plasmid copy TPKI (Table 2).           form plasmid pMUT, identical pWT the <br /> The finding pKIN segregated pWT           exception single point mutation. <br /> a haploid strain lacking chromosomal RHOI gene demon-                pWT pMUT separately introduced ura3- <br /> strates increased cAMP-dependent kinase activity does          heterozygous diploid W303VRHO1 uracil-independent <br /> not compensate loss RHO). The trivial explanation       transformants selected. Quantitating frequency of <br /> pWT integrated genome host strain           Ura+ colonies present growth 30-40 generations in <br /> ruled Southern hybridization DNA isolated          nonselective medium revealed pWT retained 75% <br /> His+ Ura+ segregant, revealed presence RHOI            segregants, 14% segregants a <br /> in high copy number single chromosomal form RHO)           colony containing pMUT retained plasmid. There- <br /> corresponding disrupted allele (data shown).             fore, RHOl-His68 likely detrimental effect on <br />    In analogous experiment, high copy number plasmid         cells, segregants contain pMUT a <br /> pVall9 containing mutant allele RAS2-Vall9 allowed          selective advantage vegetative growth segregants <br /> to segregate independently pWT absence             retain plasmid. Nonetheless, pMUT main- <br /> functional RHO) gene chromosome. RAS2-Vall9             tained selection imposed population. <br /> mutation analogous activates transforming              The heterozygous diploids carrying pMUT pWT <br /> potential mammalian ras genes causes increased               induced sporulate effort isolate RHO] -His68 <br /> intracellular cAMP levels (28). None surviving segre-        sole active allele RHOI cell. Haploid progeny <br /> gants contained pVall9 isolated pWT (Table 2), pro-            assayed ability grow minimal plates lacking <br /> viding evidence stimulation adenylate           uracil histidine. pWT transmitted haploid <br /> cyclase-cAMP-dependent protein kinase cascade does              progeny containing wild type disrupted allele of <br />          Table 3. Meiotic segregation high copy number plasmids carrying RHO) RHOI-His68 <br />                                             Meiotic segregation pWT                   Meiotic segregation pMUT <br />               Phenotype                                           Number                                      Number of <br />          HIS3           URA3               Genotype                 haploids             Genotype                 haploids <br />            +              +           RHOI::HIS3; pWT                  52           RHOI::HIS3; pMUT                   0 <br />            +              -           RHO1::HIS3 -                      0           RHOI::HIS3; -                      0 <br />            -              +                   RHO1; pWT                27                  RHOI; pMUT                  0 <br />            -              -                   RHO); -                  46                  RHO); -                   245 <br />            Diploid strain W303VRHO1 (RHOI /RHOI::HIS3) transformed uracil independence pWT (carrying <br />          RHO)) pMUT (carrying RHOI-His68). Individual transformant colonies induced sporulate absence of <br />          nitrogen, random spore colonies tested presence URA3 HIS3. pWT transmitted meiotic <br />          products containing RHO) RHOI::HIS3; however, pMUT transmitted haploid progeny. <br />            Genetics: Madaule et al.                                               Proc. Natl. Acad. Sci. USA 84 (1987)                 783 <br />  <br /> RHOI (Table 3). However, diploids containing pMUT did           We thank Dr. Alexander Tzagoloff helpful discussions the <br /> give rise Ural haploid progeny, indicating       analysis data Dr. Michael Wigler use cloned <br /> plasmid transmitted meiosis. These data            TPKI RAS2-Vall9 genes. This work supported grants <br /> suggest diploids containing RHO]-His68 sporu-         Howard Hughes Medical Institute (P.M. R.A.), the <br />                                                                   Public Health Service (R.A.), Government France (P.M.), and <br /> late.                                                             National Institutes Health Research Service Award GM09109 <br />    The effect RHO]-His68 sporulation confirmed          (A.M.M.). <br /> by integrating mutant allele genome W303. The <br /> 1.6-kb EcoRI fragments containing RHOJ-His68 cor-           1. Cooper, G. M. &amp; Lane, M. A. (1984) Biochim. Biophys. Acta <br /> responding wild-type gene ligated separately             738, 9-20. <br /> integrative vector YIp352 bearing URA3 (31). The recombi-          2. Shilo, B. &amp; Weinberg, R. (1981) Proc. Natl. Acad. Sci. USA <br /> nant plasmids linearized cleavage URA3              78, 6789-6792. <br /> introduced W303. Transformants isolated         3. Reymond, C. D., Gomer, R. H., Mehdy, M. C. &amp; Firtel, <br /> integrated linear DNA genomes, demon-                R. A. (1984) Cell 39, 141-148. <br /> strated 100% retention Ura+ phenotype             4. Defeo-Jones, D., Scolnik, E., Koller, R. &amp; Dhar, R. (1983) <br />                                                                        Nature (London) 306, 707-709. <br /> growth nonselective medium. The stable diploid transfor-        5. Powers, S., Kataoka, T., Fasano, O., Goldfarb, M., Strathern, <br /> mants patched sporulation medium incubated               J., Broach, J. &amp; Wigler, M. (1984) Cell 36, 607-612. <br /> at room temperature 5 days. The strain containing           6. Fukui, Y. &amp; Kaziro, Y. (1985) EMBO J. 4, 687-691. <br /> extra integrated copy wild-type RHO1 sporulated rate       7. Madaule, P. &amp; Axel, R. (1985) Cell 41, 31-40. <br /> of -25%, asci observed strain containing       8. Myers, A. M. &amp; Tzagoloff, A. (1985) J. Biol. Chem. 260, <br /> an integrated copy RHOJ-His68. Therefore, presence              15371-15377. <br /> of RHO]-His68 low high copy number prevents            9. Mortimer, R. K. &amp; Hawthorne, D. C. (1969) The Yeasts, <br />  sporulation, effect dominant wild-type           Volume 1, eds. Rose, A. H. &amp; Harrison, J. H. (Academic, <br /> RHO] allele.                                                           New York), pp. 385-460. <br />                                                                   10. Hinnen, A., Hicks, J. B. &amp; Fink, G. R. (1978) Proc. Natl. <br />                                                                        Acad. Sci. USA 75, 1929-1933. <br />                      DISCUSSION                                   11. Maniatis, T., Fritsch, E. F. &amp; Sambrook, J. (1982) Molecular <br />                                                                        Cloning: A Laboratory Manual (Cold Spring Harbor Labora- <br /> The rho ras genes appear representatives              tory, Cold Spring Harbor, NY). <br /> highly conserved members superfamily ras genes.         12. Myers, A. M., Pape, L. K. &amp; Tzagoloff, A. (1985) EMBO J. 4, <br /> Sequence analysis ras rho genes yeast,              2087-2092. <br /> the mollusc Aplysia, humans indicates ras        13. Maxam, A. &amp; Gilbert, W. (1980) Methods Enzymol. 65, <br />                                                                        499-560. <br /> rho subfamilies diverged early evolution. The ras     14. Sanger, F., Nicklen, S. &amp; Coulson, A. (1977) Proc. Natl. <br /> rho genes reveal -30% homology. However, yeast RHO]                Acad. Sci. USA 74, 5463-5467. <br /> gene 70% homologous Aplysia rho, turn 90%       15. Vieira, J. &amp; Messing, J. (1982) Gene 19, 259-268. <br /> homologous human rho gene. Thus, rho ras             16. Kramer, W., Drutsa, V., Jansen, H. W., Kramer, B., <br /> subfamilies likely arisen common ancestor            Pflugfelder, M. &amp; Fritz, H.-J. (1984) Nucleic Acids Res. 12, <br /> but diverged generate distinct gene families early          9441-9456. <br />                                                                   17. Woolford, J. &amp; Rosbash, M. (1981) Nucleic Acids Res. 9, <br /> in evolutionary time. The individual families, distinct         5021-5036. <br /> in yeast, remarkable conservation subsequent         18. Struhl, K. (1985) Nucleic Acids Res. 13, 8587-8601. <br /> evolution.                                                        19. Rose, M., Grisafi, P. &amp; Botstein, D. (1984) Gene 29, 113-124. <br />    A variety observations led suggestion      20. Fasano, O., Aldrich, T., Tamanoi, F., Taparowsky, E., Furth, <br /> the ras proteins intermediary GTP binding proteins                  M. &amp; Wigler, M. (1984) Proc. Natl. Acad. Sci. USA 71, <br /> capable transducing information cell surface             4008-4012. <br />                                                                   21. Bos, J. L., Toksoz, D., Marshall, C. J., Verlaan-deVries, M., <br /> effector proteins cell adenylate cyclase-            Veeneman, G. H., van der Eb, A. J., van Boom, J. H., <br /> cAMP-dependent protein kinase cascade. The homology                     Janssen, J. W. G. &amp; Steenvoorden, A. C. M. (1985) Nature <br /> between rho ras gene products, finding          (London) 315, 726-730. <br /> certain mutant alleles genes prevent yeast cells     22. Der, C. J., Finkel, T. &amp; Cooper, G. M. (1986) Cell 44, 167-176. <br /> sporulating, suggested proteins         23. Tabin, C. J., Bradley, S. M., Bargmann, C. I., Weinberg, <br /> link different surface receptors common intracellular              R. A., Papageorge, A. G., Scolnik, E. M., Lowy, D. R. &amp; <br /> effector pathway. Alternatively, ras rho proteins               Charge, E. H. (1982) Nature (London) 300, 143-149. <br />                                                                    24. Reddy, E. P., Reynolds, R. K., Santos, E. &amp; Barbacid, M. <br /> could link surface receptor different intracellular         (1982) Nature (London) 300, 149-152. <br /> effectors. The observation RHO] essential gene       25. Yuasa, Y., Srivastava, S. K., Dunn, C. Y., Rhim, J. S., <br /> yeast allowed test possibilities determining             Reddy, E. P. &amp; Aaronson, S. A. (1983) Nature (London) 303, <br /> whether gene acts cyclase-kinase                775-779. <br />  pathway. Overexpression catalytic subunit cAMP-         26. Taparowsky, E., Suard,  <br /> </body></html>