<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Knockdown Hsc70-5/mortalin Induces Loss Synaptic <br /> Mitochondria Drosophila Parkinson&#226;&#8364;&#8482;s Disease Model <br /> Jun-yi Zhu1,2, Natalia Vereshchagina1, Vrinda Sreekumar1,2,3, Lena F. Burbulla3,4, Ana C. Costa5, <br /> Katharina J. Daub1, Dirk Woitalla6, L. Miguel Martins5, Rejko Kru&#194;&#168;ger3,4,7*, Tobias M. Rasse1* <br /> 1 Junior Research Group Synaptic Plasticity, Hertie-Institute Clinical Brain Research, University Tu&#194;&#168;bingen, Tu&#194;&#168;bingen, Germany, 2 Graduate School Cellular &amp; <br /> Molecular Neuroscience, University Tu&#194;&#168;bingen, Tu&#194;&#168;bingen, Germany, 3 German Center Neurodegenerative Diseases, Tu&#194;&#168;bingen, Germany, 4 Department of <br /> Neurodegenerative Diseases Hertie-Institute Clinical Brain Research, University Tu&#194;&#168;bingen, Tu&#194;&#168;bingen, Germany, 5 Cell Death Regulation Laboratory, MRC <br /> Toxicology <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Leicester, United Kingdom, 6 Department Neurology, St. Josef-Hospital, Ruhr-University Bochum, Bochum, Germany, 7 Werner Reichardt Centre for <br /> Integrative Neuroscience, University Tu&#194;&#168;bingen, Tu&#194;&#168;bingen, Germany <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Mortalin essential component molecular machinery imports nuclear-encoded proteins mitochondria, <br />      assists folding, protects damage accumulation dysfunctional, unfolded proteins aging <br />      mitochondria. Mortalin dysfunction associated Parkinson&#226;&#8364;&#8482;s disease (PD) increases vulnerability cultured cells <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      proteolytic stress leads changes mitochondrial function morphology. To date, Drosophila melanogaster has <br />      successfully used investigate pathogenesis following loss PD-associated genes. We generated <br />      loss-of-Hsc70-5/mortalin-function Drosophila model. The reduction Mortalin expression recapitulates the <br />      defects observed existing Drosophila PD-models, include reduced ATP levels, abnormal wing posture, <br />      shortened life span, reduced spontaneous locomotor climbing ability. Dopaminergic neurons more <br />      sensitive loss mortalin neuronal sub-types non-neuronal tissues. The loss synaptic mitochondria <br />      early pathological change cause later degenerative events. It precedes behavioral abnormalities and <br />      structural changes neuromuscular junction (NMJ) mortalin-knockdown larvae exhibit increased mitochondrial <br />      fragmentation. Autophagy concomitantly up-regulated, suggesting mitochondria degraded mitophagy. Ex <br />      vivo data human fibroblasts identifies increased mitophagy early pathological change precedes apoptosis. <br />      Given specificity observed defects, confident loss-of-mortalin model presented study will <br />      useful dissection complex network pathways underlie development mitochondrial <br />      parkinsonism. <br />  <br />   Citation: Zhu J-y, Vereshchagina N, Sreekumar V, Burbulla LF, Costa AC, et al. (2013) Knockdown Hsc70-5/mortalin Induces Loss Synaptic Mitochondria a <br />   Drosophila Parkinson&#226;&#8364;&#8482;s Disease Model. PLoS ONE 8(12): e83714. doi:10.1371/journal.pone.0083714 <br />   Editor: Hemachandra Reddy, Oregon Health &amp; Science University, United States America <br />   Received September 23, 2013; Accepted November 7, 2013; Published December 30, 2013 <br />   Copyright: &#195;&#376; 2013 Zhu et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grants Fritz Thyssen Foundation (10.11.2.153 RK 10.12.1.192 TMR); German Research Council [DFG, <br />   KR2119/8-1 RK, RA 1804/2-1 TMR]; Federal Ministry Education Research [BMBF, NGFNplus; 01GS08134 RK]. JZ supported fellowship of <br />   China Scholarship Council, LFB doctoral scholarship charitable Hertie Foundation. We acknowledge support Deutsche <br />   Forschungsgemeinschaft Open Access Publishing Fund Tu&#194;&#168;bingen University. The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: tobias.rasse@gmail.de (TMR); rejko.krueger@uni-tuebingen.de (RK) <br />  <br />  <br />  <br /> Introduction                                                                          leads growth arrest [10]. In contrast, overexpression the <br />                                                                                       mitochondrial chaperone Mortalin sufficient extend life <br />    Parkinson&#226;&#8364;&#8482;s disease (PD) common movement disorder                             span Caenorhabditis elegans [11] cultured human <br /> characterized progressive degeneration dopaminergic (DA)                      fibroblasts [12]. <br /> neurons substantia nigra (SNc) [1&#226;&#8364;&#8220;3]. Different genetic                       Mortalin acts buffer prevent damage accumulation <br /> environmental factors contribute disease etiology. Mitochon-                       dysfunctional, unfolded proteins aging mitochondria. Unlike <br /> drial dysfunction plays profound role PD progression [4&#226;&#8364;&#8220;                     heat shock proteins, Mortalin expression induced by <br /> 6] genes associated familial PD, parkin,                     heat shock; promoted cellular stress, glucose <br /> PTEN-induced putative kinase 1 (PINK1) DJ-1                             deprivation, oxidative injury, radiation, cytotoxins (for review, <br /> proposed regulate distinct features mitochondrial function                      [7,8]). <br /> [5].                                                                                     Loss Mortalin function associated PD. Decreased <br />    The ATPase domain-containing protein Mortalin                       levels Mortalin reported advanced PD cases [13] <br /> molecular machinery imports nuclear-encoded proteins                        6-hydroxydopamine rat PD model [14]. Mortalin <br /> mitochondria, sorts them, assists folding (for review,                   variants identified Spanish [15] German patients with <br /> see [7,8]). Mortalin function regarded critical mitochon-                   PD [16]. In vitro studies revealed PD-related Mortalin variants <br /> drial biogenesis; deletion yeast mortalin homolog SSC1                      associated mitochondrial impairments, including mor- <br /> lethal [9], knockdown mortalin immortalized human cells                     phological changes mitochondria, increased reactive oxygen <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                          December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                          Mortalin Knockdown Affects Mitochondria <br />  <br />  <br /> species production, reduced mitochondrial membrane poten-               Quantification Mitochondria Autophagosomes <br /> tial [16]. Importantly, defects exacerbated                We used circularity, inverse form factor, assess <br /> cells challenged proteolytic oxidative stress [16]            shape mitochondria. The circularity object approaches <br /> could partially rescued parkin overexpression [17].                   1 circular is. It approaches 0 branched or <br />    Drosophila melanogaster successfully used uncover            complex object is. Circularity measured using ImageJ. <br /> molecular pathways underlying pathobiology caused loss            Mitochondria circularity .0.8 defined &#226;&#8364;&#732;&#226;&#8364;&#732;round.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> several PD-associated genes, including pink1, parkin, DJ-1 [18&#226;&#8364;&#8220;         Images thresholded allow semi-automated segmenta- <br /> 22]. To generate fly model mortalin-associated PD, used          tion, counting, classification mitochondria using ImageJ <br /> targeted knockdown Hsc70-5, Drosophila homolog human              Version 1.43e. <br /> mortalin. We pan-neuronal silencing Drosophila mortalin          ATG8-mRFP, widely used marker autophagosomes [30], <br /> by <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) resulted reduced fly viability,               used quantify autophagosome abundance size the <br /> locomotion impairment, body posture defects, reduced ATP                Drosophila NMJ. In absence autophagy, ATG8-mRFP is <br /> levels. These phenotypes highly reminiscent defects                  diffusely distributed cytoplasm. Autophagy induction leads <br /> described Drosophila models PD-associated mitochon-            recruitment ATG8-mRFP different sized puncta. <br /> drial dysfunction [21,23,24]. In vivo model, loss                 While large ATG8-mRFP positive puncta generally represent <br /> mitochondria precedes behavioral abnormalities structural               autolysosomes, small puncta frequently did overlap with <br /> changes neuromuscular junction (NMJ) Drosophila larvae.           lysotracker labeling, suggesting represent early autopha- <br /> Mitochondrial fragmentation degradation early                  gosomal structures [30]. We used threshold differentiate <br /> defects up-stream later pathological events. This          autophagosomes (Figure S1, arrowheads) diffuse cyto- <br /> order pathological events mortalin Drosophila model           plasmic ATG-mRFP signal (Figure S1, arrows). The signal <br /> then confirmed human ex vivo model. Our results suggest           intensity observed autophagosomes generally 1.5&#226;&#8364;&#8220;3-fold <br /> mitophagy used biomarker monitoring                   higher background. To improve visualization, either <br /> predisposition mitochondrial Parkinsonism.                               used false color look-up table &#226;&#8364;&#732;&#226;&#8364;&#732;Green-Fire-Blue&#226;&#8364;&#8482;&#226;&#8364;&#8482; (compare <br />                                                                             Figures S1A B) displayed autophagosomes elimination <br /> Materials Methods                                                       cytosolic background adjustment brightness and <br />                                                                             contrast followed adjustment Gamma values 0.75 <br /> Drosophila Strains Culture Conditions                                   (compare Figures S1A C). <br />   All flies raised standard corn meal/agar medium. <br /> Transgenic fly stocks obtained Indiana University             Behavioral Analysis <br /> Stock Center (Bloomington, IN, USA), unless noted. <br />                                                                                The walking behavior 1-day-old female flies assessed as <br /> Transgenic RNAi stocks obtained VDRC stock <br />                                                                             previously described [31]. Flies assayed times in <br /> center: wGD30033, khcGD44337, mortGD47745, mortKK106236 [25]. <br />                                                                             15615-cm petri plates. Each trial lasted 30 s. Individual trials were <br />                                                                             spaced 30 s apart. Walking ability assessed by <br /> Eye <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>Phenotype</span> Scoring                                                       recording number crossings 161-cm square grid lines <br />   To examine external Drosophila eye phenotype, adult flies             marked the plate. At 25 flies each <br /> were raised 29uC 12-h day/night cycles. For analysis,           genotype individually tested. Flies raised assayed at <br /> used frozen flies stored longer 6 d 220uC          18uC. <br /> avoid changes hue eye. Before taking images, flies               Climbing assays conducted previously described [32]. <br /> were thawed dried room temperature 10&#226;&#8364;&#8220;15 min.                    On 6th day egg laying (AEL), larvae transferred <br /> Images obtained DCM510 (ScopeTek, Hangzhou,                     18uC 25uC induce expression UAS-constructs. <br /> P.R. China) camera mounted Zeiss Stemi 2000 stereomicro-               Motor function 4-day-old male flies monitored analyzing <br /> scope (Carl Zeiss, Oberkochen, Germany).                                    ability climb 6 cm 14 s. A successful trial scored <br />                                                                             1, non-successful trial counted 0. Each fly was <br /> Immunocytochemistry Microscopy                                          assessed times calculate average climbing score. At <br />    The size-matched mid-L3 larvae dissected stained                40 flies genotype analyzed. <br /> essentially previously described [26&#226;&#8364;&#8220;28]. Larvae carrying native            The righting assay preformed essentially previously <br /> GFP mRFP constructs fixed 3 min (4% paraformal-                 described [33]. Size-matched mid L3 larvae collected <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> dehyde phosphate-buffered saline) instead 10 min. The goat            food adapted experimental conditions previously <br /> a-horseradish peroxidase (HRP)-Cy3 antibody obtained               described [28]. Next, larvae placed upside the <br /> Dianova (Hamburg, Germany).                                                 agar plate measure total time required reposition the <br />    The larval filets imaged Zeiss LSM 710 Confocal              body posture ventral initiating first <br /> Microscope using 636Plan-Apochromat 1.4 N.A. oil objective.               contraction wave. At 20 larvae sex analyzed <br /> The voxel dimensions (x/y/z) 10061006500 nm. The                       genotype. Each larva assayed times. The average <br /> pinhole size 1 Airy Disc. The images processed                     righting time larva used analysis. N represents <br /> essentially previously described [29]. In brief, images             number larvae analyzed. <br /> scaled factor 2 Gaussian blur filtering applied <br /> (pixel radius = 2). Gamma values adjusted unless                   Longevity Assay <br /> otherwise indicated. For quantitative comparisons intensities,             Flies maintained 18uC single-sex groups more <br /> common settings chosen avoid oversaturation           15 flies group. No anesthesia used longevity <br /> genotypes. Image processing performed using Image J                     experiments. <br /> Software Version 1.43e (National Institutes Health, Bethesda, <br /> MD, USA). <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              2                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                          Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />  <br />  <br /> Figure 1. Hsc70-5 (CG8542, mortalin) Drosophila homolog PD-associated gene mortalin. (A) The genomic organization Hsc70-5 <br /> (CG8542, mortalin) located second chromosome cytological position 50E6. Genes transcripts displayed blue gray/yellow, <br /> respectively. Coding exons depicted yellow boxes, 59-UTR 39-UTR shown gray box gray triangle, respectively. The exact <br /> sequence location (2R:10,140,103&#226;&#8364;&#166;10,143,697 [2]) given panel. mortalin expression repressed using UAS-RNAi stocks <br /> named mortGD47745 (mortGD) mortKK106236 (mortKK). In mortGD (purple arrow) mortKK (cyan arrow), 303-bp 415-bp-long hairpin RNAs <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                             Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />  <br /> directed gene fragments located partially overlapping domains fifth exon mortalin expressed. These double-stranded <br /> RNAs processed short siRNAs predicted induce mortalin mRNA degradation. (B) Drosophila Mortalin (black box) high <br /> sequence similarity human Mortalin. The 686-amino acid-long Drosophila Mortalin protein shares overall 73% identity 84% similarity with <br /> the 679-amino acid-long human Mortalin. The percent homology, color coded panel, human Drosophila mortalin the <br /> highest central domain protein. (C) The ubiquitous pan-neuronal knockdown mortalin resulted larval pupal lethality, while <br /> mortalin knockdown muscle did impair viability. (D) The protein level Mortalin ventral nerve cord (VNC) mid instar larvae was <br /> measured western blot pan-neuronal expression (elav-GAL4, 29uC) mortGD mortKK (E) Eye-specific knockdown mortalin did not <br /> cause visible defects external adult eye young ageing flies. All flies carrying induced RNAi constructs raised 29uC. <br /> Scale bar: 0.1 mm (F) Mortalin deficiency DA neurons lethal, GMR- ey- driven expression mortalinRNAi does affect viability. <br /> Knockdown mortalin DA neurons using Ddc- TH-GAL4 resulted lethality larval pupal stages; effect seen following <br /> knockdown sensory neurons. mortalin knockdown led lethality GAL4 drivers induce expression motoneurons (OK6-, OK371-, <br /> D42-GAL4). <br /> doi:10.1371/journal.pone.0083714.g001 <br />  <br />  <br /> Statistical Analysis                                                          examine physiological consequences knockdown in <br />   Statistical significance assessed previously described [32].         Drosophila. The functionality RNAi constructs assessed <br /> For behavioral experiments NMJ analyses, &#226;&#8364;&#732;&#226;&#8364;&#732;n&#226;&#8364;&#8482;&#226;&#8364;&#8482; represents             ubiquitous (act-5C-GAL4) silencing mortalin expression. <br /> the number flies number NMJs assayed, respectively. p-              Consistent previous reports highlighting importance of <br /> values ,0.05 considered statistically significant. Data            mortalin mitochondrial function cell viability [16,17,34], <br /> are expressed means 6 standard error values (*p,0.05,                      ubiquitous inactivation mortalin expression lethal early <br /> **p,0.01, ***p,0.001).                                                        larval stages (Figure 1C). Neurons particularly vulnerable to <br />                                                                               mitochondrial impairments. Consistently, pan-neuronal (elav- <br /> ATP Measurements                                                              GAL4) silencing mortalin expression (elav.mort) lethal in <br />    ATP levels measured head homogenates using                       late larval early pupal stage, RNAi-mediated <br /> luciferase-based bioluminescence assay. Five heads female flies            mortalin silencing muscle (Mhc-GAL4) did affect viability. <br /> were homogenized 6 M guanidine-HCl frozen liquid                    Analysis Mortalin levels ventral nerve cords third <br /> nitrogen. Next, samples boiled 3 min, cleared                     instar larvae confirmed constructs efficiently suppressed <br /> centrifugation 14,000 g 5 min, diluted measure                  mortalin expression, stronger reduction protein level due <br /> protein concentration (1:10 diluted samples, Bradford Assay Kit,              elav.mortKK activation (Figure 1D). <br /> Sigma, St. Louis, MO, USA) ATP level (1:2,000 diluted <br /> samples, ATP Determination Kit Sensitive Assay, Biaffin GmbH                  Loss mortalin DA Neurons Lethal <br /> &amp; Co KG, Kassel, Germany). ATP levels normalized                     Motor symptoms PD primarily attributed the <br /> protein concentration.                                                        progressive loss DA neurons. Non-movement symptoms of <br />                                                                               PD, anxiety, depression, memory loss, dementia, are <br /> Ethics Statement Analysis Human Cells                                  thought caused degeneration noradrenergic, seroto- <br />    We obtained skin biopsies offspring PD patient,              nergic, cholinergic neurons [35]. To test vulnerability of <br /> one carrying heterozygous A476T mortalin variant                  various neuronal sub-types Mortalin loss, employed different <br /> representing wild-type sibling control. The A476T variant                 GAL4 drivers express mortGD mortKK. RNAi white <br /> carrier did signs PD time biopsy. The               gene (whiteRNAi) used negative control, RNAi against <br /> participants provided written informed consent participate              hereditary spastic paraplegia (HSP)-related gene kinesin heavy <br /> this study. The study approved ethics committee             chain (khcRNAi) used specificity control. Gene expression <br /> Medical Faculty, Eberhard Karls University Tu&#194;&#168;bingen, Germany.                silenced following neuronal sub-types: (1) photoreceptor <br /> No minors/children participants involved study.                   neurons; pigment cells, neurons optic lobe, mushroom <br />    Human fibroblasts cultured, fixed, analyzed essentially           body, medulla cortex, lateral horn, pars intercerebralis <br /> as previously described [16]. The passage number fibroblasts               (GMR-GAL4, ey-GAL4); (2) DA serotonergic neurons <br /> was 10 experiments. Only fibroblasts          (Ddc-GAL4, TH-GAL4); (3) sensory neurons (ppk-GAL4); and <br /> passage number taken experiments. For visualization               (4) glutamatergic motoneurons (OK6-GAL4, OK371-GAL4, <br /> lysosomes mitochondria, cells incubated 15 min                D42-GAL4, OK319-GAL4). Photoreceptor neuron degeneration <br /> 100 nM Lysotracker Red DND-99 (Invitrogen, Carlsbad, CA,                      scored basis structural abnormalities external <br /> USA) 100 nM MitoTracker Green FM (Invitrogen), respec-                     surface eye, formation black lesions and <br /> tively. Secondary antibodies purchased Molecular                    changes eye pigmentation. Strong degeneration following the <br /> Probes (Invitrogen) Zymed (San Francisco, CA, USA). Hoechst                expression toxic proteins using GMR-GAL4 induce <br /> 33342 (Molecular Probes) used stain nuclei. Images                pupal lethality [36]. <br /> analyzed Zeiss software AxioVision 4.6 Image J Software                   GMR.mortKK did cause morphological changes the <br /> Version 1.41o.                                                                external eyes fly aging (Figure 1E). Neither ey- GMR- <br />                                                                               GAL4 induced mortalin silencing resulted lethality <br /> Results                                                                       (Figure 1F). Knockdown mortalin DA neurons using Ddc- <br />                                                                               TH-GAL4, sensory neurons, resulted lethality <br /> Mortalin Important Neuronal Viability                                  larval pupal stages. These results suggest that <br />    Mortalin highly conserved mitochondrial heat shock                    Mortalin particularly important DA neurons. <br /> protein. We identified Hsc70-5 (CG8542) fly ortholog                   However, direct comparison phenotypes complicated <br /> human mortalin (Figure 1A) shares 73% identity 84%                   variations knockdown efficiency different GAL4 <br /> similarity (Figure 1B). We utilized transgenic RNAi stocks [25]           drivers: instance mortalin silencing motoneurons driven by <br /> targeting mortalin: UAS-mortalin-RNAiGD47745 (mortGD) UAS-                OK6-, OK371-, D42-GAL4, OK319-GAL4, <br /> mortalin-RNAiKK106236 (mortKK) (Figure 1A, purple cyan arrows)            resulted lethality. A similar lethality <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> different <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                4                        December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                          Mortalin Knockdown Affects Mitochondria <br />  <br />  <br /> motoneuron drivers observed inactivation HSP- <br /> related gene khc; silencing khc expression DA <br /> neurons did affect viability (Figure 1F). <br />    To systematically assess DA neurons particularly <br /> vulnerable loss mortalin, 14 housekeeping genes were <br /> screened defects following GMR- TH-GAL4 induced <br /> knockdown (Figure 2A). All RNAi-constructs previously <br /> been validated efficacy using mef2-GAL4 [37]. All selected <br /> genes human orthologs cover broad range molecular <br /> functions, metabolism, cytoskeleton organization, signal- <br /> ing, translation, transcription. Eight examined RNAi <br /> lines induced lethality expression TH- and <br /> GMR-GAL4 drivers. Only RNAi constructs that <br /> induced eye degeneration caused lethality expression DA <br /> neurons (Figure 2B). If effects mortalin knockdown were <br /> simply higher efficacy TH-GAL4 compared with <br /> GMR-GAL4, gene silencing the <br /> eyes DA neurons reminiscent lethality pattern <br /> induced mortalin inactivation. However, genes <br /> that failed produce clear phenotype eye GMR- <br /> induced silencing able cause lethality following knock <br /> down DA neurons (Figure 2B). We concluded the <br /> vulnerability DA neurons loss mortalin unlikely be <br /> an artifact. <br />    Our step investigate cellular basis the <br /> increased susceptibility DA neurons mortalin silencing. DA <br /> neurons particularly vulnerable loss mitochondrial <br /> function general susceptible specific mitochondrial <br /> dysfunction caused loss mortalin. To differentiate between <br /> these possibilities, selected set genes known be <br /> important mitochondrial function [38]. Using RNAi, genes <br /> were inactivated eyes TH-positive neurons [25]. We <br /> found 3 10 RNAi constructs expressed GMR-GAL4 <br /> caused pronounced degeneration eye, the <br /> investigated RNAi constructs induced lethality TH-specific <br /> expression (Figure S2A,B). <br />    We concluded DA neurons particularly susceptible <br /> to specific mitochondrial dysfunction caused loss mortalin. <br />  <br /> Loss mortalin Function Affects Body Posture and <br /> Locomotion <br />    PD movement disorder characterized muscular rigidity, <br /> tremor rest, postural instability [3]. Drosophila PD models <br /> exhibit locomotor body posture dysfunctions include <br /> abnormal wing posture, rigidity, defects flight climbing <br /> abilities [19,21,23]. <br />    To test chronic reduction mortalin expression caused <br /> similar symptoms adult flies, pan-neuronally expressed <br /> mortGD mortKK 18uC, temperature UAS/                  Figure 2. Analysis effects housekeeping gene <br /> GAL4 active. Under experimental conditions,            knockdown Drosophila eye. (A) The eye-specific knockdown of <br /> control flies mean life span 50 days,               Drosophila housekeeping genes resulted diverse phenotypes. <br /> median life span flies expressing elav.mortGD reduced 2           Examination eyes revealed effect GMR-GAL4 driven RNAi <br />                                                                             silencing 29uC. (B) GMR.mortKK did cause degeneration the <br /> days, flies expressing elav.mortKK emerged. Ten days           external eyes adult flies compared GMR-GAL4 (control), which <br /> emergence, elav.mortGD flies died, essentially           displayed minor basal toxicity compared uninduced flies (mortKK). <br /> controls alive (elav.mortGD: 86% mortality, n = 50;              The eye-specific inactivation Drosophila housekeeping genes <br /> control: 1% mortality, n = 100) (Figure 3A). Reduced longevity              induced strong degeneration. The arrowheads point black <br /> and locomotion defects common features reported Drosophila           lesions indicative necrosis. Scale bar: 0.1 mm. <br /> models neurodegenerative diseases. Pan-neuronal knockdown             doi:10.1371/journal.pone.0083714.g002 <br /> mortalin strongly affected body posture locomotion of <br /> elav.mortGD flies.                                                          horizontal surface. Walking assays revealed substantial impair- <br />    Climbing assays commonly used test locomotion,                    ment motor capacities 1-day-old mortalin knockdown flies <br /> however impaired body posture overall weakness               (elav.mortGD: walking score 160.08; control: walking score: <br /> elav.mortGD flies required assess locomotor deficits using          0.2560.05; n = 25 each; p,0.001) (Figure 3B). This locomotor <br /> less challenging assay quantifies voluntary locomotion            disability worsened aging, suggesting rapid progressive <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              5                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                             Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />                                                                                 starting 6 days AEL using GAL4/GAL80 system. Analysis <br />                                                                                 4-day-old elav,tub-GAL80.mortKK flies confirmed locomo- <br />                                                                                 tion (Figure 3D) body posture (Figure 3E) defects observed in <br />                                                                                 elav.mortGD flies. Of note, wing posture defects developed <br />                                                                                 mortalin knockdown reminiscent pink1 <br />                                                                                 mutant flies [39] flies expressing dOTC serve model <br />                                                                                 PD-related intra-mitochondrial protein misfolding [24]. We <br />                                                                                 sought investigate mortalin loss induces mitochon- <br />                                                                                 drial dysfunction measuring total ATP levels heads 4- <br />                                                                                 day-old flies following conditional, pan-neuronal mortalin knock- <br />                                                                                 down. Expression elav,tub-GAL80.mortKK resulted 50% <br />                                                                                 decrease cellular ATP levels (Figure 3F). Given mitochon- <br />                                                                                 dria major source ATP, results suggest loss of <br />                                                                                 mortalin impairs mitochondrial ATP production. <br />  <br />                                                                                 Loss Mitochondria Early Pathological <br />                                                                                 Manifestation Presymptomatic Loss mortalin <br />                                                                                 Function Model <br />                                                                                    Next, sought investigate order pathological changes <br />                                                                                 development. The differentiation primary causes, <br />                                                                                 secondary consequences, compensatory adaptations becomes <br />                                                                                 increasingly difficult degenerative processes progress. The most <br />                                                                                 common larval behavioral abnormalities caused neurodegen- <br />                                                                                 erative processes include general larval locomotion impairment, <br />                                                                                 sluggishness, selective impairment posterior segments. <br />                                                                                 For example, Parkin-deficient larvae characterized brady- <br />                                                                                 kinesia-like impairment larval locomotion [40], the <br />                                                                                 expression HSP-related mutations khc leads dystonic <br />                                                                                 posterior paralysis (tail-flip phenotype) [32]. We analyzed <br />                                                                                 crawling larvae aim isolating mortalinRNAi larvae do <br />                                                                                 display impairment locomotion body posture. <br />                                                                                    Strong pan-neuronal expression mortKK mortGD <br />                                                                                 impaired larval locomotion dynamic control body-posture <br />                                                                                 position (Figure 4A) determined righting assay [33]. <br /> Figure 3. Pan-neuronal knockdown mortalin caused behav-                      elav.mortGD larvae displayed abnormal body-posture rest or <br /> ioral defects reduced adult Drosophila lifespan. (A) Kaplan-                locomotion (Figure 4B, Movies S3, S4). Consistently, <br /> Meier <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>survival</span> curve recorded 18uC. Lifespan reduction detected          structural analyses NMJs synaptic boutons (Figure 4C) <br /> upon pan-neuronal (elav-GAL4) mortalin knockdown. Female flies             revealed changes NMJ size bouton number shape <br /> examined. Statistical significance data determined series <br /> of Mantel-Cox tests. (B) Walking tests showed pan-neuronal                 (Figure 4D&#226;&#8364;&#8220;G). We concluded elav.mortGD larvae might <br /> mortalin silencing resulted reduced locomotor function. All flies        considered presymptomatic. <br /> were raised 18uC. Statistical significance determined using              Despite absence obvious changes NMJ morphology, we <br /> unpaired, two-tailed Student&#226;&#8364;&#8482;s t-test. (C) Characteristic wing posture          detected significant reductions mitochondria number, density, <br /> phenotype caused weak pan-neuronal expression mortalinR-              size NMJs elav.mortGD larvae (Figure 5A&#226;&#8364;&#8220;D). We <br /> NAi <br />     . The image displays normal wings control (elav.whiteRNAi)       observed significant increase percentage round <br /> flies; picture shows abnormal wing posture of <br /> elav.mortGD-expressing flies. All flies raised 18uC. Scale          mitochondria (Figure 5E). Small round mitochondria more <br /> bar: 0.25 mm (D) Climbing tests used assess locomotor                   easily engulfed autophagic membrane large branched <br /> behavior. Statistical significance determined using Kruskal-           networks. We hypothesized loss mortalin might <br /> Wallis H-test followed Dunn&#226;&#8364;&#8482;s test comparisons multiple          induce mitophagy. To assess autophagy vivo, induced pan- <br /> groups. (E) The characteristic wing posture phenotype caused             neuronal (elav-GAL4) expression autophagy marker ATG8- <br /> pan-neuronal expression mortalin RNAi. Pan-neuronal mortalin                 mRFP, combination mortalin control RNAi [30]. <br /> silencing mortKK resulted increased wing phenotype <br /> percentage. (F) ATP level measured heads 4-day old                We investigated potential generalized increase autophagy <br /> female flies. Statistical significance determined using Kruskal-          ventral nerve cord presymptomatic larvae. Pan-neuronal <br /> Wallis H-test followed Dunn&#226;&#8364;&#8482;s test comparisons                 knockdown mortalin did cause obvious increase in <br /> multiple groups.                                                                ATG8-mRFP levels central nervous (CNS) the <br /> doi:10.1371/journal.pone.0083714.g003                                           affected larvae (Figure 6A). However, NMJs (Figure 6B,C), the <br />                                                                                 site marked changes mitochondrial size, shape, and <br /> degeneration (compare 5-day-old flies, Movies S1, S2). We                 density detected, observed significant differences in <br /> that 5-day-old flies generally moved climbed.                       autophagy. While ATG8 puncta present control <br />    Body posture defects observed mortalin knockdown                        NMJs, puncta detected mortalin knockdown <br /> included abnormal wing posture: wings down-                    (Figure 6D). These puncta larger present at <br /> turned held erect (Figure 3C). The abnormally down-turned                    control NMJs (Figure 6E). This increase autophagy might <br /> posture wing (Figure 3C) commonly observed                   suggest mitochondria degraded mitophagy. Conse- <br /> phenotype 1-day-old female flies. To bypass early lethality           quently, autophagy marker ATG8-mRFP co-localize <br /> elav.mortKK suppressed mortalin expression pan-neuronally                    mitochondria. Consistently, 25% mitochondria were <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  6                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                             Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />                                                                                autophagosomes revealed organelles preferentially co- <br />                                                                                localize mitochondria (Figure 7B), directly <br />                                                                                adjacent (Figure 7A) overlapping mitochondria (Figure 7A). <br />                                                                                This preferential association mitochondria most <br />                                                                                pronounced medium- large-sized autophagosomes <br />                                                                                (Figure 7B), suggesting autophagosomes contribute the <br />                                                                                phagocytosis mitochondria. Small autophagosomes <br />                                                                                generally associated mitochondria (Figure 7A, <br />                                                                                arrow; 7B). Thus, mitophagy induced loss mortalin <br />                                                                                function earliest cellular hallmarks mortalin <br />                                                                                dysfunction-associated PD. <br />                                                                                   To test hypothesis human cells, examined fibroblasts <br />                                                                                derived carrier A476T mortalin variant did not <br />                                                                                signs PD time biopsy. Previous reports <br />                                                                                using fibroblasts revealed alterations mitochondrial <br />                                                                                morphology compared healthy sibling control [16]. The <br />                                                                                impairments reminiscent defects observed the <br />                                                                                presymptomatic larvae, suggesting human cells a <br />                                                                                good model monitoring changes caused chronic <br />                                                                                reduction Mortalin function early stages PD. <br />  <br />  <br />  <br />  <br /> Figure 4. Quantification synaptic terminals mortalin <br /> knockdown larvae. (A) Larvae locomotor behavior body posture <br /> control assessed righting assay 4-day old mid-L3 stage <br /> larvae. The average righting time determined larvae placed upside <br /> down agar plate. Pan-neuronal mortalin silencing impaired <br /> locomotor function elav.mortKK elav.mortGD larvae. <br /> Statistical significance determined using Kruskal-Wallis H-test <br /> followed Dunn&#226;&#8364;&#8482;s test comparisons multiple groups. (B) <br /> Analysis larval crawling did reveal body-posture defect 4- <br /> day old mid-L3 stage elav.mortGD larvae rest locomotion. <br /> Scale bar: ,0.25 mm (C&#226;&#8364;&#8220;G) Confocal images NMJ 4 Segment A5 of <br /> the mid instar larvae raised 29uC. Visualization neuronal          Figure 5. Quantification mitochondria Drosophila larvae <br /> membranes marked HRP-Cy3 allowed assessment NMJ                        silencing mortalin expression. (A) Confocal images of <br /> morphology. Pan-neuronal expression mortGD did affect (D)               synaptic boutons control (elav.whiteRNAi) elav.mortGD larvae. <br /> muscle <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, (E) NMJ size, number (F) size (G) synaptic         The membrane marker HRP-Cy3 shown green, native <br /> boutons. Scale bar: 5 mm. Statistical significance determined using        fluorescence mito-GFP shown magenta. Scale bar: 5 mm. <br /> an unpaired, two-tailed Student&#226;&#8364;&#8482;s t-test.                                      mortalin silencing significantly reduced (B) number mitochondria <br /> doi:10.1371/journal.pone.0083714.g004                                          NMJ, (C) area fraction NMJ positive mitochondria. <br />                                                                                Furthermore, average size (D) mitochondria reduced, while <br />                                                                                fraction circular mitochondria (E) increased. Statistical <br /> associated autophagosomes elav.mortGD larvae compared                  significance determined using unpaired, two-tailed Student&#226;&#8364;&#8482;s t- <br /> to 5% controls (Figure 7A,B). A quantification                 test. <br />                                                                                doi:10.1371/journal.pone.0083714.g005 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 7                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                             Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />  <br />  <br /> Figure 6. Pan-neuronal knockdown mortalin induced au- <br /> tophagy larval NMJ. (A) Drosophila VNCs control <br /> (elav.whiteRNAi) elav.mortGD larvae labeled autophago- <br /> somal ATG8-mRFP marker. No obvious change ATG8-mRFP signal <br /> was detected mortalin knockdown. Gamma values adjusted             Figure 7. Loss mortalin function induces mitophagy. (A) <br /> to 0.75 Scale bar: 50 mm. (B) Autophagosomes detected            Confocal images NMJ 4 Segment A5 mid instar larvae <br /> strong accumulation ATG8-mRFP signal Drosophila NMJ. The           control (elav.whiteRNAi) elav.mortGD larvae. Neuronal mem- <br /> false color look-up table &#226;&#8364;&#732;&#226;&#8364;&#732;Green-Fire-Blue&#226;&#8364;&#8482;&#226;&#8364;&#8482; allows separation       branes (HRP), autophagosomes, mito-GFP shown. In elav.- <br /> autophagosomes diffuse ATG8-mRFP signal. Scale bar: 10 mm.          mortGD larvae, mitochondria frequently co-localized autophago- <br /> (C) Confocal images synaptic boutons NMJ 4 control                  somes. Scale bar: 10 mm, Enlargement: 2 mm (B) The number of <br /> (elav.whiteRNAi) elav.mortGD larvae. Neuronal membranes              mitochondria autophagosomes NMJ shown. Most autopha- <br /> autophagosomes shown green magenta, respectively. Scale           gosomes elav.mortGD larvae co-localized mitochondria, either <br /> bar: 5 mm. (D, E) Statistical analysis revealed increases ATG8-mRFP       directly adjacent overlapping. (C) In human fibroblasts <br /> puncta abundance (D) size (E). Statistical significance              (n = 56 cells) mitochondrial-lysosomal colocalization higher in <br /> determined using unpaired, two-tailed Student&#226;&#8364;&#8482;s t-test.                cells carrier loss mortalin function variant compared <br /> doi:10.1371/journal.pone.0083714.g006                                        cells healthy sibling control. Colocalization indicated a <br />                                                                              yellow signal overlapping Lysotracker red Mitotracker green <br />                                                                              staining. Scale bar: 10 mm 2 mm. Statistical analysis revealed a <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               8                        December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                        Mortalin Knockdown Affects Mitochondria <br />  <br />  <br />  <br /> higher number mitochondria colocalized lysosomes            tently, mortalin knockdown leads reductions cellular ATP <br /> mutant compared control cells. Statistical significance           levels. <br /> determined using unpaired, two-tailed Student&#226;&#8364;&#8482;s t-test.                    Although mortalin silencing DA neurons caused early pupal <br /> doi:10.1371/journal.pone.0083714.g007                                      lethality, inactivation mitochondrial genes same <br />                                                                            neurons did affect viability, suggesting non-specific <br /> Consistently, differences basal level apoptosis              mitochondrial disturbances sufficient adequately <br /> detected fibroblasts healthy sibling              recapitulate increased susceptibility DA neurons observed <br /> carrier A476T mortalin variant (unpublished results). In        mortalin silencing. Among 150 distinct <br /> latter cells, however, strong decrease mitochondrial mass         mitochondrial syndromes affect 1 in <br /> reported, suggesting mitochondria degraded mito-              5,000 live births, large heterogeneity tissue- organ-specific <br /> phagy (unpublished results) suffer decreased mitochondrial         defects reported [46,47]. Mitochondria remarkably <br /> biogenesis. Next, assessed mitophagy human ex vivo               diverse meet specific demands specific cellular <br /> model differentiate possibilities. For           environment face. For example, cardiac mitochondria are <br /> purpose, calculated Pearson coefficient determine            robust, allowing maintain constant ATP-to-ADP <br /> correlation fluorescence signals lysosomal mitochondrial         ratio fivefold workload range exercise [48]. Thus, it <br /> structures. The fibroblasts carrier A476T mortalin           surprising mitochondria distinct organs are <br /> variant revealed higher degree colocalization mitochondria         morphologically distinct share &#226;&#8364;&#732;&#226;&#8364;&#732;only&#226;&#8364;&#8482;&#226;&#8364;&#8482; 75% common compo- <br /> with lysosomes compared fibroblasts healthy sibling          nents [47,49]. This heterogeneity explains selective pathology <br /> control (Figure 7C), indicating dysfunctional mitochondria            resulting exposure distinct mitochondrial toxins. While <br /> might cleared autophagic pathway [41].                          abuse 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP) <br />    In summary, analysis presymptomatic vivo model            causes selective DA degeneration PD complex I <br /> and ex vivo data human fibroblasts suggest loss          inhibition [50], epidemic blindness resembling Leber&#226;&#8364;&#8482;s hered- <br /> mitochondria represent earliest pathological change         itary optic neuropathy caused combined folate deficiency <br /> course disease progression associated impaired Mortalin            consumption home-made rum containing methanol <br /> function.                                                                  induced complex IV inhibition [51,52]. It proposed <br />                                                                            lack redundancy mitochondrial quality control <br /> Discussion                                                                 systems DA neurons [53,54] render DA neurons more <br />                                                                            susceptible mitochondrial dysfunction. Mortalin central <br /> Establishment Drosophila Model PD-related <br />                                                                            importance intra-mitochondrial quality control, suggesting that <br /> Mortalin Dysfunction                                                       combination specific non-specific disturbances might <br />    Mortalin essential mitochondrial biogenesis, organellar          affect mitochondrial function DA neurons, resulting selective <br /> quality control, suppression apoptosis (for review, see <br />                                                                            vulnerability cells. <br /> [7,8,34]). Mortalin variants adversely affect mitochondrial <br />                                                                               But motoneurons affected PD-model? <br /> function identified PD patients, supporting the <br />                                                                            Consistent previous report, morphology functional <br /> importance Mortalin mitochondrial function [16]. Pan- <br />                                                                            organization SNc DA neurons contribute increased <br /> neuronally targeted mortalin knockdown led shortened lifespan, <br />                                                                            vulnerability cells [55]. Although primarily affected in <br /> impaired walking climbing, abnormal wing posture. <br />                                                                            PD, Drosophila motoneurons share complex morphology SNc <br /> These phenotypes reminiscent symptoms observed in <br />                                                                            DA neurons, necessitating complex trafficking surveillance <br /> existing Drosophila PD models mitochondrial function is <br />                                                                            systems supply healthy mitochondria places high-energy <br /> disrupted intra-mitochondrial proteolytic stress [24] or <br /> dysfunction Pink1 Parkin [21,23].                                    demand. Drosophila motoneurons simple, accessible, well- <br />                                                                            characterized model decipher molecular mechanisms <br />    Due similarity observed defects, confident <br /> that model loss-of-mortalin-related PD presented           underlie susceptibility neurons extended <br /> study useful dissection imbalance            morphology impairments mitochondrial trafficking and <br /> complex molecular networks underlying development                   surveillance systems. Future research additionally include a <br /> mitochondrial parkinsonism. It particularly interesting         detailed investigation accessible Drosophila DA <br /> use powerful genetic tools available Drosophila validate         neurons. <br /> known molecular interactions mortalin PD-associated <br /> genes, PINK1, parkin, DJ-1 (compare Refs [16,42&#226;&#8364;&#8220;44]),          Reduction Synaptic Mitochondrial Mass Precedes <br /> as identify new Mortalin interaction partners.                  Apoptosis Developing Motor Symptoms <br />                                                                               Early pharmacological intervention important success <br /> Vulnerability DA Neurons mortalin Loss                               therapeutic approach. We able identify reduced <br />    mortalin silencing DA neurons generally affects                mitochondrial mass pivotal cellular defect precedes <br /> organism viability dramatically mortalin knockdown            behavioral symptoms. This observation consistent previous <br /> other cell types. Thus, DA neurons particularly                   studies fibroblasts carrier PD-associated A476T <br /> vulnerable loss mitochondrial function general             mortalin variant [16]. As such, mitochondrial fragmentation might <br /> susceptible specific mitochondrial dysfunction caused loss       caused imbalance mitochondrial fission and <br /> of mortalin. Alternatively, combination specific non-             fusion [6] increased mitophagy. Increased mitochondrial <br /> specific defects contribute observed selective                fission ought increase number mitochondria. However, <br /> vulnerability. We favor hypothesis reasons.         opposite observed: mortalin knockdown reduced the <br /> Firstly, loss mortalin, broadly expressed essential gene,          number mitochondria NMJ. We propose the <br /> expected lead general impairment cell viability. Both           accumulation fragmented mitochondria suppressed by <br /> deletion yeast homolog SSC1 strong mortGD knockdown             increased autophagy [6]. Indeed, silencing mortalin expression <br /> in Drosophila muscles using mef2-Gal4 lethal [37,45]. Consis-          increased abundance autophagosomes co- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             9                       December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                                                Mortalin Knockdown Affects Mitochondria <br />  <br />  <br /> localization mitochondria, suggesting mitophagy                              (TIF) <br /> early cellular hallmark Mortalin associated PD pathology. <br />                                                                                              Movie S1 Locomotion whiteRNAi expressing flies. The <br />    Consistently, reduced mitochondrial mass (unpublished results) <br />                                                                                              locomotion 5-day-old elav.whiteRNAi flies (18uC). The flies <br /> and increased co-localization lysosomal mitochondrial <br />                                                                                              normally. <br /> structures (Figure 7C) increased apoptosis (unpublished <br /> results) observed fibroblasts derived carrier                          (MPG) <br /> PD-associated A476T mortalin variant.                                                        Movie S2 Locomotion mortalinRNAi expressing flies. <br />    Thus, analysis presymptomatic Drosophila larvae                            The locomotion 5-day-old elav.mortGD flies (18uC). The flies <br /> human ex vivo model identify mitochondria loss early                              completely paralyzed, unable or <br /> pathological change precedes behavioral symptoms                                    climb. <br /> apoptosis. Future research address perspective                                 (MPG) <br /> monitoring mitophagy patient-derived fibroblasts biomark- <br /> er predisposition mitochondrial Parkinsonism.                                         Movie S3 Locomotion whiteRNAi expressing larvae. <br />                                                                                              The locomotion 4-day-old L3 elav.whiteRNAi larva (29uC). <br />                                                                                              The larva crawls normally. <br /> Supporting Information <br />                                                                                              (MPG) <br /> Figure S1 Quantification autophagosomes Dro- <br />                                                                                              Movie S4 Locomotion mortalinRNAi expressing lar- <br /> sophila mortalin knockdown larvae. Confocal images of <br />                                                                                              vae. The locomotion 4-day-old L3 elav.mortGD larva (29uC). <br /> NMJs control elav.mortGD larvae. (A) The autophagosomes <br />                                                                                              The larva crawls normally. <br /> marker ATG8-mRFP (red) shows diffuse staining entire <br /> NMJ (arrow). Autophagosomes (arrowheads) detected                                 (MPG) <br /> strong accumulation ATG8-mRFP signal. Scale bar: 5 mm. <br /> (B) The false color look table &#226;&#8364;&#732;&#226;&#8364;&#732;Green-Fire-Blue&#226;&#8364;&#8482;&#226;&#8364;&#8482; allows                                 Acknowledgments <br /> separating autophagosomes background staining. (C) Alter-                               We thank Developmental Studies Hybridoma Bank maintained the <br /> natively, autophagosomes displayed defining                                    University Iowa antibodies. We thank Ernst Hafen (ATG8-mRFP) <br /> removing &#226;&#8364;&#732;&#226;&#8364;&#732;non-punctate&#226;&#8364;&#8482;&#226;&#8364;&#8482; appropriate indirect                                supporting stocks. We grateful Bloomington VDRC <br /> thresholding using adjustment image brightness                                    Stock Centers researchers mentioned explicitly made <br /> contrast. The image adjustments mutant                                 stocks available Bloomington Stock Center. We thank Shu- <br /> control NMJs.                                                                                Fun J. Ng Raphael Zinser excellent assistance. We thank Julia <br /> (TIF)                                                                                        Fitzgerald, Doron Rapaport Aaron Voigt helpful discussions and <br />                                                                                              comments manuscript. <br /> Figure S2 Effects mitochondrial gene silencing in <br /> Drosophila eyes. The knockdown Drosophila mitochondria-                                   Author Contributions <br /> related genes cause degeneration external eyes upon <br /> the RNAi expression GMR-GAL4 failed induce                                      Conceived designed experiments: TMR RK. Performed the <br />                                                                                              experiments: JZ NV VS LFB KJD. Analyzed data: JZ NV VS LFB <br /> lethality driven TH-GAL4 (CoVIb, sesB, Trxr-1). All <br />                                                                                              KJD. Contributed reagents/materials/analysis tools: ACC LMM DW. <br /> the flies raised 29uC. The arrowheads point black                             Wrote paper: JZ NV TMR. <br /> lesions. Scale bar indicates 0.1 mm. <br />  <br /> References <br />  1. Dauer W, Przedborski S (2003) Parkinson&#226;&#8364;&#8482;s disease: mechanisms models.                13. Shi M, Jin J, Wang Y, Beyer RP, Kitsou E, et al. (2008) Mortalin: protein <br />     Neuron 39: 889&#226;&#8364;&#8220;909.                                                                          associated progression Parkinson disease? J Neuropathol Exp Neurol 67: <br />  2. Farrer MJ (2006) Genetics Parkinson disease: paradigm shifts future                   117&#226;&#8364;&#8220;124. <br />     prospects. Nat Rev Genet 7: 306&#226;&#8364;&#8220;318.                                                     14. Chiasserini D, Tozzi A, Iure A, Tantucci M, Susta F, et al. (2011) Mortalin <br />  3. Coelho M, Ferreira JJ (2012) Late-stage Parkinson disease. Nat Rev Neurol 8:                 inhibition experimental Parkinson&#226;&#8364;&#8482;s disease. Mov Disord 26: 1639&#226;&#8364;&#8220;1647. <br />     435&#226;&#8364;&#8220;442.                                                                                 15. De Mena L, Coto E, Sanchez-Ferrero E, Ribacoba R, Guisasola LM, et al. <br />  4. Orsucci D, Caldarazzo Ienco E, Mancuso M, Siciliano G (2011) POLG1-related                   (2009) Mutational screening mortalin gene (HSPA9) Parkinson&#226;&#8364;&#8482;s <br />     &#226;&#8364;&#732;&#226;&#8364;&#732;mitochondrial Parkinsonisms&#226;&#8364;&#8482;&#226;&#8364;&#8482;: overview. J Mol Neurosci 44: 17&#226;&#8364;&#8220;               disease. J Neural Transm 116: 1289&#226;&#8364;&#8220;1293. <br />     24.                                                                                      16. Burbulla LF, Schelling C, Kato H, Rapaport D, Woitalla D, et al. (2010) <br />  5. Rochet JC, Hay BA, Guo M (2012) Molecular insights Parkinson&#226;&#8364;&#8482;s disease.                 Dissecting role mitochondrial chaperone mortalin Parkinson&#226;&#8364;&#8482;s <br />     Prog Mol Biol Transl Sci 107: 125&#226;&#8364;&#8220;188.                                                       disease: functional impact disease-related variants mitochondrial <br />  6. Burbulla LF, Krebiehl G, Kruger R (2010) Balance challenge&#226;&#8364;&#8220;the impact                 homeostasis. Hum Mol Genet 19: 4437&#226;&#8364;&#8220;4452. <br />     mitochondrial dynamics Parkinson&#226;&#8364;&#8482;s disease. Eur J Clin Invest 40: 1048&#226;&#8364;&#8220;            17. Yang H, Zhou X, Liu X, Yang L, Chen Q, et al. (2011) Mitochondrial <br />     1060.                                                                                        dysfunction induced knockdown mortalin rescued Parkin. Biochem <br />  7. Deocaris CC, Kaul SC, Wadhwa R (2009) The versatile stress protein mortalin                  Biophys Res Commun 410: 114&#226;&#8364;&#8220;120. <br />     chaperone therapeutic agent. Protein Pept Lett 16: 517&#226;&#8364;&#8220;529.                         18. Menzies FM, Yenisetti SC, Min KT (2005) Roles Drosophila DJ-1 survival <br />  8. Deocaris CC, Kaul SC, Wadhwa R (2008) From proliferative neurological                     dopaminergic neurons oxidative stress. Curr Biol 15: 1578&#226;&#8364;&#8220;1582. <br />     role hsp70 stress chaperone, mortalin. Biogerontology 9: 391&#226;&#8364;&#8220;403.                  19. Park J, Kim SY, Cha GH, Lee SB, Kim S, et al. (2005) Drosophila DJ-1 mutants <br />  9. Craig EA, Kramer J, Kosic-Smithers J (1987) SSC1, member 70-kDa                     oxidative stress-sensitive locomotive dysfunction. Gene 361: 133&#226;&#8364;&#8220;139. <br />     heat shock protein multigene family Saccharomyces cerevisiae, essential        20. Clark IE, Dodson MW, Jiang C, Cao JH, Huh JR, et al. (2006) Drosophila pink1 <br />     growth. Proc Natl Acad Sci U S A 84: 4156&#226;&#8364;&#8220;4160.                                              required mitochondrial function interacts genetically parkin. <br /> 10. Wadhwa R, Takano S, Taira K, Kaul SC (2004) Reduction mortalin level                   Nature 441: 1162&#226;&#8364;&#8220;1166. <br />     antisense expression causes senescence-like growth arrest human                   21. Park J, Lee SB, Lee S, Kim Y, Song S, et al. (2006) Mitochondrial dysfunction in <br />     immortalized cells. J Gene Med 6: 439&#226;&#8364;&#8220;444.                                                   Drosophila PINK1 mutants complemented parkin. Nature 441: 1157&#226;&#8364;&#8220; <br /> 11. Yokoyama K, Fukumoto K, Murakami T, Harada S, Hosono R, et al. (2002)                        1161. <br />     Extended longevity Caenorhabditis elegans knocking extra copies              22. Poole AC, Thomas RE, Andrews LA, McBride HM, Whitworth AJ, et al. (2008) <br />     hsp70F, homolog mot-2 (mortalin)/mthsp70/Grp75. FEBS Lett 516: 53&#226;&#8364;&#8220;                      The PINK1/Parkin pathway regulates mitochondrial morphology. Proc Natl <br />     57.                                                                                          Acad Sci U S A 105: 1638&#226;&#8364;&#8220;1643. <br /> 12. Kaul SC, Yaguchi T, Taira K, Reddel RR, Wadhwa R (2003) Overexpressed                    23. Greene JC, Whitworth AJ, Kuo I, Andrews LA, Feany MB, et al. (2003) <br />     mortalin (mot-2)/mthsp70/GRP75 hTERT cooperate extend vitro                    Mitochondrial pathology apoptotic muscle degeneration Drosophila <br />     lifespan human fibroblasts. Exp Cell Res 286: 96&#226;&#8364;&#8220;101.                                     parkin mutants. Proc Natl Acad Sci U S A 100: 4078&#226;&#8364;&#8220;4083. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                              10                           December 2013 | Volume 8 | Issue 12 | e83714 <br />                                                                                                                                    Mortalin Knockdown Affects Mitochondria <br />  <br />  <br /> 24. Pimenta Castro I, Costa AC, Lam D, Tufi R, Fedele V, et al. (2012) Genetic                   inactivation Drosophila Pink1 rescued Parkin. Proc Natl Acad Sci U S A <br />     analysis mitochondrial protein misfolding Drosophila melanogaster. Cell                   103: 10793&#226;&#8364;&#8220;10798. <br />     Death Differ 19: 1308&#226;&#8364;&#8220;1316.                                                               40.   Vincent A, Briggs L, Chatwin GF, Emery E, Tomlins R, et al. (2012) parkin- <br /> 25. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) A genome-                       induced defects neurophysiology locomotion generated metabolic <br />     wide transgenic RNAi library conditional gene inactivation Drosophila.                   dysfunction oxidative stress. Hum Mol Genet 21: 1760&#226;&#8364;&#8220;1769. <br />     Nature 448: 151&#226;&#8364;&#8220;156.                                                                      41.   Klionsky DJ, Abdalla FC, Abeliovich H, Abraham RT, Acevedo-Arozena A, et <br /> 26. Rasse TM, Fouquet W, Schmid A, Kittel RJ, Mertel S, et al. (2005) Glutamate                     al. (2012) Guidelines use interpretation assays monitoring <br />     receptor dynamics organizing synapse formation vivo. Nat Neurosci 8: 898&#226;&#8364;&#8220;                    autophagy. Autophagy 8: 445&#226;&#8364;&#8220;544. <br />     905.                                                                                      42.   Rakovic A, Grunewald A, Voges L, Hofmann S, Orolicki S, et al. (2011) PINK1- <br /> 27. Schmid A, Sigrist SJ (2008) Analysis neuromuscular junctions: histology                  Interacting Proteins: Proteomic Analysis Overexpressed PINK1. Parkinsons <br />     vivo imaging. Methods Mol Biol 420: 239&#226;&#8364;&#8220;251.                                                 Dis 2011: 153979. <br /> 28. Kern JV, Zhang YV, Kramer S, Brenman JE, Rasse TM (2013) The Kinesin-3,                   43.   Jin J, Li GJ, Davis J, Zhu D, Wang Y, et al. (2007) Identification novel <br />     Unc-104 Regulates Dendrite Morphogenesis Synaptic Development                            proteins associated alpha-synuclein DJ-1. Mol Cell Proteomics 6: <br />     Drosophila. Genetics. <br />                                                                                                     845&#226;&#8364;&#8220;859. <br /> 29. Fuger P, Behrends LB, Mertel S, Sigrist SJ, Rasse TM (2007) Live imaging of <br />                                                                                               44.   Davison EJ, Pennington K, Hung CC, Peng J, Rafiq R, et al. (2009) Proteomic <br />     synapse development measuring protein dynamics using two-color <br />                                                                                                     analysis increased Parkin expression interactants provides evidence for <br />     fluorescence recovery photo-bleaching Drosophila synapses. Nat Protoc <br />                                                                                                     role modulation mitochondrial function. Proteomics 9: 4284&#226;&#8364;&#8220;4297. <br />      2: 3285&#226;&#8364;&#8220;3298. <br /> 30. Kohler K, Brunner E, Guan XL, Boucke K, Greber UF, et al. (2009) A                        45.   Schneider HC, Berthold J, Bauer MF, Dietmeier K, Guiard B, et al. (1994) <br />     combined proteomic genetic analysis identifies role lipid desaturase              Mitochondrial Hsp70/MIM44 complex facilitates protein import. Nature 371: <br />     Desat1 starvation-induced autophagy Drosophila. Autophagy 5: 980&#226;&#8364;&#8220;990.                     768&#226;&#8364;&#8220;774. <br /> 31. Godena VK, Romano G, Romano M, Appocher C, Klima R, et al. (2011) TDP-                    46.   Skladal D, Halliday J, Thorburn DR (2003) Minimum birth prevalence of <br />     43 regulates Drosophila neuromuscular junctions growth modulating Futsch/                    mitochondrial respiratory chain disorders children. Brain 126: 1905&#226;&#8364;&#8220;1912. <br />     MAP1B levels synaptic microtubules organization. PLoS ONE 6: e17808.                  47.   Vafai SB, Mootha VK (2012) Mitochondrial disorders windows an <br /> 32. Fuger P, Sreekumar V, Schule R, Kern JV, Stanchev DT, et al. (2012) Spastic                     ancient organelle. Nature 491: 374&#226;&#8364;&#8220;383. <br />     paraplegia <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> N256S neuronal microtubule motor KIF5A disrupts                48.   Balaban RS, Kantor HL, Katz LA, Briggs RW (1986) Relation work <br />     axonal transport Drosophila HSP model. PLoS Genet 8: e1003066.                             phosphate metabolite vivo paced mammalian heart. Science 232: <br /> 33. Daigle JG, Lanson NA Jr, Smith RB, Casci I, Maltare A, et al. (2013) RNA-                       1121&#226;&#8364;&#8220;1123. <br />     binding ability FUS regulates neurodegeneration, cytoplasmic mislocalization           49.   Pagliarini DJ, Calvo SE, Chang B, Sheth SA, Vafai SB, et al. (2008) A <br /> </body></html>