<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                      REPORT <br /> Exome Sequencing Functional Validation <br /> in Zebrafish Identify GTDC2 Mutations <br /> as Cause Walker-Warburg Syndrome <br /> M. Chiara Manzini,1,2 Dimira E. Tambunan,1,2 R. Sean Hill,1,2 Tim W. Yu,1,2 Thomas M. Maynard,3 <br /> Erin L. Heinzen,4 Kevin V. Shianna,4 Christine R. Stevens,5 Jennifer N. Partlow,1,2 Brenda J. Barry,1,2 <br /> Jacqueline Rodriguez,1,2 Vandana A. Gupta,1,6 Abdel-Karim Al-Qudah,7 Wafaa M. Eyaid,8 <br /> Jan M. Friedman,9,10 Mustafa A. Salih,11 Robin Clark,12 Isabella Moroni,13 Marina Mora,14 <br /> Alan H. Beggs,1,6 Stacey B. Gabriel,5 Christopher A. Walsh1,2,5,* <br />  <br /> Whole-exome sequencing (WES), analyzes coding sequence annotated genes human genome, ideal <br /> approach studying fully penetrant autosomal-recessive diseases, powerful identifying disease-causing muta- <br /> tions enrollment affected individuals limited reduced <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>survival</span>. In study, combined WES homozygosity <br /> analysis consanguineous pedigrees, informative single affected individual available, identify genetic <br /> mutations responsible Walker-Warburg syndrome (WWS), genetically heterogeneous autosomal-recessive disorder severely <br /> affects development brain, eyes, muscle. Mutations seven genes known cause WWS explain 50%&#226;&#8364;&#8220;60% of <br /> cases, multiple additional genes expected mutated unexplained cases suggestive linkage diverse loci. Using <br /> WES consanguineous WWS-affected families, multiple deleterious mutations GTDC2 (also known AGO61). GTDC2&#226;&#8364;&#8482;s <br /> predicted role uncharacterized glycosyltransferase consistent function genes known mutated in <br /> WWS involved glycosylation transmembrane receptor dystroglycan. Therefore, explore role GTDC2 <br /> loss function development, used morpholino-mediated knockdown zebrafish ortholog, gtdc2. We gtdc2 <br /> knockdown zebrafish replicates WWS features (hydrocephalus, ocular defects, muscular dystrophy), strongly suggesting <br /> that GTDC2 mutations cause WWS. <br />  <br /> Walker-Warburg syndrome (WWS, known                                    19; Table S1, available online). All subjects enrolled <br /> MDDGA1&#226;&#8364;&#8220;7 [MIM 236670, 613150, 253280, 253800,                                  informed consent obtained, research <br /> 613153, 613154, 614643]) severe neuromuscular                         conducted according protocols approved the <br /> disorder characterized congenital muscular dystrophy,                       institutional review board Boston Children&#226;&#8364;&#8482;s Hospital. <br /> ocular malformations, smooth appearance cortical                        All affected individuals diagnosed classic WWS <br /> surface accompanied neuronal migration defects                              basis following criteria: (1) brain malfor- <br /> (cobblestone lissencephaly), ventricular enlargement                       mation characterized cobblestone lissencephaly on <br /> (hydrocephalus).1,2 Mutations seven genes (POMT1                            neuroimaging, severe hydrocephalus, cerebellar <br /> [MIM 607423], POMT2 [MIM 607439], POMGNT1 [MIM                                 hypoplasia; (2) presence ocular malformations <br /> 606822], FKTN [MIM 607440], FKRP [MIM 606596],                                 (microphthalmia macrophthalmia, retinal dysplasia, <br /> LARGE [MIM 603590], ISPD [MIM 614631]) known                           optic nerve hypoplasia, anterior chamber defects); <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> cause WWS2&#226;&#8364;&#8220;5 identified 50%&#226;&#8364;&#8220;60%                          (3) clinical diagnosis, usually based severe hypo- <br /> cases.4,6&#226;&#8364;&#8220;8 Identification additional genes mutated                      tonia, congenital muscular dystrophy.1,9 Serum creatine <br /> WWS hindered small size available                       kinase histopathological confirmation muscular <br /> pedigrees, DNA available                        dystrophy available cases of <br /> two affected individuals. To increase power under-                      clinical severity reduced lifespan associated with <br /> stand genetic architecture WWS identify                          condition. <br /> mutations additional genes, collected cohort                          We determined <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> homozygosity in <br /> affected individuals consanguineous marriages (n &#194;&#188;                        consanguineous pedigrees hybridizing genomic <br />  <br /> 1 <br />   Division Genetics, The Manton Center Orphan Disease Research, Boston Children&#226;&#8364;&#8482;s Hospital, Boston, MA 02115, USA; 2Howard Hughes Medical <br /> Institute, Boston Children&#226;&#8364;&#8482;s Hospital, Boston, MA 02115, USA; 3Department Pharmacology Physiology, The George Washington University School <br /> of Medicine, Washington, DC 20037, USA; 4Department Medicine, Center Human Genome Variation, Duke University School Medicine, Durham, <br /> NC 27708, USA; 5Program Medical Population Genetics, Broad Institute MIT Harvard, Cambridge, MA 02142, USA; 6Genomics Program, <br /> Boston Children&#226;&#8364;&#8482;s Hospital, Boston, MA 02115, USA; 7Department Pediatrics, The University Jordan, 11941 Amman, Jordan; 8Department of <br /> Pediatrics, King Abdulaziz Medical City, Riyadh 11426, Saudi Arabia; 9Department Medical Genetics, University British Columbia, Vancouver, BC <br /> V5Z 4H4, Canada; 10Child Family Research Institute, Vancouver, BC V5Z 4H4, Canada; 11Division Pediatric Neurology, Department Pediatrics, <br /> King Saud University College Medicine, Riyadh 11461, Saudi Arabia; 12Division Genetics, Department Pediatrics, Loma Linda University Medical <br /> Center, Loma Linda, CA 92354, USA; 13Division Pediatric Neurology, Fondazione Istituto di Ricerca e Cura Carattere Scientifico Istituto Neurologico C. <br /> Besta, 20126 Milan, Italy; 14Division Neuromuscular Diseases Neuroimmunology, Fondazione Istituto di Ricerca e Cura Carattere Scientifico <br /> Istituto Neurologico C. Besta, 20126 Milan, Italy <br /> *Correspondence: christopher.walsh@childrens.harvard.edu <br /> http://dx.doi.org/10.1016/j.ajhg.2012.07.009. &#195;&#8220;2012 The American Society Human Genetics. All rights reserved. <br />  <br />  <br />  <br />  <br />                                                                 The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 541 <br />  Table 1.   Clinical Features Individuals FKTN, POMT2, GTDC2 Mutations <br />                                                 Cobblestone            Enlarged       Cerebellar <br /> Person        Age Exam         Gene          Lissencephaly          Ventricles     Hypoplasia    Ocular Defects        Muscular Defects <br />  <br /> 82-1          2 months            FKTN          &#195;&#190;                      &#195;&#190;              &#195;&#190;             retinal dysplasia,    CK &#194;&#188; 44,566 U/la <br />                                                                                                     glaucoma <br />  <br /> 144-1         6 months            POMT2         &#195;&#190;                      &#195;&#190;              &#195;&#190;             glaucoma              hypotonia <br />  <br /> 86-1          23 weeks            GTDC2         &#195;&#190;                      &#195;&#190;              &#195;&#190;                                     <br />  <br /> 86-2          20 weeks                          &#195;&#190;                      &#195;&#190;              &#195;&#190;             N/A                   N/A <br />  <br /> 90-1          2 months            GTDC2         &#195;&#190;                      &#195;&#190;              &#195;&#190;             retinal dysplasia     hypotonia <br />  <br /> 100-1         1 month             GTDC2         &#195;&#190;                      &#195;&#190;              &#195;&#190;             microphthalmia    hypotonia <br />                                                                                                     macrophthalmia <br />  <br /> The following abbreviations used: CK, creatine kinase; N/A, available. <br /> a <br />  Normal CK range: 40&#226;&#8364;&#8220;180 U/l. <br />  <br />  <br />  <br /> DNA probands available family members                           (1) likely pathogenic (missense nonsense <br /> on whole-genome SNP arrays&#226;&#8364;&#8221;Affymetrix 250K StyI Il-                           variants, frameshift indels, splicing errors within <br /> lumina 610-Quad&#226;&#8364;&#8221;at W.M. Keck Foundation Biotech-                             10 bp splice site), (2) novel rare (minor allele <br /> nology Resource Laboratory Yale University. Genotyp-                          frequency &lt; 0.5%) control cohort than <br /> ing data analyzed in-house-developed software                          7,000 control individuals (1,000 Genomes November <br /> for identification regions homozygosity (ROHs)                         2011 release, 5,379 exomes National Heart, <br /> throughout genome, genes known mutated                             Lung, Blood Institute [NHLBI] Grand Opportunity <br /> in WWS sequenced contained                               Exome Sequencing Project, 69 genomes Complete <br /> in ROH present affected individual. As                          Genomics, 821 in-house Walsh lab exomes), (3) found <br /> previously reported,6 homozygous mutations                            ROH propositus, (4) inherited <br /> in families (8 19; 42%) (Table S1). Homozygosity                        autosomal-recessive pattern family upon <br /> analysis remaining 11 families identified average                      Sanger sequencing parents unaffected siblings. <br /> percentage homozygosity 9.3% (1.3% 5 standard                              Filtered variants Middle Eastern individuals, are <br /> error mean), larger expected 6%                         widely represented exome databases, were <br /> for first-cousin union10 suggests increased back-                          genotyped 384 ethnically matched individ- <br /> ground consanguinity captured                           uals mass-spectrometry-based Sequenom SNP geno- <br /> available nuclear pedigrees (Table S1). Multiple regions                         typing Intellectual Developmental Disabilities <br /> were shared families, expected                        Research Center (IDDRC) Molecular Genetics Core at <br /> chance type cohort, fact region                       Boston Children&#226;&#8364;&#8482;s Hospital confirming vari- <br /> overlapped families indicates                            ants rare population-specific polymorphisms. <br /> multiple additional genes remain identified.                               This approach narrowed number variants an <br />    The percentage homozygosity cohort high                         average number candidates family (Table S2). <br /> but sufficient reduce number candidate genes                          Homozygous mutations genes known involved <br /> to thousand sample. Therefore,                           WWS identified affected individuals two <br /> performed whole-exome sequencing (WES) analysis                               families (Figure S1); homozygous 1 bp inser- <br /> study candidate genes once.11 Seven remain-                        tion leading frameshift change (c.642dup [p.Asp215*]) <br /> ing probands suitable WES, whole-exome                              (NM_006731.2) FKTN family 82, first-cousin <br /> DNA libraries prepared genomic DNA                                  union Hispanic ethnicity United States (Table <br /> hybridization Agilent SureSelect All Exon 37Mb                          1 Figures S1A S1B), homozygous missense <br /> kit run Illumina HiSeq2000 machine                                change (c.1433A&gt;G [p.His478Arg]) (NM_013382.4) in <br /> either Duke Center Human Genome Variation                             POMT2 family 144, second-cousin union <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> in Durham, NC Broad Institute MIT                               Sudan (Table 1 Figures S1C S1D). The missense <br /> Harvard Cambridge, MA. Sequencing reads                                  change obtained highest pathogenicity score two <br /> aligned Human Reference Genome (UCSC hg19)                                independent pathogenicity prediction algorithms, Poly- <br /> with Burrows-Wheeler Aligner software.12 Variant calling                         phen214 SIFT15 (Polyphen2 &#194;&#188; 1 [probably damaging]; <br /> was performed SAMtools software,12                               SIFT &#194;&#188; 0 [damaging]). Despite present large <br /> SequenceVariantAnalyzer ANNOVAR13 used                                ROH affected individuals, genes not <br /> annotating variants. Sequencing yielded average                            previously sequenced family, a <br /> 49,538 variants individual (see Table S2 WES statis-                     smaller homozygous region identical haplotype <br /> tics). Variants filtered custom MySQL                             present gene parents <br /> queries identification recessive mutations,                     likely common haplotype the <br />  <br />  <br /> 542 The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 <br />  Figure 1. Three Independent Alleles GTDC2 Cause WWS <br /> (A) Pedigree structure family 90 (DNA available individuals marked asterisks) T2-weighted axial magnetic resonance <br /> images (MRIs) propositus. The MRI left shows microphthalmia macrophthalmia (asterisks) cerebellar hypoplasia <br /> (red arrow), MRI right shows severe ventricular enlargement cortical-migration defects characteristic cobblestone <br /> lissencephaly (inset). <br /> (B) Pedigree structure family 100 coronal MRIs propositus. These images degree malformation similar that <br /> seen family 90. <br /> (C&#226;&#8364;&#8220;D) Sanger sequencing confirming mutations identified WES families 90 100. <br /> (E) Conservation arginine 158 surrounding region vertebrates. <br /> (F) Sanger sequencing additional GTDC2 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> identified family 86. <br /> (G) Schematics showing overlapping regions homozygosity families affected GTDC2 mutations GTDC2 <br /> protein structure including predicted functional domains. Locations mutations listed protein. <br />  <br />  <br /> population. These findings illustrate power WES            90 100, respectively (Figure 1G). The phenotypes of <br /> identify disease-causing mutations unbiased                    families undistinguishable other <br /> manner known disease genes tested                 classic WWS cases&#226;&#8364;&#8221;they included <br /> independently.                                                       cobblestone lissencephaly, severe hydrocephalus, severe <br />    We analyzed variants remaining WWS-               hypoplasia cerebellar vermis, muscle hypotonia <br /> affected families identified independent homozy-             (Table 1 Figures 1A 1B). No muscle biopsy was <br /> gous changes glycosyltransferase-like domain-containing           performed affected individuals died within <br /> 2 (GTDC2), single-exon gene short arm (3p22.1)           days weeks birth. <br /> chromosome 3 (Figure 1). The variant nonsense               WWS severe spectrum dystroglycan- <br /> change (c.1333C&gt;T [p.Arg445*]) (NM_032806.4) male               glycosylation disorders affecting muscle, eyes, and <br /> from Jordanian first-cousin union (family 90, Figure 1C)           brain. It variable severity commonly defined as <br /> and led possible early truncation GTDC2 (Figure 1G);         muscular-dystrophy dystroglycanopathies (MDDGB1&#226;&#8364;&#8220;6 <br /> the second variant missense change predicted                [MIM 613155, 613156, 613151, 613152, 606612, and <br /> be deleterious affected conserved amino acid              608840] MDDGC1&#226;&#8364;&#8220;5 MDDGC7 [MIM 609308, <br /> (c.473G&gt;A [p.Arg158His]; Polyphen2 &#194;&#188; 1 [probably                     613158, 613157, 611588, 607155, 613818]).7,9,16 We <br /> damaging] SIFT &#194;&#188; 0 [damaging]) female Saudi          sequenced GTDC2 validation cohort nonconsangui- <br /> family (family 100, Figures 1D 1E). Each change              neous cases (n &#194;&#188; 52) spanning dystroglycanopathy <br /> contained large ROH: 25.8 Mb 28.1 Mb families            phenotypic spectrum identify additional alleles. We <br />  <br />  <br />                                                        The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 543 <br />                                                                                      Figure 2. Expression GTDC2 Its <br />                                                                                     Orthologs Mice Zebrafish <br />                                                                                     (A) qPCR analysis GTDC2 mRNA expres- <br />                                                                                     sion fetal adult human tissues. High <br />                                                                                     expression stages the <br />                                                                                     brain, muscle, kidneys, heart. In the <br />                                                                                     adult brain, cerebellum cortex <br />                                                                                     highest expression. Error bars <br />                                                                                     represent standard error three <br />                                                                                     sample repeats normalized beta-actin. <br />                                                                                     (B) Murine Gtdc2 expressed brain <br />                                                                                     development. At E14.5 (left panels), <br />                                                                                     expression observed the <br />                                                                                     cortical plate (cp) developing <br />                                                                                     brain, eye, <br />                                                                                     including lens cornea. No staining <br />                                                                                     evident sense control. The scale <br />                                                                                     bars left panels represent 200 mm. <br />                                                                                     Cortical expression (right panels) peaks <br />                                                                                     E14.5 E16.5 reduced <br />                                                                                     birth (P0). The scale bar right <br />                                                                                     panel represents 100 mm. <br />                                                                                     (C) In situ hybridization zebrafish gtdc2 <br />                                                                                     1 dpf shows expression the <br />                                                                                     body; highest staining brain <br />                                                                                     eye boundaries between <br />                                                                                     somites (arrowheads inset). The scale <br />                                                                                     bar represents 150 mm. <br />  <br />  <br />  <br />  <br />                                                                                      LARGE (another gene involved in <br />                                                                                      dystroglycanopathies), pre- <br />                                                                                      dicted encode xylosyltrans- <br />                                                                                      ferase.20 We studied expression of <br />                                                                                      GTDC2 mRNA multiple human <br />                                                                                      tissues using quantitative PCR <br />                                                                                      (qPCR) fetal develop- <br /> identified additional nonsense allele family 86,      ment adult. cDNA samples postmortem <br /> Indian family pregnancies terminated         fetal adult human tissues obtained commercially <br /> as result diagnosis ventricular enlargement   (BioChain Institute Clontech). qPCR performed <br /> gestation. Both fetuses female diagnosed         SYBR Green reagents (Applied Biosystems) an <br /> with WWS autopsy (Table 1). DNA available             ABI 7500 qPCR platform previously described21 with <br /> from affected individuals, sequencing       primers hGTDC2_F 50 -GCGGAGTCCAGGCTTCAGGCT <br /> revealed homozygous GTDC2 <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> (c.590G&gt;A)                TTC-30 hGTDC2_R 50 -CCGCCGAGAGGTGCATCCT <br /> generating stop codon amino acid 197 (p.Trp197*;          AATG-30 normalizing expression values beta-actin <br /> Figure 1F). The family displayed small total       (ACTB). <br /> homozygosity (1%)&#226;&#8364;&#8221;only regions larger               At fetal adult stages, GTDC2 expression was <br /> 5 cM, GTDC2 contained largest (7.5 Mb       especially high brain, muscle, heart, kidneys <br /> 5.9 cM) regions chromosome 3 (Figure 1G)&#226;&#8364;&#8221;          (Figure 2A), variably affected dystroglycan- <br /> suggesting parents distantly related.        opathy cases.6,22 Brain expression highest cortex <br />    GTDC2 ideal candidate gene WWS            cerebellum (Figure 2A), high expression was <br /> it predicted multiple protein-domain prediction algo-    noted pancreas (Figure S2A). To analyze possible <br /> rithms contain uncharacterized glycosyltransferase       changes cellular distribution, analyzed expression of <br /> domain (Figure 1G). All genes mutations          murine ortholog GTDC2, Gtdc2, situ hybridiza- <br /> been identified date involved glycosylation       tion (performed In Situ Hybridization Core Facility at <br /> the transmembrane protein dystroglycan,          University North Carolina School Medicine in <br /> glycosylated order interact extracellular matrix    Chapel Hill, NC) probe amplified embryonic <br /> components laminins, binding essential     day (E)12.5 mouse brain cDNA library (a kind gift of <br /> for brain, eye, muscle integrity.2,17&#226;&#8364;&#8220;19 GTDC2 (which   Byoung-Il Bae Boston Children&#226;&#8364;&#8482;s Hospital) (primers <br /> alternatively called AGO61) predicted homology        mGtdc2-forward 50 -TTCCCTTACGCTGTCAATCC-30 <br /> belong glycosyltransferase family 61, and, similar       mGtdc2-reverse 50 -TACGTGTTCTCCCCTTGCTC-30 ). <br />  <br />  <br /> 544 The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 <br />  We Gtdc2 highly expressed brain          gtdc2 knockdown severely affected survival: 43.4% of <br /> eye development; peaks cerebral cortex        embryos died 3 days injection. <br /> the week gestation neuronal progenitors     However, clutchmates uninjected injected <br /> the ventricular zone migrating differentiating         control MO displayed survival rates 98.4% and <br /> neurons declines birth, supporting role       89.0%, respectively (Figure S3A). Surviving morphants at <br /> this gene neuronal development (Figure 2B),         3 dpf shorter controls bent <br /> we did observe differential expression neuronal         tail, impaired motility, smaller eyes retina <br /> progenitors neuronal subtypes.                              failed fuse ventrally, domed appearance the <br />     To test GTDC2 loss function affect brain      head (Figures 3D 3G Figure S3B). <br /> and muscle development, used zebrafish,           When pigmentation blocked 1 dpf the <br /> has recently established animal model           addition 0.003% 1-phenyl 2-thiourea water, <br /> dystroglycanopathies.23&#226;&#8364;&#8220;26 The mRNA zebrafish          external analysis head revealed extreme ventricular <br /> ortholog (gtdc2) GTDC2 expressed zebrafish       enlargement (hydrocephalus), accompanied by <br /> embryo larva development;             hemorrhaging (Figure 3G), affected larvae (Fig- <br /> especially high expression developing brain, eyes,      ure 3E). Ventral compression reduction brain <br /> and somites (Figure 2C Figure S2B), consistent        volume evident histological analysis coronal <br /> expression zebrafish orthologs genes           cryosections stained DAPI (Figure 3E). Disorganiza- <br /> mutated WWS.27,28 Zebrafish situ hybridization       tion retinal epithelium observed few <br /> performed according standard protocols29 probe       morphants; variable severity ultrathin <br /> amplified mixed 20-somite 5 day postfertilization    plastic-embedded sections stained 1% toluidine blue <br /> (dpf) zebrafish cDNA library (a kind gift Peter Wang     1% sodium borate (Figure S3C) Conventional <br /> Boston Children&#226;&#8364;&#8482;s Hospital) (primers zfgtdc2-forward      Electron Microscopy Facility Harvard Medical School. <br /> 50 -ATGCACATTTTCCACGATGA-30 zfgtdc2-reverse                Finally, muscle development severely disrupted by <br /> 50 -AACGACTGCTCCTCCAAAGA-30 ). To knock                   loss dystrophin glycosylated dystroglycan <br /> gtdc2, tested different antisense morpholino oligo-     myosepta, junction muscle fibers in <br /> nucleotides (MOs) targeted interfere gtdc2 trans-      extracellular matrix enriched (Figure 3F). <br /> lation blocking start codon (gtdc2Start:         Disorganization muscle fibers observed <br /> 50 -CCGGCAGGTTCATCCTACACCCGAT-30 ) splice               histology (Figure S3D). These phenotypes (hydro- <br /> donor site intron present gene          cephalus retinal muscle defects) hallmarks <br /> the fish (gtdc2Splice: 50 -TTATCATAACCCTGTGCACCT               WWS consistent eye, muscle, and <br /> GTGA-30 ). When MOs injected fertilized        brain phenotypes observed morphants genes (such <br /> zebrafish oocytes, provided similar results (gtdc2Start   POMT1, POMT2, FKTN, FKRP, ISPD) previously <br /> at 3&#226;&#8364;&#8220;5 ng gtdc2Splice 7&#226;&#8364;&#8220;10 ng). Therefore, data    mutated WWS.24&#226;&#8364;&#8220;26,28 All phenotypes <br /> only gtdc2Start shown. The standard control MO         ameliorated GTDC2 mRNA injections when <br /> provided Gene Tools used control       embryos analyzed 2 3 dpf (Figure 3G and <br /> concentration used experimental MO injections.         Figure S3E). <br /> Injected embryos/larvae uninjected clutchmate                 In summary, combined whole-exome se- <br /> controls analyzed 0, 1, 2, 3 dpf survival      quencing, homozygosity mapping, functional anal- <br /> and morphology tail, head, eyes. Develop-           ysis zebrafish identify validate GTDC2 mutations <br /> mental defects gtdc2 morphants evident 1          cause WWS. In addition, reduced the <br /> dpf&#226;&#8364;&#8221;one quarter embryos severely affected          number candidate variants families a <br /> with short body, poorly developed tail,       manageable number variants family. This <br /> and discernible eyes head structures. Half        approach likely yield additional mutated genes <br /> morphants mildly affected scored            future help unravel pathways involved in <br /> combination phenotypes: tail, U-shaped somites,       WWS determine role different glycosyltrans- <br /> reduced mobility inside chorion, small head,         ferases brain muscle development. <br /> underdeveloped eyes, ventral fusion retina <br /> was delayed (Figure 3A). We asked coinjection of <br /> the in-vitro-synthesized human GTDC2 mRNA             Supplemental Data <br /> gtdc2 MO improve phenotypes              Supplemental Data include figures tables be <br /> this case (Figure 3B). However, rescue     article online http://www.cell.com/AJHG. <br /> effect observed tested mRNA mutated <br /> to include p.Arg158His allele (Figure 3C). The Arg158 <br /> residue highly conserved GTDC2, located     Acknowledgments <br /> the beginning glycosyltransferase domain; there-        We thank families enrolled studies the <br /> fore, possible change severely impair    physicians contributed individuals validation <br /> protein function.                                              cohort. The EuroBioBank Telethon Network Genetic Biobanks <br />  <br />  <br />  <br />                                                   The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 545 <br />                                                                                             Figure 3. Loss gtdc2 Zebrafish <br />                                                                                            Embryo Recapitulates Phenotypes Ob- <br />                                                                                            served WWS <br />                                                                                            (A) Mild (center) severe (right) pheno- <br />                                                                                            types observed 1 dpf zebrafish <br />                                                                                            embryo injection MO designed <br />                                                                                            start site gtdc2. <br />                                                                                            (B) Phenotypes observed 1 dpf be <br />                                                                                            rescued coinjection gtdc2 MO <br />                                                                                            150 pg GTDC2 mRNA, no <br />                                                                                            effect observed injection the <br />                                                                                            mRNA (hGTDC2). <br />                                                                                            (C) Coinjection 150 pg GTDC2 <br />                                                                                            mRNA containing p.Arg158His alter- <br />                                                                                            ation able rescue morphant <br />                                                                                            <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span>. Representative experiments <br />                                                                                            shown. <br />                                                                                            (D) Surviving morphants 3 dpf show <br />                                                                                            developmental defects: reduced <br />                                                                                            <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, bent tail, failure ventral retinal <br />                                                                                            fusion, dysmorphic head shape. <br />                                                                                            (E) 3 dpf morphants display hydro- <br />                                                                                            cephalus (lower left panel: whole-mount <br />                                                                                            nonpigmented larva [asterisk]) and <br />                                                                                            ventral compression remaining <br />                                                                                            brain tissue (lower right panel: DAPI stain- <br />                                                                                            ing histological section location <br />                                                                                            matching wild-type [WT] section <br />                                                                                            above). The scale bar represents 200 mm. <br />                                                                                            (F) Immunohistochemistry sections <br />                                                                                            WT morphant muscle shows <br />                                                                                            expression dystrophin <br />                                                                                            (green, left) glycosylated dystroglycan <br />                                                                                            (green, right) reduced myosepta <br />                                                                                            (marked arrowheads). Nuclei are <br />                                                                                            labeled DAPI (blue). The scale bar repre- <br />                                                                                            sents 50 mm. <br />                                                                                            (G) Rescue GTDC2 mRNA (on <br />                                                                                            right) ameliorates severe head <br />                                                                                            phenotypes observed morphant <br />                                                                                            (center). These phenotypes failure of <br />                                                                                            retinal fusion (arrowhead), hydrocephalus <br />                                                                                            (asterisk), hemorrhage (arrow). The <br />                                                                                            scale bar represents 200 mm. <br />  <br />  <br />  <br />  <br /> (GTB07001F) acknowledged providing biological          supported Intellectual Developmental Disabilities <br /> samples. We grateful A.J. Barkovich help   Research Centers (P30HD19655). Sequencing Broad Insti- <br /> review magnetic resonance imaging scans. M.C.M.          tute supported grant NIH American <br /> supported Junior Faculty Career Development Award         Recovery &amp; Reinvestment Act (National Institute Mental <br /> the Manton Center Orphan Disease Research K99/R00         Health RC2MH089952). C.A.W. Investigator Howard <br /> Transition Independence award National Institutes       Hughes Medical Institute. <br /> of Health (NIH) (National Institute Child Health &amp; Human <br /> Development, K99HD067379). T.W.Y. supported NIH            Received: April 10, 2012 <br /> T32 grant (T32NS007484-08), Clinical Investigator Training      Revised: June 16, 2012 <br /> Program Harvard-MIT Health Science Technology,           Accepted: July 11, 2012 <br /> the Nancy Lurie Marks Junior Faculty MeRIT Fellowship. Research     Published online: September 6, 2012 <br /> was supported grants NIH (National Institute of <br /> Neurological Disorders Stroke R01NS035129 C.A.W. <br /> and National Institute Arthritis Musculoskeletal         Web Resources <br /> Skin Diseases R01AR044345 A.H.B), Muscular Dystrophy <br />                                                                     The URLs data presented follows: <br /> Association M.C.M. A.H.B., William Randolph Hearst <br /> Fund M.C.M. V.A.G., Manton Center Orphan         ANNOVAR, http://www.openbioinformatics.org/annovar/ <br /> Disease Research M.C.M., A.H.B., C.A.W. Sequencing           Carbohydrate-Active Enzymes Database, www.cazy.org <br /> and Sequenom genotyping Boston Children&#226;&#8364;&#8482;s Hospital          dCHIP Software, http://biosun1.harvard.edu/complab/dchip/ <br />  <br />  <br />  <br /> 546 The American Journal Human Genetics 91, 541&#226;&#8364;&#8220;547, September 7, 2012 <br />  NHLBI Exome Variant Server, http://evs.gs.washington.edu/EVS/                   (2010). A method server predicting damaging <br /> Online Mendelian Inheritance Man (OMIM), http://www.                         missense mutations. Nat. Methods 7, 248&#226;&#8364;&#8220;249. <br />   omim.org                                                                15.   Kumar, P., Henikoff, S., Ng, P.C. (2009). Predicting the <br /> PolyPhen-2, http://genetics.bwh.harvard.edu/pph2/                               effects coding non-synonymous variants protein func- <br /> SIFT, http://sift.jcvi.org/                                                     tion using SIFT algorithm. Nat. Protoc. 4, 1073&#226;&#8364;&#8220;1081. <br />                                                                           16.   Clement, E., Mercuri, E., Godfrey, C., Smith, J., Robb, S., Kinali, <br />                                                                                 M., Straub, V., Bushby, K., Manzur, A., Talim, B., et al. (2008). <br /> References <br />                                                                                 Brain involvement muscular dystrophies defective <br />  1. Dobyns, W.B., Pagon, R.A., Armstrong, D., Curry, C.J.,                      dystroglycan glycosylation. Ann. Neurol. 64, 573&#226;&#8364;&#8220;582. <br />     Greenberg, F., Grix, A., Holmes, L.B., Laxova, R., Michels,           17.   Michele, D.E., Barresi, R., Kanagawa, M., Saito, F., Cohn, R.D., <br />     V.V., Robinow, M., et al. (1989). Diagnostic criteria                   Satz, J.S., Dollar, J., Nishino, I., Kelley, R.I., Somer, H., et al. <br />     Walker-Warburg syndrome. Am. J. Med. Genet. 32, 195&#226;&#8364;&#8220;210.                    (2002). Post-translational disruption dystroglycan-ligand <br />  2. Godfrey, C., Foley, A.R., Clement, E., Muntoni, F. (2011).              interactions congenital muscular dystrophies. Nature 418, <br />     Dystroglycanopathies: Coming focus. Curr. Opin. Genet.                 417&#226;&#8364;&#8220;422. <br />     Dev. 21, 278&#226;&#8364;&#8220;285.                                                     18.   Hu, H., Li, J., Gagen, C.S., Gray, N.W., Zhang, Z., Qi, Y., and <br />  3. Barresi, R., Campbell, K.P. (2006). Dystroglycan: From                  Zhang, P. (2011). Conditional <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>knockout</span> protein O-manno- <br />     biosynthesis pathogenesis human disease. J. Cell Sci.                 syltransferase 2 reveals tissue-specific roles O-mannosyl <br />     119, 199&#226;&#8364;&#8220;207.                                                               glycosylation brain development. J. Comp. Neurol. 519, <br />  4. Roscioli, T., Kamsteeg, E.J., Buysse, K., Maystadt, I., van Reeu-           1320&#226;&#8364;&#8220;1337. <br />     wijk, J., van den Elzen, C., van Beusekom, E., Riemersma, M.,         19.   Manzini, M.C., Walsh, C.A. (2011). What disorders of <br />     Pfundt, R., Vissers, L.E., et al. (2012). Mutations ISPD cause           cortical development tell cortex: One plus one <br />     Walker-Warburg syndrome defective glycosylation                      does make two. Curr. Opin. Genet. Dev. 21, 333&#226;&#8364;&#8220;339. <br />     a-dystroglycan. Nat. Genet. 44, 581&#226;&#8364;&#8220;585.                              20.   Inamori, K., Yoshida-Moriguchi, T., Hara, Y., Anderson, M.E., <br />  5. Willer, T., Lee, H., Lommel, M., Yoshida-Moriguchi, T.,                     Yu, L., Campbell, K.P. (2012). Dystroglycan function <br />     Bernabe, D.B., Venzke, D., Cirak, S., Schachter, H., Vajsar,             requires xylosyl- glucuronyltransferase activities of <br />     J., Voit, T., et al. (2012). ISPD loss-of-function mutations                LARGE. Science 335, 93&#226;&#8364;&#8220;96. <br />     disrupt dystroglycan O-mannosylation cause Walker-                21.   Maynard, T.M., Meechan, D.W., Dudevoir, M.L., Gopalak- <br />     Warburg syndrome. Nat. Genet. 44, 575&#226;&#8364;&#8220;580.                                  rishna, D., Peters, A.Z., Heindel, C.C., Sugimoto, T.J., Wu, Y., <br />  6. Manzini, M.C., Gleason, D., Chang, B.S., Hill, R.S., Barry, B.J.,           Lieberman, J.A., Lamantia, A.S. (2008). Mitochondrial <br />     Partlow, J.N., Poduri, A., Currier, S., Galvin-Parton, P., Shapiro,         localization function subset 22q11 deletion <br />     L.R., et al. (2008). Ethnically diverse causes Walker-Warburg            syndrome candidate genes. Mol. Cell. Neurosci. 39, 439&#226;&#8364;&#8220;451. <br />     syndrome (WWS): FCMD mutations common cause                22.   Bello, L., Melacini, P., Pezzani, R., D&#226;&#8364;&#8482;Amico, A., Piva, L., <br />     WWS outside Middle East. Hum. Mutat. 29, E231&#226;&#8364;&#8220;E241.               Leonardi, E., Torella, A., Soraru, G., Palmieri, A., Smaniotto, <br />  7. Mercuri, E., Messina, S., Bruno, C., Mora, M., Pegoraro, E.,                G., et al. (2012). Cardiomyopathy patients POMT1- <br />     Comi, G.P., D&#226;&#8364;&#8482;Amico, A., Aiello, C., Biancheri, R., Berardinelli,           related congenital limb-girdle muscular dystrophy. Eur. <br />     A., et al. (2009). Congenital muscular dystrophies                     J. Hum. Genet. Published online May 2, 2012. http://dx.doi. <br />     defective glycosylation dystroglycan: A population study.                org/10.1038/ejhg.2012.71. <br />     Neurology 72, 1802&#226;&#8364;&#8220;1809.                                              23.   Gupta, V., Kawahara, G., Gundry, S.R., Chen, A.T., Lencer, <br />  8. Bouchet, C., Gonzales, M., Vuillaumier-Barrot, S., Devisme, L.,             W.I., Zhou, Y., Zon, L.I., Kunkel, L.M., Beggs, A.H. <br />     Lebizec, C., Alanio, E., Bazin, A., Bessie`res-Grattagliano, B.,            (2011). The zebrafish dag1 mutant: A novel genetic model <br />     Bigi, N., Blanchet, P., et al. (2007). Molecular heterogeneity           dystroglycanopathies. Hum. Mol. Genet. 20, 1712&#226;&#8364;&#8220;1725. <br />     fetal forms type II lissencephaly. Hum. Mutat. 28, 1020&#226;&#8364;&#8220;1027.      24.   Lin, Y.Y., White, R.J., Torelli, S., Cirak, S., Muntoni, F., and <br />  9. Cormand, B., Pihko, H., Baye&#194;&#180;s, M., Valanne, L., Santavuori, P.,            Stemple, D.L. (2011). Zebrafish Fukutin family proteins link <br />     Talim, B., Gershoni-Baruch, R., Ahmad, A., van Bokhoven, H.,                unfolded protein response dystroglycanopathies. <br />     Brunner, H.G., et al. (2001). Clinical genetic distinction              Hum. Mol. Genet. 20, 1763&#226;&#8364;&#8220;1775. <br />     Walker-Warburg syndrome muscle-eye-brain                  25.   Kawahara, G., Guyon, J.R., Nakamura, Y., Kunkel, L.M. <br />     disease. Neurology 56, 1059&#226;&#8364;&#8220;1069.                                           (2010). Zebrafish models human FKRP muscular dystro- <br /> 10. Lander, E.S., Botstein, D. (1987). Homozygosity mapping:                phies. Hum. Mol. Genet. 19, 623&#226;&#8364;&#8220;633. <br />     A way map human recessive traits DNA inbred            26.   Thornhill, P., Bassett, D., Lochmu     &#194;&#168; ller, H., Bushby, K., and <br />     children. Science 236, 1567&#226;&#8364;&#8220;1570.                                           Straub, V. (2008). Developmental defects zebrafish model <br /> 11. Ng, S.B., Buckingham, K.J., Lee, C., Bigham, A.W., Tabor, H.K.,             muscular dystrophies associated loss fukutin- <br />     Dent, K.M., Huff, C.D., Shannon, P.T., Jabs, E.W., Nickerson,               related protein (FKRP). Brain 131, 1551&#226;&#8364;&#8220;1561. <br />     D.A., et al. (2010). Exome sequencing identifies cause         27.   Moore, C.J., Goh, H.T., Hewitt, J.E. (2008). Genes required <br />     mendelian disorder. Nat. Genet. 42, 30&#226;&#8364;&#8220;35.                                functional glycosylation dystroglycan conserved <br /> 12. Li, H., Durbin, R. (2009). Fast accurate sho <br /> </body></html>