<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The N-terminal extension S12 influences small ribosomal <br /> subunit assembly Escherichia coli <br />  <br />  <br /> DEEPIKA CALIDAS,1,2 HIRAM LYON,1 GLORIA M. CULVER1,3 <br /> 1 <br />  Department Biology, Center RNA Biology: <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> Genome Therapeutics, University Rochester Medical Center, Rochester, <br /> New York 14627, USA <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> The small subunit (SSU) ribosome E. coli consists core ribosomal RNA (rRNA) surrounded peripherally by <br /> ribosomal proteins (r-proteins). Ten 15 universally conserved SSU r-proteins possess nonglobular regions called <br /> extensions. The N-terminal noncanonically structured extension S12 traverses solvent intersubunit surface the <br /> SSU followed C-terminal globular region adjacent decoding center SSU. The role the <br /> globular region maintaining translational fidelity characterized, role S12 extension SSU structure and <br /> function unknown. We examined effect stepwise truncation extension S12 SSU assembly function in <br /> vitro vivo. Examination vitro assembly presence sequential N-terminal truncated variants S12 reveals <br /> that N-terminal deletions greater amino acids exhibit decreased tRNA-binding activity altered 16S rRNA <br /> architecture particularly platform SSU. While wild-type S12 expressed plasmid rescue genomic <br /> deletion essential gene S12, rpsl; N-terminal deletions S12 exhibit deleterious phenotypic consequences. Partial <br /> N-terminal deletions S12 slow growing cold sensitive. Strains bearing truncations sole copy S12 have <br /> increased levels free SSUs immature 16S rRNA compared wild-type S12. These differences hallmarks of <br /> SSU biogenesis defects, indicating extension S12 plays important role SSU assembly. <br /> Keywords: ribosome; small subunit; biogenesis; assembly; S12; intrinsically disordered proteins <br />  <br />  <br />  <br /> INTRODUCTION                                                                      role extension S12 establishing SSU structure <br />                                                                                   assembly. <br /> Universally conserved small subunit (SSU, 30S) ribosomal <br />                                                                                      The region SSU functions tRNA binding and <br /> protein (r-protein) S12 composed distinct structural <br />                                                                                   decoding lies intersection structural domains: <br /> elements domains life: N-terminal nonglobular <br />                                                                                   body, platform, head, penultimate ultimate stems <br /> extension varying <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> sequence, conserved <br />                                                                                   (Woese et al. 1980; Moazed Noller 1990; Powers and <br /> C-terminal globular region (see Fig. 1; Smith et al. 2008; <br />                                                                                   Noller 1991; Alksne et al. 1993; Lee et al. 1997; Lodmell and <br /> Ben-Shem et al. 2010). S12 essential growth organ- <br />                                                                                   Dahlberg 1997; Clemons et al. 1999; Yusupov et al. 2001). <br /> isms ranging E. coli (Sharma et al. 2007; Shoji et al. <br />                                                                                   The localization globular region S12 proximal <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> 2011) Danio rerio (zebrafish) (Zhang Lin 2009) and <br />                                                                                   decoding center unique SSU r-proteins, and <br /> is key mediator fidelity translation prokaryotes <br />                                                                                   well-characterized role maintaining fidelity trans- <br /> and eukaryotes (Alksne et al. 1993). Examination crys- <br />                                                                                   lation (Gorini Kataja 1964b; Lodmell Dahlberg <br /> tal structure E. coli SSUs (Schuwirth et al. 2005) reveals <br />                                                                                   1997; Clemons et al. 1999; Ogle et al. 2002; Schuwirth et al. <br /> that N-terminus protein binds solvent surface <br />                                                                                   2005). Most contacts N-terminal segment <br /> of SSU, extension contacting rRNA dense re- <br />                                                                                   S12 16S rRNA lie platform inter- <br /> gions end C-terminal globular region localized the <br />                                                                                   acts body 30S subunit (see Fig. 1A; Allers and <br /> intersubunit face SSU (see Fig. 1A). The length lo- <br />                                                                                   Shamoo 2001; Schuwirth et al. 2005). In addition binding <br /> calization extension hints possible func- <br />                                                                                   rRNA, extension interacts S8 S17, these <br /> tional significance (Agarwal et al. 2011) led examine the <br />                                                                                   r-proteins participate assembly body, <br />    2 <br />     Present address: Department Molecular Biology Genetics, <br /> Howard Hughes Medical Institute/Johns Hopkins School Medicine,                 &#194;&#169; 2014 Calidas et al. This article distributed exclusively RNA <br /> Baltimore, MD 21205, USA                                                          Society 12 months full-issue publication date (see <br />    3 <br />     Corresponding author                                                          http://rnajournal.cshlp.org/site/misc/terms.xhtml). After 12 months, is <br />    E-mail gloria.culver@rochester.edu                                             available Creative Commons License (Attribution-NonCommercial <br />    Article published online ahead print. Article publication date   3.0 Unported), described http://creativecommons.org/licenses/by-nc/ <br /> http://www.rnajournal.org/cgi/doi/10.1261/rna.042432.113.                         3.0/. <br />  <br />  <br /> RNA 20:321&#226;&#8364;&#8220;330; Published Cold Spring Harbor Laboratory Press RNA Society                                                                  321 <br />  Calidas et al. <br />  <br />  <br />  <br />                                                                                                           (S12&#226;&#8211;&#181;15N), truncation three-fourths <br />                                                                                                           extension (S12&#226;&#8211;&#181;24N), trunca- <br />                                                                                                           tion entire extension (S12&#226;&#8211;&#181;27N). <br />                                                                                                           SSUs reconstituted vitro full <br />                                                                                                           complement recombinant r-proteins <br />                                                                                                           different S12 variants exhibited di- <br />                                                                                                           minishing functionality decrease <br />                                                                                                           length extension (Fig. 2). In ad- <br />                                                                                                           dition, minimal ribonucleoprotein parti- <br />                                                                                                           cles (RNPs) reconstituted vitro the <br />                                                                                                           presence S12&#226;&#8211;&#181;15N S12&#226;&#8211;&#181;27N <br />                                                                                                           altered structure 16S rRNA <br />                                                                                                           compared RNPs containing <br />                                                                                                           full-length S12, notably vicinity of <br />                                                                                                           central pseudoknot (Fig. 3). Thus, <br /> FIGURE 1. Structural relationship S12 SSU components. (A) Positions S12 (ma- truncation extension alters SSU <br /> genta), S4 (orange), S5 (violet) three-dimensional structure E. coli 30S subunit. <br /> 16S rRNA gray r-proteins omitted clarity. All figures containing three-dimension- <br />                                                                                                           structure function vitro. <br /> al structures prepared using PyMOL Protein Data Bank file 2AW7 (Schuwirth et al.                    We examined ex- <br /> 2005). (B) Location S12 (magenta), S8 (black), S17 (teal) three-dimensional struc- tension played role SSU assembly in <br /> ture 30S subunit. 16S rRNA gray r-proteins omitted clarity. (C) Three-di- vivo. As gene encoding S12 (rpsL) is <br /> mensional structure S12 30S subunit, rotated 180&#194;&#176; respect B, the <br /> variants color coded. S12WT magenta, S12Full Length blue, S12&#226;&#8211;&#181;10N purple, S12&#226;&#8211;&#181;15N essential (Sharma et al. 2007; Shoji et al. <br /> red, S12&#226;&#8211;&#181;27N orange.                                                                               2011), used genomic deletion strain <br />                                                                                                           (&#226;&#8211;&#181;rpsl pSTR) viability dependent <br />                                                                                                           plasmid expression wild-type <br /> shoulder, platform SSU (see Fig. 1B; Stern et al.                    S12 (pSTR). The wild-type plasmid replaced by <br /> 1988a,b; Svensson et al. 1988; Schuwirth et al. 2005; Calidas                   plasmids expressing S12 variants (pS12&#226;&#8211;&#181;6N, pS12&#226;&#8211;&#181;11N, <br /> and Culver 2011). Little known role extension                  pS12&#226;&#8211;&#181;15N, pS12&#226;&#8211;&#181;24N) plasmid shuffle create &#226;&#8211;&#181;rpsl <br /> in structure function SSU. The widespread con-                   pS12&#226;&#8211;&#181;6N, &#226;&#8211;&#181;rpsl pS12&#226;&#8211;&#181;11N, &#226;&#8211;&#181;rpsl pS12&#226;&#8211;&#181;15N (Fig. 4A). <br /> tacts extension 16S rRNA r-pro-                      Diminution N-terminal segment correlated with <br /> teins led hypothesize plays role aligning distant            increasing growth defects subunit profile abnormali- <br /> structural elements SSU.                                                 ties (Figs. 4B,C, 5A,B). Precursor 16S rRNA (17S rRNA) ob- <br />    More one-third r-proteins possess extensions                     served SSU fractions &#206;&#8221;rpsl pS12&#226;&#8211;&#181;15N strain, also <br /> (Brodersen et al. 2002; Schuwirth et al. 2005)                       lesser degree &#226;&#8211;&#181;rpsl pS12&#226;&#8211;&#181;6N &#226;&#8211;&#181;rpsl pS12&#226;&#8211;&#181;11N <br /> been hypothesized extensions scaffold tertiary structure                   (Fig. 5C). Data vitro vivo studies suggest <br /> (Ban et al. 2000; Wimberly et al. 2000; Wilson Nierhaus                     N-terminal extension S12 plays critical role in <br /> 2005). However, exact role extensions bacterial <br /> ribosome function assembly unclear. Of r-proteins <br /> with extensions studied, complete truncation <br /> of extension E. coli L20 profound effect, <br /> leading dominant lethal <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, partial dele- <br /> tions resulted slow growth defects large subunit <br /> (LSU 50S) assembly (Guillier et al. 2005). In contrast, dele- <br /> tion long loops E. coli L4 L22 effect the <br /> assembly 50S subunit (Zengel et al. 2003). In yeast, mu- <br /> tations C-terminal extension r-protein S14, homo- <br /> log bacterial r-protein S11 blocked maturation precursor <br /> SSU particles (Jakovljevic et al. 2004). Thus, common role <br /> and significance r-protein extensions ribosomal subunit <br /> biogenesis function remains unclear. <br />    In work examined role extension S12 in <br /> SSU assembly function vitro vivo. We created <br /> multiple truncations extension S12 starting                      FIGURE 2. Analysis effect truncation S12&#226;&#8364;&#8482;s extension in <br />                                                                                 vitro. Truncation extension 10 amino acids affects <br /> the N-terminus (Figs. 1C, 4A, below): truncation the <br />                                                                                 function 30S particles vitro. Activity particles measured by <br /> two-turn helix (S12&#226;&#8211;&#181;6N), truncation past turn helix                     ability bind labeled tRNA. Results normalized tRNA bind- <br /> (S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;11N), truncation half extension                            ing 30S subunits reconstituted S12WT. <br />  <br />  <br /> 322       RNA, Vol. 20, No. 3 <br />                                                                                        N-terminus S12 influences SSU assembly E. coli <br />  <br />  <br />  <br />                                                                              achieve end, generated series N-terminal trunca- <br />                                                                              tions S12. Four different C-terminal His-tagged constructs <br />                                                                              overexpression purification (S12Full <br />                                                                              Length, S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;15N, S12&#226;&#8211;&#181;27N) (Fig. 1C). The <br />                                                                              C-terminal His-tag included aid purification of <br />                                                                              mutant proteins possible contamination en- <br />                                                                              dogenous wild-type S12. Wild-type untagged (S12WT) S12 <br />                                                                              used control monitor C-terminal <br />                                                                              His-tag altered S12 function. The overexpressed, purified <br />                                                                              S12 variants assembled 16S rRNA pres- <br />                                                                              ence different r-proteins examine role S12 as- <br />                                                                              sembly function 30S subunit vitro. <br />                                                                                 Functional, active SSUs bind tRNAs message-de- <br />                                                                              pendent manner vitro making ideal assay as- <br />                                                                              sess activity SSUs reconstituted S12 variants <br />                                                                              (Fig. 2). In assay, labeled tRNA supplied excess, <br />                                                                              tRNA-binding site SSU sat- <br />                                                                              urated. S12WT, S12Full Length, truncated proteins <br />  <br />  <br />  <br />  <br /> FIGURE 3. Primer extension analysis RNPs containing S12 variants. <br /> Sequencing gel 16S rRNA following reactions: untreated 16S <br /> rRNA (lane 1); dimethyl sulfate treated 16S rRNA (lanes 2,10); dimethyl <br /> sulfate-treated minimal complex (S4, S7, S8, S15, S16, S17, S20, 16S <br /> rRNA) (lanes 3,9); dimethyl sulfate-treated minimal complex with <br /> S12WT (lane 4), S12 full-length (lane 5), S12&#226;&#8211;&#181;10N (lane 6), S12&#226;&#8211;&#181;15N <br /> (lane 7), S12&#226;&#8211;&#181;27N (lane 8). A G sequencing lanes. Bars in- <br /> dicate changes methylation patterns compared complexes lack- <br /> ing S12 (lanes 3,9). The primers utilized A, B, C 232, 683, and <br /> 1046, respectively. <br />  <br />  <br />  <br /> assembly E. coli SSUs indicates r-protein exten-                 FIGURE 4. In vivo analysis strains bearing truncations S12. (A) <br /> sions greatly alter rRNA maturation architecture.                    Three-dimensional structure S12 30S subunit, <br />                                                                              positions truncations color coded Figure 2, except <br />                                                                              S12&#226;&#8211;&#181;6N green. (B) Serial dilution analysis growth of <br /> RESULTS                                                                      &#226;&#8211;&#181;rpsl strains dependent WT truncated proteins 37&#194;&#176;C <br />                                                                              25&#194;&#176;C. Overnight cultures diluted equal OD600 then <br />                                                                              serially diluted 10-fold increments. These spotted LB sup- <br /> The N-terminal extension S12 plays role SSU                          plemented kanamycin (50 &#194;&#181;g/mL), ampicillin (100 &#194;&#181;g/mL), and <br /> tRNA binding vitro                                                        arabinose (0.2%). (C) Growth &#226;&#8211;&#181;rpsl strains liquid media mea- <br />                                                                              sured Bioscreen automatic growth analyzer (Growth Curves) and <br /> We wished determine extension S12 mod-                     plotted semi-log plot. Doubling times presented tabular <br /> ulates SSU formation, function, 16S rRNA folding. To                 form. <br />  <br />  <br />                                                                                                                    www.rnajournal.org         323 <br />  Calidas et al. <br />  <br />  <br />  <br />                                                                                                  RNPs and/or improper assembly <br />                                                                                                  folding particles extension <br />                                                                                                  reduced length half more. <br />  <br />                                                                                                  N-terminal truncation S12 results <br />                                                                                                  changes 16S rRNA conformation <br />                                                                                                     Given differences function 30S sub- <br />                                                                                                     units different S12 constructs, we <br />                                                                                                     examined effect truncation <br />                                                                                                     S12 architecture 16S rRNA <br />                                                                                                     using chemical probing primer ex- <br />                                                                                                     tension. Differences 16S rRNA reactiv- <br />                                                                                                     ity observed S12 added in <br />                                                                                                     vitro complexes containing 16S rRNA <br />                                                                                                     early binding r-proteins S4, S7, <br />                                                                                                     S8, S15, S16 S17, S20 (Stern et al. <br />                                                                                                     1988b); however, addition later <br />                                                                                                     binding r-proteins complexes, <br />                                                                                                     changes attributed S12 lon- <br />                                                                                                     ger observed. Thus, probing min- <br />                                                                                                     imal complexes (16S rRNA S4, S7, S8, <br />                                                                                                     S15, S16 S17, S20) &#194;&#177;S12 allows struc- <br />                                                                                                     tural changes associated S12 binding <br />                                                                                                     examined. RNPs assembled <br />                                                                                                     minimal complex different var- <br />                                                                                                     iants S12 (WT, Full-Length, &#226;&#8211;&#181;10N, <br />                                                                                                     &#226;&#8211;&#181;15N, &#226;&#8211;&#181;27N), subsequently probed <br />                                                                                                     dimethyl sulfate, 16S rRNA <br />                                                                                                     analyzed primer extension. <br />                                                                                                        Addition S12WT minimal <br />                                                                                                     complex (Fig. 3A, cf. lanes 3,9 lane <br />                                                                                                     4) results enhancement reac- <br />                                                                                                     tivity A26, proximal 5&#226;&#8364;&#178; terminus <br /> FIGURE 5. In vivo analysis ribosomes formed strains bearing truncations S12. (A) 16S rRNA previously observed <br /> Representative sucrose gradient sedimentation profiles ribosomal subunits &#226;&#8211;&#181;rpsl strains (Stern et al. 1988b). Similar levels en- <br /> bearing pS12WT pS12&#206;&#8221;XN. The 10%&#226;&#8364;&#8220;40% gradients pumped hancement observed addition <br /> using Biocomp Piston Gradient Fractionator elution small large subunits fol- <br /> lowed 70S ribosomes detected monitoring OD254. Area curve calcu- S12Full-Length, S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;15N, <br /> lated cross-hatched (free SSU) gray-shaded regions (70S) profile. (B) S12&#226;&#8211;&#181;27N (Fig. 3A, lanes 5,6,7,8, re- <br /> Comparison strains percent area occupied free subunits (cross-hatched) sub- spectively). This suggests truncation <br /> units ribosomes (gray) represented bar graph. (C) Agarose gel analysis RNA con- N-terminal region does cause <br /> tent free SSUs ribosomes. <br />                                                                                                     global misfolding entire popula- <br />                                                                                                     tion protein molecules complete <br /> (S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;15N, S12&#226;&#8211;&#181;27N) reconstituted                            inability variants associate 16S rRNA. In <br /> with remaining SSU r-proteins 16S rRNA (S12WT/                        contrast, central domain, residues A560 and <br /> 30S, S12Full Length/30S, S12&#226;&#8211;&#181;10N/30S, S12&#226;&#8211;&#181;15N/30S,                            A563 (Fig. 3B), protection observed addition of <br /> S12&#226;&#8211;&#181;27N/30S). S12WT/30S S12Full Length/30S                           S12WT (Fig. 3B, lane 4) S12Full-Length (Fig. 3B, lane <br /> comparable binding activity, demonstrating C-termi-                  5), reduced truncation protein 10 amino ac- <br /> nal tag does affect S12 activity incorporation              ids (Fig. 3B, lane 6) absent truncation <br /> resulting SSU. Complexes assembled S12&#226;&#8211;&#181;10N bind                     (Fig. 3B, lanes 7,8). Indeed, presence S12&#226;&#8211;&#181;15N <br /> tRNA comparable S12WT/30S S12Full Length/30S.                          (Fig. 3B, lane 7) S12&#226;&#8211;&#181;27N (Fig. 3B, lane 8) <span id='am-241' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> On hand, S12&#226;&#8211;&#181;15N/30S S12&#226;&#8211;&#181;27N/30S                         appears revert seen absence S12 <br /> deficient binding tRNA RNPs reconstituted ab-                    (Fig. 3B, lanes 3,9; Stern et al. 1988b). Analysis contacts be- <br /> sence S12 (Protein mix-S12). This change functional ca-                 tween S12 16S rRNA crystal structure 30S <br /> pacity result reduced incorporation shorter S12                 subunit indicates loss protection nucleotide <br />  <br />  <br /> 324      RNA, Vol. 20, No. 3 <br />                                                                                      N-terminus S12 influences SSU assembly E. coli <br />  <br />  <br />  <br /> 563 likely deletion residue 11 protein            N-terminal extension nucleotide A563. The <br /> (Fig. 6D&#226;&#8364;&#8220;F).                                                               changes reactivity nucleotides A560 A914 more <br />    Addition S12WT (Fig. 3C, lane 4) minimal com-                 likely long-range structural changes associated with <br /> plex (Fig. 3C, lanes 3,9) enhances reactivity A914,             absence N-terminus S12. These findings con- <br /> this change observed lesser extent incorporation              sistent N-terminal extension S12 playing role in <br /> of S12Full-Length (Fig. 3C, lane 5) S12&#226;&#8211;&#181;10N (Fig.                 shaping architecture assembling SSUs orches- <br /> 3C, lane 6). Truncation extension 15 amino acids                 trating changes proximal distant structural <br /> (Fig. 3C, lane 7) (Fig. 3C, lane 8) abolishes                 elements. <br /> change reactivity, indicating conformation of <br /> 16S rRNA similar seen absence S12. <br />                                                                            The N-terminal extension S12 plays role vivo <br />    Our results indicate truncation N-terminus of <br /> S12 10 amino acids does abolish contact be-                     The gene encoding S12 (rpsL) essential E. coli (Sharma <br /> tween globular region S12 16S rRNA. However,                      et al. 2007; Shoji et al. 2011). Therefore, analyze effect <br /> change observed correlates loss direct contact               changes N-terminus S12 used genomic dele- <br />                                                                                                      tion rpsL (&#206;&#8221;rpsL pSTR) growth is <br />                                                                                                      supported expression S12 a <br />                                                                                                      covering plasmid (pSTR) (Link et al. <br />                                                                                                      1997; Sharma et al. 2007). Plasmid shuffle <br />                                                                                                      negative selection used in- <br />                                                                                                      troduce plasmids expressing S12WT, <br />                                                                                                      S12&#226;&#8211;&#181;6N, S12&#226;&#8211;&#181;11N, S12&#226;&#8211;&#181;15N, and <br />                                                                                                      S12&#226;&#8211;&#181;24N (Fig. 4A) control of <br />                                                                                                      inducible promoter (Shoji et al. <br />                                                                                                      2011). We generated strains using <br />                                                                                                      technique: &#226;&#8211;&#181;rpsL pS12WT, &#226;&#8211;&#181;rpsL <br />                                                                                                      pS12&#226;&#8211;&#181;6N, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N, &#226;&#8211;&#181;rpsL <br />                                                                                                      pS12&#226;&#8211;&#181;15N. To aid plasmid shuffle, a <br />                                                                                                      specific plasmid expression the <br />                                                                                                      truncated S12 proteins used that <br />                                                                                                      simultaneously allowed negative selection <br />                                                                                                      full-length rpsL gene S12 mu- <br />                                                                                                      tant expression (Link et al. 1997). As a <br />                                                                                                      consequence, set truncation mu- <br />                                                                                                      tants similar possible, not <br />                                                                                                      identical, S12 truncations studied <br />                                                                                                      vitro. Despite multiple efforts and <br />                                                                                                      reoptimized conditions, unable <br />                                                                                                      generate &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;24N, leading us <br />                                                                                                      speculate presence some <br />                                                                                                      amino acids residues 15 and <br />                                                                                                      24 S12 necessary cellular <br />                                                                                                      viability. <br />                                                                                                         The growth strains dependent on <br />                                                                                                      wild-type truncated forms S12 <br />                                                                                                      examined plates (Fig. 4B) in <br />                                                                                                      liquid media (Fig. 4C). In condi- <br />                                                                                                      tions, clear truncation S12 by <br />                                                                                                      amino acids N-termi- <br />                                                                                                      nus deleterious cell growth at <br /> FIGURE 6. Modification data Figure 3 mapped spheres three-dimensional struc- <br /> ture 30S ribosomal subunit. 16S rRNA gray, various S12 variants color coded optimum growth temperature (37&#194;&#176; <br /> Figure 2, r-proteins omitted clarity. (A) S12WT 16S rRNA 30S sub- C); truncation three-fourths ex- <br /> <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. (B&#226;&#8364;&#8220;F) Data complexes containing S12WT, S12Full-Length, S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;15N, tension S12 (S12&#226;&#8211;&#181;24N) does ap- <br /> S12&#226;&#8211;&#181;27N plotted spheres 16S rRNA. Black spheres indicate nucleotides protected ad- <br />                                                                                                      pear support growth. Also, strains <br /> dition S12WT, green spheres indicate positions enhancement reactivity. Strength of <br /> modification represented size sphere. The view rotated 180&#194;&#176; zoomed dependent truncated forms S12 <br /> greater clarity.                                                                                     (&#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6N, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N, and <br />  <br />  <br />                                                                                                                  www.rnajournal.org         325 <br />  Calidas et al. <br />  <br />  <br />  <br /> &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N) exhibit cold sensitivity (Fig. 4B), hall-        cally structured segment, vitro vivo (Figs. 1C, <br /> mark ribosome biogenesis defects.                               4A). Changes tRNA binding architecture 16S <br />    This consistent data initial plasmid shuffle   rRNA observed vitro length N-termi- <br /> to generate strains. The success rate shuffle      nal extension S12 reduced half more. In vivo <br /> defined percent colonies observed shuffling        truncation N-terminus S12 change growth at <br /> plasmids expressing S12 variants (pS12WT, pS12&#226;&#8211;&#181;6N,                 normal cold temperatures, ratio free SSUs 70S <br /> pS12&#226;&#8211;&#181;11N, pS12&#226;&#8211;&#181;15N, pS12&#226;&#8211;&#181;24N) phenotyp-              ribosomes, 16S rRNA maturation. These data suggest a <br /> ically consistent loss plasmid (pSTR). We    critical role N-terminal extension S12 SSU biogen- <br /> had average success rate (out experiments)           esis function. <br /> 100% &#226;&#8211;&#181;rpsL pS12WT, 86% &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6N, 83%                     In vitro analyses offer initial information role the <br /> for &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N, 36% &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N, 0%                 N-terminal segment S12 30S subunit function struc- <br /> for &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;24N (data shown).                               turing SSU platform. Truncation extension half <br />    At 37&#194;&#176;C, &#226;&#8211;&#181;rpsL pS12WT &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6 exhibit similar          (S12&#226;&#8211;&#181;15N S12&#226;&#8211;&#181;27N) caused marked reduction <br /> growth properties (Fig. 4B,C) consistent vitro analysis    ability RNPs bind tRNA reduction resulted <br /> indicating deletion 10 amino acids N-         binding data comparable particles lacking S12 (Fig. 2). <br /> terminus does affect assembly significantly. &#226;&#8211;&#181;rpsL             However, unlikely failure truncated <br /> pS12&#226;&#8211;&#181;11N &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N demonstrate increased dou-             S12 incorporate SSUs probing data reveal reactivity <br /> bling lag times liquid culture (Fig. 4C). Expression     change 16S rRNA residues correlated S12WT <br /> S12&#226;&#8211;&#181;6N, S12&#226;&#8211;&#181;11N, S12&#226;&#8211;&#181;15N, S12&#226;&#8211;&#181;24N wild-type               association (Figs. 3A, 6). Moreover, change at- <br /> background significant effect growth (data           tributed globular region, indicating absence the <br /> shown). Therefore, mutant proteins likely domi-        extension does cause global unfolding S12 that <br /> nant, stability expression variant     deletions assemble RNPs. However, did ob- <br /> unknown. In conclusion, truncation half ex-      serve changes reactivity regions 16S rRNA asso- <br /> tension deleterious effects growth cell, espe-   ciated systematic truncation S12. For example, loss <br /> cially low temperatures, suggesting role N-terminal   protection A563 directly attributed truncating <br /> extension S12 SSU biogenesis.                                S12 past amino acid 11 (Figs. 3B, 6E), Arginine 11 interacts <br />                                                                    A563 SSUs (see Fig. 6D). Our previous work suggested <br />                                                                    alternative conformations 16S rRNA existed nucle- <br /> Truncation N-terminus S12 alters ribosome <br />                                                                    otides 566&#226;&#8364;&#8220;568 minimal complex lacking 5&#226;&#8364;&#178; domain <br /> profiles <br />                                                                    binding proteins N-terminal segment S12 <br /> One explanation growth defects observed &#226;&#8211;&#181;rpsL           played role assembly interdomain region between <br /> pS12&#226;&#8211;&#181;11N &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N lethal pheno-           SSU body platform (Calidas Culver 2011). In ad- <br /> type conferred &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;24N improper SSU assem-            dition direct effect, deletion N-terminus <br /> bly. Consequently, examined sucrose gradient                S12, differences reactivity observed are <br /> sedimentation profiles free subunits ribosomes       distal S12 SSUs (see Fig. 6). Nucleotides 560 914 ex- <br /> strains (&#226;&#8211;&#181;rpsL pS12WT, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6N, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N,              hibit differential reactivity presence S12&#226;&#8211;&#181;15N and <br /> and &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N) 37&#194;&#176;C (Fig. 5A). There signifi-        S12&#226;&#8211;&#181;27N (Figs. 3B,C, 6E,F) compared S12Full <br /> cant change percent area free SSUs comparing        Length (Figs. 3C, lane 5, 6C) S12&#226;&#8211;&#181;10N (Figs. 3C, lane 6, <br /> &#226;&#8211;&#181;rpsL pS12WT &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6N (P = 0.34) (Fig. 5A,B).             6D). Changes 16S rRNA architecture associated trun- <br /> There significant increase free SSUs &#226;&#8211;&#181;rpsL              cation S12 appear attributable direct long- <br /> pS12&#226;&#8211;&#181;11N (P = 0.03) &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N (P = 0.013)                 range effects. Since main aim publication deter- <br /> strains compared wild-type S12 control strain (Fig.       role extension S12 stabilizing 16S rRNA <br /> 5A,B). Thus, SSUs appear somewhat defective            architecture, work carried minimal par- <br /> in subunit association, strongly suggesting defect initia-    ticles. It possible changes 16S rRNA architecture <br /> tion and/or biogenesis. Consistent hypothesis,     affect r-proteins binding regions particle; <br /> we observe presence precursor 16S rRNA (p16S rRNA)           however, striking reactivity differences ob- <br /> free SSUs various rpsL truncation strains (Fig. 5C). To       served regions directly associated translational fidelity. <br /> summarize, strong correlation changes           This consistent vast majority fidelity mutations <br /> in growth, changes ratio free SSUs ribosomes, pres-       mapping globular region S12 (Gorini Kataja <br /> ence p16S rRNA, increasing truncation S12.               1964a; Sharma et al. 2007; Agarwal et al. 2011). Our probing <br />                                                                    experiments indicate S12 protein variants bind to <br />                                                                    16S rRNA (at minimally), changes consistent <br /> DISCUSSION <br />                                                                    previous structural (Schuwirth et al. 2005), probing <br /> We created partially overlapping subsets N-terminal trun-       (Stern et al. 1988b), assembly (Powers Noller 1995; <br /> cation mutants S12 analyze role noncanoni-        Talkington et al. 2005; Adilakshmi et al. 2008) data. <br />  <br />  <br /> 326      RNA, Vol. 20, No. 3 <br />                                                                              N-terminus S12 influences SSU assembly E. coli <br />  <br />  <br />  <br />    After plasmid shuffle, cloned truncations         ing S12&#226;&#8211;&#181;15N; protection nucleotide 563, direct <br /> were sole source S12 vivo, major changes     contact R11 X-ray crystal structure lost. Thus, <br /> be observed. First, growth defects observed test-    observed importance residue SSU function <br /> ed strains expressing truncated S12 (Fig. 4B,C). The changes       biogenesis consistent data. <br /> were exacerbated low temperature decreasing               These observations suggest model role non- <br /> length N-terminal segment, deletion N-         canonically structured region S12 SSU biogenesis. <br /> terminal 15 amino acids affected growth deletion         Deletion extension results structural perturbations <br /> of 11 amino acids, grew slowly variant         platform 16S rRNA vitro, suggesting ex- <br /> lacking residues. Moreover, unable obtain col-      tension important architecture region of <br /> onies expressing deletions 24 amino acids. Thus, N-         assembled SSU (Figs. 3, 6). Previous vitro studies sug- <br /> terminal extension S12 likely contributes essential      gest stabilization 16S rRNA architecture plat- <br /> role S12. In contrast, significant changes         form occurs late SSU assembly (Talkington et al. 2005; <br /> growth presence antibiotics alter decoding          Adilakshmi et al. 2008), work supports role <br /> (data shown), implying N-terminus S12 plays        S12 N-terminal extension scaffolding 16S rRNA <br /> a predominant role SSU assembly. Second, observe        structure. <br /> upon truncation half extension vivo, sig-        Roles extended loops unstructured regions r- <br /> nificant increase population free SSUs (Fig. 5A,B).      proteins subject speculation, little is <br /> This phenotype similar observed <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span>           known function assembly translation (see <br /> of C-terminal extension yeast S14, results         Ban et al. 2000; Wimberly et al. 2000; Wilson Nierhaus <br /> an accumulation 20S pre rRNA 43S pre-ribosomes               2005; Timsit et al. 2009). One hypothesis r-proteins <br /> (Jakovljevic et al. 2004). Finally, observed increase     long extensions represent subset intrinsically disor- <br /> p16S rRNA strains bearing truncated variants S12. In         dered proteins function depends structural <br /> E. coli, increase free SSUs associated p16S rRNA        transition disordered form ordered structure <br /> has strains perturbed ribosome biogenesis       binding partner (Uversky et al. 2000; <br /> cascades era-depleted cells, &#206;&#8221;yjeQ &#206;&#8221;ksgA (Inoue        Iakoucheva et al. 2002; Timsit et al. 2009). For example, <br /> et al. 2003; Campbell Brown 2008; Connolly et al. 2008).       N-terminal extension r-protein L20 undergoes he- <br /> Changes growth aberrant subunit/ribosome ratios          lix&#226;&#8364;&#8220;coil transition binding 23S rRNA, essential <br /> well increased p16S rRNA content strongly implies role        large subunit assembly (Guillier et al. 2005; Timsit et al. <br /> for extension S12 SSU biogenesis.                        2006). Similar data S4 vitro assembly the <br />    While ideal directly compare          30S subunit lend credence idea r-proteins <br /> same truncated S12 proteins vitro vivo,         long noncanonically structured regions play impor- <br /> complicated requirement plasmid allowing ex-       tant role assembly cofolding 16S rRNA (Mayerle <br /> pression truncated rpsL gene encode gene        Woodson 2013). Our study suggests N-terminal <br /> allows selection plasmid expressing wild-          extension S12 plays role stabilizing SSU structure <br /> type full-length rpsL. This compromise allowed            biogenesis. <br /> similar comparison ensuring possible wild-type <br /> S12 expression minimized. While vivo vitro <br /> data general agreement, difference phe-   MATERIALS AND METHODS <br /> notype observed vivo &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N strain and <br /> the vitro data S12&#226;&#8211;&#181;10N. While vitro-formed par-         Plasmids <br /> ticles incorporating S12&#226;&#8211;&#181;10N tRNA-binding activity            Primers introducing translation initiation codon desired sites <br /> and structures similar containing S12WT,              rpsL utilized PCR amplification MG1655 geno- <br /> &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N strain exhibits growth defects aber-         mic DNA generate full-length truncation variants S12. The <br /> rant subunit profile. The differences vivo SSU          truncations designated S12&#226;&#8211;&#181;XN, X signifies num- <br /> biogenesis vitro reconstitution SSUs minimal          ber truncated residues, N indicates protein been <br /> 16S rRNA-containing particles account ob-            truncated N-terminus. C-terminal His-tagged versions S12 <br /> served differences. Notably, increase pre-16S       expressed purified use vitro. Full-length truncat- <br /> rRNA &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N strain, suggesting el-        ed variants (&#226;&#8211;&#181;10N, &#226;&#8211;&#181;15N, &#226;&#8211;&#181;27N) S12 cloned the <br />                                                                    NdeI HindIII sites pET24 (Novagen, kanamycin resistant) <br /> ements vivo biogenesis cascade influenced by <br />                                                                    stop codon C-terminus gene. S12WT, which <br /> this truncation involvement evident <br />                                                                    full-length untagged protein cloned pET21b gift from <br /> in vitro experiments. Additionally, 11th residue        laboratory Dr. Noller. <br /> of S12 arginine, mutation residue previ-        The str operon including genes rpsL, rpsG, fusA, tufA was <br /> ously implicated changes SSU function vivo           cloned NotI site pKOV generate pSTR. pKOV express- <br /> (Agarwal et al. 2011). In fact, consistent       es sacB, gene causes loss host plasmid presence <br /> data vitro structural analysis particles incorporat-    sucrose (Link et al. 1997). Primers introducing translation <br />  <br />  <br />                                                                                                           www.rnajournal.org         327 <br />  Calidas et al. <br />  <br />  <br />  <br /> initiation codon desired sites rpsL utilized PCR    RNPs, reactions incubated 42&#194;&#176;C 20 min. These <br /> amplification MG1655 genomic DNA generate full-length          cooled ice 10 min proceeding next <br /> and truncation variants S12. The wild-type (S12WT) truncat-     experiment. <br /> ed (S12&#226;&#8211;&#181;6N, S12&#226;&#8211;&#181;11N, S12&#226;&#8211;&#181;15N, S12&#226;&#8211;&#181;24N) rpsl genes were <br /> cloned NcoI HindIII sites pBAD HisB (Invitrogen, <br /> ampicillin resistant) C-terminal stop codon, express un-     In vitro transfer RNA (tRNA) binding <br /> tagged protein vivo. The protein truncated different set <br />                                                                        A poly-U-dependent tRNA-binding experiment performed es- <br /> of residues constraints restriction enzyme choice. <br />                                                                        sentially described (Culver Noller 1999) saturating ex- <br />                                                                        cess tRNA assay monitored factor-independent, <br /> Purification proteins                                               multi-site tRNA-binding activity SSU. The assay carried <br />                                                                        presence 30 mM urea. The results presented are <br /> Strains expressing S12WT pET21b S12Full Length,               independent experiments. <br /> S12&#226;&#8211;&#181;10N, S12&#226;&#8211;&#181;15N, S12&#226;&#8211;&#181;27N pET24 grown to <br /> OD600 = 0.7, induced 1 mM IPTG 3 h. The cells <br /> were pelleted 7000 rpm 10 min, lysed Buffer E (20       Chemical probing RNP conformation <br /> mM K+-HEPES pH 7.6, 20 mM KCl, 6 mM &#206;&#178;ME) an <br />                                                                        Chemical probing minimal RNPs dimethyl sulfate car- <br /> Avestin Emulsiflex-C3 cell lyser. The lysate cleared spinning <br />                                                                        ried essentially described (Moazed et al. 1986). The complexes <br /> at 10,000 rpm JA-17 rotor Beckman high-speed centrifuge <br />                                                                        assembled, chilled ice 10 min, treated with <br /> for 5 min 4&#194;&#176;C. <br />                                                                        dimethyl sulfate final concentration 33.8 mM incubated <br />    S12WT supernatant dialyzed overnight Buffer B (20 mM <br />                                                                        1 h ice. The reactions visualized primer extension as <br /> sodium acetate pH 5.6, 20 mM KCl, 6 M urea, 6 mM &#206;&#178;ME). <br />                                                                        described (Moazed et al. 1986) using primers initiating cDNA syn- <br /> Proteins purified fast protein liquid chromatography using a <br />                                                                        thesis nucleotides 232, 683, 1046. Results representative of <br /> Resource-S cation exchange column (GE) eluted 35-mL <br />                                                                        independent experiments. <br /> salt gradient 100 350 mM KCl Buffer C (20 mM sodium <br /> acetate pH 5.6, 1 M KCl, 6 M urea, 6 mM &#206;&#178;ME). The proteins <br /> were eluted &#226;&#710;&#188;230 mM KCl. Purified proteins dialyzed       Generation E. coli strains <br /> Buffer D (80 mM K+ -HEPES pH 7.6, 1 M KCl, 6 mM <br /> &#206;&#178;ME), aliquoted, frozen liquid nitrogen.                        A strain lacking genomic copy S12 (S12koBW) created by <br />    S12&#226;&#8211;&#181;XN S12Full Length pellets resuspended Buffer        Sharma et al. (2007). It used donor strain P1 transduction <br /> BH (100 mM sodium dihydrogen phosphate, 330 mM KCl, 6 M                experiments replace rpsL gene strain MG1655str a <br /> urea, 20 mM Imidazole, 6 mM &#206;&#178;ME pH 7.4) rocking pellet       kanamycin resistance marker, generating &#226;&#8211;&#181;rpsl pSTR. &#226;&#8211;&#181;rpsl pSTR is <br /> overnight 4&#194;&#176;C. The solution cleared 7000 rpm 20 min,     temperature sucrose sensitive. It transformed pBAD <br /> twice, loading sample affinity purification HiTrap     expressing WT &#226;&#8211;&#181;X rpsl transformants grown overnight <br /> Chelating HP column (GE). The column washed Buffer            5 mL LB. The overnight culture plated LB agar <br /> CH (100 mM sodium dihydrogen phosphate, 330 mM KCl, 6 M                plates containing kanamycin (50 &#194;&#181;g/mL), ampicillin (100 &#194;&#181;g/mL), <br /> urea, 50 mM imidazole, 6 mM &#206;&#178;ME pH 7.4). The proteins          arabinose (2%), sucrose (5%) replace pSTR pS12, gen- <br /> eluted gradient 125 mM 250 mM Imidazole 20          erating &#226;&#8211;&#181;rpsL pS12WT, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;6N, &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;11N, and <br /> mL Buffer EH (100 mM sodium dihydrogen phosphate, 330 mM               &#226;&#8211;&#181;rpsL pS12&#226;&#8211;&#181;15N. In order avoid picking suppressors rever- <br /> KCl, 6 M Urea, 250 mM imidazole, 6 mM &#206;&#178;ME pH 7.4). The pro-         tants, slow growing (&#226;&#8240;&#710;48 h) colonies expressing truncated forms <br /> teins eluted &#226;&#710;&#188;163 mM imidazole concentration. The proteins     S12 chosen analysis. The presence compensa- <br /> step dialyzed gradations 0.5 M urea 50 mM imidazole        tory mutations strains dependent truncated proteins can- <br /> into Buffer DH (80 mM K+-HEPES pH 7.6, 1 M KCl, 3 M urea,           ruled out, slow growth phenotype leads believe <br /> and 6 mM &#206;&#178;ME), aliquoted, stored liquid nitrogen.               unlikely. <br />  <br />  <br /> Assembly ribonucleoprotein particles vitro                       Analysis growth properties <br /> 16S rRNA isolated described (Moazed et al. 1986), heat acti-    All overnight cultures grown 37&#194;&#176;C presence kana- <br /> vated incubating 42&#194;&#176;C 15 min Buffer A (80 mM K+-          mycin (50 &#194;&#181;g/mL), ampicillin (100 &#194;&#181;g/mL), arabinose (0.2%) <br /> Hepes pH 7.6, 330 mM KCl, 20 mM MgCl2, 0.01%                    shaking (200 rpm). For growth solid media, overnight cul- <br /> Nikkol), incubated ice 10 min. Ribonucleoprotein       tures diluted OD600 = 0.6, serially diluted factor of <br /> particles reconstituted essentially described (Culver      10. The dilutions spotted LB-agar plates containing kana- <br /> Noller 2000) reconstitution carried     mycin (50 &#194;&#181;g/mL), ampicillin (100 &#194;&#181;g/mL), arabinose (0.2%) <br /> presence 30 mM urea. 30S particles formed mixing 280        plates incubated 37&#194;&#176;C 25&#194;&#176;C examine growth op- <br /> pmol variant S12 (wild-type, full-length, &#226;&#8211;&#181;10, &#226;&#8211;&#181;15,       timal nonpermissive temperatures. <br /> &#226;&#8211;&#181;27) 240 pmol total r-protein mix (all r-proteins          Growth curves 37&#194;&#176;C obtained methods. In one <br /> S12) 40 pmol 16S rRNA final volume 100 &#194;&#181;L        method overnight cultures diluted equal OD600 50 mL <br /> final KCl concentration 330 mM. In minimal particles, 240           LB media OD600 determined aliquots taken at <br /> pmol minimal r-protein mix (S4, S7, S8, S15, S16, S17,          30-min intervals. In second technique, overnight cultures <br /> S20) substituted total r-protein mix. In order form     strain diluted equal OD600 200 &#194;&#181;L media a <br />  <br />  <br /> 328      RNA, Vol. 20, No. 3 <br />                                                                                       N-terminus S12 influences SSU assembly E. coli <br />  <br />  <br />  <br /> microtiter plate. This replicated wells overnight     Calidas D, Culver GM. 2011. Interdependencies govern multidomain <br /> culture. The plates grown Bioscreen automatic growth an-            architecture ribosomal small subunit assembly. RNA 17: 263&#226;&#8364;&#8220;277. <br /> alyzer shaking periods 20 sec maximum intensity          Campbell TL, Brown ED. 2008. Genetic interaction screens or- <br />                                                                               dered overexpression deletion clone sets implicate the <br /> minute. Growth monitored wideband setting 20 <br />                                                                               Escherichia coli GTPase YjeQ late ribosome biogenesis. J <br /> min (Weichart Kell 2001). The data collected sin-           Bacteriol 190: 2537&#226;&#8364;&#8220;2545. <br /> gle run plate 37&#194;&#176;C. Growth microtiter plates yielded longer     Clemons WM Jr, May JL, Wimberly BT, McCutcheon JP, Capel MS, <br /> doubling times growth flasks strain. However,          Ramakrishnan V. 1999. Structure bacterial 30S ribosomal sub- <br /> was significant difference ratio doubling time mu-        <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> 5.5 &#195;&#8230; resolution. Nature 400: 833&#226;&#8364;&#8220;840. <br /> tants wild-type strains, data independent      Connolly K, Rife JP, Culver G. 2008. Mechanistic insight ribo- <br />                                                                               biogenesis functions ancient protein KsgA. Mol <br /> growths Bioscreen analyzer presented. <br />                                                                               Microbiol 70: 1062&#226;&#8364;&#8220;1075. <br />                                                                            Culver GM, Noller HF. 1999. Efficient reconstitution functional <br />                                                                               Escherichia coli 30S ribosomal subunits complete set re- <br /> Analysis ribosome sedimentation profiles                                   combinant small subunit ribosomal proteins. RNA 5: 832&#226;&#8364;&#8220;843. <br />                                                                            Culver GM, Noller HF. 2000. In vitro reconstitution 30S ribosomal <br /> All strains inoculated equal OD600 500 mL LB. All           subunits using complete set recombinant proteins. Methods <br /> strains grown OD600 0.6 maximum 8 h. The            Enzymol 318: 446&#226;&#8364;&#8220;460. <br /> cells treated described (Connolly et al. 2008) obtain sub-      Gorini L, Kataja E. 1964a. Phenotypic repair streptomycin defec- <br /> <span id='am-242' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> profiles. The data derived independent growths.            tive genotypes E. coli. Proc Natl Acad Sci 51: 487&#226;&#8364;&#8220;493. <br /> The area peak profile calculated using          Gorini L, Kataja E. 1964b. Streptomycin-induced oversuppression E. <br /> program QTiPlot. Ribosome containing peak intensity correct-              coli. Proc Natl Acad Sci 51: 995&#226;&#8364;&#8220;1001. <br />                                                                            Guillier M, Allemand F, Graffe M, Raibaud S, Dardel F, Springer M, <br /> ed represent SSUs Connolly et al. (2008). Data rep-         Chiaruttini C. 2005. The N-terminal extension Escherichia coli ri- <br /> resentative independent experiments.                                 bosomal protein L20 important ribosome assembly, dis- <br />                                                                               pensable translational feedback control. RNA 11: 728&#226;&#8364;&#8220;738. <br />                                                                            Iakoucheva LM, Brown CJ, Lawson JD, Obradovic Z, Dunker AK. 2002. <br /> Analysis rRNA SSUs ribosomes                                      Intrinsic disorder cell-signaling cancer-associated proteins. <br />                                                                               J Mol Biol 323: 573&#226;&#8364;&#8220;584. <br /> SSUs ribosomes collected sucrose gradient centrifu-         Inoue K, Alsina J, Chen J, Inouye M. 2003. Suppression defective <br /> gation cell lysate. RNA extracted described Culver           ribosome assembly rbfA deletion mutant overexpression of <br /> Noller (2000), analyzed agarose gel electrophoresis. Data              Era, essential GTPase Escherichia coli. Mol Microbiol 48: <br /> are representative independent experiments.                            1005&#226;&#8364;&#8220;1016. <br />                                                                            Jakovljevic J, Mayolo PA, Miles TD, Nguyen TM, Leger-Silvestre I, <br />                                                                               Gas N, Woolford JL Jr. 2004. The carboxy-terminal extension of <br />                                                                               yeast ribosomal protein S14 necessary maturation 43S pre- <br /> ACKNOWLEDGMENTS                                                               ribosomes. Mol Cell 14: 331&#226;&#8364;&#8220;342. <br /> The donor strain P1 transduction, S12koBW, gift Dr.         Lee K, Varma S, SantaLucia J Jr, Cunningham PR. 1997. In vivo deter- <br />                                                                               mination RNA structure-function relationships: Analysis the <br /> Rachel Green. We thank Drs. George Church pKOV plasmids, <br />                                                                               790 loop ribosomal RNA. J Mol Biol 269: 732&#226;&#8364;&#8220;743. <br /> Scott Butler David Goldfarb laboratories help            Link AJ, Phillips D, Church GM. 1997. Methods generating precise <br /> with equipment materials, Drs. Elaine Sia Martin                  deletions insertions genome wild-type Escherichia coli: <br /> Pavelka laboratories technical advice.                   Application open reading frame characterization. J Bacteriol 179: <br />                                                                               6228&#226;&#8364;&#8220;6237. <br /> Received September 11, 2013; accepted November 21, 2013.                   Lodmell JS, Dahlberg AE. 1997. A conformational switch Escherichia <br />                                                                               coli 16S ribosomal RNA decoding messenger RNA. Science <br />                                                                               277: 1262&#226;&#8364;&#8220;1267. <br />                                                                            Mayerle M, Woodson SA. 2013. Specific contacts protein S4 <br /> REFERENCES                                                                    ribosomal RNA required multiple stages ribosome as- <br /> Adilakshmi T, Bellur DL, Woodson SA. 2008. Concurrent nucleation           sembly. RNA 19: 574&#226;&#8364;&#8220;585. <br />     16S folding induced fit 30S ribosome assembly. Nature 455:      Moazed D, Noller HF. 1990. Binding tRNA ribosomal A P <br />     1268&#226;&#8364;&#8220;1272.                                                                sites protects distinct sets nucleotides 16S rRNA. J Mol Biol <br /> Agarwal D, Gregory ST, O&#226;&#8364;&#8482;Connor M. 2011. Error-prone error-re-            211: 135&#226;&#8364;&#8220;145. <br />     strictive mutations affecting ribosomal protein S12. J Mol Biol 410:   Moazed D, Stern S, Noller HF. 1986. Rapid chemical probing confor- <br />     1&#226;&#8364;&#8220;9.                                                                      mation 16S ribosomal RNA 30S ribosomal subunits using <br /> Alksne LE, Anthony RA, Liebman SW, Warner JR. 1993. An accuracy               primer extension. J Mol Biol 187: 399&#226;&#8364;&#8220;416. <br />     center ribosome conserved 2 billion years. Proc Natl       Ogle JM, Murphy FV, Tarry MJ, Ramakrishnan V. 2002. Selection of <br />     Acad Sci 90: 9538&#226;&#8364;&#8220;9541.                                                   tRNA ribosome requires transition open closed <br /> Allers J, Shamoo Y. 2001. Structure-based analysis protein-RNA in-         form. Cell 111: 721&#226;&#8364;&#8220;732. <br />     teractions using program ENTANGLE. J Mol Biol 311: 75&#226;&#8364;&#8220;86.          Powers T, Noller HF. 1991. A functional pseudoknot 16S ribosomal <br /> Ban N, Nissen P, Hansen J, Moore PB, Steitz TA. 2000. The complete            RNA. EMBO J 10: 2203&#226;&#8364;&#8220;2214. <br />     atomic structure large ribosomal subunit 2.4 &#195;&#8230; resolution.   Powers T, Noller HF. 1995. Hydroxyl radical footprinting ribosomal <br />     Science 289: 905&#226;&#8364;&#8220;920.             <br /> </body></html>