<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>et al. <br /> Koonin <br /> 2004 <br /> Volume 5, Issue 2, Article R7                                                                                                Open Access <br /> Research <br /> A comprehensive evolutionary classification proteins encoded in <br />  <br />  <br />  <br />  <br />                                                                                                                                                              <span id='am-4' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br /> complete eukaryotic genomes <br /> Eugene V Koonin*, Natalie D Fedorova*, John D Jackson*, Aviva R Jacobs*, <br /> Dmitri M Krylov*, Kira S Makarova*, Raja Mazumder*&#226;&#8364;&#160;, <br /> Sergei L Mekhedov*, Anastasia N Nikolskaya*, B Sridhar Rao*, <br /> Igor B Rogozin*, Sergei Smirnov*, Alexander V Sorokin*, <br /> Alexander V Sverdlov*, Sona Vasudevan*, Yuri I Wolf*, Jodie J Yin* and <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reviews <br /> Darren A Natale*&#226;&#8364;&#160; <br /> Addresses: *National Center Biotechnology Information, National Library Medicine, National Institutes Health, Bethesda, MD 20894, <br /> USA. &#226;&#8364;&#160;Current address: Protein Identification Resource, Georgetown University Medical Center, 3900 Reservoir Road, NW, Washington, DC <br /> 20007, USA. <br />  <br /> Correspondence: Eugene V Koonin. E-mail: koonin@ncbi.nlm.nih.gov <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reports <br /> Published: 15 January 2004                                                    Received: 23 October 2003 <br />                                                                               Revised: 1 December 2003 <br /> Genome Biology 2004, 5:R7 <br />                                                                               Accepted: 4 December 2003 <br /> The electronic version article complete be <br /> found online http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br /> &#194;&#169; 2004 Koonin et al; licensee BioMed Central Ltd. This Open Access article: verbatim copying redistribution article permitted all <br /> media purpose, provided notice preserved article&apos;s original URL. <br /> A comprehensive <br /> <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Sequencing <br /> currently <br />    help uncharacterized <br />            reconstructing <br />              genomes    ancestral <br />                    evolutionary <br />                             ofconserved <br />                                multiple, <br />                                   classification <br />                                       eukaryotic <br />                                          genes. <br />                                          taxonomically <br />                                                   ofgenomes <br />                                                      proteinsand <br />                                                           diverse <br />                                                               encoded <br />                                                                   major <br />                                                                   eukaryotes     enables <br />                                                                                    eukaryotic <br />                                                                             complete <br />                                                                         inevents        eukaryoticevolution <br />                                                                                            in-depth genomes <br />                                                                                                     comparative-genomic <br />                                                                                                             makinganalysis <br />                                                                                                                              functionalwhich <br />                                                                                                                                           predictions <br />                                                                                                                                               expected <br />                                                                                                                                                        for <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  <br />     Background: Sequencing genomes multiple, taxonomically diverse eukaryotes enables in-depth comparative-genomic <br />     analysis expected help reconstructing ancestral eukaryotic genomes major events eukaryotic evolution in <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />     making functional predictions currently uncharacterized conserved genes. <br />  <br />     Results: We examined functional evolutionary patterns recently constructed set 5,873 clusters predicted <br />     orthologs (eukaryotic orthologous groups KOGs) seven eukaryotic genomes: Caenorhabditis elegans, Drosophila <br />     melanogaster, Homo sapiens, Arabidopsis thaliana, Saccharomyces cerevisiae, Schizosaccharomyces pombe Encephalitozoon cuniculi. <br />     Conservation KOGs phyletic range eukaryotes strongly correlates functions effect of <br />     gene <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> organism&apos;s viability. The approximately 40% KOGs represented seven species are <br />     enriched proteins responsible housekeeping functions, particularly translation RNA processing. These conserved KOGs <br />     essential <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span> approximate minimal set essential eukaryotic genes. The 131 single-member, pan- <br />     eukaryotic KOGs identified examined detail. For 20 remained uncharacterized, functions predicted <br />                                                                                                                                                              interactions <br />     in-depth sequence analysis examination genomic context. Nearly proteins subunits known predicted <br />     multiprotein complexes, agreement balance hypothesis evolution gene copy number. Other KOGs variety <br />     phyletic patterns, points major contributions lineage-specific gene loss &apos;invention&apos; genes new eukaryotic <br />     evolution. Examination sets KOGs lost individual lineages reveals co-elimination functionally connected genes. <br />     Parsimonious scenarios eukaryotic genome evolution gene sets ancestral eukaryotic forms reconstructed. The <br />     gene set common ancestor crown group consists 3,413 KOGs largely includes proteins involved genome <br />     replication expression, central metabolism. Only 44% KOGs, reconstructed gene set last <br />     common ancestor crown group, detectable homologs prokaryotes; remainder apparently evolved duplication <br />     divergence invention new genes. <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br />     Conclusions: The KOG analysis reveals conserved core largely essential eukaryotic genes major diversification and <br />     innovation associated evolution eukaryotic genomes. The results provide quantitative support major trends of <br />     eukaryotic evolution noticed previously qualitative level basis detailed reconstruction evolution eukaryotic <br />     genomes biology ancestral forms. <br />  <br />  <br />                                                                Genome Biology 2004, 5:R7 <br />  R7.2 Genome Biology 2004,   Volume 5, Issue 2, Article R7   Koonin et al.                                   http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />      Background                                                               (COGs) proteins [20,23,24]. The current COG set includes <br />      Comparative analysis genomes distant species pro-                approximately 70% proteins encoded 69 genomes of <br />      vides new insights gene functions, genome evolution             prokaryotes unicellular eukaryotes [25]. The COGs have <br />      phylogeny. In particular, comparative genomics                    used functional annotation new genomes [26-29], <br />      prokaryotes revealed previously underappreciated major               target selection structural genomics [30-32], identification <br />      trends genome evolution, namely, extensive lineage-spe-               potential drug targets [33,34] genome-wide evolution- <br />      cific gene loss horizontal gene transfer (HGT) [1-7]. To             ary studies [4,13,35-38]. Sonnhammer co-workers inde- <br />      efficiently extract functional evolutionary information              pendently developed similar methodology identification <br />      multiple genomes, rational classification genes based            co-orthologous protein sets pairwise genome com- <br />      homologous relationships indispensable. The prin-              parisons applied sequenced eukaryotic genomes <br />      cipal classes homologs orthologs paralogs [8-11].             [39]. <br />      Orthologs defined homologous genes evolved via <br />      vertical descent single ancestral gene com-           A central notion introduced context COG analysis <br />      mon ancestor compared species. Paralogs homolo-               phyletic <span id='am-399' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, is, pattern representa- <br />      gous genes, which, stage evolution, evolved           tion (presence-absence) analyzed species COG <br />      duplication ancestral gene. Orthology paralogy             [13,20]. Similar concepts independently developed <br />      intimately linked because, duplication (or series              applied [40,41]. The COGs remarkable <br />      duplications) occurs speciation event separated           scatter phyletic patterns, small minority repre- <br />      compared species, orthology relationship                   sented sequenced genomes. A recent quantitative study <br />      sets paralogs, individual genes (in               showed parsimonious evolutionary scenarios most <br />      case, genes called co-orthologs).                         COGs involve multiple events gene loss HGT [7]. Both <br />                                                                               similarity complementarity phyletic patterns <br />      Correct identification orthologs paralogs central           COGs, conjunction information, con- <br />      importance functional evolutionary aspects              servation gene order, successfully employed to <br />      comparative genomics [12,13]. Orthologs typically occupy              predict gene functions [13,42,43]. The comparison phyletic <br />      functional niche different organisms; contrast,           pattern formalized set-theoretical algorithms and <br />      paralogs evolve functional diversification diverge            systematically applied computational experimen- <br />      duplication [14-16]. Therefore, robustness                  tal analysis bacterial flagellar systems, demonstrated <br />      genome annotation depends accurate identification                  considerable robustness approach [44]. <br />      orthologs. A clear demarcation orthologs paralogs is <br />      required constructing evolutionary scenarios,             We recently extended orthologous protein clus- <br />      include, vertical inheritance, lineage-specific gene          ters complex, multicellular eukaryotes [25]. Here, we <br />      loss HGT [5,7].                                                      examine phyletic patterns KOGs connection with <br />                                                                               known predicted protein functions. In-depth analysis of <br />      In principle, orthologs, including co-orthologs,               KOGs resulted prediction previously <br />      identified means phylogenetic analysis entire families          uncharacterized, apparently essential, conserved eukary- <br />      homologous proteins, expected define ortholo-             otic protein functions. We reconstruct parsimonious <br />      gous protein sets clades [17-19]. However, genome-                scenario evolution crown-group eukaryotes by <br />      wide protein sets, analysis remains extremely labor-                assigning loss genes (KOGs) emergence new <br />      intensive, error-prone well. Accordingly, procedures              genes branches phylogenetic tree explicitly <br />      developed identifying sets likely orthologs             delineate minimal gene sets various ancestral forms. <br />      explicit referral phylogenetic analysis. These pro-           To knowledge, systematic, genome-wide <br />      cedures based notion genome-specific best hit            examination sets orthologous genes eukaryotes. <br />      (BeT), is, protein target genome most <br />      similar (typically terms similarity scores computed using <br />      BLAST sequence-comparison method) given                  Results discussion <br />      protein query genome [20,21]. The assumption cen-               KOGs seven sequenced eukaryotic genomes: <br />      tral approach orthologs greater similarity        functional evolutionary implications phyletic <br />      to protein respective              patterns <br />      genomes. When multiple genomes analyzed, pairs                    Eukaryotic KOGs constructed basis com- <br />      probable orthologs detected basis BeTs com-                parison proteins encoded genomes animals <br />      bined orthologous clusters represented subset           (Homo sapiens [45], fruit fly Drosophila melanogaster <br />      analyzed genomes [20,22]. This approach, amended                  [46] nematode Caenorhabditis elegans [47]), the <br />      additional procedures detecting co-orthologous pro-             green plant Arabidopsis thaliana (thale cress) [48], fungi <br />      tein sets treating multidomain proteins, imple-              (budding yeast Saccharomyces cerevisiae [49] fission <br />      mented database Clusters Orthologous Groups                 yeast Schizosaccharomyces pombe [50]) the <br />  <br />                                                                Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                        Genome Biology 2004,                     Volume 5, Issue 2, Article R7   Koonin et al. R7.3 <br />  <br />  <br />  <br />  <br /> microsporidian Encephalitozoon cuniculi [51]. The proce- <br /> dure KOG construction modification pre- <br /> viously used COGs [20,24] described greater                               14,000 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           comment <br /> detail ([25]; Materials methods). An <br />                                                                                         12,000 <br /> important difference stems fact complex <br /> eukaryotes encode multidomain proteins than <br />  <br />  <br />  <br />  <br />                                                                        Number proteins <br />                                                                                         10,000 <br /> prokaryotes and, furthermore, orthologous eukaryotic pro- <br /> teins differ domain composition, additional                                   8,000 <br /> domains accrued complex forms [3,45]. Accordingly, <br />                                                                                             6,000 <br /> and unlike original COG construction procedure, proba- <br /> ble orthologs different domain architectures                                      4,000 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           reviews <br /> assigned KOG split shared com-                                                                                                 Hsa <br />                                                                                             2,000 <br /> mon core domains. In addition KOGs, con-                                                                                                    Dme <br />                                                                                                                                                           Cel <br /> sisted species, clusters putative orthologs                                    0                                                    Ath <br />                                                                                                                                                         Spo <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> species (TWOGs) lineage-specific expansions                                                0   1    2 <br />                                                                                                                                                       Sce <br />                                                                                                                       3                              Ecu <br /> (LSEs) paralogs analyzed genomes                                                                   4     5   6        7 <br /> identified ([25,52]; Materials methods). In                                               Number species KOGs <br /> of analyses discussed below, KOGs TWOGs are <br /> treated together, unless specified.                         Figure 1 of <br />                                                                       Assignment <br />                                                                       genomes     KOGs <br />                                                                                      proteins from <br />                                                                                           different <br />                                                                                                      ofnumbers <br />                                                                                                              sevenofanalyzed <br />                                                                                                                          species and <br />                                                                                                                                   eukaryotic <br />                                                                                                                                        LSEs <br />                                                                       Assignment proteins seven analyzed eukaryotic <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           reports <br />                                                                       genomes KOGs different numbers species LSEs. 0, <br /> Figure 1 shows assignment proteins            Proteins detectable homologs (singletons); 1, LSEs. Species <br /> the analyzed eukaryotes KOGs different numbers             abbreviations: Ath, Arabidopsis thaliana; Cel, Caenorhabditis elegans; Dme, <br /> species, TWOGs LSEs. The fraction proteins assigned            Drosophila melanogaster; Ecu, Encephalitozoon cuniculi; Hsa, Homo sapiens; <br /> to KOGs tends decrease increasing genome size,            Sce, Saccharomyces cerevisisae; Spo, Schizosaccharomyces pombe. <br /> from 81% S. pombe 51% largest, human <br /> genome. (For reasons remain unclear, be <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           deposited research <br /> related intracellular parasitic lifestyle, E. cuniculi a <br /> relatively small fraction conserved proteins belonged <br /> to KOGs: approximately 60%.) The contribution LSEs                 unexpected patterns, example, animal, plant and <br /> shows opposite trend, greatest largest           fungal species (see [53] examples Table 1). <br /> genomes, is, human Arabidopsis, minimal the <br /> microsporidian (Figure 1). A notable difference observed          During manual curation KOG set, KOGs with <br /> between eukaryotes terms representation KOGs           unexpected patterns scrutinized effort detect <br /> found different numbers species. While unicel-        potential highly diverged members the <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           refereed research <br /> lular organisms represented mainly highly con-             analyzed genomes. Some unexpected patterns might <br /> served seven- six-species KOGs, larger fraction          indicate gene missing analyzed set pro- <br /> the gene set animals Arabidopsis accounted           tein sequences species included; <br /> LSEs, KOGs genomes. These               reports newly discovered genes appeared the <br /> include animal-specific genes genes shared            release initial reports genome sequences complex <br /> plants animals fungi microsporidian            eukaryotes, example, result massive sequencing of <br /> (Figure 1). The large number KOGs group (700         human cDNAs [54], exhaustive annotation Drosophila <br /> KOGs represented Arabidopsis animal               genome [55] comparative analysis closely related yeast <br /> species) notable probably results massive, line-          genomes [56]. The unexpected phyletic patterns seem, how- <br /> age-specific loss genes eukaryotic evolution (see           ever, largely reflect extensive, lineage-specific gene loss                                  interactions <br /> below).                                                               characteristic eukaryotic evolution [57]; many <br />                                                                       occasions, scenario supported presence of <br /> The phyletic patterns KOGs reveal existence          orthologs eukaryotic lineages and/or prokaryotes <br /> conserved eukaryotic gene core substantial diversity. The         (Table 1). However, interesting exceptions multiple loss <br /> &apos;pan-eukaryotic&apos; genes, represented          explanation exist exemplified ATP/ADP- <br /> seven analyzed genomes, account 20%                 translocase, present Arabidopsis Encepha- <br /> KOGs, approximately number KOGs include               litozoon evolved independent HGT from <br />                                                                                                                                                                           information <br />  <br />  <br />  <br />  <br /> all species microsporidian, intracellular par-      intracellular bacterial parasites ([58] Table 2). <br /> asite highly degraded genome [51]. Among remain- <br /> ing KOGs, large group includes representatives         Common phyletic patterns genes not <br /> analyzed animal species (worm, fly humans) sub-             suspected functionally linked suggest exist- <br /> stantial fraction (approximately 30%) KOGs                   ence connections prompt additional analysis <br />  <br />                                                         Genome Biology 2004, 5:R7 <br />  R7.4 Genome Biology 2004,      Volume 5, Issue 2, Article R7      Koonin et al.                                            http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />      Table 1 <br />  <br />      KOGs TWOGs unexpected phyletic patterns (examples) <br />  <br />      KOG/TWOG        Phyletic pattern*     (Predicted) structure       Prokaryotic homologs             Comments <br />      number                                function <br />  <br />      TWOG0892        ---H--E               Discoidin domain protein,       Detected few <br />                                            potential regulator          phylogenetically scattered <br />                                            proteasome activity             bacteria, COG far <br />                                                                            [69] <br />      TWOG0263        A-----E               ATP/ADP translocase             ATP/ADP translocases          ATP/ADP translocase hallmark intracellular <br />                                                                            chlamydia, rickettsia, Xylella   parasites symbionts, allows to <br />                                                                            fastidiosa                       scavenge ATP host cell; chloroplast <br />                                                                                                             protein plants. Could acquired plants and <br />                                                                                                             microsporidia independent HGT bacteria. <br />                                                                                                             [58] <br />      TWOG0689        ---HY--               Uncharacterized protein         PrpD protein          The yeast human (and orthologs other <br />                                            essential propionate        bacteria archaea             vertebrates) proteins greatest similarity to <br />                                            metabolism                      (COG2079)                        different subsets bacterial orthologs, might <br />                                                                                                             suggest independent HGT events. <br />      TWOG0871        ---H-P-               Uncharacterized conserved       COG4336, sporadic                The human (and mouse) protein additional <br />                                            protein, probably enzyme        representation        domain conserved archaeon Pyrococcus. <br />                                                                            bacterial lineages               Human S. pombe proteins similar to <br />                                                                                                             different subsets bacterial homologs, which <br />                                                                                                             suggests possibility independent HGT events. <br />      TWOG0788        A----P-               Urease                          Ureases bacterial        Highly conserved enzyme present plants and <br />                                                                            species                          fungi S. cerevisiae. Plant fungal <br />                                                                                                             ureases common domain architecture <br />                                                                                                             distinct bacterial orthologs, which <br />                                                                                                             suggests monophyletic origin. Might evolved <br />                                                                                                             early HGT bacteria (proto-mitochondria?) <br />                                                                                                             subsequent loss animals fungi. <br />      4751            A--H--E               Recombination repair            None                             Although sequence conservation limited the <br />                                            protein BRCA2, contains                                          BRC repeats [101] number varies <br />                                            varying number BRCA2                                          substantially, statistical significance observed <br />                                            repeats                                                          sequence similarity absence other <br />                                                                                                             homologs suggests proteins KOG <br />                                                                                                             true orthologs. Apparent orthologs BRCA2 <br />                                                                                                             detectable species taxa <br />                                                                                                             represented KOGs (mosquito Anopheles <br />                                                                                                             gambiae, fungus Ustilago maydis) [102] early- <br />                                                                                                             branching eukaryotes (Leishmania, Trypanosoma; <br />                                                                                                             E.V.K., unpublished work), suggesting evolution <br />                                                                                                             BRCA2 involved multiple gene losses <br />      4597            A--H--E               TATA-binding protein 1-         None                             Probable multiple gene losses <br />                                            interacting protein <br />      4486            A--H--E               3-methyl-adenine DNA            Orthologs bacteria       The plant protein mammals and <br />                                            glycosylase                     (COG2094)                        microsporidia greatest similarity to <br />                                                                                                             different subsets bacterial orthologs. Evolution <br />                                                                                                             included combination gene loss and <br />                                                                                                             independent HGT events <br />      1594            A-D-Y--               Predicted epimerase             Bacterial orthologs,             Eukaryotic proteins closely related to <br />                                            related aldose 1-            primarily proteobacteria         bacterial orthologs, indicating <br />                                            epimerase                       (COG0676)                        monophyletic origin. Function remains unknown; <br />                                                                                                             involved distinct still <br />                                                                                                             uncharacterized pathway polysaccharide <br />                                                                                                             biosynthesis. LSE Arabidopsis (seven paralogs). <br />      4141            ---HYPE               Rad52/22, protein involved      None                             Probable gene loss plants, insects nematodes <br />                                            double-strand break <br />                                            repair <br />  <br />  <br />  <br />  <br />                                                                      Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                              Genome Biology 2004,      Volume 5, Issue 2, Article R7      Koonin et al. R7.5 <br />  <br />  <br />  <br />  <br /> Table 1 (Continued) <br />  <br /> KOGs TWOGs unexpected phyletic patterns (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     comment <br /> 4528             -CDH--E             Uncharacterized predicted      Conserved archaea       Context analysis archaeal bacterial genomes <br />                                      enzyme, possibly             bacteria           suggests functional interaction proteins of <br />                                      polynucleotide kinase          (COG1371)                      KOG5324 KOG4246, RNA 3&apos;-terminal <br />                                      (structure ortholog                                    phosphate cyclase (KOG4398, COG0430), and <br />                                      bacterium                                            tRNA/rRNA cytosine C5-methylase (KOG1299/ <br />                                      Thermotoga maritima                                       COG0144) ([103] E.V.K., unpublished <br />                                      determined - pdb                                         observations). Taken together, observations <br />                                      <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1j5u)                                                    appear implicate KOG5324 KOG4246 a <br />                                                                                                    uncharacterized pathway rRNA and/or tRNA <br />                                                                                                    processing modification. Conservation these <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     reviews <br />                                                                                                    proteins archaea early-branching eukaryotes <br />                                                                                                    suggests lineage-specific gene loss plants fungi. <br /> 3833             -CDH--E             Uncharacterized predicted      Conserved archaea       See comment KOG5324 <br />                                      enzyme, possibly             bacteria <br />                                      polynuclotide phosphatase      (COG1690) <br />  <br /> *Abbreviations: A, thale cress A. thaliana; C, nematode C. elegans; D, fruit fly D. melanogaster; E, microsporidian Encephalitozoon cuniculi; H, Homo <br /> sapiens; S, budding yeast S. cerevisiae; P, fission yeast S. pombe; letter indicates presence respective species given KOG dash <br /> indicates absence. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     reports <br /> leading concrete functional predictions [42,59-61]. The                      general functional prediction feasible, and <br /> pair KOG5324 KOG4246 case point                     functions remain unknown, sub- <br /> been described previously. The initial observation                   set represented seven eukaryotic species, <br /> KOGs share unusual pattern presence-absence                      emphasizes current understanding eukaryotic <br /> eukaryotes, similar phyletic patterns prokaryotes,                  biology seriously lacking respect func- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     deposited research <br /> with ubiquitous presence archaea, prompted                          tions highly conserved genes. <br /> detailed examination multiple alignments the <br /> respective proteins conservation (predicted)                     The distribution KOGs number paralogs each <br /> operon organization archaea bacteria (Table 2                        genome shown Figure 3. The preponderance lineage- <br /> data shown). The combination clues analy-                     specific duplication conserved genes, is, intra-KOG <br /> ses suggests proteins interact uncharac-                LSEs, multicellular eukaryotes obvious. Cases a <br /> terized pathway RNA processing, includes RNA                      single gene yeast or, particularly, Encephalitozoon, two <br /> 3&apos;-phosphate cyclase (KOG3980)) [62] cytosine-C5-                           co-orthologs animals and/or plants com- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     refereed research <br /> methylase (NOL1/NOP2 eukaryotes; KOG1122). The pro-                          mon KOGs, reverse situation rare. These <br /> teins KOG3833 KOG4528 likely represent novel                      observations support notion major contribution of <br /> enzyme families, possibly kinase-phosphatase pair (E.V.K.                     LSE evolution eukaryotic complexity [52]. However, <br /> and L. Aravind, unpublished data). Notably, predicted                     131 KOGs represented single ortholog genomes <br /> new enzymes present animals E. cuniculi                   compared (Table 2) substantial number KOGs have <br /> Arabidopsis yeasts. In contrast, KOG3980 present                   member majority genomes (data not <br /> analyzed eukaryotic genomes Arabidopsis,                             shown). Recent theoretical modeling evolution par- <br /> whereas KOG1122 pan-eukaryotic. These differences                     alogous families suggested that, general, ancient pro- <br /> phyletic patterns components predicted pathway                    tein families tend multiple paralogs [5,63]. Therefore, <br /> are concordant patterns eukaryotes that.                         KOG single member species,                          interactions <br />                                                                                 attributed selection duplication of <br /> Figure 2 shows distribution known predicted func-                    particular gene. A prominent cause selection <br /> tions eukaryotic proteins 20 functional categories                     involvement respective gene products in <br /> for entire set KOGs and, separately, KOGs repre-                     essential multisubunit complexes, imbalance <br /> sented seven species animal-specific KOGs.                    subunits leads deleterious effects [64]. <br /> Compared functional breakdown prokaryotic COGs <br /> [25], prevalence signal transduction notable                    Known new functions single-member, pan- <br />                                                                                                                                                                     information <br />  <br />  <br />  <br />  <br /> eukaryotes. This feature particularly prominent animal-                   eukaryotic KOGs <br /> specific KOGs, highly conserved set compara-                     We examined greater 131 KOGs repre- <br /> tively enriched proteins involved translation,                   sented single gene seven genomes (Table <br /> transcription, chaperone-like functions, cell cycle control                 2). As envisaged presence diverse eukary- <br /> chromatin dynamics (Figure 2). The large number KOGs                         otic taxa, including &apos;minimal&apos; genome of <br />  <br />                                                                Genome Biology 2004, 5:R7 <br />  R7.6 Genome Biology 2004,   Volume 5, Issue 2, Article R7        Koonin et al.                                        http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />      Table 2 <br />  <br />      KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />      KOG number     (Predicted) function          Multiprotein        Functional   Prokaryotic               Fitness class&#226;&#8364;&#160;   Comments <br />                                                   complex               class*     homologs <br />  <br />                                                                                                             Yeast&#226;&#8364;&#161;   Worm&#194;&#167; <br />  <br />      Genes experimentally computationally characterized previously <br />      0392           SNF2 family DNA-              TBP-DNA                          Many bacteria          0         1     Involved regulation of <br />                     dependent ATPase              complex                          archaea (COG0553)                          transcription POL II <br />                                                                                                                               promoters [104] <br />      0121           Nuclear cap-binding           Cap-binding              A       Several bacteria           1        X      RRM-domain proteins show <br />                     protein complex, subunit      complex                          (COG0724)                                  scattered presence in <br />                     CBP20 (RRM-domain-                                                                                        bacteria been <br />                     containing RNA-binding                                                                                    horizontally transferred from <br />                     protein)                                                                                                  eukaryotes <br />      0213           U2-snRNP associated           Spliceosome              A       None                       0         0 <br />                     splicing factor 3b, subunit <br />                     1 <br />      0227           snRNA-associated              Spliceosome              A       None                       0         0 <br />                     protein, splicing factor <br />                     3a, subunit b (Prp11p) <br />      2268           Predicted nucleic-acid-       Pre-40S subunit          A       Orthologs          0        X      One small <br />                     binding protein kinase                                      archaea                         number protein kinases <br />                     RIO1 family; 40S                                           bacteria (COG0478)                         clear-cut <br />                     ribosomal subunit                                                                                         orthologous relationship <br />                     biogenesis/18S rRNA                                                                                       eukaryotes and <br />                     processing                                                                                                archaea, and, <br />                                                                                                                               apparently, one <br />                                                                                                                               containing helix-turn-helix <br />                                                                                                                               nucleic-acid-binding domain. <br />                                                                                                                               [105] Associated yeast <br />                                                                                                                               pre-40S subunit required <br />                                                                                                                               maturation. [106] <br />      3031           Protein required 60S      Processosome             A       Distantly related       0        X      The COG2136 proteins <br />                     ribosomal subunit                                              COG2136,                                   appear subunits the <br />                     biogenesis; [107]                                              represented                             predicted archaeal exosome <br />                     contains IMP4                                              orthologs                          [109]. Apparently, gene <br />                     domain, involved                                      archaea,                        undergone two <br />                     rRNA processing                                             bacteria (KSM,                             ancient duplications in <br />                     [108]; paralog                                              unpublished)                               eukaryotes <br />                     KOG3095 and <br />                     KOG3292, are <br />                     represented all <br />                     analyzed genomes. <br />      3045           Predicted RNA                 Processosome?            A       Distantly related       1         1     This protein (Rrp8p yeast) <br />                     methylase involved                                          numerous Rossmann-                         shown to <br />                     rRNA processing                                                fold methylases                        participate processing <br />                                                                                    prokaryotic orthologs                      rRNA sequence <br />                                                                                                                  analysis reveals presence <br />                                                                                    confidently identified                     Rossmann-fold <br />                                                                                                                               methylase domain [110]. <br />                                                                                                                               Therefore Rrp8p probably <br />                                                                                                                               methylates snoRNA <br />                                                                                                                               rRNA itself. <br />      3064           RNA-binding nuclear           Processosome             A       None                       0         0     Initially identified yeast as <br />                     protein containing                                                                                      MAK16 protein required <br />                     distinct C4 Zn-finger;                                                                                    dsRNA virus <br />                     implicated                                                                                         reproduction [112] <br />                     biogenesis 60S <br />                     ribosomal subunits [111] <br />  <br />  <br />  <br />  <br />                                                                     Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                         Genome Biology 2004,        Volume 5, Issue 2, Article R7   Koonin et al. R7.7 <br />  <br />  <br />  <br />  <br /> Table 2 (Continued) <br />  <br /> KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                               comment <br /> 0291, 0302,     WD40-repeat proteins,      Processosome           A       WD40-repeat               0     X,X,1, <br /> 0306, 310,      subunits rRNA                                          proteins present                X,1,1,1 <br /> 0319, 0650,     processing complexes                                      bacterial <br /> 1272            [69,70]                                                   lineages are <br />                                                                           particularly abundant <br />                                                                           cyanobacteria but <br />                                                                           missing most <br />                                                                           archaea; of <br />                                                                           appear be <br />                                                                           obvious orthologs of <br />  <br />  <br />  <br />  <br />                                                                                                                                                               reviews <br />                                                                           protein <br />                                                                           (COG2319) <br /> 0284            Polyadenylation factor I   Poly-                  A       Same              0          X <br />                 complex, subunit PFS2,     adenylation                    (COG2319) <br />                 WD40-repeat protein        complex <br /> 0337            RNA helicase involved   Processosome           A       Most archaea        0          X <br />                 28S rRNA processing                                       bacteria <br />                                                                           (COG0513) <br /> 0343            RNA helicase involved   Processosome           A       Most archaea        0          X <br />  <br />  <br />  <br />  <br />                                                                                                                                                               reports <br />                 28S rRNA processing                                       bacteria <br />                                                                           (COG0513) <br /> 1069            3&apos;-5&apos; exoribonuclease      Exosome                A       Most bacteria          0           1 <br />                 (RNAse PH), exosome                                       archaea (COG0689) <br />                 subunit Rrp46 <br /> 1070            Exosome subunit Rrp5       Exosome                A       Most bacteria              0           1 <br />                 (RNA-binding S1 domain                                    (COG0539, <br />  <br />  <br />  <br />  <br />                                                                                                                                                               deposited research <br />                 fused TPR repeats)                                     COG0457) <br /> 1135            mRNA cleavage          Cleavage           A       Most archaea           0           0 <br />                 polyadenylation complex    polyadenylation                bacteria <br />                 subunit CFT2 (CPSF)        complex                        (COG1236) <br /> 1914            mRNA cleavage          Cleavage           A       None                       0          X <br />                 polyadenylation factor I   polyadenylation <br />                 complex, subunit RNA14     complex <br /> 1975            RNA (guanine-7-)           Capping                A       Numerous                   0           1 <br />                 methyltransferase          enzyme                         methyltrans-ferases <br />                 (capping enzyme subunit)                                  (COG0500) no <br />  <br />  <br />  <br />  <br />                                                                                                                                                               refereed research <br />                                                                           ortholog <br /> 2051            Nonsense-mediated          NMD complex            A       None                       1          X <br />                 mRNA decay complex, <br />                 subunit 2 <br /> 2554            Pseudouridylate synthase   ?                      A       Most archaea           1           1 <br />                                                                           bacteria (COG0101) <br /> 2613            Upf1p-interacting          NMD complex            A       All archaea,            0          X <br />                 protein, NMD complex                                      bacteria (COG1499) <br />                 subunit Nmd3p <br /> 2771            tRNA-specific adenosine-   Heterodimeric          A       Most bacteria          0          X <br />                                                                                                                                                               interactions <br />                 34 deaminase subunit       RNA-specific                   archaea <br />                 Tad3p                      deaminase                      (COG0590) <br /> 2780            Protein involved        Processosome           A       Most archaea,           0           1 <br />                 ribosomal large subunit                                   bacteria (COG2136) <br />                 assembly (RPF1), <br />                 contains IMP4 domain <br /> 2781            Subunit small       Processosome           A       Most archaea,           0           1 <br />                 (ribosomal) subunit                                       bacteria (COG2136) <br />                 (SSU) processosome <br />                                                                                                                                                               information <br />  <br />  <br />  <br />  <br />                 (snoRNP), IMP4 <br /> 2874            Protein involved rRNA   ?                      A       All archaea,            0           1      Predicted RNA-binding <br />                 processing ribosomal                                  bacteria (COG1094)                            protein containing KH <br />                 assembly                                                                                                domain <br /> 3013            Exosome subunit Rrp4       Exosome                A       Most archaea,           0          X <br />                                                                           bacteria (COG1097) <br />  <br />  <br />                                                              Genome Biology 2004, 5:R7 <br />  R7.8 Genome Biology 2004,    Volume 5, Issue 2, Article R7     Koonin et al.                              http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />      Table 2 (Continued) <br />  <br />      KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />      3031            Protein involved large   Processosome            A      None                  0    X     Contains BRIX domain <br />                      ribosome subunit <br />                      assembly 28S rRNA <br />                      processing (Rrf2) <br />      3322            RNAse P/MRP subunit,        RNAse P/MRP             A      None                  0    1 <br />                      involved processing   holoenzyme <br />                      pre-tRNAs 5.8S <br />                      rRNA <br />      3448            Predicted snRNP core        Spliceosome             A      All archaea,       0    1 <br />                      protein                                                    bacteria (COG1958) <br />      3482            Small nuclear               Spliceosome             A      All archaea,       0    0 <br />                      ribonucleoprotein                                          bacteria (COG1958) <br />                      (snRNP) SMF subunit <br />      2463            Predicted RNA-binding       26S                   A,O      Represented        0    X     PIN domain been <br />                      protein, consisting    proteasome,                    orthologs                 detected exosome <br />                      PIN domain Zn-        pre-40S subunit                archaea                   subunits thought to <br />                      ribbon. Involved 26S                                    bacteria (COG1349)               RNA-binding properties <br />                      proteasome assembly                                                                         nuclease activity <br />                                                                                                                  [113,114]. The <br />                                                                                                                  demonstration role of <br />                                                                                                                  protein (Nob1p) in <br />                                                                                                                  proteasome assembly [115], <br />                                                                                                                  40S ribosome subunit <br />                                                                                                                  assembly, processing <br />                                                                                                                  18S rRNA 3&apos;-end [116] <br />                                                                                                                  supports connection <br />                                                                                                                  degradation RNA <br />                                                                                                                  proteins to <br />                                                                                                                  established already <br />                                                                                                                  archaea [109]. <br />      3273            Predicted RNA-binding       26S                   A,O      Orthologs      0    0     This second predicted <br />                      protein containing KH       proteasome,                    archaea                   RNA-binding protein <br />                      domain, interacts      pre-40S subunit                bacteria (COG1094)               involved proteasome <br />                      Nob1p                                                                                       assembly, [115] which <br />                                                                                                                  emphasizes the <br />                                                                                                                  aforementioned link between <br />                                                                                                                  RNA protein processing <br />      1831            Deadenylating 3&apos;-5&apos;         CCR4-NOT               AK      None                  0    0 <br />                      exonuclease, negative       core complex <br />                      regulator PolII <br />                      transcription <br />      1159            NADP-dependent              ?                      CL      Many bacteria         0    X     Genetic evidence role in <br />                      flavoprotein reductase,                                    (COG0369)                        DNA replication [117] <br />                      probably sulfite <br />                      reductase subunit <br />      1800            Ferredoxin/adrenodoxin      ?                       C      Most bacteria     0    X <br />                      reductase                                                  archaea <br />                                                                                 (COG0493) <br />      1173            Anaphase-promoting          APC                     D      Most archaea   0    0 <br />                      complex (APC), Cdc16                                       bacteria TPR- <br />                      subunit (TPR-repeat                                        repeat proteins <br />                      protein)                                                   (COG0457) no <br />                                                                                 orthologs Cdc16 <br />      3437            Anaphase-promoting          APC                     D      None                  1    1 <br />                      complex (APC), subunit <br />                      10 <br />      1358            Serine                      ?                       I      Most bacteria     0    0 <br />                      palmitoyltransferase                                       archaea <br />                                                                                 (COG0156) <br />      1511            Mevalonate kinase           ?                       I      Most archaea      0    X <br />                                                                                 bacteria <br />                                                                                 (COG1577) <br />  <br />  <br />                                                                    Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                            Genome Biology 2004,       Volume 5, Issue 2, Article R7   Koonin et al. R7.9 <br />  <br />  <br />  <br />  <br /> Table 2 (Continued) <br />  <br /> KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 comment <br /> 3059            N-acetylglucosaminyl-        N-acetylglucos-         I       None                      0           1 <br />                 transferase complex,         aminyltransferase <br />                 subunit PIG-C/GPI2,          complex <br />                 involved phosphatidyli- <br />                 nositol biosynthesis <br /> 0467            Translation elongation       ?                       J       All (COG0480)             0          X       Involved 60S ribosomal <br />                 factor 2 paralog                                                                                          subunit maturation [118] <br />                 (GTPase) <br /> 1147            Glutamyl-tRNA                Multispecificity        J       All (COG0008)             0          X <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 reviews <br />                 synthetase                   aminoacyl- <br />                                              tRNA <br />                                              synthetase <br />                                              complex <br /> 2784            Phenylalanyl-tRNA            Heterodimeric           J       All (COG0016)             0          X <br />                 synthetase, beta subunit     phenylalanyl- <br />                                              tRNA <br />                                              synthetase <br /> 3123            Diphtamide synthase          ?                       J       All archaea,           1           1 <br />                 (methyltransferase)                                          bacteria (COG1798) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 reports <br /> 0261            RNA polymerase III,          RNAPIII                 K       All (COG0086)             0          X <br />                 largest subunit              holoenzyme <br /> 0262            RNA polymerase I,            RNAPI                   K       All (COG0086)             0          X <br />                 largest subunit              holoenzyme <br /> 0215            RNA polymerase III,          RNAPIII                 K       All (COG0085)             0          X <br />                 second largest subunit       holoenzyme <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 deposited research <br /> 0216            RNA polymerase I,            RNAPI                   K       All (COG0085)             0          X <br />                 second largest subunit       holoenzyme <br /> 1063            RNA polymerase II            RNA                     K       WD40-repeat               1          X <br />                 elongator complex,           polymerase II                   proteins present <br />                 subunit ELP2, WD repeat      elongator                       bacterial <br />                 protein                      complex                         lineages are <br />                                                                              particularly abundant <br />                                                                              cyanobacteria but <br />                                                                              missing most <br />                                                                              archaea; of <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 refereed research <br />                                                                              appear be <br />                                                                              obvious orthologs of <br />                                                                              protein <br />                                                                              (COG2319) <br /> 1131            RNA polymerase II            RNAPII                  K       Most archaea          0          X <br />                 transcription initiation/    holoenzyme                      bacteria (COG1199) <br />                 nucleotide excision <br />                 repair factor TFIIH, 5&apos;-3&apos; <br />                 helicase subunit RAD3 <br /> 1920            RNA polymerase II            RNAP II                 K       None                      1          X <br />                 Elongator subunit            elongator <br />  <br />                                                                                                                                                                 interactions <br />                                              complex <br /> 1932            TBP-associated factor        TFIID complex           K       None                      0          X <br />                 (Taf2p) <br /> 2009            Transcription initiation     TFIIIB                  K       None                      0           0 <br />                 factor TFIIIB, Bdp1 <br />                 subunit (Myb domain) <br /> 2076            RNA polymerase III           TFIIIC                  K       Most archaea       0          X <br />                 transcription factor                                         bacteria TPR- <br />                 TFIIIC, TPR-repeat-                                          repeat proteins <br />                                                                                                                                                                 information <br />  <br />  <br />  <br />  <br />                 containing protein                                           (COG0457) no <br />                                                                              orthologs TFIIC <br /> 2487            RNA polymerase II            TFIIH                   K       None                      0           1 <br />                 transcription initiation/ <br />                 nucleotide excision <br />                 repair factor TFIIH, <br />                 subunit TFB4 <br />  <br />                                                                 Genome Biology 2004, 5:R7 <br />  R7.10 Genome Biology 2004,     Volume 5, Issue 2, Article R7        Koonin et al.                              http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 2 (Continued) <br />  <br />        KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />        2691            RNA polymerase II            RNAP II                   K     Most archaea,       1    X <br />                        subunit 9                    holoenzyme                      bacteria (COG1594) <br />        2807            RNA polymerase II            TFIIH                     K     No orthologs           0    0     Consists von Willebrand <br />                        transcription initiation/                                    von                      A domain closely <br />                        nucleotide excision                                          Willebrand A                      related the <br />                        repair factor TFIIH, SSL1                                    domains present               proteasome subunit RPN10 <br />                        subunit                                                      variety                   [119] Zn-finger domain <br />                                                                                     prokaryotic proteins <br />        2907            RNA polymerase I             TFIIS                     K     All archaea,        1    0 <br />                        transcription factor                                         bacteria (COG1594) <br />                        TFIIS, subunit A12.2/ <br />                        RPA12 <br />        3169            RNA polymerase II            Mediator                  K     None                   0    X <br />                        transcriptional regulation   complex [120] <br />                        mediator <br />        3233            RNA polymerase III           RNAP III                  K     None                   0    1 <br />                        subunit C34                  holoenzyme <br />        3297            RNA polymerase III           RNAP III                  K     All archaea,        0    0 <br />                        subunit C25                  holoenzyme                      bacteria (COG1095) <br />        3438            Subunit common RNA        RNAP I III            K                            0    1 <br />                        polymerases I (A) III    holoenzymes <br />                        (C); Rpc19p <br />        3471            RNA polymerase II            TFIIH                     K     None                   0    X <br />                        transcription initiation/ <br />                        nucleotide excision <br />                        repair factor TFIIH, <br />                        subunit TFB2 <br />        3490            Transcription elongation     Chromatin-                K     None                   1    1 <br />                        factor SPT4, Zn-ribbon       associated <br />                        protein                      transcription <br />                                                     complexes <br />        3497            RNA polymerase II            RNAP II                   K     All archaea,        0    X <br />                        subunit; Rpb10p              holoenzyme                      bacteria (COG1644) <br />        3901            Transcription initiation     TFIID                     K     None                   0    X <br />                        factor IID subunit <br />                        (Taf13p) <br />        3949            RNA polymerase II            RNAP II                   K     None                   1    1 <br />                        elongator complex,           elongator <br />                        subunit ELP4                 complex <br />        4086            SOH1 protein potentially     SMCC complex              K     None                   1    X <br />                        involved Pol II           [121] <br />                        transcription regulation <br />                        repair <br />        1532            Predicted GTPase      TBP-free                  L     All archaea        0    0     XP-A-binding protein in <br />                        XAB1 family [122]            TAF(II)                         bacteria                  humans, implicated in <br />                                                     complex                         (COG1100)                         repair ([122] references <br />                                                                                                                       therein). <br />        1533            Predicted GTPase      TBP-free                  L     All archaea        0    X     Might function in <br />                        XAB1 family (paralog      TAF(II)                         bacteria                  repair given paralogous <br />                        KOG1757) [122]               complex?                        (COG1100)                         relationship KOG1757. <br />  <br />  <br />  <br />  <br />                                                                        Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                        Genome Biology 2004,       Volume 5, Issue 2, Article R7    Koonin et al. R7.11 <br />  <br />  <br />  <br />  <br /> Table 2 (Continued) <br />  <br /> KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                               comment <br /> 1625            DNA polymerase &#206;&#177;          DNA                    L       Small subunit          0           0      The small, regulatory subunit <br />                 processivity subunit,     polymerase &#206;&#177;                   archaeal DNA                                 DNA polymerase &#206;&#177; also <br />                 inactivated phosphatase   holoenzyme                     polymerase II                                forms pan-eukaryotic <br />                                                                          (COG1311)                                    KOG3044, paralog <br />                                                                                                                       KOG0861 (the only <br />                                                                                                                       recent duplication in <br />                                                                                                                       KOG3044 seen in <br />                                                                                                                       vertebrates). In contrast, <br />                                                                                                                       paralog, small <br />                                                                                                                       subunit DNA polymerase <br />  <br />  <br />  <br />  <br />                                                                                                                                                               reviews <br />                                                                                                                       &#206;&#181;, represented animals, <br />                                                                                                                       fungi early-branching <br />                                                                                                                       protozoan Plasmodium, but <br />                                                                                                                       plants or <br />                                                                                                                       Microsporidia. Thus, the <br />                                                                                                                       history polymerase <br />                                                                                                                       subunit apparently involved <br />                                                                                                                       inactivation the <br />                                                                                                                       phosphatase (or nuclease) <br />                                                                                                                       inherited archaea, with <br />  <br />  <br />  <br />  <br />                                                                                                                                                               reports <br />                                                                                                                       subsequent duplications at <br />                                                                                                                       early stages eukaryotic <br />                                                                                                                       evolution [123] <br /> 0479            DNA replication           Pre-replication        L       All archaea,           0          X <br />                 licensing factor MCM3     complex                        bacteria (COG1241) <br /> 0481            DNA replication           Pre-replication        L       All archaea,           0          X <br />                 licensing factor MCM5     complex                        bacteria (COG1241) <br />  <br />  <br />  <br />  <br />                                                                                                                                                               deposited research <br /> 0482            DNA replication           Pre-replication        L       All archaea,           0           0 <br />                 licensing factor MCM7     complex                        bacteria (COG1241) <br /> 0964            Structural maintenance    Sister                 L       Many archaea          0          X <br />                 chromosome protein     chromatid                      bacteria (COG1196) <br />                 3 (cohesin subunit        cohesion <br />                 SMC3)                     complex <br /> 0979            Structural maintenance    Sister                 L       Many archaea          0          X <br />                 chromosome protein     chromatid                      bacteria (COG1196) <br />                 5 (cohesin subunit        cohesion <br />                 SMC5)                     complex <br />  <br />  <br />  <br />  <br />                                                                                                                                                               refereed research <br /> 1942            TBP-interacting protein   chromatin              L       Most archaea,      0           0 <br />                 TIP49 (DNA helicase)      remodeling                     bacteria <br />                                           complex                        (COG1224) <br /> 1979            DNA mismatch repair       Mismatch               L       Most bacteria         1           1 <br />                 ATPase, MLH1              repair complex                 archaea <br />                                                                          (COG0323) <br /> 2267            DNA primase, large        DNA                    L       All archaea,           0           0 <br />                 subunit                   polymerase                     bacteria (COG2219) <br />                                           &#206;&#177;:primase <br />                                           complex <br />  <br />                                                                                                                                                               interactions <br /> 2299            Ribonuclease HI           Replisome              L       All archaea,         1          X <br />                                                                          bacteria (COG0164) <br /> 2310            DNA repair exonuclease    MRN complex            L       All archaea,         1           1 <br />                 MRE11                     involved                    bacteria (COG0420) <br />                                           double-strand <br />                                           break repair <br /> 2929            Origin recognition        ORC                    L       None                      1           1 <br />                 complex, subunit 2 <br />                 (ORC2) <br />                                                                                                                                                               information <br />  <br />  <br />  <br />  <br /> 0179            20S proteasome,           20S                    O       All archaea      0           0 <br />                 regulatory subunit beta   proteasome                     actinomycetes among <br />                 type PSMB1/PRE7                                          bacteria (COG0638) <br />                 (paralog KOG0185) <br />  <br />  <br />  <br />  <br />                                                             Genome Biology 2004, 5:R7 <br />  R7.12 Genome Biology 2004,    Volume 5, Issue 2, Article R7      Koonin et al.                                http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 2 (Continued) <br />  <br />        KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />        0185            20S proteasome,            20S                     O      All archaea     0    0 <br />                        regulatory subunit beta    proteasome                     actinomycetes among <br />                        type PSMB4/PRE4                                           bacteria (COG0638) <br />                        (paralog KOG0179) <br />        2708            Predicted                  Putative                O      Represented           0    X     One remaining <br />                        metalloprotease       complex                        orthologs                    uncharacterized proteins <br />                        chaperone activity         involved                    archaea bacteria                universally <br />                        (RNAse H/HSP70 fold)       translation                    (COG0533)                           conserved cellular life <br />                        [124]                      regulation                                                         forms. The only <br />                                                   [125]                                                              experimentally demonstrated <br />                                                                                                                      activity of <br />                                                                                                                      sialoglycoprotease fusion <br />                                                                                                                      distinct protein kinase <br />                                                                                                                      archaea and <br />                                                                                                                      analysis gene <br />                                                                                                                      neighborhood suggest a <br />                                                                                                                      fundamental role signal <br />                                                                                                                      transduction, possibly <br />                                                                                                                      translation regulation. [125] <br />        0301            Protein required       Proteasome?             O      Same            1    X <br />                        normal rates ubiquitin-                                (COG2319) <br />                        dependent proteolysis, <br />                        contains WD40 repeats <br />        0358            Chaperonin complex         TCP-1                   O      All archaea nearly   0    0 <br />                        component, TCP-1 delta                                    bacteria <br />                        subunit (CCT4)                                            (COG0459) <br />        0363            Chaperonin complex         TCP-1                   O      All archaea nearly   0    0 <br />                        component, TCP-1 beta                                     bacteria <br />                        subunit (CCT2)                                            (COG0459) <br />        0687            26S proteasome             26S                     O      None                     0    0 <br />                        regulatory complex,        proteasome <br />                        subunit RPN7/PSMD6 <br />        1299            Vacuolar sorting protein   t-SNARE                 O      None                     1    X     Involved t-SNARE <br />                        VPS45/Stt10 (Sec1          complex                                                            complex assembly [126] <br />                        family) <br />        1349            GPI-anchor transamidase    GPI-anchor              O      Distantly related        0    1 <br />                        complex, GPI8 subunit      transamidase                   proteases some <br />                                                   complex                        bacteria (no COG) <br />        1943            Beta-tubulin folding    ?                          O      None                     1    1 <br />                        cofactor D, involved in <br />                        chromosome segregation <br />        2015            NEDD8-activating           NEDD8-                  O      Most bacteria        1    1 <br />                        complex, UBA3 subunit      activating                     archaea <br />                                                   complex                        (COG0476) <br />        2126            Phosphoethanolamine N-     ?                       O      Several bacteria     0    X <br />                        methyltransferase                                         archaea (COG1524) <br />                        involved GPI-anchor <br />                        biosynthesis <br />        2884            26S proteasome             26S                     O      No orthologs             1    1     Contains von Willebrand A <br />                        regulatory complex,        proteasome                     von                        domain <br />                        subunit RPN10/PSMD4        regulatory                     Willebrand A <br />                                                   complex                        domains present <br />                                                                                  variety of <br />                                                                                  prokaryotic proteins <br />        2908            26S proteasome             26S                     O      None                     0    0     Contains PINT domain <br />                        regulatory complex,        proteasome <br />                        subunit RPN9/PSMD13        regulatory <br />                                                   complex <br />        0209            Endoplasmic reticulum      ?                        P     Many bacteria        1    X <br />                        membrane P-type                                           archaea <br />                        ATPase                                                    (COG0474) <br />  <br />  <br />  <br />                                                                     Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                           Genome Biology 2004,       Volume 5, Issue 2, Article R7     Koonin et al. R7.13 <br />  <br />  <br />  <br />  <br /> Table 2 (Continued) <br />  <br /> KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   comment <br /> 3379            Uncharacterized member        ?                     R       Most archaea          1          X       Only biochemical function <br />                 histidine triad                                      bacteria (COG0537)                           predicted. <br />                 superfamily nucleotide <br />                 hydorlases <br /> 2635            Coatomer (COPI)               COPI complex          U       None                      0           0 <br />                 complex delta subunit <br /> 2927            Membrane component         Sec complex           U       None                      0           1 <br />                 ER protein translocation <br />                 apparatus (Sec62) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   reviews <br /> 2978            Dolichol-phosphate            ?                     U       All archaea,         0          X <br />                 mannosyltransferase                                         bacteria (COG0463) <br /> 3198            Signal recognition            Signal                U       All archaea,           0          X <br />                 particle, subunit Srp19       recognition                   bacteria (COG1400) <br />                                               particle <br /> 3315            Subunit targeting      TRAPP                 U       None                      0          X <br />                 complex (TRAPP) <br />                 involved ER Golgi <br />                 trafficking <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   reports <br /> 3369            Subunit targeting      TRAPP                 U       None                      0          X <br />                 complex (TRAPP) <br />                 involved ER Golgi <br />                 trafficking <br /> 1992            Nuclear export receptor       ?                    YU       None                      0          X <br />                 CSE1/CAS (importin <br />                 beta) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   deposited research <br /> New functional predictions <br /> 2316            PP-loop family ATP            ?                     A       Orthologs PP-      1          X       PP-loop ATPases been <br />                 pyrophosphatase                                             loop domain                              previously implicated <span id='am-398' about='xsp:base' typeof='owl:Thing'>base</span> <br />                 domain, fungi,                                     present archaea                       thiolation various RNAs <br />                 plants insects fused                                 (COG2102)                            [127] proteins K/ <br />                 duplicated                                             bacteria. Orthologs                       COG similar <br />                 translation inhibitor                                       translation                           function, likely be <br />                 domain. The fusion,                                   inhibitor domain                         conserved eukaryotes and <br />                 phyletic pattern                                                                  archaea. However, fusion <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   refereed research <br />                 PP-ATPase                                            bacteria                         translation inhibitor, <br />                 domain, suggests                                         archaea (COG0251)                            reported to <br />                 essential function                                                                                    endoribonuclease <br />                 translation regulation                                                                                   activity [128] eukaryote- <br />                                                                                                                          specific feature <br /> 2523            Predicted RNA-binding         Putative novel        A       Orthologs present      1          X       Several archaeal <br />                 protein containing PUA      RNA                           archaea                                  orthologs protein <br />                 domain, probable role      modification                  (COG2016)                            form fusions PP-loop <br />                 RNA modification [129]        complex                       bacteria                                  ATPase domain implicated in <br />                                                                                                                          base thiolation [127]. Thus, <br />                                                                                                                          proteins KOG <br />  <br />                                                                                                                                                                   interactions <br />                                                                                                                          interact of <br />                                                                                                                          KOG2840 (pan-eukaryotic, <br />                                                                                                                          duplications Arabidopsis <br />                                                                                                                          worm) KOG2594 <br />                                                                                                                          (missing humans and <br />                                                                                                                          microsporidia) form a <br />                                                                                                                          novel enzymatic complex <br />                                                                                                                          involved RNA modification <br />                                                                                                                                                                   information <br />  <br />  <br />  <br />  <br />                                                                Genome Biology 2004, 5:R7 <br />  R7.14 Genome Biology 2004,     Volume 5, Issue 2, Article R7    Koonin et al.                                      http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 2 (Continued) <br />  <br />        KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />        0270, 0271,     WD40-repeat proteins         Processosome            A      WD40-repeat             0   X,1,X   By analogy other <br />        1539                                                                        proteins present                    conserved WD40-repeat <br />                                                                                    bacterial                    proteins, predicted be <br />                                                                                    lineages                        subunits rRNA processing/ <br />                                                                                    particularly abundant                   ribosome assembly <br />                                                                                    cyanobacteria                    complexes <br />                                                                                    missing most <br />                                                                                    archaea; of <br />                                                                                    appear be <br />                                                                                    obvious orthologs of <br />                                                                                    protein <br />                                                                                    (COG2319) <br />        2321            Nucleolar protein,           rRNA                    A      WD40-repeat              0       1      Probable subunit an <br />                        contains WD40 repeats        processosome?                  proteins present                    rRNA-processing complex <br />                                                                                    bacterial <br />                                                                                    lineages are <br />                                                                                    particularly abundant <br />                                                                                    cyanobacteria but <br />                                                                                    missing most <br />                                                                                    archaea; of <br />                                                                                    appear be <br />                                                                                    obvious orthologs of <br />                                                                                    protein <br />                                                                                    (COG2319) <br />        1763            Uncharacterized              ?                       A      None                     1       1      CCCH fingers been <br />                        conserved protein                                                                                   shown bind 3&apos; <br />                        containing CCCH Zn-                                                                               untranslated regions in <br />                        finger; possible role                                                                            various mRNAs [130,131] <br />                        RNA processing or <br />                        splicing <br />        2837            Protein containing U1-     Spliceosome?            A      None                     0       0      U1-type fingers essential <br />                        type, RNA-binding C2H2                                                                              assembly U1 RNP <br />                        Zn-finger. Probable role                                                                            [132] <br />                        RNA splicing/ <br />                        processing <br />        3073            Predicted RNA-binding        Pre-40S subunit         A      Most archaea,      0       1      Interacts Nop14p is <br />                        protein containing PIN                                      bacteria (COG1412)                      required 40S subunit <br />                        domain involved                                                                              biogenesis 18S rRNA <br />                        18S rRNA processing                                                                                 maturation (11694595). The <br />                                                                                                                            presence PIN domain <br />                                                                                                                            suggests RNA-binding and, <br />                                                                                                                            possibly, RNAse activity <br />        3154            Uncharacterized protein      Pre-40S                A?      Most archaea,         1       X      The general functional <br />                        potential function   subunit?                       bacteria (COG2042)                      prediction stems the <br />                        translation ribosomal                                                                            observation gene for <br />                        biogenesis                                                                                          protein forms a <br />                                                                                                                            predicted conserved operon <br />                                                                                                                            gene ribosomal <br />                                                                                                                            protein L40E several <br />                                                                                                                            archaeal genomes <br />        3214            Small protein containing     ?                      A?      Conserved             1       1 <br />                        Zn-ribbon, possibly                                       Crenarchaeota <br />                        RNA-binding; potential                                      (COG4888) <br />                        role RNA processing <br />                        transcription <br />                        regulation <br />        3800            Predicted E3 ubiquitin       TFIIH                  KO      None                     0       X <br />                        ligase containing RING <br />                        finger, subunit of <br />                        transcription/repair <br />                        factor TFIIH CDK- <br />                        activating kinase <br />                        assembly factor <br />  <br />  <br />  <br />                                                                       Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                             Genome Biology 2004,        Volume 5, Issue 2, Article R7    Koonin et al. R7.15 <br />  <br />  <br />  <br />  <br /> Table 2 (Continued) <br />  <br /> KOGs represented exactly ortholog seven analyzed eukaryotic genomes (examples) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     comment <br /> 3176            Predicted &#206;&#177;-helical          A novel                L?       Conserved           0          X       A function DNA <br />                 protein, possibly involved   complex                    (possibly all) archaea                         replication/repair and/or <br />                 replication/repair;       PCNA involved                   bacteria                            transcription suggested by <br />                 paralog KOG3636           replication?                 (COG1711)                                      analysis genome <br />                                                                                                                             context archaeal <br />                                                                                                                             orthologs form an <br />                                                                                                                             evolutionarily conserved <br />                                                                                                                             association genes <br />                                                                                                                             replication sliding clamp <br />                                                                                                                             (PCNA ortholog) (K.S.M. and <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     reviews <br />                                                                                                                             E.V.K., unpublished work) <br /> 3303            Predicted &#206;&#177;-helical          A novel                L?       Conserved           0           0      A function DNA <br />                 protein, possibly involved   complex                    (possibly all) archaea                         replication/repair and/or <br />                 replication/repair        PCNA involved                   bacteria                            transcription suggested by <br />                 transcription; paralog    replication?                 (COG1711)                                      analysis genome <br />                 KOG3508                                                                                                     context archaeal <br />                                                                                                                             orthologs form an <br />                                                                                                                             evolutionarily conserved <br />                                                                                                                             association genes <br />                                                                                                                             replication sliding clamp <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     reports <br />                                                                                                                             (PCNA ortholog) (K.S.M. and <br />                                                                                                                             E.V.K., unpublished.work) <br /> 0396            Predicted E3 ubiquitin       Ub ligase              O        None                        1           1      The proteins KOG <br />                 ligase                                                                                                      contain modified RING <br />                                                                                                                             domain, be <br />                                                                                                                             capable metal-binding <br />                                                                                                                             similarly U-box <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     deposited research <br />                                                                                                                             domain [133] been <br />                                                                                                                             shown function E3 <br />                                                                                                                             [134] <br /> 1443            Multitransmembrane           ?                      R        Most archaea            1          X <br />                 protein, predicted drug/                                     bacteria (COG0697) <br />                 metabolite transporter <br /> 2647            Multitransmembrane           ?                      R        Most bacteria           0           1 <br />                 protein, potential                                           archaea <br />                 transporter                                                  (COG0628) <br /> 2488            Predicted N-                 ?                      R        Most archaea            1          X       Putative role ribosomal <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     refereed research <br />                 acetyltransferase                                            bacteria (COG0454)                             maturation? <br /> 3347            Predicted nucleotide         ?                      R        Conserved            0           1      Involved oxidative stress <br />                 kinase; nuclear protein                                      archaea                             reponse yeast [135] <br />                 (Fap7p)                                                      bacteria (COG1936) <br /> 3974            Predicted sugar kinase       Putative novel         R        All archaea        1           1      Based fusions seen in <br />                                              complex                    bacteria (COG0063)                             prokaryotes, predicted to <br />                                              KOG2585                                                                        interact functionally and, <br />                                              proteins                                                                       possibly, physically with <br />                                                                                                                             uncharacterized proteins of <br />                                                                                                                             KOG2585 (represented all <br />  <br />                                                                                                                                                                     interactions <br />                                                                                                                             eukaryotes includes <br />                                                                                                                             paralogs species) <br />  <br />  <br /> No functional prediction <br /> 2318            Uncharacterized              ?                      S        None                        0           1 <br />                 conserved protein <br /> 3237            Uncharacterized              ?                      S        None                        0           1      Coiled-coil domains are <br />                 conserved protein                                                                                           involved complex <br />                 containing coiled-coil                                                                                      assembly; an <br />                                                                                                                                                                     information <br />  <br />  <br />  <br />  <br />                 domain                                                                                                      uncharacterized component <br />                                                                                                                             chromatin the <br />                                                                                                                             spliceosome <br />  <br /> *Abbreviations functional categories Figure 3. &#226;&#8364;&#160;0, essential gene (lethal knockout); 1, non-essential gene (non-lethal knockout); X <br /> indicates data available given gene. &#226;&#8364;&#161;Data [85]. &#194;&#167;Data [86]. <br />  <br />  <br />  <br />                                                                Genome Biology 2004, 5:R7 <br />  R7.16 Genome Biology 2004,      Volume 5, Issue 2, Article R7    Koonin et al.                                          http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />                                                                                     pan-eukaryotic set include subunits molecular complexes <br />         4,000                                                                       directly related RNA processing, the <br />         3,500                                                   Ath                 proteasome, TCP-1 chaperonin complex [65] the <br />                                                                 Ecu                 TRAPP complex involved protein trafficking [66]. Alto- <br />         3,000                                                   Sce                 gether, 80% yeast proteins pan- <br />                                                                 Spo <br />         2,500                                                   Cel                 eukaryotic, single-member KOGs belong known macromo- <br />                                                                 Dme                 lecular complexes included MIPS database [67], as <br />         2,000                                                   Hsa                 compared 64% yeast proteins KOGs, <br />         1,500                                                                       moderate statistically highly significant excess <br />                                                                                     (data shown). This preponderance multiprotein com- <br />         1,000                                                                       plex formation single-member pan-eukaryotic <br />           500                                                                       KOGs fully compatible balance hypothesis [64]. <br />  <br />               0                                                                     The unexpected observation regarding single-mem- <br />                    1     2    3     4     5    6     7    8     9 10 &gt;10 <br />                                                                                     ber, pan-eukaryotic KOGs, probably 14 pro- <br />                                                                                     teins, detectable domain WD40 repeat <br />        analyzed <br />        Distribution         genomes <br />                     KOGs <br />        Figure 2eukaryotic        number paralogs           (Table 2). This particularly notable WD40-repeat <br />        Distribution KOGs number paralogs the <br />        analyzed eukaryotic genomes. The species abbreviations Figure 1.   proteins, extremely abundant eukaryotes are <br />                                                                                     present prokaryotic lineages [68], not <br />                                                                                     generally known form well-defined, one-to-one ortholo- <br />        Encephalitozoon, shown comparison                         gous relationships. The WD40 proteins pan-eukaryotic <br />        knockout <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span> data (Table 2 below), pan-                  KOGs listed Table 2 exceptions, probably due <br />        eukaryotic KOGs particular biological importance. For                 unique essential roles assembly RNA- <br />        great majority KOGs (113 131), function              processing complexes. It recently demonstrated <br />        experimentally determined confidently predicted                  that, S. cerevisiae, seven proteins subunits of <br />        varying degree using computational methods                    18S rRNA processosome, involved ribos- <br />        (Table 2). However, 20 KOGs set remained                    omal assembly [69,70]. Taking results the <br />        uncharacterized time analysis and,            unusual phyletic pattern, possible predict with <br />        these, substantial functional inferences drawn <br />        combination sequence-profile analysis, structure <br />        prediction genomic-context analysis prokaryotic                         800 <br />                                                                                                                            All <br />        homologs (Table 2). Some predicted new functions                      700                                  Six seven species <br />        variations well-known themes, predicted <br />                                                                                       600                                  Metazoa <br />        PP-loop ATPases, probably involved novel, <br />        essential RNA modifications (KOGs 2522 2316)                        500 <br />        predicted E3 components ubiquitin ligases (KOGs 0396                        400 <br />        3800). Other predicted functions appear com-                         300 <br />        pletely new, proteins KOG3176 3303 are <br />                                                                                       200 <br />        likely essential components eukaryotic replication <br />        and/or repair systems. Each uncharacterized                       100 <br />        ubiquitous largely essential eukaryotic genes attrac-                   0 <br />        tive target experimental studies.                                                 R S T O K U J A C L I D E G Z P B F HMQ Y N <br />  <br />        Examination experimentally characterized pre-                     Functional <br />                                                                                     Figure 3 breakdown KOGs <br />        dicted functions pan-eukaryotic, single-member KOGs                       Functional breakdown KOGs. Designations functional categories: <br />                                                                                     A, RNA processing modification; B, chromatin structure and <br />        leads interesting conclusions. Nearly functionally                dynamics; C, energy production conversion; D, cell-cycle control and <br />        characterized KOGs set consist proteins                  mitosis; E, amino acid metabolism transport; F, nucleotide metabolism <br />        subunits known multiprotein complexes (Table 2). The                      transport; G, carbohydrate metabolism transport; H, coenzyme <br />        prominent complexes involved rRNA                   metabolism; I, lipid metabolism; J, translation; K, transcription; L, <br />                                                                                     replication repair; M, membrane cell wall structure biogenesis; <br />        processing ribosome assembly, recently dis- <br />                                                                                     O, post-translational modification, protein turnover, chaperone functions; <br />        covered rRNA processosome pre-40S subunit,                   P, inorganic ion transport metabolism; Q, secondary metabolites <br />        spliceosome, various complexes involved                        biosynthesis, transport catabolism; T, signal transduction; U, <br />        transcription (Table 2). Accordingly, set KOGs mark-              intracellular trafficking secretion; Y, nuclear structure; Z, <br />                                                                                     cytoskeleton; R, general functional prediction (typically, prediction of <br />        edly enriched proteins involved various forms RNA <br />                                                                                     biochemical activity), S, function unknown. This breakdown for <br />        processing, assembly ribonucleoprotein (RNP) particles                    KOGs included species. <br />        transcription. In addition, KOGs single-member <br />  <br />                                                                     Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                  Genome Biology 2004,      Volume 5, Issue 2, Article R7      Koonin et al. R7.17 <br />  <br />  <br />  <br />  <br /> considerable confidence WD40 proteins 131- <br /> KOG set remain uncharacterized belong            (a) <br /> similar RNA-processing complexes (Table 2).                           1.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                          comment <br />                                                                       1.6 <br /> With notable exceptions, WD40 proteins,              1.4 <br /> the KOGs single-member, pan-eukaryotic set                1.2 <br /> remarkable patterns evolutionary conservation:              1 <br /> either (nearly) ubiquitous kingdoms life,         0.8 <br /> example, RNA polymerase subunits, universally con-             0.6 <br /> served eukaryotes archaea missing bacteria,             0.4 <br /> such proteins implicated RNA processing <br />                                                                       0.2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                          reviews <br /> (Table 2). Thus, appears elaborate molecular machines <br />                                                                         0 <br /> central functioning eukaryotic cell evolved,             0       0.5         1         1.5          2         2.5         3 <br /> largely ancestral archaeo-eukaryotic components, the <br /> onset eukaryotic evolution, loss duplication                            90% KOGs <br /> of respective genes strongly selected                            1.57/0.43 = 3.7 <br /> throughout rest eukaryotic evolution. <br />                                                                                                  All KOGs <br /> Variation evolutionary rates KOGs                                                    2.71/0.14 = 20.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                          reports <br /> Genome-wide analysis protein evolutionary rates shows           (b) <br /> broad range variation [71]. Here, investigate varia- <br /> tion evolutionary rates ubiquitous KOGs repre-           1.00 <br /> sented seven analyzed genomes connection <br /> between evolutionary rate protein function             0.75 <br /> KOG set. The characteristic evolutionary rate KOG,                                                                              J <br /> which included member(s) Arabidopsis, deter-                                                                                 L <br />  <br />  <br />  <br />  <br />                                                                                                                                                          deposited research <br />                                                                       0.50                                                              U <br /> mined measuring mean evolutionary distance from <br /> Arabidopsis (the outgroup phylogenetic tree;                                                                                 S <br /> below) species. Even KOGs                 0.25 <br /> include seven species and, accordingly, appear repre- <br /> sent conserved core eukaryotic genes, evolutionary         0.00 <br /> rates differ factor 20 fastest- slow-               0     0.5        1         1.5         2        2.5         3 <br /> est-evolving KOGs. Excluding 5% KOGs tail <br /> of distribution leaves fourfold difference in <br />  <br />  <br />  <br />  <br />                                                                                                                                                          refereed research <br />                                                                     Variation <br />                                                                     Figure 4 amino-acid substitution rates KOGs <br /> evolutionary rates (Figure 4a).                                     Variation amino-acid substitution rates KOGs. (a) Probability- <br />                                                                     density function distribution evolutionary rates set of <br /> We compared distributions evolutionary rates        KOGs including seven analyzed eukaryotic species. (b) Distribution <br /> different functional categories KOGs (Tables 3,4 Fig-        functions evolutionary rates different functional categories of <br />                                                                     KOGs. The designations functional categories Figure 3. <br /> ure 4b). Although distributions substantially <br /> overlapped, statistically highly significant differ- <br /> ence evolutionary rates proteins differ- <br /> ent functions (Tables 3,4 Figure 4b). The slowest-              contributions lineage-specific gene loss HGT <br /> evolving proteins involved translation RNA         genome evolution, relative likelihoods these <br /> processing, fastest-evolving ones involved cellular      events remaining uncertain [5,7]. The possibility substan-                        interactions <br /> trafficking transport, components replication        tial HGT major lineages eukaryotes can <br /> and transcription systems intermediate evolutionary            apparently safely disregarded, providing unambigu- <br /> rates (Tables 3,4 Figure 4b).                                   ous parsimonious scenario includes gene loss <br />                                                                     emergence new genes elementary events. <br /> A parsimonious scenario gene loss emergence in <br /> eukaryotic evolution reconstruction ancestral                Some crucial aspects phylogenetic tree eukaryo- <br /> eukaryotic gene sets                                                tic crown group remain matter contention. The consensus <br />                                                                                                                                                          information <br />  <br />  <br />  <br />  <br /> Assuming particular species tree topology, methods evo-        phylogenetic analyses appears point animal- <br /> lutionary parsimony analysis used construct par-        fungal clade clustering microsporidia fungi. <br /> simonious scenario evolution, is, mapping different      However, major uncertainty remains respect the <br /> types evolutionary events branches tree.         topology animal tree: majority studies protein <br /> With prokaryotes, problem confounded major            phylogenies support coelomate (chordate-arthropod) clade <br />  <br />                                                       Genome Biology 2004, 5:R7 <br />  R7.18 Genome Biology 2004,     Volume 5, Issue 2, Article R7    Koonin et al.                                     http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 3                                                                    Table 4 <br />  <br />        Evolutionary rates KOGs different functions: evolutionary          Statistical significance differences evolutionary rates <br />        rates different functional categories KOGs*                         selected functional categories KOGs (t-test) <br />  <br />        Functional            Number        Mean rate,         Standard                             J            L             U                S <br />        category               KOGs           substitutions       deviation <br />                                                site                           J                   - <br />                                                                                   L               3 &#195;&#8212; 10-3          - <br />        J                        227               0.98             0.37 <br />                                                                                   U               1 &#195;&#8212; 10-12     3 &#195;&#8212; 10-4         - <br />        H                        62                0.98             0.30 <br />                                                                                   S               7 &#195;&#8212; 10-33     5 &#195;&#8212; 10-13     2 &#195;&#8212; 10-4            - <br />        A                        167               1.01             0.36 <br />        C                        140               1.01             0.43 <br />        O                        307               1.01             0.40 <br />        F                        50                1.05             0.34 <br />        E                        130               1.07             0.38           In resulting parsimonious scenarios, branch was <br />        L                        139               1.11             0.38           associated gene loss emergence new genes, <br />        B                        56                1.13             0.33           exception plant branch branch leading <br />        Z                        64                1.13             0.46           common ancestor fungi animals, gene <br />        K                        209               1.15             0.42           losses assigned current set genomes <br />        G                        115               1.16             0.43           (Figure 5a,b). There little doubt that, genomes of <br />        I                        110               1.16             0.32           early-branching eukaryotes included, gene loss associated <br />        T                        200               1.18             0.39           branches apparent. The principal fea- <br />        D                        111               1.19             0.40           tures reconstructed scenarios include massive gene loss <br />        R                        415               1.23             0.42           fungal clade, additional elimination numerous <br />        M                        33                1.26             0.47           genes microsporidian; emergence large set new <br />        U                        196               1.27             0.42 <br />                                                                                   genes onset animal clade; subsequent sub- <br />                                                                                   stantial gene loss animal lineages, particularly <br />        Q                        30                1.27             0.37 <br />                                                                                   nematodes arthropods (Figure 5a,b). The esti- <br />        P                        69                1.28             0.45 <br />                                                                                   mated number genes lost S. cerevisiae diver- <br />        N                         2                1.30             0.78 <br />                                                                                   gence common ancestor yeast species, <br />        S                        348               1.40             0.41 <br />                                                                                   S. pombe, closely agreed previous estimate produced <br />        All                     3203               1.16             0.42           different approach [57]. The switch coelomate <br />        *Only KOGs included member(s) Arabidopsis             topology animal sub-tree ecdysozoan topology <br />        analyzed; evolutionary rates average distances     resulted relatively small changes distribution of <br />        Arabidopsis representative given KOG proteins          gains losses: notable difference greater <br />        species (see Material methods details). The functional <br />        categories designated Figure 5.                                  number genes lost nematode lineage smaller <br />                                                                                   number genes lost insect lineage ecdyso- <br />                                                                                   zoan scenario compared coelomate scenario (Figure <br />                                                                                   5a,b). <br />  <br />        [72-74], rRNA phylogeny protein family                    The parsimony analysis described involves explicit <br />        trees point so-called ecdysozoan (arthropod-nema-                   reconstruction gene sets ancestral eukaryotic <br />        tode) clade [75-78]. We treated phyletic pattern               genomes. Under Dollo parsimony model, used <br />        KOG <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span> binary characters (1 presence            analysis, ancestral gene (KOG) set union of <br />        given species 0 absence given KOG)                  KOGs shared respective outgroup each <br />        constructed parsimonious scenarios gene loss                    remaining species. Thus, gene set common <br />        emergence evolution eukaryotic crown group                   ancestor crown group includes KOGs which <br />        coelomate ecdysozoan topologies                Arabidopsis co-occurs analyzed species. <br />        phylogenetic tree. For purpose reconstruction,             Similarly, reconstructed gene set common ances- <br />        Dollo parsimony approach adopted [79]. Under                      tor fungi animals consists KOGs least <br />        approach, gene loss considered irreversible; thus, gene (a            fungal species co-occurs animal species. <br />        KOG member) lost independently evolution-                These conservative reconstructions ancestral gene sets <br />        ary lineages regained. This assumption justi-             because, indicated, gene losses lineages <br />        fied implausibility HGT eukaryotes (the                  branching deepest bifurcation detected. <br />        Dollo approach valid reconstruction prokaryotic              Under conservative approach, 3,413 genes (KOGs) were <br />        ancestors).                                                                assigned common ancestor crown group <br />  <br />                                                                    Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                           Genome Biology 2004,      Volume 5, Issue 2, Article R7    Koonin et al. R7.19 <br />  <br />  <br />  <br />  <br />                   (a)        Dme                     Hsa            Cel           Sce           Spo     Ecu        Ath <br />                          3,491              13,688              4,503       1,679          842 586          3,711 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 comment <br />                            520                 162               541         299           202 1,969            &#226;&#710;&#8217; <br />                                                                                         3,260 <br />                                         5,361                                        267 <br />                                             398                                       55 <br />                                             37       5,000                                      3,048 <br />                                                 1,358                                        15 <br />                                                   193                     3,835             802 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 reviews <br />                                                                       422 <br />                                                                         &#226;&#710;&#8217;                       3,413 <br />  <br />  <br />  <br />  <br />                   (b)        Dme                  Cel                Hsa          Sce           Spo     Ecu        Ath <br />                                               4,503 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 reports <br />                         3,491                                13,688        1,679           842 586          3,711 <br />                          369                   751              114         299            202 1,969            &#226;&#710;&#8217; <br />                                                                                         3,260 <br />                                         5,210                                       267 <br />                                             85                                       55 <br />                                             188 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 deposited research <br />                                                      5,313                                      3,048 <br />                                                 1,671                                       15 <br />                                                  193                     3,835             802 <br />                                                                      422 <br />                                                                        &#226;&#710;&#8217;                        3,413 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 refereed research <br /> Parsimonious <br /> Figure 5      scenarios loss emergence genes (KOGs) eukaryotic evolution <br /> Parsimonious scenarios loss emergence genes (KOGs) eukaryotic evolution. (a) The coelomate topology phylogenetic tree the <br /> eukaryotic crown group. (b) The ecdysozoan topology phylogenetic tree eukaryotic crown group. The numbers boxes indicate the <br /> inferred number KOGs respective ancestral forms. The numbers branches indicate number gene gains (emergence KOGs) <br /> (numerator) gene (KOG) losses (denominator) associated respective branches; dash indicates number losses given branch <br /> could determined. Proteins genome did belong KOGs LSEs counted gains terminal branches. The <br /> species abbreviations Figure 1. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 interactions <br />  <br /> (Figure 5a,b). More realistically, appears likely cer-             The functional profiles gene sets lost differ- <br /> tain number ancestral genes lost all,             ent lineages showed substantial differences (Table 5). Thus, <br /> one, analyzed lineages subsequent evolution,                   example, lineage leading common ancestor of <br /> such gene set eukaryotic crown group ancestor                animals, greatest loss genes assigned to <br /> might close size modern yeasts. In                  functional categories seen amino acid coenzyme <br /> terms functional composition, reconstructed core                  metabolism; contrast, fly nematode, more <br />                                                                                                                                                                 information <br />  <br />  <br />  <br />  <br /> gene set crown-group ancestor resembled                      substantial degradation observed transcription <br /> highly conserved KOGs animal-specific KOGs (Fig-                    factors proteins chaperone-like functions. Genes for <br /> ure 3) enriched housekeeping functions                   proteins involved RNA processing translation are, in <br /> translation, transcription RNA processing (data                      general, heavily affected loss highly <br /> shown).                                                                      degraded parasite E. cuniculi. On occasions, switch <br />  <br />                                                              Genome Biology 2004, 5:R7 <br />  R7.20 Genome Biology 2004,       Volume 5, Issue 2, Article R7         Koonin et al.                                      http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 5 <br />  <br />        Functional profiles genes lost different eukaryotic lineages <br />  <br />        Functional category                                                                            Lost genes (KOGs) <br />  <br />                                                                  Hs*         Dm*       Coelomates/     Ce*      Animals   Sc    Sp    Yeasts    Ec     Fungi-Ec <br />                                                                                         Ecdysozoa <br />  <br />        Total                                                162/114        520/369       37/188       541/751    193      299   202    55      1,969     802 <br />        RNA processing modification                           2/3          9/8          1/2         10/11      4       15     7      1       88       32 <br />        Translation                                               3/3         16/11         0/5         13/10      9        9     6      3      122       10 <br />        Transcription                                             5/2         16/12         0/4         29/33      2       16     9      4       83       40 <br />        Replication repair                                    4/5         28/14        1/15         29/14      2        9     7      3       60       16 <br />        Chromatin structure dynamics                          1/1          8/6          0/2          8/6       0        5     3      1       29       11 <br />        Energy production conversion                          7/10        9/10          5/4         12/10      7        6    13      1      110       37 <br />        Cell cycle control mitosis                            3/3         11/6          0/5         15/11      3       12    3       1       61       16 <br />        Amino acid metabolism transport                       5/6         16/9          1/8         15/7       38       6     9      0      110       18 <br />        Nucleotide metabolism transport                       3/3          6/3          0/3          8/5       5        0     3      1       38        9 <br />        Carbohydrate metabolism transport                     3/3         13/10         1/4         18/14      8       10    16      3       70       41 <br />        Coenzyme metabolism                                       0/2          5/5          2/2         14/12      11       1     1      0       51       12 <br />        Lipid metabolism                                          1/5         27/19        4/12         18/6       4        9    19      2       74       33 <br />        Membrane cell wall structure biogenesis           5/4         10/10         2/2         9/11       7        5     3      0       37       15 <br />        Post-translational modification, protein                  3/5         22/15         2/9         44/40      8       29    21      4      167       69 <br />        turnover, chaperone functions <br />        Inorganic ion transport metabolism                    2/4          8/8          2/2          8/7       9        2     6      4       50       14 <br />        Secondary metabolites biosynthesis, transport             1/2          6/5          1/2          5/3       2        4     1      0       23        5 <br />        catabolism <br />        Signal transduction                                       5/3         32/22        0/10         30/37      4       16     7      3      110       52 <br />        Intracellular trafficking secretion                   4/3         10/8          0/2         14/14      3        5    11      0      116       22 <br />        Nuclear structure                                         0/0          3/3          0/0          5/6       0        1     0      0       16        5 <br />        Cytoskeleton                                              0/0          2/2          0/0          6/8       0        9     0      3       44        6 <br />        General functional prediction (typically,        14/13           79/55        5/29         88/72      30      55    24     11      241       134 <br />        prediction biochemical activity) <br />        Function unknown                                      91/34         184/128        10/66       143/414     37      75    33     10      269       205 <br />  <br />        *For animals, numerator indicates number genes lost coelomate topology species tree denominator <br />        indicates number genes lost ecdysozoan topology tree. <br />  <br />  <br />  <br />  <br />        coelomate ecdysozoan topology replaces                         co-eliminated KOGs apparent (Table 5). Importantly, <br />        independent, parallel losses insect nematode                            different branches complex systems tend be <br />        clades single loss base ecdysozoan branch,                    eliminated parallel different lineages, example, <br />        although, whole, trees based gene content support                        largely non-overlapping sets genes proteins the <br />        coelomate topology [74]. In particular, ecdysozoan                         ubiquitin-proteasome-signalosome systems lost the <br />        topology, unlike coelomate topology, implies early loss                     fungal-microsporidial lineage nematodes (Table <br />        genes involved translation, transcription repair                    6). It likely elimination genes reflects <br />        (Table 6). Notably, large fraction genes lost line-                   independent trends simplification regulatory processes <br />        age general functional prediction prediction                      lineages. <br />        (Table 5). This emphasizes paucity current <br />        understanding lineage-specific gene sets.                                       An interesting trend seen data deterioration of <br />                                                                                           mitochondrial ribosome, occurred several <br />        As noticed previously analysis genes lost                     eukaryotic lineages appears partly parallel <br />        S. cerevisiae divergence common ancestor                        (as occurred independently fungi-microsporidia in <br />        S. pombe, functionally connected genes tend co-                         animals) partly consecutive: early loss ancestral <br />        eliminated evolution [57]. The present study general-                       animal line followed elimination additional genes <br />        izes conclusion functionally coherent groups                       ribosomal proteins individual lineages (Table 6). C. <br />  <br />                                                                           Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                         Genome Biology 2004,    Volume 5, Issue 2, Article R7    Koonin et al. R7.21 <br />  <br />  <br />  <br />  <br /> Table 6 <br />  <br /> Groups functionally linked genes co-eliminated evolution different eukaryotic lineages <br />  <br />  <br />  <br />  <br />                                                                                                                                                             comment <br /> Functional group/ complex                                                          Lost KOGs* <br />  <br />                                        Hs             Dm               Ce         Coelomates/        Animals          Yeasts          Fungi-Ec <br />                                                                                    Ecdysozoa <br />  <br /> Mitochondrial ribosomal           3331, 3435/     3505, 4600,      3505, 4122,    None/ 3505,       0899, 0938,                      0408,1686, <br /> proteins                          3331, 3435      4612/ None       4600, 4612/    4600, 4612        1740, 3254,                      1708, 4707 <br />                                                                       4122                          3278, 4844 <br /> Spliceosome, including putative                   1847, 1960/      1902, 1960,     None/ 1960                                        0105, 0107, <br /> associated proteins                                  1847          2991, 3414                                                        0117, 1365, <br />  <br />  <br />  <br />  <br />                                                                                                                                                             reviews <br />                                                                                                                                      1588, 1676, <br />                                                                                                                                      1847, 1996, <br />                                                                                                                                      2191, 2242, <br />                                                                                                                                      2548, 2991, <br />                                                                                                                                      4207, 4211 <br /> Exosome                                            1004, 1613 <br /> Replication origin-recognition                                     2228, 2538,                                                          4557 <br /> complex                                                               4557 <br /> Mismatch repair                             0218, 0220,     0218, 1977,    None/ 0218, <br />  <br />  <br />  <br />  <br />                                                                                                                                                             reports <br />                                                     221, 1977         4120          1977 <br /> Ubiquitin system/ proteasome-                      0170, 0428,     0168, 0170,    None/ 0170,                       0823, 1645,      0311, 0423, <br /> signalosome components                             1814, 4116,     0320, 0421,      4412                               1734          0427, 0827, <br />                                                    4185, 4412      0423, 1364,                                                       0895, 1100, <br />                                                                    1571, 1645,                                                       1139, 1464, <br />                                                                    1871, 1873,                                                       1571, 1812, <br />                                                                    1887, 2561,                                                       1887, 2561, <br />  <br />  <br />  <br />  <br />                                                                                                                                                             deposited research <br />                                                                    2932, 3061,                                                       2932, 3011, <br />                                                                    3250, 3268,                                                       3050, 3268, <br />                                                                    4146, 4159,                                                       4185, 4248, <br />                                                                    4275, 4412,                                                       4265, 4275, <br />                                                                    4413, 4414,                                                       4413, 4414, <br />                                                                    4692, 4761                                                        4427, 4642, <br />                                                                                                                                      4692, 4761 <br /> NADH-ubiquinone oxido-                                                                                                               2865, 2870, <br /> reductase/ NADH dehydro-                                                                                                             3256, 3300, <br /> genase                                                                                                                               3365, 3382, <br />  <br />  <br />  <br />  <br />                                                                                                                                                             refereed research <br />                                                                                                                                      3389, 3426, <br />                                                                                                                                      3446, 3456, <br />                                                                                                                                      3458, 3466, <br />                                                                                                                                      3468, 4009, <br />                                                                                                                                      4662, 4668, <br />                                                                                                                                      4669, 4770, <br />                                                                                                                                         4845 <br />  <br /> *For animals, numerator indicates KOGs lost coelomate topology species tree, denominator indicates <br /> KOGs lost ecdysozoan topology. <br />  <br />  <br />  <br />                                                                                                                                                             interactions <br />  <br />  <br />  <br /> elegans shortest mitochondrial rRNAs                     Evolutionary relationships eukaryotic and <br /> might &apos;minimal&apos; mitochondrial ribosome [80];                     prokaryotic orthologous gene sets <br /> present analysis details stages leading ultimate                The prokaryotic COGs eukaryotic KOGs identified <br /> degradation mitochondrial ribosome.                                  separate genome comparisons, overlap existed <br />                                                                             sets included unicellular eukaryotes, namely <br />                                                                                                                                                             information <br />  <br />  <br />  <br />  <br /> An exhaustive analysis patterns gene loss               yeasts microsporidian. To identify the <br /> the scope work. It clear potential             prokaryotic counterparts KOGs, sequences the <br /> improving understanding eukaryotic evolution                     eukaryotic proteins included KOGs compared <br /> functional predictions examination co-eliminated                 using RPS-BLAST program position-specific scor- <br /> gene groups.                                                                ing matrices (PSSMs) constructed prokaryotic COGs <br />  <br />                                                            Genome Biology 2004, 5:R7 <br />  R7.22 Genome Biology 2004,       Volume 5, Issue 2, Article R7       Koonin et al.                                 http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />                                                                                        Clearly, reconstructed gene set common ancestor of <br />         (a)                                                                            crown group and, particularly, pan-eukaryotic KOGs <br />         6,000                                                                          enriched ancient KOGs (those prokaryotic <br />                                                        Non-prokaryotes                 counterparts) compared KOG collection. In con- <br />         5,000                                          Prokaryote                      trast, KOGs inferred evolved indi- <br />                                                                                        vidual lineages crown group, significantly lower <br />         4,000                                                                          fraction detectable prokaryotic counterparts (Figure 6a). <br />  <br />         3,000                                                                          Early evolution eukaryotes known involved <br />                                                                                        duplication ancient genes inherited prokaryotes [82], <br />         2,000                                                                          apparent KOGs COGs comparison. <br />                        44%                                                             Although one-to-one relationships predominant, in <br />         1,000                     54% <br />                                                                                        30% cases, eukaryotic KOGs <br />                                                28%        28%          66%             corresponded prokaryotic COG (Figure 6b). This <br />                0                                                                       indicates extensive duplication ancestral genes early <br />                        All       CGA Non-CGA Msp                     Seven- <br />                                                                      species           stages eukaryotic evolution; moreover, substantial frac- <br />         (b)                                                                            tion genes undergone repeated duplications, <br />                                                                                        resulting one-to-many relationship prokaryotic <br />                                                                                        eukaryotic orthologs (Figure 6b). <br />                                                                              900 <br />                                                                                        An in-depth analysis relationships eukaryotic <br />                                                                              800 <br />                                                                                        prokaryotic orthologous gene clusters include an <br />                                                                              700       attempt decipher evolutionary history, is, <br />                                                                              600       classification C/KOGs represented eukaryotes <br />                                                                             500        prokaryotes into: inherited the <br />                                                                             400        universal common ancestor; archaeo-eukaryotic <br />                                                                                        subset; shared HGT between <br />                                                                             300 <br />                                                                                        bacteria eukaryotes various stages eukaryotic evolu- <br />                                                                            200 <br />                                                                                        tion. This analysis scope present work. <br />                                                                            100         Perhaps principal message stress that, using a <br />            1 <br />                                                                            0           fairly sensitive sequence comparison method, prokaryotic <br />                    3                                                   1               homologs detected 44% eukary- <br />                                                                  2 <br />                              5                             3                           otic KOGs, fraction increased 54% for <br />                Number                                 4        Number                  genes traced common ancestor <br />                COGs                          5              KOGs <br />                                           &gt;5                                           crown group (Figure 6a). This observation emphasizes <br />                                                                                        major innovation accompanied emer- <br />                                                                                        gence early evolution eukaryotes; KOGs for <br />        Correspondence <br />        sets <br />        Figure   6        eukaryotic prokaryotic orthologous gene           prokaryotic counterparts eventually identified <br />        Correspondence eukaryotic prokaryotic orthologous gene <br />        sets. (a) Representation prokaryotic counterparts different subsets       sensitive sequence structure comparison <br />        KOGs. CGA, crown group ancestor; non-CGA, KOGs represented               apparently experienced rapid evolution the <br />        crown group ancestor; MSP, metazoa-specific KOGs. (b) Evidence           prokaryote-eukaryote transition. <br />        ancient duplications eukaryotic genes revealed KOGs against <br />        COGs comparison. The connections KOGs COGs detected <br />        using RPS-BLAST (see text) analyzed single linkage clustering.       Phyletic patterns KOGs dispensability yeast <br />                                                                                        worm genes <br />                                                                                        There 860 KOGs representative from <br />        ([81] Materials methods details). The results                  seven analyzed genomes. In accord the <br />        checked manually comparing assignment                        &apos;knockout rate&apos; hypothesis [83], largely sup- <br />        proteins unicellular eukaryotes ortholo-                    ported recent, genome-wide analysis gene conservation <br />        gous gene sets. Altogether, probable orthologous relation-                      [38,84], expected highly conserved <br />        ships established 2,456 eukaryotic KOGs                        genes essential survival eukaryotic organisms. <br />        TWOGs (44% total) 1,516 prokaryotic COGs. A                          This appears particularly plausible given near-minimal <br />        detailed breakdown relationships                            eukaryotic gene complement microsporidian. The <br />        eukaryotic prokaryotic orthologous gene clusters                      prediction test using recently published <br />        reveal important evolutionary trends. Figure 6a compares                    functional profile yeast S. cerevisiae genome, which <br />        occurrence prokaryotic counterparts entire set                    includes data growth rates homozygous deletion <br />        eukaryotic KOGs subsets conserved different levels.                  strains 96% open reading frames (ORFs) the <br />  <br />                                                                         Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                          Genome Biology 2004,   Volume 5, Issue 2, Article R7   Koonin et al. R7.23 <br />  <br />  <br />  <br />  <br />                                                                             seven species. This translates statistically highly <br />  (a)                                                                        significant dependence gene&apos;s (in)dispensability <br />                            S. cerevisiae <br />                                                                             conservation long evolutionary distances. The prob- <br />  100% <br />  <br />  <br />  <br />  <br />                                                                                                                                                            comment <br />                                                                             ability set highly conserved genes enriched <br />                                                 221                         essential genes result chance estimated at <br />                     1,463 <br />    75%                                          115                         &lt;&lt;10-10. Notably, greater enrichment essential <br />                                                                    1        genes seen KOGs represented by <br />                                                                    2-5 <br />                                                 273                         one, one, ortholog seven analyzed <br />    50%              1,120                                          6 <br />                                                                             genomes: 131 KOGs, 98 (75%) included essen- <br />                                                                    7 <br />                                                                             tial yeast gene (Table 2). Such preponderance essential <br />    25%              1,004                                                   genes expected because, set KOGs, the <br />                                                 497 <br />  <br />  <br />  <br />  <br />                                                                                                                                                            reviews <br />                                                                             indispensability respective function have <br />                      717                                                    masked presence paralogs. <br />     0% <br />                        N                          E <br />                                                                             For additional set 15% non-essential yeast <br />  (b)                        C. elegans                                      genes, knockout results measurable retardation growth <br /> 100%                                                                        [85]. Unexpectedly contrast result obtained with <br />                                                                             essential genes, failed observe correlation between <br />                                                  163 <br />                                                                             magnitude gene&apos;s knockout effect yeast growth and <br />  <br />  <br />  <br />  <br />                                                                                                                                                            reports <br />   75%                                                                       phyletic pattern (data shown). This indicate <br />                    7,282                         181                        measured effect yeast growth translate <br />                                                                     1       effect fitness loss ortholog the <br />                                                                     2-5     given gene distant species. <br />   50%                                            154                6 <br />                                                                     7       In C. elegans, yeast, essentiality genes appears <br />                     3,602                                                   correlate strong evolutionary conservation, already <br />  <br />  <br />  <br />  <br />                                                                                                                                                            deposited research <br />   25% <br />                                                  312                        noticed recent genome-wide study inhibition of <br />                      917                                                    worm gene expression <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) [86]. We <br />     0%               736                                                    compared dataset, covers 86% C. ele- <br />                       N                           E                         gans genes, phyletic patterns respective KOGs. <br />                                                                             Of essential worm genes, 38% conserved seven <br /> respective <br /> Figure <br /> Gene   dispensability <br />          7 KOGs yeast worm phyletic patterns              compared species 19% conserved species (Fig- <br /> Gene dispensability yeast worm phyletic patterns          ure 7b). In contrast, 6% non-essential C. elegans <br /> respective KOGs. (a) Distribution essential non-essential genes      genes represented seven species 7% con- <br />  <br />  <br />  <br />  <br />                                                                                                                                                            refereed research <br /> among different size classes KOGs LSEs yeast Saccharomyces        served species (Figure 7b). Thus, a <br /> cerevisiae. (b) Distribution essential non-essential genes among <br /> different size classes KOGs LSEs nematode C. elegans. The <br />                                                                             strong robust connection gene&apos;s essentiality <br /> number species KOGs LSEs color-coded indicated       tendency conserved evolution wide span <br /> the right plot.                                                     taxa; connection established using two <br />                                                                             independent datasets biologically extremely different <br />                                                                             model organisms. <br /> yeast genome [85]. Growth rates previously inter- <br /> preted measure fitness [84].                                        Domain accretion orthologous sets eukaryotic <br />                                                                             proteins <br /> When phyletic patterns KOGs superimposed                    As noticed previously, complexity domain architecture                   interactions <br /> on data gene dispensability (with essential genes oper-              proteins orthologous sets increases increasing <br /> ationally defined deletion lethal effect            organismic complexity; phenomenon dubbed <br /> a rich medium) [85], 45% essential                 domain accretion [3]. With KOG set hand, sought to <br /> genes conserved seven species 25% rep-                 assess extent accretion quantitatively using data <br /> resented species (typically exception E.                 presence domains CDD (conserved domain <br /> cuniculi); 15% essential yeast genes orthologs             alignments database) collection KOG members. <br /> the analyzed genomes (Figure 7a). In striking con-                  The results summarized Table 7 relatively small but <br />                                                                                                                                                            information <br />  <br />  <br />  <br />  <br /> trast, non-essential genes, 16.5% repre-                    statistically significant excess domains proteins from <br /> sented compared genomes 28.5% detectable                  multicellular organisms compared orthologs from <br /> orthologs (Figure 7a). The reciprocal comparison equally                 unicellular organisms. Furthermore, multicellular <br /> illustrative: essential genes composed 18.5% entire set              eukaryotes, human proteins greatest complexity of <br /> of yeast genes 35% genes (KOGs) represented               domain architectures, followed Drosophila and <br />  <br />                                                               Genome Biology 2004, 5:R7 <br />  R7.24 Genome Biology 2004,     Volume 5, Issue 2, Article R7   Koonin et al.                                        http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        Table 7 <br />  <br />        Domain accretion complex eukaryotes <br />  <br />                                 Hsa                Dme               Ath                Cel               Sce               Spo                Ecu <br />  <br />        Hsa                                      &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10         &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10 <br />  <br />  <br />                                 470 <br />        Dme                      3214                               2 &#195;&#8212; 10-1         &lt;1 &#195;&#8212; 10-10         &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10 <br />                                 805 <br />  <br />  <br />                                 327                354 <br />        Ath                      2224               2085                              3 &#195;&#8212; 10-1          &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10 <br />                                 530                403 <br />  <br />  <br />                                 347                428               334 <br />        Cel                      2986               2962             2052                                1 &#195;&#8212; 10-8         &lt;1 &#195;&#8212; 10-10        &lt;1 &#195;&#8212; 10-10 <br />                                 880                650               376 <br />  <br />  <br />                                 149                161               183               197 <br />        Sce                      1789               1704             1769               1715                               1 &#195;&#8212; 10-2         &lt;1 &#195;&#8212; 10-10 <br />                                 504                411               374               336 <br />  <br />  <br />                                 100                123               135               150                158 <br />        Spo                      1880               1807             1886               1808              2360                              &lt;1 &#195;&#8212; 10-10 <br />                                 549                426               388               359                216 <br />  <br />  <br />                                  10                 17                12                14                13                 19 <br />        Ecu                      700                738               739               748                816               835 <br />                                 332                254               235               244                158               140 <br />  <br />        For given pair species numbers cell diagonal represent, bottom: number KOGs average <br />        number detected domains CDD collection (cut-off E = 10-3) proteins species left greater the <br />        species right; number KOGs equal average number domains; number KOGs average number domains is <br />        greater species right (for example, D. melanogaster greater number detected domains H. sapiens 470 KOGs, same <br />        number 3,214 KOGs, smaller number 805 KOGs). The numbers diagonal statistical significance difference, P(&#207;&#8225;2). <br />  <br />  <br />  <br />  <br />        Arabidopsis (Table 6), agreement preliminary results                phyletic patterns. In addition, evaluated range vari- <br />        reported previously. Among unicellular eukaryotes,                      ation evolutionary rates genes different functional cat- <br />        Encephalitozoon far complex domain archi-                  egories obtained statistical support important <br />        tectures (Table 6), reflects general genome reduc-                evolutionary     phenomenon            domain      accretion. <br />        tion intracellular parasite.                                        Furthermore, observed minority eukaryotic <br />                                                                                    KOGs readily detectable prokaryotic counterparts, <br />                                                                                    emphasizes extent innovation linked ori- <br />        Conclusions                                                                 gin eukaryotes subsequent major transitions in <br />        The present analysis KOGs provides quantitative backing                  eukaryotic evolution, origin multicellularity <br />        trends evolution eukaryotic genomes                 origin animals. <br />        previously noticed general, qualitative level. <br />        The important quantities reported include size              The case study KOGs represented just one <br />        conserved core eukaryotic genes, conservative recon-                 member eukaryotic genomes compared shows the <br />        structions ancestral gene sets, numbers genes                potential KOGs functional prediction inferring the <br />        appear lost gained individual eukaryotic                probable functions KOGs set had <br />        lineages, extent correlation gene dispen-                remained uncharacterized. This analysis revealed <br />        sability evolutionary conservation, reflected               unexpected facets evolution widespread essential <br />  <br />                                                                   Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                  Genome Biology 2004,   Volume 5, Issue 2, Article R7   Koonin et al. R7.25 <br />  <br />  <br />  <br />  <br /> eukaryotic proteins, counterintutitive preponder-       Annotation KOGs included critical assessment anno- <br /> ance WD40-repeat proteins single-member                tations available GenBank, public databases <br /> pan-eukaryotic KOGs.                                                primary literature additional, in-depth sequence <br />  <br />  <br />  <br />  <br />                                                                                                                                                    comment <br />                                                                     analysis aimed detection previously unnoticed homolo- <br /> The current KOG set includes proteins seven genomes            gous relationships. The annotated functions KOGs were <br /> whose sequences available 1 July, 2002. The              classified 23 categories (see legend Figure 3), which <br /> genomes mouse [87], fugu fish [88], Anopheles        adapted functional classification previously <br /> mosquito [89], urochordate Ciona instestinalis [90]         used COGs [24] including specific eukaryotic <br /> the malarial parasite Plasmodium falciparum [91]               categories. <br /> become available included, partly <br /> because problems protein annotation             Other sequence analysis procedures <br />  <br />  <br />  <br />  <br />                                                                                                                                                    reviews <br /> these genomes, partly time-consuming             During KOG annotation, proteins currently anno- <br /> labor-intensive nature KOG analysis. Inclusion          tated &apos;hypothetical&apos; &apos;unknown&apos;, vague <br /> and newly sequenced genomes proceed               suspect annotation, subject additional sequence <br /> faster rate itself established, ena-    analysis, included iterative sequence similarity <br /> ble further, deeper studies functional evolution-      searches PSI-BLAST program [92], RPS-BLAST <br /> ary patterns eukaryotic life.                                    searches conserved domains [80], additional domain <br />                                                                     architecture analysis using SMART [95]. To esti- <br />                                                                     mate sequence evolution rates, multiple alignments KOGs <br />  <br />  <br />  <br />  <br />                                                                                                                                                    reports <br /> Materials methods                                               constructed using MAP program [96] the <br /> Construction annotation KOGs                                 pairwise evolutionary distances calculated the <br /> A detailed description procedures employed          maximum likelihood method PAM model using <br /> this purpose presented [25]. The protein sets      PROTDIST program PHYLIP package [97]. When a <br /> all eukaryotic species, exception C. elegans H.     KOG included member given species, <br /> sapiens, genome division National              paralog greatest average similarity proteins <br /> Center Biotechnology Information (NCBI). The protein            organisms selected represent species in <br />  <br />  <br />  <br />  <br />                                                                                                                                                    deposited research <br /> sequences C. elegans WormPep67 database           given KOG. Since A. thaliana likely outgroup <br /> and human sequences NCBI build 30. Briefly,           species analyzed set eukaryotes, distances the <br /> the KOG construction protocol included: First, detection        Arabidopsis representative proteins species <br /> and masking common, repetitive domains using RPS-            averaged estimate characteristic evolutionary dis- <br /> BLAST program PSSMs respective domains              tance given KOG. Data KOGs excessive var- <br /> from CDD collection [81]; second, all-against-all compar-       iability distances A. thaliana species <br /> ison protein sequences analyzed genomes using           (standard deviation mean ratio &gt; 0.5) discarded. As <br /> the BLASTP program [92], masking low sequence               divergence times KOGs presumed the <br />  <br />  <br />  <br />  <br />                                                                                                                                                    refereed research <br /> complexity regions using SEG program [93]; third, iden-         (and equal time elapsed common <br /> tification triangles mutually consistent BeTs; merging        ancestor eukaryotic crown group), mean evolu- <br /> triangles BeTs common form preliminary            tionary distance KOG measure KOG&apos;s evolu- <br /> KOGs; forth, adding members co-orthologous sets missed           tionary rate. <br /> at previous step using COGNITOR procedure [24]; fifth, <br /> manual examination candidate KOG, aimed elimi-           The parsimonious evolutionary scenario, included <br /> nating false positives incorporated KOGs        gene losses emergence KOGs mapped branches <br /> automatic procedure inclusion false negatives           eukaryotic phylogenetic tree, constructed using <br /> were missed originally; sixth, assignment proteins contain-      DOLLOP program PHYLIP package [97]; this <br /> ing promiscuous domains masked step Fuzzy           program based Dollo parsimony method,                          interactions <br /> Orthologous Groups (FOGs), named respective               assumes irreversibility character loss [79]. <br /> domains (when sequence assigned KOG contained one <br /> or masked domains, sequences domains              For analysis domain accretion, conserved domains <br /> were restored); finally, examination largest prelim-     NCBI CDD database detected eukaryotic <br /> inary KOGs, included numerous proteins            proteins belonged KOGs using RPS-BLAST <br /> several genomes using phylogenetic trees, cluster analysis       program [81] E-value cut-off 0.001. Domains with <br /> with BLASTCLUST program [94], comparison domain              biased amino acid sequence composition, tend pro- <br />                                                                                                                                                    information <br />  <br />  <br />  <br />  <br /> architectures, visual inspection alignments. As result,    duce high false-positive rate RPS-BLAST searches, were <br /> some preliminary KOGs split          excluded analysis. <br /> smaller final KOGs. <br />  <br />  <br />  <br />  <br />                                                       Genome Biology 2004, 5:R7 <br />  R7.26 Genome Biology 2004,       Volume 5, Issue 2, Article R7       Koonin et al.                                          http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        The eukaryotic KOG set accessible [98] ftp                             UT, Rao BS, Kiryutin B, Galperin MY, Fedorova ND, Koonin EV: The <br />        [99]. The reconstructed ancestral gene sets available                            COG database: new developments phylogenetic classifica- <br />                                                                                                tion proteins complete genomes. Nucleic Acids Res <br />        [100].                                                                                  2001, 29:22-28. <br />                                                                                          25.   Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, Koonin <br />                                                                                                EV, Krylov DM, Mazumder R, Mekhedov SL, Nikolskaya AN et al.: <br />                                                                                                The COG database: updated version includes eukaryotes. <br />        Acknowledgements                                                                        BMC Bioinformatics 2003, 4:41. <br />        We thank Roman Tatusov major contribution construction             26.   Natale DA, Shankavaram UT, Galperin MY, Wolf YI, Aravind L, <br />        KOGs, Igor Garkavtsev participation initial stages         Koonin EV: Towards understanding genome <br />        KOG project, L. Aravind Wei Yang useful discussions shar-               sequence crenarchaeon genome annotation using <br />        ing unpublished observations.                                                     clusters orthologous groups proteins (COGs). Genome <br />                                                                                                Biol 2000, 1:research0009.1-0009.19. <br />                                                                                          27.   Nolling J, Breton G, Omelchenko MV, Makarova KS, Zeng Q, Gibson <br />                                                                                                R, Lee HM, Dubois J, Qiu D, Hitti J et al.: Genome sequence and <br />        References                                                                              comparative analysis solvent-producing bacterium <br />        1.    Doolittle WF: Lateral genomics. Trends Cell Biol 1999, 9:M5-M8.                   Clostridium acetobutylicum. J Bacteriol 2001, 183:4823-4838. <br />        2.    Doolittle WF: Phylogenetic classification universal                 28.   McClelland M, Sanderson KE, Spieth J, Clifton SW, Latreille P, Court- <br />              tree. Science 1999, 284:2124-2129.                                                ney L, Porwollik S, Ali J, Dante M, Du F et al.: Complete genome <br />        3.    Koonin EV, Aravind L, Kondrashov AS: The impact compara-                       sequence Salmonella enterica serovar Typhimurium LT2. <br />              tive genomics understanding evolution. Cell 2000,                       Nature 2001, 413:852-856. <br />              101:573-576.                                                                29.   Slesarev AI, Mezhevaya KV, Makarova KS, Polushin NN, Shcherbinina <br />        4.    Koonin EV, Makarova KS, Aravind L: Horizontal gene transfer                    OV, Shakhova VV, Belova GI, Aravind L, Natale DA, Rogozin IB et al.: <br />              prokaryotes: quantification classification. Annu Rev                          The complete genome hyperthermophile Methanopyrus <br />              Microbiol 2001, 55:709-742.                                                       kandleri AV19 monophyly archaeal methanogens. Proc <br />        5.    Snel B, Bork P, Huynen MA: Genomes flux: evolution                      Natl Acad Sci USA 2002, 99:4644-4649. <br />              archaeal proteobacterial gene content. Genome Res 2002,                 30.   Cort JR, Koonin EV, Bash PA, Kennedy MA: A phylogenetic <br />              12:17-25.                                                                         approach target selection structural genomics: solu- <br />        6.    Gogarten JP, Doolittle WF, Lawrence JG: Prokaryotic evolution                  tion structure YciH. Nucleic Acids Res 1999, 27:4018-4027. <br />              light gene transfer. Mol Biol Evol 2002, 19:2226-2238.                   31.   Brenner SE: Target selection structural genomics. Nat Struct <br />        7.    Mirkin BG, Fenner TI, Galperin MY, Koonin EV: Algorithms                      Biol 2000, 7(Suppl):967-969. <br />              computing parsimonious evolutionary scenarios genome                    32.   Gerstein M: Integrative database analysis structural <br />              evolution, universal common ancestor domi-                           genomics. Nat Struct Biol 2000, 7(Suppl):960-963. <br />              nance horizontal gene transfer evolution                       33.   Galperin MY, Koonin EV: Searching drug targets micro- <br />              prokaryotes. BMC Evol Biol 2003, 3:2.                                             bial genomes. Curr Opin Biotechnol 1999, 10:571-578. <br />        8.    Fitch WM: Distinguishing homologous analogous                          34.   Buysse JM: The role genomics antibacterial target <br />              proteins. Syst Zool 1970, 19:99-106.                                              discovery. Curr Med Chem 2001, 8:1713-1726. <br />        9.    Fitch WM: Homology personal view                           35.   Jordan IK, Kondrashov FA, Rogozin IB, Tatusov RL, Wolf YI, Koonin <br />              problems. Trends Genet 2000, 16:227-231.                                          EV: Constant relative rate protein evolution detection <br />        10.   Henikoff S, Greene EA, Pietrokovski S, Bork P, Attwood TK, Hood L:                functional diversification bacterial, archaeal and <br />              Gene families: taxonomy protein paralogs                               eukaryotic proteins. Genome Biol 2001, 2:research0053.1-0053.9. <br />              chimeras. Science 1997, 278:609-614.                                        36.   Yanai I, Derti A, DeLisi C: Genes linked fusion events gen- <br />        11.   Sonnhammer EL, Koonin EV: Orthology, paralogy proposed                        erally functional category: systematic analysis <br />              classification paralog subtypes. Trends Genet 2002,                           30 microbial genomes. Proc Natl Acad Sci USA 2001, <br />              18:619-620.                                                                       98:7940-7945. <br />        12.   Wilson CA, Kreychman J, Gerstein M: Assessing annotation                    37.   Lecompte O, Ripp R, Puzos-Barbe V, Duprat S, Heilig R, Dietrich J, <br />              transfer genomics: quantifying relations                          Thierry JC, Poch O: Genome evolution genus level: com- <br />              protein sequence, structure function traditional                      parison complete genomes hyperthermophilic <br />              probabilistic scores. J Mol Biol 2000, 297:233-249.                           archaea. Genome Res 2001, 11:981-993. <br />        13.   Koonin EV, Galperin MY: Sequence-Evolution-Function. Computational          38.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes are <br />              Approaches Comparative Genomics New York: Kluwer Academic                      evolutionarily conserved nonessential genes <br />              Publishers; 2002.                                                                 bacteria. Genome Res 2002, 12:962-968. <br />        14.   Pauling L, Zuckerkandl E: Chemical paleogenetics. Molecular                 39.   Remm M, Storm CE, Sonnhammer EL: Automatic clustering of <br />              &quot;restoration studies&quot; extinct forms life. Acta Chem Scand                   orthologs in-paralogs pairwise species <br />              1963, 17:S9-S16.                                                                  comparisons. J Mol Biol 2001, 314:1041-1052. <br />        15.   Ohno S: Evolution Gene Duplication Berlin-Heidelberg-New York:           40.   Gaasterland T, Ragan MA: Microbial genescapes: phyletic and <br />              Springer-Verlag; 1970.                                                            functional patterns ORF distribution prokaryotes. <br />        16.   Lynch M, Force A: The probability duplicate gene preserva-                     Microb Comp Genomics 1998, 3:199-217. <br />              tion subfunctionalization. Genetics 2000, 154:459-473.                   41.   Pellegrini M, Marcotte EM, Thompson MJ, Eisenberg D, Yeates TO: <br />        17.   Sicheritz-Ponten T, Andersson SG: A phylogenomic approach                      Assigning protein functions comparative genome analy- <br />              microbial evolution. Nucleic Acids Res 2001, 29:545-552.                          sis: protein phylogenetic profiles. Proc Natl Acad Sci USA 1999, <br />        18.   Zmasek CM, Eddy SR: RIO: Analyzing proteomes auto-                             96:4285-4288. <br />              mated phylogenomics using resampled inference                            42.   Galperin MY, Koonin EV: Who&apos;s neighbor? New <br />              orthologs. BMC Bioinformatics 2002, 3:14.                                         computational approaches functional genomics. Nat <br />        19.   Storm CE, Sonnhammer EL: Automated ortholog inference                             Biotechnol 2000, 18:609-613. <br />              phylogenetic trees calculation orthology                        43.   Myllykallio H, Lipowski G, Leduc D, Filee J, Forterre P, Liebl U: An <br />              reliability. Bioinformatics 2002, 18:92-99.                                       alternative flavin-dependent mechanism thymidylate <br />        20.   Tatusov RL, Koonin EV, Lipman DJ: A genomic perspective                        synthesis. Science 2002, 297:105-107. <br />              protein families. Science 1997, 278:631-637.                                44.   Levesque M, Shasha D, Kim W, Surette MG, Benfey PN: Trait-to- <br />        21.   Huynen MA, Bork P: Measuring genome evolution. Proc Natl Acad                     Gene. A computational method predicting function <br />              Sci USA 1998, 95:5849-5856.                                                       uncharacterized genes. Curr Biol 2003, 13:129-133. <br />        22.   Montague MG, Hutchison CA 3rd: Gene content phylogeny                    45.   Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, <br />              herpesviruses. Proc Natl Acad Sci USA 2000, 97:5334-5339.                         Devon K, Dewar K, Doyle M, FitzHugh W et al.: Initial sequencing <br />        23.   Tatusov RL, Galperin MY, Natale DA, Koonin EV: The COG data-                      analysis human genome. Nature 2001, 409:860-921. <br />              base: tool genome-scale analysis protein functions                 46.   Adams MD, Celniker SE, Holt RA, Evans CA, Gocayne JD, Amanati- <br />              evolution. Nucleic Acids Res 2000, 28:33-36.                                  des PG, Scherer SE, Li PW, Hoskins RA, Galle RF et al.: The <br />        24.   Tatusov RL, Natale DA, Garkavtsev IV, Tatusova TA, Shankavaram                    genome sequence Drosophila melanogaster. Science 2000, <br />                                                                                                287:2185-2195. <br />  <br />                                                                         Genome Biology 2004, 5:R7 <br />  http://genomebiology.com/2004/5/2/R7                                          Genome Biology 2004,       Volume 5, Issue 2, Article R7     Koonin et al. R7.27 <br />  <br />  <br />  <br />  <br /> 47.   The C. elegans Sequencing Consortium: Genome sequence                 417:967-970. <br />       nematode C. elegans : platform investigating biology.            71.   Grishin NV, Wolf YI, Koonin EV: From complete genomes to <br />       Science 1998, 282:2012-2018.                                                 measures substitution rate variability between <br /> 48.   Arabidopsis Genome Initiative: Analysis genome sequence               proteins. Genome Res 2000, 10:991-1000. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  comment <br />       flowering plant Arabidopsis thaliana. Nature 2000,              72.   Hedges SB: The origin evolution model organisms. Nat <br />       408:796-815.                                                                 Rev Genet 2002, 3:838-849. <br /> 49.   Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B, Feldmann H,        73.   Blair JE, Ikeo K, Gojobori T, Hedges SB: The evolutionary position <br />       Galibert F, Hoheisel JD, Jacq C, Johnston M et al.: Life 6000           nematodes. BMC Evol Biol 2002, 2:7. <br />       genes. Science 1996, 274:563-547.                                      74.   Wolf YI, Rogozin IB, Koonin EV: Coelomata Ecdysozoa: <br /> 50.   Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, Stewart A,                evidence genome-wide phylogenetic analysis. Genome <br />       Sgouros J, Peat N, Hayles J, Baker S et al.: The genome sequence             Res 2004, 14:29-36. <br />       Schizosaccharomyces pombe. Nature 2002, 415:871-880.                75.   Aguinaldo AM, Turbeville JM, Linford LS, Rivera MC, Garey JR, Raff <br /> 51.   Katinka MD, Duprat S, Cornillot E, Metenier G, Thomarat F, Prensier          RA, Lake JA: Evidence clade nematodes, arthropods <br />       G, Barbe V, Peyretaillade E, Brottier P, Wincker P et al.: Genome            moulting animals. Nature 1997, 387:489-493. <br />       sequence gene compaction eukaryote parasite                 76.   Rosa R, Grenier JK, Andreeva T, Cook CE, Adoutte A, Akam M, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  reviews <br />       Encephalitozoon cuniculi. Nature 2001, 414:450-453.                          Carroll SB, Balavoine G: Hox genes brachiopods pri- <br /> 52.   Lespinet O, Wolf YI, Koonin EV, Aravind L: The role lineage-              apulids protostome evolution. Nature 1999, 399:772-776. <br />       specific gene family expansion evolution                     77.   Mallatt J, Winchell CJ: Testing new animal phylogeny: first <br />       eukaryotes. Genome Res 2002, 12:1048-1059.                                   use combined large-subunit small-subunit rRNA gene <br /> 53.   Clusters orthologous groups eukaryotic complete                       sequences classify protostomes. Mol Biol Evol 2002, <br />       genomes [http://www.ncbi.nlm.nih.gov/COG/new/shokog.cgi]                     19:289-301. <br /> 54.   Yudate HT, Suwa M, Irie R, Matsui H, Nishikawa T, Nakamura Y,          78.   Peterson KJ, Eernisse DJ: Animal phylogeny ancestry of <br />       Yamaguchi D, Peng ZZ, Yamamoto T, Nagai K et al.: HUNT: launch               bilaterians: inferences morphology 18S rDNA gene <br />       full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> cDNA database Helix Research                       sequences. Evol Dev 2001, 3:170-205. <br />       Institute. Nucleic Acids Res 2001, 29:185-188.                         79.   Farris JS: Phylogenetic analysis Dollo&apos;s Law. Syst Zool <br /> 55.   Misra S, Crosby MA, Mungall CJ, Matthews BB, Campbell KS, Hra-               1977, 26:77-88. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  reports <br />       decky P, Huang Y, Kaminker JS, Millburn GH, Prochnik SE et al.:        80.   Mears JA, Cannone JJ, Stagg SM, Gutell RR, Agrawal RK, Harvey SC: <br />       Annotation Drosophila melanogaster euchromatic                        Modeling minimal ribosome based comparative <br />       genome: systematic review. Genome Biol 2002,                               sequence analysis. J Mol Biol 2002, 321:215-234. <br />       3:research0083.1-0083.22.                                              81.   Marchler-Bauer A, Anderson JB, DeWeese-Scott C, Fedorova ND, <br /> 56.   Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES: Sequencing           Geer LY, He S, Hurwitz DI, Jackson JD, Jacobs AR, Lanczycki CJ et <br />       comparison yeast species identify genes regu-                  al.: CDD: curated Entrez database conserved domain <br />       latory elements. Nature 2003, 423:241-254.                                   alignments. Nucleic Acids Res 2003, 31:383-387. <br /> 57.   Aravind L, Watanabe H, Lipman DJ, Koonin EV: Lineage-specific          82.   Brown JR, Doolittle WF: Archaea prokaryote-to- <br />       loss divergence functionally linked genes                          eukaryote transition. Microbiol Mol Biol Rev 1997, 61:456-502. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  deposited research <br />       eukaryotes. Proc Natl Acad Sci USA 2000, 97:11319-11324.               83.   Wilson AC, Carlson SS, White TJ: Biochemical evolution. Annu <br /> 58.   Wolf YI, Aravind L, Koonin EV: Rickettsiae Chlamydiae: evi-              Rev Biochem 1977, 46:573-639. <br />       dence horizontal gene transfer gene exchange. Trends            84.   Hirsh AE, Fraser HB: Protein dispensability rate of <br />       Genet 1999, 15:173-175.                                                      evolution. Nature 2001, 411:1046-1049. <br /> 59.   Marcotte EM, Pellegrini M, Thompson MJ, Yeates TO, Eisenberg D: A      85.   Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />       combined algorithm genome-wide prediction protein                     Lucau-Danila A, Anderson K, Andre B et al.: Functional profiling <br />       function. Nature 1999, 402:83-86.                                            Saccharomyces cerevisiae genome. Nature 2002, <br /> 60.   Huynen MJ, Snel B: Gene context: integrative approaches                  418:387-391. <br />       genome analysis. Adv Protein Chem 2000, 54:345-379.                 86.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana- <br /> 61.   Aravind L: Guilt association: contextual information                   pin A, Le Bot N, Moreno S, Sohrmann M et al.: Systematic func- <br />       genome analysis. Genome Res 2000, 10:1074-1077.                              tional analysis Caenorhabditis elegans genome using <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  refereed research <br /> 62.   Billy E, Wegierski T, Nasr F, Filipowicz W: Rcl1p, yeast protein         RNAi. Nature 2003, 421:231-237. <br />       similar RNA 3&apos;-phosphate cyclase, associates U3            87.   Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, Agarwal <br />       snoRNP required 18S rRNA biogenesis. EMBO J                       P, Agarwala R, Ainscough R, Alexandersson M, An P et al.: Initial <br />       2000, 19:2115-2126.                                                          sequencing comparative analysis mouse genome. <br /> 63.   Karev GP, Wolf YI, Rzhetsky AY, Berezovskaya FS, Koonin EV: Birth            Nature 2002, 420:520-562. <br />       death protein domains: A simple model evolution              88.   Aparicio S, Chapman J, Stupka E, Putnam N, Chia JM, Dehal P, Christ- <br />       explains power law behavior. BMC Evol Biol 2002, 2:18.                       offels A, Rash S, Hoon S, Smit A et al.: Whole-genome shotgun <br /> 64.   Papp B, Pal C, Hurst LD: Dosage sensitivity evolution             assembly analysis genome Fugu rubripes. Science <br />       gene families yeast. Nature 2003, 424:194-197.                            2002, 297:1301-1310. <br /> 65.   Kubota H, Hynes G, Willison K: The chaperonin containing t-            89.   Holt RA, Subramanian GM, Halpern A, Sutton GG, Charlab R, Nussk- <br />       complex polypeptide 1 (TCP-1). Multisubunit machinery                        ern DR, Wincker P, Clark AG, Ribeiro JM et al.: The genome <br />       assisting protein folding assembly eukaryotic                  sequence malaria mosquito Anopheles gambiae. Science <br />       cytosol. Eur J Biochem 1995, 230:3-16.                                       2002, 298:129-149. <br />                                                                                                                                                                  interactions <br /> 66.   Jones S, Newman C, Liu F, Segev N: The TRAPP complex              90.   Dehal P, Satou Y, Campbell RK, Chapman J, Degnan B, De Tomaso A, <br />       nucleotide exchanger Ypt1 Ypt31/32. Mol Biol Cell 2000,              Davidson B, Di Gregorio A, Gelpke M, Goodstein DM et al.: The <br />       11:4403-4411.                                                                draft genome Ciona intestinalis: insights chordate and <br /> 67.   Mewes HW, Frishman D, Guldener U, Mannhaupt G, Mayer K,                      vertebrate origins. Science 2002, 298:2157-2167. <br />       Mokrejs M, Morgenstern B, Munsterkotter M, Rudd S, Weil B: MIPS:       91.   Gardner MJ, Hall N, Fung E, White O, Berriman M, Hyman RW, Carl- <br />       database genomes protein sequences. Nucleic Acids                  ton JM, Pain A, Nelson KE, Bowman S et al.: Genome sequence of <br />       Res 2002, 30:31-34.                                                          human malaria parasite Plasmodium falciparum. Nature <br /> 68.   Ponting CP, Aravind L, Schultz J, Bork P, Koonin EV: Eukaryotic              2002, 419:498-511. <br />       signalling domain homologues archaea bacteria.                  92.   Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lip- <br />       Ancient ancestry horizontal gene transfer. J Mol Biol 1999,              man DJ: Gapped BLAST PSI-BLAST: new generation of <br />       289:729-745.                                                                 protein database search programs. Nucleic Acids Res 1997, <br />                                                                                                                                                                  information <br />  <br />  <br />  <br />  <br /> 69.   Pestov DG, Stockelman MG, Strezoska Z, Lau LF: ERB1, yeast               25:3389-3402. <br />       homolog mammalian Bop1, essential gene required               93.   Wootton JC, Federhen S: Analysis compositionally biased <br />       maturation 25S 5.8S ribosomal RNAs. Nucleic                   regions sequence databases. Methods Enzymol 1996, <br />       Acids Res 2001, 29:3621-3630.                                                266:554-571. <br /> 70.   Dragon F, Gallagher JE, Compagnone-Post PA, Mitchell BM, Por-          94.   NCBI BLAST server [ftp://ftp.ncbi.nih.gov/blast] <br />       wancher KA, Wehner KA, Wormsley S, Settlage RE, Shabanowitz J,         95.   Schultz J, Milpetz F, Bork P, Ponting CP: SMART, simple modu- <br />       Osheim Y et al.: A large nucleolar U3 ribonucleoprotein                      lar architecture research tool: identification signaling <br />       required 18S ribosomal RNA biogenesis. Nature 2002,                      domains. Proc Natl Acad Sci USA 1998, 95:5857-5864. <br />  <br />                                                               Genome Biology 2004, 5:R7 <br />  R7.28 Genome Biology 2004,       Volume 5, Issue 2, Article R7    Koonin et al.                                      http://genomebiology.com/2004/5/2/R7 <br />  <br />  <br />  <br />  <br />        96.    Huang X: On global sequence alignment. Comput Appl Biosci                  13:3369-3387. <br />               1994, 10:227-235.                                                     120. Myers LC, Kornberg RD: Mediator transcriptional <br />        97.    Felsenstein J: Inferring phylogenies protein sequences             regulation. Annu Rev Biochem 2000, 69:729-749. <br />               parsimony, distance, likelihood methods. Methods Enzymol          121. Gu W, Malik S, Ito M, Yuan CX, Fondell JD, Zhang X, Martinez E, Qin <br />               1996, 266:418-427.                                                         J, Roeder RG: A novel human SRB/MED-containing cofactor <br />        98.    Clusters orthologous groups eukaryotic complete                     complex, SMCC, involved transcription regulation. Mol Cell <br />               genomes [http://www.ncbi.nlm.nih.gov/COG/new/shokog.cgi]                   1999, 3:97-108. <br />        99.    The Eukaryotic Clusters Orthologous Groups proteins             122. Leipe DD, Wolf YI, Koonin EV, Aravind L: Classification evo- <br />               (KOGs): download [ftp://ftp.ncbi.nih.gov/pub/COG/KOG]                      lution P-loop GTPases related ATPases. J Mol Biol 2002, <br />        100.   Reconstructed KOG sets eukaryotic ancestral forms                      317:41-72. <br />               [ftp://ftp.ncbi.nih.gov/pub/koonin/Ancestors/]                        123. Aravind L, Koonin EV: Phosphoesterase domains associated <br />        101.   Chen PL, Chen CF, Chen Y, Xiao J, Sharp ZD, Lee WH: The BRC                DNA polymerases diverse origins. Nucleic Acids Res <br />               repeats BRCA2 critical RAD51 binding resist-                1998, 26:3746-3752. <br />               ance methyl methanesulfonate treatment. Proc Natl Acad Sci         124. Aravind L, Koonin EV: Gleaning non-trivial structural, func- <br />               USA 1998, 95:5287-5292.                                                    tional evolutionary information proteins by <br />        102.   Kojic M, Kostrub CF, Buchman AR, Holloman WK: BRCA2                        iterative database. J Mol Biol 1999, 287:1023-1040. <br />               homolog required proficiency DNA repair, recombina-            125. Wolf YI, Rogozin IB, Kondrashov AS, Koonin EV: Genome align- <br />               tion, genome stability Ustilago maydis. Mol Cell 2002,              ment, evolution prokaryotic genome organization and <br />               10:683-691.                                                                prediction gene function using genomic context. Genome <br />        103.   Genschik P, Drabikowski K, Filipowicz W: Characterization           Res 2001, 11:356-372. <br />               Escherichia coli RNA 3&apos;-terminal phosphate cyclase            126. Bryant NJ, James DE: Vps45p stabilizes syntaxin homologue <br />               sigma54-regulated operon. J Biol Chem 1998, 273:25516-25526.               Tlg2p positively regulates SNARE complex formation. <br />        104.   Dasgupta A, Darst RP, Martin KJ, Afshari CA, Auble DT: Mot1 acti-          EMBO J 2001, 20:3380-3388. <br />               vates represses transcription direct, ATPase-depend-           127. Anantharaman V, Koonin EV, Aravind L: Comparative genomics <br />               ent mechanisms. Proc Natl Acad Sci USA 2002, 99:2666-2671.                 evolution proteins involved RNA metabolism. <br />        105.   Leonard CJ, Aravind L, Koonin EV: Novel families putative pro-          Nucleic Acids Res 2002, 30:1427-1464. <br />               tein kinases bacteria archaea: evolution                128. Morishita R, Kawagoshi A, Sawasaki T, Madin K, Ogasawara T, Oka T, <br />               &quot;eukaryotic&quot; protein kinase superfamily. Genome Res 1998,                  Endo Y: Ribonuclease activity rat liver perchloric acid-solu- <br />               8:1038-1047.                                                               ble protein, potent inhibitor protein synthesis. J Biol Chem <br />        106.   Vanrobays E, Gelugne JP, Gleizes PE, Caizergues-Ferrer M: Late             1999, 274:20688-20692. <br />               cytoplasmic maturation small ribosomal subunit                 129. Aravind L, Koonin EV: Novel predicted RNA-binding domains <br />               requires RIO proteins Saccharomyces cerevisiae. Mol Cell Biol           associated translation machinery. J Mol Evol 1999, <br />               2003, 23:2083-2095.                                                        48:291-302. <br />        107.   Gonczy P, Echeverri C, Oegema K, Coulson A, Jones SJ, Copley RR,      130. Bai C, Tolias PP: Cleavage RNA hairpins mediated a <br />               Duperon J, Oegema J, Brehm M, Cassin E et al.: Functional                  developmentally regulated CCCH zinc finger protein. Mol <br />               genomic analysis cell division C. elegans using RNAi              Cell Biol 1996, 16:6661-6667. <br />               genes chromosome III. Nature 2000, 408:331-336.                    131. Cheng Y, Kato N, Wang W, Li J, Chen X: Two RNA binding pro- <br />        108.   Lee SJ, Baserga SJ: Imp3p Imp4p, specific components               teins, HEN4 HUA1, act processing AGAMOUS <br />               U3 small nucleolar ribonucleoprotein essen-                pre-mRNA Arabidopsis thaliana. Dev Cell 2003, 4:53-66. <br />               tial pre-18S rRNA processing. Mol Cell Biol 1999,                 132. Nelissen RL, Heinrichs V, Habets WJ, Simons F, Luhrmann R, van Ven- <br />               19:5441-5452.                                                              rooij WJ: Zinc finger-like structure U1-specific protein C is <br />        109.  <br /> </body></html>