<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Pou5f1/Oct4 Promotes Cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>Survival</span> Direct Activation <br /> of mych Expression Zebrafish Gastrulation <br /> Kay Kotkamp1., Esther Kur1.&#194;&#164;, Bjo&#194;&#168;rn Wendik1, Boz_ena K. Polok1, Shifra Ben-Dor2, Daria Onichtchouk1,3, <br /> Wolfgang Driever1,3* <br /> 1 Developmental Biology, Institute Biology I, Faculty Biology, Albert-Ludwigs-University Freiburg, Freiburg, Germany, 2 Biological Services, Weizmann Institute of <br /> Science, Rehovot, Israel, 3 BIOSS - Centre Biological Signalling Studies, Freiburg, Germany <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Myc proteins control cell proliferation, cell cycle progression, apoptosis, play important roles cancer in <br />      establishment pluripotency. Here investigated control myc gene expression Pou5f1/Oct4 pluripotency <br />      factor early zebrafish embryo. We analyzed expression known zebrafish Myc family members, myca, mycb, <br />      mych, mycl1a, mycl1b, mycn, mount situ hybridization blastula gastrula stages wildtype and <br />      maternal plus zygotic pou5f1 mutant (MZspg) embryos, quantitative PCR time series microarray data. We <br />      broad blastula gastrula stage mych expression, late gastrula stage mycl1b expression, both <br />      depend Pou5f1 activity. We analyzed ChIP-Seq data Pou5f1 Sox2 bind mych mycl1b <br />      control regions. The regulation mych Pou5f1 appears direct transcriptional activation, overexpression a <br />      Pou5f1 activator fusion protein MZspg embryos induced strong mych expression translation zygotically <br />      expressed mRNAs suppressed. We showed MZspg embryos develop enhanced apoptosis during <br />      early gastrula stages, apoptosis detected wildtype embryos. However, Mych knockdown did not <br />      induce early apoptosis, suggesting potentially redundant action early expressed myc genes, combination of <br />      pathways affected MZspg. Experimental mych overexpression MZspg embryos did significantly, not <br />      completely suppress apoptosis <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span>. Similarly, p53 knockdown partially suppressed apoptosis MZspg <br />      gastrula embryos. However, combined knockdown p53 overexpression Mych completely rescued MZspg <br />      apoptosis phenotype. These results reveal Mych anti-apoptotic activity early zebrafish embryo, p53- <br />      dependent Myc pathways likely act parallel control apoptosis stages. <br />  <br />   Citation: Kotkamp K, Kur E, Wendik B, Polok BK, Ben-Dor S, et al. (2014) Pou5f1/Oct4 Promotes <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>Cell Survival</span> Direct Activation mych Expression during <br />   Zebrafish Gastrulation. PLoS ONE 9(3): e92356. doi:10.1371/journal.pone.0092356 <br />   Editor: Ferenc Mu&#194;&#168;ller, University Birmingham, United Kingdom <br />   Received December 12, 2013; Accepted February 17, 2014; Published March 18, 2014 <br />   Copyright: &#195;&#376; 2014 Kotkamp et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported BMBF FORSYS center FRISYS, Excellence Initiative German Federal State Governments (BIOSS Centre for <br />   Biological Signalling Studies EXC 294), German Research Foundation (DFG-SFB592-A3 &amp; SFB850-A1, WD). The funders role study design, data <br />   collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: driever@biologie.uni-freiburg.de <br />   . These authors contributed equally work. <br />   &#194;&#164; Current address: Department Neurobiology, Harvard Medical School, Boston, United States America <br />  <br />  <br />  <br /> Introduction                                                                         bind specific DNA sites promoter regions target genes <br />                                                                                      [19,20]. It shown Myc associated gene <br />    Apoptosis plays crucial role development                             activation [21,22] repression [23,24]. The myc proto-oncogene <br /> maintenance homeostasis multicellular organisms                             family consists members c-myc [25], N-myc [26] L- <br /> eliminating damaged unneeded cells [1,2]. Programmed cell                         myc [27]. Myc deregulation linked tumor formation in <br /> death intensely studied model organisms,                           animals human [28&#226;&#8364;&#8220;30]. During normal mouse embryonic <br /> deregulation involved diseases including cancer,                          development c-myc expressed different cell types [31&#226;&#8364;&#8220; <br /> Alzheimer&#226;&#8364;&#8482;s disease, immune deficiencies [3,4]. Today                        34]. C-Myc shown correlate cell proliferation in <br /> components apoptosis pathway regulatory                             tissues, induces sensitizes cells apoptosis in <br /> interactions known [5&#226;&#8364;&#8220;7]. In recent years, zebrafish                        [35]. L-myc N-myc expression restricted specific <br /> been increasingly used model studying apoptosis,                   tissues course cell differentiation [31&#226;&#8364;&#8220;34]. The <br /> the ease experimental manipulation close homology                      importance c-Myc promoting early development shown <br /> the apoptosis pathway core factors fish mammals                          fact c-myc homozygous <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>knock-out</span> mice embryonic <br /> [8&#226;&#8364;&#8220;13].                                                                              lethal [36], role stem cell pluripotency mouse <br />    Myc proteins transcription factors involved regulation                  fibroblasts reprogrammed overexpressing c-Myc together <br /> various cellular functions including cell-cycle progression [14],                    Oct4, Sox2 Klf4 [37]. In embryonic stem cells, Myc <br /> differentiation [15,16], cell growth [17], apoptosis [18]. Myc                   makes complex contributions pluripotency [38&#226;&#8364;&#8220;40]. <br /> forms heterodimers transcriptional co-regulator Max <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                                March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                        Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />    In zebrafish, homologous genes myc family members          myca, mycb, mych, mycl1a mycl1b WT MZspg mutants at <br /> been identified [41,42]. As result genome duplication        60% epiboly (Figure 1). Early expression patterns mych mycn <br /> occurred evolution teleosts [43] fish paralogues        previously reported [44,45]. The mesodermal marker no <br /> exist c-myc (myca mycb) L-myc (mycl1a mycl1b; Figure       tail (ntl; [48] used control experiment, ntl is <br /> S1)[41,42]. In contrast, <span id='am-394' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> N-          regulated Pou5f1 [47]. Only c-myc homologs, <br /> myc known zebrafish, mycn [42,44]. Recently, new member            mycb, detected spatially restricted expression mid- <br /> the myc gene family, mych, identified zebrafish. Mych shows         gastrulation (Figure 1A-D). At 60% epiboly mycb specifically <br /> a high similarity N-Myc c-Myc C-terminal amino acid          expressed region embryonic shield (Figure 1C-D). <br /> sequence, direct orthologous gene higher vertebrates         The expression patterns myca mycb displayed no <br /> [41,45]. Zebrafish myc homologous genes differentially                 alterations Pou5f1 deficient embryos compared WT. The <br /> expressed embryonic development, specific adult              slightly higher microarray expression levels myca mycb this <br /> tissues [41,42,44,45]. The spatial distributions early gene         stage MZspg (Fig S3A,C) clearly detected WISH <br /> expression patterns previously described mych, mycn          minor increase stain intensity MZspg. mych, mycl1a <br /> and mycl1a. All genes broad expression blastula            mycl1b broadly expressed the <br /> and gastrula stages [44&#226;&#8364;&#8220;46]. Meijer et al. [41] described              blastoderm 60%-epiboly, mycl1b, contrast mych and <br /> postgastrula expression patterns known zebrafish myc                mycl1a, excluded embryonic shield (Figure 1E-J). We <br /> homologues great (see Figure S2). However,              MZspg mutants mych mycl1b expression were <br /> functions myc genes early zebrafish development poorly           strongly downregulated (Figure 1E-F I-J). Therefore, mych and <br /> understood. The recently identified mych gene linked           mycl1b expression appeared depend Pou5f1 transcription <br /> cell survival neural crest development [45].                           factor. Interestingly, expression mych depend on <br />    In mammalian ES cells, myc genes regulated master            Pou5f1 activity early gastrula cells, cells of <br /> stem cell factors Pou5f1 (Oct4) Nanog [38,40]. Previous work           embryonic shield (Figure 1E-F). <br /> in lab revealed transcript levels zebrafish myc           We quantified expression mych mycl1b WT and <br /> genes, mycl1b mych, reduced pou5f1 (also named Pou2          MZspg embryos different developmental stages ranging <br /> Pou5f3; www.zfin.org) maternal zygotic (MZspg) mutant                  1000-cell (MBT) 75% epiboly relative quantitative real- <br /> embryos judged microarray analysis [47]. Here                   time PCR (qPCR) (Figure 2). We used housekeeping gene <br /> analyzed early expression patterns zebrafish myc genes          translation elongation factor 1a (ef1a named eef1a1l1) reference <br /> and expression Pou5f1 deficient embryos. We showed           gene normalize expression levels mycl1b mych in <br /> the zebrafish mych mycl1b genes bound Pou5f1            MZspg WT. The developmental expression profiles, deter- <br /> likely direct targets Pou5f1. We showed MZspg              mined qPCR, genes agreement with <br /> gastrulae display enhanced apoptosis compared wildtype (WT)             microarray results (Figure S3G,K) [47]. The developmental <br /> controls. The apoptosis phenotype partially rescued mych            profile mycl1b showed strong signal mid-blastula transition <br /> overexpression, indicating role Mych regulation cell            WT Pou5f1 mutants expression decreased during <br /> survival gastrulation.                                              gastrulation (Figure 2A). This suggested early <br />                                                                            mycl1b mRNA maternally expressed. In comparison WT, the <br /> Results                                                                    expression level mycl1b MZspg 1.5 times higher at <br />                                                                            MBT, decline rapidly MZspg mutant embryos <br /> Pou5f1 activity required induce zygotic expression                   gastrulation, resulting 12.5 fold downregulation <br /> of mych mycl1b                                                         compared WT 60%-epiboly (Figure 2A). This difference <br />    Temporal expression profiles time series microarray                caused mechanisms: Pou5f1 protect maternal <br /> analyses (zygote 8 hours post fertilization - hpf) wildtype          mycl1b direct indirect mechanisms, Pou5f1 induce <br /> (WT) Pou5f1 maternal zygotic deficient embryos (MZspg)             zygotic expression mycl1b. In contrast mycl1b, mych not <br /> [47] revealed myc genes potentially regulated Pou5f1        detected expressed maternally. mych expression to <br /> early zebrafish embryo (Figure S3). For zebrafish c-myc                activated immediately MBT WT embryos reach <br /> homologues myca mycb, MZspg embryos compared WT,                 high point 30% epiboly declining (Figure 2B). In <br /> the mRNA levels derived maternal expression                 MZspg embryos mych expression strongly reduced all <br /> found reduced onset zygotic transcription                  analyzed developmental stages (Figure 2B). Our results demon- <br /> (midblastula transition - MBT), elevated post-MBT stages            strate Pou5f1 transcription factor important early <br /> (Figure S3A C). mycl1a expression altered Pou5f1            zygotic activation mych expression, maintaining mycl1b <br /> mutants compared WT (Figure S3E). mycl1b expression                  RNA levels. In summary, data indicate Pou5f1 function is <br /> MZspg normal blastula early                important activation early zygotic expression mych, <br /> gastrula stages, downregulated late gastrulation (Figure        mycl1b 8 hours development. <br /> S3G). The expression mych mycn shown be <br /> strongly activated MBT WT embryos, Pou5f1              mych mycl1b directly activated Pou5f1 <br /> mutants (Figure S3I K). Whereas induction mycn                     To distinguish direct indirect Pou5f1 targets, <br /> expression appeared delayed MZspg embryos (Figure            Onichtchouk et al. [47] performed pou5f1 mRNA overexpression <br /> S3I), mych continued expressed strongly reduced levels         experiments MZspg embryos, inhibited translation of <br /> MZspg gastrulation (Figure S3K). These findings                 zygotic mRNAs MBT adding translation elongation <br /> suggested Pou5f1 function crucial proper regulation          inhibitor cycloheximide (CHX) 64-cell stage. They <br /> of mycl1b mych gene expression early zebrafish development.         compared microarray analysis expression profiles 30% <br />    To confirm microarray data obtain information           epiboly embryos injected pou5f1 mRNA versus non- <br /> about spatial distribution gastrula stage myc gene expression       injected controls, treated CHX, the <br /> patterns changes MZspg embryos, performed                  expression direct Pou5f1 targets differentially affected. <br /> whole-mount situ hybridization (WISH) expression analysis           We analyzed published data answer question whether <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             2                             March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                            Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />                                                                                 addressed experimentally level Pou5f1 may <br />                                                                                 able directly activate mych mycl1b expression. We <br />                                                                                 microinjected mRNA encoding fusion protein Pou5f1 the <br />                                                                                 strong transcriptional activator domain VP16 [49] MZspg <br />                                                                                 embryos, inhibited translation zygotically expressed <br />                                                                                 mRNAs CHX. Therefore, effect Pou5f1-VP16 mych <br />                                                                                 mycl1b direct, mediated indirect effects of <br />                                                                                 Pou5f1 targets mych mycl1b. The pou5f1-VP16 <br />                                                                                 mRNA injected chosen embryos showed a <br />                                                                                 partial rescue morphological MZspg phenotype described <br />                                                                                 Lunde et al. [49] (Figure S4B), demonstrates the <br />                                                                                 functionality used fusion mRNA. To reveal specificity of <br />                                                                                 experiment, used tail control (Figure 3E), has <br />                                                                                 shown regulated Pou5f1 [49], affected <br />                                                                                 pou5f1-VP16 mRNA experiment. We analyzed the <br />                                                                                 expression mych mycl1b WISH 60% epiboly. Injection <br />                                                                                 pou5f1-VP16 fusion mRNA MZspg embryos induced strong <br />                                                                                 mych mycl1b expression (Figure 3A C, Supplemental Table <br />                                                                                 S1) presence CHX (Figure 3B D). However, <br />                                                                                 mycl1b RNA levels increased pou5f1-VP16 <br />                                                                                 overexpression CHX treated MZspg embryos, the <br />                                                                                 CHX treatment control MZspg embryos (Figure3 D). <br />                                                                                 Given strong maternal expression (Figs. 2 S3), CHX <br />                                                                                 treatment indirectly prevent degradation maternal mycl1b <br />                                                                                 RNA mechanisms depending translation zygotic protein <br />                                                                                 products. <br />                                                                                    To investigate Pou5f1 bind mych mycl1b <br />                                                                                 regulatory regions, analyzed published genome-scale chroma- <br />                                                                                 tin immunoprecipitation data (ChIP-seq) Pou5f1 [50]. Figure 4 <br />                                                                                 shows 50% epiboly stage Pou5f1 detected bound the <br />                                                                                 proximity mych mycl1b genes (+6 kb -10,5 kb <br />                                                                                 transcription start sites, respectively). In addition, smaller <br />                                                                                 Pou5f1 binding peaks marked basal promoter region mych. <br />                                                                                 Three Pou5f1 ChIP-Seq peaks correlated Sox2 ChIP- <br />                                                                                 Seq peaks, suggesting Pou5f1 Sox2 act this <br />                                                                                 regulation. In summary, data reveal zygotic transcription <br />                                                                                 mych mycl1b directly regulated Pou5f1. <br />  <br />                                                                                 Pou5f1 mutants enhanced apoptosis <br /> Figure 1. Spatial expression <span id='am-398' about='xsp:pattern' typeof='owl:Thing'>pattern</span> zebrafish myc genes in <br /> WT MZspg embryos 60% epiboly. Whole mount situ <br />                                                                                    The prominent early phenotype MZspg embryos is <br /> hybridization (WISH) analysis myca, mycb, mych, mycl1a mycl1b            delayed ultimately arrested progression epiboly (compare <br /> expression WT (right embryo panel) MZspg (left embryo            Figure 5B J) [49,51,52]. Pou5f1-dependent regulation cell <br /> in panel). All embryos shown lateral (left column) animal       adhesion cell motility shown contribute the <br /> (right column) views dorsal oriented right. All analyzed myc        epiboly delay phenotype MZspg [53], does appear to <br /> genes broadly expressed mid-gastrula embryos, c-myc <br />                                                                                 explain extent phenotypic abnormalities MZspg. <br /> homologues, myca mycb. mycb specifically expressed the <br /> shield, myca detectable stage (A-D). mych           Additional causes epiboly delay phenotype include <br /> mycl1a addition strong expression domain involuting            slowdown cell proliferation increase apoptosis. Lachnit et <br /> axial mesoderm (E-H). Only mych (E-F) mycl1b (I-J) depend            al. [51] demonstrated 5 hpf stage reduction in <br /> function Pou5f1 expression strongly decreased MZspg          number deep cells detectable MZspg mutants compared to <br /> mutants. However, mych expression domain involuting axial            wildtype. However, cell death proliferation been <br /> mesoderm affected Pou5f1 deficient embryos (E-F; arrows). We <br /> used notail control, expression altered MZspg          analyzed mid late gastrula stages MZspg far. <br /> mutants compared WT (K-L).                                                      We analyzed cell proliferation 90% epiboly calculating the <br /> doi:10.1371/journal.pone.0092356.g001                                           mitotic index MZspg mutants WT embryos (Figure S5). We <br />                                                                                 fixed embryos stained chromatin using Sytox Green <br />                                                                                 fluorescent dye. We recorded animal view confocal stacks the <br /> the myc genes regulated directly indirectly Pou5f1                 stained embryos determined total number nuclei well <br /> (Figure S3). We expression mych, mycl1b, c-myc            number metaphase anaphase nuclei, which <br /> homologous genes 2-fold increased                      characterize cells mitosis. We calculated ratio mitotic <br /> injection pou5f1 mRNA subsequent addition CHX,                        nuclei total nuclei (Figure S5). We 90% <br /> which suggests direct regulation Pou5f1 (Figure S3B, D, H                epiboly stage, MZspg epiboly phenotype very <br /> L).                                                                             pronounced, mitotic index MZspg mutants did not <br />    To determine spatial extent myc gene regulation                    differ significantly WT embryos (Figure S5A, Supplemental <br /> Pou5f1 embryo, confirm microarray data,                    Table S3). Thus, reduction cell number caused decreased <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  3                            March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                         Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />  <br />  <br /> Figure 2. Temporal analysis mych mycl1b expression WT MZspg. Relative expression levels mych mycl1b WT and <br /> MZspg embryos distinct time points, 1000-cell (MBT) 75%-epiboly stage using quantitative real-time PCR. Bars average <br /> values independent biological replicates each. The WT expression 60%-epiboly stage normalized 1 genes. mycl1b is <br /> maternally expressed declines embryonic development progresses (A, white bars). In MZspg mutants, mycl1b expression upregulated at <br /> MBT, likely reflecting higher maternal mRNA levels, comparison WT declines faster gastrulation (A, black bars). In WT mych is <br /> activated MBT, expression increases 30%-epiboly starts slowly fade (B, white bars). mych expression levels to <br /> 26-fold reduced MZspg mutants compared WT early embryogenesis (B, black bars). <br /> doi:10.1371/journal.pone.0092356.g002 <br />  <br /> proliferation late gastrulation excluded cause         bud stage (Figs. 6 7). We detect apoptotic cells <br /> the MZspg epiboly phenotype.                                                MZspg WT embryos early gastrula stage <br />   Controlled cell death, specifically apoptosis, contributes         (Figure 6A-F). Later development, starting 60% epiboly, we <br /> embryonic development animal kingdom [54]. Cole              enhanced apoptosis MZspg mutants, WT <br /> Ross [55] described temporal spatial distribution                embryos showed TUNEL staining (Figure 6G-J and <br /> apoptotic cells normal zebrafish development. The earliest           Figure 7A-D). <br /> apoptotic cells wildtype development reported around <br /> 12 hpf dorsal midline segmental plate. To                    Mych controls apoptosis zebrafish <br /> investigate MZspg embryos enhanced apoptosis                    To investigate potential contribution loss Mych the <br /> before time stage, analyzed apoptosis MZspg mutant           MZspg phenotype analyzed morphology mych <br /> WT embryos using TUNEL assay staining 32-cell stage to <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              4                             March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                          Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />  <br />  <br /> Figure 3. mych potentially mycl1b directly regulated Pou5f1. Pou5f1 overexpression MZspg mutants pou5f1-VP16 <br /> mRNA injection one-cell stage embryos (A-F; left embryos panel) non-injected controls (A, C E; MZspg middle WT right <br /> embryos panel; B, D F; MZspg right embryo panel). The embryos B, D F addition treated CHX starting 1.5 hpf. <br /> The mych expression rescued Pou5f1-VP16 both, CHX treated untreated embryos, regulatory influence Pou5f1 <br /> should direct (A-B). The expression mycl1b rescued Pou5f1-VP16 overexpression CHX untreated MZspg embryos, also <br /> strongly upregulated CHX treated MZspg embryos, injection pou5f1-VP16 mRNA (C-D). Thus experiment proove <br /> whether activation mycl1b Pou5f1 direct not. We used notail negative control, expression independent Pou5f1 function <br /> (E), depends zygotic gene products. <br /> doi:10.1371/journal.pone.0092356.g003 <br />  <br /> overexpression MZspg mutants Mych knockdown WT                     apoptosis [56]. MZspg p53 MO injected embryos showed strong <br /> embryos 90% epiboly 25 hpf (Figure 5). The knockdown               reduction apoptosis compared non-injected MZspg controls <br /> Mych activity using splice blocking (Sp-MO) morpholino,              (Figure 7G-H). The co-injection mych mRNA p53-MO <br /> blocks splicing second intron zygotic mych pre-mRNA,           completely rescued MZspg apoptosis phenotype (Figure 7I-J). <br /> or translation blocking (ATG-MO) morpholino, blocks                  We quantified numbers apoptotic cells experiments <br /> the translation maternal zygotic mych mRNAs, did             (Figure 7K, Figure S7, Supplemental Table S4, Methods). When <br /> cause delay epiboly movement (Figure 5A-B E-F). To                  MZspg embryos injected mych mRNA compared to <br /> demonstrate functionality mych ATG-MO, knocked                 control MZspg embryos, statistically significant six-fold reduction <br /> down GFP embryos injected gfp mRNA fused mych                 apoptotic cells mych overexpressing MZspg observed. p53 <br /> ATG-MO binding site (Figure S6A-H). The effectiveness                 knockdown caused pronounced reduction apoptotic <br /> mych Sp-MO shown mych RT-PCR injecting different                phenotype MZspg, combined overexpression mych <br /> concentrations splice blocking morpholino (Figure S6I).               knockdown p53 rescued apoptosis essentially to <br /> Also, broad overexpression mych MZspg mutants mRNA                  low wildtype levels (Figure 7K). While microinjected well- <br /> injection rescue MZspg phenotype (Figure 5I-J).                established amounts p53 MO knockdown, relatively <br /> One-day-old manipulated embryos developed morphologically                    large mych mRNA rescue, completely <br /> similar WT MZspg control embryos, respectively (Figure 5 C-            exclude incomplete rescue knockdown contribute to <br /> D, G-H K-L; Supplemental Table S2). A detailed                      apparent partial rescue apoptosis only. We conclude that <br /> analysis late phenotypes Mych morphants reported               Mych p53 dependent pathways likely contribute in <br /> by Hong et al. [45].                                                         parallel MZspg apoptotic phenotype. <br />    Next, analyzed Mych activity contribute to <br /> control apoptosis gastrula stages. Following mych                  Discussion <br /> knockdown analysis TUNEL assay, enhanced <br /> apoptosis Mych morphants 24 hpf (data shown),                         Myc proteins contribute control cell proliferation by <br /> comparable results reported Hong et al. [45]. To                   regulating cell cycle progression apoptosis, play important <br /> analyze mych overexpression rescue apoptosis                 roles cancer [28] establishment pluripotency [37]. <br /> phenotype, injected mych mRNA one-cell stage MZspg                   Here investigated control myc gene expression the <br /> embryos, compared numbers apoptotic cells late                 Pou5f1/Oct4 pluripotency factor early zebrafish embryo. <br /> gastrula stage injected non-injected embryos. We                 We early zygotic mych expression late gastrula <br /> found mych mRNA injection led reduced number                    stage mycl1b expression directly depend Pou5f1 activity. <br /> apoptotic cells MZspg mutants bud stage (Figure 7E-F). In          We showed Pou5f1 deficient MZspg embryos <br /> parallel performed p53 knockdown MZspg embryos, using                developed enhanced apoptosis early gastrula <br /> p53 specific morpholino, reported block                    stages, mych overexpression MZspg embryos able <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               5                             March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                        Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />                                                                             neurulation. During mouse gastrulation c-myc N-myc are <br />                                                                             widely expressed embryonic extraembryonic tissues [32]. <br />                                                                             Thus, broad myc gene expression blastula gastrula <br />                                                                             stages evolutionary conserved vertebrate <br />                                                                             subphylum. However, known potentially <br />                                                                             conserved myc gene functions blastula gastrula stages. <br />  <br />                                                                             Pou5f1 activity required proper mych, mycl1b and <br />                                                                             mycn expression <br />                                                                                We transcription factor Pou5f1 required for <br />                                                                             proper transcription mych MBT 1000-cell stage <br />                                                                             on. While Pou5f1 appears directly activate broad early <br />                                                                             zygotic expression mych MBT, Pou5f1 activity not <br />                                                                             required initiation later mych expression domain in <br />                                                                             involuted axial mesoderm dorsal embryo. <br />                                                                             Pou5f1 protein involved proper maternal regulation of <br />                                                                             mycl1b, enhanced levels mycl1b detected pre-MBT and <br />                                                                             blastula stages MZspg. Further, Pou5f1 required proper <br />                                                                             maintenance mycl1b expression level mid- late gastrula <br />                                                                             stages, mycl1b drops significantly wildtype levels in <br />                                                                             MZspg embryos. In addition, Pou5f1 required proper early <br />                                                                             post-MBT activation mycn expression, delayed in <br />                                                                             MZspg mutants. <br />                                                                                The regulation c-myc expression intensively studied <br />                                                                             systems. A number regulatory cis-elements been <br />                                                                             identified mammals shown transcription <br />                                                                             factors bind c-myc regulatory elements vivo [59]. Chen <br />                                                                             et al. [60] analyzed core transcriptional network embryonic <br />                                                                             stem cells using chromatin immunoprecipitation coupled with <br />                                                                             ultra-high-throughput DNA sequencing (ChIP-seq) map the <br />                                                                             locations key pluripotency transcription factors like Oct4, Sox2, <br />                                                                             Nanog, c-Myc n-Myc. They Oct4 Sox2 <br />                                                                             directly bind n-myc promoter region, interact with <br />                                                                             c-myc. This reported regulation myc family members consistent <br /> Figure 4. Pou5f1 binds promoters regulatory <br />                                                                             results early zebrafish embryogenesis, mych and <br /> regions mych mycl1b. ChIP-seq reads Sox2 (red)               mycl1b genes directly induced Pou5f1 potentially also <br /> Pou5f1 (blue) [50] mapped mycl1b (A) mych (B) genetic loci       bound Sox2. Thus, regulation myc gene expression Pou5f1 <br /> are shown. Colocalisation Pou5f1 (blue) Sox2 (red)            Sox2 early embryogenesis, respectively ES cells, may <br /> regulatory regions mycl1b mych detected. Y-axis: ChIP-Seq       evolutionary conserved feature vertebrate development. <br /> reads <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span>, y-axis floor set 2. <br /> doi:10.1371/journal.pone.0092356.g004 <br />                                                                             Maternal zygotic pou5f1 mutants normal <br /> to significantly suppress apoptosis phenotype. Combined                 proliferation, enhanced apoptosis gastrulation <br /> knockdown p53 overexpression Mych completely                      stages <br /> rescued MZspg apoptosis phenotype. Both results                   When analyzing cell division rates cell survival blastula to <br /> reveal Mych anti-apoptotic activity early zebrafish         gastrula stages, differences mitotic rates <br /> embryo, p53-dependent Myc pathways act parallel             MZspg WT embryos, apoptosis rates were <br /> to control apoptosis stages.                                       markedly increased MZspg embryos starting 60% epiboly. <br />                                                                             The relationship Oct4 apoptosis mammalian ES cells not <br />                                                                             understood. Oct4 central factor ES cell <br /> Zebrafish myc genes broadly expressed the <br />                                                                             transcriptional network, controlling parameters ES cell <br /> first hours development                                                  biology [38,40]. This role compatible promoting <br />    Our previous studies shown zebrafish L-             proliferation inhibiting apoptosis, ES cells immortal cell <br /> myc c-myc homologous genes maternally expressed                 population fast self-replication rates. Oct4 deprivation results <br /> mRNAs deposited egg, remain stable blastula              global expression changes, cessation rapid cell proliferation <br /> stages [42,44,45], mycn mych expression               finally differentiation ES cells trophectoderm. However, <br /> detectable mid-blastula transition [44]. Zebrafish mycl1a,            Oct4 linked survival anti-apoptotic pathways <br /> mycl1b, mych genes broadly expressed blastoderm cells            ES cells different potential mechanisms: a <br /> after MBT. Broad myc gene expression early development               STAT3/Survivin route [61], Trp53 regulation [62], a <br /> was shown vertebrate species. The Xenopus c-myc, L-           miR-125b pathway [63]. Next role ES cells, Oct4 also <br /> myc N-myc homologous genes maternally expressed,            suggested promote survival cancer stem cells by <br /> their expression levels decrease early embryogenesis                 inhibiting apoptosis [63,64]. <br /> [57,58]. xc-myc expression maintained low level                 In zebrafish, like mammals, Pou5f1 controls expression a <br /> blastula gastrula stages increases               large network developmentally important signaling molecules <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              6                            March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                         Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />  <br />  <br /> Figure 5. Analysis Mych contribution morphology MZspg mutant phenotype. Knockdown Mych function WT by <br /> injecting splice-blocking (A-D) translation-blocking (E-H) morpholinos. Morphological analysis injected (A,C,E,G) non-injected (B,D,F,H) <br /> embryos 90%-epiboly 25 hpf. Morphants delay epiboly (A, E) obvious morphological defects 25 hours development <br /> (C, G). Rescue Mych activity MZspg mutants injecting mych mRNA 1-cell stage embryos (I-L). Mych able rescue strong <br /> defects developed Pou5f1 deficient embryos gastrulation (I) 25 hpf (K). <br /> doi:10.1371/journal.pone.0092356.g005 <br />  <br /> and transcription factors [47,50]. The majority factors          myc genes contribute control apoptosis regulating p53, e.g. <br /> expressed embryonic tissue-specific patterns, starting            p19ARF dependent manner [66]. We determined whether <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> mid-gastrulation stages [47,49,50,65]. Therefore,                 Pou5f1 regulate levels tp53 mRNA reanalyzing <br /> possible, absence factors               published time-series microarray data MZspg WT embryos <br /> compatible cell survival specific tissue,            [47], significant differences tp53 mRNA levels <br /> lead activation safeguarding apoptotic mechanism.                  zygote end gastrulation. Thus, Pou5f1 does regulate <br /> However, did detect obvious tissue-specific pattern            p53 expression, involvement p53 through <br /> apoptosis MZspg mutants, suggests Pou5f1                indirect pathways controlling p53 activity. If apoptosis MZspg <br /> required embryo prevent activation            p53-dependent, apoptosis completely abol- <br /> apoptotic cascades.                                                          ished knockdown p53 injection p53-morpholino. <br />                                                                              However, p53-MO injected MZspg embryos, mych mRNA <br /> Mechanisms anti-apoptotic action Pou5f1: separate                      injected embryos, apoptosis partially <br /> Myc p53 branches                                                         suppressed. Complete suppression apoptosis MZspg embryos <br />    The expression multiple myc genes, mych, mycl1b,              achieved simultaneous increase Mych activity and <br /> lesser degree mycn, reduced MZspg embryos. Since myc genes             suppression p53. This suggests myc genes, specifically mych, <br /> are known anti-apoptotic factors acting different routes             early zebrafish embryo able suppress apoptosis <br /> including p19ARF, BIM BCL2 (reviewed [66]) MDM2                   p53-independent pathway. A similar p53-independent <br /> [67], convey anti-apoptotic action Pou5f1.                   mechanism potentially involved suppression of <br /> Enhanced early apoptosis MZspg caused reduced                   apoptosis neural plate Mych later developmental <br /> Myc activity, individual myc genes act partially                   stages [45]. Our study suggests zebrafish embryo a <br /> redundantly. Indeed, mych mRNA overexpression                  suitable model dissect p53-dependent independent <br /> is able rescue ectopic apoptosis MZspg embryos.            anti-apoptotic activities Myc proteins embryonic <br /> However, knockdown mych wildtype embryos did                   development. However, coexpression myc genes <br /> induce apoptosis gastrulation stages (10 hpf stage; data          early embryonic stages likely require combined <br /> shown), strengthening notion mych, mycl1b mycn gene             inactivation of genes, far hindered <br /> activities, broadly expressed blastula              progress analysis molecular mechanisms. <br /> gastrula embryo [44,45], required redundantly down- <br /> stream Pou5f1 prevent early activation apoptotic                Materials Methods <br /> programs. <br />    p53, co-factors depending type                Ethics statement <br /> stress cell subjected to, universal activator apoptosis           This study performed approval State of <br /> [68], acting response various stimuli. In cell systems,           Baden-Wu&#194;&#168;rttemberg Regierungspraesidium Freiburg Animal <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               7                            March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                   Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />  <br />  <br /> Figure 6. MZspg mutants enhanced apoptosis gastrulation. TUNEL staining WT (left embryo panel) MZspg (right <br /> embryo panel) embryos distinct embryonic stages 32 cells 90%-epiboly. Embryos shown animal (left column) and <br /> lateral (right column) view. Pou5f1 deficient embryos enhanced apoptosis, starting 60%-epiboly, compared WT (G-J). <br /> doi:10.1371/journal.pone.0092356.g006 <br />  <br /> Protection Authorities accordance German Animal             GACGGAACCTTTTTC-39, standard control morpholino <br /> Protection Act permission number 35-9185.81/G-12/40.              SCMO: 59-CCTCTTACCTCAGTTACAATTTATA-39 were <br />                                                                         ordered Gene Tools (Philomath, USA). The mych-ATG- <br /> Fish embryo care                                                    MO blocks translation mych mRNA protein by <br />    We used WT embryos AB x TU  &#194;&#168; B strain crosses (http://www.       binding translation start site. The mych-Sp-MO prevents <br /> ZFIN.org) MZspg embryos carrying m793 allele spg         splicing second intron resulting non-functional <br /> <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> [69](ZFIN ID: ZDBGENE-980526-485, ZDB-GENO-                    truncated protein missing DNA binding domain. To test the <br /> 081023-1). Fish raised, maintained crossed               specificity mych-ATG-MO, sequence -4 +21 bp the <br /> standard conditions described [70]. Embryos incubated           mych translation start site ATG cloned upstream GFP ORF <br /> or raised egg water 0.36 Danieau&#226;&#8364;&#8482;s solution 28.5uC.         CS2+ vector, obtain mych-GFP construct. 50 pg/ <br /> Developmental age reported hours post fertilization (hpf)         embryo vitro transcribed mych-GFP mRNA injected into <br /> when incubated 28.5uC. Developmental stages MZspg                 one-cell stage embryos 1.4 ng, 4.1 ng 8.6 ng per <br /> embryos indirectly determined observation WT                 embryo mych-ATG-MO Morpholino (Figure S6). <br /> embryos born time incubated identical             The efficiency mych-Sp-MO tested injecting 1.4 ng, <br /> conditions.                                                             4.1 ng 8.6 ng single cell WT embryos subsequent RT- <br />                                                                         PCR analysis 60% epiboly stage. The ratio correctly spliced <br /> Morpholinos                                                             mych mRNA (162 bp) mRNA containing second intron <br />  Morpholino oligos mych-Sp-MO: 59-GTAGCAAAAGACT-                        (396 bp) shows injection 4.1 ng Morpholino sufficient <br /> CACCAGAATCGC-39, mych-ATG-MO: 59-GCAGCATCTT-                            nearly completely inhibit splicing (Figure S6). To address the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                          8                           March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                          Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br />  <br />                                                                              completely suppress cell death MZspg mutants, did rescue <br />                                                                              delay epiboly movement (I-J; arrowheads). The quantification of <br />                                                                              cell death (K) revealed number apoptotic cells decreased <br />                                                                              factor MZspg embryos mych mRNA injection. By <br />                                                                              combined knockdown p53 Mych overexpression, apoptosis in <br />                                                                              MZspg embryos reduced WT levels. <br />                                                                              doi:10.1371/journal.pone.0092356.g007 <br />  <br />                                                                              contribution mych epiboly delay phenotype MZspg mutant <br />                                                                              embryos injected 4.6 ng mych-Sp-MO 1.4 ng mych- <br />                                                                              ATG-MO 1-cell stage embryos respectively. As control we <br />                                                                              injected SCMO. <br />  <br />                                                                              Cycloheximide experiment <br />                                                                                 MZspg embryos injected 10 pg pou5f1-VP16 mRNA <br />                                                                              1-cell stage left non-injected controls. Embryos were <br />                                                                              treated 15 mg/ml cycloheximide (CHX, Calbiochem) <br />                                                                              dissolved egg water. CHX added 1.5 hpf allow for <br />                                                                              translation injected mRNAs, block translation the <br />                                                                              earliest zygotic transcripts. In presence CHX, direct Pou5f1 <br />                                                                              targets transcribed MBT, mRNAs not <br />                                                                              translated, avoiding indirect downstream regulatory effects. Loss of <br />                                                                              ntl expression CHX embryos used control efficient <br />                                                                              inhibition translation [71]. <br />  <br />                                                                              Plasmids used study <br />                                                                                 The zebrafish expression construct CS2+ Pou5f1-VP16 been <br />                                                                              described [49]. The following myc gene EST clones ordered <br />                                                                              RZPD (Deutsches Ressourcenzentrum fu&#194;&#168;r Genomforschung <br />                                                                              GmbH): IMAGp998E208991Q1 (mych), IRBOp991F125D2 (myca), <br />                                                                              IRBOp991G0414D2 (mycb), IRBOp991E0238D2 (mycl1a), IRAK- <br />                                                                              p961I20283Q2 (mycl1b). <br />  <br />                                                                              Whole-mount situ hybridization situ detection <br />                                                                              apoptosis <br />                                                                                 Whole-mount situ hybridization performed described <br />                                                                              [72]. <br />                                                                                 The ApopTag Peroxidase In Situ Apoptosis Detection Kit <br />                                                                              (Chemicon/Merck Millipore) used detect apoptotic cells in <br />                                                                              early embryonic stages. To investigate MZspg embryos <br />                                                                              develop enhanced apoptosis, MZspg WT embryos fixed <br />                                                                              4% PFA developmental stages 32-cell and <br />                                                                              bud stage. The fixed embryos incubated 100% methanol <br />                                                                              make permeable TUNEL staining. To analyze the <br />                                                                              influence Mych p53 phenotype, 105 pg mych mRNA <br />                                                                              and/or 4.2 ng p53-MO injected one-cell stage MZspg <br />                                                                              embryos. Non-injected WT MZspg embryos used as <br />                                                                              controls. Embryos fixed 10 hpf. To quantify apoptosis <br />                                                                              image z-stacks 5-14 stained embryos taken each <br />                                                                              experiment using transmitted light microscopy. Thereafter, <br />                                                                              maximum intensity projections z-stacks calculated for <br />                                                                              embryo using ImageJ (Rasband, W.S., ImageJ, U.S. National <br />                                                                              Institutes Health, Bethesda, Maryland, USA, http://imagj.nih. <br />                                                                              gov/ij/, 1997&#226;&#8364;&#8220;2012) apoptotic nuclei automatically <br />                                                                              detected using following set parameters Volocity Image <br /> Figure 7. Mych overexpression p53 knockdown suppress                     Analysis Software (PerkinElmer): (1) Find Objects Intensity; (2) <br /> cell death MZspg gastrulae. TUNEL staining detect apoptosis            exclude objects smaller 10 mm2; (3) separate touching objects <br /> at bud stage (A-J) subsequent computational image analysis (K)       greater 50 mm2 (Figure S7). <br /> quantification number apoptotic cells. The images show <br /> maximum intensity projections z-stacks taken single embryos <br /> with dorsal right. WT embryos display apoptosis,            Cell proliferation analysis <br /> whereas MZspg mutants increase cell death             To determine potential role Mych proliferation, 105 pg <br /> embryo (A-D). This mutant apoptosis phenotype partially repressed        mych mRNA and/or 4.2 ng p53-MO injected one-cell <br /> either mych overexpression (E,F) p53 morpholino knockdown              stage MZspg embryos. Non-injected WT MZspg embryos were <br /> (G,H). The co-injection mych mRNA p53-morpholino could <br />                                                                              used controls. Embryos fixed 4% PFA 10 hpf. The <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               9                            March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                        Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br /> embryos incubated 100% methanol make                        control evaluate stain background levels head, where <br /> permeable Sytox-Green DNA stain. Embryos stained               notail expressed (K-L). All myc genes gene specific <br /> in 2 mM Sytox-Green (Invitrogen) room temperature (20                 expression pattern mainly expressed proliferating and <br /> 25uC) 2 hours. Mitotic nuclei (prometa-, meta- anaphase             neural tissues. The expression c-Myc orthologous genes, <br /> nuclei) manually detected characteristic chromo-              myca mycb (A-D), L-Myc orthologous genes, mycl1a <br /> some arrangement dense DNA structure, resulting higher               mycl1b (G-J), partially complementary patterns. <br /> Sytox-Green intensity. The mitotic index proportion               (TIF) <br /> mitotic nuclei total number nuclei. For comparisons WT <br />                                                                             Figure S3 myc gene expression profiles transcrip- <br /> mitotic index normalized 1. <br />                                                                             tional regulation Pou5f1 Sox2. (A, C, E, G, I, K) <br />                                                                             Microarray time series data [47] myc gene expression profiles in <br /> Microinjections                                                             WT (white squares) MZspg (black triangles) 8&#226;&#8364;&#8220; <br />    mRNAs synthesized using mMessage mMachine kit                   hours development. The highest expression value gene <br /> (Invitrogen) according manufacturer&#226;&#8364;&#8482;s instructions. mRNA              normalized 100. (B, D, F, H, J, L) Microarray analysis of <br /> morpholino injected yolk freshly fertilized zygotes        transcriptomes MZspg embryos injected mRNA encoding <br /> (younger 15 minutes) mounted 1% (w/v) agarose ramps,                Pou5f1, developed presence CHX 64-cell stage on. <br /> using microinjection pipettes connected air pressure driven <br />                                                                             Data [47]. Non-injected MZspg control normalized <br /> microinjector. A volume 0.5&#226;&#8364;&#8220;1 nl, containing mRNA or <br />                                                                             1. <br /> morpholino 0.5% (v/v) phenol red water, injected <br />                                                                             (TIF) <br /> into zygote. <br />                                                                             Figure S4 Morphological analysis CHX treated and <br /> Quantitative RT-PCR                                                         pou5f1-VP16 mRNA injected control embryos. (A) Mor- <br />    60&#226;&#8364;&#8220;100 embryos sample snap-frozen liquid                     phological phenotype MZspg embryos treated CHX from <br /> nitrogen, total RNA isolated using RNA Easy kit                 64-cell stage developed WT control embryos reached <br /> (Qiagen). cDNA synthesized using Superscript III kit                    60% epiboly. Treated embryos arrested sphere stage, <br /> (Invitrogen). cDNA amplified using gene-specific primers                degenerate 60% epiboly equivalent age. (B) The <br /> and ABsolute SYBR Green Fluorescein (ABgene, Thermo                         injection 10 pg pou5f1-VP16 mRNA 1-cell MZspg embryos <br /> Scientific) according manufacturer&#226;&#8364;&#8482;s instructions Bio-             sufficient rescue MZspg phenotype, may <br /> Rad i-Cycler. Results calculated using ddCT method             ventralize embryo Pou5f1 overexpression WT do <br /> zebrafish ef1a normalization control. Primers used: mych               [74]. The experiment demonstrates pou5f1-VP16 injected <br /> forward 59-CCAGGCACTGGAGAGCGAAGAC-39, reverse 59-                           experiments concentrations considered <br /> AGAGCGTGTCAGGGTGCTGGAG-39; mycl1b forward 59-                               physiological embryonic development. <br /> GGTCAGAATCTCGCACCGACTC-39, reverse 59-ATGTGG-                               (TIF) <br /> AAACGCTTCATGCAGG-39; ef1a forward 59-CCTGG-                             Figure S5 Analysis mitotic index 90%-epiboly. <br /> GAGTGAAACGCTGATC-39, reverse 59-CCGATCTTCTT-                                Quantification proportion cells undergoing cell division in <br /> GATGTATGCGCTG-39. Calculations based Pfaffl [73].                    WT, MZspg MZspg injected mych mRNA and/or p53 <br />                                                                             morpholinos calculating mitotic index (ratio the <br /> Statistical analysis                                                        total number nuclei nuclei undergoing cell division). (A). <br />   Error bars Figures Standard Error Mean (SEM).              The calculated mitotic indices significantly different <br /> Significance evaluated using Student&#226;&#8364;&#8482;s T-test (Microsoft                different genotypes experimental conditions. <br /> Excel).                                                                     Mitotic index WT embryos set 1. Confocal microscopy <br />                                                                             Z-stacks taken animal region 90%-epiboly stage <br /> Supporting Information                                                      embryos, nuclei stained Sytox fluorescent DNA dye <br />                                                                             (B). Chromatin highly condensed meta- anaphase of <br /> Figure S1    Phylogenetic analysis zebrafish myc <br />                                                                             cell division, leads increase Sytox stain intensity <br /> genes. Phylogenetic tree myc family genes. In zebrafish two <br />                                                                             (B; arrows). <br /> paralogous genes exist L-myc, mycl1a mycl1b, c-myc, <br /> myca mycb. In addition, single copy mycn        (TIF) <br /> and mych genes. The later closely related N-myc c-        Figure S6 Testing mych morpholino functionality. (A- <br /> myc genes, known homologues vertebrate                  H) The functionality mych translation-blocking morpholino <br /> species. Trees built using phylip proml, 100 datasets          (ATG-MO) tested injecting fusion mRNA, the <br /> bootstrapping. The alignment clustalw. Sequences              MO target sequence fused gfp ORF start ATG, <br /> used are: hs-nmyc NP_005369.2, mm-nmyc NP_032735.2, ha-myc                  different concentrations ATG-MO one- <br /> NP_002458.2, mm-myc NP_034979.3, hs-lmyc NP_001028253.1,                    cell stage embryos. The GFP signal analyzed using <br /> mm-lmyc NP_032532.1, hs-smyc E10909, mm-smyc NP_034980.1,                   fluorescence microscopy (left panel) normal morphology <br /> mm-bmyc     NP_075815.2,       dr-myca     NP_571487.2,      dr-mycb        embryos morpholino injection documented using <br /> NP_956466.1, dr-mych XP_005166306.1, dr-nmyc NP_997779.1,                   transmitted light microscopy (right panel). The translation gfp <br /> dr-mycl1a NP_998102.1, dr-mycl1b NP_001038607.1.                            completely blocked injecting little 1.4 ng ATG- <br /> (TIF)                                                                       MO (C). For splice-blocking morpholino (Sp-MO) the <br /> Figure S2    WISH analysis myc gene expression                        functionality tested RT-PCR using pair primers <br /> 24 hpf. WISH analysis myca, mycb, mych, mycl1a mycl1b                overlapping second intron (I), splicing sites targeted <br /> expression WT (A-J). All embryos shown lateral (left              mych-Sp-MO. In WT 162 bp fragment reflects the <br /> column) dorsal (right column) view. notail expression used          proper splicing pre-mRNA, injection of <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             10                            March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                                              Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br /> 4.1 ng mych-Sp-MO detected fragment contains                              Table S3 Analysis mitotic index 90%-epiboly <br /> intron size increased 396 bp (I).                                                 Sytox nuclear stain. (Referring to: Figure S5) <br /> (TIF)                                                                                        (PDF) <br /> Figure S7 Quantification apoptosis WT MZspg                                        Table S4 Mych overexpression p53 knockdown <br /> embryos bud stage. Detection apoptotic cells TUNEL                                  suppress cell death MZspg gastrulae. (Referring to: <br /> staining (A) subsequent computational image analysis (B). The                            Figure 7) <br /> images lateral maximum intensity projection z-stack                              (PDF) <br /> taken single embryo. (B) The z-stack automatic <br /> object recognition using Volocity software (Perkin-Elmer),                             Acknowledgments <br /> most apoptotic cells marked red (B). <br /> (TIF)                                                                                        We thank Karen Lunde discussion initial analysis MZspg cell <br />                                                                                              death phenotype, Manuel Leichsenring work ChIP-Seq advice <br /> Table S1 mych potentially mycl1b directly                                       qPCR, Rebecca Moessner technical assistance. <br /> regulated Pou5f1. (Referring to: Figure 3) <br /> (PDF)                                                                                        Author Contributions <br /> Table S2      Analysis Mych contribution                                       Conceived designed experiments: KK EK WD. Performed the <br /> morphology MZspg mutant phenotype. (Referring                                         experiments: KK EK BW. Analyzed data: KK EK BW BP SBD DO <br /> to: Figure 5)                                                                                WD. Wrote paper: KK EK WD. <br /> (PDF) <br />  <br /> References <br />  1. Jacobson MD, Weil M, Raff MC (1997) Programmed cell death animal                      24. Li LH, Nerlov C, Prendergast G, MacGregor D, Ziff EB (1994) c-Myc represses <br />     development. Cell 88: 347&#226;&#8364;&#8220;354.                                                               transcription vivo novel mechanism dependent initiator element <br />  2. Kerr JF, Wyllie AH, Currie AR (1972) Apoptosis: basic biological                           Myc box II. Embo J 13: 4070&#226;&#8364;&#8220;4079. <br />     phenomenon wide-ranging implications tissue kinetics. Br J Cancer                25. Vennstrom B, Sheiness D, Zabielski J, Bishop JM (1982) Isolation and <br />     26: 239&#226;&#8364;&#8220;257.                                                                                 characterization c-myc, cellular homolog oncogene (v-myc) avian <br />  3. Metzstein MM, Stanfield GM, Horvitz HR (1998) Genetics programmed cell                    myelocytomatosis virus strain 29. J Virol 42: 773&#226;&#8364;&#8220;779. <br />     death C. elegans: past, present future. Trends Genet 14: 410&#226;&#8364;&#8220;416.                 26. Schwab M, Alitalo K, Klempnauer KH, Varmus HE, Bishop JM, et al. (1983) <br />  4. Abrams JM (1999) An emerging blueprint apoptosis Drosophila. Trends                   Amplified DNA limited homology myc cellular oncogene shared by <br />     Cell Biol 9: 435&#226;&#8364;&#8220;440.                                                                        human neuroblastoma cell lines neuroblastoma tumour. Nature 305: 245&#226;&#8364;&#8220; <br />  5. Taylor RC, Cullen SP, Martin SJ (2008) Apoptosis: controlled demolition               248. <br />     cellular level. Nat Rev Mol Cell Biol 9: 231&#226;&#8364;&#8220;241.                                        27. Nau MM, Brooks BJ, Battey J, Sausville E, Gazdar AF, et al. (1985) L-myc, a <br />  6. Danial NN, Korsmeyer SJ (2004) Cell death: critical control points. Cell 116:                new myc-related gene amplified expressed human small cell lung cancer. <br />     205&#226;&#8364;&#8220;219.                                                                                     Nature 318: 69&#226;&#8364;&#8220;73. <br />  7. Fuchs Y, Steller H (2011) Programmed cell death animal development                28. Dang CV (2012) MYC path cancer. Cell 149: 22&#226;&#8364;&#8220;35. <br />     disease. Cell 147: 742&#226;&#8364;&#8220;758.                                                              29. Swartling FJ (2012) Myc proteins brain tumor development maintenance. <br />  8. Krumschnabel G, Podrabsky JE (2009) Fish model systems study                   Ups J Med Sci 117: 122&#226;&#8364;&#8220;131. <br />     vertebrate apoptosis. Apoptosis 14: 1&#226;&#8364;&#8220;21.                                                30. Wolfer A, Ramaswamy S (2011) MYC metastasis. Cancer Res 71: 2034&#226;&#8364;&#8220; <br />  9. Eimon PM, Kratz E, Varfolomeev E, Hymowitz SG, Stern H, et al. (2006)                        2037. <br />     Delineation cell-extrinsic apoptosis pathway zebrafish. Cell Death         31. Domashenko AD, Latham KE, Hatton KS (1997) Expression myc-family, <br />     Differ 13: 1619&#226;&#8364;&#8220;1630.                                                                        myc-interacting, myc-target genes preimplantation mouse develop- <br /> 10. Inohara N, Nunez G (2000) Genes homology mammalian apoptosis                         ment. Mol Reprod Dev 47: 57&#226;&#8364;&#8220;65. <br />     regulators identified zebrafish. Cell Death Differ 7: 509&#226;&#8364;&#8220;510.                        32. Downs KM, Martin GR, Bishop JM (1989) Contrasting patterns myc N- <br /> 11. Kratz E, Eimon PM, Mukhyala K, Stern H, Zha J, et al. (2006) Functional                      myc expression gastrulation mouse embryo. Genes Dev 3: 860&#226;&#8364;&#8220; <br />     characterization Bcl-2 gene family zebrafish. Cell Death Differ 13:            869. <br />     1631&#226;&#8364;&#8220;1640.                                                                               33. Mugrauer G, Alt FW, Ekblom P (1988) N-myc proto-oncogene expression <br /> 12. Eimon PM, Ashkenazi A (2010) The zebrafish model organism study                 organogenesis developing mouse revealed situ <br />     apoptosis. Apoptosis 15: 331&#226;&#8364;&#8220;349.                                                         hybridization. J Cell Biol 107: 1325&#226;&#8364;&#8220;1335. <br /> 13. Storer NY, Zon LI (2010) Zebrafish models p53 functions. Cold Spring Harb             34. Zimmerman KA, Yancopoulos GD, Collum RG, Smith RK, Kohl NE, et al. <br />     Perspect Biol 2: a001123.                                                                    (1986) Differential expression myc family genes murine development. <br /> 14. Eilers M, Schirm S, Bishop JM (1991) The MYC protein activates transcription                 Nature 319: 780&#226;&#8364;&#8220;783. <br />     alpha-prothymosin gene. Embo J 10: 133&#226;&#8364;&#8220;141.                                       35. Pelengaris S, Khan M (2003) The faces c-MYC. Arch Biochem Biophys <br /> 15. Miner JH, Wold BJ (1991) c-myc inhibition MyoD myogenin-initiated                     416: 129&#226;&#8364;&#8220;136. <br />     myogenic differentiation. Mol Cell Biol 11: 2842&#226;&#8364;&#8220;2851.                                   36. Davis AC, Wims M, Spotts GD, Hann SR, Bradley A (1993) A null c-myc <br /> 16. Coppola JA, Cole MD (1986) Constitutive c-myc oncogene expression blocks                     mutation causes lethality 10.5 days gestation homozygotes and <br />     mouse erythroleukaemia cell differentiation commitment. Nature 320:                  reduced fertility heterozygous female mice. Genes Dev 7: 671&#226;&#8364;&#8220;682. <br />     760&#226;&#8364;&#8220;763.                                                                                 37. Takahashi K, Yamanaka S (2006) Induction pluripotent stem cells from <br /> 17. van Riggelen J, Yetil A, Felsher DW (2010) MYC regulator ribosome                    mouse embryonic adult fibroblast cultures defined factors. Cell 126: 663&#226;&#8364;&#8220; <br />     biogenesis protein synthesis. Nat Rev Cancer 10: 301&#226;&#8364;&#8220;309.                                676. <br /> 18. Evan GI, Wyllie AH, Gilbert CS, Littlewood TD, Land H, et al. (1992)                     38. Boyer LA, Lee TI, Cole MF, Johnstone SE, Levine SS, et al. (2005) Core <br />     Induction apoptosis fibroblasts c-myc protein. Cell 69: 119&#226;&#8364;&#8220;128.                    transcriptional regulatory circuitry human embryonic stem cells. Cell 122: <br /> 19. Blackwood EM, Eisenman RN (1991) Max: helix-loop-helix zipper protein                      947&#226;&#8364;&#8220;956. <br />     forms sequence-specific DNA-binding complex Myc. Science 251:                39. Chappell J, Sun Y, Singh A, Dalton S (2013) MYC/MAX control ERK <br />     1211&#226;&#8364;&#8220;1217.                                                                                   signaling pluripotency regulation dual-specificity phosphatases 2 and <br /> 20. Prendergast GC, Lawe D, Ziff EB (1991) Association Myn, murine                        7. Genes Dev 27: 725&#226;&#8364;&#8220;733. <br />     homolog max, c-Myc stimulates methylation-sensitive DNA binding              40. Loh YH, Wu Q, Chew JL, Vega VB, Zhang W, et al. (2006) The Oct4 and <br />     ras cotransformation. Cell 65: 395&#226;&#8364;&#8220;407.                                                      Nanog transcription network regulates pluripotency mouse embryonic stem <br /> 21. Amati B, Dalton S, Brooks MW, Littlewood TD, Evan GI, et al. (1992)                          cells. Nat Genet 38: 431&#226;&#8364;&#8220;440. <br />     Transcriptional activation human c-Myc oncoprotein yeast requires              41. Meijer AH, van dSAM, Cunha C, Lamers GE, Laplante MA, et al. (2008) <br />     interaction Max. Nature 359: 423&#226;&#8364;&#8220;426.                                                   Identification real-time imaging myc-expressing neutrophil population <br /> 22. Amin C, Wagner AJ, Hay N (1993) Sequence-specific transcriptional activation                 involved inflammation mycobacterial granuloma formation zebrafish. <br />     Myc repression Max. Mol Cell Biol 13: 383&#226;&#8364;&#8220;390.                                     Dev Comp Immunol 32: 36&#226;&#8364;&#8220;49. <br /> 23. Herkert B, Eilers M (2010) Transcriptional repression: dark myc.             42. Schreiber-Agus N, Horner J, Torres R, Chiu FC, DePinho RA (1993) Zebra fish <br />     Genes Cancer 1: 580&#226;&#8364;&#8220;586.                                                                     myc family max genes: differential expression oncogenic activity <br />                                                                                                  vertebrate evolution. Mol Cell Biol 13: 2765&#226;&#8364;&#8220;2775. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                              11                                 March 2014 | Volume 9 | Issue 3 | e92356 <br />                                                                                                                                Pou5f1 Control Zebrafish myc Expression <br />  <br />  <br /> 43. Amores A, Force A, Yan YL, Joly L, Amemiya C, et al. (1998) Zebrafish hox                 59. Wierstra I, Alves J (2008) The c-myc promoter: MysterY challenge. Adv <br />     clusters vertebrate genome evolution. Science 282: 1711&#226;&#8364;&#8220;1714.                             Cancer Res 99: 113&#226;&#8364;&#8220;333. <br /> 44. Loeb-Hennard C, Kremmer E, Bally-Cuif L (2005) Prominent transcription                 60. Chen X, Vega VB, Ng HH (2008) Transcriptional regulatory networks in <br />     zebrafish N-myc (nmyc1) tectal retinal growth zones embryonic                   embryonic stem cells. Cold Spring Harb Symp Quant Biol 73: 203&#226;&#8364;&#8220;209. <br />     early larval development. Gene Expr Patterns 5: 341&#226;&#8364;&#8220;347.                              61. Guo Y, Mantel C, Hromas RA, Broxmeyer HE (2008) Oct-4 critical for <br /> 45. Hong SK, Tsang M, Dawid IB (2008) The mych gene required neural crest                  survival/antiapoptosis murine embryonic stem cells subjected stress: effects <br />     survival zebrafish development. PLoS One 3: e2029.                                     associated Stat3/survivin. Stem Cells 26: 30&#226;&#8364;&#8220;34. <br /> 46. Thisse B, Thisse C (2004) Fast Release Clones: A High Throughput Expression               62. Chen T, Du J, Lu G (2012) Cell growth arrest apoptosis induced Oct4 or <br />     Analysis.                                                                                     Nanog knockdown mouse embryonic stem cells: possible role Trp53. Mol <br /> 47. Onichtchouk D, Geier F, Polok B, Messerschmidt DM, Mossner R, et al. (2010)                   Biol Rep 39: 1855&#226;&#8364;&#8220;1861. <br />     Zebrafish Pou5f1-dependent transcriptional networks temporal control                63. Wang YD, Cai N, Wu XL, Cao HZ, Xie LL, et al. (2013) OCT4 promotes <br />     early development. Mol Syst Biol 6: 354.                                                      tumorigenesis inhibits apoptosis cervical cancer cells miR-125b/BAK1 <br /> 48. Schulte-Merker S, van EFJ, Halpern ME, Kimmel CB, Nusslein-Volhard C                          pathway. Cell Death Dis 4: e760. <br />     (1994) tail (ntl) zebrafish homologue mouse T (Brachyury) gene.          64. Lin Y, Yang Y, Li W, Chen Q, Li J, et al. (2012) Reciprocal regulation Akt <br />     Development 120: 1009&#226;&#8364;&#8220;1015.                                                                   Oct4 promotes self-renewal survival embryonal carcinoma cells. <br /> 49. Lunde K, Belting HG, Driever W (2004) Zebrafish pou5f1/pou2, homolog                       Mol Cell 48: 627&#226;&#8364;&#8220;640. <br />     mammalian Oct4, functions endoderm specification cascade. Curr Biol                65. Kotkamp K, Mo&#194;&#168;ssner R, Allen A, Onichtchouk D, Driever W (2013) A Pou5f1/ <br />     14: 48&#226;&#8364;&#8220;55.                                                                                    Oct4 dependent Klf2a, Klf2b, Klf17 regulatory sub-network contributes to <br /> 50. Leichsenring M, Maes J, Mossner R, Driever W, Onichtchouk D (2013) Pou5f1                     EVL ectoderm development zebrafish embryogenesis. http://dx.doi. <br />     transcription factor controls zygotic gene activation vertebrates. Science 341:            org/10.1016/j.ydbio.2013.10.025i. <br />     1005&#226;&#8364;&#8220;1009.                                                                                66. Adhikary S, Eilers M (2005) Transcriptional regulation transformation by <br /> 51. Lachnit M, Kur E, Driever W (2008) Alterations cytoskeleton               Myc proteins. Nat Rev Mol Cell Biol 6: 635&#226;&#8364;&#8220;645. <br />     embryonic lineages contribute epiboly defect Pou5f1/Oct4 deficient              67. Slack A, Chen Z, Tonelli R, Pule M, Hunt L, et al. (2005) The p53 regulatory <br />     MZspg zebrafish embryos. Dev Biol 315: 1&#226;&#8364;&#8220;17.                                                  gene MDM2 direct transcriptional target MYCN neuroblastoma. Proc <br /> 52. Reim G, Brand M (2006) Maternal control vertebrate dorsoventral axis                       Natl Acad Sci U S A 102: 731&#226;&#8364;&#8220;736. <br />     formation epiboly POU domain protein Spg/Pou2/Oct4.                            68. Haupt S, Berger M, Goldberg Z, Haupt Y (2003) Apoptosis - p53 network. <br />     Development 133: 2757&#226;&#8364;&#8220;2770.                                                                   J Cell Sci 116: 4077&#226;&#8364;&#8220;4085. <br /> 53. Song S, Eckerle S, Onichtchouk D, Marrs JA, Nitschke R, et al. (2013) Pou5f1-             69. Belting HG, Hauptmann G, Meyer D, Abdelilah-Seyfried S, Chitnis A, et al. <br />     dependent EGF expression controls E-cadherin endo <br /> </body></html>