<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Autophagy 8:7, 1124&#226;&#8364;&#8220;1135; July 2012;            G   2012 Landes Bioscience <br />  <br />  <br />  <br />  <br />                Loss starvation-induced gene Rack1 <br />                leads glycogen deficiency impaired <br />                   autophagic responses Drosophila <br /> Bal&#195;&#161;zs &#195;&#8240;rdi,1&#226;&#8364;&#160; P&#195;&#169;ter Nagy,1&#226;&#8364;&#160; &#195;?gnes Zvara,2 &#195;?gnes Varga,1 Karolina Pircs,1 Dalma M&#195;&#169;nesi,2 L&#195;&#161;szl&#195;&#179; G. Pusk&#195;&#161;s2 G&#195;&#161;bor Juh&#195;&#161;sz1,* <br /> 1 <br />     Department Anatomy, Cell Developmental Biology; E&#195;&#182;tv&#195;&#182;s Lor&#195;&#161;nd University; Budapest, Hungary; 2Laboratory Functional Genomics; Biological Research Center; Tzeged, Hungary <br />  <br />                                                                     &#226;&#8364;&#160; <br />                                                                     These authors contributed equally work. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           &#194;&#169;2012 Landes Bioscience. Do distribute. <br />         Keywords: antimicrobial peptides, Atg8, autophagy, Drosophila, fat body, glycogen, GSK-3B, microarray, Rack1, starvation <br />  <br />  <br />               Autophagy delivers cytoplasmic material lysosomal degradation eukaryotic cells. Starvation induces high levels of <br />               autophagy promote <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>survival</span> lack nutrients. We compared genome-wide transcriptional profiles fed and <br />               starved control, autophagy-deficient Atg7 Atg1 null mutant Drosophila larvae search novel regulators of <br />               autophagy. Genes involved catabolic processes including autophagy transcriptionally upregulated cases. <br />               We detected repression genes involved DNA replication autophagy mutants compared control animals. <br />               The expression Rack1 (receptor activated protein kinase C 1) increased 4.1- 5.5-fold <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation in <br />               genotypes. The scaffold protein Rack1 plays role wide range processes including translation, cell <br />               adhesion migration, <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> cancer. Loss Rack1 led attenuated autophagic response starvation, and <br />               glycogen stores decreased 11.8-fold Rack1 mutant cells. Endogenous Rack1 partially colocalized GFP-Atg8a <br />               early autophagic structures ultrastructural level, suggesting involvement autophagosome formation. <br />               Endogenous Rack1 showed high degree colocalization glycogen particles larval fat body, with <br />               Shaggy, Drosophila homolog glycogen synthase kinase 3B (GSK-3B). Our results, time, demonstrated <br />               fundamental role Rack1 autophagy glycogen synthesis. <br />  <br />  <br />  <br />  <br />                                      Introduction                                              TOR (target rapamycin), kinase coordinates growth- <br />                                                                                                promoting stimuli availability nutrients, ATP and <br /> The proper balance anabolic catabolic processes ensures                                 oxygen. Inactivation TOR rapidly activates Atg1 complex, <br /> cellular organism-wide homeostasis physiological                                     resulting autophagy induction.7 Autophagy requires a <br /> conditions. Major intracellular catabolic pathways include                                 specific phosphatidylinositol-3-kinase (PtdIns3K) complex, the <br /> ubiquitin-proteasome system, responsible regulated                                     transmembrane protein Atg9 factors involved cycling <br /> degradation selected proteins autophagy, capable                                     cellular reservoirs phagophore assembly sites, two <br /> degrading intracellular macromolecules                                      ubiquitin-like protein conjugation systems ensure lipidation <br /> organelles. The major pathway autophagy utilizes double-                                    membrane association ubiquitin-like protein Atg8. <br /> membrane vesicles called autophagosomes deliver cytoplasmic                                 Atg8 associated surface autophagosomes similar a <br /> cargo degradation lysosomes.1 Basal levels autophagy                             coat protein. Atg8 needs cleaved outer <br /> required prevent accumulation abnormal protein                                       membrane allow fusion autophagosomes lysosomes <br /> aggregates detrimental cell function survival,                                endosomes, Atg8 pool associated inner <br /> especially long-lived cells neurons.2-5 Autophagic                                  membrane degraded sequestered cargo autolyso- <br /> activity tightly controlled cells: factors increasing cell                           somes.8,9 This way, half Atg8 protein molecules are <br /> growth rate usually decrease, <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> growth factor                                  broken autophagosomal cycle.10 <br /> deprivation dramatically increases levels autophagy.1                                      Yeast studies levels Atg8 rate- <br />    A set 20 evolutionarily conserved genes required                               limiting autophagy, upregulation gene is <br /> autophagy identified yeast. These Atg (autophagy-                                  normally observed induction.1 In higher eukaryotes, <br /> related) gene products orchestrate formation early                                      increased transcription Atg8 homologs Atg genes <br /> autophagic structures autophagosome generation takes                                     documented certain settings, example murine <br /> place.6 The Atg1 protein kinase complex directly controlled                              muscles, Drosophila fat body salivary gland cells the <br />  <br />  <br />     *Correspondence <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span>: G&#195;&#161;bor Juh&#195;&#161;sz; Email: szmrt@elte.hu <br />     Submitted: 09/14/11; Revised: 03/19/12; Accepted: 03/19/12 <br />     http://dx.doi.org/10.4161/auto.20069 <br />  <br />  <br /> 1124                                                                                  Autophagy                                                                   Volume 8 Issue 7 <br />                                                                                                                                  RESOURCE <br />  <br />  <br />  <br />  <br /> onset metamorphosis, cases autophagy             autophagy-deficient <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> Atg7 (on similar genetic <br /> induction.11-15 Basal levels Atg proteins extent        background control larvae) Atg1 (<span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> distinct genetic <br /> autophagy induction determine transcriptional        background), purpose finding gene networks are <br /> upregulation genes necessary response. The     specifically regulated starved autophagy mutants.3,21 Genes that <br /> presence multiple homologs single yeast gene         showed two-fold induction repression p value <br /> supply proteins support autophagy induction. Inter-         , 0.05 considered regulated. We expression <br /> estingly, Atg8 gene family expanded higher eukaryotes,     2,819 vs. 3,690 3,971 13,613 genes (represented by <br /> for example members Drosophila (Atg8a                unique FBgn identifiers)22 starvation-regulated controls, <br /> and Atg8b), mammals specialized functions            Atg7 Atg1 mutants, respectively. Please Table S1 for <br /> during autophagosome formation (GABARAP, GABARAPL1,                   detailed results. <br /> GABARAPL2, GABARAPL3 MAP1LC3&#226;&#8364;&#8221;this gene                        Genes encoding antimicrobial peptides Drosomycin <br /> also produces different protein isoforms).6,16                   IM10 (immune-induced molecule 10) most <br />  <br />  <br />  <br />  <br />                                                                                                                                              &#194;&#169;2012 Landes Bioscience. Do distribute. <br />    Drosophila excellent model autophagy studies,         highly upregulated genes genotypes (Table S2). <br /> polyploid tissues like fat body grow rapidly larval    Growth/insulin signaling inhibited response starvation, <br /> stages, store nutrients support metamorphosis later on.        activates transcription factor Foxo (forkhead box, sub- <br /> Therefore, low levels autophagy observed feeding    group O).15 Induction immune effector molecules is <br /> period larval stages L1 (approximately 24&#226;&#8364;&#8220;48 h egg           directly controlled Foxo.23 The FoxA/Forkhead transcription <br /> laying&#226;&#8364;&#8221;AEL), L2 (48&#226;&#8364;&#8220;72 h AEL) L3 (72&#226;&#8364;&#8220;120 h                factor target gene CG6770 transcriptionally induced <br /> AEL). Larvae leave food 108 h AEL (the onset         inhibition TOR induced 37- 95-fold.19,24 <br /> the wandering period) proper, dry place pupariation,    Another highly upregulated gene CG7224, encoding small, <br /> which observed 120 h AEL. Robust induction               118-amino acid protein. As transcriptionally induced 19.5- <br /> autophagy seen larval fat body midgut wandering      30.5-fold different genotypes, named gene Sirup <br /> larvae, response starvation earlier stages.17,18      (starvation-upregulated protein). Interestingly, expression Sirup <br />    We previously used transcription profiling search      paralog CG15283 induced developmental <br /> gene expression changes dissected fat bodies feeding        autophagy fat body.15 <br /> wandering animals. By following expression levels 3,200           Of 20 genes showing strongest downregulation, only <br /> genes, Atg8 homologs upregulated,           genotypes: CG15615 encoding an <br /> have successfully shown downregulated gene FKBP39            oxidoreductase (36- 61-fold repression) CG15155 encod- <br /> (FK506-binding protein 39 kD) inhibitor autophagy.        ing transferase transfers acyl groups amino- <br /> FKBP39 likely acts inhibiting transcription factor Foxo,       acyl groups (30- 60-fold repression; Table S3). The biological <br /> and time Foxo important positive    processes gene products involved not <br /> regulator autophagy.15 Since then, Foxo-dependent transcrip-       known, phenotypic data available genes. <br /> tional upregulation Atg8 homologs autophagy           We performed gene ontology (GO) analysis significantly <br /> described murine muscles.11,12                                     regulated genes different genotypes (Fig. 1A B; <br />    We reasoned similar strategy applied looking    Tables S4 S5 details). The analysis revealed genes <br /> at gene expression changes starvation. An earlier study        belonging 10 enriched GO categories specifically induced <br /> reported whole-genome analysis starvation responses 48 h      autophagy mutants (please Table S6 list induced <br /> AEL (L2 stage) larvae, transcriptional induction     genes 10 Atg mutant-specific enriched GO categories). <br /> only Atg8a (annotated CG1534 time) Atg9     Some GO categories high hierarchy GO <br /> autophagy genes.19 We decided use sensitive newer           terms (such GO0009056: catabolic process) included genes <br /> generation microarray platform test 86 h AEL (L3 stage) larvae     present other, specific GO groups enriched <br /> as used cases studies starvation-induced     genotypes, clear limitation GO analyses. <br /> autophagy. In addition, time point developmental    Nevertheless, groups include genes involved the <br /> milestone known &#226;&#8364;&#732;70 h change.&#226;&#8364;&#8482; This simply refers       metabolism purine nucleotides various catabolic <br /> observation larvae 70 h AEL die days    processes, CG9363 encoding enzyme involved the <br /> complete starvation, fraction larvae past 70 h AEL     breakdown tyrosine L-phenylalanine. Numerous genes <br /> (near L2- L3-molt) presumably accumulated            encoding members ubiquitin-dependent protein catabolic <br /> appropriate <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> stores sustain metamorphosis able      process here, including CHIP, CG1950, <br /> pupariate rise small adults.20                            CG4661, CG18341 Ubc84D.22 Only 3 enriched GO terms <br />                                                                       specific autophagy mutants repressed genes <br />                             Results                                   (please Table S7 list repressed genes 3 Atg <br />                                                                       mutant-specific enriched GO categories). Genes these <br /> Microarray analysis starvation-induced autophagy Droso-         categories encode proteins involved DNA metabolism, <br /> phila larvae. We compared genome-wide expression changes           amplification replication, DNA polymerases <br /> 86 h AEL control larvae subjected 4-h complete starvation     CG8142, DNA polymerase 73 kD 180 kD, DNA <br /> their well-fed siblings. We analyzed previously described   polymerase delta, essential DNA replication factors Mcm5 <br />  <br />  <br /> www.landesbioscience.com                                       Autophagy                                                             1125 <br />                     Figure 1. Transcriptional analysis of <br />                    starvation responses. The number <br />                    gene ontology (GO) terms <br />                    enriched (p , 0.05) among <br />                    significantly upregulated <br />                    (A) downregulated (B) genes <br />                    (minimum two-fold change in <br />                    expression level 4-h <br />                    complete starvation, p , 0.05) <br />                    control, Atg7 Atg1 mutant <br />                    larvae. See text Tables S4 <br />                    S5 details. (C) lists star- <br />                    vation-induced transcriptional <br />                    changes (gene expression levels in <br />  <br />  <br />  <br />  <br />                                                            &#194;&#169;2012 Landes Bioscience. Do distribute. <br />                    starved animals relative to <br />                    expression levels detected in <br />                    fed animals) Drosophila homo- <br />                    logs yeast human Atg genes, <br />                    involved core mechanism <br />                    autophagy. Atg8a encoding a <br />                    ubiquitin-like protein showed <br />                    highest induction star- <br />                    vation. Interestingly, highest <br />                    average upregulation observed <br />                    genes encoding members <br />                    Atg9 cycling complex. <br />                    (D) Quantitative real-time (QT) PCR <br />                    analysis fat bodies dissected <br />                    fed starved control larvae. <br />                    Numbers represent fold change <br />                    values, nonsignificant changes <br />                    crossed (C D). <br />  <br />  <br />  <br />  <br /> 1126   Autophagy                       Volume 8 Issue 7 <br />  (Minichromosome maintenance 5), Mcm10, disc proliferation               autophagy represent negative feedback loop acts to <br /> abnormal, replication factor C subunits RfC3 RfC38, double          prevent hyperactivation. Elucidation hypothetical mecha- <br /> parked, DNA ligase I, SuUR (suppressor under-replication),           nism needs studies. <br /> mus209/PCNA (proliferating cell nuclear antigen) origin                 Interestingly, genes encoding members pre-autophago- <br /> recognition complex subunit 2.22                                        somal cycling complex showed strongest average induction, <br />    Interestingly, detected genes significantly             3.2- 6.2-fold. Atg9 transmembrane <br /> regulated 4-h complete starvation compared             protein Atg gene products, shown cycle <br /> an earlier study.19 Zinke et al. reported 369 279 genes             early autophagic structures potential intracellular <br /> significantly induced repressed, respectively,        membrane sources ensure membrane transport forming <br /> that 1,584 1,235 genes significantly up- down-              autophagosomes. Atg2 Atg18 required cycling of <br /> regulated control larvae, respectively. Comparing list hits   Atg9 yeast.29 <br /> revealed 170 genes induced 134 repressed             Half genes (5/10) encoding members ubiquitin- <br />  <br />  <br />  <br />  <br />                                                                                                                                                  &#194;&#169;2012 Landes Bioscience. Do distribute. <br /> both studies (please Tables S8 S9 list genes),   like protein conjugation systems did an <br /> which half genes identified Zinke et al. While      upregulation. Atg4a, Atg7, Atg8b Atg16 moderately <br /> some differences likely different genetic       induced, highest upregulation Atg genes was <br /> backgrounds larval stages tested, convinced         detected Atg8a, 8.4-fold average. <br /> approximately 4.3-fold higher discovery rate           Atg8a itself specific cargo selectively degraded during <br /> use different, sensitive microarray. This          autophagy. Additional substrates described, as <br /> supported previous study compared different microarray        p62/Ref(2)P, intracellular receptor ubiquitinated <br /> technologies long oligonucleotide platform           proteins selective autophagy, addition scaffolding roles <br /> (such case study utilizing arrays printed 60-     multiple signaling pathways. p62 binds Atg8 homologs, it <br /> mer oligos) sensitive produces lower levels technical    continuously degraded autophagy.30 We p62 <br /> variability compared cDNA-based systems (used              transcriptionally induced 6-fold starvation, potentially in <br /> Zinke et al.).25                                                        compensate accelerated rate degradation during <br />    Most autophagy genes transcriptionally induced            increased autophagy. A moderate upregulation detected <br /> starvation. Several GO terms include Atg genes (such            Blue cheese/Alfy, protein product involved the <br /> GO0048102: autophagic cell death) enriched             selective degradation cytosolic protein aggregates.31 <br /> the upregulated genes genotypes. To focus              Next sought expand findings obtained by <br /> autophagy, looked expression Atg gene homologs.            microarray analysis starved larvae focusing fat <br /> Many genes showed increased induction autophagy             body. We followed expression selected genes by <br /> mutants compared controls (Fig. 1C). Expression           quantitative real-time PCRs, comparing fat bodies dissected from <br /> Atg7 Atg1 loci observed mutants;       starved vs. fed control larvae (Fig. 1D). These QT-PCR <br /> alleles previously characterized genetic nulls,             experiments supported important findings: <br /> presumably truncated versions wild-type transcripts             genes encoding members Atg1 kinase lipid <br /> detected.3,21                                                           kinase complexes EDTP showed highest upregulation (6.5- <br />    Of genes encoding members Atg1 kinase complex,            fold), moderate compared genes that <br /> Atg1, FIP200 Atg101 showed moderate induction,          showed strongest induction. Atg8a far highly <br /> levels Atg13 did change. This agreement recent        upregulated gene (169-fold), members membrane <br /> findings Atg13 continuously degraded ubiquitin-          transport group strongly induced (11- 35-fold). The <br /> proteasome system, stabilized binding Atg101        expression Sirup increased 60.4-fold. There also <br /> during autophagy induction.26,27 Therefore, Atg13 protein levels        important differences, case Atg5 showed a <br /> appear mainly regulated post-translational level. It    7.6-fold upregulation fat bodies significant change was <br /> important note Atg101 showed highest upregulation        detected larvae, probably masked other <br /> this group (on average 3.5-fold 3 different genotypes),      tissues. <br /> it stronger induction autophagy mutants.           Rack1 required autophagic response starvation. <br /> In controls, FIP200 shows highest induction group           We selected set genes studies starvation- <br /> (4.6-fold).                                                             regulated genotypes, autophagic function had <br />    Genes encoding members autophagy-specific PtdIns3K            assigned spite having already <br /> complex (Vps34, Vps15, Atg6, Atg14) showed milder               characterized extent. For example, Rack1 transcription <br /> response: Vps34 codes lipid kinase itself         increased 4.1-, 5.5- 4.1-fold controls, Atg7 Atg1 <br /> upregulated genotypes. To surprise, EDTP (Egg-         mutants, respectively. QT-PCR analysis showed Rack1 <br /> derived tyrosine phosphatase) showed highest average                upregulated 21.6-fold fat body starvation. <br /> induction lipid signaling group: 2.9-fold. This gene                Silencing Rack1 expression reduced autophagy based on <br /> encodes Drosophila homolog lipid phosphatase Jumpy,          mCherry-Atg8a (Fig. 2A N), Lysotracker Red staining and <br /> a potent antagonist autophagy-promoting activity Vps34        Lamp-GFP, reporter lysosomes.32 Numerous small, <br /> in mammals.28 The upregulation potential inhibitor              presumably inactive Lamp-GFP positive lysosomes seen in <br />  <br />  <br /> www.landesbioscience.com                                         Autophagy                                                               1127 <br />  fat body cells well-fed control larvae, strongly              continuously degraded autophagy aggregated <br /> acidic revealed lack Lysotracker Red labeling (Fig. 2B). In            ubiquitinated proteins.30 <br /> contrast, large Lamp-GFP Lysotracker-positive autolysosomes                    Rack1 involved glycogen synthesis. Glycogen stores are <br /> were formed starvation (Fig. 2C). Silencing Rack1                     normally present fat body, midgut muscle cells in <br /> reduced number size Lysotracker Lamp-GFP                         Drosophila larvae. A striking <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> Rack1 mutant fat <br /> positive autolysosomes formed starvation (Fig. 2D,                   body cells 11.8-fold reduction glycogen particle area <br /> also Fig. 2O&#226;&#8364;&#8220;Q statistics).                                                 compared heterozygous controls electron micrographs, <br />    To expand RNAi studies analyzed                      effect partially rescued Rack1 expression <br /> previously described null mutant Rack1[1.8].33 Using                   (Fig. 2J&#226;&#8364;&#8220;M; Fig. 2S statistics). Reduced glycogen stores <br /> mCherry-Atg8a Lysotracker assays, observed                       likely caused defects glycogen synthesis induced <br /> strong reduction autophagy Rack1 mutants compared                    4 h starvation 20% sucrose accelerated <br /> heterozygous controls (Fig. 2E&#226;&#8364;&#8220;H; Fig. 2N O                    degradation, glycogen particles observed fat <br />  <br />  <br />  <br />  <br />                                                                                                                                                                &#194;&#169;2012 Landes Bioscience. Do distribute. <br /> statistics). As final proof role Rack1 autophagy,               bodies well-fed control Rack1 mutant larvae (Fig. S1A and <br /> we generated transgenic animals expressing Rack1                    S1B). In cases, lack glycogen probably due <br /> C-terminal FLAG-HA tag using DNA construct                        high-protein diet routinely feed larvae before <br /> ongoing Drosophila proteomics project. Heat shock-mediated                      starving them, supplementing medium live yeast <br /> expression Rack1 mutant larvae partially rescued                    paste. Taking advantage monoclonal antibody specific for <br /> autophagy defect (Fig. 2I O). To avoid possible effect             glycogen, able silencing Rack1 <br /> heat shock autophagy, genotypes heat shocked               decreased glycogen stores cell-autonomous manner larvae <br /> Lysotracker electron microscopy experiments (see below).                    fed high sugar-high protein diet, is, medium supplemented <br />    Similar findings, ultrastructural analysis showed               live yeast paste prepared 20% sugar; similar phenotypes <br /> that numerous autophagosomes large autolysosomes                      observed mutants (Fig. 2T&#226;&#8364;&#8220;V). <br /> were observed heterozygous controls, cumulative area                     Endogenous Rack1 localizes early autophagic structures <br /> these structures decreased cross-sections Rack1 mutant                glycogen. Immunofluorescence analysis revealed Rack1 <br /> fat body cells, heat shock-mediated expression Rack1                  present numerous cytoplasmic particles tissues. <br /> mutant larvae showed partial rescue (Fig. 2J&#226;&#8364;&#8220;M;                     Knockdown Rack1 expression reduced staining fat <br /> Fig. 2R statistics). The partial rescue effect reflect                  body midgut cells (Fig. 3A B). To assess potential <br /> insufficient expression levels, likely caused              role Rack1 autophagy, performed colocalization <br /> the C-terminal tag interfering protein activity. Still,           experiments. Atg8a fusion proteins best characterized <br /> reduction autophagosome area completely rescued                 fluorescent reporters autophagosome generation. We found <br /> experiments (p = 0.02 mutant rescue, p = 0.84                   endogenous Rack1 colocalized GFP-Atg8a 5.8% of <br /> control rescue sample pairs, respectively), suggesting                 cases (n = 10/171 dots) (Fig. 3C). Vice versa, 6.1% of <br /> Rack1 required efficient autophagosome formation (Fig. 2L).              GFP-Atg8a particles colocalized Rack1 (n = 11/181 dots). In <br />    Interestingly, Rack1 dispensable                       contrast, single case colocalization observed between <br /> developmental autophagy induced fat body midgut cells                    endogenous Rack1 lysosome reporter Lamp1-GFP <br /> prior metamorphosis late L3 stage wandering animals                       (Fig. 3D). We used immunogold labeling endogenous <br /> (Fig. S1C S1D). In contrast, p62 aggregates accumulated                  Rack1 localize ultrastructural level. We that <br /> Rack1 RNAi cells well-fed early L3 stage animals (Fig. S1E                   Rack1 associated 7.7% pre-autophagosomal phago- <br /> and S1F). This finding suggests Rack1 involved                  phores autophagosomes (n = 17/221) (Fig. 3E; also <br /> basal autophagy, ubiquitin- Atg8-binding protein p62                 Fig. 3G). Again, labeling lysosomes seen. <br />  <br />  Figure 2 (See opposite page). Rack1 required autophagic response starvation, efficient glycogen synthesis. (A) Knockdown <br />  Rack1 results reduction autophagy response starvation. Please compare number mCherry-positive dots RNAi cells vs. control cells in <br />  panel (red; red channel shown greyscale insets). RNAi expressing cells marked coexpression GFP (green). Cell nuclei <br />  labeled DAPI (blue). (B) Numerous small Lamp1-GFP dots seen fat body cells well-fed control larvae, co-labeled <br />  Lysotracker Red. (C) In contrast, large Lamp1-GFP Lysotracker-positive autolysosomes formed starvation. (D) Silencing Rack1 strongly <br />  reduces number Lysotracker particles formed starvation; note size Lamp1-GFP dots similar fed control <br />  cells. (E F) The number starvation-induced mCherry-Atg8a dots reduced Rack1 null mutant fat bodies compared controls. <br />  (G H) Similarly, number Lysotracker-positive autolysosomes reduced Rack1 mutants compared controls. (I) Starvation-induced <br />  autophagy rescued heat shock-mediated expression Rack1 transgene Rack1 null mutant animals. (J) Transmission electron microscopy shows <br />  presence large glycogen stores (g), numerous double-membrane autophagosomes (arrowheads) autolysosomes (arrows) fat body cells of <br />  control animals starved 4 h 20% sucrose. (K) A portion (J) shown enlarged. (L) Fewer autophagic structures glycogen particles detected <br />  Rack1 mutants. (M) Heat-shock mediated expression Rack1 transgene Rack1 null mutant animals partially rescues reduced autophagy <br />  glycogen accumulation phenotypes mutants. (N&#226;&#8364;&#8220;S) Statistical analyses mutant RNAi phenotypes autophagy glycogen stores. <br />  Graphs represent dots/cell (N&#226;&#8364;&#8220;P), average dot size (Q) normalized cytoplasmic area (R S), **p , 0.01 (two-tailed, two-sample unequal variance <br />  Student&#226;&#8364;&#8482;s t-test). (T) Knockdown Rack1 expression cell-autonomously decreases glycogen stores. (U V) No specific Rack1 labeling observed <br />  Rack1 mutants, glycogen stores strongly decreased compared heterozygous controls. Scale bars: 30 mm fluorescent images <br />  (indicated A, E T&#226;&#8364;&#8220;V), 10 mm (J, L M, indicated J) 2 mm (K). <br />  <br />  <br />  <br />  <br /> 1128                                                                    Autophagy                                                          Volume 8 Issue 7 <br />  www.landesbioscience.com <br /> Autophagy <br /> 1129 <br />                            &#194;&#169;2012 Landes Bioscience. Do distribute. <br />     Immuno-electron microscopy analysis identified            membrane sustain high level autophagosome generation, <br /> endogenous Rack1 localized periphery of, immediately         explaining increased transcription genes products are <br /> next glycogen particles fat body cells, addition     required membrane transport. <br /> present unidentified cytoplasmic clusters (Fig. 3F G). In          In work chose Rack1 (receptor activated protein <br /> indirect immunofluorescence experiments Rack1-positive            kinase C 1) analysis. Rack1 induced whole <br /> dots localized periphery glycogen particles       animals dissected fat bodies starvation, loss of <br /> in laser scanning confocal microscopy (Fig. 3H). 77%        Rack1 impaired starvation-induced basal autophagy, <br /> glycogen particles associated Rack1 dots (n = 426/553),      effect developmentally programmed autophagy <br /> 48% Rack1 dots associated glycogen (n = 394/822).         fat body. These findings indicate Rack1 involved in <br />    The activity glycogen synthase major determinant         multiple types autophagy. <br /> glycogen levels. Phosphorylation glycogen synthase Shaggy,          Autophagosomes generally short half-life 5&#226;&#8364;&#8220;10 min, <br /> the Drosophila homolog glycogen synthase kinase 3B                  likely explanation basal levels autophagy <br />  <br />  <br />  <br />  <br />                                                                                                                                                &#194;&#169;2012 Landes Bioscience. Do distribute. <br /> (GSK-3B) promotes glycogen synthesis.34 A physical interaction         difficult visualize tissues.38 In contrast, <br /> between human homologs Rack1 GSK-3B                     autophagosomes appear high numbers starvation. The <br /> already reported.35 In line that, 73.9%          autophagosomal compartment reduced Rack1 mutants <br /> endogenous Rack1 colocalized endogenous Shaggy,               rescued completely transgenic expression Rack1, <br /> 39.4% Shaggy positive dots positive Rack1             Rack1 transiently associated phagophore <br /> (n = 82/111 n = 82/208 dots, respectively) (Fig. 3I). In           assembly sites autophagosomes, altogether suggesting that <br /> contrast, colocalization Shaggy               Rack1 required efficient autophagosome formation. As we <br /> GFP-Atg8a (not shown).                                                 observed defects cell size/cell growth, DNA polyploidization <br />                                                                        lipid droplet accumulation Rack1 loss function cells, the <br />                           Discussion                                   autophagy defect highly specific phenotype and <br />                                                                        just consequence general problems normal cellular <br /> We used whole-genome microarrays identify genes                functions. <br /> regulated starvation response Drosophila,          Rack1 evolutionarily conserved guanine nucleotide- <br /> compared gene expression changes control, Atg7             binding scaffold protein WD40-repeat: 77% (243/315) <br /> Atg1 mutant larvae. Based gene ontology analysis, genes           amino acid residues identical 87% (275/315) are <br /> involved catabolic processes proteasomal degradation        similar Drosophila Rack1 human GNB2L1. A <br /> and amino acid catabolism significantly upregulated       proteomic study Atg complexes GNB2L1 interacts <br /> starved autophagy mutants, likely compensatory              human homologs Atg1, Atg4, Atg14 Atg18.39 <br /> reaction. On hand, genes required DNA replication        Although GNB2L1 classified high-confidence <br /> were specifically downregulated. Mitotic tissues continue grow      interacting partner, data support hypothesis that <br /> and divide Drosophila larvae deprived nutrients.36      Rack1 act scaffold, transiently binding multiple Atg <br /> Repression genes required DNA replication autophagy          proteins phagophore assembly sites promote maximal <br /> mutants suggests energy-consuming DNA replication processes       activity. In line that, interesting note the <br /> are strongly inhibited completely starved autophagy mutants,        potential interacting partners human Rack1 include member <br /> presumably polyploid cells able supply nutrients    Atg protein complexes. Further biochemical studies <br /> to support diploid cell divisions absence autophagy.         co-immunoprecipitation experiments necessary to <br />    We detected increased transcription Atg genes          verify hypothetical mechanism. <br /> starvation, higher upregulation seen             In addition, half Rack1-positive dots localized to <br /> genes fat bodies dissected starved vs. fed animals. The        glycogen particles, loss Rack1 prevented proper <br /> most highly induced gene Atg8a cases, agreement          formation glycogen stores larval fat body cells. The high <br /> with previous studies showed importance Atg8               degree colocalization GSK-3B strongly suggests that <br /> induction autophagy various models.1,12,15 It             Rack1 associated pool GSK-3B promotes <br /> important note Atg8b expressed adult testis,      glycogen synthesis. The known interacting partners Rack1 <br /> and low level larval expression restricted fat      include subunits AMPK (AMP-activated protein kinase): <br /> body.37 In line this, <span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> Atg8a completely blocks         PRKAA1, PRKAA2 PRKAB2. The &#206;&#178;-subunit AMPK a <br /> starvation-induced autophagy fat body confirming           glycogen-binding domain targets pool kinase bind <br /> Atg8b expression unable sustain autophagy setting,       surface glycogen particle. Activated AMPK turns on <br /> although Atg8b upregulated 118-fold starvation      catabolic processes generate ATP, inhibits glycogen <br /> the fat body.21 Induction Atg8a likely necessary make      synthesis direct phosphorylation glycogen synthase.40 <br /> for degradation half protein products involved   We hypothesize signaling complex containing Rack1, GSK- <br /> autophagosomal cycle. Members Atg9 cycling complex              3B AMPK assembles glycogen particles regulate the <br /> showed strongest average upregulation functional            rate synthesis phosphorylating different amino acid <br /> groups Atg proteins. We speculate sudden induction      residues glycogen synthase, AMPK responsible for <br /> autophagy polyploid larval Drosophila tissues requires lot     targeting complex glycogen. In scenario, activity of <br />  <br />  <br /> 1130                                                            Autophagy                                                   Volume 8 Issue 7 <br />                                                                                                                                                              &#194;&#169;2012 Landes Bioscience. Do distribute. <br />  Figure 3. Localization endogenous Rack1. (A B) Knockdown Rack1 reduces anti-Rack1 immunostaining midgut fat body cells <br />  cell-autonomous manner. (C) Rack1 shows partial colocalization (pre-) autophagosome reporter GFP-Atg8a. (D) In contrast, overlap <br />  Rack1 lysosome reporter Lamp1-GFP. Boxed areas (C D) shown enlarged. (E) Immunogold labeling Rack1 <br />  associated pre-autophagosomal phagophore near growing edges (asterisk) autophagosome (red arrowhead). Please note <br />  characteristic cleft (empty-looking space) membrane sheets phagophores autophagosomes, facilitates their <br />  recognition glutaraldehyde-fixed samples. In contrast, labeling lysosome observed (arrow E). Some gold particles highlighted <br />  yellow arrowheads. (F) Gold particles representing endogenous Rack1 frequently associated peripheral parts glycogen stores (g) in <br />  fat body cells. (G) Rack1 present cytoplasmic clusters (c); autophagosome (red arrowhead) labeled (yellow arrowhead). (H I) <br />  Rack1 closely associated glycogen granules fat body, colocalizes Shaggy, Drosophila homolog GSK-3B. Boxed areas of <br />  (A B) shown enlarged. Scale bars: 10 mm (A&#226;&#8364;&#8220;D, H I), 5 mm (C&#226;&#8364;&#8482;&#226;&#8364;&#8220;D&#226;&#8364;? H&#226;&#8364;&#8482;&#226;&#8364;&#8220;I&#226;&#8364;?) 1 mm (E&#226;&#8364;&#8220;G). All colocalization experiments <br />  performed using confocal laser scanning microscopy. <br />  <br />  <br />  <br /> multiple kinases including AMPK GSK-3B determine                    GSK-3B actually inhibits starvation-induced autophagy (our <br /> the rate glycogen synthesis. These interactions                unpublished observation). Second, exact role AMPK in <br /> relevant Rack1&#226;&#8364;&#8482;s role autophagy number reasons.               starvation-induced autophagy fat body clear: <br /> First, GSK-3B colocalized Atg8a, overexpression                mutation SNF4Ac encoding regulatory subunit AMPK <br />  <br />  <br /> www.landesbioscience.com                                               Autophagy                                                                     1131 <br />  has shown lead impaired autophagic responses               well-fed 72&#226;&#8364;&#8220;96 h AEL (after egg laying) larvae 20% <br /> mutants, recent report suggested AMPK mutants            sucrose solution Eppendorf tube 4 h. For complete <br /> show persistent starvation phenotype interfere           starvations (no calorie), placed well-fed animals filter <br /> further induction autophagy additional experimental                papers soaked PBS. Twenty 30 larvae transferred a <br /> starvation.41,42 In line that, 10 different   new vial amply supplemented live yeast culture (high protein <br /> transgenic RNAi dominant-negative AMPK lines tested               diet) 12&#226;&#8364;&#8220;24 h starvation reduce crowding make sure <br /> showed suppression starvation-induced autophagy clonal         animals well-fed onset starvation <br /> analysis (our unpublished observations), potentially supporting      experiments, necessary proper autophagic response. <br /> hypothesis autophagy phenotype AMPK mutants              For diet high protein sugar, yeast paste prepared <br /> not cell autonomous Drosophila fat body.                       20% sucrose solution. <br />    Rack1 originally described cytoplasmic receptor             Microarray analysis. Our microarray experiments conformed <br /> activated protein kinase C (PKC). The structure Rack1                 MIAME guidelines, data sets available www.ebi. <br />  <br />  <br />  <br />  <br />                                                                                                                                               &#194;&#169;2012 Landes Bioscience. Do distribute. <br /> resembles &#206;&#178;-subunit heterotrimeric G proteins. The        ac.uk/arrayexpress accession number E-MEXP-3352. <br /> individual WD40 repeats simultaneously bind different                Four microarrays used analyzing genotype: two <br /> proteins, making Rack1 candidate platform integrating              independent fed starved samples collected (biological <br /> several signaling pathways. Rack1 shown physically interact       replica), dye swap performed fed/starved <br /> with &#206;&#178;-integrin, various kinases including PKC, AMPK, GSK-3B             sample pair (technical replica&#226;&#8364;&#8221;please below). Eighty-six &#194;&#177; 2 h <br /> and Src, protein phosphatase 2A, focal adhesion components,          AEL L3 stage larvae allowed feed standard fly food <br /> even ribosomes. Indeed, Rack1 plays role wide range            supplemented yeast (fed) starved PBS 4 h (starved). <br /> processes including cell adhesion migration, cell survival       Approximately 30 mg larvae collected frozen <br /> translation.33,43 Recent findings showed Rack1 promotes         immediately liquid nitrogen RNAlater (Sigma, R0901). <br /> is required progression types cancers,         Total RNA purified using NucleoSpin RNA II RNA <br /> increased expression predicts poor clinical outcome breast           isolation kit (Macherey-Nagel, 740955). RNA samples were <br /> cancer patients.44-48                                                    stored &#226;&#8364;&#8220;80&#194;&#176;C presence 30 U Prime RNase inhibitor <br />    Here shown Drosophila Rack1 involved          (Fermentas, EO0382). For probe preparation, 1 mg total RNA <br /> the autophagic response starvation, potentially acting     reverse transcribed 10 ml volume using Oligo(dT) <br /> scaffold protein formation autophagosomes. In              Primer ArrayScript enzyme, second cDNA strand <br /> addition, Rack1 necessary proper generation glycogen       synthesized 50 ml final volume using DNA polymerase and <br /> particles larval fat body, likely recruiting Shaggy/      RNase H. Amino allyl modified aRNA synthesized by <br /> GSK-3B promote glycogen synthesis. Taken together,            vitro transcription using aaUTP T7 Enzyme mix. All these <br /> demonstrated novel roles autophagy glycogen                  steps using AminoAllyl MessageAmpTM II aRNA <br /> synthesis added diverse list functions         Amplification Kit (Ambion, AM1753). Six micrograms amino <br /> for Rack1.                                                               allyl modified amplified RNA labeled Cy5 Cy3 <br />                                                                          dyes 10 ml volume (GE Healthcare, PA23001, PA25001) and <br />                     Materials Methods                                purified using RNA purification columns (Macherey Nagel, <br />                                                                          740955). The specific activity concentration labeled <br /> Drosophila stocks culture. Flies raised 25&#194;&#176;C              aRNA samples checked spectrophotometry (NanoDrop <br /> standard cornmeal/agar media, 60% humidity 12-h light/          3.1.0, Rockland). 4 &#194;&#190; 44 k format Drosophila Gene Expression <br /> 12-h dark daily cycle. The following stocks used: CG5335            Microarrays (Agilent, G2519F-021791) used determine <br /> [d30], Atg7[d77], Atg7[d14],3 (d14 deletes promoters         gene expression changes. 300 ng Cy5 Cy3 labeled aRNA in <br /> first parts Atg7 Sec6, d77 deletes CG5335           19 ml volume, 5 ml 10 &#194;&#190; blocking agent 1 ml 25 &#194;&#190; <br /> Atg7, d30 deletes CG5335 only; therefore, d77/d14 animals            fragmentation buffer mixed incubated at <br /> are mutant Atg7 heterozygous CG5335            60&#194;&#176;C 30 min. Twenty-five microliters 2&#194;&#190; GEx hybridization <br /> Sec6, d30/d14 heterozygous genes), Atg1          fuffer added sample stop fragmentation <br /> [d3D],21 UAS-Lamp1-GFP, UAS-GFP-Atg8a,49 hs-Flp; UAS-Dcr2;               reaction. All steps using gene expression <br /> R4-mCherry-Atg8a, Act . CD2 . Gal4, UAS-GFPnls (gift                  hybridization kit (Agilent, 5188-5242). Forty-eight microliters <br /> Tom Neufeld), Df(2L)rack1-mts, Rack1[1.8],33 (the 1.8 allele             mixes used hybridization microarray <br /> carries premature stop codon instead sixth amino acid;       hybridization chambers (Agilent, G2534A). The chambers were <br /> used hemizygous animals trans deficiency Df(2L)              loaded hybridization rotator rack (~5 rpm) and <br /> rack1-mts deletes Rack1 mts experiments), hs-       incubated 65&#194;&#176;C 17 h. After hybridization slides were <br /> Gal4 (Bloomington Drosophila Stock Center, BDSC),                    washed wash buffer 1 (Agilent, 5188-5327) room <br /> transgenic RNAi lines Rack1 KK104470 (designated                  temperature 1 min wash buffer 2 37&#194;&#176;C another <br /> Rack1 RNAi text figures), HMS01173                    1 min scanning. Each array scanned 543 nm (for <br /> (designated Rack1 RNAi_2 Fig. S1F) (provided                    Cy3 labeling) 633 nm (for Cy5 labeling) Agilent Scanner <br /> the Vienna Drosophila RNAi Center, VDRC BDSC,                        using built-in XDR (Extended Dynamic Range) function with <br /> respectively). Starvations routinely performed floating          5 mm resolution. Output images analyzed Feature <br />  <br />  <br /> 1132                                                              Autophagy                                                Volume 8 Issue 7 <br />  Extraction software Agilent (ver.9.5.1.1.) using two-           Construction UAS-Rack1 transgenic lines genetic <br /> color gene expression protocol (GE2_v5_95_Feb07)           rescue experiments. The pUAST-Rack1-FLAG-HA clone <br /> raw data. All ratios normalized using Lowess                 UFO04269 obtained DGRC (Drosophila Genomics <br /> normalization method. Two-tailed two-sample unequal variance          Resource Center). Following sequencing, standard PhiC31- <br /> Student&#226;&#8364;&#8482;s t-test two-fold change cut applied        mediated transformation procedures used obtain <br /> determine genes showed significantly altered expression         transgenic Drosophila lines carrying UAS-Rack1 65B2 <br /> levels (p , 0.05). Additionally, applied multiple testing          platform chromosome (Bestgene). Expression UAS-Rack1 was <br /> correction (Benjamini-Hochberg method; p value 2 Tables S1&#226;&#8364;&#8220;        induced hs-Gal4 Rack1 null mutant background a <br /> S3) using Genspring estimate false discovery rate (corrected p     1 h heat shock 37&#194;&#176;C, followed 4 h starvation. <br /> value supplemental tables) associated large number        Histology imaging. Clonal analysis polyploid larval <br /> of comparisons.                                                       Drosophila tissues Lysotracker stainings performed as <br />    QT-PCR experiments. RNA samples prepared fat             described previously.21,49 For statistics, Lysotracker, mCherry- <br />  <br />  <br />  <br />  <br />                                                                                                                                            &#194;&#169;2012 Landes Bioscience. Do distribute. <br /> bodies dissected fed starved control larvae described     Atg8a Lamp1-GFP puncta manually counted in <br /> above. QT-PCR reactions performed 4 independently             Photoshop. Two-tailed two-sample unequal variance Student&#226;&#8364;&#8482;s <br /> collected fed 4 independently collected starved fat body          t-test used calculate p values. The following antibodies <br /> samples. The reactions performed RotorGene 3000             used immunostainings: rabbit polyclonal anti-Rack1 <br /> instrument (Corbett Research) gene-specific primers          (1:500, gift Julie Kadrmas),33 mouse monoclonal anti- <br /> SybrGreen protocol monitor gene expression. Two micrograms         glycogen (1:20, gift Otto Baba),52 mouse monoclonal <br /> of total RNA reverse transcribed using High-Capacity          anti-Shaggy (4G1E11, 1:100, gift Marc Bourouis),34 mouse <br /> cDNA Archive Kit (Applied Biosystems, 4322171) according           monoclonal anti-GFP (1:1,000, Invitrogen, A11120), Alexa488- <br /> the manufacturer&#226;&#8364;&#8482;s instructions final volume 30 mL. The       conjugated goat anti-mouse (Invitrogen, A11001, 1:1,000), <br /> temperature profile reverse transcription following:   Alexa568-conjugated goat anti-rabbit (Invitrogen, A21069, <br /> 10 min room temperature, 2 h 37&#194;&#176;C, 5 min ice             1:1,000). A Zeiss Axioimager M2 microscope equipped an <br /> finally 10 min 75&#194;&#176;C enzyme inactivation. After dilution        Apotome2 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Plan-NeoFluar 40&#194;&#190; 0.75 NA objective was <br /> with 30 mL water, 1 mL diluted reaction mix used     used capture images Axiovision software using MinMax <br /> template QT-PCR reactions, using FastStart SYBR Green              setting automatically adjusting image levels. Confocal images <br /> Master mix (Roche, 04673484001) final primer concentra-          acquired Olympus FV500 laser scanning microscope <br /> tion 250 nM following conditions: 15 min 95&#194;&#176;C,        63&#194;&#190; 1.45 NA oil-immersion objective sequential <br /> 45 cycles 95&#194;&#176;C 15 sec, 60&#194;&#176;C 25 sec 72&#194;&#176;C 25 sec.    scanning mode. Primary images processed Adobe <br /> Fluorescence intensity SYBR Green detected          Photoshop produce final figures. <br /> amplification step. Melting temperature analysis          Transmission electron microscopy (TEM). Tissues pro- <br /> each reaction check quality products. Relative          cessed TEM analysis described previously.49 For immuno- <br /> expression ratios calculated normalized ratios GAPDH.      electron microscopy, rabbit polyclonal anti-Rack1 (1:50) used <br /> As additional control, significant change observed       described previously.3 Images captured JEOL <br /> expression Actin5C locus. Nontemplate control sample         JEM-1011 transmission electron microscope using Olympus <br /> was used PCR run check primer-dimer formation.        Morada 11 megapixel camera iTEM software (Olympus). For <br /> The final relative gene expression ratios calculated DDCt     statistical analysis, 5,000 &#194;&#190; magnification images taken <br /> values. Primer sequences available request. Two-tailed       randomly larvae genotype (16 images per <br /> two-sample unequal variance Student&#226;&#8364;&#8482;s t-test used calculate    genotype), area autophagic structures glycogen has <br /> p values.                                                             manually encircled Photoshop obtain area values. <br />    Bioinformatic analysis. GO analysis performed using            Relative cytoplasmic area values two-tailed two-sample <br /> DAVID Bioinformatics Resources 6.7 http://david.niaid.nih.         unequal variance Student&#226;&#8364;&#8482;s t-tests calculated using Excel. <br /> gov.50 We selected unique FBgn identifiers genes showing at <br /> least two-fold expression change response starvation                   Disclosure Potential Conflicts Interest <br /> p , 0.05 genotypes. Separate lists induced      No potential conflicts disclosed. <br /> repressed FBgn identifiers uploaded, GO terms of <br /> biological processes functional annotation tool used                                   Acknowledgments <br /> functional annotation clustering default options. We        We thank Sarolta P&#195;&#161;lfia excellent technical assistance, <br /> compared significantly enriched (p , 0.05) GO categories          Otto Baba, Marc Bourouis Julie Kadrmas providing <br /> among genotypes. As genes represented              antibodies, Tom Neufeld stock centers BDSC VDRC for <br /> by multiple 60-mer oligos whole-genome microarray,         fly strains. This work supported Wellcome Trust <br /> p , 0.05 fold change values collected gene           (087518/Z/08/Z), Hungarian Scientific Research Fund (OTKA <br /> interest, averaged, visualized using JColorGrid      PD72095), Hungarian Academy Sciences (BO/00712/07, <br /> expression analysis individual Atg genes.51 BLASTP searches   BO/00781/11, BO/00552/11) European Union the <br /> used <span id='am-399' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> products yeast, Drosophila        European Social Fund contributed purchase electron <br /> human, top-scoring hit shown gene Figure 1C.   microscope (T&#195;?MOP 4.2.1./B-09/1/KMR-2010-0003). <br />  <br />  <br /> www.landesbioscience.com                                       Autophagy                                                           1133 <br />                                  Supplemental Materials <br /> Supplemental materials here: <br /> www.landesbioscience.com/journals/autophagy/article/20069 <br />  <br />                         References                                14. Gorski SM, Chittaranjan S, Pleasance ED, Freeman           26. Hosokawa N, Sasaki T, Iemura S, Natsume T, Hara T, <br />                                                                       JD, Anderson CL, Varhol RJ, et al. A SAGE approach             Mizushima N. Atg101, novel mammalian autophagy <br /> 1.    Klionsky DJ, Emr SD. Autophagy regulated                   discovery genes involved autophagic cell death.       protein interacting Atg13. Autophagy 2009; <br />       pathway cellular degradation. Science 2000;                  Curr Biol 2003; 13:358-63; PMID:12593804; http://              5:973-9; PMID:19597335; http://dx.doi.org/10.4161/ <br />       290:1717-21; PMID:11099404; http://dx.doi.org/10.               dx.doi.org/10.1016/S0960-9822(03)00082-4                       auto.5.7.9296 <br />       1126/science.290.5497.1717                                  15. Juh&#195;&#161;sz G, Pusk&#195;&#161;s LG, Komonyi O, Erdi B, Mar&#195;&#179;y P,           27. Mercer CA, Kaliappan A, Dennis PB. A novel, human <br /> 2.    Hara T, Nakamura K, Matsui M, Yamamoto A,                       Neufeld TP, et al. Gene expression profiling identifies        Atg13 binding protein, Atg101, interacts ULK1 <br />       Nakahara Y, Suzuki-Migishima R, et al. Suppression           FKBP39 inhibitor autophagy larval                  essential macroautophagy. Autophagy 2009; <br />       basal autophagy neural cells causes neurodegenerative        Drosophila fat body. Cell Death Differ 2007;                   5:649-62; PMID:19287211; http://dx.doi.org/10. <br />       disease mice. Nature 2006; 441:885-9; PMID:                  14:1181-90; PMID:17363962; http://dx.doi.org/10.               4161/auto.5.5.8249 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                &#194;&#169;2012 Landes Bioscience. Do distribute. <br />       16625204; http://dx.doi.org/10.1038/nature04724                 1038/sj.cdd.4402123                                        28. Vergne I, Roberts E, Elmaoued RA, Tosch V, Delgado <br /> 3.    Juh&#195;&#161;sz G, Erdi B, Sass M, Neufeld TP. Atg7-dependent        16. Weidberg H, Shpilka T, Shvets E, Abada A, Shimron F,           MA, Proikas-Cezanne T, et al. Control autophagy <br />       autophagy promotes neuronal health, stress tolerance,           Elazar Z. LC3 GATE-16 N termini mediate                    initiation phosphoinositide 3-phosphatase Jumpy. <br />       longevity dispensable metamorphosis           membrane fusion processes required autophagosome           EMBO J 2009; 28:2244-58; PMID:19590496; http:// <br />       Drosophila. Genes Dev 2007; 21:3061-6; PMID:                    biogenesis. Dev Cell 2011; 20:444-54; PMID:                    dx.doi.org/10.1038/emboj.2009.159 <br />       18056421; http://dx.doi.org/10.1101/gad.1600707                 21497758; http://dx.doi.org/10.1016/j.devcel.2011.         29. Mari M, Griffith J, Rieter E, Krishnappa L, Klionsky <br /> 4.    Komatsu M, Waguri S, Chiba T, Murata S, Iwata J,                02.006                                                         DJ, Reggiori F. An Atg9-containing compartment that <br />       Tanida I, et al. Loss autophagy central nervous   17. Rusten TE, Lindmo K, Juh&#195;&#161;sz G, Sass M, Seglen PO,              functions early steps autophagosome bio- <br />       causes neurodegeneration mice. Nature 2006;           Brech A, et al. Programmed autophagy                    genesis. J Cell Biol 2010; 190:1005-22; PMID: <br />       441:880-4; PMID:16625205; http://dx.doi.org/10.                 Drosophila fat body induced ecdysone             20855505; http://dx.doi.org/10.1083/jcb.200912089 <br />       1038/nature04723                                                regulation PI3K pathway. Dev Cell 2004; 7:179-      30. Pankiv S, Clausen TH, Lamark T, Brech A, Bruun JA, <br /> 5.    Simonsen A, Cumming RC, Brech A, Isakson P,                     92; PMID:15296715; http://dx.doi.org/10.1016/j.devcel.         Outzen H, et al. p62/SQSTM1 binds directly Atg8/ <br />       Schubert DR, Finley KD. Promoting basal levels               2004.07.005                                                    LC3 facilitate degradation ubiquitinated protein <br />       autophagy nervous enhances longevity          18. Scott RC, Schuldiner O, Neufeld TP. Role                   aggregates autophagy. J Biol Chem 2007; <br />       oxidant resistance adult Drosophila. Autophagy           regulation starvation-induced autophagy              282:24131-45; PMID:17580304; http://dx.doi.org/ <br />       2008; 4:176-84; PMID:18059160                                   Drosophila fat body. Dev Cell 2004; 7:167-78;                  10.1074/jbc.M702824200 <br /> 6.    Klionsky DJ, Cregg JM, Dunn WA, Jr., Emr SD, Sakai              PMID:15296714; http://dx.doi.org/10.1016/j.devcel.         31. Simonsen A, Birkeland HC, Gillooly DJ, Mizushima <br />       Y, Sandoval IV, et al. A unified nomenclature yeast         2004.07.009                                                    N, Kuma A, Yoshimori T, et al. Alfy, novel FYVE- <br />       autophagy-related genes. Dev Cell 2003; 5:539-45;           19. Zinke I, Sch&#195;&#188;tz CS, Katzenberger JD, Bauer M,                  domain-containing protein associated protein <br />       PMID:14536056; http://dx.doi.org/10.1016/S1534-                 Pankratz MJ. <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> control gene expression            granules autophagic membranes. J Cell Sci 2004; <br />       5807(03)00296-X                                                 Drosophila: microarray analysis starvation sugar-       117:4239-51; PMID:15292400; http://dx.doi.org/10. <br /> 7.    Hosokawa N, Hara T, Kaizuka T, Kishi C, Takamura                dependent response. EMBO J 2002; 21:6162-73;                   1242/jcs.01287 <br />       A, Miura Y, et al. Nutrient-dependent mTORC1                    PMID:12426388; http://dx.doi.org/10.1093/emboj/            32. Pulipparacharuvil S, Akbar MA, Ray S, Sevrioukov EA, <br />       association ULK1-Atg13-FIP200 complex                  cdf600                                                         Haberman AS, Rohrer J, et al. Drosophila Vps16A is <br />       required autophagy. Mol Biol Cell 2009;                 20. Beadle G, Tatum EL, Clancy CW. Food level                   required trafficking lysosomes biogenesis of <br />       20:1981-91; PMID:19211835; http://dx.doi.org/10.                relation rate development eye pigmentation        pigment granules. J Cell Sci 2005; 118:3663-73; <br />       1091/mbc.E08-12-1248                                            Drosophila melanogaster. Biol Bull 1938; 75:447-62;            PMID:16046475; http://dx.doi.org/10.1242/jcs.02502 <br /> 8.    Nakatogawa H, Ishii J, Asai E, Ohsumi Y. Atg4 recycles          http://dx.doi.org/10.2307/1537573                          33. Kadrmas JL, Smith MA, Pronovost SM, Beckerle MC. <br />       inappropriately lipidated Atg8 promote autophago-        21. Scott RC, Juh&#195;&#161;sz G, Neufeld TP. Direct induction            Characterization RACK1 function Drosophila <br />       biogenesis. Autophagy 2012; 8:177-86; PMID:                autophagy Atg1 inhibits cell growth induces             development. Dev Dyn 2007; 236:2207-15; PMID: <br />       22240591; http://dx.doi.org/10.4161/auto.8.2.18373              apoptotic cell death. Curr Biol 2007; 17:1-11; PMID:           17584887; http://dx.doi.org/10.1002/dvdy.21217 <br /> 9.    Tanida I, Sou YS, Ezaki J, Minematsu-Ikeguchi N,                17208179; http://dx.doi.org/10.1016/j.cub.2006.10.053      34. Papadopoulou D, Bianchi MW, Bourouis M. <br />       Ueno T, Kominami E. HsAtg4B/HsApg4B/autopha-                22. Tweedie S, Ashburner M, Falls K, Leyland P,                    Functional studies shaggy/glycogen synthase kinase <br />       gin-1 cleaves carboxyl termini human Atg8          McQuilton P, Marygold S, et al. FlyBase                        3 phosphorylation sites Drosophila melanogaster. <br />       homologues delipidates microtubule-associated               Consortium. FlyBase: enhancing Drosophila Gene                 Mol Cell Biol 2004; 24:4909-19; PMID:15143183; <br />       protein light chain 3- GABAA recep <br /> </body></html>