<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Comprehensive Genetic Dissection Hemocyte <br /> Immune Response Malaria Mosquito Anopheles <br /> gambiae <br /> Fabrizio Lombardo&#194;&#164;a*, Yasmeen Ghani&#194;&#164;b, Fotis C. Kafatos, George K. Christophides* <br /> Division Cell Molecular Biology, Department Life Sciences, Imperial College London, London, United Kingdom <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Reverse genetics mosquito Anopheles gambiae RNAi mediated gene silencing led recent years an <br />      advanced understanding mosquito immune response infections bacteria malaria parasites. We <br />      developed RNAi screens An. gambiae hemocyte-like cells using library double-stranded RNAs targeting 109 genes <br />      expressed highly specifically mosquito hemocytes identify novel regulators hemocyte immune response. <br />      Assays included phagocytosis bacterial bioparticles, expression antimicrobial peptide CEC1, basal induced <br />      expression mosquito complement factor LRIM1. A cell viability screen carried assess dsRNA <br />      cytotoxicity identify genes involved cell growth <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>survival</span>. Our results identify 22 novel immune regulators, <br />      including proteins putatively involved phagosome assembly maturation (Ca2+ channel, v-ATPase cyclin- <br />      dependent protein kinase), <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> recognition (fibrinogen-domain lectins Nimrod), immune modulation (peptidase <br />      serine protease homolog), immune signaling (Eiger LPS-induced factor), cell adhesion communication (Laminin <br />      B1 Ninjurin) immune homeostasis (Lipophorin receptor). The development robust functional cell-based assays <br />      paves way genome-wide functional screens study mosquito immune response infections human <br />      pathogens. <br />  <br />   Citation: Lombardo F, Ghani Y, Kafatos FC, Christophides GK (2013) Comprehensive Genetic Dissection Hemocyte Immune Response Malaria <br />   Mosquito Anopheles gambiae. PLoS Pathog 9(1): e1003145. doi:10.1371/journal.ppat.1003145 <br />   Editor: Kenneth D. Vernick, University Minnesota, United States America <br />   Received March 30, 2012; Accepted December 5, 2012; Published January 31, 2013 <br />   Copyright: &#195;&#376; 2013 Lombardo et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Wellcome Trust grants (GR077229 WT093587MA) NIH/NIAID grant (2P01AI044220-06A1). The funders no <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: g.christophides@imperial.ac.uk (GKC); fabrizio.lombardo@uniroma1.it (FL) <br />   &#194;&#164;a Current address: Division Parasitology, Department Public Health Infectious Diseases, Sapienza University Rome, Rome, Italy. <br />   &#194;&#164;b Current address: Centre Nephrology, University College London, Royal Free Hospital, London, United Kingdom. <br />  <br />  <br />  <br />  <br /> Introduction                                                                           hemolymph [8,9,10]. Two hemocyte expression datasets have <br />                                                                                        reported recently, providing comprehensive list of <br />    Anopheles gambiae major vector Plasmodium falciparum                        hemocyte-expressed genes [8,11]. <br /> malaria sub-Saharan Africa secondary vector                             An. gambiae cell lines used extensively study <br /> parasitic viral diseases [1]. Differences vector susceptibility                 mosquito immune responses [12,13,14,15]. Indeed, cells are <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> malaria parasites partly attributed ability                       capable accomplishing complex immune tasks include <br /> mosquito immune fight infections. The developmental                          phagocytosis bacteria beads [16], expression of <br /> migration Plasmodium mosquito hemolymph,                             immune factors microbial challenge. It shown that <br /> main carrier immune system, presents opportunities                      IMD pathway activation cell lines leads robust expression of <br /> vector humoral cellular immune reactions attack                             antimicrobial peptide (AMP) gene Cecropin 1 (CEC1) [15] and <br /> parasites [2]. Key functions mosquito hemolymph components                          immune factors [13]. This pathway activated the <br /> include killing Plasmodium ookinetes soon emerge                    transmembrane receptor PGRPLC binds peptidoglycan (PGN) <br /> the midgut epithelium [3] sporozoites invade                       induces nuclear translocation REL2 transcription factor <br /> salivary glands [4]. Numerous mosquito agonist antagonist                          [15,17,18]. <br /> effectors Plasmodium bacteria identified,                                The development high-throughput RNAi screens cultured <br /> principally RNAi-mediated reverse genetic tests using dsRNA                         cells major breakthrough functional genomics of <br /> (double-stranded RNA) injections adult mosquitoes [5]. These                      model organisms, basic applied research <br /> factors operate complex molecular networks involve                             [19,20,21,22]. Here report development implementa- <br /> pathogen recognition secreted membrane bound receptors,                          tion RNAi screens An. gambiae cells provide insights the <br /> activation immune signaling pathways, synthesis                              functional immune repertoire mosquito circulating hemocytes. <br /> activation effectors contribute lysis, melanization                      We generated dsRNA library targeting 109 genes <br /> phagocytosis invading pathogens [2,6,7]. Importantly,                      specifically predominantly expressed circulating hemocytes <br /> of factors produced hemocytes function                         optimized cell-based RNAi screens investigate role <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                            1                           January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                      Novel Regulators Mosquito Innate Immunity <br />  <br />  <br />  <br />   Author Summary                                                             (AGAP008001; 2 different dsRNA fragments) Rho1 small <br />                                                                              GTPase (AGAP005160) (Table S2 Text S1). <br />   The mosquito immune relies innate humoral                           Two protocols implemented assess cell growth and <br />   cellular reactions fight infections, including                viability: (a) image acquisition quantification automated <br />   malaria parasites pass mosquitoes                     fluorescence microscopy, time-consuming technically <br />   infect humans. Therefore, detailed                       challenging allows accurate informative analysis; <br />   molecular understanding reactions assist                    (b) microplate reader fluorescence quantification, is <br />   design new ways control spread malaria                    quicker accommodate large datasets user- <br />   mosquito-borne diseases. Here use                           responsive. <br />   technique silence mosquito cultured cells genes that <br />                                                                                 A subset 37 dsRNAs screened automated fluorescence <br />   highly and/or specifically expressed mosquito <br />   hemocytes, equivalent human white blood cells,                      microscopy. Staining cells Sytox green/Hoechst used to <br />   means investigate function reactions               assess effect dsRNAs cell viability. Protocols Volocity <br />   mosquito immune system. Our study identifies                       Improvision ImageJ softwares developed capture <br />   novel regulators immune reactions including phagocy-                    images quantify fluorescent cells, determine ratio of <br />   tosis, engulfment subsequent destruction                        dead cells (Sytox green positive) entire cell population <br />   bacteria pathogens hemocytes, produc-                     (Hoechst positive) (Figure 1A Materials Methods). <br />   tion antimicrobial peptides, directly kill inhibit             ANOVA statistics followed Bonferroni&#226;&#8364;&#8482;s post-test correction <br />   proliferation microbes, basal induced                   revealed dsIAP1 leads significant increase cell <br />   production important complement regulator. Com-                      mortality. Silencing Rho1 significantly reduced number cells <br />   plement robust reaction mosquitoes malaria                 did increase cell mortality (Figure S1, Figure S2 Text <br />   parasites bacteria phagocytosis, lysis                      S1). The cells nuclei larger size indicating <br />   melanization (the enclosure pathogens melanin                      defect cytokinesis cell cycle progression. A similar <br />   capsule). We reveal intriguing molecular connections                  <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>phenotype</span> observed Drosophila S2 cells silencing the <br />   reactions phagocytosis and <br />                                                                              orthologous Rho1 gene [22,27]. <br />   regulation complement. Our study provides novel <br />   insights mosquito immune reactions                        Next, screened entire collection 111 dsRNAs using <br />   infections.                                                        plate reader quantification conjunction CellTiter-Blue <br />                                                                              Cell Viability assay. Data replicates analyzed and <br />                                                                              z-score analysis performed posing thresholds +/22 at <br /> of genes phagocytosis bacteria transcriptional               replicates three. Reproducibility replicates <br /> activation immune-related genes. Our results identify novel               evaluated correlation tests shown plots Figure <br /> regulators hemocyte immune responses interactions                 S3. Silencing ubiquitin-like/ribosomal fusion protein gene <br /> with pathogens, including regulators complement-like <br />                                                                              AGAP008001 led significant decrease cell viability (Figure 1B <br /> pathway component plays key role reactions malaria <br />                                                                              Figure S3). No dsRNA treatment resulted statistically <br /> parasites. This key milestone development genome- <br /> wide RNAi screens An. gambiae cells. <br />  <br /> Results/Discussion <br /> A hemocyte-specific dsRNA library <br />    We generated set 111 dsRNAs target 109 genes that <br /> exhibit enriched expression hemocytes, differential regulation <br /> by immune challenges, presence immune-related InterPro <br /> domains and/or signal peptide transmembrane domains <br /> (Dataset S1). To populate annotate dsRNA library we <br /> used: published datasets genome-wide transcriptional <br /> repertoires An. gambiae circulating hemocytes naive or <br /> Plasmodium infected adult females [8,11], published expression <br /> profile An. gambiae hemocyte-like cell lines response to <br /> microbial challenges [13], VectorBase An. gambiae genome <br /> annotation [23] information silencing phenotypes of <br /> Drosophila orthologs GenomeRNAi database [24,25]. <br /> Analysis AMP expression dsRNA-mediated RNAi efficien- <br /> cy (Text S1) led choose Sua5.1* cell line [12,16,26]        Figure 1. Cell growth viability screen. (A) A fraction the <br /> model experimental system.                                                   dsRNA hemocyte-specific library screened twice automated <br />                                                                              fluorescence microscopy. Fluorescent cells automatically counted <br />                                                                              using protocols developed Volocity Improvision ImageJ <br /> Cell growth viability                                                    software, described Materials Methods. The ratios dead <br />    We carried viability screens assess levels dsRNA            cells (Sytox Green positive) total number cells (Hoechst <br /> toxicity determined effect gene silencing                    positive), standard deviation replicates, shown. (B) The <br /> fundamental housekeeping processes cell growth, prolif-              entire dsRNA collection screened using Cell Titer Blue/Plate <br /> eration survival, hamper true identification              Reader method. The plot shows z-score analysis one <br />                                                                              representative set experiments. A z-score threshold 2 <br /> immune regulators. An. gambiae homologs genes previously            chosen, positive hits shown open circles. No dsRNA- <br /> shown cause lethal growth-defective RNAi phenotypes                 treated samples samples treated 100 mM H2O2 also <br /> Drosophila cells [22,27] used controls: inhibitor apoptosis       indicated. Three biological replicates performed. <br /> 1 (IAP1; AGAP007294), ubiquitin-like/ribosomal fusion protein              doi:10.1371/journal.ppat.1003145.g001 <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                   2                         January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                         Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> significant deviation average cell count. A general                    (Figure 2B). As mentioned earlier, imaging analysis provide <br /> observation dsRNA treatment resulted significant cell               additional, detailed, information compared the <br /> mortality independent targeted gene.                            microplate reader method, technically challenging <br />    We validated viability data vivo injecting dsRNAs               time consuming. The discrepancies two <br /> control LacZ, IAP1 AGAP008001 newly emerged adult                      approaches technical biological reasons, <br /> female mosquitoes monitoring survival KD                        example image analysis quantify of <br /> mosquitoes daily (Figure S1). Both gene KDs led statistically                bioparticles single cell, microplate reader quantifies <br /> significant increase mortality rates compared control. In                 intensity fluorescence. Moreover, shown Figure 2D, <br /> addition support analysis provided                      distribution fluorescent bioparticles uniform cell <br /> experimental approach, good correspondence ex                   layer, introduces variable microscope <br /> vivo cell-based viability screen vivo phenotypic analysis            images captured analyzed. <br /> highlights opportunities future use viability screens               Next, investigated silencing effect 8 13 <br /> identifying targets novel mosquito insecticides.                             dsRNAs identified microplate reader method bacterial <br />                                                                                 phagocytosis vivo. For this, employed protocol used <br /> Bacterial phagocytosis screens                                                  earlier study, bacteria injected mosquito <br />    Phagocytosis highly effective immediate response                    hemolymph spread rapidly cavity phagocytosed <br /> against microbial invaders [28]. Mosquito hemocytes bind                hemocytes clusters associated tracheal <br /> phagocytose bacterial bioparticles Sephadex beads,               [29]. We injected 2-day old mosquitoes dsRNAs 4 <br /> malaria sporozoites [4,29,30,31]. We established fast reliable            days later re-injected E. coli pHRodo-conjugated <br /> cell-based assay An. gambiae cells using Escherichia coli bioparticles       bioparticles. Mosquitoes dissected 1 h bioparticle <br /> conjugated pHrodo succinimidyl ester, pH-sensitive                       injection, mounted glass slides immediately observed <br /> fluorescent dye, investigate potential role genes              fluorescence microscopy. Pictures different parts the <br /> hemocyte-enriched library bacterial phagocytosis. The phago-                 mosquito abdomens captured analyzed using protocol <br /> cytic activity cells determined increase bioparticle           developed ImageJ quantify numbers fluorescent <br /> fluorescence caused drop pH acidified phagosomes               particles (Figure 3). The results revealed strong vivo phenotypes <br /> [32].                                                                           similar cell-based analysis 4 8 <br />    Bacterial bioparticles added cells days              dsRNAs: AGAP000182, AGAP008492, AGAP004928 and <br /> incubation dsRNA, capacity cells uptake                      AGAP002243, corroborated significant statistical <br /> bioparticles assessed fluorescence measurements using                  evaluation. Three dsRNAs (AGAP003879, FBN8 and <br /> microplate reader. Four time-points assayed                   AGAP006769) showed similar albeit weaker phenotypes <br /> account kinetics bioparticle uptake (1, 3, 6 24 h post-              compared cell-based analysis dsRNA <br /> challenge). The measurements obtained subtracted                      (AGAP009459) did confirm expected phenotype. <br /> basal level measurements (0 h, immediately challenge). The <br /> entire dsRNA library screened times z-score values <br /> for dsRNA replicate screens were <br />                                                                                 Novel regulators phagocytosis <br /> calculated. We considered positive dsRNA hits z-score                   Based microplate reader analysis, silencing 7 <br /> values 2 22                     genes led significant decrease cellular capacity to <br /> replicates time point (Figure S4 Supplemental Table                phagocytose E. coli bioparticles. Some genes also <br /> S3). Because library strongly biased genes              confirmed microscopy vivo analysis, as <br /> likely play role immune reactions, z-score method                   presented above. These genes encode: protein unknown <br /> which compares effect dsRNA average effect                 function putative signal peptide peptidase domain <br /> of dsRNAs strict condition. Therefore,                    (AGAP000182); protein homodimerization BTB/POZ <br /> analysed data using ANOVA followed Tukey&#226;&#8364;&#8482;s multiple                      domain, ankyrin repeats zinc finger domain (AGAP002243); <br /> comparison test, comparing dsRNA reference                   putative transmembrane v-ATPase (AGAP003879); membrane- <br /> dsLacZ control. The results methods revealed total               bound protein zinc finger LITAF (LPS-induced <br /> 13 positive dsRNA hits, 6 z-score 11 ANOVA                    tumor necrosis factor alpha factor) domain putatively involved in <br /> (Table S3 Figure 2A&#226;&#8364;&#8220;B). Four dsRNAs showed significant                      immune signaling (AGAP004928) [33,34]; fibrinogen-domain <br /> effects bioparticle uptake methods: 2                      FBN8 (also known FREP57), previously shown play role in <br /> decreased phagocytosis (FBN8 AGAP000095) 2                      anti-Plasmodium defense (AGAP011223) [35]; putative Calcium <br /> increased phagocytosis (AGAP006769 FBN9). Cactus dsRNA                      channel protein (AGAP000095); three-transmembrane <br /> that used positive control led significant increase          protein unpredicted function (AGAP008500). <br /> in phagocytosis E. coli soon 1 h challenge,                      Two proteins likely play role phagosome <br /> consistence previous observations [29].                                    formation maturation/acidification. The putative Ca2+ <br />    We examined 13 positive hits microplate reader                  channel protein (AGAP000095) involved cellular <br /> analysis using automated fluorescence microscopy protocol                 Ca2+ balance required solubilization actin <br /> for quantification phagocytosed bioparticles developed                meshwork surrounding nascent phagosomes, fusion phagosomes <br /> ImageJ software. Bioparticle uptake monitored 2 h 20 h                  granules containing lytic enzymes, assembly activation <br /> after challenge compared dsLacZ control using                        superoxide-generating NADPH oxidase complex [36]. The <br /> ANOVA (Figure 2C&#226;&#8364;&#8220;D). An overall consistency observed                        v-ATPase (AGAP003879) known play role phagosome <br /> between microplate reader microscopy analyses. Of                   acidification model organisms [37]. Orthologs of <br /> the 13 dsRNAs, 6 showed expected phenotype statistical                 AGAP003879 AGAP004928 D. melanogaster similar <br /> significance, 5 showed expected phenotype                      phenotypes RNAi screens investigate host-pathogen <br /> statistically significant, 2 showed difference dsLacZ           interactions (Dataset S1), KDs cause decrease in <br /> control and/or phenotype opposite expected, respectively               intracellular Listeria monocytogenes infection [38]. <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                      3                         January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                         Novel Regulators Mosquito Innate Immunity <br />  <br />  <br />  <br />  <br /> Figure 2. Bacterial phagocytosis screen. (A) Venn diagram showing results z-score ANOVA analyses data obtained the <br /> microplate reader phagocytosis assay. (B) Heat map depicting performance microscopy imaging vivo phagocytosis assays 13 <br /> dsRNAs identified positive hits microplate reader assay. Dark green, significant decrease phagocytosis; green, decrease phagocytosis; <br /> dark red, significant increase phagocytosis; red, increase phagocytosis; grey, similar LacZ control; nd, determined. (C) Microscopy imaging <br /> analysis: phagocytosis rates cells 2 20 h following bioparticles challenge genes identified modulators phagocytosis by <br /> microplate reader assay silenced. Data shown percent phagocytosis compared dsLacZ-treated controls. (D) Examples fluorescence <br /> microscopy images Sua5.1* cells treated dsRNAs Cytochalasin D. Images captured 20 h challenge pHRodo bioparticles. The <br /> graphs indicate capacity cells uptake bioparticles following dsLacZ, dsCactus Cytochalasin D treatments 2 20 h challenge, as <br /> quantified image analysis. Mean values counted particles standard errors reported. Results experiments shown. Asterisks <br /> indicate statistically significant differences KD dsLacZ-treated controls (*: P,0,05; **: 0,005,P,0,05; ***: 0,0005,P,0,005). <br /> doi:10.1371/journal.ppat.1003145.g002 <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                     4                         January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                         Novel Regulators Mosquito Innate Immunity <br />  <br />  <br />                                                                                 considering fine interplay different immune processes, <br />                                                                                 pattern recognition receptor specifically promote one <br />                                                                                 favor process. <br />  <br />                                                                                 Transcriptional activation AMP CEC1 <br />                                                                                    It previously shown AMP CEC1 is <br />                                                                                 transcriptionally induced cultured cells following immune <br />                                                                                 challenge induction depends partly IMD <br />                                                                                 signaling pathway [13], triggered PGN recognition the <br />                                                                                 PGN Recognition Protein LC (PGRP-LC) [18]. To identify novel <br />                                                                                 hemocyte regulators A. gambiae IMD pathway and <br />                                                                                 potentially pathways regulating expression AMPs, <br />                                                                                 developed luciferase reporter assay Sua5.1* cells screen <br />                                                                                 hemocyte dsRNA library. A 660 bp fragment CEC1 <br />                                                                                 promoter cloned upstream luciferase gene used [15]. <br />                                                                                 Initial experiments revealed significant induction CEC1 <br />                                                                                 promoter activity 7 h PGN challenge. A dsRNA targeting the <br />                                                                                 NF-kB transcription factor REL2, previously shown regulate <br /> Figure 3. In vivo phagocytosis assay. (A) Pictures abdominal <br /> segments adult female mosquitos injected dsLacZ, dsA- <br />                                                                                 CEC1 transcriptional activation [15], included positive <br /> GAP004928 dsAGAP008492 challenged pHRodo E.                   control. Changes luciferace activity analyzed calculat- <br /> coli bioparticles. Fluorescence brightfield images captured            ing z-score values ratios average RLU (Relative Light <br /> and hemocytes containing fluorescent bioparticles quantified. The               Units) measurements PGN vs. PBS treatments (Figure 4 and <br /> experiment repeated twice 8&#226;&#8364;&#8220;10 mosquitoes treatment.               Table S4). One-way ANOVA followed Dunnet&#226;&#8364;&#8482;s multiple- <br /> Quantification fluorescent bioparticles statistical analysis were <br />                                                                                 comparison post-test performed compare dsLacZ <br /> performed described Materials Methods. (B) The graph shows <br /> the percentage phagocytosis different KDs placing dsLacZ           control KD values (Table S4). <br /> reference. Mean values standard errors reported. Asterisks                 Silencing AGAP010531 (FBN12 FREP2) AGAP006771 led <br /> indicate statistical significance (**: 0,005,P,0,05) according t-tests       increased CEC1 expression following PGN challenge. <br /> applied gene KD compared LacZKD.                                     AGAP006771 encodes putative transmembrane protein ortholo- <br /> doi:10.1371/journal.ppat.1003145.g003 <br />                                                                                 gous Drosophila Tumor Necrosis Factor-like, eiger. Interest- <br />  <br />    In contrast, silencing 6 109 genes leads significant <br /> increase E. coli phagocytosis. Proteins encoded genes <br /> represent potential novel negative regulators bacterial recogni- <br /> tion phagosome assembly. These include: secreted protein of <br /> unknown function strongly expressed mosquito <br /> hemocytes [8] cultured cells following LPS PGN challenge <br /> (AGAP006769) [13]; putative tyrosine serine/threonine <br /> kinase (AGAP009459) homologs described mosqui- <br /> toes (Aedes aegypti, AAEL008621, cell division protein kinase 1, cdk1; <br /> and Culex quinquefasciatus, CPIJ001155, cdk2) D. melanogaster <br /> (CG5363, cdc2 cell division control protein); AGAP008492 that <br /> does exhibit similarity genes regulated <br /> during immune challenges [13]; ortholog Drosophila laminin <br /> B1 chain (AGAP001381); fibrinogen-domain lectins <br /> FBN30/FREP8 FBN9/FREP13 (AGAP006914 and <br /> AGAP011197, respectively) [35]. <br />    The negative effect AGAP009459 silencing bacterial <br /> phagocytosis possibly related defects cytoskeleton <br /> regulation. Its Drosophila ortholog, cdc2, similarly involved in <br /> defense-related processes highlighted increased Listeria <br /> intracellular infection, reduced Chlamydia infection decreased <br /> Drosophila C virus influenza virus replication following <br /> silencing [38,39,40]. Similarly, fruit fly ortholog Laminin <br /> B1 play role innate immune reactions its <br /> silencing shown decrease viability intestinal infections <br /> with Serratia marcescens [41]. FBN9 previously shown be <br />                                                                                 Figure 4. Luciferase assays. (A) Transcriptional activation CEC1 <br /> upregulated malaria parasite E. coli infections [42].               promoter PGN challenge. The graph reports z-scores <br /> The involvement FBN9 defense bacteria                     calculated ratios averaged RLU measurements PGN <br /> maintenance basal immune homeostasis supported                         PBS treatments. (B C) Transcriptional regulation LRIM1 <br /> evidence protein surface non-challenged             promoter (B) PGN challenge (z-scores calculated ratios of <br /> cells strongly co-localizes bacteria malaria                averaged RLU measurements PGN PBS treatments) (C) in <br />                                                                                 basal conditions (z-scores calculated RLU measured PBS <br /> parasites following infection [35]. A specific role FBN9                treatments). Positive hits shown open circles. <br /> negative regulator phagocytosis hypothesized                doi:10.1371/journal.ppat.1003145.g004 <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                      5                         January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                   Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> ingly, Drosophila eiger induced microbial infections           We investigated lack LRIM1 promoter upregula- <br /> and required resistance tolerance infections, partly       tion Sua5.1* cells PGN challenge inhibition by <br /> by controlling expression AMP Diptericin [33,43]. Eiger         hemocyte factors (Figure 4 Table S4). Our screen <br /> functions negative regulator AMP expression following              identified genes inhibit transcriptional activation the <br /> PGN challenge IMD pathway activation blocking                   LRIM1 promoter following PGN challenge, silencing <br /> expression NF-kB factor, Relish (the ortholog REL2),             AGAP009762, AGAP007499, AGAP003473 AGAP001964 led <br /> through JNK pathway [44]. Our data consistent              increased luciferase expression compared PBS control. <br /> fruit fly model identify mosquito eiger negative regulator        AGAP009762 highly expressed circulating hemocytes [8] and <br /> of CEC1 expression. Like FBN9 identified negative             encodes EGF-like domain protein similarities Drosophila <br /> regulator phagocytosis, FBN12 belongs family putative           phagocytosis receptors eater [56] NimC1 [57] to <br /> pattern recognition receptors known involved immune               members Nimrod superfamily [58]. Interestingly, a <br /> responses maintenance homeostasis; however, contrast          screen novel regulators JNK following IMD pathway <br /> FBN9, FBN12 downregulated bacterial infections                   activation challenge PGN Drosophila cells, revealed <br /> [17,35]. This consistent role FBN12 negative             ortholog AGAP009762 caused increased P-JNK <br /> regulator CEC1 expression.                                              protein expression, which, turn, act modulate the <br />    Silencing AGAP009231 AGAP009459 reduced PGN-in-                     expression Relish-controlled effectors [59]. <br /> duced CEC1 transcriptional activation. AGAP009231 encodes                   AGAP007499 encodes chloride channel protein, orthologous <br /> transmembrane domain protein family ninjurins,              human chloride channel 7. It strongly expressed in <br /> likely A sub-family, complex class cell adhesion               circulating hemocytes [8] upregulated mosquito cultured <br /> molecules proteolytically processed shed matrix            cells hydrogen peroxide treatment [13]. The Drosophila <br /> metalloproteases (MMP). MMP1 NinjA genes co-expressed              ortholog AGAP007499 shown play role receptor <br /> and upregulated D. melanogaster S2 cells LPS challenge [45]       tyrosine kinase (RTK)-Ras-extracellular signal-regulated kinase <br /> and adult flies wounding [46]. Proteolytic cleavage            (MAPK/ERK) signaling pathway; silencing leads increased <br /> NinjA MMPs releases ectodomain involved cell adhesion             MAPK phosphorylation following EGF stimulation [60]. It has <br /> and cell-cell communication [47]. MMPs known play                   previously shown MAPK ERK signaling plays role in <br /> various roles inflammation innate immunity identity         mosquito immune response malaria parasites [61]. <br /> and function substrates mechanisms            AGAP003473 encodes transmembrane protein signifi- <br /> elucidated [48]. AGAP009231 previously shown                similarity known proteins. Finally, AGAP001964 encodes a <br /> highly expressed localized membrane An. gambiae              previously uncharacterized member clip-domain serine <br /> circulating hemocytes [8]. Our data showing involvement            protease subfamily A (CLIPA) lacks protease activity. <br /> protein AMP expression suggest important role mosquito            Importantly, CLIPAs Plasmodium infection pheno- <br /> innate immunity, probably signaling cell communication.             types [62,63], regulating hemocyte-mediated <br />    As discussed earlier, AGAP009459 encodes ortholog                melanization reaction. Since LRIM1 involved malaria parasite <br /> Drosophila cyclin-dependent protein kinase cdc2              melanization lysis, bacterial phagocytosis, we <br /> implicated processes cell cycle regulation              hypothesize identified proteins function negative <br /> cytoskeleton remodeling. Importantly, RNAi silencing cdc2          regulators reactions (e.g. melanization) are <br /> leads decreased STAT92E phosphorylation, suggesting                   potentially costly host; LRIM1 induced when <br /> regulatory role cdc2 JAK/STAT signaling [49]. The                    proteins downregulated presumably depleted during <br /> involvement JAK/STAT pathway fruit fly hemocyte                      reactions. <br /> differentiation proliferation documented, exact           Silencing AGAP001381 opposite effect, reducing <br /> role immune responses AMP expression remains                    luciferase expression driven LRIM1 promoter 7 h after <br /> unclear [50]. In mosquitoes, JAK/STAT pathway activated             challenge PGN compared mock PBS challenge. As <br /> by immune challenges involved responses                  mentioned previously, AGAP001381 encodes ortholog the <br /> pathogens [51,52,53], data suggest involvement              fruit fly Laminin B1 silencing increases phagocytosis of <br /> JAK/STAT pathway CEC1 activation. The function                       E. coli bioparticles. These data conform hypothesis a <br /> AGAP009459 phagocytosis E. coli bioparticles related          network negative positive regulators involved induction <br /> to involvement cdc2 cytoskeleton regulation,           LRIM1 expression follows infection. Intriguingly, silencing <br /> also imply novel role JAK/STAT phagocytosis.                       Laminin B1 resulted contrasting increase basal <br />                                                                            LRIM1 promoter activity (Figure 4C), suggesting dual role of <br /> Transcriptional regulation complement factor                        laminin B1 activating suppressing LRIM1 expression the <br /> LRIM1                                                                      presence absence immune challenge, respectively. <br />    We investigated expression LRIM1 gene using                  Two additional genes identified negative regulators of <br /> approach identical described CEC1. LRIM1              LRIM1 basal expression: silencing AGAP009119 and <br /> expressed An. gambiae hemocytes [8] secreted                 AGAP002186 led significant increase luciferase expression <br /> hemolymph disulfide-linked complex structurally              driven LRIM1 promoter (Figure 4C). AGAP009119 encodes <br /> related protein APL1C; complex binds solubilizes             protein tetratrico peptide structural repeats, involved in <br /> TEP1cut, cleaved, activated form complement C3-like               protein-protein interactions, heat shock chaperonin-binding <br /> factor TEP1 [54,55]. This new complex plays key role                  motif, orthologous D. melanogaster Hsp70-interacting <br /> mosquito responses invading malaria parasites. A previous          protein 2 (HIP-replacement). AGAP002186 encodes receptor of <br /> study showed LRIM1 transcriptionally induced following             lipophorin (Lp), insect equivalent low-density <br /> PGN challenge [13]. We used 1600 bp fragment LRIM1                lipoproteins co-ortholog Drosophila LpR1 LpR2 <br /> promoter fused luciferase (courtesy M.J. Povelones). Our             [64]. In An. gambiae, Apolipoprotein I II, main components <br /> preliminary data showed high luciferase activity Sua5.1* cells          Lp, shown act antagonistically TEP1-dependent <br /> but upregulation following PGN challenge.                       malaria parasite killing [65,66,67]. Interestingly, LpR2 (Lipo- <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                 6                        January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                    Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> phorin Receptor 2), fruit fly ortholog AGAP002186,              ment midgut basement membrane (basal lamina), which <br /> been recently shown suppress formation melanotic tumors              induces transient overexpression LRIM1 TEP1 [9]. <br /> [68].                                                                         Indeed complement important defense reaction the <br />    Reduction basal LRIM1 promoter activity cultured              mosquito hemolymph [16,29,70]. Upon TEP1 maturation cleav- <br /> cells detected silencing REL2 compared dsLacZ-           age, TEP1cut binds LRIM1/APL1C complex, it <br /> treated control cells, consistent earlier study showing       remains active soluble state infection occurs [54,55]. <br /> REL2, REL1, control basal LRIM1 TEP1                        However, steps TEP1 binding pathogen surface <br /> expression [69].                                                           remain unknown. Rono co-workers [67] recently <br />                                                                            suggested observed antagonistic effect Lp (and <br /> Perspective: complex regulatory network                           Vitellogenin) TEP1 binding malaria parasites due <br /> hemocyte immune response                                                   Lp masking competing TEP1 binding sites parasite <br />    Our results identify 22 novel regulators hemocyte                surface, directly interacting TEP1, modifying lipid <br /> immune response major African vector human malaria.             composition parasite membrane. Our finding Lp <br /> As summarized Figure 5 Table S5, complex network            receptor suppresses basal expression LRIM1 provides <br /> positive negative regulators immediate (phagocytosis            additional insights mechanism underlying effect of <br /> basal complement state) induced (AMP expression             Lp TEP1-mediated parasite killing. Further investigation the <br /> induced complement state) responses revealed.                          roles Lp receptor, Laminin B1 additional regulators <br />    Four membrane-bound transmembrane proteins impli-                LRIM1 expression identified screen shed light into <br /> cated E. coli phagocytosis, domains          regulatory mechanisms mosquito complement how <br /> could indicate bona fide phagocytic receptors. Two them,              impact infections malaria parasites. <br /> putative Ca2+ channel protein (AGAP000095) v-ATPase <br /> (AGAP003879) predicted functions point                  Materials Methods <br /> involvement phagosome assembly maturation. Of the <br /> remaining membrane-bound transmembrane proteins,                    Mosquito rearing dsRNA injections <br /> AGAP004928 LPS-induced tumor necrosis factor alpha                     An. gambiae G3 strain maintained according standard <br /> factor domain involved immune signaling [33,34].           insectary procedures (www.mr4.org/). DsRNA injections 2- <br />    A intriguing connection revealed phagocytosis           days-old mosquitoes performed described previously [71]. <br /> and regulation basal induced expression LRIM1, a <br /> key component mosquito complement cascade [54,55]               Cell cultures mainteinance RNAi <br /> a known facilitator E. coli phagocytosis [29]. The basement                The An. gambiae cell lines 4a2, 4a3A, 4a3B, L3-5, SuaE.1, <br /> membrane protein Laminin B1 inhibits phagocytosis                 SuaB.1, Sua4.0 Sua5.1* maintained described (www. <br /> basal expression LRIM1, promotes LRIM1 expression             mr4.org/ [12]). Briefly, cells grown Schneider&#226;&#8364;&#8482;s <br /> immune challenge. Therefore, Laminin B1 appears play dual             medium supplemented 10% fetal bovine serum (heat <br /> role maintaining basal levels complement                  inactivated), 100 U/ml Penicillin 100 mg/ml Streptomycin <br /> hemolymph promoting production complement                        27uC. Splitting carried shaking flasks detach cells <br /> components needed preparation potential reinfec-             freezing/thawing procedures performed according to <br /> tions. Whether function Laminin B1 based cell                standard cell culture protocols. RNAi-mediated gene silencing of <br /> signaling hemocyte differentiation, remains investi-          cells carried 96-well plates: approximately 105 cultured <br /> gated. Indeed, suggested immune priming            cells bathed 1 mg dsRNA dissolved serum-free <br /> mosquito hemolymph gut bacteria following midgut invasion               Schneider&#226;&#8364;&#8482;s Medium 2 h later complete medium added to <br /> by malaria parasites causes hemocyte differentiation attach-           obtain final serum concentration 10%. <br />  <br />  <br />  <br />  <br /> Figure 5. Schematic representation role identified regulators An. gambiae hemocyte immune response. A total 22 <br /> positive (green) negative (red) regulators shown. Dotted lines indicate modulations CEC1 LRIM1 gene expression PGN challenge, <br /> and solid lines effect genes phagocytosis basal expression LRIM1. <br /> doi:10.1371/journal.ppat.1003145.g005 <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                 7                        January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                       Novel Regulators Mosquito Innate Immunity <br />  <br />  <br />  <br /> RNA preparation, dsRNA synthesis Quantitative RT-PCR                       Schneider&#226;&#8364;&#8482;s Medium; 2 h later, complete medium added to <br />     Total RNA wild type, KD females cultured cells                obtain final serum concentration 10% volume 50 ml. <br /> extracted using Trizol Reagent (Invitrogen). After DNAseI (Invitro-            PHRodo E. coli bioparticles (Invitrogen) dissolved sterile <br /> gen) treatment, strand cDNA synthesized using oligo-d(T)             1xPBS, sonicated according manufacturer&#226;&#8364;&#8482;s instructions and <br /> primers (Invitrogen) Superscript Reverse Transcriptase II                  50 ml added 3 days gene silencing obtain a <br /> (Invitrogen) according manufacturer&#226;&#8364;&#8482;s instructions.                     final volume 100 ml/well. Cytochalasin D added 20 min <br />     For dsRNA synthesis, T7-tailed primers (see Table S1)                 bioparticle challenge concentration 10 mM or <br /> designed using E-RNAi web-service http://www.dkfz.de/                   100 mM. BMG Labtech FLUOstar OMEGA plate reader was <br /> signaling/e-rnai3/evaluation.php. PCR products synthesized                used measure fluorescence intensity (Ex 532 nm/Em 595 nm <br /> using cDNA female mosquitoes template purified                   filter) immediately pHRodo bioparticles challenge, at <br /> using QIAquick PCR Purification kit (QIAGEN). DsRNA                        time-points 24 h, keeping temperature 27uC <br /> synthesis performed according MEGAscript T7 Kit                         entire procedure. Three replicates performed. Plate <br /> (Ambion) manufacturer&#226;&#8364;&#8482;s protocol purification dsRNA                     reader outcomes used calculate z-score values each <br /> was performed phenol/chloroform extraction. The quality                 replicate. Hits considered dsRNAs producing z-score <br /> quantity dsRNA checked agarose gel electrophoresis                  values 2 22 replicates. <br /> and Nanodrop reading, respectively.                                            Numerical values pooled statistically analyzed using <br />     Quantitative RT-PCR performed using SYBR Green                     ANOVA followed Tukey&#226;&#8364;&#8482;s Multiple Comparison Test assess <br /> PCR mastermix analyzed using ABI PRISM 7700                            dsRNA average value relation dsLacZ. <br /> sequence detection manufacturer&#226;&#8364;&#8482;s instructions.                     Automated fluorescence microscopy employed a <br /> Expression levels calculated relative standard curve               separate method assess fluorescent pHRodo bioparticles uptake <br /> method using S7 endogenous control [72,73]. Primers used                2 h 20 h post challenge dsRNAs showing a <br /> listed Table S1.                                                            phenotype according microplate reader assay. Images were <br />                                                                                captured using Zeiss Axiovert 200 fluorescence widefield <br /> Viability assay                                                                microscopy Volocity Improvision software (106objective). <br />    Cells seeded 96-well plate concentration 105              Image analysis quantification performed using protocol <br /> cells/well. The day medium removed cells                 developed ImageJ. Briefly, images grouped stacks and <br /> treated 1 mg dsRNA dissolved 50 ml serum-free Schneider&#226;&#8364;&#8482;s              uniformly handled; background subtracted, contrast enhanced <br /> Medium; 2 h later, complete medium added obtain final                 stacks transformed binary images separate the <br /> serum concentration 10% volume 100 ml. Four days              fluorescent particles cell layer quantify spot <br /> dsRNA treatment, cells stained 1 mg/ml Hoechst 33342                 number size. Numbers obtained experiments <br /> (Invitrogen), 500 nM Sytox Green (Invitrogen), adding                   statistically analyzed using GraphPad Prism software. <br /> 25 ml chemicals obtain final volume 150 ml/well.               Numbers converted percent values compared dsLacZ. <br /> After 30 min incubation dark 28uC, plates analyzed              After normality test, values dsRNA compared to <br /> by fluorescence microscopy. Images cells captured using              dsLacZ using student&#226;&#8364;&#8482;s t-test. <br /> Zeiss Axiovert 200 widefield fluorescence microscope (106                          In vivo phagocytosis assays performed 2-day old <br /> objective) Improvision Volocity Software. Three images                 mosquitoes injected dsRNA described above. Four days <br /> per taken (bright-field, DAPI GFP). Images                  later, mosquitoes injected thorax 69 nl of <br /> analyzed using protocol developed ImageJ. Briefly, images                 pHRodo E. coli bioparticles allowed recover resume <br /> were grouped stacks uniformly handled; stacks                      phagocytosis 1 h. Mosquitoes partially dissected (wings <br /> transformed binary images, cells labeled Hoechst                 legs removed) gently compressed slide a <br /> and Sytox Green separately counted using Find                        coverslip (using clay hold together) imaging. Images of <br /> Maxima Process Tool. Numbers obtained statistically                       mosquito abdomens captured using Zeiss Axiovert 200 <br /> analyzed quantify cell viability. ANOVA statistical analysis                widefield fluorescence microscope Volocity Improvision <br /> followed Bonferroni&#226;&#8364;&#8482;s post-test applied groups           software. At replicates dsRNA treatment were <br /> six pictures dsRNA treatment.                                              carried out, 8 mosquitoes KD analyzed by <br />    To assess cell size, stacks processed subtract              microscopy pictures captured mosquito. <br /> background enhance contrast transformed                      Fluorescence images sections mosquito abdomens were <br /> binary images; cells converted particles,                  captured described shown Figure 3 (106 <br /> number, size shape calculated.                                        objective). Quantification statistical analysis carried out <br />    Viability assay performed using Cell Titer Blue kit                described pHRodo bioparticle phagocytosis in <br /> (Promega). Four days dsRNA treatment carried above,               cultured cells. <br /> 20 ml/well CellTiter Blue solution added obtain final <br /> volume 120 ml/well. Plates incubated 2 h                  Luciferase assay <br /> fluorescence produced reduction substrate resazurin                    Sua5.1* cells seeded 96-well plate concentration <br /> resorufin measured BMG Labtech FLUOstar OMEGA                         105 cells well. Upon reaching 80% confluence, medium <br /> plate reader. Three replicates performed z-score analysis         removed cells treated 1 mg dsRNA dissolved in <br /> applied identify positive phenotypes replicate. Thresholds       40 ml serum-free Schneider&#226;&#8364;&#8482;s Medium. Two h later, complete <br /> +/22 applied positive candidates considered                medium added obtain final serum concentration 10% <br /> passing threshold replicates.                 volume 60 ml. Four days dsRNA treatment, cells were <br />                                                                                washed co-transfected 50 ml final volume CEC1 or <br /> Phagocytosis assay                                                             LRIM1 promoter firefly luciferace fusion reporter constructs <br />   Sua5.1* cells seeded 96-well plate concentration             (Promega pGL3 backbone) reference pAct5C-Renilla lucifer- <br /> 105 cells/well. The day medium removed cells                  ase construct using Effectene Transfection Reagent (QIAGEN) <br /> were treated 1 mg dsRNA dissolved 40 ml serum-free                     following manufacturer&#226;&#8364;&#8482;s instructions. One day transfec- <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                     8                        January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                      Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> tion, 25 ml PGN (Sigma) added obtain final                      Figure S2     Qualitative analysis viability assays. (A) <br /> concentration 100 mg/ml. Seven h later, cells subjected              Nuclear staining Hoechst Sua4.0 cultured cells are <br /> to Dual Luciferase assay (Promega) according manufactur-              untreated (no dsRNA) treated dsRNA targeting <br /> er&#226;&#8364;&#8482;s instructions final volume 150 ml well. Relative light       LacZ (dsLacZ) AGAP005160 (ds5160). Note ds5160 <br /> units second firefly renilla luciferase                treatment, cells showed impaired cytokinesis increased nuclear <br /> measured using Luminometer mode BMG Labtech                           size. (B) Quantification number cells counts Hoechst <br /> FLUOstar OMEGA plate reader. RLU measurements                           positive nuclei (C) measurements nuclear size, using a <br /> obtained dividing firefly renilla measurements. RLUs             protocol developed ImageJ. Pictures taken three <br /> three replicates averaged z-scores calculated PGN             independent replicates analyzed. Mean values standard <br /> and PBS treatments. One-way ANOVA followed Dunnet&#226;&#8364;&#8482;s                       errors shown. One-way ANOVA followed Tukey&#226;&#8364;&#8482;s Multiple <br /> multiple-comparison post-test performed compare                  Comparison Test used statistical analyses (*, p,0,05; ***, <br /> dsLacZ control dsRNA RLUs.                                               p,0,001). <br />                                                                              (TIF) <br /> Data analysis <br />                                                                              Figure S3 Z-score correlation replicates in <br />    Z-score calculated using formula zkj = (ykj2M)/S, where <br />                                                                              viability assays. D&#226;&#8364;&#8482;Agostino &amp; Pearson omnibus normality test <br /> ykj background subtracted value kth jth <br />                                                                              revealed data fit normal distribution. Therefore, <br /> replicate, M S mean standard deviation (SD) of <br />                                                                              correlation z-score values replicates I, II III was <br /> the distribution y values, respectively [74]. We considered <br />                                                                              calculated using Spearman r correlation test. Correlation coeffi- <br /> positive hits dsRNAs exhibiting z-scores.2 ,22, which <br />                                                                              cients (r) indicated. Positive hits (z-scores.2 ,22 least <br /> corresponds SDs population mean a <br /> given replicate. Considering critical z-score values           replicates) shown open circles. Control no <br /> using 95% confidence level 21.96 +1.96 SDs, positive               dsRNA-treated samples samples treated 100 mM H2O2 <br /> hits corresponding p values,0.05. To assess z-score                     shown. <br /> correlation replicates, D&#226;&#8364;&#8482;Agostino &amp; Pearson omnibus                 (TIF) <br /> normality test applied examine data follow              Figure S4 Phagocytosis assay. (A) Sua5.1* cells treated with <br /> Gaussian distribution. Correlations computed using Spear-               dsLacZ, dsCactus dsBINT2 challenged E. coli and <br /> man r correlation test normally distributed data                 Staphylococcus aureus pHRodo bioparticles levels of <br /> Pearson r correlation test normally distributed data. GraphPad           phagocytosis measured 1 h later microtiter plate reading. <br /> Prism 5 software used statistical analyses graph design.         Values background subtracted. (B) Sua5.1* cells treated <br />                                                                              Cytochalasin D concentrations 10 mM 100 mM in <br /> Supporting Information                                                       conditioned medium challenged pHRodo E. coli. <br />                                                                              The levels phagocytosis measured different time points <br /> Dataset S1 dsRNA hemocyte-specific library. The main <br />                                                                              (TP) indicated. Values background subtracted. (C) Z-score <br /> features dsRNAs target genes used study are <br />                                                                              plots relative microplate reader measurements E. coli <br /> presented. DsRNA IDs, KD phenotypes RNAi screens, <br />                                                                              pHRodo conjugated bioparticles uptake different TPs. Each <br /> VectorBase gene IDs, Affymetrix probe codes previous <br /> ENSEMBL IDs listed columns A, B, C, D E,                          dsRNA screened triplicate positive hits considered <br /> respectively. Circulating hemocyte microarray information               dsRNAs z-score 2 3 replicates <br /> [8] summarized columns F (cluster), G (normalized                     .2 (red) ,22 (green). 0 h TP (immediately bioparticle <br /> hemocyte value), H (normalized carcass value) I (normalized              challenge) subtracted graphs TP 1-0, TP 3-0, TP 6-0 <br /> head value). Comments (name and/or homology An. gambiae                   TP 24-0 (measurements 1 h, 3 h, 6 h 24 h, <br /> genes) IPRO domain data reported columns J K,                 respectively). Z-scores TP 1, 3 6 (1, 3 6 h after <br /> respectively. D. melanogaster orthologs shown column L,           challenge, subtracting TP 0 h, respectively) shown, <br /> corresponding FlyBase IDs column M. D. melanogaster KD                    z-score plots indicating kinetic uptake between <br /> phenotypes according GenomeRNAi database reported                 1 h 3 h challenge (TP 3-1). (D) Correlation z- <br /> in column N. For details Text S1.                                score values different replicates. TPs selected indicated on <br /> (XLS)                                                                        graph, replicates compared specified left <br />                                                                              graph. Correlation coefficients (r) indicated. <br /> Figure S1 Quantitative analysis viability assays. (A)                     (TIF) <br /> Images L3&#226;&#8364;&#8220;5 confluent cell layers dsRNA treatments as <br /> indicated left figure. Nuclei stained           Figure S5 Gene-specific KD efficiency cell cultures <br /> Hoechst, dead cells stained Sytox Green. (B) Viability          (A&#226;&#8364;&#8220;D) mosquitoes (E). Confluent Sua5.1* cells were <br /> assay analysis 8 An. gambiae cell lines treated dsRNAs               incubated dsRNAs targeting AGAP004016 (A), <br /> targeting IAP1, AGAP005160 AGAP008001. Total number                   AGAP004928 (B), AGAP005227 (C) AGAP009201 (D), and <br /> cells (Hoechst) dead cells (Sytox Green) counted                 4 days later expression targeted genes analyzed qRT- <br /> image analysis using protocol developed ImageJ. Ratios               PCR. Data normalized S7 calibrated gene- <br /> dead total number cells reported graphs. Three             specific expression dsLacZ treated samples. Three wells each <br /> pictures taken independent replicates analyzed,            KD analyzed. The level silencing genes in <br /> averages standard errors shown. One-way ANOVA                        mosquitoes tested (E). dsRNAs targeting AGAP004928, <br /> followed Bonferroni&#226;&#8364;&#8482;s Multiple Comparison Test applied                AGAP005227, AGAP009201 LacZ injected into <br /> (*, p,0,05; **, p,0,01; ***, p,0,001). (C) Kaplan-Meier survival             2-day old female mosquitoes expression silenced genes <br /> curves adult mosquitoes following injection dsRNA                    measured qRT-PCR 4 days dsRNA treatments. Data <br /> targeting IAP1 AGAP008001. Percent survival compared                  normalized S7 levels calibrated gene-specific <br /> dsLacZ treated controls shown.                                            expression dsLacZ treated mosquitoes. <br /> (TIF)                                                                        (TIF) <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                   9                        January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                                        Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> Table S1 List primers used study. Name                                       PGN challenge according z-score analysis shown; KDs <br /> dsRNA (#), AGAP <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> (ID), T7-tailed primers <span id='am-317' about='Thesaurus:code' typeof='owl:Thing'>code</span> (Primer                                  modulating basal LRIM1 promoter activity (PBS challenge) are <br /> Forward Primer Reverse), relative sequences                                    summarized; IPRO domains short descriptions reported. <br /> reported. Primers Q-PCR (QF QR) listed                                  SP, Signal peptide; TD, transmembrane domain. <br /> bottom table.                                                                       (DOC) <br /> (DOC) <br />                                                                                            Table S5 Summary RNAi screens results. In column <br /> Table S2 Genes exploited controls viability assays.                                  &#226;&#8364;&#732;&#226;&#8364;&#732;#&#226;&#8364;&#8482;&#226;&#8364;&#8482; listed dsRNA labels; AGAP ID number IPRO <br /> Name dsRNA (#), AGAP code (IDs), gene name, IPRO ID                                 domain descriptions homologies reported 2 <br /> <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> T7 dsRNA products reported.                                                  columns. KD phenotypes genes gave positive phenotype <br /> (DOC)                                                                                      4 assays summarized 4 <br /> Table S3      Phagocytosis assay results. Datasets vitro                           columns. <br /> measurements using microplate reader statistically analysed                           (DOC) <br /> employing statistical approaches, z-score threshold                                Text S1 Additional information data viability <br /> ANOVA calculation. In column &#226;&#8364;&#732;&#226;&#8364;&#732;#&#226;&#8364;&#8482;&#226;&#8364;&#8482; listed dsRNA labels;                                assay, ex vivo phagocytosis assay, knockdown efficiency <br /> AGAP ID number IPRO domain descriptions reported                                assessment Drosophila melanogaster orthologs. <br /> the columns. &#226;&#8364;&#732;&#226;&#8364;&#732;Z-score&#226;&#8364;&#8482;&#226;&#8364;&#8482; column lists genes                                  (DOC) <br /> significant values indicated TP (for replicates <br /> out three) plate reader measurements. Microplate reader <br /> values averaged compared dsLacZ control values:                           Acknowledgments <br /> positive hits TP, according ANOVA statistical analysis                         We thank Tibebu Habtewold Katarzyna Sala help with <br /> followed Tukey&#226;&#8364;&#8482;s Multiple Comparison Test, listed                                mosquito rearing, Martin Spitaler rest Imperial College <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;ANOVA P,0,05&#226;&#8364;&#8482;&#226;&#8364;&#8482; column. 8 genes selected 13                                    London FILM Facility staff assistance microscopy imaging. We are <br /> significant candidates evaluated in vivo assay. The &#226;&#8364;&#732;&#226;&#8364;&#732;in vivo                      grateful Kristin Michel Sofia Pinto sharing reagents data <br /> % TP2&#226;&#8364;&#8482;&#226;&#8364;&#8482; column reports percent phagocytosis calculated                           prior publication helpful discussions. We thank Lynda <br />                                                                                            Stuart critical reading manuscript. <br /> by imaging analysis in vivo experiments 2 h challenge. SP, <br /> Signal Peptide; TD, transmembrane domain; ns, significant; <br /> nd, determined.                                                                        Author Contributions <br /> (DOC)                                                                                      Conceived designed experiments: FL FCK GKC. Performed the <br />                                                                                            experiments: FL YG. Analyzed data: FL GKC. Wrote paper: FL <br /> Table S4 Luciferase assay results. Gene knockdowns (KD) <br />                                                                                            FCK GKC. <br /> that modulate regulation CEC LRIM1 promoters upon <br />  <br /> References <br />  1. WHO (2008) World Malaria Report.                                                       14. Dimopoulos G, Richman A, Muller HM, Kafatos FC (1997) Molecular immune <br />  2. Christophides GK, Vlachou D, Kafatos FC (2004) Comparative functional                  responses mosquito Anopheles gambiae bacteria malaria parasites. <br />     genomics innate immune malaria vector Anopheles                       Proc Natl Acad Sci U S A 94: 11508&#226;&#8364;&#8220;11513. <br />     gambiae. Immunol Rev 198: 127&#226;&#8364;&#8220;148.                                                     15. Meister S, Kanzok SM, Zheng XL, Luna C, Li TR, et al. (2005) Immune <br />  3. Yassine H, Osta MA (2010) Anopheles gambiae innate immunity. Cell                          signaling pathways regulating bacterial malaria parasite infection the <br />     Microbiol 12: 1&#226;&#8364;&#8220;9.                                                                         mosquito Anopheles gambiae. Proc Natl Acad Sci U S A 102: 11420&#226;&#8364;&#8220;11425. <br />  4. Hillyer JF, Barreau C, Vernick KD (2007) Efficiency salivary gland invasion         16. Levashina EA, Moita LF, Blandin S, Vriend G, Lagueux M, et al. (2001) <br />     malaria sporozoites controlled rapid sporozoite destruction                Conserved role complement-like protein phagocytosis revealed dsRNA <br />     mosquito haemocoel. Int J Parasitol 37: 673&#226;&#8364;&#8220;681.                                           <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>knockout</span> cultured cells mosquito, Anopheles gambiae. Cell 104: 709&#226;&#8364;&#8220; <br />  5. Blandin SA, Marois E, Levashina EA (2008) Antimalarial responses                        718. <br />     Anopheles gambiae: complement-like protein complement-like                 17. Cirimotich CM, Dong Y, Garver LS, Sim S, Dimopoulos G (2010) Mosquito <br />     pathway. Cell Host Microbe 3: 364&#226;&#8364;&#8220;374.                                                     immune defenses Plasmodium infection. Dev Comp Immunol 34: 387&#226;&#8364;&#8220;395. <br />  6. Christophides GK, Zdobnov E, Barillas-Mury C, Birney E, Blandin S, et al.              18. Meister S, Agianian B, Turlure F, Relogio A, Morlais I, et al. (2009) Anopheles <br />     (2002) Immunity-related genes gene families Anopheles gambiae. Science              gambiae PGRPLC-mediated defense bacteria modulates infections with <br />     298: 159&#226;&#8364;&#8220;165.                                                                              malaria parasites. PLoS Pathog 5: e1000542. <br />  7. Waterhouse RM, Kriventseva EV, Meister S, Xi Z, Alvarez KS, et al. (2007)              19. Cherry S (2008) Genomic RNAi screening Drosophila S2 cells: we <br />     Evolutionary dynamics immune-related genes pathways disease-vector               learned host-pathogen interactions? Curr Opin Microbiol 11: 262&#226;&#8364;&#8220;270. <br />     mosquitoes. Science 316: 1738&#226;&#8364;&#8220;1743.                                                    20. Echeverri CJ, Perrimon N (2006) High-throughput RNAi screening cultured <br />  8. Pinto SB, Lombardo F, Koutsos AC, Waterhouse RM, McKay K, et al. (2009)                    cells: user&#226;&#8364;&#8482;s guide. Nat Rev Genet 7: 373&#226;&#8364;&#8220;384. <br />     Discovery Plasmodium modulators genome-wide analysis circulating              21. Boutros M, Ahringer J (2008) The art design genetic screens: RNA <br />     hemocytes Anopheles gambiae. Proc Natl Acad Sci U S A: doi:10.1073/                     interference. Nat Rev Genet 9: 554&#226;&#8364;&#8220;566. <br />     pnas.0909463106                                                                        22. Boutros M, Kiger AA, Armknecht S, Kerr K, Hild M, et al. (2004) Genome- <br />  9. Rodrigues J, Brayner FA, Alves LC, Dixit R, Barillas-Mury C (2010) Hemocyte                wide RNAi analysis growth viability Drosophila cells. Science 303: <br />     differentiation mediates innate immune memory Anopheles gambiae                         832&#226;&#8364;&#8220;835. <br />     mosquitoes. Science 329: 1353&#226;&#8364;&#8220;1355.                                                    23. Lawson D, Arensburger P, Atkinson P, Besansky NJ, Bruggner RV, et al. (2009) <br /> 10. Lavine MD, Strand MR (2002) Insect hemocytes role immunity.                   VectorBase: data resource invertebrate vector genomics. Nucleic Acids Res <br />     Insect Biochem Mol Biol 32: 1295&#226;&#8364;&#8220;1309.                                                     37: D583&#226;&#8364;&#8220;587. <br /> 11. Baton LA, Robertson A, Warr E, Strand MR, Dimopoulos G (2009) Genome-                  24. Horn T, Arziman Z, Berger J, Boutros M (2007) GenomeRNAi: database for <br />     wide transcriptomic profiling Anopheles gambiae hemocytes reveals                       cell-based RNAi phenotypes. Nucleic Acids Res 35: D492&#226;&#8364;&#8220;497. <br />     pathogen-specific signatures bacterial challenge Plasmodium berghei           25. Gilsdorf M, Horn T, Arziman Z, Pelz O, Kiner E, et al. (2010) GenomeRNAi: a <br />     infection. BMC Genomics 10: 257.                                                           database cell-based RNAi phenotypes. 2009 update. Nucleic Acids Res 38: <br /> 12. Muller HM, Dimopoulos G, Blass C, Kafatos FC (1999) A hemocyte-like cell                   D448&#226;&#8364;&#8220;452. <br />     line established malaria vector Anopheles gambiae expresses               26. Danielli A, Kafatos FC, Loukeris TG (2003) Cloning characterization of <br />     prophenoloxidase genes. J Biol Chem 274: 11727&#226;&#8364;&#8220;11735.                                      Anopheles gambiae serpin isoforms, differentially induced midgut by <br /> 13. Dimopoulos G, Christophides GK, Meister S, Schultz J, White KP, et al. (2002)              Plasmodium berghei invasion. J Biol Chem 278: 4184&#226;&#8364;&#8220;4193. <br />     Genome expression analysis Anopheles gambiae: responses injury,                  27. Somma MP, Fasulo B, Cenci G, Cundari E, Gatti M (2002) Molecular dissection <br />      bacterial challenge, malaria infection. Proc Natl Acad Sci U S A 99: 8814&#226;&#8364;&#8220;            cytokinesis <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> Drosophila cultured cells. Mol Biol Cell <br />     8819.                                                                                      13: 2448&#226;&#8364;&#8220;2460. <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                                10                            January 2013 | Volume 9 | Issue 1 | e1003145 <br />                                                                                                                             Novel Regulators Mosquito Innate Immunity <br />  <br />  <br /> 28. Blandin SA, Levashina EA (2007) Phagocytosis mosquito immune responses.                 52. Gupta L, Molina-Cruz A, Kumar S, Rodrigues J, Dixit R, et al. (2009) The <br />     Immunol Rev 219: 8&#226;&#8364;&#8220;16.                                                                         STAT pathway mediates late-phase immunity Plasmodium the <br /> 29. Moita LF, Wang-Sattler R, Michel K, Zimmermann T, Blandin S, et al. (2005)                     mosquito Anopheles gambiae. Cell Host Microbe 5: 498&#226;&#8364;&#8220;507. <br />     In vivo identification novel regulators conserved pathways phagocytosis          53. Souza-Neto JA, Sim S, Dimopoulos G (2009) An evolutionary conserved <br />     A. gambiae. Immunity 23: 65&#226;&#8364;&#8220;73.                                                             function JAK-STAT pathway anti-dengue defense. Proc Natl Acad <br /> 30. Hillyer JF, Schmidt SL, Christensen BM (2003) Rapid phagocytosis                           Sci U S A 106: 17841&#226;&#8364;&#8220;17846. <br />     melanization bacteria Plasmodium sporozoites hemocytes                    54. Povelones M, Waterhouse RM, Kafatos FC, Christophides GK (2009) Leucine- <br />     mosquito Aedes aegypti. J Parasitol 89: 62&#226;&#8364;&#8220;69.                                                 rich repeat protein complex activates mosquito complement defense against <br /> 31. Warr E, Lambrechts L, Koella JC, Bourgouin C, Dimopoulos G (2006)                              Plasmodium parasites. Science 324: 258&#226;&#8364;&#8220;261. <br />     Anopheles gambiae immune responses Sephadex beads: involvement anti-                 55. Fraiture M, Baxter RH, Steinert S, Chelliah Y, Frolet C, et al. (2009) Two <br />     Plasmodium factors regulating melanization. Insect Biochem Mol Biol 36:                     mosquito LRR proteins function complement control factors TEP1- <br />     769&#226;&#8364;&#8220;778.                                                                                       mediated killing Plasmodium. Cell Host Microbe 5: 273&#226;&#8364;&#8220;284. <br /> 32. Miksa M, Komura H, Wu R, Shah KG, Wang P (2009) A novel method                          56. Kocks C, Cho JH, Nehme N, Ulvila J, Pearson AM, et al. (2005) Eater, a <br />     determine engulfment apoptotic cells macrophages using pHrodo                        transmembrane protein mediating phagocytosis bacterial pathogens in <br />     succinimidyl ester. J Immunol Methods 342: 71&#226;&#8364;&#8220;77.                                              Drosophila. Cell 123: 335&#226;&#8364;&#8220;346. <br /> 33. Igaki T, Kanda H, Yamamoto-Goto Y, Kanuka H, Kuranaga E, et al. (2002)                     57. Kurucz E, Markus R, Zsamboki J, Folkl-Medzihradszky K, Darula Z, et al. <br />     Eiger, TNF superfamily ligand triggers Drosophila JNK pathway.                      (2007) Nimrod, putative phagocytosis receptor EGF repeats Drosophila <br />     Embo J 21: 3009&#226;&#8364;&#8220;3018.                                                                          plasmatocytes. Curr Biol 17: 649&#226;&#8364;&#8220;654. <br /> 34. Ponting CP, Mott R, Bork P, Copley RR (2001) Novel protein domains                     58. Somogyi K, Sipos B, Penzes Z, Kurucz E, Zsamboki J, et al. (2008) Evolution of <br />     repeats Drosophila melanogaster: insights structure, function,                     genes repeats Nimrod superfamily. Mol Biol Evol 25: 2337&#226;&#8364;&#8220;2347. <br />     evolution. Genome Res 11: 1996&#226;&#8364;&#8220;2008.                                                       59. Bond D, Foley E (2009) A quantitative RNAi screen JNK modifiers identifies <br /> 35. Dong Y, Dimopoulos G (2009) Anopheles fibrinogen-related proteins provide                      Pvr novel regulator Drosophila immune signaling. PLoS Pathog 5: <br />     expanded pattern recognition capacity bacteria malaria parasites.                  e1000655. <br />     J Biol Chem 284: 9835&#226;&#8364;&#8220;9844.                                                                60. Friedman AA, Tucker G, Singh R, Yan D, Vinayagam A, et al. (2011) <br /> 36. Nunes P, Demaurex N (2010) The role calcium signaling phagocytosis.                      Proteomic functional genomic landscape receptor tyrosine kinase ras <br />     J Leukoc Biol 88: 57&#226;&#8364;&#8220;68.                                                                       extracellular signal-regulated kinase signaling. Sci Signal 4: rs10. <br /> 37. Peri F, Nusslein-Volhard C (2008) Live imaging neuronal degradation                  61. Surachetpong W, Singh N, Cheung KW, Luckhart S (2009) MAPK ERK <br />     microglia reveals role v0-ATPase a1 phagosomal fusion vivo. Cell 133:              signaling regulates TGF-beta1-dependent mosquito response Plasmodium <br />     916&#226;&#8364;&#8220;927.                                                                                       falciparum. PLoS Pathog 5: e1000366. <br /> 38. Agaisse H, Burrack LS, Philips JA, Rubin EJ, Perrimon N, et al. (2005) Genome- <br />                                                                                                62. Volz J, Muller HM, Zdanowicz A, Kafatos FC, Osta MA (2006) A genetic <br />     wide RNAi screen host factors required intracellular bacterial infection. <br />                                                                                                    module regulates melanization response Anopheles Plasmodium. Cell <br />     Science 309: 1248&#226;&#8364;&#8220;1251. <br />                                                                                                    Microbiol 8: 1392&#226;&#8364;&#8220;1405. <br /> 39. Derre I, Pypaert M, Dautry-Varsat A, Agaisse H (2007) RNAi screen in <br />                                                                                                63. Schnitger AK, Kafatos FC, Osta MA (2007) The melanization reaction not <br />     Drosophila cells reveals involvement Tom complex Chlamydia <br />                                                                                                    required survival Anopheles gambiae mosquitoes bacterial infections. <br />     infection. PLoS Pathog 3: 1446&#226;&#8364;&#8220;1458. <br />                                                                                                    J Biol Chem 282: 21884&#226;&#8364;&#8220;21888. <br /> 40. Cherry S, Kunte A, Wang H, Coyne C, Rawson RB, et al. (2006) COPI activity <br />                                                                                                64. Tufail M, Takeda M (2009) Insect vitellogenin/lipophorin receptors: molecular <br />     coupled fatty acid biosynthesis required viral replication. PLoS Pathog <br />          <br /> </body></html>