<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 2 October 2012                                           Nucleic Acids Research, 2012, Vol. 40, No. 22 11545&#226;&#8364;&#8220;11553 <br />                                                                                                                 doi:10.1093/nar/gks880 <br />  <br />  <br /> The C-terminus Dpb2 required interaction <br /> with Pol2 cell viability <br /> Isabelle Isoz, Ulf Persson, Kirill Volkov Erik Johansson* <br /> Department Medical Biochemistry Biophysics, Umea&#203;&#353; University, SE-90187 Umea&#203;&#353;, Sweden <br />  <br /> Received October 6, 2011; Revised August 22, 2012; Accepted August 25, 2012 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        regulating location timing DNA synthesis by <br />                                                                                 individual DNA polymerases (3). This regulation is <br /> DNA polymerase e (Pol e) participates synthe-                            critical replication fork dynamic structure <br /> sis leading strand DNA replication                             allow different DNA polymerases <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Saccharomyces cerevisiae. Pol e comprises                                  change places functions required during <br /> subunits: catalytic subunit, Pol2, ac-                            DNA replication (1,2,4). <br /> cessory subunits, Dpb2, Dpb3 Dpb4. DPB2                                  Pol synthesizes primer required the <br /> essential gene unclear function. A genetic                                 origin beginning Okazaki fragment <br /> screen performed S. cerevisiae isolate                                lagging strand. The subunits Pol are <br /> lethal mutations DPB2. The dpb2-200 allele                                   Pri1, Pri2, Pol1 Pol12. The Pri1 subunit synthesizes <br /> carried mutations 13 codons                              short RNA primer. This followed switch DNA <br /> the open reading frame, resulted                              synthesis Pol1 subunit extends RNA <br />                                                                                 primer DNA primer 20 nucleotides <br /> six amino acid truncation. This truncated Dpb2 <br />                                                                                 long (5). The Pri2 Pol12 subunits lack catalytic <br /> subunit co-expressed Pol2, Dpb3                                    activity, Pol12 post-translationally modi&#239;&#172;?ed <br /> Dpb4 S. cerevisiae, Dpb2 variant did                            cell cycle response DNA damage <br /> co-purify Pol e subunits. This resulted                          (6). This suggests Pol12 functionally im- <br /> in purification Pol2/Dpb3/Dpb4 complex                            portant Pol a. <br /> possessed high specific activity high proces-                                  Increasing evidence suggests Pol e replicates the <br /> sivity holoenzyme assays PCNA, RFC                                 leading strand Pol d replicates lagging strand <br /> RPA single-primed circular template did                                DNA replication (7&#226;&#8364;&#8220;10). In Saccharomyces <br /> reveal defects replication efficiency. In con-                           cerevisiae, Pol d consists subunits, Pol3, Pol31 <br /> clusion, lack Dpb2 did appear                              Pol32. Pol3 catalytic subunit synthesizes <br /> negative effect Pol e activity. Thus, C-terminal                         DNA associated 30 &#226;&#8364;&#8220;50 -exonuclease activity that <br />                                                                                 remove mis-incorporated nucleotides. POL31 an <br /> motif Dpb2 identified instead be <br />                                                                                 essential gene, vivo function unclear. POL32 <br /> required Dpb2 fulfill essential structural                            essential gene S. cerevisiae (11), Pol32 <br /> role replication origin replication                            subunit plays important roles cell. The N-terminus <br /> fork.                                                                           Pol32 interacts Pol31 C-terminus interacts <br />                                                                                 PCNA, processivity clamp, Pol32 been <br />                                                                                 shown required Pol d fully processive in <br /> INTRODUCTION                                                                    holoenzyme assays (11,12). Deletion POL32 gene <br /> Three DNA polymerases, DNA polymerase (Pol a),                                renders yeast strains cold sensitive hydroxyurea sen- <br /> DNA polymerase d (Pol d) DNA polymerase e                                   sitive, POL32 required ultraviolet (UV)-induced <br /> (Pol e), participate replication fork eukary-                     mutagenesis break-induced repair (11,13,14). <br /> otic DNA replication (1,2). All polymerases                           Recently, shown Pol31 Pol32 physically <br /> multi-domain enzymes composed subunits                               interact Rev3 subunits error-prone <br /> encoded separate genes (3). The functions                             DNA polymerase z (15). Thus, Pol31 Pol32 func- <br /> smaller accessory subunits clear cases,                      tionally important carry of <br /> emerging evidence indicates important                         catalytic sites Pol d holoenzyme. <br />  <br />  <br /> *To correspondence addressed. Tel: +46 90 7866638; Fax: +46 90 7869795; Email: erik.johansson@medchem.umu.se <br /> Present address: <br /> Kirill Volkov, Abzyme Therapeutics, Inc., 271 Great Valley Parkway, Malvern, PA 19355, USA. <br />  <br /> The authors wish known that, opinion, &#239;&#172;?rst authors regarded joint First Authors. <br />  <br /> &#195;&#376; The Author(s) 2012. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted, distribution, reproduction medium, provided original work properly cited. <br />  11546 Nucleic Acids Research, 2012, Vol. 40, No. 22 <br />  <br />  <br />    Pol e subunits, Pol2, Dpb2, Dpb3          MATERIALS AND METHODS <br /> and Dpb4 (16). Pol2 catalytic subunit             Media growth conditions <br /> DNA polymerization site 30 &#226;&#8364;&#8220;50 -exonuclease site <br /> that proofreads newly synthesized strand (17).              Saccharomyces cerevisiae strains routinely grown in <br /> POL2 essential gene, deletion              yeast complete media (YPD) contained yeast extract <br /> entire region codes N-terminal domain           (1%), peptone (2%) glucose (2%). For transform- <br /> the protein contains polymerase               ation selection, used synthetic complete drop- <br /> proofreading activities did result lethality (18).       out&#226;&#8364;&#8221;media (SC-media) contained amino acid-free <br /> The cells viable, did severe growth             yeast nitrogen <span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span>, amino acids and <br /> defect. However, <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> inactivated          ammonium sulfate (0.34%), glucose (2%) and <br /> polymerase active site kept portion             ammonium sulfate (0.5%) supplemented with <br /> protein intact lethal (19). The current model            appropriate amino acids nutrients required. <br /> that Pol d capable rescuing leading strand synthesis      For selection kanamycin resistance gene (KanR) <br /> at replication fork 120 kD N-terminal              hygromycin B resistance gene (hph), G418 <br /> fragment Pol2 missing, rescue synthesis        (200 mg/ml) hygromycin B (200 mg/ml) added to <br /> in presence inactivating point <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. In          YPD media, respectively. For plasmid shuf&#239;&#172;&#8218;ing, <br /> contrast, deletion region POL2 gene           0.1% 5-&#239;&#172;&#8218;ouroorotic acid (5-FOA) added SC <br /> encoding C-terminus Pol2 resulted inviable            media. Sporulation dpb2-200 heterozygote cells was <br /> cells. DPB3 DPB4 non-essential genes                 performed solid rich sporulation media contained <br /> S. cerevisiae (20,21). Dpb3 Dpb4 form complex             potassium acetate (1%), yeast extract (0.1%), glucose <br /> that interacts double-stranded DNA vitro                (0.05%) agar (2%) supplemented the <br /> (22) explain Dpb3 Dpb4                 appropriate amino acids nutrients. <br /> important processivity Pol e (23,24). In <br /> addition, deletion DPB3               Construction DPB2-de&#239;&#172;?cient strain <br /> and DPB4 generated substrates Pol z increased           Genomic DNA S. cerevisiae strain E134 (MAT  <br /> the rate mutagenesis (20,24,25). In contrast              ade5-1 lys2::InsEA14 trp1-289 his7-2 leu2-3,112 ura3-52) <br /> the proofreading-de&#239;&#172;?cient mutant Pol e (pol2-4),             (34) used template cloning DPB2 gene by <br /> errors DNA synthesis corrected               polymerase chain reaction (PCR) primers p1 <br /> mismatch repair DPB3 DPB4                  (50 -TATGGTACCCCGAAATCCGTTACATCAA-30 ) <br /> deleted (24).                                                   p2 (50 -TATGGATCCTAGAGGCACACAATCCTG <br />    DPB2 essential gene, role Dpb2 Pol e     AGAA-30 ). The 2.7 kb PCR product digested with <br /> activity unclear (26). Temperature-sensitive (ts) alleles    KpnI BamHI restriction enzymes cloned a <br /> of DPB2 isolated cause higher mutation         pRS316 vector contained URA3 gene sequence. <br /> rate vivo (27,28). In two-hybrid assays,         The resulting vector called pRS316-DPB2 the <br /> shown increase mutation rates conferred          DPB2 gene sequenced directions. <br /> different ts dpb2 alleles correlated reduced             The chromosomal DPB2 gene deleted E134 yeast <br /> af&#239;&#172;?nity mutant Dpb2 subunits                cells carried pRS316-DPB2 plasmid transform- <br /> C-terminus Pol2 (29). Dpb2 phosphorylated          ing cells 1.7 kb PCR product contained <br /> cyclin-dependent protein kinase (CDK) Cdc28 late             DPB2-&#239;&#172;&#8218;anking regions. This PCR product ampli&#239;&#172;?ed <br /> G1-phase, functional importance phosphor-       pFA6a-KanMX6 (35) primers p3 (50 -TCTATG <br /> ylation unclear (30). A nuclear magnetic resonance           TGTGAAATGTTTGGCTCTGGGAATGTTCTGCCT <br /> solution structure N-terminal domain Dpb2             GTTAAAATTCAGCCTCCAGCTGAAGCTTCGTAC <br /> has solved, function domain remains        GC-30 ) p4 (50 -CAGAGTACCAACACTACCGTGCT <br /> unknown (31). Independent studies suggested role         TGTTTAAGTGTTCTTATTTTGATGCATTATTTAG <br /> for Dpb2 initiation DNA replication (32,33)       CATAGGCCACTAGTGGATCTG-30 ). E134 cells were <br /> and recently shown Dpb2 co-precipitates        transformed PCR product, integration was <br /> with proteins form pre-loading complex             con&#239;&#172;?rmed PCR (Supplementary Figure S1), the <br /> (pre-LC) (33).                                                  resulting strain called E134-dpb2D. <br />    In present study, isolated lethal dpb2 alleles <br /> that allowed investigate role Dpb2 Pol     Isolation alleles carry lethal mutations DPB2 <br /> e function. We Dpb2 protein expressed            The pRS316-DPB2 plasmid digested KpnI and <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> lethal alleles did co-purify         BamHI DPB2 fragment cloned TRP1- <br /> Pol2, Dpb3 Dpb4, subsequent functional                  containing pRS314 vector. The resulting vector called <br /> studies puri&#239;&#172;?ed Pol e complex lacked Dpb2           pRS314-DPB2. Random mutagenesis performed on <br /> showed Pol e does depend Dpb2 syn-          pRS314-DPB2 plasmid treating hydroxy- <br /> thesis new DNA. Thus, propose essential          lamine (36).The mutated plasmids puri&#239;&#172;?ed a <br /> function Dpb2 separate enzymatic activity        plasmid puri&#239;&#172;?cation kit (Qiagen) remove traces of <br /> of Pol e S. cerevisiae Dpb2 depends            hydroxylamine transforming E134-dpb2D cells. <br /> distinct domain C-terminal end               Transformants selected SC-Ura-Trp plates and <br /> protein physically interact Pol2.                       replica plated SC-Trp plates containing <br />                                                                       Nucleic Acids Research, 2012, Vol. 40, No. 22 11547 <br />  <br />  <br /> 5-FOA identify transformants unable prolifer-         A GST tag inserted 50 -end POL2 in <br /> ate absence plasmid wild-type DPB2.     plasmid pJL1, 50 -end DPB2 plasmid <br /> Plasmids transformants isolated         pJL9 50 -end dpb2-201 plasmid <br /> mutated dpb2 alleles subcloned untreated          pJL9-201. The fusion proteins GST-Pol2, GST-Dpb2 <br /> pRS314 vector. Lethality con&#239;&#172;?rmed              GST&#226;&#8364;&#8220;dpb2-201 over-expressed PY116 cells <br /> shuf&#239;&#172;&#8218;ing procedure. The vectors lethal dpb2 alleles       remaining wild-type subunits as <br /> were puri&#239;&#172;?ed sequenced.                                    described above. The harvested cells lysed, fraction- <br />                                                                ated ammonium sulfate precipitation, loaded and <br /> Construction diploid strain heterozygous dpb2-200     eluted phosphocellulose column, loaded a <br /> and tetrad analysis                                            glutathione sepharose column described previously <br /> Saccharomyces cerevisiae strains E134 w1588-4c             (38). The glutathione sepharose column washed with <br /> (MATa leu2-3,112 trp1-1 can1-100 ura3-1 ade2-1 his3-           20 column volumes Buffer B (25 mM Hepes-NaOH <br /> 11,15 RAD5+) (37) crossed create wild-type           pH 7.6, 10% glycerol, 1 mM ethylenediaminetetraacetic <br /> diploid strain homozygous DPB2 (DPB2/DPB2). To             acid (EDTA), 0.5 mM ethyleneglycoltetraaceticacetic <br /> obtain dpb2-200 heterozygote (dpb2-200/DPB2),          (EGTA), 0.005% Nonidet P-40, 1 mM dithiothreitol, <br /> dpb2-200 allele labeled hygromycin resistance       5 mM pepstatin A, 5 mM leupeptin 5 mM NaHSO3). <br /> gene, hph, integrated genome diploid       The bound protein eluted 20 mM glutathione <br /> strain homologous recombination. To this, DNA          (pH 8.0) Buffer B. The eluted protein colloidal <br /> fragment containing dpb2-200 cut pRS314&#226;&#8364;&#8220;          Coomassie-stained electrophoresis 10% <br /> dpb2-200 plasmid KpnI PstI endonucleases,             SDS&#226;&#8364;&#8220;PAGE (16). <br /> blunt ended T4 DNA polymerase ligated a <br /> SmaI-treated pAG32 vector. The resulting pAG32&#226;&#8364;&#8220;                DNA polymerase assays <br /> dpb2-200 plasmid used template PCR                 The speci&#239;&#172;?c activity Pol e measured previously <br /> amplify dpb2-200 fragment &#239;&#172;&#8218;anked hph gene.          described (16) primer extension assay also <br /> The ampli&#239;&#172;?cation performed primers DPB2-6             performed previously described (24). For primer <br /> (50 -AAAATTAAATGATGACCCACCG-30 ) p5                        extension assay, [g-32P]ATP-labeled 50mer oligonucleo- <br /> (50 -CAGAGTACCAACACTACCGTGCTTGTTTAAG                           tide annealed pBluescript II SK(+) <br /> TGTTCTTATTTTGATGCATTATTTAGCATAGGCC                             single-stranded DNA ratio 1:1.5. For DNA <br /> ACTAGTGGATCTG-30 ) generate PCR fragment                  synthesis processivity assay, substrate (14 nM) was <br /> containing dpb2-200 allele          incubated four-subunit wild-type Pol e the <br /> included mutation, hph gene, 60 bp DPB2         Pol2/Dpb3/Dpb4 complex indicated correspond- <br /> &#239;&#172;&#8218;anking sequences (Supplementary Figure S2). This PCR          ing &#239;&#172;?gures. Reactions performed mixture of <br /> product used transform diploid DPB2/DPB2            40 mM Tris&#226;&#8364;&#8220;HCl pH 7.8, 1 mM dithiothreitol, 0.2 mg/ml <br /> strain. Transformants selected YPD+hygro-              bovine serum albumin, 8 mM MgCl2, 125 mM NaAc and <br /> mycin B media, correct insertion dpb2-200           100 mM dNTPs. Each reaction incubated 30  C 2 <br /> into chromosome veri&#239;&#172;?ed PCR using               8 min terminated addition 15 ml stop <br /> primers described Supplementary Figure S2            solution (80% formamide 10 mM EDTA). The repli- <br /> sequencing.                                                    cation products denatured 95  C 4 min and <br />    The DPB2 homozygote dpb2-200 heterozygote           cooled ice electrophoresis 8% <br /> strains grown solid YPD media 2 days        denaturing polyacrylamide gel. The gel dried and <br /> they patched sporulation media incubated         placed phosphoimager screen (Fuji) intensity <br /> for 5 days 30  C. After incubation, tetrads dis-   bands corresponding different replication <br /> sected YPD Singer MSM System 300                     products quanti&#239;&#172;?ed Typhoon 9400 phosphoi- <br /> micromanipulator spores incubated             mager ImageQuant software package (GE <br /> 2 days 30  C.                                               Healthcare). The reaction conditions empirically <br />                                                                determined meet single-hit kinetics conditions in <br /> Co-expression puri&#239;&#172;?cation Pol2, mutated Dpb2,           polymerase molecule re-associates a <br /> Dpb3 Dpb4                                                  previously extended primer (39). <br /> The galactose-inducible over-expression vectors pJL1              Holoenzyme assays performed standard 15 ml <br /> (Pol2), pJL9-201 (Dpb2-201) pJL5 (Dpb3                 reaction contained 40 mM Tris&#226;&#8364;&#8220;HCl (pH 7.8), 1 mM <br /> Dpb4) used transform yeast strain pY116 cells.         dithiothreitol, 0.2 mg/ml bovine serum albumin, 8 mM <br /> The resulting Pol e complex puri&#239;&#172;?ed previously          MgAc2, 125 mM NaAc, 100 mM dGTP, dATP <br /> described (23) comparable yield wild-type Pol        dTTP, 50 mM dCTP, 1.5 mCi [a-32P] dCTP (Perkin <br /> e lacking Dpb2 subunit. The stoichiometry           Elmer), 70 fmol single-primed pBluescript II SK(+), <br /> Pol2:Dpb3:Dpb4 estimated 1:1.1:1.4             143 fmol DNA polymerase, 10.5 pmol RPA, 80 fmol <br /> measuring intensities Coomassie-stained             RFC 1.15 pmol PCNA. The reactions incubated <br /> protein bands 10% sodium dodecyl sulfate-polyacry-        30  C times indicated &#239;&#172;?gures. The reactions <br /> lamide gel electrophoresis (SDS&#226;&#8364;&#8220;PAGE). Overall,             terminated adding 6 ml stop solution (54 mM <br /> found biochemical properties Pol2/Dpb3/             EDTA, 1% SDS, 0.135% Orange G, 0.027% xylene <br /> Dpb4 complex comparable wild-type Pol e.                    cyanol FF, 54% glycerol 9 mM Tris&#226;&#8364;&#8220;HCl pH 7.6). <br />  11548 Nucleic Acids Research, 2012, Vol. 40, No. 22 <br />  <br />  <br /> The samples loaded 0.8% alkaline agarose gel       Table 1. Alterations amino acid sequence Dpb2 muta- <br /> that contained 30 mM NaOH 2 mM EDTA. The gels              tions described study <br /> were run 30 V 16 h neutralized 0.1 M Tris&#226;&#8364;&#8220;     Dpb2        431-PPTLLIWQGS-440 <br /> HCl pH 7.6. The gels dried, placed                   Dpb2-198    431-PPTLLI * QGS-440 <br /> phosphoimager screen (Fuji) scanned a <br /> Typhoon 9400 phosphoimager (GE Healthcare).                    Dpb2        581-DIVPIDQLVK EPDQLPQKVQ ETRKLVKTIL-610 <br />                                                                Dpb2-199    581-DIVPIDQLVK ESDQLP* KV * ETRKLVKTIL-610 <br /> Temperature sensitivity assay strains containing <br />                                                                Dpb2        671-RRARYMEYVP SSKKTIQEEI YI*-692 <br /> dpb2-200 dpb2-201 alleles                                  Dpb2-200    671-RRARYMEYVS SSKKTI * EEI YI*-692 <br /> E134-dpb2D cells contained pRS316-DPB2 <br />                                                                Dpb2        671-RRARYMEYVP SSKKTIQEEI YI*-692 <br /> plasmid transformed pRS314 vectors              Dpb2-201    671-RRARYMEYVS SSKKTI **-686 <br /> carried dpb2-200, dpb2-201 DPB2 alleles, and <br /> selected SC-Ura-Trp plates. The different             Amino acid substitutions stop codons (*) bold. <br /> transformants patched YPD plates grown <br /> overnight 30  C followed replica plating SC-        treated hydroxylamine, subcloned the <br /> Ura-Trp SC 5-FOA plates. The cells grown           dpb2-200 allele untreated pRS314 vector to <br /> 16, 23, 30 37  C.                                          con&#239;&#172;?rm lethal <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>phenotype</span> add- <br />                                                                itional mutations vector backbone. We tested <br />                                                                dpb2-200 allele temperature sensitive and <br /> RESULTS                                                        raising lowering temperature no <br /> Isolation lethal mutations DPB2                          effect viability (Figure 1A). Finally, established a <br />                                                                diploid strain carried dpb2-200 allele one <br /> We isolated lethal mutations DPB2 gene examine <br />                                                                wild-type DPB2 allele. Tetrad analysis con&#239;&#172;?rmed <br /> the role Dpb2 vivo function Pol e <br />                                                                dpb2-200 lethal mutation two <br /> complex. DPB2 essential gene, screen for <br />                                                                viable spores seven asci examined <br /> lethal mutations constructed separate low copy <br />                                                                (Figure 1B). This integration effect caused by <br /> number plasmids carried wild-type DPB2 gene <br />                                                                hygromycin resistance gene integration of <br /> under control wild-type DPB2 promoter. One <br />                                                                DPB2 using strategy gave viable spores <br /> plasmid carried TRP1 carried URA3 as <br />                                                                ascus (Supplementary Figures S2 S3). <br /> selectable markers, plasmid TRP1 was <br /> mutagenized treatment hydroxylamine. Lethal <br />                                                                Lack interaction Dpb2-201 Pol2, <br /> mutations DPB2 gene identi&#239;&#172;?ed plasmid <br />                                                                Dpb3 Dpb4 <br /> shuf&#239;&#172;&#8218;ing wild type mutagenized DPB2 <br /> genes E134-dpb2D yeast strain. That is, trans-          To determine <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> a <br /> formants carrying mutagenized plasmid         non-functional Pol e, created allele called dpb2-201 <br /> DPB2 wild-type plasmid &#239;&#172;?rst selected SC-Ura-           identical dpb2-200 &#239;&#172;?ve codons <br /> Trp plates followed replica plating SC                 Q687stop mutation removed. This done <br />  Trp+5-FOA identify unable proliferate          minimize risk accidentally reading the <br /> the absence plasmid URA3 gene         stop codon translation. We &#239;&#172;?rst con&#239;&#172;?rmed the <br /> the wild-type DPB2 gene. These cells carried potentially     dpb2-201 allele lethal low copy vector pRS314&#226;&#8364;&#8220; <br /> lethal mutation hydroxylamine-treated DPB2 gene.        dpb2-201 (Figure 1A). Next, cloned dpb2-201 allele <br /> In initial screen, 360 colonies replica plated,   over-expression vector control the <br /> 11 unable grow SC-Trp+5-FOA.               galactose-inducible promoter (plasmid pJL9-201) and <br /> These 11 isolates re-streaked inability      co-expressed Dpb2-201 Pol2, Dpb3 and <br /> grow 5-FOA con&#239;&#172;?rmed. The mutagenized                    Dpb4. The Dpb2-201 protein detected western <br /> plasmids recovered yeast cells           blot cell extracts obtained induction galactose <br /> DPB2 gene sequenced directions. We                 (Figure 2A). Next, applied previously developed <br /> identi&#239;&#172;?ed separate plasmids carried mutations       puri&#239;&#172;?cation protocol Pol e depends ion- <br /> in DPB2 resulted truncated protein.         exchange chromatography size-exclusion chromatog- <br /> Allele dpb2-198 carried nonsense codon led         raphy Dpb2-201 did co-purify the <br /> truncation residue 437 allele dpb2-199 carried          subunits (Figure 2B). This suggested Dpb2-201 <br /> three mutations (P592S, Q597stop Q600stop)                 unable interact Pol e subunits. <br /> leading truncation residue 597 (Table 1). One          To test hypothesis, positioned GST tag <br /> the alleles, dpb2-200, carried mutations located near      N-terminus Pol2 over-expression system <br /> the 30 -end DPB2 gene. These mutations              described asked over-expressed Dpb2 <br /> changed proline serine amino acid 680 (P680S)          Dpb2-201 co-purify GST-Pol2 a <br /> and glutamine nonsense codon position 687            glutathione-sepharose column. Only Dpb2, Dpb3 and <br /> (Q687stop). The dpb2-200 allele interesting       Dpb4 co-puri&#239;&#172;?ed Pol e af&#239;&#172;?nity column <br /> allele mutations located 13        (Figure 3, lanes 1 2), supporting previous result <br /> residues C-terminus resulting protein    suggested Dpb2-201 lost ability to <br /> truncation residues. As entire vector     interact Pol2 (Figure 2B). However, western blots <br />                                                                                    Nucleic Acids Research, 2012, Vol. 40, No. 22 11549 <br />  <br />  <br />  <br />  <br /> Figure 1. The isolated dpb2-200 dpb-201 alleles support <br /> growth. (A) The alleles, dpb2-200 dpb2-201, tested for <br /> lethality temperature sensitivity plasmid shuf&#239;&#172;&#8218;ing experiments in <br /> E134-dpb2D cells. 5-FOA used selection wild-type <br /> URA3-containing pRS316-DPB2 plasmid. The pRS314-DPB2 plasmid <br /> with TRP1 marker wild-type control grow          Figure 2. Co-expression Dpb2-201 Pol2, Dpb3 and <br /> 5-FOA. Four clones transformant presented. (B) Tetrad          Dpb4. (A) Western-blot analysis pY116 cells (16) over-expressed <br /> analysis DPB2/DPB2 homozygote DPB2/dpb2-200                     Dpb2-201 Dpb2 presence galactose. A rabbit polyclonal <br /> heterozygote.                                                              antibody used recognizes Dpb2. Two isolates carrying either <br />                                                                            over-expression plasmids dpb2-201 DPB2 analyzed. An <br />                                                                            equal cells loaded lane boiling a <br /> suggested Dpb2-201 protein abundant                      loading buffer SDS, b-mercaptoethanol, bromophenol blue and <br />                                                                            glycerol. The &#239;&#172;?rst lanes (1&#226;&#8364;&#8220;4) cells harvested induction <br /> than wild-type Dpb2 over-expressed yeast                           galactose lanes (5&#226;&#8364;&#8220;8) cells harvested after <br /> (Figure 2A). This potentially lead substoichio-                 induction galactose. Lanes labeled A, B, C D same <br /> metric Dpb2-201 Pol e preparations                    cultures induction, respectively. Fifteen nanograms, <br /> only impression Dpb2-201 lost ability                30 ng 60 ng puri&#239;&#172;?ed Pol e loaded indicated last <br />                                                                            lanes (9&#226;&#8364;&#8220;11). (B) The puri&#239;&#172;?ed Pol e complex missing the <br /> to interact Pol2. Thus, positioned GST tag                       Dpb2 subunit puri&#239;&#172;?ed cells galactose-induced <br /> at N-terminus Dpb2 Dpb2-201 determine                        over-expression Dpb2-201 (Figure 2A). The estimated subunit stoi- <br /> whether co-purify Pol2                                chiometry puri&#239;&#172;?ed Pol2/Dpb3/Dpb4 complex 1:1.1:1.4 based <br /> glutathione-sepharose column. Equal amounts                             plot density Coomassie-stained bands. <br /> GST-Dpb2 GST&#226;&#8364;&#8220;Dpb2-201 loaded each <br /> lane. The Pol2 co-purifying GST&#226;&#8364;&#8220; <br /> Dpb2-201 signi&#239;&#172;?cantly compared                           measured speci&#239;&#172;?c activity puri&#239;&#172;?ed Pol2/Dpb3/ <br /> GST-Dpb2 puri&#239;&#172;?cation (Figure 3, lanes 3 4).                            Dpb4 complex oligo(dT)-poly(dA) substrate. <br /> Altogether, results support model Dpb2                    We speci&#239;&#172;?c activity Pol2/Dpb3/Dpb4 <br /> depends C-terminus interact Pol2,                   complex 2-fold higher the <br /> small Pol2 co-purifying GST&#226;&#8364;&#8220;Dpb2-201                        wild-type Pol e (8860 U/mg 4551 U/mg, respectively) <br /> suggests exist second, weak interaction                   suggesting catalytic activity complex was <br /> surface. Furthermore, observed slight shift                    negatively affected. Processivity important par- <br /> stoichiometry Dpb3 Dpb4 absence                           ameter replicative polymerases, previous studies <br /> presence Dpb2 GST-Dpb2.                                              shown Pol e high intrinsic processivity <br />                                                                            non-essential small subunits Dpb3 and <br /> Biochemical characterization Pol2/Dpb3/Dpb4                         Dpb4 important processivity (24). Dpb2 was <br /> complex                                                                    previously proposed located structural domain <br /> We used assays explore enzymatic properties                   interacted nascent double-stranded DNA <br /> of puri&#239;&#172;?ed Pol2/Dpb3/Dpb4 complex. The &#239;&#172;?rst assay                      stabilized interaction template (23). <br />  11550 Nucleic Acids Research, 2012, Vol. 40, No. 22 <br />  <br />  <br />                                                                         Pol2 interacts Dpb2 (41&#226;&#8364;&#8220;44). The results X. <br />                                                                         laevis, S. cerevisiae H. sapiens suggested the <br />                                                                         Dpb3/Dpb4 subunits Dpb2 interact separate <br />                                                                         domains Pol2. Our earlier isolation homogenous <br />                                                                         Pol2/Dpb2 complex (24) current results a <br />                                                                         Pol2/Dpb3/Dpb4 complex agreement this <br />                                                                         model. <br />                                                                            Pol e S. cerevisiae &#239;&#172;?rst puri&#239;&#172;?ed 132 kDa <br />                                                                         proteolytic fragment, later referred the <br />                                                                         145 kDa fragment (45). This form Pol e missing all <br />                                                                         accessory subunits C-terminal domain where <br />                                                                         subunits normally interact Pol2 absent. The <br />                                                                         145 kDa proteolytic fragment potent polymerase <br />                                                                         activity and, fact, reported similar <br />                                                                         biochemical properties four-subunit Pol e (46&#226;&#8364;&#8220;48). <br />                                                                         The differences 145 kDa fragment Pol2 <br />                                                                         Pol e complex limited processivity, salt <br />                                                                         sensitivity af&#239;&#172;?nity single-stranded DNA. Recent <br />                                                                         studies focused effects removing Dpb2, <br /> Figure 3. Pull-down experiment GST-Pol2, GST-Dpb2 GST&#226;&#8364;&#8220; <br /> Dpb2-201. The positions subunit Pol e indicated   Dpb3 Dpb4 subunits. <br /> left. The protein loaded lane titrated      The B subunits Pol d (Pol31), Pol (Pol12) Pol e <br /> such equal GST-Pol2 loaded lanes 1 2,      (Dpb2) share conserved motifs, essential yeast <br /> and equal amounts GST-Dpb2 GST&#226;&#8364;&#8220;Dpb2-201 loaded           (3). However, roles remain elusive. Previous studies <br /> lanes 3 4. The gel colloidal Coomassie-stained (16). <br />                                                                         shown Pol31 forms bridge Pol3 and <br />                                                                         Pol32 improve ef&#239;&#172;?ciency loading Pol d the <br />                                                                         PCNA&#226;&#8364;&#8220;primer&#226;&#8364;&#8220;template          ternary complex (12,49). <br /> To address processivity affected Pol2/ <br />                                                                         However, role does appear essential <br /> Dpb3/Dpb4 complex, performed previously <br />                                                                         POL32 shown non-essential <br /> described experiments performed the <br />                                                                         gene (11). Recently, shown catalytic <br /> Pol2/Dpb2 complex wild-type Pol e (24,40). The <br />                                                                         subunits Pol z (Rev3) Pol d (Pol3) carry an <br /> reaction conditions estimated 1:10 ratio polymer- <br />                                                                         Fe&#226;&#8364;&#8220;S cluster C-termini (50) Pol31 and <br /> ase primer measured termination <br />                                                                         Pol32 interact Pol d Pol z Fe&#226;&#8364;&#8220; <br /> probability position constant 2 8 <br />                                                                         S cluster (15,50). DPB2 essential gene S. <br /> min &lt;20% primer extended. Thus, our <br />                                                                         pombe S. cerevisiae (26,32). However, puri&#239;&#172;?ed Pol2/ <br /> experiment met criteria single-hit conditions (39). <br />                                                                         Dpb3/Dpb4 complexes H. sapiens reported <br /> We high processivity Pol e the <br />                                                                         signi&#239;&#172;?cant changes biochemical <br /> primer-extension assays affected Dpb2 <br />                                                                         properties compared wild-type Pol e (42). Our <br /> was missing Pol2/Dpb3/Dpb4 complex (Figure <br />                                                                         characterization Pol2/Dpb3/Dpb4 complex con&#239;&#172;?rms <br /> 4). There is, however, strong pause site limits the <br />                                                                         loss Dpb2 does negative in&#239;&#172;&#8218;uence on <br /> analysis &#239;&#172;?rst 64 nucleotides. This substan- <br />                                                                         Pol e polymerase activity described biochemical <br /> tial <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> comparison extension Pol d              assays results loss protein stability. <br /> processivity &#239;&#172;?ve seven nucleotides              Moreover, absence Dpb2 did affect func- <br /> absence processivity clamp (40). Next, asked                  tional interaction Pol e PCNA. Thus, es- <br /> whether loss Dpb2 affected interaction                    sential function Dpb2 appears uncoupled de <br /> Pol e processivity clamp, PCNA, stimulates                novo DNA synthesis. <br /> Pol e holoenzyme assays. We tested functional inter-                Previous studies suggested Pol e contributes <br /> action Pol e PCNA single-primed                        processes addition synthesis new <br /> circular single-stranded template loss               DNA. There strong evidence Pol e plays role in <br /> of Dpb2 did affect PCNA-dependent stimulation                   initiation DNA replication. In S. pombe, a <br /> (Figure 5). Thus, Dpb2 appears essential              temperature-sensitive dpb2 mutant be <br /> interaction Pol e double-stranded DNA,                  growth-arrested early S-phase non-permissive <br /> achieving high processivity, loading Pol e                 temperature (32). In addition, Dpb2 be <br /> 30 -end primer, interaction PCNA.                associated replication origin chromatin <br />                                                                         precipitation assay (32). In S. cerevisiae, series experi- <br />                                                                         ments suggested Pol e plays role the <br /> DISCUSSION <br />                                                                         initiation DNA replication (51), co-immunopre- <br /> In study, identi&#239;&#172;?ed essential motif                  cipitation experiments suggested Dpb2 partici- <br /> S. cerevisiae Dpb2 required formation                pates pre-loading complex (pre-LC) consists of <br /> the four-subunit Pol e. Earlier studies Xenopus laevis,              Pol e, GINS, Dpb11 Sld2 (33). In complex, both <br /> S. cerevisiae, Schizosaccharomyces pombe Homo                       Sld2 Dpb2 phosphorylated CDK. The phos- <br /> sapiens shown essential C-terminal domain                 phorylation Sld2 shown important for <br />                                                                                     Nucleic Acids Research, 2012, Vol. 40, No. 22 11551 <br />  <br />  <br />  <br />  <br /> Figure 4. Primer extension assays four-subunit Pol e Pol2/Dpb3/Dpb4 complex. The experiments performed different ratios <br /> of enzyme primer template ([E]:[PT]). Results shown 2 8 min, indicated. The termination probability measured each <br /> product described previously (39). Sequencing reactions primer template used primer-extension assay loaded the <br /> right size marker. <br />  <br />  <br /> regulating start DNA replication, lack                   Pol d lagging strand PIP box-independent <br /> Dpb2 phosphorylation associated meas-                     mechanism targets Pol e leading strand (40). <br /> urable vivo effects. The role Pol e unclear               The fact Dpb2 physically interact GINS, a <br /> pre-LC, positions Pol e close assembly                CMG complex, suggests Dpb2 con- <br /> CMG helicase complex (Cdc45/Mcm2-7/GINS) (52).                             tribute targeting Pol e primarily leading strand in <br />    Dpb2 essential function progres-                     DNA synthesis. <br /> sion replication fork. There substantial evidence                   In previous study, series temperature-sensitive <br /> for asymmetric replication fork Pol e pos-                  dpb2 alleles isolated. A de&#239;&#172;?ciency the <br /> itioned leading strand Pol d lagging                     alleles, dpb2-1, allowed initiation DNA replica- <br /> strand (7,8). This raises question asym-                   tion progression replication fork per- <br /> metry established maintained. We previously                    turbed (26). Our current results suggest delayed <br /> shown processivity clamp, PCNA, unlikely                   progression S-phase unlikely to defect <br /> candidate regulating division labor Pol                catalytic activity Pol e. A caveat tempera- <br /> e Pol d loaded PCNA clamp                         ture-sensitive alleles, however, fail properly <br /> different mechanisms (40). In contrast Pol d, Pol e                     fold non-permissive temperature, possible <br /> lacks functional PIP box proposed                      mis-folded Dpb2 in&#239;&#172;&#8218;uence polymerase <br /> might PIP box-dependent mechanism targets                        activity Pol e protein interactions vivo. <br />  11552 Nucleic Acids Research, 2012, Vol. 40, No. 22 <br />  <br />  <br />                                                                        FUNDING <br />                                                                        Swedish Research Council (to E.J.); Cancerfonden <br />                                                                        (to E.J.); Sma&#194;&#168;rtafonden (to K.V. E.J.); Kempe foun- <br />                                                                        dation (to I.I., K.V. E.J.) Umea&#203;&#353; University <br />                                                                        (to I.I. E.J.). Funding open access charge: <br />                                                                        The Swedish Research Council. <br />                                                                        Con&#239;&#172;&#8218;ict statement. None declared. <br />  <br />  <br />                                                                        REFERENCES <br />                                                                         1. Burgers,P.M. (2009) Polymerase dynamics eukaryotic DNA <br />                                                                            replication fork. J. Biol. Chem., 284, 4041&#226;&#8364;&#8220;4045. <br />                                                                         2. Kunkel,T.A. Burgers,P.M. (2008) Dividing workload a <br />                                                                            eukaryotic replication fork. Trends Cell. Biol., 18, 521&#226;&#8364;&#8220;527. <br />                                                                         3. Johansson,E. Macneill,S.A. (2010) The eukaryotic replicative <br />                                                                            DNA polymerases shape. Trends Biochem. Sci., 35, 339&#226;&#8364;&#8220;347. <br />                                                                         4. Pavlov,Y.I. Shcherbakova,P.V. (2010) DNA polymerases at <br />                                                                            eukaryotic fork-20 years later. Mutat. Res., 685, 45&#226;&#8364;&#8220;53. <br />                                                                         5. Hubscher,U., Maga,G. Spadari,S. (2002) Eukaryotic DNA <br />                                                                            polymerases. Annu. Rev. Biochem., 71, 133&#226;&#8364;&#8220;163. <br /> Figure 5. Holoenzyme assay four-subunit Pol e Pol2/        6. Pellicioli,A., Lucca,C., Liberi,G., Marini,F., Lopes,M., Plevani,P., <br /> Dpb3/Dpb4 complex. A single-primed pBluescript II SK(+) template           Romano,A., Di Fiore,P.P. Foiani,M. (1999) Activation of <br /> was replicated indicated Pol e complex presence RPA       Rad53 kinase response DNA damage effect in <br /> and PCNA presence (+) absence ( ) RFC. Samples            modulating phosphorylation lagging strand DNA <br /> were collected 1, 2, 4 8 min. <br />                                                                            polymerase. EMBO J., 18, 6561&#226;&#8364;&#8220;6572. <br />                                                                         7. Pursell,Z.F., Isoz,I., Lundstrom,E.B., Johansson,E. and <br />                                                                            Kunkel,T.A. (2007) Yeast DNA polymerase epsilon participates <br />                                                                            leading-strand DNA replication. Science, 317, 127&#226;&#8364;&#8220;130. <br /> Our isolated mutation, dpb2-200, temperature                    8. Nick McElhinny,S.A., Gordenin,D.A., Stith,C.M., Burgers,P.M. <br /> sensitive, de&#239;&#172;?ciency limited local motif                  Kunkel,T.A. (2008) Division labor eukaryotic <br /> at C-terminus Dpb2. This makes de&#239;&#172;?ciency                        replication fork. Mol. Cell., 30, 137&#226;&#8364;&#8220;144. <br />                                                                         9. Larrea,A.A., Lujan,S.A., Nick McElhinny,S.A., <br /> more likely affect interaction Pol2                   Mieczkowski,P.A., Resnick,M.A., Gordenin,D.A. and <br /> not interactions unknown proteins. From                    Kunkel,T.A. (2010) Genome-wide model normal <br /> perspective, essential function Pol2 C-terminus                 eukaryotic DNA replication fork. Proc. Natl Acad. Sci. USA, <br /> might linked physical interaction Dpb2. It                    107, 17674&#226;&#8364;&#8220;17679. <br />                                                                        10. Miyabe,I., Kunkel,T.A. Carr,A.M. (2011) The major roles of <br /> was shown previously ts-dpb2 alleles associated                  DNA polymerases epsilon delta eukaryotic replication <br /> with signi&#239;&#172;?cantly increased mutation rate (27,29). It                 fork evolutionarily conserved. PLoS Genet., 7, e1002407. <br /> not clear, however, Pol2/Dpb3/Dpb4                  11. Gerik,K.J., Li,X., Pautz,A. Burgers,P.M. (1998) <br /> complexes, Pol2/ts-Dpb2/Dpb3/Dpb4 complexes                         Characterization small subunits Saccharomyces <br />                                                                            cerevisiae DNA polymerase delta. J. Biol. Chem., 273, <br /> themselves exhibit reduced &#239;&#172;?delity DNA synthesis                    19747&#226;&#8364;&#8220;19755. <br /> compared wild-type four-subunit Pol e. It                 12. Johansson,E., Garg,P. Burgers,P.M. (2004) The Pol32 subunit <br /> recently shown cases, increased mutation                  DNA polymerase delta contains separable domains for <br /> rates observed ts&#226;&#8364;&#8220;dpb2-containing strains mani-                  processive replication proliferating cell nuclear antigen <br />                                                                            (PCNA) binding. J. Biol. Chem., 279, 1907&#226;&#8364;&#8220;1915. <br /> festation perturbed replication fork (28).                        13. Huang,M.E., Rio,A.G., Galibert,M.D. Galibert,F. (2002) <br />    The biological role Dpb2 remains uncertain.                          Pol32, subunit Saccharomyces cerevisiae DNA polymerase <br /> However, shown C-terminus Dpb2                         delta, suppresses genomic deletions involved the <br /> is required interaction Pol2                    mutagenic bypass pathway. Genetics, 160, 1409&#226;&#8364;&#8220;1422. <br />                                                                        14. Lydeard,J.R., Jain,S., Yamaguchi,M. Haber,J.E. (2007) <br /> same motif essential cell viability. We                   Break-induced replication telomerase-independent telomere <br /> con&#239;&#172;?rmed previous reports shown Dpb2                        maintenance require Pol32. Nature, 448, 820&#226;&#8364;&#8220;823. <br /> is required support Pol e polymerase activity                15. Baranovskiy,A.G., Lada,A.G., Siebler,H.M., Zhang,Y., <br /> the presence accessory proteins PCNA, RFC                       Pavlov,Y.I. Tahirov,T.H. (2012) DNA polymerase delta and <br /> RPA. Altogether, accumulated evidence suggests                    zeta switch sharing accessory subunits DNA polymerase <br />                                                                            delta. J. Biol. Chem., 287, 17281&#226;&#8364;&#8220;17287. <br /> Dpb2 plays essential structural role remains             16. Chilkova,O., Jonsson,B.H. Johansson,E. (2003) The <br /> elucidated. Studies way determine                         quaternary structure DNA polymerase epsilon from <br /> function Dpb2 help identi&#239;&#172;?ed                            Saccharomyces cerevisiae. J. Biol. Chem., 278, 14082&#226;&#8364;&#8220;14086. <br /> C-terminal motif.                                                      17. Morrison,A., Araki,H., Clark,A.B., Hamatake,R.K. and <br />                                                                            Sugino,A. (1990) A essential DNA polymerase S. <br />                                                                            cerevisiae. Cell, 62, 1143&#226;&#8364;&#8220;1151. <br />                                                                        18. Kesti,T., Flick,K., Keranen,S., Syvaoja,J.E. Wittenberg,C. <br />                                                                            (1999) DNA polymerase epsilon catalytic domains dispensable <br /> SUPPLEMENTARY DATA                                                         DNA replication, DNA repair, cell viability. Mol. Cell., <br />                                                                            3, 679&#226;&#8364;&#8220;685. <br /> Supplementary Data available NAR Online:                        19. Dua,R., Levy,D.L. Campbell,J.L. (1999) Analysis the <br /> Supplementary Figures 1&#226;&#8364;&#8220;3.                                                 essential functions C-terminal protein/protein interaction <br />                                                                                  Nucleic Acids Research, 2012, Vol. 40, No. 22 11553 <br />  <br />  <br />     domain Saccharomyces cerevisiae pol epsilon                   Additional modules versatile economical PCR-based gene <br />     unexpected ability support growth absence DNA           deletion modi&#239;&#172;?cation Saccharomyces cerevisiae. Yeast, 14, <br />     polymerase domain. J. Biol. Chem., 274, 22283&#226;&#8364;&#8220;22288.                     953&#226;&#8364;&#8220;961. <br /> 20. Araki,H., Hamatake,R.K., Morrison,A., Johnson,A.L.,                  36. Strathern,J.N. Higgins,D.R. (1991) Recovery plasmids <br />     Johnston,L.H. Sugino,A. (1991) Cloning DPB3, gene                yeast Escherichia coli: shuttle vectors. Methods <br />     encoding subunit DNA polymerase II                       Enzymol., 194, 319&#226;&#8364;&#8220;329. <br />     Saccharomyces cerevisiae. Nucleic Acids Res., 19, 4867&#226;&#8364;&#8220;4872.         37. Thomas,B.J. Rothstein,R. (1989) Elevated recombination <br /> 21. Ohya,T., Maki,S., Kawasaki,Y. Sugino,A. (2000) Structure             rates transcriptionally active DNA. Cell, 56, 619&#226;&#8364;&#8220;630. <br />     function fourth subunit (Dpb4p) DNA polymerase         38. Sabouri,N. Johansson,E. (2009) Translesion synthesis of <br />     epsilon Saccharomyces cerevisiae. Nucleic Acids Res., 28,             abasic sites yeast DNA polymerase epsilon. J. Biol. Chem., <br />     3846&#226;&#8364;&#8220;3852.                                                               284, 31555&#226;&#8364;&#8220;31563. <br /> 22. Tsubota,T., Maki,S., Kubota,H., Sugino,A. Maki,H. (2003)         39. Kokoska,R.J., McCulloch,S.D. Kunkel,T.A. (2003) The <br />     Double-stranded DNA binding properties Saccharomyces                  ef&#239;&#172;?ciency speci&#239;&#172;?city apurinic/apyrimidinic site bypass by <br />     cerevisiae DNA polymerase epsilon Dpb3p-Dpb4p                 human DNA polymerase eta Sulfolobus solfataricus Dpo4. <br />     subassembly. Genes Cells, 8, 873&#226;&#8364;&#8220;888.                                    J. Biol. Chem., 278, 50537&#226;&#8364;&#8220;50545. <br /> 23. Asturias,F.J., Cheung,I.K., Sabouri,N., Chilkova,O., Wepplo,D.       40. Chilkova,O., Stenlund,P., Isoz,I., Stith,C.M., Grabowski,P., <br />     Johansson,E. (2006) Structure Saccharomyces cerevisiae            Lundstrom,E.B., Burgers,P.M. Johansson,E. (2007) The <br />     DNA polymerase epsilon cryo-electron microscopy. Nat.                 eukaryotic leading lagging strand DNA polymerases are <br />     Struct. Mol. Biol., 13, 35&#226;&#8364;&#8220;43.                                           loaded primer-ends separate mechanisms have <br /> 24. Aksenova,A., Volkov,K., Maceluch,J., Pursell,Z.F., Rogozin,I.B.,         comparable processivity presence PCNA. Nucleic Acids <br />     Kunkel,T.A., Pavlov,Y.I. Johansson,E. (2010) Mismatch                Res., 35, 6588&#226;&#8364;&#8220;6597. <br />     repair-independent increase spontaneous mutagenesis yeast      41. Dua,R., Edwards,S., Levy,D.L. Campbell,J.L. (2000) Subunit <br />     lacking non-essential subunits DNA polymerase epsilon. PLoS           interactions Saccharomyces cerevisiae DNA polymerase <br />     Genet., 6, e1001209.                                                     epsilon (pol epsilon) complex. Demonstration dimeric pol <br /> 25. Northam,M.R., Garg,P., Baitin,D.M., Burgers,P.M.                     epsilon. J. Biol. Chem., 275, 28816&#226;&#8364;&#8220;28825. <br />     Shcherbakova,P.V. (2006) A novel function DNA polymerase          42. Bermudez,V.P., Farina,A., Raghavan,V., Tappin,I. Hurwitz,J. <br />     zeta regulated PCNA. EMBO J., 25, 4316&#226;&#8364;&#8220;4325.                          (2011) Studies human DNA polymerase epsilon GINS <br /> 26. Araki,H., Hamatake,R.K., Johnston,L.H. Sugino,A. (1991)              complex role DNA replication. J. Biol. Chem., 286, <br />     DPB2, gene encoding DNA polymerase II subunit B,                  28963&#226;&#8364;&#8220;28977. <br />     required chromosome replication Saccharomyces cerevisiae.     43. Shikata,K., Sasa-Masuda,T., Okuno,Y., Waga,S. Sugino,A. <br />     Proc. Natl Acad. Sci. USA, 88, 4601&#226;&#8364;&#8220;4605.                                (2006) The DNA polymerase activity Pol epsilon holoenzyme <br /> 27. Jaszczur,M., Flis,K., Rudzka,J., Kraszewska,J., Budd,M.E.,               required rapid ef&#239;&#172;?cient chromosomal DNA replication <br />     Polaczek,P., Campbell,J.L., Jonczyk,P. Fijalkowska,I.J. <br /> </body></html>