<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Unique GGA Clathrin Adaptor Drosophila <br /> melanogaster Is Not Essential <br /> Shan Luan1, Anne M. Ilvarsonn1, Joel C. Eissenberg2* <br /> 1 Department Biology, Macelwane Hall, Saint Louis University, St. Louis, Missouri, United States America, 2 Edward A. Doisy Department Biochemistry and <br /> Molecular Biology, Saint Louis, University School Medicine, St. Louis, Missouri, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The Golgi-localized, c-ear-containing, ARF binding proteins (GGAs) highly conserved family monomeric clathrin <br />      adaptor proteins implicated clathrin-mediated protein sorting trans-Golgi network endosomes. GGA <br />      RNAi knockdowns Drosophila resulted conflicting data concerning Drosophila GGA (dGGA) is <br />      essential. The goal study define null <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> unique Drosophila GGA. We two <br />      independently derived dGGA mutations. Neither allele expresses detectable dGGA protein. Homozygous hemizygous <br />      flies allele viable fertile. In contrast previous report using RNAi knockdown, GGA mutant flies no <br />      evidence age-dependent retinal degeneration cathepsin missorting. Our results demonstrate the <br />      previous RNAi knockdown phenotypes result off-target effects. However, GGA null flies hypersensitive <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      dietary chloroquine starvation, implicating GGA lysosomal function autophagy. <br />  <br />   Citation: Luan S, Ilvarsonn AM, Eissenberg JC (2012) The Unique GGA Clathrin Adaptor Drosophila melanogaster Is Not Essential. PLoS ONE 7(9): e45163. <br />   doi:10.1371/journal.pone.0045163 <br />   Editor: Ludger Johannes, Institut Curie, France <br />   Received May 26, 2012; Accepted August 17, 2012; Published September 20, 2012 <br />   Copyright: &#195;&#376; 2012 Luan et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Saint Louis University Presidents Research Fund grant; Saint Louis University Beaumont Fund Award; The funders had <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: eissenjc@slu.edu <br />  <br />  <br />  <br /> Introduction                                                                       GGA essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span> indispensible under <br />                                                                                    certain environmental conditions. <br />    Newly synthesized proteins destined certain organelles                  RNAi knockdown single, GGAs mammalian <br /> secretion trafficked destinations membrane-                        cultured cells showed mild missorting Cathepsin D well <br /> bounded vesicles. Vesicle formation initiated recruitment                memapsin 2 (BACE, membrane associated aspartic protease) <br /> of clathrin, imposes curvature membrane surfaces                        accumulation early endosomes [22&#226;&#8364;&#8220;26], suggesting modest <br /> eventuating vesicles. The formation clathrin-coated vesicles                 contribution mammalian GGAs cellular homeostasis. <br /> (CCVs) depends adaptor proteins (APs), recruited                   However, GGA2 <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>knockout</span> <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> mice results either <br /> GTP-ARFs cellular membrane [1,2]. However, details cargo                     early embryonic neonatal lethality, depending genetic <br /> recognition vesicle formation endocytic secretory                       background [14]. Mice doubly mutant GGA1 and <br /> sorting pathways remain elusive.                                                   GGA3 neonatal lethality, mice singly mutant for <br />    In 2000, groups simultaneously identified Golgi-localized,                 GGA1 GGA3 normal lifespans fertility [27]. <br /> c-ear-containing, ARF binding proteins (GGAs) yeast two-                           Drosophila GGA protein. Compared mamma- <br /> hybrid protein interaction ARF3 [3] searching                          lian GGAs, Drosophila GGA (dGGA) similar any <br /> expressed sequence tag (EST) database genes encoding                           mammalian GGAs, identity ranging 23% <br /> proteins VHS c-adaptin ear domain motifs homologous                        GGA3 27.5% GGA1 similarity ranging between <br /> to APs [4&#226;&#8364;&#8220;7]. The structures GGA family                          41% GGA3 45% GGA2. The difference GGA2 <br /> proteins consist folded domains: (i) N-terminal VHS                   mammalian GGAs absence internal <br /> (Vps27, Hrs, STAM) domain, recognizes acidic                             dileucine motif GGA2, subjected phosphory- <br /> dileucine motif cargo protein [7,8], (ii) GAT (GGA                      lation autoinhibition [28]. In aspect, dGGA similar to <br /> and TOM1) domain, binds GTP-ARF [3,5,9]                               GGA2, lack internal dileucine motif. dGGA <br /> ubiquitin [10&#226;&#8364;&#8220;12] (iii) C-terminal GAE (c-adaptin ear)                     recognizes lysosomal enzyme receptor protein (LERP) [29], <br /> domain, shares homology &#226;&#8364;&#732;ear&#226;&#8364;&#8482; domain c-                        suggesting role protein sorting Golgi. dGGA <br /> subunit AP-1 interacts accessory proteins [6,13&#226;&#8364;&#8220;                 knockdown RNAi Drosophila Schneider S2 cultured cells <br /> 16]. An unstructured hinge region connecting GAT GAE                       results 50% decrease LERP processing [30], suggesting a <br /> domains interacts clathrin [13,17,18].                                        defect LERP sorting lysosomes. In Dmel2 cells, knockdown of <br />    Although phenotypes GGA mutants                               dGGA showed accumulation GFP-tagged LERP larger <br /> established yeast [5,6,18&#226;&#8364;&#8220;20] C. elegans [21],                vesicles total LERP protein level unchanged, also <br /> clear GGA essential animals. Since GGAs                          suggests impaired LERP processing [25]. In vivo analysis dGGA <br /> evolutionally conserved yeast mammals, postulate                        RNAi knockdown showed conflicting results. Knocking down <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                     1                          September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                     Drosophila GGA Is Not Essential <br />  <br />  <br /> dGGA Actin-Gal4 driver showed lethality semi-                    8D9 8E4 9E1, fails complement <br /> lethality groups [31,32]. The semi-lethal dGGA-RNAi3                  lethality GGAP1 chromosome. However, distal <br /> hairpin construct (<span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> Vienna Drosophila RNAi Center, VDRC                  breakpoint duplication close dGGA locus, since <br /> 3269 3270) [32] overlaps lethal dGGA-RNAi2 [32]                 Y chromosome Drosophila heterochromatic somatic <br /> lethal UAS-CG3002/dGGA-RNAi (from National Institute                      tissue, considered possibility wild type dGGA allele <br /> Genetics, Japan) [31]. A completely distinct hairpin construct,              carried translocated fragment silenced by <br /> dGGA-RNAi1, showed mild semi-lethality did knock                heterochromatic position effect. Consistent hypothesis, <br /> down dGGA protein completely [32]. Using VDRC                       different X chromosome fragment carried the <br /> 3269 3270 constructs tubulin-Gal4 driver, Hirst                 Y chromosome, Dp(1;Y)BSC144, does complement lethality of <br /> Carmichael reported dGGA knockdown flies viable                 GGAP1 chromosome. <br /> fertile apparent phenotype [33], albeit express               Since interval X chromosome carried Dp(1;Y)BSC144 <br /> low levels dGGA (less 5%) [33]. In addition, dGGA-RNAi2              relatively large (cytological interval 8A2&#226;&#8364;&#8220;8F9; .500 kb), used <br /> was tested using tubulin-Gal4 driver lethal                  nested set truncated derivatives Dp(1;Y)BSC144 refine the <br /> (Eissenberg unpublished data) Both Actin-5C tubulin                  map position lethal mutation GGAP1 chromosome <br /> housekeeping genes expressed tissues               (Table 1). Surprisingly, derivatives included X <br /> the fly development.                                              chromosome material proximal position 8C12 failed to <br />    Taken together, results GGA knockout knockdown                complement lethality, contain dGGA locus. <br /> in various model organisms yielded conflicting results                  This demonstrated lethal mutation GGAP1 <br /> concerning essential requirement GGA family proteins.                 chromosome separable GGAP1 mutation. <br /> The potential off-target effect associated RNAi lead              To localize lethal mutation GGAP1 <br /> to question dGGA essential flies. Here,            chromosome, tested chromosome BAC transgenes <br /> the generation characterization dGGA null mutant                  defined X chromosome fragments complementation <br /> alleles, generated P-element excision                activity. Together, transgenes span cytological <br /> targeted knockout using homologous recombination. Neither <br />                                                                              interval 8C1&#226;&#8364;&#8220;8C13 (Table 1). Neither Dp(1;3)DC201 nor <br /> mutant makes detectable dGGA protein flies carrying each <br />                                                                              Dp(1;3)DC202 complement lethality, Dp(1;3)DC203 does. <br /> are viable fertile normal laboratory culture conditions. <br />                                                                              Taken together, results place GGAP1 chromosome-linked <br />                                                                              lethal interval 8C8&#226;&#8364;&#8220;8C12, demonstrate GGAP1 <br /> Results                                                                      allele recessive lethal mutation. <br /> Generation P-element Excision-mediated Allele                            Intriguingly, 8C8&#226;&#8364;&#8220;8C12 interval contains gene encoding <br />                                                                              Drosophila AP-1 adaptin subunit AP-1c. Previous studies <br /> dGGA <br />                                                                              suggested dGGA functionally redundant hetero- <br />    Previous studies using <span id='am-475' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-476' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-485' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-486' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-487' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-493' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-494' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-534' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-537' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-542' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-547' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-551' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> knock dGGA <br />                                                                              tetrameric clathrin adaptor AP-1 sorting lysosomal <br /> expression suggested dGGA essential, implicated dGGA <br />                                                                              enzyme receptor LERP trans-Golgi clathrin-coated <br /> in sorting lysosomal hydrolases preventing retinal <br />                                                                              vesicles cultured cells [25,30]. We considered possibility that <br /> degeneration [31,32]. <br />                                                                              lethality 8C8&#226;&#8364;&#8220;8C12 interval arose simultaneously in <br />    To generate <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>deletion mutation</span> dGGA locus, took the <br />                                                                              screen GGAP1 allele result mutation <br /> approach generating imprecise P-element excisions [34]. As a <br />                                                                              AP-1c, reflect synthetic lethal interaction between <br /> target excision, used GGAKG05289 allele, carries a <br />                                                                              mutations. To test hypothesis, used mutant <br /> copy 11.5 kb transposon P{SUPor-P} inserted ca. 1 kb <br /> upstream dGGA transcription start site. We used scheme              PBac{RB} transposon insertion AP-1c gene test for <br /> (Figure 1A) P{SUPor-P}KG05289 element                        complementation. This mutation recessive lethal. Female flies <br /> mobilized P-element transposase provided crossing                 heteroallelic GGAP1 PBac{RB}AP-1c viable fertile, <br /> stock carrying Hop2 transposase source scored loss            demonstrating lethality GGAP1 AP-1c <br /> the y+ marker X-linked recessive lethality progeny.            mutation. <br /> Based criteria, identified candidate lines dGGA              The ability rescue recessive lethality GGAP1 <br /> mutations screen.                                                  chromosome Dp(1;3)DC203 permits assessment dGGA <br />    Among lines, defined deletion dGGA locus            protein expression GGAP1 mutant flies. No GGA protein is <br /> in line, GGAP1. This allele result             detectable Western blot GGAP1; Dp(1;3)DC203 flies <br /> deletion removes 1117 bp upstream dGGA start codon               (Figure 1C) using antibody dGGA (antibody epitope <br /> and extends 117 bp downstream start codon                       sequence amino acids 261[QLVADTL]- 660[end]) [25]. <br /> (Figure 1B). Thus, deletion removes promoter, transcrip- <br /> tion start site, 59 untranslated region, start codon 13 additional       GGAP1 does Cause Age-dependent Retinal <br /> codons, intron.                                      Degeneration <br />                                                                                 RNAi knockdown dGGA using dGGA-RNAi2 hairpin <br /> The GGAP1 Allele Recessive Lethal                                     eye-specific Gal4 drivers results age-dependent retinal <br />    The screen GGAP1 allele recovered                  degeneration [32](Fig. 2B). To test comparable <br /> designed select dGGA mutations associated              degeneration associated GGAP1 mutation, examined <br /> recessive lethality. To test lethality associated       sectioned compound eyes two-week-old GGAP1; <br /> GGAP1 chromosome maps dGGA locus, tested                 Dp(1;3)DC203 adults (Fig. 2C). When compared comparably <br /> duplication region X chromosome containing GGA                   aged wild type (Oregon R; Fig. 2A) w1118; Dp(1;3)DC203 <br /> carried Y chromosome complement lethality                   controls (Fig. 2D), significant difference in <br /> associated GGAP1 chromosome. dGGA located                 ommatidial array organization rhabdomere number/integrity. <br /> cytological region 8F1. Surprisingly, Dp(1;Y)BSC58, <br /> which carries X chromosome fragment breakpoint <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                     Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />  <br /> Figure 1. Generation GGAP1 mutant allele. A. Scheme generate candidate transposon excision-mediated deletions GGA. B. <br /> Structures GGAKG05289 GGAP1 alleles. C. Western blot showing GGA protein levels relative cytoplasmic actin wild type (WT) GGAP1 <br /> adult flies. Arrow indicates predicted position truncated GGAP1 protein, assuming in-frame AUG codon used. <br /> doi:10.1371/journal.pone.0045163.g001 <br />  <br />  <br />                                                                              GGAP1 does Cause age-dependent Retinal <br />                                                                              Degeneration <br />                                                                                 RNAi knockdown GGA using GGA-RNAi2 hairpin under <br />                                                                              eye-specific Gal4 drivers results age-dependent retinal degen- <br />                                                                              eration [32]. To test comparable degeneration is <br />  Table 1 Complementation mapping lethal mutation <br />                                                                              associated GGAP1 mutation, examined sectioned <br />  associated GGAP1. <br />                                                                              compound eyes two-week-old GGAP1; Dp(1;3)DC203 adults. <br />                                                                              When compared comparably aged wild type (Oregon R) and <br />                          Cytological              Complements GGAP1          w1118; Dp(1;3)DC203 controls, significant difference <br />  Duplication             interval                 lethality                  ommatidial array organization rhabdomere number/ <br />                                                                              integrity (Figure 2). <br />  Dp(1;Y)BSC58            [8D9&#226;&#8364;&#8220;8E4] &#226;&#8364;&#8220;9E1           no <br />  Dp(1;Y)BSC144           8A2&#226;&#8364;&#8220;8F9                  yes <br />                                                                              Generation Targeted Knockout Allele dGGA <br />  Tp(1;Y)BSC145           [8A2&#226;&#8364;&#8220;8B6] &#226;&#8364;&#8220;8F9           yes <br />                                                                                 The lack recessive lethality absence age-dependent <br />  Tp(1;Y)BSC147           [8C3&#226;&#8364;&#8220;8C4] &#226;&#8364;&#8220;8F9           yes                        degeneration GGAP1 flies contrast sharply phenotypes <br />  Tp(1;Y)BSC148           [8C4&#226;&#8364;&#8220;8C12] &#226;&#8364;&#8220;8F9                                  reported dGGA RNAi knockdown [31,32]. Since GGAP1 <br />  Tp(1;Y)BSC150           [8D1&#226;&#8364;&#8220;8D2] &#226;&#8364;&#8220;8F9                                   deletion removes 13 codons in-frame <br />  Tp(1;Y)BSC151           [8D2&#226;&#8364;&#8220;8D4] &#226;&#8364;&#8220;8F9           no <br />                                                                              methionine codon 16 codons downstream break point, <br />                                                                              formally possible sequences upstream deletion <br />  Tp(1;Y)BSC152           [8D4&#226;&#8364;&#8220;8D9] &#226;&#8364;&#8220;8F9           no <br />                                                                              contain cryptic promoter transcription start site. If were <br />  Tp(1;Y)BSC153           [8D4&#226;&#8364;&#8220;8D9] &#226;&#8364;&#8220;8F9           no <br />                                                                              case, mRNA potentially translated using that <br />  Tp(1;Y)BSC154           [8D9&#226;&#8364;&#8220;8E4] &#226;&#8364;&#8220;8F9                                   downstream methioninyl codon start codon, resulting a <br />  Tp(1;Y)BSC155           [8E4&#226;&#8364;&#8220;8E12] &#226;&#8364;&#8220;8F9                                  slightly truncated dGGA protein includes VHS <br />  Tp(1;Y)BSC156           [8E12&#226;&#8364;&#8220;8F9] &#226;&#8364;&#8220;8F9                                  domain. Thus, confidently exclude possibility a <br />  Dp(1;3)DC201            8C1&#226;&#8364;&#8220;8C4                  no <br />                                                                              small functional dGGA, undetectable Western blot <br />                                                                              analysis, expressed GGAP1 allele, this <br />  Dp(1;3)DC202            8C1&#226;&#8364;&#8220;8C8                  no <br />                                                                              allele functional null mutation. <br />  Dp(1;3)DC203            8C4&#226;&#8364;&#8220;8C13                 yes <br />                                                                                 To generate dGGA null mutation, turned site-directed <br />  doi:10.1371/journal.pone.0045163.t001                                       mutagenesis using ends-out gene replacement strategy adapted <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               3                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                     Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />  <br /> Figure 2. GGAP1 does cause age-dependent retinal degeneration. Wild type (A), ey/GMR-Gal4.; dGGA RNAi2 (B), GGAP1; Dp(1;3)DC203;(C) <br /> and w1118; Dp(1;3)DC203 (D) adult flies aged 14 d. 25uC, eyes sectioned, stained toluidine blue examined light <br /> microscopy. Ommatidial organization rhabdomere number/morphology normal panels A, C D, highly degenerated panel B. <br /> Note small number pigment granules w1118; Dp(1;3)DC203 eyes reflects weak white transgene expression stock relative the <br /> wild-type white gene expression Oregon R GGAP1; Dp(1;3)DC203 stocks. <br /> doi:10.1371/journal.pone.0045163.g002 <br />  <br /> from Chen et al. (2009). A 3 kb fragment extending 59               reciprocal control cross. Statistical analysis suggested no <br /> dGGA coding region 2.7 kb fragment extending 39               selective mortality dGGA mutants relative controls for <br /> dGGA coding region separately amplified PCR cloned               GGAP1 GGAD allele (Table 2). <br /> into pXH87 vector [35] (Figure 3A). We included FM7 <br /> balancer chromosome knockout mobilization steps                dGGA Knockout Mutation Fails Rescue RNAi <br /> order (1) rescue dGGA null mutant flies case knocking          Knockdown dGGA <br /> GGA causes lethality (2) increase efficiency recovering              The absence detectable lethality associated the <br /> targeted events X chromosome [35] We used scheme                    dGGA mutations isolated raises possibility previously <br /> (Figure 3B) p{GGA knockout transgene} targeting cassette        reported lethality associated dGGA RNAi knockdowns could <br /> is mobilized FLP recombinase linearized I-SceI                     off-target. Previous research showed GGA-RNAi2 <br /> provided P{70 FLP} P{70-Sce-I} heat shock. We collected             hairpin lethal combination Actin5C-Gal4 driver, with <br /> ,3000 mosaic- white-eyed virgin females mass-crossed                  lethal period late pupation [32]. Another GGA hairpin <br /> these males stock carrying P{70 FLP} eliminate              construct available Vienna Drosophila Resource Center <br /> residual autosomal copies p{GGA knockout transgene}. From             (VDRC) reportedly lethal [31] semi-lethal [32] the <br /> progeny, 300 flies carried w+ transgene marker seven              Actin5C-Gal4 driver. Since sequences targeted hairpins <br /> these carried transgene marker linked X chromosome.               completely deleted GGAD allele, combining GGAD <br /> Using PCR primers flanking outside targeted region inside            allele hairpins provides explicit test the <br /> the EYFP-mini-white knockout cassette, Western blot               RNAi-mediated lethality on- off-target. Since VDRC <br /> analysis, able confirm lines GGA                 hairpin sequence (construct ID 1710) entirely contained within <br /> knocked (Figure 4 data shown). The targeting rate            GGA RNAi2 hairpin sequence, generated mutant flies that <br /> calculated 0.23%, low end targeting                GGAD; GGA-RNAi2 crossed females stock with <br /> frequency range reported Chen et al. (2009) autosomal                 males stock containing Act5C-Gal4 driver. We recovered no <br /> knockouts using approach. GGAD result replacing            driver-containing sons, demonstrating lethality associated <br /> total 1131 bp, including 536 bp exon, entire             GGA-RNAi 2 1710 hairpins off-target. <br /> third intron 526 bp fourth exon, EYFP and <br /> mini-white cassette pXH87. This results loss 353 codons,        Defining Minimal dGGA Gene <br /> including C-terminal half VHS domain, entire GAT                 The sequence location GGA gene predicted span the <br /> domain N-terminal half hinge domain. All              region X: 9,495,067 9,498,313 according to <br /> GGAD lines viable fertile. Like GGAP1, GGAD flies            flybase.org. Hex-A, spans region X: 9,479,918 <br /> evidence age-dependent retinal degeneration 2 weeks (data              9,482,434, closest annotated gene upstream dGGA. <br /> not shown).                                                                  Thus, transcriptional control dGGA regulated by <br />    To test possible semi-lethality associated dGGA                  sequences 12.6 kb interval genes. <br /> mutations, compared ratio heterozygous dGGA mutant                 However, dGGA lacks TATA box, suggesting a <br /> daughters dGGA sons generated cross dGGA mutant                 housekeeping gene [36]. We inferred dGGA&#226;&#8364;&#8482;s <br /> females wild-type male flies ratio heterozygous              regulatory elements close transcription start site. <br /> dGGA mutant daughters GGA+ sons recovered                        Since 39 UTR dGGA overlaps 39 UTR the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                         Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />  <br /> Figure 3. Generation GGA knockout flies. A. Molecular basis GGA targeting. The GGA knockout plasmid created cloning ,3 kb <br /> GGA upstream downstream sequences multiple cloning sites pXH87 vector [35]. FLP recombinase catalyzes excision the <br /> knockout cassette I-SceI cleavage releases DNA fragment excised circle, undergo homologous recombination the <br /> targeted genomic DNA sequence generate GGAD. For simplicity, representations genes right GGA omitted. B. Scheme <br /> used generate candidate GGAD flies. <br /> doi:10.1371/journal.pone.0045163.g003 <br />  <br /> downstream gene CG3004, hypothesize functional                    support dGGA expression, cloned dGGA genomic sequences <br /> dGGA gene ends annotated transcription termination site X:             including 1 kb upstream dGGA transcription start site and <br /> 9,498,313. To define minimum genomic sequence necessary                inserted sequence transgene transgenic flies yw; <br />                                                                               P{GGA+}. To test autonomous expression transgene, we <br />                                                                               crossed autosomal inserts GGAP1 GGAD <br />                                                                               background. Western blot analysis showed transgene <br />                                                                               supports GGA expression near wild-type levels (Figure 4 and <br />                                                                               data shown). <br />  <br />  <br />  <br />                                                                                Table 2 Testing semi-lethality GGAD GGAP1 alleles. <br />  <br />  <br />                                                                                Crosses                           F1 female     F1 male    p-Value <br />                                                                                           D <br />                                                                                female GGA X male yw              229           183        0.5310 <br />                                                                                female yw X male GGAD             223           195 <br /> Figure 4. A GGA transgene complements protein-null <br /> phenotype GGAD flies. GGAD;P{GGA+} restored GGA protein              female GGAP1 X male yw            192           174        0.7815 <br /> wild type levels. Actin levels corresponding lane shown below.        female yw X male GGAP1            259           225 <br /> All images taken blot exposure. <br /> doi:10.1371/journal.pone.0045163.g004                                          doi:10.1371/journal.pone.0045163.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                5                         September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                             Drosophila GGA Is Not Essential <br />  <br />  <br />  <br /> GGAP1 GGAD Flies No Defect Cathepsin L                             trafficked lysosomes Drosophila places GFP the <br /> Cathepsin D Processing                                                            luminal face vesicle membrane, resulting rapid extinction of <br />    GGA family proteins previously implicated                     GFP fluorescence fusion lysosomes [38,39]. We <br /> sorting and/or processing lysosomal hydrolases [29]. Missorting                examined lysosomal proteolytic function GGA knockout <br /> of cathepsin L cathepsin D proform reported                       instar larval fluorescent microscopy salivary glands <br /> dGGA RNAi knockdowns [32]. We examined cathepsin L                            expressing LAMP1-GFP fusion protein. Compared the <br /> cathepsin D sorting processing adult flies. Western blot                   control GFP-LAMP1 expression, apparent increase in <br /> analysis extracts GGAP1 GGAD flies showed                          numbers size fluorescent vesicles mutant flies that <br /> significant defect cathepsin L cathepsin D sorting               indicative GFP stabilization defective lysosomes <br /> processing (Figure 5). In case cathepsin D,                      (data shown). By criterion, then, lysosomal proteolytic <br /> exclude possibility minor pro-form                           function GGAP1 GGAD flies normal. Effects on <br /> accumulates knockout flies, weakly cross-reacting                  classes lysosomal hydrolases, however, ruled out <br /> antigen appears position proform extracts,                   assay. <br /> including extract cathepsin D null stock [37]. Larval <br /> hemolymph showed increase accumulation pro-                          GGAD Flies Hypersensitive Dietary Chloroquine <br /> cathepsin L (data shown). Thus, defects cathepsin                         As dGGA implicated sorting hydrolytic enzymes the <br /> sorting reported dGGA RNAi knockdown [32], like lethality                 trans-Golgi network lysosomes, hypothesized the <br /> associated dGGA RNAi knockdown, appears off-                        lysosome function impaired environmental factors, <br /> target effect. dGGA does significant sorting                        additional lysosomal defects loss dGGA enhance <br /> processing effect cathepsin L cathepsin D. However,                  lysosomal impairment. To test hypothesis, tested <br /> results exclude defects sorting and/or processing              loss dGGA enhances sensitivity dietary chloroquine. <br /> lysosomal hydrolases.                                                             Chloroquine drug used malaria treatment raises <br />                                                                                   lysosomal pH, impairs lysosomal enzyme activities [40,41]. To <br /> Lysosome Proteolytic Function Unaffected GGAP1                              establish dietary chloroquine impairs lysosome function, we <br /> and GGAD Flies                                                                    fed instar larvae expressing LAMP1-GFP food containing <br />    Previously, impaired lysosomal function reported dGGA                   10 mM chloroquine 36 hours. We examined lysosomal <br /> RNAi knockdown flies, accumulation LAMP1-GFP                            function larval salivary glands fluorescent microscopy. There <br /> fusion protein vesicles [32]. The LAMP1-GFP fusion protein                  consistent increase accumulation GFP-LAMP1 treated <br />  <br />  <br />  <br />  <br /> Figure 5. Cathepsin L cathepsin D processing steady-state levels impaired GGA null flies. A. Western blot showing <br /> mature CTSL protein level relative cytoplasmic actin adult wild-type (WT), GGAP1, GGAD, GGAP1;P{GGA+} GGAD;P{GGA+} rescue flies. The <br /> unprocessed form CTSL undetectable samples. Actin levels corresponding samples shown below. B. Western blot <br /> showing mature CTSD protein level relative cytoplasmic actin adult wild-type (WT), GGAD, GGAP1, GGAD;P{GGA+} cathepsin D null mutant <br /> flies (cathD1) [37]. The CTSD pro-form comigrates cross-reactive antigen present CTSD null mutant (asterisk). An antigen ca. 63 kDa <br /> also appears samples, antigen present cathD1 flies (asterisk), unrelated CTSD. Actin levels corresponding lanes <br /> are shown below. All images using given antibody taken blot exposure. <br /> doi:10.1371/journal.pone.0045163.g005 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                    6                         September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                        Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />                                                                                  dGGA Mutations Interact Autophagy-associated <br />                                                                                  Gene Blue cheese (Bchs) <br />                                                                                      Drosophila Bchs homolog human Alfy gene. Both <br />                                                                                  gene products contain highly conserved BEACH domain, <br />                                                                                  FYVE zinc-finger domain WD40 repeats [42]. The structure <br />                                                                                  Bchs suggests function scaffolding protein autophagic <br />                                                                                  membrane trafficking [42,43]. Overexpression Bchs an <br />                                                                                  eye-specific driver leads reduced rough eye phenotype in <br />                                                                                  flies, phenotype modified mutations genes encoding <br />                                                                                  factors implicated autophagy [44,45]. When dGGA was <br />                                                                                  knocked eye RNAi Bchs overexpression <br />                                                                                  background, rough eye phenotype enhanced eye <br />                                                                                  size reduced, suggesting dGGA plays role autophagy <br />                                                                                  [32]. We tested GGA mutant lines assay confirm a <br />                                                                                  role GGA autophagy. To accurately quantify differences <br />                                                                                  eye size Bchs overexpression dGGA mutant flies and <br />                                                                                  wild-type flies, measured red eye pigment in <br />                                                                                  genotype index total eye volume. Both GGAP1; GMR- <br />                                                                                  Gal4 EP(2L)2299 GGAD; GMR-Gal4 EP(2L)2299 eye <br />                                                                                  pigment comparison +; GMR-Gal4 EP(2L)2299 (P-value <br />                                                                                  ,0.001). However, compared yw; GMR-Gal4 <br />                                                                                  EP(2L)2299 flies, GGAD; GMR-Gal4 EP(2L)2299 a <br />                                                                                  significant reduction (P-value ,0.05) (Figure 8). It formally <br />                                                                                  possible GGAP1 expresses small functional <br />                                                                                  GGA, coding region intact. The significant eye <br />                                                                                  pigment reduction GGAD Bchs overexpression, however, <br />                                                                                  suggests dGGA modifier Bchs overexpression, thus <br />                                                                                  implicating GGA autophagy. <br />  <br />                                                                                  GGAD Flies Hypersensitive Amino Acid Starvation <br />                                                                                      In starvation&#226;&#8364;&#8220;induced autophagy, intracellular proteins and <br />                                                                                  organelles cannibalized meet minimum <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br />                                                                                  requirements starving cells. Autophagy mediated the <br />                                                                                  lysosomal degradation pathway [46]. The enhancement the <br />                                                                                  Bchs overexpression phenotype dGGA mutation suggests role <br /> Figure 6. Dietary chloroquine impairs lysosome-mediated                          dGGA autophagy. We hypothesized dGGA were <br /> proteolysis. Salivary glands dissected LAMP1-GFP-express-              required efficient autophagy, lack dGGA sensitize <br /> ing instar larvae fed 36 hr instant Drosophila food <br /> reconstituted water (A) 10 mM chloroquine (B). Increased GFP             flies starvation. In order trigger autophagy, placed newly <br /> fluorescence salivary glands chloroquine-fed larvae results            emerged flies amino acid deficient media [47,48]. Crosses were <br /> from larger brighter fluorescent vesicles.                              performed previous chloroquine experiment minimize <br /> doi:10.1371/journal.pone.0045163.g006                                            genetic background effects. During starvation, median survival <br />                                                                                  time GGAD males 11.29 days, median survival <br /> larvae compared controls (Figure 6), testifying impaired                   times control yw males 14.03 days, rescue lines <br /> lysosomal proteolysis chloroquine-fed larvae.                                 GGAD; P{GGA+}E2 GGAD; P{GGA+}A2 median survival <br />    Dietary chloroquine 10 mM causes significant                      times 15.74 days 14.50 days, respectively (Figure 9). The <br /> toxicity Drosophila adults (our unpublished observations). If                 GGAD survival time significantly different control cross <br /> dGGA required lyososomal homeostasis, GGAD flies                   rescue cross P-values ,0.0001. The reduced survival of <br /> be hypersensitive dietary chloroquine. Newly eclosed adult flies              GGAD mutant stock implicates dGGA autophagy. <br /> raised standard food placed food supplemented with <br /> 20 mM chloroquine. To minimize effect background                      Discussion <br /> genotype, tested sons crosses female GGAD male yw <br />                                                                                     Our understanding GGA family protein functions animals <br /> parents used sons reciprocal cross controls. As an <br />                                                                                  increased significantly discovery family of <br /> additional control, sons cross female GGAD male <br />                                                                                  monomeric clathrin adaptors 2000. However, studies were <br /> yw;P{GGA+} used test ability wild-type dGGA <br />                                                                                  based RNAi knockdowns cultured cells animal models, <br /> transgene complement knockout phenotype. The median                       complicated potential off-target effect as <br /> survival time GGAD males 2.9 days, control yw males                  incomplete protein elimination. Therefore, null mutations are <br /> was 5.25 days, rescue lines GGAD; P{GGA+}E2                  required define true function GGAs multicellular <br /> GGAD; P{GGA+}A2 4.98 days 4.45 days, respectively                       organisms. Recently, GGA knockouts animals were <br /> (Figure 7). GGAD survival time significantly reduced                documented mice Govero, et al. (2012). Mutant phenotypes <br /> the control rescue crosses P-values ,0.0001. The                        vary murine GGAs affected the <br /> hypersensitivity GGAD mutants chloroquine consistent               different genetic background mutants [27]. Furthermore, <br /> a role dGGA lysosomal homeostasis.                                        small size perinatal hypoglycemia, phenotypes <br />                                                                                  uninformative basis lethality GGA2 mutant <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   7                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                    Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />  <br /> Figure 7. GGA mutant flies hypersensitive dietary chloroquine. Blue lines represent: GGAD males (n = 181 median survival time <br /> 2.90 days) Red lines represent a. yw males (n = 97, P-value ,0.0001 median survival time 5.25 days) b. yw; P{ GGAD} A2 males (n = 151, P-value <br /> ,0.0001 median survival time 4.45 days) c. yw; P{ GGAD} E2 males (n = 161, P-value ,0.0001 median survival time 4.98 days). <br /> doi:10.1371/journal.pone.0045163.g007 <br />  <br /> mice GGA1/GGA3 double mutant mice. The role GGA                       function research Drosophila. For example, function each <br /> family proteins mammals remains elusive. The presence            domain analyzed mutating domain dGGA <br /> a single GGA homolog Drosophila provides unique                        testing mutations function transgenic flies carrying <br /> opportunity assess requirements GGA metazoan.               GGAD allele. Although method targeted knockout was <br />    In independent screens, generated distinct gene               adapted Chen et al. (2009), knowledge, first <br /> knockout mutations. A key finding study                report targeted knockout X-linked gene method. <br /> mutations, GGAP1 GGAD, showed defect survival,                      Our results clearly demonstrate GGA essential in <br /> demonstrating dGGA essential normal development              Drosophila agree Hirst et al. (2011), reported that <br /> under laboratory conditions. Together functional dGGA                dGGA knockdown flies develop normally obvious <br /> transgene, created genetic tool kit future GGA protein             phenotype. In contrast, Eissenberg et al. (2011) and <br />  <br />  <br />  <br />  <br /> Figure 8. GGA knockout enhances eye phenotype caused Bchs overexpression. Control: Bchs overexpressing male +; GMR-Gal4 <br /> EP(2L)2299, yw; GMR-Gal4 EP(2L)2299. Bchs overexpressing male knocked GGA: GGAP1; GMR-Gal4 EP(2L)2299 GGAD; GMR-Gal4 <br /> EP(2L)2299. Amount extracted red eye pigment measured 480 nm. (*)GGAD; GMR-Gal4 EP(2L)2299 showed significant reduction red eye <br /> pigment compared +; GMR-Gal4 EP(2L)2299 yw; GMR-Gal4 EP(2L)2299 (P-value , 0.0008 0.04). GGAP1; GMR-Gal4 EP(2L)2299 showed <br /> only significant reduction red eye pigment compared +; GMR-Gal4 EP(2L)2299 (P-value , 0.0003). <br /> doi:10.1371/journal.pone.0045163.g008 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              8                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                      Drosophila GGA Is Not Essential <br />  <br />  <br />  <br />  <br /> Figure 9. GGA mutant flies hypersensitive amino acid starvation. Blue lines represent: GGAD males (n = 212 median survival time <br /> 11.30 days) Red lines represent a. yw males (n = 202, P-value ,0.0001 median survival time 13.65 days) b. yw GGAD; P{GGA+} A2 males (n = 131, P- <br /> value ,0.0001 median survival time 14.92 days) c. yw GGAD; P{GGA+} E2 males (n = 144, P-value ,0.0001 median survival time 16.4 days). <br /> doi:10.1371/journal.pone.0045163.g009 <br />  <br /> Kametaka et al. (2012) reported lethality RNAi knockdowns                   GGA family proteins involved sorting lysosomal <br /> flies, leading speculate lethality cases       hydrolases, important normal lysosomal function [22&#226;&#8364;&#8220; <br /> result off-target effect RNAi. We confirmed hypothesis           25,29,31,32]. This explained heterotetrameric <br /> by showing RNAi-induced lethality comple-                    clathrin adaptor AP-1 functions redundantly GGA under <br /> mented GGAD, lacks RNAi-targeted sequences. In                    standard lab conditions. A requirement GGA disclosed by <br /> the absence targeted dGGA sequences, lethality              hypersensitivity GGA knockout flies dietary chloroquine, <br /> to off-target effect RNAi. We 19-nucleotide-long                 lysosomal alkalinizing agent, supporting model GGA <br /> perfect match shared hairpins reported                family proteins contribute lysosomal homeostasis stress <br /> show lethality semi-lethality sequence coding                  conditions. <br /> region Arginine methyltransferase 4 (Art4) gene. However,                   Knocking GGA Bchs overexpression flies enhanced eye <br /> Western blot analysis extracts dGGA-RNAi2 knock-                   phenotype, similar results reported dGGA-RNAi2 <br /> down larvae probed Art4 protein showed significant                      knockdowns [32], suggesting dGGA directly or <br /> reduction (data shown). Thus, basis off-target effect          indirectly involved process autophagy. The role of <br /> remains unknown.                                                               dGGA autophagy unique distinct dGGA and <br />     Previous research implicated GGA family proteins sorting            AP-1 function lysosomal hydrolase sorting. This hypothesis is <br /> of lysosomal hydrolases (cathepsin B, D L) [22&#226;&#8364;&#8220;25,29,31,32].               supported observation GGAD flies are <br /> In dGGA knockout flies, detectable defect cathepsin L           hypersensitive amino acid starvation, consistent impaired <br /> or cathepsin D processing, indicating missorting                   autophagy. Since lysosomes final destination autophagy, <br /> cathepsin L cathepsin D reported dGGA knockdowns                     defects alter lysosomal composition lead lysosomal <br /> likely off-target effect generated RNA hairpin                 storage disorder, resulting impaired autophagy shortened <br /> construct. The absence cathepsin sorting defect dGGA-null              life-span [49]. <br /> flies consistent evidence dGGA functionally                    Our proposed link lysosomal stress, autophagy and <br /> redundant AP-1 cathepsin sorting cultured cells [25]. AP-1            GGA Drosophila suggests possible explanation the <br /> could substitute function dGGA lysosomal processing              puzzling results GGA knockdowns mice. Gga1/3 double <br /> in GGA <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>null mutants</span>. In addition, normal lysosomal                         mutant mice 47% death rate day birth <br /> proteolysis function GGAP1 GGAD flies suggests                 [27]. Depending genetic background, GGA2 mutant mice <br /> previously reported stabilization LAMP1-GFP cytoplasm                death rate 73% days birth die before <br /> of dGGA RNAi knockdown cells off-target. The lack role                 E9 embryonic stage (earliest testable stage) [27]. We suggest that <br /> for GGA LAMP1-GFP turnover consistent                      lethal periods explained partial impairment <br /> functional redundancy dGGA AP-1.                                        autophagy mutation GGAs. In placental mammals, <br />     We evidence age-dependent retinal degeneration                autophagy crucial specific developmental stages. Between the <br /> in GGAP1 GGAD flies reported dGGA-RNAi2 knockdowns                    cell division implantation, autophagy dramatically <br /> driven GMR-Gal4 [32]. Although dGGA enriched                          induced cleavage stage mouse zygotes [50]. Atg5 (autophagy- <br /> Drosophila testes [33], male flies hemizygous GGAP1                 related 5) knockout mice early embryonic death [51]. Early <br /> or GGAD evidence impaired fertility.                                zygotic death seen GGA2 knockout mice lines. <br />     Under standard laboratory conditions, dGGA knockout mutants                In addition, autophagy induced birth returns basal <br /> have obvious apparent phenotype; external cuticle                   level 1&#226;&#8364;&#8220;2 days [52]. This period autophagic stress could <br /> behavior appear normal. However, reports                    explain lethality 24 48 hours Gga1/3 double <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 9                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                   Drosophila GGA Is Not Essential <br />  <br />  <br /> mutation GGA2 mutant strains. The fact mice             dGGA Genomic Transgene Construct <br /> lacking a/b subunits N-acetylglucosamine-1-phosphotrans-               Using primer pairs forward 59 TCTAGA AACAGTTTTAGC- <br /> ferase [the enzyme required step synthesis          CAGACCGATTGA 39 reverse 59 TCTAGA GAAA- <br /> mannose-6-phosphate (M6P) sorting marker] fully viable                  CAGTGGGGAGAGTAGCAATG 39, 4.5 kb GGA genomic <br /> argues simple model defective sorting                 DNA generated. The GGA fragment, spanning 1 kb <br /> lysosomal hydrolases causes stage specific death periods        upstream transcription start site just the <br /> high autophagy [53]. It GGAs contribute                 polyadenylation site (X:9496311, 9498217) cloned to <br /> lysosomal function independently role M6P receptor              P{CaSpeR}. Transgenic flies generated BestGene Inc <br /> recognition. Alternatively, GGAs promote apoptosis                    (Chino Hills, CA). Transformants identified based eye <br /> through mechanism independent lysosome hydrolase sorting.              pigmentation conferred P{CaSpeR-GGA+} transgene. <br /> It interesting look molecular signatures of <br /> autophagy and/or autophagic cell death dead dying                 Drosophila Stockkeeping <br /> mice determine link autophagy GGA                  All crosses, noted, maintained standard <br /> suggested Drosophila data conserved mammals.                   cornmeal-agar-molasses medium 25uC. <br />  <br /> Materials Methods                                                       Drosophila Eye Sections <br /> Drosophila Stocks                                                              Adult Drosophila heads removed bodies the <br />   The following stocks obtained Bloomington                   antennae removed facilitate fixation. Tissue pieces fixed in <br /> Drosophila Stock Center: y GGAKG05289; Dp(1;Y)BSC58;                        3.5% glutaraldehyde 0.1 M sodium cacodylate buffer (pH 7.25) <br /> Dp(1;Y)BSC144; Tp(1;Y)BSC145; Tp(1;Y)BSC147; Tp(1;Y)BSC148;                 containing 5% sucrose 2 mM calcium chloride for <br /> Tp(1;Y)BSC150; Tp(1;Y)BSC151; Tp(1;Y)BSC152; Tp(1;Y)BSC153;                 3 d. 4uC. The tissue washed 2&#226;&#8364;&#8220;3615 min 0.1 M <br /> Tp(1;Y)BSC154; Tp(1;Y)BSC155; Tp(1;Y)BSC156; Dp(1;3)DC201;                  sodium cacodylate buffer containing 5% sucrose (this all <br /> Dp(1;3)DC202; Dp(1;3)DC203; y w; [70FLP][70I-SceI]/TM6, w;                  subsequent steps polymerization room temperature) <br /> [70FLP] w1118 PBac{RB}AP-1c304546/FM7c. The y w; CyO                    post-fixed 1% osmium tetroxide 0.1 M sodium <br /> Hop2/Bc Elp stock obtained Dr. D. Dorsett. The y w mof4/           cacodylate buffer containing 5% sucrose 3&#226;&#8364;&#8220;4 hours RT on <br /> FM7, y wa B stock obtained Dr. J.C. Lucchesi. The GMR-             rotator. The tissue washed 2 x 10 min distilled water, <br /> Gal4 EP(2L)2299 stock obtained Dr. K. Finley. The yw;              dehydrated 2 x 10 min steps 35%, 50%, 75%, 95% <br /> LAMP1-GFP stock obtained Dr H. Kra&#194;&#168;mer. Cathepsin D                100% ethanol, washed 2 x 10 min propylene oxide and <br /> mutant stock (cathD1) obtained Dr M. Feany.                        infiltrated 1:1 mixture Embed resin (Polysciences, Inc., <br />                                                                             Warrington, PA) propylene oxide overnight rotator. The <br />                                                                             tissue incubated fresh Embed resin 6 hours, <br /> Methods dGGA Knockout                                                   transferred BEEM capsules filled fresh resin, and <br />     About 3 kb upstream dGGA (using primers forward: 59 <br />                                                                             polymerized overnight 70uC. 0.5 mm sections cut <br /> attatatGGTACC-TTCGCCAACGGTAACGGTAAG 39 Re- <br />                                                                             using glass knife Reichert Ultracut E ultramicrotome <br /> verse: 59 attatatACCGGT-AATTTGGACGCCTCGTCCTGGA <br />                                                                             (Depew, NY), collected heat-attached glass slides, stained <br /> 39) 2.7kb downstream GGA (using primers forward: 59 <br />                                                                             toluidine blue cover-slipped Permount mounting <br /> attatatGCATGC-GGATCGGATGCCCAGCTTCA 39 reverse: <br />                                                                             medium. Light micrographs taken Leica DM5000 B <br /> 59 attatatCTCGAG-TTACCAGCACCGCGCCA CTTAT 39) <br />                                                                             research light microscope using 100X objective Leica DFC <br /> were amplified genomic DNA cloned the <br />                                                                             350 FX digital camera Leica Application Suite V3 software. <br /> pXH87multiple cloning sites EYFP and <br />                                                                             Images transferred PhotoShop post-acquisition <br /> mini-white cassette [35]. Plasmid DNA sent BestGene Inc <br />                                                                             processing. <br /> (Chino Hills, CA) generate transgenic flies transgenic flies <br /> were selected based eye pigmentation conferred p{GGA <br /> knockout} transgene. For knockout strategy (Fig. 3), started         Western Blots <br /> with 2000 virgin females 100 vials crossed                  Proteins adult flies prepared lysis buffer: <br /> w;P{70FLP}P{70I-SceI} males. After 3 days, flies                   25 mM Tris-HCL pH 7.5, 150 mM NaCl, 1% Triton X100, <br /> transferred new vials progeny vial             Roche protease inhibitors 1 tablet 50 ml, ES56 5 mg/ml <br /> heat shocked 38uC 90 min. 3000 mosaic white eyed                  pepstatin 50 mM, 2 mM indoleacetic acid. A single fly <br /> virgin females recovered initial crosses            transferred 100 ml lysis buffer frozen immediately. <br /> were mated P{70FLP} males. 300 non-mosaic colored eyed                 Samples sonicated twice 10 seconds, centrifuged at <br /> flies recovered cross pair-mated           16 krpm 4uC 6 minutes. Cleared supernantant used for <br /> FM7, y wa B flies. By crossing FM7, y wa B flies,             Western blot analysis. Drosophila GGA antibodies kindly <br /> determine progeny carried X-linked insertion                provided Drs. J. Hirst S. Kametaka. Cathepsin L antibody <br /> knockout cassette. PCR analysis used confirm site             purchased R&amp;D Systems, Inc. Cathepsin D antibody <br /> insertion GGA locus. Primers are:                                    kindly provided Dr. S. Kametaka. HRP-conjugated goat <br />     Forward covers 3 kb fragment: 59 GCAATGGGC-                             anti-rabbit antibody goat anti-mouse antibodies were <br /> TATTCTGGGTATCAC 39.                                                         purchased Millipore. <br />     Reverse covers 3 kb fragment: 59 TGCTCAGGGCG- <br /> GACTGGTAG 39.                                                               Analyze Bchs Overexpression Eye Phenotype <br />     Forward covers 2.7 kb fragment: 59 GGCAAGGTCATCCTG-                       Virgin females yw, OreR, GGAP1 GGAD lines crossed <br /> GAGACG 39.                                                                  Bchs overexpression line GMR-Gal4 EP(2L)2299. Sons were <br />     Reverse covers 2.7 kb fragment: 59 TCCCGGACTAATCC-                      collected aged days dissection. For each <br /> CAGTTGCT 39.                                                                replicate, 10 fly heads cut eyes placed 1 ml <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             10                       September 2012 | Volume 7 | Issue 9 | e45163 <br />                                                                                                                                           Drosophila GGA Is Not Essential <br />  <br />  <br /> acidified ethanol (pH 2) 24 hours. Absorbance measurements                             0.3% Proprionic acid, 0.3% Tegosept. The number of <br /> on replicates taken wavelength 480 nm.                                        surviving flies recorded daily. <br />                                                                                              Starvation Test. Flies raised normal fly food. After <br /> Chloroquine Diet Experiments                                                               pupation, pupae transferred amino acid deprived food (3% <br />    LAMP-1-GFP stability assay. yw; GMR-Gal4 UAS-LAMP1-                                     agar, 5% sucrose, 0.3% Tegoset, 0.3% Proprionic acid in <br /> GFP/CyO flies raised instar stage standard                                PBS) [48]. After adults eclosed, adult males collected 6 <br /> Drosophila medium. Early instar larvae transferred                           hours time transferred fresh amino acid deprived food. The <br /> vials containing 2 g. Formula 4&#226;&#8364;&#8220;24 Instant Drosophila medium                               number survivors recorded daily. <br /> (Carolina Biological Supply) reconstituted 6 ml 10 mM <br /> chloroquine (Sigma-Aldrich Co), 0.3% Proprionic acid, 0.3%                             Acknowledgments <br /> Tegosept (chloroquine supplement) 6 ml 0.3% Proprionic acid,                            We acknowledge Bloomington Stock center Drs. D. Dorsett, K. Finley <br /> and 0.3% Tegosept (control). Larvae aged 36 h 25uC,                            M. Feany fly stocks, Dr. X. Huang pXH87 vector <br /> then salivary glands dissected analyzed                                    DNA. We thank Drs. J. Hirst S. Kametaka antibody Drosophila <br /> fluorescence microscopy.                                                                   GGA, Dr. S. Kametaka antibody Drosophila cathepsin D Dr. B. <br />    Chloroquine survival curves. GGAP1 GGAD flies                                  Suter antibody Art4. We thank Drs. A. Waheed S. Kornfeld for <br />                                                                                            helpful discussion encouragement. We especially grateful Dr. <br /> crossed yw flies described text, generate knockout <br />                                                                                            Kevin Cook helpful advice sharing unpublished information. <br /> or control males, depending sex parents. As adults <br /> eclosed, adult male flies immediately transferred onto <br /> chloroquine-containing media, consists 2 g. instant <br />                                                                                            Author Contributions <br /> Drosophila media (Carolina Biological Supply Company) recon-                               Conceived designed experiments: SL JCE. Performed the <br /> stituted 6 ml 20 mM chloroquine (Sigma-Aldrich Co),                                experiments: SL AMI JCE. Analyzed data: SL AMI JCE. Contributed <br />                                                                                            reagents/materials/analysis tools: JCE. Wrote paper: SL JCE. <br />  <br /> References <br />  1. Hirst J, Robinson MS (1998) Clathrin adaptors. Biochim Biophys Acta                19. Costaguta G, Stefan CJ, Bensen ES, Emr SD, Payne GS (2001) Yeast Gga coat <br />     1404: 173&#226;&#8364;&#8220;193.                                                                             proteins function clathrin Golgi endosome transport. Mol Biol Cell 12: <br />  2. Robinson MS (2004) Adaptable adaptors coated vesicles. Trends Cell Biol                1885&#226;&#8364;&#8220;1896. <br />     14: 167&#226;&#8364;&#8220;174.                                                                           20. Hirst J, Lindsay MR, Robinson MS (2001) GGAs: roles different domains <br />  3. Boman AL, Zhang C, Zhu X, Kahn RA (2000) A family ADP-ribosylation                      comparison AP-1 clathrin. Mol Biol Cell 12: 3573&#226;&#8364;&#8220;3588. <br />     factor effectors alter membrane transport trans-Golgi. Mol        21. Kamikura DM, Cooper JA (2006) Clathrin interaction subcellular <br />     Biol Cell 11: 1241&#226;&#8364;&#8220;1255.                                                                   localization Ce-DAB-1, adaptor protein secretion Caenorhabditis <br />  4. Poussu A, Lohi O, Lehto VP (2000) Vear, novel Golgi-associated protein              elegans. Traffic 7: 324&#226;&#8364;&#8220;336. <br />     VHS gamma-adaptin &quot;ear&quot; domains. J Biol Chem 275: 7176&#226;&#8364;&#8220;7183.                       22. Mardones GA, Burgos PV, Brooks DA, Parkinson-Lawrence E, Mattera R, et al. <br />  5. Dell&#226;&#8364;&#8482;Angelica EC, Puertollano R, Mullins C, Aguilar RC, Vargas JD, et al.                  (2007) The trans-Golgi network accessory protein p56 promotes long-range <br />     (2000) GGAs: family ADP ribosylation factor-binding proteins related               movement GGA/clathrin-containing transport carriers lysosomal enzyme <br />     adaptors associated Golgi complex. J Cell Biol 149: 81&#226;&#8364;&#8220;94.                    sorting. Mol Biol Cell 18: 3486&#226;&#8364;&#8220;3501. <br />  6. Hirst J, Lui WW, Bright NA, Totty N, Seaman MN, et al. (2000) A family              23. Hida T, Ikeda H, Kametaka S, Akazawa C, Kohsaka S, et al. (2007) Specific <br />     proteins gamma-adaptin VHS domains facilitate trafficking                    depletion GGA2 causes cathepsin D missorting HeLa cells. Arch Histol <br />     trans-Golgi network vacuole/lysosome. J Cell Biol 149: 67&#226;&#8364;&#8220; <br />                                                                                                Cytol 70: 303&#226;&#8364;&#8220;312. <br />     80. <br />                                                                                            24. Ghosh P, Griffith J, Geuze HJ, Kornfeld S (2003) Mammalian GGAs act <br />  7. Takatsu H, Yoshino K, Nakayama K (2000) Adaptor gamma ear homology <br />                                                                                                sort mannose 6-phosphate receptors. J Cell Biol 163: 755&#226;&#8364;&#8220;766. <br />     domain conserved gamma-adaptin GGA proteins interact with <br />     gamma-synergin. Biochem Biophys Res Commun 271: 719&#226;&#8364;&#8220;725.                               25. Hirst J, Sahlender DA, Choma M, Sinka R, Harbour ME, et al. (2009) Spatial <br />  8. Takatsu H, Katoh Y, Shiba Y, Nakayama K (2001) Golgi-localizing, gamma-                    Functional Relationship GGAs AP-1 inDrosophilaand HeLa Cells. <br />     adaptin ear homology domain, ADP-ribosylation factor-binding (GGA) proteins                Traffic 10: 1696&#226;&#8364;&#8220;1710. <br />     interact acidic dileucine sequences cytoplasmic domains             26. He X, Li F, Chang WP, Tang J (2005) GGA proteins mediate recycling <br />     sorting receptors Vps27p/Hrs/STAM (VHS) domains. J Biol                      pathway memapsin 2 (BACE). J Biol Chem 280: 11696&#226;&#8364;&#8220;11703. <br />     Chem 276: 28541&#226;&#8364;&#8220;28545.                                                                 27. Govero J, Doray B, Bai H, Kornfeld S (2012) Analysis Gga null mice <br />  9. Puertollano R, Aguilar RC, Gorshkova I, Crouch RJ, Bonifacino JS (2001)                    demonstrates non-redundant role mammalian GGA2 development. <br />     Sorting mannose 6-phosphate receptors mediated GGAs. Science 292:                PLoS One 7: e30184. <br />     1712&#226;&#8364;&#8220;1716.                                                                             28. Ghosh P, Kornfeld S (2003) Phosphorylation-induced conformational changes <br /> 10. Puertollano R, Bonifacino JS (2004) Interactions GGA3 ubiquitin                regulate GGAs 1 3 function trans-Golgi network. J Biol Chem 278: <br />     sorting machinery. Nat Cell Biol 6: 244&#226;&#8364;&#8220;251.                                               14543&#226;&#8364;&#8220;14549. <br /> 11. Shiba Y, Katoh Y, Shiba T, Yoshino K, Takatsu H, et al. (2004) GAT (GGA                29. Dennes A, Cromme C, Suresh K, Kumar NS, Eble JA, et al. (2005) The novel <br />     Tom1) domain responsible ubiquitin binding ubiquitination. J Biol              Drosophila lysosomal enzyme receptor protein mediates lysosomal sorting in <br />     Chem 279: 7105&#226;&#8364;&#8220;7111.                                                                       mammalian cells binds mammalian Drosophila GGA adaptors. J Biol <br /> 12. Bilodeau PS, Winistorfer SC, Allaman MM, Surendhran K, Kearney WR, et al.                  Chem 280: 12849&#226;&#8364;&#8220;12857. <br />     (2004) The GAT domains clathrin-associated GGA proteins                    30. Kametaka S, Sawada N, Bonifacino JS, Waguri S (2010) Functional <br />     ubiquitin binding motifs. J Biol Chem 279: 54808&#226;&#8364;&#8220;54816.                                    characterization protein-sorting machineries trans-Golgi network in <br /> 13. Bonifacino JS (2004) The GGA proteins: adaptors move. Nat Rev Mol                   Drosophila melanogaster. J Cell Sci 123: 460&#226;&#8364;&#8220;471. <br />     Cell Biol 5: 23&#226;&#8364;&#8220;32.                                                                    31. Kametaka S, Kametaka A, Yonekura S, Haruta M, Takenoshita S, et al. (2012) <br /> 14. Wasiak S, Legendre-Guillemin V, Puertollano R, Blondeau F, Girard M, et al.                AP-1 clathrin adaptor CG8538/Aftiphilin involved Notch signaling <br />     (2002) Enthoprotin: novel clathrin-associated protein identified                 eye development Drosophila melanogaster. J Cell Sci 125: 634&#226;&#8364;&#8220;648. <br />     subcellular proteomics. J Cell Biol 158: 855&#226;&#8364;&#8220;862.                                      32. Eissenberg JC, Ilvarsonn AM, Sly WS, Waheed A, Krzyzanek V, et al. (2011) <br /> 15. Mattera R, Arighi CN, Lodge R, Zerial M, Bonifacino JS (2003) Divalent                     Drosophila GGA model: ultimate gateway GGA analysis. Traffic 12: <br />     interaction GGAs Rabaptin-5-Rabex-5 complex. EMBO J 22:                    1821&#226;&#8364;&#8220;1838. <br />     78&#226;&#8364;&#8220;88.                                                                                 33. Hirst J, Carmichael J (2011) A potential role clathrin adaptor GGA in <br /> 16. Lui WW, Collins BM, Hirst J, Motley A, Millar C, et al. (2003) Binding partners            Drosophila spermatogenesis. BMC Cell Biol 12: 22. <br />     COOH-terminal appendage domains GGAs gamma-adaptin.                 34. Hummel T, Klambt C (2008) P-element mutagenesis. Methods Mol Biol 420: <br />     Mol Biol Cell 14: 2385&#226;&#8364;&#8220;2398.                                                               97&#226;&#8364;&#8220;117. <br /> 17. Zhu Y, Doray B, Poussu A, Lehto VP, Kornfeld S (2001) Binding GGA2 to <br />                                                                                            35. Chen H, Ma Z, Liu Z, Tian Y, Xiang Y, et al. (2009) Case studies ends-out <br />     lysosomal enzyme sorting motif mannose 6-phosphate receptor. <br />                                                                                                gene targeting Drosophila. Genesis 47: 305&#226;&#8364;&#8220;308. <br />     Science 292: 1716&#226;&#8364;&#8220;1718. <br />                                                                                            36. Schug J, Schuller WP, Kappen C, Salbaum JM, Bucan M, et al. (2005) Promoter <br /> 18. Puertollano R, Randazzo PA, Presley JF, Hartnell LM, Bonifacino JS (2001) <br />                                                                                                features related tissue specificity measured Shannon entropy. Genome <br />     The GGAs promote ARF-dependent recruitment clathrin TGN. Cell <br />     105: 93&#226;&#8364;&#8220;102.                                                                <br /> </body></html>