<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Ip et al. BMC Systems Biology 2011, 5:91 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                Open Access <br />  <br /> Analysis complex metabolic behavior through <br /> pathway decomposition <br /> Kuhn Ip1,2,3,4,5, Caroline Colijn6 Desmond S Lun1,2,3,4,5* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Understanding complex systems decomposition simple interacting components a <br />   pervasive paradigm modern science engineering. For cellular metabolism, complexity be <br />   reduced decomposition pathways particular biochemical functions, concept elementary <br />   flux modes provides systematic way organizing metabolic networks pathways. While decomposition <br />   using elementary flux modes proven powerful tool understanding manipulating cellular <br />   metabolism, utility, however, severely limited number modes network increases exponentially <br />   size. <br />   Results: Here, present new method decomposition metabolic flux distributions elementary flux <br />   modes. Our method easily operate large, genome-scale networks does require relevant <br />   modes metabolic network generated. We illustrate utility method metabolic <br />   engineering Escherichia coli understanding <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span> Mycobacterium tuberculosis (MTB) during <br />   infection. <br />   Conclusions: Our method achieve computational time improvements exceeding 2000-fold requires only <br />   seconds generate elementary mode decompositions genome-scale networks. These improvements <br />   arise having generate relevant elementary modes prior initiating decomposition. The <br />   decompositions method useful understanding complex flux distributions debugging genome- <br />   scale models. <br />  <br />  <br /> Background                                                                             These computational models analyzed num- <br /> Computational analysis cellular metabolism                                      ber ways. Flux-balance analysis (FBA) elementary <br /> recently gained increasing prominence importance.                                  mode analysis popular and <br /> In particular, genome-scale computational models cap-                                  powerful. <br /> turing stoichiometric thermodynamic constraints                                      FBA determines distribution steady-state reaction <br /> have published 30 organisms [1] ranging                                  fluxes satisfies constraints model and <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> relatively simple prokaryotes E. coli,                                 optimizes biological objective function as <br /> complex eukaryotes Homo sapiens [2,3]. The                                     biomass adenosine triphosphate (ATP) production <br /> application computational models dependent                                  [4]. With appropriate constraints biological objec- <br /> on accuracy tools developed ana-                               tive, FBA shown effective method for <br /> lysis. The maintenance model accuracy, debugging,                                prediction phenotypes associated genetic manip- <br /> is ongoing process model predictions                                   ulations knockouts [5,6] intracellular <br /> validated experimental observations discre-                                metabolic fluxes [7]. A significant application FBA, <br /> pancies identified corrected. This process                                  therefore, metabolic engineering-using computational <br /> clearly linked use analysis models.                                predictions metabolic <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> genetic <br />                                                                                        manipulations guide engineering metabolically <br /> * Correspondence: dslun@camden.rutgers.edu <br />                                                                                        optimized strains [8]-and computational methods have <br /> 1 <br />  Phenomics Bioinformatics Research Centre, University South                     devised search space genetic manipula- <br /> Australia, Mawson Lakes, SA 5095, Australia                                            tions silico yield desired phenotype <br /> Full list author information available end article <br />  <br />                                          &#194;&#169; 2011 Ip et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                          Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                          medium, provided original work properly cited. <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                      Page 2 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> [9-13]. But, FBA number successful         analysis, requiring prohibitive computation <br /> applications, method gives little insight pre-   elementary modes. Our method therefore <br /> dictions, hindering human understanding model             applicable genome-scale models and, can <br /> debugging.                                                    input flux distribution, con- <br />    Elementary mode analysis, hand, yields        nected particular experimental conditions [26] or <br /> no explicit predictions metabolic behavior, seeks      genetic modifications [10]. <br /> primarily allow understanding organism&#226;&#8364;&#8482;s meta-         <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> demonstrate utility method its <br /> bolic capabilities. In elementary mode analysis, ele-     applicability genome-scale models, apply gen- <br /> mentary flux modes (EFMs) define minimal sets         ome-scale models E. coli MTB its <br /> reactions capable operating steady state gener-     results used interpret flux distributions related <br /> ated [14]. Elementary flux modes formalize concept        metabolic engineering E. coli survi- <br /> of biochemical pathway [15], studying modes         val MTB infection. <br /> associated given metabolic network been <br /> shown effective understanding function,         Results Discussion <br /> regulation, robustness [16,17]. The principal draw-       Overview <br /> back elementary mode analysis number        Our method takes given steady-state metabolic flux <br /> EFMs network suffers combinatorial explo-         distribution corresponding metabolic model, <br /> sion [18], use complete sets EFMs gives         produces decomposition flux distribution <br /> rise problems scalability applied gen-        elementary flux modes. In paper, use flux <br /> ome-wide models [19]. For example, mil-         distributions obtained FBA, method can <br /> lion modes exist simple model E. coli central        equally applied flux distributions obtained by <br /> metabolism consisting 112 reactions [20],            alternative means, derived experi- <br /> increases 26 million modes pos-         mental measurement obtained genetic perturba- <br /> sible substrates expanded [21]. iAF1260 [6],          tion analysis methods MOMA [27] ROOM <br /> most recent genome-scale metabolic network E. coli,        [28]. As elementary flux mode itself set reac- <br /> consists 2077 reactions, number reactions       tions operating steady state, flux distribution <br /> in E. coli metabolic models increased steadily    composed entirely elementary flux modes must <br /> past decades [22]. Thus, computation time         necessarily operate steady state too. Therefore, an <br /> memory storage required enumerate EFMs          input flux distribution derived experimental mea- <br /> full detailed genome-scale metabolic networks         surement need balanced produce a <br /> prohibitively large.                                          steady state flux distribution, regression fit <br />    For applications, however, understanding       measured data alternative means. <br /> particular metabolic functions organism,          Our method operates selecting reaction <br /> its ability produce desired metabolite, greater    non-zero flux maximum magnitude the <br /> interest understanding metabolic         given flux distribution. The algorithm uses mixed- <br /> capabilities. In case, necessary know   integer linear programming elementary flux <br /> the EFMs network, simply combine        mode contains selected reaction is <br /> to rise particular behavior. Previous approaches    contained given distribution. The contribution of <br /> to problem relied computing modes      elementary flux mode given distribution is <br /> of network [23,24]. More recently, approach pre-       determined removed updated flux <br /> sented Figueiredo et al. [25], biologically       distribution. The updated flux distribution used the <br /> significant EFMs identifying K-shortest EFMs        input distribution iteration algorithm, <br /> the metabolic network necessarily enumerating         procedure repeated updated flux <br /> the set. Our motivation similar       distribution zero (see Methods details). <br /> not concerned generating EFMs net-            Elementary mode decompositions unique. Our <br /> work. However, approach differs greatly        goal assist biological interpretation, are <br /> wish determine elementary modes combine         primarily interested obtaining valid decomposition <br /> to yield given flux distribution. This flux distribution    specific decomposition. A valid decom- <br /> can consist measured computationally pre-         position arise irrespective choice at <br /> dicted fluxes.                                                iteration reaction non-zero flux, but <br />    We present algorithm elementary             choice determines specific decom- <br /> modes combine produce previously-specified        position. Our choice desirable properties. As <br /> flux distribution. This links advantages elemen-       elementary flux mode mixed-integer <br /> tary mode analysis advantages flux balance        linear program includes chosen reaction, flux <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                      Page 3 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> through reaction upper bound contribution        increasing size complexity genome-scale <br /> of elementary flux mode. By selecting reaction         models [22] exponential manner the <br /> with non-zero flux maximum magnitude, avoid              number elementary modes increases size of <br /> the possibility generating elementary flux             network, follows cases, advan- <br /> modes small weightings minimal             tages offered approach simply sev- <br /> contribution flux distribution. This choice        eral-thousand fold improvement computational time, <br /> minimizes numerical instabilities arising             difference practical feasibility <br /> calculations.                                                  infeasibility. <br />   Although mixed-integer linear programming NP-                Indeed, iNJ661v [31], recent model modifies <br /> hard general, large mixed-integer linear pro-          iNJ661 aim accurately modeling MTB <br /> grams (MILPs) solved modest            in vivo infection, slightly larger iNJ661, <br /> computation solvers SCIP [29], IBM ILOG             1,049 reactions. However, complex growth <br /> CPLEX (International Business Machines Corp.,                  medium resulted significantly larger number rele- <br /> Armonk, New York), Gurobi (Gurobi Optimization,            vant elementary modes flux distribution obtained <br /> Houston, Texas). In particular, algorithm, imple-          FBA iNJ661, unable to <br /> mented using Gurobi, successfully terminates        generate using efmtool memory lim- <br /> several seconds genome-scale applications           itations. Using decomposition method, generated <br /> mentioned paper.                                       valid elementary mode decomposition consisting 27 <br />   By contrast, previous approaches decomposing flux         modes 2.5 seconds. <br /> distributions elementary modes [23,24] relied <br /> on generating complete set relevant elemen-       Metabolic engineering E. coli <br /> tary modes, calculating weight distribution             To demonstrate utility approach meta- <br /> among modes. Since elementary mode decomposi-            bolic engineering, consider metabolic engineering <br /> tions unique [23], principal advantage          E. coli increased acetate production-a problem <br /> these previous approaches comparison        received attention owing value acetic <br /> they capable selecting particular unique decom-       acid industrial food uses [32]. <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>Knockout</span> stra- <br /> position based criterion (typically minimi-        tegies increased production acetate based FBA <br /> zation Euclidean norm weight vector),            previously reported Lun et al. [10] using <br /> while approach simply generates valid decomposi-         genome-scale metabolic reconstruction E. coli by <br /> tion possibilities. It clear, however,   Feist et al. [6], iAF1260. These knockout strategies were <br /> criteria selecting weight distribution,    generated using GDLS (Genetic Design Local <br /> minimization Euclidean norm, biologically           Search) [10], efficient heuristic generating meta- <br /> meaningful. As examples           bolic engineering strategies involving multiple knockouts <br /> paper, simply having valid decomposition itself        genome-scale models, extending capabilities of <br /> useful.                                                        computationally-expensive optimal search proposed <br />   The principal drawback previous approaches          OptKnock [9]. The strategies chosen using FBA <br /> is generating complete set relevant elemen-        high predicted production acetate whilst <br /> tary modes prohibitive, particularly genome-        maintaining required energy production growth to <br /> scale models. To demonstrate fact, used efmtool        ensure viability. Specifically, predicted non-growth- <br /> [21] efficiently generate relevant elementary           associated ATP maintenance (NGAM) flux least <br /> modes genome-scale model MTB Jamshidi            8.39 h -1 biomass flux 0.05 h -1 were <br /> and Palsson [30], iNJ661, growth Middlebrook            required. <br /> 7H9-a standard growth medium MTB. With ele-              The proposed strategies make sense biologically [10] <br /> mentary modes generated efmtool, applied            include experimentally-tested knockouts acetate <br /> the quadratic programming approach proposed                 production alcohol dehydrogenase ATP <br /> Schwartz Kanehisa [23] obtain distribution         synthase [32]. However, cause increased acet- <br /> weights assign modes. The total computa-           ate production immediately clear meta- <br /> tional time approach 34 minutes 27             bolic phenotype predicted FBA alone. Table 1 shows <br /> seconds.                                                       number reactions added or <br />   By contrast, approach generates valid elementary       removed flux distributions gene-knock- <br /> mode decomposition consisting 19 modes 1.0 sec-          mutants compared wild-type flux dis- <br /> onds-a computational time improvement exceeding                tribution. As see, changes flux <br /> 2000-fold. iNJ661 consists 1,028 reactions,        distributions quite extensive and, particular, they <br /> modest current genome-scale models [1]. With           result simply shunts blockages <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                            Page 4 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> Table 1 Number changes reactions used E. coli knockout mutant compared wild-type (WT) <br /> Gene Knockouts                                                 1          2        3        4        5        6        8 <br /> Reactions zero flux WT non-zero flux mutant     12         12       22       25       25       29       36 <br /> Reactions non-zero flux WT zero flux mutant     9          9        16       21       23       17       24 <br />  <br />  <br /> created gene deletions. The inherent structure            demand chosen modes need complement <br /> the network, mechanisms pre-                 well. This seen reported <br /> dicted metabolic behavior arises, difficult ascertain         decompositions various knockout mutants. <br /> from flux distribution whole. We, therefore,                 For example, biomass-producing modes, the <br /> applied decomposition algorithm determine                mode arising five-knockout mutant most <br /> elementary flux modes make flux distribu-               efficient 1.496 mmol acetate mmol glucose. <br /> tions knockout strategies. We took best             When knockouts allowed, overall acetate <br /> knockout strategies reported Lun et al. [10]                production improved despite decrease acetate <br /> numbers knockouts ranging 1 8. For pur-              production efficiency biomass-producing mode. <br /> poses strategies, single knockout considered          This decrease offset shift using NGAM- <br /> to consist genes capable catalyzing reaction, i.        producing modes significantly efficient pro- <br /> e. enzyme complex complexes                duction acetate. For knockout cases, <br /> metabolic function.                                                 NGAM-producing mode generates 1.690 mmol of <br />    Table 2 shows elementary flux modes obtained.             acetate mmol glucose. <br /> The flux distribution knockout mutant                 Finally, FBA does necessarily yield unique dis- <br /> decomposed elementary modes,                tribution, implemented recursive MILP algorithm <br /> supply energy growth requirements                    Lee et al. [33] alternate optima then <br /> organism. We emphasize elementary mode decom-                  obtained decompositions corresponding flux dis- <br /> position identify structural components           tributions. Our results (not shown) indicate the <br /> important metabolic phenotype. The focus              decomposition modes distinct functions <br /> NGAM biomass components reflects biologi-                 related NGAM biomass production preserved <br /> cal importance underlying biology model.              alternate optima. Thus, decomposition flux <br /> The mode contributes required NGAM                distributions primarily NGAM-producing bio- <br /> flux, uses relatively reactions. In comparison,             mass-producing modes robust quality. <br /> the second mode solely responsible producing bio- <br /> mass involves active reactions. This cor-             Understanding survival MTB infection <br /> responds large number metabolites required              To illustrate application approach, we <br /> for biomass production. Furthermore, noted Stel-              consider utilization glyoxylate shunt MTB. <br /> ling et al. [16], unmodified E. coli metabolic network          The glyoxylate shunt enzyme isocitrate lyase (ICL), pre- <br /> displays degree robustness evidenced var-            sent MTB isoforms, believed required <br /> ied pathways biomass NGAM pro-                     microorganisms utilize fatty acids source of <br /> duced. The knockouts force flux alternative                    carbon energy. This shunt previously been <br /> pathways producing necessary metabolic com-               shown required vivo growth viru- <br /> ponents, pathways produce acetate side-              lence MTB [34]. Since MTB believed subsist on <br /> product. By finding examining elementary                    fatty acids infection [34,35], argued that, by <br /> modes associated acetate-enhancing knockout                removing ICL, MTB longer able utilize fatty <br /> strategies, uncover mechanism responsible                acids carbon energy unable to <br /> the increased acetate flux, apparent             grow vivo. Indeed, strains MTB absent iso- <br /> the FBA analysis.                                                   forms ICL unable grow fatty acid substrates <br />    Most interestingly, modes examined               unable survive macrophages mice [34]. <br /> terms acetate production, effi-            Therefore, ICL attracted significant attention a <br /> cient modes generally contribute             promising drug target treatment MTB infection <br /> NGAM flux. However, biomass-producing mode                     [36]. <br /> necessary satisfy growth viability requirements.               It possible, however, given robustness that <br /> Thus, problem selecting knockouts maximize                generally observed metabolic networks [16], a <br /> acetate production, given limiting resource               vital function simply rest single enzyme- <br /> glucose, necessarily finding single optimal          present isoforms. Using method, <br /> elementary mode. Rather, competing constraints                      study metabolic capabilities MTB growing <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                       Page 5 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> Table 2 Elementary modes acetate-producing E. coli knockout strategiesa <br /> Knockouts (Number brackets,            Mode Weight Overall conversion (Acetate production bold)         Biomass NGAM        Number <br /> Separated semi-colons)                                                                                     (g/mmol (mmol/      of <br />                                                                                                               Glc)    mmol        reactions <br />                                                                                                                       Glc) <br /> (0) None                                    1     2.92   1.000 Glc + 0.005 K + 0.315 NH4 + 0.028 Pi + 0.007   0.029   0.2         412 <br />                                                          SO4 &#194;&#174; 0.785 Ac + 1.661 CO2 + 0.766 EtOH + 1.073 <br />                                                          H + 1.675 H2 + 0.010 Succ <br />                                             2     7.08   1.000 Glc + 0.297 H2O + 0.004 K + 0.222 NH4 + 0.020 0.021    1.1         414 <br />                                                          Pi + 0.005 SO4 &#194;&#174; 0.849 Ac + 1.762 CO2 + 0.835 <br />                                                          EtOH + 1.051 H + 1.772 H2 + 0.007 Succ <br />                                             Combined     10.000 Glc + 2.101 H2O + 0.041 K + 0.002 Mg2 +       0.23    8.4         415 <br />                                                          2.492 NH4 + 0.222 Pi + 0.058 SO4 &#194;&#174; 8.301 Ac + <br />                                                          17.322 CO2 + 8.152 EtOH + 10.578 H + 17.434 H2 + <br />                                                          0.077 Succ <br /> (1) frmA, adhE, adhP                        1     4.18   1.000 Glc + 0.667 H2O &#194;&#174; 0.667 12PPD-R + 1.333 Ac     0       2           38 <br />                                                          + 1.333 CO2 + 1.333 H + 1.333 H2 <br />                                             2     5.82   1.000 Glc + 0.004 K + 0.228 NH4 + 0.020 Pi + 0.005   0.021   0.0045      414 <br />                                                          SO4 &#194;&#174; 0.554 12PPD-R + 1.121 Ac + 1.201 CO2 + <br />                                                          1.330 H + 1.211 H2 + 0.007 Succ <br />                                             Combined     10.000 Glc + 2.788 H2O + 0.022 K + 1.327 NH4 +       0.12    8.4         418 <br />                                                          0.118 Pi + 0.031 SO4 &#194;&#174; 6.011 12PPD-R + 12.101 Ac <br />                                                          + 12.563 CO2 + 13.313 H + 12.623 H2 + 0.041 Succ <br /> (2) mhpF, adhE; ydfG                        1     4.07   1.000 Glc + 0.750 H2O &#194;&#174; 1.500 Ac + 1.500 CO2 +       0       2.1         54 <br />                                                          0.750 EtOH + 1.500 H + 1.500 H2 <br />                                             2     5.93   1.000 Glc + 0.047 H2O + 0.004 K + 0.237 NH4 + 0.021 0.022    0           417 <br />                                                          Pi + 0.005 SO4 &#194;&#174; 1.296 Ac + 1.306 CO2 + 0.586 <br />                                                          EtOH + 1.513 H + 1.353 H2 + 0.007 Succ <br />                                             Combined     10.000 Glc + 3.331 H2O + 0.023 K + 0.001 Mg2 +       0.13    8.4         418 <br />                                                          1.406 NH4 + 0.125 Pi + 0.033 SO4 &#194;&#174; 13.791 Ac + <br />                                                          13.848 CO2 + 6.529 EtOH + 15.075 H + 14.128 H2 + <br />                                                          0.044 Succ <br /> (3) atpABCDEFGH; frmA, adhE, adhP; pgi      1     2.82   1.000 Glc + 0.444 H2O + 0.167 SO4 &#194;&#174; 0.444 12PPD-R    0       1.3         59 <br />                                                          + 1.556 Ac + 1.556 CO2 + 1.222 H + 1.556 H2 + <br />                                                          0.167 H2S <br />                                             2     7.18   1.000 Glc + 0.233 H2O + 0.001 K + 0.077 NH4 + 0.007 0.0071   0.64        418 <br />                                                          Pi + 0.159 SO4 &#194;&#174; 0.408 12PPD-R + 1.495 Ac + 1.486 <br />                                                          CO2 + 1.251 H + 1.501 H2 + 0.157 H2S + 0.002 Succ <br />                                             Combined     10.000 Glc + 2.926 H2O + 0.009 K + 0.552 NH4 +       0.051   8.4         421 <br />                                                          0.049 Pi + 1.609 SO4 &#194;&#174; 4.181 12PPD-R + 15.118 Ac <br />                                                          + 15.056 CO2 + 12.431 H + 15.166 H2 + 1.596 H2S + <br />                                                          0.017 Succ <br /> (4) atpABCDEFGH; frmA, adhE, adhP; pgi;     1     2.91   1.000 Glc + 0.444 H2O + 0.167 SO4 &#194;&#174; 0.444 12PPD-R    0       1.3         59 <br /> yahI, ybcF, yqeA                                         + 1.556 Ac + 1.556 CO2 + 1.222 H + 1.556 H2 + <br />                                                          0.167 H2S <br />                                             2     7.09   1.000 Glc + 0.233 H2O + 0.001 K + 0.077 NH4 + 0.007 0.0071   0.64        416 <br />                                                          Pi + 0.159 SO4 &#194;&#174; 0.408 12PPD-R + 1.495 Ac + 1.486 <br />                                                          CO2 + 1.251 H + 1.501 H2 + 0.157 H2S + 0.002 Succ <br />                                             Combined     10.000 Glc + 2.946 H2O + 0.009 K + 0.545 NH4 +       0.051   8.4         419 <br />                                                          0.049 Pi + 1.609 SO4 &#194;&#174; 4.185 12PPD-R + 15.124 Ac <br />                                                          + 15.062 CO2 + 12.428 H + 15.171 H2 + 1.597 H2S + <br />                                                          0.017 Succ <br /> (5) atpABCDEFGH; frmA, adhE, adhP; pgi;     1     2.87   1.000 Glc + 0.444 H2O + 0.167 SO4 &#194;&#174; 0.444 12PPD-R 0          1.3         59 <br /> tnaA; yahI, ybcF, yqeA                                   + 1.556 Ac + 1.556 CO2 + 1.222 H + 1.556 H2 + <br />                                                          0.167 H2S <br />                                             2     7.13   1.000 Glc + 0.235 H2O + 0.001 K + 0.076 NH4 + 0.007 0.0071   0.64        413 <br />                                                          Pi + 0.158 SO4 &#194;&#174; 0.408 12PPD-R + 1.496 Ac + 1.486 <br />                                                          CO2 + 1.252 H + 1.501 H2 + 0.157 H2S + 0.002 Succ <br />                                             Combined     10.000 Glc + 2.949 H2O + 0.009 K + 0.544 NH4 +       0.05    8.4         417 <br />                                                          0.049 Pi + 1.607 SO4 &#194;&#174; 4.185 12PPD-R + 15.128 Ac <br />                                                          + 15.057 CO2 + 12.435 H + 15.168 H2 + 1.595 H2S + <br />                                                          0.017 Succ <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                                   Page 6 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> Table 2 Elementary modes acetate-producing E. coli knockout strategiesa (Continued) <br /> (6) atpABCDEFGH; galP; mhpF, adhE; pgi;          1        6.53    1.000 Glc + 0.762 H2O &#194;&#174; 0.071 12PPD-R + 1.690 Ac              0    1.3      77 <br /> pitAB; ydfG                                                       + 1.357 CO2 + 0.524 EtOH + 1.690 H + 1.524 H2 <br />                                                  2        3.47    1.000 Glc + 0.280 H2O + 0.003 K + 0.156 NH4 + 0.014 0.014          0        424 <br />                                                                   Pi + 0.004 SO4 &#194;&#174; 0.114 12PPD-R + 1.404 Ac + 1.304 <br />                                                                   CO2 + 0.467 EtOH + 1.546 H + 1.388 H2 + 0.005 Succ <br />                                                   Combined        10.000 Glc + 5.945 H2O + 0.009 K + 0.542 NH4 +     0.05            8.4      427 <br />                                                                   0.048 Pi + 0.013 SO4 &#194;&#174; 0.863 12PPD-R + 15.908 Ac <br />                                                                   + 13.386 CO2 + 5.042 EtOH + 16.403 H + 14.766 H2 + <br />                                                                   0.017 Succ <br /> (8) (sapD trkA trkG), (sapD trkA        1        6.53    1.000 Glc + 0.762 H2O &#194;&#174; 0.071 12PPD-R + 1.690 Ac              0    1.3      77 <br /> or trkH), kch, kup; atpABCDEFGH; galP;                            + 1.357 CO2 + 0.524 EtOH + 1.690 H + 1.524 H2 <br /> guaB; mhpF, adhE; pgi; pitAB; ydfG <br />                                                  2        3.47    1.000 Glc + 0.282 H2O + 0.003 K + 0.155 NH4 + 0.014 0.014          0        424 <br />                                                                   Pi + 0.004 SO4 &#194;&#174; 0.114 12PPD-R + 1.406 Ac + 1.304 <br />                                                                   CO2 + 0.468 EtOH + 1.547 H + 1.389 H2 + 0.005 Succ <br />                                                   Combined        10.000 Glc + 5.953 H2O + 0.009 K + 0.539 NH4 +     0.05            8.4      427 <br />                                                                   0.048 Pi + 0.013 SO4 &#194;&#174; 0.861 12PPD-R + 15.915 Ac <br />                                                                   + 13.387 CO2 + 5.043 EtOH + 16.407 H + 14.769 H2 + <br />                                                                   0.017 Succ <br /> a <br /> Metabolite abbreviations: 12PPD-R, (R)-Propane-1,2-diol; Ac, acetate; EtOH, ethanol; Glc, glucose; Pi, phosphate; Succ, succinate. <br />  <br />  <br /> on fatty acid substrates. In particular, determined                          deletion strains E. coli change significantly after <br /> elementary modes used MTB differing uptake                                    undergoing adaptive evolution. The possible existence of <br /> rates octadecenoate using genome-scale metabolic                             inactive routes metabolizing fatty acids without <br /> reconstruction MTB Beste et al. [37] (see Figure                              use ICL MTB discussed elsewhere <br /> 1). We exist modes generate bio-                              [39] and, true, imply MTB rapidly <br /> mass and/or NGAM using using ICL. At                                 evolve resistance drugs inhibiting ICL. <br /> given NGAM flux, modes use ICL generally                                     Closer examination elementary modes reveals <br /> produce biomass efficiently                                 MTB grows silico ICL provides a <br /> not. However, given uptake octadecanoate,                              testable means confirming rejecting model&#226;&#8364;&#8482;s <br /> modes produce biomass                               predictions. All elementary modes use <br /> not use ICL. Therefore, modes available maximize                             ICL use malate synthase, second enzymes <br /> the biomass production maintaining given                                    form glyoxylate shunt. In modes, <br /> NGAM requirement depend supply flux                                  flux malate synthase comes HtrA, <br /> octadecenoate. When supply sufficiently high,                            gene predicted <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> 4-hydroxy-2-oxoglutarate <br /> NGAM requirement easily met high efficiency                               aldolase complete hydroxyproline degradation <br /> biomass producing modes use ICL,                                    pathway MTB using Bayesian method Green <br /> when lower, use ICL allows NGAM require-                               Karp [40]. HtrA supplies glyoxylate used <br /> ment met efficiently and, hence, allows                             substrate malate synthase. MTB other <br /> biomass produced.                                                             mycobacteria observed grow using hydro- <br />    Under assumption metabolic reconstruc-                              xyproline carbon source [41,42], suggesting that <br /> tion Beste et al. correct, analysis implies                          pathway exist. Further studies confirm- <br /> MTB capable producing carbon energy                                  ing characterizing pathway shed light on <br /> from fatty acids use ICL but, lower                              does fact provide MTB viable <br /> uptake rates fatty acids, ICL allows efficient                          means producing glyoxylate. <br /> utilization fatty acids. Indeed,                             The elementary mode decomposition analysis have <br /> optimal growth rate predicted FBA differs                                   presented sheds light mechanics FBA pre- <br /> slightly ICL present (see Figure 1d).                           diction difficult obtain FBA results <br /> This presents intriguing possibility MTB pos-                              alone. Specifically, HtrA combination malate <br /> sesses metabolic capability grow fatty acids                              synthase used GSMN-TB generate biomass <br /> without ICL, does knockout                               NGAM. However, space possible biomass <br /> ICL undergone adaptive evo-                              NGAM production rates achieved in <br /> lution necessary make use metabolic capabil-                             way smaller possible using the <br /> ity. This possibility consistent work Fong                           glyoxylate shunt. At low octadecenoate uptake rates, this <br /> and Palsson [38] showing growth rate gene                               difference important, leading lower biomass <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                                                            Page 7 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br />                                                                                                b <br />                                           0.08                                                                            0.08 <br />  <br />  <br />                                           0.06                                                                            0.06 <br />        Weight <br />  <br />  <br />  <br />  <br />                                                                                                        Weight <br />                                           0.04                                                                            0.04 <br />  <br />  <br />                                           0.02                                                                            0.02 <br />  <br />  <br />                                              0                                                                               0 <br />                                                  0        0.02     0.04     0.06          0.08                                   0        0.02     0.04     0.06          0.08 <br />                                                                                     &#226;&#710;&#8217;1    &#226;&#710;&#8217;1                                                                        &#226;&#710;&#8217;1    &#226;&#710;&#8217;1 <br />                                                      Octadecenoate uptake (mmol g        h )                                         Octadecenoate uptake (mmol g        h ) <br />  <br />  <br />  c                                                                                               d <br />                                           0.12                                                                              0.01 <br />        Biomass (g / mmol Octadecenoate) <br />  <br />  <br />  <br />  <br />                                            0.1                                                                            0.008 <br />                                                                                                          ) <br />                                                                                                       &#226;&#710;&#8217;1 <br />                                                                                                          Growth rate (h <br />                                           0.08 <br />                                                                                                                           0.006 <br />                                           0.06 <br />                                                                                                                           0.004 <br />                                           0.04 <br />                                                                                                                           0.002 <br />                                           0.02 <br />  <br />                                              0                                                                                   0 <br />                                                  0      50     100      150       200                                                0      0.02    0.04    0.06          0.08 <br />                                                                                                                                                                     &#226;&#710;&#8217;1 <br />                                                  NGAM (mmol / mmol Octadecenoate)                                                    Octadecenoate uptake (mmol g        h &#226;&#710;&#8217;1) <br />  Figure 1 Elementary modes used MTB growing octadecenoate sole carbon source ICL. Five unique <br />  elementary modes identified overall applying decomposition method representative flux distributions, elementary <br />  modes used characterize optimal metabolic behavior octadecenoate uptake varying 0 0.08 mmol g-1 h-1. The weights these <br />  elementary modes (colored blue, green, red, cyan, magenta) octadecenoate uptake varies shown stacked (a) (b) <br />  ICL present. The modes colored blue green use ICL, remainder not. Each mode normalized the <br />  octadecenoate uptake mode 1 mmol g-1 h-1. (c) The biomass NGAM generated mode 1 mmol octadecenoate. The <br />  region enclosed solid line space achievable pairs biomass NGAM ICL present, region enclosed the <br />  dotted line ICL present. When octadecenoate plentiful, cyan magenta modes used meet NGAM <br />  requirement produce biomass; octadecenoate limited, remaining modes needed meet NGAM requirement <br />  and, ICL present, achieved efficiently. (d) FBA-predicted growth rate varying octadecenoate uptake ICL present <br />  (solid line) (dotted line). <br />  <br />  <br /> production meet NGAM requirement. Further-                                                modes. In contrast previous approaches, method <br /> more, demonstrates utility method                                          does require initial generation set of <br /> identifying potential source discrepancy                                        elementary modes, computationally <br /> in silico predictions observed experimental results.                                         demanding and, cases, computationally infeasi- <br /> If growth experimentally possible,                                             ble practical purposes. In moderately-sized <br /> adaptive evolution, suggests model                                           instance, observed computational time improve- <br /> incorrect likely error model comes                                           ment 2000-fold using method. <br /> the inclusion predicted gene, HtrA.                                                         Overall, elementary mode analysis offers a <br />                                                                                                  great deal flux-balance analysis does not. FBA <br /> Conclusions                                                                                      yields predictions overall metabolic behavior, while <br /> We presented method decomposing given                                               elementary mode analysis allows understanding meta- <br /> flux distribution set constituent elementary                                           bolic capabilities. By decomposing flux distributions <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                            Page 8 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> obtained FBA elementary modes, gain               <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> <br /> insight metabolic networks achieve pre-           We say flux mode p conforms flux distribution v if <br /> dictions. We exploit modularity decompose com-              &#206;&#189;j &gt; 0 reactions j pj &gt; 0 &#206;&#189;j &lt; 0 all <br /> plex entity simpler entity, enables                 reactions j pj &lt; 0. <br /> debugging understanding and, ultimately,                  Our finding elementary modes con- <br /> sophisticated design engineering.                            form given flux distribution v ensures that <br />                                                                  v decomposed elementary modes reac- <br /> Methods                                                          tions flowing direction v, pur- <br /> Genome-scale FBA modeling                                        poses biological interpretation. <br /> We work genome-scale model E. coli,                    Our algorithm takes input flux distribution v in <br /> iAF1260. This model consists parts. First,         feasible set optimization problem (1) outputs <br /> m metabolites n reactions, form m &#195;&#8212; n stoi-            elementary mode decomposition v, i.e. set of <br /> chiometric matrix S, ijth element Sij stoi-         elementary flux modes {p (k) } conform v a <br /> chiometric coefficient metabolite reaction j.            corresponding set positive weights {w k } that <br /> Second, flux distribution v, jth element vj              K <br />  <br /> the flux reaction j, constrained lower            v=      wk p(k). Suppose flux distribution v that <br /> bound vector a, jth element j lower                    k=1 <br />  <br /> bound flux reaction j, upper               satisfies Sv = 0. Let k: = 1, v(k): = v. Choose jk <br />                                                                               (k) <br /> bound vector b, jth element bj &gt; 0 upper            vjk  = 0, solve following mixed- <br /> bound flux reaction j. Finally, linear        integer linear program (MILP): <br /> objective formed multiplying fluxes                               n <br /> objective vector f, jth element fj weight            min            j=0 qj <br /> reaction j biological objective. Thus, biologically         subject Sp = 0, <br />                                                                                              (k) <br /> optimal flux distribution obtained solving                                 pjk = sgn(vjk ), <br />                                                                                  &#226;&#710;&#8217;Mqj &#226;&#8240;&#164; pj &#226;&#8240;&#164; Mqj , j = 1, . . . , n, <br />    max        f  v                                                               pj = 0,           j &#226;&#710;&#710; {j|vj = 0}, <br />                                                                                                                          (2) <br />    subject Sv = 0,                                      (1) <br />                                                                                  pj &#226;&#8240;&#164; 0,           j &#226;&#710;&#710; {j|vj &lt; 0}, <br />               &#226;&#8240;&#164; v &#226;&#8240;&#164; b.                                                         pj &#226;&#8240;&#165; 0,           j &#226;&#710;&#710; {j|vj &gt; 0}, <br />                                                                                  qj &#226;&#710;&#710; {0, 1},      j = 1, . . . , n, <br />  <br />                                                                     M large constant sgn sign func- <br /> Elementary mode decomposition <br />                                                                  tion, taking value 1 argument positive -1 <br /> For given flux vector &#206;&#189;, use R(&#206;&#189;) = {i:&#206;&#189; &#226;&#8240;&#160; 0}to <br />                                                                  argument negative. This MILP similar that <br /> denote set indices reactions participating in <br />                                                                  used Figueiredo et al. [25] finding shortest <br /> &#206;&#189;. We define elementary flux mode using follow- <br />                                                                  elementary modes metabolic network. Our purpose, <br /> ing definitions [4]. <br />                                                                  however, differs significantly: seek decompose a <br /> <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> <br />                                                                  given flux distribution constituent elementary <br /> A flux mode, mode, metabolic network specified <br />                                                                  modes. The specific choice j k unimportant: all <br /> by stoichiometric matrix S lower upper                                     (k) <br /> bound constraints b non-zero n &#195;&#8212; 1 vector p           choices vjk  = 0 generate valid decompo- <br /> satisfying following conditions:                         sition, specific decomposition achieved will <br />                                                                  likely vary choice. As discussed overview, <br />     1. valid steady-state flux distribution, i.e. Sp =   choose jk = argmaxj|vj|. <br />     0;                                                              Let (p*,q*) optimal solution. We set p(k): = <br />     2. irreversible reactions flow right direction, i.                                  (k)   (k) <br />                                                                  p* wk := minj&#226;&#710;&#710;{j|p(k)  =0} (vj /pj ). Finally, set v(k   + <br />     e. j aj &#226;&#8240;&#165; 0, pj &#226;&#8240;&#165; 0.                                      j <br />                                                                  1) <br />                                                                    : = v(k) - wkp(k). If v(k + 1) zero vector, we <br /> <span id='am-319' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> <br />                                                                  done. Otherwise, choose j k + 1 that <br />                                                                   (k+1) <br /> We say flux mode elementary minimal          vjk+1  = 0, increment k one, repeat above <br /> all flux modes, i.e. does exist flux         procedure. <br /> mode R(p&#226;&#8364;&#8482;) &#226;&#352;&#8218; R(p).                                        The following proposition establishes algo- <br />   Before stating algorithm, require           rithm generates desired output. In brief, algo- <br /> <span id='am-320' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>.                                                      rithm works because, iteration, MILP finds a <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                 Page 9 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> flux mode pjk non-zero number                    It straightforward wk &gt; 0 owing defini- <br /> of non-zero elements flux mode minimized.                  tion that, iteration algorithm, |R(v(k))| <br /> Because number non-zero elements flux                  decrease 1, i.e. |R(v(k + 1))| &lt; |R(v(k))|. Thus <br /> mode minimized, flux mode minimal and,                     algorithm terminate finite number itera- <br /> hence, elementary. Since reaction non-zero                 tions K &#226;&#8240;&#164; |R(v)|. &#226;&#8211;&#161; <br /> flux participate elementary mode in <br /> the decomposition, does matter jk chosen,              Characterization optimal metabolic behavior using <br />             (k) <br /> as long vjk  = 0. A valid decomposition gener-            given elementary modes <br /> ated regardless jk chosen, particu-             When calculating elementary mode decompositions <br /> lar decomposition generated non-                   range related flux distributions, MTB <br /> unique possibilities depend choice.                     application, helpful use subset ele- <br /> Proposition                                                          mentary modes obtained, likely subset of <br /> The elementary mode decomposition algorithm stated                   modes suffices generate valid decompositions for <br /> above terminates finite number iterations K               distributions. To so, select subset of <br /> and generates set elementary flux modes {p(k)}             K modes {p(1), ..., p(k)} obtained use <br /> conform v corresponding set positive                     following approach determine suffice to <br />                              <br />                            K                                         support flux distributions. We successively <br /> weights {wk} v =    wk p(k).                               remove modes subset arrive is <br />                                  k=1                                 minimal sense additional modes be <br /> Proof                                                                removed support flux distributions. <br /> First, establish p (k) fact mode,              We represent elementary flux mode column <br /> observe p(k) generated solution problem             vector matrix P =[p (1) ... p (K) ] define non- <br /> (2) meet steady state condition   the system.            negative weight vector w = [w1, ..., wK] flux <br />                                               (k)                    distribution v = Pw. Substitution v = Pw (1) <br /> Problem (2) solution p = v(k) /  vjk   q such <br />                                                                      gives means finding biologically-optimal weight <br /> that qj = 1 pj &#226;&#8240;&#160; 0 qj = 0 feasible <br />                                                                      vector given set elementary flux modes. Spe- <br /> set problem. Further, constraining compo- <br />                                                                      cifically, solve <br /> nents p(k) sign corresponding <br /> elements v, ensure irreversible reactions flow               max        f   Pw <br /> in right direction since, j, aj &#226;&#8240;&#165; 0 vj &#226;&#8240;&#165; 0,         subject &#226;&#8240;&#164; Pw &#226;&#8240;&#164; b,                                    (3) <br /> which sets constraint pj &#226;&#8240;&#165; 0 problem (2).                                   w &#226;&#8240;&#165; 0. <br />   Second, constraints problem (2), can <br /> see p(k) conforms v.                                           If biomass derived solving (3) corresponds <br />   Lastly, establish p(k) minimal           (1), conclude given elemen- <br /> flux modes conforming v and, therefore, elementary                tary modes sufficient characterize flux distri- <br /> in set modes. We observe              bution interest, given modes utilized <br /> optimal cost problem (2) iteration k |R (p(k))|.            according weights w* obtained optimal <br /> Suppose exists mode p&#226;&#8364;&#8482; conforms v               solution (3). <br /> R(p&#226;&#8364;&#8482;) &#226;&#352;&#8218; R(p(k)). If p jk  = 0, assume loss of <br />                                                                      Implementation FBA elementary mode <br /> generality p jk = 1, (p&#226;&#8364;&#8482;,q&#226;&#8364;&#8482;), q&#226;&#8364;&#8482;j = 1 p&#226;&#8364;&#8482;j &#226;&#8240;&#160; 0      decomposition <br /> and q&#226;&#8364;&#8482;j = 0 otherwise,      feasible set problem <br />            n                     (k)                                 FBA elementary mode decomposition method <br /> (2)           q =  R(p )  &lt;  R(p ) , contradiction <br />                      <br />                                                                      implemented using MATLAB R2010b Gurobi <br />                   j=1 j <br />                                                                      4.0. This implementation available additional file 1. <br /> with p   (k) <br />                optimal solution. If p jk = 0, let p&#226;&#8364;? <br />                                                 (k)   (k) <br /> = p(k) - wp&#226;&#8364;&#8482;, wk := minj&#226;&#710;&#710;{j|p(k)  =0} (vj /pj ). Now p&#226;&#8364;?        Comparison previous decomposition methods <br />                                          j <br />  <br /> is mode conforms v R(p&#226;&#8364;&#8482;) &#226;&#352;&#8218; R(p )         (k)      We used Gurobi 4.0 solve optimization problem (1) <br /> (p&#226;&#8364;&#339;, q&#226;&#8364;?), q&quot;j = 1 p&quot;j &#226;&#8240;&#160; 0 q&quot;j = 0 otherwise,         biologically optimal flux distribution iNJ661 <br /> in         feasible   set           problem      (2)          growing Middlebrook 7H9. The resulting distribution <br />      n                   (k)                                         v contained 507 non-zero components. The reactions j <br /> and                     <br />            q = R(p ) &lt;  R(p ) , resulting same <br />           j=1 j                                                      vj = 0 removed metabolic net- <br /> contradiction. Hence does exist mode p&#226;&#8364;&#8482;             work, generating reduced S used input to <br /> conforms v R(p&#226;&#8364;&#8482;) &#226;&#352;&#8218; R(p(k)), conclude             efmtool. efmtool version 4.7.1 used default <br /> p(k) elementary.                                                  parameters MATLAB R2010b generate the <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                                        Page 10 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br /> elementary modes reduced metabolic network,                            2.    Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, <br />                                                                                      Palsson B&#195;&#732;: Global reconstruction human metabolic network <br /> resulting 507 &#195;&#8212; 131,558 matrix P containing                             based genomic bibliomic data. Proceedings National <br /> elementary modes. Finally, quadratic program                                     Academy Sciences 2007, 104:1777-1782. <br />                K      2 <br />                                                                                3.    Ma H, Sorokin A, Mazein A, Selkov A, Selkov E, Demin O, Goryanin I: The <br />    min                                                                               Edinburgh human metabolic network reconstruction functional <br />                 k=1 wk <br />                                                                                      analysis. Mol Syst Biol 2007, 3. <br />    subject Pw = v,                                                          4.    Kauffman KJ, Prakash P, Edwards JS: Advances flux balance analysis. <br />               w &#226;&#8240;&#165; 0,                                                                 Curr Opin Biotechnol 2003, 14:491-496. <br />                                                                                5.    Edwards JS, Ibarra RU, Palsson B&#195;&#732;: In silico predictions Escherichia coli <br />   proposed Schwartz Kanehisa [23] solved                               metabolic capabilities consistent experimental data. Nat <br /> using MOSEK 6.0.0.91 (MOSEK ApS, Copenhagen,                                         Biotechnol 2001, 19:125-130. <br />                                                                                6.    Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> Denmark).                                                                            Broadbelt LJ, Hatzimanikatis V, Palsson B&#195;&#732;: A genome-scale metabolic <br />   efmtool generated 131,558 elementary modes                                 reconstruction Escherichia coli K-12 MG1655 accounts 1260 <br /> the network 1 minutes 48 seconds, quadratic                             ORFs thermodynamic information. Mol Syst Biol 2007, 3:121. <br />                                                                                7.    Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objective <br /> optimization step took 32 minutes 39 seconds,                                    functions predicting intracellular fluxes Escherichia coli. Mol Syst <br /> resulting total computational time 34 minutes                            Biol 2007, 3:119. <br /> 27 seconds. Computations carried Mac                           8.    Alper H, Miyaoku K, Stephanopoulos G: Construction lycopene- <br />                                                                                      overproducing E. coli strains combining systematic combinatorial <br /> OS X 10.6.4 platform using Intel                                  <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> targets. Nat Biotechnol 2005, 23:612-616. <br /> Core 2 Duo 2.53 GHz processor 4 GB RAM.                                9.    Burgard AP, Pharkya P, Maranas CD: OptKnock: A bilevel programming <br />   For iNJ661v, biologically optimal flux distribution                            framework identifying gene knockout strategies microbial strain <br />                                                                                      optimization. Biotechnol Bioeng 2003, 84:647-657. <br /> v obtained solving optimization problem (1) con-                            10.   Lun DS, Rockwell G, Guido NJ, Baym M, Kelner JA, Berger B, Galagan JE, <br /> tained 505 non-zero components. Again, reduced S                                   Church GM: Large-scale identification genetic design strategies using <br /> was generated removing reactions j vj                               local search. Mol Syst Biol 2009, 5:296. <br />                                                                                11.   Pharkya P, Burgard AP, Maranas CD: OptStrain: A computational <br /> = 0, result used input efmtool. With                                 framework redesign microbial production systems. Genome Res <br /> maximum heap size 4 GB, efmtool failed gen-                                2004, 14:2367-2376. <br /> erating elementary modes, 657,447 modes                               12.   Pharkya P, Maranas CD: An optimization framework identifying <br />                                                                                      reaction activation/inhibition elimination candidates for <br /> generated.                                                                           overproduction microbial systems. Metab Eng 2006, 8:1-13. <br />                                                                                13.   Rocha M, Maia P, Mendes R, Pinto JP, Ferreira EC, Nielsen J, Patil KR, <br />                                                                                      Rocha I: Natural computation meta-heuristics silico <br /> Additional material                                                                  optimization microbial strains. BMC Bioinformatics 2008, 9:499. <br />                                                                                14.   Schuster S, Hilgetag C: On elementary flux modes biochemical <br />     Additional file 1: Source <span id='am-321' about='Thesaurus:code' typeof='owl:Thing'>code</span> elementary mode decomposition                 reaction systems steady state. J Biol Syst 1994, 2:165-182. <br />     method implemented using MATLAB Gurobi                                 15.   Schuster S, Fell DA, Dandekar T: A general definition metabolic <br />                                                                                      pathways useful systematic organization analysis complex <br />                                                                                      metabolic networks. Nat Biotechnol 2000, 18:326-332. <br />                                                                                16.   Stelling J, Klamt S, Bettenbrock K, Schuster S, Gilles ED: Metabolic network <br /> Acknowledgements                                                                     structure determines key aspects functionality regulation. Nature <br /> KI gratefully acknowledges support Australian Government            2002, 420:190-193. <br /> an Australian Postgraduate Award. The authors like thank Belinda      17.   Trinh CT, Wlaschin A, Srienc F: Elementary mode analysis: useful <br /> Chiera Melissa Yates careful proofreading manuscript.                 metabolic pathway analysis tool characterizing cellular metabolism. <br />                                                                                      Appl Microbiol Biotechnol 2009, 81:813-826. <br /> Author details                                                                 18.   Klamt S, Stelling J: Combinatorial Complexity Pathway Analysis in <br /> 1 <br />  Phenomics Bioinformatics Research Centre, University South                   Metabolic Networks. Mol Biol Rep 2002, 29:233-236. <br /> Australia, Mawson Lakes, SA 5095, Australia. 2School Mathematics        19.   Acu&#195;&#177;a V, Chierichetti F, Lacroix V, Marchetti-Spaccamela A, Sagot MF, <br /> Statistics, University South Australia, Mawson Lakes, SA 5095, Australia.         Stougie L: Modes cuts metabolic networks: Complexity and <br /> 3 <br />  Australian Centre Plant Functional Genomics, University South                algorithms. Biosystems 2009, 95:51-60. <br /> Australia, Mawson Lakes, SA 5095, Australia. 4Center Computational     20.   Gagneur J, Klamt S: Computation elementary modes: unifying <br /> Integrative Biology, Rutgers University, Camden, NJ 08102, USA. 5Department          framework new binary approach. BMC Bioinformatics 2004, 5:175. <br /> of Computer Science, Rutgers University, Camden, NJ 08102, USA.                21.   Terzer M, Stelling J: Large-scale computation elementary flux modes <br /> 6 <br />  Department Engineering Mathematics, University Bristol, Bristol BS8           bit <span id='am-322' about='xsp:pattern' typeof='owl:Thing'>pattern</span> trees. Bioinformatics 2008, 24:2229-2235. <br /> 1TR, UK.                                                                       22.   Feist AM, Palsson B&#195;&#732;: The growing scope applications genome-scale <br />                                                                                      metabolic reconstructions using Escherichia coli. Nat Biotechnol 2008, <br /> Authors&#226;&#8364;&#8482; contributions                                                               26:659-667. <br /> DSL CC conceived project. DSL KI designed implemented          23.   Schwartz JM, Kanehisa M: A quadratic programming approach for <br /> the method performed computational experiments. DSL, CC, KI              decomposing steady-state metabolic flux distributions elementary <br /> wrote paper. All authors read approved final manuscript.            modes. Bioinformatics 2005, 21:ii204-205. <br />                                                                                24.   Poolman MG, Venkatesh KV, Pidcock MK, Fell DA: A method the <br /> Received: 29 November 2010 Accepted: 3 June 2011                                     determination flux elementary modes, application to <br /> Published: 3 June 2011                                                               Lactobacillus rhamnosus. Biotechnol Bioeng 2004, 88:601-612. <br />                                                                                25.   Figueiredo LF, Podhorski A, Rubio A, Kaleta C, Beasley JE, Schuster S, <br /> References                                                                           Planes FJ: Computing shortest elementary flux modes genome- <br /> 1. Milne CB, Kim PJ, Eddy JA, Price ND: Accomplishments genome-scale           scale metabolic networks. Bioinformatics 2009, 25:3158-3165. <br />     silico modeling industrial medical biotechnology. Biotechnol J     26.   Colijn C, Brandes A, Zucker J, Lun DS, Weiner B, Farhat MR, Cheng TY, <br />     2009, 4:1653-1670.                                                               Moody DB, Murray M, Galagan JE: Interpreting expression data with <br />  Ip et al. BMC Systems Biology 2011, 5:91                                                                                                      Page 11 11 <br /> http://www.biomedcentral.com/1752-0509/5/91 <br />  <br />  <br />  <br />  <br />       metabolic flux models: Predicting Mycobacterium tuberculosis mycolic <br />       acid production. PLoS Comp Biol 2009, 5:e1000489. <br /> 27.   Segr&#195;&#168; D, Vitkup D, Church GM: Analysis optimality natural and <br />       perturbed metabolic networks. Proc Natl Acad Sci USA 2002, <br />       99:15112-15117. <br /> 28.   Shlomi T, Berkman O, Ruppin E: Regulatory on/off minimization of <br />       metabolic flux changes genetic perturbations. Proc Natl Acad Sci <br />       USA 2005, 102:7695-7700. <br /> 29.   Achterberg T: Constraint Integer Programming. Technische Universit&#195;&#164;t <br />       Berlin; 2007. <br /> 30.   Jamshidi N, Palsson BO: Investigating metabolic capabilities of <br />       Mycobacterium tuberculosis H37Rv using silico strain iNJ661 and <br />       proposing alternative drug targets. BMC Syst Biol 2007, 1:26. <br /> 31.   Fang X, Wallqvist A, Reifman J: Development analysis vivo- <br />       compatible metabolic network Mycobacterium tuberculosis. BMC Syst <br />       Biol 2010, 4:160. <br /> 32.   Causey TB, Zhou S, Shanmugam KT, Ingram LO: Engineering the <br />       metabolism Escherichia coli W3110 conversion sugar to <br />       redox-neutral oxidized products: Homoacetate production. Proc Natl <br />       Acad Sci USA 2003, 100:825-832. <br /> 33.   Lee S, Phalakornkule C, Domach MM, Grossmann IE: Recursive MILP model <br />       finding alternate optima LP models metabolic networks. <br />       Computers &amp; Chemical Engineering 2000, 24:711-716. <br /> 34.   Munoz-Elias EJ, McKinney JD: Mycobacterium tuberculosis isocitrate lyases <br />       1 2 jointly required vivo growth virulence. Nat Med <br />       2005, 11:638-644. <br /> 35.   Schnappinger D, Ehrt S, Voskuil MI, Liu Y, Mangan JA, Monahan IM, <br />       Dolganov G, Efron B, Butcher PD, Nathan C, Schoolnik GK: Transcriptional <br />       Adaptation Mycobacterium tuberculosis Macrophages: Insights <br />       Phagosomal Environment. J Exp Med 2003, 198:693-704. <br /> 36.   Sharma V, Sharma S, Hoener zu Bentrup K, McKinney JD, Russell DG, <br />       Jacobs WR Jr, Sacchettini JC: Structure isocitrate lyase, persistence <br />       factor Mycobacterium tuberculosis. Nat Struct Biol 2000, 7:663-668. <br /> 37.   Beste DJ, Hooper T, Stewart G, Bonde B, Avignone-Rossa C, Bushell ME, <br />       Wheeler P, Klamt S, Kierzek AM, McFadden J: GSMN-TB: web-based <br />       genome-scale network model Mycobacterium tuberculosis metabolism. <br />       Genome Biol 2007, 8:R89. <br /> 38.   Fong SS, Palsson B&#195;&#732;: Metabolic gene-deletion strains Escherichia coli <br />       evolve computationally predicted growth phenotypes. Nat Genet 2004, <br />       36:1056-1058. <br /> 39.   Purohit HJ, Cheema S, Lal S, Raut CP, Kalia VC: In Search Drug Targets <br />       Mycobacterium tuberculosis. Infect Disord Drug Targets 2007, 7:245-250. <br /> 40.   Green ML, Karp PD: A Bayesian method identifying missing enzymes <br />       predicted metabolic pathway databases. BMC Bioinformatics 2004, 5:76. <br /> 41.   Indira M, Sirsi M: Nutritional studies Mycobacterium tuberculosis&#226;&#8364;&#8220; <br />       effect amino acids &#226;&#8364;&#732;in vitro&#226;&#8364;&#8482; growth Mycobacterium <br />       tuberculosis. Indian J Tuberculosis 1960, 7:75-82. <br /> 42.   Krulwich TA, Pelliccione NJ: Catabolic pathways coryneforms, <br />       nocardias, mycobacteria. Annu Rev Microbiol 1979, 33:95-111. <br />  <br />   doi:10.1186/1752-0509-5-91 <br />   Cite article as: Ip et al.: Analysis complex metabolic behavior <br />   pathway decomposition. BMC Systems Biology 2011 5:91. <br />  <br />  <br />  <br />  <br />                                                                                     Submit manuscript BioMed Central <br />                                                                                     advantage of: <br />  <br />                                                                                     &#226;&#8364;&#162; Convenient online submission <br />                                                                                     &#226;&#8364;&#162; Thorough peer review <br />                                                                                     &#226;&#8364;&#162; No space constraints color figure charges <br />                                                                                     &#226;&#8364;&#162; Immediate publication acceptance <br />                                                                              <br /> </body></html>