<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>LmaPA2G4, Homolog Human Ebp1, Is Essential <br /> Gene Inhibits Cell Proliferation L. major <br /> Brianna Norris-Mullins1., Kaitlin VanderKolk1., Paola Vacchina1., Michelle V. Joyce2, Miguel A. Morales1* <br /> 1 Eck Institute Global Health. Department Biological Sciences. University Notre Dame, Notre Dame, Indiana, United States America, 2 Mass Spectrometry and <br /> Proteomics Facility, University Notre Dame, Notre Dame, Indiana, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      We identified LmaPA2G4, homolog human proliferation-associated 2G4 protein (also termed Ebp1), a <br />      phosphoproteomic screening. Multiple sequence alignment cluster analysis revealed LmaPA2G4 non-peptidase <br />      member M24 family metallopeptidases. This pseudoenzyme structurally related methionine aminopeptidases. A <br />      null mutant based negative selection allowed demonstrate LmaPA2G4 essential gene Leishmania <br />      major. Over-expression LmaPA2G4 did alter cell morphology ability differentiate metacyclic amastigote <br />      stages. Interestingly, over-expression affected cell proliferation virulence mouse footpad analysis. LmaPA2G4 binds a <br />      synthetic double-stranded RNA polyriboinosinic polyribocytidylic acid [poly(I:C)] shown electrophoretic mobility shift <br />      assay (EMSA). Quantitative proteomics revealed over-expression LmaPA2G4 led accumulation factors involved in <br />      translation initiation elongation. Significantly, strong reduction novo protein biosynthesis transgenic <br />      parasites using non-radioactive metabolic labeling assay. In conclusion, LmaPA2G4 essential gene potentially <br />      implicated fundamental biological mechanisms, translation, making attractive target therapeutic intervention. <br />  <br />   Citation: Norris-Mullins B, VanderKolk K, Vacchina P, Joyce MV, Morales MA (2014) LmaPA2G4, Homolog Human Ebp1, Is Essential Gene Inhibits Cell <br />   Proliferation L. major. PLoS Negl Trop Dis 8(1): e2646. doi:10.1371/journal.pntd.0002646 <br />   Editor: Ben L. Kelly, Louisiana State University, United States America <br />   Received July 9, 2013; Accepted December 3, 2013; Published January 9, 2014 <br />   Copyright: &#195;&#376; 2014 Norris-Mullins et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Eck Institute Global Health capitalization funds University Notre Dame. Kaitlin VanderKolk was <br />   supported COS-SURF fellowship. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: miguel.morales@nd.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          mainly acidic pH high temperature mammalian <br />                                                                                       host [9]. <br />    Protozoan parasites genus Leishmania causative                         Signal transduction pathways relay environmental <br /> agents leishmaniasis, disease characterized                         stimuli reversible phosphorylation, ultimately leading <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> spectrum clinical manifestations ranging ulcerative skin                      changes protein activity, interaction expression profiles [10]. <br /> lesions fatal visceral infections [1]. Leishmaniasis poverty-                 Mitogen-activated protein kinases (MAPKs) conserved virtually <br /> related disease associated malnutrition, displacement,                    eukaryotic organisms. To gain insight MAPK <br /> poor housing, illiteracy, gender discrimination, weakness                      pathway Leishmania performed comparative phosphoproteomics <br /> immune lack resources [2]. Leishmaniasis                     MPK7 [11] WT parasites objective characterizing <br /> compromised emergence co-infection human                               putative substrates kinase. As screening identified <br /> immunodeficiency virus (HIV) endemic areas [3]. Globally,                          LmaPA2G4, homolog human proliferation-associated 2G4 <br /> there estimated 1.5&#226;&#8364;&#8220;2 million new cases leishmaniasis                       (PA2G4, termed Ebp1) [12]. PA2G4 proteins highly <br /> and 70,000 deaths year, 350 million people risk                    conserved eukaryotes involved regulation cell <br /> infection disease [4]. Due absence vaccination,                         growth differentiation. The human member family, ErbB3 <br /> chemotherapy, vector control, remains                        binding protein 1 (Ebp1), ubiquitously expressed localizes in <br /> most important elements control leishmaniasis. Current                      nucleus cytoplasm [13]. The protein binds structured <br /> anti-leishmanial drugs include pentavalent antimony, amphoter-                        RNAs suggested involved linking ribosome <br /> icin B miltefosine; toxic expensive. To date,                     biosynthesis cell proliferation [14]. Here that <br /> successful vaccine exists anti-leishmanial drugs                          LmaPA2G4 essential gene L. major. The over-expression of <br /> mentioned risk ineffective emerging resistances                       LmaPA2G4 results accumulation intermediates translation <br /> [5,6]. Therefore, new drugs urgently needed [7].                                  initiation ultimately leads growth virulence defects. <br />    During infectious cycle, Leishmania differentiates the <br /> extracellular promastigote intracellular amastigote form.                      Materials Methods <br /> Flagellated promastigotes develop midgut sandflies, and <br /> following infection humans, differentiate intracellular                         Ethics statement <br /> amastigotes multiply inside macrophage lysosome [8].                          The University Notre Dame credited Animal <br /> This differentiation triggered environmental signals,                           Welfare Assurance (#A3093-01). All animal studies conducted <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                               1                                 January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                             Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br />   Author Summary                                                                     GGATCCACGGCCGTGGCATCCGTG-39 (XhoI BamHI) <br />                                                                                      59-CGCGGTACCCACGATGGGCAGAACGCC- 39 (KpnI). <br />   Leishmaniasis disease caused protozoan parasites                        Reactions performed total volume 50 ml containing <br />   genus Leishmania. Its clinical manifestations                              LongAmp high fidelity Taq- DNA polymerase (New England <br />   widespread, ranging ulcerative skin lesions life-                          Biolabs) following manufacturer&#226;&#8364;&#8482;s recommendation. Products <br />   threatening visceral infections. Approximately 1.5&#226;&#8364;&#8220;2 mil-                          cloned pGEM-T pGEM-T Easy vectors (Promega) <br />   lion new cases leishmaniasis reported year                        create pGEM-T-59UTR-39UTR. A 2.8 kb SmaI-XhoI fragment <br />   estimated 70,000 deaths. During infectious cycle,                           pX63HYG containing hygromycin B (HYG B) gene and <br />   Leishmania differentiates extracellular promasti-                         2.5 kb XhoI-BamHI fragment PX63PAC including the <br />   gote intracellular pathogenic amastigote form.                              puromycin (PAC) ligated 59UTR 39UTR to <br />   Differentiation triggered environmental signals                       generate targeting constructs. Constructs linearized <br />   mammalian host, acidic pH high temper-                              KpnI dephosphorylated. The LmaPA2G4 homolog <br />   ature. Due absence vaccination, chemotherapy, <br />                                                                                      (CAJ07101) PCR amplified genomic DNA L. major <br />   vector control, remains most <br />                                                                                      FVI using primers 59-ACCAGATCTATGTCAAAGAACG- <br />   important elements control leishmaniasis. Current <br />   anti-leishmanial drugs include pentavalent antimony,                               CTGAC- 39(BglII) 5-GCGAGATCTCTACTTCGCGCGC- <br />   amphotericin B miltefosine; toxic, expensive                          TTCTT- 39(BglII). Purified PCR products cloned into <br />   risk ineffective emerging resistance.                          pGEM-T EasyVector (Promega). N-terminal GFP-PA2G4 fusion <br />   Therefore, new drugs urgently needed. LmaPA2G4                            pXNG-PA2G4 obtained inserting 1.1 kb BglII <br />   homolog human proliferation-associated 2G4 protein                              fragment pGEM-T respective site pXG-GFP+2 <br />   (PA2G4, termed Ebp1). We essential                         [18] pXNG [19]. F1, R1 primer pair 59- CATCAATATTT- <br />   gene L. major gain-of-function approach allowed                           CATGCGC-39 59-CGTGTCCTCCTCTTCTTC- 39; F2, <br />   implicate LmaPA2G4 translation subsequent                             R1 59- GGTAGTGTCGCGTGTTGG-39; F1, R2 59-CTGCAT- <br />   protein synthesis reduction, growth defects virulence                          CAGGTCGGAGACGC-39 F1, R3 59-GGGGTCAGGG- <br />   attenuation. This work highlights essential role                            GCGTGGGTCAG-39 used corroborate absence of <br />   LmaPA2G4 biology parasite makes                          LmaPA2G4 ORF null mutant lines. LmaPA2G4 PCR <br />   attractive target drug development.                                         amplified cloned episomal vector pLEXSY (Jena <br />                                                                                      Bioscience), parasites selected 75 mg/mL hygromycin <br /> according Institutional Animal Care Use Committee                         B (Sigma). Parasites transfected vector, pXG- <br /> (IACUC) guidelines. The protocol infection mice                      GFP+2 pLEXSY used mock controls. <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>Null mutants</span> <br /> Leishmania approved University&#226;&#8364;&#8482;s IACUC (October 16,                       episomal transfectants established electroporation as <br /> 2012, protocol #15-047).                                                             previously described [20,21]. <br />  <br /> Parasite culture                                                                     RT-PCR analysis <br />    Leishmania major strain Friedlin V1 (MHOM/JL/80/Friedlin)                            Total RNA isolated L. major L. donovani WT and <br /> was cultured M199 medium supplemented 10% FBS                             transgenic parasites Trizol reagent (Life Technologies Inc., <br /> 26uC pH 7.4 [15]. L. donovani strain 1S2D (MHOM/SD/62/                           NY) using RNase-free plastic supplies. cDNA amplified using <br /> 1S-CL2D) grown M199 supplemented 10% FBS                             M-MLV Reverse Transcriptase (Sigma) oligo d(T)15 (Pro- <br /> axenic amastigotes differentiated described previously                       mega) following manufacturer&#226;&#8364;&#8482;s recommendations. PCR was <br /> [16]. For experiments L. major metacyclic promastigotes                    carried LongAmp high fidelity Taq- DNA polymerase <br /> enriched agglutination [17]. Briefly, cells incubated                    (New England Biolabs) using LmaPA2G4 specific primers 59- <br /> 30 min RT 50 mg/ml peanut agglutinin M199                         CCACGTGGACGGCTACTGCGCCG-39 59- CTTCCTT- <br /> serum, agglutinated parasites removed centrifugation                     TTCGAAGAGAATAGGG-39 GAPDH primers 59- CGAC- <br /> metacyclic parasites recovered supernatant.                            GACGGCAAAGCAGAAG-3; 59- TCAGCGCCACACCG- <br />                                                                                      TTGAAG-39. All RNA samples treated DNA-free <br /> Bioinformatics                                                                       (Ambion, Inc., TX) remove contaminating genomic DNA. Each <br />    L. major CAJ07101 (gi|68126048) used initial query                      RT-PCR product analyzed gel electrophoresis using 1% <br /> for PSI-BLAST cycles results significant E-                      agarose gels band intensity analyzed ImageQuant TL <br /> values (,10e26) selected. Sequences corresponding                            software (GE Healthcare). <br /> putative aminopeptidase proteins sequenced genomes <br /> of H. sapiens, L. major, L. infantum, L. braziliensis, L. donovani, T. brucei,       Confocal microscopy <br /> T. vivax, T. cruzi T. congolense retrieved using TriTrypDB                    Live L. major promastigotes over-expressing GFP-PA2G4 were <br /> and UniProt databases; (http://tritrypdb.org/tritrypdb/)                         immobilized poly (L-lysine)-coated 35 mm glass dishes <br /> (www.uniprot.org). Sequences aligned Clustal X (v                          (MatTek Corporation, USA) counterstained 1 mg/mL <br /> 2.0). Alignments converted MEGA compatible files                         NucBlue Live Cell stain (Hoechst 33342) (Molecular Probes). <br /> fed MEGA5.2 software package. A Neighbor-Joining tree                       Fluorescent imaging performed using spinning disk confocal <br /> was computed 500 bootstrap replicates.                                          Revolution (Andor Technology) 636 oil immersion <br />                                                                                      objective. Image acquisition using AndorIQ software <br /> Molecular constructs sequencing                                                  (Andor Technology) images processed ImageJ software <br />   In order generate null mutants, 901 bp region 59                       (NIH, USA). <br /> untranslated region (UTR) upstream LmaPA2G4 amplified <br /> with primers 59-ACCGGTACCCAATCATGGCCCACC-                                        Mouse infections <br /> GAAGG- 39 (KpnI) 59-CGCCCCGGG/CTCGAGTTTTT-                                          Virulence studies performed previously described [11]. <br /> TTGGGTGGGTGGC-39 (SmaI XhoI). A 903 bp fragment                                Briefly, groups female BALB/c mice (Charles River) were <br /> the 39UTR amplified primers 59-ACCCTCGAG/                                   injected footpad 105 metacyclic promastigotes <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                              2                          January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                      Essential Genes Drug Targets Leishmania <br />  <br />  <br /> GFP-PA2G4, cured GFP-PA2G4 mock control. Lesions                   gradient step 8 h 6000 V 5 h (60550 Vh). Following <br /> followed weekly measuring thickness footpads               IEF, strips equilibrated different solutions 15 min <br /> Vernier caliper.                                                            (6 M urea, 75 mMTris/HCl pH 8.8, 29.3% glycerol, 4% <br />                                                                             SDS, 0.002% bromophenol blue) supplemented 65 mM <br /> Western blot analysis                                                       DTT 13.5 mM iodoacetamide, respectively. The strips were <br />    Crude cell lysates separated 4&#226;&#8364;&#8220;12% Bis-Tris NuPAGE               transferred SDS polyacrylamide gels sealed 0.5% <br /> gels (Life), electro-blotted PVDF membranes (Pierce).              agarose 25 mMTris-<span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span>, 0.19 M glycine, 0.2% SDS, 0.01% <br /> Proteins revealed using following primary antibodies:              bromophenol blue. Electrophoresis carried SE 600 <br /> mouse monoclonals anti-A2 (Abcam) anti-GFP-HRP (Milte-                  Ruby cooled electrophoresis (GE Healthcare) using 12.5% <br /> nyi Biotec), mouse monoclonal anti-tubulin (Sigma), anti-               SDS-PAGE gels two-step runs (1 W/gel 15 min 7 W/ <br /> rabbit anti-mouse HRP-conjugated secondary antibodies                    gel 5 h). <br /> (Pierce). <br />                                                                             Staining procedures image analysis <br /> In vitro kinase assay                                                          After electrophoresis, gels scanned Typhoon FLA <br />    20 mL immuno-complexes (GFPK7; transgenic parasites                   9500 Imager (GE Healthcare) using 488/520 nm Cy2, 532/ <br /> over-expressing active MPK7) incubated Thermo-                 580 nm Cy3, 633/670 nm Cy5 100 mm pixel size. <br /> mixer R (Eppendorff) 30 min 30uC 1000 rpm                   Gel images normalized adjusting PMT voltage obtain <br /> 50 mL reaction mixture (Millipore) containing 5 mg myelin basic             appropriate pixel value saturation. Images were <br /> protein (MBP) substrate (positive control) recombinant substrate         analyzed Decyder v. 6.5 (GE Healthcare) Delta2D v.4.3 <br /> and 10 mCi [c-32P] ATP (3000 Ci/mmol). Reactions                       software (Decodon). Gels matched warped spots <br /> terminated heating samples 10 min 98uC                     detected images. A 2-fold difference abundance, with <br /> NuPAGE sample buffer reducing agent (Life). 30 mL                p-values,0.05, considered significant expression <br /> reaction separated SDS-PAGE. The gel Coomassie-                 profiles. Polyacrylamide gels fixed 50% methanol <br /> stained, fixed, dried analyzed autoradiography.                      7% acetic acid stained using SYPRO Ruby total protein <br />                                                                             gel stain (Life). Spots manually excised gels <br />                                                                             using blue-light transilluminator (Life). <br /> Sample preparation labeling 2D- DIGE <br />    Protein extracts logarithmic L. major WT, GFP-PA2G4 <br /> and stationary GFPK7 promastigotes differentially labelled             Protein identification <br /> with spectrally resolvable Cy3 Cy5 previously described             The gel spots subjected reduction 55 mM dithiothre- <br /> [21]. A pool extracts labelled Cy2 normalization            itol (Sigma-Aldrich) 25 mM ammonium bicarbonate (Fisher <br /> purposes, following manufacturer&#226;&#8364;&#8482;s recommendations (GE                  Scientific) 56uC 1 hour followed alkylation 100 mM <br /> Healthcare). Phosphoproteins enriched affinity IMAC               iodoacetamide (Sigma-Aldrich) 25 mM ammonium bicarbonate at <br /> columns (Qiagen) previously described [22]. Following <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span>-             room temperature dark 45 min. The spots washed <br /> ling, proteins precipitated using 2-D Clean-Up kit (GE               25 mM ammonium bicarbonate 10 min followed two <br /> Healthcare), allowing quantitative precipitation removal         consecutive washes 25 mM ammonium bicarbonate 50/50 <br /> interfering substances, detergents, salts, lipids, phenolics,       acetonitrile:water 5 min, each. The spots placed vacuum <br /> and nucleic acids.                                                          concentrator dry completely addition 12.5 ng trypsin <br />                                                                             gold (Promega) gel spot. The spots kept 4uC for <br />                                                                             30 min swell incubated 37uC overnight. <br /> Electrophoretic Mobility Shift Assays <br />                                                                             Following trypsin digestion, supernatant collected. Peptides <br />    Synthetic double stranded RNA Poly (I:C) (Sigma) labeled <br />                                                                             extracted gel spots consecutive <br /> with Label IT Cy5 labeling kit (Mirus). Briefly, 5 mg RNA was <br />                                                                             additions 50% acetonitrile/45% water/5% formic acid spots <br /> incubated 1:1 (v:w) ratio Label IT Cy5 reagent nucleic <br />                                                                             followed 30 min vortexing. The sets extracts were <br /> acid. 40 ng labeled Poly(I:C) incubated 20 ng GFP- <br />                                                                             combined supernatant gel spot vacuum <br /> PA2G4 fusion protein binding buffer (50 mM Tris pH 7.4,                  concentrated 10 mL. Each concentrated digest desalted a <br /> 0.5 mM EDTA, 150 mM NaCl) room temperature                       C18 Ziptip (EMD Millipore) according manufacturer <br /> 45 min. GFP-PA2G4 immunoprecipitated anti-GFP                      instructions. The desalted digests dried vacuum <br /> magnetic beads previously described [20]. An aliquot gel              concentrator reconstituted 10 mL 0.1% TFA water. A <br /> loading buffer (0.25% bromophenol blue, 0.25% xylene cyanol,                2 mL aliquot gel digest injected nanoAcquity <br /> 50% glycerol) added reaction mixture resolved             UPLC (Waters Corporation) BEH300 C18 100 mm6100 mm <br /> 10% non-denaturing polyacrylamide gels 16 TBE. Gels                 column (Waters Corporation) 1.7 mm particle size. A gradient of <br /> scanned Typhoon FLA 9500 Imager (GE Healthcare) using                  0.1% formic acid water (A) 0.1% formic acid acetonitrile (B) <br /> 633/670 nm Cy5 filter 489/508 nm GFP filter.                    performed starting 2% B held 6 min ramping <br />                                                                             40% B 40 min 90% B 43 min. The column washed <br /> Isoelectric focusing (IEF) two-dimensional gel                          90% B 7 min re-equilibrated 98% A: 2% B. <br /> electrophoresis (2D)                                                        The nanoAcquity coupled LTQ Orbitrap Velos mass <br />   IEF 100 120 mg protein carried using                  spectrometer (Thermo Corporation) data dependent scans the <br /> EttanIPGphor 3 System (GE Healthcare) 20uC 11                   digested samples abundant ions scan were <br /> 13 cm non-linear DryStrip (pH 4&#226;&#8364;&#8220;7). Strips passively                   selected CID fragmentation. The UPLC-MS/MS chromatograms <br /> rehydrated overnight room temperature rehydration solution            spectra analyzed using Xcalibur software (Thermo), the <br /> (GE Healthcare) containing 0.5% IPG buffer 4&#226;&#8364;&#8220;7 sample.              extracted data searched L. major custom database via <br /> The IEF maximum current setting 50 mA/strip. The following              Mascot and/or Protein Pilot. Search criteria included global <br /> conditions programmed IEF: 100 V gradient step 5 h,            modification carbamidomethylation cysteines. Proteins <br /> 300 V gradient step 5 h, 1000 V gradient step 2 h, 6000 V           identified 1% false discovery rate. <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                     3                            January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                           Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br /> Metabolic labeling <br />    The metabolic labeling novo synthetized proteins was <br /> conducted using non-radioactive assay Click-iT AHA kit and <br /> Click-iT Cell Reaction Buffer Kit (Life) following manufacturer&#226;&#8364;&#8482;s <br /> guidelines minor modifications. Briefly, 26108 mid-log phase <br /> WT GFP-PA2G4 L. major L. donovani promastigotes as <br /> GFP-PA2G4 L. donovani amastigotes initially incubated for <br /> 30 min 27uC 2 mL methionine-free RPMI medium supple- <br /> mented 10% FBS order deplete methionine reserves. <br /> Metabolic labeling performed 2 h 27uC presence of <br /> 50 mM azidohomoalanine (AHA). A culture L. major treated 2 h <br /> with 100 mg/mL cycloheximide, inhibitor protein biosynthesis, <br /> was included positive control. After labeling, cells harvested <br /> and lysed 50 mL sample employed perform the <br /> Click reaction TAMRA. Proteins precipitated, resolubi- <br /> lized 1D gel electrophoresis sample loading buffer heated for <br /> 10 min 70uC subsequently resolved precast polyacryl- <br /> amide gel (NuPAGE Novex 4&#226;&#8364;&#8220;12% Bis-Tris gels, Life). Gel was <br /> visualized Typhoon FLA 9500 (GE Healthcare) analyzed <br /> with ImageQuant TL software (GE Healthcare). After imaging the <br /> gel TAMRA-labeled samples, gel fixed stained with <br /> SYPRO Ruby order assess total protein content. Statistical <br /> comparisons using non parametric Mann&#226;&#8364;&#8220;Whitney U- <br /> test. <br />  <br /> Results <br /> Identification LmaPA2G4                                                     Figure 1. 2D-DIGE quantitative phosphoproteomics analysis of <br />    We previously shown L. major MPK7 implicated                GFPK7. An enlarged region 2D-DIGE gels showing Cy3-labeled <br /> parasite growth control, including pathogenic amastigote stage             WT stationary promastigotes Cy5-labeled stationary GFPK7 <br /> [11]. The overexpression active MPK7 (GFPK7 transgenic                   promastigotes presented. Spot ID 207 (white arrow) over- <br /> parasites overexpressing MPK7) led defects cell cycle and,               represented GFPK7. In lower panel, graphical representation of <br /> ultimately, attenuated virulence mouse model. In order                 BVA (Biological Variation Analysis) module Decyder software (GE <br />                                                                                Healthcare) statistics spot ID 207 (2.97 fold p = 0.00056). <br /> identify potential downstream targets MPK7 performed                     The spot analyzed mass spectrometry identified as <br /> comparative 2D-DIGE phosphoproteins isolated                      LmjF19.0160, putative aminopeptidase. <br /> independent stationary cultures L. major wild type (WT)                 doi:10.1371/journal.pntd.0002646.g001 <br /> GFPK7 promastigotes. MPK7 shows increased activity at <br /> stationary phase [20]. Phosphoproteins isolated immobi- <br /> lized metal affinity chromatography (IMAC) differentially                  trypanosomatid members M24 family metallopepti- <br /> labeled CyDye fluors (GE Healthcare) detailed Material              dases multiple alignment cluster analysis. LmjF19.0160 <br /> and Methods. Phosphoproteins separated 2DE pH 4&#226;&#8364;&#8220;                    (protein ID gi|68126048) used initial query PSI- <br /> 7 IPG immobiline strips SDS-PAGE. Images analyzed                     BLAST sequenced genomes H. sapiens, L. major, L. <br /> with DeCyder 6.5 software (GE Healthcare). Up-regulation                    infantum, L. braziliensis, L. donovani, T. brucei, T. cruzi, T. vivax T. <br /> spots 2-fold GFPK7, p-values,0.05                       congolense. Homologs human METAP1 2 all <br /> considered significant. Spot ID 207 over-represented                    trypanosomatids shown clustering tree (Fig. 2). Bootstrap <br /> GFPK7 (2.97 fold change p = 0.00056) excised                  values support existence METAP1 2 subclasses among <br /> gel analyzed MS/MS (Fig. 1). LmjF19.0160 (Tritryp gene                  Leishmania Trypanosoma. Interestingly, LmjF19.0160 clusters <br /> ID) putative aminopeptidase predicted MW 43 kDa.                homologs human proliferation-associated protein 2G4 <br /> Recombinant LmjF19.0160 phosphorylated vitro                     (PA2G4) [25]. These non-peptidase proteins possess the <br /> active GFPK7 using vitro kinase assay (Fig. S1). Phospho-                &#226;&#8364;&#732;&#226;&#8364;&#732;pita-bread&#226;&#8364;&#8482;&#226;&#8364;&#8482; fold typical methionyl aminopeptidases, however <br /> transferase activity recombinant GFPK7 assessed                      lack metal cofactors peptidase activity [26]. <br /> phosphorylation MBP. Given fact LmjF19.0160 was <br /> enriched IMAC (phospho-specific) column,                        RT-PCR analysis <br /> attempting characterize putative phosphorylation sites,               Although genome Leishmania constitutively <br /> the objective performing site-directed mutagenesis.                         expressed [27], 6% 9% genes display <br />                                                                                significant expression profiles. Therefore, analyzed the <br /> Bioinformatics analysis L. major LmjF19.0160                                transcript levels LmaPA2G4 semi-quantitative RT-PCR. <br />   LmjF19.0160 belongs clan MG, family M24                            Total RNA isolated L. major logarithmic metacyclic <br /> metallopeptidases according classification MEROPS                    parasites. Peanut agglutination used enrich metacyclics in <br /> database [23]. Family M24 divided subfamilies                  stationary cultures [28]. For L. donovani used host-free <br /> M24A B. Typical members subfamily M24A methionyl                    amastigotes previously described [16]. GAPDH used a <br /> aminopeptidases type I II (METAP1 2). These peptidases                 housekeeping gene semi-quantification purposes. As judged by <br /> are essential removal initiating methionine             LmaPA2G4/GAPDH ratio, significant differ- <br /> proteins [24]. We investigated relationship human                  ences expression levels different life stages (Fig. S2). <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        4                              January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                 Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br />  <br /> Figure 2. Bioinformatics analysis L. major LmjF19.0160. Relationship human trypanosomatid members M24A subfamily <br /> of metallopeptidases analyzed multiple alignment cluster analysis using Clustal X. This subfamily comprises homologs methionyl <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                  5                          January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                      Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br /> aminopeptidases 1 2 (METAP1 2), non-peptidases. Alignment fed MEGA5.2 software Neighbor-Joining tree was <br /> computed 500 bootstrap replicates. Numbers nodes indicate bootstrap support. <br /> doi:10.1371/journal.pntd.0002646.g002 <br />  <br />  <br /> Loss-of-function studies LmaPA2G4                                         infection [29]. Mock, GFP-PA2G4 cured GFP-PA2G4* <br />    In order gain insight putative function PA2G4           parasites grown G418-free medium normalized for <br /> Leishmania designed loss-of-function strategy. Leishmania           virulence passage BALB/c mice [30]. 105 mock, <br /> diploid parasites rounds targeted replacement drug-        GFP-PA2G4 GFP-PA2G4* metacyclic parasites inocu- <br /> resistance marker necessary. Unsuccessful attempts replace        lated hind footpad groups female BALB/c mice. <br /> two PA2G4 alleles resistance markers, create homozygous            Lesion formation followed measuring increase footpad <br /> KO, suggested LmaPA2G4 essential gene. To                     size Vernier caliper. Mock GFP-PA2G4* parasites <br /> demonstrate essentiality LmaPA2G4 used genetic method            elicited strong response ca. 30 days inoculation resulted <br /> based negative selection [19] guard potential lethal       necrotic lesions (Fig. 5A). Interestingly GFP-PA2G4 highly <br /> <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span>. Both LmaPA2G4 alleles removed homol-                  attenuated lesions apparent 40 days after <br /> ogous recombination presence episome expressing                 inoculation. The cured line, grown absence G418, elicited <br /> LmaPA2G4 (pXNG-PA2G4) (Fig. 3A). The loss endogenous                      response similar GFP mock parasites, suggesting the <br /> LmaPA2G4 null mutants confirmed independent                specific expression LmaPA2G4 responsible attenuated <br /> homozygous lines PCR (Fig. 3B). As expected, episomal            phenotype. At independent GFP-PA2G4 lines used <br /> copy LmaPA2G4 present. The episome pXNG [19] carries                 rule potential discrepancies clonal variations. To <br /> negative selectable thymidine kinase (TK), fluorescent protein             determine overexpression LmaPA2G4 affects the <br /> (GFP) resistance marker (SAT). TK renders parasites                differentiation pro- amastigotes, established L. donovani <br /> susceptible ganciclovir (GCV). PXNG-PA2G4/WT parasites               transgenic (GFP-PA2G4) lines allow axenic amastigote <br /> selected grown presence 250 mg/mL nourseothricin               differentiation. 26105 promastigotes inoculated low pH <br /> (SAT) GFP intensity analyzed flow cytometry. After            medium 37uC trigger differentiation [9]. We monitored the <br /> negative selection addition culture 50 mg/mL GCV          axenic amastigotes 24 48 h differentiation (Fig. 5B). As <br /> during passages, dramatic shift GFP fluorescence              judged expression amastigote-specific A2 protein <br />                                                                              family [31], transgenic parasites carrying GFP-PA2G4 bona fide <br /> observed (Fig. 3C, upper panel). However, negative selection <br />                                                                              amastigotes 48 h differences observed when <br /> with GCV, mutant LmaPA2G4 parasites retained ectopic copy of <br />                                                                              compared WT. This result suggests virulence <br /> pXNG-PA2G4, shown minimal reduction GFP intensity <br />                                                                              attenuation potentially defects cell proliferation. <br /> (Fig. 3C, lower panel). These results suggest LmaPA2G4 an <br /> essential gene L. major. <br />                                                                              2D-DIGE comparative proteomics analysis <br />                                                                                 To gain better insight function LmaPA2G4, we <br /> Phenotypic characterization GFP-PA2G4 parasites <br />                                                                              quantitatively compared protein extracts L. major GFP-PA2G4 <br />    Since essentiality LmaP2G4 precluded loss-of- <br />                                                                              mock promastigotes independent biological repeats. <br /> function analysis, followed gain-of-function strategy reveal <br />                                                                              Protein samples differentially labelled CyDye fluors (GE <br /> the implication LmaPA2G4 biology Leishmania. We <br />                                                                              Healthcare) separated two-dimensional electrophoresis <br /> created parasites over-expressing N-terminal GFP-PA2G4 <br />                                                                              (2DE) IPG strips polyacrylamide gels previously described <br /> fusion protein. LmaPA2G4 ORF cloned pXG-GFP2+ <br />                                                                              [21]. A representative merged image Cy5-labeled GFP-mock <br /> as previously described [20]. We confirmed fusion western             (red) Cy3-labeled GFP-PA2G4 (green) shown (Fig. 6A). Gels <br /> blot analysis L. major FVI wild-type (WT) transgenic GFP-             scanned Typhoon FLA-9500 Imager analyzed by <br /> PA2G4 promastigotes using monoclonal anti-GFP tubulin                 Delta2D v 4.3 (Decodon) software package. Figure 6B shows a <br /> a loading control (Fig. 4A). Fluorescence intensity GFP-PA2G4             graphical representation expression profiles mock (red) and <br /> parasites measured flow cytometry (Fig. 4B). Live log-phase           GFP-PA2G4 (green) samples. Five spots significant expression <br /> transgenic promastigotes immobilized poly(l)lysine-coated            differences GFP-PA2G4 (p-value,0.005) selected. Gels <br /> 35 mm glass dishes cells analyzed using spinning             stained fluorescent stain SYPRO Ruby five <br /> disk confocal microscopy (Fig. 4C). Nuclei counterstained               spots excised identified MS/MS. Interest- <br /> with NucBlue Live Cell stain (Molecular Probes) (red). The ectopic           ingly, homologs identified eukaryotic translation initiation <br /> expression LmaPA2G4 predominantly cytoplasmic                  factor 5 (LmjF25.0720, 6.1-fold change), 60S ribosomal protein <br /> overexpression effect morphology viability              (LmjF29.2460, 5.9-fold change) 40S ribosomal protein <br /> the parasites. Growth curves WT, GFP-PA2G4 GFP-mock                   (LmjF28.0960, 5.3-fold change) implicated translation <br /> control overexpression LmaPA2G4 results                initiation elongation [32]. The chaperonin HSP60 <br /> significant growth delay (Fig. 4D). LmaPA2G4 growth defect               (LmjF36.2030, 4.4-fold change) involved stress response and <br /> reproduced transgenic parasites expressing untagged protein               acts catalyst folding proteins [33]. Raw data protein <br /> (pLEXSY-PA2G4) independent GFP expression.                  identification Mascot searches presented Table S1. <br /> We analyzed infective metacyclic stage control and <br /> transgenic lines. Similar numbers metacyclic parasites               Electrophoretic Mobility Shift Assays (EMSA) <br /> agglutinated lines (Fig. 4E), indicating over-                 dsRNA-binding domains characterize expanding family of <br /> expression LmaPA2G4 does affect metacyclogenesis.                     proteins involved different cellular processes, ranging from <br />                                                                              RNA editing processing translational control. Human <br /> GFP-PA2G4 transgenic parasites attenuated                             homolog Ebp1 interacts double stranded RNA [34] thus <br /> virulence                                                                    tempting study GFP-PA2G4 able bind a <br />   We investigated effects LmaPA2G4 overexpression                  synthetic double stranded RNA Poly (I:C). RNA molecules <br /> parasite virulence using established experimental mouse                   (Sigma) labeled Cy5 order normalize visualize <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                      6                           January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                          Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br />  <br /> Figure 3. Establishment L. major PA2G4 conditional null-mutant parasites. (A) Schematic representation null-mutant strategy. <br /> Two alleles LmaPA2G4 replaced homologous recombination hygromycin B (HYG B) puromycin (PAC) resistance markers. <br /> Replacement performed presence ectopic copy PA2G4 (pXNG- PA2G4), carrying nourseothricin (SAT) marker, fluorescent <br /> protein (GFP) Herpes simplex virus thymidine kinase (TK). (B) PCR analysis total DNA wild-type (WT) parasites independent clone <br /> (PA2G4 2/2 [pXNG-PA2G4]). F1,R1 primers (expected size 338 bp) confirm presence endogenous copy PA2G4, F2,R1 primers <br /> (expected size 513 bp) confirm episomal PA2G4 ORF. F1, R2 primer pair (expected size 413 bp) F1, R3 (expected size 440 bp) the <br /> integration hygromycin b puromycin genes, respectively. Molecular weight marker (M) shown. (C) (Upper panel). WT parasites carrying a <br /> copy pXNG-PA2G4 selected grown SAT GFP intensity analyzed flow cytometry (GCV2). After negative selection with <br /> the addition 50 mg/mL GCV (GCV+) culture (3 passages) reduction GFP fluorescence observed (lower panel). In contrast, conditional <br /> PA2G4 null-mutant parasites retained ectopic copy pXNG-PA2G4 shown minimal reduction GFP intensity. Histograms plots one <br /> representative analysis shown. Dotted lines correspond fluorescence background levels control parasites. <br /> doi:10.1371/journal.pntd.0002646.g003 <br />  <br />  <br /> the reaction. 40 ng labeled Poly(I:C) incubated 20 ng                  amastigotes initially incubated methionine-free medium. <br /> GFP-PA2G4 protein binding buffer room temperature                     Metabolic labeling performed 2 h 27uC presence <br /> 45 min resolved 10% non-denaturing polyacrylamide gels                   azidohomoalanine (AHA). A culture L. major treated 2 h <br /> in 16 TBE. Gel scanned Typhoon imager (GE) Cy5                    100 mg/mL cycloheximide, inhibitor protein biosyn- <br /> filters. Poly (I:C) visible lanes 1 3 (Fig. 7, left panel). Cy5       thesis, included positive control. After labeling, cells were <br /> filters produce non-specific background signal xylene                    harvested, lysed subjected Click-iT (Life) reaction with <br /> cyanol, loading buffer. Taking advantage                TAMRA. Proteins resolved polyacrylamide gels and <br /> the GFP fusion, gel re-scanned GFP filter,                   visualized Typhoon FLA 9500 imager fluorescence <br /> arrows indicate apparent mobility shift GFP-PA2G4                 measured ImageQuant TL software (GE Healthcare). The <br /> presence Poly (I:C) (Fig. 7, right panel).                                   gel fixed stained SYPRO Ruby order to <br />                                                                                 assess total protein content (Fig. 8). Percentage novo protein <br /> De novo protein synthesis parasites over-expressing                          synthesis shown normalized controls L. donovani L. <br />                                                                                 major WT parasites (Fig. 8, lower panel). These data suggest than <br /> LmaPA2G4                                                                        novo protein synthesis greatly reduced pro- and <br />    The results section suggest defect protein                    amastigotes over-expressing LmaPA2G4. Defects biosynthesis <br /> translation transgenic parasites, potentially non-                likely impact cell proliferation ultimately be <br /> physiological accumulation intermediates translation initia-              responsible phenotype observed. <br /> tion elongation. In order confirm phenotype, we <br /> measured novo protein synthesis transgenic lines. <br />                                                                                 Discussion <br /> Metabolic labeling novo synthetized proteins conducted <br /> using non-radioactive assay. Mid-log L. major WT GFP-                       LmaPA2G4 homolog proliferation-associated 2G4 <br /> PA2G4 promastigotes L. donovani WT GFP-PA2G4                     protein [35] termed Ebp1 [14]. The human counterparts are <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                         7                            January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                         Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br />  <br /> Figure 4. Gain-of-function strategy study LmaPA2G4. (A) WT parasites over-expressing N-terminal GFP-PA2G4 fusion protein were <br /> lysed, resolved SDS&#226;&#8364;&#8220;PAGE, electroblotted PVDF membrane analyzed western blot using monoclonal anti-GFP antibody anti- <br /> tubulin loading control. The molecular weight standard proteins indicated kDa. (B) GFP intensity GFP-PA2G4 transgenic promastigotes <br /> was analyzed Beckman Coulter FC-500 flow cytometer. 10,000 events recorded. GFP mock control parasites clearly distinguished from <br /> GFP-PA2G4. (C) Live control transgenic promastigotes logarithmic culture analyzed using spinning disk confocal microscopy. Nuclei <br /> were counterstained 1 mg/mL NucBlue Live Cell stain (Molecular Probes) (red). The bar corresponds 7 mm. (D) Growth curve L. major WT, <br /> GFP-mock, GFP-PA2G4, pLEXSY pLEXSY-PA2G4 promastigotes measured microscopically counting cells Neubauer chamber. The <br /> average standard deviation triplicate determinations shown. (E) The infective metacyclic stage control transgenic lines analyzed. <br /> Parasites day stationary cultures incubated peanut agglutinin numbers non-agglutinating metacyclics determined <br /> microscopically. Two independent experiments performed representative triplicate experiment standard deviation denoted by <br /> the bars shown. <br /> doi:10.1371/journal.pntd.0002646.g004 <br />  <br />  <br /> involved regulation cell growth differentiation [36].           characterizing potential phosphorylated residues Lma- <br /> Human homolog Ebp1 target phosphorylation PKC                  PA2G4 combination IMAC enrichment 2D LC- <br /> vitro vivo, C-terminus suggested harbor            MS/MS. <br /> the phosphorylation site [37]. Furthermore, serine 363 (S363)                 LmaPA2G4 metallopeptidase M24A family, clan MG <br /> Ebp1 phosphorylated vivo S363A <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span>                         (Fig. 2). Bio-informatics analysis multiple alignment identified <br /> significantly decreased ability Ebp1 repress transcription          members family trypanosomatids, a <br /> and abrogated ability inhibit cell growth [38]. LmaPA2G4               remarkable conservation. The crystal structure human <br /> was isolated phospho-enrichment procedure (IMAC),                   PA2G4 determined 1.6 A        &#203;&#353; resolution [13]. The <br /> suggesting phosphoprotein. We currently                       structure revealed pita-bread fold conserved methionine <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                       8                             January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                          Essential Genes Drug Targets Leishmania <br />  <br />  <br />  <br />  <br /> Figure 5. Attenuated virulence GFP-PA2G4 transgenic parasites. (A) 105 L. major metacyclic mock transgenic parasites expressing <br /> GFP-PA2G4 cured GFP-PA2G4 grown absence G418 inoculated footpad female BALB/c mice. Lesion formation was <br /> followed measuring increase footpad size Vernier caliper. Groups mice analyzed standard deviation indicated by <br /> the bars. Two independent experiments performed representative experiment shown. (B) L. donovani transgenic (GFP-PA2G4) lines <br /> were established. 26105 promastigotes inoculated low pH medium 37uC trigger differentiation axenic amastigotes. Cells lysed <br /> 24 48 h differentiation, western blot performed anti-GFP, amastigote-specific A2 antibody anti-tubulin loading control. <br /> The molecular weight standard proteins indicated kDa. <br /> doi:10.1371/journal.pntd.0002646.g005 <br />  <br /> aminopeptidases (MetAPs). In enzymes, divalent metal                   performed. This group enzymes reflected MEROPS <br /> center catalytic site involved cleavage            database non-peptidase members M24 family. Lma- <br /> appropriate substrate. However, metal ions present              PA2G4 conserved trypanosomatids, indicating an <br /> PA2G4 [13], enzymatic activity                         essential function selection pressure despite having lost its <br />  <br />  <br />  <br />  <br /> Figure 6. 2D-DIGE analysis L.major mock GFP-PA2G4 proteome. (A) Representative 2D-DIGE gel. Protein extracts from <br /> promastigotes independent biological experiments differentially labeled CyDye fluors Cy3, Cy5 Cy2, separated by <br /> two-dimensional electrophoresis 13 cm pH 4&#226;&#8364;&#8220;7 IPG strips 12.5% polyacrylamide gels. Gels scanned Typhoon FLA 9500 (GE <br /> Healthcare) imager. A merged image Cy5-labeled GFP-mock (red) Cy3-labeled GFP-PA2G4 (green) shown. The molecular weight marker <br /> proteins pI range IEF gradient indicated. (B) Differences protein abundance revealed analyzing gel images with <br /> Delta2D v4.3 software (Decodon). Histograms graphic representation significant differences. For normalization purposes, Cy2-labeled <br /> internal standard included, corresponding pool protein extracts used analysis (yellow bars). Fold change p-values the <br /> spots selected shown, protein identification mass spectrometry. Raw data MS data Mascot searches presented Table S1. <br /> doi:10.1371/journal.pntd.0002646.g006 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        9                             January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                          Essential Genes Drug Targets Leishmania <br />  <br />  <br />                                                                                 catalytic site. Further substrate binding studies necessary to <br />                                                                                 conclusively label LmaPA2G4 pseudoenzyme. Inactive <br />                                                                                 enzyme homologs simply debris functional studies, <br />                                                                                 instance iRhom family rhomboid proteases, have <br />                                                                                 revealed important roles biological regulators [39]. With the <br />                                                                                 exception pseudokinases, lack functional <br />                                                                                 information roles inactive enzymes [40]. <br />                                                                                    We confirmed LmaPA2G4 essential gene L. <br />                                                                                 major. Study promising candidate genes loss-of- <br />                                                                                 function hindered lethal mutant phenotypes our <br />                                                                                 (Fig. 3A) allows test guarding episome <br />                                                                                 (pXNG) actively quickly removed null mutant <br />                                                                                 (Fig. 3C). Moreover, indicates loss LmaPA2G4 <br />                                                                                 compensated related genes. Our findings may <br />                                                                                 applicable trypanosomatids, viability the <br />                                                                                 LmaPA2G4 null mutant carefully examined Trypano- <br /> Figure 7. Electrophoretic Mobility Shift Assay. Poly (I:C), a <br />                                                                                 soma Leishmania spp. <br /> synthetic double stranded RNA labeled Cy5 incubated                   The overexpression LmaPA2G4 did impair ability of <br /> with 20 ng GFP-PA2G4 fusion protein room temperature 45 min.             L. major differentiate infective metacyclic promastigotes <br /> The reaction resolved 10% non-denaturing polyacrylamide gels.            (Fig. 4D) L. donovani promastigotes able fully <br /> Lane 1: 40 ng poly (I:C); lane 2: 20 ng GFP-PA2G4; lane 3: 20 ng GFP-           differentiate axenic amastigotes 48 h (Fig. 5B). These <br /> PA2G4 incubated 40 ng poly (I:C). Gel scanned Typhoon <br />                                                                                 data suggest attenuated virulence observed murine <br /> scanner (GE) using 633/670 nm Cy5 489/508 nm GFP. <br /> Arrows indicate mobility shift GFP-PA2G4.                                    model (Fig. 5A) likely defect proliferation. To better <br /> doi:10.1371/journal.pntd.0002646.g007                                           understand phenotype, performed quantitative proteomics <br />  <br />  <br />  <br />  <br /> Figure 8. Over-expression PA2G4 leads reduced novo protein synthesis. Metabolic labeling novo synthetized proteins was <br /> conducted using non-radioactive Click-iT (Life) assay. Mid-log L. major WT GFP-PA2G4 promastigotes (pro) L. donovani WT GFP-PA2G4 <br /> amastigotes (ama) used experiment. Metabolic labeling performed 2 h 27uC presence 50 mM azidohomoalanine (AHA). A <br /> culture L. major treated 2 h 100 mg/mL cycloheximide, inhibitor protein biosynthesis, included positive control. <br /> Fluorescence signal TAMRA (Life) measured determine novo protein synthesis. Gels stained SYPRO Ruby (Molecular <br /> Probes) quantify signal total protein synthesis. Fluorescence intensities plotted arbitrary units fluorescence (AUF). Two independent <br /> experiments performed representative triplicate experiment standard deviation shown. Percentage novo synthesis <br /> normalized WT values shown lower panel. Median values used non-parametric Mann&#226;&#8364;&#8220;Whitney U-test. <br /> doi:10.1371/journal.pntd.0002646.g008 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        10                            January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                                            Essential Genes Drug Targets Leishmania <br />  <br />  <br /> that allowed study significant differences protein                                   Supporting Information <br /> expression result LmaPA2G4 overexpression (Fig. 6A <br /> and B). 2D DIGE proven circumvent limitation                                    Figure S1 In vitro kinase assay. GFP-PA2G4 GFPK7 <br /> traditional in-gel proteomics, especially combined                                  immuno-precipitates isolated stationary promastigotes. <br /> bottom-up proteomics [41]. The over-represented 26.1 fold                                Phosphotransferase activity determined autoradiography. <br /> change- spot corresponds eukaryotic elongation factor 5A.                              Molecular weight indicated kDa. A replica gel analyzed <br /> eIF5A protein contains modified amino acid                               western blot using monoclonal GFP antibody. Lane 1: GFP7 <br /> hypusine [42]. Hypusine formed eIF5A post-translational                              incubated GFP-PA2G4 substrate; lane 2: GFPK7 <br /> modification lysyl residues consecutive steps                            incubated 3 mg MBP; lane 3: GFP-PA2G4 immuno-complex; <br /> through action deoxyhypusine synthase (DHS)                                        lane 4: GFPK7 immuno-complex. <br /> deoxyhypusine hydroxylase (DOHH) [43]. The hypusine pathway                                   (TIF) <br /> is conserved trypanosomatids, DHS DOHH                                   Figure S2 RT-PCR analysis PA2G4 transcript levels. <br /> recently characterized T. brucei [44] L. donovani [45],                                Total RNA isolated L. major logarithmic (l) met <br /> respectively. In higher eukaryotes, eIF5A active role in <br />                                                                                               acyclic (m) promastigotes L. donovani logarithmic (l) <br /> translation elongation, precise requirement protein <br />                                                                                               promastigotes (a) amastigotes. After cDNA synthesis, PCR <br /> synthesis remains elusive [46]. 60S 40S ribosomal subunits <br />                                                                                               performed specific primers PA2G4 GAPDH. <br /> showed 5.9 5.3 fold change respectively promastigotes <br />                                                                                               Products resolved 1% agarose gels, stained SybrSafe <br /> over-expressing LmaPA2G4. In higher eukaryotes, translation <br />                                                                                               (Life) scanned Typhoon FLA 9500 imager. Band <br /> initiation starts disassociation 80S ribosomal <br />                                                                                               intensities analyzed ImageQuant TL (GE Healthcare) <br /> complex binding eIF6 60S ribosomal subunit and <br />                                                                                               PA2G4 expression normalized GAPDH signal. <br /> the binding eIF3 eIF1A 40S ribosomal subunit [47]. <br /> The HSP60 family chaperonines 24.4 fold change                                      Reactions RT used negative control. Three <br /> analysis- widely present trypanosomatids                               independent reactions carried standard deviations are <br /> potential role folding proteins imported mitochon-                             shown. <br /> dria [48]. It noteworthy proteomic approach does                              (TIF) <br /> allow confirm potential interaction LmaPA2G4                                Table S1 Raw data mass spectrometry identification <br /> the key transcription elements discussed above. The EMSA assay                                Mascot and/or Protein Pilot searches. <br /> suggests GFP-PA2G4 binds generic synthetic double                                  (PDF) <br /> stranded RNA (Fig. 7). It tempting speculate further <br /> investigations RNA binding domains allow gain                               Acknowledgments <br /> insight biological relevance binding on, instance, <br /> translational control. The fact novo protein synthesis                              We thank Dr. Gerald F. Spa&#194;&#168;th (Institute Pasteur, Paris, France) kindly <br /> significantly reduced transgenic lines (Fig. 8) new                            sharing preliminary data. We thank Prof. Steve Beverley (Washington <br /> insights transcriptional roles human counterpart [49]                               University St Louis, USA) pXNG pXG-GFP2+ systems. <br /> suggest potential role LmaPA2G4 transcription L. major. <br /> Altered transcription lines over-expressing LmaPA2G4 lines                                 Author Contributions <br /> leads defects cell growth, including pathogenic amastigote                          Conceived designed experiments: BNM KV PV MAM. Performed <br /> stage. However, investigation required dissect                              experiments: BNM KV PV MVJ MAM. Analyzed data: MAM <br /> molecular mechanisms LmaPA2G4 involved. In                                        MVJ. Contributed reagents/materials/analysis tools: MAM MVJ. Wrote <br /> conclusion, work underscores essential role biology                        paper: MAM. <br /> the parasite opens new venues potential therapeutic <br /> intervention. <br />  <br /> References <br />  1. Herwaldt BL (1999) Leishmaniasis. Lancet 354: 1191&#226;&#8364;&#8220;1199.                                  12. Yoo JY, Wang XW, Rishi AK, Lessor T, Xia XM, et al. (2000) Interaction the <br />  2. den Boer M, Argaw D, Jannin J, Alvar J (2011) Leishmaniasis impact                        PA2G4 (EBP1) protein ErbB-3 regulation binding heregulin. <br />     treatment access. Clin Microbiol Infect 17: 1471&#226;&#8364;&#8220;1477.                                        Br J Cancer 82: 683&#226;&#8364;&#8220;690. <br />  3. Alvar J, Aparicio P, Aseffa A, Den Boer M, Canavate C, et al. (2008) The                  13. Kowalinski E, Bange G, Bradatsch B, Hurt E, Wild K, et al. (2007) The crystal <br />     relationship leishmaniasis AIDS: second 10 years. Clin                        structure Ebp1 reveals methionine aminopeptidase fold binding platform <br />     Microbiol Rev 21: 334&#226;&#8364;&#8220;359                                                                     multiple interactions. FEBS Lett 581: 4450&#226;&#8364;&#8220;4454. <br />  4. Alvar J, Velez ID, Bern C, Herrero M, Desjeux P, et al. (2012) Leishmaniasis              14. Squatrito M, Mancino M, Donzelli M, Areces LB, Draetta GF (2004) EBP1 a <br />     worldwide global estimates incidence. PLoS One 7: e35671.                          nucleolar growth-regulating protein pre-ribosomal ribonucleo- <br />  5. Perez-Victoria JM, Di Pietro A, Barron D, Ravelo AG, Castanys S, et al. (2002)                protein complexes. Oncogene 23: 4454&#226;&#8364;&#8220;4465. <br />     Multidrug resistance phenotype mediated P-glycoprotein-like transporter            15. Kapler GM, Coburn CM, Beverley SM (1990) Stable transfection human <br />     Leishmania: search reversal agents. Curr Drug Targets 3: 311&#226;&#8364;&#8220;333.                    parasite Leishmania major delineates 30-kilobase region sufficient for <br />  6. Mittal MK, Rai S, Ashutosh, Ravinder, Gupta S, et al. (2007) Characterization                 extrachromosomal replication expression. Mol Cell Biol 10: 1084&#226;&#8364;&#8220;1094. <br />     natural antimony resistance leishmania donovani isolates. Am J Trop Med             16. Goyard S, Segawa H, Gordon J, Showalter M, Duncan R, et al. (2003) An in <br />     Hyg 76: 681&#226;&#8364;&#8220;688.                                                                              vitro developmental genetic studies Leishmania donovani <br />  7. Matlashewski G, Arana B, Kroeger A, Battacharya S, Sundar S, et al. (2011)                    phosphoglycans. Mol Biochem Parasitol 130: 31&#226;&#8364;&#8220;42. <br />     Visceral leishmaniasis: elimination existing interventions. Lancet Infect Dis        17. Sacks DL (1989) Metacyclogenesis Leishmania promastigotes. Exp Parasitol <br />     11: 322&#226;&#8364;&#8220;325.                                                                                  69: 100&#226;&#8364;&#8220;103. <br />  8. McConville MJ, Naderer T (2011) Metabolic pathways required                       18. Ha DS, Schwarz JK, Turco SJ, Beverley SM (1996) Use green fluorescent <br />     intracellular <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>survival</span> Leishmania. Annu Rev Microbiol 65: 543&#226;&#8364;&#8220;561.                         protein marker transfected Leishmania. Mol Biochem Parasitol 77: 57&#226;&#8364;&#8220;64. <br />  9. Zilberstein D, Shapira M (1994) The role pH temperature                     19. Murta SM, Vickers TJ, Scott DA, Beverley SM (2009) Methylene tetrahydro- <br />     development Leishmania parasites. Annu Rev Microbiol 48: 449&#226;&#8364;&#8220;470.                          folate dehydrogenase/cyclohydrolase synthesis 10-CHO-THF are <br /> 10. Hunter T (2000) Signaling&#226;&#8364;&#8220;2000 beyond. Cell 100: 113&#226;&#8364;&#8220;127.                                 essential Leishmania major. Mol Microbiol 71: 1386&#226;&#8364;&#8220;1401. <br /> 11. Morales MA, Pescher P, Spath GF (2010) Leishmania major MPK7 protein                      20. Morales MA, Renaud O, Faigle W, Shorte SL, Spath GF (2007) Over- <br />     kinase activity inhibits intracellular growth pathogenic amastigote stage.             expression Leishmania major MAP kinases reveals stage-specific induction of <br />     Eukaryot Cell 9: 22&#226;&#8364;&#8220;30.                                                                       phosphotransferase activity. Int J Parasitol 37: 1187&#226;&#8364;&#8220;1199. <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                                      11                                 January 2014 | Volume 8 | Issue 1 | e2646 <br />                                                                                                                           Essential Genes Drug Targets Leishmania <br />  <br />  <br /> 21. Morales MA, Watanabe R, Dacher M, Chafey P, Osorio y Fortea J, et al. (2010)            34. Squatrito M, Mancino M, Sala L, Draetta GF (2006) Ebp1 dsRNA-binding <br />     Phosphoproteome dynamics reveal heat-shock protein complexes specific                protein associated ribosomes modulates eIF2alpha phosphorylation. <br />     Leishmania donovani infectious stage. Proc Natl Acad Sci U S A 107: 8381&#226;&#8364;&#8220;                   Biochem Biophys Res Commun 344: 859&#226;&#8364;&#8220;868. <br />     8386.                                                                                   35. Radomski N, Jost E (1995) Molecular cloning murine cDNA encoding a <br /> 22. Morales MA, Watanabe R, Laurent C, Lenormand P, Rousselle JC, et al. (2008)                 novel protein, p38-2G4, varies cell cycle. Exp Cell Res 220: 434&#226;&#8364;&#8220; <br />     Phosphoproteomic analysis Leishmania donovani pro- amastigote stages.                445. <br />     Proteomics 8: 350&#226;&#8364;&#8220;363.                                                                  36. Yamada H, Mori H, Momoi H, Nakagawa Y, Ueguchi C, et al. (1994) A fission <br /> 23. Rawlings ND, Barrett AJ, Bateman A (2012) MEROPS: database                           yeast gene encoding protein preferentially associates curved DNA. <br />     proteolytic enzymes, substrates inhibitors. Nucleic Acids Res 40:                 Yeast 10: 883&#226;&#8364;&#8220;894. <br />     D343&#226;&#8364;&#8220;350.                                                                               37. Ahn JY, Liu X, Liu Z, Pereira L, Cheng D, et al. (2006) Nuclear Akt associates <br /> 24. Chang PC, Lee YH (1992) Extracellular autoprocessing metalloprotease                   PKC-phosphorylated Ebp1, preventing DNA fragmentation inhibition <br />     Streptomyces cacaoi. J Biol Chem 267: 3952&#226;&#8364;&#8220;3958.                                       caspase-activated DNase. EMBO J 25: 2083&#226;&#8364;&#8220;2095. <br /> 25. Lamartine J, Seri M, Cinti R, Heitzmann F, Creaven M, et al. (1997) Molecular           38. Akinmade D, Lee M, Zhang Y, Hamburger AW (2007) Ebp1-mediated <br />     cloning mapping human cDNA (PA2G4) encodes protein highly                   inhibition cell growth requires serine 363 phosphorylation. Int J Oncol 31: <br />     homologous mouse cell cycle protein p38-2G4. Cytogenet Cell Genet 78:                851&#226;&#8364;&#8220;858. <br />     31&#226;&#8364;&#8220;35.                                                                                  39. Adrain C, Freeman M (2012) New lives old: evolution pseudoenzyme <br /> 26. Bazan JF, Weaver LH, Roderick SL, Huber R, Matthews BW (1994) Sequence                      function illustrated iRhoms. Nat Rev Mol Cell Biol 13: 489&#226;&#8364;&#8220;498. <br />     structure comparison suggest methionine aminopeptidase, prolidase,             40. Zhang H, Photiou A, Grothey A, Stebbing J, Giamas G (2012) The role of <br />     aminopeptidase P, creatinase share common fold. Proc Natl Acad Sci U S A              pseudokinases cancer. Cell Signal 24: 1173&#226;&#8364;&#8220;1184. <br />                                                                                             41. Tonge R, Shaw J, Middleton B, Rowlinson R, Rayner S, et al. (2001) Validation <br />     91: 2473&#226;&#8364;&#8220;2477. <br />                                                                                                 development fluorescence two-dimensional differential gel electrophoresis <br /> 27. Langmead B, Trapnell C, Pop M, Salzberg SL (2009) Ultrafast memory- <br />                                                                                                 proteomics technology. Proteomics 1: 377&#226;&#8364;&#8220;396. <br />     efficient alignment short DNA sequences human genome. Genome Biol <br />                                                                                             42. Silvera D, Formenti SC, Schneider RJ (2010) Translational control cancer. <br />     10: R25. <br />                                                                                                 Nat Rev Cancer 10: 254&#226;&#8364;&#8220;266. <br /> 28. Sacks DL, Perkins PV (1984) Identification infective stage Leishmania          43. Park MH, Wolff EC, Folk JE (1993) Hypusine: post-translational formation in <br />     promastigotes. Science 223: 1417&#226;&#8364;&#8220;1419.                                                      eukaryotic initiation factor 5A potential role cellular regulation. <br /> 29. Spath GF, Epstein L, Leader B, Singer SM, Avila HA, et al. (2000)                           Biofactors 4: 95&#226;&#8364;&#8220;104. <br />     Lipophosphoglycan virulence factor distinct related glycoconjugates           44. Nguyen S, Jones DC, Wyllie S, Fairlamb AH, Phillips MA (2013) Allosteric <br />     protozoan parasite Leishmania major. Proc Natl Acad Sci U S A 97:                    activation trypanosomatid deoxyhypusine synthase catalytically dead <br />     9258&#226;&#8364;&#8220;9263.                                                                                  paralog. J Biol Chem 288: 15256&#226;&#8364;&#8220;15267. <br /> 30. Moreira D, Santarem N, Loureiro I, Tavares J, Silva AM, et al. (2012) Impact         45. Chawla B, Kumar RR, Tyagi N, Subramanian G, Srinivasan N, et al. (2012) A <br />     continuous axenic cultivation Leishmania infantum virulence. PLoS Negl                   unique modification eukaryotic initiation factor 5A shows presence of <br />     Trop Dis 6: e1469.                                                                          complete hypusine pathway Leishmania donovani. PLoS One 7: e33138. <br /> </body></html>