<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Syst Synth Biol (2014) 8:73&#226;&#8364;&#8220;81 <br /> DOI 10.1007/s11693-013-9123-5 <br />  <br />  RESEARCH ARTICLE <br />  <br />  <br />  <br /> The organisational structure protein networks: revisiting <br /> the centrality&#226;&#8364;&#8220;lethality hypothesis <br /> Karthik Raman &#226;&#8364;&#162; Nandita Damaraju            &#226;&#8364;&#162; <br />  <br /> Govind Krishna Joshi <br />  <br />  <br />  <br />  <br /> Received: 3 June 2013 / Revised: 5 August 2013 / Accepted: 12 August 2013 / Published online: 27 August 2013 <br /> &#195;&#8220; Springer Science+Business Media Dordrecht 2013 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> Protein networks, describing physical interac-               lethality hypothesis hold goods large number of <br /> tions functional associations proteins,            organisms, certain limitations. Betweenness centrality <br /> have unravelled organisms recent past.           useful measure identify essential nodes, but <br /> Databases <span id='am-2' about='xsd:string' typeof='owl:Thing'>STRING</span> provide excellent resources              measures like closeness centrality pairwise disconnec- <br /> for analysis networks. In contribution,           tivity significantly higher essential nodes. <br /> revisit organisation protein networks, particularly the <br /> centrality&#226;&#8364;&#8220;lethality hypothesis, hypothesises              Keywords Protein&#226;&#8364;&#8220;protein interactions   Lethality   <br /> nodes higher centrality network likely          Centrality   Network biology <br /> produce lethal phenotypes removal, compared nodes <br /> with lower centrality. We consider protein networks a <br /> diverse set 20 organisms, essentiality information            Introduction <br /> available Database Essential Genes assess the <br /> relationship centrality measures lethality. For           Protein networks, describing functional associations as <br /> each organisms, obtained networks high-                physical interactions proteins, been <br /> confidence interactions STRING database,                 unravelled organisms recent past. A <br /> computed network parameters degree, betweenness               number methods developed identify pro- <br /> centrality, closeness centrality pairwise disconnectivity         tein&#226;&#8364;&#8220;protein interactions, using experimental (Shoe- <br /> indices. We observe networks considered             maker Panchenko 2007a) computational <br /> predominantly disassortative. Further, observe                techniques (Shoemaker Panchenko 2007b). Databases <br /> essential nodes network significantly higher              DIP (Xenarios et al. 2002) STRING <br /> average degree betweenness centrality, compared            (Szklarczyk et al. 2011) provide excellent resources for <br /> network average. Most previous studies evaluated             building analysing networks proteins. The organi- <br /> centrality&#226;&#8364;&#8220;lethality hypothesis Saccharomyces cerevisi-           sation protein networks studied the <br /> ae Escherichia coli; observe centrality&#226;&#8364;&#8220;         past, particularly examining importance highly <br />                                                                       connected proteins, &#226;&#8364;&#732;hubs&#226;&#8364;&#8482;, terms essentiality, <br /> Nandita Damaraju Govind Krishna Joshi contributed equally    lethality (Batada et al. 2006; He Zhang 2006; Jeong <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> article.                                                      et al. 2001; Ning et al. 2010; Rodrigues et al. 2011; Song <br />                                                                       Singh 2013). Previous studies addressed how <br /> Electronic supplementary material The online version          complex networks attacked, targeting specific <br /> article (doi:10.1007/s11693-013-9123-5) contains supplementary <br /> material, available authorized users.                     nodes, based centrality properties (Holme et al. 2002). <br />                                                                       Many previous studies (Batada et al. 2006; He and <br /> K. Raman (&amp;)   N. Damaraju   G. K. Joshi                              Zhang 2006; Jeong et al. 2001) focussed budding <br /> Department Biotechnology, Bhupat Jyoti Mehta School of <br />                                                                       yeast, Saccharomyces cerevisiae, model organism, <br /> Biosciences, Indian Institute Technology Madras, <br /> Chennai 600 036, India                                                focus yeast Escherichia coli (Ning et al. <br /> e-mail: kraman@iitm.ac.in                                             2010). <br />  <br />  <br />                                                                                                                          123 <br />  74                                                                                                                         K. Raman et al. <br />  <br />  <br />    In contribution, revisit organisation protein        organisms, essentiality data available <br /> networks, particularly centrality&#226;&#8364;&#8220;lethality hypothesis             Database Essential Genes (see below). The <br /> (He Zhang 2006; Jeong et al. 2001), assessing                  STRING database (Szklarczyk et al. 2011) includes inter- <br /> importance set network parameters centrality               actions published literature describing experimentally <br /> measures, protein networks diverse set 20                  identified protein interactions, functional asso- <br /> organisms. The networks consider functional asso-               ciations genome sequence analysis using well- <br /> ciation networks STRING database; however,                 established methods based phylogenetic profiling, <br /> consider interactions associations reported          domain fusion gene neighbourhood concepts. For each <br /> high confidence. We analyse various network parameters,                organism, considered high-confidence inter- <br /> such degree centrality, betweenness centrality, closeness           actions, i.e. interactions STRING score greater than <br /> centrality pairwise disconnectivity index.                         700. <br />    In particular, seek answer following questions,               Data essential genes obtained Data- <br /> across diverse set 20 organisms: How protein net-             <span id='am-84' about='xsp:base' typeof='owl:Thing'>base</span> Essential Genes (DEG version 5.0; http://tubic.tju. <br /> works organised, structurally, terms connectivity         edu.cn/deg/). While DEG indexes proteins using the <br /> essential non-essential proteins? Does centrality&#226;&#8364;&#8220;             NCBI GI numbers (GenInfo Identifiers), STRING <br /> lethality hypothesis hold good different types organ-           indexes proteins using RefSeq/ENSEMBL identifiers. We <br /> isms? Do essential proteins hold special position             translated DEG identifiers STRING identifiers, using <br /> network organisation? What important network                   aliases file provided STRING database (file <br /> metrics decide protein likely essential?        protein.aliases.v9.0.txt.gz). Based data DEG, <br />                                                                        annotated proteins networks derived <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                        STRING essential non-essential. Some essential <br /> Methods                                                                proteins high-confidence interactions; this <br />                                                                        leads small discrepancy number proteins <br /> Data                                                                   listed essential DEG organism, number <br />                                                                        column Table 1. The table lists the <br /> We obtained protein networks 20 organisms                 organisms considered study, statistics on <br /> the STRING database (STRING version 9.0; http://string.                network size, number essential proteins, the <br /> embl.de/; file protein.links.v9.0.txt.gz). We chose 20           total number interactions considered. <br />  <br />  <br /> Table 1 Summary the <br />                                   Organism (NCBI taxonomy ID)                          Nodes        Essential   (%)          Edges <br /> networks considered this <br />                                                                                        (proteins)   nodes                    (interactions) <br /> study <br />                                   Acinetobacter baylyi (62977)                         2,546         468        (18.4 %)      12,996 <br />                                   Arabidopsis thaliana (3702)                          7,090         195         (2.8 %)      69,603 <br />                                   Bacillus subtilis (224308)                           3,347         219         (6.5 %)      20,728 <br />                                   Caenorhabditis elegans (6239)                        5,184         192         (3.7 %)      46,737 <br />                                   Escherichia coli (511145)                            3,789         672        (17.7 %)      25,784 <br />                                   Francisella novicida (401614)                        1,415         362        (25.6 %)       7,587 <br />                                   Haemophilus influenzae (71421)                       1,497         592        (39.5 %)       8,877 <br />                                   Helicobacter pylori (85962)                          1,352         298        (22.0 %)       7,915 <br />                                   Mycobacterium tuberculosis (83332)                   3,295         587        (17.8 %)      18,445 <br />                                   Mycoplasma genitalium (243273)                         446         363        (81.4 %)       3,376 <br />                                   Mycoplasma pulmonis (272635)                           616         288        (46.8 %)       3,111 <br />                                   Pseudomonas aeruginosa (208963)                      4,556         296         (6.5 %)      21,818 <br />                                   Saccharomyces cerevisiae (4932)                      5,477        1,109       (20.2 %)     105,429 <br /> The table lists organisms     Salmonella enterica serovar typhi (209261)           3,491         344         (9.9 %)      19,650 <br /> considered study    Salmonella typhimurium (99287)                       3,712         204         (5.5 %)      20,985 <br /> with NCBI taxonomy ID,      Staphylococcus aureus NCTC (93061)                   2,127         328        (15.4 %)       9,500 <br /> the number nodes (proteins), <br /> the number essential nodes     Staphylococcus aureus subsp. aureus N315 (158879)    1,966         296        (15.1 %)       9,207 <br /> (as obtained DEG),   Streptococcus pneumoniae (170187)                    1,718         109         (6.3 %)       8,597 <br /> number high-confidence         Streptococcus sanguinis (388919)                     1,801         215        (11.9 %)       8,315 <br /> interactions nodes <br />                                   Vibrio cholerae (243277)                             2,958         537        (18.2 %)      15,644 <br /> (as obtained STRING) <br />  <br />  <br /> 123 <br />  The organisational structure protein networks                                                                           75 <br />  <br />  <br /> Network analyses                                                &#226;&#8364;&#732;&#226;&#8364;&#732;fraction initially connected pairs vertices a <br />                                                                 network disconnected vertex v removed <br /> A number biological networks analysed using        network&#226;&#8364;&#8482;&#226;&#8364;&#8482;: <br /> concepts graph theory science. An                         N0   N v <br /> excellent introduction network biology, science       Dis&#195;&#176;v&#195;&#382; &#194;&#188; <br />                                                                               N0 <br /> analysing biological networks, elsewhere <br /> (Baraba&#194;&#180;si Oltvai 2004). For protein networks        Here, N0 total number vertex pairs network <br /> discuss here, nodes proteins, interactions      connected path <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> network, <br /> between proteins comprise edges. Nodes networks          N-v number vertex pairs remain connected <br /> be characterised parameters, evaluate          following removal vertex v. We computed these <br /> their importance network&#226;&#8364;&#8482;s structure, different     values node networks, using <br /> perspectives. We important        MATLAB Boost Graph Library MATLAB <br /> network parameters, used study. A          (http://dgleich.github.io/matlab-bgl/). <br /> more comprehensive review networks network <br /> parameters (Boccaletti et al. 2006;      Assortativity <br /> Newman 2003b). <br />                                                                 Newman (2003a) defined measure quantify as- <br /> Degree centrality                                               sortativity networks discrete types nodes. In our <br />                                                                 networks, types nodes&#226;&#8364;&#8221;essential non- <br /> Degree centrality, degree, represents number          essential. We compute assortativity coefficient as <br /> edges links node nodes network.   defined Newman: <br />                                                                                  <br />                                                                      Tr e    e2   <br /> Betweenness centrality                                          r&#194;&#188; <br />                                                                        1   ke2 k <br /> Betweenness centrality (CB) measures participation     e 2 9 2 matrix indicating fraction edges <br /> node shortest parts network. For graph G(V,E)     nodes different types (essential non- <br /> with n vertices (nodes), betweenness centrality        essential). eij matrix represents fraction edges <br /> vertex v defined as:                                         network node type node type <br />             X rst &#195;&#176;v&#195;&#382;                                           j. The trace matrix sum main diagonal <br /> CB &#195;&#176;v&#195;&#382; &#194;&#188;                                                        elements, k k denotes sum elements in <br />          s6&#194;&#188;v6&#194;&#188;t2V <br />                    rst <br />                                                                 corresponding matrix. <br /> Here, rst number shortest paths s t, and <br /> rst(v) number shortest paths s t pass <br /> through vertex v. Betweenness centrality          Results <br /> defined Freeman (1977). <br />                                                                 We analysed relationships essentiality and <br /> Closeness centrality                                            network parameters degree centrality, betweenness <br />                                                                 centrality, closeness centrality pairwise disconnectivity <br /> Closeness centrality (CC) defined reciprocal   index. We performed analyses 20 organisms with <br /> sum geodesic distances vertex      data available DEG. While Arabidopsis thaliana had <br /> vertices graph (Sabidussi 1966):                         smallest fraction essential genes, very <br />                   1                                             small organisms large fractions essential genes, <br /> CC &#195;&#176;v&#195;&#382; &#194;&#188; P                                                      viz. Mycoplasma genitalium (81.4 %) M. pulmonis <br />              t2V=v dG &#195;&#176;v; t&#195;&#382; <br />                                                                 (46.8 %). This understandable, given organ- <br /> Here dG(v,t) represents distance v t     isms small genomes, large number of <br /> graph. Note closeness centrality generally          genes genomes redundant, rather <br /> computed fully connected graph.                      important functions <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> the <br />                                                                 organism. S. cerevisiae 5,477 proteins connected <br /> Pairwise disconnectivity index                                  large number high confidence interactions. <br />                                                                 About 20 % proteins S. cerevisiae essential. <br /> The pairwise disconnectivity index defined Win-          E. coli 3,789 proteins network, 18 % <br /> gender co-workers earlier (Potapov et al. 2008),     nodes essential. <br />  <br />  <br />                                                                                                                    123 <br />  76                                                                                                                                 K. Raman et al. <br />  <br />  <br /> Protein networks predominantly disassortative/                            networks (He Zhang 2006; Jeong et al. 2001); the <br /> essential proteins tend connect non-essential                           centrality&#226;&#8364;&#8220;lethality hypothesis essentially implies that <br /> proteins                                                                      proteins essential organism&#226;&#8364;&#8482;s survival likely to <br />                                                                               higher degree (Jeong et al. 2001). We examine <br /> We observe nearly protein networks studied                    multiple metrics, addition degree, whether <br /> here low assortativity. Table 2 illustrates distri-                  essential proteins networks differ terms metrics <br /> bution various edge types (essential&#226;&#8364;&#8220;essential, non-                       betweenness centrality, closeness centrality and <br /> essential&#226;&#8364;&#8220;non-essential essential&#226;&#8364;&#8220;non-essential)                  pairwise disconnectivity index. We address question <br /> as assortativity coefficients, computed according                      means simple statistical test: null hypothesis is <br /> Newman (2003a). If network assortative, vast                      essential nodes average value a <br /> majority edges connect like nodes. In                         metric (degree, betweenness centrality, etc.) entire <br /> protein network S. typhi, assortativity                       set nodes network. We consider set essential <br /> coefficient 0.55, observe 85 % connec-                      nodes (E) particular subsample entire set of <br /> tions like nodes. In networks,                  nodes (N). Following this, create 106 random subsam- <br /> assortativity coefficient 0.5;                     ples size |E| N. We compute p-value, the <br /> number edges connect non-essential nodes                           probability observing mean value metric (in <br /> themselves, relatively edges connect essential nodes                      random subsamples) greater equal of <br /> amongst themselves, leading low overall assortativity.                     E. Table 3 lists values degree, betweenness cen- <br />                                                                               trality, closeness centrality pairwise disconnectivity <br /> Do essential nodes differ significantly certain                            index, 20 organisms. For example, S. <br /> metrics?                                                                      cerevisiae, observe degree betweenness <br />                                                                               centrality significantly higher essential nodes, vis-a`- <br /> Many previous studies highlighted critical roles                     vis entire network (p \ 10-6). Indeed, clearly <br /> played &#226;&#8364;&#732;hubs&#226;&#8364;&#8482;, highly connected proteins protein                     seen average degree essential nodes is <br />  <br /> Table 2 Assortativity coefficients networks considered study <br /> Organism                                         Nodes (proteins)     Fractions edges different types        Total edges     Assortativity (r) <br />                                                                       EE             NE             NN <br />  <br /> Arabidopsis thaliana                             7,090                0.002          0.066          0.932           69,603         0.034 <br /> Caenorhabditis elegans                           5,184                0.005          0.100          0.895           46,737         0.041 <br /> Helicobacter pylori                              1,352                0.105          0.390          0.505            7,915         0.071 <br /> Streptococcus pneumoniae                         1,718                0.019          0.171          0.810            8,597         0.084 <br /> Mycoplasma genitalium                             446                 0.825          0.158          0.017            3,376         0.088 <br /> Haemophilus influenzae                           1,497                0.249          0.454          0.296            8,877         0.090 <br /> Salmonella typhimurium                           3,712                0.020          0.145          0.836           20,985         0.135 <br /> Saccharomyces cerevisiae                         5,477                0.193          0.371          0.436          105,429         0.212 <br /> Pseudomonas aeruginosa                           4,556                0.047          0.168          0.785           21,818         0.265 <br /> Mycobacterium tuberculosis                       3,295                0.141          0.294          0.565           18,445         0.284 <br /> Escherichia coli                                 3,789                0.149          0.261          0.590           25,784         0.352 <br /> Bacillus subtilis                                3,347                0.068          0.159          0.773           20,728         0.367 <br /> Mycoplasma pulmonis                               616                 0.658          0.216          0.126            3,111         0.397 <br /> Vibrio cholerae                                  2,958                0.176          0.251          0.573           15,644         0.404 <br /> Streptococcus sanguinis                          1,801                0.184          0.252          0.565            8,315         0.411 <br /> Staphylococcus aureus subsp. aureus N315         1,966                0.235          0.274          0.491            9,207         0.413 <br /> Acinetobacter baylyi                             2,546                0.263          0.278          0.459           12,996         0.422 <br /> Francisella novicida                             1,415                0.360          0.287          0.353            7,587         0.426 <br /> Staphylococcus aureus NCTC                       2,127                0.245          0.238          0.517            9,500         0.486 <br /> Salmonella enterica serovar typhi                3,491                0.137          0.149          0.714           19,650         0.554 <br /> EE Essential&#226;&#8364;&#8220;Essential, NE Non-essential&#226;&#8364;&#8220;Essential, NN Non-essential&#226;&#8364;&#8220;Non-essential <br /> The table lists different organisms fraction different types edges assortativity coefficient, computed indicated <br /> in text. The list sorted increasing order assortativity coefficient (r) <br />  <br />  <br /> 123 <br />  The organisational structure protein networks                                                                                                 77 <br />  <br />  <br /> Table 3 p-values significance deviation mean different metrics set essential nodes vis-a`-vis nodes network <br /> Organism                                              p (degree)            p (Betweenness            p (Closeness           p (Pairwise <br />                                                                             centrality)               centrality)            disconnectivity index) <br />  <br /> Acinetobacter baylyi                                 \ 10-6                 \ 10-6                    0.22                   0.14 <br />                                                                     -3 <br /> Arabidopsis thaliana                                  8.52 9 10             0.05                      0.18                   0.15 <br /> Bacillus subtilis                                    \ 10-6                 \ 10-6                    0.89                   0.40 <br /> Caenorhabditis elegans                                8.40 9 10-5           0.25                      0.10                   0.01 <br /> Escherichia coli                                     \ 10-6                 \ 10-6                    0.92                   0.98 <br /> Francisella novicida                                 \ 10-6                 \ 10-6                    0.50                   0.49 <br />                                                            -6 <br /> Haemophilus influenzae                               \ 10                   0.26                      0.22                   0.36 <br /> Helicobacter pylori                                  \ 10-6                 6.55 9 10-4               0.86                   0.03 <br /> Mycobacterium tuberculosis                           \ 10-6                 \ 10-6                    0.20                   0.07 <br />                                                            -6 <br /> Mycoplasma genitalium                                \ 10                   0.05                      0.76                   0.18 <br /> Mycoplasma pulmonis                                  \ 10-6                 \ 10-6                    0.87                   0.00 <br /> Pseudomonas aeruginosa                               \ 10-6                 \ 10-6                    0.14                   0.05 <br />                                                            -6 <br /> Saccharomyces cerevisiae                             \ 10                   \ 10-6                    0.28                   0.09 <br /> Salmonella enterica serovar typhi                    \ 10-6                 \ 10-6                    0.92                   0.06 <br /> Salmonella typhimurium                               \ 10-6                 0.0163                    0.85                   0.15 <br /> Staphylococcus aureus NCTC                           \ 10-6                 \ 10-6                    0.24                   0.18 <br /> Staphylococcus aureus subsp. aureus N315             \ 10-6                 \ 10-6                    0.96                   0.05 <br />                                                                -6                       -4 <br /> Streptococcus pneumoniae                              2 9 10                7.14 9 10                 0.57                   0.11 <br /> Streptococcus sanguinis                              \ 10-6                 \ 10-6                    0.61                  \ 10-6 <br /> Vibrio cholerae                                      \ 10-6                 \ 10-6                    0.57                   0.89 <br /> Poor p values (p [ 0.01) shown bold. The table clearly shows metrics closeness centrality pairwise disconnectivity <br /> index used selectively delineate essential nodes <br />  <br />  <br />  <br /> significantly higher network average,                  degree distribution. The vertical axis indicates the <br /> the 20 organisms considered. Interestingly, addition                  fraction essential nodes Nx [ N, Nx set of <br /> degree centrality, betweenness centrality signifi-                                                                               d <br />                                                                             nodes degrees xth percentile above, i.e. N90 <br /> cantly higher (p \ 0.01), 15 20 organisms con-                   set nodes degrees (indicated super- <br /> sidered here.                                                               script d) 90th percentile (top 10 % of <br />    In stark contrast, closeness centrality pairwise dis-                nodes highest degrees). In S. cerevisiae, that <br /> connectivity index yeast significantly different                      d <br />                                                                             N90 , 43.8 % nodes essential, compared 20.2 % in <br /> for essential nodes compared entire network                          entire network (also Table 4). A similar plot for <br /> (p = 0.28 p = 0.09 respectively). It clear                  betweenness centrality shown Fig. 2. For N90     bc <br />                                                                                                                                       (the <br /> table metrics closeness centrality pairwise                superscript bc denotes betweenness centrality), 29 % the <br /> disconnectivity index vary significantly essential               nodes essential. Online Resources 1 2 similar <br /> nodes, organisms considered here.                            plots remaining organisms. Table 4 summarises the <br />                                                                             N90 data degree betweenness centrality. Overall, we <br /> Degree betweenness centrality correlate                                 observe that, organisms, clear increase in <br /> with lethality organisms                                            fraction essential nodes (albeit gradual), with <br />                                                                             increase degree betweenness centrality. <br /> Does degree betweenness centrality form strong <br /> indicator lethality? To address this, computed                    Closeness centrality pairwise disconnectivity index <br /> fraction nodes essential, set proteins                  strong indicators essentiality <br /> having degree greater specified value. Figure 1 <br /> shows plot end S. cerevisiae, E. coli, Staphy-                We computed closeness centrality node 20 <br /> lococcus aureus NCTC Mycobacterium tuberculosis:                        different networks. Online Resource 3 indicates varia- <br /> the horizontal axis represents increasing node degrees,                     tion fraction essential nodes 20 organisms <br /> indicated percentiles (x); representation hints              increasing closeness centrality. Clearly, observe <br />  <br />  <br />                                                                                                                                         123 <br />  78                                                                                                                                                                                     K. Raman et al. <br />  <br /> Fig. 1 Variation fraction                                                 (a) Saccharomyces cerevisiae                                                         (b)   Escherichia coli <br /> essential nodes, increase in <br />                                                                      1                                                                               1 <br />  <br />  <br />  <br />  <br />                                       Fraction essential nodes <br />  <br />  <br />  <br />  <br />                                                                                                                       Fraction essential nodes <br /> degree. The horizontal axis <br /> represents increasing node <br />  <br />  <br />  <br />  <br />                                          higher degree <br />  <br />  <br />  <br />  <br />                                                                                                                          higher degree <br />                                                                     0.8                                                                             0.8 <br /> degrees, indicated percentiles <br /> (x), vertical axis                                        0.6                                                                             0.6 <br /> indicates fraction of <br /> essential nodes Nxd , set                                 0.4                                                                             0.4 <br /> nodes degrees xth <br /> percentile above. For                                           0.2                                                                             0.2 <br /> simplicity, example <br /> organisms shown: S.                                            0                                                                               0 <br /> cerevisiae, b E. coli, c S. aureus                                        0   10 20 30 40 50 60 70 80 90 100                                              0   10 20 30 40 50 60 70 80 90 100 <br /> NCTC d M. tuberculosis.                                                      Percentile nodes&#226;&#8364;&#8482; degree                                                    Percentile nodes&#226;&#8364;&#8482; degree <br /> To illustrate, panel (d), M.                                                                                                                               (d) <br />                                                                                (c)   Mycobacterium tuberculosis                                                     Staphylococcus aureus NCTC <br /> tuberculosis, 31 % the <br />                                       Fraction essential nodes <br />  <br />  <br />  <br />                                                                      1                                                                               1 <br />  <br />  <br />  <br />  <br />                                                                                                                       Fraction essential nodes <br /> nodes highest 30 % of <br />                       d <br />                                          higher degree <br />  <br />  <br />  <br />  <br /> degrees (nodes N70   ) are <br />  <br />  <br />  <br />  <br />                                                                                                                          higher degree <br />                                                                     0.8                                                                             0.8 <br /> essential. This increases 33 % <br />      d <br /> in N90 . In panel (a), S.                                           0.6                                                                             0.6 <br /> cerevisiae, 37 % the <br />             d <br /> nodes N70   essential,                                       0.4                                                                             0.4 <br />                             d <br /> which rises 44 % N90    . For <br /> further details, text                                           0.2                                                                             0.2 <br />  <br />                                                                      0                                                                               0 <br />                                                                           0   10 20 30 40 50 60 70 80 90 100                                              0   10 20 30 40 50 60 70 80 90 100 <br />                                                                                 Percentile nodes&#226;&#8364;&#8482; degree                                                     Percentile nodes&#226;&#8364;&#8482; degree <br />  <br />  <br /> that increasing closeness centrality,                                                possible ask questions structural <br /> appreciable change fraction essential nodes. In                                             organisation networks, examine roles <br /> most cases, observe fraction essential                                             played different proteins networks. The identi- <br /> nodes remains unaffected, instance, compares                                             fication analysis essential proteins organism <br />    cc       cc <br /> N20   N80  .                                                                                      serves purpose: (1) proteins be <br />     We computed pairwise disconnectivity index                                               functionally important, mediate number of <br /> (Potapov et al. 2008) node                                                   biological processes (Batada et al. 2006), (2) proteins, <br /> organisms. Online Resource 4 indicates variation                                           particularly pathogenic organism, particular <br /> fraction essential nodes 20 organisms                                                  drug targets (Flo&#194;&#180;rez et al. 2010; Verkhedkar <br /> increasing pairwise disconnectivity indices. The sparse                                               et al. 2007). The identification essential genes/proteins <br /> plots large gaps successive points                                             organism experimentally challenging, expensive <br /> plots indicate large fraction nodes share                                          time-consuming task. Therefore, important to <br /> same pairwise disconnectivity index; further, fraction                                            identify likely essential proteins organisms through <br /> of essential nodes does increase                                                computational analyses. It possible identify <br /> increase pairwise disconnectivity index, suggesting                                           important metabolic enzymes computational <br /> this powerful metric evaluate lethality                                          techniques flux balance analysis (Joyce Palsson <br /> of node. These observations reiterate statistical tests                                         2008; Kauffman et al. 2003). While methods are <br /> illustrated Table 3, showed set                                                  powerful, require data stoichiometry of <br /> essential nodes differ significantly set                                       individual reactions, restricted enzymes/meta- <br /> nodes network, terms metrics closeness                                           bolic genes. Protein networks, hand, encom- <br /> centrality pairwise disconnectivity index.                                                         pass entire proteome organism&#226;&#8364;&#8221;therefore, an <br />                                                                                                       analysis protein network structure provide better <br />                                                                                                       picture essential proteins organism. <br /> Discussion                                                                                               Although metrics commonly used in <br />                                                                                                       graph theory network biology, clear <br /> Understanding structural organisation protein net-                                             understanding best metric predict essentiality/ <br /> works holds key understanding biological function,                                             lethality biological systems. For example, though <br /> as design principles biological networks. As                                           betweenness centrality suggested important <br /> several protein networks readily available,                                            metric identify critical nodes network (Holme et al. <br />  <br />  <br /> 123 <br />  The organisational structure protein networks                                                                                                                                                                          79 <br />  <br />  <br /> Table 4 Fraction essential proteins 10 % nodes (by degree, betweenness centrality) <br />                                                                                      d                                                                                                            bc   <br /> Organism                                       Nodes           Essential (%)        N   Essential (%)                                                                                            N        Essential    (%) <br />                                                                                      90                                                                                                           90 <br />                                                (proteins) |N0| nodes                       nodes                                                                                                          nodes <br />                                                                N0                           d                                                                                                              bc <br />                                                                                            N90                                                                                                         N90 <br />  <br /> Acinetobacter baylyi                                                               2,546                  468        (18.4)   258                138                                  (53.5)    257       101          (39.3) <br /> Arabidopsis thaliana                                                               7,090                  195         (2.8)   717                          29                          (4.0)    710        24           (3.4) <br /> Bacillus subtilis                                                                  3,347                  219         (6.5)   338                          71                         (21.0)    335        53          (15.8) <br /> Caenorhabditis elegans                                                             5,184                  192         (3.7)   519                          34                          (6.6)    519        27           (5.2) <br /> Escherichia coli                                                                   3,789                  672        (17.7)   407                150                                  (36.9)    379       118          (31.1) <br /> Francisella novicida                                                               1,415                  362        (25.6)   147                102                                  (69.4)    142        63          (44.4) <br /> Haemophilus influenzae                                                             1,497                  592        (39.5)   160                          89                         (55.6)    150        62          (41.3) <br /> Helicobacter pylori                                                                1,352                  298        (22.0)   140                          48                         (34.3)    136        40          (29.4) <br /> Mycobacterium tuberculosis                                                         3,295                  587        (17.8)   330                109                                  (33.0)    330        99          (30.0) <br /> Mycoplasma genitalium                                                                  446                363        (81.4)   45                           45                        (100.0)      46       40          (87.0) <br /> Mycoplasma pulmonis                                                                    616                288        (46.8)   63                           61                         (96.8)      62       45          (72.6) <br /> Pseudomonas aeruginosa                                                             4,556                  296         (6.5)   468                          77                         (16.5)    456        61          (13.4) <br /> Saccharomyces cerevisiae                                                           5,477                1,109        (20.2)   548                240                                  (43.8)    548       159          (29.0) <br /> Salmonella enterica serovar typhi                                                  3,491                  344         (9.9)   355                103                                  (29.0)    350        66          (18.9) <br /> Salmonella typhimurium                                                             3,712                  204         (5.5)   395                          45                         (11.4)    372        37           (9.9) <br /> Staphylococcus aureus NCTC                                                         2,127                  328        (15.4)   221                128                                  (57.9)    213        72          (33.8) <br /> Staphylococcus aureus subsp. aureus N315                                           1,966                  296        (15.1)   200                115                                  (57.5)    197        68          (34.5) <br /> Streptococcus pneumoniae                                                           1,718                  109         (6.3)   180                          25                         (13.9)    172        22          (12.8) <br /> Streptococcus sanguinis                                                            1,801                  215        (11.9)   181                          80                         (44.2)    181        58          (32.0) <br /> Vibrio cholerae                                                                    2,958                  537        (18.2)   314                130                                  (41.4)    296        83          (28.0) <br /> The table lists organisms considered study, numbers fraction essential nodes entire network, for <br /> sets nodes highest degree betweenness centralities. The table clearly illustrates enrichment essential nodes N90 sets: in <br /> all cases, increase fraction essential proteins N90 sets, organisms compared others <br />  <br />  <br /> Fig. 2 Variation fraction                                                              (a) Saccharomyces cerevisiae                                                                   (b) Escherichia coli <br />                                        higher betwenness centrality <br />  <br />  <br />  <br />  <br />                                                                                                                                     higher betwenness centrality <br />  <br />  <br />  <br />  <br /> essential nodes, increase in <br />                                                                               1                                                                                            1 <br /> betweenness centrality. The <br />                                            Fraction essential nodes <br />  <br />  <br />  <br />  <br />                                                                                                                                         Fraction essential nodes <br />  <br />  <br />  <br />  <br /> horizontal axis represents                                                   0.8                                                                                          0.8 <br /> increasing node betweenness <br /> centralities, indicated                                                   0.6                                                                                          0.6 <br /> percentiles (x), the <br /> vertical axis indicates                                                  0.4                                                                                          0.4 <br /> fraction essential nodes in <br /> Nxbc , set nodes                                                 0.2                                                                                          0.2 <br /> betweenness centralities the <br /> xth percentile above. For                                                 0                                                                                            0 <br />                                                                                    0    10 20 30 40 50 60 70 80 90 100                                                          0   10 20 30 40 50 60 70 80 90 100 <br /> further details, text <br />                                                                                     Percentile nodes&#226;&#8364;&#8482; betwenness centrality                                                  Percentile nodes&#226;&#8364;&#8482; betwenness centrality <br />  <br />  <br />                                                                                          (c)    Mycobacterium tuberculosis                                                          (d)   Staphylococcus aureus NCTC <br />                                        higher betwenness centrality <br />  <br />  <br />  <br />  <br />                                                                                                                                     higher betwenness centrality <br />  <br />  <br />  <br />  <br />                                                                               1                                                                                            1 <br />                                            Fraction essential nodes <br />  <br />  <br />  <br />  <br />                                                                                                                                         Fraction essential nodes <br />  <br />  <br />  <br />  <br />                                                                              0.8                                                                                          0.8 <br />  <br />                                                                              0.6                                                                                          0.6 <br />  <br />                                                                              0.4                                                                                          0.4 <br />  <br />                                                                              0.2                                                                                          0.2 <br />  <br />                                                                               0                                                                                            0 <br />                                                                                    0    10 20 30 40 50 60 70 80 90 100                                                          0   10 20 30 40 50 60 70 80 90 100 <br />                                                                                    Percentile nodes &#226;&#8364;&#8482; betwenness centrality                                                  Percentile nodes&#226;&#8364;&#8482; betwenness centrality <br />  <br />  <br />  <br />                                                                                                                                                                                                                    123 <br />  80                                                                                                                       K. Raman et al. <br />  <br />  <br /> 2002), researchers argued betweenness             gaps existing knowledge essential genes in <br /> centrality informative cases (Potapov         organisms considered; particular, lim- <br /> et al. 2008).                                                     ited gaps DEG. <br />    Following analysis multiple network measures             Overall, major contribution study the <br /> this study, make broad observations, set        identification hubs highly connected proteins is <br /> diverse organisms. Firstly, observe nearly        insufficient identify essential proteins networks. <br /> protein networks considered study disassortative;     Importantly, perform extensive analysis protein <br /> this agreement previous studies Newman              networks 20 diverse organisms, which, knowl- <br /> (2002). Also, interactions happen non-               edge, carried before. These organisms <br /> essential proteins, essential non-essential        differ substantially number proteins, fraction <br /> proteins; interactions essential proteins        essential proteins, density interactions on. <br /> rarer. Secondly, observe essential nodes           Therefore, analysis diverse set organisms <br /> significantly higher average degree compared net-          scrutinises centrality&#226;&#8364;&#8220;lethality hypothesis criti- <br /> work average organisms; centrality&#226;&#8364;&#8220;lethality           cally, enables make general statements the <br /> hypothesis appears hold larger set organ-        associations centrality lethality different <br /> isms. However, rate fraction essential        organisms. Although degree centrality, betweenness cen- <br /> nodes increases, increase degree slow. For             trality lethality correlated organisms, is <br /> example, M. tuberculosis, 31 % nodes        possible predict lethal nodes organism a <br />                                         d                         large degree accuracy using merely degree/betweenness. <br /> highest 30 % degrees (nodes N70     ) essential. This <br />                              d <br /> increases 33 % N90 . Only higher degrees       Further, observe metrics like pairwise disconnec- <br /> (top 2 %), does fraction essential nodes near 49 %.        tivity index poorer indicators essentiality in <br /> Even S. cerevisiae, 37 % nodes N70    d <br />                                                               protein networks, despite fact useful metric to <br />                                        d             d <br /> essential, rises 44 % N90 ; N99         analyse critical nodes complex networks (Potapov et al. <br />                                                                   2008). Indeed, results reiterate observation Roy <br /> observe nearly 73 % essential nodes. Therefore, while <br />                                                                   (Roy 2012; Roy Filkov 2009) indi- <br /> higher degree indicator lethality general, <br />                                                                   vidual metrics sufficient analyse phe- <br /> high degree does automatically imply lethality. The <br />                                                                   notypes organism. Our results warrant further <br /> average betweenness centrality essential proteins is <br />                                                                   exploration organisation protein networks; mere <br /> significantly higher compared network average in <br />                                                                   analysis hubs network completely explain <br /> most organisms considered here; however, similar to <br />                                                                   complex organisation protein networks. <br /> degree, high betweenness centrality does automatically <br /> imply essentiality. Perhaps, fruitful explore      Acknowledgments Funding IIT Madras seed grant <br /> combinations metrics, stronger                BIO/12-13/579/NFSC/KARH gratefully acknowledged. KR wishes <br /> association lethality. Finally, observe metrics      thank Dr. Saketha Nath (IIT Bombay) useful <br /> such closeness centrality pairwise disconnectivity         discussions. <br /> index, useful predict/analyse critical nodes          Conflict   The authors declare conflict <br /> complex networks, insufficient predict lethality        interest. <br /> proteins networks. <br />    Our work does limitations. Firstly, consider <br /> protein networks composed physical           References <br /> interactions, functional associations. However, we <br /> consider high-confidence associations reported        Baraba&#194;&#180;si A-L, Oltvai ZN (2004) Network biology: understanding the <br />                                                                         cell&#226;&#8364;&#8482;s functional organization. Nat Rev Genet 5:101&#226;&#8364;&#8220;113. doi:10. <br /> the STRING database, considerably limit any <br />                                                                         1038/nrg1272 <br /> false positive associations networks. The advantage,       Batada NN, Hurst LD, Tyers M (2006) Evolutionary physiolog- <br /> however, that, interactions present complete            ical importance hub proteins. PLoS Comput Biol 2:e88. <br /> view protein function cell, comparison              doi:10.1371/journal.pcbi.0020088 <br />                                                                   Boccaletti S, Latora V, Moreno Y et al (2006) Complex networks: <br /> networks composed merely physically interacting pro- <br />                                                                         structure dynamics. Phys Rep 424:175&#226;&#8364;&#8220;308. doi:10.1016/j. <br /> tein pairs. While considered organisms                physrep.2005.10.009 <br /> some previous studies, possible results     Flo&#194;&#180;rez AF, Park D, Bhak J et al (2010) Protein network prediction and <br /> could altered consider larger               topological analysis Leishmania major tool drug target <br />                                                                         selection. BMC Bioinformatics 11:484. doi:10.1186/1471-2105- <br /> even diverse set organisms. Here, quite <br />                                                                         11-484 <br /> limited availability data gene essentiality       Freeman LC (1977) A set measures centrality based on <br /> various organisms. Further, currently limited               betweenness. Sociometry 40:35&#226;&#8364;&#8220;41. doi:10.2307/3033543 <br />  <br />  <br />  <br /> 123 <br />  The organisational structure protein networks                                                                                             81 <br />  <br /> He X, Zhang J (2006) Why hubs tend essential protein       Roy S (2012) Systems biology degree, hubs scale-free <br />     networks? PLoS Genet 2:e88. doi:10.1371/journal.pgen.0020088            networks: case multiple metrics complex networks. <br /> Holme P, Kim B, Yoon C, Han S (2002) Attack vulnerability complex        Syst Synth Biol 6:31&#226;&#8364;&#8220;34. doi:10.1007/s11693-012-9094-y <br />     networks. Phys Rev E65:056109. doi:10.1103/PhysRevE.65.            Roy S, Filkov V (2009) Strong associations microbe <br />     056109                                                                  phenotypes network architecture. Phys Rev E80:040 <br /> Jeong H, Mason SP, Baraba&#194;&#180;si AL, Oltvai ZN (2001) Lethality             902. doi:10.1103/PhysRevE.80.040902 <br />     centrality protein networks. Nature 411:41&#226;&#8364;&#8220;42. doi:10.1038/     Sabidussi G (1966) The centrality index graph. Psychometrika <br />     35075138                                                                31:581&#226;&#8364;&#8220;603. doi:10.1007/BF02289527 <br /> Joyce AR, Palsson B&#195;&#732; (2008) Predicting gene essentiality using         Shoemaker BA, Panchenko AR (2007a) Deciphering protein&#226;&#8364;&#8220;protein <br />     genome-scale silico models. Methods mole biol (Clifton, NJ)          interactions. Part I. experimental techniques databases. PLoS <br />     416:433&#226;&#8364;&#8220;457. doi:10.1007/978-1-59745-321-9_30                           Comput Biol 3:e42. doi:10.1371/journal.pcbi.0030042 <br /> Kauffman KJ, Prakash P, Edwards JS (2003) Advances flux             Shoemaker BA, Panchenko AR (2007b) Deciphering protein&#226;&#8364;&#8220;protein <br />     balance analysis. Curr Opin Biotechnol 14:491&#226;&#8364;&#8220;496. doi:10.              interactions. Part II. computational methods predict protein <br />     1016/j.copbio.2003.08.001                                               domain interaction partners. PLoS Comput Biol 3:e43. <br /> Newman M (2002) Assortative mixing networks. Phys Rev Lett               doi:10.1371/journal.pcbi.0030043 <br />     89:208701. doi:10.1103/PhysRevLett.89.208701                       Song J, Singh M (2013) From hub proteins hub modules: the <br /> Newman MEJ (2003a) Mixing patterns networks. Phys Rev E                  relationship essentiality centrality yeast <br />     67:026126. doi: 10.1103/physreve.67.026126                              interactome different scales organization. PLoS Comput <br /> Newman MEJ (2003b) The structure function complex networks.          Biol 9:e1002910. doi:10.1371/journal.pcbi.1002910 <br />     SIAM Rev 45:167&#226;&#8364;&#8220;256. doi:10.1137/S003614450342480                  Szklarczyk D, Franceschini A, Kuhn M et al (2011) The STRING <br /> Ning K, Ng H, Srihari S et al (2010) Examination relationship        database 2011: functional interaction networks proteins, <br />     essential genes PPI network hub proteins              globally integrated scored. Nucleic Acids Res 39:D561&#226;&#8364;&#8220; <br />     reverse nearest neighbor topology. BMC Bioinformatics. doi:10.          D568. doi:10.1093/nar/gkq973 <br />     1186/1471-2105-11-505                                              Verkhedkar KD, Raman K, Chandra NR, Vishveshwara S (2007) <br /> Potapov AP, Goemann B, Wingender E (2008) The pairwise                      Metabolome Based Reaction Graphs M. tuberculosis M. <br />     disconnectivity index new metric topological               leprae: Comparative Network Analysis. PLoS ONE. doi:10. <br />     analysis regulatory networks. BMC Bioinformatics 9:227.              1371/jour <br /> </body></html>