<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Gemin Associates <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>Survival</span> Motor Neuron Are <br /> Required Motor Function Drosophila <br /> Rebecca Borg1,2, Ruben J. Cauchi1* <br /> 1 Department Physiology Biochemistry, University Malta, Msida, Malta G.C, 2 Department Applied Biomedical Sciences, University Malta, Msida, Malta G.C <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Membership survival motor neuron (SMN) complex extends factors, including SMN protein, product of <br />      spinal muscular atrophy (SMA) disease gene, Gemins 2&#226;&#8364;&#8220;8 Unrip. The best-characterised function this <br />      macromolecular machine assembly Sm-class uridine-rich small nuclear ribonucleoprotein (snRNP) particles <br />      SMN complex member key role process. So far, however, little known function <br />      individual Gemin components vivo. Here, make use Drosophila model organism uncover loss-of- <br />      function phenotypes Gemin2, Gemin3 Gemin5, SMN form minimalistic fly SMN complex. We <br />      ectopic overexpression dead helicase Gem3DN mutant knockdown Gemin3 result similar motor <br />      phenotypes, restricted muscle, combination cause lethality, suggesting Gem3DN overexpression <br />      mimics loss-of-function. Based localisation <span id='am-635' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Gem3DN, predict nucleus primary site the <br />      antimorphic dominant-negative mechanism Gem3DN-mediated interference. Interestingly, phenotypes induced by <br />      human SMN overexpression Drosophila exhibit similarities induced overexpression Gem3DN. Through <br />      enhanced knockdown uncover requirement Gemin2, Gemin3 Gemin5 viability motor behaviour, <br />      including locomotion flight, muscle. Notably, case Gemin3 Gemin5, function depends on <br />      adequate levels respective protein neurons. Overall, findings lead speculate absence one <br />      member sufficient arrest SMN-Gemins complex function nucleocentric pathway, critical motor <br />      function vivo. <br />  <br />   Citation: Borg R, Cauchi RJ (2013) The Gemin Associates Survival Motor Neuron Are Required Motor Function Drosophila. PLoS ONE 8(12): e83878. <br />   doi:10.1371/journal.pone.0083878 <br />   Editor: Giorgio F. Gilestro, Imperial College London, United Kingdom <br />   Received October 4, 2013; Accepted November 9, 2013; Published December 31, 2013 <br />   Copyright: &#195;&#376; 2013 Borg, Cauchi. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported University Malta (PHBRP09; MDSIN08-24) Malta Council Science &amp; Technology National <br />   Research &amp; Innovation Programme 2012 (R&amp;I-2012-066). The funders role study design, data collection analysis, decision publish, preparation <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: ruben.cauchi@um.edu.mt <br />  <br />  <br />  <br /> Introduction                                                                          cell [22,28]. The best-characterised function SMN complex <br />                                                                                       revolves biogenesis uridine-rich small nuclear <br />    Spinal muscular atrophy (SMA) counts expanding                            ribonucleoproteins (snRNPs) central elements the <br /> ensemble motor neuron degenerative disorders resulting                        spliceosome and, hence, crucial yielding mature spliced <br /> mutations genes encode proteins crucial roles RNA                     mRNAs [6,21,29]. All SMN complex members bar Unrip are <br /> metabolism. The correct handling processing RNA                       indispensable uploading heptameric ring Sm <br /> transcription eventual translation emerging key                       proteins small nuclear RNAs (snRNAs) form core <br /> function survival motor <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, relative tissues,               structure Sm-class snRNPs [16,20,30,31,32]. <br /> reasons unclear [1,2,3,4]. A common genetic cause infant                       Details role member SMN complex in <br /> death, SMA recessively inherited characterised loss                   assembly process scant. Sm proteins thought be <br /> lower motor neurons progressive muscle weakness                        recognised Gemin2, wraps itself pentamer <br /> wasting [5]. In majority cases, devastating disorder               formed Sm D1/D2/E/F/G contact Sm proteins. <br /> result low levels ubiquitously-expressed survival motor                     Importantly, N-terminal tail Gemin2 reaches the <br /> neuron (SMN) protein [6]. SMN forms oligomers [7,8,9]                            snRNA-binding pocket Sm pentamer occlude RNA <br /> associate core group proteins including Gemin2-Gemin8                       binding, presumably delivery bona fide RNA <br /> [10,11,12,13,14,15,16,17,18] Unrip [19,20] form large                      substrates, snRNAs [33]. Gemin5 factor identifies <br /> macromolecular SMN complex (reviewed [21]).                                        snRNAs stringent recognition <span id='am-7' about='Thesaurus:code' typeof='owl:Thing'>code</span> formed of <br />    Across metazoa, SMN complex particularly concentrated                       sequences structural motifs [34]. Newly exported <br /> in prominent cellular bodies, called gems                          snRNAs captured SMN complex-independent Gemin5 via <br /> nucleus [22,23] U bodies dot perinuclear periphery                     N-terminal WD-repeat domain [35], docks the <br /> in cytoplasm [24,25]. The exact function organelles                   SMN complex, probably proximate Gemin2, deliver its <br /> not known share characteristics (reviewed                     load Sm core assembly [36]. Gemin3 functional DEAD- <br /> in [21]), including crosstalk compartment-specific                         box RNA helicase [37], necessary chaperoning <br /> organelles [22,26,27] response metabolic changes                        RNA and, eventually, RNP complexes assembly <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                          December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                  A Motor Function Gemin Associates SMN <br />  <br />  <br /> reaction intimately-associated Gemin4, possibly acting           Results <br /> a co-factor activities. Regarding SMN considering its <br /> oligomeric nature cellular milieu, highly probable          Gemin3 Knockdown Muscle Tissue Results Adult <br /> that SMN, aided SMN complex components, acts                   Age-progressive Climbing Defects <br /> scaffold assembly reaction occurring                 According Muller&#226;&#8364;&#8482;s 1932 classical classification mutations, <br /> simultaneously.                                                              antimorph defined mutant allele antagonises co- <br />     Whether decreased capacity assemble snRNPs and,                     expressed wild-type gene product, reducing functional <br /> consequently, defective splicing, accounts selective neuro-          activity [45,46]. In era molecular basis Muller&#226;&#8364;&#8482;s <br /> muscular <span id='am-167' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> SMA passionately debatable topic            antimorphs comprehended, Herskowitz [47] coined the <br /> in field. An alternative view proposes SMN motor              term dominant-negative used synonymously with <br /> neuron-specific function secondary housekeeping               antimorph since. Helicase-inactivating mutations exert a <br /> activities, loss critical function context       spectrum dominant-negative phenotypes [48]. DEAD-box <br /> sufficient snRNP assembly explain profound tissue-            RNA helicase Gemin3 present metazoans high levels <br /> specificity observed SMA patients [6,38]. However,         amino acid conservation observed N-terminal <br /> open question select members SMN complex               helicase motifs [40,49]. We recently reported generation of <br /> participate non-canonical function present. In           truncated version Gemin3 lacking helicase domains, <br /> part allured presence functional minimalistic complex            Gem3DN (FIG S1A). Whereas expression catalytically <br /> composed SMN, Gemin2, Gemin3 Gemin5                              inactive mutant fails rescue lethality associated a <br /> [21,25,39,40,41], exploited Drosophila genetic systems          Gemin3 null background, ubiquitous pan-muscular-specific <br /> model investigate workings SMN complex vivo and,            overexpression wild-type organisms, lethal ( [40] data not <br /> hence, shed light conundrums. Recently, reported            shown). A milder pan-muscular overexpression however <br /> Gemin3 null larval mutants exhibit phenotypes strikingly           associated flightless phenotype [40]. <br /> similar observed Smn mutants including decline                We attempted investigate mechanism which <br /> mobility defects neuromuscular junction [40,42].                  overexpression Gem3DN mutant confers dominant- <br /> Interestingly, overexpression truncated Gemin3 mutant                   negative antimorphic phenotype. First, asked loss <br /> lacking helicase core (Gem3DN) Gemin3 knockdown,                   Gemin3 function <span id='am-557' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-558' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-559' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-560' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-561' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-562' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-563' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-564' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-565' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-566' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-567' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-568' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-569' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-570' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-571' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-572' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-573' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-574' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-575' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-576' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-577' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-578' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-579' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-580' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-581' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-582' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-583' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-584' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-585' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-586' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-587' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-588' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-589' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-590' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-591' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-592' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-593' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-594' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-595' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-596' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-597' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-598' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-599' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-600' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-601' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-602' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-603' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-604' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-605' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-606' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-607' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-608' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-609' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-610' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-611' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-612' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-613' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-614' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-615' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-616' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-617' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-618' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-619' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-620' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-621' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-622' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-623' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-624' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-625' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-626' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-627' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-628' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-629' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-630' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-631' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-632' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-633' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-634' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi)-mediated <br /> muscle neuronal tissue, drastic impact                knockdown gives rise identical phenotype. <span id='am-10' about='protege:TO' typeof='owl:Thing'>To</span> end, <br />                                                                              making use versatile GAL4/upstream activation sequence <br /> adult viability. Remarkably, pan-muscular overexpression is <br />                                                                              (UAS) (reviewed [50]), expressed inducible RNAi <br /> milder, Gem3DN confers flight defects flight muscle atrophy, a <br />                                                                              transgene targeting Gemin3 (Gem3-IRmun; FIG S1A) muscle <br /> phenotype reminiscent observed hypomorphic Smn <br />                                                                              tissues using pan-muscular Mef2-GAL4 driver. The specificity <br /> mutant (SmnE33) reported low SMN protein levels adult <br />                                                                              transgene demonstrated previously [40], co- <br /> flight muscles [40,43]. <br />                                                                              expression Dicer-2 (Dcr-2) performed order to <br />     In present study, sought investigate dominant- <br />                                                                              enhance knockdown efficiency [51]. Flies subjected a <br /> negative antimorphic mechanism responsible overex- <br />                                                                              climbing assay 5, 15, 25 35 days post-eclosion. The first <br /> pression phenotype Gem3DN. We loss-of-function                  parameter measured time taken fly the <br /> through knockdown Gemin3 muscle tissue gives rise                   sampled population climb height 8 cm. At days <br /> similar motor phenotypes including climbing defects loss              measured, fly Mef2-GAL4.Dcr-2+Gem3-IRmun fly <br /> flight. Notably, Gemin3 knockdown overexpression                      population took, average, significantly longer reach the <br /> Gem3DN, combination singularly, muscle              target height compared controls (FIG. 1A). To better reflect the <br /> or neuronal tissue organism lethal. These findings            climbing ability flies sampled population, counted the <br /> indicate Gem3DN mimics loss-of-function presumably                 number flies reached height 8 cm 10 seconds to <br /> interfering level activity Gemin3 protein         determine second parameter, climbing success rate. <br /> its complex. Tagging Gem3DN GFP allowed delve                Similar data time fly, flies reduced levels <br /> the sub-cellular location interference, predict          Gemin3 significantly lower average success rate time <br /> be predominantly nuclear similar certain aspects           points compared controls (FIG. 1A). Furthermore, their <br /> human SMN, previously reported                          success rate showed age-dependent progressive decline in <br /> antimorphic [44]. Through enhanced knockdown screen                    climbing ability; hence, worst result obtained last <br /> diverse spatial temporal patterns, Gemin3           time point measured, namely, day 35 post-eclosion. Notably, these <br /> essential viability flight muscle        results similar obtained flies GAL4/UAS- <br /> brain. Intriguingly, uncover similar requirement Gemin5,            mediated pan-muscular overexpression Gem3DN. Indeed, <br /> hence enhanced knockdown brain muscle tissue                compared controls, performers Mef2-GAL4.Gem3DN <br /> gives rise viability flight defects. We finally tackle            population took progressively longer, average, reach the <br /> function Gemin2, important                target height contrast controls. The average climbing success <br /> motor <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> motor behaviour including flight. Placed           rate population significantly lower of <br /> context previous studies, results lead hypothesise        controls, showed age-dependent deterioration (FIG. 1B). <br /> the presence common nucleocentric pathway process                Therefore, Gemin3 attenuation muscle leads mobility deficits <br /> SMN Gemin associates critical motor function             similar resulting ectopic Gem3DN <br /> in vivo.                                                                     overexpression tissue. <br />  <br />                                                                              Gemin3 Knockdown Muscle Tissue Leads Age- <br />                                                                              dependent Flight Defects <br />                                                                                Metazoans invest substantially muscles power movement <br />                                                                              especially true insects flight muscles can <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                       December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                     A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 1. Pan-muscular Gemin3 knockdown Gem3DN overexpression gives rise identical motor phenotypes. (A) Compared to <br /> controls (Mef2-GAL4.Dcr-2 Gem3-IRmun/+), time taken fly climb 8 cm increases progressively course 35 days in <br /> sample populations having reduced levels Gemin3 muscles (Mef2-GAL4.Dcr-2+Gem3-IRmun). The genotype exhibits age- <br /> dependent progressive decline total percentage flies successfully reached 8 cm mark 10 seconds compared the <br /> same controls. (B) An identical progressive change climbing behaviour observed flies pan-muscular Gem3DN overexpression (Mef2- <br /> GAL4.Gem3DN) uncombined driver (Mef2-GAL4/+) transgene (Gem3DN/+) controls. The day 25 day 35 time points excluded <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> the chart lack difference driver control (Mef2-GAL4/+). (C) Flight behaviour determined using Droso-Drome, the <br /> height fly falls determines flight performance. Fliers tend stick upper sectors non-fliers drop sector 1, lowest sector. On <br /> day 5 day 15 post-eclosion, percentage flight-impaired organisms Gemin3 knockdown population (Mef2-GAL4.Dcr-2+Gem3-IRmun) <br /> is significantly different controls. This situation changes day 25 day 35 time points significant difference is <br /> obvious. At day 35, percentage flightless flies Mef2-GAL4.Dcr-2+Gem3-IRmun population majority significant <br /> differences flies overexpressing Gem3DN (Mef2-GAL4.Gem3DN) persist time point. Data presented mean 6 S.E.M. and <br /> statistical significance determined differences Mef2-GAL4.Dcr-2+Gem3-IRmun (A, C) Mef2-GAL4.Gem3DN (B) genotype and <br /> other genotypes, indicated respective colour. For data, ns = significant, *p,0.05, **p,0.01, ***p,0.001. <br /> doi:10.1371/journal.pone.0083878.g001 <br />  <br /> make 65% total body mass [52]. Some insects                   In context, climbing ability, wanted determine <br /> including Drosophila evolved remarkable muscle capable             flight performance sampled population flies. To this <br /> generating high power high frequency, generating                       end, developed Droso-Drome, device enables <br /> sufficient force offset gravity [53]. At time,                systematic quantification flight defects based sensitive <br /> remains substantial conservation insects mammals                 &#226;&#8364;&#732;cylinder drop assay&#226;&#8364;&#8482; described previously [56]. Basically, the <br /> basic cell developmental biology muscles [54,55]. In                     height fly falls determines flight capability, fliers able to <br /> addition powerful lightweight engine supplied flight                hold walls upper sectors flight defective <br /> muscles, animal capable active flight possess wings             organisms fall lower sectors. We observe when <br /> that capable generating sufficient aerodynamic forces            subjected assay, previously reported [40], majority <br /> as control system, motor neurons pivotal,                flies pan-muscular overexpression Gem3DN flightless <br /> the organism plummeting ground [53]. Deficiency                day 5 post-eclosion, earliest time point measured, and, <br /> protein indispensible function                   hence, 66% flies fall straight sector 1, lowest sector <br /> elements &#226;&#8364;&#8220; muscles, wings sensory-motor circuit &#226;&#8364;&#8220; predicted              (FIG. 1C). At day 5 15 post-eclosion, compared driver- <br /> to negative impact flight.                                            (Mef2-GAL4.Dcr-2) transgene-only (Gem3-IRmun/+) controls, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  3                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                 A Motor Function Gemin Associates SMN <br />  <br />  <br /> we did observe large differences fly population induced           helicase protein antagonises interferes function the <br /> to pan-muscular knockdown Gemin3 (Mef2-GAL4.Dcr-                     endogenous Gemin3 protein. We note overexpression the <br /> 2+Gem3-IRmun). Furthermore, time points, flies                 GFP.Gem3DN fusion protein muscle induces flightless <br /> behave differently Mef2-GAL4.Gem3DN fly population                  phenotype observed tag-free counterpart, hence, indicating <br /> indicating fly well. Although difference                insertion epitope tag did alter antagonistic <br /> remained, started notice differences percentage              effects Gem3DN (data shown). Importantly, staining for <br /> non-fliers Gemin3 knockdown flies (31%) controls                 GFP, observed GFP.Gem3DN ubiquitously <br /> (Mef2-GAL4.Dcr-2, 11% Gem3-IRmun/+, 7%) day 25                    present muscles, localisation pattern predomi- <br /> time point. However, dramatic difference observed day               nantly nuclear (FIG. 2B). In contrast full-<span id='am-8' about='xsp:length' typeof='owl:Thing'>length</span> wild-type <br /> 35 post-eclosion distribution Mef2-GAL4.Dcr-                  Gemin3, localises intensely-stained foci (gems) in <br /> 2+Gem3-IRmun flies significantly different exhibited          nucleus [23], Gem3DN gives diffuse nuclear staining pattern. <br /> the control populations similar Mef2-GAL4.Gem3DN <br /> flies. In regard, time point, flight-defective organisms        Phenotypes Resulting Ectopic Human SMN <br /> that tumbled straight sector 1 majority the <br />                                                                              Overexpression Similar Resulting from <br /> Mef2-GAL4.Dcr-2+Gem3-IRmun (57%) Mef2-GAL4.Gem3DN <br /> (80%) populations (FIG. 1C). This indicates similar                  Gem3DN-mediated Interference <br /> Gem3DN overexpression, Gemin3 knockdown leads flight                       Ectopic overexpression human SMN (hSMN) protein in <br /> defective phenotype, albeit age-dependent.                                   wild-type flies reported antagonise endogenous <br />                                                                              Drosophila SMN (dSMN) function. The presumed mechanism <br />                                                                              responsible resulting dominant-negative phenotype has <br /> Gemin3 knockdown And Gem3DN Overexpression, in <br />                                                                              ascribed binding hSMN endogenous dSMN, <br /> Combination, Lead Lethality                                               eventually sequestering non-functional hSMN/ <br />    If Gem3DN overexpression mimics loss-of-function,                    dSMN hetero-oligomers [44]. Ubiquitous expression hSMN but <br /> predicted Gem3DN overexpression combination                     dSMN Smn null background (SmnX7/SmnX7) fails to <br /> Gemin3 knockdown organism lead                    rescue lethality associated genetic background (data <br /> phenotype severe resulting both <br />                                                                              shown), probably result divergence dSMN. <br /> genetic manipulations applied singularly. To test this <br />                                                                              To shed light mechanism action Gem3DN, we <br /> hypothesis, generated organisms Mef2-GAL4-driven <br />                                                                              probed similarities differences down- <br /> Gemin3 knockdown Gem3DN overexpression. Remarkably, in <br />                                                                              stream events resulting hSMN Gem3DN overexpression. <br /> combination muscle, genetic manipulations result in <br />                                                                              To end, noted similar previously reported for <br /> lethality total loss adult viability (FIG. 2A). This phenotype <br />                                                                              Gem3DN [40], pan-muscular pan-neuronal overexpression <br /> contrasts heavily observed elements, distinctive- <br />                                                                              hSMN induces lethality reduced adult viability (FIG. 3A). <br /> ly. In regard, pan-muscular Gem3DN overexpression has <br />                                                                              Furthermore, similar Gem3DN overexpression (see <br /> no effect adult viability flight impaired (FIG. 2A and <br />                                                                              above), flies hSMN overexpression muscle exhibited <br /> above). Furthermore, RNAi-induced depletion Gemin3 levels in <br />                                                                              climbing defects (FIG. 3B). Despite observations, in <br /> muscle, itself, lethal, results age-dependent <br />                                                                              contrast Gem3DN, did identify large differences in <br /> progressive decline adult survival (day 5, 100%; day 15, 75%; <br />                                                                              flight behaviour flies pan-muscular hSMN <br /> day 25, 66%; day 35, 62%) flight ability (FIG. 2A <br />                                                                              overexpression respective controls (FIG S2). However, <br /> and above). <br />                                                                              overexpression Gem3DN hSMN resulted increase <br />    Although overexpression full-length Gemin3 muscle of <br /> wild-type flies effect adult viability flight ability,          puparial axial ratios (FIG. 4A), phenotype described <br /> transgene levels adequate rescue flight defects                  previously Gemin3 <span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>null mutants</span> [40], modulation of <br /> associated RNAi-mediated Gemin3 knockdown Gem3DN                     Drosophila SMN protein levels [58,59]. Interestingly, hSMN was <br /> overexpression. Furthermore, Gemin3 added                 detected intensely-stained nuclear and, times, cytoplasmic <br /> background having genetic manipulations, transgenic protein             puncta muscle tissue. Inside nucleus, hSMN concen- <br /> levels capable rescuing lethality           trates multiple spherical puncta different sizes, are <br /> to revert flight performance normal (FIG. 2A). Finally,           restricted nucleolus, devoid Hoechst-labelled <br /> that overexpression SMN Gem3DN background does                   chromatin (FIG. 4B). <br /> rescue associated flight defects indicating not <br /> arise downstream events result deficient SMN                  Gemin3 Required Viability Normal Motor <br /> function. Taken together, findings confirm Gem3DN           Behaviour Neurons <br /> overexpression mimics strong loss Gemin3 function. Second,              Previously demonstrated global reduction Gemin3 <br /> drastic reduction Gemin3 function muscle results                function elicits <span id='am-479' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-480' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-485' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-487' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-489' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-493' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-506' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-509' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-513' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-527' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-529' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-531' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-533' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-536' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-538' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-543' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-551' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-552' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-553' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-554' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-555' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-556' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>, recapitulated if <br /> lethality low levels sustain normal                Gemin3 reduced solely muscle neuronal lineages [40]. <br /> motor behaviour. Third, Gemin3 indispensable motor                    The conclusion reached contrast SMN <br /> function adequately fulfilled simple expansion           essential tissue [42,60], Gemin3 required <br /> in functional capacity SMN.                                           exclusively muscle survival normal motor function. <br />                                                                              However, did exclude possibility pan-neuronal <br /> Gem3DN Localises Primarily Nucleus                                    GAL4 drivers used induced RNAi-mediated Gemin3 knock- <br />   Epitope tagging fusion known epitope                      potent and/or expression started late during <br /> recombinant protein revolutionised characterisation               development contrast GAL4 drivers pan-muscular <br /> proteins mutant versions, especially low                expression pattern. Furthermore, RNAi efficient in <br /> immunogenicity [57]. Taking advantage technique,                  neurons compared muscle. In context, rule out <br /> engineered GFP-tagged Gem3DN fusion protein goal               neuronal function Gemin3, end, devised <br /> determining sub-cellular location mutant dead                  strategies aimed providing definitive answer. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                      December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                               A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 2. Gem3DN combination Gemin3 knockdown lethal localisation pattern predominantly nuclear. (A) <br /> Perturbation endogenous Gemin3 wild-type muscle consequences adult viability flight behaviour. Bottom Top. Overexpression <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             5                       December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                       A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> of full-length Gemin3 repercussions. Reduced levels Gemin3 induced expression Gem3-IRmun associated a <br /> progressive age-dependent decline flight adult viability, rescued overexpression full-length Gemin3. Overexpression of <br /> Gem3DN effect adult viability drastic impact flight straight eclosion, rescued overexpression full- <br /> length Gemin3. Gemin3 knockdown combination Gem3DN overexpression results lethality. Overexpression full-length Gemin3 this <br /> background rescues associated lethality flight defects. SMN overexpression Gem3DN background influence the <br /> anomalous flight behaviour (assessment day 5 post-eclosion) associated genotype. Co-expression unrelated transgene (generic) <br /> with Gem3DN results flightless phenotype excluding possibility GAL4 dilution effect resulting multiple UAS-constructs in <br /> the organism. Pan-muscular expression driven Mef2-GAL4 experiments performed 25uC. Protein levels transgenic Gemin3 <br /> are indicated bar strength. (B) Subcellular localisation GFP.Gem3DN Drosophila larval muscles (delineated dashed white outline). In flies <br /> overexpressing GFP.Gem3DN, GFP signal ubiquitous predominantly nuclear co-localises Hoechst-stained nuclei. This subcellular <br /> expression pattern visible driver control (da-GAL4/+), hence, excluding possibility result non-specific reactivity <br /> of primary and/or secondary antibodies used detect GFP. <br /> doi:10.1371/journal.pone.0083878.g002 <br />  <br />    We noticed similar observed restricted             mortis ecdysone signalling [61]. In context, wished to <br /> muscle (above), Gem3DN overexpression combination                        clarify function Gemin5 hypothesised is <br /> Gemin3 knockdown exclusively neuronal tissues (via                    functionally associated SMN complex, loss function <br /> pan-neuronal elav-GAL4 driver incubation temperature                    mimic key phenotypic outcomes resulting loss of <br /> 25uC) induces lethality. This remarkable observation hinted              Gemin3, itself integral SMN complex member. We <br /> a critical role Gemin3 central nervous (CNS),                  noticed overexpression GFP-tagged Gemin5 fusion <br /> which apparent following severe reduction                      protein gave similar sub-cellular localisation pattern we <br /> function. To confirm finding, attempted markedly                      previously reported Gemin3 [23]. Indeed, Gemin5 forms <br /> attenuating level Gemin3 increasing dose hairpin                multiple nuclear foci variable size nucleus several <br /> RNAs and, eventually, short interfering RNAs (siRNAs) targeting                  smaller counterparts cytoplasm (FIG. 6A). Depending on <br /> Gemin3. To end, expression RNAi transgenic                           cellular location, structures likely gems U <br /> constructs (Gem3-IRmun+Gem3-IRdwe) muscle similar                  bodies present nucleus cytoplasm, respectively. We next <br /> worse effect reported previously using single transgenic               attempted addressing relative requirement Gemin5 the <br /> construct (FIG. 5A) [40]. Furthermore, RNAi                     important elements motor <span id='am-636' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> asked whether <br /> phenotype off-target effects observe rescue               Gemin5 required muscle neuron similar <br /> on co-expression full-length Gemin3 transgene. The possibility              manner Gemin3. <br /> that result GAL4 dilution effects arising multiple               Enhanced Gemin5 knockdown expression two <br /> UAS-constructs background excluded                        RNAi transgenes (Gem5-IRnan+Gem5-IRsac) targeting exon <br /> rescue observed truncated Gemin3 transgene (Gem3DC)                   Gemin5 mRNA transcript (FIG S1B) confirmed previous <br /> was utilised. Interestingly, observed expression             report demonstrating global reduction Gemin5 function <br /> the RNAi transgenic constructs brain pan-                     lethal [61] (FIG. 6A). We provide evidence specificity of <br /> neuronal elav-GAL4 n-syb-GAL4 drivers negligible effects                  RNAi-based knockdown showing co-expression a <br /> on adult viability development restricted                          full-length Gemin5 transgene (Gem5) version lacking the <br /> temperature 25uC, significant reduction (close 50%)                   WD domain-rich N-terminus (Gem5DN) rescues lethality <br /> adult viability achieved flies cultured                       associated loss Gemin5 function (FIG. 6A; FIG S1B). <br /> temperature 29uC allow maximal GAL4 activity                           The entire WD repeat domain necessary and <br /> (FIG. 5A). Furthermore, efficiency                      sufficient sequence-specific, high-affinity binding Gemin5 to <br /> knockdown intensified increasing Dicer-2 levels,               snRNAs [35]. Interestingly, lethality associated ubiquitous <br /> achieved dramatic reduction adult viability total lethality              Gemin5 knockdown recapped knockdown is <br /> (FIG. 5A). Interestingly, repeat result Gemin3                restricted muscle lineages pan-muscular how-GAL4 or <br /> knockdown restricted exclusively motor neurons OK6-                C179-GAL4 drivers. Furthermore, dramatic reduction adult <br /> GAL4 driver (FIG. 5A), result indicates requirement                  viability observed culture temperatures associated with <br /> Gemin3 viability specifically motor neurons. Importantly,                 maximal GAL4 activity (29uC) RNAi driven pan- <br /> we similar observed restricted muscle                  muscular Mef2-GAL4 boosted elevated Dicer-2 levels <br /> (above), enhanced Gemin3 knockdown explicitly CNS                     (FIG. 6B). Reduction Gemin5 larval somatic muscles <br /> leads age-dependent progressive flight defects (FIG. 5B)                  G7-GAL4 resulted significant decrease adult <br /> loss adult viability (day 5, 100%; day 15, 100%; day 25, 72%;                 viability incubation temperature 29uC. Intriguingly, we <br /> and, day 35, 26%) incubation temperature 25uC. This                     pan-neuronal Gemin5 reduction elav-GAL4 <br /> finding indicates addition muscle, sufficient Gemin3                  driver increase Dicer-2 levels translates in <br /> levels neurons paramount normal motor behaviour.                      lethality flies cultured 29uC, and, remarkably, this <br />                                                                                  phenotype repeated knockdown restricted only <br /> Adequate Muscle Neuronal Levels Gemin5                                motor neurons OK6-GAL4 driver. In summary, these <br /> Required Viability                                                           studies underscore similar Gemin3, SMN complex member <br />   We recently showed member SMN complex,                    Gemin5 forms nuclear cytoplasmic foci required <br /> Drosophila orthologue Gemin5 (previously named Rigor mortis                   adequate levels muscle neurons viability. <br /> [61]) localises U bodies ovarian cytoplasm [25]. <br /> Whilst findings indicative role Gemin5 SMN                Loss Gemin5 Function Either Muscle Neurons <br /> complex-related activity, Kroiss et al. [39] argued based              Impairs Normal Motor Behaviour <br /> on phylogenetic analysis revealing Drosophila Gemin5                     Our findings indicating requirement Gemin5 viability in <br /> evolving significantly faster compared orthologues               major constituents motor <span id='am-637' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> posed question of <br /> organisms study reporting function Rigor              loss Gemin5 function muscle neurons leads <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   6                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                   A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   7    December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                    A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> Figure 3. Ectopic overexpression hSMN wild-type organisms affects viability motor behaviour restricted muscle. <br /> (A) Ubiquitous overexpression hSMN wild-type flies a-Tub-GAL4 da-GAL4 lethal. Restricting hSMN overexpression mesoderm and <br /> larval muscles (via how-GAL4, C179-GAL4 Mef2-GAL4) larval muscles (via G7-GAL4, C57-GAL4 G14-GAL4), leads lethality drastic <br /> reduction viability. Fly viability remains unaffected hSMN overexpressed CNS neurons (via elav-GAL4, 1407-GAL4, nrv2-GAL4 n-syb- <br /> GAL4) restricted motor neurons (via D42-GAL4 OK6-GAL4) eye (GMR-GAL4). Bar chart (right panel) adult fly viability <br /> assayed 25uC 29uC, resulting enhanced GAL4 activity. Individual bars represent mean viability 6 S.E.M. 4 <br /> independent experiments. Left panel shows tissue expression pattern GAL4 drivers utilised. Abbreviations: pan-ms, pan-muscular; mes, <br /> mesoderm; ms, larval muscles; mn, motor neurons; n, CNS neurons motor neurons. (B) The climbing behaviour flies pan-muscular <br /> hSMN overexpression (Mef2-GAL4.hSMN) significantly altered day 15 post-eclosion compared control sample populations (Mef2-GAL4/+ <br /> or hSMN/+). Data presented mean 6 S.E.M. statistical significance (*p,0.05, ***p,0.001) determined differences the <br /> Mef2-GAL4.hSMN genotype control genotypes, denoted respective colour. <br /> doi:10.1371/journal.pone.0083878.g003 <br />  <br /> to defects motor behaviour similar manner Gemin3. In                When sampled flies genotype (elav-GAL4.Dcr- <br /> this regard, assessed climbing ability flies           2+Gem5-IRnan+Gem5-IRsac), noticed fly was <br /> enhanced Gemin5 knockdown CNS, viable                      capable reaching target height took, average, signifi- <br /> their development restricted permissive temperature (25uC).            cantly longer compared controls early day 5 post- <br />  <br />  <br />  <br />  <br /> Figure 4. Overexpression hSMN wild-type organisms interferes normal puparial axial ratio similar manner to <br /> Gem3DN, induces formation spherical aggregates. (A) Top, Puparia flies ubiquitous overexpression hSMN (da- <br /> GAL4.hSMN), pan-muscular overexpression hSMN (how-GAL4.hSMN) Gem3DN (how-GAL4.Gem3DN), respective GAL4 driver controls <br /> (da-GAL4/+ how-GAL4/+). Bottom, Chart displaying axial ratios puparia indicated genotype. Ubiquitous pan-muscular <br /> overexpression hSMN results significantly larger puparial axial ratios. Although ubiquitous overexpression Gem3DN results lethality prior to <br /> puparium formation, pan-muscular overexpression results puparia significantly large axial ratio. The mean marked horizontal line <br /> running data points error bars 6 S.E.M. (***p,0.001). (B) Larval muscles (delineated dashed white outline) showing the <br /> formation hSMN-positive spherical aggregates variable size cytoplasm nucleus flies ubiquitous ectopic overexpression of <br /> hSMN. Within nucleus, foci localise nucleolus, region low Hoechst reactivity (insert, scale bar = 5 mm). Aggregates <br /> were visible control (da-GAL4/+) tissues subjected immunostaining reaction, excluding possibility non-specific <br /> reactivity. <br /> doi:10.1371/journal.pone.0083878.g004 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 8                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                  A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 5. Enhanced Gemin3 knockdown neuronal lineages leads loss adult viability flight defects. (A) Enhanced Gemin3 <br /> knockdown involved simultaneous application RNAi transgenes (Gem3-IRmun+Gem3-IRdwe) targeting Gemin3. Culture temperature of <br /> 29uC allow maximal GAL4 activity and/or Dicer-2 overexpression resulted increase RNAi potency. Gemin3 knockdown tissues <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               9                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                    A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> via da-GAL4, a-Tub-GAL4 1032-GAL4 drivers results lethality, rescued co-expression full-length (Gem3) C- <br /> terminal truncated (Gem3DC) Gemin3 transgene, demonstrating RNAi specificity. When Gemin3 knockdown restricted muscle <br /> lineages (how-GAL4, C179-GAL4 Mef2-GAL4) somatic musculature (G7-GAL4), loss adult viability complete lethality was <br /> observed, phenotype similar worse previously reported dose RNAi transgene [40]. Remarkably, note pan- <br /> neuronal Gemin3 knockdown (via n-syb-GAL4 elav-GAL4), achieve significant decrease adult viability culture temperature <br /> was raised 29uC and, eventually, complete lethality concomitant overexpression Dicer-2. Knockdown exclusively motor neurons was <br /> sufficient markedly impact adult viability cause lethality incubation temperature 25uC 29uC, respectively, enhanced <br /> Dicer-2 levels. Compared controls, enhanced knockdown Gemin3 eyes GMR-GAL4 driver does lead rough eye phenotype <br /> (data shown). Note that, expected, drivers similar tissue expression pattern (left panel) analogous result reason <br /> being drive strong GAL4 levels comparable effective counterparts. Bar chart (right panel) shows adult fly viability assayed <br /> at 25uC 29uC. Individual bars represent mean viability 6 S.E.M. 4 independent experiments. Abbreviations: pan-ms, pan-muscular; <br /> mes, mesoderm; ms, larval muscles; mn, motor neurons; n, CNS neurons motor neurons. (B) Flight behaviour population brain- <br /> restricted enhanced Gemin3 knockdown (elav-GAL4.Dcr-2+Gem3-IRdwe+Gem3-IRmun) significantly different controls starting day 15 <br /> post-eclosion. In regard, number non-fliers drop straight lowest sector (sector 1) exhibits progressive age-dependent increase <br /> that obvious driver-only (elav-GAL4.Dcr-2) responder-only (Gem3-IRdwe/+ Gem3-IRmun/+) control populations. Data presented the <br /> mean 6 S.E.M. statistical significance determined differences elav-GAL4.Dcr-2+Gem3-IRdwe+Gem3-IRmun genotype the <br /> control genotypes, indicated respective colour. For data, ns = significant, *p,0.05, **p,0.01, ***p,0.001. <br /> doi:10.1371/journal.pone.0083878.g005 <br />  <br /> eclosion. In vein, time point, brain-specific                 embryos, knockdown Gemin2 reported conflicting <br /> Gemin5 knockdown depressed markedly climbing success rate                  effects motor axon outgrowth [63,64], typically <br /> or average number flies successfully climbed              defective depletion SMN [65]. We sought clarify issue <br /> target height 8 cm 10 seconds (FIG. 7A). Furthermore,                 Drosophila targeting Gemin2 levels expression an <br /> did fail observe small percentage flies (36%)         RNAi transgene (Gem2-IRgau) directed exon 1 exon 2 <br /> population curved instead normal straight wing blades and,              (FIG S1C). We note similar previous studies other <br /> less frequently (9%), wings crumpled unopened                     organisms [62,64,65,66], global depletion Gemin2 driven the <br /> (FIG. 7B).                                                                     ubiquitous da-GAL4 driver results lethality culture temper- <br />     The results spurred investigate drastic              atures induce highest GAL4 activity (FIG. 8A). Impor- <br /> reduction Gemin5 CNS adverse effect                    tantly, phenotype reversed co-expression full-length <br /> flight performance. To end, subjected flies           Gemin2 transgene demonstrating RNAi specificity. <br /> Droso-Drome trials compared resulting mean                  Furthermore, lack rescue unrelated truncated <br /> controls. Remarkably, majority flies pan-neuronal                transgene (Gem3DC) eliminates possibility rescue the <br /> Gemin5 knockdown (elav-GAL4.Dcr-2+Gem5-IRnan+Gem5-IRsac,                       result GAL4 dilution effects induced multiple UAS- <br /> 73%) flightless day 5 post-eclosion, result                  constructs background. Although Gemin2 knockdown was <br /> significantly different obtained wild-type (y w, 10%),            maximal considering limited use a <br /> RNAi transgenes-only (Gem5-IRnan/+; Gem5-IRsac/+, 21%)                      single RNAi transgene, observed restricting effect in <br /> driver-only (elav-GAL4.Dcr-2, 26%) controls. However,                      various tissues, achieve drastic reduction adult <br /> resulting phenotype did differ flies curly             viability use pan-muscular C179-GAL4 how- <br /> wing phenotype (In[2LR]Gla/CyO, 57%), known                  GAL4 drivers (FIG. 8A). <br /> an impaired flight ability (FIG. 7C). We asked                       Following observations, asked Gemin2 <br /> restricting Gemin5 knockdown muscle similar effect                 knockdown motor unit negative impact motor <br /> motor behaviour. Although, Dicer-boosted reduction Gemin5                 behaviour similar observed depletion Gemin3 <br /> levels pan-muscular Mef2-GAL4 driver negative                   Gemin5 (above). Interestingly, confined to <br /> impact climbing behaviour (data shown), demonstrate                muscle, Gemin2 reduction enhanced high Dicer-2 levels <br /> striking progressive decline flight ability adulthood            interferes mobility. In respect, time taken the <br /> (FIG. 7D). Indeed, final recorded time point (day            initial fly sampled population cross target height of <br /> 35 post-eclosion) majority non-fliers (Mef2-                  8 cm significantly longer taken controls day 5 <br /> GAL4.Dcr-2+Gem5-IRnan+Gem5-IRsac, 77%) contrast wild-                    day 15 post-eclosion. Furthermore, climbing success rate is <br /> type (y w, 30%), transgenic responder-only (Gem5-IRnan/+; Gem5-                depressed early day 5 post-eclosion remains same <br /> IRsac/+, 21%), driver-only (Mef2-GAL4.Dcr-2, 28%) controls,                 level course adulthood (FIG. 8B). Notably, we <br /> and similar curly-positive flies (In[2LR]Gla/CyO, 74%).                 demonstrate progressive age-dependent decline flight perfor- <br /> Overall, findings indicate loss Gemin5 function               mance population flies having pan-muscular <br /> either muscle neurons impairs normal motor behaviour, albeit                Gemin2 knockdown (Mef2-GAL4.Dcr-2+Gem2-IRgau) (FIG. 8C). <br /> with different impact.                                                       In regard, day 15, half flies genotype were <br />                                                                                average non-fliers (50%), portion significantly <br /> Gemin2 Required Muscle Viability Normal                          different RNAi transgene-only (Gem2-IRgau/+, 6%) <br /> Motor Behaviour                                                                driver-only (Mef2-GAL4.Dcr-2, 18%) controls. On follow- <br />   Gemin2 SMN complex component                       ing recorded time points, fly population muscle- <br /> phylogenetically conserved sequence domain structure [21,33]               specific Gemin2 depletion, registered increase the <br /> (FIG S4). Its pivotal role snRNP assembly recently                 percentage flies flightless. Indeed, majority flies <br /> been revealed elegant structural study [33]. It                  (Mef2-GAL4.Dcr-2+Gem2-IRgau: day 25, 69%; day 35, 88%) drop <br /> however unclear key SMN complex member                         straight lowest sector Droso-Drome contrast to <br /> required normal motor function. In regard, double                     controls consisting responder RNAi transgene (Gem2- <br /> heterozygous Gemin2 SMN <span id='am-401' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-402' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-476' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-477' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> mice develop                           IRgau/+: day 25, 14%; day 35, 15%) driver (Mef2-GAL4.Dcr-2: <br /> enhanced motor neurodegenerative phenotype, correlated                   day 25, 11%; day 35, 28%) similar phenotypic outcome <br /> with disturbed snRNP assembly [62]. However, zebrafish                      flies homozygous hypomorphic SmnE33 allele (FIG. 8C). <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                10                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                    A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   11    December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                     A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> Figure 6. Gemin5 concentrates predominantly nuclear foci adequate levels muscle neurons required for <br /> viability. (A) Larval muscles (delineated dashed white outline) flies ubiquitously expressing GFP-tagged Gemin5 functional fusion protein. <br /> Gemin5 concentrates puncta variable size predominantly nuclear (insert, scale bar = 5 mm). For reasons unclear, GFP also <br /> detected tracheal supplies muscles oxygen. (B) A severe Gemin5 knockdown concomitant expression of <br /> two RNAi transgenes (Gem5-IRnan+Gem5-IRsac) targeting Gemin5 mRNA transcripts causes lethality expression driven ubiquitously via <br /> the 1032-GAL4, da-GAL4 Act5c-GAL4 drivers. RNAi specific co-expression full-length Gemin5 transgene (Gem5) truncated <br /> version lacking N-terminus (Gem5DN) restores viability. Ubiquitous-associated lethality replicated Gemin5 depletion restricted muscle <br /> lineages C179-GAL4 how-GAL4 driver. A drastic reduction viability observed RNAi driven pan-muscular Mef2-GAL4 <br /> driver temperature 29uC presence enhanced Dicer-2 levels, strategies employed heighten knockdown effect. <br /> Reduced levels Gemin5 somatic musculature G7-GAL4 driver marked negative impact adult viability. A lethal <br /> outcome obtained RNAi constrained CNS neurons (elav-GAL4) exclusively motor neurons (OK6-GAL4) a <br /> temperature 29uC upregulation Dicer-2. Compared controls, enhanced knockdown Gemin5 eyes GMR-GAL4 driver <br /> does lead rough eye phenotype (data shown). Note that, expected, drivers similar tissue expression pattern (left panel) <br /> give analogous result reason drive strong GAL4 levels comparable effective counterparts. Bar chart <br /> (right panel) shows adult fly viability assayed 25uC 29uC. Individual bars represent mean viability 6 S.E.M. 4 independent <br /> experiments. Abbreviations: pan-ms, pan-muscular; mes, mesoderm; ms, larval muscles; mn, motor neurons; n, CNS neurons motor neurons. <br /> doi:10.1371/journal.pone.0083878.g006 <br />  <br /> Flies muscle-confined Gemin2 knockdown experience                 supports ability SMN, Gemin2 Gemin8 Gemin3 <br /> progressive decline viability age (day 5, 100%; day 15,                 self-associate [8,9,11,31,67]. This model most <br /> 100%; day 25, 62%; day 35, 47%). However, did                        probable explaining antimorphic mechanism hSMN <br /> detect significant motor defects knockdown restricted               overexpression phenotype ( [44] present study). The resulting <br /> the CNS (data shown) outcome likely reflects              high hSMN levels thought titre dSMN oligomers that <br /> fact knockdown severe enough. In summary,                    unable support addition Gemin members of <br /> findings emphasise similar counterparts SMN              SMN complex and, thereby, incapable forming functional <br /> complex, Gemin2 required motor unit viability                 SMN complexes. Our demonstration hSMN aggregates <br /> normal motor function.                                                          corroborates mechanism work needed to <br />                                                                                 explore toxic effect cellular processes especially those <br /> Discussion                                                                      confined nucleolus. In line previous study [60], <br />                                                                                 muscle susceptible attenuation SMN activity, which <br />   Our study sheds light antimorphic mechanism                     case induced ectopic hSMN overexpression. <br /> Gem3DN overexpression phenotype, importantly unravels                        In model, Gem3DN capable binding its <br /> requirement Gemin associates SMN viability                 partner high concentrations competes partner from <br /> motor function vivo.                                                         wild-type endogenous protein. This model likely <br />                                                                                 scenario case Gem3DN protein, having retained <br /> Mechanism Gem3DN Overexpression Phenotype                                SMN-binding domain, hijacks SMN associates the <br />    We demonstrate loss-of-function phenotype Gemin3                 endogenous wild-type Gemin3, hence, forming &#226;&#8364;&#732;poisonous&#226;&#8364;&#8482; or <br /> is similar resulting Gem3DN overexpression. In                non-functional SMN complex (FIG. 9A,B). In regard, <br /> regard, Gemin3 knockdown Gem3DN overexpression                     SMN complex bind target RNAs, it <br /> muscle disrupt normal motor performance. Because overexpres-                    probably incapable processing loss of <br /> sion mimics loss function, Gem3DN presumably interferes                 Gemin3 ATPase activity. This competition-based mechanism is <br /> some level function Gemin3, complex RNA                  supported observation Gem3DN dominant-negative <br /> substrate, acting dominant-negative mutant Muller&#226;&#8364;&#8482;s                   phenotype reversed co-overexpression target <br /> antimorph. It fair say case flight ability,                protein, Gemin3, aggravated background reduced <br /> phenotypes completely identical resulting              levels Gemin3. Furthermore, localisation pattern the <br /> Gemin3 knockdown develop late adulthood. This                           fluorescently-tagged version Gem3DN encourages to <br /> observation highlights fact contrast RNAi-based                  hypothesise antagonistic action primarily nuclear. It is <br /> methods, use inhibitory mutants act protein level            possible Gem3DN causes mislocalisation associated <br /> generate severe phenotypes view                      complex cytoplasm nucleus, displacing it <br /> direct.                                                                         away site action. Nevertheless, plausible the <br />    Several models explain overexpression catalytically             smaller Gem3DN cytoplasm be <br /> inactive helicase protein antagonises endogenous wild-type                  sufficient inhibit cytoplasmic function. Interestingly, Almstead <br /> protein exert dominant-negative antimorphic phenotype                   Sarnow [68] reported poliovirus-encoded proteinase <br /> [46,48]. Overexpression mutant RNA helicase protein                   2Apro specifically cleaves human Gemin3 Tyr462 and <br /> has defective ATPase RNA unwinding activity retains                  Gly463 generate cleavage product lacks helicase core <br /> ability stably bind RNA form static protein-RNA                        and, hence, identical Gem3DN. Notably, <br /> complex prevents RNA substrates taking                        poliovirus causative agent poliomyelitis, is <br /> downstream reactions. This model highly unlikely case                 characterised destruction motor neurons, phenotype <br /> considering Gem3DN protein lacks entire helicase core              surprisingly similar observed SMA. Future studies that <br /> and, thereby, probably unable bind RNA, let                 focus interaction profile Gem3DN biochemical <br /> engage ATPase-dependent RNA chaperoning activities. Alter-                   approaches help understanding molecular <br /> natively, mutant dead helicase protein retained                mode action dead helicase mutant. <br /> ability interact wild-type counterpart mutant/ <br /> wild-type helicase dimer multimer defective                  Composition Drosophila SMN Complex <br /> catalytic activity. Again, model highly improbable consid-                 Throughout evolution, SMN complex experienced an <br /> ering knowledge biochemical evidence date                    increase membership addition set Gemin <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 12                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                     A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 7. Loss Gemin5 function neurons muscle impedes normal motor behaviour. (A) Flies pan-neuronal Gemin5 <br /> knockdown (elav-GAL4.Dcr-2+Gem5-IRnan+Gem5-IRsac) significantly impaired climbing ability compared wild-type (y w), RNAi transgenes- <br /> only (Gem5-IRnan/+; Gem5-IRsac/+) driver-only (elav-GAL4.Dcr-2) controls, starting early day 5 post-eclosion. In respect, performer <br /> in population depleted levels Gemin5 took significantly longer climb 8 cm mark population-level, number <br /> of flies successful reached target height required timeframe was, average, significantly lower compared controls. (B) <br /> Compared wild-type controls, close half flies, CNS-restricted Gemin5 knockdown, abnormal wing blades defects ranging <br /> from curved crumpled unopened wing phenotype (arrow). (C) Flies brain-specific Gemin5 (elav-GAL4.Dcr-2+Gem5-IRnan+Gem5- <br /> IRsac) knockdown average different distribution Droso-Drome compared controls starting day 5 post-eclosion. In regard, <br /> the majority flies drop straight sector 1 indicating flightless, phenotypic outcome differs significantly controls <br /> but similar flies curly wings (In[2LR]Gla/CyO), known weak fliers. (D) Pan-muscular Gemin5 knockdown leads an <br /> age-dependent progressive decline flight ability worse phenotype observed day 35 post-eclosion. On time-point, majority of <br /> flies muscle-specific depleted levels Gemin5 non-fliers and, thereby, fall straight sector 1 contrast controls similar curly- <br /> positive flies. Data presented mean 6 S.E.M. statistical significance determined differences elav-GAL4.Dcr-2+Gem5- <br /> IRnan+Gem5-IRsac (A, C) Mef2-GAL4.Dcr-2+Gem5-IRnan+Gem5-IRsac (D) genotype genotypes, indicated respective <br /> colour. For data, ns = significant, *p,0.05, **p,0.01, ***p,0.001. <br /> doi:10.1371/journal.pone.0083878.g007 <br />  <br /> proteins ancestral core complex formed SMN                       assembly conserved Drosophila since, similar observed <br /> Gemin2 [21]. Focusing Drosophila, Kroiss et al. [39] used                  SMN, depletion Gemin3 significantly reduced Sm core <br /> biochemical approach successfully isolate assembly-active                 assembly in vitro assay [49]. Such findings indicate the <br /> SMN complex surprisingly consisted SMN Gemin2                    Drosophila SMN complex intricate present S. <br /> the stoichiometric components despite bioinformatic data                   pombe plants [21]. Taking account vivo work making use <br /> that predicted presence Gemin3 Gemin5 orthologues                 genetic cell biology approaches, provide support <br /> this model organism. Later, presented biochemical                 direction. The present study combination previous <br /> evidence showing interaction Gemin3 SMN                             work Gemin3 SMN [40,42,43,49,60], unequivocally shows <br /> [40,49,69]. In addition, function Gemin3 snRNP                        remarkably similar phenotypes arise loss function of <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 13                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                      A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 8. Gemin2 essential viability normal motor function. (A) Gemin2 knockdown expression single RNAi <br /> transgene (Gem2-IRgau) causes drastic reduction adult viability (25uC) complete lethality (29uC) driven tissues da-GAL4 driver. <br /> Expression transgenic Gemin2 genetic background restores viability. No significant rescue observed co-expression an <br /> unrelated transgene (Gem3DC). A marked reduction adult viability observed Gemin2 depleted specifically muscle lineages <br /> (C179-GAL4 how-GAL4). Note that, expected, drivers similar tissue expression pattern (left panel) analogous result one <br /> reason drive strong GAL4 levels comparable effective counterparts. Bar chart (right panel) shows adult fly viability <br /> assayed 25uC 29uC, temperature inducing maximal GAL4 activity. Abbreviations: pan-ms, pan-muscular; mes, mesoderm; ms, <br /> larval muscles; mn, motor neurons; n, CNS neurons motor neurons. (B) Pan-muscular Gemin2 knockdown boosted enhanced Dicer-2 <br /> levels leads impaired climbing behaviour. In respect, time taken fly sampled population reach target height of <br /> 8 cm significantly increased day 5 day 15 post-eclosion. In similar vein, climbing success rate percentage flies reach <br /> beyond 8 cm 10 seconds, dramatically decreased recorded time points. (C) Flies muscle-specific Gemin2 depletion exhibit a <br /> progressive age-dependent decline flight performance. In regard, portion flightless flies significantly different controls day 15 <br /> post-eclosion increases age reaching maximal level day 35 adulthood. On time point, behaviour similar that <br /> of flies reduced levels SMN muscle (SmnE33), 100% non-fliers. For data, individual bars represent mean 6 S.E.M. and <br /> statistical significance determined differences Mef2-GAL4.Dcr-2+Gem2-IRgau genotype control genotypes, are <br /> indicated respective colour (B, C). For data, ns = significant, *p,0.05, **p,0.01, ***p,0.001. <br /> doi:10.1371/journal.pone.0083878.g008 <br />  <br /> SMN Gemin components including conserved                    SMN-Gemins complex heart reported motor <br /> Gemin3 Gemin5 (FIG S3 FIG. 9A,C&#226;&#8364;&#8220;F). This itself                    defects. <br /> highly indicative common role Gemins the <br /> Drosophila SMN complex. Notably, evidence supported                  A common Process key Normal Motor Function <br /> recent study Gemins 2, 3 5 co-localise                    In study report that, depending strength, loss of <br /> with SMN snRNPs cytoplasmic U bodies                           function Gemin2, Gemin3 Gemin5 motor unit impairs <br /> Drosophila egg chamber [25]. Furthermore, overexpression                   adult viability normal motor behaviour including <br /> Gemin3 Drosophila, induce formation SMN-                      climbing ability flight performance. In particular, flight a <br /> enriched nuclear gems interact Cajal bodies similar               highly demanding motor activity flies addition wings <br /> manner described vertebrate counterparts [22,23].              capable generating aerodynamic lift, depends a <br /> Interestingly, study reported gems                 motor-sensory circuit allows exquisite control, high- <br /> triggered upregulation Gemin5 protein levels. Although                  power high-frequency muscle [53]. It interesting note that <br /> cannot exclude possibility secondary SMN complex-                         exception Gemin2, required muscle, all <br /> independent activities, overall, data captured far, favours               Gemins required key compartments the <br /> a Drosophila SMN complex formed SMN, Gemin2, Gemin3                       motor unit survival and, importantly, motor function. In this <br /> Gemin5, essential correct functioning vivo.          regard, restricted muscle neurons, showed a <br /> Hence, hypothesise absence member                          strong loss function Gemin3 Gemin5 a <br /> sufficient &#226;&#8364;&#732;rob&#226;&#8364;&#8482; SMN-Gemins complex function                       significant impact adult viability milder knock- <br /> (FIG. 9A,C-F). We presently attempting genetic interaction                   down, detect defects motor function. A similar tissue- <br /> studies better understand relationship complex                    selective trend reported SMN various studies <br /> members vivo strengthen conviction disruption              [42,59,60]. We corroboration previous <br />                                                                                  study [49], forced expression normal SMN Gem3DN <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  14                        December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                   A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br />  <br /> Figure 9. A working model evaluation motor phenotypes resulting perturbation individual members the <br /> Drosophila SMN-Gemins complex. (A) The interaction map Drosophila SMN-Gemins complex unclear and, here, modelled that <br /> drafted human version (reviewed [21]). In regard, SMN associates Gemin2 Gemin3. Gemin2, turn, interacts Gemin5. All <br /> components SMN complex individually required viability and, importantly, motor behaviour. In particular, key normal flight <br /> performance, highly demanding motor activity relies correct function motor unit. The process pathway linking SMN <br /> complex motor behaviour unclear (indicated &#226;&#8364;&#732;?&#226;&#8364;&#8482;) speculate predominantly nuclear and, hence, revolve <br /> around snRNP biogenesis and/or recycling. (B) The dead helicase Gem3DN mutant lacks helicase core retains SMN-binding domain. We <br /> hypothesise hijacks SMN associates endogenous wild-type Gemin3, hence, forming non-functional inactive SMN complex. <br /> Gem3DN overexpression mimics flightless phenotype associated loss Gemin3 function phenotype stronger. (C&#226;&#8364;&#8220;E) <br /> Reduced Gemin3, Gemin5 Gemin2 levels RNAi-mediated knockdown results flightless phenotype. These observations <br /> demonstrate factors performs critical role compensated presence remaining components. (D) <br /> Similar Gemin associates, SMN required adequate levels normal flight behaviour reported hypomorphic SmnE33 fly mutant [43]. <br /> doi:10.1371/journal.pone.0083878.g009 <br />  <br /> mutant background does rescue associated motor defects,               responsible RNA unwinding RNP restructuring events <br /> thereby indicating SMN does function downstream                   miRNA maturation and/or downstream events that <br /> Gemin3 probably parallel Gemin3. Congruent                include target RNA recognition. Furthermore, association of <br /> this, Gemins genetic modifiers Smn-                    Gemin3 multitude transcription factors well- <br /> dependent phenotypes ([60,70] data shown). Although                 documented [reviewed 21], additional role in <br /> requirement Gemin2 neurons obvious,                    transcriptional regulation excluded. An SMN com- <br /> predict that, similar way Gemin 3 5, role          plex-independent inclination reported Gemin5, <br /> exposed significant knockdown. Hence, presently                forms distinct complexes, specific internal <br /> developing strategies aimed increasing impact Gemin2                ribosome entry site (IRES)-ribonucleoprotein complex an <br /> loss function neuronal tissues test hypothesis.                 IRES-independent complex containing eIF4E [76,77], hence <br />    Key Gemin members SMN-Gemins complex                      establishing link Gemin5 modulation mRNA <br /> reported form additional multiprotein complexes                 translation. Turning Gemin2, note evidence <br /> which function independently SMN complex [21,71].                 pointing existence SMN-Gemin2 subcomplex <br /> For instance, Gemin3 Gemin4 form abundant complex                  functions DNA double-strand break repair through <br /> that co-sediments polyribosomes contains Argonaute2                  homologous recombination [78,79]. This body evidence seems <br /> (AGO2) numerous microRNAs (miRNAs) [72,73,74]. In                  suggest alternative functions Gemin components, at <br /> a following report, Gemin3 detected AGO2, fragile               vertebrates. However, based present work that <br /> X mental retardation protein (FMRP) p100 murine                    demonstrating similar phenotype loss function of <br /> peripheral axons sciatic nerve proteins             members Drosophila SMN-Gemins complex <br /> reported engage formation RNA-induced silencing               [26,27,40,42,43,49,58,60,64], tempting speculate they <br /> complex (RISC) [75]. These studies implicate Gemin3 RNA                    engage common pathway process vivo. <br /> silencing, aided co-factors Gemin4, be <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               15                       December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                              A Motor Function Gemin Associates SMN <br />  <br />  <br />    The most-extensively studied pathway unifies members           Center, RWTH Aachen, Germany), elav-GAL4;UAS.Dcr-2 (Bloo- <br /> of SMN complex snRNP biogenesis. It                mington Drosophila Stock Centre [BDSC] Indiana University, <br /> unclear defects pathway linked impaired motor           USA), 1407-GAL4 (BDSC), nrv2-GAL4 (gift Paul Salvaterra, <br /> function. Recent studies SMA animal models uncovered                 City Hope National Medical Center, Duarte, California, USA), <br /> negative impact splicing expression levels U12 intron-           n-syb-GAL4 (gift Brian McCabe, Columbia University, NY, <br /> containing genes essential motor-sensory circuit              USA), GMR-GAL4 (DGRC), D42-GAL4 (BDSC), OK6-GAL4 (gift <br /> function [80,81] preferential reduction snRNPs          Cahir O&#226;&#8364;&#8482;Kane, University Cambridge, Cambridge, UK), <br /> constitute minor spliceosome [82,83]. In respect, early        C179-GAL4 (BDSC), how-GAL4 (BDSC), Mef2-GAL4 (gift from <br /> study showed motor axon defects observed silencing          Barry Dickson, Research Institute Molecular Pathology, <br /> SMN Gemin2 zebrafish embryos rescued                    Vienna, Austria), UAS.Dcr-2;Mef2-GAL4 (BDSC), G7-GAL4 (gift <br /> injection purified snRNPs [64] recent work Workman               Aaron DiAntonio, Washington University, St. Louis, <br /> et al. [84] showed restoring normal snRNP levels                Missouri, USA), C57-GAL4 (gift Vivian Budnik, University <br /> significant phenotypic rescuing effect severe SMA mouse               Massachusetts, Worcester, Massachusetts, USA), MHC82- <br /> model. The link snRNP assembly selective neuro-                GAL4 (gift Cahir O&#226;&#8364;&#8482;Kane). The spatial temporal <br /> muscular degeneration refuted studies demonstrat-            expression patterns described Results section. <br /> ing role SMN snRNP assembly uncoupled                    The generation UAS.Dcr-2;OK6-GAL4, UAS.Gem3-IRd- <br /> from axonal defects zebrafish [85] organismal viability          we <br />                                                                               ;UAS.Gem3-IRmun, UAS.Gem5-IRnan;UAS.Gem5-IRsac double trans- <br /> locomotor defects Drosophila [86]. Furthermore,                genic stocks carried according standard genetic crossing <br /> evidence suggesting splicing defects SMA mice likely           schemes. <br /> a secondary consequence severe SMN loss [83,87]. Against this <br /> backdrop, SMN complex implicated key non-              Transgenic Constructs <br /> canonical functions. In neuronal processes, SMN complex                The generation UAS.GFP.Gem3DN transgenic construct <br /> present large stationary small actively-transported granules        involved PCR-amplification C-terminus Gemin3 and <br /> devoid Sm proteins [88,89,90,91,92]. On postsynaptic side, <br />                                                                            ligation, N-terminus, enhanced cyan fluorescent <br /> the Drosophila SMN complex reported localise the <br />                                                                            protein (eCFP) coding portion derived pECFP-C1 (BD <br /> sarcomeric Z-disc [43,93]. In compartments, SMN <br />                                                                            Biosciences Clontech, Palo Alto, California, USA) vector. The NotI <br /> complex involved mRNA trafficking, process <br />                                                                            XbaI restriction sites used insert fusion <br /> required function maintenance neuromuscular <br />                                                                            construct pUAST vector. For UAS.Gem3DC transgenic <br /> junctions myofibrils [21,38]. Based evidence showing <br />                                                                            construct, N-terminus Gemin3 PCR-amplified and <br /> that localisation Gem3DN mutant predominantly nuclear <br />                                                                            ligated KpnI XbaI restriction sites pUAST <br /> coupled induction nuclear gems following upregulation <br />                                                                            vector. The UAS.Gem2 transgenic construct involved PCR- <br /> of Gemin3 Gemin5, tempting speculate the <br />                                                                            amplification full-length coding sequence Gemin2 followed <br /> SMN-Gemins complex participates nucleocentric process or <br />                                                                            insertion pUAST vector utilising NotI KpnI <br /> pathway key normal motor function vivo. We <br />                                                                            restriction sites. PCR-amplification full-length coding <br /> hypothesise snRNP assembly and/or recycling the <br />                                                                            sequence Gemin5 C-terminus, followed insertion into <br /> heart pathway, studies understanding <br />                                                                            EcoRI KpnI restriction sites pUAST vector, was <br /> of components, workings, manipulation through <br /> genetic pharmacological means potentially open new              carried generate respective UAS.Gem5.GFP and <br /> avenues SMA therapeutics.                                              UAS.Gem5DN transgenic constructs, involving <br />                                                                            ligation eCFP tag C-terminus. <br />                                                                               cDNA clones Gemin2 (LD47479), Gemin3 (LD05563) and <br /> Materials Methods <br />                                                                            Gemin5 (SD03652) obtained Drosophila Genomics <br /> Fly Stocks                                                                 Resource Centre (Indiana University, USA). In cases, the <br />    Fly stocks maintained 25uC standard molasses/                ligation products used transform NEB 5-alpha competent <br /> maizemeal agar medium plastic vials. Except                   E. coli cells (New England Biolabs, Hitchin, UK) using standard <br /> indicated, wild-type strains y w Oregon R. The                 protocols. Correct transformants propagated, their <br /> In(2LR)Gla/CyO line obtained Drosophila Genetic               harbouring plasmids purified sequenced prior microinjec- <br /> Resource Center (DGRC) Kyoto Institute Technology,               tion w1118 embryos. <br /> Kyoto, Japan. The SmnE33 hypomorphic allele SmnX7 <br /> microdeletion generous gifts Greg Matera (University             Puparial Axial Ratios <br /> of North Carolina, Chapel Hill, North Carolina, USA) Spyros              Puparial axial ratios calculated dividing length by <br /> Artavanis-Tsakonas (Harvard Medical School, Boston, Massachu-              width puparia, measured still <br /> setts, USA), respectively. The UAS.SMN-YFP UAS.hSMN lines              images. <br /> were kindly provided Ji-Long Liu (MRC Functional Genomics <br /> Unit, University Oxford, Oxford, UK). The UAS.Gem3DN                    Adult Viability Studies <br /> transgenic construct RNAi transgenic constructs,                Adult viability assays conducted crossing GAL4 driver <br /> UAS.Gem3-IRdwejra UAS.Gem3-IRmunxar, characterised                stocks lines harbouring single combination transgenes. <br /> previously [40]. The RNAi transgenic constructs, UAS.Gem2-                 Culture temperature 25uC 29uC. Following eclosion, <br /> IRgaulos (10419R-1), UAS.Gem5-IRnanni (11171R-2), UAS.Gem5-            adult flies screened counted regular intervals. Adult <br /> IRsacher (11171R-1), obtained National Institute          viability calculated percentage number adult <br /> Genetics Fly Stock Center, Japan.                                          flies appropriate genotype divided expected <br />    GAL4 lines used work drive expression UAS-linked          number cross. <br /> transgenes included 1032-GAL4, da-GAL4, a-Tub-GAL4 <br /> (DGRC), elav-GAL4 (gift Aaron Voigt, University Medical <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            16                      December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                 A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> Flight Assay                                                                 6.8 Kbp nine-exon gene thirteen WD repeat domains <br />     Based originally designed behavioural genetics            located N-terminus. The Gem5DN construct lacks N- <br /> mastermind, Seymour Benzer [56], developed Droso-                     terminus and, hence, region harbouring snRNA-binding <br /> Drome, simple device systematically measure flying ability          [35] WD-repeat domains. Gemin5 mRNA transcripts targeted <br /> of flies. The Droso-Drome consisted 1L glass bottle divided             inducible RNAi constructs (Gem5-IRnan Gem5-IRsac), <br /> into 4 sectors, 5 cm each, spanning total height 20 cm.              consisting inverted repeat small fragment present in <br /> Before assay, internal walls Droso-Drome                exon1 (highlighted lavender). A Ret oncogene fragment (exon 5 <br /> coated alcohol-based sticky fluid. Flies introduced             7 including intron) present IR fragments, thereby <br /> into device funnel number flies          enhancing RNAi efficiency. (C) Gemin2 1.2 Kbp two-exon <br /> stuck sector counted, divided total number            gene, highly conserved lacks computationally <br /> flies assayed multiplied 100 generate percentage               identifiable domains. Both N- C-terminus contain SMN- <br /> number flies sector. The flight capability determines             self (G2)-binding domains [31]. Furthermore, crystal studies <br /> height sector flies stick Droso-Drome.                             showed SmF/E SmD1/D1 make contact Gemin2&#226;&#8364;&#8482;s <br />                                                                              N-terminal C-terminal domains, respectively [33]. Knock- <br /> Climbing Assay                                                               Gemin2 achieved expression an <br />     The climbing apparatus consisted polystyrene                inducible RNAi construct (Gem2-IRgau) consisting entire <br /> vials vertically joined tape facing other. For          exon1 exon2 inverted repeat also <br /> lower vial, vertical distance 8 cm surface             separated Ret oncogene fragment boost RNAi efficiency. <br /> was measured marked drawing circle entire                (TIFF) <br /> circumference vial. For climbing assay, group 10-15 <br />                                                                              Figure S2 Pan-muscular overexpression hSMN has <br /> flies transferred lower vial allowed acclimatize <br />                                                                              effect flight behaviour. The distribution organisms <br /> to new setting 1 minute. The climbing assay involved gently <br />                                                                              pan-muscular overexpression hSMN (Mef2- <br /> tapping flies the vial measuring (1) <br />                                                                              GAL4.hSMN) significantly different control <br /> the number flies group, climb 8 cm mark <br /> by 10 seconds tap, recorded percentage success              populations (Mef2-GAL4/+ hSMN/+) course 35 days <br /> rate, (2) time fly group cross         post-eclosion. Note final time point (day 35), the <br /> 8 cm mark. Three trials performed group                  performance test genotype significantly different that <br /> minimum groups assayed genotype.                      driver-only control (Mef2-GAL4/+) responder- <br /> Experiments performed daylight minimize                       (hSMN/+) control. Data presented mean 6 S.E.M. <br /> potential effects circadian oscillation.                                  statistical significance determined differences, at <br />                                                                              sector 1, Mef2-GAL4.hSMN genotype control <br />                                                                              genotypes, indicated respective colour. For all <br /> Immunohistochemistry <br />                                                                              data, ns = significant, *p,0.05, **p,0.01, ***p,0.001. <br />    Larvae dissected 16 PBS, fixed 4% paraformalde- <br /> hyde PBS washed 16 PBS +0.1% TritonH X-100                    (TIFF) <br /> (PBT). The tissues subjected overnight staining              Figure S3 Multiple protein sequence alignment of <br /> room temperature mouse anti-GFP (1:1000; Roche Diagnostics                Gemin5 orthologues. The protein sequence alignment was <br /> Ltd.) mouse anti-hSMN 11F3 (1:20; generous gift Glenn               generated ClustalW EMBL-EBI [95,96] displayed <br /> Morris, Wolfson Centre Inherited Neuromuscular Disease,                  using GeneDoc (http://www.nrbsc.org/gfx/genedoc/). Human, <br /> RJAH Orthopaedic Hospital, Oswestry, UK) [94] antibodies. The                Homo sapiens (Ensembl Protein ID: ENST00000285873); Mouse, <br /> next day, tissues washed PBT stained overnight                Mus musculus (ENSMUST00000172035); Zebrafish, Danio rerio <br /> room temperature anti-mouse Alexa Fluor 546-conjugated                  (ENSDART00000137309);           Fly,    Drosophila      melanogaster <br /> secondary goat antibodies (1:50) nuclear-staining Hoechst                (FBtr0086252). Conservation sequence represented based <br /> 33342 (1:500). Following final wash PBT, samples               Gonnet Protein Weight Matrix, conserved <br /> mounted 90% glycerol anti-fade. Epifluorescent pictures              residues shown light grey (weakly conserved) black <br /> were acquired Optika B-600TiFL microscope (206                    (highly conserved). The N-terminus hosts WD repeats <br /> 406 objectives).                                                             (highlighted red) coiled-coil motif (highlighted blue) <br />                                                                              present C-terminus. <br /> Statistical methods                                                          (TIFF) <br />   Significance tested unpaired t-test. <br />                                                                              Figure S4 Multiple protein sequence alignment of <br />                                                                              Gemin2 orthologues. The protein sequence alignment was <br /> Supporting Information <br />                                                                              generated ClustalW EMBL-EBI [95,96] displayed <br /> Figure S1 Features transgenic constructs utilised                     using GeneDoc (http://www.nrbsc.org/gfx/genedoc/). Human, <br /> in study. (A) Gemin3 4 Kbp two-exon gene set              Homo sapiens (Ensembl Protein ID: ENST00000308317); Mouse, <br /> nine DEAD-box helicase motifs present N-terminus               Mus musculus (ENSMUST00000021379); Zebrafish, Danio rerio <br /> SMN-binding region located middle. The Gem3DN construct               (ENSDART00000149779);          Fly,  Drosophila    melanogaster <br /> is devoid N-terminal region, hosts helicase core            (FBtr0075032). Conservation sequence represented based <br /> but SMN-binding [11,49] region. On                Gonnet Protein Weight Matrix, conserved <br /> hand, Gem3DC devoid C-terminal region, hence,                  residues shown light grey (weakly conserved) black <br /> consisting helicase core. The inducible RNAi                     (highly conserved). <br /> constructs targeting Gemin3 (Gem3-IRmun Gem3-IRdwe)                 (TIFF) <br /> have short fragment (highlighted lavender) derived from <br /> exon1 exon2 inverted repeat (IR), attached to <br /> 10 copies UAS sites enhance RNAi efficiency [51]. (B) Gemin5 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              17                       December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                                      A Motor Function Gemin Associates SMN <br />  <br />  <br />  <br /> Acknowledgments                                                                               stimulating discussions. Finally, study possible <br />                                                                                               unwavering support fundamental research Rectorship <br /> This paper dedicated life Assunta (Sunta) Curmi. We                          University Malta, immensely appreciative. <br /> indebted colleagues University Malta sharing equipment <br /> and reagents. We grateful Aaron DiAntonio, Aaron Voigt, <br /> Barry Dickson, Brian McCabe, Cahir O&#226;&#8364;&#8482;Kane, Glenn Morris, Greg <br />                                                                                               Author Contributions <br /> Matera, Ji-Long Liu, Paul Salvaterra, Spyros Artavanis-Tsakonas                           Conceived designed experiments: RJC. Performed exper- <br /> Vivian Budnik antibodies and/or fly stocks. Thanks Ji-Long                     iments: RB RJC. Analyzed data: RB RJC. Contributed reagents/ <br /> Liu helpful suggestions Neville Vassallo Michael Briese                    materials/analysis tools: RJC. Wrote paper: RJC. <br />  <br /> References <br />  1. Kolb SJ, Sutton S, Schoenberg DR (2010) RNA processing defects associated                 24. Liu JL, Gall JG (2007) U bodies cytoplasmic structures contain uridine- <br />     diseases motor neuron. Muscle Nerve 41: 5&#226;&#8364;&#8220;17.                                     rich small nuclear ribonucleoproteins associate P bodies. Proc Natl <br />  2. Baumer D, Ansorge O, Almeida M, Talbot K (2010) The role RNA                               Acad Sci U S A 104: 11655&#226;&#8364;&#8220;11659. <br />     processing pathogenesis motor neuron degeneration. Expert reviews            25. Cauchi RJ, Sanchez-Pulido L, Liu JL (2010) Drosophila SMN complex proteins <br />     molecular medicine 12: e21.                                                                   Gemin2, Gemin3, Gemin5 components U bodies. Exp Cell Res 316: <br />  3. Lemmens R, Moore MJ, Al-Chalabi A, Brown RH, Jr., Robberecht W (2010)                         2354&#226;&#8364;&#8220;2364. <br />     RNA metabolism pathogenesis motor neuron diseases. Trends                   26. Lee L, Davies SE, Liu JL (2009) The spinal muscular atrophy protein SMN <br />     neurosciences 33: 249&#226;&#8364;&#8220;258.                                                                    affects Drosophila germline nuclear organization U body-P body <br />  4. Sheean RK, Turner BJ (2013) Genetics Motor Neuron Disorders: gene                     pathway. Dev Biol 332: 142&#226;&#8364;&#8220;155. <br />     diversity common cellular conspirators selective neuronal killing. In:              27. Cauchi RJ (2012) Conserved requirement DEAD-box RNA helicase Gemin3 <br />     Cauchi RJ, editor. Drosophila melanogaster Models Motor Neuron Disease.                    Drosophila oogenesis. BMC Res Notes 5: 120. <br />     N.Y.: Nova Biomedical. 1&#226;&#8364;&#8220;34.                                                              28. Buckingham M, Liu JL (2011) U bodies respond <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> stress Drosophila. <br />  5. Kolb SJ, Kissel JT (2011) Spinal muscular atrophy: timely review. Archives               Experimental cell research 317: 2835&#226;&#8364;&#8220;2844. <br />     neurology 68: 979&#226;&#8364;&#8220;984.                                                                    29. Workman E, Kolb SJ, Battle DJ (2012) Spliceosomal small nuclear ribonucleo- <br />  6. Burghes AH, Beattie CE (2009) Spinal muscular atrophy: low levels                   protein biogenesis defects motor neuron selectivity spinal muscular <br />     survival motor neuron protein make motor neurons sick? Nat Rev Neurosci 10:                   atrophy. Brain research 1462: 93&#226;&#8364;&#8220;99. <br />     597&#226;&#8364;&#8220;609.                                                                                  30. Shpargel KB, Matera G (2005) Gemin proteins required efficient <br />  7. Pellizzoni L, Charroux B, Dreyfuss G (1999) SMN mutants spinal muscular                    assembly Sm-class ribonucleoproteins. Proc Natl Acad USA 102: 17372&#226;&#8364;&#8220; <br />     atrophy patients defective binding snRNP proteins. Proc Natl Acad                   17377. <br />     Sci U S A 96: 11167&#226;&#8364;&#8220;11172.                                                                31. Ogawa C, Usui K, Aoki M, Ito F, Itoh M, et al. (2007) Gemin2 plays an <br />  8. Lorson CL, Strasswimmer J, Yao JM, Baleja JD, Hahnen E, et al. (1998) SMN                     important role stabilizing survival motor neuron complex. J Biol Chem <br />     oligomerization defect correlates spinal muscular atrophy severity. Nat                  282: 11122&#226;&#8364;&#8220;11134. <br />     Genet 19: 63&#226;&#8364;&#8220;66.                                                                          32. Feng W, Gubitz AK, Wan L, Battle DJ, Dostie J, et al. (2005) Gemins modulate <br />  9. Young PJ, Man NT, Lorson CL, Le TT, Androphy EJ, et al. (2000) The exon 2b                    expression activity SMN complex. Hum Mol Genet 14: 1605&#226;&#8364;&#8220; <br />     region spinal muscular atrophy protein, SMN, involved self-                      1611. <br />     association SIP1 binding. Hum Mol Genet 9: 2869&#226;&#8364;&#8220;2877.                                 33. Zhang R, So BR, Li P, Yong J, Glisovic T, et al. (2011) Structure key <br /> 10. Liu Q, Fischer U, Wang F, Dreyfuss G (1997) The spinal muscular atrophy                       intermediate SMN complex reveals Gemin2&#226;&#8364;&#8482;s crucial function snRNP <br />     disease gene product, SMN, associated protein SIP1 complex                   assembly. Cell 146: 384&#226;&#8364;&#8220;395. <br />     spliceosomal snRNP proteins. Cell 90: 1013&#226;&#8364;&#8220;1021.                                     34. Battle DJ, Lau CK, Wan L, Deng H, Lotti F, et al. (2006) The Gemin5 protein <br /> 11. Charroux B, Pellizzoni L, Perkinson RA, Shevchenko A, Mann M, et al. (1999)                   SMN complex identifies snRNAs. Mol Cell 23: 273&#226;&#8364;&#8220;279. <br />     Gemin3: novel DEAD box protein interacts SMN, spinal                      35. Lau CK, Bachorik JL, Dreyfuss G (2009) Gemin5-snRNA interaction reveals an <br />     muscular atrophy gene product, component Gems. J Cell Biol 147:                   RNA binding function WD repeat domains. Nat Struct Mol Biol 16: 486&#226;&#8364;&#8220; <br />     1181&#226;&#8364;&#8220;1193.                                                                                    491. <br /> 12. Charroux B, Pellizzoni L, Perkinson RA, Yong J, Shevchenko A, et al. (2000)               36. Yong J, Kasim M, Bachorik JL, Wan L, Dreyfuss G (2010) Gemin5 delivers <br />     Gemin4: novel component SMN complex gems                       snRNA precursors SMN complex snRNP biogenesis. Mol Cell 38: <br />     nucleoli. J Cell Biol 148: 1177&#226;&#8364;&#8220;1186.                                                     551&#226;&#8364;&#8220;562. <br /> 13. Gubitz AK, Mourelatos Z, Abel L, Rappsilber J, Mann M, et al. (2002) Gemin5,              37. Yan X, Mouillet JF, Ou Q, Sadovsky Y (2003) A novel domain the <br />     novel WD repeat protein component SMN complex binds Sm                          DEAD-box protein DP103 essential transcriptional repression helicase <br />     proteins. J Biol Chem 277: 5631&#226;&#8364;&#8220;5636.                                                         activity. Mol Cell Biol 23: 414&#226;&#8364;&#8220;423. <br /> 14. Pellizzoni L, Baccon J, Rappsilber J, Mann M, Dreyfuss G (2002) Purification           38. Fallini C, Bassell GJ, Rossoll W (2012) Spinal muscular atrophy: role of <br />     native survival motor neurons complexes identification Gemin6                  SMN axonal mRNA regulation. Brain research 1462: 81&#226;&#8364;&#8220;92. <br />     novel component. J Biol Chem 277: 7540&#226;&#8364;&#8220;7545.                                              39. Kroiss M, Schultz J, Wiesner J, Chari A, Sickmann A, et al. (2008) Evolution of <br /> 15. Baccon J, Pellizzoni L, Rappsilber J, Mann M, Dreyfuss G (2002) Identification                RNP assembly system: minimal SMN complex facilitates formation of <br />     characterization Gemin7, novel component survival motor                    UsnRNPs Drosophila melanogaster. Proc Natl Acad Sci U S A 105: 10045&#226;&#8364;&#8220; <br />     neuron complex. J Biol Chem 277: 31957&#226;&#8364;&#8220;31962.                                                 10050. <br /> 16. Carissimi C, Saieva L, Baccon J, Chiarella P, Maiolica A, et al. (2006) Gemin8         40. Cauchi RJ, Davies KE, Liu JL (2008) A motor function DEAD-box RNA <br />     novel component survival motor neuron complex functions small                 helicase, Gemin3, Drosophila. PLoS Genet 4: e1000265. <br />     nuclear ribonucleoprotein assembly. J Biol Chem 281: 8126&#226;&#8364;&#8220;8134.                           41. Grice SJ, Sleigh JN, Liu JL, Sattelle DB (2011) Invertebrate models spinal <br /> 17. Carissimi C, Saieva L, Gabanella F, Pellizzoni L (2006) Gemin8 required                muscular atrophy: insights mechanisms potential therapeutics. <br />     architecture function survival motor neuron complex. J Biol                    BioEssays : news reviews molecular, cellular developmental biology <br />     Chem 281: 37009&#226;&#8364;&#8220;37016.                                                                        33: 956&#226;&#8364;&#8220;965. <br /> 18. Campbell L, Unter KMD, Mohaghegh P, Tinsley JM, Brash MA, et al. (2000)                   42. Chan YB, Miguel-Aliaga I, Franks C, Thomas N, Trulzsch B, et al. (2003) <br />                                                                                                   Neuromuscular defects Drosophila survival motor neuron gene mutant. <br />     Direct interaction Smn dp103, putative RNA helicase: role Smn in <br />                                                                                                   Hum Mol Genet 12: 1367&#226;&#8364;&#8220;1376. <br />     transcription regulation? Hum Mol Genet 9: 1093&#226;&#8364;&#8220;1100. <br />                                                                                               43. Rajendra TK, Gonsalvez GB, Walker MP, Shpargel KB, Salz HK, et al. (2007) <br /> 19. Carissimi C, Baccon J, Straccia M, Chiarella P, Maiolica A, et al. (2005) Unrip is <br />                                                                                                   A Drosophila melanogaster model spinal muscular atrophy reveals function <br />     component SMN complexes active snRNP assembly. FEBS Lett 579: <br />                                                                                                   SMN striated muscle. J Cell Biol 176: 831&#226;&#8364;&#8220;841. <br />     2348&#226;&#8364;&#8220;2354. <br />                                                                                               44. Miguel-Aliaga I, Chan YB, Davies KE, van den Heuvel M (2000) Disruption of <br /> 20. Grimmler M, Otter S, Peter C, Muller F, Chari A, et al. (2005) Unrip, factor <br />                                                                                                   SMN function ectopic expression human SMN gene Drosophila. <br />     implicated cap-independent translation, associates cytosolic SMN <br />                                                                                                   FEBS Lett 486: 99&#226;&#8364;&#8220;102. <br />     complex influences intracellular localization. Hum Mol Genet 14: 3099&#226;&#8364;&#8220; <br />                                                                                               45. Muller HJ (1932) Further studies nature causes gene mutations. <br />     3111. <br />                                                                                                   Proceedings 6th Internatioinal Congress Genetics, Ithaca, NY: 213&#226;&#8364;&#8220; <br /> 21. Cauchi RJ (2010) SMN Gemins: &#226;&#8364;&#732;we family&#226;&#8364;&#8482; &#226;&#8364;&#166; we? Insights into <br />                                                                                                   255. <br />     partnership Gemins spinal muscular atrophy disease <br />                                                                                               46. Prelich G (2012) Gene overexpression: uses, mechanisms, interpretation. <br />     protein SMN. Bioessays 32: 1077&#226;&#8364;&#8220;1089. <br />                                                                                                   Genetics 190: 841&#226;&#8364;&#8220;854. <br /> 22. Liu Q, Dreyfuss G (1996) A novel nuclear structure containing survival of <br />                                                                                               47. Herskowitz I (1987) Functional inactivation genes dominant negative <br />     motor neurons protein. Embo J 15: 3555&#226;&#8364;&#8220;3565. <br />                                                                                                   mutations. Nature 329: 219&#226;&#8364;&#8220;222. <br /> 23. Cauchi RJ (2011) Gem formation constitutive Gemin3 overexpression in <br />                                                                                               48. Wu Y, Brosh RM, Jr. (2010) Helicase-inactivating mutations basis for <br />     Drosophila. Cell Biol Int 35: 1233&#226;&#8364;&#8220;1238. <br />                                                                                                   dominant negative phenotypes. Cell cycle 9: 4080&#226;&#8364;&#8220;4090. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                               18                           December 2013 | Volume 8 | Issue 12 | e83878 <br />                                                                                                                      A Motor Function Gemin Associates SMN <br />  <br />  <br /> 49. Shpargel KB, Praveen K, Rajendra TK, Matera AG (2009) Gemin3                       76. Pacheco A, Lopez Quinto S, Ramajo J, Fernandez N, Martinez-Salas E <br />     essential gene required larval motor function pupation Drosophila.                (2009) A novel role Gemin5 mRNA translation. Nucleic Acids Res 37: <br />     Mol Biol Cell 20: 90&#226;&#8364;&#8220;101.                                                                    582&#226;&#8364;&#8220;590. <br /> 50. Cauchi RJ, van den Heuvel M (2006) The fly model neurodegenerative              77. Fierro-Monti I, Mohammed S, Matthiesen R, Santoro R, Burns JS, et al. (2006) <br />     diseases: worth jump? Neurodegener Dis 3: 338&#226;&#8364;&#8220;356.                                 Quantitative proteomics identifies Gemin5, scaffolding protein involved in <br /> 51. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) A genome-                    ribonucleoprotein assembly, novel partner eukaryotic initiation factor <br />     wide transgenic RNAi library conditional gene inactivation Drosophila.                4E. J Proteome Res 5: 1367&#226;&#8364;&#8220;1378. <br />     Nature 448: 151&#226;&#8364;&#8220;156.                                                                     78. Takizawa Y, Qing Y, Takaku M, Ishida T, Morozumi Y, et al. (2010) GEMIN2 <br /> 52. Klowden MJ (2007) Physiological Systems Insects. NY: Academic Press.                      promotes accumulation RAD51 double-strand breaks homologous <br /> 53. Dickinson M (2006) Insect flight. Current biology : CB 16: R309&#226;&#8364;&#8220;314.                         recombination. Nucleic Acids Res. <br /> 54. Vigoreaux JO (2006) Molecular Basis Muscle Structure. In: Sink H, editor.             79. Takaku M, Tsujita T, Horikoshi N, Takizawa Y, Qing Y, et al. (2011) <br />     Muscle Development Drosophila. Texas: Landes Bioscience.                                  Purification human SMN-GEMIN2 complex assessment its <br /> 55. Taylor MV (2006) Comparison Muscle Development Drosophila                          stimulation RAD51-mediated DNA recombination reactions. Biochemistry <br />     Vertebrates. In: Sink H, editor. Muslce Development Drosophila. Texas:                    50: 6797&#226;&#8364;&#8220;6805. <br />     Landes Bioscience.                                                                       80. Lotti F, Imlach WL, Saieva L, Beck ES, Hao le T, et al. (2012) An SMN- <br /> 56. Benzer S (1973) Genetic dissection behavior. Sci Am 229: 24&#226;&#8364;&#8220;37. <br />                                                                                                  dependent U12 splicing event essential motor circuit function. Cell 151: 440&#226;&#8364;&#8220; <br /> 57. Crivat G, Taraska JW (2012) Imaging proteins inside cells fluorescent tags. <br />                                                                                                  454. <br />     Trends biotechnology 30: 8&#226;&#8364;&#8220;16. <br />                                                                                              81. Imlach WL, Beck ES, Choi BJ, Lotti F, Pellizzoni L, et al. (2012) SMN is <br /> 58. Grice SJ, Liu JL (2011) Survival motor neuron protein regulates stem cell <br />                                                                                                  required sensory-motor circuit function Drosophila. Cell 151: 427&#226;&#8364;&#8220;439. <br />     division, proliferation, differentiation Drosophila. PLoS Genet 7: <br />     e1002030.                                                                                82. Gabanella F, Butchbach ME, Saieva L, Carissimi C, Burghes AH, et al. (2007) <br /> 59. Timmerman C, Sanyal S (2012) Behavioral electrophysiological outcomes                    Ribonucleoprotein Assembly Defects Correlate Spinal Muscular Atrophy <br />     tissue-specific Smn knockdown Drosophila melanogaster. Brain research                  Severity Preferentially Affect Subset Spliceosomal snRNPs. PLoS ONE <br />     1489: 66&#226;&#8364;&#8220;80.                                                                                 2: e921. <br /> 60. Chang HC, Dimlich DN, Yokokura T, Mukherjee A, Kankel MW, et al. (2008)                  83. Zhang Z, Lotti F, Dittmar K, Younis I, Wan L, et al. (2008) SMN deficiency <br />     Modeling spinal muscular atrophy Drosophila. PLoS ONE 3: e3209.                           causes tissue-specific perturbations repertoire snRNAs widespread <br /> 61. Gates J, Lam G, Ortiz JA, Losson R, Thummel CS (2004) rigor mortis encodes                 defects splicing. Cell 133: 585&#226;&#8364;&#8220;600. <br />     novel nuclear receptor interacting protein required ecdysone signaling        84. Workman E, Saieva L, Carrel TL, Crawford TO, Liu D, et al. (2009) A SMN <br />     Drosophila larval development. Development 131: 25&#226;&#8364;&#8220;36.                                       <span id='am-245' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>missense mutation</span> complements SMN2 restoring snRNPs rescuing SMA <br /> 62. Jablonka S, Holtmann B, Meister G, Bandilla M, Rossoll W, et al. (2002) Gene                 mice. Human molecular genetics 18: 2215&#226;&#8364;&#8220;2229. <br />     targeting Gemin2 mice reveals correlation defects                 85. Carrel TL, McWhorter ML, Workman E, Zhang H <br /> </body></html>