<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Functional Characterization Aspergillus nidulans <br /> ypkA, Homologue Mammalian Kinase SGK <br /> Ana Cristina Colabardini2, Neil Andrew Brown2, Marcela Savoldi2, Maria Helena S. Goldman3, Gustavo <br /> Henrique Goldman1,2* <br /> 1 Laborato&#194;&#180;rio Nacional Cie&#203;&#8224;ncia e Tecnologia Bioetanol &#226;&#8364;&#8220; CTBE, Sa&#203;&#339;o Paulo, Brazil, 2 Faculdade Cie&#203;&#8224;ncias Farmace&#203;&#8224;uticas Ribeira&#203;&#339;o Preto, Universidade Sa&#203;&#339;o <br /> Paulo, Sa&#203;&#339;o Paulo, Brazil, 3 Faculdade Filosofia, Cie&#203;&#8224;ncias e Letras Ribeira&#203;&#339;o Preto, Universidade Sa&#203;&#339;o Paulo, Sa&#203;&#339;o Paulo, Brazil <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The serum- glucocorticoid-regulated protein kinase (SGK) AGC kinase involved signal cascades regulated by <br />      glucocorticoid hormones serum mammals. The Saccharomyces cerevisiae ypk1 ypk2 genes identified SGK <br />      homologues Ypk1 shown regulate balance sphingolipids inner outer plasma membrane. <br />      This investigation characterized Aspergillus nidulans YPK1 homologue, YpkA, representing filamentous fungal <br />      YPK1 homologue. Two conditional mutant strains constructed replacing endogenous ypk1 promoter two <br />      different regulatable promoters, alcA (<span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> alcohol dehydrogenase gene) niiA (from nitrate reductase gene). <br />      Both constructs confirmed ypkA essential gene A. nidulans. Repression ypkA caused decreased radial <br />      growth, delay conidial germination, deficient polar axis establishment, intense branching late stages growth, a <br />      lack asexual spores, terminal <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span>. Membrane lipid polarization, endocytosis, eisosomes vacuolar <br />      distribution affected ypkA repression, suggesting YpkA plays role hyphal morphogenesis via <br />      coordinating delivery cell membrane wall constituents hyphal apex. The A. nidulans Pkh1 homologue pkhA <br />      shown essential gene, preliminary genetic analysis suggested ypkA gene directly <br />      downstream pkhA epistatic pkhA, rather, ypkA pkhA genetically independent parallel. BarA a <br />      homologue yeast Lag1 acyl-CoA-dependent ceramide synthase, catalyzes condensation phyto- <br />      sphingosine fatty acyl-CoA form phytoceramide. When barA absent, ypkA repression lethal cell. <br />      Therefore, appears genetic interaction ypkA, barA, sphingolipid synthesis. Transcriptional <br />      profiling ypkA overexpression down-regulation revealed putative YpkA targets associated observed <br />      phenotypes. <br />  <br />   Citation: Colabardini AC, Brown NA, Savoldi M, Goldman MHS, Goldman GH (2013) Functional Characterization Aspergillus nidulans ypkA, Homologue the <br />   Mammalian Kinase SGK. PLoS ONE 8(3): e57630. doi:10.1371/journal.pone.0057630 <br />   Editor: Jae-Hyuk Yu, University Wisconsin - Madison, United States America <br />   Received November 28, 2012; Accepted January 23, 2013; Published March 5, 2013 <br />   Copyright: &#195;&#376; 2013 Colabardini et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: Fundac&#194;&#184;a&#203;&#339;o Amparo a` Pesquisa Estado Sa&#203;&#339;o Paulo (FAPESP) Conselho Nacional Desenvolvimento Cient&#196;&#177;&#194;&#180;fico e Tecnolo&#194;&#180;gico (CNPq). The <br />   funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. Gustavo Henrique Goldman PLOS ONE Editorial Board member. This <br />   does alter authors&#226;&#8364;&#8482; adherence PLOS ONE policies sharing data materials. <br />   * E-mail: address: ggoldman@usp.br <br />  <br />  <br />  <br /> Introduction                                                                            mammalian SGK restored viability. The S. cerevisiae pkh1 protein <br />                                                                                         kinase possesses catalytic domain resembles mammalian <br />    Protein kinases key regulators cellular function act                     PDK1 catalytic domain. In S. cerevisiae, Pkh1 essential. <br /> via phosphorylating target proteins directing activity,                      Complementation Dpkh1 PDK1 restored viability, <br /> location function. Serum- glucocorticoid-regulated                              Ypk1 demonstrated phosphorylated Pkh1 <br /> protein kinases (SGK) members AGC kinase subfamily                           Thr504 residue, indicating PDK1 pkh1 are <br /> (PKA, PKC, PKC) involved signal cascades                           functional homologues [6]. <br /> regulated mammals serum glucocorticoid hormones [1].                             Sphingolipids essential components eukaryotic cells <br /> Additionally, mammalian SGK activated response                         providing membrane structure, acting secondary messengers <br /> follicle-stimulating hormone produced brain damage                       interacting sterols form lipid-rafts [7,8]. In eukaryotic <br /> [2] transfection mammary epithelial cells p53                       cells, sphingolipids involved important cellular <br /> transcription factor [3]. The threonine residue (Thr256), located                    processes endocytosis [9,10], intracellular trafficking of <br /> the SGK activation loop catalytic domain, phosphorylated                      proteins lipids [11,12], cell cycle control [13,14], heat <br /> by 3-phosphoinositide-dependent protein kinase-1 (PDK1),                            shock response [15,16]. In S. cerevisiae Ypk1 regulates flipase <br /> which serves central integrator signaling cascades.                             protein kinases, Fpk1 Fpk2, responsible maintaining the <br /> Additional PDK1 targets include p70 S6 kinase, PKC                                  balance sphingolipids inner outer plasma <br /> isoforms, PKB SGK [4]. Two S. cerevisiae genes, ypk1                            membrane activating flipases proteins, maintain layer <br /> ypk2, identified SGK homologues. The ypk1 shown                          asymmetry expulsion amino phospholipids the <br /> confer resistance ISP-1, inhibitor sphingolipid synthesis                      outer layer. Ypk1 negatively regulates Fpk1, ypk1 null <br /> [5]. Complementation Dypk1 S. cerevisiae strain                         mutant possesses defects result flipases hyper-activity <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                          1                                 March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                             Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> which deleterious cell viability [17]. In mammals,          Furthermore, cell wall dynamic structure functions as <br /> characterized secondary messengers, derived                  sensor turgor continuously remodeling response <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> sphingolipids, sphingosine 1-phosphate ceramides,              environmental stimuli growth [35]. The S. cerevisiae protein <br /> involved growth apoptosis signaling [18,19]. In S. cerevisiae,       kinase C (Pkc1) activates cell wall maintenance MAP kinase <br /> phytosphingosine activates Pkh1 turn activates Ypk1                signaling cascade response osmotic stress restrictive <br /> [20]. Ypk1 responsible inactivation endoplasmic           temperatures, mediating signal TORC2, regulate the <br /> reticulum membrane proteins, Orm1 Orm2, inhibit               actin cytoskeleton [36]. Pkh2 possibly Pkh1 phosphorylate <br /> responsible catalytic step sphingolipid                activate Pkc1 [37]. In addition, ypk mutants demonstrate a <br /> biosynthesis [21].                                                          random distribution actin cytoskeleton deficient in <br />    The integral highly conserved serine/threonine protein kinase,           activation MAP cascade [38]. <br /> target rapamycin (TOR), forms complexes (TORC1                      This study presents YPK1 homologue, ypkA, be <br /> TORC2) regulate cell growth metabolism response              characterized filamentous fungus. In A. nidulans, ypkA was <br /> the environment. AGC kinases activated phosphorylation            shown essential gene involved polar axis <br /> the activation loop, turn motif hydrophobic motif [22].                 establishment germling growth. Membrane lipid polarization <br /> Hydrophobic motif phosphorylation Ypk1 (Thr 662)                  deficient ypkA conditional mutant grown in <br /> mediated TORC2, regulates cytoskeleton organization,               repressing conditions, suggesting YpkA plays important <br /> [23] phosphorylation site required resistance        role hyphal morphogenesis affects endocytosis and <br /> myriocin, inhibitor sphingolipid synthesis. In response            vacuolar distribution A. nidulans. <br /> sphingolipids depletion, S. cerevisiae ypk1T662A mutant low <br /> Orm phosphorylation vivo, as, low activation vitro.           Results <br /> Thus Ypk1 sensor effector sphingolipids levels, <br /> with sphingolipid reduction, stimulating Ypk1          The A. nidulans ypkA Gene Essential <br /> TORC2 mediated phosphorylation [21]. Besides interaction                   A BLASTp search A. nidulans genome database (http:// <br /> with TORC2, Ypk1 interacts TORC1. The TORC1                       www.aspgd.org) using S. cerevisiae Ypk1p Ypk2p queries <br /> complex positively regulates translation initiation, biogenesis          revealed single open reading frame (ORF) significant <br /> ribosomes, uptake amino acids sensing             similarity. The potential homologue, AN5973 (here named ypkA), <br /> <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability. Protein translation rapidly interrupted        predicted 640-amino acid protein kinase high <br /> response lack nitrogen autophagic proteolysis           identity Ypk1p (2e2135; 47.5% identity 62.8% similarity) <br /> of Ypk1 [24]. TORC1 Ypk1 differentially                   Ypk2 (1.1e2134; 47.8% identity; 62.1% similarity) mainly <br /> controlled lack nitrogen, share downstream           C-terminal region. YpkA defined domains, a <br /> targets, translation initiation factor eIF4G [25].              protein kinase domain (PS50011) residues 293 550 an <br />    The highly polarized nature fungal cells hallmark         AGC-kinase C-terminal domain (PS51285) residues 551 to <br /> their morphology grow insertion new                622. An vivo S. cerevisiae fusion-based approach (see Materials <br /> membrane cell wall surface. The tubular cell shape          Methods) used attempt generate A. nidulans <br /> to fact growth confined hyphae apical hub [26]. For          ypkA null mutant. Several primary transformants isolated. <br /> this purpose, vesicles loaded components required cell         However, transformants displayed pronounced phenotypic <br /> wall expansion transported active sites growth             defect MM, i.e. small non-sporulating colonies (Figure 1A; <br /> network polarized microtubes [27]. Therefore, polarized growth           middle right panels). These colonies unable grow or <br /> requires proteins involved cytoskeleton functions secretory          produce significant mycelium liquid medium, <br /> endocytic machinery [28]. The plasma membrane consists                   suggesting terminal phenotype strains. Heterokaryon <br /> different sub-domains defined distribution sphingolipids          rescue [39] confirmed ypkA essential gene (data not <br /> and sterols. These sphingolipids grouped sub-domains,           shown). Subsequently, conditional mutants ypkA were <br /> rather distributed homogeneously                  constructed replacing endogenous ypkA promoter the <br /> glyceroglycolipid membrane [29]. Sterols four-ring structures           alcA niiA promoters. The alcA promoter repressed <br /> that possess aliphatic tail, &#226;&#8364;&#732;&#226;&#8364;&#732;pack&#226;&#8364;&#8482;&#226;&#8364;&#8482; sphingolipids            glucose, derepressed glycerol induced high levels by <br /> together [30]. These &#226;&#8364;&#732;&#226;&#8364;&#732;packs&#226;&#8364;&#8482;&#226;&#8364;&#8482;, referred lipid-rafts, play         ethanol L-threonine [40]. Transformants accumulated <br /> important role protein localization signal transduction.             approximately 3-fold higher ypkA mRNA transferred to <br /> Lipid-rafts serve mounting organizing centers signaling          glycerol 2%+threonine transferred glucose 4% were <br /> molecules important polar organization         selected (Figure 1B). The second promoter niiA (encoding a <br /> cell [31].                                                                  nitrite reductase) induced sodium nitrate repressed by <br />    Lipids implicated performing role                 ammonium tartrate [41]. Transformants showed approx- <br /> membrane trafficking [32]. In mammalian cells, sphingosine                  imate four-fold decrease ypkA mRNA accumulation when <br /> induces phosphorylation certain protein kinases vitro [33].       transferred ammonium tartrate transferred to <br /> In S. cerevisiae, sphingolipids induce Pkh1 auto-phosphorylation,           sodium nitrate selected (Figure 1C). <br /> which turn activates proteins kinases, including              Repression growing alcA::ypkA mutant strain the <br /> Ypk1 [6]. Mutations block early stages endocytosis              presence 4% glucose decreased colony diameter approximately <br /> show reduction assimilation fatty acids, indicating         four-fold (Figure 1D). In contrast, wild-type alcA::ypkA <br /> that Ypk1 affects uptake fatty acids regulating                   strains radial diameter grown 2% glycerol <br /> endocytosis [34]. In addition signaling events vesicular             (Figure 1D). Overexpression ypkA did cause detectable <br /> transport, control cell growth involves synthesis new       phenotypic change (data shown). Repression growing the <br /> cell wall skeleton reorganization actin. The actin               niiA::ypkA mutant strain presence ammonium tartrate <br /> cytoskeleton directs delivery components hyphal apex          caused dramatic ten-fold decrease colony diameter <br /> and essential maintaining polarized growth. In fungi,         (Figure 1E). These results strongly indicate ypkA essential A. <br /> cell wall determines shape of, confers rigidity to, cell.       nidulans gene. The construction strains, different <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              2                           March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                               Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 1. The ypkA gene essential A. nidulans. (A) Wild-type primary DypkA transformant grown (left center panel) or <br /> streaked (right panel) YAG medium 96 hours 37uC. (B) The alcA::ypkA strain grown 6 hours MM+4% glucose MM+2% glycerol <br /> +threonine 100 mM (C) The niiA::ypkA strain grown 6 hours MM+sodium nitrate 10 mM MM+ammonium tartrate 50 mM. The relative <br /> quantitation ypkA tubulin gene expression determined standard curve (i.e., CT &#226;&#8364;&#8220;values plotted logarithm DNA copy <br /> number). The presented results means (6 standard deviation) biological replicates. The growth phenotypes alcA::ypkA (D) and <br /> niiA::ypkA (E) mutant strains. The A. nidulans wild-type, alcA::ypkA, niiA::ypkA mutant strains grown 72 hours 37uC MM+4% <br /> glucose MM+2% glycerol MM+sodium nitrate 10 mM MM+ammonium tartrate 50 mM. <br /> doi:10.1371/journal.pone.0057630.g001 <br />  <br /> levels YpkA expression, desired enable different                     To evaluate sub-cellular localisation YpkA, YpkA::GFP <br /> experimental approaches assist investigation           strain constructed. The YpkA::GFP strain behaved exactly the <br /> YpkA function. For instance, niiA promoter allows little           wild-type strain (data shown). When the <br /> leakage repression used study impact            YpkA::GFP strain grown YG 16 hours 30oC, <br /> YpkA loss function while, alcA promoter allows study          disperse fluorescence observed cytoplasm appeared <br /> the ypkA overexpression.                                                    similar distribution tubulin (Figure 2A). To test this <br />                                                                             hypothesis, double YpkA::GFP TubC::mRFP strain was <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                             March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                             Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 2. YpkA::GFP partially co-localizes TubC::mRFP. Conidia grown 16 hours 30uC YG medium, YpkA::GFP (A) and <br /> YpkA::GFP TubC::mRFP (B). Bars, 5 mm. <br /> doi:10.1371/journal.pone.0057630.g002 <br />  <br /> constructed crossing corresponding parental strains. Partial         tubes showed single polar axis (Figure 3C). However, we <br /> co-localization proteins observed (Figure 2B).               observed multiple polar axes niiA::ypkA strain grown in <br />                                                                             presence ammonium tartrate (Figure 3C). These results <br /> A. nidulans YpkA Involved Polar Growth                        suggest YpkA play role recruiting morphoge- <br /> Endocytosis                                                                 netic machinery, including instance components a <br />     The morphogenetic program associated germination            multiprotein complex termed polarisome [26] site of <br /> A. nidulans conidiospores includes; (i) brief period isotropic         polarization. <br /> spore swelling, (ii) establishment stable polarity axis,          Subsequently, constructed strains used to <br /> (iii) emergence polarized germ tube [42]. This program             investigate effects ypkA repression overexpression under <br /> leads formation multinucleate hyphal cell grows         different experimental conditions (Figure 4). Under repressing <br /> apical extension. We assess YpkA affects polar growth, germ-             conditions plus high temperature (44uC) presence of <br /> tube emergence number nuclei wild-type            lovastatin, growth alcA::ypkA strain reduced greater <br /> niiA::ypkA hyphae grown presence sodium nitrate         extent wild-type strain (Figures 4A B). However, <br /> ammonium tartrate (Figures 3A&#226;&#8364;&#8220;B). A delay germ-tube                      repressing conditions alcA::ypkA strain resistant <br /> emergence observed wild-type strain grown               SDS (Sodium Dodecyl Sulfate) Calcofluor white (CFW) <br /> media containing ammonium tartrate (Figure 3A). No significant              wild-type strain (Figure 4A). Surprisingly, either <br /> difference germ-tube emergence number nuclei               repressing overexpressing conditions, growth alcA::ypkA <br /> observed wild-type niiA::ypkA mutant strains            strain affected myriocin phytosphingosine <br /> in hours germination (Figures 3A&#226;&#8364;&#8220;B 4A).              (Figure 4B). Overexpression ypkA slightly increased the <br /> However, nuclei niiA::ypkA mutant grown                    resistance lovastatin (Figure 4B). Accordingly, repressing <br /> repressing conditions appeared fragmented (Figure 5A). The                  conditions, growth niiA::ypkA strain highly <br /> overexpression YpkA did affect germ-tube emergence            affected higher temperatures (44uC) lovastatin, being <br /> number nuclei hours germination (data          resistant SDS CFW (Figures 4C D). Again, under <br /> shown). When niiA::ypkA strain grown longer periods             repressing conditions, niiA::ypkA strain affected by <br /> (10 16 hours) presence sodium nitrate, 100% germ-           phytosphingosine (Figure 4B). <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              4                           March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                             Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 3. A. nidulans YpkA involved polarized growth. The percentage wild-type niiA::ypkA mutant germlings exhibited polar <br /> growth (defined emergence germ tube) (A) number nuclear germling (B). Conidia grown 2 8 hours at <br /> 37uC. Averages (6 standard deviation) represent 100 germlings independent experiments (Wt = Wild-type). (C) The germination <span id='am-321' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of <br /> the wild-type niiA::ypkA conidiospores. Conidia allowed germinate MM media 8 16 hours. Conidia possessing germ tubes were <br /> classified displaying (left right) unipolar (1), bipolar (2), unipolar plus lateral branches (3) bipolar plus lateral branches (4). <br /> doi:10.1371/journal.pone.0057630.g003 <br />  <br />    Germlings wild-type niiA::ypkA mutant strains           nitrate ammonium tartrate (Figure 5B, upper panel). Filipin <br /> stained filipin, fluorescent polyene antibiotic binds          staining localized hyphal apex niiA::ypkA mutant <br /> sterols [43], determine membrane lipids              inducing conditions, staining uniformly dispersed <br /> delivered hyphal apex polar growth (Figure 5B).              membrane repressing conditions (Figure 5B, <br /> Intense filipin staining observed hyphal apex            lower panel). In A. nidulans, FITC-conjugated lectin WGA can <br /> wild-type strain grown presence sodium               used detect sites cell wall deposition [26]. The confinement <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             5                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                         Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   6   March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                                 Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br /> Figure 4. Evaluation effect YpkA depletion overexpression growth A. nidulans different experimental <br /> conditions. Five ml ten-fold dilution series starting concentration 26107 wild-type (A-D), alcA::ypkA (A-B), niiA::ypkA (C&#226;&#8364;&#8220;D) <br /> strains spotted different growth media grown 72 hours 37uC, experiments temperature evaluated (A and <br /> C). CFW = calcofluor white; CR = congo red; MR = myriocin; PHS = phytosphingosine; SDS = sodium dodecyl sulphate. <br /> doi:10.1371/journal.pone.0057630.g004 <br />  <br /> of FITC-WGA staining hyphal apex niiA::ypkA                     vesicles carrying cell wall precursors inappropriately <br /> mutant, observed inducing conditions (Figure 5C, upper               distributed hyphae mutant germlings ypkA <br /> panel), lost ypkA repression (Figure 5C, lower panel).             repression. <br /> CFW staining demonstrated similar results, CFW localiza-                   It demonstrated S. cerevisiae Ypk1 acts downstream <br /> tion hyphal apex lost, niiA::ypkA strain,             Pkh kinases control endocytosis phosphorylating <br /> repressing conditions (Figure 5D). Additional septa noted           components endocytic machinery [44]. Ypk1 possibly <br /> in niiA::ypkA germlings grown repressing                       human Sgk1 kinase affect fatty-acid uptake energy <br /> conditions (Figure 5D). These observations suggest pool              homeostasis regulating endocytosis [34]. FM4-64 assays <br />  <br />  <br />  <br />  <br /> Figure 5.The polarized delivery membrane lipids cell wall deposition confined hyphal apex niiA::ypkA <br /> mutant repression. In experiments, germlings grown 16 hours 37uC inducing (sodium nitrate) repressing conditions <br /> (ammonium tartrate). Stains utilized: (A) Hoescht, (B) Filipin, (C) FITC-conjugated wheat germ, (D) CFW. Bars, 5 10 mm. <br /> doi:10.1371/journal.pone.0057630.g005 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                7                             March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                                Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> were performed investigate intracellular trafficking, secretion,          logues Pil1/Lsp1, PilA PilB, identified A. nidulans <br /> and vesicular transport. Under inducing conditions, FM4-64                   [48]. In A. nidulans wild-type mycelia, punctate structures <br /> staining revealed Spitzenko&#194;&#168;rper hyphal apex             composed PilA present, PilB diffused the <br /> structures probably represent mature endosomes/vacuoles                 cytoplasm. The construction pilA::gfp pilB::gfp the <br /> [45] wild-type niiA::ypkA strains. In contrast,             niiA::ypkA background enabled evaluation PilA PilB <br /> repressing conditions Spitzenko&#194;&#168;rper visualized          localization ypkA induction repression. When grown <br /> the niiA::ypkA strain significant decrease       inducing conditions, previously observed wild-type <br /> endosome/vacuole structures (Figure 6A). To verify function              strain, PilA localized punctate structures cytoplasm, while <br /> of YpkA endocytosis, live cells niiA::ypkA strain             PilB diffused cytoplasm (Figure 7A). Upon <br /> stained FM4-64 uptake dye tracked time.             ypkA repression, increase punctate distribution <br /> Under inducing conditions, FM4-64 visible plasma                  PilA PilB cytoplasm (Figure 7B). These <br /> membrane cell 10 min, taken cells             results suggest depletion YpkA affect eisosome <br /> and localized endomembranes, mature                          turnover, increasing number structures. <br /> endosomes vacuoles, 30 min (Figure 6B&#226;&#8364;&#8220;C). Under                        Taken results imply ypkA performs an <br /> repressing conditions, FM4-64 uptake delayed. After 10 min               essential role hyphal morphogenesis filamentous growth, <br /> the dye remained plasma membrane structures               reduction ypkA expression resulting deficiencies in <br /> resembled septa (Figure 6B). Even 60 min more, staining             polarization related endocytosis, vesicle transport the <br /> of endomembrane remained diffuse (Figure 6D).                                polarized delivery chitin/lipid hyphal apex. <br />    Eisosomes fungal subcortical organelles play roles in <br /> endocytosis organization plasma membrane <br /> [32,46]. They include homologous proteins, Pil1 Lsp1, <br />                                                                              A. nidulans YpkA does Interact PkhA <br /> which colocalize transmembrane protein Sur7 [32,46]. In                In S. cerevisiae, Pkh1 activates Ypk1 [20]. Thus, step to <br /> S. cerevisiae Pkh1/2-Ypk1/2 signaling pathway regulates                  verify A. nidulans Pkh1 homologue interacts A. <br /> eisosome assembly turnover [47]. Recently, homo-                 nidulans YpkA, BLASTp search A. nidulans genome <br />  <br />  <br />  <br />  <br /> Figure 6.YpkA expression affects endocytosis vacuolar distribution A. nidulans. (A) Conidiospores grown glass-bottom <br /> dishes (Mattek Corporation, USA) 2 ml MM-G+supplements 12 hours 44uC exposed pre-warmed media containing 10 mM FM4-64 <br /> for 2 minutes. The coverslips briefly rinsed fresh media incubated MM-G FM4-64 3 hours allow dye internalization <br /> and vesicle trafficking. The living cells visualized room temperature using fluorescence microscope. (B, C, D) incubation the <br /> FM4-64 10, 30, 60 minutes. Arrows Spitzenko&#194;&#168;rper. Bars, 5 mm. <br /> doi:10.1371/journal.pone.0057630.g006 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               8                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                               Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 7. YpkA expression affects eisosomes distribution A. nidulans. The niiA::ypkA pilA::gfp (A) niiA::ypkA pilB::gfp (B) strains were <br /> grown 16 hours 37uC MM+10 mM sodium nitrate 50 mM ammonium tartrate. Bars, 5 mm. <br /> doi:10.1371/journal.pone.0057630.g007 <br />  <br /> database (http://www.aspgd.org) using S. cerevisiae Pkh1            niiA::ypkA barA1 double mutant showed synthetic lethality <br /> query revealed single ORF significant similarity. The                 phenotype grown presence ammonium tartrate <br /> potential homologue, AN3110 (here named pkhA), predicted               (Figure 9), suggesting genetic interaction YpkA <br /> be 813 amino acid high identity PkhA (8e298; 449.8%               BarA. <br /> identity 69.4% similarity). PkhA well-defined protein <br /> kinase domain (PS50011). The generation A. nidulans pkhA               Transcriptional Profiling Increased Reduced YpkA <br /> null mutant, using vivo S. cerevisiae fusion-based approach (see       Expression <br /> Materials Methods) unable generate primary                       To gain insight genes pathways were <br /> transformant (data shown). Thus conditional mutant pkhA            influenced YpkA, transcriptional profile A. nidulans <br /> was constructed replacing endogenous pkhA promoter               alcA::ypkA strain, cultured ypkA overexpression repres- <br /> the niiA promoter. When pkhA repressed, growing               sion conditions, determined. The alcA::ypkA mutant was <br /> niiA::pkhA mutant strain presence ammonium tartrate,               selected instead niiA::ypkA mutant permitted the <br /> there dramatic ten-fold decrease colony diameter                investigation genes affected ypkA overexpression. <br /> (Figure 8A). These results strongly indicate pkhA            The A. nidulans wild-type alcA::ypkA mutant strains grown <br /> essential A. nidulans gene. A niiA::pkhA alcA::ypkA double mutant        16 24 hours presence 4% glucose (repression) or <br /> constructed. When double mutant grown 4% glucose                  2% glycerol plus 100 mM threonine (overexpression). Under ypkA <br /> plus ammonium tartrate, representing pkhA ypkA repressing                repression conditions, 186 genes differentially modulated in <br /> conditions, radial growth comparable radial growth             time point (85 101 genes regulated, <br /> the alcA::ypkA mutant strain grown conditions                 respectively, p,0.001), ypkA overexpression conditions <br /> (Figure 8B). The alcA::ypkA strain showed radial diameter similar          132 genes differentially modulated time point <br /> to double mutant repressing conditions niiA               (57 75 regulated, respectively, p,0.001) <br /> promoter inducing conditions alcA promoter (Figure 8B).              compared wild-type control. Hierarchical clustering the <br /> The radial diameter double mutant similar                  significant genes sets identified main regulated <br /> growth glucose plus sodium nitrate, representing ypkA                     clusters ypkA repression main down <br /> repression (Figure 8B, second lane) glucose plus ammonium                regulated clusters ypkA overexpression (Supplementary <br /> tartrate, representing pkhA repression (Figure 8B). Taken together,          Figure 1). The microarray data deposited https://www. <br /> these results suggested ypkA gene directly                   ncbi.nlm.nih.gov/geo/query/acc.cgi?acc = GSE39755, the <br /> downstream pkhA epistatic pkhA, rather, ypkA pkhA               significant gene sets presented Supplementary Table S1. <br /> are genetically independent parallel.                                     The differentially expressed genes alcA::ypkA strain, under <br />    Ceramides sphingoid long-chain bases (LCBs) precur-               conditions, classified MIPS functional categories <br /> sors complex sphingolipids play distinct signaling              (http://mips.helmholtz-muenchen.de/proj/funcatDB/ <br /> roles crucial cell growth <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>survival</span> [49]. It shown           search_main_frame.html) [51]. Within glucose repressed or <br /> that A. nidulans ceramide synthases regulate hyphal             threonine induced gene sets single FunCat significantly <br /> morphogenesis them, BarA, unique filamentous                overrepresented compared ratio genome (Fisher&#226;&#8364;&#8482;s test, <br /> fungi [50]. Since YPK1 important sphingolipid-mediated             p,0.01; data shown). Close inspection gene function <br /> signaling pathway yeast [5] interactions                  revealed conserved themes gene sets correlated with <br /> niiA::ypkA barA1 mutants investigated (Figure 9). The                observed phenotypes conditions including <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               9                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                               Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 8. The pkhA gene essential A. nidulans interacts ypkA. (A) The wild-type niiA::pkhA mutant strains grown 72 <br /> hours 37uC MM+sodium nitrate 10 mM MM+ammonium tartrate 50 mM (Wt = Wild-type). (B) The alcA::ypkA, niiA::pkhA, alcA::ypkA <br /> niiA::pkhA strains grown 72 hours 37uC different combinations MM+glucose 2% glycerol 2% plus threonine 100 mM plus sodium <br /> nitrate 10 mM ammonium tartrate 50 mM (Wt = Wild-type). <br /> doi:10.1371/journal.pone.0057630.g008 <br />  <br /> <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation related metabolism uptake, cell membrane             expression including regulation inositol phosphate <br /> and wall biosynthesis, protein modification, membrane trafficking            uptake ypkA repression (AN3781, AN2864), iron (AN0176) <br /> and cell cycle (Supplementary Table S2). However, despite having             sulfur (AN2200, AN1825) ypkA overexpression, while <br /> a similar functional profile majority genes        fatty acid (AN5028, AN7320, AN9408) potassium transport- <br /> data sets distinct (Figure 10).                                         ing ATPase energy metabolism (AN6642) regulated by <br />    Under ypkA repression regulation genes                ypkA repression. <br /> involved carbohydrate metabolism including alcohol                        The influence ypkA impacted protein level <br /> dehydrogenase (AN2860) plays role fermentation               including elevated expression post ypkA overexpression the <br /> phosphogluconate dehydrogenase (AN6135) forms               cyclin (AN9500) interacts phosphorylated by <br /> pentose phosphate shunt. Simultaneously                      Pho85p cyclin-dependent kinase (Cdk), induced Gcn4p level <br /> regulation high affinity glucose sensor RGT2 (AN6669),                transcription, specifically required Gcn4p degradation <br /> sugar transporter HXT17 (AN1865) plus multiple polysaccharide                act sensor cellular capacity protein synthesis <br /> degrading enzymes (AN5727, AN8421, AN0479, AN9365).                          [52]. Genes involved protein glycosylation (repression: AN5888, <br /> Conversely, ypkA overexpression caused regulation                 AN2045) proteolysis (repression: AN6686; overexpression: <br /> pyruvate dehydrogenase kinase (AN10800) inactivates                 AN3932, AN8681) regulated ypkA repression and <br /> pyruvate dehydrogenase complex RGT1 (AN8177) glucose                 overexpression. While regulation genes which <br /> responsive transcription factor, regulation SNF3             transcribe GPI anchored proteins (repression: AN11048, AN5033, <br /> (AN8737) low glucose sensor xylolytic enzymes (AN0942,               AN10790, AN4381: overexpression AN6969) different set of <br /> AN8149). Real-time PCR experiments showed AN6669                    peptidase (repression overexpression: AN6438; overexpres- <br /> AN8737 transcriptionally modulated observed                   sion: AN8445, AN3959, AN9003). <br /> microarray hybridization experiments (Figures 10E&#226;&#8364;&#8220;F). The                       Genes involved sterol biosynthesis cell wall formation <br /> repression ypkA impacted nitrogen status cell             influenced transcriptional level ypkA expression, <br /> with regulation genes involved amino acid synthesis              including glucan modifying, phospholipid metabolism ergos- <br /> (AN1857, AN1858, AN1034) regulation multiple                 terol biosynthesis enzymes. Under ypkA repression and <br /> amino acid permeases (AN2201, AN11211, AN6770, AN2781).                      overexpression greater number genes involved in <br /> Overexpression resulted regulation amino acid              processes regulated (Supplementary Table <br /> biosynthesis genes (AN8608, AN8009, AN3928, AN5200). The                     S2). In addition, genes involved establishment polarized <br /> acquisition nutrients modulated ypkA                    growth membrane-to-vacuole trafficking influenced <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              10                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                                  Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />                                                                                S. cerevisiae Bar1 (not ortholog BarA A. nidulans) [53] which <br />                                                                                mutated causes hypersensitivity alpha factor-induced G1 <br />                                                                                arrest, suggesting repression ypkA activity was <br />                                                                                influencing successful progression cell cycle A. <br />                                                                                nidulans. <br />  <br />                                                                                Discussion <br />                                                                                   The serum- glucocorticoid-inducible kinase (SGK) was <br />                                                                                initially identified screen cDNA library generated from <br />                                                                                mammary tumor cells rats treated glucocorticoids [1]. The <br />                                                                                SGK kinase revealed expressed wide <br />                                                                                variety species. The SGK isoforms exist mammals <br />                                                                                regulate metabolism, transport, transcription enzymatic <br />                                                                                activation diverse range functions epithelial <br />                                                                                transport, excitability, cell proliferation, apoptosis [54]. The S. <br />                                                                                cerevisiae SGK homologues, Ypk1 Ypk2 [6], required for <br />                                                                                cell growth [55], cell wall integrity [38,56], involved in <br />                                                                                endocytosis [16,44], actin polarization [23] sphingolipid <br />                                                                                metabolism [57]. Both Dypk1 Dypk2 deficient strains grew <br />                                                                                slower wild-type strain, double Dypk1 Dypk2 <br />                                                                                mutant viable, germilings undergoing three <br />                                                                                rounds cell division prior arresting growth [55]. <br />                                                                                   The presented study demonstrated A. nidulans single <br />                                                                                ypk1 homologue significant similarity Ypk1 Ypk2p, <br />                                                                                proved essential cell viability. Replacement the <br />                                                                                endogenous ypkA promoter different regulatable <br />                                                                                promoters, alcA niiA, confirmed ypkA essential gene. <br />                                                                                The A. nidulans ypkA conditional mutants showed reduced ypkA <br />                                                                                expression caused decreased radial growth, delayed conidial <br />                                                                                germination, deficient polar axis establishment, intense branching <br />                                                                                late stages growth, lack asexual spores, a <br />                                                                                terminal phenotype. These effects noticeable after <br />                                                                                approximately 16 hours growth, point than <br />                                                                                90% germlings showed increasingly unordered and <br />                                                                                branched growth pattern, defect hyphal elongation. <br />                                                                                   The spatial control cell growth involves cell wall synthesis and <br /> Figure 9. The A. nidulans YpkA interacts BarA. The radial <br /> growth wild-type, niiA::ypkA, barA1, niiA::ypkA barA1 mutant <br />                                                                                organization cytoskeleton. A polarized cytoskeleton is <br /> strains grown 72 hours 37uC MM+sodium nitrate 10 mM             important secretion hyphal apex essential the <br /> or MM+ammonium tartrate 50 mM (Wt = Wild-type).                                establishment maintenance polarized growth [38]. The <br /> doi:10.1371/journal.pone.0057630.g009                                          repression ypkA caused CFW SDS resistance, suggesting <br />                                                                                modification cell wall structure compared wild type <br /> by ypkA. Repression resulted regulation voltage-              strain. FITC-conjugated WGA staining demonstrated the <br /> gated calcium channel CCH1 (AN1168) subunit                       sites cell wall deposition mislocalized absent ypkA <br /> ESCRT I complex (AN2521) involved trafficking vesicles                 repression. The transcriptomic analysis revealed ypkA <br /> the plasma membrane vacuole. Overexpression ypkA caused                  modulates transcription GPI anchored proteins, are <br /> the regulation ARP2/3 complex (AN8698) involved                    maybe involved cell wall formation polarized growth. <br /> in establishment actin cortical patch             The fluorescently labeled YpkA protein distributed through- <br /> the polarized regions cells, causing regulation           cytoplasm partial co-localized microtubules <br /> palmitoyltransferase (AN1907) acts SNARE proteins                    indicating YpkA participate intracellular trafficking <br /> which play role fusion vesicles membrane                  delivery cell membrane and/or wall constituents the <br /> external vacuolar secretion. A polyphosphate kinase (AN12017)               hyphal apex. Low levels YpkA make cell wall other <br /> that plays role vacuolar transport fusion, vacuolar                 precursors distributed erroneously hyphae. <br /> import protein (AN5157) subunit exocyst complex                      The sterol composition animals yeasts influences the <br /> involved secretion (AN6210) regulated ypkA               apical localization proteins. Ergosterol important raw <br /> overexpression.                                                                material new cell membranes low ergosterol concentra- <br />    Cell cycle influenced activity ypkA,           tions stimulate Ypk1 activity, Ypk1 act sensor of <br /> as putative transcription factor (AN0273), required             ergosterol levels coordinating cell wall synthesis budding <br /> septum destruction cytokinesis, cell polarity morpho-               [38,58]. The lipid rafts specialized membrane structures, <br /> genesis, protein involved cell cycle arrest (AN6617)           consisting aggregation sphingolipids ergosterol, that <br /> were regulated ypkA repression. In addition,                 mediate biosynthetic endocytic processes anchoring <br /> regulation arginase (AN6869), mutated S.                   compounds plasma membrane. These domains rich in <br /> cerevisiae results arrest cell growth unbudded G1 phase,          sterol sphingolipids play important role cellular <br /> and regulation endopeptidase (AN6686) similar              processes including addressing proteins, polarity signal <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                11                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                               Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br />  <br />  <br /> Figure 10. The representation MIPS FunCats functionally annotated genes wild-type A. nidulans genome (A) and <br /> the genes differentially expressed (p,0.001) alcA::ypkA strain repression (B) overexpression (C) conditions when <br /> compared wild-type strain. The venn diagram demonstrates limited overlap differentially expressed genes ypkA <br /> repression overexpression conditions (D). Real-time RT-PCR AN6669 AN8737 genes increased decreased mRNA accumulation in <br /> the wild-type alcA::ypkA mutant strains grown glucose 4% threonine 100 mM, respectively, 16 24 hours. <br /> doi:10.1371/journal.pone.0057630.g010 <br />  <br /> transduction [31]. The repression ypkA increased sensitivity           ypkA gene directly downstream pkhA epistatic pkhA. <br /> lovastatin, inhibitor HGM-CoA reductase sterol                     These results strongly indicate A. nidulans, differently from <br /> synthesis, suggesting lower production ergosterol             observed S. cerevisiae, ypkA pkhA genetically <br /> when YpkA reduced. Furthermore, filipin staining showed                  independent parallel. It remains investigated which <br /> that ergosterol-rich membrane domains disorganized                 pathways responsible YpkA activation. <br /> ypkA repression, ypkA shown modulate                      The activity mammalian growth factor-regulated PDK-1 <br /> transcription genes involved sterol biosynthesis. Collectively,        induced increased level phosphatidylinositol. Apart <br /> these results indicate YpkA involved proper synthesis         SGK, PDK1 phosphorylates kinases including <br /> and localization lipid rafts A. nidulans.                              PKB/Akt, p70 PKC isoforms. The activation PKB by <br />    In mammals, SGK activated phosphatidylinositol                  PDK1 occurs presence vesicles containing phosphatidylino- <br /> dependent kinase-1 (PDK-1), responds increased levels            sitol tri-phosphate, results interaction second <br /> phosphatidylinositol. The S. cerevisiae homologues Pdk1 Pdk2             messenger PKB PH domains. The presented transcrip- <br /> are essential involved activation ypk1            tome analysis ypkA repressed A. nidulans demonstrated the <br /> ypk2. Sphingolipids stimulate Pkh1 auto&#226;&#8364;&#8220;phosphorylation,               increased expression genes regulate uptake inositol <br /> Pkh1 required Ypk1 phosphorylation [6,9]. The pkh1 mutant             phosphate, possibly representing attempt compensate <br /> showed defects actin polarization restrictive temperatures         lack signaling cascade triggered PkhA-YpkA <br /> an inability initiate endocytic processes [9]. Phytosphingo-          cascade. <br /> sine (PHS) generation depends concentration phytocer-                 Besides internalization <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>, endocytosis allows <br /> amides plasma membrane. Thus, Pkh1-Ypk1 signaling cascade                 eukaryotic cells recycle components plasma membrane, <br /> represents feed-back mechanism coordinates membrane                  regulating expression cell signaling receptors maintain- <br /> growth cell wall expansion [56]. In S. cerevisiae, Ypk1                  ing proper supply membrane precursors removal of <br /> phosphorylation increased presence PHS [5]. The A.                  excess material supporting hyphal growth [45]. The vesicle <br /> nidulans Pkh1 homologue pkhA essential gene. In               supply center hyphal apex, called Spitzenko&#194;&#168;rper, is <br /> initial attempt verify PkhA YpkA interacting A.            essential filamentous growth determination polarization, <br /> nidulans, double mutant niiA::pkhA alcA::ypkA constructed          performing role endo- exocytosis [59]. In S. cerevisiae, <br /> both genes individually simultaneously induced                    Pkh&#226;&#8364;&#8220;Ypk1 kinase cascade important regulatory <br /> repressed. When pkhA, ypkA genes repressed,                 component endocytic machinery [44]. In presented <br /> mutant displayed similar growth phenotype, suggesting             study, FM4-64 staining used monitor the vacuolar <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              12                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                               Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> system endomembranes A. nidulans [31]. During ypkA                    Expression ypkA gene control alcA promoter <br /> repression Spitzenko&#194;&#168;rper absent FM4-64 uptake               regulated carbon source: repression glucose 4% w/v, <br /> delayed. Additionally, transcriptomic analysis ypkA repres-           derepression glycerol, induction ethanol threonine. <br /> sion revealed modulation genes involved vacuolar                 Therefore, MM-G MM-E identical MM, that <br /> trafficking. Accordingly, A. nidulans YpkA required               glycerol (2% v/v) and/or ethanol (2% v/v) threonine (100 mM) <br /> proper function endocytic machinery. Furthermore, S.               used, respectively, place glucose sole carbon <br /> cerevisiae Ypk1 affects fatty-acid uptake energy homeo-             source. Trace elements, vitamins, nitrate salts included <br /> stasis regulating endocytosis [34]. During ypkA repression A.         described [63]. Strains grown 37uC unless indicated <br /> nidulans, genes involved fatty acid potassium transporting            otherwise. Expression ypkA gene control niiA <br /> ATPase energy metabolism regulated suggesting                 promoter regulated nitrate source: repression a <br /> YpkA affects uptake fatty acids.                                      modified minimal medium (AMM) (1% w/v glucose, 2% w/v <br />    BarA A. nidulans homologue S. cerevisiae Lag1 acyl-         agar) plus ammonium tartarate 50 mM induction AMM <br /> CoA-dependent ceramide synthase, catalyzes conden-                 plus sodium nitrate 10 mM. <br /> sation phytosphingosine fatty acyl-CoA form <br /> phytoceramide. The barA null mutant showed heat sensitivity,               Molecular Techniques <br /> delay emergence germ tubes fails maintain stable             Standard genetic techniques A. nidulans used all <br /> axis polarity absence filipin staining hyphal tips [50].        strain constructions genetic transformation [63,64]. DNA <br /> These phenotypic defects similar observed                 manipulations performed according [65]. All PCR <br /> sphingolipid biosynthesis blocked [60]. This appeared similar            reactions performed using Platinum Taq DNA Polymerase <br /> to phenotype observed ypkA repression,                 High Fidelity (Invitrogen). The primers used work listed <br /> absence barA, ypkA repression lethal cell viability.           Supplementary Table S4. For construction alcA::ypkA <br /> Therefore appears genetic interaction ypkA,            strain, ypkA encoding region cloned pMCB17apx <br /> barA, sphingolipid synthesis. Ceramides sphingoid                vector downstream alcA promoter. This construction was <br /> long-chain bases (LCBs) precursors complex                       transformed A. nidulans replace endogenous ypkA <br /> sphingolipids play distinct signaling roles crucial cell             promoter yielding strain alcA::ypkA. The transformants were <br /> growth survival [49]. LCBs induce cellular responses                 confirmed PCR. The DypkA deletion cassette constructed <br /> during heat stress [61]. In S. cerevisiae, Ypk1 activates          amplifying 1.5-kb region ORF. The A. <br /> enzyme sphingolipid biosynthesis pathway                  fumigatus pyroA gene, localized center cassette, <br /> control Orm protein activity [21]. Recent studies reported            acted selectable marker enabling transformed cells to <br /> that Ypk1 regulates sphingolipid homeostasis heat stress          grow media addition pyridoxine amplified <br /> through Orm protein phosphorylation [62]. A similar mechanism                A.fumigatus CEA17 strain. Transformants confirmed by <br /> may occur A. nidulans, low levels YpkA increased                PCR using heterokaryon rescue technique described by <br /> susceptibility cells restrictive temperatures, probably        Osmani et al. [39]. The cassette niiA::ypkA constructed <br /> to absence key elements cellular structure.                    PCR-mediated technique. About 1.5-kb region 59 and <br />    In S. cerevisiae, Ypk1 rapidly degraded nitrogen absent        39-ends ORF selected primer design. The pyrG::niiA <br /> leading decreased levels eIF4G disruption protein           fragment amplified niiA::pikA strain. The transfor- <br /> translation [24]. A. nidulans genes involved protein glycosyl-          mants confirmed Southern blot analysis (Supplementary <br /> ation, proteolysis set peptidases repressed                 Figure S2). <br /> overexpressed ypkA expression modulated. Furthermore,                  The construction YpkA::GFP fusion gene performed <br /> genes involved amino acid transport metabolism                   according Colot et al. [66]. Briefly, 1.5-kb regions of <br /> modulated repressing inducing conditions,                     ypkA ORF selected primer design. The primers <br /> possibly attempt restore normal levels nitrogen              amplified 59-UTR flanking region plus ypkA ORF the <br /> within cell.                                                             39-UTR flanking region. Both fragments PCR amplified <br />    Collectively investigation suggests filamentous          genomic DNA A. nidulans A4 strain. The pyrG gene, <br /> fungus A. nidulans YpkA acts sensor effector protein             used selective marker uridine uracil prototrophy was <br /> coordinating maintenance expansion cell mem-                  amplified pCDA21 plasmid, GFP gene amplified <br /> brane wall. Depending stimulus, YpkA possibly                  pMCB17apx plasmid. The external primers possessed <br /> phosphorylated PkhA. Subsequently, active YpkA regulates              cohesive ends matched pRS426 vector used vivo <br /> the synthesis transport ergosterol, amino phospholipids           recombination S. cerevisiae. The pRS426 vector double <br /> shingolipids, packed lipid rafts, hyphal apex          digested EcoRI BamHI linearization S. <br /> permitting polarized growth.                                                 cerevisiae strain SC9721 transformed DNA <br />                                                                              fragments using lithium acetate method [67]. The DNA the <br /> Materials Methods                                                        S. cerevisiae transformants extracted method described <br />                                                                              Goldman et al. [68], dialysed transformed electropora- <br /> Strains, Media Culture Methods                                           tion Escherichia coli strain DH10B, order rescue the <br />   The A. nidulans strains used described Supplementary                pRS426 plasmid harboring cassettes. <br /> Table S3. The media used basic types, complete                   All cassettes PCR-amplified used transforma- <br /> minimal. The complete media comprised following                    tion A. nidulans according procedure OSMANI et al. <br /> variants: YAG (2% w/v glucose, 0.5% w/v yeast extract, 2% w/v                [64]. Transformants scored ability grow on <br /> agar, trace elements), YUU (YAG supplemented 1.2 g/liter                minimal medium. <br /> [each] uracil uridine), liquid YG YG+UU medium <br /> with composition (but agar). The minimal media              Staining Microscopy <br /> were modified minimal medium (MM; 1% w/v glucose, original                   For germling nuclear septum staining, conidia were <br /> high-nitrate salts, trace elements, 2% w/v agar, pH 6.5).                    inoculated coverslips. After incubation appropriate <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              13                            March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                              Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> conditions experiment, coverslips adherent                 submitted Agilent E-array. Based quality <br /> germlings fixed (3.7% formaldehyde, 50 mm sodium                        parameter implemented Aglilent E-array (such sequences <br /> phosphate buffer pH 7.0, 0.2% Triton X-100) 30 min                    high score cross-hybridization potential the <br /> room temperature, rinsed PBS buffer (140 mm NaCl, 2 mm                  genome sequences appropriate regions be <br /> KCl, 10 mm NaHPO4, 1.8 mm KH2PO4, pH 7.4) incubated                      target represented slides), 11,143 probes were <br /> for 5 min solution 100 ng/ml DAPI (Sigma Chemical,              designed form uploaded sequence A4 strain. These probes <br /> St. Louis) and/or 100 ng/ml calcofluor (fluorescent brightener,           represented three-four times microarray slides the <br /> Sigma Chemical). After incubation dyes, coverslips           annotation based [1,2] used generate annotation file <br /> washed PBS 10 min room temperature finally              used analysis. Therefore, microarray slides comprise <br /> rinsed distilled water mounted microscopical                    45,220 features representing 1,417 Agilent internal controls and <br /> examination. For cell imaging YpkA protein fused GFP,                  800 internal controls represent 80 randomly chosen A. <br /> conidiospores grown glass-bottom dishes (Mattek, Ash-                nidulans ORFs (ten times replicated each). <br /> land, MA) 30uC. Germlings fixed 10 min fixative                The gene expression analysis used work carried out <br /> solution containing 16PBS, 5% DMSO, 3.7% formaldehyde,                   using custom-designed oligonucleotides slides (4644 K micro- <br /> 10% methanol. The nucleus DAPI stained described above.               array) Agilent TechnologiesTM, based A. nidulans genome <br /> For lectin staining, coverslips incubated prewarmed media            annotation publicly available. After RNA isolation purifica- <br /> containing 5 mg/ml FITC-conjugated wheat germ agglutinin                     tion, described above, samples labeled Cy-3 or <br /> (WGA) 5 min fixed, washed, mounted. For                     Cy-5-dUTP using Two-Color Microarray-based gene expres- <br /> lipid raft staining, cells stained filipin (25 mg ml21)        sion analysis (Quick Amp Labeling Kit, Agilent TechnologiesTM, <br /> for 5 min analysed. In order monitor establish-              USA) following manufacturer&#226;&#8364;&#8482;s protocol. Initially, 5mg total <br /> ment polarization, conidia incubated appropriate              RNA incubated AgilentTM RNA Spike-In controls <br /> experimental conditions samples taken 2 hour intervals                probes (RNA Spike A B mix). Prior labeling, synthesis of <br /> over 16 hours. A conidium counted polarized possessed           cDNA carried incubating samples 1.2 mL T7 <br /> a germ tube readily detectable small protuberances                 promoter primer, nuclease-free water appropriate <br /> spherical shape conidium surface.                                         volume. The template primer denatured incubating <br />    Slides viewed Carl Zeiss (Jena, Germany)                      reaction 65uC circulating water bath 10 minutes, <br /> microscope using 1006 magnification oil immersion objective                  reactions placed ice 5 min. To the <br /> (EC Plan-Neofluar, NA 1.3) equipped 100 W HBO                         samples added cDNA Master Mix (4 mL 5X First Strand <br /> mercury lamp. Phase contrast brightfield images                  Buffer, 2 mL 0.1 M DTT, 1 mL 10 mM dNTP mix, 1 mL MMLV- <br /> fluorescent images captured AxioCam camera (Carl                RT 0.5 mL RNaseOut), mixture incubated at <br /> Zeiss), processed using AxioVision software version 3.1              40uC circulating water bath 2 hours. After, samples <br /> saved TIFF files. Further processing performed using                  moved 65uC circulating water bath incubated 15 <br /> Adobe Photoshop 7.0 (Adobe Systems Incorporated, CA).                        minutes. cRNA amplification labeling performed by <br />                                                                              adding samples AgilentTM Transcription Master Mix <br />                                                                              (20 mL 46 Transcription Buffer, 6 mL 0.1 M DTT, 8 mL NTP <br /> RNA Extraction, cDNA Synthesis Real-time PCR <br />                                                                              mix, 6.4 mL 50% PEG, 0.5 mL RNase OUT, 0.6 mL inorganic <br /> Reactions                                                                    pyrophosphatase, 0.8 mL T7 RNA Polymerase, 2.4 mL Cyanine 3- <br />    After harvesting, mycelia disrupted grinding                CTP control samples, cyanine 5-CTP treated samples, and <br /> liquid nitrogen, total RNA extracted using Trizol                    15.3 mL nuclease-free water), incubating mixtures a <br /> (Invitrogen, USA). RNA (10 mg) treatment                       circulating water bath 40uC 2 hours. The labeled cRNA was <br /> fractionated 2.2 M formaldehyde, 1.2% agarose gel, stained                purified using RNeasyH Mini Kit (Qiagen), quantified in <br /> with ethidium bromide, visualized UV-light order              NanoDropH 2000 Thermo Scientific (Uniscience). <br /> check RNA integrity. The samples submitted RNAse-free                   For hybridization, 825 ng labeled cRNA mixed <br /> DNAse treatment previously described [69], purified                  AgilentTM Fragmentation Mix (11 mL 106 Blocking agent, <br /> RNeasyH Mini Kit (Qiagen), quantified                        2.2 mL 25X fragmentation buffer, nuclease-free water bring <br /> NanoDropH 2000 Thermo Scientific (Uniscience).                               volume 52.8 mL), incubated 60uC exactly 30 <br />    All RT-qPCR reactions performed using ABI 7500                minutes fragment RNA. The fragmentation interrupted by <br /> Fast Real-Time PCR System (Applied Biosystems, USA) Taq-                 adding 55 mL 2X GE Hybridization Buffer HI-RPM. Finally, <br /> ManTM Universal PCR Master Mix kit (Applied Biosystems,                      100 mL sample placed microarray slide, <br /> USA). The reactions calculations performed according                mounted AgilentTM Microarray Hybridization <br /> to Semighini et al. [69]. The primers LuxTM fluorescent                  Chamber Kit. The hybridization carried oven <br /> probes (Invitrogen, USA) used work described                  (Agilent G2545A Hybridization Oven) set 65uC 17 hours. <br /> Supplementary Table S4.                                                      After, microarray slides washed according Agilent&#226;&#8364;&#8482;s <br />                                                                              instruction scanned using GenePixH 4000B microarray <br /> Microarray Slides Construction Gene Expression                           scanner (Molecular Devices, USA). <br /> Methods <br />    To construct microarray slides Agilent E-array software           Gene Expression Analysis <br /> tool used (available https://earray.chem.agilent.com/                     The extraction data TIFF images generated through <br /> earray/). Briefly, uploaded gene sequences representing               scanning microarray slides performed using Agilent <br /> whole A. nidulans A4 gene sequences. The ORF number                      Feature Extraction (FE) Software version 9.5.3.1 (Agilent Tech- <br /> carefully validated comparing sequences deposited            nologies, USA), using Linear Lowess algorithm obtain <br /> databanks [CADRE (The Central Aspergillus Resource); AspGD                   background subtracted normalized intensity values. The <br /> (Aspergillus Genome Database) BROAD Institute] aiming                 dye-normalyzed values generated Feature Extraction data <br /> identify validate sequences probe design. 11,251 ORFs            files used upload software Express Converter. (version <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              14                           March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                                                         Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> 2.1, TM4 platform available http://www.tm4.org/utilities.                                 Table S1 Significant gene sets. <br /> html), conveniently converts Agilent file format mev                            (XLSX) <br /> (multi experiment viewer) file format compatible TM4 <br />                                                                                              Table S2 Log2 fold change compared wild-type. <br /> softwares microarray analysis (available http://www.tm4. <br />                                                                                              (XLSX) <br /> org/). The mev files uploaded MIDAS software <br /> (TM4 platform), resulting data averaged                                  Table S4 Strains used work. <br /> replicated genes array, biological replicates                          (DOCX) <br /> each treatment. The generated mev files finally analyzed by <br />                                                                                              Table S4 Primers probes used work. <br /> using TIGR MeV (TM4 platform, Multi Experiment Viewer, <br /> available http://www.tigr.org/software/microarray.shtml),                                 (DOCX) <br /> where differentially expressed genes statistically identified <br /> using one-class t test (p.0.01). Significantly different genes                          Acknowledgments <br /> those mean log2 expression ratio included samples                             We thank Fundac&#194;&#184;a&#203;&#339;o Amparo a` Pesquisa Estado Sa&#203;&#339;o Paulo <br /> was statistically different 0, indicates absence                           (FAPESP) Conselho Nacional Desenvolvimento Cient&#196;&#177;&#194;&#180;fico e <br /> gene modulation.                                                                             Tecnolo&#194;&#180;gico (CNPq), Brazil financial support. We thank Dr. <br />                                                                                              Vicky Sophianopoulou providing PilA::GFP PilB::GFP strains. <br /> Supporting lnformation                                                                       We like thank reviewers suggestions comments. <br />  <br /> Figure S1 The heat map centroid plot                                          Author Contributions <br /> hierarchal clusters genes identified differ- <br /> entially expressed (p,0.001) alcA::ypkA strain                                        Conceived designed experiments: GHG. Performed experi- <br />                                                                                              ments: ACC MS. Analyzed data: GHG MHSG ACC NAB. <br /> under repression (G1-10) overexpression (T1-10) <br />                                                                                              Contributed reagents/materials/analysis tools: ACC NAB GHG. Wrote <br /> conditions compared wild-type strain.                                            paper: GHG ACC NAB. <br /> (PDF) <br />  <br /> References <br />  1. Webster MK, Goya L, Ge Y, Maiyar AC, Firestone GL (1993) Characterization                      yeast differs depending downstream events, including endocytosis, cell <br />     sgk, novel member serine/threonine protein kinase gene family                growth, resistance sphingolipid biosynthesis inhibitor, ISP-1. Arch. <br />     transcriptionally induced glucocorticoids serum. Mol. Cell Biol. 13:                 Biochem. Biophys. 437: 29&#226;&#8364;&#8220;41. <br />     2031&#226;&#8364;&#8220;2040.                                                                               17.   Roelants FM, Baltz AG, Trott AE, Fereres S, Thorner J (2010) A protein <br />  2. Hollister RD, Page KJ, Hyman BT (1997) Distribution messenger                       kinase network regulates function aminophospholipid flippases. Proc. Natl. <br />     RNA extracellularly regulated kinases 1, 2 3 rat brain: effects              Acad. Sci. USA 107: 34&#226;&#8364;&#8220;39. <br />     excitotoxic hippocampal lesions. Neuroscience 79: 1111&#226;&#8364;&#8220;1119.                             18.   Spiegel S, Milstien S (2003) Exogenous intracellularly generated <br />  3. Mayar AC, Huang AJ, Phu PT, Cha HH, Firestone GL (1996) p53                                sphingosine 1-phosphate regulate cellular processes divergent pathways. <br />     stimulates promoter activity SGK serum/glucocorticoid-inducible serine/                 Biochem. Soc. Trans. 31: 1216&#226;&#8364;&#8220;1219. <br />     threonine protein kinase gene rodent mammary epithelial cells. J. Biol. Chem.         19.   Pettus BJ, Chalfant CE, Hannun YA (2002) Ceramide apoptosis: an <br />     271: 12414&#226;&#8364;&#8220;12422.                                                                              overview current perspectives. Biochim. Biophys. Acta. 1585: 114&#226;&#8364;&#8220;125. <br />  4. Kobayashi T, Cohen P (1999) Activation serum- glucocorticoid-                 20.   Liu K, Zhang X, Sumanasekera C, Lester RL, Dickson RC (2005) <br />     regulated protein kinase agonists activate phosphatidylinositide 3-kinase              Signalling functions sphingolipid long-chain bases Saccharomyces cerevisiae. <br />     mediated 3-phosphoinositide-dependent protein kinase-1 (PDK1)                        Biochem. Soc. Trans. 33: 1170&#226;&#8364;&#8220;1173. <br />     PDK2. Biochem. J. 339: 319&#226;&#8364;&#8220;328.                                                          21.   Roelants FM, Breslow DK, Muir A, Weissman JS, Thorner J. (2011) Protein <br />  5. Sun Y, Taniguchi R, Tanoue D, Yamaji T, Takematsu H, Mori K, Fujita T,                         kinase Ypk1 phosphorylates regulatory proteins Orm1 Orm2 control <br />     Kawasaki T, Kozutsumi Y (2000) Sli2 (Ypk1), homologue mammalian                           sphingolipid homeostasis Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA <br />     protein kinase SGK, downstream kinase sphingolipid-mediated                        108: 19222&#226;&#8364;&#8220;19227. <br />     signaling pathway yeast. Mol. Cell Biol. 20: 4411&#226;&#8364;&#8220;4419.                               22.   Newton AC (2010) Protein kinase C: poised signal. Am. J. Physiol. <br />  6. Casamayor A, Morrice NA, Alessi DR (1999) Phosphorylation Ser-241                        Endocrinol. Metab. 298: E395&#226;&#8364;&#8220;402. <br />     essential activity 3-phosphoinositide-dependent protein kinase-1:             23.   Kamada Y, Fujioka Y, Suzuki NN, Inagaki F, Wullschleger S, Loewith R, Hall <br />     identification sites phosphorylation vivo Biochem. J. 342: 287&#226;&#8364;&#8220;292.              MN, Ohsumi Y (2005)Tor2 directly phosphorylates AGC kinase Ypk2 <br />  7. Ohanian J, Ohanian V (2001) Sphingolipids mammalian cell                                regulate actin polarization. Mol. Cell Biol. 25: 7239&#226;&#8364;&#8220;7248. <br />     signalling.Cell Mol. Life Sci. 58: 2053&#226;&#8364;&#8220;2068.                                            24.   Shimobayashi M, Takematsu H, Eiho K, Yamane Y, Kozutsumi Y (2010) <br />  8. Futerman AH, Hannun YA (2004) The complex life simple sphingolipids.                        Identification Ypk1 novel selective substrate nitrogen starvation- <br />     EMBO Rep. 5(8): 777&#226;&#8364;&#8220;82.                                                                        triggered proteolysis requiring autophagy endosomal sorting <br />  9. Friant S, Lombardi R, Schmelzle T, Hall MN, Riezman H (2001)                               complex required transport (ESCRT) machinery components. J. Biol. <br />     Sphingoid <span id='am-320' about='xsp:base' typeof='owl:Thing'>base</span> signaling Pkh kinases required endocytosis yeast                  Chem. 285: 36984&#226;&#8364;&#8220;36994. <br />     EMBO J. 20: 6783&#226;&#8364;&#8220;6792.                                                                   25.   Gelperin D, Horton L, Dechant A, Hensold J, Lemmon SK (2002) Loss ypk1 <br /> 10. Zanolari B, Friant S, Funato K, Sutterlin C, Stevenson BJ, Riezman H                       function causes rapamycin sensitivity, inhibition translation initiation and <br />     (2000) Sphingoid base synthesis requirement endocytosis Saccharomyces                   synthetic lethality 14-3-3-deficient yeast. Genetics 161: 1453&#226;&#8364;&#8220;1464. <br />     cerevisiae. EMBO J. 19: 2824&#226;&#8364;&#8220;2833. <br />                                                                                              26.   Harris SD, Read ND, Roberson RW, Shaw B, Seiler S, Plamann M, Momany <br /> 11. Horvath A, Sutterlin C, Manning-Krieg U, Movva NR, Riezman H (1994)                        M (2005) Polarisome meets spitzenko&#194;&#168;rper: microscopy, genetics, genomics <br />     Ceramide synthesis enhances transport GPI-anchored proteins Golgi                    converge. Eukaryot. Cell 4: 225&#226;&#8364;&#8220;229. <br />     apparatus yeast. EMBO J. 13: 3687&#226;&#8364;&#8220;3695. <br />                                                                                              27.   Gow NA (1994) Growth guidance fungal hypha. Microbiology 140: <br /> 12. Sutterlin C, Doering TL, Schimmoller F, Schroder S, Riezman H (1997) <br />                                                                                                    3193&#226;&#8364;&#8220;3205. <br />     Specific requirements ER Golgi transport GPI-anchored proteins in <br />                                                                                              28.   Govindan B, Novick P (1995) Development cell polarity budding yeast. <br />     yeast.J. Cell Sci. 110: 2703&#226;&#8364;&#8220;2714. <br />                                                                                                    J. Exp. Zool. 273: 401&#226;&#8364;&#8220;424. <br /> 13. Jenkins GM, Hannun YA (2001) Role novo sphingoid base <br />                                                                                              29.   Mouritsen OG, Jorgensen K (1997) Small-scale lipid-membrane structure: <br />     biosynthesis heat-induced transient cell cycle arrest Saccharomyces <br />                                                                                                    simulation versus experiment. Curr. Opin. Struct. Biol. 7: 518&#226;&#8364;&#8220;527. <br />     cerevisiae. J. Biol. Chem. 276: 8574&#226;&#8364;&#8220;8581. <br />                                                                                              30.   Megha OB, London E (2006) Cholesterol precursors stabilize ordinary and <br /> 14. Kanazawa T, Nakamura S, Momoi M, Yamaji T, Takematsu H, Yano H, Sabe <br />                                                                                                    ceramide-rich ordered lipid domains (lipid rafts) different degrees. <br />     H, Yamamoto A, Kawasaki T, KozotsumiI Y (2000) Inhibition of <br />     cytokinesis lipid metabolite, psychosine. J. Cell Biol. 149: 943&#226;&#8364;&#8220;950.                     Implications Bloch hypothesis sterol biosynthesis disorders. J. Biol <br />                                                                                                    Chem. 281: 21903&#226;&#8364;&#8220;21913. <br /> 15. Dickson RC, Nagiec EE, Skrzypek M, Tillman P, Wells GB, Lester RL <br />     (1997) Sphingolipids potential heat stress signals Saccharomyces. J. Biol         31.   Alvarez FJ, Douglas LM, Konopka JB (2007) Sterol-rich plasma membrane <br />     Chem. 272: 30196&#226;&#8364;&#8220;30200.                                                                        domains fungi. Eukaryot. Cell 6: 755&#226;&#8364;&#8220;763. <br /> 16. Tanoue D, Kobayashi T, Sun Y, Fujita T, Takematsu H, Kozutsumi Y                     32.   Walther TC, Brickner JH, Aguilar PS, Bernales S, Pantoja C, Walter P <br />     (2005) The requirement hydrophobic motif phosphorylation Ypk1                    (2006) Eisosomes mark static sites endocytosis. Nature 439: 998&#226;&#8364;&#8220;1003. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                              15                                    March 2013 | Volume 8 | Issue 3 | e57630 <br />                                                                                                                                         Aspergillus Nidulans YPK1 Homologue <br />  <br />  <br /> 33. Pushkareva MYU, Khan WA, Alessenko AV, Sahyoun N, Hannun YA                            53. Sprague GF Jr, Herskowitz I (1981) Control yeast cell type mating <br />     (1992) Sphingosine activation protein kinases Jurkat T cells. In vitro                   type locus. I. Identification control expression a-specific gene <br />     phosphorylation endogenous protein substrates specificity action. J.                 BAR1. J. Mol. Biol. 153: 305&#226;&#8364;&#8220;321. <br />     Biol. Chem. 267: 15246&#226;&#8364;&#8220;15251.                                                              54. Lang F, Bo&#194;&#168;hmer C, Palmada M, Seebohm G, Strutz-Seebohm N, Vallon V. <br /> 34. Jacquier N, Schneiter R (2010) Ypk1, yeast orthologue human serum-                  (2006) (Patho)physiological significance serum- glucocorticoid- <br />     glucocorticoid-induced kinase, required efficient uptake fatty acids.            inducible kinase isoforms. Physiol Rev. 86(4): 1151&#226;&#8364;&#8220;78. <br />     J. Cell Sci. 123: 2218&#226;&#8364;&#8220;2227.                                                               55. Chen P, Lee KS, Levin DE (1993) A pair putative protein kinase genes (YPK1 <br /> 35. Delley PA, Hall MN (1999) Cell wall stress depolarizes cell growth                     YPK2) required cell growth Saccharomyces cerevisiae. Mol Gen Genet. <br />     hyperactivation RHO1. J. Cell Biol. 147: 163&#226;&#8364;&#8220;174.                                           236(2&#226;&#8364;&#8220;3): 443&#226;&#8364;&#8220;7. <br /> 36. Helliwell SB, Schmidt A, Ohya Y, Hall MN. (1998) The Rho1 effector                     56. Roelants FM, Torrance PD, Bezman N, Thorner J (2002) Pkh1 Pkh2 <br />     Pkc1, Bni1, mediates signalling Tor2 actin cytoskeleton.                   differentially phosphorylate activate Ypk1 Ykr2 define protein <br />     Curr. Biol. 8: 1211&#226;&#8364;&#8220;1214.                                                                      kinase modules required maintenance cell wall integrity. Mol Biol Cell. <br /> 37. Inagaki M, Schmelzle T, Yamaguchi K, Irie K, Hall MN, Matsumoto K                          13(9): 3005&#226;&#8364;&#8220;28. <br />     (1999) PDK1 homologs activate Pkc1-mitogen-activated protein kinase <br />                                                                                                57. Aronova S, Wedaman K, Aronov PA, Fontes K, Ramos K, Hammock BD, <br />     pathway yeast. Mol. Cell Biol. 19: 8344&#226;&#8364;&#8220;8352. <br />                                                                                                    Powers T (2008) Regulation ceramide biosynthesis TOR complex 2. Cell <br /> 38. Schmelzle T, Helliwell SB, Hall MN (2002)Yeast protein kinases the <br />                                                                                                    Metab. 7(2): 148&#226;&#8364;&#8220;58. <br />     RHO1 exchange factor TUS1 novel components cell integrity <br />                                                                                                58. Li X, Gianoulis TA, Yip KY, Gerstein M, Snyder M (2010) Extensive vivo <br />     pathway yeast. Mol Cell Biol. 22: 1329&#226;&#8364;&#8220;1339. <br /> 39. Osmani AH, Oakley BR, Osmani SA (2006). Identification analysis                         metabolite-protein interactions revealed large-scale systematic analyses. Cell. <br />     essential Aspergillus nidulans genes using heterokaryon rescue technique. Nat.             12;143(4): 639&#226;&#8364;&#8220;50. <br />     Protoc. 1: 2517&#226;&#8364;&#8220;2526.                                                                      59. Steinberg G (2007) On move: endosomes fungal growth and <br /> 40. Flipphi M, Kocialkowska J, Felenbok B (2002) Characteristics                            pathogenicity. Nat Rev Microbiol. 5(4): 309&#226;&#8364;&#8220;16. <br />     physiological inducers ethanol utilization (alc) pathway Aspergillus             60. Cheng J, Park TS, Fischl AS, Ye XS (2001) Cell cycle progression cell <br />     nidulans. Biochem. J. 15: 25&#226;&#8364;&#8220;31.                                                               polarity require sphingolipid biosynthesis Aspergillus nidulans. Mol Cell Biol. <br /> 41. Punt PJ, Greaves PA, Kuyvenhoven A, van Deutekom JC, Kinghorn JR,                              21(18): 6198&#226;&#8364;&#8220;209. <br />     Pouwels PH, van den Hondel CA (1991) A twin-reporter vector simultaneous               61. Dickson RC, Sumanasekera C, Lester RL (2006) Functions metabolism of <br />     analysis expression signals divergently transcribed, contiguous genes                 sphingolipids Saccharomyces cerevisiae. Prog Lipid Res. 45(6): 447&#226;&#8364;&#8220;65. <br />     filamentous fungi. Gene 104: 119&#226;&#8364;&#8220;122.                                                      62. Sun Y, Miao Y, Yamane Y, Zhang C, Shokat KM, Takematsu H, Kozutsumi Y, <br /> 42. Momany M. (2002) Polarity filamentous fungi: establishment, maintenance                     Drubin DG (2012) Orm protein phosphoregulation mediates transient <br />     new axes. Curr Opin Microbiol. 5(6): 580&#226;&#8364;&#8220;5.                                                sphingolipid biosynthesis response heat stress Pkh-Ypk Cdc55- <br /> 43. Ghannoum MA, Rice LB. (1999) Antifungal agents: mode action,                                PP2A pathways. Mol Biol Cell. 23(12): 2388&#226;&#8364;&#8220;98. <br />     mechanisms resistance, correlation mechanisms bacterial               63. Kafer E (1977) Meiotic mitotic recombination Aspergilllus its <br />     resistance. Clin Microbiol Rev. 12(4): 501&#226;&#8364;&#8220;17.                                                 chromosomal aberrations. Adv. Genet. 19: 33&#226;&#8364;&#8220;131. <br /> 44. Hart AK, Schnell JD, Allen DA, Hicke L (2002) The conserved Pkh-Ypk                     64. Osmani SA, May GS, Morris NR. (1987) Regulation mRNA levels of <br />     kinase cascade required endocytosis yeast. J. Cell Biol. 156: 241&#226;&#8364;&#8220;248.               nimA, gene required G2-M transition Aspergillus nidulans. J Cell Biol. <br /> 45. Pen&#203;&#339;alva MA (2005) Tracing endocytic pathway Aspergillus nidulans                  104(6): 1495&#226;&#8364;&#8220;504. <br />     FM4&#226;&#8364;&#8220;64. Fungal Genet Biol. 42(12): 963&#226;&#8364;&#8220;75.                                                 65. Sambrook J Russel DW (2001) Molecular Cloning: A Laboratory Manual, <br /> 46. Olivera-Couto A, Aguilar PS (2012) Eisosomes plasma membrane                           3rd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press. <br />     organization. Mol. Genet. Genomics 287: 607&#226;&#8364;&#8220;620.                                           66. Colot HV, Park G, Turner GE, Ringelberg C, Crew CM, Litvinkova L, Weiss <br /> 47. Luo G, Gruhler A, Liu Y, Jensen ON, Dickson RC (2008) The sphingolipid                         RL, Borkovich KA, Dunlap JC. (2006) A high-throughput gene <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span> <br />     long-chain base-Pkh1/2-Ypk1/2 signaling pathway regulates eisosome assembly                    procedure Neurospora reveals functions multiple transcription factors. Proc <br />     turnover. J. Biol. Chem. 283: 10433&#226;&#8364;&#8220;10444.                                                 Natl Acad Sci U S A. 5;103(27): 10352&#226;&#8364;&#8220;7. <br /> 48. Vangelatos I, Roumelioti K, Gournas C, Suarez T, Scazzocchio C,                            67. Schiestl RH, Gietz RD (1989) High efficiency transformation intact yeast <br />     Sophianopoulou V (2010) Eisosome organization filamentous ascomycete                    cells using single stranded nucleic acids carrier. Curr. Genet. 16: 339&#226;&#8364;&#8220;346. <br />     Aspergillus nidulans. Eukaryot. Cell 9: 1441&#226;&#8364;&#8220;1454. <br />                                                                                                68. Goldman GH, dos Reis Marques E, Duarte Ribeiro DC, Souza Bernardes <br /> 49. Mullen TD, Hannun YA, Obeid LM (2012) Ceramide synthases the <br />                                                                                                    LA, Quiapin AC, Vitorelli PM, Savoldi M, Semighini CP, Oliveira RC, <br />     centre sphingolipid metabolism biology. Biochem. J. 441: 789&#226;&#8364;&#8220;802. <br />                                                                                                    Nunes LR, Travassos LR, Puccia R, Batista WL, Ferreira LE, Moreira JC, <br /> 50. Li S, Du L, Yuen G, Harris SD (2006) Distinct ceramide synthases regulate <br />                                                 <br /> </body></html>