<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAUR ANID CELLULAR BIOLOGY, Jan. 1993, p. 341-350                                                                 Vol. 13, No. 1 <br /> 0270-7306/93/010341-10$02.00/0 <br /> Copyright &#194;&#169; 1993, American Society Microbiology <br />  <br />  <br />         An Essential Yeast Gene Homology Exonuclease- <br />                Encoding XRNJ/KEM1 Gene Also Encodes a <br />                    Protein Exoribonuclease Activity <br />          MARGARET KENNA,1t AUDREY STEVENS,2 MARK McCAMMON,3 AD MICHAEL G. DOUGLAS`* <br />           Department Biochemistry Biophysics, University North Carolina Medical School, University of <br />                 North Carolina Chapel Hill, Chapel Hill, North Carolina 27599-72601; Biology Division, <br />                   Oak Ridge National Laboratory, Oak Ridge, Tennessee 37831-80772; Department <br />                          Biochemistry, Southwestern Graduate School Biomedical Sciences, <br />                              University Texas Health Science Center, Dallas, Texas 752353 <br />                         Received 2 July 1992/Returned modification 18 August 1992/Accepted 12 October 1992 <br />  <br />               An essential gene, designated HKE1/RATI, isolated yeast Saccharomyces cerevisiae and <br />            characterized. The gene encodes protein 116 kDa (p116) significant homology yeast gene <br />            (XRNI/KEMI) encoding related protein (p175) 5&apos;--&apos;3&apos; exonuclease activity activities involving <br />            chromosomal DNA pairing mechanics. Preliminary analysis hkel) mutant reveals precipitous <br />            decline translation mRNA nonpermissive temperature. Sporulation heterozygous <br />            HKE1/hkel::URA3 diploids reveals gene, unlike highly related XRN1/KEM1 gene, essential for <br />            cell viability. Overexpression <span id='am-163' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> product, p175, failed rescue cells lacking functional <br />            p116. In vitro studies demonstrate p116 protein 5&apos;- 3&apos; exoribonuclease activity, major activity <br />            related p175. An immunoreactive RNase activity 116 kDa abolished antiserum p116. <br />            Both level protein RNase activity correlate HKEI gene dosage. The RNase activity <br />            purifies coincidentally previously described 116-kDa RNase having 5&apos;- 3&apos; exoribonuclease activity. <br />  <br />    We characterized temperature-sensitive (ts) mutant              In order understand activities operate <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Saccharomyces cerevisiae obtained                    ensure correct processing transport RNA, two <br /> screen mutants reduced synthesis binding             studies initiated yeasts advantage of <br /> hybrid protein competes transport protein              genetic approaches available (3, 16). One these <br /> precursors mitochondria (8). Analysis mutant           studies, designed uncover RNA trafficking mutants in <br /> led characterization gene significant primary          Saccharomyces cerevisiae, yielded temperature-sensi- <br /> sequence homology previously identified gene, XRNJ                 tive mutant complemented gene identical to <br /> (25). XRNJ independently identified KEMI (18),              HKE1 (named RATl RNA trafficking [3]). In study, <br /> SEP1 (42), DST2 (10), RARS (19). The gene referred            poly(A) RNA localization methods used screen for <br /> as XRNI/KEMI paper. It encodes protein 175                 conditional mutants appear blocked the <br /> kDa (pl75) appears multitude properties,             transport RNA molecules nucleus restric- <br /> including involvement seemingly diverse functions               tive temperature. Mutants obtained screens (3, <br /> as recombination, RNA processing turnover, microtu-                 16) appear distinct previously described mutants <br /> bule function, karyogamy, DNA replication (for sum-               exhibit defects cytoplasmic mRNA production to <br /> mary, reference 17). The protein 5&apos;--3&apos; exoribonu-              mRNA splicing (26; review, reference 32). <br /> clease DNase activity DNA strand-exchange                  The studies presented characteriza- <br /> activity (10, 25, 42). XRN1IKEM1 gene disruptions                  tion HKEJ/RAT1 gene hkeltS mutant show <br /> shown nonessential gene. The related gene                  pl16 protein having 5&apos;-33&apos; exoribonuclease activity, <br /> described report originally designated HKE1 (for            major activity product related XRNJ/KEMI <br /> homology KEMI). Yeast cells lacking functional HKE1                  gene. <br /> are viable rescued XRN1/KEMJ, even <br /> in high copy. Analysis hkel temperature-sensitive <br /> mutant shows reduction translation RNA                             MATERIALS AND METHODS <br /> nonpermissive temperature. Furthermore, p116 appears to <br /> have 5&apos;-.3&apos; exoribonuclease activity, does related                 Strains media. The following yeast strains used <br /> p175 protein. The nuclear immunolocalization p116 sug-               study: SEY2101 (MATa ura3-52 leu2-3,112 suc2-A9 <br /> gests participation utilization mRNA             ade2-1 gal2); SEY2102 (AL4Ta ura3-52 leu2-3,112 suc2-A9 <br /> occur prior translation mRNA cytoplasm.                his4-519 gal2) (11); GS8-44 (AL4Ta ura3-52 leu2-3,112 <br /> These phenotypes associated hkel-l&apos;S indicate                suc2-A9 his4-519 gal2 hkel-l&apos;&apos;); W303 (ALATa/a ura3-52 <br /> role protein processing and/or trafficking           leu2-3,112 his3-11,15 trpl-1 ade2-1); YPH500 (MATa <br /> nuclear mRNAs.                                                          ura3-52 lys2-801 ade2-101 trpl-A63 his3-A200 leu-Al); and <br />                                                                         YPH500xmnl (MATa ura3-52 lys2-801 ade2-101 trpl-A63 <br />                                                                         his3-A200 leu-Al xrn1ABglII::URA3) (25). <br />   * <br />       Corresponding author.                                               Yeast cells grown YPD rich medium (1% yeast <br />   t Present address: Howard Hughes Medical Institute, Duke Uni-         extract, 2% peptone, 2% glucose) minimal medium <br /> versity Medical Center, Durham, NC 27713.                               (YNBD [0.67% yeast nitrogen <span id='am-166' about='xsp:base' typeof='owl:Thing'>base</span>, 2% glucose] YNBG <br />                                                                   341 <br />  342     KENNA ET AL.                                                                                          MOL. CELL. BIOL. <br />  <br /> [0.67% yeast nitrogen base, 2% glycerol]) supplemented        plasmids shared common region 4.1 kb which <br /> appropriate amino acids (34). LSM used labeling            contained full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> gene. <br /> experiments (8).                                                      Complementation HKEI centromere plasmid. A <br />    The Escherichia coli strain used plasmid construction       5.1-kb fragment bearing HKEI gene 5&apos; 3&apos; <br /> was MC1066 (F- AlacX74 galU galK rpsL hsdR trpC9830                flanking DNA amplified polymerase chain reaction <br /> leuB600 pyrF74::TnS) (29). Bacteria grown LB               (PCR) primers flanking BamHI site YEp13 the <br /> medium (30). Ampicillin-resistant transformants se-           plasmid pMKHKE1. This insert cloned the <br /> lected LB plates supplemented 50 ,ug ampicillin         BamHI site pRS316, CEN6 plasmid (35), trans- <br /> per ml (30).                                                       formed GS8-44 derivatives. The ability the <br />    Plasmids. The plasmid, pMKHKE1, 2,um LEU2                gene complement ts <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> analyzed grow- <br /> YEp13-based plasmid containing 5.1-kb insert, including          ing cells containing plasmid 23 37&#194;&#176;C. <br /> the entire HKEJIRATI gene, BamHI site (see below).             Integrative mapping. Integrative mapping used to <br /> The plasmid pExo2, YEp24-based plasmid Larimer              demonstrate gene cloned complementation the <br /> and Stevens (25), contains XRNI gene. An -8-kb                 ts mutant mapped genetic location allele <br /> BamHI-HindIII fragment containing entire XRNJ gene             responsible temperature-sensitive <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. A <br /> was subcloned YEpl3 create YEp13XRN1.                      2.3-kb SalI fragment YEp44-7, comple- <br />    Isolation temperature-sensitive mutant. The original     menting plasmids, subcloned yeast integrating <br /> temperature-sensitive strain GS8-44 (hkel-1) identified        vector YIp5. Integration plasmid directed the <br /> as ts mutant screen yeast cells designed detect     HKEJ region genome transforming wild-type strain <br /> the mislocalization fusion protein vivo. Earlier studies   SEY2101 KpnI-linearized DNA. Correct integration at <br /> had demonstrated mitochondrial association              HKEI locus confirmed Southern analysis of <br /> ATP2-lacZ-encoded hybrid protein mitochondrial <br /> inner membrane prevents growth nonfermentable carbon            genomic DNA prepared transformant. This trans- <br /> sources wild-type host (8, 11). A genetic          formed strain mated GS8-44 containing the <br /> screen devised select conditional mutants        temperature-sensitive hkel-1 mutation. Diploids sporu- <br /> suppress no-growth phenotype hybrid protein        lated, Ura+ phenotype meiotically mapped in <br /> still harbor plasmid, assayed growth glycerol     relation hkel-1 allele. Tetrad analysis confirmed that <br /> and 13-galactosidase activity.                                     cloned sequence represents wild-type gene the <br />    Mutagenesis genetic analysis. Strains SEY2101           mutant allele hkel-1 strain. <br /> SEY2102, carrying plasmid pCIZ1 (the ATP2-lacZ                   Yeast transformation. Yeast cells competent <br /> plasmid [8]), mutagenized ethyl methane           transformation lithium acetate (14). <br /> sulfonate (34) UV irradiation. For UV mutagenesis, cells           Nucleic acid methods. Standard procedures used for <br /> were treated cell density 107/ml H20            restriction digestions plasmid constructions. Restriction <br /> approximately 40% killing. After mutagenesis, cells           enzymes T4 DNA ligase used according the <br /> plated YNBG plus 5-bromo-4-chloro-3-indolyl-o-D-ga-           instructions supplier. DNA fragments purified <br /> lactopyranoside amino acid supplements 0.1%               agarose, DNA transformation E. coli was <br /> glucose. Colonies growing plates 7 10 days       performed described Maniatis et al. (27). <br /> at room temperature expressed observable 3-galacto-             The HKE1 gene sequenced single-stranded <br /> sidase activity (30) selected study (approx-      double-stranded template. Dideoxy sequencing DNA <br /> imately 10&apos; survivors). Candidates subjected            accomplished using kit New England Biolabs, <br /> multiple rounds screening checked temper-          single-stranded DNA cloning vectors, universal primers. <br /> ature-sensitive growth 30 37&#194;&#176;C glucose-         Double-stranded plasmid DNA sequenced either <br /> glycerol-containing media. This screening eliminated ap-           Sequenase (United States Biochemical Corp.) double- <br /> proximately 85% candidates. To confirm             stranded DNA cycling sequencing (Bethesda Re- <br /> lesion chromosome linked, original pC,Z1               search Laboratories) using Taq DNA polymerase. The se- <br /> segregated screening uracil prototrophy loss        quencing vector Bluescript (Stratagene) used to <br /> of 0-galactosidase expression. Strains retransformed          subclone certain restriction fragments HKE1, se- <br /> with fresh pCIZ1 screened growth YNBG.                  quence obtained universal sequencing primers. <br /> Approximately 60% final-found candidates lost           Oligonucleotides synthesized used primers to <br /> ability grow YNBG retransformation                 complete regions. DNA data analysis performed <br /> pCIZ1.                                                             using MacVector (IBI) program. Protein sequence <br />    Complementation analysis SEY2101-                   alignment sequences NBRF GenBank data <br /> SEY2102-derived mutants backcrosses followed stan-             bases performed using TFASTA program within <br /> dard genetic protocols (34), did tetrad analysis            University Wisconsin Genetics Computer Group <br /> random spore analysis select haploids using ade2-1       program (7). The DNA sequence protein sequence <br /> allele. GS8-44 (hkel-l&apos;s) strain chosen genetic    alignment identical previously presented (3). <br /> and molecular analyses described report. Other                Disruption HKEI. A 5.1-kb fragment bearing HKEI <br /> mutants described elsewhere.                               gene 5&apos; 3&apos; flanking DNA amplified PCR with <br />     Cloning HKEI/RATI gene. Strain GS8-44 (hkel-l&apos;s)       primers flanking BamHI site YEpl3, plasmid <br /> transformed yeast Sau3A genomic library inserted            pMKHKE1. Additional BamHI sites created novo <br /> into YEp13. Cells plated media selective trans-        PCR method used convenient subcloning of <br /> formants room temperature 1 day shifting          entire genomic fragment containing HKE1 single <br /> 37&#194;&#176;C. After 2 days 37&#194;&#176;C, fast-growing colonies             BamHI fragment. The 5.1-kb BamHI fragment cloned <br /> isolated rescreened ability complement            BglII site polylinker reglon pSP71 <br /> hkel-1. Four independent plasmids identified        (Promega) create pHKE71. pHKE71 digested with <br />  19,000 Leu+ transformants. The insert DNA            SalI-SacII remove 430-bp internal fragment near 5&apos; <br />  VOL. 13, 1993                                                                   AN ESSENTIAL YEAST EXORIBONUCLEASE                  343 <br />  <br />                                  ll.7 kb                                  lows: fusion protein obtained E. coli extracts, <br />                                                                           electrophoresed preparative sodium dodecyl sulfate <br />                                                                           (SDS)-polyacrylamide gel electrophoresis, transferred <br />                                                                           nitrocellulose. The nitrocellulose stained Ponceau <br />           RI H S S <br />                      \\\\\\\k\\ <br />                   I ~~~~~ si <br />                               ~II                         RI <br />                                                                II         S visualize protein, strip containing fusion <br />                                                                           protein excised. Crude antisera p116 incubated <br />                                                                           strip following standard Western blot procedures. <br />                                                                           The bound antibodies eluted nitrocellulose by <br />                                                                           treatment 0.2 M glycine-1 mM EGTA [ethylene glycol- <br />                                                                           bis(,-aminoethyl ether)-N,N,N&apos;,N&apos;-tetraacetic acid] (pH <br />                                                                           2.7) neutralized 3 N NaOH. Immunopurified anti- <br />                                                                           sera tested Western analysis immunoreactivity <br />               /      &lt;0          3.8 kb\                                  p116. <br />                           hisG- -URA3- -hisG               I                 Indirect immunofluorescence. Anti-trpE::p116 antibodies <br />                                                                           used immunofluorescence described Adams <br />           I                                         :I         I          Pringle (1). To visualize nuclear DNA, cells stained <br />        S Rl B/Bg                                   B/Bg    X        Ss2   0.5 ,ug 4,6-diamino-2-phenylindole (DAPI) ml. <br />                                     5-FOA                                 Treated cells mounted glass slides 1 mg of <br />                                                                           p-phenylenediamine ml 90% glycerol. Cells were <br />                                                                           examined photographed using fluorescence micros- <br />                                  hisG                                     copy Nikon Optiphot Microscope fitted UV, <br />                                                                           fluorescein isothiocyanate, rhodamine filters and <br />                   S RI    B/Bg             B/Bg X Ss2 <br />                                                                           Nomarski (DIC) optics. Kodak T-Max 400 35-mm film was <br />                                                                           used photography. NOPJ antibody gift from <br />    FIG. 1. HKEI/RATI essential gene. DNA constructions           John Aris (4). <br /> disrupt HKEIIRATI. A 3.8-kb consists yeast URA3 gene             Protein synthesis. Cells grown overnight log phase <br /> flanked direct repeats Salmonella hisG DNA (2) used          23&#194;&#176;C YPD. Cells harvested, washed twice with <br /> disrupt HKEI/RATI insertion SalI-SacII sites       water, resuspended 107 cells ml low-sulfate medium <br /> 5&apos; region gene (restriction sites used follows: B/Bg,      (LSM [see above]), incubated 30 min 23&#194;&#176;C prior to <br /> BamHI-BglII; H, HindIII; Rl, EcoRI; S, Sall; Ssl, Sacl; Ss2, <br /> SacII; X, XbaI) described in Materials Methods.      zero-time labeling point. At time zero, cells split <br /> A linear fragment containing construct used              half culture shifted 37&#194;&#176;C. For each <br /> transform diploid strain W303. Tetrad analysis HKEI/           labeling point, 1 ml culture removed labeled 5 <br /> hkel::URA3 diploid revealed segregation <span id='am-167' about='xsp:pattern' typeof='owl:Thing'>pattern</span> 2 viable             min 23&#194;&#176;C 5 ,uCi Na235SO4. Labeling termi- <br /> spores:2 inviable spores. Treatment Ura+ transformants            nated addition 1 ml 5% trichloroacetic acid plus 1% <br /> 5-FOA selects cells excised URA3 gene        Casamino Acids. Samples collected counted by <br /> recombination direct repeats hisG gene         using GFA glass filters. <br /> therefore ura3 mutants maintain hkel::hisG gene disruption.          Protein manipulations. Yeast cell extracts were <br /> Tetrad analysis diploid gave rise 2 live:2 dead spores.   prepared glass bead lysis radioimmunoprecipitation <br />                                                                           assay buffer (150 mM NaCl, 1% Nonidet P-40, 0.5% sodium <br />                                                                           deoxycholate, 0.1% SDS, 50 mM Tris-HCl [pH 8.0]) plus <br /> end HKEI. The linear -7-kb fragment isolated                   protease inhibitors (13). Total insoluble E. coli proteins <br /> used recipient input DNA described below.                  prepared method Koerner et al. (20). Proteins <br />   A 3.8-kb fragment consists yeast URA3 gene                  RNase-containing fractions precipitated tri- <br /> flanked direct repeats Salmonella hisG DNA (2)                  chloroacetic acid 5%. Proteins separated electro- <br /> subcloned polylinker region Bluescript vector                 phoretically 7.5 10% gels method Laemmli <br /> (Stratagene). A Sall-SacII fragment isolated                (23) transferred nitrocellulose using Bio- <br /> construct subcloned SalI-SacII site               Rad semidry transfer apparatus (13). For immunoblotting of <br /> HKEI gene pHKE71. The resulting plasmid, pHKE71::                      hydroxylapatite column fractions, nitrocellulose membranes <br /> hisGURA43, digested XhoI AatII release                 blocked 1 h 5% instant milk TTBS (20 mM Tris <br /> -8-kb fragment useful integrative transformation             [pH 7.5], 500 mM NaCl, 0.05% Tween 20) washed in <br /> diploid yeast strain W303 (31). Ura+ transformants                   TIBS. Washed blots incubated 1 2 h the <br /> selected confirmed Southern analysis. Individual                   appropriately diluted antiserum. Blots washed <br /> transformants subsequently used detect excision                   reacted anti-rabbit secondary antiserum <br /> events lead Ura- phenotype                       (horseradish peroxidase conjugated), immunoreactive <br /> selected 5-fluoro-orotic acid (5-FOA) (5). Figure 1                bands detected enhanced chemiluminescence a <br /> depicts disrupted HKE1 gene.                                          kit Amersham. Immunoblotting heparin-agarose <br />   Antibody production. An in-frame gene fusion                column fractions carried crude p116 antisera as <br /> E. coli trpE gene HKEI/RATI ligating                   described Larimer Stevens (25) using &quot;2I-protein <br /> internal HindIII 706-bp fragment HindIII site                 A (ICN). <br /> pATH1l (20). The resulting plasmid coded fusion                        Exoribonuclease analysis. Enzymatic assay exoribonu- <br /> protein consisting amino acids 168 403 p116 fused              clease activity [3H]poly(A) substrate carried <br /> to 324 amino acids trpE protein, creating              described previously (36, 41) <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> activity, <br /> hybrid protein approximately 66 kDa. Rabbit polyclonal                 previously defined (41), needed release 5 <br /> antibodies generated gel-purified trpE fusion                 nmol l H]AMP 150 nmol [3H]poly(A) 30 min <br /> protein. For immunofluorescence Western immunoblot                    37&#194;&#176;C. Hydroxylapatite column fractions assayed <br /> applications, crude sera immunopurified fol-                 addition 20 mM EDTA reaction mixtures, <br />  344     KENNA ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br /> and values obtained subtracted plus Mg2+                                1      2       3         1      2       3 <br /> values. The Mg2 -independent activity 10% of <br /> the Mg2+-dependent activity cases. Cells grown <br /> in 1 liter minimal medium A650 1.7 1.9 and <br /> harvested. Crude extracts prepared yeast            WT <br /> parental strain grown 2pm plasmid <br /> harboring HKEI (PMKHKE1) procedure described in <br /> Fig. 1 Larimer Stevens (25). The parental strain       mutant <br /> the hkel&apos;S mutant grown similarly. Extracts (1 ml) were <br /> resolved hydroxylapatite (Bio-Rad) chromatography              mutant + pl 16 <br /> (2.7-ml columns) exactly previously described (25). A <br /> 40-ml linear gradient (50 500 mM potassium phosphate <br /> buffer [pH 7.6]) used, 1-ml fractions collected, 15      mutant + p175 <br /> pl fraction assayed, results were <br /> plotted graphically units activity versus fraction num- <br /> ber (see Fig. 5). The heparin-agarose chromatography was <br /> carried described legend Fig. 6.                                         230C                  370C <br />                                                                    FIG. 2. XRNJ/KEMl fails rescue temperature sensitivity <br />                            RESULTS                               phenotype hkel-1. hkel-I temperature sensitive growth <br />                                                                  reversed presence XRN1 gene 2,m <br />    Cloning sequence analysis HKEI gene. A genetic     plasmid (YEpl3XRN1). Shown duplicate plates (23 37&#194;&#176;C) of <br /> screen select temperature-sensitive (ts) mutants ex-     serial dilutions (columns: 1, -103 cells; 2, -102 cells; 3, -101 cells) <br /> hibited altered transport mitochondrial protein precursors    wild-type (WT) mutant strains indicated mutant (hkel&apos;s), <br /> yielded mutants falling different complementa-      mutant + p116 (hkel&apos;s transformed HKEl/RATl 2,um <br /> tion groups, defective mitochondrial        plasmid, pMKHKEl), mutant + p175 (XRNIKEM1 2tm <br /> translocation. These studies presented elsewhere. In     plasmid, pExo2). <br /> the course study, however, arrived mutant <br /> in temperature sensitivity phenotype segregated <br /> independently mitochondrial defect. We cloned           HKEI/RA4T1 essential gene rescued by <br /> this temperature-sensitive gene showed essen-      multicopy XRNI/KEMI. A URA3 gene flanked hisG <br /> tial gene preliminary sequence analysis provoked        repeats (see Materials Methods) (Fig. 1) inserted <br /> interest pursuing characterization (see     HKE1 integrated genome generate a <br /> Discussion). In report, molecular char-     heterozygous HKElIhkel::hisG diploid. The correct physi- <br /> acterization gene biochemical analysis    cal insertion integration confirmed Southern <br /> gene product.                                                    analysis (data shown). Sporulation tetrad analysis of <br />    To characterize temperature-sensitive mutant,     diploid revealed spores harboring hkel::hisG <br /> the recessive ts allele genetically complemented         disruption viable. Microscopic examination the <br /> shown allele responsible temperature-          disruptive spores revealed early block cell division. <br /> sensitive growth integrative mapping (see Materiats       The spores did germinate arrested growth after <br /> Methods). The complementing fragment sequenced           single division. These data suggest important role for <br /> contained open reading frame 1,007 codons (3,021 bp)       HKEJIRATJ gene product cell growth. A diploid strain <br /> encoding protein approximately 116 kDa (calculated          bearing disrupted gene wild-type copy gene <br /> molecular weight, 116,773).                                      UR43 plasmid sporulated, tetrads dis- <br />    The oldest complete sequence data base        sected. Spores allowed germinate rich medium <br /> revealed significant homology open reading frame      later replicated plates containing 5-FOA (5). All <br /> was sequence yeast gene KEMI (18). In       cells carry hisG gene insertion (confirmed by <br /> four independent studies, KEM1 gene identi-        PCR strategies) grow 5-FOA, demonstrating <br /> fied different activities named XRN1, SEP1, RAR5,       disrupted strain requires plasmid-borne copy of <br /> and DST2 (for summary, reference 17). For              gene mitotic growth (data shown). This result <br /> purposes report, refer gene XRN1I      confirms Amberg et al. (3) disruption RATIJ. <br /> KEM1, protein encodes (175 kDa) referred         To confirm independence cellular activities <br /> to p175.                                                      ascribed HKEJ/RATJ homologous XRNI/KEM1 <br />    Because high homology complementing gene        gene products, hkel-I mutant transformed with <br /> with XRNI/KEMI potential related function,        XRNJ/KEMI multicopy plasmid (pExo2) determine <br /> was originally named HKEJ homology         increased levels p175 rescue cells nonper- <br /> karyogamy enhancing gene, KEMI. (Alignment HKE1           missive temperature. The transformed cells compared <br /> amino acid sequence KEMI revealed          hkel-l&apos;s mutant cells rescued HKEJ/RATJ <br /> regions significant stretches homology, including 37%      plasmid. As shown Fig. 2, thermosensitive growth of <br /> identity 414 amino acid residues [3; data shown].)      hkel-l&apos; mutant rescued XRNJIKEMJ under <br /> Concurrent studies, present     conditions HKEJ/RATI rescued growth 37&#194;&#176;C. <br /> data base, sequence molecular characterization      Additionally, XRNI/KEMJ gene multicopy plasmid <br /> of gene (RATI), know identical        (pExo2) did rescue lethality hkel/ratl ::URA3 <br /> to HKEJ (3). Since sequence same,          disruption strain sporulation tetrad analysis. The <br /> presented here. In order consistent literature,   meiotic products disrupted diploid strain bearing <br /> we use HKEI/RATI referring gene            XRN1IKEMI oii multicopy plasmid segregated 2 via- <br /> and p116 referring protein.                          ble:2 inviable spores (not shown). Parallel, control experi- <br />  VOL. 13, 1993                                                                  AN ESSENTIAL YEAST EXORIBONUCLEASE                     345 <br />  <br />                                                                          cells, antibody localization requires expression p116 <br />                                                                          multicopy plasmid. This affect normal <br />                                                                          subcellular localization protein. <br />                                                                             Translation defect HKE1&apos;3. In study Amberg et al. <br />           A                                                              (3), RATl identified screen designed isolate <br />                                                                          temperature-sensitive mutants involved trafficking of <br />               DIC                                                        mRNA nucleus. In situ localization mRNA in <br />                                                                          mutant reveals efficiently released from <br />                                                                          nucleus cytoplasm nonpermissive tempera- <br />                                                                          ture. Mutant cells held 37&#194;&#176;C exhibit marked nuclear <br />                                                                          staining mRNA loss cytoplasmic poly(A) signal <br />                                                                          compared cells 23&#194;&#176;C wild type at <br />                                                                          temperature. In similar study Kadowaki et al. (16) <br />                                                                          mutant (mtrl) shows similar mRNA nuclear reten- <br />                                                                          tion phenotype described. These authors additionally <br />           B                                                              demonstrate protein synthesis decreased this <br />                                                                          mutant mRNA levels remain constant. To determine if <br />               DAPI                                                       hkel-l&apos;s mutant exhibited similar phenotype, exam- <br />                                                                          ined protein synthesis mutant. <br />                                                                             Wild-type hkel-ls cells labeled Na235SO4 at <br />                                                                          permissive nonpermissive temperatures, the <br />                                                                          abilities cells incorporate amino acids assayed <br />                                                                          (Materials Methods). Protein synthesis hkel-1 <br />                                                                          temperature-sensitive mutant significantly lower than <br />           c <br />                                                                          wild-type cells following shift 37&#194;&#176;C. After shifting <br />                                                                          nonpermissive temperature, protein synthesis the <br />               anti trpE::p116                                            mutant begins reduced levels 10% of <br />                                                                          wild type (Fig. 4B). The decrease protein synthesis could <br />                                                                          reflect posttranscriptional event blocks acces- <br />                                                                          sibility RNA translation apparatus. <br />                                                                             Evidence HKEI/RATI gene encodes protein with <br />                                                                          exonuclease (RNase) activity. The homologous p175 protein <br />                                                                          encoded XRNJIKEMI gene have <br />   FIG. 3. Intracellular localization p116. Polyclonal antibodies <br />                                                                          5&apos;--3&apos; exoribonuclease activity 5&apos;-)3&apos; DNase activity <br /> were  generated trpE fusion protein amino acids 168       catalyze DNA strand exchange (for review, see <br /> to 403 p116. Crude sera immunopurified          reference 17). It identified studies 5&apos;--*3&apos; <br /> fusion protein obtained E. coli extracts. Yeast cells bearing   exoribonuclease activities S. cerevisiae 10 years ago <br /> hkel/ratl::URA3 disruption HKE1/RATJ high-copy-num-             (36, 41). Yeast exoribonuclease activities elute hydrox- <br /> ber vector (pMKHKEl) fixed prepared immunofluo-             ylapatite columns discrete peaks (36) the <br /> rescence trpE::pll6 antibody described Materials          predominant activities hydrolyze poly(A) pH 9.4 and <br /> Methods. (A) Cell field viewed DIC optics; (B) cells      require Mg2+. These exoribonuclease activities account for <br /> stained DAPI visualize nuclei; (C) fluorescence       &gt;80% Mg2+-dependent RNase activity cytoplasmic <br /> trpE::pll6 antibody, indicating nuclear location protein   ribosome fractions yeast cells (38). Earlier studies <br /> some cells. <br />                                                                          demonstrated exonuclease peak eluting hy- <br />                                                                          droxylapatite higher salt concentrations p175, <br />                                                                          XRNI/KEM1 gene product (25, 36). The RNase <br /> ments  performed pExo2 disruption mutant                peak, eluting lower salt concentration, as <br /> YPH500km1 ensure plasmid rescue slow                extensively characterized, purified <br /> growth phenotype XRNI/KEMJ disruption mutant                       (37) degrade 5&apos; terminally labeled rRNA and <br /> (25).                                                                    [5,_32p, 3H](pA)4 manner described p175 <br />    Antibody localization. Indirect immunofluorescence                 (36, 38) (see Fig. 7). It referred exo-2 below. To <br /> wild-type yeast cells bearing HKEI/RATl 2p,m plasmid                determine exoribonuclease activity lower-salt <br /> (pMKHKE1) using anti-trpE::pll6 antisera revealed                  hydroxylapatite column peak highly <br /> nuclear location p116 approximately 30 40%              conserved HKEJIRATJ gene product, prepared protein <br /> cells given field. The staining distributed             extracts hydroxylapatite column chromatography a <br /> nucleus space defined DAPI detection                   yeast strain harboring HKE1/RATl multicopy plasmid <br /> nuclear DNA (Fig. 3). Not shown control experiment                  strain plasmid. Quantitation <br /> using NOP1 antibodies. NOP1 nucleolar protein (4),              Mg2 -requiring poly(A)-hydrolyzing activities fractions <br /> staining antibody evident nuclei             eluted hydroxylapatite columns revealed four- to <br /> cells. This uniform nuclear staining NOP1 indicates              fivefold increase level nuclease activity cofraction- <br /> the lower percentage nuclear staining                  ating peak eluting lower salt concentrations in <br /> anti-p116 antibodies likely artifactual effects           yeast cells bearing multiple copies HKEJIRA T1 (Fig. SA). <br /> from fixation spheroplasting. It possible nuclear         Because high level peak, p175 activity found <br /> localization cells reflects cell cycle-specific           shoulder activity. The overproduced HKEI/ <br /> regulation localization p116. However,              RATI activity eluted location peak of <br /> the relatively low abundance protein wild-type                activity shown previously remain yeast cells harboring <br />  346            KENNA ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br /> A. GROWTH                                                                 A. <br />  <br />  <br />  <br />         E <br />                                                                                    F- <br />  J)C,D <br />      X  CD <br />                                                                                    C) <br />                                                                                    LU <br />  <br />  <br />  <br />  <br />   0i   O                                                                                 ) <br />                                                                                    L] <br />                                                                                    (z <br />                                                                                    z <br />                                                                                    m <br />  B.     PROTEIN SYNTHESIS                                                          rr <br />                                                                                    0 <br />                                                                                    x <br />                                                                                    IlJ <br />   CU)                                               WT (37-C) <br />                  3- <br />  <br />  <br />                                                     WT (23-C) <br />   = co <br />                                                                                              0              10               20             30 <br />   Z        _                                                                                                 FRAC,TION NO. <br />                                                     Mutant (23-C) <br />   z E <br />  <br />                  1- <br />        0 <br />  <br />  <br />   IL                                                Mutant (37-C) <br />                                                                           B.                          WT            WT   +   (pMKHKE1) <br />                       0   2       4         6          8            10                                              0 <br />  <br />  <br />   FIG. 4. Protein synthesis hkel-l&apos;s. (A) Log-phase cells grown       Fraction No.          14    16    18     x <br />                                                                                                                              14   16   18 <br /> at 23&#194;&#176;C shifted fresh medium indicated tempera- <br /> tures. Aliquots taken times indicated, optical <br /> density (OD) 600 nm determined. Open symbols represent              HKE1p <br /> cultures growing 23&#194;&#176;C filled symbols represent cultures             116 KDa <br />  <br /> growing 37&#194;&#176;C. (B) Protein synthesis assayed incorporation <br /> of Na235SO4 described Materials Methods. At times <br /> indicated, aliquots containing numbers cells               FIG. 5. (A) Separation exoribonuclease activities the <br /> removed culture flask labeled 5 min addition     p116 overproducing strain. Crude extracts prepared the <br /> of 5 pCi Na235SO4. Samples treated described plotted      yeast parental strain grown pMKHKE1 (2,um) and <br /> as counts minute versus time. Open symbols represent cultures        chromatographed hydroxylapatite column, fractions were <br /> growing 23&#194;&#176;C LSM filled symbols represent cultures             assayed exoribonuclease activity (see Materials Methods). <br /> growing 37&#194;&#176;C LSM. WT, wild type.                                   The values obtained plotted units RNase activity versus <br />                                                                          fraction number. The peak exoribonuclease activity higher salt <br />                                                                          concentrations fraction 24 wild-type extract p175 <br />                                                                          (21). This peak activity noted shoulder major <br /> a disruption p175-encoding gene (25). The increase in <br />                                                                          low-salt peak HKEJ/RATI overproducing strain. (B) Western <br />                                                                          blot p116 exonuclease-containing fractions. Immunopurified <br /> RNase activity comparable observed overpro-               rabbit antibody TrpE::pll6 fusion protein (see Materials and <br /> duction p175 identical conditions earlier studies            Methods) used probe Western blot containing hydroxylap- <br /> (25). The hkel-lts mutant parental strain                  atite column fractions (Fig. 5A) increased 5&apos;--3&apos; exoribonu- <br /> grown 23&#194;&#176;C, extracts examined                    clease activity crude extracts. Lanes: extract, equal <br /> manner hydroxylapatite chromatography. The extract                 protein amounts crude extracts wild-type parental strain <br /> the parental strain showed results similar Fig. SA;          (WT) WT + pMKHKE1; 14, 16, 18, equal loadings total <br /> however, poly(A)-hydrolyzing peak eluting lower salt              protein associated exoribonuclease peak (fractions 14, 16, <br /> concentrations reduced 30% parental                  18, respectively) WT WT + pMKHKE1. <br /> level hkeltS mutant (data shown). The hkel-l&apos;s <br /> strain grows slowly, 23&#194;&#176;C (Fig. 4A). <br />    The increased level RNase activity observed Fig. SA             reacted RNase activity fractions. Immu- <br /> was consistent increased synthesis HKEI/                  noaffinity-purified antiserum p116 prepared (see Ma- <br /> RATI gene product multicopy HKEI/RATJ plas-                     terials Methods) used probe crude extracts <br /> mid. However, increase result regulation             partially purified hydroxylapatite peak fractions (14, <br /> of synthesis activity RNase HKEI/RATl               16, 18) RNase activity Fig. 5A. These studies <br /> gene. To eliminate possibility, used p116 antibody            showed protein approximately 116 kDa found <br /> show recognized protein expected size            fractions recognized antiserum p116 <br /> HKEJIRAT1 gene product (116 kDa), protein                  (Fig. 5B). The level protein considerably in- <br /> overexpressed HKE1/RATI plasmid,                      creased (at fivefold) identical extracts cells <br />  VOL. 13, 1993                                                                                   AN ESSENTIAL YEAST EXORIBONUCLEASE                     347 <br />  <br />          TABLE 1. Immunodepletion RNase activity                                    overexpressed activity eluted peak shown. Figure 6B <br />                      anti-p116 antibodiesa                                                shows Western blot peak fractions column. <br />          Enzyme fraction                                                     % Activity   A protein band approximately 116 kDa immunoreactive <br />           serum (,ul)                                                    remaining    case elution profile parallels the <br /> 16 (WT)                                                                                   activity. Although difference specific activity, i.e., <br />   None ...........................................                              100       RNase activity immunoreactive protein, the <br />   Preimmune (1) .......................            ....................          97       overexpressed exo-2 fractions (approximately 1.5- to <br />   Preinmune (2.5) ...........................................                    95       2-fold; compare overexpressed fraction 37 exo-2 frac- <br />   Immune (1) .................             ..........................            12       tion 32 [Fig. 6B]) apparent, quantitation actual <br />   Immune (2.5) .....................            ......................           10       amounts protein loaded present bands (e.g., by <br />                                                                                           staining) possible low levels of <br /> 16 (WT + pMKHKEl)                                                                         protein (less 20 ,ug/ml case exo-2 fractions). <br />   None ...........................................                              100       It possible overexpressed protein differs a <br />   Preimmune (1) .....                                                           104 <br />                                     ...................................... <br />  <br />  <br />   Preimmune (2.5) ...........................................                   107       structural modification(s) affecting RNase activity immu- <br />   Immune (1) .................             ..........................            19       noreactivity. <br />   Immune (2.5) .....................            ......................           10          The poly(A) hydrolytic activity columns was <br />                                                                                           Mg2e dependent, 98% product being <br /> Heparin-agarose-purified exo-2                                                            5&apos;-AMP (data shown). Most studies mode of <br />   None ...........................................                              100       hydrolysis exo-2, purified manner de- <br />   Preimmune (1) .......................            ....................         100       scribed legend Fig. 6, with <br />   Preimmune (2.5) ...........................................                    93       [3H]poly(A) substrate (38). These studies showed <br />   Immune (1) .................             ..........................            17       98% hydrolysis product 5&apos;-AMP, the <br />   Immune (2.5) .        ...........................................               7 <br />                                                                                           remaining 2% short adenylate oligonucleotides (2 4 <br />   Aliquots hydroxylapatite column fraction 16 wild-type                   nucleotides long). These probably derived the <br /> extract (3 pg) wild-type plus pMKHKE1 extract (3.5 pg) (Fig. 5A, WT               termini chains, exoribonucleases fail to <br /> and WT + pMKHKEl, respectively) 0.9 ig heparin-agarose-purified                  hydrolyze chain completely (36). With [5_-32P, <br /> exo-2 (36) mixed antiserum shown reaction mixtures (60 <br /> pl) containing 16 mg albumin ml, 1.6% Nonidet P-40, 50 mM NaCI,                3H]poly(A) substrate, 32P-labeled product is <br /> 3.3 mM sodium phosphate buffer (pH 7.2). After 4 h 4&apos;C, protein A-agarose              5&apos;-AMP. Poly(A) lacking 5&apos;-phosphate end group hydro- <br /> (Pierce) (30 pL1 50% suspension) added, suspensions                 lyzed exo-2 25% rate poly(A) containing a <br /> mixed 2.5 h 4&#194;&#176;C. Aliquots supematant solutions following                    terminal phosphate. Similar findings for <br /> centrifugation assayed described Materials Methods, the <br /> percent activity remaining supematant shown. The crude                p175 (41). The hydrolysis RNA substrates yet <br /> rabbit antisera used indicated.                                                        studied detail, Fig. 7 shows analysis the <br />                                                                                           type exoribonuclease activity purified exo-2 fractions, <br />                                                                                           prepared manner shown Fig. 6, using <br />                                                                                           yeast [5&apos;-32P, 3H]rRNA (25S) substrate 0&apos;C. <span id='am-4' about='rdfs:label' typeof='owl:Thing'>Label</span> <br /> harboring multicopy plasmid pMKHKE1. The size                                      release incubation RNase measured. The <br /> this protein correlated estimated size (116 kDa)                            results 5,_32p label released considerably <br /> from DNA sequence analysis HKEI/RATI. The protein                                   faster 3H label, showing 5&apos;-&gt;3&apos; mode of <br /> barely detected cell extracts protein level                              hydrolysis predominant one. The enzyme fractions do <br /> which overproduced protein readily apparent [Fig. 5B,                              contain phosphatase activity conversion of <br /> WT versus WT + (pMKHKE1)]. The activity increase                                          5&apos;-AMP adenosine [5&apos;-32P]poly(A) 32Pi found. <br /> measured (Fig. 5A) represent low estimate                                    Further analysis substrate reaction specificity of <br /> actual increase vivo.                                                                  exo-2 progress. <br />    Immunodepletion RNase activity tested the <br /> addition p116 antiserum peak hydroxylapatite <br /> fractions Fig. 5A, preparation exo-2                                                         DISCUSSION <br /> purified YPH50QnJl yeast strain described by <br /> Stevens (37). Table 1 shows results immunopre-                                    In present study characterized conditional <br /> cipitation RNase activity trpE::pll6 fusion protein                             yeast mutant defective nucleolytic hydrolysis of <br /> antibody. Approximately 90% RNase activity                                     RNA. The essential HKEJ/RATI gene uncovered this <br /> depleted RNase-containing fractions incu-                               mutation encodes protein significant homology to <br /> bation immune antiserum incubation                                  yeast protein (p175) exhibits variety bio- <br /> with preimmune serum.                                                                 chemical activities, strand exchange exoribonu- <br />    Further purification heparin-agarose column chroma-                                 clease activity (17). In addition mRNA trafficking data <br /> tography Western immunoblot analysis overex-                                   pre&quot;iously published RATl (3), present <br /> pressed (HKElIRATl plasmid-dependent) RNase activity                                      sty. J HKEI/RATJ gene product exhibits 5&apos;--3&apos; <br /> was coincidental purification activity                              exoribonuclease activity like previously demonstrated <br /> with previously described exo-2 (36, 37)                             related p175. <br /> demonstrated. These data presented Fig. 6. The                                        p175 purified homogeneity based 5&apos;--3&apos; <br /> overexpressed poly(A)-hydrolyzing activity hydroxy-                                exoribonuclease activity (36, 41), gene encoding it <br /> lapatite column fractions obtained chromatography                                   designated subsequent studies XRNJ (exoribonu- <br /> crude extracts (Fig. SA) purifies similarly heparin-                                 clease 1 [25]). These earlier results analysis poly(A) <br /> agarose column exo-2 activity purified high-salt                            hydrolytic activity cells lacking active XRNJ gene <br /> ribosome wash yeast cells containing disruption                                 suggested second 5&apos;--3&apos; exoribonuclease could <br /> XRNJ (37). The heparin-agarose chromatography results                                  partially compensate loss p175 xrnl disruption <br /> an 8- 10-fold purification RNase activity. All                              mutants existed (25). We provided evidence this <br />  348            KENNA ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />        A.                                                                       0 <br />                 14                                                              U) <br />                                                                                                                                           51-32p <br />                                                                                       &gt;--J 30- <br />                 12 <br />                                                                                 az:    &lt; 20- <br />                                                                                                                            0.~~~~~~ <br />                                                                                                      0~~~~~ <br />                                                                                                     o0 <br />                 10 <br />                                                                                      -                                                        3H <br />                                                                                               --,_0                  -~00 <br />       E-                                                                                    I1OI                       I <br />       u                                                                                                                5                    10 <br />   U) .-. <br />                  8                                                                                         INCUBATION TIME, min <br />                                                                                   FIG. 7. Hydrolysis [5&apos;-32P, 3H]rRNA exo-2. [5 _32p, 3H] <br />                  6 <br />                                                                                 rRNA (25S) prepared described Stevens Maupin (40). <br />       z                                                                         The RNA (1.3 nmol) incubated times shown 0&#194;&#176;C with <br />                                                                                 50 U exo-2 (purified described [37]) reaction <br />   0m_s4                                                                         mixtures (50 ,ul) containing Tris-HCl buffer (pH 8.0) (25 mM), <br />            :     4                                                              NH4Cl (50 mM), MgCl2 (3 mM), dithiothreitol (0.5 mM), 30 p,g <br />            C                                                                    albumin. Assay acid-soluble radioactivity previ- <br />                                                                                 ously described (41).   Control values reaction mixtures   lacking <br />                  2                                                              enzyme subtracted. <br />  <br />                  0    ___wi__s_------ 7- r              -r--- -t     --r----&apos;   second enzyme encoded essential gene HKEJ/ <br />                       0            10         20          30          40        RAT1. <br />                                         FRACTION NO.                               The gene p175 cloned sequenced four <br />                                                                                 laboratories. By using assay detects trans- <br />                                                                                 fer strand DNA molecule complementary <br />                                                                                 single-stranded partner (21), gene strand-exchange <br />       B.                      W T + pM K H KE1            E xo -2               protein, designated SEPI, characterized (42) was <br />                             -I                                                  independently confirmed designated DST2 (9, <br />  Frction No.              23 26 29 32 35 37 26 28 30 32 34 36                   10). SEPI shown contain 5&apos;--3&apos; exonuclease <br />  Units, 10 2              0.6 1.7 3.4 3.4 2.3 1,3 0.3 0.8 1.7 2.2 1.8 0.8       activity hydrolysis single- double- <br />                                                                                 stranded DNA (15). It activity likely <br />    kDo 205-                                                                     responsible synapsis observed 5&apos;-*3&apos; polarity of <br />                                                                                 strand-exchange reaction. However, mechanistic <br />                116-            *          u                                     details activity resolved present (17). <br />                                                                                 Kolodner et al. (21) data suggesting p175 plays an <br />                                                                                 important role meiotic recombination. Finally, other <br />                80 -                                                             studies utilizing vivo mutant screens report the <br />                                                                                 fidelity nuclear fusion (18) mitotic stability of <br />                                                                                 minichromosomes containing altered origins (19) have <br />                49 -                                                             yielded gene, designated KEMI RAR5, respec- <br />                                                                                 tively. <br />                                                                                   What seemingly different activities do <br />   FIG. 6. (A) Heparin-agarose chromatography overex- <br />                                                                                 another? Certainly, exonuclease be <br />                                                                                 involved number different hydrolytic reactions and <br /> pressed RNase activity exo-2.Hydroxylapatite chromatography                 reaction affect distinct processes. Loss the <br /> of extracts yeast strain (0.4 g) containing high-copy- <br /> number HKE. l/RATI plasmid carried described                      exonuclease result confusing array phenotypes. <br /> Materials 1Methods using 9-ml column 80-ml gradient             Events 5&apos;-*3&apos; exoribonucleases involved <br /> of potassium pphosphate buffer described previously (36). Assay              described previously (41) discussed again <br /> of fractions shiowed elution profile similar Fig. SA.             below. Recent studies p175 mutants mRNA <br /> Fractions 13 tRo 17, containing 4 x 103 U RNase activity,               turnover affected, resulting differences cellular <br /> combined, diluited equal volume 20 mM Tris-HCl buffer                mRNA protein levels (24). Vreken Raue have <br /> (pH 7.7)-10% glycerol, applied 4-ml column heparin-                 recently presented results suggesting involvement of <br /> agarose (Sigmaa) prewashed buffer containing 100 mM               exonuclease mRNA turnover (43). A role an <br /> (NH4)2SO4. Tihe column washed 8 ml same <br /> buffer elu ted 40-ml linear gradient 100 600 mM                exonuclease process affect other <br /> (NT A V-       In 1Im;111ltPr iractions <br />          n r(une-mllninlter <br /> kI&apos;n4)2Z5k4.                            xw .-r..nih4tfi-d <br />                               frnrtinne weoreconcult;lcu,  snt1 kiiquoLs <br />                                                           anlu  nliniint        activities cell. <br /> were assayed using [3H]poly(A) substrate described in <br /> Materials Methods. Approximately 5.5 x 103 U RNase <br /> activity recovered. For exo-2 chromatography, exo-2 was <br /> purified described Stevens (37) 5 g YPH5Oxrml                 protein RNase activity elute current heparin-agarose <br /> yeast strain hydroxylapatite chromatography step.                   columns higher salt concentration previously described [37], <br /> Hydroxylapatite fractions (fractions 13 17) combined,               apparently difference heparin-agarose.) (B) Western <br /> 12 x 103 U RNase activity chromatographed heparin-                    blotting carried described Materials Methods, by <br /> agarose described above. Aliquots fractions assayed,             using 500 p.l designated overexpressed RNase fractions 200 <br /> and 9.8 x 103 U activity recovered. The elution profile,                 ,ul exo-2 fractions. The units RNase activity were <br /> normalized 50 U activity, plotted column. (Both               contained fraction shown. WT, wild type. <br />  VOL. 13, 1993                                                            AN ESSENTIAL YEAST EXORIBONUCLEASE                          349 <br />  <br />    HKE1IRATI exoribonuclease activity related pheno-          activities begin better understand temporal and <br /> types.  An array phenotypes condi-      spatial order events involved nuclear cytoplasmic RNA <br /> tional HKEJIRAT1 gene mutants. In vivo labeling studies        trafficking relationship transcription, process- <br /> characterize consequences conditional defects asso-        ing events, trafficking. Additional study reaction <br /> ciated loss HKEJ/RATl activity reveal post-         specificity p116 currently way. <br /> transcriptional block RNA utilization. Translation of <br /> mRNA severely reduced ,at restrictive temperature.                              ACKNOWLEDGMENTS <br /> This result expected light findings            We thank Eva Hoke excellent technical assistance. We <br /> Amberg et al. RAT1 (3). In situ localization mRNA       acknowledge Ben Hall Terni Aldridge providing unpub- <br /> this mutant reveals efficiently released   lished sequence data compare HKEI, Chuck Cole and <br /> the nucleus cytoplasm nonpermissive tempera-        David Amberg sharing unpublished data sequence, and <br /> ture. Mutant cells held 37&#194;&#176;C exhibit marked nuclear          Mark Goebl (Indiana School Medicine) drawing attention <br /> staining mRNA loss cytoplasmic poly(A) signal         KEM1 homology. <br /> compared cells 23&#194;&#176;C wild-type cells       This work supported grants National Institutes of <br /> at temperature. This phenotype defects hkell        Health (GM41758) Robert A. Welch Foundation (AQ-814) <br /> ratl mutants failure nuclear mRNA       American Heart Association established investigator <br />                                                                   award, M.G.D. A.S. supported Office Health and <br /> efficiently clear transcriptional process nucleocy-    Environmental Research, U.S. Department Energy, con- <br /> toplasmic translocation apparatus (for review, refer-       tract DE-AC05-84 OR 21400 Martin Marietta Energy <br /> ence 28). HKEJ/RAT1 cloned independently          Systems. Direct support A.S. grant Explor- <br /> other laboratories (12, 33), uncovered          atory Studies Program, Oak Ridge National Laboratory. M.M. was <br /> temperature-sensitive suppressor tRNA promoter mu-           supported NRSA National Institutes Health. <br /> tation (12) gene overexpression                                         REFERENCES <br /> restore wild-type growth yeasts bearing overproduced <br /> amount mutated form transcription factor TFIID         1. Adams, A. E. M., J. R. Pringle. 1984. Relationship actin <br /> (33). These data suggest p116 involved            tubulin distribution bud growth wild-type morpho- <br />                                                                       genetic mutant Saccharomyces cerevisiae. J. Cell Biol. 98:934- <br /> transcriptional processes observed transport             945. <br /> defects secondary effects lack produc-        2. Alani, E., L. Cao, N. Kleckner. 1987. A method gene <br /> tion components essential RNA processing traf-              disruption allows repeated use URA3 selection the <br /> ficking.                                                              construction multiply disrupted yeast strains. Genetics 116: <br />    As far transcriptional involvement p116 con-              541-545. <br /> cerned, spectrum nuclease strand-exchange ac-           3. Amberg, D. C., A. L. Goldstein, C. N. Cole. 1992. Isolation <br /> tivities related XRNJIKEM1 protein suggest             characterization RAT1: essential gene Saccharo- <br /> similar potential role HKEJIRATl gene product              myces cerevisiae required efficient nucleocytoplasmic <br /> transcript displacement DNA template (for addi-              trafficking mRNA. Genes Dev. 6:1173-1189. <br />                                                                    4. Aris, J. P., G. Blobel. 1988. Identification characteriza- <br /> tional discussion, reference 17). However, factors           tion yeast nucleolar protein similar rat liver <br /> as associated RNA polymerase II,              nucleolar protein. J. Cell Biol. 107:17-31. <br /> thus role p116 unlikely, p116 does      5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive <br /> appear abundant protein (as revealed                  selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> Western analysis protein levels Northern [RNA]                 ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> analysis message levels) relative polymerase II. Possi-         Genet. 197:345-346. <br /> ble involvement 5&apos;- &apos;3&apos; exonuclease transcription          6. Connelly, S., J. L. Manely. 1988. A functional mRNA <br /> termination suggested Connelly Manley (6).            polyadenylation signal required transcription RNA <br /> If exonuclease actually involved termi-           polymerase II. Genes Dev. 2:440-452. <br />                                                                    7. Devereux, J., P. Haeberli, 0. Smithies. 1984. A comprehen- <br /> nation event, affect transcription processing            sive set sequence analysis programs VAX. Nucleic <br /> pathway mRNA removing 3&apos;-terminal fragments                 Acids Res. 12:387-395. <br /> resulting transcript cleavage polyadenylation.            8. Douglas, M., B. Geller, S. Emr. 1984. Intracellular targeting <br />    Rather transcription related function, p116        import Fl 1-subunit-,B-galactosidase hybrid protein <br /> involved RNA processing events nucleus.                 yeast mitochondria. Proc. Natl. Acad. Sci. USA 81:3983- <br /> Amberg et al. (3) reported pre-rRNA process-           3987. <br /> ing defect RAT1. The 5.8S rRNA suffer           9. Dykstra, C. C., R. K. Hamatake, A. Sugino. 1990. DNA <br /> processing maturation defect, slightly longer           strand protein f yeast mitotic cells differs strand <br /> forms rRNA produced nonpermissive                 transfer meiotic cells. J. Biol. Chem. 265:10968-10973. <br />                                                                   10. Dykstra, C. C., K. Kitada, A. B. Clark, R. K. Hamatake, A. <br /> temperature. The exoribonuclease activity ascribed p116            Sugino. 1991. Cloning characterization DST2, gene <br /> may responsible trimming 5&apos; ends            DNA strand transfer 1B Saccharomyces cerevisiae. <br /> 5.8S rRNA molecules, suggested authors. It            Mol. Cell. Biol. 11:2583-2592. <br /> previously suggested enzymatic properties p175        11. Emr, S., A. Vassarotti, J. Garrett, B. Geller, M. Takeda, and <br /> make good candidate 5&apos; trimming reactions rRNA            M. G. Douglas. 1986. The amino terminus F1-ATPase <br /> maturation (25, 41). Additionally, frag-            1-subunit precursor functions mitochondrial import signal. <br /> ments internal transcribed spacer 1 pre-rRNA                J. Cell Biol. 102:523-533. <br /> accumulate yeast cells lacking p175 (39). This suggests      12. Hall, B. D. (University Washington). 1991. Personal commu- <br /> role enzyme scavenger RNase pre-rRNA           nication. <br />                                                                   13. Harlow, E., D. Lane. 1988. Antibodies: laboratory man- <br /> processing. It plausible p116 involved rRNA             ual, p. 453. Cold Spring Harbor Laboratories, Cold Spring <br /> processing, trimming scavenger RNase               Harbor, N.Y. <br /> which coupled translocation nu-          14. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br /> clear envelope. Further investigation necessary un-             mation intact yeast cells treated alkali cations. J. <br /> cover interplay seemingly diverse               Bacteriol. 153:163-168. <br />  350     KENNA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br /> 15. Johnson, A. W., R. D. Kolodner. 1991. Strand exchange                Saccharomyces cerevisiae LEU2 gene Escherichia coli <br />     protein 1 Saccharomyces cerevisiae. J. Biol. Chem. 266:             ,B-galactosidase gene. Mol. Cell. Biol. 3:580-586. <br />     14046-14054.                                                       30.   Miller, J. 1972. Experiments molecular genetics, p. 352-355. <br /> 16. Kadowaki, T., Y. Zhao, A. Tartakoff. 1992. A conditional             Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     yeast mutant deficient mRNA t <br /> </body></html>