<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Cdc34 F-box protein Met30 <br /> are required degradation the <br /> Cdk-inhibitory kinase Swe1 <br /> Peter Kaiser,1 Rey A.L. Sia,2 Elaine G.S. Bardes,2 Daniel J. Lew,2 Steven I. Reed1,3 <br /> 1 <br /> The Scripps Research Institute (TSRI), La Jolla, California 92037 USA; 2Department Pharmacology Cancer Biology, <br /> Duke University Medical Center, Durham, North Carolina 27710 USA <br />  <br />  <br />  <br /> Ubiquitin-mediated proteolysis controls abundance cell cycle regulatory proteins. Recent work in <br /> Saccharomyces cerevisiae suggests complex consisting Cdc53, Skp1, component known <br /> as F-box protein (termed SCF) combination Cdc34 specifically targets regulatory proteins for <br /> degradation, substrate specificity likely mediated F-box subunit. A screen genetic <br /> interactions cdc34 <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> yielded MET30, encodes F-box protein. MET30 essential <br /> gene required cell cycle progression met30 mutations interact genetically mutations SCF <br /> components. Furthermore, physical interactions Met30, Cdc53, Cdc34, Skp1 vivo provide <br /> evidence SCFMet30 complex. We demonstrate involvement Met30 degradation the <br /> Cdk-inhibitory kinase Swe1. Swe1 stabilized met30 mutants GST-Met30 pull-down experiments <br /> reveal Met30 specifically binds Swe1 vivo. Furthermore, extracts prepared cdc34 met30 <br /> mutants defective polyubiquitination Swe1. Taken together, data suggest SCF-mediated <br /> proteolysis contribute regulation entry mitosis. Our data, combination previously <br /> published results, provide evidence distinct SCF complexes vivo support idea their <br /> F-box subunits mediate SCF substrate specificity. <br /> [Key Words: Met30; F-box; Swe1; SCF; ubiquitination] <br /> Received March 6, 1998; revised version accepted July 2, 1998. <br />  <br />  <br />  <br />  <br /> Proteolysis principal mechanisms                   Eukaryotes express family distinct E2 enzymes. <br /> cellular protein levels regulated. Therefore,           All share characteristic motifs allowed the <br /> surprising, cell cycle transitions frequently con-           identification complete set E2 enzymes Sac- <br /> trolled degradation regulatory proteins. Proteolysis            charomyces cerevisiae (Ubc1&#226;&#8364;&#8220;Ubc13) sequence analy- <br /> is particularly suitable cell cycle regulation,           sis S. cerevisiae genome. Although E2 enzymes <br /> it rapid irreversible process and, thus, ensures uni-            confer substrate specificity, general view is <br /> directional progression cell cycle phase transi-              E3 enzymes responsible majority sub- <br /> tions.                                                                strate recognition targeting, the <br />    In eukaryotes, regulated proteolysis initiated             components subjected regulation. Despite cen- <br /> by polyubiquitination target proteins. Ubiquitination              tral role, E3 enzymes well-characterized fac- <br /> requires ATP-dependent activation ubiquitin ubiq-               tors involved ubiquitination, probably they <br /> uitin-activating enzymes (E1), transfer conserved                generally share single identifying sequence mo- <br /> cystein residue ubiquitin-conjugating enzymes (E2               tif mode action. However, families pro- <br /> Ubc) and, finally, conjugation ubiquitin target pro-            teins E3 function identified, including <br /> teins. Ubiquitin conjugation requires specificity               HECT domain proteins (Huibregtse et al. 1995), <br /> factors (E3 ubiquitin ligase) thought recog-           Ubr1-related ubiquitin ligases (Varshavsky 1996), and <br /> nize directly interact substrate proteins, and,           anaphase-promoting complex (APC, called the <br /> some cases, serve thiol&#226;&#8364;&#8220;ubiquitin intermediates. The               cyclosome) (Lamb et al. 1994; Irniger et al. 1995; King et <br /> E1&#226;&#8364;&#8220;E2&#226;&#8364;&#8220;E3 reaction cascade results polyubiquitination               al. 1995; Sudakin et al. 1995). <br /> of substrate proteins lysine residues. These polyubiq-                Recent work S. cerevisiae suggests complex <br /> uitinated proteins subsequently recognized de-                consisting Cdc53, Skp1, Cdc4 functions E3 <br /> graded 26S proteasome, complex protease-con-                 ubiquitin ligase Cdk-inhibitor Sic1 (Bai et al. <br /> taining organelle (Hochstrasser 1996).                                1996; Feldmann et al. 1997; Skowyra et al. 1997). This <br />                                                                       complex termed SCFCdc4 (Skp1-Cullin (or <br /> 3 <br />  Corresponding author.                                                Cdc53)-F-box protein) indicating F-box protein <br /> E-MAIL sreed@scripps.edu; FAX (619) 784-2781.                         particular complex Cdc4. F-box proteins share <br />  <br />  <br />     GENES &amp; DEVELOPMENT 12:2587&#226;&#8364;&#8220;2597 &#194;&#169; 1998 Cold Spring Harbor Laboratory Press ISSN 0890-9369/98 $5.00; www.genesdev.org   2587 <br />  Kaiser et al. <br />  <br />  <br />  <br /> a short sequence motif, F-box, shown        Results <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> mediate binding Skp1 (Bai et al. 1996). Data <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> <br />                                                                Isolation MET30 enhancer cdc34-3 mutants <br /> searches revealed F-box proteins exist a <br /> small family. Interestingly, yeast F-box protein Grr1      The S. cerevisiae ubiquitin-conjugating enzyme Cdc34 <br /> has linked G1-cyclin proteolysis (Barral et al.        implicated proteolysis key <br /> 1995) physical interaction G1 cyclins           regulatory proteins, including Cdk inhibitors Sic1 <br /> Cln1 Cln2 demonstrated vitro (Skowyra          Far1 (McKinney et al. 1993; Feldmann et al. 1997; <br /> et al. 1997) suggesting existence SCFGrr1 com-       Skowyra et al. 1997), G1 cyclins Cln1, Cln2, Cln3 <br /> plex. These observations led proposal         (Deshaies et al. 1995; Yaglom et al. 1995; Lanker et al. <br /> model different F-box proteins associated        1996), DNA replication factor Cdc6 (Piatti et al. <br /> Skp1 Cdc53 define substrate specificity SCF         1996; Drury et al. 1997). The current model predicts that <br /> ubiquitin ligase complexes (Bai et al. 1996; Feldmann et       different E3 ligases mediate substrate specificity. To <br /> al. 1997; Skowyra et al. 1997). SCF-dependent ubiquitin        identify E3 enzymes, used haploinsufficiency <br /> conjugation budding yeast shown cata-        screen temperature-sensitive cdc34 mutant. We <br /> lyzed E2 enzyme Cdc34, interacts             reasoned 50% reduction limiting component <br /> Cdc53 ubiquitinate Sic1 Cln2 (Deshaies et          Cdc34 pathway result enhancement of <br /> al. 1995; Mathias et al. 1996; Willems et al. 1996; Verma      temperature sensitivity diploid cdc34-3 cells (for <br /> et al. 1997). Currently, however, ruled       description rationale screen, Marini et <br /> that E2 enzymes function conjunction             al. 1996). Therefore, mutagenized diploid cdc34-3 <br /> SCF.                                                           mutants random heterozygous transposon-mediated <br />    In genetic screen designed identify limiting com-      gene disruptions (see Materials Methods), se- <br /> ponents Cdc34-dependent ubiquitination path-            lected mutants unable grow semiper- <br /> way, isolated gene encoding F-box protein           missive temperature parental strain (32&#194;&#176;C). Three <br /> Met30. MET30 essential gene recently        mutants exhibited 2:2 segregation lethality <br /> implicated repression genes required methio-         linked inserted URA3 marker associated the <br /> nine metabolism. Mutants harboring dominant                transposon. Sequence analysis DNA flanking the <br /> MET30-1 allele shown fail repress expression        transposon insertion sites revealed ORF <br /> of MET25, gene required methionine biosynthesis,         disrupted mutants site of <br /> in response S-adenosylmethionine (Thomas et al.             integration mutants identical. Analysis of <br /> 1995). However, defect explain lethality       disruption sites use Yeast Genome Data <br /> confered met30 loss-of-function mutations.                  Base identified mutated gene MET30. To verify <br />    Here, report Met30 required proteolysis      mutation MET30 caused enhancement the <br /> of Cdk-inhibitory kinase Swe1. Swe1 S. cere-        temperature sensitivity mutants, re- <br /> visiae homolog fission yeast Wee1 phosphorylates        placed copy entire MET30 ORF diploid <br /> the cyclin-dependent kinase Cdc28 tyrosine 19               cdc34-3 mutant gene conferring resistance the <br /> (Booher et al. 1993). Phosphorylation specific ty-          drug G418. This heterozygous gene disruption resulted <br /> rosines Wee1-like kinases renders cyclin-dependent          reduced permissive temperature cdc34-3 <br /> kinases inactive. Entry mitosis organisms         mutants observed original enhancer <br /> is largely regulated Wee1-dependent tyrosine phos-          mutants (Fig. 1A). A disruption copy MET30 in <br /> phorylation followed dephosphorylation Cdks              diploid wild-type strain did affect growth any <br /> (Dunphy 1994). Although enzymatic machinery                temperature (data shown). Tetrad analysis het- <br /> that controls tyrosine phosphorylation dephosphory-        erozygously disrupted wild-type strain revealed 2:2 seg- <br /> lation Cdks conserved budding yeast, phosphory-       regation lethality viable spores sensitive <br /> lation tyrosine 19 Cdc28 required proper      G418, confirming MET30 essential gene, as <br /> timing mitosis unperturbed cell cycle            reported previously (Thomas et al. 1995). <br /> (Russell et al. 1989; Amon et al. 1992; Booher et al. 1993). <br /> However, critical inhibition mitosis in <br />                                                                Met30 interacts genetically Cdc53, Cdc34, <br /> response activation morphogenesis checkpoint <br />                                                                Skp1 <br /> (Lew Reed 1995a,b) participates adapta- <br /> tion checkpoint arrests (Rudner Murray          To analyze functional relationship Met30 <br /> 1996).                                                         proteins involved Cdc34-dependent ubiquiti- <br />    In paper F-box protein Met30       nation pathway, tested genetic interactions <br /> part SCFMet30 complex vivo specifically tar-      SCF components. To carry detailed ge- <br /> gets Swe1 Cdc34-dependent proteolysis pathway.          netic analysis, constructed temperature-sensitive <br /> These results link regulation tyrosine phosphoryla-         met30 mutants. For purpose, coding region of <br /> tion Cdks SCF-regulated proteolysis. Because           MET30 mutagenized vitro PCR mutants <br /> highly conserved Met30 homologs eu-         unable grow 37&#194;&#176;C selected. One these, <br /> karyotes, including mammals, SCF-mediated ubiquiti-            met30-6, chosen analysis. <br /> nation Wee1-like kinases play significant role          We demonstrated genetic interaction be- <br /> in control entry mitosis.                              tween cdc34 temperature sensitivity MET30 gene <br />  <br />  <br /> 2588        GENES &amp; DEVELOPMENT <br />                                                                                                   Met30 required Swe1 proteolysis <br />  <br />  <br />  <br />                                                                     tagged Met30(HA)3 purified glutathione beads. <br />                                                                     Proteins bound beads eluted glutathi- <br />                                                                     analyzed immunoblotting. Met30 specifically <br />                                                                     copurified Skp1 (Fig. 2A). We performed coim- <br />                                                                     munoprecipitation experiments strains expressing <br />                                                                     Met30 tagged RGS6H epitope Cdc34(HA)3 <br />                                                                     Cdc53(HA)3, respectively. Met30 im- <br />                                                                     mune complexes Cdc34 Cdc53 (Fig. 2B). The <br />                                                                     mobility species Cdc53 detected consistent <br />                                                                     report suggesting slower migrating iso- <br />                                                                     form Cdc53 modified ubiquitin-related <br />                                                                     protein Rub1 (Lammer et al. 1998). Whereas stoichi- <br />                                                                     ometry Met30/Cdc53 association unchanged <br />                                                                     different experiments, Met30 associ- <br />                                                                     ated Cdc34 variable signifi- <br />                                                                     cant. <br />                                                                       Taken together, genetic biochemical <br />                                                                     results suggest Met30 multiprotein com- <br /> Figure 1. Genetic interactions MET30, CDC34, and <br /> SCF components. (A) Herterozygous disruption MET30 en- <br /> hances temperature sensitivity diploid cdc34-3 mutants. <br /> Cells grown 23&#194;&#176;C, sequentially replica plated four <br /> new plates, incubated temperatures indicated. (B) <br /> High-copy suppression. met30-6, cdc53-1, cdc4-3 mutants <br /> were transformed 2&#194;&#181;-based plasmids harboring insert <br /> (vector), MET30 (2&#194;&#181;-MET30), CDC34 (2&#194;&#181;-CDC34). Transfor- <br /> mants grown 23&#194;&#176;C replicas incubated 34&#194;&#176;C <br /> (met30-6 cdc53-1) 30&#194;&#176;C (cdc4-3). To analyze genetic in- <br /> teractions MET30 SKP1, met30-6 mutants express- <br /> ing SKP1 control GAL1 promoter grown on <br /> glucose plates 23&#194;&#176;C, replica plated galactose plates in- <br /> duce expression SKP1 incubated restrictive tem- <br /> perature 34&#194;&#176;C. <br />  <br />  <br /> dosage (Fig. 1A). To provide additional support this <br /> interaction, transformed haploid met30-6 mutants <br /> with CDC34 cloned high-copy plasmid. Increased <br /> dosage Cdc34 suppressed lethality met30-6 mu- <br /> tants normally nonpermissive temperature (34&#194;&#176;C).               Figure 2. Physical interaction Met30 Skp1, Cdc34, and <br /> As expected, MET30 high-copy plasmid rescued               Cdc53. (A) Met30 forms complex Skp1 vivo. Cells <br /> mutant (Fig. 1B). We extended                 harboring endogenously expressed (HA)3-tagged MET30 and <br /> kind analysis cdc53-1 cdc4-3 mutants. The                 expressing GST&#226;&#8364;&#8220;SKP1 GST control CUP1 <br /> growth defect cdc53-1 mutants restrictive tempera-            promotor grown presence 100 &#194;&#181;M copper an <br /> ture suppressed high-copy MET30 high-               OD600 &#226;&#8240;&#8230; 0.5. Protein extracts incubated glutathione <br /> copy CDC34. However, increased levels Met30                   beads. After washes binding buffer beads were <br />                                                                     washed binding buffer, including 0.8 M sodium chloride <br /> Cdc34 did rescue cdc4-3 mutants. Finally, genetic <br />                                                                     (wash). Bound proteins eluted glutathione sub- <br /> link SKP1 MET30 established, because <br />                                                                     jected Western blot analysis antibody directed <br /> overexpression Skp1 suppressed temperature sen-              HA epitope. Probing blot antibodies di- <br /> sitivity met30-6 mutants (Fig. 1B).                              rected GST revealed two- threefold excess GST <br />                                                                     compared GST&#226;&#8364;&#8220;Skp1 (data shown). (B) Met30 a <br />                                                                     complex Cdc34 Cdc53 vivo. Cells expressing a <br /> Met30 physically interacts Cdc34, Cdc53,                       RGS6H epitope-tagged MET30 control GAL1 pro- <br /> and Skp1                                                            motor harboring endogenously expressed (HA)3- <br /> These genetic data suggest Cdc34, Cdc53, Skp1              tagged CDC34 CDC53 grown galactose an <br />                                                                     OD600 &#226;&#8240;&#8230; 0.5. Immunocomplexes purified 12CA5 an- <br /> interact Met30. However, genetic interactions can <br />                                                                     tibodies (directed HA epitope) coupled protein A, <br /> result indirect physiological interactions. There-             separated SDS-PAGE, analyzed immunoblotting with <br /> fore, sought direct evidence physical association          antibodies directed HA RGS6H epitopes to <br /> between Met30 components Cdc34-dependent                 detect Cdc53 Cdc34 Met30, respectively. Cells express- <br /> ubiquitination machinery. Proteins cells express-              ing Met30(RGS6H) HA-tagged protein used a <br /> ing biologically active GST-tagged Skp1 epitope-                control (no HA tag). <br />  <br />  <br />                                                                                                  GENES &amp; DEVELOPMENT              2589 <br />  Kaiser et al. <br />  <br />  <br />  <br /> plex Cdc34, Cdc53, Skp1. In accordance                tase Mih1 leading activation Cdc28 kinase <br /> the previously proposed nomenclature, refer               (Russell et al. 1989). To determine regula- <br /> Skp1&#226;&#8364;&#8220;Cdc53&#226;&#8364;&#8220;Met30 complex SCFMet30. While pa-                tory pathway affected met30 mutants, we <br /> per revised, similar analysis arrived           generated      met30-6     mih1::LEU2          met30-6 <br /> the conclusion published (Patton et al. 1998).             mih1::LEU2 swe1::URA3 strains. At 37&#194;&#176;C, met30-6 <br />                                                                     mih1::LEU2 cells arrested higher propor- <br />                                                                     tion cells exhibiting elongated bud <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. <br /> Swe1 potential target SCFMet30                              3D), elongated buds absent the <br /> The arrest phenotype met30-6 mutants 37&#194;&#176;C                 met30-6 mih1::LEU2 swe1::URA3 cells (Fig. 3E). As de- <br /> heterogeneous, including budded unbudded                   scribed previously, deletion MIH1 otherwise <br /> cells G1 G2 /M DNA content. The major-               wild-type strain did cause significant cell cycle <br /> ity population arrested round unbudded cells.             delay lead generation cells elongated buds <br /> We noticed fraction budded met30-6 cells              (Fig. 3A,B). This result suggested Swe1 be <br /> displayed elongated bud phenotype (Fig. 3C),            abundant active met30-6 cells in <br /> characteristic cells activation Clb1-4/              wild-type cells. <br /> Cdc28 kinase delayed prevented (Lew                    A role Cdc34 control Swe1/Mih1 pathway <br /> Reed 1993). One mechanism Clb1-4/Cdc28 ac-                  reported, role been <br /> tivation blocked inhibitory phosphorylation            missed failure cdc34 mutants degrade <br /> Cdc28 tyrosine 19 kinase Swe1 (Booher et al.              Cdk inhibitor Sic1 leads cell cycle arrest prior to <br /> 1993). This phosphorylation reversed phospha-             G2, Swe1 Mih1 relevant cell cycle <br />                                                                     progression. cdc34-3 sic1::URA3 cells arrested G2 /M <br />                                                                     predominantly large round bud phenotype (Fig. <br />                                                                     3F), reported previously (Schwob et al. 1994). To de- <br />                                                                     termine increase Swe1 activity occurred <br />                                                                     cdc34-3 mutants, generated cdc34-3 sic1::URA3 <br />                                                                     mih1::LEU2 strain. At 37&#194;&#176;C, large proportion (65%) of <br />                                                                     cells arrested elongated buds (Fig. 3G). Dele- <br />                                                                     tion SWE1 cdc34-3sic1::URA mutants did su- <br />                                                                     press G2 /M arrest phenotype, suggesting accu- <br />                                                                     mulation Cdc34 target responsible this <br />                                                                     cell cycle arrest (data shown). However, cells <br />                                                                     longer elongated buds (data shown), indicat- <br />                                                                     ing component phenotype caused by <br />                                                                     Swe1. <br />                                                                        In combination, findings suggest defect <br />                                                                     met30 cdc34 mutants higher level of <br />                                                                     Swe1 protein activity. This does make major <br />                                                                     contribution arrest phenotype mutant <br />                                                                     Mih1 suffices Cdc28 sufficiently <br />                                                                     active; deletion MIH1 uncovers role pro- <br />                                                                     teins controlling Cdc28 tyrosine phosphorylation. <br />                                                                        Given expectation Met30 Cdc34 act by <br />                                                                     targeting proteins ubiquitination subsequent deg- <br />                                                                     radation, simplest explanation findings <br />                                                                     Swe1 itself target SCFMet30. Con- <br />                                                                     sistent explanation, analysis Swe1 se- <br />                                                                     quence revealed regions PEST motifs charac- <br />                                                                     teristic unstable proteins (Rogers et al. 1986) are <br />                                                                     targets Cdc34-dependent ubiquitination <br /> Figure 3. Hyperactivation Swe1 pathway met30          pathway. To examine stability Swe1 protein, a <br /> cdc34 mutants. Cells grown 23&#194;&#176;C YEPD shifted         12myc epitope tag fused carboxyl terminus of <br /> to 37&#194;&#176;C 5 hr, followed fixation 70% ethanol visu-     Swe1, expression construct placed under <br /> alization DIC microscopy. Strains follows (percent    control inducible GAL1 promoter. Wild-type <br /> elongated cells buds population indicated paren-   met30-6 mutant cells containing construct were <br /> theses): (A) wild type (0%), (B) mih1::LEU2 (0%), (C) met30-6 <br />                                                                     incubated 37&#194;&#176;C 2 hr, induced express Swe1&#226;&#8364;&#8220; <br /> (23%), (D) met30-6 mih1::LEU2 (50%), (E) met30-6 mih1::LEU2 <br /> swe1::URA3 (10%&#226;&#8364;&#8221;these smaller straight than <br />                                                                     myc12 galactose 10 min, labeled with <br /> the elongated buds C D), (F) cdc34-3 sic1::URA3 (10%),       [35S]methionine/cysteine 10 min. Swe1&#226;&#8364;&#8220;myc12 ex- <br /> (G) cdc34-3 sic1::URA3 mih1::LEU2 (65%). Two cells          pression subsequently turned transferring the <br /> were counted sample. Bar, 10 &#194;&#181;m. The graph illustrates      cells dextrose medium, excess cold methi- <br /> the percentage cells elongated buds.                        onine casamino acids added. Degradation la- <br />  <br />  <br /> 2590        GENES &amp; DEVELOPMENT <br />                                                                                                     Met30 required Swe1 proteolysis <br />  <br />  <br />  <br /> beled Swe1&#226;&#8364;&#8220;myc12 significantly inhibited                   This physical interaction likely required tar- <br /> met30-6 mutant relative wild type (Fig. 4A,C). In con-             Swe1 Cdc34-degradation pathway. <br /> trast, Swe1&#226;&#8364;&#8220;myc12 degradation cdc4-3 mutants pro-                    In experiment described above, Swe1 ex- <br /> ceeded wild-type rates (Fig. 4A,C). To exclude possible            pressed control GAL1 promoter that <br /> cell cycle position effects, analyzed Swe1 turnover             leads cell cycle arrest result inhibition Cdc28 <br /> cells synchronized S phase. Cells arrested                 kinase activity. However, simultaneous overexpression <br /> addition hydroxyurea pulse-chase analysis                  B-type cyclin Clb2 suppresses cell cycle arrest, <br /> performed described above. Again, Swe1&#226;&#8364;&#8220;myc12 deg-                  presumably excess Clb2 hyperactivated the <br /> radation significantly inhibited met30-6 mutants               Cdc28 kinase. Interestingly, overexpression Clb2 in- <br /> relative wild type (Fig. 4B,C). The G1 cyclin, Cln2,           creased Swe1 copurified Met30 <br /> not stabilized met30-6 mutant, confirming                   (data shown), suggesting potential link between <br /> Swe1 stabilization caused general proteolysis             Clb2/Cdc28 mediated phosphorylation Swe1 and <br /> defect associated met30-6 mutation (data                 SCFMet30 mediated ubiquitination. <br /> shown). <br />  <br />                                                                       Extracts prepared cdc34 met30 mutants are <br /> Met30 physically interacts Swe1 vivo                          defective Swe1 polyubiquitination <br /> In current model SCF-mediated ubiquitination,                 Our data suggest Met30 required Swe1 ubiq- <br /> F-box proteins specifically recognize bind target pro-            uitination. However, establish direct link be- <br /> teins. A physical interaction Cdc4 target             tween Met30 function Swe1 ubiquitination, ana- <br /> Sic1, binding Grr1 Cln1, dem-               lyzed ability wild-type mutant yeast cell ex- <br /> onstrated vitro (Feldmann et al. 1997; Skowyra et al.              tracts ubiquitinate Swe1. To end, incubated <br /> 1997). We asked complex Met30                    radiolabeled Swe1, produced vitro translation, with <br /> putative target Swe1 detected vivo. Extracts              yeast cell extracts analyzed reaction prod- <br /> were prepared yeast cells expressing epitope-                 ucts SDS-PAGE. Incubation extracts prepared <br /> tagged allele Swe1 GST&#226;&#8364;&#8220;Met30 fusion,                     wild-type cells resulted accumulation more <br /> Met30-associated complexes purified glutathi-                 slowly migrating forms Swe1 (Fig. 6A). This reaction <br /> one beads. Swe1 specifically copurified GST&#226;&#8364;&#8220;                     dependent addition ATP (data shown). <br /> Met30, binding GST detected (Fig. 5).               To ask slower migrating forms Swe1 were <br />  <br />  <br />  <br />  <br /> Figure 4. Swe1 stabilized met30 cdc4 mutants. (A) Wild-type (top), cdc4-3 (middle), met30-6 (bottom) cells containing <br /> the GAL1&#226;&#8364;&#8220;SWE1&#226;&#8364;&#8220;myc12 construct shifted 37&#194;&#176;C 2 hr. Galactose added 2% cells incubated a <br /> further 10 min induce Swe1&#226;&#8364;&#8220;myc12 expression. Cells labeled 10 min 37&#194;&#176;C followed termination Swe1 synthesis <br /> and labeling 4-hr time course. The immunoprecipitated Swe1 analyzed SDS-PAGE PhosphorImaging. (Left lane) <br /> Immunoprecipitates cells lacking Swe1&#226;&#8364;&#8220;myc construct. The band running Swe1&#226;&#8364;&#8220;myc12 (asterisk) background <br /> band. (B) A similar experiment conducted synchronized cells. Wild-type met30-6 cells arrested addition 200 <br /> mM hydroxyurea (HU) 3 hr 23&#194;&#176;C prior temperature shift 37&#194;&#176;C. Cell cycle arrest monitored visually (&gt;80% large <br /> budded cells). Pulse-chase analysis performed described A continuous presence 200 mM HU. (C) The 35S-labeled <br /> Swe1&#226;&#8364;&#8220;myc12 signal quantitated PhosphorImager ImageQuant v1.2 software, normalized background band and <br /> expressed percent signal t = 0. <br />  <br />  <br />                                                                                                    GENES &amp; DEVELOPMENT              2591 <br />  Kaiser et al. <br />  <br />  <br />  <br />                                                                     While paper revised, similar results were <br />                                                                     reported Patton et al. (1998). Whereas F-box motif <br />                                                                     Met30 suggests direct interaction Skp1 the <br />                                                                     basis previous work, evidence direct <br />                                                                     physical contact Met30 Cdc53 Cdc34. By <br />                                                                     use insect cells expressing various combinations of <br />                                                                     Cdc4, Cdc53, Skp1, Cdc34, direct contact between <br />                                                                     Cdc4/Skp1, Skp1/Cdc53, Cdc53/Cdc34, respec- <br />                                                                     tively, proposed (Skowyra et al. 1997). We envision a <br /> Figure 5. Met30 interacts physically Swe1. Cells          similar situation, Met30 replacing Cdc4 F- <br /> grown YEP raffinose OD600 &#226;&#8240;&#8230; 0.3 expression          box protein complex. This model suggests a <br /> Swe1(myc)12, Clb2 GST&#226;&#8364;&#8220;Met30 induced addition          nonspecific role Skp1, Cdc53, Cdc34 in <br /> galactose 3 hr. As control, strain expressing GST instead   SCF-mediated proteolysis, Cdc53 Skp1 serving <br /> of GST&#226;&#8364;&#8220;Met30 processed parallel. Protein complexes <br />                                                                     principally bridge F-box proteins the <br /> were purified glutathione beads parallel samples were <br />                                                                     ubiquitin-conjugating enzyme Cdc34. The F-box pro- <br /> separated SDS&#226;&#8364;&#8220;polyacrylamide gels. After immunoblot- <br /> ting, blots probed polyclonal antibody        teins represent specificity factors physi- <br /> directed GST detect GST&#226;&#8364;&#8220;Met30 GST, respec-           cally interact substrate proteins enable their <br /> tively. The blot analyzed 9E10 antibody          ubiquitination Cdc34. On basis current data, <br /> against myc epitope Swe1. (Asterisk) A protein crossre-      precise molecular organization SCF complexes in <br /> acting anti-GST antibody.                                  vivo clear. For example, estab- <br />                                                                     lished individual SCF complexes consist one <br /> a result polyubiquitination, added mutant ubiqui-             molecule Cdc53, Skp1, F-box protein, or <br /> tin (Ubi-R48) reaction. Mutation lysine 48             complexes higher stoichiometry exist. <br /> arginine impairs chain elongation polyubiquitination                Little known regulation SCF activity. <br /> reactions lysine 48 major site isopeptide         Two-hybrid experiments F-box protein Grr1 <br /> linkage ubiquitin molecules (Chau et al. 1989).             shown Grr1/Skp1 interaction modulated <br /> Addition Ubi-R48 vitro assay significantly             carbon source (Li Johnston 1997), suggesting that <br /> reduced mobility shift Swe1 response incu-             complex formation Skp1 F-box compo- <br /> bation wild-type extract, indicating             nent subjected regulation, consequently in- <br /> slowly migrating Swe1 forms correspond polyubiqui-               fluencing rate substrate proteolysis. <br /> tinated species (Fig. 6A). To address question a <br /> requirement Cdc34 Met30 function Swe1 <br /> polyubiquitination, prepared extracts cdc34 and <br /> met30 mutants. Whereas incubation Swe1 wild- <br /> type cell extracts resulted accumulation poly- <br /> ubiquitinated species Swe1, change Swe1 mo- <br /> bility observed incubation mutant <br /> extract (Fig. 6B). This deficiency catalyze Swe1 poly- <br /> ubiquitination result impaired Cdc34 Met30 <br /> activity, respectively, result general <br /> inactivity extracts, Swe1 ubiquitination is <br /> restored mutant extracts combined. <br /> Cdc34 Met30 functions required for <br /> polyubiquitination Swe1 vitro. <br />                                                                     Figure 6. Swe1 vitro ubiquitination. (A) Swe1 ubiquiti- <br />                                                                     nated cell extracts. 35S-labeled Swe1 incubated <br /> Discussion                                                          yeast cell extracts 23&#194;&#176;C (lanes 2,3) ice (lane <br /> The F-box components SCF target distinct                         1) presence ATP-regenerating ubiquitin <br />                                                                     (lanes 1,2) mutant R48-ubiquitin (lane 3), respectively. The <br /> substrates Cdc34-dependent ubiquitination <br />                                                                     reaction mix separated SDS-PAGE analyzed a <br /> pathway <br />                                                                     PhosphorImager. (B) Swe1 ubiquitination defective cdc34 <br /> We isolated MET30 genetic screen basis           met30 mutants. In vitro ubiquitination activity extracts <br /> strong specific interaction temperature-sensitive            prepared cdc34-3 met30-6 mutants compared with <br /> cdc34 mutation. The presence F-box motif                   wild-type extracts. (Lane 5) Complementation experiment in <br />                                                                     mutant extracts mixed (ratio, 2:1 = cdc34- <br /> Met30 immediately suggested Met30 part <br />                                                                     3:met30-6). Extracts preincubated 23&#194;&#176;C 5 min to <br /> of E3 complex targeting selected proteins                 allow complex formation. 35S-labeled Swe1, ATP-regenerating <br /> Cdc34-dependent ubiquitination pathway. In paper,              ubiquitin added reactions incubated <br /> we demonstrate existence E3 complex               23&#194;&#176;C (lanes 2&#226;&#8364;&#8220; 6) ice (lane 1). No extract added to <br /> vivo showing Met30 physically associates               reaction shown lane 6. Reaction products analyzed <br /> Skp1 Cdc53 refer complex SCFMet30.               described A. <br />  <br />  <br /> 2592        GENES &amp; DEVELOPMENT <br />                                                                                          Met30 required Swe1 proteolysis <br />  <br />  <br />  <br />   On basis current model SCF-mediated           sequence comparison revealed Met30 is, fact, the <br /> ubiquitination, ruled ubiquitin con-    closest budding yeast relative Slimb (Fig. 7). Although <br /> jugation substrates catalyzed E2 en-        budding yeast does close homolog b- <br /> zymes Cdc34. It shown glucose-       catenin, conceivable Met30 Slimb target <br /> induced HXT1 expression depends functional Cdc53           proteins conserved evolu- <br /> and Skp1, Cdc34 (Li Johnston 1997).            tion metazoa yeast. It remains deter- <br />                                                               mined, example, metazoan homologs of <br />                                                               Swe1 targeted SCF complex containing Slimb <br /> Targets Met30 <br />                                                               (see below). <br /> MET30 originally identified screen mutants          We report Swe1 stabilized met30 mu- <br /> defective transcriptional repression MET25 (Thom-       tants, suggesting Cdc28-inhibitory kinase de- <br /> as et al. 1995). This screen resulted isolation   graded Met30-dependent manner. However, MET30 <br /> dominant allele, MET30-1, shown impair ad-        essential gene, deletion mutants sev- <br /> enosylmethionine-mediated transcriptional regulation.         eral different temperature-sensitive mutants arresting <br /> Furthermore, two-hybrid interaction demon-         predominantly G1 unbudded cells. Defects re- <br /> strated Met30 transcriptional activator       pression genes involved methionine biosynthesis or <br /> of MET25, Met4 (Thomas et al. 1995). These results, to-       failure degrade Swe1 explain cell cycle <br /> gether experiments link Met30 ubiq-        arrest phenotype, suggesting proteins sub- <br /> uitin-dependent proteolysis, suggest Met4 possible       jected Cdc34-dependent ubiquitination Met30. <br /> target Met30-mediated degradation. The constitutive           All budding yeast F-box proteins analyzed ge- <br /> transcription MET25 observed result           netically date appear involved proteoly- <br /> hyperaccumulation Met4 linked defect Met30-        sis proteins. Remarkably, seem <br /> dependent proteolysis. However, Met4 stabilization         target simultaneously proteins implicated in <br /> met30 mutants remains shown.                            cell cycle regulation control general metabolism. <br />   Recent work Hedgehog Wingless signaling          For example, Cdc4 implicated ubiquitina- <br /> pathway Drosophila suggests F-box protein         tion cell cycle regulatory proteins Sic1, Far1, and <br /> Slimb required degradation b-catenin        Cdc6 (McKinney et al. 1993; Drury et al. 1997; Feldmann <br /> other proteins. It proposed Slimb ho-      et al. 1997; Henchoz et al. 1997; Skowyra et al. 1997) <br /> molog S. cerevisiae Cdc4 (Jiang Struhl 1998),      transcriptional activator Gcn4 (D. Kor- <br />  <br />  <br />  <br />  <br />                                                                                  Figure 7. Met30 closely related Dro- <br />                                                                                  sophila Slimb. (A) Dendrogram illustrat- <br />                                                                                  ing Met30 closely related to <br />                                                                                  Slimb human homolog h-bTrCP <br />                                                                                  (Margottin et al. 1998) Cdc4. The <br />                                                                                  dendrogram generated Gene- <br />                                                                                  Works 2.5 software. (B) Met30 Slimb <br />                                                                                  sequences aligned Clustalw. <br />                                                                                  Identical amino acids shown white <br />                                                                                  letters black background, related <br />                                                                                  residues black letters gray back- <br />                                                                                  ground. <br />  <br />  <br />                                                                                         GENES &amp; DEVELOPMENT              2593 <br />  Kaiser et al. <br />  <br />  <br />  <br /> nitzer, pers. comm.). Similarly, Grr1 required pro-       medium standard yeast genetic methods used (Guthrie <br /> teolysis G1 cyclins Cln1 Cln2 (Barral et al.          Fink 1991). <br /> 1995) unknown target mediates glu- <br />                                                                  Enhancer screen cloning MET30 <br /> cose repression. Thus, recognition targeting par- <br /> ticular F-box protein appears depend protein          A diploid cdc34-3 strain transformed different <br /> structure function. Surprisingly though,           pools mTn-3xHA/lacZ transposon mutagenized yeast ge- <br /> are obvious sequence homologies relating substrates           nomic library resulting random heterozygous gene disrup- <br />                                                                  tions marked URA3 gene (Burns et al. 1994, Ross et al. <br /> of particular F-box protein. Thus, mechanism <br />                                                                  1997). A total 8000 uracil prototrophic transformants were <br /> whereby substrate specificity achieved remains be <br />                                                                  replica plated YEPD plates incubated semipermissive <br /> determined.                                                      temperature (32&#194;&#176;C). Colonies failed grow 32&#194;&#176;C were <br />                                                                  picked master plate, retested, sporulated. The dis- <br /> SCF-mediated proteolysis entry mitosis                  rupted genes mutants showed cosegregation lethal- <br />                                                                  ity uracil prototrophy identified rescue mTn- <br /> In eukaryotes, entry mitosis controlled          3xHA/lacZ insertion including flanking regions (see http://yc- <br /> phosphorylation dephosphorylation cyclin-depen-           mi.med.yale.edu/YGAC/home.html). The MET30 ORF <br /> dent kinases. Wee1-like kinases phosphorylate Cdks            including 1000 bp up- downstream amplified PCR <br /> conserved tyrosine residues, inhibiting activ-        (primer PK55, 58-GTCTGGTATACCGACACAG-38, PK56, <br /> ity. Dephosphorylation Cdc25 phosphatases results          58-CCTCGGATTCAT ACTGGAA-38) 3.2-kb XbaI&#226;&#8364;&#8220;SphI <br />                                                                  fragment subcloned puc19 (resulting plasmid Pp70) <br /> activation Cdk/cyclin complexes triggers en- <br />                                                                  YCplac33 (resulting plasmid Pp88; Gietz Sugino <br /> try mitosis. Our results demonstrate Swe1,         1988). To disrupt MET30, HincII&#226;&#8364;&#8220;BglII fragment containing <br /> budding yeast homolog Wee1, degraded Met30-           entire coding region replaced DNA fragment con- <br /> dependent manner. Wee1 regulated Met30               ferring kanamycin G418 resistance [HincII&#226;&#8364;&#8220;HpaI fragment <br /> homologs eukaryotes, proteins highly            plasmid kanMX2 (Wach et al. 1994)]. <br /> homologous Met30 eukary- <br /> otes, including mammals. Furthermore, analyzing Wee1             Construction temperature-sensitive met30 mutants <br /> protein sequences various organisms (human,                 Temperature-sensitive met30 alleles isolated following a <br /> Xenopus, Drosophila, fission yeast), identified        procedure similar described Muhlrad et al. (1992). <br /> least PEST motif protein, suggesting            Briefly, primers PK55 PK56 used amplify MET30 <br /> they degraded Cdc34&#226;&#8364;&#8220;SCF pathway. In fact,             mutagenic conditions (1 mM dCTP, dTTP, 0.2 mM dGTP, <br /> synchrony experiments Schizosaccharomyces pombe               dATP, 30 pmole primer, 7 mM MgCl2, 0.5 mM MnCl2, 20 <br /> have shown Wee1 protein levels greatly reduced          mM Tris-HCl pH 8.4, 50 mM KCl, 0.5 units Taq DNA <br />                                                                  polymerase total volume 100 &#194;&#181;l). The PCR product was <br /> at onset mitoses despite constant transcript levels <br />                                                                  cotransformed PstI-cut Pp88 strain disrupted for <br /> (Aligue et al. 1997).                                            MET30 kept alive GAL1&#226;&#8364;&#8220;MET30 inducible allele. The <br />    The abruptness G2 /M transition thought       PstI digestion removes MET30 coding region the <br /> the result positive feedback loops, cyclin B/Cdk         gap repaired vivo homologous recombination the <br /> activation leading Wee1 inactivation Cdc25 acti-          mutagenized MET30 PCR fragment. Transformants grown <br /> vation. It possible cyclin B/Cdk-mediated phos-          glucose plates 25&#194;&#176;C, replica plated colonies unable to <br /> phorylation Wee1 creates signal ubiquitination          grow 37&#194;&#176;C selected. Plasmids containing temperature- <br /> and leads Wee1 degradation. Thus, SCF-mediated pro-           sensitive met30 alleles rescued, XbaI&#226;&#8364;&#8220;SphI fragment con- <br /> teolysis play significant role regulation        taining gene subcloned YIplac204 (Gietz and <br /> the phosphorylation state cyclin-dependent kinases,           Sugino 1988), met30 mutant alleles inte- <br />                                                                  grated TRP1 locus. <br /> contributing control entry mitosis. <br />    In budding yeast, presence Swe1 does affect        Immunoprecipitations <br /> cell cycle progression unperturbed cell cycle. <br />                                                                  Strains expressing carboxy-terminal epitope-tagged proteins <br /> However, cell polarity impaired, Swe1 intro- <br />                                                                  constructed inserting DNA sequence encoding the <br /> duces prolonged G2 delay, ascribed           epitope tag stop codon. All tagged proteins were <br /> a checkpoint monitoring cell morphogenesis (Lew                  functional rescued strain deleted corre- <br /> and Reed 1995a,b; Sia et al. 1996). Our findings suggest         sponding gene. Amino-terminal GST-tagged Skp1 ex- <br /> that Met30-dependent ubiquitination contributes               pressed control CUP1 promoter single <br /> maintaining low Swe1 levels unperturbed cells,         copy plasmid. An XbaI&#226;&#8364;&#8220;KpnI fragment (<span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> plasmid <br /> will interesting determine pathway            pYEXTM4T, Amrad Biotech) containing CUP1 promotor in <br /> may blocked response morphogenesis check-           GST-coding region ligated XbaI&#226;&#8364;&#8220;KpnI- <br /> point.                                                           digested YCplac33 (resulting plasmid Pp87; Gietz Sugino <br />                                                                  1988). The Skp1 coding region amplified PCR primers <br />                                                                  added BamHI site end cloned single <br /> Materials methods                                            BamHI site Pp87. Expression GST&#226;&#8364;&#8220;Skp1 induced by <br />                                                                  addition CuSO4 final concentration 100 &#194;&#181;M. For im- <br /> Yeast strains culture <br />                                                                  munoprecipitations cells broken glass beads buffer <br /> All yeast strains isogenic 15DaubD, bar1D ura3Dns      A (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.1% NP40, 10 mM <br /> derivative BF264-15D (MATa ade1 his2 leu2-3,112 trp1-1a)      sodium pyrophosphate, 5 mM EDTA, 5 mM EGTA, 0.1 mM or- <br /> (Reed et al. 1985). All strains grown standard culture   thovanadate, 1 mM PMSF, 2 &#194;&#181;g/ml aprotinin, leupeptin, and <br />  <br />  <br /> 2594        GENES &amp; DEVELOPMENT <br />                                                                                                  Met30 required Swe1 proteolysis <br />  <br />  <br /> pepstatin) 5 &#195;&#8212; 1 min vortexing 1-min cooling intervals     Preparation ubiquitination-competent yeast extracts <br /> on ice. Cell debris sedimented 15 min 14000g and <br />                                                                    All extracts prepared cells deleted PEP4 gene. <br /> protein concentration determined absorption 280 nm. <br />                                                                    Cells grown OD600 &#226;&#8240;&#8230; 1, washed twice ice cold <br /> A total 500 &#194;&#181;g protein incubated 12CA5 antibod- <br />                                                                    water, cell pellet resuspended YEB [30 mM <br /> ies (ascites fluid) directed HA epitope (covalently <br />                                                                    HEPES pH 7.3, 100 mM sodium acetate, 1 mM EDTA, 1 mM <br /> coupled protein A beads) glutathione beads, respectively. <br />                                                                    EGTA, 10% glycerol, 2 mM DTT, proteinase inhibitor cock- <br /> Unbound proteins removed washes 1 ml of <br />                                                                    tail (Complete minus EDTA, Boehringer)]. Sodium acetate was <br /> buffer A, bound proteins eluted boiling 100 &#194;&#181;l of <br />                                                                    added final concentration 200 mM cells lysed <br /> gel-loading buffer. Proteins separated SDS-PAGE and <br />                                                                    French press. Extracts centrifuged 10 min 27000g <br /> analyzed immunoblotting anti-HA (12CA5, BAbCO) <br />                                                                    15 min 136000g. After filtration 45-&#194;&#181;m filter, <br /> anti-RGS6H (Qiagen), anti-GST antibodies (prepared im- <br />                                                                    extracts desalted G25 column (8 ml) pre- <br /> munizing rabbits purified GST purified Escherichia <br />                                                                    equlibrated YEB concentrated amicon ultra- <br /> coli), respectively. <br />                                                                    filtration cell (membrane YM10). Alliquots extracts (30&#226;&#8364;&#8220;45 <br />                                                                    &#194;&#181;g/&#194;&#181;l) flash frozen liquid nitrogen stored &#226;&#710;&#8217;70&#194;&#176;C. <br /> Construction GAL&#226;&#8364;&#8220;SWE1&#226;&#8364;&#8220;myc12 plasmid <br /> A 3.7-kb XbaI&#226;&#8364;&#8220;BamHI fragment pSWE1-HA (Booher et <br /> al.1993) containing GAL1&#226;&#8364;&#8220;SWE1&#226;&#8364;&#8220;HA cloned altered        In vitro ubiquitination assay <br /> version pRS306 polylinker sites KpnI       35 <br />                                                                      S-Labeled Swe1 produced vitro transcription/trans- <br /> SmaI deleted. The NheI site HA tag sequence con-        lation PCR fragment containing SWE1 coding region <br /> verted SalI site, DNA digested NheI        T7 promoter (TNT T7 Quick Coupled Transcription/ <br /> ligated 100-fold molar excess synthetic oligo-       Translation System, Promega). Mutant ubiquitin (R48) ex- <br /> nucleotide 58-CTAGCGTCGACG-38, anneals itself          pressed E. coli purified described (Beers Callis <br /> generate SalI site ligates NheI site. A 400-bp     1993). <br /> XhoI&#226;&#8364;&#8220;SalI fragment containing 12 tandem copies myc            Ubiquitination reactions (total volume 10 &#194;&#181;l) contained 50 &#194;&#181;g <br /> epitope (P. Russell, TSRI) cloned generated SalI      extract 75 &#194;&#181;g case complementation experiment (50 <br /> site. The construct digested StuI integration      &#194;&#181;g cdc34-3 + 25 &#194;&#181;g met30-6), 5 &#194;&#181;g ubiquitin (Sigma), 1 &#194;&#181;l 10&#195;&#8212; <br /> URA3.                                                              ATP regenerating (10 mM ATP, 600 mM creatine phos- <br />                                                                    phate, 10 mM magnesium acetate, 1.5 mg/ml creatine kinase in <br /> Determination Swe1 stability                                    YEB), 2 &#194;&#181;l 5&#195;&#8212; SCF-buffer (25 mM magnesium acetate, 5 mM <br /> Cells containing GAL1&#226;&#8364;&#8220;SWE1&#226;&#8364;&#8220;myc12 construct                DTT, 5 &#194;&#181;M okadaic acid, 150 nM MG132 (Calbiochem), and <br /> grown YEPS (1% yeast extract, 2% Bacto-peptone, 2% su-          proteinase inhibitor cocktail (Complete minus EDTA, Boeh- <br /> crose) 23&#194;&#176;C shifted 37&#194;&#176;C 2 hr. Galactose    ringer) 1.5 &#194;&#181;l 35S-labeled Swe1. Reactions incubated <br /> added 2% cells incubated 10 min      23&#194;&#176;C 5&#226;&#8364;&#8220;15 min terminated addition gel-loading <br /> to induce Swe1&#226;&#8364;&#8220;myc12 expression. Cells harvested         buffer. Samples separated 7.5% SDS&#226;&#8364;&#8220;polyacrylamide <br /> by centrifugation, resuspended 108 cells/ml labeling me-     gel analyzed exposing gel PhosphorImaging <br /> dium [yeast nitrogen base minus methionine, 2% sucrose,            screen. <br /> and 2% galactose 0.25 mCi/ml (0.183 mM) Trans35S-<span id='am-4' about='rdfs:label' typeof='owl:Thing'>Label</span> <br /> (ICN)], incubated 10 min 37&#194;&#176;C. Labeled <br /> cells collected filtration, washed prewarmed          Acknowledgments <br /> YEPD (1% yeast extract, 2% Bacto-peptone, 2% dextrose), <br />                                                                    We grateful C. Wittenberg pointing F-box in <br /> and resuspended 3 &#195;&#8212; 107 cells/ml fresh YEPD sup- <br />                                                                    Met30, D. Finley S. Sadis GST&#226;&#8364;&#8220;MET30 6xHIS- <br /> plemented 3 mM methionine 0.5% casamino acids             ubiquitin(R48) expression plasmids helpful discussions dur- <br /> to prevent labeling. Incubation continued           ing course work. We thank M. Snyder P. Ross- <br /> 37&#194;&#176;C 4 hr. Aliquots cells diluted ice-cold        Mcdonald generous gift transposon mutagenized <br /> 10 mM NaN3, harvested centrifugation, washed ice-          yeast library. We thank K. Flick, S. Haase, C. Wittenberg for <br /> cold 10 mM NaN3, frozen &#226;&#710;&#8217;80&#194;&#176;C. To determine Swe1            helpful suggestions manuscript members the <br /> stability synchronized cultures, cells grown YEPS       Scripps-cell cycle group McGowan, Reed, Russell, and <br /> at 23&#194;&#176;C, arrested addition 200 mM hydroxyurea 3 hr       Wittenberg laboratories stimulating environment and <br /> (&gt;80% accumulation large budded cells), shifted        constructive discussions. P.K. supported Max <br /> to 37&#194;&#176;C additional hour inactivate tempera-          Kade Fellowship. This work supported National Insti- <br /> ture-sensitive alleles. Cells processed described    tutes Health grant GM-38328 S.I.R. U.S. Public Health <br /> but presence 200 mM hydroxyurea. Cell pellets       Service grant GM-53050 funds Searle Scholars <br /> lysed ice-cold 50 mM Tris (pH 7.5), 5 mM EDTA, 1 mM so-         Program/The Chicago Community Trust D.J.L. <br /> dium pyrophosphate, 150 mM sodium chloride, 1% NP-40,                The publication costs article defrayed by <br /> 1 mM sodium orthovanadate, 1 mM PMSF, 1 &#194;&#181;g/ml leupeptin,           payment page charges. This article hereby <br /> and 1 &#194;&#181;g/ml pepstatin. Swe1&#226;&#8364;&#8220;myc12 immunoprecipitated           marked &#226;&#8364;&#732;&#226;&#8364;&#732;advertisement&#226;&#8364;&#8482;&#226;&#8364;&#8482; accordance 18 USC section <br /> with anti-c-Myc (9E10) antibody (Santa Cruz Biotechnology)     1734 solely indicate fact. <br /> from lysates containing equal amounts radioactive label, <br /> and immunoprecipitates washed times ly- <br /> sis buffer, boiled 10 min 1&#195;&#8212; SDS sample loading buffer, and <br />                                                                    References <br /> separated SDS-PAGE 8% polyacrylamide gels. Dried gels <br /> were exposed Molecular Dynamics storage phosphor screen         Aligue, R., L. Wu, P. Russell. 1997. Regulation Schizo- <br /> for 24&#226;&#8364;&#8220;48 hr, scanned Molecular Dynamics 445 SI Phos-            saccharomyces pombe Wee1 tyrosine kinase. J. Biol. Chem. <br /> phorImager, analyzed ImageQuant version 1.2 soft-            272: 13320&#226;&#8364;&#8220;13325. <br /> ware.                                                              Amon, A., U. Surana, I. Muroff, K. Nasmyth. 1992. Regu- <br />  <br />  <br />                                                                                                 GENES &amp; DEVELOPMENT              2595 <br />  Kaiser et al. <br />  <br />  <br />     lation p34CDC28 tyrosine phosphorylation required          Cdc16p, Cdc23p Cdc27p form complex essential for <br />     entry mitosis S. cerevisiae. Nature 355: 368&#226;&#8364;&#8220;371.        mitosis. EMBO J. 13: 4321&#226;&#8364;&#8220;4328. <br /> Bai, C., P. Sen, K. Hofman, L. Ma, M. Goebl, J.W. Harper, S.J.   Lammer, D., N. Mathias, J.M. Laplaza, W. Jiang, Y. Liu, J. Callis, <br />     Elledge. 1996. SKP1 connects cell cycle regulators            M. Goebl, M. Estelle. 1998. Modification yeast <br />     ubiquitin proteolysis machinery novel motif,           Cdc53p ubiquitin-related protein Rub1p affects func- <br />     F-box. Cell 86: 263&#226;&#8364;&#8220;274.                                             tion SCFCdc4 complex. Genes &amp; Dev. 12: 914&#226;&#8364;&#8220;926. <br /> Barral, Y., S. Jentsch, C. Mann. 1995. G1 cyclin turnover        Lanker, S., M.H. Valdivieso, C. Wittenberg. 1996. Rapid <br />     <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake controlled common pathway            degradation G1 cyclin Cln2 induced CDK-depen- <br />     yeast. Genes &amp; Dev. 9: 399&#226;&#8364;&#8220;409.                                      dent phosphorylation. Science 271: 1597&#226;&#8364;&#8220;1601. <br /> Beers, E.P. J. Callis. 1993. Utility polyhistidine-tagged     Lew, D.J. S.I. Reed. 1993. Morphogenesis yeast cell <br />     ubiquitin purification ubiquitin-protein conjugates        cycle: Regulation Cdc28 cyclins. J. Cell. Biol. <br />     affinity ligand purification ubiquitin-         120: 1305&#226;&#8364;&#8220;1320. <br />     specific hydrolases. J. Biol. Chem. 268: 21645&#226;&#8364;&#8220;21649.            &#226;&#8364;&#8221;&#226;&#8364;&#8221;&#226;&#8364;&#8221;. 1995a. A cell cycle checkpoint monitors cell morpho- <br /> Booher, R.N., R.J. Deshaies, M.W. Kirschner. 1993. Proper-           genesis budding yeast. J. Cell. Biol. 129: 739&#226;&#8364;&#8220;749. <br />     ties Saccharomyces cerevisiae wee1 differential       &#226;&#8364;&#8221;&#226;&#8364;&#8221;&#226;&#8364;&#8221;. 1995b. Cell cycle control morphogenesis budding <br />     regulation p34CDC28 response G1 G2 cyclins.             yeast. Curr. Opin. Genet. Dev. 5: 17&#226;&#8364;&#8220;23. <br />     EMBO J. 2: 3417&#226;&#8364;&#8220;3426.                                            Li, F.N. M. Johnston. 1997. Grr1 Saccharomyces cerevi- <br /> Burns, N., B. Grimwald, P.B. Ross-Macdonald, E.Y. Choi, K.               siae connected ubiquitin proteolysis machinery <br />     Finberg, S.G. Roeder, M. Snyder. 1994. Large-scale               Skp1: Coupling glucose sensing gene expression <br />     analysis gene expression, protein localization, gene          cell cycle. EMBO J. 16: 5629&#226;&#8364;&#8220;5638. <br />     disruption Saccharomyces cerevisiae. Genes &amp; Dev.             Margottin, F., S.P. Bour, H. Durand, L. Selig, S. Benichou, V. <br />     8: 1087&#226;&#8364;&#8220;1105.                                                        Richard, D. Thomas, K. Strebel, R. Benarous. 1998. A <br /> Chau, V., J.W. Tobias, A. Bachmair, D. Marriott, D.J. Ecker,             novel human WD protein, h-bTrCP, interacts with <br />     D.K. Gonda, A. Varshavsky. 1989. A multiubiquitin                HIV-1 Vpu connects CD4 ER degradation pathway <br />     chain confined specific lysine targeted short-lived       F-box motif. Mol. Cell 4: 565&#226;&#8364;&#8220;574. <br />     protein. Science 243: 1576&#226;&#8364;&#8220;1583.                                 Marini, N.J., E. Meldrum, B. Buehrer, A.V. Hubberstey, D.E. <br /> Deshaies, R.J., V. Chau, M.W. Kirschner. 1995. Ubiquitina-           Stone, A. Traynor-Kaplan, S.I. Reed. 1996. A pathway in <br />     tion G1 cyclin Cln2p Cdc34p-dependent path-            yeast cell division cycle linking protein kinase C (Pkc1) <br />     way. EMBO J. 14: 303&#226;&#8364;&#8220;312.                                            activation Cdc28 START. EMBO J. 15: 3040&#226;&#8364;&#8220;3052. <br /> Drury, L.S., G. Perkins, J. Diffley. 1997. The Cdc4/34/53        Mathias, N., S.L. Johnson, M. Winey, A.E.M. Adams, L. <br />     pathway targets Cdc6p proteolysis budding yeast.              Goetsch, J.R. Pringle, B. Byers, M.G. Goebl. 1996. <br />     EMBO J. 16: 5966&#226;&#8364;&#8220;5976.                                               Cdc53p acts concert Cdc4p Cdc34p control <br /> Dunphy, W.G. 1994. The decision enter mitosis. Trends Cell            G1-to-S-phase transition identifies conserved fam- <br />     Biol. 4: 202&#226;&#8364;&#8220;207.                                                    ily proteins. Mol. Cell. Biol. 16: 6634&#226;&#8364;&#8220;6643. <br /> Feldmann, R.M.R., C.C. Correll, K.B. Kaplan, R.J. Deshaies.      McKinney, J.D., F. Chang, N. Heintz, F.R. Cross. 1993. <br />     1997. A complex Cdc4p, Skp1p, Cdc53p/cullin cata-             Negative regulation FAR1 start cell cycle. <br />     lyzes ubiquitination phosphorylated CDK inhibitor             Genes &amp; Dev. 7: 833&#226;&#8364;&#8220;843. <br />     Sic1p. Cell 91: 221&#226;&#8364;&#8220;230.                                         Muhlrad, D., R. Hunter, R. Parker. 1992. A rapid method for <br /> Gietz, R.D. A. Sugino. 1988. New yeast-Escherichia coli              localized mutagenesis yeast genes. Yeast 8: 79&#226;&#8364;&#8220;82. <br />     shuttle vectors constructed vitro mutagenized yeast      Patton, E.E., A.R. Willems, D. Sa, L. Kuras, D. Thomas, K.L. <br />     genes lacking six-base pair restriction sites. Gene 74: 527&#226;&#8364;&#8220;         Craig, M. Tyers. 1998. Cdc53 scaffold protein for <br />     534.                                                                 multiple Cdc34/Skp1/F-box protein complexes regulate <br /> Guthrie, C. G.R. Fink. 1991. Guide yeast genetics             cell division methionine biosynthesis yeast. Genes &amp; <br />     molecular biology, methods enzymology, Vol. 194. Aca-             Dev. 12: 692&#226;&#8364;&#8220;705. <br />     demic Press, San Diego, CA.                                      Piatti, S., T. Bo&#194;&#168;hm, J.H. Cocker, J.F.X. Diffley, K. Nasmyth. <br /> Henchoz, S., Y. Chi, B. Catarin, I. Herskow <br /> </body></html>