<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                 GBE <br />  <br /> Evolutionary Rate Heterogeneity Core Attachment <br /> Proteins Yeast Protein Complexes <br /> Sandip Chakraborty Tapash Chandra Ghosh* <br /> Bioinformatics Centre, Bose Institute, Kolkata, West Bengal, India <br /> *Corresponding author: E-mail: tapash@jcbose.ac.in. <br /> Accepted: June 23, 2013 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> In general, proteins work alone; form macromolecular complexes play fundamental roles diverse cellular functions. <br /> On basis iterative clustering procedure frequency occurrence macromolecular complexes, protein <br /> subunits categorized core attachment. Core protein subunits main functional elements, attach- <br /> ment proteins act modifiers activators protein complexes. In article, using current data set yeast protein complexes, <br /> we core proteins evolving faster rate attachment proteins spite functional importance. Interestingly, <br /> our investigation revealed attachment proteins present higher number macromolecular complexes core proteins. <br /> We observed protein complex number (defined number protein complexes protein subunit belongs) <br /> has stronger influence gene/protein essentiality multifunctionality. Finally, results suggest observed differences <br /> in rates protein evolution core attachment proteins differences protein complex number and <br /> expression level. Moreover, conclude proteins present higher numbers macromolecular complexes enhance <br /> their overall expression level increasing transcription rate translation rate, protein complex number <br /> imposes strong selection pressure evolution yeast proteome. <br /> Key words: core protein, attachment protein, evolutionary rate, yeast, protein complex, protein complex number. <br />  <br />  <br /> Introduction <br /> Different proteins evolve different rates. The central prob-                                researchers (Batada et al. 2006). On hand, <br /> lem molecular evolution comprehend factors                                    demonstrated gene expression level an <br /> the differential evolutionary rates proteins. The origins                                important constraint protein evolution (Subramanian and <br /> variation protein evolutionary rates remained core                                 Kumar 2004; Drummond et al. 2005, 2006; Drummond and <br /> issue selectionists neutralists (Fisher                                    Wilke 2008); highly expressed proteins conserved <br /> 1930; Kimura Ohta 1974; Kimura 1983; Razeto-Barry                                          lowly expressed proteins. This phenomenon been <br /> et al. 2011). The neutral theory molecular evolution accen-                                 attributed natural selection (Popescu et al. 2006). <br /> tuates nucleotide substitutions gene                                 Moreover, shown half variation in <br /> due random fixation neutral mutations (Kimura                                        protein evolutionary rates explained expression level <br /> 1983), accordance proposal function-                                 (Drummond et al. 2005). <br /> ally important genes evolve slower important                                      Proteins carry functions alone, often <br /> genes (Kimura Ohta 1974). According proposal,                                   act participating macromolecular complexes play <br /> has demonstrated essential genes evolve slower                                       different functional roles (Qiu Noble 2008; Chakraborty <br /> than genes dispensable (Jordan et al. 2002).                                     et al. 2010). Additionally, multipro- <br /> Moreover, proteins interact large number part-                                 tein complexes enriched essential genes (Semple et al. <br /> ners evolve slower rate compared proteins                                   2008), proteins present protein com- <br /> fewer number interacting partners (Hirsh Fraser 2001;                                   plex assemblies tendency essential than <br /> Jordan et al. 2002; Fraser et al. 2003; Hahn Kern 2005;                                    proteins present fewer protein complex assemblies <br /> Lemos et al. 2005; Chakraborty et al. 2011; Alvarez-Ponce                                      (Pereira-Leal et al. 2006). In contrast, Pache et al. (2009) <br /> and Fares 2012). However, results questioned                                   reported significant correlation between <br />  <br />  <br /> &#195;&#376; The Author(s) 2013. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), permits <br /> non-commercial re-use, distribution, reproduction medium, provided original work properly cited. For commercial re-use, contact journals.permissions@oup.com <br />  <br />  <br />  <br />  <br /> 1366 Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013 <br />  Evolutionary Rates Yeast Protein Complexes                                                                       GBE <br />  <br /> gene essentiality number protein complex assemblies       attuned natural selection protein evolution <br /> which protein present. In previous studies,        (Popescu et al. 2006). <br /> demonstrated complex-forming proteins evolve more <br /> slowly noncomplex forming proteins (Teichmann <br />                                                                     Materials Methods <br /> 2002), proteins associated large number <br /> of complexes evolutionary conserved             Protein-Complex Information <br /> proteins associated fewer numbers com-            The yeast protein complex data Gavin et al. (2006) was <br /> plexes (Chakraborty et al. 2010; Das et al. 2013). Moreover,        collected supplementary data set Pache et al. <br /> it reported exists significant positive       (Additional file 3: Gavin complexes: http://www.biomedcen <br /> correlation expression level protein complex            tral.com/1752-0509/3/74/additional, accessed July 15, <br /> number (i.e., number protein complex assemblies           2013) (Pache et al. 2009). There 491 complexes with <br /> which protein present) yeast (Chakraborty et al.             1,487 unique protein subunits. Gavin et al. (2006) used the <br /> 2010), proposed proteins associated       affinity purification technique coupled mass spectrome- <br /> with large number complexes (i.e., higher protein com-         try purify subunits protein complexes. After repeated <br /> plex number) need produced greater quantities          purification yeast protein complexes, found <br /> hence higher expression level.                                 protein complexes contain different pro- <br />    Additional studies protein complex assembly tell       tein subunits termed &#226;&#8364;&#339;complex isoform.&#226;&#8364;? <br /> about architecture function different protein        Proteins complex isoform classified into <br /> subunits (Gavin et al. 2006). <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> perspective         major classes clustering property, is, core <br /> topology complex assemblies, protein subunits catego-        attachment protein. In data set, 357 and <br /> rized mainly different groups, is, core attach-     339 proteins core attachment, respectively; however, <br /> ment (Gavin et al. 2006; Pang et al. 2008). Core proteins       791 proteins act core attachment protein <br /> always present isoforms execute main functions       subunit complex isoforms (supplementary table S1, <br /> (Gavin et al. 2006), attachment proteins present        Supplementary Material online). The protein complex <br /> only isoforms act modifiers com-       number defined number protein complex assem- <br /> plex&#226;&#8364;&#8482;s function (Dezso et al. 2003). Analysis subunits    blies protein present (Chakraborty et al. 2010). <br /> protein complexes revealed biochemical role core        Recently, Benschop et al. (2010) identified 518 protein com- <br /> proteins essential complex assembly        plexes, 409 determined forward&#226;&#8364;&#8220;back- <br /> are irreplaceable (Dezso et al. 2003); protein         ward module detection (i.e., FBMD). They (Benschop et al. <br /> abundance core subunits cell             2010) determined 350 consensus protein complexes by <br /> lower level attachment subunits (Pang et al. 2008).            integrating data Pu et al. (2007) Hart et al. <br /> However, recent studies (Semple et al. 2008; Pache              (2007). This consensus data set accurate other <br /> et al. 2009) showed core attachment subunits are <br />                                                                     predicted data set (Benschop et al. 2010), no <br /> equally important complex machinery function. <br />                                                                     information core attachment subunits their <br />    In work, used yeast protein complex data set <br />                                                                     data set. Thus, used consensus data sets <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (Gavin et al. 2006) classification pertaining core and <br />                                                                     compute protein complex number. <br /> attachment proteins analyzed evolutionary rates to <br /> address influences gene dispensability, protein multi- <br /> functionality, protein connectivity, protein complex number, <br />                                                                     Protein Multifunctionality <br /> and gene expression level protein evolutionary rates.            The number unique biological processes (i.e., multifunction) <br />    We report communication core attachment         (Salathe&#194;&#180; et al. 2006; Podder et al. 2009) protein was <br /> proteins equally essential <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>survival</span>. We observed       calculated Gene Ontology (GO) project (Camon <br /> that core proteins evolve faster attachment proteins        et al. 2004); yeast protein, corresponding <br /> spite higher multifunctionality, incompatible     child term GO biological processes (GO-BP) retrieved <br /> with proposal neutral theory molecular evolution.     Ensembl BioMart (version 69) (Flicek et al. 2011). <br /> Finally, demonstrated proteins present         Alternatively, used number biological pathways <br /> higher numbers macromolecular complexes increase           measure multifunctionality. To calculate unique <br /> transcription rate (number mRNA molecules hour)           pathway number, used MIPS-CYGD (Munich <br /> well translation rate (number proteins molecules          Information centre Protein Sequences-Comprehensive <br /> second). Moreover, observed attachment proteins             Yeast Genome Database) (http://mips.helmholtz-muenchen. <br /> have higher protein complex number core proteins             de/genre/proj/yeast/Search/Catalogs/searchCatfirstFun.html, <br /> and subsequently increases expression level attachment       accessed July 15, 2013) (Guldener et al. 2005; Mewes <br /> proteins. Thus, protein complex number imposes strong         et al. 2011) KEGG database (Kanehisa et al. 2012) <br /> selection pressure attachment proteins          separately. <br />  <br />  <br /> Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013                          1367 <br />  Chakraborty Ghosh                                                                                                 GBE <br />  <br /> Essential Genes Calculation Essentiality                       alignment performed using ClustalW (version 2.0) <br /> To obtain data set essential genes yeast, down-            (Larkin et al. 2007) set orthologs gene pair. dN/dS <br /> loaded essential genes list OGEE database (http://       values calculated Yang Nielsen method (Yang and <br /> ogeedb.embl.de/#summary, accessed July 15, 2013)                 Nielsen 2000) using PAML package (version 4) (Yang 2007). <br /> (Chen et al. 2012). Moreover, used fitness quantitative        We took average values dN/dS S. cerevisiae ORF <br /> data Steinmetz et al. (2002) calculate essentiality     orthologous pair present (supplemen- <br /> each gene. The fitness data downloaded Pache                tary table S2, Supplementary Material online). <br /> et al. (2009) (Additional file 3: Quantitative fitness data: <br /> http://www.biomedcentral.com/1752-0509/3/74/additional,               Protein&#226;&#8364;&#8220;Protein Interactions Data <br /> last accessed July 15, 2013) considered growth        We collected protein&#226;&#8364;&#8220;protein interactions (PPIs) informa- <br /> the YPD medium (1% Bacto-peptone, 2% yeast extract,               tion DIP (Database Interacting Proteins: http://dip. <br /> 2% glucose). The normalized essentiality (Ei) calculated       doe-mbi.ucla.edu/, accessed July 15, 2013) (Salwinski <br />                                                                       et al. 2004), MINT (Molecular INTeraction Database: http:// <br />                                     fi <br />                         Ei &#194;&#188; 1                                        mint.bio.uniroma2.it/mint/, accessed July 15, 2013) <br />                                    fmax <br />                                                                       (Licata et al. 2012), IntAct (http://www.ebi.ac.uk/intact, <br /> where fi represents fitness values deletion ith    accessed July 15, 2013) (Kerrien et al. 2012). In those <br /> gene fmax represents maximum fitness value.                   databases, PPIs documented experimentally by <br /> Therefore, gene essentiality ranges 0 1,          genome wide two-hybrid screen, immune precipitation, affin- <br /> essentiality 0 denoting gene measurable effect       ity binding, antibody blockage, on. To collect high- <br /> on fitness, 1 denotes gene essential survival.     throughput PPIs data DIP database, used the <br />                                                                       CORE data set S. cerevisiae (baker&#226;&#8364;&#8482;s yeast) <br /> Calculation Evolutionary Rate                                      (Scere20120228CR). In CORE data set, PPIs iden- <br />                                                                       tified high-throughput methods small-scale experi- <br /> We calculated average dN/dS ratio measure evolu- <br />                                                                       ments, data CORE highly reliable (Deane <br /> tionary rate, ratio number nonsynony- <br />                                                                       et al. 2002). However, predefined high confidence <br /> mous substitutions nonsynonymous site (dN) number <br />                                                                       data MINT IntAct, used confidence score to <br /> of synonymous substitutions synonymous site (dS). To cal- <br />                                                                       collect high-throughput data MINT IntAct. We <br /> culate dN/dS ratio, compared S. cerevisiae sequences <br />                                                                       calculated average confidence score data set and <br /> with orthologous sequences S. paradoxus, S. bayanus, and <br />                                                                       used average value cutoff identify high- <br /> S. mikatae, sibling species (Naumov et al. 1992). We <br />                                                                       throughput PPIs (for MINT, cutoff score 0.30, for <br /> obtained coding sequences yeast strains the <br />                                                                       IntAct, cut score 0.40). After merging three <br /> Saccharomyces Genome Database (Cherry et al. 2012) (for <br />                                                                       databases (DIP, MINT, IntAct) removing redun- <br /> S. cerevisiae: http://downloads.yeastgenome.org/sequence/ <br />                                                                       dant self-interactions, obtained 3,580 individual pro- <br /> S288C_reference/orf_dna/orf_coding.fasta.gz [last accessed <br />                                                                       teins 12,624 binary interactions (fig. 1) (supplementary <br /> July 15, 2013]; S. paradoxus: http://downloads.yeastgenome. <br />                                                                       table S3, Supplementary Material online). <br /> org/sequence/fungi/S_paradoxus/archive/MIT/orf_dna/orf_ge <br /> nomic.fasta.gz [last accessed July 15, 2013]; S. bayanus: http:// <br />                                                                       Protein Expression, Transcription Rate, <br /> downloads.yeastgenome.org/sequence/fungi/S_bayanus/ar <br />                                                                       Translation Rate <br /> chive/MIT/orf_dna/orf_genomic.fasta.gz [last accessed July 15, <br /> 2013]; S. mikatae: http://downloads.yeastgenome.org/              The gene expression (number mRNAs molecules cell) and <br /> sequence/fungi/S_mikatae/archive/MIT/orf_dna/orf_genomic.             transcription rate (number mRNAs molecules hour) were <br /> fasta.gz [last accessed July 15, 2013]) calculate evolution-   collected genome wide expression analysis Holstege <br /> ary rates (dN/dS). Then, translate sequences pro-       et al. (1998) (http://web.wi.mit.edu/young/pub/data/orf_tran <br /> tein sequence using EMBOSS Transeq program (http://               scriptome.txt, accessed July 15, 2013). We used a <br /> www.ebi.ac.uk/Tools/st/emboss_transeq/, accessed July            up-to-date (Miller et al. 2011) expression data set for <br /> 15, 2013). By using National Center Biotechnology             validation purposes. Dynamic transcriptome analysis was <br /> Information BlastP program (version 2.2.17) (Altschul et al.          used realistically monitor mRNA metabolism yeast <br /> 1997), identified orthologous proteins S. cerevisiae        (Miller et al. 2011). We downloaded translation rate <br /> in S. paradoxus, S. baynus, S. mikatate. We used ex-          (number proteins molecules seconds) yeast proteins <br /> pectation value 1.0   10 5 cutoff maintained        data set Arava et al. (2003). <br /> 75% sequence similarity sequences a <br /> minimum alignment overlap 80%. The gaps allowed                Software <br /> alignment 3%. We verified results             We used SPSS (version 13.0) (Nie et al. 1970) Tanagra <br /> with results Kellis et al. (2003). Then pair-wise          (version 1.4.36) (Rokotomalala 2005) statistical <br />  <br />  <br /> 1368 Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013 <br />  Evolutionary Rates Yeast Protein Complexes                                                                                    GBE <br />  <br />                                                                               units complex assembly (Dezso et al. 2003; Gavin et al. <br />                                                                               2006). Therefore, role core protein crucial for <br />                                                                               functional integrity protein complex (Dezso et al. <br />                                                                               2003). Hence, measured multifunctionality core <br />                                                                               attachment proteins counting unique GO-BP terms, <br />                                                                               widely used calculate multifunctionality a <br />                                                                               protein (Camon et al. 2004; Pa&#194;&#180;l et al. 2006; Salathe&#194;&#180; et al. <br />                                                                               2006; Podder et al. 2009; Su et al. 2010; Flicek et al. 2011; <br />                                                                               Yang Gaut 2011). We observed core proteins are <br />                                                                               involved higher number biological processes than <br />                                                                               attachment proteins (table 1). Nonetheless, GO terms, <br />                                                                               widely used functional characterization draw- <br />                                                                               backs reported associated GO terms <br />                                                                               (Dolan et al. 2005; Park et al. 2011). In particular, GO anno- <br />                                                                               tation systematically redundant biological domain is <br />                                                                               inconsistent (Park et al. 2011). Therefore, used the <br />                                                                               MIPS-CYGD (Guldener et al. 2005; Makino Gojobori <br />                                                                               2006; Chakraborty et al. 2010; Mewes et al. 2011) the <br />                                                                               KEGG (Kanehisa et al. 2012) databases, counted <br />      FIG. 1.&#226;&#8364;&#8221;Venn diagram PPI information collected DIP,          number pathways protein takes part. We <br /> MINT, IntAct. Here protein abbreviated &#226;&#8364;&#339;P&#226;&#8364;? PPI abbreviated   similar trends, is, core proteins involved a <br /> by &#226;&#8364;&#339;I.&#226;&#8364;? The &#226;&#8364;&#339;light green&#226;&#8364;? color indicates DIP data set domain, &#226;&#8364;&#339;light     higher number pathways/functions attachment <br /> blue&#226;&#8364;? color indicates MINT data set domain, &#226;&#8364;&#339;light saffron&#226;&#8364;? color <br />                                                                               proteins (table 1). However, core proteins <br /> indicates IntAct data set domain. <br />                                                                               evolve faster attachment proteins spite their <br />                                                                               higher multifunctionality. Thus prevailing idea that, <br /> calculations. In statistical analysis, used 95% level of <br />                                                                               &#226;&#8364;&#339;multifunctional proteins evolve slower rate&#226;&#8364;? (Wilson <br /> confidence measure significance. The network statistic <br />                                                                               et al. 1977; Podder et al. 2009) does compat- <br /> (Degree) calculated using Pajek software package <br />                                                                               ible explaining evolutionary rate differences between <br /> (Batagelj Mrvar 2004). <br />                                                                               core attachment proteins. <br />                                                                                   It reported gene essentiality protein <br /> Results                                                                       multifunctionality negatively associated protein <br />                                                                               evolutionary rates (Jordan et al. 2002; Podder et al. 2009). <br /> Evolutionary Rates, Multifunctionality, Essentiality of <br />                                                                               Therefore, expected essential genes are <br /> Core Attachment Proteins <br />                                                                               multifunctional. Indeed, positive association <br /> Yeast protein complexes taken supplementary                     gene essentiality multifunctionality our <br /> data set Gavin et al. (2006), comprised 1,487 pro-                   data set (Spearman&#226;&#8364;&#8482;s rmultifunctionality vs. essentiality &#194;&#188; 0.1689, <br /> teins belonging 491 protein complexes. In work, Gavin                P &#194;&#188; 1.0   10 6), is, proportion essential genes is <br /> et al. (2006) identified 357 339 subunits core                     higher highly multifunctional proteins. To validate above <br /> attachment proteins, respectively. The remaining 791 subunits                 results, divided data set groups by <br /> are present core attachment proteins. We                     using mean value GO-multifunctionality cutoff. <br /> not considered 791 proteins evolutionary rate anal-                 Genes multifunctionality higher corre- <br /> yses presence core attachment pro-                   sponding mean value considered highly functional <br /> teins. Estimation evolutionary rates core attachment                (HF) (N &#194;&#188; 436) lower one, low-functional <br /> proteins shows attachment proteins conserved                    (LF) (N &#194;&#188; 260) groups. We significantly (two-sided <br /> than core proteins (average evolutionary rates core pro-                   Fisher&#226;&#8364;&#8482;s exact test, P &#194;&#188; 1.1   10 3) higher proportion essen- <br /> teins: 0.0901 (&#195;&#190;0.0031) (N &#194;&#188; 348), attachment proteins:                       tial genes HF group (41.28%) compared LF <br /> 0.0772 (&#195;&#190;0.0040) (N &#194;&#188; 325); Mann&#226;&#8364;&#8220;Whitney U test,                              group (28.85%); indicating HF proteins essential <br /> P &#194;&#188; 2.5   10 5) (fig. 2).                                                     low functional proteins. Hence, expect a <br />    The function protein important                     higher proportion essential genes core proteins than <br /> parameters influence protein evolution,                  attachment proteins. Surprisingly, observed similar pro- <br /> reported multifunctional genes evolve slowly (Wilson                     portion essential genes core attachment proteins <br /> et al. 1977; Podder et al. 2009). Moreover, core proteins                 (two-sided Fisher&#226;&#8364;&#8482;s exact test, Core: 38.66%, Attachments: <br /> always present different subsets isoforms complex              34.51%, P &#194;&#188; 2.7   10 1). Moreover, core <br /> assembly, reported functional                  attachment proteins exhibit similar essentiality (average <br />  <br />  <br /> Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013                                        1369 <br />  Chakraborty Ghosh                                                                                                        GBE <br />  <br />  <br />  <br />  <br />    FIG. 2.&#226;&#8364;&#8221;Average evolutionary rates (dN/dS) core attachment proteins yeast. The statistical comparison performed two-tailed <br /> Mann&#226;&#8364;&#8220;Whitney U test. <br />  <br />           Table 1 <br />           Average Multifunctionality Pathways Core Attachment Proteins <br />           Database Used                       Core                              Attachment                    Signi&#239;&#172;?cance Level (P) <br />           KEGG                      1.8992 (&#195;&#190;0.1209) (N &#194;&#188; 129)           1.6994 (&#195;&#190;0.0939) (N &#194;&#188; 173)                3.3   10 2 <br />           MIPS                      2.6627 (&#195;&#190;0.0873) (N &#194;&#188; 335)           2.4174 (&#195;&#190;0.0801) (N &#194;&#188; 321)                4.4   10 2 <br />           GO-BP                     4.2068 (&#195;&#190;0.1934) (N &#194;&#188; 266)           3.3746 (&#195;&#190;0.1814) (N &#194;&#188; 283)                2.1   10 5 <br />  <br />  <br /> essentiality core proteins: 0.1496 ( &#195;&#190; 0.0091) (N &#194;&#188; 214),             variation rates protein evolution better gene <br /> attachment proteins: 0.1685 (&#195;&#190;0.0086) (N &#194;&#188; 213); Mann&#226;&#8364;&#8220;                   essentiality multifunctionality. <br /> Whitney U test, P &#194;&#188; 5.7   10 2). The neutral theory molec- <br /> ular evolution postulates functionally important genes              Protein Connectivity Evolutionary Rates Core <br /> evolve slowly important genes (Hirsh              Attachment Proteins <br /> Fraser 2001; Jordan et al. 2002; Liao et al. 2006). In our <br />                                                                          In general, biological cell considered a <br /> data set separated genes essential                  complex network, PPIs network acts <br /> nonessential genes, essential genes                   source biological functions (Mete et al. 2008). <br /> evolve slower nonessential genes (average evolutionary              Proteins interaction partners expected be <br /> rate essential genes: 0.0735 (&#195;&#190;0.0039) (N &#194;&#188; 251), nones-              multifunctional. We analyzed correlation the <br /> sential genes: 0.0904 (&#195;&#190;0.0032) (N &#194;&#188; 416); Mann&#226;&#8364;&#8220;Whitney U                number interacting partners core/attachment proteins <br /> test, P &#194;&#188; 8.5   10 5). Moreover, separated essential             (whose GO-BP annotations connectivity data avail- <br /> and nonessential genes core attachment proteins,             able) multifunctionality. As expected, a <br /> found evolutionary rates attachment proteins             significant positive correlation (Spearman&#226;&#8364;&#8482;s rGO-BP vs. connectivity <br /> significantly lower core proteins gene pools            &#194;&#188; 0.3005, P &#194;&#188; 1.0   10  6) GO-BP connectivity <br /> (fig. 2). Therefore, observed evolutionary rate differences          PPI network. Previously, reported the <br /> between core attachment proteins independent                  protein connectivity PPI network negatively correlated <br /> gene essentiality. Previously, Hurst Smith (1999) demon-             evolutionary rates (Hirsh Fraser 2001; Fraser et al. <br /> strated gene essentiality weak correlate           2002, 2003; Jordan et al. 2002; Hahn Kern 2005; Lemos <br /> rates protein evolution. They observed &#226;&#8364;&#339;non-                 et al. 2005; Chakraborty et al. 2010, 2011; Alvarez-Ponce and <br /> immune non-essential genes&#226;&#8364;? (&#226;&#8364;&#339;non-immune&#226;&#8364;? genes refers                   Fares 2012). Similar trend observed core/ <br /> to genes belong immune system)                  attachment data set (Spearman&#226;&#8364;&#8482;s rconnectivity vs. dN/ <br /> evolve similar rate &#226;&#8364;&#339;essential genes.&#226;&#8364;? Even                                         6 <br />                                                                          dS &#194;&#188;  0.1907, P &#194;&#188; 1.0   10 ). The observations moti- <br /> study observed significant differences evolution-          vated investigate exists variation connec- <br /> ary rates &#226;&#8364;&#339;essential core proteins&#226;&#8364;? &#226;&#8364;&#339;non-                    tivity core attachment proteins not. Surprisingly, we <br /> essential attachment proteins&#226;&#8364;? (average evolutionary rates               did significant difference (average connectivity of <br /> of essential core proteins: 0.0760 (&#195;&#190;0.0042) (N &#194;&#188; 137), nones-           core proteins: 8.0364 (&#195;&#190;0.3565) (N &#194;&#188; 357), attachment pro- <br /> sential attachment proteins: 0.0818 (&#195;&#190;0.0050) (N &#194;&#188; 207);                 teins: 13.8260 (&#195;&#190;2.3445) (N &#194;&#188; 339); Mann&#226;&#8364;&#8220;Whitney U test, <br /> Mann&#226;&#8364;&#8220;Whitney U test, P &#194;&#188; 1.0   10 1). These results in-              P &#194;&#188; 5.4   10 1) connectivity core attach- <br /> dicate constraints explain           ment proteins. This result indicates observed <br />  <br />  <br /> 1370 Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013 <br />  Evolutionary Rates Yeast Protein Complexes                                                                                        GBE <br />  <br />  <br />  <br />  <br />      FIG. 3.&#226;&#8364;&#8221;Average evolutionary rates (dN/dS) core attachment <br /> proteins bines. The statistical comparison performed two-tailed <br /> Mann&#226;&#8364;&#8220;Whitney U test. <br />  <br /> difference protein evolution core attachment <br /> proteins independent protein connectivity. To confirm the <br /> above observation, binned core attachment proteins <br /> into groups according connectivity PPI net- <br /> work analyzed evolutionary rates. Interestingly, we <br /> found core proteins evolving higher rate than <br /> attachment proteins PPI bin (fig. 3). Thus say                    FIG. 4.&#226;&#8364;&#8221;(a) Average protein complex number (protein complex <br />                                                                             number calculated using Gavin et al. [2006] data set) core and <br /> that observed difference evolutionary rates between <br />                                                                             attachment proteins yeast. The statistical comparison performed by <br /> core attachment proteins steered factors <br />                                                                             two-tailed Mann&#226;&#8364;&#8220;Whitney U test. (b) Average protein complex number <br /> other protein connectivity.                                        (protein complex number calculated using Benschop et al. [2010] <br />                                                                             data set) core attachment proteins yeast. The statistical compar- <br /> Protein Complex Number Evolutionary Rates                            ison performed two-tailed Mann&#226;&#8364;&#8220;Whitney U test. <br /> Core Attachment Proteins <br /> Earlier, demonstrated protein complex                      complex number 1 3, named Attach-I, the <br /> number significantly modulates rates protein evolution               rest ones Attach-II (protein complex number ranges from <br /> when compared protein connectivity PPI networks                     4 22). Comparing protein evolutionary rates between <br /> (Chakraborty et al. 2010; Das et al. 2013). Thus, calculated             core Attach-I proteins, marginally significant <br /> protein complex number core attachment pro-                     (Mann&#226;&#8364;&#8220;Whitney U test, P &#194;&#188; 5.3   10 2) differences between <br /> teins. We observed protein complex number signifi-                  (fig. 5). At point, mentioned is <br /> cantly higher attachment proteins core                      core protein complex number 3, and <br /> proteins (fig. 4a). Furthermore, reconfirmed result              core protein excluded data set and <br /> using data set protein complexes developed                        rests compared attachment proteins having <br /> Benschop et al. (2010) similar trends (fig. 4b).                  complex numbers 1and 2, significant difference <br /> We significant negative correlation pro-               evolutionary rates (average evolutionary rates core <br /> tein complex number dN/dS (table 2) agree-                  proteins: 0.0899 (&#195;&#190;0.0031) (N &#194;&#188; 347), attachment proteins: <br /> ment previously published results (Chakraborty et al.              0.0811 (&#195;&#190;0.0057) (N &#194;&#188; 114); Mann&#226;&#8364;&#8220;Whitney U test, <br /> 2010). Therefore, protein complex number                   P &#194;&#188; 8.6   10 2). However, evolutionary rate differences <br /> cause evolutionary rate differences core                     significant core Attach-II proteins, as <br /> attachment proteins. Interestingly, consider-                 Attach-I Attach-II proteins (fig. 5). These <br /> able differences distribution protein complex num-                results indicate differences evolutionary rates <br /> bers core attachment proteins, ranging                     core attachment proteins disappear the <br /> 1 3 core proteins 1 22 attachment                    complex number factored out. <br /> proteins considering data set Gavin et al. (2006). <br /> To verify effect protein complex number controlling <br /> the evolutionary rate differences core attach-                  Protein Complex Number Expression Level <br /> ment proteins, divided data attachment pro-                 A large number studies using organisms ranging from <br /> teins groups; attachment proteins protein                   bacteria mammals (Akashi 2001; Subramanian and <br />  <br />  <br /> Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013                                             1371 <br />  Chakraborty Ghosh                                                                                                                   GBE <br />  <br /> Table 2 <br /> Spearman&#226;&#8364;&#8482;s Correlation Analysis Protein Complex Number (Taking Core Attachment Proteins Gavin et al. [2006]) versus Evolutionary <br /> Rates (dN/dS), Essentiality, Connectivity, Expression Level, Transcription Rate, Translation Rate <br />                       dN/dS         Essentiality    Connectivity      Expression       Expression       Transcription   Transcription    Translation <br />                                                                          Level        Level (Miller    Rate (Holstege    Rate (Miller    Rate (Arava <br />                                                                        (Holstege      et al. 2011)       et al. 1998)    et al. 2011)    et al. 2003) <br />                                                                       et al. 1998) <br /> Protein           r &#194;&#188;  0.1572      r &#194;&#188; 0.1659       r &#194;&#188;  0.0284      r &#194;&#188; 0.3987       r &#194;&#188; 0.3990       r &#194;&#188; 0.4032       r &#194;&#188; 0.3977       r &#194;&#188; 0.3239 <br />   complex         P &#194;&#188; 4.2   10 5   P &#194;&#188; 5.7   10 4   P &#194;&#188; 4.5   10 1   P &#194;&#188; 1.0   10 6   P &#194;&#188; 1.0   10 6   P &#194;&#188; 1.0   10 6   P &#194;&#188; 1.0   10 6   P &#194;&#188; 1.0   10 6 <br />   number          N &#194;&#188; 673          N &#194;&#188; 428          N &#194;&#188; 696          N &#194;&#188; 687          N &#194;&#188; 657          N &#194;&#188; 649          N &#194;&#188; 649          N &#194;&#188; 659 <br />   (Gavin et al. <br />   2006) <br />  <br />  <br />  <br />                                                                                level increased possible steps 1) increasing <br />                                                                                transcription rate, 2) increasing translational rate, 3) increas- <br />                                                                                ing transcription translation rate. Here, analyzed <br />                                                                                transcription rate translation rate protein <br />                                                                                complex data set observed transcription rate and <br />                                                                                translation rate increases increment protein com- <br />                                                                                plex number (table 2). We attachment pro- <br />                                                                                teins showed higher values average transcription and <br />                                                                                translation rates core proteins (for Holstege data set: av- <br />                                                                                erage transcription rate core proteins: 3.8349 (&#195;&#190;0.4394) <br />                                                                                (N &#194;&#188; 335), attachment proteins: 20.9643 (&#195;&#190;2.0421) <br />                                                                                (N &#194;&#188; 322); Mann&#226;&#8364;&#8220;Whitney U test, P &#194;&#188; 9.6   10 16; Miller <br />      FIG. 5.&#226;&#8364;&#8221;Average evolutionary rates (dN/dS) Core, Attach-I,         data set: average transcription rate core proteins: 22.1872 <br /> Attach-II proteins yeast. The pair wise comparison performed             (&#195;&#190;0.9876) (N &#194;&#188; 329), attachment proteins: 51.9403 (&#195;&#190;3.0335) <br /> two-tailed Mann&#226;&#8364;&#8220;Whitney U test. <br />                                                                                (N &#194;&#188; 320); Mann&#226;&#8364;&#8220;Whitney U test, P &#194;&#188; 6.9   10 16; for <br />                                                                                Arava data set: average translation rate core proteins: <br /> Kumar 2004; Drummond et al. 2006) demonstrated gene                       0.1893 (&#195;&#190;0.0296) (N &#194;&#188; 331), attachment proteins: 0.9446 <br /> expression levels strongly correlated protein evolu-                  (&#195;&#190;0.1325) (N &#194;&#188; 328); Mann&#226;&#8364;&#8220;Whitney U test, P &#194;&#188; 5.8   10 9). <br /> tionary rates, hypothesized highly ex-                    Therefore, clear proteins higher complex number <br /> pressed genes strong selection pressure reduce                    increase transcription translation rates, this <br /> mistranslation-induced protein misfolding (Drummond et al.                     increased transcription translation rates enhance the <br /> 2005; Drummond Wilke 2008). In previous study,                      overall protein expression levels. Finally, elevated expres- <br /> found protein complex number important                      sion levels decrease evolutionary rates (Spearman&#226;&#8364;&#8482;s revolu- <br /> constraints guiding protein evolutionary rates ob-                                                                                    6 <br />                                                                                tionary rate vs. expression level_Holstage &#194;&#188;  0.4894, P &#194;&#188; 1.0   10 ; <br /> served proteins associated large number                  Spearman&#226;&#8364;&#8482;s revolutionary rate vs. expression level_Miller &#194;&#188;  0.4857, <br /> of complexes (higher protein complex number) higher                       P &#194;&#188; 1.0   10 6) avoidance mistranslational-induced <br /> expression levels (Chakraborty et al. 2010). Once deter-                    protein misfolding (Drummond et al. 2005, 2006; Drummond <br /> mined average expression level core attachment                      Wilke 2008). Thus, controlled expression <br /> proteins, able attachment proteins                   levels, did obtain significant difference the <br /> a higher average expression level compared core                   rates protein evolution core attachment pro- <br /> proteins (fig. 6a b), strong significant positive                    teins (fig. 7). <br /> correlation exists protein complex number and <br /> expression level (table 2). Moreover, com- <br /> plex number expression level independently af-                         Discussion <br /> fect evolutionary rates (multivariate regression analysis:                     It reported complex-forming proteins evolve <br /> bexpression level &#194;&#188;  0.0601, P &#194;&#188; 8.1   10 5; bcomplex number &#194;&#188;                 slower noncomplex forming proteins (Teichmann <br />  0.1012, P &#194;&#188; 3.2   10 11), consistent pre-                   2002), observed yeast <br /> vious study (Chakraborty et al. 2010). According                            (Chakraborty et al. 2010). Moreover, complex- <br /> Drummond et al. (2005), expression level protein                      forming proteins, subunits associated a <br /> influences rates protein evolution transla-                 large number protein complexes evolve slower the <br /> tion events. Furthermore, know protein expression                      proteins associated lower number protein complexes <br />  <br />  <br /> 1372 Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013 <br />  Evolutionary Rates Yeast Protein Complexes                                                                                      GBE <br />  <br />  <br />  <br />  <br />                                                                                    FIG. 7.&#226;&#8364;&#8221;Average evolutionary rates (dN/dS) core attachment <br />                                                                               proteins different bins. The statistical comparison performed by <br />                                                                               two-tailed Mann&#226;&#8364;&#8220;Whitney U test. <br />  <br />  <br />                                                                               features genes (Liao et al. 2006). Therefore, measured <br />                                                                               multifunctionality core attachment proteins found <br />                                                                               core proteins associated higher number of <br />                                                                               biological processes compared attachment proteins. <br />                                                                               However, data set, did observe significant <br />                                                                               difference essentiality core attachment <br />                                                                               proteins. These results indicate essentiality a <br />                                                                               given gene protein does depend multifunc- <br />                                                                               tionality, parameters in- <br />                                                                               fluence gene/protein essentiality. In study, we <br />      FIG. 6.&#226;&#8364;&#8221;(a) Average expression level (using Holstege et al. [1998]   observed attachments proteins present com- <br /> data set) core attachment proteins yeast. The statistical compar-   plex assemblies core proteins, increase their <br /> ison performed two-tailed Mann&#226;&#8364;&#8220;Whitney U test. (b) Average expres-         essentiality modify enhance activity large <br /> sion level (using Miller et al. [2011] data set) core attachment   number complex assemblies. Therefore, reasonably <br /> proteins yeast. The statistical comparison performed two-tailed         assume protein essentiality protein complex number <br /> Mann&#226;&#8364;&#8220;Whitney U test. <br />                                                                               interrelated other. To verify hypothesis, we <br />                                                                               determined correlation gene essentiality pro- <br /> (Chakraborty et al. 2010; Das et al. 2013). The protein com-                  tein complex number data set, indeed, found <br /> plex number showed strong positive correlation gene                    strong positive association (Spearman&#226;&#8364;&#8482;s <br /> expression levels negative correlation evolutionary                  rprotein complex number vs. essentiality &#194;&#188; 0.2681, P &#194;&#188; 1.0   10 6), <br /> rates, suggesting protein complex number significant                contradicts results Pache et al. (2009) the <br /> factor modulating protein evolutionary rates. In work,                essentiality does depend protein complex number. <br /> utilized yeast protein complex data set (Gavin et al. 2006)               Thus multifunctionality protein complex number in- <br /> for classification pertaining core attachment                      fluence gene/protein essentiality. We obtained sim- <br /> proteins analyzed evolutionary rates address                 ilar proportion essential proteins core and <br /> influence gene dispensability, protein multifunctionality,                 attachment proteins suggesting core proteins execute <br /> protein connectivity, protein complex number, gene                        higher multifunctionality, attachment proteins have <br /> expression level protein evolutionary rates.                               higher protein complex number. These results consistent <br />     Analysis complex forming versus noncomplex proteins                    recent study fitness effect protein complex <br /> demonstrates proteins complexes higher                           core attachment proteins equally important for <br /> essentiality noncomplex proteins (average essentiality               function organism (Pache et al. 2009). Evolutionary <br /> of complex proteins: 0.1665 (&#195;&#190;0.0045) (N &#194;&#188; 867), noncom-                      rate differences core attachment proteins be <br /> plex proteins: 0.1002 (&#195;&#190;0.0014) (N &#194;&#188; 3,348); Mann&#226;&#8364;&#8220;Whitney                     interpreted framework neutralist/selectionist contro- <br /> U test, P &#194;&#188; 1.1   10 65). Within complex-forming pro-                     versy. Core proteins evolve faster attachment proteins, <br /> teins, core proteins reported crucial func-                core proteins multifunctional attach- <br /> tion protein complex assembly (Dezso et al. 2003; Gavin                ment proteins, results compatible the <br /> et al. 2006), indicating core proteins contribute                   neutral theory molecular evolution. However, attachment <br /> toward fitness organism. Moreover, reported                 proteins having higher expression levels higher <br /> that essential genes associated multifunctional                      impact essentiality, slowly evolving nature of <br />  <br />  <br /> Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013                                           1373 <br />  Chakraborty Ghosh                                                                                                           GBE <br />  <br /> attachment proteins core proteins apparently             Department Computational Medicine and <br /> accommodated neutral theory. Moreover, demon-                  Bioinformatics, University Michigan Medical School, Ann <br /> strated core proteins higher multifunctionality        Arbor, MI 48109 critical reading manuscript. They <br /> also higher impact gene/protein essentiality. Indeed,        thankful anonymous reviewers their <br /> we significant difference gene essentiality              valuable comments improve manuscript. <br /> between core attachment proteins. However, ob- <br /> served significant evolutionary rate differences core <br /> and attachment proteins, inconsistent neu-        Literature Cited <br /> tral theory molecular evolution. Recently, Razeto-Barry et al.    Akashi H. 2001. Gene expression molecular evolution. Curr Opin <br /> (2011) predicted selective scenario, multifunc-            Genet Dev. 11:660&#226;&#8364;&#8220;666. <br />                                                                      Altschul SF, et al. 1997. Gapped BLAST PSI-BLAST: new generation <br /> tional proteins evolve faster multifunc- <br />                                                                           protein database search programs. Nucleic Acids Res. 25: <br /> tional proteins difference size fitness        3389&#226;&#8364;&#8220;3402. <br /> effects. They conclude higher number mutations        Alvarez-Ponce D, Fares MA. 2012. Evolutionary rate duplicability the <br /> have fixed multifunctional proteins positive              arabidopsis thaliana protein-protein interaction network. Genome Biol <br /> selection. Our results confirm prediction Razeto-Barry             Evol. 4:1263&#226;&#8364;&#8220;1274. <br />                                                                      Arava Y, et al. 2003. Genome-wide analysis mRNA translation profiles in <br /> et al. (2011). <br />                                                                           Saccharomyces cerevisiae. Proc Natl Acad Sci U S A. 100:3889&#226;&#8364;&#8220;3894. <br />    In study, observed protein complex              Batada NN, Hurst LD, Tyers M. 2006. Evolutionary physiological <br /> number emerged important parameter explaining                   importance hub proteins. PLoS Comput Biol. 2:748&#226;&#8364;&#8220;756. <br /> the differences evolutionary rates core attach-       Batagelj V, Mrvar A. 2004. Pajek&#226;&#8364;&#8221;analysis visualization large net- <br /> ment proteins. However, attachment proteins                      works. In: Ju&#194;&#168;nger M, Mutzel P, editors. Graph drawing software. Berlin <br /> multifunctional, participate high number pro-            Heidelberg (Germany): Springer. p. 77&#226;&#8364;&#8220;103. <br />                                                                      Benschop JJ, et al. 2010. A consensus core protein complex composi- <br /> tein complexes transcription rates transla- <br />                                                                           tions Saccharomyces cerevisiae. Mol Cell. 38:916&#226;&#8364;&#8220;928. <br /> tion rates increase. The increased transcription rate            Camon E, et al. 2004. The Gene Ontology Annotation (GOA) Database: <br /> translation rate turn enhances overall protein expression          sharing knowledge Uniprot Gene Ontology. Nucleic Acids Res. <br /> levels expression level imposes strong selection pres-          32:D262&#226;&#8364;&#8220;D266. <br /> sure attachment proteins. Previously, Drummond et al.             Chakraborty S, Kahali B, Ghosh TC. 2010. Protein complex forming ability <br />                                                                           favored features interacting partners determining the <br /> (2005) demonstrated majority variation <br />                                                                           evolutionary rates proteins yeast protein-protein interaction <br /> in protein evolution mainly guided expression level.            networks. BMC Syst Biol. 4:155. <br /> This confirmed study, control          Chakraborty S, et al. 2011. Insights eukaryotic interacting protein <br /> expression level complex number, difference evolu-              evolution. In: Pontarotti P, editor. Evolutionary biology: concepts, bio- <br /> tionary rates core attachment proteins disap-                 diversity, macroevolution genome evolution. Berlin Heidelberg <br /> pears. Hence, analysis, showed rates                 (Germany): Springer. p. 51&#226;&#8364;&#8220;70. <br />                                                                      Chen W-H, Minguez P, Lercher MJ, Bork P. 2012. OGEE: online gene <br /> of protein evolution core attachment proteins <br />                                                                           essentiality database. Nucleic Acids Res. 40:901&#226;&#8364;&#8220;906. <br /> are mainly guided protein complex number expression           Cherry JM, et al. 2012. Saccharomyces Genome Database: genomics <br /> level. To relative influence evolutionary rates,          resource budding yeast. Nucleic Acids Res. 40:D700&#226;&#8364;&#8220;D705. <br /> we performed multivariate regression analysis            Das J, Chakraborty S, Podder S, Ghosh TC. 2013. Complex-forming pro- <br /> that expression level protein complex number inde-               teins escape robust regulations miRNA human. FEBS Lett. <br />                                                                           587:2284&#226;&#8364;&#8220;2287. <br /> pendently control evolutionary rate essentiality did not <br />                                                                      Deane CM, Salwinski L, Xenarios I, Eisenberg D. 2002. Protein <br /> show significant contribution.                                        interactions&#226;&#8364;&#8221;two methods assessment reliability high <br />                                                                           throughput observations. Mol Cell Proteomics. 1:349&#226;&#8364;&#8220;356. <br />                                                                      Dezso Z, Oltvai ZN, Baraba&#194;&#180;si AL. 2003. Bioinformatics analysis experi- <br /> Supplementary Material                                                    mentally determined protein complexes yeast Saccharomyces <br /> Supplementary tables S1&#226;&#8364;&#8220;S3 available Genome Biology                cerevisiae. Genome Res. 13:2450&#226;&#8364;&#8220;2454. <br /> and Evolution online (http://www.gbe.oxfordjournals.org/).           Dolan ME, Ni L, Camon E, Blake JA. 2005. A procedure assessing GO <br />                                                                           annotation consistency. Bioinformatics 21:I136&#226;&#8364;&#8220;I143. <br />                                                                      Drummond DA, Bloom JD, Adami C, Wilke CO, Arnold FH. 2005. Why <br /> Acknowledgments                                                           highly expressed proteins evolve slowly. Proc Natl Acad Sci U S A. 102: <br />                                                                           14338&#226;&#8364;&#8220;14343. <br /> This work supported Department Biotechnology,          Drummond DA, Raval A, Wilke CO. 2006. A single determinant dominates <br /> Government India Department Science                         rate yeast protein evolution. Mol Biol Evol. 23:327&#226;&#8364;&#8220;337. <br /> Technology, Government India (Sanction No. BT/PR692/              Drummond DA, Wilke CO. 2008. Mistranslation-induced protein misfold- <br />                                                                           ing dominant constraint coding-sequence evolution. Cell 134: <br /> BID/7/369/2011). The authors thankful Professor <br />                                                                           341&#226;&#8364;&#8220;352. <br /> Giorgio Bernardi, Department Biology, University Rome 3,          Fisher RA. 1930. The genetical theory natural selection. Oxford: <br /> Viale Marconi 446, Rome 00146, Italy, Dr Bratati Kahali,              Clarendon Press. <br /> Department Internal Medicine, Division Gastroenterology        Flicek P, et al. 2011. Ensembl 2011. Nucleic Acids Res. 39:D800&#226;&#8364;&#8220;D806. <br />  <br />  <br />  <br /> 1374 Genome Biol. Evol. 5(7):1366&#226;&#8364;&#8220;1375. doi:10.1093/gbe/evt096 Advance Access publication June 27, 2013 <br />  Evolutionary Rates Yeast Protein Complexes                                                                                              GBE <br />  <br /> Fraser HB, Hirsh AE, Steinmetz LM, Scharfe C, Feldman MW. 2002.                  Nie NH, Bent DH, Hull CH. 1970. SPSS: statistical package social <br />      Evolutionary rate protein interaction network. Science 296:               sciences. NewYork: McGraw-Hill. <br />      750&#226;&#8364;&#8220;752.                                                                    Pache RA, Babu MM, Aloy P. 2009. Exploiting gene deletion fitness effects <br /> Fraser HB, Wall DP, Hirsh AE. 2003. A simple dependence protein               yeast understand modular architecture protein complexes <br />      evolution rate number protein-protein interactions. BMC               different growth conditions. BMC Syst Biol. 3:74. <br />      Evol Biol. 3:11.                                                            Pa&#194;&#180;l C, Papp B, Lercher MJ. 2006. An integrated view protein evolution. <br /> Gavin AC, et al. 2006. Proteome survey reveals modularity yeast cell           Nat Rev Genet. 7:337&#226;&#8364;&#8220;348. <br />      machinery. Nature 440:631&#226;&#8364;&#8220;636.                                              Pang CNI, Krycer JR, Lek A, Wilkins MR. 2008. Are protein <br /> Guldener U, et al. 2005. CYGD: comprehensive Yeast Genome                         complexes cores, modules attachments? Proteomics 8: <br />      Database. Nucleic Acids Research 33:D364&#226;&#8364;&#8220;D368.                                   425&#226;&#8364;&#8220;434. <br /> Hahn MW, Kern AD. 2005. Comparative genomics centrality                   Park YR, Kim J, Lee HW, Yoon YJ, Kim JH. 2011. GOChase-II: correcting <br />      essentiality eukaryotic protein-interaction networks. Mol Biol          semantic inconsistencies gene ontology-based annotations for <br />      Evol. 22:803&#226;&#8364;&#8220;806.                                                                gene products. BMC Bioinformatics 12:S40. <br /> Hart GT, Lee I, Marcotte ER. 2007. A high-accuracy consensus map              Pereira-Leal JB, Levy ED, Teichmann SA. 2006. The origins evolution of <br />      yeast protein complexes reveals modular nature gene essentiality.             functional modules: lessons protein complexes. Philos Trans R <br />      BMC Bioinformatics 8:236.                                                        Soc Lond B Biol Sci. 361:507&#226;&#8364;&#8220;517. <br /> Hirsh AE, Fraser HB. 2001. Protein dispensability rate evolution.         Podder S, Mukhopadhyay P, Ghosh TC. 2009. Multifunctionality domi- <br />      Nature 411:1046&#226;&#8364;&#8220;1049.                                                            nantly determines rate human housekeeping tissue specific <br /> Holstege FCP, et al. 1998. Dissecting regulatory circuitry eukaryotic        interacting protein evolution. Gene 439:11&#226;&#8364;&#8220;24. <br />      genome. Cell 95:717&#226;&#8364;&#8220;728.                                                    Popescu CE, Borza T, Bielawski JP, Lee RW. 2006. Evolutionary rates and <br /> Hurst LD, Smith NGC. 1999. Do essential genes evolve slowly? Curr Biol. 9:            expression level Chlamydomonas. Genetics 172:1567&#226;&#8364;&#8220;1576. <br />      747&#226;&#8364;&#8220;750.                                                                    Pu S, Vlasblom J, Emili A, Greenblatt J, Wodak SJ. 2007. Identifying <br /> Jordan IK, Rogozin IB, Wolf YI, Koonin EV. 2002. Essential genes             functional modules physical interactome Saccharomyces <br />      evolutionarily conserved nonessential genes bacteria.                cerevisiae. Proteomics 7:944&#226;&#8364;&#8220;960. <br />      Genome Res. 12:962&#226;&#8364;&#8220;968.                                                     Qiu J, Noble WS. 2008. Predicting co-complexed protein pairs from <br /> Kanehisa M, Goto S, Sato Y, Furumichi M, Tanabe M. 2012. KEGG                     heterogeneous data. PLoS Comput Biol. 4:e1000054. <br />      integration interpretation large-scale molecular data sets.          Razeto-Barry P, Diaz J, Cotoras D, Vasquez RA. 2011. Molecular evolution, <br />      Nucleic Acids Res. 40:D109&#226;&#8364;&#8220;D114.                            <br /> </body></html>