<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An Essential Nuclear Protein Trypanosomes Is a <br /> Component mRNA Transcription/Export Pathway <br /> Mariana Serpeloni1,2, Carolina Borsoi Moraes3, Joa&#203;&#339;o Renato Carvalho Muniz4, Maria Cristina Machado <br /> Motta5, Augusto Savio Peixoto Ramos2, Rafael Luis Kessler1,2, Alexandre Haruo Inoue1,2, Wanderson <br /> Duarte daRocha6, Sueli Fumie Yamada-Ogatta7, Stenio Perdiga&#203;&#339;o Fragoso2, Samuel Goldenberg2, <br />                                             &#194;&#180; vila2* <br /> Lucio H. Freitas-Junior3, Andre&#194;&#180;a Rodrigues A <br /> 1 Departamento Biologia Celular e Molecular, Universidade Federal Parana&#194;&#180; (UFPR), Curitiba, Brazil, 2 Laborato&#194;&#180;rio Regulac&#194;&#184;a&#203;&#339;o da Expressa&#203;&#339;o ge&#203;&#8224;nica, Instituto Carlos <br /> Chagas (ICC), Curitiba, Brazil, 3 Center Neglected Diseases Drug Discovery (CND3), Institut Pasteur Korea (IPK), Gyeonggi-do, South Korea, 4 University Oxford, <br /> Oxford, United Kingdom, 5 Departamento Biologia Celular e Parasitologia, Instituto Biof&#196;&#177;&#194;&#180;sica Carlos Chagas Filho, Universidade Federal Rio Janeiro (UFRJ), Rio <br /> de Janeiro, Brazil, 6 Departamento Bioqu&#196;&#177;&#194;&#180;mica, Universidade Federal Parana&#194;&#180; (UFPR), Curitiba, Brazil, 7 Departamento Microbiologia, Centro Cie&#203;&#8224;ncias <br /> Biolo&#194;&#180;gicas, Universidade Estadual Londrina (UEL), Londrina, Brazil <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      In eukaryotic cells, different RNA species exported nucleus specialized pathways. The mRNA export <br />      machinery highly integrated mRNA processing, includes different set nuclear transport adaptors as <br />      mRNA binding proteins, RNA helicases, NPC-associated proteins. The protozoan parasite Trypanosoma cruzi the <br />      causative agent Chagas disease, widespread neglected human disease endemic Latin America. Gene <br />      expression Trypanosoma unique characteristics, constitutive polycistronic transcription protein-encoding <br />      genes mRNA processing trans-splicing. In general, post-transcriptional events major points regulation of <br />      gene expression parasites. However, export pathway mRNA nucleus poorly understood. The <br />      present study investigated function TcSub2, highly conserved protein ortholog Sub2/ UAP56, a <br />      component Transcription/Export (TREX) multiprotein complex connecting transcription mRNA export yeast/ <br />      human. Similar orthologs, TcSub2 nuclear protein, localized dispersed foci nuclei &#226;&#8364;&#8221;except the <br />      fibrillar center nucleolus&#226;&#8364;&#8221; interface dense non-dense chromatin areas, proposing the <br />      association TcSub2 transcription/processing sites. These findings analyzed BrUTP incorporation <br />      assays confirmed TcSub2 physically associated active RNA polymerase II (RNA pol II), RNA <br />      polymerase I (RNA pol I) Spliced Leader (SL) transcription, demonstrating participation particularly nuclear mRNA <br />      metabolism T. cruzi. The double <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>knockout</span> TcSub2 gene lethal T. cruzi, suggesting essential function. <br />      Alternatively, <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>RNA interference</span> assays performed Trypanosoma brucei. It allowed demonstrating being <br />      essential protein, knockdown causes mRNA accumulation nucleus decrease translation levels, reinforcing <br />      Trypanosoma-Sub2 (Tryp-Sub2) component mRNA transcription/export pathway trypanosomes. <br />  <br />   Citation: Serpeloni M, Moraes CB, Muniz JRC, Motta MCM, Ramos ASP, et al. (2011) An Essential Nuclear Protein Trypanosomes Is Component mRNA <br />   Transcription/Export Pathway. PLoS ONE 6(6): e20730. doi:10.1371/journal.pone.0020730 <br />   Editor: Fabio T. M. Costa, State University Campinas, Brazil <br />   Received November 26, 2010; Accepted May 11, 2011; Published June 8, 2011 <br />   Copyright: &#195;&#376; 2011 Serpeloni et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />    Funding: This work funded grants Conselho Nacional Desenvolvimento Cient&#196;&#177;&#194;&#180;fico e Tecnolo&#194;&#180;gico (CNPq) Fundac&#194;&#184;a&#203;&#339;o Arauca&#194;&#180;ria ARA, <br />    grants SG Fiocruz CNPq. MS funded studenship CNPq AHI studentship CAPES CBM LHFJ funded the <br />    Ministry Education, Science Technology (MEST), Gyeonggi government, Korean Institute Science Technology Information South <br />    Korea. SG, SPF MCMM research fellows CNPq. The funders role study design, data collection analysis, decision publish, or <br />    preparation manuscript. <br />    Competing Interests: The authors declared competing interests exist. <br />    * E-mail: aravila@fiocruz.br <br />  <br />  <br />  <br /> Introduction                                                                              RanGTP-dependent manner; specific exportins involved <br />                                                                                           different RNA types [2]. In contrast, nucleocytoplasmic <br />   In eukaryotic cells, transcription translation physically                       export mRNAs does follow RanGTP-exportin <br /> and temporally separated nuclear membrane. Control                              pathway. In yeast humans, mRNAs associate protein <br /> mRNA transport regions translation defines                           factors messenger ribonucleoprotein complexes (mRNPs) which <br /> proteins expressed. This transport initiated nucleus,                       exported NPC essential general <br /> where association protein complexes determinates fate                         receptor-shuttling heterodimer: Mex67/Mtr2 yeast TAP/ <br /> mRNAs cytoplasm [1]. Different events nuclear                               p15 humans [3,4]. The dimeric export receptor operates in <br /> metabolism mRNA crucial roles controlling gene                                 association TREX (Transcription/Export), multiprotein <br /> expression. The mRNA correctly processed                           complex connects transcription mRNA export. Yeast <br /> shuttled nucleus cytoplasm nuclear pore                             TREX consists RNA helicase Sub2 (UAP56 human), the <br /> complex (NPC). The general model RNA export involves                                   RNA-binding adaptor protein Yra1 (ALY human), the <br /> exportins transport receptors carry RNA NPC                           THO complex [5&#226;&#8364;&#8220;7]. <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                    1                                    June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                             Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />    Excluding model eukaryotic organisms, export machinery             59GGGGACAAGTTTGTACAAAAAAGCAGGCTTCATGAG- <br /> other eukaryotes determined. Using comparative                 CAGTGGACTTGCCGAC39 (forward) 59GGGGACCAC- <br /> genomics, recently showed mRNA export pathway                 TTTGTACAAGAAAGCTGGGTCCTATTACTGATTCATG- <br /> the conserved early divergent eukaryotes, especially          TACTGGCTCTG39 (reverse). Genomic DNA Dm28c was <br /> excavates, major kingdom unicellular eukaryotes known           used template. Both oligonucleotides contain recombination <br /> Excavata [8]. In lineage, suggested mRNA export            sites cloning TcSub2 ORF entrance pDONR <br /> is quite different members. The phylogenetic category            vector (Invitrogen &#226;&#8364;&#8220; GatewayH technology). This ORF was <br /> Excavata contains variety free-living symbiotic forms,          recombined expression vector pDESTTM 17, in <br /> also includes major parasites affecting humans [9,10].                  accordance manufacturer&#226;&#8364;&#8482;s protocol production of <br />    The protozoan parasite Trypanosoma cruzi member                   His6-tagged recombinant protein Escherichia coli BL21 (DE3) <br /> Excavata kingdom causative agent Chagas disease,              strain. The recombinant protein expression induced with <br /> a widespread neglected disease humans endemic             1 mM IPTG 3 h 37uC; protein separated SDS&#226;&#8364;&#8220; <br /> Latin America. Over 16 million people worldwide                    PAGE, excised gel eluted inoculation rabbit for <br /> estimated infected parasite, 100               polyclonal antiserum production. <br /> million live endemic areas [11&#226;&#8364;&#8220;13]. T. cruzi gene expression <br /> displays unique characteristics constitutive polycis-           Immunoblotting assay <br /> tronic transcription protein-coding genes mRNA processing                Cell lysates 56106 parasites loaded SDS-PAGE <br /> by trans-splicing. Because genes polycistronic <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>              proteins transferred nitrocellulose membrane <br /> display different levels processed mRNA, consensus notion          (Hybond C, Amersham Biosciences, England) using standard <br /> that post-transcriptional events major points gene                protocols. The membrane incubated anti-TcSub2 <br /> expression regulation play important roles adaptation             polyclonal antiserum diluted 1:1000 blocking solution (5% of <br /> of parasite life cycle [14,15]. However,              nonfat powdered milk, 0.1% Tween-20 phosphate buffered <br /> machinery mRNA export pathway functionally studied             saline, pH 7.4) 1 h. After washing PBS, membrane was <br /> in T. cruzi. We previously shown Sub2/UAP56,                     incubated polyclonal anti-GAPDH antiserum diluted 1:500 <br /> component TREX complex, conserved                blocking solution. This incubated 45 min room <br /> protein factors involved mRNA export pathway                temperature (R.T.) anti-rabbit horseradish peroxidase- <br /> [8]. In present study demonstrate, employing comparative <br />                                                                              conjugated IgG (Amershan Biosciences, England) diluted in <br /> modeling, T. cruzi Sub2 (TcSub2) similar 3D structure <br />                                                                              PBS/Tween 0.5%. Detection protein performed according <br /> when compared crystallized human ortholog. TcSub2 a <br />                                                                              manufacturer&#226;&#8364;&#8482;s instructions. <br /> nuclear protein localized foci dispersed nucleus apart <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> nucleolus. Furthermore, electron scanning microscopy <br /> revealed TcSub2 localized interface dense         Comparative analysis protein sequence alignments <br /> non-dense chromatin areas, <span id='am-398' about='xsp:pattern' typeof='owl:Thing'>pattern</span> distribution typically               molecular modeling <br /> associated mRNA transcription/processing sites. BrUTP                      BLAST searches carried TcSub2 T. cruzi protein&#226;&#8364;&#8221; <br /> incorporation assays showed TcSub2 colocalizes RNA                 homologues GenBank databases (http://www.ncbi.nlm.nih. <br /> pol II transcription sites. These findings strongly suggest             gov) using sequences human Saccharomyces cerevisae <br /> TcSub2 associated transcription sites participate         proteins queries. Sequences aligned using Clustal W <br /> nuclear mRNA metabolism. The double knockout TcSub2                   (http://www.cmbi.kun.nl/bioinf/tools/clustalw.shtml) ho- <br /> gene lethal T. cruzi, indicating that, similarly              molog proteins different organisms; including putative proteins <br /> eukaryotes, TcSub2 essential protein trypanosomes.                 trypanosomatids occasionally manual refinement the <br /> Alternatively, RNA interference (RNAi) assays T. bruceii                  alignments performed. For molecular modeling of <br /> confirmed essential function. Besides, knockdown               TcSub2, GenTHREADER software [24] used perform <br /> causes mRNA accumulation nucleus decreasing                    structural alignments based secondary structure available <br /> translation levels, confirming protein component           proteins known structure. The best match human <br /> mRNA transcription/export pathway trypanosomes.                           HsaUAP56 (1XTK) protein [25]. The atomic coordinates from <br />                                                                              structure submitted alignment results the <br /> Materials Methods                                                        program MODELLER [26] produce atomic-resolution <br />                                                                              models, validated using software PRO- <br /> Trypanosoma cultures                                                         CHECK [27], Verify 3D [28] WHATIF [29]. <br />   Epimastigotes T. cruzi Dm28c [16] cultured 28uC in <br /> LIT medium supplemented 10% bovine fetal serum [17].                    Ultrastructural immunocytochemistry <br /> The culture seeded 16106 cells ml21 parasites               Parasites fixed 0.3% glutaraldehyde, 4% formaldehyde, <br /> harvested cultures reached cell density 16107 cells ml21           1% picric acid diluted 0.1 M cacodylate buffer (pH 7.2) <br /> (during log phase growth). Different forms parasite             dehydrated 220uC graded series ethanol <br /> metacyclogenesis obtained vitro previously described              solutions. The material progressively infiltrated Unicryl <br /> [18,19]. Amastigotes obtained LLC-MK2 (ATCCH &#226;&#8364;&#8220;                       lower temperatures resin polymerization carried in <br /> number: CCL-7TM) [20] infected cells previously described                 BEEM capsules 20uC 5 days UV light. Ultrathin <br /> [21]. The culture T. brucei procyclic forms Lister 427 29-13           sections obtained Leica ultramicrotome (Reichert, <br /> [22] performed previously described [23].                            Ultracuts) grids containing sections incubated with <br />                                                                              50 mM NH4Cl 30 min. They incubated with <br /> Cloning expression TcSub2 recombinant protein                         blocking solution (3% BSA, 0.5% teleostean gelatin diluted in <br /> for polyclonal antiserum production                                          PBS, pH 8.0) 30 min. This followed 1 h incubation <br />   The TcSub2 coding region (GenBank identification number                    anti-TcSub2 serum diluted 1:200 blocking solution. The <br /> 3538341) amplified PCR using follow oligonucleotides:             grids incubated 30 min gold-labeled goat anti- <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                            June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                                  Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> rabbit IgG (Sigma) diluted 1:200 blocking solution. Grids                 (Invitrogen), 5 mg.ml21 DAPI. All slides mounted <br /> washed stained uranyl acetate lead citrate               VectaShield (Vector Laboratories). 2D images collected <br /> observation Zeiss 900 transmission electron microscope. For                  Nikon E90i epifluorescence microscope using 1006/1.4 <br /> control assays, incubation primary antiserum                         PlanApoVC objective Nikon DS-QiMc camera. 3D images <br /> omitted.                                                                          (z series) collected confocal Leica TCS SP2 AOBS using <br />                                                                                   63x/HCX 1.4 PL Apo lbdBL oil immersion objective. <br /> In situ labeling nascent RNAs <br />    The assay performed previously described Dossin                  TcSub2 knockout assay <br /> Schenkman (2005) [30]. The parasites washed times                          Regions located upstream (UPS) downstream (DWN) the <br /> and resuspended cell density 16107 ml21 transcription                  TcSub2 coding sequence cloned pKS/NEO and <br /> buffer containing 80 U mg ml21 RNasin (Promega). These cells                   pKS/HYG vectors respectively contain neomycin <br /> were permeabilized 2.5 mg ml21 lysolecithin (Sigma)                  (NEO) hygromycin (HYG) resistance genes [34]. The UPS <br /> for 1 min ice, centrifuged, washed transcription buffer,                DWN cloned flanking regions NEO HYG genes. <br /> suspended cell density transcription buffer                        The oligonucleotides used amplification UPS DWN are <br /> containing 2 mM ATP, 1 mM CTP, 1 mM GTP, 0.5 mM                                   listed supplementary material (Table 1). <br /> bromo-UTP (BrUTP) (Sigma), 200 mg ml21 creatine kinase                             The UPS/NEO/DWN construction (approximately 2100 bp) <br /> (Roche), 50 mM creatine phosphate (Roche). Cells                    PCR-amplified purified electroelution. A total 10 mg <br /> incubated 15 min 28uC. As control, parasites                    DNA used transfect T. cruzi, based method <br /> incubated transcription buffer 15 min different                       described [33]. After 24 h, neomycin (G418 - Sigma, St. Louis, <br /> concentrations a&#226;&#8364;&#8220;amanitin (Sigma) prior addition                     MO, USA) added concentration 300 mg ml21 select <br /> nucleotides. Based Campbell et al. (2003) [31], incubation              resistant parasites. The selected parasites cloned serial <br /> parasites 75 mg ml21 a&#226;&#8364;&#8220;amanitin inhibits RNA pol II                  dilution 24-well plates. Single allele knockout TcSub2 was <br /> while 200 mg.ml21 inhibits RNA pol II III.                               confirmed Pulse-Field electrophoresis followed Southern <br />                                                                                   blot, TcSub2 NEO genes labeled a- <br /> Fluorescent microscopy analysis TcSub2                                         [P32]-dCTP (10 mCi/ml; 3000 Ci/mmol) used probes. <br />    For N-terminal fusion GFP TcSub2 ORF cloned                       Knockout confirmed Western blot assays where <br /> into pTcGFPN vector [32] construct transfected                    protein extracts knockout wild-type parasites used <br /> in T. cruzi epimastigotes previously described [33]. For analysis              detection TcSub2, using GAPDH level loading control. <br /> of cells expressing TcSub2-GFP, aliquots cells                        Comparative analysis proteins performed through <br /> harvested, washed PBS, resuspended PBS density                      signal quantification using Scion Image program [35]. A single <br /> 107 cells ml21, fixed 4% paraformaldehyde PBS                         allele knockout parasites transfected UPS/ <br /> 20 min. The fixed cells placed poly-l-lysine-coated                HYG/DWN construction (approximately 2300 bp), was <br /> slides 20 min room temperature analyzed                        PCR-amplified using oligonucleotides UPSF e DWNR. The <br /> fluorescence microscopy. For indirect immunofluorescence (IF)                 transfection performed described UPS/NEO/DWN <br /> assays, cells fixed described above, permeabilized               construction, 300 mg ml21 G418 300 mg ml21 hygro- <br /> 0.1% Triton X-100 PBS 5 min blocked 4% BSA                       micin B (Sigma) added LIT medium 24 h after <br /> PBS 60 min. For TcSub2 localization analysis, cells                  transfection. <br /> incubated 45 min anti-TcSub2 serum diluted 1:1000. <br /> This followed 45 min incubation Alexa Fluor 488-                    RNAi assay T. brucei <br /> conjugated donkey anti-rabbit IgG diluted 1:1600 (Invitrogen)                   According previous data comparative genomics [8], <br /> 5 mg ml21 DAPI. For transcription colocalization analysis, cells                  Trypanosoma brucei ortholog protein (TbSub2) TcSub2 <br /> were incubated 45 min anti-TcSub2 serum diluted 1:1000                   accession number Tb10.70.7730 GenBank database. <br /> and mouse anti-BrdU (Invitrogen) diluted 1:300. This                          For TbSub2 RNAi approach, target sequence primers <br /> followed incubation 45 min Alexa Fluor 594-                           chosen using RNAit software [36]. The target sequence was <br /> conjugated donkey anti-rabbit IgG diluted 1:1600 (Invitrogen),                    PCR-amplified cloned p2T7-177 vector [37]. The <br /> Alexa Fluor 488-conjugated goat anti-mouse IgG diluted 1:400                      vector linearized NotI transfected procyclic forms <br />  <br />  Table 1. Oligonucleotides used amplification UPS DWN regions TcSub2 gene. <br />  <br />  <br />  Oligonucleotides                            Sequence (59- 39)                                                          Enzyme            bp <br />  <br />  UPS-NEO F                                   GGGGTCGACGGTGTTATTTGCGTCACGATGTGC                                          SalI              700 <br />  UPS-NEO R                                   GGGCGTCGACTTGCTTATGTCTATCTGGTTTCTTA                                        SalI <br />  UPS HIGRO F                                 GGGTCTAGAGGTGTTATTTGCGTCACGATGTGC                                          XbaI <br />  UPS HIGRO R                                 GGGCTCTAGATTGCTTATGTCTATCTGGTTTCTTA                                        XbaI <br />  DOWN-NEO F                                  GGGGAATTCGATTTAAGGCCTCTGGAAAATAAGAATGAGG                                   EcoRI             450 <br />  DOWN-NEO R                                  GGGGAATTCAACACAAAAATAAGAAGAAGTATTGGAATGGCAATTGC                            EcoRI <br />  DOWN-HIGRO F                                GGGGGATCCGATTTAAGGCCTCTGGAAAATAAGAATGAGG                                   BamHI <br />  DOWN-HIGRO R                                GGGAAGCTTAACACAAAAATAAGAAGAAGTATTGGAATGGCAATTGC                            HindIII <br />  <br />  NEO: neomycin; HIGRO: hygromycin. bp <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span> pair. The enzyme cleavage sites bold. <br />  doi:10.1371/journal.pone.0020730.t001 <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                              3                            June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                            Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> of T. brucei 29-13 strain [22]. The stable DNA integration              ethanol, filters dried R.T. radioactivity was <br /> selected using phleomycin (5 mg ml21) RNAi induction                counted liquid scintillation. <br /> performed addition 1 mg ml21 tetracycline. Cultures <br /> grown mid-log phase (106&#226;&#8364;&#8220;107 cells ml21) harvested              Flow cytometry assays <br /> production total protein extracts. The knockdown                     All experiments performed FACSCalibur flow <br /> TbSub2 expression confirmed immunoblotting assay, using              cytometer (Becton-Dickinson, San-Jose, USA). A total 20,000 <br /> GAPDH level loading control. Comparative analysis               events acquired data analysis performed using <br /> proteins performed signal quantification using Scion            FlowJo software (Treestar software). All experiments at <br /> Image program [35].                                                         duplicate. <br />                                                                                Cell viability assays performed using parameters: <br /> Fluorescent situ hybridization (FISH)                                    propidium iodide staining mitochondrial transmembrane <br />    For TcSub2- telomere colocalization analysis, log phase                  potential [adapated 41]. For assay, 26106 cells were <br /> epimastigotes subjected indirect immunoflourescence                 harvested, washed PBS stained 5 ng ml21 <br /> followed telomere detection using Telomere PNA FISH Kit/                 propidium iodide PBS 28uC 15 min. The cells were <br /> FITC (DakoCytomation) accordance manufacturer&#226;&#8364;&#8482;s                 analyzed flow cytometry using FL2-H detector. For second <br /> instructions. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> observe colocalization TcSub2 SL                   assay, 26106 cells harvested, washed PBS incubated <br /> transcription sites T. cruzi, log phase epimastigotes               10 mg ml21 rhodamine 123 PBS 15 min 28uC. <br /> subjected IF followed Fluorescent situ hybridization               The cells resuspended PBS flow cytometry <br /> (FISH), using SL-RNA probe. The SL-RNA probe                   analyses using FL1-H detector. Data analysis executed <br /> prepared PCR using template T. cruzi (Y strain) SL RNA            considering viable cells gated forward scatter. <br /> gene cloned vector pGEM-T Easy. The SL probe labeled                 Relative mitochondrial membrane potential (in comparison to <br /> with digoxigenin PCR using High prime DNA labeling kit                   control cells) determined considering ratio (treated/control <br /> (Roche Diagnostics), purified used hybridization assays           cells) geometric mean rhodamine 123 fluorescence. <br /> described Dossin et al., 2005 [30]. For detection polyA-                 The cell cycle analysis performed DNA staining with <br /> containing mRNA, FISH performed based previous works                 propidium iodide. 26105 cells harvested, washed PBS and <br /> with modifications [38,39]. Aliquots cells (T. cruzi epimastigotes       resuspended 500 ml DNA staining solution containing <br /> and T. brucei procyclic forms) harvested, washed PBS,               0.425 mM Tris-HCl, 9.375 mM Propidium Iodide, 0.0125% <br /> resuspended PBS density 107 cells ml21 fixed            NP40 detergent, 1.25 mM NaCl 8.75 U/L Rnase A. <br /> 4% paraformaldehyde PBS 20 min. Fixed cells washed              The cells kept ice immediate flow cytometry <br /> three times PBS, resuspended 200 ml PBS placed               quantification. Data analysis FL2-Widht6FL2-Area <br /> onto poly-l-lysine-coated slides 20 min R.T. The cells          gated cells exclude debris doublets. The Dean-Jett-Fox <br /> then permeabilized 0.2% Triton X-100 PBS 30 min                 algorithm FlowJo software used estimate percentage of <br />                                                                             cells G0/G1, S G2/M phases cell cycle. <br /> and washed times PBS. Pre-hybridization was <br /> performed 106 Denhardt&#226;&#8364;&#8482;s solution, 46SSC, 1 mM EDTA,                        Apoptosis detected phosphatidylserine exposure using <br />                                                                             BD PharMingenTM Annexin V-FITC Apoptosis Detection Kit <br /> 35% deionized formamide, 0.5 mg ml21 tRNA, 2 mU ml21 <br />                                                                             according manufacturer&#226;&#8364;&#8482;s instructions. Briefly, 26106 cells were <br /> RNase OUT 30 min room temperature. For control, cells <br />                                                                             harvested, washed staining annexin-V-FITC (AV) for <br /> were previously treated 100 mg ml21 boiled RNase A in <br />                                                                             30 min 28uC followed staining propidium iodide (PI) for <br /> PBS 60 min 37uC. Six ng/ml digoxigenin (DIG) <br />                                                                             15 min 28uC. Unstained single-stained cells used to <br /> conjugated-59 oligo d(T)30 added pre-hybridization buffer. <br />                                                                             set compensation signals detectors. To observe <br /> The cells heated 65uC 3 min hybridization at <br />                                                                             morphological alterations, cell samples assay were <br /> R.T. overnight. The cells washed twice 26 SSC <br />                                                                             treated using staining method &#226;&#8364;&#732;&#226;&#8364;&#732;Pano&#194;&#180;tico Ra&#194;&#180;pido&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Laborclin, <br /> for15 min, twice 16 SSC 15 min, twice PBS                 Pinhais, PR, Brazil) described previously [34]. The stained <br /> 5 min incubated 45 min R.T. anti-                      parasites examined bright field using Leica SP5-AOBS <br /> digoxigenin monoclonal antibody (Sigma) diluted PBS/ BSA                 confocal microscopy. <br /> (0.1 mg ml21). This followed incubation Alexa Fluor <br /> 488-conjugated goat anti-mouse IgG diluted 1:400 (Invitrogen) in <br />                                                                             Results <br /> PBS/BSA 0.1 mg ml21 100 ng ml21 DAPI 45 min at <br /> R.T. The slides mounted N-propyl gallate examined               Cloning TcSub2 comparative analysis ortholog <br /> with Nikon E600 microscope. Images acquired                 proteins molecular modeling <br /> Image Pro program (Media Cybernetics, Bethesda, MD, USA).                      Previous studies shown small group proteins <br />                                                                             involved mRNA export conserved eukaryotic <br /> Metabolic Labeling                                                          phylogeny. Among components mRNA export pathway, <br />    The metabolic labeling procyclic forms T. brucei               Sub2 conserved protein [8]. Its ortholog T. cruzi <br /> performed according [40] modifications. A total 56106               (TcSub2) annotated putative RNA helicase <br /> cells 48 h-induced non- induced cultures harvested             (Tc00.1047053508319.40) highly similar SceSub2 <br /> from medium, washed methionine-free RPMI medium (Sigma)                  (68%) HsaUAP56 (69%). Multiple alignments TcSub2 <br /> supplemented fetal bovine serum 0.5% labeled                    similar proteins different eukaryotes (Figure 1A) a <br /> incubation, medium containing 50 mCi ml21 [35S]-                highly conserved aminoacid sequence eukaryotic lineages. <br /> methionine (Amershan Bioscience), hour 28uC. For                 Sequence analysis revealed TcSub2 exhibits canonical <br /> control, parasites treated 100 mg ml21 cycloheximide           helicase motifs, typical DEADX box-RNA helicase proteins, <br /> for 5 min addition [35S]-methionine. Aliquots 10 ml            associated nearly processes involving RNA, <br /> were spotted 3MM filters boiled 10% trichloroace-              transcription RNA degradation [42]. We obtained the <br /> ticacid (TCA). After consecutives washes 5% TCA, acetone              molecular model TcSub2 based crystal structure of <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        4                             June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                             Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> human HsaUAP56 (Figure 1B). Comparative analysis                 RNA binding activity involved ATPase mRNA <br /> molecular model TcSub2 HsaUAP56 shows                          processing, similar UAP56 humans [43]. Since TcSub2 <br /> conservative folding RNA helicase motifs major                    structure highly conserved, assayed functional ability of <br /> helicase domains. Despite amino acid differences                TcSub2 protein rescue <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> S. cerevisiae sub2 null <br /> the sequences (Figure 1C) does affect quality             mutants [44]. SUB2 essential gene yeast viability of <br /> overall aspect modeled structure TcSub2 RNA                    mutant cells maintained episomal presence the <br /> helicase (Figure 1D E). Modifications N-terminal                  gene URA-3 plasmid. Functional complementation of <br /> (regions Ia II) non-conservative (figure 1D)             heterologous genes strains accessed growth <br /> modifications C-terminal regions (regions IV, V VI)               FOA-containing plates. Since Sub2 levels critical yeast <br /> conservative (figure 1E). In N-terminal, V116C substitu-             [45], tested expression TcSub2 low- high-copy <br /> tion located hydrophobic core A118T Y123F               plasmids (as control S. cerevisiae SUB2 gene cloned into <br /> are exposed solvent. These substitutions                vectors). However, growth observed case <br /> unlikely cause conformational changes structure           5 days culture 30uC (data shown). <br /> whole. The substitution F194C, located region II, a <br /> hydrophobic core conservative substitutions surround-              TcSub2 nuclear protein localized mRNA <br /> ing volumetrically bigger chain phenylalanine             transcription sites <br /> enough accommodate residue position. At C-                      The subcellular localization TcSub2 analyzed by <br /> terminal domain, region V presents conservative                 GFP fusion, exclusively nuclear distribution was <br /> substitution V347M. This <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>mutation</span> located              observed fluorescence microscopy epimastigote forms <br /> the IPA biding sites human structure directly related       (Figure 2, left). We observed protein evenly <br /> on coordination. The substitution D350E located               distributed nucleus, regions showing more <br /> regions V VI structurally exposed, minimizing              intense signal typical compartmentalized distribution. This <br /> impact protein folding. The conservative variable residues            distribution pattern confirmed indirect immuno- <br /> regions IV VI (mutations I287V G372A, respectively)              fluorescence, using anti-TcSub2 polyclonal specific antibodies <br /> buried destabilize TcSub2 structure. However,             (Figure 2, right). This constitutive protein pattern <br /> are limited C-terminal domain unlikely           protein localization occurs different forms life cycle of <br /> cause disruption protein folding. Some aminoacid             parasite Trypanosoma brucei Leishmania major (data <br /> substitutions (Y123F, I287V, G372A) observed                shown). To investigate nuclear distribution of <br /> Sub2 ortholog Plasmodium falciparum (pfU52) did affect            TcSub2, ultrastructural immunocytochemical analyses were <br /> the activity protein domains (Figures 1C&#226;&#8364;&#8220;E). PfU52           performed T. cruzi epimastigotes (Figure 3). The results showed <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison amino acid sequences TcSub2 homolog proteins. (A) Amino acids shaded according similarity: <br /> black identical, dark grey present 60% sequences, light grey similarity follows BLOSUM 62 Matrix. When <br /> necessary, gaps inserted various sequences (dashes) allow alignment optimization. The motifs typical DEAD-box <br /> RNA helicases highlighted predicted secondary structures shown alignment. The GenBank accession numbers entries <br /> are listed follows: TcSub2, (T. cruzi NP_808002.1); Tbr, (T. brucei XP_822312.1); Lma, (Leishmania major XP_001683110.1); PfaSub2, (Plasmodium <br /> falciparum XP_001349607.1); HsaUAP56, (Homo sapiens NP_004631.1); SceSub2, (Saccharomyces cerevisiae NP_010199.1); Ath, (Arabidopsis thaliana <br /> NP_568245.1); Ddi, (Dictyostelium discoideum XP_646415.1). (B) Molecular modeling TcSub2 (light grey), based human homologue, <br /> HsaUAP56 (dark grey). Motifs highlighted. Representations three-dimensional models created software PyMol (http://www. <br /> pymol.org). (C) Analysis TcSub2 motifs, compared general DEAD-box consensus HsaUAP56 aminoacid disposition. (D&#226;&#8364;&#8220;E). The <br /> arrangement different amino acids differ TcSub2 (light grey) HsaUAP56 (dark grey) evidenced N-terminal (D) C- <br /> terminal (E) regions. <br /> doi:10.1371/journal.pone.0020730.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                              June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                            Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> that protein localized mainly less-compact chromatin             observed compared 200 mg ml21 (data shown). We <br /> regions (Figure 3A) absent fibrillar center              decided use 200 mg ml21 drug inhibition <br /> nucleolus (Figure 3B). Moreover, observed presence                concentration RNA pol II. When BrUTP incorporation was <br /> grouped gold particles edge electron-dense chromatin              carried presence RNA pol II inhibitor a-amanitin, <br /> regions (Figure 3A). This distribution similar                colocalization TcSub2 RNA pol I <br /> transcription sites [46], indicating TcSub2 located active       transcription site. This confirmed previous finding that <br /> transcription regions. Analyses using indirect immunofluorescence           TcSub2 excluded fibrillar center nucleolus. The <br /> combined telomere FISH reinforced notion             distribution TcSub2 relation RNA pol II RNA pol I <br /> of protein does colocalize telomeres indicated           transcription sites better observed comparison the <br /> the absence TcSub2 signal telomeric repeats             fluorescence intensity profiles confocal pictures BrUTP- <br /> (Supplementary Figure S1). To test hypothesis, situ labeling        labeled nuclei. In absence a-amanitin treatment, fluores- <br /> of nascent RNAs followed immunofluorescence TcSub2                cence intensity peaks nascent RNAs TcSub2 often <br /> was analyzed confocal microscopy. This approach allowed              correlated (Figure 4B). However, a-amanitin treatment, poor <br /> detection BrUTP-labeled nascent RNAs (BrRNA),                      correlation observed BrRNA TcSub2 <br /> corresponded active transcription sites, TcSub2 (Figure 4).          fluorescent peaks (Figure 4C). This colocalization pattern was <br /> The results showed TcSub2 colocalizes RNA pol II                  observed different focal planes (Supplementary Figure S2). In <br /> active transcription sites (Figure 4A). Based previous work [31]         addition, colocalization TcSub2 Spliced Leader <br /> incubation parasites 75 mg ml21 &#226;&#8364;&#8220;amanitin                  RNA observed FISH immunolocalization analysis <br /> inhibits RNA pol II 200 mg ml21 inhibits RNA              (Figure 5). Taking together, results strongly suggest that <br /> pol II III. We treated cells using 75 mg ml21 &#226;&#8364;&#8220;           TcSub2 related mRNA transcription and/or processing the <br /> amanitin, similar profile signal detection                nucleus T. cruzi. We observed uncorrelated peaks of <br />  <br />  <br />  <br />  <br /> Figure 2. Localization TcSub2 T. cruzi Dm28c epimastigotes. The GFP fusion protein (TcSub2-GFP) native protein (TcSub2) were <br /> localized fluorescence microscopy. Specific polyclonal antibodies raised TcSub2 used detect native protein. Secondary <br /> antibodies conjugated Alexa-488. Cells stained DAPI locate nuclear kinetoplast DNA. Bars = 10 mm. <br /> doi:10.1371/journal.pone.0020730.g002 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        6                            June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                            Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />  <br />  <br /> Figure 3. Immunoelectron microscopy localization TcSub2 T. cruzi Dm28c epimastigotes. (A) nuclear distribution TcSub2: gold <br /> particles showing TcSub2 seen edge electron-dense chromatin (arrows). (B) Gold particles present perinucleolar regions <br /> (arrows) absent fibrillar center (FC) nucleolus (Nu). Bars = 1 mm. k = kinetoplast, Chr = dense chromatin. <br /> doi:10.1371/journal.pone.0020730.g003 <br />  <br /> TcSub2 BrRNA, suggesting RNA pol II transcription                  does trigger death apoptosis (assessed phosphatidylserine <br /> sites nuclear domain associated TcSub2.               exposure, Figure S4). In addition, analysis cell viability by <br />                                                                             propidium iodide internalization (Supplementary Figure S3C and <br /> Trypanosome Sub2 ortholog proteins essential                        S5A) mitochondrial transmembrane potential (Supplementary <br /> parasite <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span>                                                           Figure S3D S5B) shows strong increase the <br />    To analyze function TcSub2 used gene-knockout                percentage non-viable cells occurs 96 hours RNAi <br /> approach abolish TcSub2 expression observed                      induction. Therefore, results observed present study can <br /> subsequent phenotypic alterations. A single-allele TcSub2 knock-            explained cell death patterns, necrotic cell death <br /> out T. cruzi successfully obtained (Figure 6). However, cell         late secondary event knockdown TbSub2. <br /> growth affected reduction protein (Figure 6A). <br /> Western analysis using protein extracts wild-type single-          Knockdown TbSub2 causes mRNA accumulation the <br /> allele knockout parasites, confirmed 50% reduction                nuclei translation inhibition <br /> TcSub2 expression (Figure 6B). This reduction protein                       FISH assays detect poliA+ RNA performed infer the <br /> expression replacement TcSub2 allele                distribution pattern mRNA investigate changes after <br /> neomycin selective marker observed pulse field                    decreasing Tryp-Sub2 protein levels parasite. As shown in <br /> electrophoresis Southern blot analysis (Figure 6C). We failed           supplementary figure S6, distribution patterns mRNA are <br /> to achieve null knockout cells viable         similar wild type single knockout cells T. cruzi. <br /> selection hygromycin resistance. Three unsuccessful attempts            It observed major signal cytoplasm the <br /> were obtain null knockout believe              nucleus, weak signal inside nucleus. The analysis <br /> result consequence essential function TcSub2 T.           different time-points growth evident changes <br /> cruzi. Trypanosoma brucei relevant model analysis gene          observed. <br /> function using inducible based RNA interference                   By contrast, knockdown TbSub2 affected mRNA <br /> [47]. Moreover, Sub2 highly conserved protein                    distribution pattern T. brucei. Differently wild type cells, <br /> eukaryotic evolution [8] T. brucei ortholog protein              48 h RNAi induction possible observe slight signal <br /> which 97% similar TcSub2 (see alignment Figure 1A). This           increase inside nucleus (Figure 8A, top; Supplementary <br /> encouraged use T. brucei suitable model investigate       Figure S7); signal visible cytoplasm. This <br /> function Sub2 trypanosomes. First data showed TbSub2             correspond mRNA accumulation nucleus is <br /> has essential function knockdown procyclic forms            clearly visible 72 h induction. (Figure 8A, below; <br /> resulted life cycle arrest G2 phase (Figure 7A)                  Supplementary Figure S7). By analysis using metabolic labeling <br /> morphological changes (Figure 7B). Life cycle analysis flow              [35S]-methionine, demonstrated silencing of <br /> cytometry showed presence cells arrested G2 phase                 TbSub2 expression caused translation inhibition 48 h of <br /> (containing nuclei and/or kinetoplasts) 48 h              induction (Figure 8B). It possible observe deep decrease of <br /> RNAi induction, increasing number G2 arrested -               radioactive counting induced cells compared non- <br /> cells 72 h (Figure 7A). After 24 h RNAi induction, level       induced cells, similar result observed non-induced cells <br /> of TbSub2 decreases 45% reaches 70% 72 h                treated translation inhibitor cycloheximide. We believe <br /> induction (Supplementary Figure S3A). The knockdown inhibits                translation inhibition mRNA accumulation due <br /> the cellular division 24 h (Supplementary Figure S3B)          cell death knockdown TbSub2 strongly affects cell <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        7                              June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                                Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />  <br />  <br /> Figure 4. Nuclear colocalization TcSub2 active sites transcription detection nascent RNAs T. cruzi. (A) <br /> Immunolocalization TcSub2 active transcription sites (BrRNA) confocal microscopy, (2) (+) 200 mg ml21 a-amanitin. (B) <br /> Analysis colocalization TcSub2 (red curves) transcription sites (green curves) a-amanitin. Squares colocalization TcSub2 and <br /> transcription sites. (C) Analysis colocalization TcSub2 (red curves) transcription sites (green curves) a-amanitin. The fluorescence <br /> intensity profiles TcSub2 transcription sites distribution fluorescence dotted line (x-axis). The fluorescence intensities <br /> are plotted y-axis squares indicate colocalization. Bars = 1 mm. <br /> doi:10.1371/journal.pone.0020730.g004 <br />  <br />  <br /> viability 96 h induction, shown (Supple-               [15]. Mature mRNAs generated primary transcripts by <br /> mentary Figure S3C,D; Supplementary Figures S4 S5).                        trans-splicing polyadenylation moved the <br />                                                                                cytoplasm translated [48]. In context post- <br /> Discussion                                                                     transcriptional events, machinery mRNA export poorly <br />                                                                                understood trypanosomes pathway an <br />    Post-transcriptional events crucial regulation gene              important step regulation gene expression parasites. <br /> expression trypanosomatids absence specific               We interested investigating factors could <br /> control mechanisms transcription. Unlike eukaryotic                involved pathway. <br /> organisms, gene transcribed RNA Pol II                   The export mRNAs T. cruzi mediated by <br /> own promoter, transcription trypanosomatids polycistronic                CRM1, component RanGTP-exportin pathway. This <br /> without traditional promoter elements genes individual                  pathway commonly responsible protein export no <br /> clusters necessarily <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> functionally related proteins             major role mRNA export higher eukaryotes [49]. Most <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          8                               June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                              Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />  <br />  <br /> Figure 5. Nuclear colocalization TcSub2 SL transcription sites T. cruzi. Immunofluorescence TcSub2 (green) followed FISH of <br /> SL transcription sites (red) performed epimastigote forms T. cruzi. Cells stained DAPI locate nuclear kinetoplast DNA. <br /> Bars = 1 mm. <br /> doi:10.1371/journal.pone.0020730.g005 <br />  <br /> reports related topic come model organisms, especially          conserved motifs distributed domains. In general, motifs I <br /> S. cerevisiae, bulk mRNA exported RanGTP                  II implicated ATP binding hydrolysis, with <br /> independent pathway involving THO/TREX complex                           contributions motif VI. Motif III believed couple ATP <br /> [revised 2]. Based previous investigations, started             hydrolysis RNA unwinding, motifs IV, V, VI <br /> biological analysis conserved component                   contribute RNA binding [42,51]. We noted changes of <br /> eukaryotic mRNA export pathway, yeast DEAD-box RNA                       amino acid composition TcSub2 sequence compared <br /> helicase Sub2 (UAP56 humans). In present study cloned              human sequence (UAP56). However, changes did not <br /> the gene encoding T. cruzi protein highly similar             affect molecular model TcSub2 similar the <br /> Sub2/UAP56, named TcSub2. Sub2/UAP56                       UAP56 crystal structure. <br /> component TREX multiprotein complex links                           Although TcSub2 highly conserved, unable function a <br /> transcription mRNA export [revised 50]. DEAD-box                     substitute SUB2. We speculate the <br /> proteins involved ATP-dependent unwinding double-              specific functions TcSub2 T. cruzi transcripts in <br /> stranded RNA (dsRNA), displacement, RNA remodeling,                       parasite processed trans-splicing cis-splicing as <br /> RNA/protein complexes. They characterized                        eukaryotic cells [52]. Specific point mutations be <br />  <br />  <br />  <br />  <br /> Figure 6. Cell proliferation analysis single allele TcSub2 knockout T. cruzi. (A) Growth curve wild-type T. cruzi epimastigotes (blue <br /> squares) single-allele knockout mutants (red squares). (B) Western-blot analysis comparing expression TcSub2 wild-type (WT) and <br /> mutant parasites (KO). The membrane incubated 1:1000 polyclonal anti-TcSub2 anti-GAPDH serum. The level TcSub2 protein was <br /> decreased single-allele TcSub2-knockout parasites. (C) Southern blot analysis confirming knockout allele TcSub2. The membrane <br /> containing chromosomes wild-type (WT) single-allele knockout mutant hybridized TcSub2 (TcSub2) Neomycin genes (NEO) <br /> labeled 32P. <br /> doi:10.1371/journal.pone.0020730.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                               June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                                 Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />  <br />  <br /> Figure 7. Analysis cell cycle morphology. (A) The cell cycle analyzed flow cytometry graphs correspond data from <br /> RNAi assays 48 72 hours induction. Non-induced cells (NI) indicated black lines, induced cells (I) indicated orange lines. The <br /> cell cycle phases indicated graph: G1, S G2/M. (B) Morphology images stained cells 48 72 hours RNAi induction <br /> obtained confocal microscopy. Bars = 10 mm. <br /> doi:10.1371/journal.pone.0020730.g007 <br />  <br /> enough create specific characteristics TcSub2 protein,            In export mRNA, revealing example involving <br /> could, example, prevent interaction protein                yeast Mex67-Mtr2 protein, adaptor responsible for <br /> components TREX S.cerevisiae. Such example                  mRNA export, human counterpart, TAP-p15. TAP is <br /> recently reported Va&#194;&#180;zquez colleagues (2009) [53].                       unable complement mex67- mutant cells does p15 <br /> TcU2AF35, involved initial steps trans-splicing           complement mrt22 mutants. However, TAP-p15 complex <br /> T. cruzi, able functionally complement yeast cells          partially restores growth defect mex672/mtr22 double <br /> aminoacid residues modified. This demonstrates importance               mutants showing TAP p15 interact functional in <br /> of conserved residues functional substitution ortholog proteins          yeast TAP-Mtr2 Mex67-p15 complexes not <br /> between different organisms. Another possibility                   form [54]. Like Sub2 UAP56, TcSub2 exclusively nuclear <br /> development novel components T. cruzi pathway,                  dispersed loci nuclei, present the <br /> specific factors necessary functional activity TcSub2.          periphery nucleolus, excluding fibrillar center nucleolus. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          10                               June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                               Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br />  <br />  <br /> Figure 8. FISH poliA+mRNA translation level TbSub2 knockdown. (A) Fluorescence microscopy poliA+ mRNAs FISH using <br /> DIG-labeled oligo d(T)30 48 72 hours RNAi induction. Cells stained DAPI localize nuclear kinetoplast DNA. <br /> Bars = 1 mm. (B) Metabolic labeling 48 hours RNAi induction. The graph shows average radioactive counting triplicate experiments <br /> of non-induced cells (NI), non-induced cells treated 100 mg ml21 cycloheximide (NIC), induced cells (I). The p value, calculated by <br /> unpaired t-test, indicated graph. <br /> doi:10.1371/journal.pone.0020730.g008 <br />  <br />    The ultrastructural immunocytochemical assays showed                  TcSub2 functionally similar homolog proteins other <br /> TcSub2 concentrated non-dense chromatin areas grouped               eukaryotes. However unable confirm TcSub2 <br /> mainly interface dense non-dense chromatin.                involved pathway nuclear mRNA metabolism this <br /> This speckled pattern distribution usually associated              parasite. We observed uncorrelated peaks TcSub2 and <br /> mRNA transcription, processing, nucleocytoplasmatic export                BrRNA, suggesting RNA pol II transcription sites the <br /> [45,55,56]. This distribution pattern similar human                 nuclear domain associated TcSub2. We speculate the <br /> arthropod (Chironomus tentans) TcSub2 homologues                    partial association nascent mRNAs explained the <br /> localized periphery dense chromatin domains, termed                 dynamics interaction different events cell, where <br /> interchromatin granule clusters (IGCs) [57]. These clusters contain           DNA regions different stages transcription. Another <br /> mainly proteins related mRNA processing, especially SC-35,             explanation TcSub2 related transcription of <br /> are closely related perichromatic fibrils, mRNA                    specific mRNAs consequence role regulation gene <br /> transcription occurs [revised 58,59]. Many studies                    expression. To investigate function TcSub2, we <br /> demonstrated nascent mRNAs deposited                        attempted knockout TcSub2 gene obtain phenotypic <br /> interchromatin spaces [46,60&#226;&#8364;&#8220;62]. A similar distribution pattern              alterations related mRNA export. However, null knockout <br /> has observed trypanosomatids, interface                lineages T. cruzi viable. Studies an <br /> dense non-dense chromatin bromodomain factors (BDFs)                  essential protein eukaryotes knockdown of <br /> and acetylated histones [63&#226;&#8364;&#8220;65]. Recent investigations              Sub2 homologs inhibits cell growth [45,55,70,71]. Unfortunately, <br /> these histone modifications serve indicators regions            knockout single TcSub2 allele affect <br /> initiation RNA Pol II transcription [64,66,67]. Based               morphology cell growth. Although single-allele TcSub2 <br /> findings TcSub2 localization, decided investigate               knockouts 50% TcSub2 protein, did alter <br /> relationship TcSub2 active transcription sites using BrUTP             mRNA export. It necessary apply regulated system <br /> incorporation followed immunocolocalization. This approach                 completely silence TcSub2 expression information <br /> has successfully used observe association RNA Pol II           regarding function T. cruzi. The absence approaches of <br /> transcription sites proteins Hrp59 Hrp65,                inducible silencing, RNA interference, T. cruzi preclude <br /> hnRNPs C. tentans. It used define localization        suitable studies this. On hand, T. bruceii suitable <br /> of RNA Pol II transcription sites T. cruzi [30,68,69]. Our results         model study essential genes using inducible based on <br /> demonstrated TcSub2 colocalizes nascent RNAs &#226;&#8364;&#8221;                 RNAi trypanosomes. Taking account Sub2 highly <br /> specifically transcribed RNA pol II&#226;&#8364;&#8221; responsible            conserved trypanosomes [8], decided use T. bruceii study <br /> mRNA, snRNA spliced-leader (SLRNA) transcription. Block-                  function protein. The data obtained demonstrated it <br /> ing RNA pol II activity a-amanitin abolished                      really essential protein knockdown inhibits cellular <br /> transcription nascent RNAs, resulting RNAs transcribed             growth. Similar eukaryotes, silencing TbSub2 <br /> by RNA pol I. Under conditions, observed absence              expression caused translation inhibition robust accumulation of <br /> colocalization TcSub2, indicating TcSub2                   bulk polyadenylated mRNAs nucleoplasm. These phenotypes <br /> associated RNA pol II, RNA pol I transcription.             observed expression homologs silenced in <br /> Besides, protein does localize SL-RNA, reinforcing          metazoan [45,55,71&#226;&#8364;&#8220;76]. Interestingly, observed arrest of <br /> the protein related mRNA transcription sites. The association           cell cycle G2 phase, mainly 72 hours RNAi induction <br /> of TcSub2 RNA pol II transcription sites strongly suggests          level proteins reduced 70%. In same <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        11                              June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                                               Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> way, knockdown UAP56, homolog humans, resulted                                Figure S5 Flow cytometry analysis cell viability and <br /> mitotic defects. Taking together, trypanosome protein                              cell cycle TbSub2 knockdown RNAi. (A) Overlay <br /> similar characteristics orthologs higher eukaryotes, possibly                    histograms induced (I) non-induced (NI) cells stained with <br /> reflecting similar function parasites.                                          Propidium Iodide 24 96 hours (h, indicated inside the <br />    In context nuclear mRNA metabolism trypanosoma-                               graphs) RNAi induction; percentage PI positive cells <br /> tids, nucleocytoplasmic transport understood data                          (&#226;&#8364;&#732;&#226;&#8364;&#732;Dead&#226;&#8364;&#8482;&#226;&#8364;&#8482; region) indicated. (B) Overlay histograms I NI <br /> in area lend insights regulation gene                            cells stained rhodamine 123 mitochondrial membrane <br /> expression parasites. Herein, identified novel                               potential analysis. (C) Analysis RNAi induction DNA <br /> trypanosome protein related nuclear mRNA                                     content staining permeabilized cells propidium iodide; <br /> metabolism proposed Tryp-Sub2 component                               note gradual increase cells G2/S phase 48 hours of <br /> mRNA transcription/export pathway parasites.                                      RNAi induction. (D) Cell cycle analysis Dean-Jett-Fox <br />                                                                                            algorithm FlowJo software; data non-induced (left histogram) <br /> Supporting Information                                                                     induced (right histogram) cells 48 hours RNAi <br />                                                                                            induction shown; percentage cells phase cell <br /> Figure S1 Nuclear colocalization TcSub2 telo-                                      cycle indicated inside graphs. <br /> mere repeats. FISH telomeric repeat regions followed                                 (TIF) <br /> immunofluorescence TcSub2 perfomed epimastigote <br /> forms. Telomeric repeats (red); TcSub2 (green). Cells stained                         Figure S6 FISH poliA+mRNA T. cruzi. Localization of <br /> with DAPI locate nuclear kinetoplast DNA. Bars = 1 mm.                          mRNAs FISH using DIG-labeled oligo d(T)30 wild-type <br /> (TIF)                                                                                      (WT) single knockout TcSub2 (KO) epimastigotes. Cells <br />                                                                                            stained DAPI locate nuclear kinetoplast <br /> Figure S2 (A, B) Nuclear colocalization TcSub2                                     DNA. For control, cells treated 100 mg ml21 RNAse <br /> transcription sites different focal planes. Analysis                                 A hybridization. Bars = 1 mm. <br /> colocalization TcSub2 (red) transcription sites (BrRNA,                            (TIF) <br /> green) a-amanitin. Bars = 1 mm. <br />                                                                                            Figure S7 FISH poliA+mRNA T. brucei. Localization <br /> (TIF) <br />                                                                                            mRNAs FISH using DIG-labeled oligo d(T)30 48 and <br /> Figure S3 Cell proliferation viability analysis. (A)                                   72 hours RNAi induction. Cells stained DAPI to <br /> Growth curves induced (I) non-induced (NI) cells direct                          locate nuclear kinetoplast non-induced cells (NI) and <br /> counting Neubauer chamber; point represents mean                             induced cells (I). DNA. For control, cells treated with <br /> and standard deviations triplicate experiments. Western blot                         100 mg ml21 RNAse A hybridization. Bars = 1 mm. <br /> TbSub2 RNAi induction. Antiserum GAPDH                                  (TIF) <br /> used protein-loading control. (B) Percentage cells G2 <br /> phase cell cycle calculated flow cytometry (details                               Acknowledgments <br /> methods); (C) Percentage non-viable cells (dead cells) calculated <br />                                                                                            We thank Newton M. Vidal Christian M. Probst bioinformatic <br /> by flow cytometry using propidium iodide staining; (D) Compar- <br />                                                                                            assistance, Nilson Fide&#203;&#8224;ncio Vanessa M. dos Santos technical <br /> ison R123 fluorescence intensity induced non                                 assistance, Miriam Giardini providing amastigote forms, Andre&#194;&#180; Luis <br /> induced cells plotted fold change graph using geometric mean                         Bombo figure assistance, Lysangela Ronalte Alves, Alejandro Correa <br /> FL1-H detector intensity. For flow cytometry data (B&#226;&#8364;&#8220;D),                              Bruno Dallagiovana valuable comments, Michel Baptista and <br /> experimental point represents average respective standard                          Fla&#194;&#180;via S. Pereira Souza providing vectors localization and <br /> deviation duplicate experiments.                                                        knockout analysis Se&#194;&#180;rgio Schenkman providing vector SL <br /> (TIF)                                                                                      gene. <br />  <br /> Figure S4 Analysis phosphatidylserine exposure                                    Author Contributions <br /> TbSub2 knockdown RNAi. The density plots non- <br /> induced (NI, row) induced (I, row) cells co-stained                          Conceived designed experiments: MS CBM ASPR SPF LHF-J <br />                                                                                            ARA. Performed experiments: MS JRCM MCMM ASPR RLK WDR <br /> with Annexin-V-FITC Propidium Iodide 48 96 hours                              AHI. Analyzed data: MS JRCM ARA. Contributed reagents/ <br /> (indicated plots) RNAi induction. The values inside                       materials/analysis tools: SFY-O MCMM SG LHF-J ARA. Wrote the <br /> density plots represent percentage cells inside region.                        paper: MS ARA. Provided guidance, direction, decision-making, and <br /> (TIF)                                                                                      oversight: MS CBM SG LHF-J WDR ARA. <br />  <br /> References <br />  1. Du TG, Schmid M, Jansen RP (2007) Why cells messages: biological               7. Aguilera A (2005) Cotranscriptional mRNP assembly: DNA the <br />     functions mRNA localization. Semin Cell Dev Biol 18: 171&#226;&#8364;&#8220;177.                           nuclear pore. Curr Opin Cell Biol 17: 242&#226;&#8364;&#8220;250. <br />  2. Kohler A, Hurt E (2007) Exporting RNA nucleus cytoplasm. Nat            8. Serpeloni M, Vidal NM, Goldenberg S, Avila AR, Hoffmann FG (2011) <br />     Rev Mol Cell Biol 8: 761&#226;&#8364;&#8220;773.                                                              Comparative genomics proteins involved RNA nucleocytoplasmic export. <br />  3. Santos-Rosa H, Moreno H, Simos G, Segref A, Fahrenkrog B, et al. (1998)                    BMC Evol Biol 11: 7. <br />     Nuclear mRNA export requires complex formation Mex67p                       9. Simpson AG, Inagaki Y, Roger AJ (2006) Comprehensive multigene phylogenies <br />     Mtr2p nuclear pores. Mol Cell Biol 18: 6826&#226;&#8364;&#8220;6838.                                   excavate protists reveal evolutionary positions &#226;&#8364;&#732;&#226;&#8364;&#732;primitive&#226;&#8364;&#8482;&#226;&#8364;&#8482; eukaryotes. <br />  4. Segref A, Sharma K, Doye V, Hellwig A, Huber J, et al. (1997) Mex67p, novel              Mol Biol Evol 23: 615&#226;&#8364;&#8220;625. <br />     factor nuclear mRNA export, binds poly(A)+ RNA nuclear                 10. Hampl V, Hug L, Leigh JW, Dacks JB, Lang BF, et al. (2009) Phylogenomic <br />     pores. Embo J 16: 3256&#226;&#8364;&#8220;3271.                                                               analyses support monophyly Excavata resolve relationships among <br />  5. Chavez S, Beilharz T, Rondon AG, Erdjument-Bromage H, Tempst P, et al.                     eukaryotic &#226;&#8364;&#732;&#226;&#8364;&#732;supergroups&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Proc Natl Acad Sci U S A 106: 3859&#226;&#8364;&#8220;3864. <br />     (2000) A protein complex containing Tho2, Hpr1, Mft1 novel protein,              11. Hotez PJ, Molyneux DH, Fenwick A, Kumaresan J, Sachs SE, et al. (2007) <br />     Thp2, connects transcription elongation mitotic recombination                      Control neglected tropical diseases. N Engl J Med 357: 1018&#226;&#8364;&#8220;1027. <br />     Saccharomyces cerevisiae. Embo J 19: 5824&#226;&#8364;&#8220;5834.                                        12. Rassi A, Jr., Rassi A, Marin-Neto JA (2010) Chagas disease. Lancet 375: <br />  6. Strasser K, Masuda S, Mason P, Pfannstiel J, Oppizzi M, et al. (2002) TREX            1388&#226;&#8364;&#8220;1402. <br />     conserved complex coupling transcription messenger RNA export. Nature             13. Coura JR, Borges-Pereira J (2010) Chagas disease: 100 years discovery. <br />     417: 304&#226;&#8364;&#8220;308.                                                                              A systemic review. Acta Trop 115: 5&#226;&#8364;&#8220;13. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                    12                                    June 2011 | Volume 6 | Issue 6 | e20730 <br />                                                                                                                Sub2 Is Component mRNA Export Trypanosome <br />  <br />  <br /> 14. Teixeira SM, daRocha WD (2003) Control gene expression genetic                   43. Shankar J, Pradhan A, Tuteja R (2008) Isolation characterization of <br />     manipulation Trypanosomatidae. Genet Mol Res 2: 148&#226;&#8364;&#8220;158.                             Plasmodium falciparum UAP56 homolog: evidence coupling RNA <br /> 15. Martinez-Calvillo S, Vizuet-de-Rueda JC, Florencio-Martinez LE, Manning-                    binding splicing activity site-directed mutations. Arch Biochem Biophys <br />     Cela RG, Figueroa-Angulo EE (2010) Gene expression trypanosomatid                        478: 143&#226;&#8364;&#8220;153. <br />     parasites. J Biomed Biotechnol 2010: 525241.                                            44. Kistler AL, Guthrie C (2001) Deletion MUD2, yeast homolog U2AF65, <br /> 16. Contreras VT, Salles JM, Thomas N, Morel CM, Goldenberg S (1985) In vitro                   bypass requirement sub2, essential spliceosomal ATPase. Genes <br />     differentiation Trypanosoma cruzi chemically defined conditions. Mol               Dev 15: 42&#226;&#8364;&#8220;49. <br />     Biochem Parasitol 16: 315&#226;&#8364;&#8220;327.                                                          45. Strasser K, Hurt E (2001) Splicing factor Sub2p required nuclear mRNA <br /> 17. Camargo EP (1964) Growth Differentiation Trypanosoma Cruzi. I.                       export interaction Yra1p. Nature 413: 648&#226;&#8364;&#8220;652. <br />     Origin Metacyclic Trypanosomes Liquid Media. Rev Inst Med Trop Sao                46. Verschure PJ, van Der Kraan I, Manders EM, van Driel R (1999) Spatial <br />     Paulo 6: 93&#226;&#8364;&#8220;100.                                                                            relationship transcription sites chromosome territories. J Cell Biol <br /> 18. Elias MC, Marques-Porto R, Freymuller E, Schenkman S (2001) Transcription                   147: 13&#226;&#8364;&#8220;24. <br />     rate modulation Trypanosoma cruzi life cycle occurs parallel        47. Djikeng A, Shen S, Tschudi C, Ullu E (2004) Analysis gene function in <br />     changes nuclear organisation. Mol Biochem Parasitol 112: 79&#226;&#8364;&#8220;90.                          Trypanosoma brucei using RNA interference. Methods Mol Biol 270: 287&#226;&#8364;&#8220;298. <br /> 19. Avila AR, Yamada-Ogatta SF, da Silva Monteiro V, Krieger MA,                            48. Clayton CE (2002) Life transcriptional control? From fly man and <br />     Nakamura CV, et al. (2001) Cloning characterization metacyclogenin               again. Embo J 21: 1881&#226;&#8364;&#8220;1888. <br />     gene, specifically expressed Trypanosoma cruzi metacyclogen-            49. Cuevas IC, Frasch AC, D&#226;&#8364;&#8482;Orso I (2005) Insights CRM1-mediated RNA- <br />     esis. Mol Biochem Parasitol 117: 169&#226;&#8364;&#8220;177.                                                   nuclear export pathway Trypanosoma cruzi. Mol Biochem Parasitol 139: <br /> 20. Hull RN, Cherry WR, Tritch OJ (1962) Growth characteristics monkey                       15&#226;&#8364;&#8220;24. <br />     kidney cell strains LLC-MK1, LLC-MK2, LLC-MK2(NCTC-3196)                        50. Moore MJ, Proudfoot NJ (2009) Pre-mRNA processing reaches to <br />     utility virus research. J Exp Med 115: 903&#226;&#8364;&#8220;918.                                    transcription ahead translation. Cell 136: 688&#226;&#8364;&#8220;700. <br /> 21. Schenkman S, Jiang MS, Hart GW, Nussenzweig V (1991) A novel cell surface               51. Hilbert M, Karow AR, Klostermeier D (2009) The mechanism ATP- <br />     trans-sialidase Trypanosoma cruzi generates stage-specific epitope required            dependent RNA unwinding DEAD box proteins. Biol Chem 390: <br />     invasion mammalian cells. Cell 65: 1117&#226;&#8364;&#8220;1125.                                        1237&#226;&#8364;&#8220;1250. <br /> 22. Wirtz E, Leal S, Ochatt C, Cross GA (1999) A tightly regulated inducible                52. Liang XH, Haritan A, Uliel S, Michaeli S (2003) trans cis splicing in <br />     expression conditional gene knock-outs dominant-negative                     trypanosomatids: mechanism, factors, regulation. Eukaryot Cell 2: 830&#226;&#8364;&#8220;840. <br />     genetics Trypanosoma brucei. Mol Biochem Parasitol 99: 89&#226;&#8364;&#8220;101.                       53. Vazquez M, Atorrasagasti C, Bercovich N, Volcovich R, Levin MJ (2003) <br /> 23. Dhalia R, Marinsek N, Reis CR, Katz R, Muniz JR, et al. (2006) The eIF4A                Unique features Trypanosoma cruzi U2AF35 splicing factor. Mol <br />     helicases Trypanosoma brucei functionally distinct. Nucleic Acids Res 34:            Biochem Parasitol 128: 77&#226;&#8364;&#8220;81. <br />     2495&#226;&#8364;&#8220;2507.                                                                              54. Katahira J, Strasser K, Podtelejnikov A, Mann M, Jung JU, et al. (1999) The <br /> 24. McGuffin LJ, Jones DT (2003) Improvement GenTHREADER method                          Mex67p-mediated nuclear mRNA export pathway conserved yeast to <br />     genomic fold recognition. Bioinformatics 19: 874&#226;&#8364;&#8220;881.                                   human. Embo J 18: 2593&#226;&#8364;&#8220;2609. <br /> 25. Shi H, Cordin O, Minder CM, Linder P, Xu RM (2004) Crystal structure             55. Gatfield D, Le Hir H, Schmitt C, Braun IC, Kocher T, et al. (2001) The DExH/ <br />     human ATP-dependent splicing export factor UAP56. Proc Natl Acad                        D box protein HEL/UAP56 essential mRNA nuclear export in <br />     Sci U S A 101: 17628&#226;&#8364;&#8220;17633.                                                                 Drosophila. Curr Biol 11: 1716&#226;&#8364;&#8220;1721. <br /> 26. Sali A, Blundell TL (1993) Comparative protein modelling satisfaction             56. Kota KP, Wagner SR, Huerta E, Underwood JM, Nickerson JA (2008) Binding <br />     spatial restraints. J Mol Biol 234: 779&#226;&#8364;&#8220;815.                                                ATP UAP56 necessary mRNA export. J Cell Sci 121: 1526&#226;&#8364;&#8220;1537. <br /> 27. Laskowski RA, Rullmannn JA, MacArthur MW, Kaptein R, Thornton JM                        57. Kiesler E, Miralles F, Visa N (2002) HEL/UAP56 binds cotranscriptionally to <br />     (1996) AQUA PROCHECK-NMR: programs checking quality                          Balbiani ring pre-mRNA intron-independent manner accompa- <br />     protein structures solved NMR. J Biomol NMR 8: 477&#226;&#8364;&#8220;486.                                  nies BR mRNP nuclear pore. Curr Biol 12: 859&#226;&#8364;&#8220;862. <br /> 28. Luthy R, Bowie JU, Eisenberg D (1992) Assessment protein models                 58. Spector DL (2003) The dynamics chromosome organization gene <br />     three-dimensional profiles. Nature 356: 83&#226;&#8364;&#8220;85.                                              regulation. Annu Rev Biochem 72: 573&#226;&#8364;&#8220;608. <br /> 29. Vriend G (1990) WHAT IF: molecular modeling drug design program.                  59. Zhao R, Bodnar MS, Spector DL (2009) Nuclear neighborhoods gene <br />     J Mol Graph 8: 52&#226;&#8364;&#8220;56, 29.                                                                   expression. Curr Opin Genet Dev 19: 172&#226;&#8364;&#8220;179. <br /> 30. Dossin Fde M, Schenkman S (2005) Actively transcribing RNA polymerase II                60. Iborra FJ, Pombo A, Jackson DA, Cook PR (1996) Active RNA polymerases are <br />     concentrates spliced leader genes nucleus Trypanosoma cruzi.                   localized discrete transcription &#226;&#8364;&#732;&#226;&#8364;&#732;factories&#226;&#8364;&#8482; human nuclei. J Cell Sci <br />     Eukaryot Cell 4: 960&#226;&#8364;&#8220;970.                                                                   109(Pt 6): 1427&#226;&#8364;&#8220;1436. <br /> 31. Campbell DA, Thomas S, Sturm NR (2003) Transcription kinetoplastid                   61. Cmarko D, Verschure PJ, Martin TE, Dahmus ME, Krause S, et al. (1999) <br />     protozoa: normal? Microbes Infect 5: 1231&#226;&#8364;&#8220;1240.                                      Ultrastructural analysis transcription splicing cell nucleus after <br /> 32. Batista M, Marchini FK, Celedon PA, Fragoso SP, Probst CM, et al. (2010) A                  bromo-UTP microinjection. Mol Biol Cell 10: 211&#226;&#8364;&#8220;223. <br />     high-throughput cloning reverse genetics Trypanosoma cruzi.               62. Cremer T, Cremer C (2001) Chromosome territories, nuclear architecture and <br />     BMC Microbiol 10: 259.                                                                      gene regulation mammalian cells. Nat Rev Genet 2: 292&#226;&#8364;&#8220;301. <br /> 33. Lu HY, Buck GA (1991) Expression exogenous gene Trypanosoma cruzi              63. Nardelli SC, da Cunha JP, Motta MC, Schenkman S (2009) Distinct acetylation <br />     epimastigotes. Mol Biochem Parasitol 44: 109&#226;&#8364;&#8220;114.                                           Trypanosoma cruzi histone H4 cell cycle, parasite differentiation, and <br /> 34. Souza FS, Rampazzo Rde C, Manhaes L, Soares MJ, Cavalcanti DP, et al.                    DNA damage. Chromosoma 118: 487&#226;&#8364;&#8220;499. <br />     (2010) Knockout gene encoding kinetoplast-associated protein 3               64. Siegel TN, Hekstra DR, Kemp LE, Figueiredo LM, Lowell JE, et al. (2009) Four <br />     (KAP3) Trypanosoma cruzi: effect kinetoplast organization, cell                       histone variants mark boundaries polycistronic transcription units in <br />     proliferation differentiation. Mol Biochem Parasitol 172: 90&#226;&#8364;&#8220;98.                        Trypanosoma brucei. Genes Dev 23: 1063&#226;&#8364;&#8220;1076. <br /> 35. Tolivia J, Navarro A, del Valle E, Perez C, Ordonez C, et al. (2006) Application        65. Villanova GV, Nardelli SC, Cribb P, Magdaleno A, Silber AM, et al. (2009) <br />     Photoshop Scion Image analysis quantification signals                       Trypanosoma cruzi bromodomain factor 2 (BDF2) binds acetylated histones <br />     histochemistry, immunocytochemistry hybridocytochemistry. Anal Quant                    accumulated UV irradiation. Int J Parasitol 39: 665&#226;&#8364;&#8220;673. <br />     Cytol Histol 28: 43&#226;&#8364;&#8220;53.                                                                 66. Respuela P, Ferella M, Rada-Iglesias A, Aslund L (2008) Histone acetylation and <br /> 36. Redmond S, Vadivelu J, Field MC (2003) RNAit: automated web-based tool                   methylation sites initiating divergent polycistronic transcription Trypano- <br />     selection RNAi targets Trypanosoma brucei. Mol Biochem Parasitol              soma cruzi. J Biol Chem 283: 15884&#226;&#8364;&#8220;15892. <br />     128: 115&#226;&#8364;&#8220;118.                                                                           67. Figueiredo LM, Cross GA, Janzen CJ (2009) Epigenetic regulation African <br /> 37. Wickstead B, Ersfeld K, Gull K (2002) Targeting tetracycline-inducible                 trypanosomes: new kid block. Nat Rev Microbiol 7: 504&#226;&#8364;&#8220;513. <br />     expression transcriptionally silent minichromosomes Trypano-           68. Percipalle P, Fomproix N, Kylberg K, Miralles F, Bjorkroth B, et al. (2003) An <br />     soma brucei. Mol Biochem Parasitol 125: 211&#226;&#8364;&#8220;216.                                            actin-ribonucleoprotein interaction involved transcription RNA <br /> 38. Cassola A, Frasch AC (2009) An RNA recognition motif mediates                           polymerase II. Proc Natl Acad Sci U S A 100: 6475&#226;&#8364;&#8220;6480. <br />     nucleocytoplasmic transport trypanosome RNA-binding protein. J Biol                69. Kiesler E, Hase ME, Brodin D, Visa N (2005) Hrp59, hnRN <br /> </body></html>