<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Changes in transcriptional orientation are <br /> associated with increases in evolutionary <br /> rates of enterobacterial genes <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Changes in transcriptional orientation (&#226;&#8364;&#339;CTOs&#226;&#8364;&#65533;) occur frequently in prokaryotic genomes. Such <br />   changes usually result <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> genomic inversions, which may cause a conflict between the directions of replication <br />   and transcription and an increase in <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span> rate. However, CTOs do not always lead <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the replication- <br />   transcription confrontation. Furthermore, CTOs may cause deleterious disruptions of operon structure and/or gene <br />   regulations. The currently existing CTOs may indicate relaxation of selection pressure. Therefore, it is of interest to <br />   investigate whether CTOs have an independent effect on the evolutionary rates of the affected genes, and <br />   whether these genes are subject to any type of selection pressure in prokaryotes. <br />   Methods: Three closely related enterbacteria, Escherichia coli, Klebsiella pneumoniae and Salmonella enterica serovar <br />   Typhimurium, were selected for comparisons of synonymous (dS) and nonsynonymous (dN) substitution rate <br />   between the genes that have experienced changes in transcriptional orientation (changed-orientation genes, <br />   &#226;&#8364;&#339;COGs&#226;&#8364;&#65533;) and those that do not (same-orientation genes, &#226;&#8364;&#339;SOGs&#226;&#8364;&#65533;). The dN/dS ratio was also derived to evaluate the <br />   selection pressure on the analyzed genes. Confounding factors in the estimation of evolutionary rates, such as <br />   gene essentiality, gene expression level, replication-transcription confrontation, and decreased dS at gene terminals <br />   were controlled in the COG-SOG comparisons. <br />   Results: We demonstrate that COGs have significantly higher dN and dS than SOGs when a series of confounding <br />   factors are controlled. However, the dN/dS ratios are similar between the two gene groups, suggesting that the <br />   increase in dS can sufficiently explain the increase in dN in COGs. Therefore, the increases in evolutionary rates in <br />   COGs may be mainly mutation-driven. <br />   Conclusions: Here we show that CTOs can increase the evolutionary rates of the affected genes. This effect is <br />   independent of the replication-transcription confrontation, which is suggested to be the major cause of inversion- <br />   associated evolutionary rate increases. The real cause of such evolutionary rate increases remains unclear but is <br />   worth further explorations. <br />  <br />  <br /> Introduction                                                 has been the major driver of the changes in evolutionary <br /> Genome rearrangements occur frequently in the evolu-         rates, the dN/dS ratios are expected to differ signifi- <br /> tion of prokaryotes. Among the rearrangement events,         cantly between the two gene groups. Clarifying the <br /> inversions usually occur symmetrically around the origin     molecular mechanisms by which prokaryotic genomes <br /> (designated as &#226;&#8364;&#339;Ori&#226;&#8364;&#65533;) or terminus (&#226;&#8364;&#339;Ter&#226;&#8364;&#65533;) of replication     evolve may help us understand how prokaryotes develop <br /> between closely related bacterial genomes [1]. These         novel functions, which is in turn relevant to ecological <br /> rearrangement events often lead to changes in transcrip-     and biomedical studies. <br /> tional orientation (designated as &#226;&#8364;&#339;CTOs&#226;&#8364;&#65533;) [2] and              Accordingly, we compared the genomes of three clo- <br /> increases in mutation pressure in the affected genes         sely related enterobacteria [5,6]: Escherichia coli (ECO), <br /> because of a conflict between the directions of transcrip-   Klebsiella pneumonia (KPN) and Salmonella enterica <br /> tion and replication [1,3,4]. However, when the inver-       serovar Typhimurium (STM). A series of analyses were <br /> sion events result from the flipping across Ori or Ter,      performed to control for potential confounding factors, <br /> replication-transcription conflict may not occur [1].        including gene essentiality, expression level, background <br /> Furthermore, CTOs may cause changes in homology-             mutation rate, the <span id='am-84' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of replication-transcription <br /> based recombination or impediments in DNA replica-           confrontation, and codon usage bias. These potential <br /> tion (by altering, for example, DNA protein binding          confounding factors have been reported to be associated <br /> sites or secondary structure), which may increase muta-      with evolutionary rates. For example, highly expressed <br /> tion rate. Meanwhile, CTOs may lead to disruptions of        genes, essential genes, and genes with large codon usage <br /> operon structure and transcriptional regulations, which      bias tend to evolve slowly [7-9]. Meanwhile, a higher <br /> are potentially deleterious. Therefore, currently existing   mutation rate was observed in the genes near Ter[10] <br /> CTOs may signify relaxation of selection pressure on         and the genes that were subject to orientation conflicts <br /> the affected genes. Nevertheless, it remains unknown         between DNA replication and transcription [3]. Our <br /> whether selection actually plays an important role in        results suggest that COGs have significantly higher dN <br /> maintaining CTOs. In view of the potential influences of     and dS than SOGs. Such increases are independent of <br /> CTOs on gene evolution, we are interested in investigat-     the analyzed confounding factors, and are mainly muta- <br /> ing whether CTO per se has any effect on the evolution-      tion-driven. <br /> ary rates of prokaryotic genes. <br />   Note that the evolution of prokaryotic genes is driven     Methods <br /> by two major forces: mutation and natural selection. An      Data sources <br /> increased mutation rate can accelerate short-term            The complete genomic sequences and gene annotations <br /> nucleotide substitutions, which are then retained or         of Escherichia coli K-12 MG 1655 (GenBank accession <br /> eliminated by natural selection according to their fitness   number U00096), Klebsiella pneumonia MGH 78578 <br /> effects. The evolutionary rates of prokaryotic genes         (CP000647) and Salmonella enterica serovar Typhimur- <br /> mainly reflect the combinatorial effects of these two        ium LT2 (AE006468) were retrieved from the GenBank <br /> forces. In general, substitutions that cause protein         [11]. These species were selected because their genomes <br /> sequence changes (nonsynonymous substitutions) are           had been completely sequenced and carefully annotated, <br /> subject to stronger selection pressure than those that do    and because genome-scale gene expression data were <br /> not (synonymous substitutions). By comparing nonsy-          available for these species. The orthologous gene pairs <br /> nonymous (dN) and synonymous substitution rate (dS),         were identified according to reciprocal BLASTP best <br /> we may evaluate whether mutation or selection plays a        matches with the following parameters: (1) E value &lt; 1.0 <br /> more important role in the evolution of prokaryotic          x 10 -5 ; and (2) &gt; 50% amino acid sequence identity. <br /> genes.                                                       Only the orthologous genes that were found in all of the <br />   Here we attempt to examine (1) the effects of CTOs         three compared species were retained. A total of 2,574 <br /> on prokaryotic gene evolution; and (2) the relative con-     one-to-one orthologous gene groups were therefore <br /> tributions of the two abovementioned driving forces to       obtained for subsequent analyses. Since one of our ana- <br /> dS and dN in the genes that have experienced CTOs.           lyses measured the evolutionary rates separately for the <br /> To this end, we compared the dS and dN between the           terminal and central regions for individual genes, the <br /> genes that have experienced CTOs (changed-orientation        analyzed genes must be sufficiently long to reduce varia- <br /> genes, &#226;&#8364;&#339;COGs&#226;&#8364;&#65533;) and those that do not (same-orientation       tions in evolutionary rate estimates. A previous study <br /> genes, &#226;&#8364;&#339;SOGs&#226;&#8364;&#65533;) in closed related prokaryotic genomes.        suggested that the first 50 and the last 20 codons usually <br /> In the case of mutation-driven evolution, COGs and           evolved more slowly than the rest [12,13]. Therefore, we <br /> SOGs should have significantly different dS but approxi-     took twice the <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> of 70 (2*(50+20) = 140) to include <br /> mately the same dN/dS ratios. Alternatively, if selection    a central region with a minimal length of 70 codons. <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                           Page 3 of 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> Accordingly, genes that are shorter than 420 bp (140*3)       expression was log2-transformed and normalized by <br /> were discarded. Note that this practice will lead to dif-     using quantile normalization. For cross-species compari- <br /> ferences in the number of analyzed gene pairs between         sons, the gene expression levels for each species were <br /> different pair-wise species comparisons. The numbers of       standardized to a median value of 0 and a variance of 1. <br /> analyzed gene pairs are listed in Additional file 1. Since <br /> the reciprocal-BLASTP approach may not be an optimal          Functional enrichment analysis <br /> solution for finding orthologous genes, we also retrieved     Gene Ontology (GO) enrichment analysis was per- <br /> orthologous genes for the analyzed species from the           formed by using the DAVID Bioinformatics Resources <br /> OMA database for comparison [14]. In fact, over 98% of        [20,21]. Since the genes of ECO are more extensively <br /> the orthologous gene pairs retrieved from OMA were            studied than those of the other two species, only the <br /> identical to those identified by using reciprocal BLASTP      COGs in ECO-STM and ECO-KPN comparisons were <br /> matches (see Additional file 2). We actually observed         included in this analysis. The gene clustering and GO <br /> similar results when these 98% of orthologous genes           term enrichment were assessed with reference to the <br /> were analyzed (i.e. COGs have significantly higher dS         enrichment score and the p-values of the modified Fish- <br /> and dN but similar dN/dS when compared with SOGs;             er&#226;&#8364;&#8482;s Exact test. <br /> all p&lt;0.05, see Additional file 3). <br />   The locations of Ori and Ter were determined by             Results and discussion <br /> using the program Oriloc [15].                                COGs have higher dN and dS but similar dN/dS ratio <br />                                                               when compared with SOGs <br /> Estimation of evolutionary rates and statistical analyses     In all the pair-wise species comparisons (ECO-KPN, <br /> The amino acid sequences of orthologous gene pairs            STM-KPN, and ECO-STM), the COGs have significantly <br /> were aligned by using ClustalW 2.0 [16] and back-trans-       higher dS and dN than SOGs (all p&lt;0.05). However, the <br /> lated to nucleotide sequences. dS and dN were esti-           dN/dS ratio does not show significant difference <br /> mated by using the CODEML module of PAML 4.1                  between COGs and SOGs in any of the pair-wise com- <br /> [17]. Only orthologous gene pairs with a dS value of &lt; 3      parisons (Figure 1). We also used a three-way (ECO- <br /> were considered [18]. Note that this will also lead to dif-   STM-KPN) comparison to infer lineage-specific COGs, <br /> ferences in the number of analyzed genes in different         and &#226;&#8364;&#339;completely conserved SOGs&#226;&#8364;&#65533; (i.e. genes that never <br /> pair-wise species comparisons. In the end, 1,784, 1,809       changed transcriptional orientation among the three <br /> and 2,069 orthologous gene pairs were obtained, respec-       compared species). As shown in Additional file 5, line- <br /> tively, for the ECO-KPN, STM-KPN, and ECO-STM                 age-specific (ECO-only and STM-only) COGs have sig- <br /> comparison (Additional file 1). Since dS &gt; 3 may indi-        nificantly higher dS and similar dN/dS ratios when <br /> cate an extremely high mutation rate and loss of func-        compared with completely conserved SOGs. These <br /> tion, we also investigated whether COGs tend to have          observations suggest that the increases in evolutionary <br /> dS &gt; 3. Indeed, the proportion of COGs with dS &gt; 3 is         rates in COGs are mainly mutation-driven. However, a <br /> higher than that of all of the analyzed genes (see Addi-      number of confounding factors need to be excluded to <br /> tional file 4). This observation is in fact consistent with   further confirm these results. The confounding factors <br /> our results that COGs tend to evolve faster than SOGs.        to be controlled include gene essentiality, gene expres- <br />   Spearman&#226;&#8364;&#8482;s rank correlations and partial correlations       sion level, background mutation rate, the transcription- <br /> between evolutionary rates and the rate-determining fac-      replication confrontation, and codon usage bias. <br /> tors were performed by using the R program (http:// <br /> www.r-project.org). The statistical significance of the       Confounding factor 1: gene essentiality <br /> evolutionary rate differences was evaluated by using the      Essential genes are known to be more evolutionarily <br /> Wilcoxon Rank Sum test throughout the study.                  conserved than nonessential genes [8]. It is therefore <br />                                                               necessary to clarify whether COGs tend to be non- <br /> Analysis of gene expression data                              essential, so that they evolve faster than SOGs. We thus <br /> The genome-scale gene expression data of ECO                  retrieved the gene essentiality information of E.coli[22], <br /> (GSE15534) and STM (GSE11486) were downloaded                 and divided the orthologous genes into essential and <br /> from the Gene Expression Omnibus (GEO) database               nonessential genes in the ECO-KPN and ECO-STM <br /> [19]. The gene expression data of KPN were generated          comparisons. Our results indicate that dN and dS are <br /> by using a custom-made NimbleGen tiling array (pro-           significantly higher for COGs than for SOGs in nones- <br /> vided by Dr. Bernhard O. Palsson at University of Cali-       sential genes (Table 1). We observed similar trends in <br /> fornia, San Diego). In all of the three data sets, the gene   essential genes, although the differences in dN and dS <br /> expression levels were measured separately at the log         are statistically insignificant possibly due to the small <br /> and stationary phase. The signal intensity of gene            sample sizes and the reduced variations in evolutionary <br />  <br />   Figure 1 The evolutionary rates of COGs and SOGs in the ECO-KPN and STM-KPN comparisons. *: p-value &lt; 0.05; **: p-value &lt; 0.01; ***: <br />   p-value &lt; 0.0001 <br />  <br />  <br />  <br /> rates in this gene group. Meanwhile, the dN/dS ratios                                       evolve more slowly than lowly expressed genes [23,24]. <br /> are similar between COGs and SOGs, consistent with                                          The question now is whether SOGs tend to be highly <br /> our previous observations. Therefore, the difference in                                     expressed, so that they evolve more slowly than COGs. <br /> the level of gene essentiality does not seem to affect our                                  We thus classified the analyzed genes into highly (top <br /> result. This is expected if the COG-SOG differences in                                      20%) and non-highly (other 80%) expressed genes. In <br /> evolutionary rates are mainly mutation-driven.                                              the ECO-KPN and STM-KPN comparisons, the dS of <br />                                                                                             COGs is significantly larger than SOGs in both highly <br /> Confounding factor 2: gene expression level                                                 and non-highly expressed genes for both growth phases <br /> Highly expressed genes are under strong selection pres-                                     (stationary and log phase, Table 1). dN values also <br /> sure to maintain functional stability, and thus usually                                     show similar trends. However, the dN/dS ratios are <br />  <br /> Table 1 The median evolutionary rates of COGs and SOGs when different confounding factors are controlled. <br /> confounding factors                        comparison                     #gene (COG/                dN                                dS                       dN/dS <br />                                                                              SOG) <br />                                                                                              COGs SOGs       p-     COGs              SOGs      p-      COGs    SOGs      p- <br />                                                                                                             value                              value                     value <br /> nonessential gene <br />                                              ECO-KPN                       121/1426          0.076 0.067      *     1.6811 1.466                ***    0.0481 0.0463       - <br />                                              ECO-STM                       240/1582          0.057 0.045     ***    1.5921 1.1926               ***    0.0395 0.0381       - <br /> essential gene <br />                                              ECO-KPN                        8/224            0.045 0.041      -     1.374             0.978      -      0.042   0.046      - <br />                                              ECO-STM                        17/225           0.025 0.025      -     0.785             0.825      -      0.029   0.032      - <br /> non-highly expressed <br /> gene <br />                                       ECO-KPN (log phase)                  106/1240          0.082 0.074      -     1.721             1.561     **      0.049   0,047      - <br />                                   ECO-KPN (stationary phase)               103/1342          0.077 0.066      -     1.686             1.457     ***     0.048   0.046      - <br />                                       STM-KPN (log phase)                   88/1416          0.077 0.068      -     1.845             1,507     ***     0.042   0.045      - <br />                                   STM-KPN (stationary phase)                80/1475          0.077 0.065      -      1.77             1.458     ***     0.042   0.046      - <br /> highly expressed <br />                                       ECO-KPN (log phase)                   23/415           0.045 0.037      -     1.352             0.896     **      0.042   0.044      - <br />                                   ECO-KPN (stationary phase)                26/313           0.064 0.047      -     1.459             1.142      *      0.047   0.045      - <br />                                     STM-KPN (log phase)                     18/287           0.06 0.037       -     1.384             0.895      *      0.046   0.046      - <br />                                   STM-KPN (stationary phase)                26/228           0.075 0.053      *     1.845             1.273     ***     0.042   0.044      - <br /> * : p &lt; 0.05: **: p &lt; 0.01: ***: p &lt; 0.001; -: insignificant <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                Page 5 of 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> approximately the same between COGs and SOGs                             location-independent effect on dS. Meanwhile, the <br /> (Table 1). Notably, it has been previously reported that                 increase in dN in COGs is likely affected by the chro- <br /> mutation rate may increase with gene expression level                    mosomal locations of these genes. However, since the <br /> [25-27]. Therefore, it is of interest to investigate whether             COGs that are close to Ter far outnumber those that <br /> the increases in evolutionary rate in COGs are asso-                     are not, more evidence is needed to clarify the correla- <br /> ciated with increased expression levels in these genes.                  tion between dN and the COG/SOG gene type in the <br /> However, our results indicate that this is not the case.                 context of background mutation rate. <br /> Highly expressed COGs actually have lower dN and dS <br /> than non-highly expressed COGs (Additional file 6).                      Confounding factor 4: replication&#226;&#8364;&#8220;transcription <br /> Therefore, the increases in evolutionary rates (particu-                 confrontation <br /> larly dS) in COGs do not result from increased expres-                   The head-on collision between the directions of tran- <br /> sion levels in these genes.                                              scription and replication has been reported to increase <br />                                                                          the mutation rate of the affected genes [3,10,29]. When <br /> Confounding factor 3: background mutation rate                           a gene is involved in an inversion event, its transcrip- <br /> Although gene inversions usually occur in the vicinity of                tional orientation is likely to change, causing replica- <br /> both Ori and Ter[1,2], we actually found that most                       tion-transcription confrontation. We thus investigated <br /> COGs analyzed in this study were located near Ter                        whether COGs have replication-transcription confronta- <br /> (Figure 2). It is known that the genes located near Ter                  tion more often than SOGs. Interestingly, most of the <br /> have an increased mutation rate [28]. We then ask                        COGs maintain their status in terms of replication-tran- <br /> whether the uneven chromosomal distribution of COGs                      scription confrontation. For example, using KPN as an <br /> has contributed to the COG-SOG differences in evolu-                     outgroup, we found that 157 of 162 COGs in the ECO- <br /> tionary rates. Therefore, we performed partial correla-                  STM comparison maintain the status of head-on colli- <br /> tion analyses to examine the relationship between                        sion or co-orientation between replication and transcrip- <br /> evolutionary rates and gene type (COG or SOG). Our                       tion after the occurrences of the inversion events. <br /> results indicate that dS is significantly correlated with                Meanwhile, the ratio of head-on-collision to co-orienta- <br /> the COG/SOG gene type even when the distance effect                      tion genes in COGs and SOGs are 1.1 and 0.64, respec- <br /> is controlled (Spearman&#226;&#8364;&#8482;s r &gt; 0.05 and p &lt; 0.05) (Addi-                  tively in the ECO&#226;&#8364;&#8220;STM comparison. Therefore, COGs <br /> tional file 7). The positive correlation indicates that                  are in fact more likely to be head-on-collision genes <br /> CTOs have a location-independent effect on the                           than SOGs. To further clarify whether the difference in <br /> increase of dS. Meanwhile, the distance-controlled cor-                  the degree of head-on collision is a major determinant <br /> relations are statistically insignificant between dN (and                of the COG-SOG differences in evolutionary rate, we <br /> also dN/dS) and the COG/SOG gene type. Together,                         compared the evolutionary rates of COGs and SOGs by <br /> these results indicate that CTOs actually have a                         controlling the replication-transcription confrontation <br />  <br />  <br />  Figure 2 Dot plot for the analyzed orthologous genes in the (a) ECO-KPN; and (b) STM-KPN comparison. The Y and X axis represents, <br />  respectively, the chromosomal positions of the KPN genes (in <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> pair) and those of the orthologous genes in ECO (left panel) or STM (right <br />  panel). The black and red dots represent SOGs and COGs, respectively. Most of the COGs are located close to Ter. The positions of Ter are <br />  1,892,000, 1,549,000 and 1,635,000 bp for KPN, ECO and STM, respectively. <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                        Page 6 of 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> pattern. In the ECO-STM comparison, the rates of dN,                         Functional enrichment analysis of COGs <br /> dS and dN/dS of head-on-collision COGs do not differ                         We next ask whether the rapidly evolving COGs are <br /> significantly from co-orientation COGs (Additional File                      enriched in certain functional categories. Additional file <br /> 8). By contrast, head-on-collision SOGs have signifi-                        10 indicates that the protein products of COGs tend to <br /> cantly higher evolutionary rates than co-orientation                         be located on membranes and cell walls, and be <br /> SOGs. These results suggest that the higher evolutionary                     involved in a variety of metabolic reactions. This obser- <br /> rates of COGs do not result from a higher proportion of                      vation is biologically sensible because (1) proteins that <br /> head-on-collision genes. Rather, CTO by itself may be                        are located on the cell surface tend to evolve faster; and <br /> the major reason for the increased evolutionary rate.                        (2) different bacterial species have very different meta- <br />                                                                              bolic capacities. However, the relationship between <br /> Confounding factor 5: decreased dS at gene terminals                         CTOs and the functional preferences of these genes <br /> (codon usage bias)                                                           remains unknown. <br /> It has been reported that the synonymous substitution <br /> rate decreases near the first 50-100 codons [12] and the                     Conclusions <br /> last 20 codons of a gene [13], possibly because codon                        To our knowledge, this is the first study to demonstrate <br /> usage is less biased at these regions. We then ask                           that changes in transcriptional orientation may increase <br /> whether SOGs tend to have lowered dS at both ends as                         the evolutionary rates (dN and dS) of prokaryotic genes. <br /> compared with COGs, so that the overall dS is                                We show that the evolutionary effects of CTOs are inde- <br /> decreased. Accordingly, we divided each of the analyzed                      pendent of gene essentiality, gene expression level, replica- <br /> genes into terminal (first 50 and last 20 codons) and                        tion-transcriptional confrontation, or the decrease in dS at <br /> middle region (the rest of the codons). We then ran-                         gene terminals. However, the increase in dN may be par- <br /> domly sampled 70 codons from the middle region for                           tially related to gene locations. Furthermore, our results <br /> comparison. The differences in evolutionary rates (&#206;&#8221;dS,                      suggest that the increases in evolutionary rates in COGs <br /> &#206;&#8221;dN, and &#206;&#8221;dN/dS) between the terminal codons and                             are mainly mutation-driven, as the dN/dS ratios are similar <br /> the randomly sampled codons were calculated and com-                         between COGs and SOGs. The real cause of the increases <br /> pared between COGs and SOGs. Our results show that                           in evolutionary rate in COGs (particularly dS) remains <br /> &#206;&#8221;dS, &#206;&#8221;dN, and &#206;&#8221;dN/dS are similar between COGs and                            unclear but is worth further explorations. It is speculated <br /> SOGs (Figure 3 for ECO-KPN comparison and Addi-                              that CTOs may somehow result in impediments in DNA <br /> tional file 9 for STM-KPN comparison; all of the pair-                       replication (by altering, for example, DNA protein binding <br /> wise differences are statistically insignificant). Therefore,                sites or secondary structure), which may in turn lead to <br /> the difference in codon usage bias in evolutionary rate                      the recruitment of error-prone DNA repair polymerases <br /> does not appear to affect our overall results.                               and the increase in mutation rate [30,31]. <br />  <br />                  all genes COGs      SOGs                         all genes COGs        SOGs                              all genes COGs    SOGs <br />  Figure 3 Difference in evolutionary rates between the terminal and middle region of ECO-KPN orthologous genes. The Y axis shows <br />  the difference (terminal minus middle) in dN, dS, and dN/dS ratio, respectively, for the left, middle, and right panel. The differences in dN, dS and <br />  dN/dS ratio are statistically insignificant between any pair-wise gene group comparisons in each panel. <br />  <br />  <br /> Additional material                                                             Authors&#226;&#8364;&#8482; contributions <br />                                                                                 CHL conceived the study and carried out the data analyses. CHL, FCC and <br />                                                                                 CAH designed the conceptual framework of the study, interpreted the <br />     Additional file 1: The numbers and detailed information of the              results, and drafted the manuscript. CYL pre-processed gene expression data <br />     orthologous gene pairs analyzed in this study.                              and assisted in the statistical analyses. <br />     Additional file 2: The numbers of ECO-KPN, ECO-STM, and STM-KPN <br />     orthologous gene pairs identified by reciprocal BLASTP and the             <br />     OMA database                                                                <br />     Additional file 3: The evolutionary rates of COGs and SOGs in the <br />                                                                              <br />     ECO-KPN and STM-KPN comparisons. The genes included in this <br />     analysis were identified by both reciprocal BLASTP and OMA <br />     database as orthologous genes. *: p-value &lt; 0.05; **: p-value &lt;             <br />     0.01; ***: p-value &lt; 0.0001                                           <br />     Additional file 4: The numbers of COGs with dS &gt; 3 in the ECO-KPN, <br />                                                                            <br />     ECO-STM, and STM-KPN comparisons. <br />                                                                      <br />     Additional file 5: The evolutionary rates in (a) ECO-KPN comparison;         <br />     and (b) STM-KPN comparison. Here we compare the evolutionary                    <br />     rates of the genes that never changed transcriptional orientation in      <br />     ECO, STM, and KPN, (&#226;&#8364;&#339;ESK&#226;&#8364;&#65533;), the genes that changed orientation in               <br />     either the ECO-STM lineage or in the KPN lineage (&#226;&#8364;&#339;ES/K&#226;&#8364;&#65533;), and the           <br />     genes that changed orientation in only one species (&#226;&#8364;&#339;ECO-only&#226;&#8364;&#65533; or          <br />     &#226;&#8364;&#339;STM-only&#226;&#8364;&#65533;).                                                             <br />     Additional file 6: Comparison of the evolutionary rates of highly           <br />     and non-highly expressed COGs. (a) ECO-KPN comparison at the         <br />     log phase; (b) ECO-KPN comparison at the stationary phase; (c)                 <br />     STM-KPN comparison at the log phase; (d) STM-KPN comparison at            <br />     the stationary phase. *: p-value &lt; 0.05; **: p-value &lt; 0.01             <br />     Additional file 7: Partial correlations between evolutionary rates <br />                                                                                  <br />     and the COG/SOG gene type while the physical distance from Ter is <br />                                                                             <br />     controlled. <br />                                                                                     <br />     Additional file 8: The evolutionary rates of COGs and SOGs that are          <br />     subject to head-on collision or co-orientation between DNA       <br />     replication and transcription in the ECO-STM comparison. *                   <br />     represents p value &lt; 0.05.                                                <br />     Additional file 9: Comparison of &#206;&#8221;dN, &#206;&#8221;dS and &#206;&#8221;dN/dS between                <br />     COGs and SOGs in the STM-KPN comparison. &#206;&#8221;dN, &#206;&#8221;dS and &#206;&#8221;dN/dS                    <br />     were calculated by subtracting the evolutionary rates of the middle        <br />     region from those of the terminal regions of each gene. Note that          <br />     none of the pair-wise comparisons in any of the three panels is              <br />     statistically significant.                                                 <br />     Additional file 10: Functional enrichment analyses for COGs in the <br />     (a) ECO-KPN and (b) ECO-STM comparisons. Note that the <br />                                                                                    <br />     functional assignments were based on the gene annotations for <br />                                                                                     <br />     ECO. <br /> </body></html>