<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Role Protein Interactions Mediating Essentiality <br /> and Synthetic Lethality <br /> David Talavera, David L. Robertson, Simon C. Lovell* <br /> Faculty Life Sciences, University Manchester, Manchester, United Kingdom <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Genes characterized essential <span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>knockout</span> associated lethal <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span>, &#226;&#8364;&#732;&#226;&#8364;&#732;essential genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; play <br />      central role biological function. In addition, genes essential deleted pairs, phenomenon known <br />      synthetic lethality. Here consider genes displaying synthetic lethality &#226;&#8364;&#732;&#226;&#8364;&#732;essential pairs&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes, analyze the <br />      properties yeast essential genes synthetic lethal pairs together. As gene duplication initially produces identical <br />      pair sets genes, invoked explanation synthetic lethality. However, duplication explains <br />      minority cases synthetic lethality. Similarly, disruption metabolic pathways leads relatively examples of <br />      synthetic lethality. By contrast, vast majority synthetic lethal gene pairs <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins related functions that <br />      share interaction partners. We essential genes synthetic lethal pairs cluster protein-protein <br />      interaction network. These results suggest synthetic lethality strongly dependent formation protein-protein <br />      interactions. Compensation duplicates does usually occur mainly genes involved recent duplicates, <br />      commonly functional similarity permits preservation essential protein complexes. This unified <br />      view, combining genes individually essential form essential pairs, suggests essentiality is <br />      feature physical interactions proteins protein-protein interactions, inherent gene and <br />      protein products themselves. <br />  <br />   Citation: Talavera D, Robertson DL, Lovell SC (2013) The Role Protein Interactions Mediating Essentiality Synthetic Lethality. PLoS ONE 8(4): e62866. <br />   doi:10.1371/journal.pone.0062866 <br />   Editor: John Parkinson, Hospital Sick Children, Canada <br />   Received September 7, 2012; Accepted March 27, 2013; Published April 29, 2013 <br />   Copyright: &#195;&#376; 2013 Talavera et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Biotechnology Biological Sciences Research Council (www.bbsrc.ac.uk), grant number BB/F013337/1. The funders <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: simon.lovell@manchester.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                         synthetic-lethality, genes essential mutated <br />                                                                                      deleted individually, lethal altered simultaneously. <br />    In organisms complex relationship                          This phenomenon understood buffering effect <br /> genotype phenotype. A key tool genetic research                        [10]: gene X buffer phenotypic effect loss gene Y <br /> generation analysis null mutations, particular                      vice versa. However, lost simultaneously no <br /> gene rendered non-functional, deletion. Analysis                    buffering possible. Alternatively, direct, <br /> of <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>null mutants</span> allows identification &#226;&#8364;&#732;&#226;&#8364;&#732;essential genes&#226;&#8364;&#8482;&#226;&#8364;&#8482;,              additive effect, deletion causes decrease ability <br /> are inferred significant contributions function                  pathway function. A number mechanisms involving the <br /> in given environment.                                                              protein products explain buffering come about <br />    In yeast (Saccharomyces cerevisiae) 20% genes                  (Figure 1), including previously proposed [10]. Using the <br /> essential grown laboratory [1]. Many factors                             nomenclature Kelley Ideker [11], A, B, D E are <br /> determine given gene classed essential: essentiality                 &#226;&#8364;&#732;&#226;&#8364;&#732;within pathway&#226;&#8364;&#8482;&#226;&#8364;&#8482; explanations genetic interactions, C <br /> has linked protein product highly connected                   F &#226;&#8364;&#732;&#226;&#8364;&#732;between pathway&#226;&#8364;&#8482;&#226;&#8364;&#8482; explanations. The terms &#226;&#8364;&#732;&#226;&#8364;&#732;within <br /> networks protein-protein interactions [2], so-called                module&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;between module&#226;&#8364;&#8482;&#226;&#8364;&#8482; used similar <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;centrality-lethality rule&#226;&#8364;&#8482;&#226;&#8364;&#8482; disputed [3]. Alternatively,                  concepts [12]. <br /> genes essential proteins <span id='am-479' about='Thesaurus:code' typeof='owl:Thing'>code</span> involved in <br />                                                                                         Protein-protein interactions account relatively small <br /> specific biological modules protein complexes [4] they <br />                                                                                      proportion genetic interactions [9]. Indeed, protein complexes <br /> participate subset interactions essential <br />                                                                                      account subsets monochromatic genetic <br /> [5]. Nevertheless, clear essential nature (or otherwise) <br />                                                                                      interactions (mostly positive negative, equally <br /> of gene strongly dependent context environmental <br />                                                                                      mixed) grouped functional modules [12]. Interestingly, one <br /> conditions [6,7]. <br />                                                                                      recent study related strength genetic interaction <br />    An important aspect context gene functions is <br />                                                                                      (either positive negative) proportion common <br /> the genetic background expressed [8]. Genetic <br />                                                                                      physical interactors [13]. Similarly, genetic interactions been <br /> interactions common [9], arise simultaneous <br />                                                                                      rationalized using metabolic models [14]. Although are <br /> <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span> gene results phenotypic effects <br />                                                                                      discrepancies predicted genetic interactions those <br /> greater multiplicative effects mutating gene <br /> individually. The extreme case negative genetic interaction                  measured experimentally, pleiotropy explained <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                                  April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                  Protein Interactions Essentiality <br />  <br />  <br />                                                                              interactions. For instance, unknown synthetic lethality <br />                                                                              (and synthetic sickness) regulated mechan- <br />                                                                              isms, different origins, e.g. set Mendelian traits <br />                                                                              quantitative traits. <br />                                                                                 Of basic potential mechanisms synthetic lethality <br />                                                                              arise (Figure 1), mechanisms A, C, D F clearly arise <br />                                                                              gene duplication. In mechanism A, proteins X Y, <br />                                                                              coded synthetic lethal pair genes X Y, functionally <br />                                                                              compensate binding partner the <br />                                                                              interface. If X Y recently duplicated paralogs, their <br />                                                                              protein products expected similar binding <br />                                                                              specificities, able compensate way. Similarly, in <br />                                                                              mechanism D, proteins X Y carry step in <br />                                                                              metabolic signaling pathway. Mechanisms C F arise <br />                                                                              duplication subsystem. In mechanism C, the <br /> Figure 1. Potential mechanisms rise synthetic               entire complexes X Y members be <br /> lethality (essential pairs). X Y indicate proteins coded <br />                                                                              functionally redundant, paralogous. Duplication of <br /> for synthetic-lethal gene pairs. A&#226;&#8364;&#8220;C gray indicates protein <br /> complexes; D&#226;&#8364;&#8220;E gray circles indicate proteins         complexes happen concert remain <br /> a pathway. A. Proteins X Y functionally redundant members         functional; achieved whole-genome duplica- <br /> a multi-protein complex. B. Proteins X Y members         tion [33]. Similarly mechanism F arise duplication of <br /> same protein complex. C. Proteins X Y essential members           subsection pathway. Alternatively both <br /> two different protein complexes functionally compensate        mechanisms C F alternative independent <br /> other. D. Proteins X Y redundant members metabolic <br /> pathway. E. Proteins X Y play synergistic role metabolic          mechanism produce function. In case <br /> pathway. F. Proteins X F essential members alternate              functional redundancy arise members the <br /> metabolic pathways.                                                          complexes pathways paralogous. Obviously, basic <br /> doi:10.1371/journal.pone.0062866.g001                                        mechanisms complicated; e.g.: parallel <br />                                                                              pathways, present. <br /> through use models [14]. An alternative hypothesis                  Many proteins pairs coded synthetic lethal genes <br /> relates synthetic lethality rearrangements genetic                 participate protein complexes [34], synthetic lethality may <br /> interaction network gene deleted [15]; however,           arise paralogs compensate complexes <br /> we understand rearrangements occur,                (Figure 1A) [35]. However, studies relationship <br /> possible predict outcome novel genetic interactions.               genetic interactions physical interaction networks <br />    A key aspect genetic context gene presence              concluded majority genetic interactions occur <br /> a duplicate copy. Gene duplication initially produces            proteins different complexes [11,36], suggesting <br /> identical genes diverge function [16]. These duplicated          minimal functional compensation. Interestingly, genome-wide <br /> genes functionally compensate [17,18],               association study reached conclusions [37]. Mapping the <br /> up-regulating expression remaining gene needed [19].           majority genetic interactions yeast demonstrates 1) there <br /> More 30% yeast duplicate genes functionally                      modest overlap genetic interactions direct <br /> redundant, case arise                physical interactions 2) proteins similar functions <br /> genome duplication smaller scale duplications [16,20]. More-              similar <span id='am-480' about='xsp:pattern' typeof='owl:Thing'>pattern</span> genetic interactions [38]. Exceptions <br /> over, little evidence functional redundancy             include sets interactions termed &#226;&#8364;&#732;&#226;&#8364;&#732;monochromatic&#226;&#8364;&#8482;&#226;&#8364;&#8482; (i.e., <br /> unrelated singleton genes. Thus, genes                    predominantly positive predominantly negative); modules <br /> duplicated likely essential              defined way frequently associated protein <br /> duplicated [17,21,22]. However, true mammals                  complexes, account minority cases <br /> [21,23&#226;&#8364;&#8220;25], probably complexity development                 [12]. Two studies demonstrated protein complexes <br /> higher organisms [26,27].                                                    negative interactions components coded by <br />    Duplicates high degree sequence similarity             essential genes [39,40]. Surprisingly, studies included genetic <br /> more likely able functionally substitute [28].       interactions essential genes carrying hypomorphic mutations, <br /> Conversely, genes duplicated genes               affect results proteins time <br /> the duplicate pairs high degree sequence divergence            criterion analyzing data (complexes with <br /> unlikely able compensate [22,29]. Although           monochromatic negative genetic interactions), object of <br /> the maintenance highly-similar paralogs detrimental              study. <br /> [30], examples network redundancy ensure                    Here, present study relationship essential <br /> a high metabolic flux [6] provide rapid response changing            genes synthetic lethal pairs yeast, consider having <br /> conditions. An example the regulation gene               paired essentiality. We restrict analysis synthetic lethal pairs, <br /> expression transcription factors [31].                                    differentiate negative genetic interactions, an <br />    There estimates percentage negative           approach differs previous studies [12,39,40]. We <br /> genetic interactions caused simultaneous deletion           demonstrate 1) presence duplicate copies the <br /> of duplicate genes [18,32]; however, estimates greatly differ.           major determinant gene essential and, 2) products <br /> Some differences size dataset negative         essential genes members synthetic lethal pairs frequently <br /> interactions, methodology finding duplicates. As            interact other. We expand analysis He Zhang <br /> study exclusively devoted synthetic lethal pairs,          [5] include essential genes essential pairs. We therefore <br /> not know proportions particular set negative genetic         present new view essentiality, emphasizing importance of <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                    Protein Interactions Essentiality <br />  <br />  <br /> protein complexes. This view focuses protein-protein interac-               robustness existence alternative pathways or <br /> tions, proteins themselves.                                 complexes perform function paralogs <br />                                                                                [10,22]. Alternatively, members essential pair be <br /> Results                                                                        present complex metabolic pathway [10], adding <br />                                                                                small synergistic effect (see Figure 1B&amp;E). <br /> Gene Duplication does Explain Synthetic Lethality <br />    Presence absence duplicate genes used explain             Members Synthetic Lethal Pairs Functionally <br /> the existence synthetic lethal pairs essential genes,                   Related <br /> respectively. If duplicate genes main explanation we <br />                                                                                    Synthetic-lethal pairs able compensate other <br /> would expect 1) nearly essential genes singletons (i.e., <br />                                                                                physical substitution are, <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, interchangeable, <br /> without paralog); and, 2) synthetic lethal pairs be <br />                                                                                functionally related. Valente co-workers <br /> paralogous. Importantly, expect gene <br />                                                                                [35] examples paralogs likely be <br /> duplications rise synthetic lethal pair [18]. <br />                                                                                interchangeable obligate transient complexes. In <br />    We yeast 79.1% essential genes selected using the <br />                                                                                agreement low percentage synthetic lethal pairs that <br /> stringent criteria singletons (78.5% selected using the <br />                                                                                paralogs (Figure 2 Table S1), 5.4% <br /> tolerant criteria, methods section selection criteria; Figure 2 <br />                                                                                (3.6% using tolerant criterion) protein products the <br /> and Table S1). This higher expected chance <br />                                                                                essential gene pairs functional domain, defined <br /> (58,9%; p-value ,1024). By contrast, 50.1% genes in <br />                                                                                InterPro, common (Figure 3 Table S2). Even this <br /> synthetic lethal pairs selected using stringent criterion are <br />                                                                                percentage higher random expectation, confirms that <br /> singletons (49.5% selected using tolerant criteria), <br />                                                                                small minority synthetic lethal pairs generated by <br /> which lower expected (56.3%; p-value = 361024). These <br />                                                                                simultaneous knockout evolutionary-related genes. <br /> results agree previous research, indicating single-gene- <br />                                                                                    By contrast, disruption related functions explain <br /> associated essentiality derives lack paralogs [21], that <br />                                                                                large proportion synthetic lethality, demonstrated in <br /> synthetic lethality originates gene duplication [22,29,35,41]. <br />                                                                                studies using synthetic genetic arrays [42]. We <br /> However, complete set negative interactions <br /> [17,18,32], complete explanation. First,               identified functions gene compared assignments <br /> 20% yeast essential genes paralogs yeast genome.                members synthetic lethal pairs. More than <br /> Second, half genes synthetic lethal pairs            expected chance (60.0% vs. 12.6% using stringent <br /> paralog. Third, half genes members                 criterion; p-value ,1024) synthetic lethality involves deletion <br /> synthetic lethal pairs paralogs, percentage synthetic              genes working related functions (Figure 3 and <br /> lethal pairs members paralogous genes just 4.3%               Table S2). This indicates examples synthetic lethality <br /> (2.5%, using tolerant criteria). This percentage              caused single functional process disrupted, by <br /> similar originally set quantitative             deletion redundant proteins, disruption alternate <br /> negative interactions [32] recent estimates [17,18].              pathways. Our figures higher Tong and <br />    Some paralogs essential genes correspond                 coworkers [32]; results difficult compare <br /> highly diverged duplicates paralogs substitution                    consider synthetic sick pairs synthetic lethal <br /> functionally important residues. In cases, modifications                  pairs, restrict analysis synthetic lethality. It <br /> might prevent functional compensation [16,28]. In case                  functional compensation mainly related <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br /> synthetic lethal gene pairs, unlikely pairs        synthetic lethality, synthetic sick interactions emerging to <br /> duplicates able compensate functionally,                  different causes (Table S3). However, differences sample size <br /> diverged prevent identification paralogs. It            ambiguity <span id='am-7' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> genetic interactions <br /> therefore highly probable mechanisms duplication               suggest caution interpreting result. There possibility of <br /> events contribute determining gene                 &#226;&#8364;&#732;&#226;&#8364;&#732;study bias&#226;&#8364;&#8482;&#226;&#8364;&#8482;, genes selected experimental testing are <br /> essential.                                                                     likely functionally related. This bias functional relatedness <br />    Some synthetic lethal pairs participate functional             introduced 1) performing small-scale experiments <br /> redundancy duplication. Rather contribute to <br />  <br />  <br />  <br />  <br /> Figure 2. Duplication essential genes members                        Figure 3. Evolutionary functional similarity essential <br /> essential pairs. Genes selected using stringent criteria. The inset       pairs. Genes selected using stringent criteria. For expected values <br /> shows percentage essential gene pairs paralogs.                mean standard deviation shown. <br /> doi:10.1371/journal.pone.0062866.g002                                          doi:10.1371/journal.pone.0062866.g003 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 3                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                     Protein Interactions Essentiality <br />  <br />  <br /> (in 323 440 sources reporting just synthetic lethal pair, <br /> that pair functionally related); or, 2) using epistatic <br /> miniarray profiles (E-MAP), designed testing only <br /> a subset interactions suspected enriched positive or <br /> negative interactions [43,44]. The majority data data set <br /> came experiments using functional information their <br /> design; subsets displayed degree of <br /> functional relatedness (Table S4). <br />  <br /> Most Synthetic Lethal Pairs Caused Deletion of <br /> Members Protein Complexes <br />    Synthetic lethality functional redundancy occur if <br /> two genes code proteins buffer other. Thus, in <br /> protein complexes acting metabolic pathways, deletion of <br /> one gene buffered other&#226;&#8364;&#8482;s presence (Figure 1); wider <br /> explanation causes epistasis, synthetic lethality is <br /> the ultimate consequence, given Lehner [45]. In order               Figure 4. Essential pairs members bind same <br /> determine membership protein complex                          interactor. The interaction networks built combining two <br /> participation pathway equally important understand-              criteria selection essentiality physical interactions (SS two <br /> ing synthetic lethality, calculated percentage synthetic            stringent criteria used; ST stringent criteria used for <br />                                                                               selecting physical interactions; TS essentiality stringently <br /> lethal pairs members assigned metabolic                  selected; TT tolerant criteria used). For expected <br /> pathway detected affinity capture experiments. We                  values mean standard deviation shown. <br /> assume affinity capture experiments retrieve multi-                 doi:10.1371/journal.pone.0062866.g004 <br /> protein complexes. Thus assign status member protein <br /> complex protein involved protein-protein interactions               indicate number synthetic lethal pairs code putatively <br /> detected affinity capture experiments. Our results               interchangeable proteins form protein complexes. In case <br /> that 8961% synthetic lethal pairs contain genes               simultaneous knockout proteins produces lethal <br /> code proteins, suggesting gene products                   phenotype complexes formed. <br /> members different protein complexes the <br /> main reasons synthetic lethality. As proteins be <br />                                                                               &#226;&#8364;&#732;&#226;&#8364;&#732;Essential Proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482; Group Interaction Sub- <br /> found hits affinity capture experiments despite being <br /> involved protein complexes, used alternative                network <br /> more restrictive <span id='am-477' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> complex membership, based GO                   The importance protein complexes synthetic lethality <br /> terms. Although percentage synthetic lethal pairs                indicates formation protein complexes crucial <br /> both genes members protein complexes lower,                 <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span> cell. This implies components <br /> this <span id='am-478' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> figures 4060% using         complexes essential [39]. Accordingly, we <br /> annotations IEA, NAS ND ones, 3860%                hypothesize &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;, regardless they <br /> using annotations experimental evidence code. Converse-               product essential genes members synthetic lethal <br /> ly, disruption metabolic pathways explain                     pairs, highly interconnected. This expansion the <br /> a minority (1260%) cases synthetic lethality. Moreover,                 original essential interactions hypothesis proposed He and <br /> only 2265% synthetic lethal pairs caused metabolic              Zhang [5] supported previous finding some <br /> factor occur pathway, suggesting alternate                   protein complexes enriched single-gene essential proteins <br /> pathways account ,10% synthetic lethal pairs.                          products synthetic lethal pairs [34]. <br />    Synthetic-lethal pairs gene products different complexes              In order verify expected enrichment interactions <br /> are common pairs complex [11,36],                       essential proteins, determined number essential <br /> although methodological statistical biases [11,36].           proteins physically interact essential proteins. As <br /> Nevertheless, examples products                    control, calculated percentage non-essential proteins <br /> synthetic lethal pairs present protein complex                interacting single-gene essential proteins members of <br /> [11,34,36], expected directly interact [9].           synthetic pairs. We 7569% single-gene essential <br /> Indeed, proteins compensatory role expect              proteins 8165% members synthetic lethal pairs interact <br /> will interact other;                    essential proteins, just 57610% non- <br /> interacting partners common [13]. This case                essential proteins interact essential proteins. This <br /> synthetic lethal pairs (Figure 4 Table S5). The observed                  products coded essential genes tend be <br /> percentages higher obtained chance (7.6%               highly connected. <br /> vs. 0.1&#226;&#8364;&#8220;0.7% analyzing SS network; p-value ,1024)                   We determined number interactions between <br /> are independent structure synthetic lethality network,          essential proteins, comparing results networks <br /> since altering network topology substituting reshuffling           annotated using purely randomized lists interactions. Annota- <br /> nodes produce significant decrease number synthetic               tion independent synthetic lethality network (i.e. degree <br /> lethal pairs common physical interactor. Most importantly,             distribution), takes account gene is <br /> for 6364% pairs interacting partner common,                 involved synthetic lethality interaction. First, determined <br /> shared interactor product essential gene               frequency essential-gene proteins interact each <br /> a member synthetic lethal pair. Increase number                other. Although just 2.6%&#226;&#8364;&#8220;3.8% physical interactions occur <br /> experimentally-identified protein-protein interactions               products essential genes, results are <br /> increase number pairs sharing interactors. These findings              higher expected chance (0.3&#226;&#8364;&#8220;0.4%; p-value ,1024; <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                4                                April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  <br /> Figure 5 Table S6). We find, therefore, single-gene                fact proportion products single-gene <br /> essential proteins tend interact [5,46], meaning         essentiality higher networks obligate physical <br /> that cluster functional modules [4], tend         interactions (Table S14). Nonetheless, prefer cautious <br /> to complexes [34,47]. In addition, number             results, caused definition of <br /> physical interactions involving members synthetic lethal pairs        obligate transient interactions (see Methods). <br /> higher expected chance (Figure 5 Table S7). <br /> Finally, observe enrichment interactions              Functional Convergence Diminish Number of <br /> essential genes members synthetic lethal pairs (Figure 5         Essential Genes <br /> Table S8). Since enrichment independent dataset                 Clearly, randomly removing essential proteins analysis <br /> randomisation used, conclude interactions             diminish enrichment essential sub-network. We <br /> caused distribution just type essentiality.                separately excluded (a) members synthetic lethal pairs were <br /> Importantly, removing physical interactions detected                paralogs (b) members synthetic lethal pairs shared <br /> affinity capture methods causes small, non-significant               common interacting partner. However, effect the <br /> decrease proportion physical interactions involving               exclusion paralogs expected, exclusion pairs an <br /> essential genes members synthetic lethal pairs (Table S9).            interacting partner common greater effect (Figure 7 and <br /> These results corroborate essential interactions hypothesis         Table S15). <br /> demonstrate essential proteins highly inter-connected,            These results demonstrate ability form correct, <br /> so essential proteins define interaction sub-network (Figure 6).         cognate interaction leads enrichment physical interactions <br /> Although apparently members synthetic lethal pairs              &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;, evolutionary re- <br /> likely self-interact (Table S10), probably by-product       lationship. Moreover, functional convergence as <br /> the different combination rules hetero- homo-complexes.             preserve essential protein complexes. We speculate of <br /> Indeed, essential interaction sub-network mainly formed           singletons members synthetic lethal pairs have <br /> hetero-interactions (Table S10).                                            non-essential buffering mechanism. <br />    Finally, determined differences between <br /> networks exclusively composed obligate transient protein              Discussion <br /> interactions. These types protein interactions quite <br /> disparate [48]; so, potentially, introduce differences            It suggested relatively simple organisms, such <br /> the network. Although clear differences       yeast, phenomenon gene essentiality easily related <br /> proportion interactions involved essential subnetwork             duplication [21]: singleton genes (i.e., clear <br /> (Table S11), clear physical interactions          paralog) likely essential, duplicated genes may <br /> products essential genes relevant obligate                   compensate [28]. Our results in <br /> interactions (Table S12), transient interactions contain            yeast straightforward relationship essentiality <br /> a greater percentage interactions proteins coded          gene duplication. <br /> genes involved synthetic lethality (Table S13). These differences            We fifth essential yeast genes paralogs; <br />                                                                             however, synthetic lethal pairs considered, 4.3% <br />                                                                             cases members pair paralogous. This figure is <br />                                                                             lower estimates weak phenotypic effect to <br />                                                                             duplicate redundancy [17,18], similar older <br />                                                                             study demonstrated just 2% negative genetic <br />                                                                             interactions involved duplicate genes [32], notwithstanding <br />                                                                             methodological differences. Nevertheless, clear that, in <br />                                                                             yeast, synthetic lethality explained duplication alone. <br />                                                                                 Perhaps unsurprisingly, ability synthetic lethal gene pairs <br />                                                                             compensate functionally important. This the <br />                                                                             case divergently convergently evolved. We <br />                                                                             functional compensation highly likely to <br />                                                                             membership protein complex, frequently to <br />                                                                             members synthetic lethal pair sharing common <br />                                                                             interacting partner. Thus, common type functional <br />                                                                             compensation arises proteins able compensate for <br />                                                                             other&#226;&#8364;&#8482;s protein-protein interactions, other <br />                                                                             mechanism (Figure 1). Conversely, genes property <br />                                                                             single-essentiality protein products make interactions <br />                                                                             compensated. The key observations are, therefore <br />                                                                             (i) clustering essential genes synthetic lethal pairs in <br />                                                                             protein interaction network suggests synthetic lethality is <br />                                                                             strongly dependent formation protein-protein interac- <br />                                                                             tions (ii) compensation frequently arises to <br /> Figure 5. Essential interactions. A. SS network. B. ST network. C. TS       functional similarity results preservation essential <br /> network. D. TT network. &#226;&#8364;&#732;&#226;&#8364;&#732;Randomized genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates mean         protein complexes. The central role interactions synthetic <br /> standard deviation expected values random assignments of <br /> essential genes. &#226;&#8364;&#732;&#226;&#8364;&#732;Randomize pairs&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates mean standard <br />                                                                             lethality allows alternative phrasing: the <br /> deviation expected values random assignments essential           interaction essential, gene. <br /> pairs.                                                                          If proposed interaction-centric view correct, help <br /> doi:10.1371/journal.pone.0062866.g005                                       understanding human disease [49]. Here applied the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              5                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  <br />  <br />  <br /> Figure 6. Biggest component physical interaction network. Proteins coded essential gens coloured dark blue; proteins that <br /> are members synthetic lethal pairs coloured light blue; and, non-essential proteins coloured red. <br /> doi:10.1371/journal.pone.0062866.g006 <br />  <br /> understanding essentiality, extending work He               ally-related &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;. This essential sub-network <br /> Zhang [5]. An interaction focus allows unify analysis           apparent essential genes genes paired- <br /> essential genes, synthetic lethality functional compensation             essentiality (synthetic lethal pairs) considered together. <br /> within single framework. We propose single framework             Although cautiously, results suggests macromolecular <br /> helps overcome contradictions described                      machinery make physical interactions involving <br /> researchers [4].                                                             protein products single-essential genes, the <br />    Previous research shown presence products               members synthetic lethal pairs relatively more <br /> essential genes genes negative genetic interactions          important role transient interactions. Previous research found <br /> same complex [39,40]. Here, demonstrate enrichment                    essential interactions evolutionary-conserved than <br /> physical interactions subset products: i.e.       non-essential interactions [5,50]. This true significant <br /> there interaction sub-network physically links function-          genetic interaction network [51]. It possible this <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               6                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                      Protein Interactions Essentiality <br />  <br />  <br />                                                                                  essential functional <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> [12], specificity protein- <br />                                                                                  protein interaction key compensation. <br />                                                                                     Due expected conservation interactions involving <br />                                                                                  essential proteins [5,50,51], likely definition of <br />                                                                                  comprehensive essential interaction sub-network aid the <br />                                                                                  prediction protein-protein interactions related organ- <br />                                                                                  isms. Experimentally-generated genetic interactions give <br />                                                                                  mechanistic explanation phenotype associated with <br />                                                                                  range naturally-occurring mutations [37]. Moreover, protein <br />                                                                                  complexes form functional units, play central <br />                                                                                  role linking functional units [12]. There be <br />                                                                                  great benefit biotechnology medicine studying the <br />                                                                                  consequences protein mutations associated disruption <br />                                                                                  essential interactions, protein itself. Further, we <br />                                                                                  hypothesize interaction-centric view essentiality likely <br />                                                                                  facilitate understanding essentiality mammals. <br />  <br />                                                                                  Methods <br />                                                                                  Data <br />                                                                                     The complete list physical synthetic lethal interactions <br />                                                                                  data downloaded BIOGRID repository (http:// <br />                                                                                  thebiogrid.org/; version 2.0.63 downloaded 25th March 2010) <br />                                                                                  [53]. Essential genes recovered different sources: 1083 <br />                                                                                  genes Saccharomyces Genome Deletion Project [1,54]; <br />                                                                                  1007 genes CYGD database (http://mips.helmholtz- <br />                                                                                  muenchen.de/genre/proj/yeast/) [55]; 1136 genes SGD <br />                                                                                  database (http://www.yeastgenome.org/) [56] 1107 genes <br />                                                                                  recent study using synthetic genetic array technology [9]. <br />                                                                                  In order proper balance quality size <br />                                                                                  dataset, used different criteria select type of <br />                                                                                  data. Paralogs InterPro domains assignments down- <br />                                                                                  loaded Ensembl Genomes [57] BioMart portal [58]. <br />                                                                                  Functional clusters extracted DAVID bioinfor- <br />                                                                                  matics resources (http://david.abcc.ncifcrf.gov/) [59], using the <br /> Figure 7. Contributors essential interactions. A. Exclusion                highest stringency setting initial final group member- <br /> essential pairs genes paralogs. B. Exclusion essential pairs <br /> where proteins share interactors. &#226;&#8364;&#732;&#226;&#8364;&#732;Before exclusion&#226;&#8364;&#8482;&#226;&#8364;&#8482; original values <br />                                                                                  ship 2. <br /> before exclusion essential pairs; blue; &#226;&#8364;&#732;&#226;&#8364;&#732;After exclusion&#226;&#8364;&#8482;&#226;&#8364;&#8482; observed <br /> values exclusion selected pairs; &#226;&#8364;&#732;&#226;&#8364;&#732;Model 10 mean                Physical Interactions <br /> standard deviation expected values based model 1 (exclusion                   In case physical interactions, encountered two <br /> based number genes); &#226;&#8364;&#732;&#226;&#8364;&#732;Model 20 mean standard deviation of <br />                                                                                  problems: (1) methodology used detecting interaction, <br /> expected values based model 2 (exclusion based number of <br /> pairs).                                                                          (2) reproducibility results. Different methodologies <br /> doi:10.1371/journal.pone.0062866.g007                                            account different types detected interactions, with <br />                                                                                  strengths weaknesses. For instance, yeast-two-hybrid experi- <br /> set interactions represents ancestral network, core            ments identify proteins bind other; however, <br /> set functions common wide range organisms. We                biologically irrelevant interactions detected. Affinity <br />                                                                                  capture methods believed account false positives, <br /> therefore expect essential sub-network be <br />                                                                                  allow detection interactions need for <br /> found related organisms. Interestingly, research bacterial <br />                                                                                  protein over-expression. The relative merits discussed <br /> metabolic networks shows network core most <br />                                                                                  previously [60&#226;&#8364;&#8220;62]. Moreover, Yu coworkers that <br /> conserved, peripheral interactions originated niche <br />                                                                                  yeast-two-hybrid technique better finding direct <br /> specialization different organisms [52]. <br />                                                                                  interactions affinity purification methodologies, were <br />    Since compensation does mainly occur an <br />                                                                                  prone biased essential genes genes encoding <br /> evolutionary relationship, removing paralogous synthetic lethal <br />                                                                                  specific functions [46]. <br /> pairs network analysis greater effect the <br />                                                                                     As methods sources errors, (1) selected <br /> essential sub-network removing random essential pairs. By <br />                                                                                  detected using different experimental systems and <br /> contrast, higher depletion essential interactions observed <br />                                                                                  reported different scientific papers (2350 nodes, 4951 edges, <br /> when removing pairs interacting partner                stringent criteria) and, (2) selected different pieces of <br /> common. Thus functional similarity, commonly manifested                     evidence, i.e., reported twice detected using two <br /> as similarity protein-protein interactions, permits preservation              different methods (3335 nodes, 12153 edges, tolerant criteria)(see <br /> of essential protein complexes. Consequently, cases,                     Figure S1 Tables S16 S17 details). Most the <br /> lethality lack essential component protein            physical interactions selected using stringent criterion likely <br /> complex. These results highlight protein complex basic                true positives terms protein&#226;&#8364;&#8482;s ability bind in <br />                                                                                  biological relevance. However, rule some <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   7                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  <br /> of artifacts identified using different experimental               biological data available (ND evidence code) inferred <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> designs.                                                                    electronic annotation (IEA evidence code). <br />  <br /> Essential Genes Synthetic Lethal Pairs                                  Obligate Transient Interactions <br />    In order clearly differentiate essential genes genes              We divided set physical interactions different <br /> synthetic-lethal pairs, essential genes members         subsets: obligate transient interactions. Unambiguous differ- <br /> of synthetic-lethal pairs removed datasets.                   entiation interaction types challenging since <br /> Sometimes genetic interactions measured non-              continuum binding affinity [48]. Here, classified as <br /> essential gene essential gene containing hypomorphic               obligate interactions detected affinity capture <br /> mutation. Although interactions biologically relevant,            experiments involving proteins annotated participating in <br /> they harder interpret. Moreover, logically              macromolecular complexes (GO:0032991) experimental <br /> correct classify gene essential non-essential             evidence. Conversely, transient interactions included those <br /> simultaneously. A possible consequence using data              detected method affinity capture involving <br /> would results members synthetic-lethal pairs           proteins assigned participate protein <br /> be heavily influenced results essential genes. Thus,              complexes. It noteworthy 1) interactions could <br /> contrast previous research [39,40], removed genes               assigned subsets; and, 2) way built the <br /> from analyses. Another problem encountered                   networks does allow crosstalk proteins obligate <br /> genetic interactions comprehensively tested, high-          interactions proteins transient interactions. <br /> throughput studies. Experiments focus specific <br /> sets genes. In addition, high-throughput experiments limited         Annotated Networks Physical Interactions <br /> to small number research groups. Nevertheless believe             Combining selection criteria, built networks of <br /> the numbers synthetic-lethal interactions analyzed           physical interactions annotated essentiality information (see <br /> a fair representation current knowledge.                             Figure S3). The networks labeled SS (stringent physical <br />    Data set selection illustrated Figure S2, described         interactions; stringent essentiality), ST (stringent physical interac- <br /> Tables S18 S19. When using stringent criterion,                  tions; tolerant essentiality), TS (tolerant physical interactions; <br /> selected synthetic pairs reported twice independently (694              stringent essentiality) TT (tolerant physical interactions; <br /> nodes, 1621 edges) essential genes present          tolerant essentiality). Additionally, built control <br /> datasets (387 nodes). When using tolerant criterion, selected        networks account affinity capture methods could <br /> all synthetic-lethal pairs (2007 nodes, 8055 edges)             identify members complex having straight <br /> essential proteins reported datasets (437 nodes).         physical interaction. Starting strictest network (SS), 1) we <br /> Although synthetic-lethal interactions reported               removed analysis physical interactions only <br />                                                                             affinity capture methods, resulting in <br /> twice research group, possible existence an <br />                                                                             network containing 3811 edges; and, 2) removed the <br /> ambiguity synthetic-lethality synthetic-sickness, the <br />                                                                             network physical interaction detected at <br /> variety sources (see Tables S18 S19) make confident that <br />                                                                             different non-affinity capture methods, getting network <br /> most synthetic-lethal interactions selected using stringent <br />                                                                             having 1762 edges. <br /> criteria true positives. Nevertheless, built additional <br /> control lists: high-confident synthetic-lethal pairs, the <br /> other possible synthetic sick pairs. In order generate set       Random Models <br /> pairs small chance containing false positives, explored              All analyses compared null hypothesis the <br /> the list generated stringent criteria, selected 113       emergence essentiality synthetic lethality purely random <br />                                                                             events. The pool genes list yeast genes, except <br /> pairs reported different publications. The <br />                                                                             previously removed analysis <br /> fact 92 different sets publications reassures that <br />                                                                             (essential genes participating synthetic-lethality pairs through <br /> there small chance reporting bias. For generating list of <br />                                                                             hypomorphic mutations). Moreover, ensured random <br /> synthetic sick pairs, selected pairs genes involved in <br />                                                                             networks essentiality ambiguity excluding the <br /> genetic interactions termed &#226;&#8364;&#732;&#226;&#8364;&#732;Negative Genetic&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;Synthetic <br />                                                                             randomisation fixed-state genes; i.e. creating random <br /> Growth Defect&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Then, discarded pairs been <br />                                                                             lists essential genes, excluded members synthetic-lethal <br /> identified participating synthetic lethal interaction, those <br />                                                                             pairs; generating random synthetic-lethal pairs, did not <br /> pairs containing gene identified essential 4             use essential genes. For criterion, generated 10000 <br /> datasets previously mentioned. This resulted list 53902 pairs       random lists genes assigned essential genes 10000 <br /> of putative synthetic sick pairs.                                           lists pairs proteins assigned synthetic-lethal pairs. The <br />                                                                             lists contained number genes (and pairs) original <br /> Protein Complexes                                                           list. These lists subsequently used re-annotate original <br />    We used different definitions protein complex. In one,         network physical interactions (see Figure S3). P-values were <br /> defined protein member protein complex           calculated number random models having result equal <br /> detected participating protein interactions means affinity         higher obtained using real data. <br /> capture methods. In other, used similar strategy                    Being network itself, randomisation lists synthetic <br /> Michaut coworkers [12]: considered genes annotated               lethal genes poses additional problem: sure the <br /> with GO term macromolecular complex (GO:0032991)                observed differences identity nodes (the genes <br /> children terms, excluding annotations qualifiers               involved synthetic lethality) changes topology the <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;NOT&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;colocalizes_with&#226;&#8364;&#8482;&#226;&#8364;&#8482;. We generated different sets           network (see Table S20). As control added two <br /> of proteins: 1) experimental evidence         additional random models synthetic lethality same <br /> the annotation; and, 2) annotated means            degree distribution real network. Thus, additional <br /> there non-traceable author statement (NAS evidence code),             models topology original synthetic-lethality <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              8                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                Protein Interactions Essentiality <br />  <br />  <br /> network. In model, substituted node synthetic             Table S4 Number synthetic lethal pairs their <br /> lethality network gene pool genes. In           functional relatedness main contributing <br /> other model just used genes present synthetic          sources. On top, pairs selected using tolerant criterion <br /> lethality network reshuffled nodes. In model          considered. On bottom, pairs reported single <br /> only difference networks position nodes             study taken account. <br /> them, model network contain                   (DOCX) <br /> different nodes despite keeping original topology (see Table <br />                                                                              Table S5 Percentage synthetic-lethal pairs sharing at <br /> S20). In cases, built 10000 lists, used re- <br />                                                                              interactor. <br /> annotate original network. P-values calculated above- <br />                                                                              (DOCX) <br /> mentioned. <br />                                                                              Table S6 Percentage physical interactions occurring <br /> Random Exclusion                                                             essential proteins. <br />    We randomly excluded synthetic lethal pairs                 (DOCX) <br /> analyses. We used different protocols: 1) excluding             Table S7 Percentage physical interactions occurring <br /> number genes (regardless number pairs) real           members synthetic-lethal pairs (it includes <br /> data (Model 1); 2) excluding number pairs                    pairs). <br /> (regardless number genes) real data (Model 2).           (DOCX) <br /> We ran 10000 simulations analysis. P-values were <br /> calculated above.                                                         Table S8 Percentage physical interactions occurring <br />                                                                              member synthetic-lethal pair an <br />                                                                              essential protein. <br /> Supporting Information <br />                                                                              (DOCX) <br /> Figure S1 Flowchart selection data physical <br />                                                                              Table S9 Analysis effect affinity capture <br /> interactions. We discarded data just evidence the <br />                                                                              methods detection physical interactions in- <br /> BioGRID database. Physical interactions detected using two <br />                                                                              volving essential genes members synthetic lethal <br /> different methods reported twice independently were <br />                                                                              pairs SS network. P-values calculated comparing the <br /> selected using stringent criterion. The rest physical <br />                                                                              proportions obtained control original <br /> interactions selected tolerant criterion. All the <br />                                                                              network assuming binomial distribution. <br /> interactions selected using stringent criterion also <br />                                                                              (DOCX) <br /> included dataset obtained using tolerant criterion. <br /> (PPTX)                                                                       Table S10 Analysis effect self-interaction upon <br />                                                                              essential subnetwork. P-values calculated comparing <br /> Figure S2     Flowchart selection data essentiality <br />                                                                              proportions obtained control original <br /> relationships (gene essentiality synthetic lethality). <br />                                                                              network assuming binomial distribution. <br /> All synthetic lethality interactions present BioGRID <br />                                                                              (DOCX) <br /> database selected using tolerant criterion. However, only <br /> those multiple evidences selected using stringent              Table S11 Analysis essentiality transient and <br /> criterion. In case gene essentiality, selected genes       obligate interaction networks (essential subnetwork). <br /> present 3 datasets using tolerant criterion,          P-values calculated comparing proportions assuming <br /> selected present datasets using stringent           binomial distribution. <br /> criterion. We did use analyses data leading                    (DOCX) <br /> ambiguity: essential genes involved synthetic lethality interac- <br />                                                                              Table S12 Analysis essentiality transient and <br /> tions (probably result hypomorphic mutation), their <br />                                                                              obligate interaction networks (interactions between <br /> corresponding synthetic lethal pairs. Obviously, dataset <br />                                                                              products single-essentiality genes). P-values calcu- <br /> selected using stringent criterion subset data <br />                                                                              lated comparing proportions assuming binomial <br /> selected using tolerant criterion. <br />                                                                              distribution. <br /> (PPTX) <br />                                                                              (DOCX) <br /> Figure S3  Annotation physical interaction network <br />                                                                              Table S13 Analysis essentiality transient and <br /> with information &#226;&#8364;&#732;&#226;&#8364;&#732;essentiality&#226;&#8364;&#8482;&#226;&#8364;&#8482;. <br />                                                                              obligate interaction networks (interactions between <br /> (PPTX) <br />                                                                              members synthetic lethal pairs). P-values calculated <br /> Table S1 Analysis paralogy essential genes                         comparing proportions assuming binomial distribution. <br /> members synthetic-lethal pairs.                                           (DOCX) <br /> (DOCX) <br />                                                                              Table S14 Composition transient obligate phys- <br /> Table S2   Analysis function similarity synthetic-                     ical interaction networks. The figure corresponds the <br /> lethal pairs.                                                                proportion proteins transient network. The second <br /> (DOCX)                                                                       figure corresponds proportion proteins obligate <br />                                                                              network. P-values calculated comparing proportions and <br /> Table S3 Control different features synthetic <br />                                                                              assuming binomial distribution. <br /> lethal synthetic sick pairs. P-values calculated <br />                                                                              (DOCX) <br /> comparing proportions obtained control of <br /> the synthetic lethal pairs selected using stringent criteria         Table S15 Analysis importance evolutionary <br /> assuming binomial distribution.                                            functional factors essential interactome. <br /> (DOCX)                                                                       (DOCX) <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               9                             April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                                            Protein Interactions Essentiality <br />  <br />  <br /> Table S16 Description filtering selecting                                           (DOCX) <br /> datasets physical interactions. <br />                                                                                                Table S20 Features randomized networks of <br /> (DOCX) <br />                                                                                                synthetic lethal interactions. <br /> Table S17 Sets methodologies commonly used                                             (DOCX) <br /> to detect physical interactions selected using strin- <br /> gent criterion.                                                                                Acknowledgments <br /> (DOCX) <br />                                                                                                The authors wish acknowledge Brad T. Sherman assistance the <br /> Table S18    Description filtering origin                                        DAVID analysis, Kathyrn Hentges critical reading the <br /> synthetic-lethal interactions. We considered high-throughput                                   manuscript. <br /> experiments reporting 50 interactions, small-scale <br /> experiments reporting interactions, medium-                             Author Contributions <br /> scale experiments reporting 6 50 interactions. <br /> (DOCX)                                                                                         Conceived designed experiments: DT DLR SCL. Performed the <br />                                                                                                experiments: DT. Analyzed data: DT SCL. Wrote paper: DT DLR <br /> Table S19 Papers reporting multiple synthetic-lethal                                           SCL. <br /> interactions selected using stringent criterion. <br />  <br /> References <br />  1. Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al. (2002) Functional                     26. Makino T, Hokamp K, McLysaght A (2009) The complex relationship gene <br />     profiling Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391.                         duplication essentiality. Trends Genet 25: 152&#226;&#8364;&#8220;155. <br />  2. Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality centrality               27. Liang H, Li WH (2009) Functional compensation duplicated genes mouse. <br />     protein networks. Nature 411: 41&#226;&#8364;&#8220;42.                                                           Trends Genet 25: 441&#226;&#8364;&#8220;442. <br />  3. Coulomb S, Bauer M, Bernard D, Marsolier-Kergoat MC (2005) Gene                            28. VanderSluis B, Bellay J, Musso G, Costanzo M, Papp B, et al. (2010) Genetic <br />     essentiality topology protein interaction networks. Proc Biol Sci 272:              interactions reveal evolutionary trajectories duplicate genes. Mol Syst Biol <br />     1721&#226;&#8364;&#8220;1725.                                                                                     6: 429. <br />  4. Zotenko E, Mestre J, O&#226;&#8364;&#8482;Leary DP, Przytycka TM (2008) Why hubs                    29. Conant GC, Wagner A (2004) Duplicate genes robustness transient gene <br />     yeast protein interaction network tend essential: reexamining                        knock-downs Caenorhabditis elegans. Proc Biol Sci 271: 89&#226;&#8364;&#8220;96. <br />     connection network topology essentiality. PLoS Comput Biol                 30. Papp B, Pal C, Hurst LD (2003) Dosage sensitivity evolution gene <br />     4: e1000140.                                                                                   families yeast. Nature 424: 194&#226;&#8364;&#8220;197. <br />  5. He X, Zhang J (2006) Why hubs tend essential protein networks?                 31. Stuart GR, Copeland WC, Strand MK (2009) Construction application of <br />     PLoS Genet 2: e88.                                                                             protein genetic interaction network (yeast interactome). Nucleic Acids Res <br />  6. Papp B, Pal C, Hurst LD (2004) Metabolic network analysis causes                    37: e54. <br />     evolution enzyme dispensability yeast. Nature 429: 661&#226;&#8364;&#8220;664.                          32. Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al. (2004) Global mapping of <br />  7. D&#226;&#8364;&#8482;Elia MA, Pereira MP, Brown ED (2009) Are essential genes really essential?                   yeast genetic interaction network. Science 303: 808&#226;&#8364;&#8220;813. <br />     Trends Microbiol 17: 433&#226;&#8364;&#8220;438.                                                              33. Hakes L, Pinney JW, Lovell SC, Oliver SG, Robertson DL (2007) All duplicates <br />  8. Dowell RD, Ryan O, Jansen A, Cheung D, Agarwala S, et al. (2010) Genotype                      equal: difference small-scale genome duplication. <br />     phenotype: complex problem. Science 328: 469.                                             Genome Biol 8: R209. <br />  9. Costanzo M, Baryshnikova A, Bellay J, Kim Y, Spear ED, et al. (2010) The                   34. Hollunder J, Beyer A, Wilhelm T (2005) Identification characterization of <br />     genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431.                                             protein subcomplexes yeast. Proteomics 5: 2082&#226;&#8364;&#8220;2089. <br /> 10. Kaelin WG Jr (2005) The concept synthetic lethality context                   35. Valente AX, Roberts SB, Buck GA, Gao Y (2009) Functional organization the <br />     anticancer therapy. Nat Rev Cancer 5: 689&#226;&#8364;&#8220;698.                                                 yeast proteome yeast interactome map. Proc Natl Acad Sci U S A 106: <br /> 11. Kelley R, Ideker T (2005) Systematic interpretation genetic interactions using              1490&#226;&#8364;&#8220;1495. <br />     protein networks. Nat Biotechnol 23: 561&#226;&#8364;&#8220;566.                                              36. Le Meur N, Gentleman R (2008) Modeling synthetic lethality. Genome Biol 9: <br /> 12. Michaut M, Baryshnikova A, Costanzo M, Myers CL, Andrews BJ, et al. (2011)                     R135. <br />     Protein complexes central yeast genetic landscape. PLoS Comput Biol             37. Hannum G, Srivas R, Guenole A, van Attikum H, Krogan NJ, et al. (2009) <br />     7: e1001092.                                                                                   Genome-wide association data reveal global map genetic interactions <br /> 13. Feiglin A, Moult J, Lee B, Ofran Y, Unger R (2012) Neighbor overlap                         protein complexes. PLoS Genet 5: e1000782. <br />     enriched yeast interaction network: analysis implications. PLoS One             38. Costanzo M, Baryshnikova A, Bellay J, Kim Y, Spear ED, et al. (2010) The <br />     7: e39662.                                                                                     genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431. <br /> 14. Szappanos B, Kovacs K, Szamecz B, Honti F, Costanzo M, et al. (2011) An                    39. Bandyopadhyay S, Kelley R, Krogan NJ, Ideker T (2008) Functional maps of <br />     integrated approach characterize genetic interaction networks yeast                      protein complexes quantitative genetic interaction data. PLoS Comput Biol <br />     metabolism. Nat Genet 43: 656&#226;&#8364;&#8220;662.                                                             4: e1000065. <br /> 15. Tischler J, Lehner B, Fraser AG (2008) Evolutionary plasticity genetic                  40. Baryshnikova A, Costanzo M, Kim Y, Ding H, Koh J, et al. (2010) Quantitative <br />     interaction networks. Nat Genet 40: 390&#226;&#8364;&#8220;391.                                                   analysis fitness genetic interactions yeast genome scale. Nat <br /> 16. Li J, Yuan Z, Zhang Z (2010) The cellular robustness genetic redundancy                  Methods 7: 1017&#226;&#8364;&#8220;1024. <br />     budding yeast. PLoS Genet 6: e1001187.                                                     41. Guan Y, Dunham MJ, Troyanskaya OG (2007) Functional analysis gene <br /> 17. Wagner A (2005) Distributed robustness versus redundancy causes                          duplications Saccharomyces cerevisiae. Genetics 175: 933&#226;&#8364;&#8220;943. <br />     mutational robustness. Bioessays.                                                          42. Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, et al. (2001) <br /> 18. Ihmels J, Collins SR, Schuldiner M, Krogan NJ, Weissman JS (2007) Backup                       Systematic genetic analysis ordered arrays yeast deletion mutants. <br />     redundancy: genetic interactions reveal cost duplicate gene loss.               Science 294: 2364&#226;&#8364;&#8220;2368. <br />     Molecular Systems Biology 3: 86.                                                           43. Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati A, et al. (2005) <br /> 19. DeLuna A, Springer M, Kirschner MW, Kishony R (2010) Need-based up-                            Exploration function organization yeast early secretory <br />     regulation protein levels response deletion duplicate genes. PLoS            pathway epistatic miniarray profile. Cell 123: 507&#226;&#8364;&#8220;519. <br />     Biol 8: e1000347.                                                                          44. Collins SR, Roguev A, Krogan NJ (2010) Quantitative genetic interaction <br /> 20. Dean EJ, Davis JC, Davis RW, Petrov DA (2008) Pervasive persistent                         mapping using E-MAP approach. Methods Enzymol 470: 205&#226;&#8364;&#8220;231. <br />     redundancy duplicated genes yeast. PLoS Genet 4: e1000113.                        45. Lehner B (2011) Molecular mechanisms epistasis genes. <br /> 21. Doyle MA, Gasser RB, Woodcroft BJ, Hall RS, Ralph SA (2010) Drug target                        Trends Genet 27: 323&#226;&#8364;&#8220;331. <br />     prediction prioritization: using orthology predict essentiality parasite         46. Yu H, Braun P, Yildirim MA, Lemmens I, Venkatesan K, et al. (2008) High- <br />     genomes. BMC Genomics 11: 222.                                                                 quality binary protein interaction map yeast interactome network. Science <br /> 22. Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, et al. (2003) Role                           322: 104&#226;&#8364;&#8220;110. <br />     duplicate genes genetic robustness null mutations. Nature 421: 63&#226;&#8364;&#8220;66.           47. Hart GT, Lee I, Marcotte ER (2007) A high-accuracy consensus map yeast <br /> 23. Liao BY, Zhang J (2007) Mouse duplicate genes essential singletons.                  protein complexes reveals modular nature gene essentiality. BMC Bioinfor- <br />     Trends Genet 23: 378&#226;&#8364;&#8220;381.                                                                      matics 8: 236. <br /> 24. Liang H, Li WH (2007) Gene essentiality, gene duplicability protein                    48. Ozbabacan SE, Engin HB, Gursoy A, Keskin O (2011) Transient protein- <br />     connectivity human mouse. Trends Genet 23: 375&#226;&#8364;&#8220;378.                                     protein interactions. Protein Eng Des Sel 24: 635&#226;&#8364;&#8220;648. <br /> 25. Su Z, Gu X (2008) Predicting proportion essential genes mouse                    49. Zhong Q, Simonis N, Li QR, Charloteaux B, Heuze F, et al. (2009) Edgetic <br />     duplicates based biased mouse knockout genes. J Mol Evol 67: 705&#226;&#8364;&#8220;709.                       perturbation models human inherited disorders. Mol Syst Biol 5: 321. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                                10                                    April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                                        Protein Interactions Essentiality <br />  <br />  <br /> 50. Pereira-Leal JB, Audit B, Peregrin-Alvarez JM, Ouzounis CA (2005) An                    56. Cherry JM, Ball C, Weng S, Juvik G, Schmidt R, et al. (1997) Genetic and <br />     exponential core heart yeast protein interaction network. Mol Biol            physical maps Saccharomyces cerevisiae. Nature 387: 67&#226;&#8364;&#8220;73. <br />     Evol 22: 421&#226;&#8364;&#8220;425.                                                                       57. Kersey PJ, Lawson D, Birney E, Derwent PS, Haimel M, et al. (2010) Ensembl <br /> 51. Dixon SJ, Fedyshyn Y, Koh JL, Prasad TS, Chahwan C, et al. (2008) Significant               Genomes: extending Ensembl taxonomic space. Nucleic Acids Res 38: <br />     conservation synthetic lethal genetic interaction networks distantly             D563&#226;&#8364;&#8220;569. <br />     related eukaryotes. Proc Natl Acad Sci U S A 105: 16653&#226;&#8364;&#8220;16658.                          58. Smedley D, Haider S, Ballester B, Holland R, London D, et al. (2009) BioMart&#226;&#8364;&#8220; <br /> 52. Kreimer A, Borenstein E, Gophna U, Ruppin E (2008) The evolution                         biological <br /> </body></html>