<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Statistical Framework Improving Genomic <br /> Annotations Prokaryotic Essential Genes <br /> Jingyuan Deng1,2, Shengchang Su3, Xiaodong Lin5, Daniel J. Hassett3, Long Jason Lu1,2,4* <br /> 1 Division Biomedical Informatics, Cincinnati Children&#226;&#8364;&#8482;s Hospital Medical Center, Cincinnati, Ohio, United States America, 2 Department Environmental Health, <br /> University Cincinnati, Cincinnati, Ohio, United States America, 3 Department Biochemistry Molecular Genetics, University Cincinnati, Cincinnati, Ohio, United <br /> States America, 4 Department Computer Science, University Cincinnati, Cincinnati, Ohio, United States America, 5 Department Management Science and <br /> Information Systems, Rutgers University, Piscataway, New Jersey, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Large-scale systematic analysis gene essentiality important step closer unraveling complex relationship <br />      genotypes phenotypes. Such analysis accomplished unbiased accurate annotations of <br />      essential genes. In current genomic databases, essential gene annotations derived whole-genome <br />      transposon mutagenesis (TM), frequently used experimental approach determining essential genes in <br />      microorganisms defined conditions. However, substantial systematic biases associated TM <br />      experiments. In study, developed novel Poisson model&#226;&#8364;&#8220;based statistical framework simulate TM insertion <br />      process subsequently correct experimental biases. We quantitatively assessed effects major factors that <br />      potentially influence accuracy TM subsequently incorporated relevant factors framework. Through <br />      iteratively optimizing parameters, inferred actual insertion events occurred described gene&#226;&#8364;&#8482;s essentiality on <br />      probability measure. Evaluated definite mapping essential gene profile Escherichia coli, model significantly <br />      improved accuracy original TM datasets, resulting accurate annotations essential genes. Our method also <br />      showed encouraging results improving subsaturation level TM datasets. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> test model&#226;&#8364;&#8482;s broad applicability other <br />      bacteria, applied Pseudomonas aeruginosa PAO1 Francisella tularensis novicida TM datasets. We validated our <br />      predictions literature allelic exchange experiments PAO1. Our model correct seven tested <br />      genes. Remarkably, cases predictions contradicted TM assignments, experimental validations <br />      supported predictions. In summary, method promising tool improving genomic annotations essential <br />      genes enabling large-scale explorations gene essentiality. Our contribution timely considering rapidly <br />      increasing essential gene sets. A Webserver set provide convenient access tool. All results source <br />      codes available download publication http://research.cchmc.org/essentialgene/. <br />  <br />   Citation: Deng J, Su S, Lin X, Hassett DJ, Lu LJ (2013) A Statistical Framework Improving Genomic Annotations Prokaryotic Essential Genes. PLoS ONE 8(3): <br />   e58178. doi:10.1371/journal.pone.0058178 <br />   Editor: Philip M. Kim, University Toronto, Canada <br />   Received October 23, 2012; Accepted January 31, 2013; Published March 8, 2013 <br />   Copyright: &#195;&#376; 2013 Deng et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported CCHMC Trustee Award LJL; Cystic Fibrosis Foundation grants HASSETT07G0 R457-CR01 DJH. The funders <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: long.lu@cchmc.org <br />  <br />  <br />  <br /> Introduction                                                                             understudied species. Constraint-based methods limited <br />                                                                                          metabolic signaling pathways require prior knowledge <br />     Large-scale systematic analysis gene essentiality                           pathways. Homology mapping works best model and <br /> important step closer unraveling complex relationship                         target organisms closely related. Otherwise, prediction <br /> between genotypes phenotypes [1]. However, analysis                             coverage low ignores unique physiology the <br /> cannot accomplished unbiased accurate annota-                             subject species people orthologs not <br /> tions essential genes.                                                                necessarily degree essentiality [16]. <br />     Whole-genome <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>knockout</span> experiments produce accu-                                Therefore, interrogate essential genes understudied <br /> rate essential gene annotations, consortium                     species, whole-genome transposon mutagenesis (TM) followed by <br /> labs years complete organism. It                              sequence-based identification insertion sites most <br /> surprising single-<span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>gene knockout</span> results currently                          practical frequently used experimental approach <br /> available handful well-studied model organisms,                          [17]. Using approach, essential genes variety of <br /> Escherichia coli [2] Saccharomyces cerevisiae [3]. Computational                     understudied bacterial species, Pseudomonas aeruginosa and <br /> methods predicting essential genes, e.g., homology mapping                           Francisella tularensis novicida, identified, greatly increasing <br /> [4,5,6], constraint-based methods [7,8,9,10] supervised                              insights essential processes necessary growth of <br /> learning [11,12,13,14,15], useful reducing cost                         bacteria defined conditions. As May 2012, have <br /> and labor. However, limited applicability understud-                        24 genomic scale TM studies 19 bacterial species <br /> ied species. For example, supervised learning depends partial                       literature (Table S1), generated <br /> list known essential genes unavailable                             past years. For past years, seen at <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                           1                                  March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                                 Improving Essential Gene Annotations <br />  <br />  <br /> least new datasets different organisms. As                    subsequently incorporated relevant factors framework. <br /> researchers starting analyze conditional essential genes,             Through iteratively optimizing parameters, finalized model <br /> that is, essential genes virtually unlimited growth conditions,        inferred actual insertion events occurred gene <br /> we expect number available TM datasets rapidly                   given observed insertion information. Finally described <br /> increase future years.                                                gene&#226;&#8364;&#8482;s essentiality probability measure, provided <br />    As result rapid increase, current genomic databases,         corrections possible biases TM assigned annota- <br /> e.g., [18,19,20], annotations microbial essential             tions. <br /> genes derived directly indirectly TM experiments.                   We took advantage definitive mapping essential genes <br /> However, unlike single-gene knockouts, TM intrinsic biases               E. coli MG1655 strains determined single gene knockout <br /> (see below). Without correcting biases, TM assignments             experiments (PEC set) [2] identify errors essential <br /> will contain hundreds mis-annotated essential non-essential           gene annotations produced TM experiments (Gerdes set) [23] <br /> genes organism, caused substantial confusion               comparing assignments. Although single-gene <br /> among genetics community.                                                knockout experiments completely error free, have <br />    Transposons segments DNA (transpose)                 considered error-prone [2]. We realized the <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> location genome [21,22]. The locations              essential genes uniquely identified TM biological <br /> in transposon depends sequence              significance represent genes essential fitness as <br /> transposase recognizes cleaves, recognition                 suggested Gerdes et al. [32]; however, focus on <br /> sequence transposons unclear                    essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span>, referred &#226;&#8364;&#732;&#226;&#8364;&#732;errors&#226;&#8364;&#8482;&#226;&#8364;&#8482;. <br /> determined. TM results disruption region genome             Also note, model dependent single-gene knockout <br /> where transposon inserted. If insertion predicted         results target organism (see Discussion section). The <br /> ORF allows resulting strain form colony appropriate              PEC dataset used assessing errors TM dataset <br /> solidified media, unlikely ORF essential viability         evaluating model&#226;&#8364;&#8482;s performance. <br /> under conditions (Fig. S1). Therefore, TM identifies <br /> essential genes using &#226;&#8364;&#732;&#226;&#8364;&#732;negative&#226;&#8364;&#8482;&#226;&#8364;&#8482; approach, i.e., identifying        Results <br /> regions essential presuming is <br /> essential.                                                                   1. Assessing Overall Error Rates TM Annotations of <br />    Due random nature transposon insertion events,            Essential Genes <br /> are number factors create systematic biases TM                 Using PEC set reference, assessed overall error <br /> experiments. For example, inevitable genes,                  rates Gerdes dataset. We examined subset genes <br /> especially shorter ones, missed simply chance                     appearing datasets. This subset contains 3833 genes in <br /> [23,24,25]. This create false positive errors non-             total, including 615 TM-assigned essential genes (TmEs) 3218 <br /> essential genes determined essential TM. On              TM-assigned non-essential genes (TmNs), covers 90% (3833 <br /> hand, insertion place gene,            4291) genes TM dataset. <br /> the extreme ends, fully disrupt function               According intersection PEC set, Gerdes <br /> gene product [23,24,25,26,27,28,29,30]. This create false               dataset divided mutually exclusive fractions <br /> negative errors essential genes determined non-              (Table 1): true essential TM (TETmE, essential assigned by <br /> essential TM. These biases TM experiments                       TM PEC), false essential TM (FETmE; essential assigned <br /> introduced substantial errors essential gene annotations               TM non-essential PEC), false non-essential TM <br /> current genomic databases [19]. In order render large-scale           (FNTmN; non-essential TM essential PEC) true <br /> integrative comparative analysis essential genes possible,            non-essential TM (TNTmN; non-essential TM and <br /> these biases quantitatively assessed corrected.                  PEC). In report, defined TM essential error rate <br />    Several models previously developed tackle biases          (TmER) proportion FETmEs total TmEs; <br /> TM. For example, Lamichhane et al. used Bayesian framework                 likewise, TM non-essential error rate (TmNR) defined as <br /> to estimate essential genes Mycobacterium tuberculosis             proportion FNTmNs total TmNs. Among the <br /> subsaturation level TM [28,31]. Jacobs et al. used neutral <span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span>         615 TmEs, 186 TETmEs, yielding TmER 70%. <br /> pair model reach estimation 307 essential genes P.               Similarly, TmNR 2.3%. Typically, TM experiments the <br /> aeruginosa PAO1 saturation level TM [29]. However,             TmNR low, TmER relatively high. <br /> key limitation existing models into <br /> account insertions viable mutants ignore               2. Assessing Effects Four Factors Accuracy <br /> disrupt essential genes mutants die and, such,          TM Experiments <br /> not observable. Therefore, number real insertions                    Previous studies suggested accuracy TM experiments may <br /> underestimated equal observed insertions. Also,               affected main factors: gene <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, insertions the <br /> the existing models assume gene&#226;&#8364;&#8482;s probability inserted <br /> only depends length insertion sufficient disrupt         Table 1. Using PEC dataset gold-standards identify <br /> a gene&#226;&#8364;&#8482;s function. However, TM results &#226;&#8364;&#732;&#226;&#8364;&#732;hot&#226;&#8364;&#8482;&#226;&#8364;&#8482;             false essential non-essential genes TM dataset. <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;cold&#226;&#8364;&#8482;&#226;&#8364;&#8482; spots genome transposon insertions. Further- <br /> more, require multiple insertions disrupt gene&#226;&#8364;&#8482;s <br /> functions, depending site insertion.                            *E &#226;&#8364;&#8220; Essential                          PEC dataset (Gold Standard) <br />    Given caveats, developed novel Poisson model based <br /> statistical framework simulate TM insertion process                *N &#226;&#8364;&#8220; Non-essential                      E (259)         N (3574) <br /> subsequently correct experimental biases. Briefly, statistical        TM dataset        TmEs (615)            186 (TETmEs)    429 (FETmEs) <br /> framework works follows: We quantitatively assessed              (Gerdes set)      TmNs (3218)           73 (FNTmNs)     3145 (TNTmNs) <br /> effects potential factors affect accuracy TM <br /> results, gene length relative insertion positions,            doi:10.1371/journal.pone.0058178.t001 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                              March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                             Improving Essential Gene Annotations <br />  <br />  <br />  <br />  <br /> Figure 1. Three factors strong associations false TM assignments. (A) Gene length. The lengths TmEs significantly shorter <br /> than PEC dataset total genes. Many short genes false essential genes. (B) Position insertions. Essential genes <br /> mistakenly assigned non-essential TM insertions 25% extreme-ends (5% 59 end 20% 39 end). These insertions do <br /> not completely disrupt gene&#226;&#8364;&#8482;s function. (C) Number insertions. 75% essential genes mistakenly assigned non-essential TM only <br /> have insertion them. <br /> doi:10.1371/journal.pone.0058178.g001 <br />  <br /> distal regions, number insertions gene polar effects             insertion number gene FNTmNs 1.56, significantly <br /> [23,29]. To quantitatively assess factor&#226;&#8364;&#8482;s influence            smaller TNTmNs (4.13) (P-value #1E-26). <br /> accuracy TM results, examined association                   (4) Polar effects: potentially causing false positive <br /> factor FETmEs FNTmNs, respectively (Fig. 1).                   errors. In TM experiments, polar effects occur a <br />   (1) Gene length: potentially causing false positive                       transposon inserts dispensable gene prevents the <br /> errors. In TM experiments, genes                   transcription downstream essential genes operon <br /> detected transposon hits assigned essential (Fig.           [23,24,25]. Because insertion dispensable gene actually <br /> S1). The average detectable insertion density 1 400 bp         disrupts entire downstream essential genes causes death of <br /> in TM experiments saturation levels [23,24,25]. This                  mutants, dispensable gene incorrectly labeled as <br /> suggests relatively short genes (e.g. #300 bp) easily              essential, causing false positive error. <br /> missed simply chance, incorrectly labeled               To detect number FETmEs caused polar effects, we <br /> essential.                                                                  examined TmEs 2,665 operons were <br />    To quantitatively assess influence, compared length        inferred experimentally computationally (see Methods). If <br /> essential genes Gerdes PEC sets, using total genes        exists TETmE resides downstream FETmE, this <br /> a control (Fig. 1A). The student t-test shows average              FETmE considered potentially caused polar effect. <br /> length essential genes Gerdes set (730 bp) significantly       Among 429 FETmEs, 46 caused polar <br /> shorter PEC set (1,003 bp) (P-value ,1E-11). It         effects. The small number likely fact TM <br /> also significantly shorter total genes (982 bp) (P-value ,1E-          experiments designed prevent polar effects, typically <br /> 25) difference PEC set total genes             designing transposons strong regulatable promoter <br /> considered significant (P-value .0.05).                                 downstream transposase transposon. <br />   (2) Insertions 59- 39-ends genes: potentially <br /> causing false negative errors. In TM experiments,                 3. Developing Statistical Framework Correct TM <br /> insertions occurring extreme ends gene&#226;&#8364;&#8482;s coding                 Errors <br /> sequence sufficiently disrupt function                             We developed statistical framework capable of <br /> [23,24,25,26,27,28,29,30]. In cases, essential genes           incorporating potential factors strong associations with <br /> mistakenly assigned non-essential genes, creating false                FETmE FNTmN assignments. This model estimates <br /> negative error. We compared distributions position            overall error rates, assigns score indicate the <br /> insertions ORFs FNTmNs TNTmNs. As                    probability individual gene essential given TM <br /> expected, FNTmNs higher percentage transposon                 assignments. We incorporated potential <br /> insertions 39- 59-ends TNTmNs (Fig. 1B). To                 factors model. Since polar effects responsible for <br /> assess significance difference, simulated pure random        relatively number FETmEs, chose include this <br /> insertion experiments coding sequences (see Methods).            factor model. The general idea model illustrated <br /> The P-values showed 20%-most 39-end              Fig. 2. <br /> 5%-most 59-end regions, FNTmNs significantly                   This model requires assumptions: <br /> more insertions TNTmNs (Table S2). We named the <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;25% extreme ends&#226;&#8364;&#8482;&#226;&#8364;&#8482; rule used later model.                   (A)   Transposons insert randomly independently the <br />   (3) Number insertions gene: potentially causing                          coding region gene; and <br /> false negative errors. Occasionally, insertions gene             (B)   Each transposon ability disrupt gene&#226;&#8364;&#8482;s <br /> are insufficient completely disrupt function, especially              function, ability vary different regions <br /> the target gene relatively long [23,24,25,26,29,33]. We plotted                gene. <br /> the distribution number insertions gene both <br /> FNTmNs TNTmNs (Fig. 1C). The histogram showed                         Assumption (A) does require transposon insertions be <br /> about 75% FNTmNs mistakenly assigned                 uniformly distributed entire genome the <br /> non-essential TM harbored single insertion. The average           insertion &#226;&#8364;&#732;&#226;&#8364;&#732;hot&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;cold&#226;&#8364;&#8482;&#226;&#8364;&#8482; spots observed microbial genomes <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                            March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                                    Improving Essential Gene Annotations <br />  <br />  <br />  <br />  <br /> Figure 2. Illustration statistical model. In TM experiment, gene observed insertions, meaning TM essential TmEs, what <br /> could be? There possibilities: (1) Part A: It insertion missed transposons chance. This means have <br /> useful information infer gene be, completely blind us. For blind gene, try best guess assume <br /> that chance gene essential (Pr (E~1Dnreal &#194;&#167;nobs ~0)) equal overall essential gene rate (Pr(overall essential)), gene <br /> to non-essential equal Pr (E~0Dnreal &#194;&#167;nobs ~0) = 1-Pr (E~1Dnreal &#194;&#167;nobs ~0). (2) Part B: It actually insertions, inserted mutations <br /> died. (Pr (E~1,nreal w0Dnreal &#194;&#167;nobs ~0)) This means gene truly essential. In way, split TM assigned essential genes into <br /> two parts, TETmE FETmE. Similarly, TM experiment, gene observed insertions, meaning TM nonessential, it <br /> really be? There possibilities: (1) Part C: All observed insertions ineffective, did interrupt gene function. This means <br /> again blind gene. So certain chance essential (Pr (E~1,allinsertionsineffectiveDnreal ~nobs w0)), certain <br /> chance nonessential (1-Pr (E~1,allinsertionsineffectiveDnreal ~nobs w0)). (2) Part D: There effective insertion, did <br /> interrupt gene function. (Pr (E~1,atleastoneinsertioneffectiveDnreal ~nobs w0)~0). This means gene truly non-essential. <br /> doi:10.1371/journal.pone.0058178.g002 <br />  <br /> and provides realistic approximation process               Here E binary variable E = 1 gene essential; <br /> transposon insertions.                                                           otherwise, non-essential. nreal represents real number of <br />    If assume transposons insert gene independently                insertions occurring gene TM experiment and <br /> and insertions occur constant rate mutagenesis,                  nobs represents observed number insertions TM <br /> then process characterized Poisson distribution                 dataset. Here nobs = 0 gene assigned essential TM, <br /> [34,35]. The probability k insertions occurring                   nobs .0 assigned non-essential. nobs further <br /> within gene length L expressed as:                                 separated parts n30 or50 ends nmiddle represent the <br />                                                                                  observed insertions occurring 25% extreme ends the <br />                                                                                  middle gene, respectively. In transposon insertion process, <br />                    Poisson&#195;&#176;k; rL&#195;&#382;~e{rL &#195;&#176;rL&#195;&#382;k =k!                                 insertion hits true essential gene disrupts function, <br />                                                                                  inserted mutant die insertion be <br />    Here r local insertion density DNA fragment, <br />                                                                                  observable TM dataset; real insertion number <br /> estimated counting number insertions 30 kb- <br />                                                                                  nreal greater equal observed insertion number <br /> long region flanking coding sequence. If k = 0, equation <br />                                                                                  nobs . While non-essential gene, matter an <br /> describes probability ORF missed TM <br />                                                                                  insertion disrupts function not, die. Thus real <br /> experiments. <br />                                                                                  insertion number nreal equals observed insertion <br />    Based assumption (B), defined parameters P1 P2                  number nobs . <br /> both range (0, 1) representing probability                    After derivation equations, used iterative <br /> individual transposon insertion disrupts gene&#226;&#8364;&#8482;s function              procedure [36,37,38] estimate values unknown param- <br /> insertion occurs 25% extreme ends middle                   eters determine Eqs. (1) (2) (details Methods). <br /> gene, respectively. We assumed probability (P1) the <br /> 5%-most 59-end 20%-most 39-end disrupt gene&#226;&#8364;&#8482;s <br /> function. <br />                                                                                  4. Validating Model E. coli TM Dataset <br />                                                                                     The Gerdes dataset contains 615 TmEs 3218 TmNs. Using <br />    Under assumptions, calculate probability <br />                                                                                  algorithm, converged P1 = 0.942, <br /> of essential gene given TM assignments. <br />                                                                                  P2 = 0.984 overall essential rate ress = 12.84%. Since our <br /> (A)    If TM assigned essential gene (TmE), have:                     model assigned individual gene score indicate its <br />                                                                                  probability essential, ranked genes TmEs <br />                                                                                  TmNs separately. Among   P      615 TmEs, expected <br />                 Pr (E~1DTmE)~ Pr (E~1Dnreal &#194;&#167;nobs ~0)                  &#195;&#176;1&#195;&#382;       number essential genes ( ni P(E~1)) estimated 480. <br />                                                                                  Using expected number essential genes cutoff, top <br /> (B)    Similarly, TM assigned non-essential gene (TmN), <br />                                                                                  480 genes named predicted essential genes model <br />        have: <br />                                                                                  TM-assigned essential genes (PETmEs) the <br />                                                                                  remaining 135 genes predicted non-essential genes our <br />                                                                                  model TM-assigned essential genes (PNTmEs). <br />                                                                                  Similarly, 3218 TmNs, expected number of <br />             Pr (E~1DTmN)~ Pr (E~1Dnreal ~nobs w0)                      &#195;&#176;2&#195;&#382;       essential genes estimated 12. Using cutoff, top <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   4                             March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                              Improving Essential Gene Annotations <br />  <br />  <br /> 12 genes named predicted essential genes model <br /> among TM-assigned non-essential genes (PETmNs) the <br /> remaining ones predicted non-essential genes the <br /> TM-assigned non-essential genes (PNTmNs). <br />    To assess accuracy predictions, compared our <br /> results PEC dataset (Table 2). Among 480 PETmEs, <br /> 176 (or 37%) true essential, significantly higher in <br /> the original TmEs (186/615 = 30%) (P-value = 0.013, Fisher&#226;&#8364;&#8482;s <br /> exact test). Remarkably, 135 PNTmEs filtered <br /> out, 10 (or 7.4%) true essential, significantly lower than <br /> that original TmEs, i.e., 30% (P-value ,1E-8). On other <br /> hand, 12 PETmNs, 5 (or 42%) true essential, <br /> significantly higher original TmNs (2.3%) (P-value <br /> ,1E-6). These results strongly indicated model success- <br /> fully enhanced accuracy original TM assignments. <br />    Our results showed positive correlation              Figure 3. Enrichment true essential genes using different <br /> confidence score enrichment essential genes (Fig. 3). In        thresholds confidence score. <br /> other words, higher confidence scores chose cutoff,          doi:10.1371/journal.pone.0058178.g003 <br /> the higher percentage true essential genes predictions <br /> contained, indicating score agreement           6. Testing Model&#226;&#8364;&#8482;s Applicability P. aeruginosa by <br /> the distribution essential genes. For instance, 100 <br />                                                                            Allelic Exchange Experiments <br /> PETmEs contained 50 true essential genes, ratio signifi- <br />                                                                               The ultimate test model applicable other <br /> cantly higher original TmEs (30%) P-value <br />                                                                            microorganisms. A set essential genes determined by <br /> ,1E-4. <br />                                                                            TM saturated level c-Proteobacteria, P. aeruginosa <br />                                                                            PAO1 (Jacobs dataset) [29]. This TM dataset contains 678 putative <br /> 5. Testing Model&#226;&#8364;&#8482;s Robustness Analyzing                             essential genes 4892 non-essential genes. If assume the <br /> Subsaturation Level TM Datasets                                            probability individual transposon insertion disrupts a <br />    Compared saturated TM experiment, unsaturated                 gene&#226;&#8364;&#8482;s function, different species, i.e., P1 = 0.942 <br /> TM experiment generally contains higher TmER, genes              P2 = 0.984 E. coli, overall essential rate ress <br /> are likely missed transposon insertions             estimated PAO1 10.1% expected numbers of <br /> incorrectly assigned essential.                                         PETmEs PETmNs 540 15, respectively. <br />    To test model&#226;&#8364;&#8482;s applicability unsaturated TM datasets,              Because whole-genome single gene knockout dataset yet <br /> we randomly removed 10%, 30% 50% total insertions               available organism, chose pursue allelic exchange <br /> from Gerdes dataset simulate effects different               experiments validate predictions PAO1. <br /> subsaturation levels TM experiments. We applied                   In order make sure experimental procedure correctly <br /> model subsaturated datasets.                                      identify essential genes, tested PA4238 positive <br />    The results suggested strong robustness analyzing                  control. PA4238 subunit RNA polymerase, the <br /> subsaturation level TM datasets (Fig. 3). As shown Fig. 4, <br /> the lower curve (dashed line) showed p-values Fisher&#226;&#8364;&#8482;s <br /> exact test examine true essential rate PNTmEs is <br /> significantly lower original TmEs set. Similarly, <br /> the upper curve (solid line) showed p-values Fisher&#226;&#8364;&#8482;s <br /> exact test examine true essential rate our <br /> PETmNs significantly higher original TmNs set. <br /> For (10%, 30% 50%) random experiments, <br /> we repeated 100 times obtain error bars. The results showed <br /> that subsaturation conditions, model still <br /> significantly improved TM results. <br />  <br />  <br />  Table 2. Improvement overlaps PEC dataset using <br />  model. <br />  <br />  <br />  TM dataset        Our Statistical       PEC dataset (Gold Standard) <br />  <br />  (Gerdes set)      Model                 E (259)         N (3574) <br />                                                                            Figure 4. Robustness model subsaturation levels of <br />  TmEs (615)        PETmEs (480)          176             304               transposon insertions. The dashed line showed p-values the <br />                    PNTmEs (135)          10              125               Fisher&#226;&#8364;&#8482;s exact test examine true essential rate PNTmEs <br />                                                                            significantly lower original TmEs set. Similarly, the <br />  TmNs (3218)       PETmNs (12)           5               7 <br />                                                                            solid line showed p-values Fisher&#226;&#8364;&#8482;s exact test examine whether <br />                    PNTmNs (3206)         68              3138              true essential rate PETmNs significantly higher in <br />                                                                            original TmNs set. <br />  doi:10.1371/journal.pone.0058178.t002                                     doi:10.1371/journal.pone.0058178.g004 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             5                              March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                               Improving Essential Gene Annotations <br />  <br />  <br /> target microbicidal antibiotic, Rifampin [39]. The                    non-essential. Our model correct three <br /> essentiality PA4238 confirmed independent gene knockout             cases. <br /> efforts [24,29]. As expected, PA4238 determined allelic              Overall, seven genes tested allelic exchange <br /> exchange experiments essential. According model,                   experiments, model agreed experimental validations <br /> PA4238 rank 46 678, higher expected                    them. Remarkably, cases directly <br /> number essential genes TmEs, i.e., 540;            contradicted TM assignments, allelic exchange experiments <br /> predicted essential model. In case, results          supported predictions. Details allelic exchange experiments <br /> the TM assignment, model allelic exchange                        provided Supporting Information (Fig. S2 and <br /> experiment consistent (Table 3).                                         Table S3). <br />    We selected ORFs PAO1 TM dataset to <br /> further test model: PA0723, PA2954 PA2143                       7. Examples Identified TM Annotation Errors <br /> selected TmEs; PA3746, PA4260 PA0985 were <br />                                                                              Confirmed Literature <br /> selected TmNs. The main purpose conduct allelic <br />                                                                                 Among TM annotation errors PAO1 identified our <br /> exchange experiments validations demonstrate TM <br />                                                                              model, number confirmed literature. <br /> indeed contains errors method capable correcting <br />                                                                              Below examples: <br /> some errors. To serve purpose, chose genes <br /> that likely TM errors testing cases based              Hfq (PA4944) RNA-binding protein PAO1 involved <br /> the following considerations: First, testing cases TmEs            Bacterial RNA degradation pathway. It identified an <br /> should lower ranked (close cutoff 540)           essential gene TM experiments [29], assigned non- <br /> the testing cases TmNs higher ranked (close            essential model (ranked 572 678 TmEs). A recent <br /> above cutoff 15). The higher-ranked genes TmEs                 study [40] investigated effect Hfq virulence stress <br /> lower-ranked genes TmNs agreements                  response PAO1 compared growth rate wild-type <br /> TM model likely correct assignments;                  strain Dhfq strain. The Dhfq strain showed reduced <br /> therefore, interesting testing cases. Second,              growth rate compared wild-type strain, indicates <br /> among lower-ranked TmEs higher-ranked TmNs,                     gene essential fitness survival should <br /> challenged selecting difficult cases, i.e.,       referred non-essential gene here. <br /> either ortholog E. coli assignments based              FpvI (PA2387) RNA polymerase sigma factor a <br /> on orthologs contradictory TM assignments. Third,                  TM-assigned essential gene PAO1. Our model predicted as <br /> chose testing cases sides cutoffs test       non-essential assigning rank 582 678 TmEs. <br /> robustness cutoff setting. Finally, short list           Beare et.al studied role siderophore-mediated cell signaling <br /> genes met criteria, nailed               [41]. They FpvI required synthesis of <br /> based experimental feasibility.                                           ferripyoverdine receptor FpvA signaling <br />    Among TmEs (PA2954, PA0723 PA2143),                         pathway regulating pyoverdine (a form siderophore) production. <br /> PA0723 true essential gene PA2954 PA2143                       In experiment, constructed DfpvI strain that <br /> turned non-essential. According model, PA0723               mutant produced lower amounts FpvA wild- <br /> was ranked 414 678. Because expected number                    type stain, suggests FpvI required normal <br /> essential genes TmEs 540, correctly predicted             amounts FpvA production survival. <br /> be essential. In contrast, PA2954 PA2143 given rank               PcrG (PA1705) regulator type III secretion (TTSS) <br /> 588 663, respectively. They predicted false                 enables bacteria translocate virulence effectors <br /> positive error (i.e., non-essential) genes ranks          directly cytosol host cells. It ranks 559 the <br /> lower 540. Our model correct cases.                    678 TmEs model predicted non-essential gene, <br />    Among TmNs (PA3746, PA4260 PA0985),                         contradicting TM assignment. To understand its <br /> PA0985 true non-essential gene PA3746 PA4260                   mechanism, Sundin&#226;&#8364;&#8482;s group constructed in-frame deletion <br /> were tested essential. Our model assigned PA3746 rank             mutant PcrG cultured mutant LB medium. They <br /> 8 4892. Because rank higher 15, expected             PcrG mutant up-regulate expression of <br /> number essential genes TmNs, PA3746                         exoenzyme S (ExoS) identified effectors <br /> predicted essential. In contrast, PA0985 PA4260               targeted host cells TTSS PAO1 [42]. This <br /> assigned rank 113 103, respectively; predicted            experiment proved PcrG non-essential gene. <br />  <br />  Table 3. Validation using allelic exchange experiments Pseudomonas aeruginosa PAO1. E &#226;&#8364;&#8220; Essential; N &#226;&#8364;&#8220; Non-essential. <br />  <br />  <br />                                 Local Insertion                                      Assignments                   Assignments Allelic <br />  PAO1 genes       Length        Density         Assignments TM Ranks Our Model model                  exchange experiments <br />  <br />  PA3746           1374          4.9521        N                    8/4289                  E                       E <br />  PA4260           822           3.4102        N                    103/4289                N                       E <br />  PA0985           1497          7.5415        N                    113/4289                N                       N <br />  PA4238           1002          3.3564        E                    46/678                  E                       E (Positive Control) <br />  PA0723           249           7.4446        E                    414/678                 E                       E <br />  PA2954           570           2.0336        E                    588/678                 N                       N <br />  PA2143           288           2.1479        E                    663/678                 N                       N <br />  <br />  doi:10.1371/journal.pone.0058178.t003 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               6                            March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                                  Improving Essential Gene Annotations <br />  <br />  <br />  <br /> 8. Testing Model&#226;&#8364;&#8482;s Broad Applicability Other                            essential rate original TmNs. The confidence <br /> Organisms                                                                      scores generated model shown positive <br />    Francisella tularensis Gram-negative, intracellular pathogen           correlations enrichment true essential genes. <br /> that causes disease tularemia. F. tularensis novicida commonly              Being able assign PETmNs demonstrates advantage of <br /> used surrogate virulence studies particularly virulent            approach previous studies recovering true essential <br /> mice able cause tularemia-like disease rodents.              genes TmNs. Most existing models, e.g., [28], only <br /> Genome-scale TM experiments performed                         focused removing false essential genes TmEs, are <br /> subspecies F. tularensis novicida [25]. This TM library consists            incapable recovering false non-essential genes, although <br /> 16,508 unique insertions covering 1,434 1,767 genes,             relatively few, TmNs. In E. coli, 12 false non- <br /> average .9 insertions gene, achieves highest                  essential genes recovered, true essential, significantly <br /> coverage available TM experiments bacterial                       higher original TmNs (Table 2). <br /> species. We applied model TM library estimated                     Second, model adopts simple realistic <br /> the overall essential rate 14.5%. Among 333 TmEs,                    assumptions, applicable multiple microorganisms. It is <br /> predicted 251 genes essential labeled PETmEs. Among                 important note method dependent single <br /> the 1434 TmNs, predicted 5 essential labeled                      gene knockout results make predictions. The single gene <br /> PETmNs. The prediction results available                 knockout results E. coli used assessing TM <br /> Web server described section. The TM annotations                   errors evaluating performance predictions. In P. <br /> from microorganisms processed deposited               aeruginosa PAO1 single gene knockout results unavailable, <br /> Web server results available.                                  demonstrated method remarkably accurate based on <br />                                                                                literature allelic exchange experiments. Among the <br /> 9. Developing Web Server Provides Convenient                            seven chosen genes experimental test, were <br /> Access Our Method                                                           assigned correctly model overall accuracy 86%. <br />    To make model readily available research                         These results clearly demonstrated method&#226;&#8364;&#8482;s reliability and <br /> community, developed convenient user-friendly Web                     applicability organisms single gene knockout <br /> service named EGTEC (The Essential Gene TM Annotation                          results. We demonstrated model&#226;&#8364;&#8482;s broad applicability <br /> Error Corrector) hosted at: http://research.cchmc.org/                         F. tularensis novicida, single gene knockout results also <br /> essentialgene/. The EGTEC interface implemented using                       unavailable. Therefore, believe, granted access the <br /> PHP-HTML comprises main functions (Fig. 5):                         information transposon insertion positions bacterial <br />                                                                                genomes, model readily applicable 24 available <br /> (1) A query tool quick exploration corrected TM                   TM datasets listed Table S1. <br />     sample results bacteria, currently includes E. coli, P.               Third, model displays robustness analyzing unsaturation <br />     aeruginosa F. tularensis novicida. Once query received,       level TM datasets resisting experimental errors as <br />     tool check gene&#226;&#8364;&#8482;s presence database                demonstrated simulated unsaturation level TM datasets. <br />     displays matching records input, including                 This potentially useful significantly reducing time and <br />     original TM annotations EGTEC&#226;&#8364;&#8482;s prediction                 costs currently associated TM experiments. <br />     results single-gene knockout experiments                        Finally, model flexible able incorporate more <br />     available. The tool accept input query gene&#226;&#8364;&#8482;s                potential factors. For example, assigned different weights the <br />     following formats: gene symbol/name, GenBank                        insertions based positions inserted the <br />     accession number Entrez Gene ID.                                        genes. In future, consider factors affect <br /> (2) An uploading tool submitting new TM experimental data                  accuracy TM experiments [33]. <br />     users. The EGTEC accepts user-generated TM experi-                          To benchmark method&#226;&#8364;&#8482;s performance different <br />     mental results makes corrections results. The                 types computational methods, compared prediction <br />     input contain following information: (a) Each                   performance Table S4. We included studies <br />     ORF&#226;&#8364;&#8482;s name, (b) Start end position ORF (c)                 conducted E. coli order make objective comparison. <br />     The transposon insert position ORF. A sample input                 It surprising supervised methods rely gold <br />     file provided uploading box. When finished,              standard datasets outperform non-supervised methods that <br />     corrected TM annotations including predicted                       rely gold standard datasets. The main problem of <br />     probability ORF essential sent                supervised learning applicable understudied <br />     user email.                                                        species essential genes known; <br />                                                                                therefore, superior performance sensitivity, specificity and <br />                                                                                precision meaningful problem intend solve. <br /> Discussion                                                                     Although constraint-based method outperforms model in <br />    The intrinsic biases TM experiments motivated develop              precision, sensitivity specificity poor. This reflects <br /> a statistical framework systematically filter errors (both              limitation dependent priori knowledge pathways <br /> FETmEs FNTmNs) improved accuracy TM-                       reactions. Homology mapping similar level of <br /> determined essential gene annotations. This model significant            performance model, requires closely related model <br /> four ways:                                                                     organism, accuracy assignment highly dependent on <br />    First, model significantly enhances accuracy                 distance model organism target organism. <br /> original TM assignments. In E. coli TmEs, PNTmEs, i.e.,                In case, homology mapping performs mainly because <br /> the false essential genes filtered method,                    single gene knockout results available well-studied and <br /> significantly lower true essential rate original              closely-related organism, Acinetobacter baylyi [43]. It unrealistic to <br /> TmEs. In contrast, TmNs, PETmNs, i.e., predicted                expect model organism available an <br /> essential genes TmNs, significantly higher true                     understudied species. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 7                             March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                             Improving Essential Gene Annotations <br />  <br />  <br />  <br />  <br /> Figure 5. Interface EGTEC Web server. <br /> doi:10.1371/journal.pone.0058178.g005 <br />  <br />    Our approach represents significant advancement                   gold-standard dataset available M. tuberculosis, we <br /> Gerdes approach filter TM errors. First, cutoff settings          compare subsaturation results theirs. In addition, <br /> Gerdes approach appeared somewhat arbitrary lack               issues Jacobs et al.&#226;&#8364;&#8482;s method by <br /> statistical rigorousness. For example, genes longer 240 bp         disregarding false negative errors differences insertion <br /> and free inserts assigned essential, genes         density. By taking account features, model is <br /> least insertion designated non-essential unless        expected realistic accurate. <br /> relatively long (.900 bp). In contrast, parameters             Our effort timely given large number existing <br /> determined iterative learning process. Second, Gerdes           annotations essential genes rapidly increased TM datasets. <br /> annotations strictly based TM data involved            The need assess TM errors even <br /> manual inspection requires prior knowledge E. coli                  urgent researchers start explore conditional essentials, <br /> physiology. Therefore, Gerdes approach easily extended            hold key understanding gene essentiality. For <br /> to understudied organism expected achieve                past years, seen numerous new bacterial <br /> performance. In contrast, approach relies TM                essential gene sets published majority were <br /> data, i.e., transposon insertion positions bacterial genome.         generated TM approach [44,45,46,47,48,49,50,51,52,53] <br /> Even using prior knowledge improve assignments,              (Table S1). We expect growth essential gene datasets to <br /> our model significantly outperformed Gerdes approach               greatly accelerate. Therefore contribution valuable <br /> precision specificity (Table S4).                                       timely. As long TM experiments remain dominant <br />    In organisms, performance metrics              method determine essential gens prokaryotes, method is <br /> be easily assessed definite mappings essential genes         expected remain useful. <br /> not available species, approach exhibits              A logical step investigate model&#226;&#8364;&#8482;s applicability to <br /> advantages: Our model capable estimating probability              eukaryotes. However, eukaryotes more <br /> score essential individual genes,             complex genome structure prokaryotes, e.g., introns, the <br /> estimating overall essential rate genome             mechanism eukaryotic essential genes expected quite <br /> Jacobs et al.&#226;&#8364;&#8482;s method. In addition, Jacobs et al. used multinomial       different prokaryotic essential genes. For reason, <br /> distribution incorporate difference                transposons likely work differently eukaryotes in <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;hot&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;cold&#226;&#8364;&#8482;&#226;&#8364;&#8482; spots. Furthermore, method                prokaryotes. A thorough investigation aspects be <br /> applicable correcting false negative errors, i.e., detecting true       performed apply model eukaryotic genomes. <br /> essential genes TmNs. In Lamichhane et al.&#226;&#8364;&#8482;s study,                In summary, developed promising tool crucial <br /> method applicable subsaturation level TM. Since               large-scale data mining essential genes. The analysis <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              8                            March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                               Improving Essential Gene Annotations <br />  <br />  <br /> enabled tool eventually lead better understanding <br /> on mechanistic basis underlying gene essentiality.                                       z Pr (E~1,nreal w0Dnreal &#194;&#167;nobs ~0)           &#195;&#176;PartB&#195;&#382; <br />  <br /> Materials Methods <br />                                                                               Part A: <br /> 1. Data Sources <br />    In study, tested model different prokaryotic <br /> organisms: E. coli, P. aeruginosa PAO1 F. tularensis novicida. We        Pr (E~1,nreal ~0Dnreal &#194;&#167;nobs ~0) <br /> chose study organisms genomic <br /> sequences TM datasets publicly available. <br />    The genomic protein sequences E. coli MG1655                  ~ Pr (E~1Dnreal ~0,nreal &#194;&#167;nobs ~0) Pr (nreal ~0Dnreal &#194;&#167;nobs ~0) <br /> downloaded Comprehensive Microbial Resource (CMR) <br /> at http://cmr.jcvi.org/. It contained 4,289 protein coding genes         Pr (E~1Dnreal ~0,nreal &#194;&#167;nobs ~0)~ Pr (E~1Dnreal ~0) is <br /> total.                                                                      probability gene essential given actual insertion <br />    The E. coli essential dataset downloaded Profiling E.        number equal 0. This means insertion was <br /> coli Chromosome (PEC) v4 http://www.shigen.nig.ac.jp/. This              missed transposons chance. Thus, infer <br /> dataset (PEC set) contained 302 essential genes 4477 non-               information gene, assume probability being <br /> essential genes.                                                            essential equal overall percentage essential genes <br />    The E. coli TM dataset downloaded [23]. Of total            genome. We have: <br /> 4291 protein-coding genes dataset, 3,311 ORFs have <br /> observed transposon insertions 649 ORFs not. The                                Pr (E~1Dnreal ~0)~ Pr (overall essential): <br /> remaining 331 genes excluded analysis no <br /> reliable PCR data obtained corresponding region of <br /> the E. coli chromosome technical reasons.                              Since transposon insertion process follows Poisson <br />    The total 5,568 protein sequences P. aeruginosa PAO1             distribution, get: <br /> downloaded                     http://www.pseudomonas.com/ <br /> (Pseudomonas_aeruginosa_PAO1.faa, revision 2009-07-17). <br />                                                                                                                  Pr (nreal ~0,nreal &#194;&#167;nobs ~0) <br />    The P. aeruginosa PAO1 TM dataset downloaded [29].                  Pr (nreal ~0Dnreal &#194;&#167;nobs ~0)~ <br /> This dataset contained 4,892 ORFs observed transposon                                                      Pr (nreal &#194;&#167;nobs ~0) <br /> insertions 678 ORFs not.                                                                         ~Poisson(0; rL) <br />    The F. tularensis novicida TM dataset downloaded [25]. <br /> This datasets contain 333 putative essential genes 1,767 <br /> putative non-essential genes.                                                 Part B: <br />    The E. coli operon dataset downloaded Regulon DB <br /> version 6.4 http://regulondb.ccg.unam.mx/. This dataset                                  Pr (E~1,nreal w0Dnreal &#194;&#167;nobs ~0) <br /> contained 2,665 operons inferred experimentally compu- <br /> tationally E. coli genome. <br />                                                                                             X       &#194;&#189;Pr (E~1jnreal ~m,nreal &#194;&#167;nobs ~0) <br /> 2. Simulation Random Insertions                                                      ~     mw0 <br />    Assuming uniform distribution transposon insertions                                          Pr (nreal ~mjnreal &#194;&#167;nobs ~0)  <br /> within ORF, conducted simulation experiment for <br /> random insertions. In Gerdes set, observed 12,966 total                 Pr (E~1Dnreal ~m, nreal &#194;&#167;nobs ~0), mw0 proba- <br /> transposon insertions inside ORFs, excluding intergenic insertions.         bility gene essential given real insertion number <br /> We randomly generated 12,966 insertions genome,                   m (m .0), insertions survived. Since didn&#226;&#8364;&#8482;t <br /> excluding intergenic regions. For random insertion,                 observe insertion, m insertions effective; <br /> recorded relative position inside ORF. Repeating               otherwise, mutants ineffective insertions have <br /> simulation 1,000 times, obtained distribution                 survived. In case, target gene essential, all <br /> positions simulated insertions. We evaluated empirical          insertions interrupted function caused mutant&#226;&#8364;&#8482;s death. <br /> insertion distribution TmNs (both TNTmNs FNTmNs)                  Thus, have: <br /> calculating probability P randomized insertions <br /> appearing certain position equal greater number <br /> of times real experiment. We concluded                            Pr (E~1Dnreal ~m,nreal &#194;&#167;nobs ~0)~1 <br /> a significant difference P-values #0.01 (Table S2). <br />  <br /> 3. Derivation Equations                                                              Pr (nreal ~mDnreal &#194;&#167;nobs ~0)~Poisson(m; rL) <br />   Based probability theory, Eq. (1) rewritten into <br /> two parts (Fig. 2): <br />                                                                               Similarly, Eq. (2) rewritten parts (Fig. 2): <br />                 Pr (E~1Dnreal &#194;&#167;nobs ~0) <br />                                                                                     Pr (E~1Dnreal ~nobs w0) <br />  <br />                ~ Pr (E~1,nreal ~0Dnreal &#194;&#167;nobs ~0)           &#195;&#176;PartA&#195;&#382; <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              9                              March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                                                                                          Improving Essential Gene Annotations <br />  <br />  <br />  <br />                                                                                                                                                                  X <br /> ~ Pr (E~1, insertions ineffective Dnreal ~nobs w0)                                                &#195;&#176;PartC&#195;&#382;                            E(TETmE)~                      Pr (Ei ~1Dnireal &#194;&#167;niobs ~0) <br />                                                                                                                                                                 i[TmE <br />                                                                                                                                                                  X                                                        &#195;&#176;4&#195;&#382; <br />                                                                                                                                          E(FNTmN)~                      Pr (Ei ~1Dnireal ~niobs w0) <br /> z Pr (E~1, insertion effectiveDnreal ~nobs w0)&#195;&#176;PartD&#195;&#382;                                                                                              i[TmN <br />  <br />  <br />  <br />    Part C:                                                                                                                     Then ress estimated by: <br />    Since insertions disrupted gene&#226;&#8364;&#8482;s function <br /> and infer information gene,                                                                                             E(TETmE)zE(FNTmN) <br /> assume probability essential equal overall                                                                          ress ~                                                                     &#195;&#176;5&#195;&#382; <br />                                                                                                                                                                 NTmE zNTmN <br /> percentage essential genes genome, thus, <br />  <br />        Pr (E~1, insertions ineffectiveDnreal ~nobs w0)                                                                     Substituting (4) (5), get: <br />  <br />       ~ Pr (overallessential)(1{P1 )n30 or50 ends (1{P2 )nmiddle                                                                                                P P <br />                                                                                                                                                                            Poisson(m; ri Li ) <br />                                                                                                                                                                i[TmE mw0 <br />                                                                                                                             ress ~                   P                             P                                          &#195;&#176;6&#195;&#382; <br />                                                                                                                                                                                                 ni 0 0               i <br />                                                                                                                                      NTmE zNTmN {           Poisson(0; ri Li ){              ^1) <br />                                                                                                                                                                                           (1{P                 ^ 2 )nmiddle <br />                                                                                                                                                                                                  3 or5 ends (1{P <br />                                                                                                                                                     i[TmE                         i[TmN <br />   Part D: <br />   Since observed effective insertions target gene the <br /> mutant gene disrupted survived, target gene                                                                                                    ^ 1 ,P <br />                                                                                                                                 Step 3. Using current ress P    ^ 2 , assign essential <br /> cannot essential.                                                                                                        probability score individual gene TM dataset. <br />                                                                                                                                 Step 4. Ranking essential probability scores TmEs set <br />     Pr (E~1,atleastoneinsertioneffectiveDnreal ~nobs w0)~0                                                                  using expected number essential genes dataset as <br />                                                                                                                             cutoff, genes considered PETmEs. Similarly, <br />                                                                                                                             ranking essential probability scores TmNs dataset using <br />                                                                                                                             expected number essentials dataset cutoff, the <br /> 4. Estimating Parameters Using Iterative Procedure                                                                       genes considered PETmNs, filtered essential <br />    We iteratively estimated unknown parameters P1 (the                                                                  dataset combination PETmEs PETmNs. <br /> probability individual insertion disrupts gene&#226;&#8364;&#8482;s function                                                             Step 5. Updating current essential dataset S&#195;&#176;Ess&#195;&#382; using this <br />                                                                                                                             filtered essential dataset remaining genes update the <br /> when occurs 25% extreme ends gene), P2 (the <br />                                                                                                                             current non-essential dataset S &#195;&#176;Non{ess&#195;&#382;. <br /> probability individual insertion disrupts gene&#226;&#8364;&#8482;s function <br />                                                                                                                                 Step 6. Going Step 1 result converges. <br /> when occurs middle gene) and <br />                                                                                                                                 In model, initial P1 P2 randomly assigned <br /> ress ~ Pr (OverallEssential) (the true essential rate genome) <br />                                                                                                                             0 1, updated iteration process until <br /> as follows: <br />                                                                                                                             converge. <br />    Step 1. Based model assumptions, defined the <br /> empirical estimators P1 P2 calculating number of <br /> corresponding insertions (denominator) effectiveness                                                              5. Experimental Validation P. aeruginosa PAO1 <br /> (numerator) using <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> follows:                                                                                   To verify predictions, conducted allelic exchange <br />                                                                                                                             experiments seven chosen genes: PA4238, PA0723, <br />                                             P                        P                                                      PA0985, PA2954, PA2143, PA3746 PA4260. First, using <br />                                                   ni30 or50 ends =          Li                                              genomic DNA strain PAO1 template standard PCR <br />                                          i[S(Ess)                  i[S(Ess) <br /> ^ 1 ~1{ <br /> P                 P                               P                     P                      P                            techniques, cloned PCR fragments containing desired <br />           (                  ni30 or50 ends =                 Lj z            ni30 or50 ends =   Li ) <br />               j[S(Non{ess)                    j[S(Non{ess)               i[S(Ess)                     i[S(Ess)              genes 1 kb flanking DNA pUC19. We inserted a <br />                                             P                        P                                           &#195;&#176;3&#195;&#382;        non-polar aacC1 (GmR) cassette pUCGM [54] target <br />                                                      nimiddle =:              Li <br /> ^ 2 ~1{                                   i[S(Ess)                 i[S(Ess)                                                 gene transcriptional orientation ensure was <br /> P                 P                            P                       P                             P <br />           (                  nimiddle =                    Lj z                 ni30 or50 ends =              Li )          polar effects downstream genes. The inserted fragments <br />               j[S(Non{ess)                j[S(Non{ess)               i[S(Ess)                      i[S(Ess)                 subcloned gene replacement vector, pEX100T, <br />                                                                                                                             resultant constructs conjugated PAO1 with <br />                                                                                                                             selection GmR. Transconjugates resolved LB agar <br />                                                                                                                             containing 7% sucrose counter-selectable marker. Isogenic <br />     Here S&#195;&#176;Ess&#195;&#382; S &#195;&#176;Non{ess&#195;&#382; denote set essential                                                             mutants confirmed initially PCR Southern blot. <br /> non-essential genes current step, respectively. nimiddle                                                      Complementation single copy using unique attB <br /> observed insert number middle regions ith gene;                                                           locus stable plasmids (details Text S1). <br /> ni30 or50 ends observed insert number 39 59ends Li is <br /> the length gene.                                                                                                    Supporting Information <br />     Step 2. Using P     ^ 1 ,P <br />                              ^ 2 estimate ress : <br />     In step, first, calculated expected number essential                                                     Figure S1       Identification essential genes TM. <br /> genes TmEs TmNs respectively, following equation:                                                             (DOC) <br />                                                                                                                             Figure S2 PCR analysis PA0985 merodiploid (single <br />                                                                                                                             crossover) mutant (double crossover) recombinant <br />                                                                                                                             strains. <br />                                                                                                                             (DOCX) <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                                                             10                                      March 2013 | Volume 8 | Issue 3 | e58178 <br />                                                                                                                                    Improving Essential Gene Annotations <br />  <br />  <br /> Table S1.                                                                                  Text S1. <br /> (DOC)                                                                                      (DOC) <br /> Table S2. <br /> (DOC)                                                                                      Author Contributions <br /> Table S3.                                                                                  Conceived designed experiments: JD LJL. Performed the <br />                                                                                            experiments: SS DJH. Analyzed data: JD XL. Wrote paper: JD <br /> (DOC) <br />                                                                                            LJL SS DJH. <br /> Table S4. <br /> (DOC) <br />  <br /> References <br />  1. Dowell RD, Ryan O, Jansen A, Cheung D, Agarwala S, et al. (2010) Genotype              27. Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, et al. (2006) <br />     <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span>: complex problem. Science 328: 469.                                         Essential genes minimal bacterium. Proc Natl Acad Sci U S A 103: 425&#226;&#8364;&#8220; <br />  2. Kato J, Hashimoto M (2007) Construction consecutive deletions                    430. <br />     Escherichia coli chromosome. Mol Syst Biol 3: 132.                                     28. Lamichhane G, Zignol M, Blades NJ, Geiman DE, Dougherty A, et al. (2003) A <br />  3. Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al. (1999)                 postgenomic method predicting essential genes subsaturation levels of <br />     Functional characterization S. cerevisiae genome gene deletion               mutagenesis: application Mycobacterium tuberculosis. Proc Natl Acad <br />     parallel analysis. Science 285: 901&#226;&#8364;&#8220;906.                                                   Sci U S A 100: 7213&#226;&#8364;&#8220;7218. <br />  4. Mushegian A (1999) The minimal genome concept. Curr Opin Genet Dev 9:                  29. Jacobs MA, Alwood A, Thaipisuttikul I, Spencer D, Haugen E, et al. (2003) <br />     709&#226;&#8364;&#8220;714.                                                                                   Comprehensive transposon mutant library Pseudomonas aeruginosa. Proc <br />  5. Arigoni F, Talabot F, Peitsch M, Edgerton MD, Meldrum E, et al. (1998) A                   Natl Acad Sci U S A 100: 14339&#226;&#8364;&#8220;14344. <br />     genome-based approach identification essential bacterial genes. Nat         30. Hutchison CA, Peterson SN, Gill SR, Cline RT, White O, et al. (1999) Global <br />     Biotechnol 16: 851&#226;&#8364;&#8220;856.                                                                    transposon mutagenesis minimal Mycoplasma genome. Science 286: <br />  6. Bruccoleri RE, Dougherty TJ, Davison DB (1998) Concordance analysis                     2165&#226;&#8364;&#8220;2169. <br />     microbial genomes. Nucleic Acids Res 26: 4482&#226;&#8364;&#8220;4486.                                    31. Blades NJ, Broman KW (2002) Estimating number essential genes a <br />  7. Joyce AR, Palsson BO (2008) Predicting gene essentiality using genome-scale             genome random transposon mutagenesis. Technical Report MS02&#226;&#8364;&#8220;20, <br />     silico models. Methods Mol Biol 416: 433&#226;&#8364;&#8220;457.                                              Department Biostatistics, Johns Hopkins University Working Paper 15. <br />  8. Perumal D, Samal A, Sakharkar KR, Sakharkar MK (2011) Targeting multiple               32. Gerdes S, Edwards R, Kubal M, Fonstein M, Stevens R, et al. (2006) Essential <br />     targets Pseudomonas aeruginosa PAO1 using flux balance analysis                    genes metabolic maps. Curr Opin Biotechnol 17: 448&#226;&#8364;&#8220;456. <br />     reconstructed genome-scale metabolic network. J Drug Target 19: 1&#226;&#8364;&#8220;13.                  33. Gerdes SY, Scholle MD, D&#226;&#8364;&#8482;Souza M, Bernal A, Baev MV, et al. (2002) From <br />                                                                                                genetic footprinting antimicrobial drug targets: examples cofactor <br />  9. Plata G, Hsiao TL, Olszewski KL, Llinas M, Vitkup D (2010) Reconstruction <br />                                                                                                biosynthetic pathways. J Bacteriol 184: 4555&#226;&#8364;&#8220;4572. <br />     flux-balance analysis Plasmodium falciparum metabolic network. Mol <br />                                                                                            34. Good IJ (1986) Some statistical applications Poisson&#226;&#8364;&#8482;s work. Statistical science <br />     Syst Biol 6: 408. <br />                                                                                                1 (2): 157&#226;&#8364;&#8220;180. <br /> 10. Imam S, Yilmaz S, Sohmen U, Gorzalski AS, Reed JL, et al. (2011) iRsp1095: a <br />                                                                                            35. Ross SM (1996) Stochastic processes. New York: Wiley. xv, 510 p. p. <br />     genome-scale reconstruction Rhodobacter sphaeroides metabolic network. <br />                                                                                            36. Lehmann EL, Casella G (1998) Theory point estimation. New York: Springer. <br />     BMC Syst Biol 5: 116. <br />                                                                                                xxvi, 589 p.p. <br /> 11. Chen Y, Xu D (2005) Understanding protein dispensability machine- <br />                                                                                            37. Zolman JF (1993) Biostatistics : experimental design statistical inference. <br />     learning analysis high-throughput data. Bioinformatics 21: 575&#226;&#8364;&#8220;581. <br />                                                                                                New York: Oxford University Press. xv, 343 p.p. <br /> 12. Saha S, Heber S (2006) In silico prediction yeast deletion phenotypes. Genet        38. Balakrishnan N, Melas VB, Ermakov SM (2000) Advances stochastic <br />     Mol Res 5: 224&#226;&#8364;&#8220;232.                                                                        simulation methods. Boston: Birkha&#194;&#168;user. xxvi, 386 p.p. <br /> 13. Gustafson AM, Snitkin ES, Parker SC, DeLisi C, Kasif S (2006) Towards              39. Knox C, Law V, Jewison T, Liu P, Ly S, et al. (2011) DrugBank 3.0: a <br />     identification essential genes using targeted genome sequencing                     comprehensive resource &#226;&#8364;&#732;omics&#226;&#8364;&#8482; research drugs. Nucleic Acids Res 39: <br />     comparative analysis. BMC Genomics 7: 265.                                                 D1035&#226;&#8364;&#8220;1041. <br /> 14. Seringhaus M, Paccanaro A, Borneman A, Snyder M, Gerstein M (2006)                     40. Sonnleitner E, Hagens S, Rosenau F, Wilhelm S, Habel A, et al. (2003) Reduced <br />     Predicting essential genes fungal genomes. Genome Res 16: 1126&#226;&#8364;&#8220;1135.                    virulence hfq mutant Pseudomonas aeruginosa O1. Microb Pathog 35: <br /> 15. Deng J, Tan L, Lin X, Lu Y, Lu LJ (2012) Exploring optimal strategy                 217&#226;&#8364;&#8220;228. <br />     predict essential genes microbes. Biomolecules: 1&#226;&#8364;&#8220;22.                               41. Beare PA, For RJ, Martin LW, Lamont IL (2003) Siderophore-mediated cell <br /> 16. Deng J, Deng L, Su S, Zhang M, Lin X, et al. (2011) Investigating                      signalling Pseudomonas aeruginosa: divergent pathways regulate virulence <br />     predictability essential genes distantly related organisms using              factor production siderophore receptor synthesis. Mol Microbiol 47: 195&#226;&#8364;&#8220; <br />     integrative approach. Nucleic Acids Res 39: 795&#226;&#8364;&#8220;807.                                       207. <br /> 17. Judson N, Mekalanos JJ (2000) Transposon-based approaches identify                  42. Sundin C, Thelaus J, Broms JE, Forsberg A (2004) Polarisation type III <br />     essential bacterial genes. Trends Microbiol 8: 521&#226;&#8364;&#8220;526.                                    translocation Pseudomonas aeruginosa requires PcrG, PcrV PopN. <br /> 18. Winsor GL, Lam DK, Fleming L, Lo R, Whiteside MD, et al. (2011)                            Microb Pathog 37: 313&#226;&#8364;&#8220;322. <br />     Pseudomonas Genome Database: improved comparative analysis popula-                 43. Berardinis V, Vallenet D, Castelli V, Besnard M, Pinet A, et al. (2008) A <br />     tion genomics capability Pseudomonas genomes. Nucleic Acids Res 39:                    complete collection single-gene deletion mutants Acinetobacter baylyi <br />     D596&#226;&#8364;&#8220;600.                                                                                  ADP1. Mol Syst Biol 4: 174. <br /> 19. Zhang <br /> </body></html>