<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell Metab. 2012 October 3; 16(4): 526&#226;&#8364;&#8220;537. doi:10.1016/j.cmet.2012.09.007. <br />  <br />  <br />  <br />                            Mitochondrial SKN-1/Nrf Mediates Conserved Starvation <br />                            Response <br />                            Jennifer Paek1, Jacqueline Y. Lo2, Sri Devi Narasimhan3, Tammy N. Nguyen1, Kira Glover- <br />                            Cutter3, Stacey Robida-Stubbs3, Takafumi Suzuki4, Masayuki Yamamoto4, T. Keith <br />                            Blackwell3, Sean P. Curran1,2,5,* <br />                            1Leonard Davis School Gerontology, University Southern California, Los Angeles, CA <br />  <br />                            90089, USA <br />                            2College Letters, Arts, Science&#226;&#8364;&#8221;Department Molecular Computational Biology, <br />                            University Southern California, Los Angeles, CA 90089, USA <br />                            3Joslin <br />                                  Diabetes Center, Harvard Stem Cell Institute, Department Genetics, Harvard <br />                            Medical School, Boston, MA 02114, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Tohoku    University School Medicine, 2-1 Seiryo-machi, Aoba-ku, Sendai, Miyagi, 980-8575, <br />                            Japan <br />                            5KeckSchool Medicine&#226;&#8364;&#8221;Department Biochemistry Molecular Biology, University of <br />                            Southern California, Los Angeles, CA 90089, USA <br />  <br />                            SUMMARY <br />                                SKN-1/Nrf plays multiple essential roles development cellular homeostasis. We <br />                                demonstrate SKN-1 executes specific appropriate transcriptional response changes in <br />                                available nutrients, leading metabolic adaptation. We isolated gain-of-function (gf) alleles of <br />                                skn-1, affecting domain SKN-1 binds transcription factor MXL-3 the <br />                                mitochondrial outer membrane protein PGAM-5. These skn-1(gf) mutants perceive state of <br />                                starvation presence plentiful food. The aberrant monitoring cellular nutritional <br />                                status leads altered <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> response skn-1(gf) mutants transcriptionally activate <br />                                genes associated metabolism, adaptation starvation, aging, survival. The triggered <br />                                starvation response conserved mice constitutively activated Nrf contribute <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                tumorgenicity associated activating Nrf mutations mammalian somatic cells. Our <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                findings delineate evolutionarily conserved metabolic axis SKN-1/Nrf, establishing <br />                                complexity pathway. <br />  <br />  <br />                            INTRODUCTION <br />                                            Transcription factors influence multiple pathways choose variety targets <br />                                            ensure appropriate cellular response. SKN-1/Nrf (NF-E2-related factor) responds to <br />                                            environmental endogenous stressors including electrophiles, pathogens, xenobiotics, <br />                                            plays additional essential roles regulating development cellular homeostasis <br />                                            (Bowerman et al., 1992; Wang et al., 2010). In mammals, Kelch-like ECH-associated <br />                                            protein 1 (Keap1) negatively regulates Nrf sequestration cytoplasm and <br />  <br />  <br />                            &#194;&#169;2012 Elsevier Inc. <br />                            * <br />                              Correspondence: spcurran@usc.eduhttp://dx.doi.org/10.1016/j.cmet.2012.09.007. <br />                            SUPPLEMENTAL INFORMATION Supplemental Information includes figures, seven tables, Supplemental Experimental <br />                            Procedures article online http://dx.doi.org/10.1016/j.cmet.2012.09.007. <br />                             Paek et al.                                                                                             Page 2 <br />  <br />  <br />                                           presentation proteasome degradation (Itoh et al., 1999). An orthologous <br />                                           mechanism modulating SKN-1 activity orchestrated C. elegans WDR-23 (Choe et <br />                                           al., 2009). Mutations skn-1 wdr-23 shown influence cell organism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           survival C. elegans (Bishop Guarente, 2007; Curran Ruvkun, 2007; Tullet et al., <br />                                           2008; Wang et al., 2010). SKN-1 activate repress expression number of <br />                                           transcripts basal stress-activated conditions (An et al., 2005; Inoue et al., 2005; <br />                                           Bishop Guarente, 2007; Kell et al., 2007; Kahn et al., 2008; Tullet et al., 2008; Oliveira <br />                                           et al., 2009; Li et al., 2011). Although SKN-1 Nrf shown occupy consensus <br />                                           DNA elements promoters target genes, mechanism target selection by <br />                                           SKN-1/Nrf varying stress conditions understood (Blackwell et al., 1994; <br />                                           Moi et al., 1994; Oliveira et al., 2009) (Figure S1). <br />  <br />                                           The breadth cellular pathways influenced SKN-1 requires sophisticated mechanism <br />                                           regulating appropriate transcriptional response importantly restraining the <br />                                           activation unnecessary transcripts. In C. elegans SKN-1 essential embryonic <br />                                           development digestive required normal life span adaptation <br />                                           electrophile pathogen stress larvae adult animals (Bowerman et al., 1992; <br />                                           Bowerman et al., 1993; Maduro et al., 2001; An Blackwell, 2003; Maduro et al., 2007; <br />                                           Lin et al., 2009). Previous studies skn-1 utilize loss-of-function alleles activation of <br />                                           SKN-1 inhibition negative regulator wdr-23 (Choe et al., 2009; Hasegawa and <br />                                           Miwa, 2010). SKN-1 modulated insulin-like/IGF-1 pathway promotion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           longevity (Tullet et al., 2008). <br />  <br />                                           The ability organism balance energy demands nutritional supply paramount <br />                                           survival. Under conditions <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation organisms induce expression of <br />                                           starvation response genes facilitate adaptation change available nutrients a <br />                                           reliance energy stores (Lakowski Hekimi, 1998; Longo, 1999). Here the <br />                                           gain-of-function alleles skn-1, alter expression genes tied to <br />                                           metabolism, starvation adaptation, growth, reproduction. These mutations disrupt the <br />                                           association SKN-1 mitochondria outer-membrane protein, PGAM-5, <br />                                           independent canonical wdr-23 pathway. SKN-1 C. elegans ortholog of <br />                                           mammalian Nrf, constitutive activation Nrf induces similar starvation <br />                                           adaptation response mice. We conclude SKN-1/Nrf functions evolutionarily <br />                                           conserved pathway modulate metabolism adaptation starvation. <br />  <br />                               RESULTS <br />                               Identification Novel SKN-1 Regulatory Domain That Interacts PGAM-5 MXL-3 <br />                                           To identify novel regulators SKN-1 activation, mutagenized C. elegans harboring the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           well-established gst-4p::gfp reporter SKN-1 activity, hypothesizing be <br />                                           feasible identify genetic mutants disrupt specialized SKN-1 activation (Figure S2). In <br />                                           support idea, identified classes mutants range SKN-1 activity <br />                                           (our unpublished results). One class mutants includes alleles lax120 lax188 was <br />                                           dominant (Figure S2) mapped genetic position near center LG IV (Figure <br />                                           1A). An <span id='am-476' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-477' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-478' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-480' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-488' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-493' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-496' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-498' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-499' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-518' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-522' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-528' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-531' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-532' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-534' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-538' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-544' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-547' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-550' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-551' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-553' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) screen genes genomic region identified a <br />                                           single RNAi clone targeted T19E7.2/skn-1, suppress dominant gain-of- <br />                                           function <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Figure 1B). Sequencing skn-1 locus identified single missense <br />                                           <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> lax120 lax188, changing serine 245 leucine glutamic acid 237 to <br />                                           lysine, respectively. The similar phenotypes close proximity mutations <br />                                           suggested presence functional protein domain previously uncharacterized <br />                                           region SKN-1A/C protein (Figure 1C). <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                             Page 3 <br />  <br />  <br />                                          To determine region SKN-1 polypeptide facilitates interaction specific <br />                                          proteins, performed yeast-two-hybrid (Y2H) screen using bait construct consisting of <br />                                          200 amino acid region centered lax120 lax188 mutations (Figure 1C). We <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          identified 56 prey constructs initial screen sequenced plasmids identify <br />                                          potential protein interactors SKN-1 domain. To enhance specificity the <br />                                          interaction domain retested 56 plasmids directed Y2H assay utilizing refined <br />                                          bait construct containing 100 amino acids new SKN-1 domain. This second tier of <br />                                          screening eliminated 16 constructs failed retest. We performed final test the <br />                                          remaining 40 prey plasmids refined bait construct contained 50 <br />                                          amino acids SKN-1 domain did contain upstream DIDLID <br />                                          sequence (Walker et al., 2000). Bioinformatic analysis new 50 amino acid domain <br />                                          reveals casein kinase 2 domain, disrupted lax188, multiple potential <br />                                          phosphorylation sites, enrichment charged residues (Figure S2). <br />  <br />                                          Ten plasmids passed tiers screening. Four plasmids contained the <br />                                          pgam-5 locus, codes mitochondrial outer membrane protein. Two plasmids <br />                                          contained sequence <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-responsive transcription factor mxl-3; two <br />                                          plasmids coded conserved serine proteinase inhibitor K10D3.4; plasmid coded <br />                                          high-mobility group protein, HMG-1.1; plasmid coded protein with <br />                                          unknown function, D1025.1. The identification multiple independent isolates these <br />                                          genes combined fact plasmids facilitated strongest interaction all <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          skn-1 bait constructs suggests PGAM-5, MXL-3, K10D3.4 bona fide SKN-1 <br />                                          interactors (Figure S2). The ability SKN-1A SKN-1C bind MXL-3 PGAM-5 <br />                                          confirmed vitro coimmunoprecipitation studies utilizing engineered carboxyl- <br />                                          terminal HA epitope tag SKN-1 (Figure S2). These results indicate direct biochemical <br />                                          interaction MXL-3 PGAM-5 SKN-1. <br />  <br />                                          The dominant mutations skn-1 alter association SKN-1 protein with <br />                                          new interactors. We tested directly repeating Y2H assay 50 amino <br />                                          bait construct encoding lax188(E237K) lax120(S245L) mutation combination with <br />                                          isolated prey plasmids. Yeast harboring plasmids screened change in <br />                                          bait-prey interaction, measured X-&#206;&#177;-Gal activity resistance antibiotic <br />                                          aureobasidin A. The lax188 lax120 mutations decreased interaction SKN-1 <br />                                          domain PGAM-5 72% 75%, respectively (Figure 1D). Interestingly, both <br />                                          mutations resulted modest increase interaction MXL-3, HMG-1.1, and <br />                                          D1025.1. These findings suggest SKN-1 domain interacts PGAM-5, <br />                                          presumably mitochondria, MXL-3, help SKN-1 specify <br />                                          transcriptional targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          PGAM-5 predicted phosphoglycerate mutase uses alternate catalytic activity a <br />                                          protein serine/threonine phosphatase (Takeda et al., 2009). The interaction PGAM-5 at <br />                                          mitochondria optimize position SKN-1 function sensor of <br />                                          mitochondrial function, stress, energy output, cellular metabolic status. The serine 245 <br />                                          mutation lax120 combined fact PGAM-5 function serine/threonine <br />                                          phosphatase intriguing clue biochemical function SKN-1/ <br />                                          PGAM-5 interaction. Based data, lax120 lax188 mutations destabilize the <br />                                          association SKN-1 PGAM-5 decrease affinity SKN-1 biochemical <br />                                          substrate PGAM-5 catalytic activity (Figure S1). <br />  <br />                                          To identify domain interacting proteins facilitate binding SKN-1, we <br />                                          created new Y2H prey constructs coding truncated versions binding proteins and <br />                                          tested ability interact SKN-1 domain (Figure S2). The N-terminal domain of <br />                                          MXL-3 capable binding SKN-1 50-, 100-, 200-mer constructs. The N <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                             Page 4 <br />  <br />  <br />                                           terminus MXL-3 highly charged, facilitate interaction binding <br />                                           equally highly charged 50-mer domain SKN-1 electrostatic interactions. <br />                                           None truncated versions PGAM-5 revealed positive yeast-2-hybrid interaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SKN-1, potentially misfolding prey fusion protein. <br />  <br />                               Identification Mitochondria Associated SKN-1 <br />                                           A mitochondrial pool SKN-1 previously identified. To determine if <br />                                           endogenous SKN-1 associated organelle, isolated enriched <br />                                           mitochondrial fraction worm lysates differential centrifugation. We identified <br />                                           SKN-1-specific band predicted molecular weight SKN-1A isoform was <br />                                           enriched mitochondria subcellular fraction mitochondrial marker <br />                                           PDHE1 (Brys et al., 2010; Greiss et al., 2008; Li et al., 2009) (Figures 2A S3). While <br />                                           strength function SKN-1 mitochondrial surface unclear, steady- <br />                                           state presence SKN-1 isolated mitochondria suggest association is <br />                                           relatively stable. It notable skn-1(lax120) skn-1(lax188) gain-of-function (gf) <br />                                           mutants display Sma (small) body-size phenotype (Figure S2), trait shared many <br />                                           mitochondria mutants reduced ATP production (Lee et al., 2003). Taken together, these <br />                                           observations suggest pool SKN-1 (hereafter referred mito-SKN-1) present <br />                                           mitochondria, potentially facilitated interaction mitochondrial <br />                                           outer membrane protein PGAM-5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Regulation SKN-1(gf) Is Independent WDR-23 GSK-3 Pathways <br />                                           Our data indicate mitochondria-associated pool SKN-1 modulated a <br />                                           WDR-23-independent manner. WDR-23 molecularly well-defined regulator SKN-1 <br />                                           activity, did pull wdr-23 prey constructs Y2H screen using the <br />                                           new SKN-1 protein domain. In addition, directly tested wdr-23 prey vector the <br />                                           SKN-1 50-mer-bait construct detect measureable interaction (Figure S2). To <br />                                           confirm hypothesis 50-mer-bait regulated independently of <br />                                           WDR-23 measured activation gst-4p::gfp reporter skn-1(lax120) and <br />                                           skn-1(lax188) backgrounds wdr-23 (Figure 2D). In support two <br />                                           independent mechanisms SKN-1 regulation observed synergistic activation the <br />                                           SKN-1 target gene gst-4 skn-1(gf) alleles combined wdr-23 inactivation <br />                                           (Figure 2E). <br />  <br />                                           WDR-23 modulates activity abundance SKN-1 sequestering SKN-1 and <br />                                           delivering proteasome degradation. As verification model measured an <br />                                           increase expression wdr-23 mutant backgrounds (Table 1). The activation <br />                                           wdr-23 dependent skn-1, suggesting molecular feedback loop this <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           response pathway identifying wdr-23 major mechanism terminating SKN-1 <br />                                           activity regardless mode activation. <br />  <br />                                           GSK-3 shown inhibit localization SKN-1 nucleus subsequently <br />                                           repress activation SKN-1 targets (An et al., 2005). To determine skn-1(gf) <br />                                           mutants responsive GSK-3 regulation, treated skn-1(lax120) and <br />                                           skn-1(lax188) mutant animals gsk-3 RNAi. The animals fed gsk-3 RNAi clone had <br />                                           higher expression SKN-1 target gst-4 animals fed control RNAi clones (Figure <br />                                           S3). These data indicate SKN-1 gain-of-function mutants responsive the <br />                                           gsk-3 pathway suggesting mitochondrial pool SKN-1 activated mechanisms <br />                                           independent wdr-23 gsk-3. <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                            Page 5 <br />  <br />  <br />                               SKN-1(gf) Induces Expression Metabolism Starvation Response Genes <br />                                          Our model predicts activation mito-SKN-1 induce specific <br />                                          transcriptional response C. elegans. We examined transcriptional profile of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          synchronous skn-1(lax120) skn-1(lax188) animals compared age-matched wild-type <br />                                          controls (Figure 3A; Tables 1 S1). The functional GO terms highly represented in <br />                                          mito-SKN-1 activation mutants genes tied metabolism starvation response <br />                                          pathways followed genes involved growth, development, life span. Two <br />                                          metabolism-related genes upregulated skn-1 gain-of-function mutants previously <br />                                          identified upregulated SKN-1. Y42G9A.3 induced SKN-1 normal <br />                                          conditions zk742.4 induced SKN-1 response exposure arsenic t- <br />                                          BOOH. Of particular metabolism starvation adaption genes previously <br />                                          reported SKN-1 responsive following exposure environmental electrophile <br />                                          stress conditions. These metabolism genes include tyr-5/Tyrosinase, F09C8.1/ <br />                                          Phospholipase, sqrd-1/mitochondrial oxidoreductase, Y4C6B.6/glucosylceramidase, acdh-6/ <br />                                          acyl CoA dehydrogenase, lips-3/lipase-related, aat-5/amino acid transporter catalytic <br />                                          subunit, atgp-2/amino acid transporter glycoprotein subunit, aat-2/ amino acid transporter <br />                                          catalytic subunit, lips-10/lipase related. In addition, aging/dauer-related genes <br />                                          zk180.6/uncharacterized protein, R08E5.3/methyltrasnferase, wdr-23/SKN-1 negative <br />                                          regulator, previously identified SKN-1 targets, differentially <br />                                          expressed starvation conditions. The expression profile genes altered the <br />                                          direction wild-type worms subjected starvation (Kang Avery, 2009). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Taken together, results indicate worms harboring skn-1 gain-of-function <br />                                          alleles inappropriately perceive state starvation. We confirmed specificity of <br />                                          gain-of-function alleles WDR-23 pathways comparing abundance mito- <br />                                          SKN-1 target mRNAs activated SKN-1(gf) strains wdr-23 <br />                                          regulation. We identified synergistic change gene expression mito-SKN-1(gf)- <br />                                          dependent transcripts depleted wdr-23 RNAi (Table S2). Additivity was <br />                                          observed transcripts induced repressed mito-SKN-1 activation indicating the <br />                                          existence parallel pathways activate SKN-1. <br />  <br />                                          The altered expression metabolism <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-mobilization genes notable skn-1 <br />                                          loss-of-function mutants shown suppress longevity phenotype associated <br />                                          dietary restriction (Bishop Guarente, 2007). The genes changed most <br />                                          significantly direction change reminiscent sensed state starvation or <br />                                          severe dietary restriction (Szewczyk et al., 2006; Baugh et al., 2009). Although the <br />                                          transcriptional analysis mito-SKN-1 mutants representative entire organism <br />                                          tissue specific, observed strongest mito-SKN-1 activation hypodermal <br />                                          intestinal tissues (Figure 1B), collectively comprise significant portion the <br />                                          total worm body. To test starvation capable inducing SKN-1 transcriptional <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          response compared expression gst-4p::gfp reporter SKN-1 activity the <br />                                          presence absence food. Consistent activation observed skn-1(gf) mutants <br />                                          detected GFP expression hypodermal tissues animals removed <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                          food (Figures 3B 3C). The intestine hypodermis major sites lipid <br />                                          storage C. elegans, explain observed changes metabolism and <br />                                          starvation response transcripts. <br />  <br />                                          Recent work shown bacterial diet itself strong impact the <br />                                          physiology worm, cases reverse suppress genetic phenotypes <br />                                          (Soukas et al., 2009; Maier et al., 2010). The HT115 RNAi strain E. coli K12 <br />                                          background standard C. elegans food source, OP50, E.coli B genotype. <br />                                          The genetic differences K12 B genotypes alter nutritional <br />                                          composition food source &#226;&#8364;&#339;taste&#226;&#8364;? food worm measured by <br />                                          occupancy lawn bacteria (Watts Browse, 2002; Shtonda Avery, 2006; <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                               Page 6 <br />  <br />  <br />                                           Soukas et al., 2009; Watts, 2009). Worms HT115 diet spend time bacterial <br />                                           lawn experience oscillating periods caloric restriction. To test E.coli B <br />                                           K12 diets influence SKN-1 activity grew gst-4p::gfp sensor strain both <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           strains. On HT115 diet observed GFP expression intestinal cells normally <br />                                           observed animals fed OP50 diet (Figures 3D&#226;&#8364;&#8220;3I). <br />  <br />                                           Since diet strong impact physiology, examined expression mito- <br />                                           SKN-1 transcripts animals fed OP50/E.coli B compared animals raised a <br />                                           HT115/E.coli K12 food source. We identified mito-SKN-1 transcripts were <br />                                           differentially expressed fed E.coli K12 diet. Notably, identified switch from <br />                                           transcriptional repression OP50 activation HT115 genes involved aging and <br />                                           metabolic pathways (Table S2). These changes obvious aging/dauer- <br />                                           related genes dao-4/novel protein, ZK180.6/uncharacterized protein, R05G9R.1/ <br />                                           mucin-2-like, metabolism/signaling-related genes mxl-3/Max-like <br />                                           transcription factor, acdh-6/acyl CoA dehydrogenase, lips-3/lipase related (Table S2). <br />  <br />                                           The physical interaction SKN-1 MXL-3 help specify transcription of <br />                                           metabolism starvation genes skn-1(gf) background. To test combined <br />                                           skn-1(gf) alleles loss-of-function allele mxl-3(ok1947) examined transcript <br />                                           levels genes altered gain-of-function background (Figure 1, Table S3). The <br />                                           genes tied metabolism aging/dauer regulatory pathways affected the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           absence mxl-3. The mxl-3 <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>null allele</span> partially suppressed increased expression of <br />                                           acdh-6/acyl CoA dehydrogenase, lips-10/lipase-related, R01E6.5/Filaggrin-2-like, dao-4/ <br />                                           novel protein, dod-24/downstream DAF-16, dod-17/downstream DAF-16, ugt-11/ <br />                                           UDP-Glucuronosyl Transferase, C32H11.4/hydrolase, ins-35/insulin peptide genes. The <br />                                           mxl-3 deletion reversed decreased expression F09C8.1/Phospholipase and <br />                                           R08E5.3/methyltransferase skn-1(gf) background. Intriguingly, skn-1;mxl-3 <br />                                           double mutants synergized increased expression wdr-23, indicating concerted effort <br />                                           suppress gain-of-function activity SKN-1. The dysregulated activation SKN-1 <br />                                           results transcriptional profile indicative perceived state starvation. The change in <br />                                           expression metabolism genes SKN-1(gf) mutants dependence mxl-3 <br />                                           introduces new regulatory axis organismal <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> mobilization maintain energy <br />                                           homeostasis. <br />  <br />                               A Perceived State Starvation Alters Metabolic Adaptation Capacity skn-1(gf) Mutants <br />                                           The presence specific independent mechanisms SKN-1 activation predicts that <br />                                           unique physiological role pathways. To identify specific <br />                                           function mito-SKN-1 characterized mito-SKN-1(gf) mutants reproduction and <br />                                           life-span phenotypes challenged ability adapt physiologic stress conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The skn-1 signaling pathway functions multiple longevity regulatory cascades; <br />                                           however, clear gain-of-function mutants aging phenotype <br />                                           based altered expression pro- anti-aging genes (Figure 3A, Table 1). We <br />                                           assessed effect SKN-1 activating mutations adult lifespan (Figures 4A S4). <br />                                           Neither allele skn-1 significant effect rate aging compared wild-type <br />                                           animals. <br />  <br />                                           Food availability strong influence life span. As such, tested ability of <br />                                           mutants respond dietary restriction (Figures 4A S4). We examined life span <br />                                           skn-1(lax120) skn-1(lax188) mutants range bacteria concentrations. <br />                                           skn-1(lax188) failed display lifespan extension phenotype bacterial dilution (bDR), <br />                                           skn-1(lax120) mutant animals showed attenuated response. These results are <br />                                           consistent diminished L1 survival phenotype (Figure S5) skn-1(lax188) <br />                                           mutant support idea animals perceive reduction food supply <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                           Page 7 <br />  <br />  <br />                                          ad libitum conditions leading extreme starvation response bDR <br />                                          growth conditions. An alternative mechanism caloric restriction C. elegans results from <br />                                          genetic mutations decrease pharyngeal pumping reduce ingestion E. coli <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (Lakowski Hekimi, 1998). As such, measured pharyngeal pumping, feeding rates, <br />                                          bacterial clearance skn-1(gf) mutants statistically significant <br />                                          difference wild-type (Figures 4B S4, Table S4). <br />  <br />                                          skn-1 essential development skn-1(z-m-) embryos viable (Bowerman et al., <br />                                          1992). However, lax120 lax188 mutations facilitate extended reproductive <br />                                          period hermaphrodites approximately 2 days (Figure 5A). Intriguingly, observed <br />                                          suppression reproductive senescence does come expense reduced brood size <br />                                          worms mutation yield similar number viable progeny compared to <br />                                          wild-type animals (Figure 5B). We did notice low penetrance (less 10%) bag <br />                                          worms (Bag) phenotype, progeny hatch parent higher temperatures <br />                                          (unpublished observation) (Lakowski Hekimi, 1998). C. elegans require demanding <br />                                          supply nutrients reach reproductive maturity. If removed food, <br />                                          hermaphrodites cease egg laying, allowing progeny hatch utero leading Bag <br />                                          phenotype (Dong et al., 2000). The low penetrance Bag phenotype mito-SKN-1(gf) <br />                                          mutants consistent idea animals experience perceived state of <br />                                          starvation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The transcriptional changes skn-1(gf) mutants reminiscent animal coping <br />                                          nutrient deprivation conditions. C. elegans survive extended periods starvation <br />                                          stages diapause&#226;&#8364;&#8221; larval stage 1 (L1) dauers, <br />                                          alternative larval stage (L3). Since skn-1(gf) worms demonstrated <br />                                          transcriptional changes associated metabolic profile similar response to <br />                                          starvation, directly tested ability mutants survive food for <br />                                          extended periods time. We performed L1 survival assays previously reported (Baugh et <br />                                          al., 2009) assessed ability skn-1(lax120), skn-1(lax188), skn-1- <br />                                          overexpressing skn-1b/c::gfp worms withstand starvation 5, 7, 12 days room <br />                                          temperature (~21&#194;&#176;C) (Figures 5C S5, Table S5). At 5 days, clear difference <br />                                          survival skn-1(gf) worms compared wild-type skn-1b/c::gfp worms. <br />                                          However, increasing &#226;&#8364;&#339;age,&#226;&#8364;? skn-1b/c::gfp worms start exhibit significantly <br />                                          increased survival wild-type worms. Intriguingly, stronger skn-1(gf) allele, <br />                                          skn-1(lax188), modest increase day 7, day 12 dramatically decreased <br />                                          survival. In contrast, skn-1(lax120) mutants somewhat comparable wild-type <br />                                          worms assay. Our results suggest given skn-1(lax188) animals already <br />                                          experiencing starvation-like response, prolonged absence food detrimental their <br />                                          long-term survival. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Worms constantly sensing environment nutritional cues, cultured on <br />                                          regular growth media, crawl bacterial lawn feed. Indeed, L1-arrested <br />                                          animals typically recover diapause resume reproductive growth transfer <br />                                          regular growth media (Figure S5, dark blue bars). We assessed ability the <br />                                          &#226;&#8364;&#339;starving&#226;&#8364;? skn-1(gf) mutants recover 8 days starvation regular media (Figures <br />                                          5D S5; Table S5). As shown Figure 5E, 2 days incubation 20 degrees, the <br />                                          majority wild-type worms (&gt;80%) grow gravid adults. In contrast, skn-1(gf) <br />                                          worms slower growth profile. In particular, small fraction skn-1(gf) <br />                                          worms arrested L1s, rest population demonstrating asynchronous <br />                                          growth primarily comprising L2 L3 stage animals. Despite obvious lag in <br />                                          growth, worms eventually develop fertile adults (data shown). The skn-1 <br />                                          transgenic worms markedly reduced growth rate able form gravid <br />                                          adults (~30% adults) 2 days. We conclude presence food, these <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                              Page 8 <br />  <br />  <br />                                           skn-1(gf) animals behave undergoing constant state starvation. In support <br />                                           this, upregulation metabolic transcripts, inability survive <br />                                           extended starvation, slow recovery time despite plentiful levels food. Additionally, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           data indicate gf alleles behave similarly skn-1 overexpressing <br />                                           strain, suggesting increased abundance SKN-1 does necessarily correlate with <br />                                           increased function/activity. Thus stressful survival conditions, food <br />                                           resources limited, SKN-1 activation provide protective role ensuring the <br />                                           animal cope extended periods starvation. <br />  <br />                               Constitutive Activation Mammalian Nrf Induces Starvation Response <br />                                           Cellular adaptation starvation conserved stress response (Hardie, 2011). In light the <br />                                           evolutionary conservation SKN-1-like cap-n-collar transcription factors, were <br />                                           curious Nrf activation mediate starvation-like response mammals. To <br />                                           determine Nrf activation leads perceived state starvation, measured the <br />                                           transcriptional profile tissue established Keap1&#226;&#710;&#8217;/&#226;&#710;&#8217; <span id='am-398' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-399' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>knockout</span> mouse line (Ohta et <br />                                           al., 2008) (Tables 2 S6). Despite fact tissue display dysregulation of <br />                                           transcripts Keap1-Nrf dependent pathways, curious detect any <br />                                           alteration starvation-response genes. Genes tied metabolism GO terms were <br />                                           significantly enriched Keap1&#226;&#710;&#8217;/&#226;&#710;&#8217; tissues, similar C. elegans (Table S7). <br />                                           In support SKN-1/Nrf function maintaining metabolic homeostasis, discovered <br />                                           genes associated starvation responses significantly upregulated Keap1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           knockout mice compared wild-type litter mates (Table 2). We confirmed the <br />                                           transcriptional changes identified microarray data additional qRT-PCR analysis. A <br />                                           hallmark state starvation activation energy sensors AMP-activated <br />                                           protein kinase adenylate kinase (Hardie, 2011). The catalytic domain AMPK, Prkaa2 <br />                                           Ak7, upregulated lung tissue Keap1 knockout mice 2.07- and <br />                                           3.04-fold, respectively. Importantly, levels cAMP-dependent protein kinase subunits, <br />                                           opposite effects energy homeostasis, remained unchanged. Of additional <br />                                           upregulation genes involved mitochondrial peroxisomal fatty acid <br />                                           oxidation, lipases, lipid transporters (Table 2). These genes shown change <br />                                           direction mouse liver periods starvation (Bauer et al., 2004). <br />                                           Remarkably, activated Nrf lung tissue 300-fold increase Ces1g. <br />                                           Overexpression mouse esterase-x/carboxylesterase 1 (Ces1g) shown promote <br />                                           beta-oxidation (Ko et al., 2009). Two C. elegans Ces1g-like orthologs coded operon <br />                                           T28C12.4 T28C12.5 significantly upregulated skn-1(lax120) skn-1 <br />                                           (lax188) mutant backgrounds (Table S1). These results suggest constitutively <br />                                           activated Nrf lung tissue Keap1 knockout mice induces perceived state of <br />                                           starvation, similar observed intestine hypodermis C. elegans mito- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SKN-1(gf) mutants. As such, addition canonical electrophile stress responses, the <br />                                           SKN-1/Nrf functions evolutionarily conserved pathway modulate <br />                                           cellular metabolism. <br />  <br />                               DISCUSSION <br />                                           Transcription factors like SKN-1/Nrf, involved multiple aspects cell biology, <br />                                           capable initiating appropriate transcriptional response activated a <br />                                           specific cellular condition. This specification accomplished posttranslational <br />                                           modification, changing association transcriptional activators repressors, <br />                                           sequestration away nucleus, and/or degradation transcription factor (Wang et <br />                                           al., 2006; Wolff Dillin, 2006; Villeneuve et al., 2010). The SKN-1/Nrf family of <br />                                           transcription factors well-established regulator cellular response electrophiles <br />                                           pathogens. By characterizing skn-1(gf) alleles, identified novel <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                              Page 9 <br />  <br />  <br />                                          function SKN-1 regulation metabolism adaptation starvation. This new <br />                                          function mediated pool SKN-1 sequestered the <br />                                          mitochondrial presumably outer membrane. The idea distinct organelle population <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          SKN-1 adapted respond specific changes cellular environment new <br />                                          exciting way thinking mechanism organism utilizes directing an <br />                                          appropriate transcriptional response. <br />  <br />                                          Despite functional conservation Nrf SKN-1, clear KEAP1 ortholog C. <br />                                          elegans identified. Our results suggest despite little sequence identity <br />                                          KEAP1 WDR-23, proteins function evolutionarily conserved <br />                                          manner negatively regulate SKN-1 activity formation mitochondrial <br />                                          SKN-1 complex. Although data support model SKN-1 interact with <br />                                          PGAM-5 independently WDR-23, interaction Nrf PGAM5 dependent on <br />                                          KEAP1. These results intriguing evolutionary standpoint imply the <br />                                          mechanism recruiting SKN-1/Nrf evolved twice, clearly demonstrating importance of <br />                                          mitochondrial localized SKN-1 population. WDR-23, however, capable of <br />                                          binding PGAM-5 MXL-3, suggesting aspects coordinated control two <br />                                          pathways (Figure S1). Despite mechanistic difference, increased activity of <br />                                          mitochondrial SKN-1 gain-of-function mutants activated Nrf KEAP1 <br />                                          knockout mutant results perceived state starvation consequently activation of <br />                                          metabolism starvation-response pathways. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          What facilitates interaction SKN-1 mitochondria? Our data suggest the <br />                                          SKN-1 gain-of-function mutants reduced interaction PGAM-5. If PGAM-5 is <br />                                          anchor SKN-1 mitochondria reduced interaction promotes <br />                                          activation. An alternative hypothesis SKN-1 anchored mitochondrial <br />                                          outermembrane predicted transmembrane domain N terminus the <br />                                          SKN-1A isoform. Future studies directed uncover mechanisms coordinate <br />                                          SKN-1 association mitochondria subsequent activation/dissociation. <br />  <br />                                          The aberrant regulation SKN-1 gain-of-function mutants independent the <br />                                          glycogen synthase kinase pathway, GSK-3 (An et al., 2005). SKN-1 shown be <br />                                          regulated GSK-3 phosphorylation event occurs serine residues 300&#226;&#8364;&#8220;400 <br />                                          amino acid region SKN-1. lax120 mutation serine 245 SKN-1A protein. <br />                                          This serine residue poor GSK-3 target lacks required priming serine <br />                                          residue GSK-3 function (An et al., 2005). <br />  <br />                                          We identified new SKN-1-interacting proteins refine appropriate SKN-1 <br />                                          transcriptional response. The bZip transcription factor MXL-3 partners SKN-1 the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          specification response starvation. The new highly charged 50 amino acid SKN-1 <br />                                          domain identified responsible binding MXL-3 facilitates an <br />                                          interaction mitochondrial phosphatase PGAM-5 serine proteinase inhibitor <br />                                          K10D3.4. The expression pgam-5 mxl-3 clearly expressed worm nervous <br />                                          pharyngeal tissues (Figure S2). The behavioral response limited nutrients and <br />                                          starvation controlled well-established neuroendocrine circuit (Avery et al., 1993; <br />                                          Baumeister et al., 2006; Bishop Guarente, 2007). Our findings incorporate mito-SKN-1, <br />                                          PGAM-5, MXL-3 behavioral circuit. We tested mechanistic role of <br />                                          mxl-3 expression transcriptional targets altered skn-1 gain-of-function <br />                                          mutants. In support metabolic specificity pathway, expression <br />                                          metabolism transcripts altered SKN-1 influenced MXL-3. Recent <br />                                          examination SKN-1 function normal conditions identifies different overlapping <br />                                          sets genes regulated response different stresses (Oliveira et al., 2009). <br />                                          MXL-3 provide paradigm SKN-1 interact combinatorially other <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                 Page 10 <br />  <br />  <br />                                            transcription factors regulate different subsets genes conditions variable <br />                                            availability nutrients. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Although gain-of-function alleles lead perceived state starvation similar <br />                                            transcriptional profiles, molecular physiological differences two <br />                                            alleles. We identified allele-specific changes effect E. coli K12 diet of <br />                                            wdr-23 inactivation mito-SKN-1-dependent transcripts (signaling: ins-35, kin-15, <br />                                            kin-16; metabolism: ZK742.4, lips-10, aat-2; aging/dauer: ZK180.6 R05G9R.1) <br />                                            (Table S2). These allele-specific changes particular based observation <br />                                            skn-1(lax120) mutant animals remain partially responsive dietary restriction <br />                                            skn-1(lax188) mutants not. These results specify unique properties the <br />                                            different mito-SKN-1(gf) mutants substantiate idea mito-SKN-1, in <br />                                            concert MXL-3, influence metabolism organismal response changes in <br />                                            available nutrients. <br />  <br />                                            Activating Nrf mutations identified numerous somatic tumors, especially <br />                                            lung cell carcinomas (Ohta et al., 2008; Shibata et al., 2008). These tumors tend be <br />                                            extremely resistant chemical radiation-based therapies. It shown that <br />                                            resistance phenotype partially dependent activation Nrf-dependent transcripts <br />                                            (Singh et al., 2008). Hyperproliferative tumor cells rapidly exhaust available energy <br />                                            supplies forcing necessity adapt nutritionally limiting environment (Jang et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2011). The lung canonically thought metabolic tissue does a <br />                                            major contribution maintaining energy homeostasis organism. The lung is, <br />                                            however, exceptionally active metabolically, fabricating lipids form of <br />                                            surfactants rich phospholipids sterols. The activation mitochondrial lipid oxidation <br />                                            provide mechanism ensure adequate ATP levels continued cancer growth prior <br />                                            new vasculature provided angiogenesis. Nrf identified potential target for <br />                                            cancer treatment suppress cellular stress adaption mechanisms. Our results suggest an <br />                                            understanding different specific mechanisms Nrf activation likely important <br />                                            elucidating effects <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>cell survival</span> carcinogenesis (Mitsuishi et al., 2012). Further <br />                                            characterization role mito-SKN-1/Nrf provide insights physiologic <br />                                            roles mitochondrial-specific population essential biological regulator may <br />                                            lead novel strategies combatting cancers resistant traditional therapies. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               Standard Growth Conditions <br />                                            C. elegans raised standard 6 cm nematode growth media plates supplemented with <br />                                            streptomycin seeded Escherichia coli OP50. For RNAi experiments NGM plates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            containing 5 mM IPTG 100 ug ml&#226;&#710;&#8217;1 carbencillin seeded overnight cultures of <br />                                            double-stranded RNAi-expressing HT115 bacteria. Plates allowed induce overnight <br />                                            followed transfer age-synchronous populations C. elegans. <br />  <br />                               Strains Used This Study <br />                                            Strains used study N2 Bristol (wild-type), CB4856 (Hw), CL2166 <br />                                            [gst-4p::gfp]III, SPC167[skn-1(lax120)IV;gst-4p::gfp], SPC168[skn-1(lax188) <br />                                            IV;gst-4p::gfp], SPC207[skn-1(lax120)IV], SPC227[skn-1(lax188)IV]. <br />  <br />                               Statistics <br />                                            Statistical analyses performed JMP 8 software. <br />  <br />  <br />  <br />  <br />                                                 Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                            Page 11 <br />  <br />  <br />                               Isolation Dominant SKN-1 Activation Mutants <br />                                             A C. elegans strain harboring SKN-1 transcriptional reporter gst-4p::gfp was <br />                                             mutagenized Ethyl methanesulfonate (EMS). F1 generation worms increased GFP <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             expression, indicating SKN-1 activation, isolated characterization. F2 <br />                                             generation animals singled F3 progeny analyzed 100% penetrance the <br />                                             activated GFP phenotype. These homozygous animals mated the <br />                                             unmutagenized parental strain. Successful matings yielded 100% F1 progeny expressing <br />                                             activated GFP phenotype confirmed dominant mutants. <br />  <br />                               Yeast-2-Hybrid <br />                                             Bait prey plasmids generated cloning 50-mer, 100-mer, 200-mer <br />                                             oligonucleotides skn-1(lax120) skn-1(lax188) mutations pLexA C. <br />                                             elegans cDNA library pACT2.2 (Addgene), respectively. Interaction tested on <br />                                             synthetic complete agar lacked leucine tryptophan supplemented X-&#206;&#177;- <br />                                             gal. Interactors identified transforming Y2HGold (Clontech) bait (pLexA- <br />                                             SKN-1) cDNA prey library. Positive clones grown absence of <br />                                             tryptophan sequenced. Sequenced clones retested individually. <br />  <br />                               Microarrays <br />                                             RNA processed using Affymetrix GeneChip 3&#226;&#8364;&#178; IVT express Kit. Mouse samples <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             hybridized Affymetrix MouseGene 1.0 ST array. Five knockout mice three <br />                                             wild-type littermates used studies. C. elegans samples hybridized to <br />                                             Affymetrix GeneChip C. elegans Genome Arrays. Three biologically unique wild-type <br />                                             samples compared biological replicates skn-1(lax120) biological <br />                                             replicates skn-1(lax188) utilized study. <br />  <br />                               L1 Arrest, Survival, Recovery Assays <br />                                             Following hypochlorite treatment, eggs seeded 24 plates (4 wells strain) <br />                                             S-Basal + Cholesterol. For scoring viability, drop spotted <br />                                             coverslip. Movement used parameter score viability. Bromophenol blue <br />                                             staining used confirm dead worms. Animals scored 5, 7, 12 days. At day 8 <br />                                             cohort animals placed OP50 starvation recovery assays. Animals were <br />                                             scored developmental progress wild-type animals reached reproductive <br />                                             maturity. <br />  <br />                               Life-Span Analysis <br />                                             Life-span experiments performed previously described (Curran Ruvkun, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             2007). The strains egg prepped generate synchronous populations, 20&#226;&#8364;&#8220;30 L4 <br />                                             animals placed NGM plates seeded OP50 containing 50 &#206;&#188;g ml&#226;&#710;&#8217;1 5- <br />                                             fluoro-2&#226;&#8364;&#178;-deoxyuridine (FUdR) prevent progeny hatching. The worms kept 20&#194;&#176;C <br />                                             scored days survival gentle prodding platinum wire. Dietary <br />                                             restriction lifespan experiments performed 25&#194;&#176;C plates containing ampicillin to <br />                                             inhibit bacterial growth. Worms transferred daily fresh plates same <br />                                             bacterial concentration. <br />  <br />                                             Additional methods Supplemental Information. <br />  <br />                               Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />  <br />                                                  Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                            Page 12 <br />  <br />  <br />                               Acknowledgments <br />                                          We thank Chandra Tucker, Duke University C. elegans Y2H library (Developed Maureen Barr&#226;&#8364;&#8482;s <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          laboratory). We grateful Daniel Wai, Children&#226;&#8364;&#8482;s Hospital Los Angeles analysis Affymetrix microarray <br />                                          data. We thank Drs. Caleb Finch, Alexander Soukas, Shanshan Pang critically reading manuscript <br />                                          members Curran laboratory technical support fruitful discussions. S.P.C. conceived and <br />                                          designed study. T.K.B. M.Y. collaborated SPC plan organize experiments. J.P., J.Y.L., S.D.N., <br />                                          T.N.N., K.G.C., S.R.S., T.S., S.P.C. conducted experiments. Some strains used study were <br />                                          obtained Caenorhabditis Genetics Center (CGC), supported National Institutes Health&#226;&#8364;&#8221; <br />                                          NCRR. Research study supported NIH R01GM62891 (T.K.B.) R00AG032308 (S.P.C.). S.P.C. is <br />                                          Ellison Medical Foundation Junior Scholar Aging. <br />  <br />  <br />                               REFERENCES <br />                                          An JH, Blackwell TK. SKN-1 links C. elegans mesendodermal specification conserved oxidative <br />                                            stress response. Genes Dev. 2003; 17:1882&#226;&#8364;&#8220;1893. [PubMed: 12869585] <br />                                          An JH, Vranas K, Lucke M, Inoue H, Hisamoto N, Matsumoto K, Blackwell TK. Regulation the <br />                                            Caenorhabditis elegans oxidative stress defense protein SKN-1 glycogen synthase kinase-3. <br />                                            Proc. Natl. Acad. Sci. USA. 2005; 102:16275&#226;&#8364;&#8220;16280. [PubMed: 16251270] <br />                                          Avery L, Bargmann CI, Horvitz HR. The Caenorhabditis elegans unc-31 gene affects multiple nervous <br />                                            system-controlled functions. Genetics. 1993; 134:455&#226;&#8364;&#8220;464. [PubMed: 8325482] <br />                                          Bauer M, Hamm AC, Bonaus M, Jacob A, Jaekel J, Schorle H, Pankratz MJ, Katzenberger JD. <br />                                            Starvation response mouse liver shows strong correlation life-span-prolonging processes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Physiol. Genomics. 2004; 17:230&#226;&#8364;&#8220;244. [PubMed: 14762175] <br />                                          Baugh LR, Demodena J, Sternberg PW. RNA Pol II accumulates promoters growth genes during <br />                                            developmental arrest. Science. 2009; 324:92&#226;&#8364;&#8220;94. [PubMed: 19251593] <br />                                          Baumeister R, Schaffitzel E, Hertweck M. Endocrine signaling Caenorhabditis elegans controls <br />                                            stress response longevity. J. Endocrinol. 2006; 190:191&#226;&#8364;&#8220;202. [PubMed: 16899554] <br />                                          Bishop NA, Guarente L. Two neurons mediate diet-restriction-induced longevity C. elegans. Nature. <br />                                            2007; 447:545&#226;&#8364;&#8220;549. [PubMed: 17538612] <br />                                          Blackwell TK, Bowerman B, Priess JR, Weintraub H. Formation monomeric DNA binding <br />                                            domain Skn-1 bZIP homeodomain elements. Science. 1994; 266:621&#226;&#8364;&#8220;628. [PubMed: <br />                                            7939715] <br />                                          Bowerman B, Draper BW, Mello CC, Priess JR. The maternal gene skn-1 encodes protein is <br />                                            distributed unequally early C. elegans embryos. Cell. 1993; 74:443&#226;&#8364;&#8220;452. [PubMed: 8348611] <br />                                          Bowerman B, Eaton BA, Priess JR. skn-1, maternally expressed gene required specify fate of <br />                                              ventral blastomeres early. C. elegans embryo. Cell. 1992; 68:1061&#226;&#8364;&#8220;1075. <br />                                          Brys K, Castelein N, Matthijssens F, Vanfleteren JR, Braeckman BP. Disruption insulin signalling <br />                                              preserves bioenergetic competence mitochondria ageing Caenorhabditis elegans. BMC Biol. <br />                                              2010; 8:91. [PubMed: 20584279] <br />                                          Choe KP, Przybysz AJ, Strange K. The WD40 repeat protein WDR-23 functions CUL4/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              DDB1 ubiquitin ligase regulate nuclear abundance activity SKN-1 Caenorhabditis <br />                                              elegans. Mol. Cell. Biol. 2009; 29:2704&#226;&#8364;&#8220;2715. [PubMed: 19273594] <br />                                          Curran SP, Ruvkun G. Lifespan regulation evolutionarily conserved genes essential viability. <br />                                              PLoS Genet. 2007; 3:e56. [PubMed: 17411345] <br />                                          Dong MQ, Chase D, Patikoglou GA, Koelle MR. Multiple RGS proteins alter neural G protein <br />                                              signaling allow. C. elegans rapidly change behavior fed. Genes Dev. 2000; 14:2003&#226;&#8364;&#8220; <br />                                              2014. <br />                                          Greiss S, Hall J, Ahmed S, Gartner A. C. elegans SIR-2.1 trans-location linked proapoptotic <br />                                              pathway parallel cep-1/p53 DNA damage-induced apoptosis. Genes Dev. 2008; <br />                                              22:2831&#226;&#8364;&#8220;2842. [PubMed: 18923081] <br />                                          Hardie DG. Sensing energy nutrients AMP-activated protein kinase. Am. J. Clin. Nutr. 2011; <br />                                              93:891S&#226;&#8364;&#8220;6. [PubMed: 21325438] <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                    Page 13 <br />  <br />  <br />                                          Hasegawa K, Miwa J. Genetic cellular characterization Caenorhabditis elegans mutants <br />                                               abnormal regulation phase II enzymes. PLoS ONE. 2010; 5:e11194. [PubMed: <br />                                               20585349] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Inoue H, Hisamoto N, An JH, Oliveira RP, Nishida E, Blackwell TK, Matsumoto K. The C. elegans <br />                                               p38 MAPK pathway regulates nuclear localization transcription factor SKN-1 oxidative <br />                                               stress response. Genes Dev. 2005; 19:2278&#226;&#8364;&#8220;2283. [PubMed: 16166371] <br />                                          Itoh K, Wakabayashi N, Katoh Y, Ishii T, Igarashi K, Engel JD, Yamamoto M. Keap1 represses <br />                                               nuclear activation antioxidant responsive elements Nrf2 binding amino- <br />                                               terminal Neh2 domain. Genes Dev. 1999; 13:76&#226;&#8364;&#8220;86. [PubMed: 9887101] <br />                                          Jang T, Calaoagan JM, Kwon E, Samuelsson S, Recht L, Laderoute KR. 5&#226;&#8364;&#178;-AMP-activated protein <br />                                               kinase activity elevated early primary brain tumor development rat. Int. J. Cancer. <br />                                               2011; 128:2230&#226;&#8364;&#8220;2239. [PubMed: 20635388] <br />                                          Kahn NW, Rea SL, Moyle S, Kell A, Johnson TE. Proteasomal dysfunction activates transcription <br />                                               factor SKN-1 produces selective oxidative-stress response Caenorhabditis elegans. <br />                                               Biochem. J. 2008; 409:205&#226;&#8364;&#8220;213. [PubMed: 17714076] <br />                                          Kang C, Avery L. Systemic regulation starvation response Caenorhabditis elegans. Genes Dev. <br />                                               2009; 23:12&#226;&#8364;&#8220;17. [PubMed: 19136622] <br />                                          Kell A, Ventura N, Kahn N, Johnson TE. Activation SKN-1 novel kinases Caenorhabditis <br />                                               elegans. Free Radic. Biol. Med. 2007; 43:1560&#226;&#8364;&#8220;1566. [PubMed: 17964427] <br />                                          Ko KW, Erickson B, Lehner R. Es-x/Ces1 prevents triacylglycerol accumulation McArdle-RH7777 <br />                                               hepatocytes. Biochim. Biophys. Acta. 2009; 1791:1133&#226;&#8364;&#8220;1143. [PubMed: 19651238] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Lakowski B, Hekimi S. The genetics caloric restriction Caenorhabditis elegans. Proc. Natl. Acad. <br />                                               Sci. USA. 1998; 95:13091&#226;&#8364;&#8220;13096. [PubMed: 9789046] <br />                                          Lee SS, Lee RY, Fraser AG, Kamath RS, Ahringer J, Ruvkun G. A systematic RNAi screen identifies <br />                                               critical role mitochondria C. elegans longevity. Nat. Genet. 2003; 33:40&#226;&#8364;&#8220;48. [PubMed: <br />                                               12447374] <br />                                          Li J, Cai T, Wu P, Cui Z, Chen X, Hou J, Xie Z, Xue P, Shi L, Liu P, et al. Proteomic analysis of <br />                                               mitochondria Caenorhabditis elegans. Proteomics. 2009; 9:4539&#226;&#8364;&#8220;4553. [PubMed: 19670372] <br />                                          Li X, Matilainen O, Jin C, Glover-Cutter KM, Holmberg CI, Blackwell TK. Specific SKN-1/Nrf stress <br />                                               responses perturbations translation elongation proteasome activity. PLoS Genet. 2011; <br />                                               7:e1002119. [PubMed: 21695230] <br />                                          Lin KT, Broitman-Maduro G, Hung WW, Cervantes S, Maduro MF. Knockdown SKN-1 the <br />                                               Wnt effector TCF/POP-1 reveals differences endomesoderm specification C. briggsae as <br />                                               compared C. elegans. Dev. Biol. 2009; 325:296&#226;&#8364;&#8220;306. [PubMed: 18977344] <br />                                          Longo VD. Mutations signal transduction proteins increase stress resistance longevity yeast, <br />                                               nematodes, fruit flies, mammalian neuronal cells. Neurobiol. Aging. 1999; 20:479&#226;&#8364;&#8220;486. <br />                                               [PubMed: 10638521] <br />                                          Maduro MF, Broitman-Maduro G, Mengarelli I, Rothman JH. Maternal deployment embryonic <br />                                               SKN-1&#226;&#8364;&#8221;&gt;MED-1,2 cell specification pathway C. elegans. Dev. Biol. 2007; 301:590&#226;&#8364;&#8220;601. <br />                                               [PubMed: 16979152] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Maduro MF, Meneghini MD, Bowerman B, Broitman-Maduro G, Rothman JH. Restriction of <br />                                               mesendoderm single blastomere combined action SKN-1 GSK-3beta homolog <br />                                               mediated MED-1 -2 C. elegans. Mol. Cell. 2001; 7:475&#226;&#8364;&#8220;485. [PubMed: 11463373] <br />                                          Maier W, Adilov B, Regenass M, Alcedo J. A neuromedin U receptor acts sensory to <br />                                               modulate food type-dependent effects C. elegans lifespan. PLoS Biol. 2010; 8:e1000376. <br />                                               [PubMed: 20520844] <br />                                          Mitsuishi Y, Taguchi K, Kawatani Y, Shibata T, Nukiwa T, Aburatani H, Yamamoto M, Motohashi H. <br />                                               Nrf2 redirects glucose glutamine anabolic pathways metabolic reprogramming. Cancer <br />                                               Cell. 2012; 22:66&#226;&#8364;&#8220;79. [PubMed: 22789539] <br />                                          Moi P, Chan K, Asunis I, Cao A, Kan YW. Isolation NF-E2-related factor 2 (Nrf2), NF-E2-like <br />                                               basic leucine zipper transcriptional activator binds tandem NF-E2/AP1 repeat the <br />                                               beta-globin locus control region. Proc. Natl. Acad. Sci. USA. 1994; 91:9926&#226;&#8364;&#8220;9930. [PubMed: <br />                                               7937919] <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                   Page 14 <br />  <br />  <br />                                          Ohta T, Iijima K, Miyamoto M, Nakahara I, Tanaka H, Ohtsuji M, Suzuki T, Kobayashi A, Yokota J, <br />                                              Sakiyama T, et al. Loss Keap1 function activates Nrf2 provides advantages lung cancer <br />                                              cell growth. Cancer Res. 2008; 68:1303&#226;&#8364;&#8220;1309. [PubMed: 18316592] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Oliveira RP, Porter Abate J, Dilks K, Landis J, Ashraf J, Murphy CT, Blackwell TK. Condition- <br />                                              adapted stress longevity gene regulation Caenorhabditis elegans SKN-1/Nrf. Aging Cell. <br />                                              2009; 8:524&#226;&#8364;&#8220;541. [PubMed: 19575768] <br />                                          Shibata T, Ohta T, Tong KI, Kokubu A, Odogawa R, Tsuta K, Asamura H, Yamamoto M, Hirohashi <br />                                              S. Cancer related mutations NRF2 impair recognition Keap1-Cul3 E3 ligase promote <br />                                              malignancy. Proc. Natl. Acad. Sci. USA. 2008; 105:13568&#226;&#8364;&#8220;13573. [PubMed: 18757741] <br />                                          Shtonda BB, Avery L. Dietary choice behavior Caenorhabditis elegans. J. Exp. Biol. 2006; 209:89&#226;&#8364;&#8220; <br />                                              102. [PubMed: 16354781] <br />                                          Singh A, Boldin-Adamsky S, Thimmulappa RK, Rath SK, Ashush H, Coulter J, Blackford A, <br />                                              Goodman SN, Bunz F, Watson WH, et al. RNAi-mediated silencing nuclear factor erythroid-2- <br />                                              related factor 2 gene expression non-small cell lung cancer inhibits tumor growth increases <br />                                              efficacy chemotherapy. Cancer Res. 2008; 68:7975&#226;&#8364;&#8220;7984. [PubMed: 18829555] <br />                                          Soukas AA, Kane EA, Carr CE, Melo JA, Ruvkun G. Rictor/TORC2 regulates fat metabolism, <br />                                              feeding, growth, life span Caenorhabditis elegans. Genes Dev. 2009; 23:496&#226;&#8364;&#8220;511. <br />                                              [PubMed: 19240135] <br />                                          Szewczyk NJ, Udranszky IA, Kozak E, Sunga J, Kim SK, Jacobson LA, Conley CA. Delayed <br />                                              development lifespan extension features metabolic lifestyle alteration C. elegans under <br />                                              dietary restriction. J. Exp. Biol. 2006; 209:4129&#226;&#8364;&#8220;4139. [PubMed: 17023606] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Takeda K, Komuro Y, Hayakawa T, Oguchi H, Ishida Y, Murakami S, Noguchi T, Kinoshita H, <br />                                              Sekine Y, Iemura S, et al. Mitochondrial phosphoglycerate mutase 5 uses alternate catalytic <br />                                              activity protein serine/threonine phosphatase activate ASK1. Proc. Natl. Acad. Sci. USA. <br />                                              2009; 106:12301&#226;&#8364;&#8220;12305. [PubMed: 19590015] <br />                                          Tullet JM, Hertweck M, An JH, Baker J, Hwang JY, Liu S, Oliveira RP, Baumeister R, Blackwell TK. <br />                                              Direct inhibition longevity-promoting factor SKN-1 insulin-like signaling C. elegans. <br />                                              Cell. 2008; 132:1025&#226;&#8364;&#8220;1038. [PubMed: 18358814] <br />                                          Villeneuve NF, Lau A, Zhang DD. Regulation Nrf2-Keap1 antioxidant response ubiquitin <br />                                              proteasome system: insight cullin-ring ubiquitin ligases. Antioxid. Redox Signal. 2010; <br />                                              13:1699&#226;&#8364;&#8220;1712. [PubMed: 20486766] <br />                                          Walker AK, See R, Batchelder C, Kophengnavong T, Gronniger JT, Shi Y, Blackwell TK. A <br />                                              conserved transcription motif suggesting functional parallels Caenorhabditis elegans <br />                                              SKN-1 Cap&#226;&#8364;&#732;n&#226;&#8364;&#8482;Collar-related basic leucine zipper proteins. J. Biol. Chem. 2000; 275:22166&#226;&#8364;&#8220; <br />                                              22171. [PubMed: 10764775] <br />                                          Wang J, Robida-Stubbs S, Tullet JM, Rual JF, Vidal M, Blackwell TK. RNAi screening implicates a <br />                                              SKN-1-dependent transcriptional response stress resistance longevity deriving from <br />                                              translation inhibition. PLoS Genet. 2010; 6:6. <br />                                          Wang Y, Oh SW, Deplancke B, Luo J, Walhout AJ, Tissenbaum HA. C. elegans 14-3-3 proteins <br />                                              regulate life span interact SIR-2.1 DAF-16/FOXO. Mech. Ageing Dev. 2006; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              127:741&#226;&#8364;&#8220;747. [PubMed: 16860373] <br />                                          Watts JL. Fat synthesis adiposity regulation Caenorhabditis elegans. Trends Endocrinol. Metab. <br />                                              2009; 20:58&#226;&#8364;&#8220;65. [PubMed: 19181539] <br />                                          Watts JL, Browse J. Genetic dissection polyunsaturated fatty acid synthesis Caenorhabditis <br />                                              elegans. Proc. Natl. Acad. Sci. USA. 2002; 99:5854&#226;&#8364;&#8220;5859. [PubMed: 11972048] <br />                                          Wolff S, Dillin A. The trifecta aging Caenorhabditis elegans. Exp. Gerontol. 2006; 41:894&#226;&#8364;&#8220;903. <br />                                              [PubMed: 16919905] <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                      Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. The Dominant SKN-1 Domain Functions Independently Canonical WDR-23 <br />                                          Pathway <br />                                          (A) SNP mapping Hawaiian wild-type strain CB4856 assigned alleles lax120 and <br />                                          lax188 LGIV B0273 C49H3. 576 RNAi clones covering genes between <br />                                          SNPs tested suppression dominant phenotype. <br />                                          (B) A skn-1/T19E7.2 RNAi clone construct capable suppression the <br />                                          dominant phenotype. <br />                                          (C) lax120 lax188 encode single missense mutations skn-1 coding sequence. Y2H <br />                                          analysis performed bait constructs <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> 200, 100, 50 amino acid <br />                                          region mutated dominant skn-1 alleles. <br />                                          (D) skn-1(lax120) skn-1(lax188) mutations alter interaction SKN-1 50 amino <br />                                          acid domain Y2H targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                           Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. Identification Unique Mitochondria-Associated SKN-1 Activation Pathway <br />                                          (A) SKN-1 enriched mitochondria fraction C. elegans. The pL4440 and <br />                                          skn-1 RNAi controls whole-worm extracts. In left panel, asterisks denote bands <br />                                          clearly decreased skn-1 RNAi, contrast 85 kD SKN-1 itself. <br />                                          (B) Partial purification achieved spinning mito pellet (mito), indicated <br />                                          presence remaining mitochondrial marker, PDHE1 (Greiss et al., 2008; Li et al., <br />                                          2011; Brys et al., 2010), cytosolic fraction (cyto), mitochondria fraction lacks <br />                                          cytoplasmic marker GAPDH. <br />                                          (C) Arrowhead indicates remaining PDHE1 removed stripping. <br />                                          (D) Synergistic activation SKN-1 transcriptional reporter (gst-4p::gfp) the <br />                                          canonical wdr-23 pathway dominant skn-1 mutations. Percentage represents fraction <br />                                          total animals, n, phenotype. <br />                                          (E) GFP score indicated genotypes based fluorescence intensity. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. SKN-1 Is Activated Starvation Changes Available Nutrients <br />                                          (A) GO-terms genes identified dysregulated transcriptional analysis lax120 <br />                                          lax188 mutants. <br />                                          (B) SKN-1 activity sensor worms grown food starvation conditions and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          imaged GFP expression indicative SKN-1 activation gst-4p::gfp reporter. <br />                                          (C) Quantification GFP levels observed described (B). <br />                                          (D&#226;&#8364;&#8220;I) Indicated strains fed E. coli B strain OP50. <br />                                          (D)&#226;&#8364;&#8220;(F) E.coli K12 strain HT115 (G)&#226;&#8364;&#8220;(I) imaged GFP expression described <br />                                          (B). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                        Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. Activated Mito-SKN-1 Promotes Perceived State Starvation <br />                                          (A) Bacterial dilution (bDR) life span; mean life span adult worms grown solid <br />                                          media concentration bacteria 25&#194;&#176;C. (B) Feeding rates assessed by <br />                                          pharyngeal pumping assays indicated strains. Error bars represent average +SD. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                         Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. Activated SKN-1 Promotes Extended Reproduction Developmental Delay in <br />                                          Response Food <br />                                          (A) Nonmated reproductive span worms fed OP50 bacteria 20&#194;&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (B) Total progeny (A). <br />                                          (C) L1 diapause survival measured movement staining bromophenol blue <br />                                          following 5, 7, 12 days absence food. <br />                                          (D) Growth development animals L1 diapause following feeding. <br />                                          (E) Percentage animals arrested early development recovery starvation. (* &lt; <br />                                          0.05, ** &lt; 0.01, *** &lt; 0.001). Error bars represent SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                                     Page 20 <br />  <br />  <br />  <br />  <br />                                                                                                 Table 1 <br />                            Activation Mito-SKN-1 Regulates Unique Transcriptional Response <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                               skn-1(lax120)                           skn-1(lax188) <br />  <br />                                                                               fold changeb     std devc     t-testd   fold changeb    std devc   t-testd <br />                                Stress         Glutathione       gcs-1                   4.48   1.00         0.001             4.00    0.95       &lt;0.001 <br />                                                                 gst-4                 21.31    1.06         &lt;0.001           22.16    1.01       &lt;0.001 <br />  <br />                                               Detox             ugt-11                8.75   0.96         0.001             4.61    1.05       &lt;0.001 <br />  <br />                                Aging/Dauer    Aging             wdr-23                2.81   1.01         0.001             2.85    0.98       &lt;0.001 <br />  <br />                                                                 dod-17                  5.00   0.94         0.002             4.38    1.01       &lt;0.001 <br />                                                                 dod-24                  7.69   1.15         0.005             7.49    1.20       &lt;0.001 <br />                                               Dauer             dod-6                   2.60   0.92           NS              2.33    1.22       0.004 <br />                                                                 C32H11.4              18.85    1.08         &lt;0.001           14.68    1.15       &lt;0.001 <br />                                               Unknown           dao-4                 &#226;&#710;&#8217;9.92    1.21         0.001            &#226;&#710;&#8217;2.37    1.09       0.001 <br />                                                                 R05G9R.1              &#226;&#710;&#8217;8.38    2.05         0.018            &#226;&#710;&#8217;4.49    1.11       &lt;0.001 <br />  <br />                                                                 zk180.6             &#226;&#710;&#8217;7.66    2.49         0.031            &#226;&#710;&#8217;3.78    1.09       &lt;0.001 <br />  <br />                                Metabolism     Amino Acid        tyr-5              &#226;&#710;&#8217;42.71    2.38         0.009          &#226;&#710;&#8217;48.67     1.01       &lt;0.001 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                 aat-5                   1.77   0.87           NS              1.51    1.12       0.004 <br />                                                                 atgp-2                  1.88   0.85         0.045             1.81    1.07       0.002 <br />                                                                 aat-2                   2.31   1.02         &lt;0.001            3.21    1.00       &lt;0.001 <br />                                                                 K07E1.1                 4.97   1.08         0.002             2.83    0.98       &lt;0.001 <br />                                               Lipid             F09C8.1              &#226;&#710;&#8217;14.06    1.89         0.009            &#226;&#710;&#8217;9.03    0.82       &lt;0.001 <br />                                                                 Y4C6B.6               &#226;&#710;&#8217;3.23    0.95         0.010            &#226;&#710;&#8217;3.20    0.96       &lt;0.001 <br />  <br />                                                                 acdh-6              &#226;&#710;&#8217;2.93    1.18         0.010            &#226;&#710;&#8217;2.89    1.00       &lt;0.001 <br />  <br />                                                                 lips-3              &#226;&#710;&#8217;2.37    0.96         0.010            &#226;&#710;&#8217;2.96    1.06       0.011 <br />  <br />                                                                 lips-10                 2.74   1.14         0.007             1.90    1.13       &lt;0.001 <br />                                                                 zk742.4                 9.36   1.08         0.001             7.80    1.09       &lt;0.001 <br />  <br />                                               Quinone           R08E5.3             &#226;&#710;&#8217;6.67    2.67         0.039            &#226;&#710;&#8217;4.67    0.94       &lt;0.001 <br />  <br />                                                                 sqrd-1                &#226;&#710;&#8217;6.18    2.40         0.040            &#226;&#710;&#8217;3.59    1.15       &lt;0.001 <br />  <br />                                               Unknown           Y42G9A.3              3.77   1.08         0.004             3.42    1.07       &lt;0.001 <br />  <br />                                Reproduction   Germline          bcc-1                   2.32   0.96         0.001             2.35    0.95       &lt;0.001 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                 glh-1                   2.42   0.80         0.007             2.24    1.14       &lt;0.001 <br />                                               Piwi/Paz          ppw-2                   2.75   0.65           NS              2.38    1.21       0.001 <br />  <br />                                Signaling      Insulin           ins-35                3.20   1.00         0.001             3.64    1.01       &lt;0.001 <br />  <br />                                               Kinase            kin-15                  3.61   0.90         0.003             1.52    0.91       0.002 <br />  <br />                                                                 kin-16                4.04   1.05         0.003             2.15    1.00       &lt;0.001 <br />  <br />                                               Transcription     mxl-3               &#226;&#710;&#8217;1.03    0.92           NS             &#226;&#710; <br /> </body></html>