<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>AtVPS34, a phosphatidylinositol 3-kinase of Arabidopsis thaliana, is <br /> an essential protein with homology <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> a calcium-dependent lipid <br /> binding domain <br />      (vacuoe/prIn t            rt/yeast come                 on/      c gene/ant-ene) <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>          The cDNA e pho atldylinositol (PI)                                 PI 3-kinase is required for internalization and degradation of <br /> 3-knim was cloned bomArabidpsis t              and the derived                       platelet-derived growth factor receptor (16). <br /> amino acid seuence (AtVPS34) has a s              antl higher                          Using PCR with oligonucleotides based on conserved <br /> homology to yeast PI 34-dase (VPS34) than to the mammalian                           sequences in PI 3-kinases <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> bovine tissues (17), yeast (9), <br /> (plO). The protein has two conserved domains: a catlic site                          and a PI 4-kinase of yeast (18), we identified an A. thaliana <br /> with the ATP-bdng site near the C terinns and a calcium-                             PI 3-kinase cDNA (AtVPS34). We found considerable amino <br /> depenldent lipid-binding domain near the N te s. The                                 acid sequence identity between AtVPS34 and yeast VPS34, <br /> plant cDNA does not rescue a yeast vps34 deletion mutat, but                         as well as mammalian p110, especially in the C-terminal half <br /> a chenrV gene In which the coding sequence for the C-termi-                          with its catalytic domain. In the N-terminal half of the plant <br /> nal third of VPS34 Is repl by the correspondig sequence                              protein homology to a calcium-dependent phospholipid- <br /> from the plant gene does rescue the yeast mutant. PI 3-kinase                        binding (CaLB) domain is present, as found in proteins such <br /> activit s d b in extracts from plants that overexpres                                as rabphilin-3A, a protein involved in fusion of synaptic <br /> the plant PI 3-kinase. Expressn of antse c                gives                      vesicles to the plasma membrane (7). <br /> rise to second-generation transformed plants severely inhibited <br /> in growth and development.                                                                        MATERIALS AND METHODS <br /> Sigaling cascades involving phosphorylated phospholipids                               Mdecular Biolgy Techniques. Cloning, transformation, <br /> that function in mammalian and yeast cells may also operate                          blotting, and hybridization were done by standard methods, <br /> in plant cells (1). Because of the importance of phosphati-                          as described (19). PCR was conducted by denaturation at <br /> dylinositol (PI) 3-kinases in signaling cascades of mammalian                        94&#194;&#176;C for 3 min, followed by 25-40 cycles of 1-min denatur- <br /> cells (2) and for vacuolar targeting in yeast (3), we studied the                    ation at 94&#194;&#176;C, 1-min annealing at 65&#194;&#176;C, and 1-min elongation <br /> role of PI 3-kinase in Arabidopsis thaliana. This enzyme has                         at 72&#194;&#176;C. <br /> not yet been found in plant extracts, although its activity has                         Plasnid Consthucti. Oligonucleotides used for amplifi- <br /> been demonstrated by metabolic labeling of lipids with ra-                           cation of the VPS34 homolog from A. thaliana were 2048-fold <br /> dioactive precursors (4, 5).                                                         degenerated covering the following sequences: 1, CAGY- <br />   Targeting of vacuolar proteins requires both information                           CVA/I; 2, GDDLRQD/E; and 3, HI/AD-FGH/Y/F. Se- <br /> within the proteins themselves (vacuolar sorting signals) and                        quences 1 and 3 had a BamHI restriction site at their 5&apos; ends <br /> machinery for protein targeting: receptors as well as proteins                       and amplified the complementary strand; sequence 2 had a <br /> on the outside of the vesicles (coat and regulatory proteins)                        Pst I site and is specific for the coding strand. All three started <br /> and on the vacuolar membrane. GTPases are likely to be                               with GTTT to ensure function of the restriction enzyme. The <br /> important in these processes (6), as well as proteins necessary                      resulting PCR fiagment (291 bp) was isolated from an agarose <br /> for vesicle fusion with the target membrane, possibly by                             gel and without further treatment cloned into pCRII (Invit- <br /> binding to specific phospholipids in these membranes (7).                            rogen). pMM9.1 (AtVPS34) was isolated from a cDNA li- <br /> How these proteins interact to deliver vacuolar protein                              brary of A. thaliana seedling mRNA in pFL61 (20). For <br /> remains to be elucidated, but VPS34 and VPS15 have been                              antisense and sense expression in plants, a 2.8-kb Not I <br /> shown to be components of a membrane-associated protein                              fragment of AtVPS34 was cloned into Xba I/Sst I sites of <br /> complex needed for the correct targeting ofvacuolar proteins                         pBI121 after a Klenow fragment fill-in reaction (p9.1A and <br /> in Saccharomyces cerevisiae (8). VPS15 encodes a protein                             p9.1S, respectively). ScAtI is a fusion of the N-terminal yeast <br /> kinase (9) and VPS34 encodes a PI 3-kinase (10, 11).                                 and C-terminal plant clone at the single Dra I restriction site <br />    The importance of PI for vesicle transport is underscored                         in AtVPS34 conserved in ScVPS34 yielding a fusion protein <br /> by the crucial role of PI transfer protein, the product of the                       of amino acids 1-601 of the yeast and 540-815 of the plant <br /> SEC14 gene in yeast, for exit of proteins from a late Golgi                          VPS34. ScAtH is a fusion at the filled in Bgi II ends of <br /> compartment (12). This protein transfers PI between lipid                            AtVPS34 and Ava I ends of ScVPS34, resulting in an in-frame <br /> bilayers and is required to maintain the correct PI level in                         fusion of amino acids 1-463 of ScVPS34 and 402-815 of <br /> Golgi membranes. <br />    The association of PI 4-kinases with the cytoskeleton (13)                        Abbreviations: CaLB, calcium-dependent lipid-binding; PI, phospha- <br /> or their colocalization with vesicles (14) and the interaction                       tidylinositol. <br /> of their products with proteins like profilin (15) also point to                     tTo whom reprint requests should-be sent at present address: Centre <br /> involvement of PI kinases in protein transport. Furthermore,                          National de la Recherche Scientifique, Unite de Recherche Asso- <br />                                                                                       ciee 203, Faculte des Sciences, Universite de Rouen, 76821 Mont <br />                                                                                       Saint Aignan Cedex, France. <br /> The publication costs of this article were defrayed in part by page charge           &#194;&#167;Present address: Department of Bioresource Science, Faculty of <br /> payment. This article must therefore be hereby marked &quot;advertisement&quot;                 Agriculture, Kagawa University, Miki-cho, Kagawa-ken, Kagawa <br /> in accordance with 18 U.S.C. &#194;&#167;1734 solely to indicate this fact.                      761-07 Japan. <br />                                                                              11398 <br />            Plant Biology: Welters et A                                                                   Proc. Natl. Acad. Sci. USA 91 (1994)   11399 <br /> AtVPS34. ScAtI, ScAtII, and ScVPS34 are derivatives of a                      resulting fragments with yeast VPS34 as probe (data not <br /> single-copy yeast vector (pRS316).                                            shown). Only under very stringent conditions for the PCR <br />   Yeast Strains. S. cerevisiae strains used were SEY6210                      (650C annealing temperature compared with a calculated melt- <br /> [MATa, leu2-3, 112 ura3-52, his3-4200, trpl-A901, lys2-801,                   ing temperature of 62&#194;&#176;() was amplification specific enough to <br /> suc2-A9 (ref. 21)] and the vps34 deletion mutant PHY102                       yield a fragment hybridizing to the yeast probe. Moreover, <br /> [SEY6210 vps34A1::TRP1 (ref. 10)].                                            only the inner primer pair (2/1, Fig. 1) could amplify the <br />   Yeast Transformation. A modified version of a published                     sequence. The deduced amino acid sequence of the 291-bp <br /> method (22) was used for transformation. The highest effi-                    PCR fiagment was 61% identical to the corresponding yeast <br /> ciency was achieved by applying a heat shock of 15 min at                     VPS34 region. Identity with bovine PI 3-kinase (36%6) and <br /> 420C despite the heat sensitivity of the strains and using                    yeast PI 4-kinase (31%) was also significant. To get a full- <br /> selection plates containing 1 M sorbitol.                                     <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> cDNA clone, this PCR fragment was used to screen an <br />   Plant Transformation. Calli induced from roots of A.                        A. thaliana cDNA library in pFL61 (20). Out of 40,000 <br /> thaliana ecotype Landsberg were transformed by cocultiva-                     colonies, 12 clones were isolated. One of them (pMM9.1) <br /> tion with Agrobacterium tumefaciens LBA4044 bearing plas-                     yielded PCR fragments of the appropriate size with the de- <br /> mids pBI121, p9.1A, and p9.1S. Transgenic plants were                         generate primers; this 2.8-kb clone was sequenced in its <br /> obtained after regeneration on kanamycin-containing me-                       entirety. <br /> dium, as described (23).                                                        The cDNA clone consists of a 121-bp untranslated leader <br />   PI 3-Kinase Assay. A described method (3) was used for                      sequence, 2442-bp coding sequence, and 198-bp 3&apos; region <br /> yeast protein extraction and assay conditions. For plant pro-                 followed by a stretch of poly(A) (GenBank accession no. <br /> tein extracts, plant tissue was pulverized in liquid nitrogen.                U10669). The deduced amino acid sequence was 25% identical <br /> The resulting powder was suspended in yeast protein extrac-                   to the bovine p110 and 40% identical to the yeast PI 3-kinase <br /> tion buffer [15 mM Hepes, pH 7.5/100 mM KCI/3 mM                              (ScVPS34 in Fig. 1). The lower identity with the mammalian <br /> EGTA/10%o (vol/vol) glycerol/2 mug of tissue] with a pro-                     peptide sequence is caused, in part, by the more variable <br /> teinase inhibitor mixture (leupeptin, pepstatin, aprotenin, phe-              N-terminal sequence of the p110 polypeptide. Common to the <br /> nylmethylsulfonyl fluoride in concentrations between 1 and 10                 plant and mammalian PI 3-kinases is the deletion of a 60-aa <br /> ng/ml). The supernatant after centrifugation at 700 x g for 5                 stretch from the yeast sequence, 400 aa from the C terminus <br /> min was used for PI 3-kinase assays. Briefly, a suspension of                 (Fig. 1, black box). The 60-aa insertion in the yeast protein is <br /> 1.5 pg of PI in 15 mM Hepes, pH 7.5/100 mM KCl was                            a serine-rich sequence with unknown function. <br /> incubated with protein extracts of yeast or plants, 10 pCi of                   AVPS34 Contains a Lipid-Bin          Domain. A BLAST Net- <br /> [y-32P]ATP (1 Ci = 37 GBq), and 0.3 mM unlabeled ATP, in                      work Service search with the entire amino acid sequence of <br /> 10 mM MgCl2 and 20 mM Hepes, pH 7.5, in a final 5SO- vol                      AtVPS34 found the closest matches to yeast VPS34, bovine <br /> for 5 min at room temperature. The reaction was stopped with                  and murine pl10, PI 4-kinase from yeast, and TOR2, a protein <br /> 80 A&amp; of 1 M HCl; the lipids were extracted with 160 A4 of                    involved in cell cycle control, in that order of decreasing <br /> CHCl3/CH30H, 1:1, and after evaporation of the solutes and                    similarity. The highest similarity was always in the ATP- <br /> redissolving of the lipids in CHC13 separated by TLC on silica                binding domains found near the C terminus ofthese proteins. <br /> gel 60 TLC plates (Merck) with a borate buffer system (24).                   A search with parts of the N-terminal sequences (aa 85-150 <br />   Computer Programs. The programs used were developed                         of AtVPS34) revealed a strong homology to a CaLB domain <br /> by the Genetics Computer Group (25), Smith (11), Henikoff                     of rabphilin-3A (24% identity and 47% similarity; Fig. 2). <br /> and Henikoff (ref. 26; BLOCKS search), and Altschul et al. (ref.              Because of the high homology between a portion of AtVPS34 <br /> 27; BLAST Network Service).                                                   and core stretches of the CaLB domain of rabphilin-3A (47% <br />                                                                               identity in aa 96-119 of AtVPS34; box I, Fig. 2) and of p110 <br />                            RESULTS                                            (44% identity in aa 122-139 of AtVPS34; box II, Fig. 2), we <br />                                                                               postulate that a phospholipid-binding site is located in this <br />   Cloning of Plant PI 3-Kinase cDNA. The cloning of a                         region of the plant protein. In spite of the moderate identity <br /> fragment of AtVPS34 was achieved by using the PCR with a                      value for these regions of the yeast and plant VPS34 proteins <br /> whole-plant A. thaliana cDNA library and degenerate primers                   (21%), the good similarity scores (46%) lead us to suggest in <br /> derived from conserved amino acid sequence domains in the                     contrast to other reports (2) that yeast VPS34 may have a <br /> mammalian (17) and yeast PI 3-kinases (3, 9), as well as a yeast              similar functional domain in this region. <br /> PI 4-kinase (18). Positions of the three primers are indicated in                A Chileric Gene Rescues a S. cerevmuja Mutant. The <br /> Fig. 1. The PCR conditions were optimized by probing the                      AtVPS34 gene on a 2,u plasmid expressed under the control <br />                                                                                      4       4. <br />                             s cV <br />                                v P3 4                      1\\\\\ <br />                             40%(61 %) <br />                                                                                      L       .w   ... <br />  <br />                             A tvPS 34 <br />                             26%(52%) <br />                                                14 8                                      C                             _ <br />                                                              ~~~~.... <br />                                                                  ~ <br />                              c11o <br />  <br />  <br />                                                              G:I, <br />                                                                 D L.I. <br />                                                                &apos;D <br />  <br />                                                             GDD LRQ D     -                                    - <br />  <br />                                                       C)    C-D DLRQADD        .1l           -               I- <br />  <br />  <br />  <br />   FiG. 1. PI 3-kinases of A. thaliana (open box), S. cerevisiae (hatched), and mammal (bovine p110, stippled). The amino acid sequence <br /> AtVPS34 has 40%o identity (61% similarity) to that of ScVPS34 but has only 26% (52%) to p110. The N-terminal half is less conserved with 29%O <br /> identity between plant (aa 1-401) and fungal (aa 1-418) domains and 22% identity between plant and mammalian (aa 148-646) polypeptides. <br /> Black boxes are regions with homology gaps &gt;20 aa. Bars a-c indicate conserved regions used to design oligonucleotides; arrowheads indicate <br /> their orientations and positions. <br />  11400      Plant Biology: Welters et al.                                                 Proc. Nad. Acad Sci. USA 91                                       (1994) <br />                                                                                                               7.           -- <br /> A                                                                      A                      30 C <br />                                                                                                             AtVPS34 ScAt                   \ <br />     p110 <br />     AtVPS34                                                                                        __ <br />     rabphilin-3A         I <br />  <br />  <br /> B <br />     p110                      369     .DNVNTQRVPCSNPRWNEWLNYDIYI <br />     AtVPS34                      68   RTRLKTTGPPYC ... WNELITLSSKY <br />                                       :1.:1...         :11 : .. I. <br />      rabphilin-3A             615     KTQIKKKTLNPE ... FNEEFFYDIKH <br />  <br />     PDLPRAARLCLSICSVKGRKGAKEEHCPLAWGNI LFDYTDTLVSGKMA. LNLW <br />                                                                        B <br />     R DLTAHSQLAITVWDVSCGKT .... EGLIGG ATVL.LFNSKMOMKSGKQK   LgkWj             ScAt                \X-N\E-bCKNENS\XES                                      ~~~~~~~----- <br />      .1..11.1.111 ..11.          :.1     :11.1                                 (58976i         z~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~9 I., 1 <br />  <br />     S bLAKKS.LDISVWDYDIGKS....   NDYIGGCQL ........ GISAKGERLKHW           2 <br />                                                                                                               r -- -- ---         S~4~-I    40 2             _ <br />                                                                                ArVPS34         F <br />                                                                                                                                                7 <br />  <br />     PVPHGLEDLLNPIGVTGSNPNKETPCLELEFDWFSSV        483 <br />           :.: <br />       ....:          ..1.1.1 ..     1::..                                  3   ScAti                                                                         ____        _- <br />  <br />                                                  170                           (50I69                                                      463 <br />     QGKEADGSF ...... PTSTPGKVPRHERGELERLEKL <br />                                                                            4   Sc VPS34 <br />     YECLKNKDKKI .......... ERWHQLQNE.NHVSSD      704 <br />                                                                        C <br />   FiG. 2. Homology of AtVPS34 CaLB-domain to plo and rab- <br /> philin-3A. (A) Schematic location ofCaLB domains (hatched boxes);                                                                                                -13*. <br /> black bars, ATP-binding domain in p110 and AtVPS34. (B) Compar-                                                                                                  -N&amp;I- <br />                                                                            PI 3-P       IN.          &quot;-&quot; <br />  <br /> ison ofamino acid sequences ofrabphilin-3A and pliOCaLB-domain                                       --gl <br />                                                                                                       im <br />  <br /> to AtVPS34. Dots represent different degrees of similarity; dashes                                                         15MM                    -jai <br /> indicate identity. Box I emphasizes regions of highest identity                                                    ,I <br />                                                                                                                        I                           .17-. <br /> betweenAtVPS34 and p110 (44%) CaLB-domains; box I emphasizes               PI 4-P       00-                        - <br /> those between AtVPS34 and the second CaLB-domain of rab- <br /> philin-3A (47%). <br /> of the phosphoglycerate kinase promoter and terminator (20)             FiG. 3. Growth at different temperatures and PI 3-kinase assay <br /> failed to rescue a vps34 deletion mutant that is temperature         for a vps34 deletion mutant of S. cerevisiae (PHY102) transformed <br /> sensitive for growth at 3rC (Fig. 3A, upper left quadrant).          with ScVPS34, AtVPS34, and chimeric genes of both. (A) Growth of <br /> Extracts of the cells grown at 30TC expressed AtVPS34 (as            PHY102 transformed withAtVPS34, ScVPS34, ScAtII, and ScAtI on <br /> shown in Northern blots) but contained no PI 3-kinase                yeast extract/peptone/dextrose at 30&#194;&#176; and 37C. The ScVPS34 and <br /> activity (Fig. 3C, lane 2).                                          the ScAtI fusion complemented the temperature-sensitive <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br />   To test whether the phosphoglycerate kinase promoter was           of the yeast mutant. PHY102 transformed with ScAtI or with <br />                                                                      AtVPS34 were unable to grow at 37TC. [Fewer colonies in the upper <br /> responsible for the lack of enzyme activity, we made a               left of the plate grown at 30&#194;&#176; (AtVPS34) is not caused by the gene <br /> construct in which the plant coding sequence was controlled          presence and was not found on other plates.] (B) Schematic drawing <br /> by yeast VPS34 promoter. This construct, which contained             of PI 3-kinases of yeast and A. thaliana and chimeric fusions tested <br /> the proper yeast expression signals, also failed to rescue the       in PHY102 complementation. Numbers above and below barsdenote <br /> yeast deletion mutant (data not shown).                              amino acids at point of fusion, respectively, in Arabidopsis and yeast <br />   The extent of homology in the N-terminal half ofthe VPS34          proteins; in brackets are percentage identity/percentage similarity <br /> proteins is relatively low (29%, Fig. 1). Therefore, the plant       ratios ofthe exchangedArabidopsis segment to its yeast counterpart. <br /> protein may not associate with the necessary effector pro-           (C) PI 3-kinase assay of protein extracts of the strains described <br />                                                                      above grown in liquid medium at 30TC. Corresponding to the results <br /> teins, such as VPS15. To test this hypothesis, we made               of the growth phenotype, the strains able to grow at 37C, PHY102/ <br /> chimeric yeast-plant cDNAs in which domains between the              ScVPS34 (lane 4) and PHY102/ScAtI (lane 1), have PI 3-kinase <br /> two proteins were swapped. Two of these are shown in Fig.            activity (PI 3-P), whereas no PI 3-kinase activity was detectable in <br /> 3B. In the construct ScAtI the terminal third of the plant           PHY102/AtVPS34 (lane 2) and PHY102/ScAtl (lane 3). PI 4-P, PI <br /> cDNA is fused to two-thirds of the yeast gene from the N             4-kinase activity. <br /> terminus. Taking advantage of a conserved Dra I restriction          right quadrant), and cells grown at 30TC had no detectable PI <br /> site in both genes, this fusion was made at the N-terminal end       3-kinase activity (Fig. 3C, lane 3). <br /> of the most conserved domain (58% identity, Fig. 3B). ScAtII           AtPS34 Is Expresed In Transgenc Cels and Encdes an <br /> has the C-terminal half of AtVPS34 (aa 402-815) and aa 1-463 <br /> ofScVPS34 (Fig. 3B). Both coding sequences were expressed            Active PI 3-Klnase. The presence of two stretches of amino <br /> with yeast expression signals (ScVPS34 promoter and phos-            acid identity (100% match, aa 563-577 and 665-676), defined <br /> phoglycerate kinase terminator). Of these two constructs,            in Prosite 11 (keyed to SwissProt 27) as specific for PI <br /> only the one with the shorter plant coding segment rescued           3-kinases in AtVPS34 and the high homology between these <br /> the temperature-sensitive growth defect of the yeast vps34-          proteins, especially in the catalytic domain, suggest a similar <br /> deletion strain (Fig. 3A, upper right quadrant). When this           enzymatic activity for AtVPS34. PI 3-kinase activity has <br /> complemented strain was grown at 300C, cell extracts con-            never been found in plant protein extracts. To demonstrate <br /> tained PI 3-kinase activity (Fig. 3C, lane 1). The substitution      AtVPS34 PI 3-kinase activity in plant extracts, the gene was <br /> in ScAtI involves the most conserved domains of the yeast            overexpressed in plants. A. thaliana root explants were <br /> and plant proteins (76% similarity; 58% identity), and this          transformed with an AtVPS34 construct in which expression <br /> domain contains the protein catalytic activity (3). When a           was driven by the cauliflower mosaic virus 35S promoter <br /> slightly larger part of the plant protein was present in the         (p9.1S) or with a control vector (pBI121) that lacked the <br /> chimeric construct, complementation failed (Fig. 3A, lower           AtVPS34 gene. The resulting transformed cells were propa- <br />             Plant Biology: Welters et al.                                              Proc. Nati. Acad. Sci. USA 91 (1994)           11401 <br /> gated as a callus and also regenerated into plants. These <br /> plants appeared normal and had normal seed set; the seeds <br /> germinated on kanamycin-containing medium. RNA was <br /> extracted from control, pBI121-transformed, and AtVPS34- <br /> transformed plants, and a Northern blot was probed with <br /> AtVPS34 (Fig. 4A). The same blot was probed with a (-tu- <br /> bulin cDNA as control. Transformed plants highly expressed <br /> a 2.8-kb RNA, for which a dilution series is shown in lanes <br /> 1-3. Control plants and vector-transformed plants had much <br /> lower levels of this mRNA (lanes 5 and 6, respectively). The <br /> mRNA was also readily detected in the transformed callus <br /> cells at a level considerably above that in control callus (data <br /> not shown). Extracts of control and transformed callus were <br /> assayed for PI 3-kinase activity (Fig. 4B). AtVPS34- <br /> transformed callus cells contained low, but detectable, levels <br /> of PI 3-kinase (lane 3, the radioactivity applied represents <br /> enzyme activity 2.5 times more protein for plant extract than             FiG. 5. Phenotype of AtVPS34 antisense plants. Shown are <br /> for yeast extract). No enzyme activity was found in cells               3-mo-old plants germinated from seeds of regenerated transgenic <br /> transformed with pBI121 vector alone (lane 2) or in wild-type           plants derived from a transformation with p9.1A. (Left) Most severe <br /> cells, even when 8 times more protein was used (data not                viable phenotype without shoot elongation. (Right) Plant with nearly <br /> shown). No other differences were detected in the radioac-              wild-type phenotype. <br /> tivity-labeled lipids when comparing extracts of control and <br /> overexpressing plants. PI 3-kinase activity was very abun-              inward and dipped into the agar. Then growth stopped, al- <br /> dant in extracts of yeast cells (lane 1).                               though a callus eventually was formed where small leaves <br />   AtVPS34 Enes an Esl Protein. To gain insight into                     grow out (Fig. 5 Left). In a less severe phenotype, (four plants) <br /> the role of AtVPS34 in plants, the plant gene was inactivated           the leaves were less rolled, and a short shoot formed. In both <br /> by expressing the cDNA in A. thaliana in the antisense                  types, the leaves appeared shiny, and water covered the leaf <br /> orientation (28) using the cauliflower mosaic virus 35S pro-            surface. One plant containing the antisense gene exhibited a <br /> moter and kanamycin as the selectable marker. Plants ex-                nearly wild-type phenotype (Fig. 5 Right). The other four <br /> pressing the antisense gene are used to determine the phe-              plants with wild-type phenotypes were not tested by PCR. <br /> notype resulting from a lower level or the absence of PI <br /> 3-kinase.                                                                                        DISCUSSION <br />    Characteristic of the initial transformants (T.) with the <br /> antisense construct (p9.1A) was the slower regeneration of              In plant, mammalian, and yeast cells, proteins are synthesized <br /> shoots (6 weeks instead of 4 weeks) and fewer shoots arising            on the endoplasmic reticulum, sorted in the Golgi body, and <br /> from the calli. About 20 independent shoots were obtained.              delivered to the extracellular space or to vacuoles (lysos- <br /> From seven of these, callus lines were generated and main-              omes), depending on specific targeting information. This tar- <br /> tained. The regenerated plantlets flowered and after selling            geting information differs among organisms, but some com- <br /> produced seeds; seed set was poor, as was gemination of the             ponents ofthe secretory machinery appear conserved. Indeed, <br /> seeds; from only three plants were fertile seeds obtained.              some proteins ofthe secretory system ofplant cells have been <br /> With kanamycin (50 pg/ml) seedlings died within 2 weeks.                identified by complementing yeast secretion mutants (29,30). <br /> Without kanamycin, we recovered 11 plants, and the trans-               The enzyme PI 3-kinase, which catalyzes phosphorylation of <br /> gene in these plants was shown by PCR.                                  PI, is involved in protein sorting in both mammalian and yeast <br />    The three most severely affected T, plants had leaves that           cells (3, 16). In yeast, vps34 mutants secrete vacuolar en- <br /> grew more or less normally at first, but the petioles failed to         zymes, and the protein encoded by VPS34, PI 3-kinase, binds <br /> elongate, and no stem was formed. The leaf edges curled                 directly to a membrane-associated protein kinase encoded by <br />                                                                         VPS1S. This protein complex is postulated to regulate the <br />        A                p9lS       pB121 wt    B                        sorting/packaging of vacuolar hydrolases in a late Golgi com- <br />                                                                         partment (8). In mammals, PI 3-kinase plays a role in the <br />        AtVPS34                                 PI   3-P -               internalization of human platelet-derived growth factor recep- <br />        RNA   (L4g) 20 2 0.2 50 20 20                        9           tors and other receptor tyrosine kinases (16). <br />        I -tubulin   _W#M                       PI   4-P *       9 <br />                                                                 |          Structure and Functon of AtVPS34, an A. tan PI <br />                                                                         3-Kinase. In addition to size similarity, the homology of A. <br />                     1    2     3   4   5   6                    2   3 <br />                                                                         thaliana PI 3-kinase to yeast VPS34 (40% identity) is signif- <br />    FIG. 4. (A) Detection of AtVPS34 by Northern blotting. Total <br />                                                                         icantly higher than to mammalian p110 (26% identity). This <br /> RNA of shoots of wild-type and transgenic plants were probed with       result favors the conclusion that AtVPS34 has similar prop- <br /> a 1.6-kb C-terminal Sac I frgent of AtVPS34, detecting a 2.8-kb         erties as ScVPS34. The C-terminal domains of both proteins <br /> band. The dilution series to evaluate extent of overexpression (lanes   contain ATP-binding and kinase motifs that define the pre- <br /> 1-3) suggest a2- to 5O-fold higherAtVPS34 expressioninp9.1S (lane       sumed catalytic domain. Point mutations in this part of yeast <br /> 1) compared with the same amount of RNA in transgenic control           VPS34 abolish any detectable PI 3-kinase activity and do not <br /> (lane 5) or wild-type (ane 6) plants. The blot was reprobed with        complement a yeast vps34 deletion mutant (3). Exchange of <br /> P-tubulin as internal control. The callus tissue of p9.1S showed a      the last 255 aa of the yeast protein for the plant sequence <br /> similar increase in AtVPS34 expression as regenerated p9.1S plants.     created a fusion protein able to restore temperature resis- <br /> (B) PI 3-kinase activity (PI 3-P) of callus tissue overexpressing       tance and PI 3-kinase activity to a yeast vps34 deletion <br /> AtVPS34. Protein extracts of kanamycin-resistant calli from a trans- <br /> formation with a p9.1S and a control plasmid were assayed for PI        mutant (Fig. 3). <br /> 3-kise activity, and labeled lipids were separated by TLC. Over-          The N-terminal part of the protein is thought to have a <br /> expressing Atp9.1S calli (lane 3) show four times less PI 3-kinase      ligand binding and/or regulatory role by interacting with <br /> activity than yeast cells (lane 1) noalizd to equal PI 4-kinase         other proteins, such as VPS15 or p85. The low sequence <br /> activity (PI 4-F). PI 3-kinase activity is below detection limit in     conservation in this domain may account for our failure to <br /> extracts of control calli (lane 2).                                     complement the yeast mutant, as plant VPS34 may not <br />  11402    Plant Biology: Welters et al.                                               Proc. Nad. Acad. Sci. USA 91 (1994) <br /> efficiently recognize yeast VPS15. The interaction of p110           endocytosis of these receptors (16). In yeast, PI 3-kinase is <br /> with its regulatory subunit p85 also maps to this less con-          essential for vesicle-mediated delivery of proteins to the <br /> served N-terminal region (2).                                        vacuole (3). Therefore, AtVPS34 may serve an essential role <br />    Our finding that AtVPS34 contains a putative lipid-binding        in similar targeting events and, as a consequence of their <br /> domain at its N-terminal end (aa 39-170) similar to that found       defect, prevent normal plant development and growth in <br /> in p110, rabphilin-3A, synaptotagmin (p65), and several other        antisense plants. <br /> proteins (2) strengthens the hypothesis that AtVPS34 is                 <br /> involved in vesicle-transport events. This CaLB domain may         <br /> help bind the substrate (PI) or it may permit PI 3-kinase           <br /> association with the membrane to trigger specific transport        <br /> events. The resulting phosphorylation of PI may recruit            <br /> additional proteins to the membrane, either by ionic inter-          <br /> action as for the CaLB-containing GTPase activating protein          <br />                                                                     <br /> (GAP) protein (2) or through covalent binding as shown for           <br /> elongation factor (EF) la. The membrane-bound portion of <br /> the pool of this GTP-binding protein has a PI anchor cova-            <br /> lently attached to an internal amino acid (31). Another              <br />                                                                           <br /> function of this protein is the binding and bundling ofF-actin.      <br /> The recent finding that an EF-la plant homolog is also a                   <br /> membrane-bound activator of PI 4-kinase (32), together with           <br /> reports of PI 4-kinases found on the surface of vesicles (14)             <br /> and isolated together with F-actin (33), may indicate that both        <br /> kinase activities are crucial in vesicle budding processes, as             <br /> shown for the internalization of tyrosine receptor kinases by        <br /> endocytotic events in mammalian cells (2).                                 <br />    PI 3-Klnase Actvity In Prnts. PI 3-kinase activity has so far           <br /> not been found in plant protein extracts (13), and we also                 <br /> failed to detect this activity in control plant extracts (Fig. 4B,    <br /> lane 2). The existence of PI 3-kinase could be inferred from              <br /> labeling patterns of lipids extracted from duckweed (5) or            <br /> guard cells of Commelina communes (34) labeled with appro-           <br /> priate radioactive precursors. We detected very low levels of        <br /> PI 3-kinase activity (compared with PI 4-kinase activity) in         <br /> transgenic callus cells overexpressing AtVPS34 (Fig. 4B). PI                <br /> 3-phosphate and PI 4-phosphate were not completely sepa-             <br /> rated as unique spots in this chromatographic system. Nev-                 <br /> ertheless, we believe that the spot identified as PI 3 is not       <br /> caused by &quot;fronting&quot; from the PI 4 spot because increased                   <br /> loading (5-fold) did not produce such a spot when extract                  <br /> from an untransformed plant was used. Headgroup analysis,            <br /> as done for yeast (3), will be necessary to confirm unequiv-               <br /> ocally that this spot represents PI 3-kinase activity. mRNA                <br /> for the enzyme was more readily detected than enzyme                        <br /> activity (Fig. 4A). The low level of active enzyme may be                  <br /> caused by rapid inactivation in cell extracts or the absence of     <br /> a required activation step, leaving most of the enzyme                     <br /> inactive, as occurs in ScVPS34-overexpressing yeast cells <br /> </body></html>