<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Accumulation Squalene Microalga <br />                                                           Chlamydomonas reinhardtii Genetic <br />                                                           Modification Squalene Synthase and <br />                                                           Squalene Epoxidase Genes <br />                                                           Masataka Kajikawa1, Seiko Kinohira1, Akira Ando2, Miki Shimoyama2, Misako Kato2, <br />                                                           Hideya Fukuzawa1* <br />                                                           1 Graduate School Biostudies, Kyoto University, Kyoto, Japan, 2 Graduate School Humanities and <br />                                                           Science, Ochanomizu University, Tokyo, Japan <br />  <br />                                                           * fukuzawa@lif.kyoto-u.ac.jp <br />  <br />  <br />  <br />  <br />                                                           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     OPEN ACCESS <br />                                                           Several microalgae accumulate high levels squalene, provide potentially <br /> Citation: Kajikawa M, Kinohira S, Ando A, <br />                                                           valuable source useful compound. However, molecular mechanism squalene <br /> Shimoyama M, Kato M, Fukuzawa H (2015) <br /> Accumulation Squalene Microalga                   biosynthesis microalgae largely unknown. We obtained sequences en- <br /> Chlamydomonas reinhardtii Genetic Modification         zymes involved squalene synthesis metabolism, squalene synthase (CrSQS) and <br /> of Squalene Synthase Squalene Epoxidase               squalene epoxidase (CrSQE), model green alga Chlamydomonas reinhardtii. <br /> Genes. PLoS ONE 10(3): e0120446. doi:10.1371/ <br />                                                           CrSQS functionally characterized expression Escherichia coli CrSQE by <br /> journal.pone.0120446 <br />                                                           complementation budding yeast erg1 mutant. Transient expression CrSQS and <br /> Academic Editor: Wei Ning Chen, Nanyang <br />                                                           CrSQE fused fluorescent proteins onion epidermal tissue suggested pro- <br /> Technological University, SINGAPORE <br />                                                           teins co-localized endoplasmic reticulum. CrSQS-overexpression increased the <br /> Received: September 17, 2014 <br />                                                           rate conversion 14C-labeled farnesylpyrophosphate squalene did lead <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Accepted: January 22, 2015                                over-accumulation squalene. Addition terbinafine caused accumulation squa- <br /> Published: March 12, 2015                                 lene suppression cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span>. On hand, CrSQE-knockdown lines, ex- <br /> Copyright: &#194;&#169; 2015 Kajikawa et al. This open         pression level CrSQE reduced 59&#226;&#8364;&#8220;76% wild-type cells, significant <br /> access article distributed terms         levels squalene (0.9&#226;&#8364;&#8220;1.1 &#206;&#188;g mg&#226;&#8364;&#8220;1 cell dry weight) accumulated growth inhibi- <br /> Creative Commons Attribution License, permits       tion. In co-transformation lines CrSQS-overexpression CrSQE-knockdown, the <br /> unrestricted use, distribution, reproduction any <br />                                                           level squalene increased significantly compared solitary CrSQE- <br /> medium, provided original author source are <br /> credited.                                                 knockdown lines. These results indicated partial knockdown CrSQE effective <br />                                                           strategy increase squalene production C. reinhardtii cells. <br /> Data Availability Statement: All sequences genes <br /> in study available Chlamydomonas <br /> reinhardtii genome database v5.5 (Phytozome v10; <br /> http://phytozome.jgi.doe.gov/pz/portal.html#!info? <br /> alias=Org_Creinhardtii). <br />  <br /> Funding: This study supported Advanced             Introduction <br /> Low Carbon Technology Research Development <br />                                                           Squalene important intermediate sterol biosynthesis variety organisms bac- <br /> (ALCA) Program (to H. F. M. Kato). The funders <br /> had role study design, data collection and <br />                                                           teria humans [1]. Squalene used natural antioxidant, adjuvant vaccines, die- <br /> analysis, decision publish, preparation      tary supplement skin moisturizer therapeutic, pharmacological cosmetic purposes <br /> manuscript.                                               [1,2]. The main source squalene shark liver oil, alternative sources desired order <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                                   1 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br /> Competing Interests: The authors declared   secure sustainable supply squalene conservation marine organisms [1]. In <br /> that competing interests exist.               microorganisms, squalene content shows wide variety depending species. For instance, <br />                                                  squalene contents Saccharomyces cerevisiae Torulaspora delbrueckii be <br />                                                  0.04 0.24 &#206;&#188;g mg&#226;&#710;&#8217;1 respectively, dry weight cells [3]. In microalgae Chlamydomonas rein- <br />                                                  hardtii (in study) Phaeodactylum tricornutum [4], squalene detected the <br />                                                  wild-type cells. In contrast, Aurantiochytrium mangrovei accumulates squalene (0.2&#226;&#8364;&#8220;0.4 &#206;&#188;g mg&#226;&#710;&#8217;1 <br />                                                  dry weight) [5,6]. The Botryococcus braunii Euglena reported accumulate squa- <br />                                                  lene cells [1]. Recently, Aurantiochytrium sp. strain 18W-13a reported accumu- <br />                                                  late huge squalene (171 &#206;&#188;g mg&#226;&#710;&#8217;1 dry weight) cells [7] expected be <br />                                                  useable industrial source useful compound. For accumulation squalene <br />                                                  controls squalene production microalgal cells, genetic modification genes involved <br />                                                  squalene metabolism required. However, transformation available the <br />                                                  microalgae including Aurantiochytrium sp. examples characteriza- <br />                                                  tion genes directly responsible squalene metabolism reported microalgae <br />                                                  [8,9]. <br />                                                     In land plants, squalene produced pathways, mevalonate (MVA) pathway and <br />                                                  methylerythritol phosphate (MEP) pathway [1]. In contrast, green algae the <br />                                                  MEP pathway [1,10,11]. In model green microalga C. reinhardtii, squalene predicted to <br />                                                  synthesized MEP pathway, green algae [12]. However, enzymes the <br />                                                  MEP pathway C. reinhardtii characterized yet. In C. reinhardtii, major ste- <br />                                                  rols, ergosterol 7-dehydroporiferasterol, produced squalene branched <br />                                                  pathways [13&#226;&#8364;&#8220;16]. In step pathway, squalene produced farnesyl diphos- <br />                                                  phate (FPP) squalene synthase (SQS) converted 2,3-oxidosqualene squalene epox- <br />                                                  idase (SQE). <br />                                                     SQS catalyzes reductive dimerization molecules FPP two-step reaction to <br />                                                  produce squalene. This reaction proceeds head-to-head coupling FPP molecules to <br />                                                  form squalene stable cyclopropylcarbinyl diphosphate intermediate [17]. SQS genes have <br />                                                  characterized organisms, yeast, rat, human land plants [1]. The <br />                                                  enzymatic activities SQS oil-producing microalgae, Botryococcus braunii [8] Aur- <br />                                                  antiochytrium sp. KRS101 [9], characterized vitro. However, no <br />                                                  reports vivo functional analysis SQS algae. On hand, SQE catalyzes <br />                                                  insertion oxygen atom C-C double bond form epoxide P450-type <br />                                                  reaction. SQE genes characterized organisms, yeast, rat, <br />                                                  human land plants [1], algae. <br />                                                     In order understand functional importance genes encoding enzymes directly re- <br />                                                  sponsible squalene synthesis metabolism squalene accumulation microalgal cells, <br />                                                  isolation characterization putative C. reinhardtii SQS SQE <br />                                                  genes. Transgenic studies using genes C. reinhardtii cells revealed partial knock- <br />                                                  (KD) CrSQE expression enhanced accumulation significant amounts squa- <br />                                                  lene growth inhibition. In contrast, overexpression CrSQS did influence the <br />                                                  squalene cells. <br />  <br />  <br />                                                  Results <br />                                                  Identification SQS SQE genes C. reinhardtii <br />                                                  Two genes, Cre03.g175250 Cre17.g734644, encoding putative enzymes involved sterol <br />                                                  metabolism predicted C. reinhardtii genome database v5.5 (Phytozome v10; http:// <br />                                                  phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Creinhardtii) [18,19]. The predicted <br />                                                  amino acid sequences showed high levels similarity SQS SQE enzymes other <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                   2 / 21 <br />                                                                            Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          organisms, respectively. Therefore, genes prospectively designated CrSQS and <br />                                          CrSQE, respectively. CrSQS shared 50% sequence identity Arabidopsis thaliana SQS1 <br />                                          (AtSQS1) [20&#226;&#8364;&#8220;22], 47% B. braunii SQS (BbSQS) [8], 41% human SQS (HsSQS) <br />                                          [17,23&#226;&#8364;&#8220;25], 34% S. cerevisiae SQS (ScERG9) [23] (S1A Fig.). CrSQE shared 51% se- <br />                                          quence identity A. thaliana SQE1 (AtSQE1) [26], 36% human SQE (HsSQE) <br />                                          [27], 34% rat SQE (RnSQE) [28] 33% S. cerevisiae SQE (ScERG1) [29,30] (S1B <br />                                          Fig.). Southern blotting analysis indicated genes present single-copy genes in <br />                                          C. reinhardtii (S2 Fig.). The SOSUI program (http://harrier.nagahama-i-bio.ac.jp/sosui/sosui_ <br />                                          submit.html) [31] predicted CrSQS protein putative transmembrane domains <br />                                          (TMDs) C-terminus (Phe-398 Glu-420 Gln-436 Leu-458) (S1A Fig.). Likewise, <br />                                          TMD N-terminus (Pro-29 Ala-51) TMDs C-terminus (Ala-424 to <br />                                          Phe-446 Leu-472 Phe-494) predicted CrSQE protein (S1B Fig.). The above <br />                                          results suggested CrSQS CrSQE genes encode membrane-bound SQS and <br />                                          SQE, respectively, involved squalene synthesis sterol metabolism C. reinhardtii. <br />  <br />                                          In vitro enzymatic activity recombinant CrSQS protein <br />                                          To confirm CrSQS encodes functional SQS, CrSQS expressed fusion protein <br />                                          His-tag C-terminus Escherichia coli. The sequence encoding C-terminal 64 <br />                                          amino acid residues, included hydrophobic transmembrane regions, removed, <br />                                          remaining 1,191-bp fragment encoding 397 amino acid residues (Met-1 Asp-397) <br />                                          cloned upstream 6&#195;&#8212;His-tag sequence pET-21b vector. A 46-kDa band, corre- <br />                                          sponded predicted size recombinant CrSQS protein, detected soluble <br />                                          protein fraction IPTG-induced E. coli cells harboring CrSQS-expression plasmid, not <br />                                          extracts cells harboring vector (Fig. 1A). The recombinant CrSQS protein <br />                                          purified soluble protein fractions Co2+-affinity (TALON) column chromatog- <br />                                          raphy (lane 3 Fig. 1A). <br />                                              To evaluate squalene synthase activity CrSQS, crude soluble extract purified <br />                                          CrSQS protein used enzyme assays presence (1&#226;&#8364;&#8220;14C) FPP, nicotinamide ade- <br />                                          dinucleotide phosphate (NADPH) Mg2+ cofactor. Normal-phase layer chro- <br />                                          matography (TLC) analysis performed separate reaction products. 14C-labeled <br />                                          squalene detected samples containing crude extract purified CrSQS protein (lanes <br />                                          4, 5 7, 8 Fig. 1B) sample cells containing vector (lanes 1&#226;&#8364;&#8220;3 <br />                                          Fig. 1B). These results indicated CrSQS protein SQS activity. When NADPH was <br />                                          removed, radiolabeled squalene detected sample (lane 3, 6 9 Fig. 1B). In- <br />                                          stead, weak signals, dehydrosqualene 12-hydroxysqualene [20,22], <br />                                          detected purified CrSQS incubated (1&#226;&#8364;&#8220;14C) FPP NADPH <br />                                          (lane 9 Fig. 1B). To characterize enzymatic activity, crude purified CrSQS <br />                                          protein incubated mixture containing (1&#226;&#8364;&#8220;14C) FPP, NADPH, ei- <br />                                          ther Mg2+ Mn2+ cofactor, reaction products separated normal re- <br />                                          verse-phase TLC (S3 Fig.). TLC analyses indicated CrSQS protein showed SQS activity in <br />                                          presence NADPH Mg2+ Mn2+ (lanes 1 4 S3 Fig.). <br />  <br />                                          Complementation analysis CrSQE yeast erg1 mutant <br />                                          To characterize enzymatic activity putative C. reinhardtii SQE, coding region the <br />                                          CrSQE gene amplified cDNAs C. reinhardtii wild-type C-9 cells RT-PCR. The <br />                                          product cloned yeast expression vector pAUR123, containing aureobasidin A-re- <br />                                          sistance gene AUR1-C selectable marker, control constitutive ADH1 pro- <br />                                          moter. The S. cerevisiae ScERG1 ORF cloned vector positive control. The <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                            3 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br />                                          Fig 1. Functional characterization CrSQS E. coli. A) SDS-polyacrylamide gel electrophoresis of <br />                                          crude extracts cells harboring vector pET-21b (lane 1), cells expressing CrSQS (lane 2) and <br />                                          purified CrSQS using TALON Metal affinity chromatography. B) Normal-phase thin-layer chromatogram of <br />                                          reaction products derived (1&#226;&#8364;&#8220;14C) farnesyl diphosphate (FPP). Crude extracts cells harboring <br />                                          vector (lanes 1&#226;&#8364;&#8220;3), cells expressing CrSQS (lanes 4&#226;&#8364;&#8220;6) purified CrSQS protein (lanes <br />                                          7&#226;&#8364;&#8220;9) assayed SQS activity using (1&#226;&#8364;&#8220;14C) FPP Mg2+ NADPH (lanes 1&#226;&#8364;&#8220;2, 4&#226;&#8364;&#8220;5, 7&#226;&#8364;&#8220;8 are <br />                                          experimental replicates) NADPH (lanes 3, 6, 9). Authentic (1&#226;&#8364;&#8220;14C) FPP loaded lane 10 a <br />                                          negative control. The positions origin, solvent authentic squalene indicated left. Open <br />                                          triangles right indicate position signals putative dehydrosqualene 12-hydroxysqualene in <br />                                          lane 9. <br />                                          doi:10.1371/journal.pone.0120446.g001 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                         4 / 21 <br />                                                                               Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br /> Fig 2. Complementation yeast erg1 mutant KLN1 CrSQE expression. Three independent lines harboring pAUR123-CrSQE (CrSQE-1 to <br /> CrSQE-3), yeast lines harboring pAUR123-ScERG1 (ScERG1-1 ScERG1-2), control line harboring pAUR123 vector (VC) were <br /> streaked YPD medium ergosterol aerobic conditions (A) ergosterol anaerobic conditions (B). <br /> doi:10.1371/journal.pone.0120446.g002 <br />  <br />                                            resultant expression plasmids vector introduced S. cerevisiae erg1 <br />                                            mutant KLN1 [30], corresponding transformants isolated aureobasidin A- <br />                                            containing selective Yeast extract-Peptone-Dextrose (YPD) medium. Three transformants har- <br />                                            boring CrSQE expression plasmid (CrSQE-1 CrSQE-3) grew YPD medium without <br />                                            ergosterol aerobic conditions, did positive-control transformants, contrast to <br />                                            vector control, did grow (Fig. 2A). All transformants grew YPD me- <br />                                            dium ergosterol anaerobic conditions (Fig. 2B). Ergosterol detected the <br />                                            complementation lines expressing CrSQE ScERG1, large amounts squalene <br />                                            accumulated CrSQE-expressing line (S4 Fig.). These results indicated ergos- <br />                                            terol biosynthetic pathway yeast erg1 mutant reconstituted heterologous ex- <br />                                            pression CrSQE, CrSQE protein functions SQE enzyme. <br />  <br />  <br />                                            Subcellular localization CrSQS CrSQE <br />                                            To examine subcellular localization CrSQS, expressed CrSQS C-terminal two <br />                                            TMDs green fluorescent protein (GFP)-fusion proteins onion epidermal cells (Fig. 3A). <br />                                            Each expression plasmid co-transformed expression plasmid mCherry- <br />                                            AtSQS1-TMD endoplasmic reticulum (ER)-localized marker [22]. The fluorescence im- <br />                                            ages GFP-CrSQS GFP-CrSQSTMD proteins completely coincident that <br />                                            mCherry-AtSQS1-TMD protein onion epidermal cells (Fig. 3B 3C), suggesting that <br />                                            CrSQS ER-localized protein C-terminal TMDs function ER-localization signal <br />                                            similarly C-terminal TMD AtSQS1 protein [21,22]. <br />                                               To analyze subcellular localization CrSQE protein, mCherry protein fused N ter- <br />                                            minus N-terminal 100 residues CrSQE containing N-terminal TMD C <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                5 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br />                                          Fig 3. Subcellular localization CrSQS CrSQE onion epidermal cells. A) Schematic diagrams of <br />                                          expression plasmids GFP- mCherry-fusion proteins. pGEP-CrSQS: GFP fused C terminus <br />                                          full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> CrSQS. pGFP-CrSQSTMD: GFP fused C terminus C-terminal 118 residues of <br />                                          CrSQS including predicted transmembrane domains (TMDs). pTMD-mCherry-CrSQE: mCherry was <br />                                          fused N terminus N-terminal 100 residues CrSQE including predicted TMD C <br />                                          terminus C-terminal 423 residues CrSQE including predicted TMDs. pmCherry-AtSQS1TMD: <br />                                          mCherry fused C-terminus C-terminal 67 residues A. thaliana SQS1 including predicted <br />                                          TMD, shown localized endoplasmic reticulum [22]. 35S, cauliflower mosaic virus <br />                                          35S promoter; NOS, terminator nopaline synthase. B) The fusion proteins expressed transiently in <br />                                          onion epidermal cells particle bombardment. Fluorescence observed using laser scanning confocal <br />                                          microscope. Bars = 50 &#206;&#188;M <br />                                          doi:10.1371/journal.pone.0120446.g003 <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                     6 / 21 <br />                                                                            Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          terminus C-terminal 423 residues CrSQE containing C-terminal TMDs <br />                                          (TMD-mCherry-CrSQE protein) expressed GFP-CrSQSTMD protein onion epi- <br />                                          dermal cells. The fluorescence images TMD-mCherry-CrSQE protein completely coin- <br />                                          cident GFP-CrSQSTMD protein onion epidermal cells (Fig. 3D), suggesting <br />                                          CrSQE ER-localized. <br />  <br />  <br />                                          Overexpression CrSQS gene C. reinhardtii <br />                                          To characterize vivo function CrSQS squalene sterol biosynthetic pathway, <br />                                          CrSQS overexpressed C. reinhardtii cells. A plasmid overexpression CrSQS <br />                                          (CrSQS-ox), comprising genomic sequence CrSQS coding region regulation of <br />                                          constitutive PsaD promoter expressed UVM4 mutant strain, suitable <br />                                          stable effective expression exogenous genes [32]. Two lines transformants, <br />                                          CrSQS-ox-2 CrSQS-ox-5, showed positive signals 57 kDa immunoblotting <br />                                          analysis using anti-HA antibody, used analysis (Fig. 4A). Total expression <br />                                          level CrSQS endogenous exogenous loci estimated qRT-PCR (Fig. 4B). In <br />                                          CrSQS-ox lines, expression levels CrSQS increased 3- 4-fold compared that <br />                                          parental line. Squalene synthesizing activity (1&#226;&#8364;&#8220;14C) FPP 24 h CrSQS-ox <br />                                          cell cultures 1.6- 1.7-fold higher parental cells (Fig. 4C). <br />                                              However, squalene detected CrSQS-ox lines parental line UVM4. The <br />                                          amounts major sterols, ergosterol putative 7-dehydroporiferasterol, CrSQS-ox <br />                                          lines, changed significantly UVM4 cells (Fig. 4D). These results sug- <br />                                          gested squalene did accumulate significantly CrSQS-ox cells, conversion of <br />                                          14 <br />                                             C-labeled FPP squalene enhanced overexpression CrSQS. <br />  <br />  <br />                                          Effect SQE inhibitor terbinafine cell viability biosynthesis of <br />                                          squalene sterols <br />                                          To verify contribution SQE activity downstream sterol biosynthesis influence <br />                                          SQE inhibition accumulation squalene, dose-dependent inhibition SQE activity <br />                                          evaluated addition 10 150 mg L&#226;&#8364;&#8220;1 terbinafine (TBF) [6,33&#226;&#8364;&#8220;34] cultures of <br />                                          UVM4 cells. The squalene increased TBF treatment dose dependently, and <br />                                          reached 18.7 nmol mg&#226;&#8364;&#8220;1 dry weight (DW) presence 50 mg L&#226;&#8364;&#8220;1 TBF (Fig. 5A). In con- <br />                                          trast, total amounts ergosterol putative 7-dehydroporiferasterol, down- <br />                                          stream metabolites SQE reaction, reduced 72% cells treated 100 mg L&#226;&#8364;&#8220;1 <br />                                          TBF compared TBF-untreated control (Fig. 5A). With higher concentration TBF of <br />                                          150 mg L&#226;&#8364;&#8220;1, cells bleached (Fig. 5B 5C) squalene sterols detected in <br />                                          cell extracts (Fig. 5A), indicating SQE activity indispensable sterol biosynthesis <br />                                          <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>. <br />  <br />  <br />                                          Knockdown CrSQE UVM4 line <br />                                          To determine functional importance CrSQE squalene sterol biosynthesis, ex- <br />                                          pression CrSQE suppressed UVM4 cells expression artificial microRNA <br />                                          (amiRNA) [35]. Twenty-two 30 transformants carry CrSQE-KD plas- <br />                                          mid genomic PCR. The transcript level CrSQE significantly reduced 59&#226;&#8364;&#8220;76% in <br />                                          independent CrSQE-KD lines compared parental UVM4 line (Fig. 6A). <br />                                          In lines, 0.9&#226;&#8364;&#8220;1.1 &#206;&#188;g (2.3&#226;&#8364;&#8220;2.7 nmol) squalene mg&#226;&#8364;&#8220;1 cell DW accumulated (Fig. 6B and <br />                                          S5 Fig.). Unexpectedly, squalene CrSQE-KD cells 12&#226;&#8364;&#8220;15% in <br />                                          parental UVM4 cells treated 50 mg L&#226;&#8364;&#8220;1 TBF. The amounts ergosterol putative <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                             7 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br />                                          Fig 4. Overexpression CrSQS gene C. reinhardtii line UVM4. A) Immunoblotting analysis with <br />                                          antibodies HA epitope tag &#206;&#177;-tubulin transgenic UVM4 lines harboring CrSQS-expression <br />                                          plasmid. B) qRT-PCR analysis CrSQS gene expression parental line (PL) UVM4 two <br />                                          overexpression lines, CrSQS-ox-2 CrSQS-ox-5. Expression gene normalized CRY1, <br />                                          encodes ribosomal protein S14 [43]. C) Incorporation (1&#226;&#8364;&#8220;14C) FPP squalene line. D) <br />                                          Content ergosterol (grey bars) putative 7-dehydroporiferasterol (open bars) line. Asterisks <br />                                          bars indicate significant differences (**p &lt; 0.01). DW; cell dry weight. ND; detected. Data all <br />                                          experiments indicate mean value &#194;&#177; SD biological replicates. <br />                                          doi:10.1371/journal.pone.0120446.g004 <br />  <br />                                          7-dehydroporiferasterol unchanged CrSQE-KD lines parental UVM4 <br />                                          line (Fig. 6C), growth inhibition observed CrSQE-KD lines (Fig. 6D). <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                         8 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br />                                          Fig 5. Effects terbinafine (TBF) squalene sterol biosynthesis cell viability. A) Contents of <br />                                          ergosterol (grey bars), putative 7-dehydroporiferasterol (open bars) squalene (hatched bars) cells <br />                                          treated different concentrations TBF. DW; cell dry weight. B) Images cultures treated TBF. C) <br />                                          Chlorophyll content cells treated TBF. Data experiments mean value &#194;&#177; SD three <br />                                          biological replicates. <br />                                          doi:10.1371/journal.pone.0120446.g005 <br />  <br />                                             To evaluate effects CrSQE-KD gene expression involved MEP pathway, <br />                                          mRNA levels 11 genes putatively encoding: 1-deoxy-D-xylulose 5-phosphate synthase <br />                                          (CrDXS), 1-deoxy-D-xylulose 5-phosphate reductoisomerase (CrDXR), 2-C-methyl-D-ery- <br />                                          thriol 4-phosphate cytidyltransferase (CrCMS), 4-diphosphocytidyl-2-C-methyl-D-erythritol <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                       9 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br /> Fig 6. Knockdown CrSQE gene parental line (PL) UVM4. PL UVM4 knockdown lines (SQEKD-1 SQEKD-5) analyzed. A) qRT-PCR <br /> analysis CrSQE gene expression gene normalized CRY1 gene. Asterisks bars indicate significant differences <br /> (*p &lt; 0.05, **p &lt; 0.01). B) Squalene content measured line. DW; cell dry weight. ND; detected. C) Contents ergosterol (grey bars), <br /> putative 7-dehydroporiferasterol (open bars) squalene (hatched bars) measured line. D) Doubling time line calculated by <br /> measuring absorbance 730 nm. Data experiments indicate mean value &#194;&#177; SD biological replicates. <br /> doi:10.1371/journal.pone.0120446.g006 <br />  <br />                                              2-phosphate kinase (CrCMK), 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase <br />                                              (CrMECPS), 4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase (CrHDS), isopentenyl <br />                                              diphosphate synthase (CrIDS), isopentenyl diphosphate isomerase (CrIDI), geranyl pyrophos- <br />                                              phate synthase (CrGSP), farnesyl diphosphate synthase (CrFPS) CrSQS annotated the <br />                                              Phytozome database [18,19] (S1 Table), measured qRT-PCR. Expression levels only <br />                                              CrCMK CrSQE-KD lines increased 2.2- 4.9-fold compared pa- <br />                                              rental line (S6A Fig.). In contrast, similar changes mRNA levels 8 genes, except <br />                                              CrIDS CrIDI, seen CrSQE-KD lines UVM4 parental line <br />                                              (S6B Fig.). Amplification cDNAs CrIDS CrIDI parental line the <br />                                              CrSQE-KD lines weak analyze. These results suggested CrSQE-KD up-regulat- <br />                                              ed expression putative CMK gene mRNA level. <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                      10 / 21 <br />                                                                           Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          Screening CrSQS-ox&#226;&#8364;&#8220;CrSQE-KD UVM4 lines accumulation of <br />                                          squalene sterols <br />                                          To evaluate synergistic effect squalene accumulation CrSQS-ox CrSQE-KD a <br />                                          single line, transformed CrSQE-KD-5 CrSQS-ox plasmid. The CrSQS-ox and <br />                                          CrSQE-KD lines screened immunoblotting analysis using anti-HA-tag antibody <br />                                          (Fig. 7A). Positive signals detected 3 10 transformants, designated CrSQS-ox- <br />                                          CrSQE-KD-1, CrSQS-ox-CrSQE-KD-5 CrSQS-ox-CrSQE-KD-10. As expected, expres- <br />                                          sion levels CrSQS strains 5.7- 7.3-fold higher UVM4 and <br />                                          CrSQE-KD-5 lines (Fig. 7B). The expression levels CrSQE similar the <br />                                          CrSQE-KD-5 line reduced 68&#226;&#8364;&#8220;82% compared UVM4 line <br />                                          (Fig. 7C). However, squalene changed CrSQE-KD-5 <br />                                          line (Fig. 7D), indicating CrSQS-ox did enhance production accumulation <br />                                          squalene. <br />  <br />                                          Discussion <br />                                          We isolated CrSQS CrSQE, responsible squalene synthesis metabolism, <br />                                          model green alga C. reinhardtii, demonstrated enzymatic activities. Further- <br />                                          more, analyses transgenic C. reinhardtii indicated squalene, undetectable in <br />                                          parental strain, accumulated approximately 0.1% (w/w) cell DW 59&#226;&#8364;&#8220;76% reduc- <br />                                          tion CrSQE expression CrSQE-KD lines. As levels sterols, down- <br />                                          stream products squalene metabolic pathway, did decrease significantly by <br />                                          suppression CrSQE expression, growth CrSQE-KD lines did differ significantly <br />                                          parental strain. These results explained following hypotheses; <br />                                          (1) sufficient 2,3-oxidesqualene sterol production supplied re- <br />                                          maining CrSQE protein CrSQE-KD lines; (2) gene(s) encoding SQE could <br />                                          exist addition CrSQE C. reinhardtii genome, expression sup- <br />                                          pressed CrSQE-KD lines; (3) C. reinhardtii unknown bypass pathways for <br />                                          SQE reaction sterol supply. Based results treatment SQE inhibitor on <br />                                          parental UVM4 strain, examined hypotheses. Treatment TBF decreased the <br />                                          sterols increased squalene accumulation dose-dependent manner. The <br />                                          squalene UVM4 cells treated 50 mg L&#226;&#8364;&#8220;1 TBF 7- 8-fold larger that <br />                                          CrSQE-KD lines. The chlorophyll content cells decreased dose dependently TBF <br />                                          treatment, cells finally bleached presence 150 mg L&#226;&#8364;&#8220;1 TBF. These results in- <br />                                          dicated TBF-sensitive SQE activity indispensable sterol biosynthesis cell sur- <br />                                          vival algal cells. Based results fact CrSQE high levels sequence <br />                                          similarity sequences TBF-sensitive SQEs, ScERG1 [34], considered <br />                                          CrSQE-KD lines retained sufficient CrSQE activity supply 2,3-oxidesqualene for <br />                                          downstream sterol production, completely exclude second third <br />                                          hypotheses. For accumulation additional amounts squalene, inducible-promoter-mediat- <br />                                          ed transient KD CrSQE effective. <br />                                             On hand, tried increase squalene cells overexpression of <br />                                          CrSQS. However, squalene CrSQS-ox lines increased compared with <br />                                          parental cells, amounts sterols unchanged CrSQS-ox cells and <br />                                          parental cells. Uptake experiments 14C-labeled FPP suggested conversion FPP <br />                                          squalene enhanced CrSQS-ox lines. Therefore, following possibilities be <br />                                          considered reason CrSQS-ox did influence squalene accumulation; (1) the <br />                                          FPP supplied MEP pathway accumulation squa- <br />                                          lene, (2) excess squalene converted rapidly sterol intermediate(s) structure <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                          11 / 21 <br />                                                                                   Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />  <br />                                          Fig 7. Overexpression CrSQS gene knockdown CrSQE gene parental line (PL) UVM4. A) <br />                                          Immunoblotting analysis antibodies HA epitope tag &#206;&#177;-tubulin CrSQE-KD-5 lines <br />                                          harboring CrSQS-expression plasmid. qRT-PCR analysis CrSQS (B) CrSQE (C) genes each <br />                                          line. Expression gene normalized CRY1 gene. D) Squalene content line. <br />                                          DW; cell dry weight. Data experiments indicate mean value &#194;&#177; SD biological replicates. <br />                                          doi:10.1371/journal.pone.0120446.g007 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                    12 / 21 <br />                                                                             Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          suited detection gas chromatography analysis, (3) relatively insufficient in- <br />                                          crease accumulated level CrSQS protein, detected significant level <br />                                          increase CrSQS mRNA qRT-PCR, (4) insufficient activity transgenic CrSQS <br />                                          protein HA-tag C-terminal end, interfere anchoring ER mem- <br />                                          brane, (5) Km value higher concentration intracellular FPP, <br />                                          (6) types SQS proteins operating generate squalene vivo, in- <br />                                          troduced CrSQS shown SQS activity vitro, (cf. Botryococcene synthases <br />                                          showing high level sequence similarity SQS did SQS activity B. braunii <br />                                          [36]. In order examine possibilities, investigation necessary reveal rea- <br />                                          son overexpression CrSQS effective squalene accumulation. <br />                                              Heterologous expression analysis onion epidermal cells suggested CrSQS CrSQE <br />                                          ER-localized proteins. A. thaliana AtSQS1 reported ER-localized pro- <br />                                          tein [22]. A predicted TMD C-terminus AtSQS1 necessary localization ER <br />                                          [22]. Fungal animal SQS proteins predicted C-terminal TMDs, are <br />                                          thought anchor enzyme ER membrane [23]. CrSQS predicted TMDs at <br />                                          C-terminus. A fusion protein C-terminal region including TMDs GFP <br />                                          showed ER-localized <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, indicating C-terminal TMDs CrSQS play im- <br />                                          portant role ER localization. In contrast SQS proteins TMD C- <br />                                          terminus, TMDs predicted CrSQS sequence SOSUI program. The region <br />                                          TMD predicted CrSQS predicted TMD SQS se- <br />                                          quences. Because region including TMDs cloned pGFP-CrSQSTMD plas- <br />                                          mid study, examination necessary reveal effect TMD ER <br />                                          localization. These results suggested TMDs CrSQE play important role ER lo- <br />                                          calization, case CrSQS. The subcellular ER- lipid droplet-localization bud- <br />                                          ding yeast SQE (ScERG1) reported means expression GFP fusion protein <br />                                          [37] immunostaining [38]. Two TMDs N- C-termini predicted ScERG1 <br />                                          sequence SOSUI program. However, roles TMDs subcellular localiza- <br />                                          tion SQE clarified organisms. In land plants, sterol biosynthesis <br />                                          downstream squalene predicted occurred ER [1]. These results suggested that <br />                                          steps catalyzed CrSQS CrSQE located ER microalgal cells. <br />                                              The mRNA level putative CrCMK gene upregulated CrSQE-KD lines. <br />                                          CMK enzyme functions middle step MEP pathway converts 4- <br />                                          diphosphocytidyl-2C-methyl-D-erythritol 4-diphosphocytidyl-2-C-methyl-D-erythritol 2- <br />                                          phosphate. Only gene showing sequence similarity characterized CMKs organ- <br />                                          isms annotated C. reinhardtii genome database Phytozome (CrCMK; Cre16. <br />                                          g679669) [18,19]. In A. thaliana, regulation enzymes MEP pathway reported <br />                                          [39]. In mutants DXR, CMS, HDS HDR, protein level DXS, catalyzes <br />                                          step MEP pathway, higher wild type. In contrast, the <br />                                          mRNA level DXS mutants lower wild type, suggesting pres- <br />                                          ence post-transcriptional regulation DXS protein [39]. However, mRNA pro- <br />                                          tein levels CMK mutants lower wild type. These results <br />                                          indicated KD CrSQE generated change gene expression related upstream <br />                                          MEP pathway. In CrSQE-KD lines, similar changes mRNA levels observed other <br />                                          genes MEP pathway, regulated post-transcriptionally. <br />                                              Based results study transgenic microalgae, conclude partial sup- <br />                                          pression CrSQE expression effective strategy increase squalene accumulation. A fea- <br />                                          sible strategy accumulate higher levels squalene cells increase of <br />                                          FPP supplied enhancement MEP pathway overexpression CrDXS, CrDXR, <br />                                          CrHDS, encode candidates rate-limiting enzymes land plants [10], other <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                            13 / 21 <br />                                                                           Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          genes involved MEP pathway background CrSQE-KD and/or CrSQS-ox <br />                                          transgenic lines. <br />  <br />                                          Material Methods <br />                                          Strains culture conditions <br />                                          For isolation genomic DNA total RNA C. reinhardtii, wild-type C-9 strain (mt&#226;&#8364;&#8220;) <br />                                          grown 2 days 50 mL Tris-Acetate-Phosphate (TAP) medium [40] 25&#194;&#176;C con- <br />                                          tinuous light illumination 50 &#206;&#188;mol photons m&#226;&#8364;&#8220;2 s&#226;&#8364;&#8220;1 orbital shaker 100 rpm. A mu- <br />                                          tant strain UVM4 [32] efficiently expresses transgenes used functional analysis of <br />                                          CrSQS CrSQE. For metabolic expression analysis, UVM4 transgenic lines were <br />                                          cultured 2 days conditions C-9 culture. For inhibitor studies, the <br />                                          UVM4 cells cultured 1 day 50 mL TAP liquid medium OD730 0.1, SQE-in- <br />                                          hibitor, TBF (Sigma-Aldrich, St. Louis, MO), added culture final concentration <br />                                          0, 10, 100, 150, 300 mM, respectively, incubated 1 day. <br />  <br />                                          Quantitative (q)RT-PCR <br />                                          qRT-PCR performed using SYBR Premix Ex Taq GC (Takara Bio, Shiga, Japan) a <br />                                          LightCycler 480 Instrument (Roche, Basel, Switzerland) described previously [41]. CRY1 en- <br />                                          coding ribosomal protein S14 [42] used internal control [43]. The primers used for <br />                                          qRT-PCR listed S1 Table. <br />  <br />                                          Southern blotting analysis <br />                                          Genomic DNA extracted C-9 cells using DNeasy Plant Mini Kit (Qiagen, KJ Venlo, <br />                                          Netherlands). Five &#206;&#188;g genomic DNA digested restriction enzymes separated by <br />                                          electrophoresis, transferred nylon membrane (Biodyne B; Pall Corporation, Port Washing- <br />                                          ton, NY), hybridized using standard protocols [44]. For detection CrSQS CrSQE, <br />                                          coding sequences amplified PCR primers construction expres- <br />                                          sion vectors C. reinhardtii yeast, respectively. The PCR products labeled with <br />                                          (&#206;&#177;-32P) dCTP using Ex-Taq DNA polymerase (Takara Bio) used probes. Hybridization <br />                                          performed using ExpressHyb hybridization solution (Clontech, Mountain View, CA) in <br />                                          accordance manufacturer&#226;&#8364;&#8482;s instructions. <br />  <br />                                          Immunoblotting analysis <br />                                          Immunoblotting analysis performed described previously [41,45]. The following anti- <br />                                          bodies used: horseradish peroxidase (HRP)-linked monoclonal anti-HA antibody (1: <br />                                          2,000 dilution; Roche, clone 3F10) anti-&#206;&#177;-tubulin antibody (1:5,000 dilution; Sigma-Al- <br />                                          drich, clone B-5-1-2). HRP-conjugated goat anti-mouse IgG antibody (1:10,000 dilution; GE <br />                                          Healthcare, Buckinghamshire, United Kingdom) detect &#206;&#177;-tubulin used secondary <br />                                          antibody. Tris-buffered saline 0.05% Triton X-100 used dilution antibodies. <br />  <br />                                          Expression recombinant CrSQS protein Escherichia coli <br />                                          A partial coding region CrSQS cDNA (+1 +1191) excluding putative TMD ampli- <br />                                          fied using primer set CrSQS-pET-F CrSQS-pET-R. cDNAs C-9 cells cultured for <br />                                          2 days described used template. Sequences primers plasmid con- <br />                                          struction transgenic analysis study listed S2 Table. PCR carried by <br />                                          PrimeScript GXL Polymerase (Takara Bio). The resultant PCR product cloned pET- <br />                                          21b vector (Merck Millipore, Darmstadt, Germany) using In-Fusion HD Cloning kit <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                          14 / 21 <br />                                                                           Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          (Clontech). The recombinant His-tagged CrSQS protein induced E. coli strain BL21 <br />                                          (DE3) (Life Technologies, Carlsbad, CA) addition 0.5 mM isopropylthio-&#195;&#376;-galacto- <br />                                          22&#194;&#176;C 6 h. The recombinant protein extracted anaerobic conditions using <br />                                          BugBuster Master Mix protein extraction reagents (Merck Millipore) purified using a <br />                                          TALON Metal Affinity Resin (Clontech) accordance manufacturer&#226;&#8364;&#8482;s instruc- <br />                                          tions. Eluted fractions containing recombinant protein combined dialyzed against <br />                                          phosphate-buffered saline buffer (pH 7.4) containing 100 mM sucrose, 5 mM dithiothreitol <br />                                          (DTT), Complete Protease Inhibitor Cocktail (Roche) using Slide-A-Lyzer Dialysis Cas- <br />                                          sette 3.5K (Thermo Fisher Scientific, Waltham, MA). The crude cell lysate purified and <br />                                          CrSQS protein used enzyme assay. BL21 (DE3) cells harboring pET-21b vec- <br />                                          tor cultured induction condition, cell lysates used control. <br />  <br />  <br />  <br />                                          Assay CrSQS activity <br />                                          The enzymatic activity recombinant CrSQS protein measured described previous- <br />                                          ly [22,46] modifications. The reaction mixture contained total volume <br />                                          100 &#206;&#188;L: 25 &#206;&#188;M (1&#226;&#8364;&#8220;14C) FPP (1.2 kBq; American Radiolabeled Chemicals, St. Louis, MO), <br />                                          2.5 mM NADPH (Sigma-Aldrich), 50 mM glucose-6-phosphate (Sigma-Aldrich), 2U glucose- <br />                                          6-phosphate dehydrogenase (Sigma-Aldrich), 5 mM MgCl2 (or 2 mM MnCl2), 10 mM DTT, <br />                                          50 mM potassium phosphate (pH 7.4), 1 mg mL&#226;&#8364;&#8220;1 bovine serum albumin 15 &#206;&#188;g purified <br />                                          CrSQS (or 15 &#206;&#188;g crude cell lysate). The reaction mixture incubated 30&#194;&#176;C 1 h, and <br />                                          reaction stopped addition 10 &#206;&#188;L solution containing 0.4% v/v squalene, <br />                                          1 M Na2EDTA (pH 9.2), 1% v/v ethanol. The mixture quickly placed ice bath <br />                                          used directly TLC analysis. For normal-phase TLC analysis reaction products, <br />                                          10 &#206;&#188;L reaction mixture applied silica gel 60G F254 plate (Merck Millipore). <br />                                          The plates developed cyclohexane:ethyl acetate (9:1, v/v). For reverse-phase TLC <br />                                          analysis, 20 &#206;&#188;L reaction mixture applied silica gel 60 RP-18 F254S plate (Merck <br />                                          Millipore), developed acetone:water (19:1, v/v). After development, plates <br />                                          exposed analyzed using bio-image analyzer, Typhoon FLA7000 (GE Healthcare). <br />                                          Squalene standard (Sigma-Aldrich) visualized UV irradiation. <br />  <br />  <br />  <br />                                          Expression CrSQE yeast KLN1 mutant <br />                                          Functional identification CrSQE achieved heterologous expression yeast <br />                                          ScERG1-defective mutant KLN1 (MAT&#206;&#177;, erg1::URA3, leu2, ura3, trp) [30]. The coding region <br />                                          CrSQE cDNA amplified using primer set CrSQE-pAUR123-F CrSQE- <br />                                          pAUR123-R. cDNAs C-9 cells cultured 2 days described used the <br />                                          template. PrimeScript GXL Polymerase (Takara Bio) used PCR. As positive control, <br />                                          ScERG1 gene amplified using primer set ScERG1-pAUR123-F ScERG1-pAUR123- <br />                                          R. Genomic DNA yeast wild-type InvSc1 strain (Life Technologies) used tem- <br />                                          plate. Each PCR product cloned pAUR123 vector (Takara Bio). The resultant con- <br />                                          structs, pAUR123-CrSQE pAUR123-ScERG1, used transform KLN1 mutant <br />                                          lithium acetate method [47]. Transformants selected YPD agar medium con- <br />                                          taining 1% yeast extract, 2% peptone, 2% dextrose 2% agar 0.5 &#206;&#188;g mL&#226;&#8364;&#8220;1 aureobasidin <br />                                          A (Takara Bio). Anaerobic conditions achieved culturing yeast transformants an <br />                                          anaerobic jar AnaeroPack-Anaero (Mitsubishi Gas Chemical, Tokyo, Japan). Ergosterol <br />                                          dissolved Tween-80:ethanol (1:1, v/v) supplied YPD medium final concentra- <br />                                          tion 20 &#206;&#188;g mL&#226;&#8364;&#8220;1. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                          15 / 21 <br />                                                                            Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          Expression GFP-CrSQS mCherry-CrSQE fusion proteins onion <br />                                          epidermal cells <br />                                          A PCR fusion-based approach applied create reporter gene constructions transient <br />                                          expression analysis onion epidermal cells. All PCR reactions using Prime- <br />                                          Script GXL Polymerase (Takara Bio). Firstly, sequences encoding full-length CrSQS <br />                                          CrSQS-TMD cDNAs amplified using cDNAs C-9 cells cultured 2 days de- <br />                                          scribed templates primer sets sGFP-CrSQS-F CrSQS-pBI221-R full-length <br />                                          CrSQS cDNA, sGFP-CrSQSTMD-F CrSQS-pBI221-R CrSQS-TMD cDNA. <br />                                          The sequences encoding 50 - 30 -partial fragments CrSQE amplified using <br />                                          pAUR123-CrSQE plasmid template primer set CrSQE-pBI221-F SQE300- <br />                                          mCherry-R 50 -partial fragment mCherry-CrSQE301-F CrSQE-pBI221-R for <br />                                          30 -partial fragment. The sequence encoding AtSQS1TMD [22] amplified using the <br />                                          primers mCherry-AtSQS1TMD-F AtSQS1-pBI221-R cDNAs seedlings wild- <br />                                          type A. thaliana Col-0 template. Translational start stop codons shown bold let- <br />                                          ters (S2 Table). <br />                                             Secondly, coding sequence GFP gene amplified using primer set sGFP- <br />                                          pBI221-F sGFP-CrSQS-R fusion full-length CrSQS fragment sGFP- <br />                                          pBI221-F sGFP-CrSQSTM-R fusion CrSQS-TMD fragment. The coding se- <br />                                          quence mCherry gene amplified using primers SQE300-mCherry-F mCherry- <br />                                          SQE301-R fusion 50 - 30 -partial fragments CrSQE cDNA, sGFP-pBI221- <br />                                          F mCherry-AtSQS1TM-R fusion AtSQS1-TMD cDNA fragment. <br />                                             Finally, amplicons construct mixed used templates fuse each <br />                                          PCR primer pairs sGFP-pBI221-F CrSQS-pBI221-R fusion GFP <br />                                          full-length CrSQS CrSQS-TMD fragments; sGFP-pBI221-F CrSQE-pBI221-R <br />                                          fusion mCherry 50 - 30 -partial fragments CrSQE; sGFP-pBI221-F and <br />                                          AtSQS1-pBI221-R fusion mCherry AtSQS1-TMD fragment. The resultant fused <br />                                          fragments cloned XbaI SacI sites pBI221 expression vector (Clontech) <br />                                          using In-Fusion cloning kit (Clontech). The resultant plasmids transformed into <br />                                          onion epidermal cells particle bombardment using PDS-1000/He Biolistic Particle Deliv- <br />                                          ery System (Bio-Rad, Hercules, CA) equipped 1,100 psi rupture disks. After incubation in <br />                                          dark 22&#194;&#176;C 24 h, fluorescence signals reporter protein observed using a <br />                                          Leica TCS SP8 laser scanning confocal microscope (Leica, Solms, Germany) excitation at <br />                                          488 nm GFP 552 nm mCherry fluorescence. <br />  <br />  <br />  <br />                                          Transgenic analysis C. reinhardtii <br />                                          For overexpression CrSQS, coding sequence amplified using genomic DNA C-9 <br />                                          cells template primer set CrSQSox-F CrSQSox-R. The resultant PCR product was <br />                                          cloned PsaD promoter [48] HA-tag pGenD-cHA-Hyg3 plasmid, which <br />                                          constructed pGenD-cHA plasmid [49] replacement aphVIII expression <br />                                          cassette aph7&quot; expression cassette derived pHyg3 plasmid [50]. <br />                                             For amiRNA-mediated KD CrSQE, pairs oligonucleotides, CrSQEKD-F and <br />                                          CrSQEKD-R, designed described previously [35] using WMD3-Web MicorRNA <br />                                          Designer site (http://wmd3.weigelworld.org/). These oligonucleotides annealed and <br />                                          cloned pChlamiRNA3-int vector [35]. The resultant constructs transformed the <br />                                          parental line UVM4 electroporation method [51]. Transformants screened PCR <br />                                          using primers PsaD-F CrSQSox-02R CrSQS-ox lines, AmiRNAprec pChla- <br />                                          miRNA3-R CrSQE-KD lines. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                           16 / 21 <br />                                                                            Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          Measurements squalene sterol content <br />                                          Squalene sterols extracted C. reinhardtii cells described previously [14,15] <br />                                          slight modifications. C. reinhardtii cells cultured 50 mL TAP medium 2 days (ap- <br />                                          proximately 1&#195;&#8212;107 cells mL&#226;&#8364;&#8220;1) dried mixed 20 mL 1 mg mL&#226;&#8364;&#8220;1 cholesterol <br />                                          (Sigma-Aldrich) internal control saponified 2 mL 10% KOH containing 50% <br />                                          methanol 30 min sonication. After extraction volume hexane, sol- <br />                                          vent evaporated residue dissolved 20 &#206;&#188;L chloroform. Then, 20 &#206;&#188;L silylat- <br />                                          ing reagent, N,O-bis(trimethylsilyl)trifluoroacetamide (Sigma-Aldrich) added lipid <br />                                          samples incubated 80&#194;&#176;C 30 min. Derivatized samples increased volume of <br />                                          40 &#206;&#188;L chloroform. A 2 &#206;&#188;L aliquot solution analyzed using gas chromatogra- <br />                                          phy (GC-4000; GL Science, Tokyo, Japan) equipped InertCap 5 capillary column <br />                                          (30 m &#195;&#8212; 0.25 mm, film thickness 0.25 mm, GL Science) gas chromatography-mass spec- <br />                                          trometry (GCMS-QP2010 Ultra; Shimadzu, Kyoto, Japan) equipped DB-5MS capillary <br />                                          column (30 m &#195;&#8212; 0.25 mm, film thickness 0.25 mm, Agilent Technologies, Santa Clara, CA), <br />                                          carrier gas: He (1 mL min&#226;&#8364;&#8220;1), oven temperature: 150&#226;&#8364;&#8220;300&#194;&#176;C (increase rate 20&#194;&#176;C min&#226;&#8364;&#8220;1). The <br />                                          ionization voltage 70 eV scan range 40&#226;&#8364;&#8220;500 Da GC-MS analysis. The squalene <br />                                          ergosterol contents calculated ratios peak areas standard <br />                                          (Sigma-Aldrich). The second largest peak endogenous sterols C. reinhardtii <br />                                          wild-type cells estimated putative 7-dehydroporiferasterol compared gas-chromato- <br />                                          gram pattern (S5A Fig.) mass fragmentation pattern electron ionization (EI) GC-MS <br />                                          analysis (S5B Fig.) reference [15]. The content putative 7-dehydroporifer- <br />                                          asterol estimated based standard curve authentic ergosterol. <br />  <br />  <br />                                          Squalene-synthesizing activity 14C-FPP <br />                                          After 2-days incubation, cells collected centrifugation 600 g 5 min, ob- <br />                                          tained total 3&#195;&#8212;108 cells suspended new TAP medium (3 mL) 30 mL Erlenmeyer <br />                                          flask fitted glass tube contained piece folded filter paper impregnated with <br />                                          0.1 mL 20% KOH center collect 14CO2, incubated 7.4 kBq of <br />                                          (1&#226;&#8364;&#8220;14C) FPP (2.22 GBq mmol&#226;&#8364;&#8220;1, American Radiolabeled Chemicals) light (10.2 &#206;&#188;mol <br />                                          photons m&#226;&#8364;&#8220;2 s&#226;&#8364;&#8220;1) 24 h. The hydrophobic fraction extracted harvested cells the <br />                                          method described previously [52]. The hydrophobic fraction separated TLC using cy- <br />                                          clohexane/ethylacetate (9:1, v/v). The plate sprayed 0.01% (w/v) primulin (Tokyo <br />                                          Chemical Industry, Tokyo, Japan) 80% (v/v) acetone, lipid spots located using <br />                                          ultraviolet light. The 14C-labeled compounds TLC plates detected using Bio-Im- <br />                                          aging Analyzer (Typhoon FLA7000, GE Healthcare). <br />  <br />  <br />                                          Supporting Information <br />                                          S1 Fig. Amino acid sequence alignments CrSQS (A) CrSQE (B) related se- <br />                                          quences. Sequences aligned using ClustalW [53] method. In alignment SQS <br />                                          proteins, conserved domains involved catalysis [22] underlined blue. In the <br />                                          alignment SQE proteins, highly conserved sequences containing putative FAD-binding <br />                                          domain [26,46] boxed red. Predicted transmembrane domains shown yellow <br />                                          boxes sequence. Identical residues sequences shown black back- <br />                                          ground. Similar residues shown gray background. See S3 Table accession num- <br />                                          bers protein sequences. See S1 Text sequence information SQS SQE as <br />                                          FASTA format data. <br />                                          (TIF) <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                           17 / 21 <br />                                                                           Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          S2 Fig. Southern blotting analysis CrSQS CrSQE genes. Genomic DNA isolated <br />                                          wild-type C-9 cells digested XhoI BamHI detection CrSQS gene <br />                                          HindIII XhoI detection CrSQE gene, hybridized 32P-labeled DNA <br />                                          fragment ORF. No recognition sites XhoI, BamHI HindIII occurred within <br />                                          coding sequence gene. One recognition site XhoI occurred coding se- <br />                                          quence CrSQE. The size restriction fragment detected given right kb. <br />                                          (TIF) <br />                                          S3 Fig. Mg2+ Mn2+ co-factors SQS enzymatic assays. The metal co-factor require- <br />                                          ments examined enzymatic assays CrSQS, products analyzed nor- <br />                                          mal-phase (A) reverse-phase (B) thin-layer chromatography. The different components <br />                                          included reaction mixtures indicated top. Authentic (1&#226;&#8364;&#8220;14C) farnesyl diphos- <br />                                          phate (FPP) loaded negative control. The positions origin, solvent au- <br />                                          thentic squalene indicated left. Open triangles right indicate positions of <br />                                          signals putative dehydrosqualene 12-hydroxysqualene. <br />                                          (TIF) <br />                                          S4 Fig. Squalene sterol profiles complementation lines KLN mutant express- <br />                                          ing CrSQE ScERG1. Complementation lines KLN mutant expressing CrSQE and <br />                                          ScERG1 established. Squalene sterols extracted yeast cells harvested <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                          YPD culture plates aerobic conditions (Fig. 2A) measured gas chromatography anal- <br />                                          ysis. 1, squalene; 2, cholesterol (internal standard); 3, ergosterol. Gas-chromatograms each <br />                                          authentic standard shown lower part. <br />                                          (TIF) <br />                                          S5 Fig. Squalene sterol profiles CrSQE-KD-1 parental UVM4 lines. A) Squalene <br />                                          sterols extracted C. reinhardtii cells cultured TAP liquid medium 2 day as <br />                                          Fig. 6, measured gas chromatography analysis. Gas-chromatograms authentic <br />                                          standard shown lower panels. 1, squalene; 2, cholesterol (internal standard); 3, ergos- <br />                                          terol. The peak 4 estimated putative 7-dehydroporiferasterol. B) Mass spectrum of <br />                                          peak 4 measured gas chromatography-mass spectrometry analysis. <br />                                          (TIF) <br />                                          S6 Fig. Expression analysis putative genes related methylerythritol phosphate <br />                                          (MEP) pathway. Expression levels genes putatively involved MEP pathway ex- <br />                                          amined CrSQE-knockdown (KD) lines. Nine genes encoding putative enzymes MEP <br />                                          pathway Phytozome database. Among them, expression level puta- <br />                                          tive CMK gene examined CrSQE-KD lines (A). Expression levels other <br />                                          genes examined CrSQE-KD lines, CrSQE-KD-2 CrSQE-KD-4, repre- <br />                                          sentative lines (B). Primer sequences, gene ID Phytozome encoded enzyme <br />                                          names listed S1 Table. Expression gene normalized CRY1. Data in <br />                                          experiments indicate mean value &#194;&#177; SD biological replicates. Asterisks the <br />                                          bars indicate significant differences (  p &lt; 0.05,    p &lt; 0.01). <br />                                          (TIF) <br />                                          S1 Table. Primer sequences qRT-PCR. <br />                                          (DOC) <br />                                          S2 Table. Primer sequences transgenic analysis. <br />                                          (DOC) <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                          18 / 21 <br />                                                                                 Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          S3 Table. Accession numbers SQS SQE sequences used alignment S1 Fig. <br />                                          (DOC) <br />                                          S1 Text. Amino acid sequences SQS SQE used alignment S1 Fig. listed in <br />                                          S3 Table. <br />                                          (TXT) <br />  <br />  <br />                                          Acknowledgments <br />                                          We thank Dr Ralph Bock (Max Planck Institute) providing C. reinhardtii strain UVM4, <br />                                          Dr Friederike Turnowsky (Graz University) providing yeast KLN1 strain, Dr Haruaki <br />                                          Yanagisawa (The University Tokyo) providing pGenD-cHA vector, Dr Fumihiko <br />                                          Sato Mr Yasuyuki Yamada (Kyoto University) use Particle Delivery PDS-1000/ <br />                                          He system. <br />  <br />  <br />                                          Author Contributions <br />                                          Conceived designed experiments: M. Kajikawa HF. Performed experiments: M. <br />                                          Kajikawa SK AA MS M. Kato. Analyzed data: M. Kajikawa SK AA MS M. Kato HF. Con- <br />                                          tributed reagents/materials/analysis tools: M. Kajikawa M. Kato HF. Wrote paper: M. Kaji- <br />                                          kawa HF. <br />  <br />  <br />                                          References <br />                                           1.   Spanova M, Daum G. Squalene&#226;&#8364;&#8221;biochemistry, molecular biology, process biotechnology, appli- <br />                                                cations. Eur J Lipid Sci Technol. 2011; 113: 1299&#226;&#8364;&#8220;1320. <br />                                           2.   Reddy LH, Couvreur P. Squalene: neutral triterpene use disease management therapy. <br />                                                Adv Drug Deliv Rev. 2009; 61: 1412&#226;&#8364;&#8220;1426. doi: 10.1016/j.addr.2009.09.005 PMID: 19804806 <br />                                           3.   Bhattacharjee P, Shukla VB, Singhal RS, Kulkarni PR. Studies fermentative production squalene. <br />                                                World J Microbiol Biotechnol. 2001; 17: 811&#226;&#8364;&#8220;816. <br />                                           4.   Fabris M, Matthijs M, Carbonelle S, Moses T, Pollier J, Dasseville R, et al. Tracking sterol biosyn- <br />                                                thesis pathway diatom Phaeodactylum tricornutum. New Phytol. 2014; 204: 521&#226;&#8364;&#8220;535. doi: 10. <br />                                                1111/nph.12917 PMID: 24996048 <br />                                           5.   Jiang Y, Fan KW, Wong RDY, Chen F. Fatty acid composition squalene content marine <br />                                                microalga Schizochytrium mangrovei. J Agric Food Chem. 2004; 52: 1196&#226;&#8364;&#8220;1200. PMID: 14995120 <br />                                           6.   Fan KW, Aki T, Chen F, Jiang Y. Enhanced production squalene thraustochytrid Aurantiochy- <br />                                                trium mangrovei medium optimization treatment terbinafine. World J Microbiol Biotechnol. <br />                                                2010; 26: 1303&#226;&#8364;&#8220;1309. doi: 10.1007/s11274-009-0301-2 PMID: 24026934 <br />                                           7.   Nakazawa A, Matsuura H, Kose R, Kato S, Honda D, Inouye I, et al. Optimization culture conditions <br />                                                thraustochytrid Aurantiochytrium sp. strain 18W-13a squalene production. Bioresour Technol. <br />                                                2012; 109: 287&#226;&#8364;&#8220;291. doi: 10.1016/j.biortech.2011.09.127 PMID: 22023965 <br />                                           8.   Okada S, Devarenne TP, Chappell J. Molecular characterization squalene synthase green <br />                                                microalga Botryococcus braunii, race B. Arch Biochem Biophys. 2000; 373: 307&#226;&#8364;&#8220;317. PMID: <br />                                                10620354 <br />                                           9.   Won-Kyung H, Heo S-Y, Park H-M, Kim CH, Sohn J-H, Kondo A, et al. Characterization squalene <br />                                                synthase Thraustochytrid Microalga Aurantiochytrium sp. KRS101. J Microbiol Biotechnol. <br />                                                2013; 23: 759&#226;&#8364;&#8220;765. PMID: 23676912 <br />                                          10.   Lohr M, Schwender J, Polle JEW. Isoprenoid biosynthesis eukaryotic phototrophs: spotlight on <br />                                                algae. Plant Sci. 2012; 185&#226;&#8364;&#8220;186: 9&#226;&#8364;&#8220;22. <br />                                          11.   Matsushima D, Jenke-Kodama H, Sato Y, Fukunaga Y, Sumimoto K, Kuzuyama T, et al. The single cel- <br />                                                lular green microalga Botryococcus braunii, race B possesses distinct 1-deoxy-D-xylulose 5- <br />                                                phosphate synthases. Plant Sci. 2012; 185&#226;&#8364;&#8220;186: 309&#226;&#8364;&#8220;320. <br />                                          12.   Kong F, Yamasaki T, Ohama T. Expression levels domestic cDNA cassettes integrated nucle- <br />                                                ar genomes various Chlamydomonas reinhardtii strains. J Biosci Bioeng. 2014; 117: 613&#226;&#8364;&#8220;616. doi: <br />                                                10.1016/j.jbiosc.2013.10.025 PMID: 24342172 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120446 March 12, 2015                                                                                        19 / 21 <br />                                                                                 Characterization SQS SQE Genes Algal Squalene Metabolism <br />  <br />  <br />  <br />                                          13.   Salimova E, Boschetti A, Eichenberger W, Lutova L. Sterol mutants Chlamydomonas reinhardtii: <br />                                                Characterization strains deficient C24(28) reductase. Plant Physiol Biochem. 1999; 37: 241&#226;&#8364;&#8220; <br />                                                249. <br />                                          14.   Brumfield KM, Moroney JV, Moore TS, Simms TA, Donze D. Functional characterization Chlamy- <br />                                                domonas reinhardtii ERG3 ortholog, gene involved biosynthesis ergosterol. PLoS One. <br />                                                2010; 5: e8659. doi: 10.1371/journal.pone.0008659 PMID: 20084111 <br />                                          15.   Miller MB, Haubrich BA, Wang Q, Snell WJ, Nes WD. Evolutionarily conserved &#206;&#8221;25(27)-olefin ergosterol <br />                                                biosynthesis pathway alga Chlamydomonas reinhardtii. J Lipid Res. 2012; 53: 1636&#226;&#8364;&#8220;1645. doi: <br />                                                10.1194/jlr.M027482 PMID: 22591742 <br />                                          16.   Haubrich BA, Collins EK, Howard AL, Wang Q, Snell WJ, Miller MB, et al. Characterization, mutagene- <br />                                                sis mechanistic analysis ancient algal sterol C24-methyltransferase: implications under- <br />                                                standing sterol evolution green lineage. Phytochem. 2014;In press: doi: 10.1016/j.phytochem. <br />                                                2014.07.019 <br />                                          17.   Pandit D, Danley DE, Schulte GK, Mazzalupo S, Pauly TA, Hayward CM, et al. Crystal structure of <br />                                                human squalene synthase. J Biol Chem. 2000; 275: 30610&#226;&#8364;&#8220;30617. PMID: 10896663 <br />                                          18.   Merchant SS, Prochnik SE, Vallon O, Harris EH, Karpowicz SJ, Witman GB, et al. The Chlamydomo- <br />                                                nas genome reveals evolution key animal plant functions. Science. 2007; 318: 245&#226;&#8364;&#8220;250. <br />                                                PMID: 17932292 <br />                                          19.   Blaby IK, Blaby-Haas CE, Tourasse N, Hom EFY, Lopez D, Aksoy M, et al. The Chlamydomonas ge- <br />                                                nome project: decade on. Trends Plant Sci. 2014; 19: 672&#226;&#8364;&#8220;680. doi: 10.1016/j.tplants.2014.05.008 <br />                                                PMID: 24950814 <br />                                          20.   Nakashima T, Inoue T, Oka A, Nishino T, Osumi T, Hata S. Cloning, expression, characterization <br />                                                cDNAs encoding Arabidopsis thaliana squalene synthase. Proc Natl Acad Sci USA. 1995; 92: 2328&#226;&#8364;&#8220; <br />                                                2332. PMID: 7892265 <br />                                          21.   Kribii R, Arro M, Del Arco A, Gonz&#195;&#161;lez V, Balcells L, Deloume D, et al. Cloning characterization of <br />                                                Arabidopsis thaliana SQS1 gene encoding squalene synthase. Involvement C-terminal region <br />                                                enzyme channeling squalene sterol pathway. Eur J Biochem. 1997; 249: <br />                                                61&#226;&#8364;&#8220;69. PMID: 9363754 <br />                                          22.   Busquets A, Keim V, Closa M, del Arco A, Boronat A, Arr&#195;&#179; M, et al. Arabidopsis thaliana contains sin- <br />                                                gle gene encoding squalene synthase. Plant Mol Biol. 2008; 67: 25&#226;&#8364;&#8220;36. doi: 10.1007/s11103-008- <br />                                                9299-3 PMID: 18236008 <br />                                          23.   Robinson GW, Tsay YH, Kienzle BK, Smith-Monroy CA, Bishop RW. Conservation human <br />                                                fungal squalene synthases: similarities structure, function, regulation. Mol Cell Biol. 1993; <br />                                                13: 2706&#226;&#8364;&#8220;2717. PMID: 8474436 <br />                                          24.   Jiang G, McKenzie TL, Conrad DG, Schechter I. Transcriptional regulation lovastatin 25-hydro- <br />                                                xycholesterol HepG2 cells molecular cloning expression cDNA human hepatic <br />                                                squalene synthase. J Biol Chem. 1993; 268: 12818&#226;&#8364;&#8220;12824. PMID: 7685352 <br />                                          25.   Summers C, Karst F, Charles AD. Cloning, expression characterization cDNA encoding <br />                                                human hepatic squalene synthase, relationship phytoene synthase. Gene. 1993; 136: 185&#226;&#8364;&#8220; <br />                                                192. PMID: 8294001 <br />                                          26.   Rasbery JM, Shan H, LeClair RJ, Norman M, Matsuda SPT, Bartel B. Arabidopsis thaliana squalene <br />                                                epoxidase 1 essential root seed development. J Biol Chem. 2007; 282: 17002&#226;&#8364;&#8220;17013. PMID: <br />                                                17426032 <br />                                          27.   Laden BP, Tang Y, Porter TD. Cloning, heterologous expression, enzymological characterization <br />                                                human squalene monooxygenase. Arch Biochem Biophys. 2000; 374: 381&#226;&#8364;&#8220;388. PMID: 10666321 <br />                                          28.   Lee HK, Denner-Ancona P, Sakakibara J, Ono T, Prestwich GD. Photoaffinity labeling site-directed <br />                                                mutagenesis rat squalene epoxidase. Arch Biochem Biophys. 2000; 381: 43&#226;&#8364;&#8220;52. PMID: 11019818 <br />                                          29.   Jandrositz A, Turnowsky F, Hogenauer G. The gene encoding squalene epoxidase Saccharomy- <br />                                                ces cerevisiae: cloning characterization. Gene. 1991; 107: 155&#226;&#8364;&#8220;160. PMID: 1743514 <br />                                          30.   Landl KM, Kl&#195;&#182;sch B, Turnowsky F. ERG1, encoding squalene epoxidase, located right arm of <br />                                                chromosome VII Saccharom <br /> </body></html>