<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Nail. Aead. Sci. USA <br /> Vol. 89, pp. 28-32, January 1992 <br /> Genetics <br />  <br />  <br /> Yeast casein kinase I homologues: An essential gene pair <br />      (Saccharomyces cerevisiae/protein kinase/snf mutations/salt tolerance) <br /> Lucy C. ROBINSON*, E. JANE ALBERT HUBBARDt, PAUL R. GRAVESt, ANNA A. DEPAOLI-ROACHt, <br /> PETER J. ROACHt, CHING KUNG*, DAVID W. HAAS&#194;&#167;, CURT H. HAGEDORN&#194;&#167;, MARK GOEBL, <br /> MICHAEL R. CULBERTSON*, AND MARIAN CARLSONt <br /> *Molecular Biology Genetics, University Wisconsin, 1525 Linden Drive, Madison, WI 53706; tDepartment Genetics Development, Columbia <br /> University, 701 West 168th Street, New York, NY 10032; tDepartment Biochemistry Molecular Biology, Indiana University School Medicine, <br /> 635 Barnhill Drive, Indianapolis, IN 46223; &#194;&#167;Departments Medicine Cell Biology, Vanderbilt University School Medicine the <br /> Veterans Administration Medical Center, Nashville, TN 37232-2279 <br /> Communicated Michael H. Wigler, October 10, 1991 (received review August 2, 1991) <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>           We report isolation essential pair                   Casein kinase I activity inferred central <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Saccharomyces cerevisiae genes encode protein kinase                         cellular function wide distribution, numerous vitro <br /> homologues. The genes independently isolated                          substrates, multiple subcellular localizations. However, <br /> dosage-dependent suppressors. Increased dosage YCKI sup-                       importance phosphorylation enzyme vivo has <br /> pressed defects caused reduced SNF1 protein kinase activ-                      demonstrated cases, notably glycogen synthase. <br /> ity, increased dosage YCK2 relieved sensitivity                         Phosphorylation glycogen synthase casein kinase I in <br /> wild-type cells salt stress. The genes function identically                vitro results inactivation enzyme (7) phosphor- <br /> in growth assays, loss function gene                     ylation site, Ser10, occurs rabbit muscle vivo in <br /> alone discernible effect growth. However, loss                       response stimulation epinephrine (9). Other physiolog- <br /> function genes results inviability. The predicted                  ically important phosphorylations casein kinase I have <br /> protein products share 77% overall amino acid identity                        suggested overall physiological roles remain <br /> contain sequence elements conserved protein kinases.                        unclear. The study casein kinase I function be <br /> Partial sequence obtained rabbit casein kinase I shares 64%                   greatly facilitated approaches possible with <br /> identity yeast gene products. Moreover,                           genetically tractable budding yeast Saccharomyces cere- <br /> increase casein kinase I activity observed extracts                 visiae. Enzymological studies revealed multiple forms <br /> cells overexpressing YCK2. Thus YCKI YCK2 appear                           casein kinase yeast share properties mamma- <br /> encode casein kinase I homologues.                                                lian casein kinase I (10-12), biological function has <br />                                                                                   assigned activities. <br /> Casein kinase I denotes classes widely distrib-                        We isolation structural homo- <br /> uted mammalian cAMP- Ca2+-independent protein ki-                             logues casein kinase I S. cerevisiae. The YCKI and <br /> nases originally defined ability phos-                       YCK2 (yeast casein kinase I homologue) genes identi- <br /> phorylate acidic substrates, notably caseins (reviewed ref.                    fied independently based effects increased gene <br /> 1). Casein kinase I activities detected                         dosage. YCKJ isolated suppressor require- <br />                                                                                   ment SNF4 function. The SNF4 protein positive <br /> eukaryotic cells membrane-bound, cytosolic,                         effector SNF1 protein kinase (13), required for <br /> nuclear. A wide variety vitro substrates                          carbon catabolite derepression (14). Increased dosage of <br /> identified mammalian casein kinase I. Among                          YCK2 allows wild-type cells withstand extreme salinity <br /> the simian virus 40 large tumor antigen (1); cytoskeletal                     stress prior adaptation. We report YCKJ and <br /> proteins myosin troponin (1, 2); membrane components                           YCK2 high degree sequence similarity other <br /> such spectrin (1) neural cell adhesion molecule (N-                        partial sequence rabbit casein kinase I. We that <br /> CAM; ref. 3); nonhistone nuclear proteins including RNA                           increased gene dosage YCK2 correlated increased <br /> polymerases I II (1); components translation                           casein kinase I activity cell extracts. Finally, demon- <br /> apparatus including initiation factors 4B, 4E, 5 tRNA                     strate genes comprise essential gene family.l <br /> synthetases (1, 4, 5); metabolic enzymes, notably gly- <br /> cogen synthase (1). <br />    The site preference casein kinase I defined                                  MATERIALS AND METHODS <br /> acidic residues amino-terminal modified residue (1, 6).                      Genetic Methods. S. cerevisiae strains derived <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> However, sequence motif -Ser(P)-Xaa-Xaa-Ser- re-                          JC482 (MATa his4 leu2 ura3; ref. 15), LRB188 (MATa/a <br /> cently demonstrated specify casein kinase I action,                       his3/his3 leu2/leu2 ura3/ura3; ref. 16), MCY1093 (MATa <br /> suggesting model casein kinase I biological activity                        his4 lys2 ura3 SUC2), MCY1094 (MATa ade2 ura3 SUC2), or <br /> termed hierarchical protein phosphorylation (7). For exam-                        MCY2372 (MATa his3 leu2 ura3 SUC2). The snf strains were <br /> ple, phosphorylation N-CAM casein kinase I vitro                      MCY1853 (MA Ta snf4-A2 his4 lys2 ura3 SUC2) and <br /> abolished prior treatment phosphatase (3). Also,                          MCY1845 (MATa snfl-A10 ade2 ura3 SUC2). Genetic anal- <br /> phosphorylation glycogen synthase cAMP-dependent                            ysis transformation standard methods (17, 18). <br /> protein kinase (cAPK) potentiates casein kinase 1 (6).                      Nucleic Acid Methods. Standard methods DNA manip- <br /> Replacement phosphoserine block                          ulations used (19, 20). DNA sequence determined <br /> acidic residues synthetic peptide substrates results poor                   strands (21) using Sequenase (United States <br /> but relatively specific casein kinase I substrates (8). <br />                                                                                   Abbreviation: cAPK, cAMP-dependent protein kinase. <br /> The publication costs article defrayed page charge        $The sequences reported paper deposited the <br /> payment. This article marked &quot;advertisement&quot;              GenBank data <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span> [accession nos. M74552 (YCKI) M74453 <br /> in accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                   (YCK2)]. <br />                                                                              28 <br />            Genetics: Robinson                   et   al.                                      Proc. Natl. Acad. Sci. USA 89 (1992)           29 <br /> Biochemical) Amplitaq (Cetus) enzymes, using                        Edman degradation Porton Instruments (Tarzana, CA) <br /> the Applied Biosystems 373A DNA sequencer.                                      model 2090 integrated microsequencing equipped <br />    Gene Disruption. A deletion YCKI (yckl-AJ:: URA3)                         on-line HPLC detection phenylthiohydantoin <br /> was constructed replacing Bgl II fragment URA3                      derivatives. Amino acid sequences obtained four <br /> (Fig. 1A). yck2::HIS3 constructed TnS mutagenesis                        peptides. Degenerate oligonucleotide primers polymerase <br /> (22) pLS2.3 followed replacement portion TnS                      chain reaction (PCR) DNA amplification designed from <br /> with HIS3 (Fig. 1B). The yckl-Al::URA3 yck2::HIS3                           peptide sequences DNFLMG (sense strand) and <br /> alleles introduced (23) wild-type diploid strains                 MYFNLQ (antisense strand). A BamHI site Sma I site <br /> MCY1093 x MCY1094 LRB188 generate strains                                added 5&apos; ends sense antisense primers, <br /> heterozygous disruption <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>mutation</span>. The diploid                          respectively. PCR carried separate phases <br /> strains LRB237 LRB238 (yckl/+ yck2/+) gener-                           (25), using cDNA synthesized total rabbit testis RNA. <br /> ated introduction yckl -Al:: URA3 diploid strain                   The phase 10 cycles 10 pmol primers the <br /> heterozygous yck2::HIS3. LRB263 (MATa his3 leu2 ura3                        second phase 40 cycles 50 pmol primers. The <br /> yck2) LRB266 (MATa his3 leu2 ura3 yckl) meiotic                         reaction yielded discrete DNA product 230 base pairs <br /> progeny LRB237. Strains LRB271 (MATa/MA Ta his3/                             (bp) subcloned phage vector M13 using the <br /> his3 leu2/leu2 ura3/ura3 yckl/yckl yck2/+) LRB272                           BamHI Sma I sites subjected sequence analysis. <br /> (MATa/MATa his3/his3 leu2/leu2 ura3/ura3 yckl/+ yck2/                              Measurement Yeast Casein Kinase I Activity. Cells from <br /> yck2) derived meiotic progeny LRB237. Diploid                      overnight cultures disrupted shaking glass <br /> strains MCY2394 MCY2395 generated introduc-                         beads 25 mM Hepes/2 mM EDTA/2 mM EGTA, pH 7.5. <br /> tion yck2::HIS3 MCY2393 (MATa/MATa ade2/+                               The extract centrifuged 11,000 x g supernatant <br /> his3/his3 + /Ieu2 + /Ilys2 ura3/ura3 yckl/yckl). Replacement                    assayed casein kinase I activity (8). 32p incorporation <br /> of wild-type sequences disruption alleles veri-                    [y-32P]ATP (310 cpm/pmol) synthetic peptide <br /> fied Southern blot analysis.                                                 DDDDVASLPGLRRR (1 mM) measured using P81 <br />    Characterization Amplifled DNA Fragment Encoding                       paper (Whatman) filter binding assay (26). Values from <br /> Rabbit Casein Kinase I. Casein kinase I used protein                        control reactions lacking peptide subtracted. <br /> sequencing purified rabbit reticulocyte lysate (4). <br /> This enzyme identical casein kinase I purified                                               RESULTS <br /> rabbit skeletal muscle respect molecular mass and <br /> tryptic phosphopeptide map (4, 6). Phosphopeptide maps                         Isolation YCK1 YCK2 Genes. YCKI isolated <br /> initiation factor 4E phosphorylated reticulocyte skel-                   multicopy suppressor growth defects reduced <br /> etal muscle casein kinase I identical, specificity                 SNF1 protein kinase activity. The SNF1 protein kinase is <br /> of reticulocyte enzyme substrates consis-                     required expression genes response glucose <br /> tent casein kinase I (4). Reticulocyte casein                      deprivation (14). The SNF4 protein physically associated <br /> kinase 1 (30 ,ug) subjected SDS/PAGE transferred                     SNF1 required maximal SNF1 protein kinase <br /> to nitrocellulose (4, 24). The polypeptide 35 kDa                        activity (13, 27). To recover genes functionally <br /> excised digested trypsin (trypsin/protein ratio, 1:20)                 related SNFI SNF4, isolated multicopy (2 am) <br /> at 37&#194;&#176;C 24 hr. Tryptic peptides separated                         plasmids suppress raffinose growth defect snf4 <br /> reverse-phase HPLC C18 cartridge (OD-300, Applied Biosys-                       mutants. The snf4 mutant strain MCY1853 transformed <br /> tems) gradient acetonitrile 0.1% trifluoroacetic                   YEp24 yeast genomic library (28), transformants <br /> acid. The purified peptides subjected automated                         able grow raffinose 30&#194;&#176;C isolated described <br />                                                                                 (29). Among plasmids recovered pB65, a <br />           A. YCKJ <br />                                   B H <br />                                                                                 weak suppressor recovered transformants. <br />                          SB               Sm                            p <br />           pB65       - <br />                              II, f                                                 To localize functional region pB65, con- <br />           pJH21 <br />                                      O-                                         structed subclones tested ability allow growth of <br />           pB65R7 <br />                                                                                 snf4-A2 mutant raffinose. The functional region lies <br />           pJH2O <br />                                                                                 Sal I site (Fig. 1A). A 1.8-kb RNA hybridized to <br />           pUJ721 <br />                                                                                 probe derived region (data shown). <br />           (YEp352)                                                                 YCK2 isolated ability act suppressor of <br />           pJH79 <br />                                                                                 high salinity intolerance. Wild-type cells require period of <br />           (pUC19) <br />                               2 <br />  <br />                                                                                 adaptation conditioning 0.4-0.6 M NaCl enable <br />                                                                                 growth medium containing &gt;1 M NaCl. During period <br />           B. YCK2 <br />                                                                                 cells accumulate high levels protective osmolyte <br />                                   EX <br />                                    II <br />                                                N           H <br />                                                                                 glycerol (30, 31). The multicopy plasmid pLS2 recovered <br />           pIS2                                                              + <br />                                                                                 transformants strain JC482 YEp352 <br />           pI]S2.3                                                               genomic library (provided D. Conklin, University of <br />           pLS2.33 (YEp351) <br />           pLS2.6                                                                Wisconsin, Madison). These transformants longer re- <br />           pIS2.3T                                                               quired conditioning grow high salinity. The functional <br />                                                                                 region pLS2 determined subclone analysis (Fig. <br />                                                Tn5:iIIS3       1   kb           1B). A probe derived HindIII-Xba I fragment <br />                                                                                 hybridized 1.9-kb RNA (data shown). <br />    FIG. 1. YCKI YCK2 plasmids. Only yeast DNA sequences                        YCK1 YCK2 Show Sequence Similarity Each Other <br /> are shown. Arrows indicate position 5&apos; 3&apos; direction              Protein Kinases. The nucleotide sequence pB65 <br /> open reading frames, starting ATG. Bg, Bgl II; E, EcoRI;           functional region contains open reading frame 1.6 kb. <br /> H, HindIII; N, Nco I; P, Pst 1; S, Sal I; Sm, Sma I; X, Xba I; Xh,              The 538 codons ATG insert the <br /> Xho I. (A) The vector YEp24 unless indicated. The                  stop codon encode 61.7-kDa protein. The nucleotide <br /> ability multicopy plasmid suppress growth defect an <br /> snf4 mutant raffinose 30&apos;C indicated right. *, pLJ721 was <br />                                                                                 sequence pLS2 functional region contains 547-codon <br /> tested salt tolerance. HindIll Bgl II sites mapped            open reading frame encode protein 62 kDa. <br /> only left Sma I site. (B) The vector YEp352 unless                Both predicted amino acid sequences compared with <br /> otherwise indicated. The ability multicopy plasmid confer            sequences GenBank EMBL data bases (32) (Feb- <br /> salt tolerance indicated right. kb, Kilobase.                             ruary 1991) protein sequences data base <br />  30       Genetics: Robinson            et   al.                                                   Proc. Natl. Acad. Sci. USA 89 (1992) <br /> maintained M.G. The YCK1 YCK2 proteins                             receptor tyrosine kinases prefer acidic substrates <br /> similar (Fig. 2), overall amino acid identity                   (33). Thus, basic residue position correlate <br /> of 77%. The N-terminal two-thirds predicted product                        recognition acidic substrates. However, corre- <br /> contains sequence elements conserved pro-                           sponding residue casein kinase I YCK products is <br /> tein kinases (33). Within region, YCK products                        Asp. Three residues important substrate recog- <br /> 96% identical. The C termini YCK proteins                         nition cAPK-Leu1&quot;, Pro202, Leu205-are replaced in <br /> similarity other, rich glutamine                           casein kinase II Arg, Arg, Lys, respectively, consis- <br /> residues terminate sequence Gly-Cys-Cys.                              tent preference acidic substrates. However, the <br />    YCK1 YCK2 Resemble Rabbit Casein Kinase I. Both                             casein kinase I family, residues Leu, Ala, and <br /> YCK1 YCK2 closely related partial sequences                             Met/Ala, respectively. Thus, casein kinase I distinct from <br /> obtained rabbit casein kinase I. Degenerate oligonucleo-                       casein kinase II asjudged overall sequence these <br /> tide primers corresponding peptide sequence obtained                        key residues potentially involved substrate recognition. <br /> rabbit reticulocyte casein kinase I used PCR                        Another characteristic casein kinase I family the <br /> rabbit testis cDNA, sequence product                            absence highly conserved Ala-Pro-Glu sequence of <br /> determined. The predicted amino acid sequence (Fig. 2)                             subdomain VIII (33), possible assign unequiv- <br /> contained sequences peptides used design                         ocally residue corresponding invariant Glu (cAPK <br /> primers sequences tryptic pep-                         Glu21). In summary, appears gene products <br /> tides. There conservative substitutions (see leg-                       define novel subfamily protein kinases distinctive <br /> end Fig. 2) reflect PCR artifacts                      sequence features. <br /> presence distinct casein kinase I isoforms rabbit testis                        YCKI YCK2 Are Functionally Homologous. The high <br /> and reticulocytes.                                                                 degree sequence similarity YCK1 YCK2 <br />    When sequence predicted rabbit testis casein kinase                     prompted assess functional similarity. Plasmid <br /> I compared sequences data base maintained                         pB65 weakly suppressed raffinose sucrose growth <br /> M.G., highest similarity YCK1 YCK2 (Fig. 2),                        defects snf4-A2 snfl-AJO mutants MCY1853 and <br /> with 64% amino acid identity 77 residues. Con-                            MCY1845. Transformation strains pLS2 <br /> versely, YCK1 YCK2 showed greater similarity                            showed increased dosage YCK2 weakly sup- <br /> rabbit casein kinase I sequence yeast sequence.                        pressed growth defects. Because invertase catalyzes the <br /> All sequences distantly related                            hydrolysis sugars, assayed invertase activity (36) <br /> other protein kinases, showing &lt;26% identity enzymes                         snf4 mutants carrying pB65, pB65R7, pLS2. None of <br /> the Ca2+/calmodulin-dependent, cdc2, cAMP-depen-                               plasmids restored significant invertase expression after <br /> dent protein kinase families. Casein kinases I II                          shift low glucose (2-3.5 units YCKJ YCK2 <br /> among known kinases substrate specificity                         plasmids; 1.5-2 units vectors; 160 units SNF4 plas- <br /> acidic sites, terms primary sequence, casein                         mid). <br /> kinase II enzymes quite distinct putative                              In reciprocal experiment, multicopy subclone of <br /> casein kinase I sequences. Although yeast Drosophila                            YCKI (pLJ721; Fig. 1A) introduced JC482 and <br /> casein kinase II sequences share 64% amino acid identity (1),                      transformants tested ability grow medium <br /> they &lt;20% identity catalytic domains ofthe YCK                         containing 1.2 M NaCl. pLJ721 effective pLS2 in <br /> products partial rabbit casein kinase I sequence.                       assay, cells carrying vector unable to <br />    The crystal structure cAPK substrate-analogue                           grow (data shown). <br /> inhibitory peptide bound (34, 35) allowed <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>                           Loss YCK Function Is Lethal. The <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> caused by <br /> side chains involved substrate recognition,                       loss YCKJ function examined constructing a <br /> lie conserved kinase domain. For example, Glu170                        diploid strain heterozygous yckl-AJ::URA3 allele. <br /> in cAPK contacts substrate Arg residue. This Glu                              The URA3 marker segregated Mendelian fashion among <br /> replaced His casein kinase II Arg                             tetrads derived strain, Ura+ <br />                1    .                                                                                          .VLFEGT <br />                                                                                         SRDDSTIVGLHYKIGKKIDET.SF <br />                           MSMPIASTTLAVNNLTNINGNANFNVQANKQLHHQAVDSPAR. .SSMTATTAANSNSNS...                                          YCK1 <br />                               :     11111 1                1: :           :    1:           1111111111111111 <br />                1     MSQVQSPLTATNSGLAVNNNT ... MNSQMPNRSNVRLVNGTLPPSLHVSSNLNHNTGNSSASYSGSQSRDDSTIVGLHYKIGKKIfEcSFcVLFEGT           YCK2 <br />                                                                                                                        II <br />                88    NMINGVPVAIKFEPRKTEAPQLRDEYKTYKILNGTPNIPYAYYFGQEGLHNILVIDLLGPSLEDLFDWCGRKFSVKTVVQVAVQMITLIEDLHAHDL             YCK1 <br />                      11111:1111111111111111:111:11111 111 11 111111111ll111111111111111111111:1111111111111111111111111 <br />                95    NMINGLPVAIgEPRKTEAPQLKDEYRTYKI LAGTPGIPQEYYFGQEGLHNILVIDLLGPSLEDLFDWCGRRFSVKTVVQVAVQMITLIEDLHAHDL             YCK2 <br />                                III <br />                185   IYRDIKPD.fFLIGRPGQPDANNIHLIDEIMAKQYRDPKTKQHIPYREKKSLSGTARYMSINTHLGREQSRRDAMEALcHVFFYFLRGHLPWQGLKAP            YCK1 <br />                      111111111111111111111          :1111111111111111111111111111111111111111111111111 1111:1111111111:111111111 <br />                192   IYRwIKPDEFLIGRPGQPDANKVHLIDEMAKQYRDPKTKQHIPYREKKSLSGTARYMSINTHLGREQSRRDnMEAMcHVFFYFLRGQLPWQGLKAP              YCK2 <br />                               1111:1              :II:IIII:III :IIIIIIIIIIIIIIIIIIII1lIIII1lI:1                 I11 <br />                               DIJFLMGLG. . KKGNLVYI IDFlLAKKYRDARTHQHIPYRENKNLTGTARYASINTHLGIEQSRRDDLESLcYVLMYFNL                  CK1 <br />                          VI                            VII                                               Ix <br />                282   NNKQKYEKIGEKKRSTNVYDLAQGLPVQFGRYLEIVRSLSFEECPDYEGYRKLLLSVLDDLGETADGQYDWMKLNDGRGWDLNINKKPNLHGYGHPN             YCK1 <br />                289 NNKQKYEKIGEKKRLTNVYDLAQGLP IQFGRYLEIVRNLSFEETPDYEGYRMLLLSVLDDLGETADGQYDWMKLNGGRGWDLSINKKPNLHGYGHPN              YCK2 <br />  <br />                379 PPNEKSRKHR .............. NKQLQMQQ ............ LQMQQLQQQQQQQQYAQKTEADMRNSQYK ..... PKLDPTSYEAYQHQTQQK          YCK1 <br />                      111111::11              ~~1111111111111           1:IIII:I       :    :1:1         111111:11 <br />                386 PPNEKSKRHRSKNHQYSSPDHHHHYNQQQQQQQAQAQAQAQAQAKVQQQQLQQAQAQQQ . ANRYQLQPDDSHYDEEREASKLDPTSYEAYQQQTQQK             YCK2 <br />  <br />                444 YLQEQQKRQQQQKLQEQQLQEQQLQQQQQQQQQLRATGQPPSQPQAQTQSQQFGARYQPQQQPSAALRTPEQHPNDDNSSLAASHKGFFQKLGCC*                YCK1 <br />                    1 1:111                   I 11   1: 11        I I         11         ::     1: 11       1111 11111 <br />                483 YAQQQQK .....     ..... QMQQKSKQFANTG ....... ANGQTNKYPYNAQPTANDEQNAKNAAQDRNSNKSS ..... KGFFSKLGCC*             YCK2 <br />  <br />   FIG. 2. Alignment YCK1 YCK2 predicted amino acid sequences partial rabbit testis casein kinase I (CK1) sequence. The <br /> comparison generated GAP program (32). Vertical lines denote identity double dots denote conservative changes. Underlined <br /> residues correspond readily identifiable conserved protein kinase subdomains (ref. 33; subdomains numbered sequence). The <br /> YCKI clone includes additional 54 codons 5&apos; ATG (not shown). The sequences determined tryptic peptides rabbit <br /> reticulocyte casein kinase I XIFPDNFLMGIG, XDDXESLGYVLMYFNL, XHIPYXEDKNLTGT, YASINAXLGIEQS (where X <br /> denotes cycle assignment made); peptide positions depicted bold lettering differ testis sequence. <br />           Genetics: Robinson    et   al.                                          Proc. Natl. Acad. Sci. USA 89 (1992)                  31 <br /> segregants showed growth differences Ura- segregants <br /> on carbon source tested 300C 37TC. Regulation of <br /> invertase expression identical yckl segregants <br /> and wild-type siblings. <br />    Tetrad analysis carried progeny two <br /> diploid transformants heterozygous yck2: :HIS3 allele. <br /> All spore clones viable, His&apos; spore clones showed <br /> no obvious growth differential wild-type (His-) siblings                                                                Ir <br /> under conditions tested, including high salt, fermentable                      i <br />  <br /> and nonfermentable carbon sources, high (37TC) low <br /> (16TC) temperature. <br />    To determine phenotype strains lacking YCKI <br /> and YCK2 function, diploid strains heterozygous both <br /> yckl yck2 sporulated tetrads analyzed. <br /> The <span id='am-164' about='xsp:pattern' typeof='owl:Thing'>pattern</span> spore viability observed (2:2, 3:1, 4:0) was <br /> consistent presence unlinked mutations that <br /> together lethal. Single yckl yck2 disruption mutants <br /> were recovered expected frequency His&apos; <br /> Ura&apos; double disruption mutants recovered. To confirm <br /> that yckl yck2 lethal combination, examined the <br /> meiotic progeny strains LRB271, MCY2394, and <br /> MCY2395 (yckl/yckl yck2/+) LRB272 (yckl/+ yck2/ <br /> yck2). In case, observed 2:2 segregation for <br /> viability expected double mutant inviable <br /> (45/45 total tetrads; Fig. 3). None viable spore clones <br /> tested carried mutations. <br />    The spore clones corresponding predicted double <br /> mutants showed unusual growth pattern (Fig. 3). All such <br /> spores germinated divided times rich medium <br /> containing glucose 30&#194;&#176;C, normal division ceased        FIG. 3. Loss YCK function lethal. (Upper) Ten tetrads that <br /> 3-10 rounds. Most cells formed multiple projec-       progeny diploid strain LRB271 (yckl/yckl yck2/+). (Lower) <br /> tions continued elongate 24-48 hr,       Clusters cells corresponding inviable double mutants, illustrat- <br /> growth ceased. The lethality double mutation       ing elongated terminal morphology variability in <br /> rescued alteration carbon source (ethanol raffinose),    number normal divisions terminal morphology was <br />                                                                   manifested. The plate incubated 300C, photographs were <br /> temperature incubation (16&#194;&#176;C, 25&#194;&#176;C, 37&#194;&#176;C) ionic (0.1     taken 72 hr (plate) 96 hr (spore clones) incubation. (x480.) <br /> or 1 M KCI; 0.9 M NaCI) osmotic (1 M sorbitol) conditions. <br />    To confirm mitotic growth requires YCK function,       rabbit casein kinase I, increased gene dosage YCK2 <br /> analyzed mitotic progeny double mutants carrying           increases casein kinase I activity cell extracts. Therefore, <br /> pLS2.33 retention plasmid nonselective           propose yeast genes encode casein kinase I <br /> conditions. Four transformants strain LRB271               homologues, designate YCKI YCK2. <br /> (yckl/+ yck2/yck2) carrying pLS2.33 sporulated              The YCKI YCK2 predicted products -62 kDa, <br /> all yckl yck2 (Ura+ His+) segregants carried plasmid          significantly larger mammalian casein kinase I, which <br /> (Leu+). Four spore clones grown nonselective         exists monomer 30-38 kDa. However, nuclear casein <br /> medium 20 generations, cultures plated       kinase I species plants range sizes 60 <br /> on nonselective medium. All resulting colonies (&gt;400       kDa (1). Also, size difference unusual yeast <br /> colonies strain) carried plasmid marker. In contrast,     mammalian homologues; yeast products N- <br /> identical experiments single yckl (LRB263) yck2           C-terminal extensions relative mammalian coun- <br /> (LRB266) mutant strains wild-type strain JC482            terparts (37-39). The high degree sequence similarity <br /> carrying pLS2.33 resulted 30-50% plasmid retention.       YCK predicted products rabbit casein kinase <br /> These results indicate plasmid loss results inviability   I indicates proteins members same <br /> and YCK function required mitotic growth. <br />    YCK2 Affects Casein Kinase I Activity. The extensive se-       protein kinase subfamily. <br /> quence similarity YCK gene products rabbit           Biochemical studies casein kinase I activity yeast <br /> casein kinase I suggests YCKI YCK2 encode casein         detected multiple isoforms 30-50 kDa (10-12). The <br /> kinase I species. We tested effects increased    YCKI YCK2 products predicted larger, and <br /> dosage genes, YCK2, casein kinase I          relationship activities clear. We observed <br /> activity. A synthetic peptide substrate phosphorylated    additional cross-hybridizing sequences genomic Southern <br /> specifically mammalian casein kinase I (8) used         blots, hybridized YCKJ YCK2 sequence (L.C.R., <br /> assay casein kinase I activity yeast cell extracts. Extracts   unpublished work). These related sequences encode <br /> from cells overexpressing YCK2 control strains           related proteins different biological function. <br /> carrying vector assayed. Increased gene dosage           We understand YCK genes isolated <br /> YCK2 correlated 2- 3-fold increase casein        using genetic screens described here. The ability casein <br /> kinase I activity (Fig. 4).                                       kinase I homologues suppress raffinose growth defect <br />                                                                   caused decreased SNF1 protein kinase activity be <br />                        DISCUSSION                                 simply promiscuous phosphorylation overex- <br /> In report, structurally functionally     pressed YCK products compensates loss phos- <br /> homologous genes isolated independent genetic        phorylation SNF1. However, observation in- <br /> screens. Together, genes essential yeast cell       creased dosage YCKI YCK2 allows growth snf4 <br /> growth. The predicted products genes              mutant raffinose significantly restoring invertase <br /> extensive sequence similarity partial sequence obtained        activity suggests mode suppression. The snf4 <br />  32        Genetics: Robinson et al.                                                   Proc. Natl. Acad. Sci. USA 89 (1992) <br />                                                                            Tam, S. W. &amp; Wang, J. H. (1982) Biochem. Biophys. Res. <br />                                                                            Commun. 107, 676-683. <br />           C                                                             3. Mackie, K., Sorkin, B. C., Nairn, A. C., Greengard, P., Edel- <br />                 Ii1500                                                     man, G. M. &amp; Cunningham, B. A. (1989) J. Neurosci. 9, <br />                                                                              1883-18%. <br />                                                                          4. Haas, D. W. &amp; Hagedorn, C. H. (1991) Arch. Biochem. Bio- <br />                 CE1000                                                       phys. 284, 84-89. <br />                                                                          5. Pendergast, A. M. &amp; Traugh, J. A. (1985) J. Biol. Chem. 260, <br />                 O                                                            11769-11774. <br />                                                                          6. Flotow, H., Graves, P. R., Wang, A., Fiol, C. J., Roeske, <br />             0                                                                R. W. &amp; Roach, P. J. (1990) J. Biol. Chem. 265, 14264-14269. <br />                                                                           7. Roach, P. J. (1990) FASEB J. 4, 2961-2968. <br />                      0                                   --               8. Flotow, H. &amp; Roach, P. J. (1991) J. Biol. Chem. 266, 3724- <br />                                                                              3727. <br />                          0    5       10      15        20         <br /> </body></html>