<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                        Europe PMC Funders Group <br />                                         Author Manuscript <br />                                         Cell. Author manuscript; available PMC 2015 April 14. <br />                                               Published final edited form as: <br />                                                Cell. 2013 May 23; 153(5): 1064&#226;&#8364;&#8220;1079. doi:10.1016/j.cell.2013.04.055. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                               The eEF2 Kinase Confers Resistance <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> Deprivation by <br />                                               Blocking Translation Elongation <br />                                               Gabriel Leprivier1, Marc Remke2,3,4, Barak Rotblat1, Adrian Dubuc2,3, Abigail-Rachele F. <br />                                               Mateo3,6, Marcel Kool4, Sameer Agnihotri2, Amal El-Naggar1, Bin Yu6, Syam Prakash <br />                                               Somasekharan1, Brandon Faubert7, Ga&#195;&#171;lle Bridon8, Cristina E. Tognon1, Joan Mathers1, <br />                                               Ryan Thomas1, Amy Li1, Adi Barokas1, Brian Kwok1, Mary Bowden9, Stephanie Smith9, <br />                                               Xiaochong Wu2,3, Andrey Korshunov10, Thomas Hielscher5, Paul A. Northcott2, Jason D. <br />                                               Galpin11, Christopher A. Ahern11, Ye Wang12, Martin G. McCabe13,14, V. Peter Collins14, <br />                                               Russell G. Jones7, Michael Pollak12, Olivier Delattre15, Martin E. Gleave9, Eric Jan16, Stefan <br />                                               M. Pfister4,17, Christopher G. Proud18, W. Brent Derry3,6, Michael D. Taylor2,3, Poul H. <br />                                               Sorensen1,* <br />                                               1Department  Molecular Oncology, British Columbia Cancer Research Centre Department <br />                                               Pathology, University British Columbia (UBC), Vancouver, BC V5Z1L4, Canada <br />                                               2TheArthur Sonia Labatt Brain Tumor Research Centre Division Neurosurgery, <br />                                               Hospital Sick Children (HSC), University Toronto, ON M5G1L7, Canada <br />                                               3Program     Developmental Stem Cell Biology, HSC, University Toronto, ON M5G1X8, <br />                                               Canada <br />                                               4Divisionof Pediatric Neurooncology, German Cancer Research Center (DKFZ), 69120 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                               Heidelberg, Germany <br />                                               5Division   Biostatistics, German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany <br />                                               6Department      Molecular Genetics, University Toronto, ON M5G2M9, Canada <br />                                               7Goodman Cancer Research Centre, Department Physiology, McGill University, Montreal, QC <br />                                               H3G1Y6, Canada <br />                                               8Metabolomics <br />                                                           Core Facility, Goodman Cancer Research Centre, McGill University, Montreal, QC <br />                                               H3A1A3, Canada <br />                                               9Departmentof Urological Sciences, UBC Vancouver Prostate Centre, Vancouver, BC <br />                                               V6H3Z6, Canada <br />                                               10Department       Neuropathology, University Heidelberg, DKFZ, 69120 Heidelberg, Germany <br />                                               11Department       Molecular Physiology Biophysics, University Iowa, Iowa City, IA 52242, <br />                                               USA <br />  <br />  <br />  <br />                                               &#194;&#169;2013 Elsevier Inc. <br />                                               * <br />                                                Correspondence: psor@mail.ubc.ca. <br />                                               SUPPLEMENTAL INFORMATION <br />                                               Supplemental Information includes Extended Experimental Procedures seven figures article online at <br />                                               http://dx.doi.org/10.1016/j.cell.2013.04.055. <br />                                          Leprivier et al.                                                                                             Page 2 <br />  <br />                                             12Lady    Davis Research Institute McGill University, Montreal, QC H3T1E2, Canada <br />                                             13School       Cancer Enabling Sciences, University Manchester, Manchester M204BX, UK <br />                                             14Department       Pathology, University Cambridge, Cambridge CB21QP, UK <br />                                             15INSERM        U830, Laboratoire G&#195;&#169;n&#195;&#169;tique et Biologie des Cancers, Institut Curie, 75248 Paris, <br />                                             France <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             16Department       Biochemistry Molecular Biology, UBC, Vancouver, BC V6T1Z3, Canada <br />                                             17Department  Pediatric Oncology, Hematology Immunology, University Hospital <br />                                             Heidelberg, 69120 Heidelberg, Germany <br />                                             18Centre       Biological Sciences, University Southampton, Southampton SO171BJ, UK <br />  <br />                                             SUMMARY <br />                                                   Metabolic adaptation essential cell <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>survival</span> <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation. We report that <br />                                                   eukaryotic elongation factor 2 kinase (eEF2K), activated AMP-kinase (AMPK), <br />                                                   confers <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>cell survival</span> acute <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> depletion blocking translation elongation. Tumor cells <br />                                                   exploit pathway adapt <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation reactivating AMPK-eEF2K axis. <br />                                                   Adaptation transformed cells nutrient withdrawal severely compromised cells lacking <br />                                                   eEF2K. Moreover, eEF2K knockdown restored sensitivity acute nutrient deprivation highly <br />                                                   resistant human tumor cell lines. In vivo, overexpression eEF2K rendered murine tumors <br />                                                   remarkably resistant caloric restriction. Expression eEF2K strongly correlated overall <br />                                                   survival human medulloblastoma glioblastoma multiforme. Finally, C. elegans strains <br />                                                   deficient efk-1, eEF2K ortholog, severely compromised response nutrient <br />                                                   depletion. Our data highlight conserved role eEF2K protecting cells nutrient <br />                                                   deprivation conferring tumor cell adaptation metabolic stress. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             INTRODUCTION <br />                                                                Nutrient deprivation (ND) severe physiological stress dire consequences cell <br />                                                                viability. Living organisms evolved molecular mechanisms respond ND, <br />                                                                including metabolic reprogramming preserve energy balance (Caro-Maldonado and <br />                                                                Mu&#195;&#177;oz-Pinedo, 2011). A key mediator highly conserved energy sensor AMP-activated <br />                                                                protein kinase (AMPK), activated cellular AMP:ATP ADP:ATP ratios <br />                                                                increase (Hardie, 2011). AMPK limits energy-consuming processes proliferation <br />                                                                protein synthesis induces catabolic processes glycolysis fatty acid <br />                                                                oxidation preserve energy (Hardie, 2011). Another critical nutrient sensor mammalian <br />                                                                target rapamycin complex 1 (mTORC1), regulated ATP amino acid <br />                                                                levels (Zoncu et al., 2011). This complex couples nutrient abundance control protein <br />                                                                synthesis phosphorylation 4EBP1 p70S6K (Hay Sonenberg, 2004). <br />                                                                When nutrient availability compromised, mTORC1 inactivated, AMPK <br />                                                                (Inoki et al., 2003), blocking protein synthesis, energy-demanding process <br />                                                                cell (Buttgereit Brand, 1995). <br />  <br />  <br />  <br />  <br />                                                                        Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                            Page 3 <br />  <br />  <br />                                                            Pathologic ND occurs hypoxia early stages tumor development new <br />                                                            blood vessels form later stages abnormal tumor vasculature (Nagy et al., 2009). <br />                                                            While metabolic stress prevents tumor development inducing growth arrest necrosis, <br />                                                            select metabolically adapted cells form aggressive tumors (Jones and <br />                                                            Thompson, 2009). Proto-oncogenes MYC, AKT, mTOR stimulate anabolic <br />                                                            metabolism sensitize cells ND (Buzzai et al., 2005; Choo et al., 2010; Shim et al., 1998). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            This argues that, balance initial oncogenic events driving energy-demanding processes <br />                                                            proliferation, tumors develop adaptive responses protect cells ND <br />                                                            (Jones Thompson, 2009). Several factors linked responses, including <br />                                                            ATF4, NF&#206;&#186;B, CPT1C, affect amino acid synthesis, mitochondrial respiration, <br />                                                            fatty acid oxidation, respectively (Mauro et al., 2011; Ye et al., 2010; Zaugg et al., <br />                                                            2011). However, understanding process incomplete, uncovering the <br />                                                            molecular pathways involved critical potential therapeutic targeting tumors. <br />  <br />                                                            In study, report eukaryotic translation elongation factor 2 kinase (eEF2K) a <br />                                                            conserved mediator cellular response ND. EEF2K inhibits activity translation <br />                                                            elongation factor eEF2, mediates translocation stage translation elongation, <br />                                                            polypeptidyl-tRNAs A P site ribosome (Carlberg et al., <br />                                                            1990). Activity eEF2K tightly controlled nutrient availability, notably direct <br />                                                            positive regulation AMPK inhibition mTORC1 Ras-Erk-p90RSK pathways <br />                                                            (Proud, 2007). In absence nutrients, eEF2K activated phosphorylate and <br />                                                            inactivate eEF2 (Ryazanov et al., 1988), blocking energy-demanding messenger <br />                                                            RNA (mRNA) translation elongation (Carlberg et al., 1990). Our data demonstrate critical <br />                                                            role eEF2K protecting normal tissues acute ND inhibition eEF2 and <br />                                                            pathway exploited tumor cells adapting metabolic stress. <br />  <br />  <br />                                             RESULTS <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             Oncogenic Transformation Sensitizes Fibroblasts Acute ND Association with <br />                                             Defective eEF2 Signaling <br />                                                            We tested effects oncogenic transformation responses acute ND using National <br />                                                            Institutes Health (NIH) 3T3 fibroblasts transformed activated K-RasV12 (RasV12) or <br />                                                            ETV6-NTRK3 (EN) chimeric tyrosine kinase (Knezevich et al., 1998). Both <br />                                                            oncoproteins constitutively activate Ras-Erk PI3K-Akt (Tognon et al., 2002), allowing <br />                                                            study pathways impact acute responses ND. Transformed fibroblasts <br />                                                            cultured media lacking glucose, amino acids, serum showed massive cell death <br />                                                            compared nontransformed control cells ND (Figures 1A S1A available online). <br />                                                            Apoptosis confirmed Annexin V staining (Figure S1B). Glucose depletion alone <br />                                                            induced cell death transformed cells, amino acid depletion little effect <br />                                                            (Figure S1C). Nevertheless, withdrawal glucose amino acids significantly <br />                                                            enhanced cell death transformed cells (Figure S1C). Increased cell death linked <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                            increased proliferation reactive oxygen species (ROS), similarly reduced or <br />                                                            increased, respectively, control transformed cells ND (Figures S1D S1E). <br />                                                            Acute ND precipitously reduced ATP levels control cells, but, unexpectedly, not <br />                                                            observed transformed cells (Figure S1F), arguing severe energetic crisis the <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                           Page 4 <br />  <br />  <br />                                                            latter. Finally, autophagy similarly induced cell lines based LC3-II <br />                                                            accumulation (data shown), ruling differential autophagy flux basis the <br />                                                            observed <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br />  <br />                                                            We assessed mTORC1 signaling, rapidly profoundly inhibited all <br />                                                            cell lines ND (Figure 1C). However, activation key energy sensor, <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            AMPK, compromised transformed cells ND, indicated reduced <br />                                                            phosphorylation AMPK acetyl CoA-carboxylase (ACC) (Figure 1C). Moreover, <br />                                                            phosphorylation eEF2 Thr56, known marker AMPK activation, markedly <br />                                                            blunted ND transformed versus control cells (Figure 1C). EEF2 essential for <br />                                                            translation elongation (Buttgereit Brand, 1995), Thr56 phosphorylated eEF2 <br />                                                            kinase (eEF2K), known kinase eEF2, block eEF2 activity (Carlberg et al., <br />                                                            1990). This suggested eEF2 activity partially retained transformed cells under <br />                                                            ND. In addition, phosphorylation eEF2K Ser366, catalyzed p90RSK downstream of <br />                                                            Ras-Erk inhibit eEF2K (Wang et al., 2001), retained transformed cells ND <br />                                                            (Figure 1C). <br />  <br />                                                            These results point eEF2K inhibition deregulated eEF2 activity transformed cells <br />                                                            ND. We examined contributions Ras-Erk-p90RSK AMPK this <br />                                                            process, regulate eEF2K (Proud, 2007). Chemical inhibitors p90RSK (PD184352 <br />                                                            BI-D1870; data shown) dominant-negative p90RSK (DN-RSK; Figure S1G) each <br />                                                            reduced eEF2K-Ser366 phosphorylation markedly enhanced eEF2 phosphorylation in <br />                                                            EN RasV12 cells ND. Next, ectopically activated AMPK ND using <br />                                                            AICAR SMER28, chemical inducers AMPK, constitutively active AMPK. Each <br />                                                            strongly induced eEF2 phosphorylation changing eEF2K mRNA (data shown) <br />                                                            protein levels (Figures S2A S2B). Therefore, eEF2K activity deregulated in <br />                                                            transformed cells ND sustained Ras-Erk-p90RSK activity reduced AMPK <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            activation. The likely occurs constitutively high Ras activity RasV12 or <br />                                                            EN-transformed cells (Figure 1C). To explore basis defective AMPK activation, we <br />                                                            measured AMP:ATP ADP:ATP ratios, known physiological inducers AMPK <br />                                                            (Hardie, 2011), ATP levels elevated transformed cells (Figure S1F). <br />                                                            Whereas ratios elevated control cells ND correlated AMPK <br />                                                            induction (Figure 1C), ratios virtually unchanged transformed cells (Figures 1D and <br />                                                            S2C). Therefore EN- RasV12-transformed cells unexpectedly exhibit defect the <br />                                                            energy response ND maintaining high ATP failing raise AMP:ATP and <br />                                                            ADP:ATP ratios. To confirm sustained ATP levels inhibit AMPK activation, we <br />                                                            blocked ATP production 2-deoxyglucose (2-DG) rotenone. Both agents, an <br />                                                            Akt inhibitor, induced AMPK activity ND markedly increased eEF2 <br />                                                            phosphorylation (Figure 1E). Together, indicates transformation sensitizes <br />                                                            fibroblasts ND association defects AMPK p90RSK regulation eEF2. <br />  <br />                                             Adaptation Transformed Cells Chronic ND Correlates Increased AMPK-eEF2K <br />                                             Pathway Activation <br />                                                            We hypothesized rare transformed cells adapt acquire resistance to <br />                                                            prolonged ND. Therefore, transformed cells subjected repeated cycles prolonged <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 5 <br />  <br />  <br />                                                            ND followed nutrient resupplementation (Figure 1F Extended Experimental <br />                                                            Procedures). Indeed, weeks, stable populations &#226;&#8364;&#339;selected&#226;&#8364;? EN RasV12 <br />                                                            cells, designated EN selected (EN-S) RasV12 selected (RasV12-S), respectively, emerged <br />                                                            capacity survive acute ND (Figure 1F), retained multiple <br />                                                            passages (tested serial passages). Importantly, unselected EN- RasV12- <br />                                                            transformed cells subjected adaptation protocol glucose amino <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            acids (EN-S+Glc+AA RasV12-S+Glc+AA), equally sensitive ND parental <br />                                                            counterparts (Figure S2D), indicating adaptive reprogramming requires chronic <br />                                                            exposure ND. Notably, EN-S RasV12-S cells retained soft-agar-colony-forming <br />                                                            activity (data shown) tumorigenicity (Figure S2E). As observed unselected cells, <br />                                                            EN-S RasV12-S cells rapidly shut mTORC1 signaling acute ND and <br />                                                            exhibited similar levels Akt activation autophagic flux compared unselected <br />                                                            counterparts (Figures 1G S2F). In contrast, displayed increased AMPK ACC <br />                                                            phosphorylation ND, suggesting that, like non-transformed cells, selected cells had <br />                                                            reacquired ability activate AMPK ND (Figure 1G). This correlated with <br />                                                            massively increased eEF2 phosphorylation, altered eEF2K transcript <br />                                                            protein levels (Figures 1G S2G). This confirms correlation eEF2 <br />                                                            phosphorylation survival capacity acute ND. This inhibition the <br />                                                            Ras-Erk-p90RSK pathway, eEF2K Ser366 phosphorylation preserved selected <br />                                                            cells ND (Figure 1G). However, blocking AMPK activity dominant-negative <br />                                                            AMPK (DN-AMPK) markedly reduced eEF2 phosphorylation selected cells (Figure 1H) <br />                                                            significantly increased ND-induced cell death (Figure 1I), highlighting importance <br />                                                            AMPK reinduction resistance ND. Mechanistically, changes in <br />                                                            LKB1 expression (Figure 1G), directly activates AMPK (Hardie, 2011). However, <br />                                                            AMP:ATP ADP:ATP ratios markedly increased EN-S RasV12-S cells <br />                                                            ND compared parental EN RasV12 cells (Figures 1D S2C). Therefore, <br />                                                            unlike nonselected tumor cells, selected cells able raise AMP:ATP ADP:ATP <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            ratios nutrient depleted, strongly suggesting adaptation EN RasV12 cells to <br />                                                            ND linked reactivation AMPK-eEF2K axis. <br />  <br />                                             Translation Elongation Is Deregulated Transformed Fibroblasts ND Restricts <br />                                             Cell Survival <br />                                                            We asked eEF2 deregulation impacts protein synthesis ND, predicting <br />                                                            high eEF2 activity sustain protein synthesis ND, sensitizing <br />                                                            nonselected tumor cells apoptosis. However, rates global protein synthesis were <br />                                                            equivalently reduced ND transformed nontransformed cells using [35S]- <br />                                                            methionine/cysteine incorporation (Figure 2A) pulse-labeling L-azidohomoalanine <br />                                                            (AHA) (Somasekharan et al., 2012) (Figure S3A). This consistent observed block <br />                                                            mTORC1 signaling ND (Figure 1C) argues major role sustained <br />                                                            global protein synthesis hypersensitivity nonselected transformed cells ND. It is <br />                                                            noteworthy low level protein synthesis observed 6 hr ND <br />                                                            (Figure S3B). <br />  <br />                                                            We examined effects ND translation initiation versus elongation analyzing <br />                                                            polysome profiles sucrose gradients. Ribosomal 40S-, 60S-, 80S-mRNA <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                            Page 6 <br />  <br />  <br />                                                            complexes represent translationally inactive complexes (subpolysomal fractions), whereas <br />                                                            polyribosome (polysome) fractions correspond translationally active complexes. ND <br />                                                            markedly increased 60S fractions control transformed cells, indicating global <br />                                                            block translation initiation (Figure 2B). This consistent loss 4EBP1 <br />                                                            phosphorylation (Figure 1C), blocks cap-dependent translation initiation, major <br />                                                            rate-limiting step mRNA translation (Mathews et al., 2007). Whereas ratios polysomal <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            subpolysomal fractions (P/S) significantly affected ND nontransformed <br />                                                            cells (4% &#194;&#177; 0.2% reduction), P/S ratios dramatically decreased EN RasV12 cells <br />                                                            (82% &#194;&#177; 2% 91% &#194;&#177; 3%, respectively) ND (Figure 2B). Because initiation is <br />                                                            blocked, strongly suggests that, transformed cells, ribosomes continuously <br />                                                            running transcripts (i.e., minimal ribosome stalling) ND, leading to <br />                                                            polysomal disaggregation sustained translation elongation, decreasing P/S ratios <br />                                                            (Mathews et al., 2007). Furthermore, ND, eEF2 remained unphosphorylated <br />                                                            state mouse embryonic fibroblasts (MEFs) lacking eEF2K (eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217;), indicating that <br />                                                            translation elongation maintained eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; cells ND, contrast wild-type <br />                                                            eEF2K+/+ cells eEF2 strongly phosphorylated (Figure S3C). ND correlated <br />                                                            accumulation 60S fractions eEF2K+/+ eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; cells, is <br />                                                            consistent blocked translation initiation (Figure S3D). However, P/S ratio more <br />                                                            dramatically reduced ND eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; cells (89% &#194;&#177; 2%) compared eEF2K+/+ cells <br />                                                            (49% &#194;&#177; 1%) (Figure S3D). This indicates retained eEF2 activity leads sustained <br />                                                            translation elongation polysome disaggregation ND. <br />  <br />                                                            To conclusively demonstrate differential translation elongation control versus transformed <br />                                                            cells, measured ribosome half-transit times ND. These refer time required <br />                                                            ribosome translate average-sized mRNA inverse elongation rate <br />                                                            (Fan Penman, 1970). Under ND, ribosome half-transit times ~2-fold longer in <br />                                                            nontransformed cells, indicative slower elongation rates (Figure 2C). In contrast, <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            half-transit times slightly increased ND EN RasV12 cells (i.e., ~1.25- <br />                                                            fold ~1.2-fold, respectively) (Figure 2C). This differences intracellular <br />                                                            amino acid levels control transformed cells, decreased equivalently <br />                                                            ND (Figure S3E). These data support model whereby, nonselected tumor cells <br />                                                            ND, retained eEF2 activity leads sustained translation elongation. To link to <br />                                                            cell death ND, specifically impaired translation elongation EN RasV12 cells <br />                                                            eEF2 knockdown (Figure 2D). Indeed, ND-induced cell death substantially reduced <br />                                                            transformed cells eEF2 knockdown compared small interfering RNA (siRNA) <br />                                                            controls (Figure 2E). This supports role deregulated eEF2 activity in <br />                                                            hypersensitivity transformed cells ND. In addition, EN-S/DN-AMPK RasV12- <br />                                                            S/DN-AMPK cells, highly susceptible ND (Figure 1I), rescued cell <br />                                                            death eEF2 knockdown, suggesting AMPK promotes survival adapted <br />                                                            transformed cells inhibiting translation elongation (Figures 2F 2G). Therefore, <br />                                                            sustained translation elongation renders transformed cells hypersensitive ND. <br />  <br />                                             eEF2K Is Critical Cell Survival ND <br />                                                            Because translation elongation tightly controlled eEF2K (Carlberg et al., 1990), we <br />                                                            examined impact eEF2K expression cell survival ND. Similar to <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 7 <br />  <br />  <br />                                                            nonselected tumor cells, eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs highly sensitive ND compared with <br />                                                            eEF2K+/+ MEFs underwent apoptotic cell death ND (Figures 3A S4A). In <br />                                                            contrast previous reports (Wu et al., 2006), associated defects in <br />                                                            autophagy (Figures S4B S4C). To establish eEF2 linked survival <br />                                                            responses ND, performed eEF2 knockdown eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; cells, harbor high <br />                                                            eEF2 activity ND (Figure S3C). This markedly increased cell survival compared to <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            siRNA controls (Figures 3B 3C). Moreover, eEF2K knockdown NIH 3T3 cells with <br />                                                            independent eEF2K siRNAs strongly sensitized cells ND (Figures 3D 3E). <br />                                                            However, coknockdown eEF2 completely rescued cells cell death (Figures 3D and <br />                                                            3E), confirming eEF2K mediates survival ND control eEF2. <br />                                                            Conversely, stable overexpression eEF2K EN- RasV12-transformed NIH 3T3 cells <br />                                                            led increased eEF2 phosphorylation cell survival ND (Figures 3F 3G). This <br />                                                            observed kinase-dead eEF2K, implying eEF2 phosphorylation and <br />                                                            inhibition translation elongation critical survival ND (Figures 3G S4D). <br />  <br />                                                            To directly test role eEF2K adaptive response transformed fibroblasts ND, <br />                                                            performed eEF2K knockdown EN-S RasV12-S cells (which high phospho- <br />                                                            eEF2 levels; Figure 1G) using independent siRNAs. This blocked eEF2 phosphorylation <br />                                                            (Figure 3H) rendered cells significantly susceptible ND compared siRNA <br />                                                            control cells (Figure 3I). Finally, eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs transformed EN RasV12 exhibited <br />                                                            similar growth vitro ambient conditions eEF2K+/+ counterparts (Figures <br />                                                            S4E S4F) similarly tumorigenic athymic mouse (Figure S4G). However, <br />                                                            cells subjected adaptation protocol (Figure 1F), transformed <br />                                                            eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs strikingly reduced capacity adapt repeated ND compared to <br />                                                            transformed eEF2K+/+ MEFs (Figure 3J). These findings clearly establish eEF2K a <br />                                                            critical survival factor ND adaptive response transformed fibroblasts to <br />                                                            nutrient stress. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             eEF2K Is Critical Survival Human Tumor Cells Nutrient Stress <br />                                                            To extend findings human cells, screened tumor cell lines eEF2 <br />                                                            phosphorylation ND. This identified lines widely divergent levels: HeLa <br />                                                            cells showed minimal eEF2 phosphorylation, MG63 osteosarcoma cells exhibited <br />                                                            high eEF2 phosphorylation ND (Figure 4A). HeLa cells exhibited extremely low <br />                                                            AMPK activity ND reported (Liang et al., 2007), whereas, MG63 cells, AMPK <br />                                                            activity high (Figure 4A). This tightly associated survival: HeLa cells were <br />                                                            highly sensitive ND, MG63 cells completely resistant cell death <br />                                                            conditions (Figures 4B 4C), HeLa cells particularly sensitive to <br />                                                            glucose withdrawal (Figure S5A). To link eEF2 translation elongation, we <br />                                                            knocked eEF2 HeLa cells, significantly reduced cell death ND <br />                                                            compared siRNA controls (Figures 4D 4E). Moreover, stable eEF2K overexpression <br />                                                            increased eEF2 phosphorylation cell survival HeLa cells ND compared to <br />                                                            parental cells (Figures 4F, 4G, S5B). This strongly argues hypersensitivity HeLa <br />                                                            cells related reduced eEF2K activity sustained translation elongation. Knockdown <br />                                                            eEF2K, reduced eEF2 phosphorylation compared controls (Figure 4H), strongly <br />                                                            increased sensitivity MG63 cells ND (Figures 4I S5C). This phenocopied by <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 8 <br />  <br />  <br />                                                            knockdown AMPK-subunits &#206;&#177;1/&#206;&#177;2, sensitized MG63 cells ND-induced cell <br />                                                            death, supporting importance AMPK-eEF2K axis protecting human tumor cells <br />                                                            ND (Figures S5D S5E). Overall, results demonstrate eEF2K critical <br />                                                            survival human tumor cells ND. <br />  <br />                                             eEF2K Protects Tumors Caloric-Restriction-Induced Cell Death In Vivo <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            To determine relevance findings vivo, EN- RasV12-transformed NIH 3T3 <br />                                                            cells overexpressing eEF2K MSCV vector control subcutaneously implanted in <br />                                                            nu/nu immunocompromised mice. Mice fed standard ad libitum (AL) diet or, <br />                                                            prior tumor development, caloric restriction (CR) corresponding 60% caloric intake of <br />                                                            AL diet. As described (Kalaany Sabatini, 2009), CR led significant reductions of <br />                                                            blood IGF1, insulin, glucose levels (Figures S6A&#226;&#8364;&#8220;S6C). Tumor sizes EN RasV12 <br />                                                            vector control mice severely reduced (by ~50%) CR compared AL <br />                                                            (compare MSCV bars Figures 5A 5B). Although eEF2K-overexpressing tumors were <br />                                                            smaller control tumors AL diets, likely reduced translation elongation and <br />                                                            overall growth, sizes virtually unaffected CR (Figures 5A, 5B, and <br />                                                            S6D). Notably, EN/eEF2K RasV12/eEF2K tumors eventually grew larger sizes than <br />                                                            controls mice reverted AL diet following CR (Figures S6E S6F), <br />                                                            highlighting survival advantage high eEF2K expression tumors. Morphologic <br />                                                            examination revealed massive necrosis EN RasV12 vector control tumors after <br />                                                            CR compared AL diets. In dramatic contrast, EN/eEF2K RasV12/eEF2K tumors <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                            CR animals exhibited apparent necrosis (Figures 5C S6G). In EN RasV12 control <br />                                                            tumors, CR massively increased apoptosis measured IHC cleaved caspase-3, which <br />                                                            observed EN/eEF2K RasV12/eEF2K tumors (Figures 5D S6H), further <br />                                                            accentuating protective effect eEF2K vivo. We assessed eEF2K activity AL <br />                                                            versus CR tumors analyzing phospho-eEF2 levels IHC immunoblotting. <br />                                                            Phosphorylation eEF2 poorly induced CR EN RasV12 control tumors (see <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            MSCV panels Figures 5E, 5F, S6I, S6J), correlated sustained eEF2K <br />                                                            inhibition CR indicated retention p90RSK-mediated eEF2K Ser366- <br />                                                            phosphorylation (Figure S6K). In contrast, eEF2 phosphorylation strongly induced in <br />                                                            EN/eEF2K RasV12/eEF2K tumors AL CR pronounced <br />                                                            CR (see eEF2K panels Figures 5E, 5F, S6I, S6J). This confirmed under <br />                                                            CR using eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; +/+ MEFs transformed RasV12 (Figure S6L). Although tumor <br />                                                            sizes reduced CR RasV12/eEF2K+/+ MEF tumors, effects significant <br />                                                            RasV12/eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEF tumors (Figure 5G). CR led significantly increased <br />                                                            necrosis apoptosis RasV12/eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; compared RasV12/eEF2K+/+ tumors <br />                                                            (Figures 5H 5I), indicating genetic loss eEF2K increases sensitivity of <br />                                                            tumors CR. These results consistent model eEF2K activity protects <br />                                                            tumors CR-induced cell death vivo. <br />  <br />                                             Expression eEF2K Is Associated Poor Prognosis Medulloblastoma and <br />                                             Glioblastoma Multiforme <br />                                                            To probe findings primary human tumors, focused medulloblastoma (MB), <br />                                                            common malignant brain tumor childhood, glioblastoma multiforme <br />                                                            (GBM), highly malignant adult brain tumor, eEF2K expression data were <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                            Page 9 <br />  <br />  <br />                                                            publically available tumors (Northcott et al., 2011). In MB, high eEF2K transcript <br />                                                            levels correlated aggressive metastatic subtype MB, group 3 <br />                                                            tumors (Northcott et al., 2011). Gene expression profiles independent MB cohorts <br />                                                            (total n = 286) demonstrated highly significant upregulation eEF2K group 3 relative to <br />                                                            non-group-3 tumors (Figures 6A 6B). Furthermore, high eEF2K expression correlated <br />                                                            significantly decreased overall survival disease variants (p = 0.00003; Figure 6C), <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            aggressive non-WNT/non-SHH subtypes (p = 0.002; Figure 6D). EEF2K <br />                                                            expression significantly upregulated GBM compared normal human brain tissue <br />                                                            (~2.5-fold; Figure 6E). Among GBM subtypes, eEF2K expression specifically <br />                                                            increased classical mesenchymal subtypes (Figure 6F), associated <br />                                                            poor overall progression-free survival (Verhaak et al., 2010). EEF2K expression <br />                                                            strongly correlated decreased overall survival human glioma subtypes <br />                                                            (astrocytoma, oligodendroglioma, mixed glioma, GBM) (p = 0.0001; Figure 6G), as <br />                                                            GBM itself (p = 0.001; Figure 6H). Together, findings highlight eEF2K <br />                                                            expression prognostic biomarker aggressive human brain tumors. <br />  <br />                                                            We evaluated eEF2K activity MB tumor samples IHC phospho-eEF2. In a <br />                                                            mouse model MB (Wu et al., 2012), primary tumors (Figure 6I, panels 1 2) and <br />                                                            matching metastases (Figure 6I, panels 3 4) showed extensive tumor-specific eEF2 <br />                                                            phosphorylation (i.e., inactivation), adjacent normal cerebellar tissue negative. <br />                                                            Similar findings observed primary human MB tumors (n = 7) (Figure 6J). Thus, <br />                                                            eEF2K activity strongly induced MB tissues, primary tumor metastatic <br />                                                            compartments, compared normal cerebellar tissue. Finally, tested potential <br />                                                            involvement eEF2K response MB cells ND. The group 3 human MB d283 <br />                                                            cell line highly resistant ND (Figures S7A S7B). However, d283 cells transfected <br />                                                            eEF2K siRNAs exhibited significantly higher cell death ND compared to <br />                                                            controls (Figures 6K S7C). This suggests eEF2K promotes survival MB <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            cells ND. Together, findings indicate eEF2K expression predictive of <br />                                                            outcome MB GBM eEF2K activity protects MB cells ND. <br />  <br />                                             The eEF2K Ortholog efk-1 Is Critical Survival C. elegans ND <br />                                                            To test protective role eEF2K conserved, examined physiological <br />                                                            responses ND vivo using nematode worm Caenorhabditis elegans. In absence <br />                                                            nutrients, newly hatched larval stage (L1) C. elegans worms enter dormant state <br />                                                            termed &#226;&#8364;&#339;L1 diapause,&#226;&#8364;? development halted survival sustained 2 3 <br />                                                            weeks (Johnson et al., 1984), providing useful model study survival mechanisms <br />                                                            ND independently concomitant developmental programs. We assessed deletion <br />                                                            allele C. elegans eEF2K ortholog, efk-1(ok3609), survival starved L1 larvae to <br />                                                            determine conserved role efk-1 managing ND. Whereas deletion <br />                                                            efk-1 minor effect lifespans animals grown nutrient-rich <br />                                                            conditions, survival efk-1(ok3609) substantially reduced ND compared with <br />                                                            wild-type (wt) (N2) worms (Figures 7A 7B). Indeed, mean lifespan N2 animals in <br />                                                            absence nutrients 13.1 days, efk-1 mutant worms mean lifespan of <br />                                                            7.4 days (p &lt; 0.00058; Figure 7B). Moreover, efk-1 transcripts markedly induced <br />                                                            ND wt L1 larvae (Figure 7C), eEF2K levels various mouse human <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                           Page 10 <br />  <br />  <br />                                                            cell lines subjected ND (Figure 7D). These findings provide compelling evidence the <br />                                                            eEF2K ortholog efk-1 component physiological response C. elegans ND and, <br />                                                            thus, role eEF2K pathway highly conserved. <br />  <br />  <br />                                             DISCUSSION <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             efk-1/eEF2K Is Component Stress Response ND <br />                                                            Nutrient deprivation activates multiple transcriptional cascades reprogram metabolism <br />                                                            nutrients limited, induction AMPK subunit &#206;&#177;-2 (prkaa-2/aak-2) (Baugh <br />                                                            et al., 2009; Jagoe et al., 2002). Here, report mammalian eEF2K C. elegans <br />                                                            ortholog efk-1 ND-responsive genes components nutrient stress adaptive <br />                                                            program. In mice, eEF2K expression upregulated neonates deprived of <br />                                                            transplacental nutrient supply (Sakagami et al., 2002). Along findings, suggests <br />                                                            efk-1/eEF2K evolved preserve cell survival during <br />                                                            physiological ND (Figure 7E). Given eEF2K blocks mRNA translation elongation, this <br />                                                            reflect need block translation elongation initiation optimal <br />                                                            adaptive response. Indeed, 4EBP1 expression increases ND (Jagoe et al., 2002) to <br />                                                            support adaptation metabolic stress, including ND hypoxia (Braunstein et al., 2007; <br />                                                            Teleman et al., 2005). <br />  <br />                                             eEF2K Blocks Translation Elongation Protect Cells ND <br />                                                            Inhibiting global mRNA translation increases resistance ND vivo (Pan et al., 2007; <br />                                                            Teleman et al., 2005). Indeed, mTORC1 inactivated ND preserve energy balance <br />                                                            amino acids (Choo et al., 2010; Teleman et al., 2005). Our findings suggest that <br />                                                            mTORC1 inhibition insufficient fully protect cells ND eEF2K <br />                                                            activation critical. The relies inhibition eEF2K substrate, eEF2, <br />                                                            supporting notion translation initiation elongation inhibited to <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            promote survival (Figure 7E). Unexpectedly, inhibition translation elongation not <br />                                                            linked preservation ATP altered ROS autophagic flux. Moreover, inhibition <br />                                                            elongation did affect global protein synthesis rates nutrient-deprived cells, as <br />                                                            overall protein synthesis strongly reduced block translation <br />                                                            initiation. Nevertheless, eEF2K activation does appear promote stalling ribosomes on <br />                                                            transcripts, secondary blocked translation elongation. This suggests that, absence of <br />                                                            eEF2K activation, polypeptides elongate ND transcripts even <br />                                                            bypass block initiation alternative initiation mechanisms (cap-independent or <br />                                                            uORF-based processes), supporting continued translation nutrient stress. <br />                                                            Therefore, eEF2K block synthesis specific subgroups proteins instead affecting <br />                                                            overall synthesis. This particularly relevant include proapoptotic factors or <br />                                                            anabolic drivers inhibition critical adaptation ND. In keeping this <br />                                                            notion, 4E-BP, protective ND (Teleman et al., 2005), blocks synthesis of <br />                                                            specific subsets proteins involved proliferation polyamine biosynthesis (Dowling <br />                                                            et al., 2010). <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                           Page 11 <br />  <br />  <br />                                             eEF2K Is Hijacked Tumor Cells Adaptation ND <br />                                                            During tumor development, transformed cells adapt metabolic stress ND <br />                                                            anabolic processes typically driven oncogenic pathways deleterious <br />                                                            effects cell nutrients scarce (Jones Thompson, 2009). Our vitro <br />                                                            studies indicate hypersensitivity RasV12- EN-transformed cells ND a <br />                                                            block eEF2K activation combination sustained Ras-Erk-p90RSK signaling <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            dampened AMPK induction. Given Ras-Erk pathway overactive ~25% of <br />                                                            human tumors, eEF2K inhibited diverse tumors, restricting their <br />                                                            capacity survive nutrients limited. However, observed RasV12- EN- <br />                                                            transformed cells adapt nutrient stress, sustained Ras-Erk activity. This is <br />                                                            agreement previous studies reporting that, oncogenic Ras primarily <br />                                                            sensitizes fibroblasts reduced nutrients (Chiaradonna et al., 2006), support <br />                                                            adaptation ND increasing Glut1 glucose uptake tumor cells (Yun et al., 2009). <br />                                                            We propose additional mechanism adaptation reactivation AMPK- <br />                                                            eEF2K axis occurs ability cells, unexpectedly, raise AMP:ATP and <br />                                                            ADP:ATP ratios. In contrast, defective AMPK activation nonadapted cells results from <br />                                                            sustained ATP levels, blocking increase AMP:ATP ADP:ATP ratios. This <br />                                                            suggests nonadapted tumor cells maintain specific anabolic processes ND <br />                                                            sustain ATP. <br />  <br />                                                            It tempting speculate eEF2K-mediated adaptation ND mimics process <br />                                                            occurring primary tumors subjected repeated cycles metabolic stress. The <br />                                                            mechanism ND-induced eEF2K activation vitro appears involve, part, <br />                                                            activation AMPK-eEF2K pathway modulation cellular AMP:ATP and <br />                                                            ADP:ATP ratios. In vivo, defects eEF2K activity rendered tumor xenografts sensitive to <br />                                                            CR reported models (Kalaany Sabatini, 2009). Although reduction of <br />                                                            glucose levels CR modest compared severe conditions imposed vitro, this <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            suggests eEF2K-deficient tumors sensitive moderate reductions in <br />                                                            glucose vivo. In addition, major effect CR reduce plasma levels insulin and <br />                                                            IGF1 (Kalaany Sabatini, 2009). Because eEF2K negatively regulated factors <br />                                                            (Wang et al., 2001), vivo necrosis eEF2K-deficient tumors CR reflect <br />                                                            enhanced sensitivity tumors decreased plasma levels insulin IGF1. We speculate <br />                                                            combined effect compromised plasma nutrient availability, including modestly <br />                                                            reduced glucose levels, coupled decreased insulin IGF1 levels, collaborate to <br />                                                            render eEF2K-deficient tumors sensitive CR. <br />  <br />                                                            In summary, data reinforce importance AMPK management nutrient stress <br />                                                            tumors, AMPK supports adaptation ND activating eEF2K blocking <br />                                                            translation elongation, addition inhibiting ACC fatty acid synthesis reported <br />                                                            (Jeon et al., 2012). Thus, eEF2K emerges critical prosurvival factor be <br />                                                            exploited tumors support adaptation metabolic stress. The development of <br />                                                            therapeutic strategies target eEF2K kinase tumors, especially aggressive human <br />                                                            brain tumors, warrants investigation. Such strategies particularly relevant if <br />                                                            combined CR mimetics 2-DG, angiogenesis inhibitors, resveratrol, eEF2K <br />                                                            selectively protect tumors CR nonstress conditions. In <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                                              Page 12 <br />  <br />  <br />                                                            conclusion, inhibition mRNA translation elongation represents important mechanism <br />                                                            tumor cells adapt ND, highlighting previously unrecognized possibilities for <br />                                                            therapeutic targeting metabolically adapted tumor cells. <br />  <br />  <br />                                             EXPERIMENTAL PROCEDURES <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                             Cell Culture Xenografts <br />                                                            NIH 3T3, MG63, HeLa, d283 cells purchased ATCC. NIH 3T3 cells stably <br />                                                            expressing EN K-RasV12 described (Ng et al., 2012). Immortalized WT and <br />                                                            eEF2K-deficient MEFs gifts Dr. Alexey Ryazanov (University Medicine and <br />                                                            Dentistry New Jersey). ND performed ~50% subconfluent cultures Hanks <br />                                                            buffered saline solution (HBSS)-HEPES (CaCl2&#194;&#183;2H2O, 0.185 g/l; MgSO4&#194;&#183;7H2O, 0.2 g/l; <br />                                                            KCl, 0.4 g/l; KH2PO4 [anhydrous], 0.06 g/l; NaHCO3, 0.35 g/l; NaCl, 8 g/l; <br />                                                            Na2HPO4&#194;&#183;7H2O, 0.09 g/l; 20 mM HEPES, pH 7.4) containing glucose, the <br />                                                            indicated times, resupplemented glucose (4.5 g/l; Fisher) and/or 1&#195;&#8212; MEM essential <br />                                                            nonessential amino acid solutions (Invitrogen) glutamine (2 mM; Invitrogen) as <br />                                                            indicated. For tumor xenografts, cells injected subcutaneously (5 &#195;&#8212; 105 cells site) <br />                                                            flanks nu/nu mice (Harlan Laboratories). CR experiments carried as <br />                                                            described (Kalaany Sabatini, 2009); mice CR group received 60% daily <br />                                                            food intake ad libitum counterparts. Nine days postinjection, tumor tissues either <br />                                                            snap frozen liquid N2 formalin fixed IHC. <br />  <br />                                             Protein Synthesis Rate <br />                                                            For [35S]-methionine-cysteine incorporation, cells pulse labeled 10 &#206;&#188;Ci [35S]- <br />                                                            methionine cysteine mix ml (EasyTag EXPRESS, Perkin Elmer) 5 min HBSS- <br />                                                            HEPES described (Ng et al., 2012). For AHA incorporation assays, cells incubated <br />                                                            methionine-free DMEM containing 10% CS 1 hr pulse labeled 50 &#206;&#188;M L- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            AHA 45 min methionine-free DMEM containing 10% CS HBSS-HEPES. Cells <br />                                                            processed described Somasekharan et al. (2012) fixed, permeabilized, <br />                                                            labeled using Click-iT Cell Reaction Buffer Kit (Invitrogen) Alexa Fluor 488 <br />                                                            azide probe (Invitrogen) sorted FACS (BD Bioscience). <br />  <br />                                             C. elegans Studies <br />                                                            C. elegans strains cultured 20&#194;&#176;C nematode growth media (NGM) plates using <br />                                                            standard protocols (Brenner, 1974). WT Bristol N2 strain worms efk-1(ok3609) deletion <br />                                                            mutants obtained Caenorhabditis Genetics Centre (CGC) National <br />                                                            Institutes Health Center Research Resources. <br />  <br />  <br />                                             Supplementary Material <br />                                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                             ACKNOWLEDGMENTS <br />                                                            We thank T. Tang, M. Robertson, C. Chow technical assistance B. Fonseca (McGill University), S. <br />                                                            Aparicio, S. Mullaly, T. Ng (UBC), J.A. Pospisilik, R. Teperino (Max Planck Institute, Freiburg) helpful <br />  <br />  <br />  <br />                                                                       Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                                           Page 13 <br />  <br />                                                            discussions. This work supported Canadian Cancer Society Research Institute British Columbia <br />                                                            Cancer Foundation (to P.H.S.) generous donations Team Finn riders Ride Conquer <br />                                                            Cancer National Institutes Health (CA159859), Terry Fox Research Institute, Pediatric <br />                                                            Brain Tumor Foundation (to M.D.T.). <br />  <br />  <br />                                             REFERENCES <br />                                                            Baugh LR, Demodena J, Sternberg PW. RNA Pol II accumulates promoters growth genes during <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               developmental arrest. Science. 2009; 324:92&#226;&#8364;&#8220;94. [PubMed: 19251593] <br />                                                            Braunstein S, Karpisheva K, Pola C, Goldberg J, Hochman T, Yee H, Cangiarella J, Arju R, Formenti <br />                                                               SC, Schneider RJ. A hypoxia-controlled cap-dependent cap-independent translation switch in <br />                                                               breast cancer. Mol. Cell. 2007; 28:501&#226;&#8364;&#8220;512. [PubMed: 17996713] <br />                                                            Brenner S. The genetics Caenorhabditis elegans. Genetics. 1974; 77:71&#226;&#8364;&#8220;94. [PubMed: 4366476] <br />                                                            Buttgereit F, Brand MD. A hierarchy ATP-consuming processes mammalian cells. Biochem. J. <br />                                                               1995; 312:163&#226;&#8364;&#8220;167. [PubMed: 7492307] <br />                                                            Buzzai M, Bauer DE, Jones RG, Deberardinis RJ, Hatzivassiliou G, Elstrom RL, Thompson CB. The <br />                                                               glucose dependence Akt-transformed cells reversed pharmacologic activation fatty <br />                                                               acid beta-oxidation. Oncogene. 2005; 24:4165&#226;&#8364;&#8220;4173. [PubMed: 15806154] <br />                                                            Carlberg U, Nilsson A, Nyg&#195;&#165;rd O. Functional properties phosphorylated elongation factor 2. Eur. J. <br />                                                               Biochem. 1990; 191:639&#226;&#8364;&#8220;645. [PubMed: 2390990] <br />                                                            Caro-Maldonado A, Mu&#195;&#177;oz-Pinedo C. Dying eat: cells respond starvation. <br />                                                               The Open Cell Signaling Journal. 2011; 3:42&#226;&#8364;&#8220;51. <br />                                                            Chiaradonna F, Sacco E, Manzoni R, Giorgio M, Vanoni M, Alberghina L. Ras-dependent carbon <br />                                                               metabolism transformation mouse fibroblasts. Oncogene. 2006; 25:5391&#226;&#8364;&#8220;5404. [PubMed: <br />                                                               16607279] <br />                                                            Choo AY, Kim SG, Vander Heiden MG, Mahoney SJ, Vu H, Yoon SO, Cantley LC, Blenis J. Glucose <br />                                                               addiction TSC null cells caused failed mTORC1-dependent balancing metabolic demand <br />                                                               supply. Mol. Cell. 2010; 38:487&#226;&#8364;&#8220;499. [PubMed: 20513425] <br />                                                            Dowling RJ, Topisirovic I, Alain T, Bidinosti M, Fonseca BD, Petroulakis E, Wang X, Larsson O, <br />                                                                Selvaraj A, Liu Y, et al. mTORC1-mediated cell proliferation, cell growth, controlled by <br />                                                                4E-BPs. Science. 2010; 328:1172&#226;&#8364;&#8220;1176. [PubMed: 20508131] <br />                                                            Fan H, Penman S. Regulation protein synthesis mammalian cells. II. Inhibition protein <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                synthesis level initiation mitosis. J. Mol. Biol. 1970; 50:655&#226;&#8364;&#8220;670. [PubMed: <br />                                                                5529301] <br />                                                            Hardie DG. AMP-activated protein kinase: energy sensor regulates aspects cell function. <br />                                                                Genes Dev. 2011; 25:1895&#226;&#8364;&#8220;1908. [PubMed: 21937710] <br />                                                            Hay N, Sonenberg N. Upstream downstream mTOR. Genes Dev. 2004; 18:1926&#226;&#8364;&#8220;1945. <br />                                                                [PubMed: 15314020] <br />                                                            Inoki K, Zhu T, Guan KL. TSC2 mediates cellular energy response control cell growth survival. <br />                                                                Cell. 2003; 115:577&#226;&#8364;&#8220;590. [PubMed: 14651849] <br />                                                            Jagoe RT, Lecker SH, Gomes M, Goldberg AL. Patterns gene expression atrophying skeletal <br />                                                                muscles: response food deprivation. FASEB J. 2002; 16:1697&#226;&#8364;&#8220;1712. [PubMed: 12409312] <br />                                                            Jeon SM, Chandel NS, Hay N. AMPK regulates NADPH homeostasis promote tumour cell survival <br />                                                                energy stress. Nature. 2012; 485:661&#226;&#8364;&#8220;665. [PubMed: 22660331] <br />                                                            Johnson TE, Mitchell DH, Kline S, Kemal R, Foy J. Arresting development arrests aging the <br />                                                                nematode Caenorhabditis elegans. Mech. Ageing Dev. 1984; 28:23&#226;&#8364;&#8220;40. [PubMed: 6542614] <br />                                                            Jones RG, Thompson CB. Tumor suppressors cell metabolism: recipe cancer growth. Genes <br />                                                                Dev. 2009; 23:537&#226;&#8364;&#8220;548. [PubMed: 19270154] <br />                                                            Kalaany NY, Sabatini DM. Tumours PI3K activation resistant dietary restriction. Nature. <br />                                                                2009; 458:725&#226;&#8364;&#8220;731. [PubMed: 19279572] <br />                                                            Knezevich SR, McFadden DE, Tao W, Lim JF, Sorensen PH. A novel ETV6-NTRK3 gene fusion in <br />                                                                congenital fibrosarcoma. Nat. Genet. 1998; 18:184&#226;&#8364;&#8220;187. [PubMed: 9462753] <br />                                                            Liang J, Shao SH, Xu ZX, Hennessy B, Ding Z, Larrea M, Kondo S, Dumont DJ, Gutterman JU, <br />                                                                Walker CL, et al. The energy sensing LKB1-AMPK pathway regulates p27(kip1) phosphorylation <br />  <br />  <br />                                                                      Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                                  Page 14 <br />  <br />  <br />                                                                mediating decision enter autophagy apoptosis. Nat. Cell Biol. 2007; 9:218&#226;&#8364;&#8220;224. <br />                                                                [PubMed: 17237771] <br />                                                            Mathews, MB.; Sonenberg, N.; Hershey, JWB. Origin principles translational control. In: <br />                                                                Mathews, MB.; Sonenberg, N.; Hershey, JWB., editors. Translational Control Biology and <br />                                                                Medicine. Cold Spring Harbor Laboratory Press; Cold Spring Harbor, NY: 2007. p. 1-40. <br />                                                            Mauro C, Leow SC, Anso E, Rocha S, Thotakura AK, Tornatore L, Moretti M, De Smaele E, Beg AA, <br />                                                                Tergaonkar V, et al. NF-&#206;&#186;B controls energy homeostasis metabolic adaptation upregulating <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                mitochondrial respiration. Nat. Cell Biol. 2011; 13:1272&#226;&#8364;&#8220;1279. [PubMed: 21968997] <br />                                                            Nagy JA, Chang SH, Dvorak AM, Dvorak HF. Why tumour blood vessels abnormal it <br />                                                                important know? Br. J. Cancer. 2009; 100:865&#226;&#8364;&#8220;869. [PubMed: 19240721] <br />                                                            Ng TL, Leprivier G, Robertson MD, Chow C, Martin MJ, Laderoute KR, Davicioni E, Triche TJ, <br />                                                                Sorensen PH. The AMPK stress response pathway mediates anoikis resistance inhibition <br />                                                                mTOR suppression protein synthesis. Cell Death Differ. 2012; 19:501&#226;&#8364;&#8220;510. [PubMed: <br />                                                                21941369] <br />                                                            Northcott PA, Korshunov A, Witt H, Hielscher T, Eberhart CG, Mack S, Bouffet E, Clifford SC, <br />                                                                Hawkins CE, French P, et al. Medulloblastoma comprises distinct molecular variants. J. Clin. <br />                                                                Oncol. 2011; 29:1408&#226;&#8364;&#8220;1414. [PubMed: 20823417] <br />                                                            Pan KZ, Palter JE, Rogers AN, Olsen A, Chen D, Lithgow GJ, Kapahi P. Inhibition mRNA <br />                                                                translation extends lifespan Caenorhabditis elegans. Aging Cell. 2007; 6:111&#226;&#8364;&#8220;119. [PubMed: <br />                                                                17266680] <br />                                                            Proud CG. Signalling translation: signal transduction pathways control protein synthetic <br />                                                                machinery. Biochem. J. 2007; 403:217&#226;&#8364;&#8220;234. [PubMed: 17376031] <br />                                                            Ryazanov AG, Shestakova EA, Natapov PG. Phosphorylation elongation factor 2 EF-2 kinase <br />                                                                affects rate translation. Nature. 1988; 334:170&#226;&#8364;&#8220;173. [PubMed: 3386756] <br />                                                            Sakagami H, Nishimura H, Saito R, Kondo H. Transient upregulation elongation factor-2 kinase <br />                                                                (Ca2+/calmodulin-dependent protein kinase III) messenger RNA developing mouse brain. <br />                                                                Neurosci. Lett. 2002; 330:41&#226;&#8364;&#8220;44. [PubMed: 12213630] <br />                                                            Shim H, Chun YS, Lewis BC, Dang CV. A unique glucose-dependent apoptotic pathway induced by <br />                                                                c-Myc. Proc. Natl. Acad. Sci. USA. 1998; 95:1511&#226;&#8364;&#8220;1516. [PubMed: 9465046] <br />                                                            Somasekharan SP, Stoynov N, Rotblat B, Leprivier G, Galpin JD, Ahern CA, Foster LJ, Sorensen PH. <br />                                                                Identification quantification newly synthesized proteins translationally regulated YB-1 <br />                                                                using novel Click-SILAC approach. J. Proteomics. 2012; 77:e1&#226;&#8364;&#8220;e10. [PubMed: 22967496] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Teleman AA, Chen YW, Cohen SM. 4E-BP functions metabolic brake used stress <br />                                                                conditions normal growth. Genes Dev. 2005; 19:1844&#226;&#8364;&#8220;1848. [PubMed: 16103212] <br />                                                            Tognon C, Knezevich SR, Huntsman D, Roskelley CD, Melnyk N, Mathers JA, Becker L, Carneiro F, <br />                                                                MacPherson N, Horsman D, et al. Expression ETV6-NTRK3 gene fusion primary event <br />                                                                human secretory breast carcinoma. Cancer Cell. 2002; 2:367&#226;&#8364;&#8220;376. [PubMed: 12450792] <br />                                                            Verhaak RG, Hoadley KA, Purdom E, Wang V, Qi Y, Wilkerson MD, Miller CR, Ding L, Golub T, <br />                                                                Mesirov JP, et al. Cancer Genome Atlas Research Network. Integrated genomic analysis identifies <br />                                                                clinically relevant subtypes glioblastoma characterized abnormalities PDGFRA, IDH1, <br />                                                                EGFR, NF1. Cancer Cell. 2010; 17:98&#226;&#8364;&#8220;110. [PubMed: 20129251] <br />                                                            Wang X, Li W, Williams M, Terada N, Alessi DR, Proud CG. Regulation elongation factor 2 kinase <br />                                                                p90(RSK1) p70 S6 kinase. EMBO J. 2001; 20:4370&#226;&#8364;&#8220;4379. [PubMed: 11500364] <br />                                                            Wu H, Yang JM, Jin S, Zhang H, Hait WN. Elongation factor-2 kinase regulates autophagy human <br />                                                                glioblastoma cells. Cancer Res. 2006; 66:3015&#226;&#8364;&#8220;3023. [PubMed: 16540650] <br />                                                            Wu X, Northcott PA, Dubuc A, Dupuy AJ, Shih DJ, Witt H, Croul S, Bouffet E, Fults DW, Eberhart <br />                                                                CG, et al. Clonal selection drives genetic divergence metastatic medulloblastoma. Nature. 2012; <br />                                                                482:529&#226;&#8364;&#8220;533. [PubMed: 22343890] <br />                                                            Ye J, Kumanova M, Hart LS, Sloane K, Zhang H, De Panis DN, Bobrovnikova-Marjon E, Diehl JA, <br />                                                                Ron D, Koumenis C. The GCN2-ATF4 pathway critical tumour cell survival and <br />                                                                proliferation response nutrient deprivation. EMBO J. 2010; 29:2082&#226;&#8364;&#8220;2096. [PubMed: <br />                                                                20473272] <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                             Page 15 <br />  <br />  <br />                                                            Yun J, Rago C, Cheong I, Pagliarini R, Angenendt P, Rajagopalan H, Schmidt K, Willson JK, <br />                                                               Markowitz S, Zhou S, et al. Glucose deprivation contributes development KRAS pathway <br />                                                               mutations tumor cells. Science. 2009; 325:1555&#226;&#8364;&#8220;1559. [PubMed: 19661383] <br />                                                            Zaugg K, Yao Y, Reilly PT, Kannan K, Kiarash R, Mason J, Huang P, Sawyer SK, Fuerth B, Faubert <br />                                                               B, et al. Carnitine palmitoyltransferase 1C promotes cell survival tumor growth under <br />                                                               conditions metabolic stress. Genes Dev. 2011; 25:1041&#226;&#8364;&#8220;1051. [PubMed: 21576264] <br />                                                            Zoncu R, Efeyan A, Sabatini DM. mTOR: growth signal integration cancer, diabetes and <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               ageing. Nat. Rev. Mol. Cell Biol. 2011; 12:21&#226;&#8364;&#8220;35. [PubMed: 21157483] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                         Page 16 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 1. Oncogenic Transformation Sensitizes Cells ND Alters eEF2 Signaling Pathways <br />                                                            (A) Caspase-3 activity assays NIH 3T3 MSCV, EN, RasV12 cells grown complete <br />                                                            media deprived nutrients (ND) indicated times (n = 3). <br />                                                            (B) Schematic representation nutrient-responsive signaling pathways assessed this <br />                                                            study. Gray arrows bars indicate release regulatory effects upstream pathways. <br />                                                            (C) Immunoblot analysis NIH 3T3 MSCV, EN, RasV12 cells grown complete <br />                                                            media ND indicated times. <br />  <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                      Page 17 <br />  <br />  <br />                                                            (D) Intracellular levels AMP ATP NIH 3T3 MSCV, EN, RasV12, EN-S, and <br />                                                            RasV12-S cells grown complete media (Ctrl) ND 6 hr. Results expressed <br />                                                            relative fold increases AMP:ATP ratio MSCV Ctrl n = 3. <br />                                                            (E) Immunoblot analysis NIH 3T3 EN RasV12 cells deprived nutrients (1 hr) and <br />                                                            treated Akt inhibitor VII (Akti; 1 &#206;&#188;M), rotenone (0.5 &#206;&#188;M), 2-deoxyglucose (2-DG; 25 <br />                                                            mM), vehicle. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            (F) Scheme generation EN-S RasV12-S cells NIH 3T3 EN RasV12 cells <br />                                                            (top). Caspase-3 activity assays NIH 3T3 EN, EN-S, RasV12, RasV12-S cells grown in <br />                                                            complete media ND indicated (n = 3) (bottom). <br />                                                            (G) Immunoblot analysis NIH 3T3 EN, EN-S, RasV12, RasV12-S cells grown in <br />                                                            complete media ND indicated times. <br />                                                            (H) Immunoblot analysis NIH 3T3 EN-S/MSCV, EN-S/DN-AMPK, RasV12-S/MSCV, <br />                                                            RasV12-S/DN-AMPK cells grown complete media ND indicated <br />                                                            times. <br />                                                            (I) Caspase-3 assays NIH 3T3 EN-S/MSCV, EN-S/DN-AMPK, RasV12-S/MSCV, and <br />                                                            RasV12-S/DN-AMPK cells grown complete media (Ctrl) ND 48 hr (n = 3). <br />                                                            Where shown, data reported means &#194;&#177; SD indicated significance (*p &lt; 0.05, **p <br />                                                            &lt; 0.01, ***p &lt; 0.005). See Figures S1 S2. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 18 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 2. Translation Elongation Activity Is Sustained Transformed Cells ND and <br />                                                            Reduces Their Survival <br />                                                            (A) Protein synthesis levels NIH 3T3 MSCV, EN, RasV12 cells deprived nutrients <br />                                                            5 min 1, 3, 6 hr determined [35S]-methionine/cysteine(Met/Cys) incorporation. <br />                                                            Results expressed percentage [35S]-Met/Cys incorporation/mg protein relative to <br />                                                            MSCV cells 5 min (n = 2). <br />                                                            (B) Polysome profiles NIH 3T3 MSCV, EN, RasV12 cells grown complete media <br />                                                            ND 70 min described Extended Experimental Procedures. P/S indicates <br />                                                            ratio polysomal subpolysomal (40S, 60S, 80S) fractions. <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                        Page 19 <br />  <br />  <br />                                                            (C) Ribosome half-transit times NIH 3T3 MSCV, EN, RasV12 cells grown in <br />                                                            complete media ND 70 min. [35S]-Met/Cys incorporation polypeptides <br />                                                            (postmitochondrial supernatant, PMS) polypeptides released ribosomes <br />                                                            (postribosomal supernatant, PRS) obtained linear regression analysis. Representative <br />                                                            results different experiments shown. <br />                                                            (D E) siRNA-mediated knockdown eEF2 NIH 3T3 EN RasV12 cells. Cells <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            transiently transfected 12.5 nM control (CTRL) eEF2-directed siRNAs, <br />                                                            grown complete media 48 hr, placed ND 48 hr. Lysates either <br />                                                            analyzed immunoblotting (D) assayed caspase-3 activity (n = 3) (E). <br />                                                            (F G) siRNA-mediated knockdown eEF2 NIH 3T3 EN-S/MSCV, EN-S/DN- <br />                                                            AMPK, RasV12-S/MSCV, RasV12-S/DN-AMPK cells. Cells transfected, treated, <br />                                                            analyzed (D) (E). <br />                                                            Where shown, data reported means &#194;&#177; SD indicated significance (**p &lt; 0.005). <br />                                                            See Figure S3. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                   Page 20 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 3. eEF2K Is Critical Adaptation Fibroblasts ND <br />                                                            (A) Caspase-3 activity assays eEF2K+/+ eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs grown complete media <br />                                                            ND indicated times (n = 3). <br />                                                            (B C) siRNA-mediated knockdown eEF2 eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs. Cells transiently <br />                                                            transfected 12.5 nM control (CTRL) eEF2-directed siRNAs, grown complete <br />                                                            media 48 hr, placed ND 48 hr. Cell lysates analyzed by <br />                                                            immunoblotting (B) assayed caspase-3 activity (n = 3) (C). <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                      Page 21 <br />  <br />  <br />                                                            (D E) siRNA-mediated knockdown eEF2K eEF2 NIH 3T3 MSCV cells by <br />                                                            transient transfection 37.5 nM control (siCTRL) 25 nM eEF2K siRNAs (si <br />                                                            eEF2K#1 eEF2K#2) combined 12.5 nM control eEF2 siRNAs. Cells were <br />                                                            treated described (B) (C). Lysates analyzed immunoblotting (D) or <br />                                                            assayed cell death (n = 3) (E). <br />                                                            (F) Immunoblot analysis NIH 3T3 EN/MSCV, EN/eEF2K, RasV12/MSCV, RasV12/ <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            eEF2K grown complete media ND indicated times. <br />                                                            (G) Caspase-3 activity assays NIH 3T3 EN/MSCV, EN/eEF2K, EN/eEF2K-K170M <br />                                                            (kinase-dead), RasV12/MSCV, RasV12/eEF2K, RasV12/eEF2K-K170M cells grown in <br />                                                            complete media (Ctrl) ND 48 hr (n = 3). <br />                                                            (H I) siRNA-mediated knockdown eEF2K NIH 3T3 EN-S RasV12-S cells. <br />                                                            Cells transiently transfected 25 nM control (siCTRL) eEF2K (si eEF2K#1 <br />                                                            eEF2K#2) siRNAs grown complete media 72 hr. Cells placed ND <br />                                                            3 hr analyzed immunoblotting (H) 72 hr assayed caspase-3 <br />                                                            activity (n = 3) (I). <br />                                                            (J) Cell adaptation assays eEF2K+/+ eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; MEFs expressing EN RasV12. <br />                                                            Cells subjected adaptation protocol described Extended Experimental <br />                                                            Procedures. Results expressed number cells alive completion the <br />                                                            experiment (n = 3). <br />                                                            Where shown, data reported means &#194;&#177; SD indicated significance (*p &lt; 0.05 and <br />                                                            **p &lt; 0.01; ns, nonsignificant). See Figure S4. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                      Page 22 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 4. eEF2K Facilitates Survival Human Tumor Cells Response ND <br />                                                            (A) Immunoblot analysis HeLa MG63 cells grown complete media ND <br />                                                            indicated times. <br />                                                            (B C) Caspase-3 activity assays (B) cell death assays (C) HeLa MG63 cells <br />                                                            grown complete media ND indicated times (n = 3). <br />                                                            (D E) siRNA-mediated knockdown eEF2 HeLa cells. Cells transiently <br />                                                            transfected 12.5 nM control (CTRL) eEF2-directed siRNAs, grown complete <br />                                                            media 48 hr, placed ND 48 hr. Cell lysates analyzed by <br />                                                            immunoblotting (D) assayed caspase-3 activity (n = 3) (E). <br />                                                            (F G) Immunoblot analysis (F) caspase-3 activity assays (n = 3) (G) HeLa MSCV <br />                                                            HeLa eEF2K cells grown complete media ND indicated times. <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                       Page 23 <br />  <br />  <br />                                                            (H I) siRNA-mediated knockdown eEF2K MG63 cells. Cells transiently <br />                                                            transfected 25 nM control (siCTRL) eEF2K (si eEF2K#1&#226;&#8364;&#178; eEF2K#2&#226;&#8364;&#178;) siRNAs <br />                                                            grown complete media 72 hr. Cells placed ND 3 hr and <br />                                                            analyzed immunoblotting (H) 72 hr assayed caspase-3 activity (n = 3) (I). <br />                                                            Where shown, data reported means &#194;&#177; SD indicated significance (*p &lt; 0.05, **p <br />                                                            &lt; 0.01, ***p &lt; 0.005). See Figure S5. <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                         Page 24 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 5. eEF2K Promotes Resistance Tumors Caloric Restriction-Induced Cell Death In <br />                                                            Vivo <br />                                                            (A B) Tumor volumes NIH 3T3 RasV12/MSCV RasV12/eEF2K (A) EN/MSCV <br />                                                            EN/eEF2K (B) xenografts implanted subcutaneously nu/nu mice. Mice fed <br />                                                            AL placed CR diets (n = 10 mice/group). <br />                                                            (C) Hematoxylin eosin staining (H&amp;E) tumor xenografts (A). Black arrows <br />                                                            indicate regions necrosis. Results expressed percentages necrotic/total tumor <br />                                                            areas (n = 3 mice group). <br />  <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                            Page 25 <br />  <br />  <br />                                                            (D) Immunohistochemical (IHC) staining tumor xenografts (A) antibodies <br />                                                            cleaved caspase-3. Graphs indicate percent total cells positive cleaved <br />                                                            caspase-3 (n = 3 mice group). <br />                                                            (E) IHC staining tumor xenografts (A) anti-phospho-eEF2 antibodies. <br />                                                            (F) Immunoblot analysis tumor lysates tumor xenografts (A). <br />                                                            (G) Tumor volumes eEF2K+/+ RasV12 eEF2K&#226;&#710;&#8217;/&#226;&#710;&#8217; RasV12 xenografts implanted <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            subcutaneously nu/nu mice. Mice fed AL placed CR diets (n = 10 mice <br />                                                            group). <br />                                                            (H) H&amp;E staining tumor xenografts (G). Black arrows indicate areas necrosis. <br />                                                            Results expressed percentages necrotic/total tumor areas (n = 3 mice group). <br />                                                            (I) IHC staining tumor xenografts (G) antibodies cleaved caspase-3. Graphs <br />                                                            indicate percent total cells positive cleaved caspase-3 (n = 3 mice group). <br />                                                            Where shown, data reported means &#194;&#177; SEM indicated significance (*p &lt; 0.05 and <br />                                                            **p &lt; 0.01; ns, nonsignificant). See Figure S6. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                        Page 26 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 6. eEF2K Transcript Levels Are Associated Poor Prognosis Medulloblastoma and <br />                                                            Glioblastoma Multiforme <br />                                                            (A B) Expression levels eEF2K Toronto MB cohort (A) European MB <br />                                                            data set (B) (see Extended Experimental Procedures). FC CB indicate fetal adult <br />                                                            cerebellum, respectively; MBs indicate total cohort, WNT, SHH, group 3, group <br />                                                            4 indicate specific disease subgroups. p values shown panel were <br />                                                            generated using Mann-Whitney U test. <br />                                                            (C D) Kaplan-Meier estimates overall survival MB patients subgroups <br />                                                            (C) non-WNT/non-SHH subgroups (D) classified eEF2K mRNA expression <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                           Page 27 <br />  <br />  <br />                                                            levels. The number patients risk indicated time increments 24 months. p values <br />                                                            calculated using log rank test. <br />                                                            (E F) Expression eEF2K mRNA normal human brain (NHB; n = 10) compared to <br />                                                            GBM (n = 25) (E) specific disease subtypes GBM (proneural, neural, classical, and <br />                                                            mesenchymal) (F). p values determined ANOVA Bonferroni post hoc test. <br />                                                            (G H) Kaplan-Meier estimates overall survival glioma patients (G) GBM <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            patients (H) classified eEF2K mRNA expression levels. Survival indicated time <br />                                                            increments 2,000 (G) 500 days (H). p values calculated using log rank test. <br />                                                            (I J) IHC staining phospho-eEF2 primary tumors (I, 1&#226;&#8364;&#8220;2) corresponding <br />                                                            metastases (I, 3&#226;&#8364;&#8220;4) mouse model MB (Wu et al., 2012) human primary MB <br />                                                            (J). Black arrows indicate tumor regions, white triangles indicate adjacent normal <br />                                                            cerebellar tissue. <br />                                                            (K) siRNA-mediated knockdown eEF2K d283 MB cells. Cells transiently <br />                                                            transfected 25 nM control (siCTRL) eEF2K siRNAs (si eEF2K#1&#226;&#8364;&#178; and <br />                                                            eEF2K#2&#226;&#8364;&#178;), grown complete media 72 hr, harvested placed ND 24 hr. <br />                                                            Cell lysates assayed caspase-3 activity. Error bars indicate SD n = 3 (*p &lt; 0.05). <br />                                                            See Figure S7. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                     Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 28 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Figure 7. The eEF2K Ortholog efk-1 Supports Survival C. elegans ND In Vivo <br />                                                            (A) Lifespans C. elegans WT (N2) efk-1 (ok3609) homozygous deletion mutants <br />                                                            ambient growth conditions. <br />                                                            (B) Lifespans N2 efk-1 (ok3609) mutants L1 stage development the <br />                                                            indicated days absence food. Error bars indicate SEM n = 142&#226;&#8364;&#8220;150. <br />                                                            (C) Quantitative RT-PCR efk-1 mRNA levels N2 worms held L1 stage 24 hr <br />                                                            absence food (starved) placed food (OP50 bacteria) 3 hr (fed). <br />                                                            Transcript levels normalized &#206;&#179;-tubulin (tbg-1) levels. Error bars indicate SD n = <br />                                                            3 (*p &lt; 0.05). <br />                                                            (D) Quantitative RT-PCR eEF2K transcripts indicated mammalian cell lines <br />                                                            grown complete media (Ctrl) ND 24 hr. Transcript levels normalized to <br />  <br />  <br />                                                                    Cell. Author manuscript; available PMC 2015 April 14. <br />                                          Leprivier et al.                                                                                          Page 29 <br />  <br />  <br />                                                            &#206;&#178;-actin (actb). Results shown relative levels ND versus control conditions each <br />                                                            cell line. Error bars indicate SD n = 2 (*p &lt; 0.05). <br />                                                            (E) Model proposed role eEF2K/efk-1 adaptation ND normal tumor <br />                                                            tissues. <br /> </body></html>