<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast DJ-1 superfamily members required for <br /> diauxic-shift reprogramming cell <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>survival</span> in <br /> stationary phase <br /> Leonor Miller-Fleminga,b, Pedro Antasa, Teresa Faria Paisa, Joshua L. Smalleyb,c, Flaviano Giorginib,1, <br /> and Tiago Fleming Outeiroa,d,e,1 <br /> a <br />  Instituto Medicina Molecular, 1649-028 Lisbon, Portugal; bDepartment Genetics, University Leicester, Leicester LE1 7RH, United Kingdom; <br /> c <br />  Systems Toxicology Group, Medical Research Council Toxicology <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, University Leicester, LE1 9HN Leicester, United Kingdom; dInstituto Fisiologia, <br /> Faculdade Medicina, Universidade Lisboa, 1649-028 Lisbon, Portugal; eDepartment Neurodegeneration Restorative Research, University <br /> Medical Center G&#195;&#182;ttingen, 37073 G&#195;&#182;ttingen, Germany <br />  <br /> Edited Gregory A. Petsko, Weill Cornell Medical College, New York, NY, approved March 7, 2014 (received review October 14, 2013) <br />  <br /> The yeast Hsp31 minifamily proteins (Hsp31, Hsp32, Hsp33, Hsp34)               contribute understanding general function the <br /> belong highly conserved DJ-1 superfamily. The human DJ-1                DJ-1 superfamily. <br /> protein associated cancer neurodegenerative disorders, <br /> such Parkinson disease. However, precise function human              Results <br /> and yeast DJ-1 proteins unclear. Here yeast DJ-1           The HSP31 Minifamily Is Required Normal DS SP. Because <br /> homologs role diauxic-shift (DS), characterized meta-             Hsp31 induced postdiauxic phase (5), started assessing <br /> bolic reprogramming glucose limitation. We            levels HSP31, HSP32, HSP33 different <br /> Hsp31 genes strongly induced DS stationary phase (SP),           phases yeast growth liquid medium (Fig. S1A). Because <br /> and deletion genes reduces chronological lifespan,               HSP32, HSP33, HSP34 extremely high sequence identity <br /> impairs transcriptional reprogramming DS, impairs acqui-            (Fig. S1B), possible design unique primers HSP34. In <br /> sition typical characteristics SP, including autophagy           parallel, determined expression profile GIS1 (GIg1-2 <br /> induction. In addition, carbon starvation, HSP31 family              Suppressor), transcription factor required gene expression <br /> gene-deletion strains display impaired autophagy, disrupted target             <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation (9). We HSP31&#226;&#8364;&#8220;33 GIS1 <br /> of rapamycin complex 1 (TORC1) localization P-bodies,                   similar expression patterns (Fig. 1A): genes highly <br /> caused abnormal TORC1-mediated Atg13 phosphorylation. Repres-                  induced DS levels maintained SP. We detected <br /> sion TORC1 rapamycin gene-deletion strains completely             slight differences timing levels expression between <br /> reversed sensitivity heat shock. Taken together, data in-         members Hsp31 minifamily. HSP31 HSP33 mRNAs <br /> dicate Hsp31 minifamily required DS reprogramming              reached maximum levels early SP, HSP32 mRNA <br /> <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>cell survival</span> SP, plays role upstream TORC1. The en-               levels peaked DS (Fig. 1A). <br /> hanced understanding cellular function genes sheds                As previously shown hsp31&#206;&#8221; cells (5), deletion HSP32, <br /> light biological role members superfamily,            HSP33, HSP34 effect yeast growth normal <br /> including DJ-1, attractive target therapeutic inter-           conditions resulted higher sensitivity oxidative stress <br /> vention cancer Parkinson disease.                                    (Fig. S1C), reduced thermotolerance (Fig. S1D), accumulation <br />                                                                                higher levels reactive oxygen species (ROS) (Fig. S1 E F). <br />                                                                                Given expression HSP31, HSP32, HSP33 signifi- <br /> T   Saccharomyces cerevisiae Hsp31 minifamily comprised <br />     Hsp31 (YDR533C), Hsp32 (YMR322C), Hsp33 (YOR391C), <br /> and Hsp34 (YPL280W) proteins, belongs DJ-1 super- <br />                                                                                cantly increased DS, tested absence these <br />  <br />                                                                                    Significance <br /> family (1). Human DJ-1, extensively studied su- <br /> perfamily, implicated Parkinson disease cancer. Although <br />                                                                                    <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> knowledge, time yeast <br /> there consensus precise role, DJ-1 appears be <br />                                                                                    DJ-1 homologs required diauxic-shift, important <br /> a multifunctional protein associated numerous functions, in- <br />                                                                                    metabolic reprogramming stage triggered glucose <br /> cluding protection oxidative stress (2). DJ-1 acts redox- <br />                                                                                    deprivation. Deletion HSP31-34 genes results reduced <br /> dependent chaperone, preventing &#206;&#177;-synuclein aggregation, a <br />                                                                                    lifespan alterations hallmarks stationary- <br /> protease, RNA-binding protein, free-radical scav- <br />                                                                                    phase, including impairment autophagy induction through <br /> enger (3). DJ-1 linked survival oxidative stress <br />                                                                                    deregulation target rapamycin complex 1 (TORC1). As both <br /> signaling pathways, PTEN, AKT NRF2 (3). <br />                                                                                    autophagy TORC1 associated human disorders, <br />    Among Hsp31 minifamily, Hsp31 divergent <br /> and likely gave rise paralogs genome du-                     work broad relevance understanding of <br /> plication (4), share strong sequence homology &gt;90%.                       processes health disease. <br /> Although HSP32, HSP33, HSP34 localize subtelomeric <br />                                                                                Author contributions: L.M.-F., T.F.P., F.G., T.F.O. designed research; L.M.-F., P.A., T.F.P. <br /> regions, HSP31 interstitial gene. Although little known               performed research; F.G. T.F.O. contributed new reagents/analytic tools; L.M.-F., P.A., <br /> about function proteins, Hsp31 likely plays role             T.F.P., J.L.S., F.G., T.F.O. analyzed data; L.M.-F., F.G., T.F.O. wrote paper. <br /> stress response (5). Indeed, proteomic transcriptomic studies              The authors declare conflict interest. <br /> have HSP31 expression increases exposure a <br />                                                                                This article PNAS Direct Submission. <br /> variety cellular insults (4, 6&#226;&#8364;&#8220;8). <br />    Here, Hsp31 minifamily required                     Freely available online PNAS open access option. <br />  <br /> normal transition diauxic-shift (DS) acquisition             Data deposition: The data reported paper deposited Gene Ex- <br /> typical characteristics stationary phase (SP). In absence,            pression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE56124). <br />  <br /> yeast cells lose capacity survive SP conditions                  See Commentary page 6858. <br /> display target rapamycin complex 1 (TORC1) signaling                        1 <br />                                                                                 To correspondence addressed. E-mail: fg36@le.ac.uk touteiro@gmail.com. <br /> dysfunction. In report, provide novel insights                This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> function DJ-1-related Hsp31 minifamily,                       1073/pnas.1319221111/-/DCSupplemental. <br />  <br />  <br />  <br /> 7012&#226;&#8364;&#8220;7017 | PNAS | May 13, 2014 | vol. 111 | no. 19                                                                     www.pnas.org/cgi/doi/10.1073/pnas.1319221111 <br />                                                                                                                                                                     SEE COMMENTARY <br /> Fig. 1. HSP31, HSP32, HSP33 required normal DS CLS. (A) Gene-expression quantification performed qRT-PCR midlog, DS, early SP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                    CELL BIOLOGY <br /> (1 d growth) SP (2 d growth) normalized TAF10 UBC6. Error bars represent SD (SD) mean (n = 3). Fold-induction indicated of <br /> each bar. Statistical analyses performed using Student t test (***P &lt; 0.001, **P &lt; 0.01, *P &lt; 0.05). (B) Strains monitored CLS counting colony- <br /> forming units. Day 0 time cultures reached SP (survival = 100%). Mean SD (n = 3) indicated graph. (C&#226;&#8364;&#8220;F) Expression analysis of <br /> hsp31&#206;&#8221;, hsp32&#206;&#8221;, hsp33&#206;&#8221; DS. (C) Overlap up- down-regulated genes hsp31&#206;&#8221;, hsp32&#206;&#8221;, hsp33&#206;&#8221; versus WT cells. (D E) Overlapping <br /> genes grouped terms GO. The significant gene clusters shown. Frequency percentage overlapping genes clustered in <br /> a given ontology group; yeast genome frequency total percentage genes annotated term parental strain. (F) Log DS ex- <br /> pression profiles downloaded ArrayExpress (E-TABM-496) subsequently analyzed calculate fold-change conditions. <br /> DEGs &gt; twofold (P &lt; 0.05) divided up- down-regulated groups compared overlapping genes obtained C. <br />  <br />  <br />  <br /> genes, HSP34, affected yeast survival SP, known                  confirm DEGs overlap reprogrammed <br /> chronological lifespan (CLS). We included gis1&#206;&#8221; strain                    genes DS, compared results existing data available at <br /> comparison shorter CLS (10). Deletion HSP31 family                   ArrayExpress (12). We compared existing log-phase expression <br /> genes resulted reduced CLS, lesser extent                   profiles DS profiles, allowing determine genes <br /> deletion GIS1 (Fig. 1B). Reintroduction HSP31 family                    change DS. Next, compared dataset &#226;&#8364;&#339;overlapping&#226;&#8364;? <br /> genes <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>knockout</span> strains reverted <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. S1 G                  dataset. This analysis revealed &#226;&#710;&#188;62% genes our <br /> H). In addition, absence hsp31&#206;&#8221; thermotolerance                   dataset overlapped reprogrammed genes (Fig. 1F). How- <br /> reverted HSP31 family members, suggesting                ever, &#226;&#710;&#188;59% overlapping genes differentially expressed <br /> have partially overlapping functions (Fig. S1I).                         opposing directions (e.g., anticorrelated), indicating normal <br />    The strong up-regulation HSP31&#226;&#8364;&#8220;33 DS,                      diauxic transition altered knockout strains (Fig. 1F). <br /> the loss cell viability aged cultures knockout strains,                     Because glucose limitation DS, genes glu- <br /> prompted investigate role genes                 cose repression derepressed. Indeed, several <br /> DS. We performed gene-expression microarrays hsp31&#206;&#8221;,                           DEGs involved glucose transport metabolism, including <br /> hsp32&#206;&#8221;, hsp33&#206;&#8221;, WT strains DS, determined                        MIG2, NRG2, HXT5 (Table S3). MIG2 NRG2, are <br /> based growth rate glucose consumption. The micro-                        involved glucose repression, up-regulated knockout <br /> array analyses validated testing selected differentially                  strains, HXT5&#226;&#8364;&#8221;a glucose transporter induced DS (13)&#226;&#8364;&#8221; <br /> expressed genes (DEGs) quantitative real-time PCR (qRT-                        highly down-regulated. Interestingly, expression HXT4, <br /> PCR) (Table S1). The expression profiles knockout strains                      glucose transporter induced DS, up-regulated in <br /> were independently compared WT strain,                       knockout strains, possible compensatory mechanism result- <br /> only genes differently expressed twofold (P &#226;&#8240;&#164; 0.05)                     ing reduced expression HXT5. <br /> were subsequently analyzed. Among pair-wise com-                           To confirm glucose repression enhanced the <br /> parisons, hsp32&#206;&#8221; showed highest number DEGs                           knockout strains, expressed Leu2 WT hsp31&#206;&#8221; <br /> alterations one-quarter genome (Fig. 1C). To identify                   strains control GAL1 promoter. Whereas ex- <br /> overlapping DEGs knockout strains, performed                       pression Leu2 increased progressively galactose in- <br /> separate comparisons up- down-regulated genes (Fig. 1C).                   duction WT strain, hsp31&#206;&#8221; cells exhibited lower levels of <br /> hsp32&#206;&#8221; hsp33&#206;&#8221; strains shared high number DEGs                       Leu2, did increase time-frame experi- <br /> did overlap hsp31&#206;&#8221;, agreement high con-                      ment. These data suggest glucose repression enhanced in <br /> servation Hsp32 Hsp33 (Fig. 1C). We selected                          hsp31&#206;&#8221; comparison WT strain, confirming Hsp31 <br /> DEGs common knockout strains clustered                      required glucose derepression (Fig. S1J). <br /> using gene ontology (GO) analysis based biological process <br /> annotations (Fig. 1 D E). Among down-regulated GO                         Typical SP Characteristics Are Altered HSP31 Minifamily Knockout <br /> terms genes involved metabolic processes, cellular                    Strains. Several changes occur postdiauxic phase are <br /> response stress, autophagy (Fig. 1D Table S2),                         maintained SP (14). Therefore, evaluated <br /> whereas up-regulated group included genes involved                         typical characteristics SP altered knock- <br /> translation (Fig. 1E). Intriguingly, biological processes                   strains. We tested thermotolerance strains to <br /> are normally reprogrammed DS WT cells, suggesting                      50 &#194;&#176;C heat shock, indicator cellular steady-state defenses. <br /> in knockout cells metabolic regulation perturbed (11). To                 Consistently, knockout strains thermotolerant than <br />  <br /> Miller-Fleming et al.                                                                                      PNAS | May 13, 2014 | vol. 111 | no. 19 | 7013 <br />  the WT strain, exhibiting decreased viability 10 min of <br /> heat shock (Fig. 2A). The thickness cell wall&#226;&#8364;&#8221;known to <br /> increase SP cells&#226;&#8364;&#8221;was evaluated resistance lytic <br /> enzyme zymolyase. Cells SP incubated zymolyase in <br /> a hyperosmotic environment monitored optical density <br /> over time. As cell wall digested, cells burst optical <br /> density decreases. The cell walls HSP31 minifamily GIS1 <br /> knockout strains digested rapidly of <br /> the WT strain, indicating reduced resistance zymolyase and <br /> therefore thinner cell wall (Fig. 2B). <br />    We evaluated autophagy knockout strains SP by <br /> monitoring ATG8, autophagy gene down-regulated the <br /> knockout strains. Atg8 involved formation autopha- <br /> gosomes and, process, degraded vacuole                        Fig. 3. Autophagic response hsp31&#206;&#8221; impaired carbon starvation. <br /> (15). We assessed autophagy using GFP-Atg8 reporter                         WT hsp31&#206;&#8221; strains expressing pATG8-GFP-Atg8 grown midlog <br /> the control ATG8 promoter, permits monitoring                       phase treated rapamycin starved carbon (SC-C) the <br /> both autophagy induction autophagic flux Atg8 rapidly                     specified time. (A) Total lysates analyzed immunoblotting using <br /> degraded vacuole GFP (16). Immunoblot                             antibodies GFP housekeeping control Pgk1. (B) After 5 h of <br /> analysis anti-GFP antibody detects protein bands:                       incubation rapamycin starvation, GFP-Atg8 localization ana- <br /> the higher molecular-weight band corresponding cytosolic                         lyzed microscopy. (Scale bar, 5 &#206;&#188;m.) <br /> GFP-Atg8, lower band corresponding GFP-Atg8 <br /> degraded vacuole (16). We analyzed autophagy WT, <br /> hsp31&#206;&#8221;, hsp33&#206;&#8221;, hsp34&#206;&#8221; strains SP log phase.                        DS, hypothesized ROS trigger observed up- <br /> Strikingly, observed impaired induction autophagy                      regulation HSP31-33 DS (Fig. 1A). Intriguingly, oxidative <br /> knockout strains SP reduced levels basal autophagy                    stress induced H2O2 failed increase expression HSP32 <br /> during log phase (Fig. 2C). These results indicate autophagy                HSP33, highest concentration (2 mM) able <br /> impaired absence Hsp31 family                         induce HSP31 (Fig. S2A). Because yeast viability decreased <br /> members. Finally, evaluated cell-cycle progression                        concentration, did exceed level our <br /> knockout strains. To measure DNA content, stained cells                     experiments. <br /> propidium iodide performed flow cytometry analyses,                          Next, tested HSP31 minifamily induced by <br /> indicated higher percentage cells remained mitosis                 starvation, hallmark DS SP. HSP32 HSP33 <br /> failed enter G0 knockout cells (Fig. 2D).                                 expression highly induced nitrogen carbon <br />                                                                                     starvation, higher carbon starvation conditions (Fig. <br /> Autophagic Response Is Impaired During Carbon Starvation hsp31&#206;&#8221;                  S2B). On hand, HSP31 significantly up-reg- <br /> Cells. As previously shown HSP31 induced oxi-                     ulated nitrogen depletion (Fig. S2B). These results suggest <br /> dative stress conditions (5) levels ROS increase                     expression Hsp31 minifamily members during <br />                                                                                     DS SP mainly regulated starvation opposed to <br />                                                                                     oxidative stress. <br />                                                                                        We assessed autophagy impairment during <br />                                                                                     SP knockout strains observed <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br />                                                                                     (nitrogen carbon) starvation rapamycin treatment, <br />                                                                                     compound widely-used induce autophagy TORC1 in- <br />                                                                                     activation (17). Cells expressing GFP-Atg8 analyzed before <br />                                                                                     1, 3, 5 h posttreatment. Because autophagy phe- <br />                                                                                     notypes knockout strains identical, focused our <br />                                                                                     efforts hsp31&#206;&#8221; strain subsequent experiments. <br />                                                                                     After 1 h rapamycin treatment, observed induction of <br />                                                                                     autophagy, increased time WT hsp31&#206;&#8221; <br />                                                                                     cells (Fig. 3A). Nitrogen depletion induced autophagy in <br />                                                                                     hsp31&#206;&#8221; cells (Fig. S3A); however, induction flux remained <br />                                                                                     basal levels carbon starvation (Fig. 3A Fig. S3B). <br />                                                                                     Similar results obtained hsp33&#206;&#8221; hsp34&#206;&#8221; strains (Fig. <br />                                                                                     S3C). Next, confirmed findings fluorescence mi- <br />                                                                                     croscopy that, hsp31&#206;&#8221; cells displayed faint <br />                                                                                     GFP-Atg8 fluorescence signal cytoplasm absence of <br />                                                                                     glucose, WT cells displayed strong GFP-Atg8 signal localized to <br />                                                                                     vacuole (Fig. 3B). On hand, nitrogen star- <br />                                                                                     vation rapamycin treatment, GFP-Atg8 localized the <br />                                                                                     vacuole WT hsp31&#206;&#8221; cells (Fig. 3B Fig. S3A). <br /> Fig. 2. hsp31&#206;&#8221;, hsp32&#206;&#8221;, hsp33&#206;&#8221;, hsp34&#206;&#8221; cells fail acquire typical SP <br /> characteristics. WT gene-deletion strains analyzed resistance       TORC1 Signaling Is Perturbed hsp31&#206;&#8221; Cells. Entry postdiauxic <br /> heat stress (A), cell wall integrity (B), autophagy flux (C), cell cycle (D).   phase progression SP orchestrated <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing <br /> (A) Cultures grown SP (2 d growth) subjected 50 &#194;&#176;C heat         signaling pathways. Two major signaling pathways asso- <br /> shock specified time. (B) SP cultures incubated zymolyase <br />                                                                                     ciated transition regulated TORC1 and <br /> cell lysis measured reading OD600. (C) Log SP cells expressing <br /> GFP-Atg8 lysed analyzed immunoblotting. The presence free <br />                                                                                     protein kinase A (PKA) (18). These kinases&#226;&#8364;&#8221;in addition to <br /> GFP indicator autophagic flux. (D) To quantify DNA content, cells          playing role cellular growth translation stress re- <br /> were stained propidium iodide analyzed flow cytometry. The              sponse repression&#226;&#8364;&#8221;also negatively regulate autophagy (18). <br /> graph represents percentage cells remaining G2/M SP.               Because dramatic gene-expression changes observed the <br /> Error bars represent SD mean assay (n = 3, Student t test,          knockout strains DS, asked pathways were <br /> **P &lt; 0.01, *P &lt; 0.05).                                                             deregulated hsp31&#206;&#8221; cells. We used Atg13 reporter, is <br />  <br /> 7014 | www.pnas.org/cgi/doi/10.1073/pnas.1319221111                                                                                       Miller-Fleming et al. <br />                                                                                                                                                             SEE COMMENTARY <br /> a key player autophagy regulated phosphorylation PKA <br /> and TORC1 (18). <br />    Because TORC1 hyperphosphorylates Atg13, monitored <br /> phosphorylation-status Atg13 mobility SDS-polyacrylamide <br /> gels (18). In SP TORC1 repressed levels of <br /> Atg13 phosphorylation decrease. As expected WT cells, Atg13 <br /> migrated faster SP (Fig. 4A). However, hsp31&#206;&#8221; cells the <br /> difference Atg13 migration log SP reduced <br /> (Fig. 4A). This finding suggests Atg13 phosphory- <br /> lated hsp31&#206;&#8221; cells, possibly TORC1 hyperactivity.              Fig. 5. Rapamycin increases thermotolerance hsp31&#206;&#8221; cells. Yeast strains <br /> To confirm differences observed migration                   midlog phase treated DMSO rapamycin 3 h subjected <br /> caused phosphorylation posttranslational                 5-, 10-, 20-min heat shock 50 &#194;&#176;C. At postdiauxic phase cultures <br /> modifications, treated SP Atg13 &#206;&#187;-phosphatase ob-                incubated rapamycin 16 h heat shock. Cultures <br /> served Atg13 hsp31&#206;&#8221; cells migrated side               serial diluted spotted media plates analyze cell survival. <br /> Atg13 WT cells, confirming migrational differences in <br /> SP caused phosphorylation (Fig. S3D). We ana- <br /> lyzed phosphorylation Atg13 carbon nitrogen                 granules form stress responses translation initiation <br /> starvation conditions, led Atg13 dephosphorylation               affected (21). <br /> both parental hsp31&#206;&#8221; strains. Migration Atg13                   Next, subjected WT cells expressing Hsp31-GFP and <br /> hsp31&#206;&#8221; cells slower conditions tested              Hsp32-GFP glucose starvation heat shock, conditions <br /> more pronounced carbon starvation (Fig. 4B Fig. S3E),              associated SG formation. In nonstressed cells observed <br /> suggesting difference phosphorylation-status                presence Hsp31-GFP foci, significatly increased in <br /> strains. To investigate activity PKA immunoprecipitated            conditions (Fig. 6A Fig. S4A). Because SGs dynamic <br /> Atg13 analyzed phosphorylation immunoblotting using                   structures, characterized foci reintroducing <br /> a PKA substrate antibody [&#206;&#177;-(P)PKA]. The levels PKA-                      glucose media percentage cells with <br />  <br />  <br />  <br />  <br />                                                                                                                                                            CELL BIOLOGY <br /> mediated Atg13 phosphorylation similar WT                   foci decreased approximately basal levels, indicating the <br /> hsp31&#206;&#8221; cells conditions tested (SP, carbon, nitrogen              assembly foci reversible (Fig. 6A). SG assembly de- <br /> starvation) (Fig. 4 C D Fig. S3F), indicating              pendent presence nontranslating mRNA. Thus, we <br /> kinase affected knockout strain.                               treated cells cycloheximide (CHX) heat shock trap <br />    In total, results suggest TORC1 deregulated            mRNAs polysomes (21). Similar SG assembly, number of <br /> HSP31 deletion, evident SP conditions. Because              Hsp31-GFP foci did increase heat shock treated <br /> heat shock leads TORC1 inactivation (19, 20),                CHX (Fig. 6A), suggesting assembly requires free <br /> assessed blocking TORC1 reverted hsp31&#206;&#8221; lack ther-                mRNAs. We obtained similar results Hsp32-GFP (Fig. 6B <br /> motolerance. Using growth assays observed knockout cells             Fig. S4A), foci detected nontreated cells <br /> treated rapamycin exhibited decreased sensitivity heat               localization <span id='am-320' about='xsp:pattern' typeof='owl:Thing'>pattern</span> different (Fig. 6). <br /> shock, indicating TORC1 inhibition restored normal heat-                   To characterize foci, coexpressed Hsp31-GFP <br /> shock response (Fig. 5). This finding supports absence          Hsp32-GFP Pub1, marker SGs, Edc3, marker <br /> of HSP31 affects normal upstream regulation TORC1.                    Processing bodies (P-bodies, Pbs). P-bodies RNA- <br />                                                                              protein structures contain translationally silenced mRNAs, <br /> Hsp31 Hsp32 Are Present Stress Granules P-Bodies. To              diverge major components. Whereas SGs contain <br /> further characterize cellular role Hsp31 minifamily mem-              translation initiation factors, P-bodies contain several <br /> bers identify potential interaction partners, performed               mRNA decapping proteins (22). Although structures likely <br /> proteomic analyses GFP-Trap immunoprecipitation Hsp31-                different roles (22), interact dynamically and, cer- <br /> GFP Hsp32-GFP followed mass spectrometry. Interestingly,              tain conditions, SG formation depends P-body assembly (21). <br /> we identified components stress granules (SGs) in-             Our results clearly showed Hsp31-GFP Hsp32-GFP are <br /> teraction partners (Table S4), including core proteins Pbp1          present P-bodies SG foci (Fig. 6 C D). <br /> and Pab1. SGs dynamic cytoplasmic RNA-protein structures <br /> where nontranslating mRNAs rapidly stored (21). These                    P-Body SG Assembly Is Affected Absence HSP31. We <br />                                                                              investigated formation SG P-bodies was <br />                                                                              affected absence Hsp31. Similar de- <br />                                                                              scribed (23), P-bodies (Dcp1 foci) appeared early SP (Fig. <br />                                                                              7A), SGs (Pab1 foci) assembled SP (Fig. 7B). <br />                                                                              In absence Hsp31 number cells containing either <br />                                                                              P-bodies SGs significantly decreased (Fig. 7 A B), <br />                                                                              indicating formation impaired. <br />                                                                                 Although TORC1 known sequestered SGs upon <br />                                                                              heat shock (19), localization SP glucose starvation <br />                                                                              described. Given P-bodies struc- <br />                                                                              tures formed SP (23), SG formation depends on <br />                                                                              P-body assembly glucose starvation (21), hypothesized that <br />                                                                              TORC1 trapped P-bodies early SP. Thus, we <br />                                                                              genomically fused GFP Kog1, TORC1 subunit, monitor <br />                                                                              TORC1 localization SP. As predicted, Kog1 lost typ- <br /> Fig. 4. Atg13 phosphorylation altered hsp31&#206;&#8221; cells. Protein extracts <br />                                                                              ical vacuolar membrane localization early SP WT cells (24) <br /> were collected cultures midlog phase (Log) SP (A),      (Fig. S4B) and, instead, localized cytoplasmic foci (Fig. 7C). <br /> cultures starved carbon (SC-C) (B). Migration HA-Atg13   When coexpressed Kog1-GFP P-body marker Dcp1, <br /> SDS-polyacrylamide gels assessed using &#206;&#177;-HA. An &#206;&#177;-Pgk1 used      observed subset Kog1 granules clearly colocalized <br /> loading control. (C D) Atg13 immunoprecipitated &#206;&#177;-HA        Dcp1 foci, Kog1 foci positive Dcp1 <br /> analyzed PKA mediated phosphorylation using &#206;&#177;-(P)PKA. The numbers        vice versa. However, hsp31&#206;&#8221; cells observed only <br /> below immunoblots indicate ratio phosphorylated          fewer cells P-bodies, dramatic &#226;&#710;&#188;fourfold decrease <br /> total Atg13.                                                                 cells Kog1/P-body colocalization (P &lt; 0.01) (Fig. 7C). <br />  <br /> Miller-Fleming et al.                                                                                PNAS | May 13, 2014 | vol. 111 | no. 19 | 7015 <br />                                                                                  deficient G0 arrest reduced thermotolerance, are <br />                                                                                 similar observed strains deleted kinase Rim15 <br />                                                                                 integrates signals Sch9, PKA, TORC1 (33), and <br />                                                                                 transduces Gis1 [the transcription factor responsible for <br />                                                                                 expression postdiauxic-shift genes (9)] Msn2/Msn4 <br />                                                                                 (transcription factors responsible expression stress-in- <br />                                                                                 duced genes). We autophagy knockout strains <br />                                                                                 completely blocked carbon source depleted, it <br />                                                                                 identical WT response rapamycin treatment and <br />                                                                                 nitrogen starvation. This finding consistent impair- <br />                                                                                 ment observed SP, entry phase char- <br />                                                                                 acterized depletion carbon source; indicates <br />                                                                                 Hsp31 involved transducing carbon starvation signal <br />                                                                                 leads autophagy induction. This finding shows that <br />                                                                                 autophagic machinery working properly, importantly, <br />                                                                                 Hsp31 role downstream TORC1. <br />                                                                                    We assessed kinase activity TORC1 PKA using <br />                                                                                 Atg13 reporter. Phosphorylation Atg13 of <br />                                                                                 kinases represses autophagy, making key player this <br />                                                                                 process. We striking differences Atg13 phosphorylation <br />                                                                                 mediated TORC1 SP hsp31&#206;&#8221; control cells, <br />                                                                                 suggesting TORC1 hyperactive knockout strain. We <br /> Fig. 6. Hsp31 Hsp32 colocalize SG P-bodies. Midlog yeast WT        differences TORC1-mediated phosphorylation in <br /> W303 cells, grown presence glucose (+Glu), expressing          carbon starvation conditions and, unexpectedly, nitrogen <br /> Hsp31-GFP (A) Hsp32-GFP (B) driven constitutive GPD promoter        starvation. Interestingly, differences phosphorylation were <br /> (centromeric vector) subjected following treatments: 1 h         detected log-phase cells growing rich media, suggesting that <br /> glucose deprivation (&#226;&#710;&#8217;Glu), 30 min glucose readdition (&#226;&#710;&#8217;Glu+Glu), heat       HSP31 deletion leads higher basal TORC1 activation, is <br /> shock 46 &#194;&#176;C 30 min (HS), treatment CHX 2 min heat        independent PKA-mediated phosphorylation. <br /> shock 46 &#194;&#176;C (30 min) (CHX + HS). The percentage cells containing Hsp31-      Notably, Hsp31 Hsp32 associate components of <br /> GFP (A) Hsp32-GFP (B) foci displayed graphs. Error bars repre-    SGs P-bodies, cytoplasmic granules rapidly formed under <br /> sent SD. Statistical analysis performed using Student t test (**P &lt; <br />                                                                                 stress conditions (21) contain nontranslating mRNAs and <br /> 0.01, *P &lt; 0.05; 500 cells scored condition, n = 3). (C D) Cells <br /> coexpressing Hsp31-GFP Hsp32-GFP Pub1-mCherry (C), SG marker <br />                                                                                 proteins, allowing selective translation genes essential for <br /> or Edc3-mCherry (D), P-body marker, subjected heat shock as <br /> described analyzed confocal microscopy. (Scale bars, 5 &#206;&#188;m.) <br />  <br />  <br />  <br /> Curiously, P-bodies adjacent Kog1 foci sev- <br /> eral knockout cells. Taken together, data strongly suggest <br /> that Hsp31 plays critical role sequestration TORC1 <br /> into P-bodies glucose starvation. <br />  <br /> Discussion <br /> This study provides novel insights function Hsp31 <br /> minifamily. We HSP31 orthologs required <br /> for DS, occurs glucose limiting, leading to <br /> extensive metabolic transcriptional reprogramming (11, 14, 25). <br />    We <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> starvation induces expression of <br /> the Hsp31 minifamily. In agreement, Escherichia coli EcHsp31 <br /> is induced SP decreases cell survival phase <br /> when deleted (26). In addition, Caenorhabditis elegans DJ-1 <br /> homolog DJR-1.2 induced starvation dauer stage <br /> (starvation-induced alternative developmental stage) (27). In- <br /> terestingly, human leukemia cell lines depleted DJ-1 re- <br /> duced viability serum-starvation conditions (28). <br />    The TORC1 pathway major signaling pathways <br /> implicated DS progression SP (14, 29&#226;&#8364;&#8220;31). TORC1                  Fig. 7. P-body SG formation affected HSP31 deletion early SP. <br /> highly conserved nutrient-sensitive kinase controls growth              (A) WT hsp31&#206;&#8221; strains expressing Dcp1-DsRed, marker P-bodies, <br /> promoting transcription, protein translation, mRNA stability,               grown early SP (1 d growth) analyzed epifluor- <br /> and repressing stress responses autophagy (24, 32). Be-                  escence microscopy score percentage cells containing Dcp1 foci <br /> cause high complexity TORC1 signaling pathway                     (Student t test, *P &lt; 0.05, n = 3). Error bars represent SD. (B) WT hsp31&#206;&#8221; <br />                                                                                 cells expressing Pab1-GFP, marker SGs, analyzed 1 2 d of <br /> many questions unanswered, carbon avail- <br />                                                                                 growth fluorescence microscopy. The percentage cells containing Pab1 <br /> ability signaled TORC1 (24). The study pathway is <br />                                                                                 granules indicated left corner panel representing 2- <br /> difficult cross-talk signaling pathways,              d growth cultures; SD shown brackets. Five cells scored <br /> such PKA pathway (another key kinase signals                    condition (Student t test, P &lt; 0.05, n = 3). (C) Cells bearing genomically <br /> entry DS), overlapping readouts.                 tagged Kog1-GFP Dcp1-DsRed analyzed confocal microscopy <br /> Our data reveal HSP31 minifamily members potential                     1 d growth. The numbers indicated left corner the <br /> upstream effectors TORC1. We observed dramatic tran-                       panel merged field correspond percentage cells contain <br /> scriptional effect result deletion Hsp31 mini-                   Kog1-GFP foci colocalized Dcp1-DsRed. Only cells positive Dcp1 <br /> family members consequent absence typical SP                         used calculation (60 cells scored strain). SD indicated below <br /> characteristics. Interestingly, knockout phenotypes,                (Student t test, P &lt; 0.005, n = 3). (Scale bars, 5 &#206;&#188;m.) <br />  <br />  <br /> 7016 | www.pnas.org/cgi/doi/10.1073/pnas.1319221111                                                                                         Miller-Fleming et al. <br />                                                                                                                                                                                            SEE COMMENTARY <br /> survival (21, 22). We foci containing Hsp31 Hsp32                             critical investigate human DJ-1 plays <br /> share characteristics SGs P-bodies, including increased                              similar role modulation particles, this <br /> assembly stress, dependence presence non-                                  function contributes Parkinson disease pathogenesis. <br /> translating mRNAs, reversibility. Importantly, Hsp31/32 foci <br /> colocalize P-bodies SGs, known interact                               Materials Methods <br /> with (21). Curiously, different pattern                           Strains Plasmids. All strains plasmids used study described <br /> SG/P-body colocalization Hsp31 Hsp32 foci, sug-                                 Table S5. Primers used qRT-PCR listed Table S6. Further in- <br /> gesting slightly different roles process. The absence                            formation described SI Materials Methods. <br /> Hsp31 affected formation SGs P-bodies SP, <br /> further supporting role Hsp31 granules. We                              Media Growth Conditions. Yeast strains grown standard media. <br /> investigated subcellular localization TORC1, known                          For starvation experiments, cultures grown midlog phase in <br /> to sequestered SGs heat shock (19). We                                synthetic complete (SC) media, resuspended carbon starvation <br /> Kog1, TORC1 subunit, moves vacuolar membrane                                  media (SC-C), nitrogen starvation media (SC-N). Rapamycin (Sigma) was <br /> P-bodies. Interestingly, P-bodies previously shown                             added midlog-phase cultures growing SC-URA final concentration <br /> colocalize signaling molecules, including PKA sub-                           200 ng/mL. For GAL1 induction analysis, cultures grown SC-URA <br /> units Tpk2 Tpk3 SP (34). We observed presence                            2% (wt/vol) glucose midlog phase, washed, resuspended in <br /> Kog1 P-bodies clearly reduced hsp31&#206;&#8221;, strongly sup-                               SC-URA 2% galactose. <br /> porting TORC1 properly regulated deletion                              Microscopy, immunoblotting, DNA microarrays, qRT-PCR procedures <br /> Hsp31 minifamily genes. Future studies clarify                            described in SI Materials Methods. <br /> mis-localization directly caused absence Hsp31 and <br /> whether localization TORC1 P-bodies mechanism                            ACKNOWLEDGMENTS. We thank Prof. Paula Ludovico helpful discus- <br />                                                                                             sions; Prof. Claudina Rodrigues-Pousada access tetrad dissection <br /> reduction TORC1 activity glucose limiting.                                  microscope; Dr. Nicolas Sylvius NUCLEUS facility (University of <br />    Our findings suggest Hsp31 minifamily modulates                                 Leicester) assistance microarrays; Drs. Daniel Klionsky, Yosh- <br /> TORC1 activity yeast, potentially serving negative regulators                         inori Ohsumi, Tony Hazbun, Roy Parker, Uesono Yukifumi, Marek <br /> of autophagy (Fig. S5). Human DJ-1 previously asso-                                Skoneczny providing plasmids strains. This work supported <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           CELL BIOLOGY <br /> ciated autophagy <br /> </body></html>