<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast Wbpl p and Swpl p form a protein complex <br /> essential for oligosaccharyl transferase activity <br />  <br />  <br />                                     (Kelleher et al., 1992). Ribophorins are highly conserved <br />                                         proteins in the rough endoplasmic reticulum and found <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                       be associated with the translocation machinery in the ER <br />    membrane (Kreibich et al., 1983; Marcantonio et al., 1984; <br />      Rapoport, 1991). <br />  <br />                                                                          We have recently shown that in the yeast Saccharomyces <br />                                          cerevisiae the essential gene WBPJ encodes a protein <br /> Asparagine-linked N-glycosylation is an essential protein             necessary for N-oligosaccharyl transferase activity in vivo <br /> modification occurring in all eukaryotic cells. The central           and in vitro. Depletion of Wbplp led to a loss of N- <br /> step is the co-translational transfer of the core                     oligosaccharyl transferase activity (te Heesen et al., 1992). <br /> oligosaccharide assembled on the lipid carrier dolichol               We proposed that Wbplp is an essential component of the <br />                                                                       N-oligosaccharyl transferase complex. This conclusion is <br /> phosphate to selected Asn-X-Ser/Thr residues of nascent               supported by the recent finding that the 48 kDa component <br /> polypeptide chains in the endoplasmic reticulum. This                 of the canine pancreas N-oligosaccharyl transferase (Kelleher <br /> reaction is catalyzed by the enzyme N-oligosaccharyl                  et al., 1992) has sequence homology to the yeast Wbpl <br /> transferase. In yeast, Wbplp is an essential component                protein (Silberstein et al., 1992). Therefore, the N- <br /> of this enzyme. Using a high copy number suppression                  oligosaccharyl transferase (or at least one component of it) <br /> approach, the SWPI gene was isolated as an allele specific            is structurally conserved <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to mammals. <br /> suppressor of a wbpl <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. Swplp is a 30 kDa type                    In this report, we describe the identification and <br /> I transmembrane protein and essential for cell viability.             characterization of an additional component, Swplp, of the <br /> Similar to Wbplp, depletion of Swplp results in reduced               yeast N-oligosaccharyl transferase. We show that this <br /> N-oligosaccharyl transferase activity in vivo and in vitro.           component, like Wbplp, is essential for N-oligosaccharyl <br /> Wbplp and Swplp can be chemically cross-linked,                       transferase activity in vivo and in vitro. Cross-linking <br /> suggesting that both proteins are essential constituents              experiments and genetic data indicate that the two proteins <br /> of the N-tligosaccharyl transferase complex.                          form a complex in vivo. <br /> Key words: endoplasmic reticulum/glycosylation/oligo- <br /> saccharyl transferase/Saccharomyces cerevisiaeltransmem- <br /> brane protein                                                         Results <br />                                                                       Isolation of the SWP1 gene <br />                                                                       As noted previously, the temperature-sensitive <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br /> Introduction                                                          wbpl mutants is due to a reduced level of mutant Wbplp <br /> N-linked glycosylation of proteins is an essential modification       at permissive and non-permissive temperatures (te Heesen <br /> of secreted proteins in the endoplasmic reticulum and follows         et al., 1992). It is possible that a reduced stability of the <br /> a highly conserved pathway in all eukaryotic cells. A core            mutant Wbpl protein is the cause of this defect. We <br /> oligosaccharide of the composition [Glc3Man9(GlcNAc)2] is             hypothesized that stability could be (partially) restored by <br /> first assembled on the lipid carrier dolichol phosphate and           an increased concentration of an appropriate Wbplp complex <br /> then transferred to selected asparagine residues of nascent           partner. Therefore, we applied a high copy number <br /> polypeptide chains by the enzyme N-oligosaccharyl                     suppressor approach (Rine, 1991) and screened a high copy <br /> transferase (OTase) (Kaplan et al., 1987; Tanner and Lehle,           number plasmid library containing yeast genomic DNA for <br /> 1987).                                                                the presence of a wbpl suppressor. To construct a high copy <br />   A biochemical characterization of the N-oligosaccharyl              number plasmid library, total genomic DNA of the strain <br /> transferase has proven to be difficult: attempts to purify this       45-C3 (Awbpl::HIS3 URA3:Gall-WBPI ade2-101 tyri) (te <br /> enzyme have failed due to the lability of the enzyme upon             Heesen et al., 1992) was isolated. This strain contains a <br /> solubilization (Das and Heath, 1980; Sharma et al., 1981;             hybrid WBPI gene: the expression of the wild-type WBPI <br /> Kaplan et al., 1987). Recently, it has been shown that                protein is directed by the GAL] promoter. Since the GAL] <br /> mammalian N-oligosaccharyl transferase activity co-                   promoter activity is repressed in the presence of glucose, <br /> fractionated with a protein complex of ribophorins I and II           this modified WBPI locus should not be able to complement <br /> and a 48 kDa protein (Kelleher et al., 1992). Antibodies              a wbpl mutation when the cells are grown on glucose- <br /> directed against ribophorin I were able to precipitate N-             containing medium. DNA was partially digested with Sau3A, <br /> oligosaccharyl transferase activity (Kelleher et al., 1992).          fragments between 4 and 10 kb were isolated and ligated <br /> The presence of a potential binding site for dolichol (Albright       into the BamHI-cleaved URA3-containing vector YEp352 <br /> et al., 1989) in the transmembrane domain of ribophorin I             (Hill et al., 1986). This plasmid pool, resulting from <br /> led to the suggestion that this protein might be involved in          120 000 individual transformants, was used to transform the <br /> the binding of the lipid-linked core oligosaccharide substrate        temperature-sensitive strain MA9-D (relevant phenotype: <br />     Oxford University Press <br />  <br />  <br /> Fig. 1. (A) Restriction map of the SWPI locus. The cleavage site for selected restriction enzymes is shown. The location of the Swplp encoding <br /> open reading frame is shown as an arrow. (B) Nucleotide sequence of the PstI-BamHI fragment of the SWPI locus and the predicted amino acid <br /> sequence of Swplp. Numbering of the nucleotides, starting with the ATG initiation codon, is indicated above the sequence. Numbering of the amino <br /> acid residues of Swplp is given at the right side of the sequence. (C) Hydropathy analysis of Swplp according to Kyte and Doolittle. The N- <br /> terminal and the three C-terminal hydrophobic domains are indicated by bars. <br />  <br /> wbpl-2 ura3-52) to uracil prototrophy at 23&#194;&#176;C on glucose-                                                                                                         the suppressing activity on a 1.7 kb PstI-BamHI DNA <br /> containing medium. The resulting 60 000 colonies were                                                                                                             fragment (Figure IA). This region was sequenced <br /> screened by replica-plating for a temperature-resistant                                                                                                           (Figure 1B) and the corresponding locus was termed SWPI <br /> phenotype. Seventeen revertants were isolated, the plasmids                                                                                                       (suppressor of wbpl). <br /> conferring the temperature-resistant phenotype were <br /> recovered in Escherichia coli and analyzed by restriction                                                                                                         The SWP1 gene encodes a potential transmembrane <br /> analysis. All except one of these plasmids contained WBPI                                                                                                         protein essential for vegetative growth <br /> specific sequences. Restriction analysis of WBPI positive                                                                                                         The 1.7 kb PstI-BamHI fragment capable of suppressing <br /> plasmids revealed small deletions within the GALI-JO                                                                                                              the temperature-sensitive phenotype of wbpl-2 strains <br /> promoter region in some of these plasmids, but no detailed                                                                                                        contains one large open reading frame predicted to encode <br /> analysis was performed. The remaining plasmid was able                                                                                                            a protein of 286 amino acid residues (Figure lB). <br /> to confer temperature resistance when transformed into strain                                                                                                     Hydrophilicity analysis of the corresponding protein revealed <br /> MA9-D (wbpl-2). Interestingly, no temperature-resistant                                                                                                           a potential N-terminal signal sequence and a long C-terminal <br /> transformants were recovered when strain MA7-B (wbp]-l)                                                                                                           hydrophobic region with the capacity to form three <br /> was used as recipient, indicating an allele specificity of the                                                                                                    transmembrane domains (Figure 1C). The predicted <br /> suppressor plasmid (see below). Deletion analysis located                                                                                                         molecular weight of mature Swplp is 29.3 kDa. No <br />  <br />  A                                     B <br /> Fig. 2. Swplp is an essential protein. Tetrad analysis of diploid strains                   VW               ,*         uwm    4_        - <br />                                                                                                                                              mature CPY <br />  <br /> heterozygous for the SWPI deletion. Tetrads obtained from sporulation <br /> of the strain ATH7 (A), heterozygous for the deletion in the SWPI <br /> coding sequence, and the strain ATH7, carrying the SWPI gene on the <br /> plasmid pRS313(SWPI-Pst-Bam) (B), were dissected on YPD medium, <br /> the four spores of one tetrad in a column, and grown for 4 days at          Fig. 3. SWPI is an allele specific wbpl suppressor. The two strains <br /> 30&#194;&#176;C. The colonies were replica-plated to determine their phenotype.        MA-7B (wbpl-J) and MA9-D (wbpl-2) were transformed with either <br /> All growing colonies resulting from strain ATH7 were ura-,                  YEp352 (+ vector), YEp352(WBP1) (+ WBP1) or YEp352(SWP-Pst- <br /> indicating the lethal effect of the SWP1 deletion. This lethality could     Bam) (+ SWP1) as indicated. Strains were grown under selective <br /> be overcome by the expression of the SWPI gene on the plasmid               conditions at 23&#194;&#176;C, labeled for 1 h, lysed and used for CPY specific <br /> pRS313(SWPI-Pst-Bam). Improper segregation of the complementing             immunoprecipitation. The position of mature CPY is indicated; the two <br /> plasmid pRS313(SWPI-Pst-Bam) can account for the 1:3 and 2:2                wbpl mutations lead to underglycosylation of CPY. <br /> segregation of lethality observed in Figure 2B. <br />                                                                             Table I. In vitro activity of OTase in different cell extracts <br /> significant homology to other known sequences was detected <br /> in several databases using the FASTA or WORDSEARCH                          Strain                                           Glycosyl transfer to peptide (S) <br /> programs. No potential N-linked glycosylation site is present <br /> within the Swplp coding sequence. To test if SWPJ encodes                   SS328 (wild-type)                                100 <br /> an essential protein, a deletion version of the SWPJ locus                  MA9-D (wbpl-2)                                    10 <br /> was constructed in which the ClaI -BglII fragment of SWPI                   MA9-D (YEp352/SWPl-Pst-Bam)                       50 <br /> (Figure lA) was replaced by the 1. I kb HindIll fragment                    MA9-D (YEp352/WBPI)                               90 <br /> of the URA3 locus (Rose et al., 1983). This construct was                   The different strains were grown to mid-log phase, extracts were <br /> used to replace one wild-type copy of the SWPI locus in                     prepared and oligosaccharyl transferase activity was measured as given <br /> the diploid strain SS328 x SS330. Correct replacement was                   in Materials and methods. The value obtained from wild-type extracts <br /> monitored by genomic Southern blot analysis of resulting                    was taken as 100%. <br /> Ura+ transformants. Sporulation of such strains and <br /> resulting tetrad analysis showed that haploid segregants                    glycosylation <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> was seen in the wbpl-l strain. To <br /> containing the SWPI deletion were inviable (Figure 2). The                  demonstrate unambiguously that core oligosaccharide <br /> deletion-containing spores were able to germinate but only                  transfer was affected, N-oligosaccharyl transferase activity <br /> micro-colonies of 20 cells were formed (data not shown). <br />                        - <br />                                                                             was determined in extracts prepared from wbpl-2 mutant <br /> The <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> of the SWPI deletion could be rescued                  cells transformed with either SWPI or WBPI (Table I). <br /> by a plasmid that contains the intact SWPJ allele (Figure 2).               Extracts from wbpl-2 mutant cells contained a 10-fold <br /> We conclude that like Wbplp, the SWPl protein is essential                  reduced OTase activity compared with wild-type extracts. <br /> for the vegetative growth of yeast.                                         A high copy number plasmid containing the SWPI locus <br />                                                                             restored OTase activity to 50% of the wild-type level, <br /> Overexpression of SWP1 restored the glycosylation                           whereas the WBPI gene could fully complement the wbpl-2 <br /> deficiency in wbp 1-2 but not in wbp 1-1 strains                            defect. These results establish that SWPI is a wbpl-2 specific <br /> As outlined above, overexpression of SWPI was capable of                    high copy number suppressor for both the temperature- <br /> suppressing the temperature-sensitive phenotype in wbpl-2                   sensitive phenotype and the constitutive underglycosylation <br /> but not in wbpl-l strains. Both wbpl alleles result in                      of proteins. <br /> underglycosylation of proteins in permissive as well as non- <br /> permissive conditions (te Heesen et al., 1992). If the growth               Depletion of the SWP1 protein results in <br /> defect in the wbpl strains were due to this                                 underglycosylation of CPY in vivo and a reduced N- <br /> underglycosylation, we would expect glycosylation to be                     oligosaccharyl transferase activity in vitro <br /> restored in a wbpl-2 but not in a wbpl-l strain transformed                 The depletion of the WBPI protein leads to <br /> with a high copy number SWPI plasmid. The glycosylation                     underglycosylation of CPY due to a reduced N- <br /> capacity of the cells was measured by the analysis of                       oligosaccharyl transferase activity. We asked if depletion of <br /> carboxypeptidase Y (CPY) (Stevens et al., 1982). Mature                     the SWPI protein results in a similar phenotype. For that <br /> CPY contains four N-linked oligosaccharides and has a                       purpose, the diploid strain heterozygous for the SWPJ <br /> molecular weight of 61 kDa (Hasilik and Tanner, 1978).                      deletion was transformed with a low copy number plasmid <br /> The wbpl-l and wbpl-2 mutations affect CPY glycosylation                    carrying a GALI-SWPI hybrid gene. In this gene, the <br /> at the permissive as well as the non-permissive temperature,                expression of the SWPI coding region is controlled by the <br /> resulting in underglycosylated CPY (Figure 3) (te Heesen                    GAL] promoter. After sporulation of such a strain and tetrad <br /> et al., 1992). Whereas expression of wild-type Wbplp                        dissection, haploid strains were recovered carrying a deletion <br /> restored the glycosylation defect in both mutant strains, high              of the chromosomal SWPJ gene but containing the <br /> copy number expression of SWPI had an effect only in the                    complementing plasmid. Such strains grow on galactose- but <br /> wbpl-2 strain: fully glycosylated and small amounts of CPY                  not on glucose-containing medium. We analyzed the time- <br /> lacking one oligosaccharide chain were detected in this strain.             dependent effect of Swplp depletion on CPY glycosylation <br /> Only a slight influence of SWPI expression on the CPY                       (Figure 4). We observed the first reduced glycosylation of <br />                                                                                                                                                           281 <br />  S.te Heesen et al. <br />  <br />                     Oh   31--   6h   (411   12h I 8h 24h                             Table II. In vitro activity of OTase in Swplp-depleted cell extracts <br /> mliature   PY   - <br />                                             ... <br />                                                                 unnderglycosylated <br />                                                                                      Strain                  Time after shift               Glycosyl transfer <br />                                                                                                              to glucose (h)                 to peptide (%) <br />                                                        .: <br />  <br />  <br />                                                             I          cpv <br />                                                                                      AT- <br /> Fig. 4. Depletion of Swplp alters CPY glycosylation. Strain ATH8                     H8                       0                             100 <br /> was shifted from galactose- to glucose-containing selective rmiinimal                                         6                              70 <br /> medium, resulting in the repression of Swplp synthesis. Samples were                                         12                              23 <br /> taken at the time indicated, labeled for 1 h, lysed and used for CPY                                         18                               9 <br /> specific immunoprecipitation as given in Materials and methods. The                                          24                               8 <br /> position of fully glycosylated, mature CPY is indicated. Depletion of <br /> Swplp results in the accumulation of underglycosylated CPY.                          Strain ATH8 was grown in selective mninimal medium containing <br />                                                                                      galactose, washed and used to innoculate minimal medium with <br /> CPY after 6 h in glucose-containing medium, and the amount                           glucose as carbon source. Cultures were grown at 30&#194;&#176;C for the time <br /> of underglycosylated CPY increased with time. Under-                                 indicated, extracts were prepared and oligosaccharyl transferase activity <br />                                                                                      was measured as given in Materials and methods. The value obtained <br /> glycosylated CPY gave rise to a multiplet of bands with a                            at the shift from galactose to glucose medium was taken as 100%. <br /> higher mobility than mature CPY, representing <br /> underglycosylated forms lacking one, two, three or all four <br /> N-linked oligosaccharide chains. The same CPY pattern was                                        -DSP             +DSP <br /> observed in strains depleted for the WBPI protein (te Heesen                                                             C <br /> et al., 1992) or carrying the wbpl-l (or wbpl-2) mutation.                                       a)     X          a) <br />                                                                                                                    4-~       - <br />  <br />  <br />  <br /> A similar underglycosylation due to Swplp depletion was                                                            &gt;Q <br />                                                                                                                   -0--   1 <br />  <br />  <br />  <br /> observed for proteinase A (PrA) (data not shown). The                                           -6     a <br /> hypothesis that this underglycosylation is due to a reduced                                            cl, <br /> N-oligosaccharyl transferase activity was verified by <br /> determining the in vitro activity of this enzyme in Swplp-                                                                       -   ORI <br /> depleted cells (Table II). It is evident that growth in glucose <br /> medium of a GAL] -SWPI strain leads to a time dependent <br /> decrease in OTase activity. Therefore, the SWP1 protein <br /> is as essential for N-oligosaccharyl transferase activity in vivo                                        I... <br />                                                                                                         I-,                          8OkDa <br /> and in vitro as Wbplp.                                                                                                           -   7OkDa <br />                                                                                                                                      68kDa <br /> Wbp lp and Swp lp form a protein complex                                                                                         -   Wbpl p <br /> The observation that (i) high copy number expression of the <br /> SWPI gene could suppress the wbpl-2 mutation and (ii) that <br /> Wbplp and Swplp depletion lead to a similar phenotype                                                                                Swpl p-tag <br /> suggests that these two proteins form a complex in vivo                                                                              Swplp <br /> essential for N-oligosaccharyl transferase activity. To test                                                                         3OkDa <br /> this hypothesis, cross-linking experiments were performed.                                       1     2          3      4 <br /> Membrane fractions of metabolically labeled wild-type cells <br /> were prepared, solubilized with Triton X-100 and treated                             Fig. 5. Cross-linking of Wbplp to Swplp. Radiolabeled, solubilized <br /> with the cross-linking reagent dithiobis(succinimidylpropio-                         membrane extracts were treated with dithiobis(succinimidylpropionate) <br /> nate) (DSP) (Deshaies et al., 1991). Cross-linked products                           (+DSP) at 23&#194;&#176;C (lanes 3 and 4). Products were immunoprecipitated <br />                                                                                      with affinity-purified anti-Wbplp antibodies, the cross-linker was <br /> were denatured and immunoprecipitation was performed                                 reduced and the samples were analyzed by SDS-PAGE. In control <br /> under non-reducing conditions using affinity-purified anti-                          experiments, the cross-linking step was omitted and SDS was added <br /> Wbplp antibodies. After cleavage of the cross-linker with                            before immunoprecipitation (-DSP) (lanes 1 and 2). Extracts were <br /> DTT, the products were analyzed on SDS -PAGE. Beside                                 prepared from strain SS328 (wild-type) (lanes 1 and 3) and ATH1 <br /> Wbplp, additional proteins were precipitated from cross-                             (SWPI-tag) (lanes 2 and 4). The positions of Wbplp, Swplp (30 <br />                                                                                      kDa), the tagged Swplp (31.5 kDa) and additional cross-linked <br /> linked extracts: proteins migrating with a mobility of 80,                           proteins are indicated. <br /> 70, 68 and 30 kDa were detected (Figure 5, lane 3). Among <br /> these proteins, the 30 kDa protein is cross-linked most <br /> efficiently to Wbplp. Immunoprecipitation without cross-                             used this strain to perform cross-linking experiments as <br /> linking resulted primarily in the detection of Wbplp                                 described above. If the 30 kDa protein cross-linked to Wbplp <br /> (Figure 5, lanes 1 and 2). Since 30 kDa is the expected                              is Swplp, we expected that the 30 kDa protein from the <br /> molecular weight of Swplp, we tested if the 30 kDa protein                           SWPI modified cells would be replaced by a protein of <br /> is Swplp. We constructed a yeast strain with a modified                               - <br />                                                                                         31.5 kDa. Indeed, the expected mobility-shift of the small <br /> SWPI locus. A tag sequence encoding 12 amino acids was                               cross-linked protein was detected in extracts derived from <br /> inserted at the 3&apos; end of the SWPI open reading frame. A                             SWPI modified cells as compared with wild-type cells <br /> yeast strain, ATH1, containing only the tagged Swplp was                             (Figure 5, lanes 3 and 4). Our results prove that Swplp can <br /> constructed. Phenotypically, this strain could not be                                be specifically cross-linked to Wbplp in solubilized <br /> distinguished from a wild-type strain, indicating that the C-                        membrane fractions, arguing for a direct interaction between <br /> terminal extension did not impair Swplp function. We have                            these two proteins. <br />  <br /> 282 <br />                                                                                                Yeast oligosaccharyl transferase complex <br />  <br /> Discussion                                                        dolichol consensus sequence: within amino acid positions <br /> We have isolated the essential SWPI gene of S. cerevisiae         196-206 and 234-244 (see Figure 1) we observe a four <br /> based on its capacity to suppress the temperature-sensitive       out of seven and three out of seven match to the consensus <br /> phenotype of the wbpl-2 mutation. Several experimental            sequence. However, we have no direct proof that these <br /> findings suggest that Swplp and Wbplp form a complex              regions interact with dolichol. <br /> essential for N-oligosaccharyl transferase: (i) Depletion of         Recently, Kelleher et al. (Kelleher et al., 1992) have <br /> Wbplp and Swplp lead to underglycosylation of proteins            found that N-oligosaccharyl transferase activity from dog <br /> and to a concomitant reduction of oligosaccharyl transferase      pancreas is present in a complex consisting of ribophorins <br /> activity in extracts from depleted cells. (ii) Swplp could be     I and II and a 48 kDa protein. This 48 kDa protein, called <br /> cross-linked to Wbplp in solubilized membranes, suggesting        OST48p, is homologous to the yeast Wbplp (Silberstein, <br /> a close contact between the two proteins. (iii) The presence      1992) suggesting that N-oligosaccharyl transferase is <br /> of SWPI on a high copy number plasmid suppresses only             structurally conserved during evolution. However, the 48 <br /> a wbpl-2 but not a wbpl-l mutation. This allele specificity       kDa Wbplp homolog OST48p was found in a complex with <br /> is best explained by a direct interaction between the two         ribophorins I and H (Kelleher et al., 1992). two proteins <br /> proteins (Huffaker et al., 1987).                                 which have not been identified so far in yeast (Sanderson <br />    Based on the primary amino acid sequence we propose            et al., 1990). The purification of dog pancreas N- <br /> that Swplp (like Wbplp) is a type I transmembrane protein.        oligosaccharyl transferase did not yield a protein similar in <br /> The first 22 amino acids represent a potential signal sequence    size to Swplp (30 kDa) (Kelleher et al., 1992) and Swplp <br /> to direct import into the endoplasmic reticulum (von Heijne,      has no sequence similarity to either ribophorin I or II. It is <br /> 1986). The C-terminal part of the protein is very hydrophobic     possible that only one component of N-oligosaccharyl <br /> and three potential transmembrane domains can be predicted.       transferase (Wbplp/OST48p) is conserved in evolution. <br /> We therefore propose that the N-terminal portion of Swplp         Alternatively, the multi-component complex is conserved <br /> is located in the lumen of the endoplasmic reticulum,             between different species, but the yeast ribophorin homologs <br /> separated by three membrane-spanning domains from the             and the Swplp homolog of higher eukaryotes have not been <br /> short, cytoplasmic tail. In support of this hypothesis, we        isolated so far. Our results that additional proteins beside <br /> could recently show that fusing the histidinol dehydrogenase      Swplp can be cross-linked to Wbplp (Figure 5) suggest that <br /> protein domain to the C-terminus of Swplp allowed growth          additional component(s) of yeast N-oligosaccharyl transferase <br /> of a his4 strain on histidinol (M.Zecca, S.te Heesen and          complex may exist. Indeed, we observe that simultaneous <br /> M.Aebi, unpublished). This result indicates that the C-           overexpression of both Wbplp and Swplp does not lead to <br /> terminus of Swplp is located in the cytoplasm, since              a significant increase of N-oligosaccharyl transferase activity <br /> histidinol can only be converted to histidine by a cytoplasmic    in extracts from such cells, though a Wbplp-Swplp <br /> histidinol dehydrogenase (Sengstag et al., 1990). Since           complex accumulates in these cells (data not shown). Both <br />                                                                   Wbplp and Swplp are therefore non-limiting components <br /> Wbplp is also a type I transmembrane protein, the                 of the yeast N-oligosaccharyl transferase. Combined genetic <br /> Wbplp-Swplp complex is exposed to the lumen of the ER.            and biochemical approaches will allow us to identify such <br />    At present, we have isolated two essential components of       additional components. With these components identified we <br /> the yeast N-oligosaccharyl transferase, but we have no direct     should be able to define the functional role of the different <br /> indication for the individual functional role of these            subunits in the transfer of the core oligosaccharide to nascent <br /> components in the transfer reaction of the core                   polypeptide chains. <br /> oligosaccharide to asparagine residues of nascent <br /> polypeptides. wbpl-l and wbpl-2 mutants are characterized <br /> by a reduced level of mutant Wbplp, resulting in a decreased <br /> N-oligosaccharyl transferase activity (te Heesen et al., 1992).   Materials and methods <br /> This reduction of N-linked glycosylation results in a             Yeast strains and media <br /> temperature-sensitive phenotype. The observation that the         The following yeast strains were used in this study: X2180-1A (Yeast Genetic <br /> presence of SWPI on a high copy number plasmid can rescue         Stock Center, Berkley, CA); SS328 (MATa ade2-101 his3A200 ura3-52 <br /> a temperature-sensitive wbpl-2 mutation suggests that             lys2-801) (Vijayraghavan et al., 1989); SS330 (MATa ade2-101 his3A200 <br /> Wbplp is stabilized in a complex with Swplp. In support           ura3-52 tyri) (Vijayraghavan et al., 1989); MA7-B (MATa wbpl-l <br />                                                                   ade2-101 his3A200 ura3-52 lys2-801) (te Heesen et al., 1992); MA9-D <br /> of this notion we observe that depletion of Swplp results         (MATa wbpl-2 ade2-101 his3A200 ura3-52 lys2-801) (te Heesen et al., <br /> in a concomitant reduction of Wbplp (data not shown), but         1992); 45-C3 (MAThx his3A200 Awbpl::HIS3 URA3:GALJ-WBP1 ade2-101 <br /> we do not know at present whether depletion of Wbplp also         tyri) (te Heesen et al., 1992); ATH1 (MATa ade2-101 his3A200 ura3-52 <br /> results in a similar reduction of Swplp.                          lys2-801 SWPI-tag:URA3) (this study); ATH7 (MATa/a <br />    One substrate of N-oligosaccharyl transferase is the           ade2-10lade2-101 his3A2()/his3vA200 ura3-521ura3-52 Iys2/+ tyrll + <br />                                                                   Aswpl::URA31+) (this study); ATH8 [MATa ade2-101 his3A200 ura3-52 <br /> dolichol-bound core oligosaccharide. It has been noted that       Aswpl::URA3 pRS313(SWP1-Pst-Bam)] (this study). Standard yeast media <br /> several dolichol binding enzymes contain an 11 residue            (Sherman et al., 1983) were used. <br /> consensus sequence in the membrane-spanning domains. <br /> This sequence was implicated in the binding of dolichol, but      Isolation of the SWP1 locus <br /> no experimental evidence exists to support this hypothesis        DNA techniques have been described (Sambrook et al., 1989). Chromosomal <br /> (Albright et al., 1989; Lehrman, 1991). Interestingly, this       DNA was isolated from strain 45-C3 and partially digested with Sau3A. <br />                                                                   DNA fragments between 4 and 10 kb in size were fractionated by agarose- <br /> consensus sequence was also observed in the transmembrane         gel electrophoresis and ligated into the vector YEP352 (Hill et al., 1986). <br /> domain of ribophorin I (Kelleher et al., 1992) (see below).       After transformation and amplification in the E. coli strain DH5ai (Hanahan, <br /> We note that two of the three potential membrane-spanning         1983), the plasmid pool was transformed into the yeast strain MA9-D (Ito <br /> domains of Swplp also contain sequences similar to the            et al., 1983) and uracil prototrophic transformants were selected at permnissive <br />  <br />  <br />  <br /> temperature (23&#194;&#176;C). Temperature-resistant colonies were screened by replica-     was omitted. Immunoprecipitation of CPY was as described (te Heesen et al., <br /> plating on selective minimal medium and YEPD medium and incubation               1992). Labeled proteins were analyzed by 9% (Wbplp/Swplp) or 8% (CPY) <br /> of the plates at 37&#194;&#176;C. Resistant strains were colony-purified, total DNA         SDS-PAGE, followed by fluorography (Chamberlain, 1979) and exposure <br /> was isolated and plasmid DNA was recovered by transformation into E. coli        to Kodak XAR-5 films at -80&#194;&#176;C. <br /> strain DH5S, analyzed by restriction digest analysis and tested if they were <br /> able to transform yeast strain MA9-D to temperature resistance. Plasmid          Determination of N-oligosaccharyl transferase activity <br /> YEP352/SWP1 was analyzed further and the wbpl-2 complementing                    Preparation of membrane fractions and determination of OTase activity have <br /> PstI-BamHI fragment was sequenced. The DNA sequence was submitted                been described (te Heesen et al., 1992). <br /> to the EMBL Data Bank, accession number X67705. For further <br /> experiments, the PstI-BamHI fragment of the SWPJ locus was ligated into <br /> the 2Am-derived vector YEp352 [YEp352(SWP1-Pst-Bam)] and the CEN-               <br /> ARS plasmid pRS313 (Sikorski and Hieter, 1989) [pRS313(SWPI-Pst-                <br /> Bam)].                                                                      <br /> Disruption of the SWP1 locus                                                     <br /> The 1.1 kb HindIII fragment of the URA3 locus (Rose et al., 1983) was            <br /> inserted between the ClaI and BglH sites of the SWPJ locus (see Figure 1),        <br /> resulting in the deletion of codons 51-280 of the SWPJ ORF. This version <br /> of the SWPJ locus was used to replace one wild-type copy of the SWPJ <br /> gene of the diploid strain SS328 xSS330 by homologous recombination,              <br /> selecting for uracil prototrophy. <br />                                                                                  <br /> Construction of pRS313(GAL 1- SWP1)                                                 <br /> A Hindm site was introduced at position -29 of the SWPI gene by standard         <br /> PCR methods, resulting in the sequence 5&apos;-AAGCTTAGACCATAAAA-3&apos;.                   <br /> The resulting SWPI gene was cleaved with HindIII and BamHI, located                 <br /> downstream of the SWPJ coding sequence, mixed with the EcoRI-HindIII             <br /> GALI-JO promoter fragment of YIpGAL(WBP1) (te Heesen et al., 1992)                   <br /> and ligated into the EcoRI-BamHI cleaved vector pRS303 (Sikorski and             <br /> Hieter, 1989). The GALl -SWPI <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> was then transferred as a 2 kb                <br /> Sall-BamHI fragment to the CEN-ARS vector pRS313 (Sikorski and Hieter,             <br /> 1989) to yield pRS313(GAL1 -SWP1). The coding sequence of this modified          <br /> SWPJ gene was sequenced. The diploid strain ATH7 was transformed with            <br /> this plasmid to histidine prototrophy, sporulated and tetrads were dissected        <br /> on plates containing galactose as sole carbon source.                           <br /> Construction of the SWP1 -tag strain ATH1                                         <br /> First, a Narn site was introduced at position 63 of the SWPI gene by standard       <br /> PCR methods. The resulting sequence was 5&apos;-GGCGCCAGATGTTCA-                      <br /> TG-3&apos;. The NarI-KpnI fragment containing a 5&apos;-truncated SWPI locus                  <br /> was ligated into the ClaI-KpnI cleaved vector pBS/URA3. This vector               <br /> contains the 1.1 kb Hindm fragment of the URA3 locus (Rose et al., 1983)         <br /> within the polylinker sequence of pBluescript KS(+). Transcription of the          <br /> URA3 gene is toward the Sacd site of the polylinker. The resulting plasmid        <br /> was called pBS(SWPANarI)URA3. In this plasmid, the 3&apos; non-translated             <br /> region of the SWPI gene was replaced by a synthetic, tag encoding sequence        <br /> and 0.7 kb of the 3&apos; non-translated region of the WBPI gene. The new             <br /> C-terminus encoding sequence is as follows: 5&apos;-GATCTTTTGGTCAA-                   <br /> ACAATCctcgagcaaaagttgatctctgaagaagacttgaactagTATATGTITT-3&apos;. The                   <br /> bold letters represent SWPJ sequence from position 839-858, the lower         <br /> case letters an artificial sequence encoding the c-myc tag-containing sequence      <br /> LEQKLISEEDLN and the italic letters WBPI sequence from position                     <br /> 1294-1303 of the WBPI locus (te Heesen et al., 1991). This plasmid,            <br /> pBS(SWPIANarI/tag)URA3, was linearized with ClaI, located at position             <br /> 145 in the SWPI coding sequence, and used to transform strain SS328 to           <br /> uracil prototrophy. Correct integration of the plasmid into the SWPI locus        <br /> was monitored by PCR analysis of genomic DNA of the transformants.               <br /> Integration of the plasmid at the ClaI site results in an inactive copy due        <br /> to the NarI deletion and an active copy expressing the tagged Swplp. The        <br /> resulting strain was called ATH1.                                                   <br /> Immunological methods                                                             <br /> Recombinant Wbplp (codons 14-396) was produced in E.coli BL21 cells               <br /> using the pET system (Studier and Moffat, 1986) and purified from inclusion      <br /> bodies by preparative SDS -PAGE. The material was used to immunize               <br /> a New Zealand White rabbit. The serum obtained was purified by affinity           <br /> columns containing the pure recombinant rWbplp coupled in 100 mM MOPS             <br /> (pH 7.5) to AffiGel 15 matrix using standard techniques (Harlow and Lane,        <br /> 1988).                                                                           <br />   For cross-linking experiments, yeast strains were grown in minimal                <br /> medium containing the appropriate amino acid and nucleoside supplements           <br /> to a density of - 107 cells/mi. 109 cells were labeled with 1.5 mCi of              <br /> [35S]methionine (&gt; 1000 Ci/mmol, Amersham) for 1 h at 30&#194;&#176;C.                      <br /> Preparation of labeled yeast membranes, cross-linking with DSP (Fluka, <br /> Switzerland) and immunoprecipitation were as described (Deshaies et al.,         <br /> 1991) except that 20 Al of affinity-purified anti-Wbplp antiserum was used <br /> and the treatment of extracts with IgSorb prior to incubation with antibody <br /> </body></html>