<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Intrinsic biocontainment: Multiplex genome safeguards <br /> combine transcriptional recombinational control <br /> of essential yeast genes <br /> Yizhi Caia,b,1, Neta Agmonb,c, Woo Jin Choib, Alba Ubidea, Giovanni Stracquadaniob,d, Katrina Caravellib, Haiping Haob, <br /> Joel S. Baderb,d, Jef D. Boekeb,c,1 <br /> a <br />  School Biological Sciences, University Edinburgh, Edinburgh EH9 3BF, Scotland, United Kingdom; bSchool Medicine, The Johns Hopkins University, <br /> Baltimore, MD 21205; cInstitute Systems Genetics, NYU Langone Medical Center, New York, NY 10016; dWhiting School Engineering, The Johns <br /> Hopkins University, Baltimore, MD 21218 <br />  <br /> Contributed Jef D. Boeke, January 2, 2015 (sent review November 12, 2014; reviewed James J. Collins) <br />  <br /> Biocontainment required wide variety situations                complexity likely defeat wants decode the <br /> such work pathogens, field release applications engi-              safeguard strains. <br /> neered organisms, protection intellectual properties. Here,               Previous work biocontainment largely focused on <br /> we control growth brewer&#226;&#8364;&#8482;s yeast, Saccharo-            auxotrophic mutations inducible lethality based toxin&#226;&#8364;&#8220; <br /> myces cerevisiae, using transcriptional recombinational              antitoxin pairs bacteria (18&#226;&#8364;&#8220;21). There recent work <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> &#226;&#8364;&#339;safeguard&#226;&#8364;? control essential gene function. Practical biocontain-         reduce unintended plasmid propagation bacteria (22). Exist- <br /> ment strategies dependent presence small molecules re-              ing biocontainment technologies usually depend single- <br /> quire active low concentrations, rendering            cellular mechanism. Such &#226;&#8364;&#339;uniplex&#226;&#8364;? approaches lack redundancy, <br /> inexpensive difficult detect. Histone genes controlled            leakiness toxin genes compromises fitness and <br /> by inducible promoter controlled 30 nM estradiol. The               normal behavior engineered microbes, reducing like- <br /> stability engineered genes separately regulated             lihood accepted researchers industrial <br /> expression site-specific recombinase. The combined frequency             biotechnologists. Also, ongoing selection toxin gene is <br /> of generating viable derivatives systems active            required. Reduced fitness undesirable features: (i) it <br /> below detection (&lt;10&#226;&#710;&#8217;10), consistent orthogonal nature             reduces usefulness models behavior natural <br /> and individual escape frequencies &lt;10&#226;&#710;&#8217;6. Evaluation es-             organism, (ii) reduce &#226;&#8364;&#339;performance&#226;&#8364;? industrial <br /> caper mutants suggests strategies reducing emergence.               applications, (iii) increase frequency escape <br /> Transcript profiling growth test suggest high fitness safe- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              GENETICS <br />                                                                               mutants (revertants able grow absence com- <br /> guarded strains, important characteristic wide acceptance.             pound). Finally, auxotrophic strains rely supplementation of <br />                                                                               corresponding nutrients micromolar concentrations, rendering <br /> biotechnology   | biosafety | Saccharomyces cerevisiae                        costly industrial scale-up. We developed <br />                                                                               multiplex safeguard switches regulated combination small <br />  <br /> T   rapid development biotechnology depends heavily on <br />     engineered microbes. In particular, booming field syn- <br />                                                                               molecules, active submicromolar concen- <br />                                                                               trations. In future, envision supplying blend such <br /> thetic biology (1) demonstrates feasibility novo syn-               compounds kind &#226;&#8364;&#339;special sauce&#226;&#8364;? designer mi- <br /> thesis viral genomes (2&#226;&#8364;&#8220;5), bacterial genomes (6, 7),                  crobe alive. This laced decoy compounds to <br /> eukaryotic chromosomes (8, 9). The technologies underpinning <br /> synthetic biology advanced DNA synthesis assembly (10),                   Significance <br /> genome editing (11, 12), computational assisted designs (13&#226;&#8364;&#8220; <br /> 15), commoditized increasingly                    The advance biotechnology opens greater possibilities <br /> available public. The advance synthetic biology prom-                   bioterror bioerror. Here, propose multiplexed safe- <br /> ises ultimately improve human living conditions                      guard switches rooted development foundational <br /> better understanding fundamental sciences multi-                  genomic, regulatory, metabolic technologies. Safeguard <br /> tude practical applications. However, biosafety mechanisms                     switches regulated submicromolar small molecule(s) <br /> should carefully considered minimize prevent dual use                    combined modular fashion. The resulting safeguard <br /> (16). Professionals chemically synthesized infectious virus                  strains high fitness low reversion rates. Moreover, <br /> in absence natural templates (2) reconstitute in-                      distinct classes safeguard switches orthogonal, <br /> fectious human retroviruses (3) 1918 &#226;&#8364;&#339;Spanish&#226;&#8364;? in-                        providing potential fail-safe mechanism. The safeguard <br /> fluenza virus (5). At time, relative amateurs trying                 technologies provide practical generic approach con- <br /> to engineer microbes do-it-yourself fashion (www.diybio.                     taining engineered microbes defined laboratory and/or <br /> org). Proactive measures warranted minimize bio-                      industrial environments, principle used the <br /> terror (e.g., anthrax attack United States 2000)                field well. <br /> &#226;&#8364;&#339;bioerror&#226;&#8364;? (accidental environment releases self-infection by <br />                                                                               Author contributions: Y.C. J.D.B. designed research; Y.C., N.A., W.J.C., A.U., K.C., and <br /> laboratory-adapted microbes case laboratory in-                H.H. performed research; Y.C., N.A., A.U., G.S., J.S.B., J.D.B. analyzed data; Y.C. <br /> fection individual hemochromatosis, victim               J.D.B. wrote paper. <br /> scientist&#226;&#8364;&#8482;s high iron levels caused hemochromatosis com-                   Reviewers included: J.J.C., Boston University. <br /> plemented natural iron requirement attenuated Yersinia                 The authors declare conflict interest. <br /> pestis) (17). Intrinsic biocontainment used prevent            Freely available online PNAS open access option. <br /> industrial espionage protecting intellectual property               1 <br />                                                                               To correspondence addressed. Email: jef.boeke@nyumc.org yizhi.cai@ <br /> biotechnology companies. In light wide accessibility                ed.ac.uk. <br /> whole-genome sequencing, envision future embedding                  This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> &#226;&#8364;&#339;decoy&#226;&#8364;? circuits safeguard strains combinatorial             1073/pnas.1424704112/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1424704112                                                     PNAS | February 10, 2015 | vol. 112 | no. 6 | 1803&#226;&#8364;&#8220;1808 <br />  defy biohackers. Each safeguard switch technology targets dif- <br /> ferent cellular process combined multiplex <br /> fashion. With safeguard switches, able regulate engi- <br /> neered microbes small molecules contain in <br /> defined laboratory conditions prevent escape environ- <br /> ment. The safeguard switch concept provides potentially <br /> effective way protect intellectual property biotechnology, by <br /> containing engineered strains defined combinations of <br /> small molecules. <br />    Here, report orthogonal safeguard switches, with <br /> one targeting transcriptional control based on <br /> recombination-induced lethality; cases, es- <br /> sential genes targeted, instead autotrophic markers and/or <br /> toxin genes minimize physiological impacts cell fitness. <br /> Transcriptome profiling growth tests confirm fitness of <br />                                                                        Fig. 1. The structure triplex histone switch. Duplex integrated safeguard <br /> several safeguard strains close wild type, perfor- <br />                                                                        based pair histone genes. Histone H3 gene HHTS regulated by <br /> mance multiple media types suggests robustness varying           galactose promoter pGAL1, histone H4 gene HHFS controlled <br /> conditions. The escape rates individual safeguards 10&#226;&#710;&#8217;8     galactose promoter pGAL7. Each histone switch flanked by <br /> 10&#226;&#710;&#8217;6. Analysis escape variants (revertants) sheds useful light   pair loxP sites. The duplex histone safeguard switches on <br /> guide design next-generation safeguard development.             CEN/ARS plasmid (A) integrated genomic locus (B). Either the <br />                                                                        native Gal4 protein engineered tribrid protein called GEV turns the <br /> Results                                                                transcription histone genes presence ligands, that <br /> Goals Overall Design Principles. To establish design principles    engineered yeast survives. The level redundancy use an <br /> for generic genomic safeguards, evaluated designs           orthogonal control mechanism, site-specific recombination. Any pair loxP <br />                                                                        sites (yellow diamonds) construct recombine delete or <br /> based premise desired combination high fit- <br />                                                                        histone genes Cre, activation, leading inviability. Both proteins <br /> ness, normal physiology, tight control small molecule         independently controlled small molecule(s). Hooked arrows, pro- <br /> could obtained putting essential genes multiple            moters; filled arrows, histone genes; blue lozenges, standardized vector <br /> forms chemical control. Here, that, putting         components; HOL, HOR, sequences Left Right HO gene, the <br /> the presence/absence gene transcription           safeguard integrated. <br /> chemical control, possible produce yeast strains live <br /> or die according presence/absence ligand, show <br /> high-fitness low-escape frequencies. Analysis escapers          favorable 10&#226;&#710;&#8217;7, observed significant fitness defect by <br /> (revertants) provides insight mechanisms escape       comparing colony size PCy230 control strain <br /> can used subsequent round engineering reduce            (Fig. S2). Subsequent transcriptome profiling indicated that <br /> those escape frequencies. Here, evaluated essential             large number genes dysregulated, possibly result of <br /> histone genes performance genome safeguards                 combined effects variations plasmid copy number and <br /> Saccharomyces cerevisiae various contexts. As       accumulation fraction cells lost plasmid <br /> histones studied encoded gene copies,           permissive conditions (Fig. S3). <br /> the studies performed strains native copies <br /> of histone H3 native copies histone H4 genes           Chromosomal Integration Histone Switches Restores Fitness. We <br /> been deleted genome (Fig. S1). We screened         hypothesized fitness decrease PCy230 reflect <br /> essential genes performance safeguards.                         variations histone abundance individual cells copy <br />                                                                        number centromeric plasmids fluctuate and <br /> Plasmid-Based Chemically Regulated Transcriptional Switches. As        copies. Alternatively, possible expression of <br /> a proof principle, constructed chemically regulated      histone genes rise aberrant histone ratios <br /> transcriptional switch based essential synthetic histone        differed native situation, deviations expres- <br /> genes called HHTS HHFS (encoding histone H3 H4,                sion dynamics galactose inducible native histone <br /> respectively; &#226;&#8364;&#339;S&#226;&#8364;? refers synthetic nature genes)     promoters. Integration histone-based safeguard the <br /> described in ref. 23. In safeguard strains described     genome ensure stable single copy. Thus, integrated <br /> here, HHTS HHFS genes regulated GAL1                histone safeguard (tagged URA3) HO locus <br /> GAL7 promoters, respectively, histone transcriptional         PCy230, spontaneous mitotic loss original <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> flanked pair loxP sites (see following sections    pPC012 plasmid (tagged LEU2) YP-galactose medium, <br /> detail). Thus, transcription genes regu-        constructed integrated safeguard strains PCy596&#226;&#8364;&#8220;599. These <br /> lated presence galactose glucose growth me-        integrated safeguard strains similar low escape rates as <br /> dium, related methods, presence/absence genes        PCy230 glucose (Table S1), fitness gene expression <br /> can regulated expression Cre recombinase activity.        profiles significantly improved determined measuring <br /> This histone-based triplex safeguard cloned pRS415          colony sizes (Fig. S4A), doubling time (Fig. S4B), transcriptome <br /> centromeric (episomal) vector (24) construct plasmid pPC012         profiling (Fig. S4C). <br /> (Fig. 1). <br />    To evaluate ability function genome safeguard,           Regulating Transcriptional Switches 30 nM Estradiol. For many <br /> generated yeast strain bearing construct, PCy230 [his3&#206;&#8221;200         applications, low ligand concentration, nonnative ligand <br /> leu2&#206;&#8221;1 lys2&#206;&#8221;202 trp1&#206;&#8221;63 ura3-52 hht1-hhf1&#206;&#8221;::natMX4 hht2-               unlikely alter cell physiology desirable. To demonstrate <br /> hhf2&#206;&#8221;::hygMX4 (pRS415-loxPWT-HHTS-loxPWT-HHFS-loxpWT);                 ability regulate cell viability using nonnative small mole- <br /> Materials Methods]. Strain PCy230 subsequently plated          cules nanomolar concentration, transformed second <br /> on permissive (galactose) restrictive (glucose) con-          plasmid pHCA/GAL4(1-93).ER.VP16 (henceforth called GEV) <br /> ditions, escape (reversion) rate calculated according      produces GAL4 DBNA binding domain&#226;&#8364;&#8220;estrogen receptor&#226;&#8364;&#8220; <br /> to Luria Delbruck fluctuation analysis using method         VP16 tribrid protein (GAL4DB.ER.VP16; GEV) (26), safe- <br /> the median (25) (Table S1). Although escape rate quite         guard strain PCy599 construct strain NAy236. This safeguard <br />  <br /> 1804 | www.pnas.org/cgi/doi/10.1073/pnas.1424704112                                                                                         Cai et al. <br />  strain regulated galactose, used millimolar <br /> concentrations, estradiol, small molecule nonnative <br /> to yeast able maintain viability submicromolar (30 nM) <br /> concentrations. The GEV localizes nucleus binds to <br /> Gal4p consensus sequences activate transcription tar- <br /> geted gene, induction &#206;&#178;-estradiol. The GEV protein is <br /> retained cytoplasm Hsp90 absence estradiol, <br /> resulting fairly tight control gene expression. We able <br /> to regulate histone safeguard strains (PCy230 PCy599) <br /> with 30 nM estradiol normal glucose-containing yeast growth <br /> medium, escape reversion frequency 10&#226;&#710;&#8217;7 10&#226;&#710;&#8217;9 <br /> (Fig. 2 Table S1). <br />  <br /> Escape Mutant Analysis: GEV-Regulated GAL-Histone Transcriptional <br /> Safeguards. We isolated independent escape mutants the <br /> GEV-regulated histone safeguard strains, grow glucose <br /> media absence estradiol. The GEV plasmids those <br /> five escapers recovered yeast Escherichia coli, <br /> transformed parental strain PCy599, tested their <br /> ability recapitulate &#226;&#8364;&#339;growth absence estradiol&#226;&#8364;? phe- <br /> notype (Fig. S5A), sequenced primers (Table <br /> S2). Interestingly, escape mutants seemingly grow <br />                                                                             Fig. 3. Escape mutant analysis. Both GEV Cre-EBD plasmids escape <br /> less parental strain presence estradiol,            mutants isolated analyzed. (A) Most plasmids bore in-frame <br /> suggesting selective pressure accu-             deletions hER region retained C-terminal VP16 activator se- <br /> mulation (Fig. S5A). All plasmids recapitulated               quence. One plasmid 4-bp insertion, protein predicted to <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>phenotype</span> mutations, large internal deletions             out-of-frame C-terminal tail 26 aa, 5 acidic residues, <br /> human estrogen receptor (hER) domain gene delete                sequence serve transcriptional activator. See Table S3. <br /> the estradiol-responsive domain remain in-frame                (B) The mutations Cre-EBD constructs missense <br /> downstream VP16 (Fig. 3A Table S3). Similarly, tested                mutations Cre coding region deletions (deletions out <br />                                                                             frame indicated stippled line out-of-frame segment) in <br /> that reintroducing fresh GEV plasmid restored estradiol de- <br />                                                                             Cre and/or EBD domains. See Table S3. <br /> pendence (Fig. S5B). <br />  <br />  <br />  <br />  <br />                                                                                                                                                              GENETICS <br /> Chemically Regulated Site-Specific Recombination Switches. In           killing effect complete expected. We hypothesized <br /> previous work synthetic yeast chromosomes (8, 9),                SCW11 promoter strength/expression <br /> adapted chemically regulated Cre (27) shown              properties, constructed set Cre-EBD plasmids <br /> effectively kills yeast semisynthetic genome containing              replacing SCW11 promoter various constitutive yeast <br /> numerous loxP sites flanking number essential genes. The               promoters (Materials Methods). After transforming set of <br /> specialized Cre protein ref. 27 tightly regulated              constructed Cre-EBD plasmids PCy230 screening on <br /> transcriptionally posttranslationally. A daughter-cell&#226;&#8364;&#8220;spe-             Gal-estradiol plates, pTDH3-Cre-EBD turned best <br /> cific promoter, pSCW11, ensures expression daughter             candidate application, effectively killed safe- <br /> cell state. Furthermore, Cre fused estrogen binding           guard strain induction remained inactive absence <br /> domain (EBD), unfolded absence estradiol,                estradiol. Upon estradiol induction, Cre protein localizes <br /> causing retained cytoplasm binding                nucleus excises histone transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>(s) <br /> Hsp90 chaperone. In following proof-of-principle experi-                residing loxP sites. The reversion rate this <br /> ments, induced Cre estradiol, leading excision            site-specific recombination-based safeguard systematically <br /> histone gene(s) loss viability. Ultimately, desir-        measured 10&#226;&#710;&#8217;6. <br /> able &#226;&#8364;&#339;reverse&#226;&#8364;? logic, small molecule maintains <br /> the viability cell.                                                  Escape Mutant Analysis: CRE-EBD Sensitivity GAL-Histone <br />    We transformed pSCW11-Cre-EBD plasmid safe-                     Recombinational Safeguards. We evaluated site-specific re- <br /> guard strain PCy230 construct PCy251, induced killing             combination regulation original GAL-histone safeguard <br /> Gal plates (galactose required maintain histone gene ex-              presence CRE-EBD plasmid studying escape <br /> pression) 1 &#206;&#188;M estradiol, intended delete histone               mutants. We isolated 21 independent escape mutants the <br /> genes, despite expression. However, context,              safeguard strains grew restrictive condition Cre <br />                                                                             expression, case galactose containing 1 &#206;&#188;M es- <br />                                                                             tradiol. The Cre-EBD plasmids recovered retrans- <br />                                                                             formed parent strain PCy599, resulting strains <br />                                                                             insensitive estradiol induction, confirming <br />                                                                             responsible <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span> resided plasmid. The Cre- <br />                                                                             EBD mutant plasmids sequenced identify mutations <br />                                                                             leading phenotype (Fig. 3B Table S3), ob- <br />                                                                             served wide variety mutations predicted lead loss of <br />                                                                             function. Of 21 plasmids, 2 insertions (of 3 8 bp), 7 <br /> Fig. 2. Regulating histone-based safeguard switches 30 nM estradiol. <br /> The GEV-regulated histone safeguard strains grow permissive <br />                                                                             missense nonsense mutations Cre coding region, 5 <br /> medium die restrictive conditions. The histone strain PCy599      various deletions Cre and/or EBD regions, finally <br /> with GEV plasmid grows galactose medium estradiol-         7 complete lost Cre-EBD insert, instead containing <br /> supplemented glucose medium, survive glucose medium           single loxP site loxP sites flanking histone ex- <br /> without estradiol.                                                          pression cassette (unlike parental plasmid, none). <br />  <br /> Cai et al.                                                                                        PNAS | February 10, 2015 | vol. 112 | no. 6 | 1805 <br />  The category mutant explained assuming <br /> there copies histone plasmid. We propose that <br /> homologous recombination copy safeguard <br /> plasmid (based pRS415-LEU2) pTdh-Cre-EBD <br /> plasmid (based largely identical pRS413-HIS3) occurred, <br /> producing Cre-EBD-LEU2 plasmid histone HIS3 plas- <br /> mid. Upon Cre-EBD induction, &#226;&#8364;&#339;empty&#226;&#8364;? HIS3 plasmid con- <br /> taining just loxP site produced high frequency. <br /> Subsequent mitotic loss Cre-EBD plasmid, frequent <br /> event, produce observed configurations. <br />  <br /> Combined Escape Transcriptional Recombinational Safeguard <br /> Switches Not Observed. We performed combined escape ex- <br /> periment plating yeast strain PCy599 carrying TDH3-Cre- <br /> EBD growing cells permissively, plating nearly 1010 <br /> cells restrictive medium (glucose plus estradiol). Even after <br /> replica plating (i.e., rounds growth restrictive medium),      Fig. 4. Construction multiplex integrated safeguard strains. A Golden <br /> no escaper colonies observed. We estimate cells went       Gate assembly-based method developed construct various combina- <br /> through doubling hitting selective plates, pos-   tions safeguards. A selected essential gene [your favorite essential gene <br />                                                                          (YFEG)] quickly assembled given promoter 3&#226;&#8364;&#178;-UTR into <br /> sibly more. Thus, conclude reversion frequency was <br />                                                                          acceptor vector [A; red colonies vector reassembly (they <br /> &lt;10&#226;&#710;&#8217;10, consistent predicted frequency 10&#226;&#710;&#8217;12.          express RFP) white colonies right assemblies] integrate into <br />                                                                          corresponding yeast mutant (B) test behavior safeguard <br /> Construction Additional Safeguard Switches. To screen ad-         strain permissive restrictive conditions (C). <br /> ditional essential genes suitable candidate safeguards, used <br /> a variant Golden Gate DNA assembly method (28) to <br /> efficiently design construct systematic set safeguard           bioeconomy producing wide variety chemicals fuels, <br /> switches (Fig. 4). A library parts (promoters, genes 3&#226;&#8364;&#178;-          ease constructing more complex bi- <br /> UTRs), set acceptor vectors, constructed         ological systems needed accomplish grows. Containment <br /> sequence verified. Using Golden Gate reaction, assembled            critical field application engineered organisms <br /> 17 essential genes different galactose promoters             extremely desirable work pathogenic <br /> (pGAL1, pGAL7, pGAL10), respectively.                                organisms, perspectives investigator safety pre- <br />    These tested individually corresponding hetero-           vention accidental intentional release harmful organisms <br /> zygous diploid yeast deletion mutant. Diploids transformed          laboratory environment. Finally, biotechnology com- <br /> the appropriate test plasmid sporulated, tetrads           panies make large financial investments building specific <br /> dissected permissive agar plates, resulting haploid spore         strains bacteria yeast large-scale fermentation. Pre- <br /> clones replica plated restrictive plates identify po-       venting theft organisms challenging problem, but <br /> tential safeguard strains. We identified essential genes,          genomic safeguards described po- <br /> namely SUI1, HSP10, RPC11, preferred gene candidates              tential minimize risk intellectual property. <br /> by method. The selection criteria follows: safe-           In order safeguards practically useful, require <br /> guard strain (i) grow permissive medium and <br />                                                                          engineered features: redundancy, robustness, low escape <br /> (ii) low escape rate restrictive conditions. <br />                                                                          frequencies, low cost. The likelihood widespread <br /> Transcriptome profiling carried constructed <br />                                                                          adoption experimentalists industrial biotechnologists <br /> safeguard strains identify potential fitness defects (Fig. 5). The <br /> GEV plasmid CreEBD plasmid separately transformed               alike requires strong evidence safeguard itself does <br /> into RPC11 safeguard strain, construct estradiol-             affect cellular metabolism phenotype. <br /> regulated transcriptional switch safeguard strain site-specific <br />                                                                          Redundancy. We different forms redundancy. <br /> recombination-regulated safeguard strains, respectively. <br />                                                                          The use essential genes just one. The <br /> Reducing Leakiness Safeguard Switches. We discovered         second use modes dependency, tran- <br /> non&#226;&#8364;&#8220;histone-based GEV-regulated safeguard strains grew             scriptional other, presence/absence gene, i.e., <br /> restrictive conditions hypothesized resulted          recombinational. In case, distinct small molecules, <br /> leakiness galactose promoters (pGAL1/pGAL7/pGAL10)                galactose estradiol, used control switch. <br /> in context, lower amounts protein products <br /> might required viability histone genes. In previous      Robustness. Cells environment face wide variety of <br /> work (31), various numbers (2&#226;&#8364;&#8220;10) Gal4 binding sites             conditions. It useful know safeguard has <br /> fused SPO13 promoter, contains repressive up-             little effect growth presence ligand also <br /> stream regulatory sequence (URS) sequence, generate set          growth absence ligand similarly absent <br /> tightly regulated galactose-dependent promoters called SPAL              multiple conditions. With histone safeguard, we <br /> promoters (SPO/GAL). We retrieved set promoters             showed performance excellent minimal rich <br /> strains containing promoters (Materials Methods)            medium conditions, environments routinely used the <br /> use construct second generation safeguard strains           laboratory. We observed histone safeguard the <br /> based RPC11. The resulting safeguard strains ap-              GEV function wide range starting 5 nM <br /> propriate growth phenotypes compared with native              estradiol concentration, positive least <br /> galactose promoters (Fig. S6).                                           100 nM. This advantageous complex biological environ- <br />                                                                          ment bioreactor fermenter, levels exog- <br /> Discussion                                                               enous molecules homogenously distributed in <br /> Engineering containment anticipated critical          well-controlled laboratory environment. It interesting <br /> ever world transitions petroleum economy              expand studies wider ranges conditions. <br />  <br /> 1806 | www.pnas.org/cgi/doi/10.1073/pnas.1424704112                                                                                         Cai et al. <br />                                                                                       One key limitation safeguard design construction the <br />                                                                                   paucity well-regulated transcriptional regulatory systems <br />                                                                                   controlled low-cost nontoxic small molecules work <br />                                                                                   prokaryotic eukaryotic systems generically. Besides <br />                                                                                   estradiol, low-cost natural synthetic hor- <br />                                                                                   mone hormone binding domains proven to <br />                                                                                   regulate protein activity various systems including budding <br />                                                                                   yeast, androgen (29), glucocorticoid (30), pro- <br />                                                                                   gesterone/RU486 (31). Once carefully characterized, small <br />                                                                                   molecular/binding domain pairs potentially used build <br />                                                                                   orthogonal safeguard switches future. Also, prokaryotic <br />                                                                                   strains engineered parallel, using same <br />                                                                                   design philosophies (32, 33). We anticipate improve- <br />                                                                                   ments safeguards coming years to <br />                                                                                   anticipate explosion recombinant organisms study in <br />                                                                                   basic industrial research laboratories, resulting increased <br />                                                                                   focus safety mechanisms integral systems study. <br /> Fig. 5. Transcriptome profiling safeguard strains. Transcriptome profiling <br /> of various safeguard strains. The graph organized gene/promoter pairs. <br />                                                                                   Materials Methods <br /> The blue dots volcano plots represent statistically significant dysregu- <br /> lated genes (see Table S4 lists genes affected). The transcriptome pro-    Strains, Plasmids, Oligonucleotides. Yeast strains plasmids con- <br /> filing shows limited transcriptome changes safeguard strains compared      tained listed Table S5. Oligonucleotides used listed Table S2. <br /> with wild type, GAL10-RPC11 showing best performance. <br />                                                                                   Plasmid Shuffling. Plasmid pPC012 transformed yeast strain, JDY6, <br />                                                                                   genomic copies histones H3 H4 genes been <br /> Escape Frequencies. Because safeguard described pro-                    previously deleted (23) viability maintained plasmid pDM9, plasmid <br /> duces escapers frequency million                     carrying URA3 marker wild-type histone H3 H4 genes HHT1 and <br /> some cases significantly less, anticipated duplexing                HHF1. After introducing pPC012, centromeric (CEN) LEU2 plasmid, the <br /> very favorable escape rates 10&#226;&#710;&#8217;12 lower obtained.                  strain LioAc/SS/PEG transformation (34), plasmid shuffling 5-fluoro- <br />                                                                                   orotic acid [5-Foa (35)], 2% (wt/vol) galactose medium successful, indi- <br /> In experiment evaluate combined transcriptional and <br />                                                                                   cating GAL promoters successfully express histone <br /> recombinational switches, failed identify escapers, consis-                 genes. More importantly, 5-FoaR strains unable grow glu- <br /> tent prediction. Interestingly, observed                    cose, consistent effective shutoff histone gene expression. <br /> revertants grow parental presence the <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     GENETICS <br /> ligand, suggesting persistence population mini-                  Golden Gate DNA Assembly. The Golden Gate assembly protocol described <br /> mized. Similarly, triplex higher-plex safeguards pro-                   ref. 28. Specifically, 15-&#206;&#188;L Golden Gate reaction containing 1.5 &#206;&#188;L 10&#195;&#8212; <br /> duce increasingly stringent levels control.                                    T4 DNA ligase reaction buffer (New England Biolabs), 0.15 &#206;&#188;L 100&#195;&#8212; BSA <br />                                                                                   (New England Biolabs), 1 &#206;&#188;L 600 units/&#206;&#188;L T4 DNA ligase (Enzymatics), 10 &#206;&#188;L <br /> Cost. The key low cost identify ligands                    H2O, 1 &#206;&#188;L acceptor vector DNA (&#226;&#710;&#188;100 ng/&#206;&#188;L), 0.5 &#206;&#188;L insert DNA <br /> relatively inexpensive gram operate low con-                      (&#226;&#710;&#188;100 ng/&#206;&#188;L). The following temperature cycles used: 1 h 37 &#194;&#176;C, <br /> centrations, preferably micromolar nanomolar. Our                         5 min 50 &#194;&#176;C, 5 min 80 &#194;&#176;C followed incubation 4 &#194;&#176;C. One microliter <br /> work demonstrates estradiol-based GEV-based safe-                        finished Golden Gate reaction transformed 100 &#206;&#188;L of <br /> guard operates concentration 30 nM escape rate                       Top10/DH5&#206;&#177; chemical-competent cells. <br /> characteristics. Because estradiol commodity be <br /> purchased cheaply, cost treating 1,000-L fermentor                    Plasmid Recovery Yeast. Plasmid recovery yeast carried out <br />                                                                                   using Zymoprep yeast plasmid miniprep kit (Zymo Research) following the <br /> estimated US$0.13. <br />                                                                                   manufacturer&#226;&#8364;&#8482;s instructions. <br /> Minimal Phenotype Effects. We evaluated impact safe- <br />                                                                                   Cloning Sequencing SPALX Promoters. Twelve yeast MAV strains (36) <br /> guards fitness examining colony size. We hypothesize that <br />                                                                                   containing various numbers Gal4 binding sites used template to <br /> the use essential genes, toxins, control cell                   clone SPAL promoters varying numbers Gal4 binding sites. Ge- <br /> growth inherently favorable overall growth properties                  nomic DNAs yeast strains extracted using phenol/chloroform <br /> and lack perturbation cellular systems. Indeed, histone                 isolation method. Two rounds PCR used amplify SPALX pro- <br /> switch, integrated single copy, surprisingly fit. As                moters genomic DNA: first, F primer PC_oligo379 Ura3 R primer <br /> first evaluation impact safeguards built, ex-                 PC_oligo380 used; PCR product diluted 1,000-fold. Pri- <br /> amined transcriptome effects essential genes used                 mers PC_oligo401 PC_oligo324 used amplify SPAL promoter <br /> in safeguards ligands commonly used                       fragments appropriate Golden Gate overhangs. The PCR products were <br /> &#226;&#8364;&#339;on&#226;&#8364;? state. Although systems use perfect                  cloned using Zero Blunt Topo kit (Life Technologies). The resulting plas- <br /> effects observed gene expression, extent                     mids sequenced identify exact number Gal4 binding sites. <br /> perturbations varied significantly different essential genes               These promoters used subsequent assembly safeguard switches to <br /> and small molecules. This helps prioritization                   identify tighter constructs. <br /> promising designs. <br />                                                                                   &#206;&#178;-Estradiol Induction Cre Expression. Safeguard strains grown on <br />    Escaper (revertant) analyses suggest means reduce escape <br />                                                                                   appropriate selective growth medium, plated spotted on <br /> frequencies. For Cre-EBD GEV constructs, escapers <br />                                                                                   appropriate selective solid medium containing 1 &#206;&#188;M &#206;&#178;-estradiol, in- <br /> affecting function proteins isolated. For                   cubated 30 &#194;&#176;C 2&#226;&#8364;&#8220;3 d. <br /> Cre-EBD, mutations isolated loss-of-function muta- <br /> tions. This frequency greatly lowered simultaneously                  &#206;&#178;-Estradiol Induction GEV System. Safeguard strains grown in <br /> expressing second recoded copy gene,                        appropriate glucose selective growth medium containing 30 nM &#206;&#178;-estradiol, <br /> mutations type recessive. The GEV mutants                       plated spotted appropriate glucose selective solid medium <br /> could similarly reduced frequency incorporating                      containing 30 nM &#206;&#178;-estradiol (or restrictive plates lacking estradiol), and <br /> ligand-binding domains protein.                                     incubated 30 &#194;&#176;C 2&#226;&#8364;&#8220;3 d. <br />  <br />  <br /> Cai et al.                                                                                              PNAS | February 10, 2015 | vol. 112 | no. 6 | 1807 <br />  Reversion Rate Measurement Colony Picking Sequencing. Escape rates                     histone, RPC11, SUI1, HSP10 safeguards. mRNA sequenced using an <br /> were calculated using method median (25). For measurement rates                     Illumina HiSeq standard TruSeq preparation kits. For strain, we <br /> in 10&#226;&#710;&#8217;5 10&#226;&#710;&#8217;9 range, 5&#226;&#8364;&#8220;12 independent cultures (each grown                          obtained &#226;&#710;&#188;12 million 50-bp single-end reads. Reads mapped using <br /> a single parent colony) inoculated 20 mL SC&#226;&#8364;&#8220;His supplemented                      TOPHAT (37) reference S. cerevisiae genome (S288c). Approximately <br /> with 2% galactose liquid cultures. In total, 108 107 cells plated                  95% reads mapped. For gene, read counts computed <br /> on restrictive medium. Viable titer determined plating 100 &#206;&#188;L a <br />                                                                                                using HTSEQ (38) analyzed differential expression using DESEQ (39), <br /> five serial 10-fold dilution permissive medium. The reversion fre- <br />                                                                                                standard parameters following no-replicates scenario. For each <br /> quency obtained dividing colony-forming units restrictive plates <br /> by colony-forming units permissive plates. The median reversion fre-                        gene, obtained raw P value adjusted P value using standard <br /> quency used calculate rate using method median.                     Benjamini&#226;&#8364;&#8220;Hochberg procedure (40). We used 1% false-discovery rate <br />    For safeguard strain measured, &#226;&#8364;&#339;escaper&#226;&#8364;? revertant col-                     (adjusted P value &#226;&#8240;&#164; 0.01) identify genes significantly differentially <br /> onies picked independently grown cultures grown                            expressed. Differentially expressed genes reported Table S4. <br /> 10 mL permissive liquid culture (with plasmid selection applicable), for <br /> 48 h 30 &#194;&#176;C. Tenfold serial dilutions plated restrictive per-                    ACKNOWLEDGMENTS. We thank Qiuhui Lin Bin Jia help Cre-EBD <br /> missive agar plates, incubated 30 &#194;&#176;C 2&#226;&#8364;&#8220;3 d single colonies                    escaper analysis experiments helpful discussions. Y.C., N.A., W.J.C., G.S., <br /> appeared. One colony chosen culture assure independence.                            K.C., H.H., J.S.B., J.D.B. supported Defense Advanced <br />                                                                                                Research Project Agency [N66001-12-C4020 (to J.D.B.)]. Y.C. A.U. were <br />                                                                                                supported Chancellor&#226;&#8364;&#8482;s Fellowship University Edinburgh, <br /> RNASeq. A single colony picked grown 10 mL permissive                         start-up fund Scottish Universities Life Sciences Alliance, Biotech- <br /> liquid medium, incubated 30 &#194;&#176;C A600 0.8                       nology Biological Sciences Research Council Grant BB/M005690/1 <br /> 1.2, RNA isolated described using Qiagen RNAEasy kit using                    (to Y.C.). The open access charge supported Research Councils UK Open <br /> manufacturer&#226;&#8364;&#8482;s protocol. We performed RNASeq strains integrating                        Access Fund. <br />  <br />  <br />  1. Way JC, Collins JJ, Keasling JD, Silver PA (2014) Integrating biological redesign: Where   22. Wright O, Delmans M, Stan G-B, Ellis T (2014) GeneGuard: A modular plasmid system <br />     synthetic biology came needs go. Cell 157(1):151&#226;&#8364;&#8220;161.                     designed biosafety. ACS Synth Biol, 10.1021/sb500234s. <br /> </body></html>