<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 15, 3811&#226;&#8364;&#8220;3828, August 2004 <br />  <br />  <br /> Characterization Saccharomyces cerevisiae Fol1 <br /> Protein: Starvation C1 Carrier Induces <br /> Pseudohyphal Growth <br /> Ulrich Gu&#194;&#168;ldener,*&#226;&#8364;&#160;&#226;&#8364;&#161; Gabriele J. Koehler,*&#226;&#8364;&#160; Christoph Haussmann,&#194;&#167; <br /> Adelbert Bacher,&#194;&#167; Jo&#194;&#168;rn Kricke,&#229;&#8218;&#168; Dietmar Becher,&#194;&#182; and <br /> Johannes H. Hegemann*# <br />  <br /> *Heinrich-Heine-Universita&#194;&#168;t, Funktionelle Genomforschung der Mikroorganismen, 40225 Du&#194;&#168;sseldorf, <br /> Germany; &#194;&#167;Technische Universita&#194;&#168;t Mu&#194;&#168;nchen, Institut fu&#194;&#168;r Organische Chemie und Biochemie, 85747 <br /> Garching, Germany; &#229;&#8218;&#168;AG Anatomie und Zellbiologie der Charite, 13353 Berlin, Germany; and <br /> &#194;&#182; <br />  MICROMUN Private Institute Microbiological Research GmbH, Biotechnikum Greifswald, 17489 <br /> Greifswald, Germany <br />  <br /> Submitted September 18, 2003; Revised May 10, 2004; Accepted May 16, 2004 <br /> Monitoring Editor: Douglas Koshland <br />  <br />  <br />  <br /> Tetrahydrofolate (vitamin B9) folate derivatives essential cofactors one-carbon (C1) transfer reactions and <br /> absolutely required synthesis variety different compounds including methionine purines. Most plants, <br /> microbial eukaryotes, prokaryotes synthesize folate novo. We characterized important enzyme this <br /> pathway, Saccharomyces cerevisiae FOL1 gene. Expression budding yeast gene FOL1 Escherichia coli <br /> identified folate biosynthetic enzyme activities dihydroneopterin aldolase (DHNA), 7,8-dihydro-6-hydroxymethylp- <br /> terin-pyrophosphokinase (HPPK), dihydropteroate synthase (DHPS). All enzyme activities detected <br /> in wild-type yeast strains, fol1&#226;&#338;&#172; deletion strains showed background activities, demonstrating that <br /> Fol1p catalyzes sequential steps tetrahydrofolate biosynthetic pathway central enzyme this <br /> pathway, starting GTP consists seven enzymatic reactions total. Fol1p exclusively localized <span id='am-10' about='protege:TO' typeof='owl:Thing'>to</span> <br /> mitochondria shown fluorescence microscopy immune electronmicroscopy. FOL1 essential gene the <br /> nongrowth <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> fol1 deletion leads recessive auxotrophy folinic acid (5&#226;&#180;&#8226;-formyltetrahydrofolate). <br /> Growth fol1&#226;&#338;&#172; deletion strain folinic acid&#226;&#8364;&#8220;supplemented rich media induced dimorphic switch haploid <br /> invasive filamentous pseudohyphal growth presence glucose ammonium, known suppressors <br /> of filamentous invasive growth. The invasive growth phenotype induced depletion C1 carrier dependent <br /> on transcription factor Ste12p flocullin/adhesin Flo11p, filamentation phenotype independent <br /> of Ste12p, Tec1p, Phd1p, Flo11p, suggesting signaling pathways adhesion proteins. <br />  <br />  <br /> INTRODUCTION                                                         organisms, cells use carrier-mediated active trans- <br />                                                                      port uptake folate. Folate deficiency causes <br /> Reduced folate supplies cellular one-carbon pool            massive incorporation uracil DNA chromo- <br /> C1 carrier units, act enzyme cofactors one-              breaks repair uracil-DNA glycosylase <br /> carbon transfer reactions essential synthesis pu-         apyrimidinic endonuclease (Blount et al., 1997). Diets <br /> rines, thymidylate, glycine, methionine, pantothenic acid            low intake folate associated increased risk <br /> and N-formylmethionyl-tRNA organisms (Blakley             developing cancer (Ames et al., 1995; Duthie, 1999). In <br /> Benkovic, 1984; Appling, 1991). In eukaryotes cytoplas-          addition, shown 70% neural tube <br /> mic mitochondrial compartments possess paral-             defects prevented folic acid supplementation in <br /> lel array enzymes catalyzing interconversion folate        early pregnancy (Greene Copp, 1997). Most microbial <br /> coenzymes, differ oxidation state one-         cells plants synthesize folates novo because <br /> carbon units (Appling, 1991).                                        lack carrier-mediated active transport of <br />    Mammalian cells unable carry novo synthe- <br />                                                                      mammalian cells. Therefore early folate biosynthetic en- <br /> sis folate. Therefore folate considered vitamin these <br />                                                                      zymes attractive chemotherapeutic targets treat- <br />                                                                      ment microbial diseases (Roland et al., 1979). <br />  Article published online ahead print. Mol. Biol. Cell 10.1091/      The molecular biochemical characterization folate <br /> mbc.E03&#226;&#8364;&#8220; 09 &#226;&#8364;&#8220; 0680. Article publication date available    biosynthetic pathways enzymes involved been <br /> www.molbiolcell.org/cgi/doi/10.1091/mbc.E03&#226;&#8364;&#8220; 09 &#226;&#8364;&#8220; 0680.              limited small number species, particular bacteria <br />  &#226;&#8364;&#160; <br />    These authors contributed equally work                    plants (Talarico et al., 1992; Volpe et al., 1993; Cossins <br />  &#226;&#8364;&#161; <br />    Present address: Institute Bioinformatics, GSF-National Re- <br />    search Center Environment Health, 85764 Neuherberg,       Chen, 1997; review Rebeille, 1999). In most <br />    Germany.                                                          bacteria single enzymatic activities performed by <br />  # <br />    Corresponding author. E-mail address: hegemann@uni-               monofunctional proteins, eukaryotes enzymatic <br />    duesseldorf.de.                                                   steps carried bi- trifunctional proteins. In <br />  <br />  <br /> &#194;&#169; 2004 The American Society Cell Biology                                                                               3811 <br />  U. Gueldener et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Schematic representation biosynthetic pathway tetrahydrofolate. The successive reactions shown catalyzed by <br /> the following enzymes: GTPCH, GTP-cyclohydrolase; DHNA, dihydroneopterin aldolase; HPPK, 7,8-dihydro-6-hydroxymethylpterin- <br /> pyrophosphokinase; DHPS, 7,8-dihydropteroate synthase; DHFS, dihydrofolate synthase (folylpolyglutamate synthase); DHFR, dihydrofo- <br /> late reductase. In S. cerevisiae GTPCH activity encoded FOL2 (Nardese et al., 1996). The phosphatase(s) (Pase) presumably <br /> removes phosphate residues 7,8-dihydroneopterin triphosphate encoded PHO8 (Kaneko et al., 1987). FOL1 described in <br /> this work encodes enzymatic activities. The DHFS activity brought FOL3 gene (Cherest et al., 2000), whereas <br /> the DHFR activity encoded DFR1 (Lagosky et al., 1987). Tetrahydrofolate supplies one-carbon pool C1 carrier units, act <br /> as enzyme cofactors one-carbon transfer reactions involved various biosynthetic activities (Blakley Benkovic, 1984). <br />  <br />  <br /> Saccharomyces cerevisiae pathway leading tetrahydro-            alkaline phosphatase identified S. cerevisiae far <br /> folate (THF) far elucidated completely (Fig-           (Kaneko et al., 1987), probably capable catalyzing this <br /> ure 1). THF synthesis starts guanosine triphosphate,              reaction yielding 7,8 dihydroneopterin. We this <br /> which transformed 7,8-dihydroneopterin triphospate               work enzymes needed (dihydroneopterin <br /> by GTP-cyclohydrolase I encoded FOL2 (Nardese et                aldolase [DHNA], 7,8-dihydro-6-hydroxymethylpterin-py- <br /> al., 1996). The phosphatase(s) presumably removes             rophosphokinase [HPPK], dihydropteroate synthase <br /> phosphate residues identified yet. PHO8,              [DHPS]) convert 7,8 dihydroneopterin 7,8-dihydrop- <br />  <br />  <br /> 3812                                                                                                         Molecular Biology Cell <br />                                                                                                  Yeast C1 Carrier Filamentous Growth <br />  <br />  <br /> teroate carried single protein, Fol1           variety signals contribute initiate dim- <br /> protein. Subsequently, dihydrofolate synthase (DHFS) en-            porphic switch (Breitkreutz et al., 2003). <br /> coded FOL3 adds glutamate 7,8-dihydrop-                In report FOL1 gene encodes three <br /> teroate, yielding 7,8-dihydrofolate (DHF; Cherest et al.,           subsequent enzymatic activities biosynthesis folic <br /> 2000). Finally, DHF reduced dihydrofolate reductase           acid, DHNA, HPPK, DHPS. The trifunctional <br /> (DHFR) encoded DFR1 gene yield tetrahydrofolate           Fol1 protein expressed low level localized <br /> (THF; Barclay et al., 1988; Figure 1). Although genes           mitochondrial membranes. Deletion FOL1 leads a <br /> encoding enzymes THF biosynthetic pathway                nongrowth phenotype suppressed addi- <br /> nuclear, unclear subcellular compartment(s)         tion folinic acid media. Under conditions <br /> THF produced. In yeast biochemical analysis         haploid fol1&#226;&#338;&#172; strains exhibited strong invasive growth on <br /> revealed dihydrofolate reductase (DHFR), cata-           rich media containing glucose ammonium compounds <br /> lyzes step THF synthesis present         known suppress invasive growth. Likewise haploid and <br /> cytoplasm mitochondria (Zelikson Luzzati, 1977).            diploid fol1&#226;&#338;&#172; strains showed switch filamentous <br />    In fungi <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deprivation changes             pseudohyphal growth media, indepen- <br /> environment result distinct morphological differentia-         dent transcription factors Ste12p, Tec1p, Phd1p, and <br /> tion. This dimorphic switch filamentous growth                cell wall protein Flo11p. Thus results indicate that <br /> thought adaptive nonmotile microorganisms be-             loss C1 carrier activity mediated folate triggers a <br /> cause facilitates foraging environment           signal pseudohyphae development rich <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> <br /> scarce nutrients. This dimorphic switch similar mor-          supply conditions. <br /> phological transitions observed fungal pathogens as <br /> Candida albicans Ustilago maydis (for reviews Len-          MATERIALS AND METHODS <br /> geler et al., 2000; Sanchez-Martinez Perez-Martin, 2001). <br /> When starved nitrogen diploid strains S. cerevisiae          Media <br /> undergo developmental transition single yeast cell         The YP media plus 2% glucose (YPD) 2% galactose (YPG) synthetic <br />                                                                     media plus 2% glucose (SD) 2% galactose (SG) prepared described <br /> form filaments elongated pseudohyphal cells (for re-          (Sherman, 1991). All YP media supplemented adenine (4 mg/l) and <br /> views Lengeler et al., 2000; Palecek et al., 2002).             tryptophan (20 mg/l). G418 (geneticin, final concentration 200 &#226;?&#174;g/ml, Cal- <br /> Pseudohyphal cells exhibit altered morphology bud-           biochem, Novabiochem GmbH, Bad Soden, Germany), phleomycin (final <br /> ding <span id='am-90' about='xsp:pattern' typeof='owl:Thing'>pattern</span> invade growth substrate (Gimeno            concentration 7.5 &#226;?&#174;g/ml, Cayla, Toulouse, France), 5&#226;&#172;&#732;fluoro orotic acid (5- <br />                                                                     FOA, final concentration 1000 &#226;?&#174;g/ml, PCR) folinic acid (final concentra- <br /> et al., 1992; Kron et al., 1994). In haploid S. cerevisiae cells    tion 50 &#226;&#8364;&#8220;1000 &#226;?&#174;g/ml, F-7878, Sigma, Taufkirchen, Germany) added as <br /> <span id='am-8' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation, specifically loss fermentable sugars        required. <br /> such glucose causes related process called haploid in- <br /> vasive growth (Roberts Fink, 1994; Cullen Sprague,          Yeast Strains <br /> 2000). Presence short-chain alcohols (fusel oils; e.g.,          All yeast strains used study derivatives CEN.PK2 CGX31 <br />                                                                     (genetic background: &#226;&#338;&#186;1278b) indicated Table 1 (Gimeno et al., 1992; <br /> isoamyl isobutyl alcohols) liquid solid media        Entian Ko&#194;&#168; tter, 1998). Transformation yeast cells, sporulation, and <br /> also induce pseudohyphal-like growth haploid cells               tetrad analysis carried described (Sherman Hicks, 1991). <br /> (Dickinson, 1996), indicating yeast sense <br /> <span id='am-9' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation metabolic byproducts regulate       Databank Searches <br /> differentiation (Lorenz et al., 2000a). Starch-degrading hap-       The NCBI&#226;&#8364;&#8482;s protein database searched using BLASTP (Altschul et al., <br /> loid diploid S. cerevisiae cells able switch   1997). Sequence alignments performed using CLUSTAL W (Thompson <br />                                                                     et al., 1994). Pfam domains calculated according Bateman et al. (2002). <br /> a yeast cell form pseudohyphal forms response              Isolated Pfam domains subjected FASTA analysis (Pearson, 1994). <br /> carbon limitation (Lambrechts et al., 1996). <br />    Although little known sensors effectors      Gene Deletions <br /> pseudohyphal differentiation, considerably insight         ORF YNL256w (FOL1) deleted strain CEN.PK2 resulting strain <br /> been gained signal transduction pathways. The             CEN.HE4 using PCR-based disruption strategy. Briefly deletion cas- <br /> two best characterized pathways mating Cdc42p/              sette created template plasmid pFA6-kanMX4 using oligonucleotides <br />                                                                     278 279 (Table 2) transformed CEN.PK2 (Wach et al., 1994). The <br /> Ste20p/Ste12p/Tec1p MAPK cascade, including                 N terminal 357 nucleotides (119 amino acids) FOL1 deleted. The <br /> unique filamentation specific components, recep-            correct deletion verified Southern blot analysis (unpublished data). <br /> tor/G protein/cAMP-PKA/Flo8p signaling pathway (for re-                All gene deletions diploid yeast strain CGX31 using a <br /> views Borges-Walmsley Walmsley, 2000; Lengeler et           PCR-generated deletion cassette carrying loxP-kanMX-loxP marker created <br />                                                                     template plasmid pUG6 (Gu&#194;&#168; ldener et al., 1996). Deletion ORF YNL256w <br /> al., 2000; D&#226;&#8364;&#8482;Souza Heitman, 2001; Palecek et al., 2002).        verified PCR using genomic primers 1068 1069 combination <br /> Other transcription factors known regulate filamentous           kanR-specific primers 363 335. Where appropriate kanMX <br /> growth include Sok2p, Phd1p, Ash1p evi-            marker resulting diploid heterozygous deletion strain rescued <br />                                                                     using Cre recombinase described (Gu&#194;&#168; ldener et al., 1996). Next the <br /> dence Phd1p act pathway distinct           FOL1/fol1&#226;&#338;&#172;::loxP strain, STE12 gene deleted using kanMX deletion <br /> cAMP MAP kinase pathways (for review Pan et al.,            cassette generated using oligonucleotides 1052 1053. PCR verification was <br /> 2000). Few targets signal transduction pathways        using STE12-specific genomic primers 638 1055 combination <br /> known, FLO11 gene encoding im-              363 335. Likewise FOL1/fol1&#226;&#338;&#172;::loxP strain FLO11 gene was <br />                                                                     deleted using oligonucleotides 1056 1057. PCR verification done <br /> portant flocculin relevant cell-cell adhesion fila-      using FLO11-specific genomic primers 1058 1061 combination with <br /> mentous growth haploids diploids (Rupp et al., 1999;         363 335. The PHD1 TEC1 genes individually deleted the <br /> Tamaki et al., 2000). The FLO11 gene expression highly           FOL1/fol1&#226;&#338;&#172;::loxP-kanMX-loxP strain using ble deletion cassette created on <br /> regulated transcription factors;              template plasmid pUG66 using oligonucleotides 1425 1426 (PHD1) and <br />                                                                     oligonucleotides 1427 1428 (TEC1; Gueldener et al., 2002). PCR verifica- <br /> Ste12p Tec1p regulate transcription MAPK               tion using PHD1-specific genomic primers 1431 1432 or <br /> cascade, Flo8p regulates Flo11p expression down-            TEC1-specific genomic primers 1429 1430 combination 363 <br /> stream PKA (Lo Dranginis, 1998; Madhani Fink,            335. <br />                                                                        Tetrad dissection resulting diploid heterozygous deletion strains was <br /> 1998a; Rupp et al., 1999). New transcriptional profiling data       YPD &#226;&#171;&#185; 1 mg/ml folinic acid, mating type determined <br /> indicate invasive growth controlled solely         PCR analysis (Huxley et al., 1990). To generate diploid homozygous <br /> dedicated invasion-specific transcriptional program             fol1&#226;&#338;&#172;/fol1&#226;&#338;&#172; deletion strain haploid fol1&#226;&#338;&#172; deletion strain YUG1 trans- <br />  <br />  <br />  <br /> Vol. 15, August 2004                                                                                                                         3813 <br />  U. Gueldener et al. <br />  <br />  <br />  <br /> Table 1. Yeast strains used study <br />  <br /> Strain                                                                  Genotype                                                                Source <br />  <br /> CEN.PK2a                  MATa/MAT&#226;?&#163; ura3-52/ura3-52 trp1-289/trp1-289 leu2-3,112/leu2-3,112 his3-&#226;&#338;&#172;1/his3-&#226;&#338;&#172;1                          Entian Ko&#194;&#168; tter (1998) <br /> CEN.PK2-1ca               MATa ura3-52 trp1-289 leu2-3,112 his3-&#226;&#338;&#172;1                                                                   Entian Ko&#194;&#168; tter (1998) <br /> CEN.HE4a                  MATa/MAT&#226;?&#163; ura3-52/ura3-52 trp1-289/trp1-289 leu2-3,112/leu2-3,112 his3-&#226;&#338;&#172;1/his3-&#226;&#338;&#172;1                          Sen-Gupta et al. (1997) <br />                            YNL256w/ynl256w&#226;&#338;&#172;&#226;&#172;&#8249;kanMX4 <br /> YGK35b                    MATa ura3-52 phd1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-ble-loxP                                                                           This   study <br /> YGK36b                    MATa ura3-52 phd1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-ble-loxP fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                     This   study <br /> YGK37b                    MATa ura3-52 tec1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-ble-loxP                                                                           This   study <br /> YGK38b                    MATa ura3-52 tec1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-ble-loxP fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                     This   study <br /> YUG1a                     MATa ura3-52 trp1-289 leu2-3,112 his3-&#226;&#338;&#172;1 ynl256w&#226;&#338;&#172;&#226;&#172;&#8249;kanMX4                                                   This   study <br /> YUG3a                     MATa ura3-52 trp1-289 leu2-3,112 his3-&#226;&#338;&#172;1[pUG15 GALS-FOL1 CEN6 ARSH4 TRP1                                   This   study <br />                            Amp] <br /> YUG31a                    MATa/MAT&#226;?&#163; ura3-52/ura3-52 trp1-289/trp1-289 leu2-3,112/leu2-3,112 his3-&#226;&#338;&#172;1/his3-&#226;&#338;&#172;1                          This study <br />                            FOL1/FOL1-yEGFP3-loxP-kanMX-loxP <br /> YUG28a                    MATa/MAT&#226;?&#163; ura3-52/ura3-52 trp1-289/trp1-289 leu2-3,112/leu2-3,112 his3-&#226;&#338;&#172;1/his3-&#226;&#338;&#172;1                          This study <br />                            URA3/ura3&#226;&#338;&#172;&#226;&#172;&#8249;yEGFP3-loxP-kanMX-loxP <br /> CGX31b                    MATa/MAT&#226;?&#163; ura3-52/ura3-52                                                                                  Gimeno et al. (1992) <br /> YUG72b                    MATa/MAT&#226;?&#163; ura3-52/ura3-52 FOL1/fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP FLO11/flo11&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                     This study <br /> YUG74b                    MATa/MAT&#226;?&#163; ura3-52/ura3-52 FOL1/fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP STE12/ste12&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                     This study <br /> YUG94b                    MATa ura3-52                                                                                               This study <br /> YUG96b                    MATa ura3-52 fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP                                                                                    This study <br /> YUG98b                    MATa ura3-52 ste12&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                                        This study <br /> YUG100b                   MATa ura3-52 flol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                                        This study <br /> YUG102b                   MATa ura3-52 fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP ste12&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                             This study <br /> YUG104b                   MATa ura3-52 fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP flol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP-kanMX-loxP                                                             This study <br /> YUG105b                   MATa/MAT&#226;?&#163; ura3-52/ura3-52 fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP/fol1&#226;&#338;&#172;&#226;&#172;&#8249;loxP                                                            This study <br /> a <br />     Genetic background:W303. <br /> b <br />     Genetic background: &#226;&#338;&#186;1278b. <br />  <br />  <br />  <br /> formed HO plasmid Ycp50-HO-12 (gift C. Hollenberg). Several            expressed &#226;&#338;&#186;1278b-based strains analyzed FACS described <br /> uracil-prototrophic transformants streaked diploids iden-          (Niedenthal et al., 1999). Wild-type yEGFP3 integration strains <br /> tified mating type-specific PCR. The HO plasmid recovered 5-FOA            grown overnight 30&#194;&#176;C liquid synthetic minimal glucose media. <br /> counterselection (Boeke et al., 1984).                                               <span id='am-5' about='protege:FROM' typeof='owl:Thing'>From</span> cultures cells freshly inoculated media (starting <br />                                                                                      OD600 0.1) grown OD600 0.3&#226;&#8364;&#8220; 0.5 analyzed FACS as <br /> Quantification Fol1 Protein                                                       described above. <br /> The Fol1 protein quantified fluorescence-activated cell <br /> analysis Fol1-GFP fusion protein. The genomic fusion yEGFP            ii) LacZ/Overlay Assay Yeast cells transformed pFLO11-2/1, pFLO11- <br /> C-terminus FOL1 yeast strain CEN.PK2 homologous                    4/3, pFLO11-6/5, pFLO11-8/7, pFLO11-9/10, pFLO11-12/11, pFLO11-14/13, <br /> integration yEGFP3-loxP-kanMX-loxP integration cassette,              pFLO11::LacZ vector (Rupp et al., 1999) pregrown in <br /> created PCR template plasmid pUG24 using oligonucleotides 609              SC-ura medium containing 200 &#226;?&#174;g/ml folinic acid 30&#194;&#176;C O/N. Approxi- <br /> 636. In resulting strain YUG31 yEGFP3 gene integrated frame           mately 1 &#226;&#171;&#187; 106 cells (wt) 28 &#226;&#171;&#187; 106 cells (fol1&#226;&#338;&#172;) spotted SD plates <br /> at C-terminal end FOL1, replacing FOL1 stop codon                 amino acids containing 50 &#226;?&#174;g/ml folinic acid incubated 30&#194;&#176;C <br /> following 60 <span id='am-89' about='xsp:base' typeof='owl:Thing'>base</span> pairs downstream. The correct integration verified          25 h. On average spots carried 4 &#226;&#171;&#187; 107 cells. The overlay assay was <br /> two PCRs using 1) yEGFP3 specific primer 604 FOL1 specific               performed previously described (Suckow Hollenberg, 1998). Briefly <br /> primer S52, 2) kanMX module-specific primer 335 primer          spots covered 5 ml 70&#194;&#176;C hot agarose (0.5% agarose, 0.5 M sodium <br /> FOL1 terminator region (S38). Plasmid pUG24 obtained cloning                phosphate buffer, pH 7.0, 0.2% SDS, 0.4 mg/ml 5-bromo-4-chloro-3- <br /> PCR-generated yEGFP3 SalI fragment (template plasmid: pYGFP3; Cormack                indolyl-&#226;?&#164;-d-galactopyranoside) incubated 24 h 37&#194;&#176;C. <br /> et al., 1997; oligonucleotides 556 557) SalI cut pUG6 (Gu&#194;&#168; ldener et al., <br /> 1996).                                                                               Quantification Fol1p Enzymatic Activities <br />    Wild-type yEGFP3 integration strains grown overnight 30&#194;&#176;C in <br /> the appropriate liquid media indicated. From cultures cells            (i) FOL1 Overexpression Yeast A 2.6-kb-long FOL1 fragment obtained <br /> freshly inoculated media (starting OD600 0,1) grown            PCR EcoRI subclone cosmid 14-4 (obtained P. Philippsen, <br /> OD600 0.3&#226;&#8364;&#8220; 0.5. Cells diluted OD600 0.03 10 mM Tris, pH 7.5,     Basel) using oligonucleotides 327 328. The SpeI/SalI-cleaved PCR product <br /> and analyzed directly. Flow cytometry carried using FACSort                cloned SpeI/SalI cut p414GALS (Mumberg et al., 1994) yielding <br /> system (Becton Dickinson, Heidelberg, Germany; Niedenthal et al., 1996,              pUG15 (selection marker: TRP1). pUG15 transformed strain <br /> 1999). Ten thousand particles (living cells) analyzed sample. The           CEN.HE4 (relevant genotype: FOL1/fol1&#226;&#338;&#172;::KanMX). Subsequent tetrad anal- <br /> autofluorescence obtained wild-type (wt) strain set electronically       ysis replica platting identified G418-resistant Trp&#226;&#171;&#185; haploid seg- <br /> to channel 200 deletion strains analyzed using            regants (YUG3). The GAL1/2 &#226;?&#174; FOL1 overexpression vector pUG14 was <br /> parameters. For data point independent cultures measured.             constructed subcloning SpeI/SalI fragment pUG15 SpeI/SalI-cut <br />                                                                                      p424GAL1 (Mumberg et al., 1994). <br /> Quantification FLO11 Promoter Activity <br />                                                                                      (ii) FOL1 Expression E. coli The FOL1 gene cloned tac <br /> i) GFP/FACS Analysis For quantification FLO11 promoter activity             promoter plasmid pHK255-I (Duesterhoeft, 1990). To end FOL1 <br /> 3-kb-long FLO11 promoter fragment cloned yEGFP gene              gene PCR-amplified EcoRI subclone cosmid 14-4 using <br /> located CEN plasmid. The FLO11 promoter fragment obtained PCR              oligonucleotides 345 346, cleaved EcoRV, cloned pUG7 <br /> on genomic DNA strain CGX31 using oligonucleotides 1098 1099. The             (pBluescript derivative; Gu&#194;&#168; ldener et al., unpublished results) resulting in <br /> resulting PCR product digested SacI/SalI cloned SacI/SalI          plasmid pSH50. pSH50 cleaved NcoI SacI DNA fragment <br /> cut pUG35 (a pUG23 derivative carrying yEGFP3, URA3, CEN/ARSH4),                 carrying complete FOL1 gene cloned NcoI/SacI cut pHK255-I <br /> thus replacing MET25 promoter yEGFP3 gene. The resulting         resulting plasmid pSH51, transformed E. coli strain XL1 <br /> plasmid pUG64 control plasmid p416MET25 (Mumberg et al., 1995)               blue expression Fol1 protein. <br />  <br />  <br />  <br /> 3814                                                                                                                            Molecular Biology Cell <br />                                                                                                             Yeast C1 Carrier Filamentous Growth <br />  <br />  <br />  <br /> Table 2. Oligonucleotides used study <br />  <br /> Oligonucleotide, no.                                                     Sequence (5&#226;&#172;&#732; 3 3&#226;&#172;&#732;)                                                  Used for <br />  <br /> Plasmid constructiona,b <br />   556                         GTACTAGTCGACcagttatttgtacaattcatcc                                                                         pUG24 <br />   557                         CCACTAGTCGACatgtctaaaggtgaagaattattc                                                                       pUG24 <br />   1098                        CCACTAGAGCTCTCTCCACATACCAATCACTCG                                                                          pUG64 <br />   1099                        GTACTAGTCGACTGGTCTTTGCATAGTGTGCG                                                                           pUG64 <br />   327                         GCCGTAAGACTAGTATGATAAATTGCAAGGTC                                                                           pUG15 <br />   328                         GGCGCCGGGTCGACTTATTCCAAACCTTTATAAAT                                                                        pUG15 <br />   345                         CGCACGTGACGTCTTATTCCAAACCTTTATAAA                                                                          pSH50 <br />   346                         CGCATGCCATGGTAAATTGCAAGGTC                                                                                 pSH50 <br />   327                         GCCGTAAGACTAGTATGATAAATTGCAAGGTC                                                                           pUG38 <br />   391                         ACTAAGAATTCTTCCAAACCTTTATAAAT                                                                              pUG38 <br /> Gene disruptionb <br />   278                         TTTGAAATATTCTCTAAATTATGCAGTTATTTCTCGTGATcagctgaagcttcgtacgc                                                FOL1 <br />   279                         TACAAAACAGAATTACATATTTAATCTATATAACTAACCTgcataggccactagtggatctg                                             FOL1 <br />   1052                        ATGAAAGTCCAAATAACCAATAGTAGAACAGAGGAAATCTcagctgaagcttcgtacgc                                                STE12 <br />   1053                        ATATCAGGTTGCATCTGGAAGGTTTTTATCGGACCTTCGAgcataggccactagtggatctg                                             STE12 <br />   1056                        ATGCAAAGACCATTTCTACTCGCTTATTTGGTCCTTTCGCcagctgaagcttcgtacgc                                                FLO11 <br />   1057                        GTATCATTAGAATACAACTGGAAGAGCGAGTAGCAACCACgcataggccactagtggatctg                                             FLO11 <br />   591                         ACCCAACTGCACAGAACAAAAACCTGCAGGAAACGAAGATAAATCatgtctaaaggtgaagaat                                           URA3 <br />   592                         AATTTGTGAGTTTAGTATACATGCATTTACTTATAATACAGTTTTgcataggccactagtgga                                            URA3 <br />   1425                        CTGGAATTCATAATTCATTAAGCAGAAAATATGTACCATGcagctgaagcttcgtacgc                                                PHD1 <br />   1426                        TTGTTCATAGAGCAAAGAGTTAACGGATTATGTTATGTGCgcataggccactagtggatc                                               PHD1 <br />   1427                        ATAATCCACCTATTTCAACAATTCTGATACCTGTTTAACCcagctgaagcttcgtacgc                                                TEC1 <br />   1428                        TGCGTATTTATGTACGAGATGTATGTATGTATGTAGACATgcataggccactagtggatc                                               TEC1 <br /> Tagging <br />   609                         TTTCCTCCCTTGGTTATTTTTAACGATTCTTTATTATGAAgcataggccactagtggatctg                                             FOL1 <br />   636                         GAGCATTAAATTAGCAGATGCTATTTATAAAGGTTTGGAAatgtctaaaggtgaagaattattcac                                         FOL1 <br /> Verification <br />   363                         GGATGTATGGGCTAAATG                                                                                         kanMX forw <br />   335                         CCTCGACATCATCTGCCC                                                                                         kanMX rev <br />   1068                        TTTGGGATCCCTCCCGTG                                                                                         YNL256w start <br />   1069                        TTGTACCATGCCAAGAAC                                                                                         YNL256w stop <br />   638                         ATAGATGAAGTTTTCGTGTG                                                                                       STE12 start <br />   1055                        GAAAGATGTATCGGTAGC                                                                                         STE12 stop <br />   1058                        CGTTCTCTTCTGATGAGG                                                                                         FLO11 start <br />   1061                        TTCATCAAAGCCTGGTCG                                                                                         FLO11 stop <br />   604                         GCATCACCTTCACCTTCACC                                                                                       yEGFP3 <br />   S52                         GTTTCAAACACATTCAAATGG                                                                                      FOL1 <br />   S38                         GGAACAATCACTTATCATGC                                                                                       FOL1 <br />   593                         GAGGCTACTGCGCCAATTG                                                                                        URA3 <br />   604                         GCATCACCTTCACCTTCACC                                                                                       URA3 <br />   1429                        GCTGGAGTTTCTCTCGATGG                                                                                       TEC1 <br />   1430                        TACTCGTTCTGGCGCGTCAG                                                                                       TEC1 <br />   1431                        ATATAGGAAGAACTCTACAG                                                                                       PHD1 <br />   1432                        TCTAACGAATTGCGCAATAG                                                                                       PHD1 <br /> a <br />   Underlines indicate restriction enzyme recognition sites. <br /> b <br />   Lower case letters indicate nucleotides homologous sequences plasmids used PCR; capital letters indicate nucleotides homologous <br /> to genomic sequences yeast used homologous recombination. <br />  <br />  <br />  <br />  <br /> Enzymatic Assays                                                                Assay 6-Hydroxymethyldihydropterin Pyrophosphokinase Assay mix- <br />                                                                                 tures contained 100 mM Tris/HCl, pH 8.0, 10 mM MgCl2, 5 mM &#226;?&#164;-mer- <br /> The following materials purchased B. Schircks (Jona, Switzerland): <br />                                                                                 captoethanol, 50 &#226;?&#174;M 6-hydroxymethyl-7,8-dihydropterin, protein. Af- <br /> 6-hydroxymethylpterin, 6-hydroxymethylpterin pyrophosphate, 7,8-di- <br />                                                                                 ter incubation 20 min reaction stopped described above. The <br /> hydro-d-neopterin. Pterines hydrated Pd-Charcoal (4 h, RT, atmo-      samples anaylzed 6-hydroxymethylpterin pyrophosphate as <br /> spheric pressure) formation 7,8-dihydropterins.                    described above. The retention volume 6-hydroxymethylpterin pyro- <br />                                                                                 phosphate 2.5 ml. <br /> Assay 7,8-Dihydroneopterin Aldolase (Haussmann et al., 1998) Assay <br /> mixtures contained 100 mM Tris/HCl, pH 8.0, 5 mM &#226;?&#164;-mercaptoethanol, 100         Assay Dihydropteroate Synthase Assay mixtures contained 0.1 M Tris <br /> &#226;?&#174;M 7,8-dihydro-d-neopterin protein. The mixture incubated 37&#194;&#176;C       hydrochloride, pH 8.0, 10 mM MgCl2, 5 mM &#226;?&#164;-mercaptoethanol, 20 &#226;?&#174;M <br /> for 20 min. The reaction terminated addition 30 &#226;?&#174;l 1 M HCl        6-hydroxymethyl-7,8-dihydropterin pyrophosphate, 0.04 nM [carboxyl-14C]4- <br /> containing 1% I2 2% KI (wt/vol). The samples incubated room         aminobenzoate, protein total volume 6 &#226;?&#174;l. The mixture was <br /> temperature (RT) dark 5 min. Excess iodine reduced        incubated 37&#194;&#176;C 30 min. The reaction terminated immersion into <br /> addition 20 &#226;?&#174;l 2% ascorbinic acid (wt/vol). Samples analyzed         boiling water bath. Aliquots (3 &#226;?&#174;l) applied Whatman 3MM paper <br /> reversed-phase HPLC using column Nucleosil RP18 (4 &#226;&#171;&#187; 250 mm). The          subjected descending paper chromatography using 0.1 M potas- <br /> eluent contained 30 mM HCOOH 7% MeOH. The flow rate 2                   sium phosphate solvent. The enzyme product remained start the <br /> ml/min. 6-hydroxymethylpterin detected fluorometrically (Ex: 365 nm;        chromatogram. Radioactivity paper segments determined scintil- <br /> Em: 446 nm). The retention volume reaction product 6.5 ml.           lation counting. <br />  <br />  <br />  <br /> Vol. 15, August 2004                                                                                                                                 3815 <br />  U. Gueldener et al. <br />  <br />  For DHNA HPPK activity measurements independent cultures              involved biosynthesis folate (Sen-Gupta et al., <br /> were measured triplicate median SD calculated.              1997). Therefore named ORF YNL256w FOL1 according <br />                                                                                    standard yeast genetic nomenclature. Furthermore a <br /> Yeast Protein Analysis <br />                                                                                    folate auxotroph mutant named fol1 mapped pre- <br /> Wild-type fol1&#226;&#338;&#172; strains (strain background: &#226;&#338;&#186;1278b) grown an <br /> OD600 0.6 &#226;&#8364;&#8220; 0.8. Then cells incubated 0.15 &#226;?&#174;M &#226;?&#163; factor 15 min. <br />                                                                                    viously region (Little Haynes, 1979). <br /> Cells 20-ml cultures pelleted, crude protein extracts              Detailed analysis Fol1p protein sequence indicated <br /> produced according protocol provided Miltenyi Biotec GmbH (Ber-            presence putative enzymatic activities: DHNA, <br /> gisch-Gladbach, Germany). Protein samples immediately boiled             HPPK, DHPS (Figure 2). These enzymatic activi- <br /> SDS loading buffer, aliquoted, frozen &#226;&#171;&#186;20&#194;&#176;C. <br />   Protein samples electrophoresed SDS-PAGE 12% polyacryl- <br />                                                                                    ties perform subsequent steps biosynthesis of <br /> amide gels (acrylamide:bis-acrylamide ratio 37.5:1) using Hoefer       folate (Figure 1). In bacteria enzymatic activities <br /> Mighty Small gele (Amersham Biosciences, Freiburg, Germany). After          usually encoded separate ORFs (e.g., Bacillus <br /> protein transfer membranes blocked TBST buffer described         subtilis; Slock et al., 1990). In contrast filamentous fungi <br /> the ECL Advance Western Blotting Detection Kit (Amersham Biosciences). <br /> Membranes incubated primary antibody (antiphospho p44/42 <br />                                                                                    probably higher eukaryotes HPPK and <br /> MAPK; Cell Signaling Technology, Inc., Beverly, MA; 9101) TBST-2% block         DHPS domains fused protein, e.g., Toxoplasma <br /> overnight 4 C&#194;&#176; followed washings TBST RT. Immunoblots          gondii pea (Triglia Cowman, 1994; Rebeille et al., <br /> were incubated secondary antibody (rabbit Ig, HRP-linked Whole anti-          1997; Figure 2). However, fungi Schizosaccharomyces <br /> body; Amersham Biosciences) TBST-2% block 7 h 4&#194;&#176;C washed <br /> three times TBST RT developed using Amersham ECL kit. <br />                                                                                    pombe C. albicans Pneumocystis carinii (zygomy- <br />                                                                                    cete), enzyme activities be <br /> Microscopy                                                                         located single polypeptide (Volpe et al., 1993; Figure 2). <br /> The FOL1 gene PCR amplified using oligonucleotides 327 391             Detailed FASTA alignments Fol1p indicated N- <br /> plasmid pUG16 template. The SpeI/EcoRI 2.6-kb PCR product cloned            terminus carries domains 24 32% identity, <br /> into SpeI/EcoRI cut yEGFP3 vector pUG23 (Gu&#194;&#168; ldener et al., unpublished            respectively, ORF1 B. subtilis encoding DHNA. This <br /> results) resulting CEN-vector pUG38.                                        organization P. carinii fas protein, <br />   For Fol1p localization experiments yeast strain CEN.PK2 transformed <br /> with plasmid pUG38. For staining mitochondria nuclei living cells,       domains FasA FasB required for <br /> exponential growth phase cells incubated YPD media supplemented            DHNA function (Volpe et al., 1995; Figure 2). The S. cerevisiae <br /> with 4&#226;&#172;&#732;6-diamino-2-phenylindoledihydrochloride (DAPI; stock:1 &#226;?&#174;g/&#226;?&#174;l, final         DHNA domains lowest identity (21%) cor- <br /> concentration: 1 &#226;?&#174;g/ml) 30 min. Cells fixed addition formal-        responding domain Neurospora crassa highest <br /> dehyde (500 &#226;?&#174;l 37% formaldehyde added 5 ml yeast culture) and <br /> incubation 30 min 30&#194;&#176;C. Cells washed PBS            identity (43%) domain C. albicans. The central <br /> adsorbed polylysine-coated coverslip. Cells viewed using Zeiss       Fol1p domain exhibits identities 31 48%, re- <br /> Axioskop (Oberkochen, Germany) microscope.                                         spectively, HPPK carrying domains T. gondii and <br />                                                                                    P. carinii. The C-terminal domain Fol1p shares between <br /> Assessment Budding Pattern                                                      25% identity corresponding domain T. gondii <br /> Haploid cells indicated genotype grown YPD plates containing 50        44% identity C-terminal domain C. albi- <br /> &#226;?&#174;g/ml folinic acid taken plates, resuspended PBS buffer, and <br /> stained chitin-specific dye calcofluor white (concentration 4 &#226;?&#174;g/ml).     cans (Figure 2). Taken data indicate Fol1p <br /> Cells spotted polylysine-coated microscope slides viewed             shares trifunctional nature corresponding pro- <br /> using Zeiss Axioskop microscope. Cells bud scars           teins limited number fungi, other <br /> classified follows: unipolar/axial (scars end), bipolar (scars   eukaryotes protein bifunctional. <br /> both ends), random (scars localized ends; number cells &#226;&#177;&#8211;100). <br />  <br /> Immunocytochemistry Electron Microscopy <br />                                                                                    FOL1 Codes Enzyme Activities DHNA, HPPK, and <br /> For electron microscopy gfp-FOL1 fusion constructed cloning a <br /> SpeI/SalI FOL1 fragment (out pUG15) plasmid pGFP-N-FUS <br />                                                                                    DHPS <br /> (Niedenthal et al., 1996) resulting pUG16. Yeast strain CEN.PK2             To confirm enzymatic activities inferred from <br /> transformed plasmid pUG16. Cells grown selective minimal              homology encoded FOL1, performed <br /> media, harvested, fixed mixture 0.5% glutaraldehyde 4% <br /> freshly depolymerized paraformaldehyde 0.1 M sodium citrate buffer, pH <br />                                                                                    enzymatic tests enzyme activities. First FOL1 <br /> 4.7, 1.5 h 30&#194;&#176;C (Zimmer et al., 1995). Subsequent treatment 1%         gene expressed E. coli inducible tac promoter. <br /> sodium metaperiodate (Sigma) 1 h 4&#194;&#176;C performed support               As expected, control E. coli cells exhibited endoge- <br /> penetration cell wall cryoprotectant sucrose. Cryoprotection      nous enzyme activities (Table 3; XL1 Blue). However E. <br /> the cells achieved infusion mixture containing 25% polyvi- <br /> nylpyrrolidone (PVP, MW 10,000, Sigma) 1.6 M sucrose 3 h 30&#194;&#176;C <br />                                                                                    coli cells expressing S. cerevisiae FOL1 gene three <br /> (Tokuyasu, 1989). Cells mounted specimen holders immediately           enzyme activities increased significantly. The DHNA and <br /> frozen liquid nitrogen. Ultrathin cryosections prepared &#226;&#171;&#186;100&#194;&#176;C          HPPK enzyme levels increased fivefold compared with <br /> and deposited formvar/carbon-coated copper grids thawing. Immu-           enzyme levels protein extracts control cells, <br /> nolabeling polyclonal anti-GFP antibody (Clontech, Palo Alto, CA) or <br /> polyclonal anti-Fol1p antibody (D. Becher, unpublished results) second- <br />                                                                                    DHPS enzyme level increased ninefold <br /> ary colloidal gold (10 nm) goat anti-rabbit IgG conjugate (British BioCell,        (Table 3; ptac:FOL1). These data demonstrate FOL1 <br /> Cardiff, Wales) performed according Griffiths et al., (1984) minor     encodes enzyme activities. <br /> modifications. As controls, sections incubated directly gold IgG        We characterized FOL1 enzyme activities in <br /> conjugate, omitting primary antibody step, cells GFP-fusion <br /> protein used. Finally, frozen-thawed sections stabilized and <br />                                                                                    yeast. Protein extracts prepared haploid wt strain <br /> stained mixture 0.3% uranyl acetate 2% methylcellulose (25 cp;       exhibited low significant levels enzymatic <br /> Sigma). Specimens examined Zeiss 906 electron microscope 60         reactions (Table 3; FOL1). Analysis isogenic FOL1 <br /> kV acceleration voltage.                                                           deletion strain (kept alive presence folinic acid in <br />                                                                                    media; later) revealed severely reduced activities for <br /> RESULTS                                                                            enzyme reactions (Table 3; fol1&#226;&#338;&#172;). In contrast mod- <br />                                                                                    erate overexpression FOL1 GALS promoter a <br /> The FOL1 Gene Product Has Been Conserved Evolution                              wt strain resulted strong increase enzymatic <br /> Data base searches using 864-amino acid-long protein                           activities (Table 3; FOL1 &#226;&#171;&#185; pGALS:FOL1). <br /> sequence derived ORF YNL256w left arm                                  These data demonstrate Fol1p carries enzymatic <br /> chromosome XIV revealed significant homologies pro-                             activities biosynthesis folic acid, DHNA, <br /> teins prokaryotic eukaryotic microorganisms known                           HPPK, DHPS (Figure 1). <br />  <br />  <br /> 3816                                                                                                                  Molecular Biology Cell <br />                                                                                                   Yeast C1 Carrier Filamentous Growth <br />  <br />  <br />  <br />  <br /> Figure 2. The Fol1 protein conserved evolution. Comparison Pfam domains Fol1 protein S. cerevisiae the <br /> corresponding proteins species (B. subtilis P28823, S66109, P28822; P. carinii P29251; S. pombe SPBC1734.03; C. albicans CAC08858; <br /> A. nidulans an4979.1, an6032.1; N. crassa ncu02798.1, b21j21_140; M. grisea mg01629.1, mg01474.1; T. gondii AAB93492.1; P. falciparium <br /> CAA83456.1; P. sativum CAA69903.1; numbers correspond GenBank entries). For protozoa P. sativum bifunctional <br /> HPPK- DHPS-carrying proteins presented. The percentage identity corresponding S. cerevisiae Pfam domain calculated <br /> using FASTA. <br />  <br />  <br /> The FOL1 Gene Is Required Growth                                     deletion unable form visible colonies YPD (Fig- <br /> Deletion allele FOL1 diploid yeast strain                 ure 3B, left panel). The terminal phenotype fol1 dele- <br /> (Figure 3A) subsequent tetrad analysis revealed                 tion rich media (YPD) microcolony uneven <br /> FOL1 essential gene, spores carrying FOL1                   shape containing 50 &#226;&#8364;&#8220;100 slightly elongated cells. The non- <br />  <br />  <br />  <br /> Table 3. Enzymatic activities encoded FOL1 <br />  <br />                                                   DHNA                                       HPPK                                 DHPS <br />                                                 (nmol/mg h)                               (nmol/mg h)                          (nmol/mg h) <br />  <br /> S. cerevisiae strain <br />    FOL1                                            h6.0 &#226;&#171;&#190; 1                                 1.0 &#226;&#171;&#190; 0.5                                 16 <br />    fol1&#226;&#338;&#172;                                            &#226;&#172;?1.0a                                    &#226;&#172;?0.3a                                     5 <br />    FOL1 &#226;&#171;&#185; pGALS:FOL1                               61.7 &#226;&#171;&#190; 20                               43.8 &#226;&#171;&#190; 15                                  60 <br /> E. coli strain <br />    XL1Blue                                         73.8 &#226;&#171;&#190; 10                               61.4 &#226;&#171;&#190; 15                                185 <br />    XL1Blue &#226;&#171;&#185; ptac:FOL1                            364.0 &#226;&#171;&#190; 25                              310.7 &#226;&#171;&#190; 25                               1714 <br />  <br /> The specific enzyme activity assays performed using protein extracts E. coli yeast. E. coli XL1Blue assayed plasmid <br /> (control) transformed pSH51 carrying yeast FOL1 gene control tac promoter (ptac-FOL1). Yeast strains that <br /> were derived genetic ground grown media complemented 1 mg/ml folinic acid. Haploid CEN.PK2-1C <br /> carries FOL1&#226;&#171;&#185; wt gene (FOL1), YUG1 deleted FOL1 (fol1&#226;&#338;&#172;). Strain YUG3 expresses FOL1 GALS promoter plasmid <br /> (FOL1 &#226;&#171;&#185; pGALS-FOL1; Mumberg et al., 1994). <br /> a <br />   The levels detection level assay. <br />  <br />  <br />  <br /> Vol. 15, August 2004                                                                                                                       3817 <br />  U. Gueldener et al. <br />  <br />  <br />                                                                           1 mg/ml FA), probably folinic acid known be <br />                                                                           inhibitor folate-dependent enzymes (Girgis et <br />                                                                           al., 1997). <br />                                                                              Additionally, fol1 deletion strains could <br />                                                                           grow media supplemented folinic acid, were <br />                                                                           unable grow nonfermentable carbon sources eth- <br />                                                                           anol, glycerol, acetate (supplemented folinic acid), <br />                                                                           indicating Fol1p 1C carrier required <br />                                                                           mitochondrial function yeast addition of <br />                                                                           exogenous folinic acid mitochondrial one- <br />                                                                           carbon carrier pools adequately (unpublished data). <br />  <br />                                                                           Moderate But Not High FOL1 Expression Rescues the <br />                                                                           Nongrowth Phenotype fol1&#226;&#338;&#172; Strain <br />                                                                           Interestingly, strong overexpression FOL1 expression <br />                                                                           GAL1 promoter 2-&#226;?&#174;&#226;&#8364;&#8220; based plasmid did not <br />                                                                           complement nongrowth phenotype fol1&#226;&#338;&#172; strain. <br />                                                                           Likewise strong overexpression FOL1 diploid wt <br />                                                                           strain toxic resulted significant increase cells <br /> Figure 3. FOL1 essential gene.(A) Schematic representation of <br /> a FOL1 deletion using kanMX PCR-generated gene deletion               2C DNA content (Figure 4A; unpublished data). <br /> cassette. (B) Tetrad analysis heterozygous diploid fol1 deletion   However FOL1 expression moderate GALS pro- <br /> strain CEN.HE4 YPD supplemented folinic acid indi-             moter CEN-based plasmid exhibited wt-like <br /> cated. The genotype spores shown. Plates incubated      growth (Figure 4A). These data suggest high levels of <br /> 30&#194;&#176;C 72 h. Spores carrying fol1&#226;&#338;&#172; deletion germinated           THF-synthesizing enzymes and/or C1 carrier units are <br /> media folinic acid went 4 &#226;&#8364;&#8220; 6 cell divisions          detrimental yeast cell. Thus assumed the <br /> before stopping divide. This nongrowth phenotype              Fol1p level cell low, data <br /> rescued addition folinic acid.                                      indicate Fol1 promoter activity low (unpublished <br />                                                                           data). <br />                                                                              Next wanted know Fol1 <br /> growth phenotype fol1&#226;&#338;&#172; strain rescued                  protein regulated conditions intracellular <br /> addition 5-formyl tetrahydrofolate (folinic acid, FA)            C1 carrier concentration changes (here: addition folinic <br /> alternative one-carbon source media (Figure 3B,                  acid). A genomic FOL1-yEGFP fusion construct replaced one <br /> middle right panels). Folinic acid metabo-                chromosomal wt FOL1 alleles diploid strain <br /> lized 5,10-methenyl-THF synthetase yielding 5,10-methe-                (Figure 4B). Quantification background fluorescence <br /> nyl-THF, folate coenzyme converted                 parental wt strain (FOL1) flow cytometry revealed <br /> other folate coenzymes (Holmes Appling, 2002). How-                   relative green fluorescence (RGF) 10.23 U, the <br /> ever fol1&#226;&#338;&#172; strain rescued addition               Fol1-yEGFP fusion strain exhibited 11.2 U, represents <br /> of folinic acid, colonies reached wt size (Figure 3B;           small increase 0.97 RGF units (see MATERIALS AND <br />  <br />  <br />                                                                                                          Figure 4. High levels Fol1p are <br />                                                                                                          detrimental yeast. (A) Serial dilu- <br />                                                                                                          tion patch tests cells expressing <br />                                                                                                          Fol1p GAL1 promoter from <br />                                                                                                          2-&#226;?&#174; plasmid GALS pro- <br />                                                                                                          moter CEN/ARS plasmid in <br />                                                                                                          diploid strain CEN.PK2 grown on <br />                                                                                                          rich media containing glucose ga- <br />                                                                                                          lactose. Dilutions shown 10-fold <br />                                                                                                          starting 104 cells. (vector &#226;&#171;&#189; <br />                                                                                                          vector p424GAL1 without <br />                                                                                                          FOL1; Mumberg et al., 1994); (B) Dia- <br />                                                                                                          grammatic representation chro- <br />                                                                                                          mosomal FOL1-yEGFP fusion. The <br />                                                                                                          yEGFP3 gene fused frame to <br />                                                                                                          3&#226;&#172;&#732; end 2 FOL1 alleles <br />                                                                                                          strain CEN-PK2 integration a <br />                                                                                                          PCR-mediated yEGFP3-loxP-kanMX- <br />                                                                                                          loxP replacement cassette. (C) The <br />                                                                                                          diploid strain CEN.PK2 the <br />                                                                                                          heterozygous FOL1/FOL1-yEGFP3 <br />                                                                                                          strainYUG31 grown YPD&#226;&#171;&#185; or <br />                                                                                                          YPD&#226;&#171;&#185; supplemented 250 <br />                                                                                                          &#226;?&#174;g/ml folinic acid, of <br />                                                                                                          green fluorescence quantified by <br />                                                                                                          flow cytometry. For strain five <br />                                                                                                          independent cultures measured <br />                                                                                                          median SD given. RGF, <br />                                                                                                          relative green fluorescence. <br />  <br />  <br />  <br /> 3818                                                                                                          Molecular Biology Cell <br />                                                                                        Yeast C1 Carrier Filamentous Growth <br />  <br />  <br />  <br />  <br /> Figure 5. Localization Fol1-yEGFP3 fu- <br /> sion protein. Cells expressing plasmid- <br /> borne version FOL1-yEGFP grown <br /> in YPD&#226;&#171;&#185; midlog followed addition <br /> of 1 &#226;?&#174;g/ml DAPI incubation 30 min <br /> at 30&#194;&#176;C. Under conditions DAPI stains <br /> mainly mitochondrial DNA. Subse- <br /> quently cells adsorbed polyly- <br /> sine-coated coverslips, photomicro- <br /> graphs taken. Bar, 5 &#226;?&#174;m. <br />  <br />  <br />  <br /> METHODS details; Figure 4C). Addition folinic acid    strain. In cells expressing GFP-Fol1 fusion protein the <br /> the media led reduction background fluorescence     gold-particles labeled mainly mitochondria (Figure <br /> the wt control strain (RGF 9.30) &#226;&#172;&#402;0.93 RGF units; how-       6a). A detailed inspection higher resolution revealed <br /> ever, FOL1-yEGFP fusion strain showed stronger re-        gold particles associated mainly with <br /> duction (RGF 9.85) 1.35 RGF units, suggesting       mitochondrial membranes rarely mito- <br /> FOL1 promoter autoregulated (Figure 4C).               chondrial matrix (Figure 6, b c). To exclude the <br />   Together data indicate excess amounts Fol1p     GFP fusion protein influenced localization <br /> in yeast hampers growth cells       localized Fol1 protein diploid wt strain <br /> evolved sensing regulating mechanisms adjust         using peptide antibodies antigenic epitope of <br /> Fol1p levels adequately.                                        Fol1p itself. Although Fol1 protein level low in <br />                                                                 yeast antibody exhibited limited specificity, <br /> Fol1p Is Localized Mitochondrial Membrane                gold particles located mito- <br />                                                                 chondrial membrane (Figure 6d). Taken data <br /> To study subcellular localization Fol1 protein, we <br />                                                                 indicate Fol1 protein located mitochon- <br /> expressed C-terminal FOL1-yEGFP3 fusion protein a <br />                                                                 drial membrane system. <br /> CEN plasmid control regulatable MET25 pro- <br /> moter. The fusion protein functional able to <br /> rescue nongrowth phenotype fol1 deletion strain      Absence FOL1 Induces Filamentous Adhesive <br /> (unpublished data). The Fol1p-yEGFPp fusion protein             Growth <br /> showed punctate staining colocalized DAPI-           Rescue nongrowth phenotype fol1&#226;&#338;&#172; cells seen <br /> stained mitochondria (Figure 5).                                media containing folinic acid. The degree rescue ob- <br />   This localization confirmed electron microscopy        served dependent concentration folinic acid <br /> of yeast cells expressing N-terminal GFP-Fol1 fusion          (Figure 3B). Microscopic inspection haploid fol1&#226;&#338;&#172; cells <br /> protein using anti-GFP antibodies (Figure 6, a&#226;&#8364;&#8220; c). Expres-     grown 8 h rich YPD media presence of <br /> sion fusion protein complemented non-         folinic acid revealed elongated connected cells forming <br /> growth phenotype fol1 deletion strain. However           branching filamentous asters (Figure 7A). The extent fila- <br /> sometimes clumped mitochondria observed            mentation dependent folinic acid <br />  <br />  <br /> Vol. 15, August 2004                                                                                                    3819 <br />  U. Gueldener et al. <br />  <br />  <br />  <br />  <br /> Figure 6. Electron microscopy Fol1 protein. Cells diploid CEN.PK2 strain expressing plasmid-borne GFP-Fol1p processed <br /> and localization fusion protein determined using gold-labeled anti-GFP antibody (a&#226;&#8364;&#8220; c). In panel c enlargement a <br /> mitochondrial membrane shown panel b presented (indicated boxed area). (d) The analysis carried using anti-Fol1p <br /> peptide antibodies. Size markers indicated. <br />  <br />  <br />  <br /> present: media 50 &#226;?&#174;g/ml cells formed filaments           strain background commonly used filamen- <br /> consisting elongated bigger cells (Figure 7A,             tation invasion studies (Gimeno et al., 1992; Palecek et <br /> panel), media 200 &#226;?&#174;g/ml 50 &#226;&#8364;&#8220; 80% cells                al., 2002). Haploid strains strain background show <br /> formed filaments (Figure 7A, middle panel, types                invasive growth rich YPD media pro- <br /> shown), media 800 &#226;?&#174;g/ml, cells                  longed incubation 4 &#226;&#8364;&#8220; 6 days (Roberts Fink, 1994; <br /> showed regular yeast-like morphology (Figure               Rupp et al., 1999; Braus et al., 2003). As expected, micro- <br /> 7A, panel). After growth 24 h filamentous growth          colonies &#226;&#338;&#186;1278b-based haploid wt strain showed <br /> of fol1&#226;&#338;&#172; cells grown media 50 200 &#226;?&#174;g/ml folinic          yeast-like growth forming round colonies rich YPD <br /> acid continued, cells media 800 &#226;?&#174;g/ml            media supplemented folinic acid (Figure 8, b) <br /> folinic acid yeast-like formed uneven micro-            showed typical substrate invasiveness after <br /> colonies (Figure 7B). These data indicated deletion          growth 5 days (Figure 9, c d, wt), after <br /> the FOL1 gene induces filamentous growth phenotype             growth 24 h agar invasion observed <br /> prompted test fol1&#226;&#338;&#172; cells showed adhesive growth.           (Figure 9, b, wt). In contrast, deletion FOL1 <br /> Haploid fol1&#226;&#338;&#172; cells grown 48 h YPD media supple-              gene strain background gave rise slightly larger <br /> mented varying amounts folinic acid invaded              elongated cells filamentous structures and <br /> the agar, isogenic wt cells did agar inva-          uneven microcolony morphologies (Figure 8, g h). <br /> sion (Figure 7C).                                                    Moreover, contrast wt cells fol1&#226;&#338;&#172; strain showed <br />    These growth phenotypes reminiscent filamen-           strong adhesive growth 24 h growth media <br /> tous growth phenotype observed diploid cells               supplemented folinic acid (Figure 9, b, fol1&#226;&#338;&#172;). <br /> nitrogen limitation, haploid cells exhibiting invasive        After prolonged growth 5 days fol1&#226;&#338;&#172; strain despite <br /> growth grown rich media extended period            significantly reduced overall growth behavior showed <br /> time starved glucose (Cullen Sprague, 2000;          adhesive growth similar observed wt strain <br /> Lengeler et al., 2000; Palecek et al., 2002).                        (Figure 9, c d, fol1&#226;&#338;&#172;). <br />    The wt fol1&#226;&#338;&#172; strains used derived                      In diploid strains, starvation nitrogen induces <br /> CEN.PK2 W303 background. We assayed                  pseudohyphal growth. Therefore tested consequences of <br /> the consequences FOL1 deletion &#226;&#338;&#186;1278b/CGX31                deletion alleles FOL1 gene diploid &#226;&#338;&#186;1278b- <br />  <br />  <br /> 3820                                                                                                     Molecular Biology Cell <br />                                                                                                   Yeast C1 Carrier Filamentous Growth <br />  <br />  <br />  <br />  <br /> Figure 7. Pseudohyphal adhesive growth fol1&#226;&#338;&#172; strain. (A B) CEN.PK2 derived wild-type (wt) fol1&#226;&#338;&#172; strains pregrown <br /> on YPD medium containing different amounts folinic acid (50, 200, 800 &#226;?&#174;g/ml) 24 h 30&#194;&#176;C. Subsequently cells resuspended <br /> in water aliquots plated colonies derived single cells YPD medium containing 50, 200, 800 &#226;?&#174;g/ml folinic acid. Single <br /> colonies photographed 8- (A) 24-h (B) incubation 30&#194;&#176;C. (C) The strains described A pregrown YPD <br /> medium containing folinic acid, restreaked YPD containing different amounts folinic acid (50, 200, 800 &#226;?&#174;g/ml), incubated 30&#194;&#176;C <br /> for 48 h. Plates photographed (surface growth) washed extensively water rephotographed (adhesive growth). <br /> Background strains CEN.PK2/W303. <br />  <br />  <br />  <br /> based strain. Growth strain 24 h rich YPD media           8m n, diploid). However, growth 24 h adhesive <br /> supplemented folinic acid resulted filamentous growth            growth properties fol1&#226;&#338;&#172; fol1&#226;&#338;&#172; strain similar of <br /> and growth 72 h uneven colony formed (Figure            diploid wt strain (Figure 9, b, diploid). <br />  <br />  <br />  <br />  <br /> Figure 8. The fol1&#226;&#338;&#172; deletion strain <br /> shows filamentous growth, is <br /> independent STE12, TEC1, PHD1, <br /> and FLO11. Cells haploid wt <br /> strain YUG94 (wt), fol1&#226;&#338;&#172; single <br /> deletion (fol1&#226;&#338;&#172;), fol1&#226;&#338;&#172; ste12&#226;&#338;&#172; <br /> (fol1&#226;&#338;&#172;/ste12&#226;&#338;&#172;), fol1&#226;&#338;&#172; tec1&#226;&#338;&#172; (fol1&#226;&#338;&#172;/ <br /> tec1&#226;&#338;&#172;), fol1&#226;&#338;&#172; phd1&#226;&#338;&#172; (fol1&#226;&#338;&#172;/phd1&#226;&#338;&#172;), <br /> and fol1&#226;&#338;&#172; flo11&#226;&#338;&#172; (fol1&#226;&#338;&#172;/flo11&#226;&#338;&#172;) <br /> double deletion strains, the <br /> homozygous diploid fol1&#226;&#338;&#172;/fol1&#226;&#338;&#172; <br /> strain (fol1&#226;&#338;&#172;/fol1&#226;&#338;&#172;) incubated <br /> for 6 h (a), 24 h (b, c, e, g, i, k, m), <br /> and 72 h (d, f, h, j, l, n) 30&#194;&#176;C on <br /> YPD &#226;&#171;&#185; 50 &#226;?&#174;g/ml folinic acid, respec- <br /> tively. All fol1&#226;&#338;&#172; strains showed fila- <br /> mentous growth 24 h and <br /> formed uneven colonies. Background <br /> of strains &#226;&#338;&#186;1278b. <br />  <br />  <br />  <br /> Vol. 15, August 2004                                                                                                                    3821 <br />  U. Gueldener et al. <br />  <br />  <br />  <br />  <br />                                                                                 Figure 9. Adhesive growth fol1&#226;&#338;&#172; strains depen- <br />                                                                                 dent Ste12p Flo11p, independent Tec1p <br />                                                                                 Phd1p. (a, b, e, f) Single double deletion <br />                                                                                 strains streaked synthetic media containing 50 <br />                                                                                 &#226;?&#174;g/ml folinic acid grown 30&#194;&#176;C 24 h. Strains <br />                                                                                 shown haploid strains white tri- <br />                                                                                 angle, homozygous FOL1 (wt) fol1&#226;&#338;&#172; <br />                                                                                 (fol1&#226;&#338;&#172;) diploid strains. (c d) Single double <br />                                                                                 deletion strains streaked solid 2&#226;&#171;&#187; YPD con- <br />                                                                                 taining 125 &#226;?&#174;g/ml folinic acid incubated 5 days <br />                                                                                 30&#194;&#176;C. All plates photographed (surface <br />                                                                                 growth) washed extensively water and <br />                                                                                 rephotographed (adhesive growth). <br />  <br />  <br />  <br />    The haploid fol1&#226;&#338;&#172; cells exhibited altered budding        lated MAPK cascade strains lacking these <br /> pattern comparison wt cells. Haploid wt cells typi-      genes deficient filamentous invasive <br /> cally exhibit axial budding pattern           growth (Liu et al., 1993; Gavrias et al., 1996; Madhani et al., <br /> choose location new bud adjacent sites    1997; Palecek et al., 2002). To test folate star- <br /> the previous ones (for review Madden Snyder,           vation signal mediated Ste12 and/or Tec1p <br /> 1998; Chant, 1999). The haploid fol1&#226;&#338;&#172; strain showed            proteins fol1&#226;&#338;&#172; ste12&#226;&#338;&#172; fol1&#226;&#338;&#172; tec1&#226;&#338;&#172; double mutant <br /> mainly bipolar distal bud site selection (91%),     strains analyzed ability pseudohy- <br /> of buds emerged distal pole opposite birth   phal adhesive growth. Surprisingly haploid fol1&#226;&#338;&#172; <br /> end, 47% wt cells exhibited bipolar <br /> budding (Table 4). <br />    The actin cytoskeleton plays central role fungal <br /> dimorphism (Kron et al., 1994). Therefore ana-         Table 4. Increased bipolar budding pattern haploid fol1 deletion <br />                                                                strains <br /> lyzed actin cytoskeleton fol1&#226;&#338;&#172; strain grown on <br /> YPD media supplemented 50 &#226;?&#174;g/ml folinic acid.             Strain                 Unipolar (%)       Random (%)        Bipolar (%) <br /> Cells fol1&#226;&#338;&#172; strain showed enhanced polarity of <br /> actin patches distal tip daughter cells          wt                           30                23               47 <br /> throughout bud emergence relative distribution       fol1&#226;&#338;&#172;                         5                 4               91 <br /> actin patches wt cells. Furthermore actin cables        fol1&#226;&#338;&#172;   ste12&#226;&#338;&#172;                6                21               73 <br /> were pronounced fol1&#226;&#338;&#172; cells compared         fol1&#226;&#338;&#172;   flol1&#226;&#338;&#172;                6                 7               87 <br /> wt cells (unpublished data). These findings indicate      fol1&#226;&#338;&#172;   phd1&#226;&#338;&#172;                 4                 8               88 <br /> in fol1&#226;&#338;&#172; strain actin cytoskeletal structures       fol1&#226;&#338;&#172;   tec1&#226;&#338;&#172;                11                 5               84 <br /> adopted characteristics pseudohyphal growth (Kron et <br /> al., 1994; Cali et al., 1998).                                 Yeast cells grown 24 hrs YPD plates supplemented with <br />                                                                50 &#226;?&#174;g/ml folinic acid, analyzed. For determination of <br />                                                                budding pattern cells having bud scars, follow- <br /> Filamentous Growth fol1&#226;&#338;&#172; Strains Is Independent          ing criteria applied: unipolar: cells bud scars pole; <br /> Ste12p, Tec1p, Pdh1p, Flo11p                               bipolar: cells bud scars poles; random: cells least <br /> The Ste12p Tec1p transcription factors critical    bud scar localized pole (Freedman, et al.; 2000). <br />                                                                Values shown percentages (n &#226;&#177;&#8211; 100 cells). <br /> the induction filamentous invasive growth stimu- <br />  <br />  <br /> 3822                                                                                                    Molecular Biology Cell <br />                                                                                        Yeast C1 Carrier Filamentous Growth <br />  <br />  <br />  <br />  <br /> Figure 10. Kss1p Fus3p phosphoryla- <br /> tion levels altered fol1&#226;&#338;&#172; strains. <br /> Cells haploid wild-type (wt) fol1&#226;&#338;&#172; (&#226;&#338;&#172;) <br /> strains incubated 24 h 30&#194;&#176;C in <br /> liquid YPD media containing different <br /> amounts folinic acid (FA; 50, 200, 800 <br /> &#226;?&#174;g/ml). Midlog phase cultures both <br /> strains treated (&#226;&#171;&#185;) treated (&#226;&#171;&#186;) <br /> for 15 min 0.15 &#226;?&#174;M &#226;?&#163;-factor mating <br /> pheromone (&#226;?&#163;-factor). MAPK phosphoryla- <br /> tion levels determined immunoblot- <br /> ting using phosphorylation-state specific <br /> antibody (&#226;?&#163;pTEpY; Sabbagh et al., 2001). To- <br /> tal protein amounts loaded deter- <br /> mined Coomassie staining (loading con- <br /> trol). <br />  <br />  <br /> ste12&#226;&#338;&#172; fol1&#226;&#338;&#172; tec1&#226;&#338;&#172; double mutant strain    The Phosphorylation Level Fus3p Kss1p MAP <br /> showed increased bipolar budding filamentous                Kinases Is Not Increased fol1&#226;&#338;&#172; Strain <br /> growth uneven colony morphologies (Figure 8,         Filamentous growth depends mitogen- <br /> c d, k l; Table 4). However, adhesive growth            activated protein kinase (MAPK) cascade, results in <br /> could longer observed fol1&#226;&#338;&#172; ste12&#226;&#338;&#172; double         activation Ste12p (for review Madhani Fink, <br /> mutant strain 24 h growth (Figure 9, b).         1998b). It shown activation final MAP <br /> Likewise, invasive growth observed          kinase Kss1p phosphorylation increases filamentation, <br /> fol1&#226;&#338;&#172; ste12&#226;&#338;&#172; strain growth 5 days,        kinase activity Fus3p appears suppress <br /> single ste12&#226;&#338;&#172; mutant strain showed residual substrate           filamentous growth (Madhani Fink, 1997; Sabbagh et <br /> invasiveness (Figure 9, c d). In contrast fol1&#226;&#338;&#172; tec1&#226;&#338;&#172;   al., 2001). Stimulation mating pheromone response <br /> double mutant strain showed adhesive growth like            pathway leads phophorylation activation <br /> single fol1&#226;&#338;&#172; mutant strain, single tec1&#226;&#338;&#172; mu-        Kss1p Fus3p (Sabbagh et al., 2001). We tested <br /> tant strain showed residual invasiveness (Figure 9, e      deletion FOL1 gene results increased <br /> and f). Thus data suggest filamentation pro-       phosphorylation level Kss1p and/or Fus3p. The fol1&#226;&#338;&#172; <br /> cess induced loss Fol1p activity presence      single mutant strain showed Kss1p Fus3p phos- <br /> folinic acid media does require Ste12p       phorylation level similar corresponding <br /> Tec1p transcription factors, invasiveness de-        wt strain (Figure 10, wt &#226;&#338;&#172;). The level Kss1p and <br /> pendent presence Ste12 protein.                   Fus3p phosphorylation fol1&#226;&#338;&#172; strain did not <br />    Next tested role transcription factor          change cells grown media containing dif- <br /> Phd1p, thought involved pseudohy-        ferent amounts folinic acid. However control exper- <br /> phal growth acting pathway distinct            iments Kss1 Fus3p phosphorylation induced <br /> cAMP MAP kinase pathways (Gimeno Fink,                  strains addition &#226;?&#163; factor mating pheromone, <br /> 1994; Palecek et al., 2002). The haploid fol1&#226;&#338;&#172; phd1&#226;&#338;&#172; double     indicating MAPK pathway activated (Fig- <br /> mutant strain exhibited filamentous growth increased        ure 10). This experiment indicates deletion FOL1 <br /> bipolar budding pattern uneven colony shape           does result increased level Kss1p Fus3p <br /> the extent fol1&#226;&#338;&#172; single mutant strain (Figure 8,    phosphorylation. <br /> e f; Table 4). Furthermore adhesion assays fol1&#226;&#338;&#172; <br /> phd1&#226;&#338;&#172; double mutant strain 24 h growth exhibited <br /> invasiveness similar observed phd1&#226;&#338;&#172; single      Deletion FOL1 Results Increased Flo11p Expression <br /> mutant strain despite significantly reduced overall         Levels <br /> growth behavior (Figure 9, e f). These results suggest      Our data suggested fol1&#226;&#338;&#172; induced invasive growth <br /> that Phd1p required pseudohyphal growth          dependent Ste12p Flo11p. This imply that <br /> for agar invasive growth fol1&#226;&#338;&#172; mutant strain.              Flo11p transcription activated fol1&#226;&#338;&#172; strain. The FLO11 <br />    One known Ste12p target genes leading          promoter, largest known promoter S. cerevisiae, is <br /> and required induction pseudohyphal growth           regulated transcription factors including Ste12p <br /> and agar invasion FLO11 gene encoding cell             Tec1p (Lo Dranginis, 1998; Rupp et al., 1999; Palecek <br /> surface flocculin protein (Lo Dranginis, 1998). We          et al., 2002). We assayed FLO11 promoter activity in <br /> therefore tested Flo11 protein required      haploid fol1&#226;&#338;&#172; background. We placed 3-kb-long DNA <br /> the filamentous invasive growth behavior fol1&#226;&#338;&#172;       fragment containing FLO11 promoter the <br /> strain. The haploid fol1&#226;&#338;&#172; flo11&#226;&#338;&#172; double mutant strain ex-       yEGFP encoding gene CEN plasmid quantified the <br /> hibited filamentous growth increased bipolar bud-           FLO11 promoter-dependent green fluorescence flow cy- <br /> ding uneven colony shape extent       tometry haploid strains grown liquid synthetic glucose <br /> fol1&#226;&#338;&#172; single mutant strain (Figure 8, j; Table 4).        media supplemented 250 &#226;?&#174;g/ml folinic acid. Cells were <br /> However, showed adhesive growth 24 h 5          analyzed early log phase depleted for <br /> days growth (Figure 9, a&#226;&#8364;&#8220; d). These data suggest        glucose ammonium. In wt strain FLO11 promoter <br /> Flo11p similar Ste12p required fol1&#226;&#338;&#172;          showed &#226;&#172;&#402;22 relative green fluorescence (RGF) units, <br /> induced filamentous growth required agar inva-          fol1&#226;&#338;&#172; deletion strain promoter to <br /> sion.                                                           strongly activated, exhibiting 104 (RGF) units (Table 5). A <br />  <br />  <br /> Vol. 15, August 2004                                                                                                     3823 <br />  U. Gueldener et al. <br />  <br />  <br />                                                                           HPPK, DHPS, perform subsequent steps in <br /> Table 5. FLO11 promoter activity different strains                     biosynthesis folate. Our data suggest biosyn- <br />                                                                           thesis C1 carrier appears carried mito- <br /> Strain                                                    RGF <br />                                                                           chondria Fol1p localized mitochondrial <br /> wt                                                    21.91 &#226;&#171;&#190; 3.36 <br />                                                                           membrane system. Interestingly, deletion FOL1 gives rise <br /> ste12&#226;&#338;&#172;                                                17.15 &#226;&#171;&#190; 5.04        induction filamentation adhesive growth media <br /> fol1&#226;&#338;&#172;                                                104.87 &#226;&#171;&#190; 18.68       supplemented folinic acid presence excess <br /> ste12&#226;&#338;&#172;fol1&#226;&#338;&#172;                                           24.11 &#226;&#171;&#190; 6.16        amounts glucose ammonium, known sup- <br />                                                                           pressors dimorphic switch. Invasive growth induced <br /> Yeast strains expressing yEGFP FLO11 promoter present            fol1&#226;&#338;&#172; strains dependent MAP kinase transcrip- <br /> on plasmid grown liquid synthetic glucose media supple-         tion factor Ste12p major adhesin Flo11p. <br /> mented 250 &#226;?&#174;g/ml folinic acid OD600 0.3&#226;&#8364;&#8220; 0.5. The           Indeed Flo11p expression levels increased Ste12p- <br /> amount yEGFP protein quantified flow cytometry. For <br />                                                                           dependent way strains FOL1 absent. Interest- <br /> each data point, independent cultures measured. Me- <br /> dian &#226;&#171;&#190; SD given. RGF, relative green fluorescence.                    ingly, however, filamentous growth observed fol1&#226;&#338;&#172; <br />                                                                           strains dependent transcription factors Ste12p, <br />                                                                           Tec1p, Phd1p Flo11p, suggesting in <br />                                                                           fol1&#226;&#338;&#172; strains unknown mechanisms/pathways/adhesins <br /> ste12&#226;&#338;&#172; strain showed FLO11 promoter activity similar                 required pseudohyphal growth. <br /> that wt strain, indicating basal FLO11 promoter <br /> activity dependent Ste12p. The fol1&#226;&#338;&#172; ste12&#226;&#338;&#172; double              The Trifunctional Nature Fol1p Is Conserved Evolution <br /> deletion strain exhibited promoter activity 24 &#226;&#171;&#190; 6 RGF, <br /> indicating elevated FLO11 promoter activity ob-                  The different enzymatic activities performed Fol1p <br /> served flo11&#226;&#338;&#172; strain dependent Ste12p                 procaryotes, microbial eukaryotes, <br /> (Table 5). Thus data indicate depletion                 plants absent mammals (Figure 2). In most <br /> cellular folate pool triggers activation FLO11 gene                bacteria enzyme activities encoded three <br /> expression Ste12p-dependent manner.                                  distinct ORFs (Slock et al., 1990; Dallas et al., 1992; Talarico et <br />    To assay FLO11 promoter regions affected                 al., 1992; Jardine et al., 2002). However prokaryotes <br /> a fol1&#226;&#338;&#172; strain, used set seven plasmid-born reporter              enzymatic activities HPPK DHNA linked a <br /> constructs containing individual nonoverlapping 400-base                  bifunctional protein, DHPS encoded sepa- <br /> pair FLO11 promoter regions covering 2800 base                  rate ORF (Lopez Lacks, 1993). In bacteria genes <br /> pairs FLO11 promoter cloned CYC1-lacZ                encoding consecutive enzymes biosynthetic pathway <br /> reporter <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (Rupp et al., 1999). Wild-type fol1&#226;&#338;&#172; cells               adjacent close chromosome, <br /> carrying plasmids plated synthetic glucose                suggesting coregulated way (Jardine et <br /> media containing 50 &#226;?&#174;g/ml folinic acid, incubated 24 h,               al., 2002). <br /> and analyzed &#226;?&#164;-gal overlay assay. The full-<span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span>                       In eukaryotes, biochemical analysis folate biosyn- <br /> FLO11 promoter showed small significant increase                 thetic genes characterized far revealed corre- <br /> &#226;?&#164;-galactosidase activity fol1&#226;&#338;&#172; strain compared                sponding proteins multifunctional, containing <br /> with wt confirming GFP measurements (Figure 11, full-                 fused HPPK DHPS activities (Allegra et al., 1990; Triglia <br /> length prom.) Furthermore FLO11 promoter segments                     Cowman, 1994; Rebeille et al., 1997) fused DHNA, <br /> 6/5 12/11 exhibited strong increase &#226;?&#164;-galactosidase              HPPK, DHPS activities (Volpe et al., 1992, 1995; this <br /> activity fol1&#226;&#338;&#172; deletion strain compared wild type,            work). Moreover yeasts S. pombe C. albicans ORFs <br /> indicating regions activated absence             exist <span id='am-6' about='Thesaurus:code' typeof='owl:Thing'>code</span> protein homology three <br /> of Fol1p (Figure 11).                                                     enzyme activities (Figure 2). However, fungi like N. <br />                                                                           crassa plant pathogenic fungus Magnaporthe grisea, <br />                                                                           bifunctional HPPK/DHPS protein separated <br /> DISCUSSION <br />                                                                           DHNA activity. The multifunctional nature this <br /> One-carbon transfer reactions mediated folate cofactors                protein appears common fungal plant species; <br /> play essential role variety cellular reactions. Similar         however, subset fungi contain trifunctional <br /> to bacteria plants S. cerevisiae produces folate novo              structure. It unclear fungi plants <br /> from pterin, p-aminobenzoate, glutamate moieties. The                 relocated gene initial fusion <br /> yeast folate-synthesis pathway completely under-                   events occurred trifunctional fusion <br /> stood. In work shown S. cerevisiae FOL1             occurred later evolution subset fungal species. <br /> gene encodes essential enzymatic activities DHNA,                     Whether enzymatic activities come several <br />  <br />  <br />                                                                                           Figure 11. The FLO11 promoter activated in <br />                                                                                           fol1&#226;&#338;&#172; strains. Wild-type (wt) fol1&#226;&#338;&#172; (fol1&#226;&#338;&#172;) cells <br />                                                                                           transformed CYC::lacZ reporter con- <br />                                                                                           structs carrying individual 400-base pair-long <br />                                                                                           FLO11 promoter elements (2/1&#226;&#8364;&#8220;14/13), full- <br />                                                                                           length promoter (wt p.) promoter (no insert; <br />                                                                                           Rupp et al., 1999). Boundaries promoter fragments <br /> are given relative FLO11 translational start site &#226;&#171;&#185; 1 (2/1: &#226;&#171;&#186;421 &#226;&#171;&#186;1; 3/4: &#226;&#171;&#186;820 &#226;&#171;&#186;381; 6/5: &#226;&#171;&#186;1220 &#226;&#171;&#186;779; 8/7: &#226;&#171;&#186;1620 &#226;&#8364;&#8220;1182; 10/9: <br /> &#226;&#171;&#186;2020 &#226;&#171;&#186;1573; 12/11: &#226;&#171;&#186;2420 to&#226;&#171;&#186;1981; 14/13: &#226;&#171;&#186;2820 &#226;&#171;&#186;2381). Strains pregrown synthetic medium uracil plus 200 &#226;?&#174;g/ml <br /> folinic acid 30&#194;&#176;C o/n. Cells spotted SD plates amino acids containing 50 &#226;?&#174;g/ml folinic acid incubated 25 h at <br /> 30&#194;&#176;C. For overlay assay X-gal&#226;&#8364;&#8220; containing agarose added spots incubated 37&#194;&#176;C 24 h. Background strains <br /> is &#226;&#338;&#186;1278b. <br />  <br />  <br />  <br /> 3824                                                                                                            Molecular Biology Cell <br />                                                                                             Yeast C1 Carrier Filamentous Growth <br />  <br />  <br /> genes affects regulation gene expression         Alternatively, growth FOL1 overexpressing strains could <br /> localization enzymes (Figure 2).                              hampered increased levels 7,8-dihydrop- <br />    Very recently folic acid biosynthetic enzymes           teroate toxic yeast cells (Bayly Macreadie, 2002). <br /> used rooting eukaryotic tree. Stechmann Cava- <br /> lier-Smith (2002) used derived gene fusion dihy-           Fol1p Is Localized Mitochondrial Membrane <br /> drofolate reductase thymidylate synthase order             System <br /> locate root eukaryotic tree bikonts           In vivo localization Fol1 fusion protein showed similar <br /> opisthokonts. However position Amoebozoa                  pattern observed mitochondria (Figure 5). Electron <br /> needs clarified root precisely pin-          microcopy confirmed mitochondrial localization a <br /> pointed. Thus analysis folic acid biosynthesis par-        GFP-Fol1 fusion protein, mitochondrial mem- <br /> ticular DHNA, HPPK, DHPS activities Amoebo-               branes labeled nearly exclusively gold particles. <br /> zoa initiate studies regard                However, overexpression GFP-Fol1p gave <br /> identification early-branching eukaryotic lineages.               rise clumped mitochondria, unable make a <br />                                                                      distinction inner outer membrane localization. <br /> Fol1p Is Essential Growth                                        The localization Fol1p analyzed EM studies <br /> Our data FOL1 gene essential S.             using anti-Fol1p peptide antibodies predominant <br /> cerevisiae vegetative life cycle, fol1&#226;&#338;&#172; strains      staining mitochondrial membranes observed (Fig- <br /> kept alive addition folinic acid. This       ure 6d). <br /> shown partial deletion just C-terminal         What implications mitochondrial mem- <br /> FOL1 gene encoding DHPS activity (Bayly et al., 2001).           brane-associated localization Fol1p central multi- <br /> Similar results obtained FOL2 (encoding GTP-           functional component folate biosynthetic machinery <br /> cyclohydrolase I) FOL3 (encoding dihydrofolate syn-              production folate S. cerevisiae? In yeast cytosol <br /> thetase; Nardese et al., 1996; Cherest et al., 2000). Folinic acid   mitochondria contain folate coenzymes both <br /> is folate derivative, supplements C1 carrier pool        compartments posses parallel array enzymes catalyzing <br /> at site distinct DHF (Holmes Appling, 2002). We           interconversion folate units different oxidation <br /> have tested supplementation fol1&#226;&#338;&#172; strains             states (Appling, 1991; McNeil et al., 1996). Based data, <br /> with folic acid restores growth reported           speculated yeast biosynthesis of <br /> folate-minus strains (Cherest et al., 2000; Bayly et al., 2001;      folate associated mitochondria 7,8-dihy- <br /> Bayly Macreadie, 2002). That intracellular folate            dropteroate (no &#226;?&#165;-glutamate residue) and/or DHF/THF (1 <br /> pool limiting growth shown fact            &#226;?&#165;-glutamate residue; Figure 1) transported the <br /> that growth rate fol1&#226;&#338;&#172; strain dependent        mitochondrial matrix cytosol con- <br /> amount extracellular folinic acid added wt-like          version physiologically relevant oligo-&#226;?&#165;-glutamyl de- <br /> growth observed (Figure 3; unpublished                rivatives occur. Recently reported the <br /> data). Moreover fact high amounts folinic acid       cytosolic mitochondrial folylpolyglutamate <br /> needed indicates yeast cell probably does pos-          synthetase activity encoded MET7 (Cherest et al., 2000; <br /> sess active uptake transport folic acid deriva-        DeSouza et al., 2000). <br /> tives.                                                                  In addition FOL1 essential folate biosynthesis, <br />    The terminal phenotype FOL1 deletion micro-             loss FOL1 impairs growth nonferment- <br /> colony &#226;&#172;&#402;50 slightly elongated cells indicating           able carbon sources (unpublished data). Similarly, has <br /> folate pool spore derived heterozygous diploid         loss dihydrofolate reductase (DHFR) <br /> deletion strain sufficient sporulation process        activity results inability yeast mutant to <br /> the subsequent 5&#226;&#8364;&#8220; 6 cell divisions. This low de-          grow glycerol (Huang et al., 1992). These data suggest <br /> mand folate explained fact carbon         C1 carrier directly indirectly essential mito- <br /> carrier reused consumed. Because             chondrial function. <br /> consumption folate derivatives, assumed that <br /> FOL1 gene expression low. Indeed quantification            Depletion Folate Coezymes Induces Pseudohyphal <br /> FOL1 transcript FOL1 promoter studies             Growth Agar Invasion <br /> (unpublished data) revealed low mRNA level (Planta            The data presented demonstrate growth haploid <br /> et al., 1999). Moreover Fol1p expression regulated wt       fol1&#226;&#338;&#172; cells media supplemented folinic acid results <br /> strains grown presence extracellular folinic acid      filamentous growth ability adhere agar. <br /> FOL1 promoter repressed (see Figure 4C). In addition              The slow-growth phenotype fol1&#226;&#338;&#172; mutant strain in <br /> glucose activates Fol1p expression, stationary           presence high amounts folinic acid suggests the <br /> phase results repression Fol1p expression (Planta et al.,      starvation cells C1 carrier units induces dimor- <br /> 1999). These data indicate complex regulation FOL1              phic switch. Interestingly filamentous growth the <br /> gene expression, suggesting Fol1 protein level           ability fol1&#226;&#338;&#172; cells adhere agar substrate is <br /> adjusted adequately nutrient situation yeast           induced presence high glucose ammonium. <br /> cell.                                                                Thus data indicate starvation C1 carrier a <br />    Interestingly strong overexpression FOL1 toxic           nutritional signal activates filamentous ad- <br /> yeast (Figure 4A) bacteria (unpublished data). Recently          hesive growth. <br /> an inverse relationship reported FOL1                  To test haploid invasive pseudohyphal growth <br /> expression levels growth rate corresponding           phenotype dependent MAP kinase pathway and <br /> yeast strains (Iliades et al., 2003). Increased FOL1 expression      major adhesin Flo11p, constructed fol1&#226;&#338;&#172; ste12&#226;&#338;&#172;, <br /> could result production excess amounts 7,8-dihy-            fol1&#226;&#338;&#172; tec1&#226;&#338;&#172;, fol1&#226;&#338;&#172; flo11&#226;&#338;&#172; double deletion strains. Ste12p <br /> dropteroate depletes GTP pool, needed          transcription factor end MAP kinase <br /> other biosynthetic pathways overproduction             cascade, Tec1p binds called FREs <br /> these folate intermediates inhibits folate-dependent en-             (filamentation invasion response elements) promot- <br /> zymes similar action folinic acid (Girgis et al., 1997).   ers genes, involved dimorphic switch <br />  <br />  <br /> Vol. 15, August 2004                                                                                                           3825 <br />  U. Gueldener et al. <br />  <br />  <br /> (Madhani Fink, 1998b). One FRE element located         flocculation, diploid invasion, diploid filamentation <br /> in promoter cell surface floccullin encoding gene       separated genetically (Palecek et al., 2000). Thus, <br /> FLO11, necessary pseudohyphae formation           regulation FLO11 transcription provides im- <br /> invasion (Lo Dranginis, 1998). The FLO11 promoter              portant mechanism invasive growth control, FLO11 up- <br /> integrates MAP kinase cascade cAMP/PKA fila-               regulation essential invasive growth not <br /> mentation signaling pathways (Pan Heitman, 1999;               induced conditions trigger dimorphic switch. <br /> Rupp et al., 1999). The agar adhesion phenotype observed           One FLO11-independent mechanisms likely to <br /> for fol1&#226;&#338;&#172; cells dependent MAPK-             exist invasive growth (Palecek et al., 2000; Rua et al., <br /> specific transcription factor Ste12p cell wall pro-     2001). It suggested invasive-growth re- <br /> tein Flo11p Tec1p transcription factor. Sur-        sponse triggered different physiological and <br /> prisingly fol1&#226;&#338;&#172; ste12&#226;&#338;&#172;, fol1&#226;&#338;&#172; tec1&#226;&#338;&#172;, fol1&#226;&#338;&#172; flo11&#226;&#338;&#172;          genetic inputs (Breitkreutz et al., 2003). Similarly, data <br /> double deletion strains showed extent filamen-         suggest FLO11-independent mechanism(s) <br /> tous growth fol1&#226;&#338;&#172; single deletion (Figure 8). This          leading fol1&#226;&#338;&#172;-induced pseudohyphal growth. <br /> suggests internal starvation signal created fol1      What signals receptors used to <br /> deletion induces filamentous growth independent Ste12p          activate dimorphic switch fol1&#226;&#338;&#172; strain? Thus far <br /> and Tec1p. In accordance result        receptors required dimorphic switch have <br /> detect increased phosphorylation level MAP ki-           identified. The Mep2p ammonium permease is <br /> nases Kss1p Fus3p, suggesting Kss1p Fus3p         used cells detect nitrogen compounds, whereas <br /> are constitutively activated fol1&#226;&#338;&#172; deletion strain. We    Gpr1p cell surface G-protein&#226;&#8364;&#8220; coupled receptor to <br /> also tested Phd1p transcription factor, thought       sense glucose, mutations protein renders the <br /> to act independent cAMP MAP kinase pathways             cells unable form filaments diploid strains grow <br /> (Palecek et al., 2002) filamentous       invasively haploid strains (Lorenz Heitman, 1998; <br /> invasive growth fol1&#226;&#338;&#172; cells dependent Phd1p.         Lorenz et al., 2000b). Similarly amino acid starvation was <br /> This suggests signaling pathways possibly acting        nutritional signal induces FLO11 gene ex- <br /> through Flo8p proteins relevant filamen-       pression induces adhesive growth GCN4-depen- <br /> tous growth consequence loss C1 carrier         dent manner (Braus et al., 2003). Also, metabolic enzymes are <br /> need tested. Likewise flocullin Flo11p         thought regulate directly indirectly filamentous phe- <br /> not necessary filamentous growth cells deleted        notypes, e.g., mutations NADP-linked glutamate de- <br /> for FOL1. The fact Flo11p required filamentous     hydrogenase GDH3, phosphoglucose isomerase PGI1, <br /> growth suggests mechanisms cell-cell adhesion        alcohol dehydrogenase ADH1 constitutively acti- <br /> and/or cytokinesis relevant filamentous growth         vate invasive growth haploids pseudohyphal growth <br /> phenotype. One candidate Fig 2p mating-            diploids (Wilkinson et al., 1996; Palecek et al., 2000). How- <br /> specific adhesin, overexpressed partially           ever, completely unclear metabolite sensed in <br /> complement loss adhesion FLO11 mutant            fol1&#226;&#338;&#172; cell depleted C1 carrier possible <br /> strain (Guo et al., 2000). Alternatively lack mother-         receptor(s) are. Starvation C1 carrier affects variety of <br /> daughter separation result increased cell-cell ad-       biosynthetic pathways including synthesis purines, <br /> hesion link septal maturation cell        methionine, glycine. Thus likely one <br /> adhesion observed (Pan Heitman, 2000).                signal activated cells starving folate the <br />    In contrast filamentous growth fol1&#226;&#338;&#172; cells adhe-      combined read-out signals results filamentous <br /> sion phenotype fol1&#226;&#338;&#172; mutant strain dependent         adhesive growth seen fol1&#226;&#338;&#172; cells. <br /> Ste12p Flo11p Tec1p. We tested <br /> whether deletion FOL1 modulates FLO11 promoter <br /> activity. Indeed fol1&#226;&#338;&#172; strains FLO11 promoter activity      ACKNOWLEDGMENTS <br /> was enhanced Ste12p-dependent manner comparison            We thank Ursula Fleig extensive discussions, critical reading, editing <br /> to wt strains, demonstrating internal starvation          manuscript Sabine Klein FACS analysis. This work sup- <br /> signal induced depletion C1 carrier able           ported FAZIT-Stiftung contract awarded J.H.H. the <br />                                                                    context EUROFAN project EC (BIO4-CT95-0080). <br /> activate expression important cell-cell adhe- <br /> sin flocculin. More specifically FLO11 promoter seg- <br /> ments &#226;&#171;&#186;800 &#226;&#171;&#186;1200 &#226;&#171;&#186;2000 &#226;&#171;&#186;2400 induced             REFERENCES <br /> deletion FOL1. Interestingly, reported that <br /> Ste12p acts segments FLO11 promoter             Allegra, C.J., Boarman, D., Kovacs, J.A., Morrison, P., Beaver, J., Chabner, <br />                                                                    B.A., Masur, H. (1990). Interaction sulfonamide sulfone com- <br /> (Rupp et al., 1999). However, Ste12p controls segment      pounds Toxoplasma gondii dihydropteroate synthase. J. Clin. Invest. 85, <br /> of FLO11 promoter located position &#226;&#171;&#186;1600 &#226;&#171;&#186;2000,          371&#226;&#8364;&#8220;379. <br /> which activated fol1&#226;&#338;&#172; strain. Thus data           Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., <br /> imply activation FLO11 expression deletion           Lipman, D.J. (1997). Gapped BLAST PSI-BLAST: new generation of <br /> FOL1 involves FLO11 promoter ele-             protein database search programs. Nucleic Acids Res. 25, 3389 &#226;&#8364;&#8220;3402. <br /> ments regulated Ste12p pathway.                             Ames, B.N., Gold, L.S., Willett, W.C. (1995). The causes prevention of <br />    Our data deletion FOL1 gene results         cancer. Proc. Natl. Acad. Sci. USA 92, 5258 &#226;&#8364;&#8220;5265. <br /> Ste12p- Flo11p-dependent Tec1p- Phd1p-inde-            Appling, D.R. (1991). Compartmentation folate-mediated one-carbon me- <br /> pendent adhesive growth, filamentous growth             tabolism eukaryotes. FASEB J. 5, 2645&#226;&#8364;&#8220;2651. <br /> the fol1&#226;&#338;&#172; strain requires Ste12p, Tec1, Phd1p,         Barclay, B.J., Huang, T., Nagel, M.G., Misener, V.L., Game, J.C., Wahl, <br /> Flo11p. This behavior fol1&#226;&#338;&#172; strain suggests agar       G.M. (1988). Mapping sequencing dihydrofolate reductase gene <br /> invasion filamentation brought different          (DFR1) Saccharomyces cerevisiae. Gene 63, 175&#226;&#8364;&#8220;185. <br /> signals. Recently screen designed mutants show-       Bateman, A. et al. (2002). The Pfam protein families database. Nucleic Acids <br /> ing increased invasiveness, strains identified pro-      Res. 30, 276 &#226;&#8364;&#8220;280. <br /> mote agar invasion FLO11 expressed              Bayly, A.M., Berglez, J.M., Patel, O., Castelli, L.A., Hankins, E.G., Coloe, P., <br /> constitutively low level. Specifically, haploid agar invasion,     Hopkins Sibley, C., Macreadie, I.G. (2001). Folic acid utilisation related to <br />  <br />  <br />  <br /> 3826                                                                                                            Molecular Biology Cell <br />                                                                                                                       Yeast C1 Carrier Filamentous Growth <br />  <br /> sulfa drug resistance Saccharomyces cerevisiae. FEMS Microbiol. Lett. 204,           Griffiths, G., McDowall, A., Back, R., Dubochet, J. (1984). On prepa- <br /> 387&#226;&#8364;&#8220;390.                                                                                ration cryosections immunocytochemistry. J. Ultrastruct. Res. 89, 65&#226;&#8364;&#8220;78. <br /> Bayly, A.M., Macreadie, I.G. (2002). Cytotoxicity dihydropteroate             Gueldener, U., Heinisch, J., Koehler, G.J., Voss, D., Hegemann, J.H. (2002). <br /> Saccharomyces cerevisiae. FEMS Microbiol. Lett. 213, 189 &#226;&#8364;&#8220;192.                          A second set loxP marker cassettes Cre-mediated multiple gene knock- <br />                                                                                         outs budding yeast. Nucleic Acids Res. 30, e23 <br /> Blakley, R.L., Benkovic, S.J. (1984). Folates pterins. Chemistry and <br /> biochemistry folates, Vol. 1, New York: John Wiley &amp; Sons.                           Gu&#194;&#168; ldener, U., Heck, S., Fiedler, T., Beinhauer, J.D., Hegemann, J.H. (1996). <br />                                                                                         A new efficient gene disruption cassette repeated use budding yeast. <br /> Blount, B.C., Mack, M.M., Wehr, C.M., MacGregor, J.T., Hiatt, R.A., Wang, G.,           Nucleic Acids Res. 24, 2519 &#226;&#8364;&#8220;2524. <br /> Wickramasinghe, S.N., Everson, R.B., Ames, B.N. (1997). Folate deficiency <br /> causes uracil misincorporation human DNA chromosome breakage:                  Guo, B., Styles, C.A., Feng, Q., Fink, G.R. (2000). A Saccharomyces gene <br /> implications cancer neuronal damage. Proc. Natl. Acad. Sci. USA 94,             family involved invasive growth, cell-cell adhesion, mating. Proc. Natl. <br /> 3290 &#226;&#8364;&#8220;3295.                                                                             Acad. Sci. USA 97, 12158 &#226;&#8364;&#8220;12163. <br /> Boeke, J.D., LaCroute, F., Fink, G.R. (1984). A positive selection              Haussmann, C., Rohdich, F., Schmidt, E., Bacher, A., Richter, G. (1998). <br /> mutants lacking orotidine-5&#226;&#172;&#732;-phosphate decarboxylase activity yeast:                 Biosynthesis pteridines Escherichia coli. Structural mechanistic sim- <br /> 5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197, 345&#226;&#8364;&#8220;346.                         ilarity dihydroneopterin-triphosphate epimerase dihydroneopterin <br />                                                                                         aldolase. J. Biol. Chem. 273, 17418 &#226;&#8364;&#8220;17424. <br /> Borges-Walmsley, M.I., Walmsley, A.R. (2000). cAMP signalling patho- <br /> genic fungi: control dimorphic switching pathogenicity. Trends Mi-               Holmes, W.B., Appling, D.R. (2002). Cloning characterization of <br /> crobiol. 8, 133&#226;&#8364;&#8220;141.                                                                    methenyltetrahydrofolate synthetase Saccharomyces cerevisiae. J. Biol. <br />                                                                                         Chem. 277, 20205&#226;&#8364;&#8220;20213. <br /> Braus, G.H., Grundmann, O., Bru&#194;&#168; ckner, S., Mo&#194;&#168; sch, H.-U. (2003). Amino <br /> acid starvation Gcn5p regulate adhesive growth FLO11 gene expres-               Huang, T., Barclay, B.J., Kalman, T.I., von Borstel, R.C., Hastings, P.J. <br /> sion Saccharomyces cerevisiae. Mol. Biol. Cell 14, 4272&#226;&#8364;&#8220; 4284.                       (1992). The phenotype dihydrofolate reductase mutant Saccharomyces <br />                                                                                         cerevisiae. Gene 121, 167&#226;&#8364;&#8220;171. <br /> Breitkreutz, A., Boucher, L., Breitkreutz, B.J., Sultan, M., Jurisica, I., Tyers, <br /> M. (2003). Phenotypic transcriptional plasticity directed yeast mito-          Huxley, C., Green, E.D., Dunham, I. (1990). Rapid assessment S. <br /> gen-activated protein kinase network. Genetics 165, 997&#226;&#8364;&#8220;1015.                           cerevisiae mating type PCR. Trends Genet. 6, 236 <br />  <br /> Cali, B.M., Doyle, T.C., Botstein, D., Fink, G.R. (1998). Multiple functions        Iliades, P., Berglez, J., Meshnick, S., Macreadie, I. (2003). Promoter <br /> for actin filamentous growth Saccharomyces cerevisiae. Mol. Biol. Cell        strength folic acid synthesis genes affects sulfa drug resistance Saccha- <br /> 9, 1873&#226;&#8364;&#8220;1889.                                                                           romyces cerevisiae. Microb. Drug Resist. 9, 249 &#226;&#8364;&#8220;255. <br />  <br /> Chant, J. (1999). Cell polarity yeast. Annu. Rev. Cell Dev. Biol. 15, 365&#226;&#8364;&#8220;391.       Jardine, O., Gough, J., Chothia, C., Teichmann, S.A. (2002). Comparison of <br />                                                                                         small molecule metabolic enzymes Escherichia coli Saccharomyces <br /> Cherest, H., Thomas, D., Surdin-Kerjan, Y. (2000). Polyglutamylation             cerevisiae. Genome Res. 12, 916 &#226;&#8364;&#8220;929. <br /> folate coenzymes necessary methionine biosynthesis maintenance <br /> of intact mitochondrial genome Saccharomyces cerevisiae. J. Biol. Chem. 275,         Kaneko, Y., Hayashi, N., Toh-e, A., Banno, I., Oshima, Y. (1987). Struc- <br /> 14056 &#226;&#8364;&#8220;14063.                                                                           tural characteristics PHO8 gene encoding repressible alkaline phospha- <br />                                                                                         tase Saccharomyces cerevisiae. Gene 58, 137&#226;&#8364;&#8220;148. <br /> Cormack, B.P., Bertram, G., Egerton, M., Gow, N.A., Falkow, S., Brown, <br /> A.J. (1997). Yeast-enhanced green fluorescent protein (yEGFP)a reporter              Kron, S.J., Styles, C.A., Fink, G.R. (1994). Symmetric cell division in <br /> gene expression Candida albicans. Microbiology 143, 303&#226;&#8364;&#8220;311.                         pseudohyphae yeast Saccharomyces cerevisiae. Mol. Biol. Cell 5, 1003&#226;&#8364;&#8220; <br />                                                                                         1022. <br /> Cossins, E.A., Chen, L. (1997). Folates one-carbon metabolism in <br />                                                                                         Lagosky, P.A., Taylor, G.R., Haynes, R.H. (1987). Molecular characteriza- <br /> plants fungi. Phytochemistry 45, 437&#226;&#8364;&#8220; 452. <br />                                                                                         tion Saccharomyces cerevisiae dihydrofolate reductase gene (DFR1). Nu- <br /> Cullen, P.J., Sprague, G.F., Jr. (2000). Glucose depletion causes haploid           cleic Acids Res. 15, 10355&#226;&#8364;&#8220;10371. <br /> invasive growth yeast. Proc. Natl. Acad. Sci. USA 97, 13619 &#226;&#8364;&#8220;13624. <br />                                                                                         Lambrechts, M.G., Bauer, F.F., Marmur, J., Pretorius, I.S. (1996). Muc1, a <br /> Dallas, W.S., Gowen, J.E., Ray, P.H., Cox, M.J., Dev, I.K. (1992). Cloning,         mucin-like protein regulated Mss10, critical pseudohyphal <br /> sequencing, enhanced expression dihydropteroate synthase gene             differentiation yeast. Proc. Natl. Acad. Sci. USA 93, 8419 &#226;&#8364;&#8220; 8424. <br /> Escherichia coli MC4100. J. Bacteriol. 174, 5961&#226;&#8364;&#8220;5970. <br />                                                                                         Lengeler, K.B., Davidson, R.C., D&#226;&#8364;&#8482;Souza, C., Harashima, T., Shen, W.C., <br /> DeSouza, L., Shen, Y., Bognar, A.L. (2000). Disruption cytoplasmic           Wang, P., Pan, X., Waugh, M., Heitman, J. (2000). Signal transduction <br /> mitochondrial folylpolyglutamate synthetase activity Saccharomyces cerevi-           cascades regulating fungal development virulence. Microbiol. Mol. Biol. <br /> siae. Arch. Biochem. Biophys. 376, 299 &#226;&#8364;&#8220;312.                                            Rev. 64, 746 &#226;&#8364;&#8220;785. <br /> Dickinson, J.R. (1996). &#226;&#8364;&#732;Fusel&#226;&#8364;&#8482; alcohols induce hyphal-like extensions              Little, J.G., Haynes, R.H. (1979). Isolation characterization yeast <br /> pseudohyphal formation yeasts. Microbiology 142, 1391&#226;&#8364;&#8220;1397.                          mutants auxotrophic 2&#226;&#172;&#732;-deoxythymidine 5&#226;&#172;&#732;-monophosphate. Mol. Gen. <br />                                                                                         Genet. 168, 141&#226;&#8364;&#8220;151. <br /> D&#226;&#8364;&#8482;Souza, C.A., Heitman, J. (2001). Conserved cAMP signaling cascades <br /> regulate fungal development virulence. FEMS Microbiol. Rev. 25, 349 &#226;&#8364;&#8220;               Liu, H., Styles, C.A., Fink, G.R. (1993). Elements yeast pheromone <br /> 364.                                                                                    response pathway required filamentous growth diploids. Science 262, <br />                                                                                         1741&#226;&#8364;&#8220;1744. <br /> Duesterhoeft, A., Klonierung von Restriktions-Modifikationssystemen aus <br /> Herpetosiphon giganteus. PhD thesis. Justus-Liebig-University, Giessen, 1990.           Lo, W.S., Dranginis, A.M. (1998). The cell surface flocculin Flo11 is <br />                                                                                         required pseudohyphae formation invasion Saccharomyces cerevi- <br /> Duthie, S.J. (1999). Folic acid deficiency cancer: mechanisms DNA                siae. Mol. Biol. Cell 9, 161&#226;&#8364;&#8220;171. <br /> instability. Br. Med. Bull. 55, 578 &#226;&#8364;&#8220;592. <br />                                                                                         Lopez, P., Lacks, S.A. (1993). A bifunctional protein folate biosyn- <br /> Entian, K.-D., Ko&#194;&#168; tter, P. (1998). Yeast mutant plasmid collections. In:       thetic pathway Streptococcus pneumoniae dihydroneopterin aldolase <br /> Methods Microbiology, Vol. 26, ed. A.J.P. Brown, M. Tuite, San Diego:                hydroxymethyldihydropterin pyrophosphokinase activities. J. Bacteriol. <br /> Academic Press, 431&#226;&#8364;&#8220; 450.                                                               175, 2214 &#226;&#8364;&#8220;2220. <br /> Gavrias, V., Andrianopoulos, A., Gimeno, C.J., Timberlake, W.E. (1996).             Lorenz, M.C., Cutler, N.S., Heitman, J. (2000a). Characterization alco- <br /> Saccharomyces cerevisiae TEC1 required pseudohyphal growth. Mol.                 hol-induced filamentous growth Saccharomyces cerevisiae. Mol. Biol. Cell 11, <br /> Microbiol. 19, 1255&#226;&#8364;&#8220;1263.                                                               183&#226;&#8364;&#8220;199. <br /> Gimeno, C.J., Fink, G.R. (1994). Induction pseudohyphal growth                Lorenz, M.C., Heitman, J. (1998). The MEP2 ammonium permease regu- <br /> overexpression PHD1, Saccharomyces cerevisiae gene related transcrip-           lates pseudohyphal differentiation Saccharomyces cerevisiae. EMBO J. 17, <br /> tional regulators fungal development. Mol. Cell. Biol. 14, 2100 &#226;&#8364;&#8220;2112.               1236 &#226;&#8364;&#8220;1247. <br /> Gimeno, C.J., Ljungdahl, P.O., Styles, C.A., Fink, G.R. (1992). Unipolar            Lorenz, M.C., Pan, X., Harashima, T., Cardenas, M.E., Xue, Y., Hirsch, J.P., <br /> cell divisions yeast S. cerevisiae lead filamentous growth: regulation        Heitman, J. (2000b). The G protein-coupled receptor gpr1 nutrient <br /> by starvation RAS. Cell 68, 1077&#226;&#8364;&#8220;1090.                                              sensor regulates pseudohyphal differentiation Saccharomyces cerevisiae. <br />                                                                                         Genetics 154, 609 &#226;&#8364;&#8220; 622. <br /> Girgis, S., Suh, J.R., Jolivet, J., Stover, P.J. (1997). 5-Formyltetrahydrofolate <br /> regulates homocysteine remethylation human neuroblastoma. J. Biol.                   Madden, K., Snyder, M. (1998). Cell polarity morphogenesis in <br /> Chem. 272, 4729 &#226;&#8364;&#8220; 4734.                                                                 budding yeast. Annu. Rev. Microbiol. 52, 687&#226;&#8364;&#8220;744. <br /> Greene, N.D., Copp, A.J. (1997). Inositol prevents folate-resistant neural          Madhani, H.D., Fink, G.R. (1997). Combinatorial control required the <br /> tube defects mouse [see comments]. Nat. Med. 3, 60 &#226;&#8364;&#8220; 66.                         specificity yeast MAPK signaling. Science 275, 1314 &#226;&#8364;&#8220;1317. <br />  <br />  <br />  <br /> Vol. 15, August 2004                                                                                                                                                3827 <br />  U. Gueldener et al. <br />  <br /> Madhani, H.D., Fink, G.R. (1998a). The control filamentous differenti-        Sabbagh, W., Jr., Flatauer, L.J., Bardwell, A.J., Bardwell, L. (2001). Spec- <br /> ation virulence fungi. Trends Cell Biol. 8, 348 &#226;&#8364;&#8220;353.                         ificity MAP kinase signaling yeast differentiation involves transient <br />                                                                                      versus sustained MAPK activation. Mol. Cell 8, 683&#226;&#8364;&#8220; 691. <br /> Madhani, H.D., Fink, G.R. (1998b). The riddle MAP kinase signaling <br /> specificity. Trends Genet. 14, 151&#226;&#8364;&#8220;155.                                              Sanchez-Martinez, C., Perez-Martin, J. (2001). Dimorphism fungal <br />                                                                                      pathogens: Candida albicans Ustilago maydis&#226;&#8364;&#8220;similar inputs, different out- <br /> Madhani, H.D., Styles, C.A., Fink, G.R. (1997). MAP kinases distinct        puts. Curr. Opin. Microbiol. 4, 214 &#226;&#8364;&#8220;221. <br /> inhibitory functions impart signaling specificity yeast differentiation. <br /> Cell 91, 673&#226;&#8364;&#8220; 684.                                                                   Sen-Gupta, M., Gu&#194;&#168; ldener, U., Beinhauer, J., Fiedler, T., Hegemann, J.H. <br />                                                                                      (1997). Sequence analysis 33 kb long region ORC5 SUI1 <br /> McNeil, J.B., Bognar, A.L., Pearlman, R.E. (1996). In vivo analysis folate    left arm chromosome XIV Saccharomyces cerevisiae. Yeast 13, <br /> coenzymes compartmentation Saccharomyces cerevisiae. Genetics           849 &#226;&#8364;&#8220; 860. <br /> 142, 371&#226;&#8364;&#8220;381. <br />                                                                                      Sherman, F. (1991). Getting started yeast. Methods Enzymol. 194, 3&#226;&#8364;&#8220;21. <br /> Mumberg, D., Muller, R., Funk, M. (1994). Regulatable promoters of <br /> Saccharomyces cerevisiae: comparison transcriptional activity use       Sherman, F., Hicks, J. (1991). Micromanipulation dissection asci. <br /> for heterologous expression. Nucleic Acids Res. 22, 5767&#226;&#8364;&#8220;5768.                       Methods Enzymol. 194, 21&#226;&#8364;&#8220;37. <br /> Mumberg, D., Muller, R., Funk, M. (1995). Yeast vectors controlled       Slock, J., Stahly, D.P., Han, C.Y., Six, E.W., Crawford, I.P. (1990). An <br /> expression heterologous proteins different genetic backgrounds. Gene           apparent Bacillus subtilis folic acid biosynthetic operon containing pab, an <br /> 156, 119 &#226;&#8364;&#8220;122.                                                                       amphibolic trpG gene, gene required synthesis para-aminoben- <br />                                                                                      zoic acid, dihydropteroate synthase gene. J. Bacteriol. 172, 7211&#226;&#8364;&#8220;7226. <br /> Nardese, V., Gu&#194;&#168; tlich, M., Brambilla, A., Carbone, M.L. (1996). Disruption <br /> of GTP-cyclohydrolase I gene Saccharomyces cerevisiae. Biochem. Bio-          Stechmann, A., Cavalier-Smith, T. (2002). Rooting eukaryote tree by <br /> phys. Res. Commun. 218, 273&#226;&#8364;&#8220;279.                                                     using derived gene fusion. Science 297, 89 &#226;&#8364;&#8220;91. <br />  <br /> Niedenthal, R., Riles, L., Gu&#194;&#168; ldener, U., Klein, S., Johnston, M., Hegemann,    Suckow, M., Hollenberg, C.P. (1998). The activation specificities wild- <br /> J.H. (1999). Systematic analysis S. cerevisiae chromosome VIII genes. Yeast       type mutant Gcn4p vivo different DNA binding <br /> 15, 1775&#226;&#8364;&#8220;1796.                                                                       specificities corresponding bZip peptides vitro. J. Mol Biol. 276, <br />                                                                                      887&#226;&#8364;&#8220;902. <br /> Niedenthal, R.K., Riles, L., Johnston, M., Hegemann, J.H. (1996). Green <br /> fluorescent protein marker gene expression subcellular localiza-        Talarico, T.L., Ray, P.H., Dev, I.K., Merrill, B.M., Dallas, W.S. (1992). <br /> tion budding yeast. Yeast 12(8), 773&#226;&#8364;&#8220;786.                                         Cloning, sequence analysis, overexpression Escherichia coli folK, the <br />                                                                                      gene coding 7,8-dihydro-6-hydroxymethylpterin- pyrophosphokinase. J. <br /> Palecek, S.P., Parikh, A.S., Kron, S.J. (2000). Genetic analysis reveals    Bacteriol. 174, 5971&#226;&#8364;&#8220;5977. <br /> FLO11 upregulation cell polarization independently regulate invasive <br /> growth Saccharomyces cerevisiae. Genetics 156, 1005&#226;&#8364;&#8220;1023.                         Tamaki, H., Miwa, T., Shinozaki, M., Saito, M., Yun, C.W., Yamamoto, K., and <br />                                                                                      Kumagai, H. (2000). GPR1 regulates filamentous growth FLO11 in <br /> Palecek, S.P., Parikh, A.S., Kron, S.J. (2002). Sensing, signalling          yeast Saccharomyces cerevisiae. Biochem. Biophys. Res. Commun. 267, 164 &#226;&#8364;&#8220;168. <br /> integrating physical processes Saccharomyces cerevisiae invasive and <br />                                                                                      Thompson, J.D., Higgins, D.G., Gibson, T.J. (1994). CLUSTAL W: improv- <br /> filamentous growth. Microbiology 148, 893&#226;&#8364;&#8220;907. <br />                                                                                      ing sensitivity progressive multiple sequence alignment se- <br /> Pan, X., Heitman, J. (1999). Cyclic AMP-dependent protein kinase regu-           quence weighting, position-specific gap penalties weight matrix choice. <br /> lates pseudohyphal differentiation Saccharomyces cerevisiae. Mol. Cell. Biol.     Nucleic Acids Res. 22, 4673&#226;&#8364;&#8220; 4680. <br /> 19, 4874 &#226;&#8364;&#8220; 4887. <br />                                                                                      Tokuyasu, K.T. (1989). Use poly(vinylpyrrolidone) poly(vinyl alcohol) <br /> Pan, X., Harashima, T., Heitman, J. (2000). Signal transduction cascades         cryoultramicrotomy. Histochem. J. 21, 163&#226;&#8364;&#8220;171. <br /> regulating pseudohyphal differentiation Saccharomyces cerevisiae. Curr.           Triglia, T., Cowman, A.F. (1994). Primary structure expression the <br /> Opin. Microbiol. 3, 567&#226;&#8364;&#8220;572.                                                         dihydropteroate synthetase gene Plasmodium falciparum. Proc. Natl. Acad. <br /> Pan, X., Heitman, J. (2000). Sok2 regulates yeast pseudohyphal differen-         Sci. USA 91, 7149 &#226;&#8364;&#8220;7153. <br /> tiation transcription factor cascade regulates cell-cell adhesion. Mol.   Volpe, F., Ballantine, S.P., Delves, C.J. (1993). The multifunctional folic <br /> Cell. Biol. 20, 8364 &#226;&#8364;&#8220; 8372.                                                         acid synthesis fas gene Pneumocystis carinii encodes dihydroneopterin <br /> Pearson, W.R. (1994). Using FASTA program search protein DNA              aldolase, hydroxymethyldihydropterin pyrophosphokinase dihydrop- <br /> sequence databases. Methods Mol. Biol. 24, 307&#226;&#8364;&#8220;331.                                  teroate synthase. Eur. J. Biochem 216, 449 &#226;&#8364;&#8220; 458. <br /> Planta, R.J. et al. (1999). Transcript analysis 250 novel yeast genes        Volpe, F., Ballantine, S.P., Delves, C.J. (1995). Two domains amino- <br /> chromosome XIV. Yeast 15, 329 &#226;&#8364;&#8220;350.                                                  acid sequence similarity required dihydroneopterin aldolase function <br />                                                                                      multifunctional folic acid synthesis Fas protein Pneumocystis carinii. <br /> Rebeille, F., Macherel, D., Mouillon, J.M., Garin, J., Douce, R. (1997).         Gene 160, 41&#226;&#8364;&#8220; 46. <br /> Folate biosynthesis higher plants: purification molecular cloning a <br /> bifunctional 6-hydroxymethyl-7,8-dihydropterin pyrophosphokinase/7,8-di-             Volpe, F., Dyer, M., Scaife, J.G., Darby, G., Stammers, D.K., Delves, C.J. <br /> hydropteroate synthase localized mitochondria. EMBO J. 16, 947&#226;&#8364;&#8220;957.               (1992). The multifunctional folic acid synthesis fas gene Pneumocystis carini <br /> </body></html>