<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Zebrafish ambra1a ambra1b Knockdown Impairs <br /> Skeletal Muscle Development <br /> Tatjana Skobo1, Francesca Benato1, Paolo Grumati2, Giacomo Meneghetti1, Valentina Cianfanelli3, <br /> Silvia Castagnaro2, Martina Chrisam2, Sabrina Di Bartolomeo3, Paolo Bonaldo2, Francesco Cecconi3,4,5*, <br /> Luisa Dalla Valle1* <br /> 1 Department Biology, University Padova, Padova, Italy, 2 Department Molecular Medicine, University Padova, Padova, Italy, 3 Department Biology, University <br /> of Tor Vergata, Rome, Italy, 4 Department Neuroscience, Istituto di Ricovero e Cura Carattere Scientifico &#226;&#8364;&#732;&#226;&#8364;&#732;Santa Lucia Foundation&#226;&#8364;&#8482;&#226;&#8364;&#8482;, Rome, Italy, 5 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> Cell Stress <br /> and <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>Survival</span>, Danish Cancer Society Research Center, Copenhagen, Denmark <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The essential role autophagy muscle homeostasis clearly demonstrated <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> analysis mice with <br />      muscle-specific inactivation genes encoding autophagy-related proteins. Ambra1 key component Beclin 1 <br />      complex and, zebrafish, encoded paralogous genes, ambra1a ambra1b, required normal <br />      embryogenesis larval development. In study focused function Ambra1, positive regulator the <br />      autophagic process, skeletal muscle development means morpholino (MO)-mediated knockdown and <br />      compared phenotype zebrafish Ambra1-depleted embryos Ambra1gt/gt mouse embryos. Morphological <br />      analysis zebrafish morphant embryos revealed silencing ambra1 impairs locomotor activity muscle <br />      development, myoD1 expression. Skeletal muscles ATG-morphant embryos displayed severe histopathological <br />      changes contained small areas organized myofibrils widely dispersed cell. Double <br />      knockdown ambra1a ambra1b resulted severe phenotype defects evident in <br />      splice-morphants. The morphants phenotypes effectively rescued co-injection human AMBRA1 mRNA. <br />      Together, results indicate ambra1a ambra1b required correct development morphogenesis of <br />      skeletal muscle. <br />  <br />   Citation: Skobo T, Benato F, Grumati P, Meneghetti G, Cianfanelli V, et al. (2014) Zebrafish ambra1a ambra1b Knockdown Impairs Skeletal Muscle <br />   Development. PLoS ONE 9(6): e99210. doi:10.1371/journal.pone.0099210 <br />   Editor: Zhiyuan Gong, National University Singapore, Singapore <br />   Received January 9, 2014; Accepted May 12, 2014; Published June 12, 2014 <br />   Copyright: &#195;&#376; 2014 Skobo et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work financially supported Telethon-Italy (Grant GGP10225) University Padova (RFO-ex 60%). FC supported AIRC <br />   (IG2010), FISM (2009), Italian Ministry University Research (PRIN 2009 FIRB Accordi di Programma 2011), Italian Ministry Health (Ricerca <br />   Finalizzata Ricerca Corrente). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: luisa.dallavalle@unipd.it (LDV); francesco.cecconi@uniroma2.it (FC) <br />  <br />  <br />  <br /> Introduction                                                                             rated dystrophic phenotype collagen VI null mice. More <br />                                                                                          recently, deregulation autophagic process also <br />    Autophagy evolutionarily conserved catabolic process                         demonstrated dystrophic mouse models [6&#226;&#8364;&#8220;9] as <br /> which cells, lysosomal machinery, degrade recycle                        Vici syndrome, human genetic disease caused recessive <br /> long-lived proteins dismantle organelles order maintain                      mutations EPG5 gene, codes key autophagy <br /> homeostatic intracellular environment. This process tightly                           regulator involved formation autolysosomes [10]. <br /> regulated plays important roles normal physiology,                           Ambra1, originally identified gene trap screening mice, is <br /> differentiation, embryo development, <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>cell survival</span>                            positive regulator Beclin 1 dependent programme of <br /> starvation [1]. Defects degradative play role                        autophagy [11]. Ambra1 intrinsically disordered protein, <br /> various diseases, neurodegenerative lysosomal storage                        capability binding number regulatory partners <br /> disorders oncogenesis cancer progression [2].                                 involved cell processes highlights crucial role a <br /> However, little known autophagy muscular pathology.                          &#226;&#8364;&#732;&#226;&#8364;&#732;relay&#226;&#8364;&#8482;&#226;&#8364;&#8482; molecule autophagy [12]. In mammalian cells, <br />    In skeletal muscle, role autophagy initially                               Ambra1 normally docked specific cytoskeletal site, <br /> demonstrated Atg5 Atg7 muscle-specific <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> mice                              corresponding dynein light chain, unleashed <br /> [3,4]. In models, muscle showed abnormal mitochondria                           autophagy induction translocate autophagosome <br /> and disorganized sarcomeres, confirming homeostatic role                            origin sites endoplasmic reticulum [13]. Ablation of <br /> autophagy tissue. A direct connection autophagy                          Ambra1, demonstrated loss-of-function mice, leads <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> deregulation muscular dystrophy initially                               embryonic lethality causes neural defects, suggesting role for <br /> collagen VI null mice, accumulation abnormal organelles                         autophagy nervous development [11]. Results obtained <br /> and spontaneous apoptosis shown strictly depend                                zebrafish model confirmed involvement protein <br /> defective autophagy regulation [5]. In agreement this,                              embryonic development demonstrated duplicated <br /> reactivation autophagy restored myofiber survival amelio-                         fish ambra1 paralogous genes required normal embryogen- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                           1                                   June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                 Ambra1 Muscle Development <br />  <br />  <br /> esis larval development. Indeed, MO-induced ablation           pCS2+ vector. Full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> RNA transcribed using the <br /> corresponding proteins associated            T3 promoter mMessage Machine kit (Ambion) according <br /> developmental abnormalities decreased viability [14].                 manufacturer&#226;&#8364;&#8482;s instructions plasmid linearization <br />    The rapid development transparency zebrafish embryos,           HindIII restriction enzyme. After preliminary experiments <br /> together high fecundity amenability genetic               different dosages 40 10 ng/embryo, 10 ng/ <br /> manipulation vertebrate model, feature        embryo dosage selected injection human AMBRA1 <br /> that skeletal muscles represent large portion body       RNA one-cell stage embryos rescue experiments. <br /> easily accessible analysis, organism attractive <br /> for investigating muscle development fiber-type specification         Birefringence assay <br /> (reviewed [15]) myopathies muscular dystrophies           Muscle birefringence analysed placing anesthetized <br /> (reviewed [16]). In study, investigated role            embryos glass polarizing filter covering second <br /> Ambra1 skeletal muscle development means knockdown               polarizing filter Leica DMR microscope. Embryos were <br /> of ambra1 paralogous genes zebrafish. Depletion zebrafish           photographed Leica DC500 digital camera. The filter <br /> Ambra1 proteins results abnormal locomotor activity              twisted possible light refracting through <br /> severe myopathy characterized irregular myofiber orientation           striated muscle. Pixel intensity trunk region was <br /> and highly disorganized sarcomeres, suggesting role Ambra1          measured ImageJ software. Values expressed the <br /> in muscle development. In agreement this, histological               percentage WT pixel intensity 6 SEM (n = 20). <br /> analysis mouse Ambra1 gene trap mutant (Ambra1gt/gt) embryos <br /> showed disorganized three-dimensional structure developing <br />                                                                           Whole mount situ hybridization (WMISH) <br /> muscles increased proliferation muscle cells. <br />                                                                              Zebrafish embryos fixed overnight 4% paraformalde- <br />                                                                           hyde (PFA, Sigma) phosphate-buffered saline (PBS) the <br /> Materials Methods                                                     required stages development. WMISH performed as <br /> Animal maintenance handling                                           previously described [19]. DIG-labeled myoD1 riboprobe was <br />                                                                           synthesized vitro transcription T7 RNA polymerases <br />    Zebrafish (AB strain) raised, staged maintained <br />                                                                           (Roche), following manufacturer&#226;&#8364;&#8482;s instructions after <br /> according standard protocols [17,18]. Embryos obtained <br />                                                                           plasmid linearization BamHI restriction enzyme. <br /> by natural spawning cultured zebrafish fish water solution <br /> (50x: 25 g Instant Ocean, 39.25 g CaSO4, 5 g NaHCO3 1 l) at <br /> 28.5uC photoperiod 14 h light/10 h dark. For vivo            Imaging <br /> imaging, embryos anesthetized 0.04% tricaine [18]. The             For confocal microscopy, fixed embryos embedded in <br /> touch-evoked motor behaviour stimulated touching               0.8% low-melting agarose placed depression slide, a <br /> embryo tip. Ambra1gt/+ mice (CD1 strain) bred         Nikon C2 confocal used record images. WMISH- <br /> order obtain Ambra1gt/gt embryos [11]. Data obtained           stained embryos mounted 87% glycerol PBT (PBS plus <br /> E13.5 embryos comparing Ambra1gt/gt wild-type (WT)                 0.1% Tween 20) cleared mounted 2:1 benzyl benzoate/ <br /> animals. Mice housed individual cages environmen-           benzyl alcohol, observed Leica DMR microscope, and <br /> tally controlled room (23 uC, 12 h light/12 h dark cycle)             photographed Leica DC500 digital camera. <br /> provided food water ad libitum. All animal procedures <br /> were approved Ethics Committee University                Histology immunofluorescence <br /> Padova Tor Vergata, Rome.                                                Zebrafish embryos fixed overnight 4% paraformalde- <br />                                                                           hyde PBS 4 uC. For histology, 5 mm paraffin sections <br /> MO microinjection                                                         cut stained haematoxylin eosin. Embryos were <br />    MO (Gene Tools) treatment performed MOs               fixed antibody staining 4% PFA whole-mount <br /> the ATG translation initiation sites ambra1a ambra1b         immunohistochemistry performed according Dolez et al. <br /> transcripts (MO-ambra1a-ATG MO-ambra1b-ATG)                  [20], using following primary antibodies: rabbit polyclonal <br /> splice-blocking MOs designed exon 3-intron 3 junction              anti-PH3 (1:1000; Millipore); mouse monoclonal anti-Pax7 (1:20; <br /> sequence genes (MO-ambra1a-splice MO-ambra1b-                 Hybridoma Bank), mouse monoclonal anti-F59 (1:100; Hybrid- <br /> splice). The designed splice-blocking MOs cause skipping           oma Bank); mouse monoclonal anti-F310 (1:100, Hybridoma <br /> exon 3, altering translation reading frame exon 4        Bank); rabbit polyclonal anti-Laminin (1:400; Sigma). The <br /> introduction premature stop codon, resulting proteins        following secondary antibodies used: Alexa Fluor 488 goat <br /> lack known binding domains (Fig. S1). As controls, used            anti-mouse IgG1(y1) (A-21121, Invitrogen); Alexa Fluor 594 goat <br /> five-nucleotide-mismatched control MOs (MO-ambra1a-5m                 anti-rabbit IgG (H+L) (A-11012, Invitrogen). E13.5 mouse <br /> MO-ambra1b-5m). All MOs previously described                     embryos fixed 4% paraformaldehyde, de-hydrated and <br /> validated [14], lower MOs dosages used               included paraffin. Haematoxylin-eosin staining performed <br /> work order reduce embryo mortality. Specifically,          following standard protocols. Images detected using Zeiss <br /> MO, 10.3 ng injected yolk 1-cell stage embryos,            Axioplan microscope equipped Leica DC500 digital <br /> whereas dosage halved co-injection experiments.            camera. <br /> Injections performed dissecting microscope using              Phospho-H3 immunostaining quantified counting <br /> microinjector attached micromanipulator (Leica Microsys-             positive cells present somites trunk region <br /> tems). MOs-injected embryos incubated 1x fish water          embryos category. <br /> solution 28.5 uC desired stages development. <br />                                                                           Transmission electron microscopy <br /> RNA synthesis injections                                                Samples fixed 6% glutaraldehyde 0.1 M cacodylate <br />   For ambra1-MO rescue experiments, human AMBRA1                      buffer (pH 6.9) overnight 4uC. After washing cacodylate <br /> cDNA removed pLPCX-AMBRA1 [11] subcloned                     buffer, specimens post-fixed 1% OsO4 same <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            2                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                   Ambra1 Muscle Development <br />  <br />  <br /> buffer 2 h dehydrated graded ethanol series followed           ambra1 depletion interferes myoD1 expression <br /> by propylene oxide. The specimens embedded EPON 812                 myogenesis <br /> resin. Thick sections (1 mm) cut Ultracut S Reichert              To assess requirement ambra1a ambra1b during <br /> ultramicrotome, counterstained toluidine blue examined             embryonic myogenesis, performed WMISH analysis the <br /> with light microscope. Thin sections (100 nm) stained           somite-specific marker myoD1. At bud stage (10 hpf), expression <br /> uranyl acetate lead citrate. Micrographs taken              myoD1 adaxial cells reduced ATG-morphants, <br /> FEI Tecnai G12 electron microscope operating 100 kV.                     width increased, particularly the <br />                                                                             posterior end (Fig. 2). This evident ambra1a ATG- <br /> Microinjection hsp70l:Lc3-RFP plasmid                           morphants co-injected morphants. Expression myoD1 <br /> fertilized eggs                                                             appeared normal splice-morphants genes in <br />    A total 25 ng/ml hsp70l:Lc3-RFP [21] plasmid co-               5m-control morphants. At 20 hpf, ambra1 ablation resulted in <br /> injected ambra1-MO zebrafish embryos one-cell             abnormally shaped distinct somites, suggesting impaired <br /> stage. Microinjected embryos raised 3 dpf stage heat            somite organization, shortened anterior/posterior <br /> shocked replacing embryo medium fish water                      axes undulated notochord. All defects evident in <br /> preheated 41 uC incubated air incubator 38             ambra1a ATG-morphants co-injected embryos. Segmenta- <br /> uC 30 min induce hsp70 expression. Lc3-RFP labeled                   tion <span id='am-320' about='xsp:pattern' typeof='owl:Thing'>pattern</span> appeared normal splice-morphants 5m- <br /> puncta analysed confocal microscopy.                                control morphants (Fig. 2). <br />  <br /> Statistical analysis                                                        ambra1 deficiency leads abnormal myogenesis <br />    Statistical analysis performed one-way analysis                   To fully appreciate microscopic level phenotype of <br /> variance (ANOVA) followed Bonferroni&#226;&#8364;&#8482;s multiple comparison               ambra1 morphants, performed histological analysis of <br /> test (GraphPad Prism Software).                                             haematoxylin/eosin stained longitudinal sections muscles <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             3 dpf embryos. This showed well-organized myofibers with <br /> Results                                                                     elongated nuclei WT 5m-control muscles. In ATG- <br />                                                                             morphants, myosepta clearly evident, particularly in <br /> Knockdown ambra1a ambra1b interferes                            ambra1a ATG-morphants co-injected embryos, myofibers <br /> embryo motility muscle integrity                                        appeared misaligned markedly disorganized shape and <br />    To investigate effects ambra1a ambra1b ablation        orientation. Moreover, ATG-morphants displayed apparently <br /> muscle development, injected validated antisense MOs [14]                increased number myonuclei (Fig. 3). Morphological changes <br /> into yolk mass 1-cell embryos suppress translation        evident splice-morphants, the <br /> maternal zygotic mRNA (ATG MOs) silence zygotic                   myosepta thinner myofibers appeared organized <br /> transcription genes (splice-blocking MOs). In agreement          respect controls. <br /> with previous work [14], ATG-morphant embryos displayed                      The irregular arrangement muscle fibers confirmed by <br /> severe abnormalities overall appearance mainly                toluidine blue stained semithin longitudinal sections muscle <br /> consisted body growth delay, curved shape, hemorrhagic                   fibers running vertical myosepta (Fig. 4). In both <br /> pericardial cavity, neural tube defects. The percentages         ambra1a ambra1b ATG-morphants, myoseptum was <br /> of normal, abnormal dead embryos 3 dpf reported               difficult distinguish places appeared interrupted. <br /> Fig. S2. More 70% ATG-morphant 35%                     Multiple areas devoid staining present myofibers of <br /> splice-morphant embryos manually dechorionated (Fig.              ATG-morphants, amorphous opaque material replaced <br /> S3). The delay hatching overall                       lost myofibers. In ambra1a ATG-morphants co-injected <br /> developmental delay [14], reduction muscle             morphants fibers appeared detached. Skeletal muscles of <br /> activity contributes exit embryos             splice-morphants displayed minor modifications. Cross <br /> protective outer chorion. Moreover, hatching, ambra1a            sections trunk region analysed toluidine blue staining <br /> and ambra1b ATG-morphants, co-injected morphant                  showed extensive disruption myofiber structure organiza- <br /> embryos, showed impaired totally absent locomotor activity           tion, spaces regions filled amorphous opaque <br /> did respond touch escape response normally                  material. Several myonuclei appeared large, abnormally rounded, <br /> observed control embryos injected mismatch MOs (Fig.                centrally localized (Fig. 4). <br /> S4). Less severe aspects phenotypes included uncoordinated              Immunostaining 3 dpf embryos phospho-histone H3, a <br /> movements response tactile stimuli swimming            mitotic marker, showed higher proliferation rate ATG- <br /> circular fashion. In agreement impaired locomotor                  morphant embryos (Fig. 5), spots phospho-histone H3 <br /> activity, ATG-morphants exhibited marked statistically                positive nuclei splice-morphants weakly increased <br /> significant reduction birefringence skeletal musculature          compared controls. The increase mitotic cells ATG- <br /> when compared control embryos, indicating decreased                    morphant embryos resulted statistically significant (Fig. S5). <br /> striated muscle formation and/or loss myofiber organization                 To evaluate effect zebrafish ambra1a ambra1b <br /> (Fig. 1). Reduction birefringence comparable co-             knockdown autophagic process, analysed autophagy <br /> injected ambra1a ATG-morphants, marked ambra1b              muscle fibers transient expression lysosomal Lc3- <br /> ATG-morphants. Birefringence slightly reduced              RFP reporter protein obtained microinjection zebrafish <br /> ambra1a ambra1b splice-morphants, normal              embryos hsp70l:Lc3-RFP reporter construct, the <br /> 5m-control embryos (Fig. 1). Co-injection ATG-MOs                   zebrafish Lc3 gene driven hsp70l promoter thus <br /> human AMBRA1 mRNA resulted rescue morphants                   induced heat-shock treatment [21]. Analysis 3 dpf showed <br /> phenotypes (Fig. 1). Moreover, analysis rescued embryos                  Lc3-RFP puncta present myofibers WT <br /> showed marked statistically significant improvement                5 m-control embryos. In contrast, knockdown either <br /> skeletal muscle birefringence compared ATG-morphants                ambra1a ambra1b led complete lack Lc3-RFP <br /> (Fig. 1) percentage hatched embryos 3 dpf (Fig. S3).       puncta muscle fibers (Fig. S6). <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                    Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 1. Ablation ambra1 results reduced birefringence zebrafish embryos. (A). Representative images normal and <br /> polarized light 3-dpf live embryos injected indicated MOs. ATG-morphant embryos reduced size, curved shape, pericardial edema <br /> and reduced birefringence compared WT 5 m-control embryos. No visible abnormalities evident splice-morphants. The <br /> phenotypic defects ATG-morphant embryos, including birefringence, rescued co-injection 10 ng/embryo human AMBRA1 mRNA. <br /> (B). Quantification embryo trunk muscles birefringence shows severe statistically significant reduction ambra1 ATG- co-injected <br /> morphants. The birefringence faintly reduced ambra1 splice-morphants, WT 5 m-control embryos display highly birefringent <br /> skeletal muscles. Muscle birefringence statistically increased ATG- co-injected morphants co-injected human AMBRA1 mRNA <br /> (***, P,0.001). <br /> doi:10.1371/journal.pone.0099210.g001 <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              4                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                   Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 2. In situ hybridization analysis myoD1 expression ambra1 knockdown embryos. Expression myoD1, analyzed embryos <br /> injected indicated MOs 10 20 hpf, affected ambra1 ATG-morphants co-injected morphants. No differences evident in <br /> ambra1 splice-morphants compared WT 5 m-control embryos. Embryos shown dorsal view, anterior top. <br /> doi:10.1371/journal.pone.0099210.g002 <br />  <br />  <br />  <br />  <br /> Figure 3. Abnormal morphology ambra1 knockdown embryos, revealed haematoxylin/eosin staining. Representative <br /> longitudinal sections 3 dpf control ambra1 morphant embryos. Myofibers ATG- co-injected morphants muscles highly disorganized <br /> and display irregular myosepta boundaries. The phenotype splice-morphants severe compared WT 5 m-control embryos. <br /> doi:10.1371/journal.pone.0099210.g003 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             5                              June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                 Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 4. Abnormal morphology ambra1 knockdown embryos, revealed toluidine blue staining. Representative longitudinal <br /> and cross sections 3 dpf control ambra1 morphant embryos. Muscles ambra1 ATG-morphants severe phenotype, misaligned <br /> myofibers scattered somitic compartment. Black arrows, areas devoid staining; white arrows, large myonuclei condensed chromatin; <br /> black arrowheads, interruption myoseptum; asterisks, opaque material replacing lost myofibers. <br /> doi:10.1371/journal.pone.0099210.g004 <br />  <br />    Immunofluorescence 2 dpf Pax7, key regulator               dispersed cells surrounded enlarged areas <br /> muscle progenitor cells [22], showed ATG- splice-           disorganized cytoplasm devoid normally appearing organ- <br /> morphants higher incidence Pax7-positive cells            elles. Remnants degenerating myofibers seen these <br /> spaces myosepta, instead regularly localized          regions (Fig. 7, panels B1&#226;&#8364;&#8220;B4, D1&#226;&#8364;&#8220;D4, F1&#226;&#8364;&#8220;F4). Although ATG- <br /> the edges somites WT 5 m-control embryos                 morphants muscle fibers contained sarcomeres, were <br /> (Fig. 6).                                                                 substantially reduced number, torn, correctly aligned and <br />                                                                           dispersed amorphous material. Areas myofibrils <br /> Ablation ambra1 causes ultrastructural defects                      showing orthogonal arrangement visible <br /> myofibers                                                                 (Fig. 7). The ultrastructural muscle defects severe in <br />    To better understand subcellular alterations responsible       splice-morphants, small regions devoid myofibrils <br /> the observed fiber disturbances, performed electron                 present milder disorganization sarcomeres <br /> microscopy analysis longitudinal cross-sections. Zebrafish         (Fig. 7). <br /> embryos 3 dpf showed WT 5 m-control embryos (not                 Patterning internal membranes affected ATG- <br /> shown) well-defined muscle fiber structure normally              morphants co-injected embryos, sarcoplasmic <br /> developed sarcoplasmic reticulum encircling myofibrils inter-         reticulum appeared dilated closely associated with <br /> myofibrillar mitochondria. In control embryos, sarcomeres            myofibrils (Fig. 8, row E). ambra1 morphant embryos showed <br /> clearly visible formed regular repeating units alignment         ultrastructural abnormalities T-tubules sarcoplasmic <br /> of well-defined Z-lines, mitochondria aligned rows            reticulum, ranging mild changes splice-morphants to <br /> had tightly packed cristae, myofibers surrounded         unrecognizable triad areas co-injected morphants, WT <br /> small areas amorphous material (Fig. 7, panels A1&#226;&#8364;&#8220;A4). In              embryos 5 m-control embryos displayed normal pattern of <br /> contrast, muscle fibers ATG-morphants co-injected               T-tubules sarcoplasmic reticulum resulting regularly spaced <br /> embryos contained small areas organized filaments, widely         triads (Fig. 8, row C). In ambra1 morphant embryos, mitochondria <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            6                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                    Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 5. Cell proliferation muscles 3 dpf control ambra1 morphant embryos. Mitotic cells, detected immunostaining for <br /> phospho-histone H3 longitudinal sections, abundant ATG-morphant embryos respect WT 5 m-control embryos. Anterior is <br /> to left dorsal up. <br /> doi:10.1371/journal.pone.0099210.g005 <br />  <br />  <br /> were scattered cytoplasm morphology                showed regular hexagonal arrays filaments <br /> was markedly affected, swollen devoid           ambra1 morphants hexagonal arrays irregular, <br /> of cristae or, present, disorganized abnormal            areas filaments associated thin <br /> (Fig. 7, panel B2 Fig. 8, row A). Ultrastructural analysis         filaments (Fig. 8, row D). <br /> confirmed myonuclei larger morphant embryos,                 Taken together, data highlight severe disorganization of <br /> with irregular shape numerous compared                  muscle tissue cells ambra1 depletion. Also, alteration of <br /> control embryos (Fig. 8, row B). The abnormal myofiber                      mitochondria ER structure causative the <br /> ultrastructure ambra1 depleted embryos particularly                  phenotype. <br /> evident cross sections, myofibers control embryos <br />  <br />  <br />  <br />  <br /> Figure 6. Pax7 expression control ambra1 morphant embryos. Lateral views 2 dpf muscles analyzed immunofluorescence for <br /> Pax7. In WT 5 m-control embryos, Pax7-positive cells localized edge somites, ambra1 morphants Pax7-positive cells <br /> are misplaced spaces myosepta. Anterior left dorsal up. <br /> doi:10.1371/journal.pone.0099210.g006 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              7                              June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                        Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 7. Ultrastructural analysis ambra1 morphants muscles reveals disorganized sarcomeres. Representative electron micrographs <br /> of cross longitudinal sections 3 dpf (WT, panels A1&#226;&#8364;&#8220;A4), ambra1a ATG-morphant (panels B1-B4), ambra1a splice-morphant (panels C1&#226;&#8364;&#8220;C4), <br /> ambra1b ATG-morphant (panels D1&#226;&#8364;&#8220;D4), ambra1b splice-morphant (panels E1-E4), co-injected morphant (panels F1&#226;&#8364;&#8220;F4) zebrafish embryos. <br /> Columns 2 4 higher magnification views boxed areas column 1 3, respectively. Muscles WT 5 m-control (not shown) <br /> embryos display well-organized myofibers, showing densely packed sarcomeres regular organization myofilaments. ambra1 <br /> depleted muscles number ultrastructural defects, small patches disorganized myofibers mitochondria scattered the <br /> cytoplasm. Black arrows, area myofibrils having different orientations; white arrow, dilated sarcoplasmic reticulum contact myofibrils; <br /> asterisks, fragments torn myofibrils; M, mitochondria; N, nucleus. <br /> doi:10.1371/journal.pone.0099210.g007 <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 8                               June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                   Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 8. Ultrastructural analysis ambra1 morphants muscles reveals disorganized subcellular structures. When compared WT <br /> and 5 m-control (not shown) embryos, ambra1 morphant embryos display alterations mitochondria (row A), nuclei (row B) triads (row C), <br /> perturbation hexagonal arrangement filaments (row D), dilations endoplasmic reticulum (row E). Muscles of <br /> morphant embryos presence areas reduced filaments (black circles row D) adjacent normal-appearing hexagonal <br /> structures (white circles row D). Co-injected double morphant embryos exacerbated defects structures, defects barely <br /> evident splice-morphants. <br /> doi:10.1371/journal.pone.0099210.g008 <br />  <br /> Ablation ambra1 affects myofibers myosepta                     present correct positions somites, myofiber <br />    Next, assess knockdown ambra1a ambra1b                organization disrupted. <br /> expression results defective specification patterning slow         To evaluate myosepta defects suggested the <br /> and/or fast muscle fibers, examined myosin filaments           morphological analysis, performed whole-mount immunoflu- <br /> ambra1 knockdown embryos immunostaining F59                orescence anti-Laminin antibody. Myotomes ATG- and <br /> F310 antibodies, <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> slow fast myosin isoforms,                co-injected morphants embryos displayed abnormal U-shape <br /> respectively. Slow muscle fibers present ambra1           appearance, myosepta presenting frequent interruptions of <br /> knockdown, myofiber density appeared lower ambra1a             structure, myosepta appeared normal control- <br /> ATG-morphants co-injected embryos (Fig. 9). However,                MOs, splice-MOs injected embryos (Fig. 9). <br /> whereas control embryos filaments nicely                   Defects myofibers myosepta morphology and <br /> organized myotomal segments V-shaped                      organization rescued co-injection human AMBRA1 <br /> regularly spaced, filaments slow muscles ATG-                  mRNA (Fig. 9). <br /> morphant embryos appeared highly disorganized, wavy and <br /> twisted myofibrils. Moreover, characteristic V-shaped appear-          Skeletal muscles Ambra1gt/gt mouse embryos display <br /> ance vertical myoseptum completely absent.               morphological defects <br /> Some muscle fibers missing detached myosepta,                To obtain insight role Ambra1 muscle <br /> generating cell-free spaces ATG-morphants. Combined injec-              development, investigated skeletal muscles Ambra1gt/gt <br /> tion ATG-MOs exacerbated phenotype. The                        mutant mice, bear gene trap insertion Ambra1 gene <br /> phenotype normal ambra1 splice-morphants,              [11]. We analyzed muscle morphology haematoxylin-eosin <br /> myofibrils presented slightly wavy morphology (Fig. 9). A           staining E13.5 mouse embryos, differentiation muscles <br /> similar disruption myofiber organization evident fast       complete myogenic cells undergoing fusion to <br /> muscles. In WT 5 m-control embryos, fast muscle fibers             form myofibers. In WT E13.5 mouse embryos, structure the <br /> appeared relatively uniform size regularly arranged         developing muscles appeared normal, myofibers well <br /> parallel rows. Conversely, ATG-morphants fast muscle fibers             organized aligned generate ordered structure the <br /> displayed variable shapes highly disorganized, showing          muscle, nuclei localized periphery <br /> dystrophic appearance detachment retraction                    (Fig. 10A, upper panels). In Ambra1gt/gt E13.5 mouse embryos, <br /> myofibers vertical myosepta forming somite                    skeletal muscles formed three-dimensional tissue <br /> boundaries irregular wavy myofibers morphology                architecture organized (Fig. 10A, lower panels). The <br /> (Fig. 9). Interestingly, splice-morphants displayed evident         abnormal structure developing muscles Ambra1gt/gt embryos <br /> phenotype fast muscle fibers slow muscle fibers. These          failure completing muscle development a <br /> data indicated slow fast muscle fibers        delay myofiber maturation. This supported the <br />                                                                            presence immature myofibers displaying centrally located <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             9                              June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                    Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 9. Knockdown ambra1 interferes myosin organization slow fast muscles myosepta. Lateral views 3 <br /> dpf embryos labeled F59 antibody (slow muscle fibers) F310 antibody (fast muscle fibers), showing abnormally organized <br /> myofilaments ATG-morphant embryos compared WT 5 m-control embryos. Slow fibers thinner ATG-morphant embryos, <br /> whereas fast fibers splice-morphant embryos display wavy phenotype, visibly different controls. The asterisks indicate broken missing <br /> muscle fibers. Laminin labeling highlights loss V-shape arrangement somites reveals interrupted myosepta (arrowheads) ATG- <br /> morphant embryos. Defects myosin organization myosepta structure rescued co-injection human AMBRA1 mRNA. <br /> Anterior left dorsal up. <br /> doi:10.1371/journal.pone.0099210.g009 <br />  <br /> nuclei Ambra1gt/gt embryos (Fig. 10A, lower panels). These                Discussion <br /> features, present muscles Ambra1gt/gt embryos <br /> we analyzed, clearly exemplified developing tongue. In               In work used targeted protein depletion approaches to <br /> WT E13.5 embryos, tongue showed well-defined array                      investigate involvement Ambra1 muscle development. <br /> parallel myofibers distinctly organized different planes        The severe phenotypes displayed zebrafish ambra1a/ambra1b <br /> of developing intrinsic tongue muscles (Fig. 10B, upper panels).         morphants mouse Ambra1gt/gt embryos suggest key role for <br /> Conversely, Ambra1gt/gt E13.5 embryos, tongue displayed             Ambra1 myogenesis. Our analysis muscles ambra1a and <br /> general disorganization developing muscles myofibers              ambra1b zebrafish morphants mainly focused 3 dpf <br /> were defined. In addition, marked increase cell          embryos, stage muscles fully developed. Our data <br /> density muscle tissue Ambra1gt/gt embryos (Fig. 10B,         indicate ablation Ambra1 leads severe myopathy with <br /> lower panels). Similar morphological cellular alterations           structural functional defects skeletal muscles, characterized <br />                                                                              marked reduction myofiber density, abnormal orientation <br /> present developing dorsal limb muscles Ambra1gt/gt <br />                                                                              decreased alignment myofibers, disorganization sarco- <br /> embryos, showed noticeable increase interstitial <br />                                                                              meres, alterations tubulo-reticular network abnormal <br /> connective tissue (Fig. 10C, D). <br />                                                                              mitochondria morphology. The reduced locomotor activity of <br />                                                                              ambra1a/ambra1b zebrafish morphants, myofiber and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              10                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                        Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> Figure 10. Morphological alteration skeletal muscles Ambragt/gt mouse embryos. Representative pictures skeletal muscles WT <br /> and Ambra1gt/gt E13.5 mouse embryos, following haematoxylin-eosin staining. (A) Details neck muscle. WT embryos display well-organized <br /> and mature myofibers (black arrows), myonuclei localized edge cell (right panel). In Ambra1gt/gt embryos, muscle <br /> is immature, poorly organized myofibers displaying centrally located nuclei (black arrows). (B) Details tongue. In WT embryos, <br /> myofiber formed well-organized (arrows), Ambra1gt/gt embryos general disorganization muscle architecture. (C, D) <br /> Representative pictures dorsal (C) limb (D) muscles (black arrows). Ambra1gt/gt embryos abnormal muscle organization, a <br /> marked thickening connective tissue (black asterisks). All analyzed muscles display noticeable increase cell density Ambra1gt/gt <br /> embryos. Scale bar, 50 mm. <br /> doi:10.1371/journal.pone.0099210.g010 <br />  <br /> myosepta defects, rescued co-injection human                      form slow muscle cells [25], remaining near to <br /> AMBRA1 mRNA, confirming defects caused                  notochord form muscle pioneer cells [26]. Later, myoD1 <br /> Ambra1 depletion pointing conservation Ambra1                    expression takes place posterior half newly formed <br /> functions evolution. Although role Ambra1 skeletal             somite, giving rise fast muscle fibers [25]. Knockdown of <br /> muscle proposed now, findings                      zebrafish ambra1 genes led marked changes myoD1 expression, <br /> agreement recent studies indicating autophagy                  resulting reduced myoD1 signal widening space <br /> plays key role skeletal muscles, shown myopathic                bilateral adaxial cells ATG-morphant embryos. At <br /> phenotype Atg5 Atg7 muscle-specific knockout mice [3,4]                 later stages, somites present broad-shaped non-homoge- <br /> and connection autophagy deregulation                       neous myoD1 expression ATG-morphant embryos. Interestingly, <br /> muscular dystrophies [5&#226;&#8364;&#8220;9].                                                    expression shh, gene coding secreted signalling protein <br />    In zebrafish, transcripts ambra1 genes present              implicated commitment muscle precursors (reviewed in <br /> maternal RNAs eggs display gradual decline 8                [15,26]), abnormal Ambra1gt/gt mouse <br /> hpf, replaced zygotic mRNAs 12 hpf onwards [14].                 embryos [11], zebrafish ambra1 ATG-morphant <br /> To verify possible role proteins early stages             embryos, resulting notochord waving [14]. Depletion of <br /> skeletal muscle development, commitment myogenic                    Ambra1 proteins zebrafish did prevent specification <br /> fate analysed WMISH myoD1, expression                         differentiation slow fast muscle fibers, clearly <br /> normally begins adaxial cells prior somite formation                 interfered myofibrillogenesis leading anomalous pattern <br /> [23,24]. These cells, migrating laterally somites,           fiber types. As different levels, range and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                11                               June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                     Ambra1 Muscle Development <br />  <br />  <br /> timing, Shh signalling specify different muscle subtypes [27,28],          related-protein plays critical evolutionally conserved <br /> the displaced expression morphogen explain                  function skeletal muscle development. In animal <br /> disorganization ambra1-morphant muscle fibers.                         models, ablation Ambra1 leads abnormal muscle morpho- <br />    The changes myoD1 shh expression,                 genesis. Ambra1 critical correct architecture and <br /> reduced locomotor activity, observed ambra1 morphants occur                maturation myofibers, implicated cell <br /> at early developmental stages, embryonic cells likely           proliferation control. Our data suggest new function Ambra1 <br /> not competent autophagy, e.g. biological process               muscle biogenesis. While presence abnormal <br /> which Ambra1 well-established role. Actually, autophagy                 organelles likely associated well-established role of <br /> can observed zebrafish embryos starting 32 hpf [29],               Ambra1 autophagy regulation, altered morphology the <br /> and relatively late compared onset muscle                hypercellularity developing skeletal muscles a <br /> developmental defects. Interestingly, ambra1 depletion               different function Ambra1. Further studies needed to <br /> interfere gene expression program responsible correct            fully elucidate functional roles Ambra1 developing <br /> muscle development, suggested displaced expression               postnatal skeletal muscles, order understand this <br /> shh [14] myoD1, unclear              protein contributes muscle homeostasis identify possible <br /> effects related Ambra1 pro-autophagic roles. On              human muscle pathologies linked Ambra1 mutations. <br /> side, ambra1 morphant embryos display reduced <br /> autophagy, measured immunoblot analysis lipidated LC3               Supporting Information <br /> at 2 dpf [14]. A decreased activity autophagic process was <br /> also suggested lower incidence puncta muscle fibers              Figure S1 Schematic diagram showing exon-intron <br /> isolated zebrafish embryos expressing hsp701:Lc3-RFP                   structure zebrafish ambra1a (A) ambra1b (B) <br /> reporter plasmid.                                                             genes. The localization known domains corresponding <br />    Pax7 key regulator myogenic progenitor cells               proteins indicated. The partial sequences abnormally <br /> vertebrates Pax7-positive cells aligned             spliced ambra1a ambra1b transcripts, following targeting with <br /> vertical horizontal myosepta 2 dpf zebrafish                    splice-MOs, highlights loss exon 3 (arrow) causing a <br /> development [30]. In ambra1 morphant embryos, regular                     codon frameshift introduction premature stop codon. <br /> arrangement Pax7-positive cells disturbed               (TIF) <br /> occurred spaces myosepta, confirming                  Figure S2Percentage dead, abnormal normal 3 <br /> disorganization somites. Although significant increase Pax7-          dpf embryos injection different MOs. <br /> positive cells zebrafish dystrophic phenotypes              (TIF) <br /> [30,31], analysis did reveal obvious difference the <br />                                                                               Figure S3 Quantification chorion hatching 72 hpf <br /> number cells control ambra1 morphant <br />                                                                               embryos injection different MOs. <br /> embryos, This result contrast apparently higher <br />                                                                               (TIF) <br /> proliferation rate seen ambra1 morphants haematox- <br /> ylin/eosin staining immunofluorescence phosphorylat-               Figure S4 Quantification touch-evoked response and <br /> ed histone H3. However, apparent discrepancy                      circular movement 3 dpf. Embryos quantified three <br /> explained higher proliferation fibroblasts, suggested         independent microinjections, number embryos each <br /> by muscles Ambra1gt/gt mouse embryos showing conspicuous                 microinjection experiments 80. <br /> thickening interstitial connective tissue.                             (TIF) <br />    As number genes, zebrafish ortholog                  Figure S5   Analysis number mitotic cells <br /> mammalian Ambra1 gene present duplicated paralogs                   present somites region 10 embryos <br /> coding functional proteins. Although knockdown                  category. Data presented mean 6SEM. ***, <br /> these paralogous genes sufficient alter muscle structure,           P,0.001. <br /> the severe phenotype embryos deficient ambra1                (TIF) <br /> paralogs suggests proteins work similar <br /> biological process play distinct roles, justifying          Figure S6 Fluorescence detection muscle fibers of <br /> retention genes functional zebrafish genome           control ambra1 morphant embryos following trans- <br /> the fish-specific whole-genome duplication [32]. Notably, knock-              fection Lc3-RFP reporter construct. Several <br /> down zygotic ambra1a ambra1b transcripts means              fluorescent puncta detected transfected muscle <br /> splice-MOs resulted severe muscle developmental defects,              fibers WT 5m-morphant embryos few <br /> indicating importance maternally supplied ambra1 tran-                 puncta visibile ATG-morphant embryos. <br /> scripts corresponding proteins early embryonic                  (TIF) <br /> development. Interestingly, difference param- <br /> eters, splice-morphants showed remarkably stronger phenotype                Acknowledgments <br /> than ATG-morphants regard fast fiber development,                   The authors wish acknowledge Dr. Michel Bagnat (Department Cell <br /> finding agreement different timing slow            Biology, Duke University Medical Center, Durham, NC) providing <br /> fast fiber differentiation [33]. Since fast fibers differentiate        hsp70l:Lc3-RFP plasmid, Dr. Gianfranco Bellipanni (Department of <br /> slow fibers, process case affected           Biology, College Science Technology, Temple University, <br /> silencing zygotic transcripts maternal transcript            Philadelphia, PA) providing myoD1 plasmid L. Pivotti Dr. <br /> be longer available stage.                                         M. Milanetto valuable assistance fish facility. <br />    A number recent studies indicated autophagy plays <br /> important physiological pathological roles mature fully            Author Contributions <br /> developed skeletal muscles (reviewed [34,35]). Our data,                   Conceived designed experiments: PB FC LDV TS PG. Performed <br /> obtained ambra1a ambra1b zebrafish morphant embryos                  experiments: TS FB PG SC MC GM. Analyzed data: TS PG VC <br /> and Ambra1gt/gt mouse embryos, indicate autophagy              SDB LDV. Wrote paper: LDV PB FC TS. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               12                             June 2014 | Volume 9 | Issue 6 | e99210 <br />                                                                                                                                           Ambra1 Muscle Development <br />  <br />  <br />  <br />  <br /> References <br />  1. Levine B, Kroemer G (2008) Autophagy pathogenesis disease. Cell 132:         19. Thisse C, Thisse B (2008) High-resolution situ hybridization whole-mount <br />     27&#226;&#8364;&#8220;42.                                                                                     zebrafish embryos. Nat Protoc 3: 59&#226;&#8364;&#8220;69. <br />  2. Mizushima N, Komatsu M (2011) Autophagy: renovation cells tissues.              20. Dolez M, Nicolas JF, Hirsinger E (2011) Laminins, heparan sulfate <br />     Cell 147: 728&#226;&#8364;&#8220;741.                                                                         proteoglycans, participate zebrafish myotome morphogenesis modulating <br />  3. Raben NV, Hill L, Shea S, Takikita R, Baum N, et al. (2008) Suppression                 pattern Bmp responsiveness. Development 138: 97&#226;&#8364;&#8220;106. <br />     autophagy skeletal muscle uncovers accumulation ubiquitinated                21. Ellis K, Bagwell J, Bagnat M (2013) Notochord vacuoles lysosome-related <br />     proteins potential role muscle damage Pompe disease. Hum Mol               organelles function axis spine morphogenesis. J Cell Biol 200: 667&#226;&#8364;&#8220; <br />     Genet 17: 3897&#226;&#8364;&#8220;3908.                                                                       679. <br />  4. Masiero E, Agatea L, Mammucari C, Blaauw B, Loro E, et al. (2009) Autophagy            22. Nord H, Nyga&#203;&#353;rd Skalman L, von Hofsten J (2013) Six1 regulates proliferation of <br />     required maintain muscle mass. Cell Metab 10: 507&#226;&#8364;&#8220;515.                               Pax7-positive muscle progenitors zebrafish. J Cell Sci 126: 1868&#226;&#8364;&#8220;1880. <br />  5. Grumati P, Coletto L, Sabatelli P, Cescon M, Angelin A, et al. (2010) Autophagy        23. Coutelle O, Blagden CS, Hampson R, Halai C, Rigby PW, et al. (2001) <br />     defective collagen VI muscular dystrophies, rea <br /> </body></html>