<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Cross Kingdom Functional Conservation Core Universally <br /> Conserved Threonylcarbamoyladenosine tRNA Synthesis Enzymes <br /> Patrick C. Thiaville,a,b,c,d Basma El Yacoubi,d Ludovic Perrochia,c Arnaud Hecker,e Magali Prigent,c Jennifer J. Thiaville,d <br /> Patrick Forterre,c Olivier Namy,c Tamara Basta,c Val&#195;&#169;rie Cr&#195;&#169;cy-Lagardb,d <br /> Genetics Genomics Graduate Program, University Florida, Gainesville, Florida, USAa; University Florida Genetics Institute, Gainesville, Florida, USAb; Institut de <br /> G&#195;&#169;n&#195;&#169;tique et Microbiologie, Universit&#195;&#169; Paris-Sud, Orsay, Francec; Department Microbiology Cell Science, University Florida, Gainesville, Florida, USAd; <br /> Universit&#195;&#169; Lorraine, UMR 1136 INRA/Universit&#195;&#169; Lorraine Interactions Arbres-Microorganismes, Labex ARBRE, FR EFABA, Facult&#195;&#169; des Sciences, Vandoeuvre, Francee <br />  <br /> Threonylcarbamoyladenosine (t6A) universal modification located anticodon stem-loop tRNAs. In yeast, both <br /> cytoplasmic mitochondrial tRNAs modified. The cytoplasmic t6A synthesis pathway elucidated requires <br /> Sua5p, Kae1p, KEOPS complex proteins. Recent vitro work suggested mitochondrial t6A ma- <br /> chinery Saccharomyces cerevisiae composed proteins, Sua5p Qri7p, member Kae1p/TsaD family <br /> (L. C. K. Wan et al., Nucleic Acids Res. 41:6332&#226;&#8364;&#8220; 6346, 2013, http://dx.doi.org/10.1093/nar/gkt322). Sua5p catalyzes step <br /> leading threonyl-carbamoyl-AMP intermediate (TC-AMP), Qri7 transfers threonyl-carbamoyl moiety TC- <br /> AMP tRNA form t6A. Qri7p localizes mitochondria, Sua5p reported cytoplasmic. We Sua5p is <br /> targeted cytoplasm mitochondria use alternative start sites. The import Sua5p mi- <br /> tochondria required organelle functional, TC-AMP intermediate produced Sua5p cytoplasm is <br /> not transported mitochondria sufficient amounts. This minimal t6A pathway characterized vitro and, the <br /> first time, vivo heterologous complementation studies Escherichia coli. The data revealed potential TC-AMP chan- <br /> neling t6A pathway, coexpression Qri7p Sua5p required complement essentiality E. coli tsaD <br /> mutant. Our results firmly established Qri7p Sua5p constitute mitochondrial pathway biosynthesis t6A <br /> and bring additional advancement understanding reaction mechanism. <br />  <br />  <br />  <br /> A    s central adaptors responsible accurate decoding of <br />      mRNA, tRNA molecules critical cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>survival</span>. In order <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> ensure accuracy decoding, tRNAs harbor posttranscrip- <br />                                                                                          verts carbon dioxide (or bicarbonate), L-threonine, ATP to <br />                                                                                          L-threonylcarbamoyl-AMP (TC-AMP) intermediate (16). <br />                                                                                          TC-AMP converted t6A YdiCBs (TsaBEc), YdiBBs <br /> tional modifications mainly anticodon stem loop (ASL) (1).                        (TsaEEc), YdiEBs (TsaDEc) presence tRNA (16). This <br /> N6-threonyl-carbamoyl-adenosine (t6A), modification aden-                           work extended TC-AMP produced TsaC or <br /> osine located position 37 anticodon (t6A37),                         Sua5p eukaryotic archaeal origin converted to <br /> highly conserved tRNA modification present domains life.                       t6A-modified tRNA E. coli TsaDEB enzymes, S. cerevi- <br /> This modification present nearly ANN-decoding tRNAs                            siae KEOPS complex, Pyrococcus abyssi KEOPS complex <br /> (one exception tRNAiMet bacteria) (2). Like modifica-                        (11). In short, enzyme family organisms, TsaC/ <br /> tions position 37, m1G, i6A, ms2i6A modifica-                        Sua5, catalyzes formation TC-AMP. For step of <br /> tions (3, 4), t6A important translation fidelity                       transferring TC tRNA, bacteria require TsaBDE, archaea <br /> stabilizes ASL structure enhances codon-anticodon inter-                         eukaryotes use KEOPS complex. Interestingly, although <br /> action (5).                                                                              TsaBDE KEOPS functional analogues, TsaD/ <br />     The enzymatic synthesis t6A studied vitro                      Kae1 proteins shared systems (Fig. 1), sug- <br /> vivo 40 years (2, 6&#226;&#8364;&#8220;9), synthesis pathways                         gesting protein ancestral t6A synthesis core <br /> were recently elucidated, revealing core set enzymes                      present universal common ancestor (LUCA) (17). This <br /> and kingdom-specific variations (Fig. 1) (10, 11). Two universal                         hypothesis strengthened demonstration S. <br /> protein families involved biosynthesis t6A: TsaC/Sua5                      cerevisiae minimal comprised just Sua5p Kae1/ <br /> and Kae1/Qri7/TsaD (12, 13). In Escherichia coli, tsaC (previously                       TsaD mitochondrial ortholog Qri7p sufficient synthesize t6A <br /> yrdC) tsaD (previously ygjD) essential growth,                        vitro (17). <br /> Saccharomyces cerevisiae, SUA5 (a tsaC homolog) KAE1 (a <br /> cytoplasmic member Kae1/Qri7/TsaD family) re- <br /> quired t6A synthesis (12, 13). Kae1 member KEOPS                            Received 17 June 2014 Accepted 12 July 2014 <br /> complex (kinase, putative endopeptidase, proteins                              Published ahead print 18 July 2014 <br /> small size) (14), known EKC (endopeptidase-like kinase                              Address correspondence Val&#195;&#169;rie Cr&#195;&#169;cy-Lagard, vcrecy@ufl.edu. <br /> chromatin-associated complex) (15). TsaC TsaD E. coli,                             P.C.T. B.E.Y. contributed equally work. <br /> along bicarbonate, L-threonine, ATP, tRNA, suf-                            Supplemental material article http://dx.doi.org/10.1128 <br /> ficient form t6A vitro; TsaB (previously YeaZ) TsaE (pre-                         /EC.00147-14. <br /> viously YjeE) required (10). This result reproduced                            Copyright &#194;&#169; 2014, American Society Microbiology. All Rights Reserved. <br /> using Bacillus subtilis orthologs, shown YwlC                        doi:10.1128/EC.00147-14 <br /> B. subtilis (YwlCBs; ortholog TsaC E. coli [TsaCEc]) con- <br />  <br />  <br /> 1222   ec.asm.org                                          Eukaryotic Cell   p. 1222&#226;&#8364;&#8220;1231                                              September 2014 Volume 13 Number 9 <br />                                                                                                                                   Mitochondrial t6A Synthesis <br />  <br />  <br />  <br />  <br /> FIG 1 Diversity biosynthesis pathway universal tRNA modification t6A. The formation TC-AMP catalyzed enzyme family all <br /> organisms (TsaC/Sua5). Sua5p contains TsaC domain plus additional C-terminal domain, called Sua5, characterized. To transfer <br /> threonylcarbamoyl (TC) tRNA, bacteria require TsaBDE, archaea eukaryotes use KEOPS complex (composed Kae1, Bud32, Cgi121, Pcc1 <br /> proteins; Gon7 exclusively fungi). <br />  <br />  <br />  <br />     In S. cerevisiae, mitochondrial DNA (mtDNA) encodes                media, agar dropout (lacking uracil <br /> the required mitochondrial tRNAs, shown                 [&#226;&#171;&#186;ura], leucine, [&#226;&#171;&#186;leu], and/or histidine [&#226;&#171;&#186;his]), purchased <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> contain t6A (18). Qri7p shown localize mito-               Clontech (Palo Alto, CA) prepared recommended manu- <br /> chondria organisms (19, 20), qri7&#226;&#338;&#172; S. cerevisiae            facturer. Glucose (Glu; 2%, wt/vol), glycerol (Gly; 4%, wt/vol), 5-fluoro- <br /> does grow glycerol, indicating mitochondrial defect (19,             orotic acid (5-FOA; 0.1%, wt/vol), G418 (300 &#226;?&#174;g/ml) used when <br />                                                                               appropriate. Yeast transformations carried using frozen compe- <br /> 21). Also, mitochondrial targeting sequence Qri7p is <br />                                                                               tent cells described previously (24), plating appropriate <br /> deleted, Qri7p functionally replace Kae1p t6A synthesis in <br />                                                                               media. The sua5::KANMX4 strain VDC9100 created PCR <br /> the cytoplasm S. cerevisiae cells (12). Therefore, proposed         amplifying sua5::KanMX4 locus sua5::KANMX4/SUA5 dip- <br /> that mitochondrial t6A minimal composed                  loid obtained Euroscarf oligonucleotides flanking allele. <br /> only Sua5p Qri7p (11, 17).                                                The PCR product transformed electroporation wild- <br />     Several questions remain answered complete un-               type BY4742 containing pBN204 (SUA5 URA3 plasmid) (13). Trans- <br /> derstanding yeast mitochondrial pathway. First,              formants selected SD&#226;&#171;&#186;ura plates containing G418. After passag- <br /> clear TC-AMP, product Sua5p-catalyzed reaction,              ing uracil-containing media, strain cured pBN204 by <br /> transported mitochondria Sua5 protein itself              counterselection 5-FOA. Disruption SUA5 confirmed PCR <br /> imported organelle. Global subcellular localization stud-           using oligonucleotides annealing inside outside locus, as <br /> ies S. cerevisiae reported Sua5p localize cytoplasm (20,         inside KANMX4. VDC9100 confirmed devoid t6A using the <br /> 22). Second, Qri7, unlike bacterial counterparts, func-           high-performance liquid chromatography (HPLC) liquid chroma- <br />                                                                               tography-tandem mass spectrometry (LC-MS/MS) analyses listed below. <br /> tion TsaB TsaE proteins? A bacteria, mainly <br />                                                                                   E. coli strains grown LB (1%, wt/vol, tryptone, 0.5%, wt/vol, <br /> intracellular pathogens Mycoplasma, lost tsaB                yeast extract, 1%, wt/vol, salt; 1.5%, wt/vol, agar added plates) <br /> tsaE (23). It clear genes kept majority          37&#194;&#176;C unless stated. When necessary, LB supplemented <br /> of sequenced bacteria date. Third, TC-AMP (produced TsaC                kanamycin (Km; 50 &#226;?&#174;g/ml), ampicillin (Ap; 100 &#226;?&#174;g/ml), chloram- <br /> or Sua5p) used indifferently TsaBDE, KEOPS, Qri7p                phenicol (Cm; 35 &#226;?&#174;g/ml), anhydrotetracycline (aTc; 50 ng/ml), ara- <br /> systems vitro, case vivo? Indeed,           binose (Ara; 0.2%, wt/vol). Plasmids transformed E. coli via <br /> TC-AMP intermediate highly unstable (16), likely            electroporation. For drop dilution assays, E. coli cells grown sat- <br /> t6A synthesis vivo requires protection mechanism                     uration overnight LB supplemented aTc appropriate <br /> TC-AMP.                                                                       antibiotics diluted 10-fold increments prior wash <br />     In work, validate hypothesis Sua5p participates          step remove aTc left culture media. Ten &#226;?&#174;l dilution <br /> in mitochondrial t6A synthesis showing Sua5p localizes             spotted appropriate media. For halo plates, cultures <br />                                                                               grown saturation overnight LB supplemented aTc the <br /> both cytoplasm mitochondria alternative <br />                                                                               appropriate antibiotics diluted 500 times. Five ml diluted <br /> translation start site explore possibility TC-AMP pro- <br />                                                                               culture added plates containing 25 ml agar media, excess <br /> tection mechanism channeling heterologous comple-                 bacterial suspension immediately removed. Plates dried, a <br /> mentation studies.                                                            10-&#226;?&#174;l drop aTc added surface plate shown Fig. S3 <br />                                                                               supplemental material. <br /> MATERIALS AND METHODS                                                             Plasmid construction protein expression. All plasmids con- <br /> Strains growth conditions. Yeast strains grown yeast extract-     structed protein expression (Table 1) sequences primers and <br /> peptone-dextrose (YPD; Difco Laboratories) 30&#194;&#176;C. Synthetic minimal         restriction sites used cloning listed Table S1 supplemental <br />  <br />  <br /> September 2014 Volume 13 Number 9                                                                                                         ec.asm.org 1223 <br />  Thiaville et al. <br />  <br />  <br />  <br /> TABLE 1 Strains plasmids used study <br /> Strain plasmid <br /> name                      Relevant characteristic(s)                                                                Reference source <br /> Yeast <br />   BY4741                  MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0                                                         39 <br />   BY4742                  MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0                                                          39 <br />   YGL169w::GFP            MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 SUA5::GFP                                               22 <br />   YDL104c::GFP            MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 QRI7::GFP                                               22 <br />   VDC9054                 MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 sua5::kanMX4 (pBN204)                                    This study <br />   VDC9100                 MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 sua5::kanMX4                                             This study <br />  <br /> E. coli <br />    DH5&#226;?&#163;                   F&#226;&#171;&#186; gyrA462 endA1 &#226;&#338;&#172;(sr1-recA) mcrB mrr hsdS20(rB&#226;&#171;&#186;, mB&#226;&#171;&#186;) supE44 ara-14 galK2 lacY1          Invitrogen <br />                             proA2 rpsL20(Smr) xyl-5 &#226;?&#173;&#226;&#171;&#186; leu mtl1 <br />    BW25113                F&#226;&#171;&#186; &#226;&#338;&#172;(araD-araB)567 &#226;&#338;&#172;lacZ4787(::rrnB-3) &#226;?&#173;&#226;&#171;&#186; rph-1 &#226;&#338;&#172;(rhaD-rhaB)568 hsdR514                   E. coli genetic stock center <br />    VDC5789                BW25113 containing knRExTETrbs::tsaDEc                                                    12 <br />    VDC5684                BW25113 containing knRExTETrbs::tsaCEc                                                    This study <br />    VDC5806                VDC5789(pBAD24)                                                                           This study <br />    VDC5732                VDC5684(pBAD24)                                                                           This study <br />    VDC6116                VDC5684(pBY265.9)                                                                         This study <br />    VDC6120                VDC5684(pBY270.2)                                                                         This study <br />    VDC6126                VDC5789(pBY265.9)                                                                         This study <br />    VDC6130                VDC5789(pBY270.2)                                                                         This study <br />    VDC5926                VDC5789(pBY245.1)                                                                         This study <br />  <br /> Plasmids <br />   pBAD24                  bla, araC, PBAD                                                                           26 <br />   pBAD33                  cat, araC, PBAD                                                                           26 <br />   pUC19                   General cloning vector                                                                    40 <br />   pRS313                  Yeast shuttle vector; HIS3                                                                41 <br />   pBN204                  pYES::SUA5; URA3                                                                          13 <br />   pYX122-mtGFP            Mitochondrial targeting GFP; HIS3                                                         25 <br />   pYX142-mtRFP            Mitochondrial targeting RFP; LEU2                                                         A. Delahodde <br />   pPCT004                 pYX122-mtGFP fused 70 bp SUA5 cloned EcoRI/BamHI fragment                  This study <br />   pPCT006                 pYX122-mtGFP fused 5=-truncated SUA5 cloned EcoRI/BamHI fragment                    This study <br />   pPCT017                 pUC19::SUA5 cloned EcoRV fragment                                                      This study <br />   pPCT019                 pUC19::SUA5(M10L) cloned EcoRV fragment                                                This study <br />   pPCT025                 pYX122-mtGFP::SUA5 subcloned pPCT017 EcoRI/BamHI fragment                         This study <br />   pPCT030                 pYX122-mtGFP::SUA5(M10L) subcloned pPCT019 EcoRI/BamHI fragment                   This study <br />   pPCT036                 pYX122-mtGFP fused 70 bp SUA5(M10L) cloned EcoRI/BamHI fragment            This study <br />   pPCT066                 Sua5(10-426) subcloned pPCT006 EcoRI/SmaI fragment pRS313 treated       This study <br />                              EcoRI/EcoRV <br />    pPCT070                GAL promoter subcloned pBY137 SpeI/PvuII fragment pPCT066 digested           This study <br />                              XbaI/SmaI <br />    pPCT074                GAL::tsaC subcloned pBY135 SpeI/PmeI fragment pRS313 digested SpeI      This study <br />                              EcoRV <br />    pPCT075                GAL::ywlC subcloned pBY137 SpeI/PmeI fragment pRS313 digested SpeI      This study <br />                              EcoRV <br />    pPCT076                Site-directed mutagenesis pBY176 change AUG SUA5 CUG                This study <br />    pPCT078                Site-directed mutagenesis pBY176 change second AUG SUA5 CUG               This study <br />    pBY130                 pBAD24::SUA5                                                                              13 <br />    pBY135                 pYesDEST52::tsaCEc; bla, URA3                                                             13 <br />    pBY137                 pYesDEST52::ywlCBs; bla, URA3                                                             13 <br />    pBY176                 pRS313::SUA5Sc; bla, URA3                                                                 13 <br />    pBY265.9               pBY130::QRI7 (primer pair 18/19)                                                          This study <br />    pBY270.2               pBAD24::QRI7 (primer pair 18/19)                                                          This study <br />    pBY245.1               pBAD24::tsaD (primer pair 16/17)                                                          This study <br />    pBY343.2               pBAD33=::tsaD (primer pair 22/23); pBAD33= pBAD33 cut EcoRV SmaI,             This study <br />                              partially removing araC PBAD promoter; tsaD constitutively expressed from <br />                              PBAD promoter brought forward primer <br />    pBY344.2               pBAD33=::QRI7 (primer pair 20/21)                                                         This study <br />    pBY346.2               pBY344.2::rbsSUA5Met1 (primer pair 24/26); SUA5 cloned methionine       This study <br />    pBY347.6               pBY344.2::rbsSUA5Met10 (primer pair 25/26); SUA5 cloned second methionine     This study <br />  <br />  <br />  <br />  <br /> 1224    ec.asm.org                                                                                                               Eukaryotic Cell <br />                                                                                                                               Mitochondrial t6A Synthesis <br />  <br />  <br />  <br /> material. All plasmid constructs confirmed sequencing. The clon-    screen. The growth curves presented averages 5 biological repli- <br /> ing procedure protein purification S. cerevisiae Sua5 (Sua5Sc)   cates. Significance determined using 2-way analysis variance <br /> described (11). QRI7 amplified PCR using genomic           (ANOVA) Fisher&#226;&#8364;&#8482;s significant differences (LSD) Prism 6 <br /> DNA S. cerevisiae strain S288C (ATCC) appropriate oligonucle-      (GraphPad). <br /> otides (see Table S1). The 90 nucleotides QRI7, encoding mi-       Preparation bulk tRNA detection t6A. Bulk tRNA pre- <br /> tochondrial targeting sequence, deleted. The gene carried hexahis-   pared double phenol-chloroform extraction, nucleoside prepara- <br /> tidine tag 3= end subsequently cloned pET28a            tions prepared previously described (13). All tRNA extractions <br /> expression vector (Novagen) NcoI NotI restriction sites.            performed triplicate independent cultures. t6A detected <br />      Plasmid construction yeast vivo assays. Plasmid construction    HPLC described previously (27) LC-MS/MS described in <br /> for microscopy based yeast mitochondrion-targeting green         reference 13. The MS/MS fragmentation data synthesized t6A stan- <br /> fluorescent protein (GFP) plasmid, pYX122-mtGFP (25). pYX122-               dard provided Darrell Davis (University Utah) used confirm <br /> mtGFP digested EcoRI BamHI remove preSu9 mito-          presence t6A. <br /> chondrion-targeting sequence. SUA5 BY4742 PCR amplified                tRNA production purification. E. coli tRNAAsn(GUU) and <br /> with oligonucleotides listed Table S1 supplemental material    tRNALys(UUU) overexpressed E. coli XL1-Blue cells (Agilent Tech- <br /> create various forms Sua5-GFP. PCR products digested       nologies) purified described previously (11). For purification, <br /> EcoRI BamHI ligated using T4 DNA ligase (NEB) simi-        RNAs loaded 8 M urea, 12% polyacrylamide gel, band <br /> larly digested pYX122-mtGFP. Plasmids complementation assays            corresponding overexpressed tRNA cut out. tRNA eluted from <br /> listed Table 1 created subcloning components previously     gel elution buffer (EB; 0.1% SDS, 0.5 M ammonium acetate [NH4Ac], <br /> described plasmids. Site-directed mutagenesis performed using       10 mM MgAc, 1 mM EDTA), precipitated, suspended water. To <br /> QuikChange lightning mutagenesis kit (Agilent, Santa Clara, CA) fol-     favor correct folding tRNAs, solution heated 70&#194;&#176;C, cooled <br /> lowing manufacturer&#226;&#8364;&#8482;s recommendations.                                  slowly room temperature, stored &#226;&#171;&#186;80&#194;&#176;C. <br />      Plasmid construction E. coli vivo assays. E. coli tsaD              In vitro assay synthesis t6A-modified tRNA. The vitro <br /> (NP_417536) amplified PCR genomic DNA prepared             assay performed essentially described previously (11). The reaction <br /> E. coli K-12 MG1655 oligonucleotides listed Table S1         performed final volume 50 &#226;?&#174;l containing 4 &#226;?&#174;M Sua5p 5 <br /> supplemental material. The yeast SUA5 (NP_011346) QRI7                  &#226;?&#174;M Qri7p, 4.8 &#226;?&#174;M tRNA, 18.2 &#226;?&#174;M [14C]L-threonine (0.05 mCi, 55 <br /> (NP_010179) genes amplified S. cerevisiae BY4741 genomic          Ci/mol; Hartmann Analytic) reaction buffer (RB). After 40 min of <br /> DNA oligonucleotides listed Table S1. PCR fragments di-        incubation 32&#194;&#176;C, macromolecules precipitated addition of <br /> gested appropriate restriction enzymes cloning pBAD     1 ml trichloroacetic acid 15% (TCA) incubated ice 1 h. <br /> pBAD-derived vectors (26). QRI7 cloned putative mito-       Precipitated material applied prewet glass microfiber GF/F filters <br /> chondrion-targeting sequence (corresponding 30 amino ac-       (Whatman) using vacuum apparatus (Millipore). Filters washed <br />                                                                             3 ml 5% TCA 3 ml 95% ethanol (EtOH) dried. Radio- <br /> ids). The ribosomal binding site and/or restriction sites added to <br />                                                                             activity recorded liquid scintillation analyzer (Packard) av- <br /> primer sequences cloning purposes described Table S1. SUA5 was <br />                                                                             erage counts minute (cpm) 2 min. The incorporated <br /> cloned (M1) second methionine (M10). <br />                                                                             threonine calculated standard curve obtained different <br />      Microscopy. To determine localization Sua5-GFP, yeast were <br />                                                                             concentrations [14C]L-threonine spotted filters (1 pmol &#226;&#171;&#189; 102.8 <br /> cotransformed mitochondrial marker pYX142-RFP of <br />                                                                             cpm). The yield reaction calculated [cpm (assay) &#226;&#171;&#186; cpm <br /> the Sua5-GFP constructs, transformants selected SD&#226;&#171;&#186; <br />                                                                             (background)/102.8]/(pmol tRNA) &#226;&#171;&#187; 100. <br /> his&#226;&#171;&#186;leu plates. For microscopy, cells grown mid-log phase in <br /> dropout media containing 1% glucose 3% glycerol. Cells washed <br /> once water prior analysis 100&#226;&#171;&#187; magnification Leica DM       RESULTS <br /> IRE2 confocal microscope using MetaMorph microscopy automation              Sua5p targeted cytoplasm mitochondria by <br /> and image analysis software (Molecular Devices, Sunnyvale, CA). Decon-      alternate in-frame AUG start sites. In vitro, Qri7p Sua5p can <br /> volution pseudocoloring images performed using Meta-             produce t6A additional proteins, separated <br /> Morph, final figures constructed using Photoshop CS6 (Adobe).      2-kDa molecular size cutoff membrane, noticeably <br />      Recombinant protein expression purification. The expression        t6A produced Qri7p Sua5p separated <br /> and purification recombinant Sua5p S. cerevisiae TsaC, TsaD, <br />                                                                             (the precise increase difficult determine Fig. 3A Wan <br /> TsaB, TsaE E. coli described previously (11). Recombinant <br /> Qri7p expressed E. coli Rosetta2 (DE3) pLysS (Novagen). Protein <br />                                                                             et al. [17]). Close examination SUA5 open reading frame <br /> overexpression induced overnight express instant TB medium           (ORF) revealed in-frame AUG codons located 28 <br /> (Novagen) supplemented 10% glycerol. Cells collected cen-      nucleotides (Fig. 2A), postulated that, vivo, Sua5p was <br /> trifugation, suspended lysis buffer (LBf; 50 mM Tris-HCl, pH 8, 500      functional cytoplasm mitochondria, SUA5 <br /> mM NaCl, 10% glycerol, 0.01% [vol/vol] Triton X-100, 10 mM imidazole,       homolog encoded mtDNA. The mitochondrial targeting <br /> protease inhibitor cocktail [Roche]), sonicated ice. His-tagged      prediction software iPSORT (28) predicted mitochondrion-tar- <br /> proteins soluble fraction purified gravity-flow chroma-    geting sequence isoform started AUG <br /> tography nickel-nitrilotriacetic acid (Ni-NTA) resin column (Qia-      (here referred M1, methionine position 1) did not <br /> gen) according manufacturer&#226;&#8364;&#8482;s recommendations. Fractions in-      predict mitochondrial targeting sequence second AUG <br /> terest pooled, concentrated, injected Superdex 200 column     (M10, methionine position 10 Sua5). This suggested a <br /> (GE Healthcare). Fractions containing pure proteins concentrated       long form Sua5p translated AUG mitochon- <br /> and stored &#226;&#171;&#186;80&#194;&#176;C storage buffer (SB; 50 mM Tris-HCl, pH 8, 500 <br />                                                                             drion-targeted protein, short form Sua5p, translated <br /> mM NaCl, 10% glycerol). <br />      Yeast growth assays. Growth curves performed using Bio- <br />                                                                             second AUG, remain cytoplasmic. <br /> screen C MBR (Oy Growth Curves AB Ltd., Finland) 30&#194;&#176;C maxi-             To test predictions, performed fluorescence micros- <br /> mum shaking. A 250-&#226;?&#174;l culture used well, 5 biological       copy chromosomally encoded Sua5-GFP fusion, con- <br /> replicates used condition. Yeast cultures grown       trol, tested Kae1-GFP Qri7-GFP. Because low abun- <br /> SD&#226;&#171;&#186;his saturation, normalized optical density 600 nm (OD600)    dance of proteins (estimated 600 <br /> of 1, diluted 200 times SD&#226;&#171;&#186;his loaded Bio-       molecules cell Sua5p [20]), signal weak pro- <br />  <br />  <br /> September 2014 Volume 13 Number 9                                                                                                     ec.asm.org 1225 <br />  Thiaville et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Subcellular localization Sua5p. (A) The SUA5 ORF contains in-frame AUG codons located 28 nucleotides. iPSORT (28) predicts Sua5p <br /> translated AUG target mitochondria. The TsaC-like domain Sua5 domain indicated scale. (B) Sua5::GFP localization using <br /> confocal microscopy 100&#226;&#171;&#187; magnification. Wild-type BY4741 transformed plasmids expressing RFP targeted mitochondria Sua5::GFP <br /> fusions. Row 1, control. Row 2, RFP fused Fo-ATPase mitochondrial targeting sequence. Full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>, wild-type Sua5::GFP indicates cytoplasmic and <br /> mitochondrial targeting Sua5p. Colocalization RFP GFP mitochondria indicated yellow merged image. Row 3, residues 10 426 Sua5 <br /> fused GFP [Sua5(10-426)::GFP] targets cytoplasm. Row 4, substitution M10L full-length Sua5 (Sua5-M10L::GFP) enhances localization the <br /> mitochondria. Row 5, 25 residues Sua5p [Sua5(1-25)] sufficient dual targeting. GFP located cytoplasm mitochondria. (6) <br /> Sua5-M10L(1-25)::GFP targets mitochondria. <br />  <br />  <br /> duced images greatly overexposed, did indicate               cytoplasm mitochondria, second AUG is <br /> the mitochondrial localization Sua5p (data shown). To in-                mutated CUG, GFP localized mitochondria, sup- <br /> crease level Sua5p cell, constructed Sua5-GFP                  porting conclusion Sua5p target mitochondria. <br /> fusions control constitutive triosephosphate                       The localization function Sua5-GFP confirmed by <br /> isomerase (TPI) promoter ARS/CEN vector pYX122-                          cotransforming VDC9100 (sua5&#226;&#338;&#172;) pYX142-RFP the <br /> mtGFP (25). The Su9 mitochondrial targeting sequence                         Sua5-GFP plasmids. The targeting Sua5-GFP sua5&#226;&#338;&#172; mimics the <br /> pYX122-mtGFP replaced SUA5 lacking stop codon,                       results seen wild-type BY4741. Full-length Sua5-GFP targeted to <br /> creating plasmid pPCT025. Wild-type S. cerevisiae BY4741                    cytoplasm mitochondria. Sua5(10-426)-GFP targeted ex- <br /> cotransformed pPCT025 pYX142-mtRFP (to specifically                    clusively cytoplasm, Sua5(M10L)-GFP targeted exclusively <br /> <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> mitochondria red fluorescent protein [a gift Agn&#195;&#168;s              mitochondria (see Fig. S1 supplemental material). <br /> Delahodde]), transformants examined 100&#226;&#171;&#187; confo-                         TC-AMP produced cytoplasm sufficient mi- <br /> cal microscopy. The experiment revealed uniform distribution                 tochondrial function. To test TC-AMP shuttled from <br /> GFP cytoplasm foci GFP mitochondria              cytoplasm mitochondria, series plasmids con- <br /> colocalizing mitochondrion-targeted red fluorescent protein                structed encoding Sua5p targeted cytoplasm only, TsaC <br /> (RFP) (Fig. 2B, row 2). We examined 5=-truncated form                 E. coli, Sua5 (YwlC) B. subtilis. To mimic nat- <br /> Sua5p created cloning Sua5(10-426) pYX122-mtGFP, giving                 urally low expression SUA5, PGAL fusions low-copy- <br /> pPCT006. Sua5p starting second Met (M10) localized exclu-                number CEN/ARS plasmid pRS313 constructed. PGAL is <br /> sively cytoplasm (Fig. 2B, row 3). Additionally, created            known leaky, presence glucose (29). Therefore, <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> SUA5 altering second AUG CUG (M10L)                    Sua5(10-426) cloned PGAL pYesDEST52 into <br /> and cloned SUA5(M10L) pYX122-mtGFP, creating pPCT030.                      pRS313 (creating pPCT070), allowing Sua5(10-426) (targeted the <br /> As seen Fig. 2B, row 4, M10L substitution Sua5p enhances              cytoplasm) expressed low levels absence the <br /> the localization mitochondria, eliminating Sua5-GFP               galactose inducer. This eliminates possible interfering growth <br /> cytoplasm. To confirm N terminus Sua5p respon-                  galactose testing growth glycerol carbon <br /> sible targeting GFP mitochondria, cloned 70             source. PGAL::tsaCEc (pPCT074), PGAL::ywlCBs (pPCT075), and <br /> nucleotides SUA5 SUA5(M10L) pYX122-mtGFP, creat-                    PGAL::SUA5Sc (pBY176) constructed using plasmid, <br /> ing pPCT004 pPCT036, respectively. As seen Fig. 2B, rows                 pRS313, backbone. <br /> 5 6, 70 nucleotides SUA5 dual-localized GFP                 Yeast strains mutations SUA5 slow growth in <br />  <br />  <br /> 1226    ec.asm.org                                                                                                                             Eukaryotic Cell <br />                                                                                                                                   Mitochondrial t6A Synthesis <br />  <br />  <br />  <br />  <br /> FIG 3 Complementation sua5&#226;&#338;&#172;. Growth curves performed Bioscreen C. Points represent averages biological replicates. Error bars represent <br /> the standard errors means. pBY176 contains SUA5 plus 200 nt sequence upstream cloned pRS313. pPCT076 contains M1L <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> SUA5. <br /> pPCT078 contains M10L mutation SUA5. (A) Complementation sua5&#226;&#338;&#172; 2% glucose. (B) Growth 2% glucose-containing plates day 4. <br /> Five-microliter drops 10-fold dilutions beginning OD600 0.1. (C) Complementation sua5&#226;&#338;&#172; 2% glycerol. (D) Same panel B, growth on <br /> 2% glycerol-containing plates day 10 examined. <br />  <br />  <br />  <br /> glucose-containing media utilize nonfermentable                 SUA5-containing plasmids effect growth of <br /> sugar alcohol glycerol sole carbon source (13, 30). As shown           BY4741 using glucose glycerol carbon source (Fig. 3). The <br /> Fig. S2 supplemental material, supplying PGAL::SUA5(10-426)             expression SUA5 AUG second AUG mu- <br /> PGAL::tsaCEc, PGAL:: ywlCBs, PGAL::SUA5Sc trans suppressed               tated complemented slow growth sua5&#226;&#338;&#172; levels <br /> the growth defect sua5&#226;&#338;&#172; strain using glucose carbon                wild-type SUA5 glucose used carbon source (Fig. <br /> source, implying corresponding proteins functional                 3A B). However, glycerol used carbon source, <br /> the cytoplasm. However, PGAL::SUA5Sc restored growth                   expression wild-type SUA5 SUA5 second AUG <br /> glycerol, albeit longer lag phase.                                      mutated [encoding Sua5(M10L)] complemented lack growth <br />    These results suggest proteins produced TC-               sua5&#226;&#338;&#172; mutant (Fig. 3C D). The expression SUA5 with <br /> AMP cytoplasm suppress glucose growth defect                 mutation AUG did allow complementation. <br /> that TC-AMP shuttled mitochondria,                         This complementation assay corroborates GFP localization <br /> seen inability PGAL::SUA5(10-426), PGAL::tsaCEc, PGAL::           data presented earlier. Sua5p dual targeted cyto- <br /> ywlCBs variants grow glycerol.                                           plasm mitochondria use alternative trans- <br />    Sua5p targeted mitochondria grow glyc-                 lational starts, translation AUG required for <br /> erol. To directly test mitochondrion-targeting Sua5p re-                localization mitochondria. <br /> quired mitochondrial function, series plasmids pre-                  In vitro, Qri7p function Sua5p TsaCEc. <br /> pared use native SUA5 promoter drive constructs                    Wan et al. showed Qri7p Sua5p S. cerevisiae are <br /> targeting Sua5p cytoplasm, mitochondria, both. Site-                 necessary sufficient synthesize t6A vitro, repro- <br /> directed mutagenesis performed pBY176 (SUA5 plus 200                    duced result using different assay (10, 11) (Fig. 4A). The <br /> nt upstream sequence cloned low-copy-number plasmid                   assay measures incorporation radioactive threonine into <br /> pRS313) mutate methionine leucine (M1L), creating              TCA-precipitated material corresponds formation of <br /> pPCT076, second methionine leucine (M10L), creating                  t6A-modified tRNA (11). As shown Fig. 4A, significant synthe- <br /> pPCT078. All SUA5-containing plasmids, pa-                sis t6A obtained ATP-dependent manner when <br /> rental pRS313, transformed BY4741 sua5&#226;&#338;&#172; mu-                    Sua5p Qri7p present reaction mixture. In <br /> tant selection SD&#226;&#171;&#186;his agar.                                            contrast, background signal levels measured using <br />    Mitochondrial function assayed testing growth                 Sua5p Qri7p alone, indicating necessary the <br /> the nonfermentable carbon source glycerol. The expression               reaction place. We asked heterologous system <br />  <br />  <br /> September 2014 Volume 13 Number 9                                                                                                         ec.asm.org 1227 <br />  Thiaville et al. <br />  <br />  <br />  <br />  <br /> FIG 4 In vitro synthesis t6A. (A) Qri7 Sua5 proteins S. cerevisiae catalyze formation t6A ATP-dependent fashion. The reaction mixture <br /> contained 4 &#226;?&#174;M Sua5p 5 &#226;?&#174;M Qri7p, 4.8 &#226;?&#174;M E. coli tRNALysUUU, 18.2 &#226;?&#174;M [14C]L-threonine reaction buffer (RB). After 40 min incubation 32&#194;&#176;C, <br /> macromolecules precipitated trichloroacetic acid, radioactivity contained tRNA recorded using liquid scintillation analyzer. Error bars <br /> indicate standard deviations means obtained independent experiments. (B) Heterologous composed TsaC Qri7p is <br /> functional vitro. Each reaction mixture contained 3 &#226;?&#174;M proteins, indicated x axis, 4.8 &#226;?&#174;M E. coli tRNAAsn(GUU), 18.2 &#226;?&#174;M [14C]L-threonine in <br /> reaction buffer (RB). After 35 min incubation 37&#194;&#176;C, macromolecules precipitated trichloroacetic acid. Radioactivity contained tRNA was <br /> recorded using liquid scintillation analyzer. Error bars indicate standard deviations means obtained independent experiments. <br />  <br />  <br /> composed TsaCEc, bacterial homolog Sua5p, Qri7p                   press yeast genes. These suppressors appear E. <br /> could catalyze t6A synthesis vitro. As shown Fig. 4B, TsaC            coli tsaD gene expressed vector (see Fig. S3; sup- <br /> Qri7p result 83% modified tRNA compared                         pressors seen inset). Additionally, low level comple- <br /> TsaCDBE control, indicating Qri7/TsaCEc combination                 mentation seen presence Qri7p condi- <br /> functional vitro.                                                            tions small chromosomal E. coli tsaD gene <br />     In vivo, Qri7p requires Sua5p does function                    produced. The diameter halos aTc-dependent <br /> TsaCEc. Surprisingly, contrast vitro results,             growth slightly larger PTET::tsaD strain expressing <br /> that expression Qri7p failed complement essential               Qri7p vector control (see Fig. S3). <br /> <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> E. coli strain expressing tsaD. As shown Fig.                The ratio Qri7p Sua5p critical allowing for <br /> 5A, E. coli PTET::tsaD strain transformed pBAD24 deriv-                complementation PTET::tsaD strain, expressing QRI7 <br /> atives expressing QRI7 (pBY270.2) did grow absence                SUA5 lower-copy-number plasmid (P15a ori) a <br /> aTc (required induction PTET) presence arabi-              constitutive promoter (PBLA) did allow growth PTET:: <br /> nose. When QRI7 SUA5 expressed operon un-                      tsaD strain absence aTc (Fig. 5C) did the <br /> der PBAD promoter (pBY265.9) (a diagram genetic con-                 absence induction high-copy-number plasmid pBAD24 <br /> structs Fig. 5B), partial complementation                       (ColE1 ori) (Fig. 5A). Expressing E. coli tsaD plasmid <br /> (reduced growth smaller colonies compared complemen-                     (pBY343.2) did allow growth aTc, establishing the <br /> tation TsaD) observed (Fig. 5A), suggesting Qri7p                 PBLA promoter functional (Fig. 5C). To ensure SUA5(M1) and <br /> and TsaC able function vivo, Qri7p             SUA5(M10) functional E. coli, plasmids PBLA::QRI7:: <br /> Sua5p able sustain partial growth. The plasmid             SUA5(M1) (pBY346.2) PBLA::QRI7::SUA5(M10) (pBY347.6) <br /> expressing QRI7 SUA5 transformed PTET::tsaC                    tested complementation PTET::tsaC strain. In this <br /> strain, allowed grow absence aTc, establishing           strain background, plasmids allowed growth ab- <br /> that SUA5 functional setup. Interestingly, complemen-                sence aTc, establishing SUA5(M1) SUA5(M10) are <br /> tation robust absence aTc arabinose                 functional E. coli (Fig. 5D). <br /> in strain expressing yeast SUA5 QRI7 genes compared <br /> to strain expressing E. coli tsaD gene.                                 DISCUSSION <br />     Similar results obtained halo assays, diluted               In eukaryotes, tRNA modification enzymes targeted cytoplasm <br /> cultures spread plates 10-&#226;?&#174;l drop aTc                  organelles encoded single nuclear gene. The dual <br /> added center surface plate (see Fig. S3             targeting modification enzymes observed several <br /> supplemental material). The aTc diffuses plate                  universally conserved tRNA modifications. Six yeast <br /> provides gradient induction PTET. Partial complementa-                 pseudouridylation enzymes Mod5p, enzyme responsi- <br /> tion observed QRI7 SUA5 expressed                  ble insertion N6-isopentenylasenosine (i6A37), targeted <br /> an operon PBAD, halo test, suppressors clearly                 cytoplasm mitochondria (31, 32). Similarly, <br /> appear zone lower levels aTc strain ex-              Trm5p, tRNA (guanine-N1-)-methyltransferase modifies mi- <br />  <br />  <br /> 1228    ec.asm.org                                                                                                                              Eukaryotic Cell <br />                                                                                                                                        Mitochondrial t6A Synthesis <br />  <br />  <br />  <br />  <br /> FIG 5 Qri7 TsaC able function vivo, Qri7 Sua5 able sustain growth. (A) Dilution drops E. coli PTET:: <br /> tsaD(pBAD24) strain derivatives expressing QRI7 did grow absence aTc (required induction PTET) presence arabinose. When <br /> SUA5 QRI7 expressed operon PBAD promoter, complementation observed, suggesting Qri7 TsaCEc able function <br /> together vivo, Qri7 Sua5 able sustain partial growth. The E. coli PTET::tsaC(PBAD::QRI7) strain does grow aTc, the <br /> PBAD::SUA5::QRI7 strain grow aTc, validating function operon. (B) Diagram construction inducible strains plasmids. (C) Direct <br /> transformation low-copy-number plasmids constitutive expression PBLA PTET::tsaD strain plating aTc. The Qri7::Sua5(M1) or <br /> Qri7::Sua5(M10) synthetic operons sustain growth absence aTc. Providing tsaD trans using setup allows growth. (D) Direct <br /> transformation low-copy-number plasmids constitutive expression PBLA PTET::tsaC strain plating aTc. Sua5 translated from <br /> either second AUG support growth E. coli absence aTc. <br />  <br />  <br /> tochondrial tRNA methylates guanosine position 37                     &#226;&#171;&#185;4 (Fig. 2A). The presence in-frame AUGs N termi- <br /> (m1G37) tRNAs nucleus, dual localization. It           nus TsaC/Sua5 proteins conserved Arabidopsis thaliana <br /> a long form exclusively mitochondria N-termi-                 Homo sapiens eukaryotic genomes sequenced <br /> nally truncated form (translated second internal, in-frame                 date (see Fig. S4 supplemental material), suggesting this <br /> AUG start site) exclusively nucleus (22, 33, 34). The                dual-targeting mechanism conserved. In plants, TsaC be <br /> Sua5-GFP data presented strongly suggest similar scenario              targeted mitochondria chloroplasts addition to <br /> Sua5p. The nucleotide context surrounding AUGs                    localized cytoplasm, copy TsaC, located <br /> nonoptimum Kozak consensus, U &#226;&#171;&#186;3 &#226;&#171;&#185;4                    nuclear genome, plant genomes analyzed. <br /> first AUG (in yeast, U &#226;&#171;&#186;3 reduces expression 2-fold [35])              This prediction experimentally validated. <br /> stronger Kozak sequence second AUG, G &#226;&#171;&#186;3 A                     We reproduced results reported Wan et al. showing that <br />  <br />  <br /> September 2014 Volume 13 Number 9                                                                                                               ec.asm.org 1229 <br />  Thiaville et al. <br />  <br />  <br />  <br /> t6A-modified tRNA formed vitro joint action Sua5p        (17), indicating t6A modification required normal <br /> and Qri7p ATP-dependent fashion (17). The maximum                  functioning mitochondria. Our data indicate TC- <br /> amount modified tRNA 16% assay conditions;           AMP produced cytoplasm transported (in sufficient <br /> is significant, high reported Wan      amounts) mitochondrial matrix, line the <br /> and colleagues (80% modified tRNA). However, tRNA sub-            unstable nature intermediate. <br /> strate used hypomodified tRNA overexpressed               In S. cerevisiae, Qri7p fully rescue t6A synthesis the <br /> in E. coli. Unfortunately, know proportion tRNAs        kae1&#226;&#338;&#172; background, partially complement telom- <br /> lacking t6A modification. The examination kinetics re-         ere-length defect, indicating KEOPS proteins or <br /> action revealed comparable maximal rate modification: 0.7%          functions cell (17). We similar observa- <br /> modified tRNA/min versus 0.6% modified tRNA/min reaction             tion E. coli cells, cells relying Qri7/Sua5 t6A <br /> reported Wan et al. (as judged Fig. 2G Wan et al.;       synthesis healthy cells expressing TsaD. Therefore, <br /> see Fig. S4 supplemental material). However,         data suggest that, bacteria, TsaD TsaDEB complex <br /> the tRNA reaction modified (see Materials        functions cell replaced Qri7p, explain- <br /> Methods; Fig. S5); therefore, vitro reaction per-        ing observed partial complementation. Interestingly, genomic <br /> forms equally better reported Wan col-          context data implicate TsaDEB synthesis peptidoglycan <br /> leagues. The modified tRNA increased linear fashion         bacteria (38). We pursuing hypothesis additional <br /> for 30 min incubation, rate decreases, per-        genetic experiments. <br /> haps instability proteins. In addition, observed        The Sua5p localization phenotype data presented here, in <br /> a similar maximal rate cytoplasmic composed            combination vitro data published previously (17), <br /> Sua5p KEOPS proteins (0.6% modified tRNA/min), indicat-              reproduced current study, firmly establish that, <br /> ing mitochondrial cytoplasmic systems perform              yeast, Sua5p Qri7p colocalize mitochondria are <br /> equally vitro (see Fig. S5). This observation line    proteins required introduce t6A modifications mi- <br /> the vivo data demonstrated Sua5p Qri7p fully             tochondrial tRNAs, Sua5p localize mitochondria <br /> restore t6A levels kae1&#226;&#338;&#172; mutant cells S. cerevisiae (17).          growth nonfermentable carbon source glycerol. This <br />     We observed heterologous TsaC Qri7p           implies TC-AMP transported cytoplasm in <br /> performs vitro (80% t6A formed compared con-           sufficient amounts mitochondrial function TC-AMP <br /> trol) (Fig. 4B) does complement TsaD mutant vivo.         produced organelle. <br /> These data indicate TC-AMP intermediate produced by <br /> TsaC captured Qri7p vitro vivo. It likely     ACKNOWLEDGMENTS <br /> that vivo conditions, TC-AMP intermediate            This work supported National Institutes Health (grant num- <br /> sequestered molecules crowded interior cell       ber R01 GM70641 V.D.C.-L.) Agence Nationale Recher- <br /> (such nucleotide-binding proteins) destroyed             che (ANR-09-BLAN-0349 P.F.). P.C.T. funded Cha- <br /> Qri7p captures it. Therefore, data suggest normal t6A       teaubriand Fellowship French Embassy United States. L.P <br /> synthesis cell requires TC-AMP intermediate           received Ph.D. fellowship ENS Lyon. <br /> delivered Kae1p/TsaD/Qri7p active site. Such delivery               We thank Agn&#195;&#168;s Delahodde sharing pYX142-mtRFP prior pub- <br /> mechanism achieved direct contact               lication Bruno Collinet Wenhua Zhang providing with <br />                                                                          runoff transcripts tRNA genes. <br /> TsaC/Sua5p Kae1p/TsaD/Qri7p, followed channeling <br /> of TC-AMP TsaC/Sua5p active site Kae1p/TsaD/ <br />                                                                          REFERENCES <br /> Qri7p. This hypothesis strongly supported data showing <br />                                                                           1. Agris PF, Vendeix FA, Graham PWD. 2007. tRNA&#226;&#8364;&#8482;s wobble decoding of <br /> that Qri7p complement TsaD mutant vivo                  genome: 40 years modification. J. Mol. Biol. 366:1&#226;&#8364;&#8220;13. http://dx.doi <br /> Sua5p present, TsaC functional present            .org/10.1016/j.jmb.2006.11.046. <br /> the cell. Further support interpretation data comes       2. Grosjean H, De Henau S, Doi T, Yamane A, Ohtsuka E, Ikehara M, <br /> from structural work obtained protein TobZ (36).                 Beauchemin N, Nicoghosian K, Cedergren R. 1987. The vivo stability, <br />                                                                              maturation aminoacylation anticodon-substituted Escherichia coli <br /> TobZ highly relevant t6A biosynthesis composed <br />                                                                              initiator methionine tRNAs. Eur. J. Biochem. 166:325&#226;&#8364;&#8220;332. http://dx.doi <br /> of TsaC-like domain Kae1-like domain. Parthier col-                .org/10.1111/j.1432-1033.1987.tb13518.x. <br /> leagues showed TsaC-like domain catalyzes formation          3. Phizicky EM, Alfonzo JD. 2010. Do modifications benefit tRNAs? <br /> of intermediate carbamoyladenylate channeled             FEBS Lett. 584:265&#226;&#8364;&#8220;271. http://dx.doi.org/10.1016/j.febslet.2009.11.049. <br /> the Kae1-like domain, site carbamoyl transfer (36). Since          4. D&#195;&#173;az I, Ehrenberg M. 1991. ms2i6A deficiency enhances proofreading in <br />                                                                              translation. J. Mol. Biol. 222:1161&#226;&#8364;&#8220;1171. http://dx.doi.org/10.1016/0022 <br /> both TsaC/Sua5 TsaD/Kae1/Qri7 families constant                  -2836(91)90599-2. <br /> part t6A-forming systems, likely systems operate    5. Morin A, Auxilien S, Senger B, Tewari R, Grosjean H. 1998. Structural <br /> by channeling TC-AMP. In line prediction, stable                requirements enzymatic formation threonylcarbamoyladenosine <br /> interaction TsaC TsaD reported (10); how-               (t6A) tRNA: vivo study Xenopus laevis oocytes. RNA 4:24 &#226;&#8364;&#8220;37. <br />                                                                           6. Phizicky EM, Hopper AK. 2010. tRNA biology charges front. <br /> ever, global interactome data Sua5p did identify Kae1p or <br />                                                                              Genes Dev. 24:1832&#226;&#8364;&#8220;1860. http://dx.doi.org/10.1101/gad.1956510. <br /> Qri7p proteins interactants, suggesting proteins        7. Elkins BN, Keller EB. 1974. The enzymatic synthesis N-(purin-6- <br /> engaged transient interaction (37). Further experimental work             ylcarbamoyl)threonine, anticodon-adjacent <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> transfer ribo- <br /> is needed order test prediction.                                  nucleic acid. Biochemistry 13:4622&#226;&#8364;&#8220; 4628. http://dx.doi.org/10.1021 <br />     We demonstrated cells containing cytoplasmic ver-            /bi00719a024. <br />                                                                           8. Sibler AP, Dirheimer G, Martin RP. 1985. Yeast mitochondrial tRNAIle <br /> sion Sua5p presented mitochondrial defect, cells            tRNAMetm: nucleotide sequence codon recognition patterns. <br /> were unable grow nonfermentable carbon source glyc-                Nucleic Acids Res. 13:1341&#226;&#8364;&#8220;1345. http://dx.doi.org/10.1093/nar/13.4 <br /> erol. The phenotype observed qri7&#226;&#338;&#172; S. cerevisiae             .1341. <br />  <br />  <br />  <br /> 1230    ec.asm.org                                                                                                                       Eukaryotic Cell <br />                                                                                                                                           Mitochondrial t6A Synthesis <br />  <br />  <br />  <br />  9. K&#195;&#182;rner A, S&#195;&#182;ll D. 1974. N-(purin-6-ylcarbamoyl)threonine: biosynthesis             transformed high efficiency using LiAc/SS carrier DNA/PEG <br />     vitro transfer RNA enzyme purified Escherichia coli. FEBS         method. Nat. Protoc. 2:1&#226;&#8364;&#8220; 4. http://dx.doi.org/10.1038/nprot.2007.17. <br />     Lett. 39:301&#226;&#8364;&#8220;306. http://dx.doi.org/10.1016/0014-5793(74)80135-3.            25.   Westermann B, Neupert W. 2000. Mitochondria-targeted green fluores- <br /> 10. Deutsch C, El Yacoubi B, Cr&#195;&#169;cy-Lagard V, Iwata-Reuyl D. 2012.                   cent proteins: convenient tools study organelle biogenesis in <br />     Biosynthesis threonylcarbamoyl adenosine (t6A), universal tRNA nu-            Saccharomyces cerevisiae. Yeast 16:1421&#226;&#8364;&#8220;1427. http://dx.doi.org/10.1002 <br />     cleoside. J. Biol. Chem. 287:13666 &#226;&#8364;&#8220;13673. http://dx.doi.org/10.1074/jbc           /1097-0061(200011)16:15&#226;&#172;?1421::AID-YEA624&#226;&#172;&#381;3.0.CO;2-U. <br />     .M112.344028.                                                                26.   Guzman LM, Belin D, Carson MJ, Beckwith J. 1995. Tight regulation, <br /> 11. Perrochia L, Crozat E, Hecker A, Zhang W, Bareille J, Collinet B, van              modulation, high-level expression vectors containing arabi- <br />     Tilbeurgh H, Forterre P, Basta T. 2013. In vitro biosynthesis univer-         nose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220; 4130. <br />     sal t6A tRNA modification Archaea Eukarya. Nucleic Acids Res.         27.   Pomerantz SC, McCloskey JA. 1990. Analysis RNA hydrolyzates by <br />     41:1953&#226;&#8364;&#8220;1964. http://dx.doi.org/10.1093/nar/gks1287.                               liquid chromatography-mass spectrometry. Methods Enzymol. 193:796 &#226;&#8364;&#8220; <br /> 12. El Yacoubi B, Hatin I, Deutsch C, Kahveci T, Rousset J-P, Iwata-Reuyl              824. http://dx.doi.org/10.1016/0076-6879(90)93452-Q. <br />     D, Murzin AG, Cr&#195;&#169;cy-Lagard V. 2011. A role universal Kae1/        28.   Bannai H, Tamada Y, Maruyama O, Nakai K, Miyano S. 2002. Extensive <br />     Qri7/YgjD (COG0533) family tRNA modification. EMBO J. 30:882&#226;&#8364;&#8220;                   feature detection N-terminal protein sorting signals. Bioinformatics <br />     893. http://dx.doi.org/10.1038/emboj.2010.363.                                     18:298 &#226;&#8364;&#8220;305. http://dx.doi.org/10.1093/bioinformatics/18.2.298. <br /> 13. El Yacoubi B, Lyons B, Cruz Y, Reddy R, Nordin B, Agnelli F, Wil-            29.   Kristoffersen P, Jensen GB, Gerdes K, Piskur J. 2000. Bacterial toxin- <br />     liamson JR, Schimmel P, Swairjo MA, Cr&#195;&#169;cy-Lagard V. 2009. The                   antitoxin gene containment control yeast cells. Appl. Environ. <br />     universal YrdC/Sua5 family required formation threonylcar-           Microbiol. 66:5524 &#226;&#8364;&#8220;5526. http://dx.doi.org/10.1128/AEM.66.12.5524 <br />     bamoyladenosine tRNA. Nucleic Acids Res. 37:2894 &#226;&#8364;&#8220;2909. http://dx               -5526.2000. <br />     .doi.org/10.1093/nar/gkp152.                                                 30.   Lin CA, Ellis SR, True HL. 2010. The Sua5 protein essential normal <br /> 14. Downey M, Houlsworth R, Maringele L, Rollie A, Brehme M, Galicia S,                translational regulation yeast. Mol. Cell. Biol. 30:354 &#226;&#8364;&#8220;363. http://dx.doi <br />     Guillard S, Partington M, Zubko MK, Krogan NJ, Emili A, Greenblatt                 .org/10.1128/MCB.00754-09. <br />     JF, Harrington L, Lydall D, Durocher D. 2006. A genome-wide screen           31.   Gillman EC, Slusher LB, Martin NC, Hopper AK. 1991. MOD5 trans- <br />     identifies evolutionarily conserved KEOPS complex telomere reg-           lation initiation sites determine N6-isopentenyladenosine modification of <br />     ulator. Cell 124:1155&#226;&#8364;&#8220;1168. http://dx.doi.org/10.1016/j.cell.2005.12.044.          mitochondrial cytoplasmic tRNA. Mol. Cell. Biol. 11:2382&#226;&#8364;&#8220;2390. <br /> 15. Kisseleva-Romanova E, Lopreiato R, Baudin-Baillieu A, Rousselle J-C,         32.   Behm-Ansmant I, Grosjean H, Massenet S, Motorin Y, Branlant C. <br />     Ilan L, Hofmann K, Namane A, Mann C, Libri D. 2006. Yeast homolog                  2004. Pseudouridylation position 32 mitochondrial cytoplasmic <br />     cancer-testis antigen defines new transcription complex. EMBO J.            tRNAs requires distinct enzymes Saccharomyces cerevisiae. J. Biol. <br />                                                                                        Chem. 279:52998 &#226;&#8364;&#8220;53006. http://dx.doi.org/10.1074/jbc.M409581200. <br />     25:3576 &#226;&#8364;&#8220;3585. http://dx.doi.org/10.1038/sj.emboj.7601235. <br />                                                                                  33.   Lee C, Kramer G, Graham DE, Appling DR. 2007. Yeast mitochondrial <br /> 16. Lauhon CT. 2012. Mechanism N6-threonylcarbamoyladenonsine <br />                                                                                        initiator tRNA methylated guanosine 37 Trm5-encoded tRNA <br />     (t6A) biosynthesis: isolation characterization intermediate <br />                                                                                        (guanine-N1-)-methyltransferase. J. Biol. Chem. 282:27744 &#226;&#8364;&#8220;27753. http: <br />     threonylcarbamoyl-AMP. Biochemistry 51:8950 &#226;&#8364;&#8220; 8963. http://dx.doi.org <br />                                                                                        //dx.doi.org/10.1074/jbc.M704572200. <br />     /10.1021/bi301233d. <br />                                                                                  34.   Bj&#195;&#182;rk GR, Jacobsson K, Nilsson K, Johansson MJ, Bystr&#195;&#182;m AS, Persson <br /> 17. Wan LCK, Mao DYL, Neculai D, Strecker J, Chiovitti D, Kurinov I, <br />                                                                                        OP. 2001. A primordial tRNA modification required evolution of <br />     Poda G, Thevakumaran N, Yuan F, Szilard RK, Lissina E, Nislow C,                   life? EMBO J. 20:231&#226;&#8364;&#8220;239. http://dx.doi.org/10.1093/emboj/20.1.231. <br />     Caudy AA, Durocher D, Sicheri F. 2013. Reconstitution character-         35.   Romanos MA, Scorer CA, Clare JJ. 1992. Foreign gene expression yeast: a <br />     ization eukaryotic N6-threonylcarbamoylation tRNA using mini-              review. Yeast 8:423&#226;&#8364;&#8220; 488. http://dx.doi.org/10.1002/yea.320080602. <br />     mal enzyme system. Nucleic Acids Res. 41:6332&#226;&#8364;&#8220; 6346. http://dx.doi.org       36.   Parthier C, G&#195;&#182;rlich S, Jaenecke F, Breithaupt C, Br&#195;&#164;uer U, Fandrich U, <br />     /10.1093/nar/gkt322.                                                               Clausnitzer D, Wehmeier UF, B&#195;&#182;ttcher C, Scheel D, Stubbs MT. 2012. <br /> 18. Sibler AP, Dirheimer G, Martin RP. 1986. Codon reading patterns                 The O-carbamoyltransferase TobZ catalyzes ancient enzymatic reac- <br />     Saccharomyces cerevisiae mitochondria based sequences mitochon-              tion. Angew. Chem. Int. Ed. Engl. 51:4046 &#226;&#8364;&#8220; 4052. http://dx.doi.org/10 <br />     drial tRNAs. FEBS Lett. 194:131&#226;&#8364;&#8220;138. http://dx.doi.org/10.1016/0014                .1002/anie.201108896. <br />     -5793(86)80064-3.                                                            37.   Yu H, Braun P, Yildirim MA, Lemmens I, Venkatesan K, Sahalie J, <br /> 19. Oberto J, Breuil N, Hecker A, Farina F, Brochier-Armanet C, Culetto E,             Hirozane-Kishikawa T, Gebreab F, Li N, Simonis N, Hao T, Rual J <br /> </body></html>